# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N

A. 510(k) Number:

K123753

# B. Purpose for Submission:

The IMDx VanR for Abbott m2000 assay was submitted to obtain a substantial equivalence determination for IMDx VanR assay on the Abbott $m 2 0 0 0$ platform using peri-rectal swab, or rectal swab, or stool specimens.

# C. Measurand:

DNA target sequences encoding the vanA and vanB genes which are associated with vancomycin resistance.

# D. Type of Test:

Qualitative nuclei acid amplification test using real-time PCR technology to amplify and detect the vanA and vanB genes.

# E. Applicant:

Intelligent Medical Devices, Inc.

F. Proprietary and Established Names: IMDx VanR for Abbott m2000

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640, Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code: NIJ- System, test, genotypic detection, resistant markers, Enterococcus species

OOI- Nucleic acid amplification systems, real time

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IMDx VanR for Abbott $m 2 0 0 0$ assay is an in vitro diagnostic assay that uses polymerase chain reaction (PCR) amplification for the qualitative detection of nucleic acids encoding the vancomycin resistance genes vanA and/or vanB. The assay is performed directly on human peri-rectal swabs, rectal swabs, or stool specimens from patients at risk for Vancomycin Resistant Enterococcus (VRE) colonization. The IMDx VanR for Abbott $m 2 0 0 0$ assay detects the presence of vanA and vanB genes that can be associated with vancomycin-resistant enterococci. The IMDx VanR for Abbott $m 2 0 0 0$ assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. The IMDx VanR for Abbott $m 2 0 0 0$ assay is not intended to diagnose VRE infection nor to guide or monitor treatment of infection. Culture methods are necessary to recover organisms for epidemiology typing and confirmation testing.

# 2. Indication(s) for use:

The IMDx VanR for Abbott $m 2 0 0 0$ assay is an in vitro diagnostic assay that uses polymerase chain reaction (PCR) amplification for the qualitative detection of nucleic acids encoding the vancomycin resistance genes vanA and/or vanB. The assay is performed directly on human peri-rectal swabs, rectal swabs, or stool specimens from patients at risk for Vancomycin Resistant Enterococcus (VRE) colonization. The IMDx VanR for Abbott $m 2 0 0 0$ assay detects the presence of vanA and vanB genes that can be associated with vancomycin-resistant enterococci. The IMDx VanR for Abbott $m 2 0 0 0$ assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. The IMDx VanR for Abbott $m 2 0 0 0$ assay is not intended to diagnose VRE infection nor to guide or monitor treatment of infection. Culture methods are necessary to recover organisms for epidemiology typing and confirmation testing.

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: Abbott $m 2 0 0 0 s p$ : sample preparation and reagent mixing

Abbott $m 2 0 0 0 r t$ : amplification reaction and detection

# I. Device Description:

The IMDx VanR for Abbott m2000 assay is a qualitative, real-time PCR-based assay that targets regions unique to the vancomycin-resistance genes vanA and vanB. These resistance genes may be associated with the presence of vancomycin resistant Enterococcus (VRE). Specimen preparation and reagent mixing occurs on the Abbott $m 2 0 0 0 s p$ instrument, while amplification and detection occurs on the Abbott $m 2 0 0 0 r t$ instrument. The Abbott $m 2 0 0 0 r t$ application software (v1.8) monitors the fluorescence emitted by each fluorescent probe, interprets all data, and provides a final result at the end of the cycling program. Differentiation of vanA from vanB is achieved by labeling the oligonucleotide probes with different colored fluorescent dyes.

A specimen processing control is introduced into each specimen during the sample extraction. This control is co-extracted with the specimen and co-amplified in the same PCR reaction as the vanA and vanB targets and serves to demonstrate that the entire assay process has proceeded within specification. Positive and negative controls are included with each run to ensure the integrity of the system.

# IMDx VanR for Abbott m2000 Kit

The IMDx VanR for Abbott m2000 assay consists of two reagent kits packaged together: a separate box for the Amplification Reagent Kit and a separate box for the Control Kit:

<table><tr><td colspan="2">1) IMDx VanR for Abbott m2000 Amplification Reagent Kit Each Amplification Reagent Kit contains two types of items:</td></tr><tr><td colspan="2">a) IMDx Process Control-A:</td></tr><tr><td colspan="2">4 vials, 0.6 mL per vial of inactivated bacteria in a buffered solution.</td></tr><tr><td colspan="2"> IMDx Amplification Reagent Packs:</td></tr><tr><td colspan="2">•4 Amplification Reagent Packs, 24 tests/pack.</td></tr><tr><td colspan="2">Each Amplification Reagent Pack contains:* 1 vial (0.408 mL) IMDx VanR for Abbott m2000 Amplification Reagent</td></tr><tr><td colspan="2">consisting of synthetic oligonucleotides in a buffered solution, located in position 1 of the reagent pack.</td></tr></table>

\*Each Amplification Reagent Pack, although capable of containing up to three reagent vials, contains only two reagent vials in each pack. These vials are located in positions 1 and 3, as marked on the Reagent Pack; position 2, in the middle of the pack, is empty.

<table><tr><td>2) IMDx VanR for Abbott m2000 Control Kit Each Control Kit contains two types of items:</td><td></td></tr><tr><td rowspan="3">a) IMDx Negative Control-A:</td><td></td></tr><tr><td></td></tr><tr><td rowspan="3">6 tubes (2.5 mL per tube) containing a buffered solution with carrier DNA isolated from</td></tr><tr><td>Bacteroides sp.</td></tr><tr><td>b) IMDx VanR for Abbott m2000 Positive Control</td></tr><tr><td rowspan="3">•6 tubes (2.5 mL per tube) containing a mixture of synthetic vanA and vanB DNA in</td><td></td></tr><tr><td></td></tr><tr><td>buffer.</td></tr></table>

# Interpretation of Results

The decision algorithm for the IMDx VanR for Abbott $m 2 0 0 0$ assay is embedded in the Abbott $m 2 0 0 0 r t$ application software (v1.8). The interpretation of assay results is provided by analyte result and IC result.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=5>Reported Output</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>ProcessControl</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Result Call</td><td rowspan=1 colspan=1>vanA detected</td><td rowspan=1 colspan=1>vanB detected</td><td rowspan=1 colspan=1>vanA and vanBdetected</td><td rowspan=1 colspan=1>vanA and vanBnot detected</td><td rowspan=1 colspan=1>Invalid;no resultreported</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD GeneOhm™ VanR Assay

2. Predicate 510(k) number(s): K102416

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">IMDx VanR forAbbott m2000</td><td colspan="1" rowspan="1">BD GeneOhmTM VanRAssay</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The IMDx VanR for Abbottm2000 assay is an in vitrodiagnostic assay that usespolymerase chain reaction(PCR) amplification for thequalitative detection ofnucleic acids encoding thevancomycin resistancegenes vanA and/or vanB.The assay is performeddirectly on human peri-rectal swabs, rectal swabs,</td><td colspan="1" rowspan="1">The BD GeneOhmTM VanRAssay is a qualitative invitro test for the rapiddetection of vancomycin-resistance (vanA and vanB)genes directly fromperianal or rectal swabs.The BD GeneOhmTM VanRAssay detects the presenceof the vanA and vanB genesthat can be associated withvancomycin-resistant</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">or stool specimens frompatients at risk forVancomycin ResistantEnterococcus (VRE)colonization. The IMDxVanR for Abbott m2000assay detects the presenceof vanA and vanB genes thatcan be associated withvancomycin-resistantenterococci. The IMDxVanR for Abbott m2000assay can be used as an aidto identify, prevent andcontrol vancomycin-resistant colonization inhealthcare settings. TheIMDx VanR for Abbottm2000 assay is not intendedto diagnose VRE infectionnor to guide or monitortreatment of infection.Culture methods arenecessary to recoverorganisms for epidemiologytyping and confirmationtesting.</td><td colspan="1" rowspan="1">enterococci (VRE). Theassay is performed on anautomated real-time PCRinstrument with perianal orrectal swabs fromindividuals at risk for VREcolonization. The BDGeneOhm™M VanR Assaycan be used as an aid toidentify, prevent andcontrol vancomycin-resistant colonization inhealthcare settings. The BDGeneOhmTM VanR Assayis not intended to diagnoseVRE infections nor toguide or monitor treatmentfor VRE infections.Concomitant cultures arenecessary to recoverorganisms forepidemiological typing,susceptibility testing andfor further confirmatoryidentification.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">peri-rectal swab, rectalswab, or stool specimens</td><td colspan="1" rowspan="1">peri-rectal swab orrectal swab</td></tr><tr><td colspan="1" rowspan="1">Type of test</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay format</td><td colspan="1" rowspan="1">Real-time PCR nucleic acidamplification</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Mode of detection</td><td colspan="1" rowspan="1">Detection using targetspecific hybridization withFluorogenic probes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Targets detected</td><td colspan="1" rowspan="1">vanA and vanB</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">IMDx VanR forAbbott m2000</td><td colspan="1" rowspan="1">BD GeneOhmTM VanRAssay</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Assay uses the Abbottm2000 System</td><td colspan="1" rowspan="1">Assay uses the CepheidSmartCycler System</td></tr><tr><td colspan="1" rowspan="1">Interpretation of TestResults</td><td colspan="1" rowspan="1">Abbott m2000rt applicationsoftware (v1.8)</td><td colspan="1" rowspan="1">SmartCycler software</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Inactivated bacteria asinternal control</td><td colspan="1" rowspan="1">Internal Control template</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Manual</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

# L. Test Principle:

The IMDx VanR for Abbott m2000 assay is a qualitative, in vitro diagnostic assay containing reagents for the real-time PCR amplification, detection, and differentiation of nucleic acids from organisms harboring vanA or vanB genes. Detection of the vanA and vanB genes is measured by the presence of fluorescently-labeled oligonucleotide probes that generate a fluorescent signal when specifically bound to amplified vanA and/or vanB PCR products; differentiation of vanA from vanB is attained by labeling the oligonucleotide probes with different colored fluorescent dyes. The IMDx VanR for Abbott $m 2 0 0 0$ assay includes inactivated bacteria as a full process control; this inactivated bacteria is unrelated to enterococci and introduced into each specimen during sample preparation such that it is coextracted and co-amplified with each specimen. This full process control serves to demonstrate that the entire assay process has proceeded within specification for individual specimens. The Abbott $m 2 0 0 0$ system consists of two instruments: the Abbott $m 2 0 0 0 s p$ (for sample preparation) and the Abbott $m 2 0 0 0 r t$ (for real-time PCR amplification and detection). The assay is intended to be used directly on rectal swab, peri-rectal swab, or stool specimens collected from patients to aid in the control of Vancomycin Resistant Enterococcus (VRE).

Rectal swab, peri-rectal swab, or stool specimens are initially subjected to sample preparation on the Abbott $m 2 0 0 0 s p$ , an automated sample preparation system. Sample preparation lyses bacteria present in the sample, including any VRE, to make the vanA and vanB target nucleic acids accessible for amplification and to remove potential amplification inhibitors. At the completion of the sample processing procedure, the resulting bacterial lysate is transferred to an Abbott 96-Deep-Well. The Abbott $m 2 0 0 0 s p$ also combines the IMDx VanR for Abbott $m 2 0 0 0$ Amplification Reagent components and dispenses the resulting Master Mix into the Abbott 96-Well Optical Reaction Plate. After manual application of the Abbott Optical Adhesive Cover, the plate is ready for transfer to the Abbott $m 2 0 0 0 r t$ .

Amplification/detection takes place on the Abbott $m 2 0 0 0 r t$ instrument using real-time PCR techniques; during each round of PCR amplification the fluorescent probes anneal to the amplified target DNA, if present. The probes are labeled with different fluorescent molecules allowing vanA, vanB and the IMDx Process Control-A targets to be distinguished from each other. The probes are single-stranded, linear DNA oligonucleotides modified with a fluorescent moiety covalently linked to one end of the probe and a quenching moiety to the other end. When the probe binds to its complementary sequence in the target during amplification, the fluorophore separates from the quenchers, thus allowing fluorescent emission and detection. Since this fluorescence occurs during every cycle, the PCR reaction can be read in real-time. Positive and negative controls are included with each run to ensure the integrity of the system.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Assay precision was measured in four independent studies:

· within-laboratory repeatability; · user-to-user reproducibility; · lot-to-lot reproducibility, and instrument-to-instrument reproducibility.

A seven-member precision panel was comprised of intact organisms from two VRE strains: $E$ . faecium (ATCC BAA-2320; vanA-type) and $E .$ .faecalis (ATCC 700802; vanB-type). Each strain was formulated at three target levels:

i. positive (corresponding to a concentration associated with ${ \sim } 2 { - } 3 \mathbf { x }$ LoD), ii. low positive (1x LoD), and iii. high negative (\~0.05-0.25x LoD, an estimation of a 20 to $80 \%$ positivity rate).

Panel members were made using TE Buffer plus fecal matrix to simulate clinical specimens. The target LoD value used for this study was $9 4 4 \mathrm { C F U / m L }$ (2435 CFU/swab) for ATCC BAA-2320 and 644 CFU/mL (1610 CFU/swab) for ATCC 700802. The seventh panel member was a true negative sample that contained TE Buffer plus fecal matrix alone, without any VRE organisms added.

# Within-Laboratory Repeatability

For the repeatability study, the seven-member panel was tested twice a day for a total of twelve days. Panel members were tested in replicates of three for each run (for a total of 504 data points for the 24 runs). The entire study was conducted by one technician using one instrument pair ( $m 2 0 0 0 s p$ and $m 2 0 0 0 r t )$ and one reagent lot of the IMDx VanR for Abbott $m 2 0 0 0$ assay. The average, standard deviation, and

coefficient of variation in cycle number (CN) for vanA, vanB and the Internal Control (IC) were reported as follows:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>vanA/vanB CN</td><td rowspan=1 colspan=3>IC CN</td><td rowspan=1 colspan=3>% Agreement</td></tr><tr><td rowspan=1 colspan=1>SAMPLE ID</td><td rowspan=1 colspan=1>AVG</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>AVG</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Expected</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>% Agree</td></tr><tr><td rowspan=1 colspan=1>vanA Pos</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>97.2%</td></tr><tr><td rowspan=1 colspan=1>vanA Low Pos</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>vanA High Neg</td><td rowspan=1 colspan=1>41.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>55.6%</td></tr><tr><td rowspan=1 colspan=1>vanB Pos</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB Low Pos</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB High Neg</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>27.8%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

# User-to-User Reproducibility

For the reproducibility study the panel members were randomized and blinded. Each panel member was tested in replicates of three, twice a day for five days, at three study sites, by two technologists at each site for a total of 30 experimental runs; each operator performed one run each day. The entire study was conducted using one instrument system $m 2 0 0 0 s p$ and $ { \mathbf Ḋ } 2 0 0 0 r t Ḍ$ at each site and one reagent lot of the IMDx VanR for Abbott $m 2 0 0 0$ assay. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>SITE</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Site #1</td><td rowspan=1 colspan=2>Site #2</td><td rowspan=1 colspan=2>Site #3</td></tr><tr><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedrsults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedrsults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedrsults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>vanA Pos</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA Low Pos</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87/90</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>vanA High Neg</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>60/90</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>vanB Pos</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB Low Pos</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB High Neg</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>15/30</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>7/30</td><td rowspan=1 colspan=1>23.3%</td><td rowspan=1 colspan=1>32/90</td><td rowspan=1 colspan=1>35.6%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>183/210</td><td rowspan=1 colspan=1>87.1%</td><td rowspan=1 colspan=1>189/210</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>167/210</td><td rowspan=1 colspan=1>79.5%</td><td rowspan=1 colspan=1>539/630</td><td rowspan=1 colspan=1>85.5%</td></tr></table>

# Lot-to Lot-Reproducibility

Lot-to-lot reproducibility was assessed using three lots of the IMDx VanR for Abbott $m 2 0 0 0$ assay. One experiment was run for each of the three lots (for a total of three runs). A single operator performed the study using a single instrument pair $( m 2 0 0 0 s p$ and $m 2 0 0 0 r t )$ . Panel members were tested in replicates of six for each run. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>IMDx VanR for Abbott m2000 Lot</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Lot #1</td><td rowspan=1 colspan=2>Lot #2</td><td rowspan=1 colspan=2>Lot #3</td></tr><tr><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>vanA Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA High Neg</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>8/18</td><td rowspan=1 colspan=1>44.4%</td></tr><tr><td rowspan=1 colspan=1>vanB Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB High Neg</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>10/18</td><td rowspan=1 colspan=1>55.6%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>38/42</td><td rowspan=1 colspan=1>90.5%</td><td rowspan=1 colspan=1>35/42</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>35/42</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>108/126</td><td rowspan=1 colspan=1>85.7%</td></tr></table>

# Instrument-to-Instrument Reproducibility

Instrument-to-instrument reproducibility was determined independently for both the Abbott $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments. To measure $m 2 0 0 0 s p$ instrument variability, the test panel was run on three different $m 2 0 0 0 s p$ instruments. Panel members were run in replicates of six. Runs were performed in succession by the same operator, using a single IMDx VanR for Abbott $m 2 0 0 0$ assay lot. Once the master mix assembly portion of the $m 2 0 0 0 s p$ protocol was executed by the different $m 2 0 0 0 s p$ instruments, the assembled reaction plates were run using a single $m 2 0 0 0 r t$ instrument. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>Abbott m2000sp Instrument-to-Instrument Reproducibility</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Instrument #1</td><td rowspan=1 colspan=2>Instrument #2</td><td rowspan=1 colspan=2>Instrument #3</td></tr><tr><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>vanA Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA High Neg</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>10/18</td><td rowspan=1 colspan=1>55.6%</td></tr><tr><td rowspan=1 colspan=1>vanB Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB High Neg</td><td rowspan=1 colspan=1>1/6</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>3/18</td><td rowspan=1 colspan=1>16.7%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>33/42</td><td rowspan=1 colspan=1>78.6%</td><td rowspan=1 colspan=1>34/42</td><td rowspan=1 colspan=1>81.0%</td><td rowspan=1 colspan=1>36/42</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>103/126</td><td rowspan=1 colspan=1>81.7%</td></tr></table>

To measure $m 2 0 0 0 r t$ variability, a single $m 2 0 0 0 s p$ instrument was used in one experiment. Six replicates of each panel member were prepared. The eluted nucleic acids from the sample extractions were automatically transferred to an Abbott 96- well Deep Well Plate by the $m 2 0 0 0 s p$ per normal operational procedures. Aliquots from one deep well plate were then used to assemble three separate, identical Abbott 96-well Optical Reaction Plates, using the same lot of the IMDx VanR for Abbott $m 2 0 0 0$ assay. Each of the three identical Abbott 96-Well Optical Reaction plates were then run on three separate Abbott $m 2 0 0 0 r t$ instruments, by the same operator.

The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>Abbott m2000rt Instrument-to-Instrument Reproducibility</td><td rowspan=2 colspan=2>Abbott m2000sp</td></tr><tr><td rowspan=1 colspan=2>Instrument #1</td><td rowspan=1 colspan=2>Instrument #1</td><td rowspan=1 colspan=2>Instrument #1</td></tr><tr><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1># expectedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>vanA Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanA High Neg</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>10/18</td><td rowspan=1 colspan=1>55.6%</td></tr><tr><td rowspan=1 colspan=1>vanB Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB Low Pos</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>vanB High Neg</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>5/18</td><td rowspan=1 colspan=1>27.8%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>36/42</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>33/42</td><td rowspan=1 colspan=1>78.6%</td><td rowspan=1 colspan=1>36/42</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>105/126</td><td rowspan=1 colspan=1>83.3%</td></tr></table>

Negative samples showed an overall negativity rate of $100 \%$ ; no false positive results were observed. High Negative samples were expected to provide a negativity rate of $20 \mathrm { - } 8 0 \%$ ; the overall observed negativity rate was $46 \%$ . Low Positive samples were expected to provide a positivity rate of $2 9 5 \%$ ; the overall observed positivity rate was $9 6 \%$ . Positive samples were expected to provide a positivity rate of $100 \%$ ; the overall observed positivity rate was $9 9 \%$ . There were two (2) false negative samples out of all samples tested. The IMDx VanR for Abbott $m 2 0 0 0$ assay provided reproducible results.

b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The IMDx Process Control-A is composed of inactivated Gram-positive bacteria unrelated to vancomycin-resistant enterococci and is introduced into each sample during specimen processing; therefore, an IMDx Process Control-A result is generated with each specimen result. Positive and negative Quality Controls are run with each IMDx VanR for Abbott $m 2 0 0 0$ assay run.

Of the 1,506 specimens enrolled in the clinical study, 45 $( 3 . 0 \% )$ “Error Message” results caused retesting: (i) A positive control run failure invalidated 30 stool samples; upon retesting all but one of the 30 samples produced a valid result. The one sample not producing a valid result was invalidated due to the internal control being non-reactive during repeat testing (e.g. PCR inhibition). (ii) One additional stool sample was invalidated due to the internal control being outside the validity range for the initial and repeat test (again, PCR inhibition). (iii) The remaining 14 samples (two peri-rectal specimens and 12 stool specimens) were invalidated due to a mechanical error arising in the $m 2 0 0 0 s p$ . These errors were related to an issue with the instrument Liquid Handling/Liquid Sensing system, and all these errors were

encountered in two runs: a single run at one site and a single run at a second site.

In summary, there were 31 $( 2 . 1 \% )$ instances where the assay controls failed; there were 14 $( 0 . 9 \% )$ instances where the instrument invalidated the assay. The rate of failure was not excessive; the assay control strategy adequately evaluated the integrity of the result.

# Specimen Stability Studies

The following sample/swab types were tested:

· Bacti-Swabs with Amies transport media (Remel, Cat# R723095); · E-swabs with liquid Amies media (Becton Dickinson (BD), Cat# 220245); · BBL culture swabs with liquid Stuart media (BD, Cat# 220099), and · Raw stool was sampled with Copan flocked swab (Copan, Cat# 502CS01).

The data provided support to show that each specimen type was stable when stored under the following conditions

30 days at ${ - 3 0 ^ { \circ } C }$ to ${ } _ { - 1 0 ^ { \circ } \mathrm { C } }$ , or   
· 7 days at $2 \mathrm { { } ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ .   
Specimens were also stable for 3 freeze thaw cycles.

# d. Detection limit:

The limit of detection (LoD) of the IMDx VanR for Abbott $m 2 0 0 0$ assay was determined using six strains of vancomycin-resistant enterococci (four vanA-type and two vanB-type). Each strain was tested in replicates of twenty with three lots of the IMDx VanR for Abbott $m 2 0 0 0$ assay. Combined results for the three lots were used for probit analysis and calculation of confidence intervals at the $9 5 \%$ level $( 9 5 \%$ CI). A swab equivalency study was performed to confirm that simulated samples containing fecal matrix used in the LoD studies were equivalent to rectal/stool specimens.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Limit of Detection (95% CI)</td></tr><tr><td rowspan=1 colspan=1>E. faecium (vanA-type)ATCC 700221 Strain: unidentified</td><td rowspan=1 colspan=1>4,300.6 CFU/swab(3,862.6  4,788.2)</td></tr><tr><td rowspan=1 colspan=1>E. faecium (vanA-type)ATCC 51559 Strain: MMC4</td><td rowspan=1 colspan=1>1,010.7 CFU/swab(975.1 - 1,047.5)</td></tr><tr><td rowspan=1 colspan=1>E. faecium (vanA-type)ATCC BAA-2318 Strain: clinical isolate</td><td rowspan=1 colspan=1>889.0 CFU/swab(777.1 - 1,017.0)</td></tr><tr><td rowspan=1 colspan=1>E. faecium (vanA-type)ATCC BAA-2320 Strain: clinical isolate</td><td rowspan=1 colspan=1>2,435.4 CFU/swab(2,043.9 - 2,901.7)</td></tr><tr><td rowspan=1 colspan=1>E. faecalis (vanB-type)ATCC 700802 Strain: V583</td><td rowspan=1 colspan=1>1610.0 CFU/swab(1,569.1 - 1,652.0)</td></tr><tr><td rowspan=1 colspan=1>E. faecalis (vanB-type)ATCC 51575 Strain: Taxo 239</td><td rowspan=1 colspan=1>810.1 CFU/swab(571.7 - 1,147.8)</td></tr></table>

The LoD for vanA-type VRE was determined to be:

4,300.6 CFU/swab.

The LoD for vanB-type VRE was determined to be:

1,610 CFU/swab.

# Analytical Reactivity

Ninety-one (91) characterized VRE strains and/or clinical isolates (43 vanA-type, 45 vanB-type, and three VRE that had multiple vancomycin resistance elements) were tested for reactivity with the IMDx VanR for Abbott $m 2 0 0 0$ assay. Strains originated from at least twelve states in the US, and six countries (USA, Canada, Chile, Italy, Germany, Belgium). Each strain was diluted in TE Buffer plus fecal matrix at 2-3X LoD; three replicates were tested. Most strains were detected by the assay, demonstrating that the IMDx VanR Assay for Abbott $m 2 0 0 0$ can detect a broad range of both vanA-type and vanB-type VRE strains.

<table><tr><td colspan="1" rowspan="1">Genus/species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">van-Type</td><td colspan="1" rowspan="1">Genus/species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">van-Type</td></tr><tr><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">2020</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">6818</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2070</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">23497</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2078</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">1555</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2075</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">5270</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">2023</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA- 51299</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2182</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">19035</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2030</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA-51858</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2079</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">10004</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2027</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">12808</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2179</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">8092</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2160</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">3251</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2157</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA-2365</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">FM 2023</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">1697</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2067</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">14175</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2155</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">1726$</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA-2319</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">13556</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA-2317</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">712</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2169</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">24§</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">BAA-2316</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecalis</td><td colspan="1" rowspan="1">5027§</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">2071§</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E.faecium</td><td colspan="1" rowspan="1">4464§</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. raffinosus</td><td colspan="1" rowspan="1">MMCI-01</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-02</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-04</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-05</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-14</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-06</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-24</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-08</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-28</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-09</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-29</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. gallinarum</td><td colspan="1" rowspan="1">MMCI-10</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-30</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-11</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-31</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-12</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-32</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-13</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-35</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-15</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-36</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-17</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-37</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-18</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-38</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-19</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-39</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">MMCI-20</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-41</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-21</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-42</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-22</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-43</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-23</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-44</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-25</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-45</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-26</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-46</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-27</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-47</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-33</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-49</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">MMCI-34</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">MMCI-58</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-51</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. gallinarum</td><td colspan="1" rowspan="1">MMCI-52</td><td colspan="1" rowspan="1">vanA, vanC</td><td colspan="1" rowspan="1">E. gallinarum</td><td colspan="1" rowspan="1">MMCI-56</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-59</td><td colspan="1" rowspan="1">vanA, vanB</td><td colspan="1" rowspan="1">E. gallinarum</td><td colspan="1" rowspan="1">MMCI-57</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">MMCI-60*</td><td colspan="1" rowspan="1">vanA, vanB</td><td colspan="3" rowspan="1">-</td></tr></table>

§These isolates could only be detected when the strain was tested at a higher concentration than the calculated LoD. \*While vanA was detected at the calculated LoD, the vanB target could only be detected when the strain was tested at a higher concentration.

While the vast majority of these isolates could be detected at the defined LOD, there was one vanA isolate and six vanB isolates that required higher input loads in order to be detected (i.e., they were not initially detected at the 2-3X of the respective LoDs). The following isolates, and the load required to achieve detection, are shown in the table below. For one vanA+vanB, double-positive strain (MMCI-60), the vanB gene was present in significantly lower copy number than the vanA gene, such that the vanB target could only be detected when the strain was tested at a significantly higher concentration. All eight of these isolates originated in the United States.

<table><tr><td rowspan=1 colspan=1>vanA Strain</td><td rowspan=1 colspan=1>LoD Required</td><td rowspan=1 colspan=1>vanB Strain</td><td rowspan=1 colspan=1>LoD Required</td></tr><tr><td rowspan=1 colspan=1>2071</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>1726</td><td rowspan=2 colspan=1>5X LoD</td></tr><tr><td rowspan=5 colspan=2></td><td rowspan=1 colspan=1>13556</td></tr><tr><td rowspan=1 colspan=1>712</td><td rowspan=2 colspan=1>10X LoD</td></tr><tr><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>5027</td><td rowspan=1 colspan=1>20X LoD</td></tr><tr><td rowspan=1 colspan=1>4464</td><td rowspan=1 colspan=1>30X LoD</td></tr></table>

# Challenge Study

A challenge study was conducted using a panel of 72 well-characterized strains of Enterococcus: 23 vanA strains, $2 5 \nu a n B$ strains, two strains with both vanA and vanB, one strain with both vanA and vanC, five vanC strains, three vanD strains, two vanE strains, one vanG strain, and 11 vancomycin-susceptible strains. All enterococci strains harboring vanA or vanB resistance genes were detected correctly with the IMDx VanR for Abbott m2000 assay at 2-3X LoD. All enterococci strains harboring vanC, vanD, vanE, or vanG resistance genes, and all vancomycin-sensitive

enterococci strains did not react with the IMDx VanR for Abbott $m 2 0 0 0$ assay at a concentration of $\geq 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L }$ .

# e. Analytical specificity:

Cross reactivity was performed using a panel of 96 test organisms to evaluate any cross-reactivity with the IMDx VanR for Abbott $m 2 0 0 0$ assay. Bacteria were tested from fresh cultures at a concentration of $\geq 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L }$ . Viruses were tested from frozen stocks at a concentration of ${ \geq } 1 \times 1 0 ^ { 5 }$ TCID50/mL. Nucleic acids were tested at genomic equivalents $\geq 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria, and ${ \ge } 1 \times 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ for viruses. All samples were prepared by diluting organisms or DNA into TE Buffer plus fecal matrix. No cross-reactivity was detected with any of the 96 test organisms included in this panel.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Abiotropia defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">Enterococcus faecalis (vanGstrain)</td><td colspan="1" rowspan="1">MMCI-16</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 8459</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">ATCC 17925</td><td colspan="1" rowspan="1">Enterococcus gallinarum†</td><td colspan="1" rowspan="1">ATCC 49573</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ZMC 0801715</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">MMCI-62</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp.faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">Enterococcus faecium*</td><td colspan="1" rowspan="1">MMCI-64</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">ATCC 35098</td><td colspan="1" rowspan="1">Enterococcus faecium*</td><td colspan="1" rowspan="1">ATCC BAA-2127</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 13472</td><td colspan="1" rowspan="1">Enterococcus faecalis‡</td><td colspan="1" rowspan="1">ATCC BAA-2128</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus HER 1414</td><td colspan="1" rowspan="1">ATCC 11778</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 49533</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ZMC 0801583</td><td colspan="1" rowspan="1">Enterococcus spp.</td><td colspan="1" rowspan="1">MMCI-69</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli DNA*</td><td colspan="1" rowspan="1">ATCC 43479</td><td colspan="1" rowspan="1">Enterococcus spp.</td><td colspan="1" rowspan="1">MMCI-69</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni DNA*</td><td colspan="1" rowspan="1">ATCC 33292</td><td colspan="1" rowspan="1">Enterococcus spp.</td><td colspan="1" rowspan="1">MMCI-70</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">Enterococcus gallinarum(vanCl strain)</td><td colspan="1" rowspan="1">ATCC 49608</td></tr><tr><td colspan="1" rowspan="1">Candida catenulata</td><td colspan="1" rowspan="1">ATCC 10565</td><td colspan="1" rowspan="1">Enterococcus gallinarum(vanCl strain)</td><td colspan="1" rowspan="1">ATCC 49609</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ZMC 0801538</td><td colspan="1" rowspan="1">Enterococcus gallinarum(vanCl strain)</td><td colspan="1" rowspan="1">ATCC 49610</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">ATCC 25405</td><td colspan="1" rowspan="1">Enterococcus gallinarum(vanCl strain)</td><td colspan="1" rowspan="1">ATCC 700425</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">Enterococcus hirae*</td><td colspan="1" rowspan="1">ATCC 8043</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27028</td><td colspan="1" rowspan="1">Enterococcus casseliflavus‡</td><td colspan="1" rowspan="1">MMCI 65</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ATCC 51115</td><td colspan="1" rowspan="1">Enterococcus raffinosus†</td><td colspan="1" rowspan="1">ATCC 49427</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic</td><td colspan="1" rowspan="1">ATCC 43593</td><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">ZMC 0810047CF</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic</td><td colspan="1" rowspan="1">ATCC 43601</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile NAP1toxigenic strain</td><td colspan="1" rowspan="1">ATCC BAA-1870</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ZMC 0801517</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">ATCC 25986</td><td colspan="1" rowspan="1">Escherichia coli (ESBLplasmid)</td><td colspan="1" rowspan="1">ATCC BAA-197</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitaliumDNA*</td><td colspan="1" rowspan="1">ATCC 33798</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">ATCC 33650</td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus</td><td colspan="1" rowspan="1">ZMC 0810075CF</td><td colspan="1" rowspan="1">Fusobacterium varium DNA*</td><td colspan="1" rowspan="1">ATCC 8501</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ZMC 0810003CF</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 33496</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio piger DNA*</td><td colspan="1" rowspan="1">ATCC 29098</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">ZMC 0810023CF</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-2146</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta DNA*</td><td colspan="1" rowspan="1">ATCC 25559</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">ATCC 23272</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">ATCC 11454</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">ATCC 33999</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium*</td><td colspan="1" rowspan="1">ATCC 49463</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus‡</td><td colspan="1" rowspan="1">CCRI-1566/IDI 1981</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35198</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum†</td><td colspan="1" rowspan="1">ATCC 43198</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6896</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus(vanC2/3 strain)</td><td colspan="1" rowspan="1">ATCC 700668</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar*</td><td colspan="1" rowspan="1">ZMC custompropagation</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9250</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis†</td><td colspan="1" rowspan="1">ATCC 23241</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis†</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis†</td><td colspan="1" rowspan="1">ATCC 29302</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ZMC 0801722</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis†</td><td colspan="1" rowspan="1">ATCC 33074</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis†</td><td colspan="1" rowspan="1">ATCC 35550</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 33862</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis‡</td><td colspan="1" rowspan="1">ATCC 49532</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 6538</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium†</td><td colspan="1" rowspan="1">ATCC 49224</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 29213</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium†</td><td colspan="1" rowspan="1">ATCC BAA-472</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">ATCC 35034</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium (vanDstrain)</td><td colspan="1" rowspan="1">MMCI-03</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">ZMC 0801516</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium (vanDstrain)</td><td colspan="1" rowspan="1">MMCI-53</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanEstrain)</td><td colspan="1" rowspan="1">MMCI-50</td><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">ATCC 19436</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanEstrain)</td><td colspan="1" rowspan="1">MMCI-55</td><td colspan="1" rowspan="1">Human Genomic DNA**</td><td colspan="1" rowspan="1">Promega G304A</td></tr></table>

\*Denotes testing on genomic nucleic acid isolate from the organism at levels equivalent to $\overline { { \geq 1 \times 1 0 ^ { 6 } \mathrm { C F U } / \mathrm { m L } } }$ for bacteria and ${ \geq } 1$ $\times 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses. \*\*Human Genomic $\mathrm { D N A } \geq 4 . 4 \mathrm { x } 1 0 ^ { 5 }$ copies/mL † vancomycin resistance status unknown ‡ vancomycin susceptible strain

# Microbial Interference

To assess if there was any microbial interference with the IMDx VanR for Abbott $m 2 0 0 0$ assay, the same panel used to assess analytical specificity was added to tubes, each containing one of four strains of VRE in TE Buffer plus fecal matrix. The four VRE strains used for this study were:

vanA-type o E. faecium strain MMC4, o E. faecium strain ATCC BAA-2320, vanB-type o E. faecalis strain V583, and o E. faecalis strain ATCC 51575.

These VRE strains were present in the samples at a concentration corresponding to 2 - 3x LoD. Each of the four strains was tested in the presence of each test organism in triplicate using the IMDx VanR for Abbott $m 2 0 0 0$ assay. No evidence of microbial interference was observed for any of the 96 test organisms included in the analysis.

# Vancomycin-Resistant Staphylococcus aureus (VRSA) testing

To examine whether vanA gene sequences in VRSA could be detected by the IMDx VanR for Abbott $m 2 0 0 0$ assay, twelve VRSA isolates (all were vanA-type) from the Network on Antimicrobial Resistance in Staphylococcus aureus (NARSA) were tested. These included: VRS1, VRS2, VRS3a, VRS4, VRS5, VRS6, VRS7, VRS8, VRS9, VRS10, VRS11a, and VRS11b. Titered stocks were diluted to $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ and tested with the IMDx VanR for Abbott $m 2 0 0 0$ assay in triplicate. The vanA gene was detected in all strains. The vanB gene was not detected in any of the strains; however, as expected the IMDx VanR for Abbott $m 2 0 0 0$ assay may produce positive results if S. aureus organisms harboring these genes are present in the clinical specimen.

# Potentially Interfering Substances

To evaluate the potential of substances typically found in fecal specimens to interfere with the IMDx VanR for Abbott m2000 assay, four strains of VRE were tested in the presence of such substances. The VRE strains used included:

vanA-type o E. faecium strain MMC4, o E. faecium strain ATCC BAA-2320, vanB-type o E. faecalis strain V583, and o E. faecalis strain ATCC 51575.

These test organisms were diluted to a concentration of $2 \textrm { - } 3 \mathrm { x }$ LoD using TE Buffer plus fecal matrix. The test panel consisted of 23 substances shown in the table below:

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Active Ingredient(s) inSubstance</td><td colspan="1" rowspan="1">Concentration of SubstanceTested</td></tr><tr><td colspan="1" rowspan="1">Miconazole nitrate cream</td><td colspan="1" rowspan="1">Miconazole Nitrate</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Preparation H®</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">Zinc oxide</td><td colspan="1" rowspan="1">40% w/w paste</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">Hemorrhoid gel</td><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Witch Hazel</td><td colspan="1" rowspan="1">N/A*</td></tr><tr><td colspan="1" rowspan="2">Gaviscon®</td><td colspan="1" rowspan="1">Aluminum hydroxide</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Magnesium carbonate</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">TUMS</td><td colspan="1" rowspan="1">Calcium carbonate</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tagamet®</td><td colspan="1" rowspan="1">Cimetidine</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Prilosec® (delayed release)</td><td colspan="1" rowspan="1">Omeprazole magnesium</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="1">Condoms</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">7% v/v</td></tr><tr><td colspan="1" rowspan="1">Imodium</td><td colspan="1" rowspan="1">Loperamide HCl</td><td colspan="1" rowspan="1">0.00667 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Pepto Bismol®</td><td colspan="1" rowspan="1">Bismuth Subsalicylate</td><td colspan="1" rowspan="1">0.87 mg/mL</td></tr><tr><td colspan="1" rowspan="1">ExLax</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Vancomycin HCI</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">12.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Aleve</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Moist Towelettes</td><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">0.12% w/v</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">Glucose, hormones, enzymes,iron, ions, etc</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid (fecal fat)</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Stearic acid (fecal fat)</td><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">4 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">5 mg/mL</td></tr></table>

${ } ^ { * } \mathrm { N } / \mathrm { A } =$ not applicable

Substances were diluted in TE Buffer plus fecal matrix to concentrations that would either replicate or exceed the highest concentration expected to be found in a clinical sample. Each of the four VRE strains was tested in triplicate at $2 - 3 \mathrm { x } \mathrm { L o D }$ in the presence of each substance. No interference was observed with any of the substances tested.

# Target Carryover Study

Six assay runs were performed with alternating high positive and negative samples to assess the potential for cross-over or carryover contamination. No vanA or vanB target was detected in any of the 269 negative samples tested.

# f. Assay cut-off:

The cut-off for the IMDx VanR for Abbott $m 2 0 0 0$ assay was determined using Abbott’s proprietary maxRatio algorithm for real-time PCR data analysis. The maxRatio method identifies a consistent point within or very near the exponential region of the PCR signal. Compared to other analysis techniques that generate only a cycle number, the maxRatio method generates several measurements of amplification including cycle numbers (CN), relative measures of amplification efficiency and curve shape. By using these values, the maxRatio method can achieve a highly reliable reactive/nonreactive determination along with quantitative evaluation. For a result to be considered reactive, the fluorescence generated must cross a reactive threshold value (reactive threshold settings are based on “maxRatio values”). Initial maxRatio threshold parameters were based on analysis of results of characterized strains of VRE. Validation of threshold parameters came from analysis of 1,417 results (196 vanA-positive, 44 vanB-positive, and 1,177 negative) from a clinical study, compiled from six test sites.

<table><tr><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=2>vanA</td><td rowspan=1 colspan=2>vanB</td></tr><tr><td rowspan=1 colspan=1>Negativen=45</td><td rowspan=1 colspan=1>Positiven = 105</td><td rowspan=1 colspan=1>Negativen =45</td><td rowspan=1 colspan=1>Positiven = 13</td></tr><tr><td rowspan=1 colspan=1>maxRatio mean</td><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>0.075</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.193</td></tr><tr><td rowspan=1 colspan=1>maxRatio St Dev</td><td rowspan=1 colspan=1>0.002</td><td rowspan=1 colspan=1>0.013</td><td rowspan=1 colspan=1>0.0004</td><td rowspan=1 colspan=1>0.026</td></tr><tr><td rowspan=1 colspan=1>maxRatio Range</td><td rowspan=1 colspan=1>0.000- 0.007</td><td rowspan=1 colspan=1>0.020 - 0.092</td><td rowspan=1 colspan=1>0.000- 0.001</td><td rowspan=1 colspan=1>0.143 - 0.229</td></tr><tr><td rowspan=1 colspan=1>Threshold set</td><td rowspan=1 colspan=2>0.012</td><td rowspan=1 colspan=2>0.010</td></tr><tr><td rowspan=1 colspan=1># standard deviations of thresholdfrom mean of negative samples</td><td rowspan=1 colspan=2>4.9</td><td rowspan=1 colspan=2>22.7</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: See clinical studies below.   
b. Matrix comparison: Not Applicable.

3. Clinical studies:

The performance of the IMDx VanR for Abbott $m 2 0 0 0$ assay was evaluated using an IRB approved protocol at five geographically diverse locations within the United States. The reference method consisted of direct culture complemented by enriched culture. All samples were first inoculated onto Bile Esculin agar with $6 ~ { \mu \mathrm { g / m L } }$ vancomycin (BEAV) and incubated at $3 5 \pm 2 ^ { \circ } \mathrm { C }$ for 24 and 48 hours. Presumptive colonies of Enterococcus were subcultured for isolation on a TSA with $5 \%$ Sheep’s Blood and incubated for 18-24 hours at $3 5 \pm 2 ^ { \circ } \mathrm { C }$ . Colonies presumptive for Enterococcus were Gram Stained; Gram-positive cocci were tested for Catalase and pyrrolidonyl arylamidase (PYR). Gram-positive cocci that were catalase negative and PYR positive were further identified using an FDA cleared method and susceptibility to vancomycin determined using an FDA cleared test. Final confirmation as either vancomycin resistant $E .$ . faecium $\left( \mathrm { V R E } _ { f m } \right)$ or $E .$ . faecalis $( { \mathrm { V R E } } _ { f s } )$ was based on biochemical identification and vancomycin susceptibility at $\geq 3 2 ~ \mu \mathrm { g / m L }$ . The presence of vanA and/or vanB gene sequences in isolated $\mathrm { V R E } _ { f m }$ or $\mathrm { V R E } _ { f s }$ was confirmed by alternative PCR.

A total of 1,506 specimens were entered in the study. Six specimens were withdrawn due to protocol deviations, and one specimen was withdrawn due to an unresolved instrument error. The remaining 1,500 specimens comprised 469 stool specimens, 444 rectal swabs, and 587 peri-rectal swabs. Among prospectively collected stool, rectal, and peri-rectal specimens, the reference method isolated 69, 63, and 50 VRE, respectively, while the IMDx VanR for Abbott $m 2 0 0 0$ assay yielded 117, 89, and 80 positive results, respectively. The analyses in the tables below report the performance of the the IMDx VanR for Abbott $m 2 0 0 0$ assay in this clinical study. The VRE isolated included: $E .$ . casseliflavus, $E .$ . faecalis, E. faecium, $E .$ . gallinarum, $E .$ . hirae, $E$ . raffinosus, as well as eight enterococci that could not be speciated. The majority of the isolates were $E$ . faecalis and $E$ . faecium; the latter comprised approximately three-quarters of the VRE isolates.

In addition, 45 suspected vancomycin-resistant Enterococcus faecalis rectal swab specimens were run retrospectively at a single site. One (1) specimen was withdrawn when organism identification could not be confirmed by reference methods. Therefore, a total of 44 specimens had reportable results. Among these retrospectively collected specimens, the IMDx VanR for Abbott m2000 assay yielded 44 positive results, while the reference method yielded 42 VRE.

# Stool Specimens: IMDx vs. Enriched Culture plus Alternative PCR. (Prospective Collection)

# Enriched Culture $+$ Alternative PCR

IMDx VanR for Abbott $\mathbf { \nabla } m 2 0 0 0$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>vanA-typeEnterococcus</td><td rowspan=1 colspan=1>vanB-typeEnterococcus</td><td rowspan=1 colspan=1>vanA-type andvanB-typeEnterococcus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>vanA and vanB</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>343</td><td rowspan=1 colspan=1>350</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>469</td></tr></table>

\* Considered as FN for the 2x2 table below

IMDx VanR for Abbott m2000

Enriched Culture $^ +$ Alternative PCR   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>343</td><td rowspan=1 colspan=1>352</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>469</td></tr></table>

<table><tr><td>Sensitivity Specificity</td><td>87.0% (77.0% - 93.0%) 85.8% (82.0% - 88.8%)</td></tr><tr><td>Positive Predictive Value</td><td>51.3% (42.3% - 60.2%)</td></tr><tr><td>Negative Predictive Value</td><td>97.4% (95.2% - 98.6%)</td></tr><tr><td>Prevalence</td><td>14.7%</td></tr></table>

# Rectal Swab Specimens: IMDx vs. Enriched Culture plus Alternative PCR. (Prospective Collection)

# Enriched Culture $+$ Alternative PCR

IMDx VanR for Abbott $\mathbf { \nabla } m 2 0 0 0$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>vanA-typeEnterococcus</td><td rowspan=1 colspan=1>vanB-typeEnterococcus</td><td rowspan=1 colspan=1>vanA-type andvanB-typeEnterococcus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>vanA and vanB</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>311</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>337</td><td rowspan=1 colspan=1>400</td></tr></table>

IMDx VanR for Abbott $\mathbf { \mu } _ { m 2 0 0 0 }$   
Enriched Culture $^ +$ Alternative PCR   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>89</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>311</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>337</td><td rowspan=1 colspan=1>400</td></tr></table>

95% CI   

<table><tr><td>Sensitivity Specificity</td><td>96.8% (89.1% - 99.1%)</td></tr><tr><td>Positive Predictive Value</td><td>91.7% (88.3% - 94.2%)</td></tr><tr><td></td><td>68.5% (58.3% - 77.2%)</td></tr><tr><td>Negative Predictive Value</td><td>99.4% (97.7% - 99.8%)</td></tr><tr><td>Prevalence</td><td>15.8%</td></tr></table>

# Rectal Swab Specimens: IMDx vs. Enriched Culture plus Alternative PCR. (Retrospective Collection)

# Enriched Culture $+$ Alternative PCR

IMDx VanR for Abbott $\mathbf { \nabla } m 2 0 0 0$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>vanA-typeEnterococcus</td><td rowspan=1 colspan=1>vanB-typeEnterococcus</td><td rowspan=1 colspan=1>vanA-type andvanB-typeEnterococcus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>vanA and vanB</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>44</td></tr></table>

IMDx VanR for Abbott m2000

Enriched Culture $^ +$ Alternative PCR   
$9 5 \%$ CI   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>44</td></tr></table>

Positive Percent Agreement $100 \%$ $( 9 1 . 6 \% - 1 0 0 . 0 \% )$ Negative Percent Agreement $0 . 0 \%$ $( 0 . 0 0 \% - 6 5 . 8 \% )$

# Peri-Rectal Specimens: IMDx vs. Enriched Culture plus Alternative PCR. (Prospective Collection)

# Enriched Culture $+$ Alternative PCR

IMDx VanR for Abbott $\mathbf { \nabla } m 2 0 0 0$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>vanA-typeEnterococcus</td><td rowspan=1 colspan=1>vanB-typeEnterococcus</td><td rowspan=1 colspan=1>vanA-type andvanB-typeEnterococcus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>vanA and vanB</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>507</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>537</td><td rowspan=1 colspan=1>587</td></tr></table>

IMDx VanR for Abbott m2000

Enriched Culture $^ +$ Alternative PCR   
95% CI   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>507</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>537</td><td rowspan=1 colspan=1>587</td></tr></table>

Sensitivity 94.0% (83.8% – 97.9%) Specificity $9 3 . 9 \%$ (91.5% - 95.6%)   
Positive Predictive Value $5 8 . 8 \%$ (47.8% - 68.9%)   
Negative Predictive Value $9 9 . 4 \%$ $( 9 8 . 3 \% - 9 9 . 8 \% )$ Prevalence $8 . 5 \%$

a. Clinical Sensitivity: See clinical performance studies above.

$b$ . Clinical specificity: See clinical performance studies above.   
c. Other clinical supportive data (when a. and b. are not applicable): N/A

4. Clinical cut-off: See discussion in Assay Cut-off section above (Section M(1)(f)).

5. Expected values/Reference range:

Expected values by specimen type among prospectively collected specimens.

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>TotalN</td><td rowspan=1 colspan=1>Number ofVRE withvanA</td><td rowspan=1 colspan=1>Number ofVRE withvanB</td><td rowspan=1 colspan=1>Number ofVRE withvanA and/orvanB</td><td rowspan=1 colspan=1>Prevalenceof VREwith vanA</td><td rowspan=1 colspan=1>Prevalenceof VREwith vanB</td><td rowspan=1 colspan=1>Prevalenceof VREwith vanAand/or vanB</td></tr><tr><td rowspan=1 colspan=1>Stool</td><td rowspan=1 colspan=1>469</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td></tr><tr><td rowspan=1 colspan=1>Rectal</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15.5%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Peri-Rectal</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

# N. Instrument Name:

Abbott m2000 System running Application Software v.1.8:

Abbott m2000sp: sample preparation and reagent mixing Abbott $m 2 0 0 0 \mathrm { r t }$ : amplification reaction and detection

# O. System Descriptions:

1. Modes of Operation:

The Abbott $m 2 0 0 0$ System is an instrument platform that automates steps to perform nucleic acid amplification assays from sample processing through amplification, detection, and data reduction. The Abbott $m 2 0 0 0$ System comprises the $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments, which are operated with separate System Control Center (SCC) workstations. Each instrument contains an independent software application; one for the $m 2 0 0 0 s p$ and a second for the $m 2 0 0 0 r t$ . The $m 2 0 0 0 s p$ instrument is a floor standing, automated sample preparation system. The $m 2 0 0 0 r t$ instrument is a real-time PCR thermal cycler/reader instrument system. Abbott Molecular is the manufacturer of the $m 2 0 0 0$ System. The principal hardware components that comprise the $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ were developed by Original Equipment Manufacturer vendors, Tecan Schweiz AG, Mannedorf, Switzerland, and Applied Biosystems, Foster City, CA, respectively. Abbott Molecular developed the software that is uniquely for use with the $m 2 0 0 0$ System. The Abbott m2000 System software processes sample preparation and amplification/detection protocols based on pre-determined, assay-specific parameters that are contained in individual assay application specification (App Spec) files that are installed on the SCC. The Abbott $m 2 0 0 0 s p$ reads and processes bar coded primary sample tubes and processes up to 96 specimens, controls, and calibrators in batch mode. The $m 2 0 0 0$ System is capable of processing samples from various matrices, depending on the specific assay application. At the completion of the automated sample preparation protocol, the operator seals and manually transfers the PCR plate to the Abbott $m 2 0 0 0 r t$ for nucleic acid detection. Bar code and $m 2 0 0 0 s p$ data is transferred to the $m 2 0 0 0 r t$ electronically via removable media (i.e., a CD).

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

3. Specimen Identification:

Specimen identification occurs on the Abbott $m 2 0 0 0 s p$ instrument when it is loaded with specimen tubes.

4. Specimen Sampling and Handling:

Fresh or frozen human peri-rectal or rectal swabs collected and transported to the laboratory are used for testing. Fresh or frozen human stool samples collected and transported to the laboratory in a sterile container and sampled with a sterile, nylon flocked swab are used for testing. Specimen processing is automated on the Abbott $m 2 0 0 0 s p$ .

# 5. Calibration:

There is no calibration procedure associated with the IMDx VanR for Abbott $m 2 0 0 0$ assay per se. Optical calibration of the Abbott $m 2 0 0 0 r t$ instrument is required for the accurate measurement and discrimination of dye fluorescence during the IMDx VanR for Abbott $m 2 0 0 0$ assay; the Calibration Procedures section in the Abbott $m 2 0 0 0 r t$ Operations Manual describes how to perform an optical calibration.

6. Quality Control:

N/A

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

1. The submitted information in this premarket notification is complete and supports a substantial equivalence decision.