# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N

A. 510(k) Number:

K123197

# B. Purpose for Submission:

Substantial equivalence determination of the Verigene® Clostridium difficile Nucleic Acid Test (CDF) on the Verigene® System.

# C. Measurand:

Targets DNA sequences of the toxin A (tcdA), toxin B (tcdB) and tcdC genes within the PaLoc of toxigenic strains of C. difficile; presumptive identification of the PCR ribotype 027 strain of C. difficile is via detection of the binary toxin (cdt) gene sequence and the single base pair deletion at nucleotide 117 in the tcdC gene (which encodes a negative regulator of toxin production).

# D. Type of Test:

Qualitative, in vitro diagnostic test using polymerase chain reaction (PCR) amplification of tcdA, tcdB, tcdC, and cdt gene sequences, as well as detection of the single base pair deletion at nucleotide 117 in the tcdC gene combined with a nanoparticle-based array hybridization detection assay.

# E. Applicant:

Nanosphere, Inc

F. Proprietary and Established Names: Verigene® Clostridium difficile Nucleic Acid Test (CDF)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130 - C. Difficile Nucleic Acid Amplification Test Assay

2. Classification: Class II

3. Product code:

OZN - Amplification assay for the detection of Clostridium difficile toxin genes from stool specimens of symptomatic patients

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Verigene® Clostridium difficile Nucleic Acid Test (CDF) is a qualitative, multiplexed in vitro diagnostic test for the rapid detection of toxin A (tcdA), toxin B (tcdB), and tcdC gene sequences of toxigenic strains Clostridium difficile and for presumptive identification of PCR ribotype 027 strains from unformed (liquid or soft) stool specimens collected from patients suspected of having $C .$ . difficile infection (CDI). Presumptive identification of the PCR ribotype 027 strain of C. difficile is by detection of the binary toxin (cdt) gene sequence and the single base pair deletion at nucleotide 117 in the tcdC gene. The tcdC gene encodes for a negative regulator in $C .$ . difficile toxin production. The test is performed on the Verigene System and utilizes automated specimen preparation and polymerase chain reaction (PCR) amplification, combined with a nanoparticle-based array hybridization assay to detect the toxin gene sequences associated with toxin-producing C. difficile.

The CDF Test is indicated for use as an aid in the diagnosis of CDI. Detection of PCR ribotype 027 strains of C. difficile by the CDF Test is solely for epidemiological purposes and is not intended to guide or monitor treatment for $C .$ difficile infections. Concomitant culture is necessary only if further typing or organism recovery is required.

# 2. Indication(s) for use:

The Verigene® Clostridium difficile Nucleic Acid Test (CDF) is a qualitative, multiplexed in vitro diagnostic test for the rapid detection of toxin A (tcdA), toxin B (tcdB), and tcdC gene sequences of toxigenic strains Clostridium difficile and for presumptive identification of PCR ribotype 027 strains from unformed (liquid or soft) stool specimens collected from patients suspected of having $C .$ . difficile infection (CDI). Presumptive identification of the PCR ribotype 027 strain of C. difficile is by detection of the binary toxin $( c d t )$ gene sequence and the single base pair deletion at nucleotide 117 in the tcdC gene. The tcdC gene encodes for a negative regulator in $C .$ . difficile toxin production. The test is performed on the Verigene System and utilizes automated specimen preparation and polymerase chain reaction (PCR) amplification, combined with a nanoparticle-based array hybridization assay to detect the toxin gene sequences associated with toxin-producing C. difficile.

The CDF Test is indicated for use as an aid in the diagnosis of CDI. Detection of PCR ribotype 027 strains of C. difficile by the CDF Test is solely for epidemiological purposes and is not intended to guide or monitor treatment for $C .$ difficile infections. Concomitant culture is necessary only if further typing or organism recovery is required.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: To be used with the Verigene® System

# I. Device Description:

The Verigene® C. difficile Nucleic Acid Test (CDF) is a molecular assay which relies on detection of specific nucleic acid targets in a microarray format. For each of the bacterial nucleic acid sequences detected by the CDF test, the assay utilizes unique Capture and Mediator oligonucleotides, followed by gold nanoparticle probe-based endpoint detection. The Capture oligonucleotides are covalently bound to the microarray substrate and hybridize to a specific portion of the nucleic acid targets. The Mediator oligonucleotides have a region which bind to a different portion of the same nucleic acid targets and also have a sequence which allows binding of a gold nanoparticle probe. Specific silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency and provide accurate detection of target capture.

The CDF Test is performed on the Verigene System, a “sample-to-result,” fully automated, bench-top molecular diagnostics workstation. The System enables automated nucleic acid extraction from unformed stool specimens (liquid or soft) and detection of specific bacterial target DNA. The Verigene System consists of two components: the Verigene Reader and the Verigene Processor $S P$ .

The Reader is the Verigene System’s user interface, which serves as the central control unit for all aspects of test processing, automated imaging, and result generation using a touchscreen control panel and a barcode scanner. The Verigene Processor $S P$ executes the test procedure, automating the steps of (1) Sample Preparation and Target Amplification (i.e., cell lysis and magnetic bead-based bacterial DNA isolation and nucleic acid amplification), and (2) Hybridization (i.e., detection and identification of bacterial-specific DNA in a microarray format by using gold nanoparticle probe-based technology). Once the specimen is loaded by the operator, all other fluid transfer steps are performed by an automated pipette that transfers reagents between wells of the trays and finally loads the specimen into the Test Cartridge for hybridization. Single-use disposable test consumables and a self-contained Verigene Test Cartridge are utilized for each sample tested with the CDF assay.

To obtain the test results after test processing is complete the user removes the Test Cartridge from the Processor $S P$ and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader and intensities from the microarray spots are used to make a determination regarding the presence (Detected) or absence (Not Detected) of a bacterial nucleic acid sequence/analyte. This determination is made by means of software-based decision algorithm resident in the Verigene Reader.

To prevent reagent dispensing errors, all reagents are prepackaged in single-use disposables, including Stool Lysis Buffer (SLB) Tubes, reagent trays, and cartridges. Several layers of controls built into the CDF test ensure that failures at any step are identified during the procedure or in the end-point image analysis of the Test Cartridge.

An artificial DNA construct serves as an amplification or PCR control (i.e., Internal Processing Control 1 [IC1]) and is included within the Amplification Tray. Bacillus subtilis serves as a specimen preparation & amplification control (i.e., Internal Processing Control 2 [IC2]) and is automatically added by the Verigene SP to each specimen prior to the extraction step. Additional positive controls are immobilized on the Test Slide and are used to determine that hybridization was performed correctly. The CDF algorithm requires that these controls be valid before decisions regarding the presence or absence of any other target on the panel can be determined.

The Verigene® CDF Test Kit contains sufficient reagents to process 20 specimens or quality control samples. The kit contains the following:

# Verigene® CDF Test Kit

20 Verigene® CDF Test Cartridges: Each Test Cartridge comes preloaded with all required reaction solutions, including wash solutions, oligonucleotide probe solutions and signal amplification solutions required to generate a test result.

20 Verigene® CDF Extraction Trays (with Tip Holder Assemblies): Each Extraction Tray comes preloaded with all required solutions, including lysis/binding buffer, wash solutions, and buffer solutions necessary to extract nucleic acids and generate a test result.

· 20 Verigene® CDF Stool Lysis Buffer (SLB) tubes and sterile swabs.

# Verigene® CDF Test Amplification Kit

20 Verigene® CDF Amplification Trays: Each Amplification Tray comes preloaded with all required solutions, including enzymes and buffers necessary to amplify nucleic acids and generate a test result.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Portrait Toxigenic C. Difficile Assay - Great Basin Scientific Xpert C. difficile/Epi Assay - Cepheid Verigene RVNATSP Test - Nanosphere

2. Predicate 510(k) number(s):

Portrait Toxigenic C. Difficile Assay - K113358   
Xpert C. difficile/Epi Assay - K110203   
Verigene RVNATSP Test - K092566

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New Device:NanosphereVerigene C. difficile AssayK123197</td><td colspan="1" rowspan="1">Predicate 1:Great Basin PortraitToxigenicC. difficile AssayK113358</td><td colspan="1" rowspan="1">Predicate 2:Cepheid Xpert C. difficile/Epi AssayK110203</td></tr><tr><td colspan="1" rowspan="1">IntendedUse</td><td colspan="1" rowspan="1">The Verigene® Clostridium difficile Nucleic Acid Test(CDF) is a qualitative, multiplexed in vitro diagnostictest for the rapid detection of toxin A (tcdA), toxin B(tcdB), and tcdC gene sequences of toxigenic strainsClostridium difficile and for presumptive identificationof PCR ribotype 027 strains from unformed (liquid orsoft) stool specimens collected from patients suspectedof having C. difficile infection (CDI). Presumptiveidentification of the PCR ribotype 027 strain of C.difficile is by detection of the binary toxin (cdt) genesequence and the single base pair deletion at nucleotide117 in the tcdC gene. The tcdC gene encodes for anegative regulator in C. difficile toxin production. Thetest is performed on the Verigene System and utilizesautomated specimen preparation and polymerase chainreaction (PCR) amplification, combined with ananoparticle-based array hybridization assay to detectthe toxin gene sequences associated with toxin-producing C. difficile.The CDF Test is indicated for use as an aid in thediagnosis of CDI. Detection of PCR ribotype 027strains of C. difficile by the CDF Test is solely forepidemiological purposes and is not intended to guideor monitor treatment for C. difficile infections.Concomitant culture is necessary only if further typingor organism recovery is required.</td><td colspan="1" rowspan="1">Portrait Toxigenic C. difficileAssay, a prescription deviceunder 21 CFR Part 801.109that is indicated for thedetection of toxigenicClostridium difficile inhuman fecal samplescollected from patientssuspected of havingClostridium difficile infection(CDI). The test utilizesautomated blocked primerenabled helicase-dependentamplification (bpHDA) todetect toxin gene sequencesassociated with toxinproducing C. difficile. ThePortrait Toxigenic C. difficileAssay is intended as an aid inthe diagnosis of CDI.</td><td colspan="1" rowspan="1">The Cepheid Xpert® C. difficile/Epi Assay is aqualitative in vitro diagnostic test for rapiddetection of toxin B gene sequences and forpresumptive identification of 027/NAP1/BI strainsof toxigenic Clostridium difficile from unformed(liquid or soft) stool specimens collected frompatients suspected of having C. difficile infection(CDI). Presumptive identification of027/NAP1/BI strains of C. difficile is by detectionof binary toxin (CDT) gene sequences and thesingle base pair deletion at nucleotide 117 in thetcdC gene. The tcdC gene encodes for a negativeregulator in C. difficile toxin production. The testis performed on the Cepheid GeneXpert® DxSystem and utilizes automated real-timepolymerase chain reaction (PCR) to detect toxingene sequences associated with toxin producing C.difficile. The Xpert C. difficile/Epi Assay isintended as an aid in the diagnosis of CDI.Detection of 027/NAP1/BI strains of C. difficileby the Xpert C. difficile/Epi Assay is presumptiveand is solely for epidemiological purposes and isnot intended to guide or monitor treatment for C.difficile infections. Concomitant culture isnecessary only if further typing or organismrecovery is required.</td></tr><tr><td colspan="1" rowspan="1">Techno-logical</td><td colspan="1" rowspan="1">Fully-automated nucleic acid amplification</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TestCartridge</td><td colspan="1" rowspan="1">Disposable single-use, multi-chambered fluidic testcartridge.</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Unformed (liquid or soft) Stool</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>New Device:Nanosphere Verigene C. difficile AssayK123197</td><td rowspan=1 colspan=1>Predicate 1:Great Basin Portrait ToxigenicC. difficile Assay11358</td><td rowspan=1 colspan=1>Predicate 2:Cepheid Xpert C. difficile/Epi AssayK110203</td></tr><tr><td rowspan=1 colspan=1>Principle</td><td rowspan=1 colspan=1>DNA: PCR</td><td rowspan=1 colspan=1>DNA: Isothermal, helicase-dependentnucleic acid amplification</td><td rowspan=1 colspan=1>DNA: real-time PCR</td></tr><tr><td rowspan=1 colspan=1>DNA TargetSequences</td><td rowspan=1 colspan=1>C. difficile toxin A (tcdA), toxin B (tcdB),tcdC, binary toxin (cdt) and the tcdC deletionat nt 117 (tcdCΔ117)</td><td rowspan=1 colspan=1>C. difficile toxin B (tcdB)</td><td rowspan=1 colspan=1>C. difficile toxin B (tcdB), binary toxin and the tcdCdeletion nt 117 (tcdCΔ117)</td></tr><tr><td rowspan=1 colspan=1>InstrumentSystem</td><td rowspan=1 colspan=1>Verigene ReaderVerigene Processor SP</td><td rowspan=1 colspan=1>Great BasinPortrait Analyzer</td><td rowspan=1 colspan=1>Cepheid GeneXpert Dx System</td></tr><tr><td rowspan=1 colspan=1>SampleExtraction</td><td rowspan=1 colspan=1>Self-contained cartridge with magnetic bead-based bacterial DNA extraction after elutionfrom prepared stool specimens.</td><td rowspan=1 colspan=1>Self-contained and automated lysisand extraction after swab elution andfiltration.</td><td rowspan=1 colspan=1>Self-contained and automated after swab elution andtwo single-dose reagent additions.</td></tr><tr><td rowspan=1 colspan=1>Probes</td><td rowspan=1 colspan=1>Capture/Mediator oligonucleotides, followedby gold nanoparticle probe-based endpointdetection. Specific silver enhancement of thebound gold nanoparticle probes at the capturesites results in gold-silver aggregates thatscatter light with high efficiency. Lightscattering is detected by the Verigene reader.</td><td rowspan=1 colspan=1>Amplification primers are biotin-labeled primers and hybridized to anarray of probes immobilized on thesilicon chip. Incubation with anti-biotin antibody conjugated to thehorseradish peroxidase withtetramethylbenzidine development isvisualized by Portrait Analyzer</td><td rowspan=1 colspan=1>TaqMan® Probes  real-time fluorescence detectionby GeneXpert Dx System</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile – Draft Guidance for Industry and FDA Staff (FDA document 1715, issued on November 29, 2010)

# L. Test Principle:

The Verigene® Clostridium difficile Nucleic Acid Test (CDF Test) is a rapid, automated in vitro diagnostic test for the qualitative detection of $C .$ . difficile DNA directly from unformed (liquid or soft) stool specimens of patients suspected of having $C .$ . difficile infection (CDI).

The CDF Test detects toxin A (tcdA), toxin B (tcdB), and tcdC gene sequences of toxigenic strains of C. difficile. Presumptive identification of the PCR ribotype 027 strain of $C .$ . difficile is by detection of the binary toxin (cdt) gene sequence and the single base pair deletion at nucleotide 117 in the tcdC gene. The tcdC gene encodes for a negative regulator in C. difficile toxin production. The test is performed on the Verigene® System and utilizes automated specimen preparation and polymerase chain reaction (PCR) amplification, combined with a nanoparticle-based array hybridization assay to detect the toxin gene sequences associated with toxin-producing C. difficile.

Prior to initiating a test on the Verigene Processor SP, the Extraction Tray, the Amplification Tray and Tip Holder are loaded into the Verigene Processor SP. The barcode located on the CDF Test Cartridge is entered via the scanner attached to the Reader, and the associated sample information is entered either using the barcode-scanner or the Reader touch-screen interface (this links specific patient information to a specific Test Cartridge number). The CDF Test Cartridge is then inserted into the Processor SP. Once the consumables are loaded, $1 5 0 \mu \mathrm { L }$ of liquid stool sample is added to Stool Lysis Buffer, vortexed, and then centrifuged. A $1 0 0 \mu \mathrm { L }$ aliquot of this Stool Lysis Buffer is then pipetted into the designated Sample Well within the Extraction Tray and the Drawer Assembly is closed to initiate the test. The Processor SP identifies the Test Cartridge via an internal barcode scanner and communicates with the Reader to receive test instructions. Once the Processor SP module completes processing (approximately 2.0 hours), the CDF Test Cartridge is removed and inserted into the Reader for automated identification of the gene-specific DNA amplification products by nanoparticle-based array hybridization detection.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

The inter-laboratory reproducibility of the CDF Test was determined by conducting a reproducibility study at three external sites. A seven sample panel for the reproducibility study was comprised of two different strains at three different

concentrations (six positive samples) and one (1) negative sample. Each of the seven samples in the panel was tested daily in triplicate by two operators for five nonconsecutive days at three sites for a total of 90 tests per sample (3 sites x 2 operators/site x 3 replicates/operator $\mathrm { ~ x ~ } 5 \mathrm { ~ d a y s = 9 0 ~ }$ tests per sample). The study tested a total of 630 samples. Positive and negative controls were run by each site once per combination of consumable lots received

The results of the Reproducibility Study are provided in the table below. The results indicate the Verigene CDF Test was reproducible in this multi-center study.

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>ExpectedOccurrence</td><td rowspan=1 colspan=4>Total Agreement with Expected Result*(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CDFNegativeStool Matrix</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>~100%Negative</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%90/90(96.0100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>ToxigenicWild TypeC. difficile</td><td rowspan=1 colspan=1>ModeratePositive(MP)</td><td rowspan=1 colspan=1>~100%Positive</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%90/90(96.0100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>LowPositive(LP)</td><td rowspan=1 colspan=1>~95%Positive</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%90/90(96.0100)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HighNegative(HN)</td><td rowspan=1 colspan=1>~20-80%Negative</td><td rowspan=1 colspan=1>30%9/30(14.749.4)</td><td rowspan=1 colspan=1>33.30%10/30(17.352.8)</td><td rowspan=1 colspan=1>16.70%5/30(5.634.7)</td><td rowspan=1 colspan=1>26.70%24/90(17.937.0)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=1>ToxigenicHypervirulentC. difficile</td><td rowspan=1 colspan=1>ModeratePositive(MP)</td><td rowspan=1 colspan=1>~100%Positive</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>100%90/90(96.0100)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>LowPositive(LP)</td><td rowspan=1 colspan=1>~95%Positive</td><td rowspan=1 colspan=1>96.70%29/30(82.899.9)</td><td rowspan=1 colspan=1>96.70%29/30(82.899.9)</td><td rowspan=1 colspan=1>100%30/30(88.4100)</td><td rowspan=1 colspan=1>97.80%88/90(92.299.7)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>HighNegative(HN)</td><td rowspan=1 colspan=1>~20-80%Negative</td><td rowspan=1 colspan=1>36.70%11/30(19.956.1)</td><td rowspan=1 colspan=1>40.00%12/30(22.759.4)</td><td rowspan=1 colspan=1>30.00%9/30(14.749.4)</td><td rowspan=1 colspan=1>35.60%32/90(25.746.4)</td></tr></table>

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Internal Control: An artificial DNA construct serves as an amplification or PCR control and is referred to as the Internal Processing Control 1 (IC1). This control

material along with the primers and detection oligonucleotides are included within the Amplification Tray. If the process control is not valid a No Call result will be obtained and the test should be repeated.

Processing Control: Bacillus subtilis serves as a specimen preparation & amplification control and is referred to as the Internal Processing Control 2 (IC2). This control is automatically added by the Verigene SP to each specimen prior to the extraction step. If the process control is not valid a No Call result will be obtained and the test should be repeated.

Additional positive controls are immobilized on the Test Slide. These are used to determine that hybridization was performed correctly. CDF algorithm requires that these controls be valid before decisions regarding the presence or absence of any other target on the panel can be determined. If these controls are not detected a no call result will be obtained and the test should be repeated.

# d. Detection limit:

Analytical sensitivity (LoD) of the CDF Test was determined for seven strains of $C .$ . difficile, representing all major toxinotypes found in North America and including two PCR Ribotype 027 strains. The LoD was defined as the concentration at which the test produces a positive result greater than $9 5 \%$ of the time. Serial dilutions of the strains were tested and the putative LoD confirmed with 20 replicates. The LoDs for the seven strains are shown in the table below and ranged from 63 to $1 2 5 0 \mathrm { C F U / m l }$ of stool. This study established the overall limit of detection of the CDF Test to be 1250 $\mathrm { C F U / m l }$ of organism present in stool.

<table><tr><td rowspan=1 colspan=1>Strain Designation(Source ID)</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>CalculatedCFU/ml Stoolat LoD</td><td rowspan=1 colspan=1>CFU perCDF Test atLoD</td><td rowspan=1 colspan=1>LoDConfirmationResults</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>ATCC 43255 (VPI 10463)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1875 (5325)</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>CDC 2007858</td><td rowspan=1 colspan=1>IX/XXIII</td><td rowspan=1 colspan=1>1250</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>CDC 2009087</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1250</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>CDC 2009292</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>1250</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598 (1470)</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20/20</td></tr></table>

# e. Analytical Reactivity:

Analytical reactivity of the CDF Test was demonstrated with a comprehensive panel of $6 3 ~ C .$ . difficile strains, tested in triplicate at three times the LoD (i.e. 3,750 $\mathrm { C F U / m L }$ ). The panel was comprised of a wide range of toxinotypes, including

toxinotypes 0, I, IV, V, VIII, IX, X, XI, XII, XXI, XXII, IX/XXIII, XIV/XV, and six PCR ribotype 027 strains (toxinotype III).

<table><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Toxinotype</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 9689</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>CDC 2005088</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 17857</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>CDC 2005283</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 17858</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>CDC 2005325</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>CDC 2005359</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC 43594</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>CDC 2006017</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC 43596</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>CDC 2006062</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC 43597</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>CDC 2006376</td><td rowspan=1 colspan=1>VIII</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ATCC 43598</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>CDC 2007070</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>ATCC 43599</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>CDC 2007217</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>ATCC 43600</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>CDC 2007302</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC 51695</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>CDC 2007435</td><td rowspan=1 colspan=1>XII</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC 700792</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>CDC 2007816</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>ATCC BAA-1382</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>CDC 2007838</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC BAA-1803</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>CDC 2007858</td><td rowspan=1 colspan=1>IX/XXIII</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>ATCC BAA-1804</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>CDC 2007886</td><td rowspan=1 colspan=1>IX/XXIII</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>CDC 2008222</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC BAA-1806</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>CDC 2009048</td><td rowspan=1 colspan=1>XIV/XV</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC BAA-1808</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>CDC 2009078</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>ATCC BAA-1811</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>CDC 2009087</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>ATCC BAA-1812</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>CDC 2009141</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>ATCC BAA-1813</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>CDC 2009287</td><td rowspan=1 colspan=1>XXI</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>ATCC BAA-1814</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>CDC 2009292</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>ATCC BAA-1815</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>CDC 2009363</td><td rowspan=1 colspan=1>XXII</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>ATCC BAA-1874</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>CDC 20100276</td><td rowspan=1 colspan=1>VIII</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>ATCC BAA-2155</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>CDC 20100286*</td><td rowspan=1 colspan=1>XI</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>ATCC BAA-2156</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>CDC 20100304</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>CDC 2004013</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>CDC 20100307</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>CDC 2004111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>CDC 20100375</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>CDC 2004118</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>CDC 20100378</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>CDC 2004205</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>CDC 20100381</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>CDC 2004206</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>CCUG 8864/20309</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>CDC 2005022</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1></td></tr></table>

\*Per information provided by the CDC, this strain was tentatively identified as toxinotype XI

All tests correctly reported the expected results for the detection of gene sequences for toxigenic $C .$ . difficile and for presumptive PCR ribotype 027, with one exception: Strain CDC 2009048 strain, classified by the CDC as Toxinotype XIV/XV, is associated with non-027 strains (Ribotypes 111/122). However, the Verigene CDF Test reported detection of the tcdA, tcdB, binary and tcdC-MUT targets as would be expected for a PCR ribotype 027 strain. Subsequent sequencing of the tcdC gene verified the presence of the Δ117 deletion.

# f. Analytical specificity:

Ninety-four (94) microorganisms, including two (2) non-toxigenic C. difficile strains and fourteen (14) non C. difficile Clostridium species, seventy-seven (77) other microorganisms, and one (1) human cell line, were tested with the CDF Test to determine analytical specificity. In addition, the cross-reactivity of Clostridium botulinum was evaluated by in silico analysis.

Each bacterial strain was prepared in Negative Stool Matrix and tested in triplicate in concentrations of $5 { \times } 1 0 ^ { 6 } \ : \dot { \mathrm { C F U / m L } }$ stool. Two (2) organisms, Cryptosporidium parvum and Giardia lamblia, were tested using genomic DNA at a concentration of $1 \times \dot { 1 } 0 ^ { 6 }$ copies of gDNA. For the viruses, Echovirus 11 and Coxsackievirus were tested at $5 { \times } \mathrm { \dot { 1 } 0 } ^ { 5 } \ : \mathrm { P F U / m L }$ stool. Adenovirus, Enterovirus, Cytomegalovirus and Rotavirus were also tested using genomic DNA or RNA at a concentration of $1 \times { 1 0 } ^ { 6 }$ copies per reaction. Noroviruses were tested as clinical samples.

<table><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Organism/Cross-Reactant</td><td colspan="1" rowspan="1">StrainID</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Organism/Cross-Reactant</td><td colspan="1" rowspan="1">StrainID</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">43601</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">13182</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">43593</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Klebsiella pneumonia ssp. pneumonia</td><td colspan="1" rowspan="1">13883</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">49176</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">4356</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">19606</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">23272</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">7966</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">53103</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Alcaligenes faecalis ssp. faecalis</td><td colspan="1" rowspan="1">15554</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">11454</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">10702</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">33999</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">43185</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">19120</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">25285</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">7644</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">43184</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">27337</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">43183</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">14029</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">43479</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">Porphyromonas asaccharolyticus</td><td colspan="1" rowspan="1">25260</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Campylobacter jejuni ssp. jejuni</td><td colspan="1" rowspan="1">33292</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">25845</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">10231</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">43071</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">33431</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">35198</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">25407</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">9886</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">8090</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">9250</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">51115</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">27853</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">638</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">35554</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">BAA-613</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">Ruminococcus bromii</td><td colspan="1" rowspan="1">27255</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">19398</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">Salmonella choleraesuis (typhimurium)</td><td colspan="1" rowspan="1">14028</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">9650</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">Salmonella enterica ssp. arizonae</td><td colspan="1" rowspan="1">13314</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">43829</td><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">Salmonella enterica ssp. enterica</td><td colspan="1" rowspan="1">7001</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">27757</td><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">Serratia liquefacians</td><td colspan="1" rowspan="1">27592</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Clostridium noyvi</td><td colspan="1" rowspan="1">19402</td><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">13880</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">49531</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">9207</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">13124</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">11835</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">35704</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">29531</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">12464</td><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">43300</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">9714</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">14990</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Clostridium spiroforme</td><td colspan="1" rowspan="1">29899</td><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">49446</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">15579</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">12386</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">25986</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">Vibrio Cholerae</td><td colspan="1" rowspan="1">25870</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">PRA-67-D</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">17802</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">29098</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">43970</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">15947</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">9610</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">13048</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">Yersinia rohdei</td><td colspan="1" rowspan="1">43380</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">29006</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Non-bacterial Organisms</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">51299</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">VR-931</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Enterococcus faecium vanA</td><td colspan="1" rowspan="1">700221</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">VR-930</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">23511</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">VR-184</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">43894</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">0810003-CF</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">35469</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">0810023-CF</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">33650</td><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">VR-213</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">8501</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">Homo sapiens</td><td colspan="1" rowspan="1">CCL-218</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">50803-D</td><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1">Norovirus Group I</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Helicobacter fennelliae</td><td colspan="1" rowspan="1">35683</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">Norovirus Group II</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">43504</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">VR-2272</td></tr></table>

In addition, the cross-reactivity of Clostridium botulinum was evaluated by in silico analysis, which assessed the specificity of the CDF Capture, Mediator and primer oligos to GenBank sequence information for $C .$ . botulinum. Analytical specificity was observed to be $100 \%$ , including that determined by in silico analysis.

# g. Microbial Interference:

The CDF Test was tested against the same ninety-five (95) organisms/cell line that were used for analytical specificity, at the same medically relevant concentrations, using two strains of toxigenic $C .$ . difficile (ATCC BAA-1805 [toxinotype III] and ATCC 43255 [toxinotype 0]) at $1 . 5 \mathrm { x }$ LoD and $3 \mathbf { x }$ LoD, respectively, to evaluate the potential for microbial interference. No interference was observed with the CDF Test for any of the samples tested.

# h. Interfering Substances:

Thirty-four products/exogenous substances (shown in the table below) that are possibly encountered in stool samples were evaluated for potential inhibitory effects. Each interfering substance was evaluated at its “worst case” concentration, against two C. difficile strains (ATCC BAA-1805, ATCC 43255). Additionally, Cary-Blair media was tested. None of the 34 substances or the Cary-Blair media tested in this study showed any inhibitory effect on the detection of $C .$ . difficile when using the CDF Test.

<table><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Product</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">Concentration(w/w %)</td></tr><tr><td colspan="1" rowspan="1">1.</td><td colspan="1" rowspan="1">Stearic Acid, Grade I</td><td colspan="1" rowspan="1">Steric Acid</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="1">2.</td><td colspan="1" rowspan="1">Palmitic Acid Free Acid Sigma Grade</td><td colspan="1" rowspan="1">Palmitic Acid</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="1">3.</td><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">Glucose, Hormones, Enzymes,Ions, Iron, etc</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="1">4.</td><td colspan="1" rowspan="1">Nasopharyngeal Swab Sample in UniversalTransport Media (UTM)</td><td colspan="1" rowspan="1">Immunoglobulins, Lysozyme,Polymers, etc</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="1">5.</td><td colspan="1" rowspan="1">Nystatin Suspension</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">6.</td><td colspan="1" rowspan="1">Monistat® 3</td><td colspan="1" rowspan="1">Miconazole</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">7.</td><td colspan="1" rowspan="1">Preparation H® Medicated Wipes</td><td colspan="1" rowspan="1">Witch Hazel</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">8.</td><td colspan="1" rowspan="1">Vagisil Anti-Itch Crème Maximum Strength</td><td colspan="1" rowspan="1">Benzocaine, Resorcinol</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">9.</td><td colspan="1" rowspan="1">Preparation H® Anti-Itch Hydrocortisone 1%</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">10.</td><td colspan="1" rowspan="1">Desitin Maximum Strength Original Paste</td><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">11.</td><td colspan="1" rowspan="1">Sarna Anti-Itch Lotion, Sensitive</td><td colspan="1" rowspan="1">Pramoxine HCl</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">12.</td><td colspan="1" rowspan="1">Preparation H® Hemorrhoidal Ointment</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">13.</td><td colspan="1" rowspan="1">Walgreens Ready to Use Enema Mineral OilLaxative</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">14.</td><td colspan="1" rowspan="1">Options Conceptrol® Vaginal Contraceptive Gel</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">15.</td><td colspan="1" rowspan="1">Dulcolax® Laxative Suppositories</td><td colspan="1" rowspan="1">Bisacodyl</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">16.</td><td colspan="1" rowspan="1">Dimenhydrinate</td><td colspan="1" rowspan="1">Dimenhydrinate</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">17.</td><td colspan="1" rowspan="1">Neosporin® First Aid Antibiotic Ointment</td><td colspan="1" rowspan="1">Bacitracin, Neomycin, Polymyxin</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">18.</td><td colspan="1" rowspan="1">Wet Ones® Antibacterial Hand Wipes</td><td colspan="1" rowspan="1">Benzalkonium Chloride, Ethanol</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">19.</td><td colspan="1" rowspan="1">K-Y® Personal Lubricant Jelly</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">20.</td><td colspan="1" rowspan="1">Vaseline Original 100% Pure Petroleum Jelly</td><td colspan="1" rowspan="1">Petroleum</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="1">21.</td><td colspan="1" rowspan="1">Bile, bovine, dried, unfractioned</td><td colspan="1" rowspan="1">Bile</td><td colspan="1" rowspan="1">20%</td></tr><tr><td colspan="1" rowspan="1">22.</td><td colspan="1" rowspan="1">Tums® Antacid with Calcium Extra Strength 750</td><td colspan="1" rowspan="1">Calcium Carbonate</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">23.</td><td colspan="1" rowspan="1">Gaviscon® Extra Strength Liquid Antacid</td><td colspan="1" rowspan="1">Aluminum Hydroxide, MagnesiumHydroxide</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">24.</td><td colspan="1" rowspan="1">Phillips'® Genuine Milk of Magnesia SalineLaxative</td><td colspan="1" rowspan="1">Magnesium Hydroxide</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">25.</td><td colspan="1" rowspan="1">Aluminum Hydroxide, Reagent Grade</td><td colspan="1" rowspan="1">Aluminum Hydroxide</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">26.</td><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">27.</td><td colspan="1" rowspan="1">Immodium® AD Anti-Diarrheal</td><td colspan="1" rowspan="1">Loperamide Hydrochloride</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">28.</td><td colspan="1" rowspan="1">Pepto-Bismol Max Strength</td><td colspan="1" rowspan="1">Bismuth Subsalicylate</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">29.</td><td colspan="1" rowspan="1">Ex·lax® Maximum Strength Stimulant Laxative</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">30.</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">31.</td><td colspan="1" rowspan="1">Metronidazole Topical Cream (0.75%)</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">32.</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">33.</td><td colspan="1" rowspan="1">Mucin from bovine submaxillary glands, Type I-S(Dehydrated)</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">34.</td><td colspan="1" rowspan="1">Barium Sulfate</td><td colspan="1" rowspan="1">Barium Sulfate</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">35.</td><td colspan="1" rowspan="1">Cary-Blair Medium</td><td colspan="1" rowspan="1">Salts, Agar, thioglycollate</td><td colspan="1" rowspan="1">300%</td></tr></table>

# i. Assay cut-off:

The presence or absence of each target analyte was determined by the mean intensity of target capture spots relative to the Signal Detection Threshold (filter 1). With PCR amplification of the extracted sample DNA, a significant difference was expected in the signal intensity from toxigenic C. difficile bacteria-containing samples versus that obtained from negative samples. The capture, mediator, and PCR primer oligonucleotides in the CDF Test were designed to eliminate sequence-related crossreactivity, thereby ensuring that the non-specific target signal intensities at capture spots are similar to the microarray background signal. In contrast, amplicon hybridization to complementary capture and mediator probes were expected to give high positive signals, well-separated from negative capture spots. When reading a test slide, multiple images of each array were taken at increasing exposures times and the final target group mean intensity value for an analyte was assigned at the shortest exposure at which the value exceeds the Signal Detection Threshold. If the target signal did not exceed the threshold for any exposure, the mean spot intensity was

evaluated at the longest exposure taken.

The tcdC target was identified by unique captures that detect the tcdC-wild type (WT) and tcdC-mutant (∆117) (MUT) genotypes. If one of the captures exceeded the Signal Detection Threshold then tcdC was “Detected.” A second filter, the tcdC Genotype Threshold, then distinguished between the tcdC-wild type and tcdC-mutant (∆117). A tcdC genotype ratio was calculated from the individual tcdC-WT and - MUT group mean intensities.

In order to demonstrate the appropriateness of the cut-off values for these two thresholds, the target mean intensity values were examined for a panel of $5 9 C .$ . difficile strains in Negative Stool Matrix at a concentration of $3 7 5 0 \mathrm { C F U / m L } - 5 0 0 0$ CFU/mL stool. Each strain was tested twice and the six probes on the microarray (tcdA, tcdB, binary, tcdC, and the internal controls IC1, IC2) were compiled to generate a set of 708 data points (118 tests x 6 targets per test).

<table><tr><td colspan="1" rowspan="2">#</td><td colspan="2" rowspan="2">Sample</td><td colspan="1" rowspan="1"></td><td colspan="4" rowspan="1">Expected Result</td><td colspan="1" rowspan="2">#</td><td colspan="2" rowspan="2">Sample</td><td colspan="2" rowspan="1">Expected Result</td></tr><tr><td colspan="1" rowspan="1">tcdA</td><td colspan="1" rowspan="1">tcdB</td><td colspan="1" rowspan="1">binary</td><td colspan="1" rowspan="1">tcdC</td><td></td><td colspan="1" rowspan="1">tcdA</td><td colspan="1" rowspan="1">tcdB</td><td colspan="1" rowspan="1">binary</td><td colspan="1" rowspan="1">tcdC</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="2" rowspan="1">ATCC 9689</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">CDC 2007217</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="2" rowspan="1">ATCC 17857</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">CDC 2007302</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">ATCC 17858</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">CDC 2007816</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="2" rowspan="1">ATCC 43255</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">CDC 2007838</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="2" rowspan="1">ATCC 43593</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">CDC 2007858</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="2" rowspan="1">ATCC 43594</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">CDC 2007886</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="2" rowspan="1">ATCC 43596</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">CDC 2008222</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="2" rowspan="1">ATCC 43597</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">CDC 2009078</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="2" rowspan="1">ATCC 43598</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">CDC 2009087</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="2" rowspan="1">ATCC 43599</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">CDC 2009141</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="2" rowspan="1">ATCC 43600</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">CDC 2009292</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="2" rowspan="1">ATCC 43601</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">ATCC BAA-1382</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="2" rowspan="1">ATCC 43602</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">ATCC BAA-1801</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="2" rowspan="1">ATCC 43603</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">ATCC BAA-1803</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="2" rowspan="1">ATCC 51695</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">ATCC BAA-1804</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="2" rowspan="1">ATCC 700057</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">ATCC BAA-1805</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="2" rowspan="1">ATCC 700792</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">ATCC BAA-1806</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="2" rowspan="1">CDC 2004013</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">ATCC BAA-1807</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">CDC 2004111</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">ATCC BAA-1808</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">CDC 2004118</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">ATCC BAA-1809</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">CDC 2004205</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">ATCC BAA-1810</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">CDC 2004206</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">ATCC BAA-1811</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">CDC 2005022</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">ATCC BAA-1812</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">CDC 2005088</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">ATCC BAA-1813</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">CDC 2005283</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">ATCC BAA-1814</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">CDC 2005325</td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">ATCC BAA-1815</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">CDC 2005359</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">ATCC BAA-1874</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">CDC 2006017</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">ATCC BAA-2155</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">CDC 2006376</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">ATCC BAA-2156</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">CDC 2007070</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

Positive signals were well separated from the negative target signals, and the threshold value distinguished the “True Positives” from the “True Negatives.”

# j. Carry-Over & Cross-Contamination Study:

The potential for carry-over and cross-contamination to occur with the CDF Test on the Verigene system was assessed by testing C. difficile negative samples after running high positive (HP) samples. The toxigenic PCR ribotype $0 2 7 ~ C .$ . difficile strain BAA-1805 was used for the study, and was added at a high titer of $5 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ stool into the Negative Stool Matrix to prepare the high positive sample. The same Negative Stool Matrix was utilized as the C. difficile negative sample. In the execution of the study, the high-titer sample was alternated with the negative sample three times on three unique Verigene SP Processors, for a total of eighteen individual tests.

CDF target analytes were not detected by the CDF Test in any of the nine negative samples run immediately following the high-titer sample on three separate SP Processors. Additionally, all nine high-titer samples accurately detected the expected tcdA, tcdB, binary, and tcdC MUT targets. The study demonstrated that the CDF assay does not exhibit carry-over or cross- contamination that could result in a false positive test result.

2. Comparison studies:

# a. Method comparison with predicate device:

Clinical performance was determined by comparing the CDF Test results to reference culture (i.e., direct toxigenic culture or enriched toxigenic culture) followed by cell cytotoxicity testing on the isolates. Subsequent strain typing of the toxigenic strains by PCR Ribotyping and bi-directional sequencing methods was used to confirm toxigenic strains of $C .$ . difficile.

b. Matrix comparison:

Not applicable

3. Clinical studies:

Clinical performance characteristics of the CDF Test were determined in a multi-site prospective investigational study at five U.S. institutions using 1,877 unformed stool specimens from subjects whose routine care called for $C .$ . difficile testing. A portion of each leftover residual unformed stool specimen was obtained for testing. The clinical evaluation was carried out by comparing the CDF Test results to reference culture followed by cell cytotoxicity testing on the isolates. Subsequent strain typing on toxigenic strains by PCR ribotyping and bi-directional sequencing methods was used to confirm toxigenic strains of C. difficile.

In parallel to Verigene CDF Testing, an aliquot of the same specimen was sent to a reference laboratory for the reference culture and cytotoxin isolate testing. Each stool specimen was inoculated onto pre-reduced CCFA-D (cycloserine-cefoxitin-fructose agardirect plate; “direct culture”) and CCMB-Tal (cycloserine cefoxitin mannitol broth with taurocholate lysozyme cysteine). After 24 hours the CCMB-TAL was sub-cultured onto a second CCFA-E plate (CCFA-Enriched; “enriched culture”).

If C. difficile was isolated from the CCFA-D plate and the isolate was positive by the cell cytotoxicity assay, the specimen was classified as “toxigenic C. difficile positive” and the CCFA-E plate was not further analyzed. If no C. difficile was isolated from the CCFA-D plate or if the isolate was negative by the cell cytoxicity assay, the CCFA-E plate was further analyzed. If CCFA-E was positive for C. difficile and the isolate was positive for cell cytotoxicity assay, the specimen was classified as “toxigenic C. difficile positive.” The specimen was reported as “negative” if CCFA-E was negative for C. difficile or the isolate was tested negative by the cell cytotoxicity assay.

After central reference laboratory culture testing, the toxigenic C. difficile positive isolates were sent for strain identification by PCR Ribotyping to an external third-party

site. In parallel, Verigene CDF Test extracted DNA from the culture-confirmed $C .$ .   
difficile positive isolates were sent for tcdC bi-directional sequencing.

a. Clinical Sensitivity:

# Assay Performance vs. Direct Culture & PCR Ribotyping

Relative to direct culture with PCR ribotyping, the CDF Test demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 8 . 7 \%$ and $8 7 . 6 \%$ , respectively. The CDF Test also demonstrated a $9 7 . 5 \%$ positive agreement and $9 7 . 8 \%$ agreement for the hypervirulent $C .$ . difficile strain 027 by PCR ribotyping. The results are summarized in the table below (this data combines all investigational sites).

Verigene CDF Test Performance vs. Direct Culture & PCR Ribotyping   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Direct Culture &amp; PCR Ribotyping</td></tr><tr><td rowspan=5 colspan=1> Ie cee</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>Tox C. difficile +027 -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totala</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>286</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1500</td><td rowspan=1 colspan=1>1502</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>1713</td><td rowspan=1 colspan=1>1869b</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficil/027</td></tr><tr><td rowspan=1 colspan=2>Sensitivity: 98.7% (154/156)(95.5%-99.8%)Specificity: 87.6% (1500/1713)(85.9%-89.1%)Accuracy: 88.5% (1654/1869)(87.0%-89.9%)PPV: 42.0% (154/367)(36.9%-47.2%)NPV: 99.9% (1500/1502)(99.5%-99.9%)</td><td rowspan=1 colspan=2>Pos Agreement: 97.5% (39/40)(86.8%-99.9%)Neg Agreement: 97.8% (1787/1828)(97.0%-98.4%)Total Agreement: 97.7% (1827/1869)(96.9%-98.3%)PPV: 48.2% (39/81)(36.9%-59.5%)NPV: 99.9% (1787/1788)(99.7%-100%)</td></tr></table>

aThere were 1,877 evaluable specimens enrolled in the clinical trial; 71 specimens $( 3 . 8 \% )$ required repeat testing: 46 specimens $( 2 . 4 \% )$ had an initial $^ { 6 6 } \mathrm { N o }$ Call” result due to assay internal control errors; 17 specimens $( \bar { 1 } . 0 \% )$ had an initial “Indeterminate” call (No Call-IND), and 8 specimens $( 0 . 4 \% )$ had preanalytical errors (four motor stalls, two tip failures, one cracked slide and one cartridge not detected). The eight specimens that experienced pre-analytical errors, and the $4 6 \mathrm { N o }$ Call specimens, were called upon repeat testing; however, two of the No Call specimens required a second repeat test before being called. Repeat testing of the $1 7 \mathrm { N o }$ Call-IND specimens called all but two specimens (five specimens required a second repeat test before calling). Therefore, two specimens had a final “Indeterminate” call and were not included in the clinical data analysis of evaluable results. Thus, 1,875 specimens were analyzed in this clinical evaluation.

# Assay Performance vs. Enriched Culture & PCR Ribotyping

Relative to enriched culture with PCR Ribotyping, the Verigene CDF Test demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 1 . 8 \%$ and $9 2 . 5 \%$ , respectively. The CDF Test also demonstrated a $9 1 . 4 \%$ positive agreement and $9 8 . 5 \%$ agreement for the hypervirulent $C .$ difficile strain 027 by PCR Ribotyping. The results are summarized in the table below (this data combines all investigational sites).

Verigene CDF Test Performance vs. Enriched Culture & PCR Ribotyping   

<table><tr><td rowspan=1 colspan=5>Enriched Culture &amp; PCR Ribotyping</td></tr><tr><td rowspan=5 colspan=1> ce ce</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=2>Tox C. difficile +027 -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totala</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>188</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>286</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=2>18</td><td rowspan=1 colspan=1>1480</td><td rowspan=1 colspan=1>1502</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=2>211</td><td rowspan=1 colspan=1>1600</td><td rowspan=1 colspan=1>1869b</td></tr><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=3>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile/027</td></tr><tr><td rowspan=2 colspan=3>Sensitivity: 91.8% (247/269)(87.9%-94.8%)Specificity: 92.5% (1480/1600)(91.1%-93.7%)Accuracy: 92.4% (1727/1869)(91.1%-93.6%)PPV: 67.3% (247/367)(62.2%-72.1%)NPV: 98.5% (1480/1502)(97.8%-99.1%)</td><td rowspan=2 colspan=2>Pos Agreement: 91.4% (53/58)(81.0%-97.1%)Neg Agreement: 98.5% (1783/1811)(97.8%-99.0%)Total Agreement: 98.2% (1836/1869)(97.5%-98.8%)PPV: 65.4% (53/81)(54.0%-75.7%)NPV: 99.7% (1783/1788)(99.4%-99.9%)</td></tr><tr><td rowspan=1 colspan=1>/367</td></tr></table>

a There were 1,877 evaluable specimens enrolled in the clinical trial; 71 specimens $( 3 . 8 \% )$ required repeat testing: 46 specimens $( 2 . 4 \% )$ had an initial $^ { 6 6 } \mathrm { N o }$ Call” result due to assay internal control errors; 17 specimens $( 1 . 0 \% )$ had an initial “Indeterminate” call (No Call-IND), and 8 specimens $( 0 . 4 \% )$ had preanalytical errors (four motor stalls, two tip failures, one cracked slide and one cartridge not detected). The eight specimens that experienced pre-analytical errors, and the $4 6 \mathrm { N o }$ Call specimens, were called upon repeat testing; however, two of the No Call specimens required a second repeat test before being called. Repeat testing of the $1 7 \mathrm { N o }$ Call-IND specimens called all but two specimens (five specimens required a second repeat test before calling). Therefore, two specimens had a final “Indeterminate” call and were not included in the clinical data analysis of evaluable results. Thus, 1,875 specimens were analyzed in this clinical evaluation.

# Assay Performance vs. Direct Culture & Bi-Directional Sequencing

Relative to direct culture with bi-directional sequencing, the Verigene CDF Test demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 8 . 7 \%$ and $8 7 . 5 \%$ , respectively. The CDF Test also demonstrated a $9 7 . 7 \%$ positive agreement and $9 7 . 8 \%$ agreement for the hypervirulent $C .$ difficile strain 027 by bi-directional sequencing. The results are summarized in the table below (this data combines all investigational sites).

Verigene CDF Test Performance vs. Direct Culture & Sequencing   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Direct Culture &amp; Sequencing</td></tr><tr><td rowspan=5 colspan=1>F ce c</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>Tox C. difficile +027 -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totala</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>82</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>289</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1500</td><td rowspan=1 colspan=1>1502</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>1715</td><td rowspan=1 colspan=1>1873b</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile/027</td></tr><tr><td rowspan=1 colspan=2>Sensitivity: 98.7% (156/158)(95.5%-99.9%)Specificity: 87.5% (1500/1715)(85.8%-89.0%)Accuracy: 88.4% (1656/1873)(86.9%-89.8%)PPV: 42.1% (156/371)(37.0%-47.3%)NPV: 99.9% (1500/1502)(99.5%-100%)</td><td rowspan=1 colspan=2>Pos Agreement: 97.7% (42/43)(87.7%-99.9%)Neg Agreement: 97.8% (1790/1830)(97.0%-98.4%)Total Agreement: 97.8% (1832/1873)(97.0%-98.4%)PPV: 51.2% (42/82)(39.9%-62.4%)NPV: 99.9% (1790/1791)(99.7%-100%)</td></tr></table>

a There were 1,877 evaluable specimens enrolled in the clinical trial; 71 specimens $( 3 . 8 \% )$ required repeat testing: 46 specimens $( 2 . 4 \% )$ had an initial $^ { 6 6 } \mathrm { N o }$ Call” result due to assay internal control errors; 17 specimens $( 1 . 0 \% )$ had an initial “Indeterminate” call (No Call-IND), and 8 specimens $( 0 . 4 \% )$ had preanalytical errors (four motor stalls, two tip failures, one cracked slide and one cartridge not detected). The eight specimens that experienced pre-analytical errors, and the $4 6 \mathrm { N o }$ Call specimens, were called upon repeat testing; however, two of the No Call specimens required a second repeat test before being called. Repeat testing of the $1 7 \mathrm { N o }$ Call-IND specimens called all but two specimens (five specimens required a second repeat test before calling). Therefore, two specimens had a final “Indeterminate” call and were not included in the clinical data analysis of evaluable results. Thus, 1,875 specimens were analyzed in this clinical evaluation.

# Assay Performance vs. Enriched Culture & Bi-Directional Sequencing

Relative to enriched culture with bi-directional sequencing, the Verigene CDF Test demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 1 . 9 \%$ and $9 2 . 5 \%$ , respectively. The CDF Test also demonstrated a $9 3 . 7 \%$ positive agreement and $9 8 . 7 \%$ agreement for the hypervirulent C. difficile strain 027 by bi-directional sequencing. The results are summarized in the table below (this data combines all investigational sites).

Verigene CDF Test Performance vs. Enriched Culture & Sequencing   

<table><tr><td rowspan=1 colspan=4>Enriched Culture &amp; Sequencing</td></tr><tr><td rowspan=5 colspan=1> Ie e</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>Tox C. difficile +027 -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totala</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027+</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>82</td></tr><tr><td rowspan=1 colspan=1>Tox C. difficile +027</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>289</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1480</td><td rowspan=1 colspan=1>1502</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>1600</td><td rowspan=1 colspan=1>1873b</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile/027</td></tr><tr><td rowspan=1 colspan=2>Sensitivity: 91.9% (251/273)(88.1%-94.9%)Specificity: 92.5% (1480/1600)(91.1%-93.7%)Accuracy: 92.4% (1731/1873)(91.1%-93.6%)PPV: 67.7% (251/371)(62.6%-72.4%)NPV: 98.5% (1480/1502)(97.8%-99.1%)</td><td rowspan=1 colspan=2>Pos Agreement: 93.7% (59/63)(84.5%-98.2%)Neg Agreement: 98.7% (1787/1810)(98.1%-99.2%)Total Agreement: 98.6% (1846/1873)(97.9%-99.1%)PPV: 72.0% (59/82)(60.9%-81.3%)NPV: 99.8% (1787/1791)(99.4%-99.9%)</td></tr></table>

a There were 1,877 evaluable specimens enrolled in the clinical trial; 71 specimens $( 3 . 8 \% )$ required repeat testing: 46 specimens $( 2 . 4 \% )$ had an initial $^ { 6 6 } \mathrm { N o }$ Call” result due to assay internal control errors; 17 specimens $( 1 . 0 \% )$ had an initial “Indeterminate” call (No Call-IND), and 8 specimens $( 0 . 4 \% )$ had preanalytical errors (four motor stalls, two tip failures, one cracked slide and one cartridge not detected). The eight specimens that experienced pre-analytical errors, and the $4 6 \mathrm { N o }$ Call specimens, were called upon repeat testing; however, two of the No Call specimens required a second repeat test before being called. Repeat testing of the $1 7 \mathrm { N o }$ Call-IND specimens called all but two specimens (five specimens required a second repeat test before calling). Therefore, two specimens had a final “Indeterminate” call and were not included in the clinical data analysis of evaluable results. Thus, 1,875 specimens were analyzed in this clinical evaluation.

bOf the 1,875 specimens evaluated, two specimens were culture positive but were not sequenced because the isolate was either not sent or the result was inconclusive. These two specimens were not included in the performance characteristics above.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

For the CDF Test clinical study, the prevalence rate for the clinical dataset of 1,875 prospectively-collected unformed stool specimens was determined; this study obtained specimens from five large hospitals geographically distributed across the United States. The number and percentage of toxigenic $C .$ difficile positive cases by culture, stratified by age, are presented in the table below.

<table><tr><td rowspan=1 colspan=1>Age Group (years)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Toxigenic C. difficilePrevalence(includes 027)</td><td rowspan=1 colspan=1>027 Prevalencea</td></tr><tr><td rowspan=1 colspan=1>Infant (&lt;2 years old)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0% (0/0)</td><td rowspan=1 colspan=1>0% (0/0)</td></tr><tr><td rowspan=1 colspan=1>Child (≥2 - &lt;12 years old)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>13.0% (3/23)</td><td rowspan=1 colspan=1>0% (0/23)</td></tr><tr><td rowspan=1 colspan=1>Adolescent (≥12 - &lt;18 years old)</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0% (0/14)</td><td rowspan=1 colspan=1>0% (0/0)</td></tr><tr><td rowspan=1 colspan=1>Transitional Adolescent (≥18 - ≤21)</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>38.1% (16/42)</td><td rowspan=1 colspan=1>4.8% (2/42)</td></tr><tr><td rowspan=1 colspan=1>Adults: &gt;21 - 59 Years of Age</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>16.2% (148/913)</td><td rowspan=1 colspan=1>2.7% (25/913)</td></tr><tr><td rowspan=1 colspan=1>Sr. Adults: ≥60 Years of Age</td><td rowspan=1 colspan=1>883</td><td rowspan=1 colspan=1>23.1% (204/883)</td><td rowspan=1 colspan=1>6.2% (55/883)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1875</td><td rowspan=1 colspan=1>19.8% (371/1875)</td><td rowspan=1 colspan=1>4.4% (82/1875)</td></tr></table>

aPrevalence based on the CDF test results.

# N. Instrument Name:

Verigene® System (Verigene Processor SP and Verigene Reader)

# O. System Descriptions:

1. Modes of Operation:

The Verigene System allows random access, fully automated ‘sample to result’ operation. The Verigene System is comprised of two components: a Reader (user interface, central control unit, optics for reading the absence or presence of bacterial DNA on the microarray) and the Processor SP. The CDF test, performed on the Verigene System, involves two steps:

Sample preparation-Cell lysis and magnetic bead-based bacterial DNA isolation from prepared Stool Lysis Buffer (SLB) samples, and Verigene Hybridization Test-Hybridization of bacterial-specific DNA to capture oligonucleotides on a microarray, using gold nanoparticles probe-based technology to aid detection.

Operationally, single use disposables (Extraction Tray, Utility Tray, and Test Cartridge) are loaded into separate modules within the Processor SP. The only ‘user-performed’ pipetting step required to perform the assay is the addition of the prepared SLB sample into the extraction tray. The sample preparation occurs within the Processor SP’s Extraction Tray and is also aided by reagents present in the Utility Tray, while the Verigene Hybridization Test occurs within the Test Cartridge. An automated pipette

using disposable pipette tips delivers and transfers reagents during the assay.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

The results of the validation and verification testing of the result mask and other general updates were provided for version 1.9.0 of the Verigene Reader software, and for version 4.1 of the Verigene Processor $S P$ software.

3. Specimen Identification: Specimens are labeled with a Barcode. The Processor SP and Reader detect the specimen ID, the results are printed with this specimen identifier.

4. Specimen Sampling and Handling: Automated Verigene System

5. Calibration: Not required

6. Quality Control:

A series of internal controls used in conjunction with procedural checks monitors instrument functionality, performance, fluidics, reagent integrity, and result determination, based on a pre-defined decision algorithm.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.