Gen-Probe Prodesse, Inc. ProGastro Assay 5 10(k) Submission

Page 1 of 11   
01/15/2013

# Attachment D 510(k) SUMMARY

CONTACT Karen Harrington, PhD Manager, Clinical Affairs Gen-Probe Prodesse, Inc. 20925 Crossroads Circle Waukesha, WI 53186

NAME OF DEVICE Trade Name: Regulation Number: Product Code:

# PREDICATE DEVICE

K121454 Luminex GPP Test

# INTENDED USE

The Prodesse $\mathfrak { Q }$ ProGastro SSCS Assay is a multiplex real time PCR in vitro diagnostic test for the qualitative detection and differentiation of Salmonella, Shigella, and Campylobacter (C. jejuni and C. coli only, undifferentiated) nucleic acids and Shiga Toxin 1 $( s t x l )$ and Shiga Toxin 2 $( s t x 2 )$ genes. Shiga toxin producing E. coli (STEC) typically harbor one or both genes that encode for Shiga Toxins  and 2. Nucleic acids are isolated and purified from preserved stool specimens obtained from symptomatic patients exhibiting signs and symptoms of gastroenteritis. This test is intended for use, in conjunction with clinical presentation and epidemiological risk factors, as an aid in the differential diagnosis of Salmonella, Shigella, Campylobacter jejuni/Campylobacter coli, and STEC infections in humans.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative ProGastro SSCS Assay results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

# PRODUCT DESCRIPTION

The ProGastro SSCS Assay enables detection and differentiation of Salmonella, Shigella, Campylobacter (. jejuni and C. coli only, undifferentiated) and an Internal Control in the SSC Mix and Shiga Toxin Producing E. coli (STEC, stx1 and $s t x 2$ differentiated) and an Internal Control in the STEC Mix.

An overview of the procedure is as follows:

1.Collect raw stool specimens from symptomatic patients and place into Cary Blair Transport Medium or ParaPak C&S (C&S) Transport Medium .   
Add the Gastro RNA/DNA Internal Control (GIC) to every sample to monitor for inhibitors present in the specimens.   
Perform isolation and purification of nucleic acids using a NucliSENS easyMAG System and the

Automated Magnetic Extraction Reagents (bioMérieux).

4. Add purified nucleic acids to the SSC Mix included in the ProGastro SSCS Assay Kit. The SSC Mix contains target-specific oligonucleotide primers and probes for detection of Salmonella, Shigella, and Campylobacter $C .$ jejuni and $C .$ coli only). The primers and probes are complementary to highly conserved regions of genetic sequences for these organisms. The probes are dual-labeled with a reporter dye and a quencher (see table below).

5. Add purified nucleic acids to the STEC Mix included in the ProGastro SSCS Assay Kit. The STEC Mix contains target-specific oligonucleotide primers and probes for detection of Shiga Toxin 1 and 2 genes ( $. s t x l$ and stx2). The primers and probes are complementary to highly conserved regions of these genes. The probes are dual-labeled with a reporter dye and a quencher (see table below).

Perform amplification of DNA in a Cepheid SmartCycler II instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProGastro SSCS Assay is based on Taqman reagent chemistry, which utilizes the $5 ' - 3 '$ EPY exonuclease activity of Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

<table><tr><td rowspan=1 colspan=1>Supermix</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=2>GeneTargeted</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>Peak</td><td rowspan=1 colspan=1>Peak</td><td rowspan=1 colspan=1>Channel</td></tr><tr><td rowspan=2 colspan=1>SSC Mix</td><td rowspan=2 colspan=1>Campylobacter(C. jejuni and C.coli only</td><td rowspan=1 colspan=1>C.jejuni</td><td rowspan=1 colspan=1>C.coli</td><td rowspan=2 colspan=1>FAM</td><td rowspan=2 colspan=1>495 nm</td><td rowspan=2 colspan=1>520 nm</td><td rowspan=2 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>glyA</td><td rowspan=1 colspan=1>cadF</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=2>orgC</td><td rowspan=1 colspan=1>CAL FluorOrange 560</td><td rowspan=1 colspan=1>538 nm</td><td rowspan=1 colspan=1>559 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=2>ipaH</td><td rowspan=1 colspan=1>CAL FluorRed 610</td><td rowspan=1 colspan=1>590 nm</td><td rowspan=1 colspan=1>610 nm</td><td rowspan=1 colspan=1>Texas Red</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=2>stx1</td><td rowspan=1 colspan=1>CAL FluorOrange 560</td><td rowspan=1 colspan=1>538 nm</td><td rowspan=1 colspan=1>559 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=2>stx2</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>SSC Mixand STECMix</td><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>670 nm</td><td rowspan=1 colspan=1>Cy5</td></tr></table>

# SUBSTANTIAL EQUIVALENCE

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element*</td><td rowspan=1 colspan=1>Prodesse ProGastro SSCS (k123274)</td><td rowspan=1 colspan=1>ELuminex xTAG GPP (k121454)MA</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Salmonella spp., Shigella spp.,Campylobacter (C. jejuni and C. coli),and STEC (stx1 and stx2 genes)</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Stool specimens</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>User Complexity</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>Up front sample processing is requiredto extract nucleic acids</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Internal control in each sample.External control processed with eachbatch of samples.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>oaC</td></tr><tr><td rowspan=1 colspan=1>Element x^</td><td rowspan=1 colspan=1>Prodesse ProGastro SSCS (k123274)</td><td rowspan=1 colspan=1>Luminex xTAG GPP (k121454)$\\frac   6}$</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>(See above for similarities)</td><td rowspan=1 colspan=1>Can also detect and distinguish C. lari.In addition, can detect and distinguishClostridium difficile toxin A/B,Cryptosporidium (C. parvum and C.hominis only), Escherichia coli (E.coli) O157, Enterotoxigenic E. coli(ETEC) LT/ST, Giardia (G. lambliaonly), Norovirus GI/GII, andRotavirus A.</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>RNA/DNA</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Real time multiplex PCR based on theTaqman reagent chemistry</td><td rowspan=1 colspan=1>Multiplex RT-PCR and beadhybridization followed byFluorescence-activated sorting oflabeled beads coupled to streptavidin-conjugated biotinylated products</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>Cepheid SmartCycler II</td><td rowspan=1 colspan=1>PCR Thermocycler and Luminex100/200 system</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>Approximately 3 hours</td><td rowspan=1 colspan=1>Approximately 6 to 12 hours</td></tr></table>

# Clinical Performance

# Prospective Study

The clinical performance of the ProGastro SSCS Assay was established during prospective studies at four U.S. clinical laboratories. Leftover stool samples were collected during July 2011  November 2011 and May 2012  July 2012 and tested during November 201 1 thru August 2012. All specimens used in the study meeting the inclusion criteria represented excess remnants of stool specimens that were prospectively collected from symptomatic individuals suspected of gastrointestinal infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded.

Demographic details for the patient population included in the prospective study are summarized in the following table.

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Samples SSC Mix Number of Samples STEC Mix</td><td rowspan=1 colspan=1>Number of Samples SSC Mix Number of Samples STEC Mix</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>615/1214 (50.6%)</td><td rowspan=1 colspan=1>615/1214 (50.6%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>581/1214 (47.9%)</td><td rowspan=1 colspan=1>581/1214 (47.9%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>18/1214 (1.5%)</td><td rowspan=1 colspan=1>18/1214 (1.5%)</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>378/1214 (31.1%)</td><td rowspan=1 colspan=1>378/1214 (31.1%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>296/1214 (24.4%)</td><td rowspan=1 colspan=1>296/1214 (24.4%)</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>357/1214 (29.4%)</td><td rowspan=1 colspan=1>357/1214 (29.4%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>164/1214 (13.5%)</td><td rowspan=1 colspan=1>164/1214 (13.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>19/1214 (1.6%)</td><td rowspan=1 colspan=1>19/1214 (1.6%)</td></tr></table>

Performance of the ProGastro SSCS Assay was assessed and compared to the reference method of culture (Campylobacter, Salmonella, and Shigella) or broth enrichment followed by FDA cleared EIA test (Shiga Toxin proucig E. col.Smples posiive or TE by broth/EIA and/or the ProGasro SCSAssay underwent PCR followed by bi-directional sequencing to confirm the presence of the stx1 and/or stx2 genes. Two PCR/sequencing assays were used that each targeted different regions of the stx/ or stx2 gene than the ProGastro SSCS Assay. "True" STEC positives were considered as any sample that tested positive for STEC by the broth/EIA method, and "True" STEC negatives were considered as any sample that tested negative for STEC by the broth/EIA method. "True" stx1 or "true" stx2 positives were considered as any sample that tested positive for STEC by the broth/EIA method and by PCR/sequencing. Bi-directional sequencing data was required to meet pre-defined quality acceptance criteria for both the forward and the reverse sequences that matched stx1 or stx2 sequences deposited in the National Center for Biotechnology Information (NCBl) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used.The lower the E-Value, the more significant the hit is.A sequence alignment that has an E-Value of le-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone.   
(http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid $=$ handbook.section.614).

Discrepant results between the ProGastro SSCS Assay and the reference methods were also evaluated u alytialy vlt u ssys n esul   e ebles below.

A total of 1214 patients were initially enrolled in the prospective clinical trial. Prospective stool specimens were initially included in the prospective clinical trial. Sixty-one (61) patient/specimens were excluded from the performance calculations due to deviations from the clinical study protocol. Fourteen (14) specimens were excluded for the SSC Mix and 14 were excluded for the STEC Mix from the prospective clinical study data analysis because they remained Unresolved" after repeat testing with the respective ProGastro SSCS Assay Mix. Unresolved results occur when the sample is negative for all target detections and the Interal Control, indicating potentially PCR-inhibited samples.This resulted in a total of 1139 eligible prospective specimens to be included in the prospective clinical study data analysis.

Campylobacter (C. jejuni / C. coli) Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PoOSSSSAssy</td><td rowspan=1 colspan=1>Possitive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>13a</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Sensitivity 100.0%(83.9% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1106</td><td rowspan=1 colspan=1>1106</td><td rowspan=1 colspan=1>Specificity 98.8%(98.0% - 99.3%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tal</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1119</td><td rowspan=1 colspan=1>1139</td><td rowspan=1 colspan=1></td></tr></table>

i analysis.

Salmonella Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>POoSOSSAsssE</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>10a</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Sensitivity 95.2%(77.3% - 99.2%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1108</td><td rowspan=1 colspan=1>1109</td><td rowspan=1 colspan=1>Specificity 99.1%(98.4% - 99.5%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1118</td><td rowspan=1 colspan=1>1139</td><td rowspan=1 colspan=1></td></tr></table>

a Ten (10) samples were positive for Salmonella by bi-directional sequence analysis. S a s Sy l ays.

Shigella Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PosOSOSSAsss</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>6a</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Sensitivity 100.0%(79.6% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1118</td><td rowspan=1 colspan=1>1118</td><td rowspan=1 colspan=1>Specificity 99.5%(98.8% - 99.8%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATotal:..</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1124</td><td rowspan=1 colspan=1>1139</td><td rowspan=1 colspan=1></td></tr></table>

Six (6) samples were positive for Shigella by bi-directional sequence analysis.

Gen-Probe Prodesse, Inc.

STEC Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Broth Enrichment/EIA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PosOSOSSAsssE</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9$</td><td rowspan=1 colspan=1>9b</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Sensitivity 100.0%(70.1% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1121</td><td rowspan=1 colspan=1>1121</td><td rowspan=1 colspan=1>Specificity 99.2%(98.5% - 99.6%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1130</td><td rowspan=1 colspan=1>1139</td><td rowspan=1 colspan=1></td></tr></table>

Six ) samples positive or $s t x l$ smple posiive or $s t x 2$ , and two ) ss poive and stx2 by bi-directional sequence analysis. analysis.

stx1 Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Broth Enrichment/EIA andsequencing for stx1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PooOSSSs Soss</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6a</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Positive Percent Agreement100.0%(67.6% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Negative Percent Agreement 40.0%(16.8% - 68.7%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1></td></tr></table>

y/EAvw os  lu were negative by Broth Enrichment/EIA.

stx2 Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Broth Enrichment/EIAand sequencing for stx2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>POSS SsS</td><td rowspan=2 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Positive Percent Agreement100.0%(43.9% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Negative Percent Agreement 80.0%(54.8% - 93.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1></td></tr></table>

$s t x 2$ ba Enrichment/EIA.

The ProGastro SSCS Assay detected one mixed infections in the prospective clinical evaluation. This represents $0 . 9 8 \%$ of the total positive specimens (1/102). The one mixed infections sample was double infections and was confirmed by the reference methods.

Distinct Co-infection Combinations Detected by the ProGastro SSCS Assay in the Prospective Clinical Trial

<table><tr><td rowspan=2 colspan=3>Distinct Co-infection CombinationsDetected by ProGastro SSCS AssayAnalyte 1         Analyte 2        Analyte 3</td><td rowspan=2 colspan=1>conToot</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=2 colspan=1>Discrepant Analyte(s)a</td></tr><tr><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Y</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Co-infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Double Infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Triple Infections</td><td rowspan=1 colspan=1>.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

iepn o-eion  iepn analy wa fs e hat wa te  he roGas SSCS Assay but not detected by the reference methods.

There were no co-infections that were detected by the reference method and not detected by the ProGastro SSCS Assay.

# Retrospective Study

In addition to the prospective clinical study, two clinical sites also performed testing using retrospective samples that were collected from 2007 - 2011. A total of 105 stool samples were included in the retrospective study. These samples had been previously determined to be positive or negative by culture and/or Broth Enrichment/EIA. The ProGastro SSCS Assay was compared to the same reference method that was employed for the prospective study to determine positive and negative percent agreement.

Demographic details for this patient population are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>Sex*</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>24/55 (43.6%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>31/55 (56.4%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>12/105 (11.4%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>24/105 (22.9%)</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>51/105 (48.6%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>18/105 (17.1%)</td></tr></table>

\*For all of the 50 specimens tested from one site the gender was unknown

Gen-Probe Prodesse, Inc.

Campylobacter Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=2 colspan=1>PoOSSss Sss</td><td rowspan=2 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Positive Percent Agreement 96.4%(82.3% - 99.4%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>Negative Percent Agreement93.5%(85.7% - 97.2%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1></td></tr></table>

Salmonella Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>POOSSSs SsS</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Positive Percent Agreement100.0% (43.4% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>Negative Percent Agreement100.0%(96.4% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1></td></tr></table>

Shigella Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=2 colspan=1>POoOSSSS SosS</td><td rowspan=2 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Positive Percent Agreement100.0%(51.0% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Negative Percent Agreement100.0%(96.3% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1></td></tr></table>

STEC Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Positive Negative Total</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PoonOSOSSAsssy</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>19a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Positive Percent Agreement 100.0%(83.2% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>Negative Percent Agreement 100.0%(95.7% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1></td></tr></table>

a Five (5) samples positive for stx1, 5 samples positive for $s ( x 2$ , and 9 samples positive for stxl and stx2.

stx1 Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIAand sequencing for stxl</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative.</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PooSOSSAsssy</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Positive Percent Agreement 100.0%(78.5% - 100.0%) 95% CI</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative Percent Agreement 100.0%(56.6% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1></td></tr></table>

stx2 Comparison Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIAand sequencing for stx2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Positive Negative Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>POrOSOSSAsss</td><td rowspan=2 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Positive Percent Agreement 100.0%(78.5% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative Percent Agreement 100.0%(56.6% - 100.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1></td></tr></table>

Of the prospective and retrospective specimens run using the SSC Mix Assay, $9 8 . 0 \%$ (1233/1258) of these specimens were successful on the first attempt. The remaining 25 $( 2 5 / 1 2 5 8 = 2 . 0 \% )$ gave Unresolved" results on the first attempt.An Unresolved" result is generated when the Gastro Internal Control (GIC) fils to be detected in a clinical specimen.A failure of the GIC to be detected canocur i inhibitors are present in a sample or due to technical error (e.g., GIC not added prior to nucleic acid extraction). Of the 5 Unreolvd" specimens on the first attempt with sufficient nucleic acid or st $4 4 . 0 \%$ (1 1/25) gave a valid result on the second attempt. The remaining 14 were "Unresolved" on the second attempt.

Of the prospective and retrospective specimens run using the STEC Mix Assay, $9 7 . 9 \%$ (1232/1258) of these specimens were successful on the first attempt. The remaining 26 $( 2 6 / 1 2 5 8 = 2 . 1 \% )$ gave Unresolved" results on the first attempt.Of the 26 "Unresolved" specimens on the first attempt with sufficient nucleic acid for retest, $4 8 . 0 \%$ (12/25) gave a valid result on the second attempt. The remaining 14 were "Unresolved" on the second attempt.

# Reproducibility

The reproducibility of the ProGastro SSCS Assay was evaluated at three laboratory sites. Reproducibility was assessed using a panel of 15 simulated samples that included medium positive, low positive (near the assay limit of detection, $\ge 9 5 \%$ positive), and high negative (below the assay limit of detection, $\leq 9 5 \%$ positive samples for each of the assay targets.The reproducibility panel and controls were run with the ProGastro SSCS Assay (SSC and STEC Mixes) at three sites by each of two operators per site for five days.

Page 10 of 1 1 01/15/2013   
Reproducibility Panel Member Results   

<table><tr><td rowspan=2 colspan=15>SSCSSC Mix                              STEC MixMixeooiid          cood m te  Toid Ca                 Pold   estid          Poidod o ineFosidcnOod ACwrpn ni   :PanelmmAeMember IDMcooM=aheSoMPsheMinCCBBA</td><td></td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>STECMixUaSHon Hh</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>20X*LoD</td><td rowspan=1 colspan=1>100X*LoD</td><td rowspan=1 colspan=1>6X*LoD</td><td rowspan=1 colspan=1>30X*LoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=2>0.0001XLoD</td></tr><tr><td rowspan=9 colspan=1>Site 1Site 2Site 3</td><td rowspan=1 colspan=1>Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3010%%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>33.2</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>33.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Agreementwith xpectedResult</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>32.6</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwith ExpectedResult</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>89/9089.9%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>270/2701000%</td><td rowspan=1 colspan=1>270/270100%.</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9%100.0%100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>94.0%99.8%</td><td rowspan=1 colspan=1>95.9%100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%100.0%</td><td rowspan=1 colspan=1>98.6%-100.0%</td><td rowspan=1 colspan=1>98.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall MeanCt Value</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>32.9</td></tr><tr><td rowspan=1 colspan=1>Overall%CV</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.5</td></tr></table>

\* Note: The Campylobacter strains were tested at higher concentrations because Campylobacter is sensitive to environmental stressors including freezing where it loses viability. However, the average Ct value for the C. jejuni (ATCC 33291) Low Positives was 37.3, which is very close to the average Ct value of38.8 for the same C. jejuni strain tested at the estimated LoD level in the Analytical Reactivity Study. The average Ct value for the C. coli (ATCC BAA-371) Low Positives was 35.6, which is very close to the average Ct value of 34.4 for the same C. coli strain tested at the estimated LoD level in the Analytical Reactivity Study. Therefore, the effective DNA concentrations of the C. jejuni (ATCC 33291) and C. coli (ATCC BAA-371) low positive samples tested in the Reproducibility Study are very close to the estimated LoD DNA concentrations for these two strains tested in the Analytical Reactivity Study.

# Precision

The precision of the ProGastro SSCS Assay was evaluated internally using a panel of 15 simulated samples that included medium positive, low positive (near the assay limit of deteçtion, $\ge 9 5 \%$ positive), and high negative (below the assay limit of detection, $\leq 9 5 \%$ positive samples for each of the assay targets. A panel of 15 contrived samples including the necessary controls was run with the ProGastro SSCS Assay (SSC and STEC Mixes) by each of two operators for twelve days.

Precision Panel Member Results   

<table><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>20X*LoD</td><td rowspan=1 colspan=1>100X*LoD</td><td rowspan=1 colspan=1>6X*LoD</td><td rowspan=1 colspan=1>30X*LoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=2>0.0001X LoD</td></tr><tr><td rowspan=1 colspan=1>TotalAgreementwith ExpectedResult</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>71/7298.6%</td><td rowspan=1 colspan=1>72/72100%</td></tr><tr><td rowspan=1 colspan=1>Overall MeanCt Value</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>32.5</td></tr><tr><td rowspan=1 colspan=1>Overall % CV</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.0</td></tr></table>

Note: The Campylobacter strains were tested at higher concentrations because Campylobacter is sensitive to environmental stressors including freezing where it loses viability. However, the average Ct value for the C. jejuni (ATCC 33291) Low Positives was 37.3, which is very close to the average Ct value o38.8 for the same C. jejuni strain tested at the estimated LoD level in the Analytical Reactivity Study. The average Ct value for the C. coli (ATCC BAA-371) Low Positives was 35.6, which is very close to the average Ct value of 34.4 or the same C. coli strain tested at the estimated LoD level in the Analytical Reactivity Study. Therefore, the effective DNA concentrations of the C. jejuni (ATCC 33291) and C. coli (ATCC BAA-371) low positive samples tested in the Precision Study are very close to the estimated LoD DNA concentrations for these two strains tested in the Analytical Reactivity Study.

Gen-Probe Prodesse, Inc.

Gen-Probe Prodesse, Inc. c/o Karen Harrington, Ph.D. Manager, Clinical Affairs 20925 Crossroads Circle Waukesha, WI 53186

Re: k123274 Trade/Device Name: ProGastro SSCS Assay Regulation Number: 21 CFR 866.3990 Regulation Name: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assay Regulatory Class: Class II Product Code: PCH, PCI, OOI Dated: October 17, 2012 Received: October 19, 2012

Dear Dr. Harrington:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or classIII (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set

Page 2 - Karen Harrington

forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000-1050.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostics and Radiological Health at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

# Uwe Scherf for

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics and Radiological Health Center for Devices and Radiological Health

# Enclosure

# Indication for Use

510(k) Number: k123274

Device Name: ProGastro SSCS Assay

Indication For Use:

The Prodesse® ProGastro SSCS Assay is a multiplex real time PCR in vitro diagnostic test for the qualitative detection and differentiation of Salmonella, Shigella, and Campylobacter (C. jejuni and C. coli only, undifferentiated) nucleic acids and Shiga Toxin 1 $( s t x l )$ and Shiga Toxin 2 $( s t x 2 )$ genes. Shiga toxin producing E. coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2. Nucleic acids are isolated and purified from preserved stool specimens obtained from symptomatic patients exhibiting signs and symptoms of gastroenteritis. This test is intended for use, in conjunction with clinical presentation and epidemiological risk factors, as an aid in the differential diagnosis of Salmonella, Shigella, Campylobacter jejuni/Campylobacter coli, and STEC infections in humans.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative ProGastro SSCS Assay results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

# Concurrence of CDRH, Office of In Vitro Diagnostics and Radiological Health (OIR)

Division Sign-Off   
Office of In Vitro Diagnostics and Radiological Health   
k123274