# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number:

K123998

# B. Purpose for Submission:

Substantial equivalence determination for the Quidel Molecular Direct C. difficile Assay on: (i) the Applied Biosystems 7500 Fast Dx instrument; (b) the Cepheid SmartCycler II instrument, and (c) the Life Technologies QuantStudio® Dx Real-Time PCR system.

# C. Measurand:

Targets DNA sequences of the toxin A (tcdA), and toxin B (tcdB) genes within the PaLoc of toxigenic strains of Clostridium difficile.

# D. Type of Test:

Nucleic acid amplification test based on multiplex, real-time.

# E. Applicant:

Quidel Corporation

F. Proprietary and Established Names: Quidel Molecular Direct C. difficile Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR §866.3130 - C. difficile Nucleic Acid Amplification Test Assay

2. Classification: II

3. Product codes:

OZN - Amplification assay for the detection of Clostridium difficile toxin genes from stool specimens of symptomatic patients

OOI - Real-Time Nucleic Acid Amplification System

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Quidel Molecular Direct C. difficile Assay is a qualitative, multiplexed in vitro diagnostic test for the direct rapid detection of toxin A gene (tcdA) or toxin B gene (tcdB) sequences of toxigenic strains of Clostridium difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile-Associated Disease (CDAD).

The Quidel Molecular Direct C. difficile Assay is a real-time PCR test and utilizes proprietary sample preparation with fluorescently labeled primers and probes. The assay can be performed using either the Life Technologies QuantStudio® Dx; the Applied Biosystems 7500 Fast Dx, or the Cepheid SmartCycler II, to detect the toxin gene sequences associated with toxin-producing C. difficile strains.

The assay is intended to be performed directly on CDAD-suspected stool specimens, and is indicated for use as an aid in the diagnosis of CDAD.

# 2. Indication(s) for use:

The Quidel Molecular Direct C. difficile Assay is a qualitative, multiplexed in vitro diagnostic test for the direct rapid detection of toxin A gene (tcdA) or toxin B gene (tcdB) sequences of toxigenic strains of Clostridium difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile-Associated Disease (CDAD).

The Quidel Molecular Direct C. difficile Assay is a real-time PCR test and utilizes proprietary sample preparation with fluorescently labeled primers and probes. The assay can be performed using either the Life Technologies QuantStudio® Dx; the Applied Biosystems 7500 Fast Dx, or the Cepheid SmartCycler II, to detect the toxin gene sequences associated with toxin-producing C. difficile strains.

The assay is intended to be performed directly on CDAD-suspected stool specimens, and is indicated for use as an aid in the diagnosis of CDAD.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: Life Technologies QuantStudio® Dx with software version 1 Applied Biosystems 7500 Fast Dx with software version 1.4 Cepheid SmartCycler II with software version 3.0b

# I. Device Description:

The Quidel Molecular Direct C. difficile Assay is a qualitative in vitro diagnostic test for the direct rapid detection of toxigenic Clostridium difficile toxin A gene (tcdA) or toxin B gene (tcdB) from stool samples of patients suspected of having Clostridium difficile-Associated Diarrhea (CDAD). The Quidel Molecular Direct C. difficile Assay utilizes proprietary sample preparation and Real-time PCR to detect the amplified DNA from patient specimens. A multiplex, real-time PCR reaction is performed under optimized conditions in a single tube/well generating amplicons for each of the targets present in the specimen. Identification occurs by the use of oligonucleotide primers and probes that are complementary to conserved regions in the tcdA and tcdB genes of the pathogenicity locus (PaLoc).

A swab is used to transfer a small amount of specimen into a process buffer tube (PB1). The diluted sample is then transferred into a second process buffer tube (PB2) which contains the assay’s process control (PRC). The processed specimen is then combined with rehydrated master mix in either a reaction tube or plate well. The Master Mix contains oligonucleotide primers, fluorophore and quencher-labeled probes targeting conserved regions of the tcdA and tcdB as well as the process control sequence. The reaction tubes or plate is then placed into either the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx instrument, the Life Technologies QuantStudio™ Dx RealTime PCR system, or the Cepheid SmartCycler $\textsuperscript { \textregistered }$ II instrument.

Once the reaction tube or plate is added to the instrument, the Quidel Molecular Direct C. difficile Assay protocol is initiated. This assay is based on Taqman $^ \mathrm { \textregistered }$ chemistry, and uses an enzyme with DNA polymerase, and $5 ' - 3 '$ exonuclease activities. During DNA amplification, this enzyme cleaves the probe bound to the complementary DNA sequence, separating the quencher dye from the reporter dye. This step generates an increase in fluorescent signal upon excitation by a light source of the appropriate wavelength. With each cycle, additional dye molecules are separated from their quenchers resulting in an increase in the fluorescent signal. If sufficient fluorescence is achieved, the sample is reported as positive for the detected nucleic acid.

The Quidel Molecular Direct C. difficile Assay contains sufficient reagents to process 96 specimens or quality control samples. The kit contains the following:

# Assay Kit (96 Reactions)

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Quantity</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Rehydration Solution</td><td rowspan=1 colspan=1>1 vial/kit 1.9 mL</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Quidel Molecular C. difficile Master Mix</td><td rowspan=1 colspan=1>12 vials/kit8 reactions/vial</td></tr></table>

# Rapid DNA Stool Sample Prep Kit (96 Specimens)

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Quantity</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Process Buffer 1</td><td rowspan=1 colspan=1>96 tubes/kit 500 μL</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Process Buffer 2Contains Process Control</td><td rowspan=1 colspan=1>96 tubes/kit 570 μL</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Neonatal flocked Swabs</td><td rowspan=1 colspan=1>96 swabs</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Great Basin Scientific Portrait Toxigenic C. difficile Assay

2. Predicate 510(k) number(s): K113358

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate (Great BasinScientific PortraitToxigenic C. difficileAssay)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Quidel MolecularDirect C. difficile Assay isa qualitative, multiplexed invitro diagnostic test for thedirect rapid detection oftoxin A gene (tcdA) ortoxin B gene (tcdB)sequences of toxigenicstrains of Clostridium</td><td colspan="1" rowspan="1">Portrait Toxigenic C.difficile Assay, aprescription device under21 CFR Part 801.109 that isindicated for the detectionof toxigenic Clostridiumdifficile in human fecalsamples collected frompatients suspected of</td></tr><tr><td>Item</td><td>Device</td><td>Predicate (Great Basin Scientific Portrait Toxigenic C. difficile Assay)</td></tr><tr><td></td><td>difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile-Associated Disease (CDAD). The Quidel Molecular Direct C. difficile Assay is a real-time PCR test and utilizes proprietary sample preparation with fluorescently labeled primers and probes. The assay can be performed using either the Life Technologies QuantStudio® Dx; the Applied Biosystems 7500 Fast Dx, or the Cepheid SmartCycler II, to detect the toxin gene sequences associated with toxin- producing C. difficile strains. The assay is intended to be performed directly on CDAD-suspected stool specimens, and is indicated for use as an aid in the</td><td>having Clostridium difficile infection (CDI). The test utilizes automated blocked primer enabled helicase- dependent amplification (bpHDA) to detect toxin gene sequences associated with toxin producing C. difficile. The Portrait Toxigenic C. difficile Assay is intended as an aid in the diagnosis of CDI.</td></tr><tr><td>Specimen Assay time</td><td>diagnosis of CDAD. Unformed stool</td><td>Same</td></tr><tr><td>Detection method</td><td>75 to 90 min Automated</td><td>Same Same</td></tr><tr><td>Technological principle</td><td>Fully automated nucleic acid amplification</td><td>Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Assay technique</td><td colspan="1" rowspan="1">Multiplex real-time PCRreaction</td><td colspan="1" rowspan="1">Isothermal, helicase-dependent nucleic acidamplification</td></tr><tr><td colspan="1" rowspan="1">Test Container</td><td colspan="1" rowspan="1">Manual amplification set-up in PCR microfuge tubesor plates with wells.</td><td colspan="1" rowspan="1">Disposable single-use,multi-chambered fluidictest cartridge.</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Toxin A gene (tcdA) andToxin B gene (tcdB)</td><td colspan="1" rowspan="1">Toxin B gene (tcdB)</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">QuantStudioDx; theApplied Biosystems 7500Fast Dx, or the CepheidSmartCycler II</td><td colspan="1" rowspan="1">Great BasinPortrait Analyzer</td></tr><tr><td colspan="1" rowspan="1">Detection technique</td><td colspan="1" rowspan="1">PCR with fluorescentlylabeled primers and probes;detection based onTaqman® chemistry:Amplification occurs withan enzyme that has bothDNA polymerase and 5'-3'exonuclease activities.During DNA amplificationthis enzyme cleaves theprobe bound to thecomplementary DNAsequence, separating thequencher dye from thereporter dye causing anincrease in fluorescentsignal with each cycle.</td><td colspan="1" rowspan="1">Amplification primers arebiotin-labeled primers andhybridized to probesimmobilized on a siliconchip. Incubation with anti-biotin antibody conjugatedto the HRP with TMBallows visualization by thePortrait Analyzer</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile – Draft Guidance for Industry and FDA Staff (issued on November 29, 2010)

# L. Test Principle:

The Quidel Molecular Direct C. difficile Assay is a rapid, in vitro diagnostic test for the qualitative detection of $C .$ . difficile DNA directly from unformed (liquid or soft) stool specimens of patients suspected of having Clostridium difficile-Associated Disease (CDAD). The assay detects nucleic acids that have been prepared from a patient sample using proprietary sample preparation. A multiplex real-time PCR reaction is performed under optimized conditions in a single tube/well generating amplicons for each of the targets present in the sample. Identification occurs by the use of oligonucleotide primers and probes that are complementary to conserved regions in the tcdA and tcdB genes of the pathogenicity locus (PaLoc).

A neonatal flocked swab is dipped into a liquid or soft stool specimen from a pediatric or adult patient suspected of having CDAD. The swab is twirled in the first process buffer tube (PB1); $3 0 ~ \mu \mathrm { L }$ of the processed sample is then added to the second process buffer tube (PB2) which contains the process control (PRC). Separately, the lyophilized Master Mix is rehydrated using the Rehydration Solution; the Master Mix contains oligonucleotide primers, fluorophore and quencher-labeled probes targeting conserved regions of the tcdA and tcdB as well as the process control sequence. A $1 5 \mu \mathrm { L }$ portion of the rehydrated Master Mix is then placed into each reaction tube or well. To each tube/well is added $5 ~ \mu \mathrm { L }$ of prepared specimen (i.e., the processed specimen in the PB2 tube with the PRC). The tube/plate is then placed into the Life Technologies QuantStudio® Dx Real-Time PCR system, the Applied Biosystems 7500 Fast Dx instrument, or Cepheid SmartCycler® II instrument. Once the reaction tube or plate is added to the instrument, the Quidel Molecular Direct C. difficile Assay protocol is initiated. This assay is based on Taqman $^ \mathrm { \textregistered }$ chemistry.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

For the Precision/Within Laboratory Repeatability study, a blinded four-member panel consisting of C. difficile positive and negative sample was tested by two operators, twice a day using a single assay lot of Quidel Molecular Direct C. difficile Assay reagents for 12 days on all three instruments.

<table><tr><td colspan="5" rowspan="1">Precision: Applied Biosystems 7500 Fast Dx</td></tr><tr><td colspan="1" rowspan="1">C. difficile</td><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">0.3X LoD</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">% Detection</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">71%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Average Ct</td><td colspan="1" rowspan="1">18.2</td><td colspan="1" rowspan="1">20.5</td><td colspan="1" rowspan="1">25.8</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">STDEV</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">5.2%</td><td colspan="1" rowspan="1">6.2%</td><td colspan="1" rowspan="1">9.4%</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="5" rowspan="1">Precision: Life Technologies QuantStudioTM Dx</td></tr><tr><td colspan="1" rowspan="1">C. difficile</td><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">0.3X LoD</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">% Detection</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Average Ct</td><td colspan="1" rowspan="1">16.51</td><td colspan="1" rowspan="1">17.70</td><td colspan="1" rowspan="1">21.13</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">STDEV</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">6.5%</td><td colspan="1" rowspan="1">N/A</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Precision: Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=1>C. difficile</td><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>0.3X LoD</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>% Detection</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>27%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Average Ct</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1>20.6</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>6.4%</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Reproducibility: In order to confirm the reproducibility of the Quidel Molecular Direct C. difficile Assay a blinded and randomized study panel containing Clostridium difficile negative and positive samples was tested at three (3) test sites, two of which were clinical sites while the third site was internal. Each site tested a reproducibility panel and assay controls for five (5) days in triplicate on each instrument. The testing was done by two operators at each site. Each operator ran the panel once a day using one lot of Quidel Molecular Direct C. difficile Assay reagents.

<table><tr><td colspan="11" rowspan="1">Reproducibility: Applied Biosystems 7500 Fast Dx</td></tr><tr><td colspan="1" rowspan="2">Panel Member ID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">TotalResults</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">High Negative0.3xx LoD</td><td colspan="1" rowspan="1">5/29</td><td colspan="1" rowspan="1">28.8</td><td colspan="1" rowspan="1">15.0</td><td colspan="1" rowspan="1">11/30</td><td colspan="1" rowspan="1">27.1</td><td colspan="1" rowspan="1">9.0</td><td colspan="1" rowspan="1">16/30</td><td colspan="1" rowspan="1">27.6</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">32/89</td></tr><tr><td colspan="1" rowspan="1">Low Positive2x LoD</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">23.2</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">22.7</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">23.1</td><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">88/90</td></tr><tr><td colspan="1" rowspan="1">Med Positive 5xLoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.5</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.2</td><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.4</td><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">NegativeSpecimen</td><td colspan="1" rowspan="1">0/29</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/89</td></tr><tr><td colspan="1" rowspan="1">Negative Control</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">Positive Control</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">15.8</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">15.7</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="11" rowspan="1">Reproducibility: Life Technologies QuantStudio Dx</td></tr><tr><td colspan="1" rowspan="2">Panel MemberID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">TotalResults</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">High Negative0.3x LoD</td><td colspan="1" rowspan="1">8/30</td><td colspan="1" rowspan="1">22.9</td><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1">15/30</td><td colspan="1" rowspan="1">22.5</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">15/30</td><td colspan="1" rowspan="1">22.5</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">38/90</td></tr><tr><td colspan="1" rowspan="1">Low Positive2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.4</td><td colspan="1" rowspan="1">5.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.0</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Med Positive5x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">18.4</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">17.5</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">17.9</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">NegativeSpecimen</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">NegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">PositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">15.7</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">15.7</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">15.5</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">90/90</td></tr></table>

<table><tr><td rowspan=1 colspan=11>Reproducibility: Cepheid SmartCycler II</td></tr><tr><td rowspan=2 colspan=1>Panel MemberID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalResults</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>High Negative0.3x LoD</td><td rowspan=1 colspan=1>17/30</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>22/30</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>65/90</td></tr><tr><td rowspan=1 colspan=1>Low Positive2x LoD</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>20.1</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>20.1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>88/89</td></tr><tr><td rowspan=1 colspan=1>Med Positive5x LoD</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.4</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>NegativeSpecimen</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/29</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/89</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/29</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/89</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>15.1</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>90/90</td></tr></table>

The reproducibility and repeatability study results are acceptable.

# b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The Quidel Molecular Direct C. difficile Assay incorporates a process control which is included in the second process buffer tube (PB2) and is used to monitor sample processing and evaluate the presence of inhibitory substances. The process control confirms the integrity of assay reagents and detection. Additional controls are performed in accordance with end user laboratory guidelines and requirements.

# d. Detection limit:

The analytical sensitivity (limit of detection or LoD) of the Quidel Molecular Direct C. difficile Assay was determined on each instrument using quantified (CFU/mL) cultures of two C. difficile strains (ATCC BAA-1870 and ATCC BAA-1872) and serially diluted in a negative fecal matrix. Analytical sensitivity (LoD) is defined as the lowest concentration at which $9 5 \%$ of all replicates tested positive. Serial dilutions of the strains were tested and the putative LoD confirmed with 60 replicates.

<table><tr><td rowspan=3 colspan=1>Instrument</td><td rowspan=1 colspan=4>Strain</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1870</td><td rowspan=1 colspan=2>ATCC BAA-1872</td></tr><tr><td rowspan=1 colspan=1>CalculatedCFU/mL atLoD</td><td rowspan=1 colspan=1>CFU perAssay atLoD</td><td rowspan=1 colspan=1>CalculatedCFU/mL atLoD</td><td rowspan=1 colspan=1>CFU perAssay atLoD</td></tr><tr><td rowspan=1 colspan=1>Applied Biosystems 7500 Fast Dx</td><td rowspan=1 colspan=1>8.4E+04</td><td rowspan=1 colspan=1>4.2E-01</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>1.2E-01</td></tr><tr><td rowspan=1 colspan=1>Life Technologies QuantStudio</td><td rowspan=1 colspan=1>8.4E+04</td><td rowspan=1 colspan=1>4.2E-01</td><td rowspan=1 colspan=1>8.0E+03</td><td rowspan=1 colspan=1>4.0E-02</td></tr><tr><td rowspan=1 colspan=1>Cepheid SmartCycler II</td><td rowspan=1 colspan=1>8.4E+04</td><td rowspan=1 colspan=1>4.2E-01</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>1.2E-01</td></tr></table>

The final assay LoD is defined as the higher of the two strain concentrations where $9 5 \%$ positivity was observed. The final assay LoD is 4.2E-01 CFU/assay.

# e. Analytical Reactivity

The inclusivity of the Quidel Molecular Direct C. difficile Assay was evaluated using twenty-four toxigenic strains for $C .$ . difficile (see table below). Strains were tested at 2 to 3X LoD in negative specimen matrix using three different lots; this experiment was conducted on the Applied Biosystems 7500 Fast Dx instrument. Strains were reported to originate from at least five states and four countries (USA, Belgium, France and Sweden). Seven (7) toxinotypes were represented: 0, IIIb, IIIc, IV, V, VIII and XXIII. The analytical reactivity testing conducted demonstrated that the Quidel Molecular Direct C. difficile Assay can detect a broad range of toxigenic Clostridium difficile strains at 2-3X LoD.

<table><tr><td rowspan=1 colspan=7>C. difficile Strains used in Inclusivity Study</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>aka</td><td rowspan=1 colspan=1>toxinotype</td><td rowspan=1 colspan=1>ribotype</td><td rowspan=1 colspan=1>PFGE</td><td rowspan=1 colspan=1>Serogroup</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>43255</td><td rowspan=1 colspan=1>VPI 10463</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>087</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8864</td><td rowspan=1 colspan=1>20309</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>37770</td><td rowspan=1 colspan=1>UCL 7701</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A5</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>BAA-1875</td><td rowspan=1 colspan=1>5325</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>NAP 7</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>43598</td><td rowspan=1 colspan=1>1470</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>017</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>F</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>37774</td><td rowspan=1 colspan=1>UCL 8785</td><td rowspan=1 colspan=1>XXIII</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A9</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>9004</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>BAA-1874</td><td rowspan=1 colspan=1>4205</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>002</td><td rowspan=1 colspan=1>NAP 6</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>43600</td><td rowspan=1 colspan=1>2149</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>014</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>H</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>BAA-1871</td><td rowspan=1 colspan=1>4111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>001</td><td rowspan=1 colspan=1>NAP 2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>BAA-1803</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>IIIc</td><td rowspan=1 colspan=1>027</td><td rowspan=1 colspan=1>NAP 1</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>700792</td><td rowspan=1 colspan=1>14797-2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>005</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>43599</td><td rowspan=1 colspan=1>2022</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>001</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>G</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>60276</td><td rowspan=1 colspan=1>LRA 0801058&amp; others</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>60275</td><td rowspan=1 colspan=1>LRA 0801040&amp; others</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>37778</td><td rowspan=1 colspan=1>UCL T218</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>S4</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>37777</td><td rowspan=1 colspan=1>UCL T215</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>S3</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>37776</td><td rowspan=1 colspan=1>UCL T048</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>S1</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>37773</td><td rowspan=1 colspan=1>UCL 8737</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A8</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>17857</td><td rowspan=1 colspan=1>870</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>001</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>43594</td><td rowspan=1 colspan=1>W1194</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>005</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>A</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>43596</td><td rowspan=1 colspan=1>545</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>012</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>C</td></tr><tr><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>BAA-1872</td><td rowspan=1 colspan=1>4206</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>NAP 4</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>4118</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>027</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

\*These strains were used for Limit of Detection, and therefore were not run in this study.

Analytical reactivity study results are acceptable.

# f. Analytical Specificity

The analytical specificity of the Quidel Molecular Direct C. difficile Assay was evaluated by testing a panel consisting of 66 bacterial, viral and yeast microorganisms, and human DNA representing common enteric pathogens, flora, and/or nucleic acid commonly present in the intestine (see table below). Microorganisms or nucleic acid was mixed with pooled negative matrix and tested directly; this experiment was conducted on the Applied Biosystems 7500 Fast Dx instrument. Bacteria were tested at concentrations greater than $1 . 0 \mathrm { E + } 0 6 \mathrm { C F U / m L }$ and viruses at greater than $1 . 0 \mathrm { E } { + 0 5 }$ PFU/mL. In addition, in silico analysis showed that the Quidel Molecular Direct C. difficile Assay had no predicted cross-reactivity for $C .$ . botulinum. The results of this study demonstrate that the Quidel Molecular Direct C. difficile Assay does not cross-react with medically relevant levels of viruses or bacteria found in stool specimens.

<table><tr><td colspan="3" rowspan="1">Analytical Specificity &amp; Microbial Interference Panel</td></tr><tr><td colspan="1" rowspan="1">Genera and Species</td><td colspan="1" rowspan="1">CFU/mL, PFU/mL, or copies/mL</td><td colspan="1" rowspan="1">Stock Type</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">4.30E+09 copies/mL</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii (307-0294)</td><td colspan="1" rowspan="1">5.27E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1 VR-1</td><td colspan="1" rowspan="1">5.67E+05 PFU/mL</td><td colspan="1" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">2.09E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">4.65E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.00E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.19E+09 copies/mL</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">5.30E+09 copies/mL</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp. jejuni</td><td colspan="1" rowspan="1">1.72E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">3.00E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">2.38E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">2.05E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">1.75E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic) strain 1</td><td colspan="1" rowspan="1">1.13E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic) strain 2</td><td colspan="1" rowspan="1">4.58E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">3.43E+09 copies/mL</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">6.50E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">5.30E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens (Type A)</td><td colspan="1" rowspan="1">3.37E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">1.62E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">2.03E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (ATCC 9714) strain 1</td><td colspan="1" rowspan="1">1.94E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii (Z077) strain 2</td><td colspan="1" rowspan="1">2.07E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 6329) strain 3</td><td colspan="1" rowspan="1">9.85E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 9284) strain 4</td><td colspan="1" rowspan="1">6.50E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 33098) strain 5</td><td colspan="1" rowspan="1">2.00E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 36938) strain 6</td><td colspan="1" rowspan="1">5.55E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 43123) strain 7</td><td colspan="1" rowspan="1">2.50E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 47545) strain 8</td><td colspan="1" rowspan="1">1.36E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordelli (CCUG 59819) strain 9</td><td colspan="1" rowspan="1">7.00E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">3.55E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.43E+07 PFU/mL</td><td colspan="1" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus Towne VR-977</td><td colspan="1" rowspan="1">1.48E+06 PFU/mL</td><td colspan="1" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.05E+09 PFU/mL</td><td colspan="1" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">2.03E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1.31E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">5.95E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">3.45E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">4.82E+05 PFU/mL</td><td colspan="1" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.92E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7 (EDL933)</td><td colspan="1" rowspan="1">2.20E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori (Z040)</td><td colspan="1" rowspan="1">3.57E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Klebsielia oxytoca</td><td colspan="1" rowspan="1">1.63E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">6.82E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes (Serotype 1/2b)</td><td colspan="1" rowspan="1">1.18E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Norovirus GII</td><td colspan="1" rowspan="1">3.92E+08 copies/mL</td><td colspan="1" rowspan="1">RNA</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">4.00E+09 copies/mL</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.40E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">1.30E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">5.10E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.06E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">9.60E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">2.60E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Rotavirus (WA)</td><td colspan="1" rowspan="1">2.32E+08 copies/mL</td><td colspan="1" rowspan="1">RNA</td></tr><tr><td colspan="1" rowspan="1">Salmonella choleraesuis (typhimurium)</td><td colspan="1" rowspan="1">3.55E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">6.80E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. Arizonae(formerly Choleraesuis arizonae)</td><td colspan="1" rowspan="1">4.22E+09 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">3.79E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">6.10E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">8.16E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">1.26E+10 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">3.36E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">6.00E+07 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">4.00E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">2.75E+08 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">9.50E+06 CFU/mL</td><td colspan="1" rowspan="1">Bacteria</td></tr></table>

Analytical specificity study results are acceptable.

# g. Microbial Interference:

Microbial interference of the Quidel Molecular Direct C. difficile Assay was evaluated by testing a panel consisting of 66 bacterial, viral and yeast microorganisms, and human DNA representing common enteric pathogens, flora, and/or nucleic acid commonly present in the intestine (see table above). Microorganisms or nucleic acid was mixed with pooled negative matrix and tested in the presence of 2 to 3x LoD level of C. difficile; this experiment was conducted on the Applied Biosystems 7500 Fast Dx instrument. Two different strains of $C .$ difficile were used in this study (see table below). Bacteria were tested at concentrations greater than $1 . 0 \mathrm { E } { + 0 6 }$ CFU/mL and viruses at greater than $1 . 0 \mathrm { E } { + 0 5 }$ PFU/mL. The results of this study demonstrate that medically relevant levels of viruses or bacteria found in stool specimens do not interfere with the Quidel Molecular Direct C. difficile Assay.

<table><tr><td rowspan=1 colspan=4>C. difficile Strains used in Microbial Interference Study</td></tr><tr><td rowspan=1 colspan=1>StrainNumber</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>C. difficileStrain</td><td rowspan=1 colspan=1>CFU/PCRreaction</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>0.84</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>BAA-1872</td><td rowspan=1 colspan=1>0.30</td></tr></table>

# h. Interfering Substances

Two toxigenic strains of $C .$ . difficile (ATCC BAA-1870 and ATCC BAA-1872) were evaluated against a test panel consisting of thirty-five substances found in stool specimens.

<table><tr><td rowspan=1 colspan=4>C. difficile Strains used in Interfering Substances Study</td></tr><tr><td rowspan=1 colspan=1>StrainNumber</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>C. difficileStrain</td><td rowspan=1 colspan=1>CFU/PCRreaction</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>0.84</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>BAA-1872</td><td rowspan=1 colspan=1>0.30</td></tr></table>

Substances were introduced into the assay dilution tubes at concentrations which were medically relevant. Each of the strains was tested for each substance; this experiment was conducted on the Applied Biosystems 7500 Fast Dx instrument. None of the substances tested were found to interfere with the Quidel Molecular Direct C. difficile Assay.

<table><tr><td rowspan=1 colspan=1>Substance ID</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Palmitic Acid</td><td rowspan=1 colspan=1>1.3 mg/mL</td><td rowspan=1 colspan=1>100% Methanol</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Triclosan</td><td rowspan=1 colspan=1>0.1%(w/v)</td><td rowspan=1 colspan=1>20% DMSO</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Methicillin</td><td rowspan=1 colspan=1>13 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Phenylephrine HCl</td><td rowspan=1 colspan=1>2% w/v</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>Phenylephrine HCl</td><td rowspan=1 colspan=1>cream</td><td rowspan=1 colspan=1>Full swab</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Stearic Acid</td><td rowspan=1 colspan=1>26 mg/mL</td><td rowspan=1 colspan=1>100% DMSO</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>2% v/v</td><td rowspan=1 colspan=1>10% DMSO</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>14 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Aluminum Hydroxide</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Magnesium Hydroxide</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>3 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Cimetidine</td><td rowspan=1 colspan=1>0.5 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Esomeprazole MagnesiumHydrate</td><td rowspan=1 colspan=1>0.5 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>10000 USP U/mL</td><td rowspan=1 colspan=1>DPBS</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Human Serum Albumin</td><td rowspan=1 colspan=1>10 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>0.87 mg/mL</td><td rowspan=1 colspan=1>100% DMSO</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Calcium Carbonate</td><td rowspan=1 colspan=1>0.5 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Glucose</td><td rowspan=1 colspan=1>1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Loperamide HCl</td><td rowspan=1 colspan=1>1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Human Hemoglobin</td><td rowspan=1 colspan=1>3.2 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Benzalkonium Cl</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5-Aminosalicylic acid</td><td rowspan=1 colspan=1>2 mg/mL</td><td rowspan=1 colspan=1>100% DMSO</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>Petroleum Jelly</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Full swab</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>Cortisol</td><td rowspan=1 colspan=1>1% hydrocortisonecream</td><td rowspan=1 colspan=1>1/2 swab</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Zinc Oxide</td><td rowspan=1 colspan=1>13%</td><td rowspan=1 colspan=1>Full swab</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>Saline</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>Full Swab</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Miconazole Nitrate Salt</td><td rowspan=1 colspan=1>2% w/v</td><td rowspan=1 colspan=1>100% Methanol</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>AluminumHydroxide/MagnesiumCarbonate</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Witch Hazel</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Alcohol</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>Vancomycin HCl</td><td rowspan=1 colspan=1>12.5 mg/mL</td><td rowspan=1 colspan=1>Water</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Human IgA</td><td rowspan=1 colspan=1>1.6 mg/mL</td><td rowspan=1 colspan=1>Tris - saline</td></tr></table>

i. Assay cut-off:

Not applicable.

j. Carry-Over & Cross-Contamination Study:

The potential for carry-over and cross-contamination to occur with the Quidel Molecular Direct C. difficile Assay was evaluated on all three platforms. High positive specimens were manufactured by using pooled negative stool spiked with $C .$ . difficile bacterial stock at a concentration such that a Ct of 10.0 or lower was obtained. These specimens were prepared and frozen in aliquots; aliquots were defrosted prior to the experiment.

For all experiments an unused sterile neonatal flocked swab was used to transfer a positive sample into a PB1 tube; similarly an unused sterile neonatal flocked swab was used to transfer a negative sample into a PB1 tube. This process of manufacturing positive and negative samples was repeated, alternating the creation of mock positive and negative specimens. In each instance an aliquot was transferred from each PB1 tube into the appropriate PB2 prior to amplification.

The Applied Biosystems 7500 Fast Dx experiment and the Life Technologies QuantaStudio Real-Time PCR system experiment each utilized 96 wells per experiment; the Cepheid SmartCycler II experiment utilized 16 tubes per experiment. Each experiment was repeated 6 times, and each experiment was carried out on a different day.

No carry over was detected as all negative samples tested negative for C. difficile.

2. Comparison studies:

a. Method comparison with predicate device:

Clinical performance was determined by comparing the Quidel Molecular Direct C. difficile Assay results to reference culture (i.e., direct culture or enriched toxigenic culture) followed by cell cytotoxicity testing on the isolates.

$b$ . Matrix comparison:

Not applicable

3. Clinical studies:

Clinical performance characteristics of the Quidel Molecular Direct C. difficile Assay were determined in a multi-site, prospective investigational study at four geographically diverse locations within the United States, as well as one internal site. Two clinical studies were conducted: one study evaluated the ABI 7500 Fast Dx and Cepheid SmartCycler II (665 specimens), and the second study evaluated the QuantStudio Dx Real-Time PCR System (792 specimens).

# i Applied Biosystems 7500 Fast Dx

Performance characteristics of the Quidel Molecular Direct C. difficile Assay on the Applied Biosystems 7500 Fast Dx instrument were established using 665 specimens collected from patients suspected of having Clostridium difficile-associated disease. This study was conducted at four distinct geographical sites across the United States. Specimens were tested with the Quidel Assay on the 7500 Fast Dx at three external facilities. The tissue culture cytotoxin assay and enhanced toxigenic culture were performed at a central reference laboratory.

Nine specimens $( 1 . 3 5 \% )$ were invalid in the Quidel Molecular Direct C. difficile Assay when initially tested. Eight specimens yielded a valid result when retested (7 were negative, 1 was positive). One specimen remained invalid upon repeat testing.

# Direct Culture Cytotoxicity Assay Comparison

Six hundred sixty-five (665) specimens were tested by both the Quidel Molecular Direct C. difficile Assay and tissue culture followed by cell cytotoxicity testing on the isolates. Three specimens $( 0 . 5 \% )$ were indeterminate in the cytotoxin assay due to toxicity in the antitoxin well. Nine specimens $( 1 . 3 5 \% )$ were invalid in the Quidel Molecular Direct C. difficile Assay when initially tested on the Applied Biosystems 7500 Fast Dx instrument. Clinical performance was based on the initial test result obtained for each specimen (e.g., 665 specimens – 3 cytotoxin indeterminate – 9 Quidel invalid results $= 6 5 3$ evaluable specimens). Therefore, the data below summarizes the performance of the Quidel Molecular Direct C. difficile Assay relative to direct culture using the remaining 653 specimens on the Applied Biosystems 7500 Fast Dx instrument.

<table><tr><td colspan="5">Performance of the Quidel Molecular Direct C difficile Assay VS Direct Culture</td></tr><tr><td colspan="5">Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, software version 1.4 Combined Sites</td></tr><tr><td colspan="5">Direct Culture</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+ + 83</td><td>33a</td><td colspan="2">116</td></tr><tr><td>5b 88</td><td>532</td><td>537 653</td><td></td></tr><tr><td></td><td>565</td><td colspan="2"></td></tr><tr><td colspan="2">Sensitivity =</td><td colspan="2">95% CI 97.6%</td></tr><tr><td colspan="2">94.3%</td><td colspan="2">87.4%</td></tr><tr><td colspan="2"></td><td colspan="2">94.2%</td><td>91.9%</td></tr><tr><td colspan="2">Specificity =</td><td colspan="2"></td><td>95.8%</td></tr><tr><td colspan="2">PPV =</td><td colspan="2">71.6%</td><td>78.0%</td></tr><tr><td colspan="2">NPV =</td><td colspan="2">64.3% 99.1% 97.9%</td><td>99.6%</td></tr></table>

aOf these 33 discordant specimens, 32 were tested with an FDA-cleared molecular device. All 32 of these specimens were positive for C. difficile. The remaining specimen was unavailable for testing. bOf these five discordant specimens, all were tested with an FDA-cleared molecular device. All five specimens were found to be negative for $C$ . difficile.

Similarly, these 665 specimens were also subjected to enriched toxigenic culture. Clinical performance was based on the initial test result obtained on the Applied Biosystems 7500 Fast Dx for each specimen (e.g., 665 specimens – 9 Quidel invalid results $= 6 5 6$ evaluable specimens). Therefore, the data below summarizes the performance of the Quidel Molecular Direct C. difficile Assay relative to enriched toxigenic culture using the remaining 656 specimens on the Applied Biosystems 7500 Fast Dx instrument.

<table><tr><td colspan="5">Performance of the Quidel Molecular Direct C difficile Assay VS Enriched Toxigenic Culture</td></tr><tr><td colspan="5">Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, software version 1.4 Combined Sites</td></tr><tr><td colspan="5">Enriched Toxigenic Culture</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+ + 112</td><td>6a</td><td colspan="2">118</td></tr><tr><td>14b 126</td><td>524 530</td><td colspan="2">538 656</td></tr><tr><td></td><td></td><td colspan="2">95% CI</td></tr><tr><td colspan="2">Sensitivity = 88.9%</td><td colspan="2">82.2% 93.3%</td></tr><tr><td colspan="2">Specificity =</td><td colspan="2">97.6%</td></tr><tr><td colspan="2"></td><td colspan="2">98.9%</td><td>99.5%</td></tr><tr><td colspan="2"></td><td colspan="2">94.9% 89.6%</td><td>97.6%</td></tr><tr><td colspan="2">PPV =</td><td colspan="2">95.9%</td><td></td></tr><tr><td colspan="2">NPV =</td><td colspan="2">97.4%</td><td>98.4%</td></tr></table>

aThese six discordant specimens were tested with an FDA-cleared molecular device; all were positive for C. difficile. bThese 12 discordant specimens were tested with an FDA-cleared molecular device. Two specimens were unavailable for testing; nine of these specimens were found negative for $C$ . difficile, while the remaining three were positive.

# ii Cepheid SmartCycler II

Performance characteristics of the Quidel Molecular Direct C. difficile Assay on the Cepheid SmartCycler II instrument were established using 665 specimens collected from patients suspected of having CDAD. This study was conducted at four distinct geographical sites across the United States. Specimens were tested with the Quidel Assay on the Cepheid SmartCycler II at three external facilities. The tissue culture cytotoxin assay and enhanced toxigenic culture were performed at a central reference laboratory.

Five specimens $( 0 . 7 5 \% )$ were invalid in the Quidel Molecular Direct C. difficile Assay when initially tested on the SmartCycler II. All five (5) specimens yielded a valid result when retested (3 were negative, 2 were positive).

# Direct Culture Cytotoxicity Assay Comparison

Six hundred sixty-five (665) specimens were tested by both the Quidel Molecular Direct C. difficile Assay and tissue culture followed by cell cytotoxicity testing of the isolates. Three specimens $( 0 . 5 \% )$ were indeterminate in the cytotoxin assay due to toxicity in the antitoxin well. Five specimens $( 0 . 7 5 \% )$ were invalid in the Quidel Molecular Direct C. difficile Assay when initially tested on the SmartCycler II. Clinical performance was based on the initial test result obtained for each specimen (e.g., 665 specimens – 3 cytotoxin indeterminate – 5 Quidel invalid results $= 6 5 7$ evaluable specimens). Therefore, the data below summarizes the performance of the Quidel Molecular Direct C. difficile Assay relative to direct culture using the remaining 657 specimens on the SmartCycler II instrument.

<table><tr><td colspan="5">Performance of the Quidel Molecular Direct C difficile Assay VS Direct Culture</td></tr><tr><td colspan="5">Cepheid SmartCycler II Instrument, software version 3.0b Combined Sites</td></tr><tr><td colspan="5">Direct Culture +</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+ 78 9b</td><td>38a</td><td colspan="2">116</td></tr><tr><td>87</td><td>532 570</td><td colspan="2">541 657</td></tr><tr><td></td><td></td><td colspan="2"></td></tr><tr><td colspan="2">Sensitivity = 89.7%</td><td colspan="2">95% CI 81.5%</td></tr><tr><td colspan="2">Specificity = PPV =</td><td></td><td>94.5%</td></tr><tr><td colspan="2"></td><td>93.3%</td><td>91.0%</td><td>95.1%</td></tr><tr><td colspan="2"></td><td>67.2%</td><td>60.0%</td><td>73.8%</td></tr><tr><td colspan="2">NPV =</td><td>98.3%</td><td>97.0%</td><td>99.1%</td></tr></table>

aThese 38 discordant specimens were tested with a FDA-cleared molecular device. Nine of these specimens were negative for C. difficile, while 29 were positive for $C .$ . difficile. bEight of the nine discordant specimens were tested with a FDA-cleared molecular device; one specimen was unavailable for testing. Five of these specimens were negative for $C .$ . difficile, while three were positive.

# Enriched Toxigenic Culture Comparison

Similarly, these 665 specimens were also subjected to enriched toxigenic culture. Clinical performance was based on the initial test result obtained for each specimen on the SmartCycler II (e.g., 665 specimens – 5 Quidel invalid results $=$ 660 evaluable specimens). Therefore, the data below is for the 660 initially evaluable specimens.

<table><tr><td colspan="6">Performance of the Quidel Molecular Direct C difficile Assay VS Enriched Toxigenic Culture</td></tr><tr><td colspan="6">Cepheid SmartCycler II Instrument, software version 3.0b Combined Sites</td></tr><tr><td colspan="6">Enriched Toxigenic Culture + 103</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+</td><td></td><td>15a</td><td colspan="2">118</td></tr><tr><td>22b</td><td>520</td><td></td><td colspan="2">542</td></tr><tr><td>125</td><td></td><td>535</td><td>660</td></tr><tr><td></td><td></td><td>95% CI</td><td></td></tr><tr><td>Sensitivity =</td><td>82.4%</td><td colspan="2">74.8%</td></tr><tr><td></td><td></td><td></td><td>95.4%</td><td>88.1% 98.3%</td></tr><tr><td></td><td>Specificity =</td><td>97.2%</td><td>80.7%</td><td>91.9%</td></tr><tr><td></td><td>PPV =</td><td>87.3%</td><td>94.3%</td><td>97.3%</td></tr><tr><td></td><td>NPV =</td><td>95.9%</td><td></td><td></td></tr></table>

aThese 15 discordant specimens were tested with a FDA-cleared molecular device. Six of these specimens were positive for C. difficile, while nine were negative.

bNineteen of these 22 discordant specimens were tested with a FDA-cleared molecular device. Three specimens were unavailable for testing. Ten of these specimens were found to be positive for C. difficile, and nine were found to be negative.

# Life Technologies QuantaStudio Real-Time PCR Instrument

Performance characteristics of the Quidel Molecular Direct C. difficile Assay on the Life Technologies QuantaStudio Real-Time PCR system were established using 792 specimens collected from patients suspected of having Clostridium difficileassociated disease (CDAD). This study was conducted at two distinct geographical sites across the United States and one central reference laboratory. Specimens were tested with the Quidel Molecular Direct C. difficile Assay on the Life Technologies QuantaStudio Real-Time PCR Instrument at three facilities. The tissue culture cytotoxin assay and enhanced toxigenic culture were performed at a central reference laboratory.

One specimen $( 0 . 1 \% )$ was invalid in the Quidel Molecular Direct C. difficile Assay when initially tested on the QuantaStudio Real-Time PCR system. The specimen yielded a valid result when retested (it was negative).

# Direct Culture Cytotoxicity Assay Comparison

Seven hundred ninety-two (792) specimens were tested by both the Quidel Molecular Direct C. difficile Assay and tissue culture followed by cell cytotoxicity testing on the isolates. Three specimens $( 0 . 4 \% )$ were indeterminate in the cytotoxin assay due to toxicity in the antitoxin well. One specimen $( 0 . 1 \% )$ was invalid in the Quidel Molecular Direct C. difficile Assay when initially tested on the QuantaStudio Real-Time PCR system. Clinical performance was based on the initial test result obtained for each specimen (e.g., 792 specimens – 3 cytotoxin indeterminate – 1 Quidel invalid result $= 7 8 8$ evaluable specimens). Therefore, the data below summarizes the performance of the Quidel Molecular Direct C. difficile Assay relative to direct culture using the remaining 788 specimens on the QuantaStudio Real-Time PCR system.

<table><tr><td colspan="5">Performance of the Quidel Molecular Direct C difficile Assay VS Direct Culture</td></tr><tr><td colspan="5">Life Technologies QuantStudio Dx Real-Time PCR Instrument, software version 1.0 Combined Sites</td></tr><tr><td colspan="5">Direct Culture +</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+ 98</td><td>45a</td><td colspan="2">143</td></tr><tr><td>7b</td><td>638</td><td></td><td>645</td></tr><tr><td>105</td><td>683</td><td></td><td>788</td></tr><tr><td></td><td></td><td colspan="2">95% CI</td></tr><tr><td>Sensitivity =</td><td>93.3%</td><td>86.9%</td><td>96.7%</td></tr><tr><td></td><td>Specificity =</td><td>93.4%</td><td>91.3%</td><td>95.0%</td></tr><tr><td></td><td>PPV =</td><td>68.5%</td><td>62.1%</td><td>74.4%</td></tr><tr><td></td><td></td><td></td><td>97.9%</td><td>99.5%</td></tr><tr><td></td><td>NPV =</td><td>98.9%</td><td></td><td></td></tr></table>

aOf these 45 discordant specimens, 44 were tested with an FDA-cleared molecular device; 35 specimens were positive for C. difficile, while nine were negative. The remaining specimen was unavailable for testing.

bThese seven discordant specimens were tested with an FDA-cleared molecular device. Two specimens were positive for C. difficile, whereas five were negative.

# Enriched Toxigenic Culture Comparison

Similarly, these 792 specimens were also subjected to enriched toxigenic culture. Clinical performance was based on the initial test result obtained for each specimen on the QuantaStudio Real-Time PCR system (e.g., 792 specimens – 1 Quidel invalid result $= 7 9 1$ evaluable specimens). Therefore, the data below is for the 791 initially evaluable specimens.

<table><tr><td colspan="5">Performance of the Quidel Molecular Direct C difficile Assay VS Enriched Toxigenic Culture</td></tr><tr><td colspan="5">Life Technologies QuantStudio Dx Real-Time PCR Instrument, software version 1.0 Combined Sites</td></tr><tr><td colspan="5">Enriched Toxigenic Culture</td></tr><tr><td rowspan="5">Quidel Molecular Direct C difficile Assay</td><td>+ + 137</td><td>8a</td><td colspan="2">145</td></tr><tr><td>20b 157</td><td>626 634</td><td colspan="2">646 791</td></tr><tr><td></td><td></td><td colspan="2"></td></tr><tr><td colspan="2">Sensitivity = 87.3%</td><td colspan="2">95% CI 81.1% 91.6%</td></tr><tr><td colspan="2"></td><td colspan="2">97.5%</td></tr><tr><td colspan="2"></td><td colspan="2">98.7%</td><td>99.4%</td></tr><tr><td colspan="2">Specificity =</td><td colspan="2"></td><td></td></tr><tr><td colspan="2">PPV =</td><td colspan="2">94.5% 89.7%</td><td>97.2%</td></tr><tr><td colspan="2"></td><td colspan="2">96.9%</td><td>98.0%</td></tr><tr><td colspan="3">NPV =</td><td>95.4%</td><td></td></tr></table>

aThese eight discordant specimens were tested with an FDA-cleared molecular device; two ere positive for C. difficile, and six were negative.

bOf these 20 discordant specimens, 17 were tested with an FDA-cleared molecular device; three specimens were unavailable for testing. Eleven of these specimens were negative for $C$ . difficile, while six were positive.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference ranges:

Applied Biosystems 7500 Fast Dx

The age and gender distribution based on the Quidel Molecular Direct C. difficile Assay result obtained on the Applied Biosystems 7500 Fast Dx was calculated. The patient age and gender data below is for the 656 initially evaluable specimens.

<table><tr><td rowspan=1 colspan=5>Age and Gender Distribution of C. difficile Positive Results</td></tr><tr><td rowspan=1 colspan=5>Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, software version 1.4Combined Sites</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Prevalence by age*</td></tr><tr><td rowspan=1 colspan=1>Unknown Gender</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>33.3% (1/3)</td></tr><tr><td rowspan=1 colspan=1>Infant (&lt;2 yrs)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.5% (1/8)</td></tr><tr><td rowspan=1 colspan=1>Child (≥2 to &lt;12 yrs)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>25.6% (10/39)</td></tr><tr><td rowspan=1 colspan=1>Adolescent (≥12 to &lt;18 yrs)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>21.1% (4/19)</td></tr><tr><td rowspan=1 colspan=1>Transitional Adolescent(&gt;18 to &lt;21 vrs)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>15.4% (2/13)</td></tr><tr><td rowspan=1 colspan=1>Adult (&gt;21 to 59 yrs)</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>278</td><td rowspan=1 colspan=1>19.1% (53/278)</td></tr><tr><td rowspan=1 colspan=1>Sr. Adult (&gt; 60 yrs)</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>15.9% (47/296)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>656</td><td rowspan=1 colspan=1>18.0% (118/656)</td></tr></table>

\*Prevalence based on $C .$ . difficile positives with the Quidel Molecular Direct C. difficile Assay on the Applied Biosystems 7500 Fast Dx instrument.

The age and gender distribution based on the Quidel Molecular Direct C. difficile Assay result obtained on the SmartCycler II was calculated. The patient age and gender data below is for the 650 initially evaluable specimens.

<table><tr><td rowspan=1 colspan=5>Age and Gender Distribution of C. difficile Positive Results</td></tr><tr><td rowspan=1 colspan=5>Cepheid SmartCycler II Instrument, software version 3.0bCombined Sites</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Prevalence by age*</td></tr><tr><td rowspan=1 colspan=1>Unknown Gender</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>33.3% (1/3)</td></tr><tr><td rowspan=1 colspan=1>Infant (&lt;2 yrs)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.5% (1/8)</td></tr><tr><td rowspan=1 colspan=1>Child (≥2 to &lt;12 yrs)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>23.1% (9/39)</td></tr><tr><td rowspan=1 colspan=1>Adolescent (12 to &lt;18 yrs)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>15.8% (3/19)</td></tr><tr><td rowspan=1 colspan=1>Transitional Adolescent(&gt;18 to &lt;21 vrs)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.7% (1/13)</td></tr><tr><td rowspan=1 colspan=1>Adult (&gt;21 to 59 yrs)</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>18.6% (52/280)</td></tr><tr><td rowspan=1 colspan=1>Sr. Adult (&gt; 60 yrs)</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>17.1% (51/298)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>357</td><td rowspan=1 colspan=1>660</td><td rowspan=1 colspan=1>17.9% (118/660)</td></tr></table>

\*Prevalence based on C. difficile positives with the Quidel Molecular Direct C. difficile Assay on the Cepheid SmartCycler II Instrument.

# iii Life Technologies QuantaStudio Real-Time PCR Instrument

The age and gender distribution based on the Quidel Molecular Direct C. difficile Assay result obtained on the QuantaStudio Real-Time PCR system was calculated. The patient age and gender data below is for the 791 initially evaluable specimens.

<table><tr><td rowspan=1 colspan=5>Age and Gender Distribution of C. difficile Positive Results</td></tr><tr><td rowspan=1 colspan=5>Life Technologies QuantStudio Dx Real-Time PCR Instrument, software version 1.0Combined Sites</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Prevalence by age*</td></tr><tr><td rowspan=1 colspan=1>Unknown Gender</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>50.0% (1/2)</td></tr><tr><td rowspan=1 colspan=1>Infant (&lt;2 yrs)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10.0% (1/10)</td></tr><tr><td rowspan=1 colspan=1>Child (≥2 to &lt;12 yrs)</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>24.5% (12/49)</td></tr><tr><td rowspan=1 colspan=1>Adolescent (≥12 to &lt;18 yrs)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20.8% (5/24)</td></tr><tr><td rowspan=1 colspan=1>Transitional Adolescent(&gt;18 to &lt;21 vrs)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.7% (1/13)</td></tr><tr><td rowspan=1 colspan=1>Adult (&gt;21 to 59 yrs)</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>328</td><td rowspan=1 colspan=1>18.3% (60/328)</td></tr><tr><td rowspan=1 colspan=1>Sr. Adult (&gt; 60 yrs)</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>365</td><td rowspan=1 colspan=1>17.8% (65/365)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>419</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>18.3% (145/791)</td></tr></table>

\*Prevalence based on C. difficile positives with the Quidel Molecular Direct C. difficile Assay on the Life Technologies QuantStudio Dx Real-Time PCR system.

# N. Instrument Name:

The Quidel Molecular Direct C. difficile Assay can be performed on the following platforms with the specified software:

Life Technologies QuantStudio® Dx with software version 1 Applied Biosystems 7500 Fast Dx with software version 1.4 Cepheid SmartCycler II with software version 3.0b

# O. System Descriptions:

1. Modes of Operation:

Please see the respective Decision Summary for the platform of interest:

Life Technologies QuantStudio® Dx – K123955   
Applied Biosystems 7500 Fast Dx – K082562   
Cepheid SmartCycler II – K062948

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

The results of the validation and verification testing results were provided for version 1.0 of the Life Technologies QuantStudio $^ \mathrm { \textregistered }$ Dx software; version 1.4 of the Applied Biosystems 7500 Fast Dx software, or version 3.0b of the Cepheid SmartCycler II software.

3. Specimen Identification: Not applicable – specimen identification is manually entered.

4. Specimen Sampling and Handling: Specimens are manually transferred to processing tubes prior to amplification.

# 5. Calibration:

Please see the respective manual for the platform of interest:

Life Technologies QuantStudio® Dx

Applied Biosystems 7500 Fast Dx

Cepheid SmartCycler II

6. Quality Control:

Users must implement their own quality control decision algorithms; such algorithms must be in compliance with local and state guidelines. The sponsor recommends use of the “Quidel Molecular C. difficile Control Set” (SKU # M108) for development of appropriate quality control procedures for this assay.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.