# 510(k) Summary BioFire Diagnostics, Inc.

# FilmArray Blood Culture Identification (BCID) Panel Kit

Introduction: According to the requirements of 21 CFR 807.92, the following information provides sufficient detail to understand the basis for a determination of substantial equivalence.

# Submitted by:

BioFire Diagnostics, Inc. 390 Wakara Way Salt Lake City, UT 84108

Telephone: 801-736-6354   
Facsimile: 801-588-0507

Contact: Beth Lingenfelter, ext. 407

Date Submitted: March 30, 2013

# Device Name and Classification:

Trade Name: FilmArray BCID Panel

Regulation Number: 21 CFR 866.3365

Classification Name: Multiplex devices that use DNA hybridization to detect bacteria and their resistance markers

# Predicate Device:

K122514 - Nanosphere Verigene® Gram-Positive Blood Culture Nucleic Acid Test (BCBG

# Intended Use:

The FilmArray Blood Culture Identification (BCID) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with the FilmArray Instrument. The FilmArray BCID Panel is capable of simultaneous detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants of antimicrobial resistance. The BCID assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system that demonstrates the presence of organisms as determined by Gram stain.

The following gram-positive bacteria, gram-negative bacteria, and yeast are identified using the FilmArray BCID Panel: Enterococci, Listeria monocytogenes, commonly encountered Staphylococci (including specific differentiation of Staphylococcus aureus). commonly encountered Streptococci (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes), Acinetobacter baumannii, commonly encountered Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens), Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa, Candida albicans, Candida glabrata, Candida krusei, Candida parapsilosis, and Candida tropicalis.

The FilmArray BCID Panel also contains assays for the detection of genetic determinants of resistance to methicillin (mecA), vancomycin (vanA and vanB), and carbapenems EY $( b l a _ { \mathrm { K P C } } )$ to aid in the identification of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, and carbapenems exist.

FilmArray BCID is indicated as an aid in the diagnosis of specific agents of bacteremia and fungemia and results should be used in conjunction with other clinical and laboratory findings. Positive FilmArray results do not rule out co-infection with organisms not included in the FilmArray BCID Panel. FilmArray BCID is not intended to monitor treatment for bacteremia or fungemia.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the FilmArray BCID Panel, and for species determination of some Staphylococci, Enterococci, Streptococci, and Enterobacteriaceae that are not specifically identified by the FilmArray BCID Panel assays.

# Device Description:

The FilmArray Blood Culture Identification (BCID) Panel is a multiplex nucleic acid test designed to be used with the FilmArray Instrument. The FilmArray BCID pouch contains freeze-dried reagents to perform nucleic acid purification and nested, multiplex PCR with DNA melt analysis. The FilmArray Blood Culture Identification (BCID) Panel simultaneously tests a single positive blood culture sample to provide results for 24 different organisms and organism groups that cause bloodstream infections and three genetic markers that are known to confer antimicrobial resistance (see Table 1).

Table 1. FilmArray BCID Panel Test Results.   

<table><tr><td>Gram-Positive Bacteria</td><td>Gram-Negative Bacteria</td><td>Yeast</td></tr><tr><td>Enterococcus</td><td>Acinetobacter baumannii</td><td>Candida albicans</td></tr><tr><td>Listeria monocytogenes</td><td>Enterobacteriaceue</td><td>Candida glabrata</td></tr><tr><td>Staphylococcus</td><td>Enterobacter cloacae complex</td><td>Candida krusei</td></tr><tr><td>Staphylococcus aureus</td><td>Escherichia coli</td><td>Candida parapsilosis</td></tr><tr><td>Streptococcus</td><td>Klebsiella oxytoca</td><td>Candida tropicalis</td></tr><tr><td>Streptococcus agalactiae</td><td>Klebsiella pneumoniae</td><td>Antimicrobial resistance genes</td></tr><tr><td>Streptococcus pneumoniae</td><td>Proteus</td><td>mecA - methicillin resistance</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">vanA/B - vancomycin resistance</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">blakpc - carbapenem resistance</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis (encapsulated)</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonus aeruginosa</td></tr></table>

A test is initiated by loading Hydration Solution and a positive blood culture sample mixed with the provided Sample Buffer into the FilmArray BCID pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray Instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed PCR reaction which includes all primers of the outer primer sets. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green® Plus+, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The $2 ^ { \mathfrak { n } \mathfrak { d } }$ stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the $2 ^ { \mathsf { n d } }$ stage PCR, the array is interrogated by mclt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the $2 ^ { \mathrm { { \dot { n } d } } }$ stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism and antimicrobial resistance gene on the panel.

# Substantial Equivalence:

The Nanosphere Verigene® Gram-Positive Blood Culture Nucleic Acid Test is a qualitative, multiplexed in vilro diagnostic test for the simultaneous detection and identification of potentially pathogenic gram-positive bacteria which may cause

bloodstream infection. Table 2 outlines the similarities between the two systems and Table 3 outlines the differences.

Table . Similarities Between the FilmArray BCID Panel and the Nanosphere Verigene® GramPositive Blood Culture Nucleic Acid Test.   

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>FilmArray BCID Panel</td><td rowspan=1 colspan=1>Nanosphere VerigeneGram-PositiveBlood Culture Nucleic Acid Test</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Enterococci, Staphylococci (includingspecific differentiation of Staphylococcusaureus), Streptococci (with specificdifferentiation of Streptococcusagalactiae, Streptococcus pneumoniae,and Streptococcus pyogenes) andresistance markers mecA, vanA, and vanB.</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr><tr><td rowspan=1 colspan=1>SampleProcessing andPurification</td><td rowspan=1 colspan=1>Automated by instrument</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>Internal procedural/instrument qualitycontrols; Internal Negative Control,Sample processing control, externalpositive and negative assay controls.</td></tr><tr><td rowspan=1 colspan=1>UserComplexity</td><td rowspan=1 colspan=1>Moderate/Low</td><td rowspan=1 colspan=1>Same</td></tr></table>

Table 3. Differences Between FilmArray Respiratory Panel Test System and the Nanosphere Verigene® Gram-Positive Blood Culture Nucleic Acid Test.   

<table><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">FilmArray BCID Panel</td><td colspan="1" rowspan="1">Nanosphere Verigene® Gram-PositiveBlood Culture Nucleic Acid Test</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Positive blood culture samples containinggram-positive, gram-negative bacteria,and/or yeast.</td><td colspan="1" rowspan="1">Positive blood culture bottles(unspecified) which contain gram-positive bacteria.</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Detection of additional targets: Listeriamonocytogenes, Acinetobacterbaumannii, Enterobacteriaceae(including specific differentiation ofEnterobacter cloacae complex species,Escherichia coli, Klebsiella oxytoca,Klebsiella pneumoniae, Proteus, andSerratia marcescens), Haemophilusinfluenzae, Neisseria meningitidis,Pseudomonas aeruginosa, Candidaalbicans, Candida glabrata, Candidakrusei, Candida parapsilosis, Candidatropicalis, and resistance marker blakpc</td><td colspan="1" rowspan="1">Tests only for gram positive bacteria.Tests for Listeria spp. rather thanListeria monocytogenes. Includes testingfor additional Staphylococcus spp.:Staphylococcus epidermidis,Staphylococcus lugdunensis, as well astesting for specific Enterococcus spp.:Enterococcus faecalis,Enterococcus faecium. Includes testingfor an additional Streptococcus spp.:Streptococcus anginosus group. Doesnot include testing for blakpc</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Nested multiplex PCR followed by highresolution melting analysis to confirmidentity of amplified product.</td><td colspan="1" rowspan="1">Qualitative, multiplexed test for thedetection of specific nucleic acid targetsin a microarray format using capture andmediator oligonucleotides for goldnanoparticle probe-based endpointdetection.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray Instrument</td><td colspan="1" rowspan="1">Verigene Reader and Processor SP</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Less than 1 hour</td><td colspan="1" rowspan="1">2.5 hours</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Diagnostic Software/DecisionAlgorithm.</td></tr></table>

# Summary of Performance Data

# Clinical Performance

The clinical performance of the FilmArray BCID Panel was established during a two armed clinical study which was conducted at eight U.S. clinical sites over an eight month time period. The study included a prospective residual blood culture arm and a seeded blood culture arm. In the prospective arm, 1635 prospectively-collected residual blood culture samples (pediatric and adult) were initially included in the study. Sixty-seven (67) specimens were excluded from the study. The most common reasons for exclusion were that the specimens were ${ > } 8$ hours past positivity, incomplete reference/comparator data were provided, or the specimen was from a subject who had a previous specimen included in the study. In the seeded culture arm, analytes proven to be of low prevalence in the prospective arm were evaluated by seeding previously characterized isolates into blood culture bottles and incubating until positivity. A total of 716 seeded cultures were initiated for the study. Seventy-seven (77) cultures were excluded from the study. The most common reasons for exclusion were that the specimens were ${ > } 8$ hours past positivity, the seeded culture was not called positive by the automated blood culture system, or the culture was contaminated or inconsistent with the intended seed organism. The final specimen set consisted of 2207 blood cultures (1568 prospective and 639 seeded). All cultures were grown in Becton Dickinson BACTECTM Plus Aerobic/F Medium. Table 4 provides a summary of demographic information for the 1568 specimens included in the prospective arm of the study.

Table 4. Demographic Summary for Prospective Arm of FilmArray BCID Clinical Evaluation   

<table><tr><td colspan="2" rowspan="1">Prospective Study Specimens _</td></tr><tr><td colspan="1" rowspan="1">Total Specimens</td><td colspan="1" rowspan="1">1568</td></tr><tr><td colspan="1" rowspan="1">Sex</td><td colspan="1" rowspan="1">Number of Specimens</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">917 (58%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">651 (42%)</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">Number of Specimens</td></tr><tr><td colspan="1" rowspan="1">≤ 1 year</td><td colspan="1" rowspan="1">57 (4%)</td></tr><tr><td colspan="2" rowspan="1">Prospective Study Specimens</td></tr><tr><td colspan="1" rowspan="1">Total Specimens</td><td colspan="1" rowspan="1">1568</td></tr><tr><td colspan="1" rowspan="1">1 - 17 years</td><td colspan="1" rowspan="1">92 (6%)</td></tr><tr><td colspan="1" rowspan="1">18 - 44 years</td><td colspan="1" rowspan="1">281 (18%)</td></tr><tr><td colspan="1" rowspan="1">45 · 64 years</td><td colspan="1" rowspan="1">583 (37%)</td></tr><tr><td colspan="1" rowspan="1">65 - 84 years</td><td colspan="1" rowspan="1">442 (28%)</td></tr><tr><td colspan="1" rowspan="1">≥ 85 years</td><td colspan="1" rowspan="1">113 (7%)</td></tr></table>

Positive blood cultures (prospective and seeded) were tested with the FilmArray BCID Panel. The performance of FilmArray BCID was evaluated by comparing the FilmArray BCID test result for each panel member with the appropriate comparator/reference methods shown in Table 5.

Table 5. Reference/Comparator Methods used to Assess FilmArray BCID Performance   

<table><tr><td rowspan=1 colspan=1>Test Result</td><td rowspan=1 colspan=1>Reference/Comparator Method(s)</td></tr><tr><td rowspan=1 colspan=1>All organism detectionsexcept Acinetobacter baumannii</td><td rowspan=1 colspan=1>Standard manual and automated microbiological/biochemical identificationmethods </td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii detection</td><td rowspan=1 colspan=1>Standard manual and automated microbiological/biochemical identificationmethodsPlus16S PCR with bi-directional sequencing of all A. calcoaceticus-baumanniicomplex isolates for characterization as A. baumanmii or non-A. baumannii </td></tr><tr><td rowspan=1 colspan=1>Antimicrobial resistance gene detectionsin specimens in which an associated organism was detected(mecA from Staphylococcus; vanA/B from Enterococcus, KPC from Enterobacteriaceae, Acinetobacter baumannii,and Pseudomonas aeruginosa)</td><td rowspan=1 colspan=1>Method 1: PCR with bi-directional sequencing for specific resistance genedirect from blood cultureMethod 2: PCR with bi-directional sequencing for specific resistance gene fromapporiate culured aInformational: Standard manual and automated phenotypic antimicrobialsusceptibility testing of appropriate cultured isolates (methicillin resistance,vancomycin resistance, and carbapenem resistance (and/or carbapenemaseproduction) according to current CLSI criteria) </td></tr></table>

adequately discriminate between members of the A. calcoaceticus-baumannii complex.   
Information (NCBl) GenBank database (www.ncbi.nlm.nih.gov), with an acceptable E-value.   
phenotypic methods were performed in accordance with current CLSl criteria.

A total of 2207 blood culture specimens (1568 prospective and 639 seeded) were evaluated in the FilmArray BCID clinical evaluation. Specimens were tested by FilmArray BCID either fresh or from frozen aliquots. A total of 1240 specimens were tested fresh (821 prospective and 419 seeded) and 967 specimens were tested frozen (747 prospective and 220 seeded). Clinical sensitivity or positive percent agreement (PPA) was calculated as $1 0 0 \% \times \mathrm { ( T P / T P + F N ) }$ . True positive (TP) indicates that both FilmArray BCID and the reference/comparator method had a positive result for a specific analyte, and false negative (FN) indicates that the FilmArray BCID result was negative while the reference/comparator method was positive. Clinical specificity or negative percent agreement (NPA) was calculated as $100 \% \textbf { X }$ $\mathrm { \Delta T N / T N + F P }$ . True negative (TN) indicates that both FilmArray BCID and the reference/comparator method had a negative result for a specific analyte, and false positive (FP) indicates that the FilmArray BCID result was positive while the reference/comparator method was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. The results are summarized in Tables 6 - 10.

Table 6. FilmArray BCID Clinical Performance Summary - Gram-Positive Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identificatiol   

<table><tr><td colspan="2">Gram-Positive Bacteria</td><td colspan="2">Sensitivity/PPA</td><td colspan="3">Specificity/NPA</td></tr><tr><td rowspan="5">Enterococcus</td><td>Prospective Fresh</td><td>TP/TP + FN 55/55 100</td><td>% 95% CI 93.5-100</td><td>TN/TN + FP 762/766</td><td>% 99.5</td><td>95% CI 98.7-99.9</td></tr><tr><td>Prospective Frozen</td><td>43/46</td><td>93.5 82.1-98.6</td><td>701/701</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>12/12</td><td>100 73.5-100</td><td>407/407</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>17/17</td><td>100 80.5-100</td><td>203/203</td><td>100</td><td>98.2-100</td></tr><tr><td>Ovverall</td><td>127/130</td><td>97.7 93.4-99.5</td><td>2073/2077b</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Listeriu monocytogenes</td><td>Prospective Fresh</td><td>0/0</td><td>-</td><td>821/821</td><td>100</td><td>99.6-100</td></tr><tr><td>Prospective Frozen</td><td>0/0</td><td>-</td><td>747/747</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>23/23</td><td>100 85.2-100</td><td>396/396</td><td>100</td><td>99.1-100</td></tr><tr><td>Sceded Frozen</td><td>13/13</td><td>100 75.3-100</td><td>207/207</td><td>100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>36/36</td><td>100 90.3-100</td><td>2171/2171</td><td>100</td><td>99.8-100</td></tr><tr><td rowspan="5">Staphylococcus</td><td>Prospective Fresh</td><td>405/418</td><td>96.9 94.7-98.3</td><td>401/403</td><td>99.5</td><td>98.2-99.9</td></tr><tr><td>Prospective Frozen</td><td>364/379</td><td>96.0 93.6-97.8</td><td>359/368</td><td>97.6</td><td>95.4-98.9</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>- -</td><td>418/419</td><td>99.8</td><td>98.7-100</td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100 2.5-100</td><td>219/219</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>770/798</td><td>96.5 95.0-97.7</td><td>1397/1409</td><td>99.1</td><td>98.5-99.6</td></tr><tr><td rowspan="5">Staphylococcus aureus</td><td>Prospective Fresh</td><td>133/136</td><td>97.8 93.7-99.5</td><td>685/685</td><td>100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>120/121</td><td>99.2 95.5-100</td><td>622/626</td><td>99.4</td><td>98.4-99.8</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>- •</td><td>419/419</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td>- -</td><td>220/220</td><td>100</td><td>98.3-100</td></tr><tr><td>Overull</td><td>253/257d</td><td>98.4 96.1-99.6</td><td>1946/1950</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Streptococcus Streptococcus</td><td>Prospective Fresh Prospective Frozen</td><td>73/77 63/64</td><td>94.8 87.2-98.6</td><td>740/744</td><td>99.5</td><td>98.6-99.9</td></tr><tr><td></td><td></td><td>98.4 91.6-100</td><td>683/683</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>18/18</td><td>100 81.5-100</td><td>401/401</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>44/44</td><td>100 92.0-100</td><td>175/176</td><td>99.4</td><td>96.9-100</td></tr><tr><td>Overall</td><td>198/203</td><td>97.5 94.3-99.2</td><td>1999/2004*</td><td>99.8</td><td>99.4-99.9</td></tr><tr><td rowspan="5">agaluctiae (Group B)</td><td>Prospective Fresh</td><td>8/8</td><td>100 63.1-100</td><td>813/813</td><td>100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>10/10</td><td>100 69.2-100</td><td>737/737</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>3/3</td><td>100 29.2-100</td><td>416/416</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>15/15</td><td>100 78.2-100</td><td>205/205</td><td>100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>36/36</td><td>100 90.3-100</td><td>2171/2171</td><td>100</td><td>99.8-100</td></tr><tr><td rowspan="5">Streptococcus pheumoniae</td><td>Prospective Fresh</td><td>15/15</td><td>100 78.2-100</td><td>805/806</td><td>99.9</td><td>99.3-100</td></tr><tr><td>Prospective Frozen</td><td>10/10</td><td>100 69.2-100</td><td>737/737</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>4/5</td><td>80.0 28.4-99.5</td><td>413/414</td><td>99.8</td><td>98.7-100</td></tr><tr><td>Seeded Frozen</td><td>7/7</td><td>100 59.0-100</td><td>213/213</td><td></td><td></td></tr><tr><td>Overall</td><td>36/37</td><td>97.3 85.8-99.9</td><td>2168/2170</td><td>100</td><td>98.3-100</td></tr><tr><td rowspan="5">Streptococcus pyogenes (Group A)</td><td>Prospective Fresh</td><td>5/5</td><td>47.8-100</td><td></td><td>99.9</td><td>99.7-100</td></tr><tr><td>Prospective Frozen</td><td>2/2</td><td>100 100</td><td>815/816 745/745</td><td>99.9</td><td>99.3-100</td></tr><tr><td>Seeded Fresh</td><td>9/9</td><td>15.8-100 100 66.4-100</td><td>410/410</td><td>100 100</td><td>99.5-100 99.1-100</td></tr><tr><td>Seeded Frozen</td><td>22/22</td><td>100 84.6-100</td><td>198/198</td><td>100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>38/38</td><td>100 90.7-100</td><td>2168/2169</td><td>99.9</td><td>99.7-100</td></tr></table>

Agreement (NPA) refer to performance with the seeded specimens. . epidermidis, and 1 was S. haemolyticus. were in sequentially-tested specimens and may be due to sample mix-up.

Table 7. FilmArray BClD Clinical Performance Summary - Gram-Negative Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification plus 16S Sequencing for Speciation for A. baumannil)   

<table><tr><td rowspan="2" colspan="2">Gram-Negative Bacteria</td><td colspan="3">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TP/TP + FP %</td><td>95% CI</td></tr><tr><td rowspan="5">Acinetobacter baumannii</td><td>Prospective Fresh</td><td>7/7</td><td>100</td><td>59.0-100 59.0-100</td><td>813/814 99.9</td><td>99.3-100</td></tr><tr><td>Prospective Frozen</td><td>7/7</td><td>100</td><td></td><td>739/740 99.9</td><td>99.2-100</td></tr><tr><td>Seeded Fresh</td><td>20/20</td><td>100</td><td>83.2-100</td><td>397/399 99.5</td><td>98.2-99.9</td></tr><tr><td>Seeded Frozen</td><td>17/17</td><td>100</td><td>80.5-100</td><td>202/203 99.5</td><td>97.3-100</td></tr><tr><td>Overall</td><td>51/51</td><td>100</td><td>93.0-100</td><td>2151/2156 99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Enterobacteriaceae</td><td>Prospective Fresh</td><td>153/156</td><td>98.1</td><td>94.5-99.6</td><td>665/665 100</td><td>99.4-100</td></tr><tr><td>Prospective Frozen</td><td>150/154</td><td>97.4</td><td>93.5-99.3</td><td>589/593 99.3</td><td>98.3-99.8</td></tr><tr><td>Seeded Fresh</td><td>93/93</td><td>100</td><td>96.1-100</td><td>326/326 100</td><td>98.9-100</td></tr><tr><td>Seeded Frozen</td><td>94/95</td><td>98.9</td><td>94.3-100</td><td>125/125 100</td><td>97.1-100</td></tr><tr><td>Overall</td><td>490/498</td><td>98.4</td><td>96.9-99.3</td><td>1705/1709 99.8</td><td>99.4-99.9</td></tr><tr><td rowspan="4">Enterobacter cloacae complex</td><td>Prospective Fresh</td><td>10/11. 11/11</td><td>90.9</td><td>58.7-99.8</td><td>809/810 99.9</td><td>99.3-100</td></tr><tr><td>Prospective Frozen</td><td></td><td>100</td><td>71.5-100</td><td>734/736 99.7</td><td>99,0-100</td></tr><tr><td>Seeded Fresh</td><td>8/8</td><td>100</td><td>63.1-100</td><td>411/411 100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>9/9</td><td>100</td><td>66.4-100</td><td>211/211 100</td><td>98.3-100</td></tr><tr><td rowspan="5">Escherichia coli</td><td>Overall Prospective Fresh</td><td>38/39 77/79</td><td>97.4</td><td>86.5-99.9</td><td>2165/2168 99.9 100</td><td>99.6-100</td></tr><tr><td></td><td>68/69</td><td>97.5</td><td>91.2-99.7</td><td>742/742</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>4/4</td><td>98.6</td><td>92.2-100</td><td>674/678 99.4</td><td>98.5-99.8</td></tr><tr><td>Seeded Fresh</td><td></td><td>100</td><td>39.8-100</td><td>414/415 99.8</td><td>98,7-100</td></tr><tr><td>Seeded Frozen Overall</td><td>1/1 150/153^</td><td>100</td><td>2.5-100</td><td>219/219 100</td><td>98.3-100</td></tr><tr><td rowspan="5">Klebsiella oxytoca</td><td>Prospective Fresh</td><td>4/4</td><td>98</td><td>94.4-99.6</td><td>2049/2051d 99.8 100</td><td>99.4-99.9</td></tr><tr><td></td><td>1/2</td><td>100</td><td>39.8-100</td><td>817/817</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td></td><td>50</td><td>1.3-98.7</td><td>744/745 99.9</td><td>99.3-100</td></tr><tr><td>Seeded Fresh</td><td>32/36</td><td>88.9</td><td>73.9-96.9</td><td>383/383 100</td><td>99.0-100</td></tr><tr><td>Seeded Frozen Overall</td><td>22/22 59/64</td><td>100</td><td>84.6-100</td><td>198/198 100</td><td>98.2-100</td></tr><tr><td rowspan="5">Klebsiella pneumoniae</td><td>Prospective Fresh</td><td>33/34</td><td>92.2</td><td>82.7-97.4</td><td>2142/2143 99.9 99.9</td><td>99.7-100</td></tr><tr><td>Prospective Frozen</td><td>35/37</td><td>97.1</td><td>84.7-99.9</td><td>786/787</td><td>99.3-100</td></tr><tr><td>Seeded Fresh</td><td></td><td>94.6</td><td>81.8-99.3</td><td>705/710 99.3</td><td>98.4-99.8</td></tr><tr><td>Seeded Frozen</td><td>13/13</td><td>100</td><td>75.3-100</td><td>403/406 99.3</td><td>97.9-99.8</td></tr><tr><td>Overall</td><td>21/21 102/105!</td><td>100</td><td>83.9-100 291.9-99.4</td><td>199/199 100</td><td>98.2-100</td></tr><tr><td rowspan="5">Proteus</td><td>Prospective Fresh</td><td>11/11</td><td>97.1 100</td><td>71.5-100</td><td>2093/2102 99.6 810/810 100</td><td>99.2-99.8</td></tr><tr><td>Prospective Frozen</td><td>11/11</td><td>100</td><td>71.5-100</td><td>736/736</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>212</td><td>100</td><td>15.8-100</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Frozen</td><td></td><td></td><td></td><td>417/417 100</td><td>99.1-100</td></tr><tr><td>Overall</td><td>15/15 39/39</td><td>100 100</td><td>78.2-100 91.0-100</td><td>205/205 100</td><td>98.2-100</td></tr><tr><td rowspan="5">Serratiu murcescens</td><td>Prospective Fresh</td><td>14/14</td><td>100</td><td>76.8-100</td><td>2168/2168 100 807/807 100</td><td>99.8-100</td></tr><tr><td>Prospective Frozen</td><td>8/8</td><td>100</td><td>63.1-100</td><td>739/739</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>28/28</td><td></td><td>87.7-100</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Frozen</td><td></td><td>100</td><td>81.0-99.9</td><td>390/391</td><td>99.7 98.6-100</td></tr><tr><td>Overall</td><td>26/27 76/778</td><td>96.3</td><td>93.0-100</td><td>193/193 100</td><td>98.1-100</td></tr><tr><td rowspan="5">Haemophilus influenzae</td><td>Prospective Fresh</td><td>5/5</td><td>98.7 100</td><td>47.8-100</td><td>2129/2130* 99.9 816/816 100</td><td>99.7-100</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100</td><td>29.2-100</td><td>744/744</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>29/29</td><td></td><td>88.1-100</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Frozen</td><td></td><td>100</td><td></td><td>390/390</td><td>100 99.1-100</td></tr><tr><td>Overall</td><td>6/6 43/43</td><td>100</td><td>54.1-100</td><td>214/214 100</td><td>98.3-100</td></tr><tr><td rowspan="5">Neisseria meningitidis</td><td>Prospective Fresh</td><td>1/1</td><td>100 100</td><td>91.8-100 2.5-100</td><td>2164/2164 100 820/820 100</td><td>99.8-100 99.6-100</td></tr><tr><td>Prospective Frozen</td><td>0/0</td><td></td><td></td><td>747/747 100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>30/30</td><td>100</td><td>88.4-100</td><td>389/389</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td></td><td></td><td>47.8-100</td><td></td><td>100</td></tr><tr><td>Overall</td><td>5/5 36/36</td><td>100 100</td><td>90.3-100</td><td>215/215 100 2171/2171 100</td><td>98.3-100 99.8-100</td></tr><tr><td rowspan="5">Pseudomonas aeruginosa</td><td>Prospective Fresh</td><td>19/19</td><td>100</td><td>82.4-100</td><td>802/802 100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>32/33</td><td>97</td><td>84.2-99.9</td><td>713/714 99.9</td><td>99.2-100</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td>-</td><td>419/419</td><td>100 99.1-100</td></tr><tr><td>Seeded Frozen</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Overall</td><td>0/0 51/52*</td><td>: 98.1</td><td>• 89.7-100</td><td>220/220 100 2154/2155 99.9</td><td>98.3-100 99.7-100</td></tr></table>

Agreement (NPA) refer to performance with the seeded specimens with the $\pmb { A }$

bereziniae, and A. radioresistens; these 6 isolates did not cross-react with the A. baumannii assay.

isolate from another false negative specimen, identified as $E$ coli by phenotypic methods, was identified as Pasteurella, and not E. coli, by bidirectional sequencing.

$E .$ l Bidirectional sequencing identified 4/5 isolates from false negative $K .$ oxyloca specimens as the closely related species, Raoultella ornithinolytica, and not $K .$ oxyca. The misidentification is a known limitation of phenotypic testing methods for this species.

T $K .$ . pneumoniae. 6/9 false positive $K .$ pneumoniae results appear to be due to cross-reactivity with Enterobacter aerogenes and Roulella ornithinolytica (misidentified as $K ,$ oxytoca by phenotypic methods).

B misidentified as $\boldsymbol { K }$ ,oxytoca by phenotypic methods).

$P$ stulzeri and not P. aeruginosu.

Table 8. FilmArray BCID Clinical Performance Summary  Yeast Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="2" rowspan="2">Yeast</td><td colspan="3">Sensitivity/PPA*</td><td colspan="3">Specificity/NPA</td></tr><tr><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="5">Candida albicans</td><td>Prospective Fresh Prospective Frozen</td><td>12/12</td><td>100</td><td>73.5-100</td><td>808/809</td><td>99.9</td><td>99.3-100</td></tr><tr><td></td><td>4/4</td><td>100</td><td>39.8-100</td><td>740/743</td><td>99.6</td><td>98.8-99.9</td></tr><tr><td>Seeded Fresh</td><td>47/47</td><td>100</td><td>92.5-100</td><td>372/372</td><td>100</td><td>99.0-100</td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100</td><td>2.5-100</td><td>219/219</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>64/64</td><td>100</td><td>94.4-100</td><td>2139/2143</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Candida glabrata</td><td>Prospective Fresh Prospective Frozen</td><td>6/6</td><td>100</td><td>54.1-100</td><td>813/815</td><td>99.8</td><td>99.1-100</td></tr><tr><td></td><td>6/6</td><td>100</td><td>54.1-100</td><td>741/741</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>32/32</td><td>100</td><td>89.1-100</td><td>387/387</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>49/49</td><td>100</td><td>92.7-100</td><td>2156/2158</td><td>99.9</td><td>99.7-100</td></tr><tr><td rowspan="5">Candida krusei</td><td>Prospective Fresh Prospective Frozen</td><td>2/2 2/2</td><td>100</td><td>15.8-100</td><td>819/819</td><td>100</td><td>99,6-100</td></tr><tr><td>Seeded Fresh</td><td></td><td>100</td><td>15.8-100</td><td>745/745</td><td>100</td><td>99.5-100</td></tr><tr><td></td><td>28/28</td><td>100</td><td>87.7-100</td><td>391/391</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall Prospective Fresh</td><td>37/37</td><td>100</td><td>90.5-100</td><td>2170/2170</td><td>100</td><td>99.8-100</td></tr><tr><td rowspan="5">Candida parapsilosis</td><td>Prospective Frozen</td><td>3/3 4/4</td><td>100</td><td>29.2-100</td><td>818/818</td><td>100</td><td>99.6-100</td></tr><tr><td>Seeded Fresh</td><td>47/49</td><td>100</td><td>39.8-100</td><td>742/743</td><td>99.9</td><td>99.3-100</td></tr><tr><td>Seeded Frozen</td><td></td><td>95.9</td><td>86.0-99.5</td><td>370/370</td><td>100</td><td>99.0-100</td></tr><tr><td></td><td>5/5</td><td>100</td><td>47.8-100</td><td>214/215</td><td>99.5</td><td>97.4-100</td></tr><tr><td>Overall</td><td>59/61b</td><td>96.7</td><td>88.7-99.6</td><td>2144/2146</td><td>99.9</td><td>99.7-100</td></tr><tr><td rowspan="5">Candida tropicalis</td><td>Prospective Fresh Prospective Frozen</td><td>0/0</td><td>-</td><td>-</td><td>821/821</td><td>100</td><td>99.6-100</td></tr><tr><td></td><td>3/3</td><td>100</td><td>29.2-100</td><td>744/744</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>31/31</td><td>100</td><td>88.8-100</td><td>388/388</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>39/39</td><td>100</td><td>91.0-100</td><td>2168/2168</td><td>100</td><td>99.8-100</td></tr></table>

Agreement (NPA) refer to performance with the seeded specimens . metapsilosis. This misidentification is a known limitation of phenotypic identification methods.

As the antimicrobial resistance gene results are not reported in the absence of a presumptively associated organism, performance was calculated only for samples in which FilmArray BCID detected an appropriate organism. Performance was calculated separately against the two comparator methods; PCR/sequencing direct from the blood culture specimens and PCR/sequencing from organisms isolated from the blood cultures. When comparing to PCR/sequencing from bacterial isolates, performance was only calculated for specimens in which FilmArray BCID detected an appropriate organism and from which an appropriate organism isolate was obtained (i.e., antimicrobial resistance gene results could be obtained for both methods). The NPA for mecA and vanA/B are lower when comparing to PCR/sequencing from bacterial isolates than to

PCR/sequencing direct from blood culture primarily due to the reference methods not isolating a resistant clone of an applicable organism. This may be due to heterogeneous resistance within a population of cultured organisms or co-culturing of multiple indistinguishable applicable organisms with different resistance profiles (e.g., culturing a resistant Staphylococcus along with a sensitive Staphylococcus).

Table 9. FilmArray BCID Clinical Performance Summary - Antimicrobial Resistance Genes (Comparator Method: PCR/Sequencing Direct from Blood Culture).   

<table><tr><td colspan="2"></td><td colspan="2">Sensitivity /PPA²</td><td colspan="3">Specificity /NPA²</td></tr><tr><td colspan="2">Antimicrobial Resistance Genes</td><td colspan="2">TP/TP + FN .% 95% CI</td><td colspan="3">TN/TN + FP %</td></tr><tr><td colspan="7">mecA - Methicillin Resistance Gene</td></tr><tr><td rowspan="4">mecA All Staphylococcus Detected</td><td>Prospective Fresh</td><td>253/257</td><td>98.4% 96.1-99.6% 98.3%</td><td>147/150</td><td>98.0%</td><td>94.3-99.6%</td></tr><tr><td>Prospective Frozen</td><td>233/237</td><td>95.7-99.5%</td><td>134/136</td><td>98.5%</td><td>94.8-99.8%</td></tr><tr><td>Seeded Fresh</td><td>1/1 100%</td><td>n/a</td><td>0/0</td><td>•</td><td>-</td></tr><tr><td>Seeded Frozen</td><td>1/1 100%</td><td>n/a</td><td>0/0</td><td>•</td><td>•</td></tr><tr><td rowspan="7">mecA Staphylococcus Detected; S. aureus Detected</td><td>Overall Prospective Fresh</td><td>488/496 98.4% 97.1%</td><td>96.8-99.3%</td><td>281/286</td><td>98.3%</td><td>96.0-99.4%</td></tr><tr><td>Prospective Frozen</td><td>67/69</td><td>89.9.99.6%</td><td>64/64</td><td>100%</td><td>94.4-100%</td></tr><tr><td>Seeded Fresh</td><td>70/70</td><td>100% 94.9-100%</td><td>54/54</td><td>100%</td><td>93.4-100%</td></tr><tr><td></td><td>0/0</td><td>-</td><td>0/0</td><td>•</td><td></td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td>• •</td><td>0/0</td><td>•</td><td></td></tr><tr><td>Overall</td><td>137/139</td><td>98.6% 94.9-99.8% 98.9%</td><td>118/118</td><td>100%</td><td>96.9-100%</td></tr><tr><td>Prospective Fresh Prospective Frozen</td><td>186/188 163/167</td><td>96.2-99.9%</td><td>83/86</td><td>96.5%</td><td>90.1-99.3%</td></tr><tr><td rowspan="5">Staphylococcus Detected; S. aureus Not Detected</td><td>Seeded Fresh</td><td>1/1</td><td>97.6% 94.0-99.3%</td><td>80/82</td><td>97.6%</td><td>91.5-99.7%</td></tr><tr><td>Seeded Frozen</td><td></td><td>100% n/a</td><td>0/0</td><td>*</td><td></td></tr><tr><td></td><td>1/1</td><td>100% n/a</td><td>0/0</td><td>-</td><td>•</td></tr><tr><td>Overall</td><td>351/357</td><td>98.3% 96.4-99.4%</td><td>163/168</td><td>97.0%</td><td>93.2-99.0%</td></tr><tr><td colspan="6">vanA/B - Vancomycin Resistance Genes</td></tr><tr><td rowspan="5">vanA/B Enterococcus Detected</td><td>Prospective Fresh Prospective Frozen</td><td>23/23 13/13</td><td>100% 85.2-100% 100% 75.3-100%</td><td>36/36 30/30</td><td>100%</td><td>90.3-100%</td></tr><tr><td>Seeded Fresh</td><td>12/12</td><td>100% 73.5-100%</td><td>0/0</td><td>100%</td><td>88.4-100%</td></tr><tr><td>Seeded Frozen</td><td></td><td></td><td></td><td></td><td>•</td></tr><tr><td></td><td>16/16</td><td>100% 79.4-100%</td><td>1/1</td><td>100%</td><td>n/a</td></tr><tr><td>Overall</td><td>64/64 100%</td><td>94.4-100%</td><td>67/67</td><td>100%</td><td>94.6-100%</td></tr><tr><td colspan="8">KPC - Carbapenem Resistance Genc (Carbapenemase)</td></tr><tr><td rowspan="4">KPC Enterobacteriaceae and/or A. baumannii and/or P. aeruginosa Detected</td><td>Prospective Fresh Prospective Frozen</td><td>3/3 3/3 100%</td><td>100% 29.2-100% 29.2-100%</td><td>177/177 187/187</td><td>100% 100%</td><td>97.9-100% 98.0-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10</td><td>100% 69.2-100%</td><td>105/105</td><td>100%</td><td>96.5-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100% 85.2-100%</td><td>89/89</td><td>100%</td><td>95.9-100%</td></tr><tr><td>Overall</td><td>39/39 100%</td><td>91.0-100%</td><td>558/558</td><td>100%</td><td>99.3-100%</td></tr><tr><td rowspan="5">KPC Enterobacteriuceae Detected KPC</td><td>Prospective Fresh Prospective Frozen</td><td>3/3 3/3</td><td>100% 29.2-100%</td><td>150/150</td><td>100%</td><td>97.6-100%</td></tr><tr><td>Seeded Fresh</td><td></td><td>100% 29.2-100% 100%</td><td>151/151</td><td>100%</td><td>97.6-100%</td></tr><tr><td></td><td>10/10</td><td>69.2-100%</td><td>83/83</td><td>100%</td><td>95.7-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100% 85.2-100%</td><td>71/71</td><td>100%</td><td>94.9-100%</td></tr><tr><td>Overall</td><td>39/39</td><td>100% 91.0-100%</td><td>455/455</td><td>100%</td><td>99.2-100%</td></tr><tr><td rowspan="4">Enterobacteriaceae Not Detected;</td><td>Prospective Fresh</td><td>0/0</td><td>• •</td><td>27/27</td><td>100%</td><td>87.4-100%</td></tr><tr><td>Prospective Frozen</td><td>0/0</td><td>• -</td><td>36/36</td><td>100%</td><td>90.3-100%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>.</td><td>22/22</td><td>100%</td><td>84.6-100%</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td></td><td>18/18</td><td>100%</td><td>81.5-100%</td></tr><tr><td>A. baumannii and/or P. aeruginosa Detected</td><td>Overall</td><td>0/0 -</td><td></td><td>103/103</td><td>100%</td><td>96.5-100%</td></tr></table>

Agreement (NPA) refer to performance with the seeded specimens.

Table 10. FilmArray BCID Clinical Performance Summary - Antimicrobial Resistance Genes (Comparator Method: PCR/Sequencing of Cultured Isolates)   

<table><tr><td colspan="2">ANTIMICROBIAL RESISTANCE GENES</td><td colspan="2">Positive Percent Agreement TP/TP + FN % 95% CI</td><td colspan="3">Negative Percent Agreement </td></tr><tr><td colspan="2"></td><td colspan="2">mecA - Methicillin Resistance Gene</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">mecA All Staphylococcus Detected</td><td>Prospective Fresh</td><td>234/236</td><td>99.2% 97.0-99.9%</td><td>149/172</td><td>86.7%</td><td>80.6-91.3%</td></tr><tr><td>Prospective Frozen</td><td>219/222</td><td>98.6% 96.1-99.7%</td><td>135/151</td><td>89.4%</td><td>83.4-93.8%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>•</td><td>0/0</td><td></td><td></td></tr><tr><td>Seeded Frozen</td><td>1/1 100%</td><td>n/a</td><td>0/0</td><td>• •</td><td>- -</td></tr></table>

<table><tr><td colspan="2">ANTIMICROBIAL RESISTANCE GENES</td><td colspan="2">Positive Percent Agreement TPATP +FN .95% CI</td><td colspan="3">Negative Percent Agreement </td></tr><tr><td rowspan="2"></td><td>Overall</td><td>454/459 98.9%</td><td>% 97.5-99.6%</td><td>TN/TN + FP 284/323</td><td>% 87.9%</td><td>95% C1 83.9-91.3%</td></tr><tr><td>Prospective Fresh</td><td>64/65</td><td>98.5% 91.7-100%</td><td>65/68</td><td>95.6%</td><td>87.6-99.1%</td></tr><tr><td rowspan="4">mecA Staphylococcus Detected; S. aureus Detected</td><td>Prospective Frozen</td><td>66/66</td><td>100% 94.6-100%</td><td>54/58</td><td>93.1%</td><td>83.3-98.1%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>- -</td><td>0/0</td><td></td><td>-</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td>•</td><td>0/0</td><td>-</td><td></td></tr><tr><td>Overall</td><td>130/131 99.2%</td><td>95.8-100%</td><td>119/126</td><td>- 94.4%</td><td>88.9-97.7%</td></tr><tr><td rowspan="5">mecA Staphylococcus Detected: S. aureus</td><td>Prospective Fresh</td><td>170/171</td><td>99.4% 96.8-100%.</td><td>84/104</td><td>80.8%</td><td>71.9-87.8%</td></tr><tr><td>Prospective Frozen</td><td>153/156</td><td>98.1% 94.5-99.6%</td><td>81/93</td><td>87.1%</td><td>78.6-93.2%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td>0/0</td><td>-</td><td></td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100% n/a</td><td>0/0</td><td></td><td></td></tr><tr><td>Ovverall</td><td>324/328 98.8%</td><td>96.9-99.7%</td><td>165/197</td><td>- 83.8%</td><td>• 77.9-88.6%</td></tr><tr><td colspan="8">vanA/B - Vancomycin Resistance Genes</td></tr><tr><td rowspan="4">vanA/B Enterococcus Detected</td><td>Prospective Fresh</td><td>20/20 100%</td><td>83.2-100%</td><td>36/39</td><td>92.3%</td><td>79.1-98.4%</td></tr><tr><td>Prospective Frozen</td><td>12/12</td><td>100% 73.5-100%</td><td>30/31</td><td>96.8%</td><td>83.3-99.9%</td></tr><tr><td>Seeded Fresh</td><td>12/12</td><td>100% 73.5-100%</td><td>0/0</td><td></td><td>-</td></tr><tr><td>Seeded Frozen Overall</td><td>16/16 100% 60/60 100%</td><td>79.4-100% 94.0-100%</td><td>1/1 67/71</td><td>100% 94.4%</td><td>n/a 86.2-98.4%</td></tr><tr><td colspan="8">KPC - Carbapenem Resistance Gene (Carbapenemase)</td></tr><tr><td>KPC Enterobacteriaceae</td><td>Prospective Fresh Prospective Frozen</td><td>3/3</td><td>100% 29.2-100%</td><td>177/177</td><td>100%</td><td>97.9-100%</td></tr><tr><td rowspan="4">and/or A. baumanmii and/or</td><td></td><td>3/3</td><td>100% 29.2-100%</td><td>187/187</td><td>100%</td><td>98.1-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10 100%</td><td>69.2-100%</td><td>105/105</td><td>100%</td><td>96.5-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100% 85.2-100%</td><td>89/89</td><td>100%</td><td>95.9-100%</td></tr><tr><td>Overall</td><td>39/39 100%</td><td>91.0-100%</td><td></td><td></td><td></td></tr><tr><td rowspan="5">KPC Enterobacteriaceae Detected</td><td>Prospective Fresh</td><td>3/3</td><td></td><td>558/558</td><td>100%</td><td>99.3-100%</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100% 29.2-100% 100%</td><td>151/151 152/152</td><td>100%</td><td>97.6-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10</td><td>29.2-100% 69.2-100%</td><td></td><td>100%</td><td>97.6-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100%</td><td>83/83</td><td>100%</td><td>95.7-100%</td></tr><tr><td>Overall</td><td>39/39</td><td>100% 85.2-100% 91.0-100%</td><td>71/71</td><td>100%</td><td>94.9-100%</td></tr><tr><td rowspan="5">KPC Enterobacteriaceae Not Detected; A. baumannii and/or P. aeruginosa</td><td>Prospective Fresh</td><td>0/0</td><td>100%</td><td>457/457</td><td>100%</td><td>99.2-100%</td></tr><tr><td>Prospective Frozen</td><td></td><td>• •</td><td>26/26</td><td>100%</td><td>86.8-100%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>- -</td><td>35/35</td><td>100%</td><td>90.0-100%</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td></td><td>22/22</td><td>100%</td><td>84.6-100%</td></tr><tr><td>Overall</td><td>0/0 0/0</td><td></td><td>18/18 101/101</td><td>100% 100%</td><td>81.5-100% 96.4-100%</td></tr></table>

Performance of FilmArray BCID as compared to phenotypic antimicrobial susceptibility testing (AST) results was calculated for informational purposes. Results stratified by AST method are presented in Tables 11-13. Some PPA are lower when comparing results from bacterial isolates than to PCR/sequencing direct from blood culture because phenotypic AST testing is capable of detecting antimicrobial resistance due to mechanisms other than acquisition of mecA, vanA/B, or KPC.

Table 11. mecA Performance  Comparison to Phenotypic Antimicrobial Susceptibility Testing (AST) Methods Note: AST results were not provided for several isolates.   
BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel   

<table><tr><td colspan="2">PHENOTYPIC METHIODS</td><td colspan="2">Positive Percent Agreement TP/TP + FN % (95%CI)</td><td colspan="2">Negative Pereent Agreement TN/TN.+ FP % (95%CI)</td></tr><tr><td>Prospective</td><td>Cefoxitin Disc Diffusion</td><td>22/22</td><td>100%</td><td>15/15</td><td>100%</td></tr></table>

<table><tr><td rowspan="3">All Staphylococcus</td><td>Chromogenic Agar</td><td>42/46 91.3%</td><td>25/32</td><td>78.1%</td></tr><tr><td>Automated Antimicrobial Susceptibility Testing</td><td>366/380 96.3%</td><td>226/262</td><td>86.3%</td></tr><tr><td>All Methods</td><td>430/448 96.0% (93.7-97.6%)</td><td>266/309</td><td>86.1% (81.7-89.7%)</td></tr><tr><td rowspan="3">Prospective Staphylococcus, S. aureus Detected</td><td>Chromogenic Agar</td><td>10/11 90.9%</td><td>8/8</td><td>100%</td></tr><tr><td>Automated Antimicrobial Susceptibility Testing</td><td>117/119 98.3%</td><td>108/112</td><td>96.4%</td></tr><tr><td>All Methods</td><td>97.7% 127/130 (93.4- 99.5%)</td><td>116/120</td><td>96.7% (91.7-99.1%)</td></tr><tr><td>Seeded Staphylococcus</td><td>Automated Antimicrobial Susceptibility Testing</td><td>1/1 100%</td><td>0/0</td><td>-</td></tr></table>

Table 12. vanA/B Performance - Comparison to Phenotypic Vancomycin AST Methods   

<table><tr><td colspan="2">PHENOTYPIC METHODS</td><td colspan="2">Positive Percent Agreement TP/TP + FN % (95%CI)</td><td colspan="2">Negative Percent Agreement TN/TN + FP % (95%CI)</td></tr><tr><td rowspan="4">Prospective Enterococcus</td><td>Vancomycin Screen Agar</td><td>3/3</td><td>100%</td><td>5/5</td><td>100%</td></tr><tr><td>Vancomycin Disc Diffusion</td><td>0/1</td><td>0.0%</td><td>-</td><td>•</td></tr><tr><td>Automated Antimicrobial Susceptibility Testing</td><td>29/30</td><td>96.7%</td><td>55/58</td><td>94.8%</td></tr><tr><td>All Methods</td><td>32/34</td><td>94.1% (80.3 - 99.3%)</td><td>60/63</td><td>95.2% (86.7 - 99.0%)</td></tr><tr><td rowspan="3">Seeded Enterococcus</td><td>Vancomycin Disc Diffusion</td><td>14/14</td><td>100%</td><td>1/1</td><td>100%</td></tr><tr><td>Vancomycin Screen Agar</td><td>14/14</td><td>100% 100%</td><td>-</td><td>-</td></tr><tr><td>All Methods</td><td>28/28</td><td>(87.7-100%)</td><td>1/1</td><td>100% (n/u)</td></tr><tr><td>Combined Prospective and Seeded Enterococcus</td><td>All Methods</td><td>60/62 </td><td>96.8% (88.8 - 99.6%)</td><td>61/64</td><td>95.3% (86.9 - 99.0%</td></tr></table>

genes by bi-directional sequence analysis.

Table 13. KPC Performance - Comparison to Phenotypic Carbapenem AST Methods Note: AST results were not provided for several isolates. NoA than acquisition of the KPC gene $\left( b { \mathit { i a } } _ { \mathrm { K P C } } \right)$ .These bacteria very rarely carry the KPC gene.   

<table><tr><td colspan="2">PHENOTYPIC METHIODS</td><td>Positive Percent Agreement TP/ % (95%Cl) TP+FN</td><td>Negative Percent Agreement TN/ TN+ FP</td><td>% (95%C1)</td></tr><tr><td>Prospective A. baumannii</td><td>Automated Antimicrobial Susceptibility Testing</td><td>0/10 0%</td><td>4/4</td><td>100%</td></tr><tr><td>Seeded A. baumannii</td><td>Meropenem Disc Diflusion</td><td>0/30 0%</td><td>9/9</td><td>100%</td></tr><tr><td colspan="2">A. baumamnii - All Methods</td><td>0/40 0% (n/a)</td><td>13/13</td><td>100% (75.3-100%)</td></tr><tr><td rowspan="4">Prospective P. ueruginosa</td><td>Automated Antimicrobial Susceptibility Testing</td><td>0/10 0%</td><td>32/32</td><td>100%</td></tr><tr><td>Meropenem Disc Diffusion</td><td>- -</td><td>6/6</td><td>100%</td></tr><tr><td>Meropenem/Ertapenem Disc Diffusion</td><td>0/1 0%</td><td>2/2</td><td>100%</td></tr><tr><td>P. aeruginosa  All Methods</td><td>0% (n/a) 0/11</td><td>40/40</td><td>100% (91.2-100%)</td></tr><tr><td colspan="2">Prospective Automated Antimicrobial K. pneumoniae Susceptibility Testing</td><td>6/6</td><td>100%</td><td>64/64 100%</td></tr><tr><td>Seeded</td><td>Meropenem Disc Diffusion</td><td>19/19</td><td>• 100%</td><td>I/1 100%</td></tr><tr><td>K. pneumoniae</td><td>Modified Hodge Test</td><td>11/11</td><td>100% 1/1</td><td>100%</td></tr><tr><td colspan="2">PHENOTYPIC METHODS $\a$</td><td>Positive Percent Agreement a TP/ % (95%CI) TP +FN</td><td>TN/</td><td>Negative Percent Agreement % (95%CI)</td></tr><tr><td></td><td>(Meropenem)</td><td></td><td>TN+FP</td><td></td></tr><tr><td colspan="2">K. pueumoniae — All Methods</td><td>100% 36/36 (90.3-100%)</td><td>66/66</td><td>100% (94.6-100%)</td></tr><tr><td>Prospective E. cloacae</td><td>Automated Antimicrobial Susceptibility Testing</td><td>•</td><td></td><td>22/22 100%</td></tr><tr><td rowspan="3">Seeded E. cloacue</td><td>Automated Antimicrobial Susceptibility Testing</td><td>-</td><td>3/3</td><td>100%</td></tr><tr><td>Meropenem Disc Diffusion Modified Hodge Test</td><td>0/1</td><td>0% :</td><td>-</td></tr><tr><td>(Meropenem)</td><td>22 100%</td><td>11/11</td><td>100%</td></tr><tr><td colspan="2">E. cloacae  All Methods</td><td>2/3</td><td>66.7% (9.4-99.2%)</td><td>100% 36/36 (90.3-100%)</td></tr><tr><td>Prospective E. coli Seeded</td><td>Aulomated Antimicrobial Susceptibility Testing</td><td>-</td><td></td><td>144/144 100%</td></tr><tr><td>E. coli</td><td>Modified Hodge Test (Meropenem)</td><td>1/1</td><td>100% 4/4</td><td>100%</td></tr><tr><td colspan="2">E. coli - All Methods</td><td>100% (n/a) 1/I</td><td>148/148</td><td>100% (97.5-100%)</td></tr><tr><td>Prospective P. mirabilis</td><td>Automated Antimicrobial Susceptibility Testing</td><td>:</td><td>- 21/21</td><td>100%</td></tr><tr><td>Seeded P. mirabilis</td><td>Meropenem Dise Diffusion Modified Hodge Test (Meropenem)</td><td>- 0/1</td><td>- 0%</td><td>4/4 100% 11/11 100%</td></tr><tr><td colspan="2">P. mirabilis— All Methods</td><td>0/1</td><td>0% (n/a) 36/36</td><td>100% (90.3-100%)</td></tr><tr><td>Prospective All Other Enterobacteriaceue</td><td>Automated Antimicrobial Susceptibility Testing</td><td></td><td>43/43</td><td>100%</td></tr><tr><td rowspan="3">Seeded All Other Enterobacteriuceae</td><td>Automated Antimicrobial Susceptibility Testing</td><td>•</td><td>42/42</td><td>100%</td></tr><tr><td>Meropenem Disc Diffusion</td><td></td><td>13/13</td><td>100%</td></tr><tr><td>Modified Hodge Test (Meropenem)</td><td></td><td>61/61</td><td>100%</td></tr><tr><td colspan="2">All Other Enterobacteriaceae  All Methods</td><td></td><td></td><td>100% 159/159 (97.7-100%)</td></tr></table>

solat n.co $P$ by bi-directional sequence analysis.

Table 14. Stratification of Enterococcus Clinical Performance by Species (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   
Table 15. Stratification of Staphylococcus Clinical Performance by Species (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td rowspan=2 colspan=1>Enterococcus species</td><td rowspan=1 colspan=2>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>E. avium</td><td rowspan=1 colspan=1>212(100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. casseliflavus</td><td rowspan=1 colspan=1>1/250%)</td><td rowspan=1 colspan=1>1/1(100%)</td></tr><tr><td rowspan=1 colspan=1>E. durans</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>55/56 (98.2%)</td><td rowspan=1 colspan=1>8/8 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. faecalis + E. faecium</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>36/37 (97.3%)</td><td rowspan=1 colspan=1>9/9 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. gallinarum</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus sp. (not speciated)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Enterococcus</td><td rowspan=1 colspan=1>98/101 (97.0%)95%CI = 91.6-99.4%</td><td rowspan=1 colspan=1>29/29 (100%)95%CI = 88.1-100%</td></tr></table>

<table><tr><td colspan="1" rowspan="2">Staphylococcus species</td><td colspan="2" rowspan="1">Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">256/257 (99.6%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. auricularis</td><td colspan="1" rowspan="1">0/1 (0%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S.capitis</td><td colspan="1" rowspan="1">15/17(88.2%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus species</td><td colspan="2" rowspan="1">2     Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">S. capitis + S. epidermidis</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. capitis + S. hominis</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">:</td></tr><tr><td colspan="1" rowspan="1">S. capitis + S. lugdunensis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. carnosus</td><td colspan="1" rowspan="1">0/1 (0%</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S.cohnii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. cohnii + S. hominis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">200/201 (99.5%)</td><td colspan="1" rowspan="1">1/I (100%)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis + S. hominis</td><td colspan="1" rowspan="1">4/4 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis + Staphylococcus sp. (notspeciated)</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. haemolyticus</td><td colspan="1" rowspan="1">19/19 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. haemolyticus + S. hominis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. hominis</td><td colspan="1" rowspan="1">65/65 (100%)</td><td colspan="1" rowspan="1">*</td></tr><tr><td colspan="1" rowspan="1">S. hominis + Staphylococcus sp. (notspeciated)</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. intermedius</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. intermedius + Staphylococcus sp. (notspeciated)</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. lentus</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. saprophyticus</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. sciuri</td><td colspan="1" rowspan="1">0/1 (0%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. simulans</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. warneri</td><td colspan="1" rowspan="1">4/5 (80%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus sp. (not speciated)</td><td colspan="1" rowspan="1">180/200 (90%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">Overall Staphylococcus</td><td colspan="1" rowspan="1">769/797 (96.5%)95%C1 = 95.0-97.7%</td><td colspan="1" rowspan="1">1/1 (100%)95%CI = 1/a</td></tr></table>

F c     T that were detected by FilmArray BCID were not sequenced.

Table 16. Stratification of Streptococcus Clinical Performance by Species arator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="1" rowspan="2">Streptococcus species</td><td colspan="2" rowspan="1">Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">GroupA (Pyogenic)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1">7/7 (100%)</td><td colspan="1" rowspan="1">31/31 (100%)</td></tr><tr><td colspan="1" rowspan="1">Group B (Pyogenic)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">18/18 (100%)</td><td colspan="1" rowspan="1">18/18 (100%)</td></tr><tr><td colspan="1" rowspan="1">Group C/G (Pyogenic)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. canis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">*</td></tr><tr><td colspan="1" rowspan="1">S. equi/S. dysgalactiae</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus group C</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">Streptococcus group G</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Group D (Bovis Group)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. bovis</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S.equinus</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Group F (Anginosus Group)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">4/4 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. intermedius</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. constellatus</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mitis Group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. gordonii</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. mitis</td><td colspan="1" rowspan="1">8/9 (88.9%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. mitis + viridans streptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. mitis/S. oralis</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">S. mitis/S. oralis + viridans streptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. oralis</td><td colspan="1" rowspan="1">5/5 100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis + viridans streptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">12/12 (100%)</td></tr><tr><td colspan="1" rowspan="1">S. sanguinis</td><td colspan="1" rowspan="1">212(100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="2" rowspan="1">Prospective                    Seeded</td></tr><tr><td colspan="3" rowspan="1">Salivarius Group</td></tr><tr><td colspan="1" rowspan="1">S. salivarius</td><td colspan="1" rowspan="1">1/2 (50%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. salivarius + S. sanguinis group</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">S. vestibularis</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridans streptococci(not further speciated)</td><td colspan="1" rowspan="1">40/43 (93.0%)</td><td colspan="1" rowspan="1">1/1 (100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sp. (not speciated)</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">Overall Streptococcus</td><td colspan="1" rowspan="1">136/141 (96.5)95%CI= 91.9-98.8%</td><td colspan="1" rowspan="1">62/62 (100%)95%CI = 94.2-100%</td></tr></table>

Table 17. Stratification of Enterobacteriaceae Clinical Performance by Genus/Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification   

<table><tr><td rowspan=2 colspan=1>Enterobacteriaceae genus/species</td><td rowspan=1 colspan=2>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>Citrobucter freundii</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii + Escherichia coli</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>1/2(50%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>5/5100%)</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes + Klebsiellaoxyloca</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>19/19 (100%)</td><td rowspan=1 colspan=1>17/17 (100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacler cloacae complex</td><td rowspan=1 colspan=1>3/3100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter gergoviae</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>Enterobacter sakusaki</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Enterobacter sp.</td><td rowspan=1 colspan=1>1/1(100%)    ,</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>141/144 (98%)</td><td rowspan=1 colspan=1>5/5 (100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli + Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli + Providencia stuartii &quot;</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxyloca</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>58/58 (100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>67/68 (99%)</td><td rowspan=1 colspan=1>34/34 (100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae + Pantoeaagglomerans</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Leclercia adacarboxylata</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiib + Proteusmirabilis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pantoea agglomerans</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>Pantoea sp.</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>21/21 (100%)</td><td rowspan=1 colspan=1>15/15 (100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2/2(100%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella group B</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Salmonella group C.</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Salmonella sp.</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhi</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>22/22 (100%)</td><td rowspan=1 colspan=1>54/55 (98%)</td></tr><tr><td rowspan=1 colspan=1>Overall Enterobacteriaceae</td><td rowspan=1 colspan=1>303/310 (97.7%)95%CI = 95.4-99.1%</td><td rowspan=1 colspan=1>187/188 (99.5%)95%Cl =97.1-100%</td></tr></table>

FiAra C does ot dete rvenca tuarti; the posiiEntac esl s keyue o the presence of Escherichia coli in the blood culture. r   ea oe   t the presence of Proteus mirabilis in the blood culture.

Table 18. Stratification of Proteus Clinical Performance by Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   
BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel   

<table><tr><td colspan="1" rowspan="2">Proteus species</td><td colspan="2" rowspan="1">Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">22/22 (100%)</td><td colspan="1" rowspan="1">15/15 (100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2 (100%)</td></tr><tr><td>Overall Proteus</td><td>22/22 (100%)</td><td>17/17(100%)</td></tr><tr><td></td><td>95%CI =84.6-100%</td><td>95%CI =80.5-100%</td></tr></table>

FilmArray BCID reported a total of 81 prospective specimens with discernible multiple organism detections $( 5 . 2 \%$ of all prospective specimens; 81/1568). The majority of multiple detections (74/81; $9 1 . 3 \% )$ contained two discernible organisms, while $6 . 2 \%$ (5/81) contained three discernible organisms, and $2 . 5 \%$ (2/81) contained four discernible organisms. The most prevalent multiple detection was Enterococcus with Staphylococcus (S. aureus not detected) ( $1 . 3 \%$ of all specimens; 20/1568). Out of the 81 polymicrobial specimens, 29 contained one or more analytes that had not been detected with the reference/comparator methods, i.e., discrepant result.

Table 19. Discernible Multiple Detection Combinations as Determined by FilmArray BCID   

<table><tr><td rowspan=1 colspan=4>Distinct Multiple Detection Combinationsas Determined by FilmArray BCID</td><td rowspan=2 colspan=1>Soes Tne</td><td rowspan=2 colspan=1>thtsSeees</td><td rowspan=2 colspan=1>Discrepant Result(s)(Organism NotDetected byReference Method)</td></tr><tr><td rowspan=1 colspan=1>Organism 1 Results</td><td rowspan=1 colspan=1>Organism 2 Results</td><td rowspan=1 colspan=1>Organism 3 Results</td><td rowspan=1 colspan=1>Organism 4 Results</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex,Enterobacteriaceae</td><td rowspan=1 colspan=1>Escherichia coli,Enterobacteriaceae</td><td rowspan=1 colspan=1>Klebsiella oxytoca,Enterobacteriaceae</td><td rowspan=1 colspan=1>Klebsiellapneumoniae,Enterobacteriaceae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>E. cloacae, E. coli, K.oxyloca</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Streplococcus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C. albicans</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C. parapsilosis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Proteus,Enterobacteriaceae</td><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>|</td><td rowspan=1 colspan=1>Streptococcus</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Streptococcusagalactiae,Streptococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus,Enterobacteriaceae</td><td rowspan=1 colspan=1>Staphylococcusaureus.Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Staphylococcus,S. aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1>Streptococcusagalactiae,Streplococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1>Streptococcuspneumoniae,Streptococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Streptococcus,S. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Enterobacteriaceae</td><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Staphylococcus.S. aureus</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C. albicans</td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus,Staphylococcus</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>P. aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus,</td><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td colspan="4" rowspan="1">.. t      Distinct Multiple Detection Combinationsas Determined by FilmArray BCID</td><td colspan="1" rowspan="2">Soe Snn</td><td colspan="1" rowspan="2">prcemSennn</td><td colspan="1" rowspan="2">Discrepant Result(s)(Organism NotDetected byReference Method)</td></tr><tr><td colspan="1" rowspan="1">Organism 1 Results</td><td colspan="1" rowspan="1">Organism 2 Results</td><td colspan="1" rowspan="1">Organism 3 Results</td><td colspan="1" rowspan="1">Organism 4 Results</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobacteriaceae, E.coli</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">A. baumannii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae,Enteric</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriacede</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">E. coli,K. pneumoniae (2)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Proteus,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. glabrata</td></tr><tr><td colspan="1" rowspan="1">Proteus.Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Staphylococcus (3),Enterococcus (3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Pseudomonusaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streplococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. glabrata</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">C. albicans</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceue</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="4" rowspan="1">Total Specimens with Multiple Detections</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">x       , 4"9Distinct Multiple Detections by Reference/Conmiparator methods</td><td colspan="1" rowspan="2">Soe i</td><td colspan="1" rowspan="2">pidesSeecnds</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganismsNot Detected byFilmArray BCID)</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Flavobacteriumspecies</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">Staphylococcus,Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Neisseria species</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter woffii</td><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coryneform bacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Streplococcus oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Streptococcusmitis/oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pantoea species</td><td colspan="1" rowspan="1">Staphylococcusintermedius</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobacteriaceue</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Streptococcusparasanguis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capittis</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcusmitis/oralis</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Abiorophia defectiva</td><td colspan="1" rowspan="1">Staphyloccocus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii (seq. = A.nosocomialis/calcoacelicus)</td><td colspan="1" rowspan="1">Acinetobacterbaumannii (seq. = A.nosocomialis/calcoacelicus)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffli</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Sireptococcus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae.Enterobacteriaceae</td></tr><tr><td colspan="1" rowspan="1">Aerococcus species</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Bacillus pumilus</td><td colspan="1" rowspan="1">. Pseudomonasfluorescens/putida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Brevundimonasdiminuta</td><td colspan="1" rowspan="1">Weeksella virosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmlowb</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumjeikeium</td><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Coryne bacteriumspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Micrococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Siaphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Diphtheroids</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel   

<table><tr><td colspan="4" rowspan="1">i2         Distinct Multiple Detections by Reference/Comparator methods</td><td colspan="1" rowspan="2">Toee Cne</td><td colspan="1" rowspan="2">jtesSeecs</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganisnisNot Detected byFilmArray BCID</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Pasteurella multocida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli.Enterobucteriaceae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Stenotrophomonasmaliophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluencae</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Luctobacillusacidophilus</td><td colspan="1" rowspan="1">Streptococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcus species</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Proleus mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Neisseria species</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rhodococcus species</td><td colspan="1" rowspan="1">Staphylococcuswarneri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Saphylococcus aureus</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cupitis</td><td colspan="1" rowspan="1">Streplococcuspneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">Stenotrophomonusmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">Streptococcus sanguisgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4">Distinct Multiple Detections by Reference/Comparator methods</td><td rowspan="2">Scis T pscem</td><td rowspan="2">Seets</td><td rowspan="2">Discrepant Result(s) (Targeted Organisms Not Detected by FilmArray BCID)</td></tr><tr><td>Isolate 1</td><td>Isolate 2</td><td>Isolate 3</td><td>Isolate 4</td></tr><tr><td>Viridans streptococci</td><td>Viridans streptococci</td><td></td><td></td><td></td><td>0</td><td></td></tr><tr><td></td><td></td><td></td><td>Total</td><td>3 86</td><td>16</td><td></td></tr></table>

The reference method detected 201 off-panel organism isolates (i.e., those not targeted by FilmArray BCID) from the 1568 prospective cultures. The majority of these isolates belong to groups of organisms commonly considered to be blood culture contaminants (49 Corynebacterium/Diphtheroids, 33 Bacillus sp., and 27 Micrococcus sp., among others). Occurrence of off-panel organisms in the prospective arm of the clinical evaluation is presented in Table 21.

Table 21. Occurrence of Off-Panel Organisms as Determined by Reference/Comparator Methods   

<table><tr><td rowspan=1 colspan=1>Off-Panel Organism</td><td rowspan=1 colspan=1>NumberIdentified</td><td rowspan=1 colspan=1>Off-Panel Organism</td><td rowspan=1 colspan=1>NumberIdentified</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia sp. or Granulicatella sp. (formerlynutritionally-deficient Streptococci)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Flavobacterium species</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Achromobacter xylosoxidans</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fusarium species</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter sp. (not A. baumannii)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Kocuria kristinae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Actinomyces odontolyticus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Actinomyces species</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Lactobacillus species</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Aerococcus species</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Aerococcus viridans</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Micrococcus luteus/lylae</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Aeromonas sobria</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Micrococcus species</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Bacillus pumilus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Bacillus species</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Moraxella species</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Brevibacterium species</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Mycobacterium fortuitum complex</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Brevibucterium ensei</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Mycobacterium species</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Brevundimonas diminuta</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Neisseria species</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Brevindimonas vesicularis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Paenibacillus species</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia complex</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Pasteurella multocida</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Pasteurella species</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Capnocytophaga species</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Propionibacterium species</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Chryseobacterium meningosepticum(Elicabethkingia/Flavobacterium)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Pseudomonas fluorescens/putida</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Chryseobacterium indologenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Pseudomonas species</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Chryseomonas luteola</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Rhizobium radiobacter</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Rothia (Stomatococcus) mucilaginosa</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium mucifaciens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Sphingomonas mucosissima</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Corynebaclerium species/Diphtheroids</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus ne oformans</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Weeksella virosa</td><td rowspan=1 colspan=1>1</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

# Selected Analytic Studies

# Growth and Detection

A study was performed to establish the range of expected organism concentrations in. blood cultures that would be tested with the FilmArray BCID Panel from the time of positivity up to eight hours after positivity. All organism growth and testing was performed using seeded blood culture bottles (BACTECTM Plus Aerobic/F Medium incubated in the BACTECTM 9050 continuously monitoring blood culture instrument). Each microorganism was mixed with human whole blood and seeded directly into blood culture bottles for growth. At the time of positivity (and/or eight hours after positivity), the blood culture was removed from the instrument for plate enumeration (determination of CFU/mL) and FilmArray BCID testing. Three independent positive cultures (bottles) were evaluated for each organism at each time point and FilmArray testing was performed in triplicate for each bottle.

Table 22 summarizes the concentration of organism (CFU/mL) determined for a representative panel of 30 isolates. The number and percent of correct positive BCID Panel test results is provided for each isolate and overall $\%$ Detected). A correct result means that both the correct organism and antimicrobial resistance gene (where applicable) were detected in the sample.

Table 22. Summary of Organism Concentration (CFU/mL) in Positive Blood Cultures and Correct Detection of Organisms in Positive Blood Cultures by the FilmArray BCID Panel   

<table><tr><td colspan="1" rowspan="2">Species/solate(s) Tested</td><td colspan="3" rowspan="1">At Positivity           </td><td colspan="3" rowspan="1">8 Hours After Positivity</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1"># Detected/Total(% Detected)</td><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1"># Detected/Total(% Detected)</td></tr><tr><td colspan="7" rowspan="1">Gram-Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis [vanB+]JMI 368</td><td colspan="1" rowspan="1">4.60E+081.80E+082.62E+08</td><td colspan="1" rowspan="1">3.01E+08</td><td colspan="1" rowspan="1">9/9(100%).</td><td colspan="1" rowspan="1">7.25E+088.90E+081.07E+09</td><td colspan="1" rowspan="1">8.95E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium [vanA+]JMI 475</td><td colspan="1" rowspan="1">1.47E+081.53E+081.59E÷08</td><td colspan="1" rowspan="1">1.53E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.23E+081.64E+081.55E+08</td><td colspan="1" rowspan="1">1.81E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hiraeATCC 49135</td><td colspan="1" rowspan="1">1.26E+082.76E+083.25E+08</td><td colspan="1" rowspan="1">2.42E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.00E+086.60E+087.20E+08</td><td colspan="1" rowspan="1">7.27E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenesCDC F2380 (ATCC 43256)</td><td colspan="1" rowspan="1">4.50E+081.22E+091.18E+09</td><td colspan="1" rowspan="1">9.50E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.76E+092.31E+091.67E+09</td><td colspan="1" rowspan="1">1.91E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusATCC 11632</td><td colspan="1" rowspan="1">1.48E+082.00E+072.56E+07</td><td colspan="1" rowspan="1">6.45E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.75E+089.80E+081.21E+08</td><td colspan="1" rowspan="1">6.59E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus[MRSA/mecA]ATCC BAA-1747</td><td colspan="1" rowspan="1">1.41E+075.65E+066.05E+06</td><td colspan="1" rowspan="1">8.60E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">5.70E+073.85E+079.75E+07</td><td colspan="1" rowspan="1">6.43E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidisATCC 12228</td><td colspan="1" rowspan="1">1.38E+089.85E+071.16E+08</td><td colspan="1" rowspan="1">1.18E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.12E+083.95E+081.56E+09</td><td colspan="1" rowspan="1">7.22E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis[MRSE/mecA]</td><td colspan="1" rowspan="1">3.60E+073.75E+07</td><td colspan="1" rowspan="1">7.65E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.35E+096.80E+08</td><td colspan="1" rowspan="1">1.44E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="2">Species/Isolate(s) Tested</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">At Positivity</td><td colspan="3" rowspan="1">8 Hours After Positivity</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1"># Detected/Total(% Detected)</td><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1"># Deterted/Total(% Detected)</td></tr><tr><td colspan="1" rowspan="1">ATCC 29887</td><td colspan="1" rowspan="1">1.56E+08</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.29E+09</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiaeATCC 13813</td><td colspan="1" rowspan="1">4.50E+081.22E+089.15E+08</td><td colspan="1" rowspan="1">4.96E+08</td><td colspan="1" rowspan="1">9/9.(100%)</td><td colspan="1" rowspan="1">3.15E+085.80E+084.30E+08</td><td colspan="1" rowspan="1">4.42E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitisATCC 15914</td><td colspan="1" rowspan="1">1.57E+081.51E+096.90E+08</td><td colspan="1" rowspan="1">7.86E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.50E+092.03E+092.91E+09</td><td colspan="1" rowspan="1">2.15E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniaeATCC BAA-255</td><td colspan="1" rowspan="1">3.45E+082.67E+081.31E+09</td><td colspan="1" rowspan="1">6.41E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.03E+096.00E+081.37E+09</td><td colspan="1" rowspan="1">1.00E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenesATCC 19615</td><td colspan="1" rowspan="1">2.53E+082.44E+083.80E+08</td><td colspan="1" rowspan="1">2.92E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.38E+085.70E+088.90E+08</td><td colspan="1" rowspan="1">5.66E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram-</td><td colspan="1" rowspan="1">Gram-Negative Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniiATCC 9955</td><td colspan="1" rowspan="1">2.17E+081.44E+082.45E+08</td><td colspan="1" rowspan="1">2.02E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">4.85E+083.85E+084.35E+08</td><td colspan="1" rowspan="1">4.35E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaeATCC 13047</td><td colspan="1" rowspan="1">4.20E+083.95E+081.50E+08</td><td colspan="1" rowspan="1">3.22E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.23E+091.46E+092.19E+09</td><td colspan="1" rowspan="1">1.96E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliATCC 43888</td><td colspan="1" rowspan="1">9.80E+076.10E+071.93E+08</td><td colspan="1" rowspan="1">1.17E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.17E+091.39E+097.70E+07</td><td colspan="1" rowspan="1">8.79E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytocaATCC 13182</td><td colspan="1" rowspan="1">7.40E+086.85E+083.85E+08</td><td colspan="1" rowspan="1">6.03E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">3.05E+091.86E+091.20E+09</td><td colspan="1" rowspan="1">2.04E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca [+KPC]JMI 7818</td><td colspan="1" rowspan="1">6.15E+079.15E+073.05E+07</td><td colspan="1" rowspan="1">6.12E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.96E+092.00E+091.13E+09</td><td colspan="1" rowspan="1">1.70E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaeATCC 13883</td><td colspan="1" rowspan="1">4.35E+082.10E+089.15E÷08</td><td colspan="1" rowspan="1">5.20E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.60E+091.65E+091.58E+09</td><td colspan="1" rowspan="1">1.61E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae [+KPC]JMI 766</td><td colspan="1" rowspan="1">1.21E+082.50E+082.05E+08</td><td colspan="1" rowspan="1">1.92E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.14E+099.10E+087.70E+08</td><td colspan="1" rowspan="1">9.40E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilisATCC 29906</td><td colspan="1" rowspan="1">3.25E+071.04E+089.10E+07</td><td colspan="1" rowspan="1">7.58E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.04E+099.80E+087.30E+08</td><td colspan="1" rowspan="1">9.17E+08</td><td colspan="1" rowspan="1">9/92(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescensATCC 27137</td><td colspan="1" rowspan="1">8.35E+081.46E+094.90E+08</td><td colspan="1" rowspan="1">9.28E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.051+091.37E+091.02E+09</td><td colspan="1" rowspan="1">1.15E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens [+KPC]JMI 697</td><td colspan="1" rowspan="1">4.90E+083.90E+081.02E+08</td><td colspan="1" rowspan="1">3.27E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.19E+091.40E+092.42E+08</td><td colspan="1" rowspan="1">1.28E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (y ATCC 10211</td><td colspan="1" rowspan="1">2.80E+083.60E+082.23E+08</td><td colspan="1" rowspan="1">2.88E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">3.25E+093.35E+092.74E+09</td><td colspan="1" rowspan="1">3.11E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidisATCC 43744</td><td colspan="1" rowspan="1">2.07E+083.90E+081.55E+08</td><td colspan="1" rowspan="1">2.51E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">6.65E+087.65E+087.85E+08</td><td colspan="1" rowspan="1">7.38E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosuATCC 27853</td><td colspan="1" rowspan="1">1.34E+081.76E+089.75E+07</td><td colspan="1" rowspan="1">1.36E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.35E+091.39E+081.76E+09</td><td colspan="1" rowspan="1">1.08E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yeast.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicansATCC 10231</td><td colspan="1" rowspan="1">9.05E+038.00E+044.65E+03</td><td colspan="1" rowspan="1">3.12E+04</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.80E+041.03E+051.00E+05</td><td colspan="1" rowspan="1">9.70E+04</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrataATCC 15545</td><td colspan="1" rowspan="1">1.26E+061.11E+06</td><td colspan="1" rowspan="1">1.45E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.47E+072.65E+07</td><td colspan="1" rowspan="1">2.01E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=2 colspan=1>Species/Isolate(s) Tested</td><td rowspan=1 colspan=3>At Positivity</td><td rowspan=1 colspan=3>8 Hours After Positivity</td></tr><tr><td rowspan=1 colspan=1>Per Bottle(CFU/mL)</td><td rowspan=1 colspan=1>Mean&quot;(CFU/mL)</td><td rowspan=1 colspan=1># Detected/Total(% Detected)</td><td rowspan=1 colspan=1>:Per Bottle(CFU/mL)</td><td rowspan=1 colspan=1>Mean(CFU/mL)</td><td rowspan=1 colspan=1># Detected/Fotal(% Detected)</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1.97E+06</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.91E+07</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida kruseiATCC 90878</td><td rowspan=1 colspan=1>5.65E+062.47E+066.35E+06</td><td rowspan=1 colspan=1>4.82E+06</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>2.68E+073.55E+073.25E+07</td><td rowspan=1 colspan=1>3.16E+07</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosisATCC 90875</td><td rowspan=1 colspan=1>2.56E+063.60E+063.20E+06</td><td rowspan=1 colspan=1>3.12E+06</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>6.70E+073.80E+075.55E+07</td><td rowspan=1 colspan=1>5.35E+07</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalisATCC 66029</td><td rowspan=1 colspan=1>1.50E+067.45E+056.65E+05</td><td rowspan=1 colspan=1>9.70E+05</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>1.10E+072.04E+079.45E+06</td><td rowspan=1 colspan=1>1.36E+07</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Correct Detection(Organism and AntimicrobialResistance Genes)</td><td rowspan=1 colspan=2>At Positivity:</td><td rowspan=1 colspan=1>270/270(100%)</td><td rowspan=1 colspan=2>8 Hours After Positivity:</td><td rowspan=1 colspan=1>270/270(100%)</td></tr></table>

and Candida krusei) were observed in a single run (1/540; $0 . 2 \% )$ The correct results were obtained when the sample was retested.

# Inclusivity

Analytical reactivity (inclusivity) of the BCID Panel was evaluated in a study that assessed the ability of the BCID Panel to detect a diverse collection of 303 isolates of genetically, phenotypically, and geographically diverse bacteria and yeast, and to properly indicate the presence of four different antimicrobial resistance genes.

Each isolate was initially tested in blood culture matrix at a concentration consistent with the levels of organism enumerated from blood cultures at the time of positivity (see Growth and Detection section above). If the expected result was obtained at the initial test level, no further testing was performed. If an isolate was not detected initially, additional testing was performed at 10-100 fold higher concentrations. If detected at the higher concentration(s), the species/isolate is indicated as detected with reduced sensitivity and the concentration of organism that was detected is indicated. If not detected at the highest concentration, the isolate is listed as not detected by the FilmArray BCID Panel. Results are provided below for each FilmArray BCID Panel test result.

When possible, in silico analysis of sequence data was used to make predictions of assay reactivity for less common species that may be detected by the FilmArray BCID Panel but were not tested.

# Gram-Positive Bacteria Enterococcus

Table 23. Enterococcus Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Enterococcus Detected[~1x108 CFU/mL]</td><td rowspan=1 colspan=2>Enterococcus Detected withReduced Sensitivity[-1x10 CFU/mL]</td><td rowspan=1 colspan=2>EnterococcusNot Detected</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>ATCC 49463</td><td rowspan=1 colspan=1>Enterococcussaccharolyticus</td><td rowspan=1 colspan=1>ATCC 43076</td><td rowspan=1 colspan=1>Enterococcuspseudoavium</td><td rowspan=1 colspan=1>ATCC 49372</td></tr><tr><td rowspan=1 colspan=1>Enterococcuscasseliflavus</td><td rowspan=1 colspan=1>ATCC 700668</td><td rowspan=1 colspan=1>Enterococcus dispar</td><td rowspan=1 colspan=1>ATCC 51266</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>ATCC 49427</td></tr><tr><td rowspan=1 colspan=1>Enterococcuscecorum</td><td rowspan=1 colspan=1>ATCC 43198</td><td rowspan=1 colspan=2>EMPY</td><td rowspan=1 colspan=2></td></tr></table>

<table><tr><td rowspan=1 colspan=2>Enterococcus Detected[~1x108 CFU/mL]</td><td rowspan=1 colspan=1>Enterococcus Detected with^}$ rReduced Sensitivity[~1x102 CFU/mL]</td><td rowspan=1 colspan=1>EnterococcusNot Detected</td></tr><tr><td rowspan=1 colspan=1>Enterococcusdurans</td><td rowspan=1 colspan=1>ATCC 11576</td><td rowspan=18 colspan=1></td><td rowspan=18 colspan=1></td></tr><tr><td rowspan=6 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC 49532</td></tr><tr><td rowspan=1 colspan=1>ATCC 49533</td></tr><tr><td rowspan=1 colspan=1>JMI 12536</td></tr><tr><td rowspan=1 colspan=1>ATCC 51299</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td></tr><tr><td rowspan=1 colspan=1>JMI 368</td></tr><tr><td rowspan=6 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC 27270</td></tr><tr><td rowspan=1 colspan=1>ATCC 35667</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2127</td></tr><tr><td rowspan=1 colspan=1>JMI 536</td></tr><tr><td rowspan=1 colspan=1>ATCC 700221</td></tr><tr><td rowspan=1 colspan=1>JMI 475</td></tr><tr><td rowspan=1 colspan=1>Enterococcusflavescens</td><td rowspan=1 colspan=1>ATCC 49996</td></tr><tr><td rowspan=1 colspan=1>Enterococcusgallinarum</td><td rowspan=1 colspan=1>ATCC 49608</td></tr><tr><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>ATCC 8043</td></tr><tr><td rowspan=1 colspan=1>Enterococcusmalodoratus</td><td rowspan=1 colspan=1>ATCC 43197</td></tr><tr><td rowspan=1 colspan=1>Enterococcusmundtii</td><td rowspan=1 colspan=1>ATCC 43187</td></tr></table>

a Not detected at the highest test concentrations \~1x10"-1x10 CFU/mL.

# Listeria monocytogenes

Table 24. Listeria monocytogenes Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Listeria monocytogenes Detected&quot;</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1/2a</td><td rowspan=1 colspan=1>FSL-C1-056</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1/2a</td><td rowspan=1 colspan=1>FSL-J2-020b</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1/2b</td><td rowspan=1 colspan=1>FSL-J2-064b</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1/2b</td><td rowspan=1 colspan=1>HUM-2009042206</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>ATCC 43256</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>ATCC 13932</td></tr></table>

" Estimated concentration in a positive blood culture is \~5x10\* CFU/mL. b lsolates obtained from Cornell University. Isolates obtained from the Colorado Department of Public Health (CDPH).

# Staphylococcus (including Staphylococcus aureus)

Table 25. Staphylococcus aureus Inclusivity Results   

<table><tr><td colspan="4">Staphylococcus/Staphylococcus aureus Detecteda</td></tr><tr><td>Species</td><td>Isolate ID</td><td>Strain Information</td><td>&#x27;PFGE Type</td></tr><tr><td>Methicillin-sensitive S. aureus (MSSA)</td><td></td><td></td><td></td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=1 colspan=4>Staphylococcus/Staphylococcus aureus Detected    6A8</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td><td rowspan=1 colspan=1>PFGE Type</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1749</td><td rowspan=1 colspan=1>96:308</td><td rowspan=1 colspan=1>USA 900</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1759</td><td rowspan=1 colspan=1>N7129</td><td rowspan=1 colspan=1>USA 900</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1765</td><td rowspan=1 colspan=1>102-04</td><td rowspan=1 colspan=1>USA 1200</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 12600</td><td rowspan=1 colspan=1>NCTC 8532 Type strain</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 11632</td><td rowspan=1 colspan=1>S13</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2419</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2420</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2421</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1060728</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Anti</td><td rowspan=1 colspan=1>n/a^{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Lem8</td><td rowspan=1 colspan=1>$n/^$</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>MAL8134</td><td rowspan=1 colspan=1>$n/a^{}$</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>MAQ</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Per2</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>RAR</td><td rowspan=1 colspan=1>n/a^{}$</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>S313</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Sal3</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Ver2</td><td rowspan=1 colspan=1>n/a{</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 10832</td><td rowspan=1 colspan=1>Wood 46</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylocccus aureus ssp.a</td><td rowspan=1 colspan=1>ATCC 14154</td><td rowspan=1 colspan=1>Rose</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 25923</td><td rowspan=1 colspan=1>Seattle/1945</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>_Borderline Oxacillin-resistant S. aureus</td><td rowspan=1 colspan=1>(BORSA)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1> SUNI</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>SUN2^d}$</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>SUN3}$</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>SUN4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>SUN5d</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>SUNG{$</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Methicillin-resistant S. aureus (MRSA)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-38</td><td rowspan=1 colspan=1>E2125 Denmark</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>F-182 Kansas</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 700698</td><td rowspan=1 colspan=1>Mu3 Japan/1996</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-1720</td><td rowspan=1 colspan=1>MRSA252 UK</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-39</td><td rowspan=1 colspan=1>HUSA304 Hungary/1993</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS705</td><td rowspan=1 colspan=1>NY-12 New York/2005</td><td rowspan=1 colspan=1>USA 100</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS701</td><td rowspan=1 colspan=1>MN-082 Minn/2006</td><td rowspan=1 colspan=1>USA 200</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-1717</td><td rowspan=1 colspan=1>TCH1516 Texas</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS703</td><td rowspan=1 colspan=1>MN-095 Minn/2006</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS683</td><td rowspan=1 colspan=1>GA-298 Georgia/2005</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS662</td><td rowspan=1 colspan=1>CO-34 Colorado/2005</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS707</td><td rowspan=1 colspan=1>NY-155 New York/2005</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1707</td><td rowspan=1 colspan=1>MW2 N. Dakota/1998</td><td rowspan=1 colspan=1>USA 400</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS691</td><td rowspan=1 colspan=1>GA-62 Georgia/2005</td><td rowspan=1 colspan=1>USA 500</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS648</td><td rowspan=1 colspan=1>CA-347 California/2005</td><td rowspan=1 colspan=1>USA 600</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS689</td><td rowspan=1 colspan=1>GA-442 Georgia/2006</td><td rowspan=1 colspan=1>USA 700</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-42</td><td rowspan=1 colspan=1>HDE288 Portugal/1996</td><td rowspan=1 colspan=1>USA 800</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS668</td><td rowspan=1 colspan=1>CO-72 Colorado/2005</td><td rowspan=1 colspan=1>USA 800</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1747</td><td rowspan=1 colspan=1>94:1013 Vermont/1993</td><td rowspan=1 colspan=1>USA 1000</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS676</td><td rowspan=1 colspan=1>CT-19 Conn/2005</td><td rowspan=1 colspan=1>USA 1000</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS745</td><td rowspan=1 colspan=1>CA-629 California/2006</td><td rowspan=1 colspan=1>USA 1000</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1764</td><td rowspan=1 colspan=1>7031 Alaska</td><td rowspan=1 colspan=1>USA 1100</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1691</td><td rowspan=1 colspan=1>HFH-30137 Michigan/2003</td><td rowspan=1 colspan=1>Not 100-1100</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1700</td><td rowspan=1 colspan=1>HFH-33798 Illinois/2004</td><td rowspan=1 colspan=1>Not 100-1100</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2312</td><td rowspan=1 colspan=1>M10/0061 Ireland/2010</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>M10/0148 Ireland/2010</td><td rowspan=1 colspan=1>CC130</td></tr></table>

Detected at the initial test concentration of 5x10°CFU/mL.

# BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

Table 26. Results of Staphylococcus (non-S. aureus) Inclusivity Testing   

<table><tr><td rowspan=1 colspan=2>Staphylococcus Detected[-5x10° CFU/mL]</td><td rowspan=1 colspan=2>Staphylococcus Detected withReduced Sensitivity[~5x107 CFU/mL]</td><td rowspan=1 colspan=2>StaphylococcusNot Detectedb</td></tr><tr><td rowspan=1 colspan=6>Coagulase-positive staphylococci (non-S.aureus)</td></tr><tr><td rowspan=1 colspan=2>Staphylococcus lutrae     ATCC 700373</td><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>ATCC 29663</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Staphylococcuspseudointermedius</td><td rowspan=1 colspan=1>ATCC 49444</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. coagulans</td><td rowspan=1 colspan=1>ATCC 49545</td></tr><tr><td rowspan=1 colspan=6>Coagulase-negative staphylococci (CoNS)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus caprae</td><td rowspan=1 colspan=1>ATCC 51548</td><td rowspan=1 colspan=1>Staphylococcus capitissubsp. capitis</td><td rowspan=1 colspan=1>ATCC 27842</td><td rowspan=1 colspan=1>Staphylococcus auricularis</td><td rowspan=1 colspan=1>Clinical isolate</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus cohnii</td><td rowspan=1 colspan=1>ATCC 29972</td><td rowspan=1 colspan=1>Staphylococcus pasteuri</td><td rowspan=1 colspan=1>ATCC 51127</td><td rowspan=1 colspan=1>Staphylococcus carnosus</td><td rowspan=1 colspan=1>ATCC 51365</td></tr><tr><td rowspan=6 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>ATCC 12228</td><td rowspan=1 colspan=1>Staphylococcussaprophyticus</td><td rowspan=1 colspan=1>ATCC 15305</td><td rowspan=1 colspan=1>Staphylococcus lentus</td><td rowspan=1 colspan=1>ATCC 700403</td></tr><tr><td rowspan=1 colspan=1>ATCC 29886</td><td rowspan=1 colspan=1>Staphylococcussimulans</td><td rowspan=1 colspan=1>Clinicalisolates</td><td rowspan=1 colspan=1>Staphylococcuspeltenkoferi</td><td rowspan=1 colspan=1>5 clinicalisolates</td></tr><tr><td rowspan=1 colspan=1>ATCC 55133</td><td rowspan=1 colspan=1>Staphylococcuswarneri</td><td rowspan=1 colspan=1>ATCC 25614</td><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. schleiferi</td><td rowspan=1 colspan=1>ATCC 43808</td></tr><tr><td rowspan=1 colspan=1>ATCC 29887</td><td rowspan=8 colspan=2></td><td rowspan=1 colspan=1>Staphylococcus sciuri</td><td rowspan=1 colspan=1>ATCC 29060</td></tr><tr><td rowspan=1 colspan=1>ATCC 51625</td><td rowspan=7 colspan=2></td></tr><tr><td rowspan=1 colspan=1>ATCC 35984</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus equorum</td><td rowspan=1 colspan=1>ATCC 43958</td></tr><tr><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>ATCC 29968</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hominisssp. hominis</td><td rowspan=1 colspan=1>ATCC 25615</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuslugdlunensis</td><td rowspan=1 colspan=1>ATCC 43809</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusxylosus</td><td rowspan=1 colspan=1>ATCC 29966</td></tr></table>

b Initial test concentration was $\mathbf { 5 } \mathbf { x } | 0 ^ { \mathbf { 5 } } \mathbf { C } \mathbf { F } \mathbf { U } / \mathbf { m } | .$ . Isolates obtained from University of Rennes, France. S UnT Tested as a seeded blood culture at the time of positivity.   
a  S e . b Not detected when tested at a concentration of ≥5x10° CFU/mL. .

Based on inclusivity testing results for Staphylococci and in silico analysis of available sequences, the following predictions of reactivity are provided for many less common CoNS species that were not tested.

Please note that performance of the FilmArray BCID Panel for these organisms has not been established.

Table 27. In Silico Predictions of Staphylococcus Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicted</td><td rowspan=1 colspan=1>Detection Predicted WithReduced Sensitivityb</td><td rowspan=1 colspan=1>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum</td><td rowspan=1 colspan=1>Staphylococcus microti</td><td rowspan=1 colspan=1>Staphylococcus arlettae</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus kloosii</td><td rowspan=1 colspan=1>Staphylococcus simiae</td><td rowspan=1 colspan=1>Staphylococcus chromogenes</td></tr><tr><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus succimus</td><td rowspan=1 colspan=1>Staphylococcus condimenti</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus fleurettii</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus piscifermentans</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus pulvereri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus rostri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus saccharolyticus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus vitulinus</td></tr></table>

Predicted result of Staphylococcus Detected when present in a blood culture sample at a concentration of $\geq 5 \mathbf { x } \mathbf { l } 0 ^ { 6 } \mathbf { C l } { \mathrm { : } } \mathbf { U } / \mathbf { m L }$ . ceresult  Stapyloocus Detectd when present ina blood cultue sampleat a coneratin f $\geq 5 \mathrm { x } 1 0 ^ { 7 }$ CFU/mL. Predicted result of Staphylococcus Not Detected at relevant concentrations.

# Streptococcus (including S. agalactiae, S. pneumoniae, and S. pyogenes)

. Table 28. Streptococcus Inclusivity Results   

<table><tr><td colspan="3" rowspan="1">Streptococcus Detected</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 19615</td><td colspan="1" rowspan="5">Group A (Pyogenic group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">PCMC 20100107C102</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 49399</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12344</td></tr><tr><td colspan="1" rowspan="1">Streplococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813Type strain - Serotypela/c</td><td colspan="1" rowspan="5">Group B (Pyogenic group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">PCMC 20100107C103Untyped clinical isolate</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12403Type III</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC BAA-611Serotype V</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">NCTC 8017Unknown serotype</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae ssp. equisimilis</td><td colspan="1" rowspan="1">ATCC 12388</td><td colspan="1" rowspan="1">Group C/G (Pyogenic group)</td></tr><tr><td colspan="1" rowspan="1">Streplococcus bovis</td><td colspan="1" rowspan="1">ATCC 33317</td><td colspan="1" rowspan="2">Group D (Bovis group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equinis</td><td colspan="1" rowspan="1">ATCC 9812</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">ATCC 25175</td><td colspan="1" rowspan="1">Group E (Mutans group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 33397</td><td colspan="1" rowspan="3">Group F (Anginosus group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">ATCC 10558</td><td colspan="1" rowspan="9">Mitis group</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">ATCC 31412</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">ATCC 10556</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 15914</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">ATCC 10557</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-960</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-255Strain R6 (no capsule)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 700672Serotype 14</td></tr><tr><td colspan="1" rowspan="1">Sireplveoccus pmeumonite</td><td colspan="1" rowspan="1">ATGC BAA 334</td></tr><tr><td colspan="3" rowspan="1">Streptococcus Detected®                                        .</td></tr><tr><td colspan="1" rowspan="1">Species                            </td><td colspan="1" rowspan="1">IsolateID</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td></td><td colspan="1" rowspan="1">Serotype 4</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 700673Serotype 19A</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-341Serotype 5</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">Salivarius group</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">ATCC BAA-2069</td><td colspan="1" rowspan="1">Uncertain grouping</td></tr></table>

Detected at the initial tes1 concentration of \~1 x10\* CFU/mL.

Table 29. Streptococcus agalactiae Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus agalactiae (Group B) Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813Type strain - Serotype 1a/c</td><td rowspan=5 colspan=1>Group B (Pyogenic group)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>PCMC 20100107CI03Untyped clinical isolate</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 12403Type II</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC BAA-611Serotype V</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>NCTC 8017Unknown serotype</td></tr></table>

' Detected at the initial tes1 concentration of \~1x10 CFU/mL.

Table 30. Streptococcus pneumoniae Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus pneumonine Detected,b</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-255Strain R6 (no capsule)</td><td rowspan=5 colspan=1>Mitis group</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 700672Serotype 14</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-334Serotype 4</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 700673Serotype 19A</td></tr><tr><td rowspan=1 colspan=1>Sireptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-341Serotype 5</td></tr></table>

Detected at the initial test concentration of \~1x10° CFU/mL. reduced sensitivity compared to other serotypes.

Table 31. Streptococcus pyogenes Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus pyogenes (Group A) Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Sireptococcus pvogenes</td><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=5 colspan=1>Group A (Pyogenic group)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>PCMC 20100107C102</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 49399</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12344</td></tr><tr><td rowspan=1 colspan=1>Sireptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12384</td></tr></table>

Detected at the initial test concentration of \~1x10® CFU/mL.

Based on results of inclusivity testing and in silico analysis of available sequences, the following predictions of reactivity are provided for less common Streptococcus species that were not tested. As shown in Table 32 below, the analysis predicts that many species

will be detected at concentrations expected in positive blood cultures $( 1 0 ^ { 8 } – 1 0 ^ { 9 } \mathrm { C F U / m L } )$ and others (particularly Mutans group species) will likely not be detected due to sequence mismatches with the assay primers.

Please note that performance of the FilmArray BCID Panel for these organisms has not been established.

Table 32. In Silico Predictions of Streptococcus Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicted®</td><td rowspan=1 colspan=1>Detection Predicted with ReducedSensitivity</td><td rowspan=1 colspan=1>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Streptococcus australis</td><td rowspan=1 colspan=1>Streptococcus parauberis</td><td rowspan=1 colspan=1>Streptococcus cricet</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equi</td><td rowspan=9 colspan=1></td><td rowspan=1 colspan=1>Streptococcus downer</td></tr><tr><td rowspan=1 colspan=1>Streptococcus ictaluri</td><td rowspan=1 colspan=1>Streptococcus macacae®</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantis</td><td rowspan=1 colspan=1>Streptococcus porcinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantarius</td><td rowspan=1 colspan=1>Streptococcus urialis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pasteurianus</td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus perois</td></tr><tr><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus thermophilus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus vestibularis</td></tr></table>

u  D h  l ul ${ \geq } 1 \mathbf { x } 1 0 ^ { 9 }$ CFU/mL. Mutans group streptcocci.

# Gram-Negative Bacteria Acinetobacter baumannii

Table 33. Acinetobacter baumannii Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Acinetobacter baumannii Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC 9955</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC BAA-1605</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC 17961</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC 19003</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC BAA-2093</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC 15308</td></tr></table>

Detected at the initial test concentration of \~1x10° CFU/mL

Enterobacteriaceae (including Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens)

Table 34. Results of Enterobacteriaceae Inclusivity Testing   

<table><tr><td colspan="2" rowspan="1">Enterobacteriaceae Detected[~5×10' CFU/mL or 1×10° CFU/mL]</td><td colspan="2" rowspan="1">Enterobacteriaceae Detectedwith Reduced Sensitivity[~5×108-1×10° CFU/mL]</td><td colspan="2" rowspan="1">EnterobacteriaceaeNot Detecteda</td></tr><tr><td colspan="1" rowspan="1">Cedeceae davisiae</td><td colspan="1" rowspan="1">ATCC 43023</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Morganella morganiisubsp. morganii</td><td colspan="1" rowspan="1">ATCC 25829</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">Enterobacter gergoviae</td><td colspan="1" rowspan="1">ATCC 33028</td><td colspan="1" rowspan="1">Pantoea (Enterobacter)agglomerans</td><td colspan="1" rowspan="1">ATCC 27155</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 29223</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td><td colspan="1" rowspan="1">Providencia (Proteus)acalifaciens</td><td colspan="1" rowspan="1">ATCC 51902</td></tr><tr><td colspan="1" rowspan="1">Cronobacter muyjensii</td><td colspan="1" rowspan="1">ATCC 51329</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">SGSC 3041</td><td colspan="1" rowspan="1">Providencia (Proteus)rettgeri</td><td colspan="1" rowspan="1">ATCC 9250</td></tr><tr><td colspan="2" rowspan="2">Enterobacteriaceae Detected[~5×107 CFU/mL or 1×10° CFU/mL]</td><td colspan="2" rowspan="2">Enterobacteriaceae Detected2  with Reduced Sensitivity[5×10^°-1×10° CFU/mL]</td><td colspan="2" rowspan="2">EnterobacteriaceneNot Detected"</td></tr><tr></tr><tr><td colspan="1" rowspan="1">Cronobacter (Enterobacter)sakazakii</td><td colspan="1" rowspan="1">ATCC 29544</td><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td><td colspan="1" rowspan="1">Providencia stuarti</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">ATCC 33077</td><td colspan="1" rowspan="1">Rahnella aquatilis</td><td colspan="1" rowspan="1">ATCC 33071</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 29751</td><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC 27593</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td><td colspan="2" rowspan="32"></td><td colspan="1" rowspan="1">Tatumella ptyseos</td><td colspan="1" rowspan="1">ATCC 33301</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 51816</td><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC 183</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacue</td><td colspan="1" rowspan="1">9 isolates²</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">ATCC 6025</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 49162</td><td colspan="2" rowspan="29"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">ATCC BAA-260d</td></tr><tr><td colspan="1" rowspan="1">Enterobacter nimipressuralis</td><td colspan="1" rowspan="1">ATCC 9912d</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5 isolates</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermanii</td><td colspan="1" rowspan="1">ATCC 33650</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">11 isolates'</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">10 isolates</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">ATCC BAA-830</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">ATCC 33433</td></tr><tr><td colspan="1" rowspan="1">Kluyvera (Enterobacter)intermedius</td><td colspan="1" rowspan="1">ATCC 33110</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">ATCC 23216</td></tr><tr><td colspan="1" rowspan="1">Proteus species</td><td colspan="1" rowspan="1">10 isolatesh</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">ATCC 31898</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">ATCC 31900</td></tr><tr><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">ATCC 33257</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica-cholerasius</td><td colspan="1" rowspan="1">ATCC 10708</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica-heidelberg</td><td colspan="1" rowspan="1">ATCC 8326</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica-paratyphi</td><td colspan="1" rowspan="1">SGSC 3222</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica-typhimurium</td><td colspan="1" rowspan="1">ATCC 13311</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">6 isolates'</td></tr><tr><td colspan="1" rowspan="1">Serratia entomophila</td><td colspan="1" rowspan="1">ATCC 43705</td></tr><tr><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC 33105</td></tr><tr><td colspan="1" rowspan="1">Shigella boydif</td><td colspan="1" rowspan="1">ATCC 8700</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">PHM-2004008089</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">ATCC 12022</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 11060</td></tr><tr><td colspan="1" rowspan="1">Yokenella regensburgei</td><td colspan="1" rowspan="1">ATCC 35313</td></tr></table>

Not Detected at the highest test concentration of 1×10-1 ×10l®CFU/mL. No h       C See Enterobacter cloacae complex table. Te   uc $0 . 6 3 \mu \mathrm { g } / \mathrm { m L }$ (equivalent to ${ \bf - } 1 . 0 \times 1 0 ^ { 8 }$ CFU/mL).

Based on results of inclusivity testing and in silico analysis of available sequences, the following predictions of reactivity are provided for less common Enterobacteriaceae that were not tested.

Please note that performance of the FilmArray BCID Panel for these organisms has not been established.

Table 35. In silico Predictions of Enterobacteriaceae Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicted withReduced Sensitivity</td><td rowspan=1 colspan=1>Detection Not Predicted</td><td rowspan=1 colspan=1>Unknown Reactivityb</td></tr><tr><td rowspan=1 colspan=1>Brenneria spp.</td><td rowspan=1 colspan=1>Photorhabdus spp.</td><td rowspan=1 colspan=1>Buttiauxella spp.</td></tr><tr><td rowspan=1 colspan=1>Dickeya spp.</td><td rowspan=1 colspan=1>Serratia grimesii</td><td rowspan=1 colspan=1>Ewingella americana</td></tr><tr><td rowspan=1 colspan=1>Ervinia spp.</td><td rowspan=1 colspan=1>Serratia proteamaculans</td><td rowspan=1 colspan=1>Leminorella spp.</td></tr><tr><td rowspan=1 colspan=1>Pectobacterium spp.</td><td rowspan=1 colspan=1>Xenorhabdus spp.</td><td rowspan=1 colspan=1>Moellerella spp.</td></tr><tr><td rowspan=1 colspan=1>&#x27;</td><td rowspan=1 colspan=1>Yersinia spp.</td><td rowspan=1 colspan=1></td></tr></table>

u   bl u CFU/mL b Sequence data not available for in silico reactivity predictions.

# Enterobacter cloacae complex

Table 36. Summary of Enterobacter cloacae complex Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Enterobacter cloacae complex Detected[~1×10° CFU/mL] &#x27;</td><td rowspan=1 colspan=2>Enterobacter cloacae complexNot Detected</td></tr><tr><td rowspan=1 colspan=1>Enterobacter asburiae</td><td rowspan=1 colspan=1>ATCC 35953</td><td rowspan=1 colspan=1>Enterobacter nimipressuralish</td><td rowspan=1 colspan=1>ATCC 9912</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp. cloacue</td><td rowspan=1 colspan=1>ATCC BAA-1143</td><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>ATCC BAA-260</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp. cloacue</td><td rowspan=1 colspan=1>ATCC 13047</td><td rowspan=5 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp. cloacae</td><td rowspan=1 colspan=1>NCTC 10005</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp. cloacae</td><td rowspan=1 colspan=1>ATCC 49141</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. dissolvensh</td><td rowspan=1 colspan=1>ATCC 23373</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>ATCC 49162</td></tr></table>

No Decih $\overline { { 1 \times 1 0 ^ { 1 0 } \mathrm { C F U / m l } } } ,$ $0 . 6 3 \mu \mathrm { g } / \mathrm { m L }$ (equivalent to $\mathord { \sim } 1 \times 1 0 ^ { \times }$ CFU/mL). Detected by Enterobactaca assay.

# Escherichia coli

Table 37. Escherichia coli Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Escherichia coli Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 43888</td><td rowspan=1 colspan=1>CDC B6914-MS1serotype 0157:H7  -</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 49105</td><td rowspan=1 colspan=1>7482-1-1serotype 015</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 25922   -</td><td rowspan=1 colspan=1>FDA-Seattle1946</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>H10407serotype O78:H11</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC BAA-201</td><td rowspan=1 colspan=1>Produces ESBL TEM-3</td></tr></table>

'Detected at the initial test concentration of $\overline { { 5 \times 1 0 } } ^ { \tau }$ CFU/mL.

# Klebsiella oxytoca

Table 38. Klebsiella oxytoca Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Klebsiella oxytoca Detected&quot;</td><td rowspan=1 colspan=3>Klebsiella oxytoca Not Detectedb</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 13182</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Klebsiella oxytocab,c</td><td rowspan=1 colspan=1>JMI 10678</td><td rowspan=1 colspan=1>MY/2011</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 49131</td><td rowspan=1 colspan=1>n/a</td><td rowspan=9 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 700324</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 43086</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxyloca</td><td rowspan=1 colspan=1>ATCC 8724</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 14611</td><td rowspan=1 colspan=1>AR/2011</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI12707</td><td rowspan=1 colspan=1>MA/2011</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>AR/2004</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>NY/2003</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Detcd  e nal e contion $\overline { { 5 \times 1 0 ^ { 7 } ~ } } \overline { { \mathrm { C F U / m L } } }$ . $5 \times 1 0 ^ { 7 }$ CFU/mL but Not Deeed   y highest test concentration of $\mathsf { l } \times \mathsf { l } 0 ^ { \mathsf { I 0 } }$ CFU/mL. Sequence analysis confirmed this isolate as a variant $\kappa$ oxa that will not be detected by the FilmArray BCID Panel Koxyoca assay.

# Klebsiella pneumoniae

Table 39. Klebsiella pneumoniae Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Klebsiella pneumoniae Detected</td></tr><tr><td rowspan=1 colspan=1>Species                                 7</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniue</td><td rowspan=1 colspan=1>ATCC BAA-1706</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. pneumoniae</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. ozaenae       †</td><td rowspan=1 colspan=1>ATCC 11296</td><td rowspan=1 colspan=1>NCTC 5050</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. rhinoscleromatis</td><td rowspan=1 colspan=1>ATCC 13884</td><td rowspan=1 colspan=1>NCTC 5046 Type strain</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JM1 328</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 8091</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 438</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCC BAA-830</td><td rowspan=1 colspan=1>F2R9/ 2001 Type strain</td></tr></table>

Delected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { \times } \mathrm { C F U / m L } } }$ . c e     o $K$ peumonide variant 342 and Klebsiella varicola have been recovered Proteus

Table 40. Proteus Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Proteus Detected&quot;</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID      .*</td></tr><tr><td rowspan=5 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>ATCC 29906</td></tr><tr><td rowspan=1 colspan=1>JMI 10793</td></tr><tr><td rowspan=1 colspan=1>ATCC 25933</td></tr><tr><td rowspan=1 colspan=1>ATCC 33583</td></tr><tr><td rowspan=1 colspan=1>ATCC7002</td></tr><tr><td rowspan=1 colspan=1>Proteus hauseri</td><td rowspan=1 colspan=1>ATCc 13315</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td></td><td rowspan=1 colspan=1>ATCC 700826</td></tr><tr><td rowspan=1 colspan=1>Proteus penneri</td><td rowspan=1 colspan=1>ATCC 33519</td></tr><tr><td rowspan=2 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>ATCC 33420</td></tr><tr><td rowspan=1 colspan=1>ATCC 27973</td></tr></table>

' Detected at the initial test concentration of 1 ×10' CFU/mL.

Serratia marcescens

Table 41. Serratia marcescens Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Serratia marcescens Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID  .</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 13880</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 14756</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 27137</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 43297</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>CT/2009</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>JMI 8089</td><td rowspan=1 colspan=1>TX/2004</td></tr></table>

a Delected at the initial test concentration of $\frac { 1 } { 1 \times 1 0 ^ { 8 } C F U / m L }$

# Haemophilus influenzae

Table 42. Haemophilus influenzae Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Haemophilus influenzae Detected</td></tr><tr><td rowspan=1 colspan=1>Species                                     i&#x27;</td><td rowspan=1 colspan=1>Isotate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 33929</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 51907</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae ssp. aegyptus</td><td rowspan=1 colspan=1>ATCC 1 1116</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 9006</td><td rowspan=1 colspan=1>Type a</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 31512</td><td rowspan=1 colspan=1>Type b</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 10211</td><td rowspan=1 colspan=1>Type b</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 49699</td><td rowspan=1 colspan=1>Type c</td></tr><tr><td rowspan=1 colspan=1>Huemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 9008</td><td rowspan=1 colspan=1>Type d</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 8142</td><td rowspan=1 colspan=1>Type e</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 700223</td><td rowspan=1 colspan=1>Type f</td></tr></table>

at the time of positivity is estimated to be $\sim 1 \times 1 0 ^ { \mathsf { \times } }$ CFU/mL.

Table 43. Neisseria meningitidis Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>Neisseria meningitidis Detected&quot;</td><td rowspan=1 colspan=3>Neisseria meningitidis Not Detectedb            :</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Serogroup</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Serogroup</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 43744</td><td rowspan=1 colspan=1>W135</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13077</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13090</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13102</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>None</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=1 colspan=3>Neisseria meningitidis Detected</td><td rowspan=1 colspan=3>Neisseria meningitidis Not Detectedb</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID 1</td><td rowspan=1 colspan=1>Serogroup</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Serogroup</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(unencapsulated)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13113</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 35561</td><td rowspan=1 colspan=1>Y</td><td></td><td></td><td></td></tr></table>

Deected in a seeded blood culture tested within  hour of positivity estimaled concentration $\mathrm { \overline { { \sim 1 \times 1 0 ^ { 8 } } } }$ CFU/mL). Not Detected in a seeded blood culture tested 1- hours after positivity. detected by the BCID Panel. A fo cnl lae i avnt  ene as este nd not dete   cn equin o $2 . 5 \times 1 0 ^ { 9 }$ CFU/mL. DNA obtained from University of Lausanne, Institute of Microbiology, Switzerland.

# Pseudomonas aeruginosa

Table 44. Pseudomonas aeruginosa Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Pseudomonas aeruginosa Detected</td></tr><tr><td rowspan=1 colspan=1>Species                           :</td><td rowspan=1 colspan=1>Isolate ID</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 27853</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 10145</td></tr><tr><td rowspan=1 colspan=1>Pseudononas aeruginosa</td><td rowspan=1 colspan=1>ATCC 19429</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 25619</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC BAA-1744</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 35554</td></tr></table>

' Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 8 } } }$ CFU/mL.

# Yeast

# Candida albicans

Table 45. Results of Candida albicans Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Candida albicans Detected&quot;</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID          .</td><td rowspan=1 colspan=1>Strain Info </td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=1>Serotype A - 3147</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC MYA-427</td><td rowspan=1 colspan=1>A39 [DUMC 136.97]</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC MYA-2876</td><td rowspan=1 colspan=1>SC5314</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 11651</td><td rowspan=1 colspan=1>171D</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 22972</td><td rowspan=1 colspan=1>M 97</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 90028</td><td rowspan=1 colspan=1>NCCLS 11</td></tr></table>

'Detected at the initial test concentration of $\frac { 1 } { 1 \times 1 0 ^ { + } ( C F U / m L }$

Table 4. Results of Candida glabrata Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Candida glabrata Detected®</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15545</td><td rowspan=1 colspan=1>NRRL YB-4025</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 32554</td><td rowspan=1 colspan=1>26247-1</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 2001</td><td rowspan=1 colspan=1>CBS138</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15126</td><td rowspan=1 colspan=1>CBS15126</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC MYA-2950</td><td rowspan=1 colspan=1>n/a</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

# Candida krusei

Table 47. Results of Candida krusei Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Candida krusei Detected</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID          i</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 90878</td><td rowspan=1 colspan=1>B74</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 201748</td><td rowspan=1 colspan=1>89-08-008</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 14243</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Candida krusei/Issatchenkiaorientalis</td><td rowspan=1 colspan=1>ATCC 28870</td><td rowspan=1 colspan=1>CBS 2052</td></tr><tr><td rowspan=1 colspan=1>Issatchenkia orientalis</td><td rowspan=1 colspan=1>ATCC 6258</td><td rowspan=1 colspan=1>NRRL Y-413</td></tr></table>

" Detected at the initial test concentration of $1 \times 1 0 ^ { 6 } C F U / \mathfrak { m L }$ . l  v

# Candida parapsilosis

Table 48. Results of Candida parapsilosis Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Candida parapsilosis Detected</td><td rowspan=1 colspan=3>Candida purapsilosis Detected with Reduced Sensitivity</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 90875</td><td rowspan=1 colspan=1>B78</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 96142</td><td rowspan=1 colspan=1>MCO462[UTHSC R-648]</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 34136</td><td rowspan=1 colspan=1>ST-89</td><td rowspan=3 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 96138</td><td rowspan=1 colspan=1>MCO433</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 22019</td><td rowspan=1 colspan=1>CBS604</td></tr></table>

a Detected at the initial test concentration of 1 × t 0" CFU/mL. Detected at a test concentration of $\mathsf { \Omega } ^ { 1 \times 1 0 ^ { 7 } }$ CFU/mL.

# Candida tropicalis

Table 49. Results of Candida tropicalis Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Candida tropicalis Detected</td></tr><tr><td rowspan=1 colspan=1>.Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 66029</td><td rowspan=1 colspan=1>AmMS 227</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 750</td><td rowspan=1 colspan=1>Type Strain</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 90874</td><td rowspan=1 colspan=1>B79</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC MYA-2734</td><td rowspan=1 colspan=1>508-12.1</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalish</td><td rowspan=1 colspan=1>ATCC 201380</td><td rowspan=1 colspan=1>API 90 01 105 (Vitek QC)</td></tr></table>

Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 3 } \mathrm { C F U / m L } } }$ . Target concentration was ${ \sf 5 } { \times } { \sf 1 0 } ^ { \sf 5 }$ CFU/mL, final test concentration was $1 \times 1 0 ^ { 6 }$ CFU/mL $( 2 \times )$ .

# Antimicrobial Resistance Genes mecA

Table 50. mecA Inclusivity Results   

<table><tr><td colspan="4">mecA Detecteda,b</td></tr><tr><td>Species</td><td>IsolateID</td><td>Strain Information</td><td>SCCmec Type</td></tr><tr><td colspan="4">Methicillin-sensitive S. aureus (MSSA) with SCCmec cassette (mecA positive)</td></tr><tr><td></td><td>ATCC BAA-2419</td><td>Mass/2010</td><td></td></tr><tr><td>Staphylococcus aureus</td><td></td><td></td><td>1I</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=1 colspan=4>mecA Detecteda,b</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>.Strain Information</td><td rowspan=1 colspan=1>SCCmecType</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2420</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2421</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Methicillin-resistant S. epidermidis (MRSE) (mecA positive)</td><td rowspan=1 colspan=1>RSE) (mecA positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 29887</td><td rowspan=1 colspan=1>255-01B</td><td rowspan=3 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 51625</td><td rowspan=1 colspan=1>CCF 15990</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>RP62A</td></tr><tr><td rowspan=1 colspan=1>Methicillin-resistant S. aureus (MRSA) (mecA positive)</td><td rowspan=1 colspan=1>(mecApositive)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-38</td><td rowspan=1 colspan=1>E2125 Denmark</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>F-182 Kansas</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC 700698</td><td rowspan=1 colspan=1>Mu3 Japan/1996</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-1720</td><td rowspan=1 colspan=1>MRSA252 UK</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS705</td><td rowspan=1 colspan=1>NY-12 New York/2005</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS701</td><td rowspan=1 colspan=1>MN-082 Minn/2006</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS648</td><td rowspan=1 colspan=1>CA-347 California/2005</td><td rowspan=1 colspan=1>1I</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-39</td><td rowspan=1 colspan=1>HUSA304 Hungary/1993</td><td rowspan=1 colspan=1>III 3A&amp;5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS703</td><td rowspan=1 colspan=1>MN-095 Minnesota/2006</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS683</td><td rowspan=1 colspan=1>GA-298 Georgia/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS662</td><td rowspan=1 colspan=1>CO-34 Colorado/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS707</td><td rowspan=1 colspan=1>NY-155 New York/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1707</td><td rowspan=1 colspan=1>MW2 N. Dakota/1998</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS691</td><td rowspan=1 colspan=1>GA-62 Georgia/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS689</td><td rowspan=1 colspan=1>GA-442 Georgia/2006</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS668</td><td rowspan=1 colspan=1>CO-72 Colorado/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1747</td><td rowspan=1 colspan=1>94:1013 Vermont/1993</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS676</td><td rowspan=1 colspan=1>CT-19 Conn/2005</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1764</td><td rowspan=1 colspan=1>7031 Alaska</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1691</td><td rowspan=1 colspan=1>HFH-30137 Michigan/2003</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1700</td><td rowspan=1 colspan=1>HFH-33798 I1linois/2004</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-1717</td><td rowspan=1 colspan=1>TCH1516 Texas</td><td rowspan=1 colspan=1>IVa</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NARSA NRS745</td><td rowspan=1 colspan=1>CA-629 California/2006</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus ssp. aureus</td><td rowspan=1 colspan=1>ATCC BAA-42</td><td rowspan=1 colspan=1>HDE288 Portugal/1996</td><td rowspan=1 colspan=1>VI</td></tr><tr><td rowspan=1 colspan=1>Methicillin-resistant S. aureus with mecAlcazsy/mecC variant</td><td rowspan=1 colspan=1>cAct/mecC varia</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2312</td><td rowspan=1 colspan=1>M10/0061 Ireland/2010</td><td rowspan=1 colspan=1>XI</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>M10/0148 Ireland/2010</td><td rowspan=1 colspan=1>XI</td></tr><tr><td rowspan=1 colspan=1>a Detected at the initial test concentration of 5x 1OCFU/ml,</td><td></td><td></td><td></td></tr></table>

Sccu Det n Sapy u Detesul o eore as Initial test concentration was $5 \times 1 0 ^ { 5 }$ CFU/mL.

Table 51. vanA/B Inclusivity Results   

<table><tr><td rowspan=1 colspan=3>vanA/B Detecteda.b</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain.Information</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>JMI 536</td><td rowspan=1 colspan=1>TX/2006</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=1 colspan=1>Connecticut</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>JMI 475</td><td rowspan=1 colspan=1>IN/2003</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanA]</td><td rowspan=1 colspan=1>JMI 12536</td><td rowspan=1 colspan=1>Mass/2002</td></tr></table>

# BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=1 colspan=3>vanA/B Detecteda,b</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate, ID   A.</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>Missouri</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>Missouri/1987</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1>JMI 368</td><td rowspan=1 colspan=1>VA/2003</td></tr></table>

Detected at the initial test concentration of $\frac { 1 } { 1 \times 1 0 ^ { 8 } C F U / m L }$ . En De es l erd

# KPC

Table 52. KPC Inclusivity Results   

<table><tr><td rowspan=1 colspan=4>KPC Detected              4</td></tr><tr><td rowspan=1 colspan=1>Species              F.    1</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>KPC Type</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>BAA-2341</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1101152</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>BAA-2082</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>BAA-2340</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1101362</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxyloca</td><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>AR/2004</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 328</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>Modified Hodge Test Control</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>CT/2009</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>KPC-3</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>KPC-3</td><td rowspan=1 colspan=1>NY/2003</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 766 </td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 8091</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JM1 438</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Delected at the initial test concentration of $5 \times 1 0 ^ { 7 }$ CFU/mL. for K. oxytoca isolates and 1×10\* CFU/mL for K. pneumoniae and S. n  c  o iv $\vec { \cal E }$ coli. than those listed above) were not evaluated.

# Exclusivity

The potential for cross-reactivity with BCID Panel assays was evaluated by testing high concentrations of organism in contrived or seeded blood culture samples. The test concentration was equal to or greater than the level of organism estimated to be in a blood culture sample 8 hours after positivity (approximately $1 0 ^ { 9 } – 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ for bacteria and $1 0 ^ { 7 } – 1 \dot { 0 } ^ { 8 } \mathrm { C F U / m L }$ for yeast), or the highest concentration possible based on the organism stock. The selection of organisms focused on species that may be found in positive blood cultures (clinically relevant) and/or those that are closely related to target organisms (nearest neighbors). Organisms were also selected based on antimicrobial resistance phenotypes and the presence or absence of the antimicrobial resistance genes identified by the BCID Panel. The tested organisms were divided into two categories: onpanel organisms and off-panel organisms.

On-panel organisms were tested to verify that they only react with the appropriate assays on the panel. On-panel exclusivity testing included gram-positive bacteria, gram-negative bacteria and yeast, representing 29 genera and 98 individual species.

Off-panel organisms were expected to have negative test results for all of the assays on the FilmArray BCID Panel (or positive organism results but negative results for the

antimicrobial resistance genes detected by the FilmArray BCID Panel). Off-panel testing included gram-positive bacteria, gram-negative bacteria, yeast, viruses, and Mycoplasmataceae.

Results are presented for all organisms that were tested and received the expected FilmArray BCID Panel test result(s) (no cross-reactivity, Tables 53 - 57), followed by a summary of species or isolates with which cross-reactivity was observed (Table 59).

Table 53. Non-Cross-Reactive Gram Positive Bacteria   

<table><tr><td rowspan=1 colspan=4>ON PANEL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus Species</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Coagulase-NegativeStaphylococci</td><td rowspan=1 colspan=1>Streptococcus Species</td></tr><tr><td rowspan=3 colspan=1>E. aviumE. casseliflavus(2 isolates)E. cecorumE. disparE. duransE. faecalis (3 isolates)E. faceium (2 isolates)aE. gallinarium(2 isolates)E. hiraeE. raffinosus</td><td rowspan=1 colspan=1>MSSA (18 isolates)Resistant S. aureus -BORSA (6 isolates)MRSA (mecA)VRSA (mecA, vanA)</td><td rowspan=5 colspan=1>S. capitis ssp. capitisS. capraeS. cohniiS. epidermidis(2 isolates)S. haemolyticusS. hominisS. lugdunensisS. pasteuriS. saprophyticusS. schleiferi ssp.schleiferiS. sciuriS. warneriS. xylosus</td><td rowspan=5 colspan=1>S. agalactiaeS. anginosusS. bovisS. dysgalactiaeS. gallolyticusS. mitisS. mutansS. parasanguinisS. pneumoniaeS. pseudopneumoniaeS. pyogenesS. salivarius</td></tr><tr><td rowspan=1 colspan=1>Coagulase-PositiveStaphylococci</td></tr><tr><td rowspan=3 colspan=1>S. intermediusS. lutraeS. pseudointermediusS. schleiferi ssp.coagulans</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>L. monocytogenes</td></tr><tr><td rowspan=1 colspan=4>OFF PANEL</td></tr><tr><td rowspan=1 colspan=1>Gram-positive Cocci</td><td rowspan=1 colspan=1>Gram-positive Bacilli</td><td rowspan=1 colspan=1>Listeria Species;</td><td rowspan=1 colspan=1>Gram-positiveAnaerobes</td></tr><tr><td rowspan=1 colspan=1>GranulicatellaadiacensbGemella morbillorumLactococcus lactisMacrococcuscaseolyticusMicrococcus luteusVagococcus fluvialis</td><td rowspan=1 colspan=1>ActinomycesodontolyticusBacillus cereusCorynebacteriumjeikeiumLactobacillusacidophilusMycobacteriumtuberculosisRhodococcus equiRothia mucilaginosa</td><td rowspan=1 colspan=1>L. (murrayi) grayiL. innocuadL. ivanoviissp. londoniensisL. seeligeriL. welshimeri</td><td rowspan=1 colspan=1>Clostridium perfringensPeptostreptococcusanaerobiusPropionibacteriumacnes</td></tr></table>

VRSA $\cdot ^ { = }$ Vancomycin Resistant Staphylococcus aureus. One isolate was tested at a concentration of $\mathfrak { s n g } / { \mathfrak { m L } }$ Extracted DNA; ${ \sim } 1 { \times } 1 0 ^ { 6 } \mathrm { C F U / m l } .$ . $^ { \mathfrak { b } } \mathbf { A }$ multiple subsequent tests. No cross-reactivity between $G$ , adiacens and the BCID Panel Streptococcus assays is predicted by sequence analysis. Tested at a concentration of $7 . 3 3 \times 1 0 ^ { 6 } \mathrm { C F U / m L }$ . possible, however, no cross-reactivity was observed in this testing.

NOTE: While not observed in this study, cross-reactivity between the Enterococcus assay and some Staphylococcus species may occur when the staphylococci are present in a sample at very high concentrations.

Table 54. Non-Cross-Reactive Gram Negative Bacteria   

<table><tr><td rowspan=1 colspan=4>ON PANEL</td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaunannii</td><td rowspan=1 colspan=3>Enterobacteriaceae Isolates&quot;</td></tr><tr><td rowspan=1 colspan=1>A. baumannii(2 isolates)</td><td rowspan=7 colspan=3>Cedeceae davisiae        Escherichia hermanii     Providencia acalifaciensCitrobacter fruendi       Escherichia vulneris      Providencia rettgeriCitrobacter koseri        Hafnia alvei               Providencia stuartiCronobacter muytjensi   Klebsiella oxytoca        Rahnella aquatilisCronobacter sakazakii       (3 isolates)             Raoultella terrigenaEnterobacter amnigenus Klebsiella pneumoniae   Raoultella planticolaEnterobacter asburiae       (6 isolates)             Salmonella entericaEnterobacter              Kluyvera ascorbata       Serratia liquefacienscancerogenus          Kluyvera intermedius     Serratia fonticolaEnterobacter cloacae     Leclercia                  Serratia marcescensEnterobacter              adecarboxylata              (2 isolates)hormaechei            Morganella morganii    Serratia plymuthicaEnterobacter gergoviae  Pantoea agglomeransh   Tatumella ptyseosEscherichia coli           Proteus mirabilis          Yersinia enterocolitica(2 isolates)             Proteus penneri           Yokenella regensburgeiProteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td></tr><tr><td rowspan=1 colspan=1>H. influenzae (type b)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>N. meningitidis</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td></tr><tr><td rowspan=1 colspan=4>OFF PANEL</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Species</td><td rowspan=1 colspan=1>Haemophilus Species</td><td rowspan=1 colspan=1>Pseudomonas Species</td><td rowspan=1 colspan=1>Gram-negative Bacilli</td></tr><tr><td rowspan=5 colspan=1>A. calcoaceticusA. haemolyticusA. johnsoniiA. juniiA. lwoffiiA. radioresistensA. schindleriA. ursingiiA. nosocomialis(genomospecies13TU; 2 isolates )</td><td rowspan=1 colspan=1>H. parahaemolyticusH. parainfluenzaeH. parasuisH. somnus</td><td rowspan=3 colspan=1>P. fluorescensP. luteolaP. nitroreducensP. oryzihabitansP. pertucinogenaP. stutzeri</td><td rowspan=3 colspan=1>Aeromonas hydrophilaBrevundimonasdiminutaMoraxella catarrhalis(3 isolates)StenotrophomonasmaltophilaVibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Neisseria Species</td></tr><tr><td rowspan=3 colspan=1>N. siccaN. elongateN. perflavaN. mucosaN. lactamica</td></tr><tr><td rowspan=1 colspan=1>Gram-negativeAnaerobes</td><td rowspan=1 colspan=1>Gram-negativeCoccobacilli</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilisVeillonella parvula</td><td rowspan=1 colspan=1>Bordetella pertussisCampylobacter fetusChlamydia trachomatisLegionellapneumophilia</td></tr></table>

Se solate were ot deteced by the FilArry BCD Panel andare iscu in the incusiviy evaluaton. agglomerans may be possible. However, no cross-reactivity was observed in this study. Tested at a concentration of $2 . 6 3 \times 1 0 ^ { 8 } \mathrm { C F U / m u l }$ .

Table 55. Non-Cross-Reactive Fungi   

<table><tr><td>ON PANEL</td><td>OFF PANEL </td><td colspan="2"></td></tr><tr><td>Candida Species</td><td>Candida Species</td><td>C. sojae</td><td>Non-Candida Fungi Aspergillus fumigatus</td></tr><tr><td>C. albicans C. glabrata C. krusei . parapsilosis . tropicalis</td><td>C. dubliniensis C. lusitaniae C. metapsilosis C. multigemmis</td><td>C. viswanathii C. guilliermondii</td><td>Debaryomyces hansenii Kluyveromyces lactis Saccharomyces cerevisiae Schizosaccharomyces </td></tr></table>

BioFire Diagnostics, Inc. $5 1 0 ( \mathrm { k } )$ FilmArray BCID Panel

reactivity was observed in this testing.

Table 56. Non-Cross-Reactive Viruses and Mycoplasmataceae   

<table><tr><td rowspan=1 colspan=2>OFF PANEL                                                                                        .:F.</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmataceae Isolates</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis (3.16×10&#x27; CFU/mL )Ureaplasma urealyticum (1.57×10 CFU/mL)</td><td rowspan=1 colspan=1>Cytomegalovirus (1.67×104 TCID5g/mL)Epstein Barr Virus (1.00×10 TCIDs/mL)Herpes Simplex Virus - Type 1 (1:30 dilution of stock)Varicella Zoster Virus (8.17× 10³ TCIDsg/mL)</td></tr></table>

Table 57. Non-Cross-Reactive Antimicrobial Resistance   

<table><tr><td rowspan=1 colspan=1>ON PANEL</td><td rowspan=1 colspan=1>OFF PANEL</td></tr><tr><td rowspan=1 colspan=2>meA</td></tr><tr><td rowspan=1 colspan=1>Methicillin Resistant Staphylococci (mecA)</td><td rowspan=1 colspan=1>Borderline Oxacillin Resistant S. aureus(BORSA)</td></tr><tr><td rowspan=3 colspan=1>Staphylococcus epidermidis-MRSE mecAStaphylococcus aureus-MRSA       mecAStaphylococcus aureus-VRSA       mecA/vanA</td><td rowspan=1 colspan=1>Staphylococcus aureus-BORSA (6 islates)</td></tr><tr><td rowspan=1 colspan=1>Methicillin Sensitive Staphylococci</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus-MSSA (18 isolates)Staphylococcus epidermidis-MRSE (1 isolate)Staphylococcus spp. (16 isolates)</td></tr><tr><td rowspan=1 colspan=2>vanA/B</td></tr><tr><td rowspan=1 colspan=1>Vancomycin Resistant Enterococci (vanA/B)</td><td rowspan=1 colspan=1>Vancomycin Resistant Enterococci (non-vanA/B)</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecalis                 vanBEnterococcus faecium                vanA</td><td rowspan=1 colspan=1>Enterococcus casseliflavus       vanCEnterococcus casseliflavus       vanCEnterococcus gallinarium        vanCEnterococcus gallinarium        vanC</td></tr><tr><td rowspan=1 colspan=1>Vancomycin Sensitive Enterocoeci</td></tr><tr><td rowspan=1 colspan=1>Enterococcus spp. (8 isolates)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Carbapenem Resistant Enterobacteriaceae (KPC)</td><td rowspan=1 colspan=1>Carbapenem Resistant Enterobacteriaceue (non-KPC)</td></tr><tr><td rowspan=5 colspan=1>Klebsiella oxytoca                    KPC-2Klebsiella pneumoniae                KPC-4Serratia marcescens                  KPC-2</td><td rowspan=1 colspan=1>Klebsiella pneumoniae           UnknownKlebsiella pneumoniae           NDM</td></tr><tr><td rowspan=1 colspan=1>Carbapenem Sensitive/Beta-lactam ResistantIsolates</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae           AmpCKlebsiella pneumoniae            SHVEscherichia coli                   TEM-3/CTX-1Acinetobacter baumannii         blaOXAMoraxella catarrhalis             blaOXAMoraxella catarrhalis             BRO-1(bla)/orf3</td></tr><tr><td rowspan=1 colspan=1>Carbapenem Sensitive Isolates</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae (51 isolates)Acinetobacter baumannii (1 isolate)Pseudomonas aeruginosa (2 isolates)</td></tr></table>

Off-panel refers to the antimicrobial resistance gene. Organisms may be positive or organism assay(s). ola   a   C

Table 58. Predicted and Observed Cross-Reactivity Between BCID Panel Assays and On-Panel or Off-Panel Organisms Tested for Exclusivity   

<table><tr><td rowspan=1 colspan=1>BCID Panel Assay/Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism(s)/lsolate(s)/Gene</td></tr><tr><td rowspan=1 colspan=2>Gram-positive Bacteria</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Some coagulase-negative staphylococcia</td></tr><tr><td rowspan=1 colspan=2>Gram-negative Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii (ACB) complex species:Acinetobacter calcoaceticus (ssp. anitratuAcinetobacter pitti (formerly genomospecies 3)b</td></tr><tr><td rowspan=1 colspan=1>Escherichia colilEnterobacteriaceae</td><td rowspan=1 colspan=1>Shigella species:Shigella boydiiShigella dysenteriaeShigella flexneriShigella sonneiEscherichia fergusonnii</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaelEnterobacteriaceae</td><td rowspan=1 colspan=1>Klebsiella variicola (or Klebsiella pneumoniae variant 342)Enterobacter aerogenesRaoultella ornithinolytica</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescenslEnterobacteriaceae</td><td rowspan=1 colspan=1>Serratia species (Serratia entomophila², Serratia ficaria, Serratiaodorifera; Serratia rubidaea)Raoultella ornithinolyticaPseudomonas aeruginosa (ATCC 25619)Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Haemophilus haemolyticus</td></tr><tr><td rowspan=1 colspan=2>Yeast</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Candida orthopsilosis (Group III Candida parapsilosis)h</td></tr><tr><td rowspan=1 colspan=2>Antimicrobial Resistance Geñes</td></tr><tr><td rowspan=1 colspan=1>vanA/B</td><td rowspan=1 colspan=1>vanMi</td></tr></table>

mrequcny in pre-analyucal stuuies anu the cumcal evalualn (estumaed occulence un o1 an pruspecive pauem sanpiesj.   
nicrobial identification methods.

Cross-reactivity was not observed when ATCC 31898 was tested in the inclusivity study at a concentration ${ \sim } 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ , but cross-reactivity was observed in clinical cultures containing R. ornithinolylica.

ross-reactivity was observed only at high organism concentration $( \geq 1 0 ^ { 9 } \mathrm { C F U / m L } )$ ; rare human pathogens.

cross-reactivity observed with five other Pseudomonas aeruginosa isolates tested at ${ \geq } 1 0 ^ { 8 }$ CFU/mL.

andida orthopsilosis is mis-identified as $C .$ parapsilosis by automated and manual microbial identification meth ntcsln a e

# Reproducibility

A multicenter reproducibility study was performed to determine between-site and overall reproducibility of the BCID Panel. Reproducibility testing occurred at three test sites using a panel of six simulated blood culture specimens, each spiked with various

combinations of two different organisms (analytes). To best represent the composition of specimens likely to be tested by the BCID Panel, half of the analytes were at a concentration consistent with the level of organism in a blood culture bottle at the time of positivity, and half of the analytes were at a concentration similar to that observed in bottles eight hours after positivity (see Growth and Detection above). Negative results for each assay were obtained from samples that were not spiked with a corresponding organism (analyte not in the sample).

The data incorporate a range of potential variation introduced by seven different operators, three different pouch lots, and ten different FilmArray Instruments. Every specimen was tested on eight different days, for a total of 90 replicates per analyte. A summary of results (percent $( \% )$ agreement with the expected result) for each analyte is provided in the following tables.

Table 59. Summary of Reproducibility Results - Organism Assays   

<table><tr><td rowspan=2 colspan=1>tBCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>1Test Site</td><td rowspan=1 colspan=2>Results</td><td rowspan=2 colspan=1>% AgreementwithExpected Result</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=12 colspan=1>Enterococcus</td><td rowspan=4 colspan=1>Enterococcus faecium [vanA]JMI4751.50E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=8 colspan=1>180/180100%[98.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Enterococcus faecalis [vanB]JMI 3688.95E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=4 colspan=1>360/360100%[99.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>360/360</td></tr><tr><td rowspan=4 colspan=1>Listeriamonocytogenes</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>540/540100%|99.3% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=8 colspan=1>Staphylococcus</td><td rowspan=4 colspan=1>Staphylococcus aureus[MRSA]ATCC BAA-17478.60E+06 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>449/45099.8%|98.8% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/150^a</td><td rowspan=1 colspan=1>149/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/450</td><td rowspan=1 colspan=1>449/450</td></tr><tr><td rowspan=8 colspan=1>Staphylococcusaureus</td><td rowspan=4 colspan=1>Staphylococcus aureus[MRSA]      ATCC BAA-1747.8.60E+06 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Streptococcus</td><td rowspan=4 colspan=1>Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%|99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=3 colspan=1>Streptococcusagalactiae</td><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=3 colspan=1>540/540100%[99.3% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr></table>

BioFire Diagnostics, Inc. $5 1 0 ( \mathrm { k } )$ FilmArray BCID Panel

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Test Concentration</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Results</td><td rowspan=2 colspan=1>% AgreementwithExpected Result</td></tr><tr><td rowspan=1 colspan=1>..</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Streptococcuspneumoniae</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>540/54000%[99.3% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=8 colspan=1>Streptococcuspyogenes</td><td rowspan=4 colspan=1>Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Acinetobucterbaumanmii</td><td rowspan=4 colspan=1>Acinetobacter baumanniiATCC 99552.00E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=12 colspan=1>Enterobacteriaceae</td><td rowspan=4 colspan=1>Klebsiella pneumoniae [KPC]JMI 7669.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=8 colspan=1>180/180100%[98.0%-100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=4 colspan=1>360/360100%[99.0%-100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>360/360</td></tr><tr><td rowspan=4 colspan=1>Enterobactercloucue complex</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>540/540100%|99.3% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>540/540100%|99.3% -100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=4 colspan=1>Klebsiella oxytoca</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>540/540100%|99.3% -100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=8 colspan=1>Klebsiellapneumoniae</td><td rowspan=4 colspan=1>Klebsiella pneumoniae [KPC]JMI 7669.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/9010%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Proteus</td><td rowspan=4 colspan=1>Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%196.0% - 100%1</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%|99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=2 colspan=1>Serratiamurcescens</td><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=2 colspan=1>540/540100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr></table>

BioFire Diagnostics, Inc. $5 1 0 ( \mathrm { k } )$ FilmArray BCID Panel

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>*Test Site</td><td rowspan=1 colspan=2>..       Results</td><td rowspan=2 colspan=1>% AgreementwithExpected Result</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=2 colspan=1>199.3% -100%|</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=4 colspan=1>Haemophilusinfluenzae</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Sitc A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>539/54098.0%|99.0% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/180</td><td rowspan=1 colspan=1>179/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/540</td><td rowspan=1 colspan=1>539/540</td></tr><tr><td rowspan=4 colspan=1>Neisseriameningitidis</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>.540/540100%[99.3% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td></tr><tr><td rowspan=8 colspan=1>Pseudomonasaeruginosa</td><td rowspan=4 colspan=1>Pseudomonas aeruginosaATCC 278531.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B.</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%|99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Candida albicans</td><td rowspan=4 colspan=1>Candida albicansATCC 102313.10E+04</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%%|96.0% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%|99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Candida glabruta</td><td rowspan=4 colspan=1>Candida glabrataATCC 155452.00E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Candida krusei</td><td rowspan=4 colspan=1>Candida kruseiATCC 908783.20E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%196.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%|99.2% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=4 colspan=1>Candidaparapsilosis</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>539/54099.8%199.0% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/180</td><td rowspan=1 colspan=1>179/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/540</td><td rowspan=1 colspan=1>539/540</td></tr><tr><td rowspan=8 colspan=1>Candida tropicalis</td><td rowspan=4 colspan=1>Candida tropicalisATCC 660299.70E+05</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% -100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr></table>

Candida parapsilosis.

Table 60. Summary of Reproducibility Results — Antimicrobial Resistance Gene Assays   

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>Tesi Site</td><td rowspan=1 colspan=3>Results</td><td rowspan=2 colspan=1>% Agreementwith ExpectedTest Result</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>vanA/B</td><td rowspan=1 colspan=1>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>180/180</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=11 colspan=1></td><td rowspan=3 colspan=1>JMI4751.50E+08 CFU/mL</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=7 colspan=1>100%[98.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Enterococcus faecalis [vanB]JMI 3688.95E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>No Associated Organism</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=4 colspan=1>360/360100%|99.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>360/360</td></tr><tr><td rowspan=8 colspan=1>mecA</td><td rowspan=4 colspan=1>Staphylococcus aureusMRSA]      ATCC BAA-17478.60E+06 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%|96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>No Associated Organism</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>449/450&quot;99.8%|98.8% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/150 </td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>149/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/450</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>449/450</td></tr><tr><td rowspan=12 colspan=1>KPC</td><td rowspan=4 colspan=1>Klebsiella pneumoniae [KPC]JMI 7669.40E+08</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Proteus mirabilis ATCC29906.andPseudomonas aeruginosaATCC 27853</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=4 colspan=1>270/270100%|98.6% - 100%|</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/270</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>0/270</td></tr><tr><td rowspan=4 colspan=1>No Associated Organism</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=4 colspan=1>180/180100%[98.0% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr></table>

A single pouch run at Site B generated a false positive mecA result

The reproducibility of Tm for each analyte and positive assay was also evaluated and a summary is provided in the following tables.

Table 61. Summary of Tm Analysis for Positive Organism Assays   
BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel   

<table><tr><td colspan="1" rowspan="2">BCID PanelAssay</td><td colspan="1" rowspan="2">Organism TestedTest Concentration</td><td colspan="1" rowspan="2">Test Site</td><td colspan="5" rowspan="1">Reproducibility of Tm</td></tr><tr><td colspan="1" rowspan="1">Tm Mean</td><td colspan="1" rowspan="1">Tm Std Dev</td><td colspan="1" rowspan="1">T'mMin</td><td colspan="1" rowspan="1">TmMax</td><td colspan="1" rowspan="1">Range(max-min)</td></tr><tr><td colspan="8" rowspan="1">Gram-Positive Bacteria</td></tr><tr><td colspan="1" rowspan="8">Enterococcus</td><td colspan="1" rowspan="4">Enterococcus faecium [vanA]JMI4751.50E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.5</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">84.0</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.6</td><td colspan="1" rowspan="1">.0.2</td><td colspan="1" rowspan="1">82.3</td><td colspan="1" rowspan="1">83.0</td><td colspan="1" rowspan="1">0.7</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.3</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">82.8</td><td colspan="1" rowspan="1">0.9</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">82.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">84.0</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis [vanB]JMI 3688.95E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">82.4</td><td colspan="1" rowspan="1">0.9</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">81.8</td><td colspan="1" rowspan="1">82.8</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">81.6</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">81.0</td><td colspan="1" rowspan="1">82.4</td><td colspan="1" rowspan="1">1.4</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">81.0</td><td colspan="1" rowspan="1">82.8</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="1">Saureus</td><td colspan="1" rowspan="1">Staphylococcus aureus [MRSA]</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">77.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">76.6</td><td colspan="1" rowspan="1">77.8</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="2">BCID PanelAssay</td><td colspan="1" rowspan="2">Organism TestedTest Concentration</td><td colspan="1" rowspan="1">-.6</td><td colspan="5" rowspan="1">Reproducibility of Tm</td></tr><tr><td colspan="1" rowspan="1">Test Site</td><td colspan="1" rowspan="1">Tm Mean</td><td colspan="1" rowspan="1">Tm Std Dev</td><td colspan="1" rowspan="1">T'mMin</td><td colspan="1" rowspan="1">TmMax</td><td colspan="1" rowspan="1">Range(max-min)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">ATCC BAA-17478.60E+06 CFU/mL</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">77.3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">76.8</td><td colspan="1" rowspan="1">77.8</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">76.9</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">76.5</td><td colspan="1" rowspan="1">77.5</td><td colspan="1" rowspan="1">·1.0</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">77.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">76.5</td><td colspan="1" rowspan="1">77.8</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="4">Streptococcus</td><td colspan="1" rowspan="4">Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">83.6</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">81.8</td><td colspan="1" rowspan="1">82.3</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">81.8</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">82.1</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">83.6</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="4">Spyogenes</td><td colspan="1" rowspan="4">Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">78.5</td><td colspan="1" rowspan="1">79.8</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">78.7</td><td colspan="1" rowspan="1">79.8</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.8</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">78.5</td><td colspan="1" rowspan="1">79.5</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">79.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">78.5</td><td colspan="1" rowspan="1">79.8</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="2" rowspan="1">EY</td><td colspan="5" rowspan="1">Gram-Negative Bacteria</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Abaumannii,</td><td colspan="1" rowspan="4">Acinetobacter baumanniiATCC 99552.00E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.6</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.8</td><td colspan="1" rowspan="1">.0.2</td><td colspan="1" rowspan="1">80.4</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.3</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">79.5</td><td colspan="1" rowspan="1">80.9</td><td colspan="1" rowspan="1">1.4</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.5</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">79.5</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="4">Enteric</td><td colspan="1" rowspan="4">Klebsiella pneumoniae [KPC]JMI 7669.40E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">88.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">88.1</td><td colspan="1" rowspan="1">89.1</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">88.8</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">88.6</td><td colspan="1" rowspan="1">89.2</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">88.3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">87.8</td><td colspan="1" rowspan="1">88.8</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">88.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">87.8</td><td colspan="1" rowspan="1">89.2</td><td colspan="1" rowspan="1">1.4</td></tr><tr><td colspan="1" rowspan="4">Kpneumoniae</td><td colspan="1" rowspan="4">Klebsiella pneumoniae [KPC]JMI 7669.40E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.9</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">87.3</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">88.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">87.8</td><td colspan="1" rowspan="1">88.4</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">87.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">88.1</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="4">Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">80.6</td><td colspan="1" rowspan="1">81.8</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.4</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">0.7</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">80.7</td><td colspan="1" rowspan="1">81.6</td><td colspan="1" rowspan="1">0.9</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">81.3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">80.6</td><td colspan="1" rowspan="1">81.9</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="4">Paeruginosa</td><td colspan="1" rowspan="4">Pseudomonas aeruginosaATCC 278531.40E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.9</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">87.3</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">88.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">87.8</td><td colspan="1" rowspan="1">89.5</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">88.1</td><td colspan="1" rowspan="1">89.1</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">88.2</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">87.3</td><td colspan="1" rowspan="1">89.5</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="7" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Calbicans</td><td colspan="1" rowspan="4">Candida albicansATCC 102313.10E+04</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.8</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">79.3</td><td colspan="1" rowspan="1">80.3</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">79.7</td><td colspan="1" rowspan="1">80.5</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">78.9</td><td colspan="1" rowspan="1">80.2</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">79.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">78.9</td><td colspan="1" rowspan="1">80.5</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="4">Cglabrata</td><td colspan="1" rowspan="4">Candida glabrataATCC 155452.00E+07</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">75.3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">74.7</td><td colspan="1" rowspan="1">76.1</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">75.4</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">74.9</td><td colspan="1" rowspan="1">76.4</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">75.7</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">75.4</td><td colspan="1" rowspan="1">76.1</td><td colspan="1" rowspan="1">0.7</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">75.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">74.7</td><td colspan="1" rowspan="1">76.4</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="4">Ckrusei</td><td colspan="1" rowspan="4">Candida kruseiATCC 908783.20E+07</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.5</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">84.1</td><td colspan="1" rowspan="1">85.2</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">84.3</td><td colspan="1" rowspan="1">85.3</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">84.6</td><td colspan="1" rowspan="1">85.8</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">84.1</td><td colspan="1" rowspan="1">85.8</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="3">Ctropicalis</td><td colspan="1" rowspan="3">Candida tropicalisATCC 660299.70E+05</td><td colspan="1" rowspan="1">Sile A</td><td colspan="1" rowspan="1">79.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">78.6</td><td colspan="1" rowspan="1">80.1</td><td colspan="1" rowspan="1">1.6</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">78.8</td><td colspan="1" rowspan="1">79.6</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">79.3</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">0.7</td></tr></table>

BioFire Diagnostics, Inc. 510(k) FilmArray BCID Panel

<table><tr><td rowspan=2 colspan=1>BCID PanelAssay</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>;Test Site</td><td rowspan=1 colspan=5>IReproducibility of Tm</td></tr><tr><td rowspan=1 colspan=1>T&#x27;m Mean</td><td rowspan=1 colspan=1>Tm Std Dev</td><td rowspan=1 colspan=1>TmMin</td><td rowspan=1 colspan=1>TmMax</td><td rowspan=1 colspan=1>Range(max-min)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>79.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.6</td><td rowspan=1 colspan=1>80.1</td><td rowspan=1 colspan=1>1.6</td></tr></table>

Table 62. Summary of Tm Analysis for Positive Antimicrobial Resistance Gene Assays   

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=5>Reproducibility of Tm</td></tr><tr><td rowspan=1 colspan=1>T&#x27;m Mean</td><td rowspan=1 colspan=1>Tm Std Dev</td><td rowspan=1 colspan=1>T&#x27;mMin</td><td rowspan=1 colspan=1>TmMax</td><td rowspan=1 colspan=1>Range(max-min)</td></tr><tr><td rowspan=8 colspan=1>vanA/B</td><td rowspan=4 colspan=1>Enterococcus faecium [vanA]JMI4751.50E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>86.5</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>86.3</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=4 colspan=1>Enterococcus fuecalis [vanB]JMI 3688.95E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>85.3</td><td rowspan=1 colspan=1>86.6</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>85.9</td><td rowspan=1 colspan=1>86.9</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>86.6</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>86.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>86.9</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=4 colspan=1>mecA</td><td rowspan=4 colspan=1>Staphylococcus aureus [MRSA]ATCC BAA-17478.60E+06 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>73.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>73.1</td><td rowspan=1 colspan=1>74.4</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>73.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>73.2</td><td rowspan=1 colspan=1>74.2</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>73.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>72.8</td><td rowspan=1 colspan=1>74.1</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>73.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>72.8</td><td rowspan=1 colspan=1>74.4</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=4 colspan=1>KPC</td><td rowspan=4 colspan=1>Klebsiella pneumoniae [KPC]JMI 7669.40E+08</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>86.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>1.5</td></tr></table>

# Interference

Substances that could be present in blood culture samples or introduced during sample handling were evaluated for their potential to interfere with assay performance. A potentially interfering substance (see Table 63) was added to a simulated positive aerobic blood culture sample which contained simulated blood culture matrix (human whole blood that had been incubated in a blood culture bottle) and one of six different organism mixes. Each organism mix contained two live pathogens at a concentration equivalent to the level determined to be present when a blood culture bottle is detected as positive by the blood culture instrument. None of the substances tested were found to compete or interfere with the assays in the BCID Panel.

Table 63. Potentially Interfering Substances   

<table><tr><td>Endogenous Substances</td><td colspan="2">Exogenous Substances</td><td>Technique-Specitic Substances</td></tr><tr><td>Hemoglobin Triglycerides Bilirubin</td><td>Fluconazole Vancomycin Ciprofloxacin Gentamicin sulfate</td><td>Ceftriaxone Tetracycline Amoxicillin/Clavulanate Heparin</td><td>Bleach Ethanol</td></tr><tr><td colspan="2">Y·globulin Human Imipenem Genomic DNA</td><td colspan="2">Sodium Polyanetholesulfonate (SPS)</td></tr><tr><td colspan="2">On-Panel Competing Micoroorganisms</td><td colspan="2">Off-Panel Compcting Micoroorganisms</td></tr><tr><td colspan="2">Staphylococcus epidermidis Escherichia coli Streptococcus mitis</td><td colspan="2">Coryne bacterium jeike ium Bacillus cereus Micrococcus luteus Clostridium perfringens</td></tr><tr><td colspan="2">Blood Culture Media/Bottle Types</td><td colspan="2">Propionibacterium acnes</td></tr><tr><td colspan="4">BACTEC Plus Acrobic/F BacT/ALERT SA Standard Aerobic</td></tr><tr><td>BACTEC Standard Aerobic</td><td colspan="2">BacT/ALERT SN Standard Anaerobic</td><td>VersaTREK REDOX 1</td></tr><tr><td></td><td colspan="2">BacT/ALERT FA Aerobic FAN</td><td>VersaTREK REDOX 2</td></tr><tr><td>BACTEC Standard Anaerobic BACTEC Plus Anaerobic/F</td><td colspan="2">BacT/ALERT FN Anaerobic FAN</td><td></td></tr><tr><td></td><td colspan="2"></td><td></td></tr><tr><td>BACTEC Pediatric Plus BACTEC Lytic/10 Anacrobic/F</td><td colspan="3">BacT/ALERT PF Pediatric FAN BacT/Alert FA Plus Aerobic</td></tr></table>

for use with the FilmArray BCID Panel.

BETH LINGENFELTER, M.S. BIOFIRE DIAGNOSTICS, INC. 390 WAKARA WAY SALT LAKE CITY UT 84108

Re: K130914 Trade/Device Name: FilmArray Blood Culture Identification Panel Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures Regulatory Class: II Product Code: PAM, PEO, PEN, OOI Dated: March 30, 2013 Received: April 02, 2013

Dear Ms. Lingenfelter:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class IIl (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Division of Small Manufacturers, International and Consumer   
Assistance at its toll-free number (800) 638 2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/lndustry/default.htm. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 87.9.For questions regarding the reporting  dverse events under theMDR regulation 1 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htn for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

# Sally A. Hojvat -S

Sally Hojvat, Ph.D., M.Sc   
Director, Division of Microbiology Devices   
Office of In Vitro Diagnostics and Radiological Health   
Center for Devices and Radiological Health

# Indications for Use

510(k) Number (if known): K130914 Device Name: FilmArray Blood Culture Identification (BCID) Panel

The FilmArray Blood Culture Identification (BCID) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with the FilmArray Instrument. The FilmArray BCID Panel is capable of simultaneous detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants of antimicrobial resistance. The BCID assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system that demonstrates the presence of organisms as determined by Gram stain.

The following gram-positive bacteria, gram-negative bacteria, and yeast are identified using the FilmArray BCID Panel: Enterococci, Listeria monocytogenes, commonly encountered Staphylococci (including specific differentiation of Staphylococcus aureus), commonly encountered Streptococci (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes), Acinetobacter baumannii, commonly encountered Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens), Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa, Candida albicans, Candida glabrata, Candida krusei, Candida parapsilosis, and Candida tropicalis.

The FilmArray BClD Panel also contains assays for the detection of genetic determinants of resistance to methicillin (mecA), vancomycin (vanA and vanB), and carbapenems $( b l a _ { k P C } )$ to aid in the identification of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, and carbapenems exist.

FilmArray BCID is indicated as an aid in the diagnosis of specific agents of bacteremia and fungemia and results should be used in conjunction with other clinical and laboratory findings. Positive FilmArray results do not rule out co-infection with organisms not included in the FilmArray BCID Panel. FilmArray BCID is not intended to mionitor treatment for bacteremia or fungemia.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the FilmArray BCiD Panel, and for species determination of some Staphylococci, Enterococci, Streptococci, and Enterobacteriaceae that are not specifically identified by the FilmArray BCID Panel assays.

# (PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE IF NEEDED)

Concurrence of CDRH, Office of In Vitro Diagnostics and Radiological Health (OIR)

# 8.1M5

Diviston Sign-Off   
OfficinViroDgnostics andRadiotogicalHeath   
510(k) K130914