# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

B. Purpose for Submission:

To obtain a substantial equivalent determination for a premarket notification for the MALDI Biotyper CA System

# C. Measurand:

Achromobacter xylosoxidans   
Acinetobacter baumannii complex   
Acinetobacter lwoffi   
Acinetobacter radioresistens   
Acinetobacter ursingii   
Aeromonas sp   
Alcaligenes faecalis   
Burkholderia gladioli   
Burkholderia multivorans   
Burkholderia cepacia complex   
Citrobacter amalonaticus complex   
Citrobacter koseri   
Citrobacter freundii complex   
Eikenella corrodens   
Enterobacter aerogenes   
Enterobacter cloacae complex   
Escherichia coli   
Haemophilus influenzae   
Haemophilus parainfluenzae   
Hafnia alvei   
Klebsiella pneumoniae   
Klebsiella oxytoca/Raoultella ornithinolytica   
Moraxella (Branhamella) catarrhalis   
Moraxella sg Moraxella osloensis   
Morganella morganii   
Pantoea agglomerans   
Pasteurella multocida   
Proteus mirabilis   
Proteus vulgaris group   
Providencia rettgeri   
Providencia stuartii   
Pseudomonas aeruginosa   
Pseudomonas fluorescens group   
Pseudomonas putida group   
Salmonella sp   
Serratia liquefaciens   
Serratia marcescens   
Stenotrophomonas maltophilia   
Yersinia enterocolitica   
Yersinia pseudotuberculosis

# D.Type of Test:

The MALDI Biotyper CA System is a qualitative in vitro diagnostic device intended for the identification of Gram-negative bacterial colonies cultured from human specimens. The device is comprised of an ionization source, a mass analyzer and a spectral database.The device is indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of Gram-negative bacterial infections.

# E. Applicant:

Bruker Daltonics, Inc

F. Proprietary and Established Names:

Trade Name: MALDI Biotyper CA System

Common Name: MBT-CA

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3361

2. Classification: Class II (special controls)

3. Product code: PEX

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Bruker Daltonics, Inc. MALDI Biotyper CA System is a qualitative in vitro diagnostic mass spectrometer system for the identification of Gram-negative bacterial colonies cultured from human specimens using matrix-assisted laser desorption/ionization - time of flight (MALDITOF) mass spectrometry technology.

The MALDI Biotyper CA System is indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of Gram-negative bacterial infections.

The following organisms are claimed:

Achromobacter xylosoxidans   
Acinetobacter baumannii complex   
Acinetobacter lwoffii   
Acinetobacter radioresistens   
Acinetobacter ursingii   
Aeromonas species   
Alcaligenes faecalis   
Burkholderia gladioli   
Burkholderia multivorans   
Burkholderia cepacia complex   
Klebsiella pneumoniae   
Klebsiella oxytoca Raoultella ornithinolytica   
Moraxella sg Branhamella catarrhalis   
Moraxella sg Moraxella osloensis   
Morganella morganii   
Pantoea agglomerans   
Pasteurella multocida   
Proteus mirabilis   
Proteus vulgaris group   
Providencia rettgeri   
Citrobacter amalonaticus_complex   
Citrobacter koseri   
Citrobacter freundii complex   
Eikenella corrodens   
Enterobacter aerogenes   
Enterobacter cloacae complex   
Escherichia coli   
Haemophilus influenzae   
Haemophilus parainfluenzae   
Hafnia alvei   
Providencia stuartii   
Pseudomonas aeruginosa   
Pseudomonas fluorescens group   
Pseudomonas putida group   
Salmonella species   
Serratia liquefaciens   
Serratia marcescens   
Stenotrophomonas maltophilia   
Yersinia enterocolitica   
Yersinia pseudotuberculosis

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): The MALDI Biotyper CA System is for prescription use only in accordance with 21 CFR 801.109.

4. Special instrument requirements:

Mass Spectrometer: microflex LT/SH mass spectrometer

Target Plates: US IVD 48 Spot Target

Reagents:

US IVD Bacterial Test Standard (BTS) US IVD HCCA portioned (α-Cyano-4-hydroxycinnamic acid)

Database:MALDI Biotyper for Clinical Applications (MBT-CA)

# Software:

MBT-CA System Software Package: MBT-CA System client software displaying the user interface MBT-CA System Server MBT-CA System DB Server

flexControl Software Package GTPS firmware flexControl acquisition software

# I. Device Description:

The MBT-CA System consists of the microflex LT/SH mass spectrometer, reference library, kit reagents (US IVD HCCA, US IVD Bacterial Test Standard), US IVD 48 Spot Target slides, and software. The MALDI Biotyper CA System with closed safety covers is a Class 1 Laser product. With the safety cover opened it becomes a Class 4 Laser product. The laser is a $3 3 7 \mathrm { n m }$ fixed focus, nitrogen laser. The MALDI Biotyper CA System also referred to as the MBT-CA System.

The reference library, MALDI Biotyper for Clinical Applications, includes type strains, clinical strains and culture collection strains. The MALDI Biotyper CA System reference library was established by analyzing the type strain from each claimed species combined with 5 to 10 additional strains from the same species provided by clinical laboratories or different commercial strain collections for a total of 528 strains. Library mass spectra used for matching contain up to 70 peaks. The MALDI Biotyper for Clinical Applications is also referred to as the MBT-CA.

US IVD HCCA portioned (α-Cyano-4-hydroxycinnamic acid) is a solution that is used when processing test organisms for identification on the MALDI Biotyper CA System. US IVD HCCA is reconstituted in accordance with instructions provided using recommended solvent. $1 . 0 \mu \mathrm { L }$ of the matrix is added to the spot with the sample and allowed to dry.

USIVD BacterialTest Stanar (BTS) is ann-vio-diaosti prodc use or ualiyol and validation of the microflex LT/SH mass spectrometers. US IVD BTS contains a manufactured extract of Escherichia coli DH5 alpha that demonstrates a characteristic peptide and protein profile mass spectrum, when tested on the MALDI Biotyper CA System. US IVD BTS is spiked with two additional proteins that extend the upper boundary of the mass range of the US IVD BTS. The overall mass range covered by US IVD BTS is 3.6 to $1 7 \ k D \mathsf { a }$

US IVD 48 Spot Target plates are reusable steel plates which have been developed for the preparation and identification f test organisms using the MALDI Biotyper CA System. The target allows for the identification of 48 test organisms. The target has five cross-joint positions which should be usd r US IVD BTS cnol.Taret plate cen is peore atr ea .

MALDI Biotyper CA Sysecli war splay usernte whic guis e ur through the MALDI Biotyper CA System workflow. The MALDI Biotyper CA System client also interfaces to the flexControl software for automated acquisition of mass spectra on the microflex LT/SH instrument.

The MALDI Biotyper CA System server communicates with the MALDI Biotyper CA System ci anh- It peaks ists agaist he Mai Spectru refere pat, or mathing andcalcuats the score value (log (score)).

The MBT-DB server stores all information for the MALDI Biotyper CA System. The MBT-DB maintains spectra data (creation information and mass/intensity lists), project data (results o defined and executed runs), method data (parameter lists for spectra preprocessing and enat u t  at  pe pu

maintenance data.

GTPS firmware communicates with the flexControl PC software, controls and monitors the vacuum, moves the sample carrier and performs the docking of the target plate, controls and monitors high voltages in the ion source, generates trigger signals, and monitors instrument status.

The flexControl acquisition software communicates with the MALDI Biotyper CA System client, loads automatic run jobs, communicates with the GTPS firmware, communicates with the laser in the microflex LT/SH instrument, sets the acquisition parameters in the digitizer and reads the acquired data from the digitizer, performs automated data acquisition, evaluates acquired spectra, adjusts the laser power during automatic data acquisition, performs a re-calibration of the time-of-flight to mass transformation, stored acquired spectra on disk and performs source cleaning. The flexControl software does not display a user interface.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Vitek® MS

2. Predicate 510(k) number(s): K124067

# 3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>DIFFERENCES</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>NEW DEVICEMALDI Biotyper CA System(K130831)</td><td rowspan=1 colspan=1>PREDICATE DEVICEVitek® MS(K124067)</td></tr><tr><td rowspan=1 colspan=1>Culture Age</td><td rowspan=1 colspan=1>Bacteria growth should be between18h to 36h.</td><td rowspan=1 colspan=1>Bacteria and yeast growth should bebetween 24 to 72 hours.</td></tr><tr><td rowspan=1 colspan=1>Calibration</td><td rowspan=1 colspan=1>Bruker US IVD Bacterial TestStandard (BTS)</td><td rowspan=1 colspan=1>E. coli ATCC 8739</td></tr><tr><td rowspan=1 colspan=1>MALDI TargetPlate</td><td rowspan=1 colspan=1>US IVD 48 Spot Target48 positions reusable steel targets</td><td rowspan=1 colspan=1>VITEK MS DS Target Slides48 positions disposable plastic targets</td></tr><tr><td rowspan=1 colspan=1>MALDI TOFMS Instrument</td><td rowspan=1 colspan=1>Bruker microflex LT/SH (benchtop)</td><td rowspan=1 colspan=1>Shimadzu AXIMA® Assurance MS(floor standing)</td></tr><tr><td rowspan=1 colspan=1>Database</td><td rowspan=1 colspan=1>MALDI Biotyper for ClinicalApplications (MBT CA)</td><td rowspan=1 colspan=1>VITEK® MS V2.0 Knowledge Base</td></tr></table>

These differences do not affect substantial equivalence of the MBT CA System and Vitek_ MS system. Both systems are mass spectrometer systems using matrix-assisted laser desorption/ionization time of flight (MALDI TOF) for the identification of microorganisms cultured from human specimens.   

<table><tr><td rowspan=1 colspan=3>SIMILARITIES</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>NEW DEVICEMALDI Biotyper CA System(K130831)</td><td rowspan=1 colspan=1>PREDICATE DEVICEVitek MS(K124067)</td></tr><tr><td rowspan=1 colspan=1>Product Codes</td><td rowspan=1 colspan=1>PEX</td><td rowspan=1 colspan=1>PEX</td></tr><tr><td rowspan=1 colspan=1>Intended use</td><td rowspan=1 colspan=1>The MALDI Biotyper CA Systemis a qualitative in vitro diagnostic massspectrometer system for theidentification of Gram negativebacterial colonies cultured fromhuman specimens using matrixassisted laser desorption/ ionization-time of flight (MALDI-TOF) massspectrometry technology.The MALDI Biotyper CA System isindicated for use in conjunction withother clinical and laboratory findingsto aid in the diagnosis of Gramnegative bacterial infections.</td><td rowspan=1 colspan=1>The Vitek® MS is a massspectrometer system using matrixassisted laser desorption/ionization-time to flight (MALDI-TOF) for theidentification of microorganismscultured from human specimen.The VITEK MS is a qualitative invitro diagnostic device indicated foruse in conjunction with other clinicaland laboratory findings to aid in thediagnosis of bacterial and yeastinfections.</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Isolated colony from any patientsample source.Acceptable media:•  Columbia blood agar with 5%sheep blood•  Trypticase soy agar with 5%sheepBloodChocolate agarMacConkey Agar</td><td rowspan=1 colspan=1>Isolated colony from any patientsample source.Acceptable media:•  Columbia blood agar with 5%sheep bloodTrypticase soy agar with 5%sheepBloodChocolate polyvitex agar•  Campylosel agarMacConkey AgarModified Sabouraud dextroseAgarChromID CPS</td></tr><tr><td rowspan=1 colspan=1>Type of Test</td><td rowspan=1 colspan=1>Automated Mass Spectrometry System</td><td rowspan=1 colspan=1>Automated Mass Spectrometry System</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>α-Cyano-4-hydroxycinnamic acid</td><td rowspan=1 colspan=1>α-Cyano-4-hydroxycinnamic acid</td></tr></table>

<table><tr><td rowspan=2 colspan=4>SIMILARITIES</td></tr><tr><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>NEW DEVICEMALDI Biotyper CA System(K130831)</td><td rowspan=1 colspan=1>PREDICATE DEVICEVitek MS(K124067)</td></tr><tr><td rowspan=1 colspan=1>Method ofTesting</td><td rowspan=1 colspan=1>Bacteria: Direct testingIf after initial analysis the log(score) isreported at &lt; 2.00, organisms areprocessed using the extractionprocedure.</td><td rowspan=1 colspan=1>Bacteria: Direct testing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Result Reporting</td><td rowspan=1 colspan=1>Organism identification is reportedwith high confidence if the log(score)is m 2.00.An organism identification is reportedwith low confidence if the log(score)is between 1.70 and &lt;2.00.</td><td rowspan=1 colspan=1>A single identification is displayed,with a confidence value from 60.0 to99.9, when one significant organism ororganism group is retained.&quot;Low-discrimination&quot; identificationsare displayed when more than one butnot more than four significantorganisms or organism groups areretained.When more than four organisms ororganism groups are found, or whenno match is found, the organism isconsidered unidentified.</td><td></td></tr><tr><td rowspan=1 colspan=1>MatchingAlgorithm</td><td rowspan=1 colspan=1>Calculates matches by comparing anew spectrum against each singlereference entry of a reference database.</td><td rowspan=1 colspan=1>Uses a proprietary process called&quot;mass binning.&quot; In this process, thespectrum between 3,000 and 17,000Daltons are divided into 1300 pre-defined intervals called &quot;bins&quot;. Next,an algorithm based on supervisedmachine learning known as the&quot;Advanced Spectrum Classifier&quot;, isused to determine how informativeeach bin was in differentiating thatspecies from all other species in thedatabase.</td><td></td></tr><tr><td rowspan=1 colspan=1>Recorded massrange</td><td rowspan=1 colspan=1>2,000 - 20,000 m/z</td><td rowspan=1 colspan=1>2,000 - 20,000 m/z</td><td></td></tr></table>

K. Standard/Guidance Document Referenced (if applicable):   

<table><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>StandardsNo.</td><td rowspan=1 colspan=1>RecognitionNo.(FDA)</td><td rowspan=1 colspan=1>Standards Title</td><td rowspan=1 colspan=1>Date</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CLSI MM-18A</td><td rowspan=1 colspan=1>7-192</td><td rowspan=1 colspan=1>Criteria for Identification of Bacteria andFungi by DNA Target Sequencing; ApprovedGuideline, 1 Edition</td><td rowspan=1 colspan=1>4/28/2008</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CLSIEP09-A2-IR</td><td rowspan=1 colspan=1>7-92</td><td rowspan=1 colspan=1>Method Comparison and Bias EstimationUsing Patient Samples; Approved Guideline--Second Edition (Interim Revision),</td><td rowspan=1 colspan=1>7/30/2010</td></tr></table>

# L. Test Principle:

Organisms to be identified with the MALDI Biotyper CA System are isolated for on appropriate isolation media. Users are instructed to first test the organism using the direct transfer technique; if results are less than $< 2 . 0$ log(score), users are then directed to perform extraction procedure.

Direct Transfer (DT): An individual colony from an overnight subculture plate is transferred to a selected position on an US IVD 48 Spot Target (target). The target is air dried and US IVD HCCA portioned (matrix) is added. The standard solvent ( $5 0 \%$ acetonitrile $/ 4 7 . 5 \% \mathrm { H } _ { 2 } \mathrm { O } / 2 . 5 \%$ trifluoroacetic acid) in the matrix solution extracts proteins (mainly ribosomal proteins, which are present in high concentration) from the microorganisms. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MALDI Biotyper CA System.

Extraction Procedure $( E x t )$ : For this purpose, isolated colonies from the overnight subculture plate are extracted using an ethanol/formic acid procedure. Afterwards they are transferred to the target and treated as described above.

Samples are analyzed using MALDI (matrix-assisted laser desorption/ionization) TOF (time-offlight) mass spectrometry. The matrix transfers protons onto the extracted proteins and absorbs UV light. After complete drying, the mixture is exposed to laser pulses, resulting in energy transfer from the matrix causing evaporation and release of positively charged intact proteins and peptides ("soft" ionization technique). The ionized molecules areaccelerated by electrical potentials through a flght tube to the mass spectrometer, with separation of the particles determined by their mass/charge ratio $\left( m / z \right)$ As different proteins/peptides have different masses, ions arrive at the detector at diffrent times (time of flight). The system measures the time (in the nanosecond range) between pulsed acceleration and the corresponding detector signal, the speed is converted into an exact molecular mas.Themass--chare t s propornal he sqt rit ti. Hly abundant microbial proteins (mainly ribosomal proteins) result in a mass spectrum with characteristcmass and intensiy disribution. It is pecies-specic or many bacteria nd interpreted as a molecular fingerprint to identify the test organism. Data acquisition is controlled with MALDI Biotyper CA System Software. The spectrum of the unknown organism is first transformed into a peak list. This peak list is compared to the reference peak list of each organism ouinhe reference lbrary database)and alogscore) is generatAhigher logscore)indicates a higher degree of similarity t the organism in the reference library.Organism identification is

reported with high confidence if the log(score) is $\ge 2 . 0 0$ . An organism identification is reported with low confidence if the log(score) is between 1.70 and $< 2 . 0 0$ .

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Precision/Repeatability:

Validation of the complete MALDI Biotyper CA System was performed on twelve working days with two runs/day following manufacturer's instructions for use. Ten test organisms were tested in triplicate in each run. The study also tested multiple sources of system variability including three test operators, three microflex LT/SH instruments, three target lots, three BTS lots and three Matrix lots. As triplicates of each test organism were prepared and tested in each run, a total of 72 MALDI Biotyper CA System identifications were carried out for each test organism. Overall results from the precision/repeatability study are presented below.

Overall Precision per Test Organism   

<table><tr><td rowspan=1 colspan=1>Test Organism</td><td rowspan=1 colspan=2># samples measured</td><td rowspan=1 colspan=1># samplespassed (DT)</td><td rowspan=1 colspan=1># samples passed(DT+Ext)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>62 (86.1%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>70 (97.2%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=2>72</td><td rowspan=1 colspan=1>70 (97.2%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>70 (97.2%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>71 (98.6%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72 (100%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72 (100%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Brevundimonas diminuta*</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72 (100%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72 (100%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72 (100%)</td><td rowspan=1 colspan=1>72 (100%)</td></tr></table>

\* Brevundimonas diminuta was tested but is not included in the claim.

Table 3: Overall Precision per Test Organism Average log(score)   

<table><tr><td rowspan=1 colspan=2>Test Organism</td><td rowspan=1 colspan=1># samples measured</td><td rowspan=1 colspan=1>Average log(score)[DT+Ext]</td></tr><tr><td rowspan=1 colspan=2>Acinetobacter baumannii</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.172 ± 0.113</td></tr><tr><td rowspan=1 colspan=2>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.368 ± 0.121</td></tr><tr><td rowspan=1 colspan=2>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.364 ± 0.078</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.138 ± 0.064</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.385 ± 0.097</td></tr><tr><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.452 ± 0.095</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Test Organism</td><td rowspan=1 colspan=1># samples measured</td><td rowspan=1 colspan=1>Average log(score)[DT+Ext]</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.587 ± 0.079</td></tr><tr><td rowspan=1 colspan=1>Brevundimonas diminuta</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.489 ± 0.045</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.344 ± 0.118</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2.523 ± 0.080</td></tr></table>

Results confirmed the repeatability and precision of the MALDI Biotyper CA System independent from System Operators, microflex LT/SH instruments, Production Lots, Matrix Lots, and BTS Lots.

# Reproducibility

Reproducibility testing was performed at four sites. Sites were provided with a test panel containing 10 organisms. Testing was performed for five days by two operators. Testing via direct transfer and extraction procedure was performed in accordance with MALDI Biotyper CA System instruction for use. Results are summarized below:

<table><tr><td rowspan=1 colspan=1>ReproducibilityPanel</td><td rowspan=1 colspan=2>Gram-negativebacteria</td><td rowspan=1 colspan=1>MBT-CA System IDof samples (DTonly)</td><td rowspan=1 colspan=1>MBT-CA System IDof samples (DT+Ext)</td></tr><tr><td rowspan=1 colspan=1>REPRO-1</td><td rowspan=1 colspan=2>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>77/80 (96%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-2</td><td rowspan=1 colspan=2>Citrobacter koseri</td><td rowspan=1 colspan=1>80/80 (100%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-3</td><td rowspan=1 colspan=2>Enterobacter aerogenes</td><td rowspan=1 colspan=1>77/80 (96%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-4</td><td rowspan=1 colspan=2>Escherichia coli</td><td rowspan=1 colspan=1>78/80 (98%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-5</td><td rowspan=1 colspan=2>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>76/80 (95%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-6</td><td rowspan=1 colspan=2>Morganella morganii</td><td rowspan=1 colspan=1>80/80 (100%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-7</td><td rowspan=1 colspan=2>Pasteurella multocida</td><td rowspan=1 colspan=1>79/80 (99%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-8</td><td rowspan=1 colspan=2>Proteus mirabilis</td><td rowspan=1 colspan=1>80/80 (100%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-9</td><td rowspan=1 colspan=2>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>78/80 (98%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr><tr><td rowspan=1 colspan=1>REPRO-10</td><td rowspan=1 colspan=1>Salmonella sp</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>78/80 (98%)</td><td rowspan=1 colspan=1>80/80 (100%)</td></tr></table>

$100 \%$ of all test organisms were correctly identified on the species level at each test site after final extraction testing confirming reproducibility of the MALDI Biotyper CA System.

E $b$ Linearity/assay reportable range: Not applicable, qualitative assay.   
cTraceability, Stability, Expected values (controls, calibrators, or methods):

# Calibrator:

US IVD BTS is used for mass spectrum calibration and optimization as well as a performance control for the identification of microorganisms with the MALDI Biotyper CA System. US IVD BTS contains a manufactured extract of Escherichia coli DH5 alpha that demonstrates a characteristic peptide and protein profile mass spectrum, when tested on the MALDI Biotyper CA System. US IVD BTS is spiked with two additional proteins that extend the upper boundary of the mass range of the US IVD BTS. The overall mass range covered by US IVD BTS is 3.6 to $1 7 \ k D \mathbf { a } .$ Two US IVD BTS control positions on a US IVD 48 Spot Target are selected and inoculated with US IVD BTS solution. The US IVD BTS solution is allowed to dry at room temperature and then overlayed with reconstituted US IVD HCCA portioned solution. If US IVD BTS does not meet all required performance specifications, the test run will be invalid. If US IVD BTS is not inoculated onto a target prior to processing, the test run will be invalid.

# Controls:

Klebsiella pneumoniae, Haemophilus influenzae, Proteus vulgaris, Pseudomonas aeruginosa, and Esherichia coli, was used as controls. Of the 193 Quality Control runs conducted during the course of the method comparison study, there were only five instances where a Quality Control organism failed to yield an expected result. As a result, all isolates included in that plate run were repeated using a fresh QC organism transfer. The overall plate repeat rate was $2 . 6 \%$ (5/193).

# d. Assay cut-off:

Using statistical analysis, a probability ranking of the organism identification is generated. The probability ranking is represented as a log(score) between 0.00 and 3.00. Oranism enfiation dire  extracte) irepor wit ig cdenc log(score) is $\ge 2 . 0 0$ .If a direct transfer organism identification log(score) is $< 2 . 0 0$ ; the user is instructed to follow the extraction procedure.

After extraction:

If the organism identification log(score) is between 1.7 and $< 2 . 0$ , the identification is reported as low confidence. If the organism identification log(score) is $< 1 . 7$ , it is reported as 'No Identification'.

# e. Detection limit:

The Limit of Detection study was designed to establish the estimated dynamic range of sample size for both the Direct Transfer and Extraction method procedure. Seven frequently occurring clinically relevant test organisms were chosen for this study.Cell density and cell concentration were estimated by measuring the optical density of the suspension at a wavelength of $6 0 0 \mathrm { n m }$ Approximately $3 \mathrm { { x } 1 0 ^ { 8 } \ c e l l s / m L }$ were reporte  correspond optical density of $\mathrm { O D } _ { 6 0 0 } = 1$ according to the McFarland Standard. All suspensions were tested in duplicate. Each cell stock was diluted to a minimum five (5) concentrations and tested in duplicate. A cell concentration was considered within the dynamic range if the

MALDI Biotyper CA System correctly identified the organism for both replicates with a log(score) of $\geq 2 . 0 0$ .

Study results concluded that the estimated dynamic range for the direct and extracted method is as follows:

<table><tr><td rowspan=1 colspan=1>Technique</td><td rowspan=1 colspan=1>Lower limitcells/uL</td><td rowspan=1 colspan=1>Upper limitcells/uL]</td></tr><tr><td rowspan=1 colspan=1>Direct Transfer</td><td rowspan=1 colspan=1>6.3x103-1.4x104</td><td rowspan=1 colspan=1>1.4x106≥6.5x107</td></tr><tr><td rowspan=1 colspan=1>Extraction</td><td rowspan=1 colspan=1>9.0x103-1.3x10^</td><td rowspan=1 colspan=1>1.1x107≥6.9x107</td></tr></table>

# f. Analytical specificity:

Database development: The MALDI Biotyper CA System reference library was established by analyzing the type strain from each claimed species combined with 5 to 10 additional strains from the same species provided by clinical laboratories or different commercial strain collections for a total of 528 strains. Matches are calculated by comparing a new spectrum against each single reference entry of a reference database library mass spectra used for matching contain up to 70 peaks. The spectrum of the unknown test organism, acquired through the MALDI Biotyper CA System Software, is electronically transformed into the peak list. Using a biostatistical algorithm, this peak list is compared to reference peak lists of all organisms in the reference library (database) and a log(score) value between 0.00 and 3.00 is calculated.

Analytical Specificity Study: The specificity study was designed to validate the performance of the proposed MALDI Biotyper CA System reference library by ensuring that organisms not included in the reference library would not yield an incorrect identification and would be reported as "No Identification." Additionally, the study was designed to further demonstrate that the MALDI Biotyper CA System identification is not impacted when closely related species not included in the reference library are run on the system.

The study was conducted in two phases. In phase one, organisms currently not included in the initial system reference library were tested via Direct Transfer and extraction method to ensure that the organisms would not be falsely identified by the MALDI Biotpyer CA system.Organisms tested fell into the following five groupings:

Anaerobe bacteria   
Mycobacteria   
Gram-Negative bacteria (not currently claimed within the library)   
Gram-Positive bacteria   
Yeast species

Results from this phase are presented below:

Phase 1: Summary Results   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2># of &quot;NoIdentification&quot;</td><td rowspan=2 colspan=1># of falseidentification</td></tr><tr><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>Ext</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>DSM 2151</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>DSM 9669</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Prevotella copri</td><td rowspan=1 colspan=1>DSM 18205</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Prevotella buccae</td><td rowspan=1 colspan=1>DSM 19025</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium fortuitum ssp. fortuitum</td><td rowspan=1 colspan=1>DSM 43477</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium fortuitum ssp. Fortuitum</td><td rowspan=1 colspan=1>DSM 46621</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>DSM 9188</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>DSM 15130</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Erwinia tasmaniensis</td><td rowspan=1 colspan=1>DSM 17949</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Erwinia tasmaniensis</td><td rowspan=1 colspan=1>DSM 17950</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Vagococcus fluvialis</td><td rowspan=1 colspan=1>DSM 5731T</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Vagococcus fluvialis</td><td rowspan=1 colspan=1>DSM 21402</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Facklamia hominis</td><td rowspan=1 colspan=1>CCUG 59179</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Facklamia hominis</td><td rowspan=1 colspan=1>CCUG 49614</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Guehomyces pullulans</td><td rowspan=1 colspan=1>CBS 2532</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Guehomyces pullulans</td><td rowspan=1 colspan=1>CBS 2542</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Cyberlindnera mississippiensis</td><td rowspan=1 colspan=1>CBS 7023</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Cyberlindnera mississippiensis</td><td rowspan=1 colspan=1>CBS 7027</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr></table>

In phase two of the study testing, Burkholderia cepacia/multivorans/gladioli were investigated via direct transfer and extraction method to ensure that closely related organisms can be differentiated when tested on the MALDI Biotyper CA System. Results from this phase of testing are reported below:

Phase 2: Summary Results   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2># of CorrectIdentifications</td><td rowspan=2 colspan=1># of falseidentification</td></tr><tr><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>Ext</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>DSM 9241</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>DSM 50181</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Burkholderia multivorans</td><td rowspan=1 colspan=1>1A11237234_4v MVD</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Burkholderia multivorans</td><td rowspan=1 colspan=1>H480 MCRF</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Burkholderia gladioli</td><td rowspan=1 colspan=1>DSM 8361</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Burkholderia gladioli</td><td rowspan=1 colspan=1>LMG 6956</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr></table>

Phase 1 data demonstrates with high confidence that Anaerobes, Mycobacteria, Gram-negative, Grampositive and Yeast organisms not included in the MALDI Biotyper CA System database are not identified confirming the specificity of the MALDI Biotyper CA System reference library when following product instructions for use for both DT and extraction method. Phase 2 data confirms that closely related species can be unambiguously identified by the MALDI Biotyper CA System.

# g. Sample stability studies:

Sample Stability after Matrix Overlay: This study was conducted to prove the stability of test organisms on the spotted target plate following matrix addition at various temperature and relative humidity conditions. In addition, the study served to prove that matrix alone will not influence MALDI Biotyper CA System identification. Three gram negative target organisms were cultured on Columbia Blood Agar (CBA) and aging experiments were done at two different temperature and relative humidity testing conditions. For each condition, two target plates were inoculated and each contained four target spots of directly transferred test organism, four spots of extracted test organism and eight spots of matrix solution alone. All spots containing test organism were then overlaid with matrix in accordance with product instructions for use and tested immediately and then stored at one of the testing conditions and retested at $4 { \pm } 1$ hour, $8 { \pm } 1$ hour and $2 4 \pm 1$ hour.

Summary of Sample Stability overlaid with Matrix   

<table><tr><td colspan="1" rowspan="1">Test Condition</td><td colspan="1" rowspan="1">Test Age</td><td colspan="1" rowspan="1">Test Organism Correct Identification</td><td colspan="1" rowspan="1">Matrix "No PeaksFound"</td></tr><tr><td colspan="1" rowspan="4">DT20 ± 1°C, 40 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="2" rowspan="1"></td><td></td><td></td></tr><tr><td colspan="1" rowspan="4">Ext20 ± 1°C, 40 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">DT20 ± 1°C, 70 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="4">Ext20 ± 1°C, 70 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td></tr><tr><td colspan="1" rowspan="4">DT25 ± 1°C, 30 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Ext25 ± 1°C, 30 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td></tr><tr><td colspan="1" rowspan="4">DT25 ± 1°C, 70 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">18/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="4">Ext25 ± 1C, 70 ± 5%</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">8/8</td></tr></table>

All study results confirmed that inoculated test organisms once overlaid with matrix are stable for up to 24 hours at room temperature. In addition, matrix alone does not interfere or influence MALDI Biotyper CA System identification.

# h. Stability studies (reagent/target plates):

US IVD Bacterial Test Standard (BTS)

BTS was assessed using three lots of BTS material. A shipping/accelerated stability study confirmed that BTS is stable for three weeks at temperatures up to $3 7 { \pm } 2 ^ { \mathrm { { \circ } } } \mathrm { { C } }$ .(Two of these weeks account for the shipping and one for the long-term storage.Real-time stability testing confirms that BTS is stable for up to 12 months. In-use stability confirms that reconstituted BTS is stable for five (5) months when stored in accordance with product claims.

# HCCA portioned (Matrix) Stability

HCCA portioned (Matrix) is used when processing test organisms for identification on the MALDI Biotyper CA System. Matrix must be reconstituted prior to use. Studies were conducted to determine the stability of un-reconstituted matrix as well as in-use (reconstituted) matrix. Shipping/accelerated stability studies confirm that matrix is stable for sixteen weeks at temperatures up to $3 7 \pm 2 ^ { \mathrm { { \circ } } } \mathrm { { C } }$ . Real-time stability studies confirm that matrix is stable for up to 18 months when stored in accordance with product instructions for use. Inuse (reconstituted) stability testing confirmed that reconstituted matrix is stable for one week when stored at controlled room temperature and for 12 hours when stressed by temperatures up to $1 5 \mathrm { { ^ \circ C } }$ or $3 0 \mathrm { { } ^ { \circ } C }$ .

Target plates stability: Four representative MSP target polished steel plates were tested over a two year period with approximately two applications per week. Target plates were used in accordance with instructions included in the user manual. No damage to the target plate was noted. No bleeding of spots was observed. The target plates do not have a shelf-life; but rather can be used until physical damage is observed.

# i. Carry-Over and Cross Contamination:

This study was conducted to determine the effect of carry over and cross-contamination, defined as microbial sample convergence between adjacent blank target spots. Two targets and two frequently occurring Gram negative bacteria were chosen for this testing. Each target was inoculated with test organism four times via direct transfer and extraction method in an alternating pattern. Allsample positions were overlaid with matrix solution including the remaining unused target positions to serve as blank measurements. Testing was performed in accordance with product instructions for use. Targets were then cleaned in accordance with the Target Cleaning procedure and organism prepared in a similar fashion but in the reverse pattern. The test cycle described above was repeated four times on each

target plate.

No cross-contamination or carry-over effects were seen in the automated MALDI Biotyper CA System identification process.

# $j$ Influence of Agar Media

This study was completed in order to demonstrate that impurities such as salts, peptides or carbohydrates introduced from culture media do not interfere with MALDI Biotyper CA System identification. In addition, the study set-out to prove that isolation media alone would not generate mass spectra leading to false identification on the MALDI Biotyper CA System. TSA, CBA, MAC and CHOC agars were tested by the following methods:

Each agar media was inoculated using the direct transfer and extraction method alone six times each. Three frequently occurring Enterobacteriaceae and non-fermenting Gram Negative Bacteria were transferred to the target plate in duplicate via DT and Ext method to serve as a control. Each target organism was then inoculated in duplicate via DT and Ext method such that a sample agar media was included with the isolate.

Summary of Influence of Agar Media Study   

<table><tr><td rowspan=2 colspan=1>Media</td><td rowspan=1 colspan=2>Agar Alone</td><td rowspan=1 colspan=2>Target Organism Alone</td><td rowspan=1 colspan=2>Target Organism + Agar</td></tr><tr><td rowspan=1 colspan=1># replicates</td><td rowspan=1 colspan=1>% No ID</td><td rowspan=1 colspan=1># replicates</td><td rowspan=1 colspan=1>% False ID</td><td rowspan=1 colspan=1># replicates</td><td rowspan=1 colspan=1>% False ID</td></tr><tr><td rowspan=1 colspan=1>TSA</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10/12</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CBA</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MAC</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CHOC</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0</td></tr></table>

The study confirms that the media recommended for use on the MALDI MALDI Biotyper CA System do not interfere with MALDI Biotyper CA System performance or organism identification.

# $k .$ Organism Stability

Media and Colony Stability

In accordance with device instructions for use, primary or secondary isolation plates of recommended media [Trypticase Soy Agar with $5 \%$ sheep blood (TSA), Columbia Blood Agar with $5 \%$ sheep blood (CBA), MacConkey Agar (MAC), and Chocolate Agar (Choc)] may be held for up to 12 hours at room temperature prior to testing on the MALDI Biotyper CA System.

Testing was conducted using seven gram-negative organisms at two different incubation time points (18h, 24h). After initial incubation, isolates were further tested at two temperatures ( $1 8 ^ { \circ } \mathrm { C }$ $2 5 ^ { \circ } C$ for 12 hours post-incubation.

Summary of Media and Colony Stability Study   

<table><tr><td rowspan=1 colspan=1>Media</td><td rowspan=1 colspan=1>≥2.0 Identification(DT)</td><td rowspan=1 colspan=1>False Identification(DT)</td><td rowspan=1 colspan=1>≥2.0 Identification(Ext)</td><td rowspan=1 colspan=1>False Identification(Ext)</td></tr><tr><td rowspan=1 colspan=1>TSA</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td></tr><tr><td rowspan=1 colspan=1>CBA</td><td rowspan=1 colspan=1>284/288</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td></tr><tr><td rowspan=1 colspan=1>MAC</td><td rowspan=1 colspan=1>263/288</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td></tr><tr><td rowspan=1 colspan=1>CHOC</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>0/288</td></tr></table>

The study results confirm that the following culture media can be used on the MALDI Biotyper CA System:

Trypticase Soy Agar with $5 \%$ sheep blood (TSA) • Columbia Blood Agar with $5 \%$ sheep blood (CBA) • MacConkey Agar (MAC) • Chocolate Agar (CHOC)

Study results conclude that sample colony is stable for up to 18-36 hours.

# Organism Stability prior to MALDI Biotyper CA System Analysis

This study was conducted to assess isolate stability on the target plate prior to matrix overlay via direct transfer and extraction method. In addition, the study set out to confirm the stability of extracted material prior to target plate inoculation.

To test for isolate stability on the target plate prior to matrix overlay via DT, three common gram negative bacteria were inoculated eight times and overlaid with matrix at five different time points. After matrix overlay, isolates were tested in accordance with product instructions. For the Ext method, colonies were prepared following the extraction technique per product instructions for use. Extracts were overlaid with matrix at five different time points and tested per product instructions. For the third phase of testing, the three gram negative isolates were extracted twice. The extracts were stored at controlled room temperature for up to 24 hours and tested at five time points in replicates of eight.

Summary of Organism Stability Prior to MALDI Biotyper CA System Analysis Stud   

<table><tr><td colspan="1" rowspan="1">Test Phase</td><td colspan="2" rowspan="1">Testing Condition</td><td colspan="1" rowspan="1">Measurands</td><td colspan="1" rowspan="1">CorrectIdentification</td><td colspan="1" rowspan="1">FalseIdentification</td></tr><tr><td colspan="1" rowspan="5">Direct Transfer (DT)</td><td colspan="1" rowspan="1">0 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">15 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">30 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">60 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">120 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="5">Extraction Method(Ext)</td><td colspan="1" rowspan="1">0 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">0/6</td></tr><tr><td colspan="1" rowspan="1">10 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">0/6</td></tr><tr><td colspan="1" rowspan="1">20 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">0/6</td></tr><tr><td colspan="1" rowspan="1">30 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">0/6</td></tr><tr><td colspan="1" rowspan="1">60 min</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">0/6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="5">Extract #1</td><td colspan="2" rowspan="1">0 hour</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="2" rowspan="1">1 hour</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="2" rowspan="1">4 hours</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="2" rowspan="1">8 hours</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="2" rowspan="1">24 hours</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="5">Extract #2</td><td colspan="1" rowspan="1">0 hour</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">1 hour</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">4 hours</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">8 hours</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr><tr><td colspan="1" rowspan="1">24 hours</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">0/24</td></tr></table>

Study results confirm that samples are stable on the target plate when tested via DT or Ext method for up to sixty minutes prior to analysis. In addition, extracts are stable for up to 24 hours when stored at room temperature.

l. Other supportive Instrument Performance Characteristics

# Mixed Culture:

To assess the effect of testing a mixed culture on MALDI Biotyper CA System identification, Pseudomonas aeruginosa was used as the target; four non-target organisms consisting of gram-negative and gram-positive bacteria were introduced with the target organisms at varying concentrations to determine the affect a mixed culture would have on MALDI Biotyper CA System identification.

Table 6: Summary of Mixed Culture Study   

<table><tr><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=1>Target OrganismAmount</td><td rowspan=1 colspan=1>Non-TargetOrganism Amount</td><td rowspan=1 colspan=1># of MBT-CA System FalseIdentifications</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0/32</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>0/32</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>0/32</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>0/32</td></tr></table>

Although system users are instructed to test a single isolated colony on the MALDI Biotyper CA System, this study demonstrated that analyzing a mixed culture on the system, no false results are obtained.

# Viability Study

Viability studies with gram negative rods mixed with matrix on the target plate were not performed. The user is advised to consider all samples, microbial cultures and inoculated products as infectious. Aseptic techniques and usual precautions for handling organisms should be observed throughout the MALDI Biotyper CA System workflow according to "CLSI M29-A, Protection of Laboratory Workers From Occupationally Acquired Infections; Approved Guideline - Current revision". For additional handling precautions, refer to

"Biosafety in Microbiological and Biomedical Laboratories - CDC/NIH -Latest edition".

Run Failures from Clinical Trial:

Of the 193 Quality Control runs conducted during the course of the method comparison study, there were only five instances where a Quality Control organism failed to yield an expected result. As a result, all isolates included in that plate run were repeated using a fresh QC organism transfer. The overall plate repeat rate was $2 . 6 \%$ (5/193).

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies section of this document.

b. Matrix comparison: Not applicable

3.Clinical studies:

a.Clinical Sensitivity:

# Proficiency/Familiarity

Prior to method comparison study initiation, each study operator from the four US Study sites underwent a proficiency/familiarity period to ensure that each operator was familiar with all aspects of instrument operation. Each operator was asked to test five QC organisms using both the Direct Transfer (DT) and Extraction (Ext) method following product instructions for use.

# Challenge Panel

To demonstrate intra-laboratory performance, a challenge panel of 100 organisms was provided to five test sites. Each site tested the challenge panel in accordance with MALDI Biotyper CA System instructions for use. The following is a summary of results:

<table><tr><td rowspan=1 colspan=1>Test procedure</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>*Site 4</td><td rowspan=1 colspan=1>**Site 5</td></tr><tr><td rowspan=1 colspan=1>DT only</td><td rowspan=1 colspan=1>99/100(99%)</td><td rowspan=1 colspan=1>98/100(98%)</td><td rowspan=1 colspan=1>99/100(99%)</td><td rowspan=1 colspan=1>96/100(96%)</td><td rowspan=1 colspan=1>86/87 (99%)</td></tr><tr><td rowspan=1 colspan=1>MBT-CA Systemworkflow (+ Ext)</td><td rowspan=1 colspan=1>99/100(99%)</td><td rowspan=1 colspan=1>99/100(99%)</td><td rowspan=1 colspan=1>99/100(99%)</td><td rowspan=1 colspan=1>97/100(97%)</td><td rowspan=1 colspan=1>86/87 (99%)</td></tr></table>

\*O    o  a culture.

# Prospective Clinical Study:

The following is a summary of a clinical study performed at five sites to confirm product performance. Testing was performed using direct transfer of organism to target plate. If the MALDI Biotyper CA System identification of the test organism did not result in a bacterial identification with a log(score) value of $\geq 2 . 0$ , repeat testing was performed using the extraction procedure. Only $6 . 8 \%$ of all clinical results needed extraction. MALDI Biotyper CA System identifications were compared to sequencing in accordance with MM18-A using GenBank and/or EzTaxon. If no match was observed, biochemical testing and protein target sequencing was applied. The following tables summarize MALDI Biotyper CA System results for all organisms included in the current reference library. In total, 2263 fresh and stored isolates were tested to support the initial reference library claims. Site summaries and an overall performance table is presented below:

# The following comments apply to site performance summaries below:

1) THE MBT-CA System reported 5 times E.cloacae, 8 times H.influenzae, and once Y.enterocolitica with high confidence [log(score $) 2 . 0 ]$ whereas the reference method reported something else. (See Table la, 1b, and 2)   
2) The MBT-CA System reported 2 times P.stuartii, 20 times H.influenzae with low confidence EY $[ 1 0 \mathbf { g } ( \mathsf { s c o r e } ) 2 \mathsf { l } . 7 ; - 2 . 0 ]$ whereas the reference method reported something else. (See Table la and 1b)   
3) THE MBT-CA System reported 2 times C.amalonaticus with high confidence [log(score)≥2.0] whereas the reference method did not report a reliable result. (See Table la and 1b)   
4) The MBT-CA System reported once A.xylosoxidans with low confidence $[ \log ( \tt s c o r e ) 2 1 . 7 ; \exp$ whereas the reference method reported something else. (See Table la and 1b)   
5) The MBT-CA System reported "no result" [log(score)<1.7] for one $P .$ putida and one S.maltophilia identified by the reference method. (See Table 2)   
6) n/a - Not Applicable. There are no "correct negatives" in the performance table for the whole study data. There are no samples where the reference method is negative and the MBT-CA System is negative, too. Samples with negative reference identification and negative MBT-CA System identification were excluded from the study.   
7) MBT-CA System identifications were compared to sequencing in accordance with MM18-A using GenBank and/or EzTaxon.If no match was observed, biochemical testing and protein target sequencing was applied.   
8) 18 strains of Haemophilus haemolyticus were collected and soley tested at Site 5 to determine whether H. haemolyticus could be differentiated from Haemophilus influenzae. As these species could not be differentiated, the performance of this site was significantly decreased

<table><tr><td rowspan=2 colspan=1>All Isolates - ALL SITES</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td rowspan=1 colspan=1>2174</td><td rowspan=1 colspan=1>161)+3)</td><td rowspan=1 colspan=1>2190</td></tr><tr><td rowspan=1 colspan=1>Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>$23²2}+4</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>$2</td><td rowspan=1 colspan=1>n/a^6)</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2224</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>2263</td></tr></table>

<table><tr><td rowspan=2 colspan=1>GramnegativesALLSITES</td><td rowspan=1 colspan=2>Correct:Genus IDCorrect:Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=2>Correct: Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=2>Incorrect Genus ID</td><td rowspan=1 colspan=1>no ID</td></tr><tr><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7.&lt;2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7.. &lt;2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7... &lt;2.0</td><td rowspan=1 colspan=1>log(score)&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>IDs</td><td rowspan=1 colspan=1>2174 /226396.07%</td><td rowspan=1 colspan=1>48 / 22632.12%</td><td rowspan=1 colspan=1>141) / 22630.62%</td><td rowspan=1 colspan=1>22) / 22630.97%</td><td rowspan=1 colspan=1>23) / 22630.09%</td><td rowspan=1 colspan=1>14) / 22630.04%</td><td rowspan=1 colspan=1>2) / 22630.09%</td></tr><tr><td rowspan=1 colspan=1>CombinedIDs</td><td rowspan=1 colspan=2>2222 / 226398.19%95% CI [95.04 ; 100.00]</td><td rowspan=1 colspan=2>36 / 22631.59%95% CI [0.00 ; 4.81]</td><td rowspan=1 colspan=2>3/22630.13%95% CI [0.01 ; 0.25]</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=2 colspan=1>All Isolates — Site 1</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td rowspan=1 colspan=1>801</td><td rowspan=1 colspan=1>51) +3</td><td rowspan=1 colspan=1>806</td></tr><tr><td rowspan=1 colspan=1>Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>$1^{2)+4</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>$n/a$</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>818</td></tr></table>

<table><tr><td rowspan=2 colspan=1>GramnegativesSite 1</td><td rowspan=1 colspan=2>Correct: Genus IDCorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=2>Correct:Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=2>Incorrect:Genus ID</td><td rowspan=1 colspan=1>no ID</td></tr><tr><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7... 2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7.. 2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7... &lt;2.0</td><td rowspan=1 colspan=1>log(score)&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>IDs</td><td rowspan=1 colspan=1>801/81897.92%</td><td rowspan=1 colspan=1>11/8181.34%</td><td rowspan=1 colspan=1>3/8180.37%</td><td rowspan=1 colspan=1>1/8180.12%</td><td rowspan=1 colspan=1>2/8180.24%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CombinedIDs</td><td rowspan=1 colspan=2>812/81899.27%</td><td rowspan=1 colspan=2>4/8180.49%</td><td rowspan=1 colspan=2>2/8180.24%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=2 colspan=5>All Isolates - Site 2</td><td rowspan=1 colspan=6>REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=5>Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td rowspan=1 colspan=2>432</td><td rowspan=1 colspan=2>21+3)</td><td rowspan=1 colspan=2>434</td></tr><tr><td rowspan=1 colspan=5>Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td rowspan=1 colspan=2>9</td><td rowspan=1 colspan=2>$2}+4</td><td rowspan=1 colspan=2>11</td></tr><tr><td rowspan=1 colspan=5>Negative</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>n/a^6)</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=5>Total</td><td rowspan=1 colspan=2>441</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>445</td></tr><tr><td rowspan=2 colspan=1>GramnegativesSite 2</td><td rowspan=1 colspan=2>Correct: Genus IDCorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=3>Correct: Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=4>Incorrect:Genus ID</td><td rowspan=1 colspan=1>no ID</td></tr><tr><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7.. &lt;2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=2>lowconfidencelog(score)≥1.7..2.0</td><td rowspan=1 colspan=2>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=2>lowconfidencelog(score)≥1.7..2.0</td><td rowspan=1 colspan=1>log(score)&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>IDs</td><td rowspan=1 colspan=1>432 / 44597.08%</td><td rowspan=1 colspan=1>9 /4452.02%</td><td rowspan=1 colspan=1>2/4450.45%</td><td rowspan=1 colspan=2>2 / 4450.45%</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CombinedIDs</td><td rowspan=1 colspan=2>441 /44599.10%</td><td rowspan=1 colspan=3>4/4450.90%</td><td rowspan=1 colspan=4>0</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td colspan="5" rowspan="2">All Isolates - Site 3</td><td colspan="6" rowspan="1">REFERENCE ALGORITHM</td></tr><tr><td colspan="2" rowspan="1">Positive</td><td colspan="2" rowspan="1">Negative</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="5" rowspan="1">Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td colspan="2" rowspan="1">355</td><td colspan="2" rowspan="1">11)+3</td><td colspan="2" rowspan="1">356</td></tr><tr><td colspan="5" rowspan="1">Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td colspan="2" rowspan="1">7</td><td colspan="2" rowspan="1">12)+4</td><td colspan="2" rowspan="1">8</td></tr><tr><td colspan="5" rowspan="1">Negative</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">n/a)</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">Total</td><td colspan="2" rowspan="1">362</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">364</td></tr><tr><td colspan="1" rowspan="2">GramnegativesSite 3</td><td colspan="2" rowspan="1">Correct: Genus IDCorrect: Species ID orGroup ID orComplex ID</td><td colspan="3" rowspan="1">Correct: Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td colspan="4" rowspan="1">Incorrect:Genus ID</td><td colspan="1" rowspan="1">no ID</td></tr><tr><td colspan="1" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="1" rowspan="1">lowconfidencelog(score)≥1.7...2.0</td><td colspan="1" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="2" rowspan="1">lowconfidencelog(score)≥1.7.… &lt;2.0</td><td colspan="2" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="2" rowspan="1">lowconfidencelog(score)≥1.7... &lt;2.0</td><td colspan="1" rowspan="1">log(score)2.0</td></tr><tr><td colspan="1" rowspan="1">IDs</td><td colspan="1" rowspan="1">355 /36497.53%</td><td colspan="1" rowspan="1">7/3641.92%</td><td colspan="1" rowspan="1">1 /3640.27%</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1 /3640.27%</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CombinedIDs</td><td colspan="2" rowspan="1">362 / 36499.45%</td><td colspan="3" rowspan="1">1 /3640.27%</td><td colspan="4" rowspan="1">1 /3640.27%</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="5" rowspan="2">All Isolates - Site 4</td><td colspan="6" rowspan="1">REFERENCE ALGORITHM</td></tr><tr><td colspan="2" rowspan="1">Positive</td><td colspan="2" rowspan="1">Negative</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="5" rowspan="1">Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td colspan="2" rowspan="1">403</td><td colspan="2" rowspan="1">11)+3</td><td colspan="2" rowspan="1">404</td></tr><tr><td colspan="5" rowspan="1">Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td colspan="2" rowspan="1">17</td><td colspan="2" rowspan="1">12)+4</td><td colspan="2" rowspan="1">18</td></tr><tr><td colspan="5" rowspan="1">Negative</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">n/a^)</td><td colspan="2" rowspan="1">2</td></tr><tr><td colspan="5" rowspan="1">Total</td><td colspan="2" rowspan="1">422</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">424</td></tr><tr><td colspan="1" rowspan="2">GramnegativesSite 4</td><td colspan="2" rowspan="1">Correct: Genus IDCorrect: Species ID orGroup ID orComplex ID</td><td colspan="3" rowspan="1">Correct: Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td colspan="4" rowspan="1">Incorrect:Genus ID</td><td colspan="1" rowspan="1">no ID</td></tr><tr><td colspan="1" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="1" rowspan="1">lowconfidencelog(score)≥1.7.. &lt;2.0</td><td colspan="1" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="2" rowspan="1">lowconfidencelog(score)≥1.7.. &lt;2.0</td><td colspan="2" rowspan="1">highconfidencelog(score)≥2.0</td><td colspan="2" rowspan="1">lowconfidencelog(score)≥1.7.….2.0</td><td colspan="1" rowspan="1">log(score)&lt;2.0</td></tr><tr><td colspan="1" rowspan="1">IDs</td><td colspan="1" rowspan="1">403 / 42495.05%</td><td colspan="1" rowspan="1">17/4244.01%</td><td colspan="1" rowspan="1">1 /4240.24%</td><td colspan="2" rowspan="1">1 /4240.24%</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">2 / 4240.47%</td></tr><tr><td colspan="1" rowspan="1">CombinedIDs</td><td colspan="2" rowspan="1">420 / 42499.06%</td><td colspan="3" rowspan="1">2/4240.48%</td><td colspan="4" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td></tr></table>

<table><tr><td rowspan=2 colspan=5>All Isolates — Site 5</td><td rowspan=1 colspan=6>REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=5>Positive Organism ID; (High Confidence); log(score) ≥2.0</td><td rowspan=1 colspan=2>183</td><td rowspan=1 colspan=2>71)+3)</td><td rowspan=1 colspan=2>190</td></tr><tr><td rowspan=1 colspan=5>Positive Organism ID; (Low Confidence); log(score) ≥1.7;&lt;2.0</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>152+4</td><td rowspan=1 colspan=2>22</td></tr><tr><td rowspan=1 colspan=5>Negative</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>n/a)</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=5>Total</td><td rowspan=1 colspan=2>187</td><td rowspan=1 colspan=2>25</td><td rowspan=1 colspan=2>212</td></tr><tr><td rowspan=2 colspan=1>GramnegativesSite 5</td><td rowspan=1 colspan=2>Correct: Genus IDCorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=3>Correct: Genus IDIncorrect: Species ID orGroup ID orComplex ID</td><td rowspan=1 colspan=4>Incorrect:Genus ID</td><td rowspan=1 colspan=1>no ID</td></tr><tr><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=1>lowconfidencelog(score)≥1.7..&lt;2.0</td><td rowspan=1 colspan=1>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=2>lowconfidencelog(score)≥1.7.… &lt;2.0</td><td rowspan=1 colspan=2>highconfidencelog(score)≥2.0</td><td rowspan=1 colspan=2>lowconfidencelog(score)≥1.7.…..&lt;2.0</td><td rowspan=1 colspan=1>log(score)&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>IDs</td><td rowspan=1 colspan=1>183 /21286.32%</td><td rowspan=1 colspan=1>4/2121.89%</td><td rowspan=1 colspan=1>7/2123.30%</td><td rowspan=1 colspan=2>18 /2128.49%</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CombinedIDs</td><td rowspan=1 colspan=1>187/21288.21%</td><td rowspan=1 colspan=1>187/21288.21%</td><td rowspan=1 colspan=3>25/21211.79%</td><td rowspan=1 colspan=4>0</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Performance and Species Claimed in the MALDI Biotyper CA System Reference Library   

<table><tr><td colspan="3" rowspan="1">EY</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">Het Hh</td><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="2">so tee</td><td colspan="9" rowspan="1">Correct Identification</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Discordant</td></tr><tr><td colspan="3" rowspan="1">SpeciesConfirmationhigh confidencelog(score)≥2.0</td><td colspan="3" rowspan="1">SpeciesConfirmationlow confidencelog(score)≥1.7.… 2.0</td><td colspan="3" rowspan="1">CombinedPerformance</td><td colspan="1" rowspan="1">NoID</td><td colspan="1" rowspan="1">SpeciesConfirmationhigh confidencelog(score)≥2.0</td><td colspan="1" rowspan="1">SpeciesConfirmationlow confidencelogscore)≥1.7.…&lt;2.0</td></tr><tr><td colspan="1" rowspan="1">a</td><td colspan="1" rowspan="1">Achromobacterxylosoxidans</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">70/75</td><td colspan="1" rowspan="1">93%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4/75</td><td colspan="1" rowspan="1">5%</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">74/75</td><td colspan="1" rowspan="1">99%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Acinetobacterbaumanicomplex</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">67/70</td><td colspan="1" rowspan="1">96%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/70</td><td colspan="1" rowspan="1">4%</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">70/70</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Acinetobacterlwoffii</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Acinetobacterradioresistens</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Acinetobacterursingii</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1/50</td><td colspan="1" rowspan="1">2%</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c</td><td colspan="1" rowspan="1">Aeromonas sp</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Alcaligenesfaecalis</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">d</td><td colspan="1" rowspan="1">Burkholderiagladioli</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Burkholderiamultivorans</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">e</td><td colspan="1" rowspan="1">Burkholderiacepaciacomplex</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">f</td><td colspan="1" rowspan="1">Citrobacteramalonaticuscomplex</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">62/64</td><td colspan="1" rowspan="1">97%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">62/64</td><td colspan="1" rowspan="1">97%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Citrobacterkoseri</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">g</td><td colspan="1" rowspan="1">Citrobacterfreeundicomplex</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Eikenellacorrodens</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">h</td><td colspan="1" rowspan="1">Enterobactercloacaecomplex</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">74/95</td><td colspan="1" rowspan="1">78%</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16/95</td><td colspan="1" rowspan="1">17%</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">90/95</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">Escherichiacoli</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">122/122</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">122/122</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">j</td><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63/95</td><td colspan="1" rowspan="1">66%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4/95</td><td colspan="1" rowspan="1">4%</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">67/95</td><td colspan="1" rowspan="1">71%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">20{}</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Haemophilusparainfluenzae</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32/34</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2/34</td><td colspan="1" rowspan="1">6%</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">k</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">101/101</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">101/</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Hrt Hh</td><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="2">so os toe</td><td colspan="9" rowspan="1">Correct Identification</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Discordant</td></tr><tr><td colspan="3" rowspan="1">SpeciesConfirmationhigh confidencelog(score)≥2.0</td><td colspan="3" rowspan="1">SpeciesConfirmationlow confidencelog(score)≥1.7.…2.00</td><td colspan="3" rowspan="1">CombinedPerformance</td><td colspan="1" rowspan="1">ND :. :</td><td colspan="1" rowspan="1">SpeciesConfirmationhigh confidencelog(score)≥2.0   :</td><td colspan="1" rowspan="1">SpeciesConfirmationlow confidencelog(score)≥1.7.…2.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">m</td><td colspan="1" rowspan="1">KlebsiellaoxytocaRaoultellaornithinolytica</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">68/68</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">68/68</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Moraxella_sgBranhamellacatarrhalis</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">66/66</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">66/66</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Moraxella_sg_Moraxellaosloensis</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">p</td><td colspan="1" rowspan="1">Pantoeaagglomerans</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Pasteurellamultocida</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Proteusmirabilis</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">q</td><td colspan="1" rowspan="1">Proteusvulgarisgroup</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">r</td><td colspan="1" rowspan="1">Providenciarettgeri</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50/55</td><td colspan="1" rowspan="1">91%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5/55</td><td colspan="1" rowspan="1">9%</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">54/56</td><td colspan="1" rowspan="1">96%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">54/56</td><td colspan="1" rowspan="1">96%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">78/78</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">78/78</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">Pseudomonasfluorescensgroup</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">t</td><td colspan="1" rowspan="1">Pseudomonasputidagroup</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">47/61</td><td colspan="1" rowspan="1">77%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13/61</td><td colspan="1" rowspan="1">21%</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60/61</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">u</td><td colspan="1" rowspan="1">Salmonella sp</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">86/86</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">86/86</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">Serratialiquefaciens</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">w</td><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">75/76</td><td colspan="1" rowspan="1">99%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">75/76</td><td colspan="1" rowspan="1">99%</td><td colspan="1" rowspan="1">13)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">43/44</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">43/44</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">11)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">y</td><td colspan="1" rowspan="1">Yersiniapseudotuberculosis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

A150 26321744822221623

Fordetailed information please reer to table ; below. F

Table 1A: Summary of Incorrect Identifications   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>noID</td><td rowspan=1 colspan=2>Discordant</td></tr><tr><td rowspan=1 colspan=1>SpeciesConfirmationhigh confidence low confidencelog(score)≥2.0</td><td rowspan=1 colspan=1>SpeciesConfirmationhigh confidence low confidencelog(score)≥1.7.…&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>Achromobacter xylosoxidans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Citrobacteramalonaticus complex</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae_complex</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Providencia rettgeri</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr></table>

Table 1B: Details of Incorrect Identifications   

<table><tr><td rowspan=1 colspan=1>EMPY</td><td rowspan=1 colspan=1>MBT-CA System result</td><td rowspan=1 colspan=1>Log(score)</td><td rowspan=1 colspan=1>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Achromobacter xylosoxidans</td><td rowspan=1 colspan=1>Achromobacter xylosoxidans</td><td rowspan=1 colspan=1>1.810</td><td rowspan=1 colspan=1>Bordetella bronchiseptica</td></tr><tr><td rowspan=2 colspan=1>C. amalonaticus_complex</td><td rowspan=1 colspan=1>C. amalonaticus complex</td><td rowspan=1 colspan=1>2.584</td><td rowspan=1 colspan=1>Ref. Method did notconfirm any species</td></tr><tr><td rowspan=1 colspan=1>C. amalonaticus complex</td><td rowspan=1 colspan=1>2.513</td><td rowspan=1 colspan=1>Ref. Method did notconfirm any species</td></tr><tr><td rowspan=5 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>2.240</td><td rowspan=1 colspan=1>Enterobacter amnigenus</td></tr><tr><td rowspan=1 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>2.461</td><td rowspan=1 colspan=1>Enterobacter amnigenus</td></tr><tr><td rowspan=1 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>2.490</td><td rowspan=1 colspan=1>Enterobacter amnigenus</td></tr><tr><td rowspan=1 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>2.447</td><td rowspan=1 colspan=1>Enterobacter amnigenus</td></tr><tr><td rowspan=1 colspan=1>E. cloacae_complex</td><td rowspan=1 colspan=1>2.415</td><td rowspan=1 colspan=1>Enterobacter amnigenus</td></tr><tr><td rowspan=2 colspan=1>Providencia stuartii / rettgeri</td><td rowspan=1 colspan=1>P.stuartii</td><td rowspan=1 colspan=1>1.806</td><td rowspan=1 colspan=1>Providencia rettgeri</td></tr><tr><td rowspan=1 colspan=1>P.stuartii</td><td rowspan=1 colspan=1>1.899</td><td rowspan=1 colspan=1>Providencia rettgeri</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>2.043</td><td rowspan=1 colspan=1>Yersinia aldovae*</td></tr></table>

Yalv      Yl o X.

Table 2: Summary of Incorrect Identifications (Haemophilus haemolyticus / influenzae)   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>sojs te #i</td><td rowspan=1 colspan=9>Correct Identification</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Discordant</td></tr><tr><td rowspan=1 colspan=3>SpeciesConfirmationhigh confidencelog(score)≥2.0</td><td rowspan=1 colspan=3>SpeciesConfirmationlow confidencelog(score)≥1.7.…2.0</td><td rowspan=1 colspan=3>CombinedPerformance</td><td rowspan=1 colspan=1>NoID</td><td rowspan=1 colspan=1>SpeciesConfirmationhigh confidencelog(score)≥2.0</td><td rowspan=1 colspan=1>SpeciesConfirmationlow confidencelog(score)≥1.7..&lt;2.0</td></tr><tr><td rowspan=1 colspan=1>Haemophilushaemolyticus</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63/95</td><td rowspan=1 colspan=1>66%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4/95</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67/95</td><td rowspan=1 colspan=1>71%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>20</td></tr></table>

Haemophilus haemolyticus was removed from the claimed organisms during the clinical study.   
Therefore, it has to be considered to be falsely identified as H. influenza; see matching hint j.

Table 3: Details of not Identified strains   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MBT-CASystemresult</td><td rowspan=1 colspan=1>log(score)</td><td rowspan=1 colspan=1>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas putida_group</td><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>1.530</td><td rowspan=1 colspan=1>Pseudomonas putida group</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>1.638</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Nr H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1">a</td><td colspan="1" rowspan="1">Achromobacterxylosoxidans</td><td colspan="1" rowspan="1">A. xylosoxidans</td><td colspan="1" rowspan="1">A.xylosoxidans,A.denitrificans, A.insolitus,A.marplatensis, A.ruhlandii,A.spanius</td><td colspan="1" rowspan="1">All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Acinetobacterbaumannii complex_[4]</td><td colspan="1" rowspan="1">A. baumanniiA. calcoaceticusA. pittiiA. nosocomialis</td><td colspan="1" rowspan="1">A.baumannii,A.calcoaceticus, A.pittii,A.nosocomialis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">A. Iwoffii</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Acinetobacterradioresistens</td><td colspan="1" rowspan="1">A. radioresistens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">A. ursingii</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">c</td><td colspan="1" rowspan="1">Aeromonas sp</td><td colspan="1" rowspan="1">A. allosaccharophilaA. caviaeA. culicicolaA. hydrophilaA. ichthiosmiaA. sobriaA. veronii</td><td colspan="1" rowspan="1">A.allosaccharophila,A.aquariorum, A.caviae,A.culicicola,A.enteropelogenes,A.fluvialis, A.hydrophila,A.ichthiosmia, A.jandaei,A.media, A.punctata,A.rivuli, A.sanarellii,A.sobria, A.taiwanensis,A.veronii.</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.A.hydrophila, A. caviae andA.sobria (A.veronii bv sobria)are the most frequentlyreported species associatedwith human infection.A.ichthiosmia is considered asynonym of A.veronii andA.punctata is considered asynonym of A.caviae.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">A. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">d</td><td colspan="1" rowspan="1">Burkholderia gladioli</td><td colspan="1" rowspan="1">B. gladioli</td><td colspan="1" rowspan="1">B.gladioli, B.glumae,B.caryopylii</td><td colspan="1" rowspan="1">B. glumae and B. caryopyliihave not been reported asisolated with human specimens.</td></tr><tr><td colspan="1" rowspan="1">W M</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="2" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Burkholderia multivorans</td><td colspan="2" rowspan="1">B. multivorans</td><td colspan="1" rowspan="1">B.multivorans</td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">e</td><td colspan="1" rowspan="1">Burkholderia_cepaciacomplex</td><td colspan="2" rowspan="1">B. ambifariaB. anthinaB. cenocepaciaB. cepaciaB. difffusaB. dolosaB. lataB. latensB. metallicaB. pyrrociniaB. seminalisB. stabilisB. vietnamiensis</td><td colspan="1" rowspan="1">B.ambifaria, B.anthina,B.cenocepacia, B.cepacia,B.diffusa, B.dolosa, B.lata,B.latens, B.metallica,B.pyrrocinia, B.seminalis,B.stabilis, B.vietnamiensis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">f</td><td colspan="1" rowspan="1">Citrobacteramalonaticus_complex</td><td colspan="2" rowspan="1">C. amalonaticusC. farmeri</td><td colspan="1" rowspan="1">C.amalonaticus, C.farmeri</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together. Both speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="2" rowspan="1">C. koseri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">g</td><td colspan="1" rowspan="1">Citrobacter _freundiicomplex</td><td colspan="2" rowspan="1">C. braakiiC. freundiiC. gilleniiC. murliniaeC. rodentiumC. sedlakiiC. werkmanniiC. youngae</td><td colspan="1" rowspan="1">C.braakii, C.freundii,C.gillenii, C.murliniae,C.rodentium, C.sedlakii,C.werkmannii, C.youngae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.C. rodentium has not beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">E. corrodens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">W Wh</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1">h</td><td colspan="1" rowspan="1">Enterobactercloacae_complex</td><td colspan="1" rowspan="1">E. asburiaeE. cancerogenusE. cloacaeE. hormaecheiE. kobeiE. ludwigii</td><td colspan="1" rowspan="1">E.asburiae, E.cancerogenus,E.cloacae, E.cowanii,E.hormaechei, E.kobei,E.ludwigii, E.mori,E.nimipressuralis, E.soli</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.E. mori and E. soli have notbeen reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E.albertii, E.coli,E.fergusonii,Shigella spp.</td><td colspan="1" rowspan="1">All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">H.aegyptius, H.influenzae,H.haemolyticus</td><td colspan="1" rowspan="1">H. haemolyticus is reported asa commensal of the oropharynxand is generally considerednonpathogenic; it can be a rarepathogen. H. aegyptius hasbeen reported as a causativeagent of conjunctivitis and arare cause of systemic disease.Further, some rare isolates thatbelong to the genusHaemophilus and cannot beassigned to a validated speciesmay be identified by the H.influenzae reference spectrum.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Haemophilusparainfluenzae</td><td colspan="1" rowspan="1">H. parainfluenzae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">k</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">H.alvei, H.paralvei,Obesumbacterium proteus</td><td colspan="1" rowspan="1">Obesumbacterium proteus ismost commonly associatedwith brewery spoilage and hasnot reported as isolated fromhuman specimens</td></tr><tr><td colspan="1" rowspan="1"> Wih</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">K.pneumoniae,K.granulomatis,K.singaporensis, K.variicola</td><td colspan="1" rowspan="1">All species associated with thedisplayed identification havebeen isolated from humanspecimens; K. pneumoniae isthe most common speciesreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">m</td><td colspan="1" rowspan="1">Klebsiella_oxytocaRaoultella_ornithinolytica</td><td colspan="1" rowspan="1">K. oxytocaR. ornithinolytica</td><td colspan="1" rowspan="1">K.oxytoca, R.ornithinolytica,R.planticola</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Moraxella_sg_Branhamellacatarrhalis</td><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Moraxella_sg_Moraxellaosloensis</td><td colspan="1" rowspan="1">M. osloensis</td><td colspan="1" rowspan="1">Enhydrobacter aerosaccus,Moraxella osloensis</td><td colspan="1" rowspan="1">The rare species Enhydrobacteraerosaccus is closely related toMoraxella osloensis, and hasnot reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">M.morganii,M.psychrotolerans</td><td colspan="1" rowspan="1">Both species associated withthe displayed identificationhave been reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">p</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">P.agglomerans, P.anthophila,P.brenneri, P.conspicua,P.eucalypti, P.vagans</td><td colspan="1" rowspan="1">All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens. Pantoeaagglomerans, is the mostcommonly reported Pantoeaspecies isolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pasteurella multocida</td><td colspan="1" rowspan="1">P. multocida</td><td colspan="1" rowspan="1">cYanmaGntaYellowb</td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">W Wi</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="2" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1">q</td><td colspan="1" rowspan="1">Proteus vulgaris_group</td><td colspan="2" rowspan="1">P. hauseriP. penneriP. vulgaris</td><td colspan="1" rowspan="1">P.hauseri, P.penneri,P.vulgaris</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together. All specieshave been reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">r</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="2" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">P.rettgeri, P.alcalifaciens,P.burhodogranariea,P.heimbachae, P.rustigianii,P.vermicola</td><td colspan="1" rowspan="1">P. burhodogranariea and P.vermicola have not beenreported as isolated fromhuman specimens</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">P. stuartii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="2" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">Pseudomonasfluorescens_group</td><td colspan="2" rowspan="1">P. congelansP. corrugataP. extremorientalisP. fluorescensP. gessardiiP. libanensisP. mandeliiP. marginalisP. migulaeP. mucidolensP. orientalisP. poaeP. rhodesiaeP. synxanthaP. tolaasiiP. trivialisP. veronii</td><td colspan="1" rowspan="1">P.congelans, P.corrugata,P.extremorientalis,P.fluorescens, P.gessardii,P.libanensis, P.mandelii,P.marginalis, P.migulae,P.mucidolens, P.orientalis,P.poae, P.rhodesiae,P.synxantha, P.tolaasii,P.trivialis, P.veronii</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.Members of the P. fluorescensgroup are environmentalorganisms. P. fluorescens isthe most commonly isolatedspecies reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">W Hm</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Includedin Database</td><td colspan="1" rowspan="1">Different species arepotentially associated withthe displayed identification.Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Confirmatory tests arerequired to differentiatebetween listed organisms.</td><td colspan="1" rowspan="1">Matching Hint Table</td></tr><tr><td colspan="1" rowspan="1">t</td><td colspan="1" rowspan="1">Pseudomonas putida_group</td><td colspan="1" rowspan="1">P. fulvaP. monteiliiP. mosseliiP. plecoglossicidaP. putida</td><td colspan="1" rowspan="1">P.fulva, P.monteilii,P.mosselii, P.plecoglossicida,P.putida</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult, whereas they aregrouped together.Members of the P. putida groupare environmental organisms.P. putida is the most commonlyisolated species reported asisolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">u</td><td colspan="1" rowspan="1">Salmonella sp</td><td colspan="1" rowspan="1">Salmonella sp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Identification is possible ongenus level only.</td></tr><tr><td colspan="1" rowspan="1">v</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">S.liquefaciens,S.proteamaculans,S.grimesii, S.plymuthica,S.ficaria</td><td colspan="1" rowspan="1">All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">W</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">S.marcescens,S.nematodiphila, S.ureilytica</td><td colspan="1" rowspan="1">S. nematodiphila and S.ureilytica have not beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">S. maltophiliaPseudomonas beteliPseudomonashibiscolaPseudomonasgeniculata</td><td colspan="1" rowspan="1">S.maltophilia, Pseudomonasbeteli, Pseudomonashibiscola, Pseudomonasgeniculata</td><td colspan="1" rowspan="1">S. maltophilia, P. beteli, P.hibiscola, P. geniculata aresynonymously usedtaxonomical names.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">Y. enterocolitica</td><td colspan="1" rowspan="1">cYanmaGntllowb</td><td colspan="1" rowspan="1">No matching hint required.</td></tr><tr><td colspan="1" rowspan="1">y</td><td colspan="1" rowspan="1">Yersiniapseudotuberculosis</td><td colspan="1" rowspan="1">Y.pseudotuberculosis</td><td colspan="1" rowspan="1">Y.pestis,Y.pseudotuberculosis,Y.similis</td><td colspan="1" rowspan="1">Y. pestis is a select agent andshould be ruled out; handleisolate with extreme cautionand handle in accordance withlocal, state, and federalaccrediting organizations'requirements as applicable.All species associated with thedisplayed identification havebeen reported as isolated fromhuman specimens.</td></tr></table>

b. Clinical specificity: See clinical sensitivity results c. Other clinical supportive data (when a. and b. are not applicable):

4. Clinical cut-off: See Assay cut-off

5. Expected values/Reference range: See Assay cut-off

N. Instrument Name: MBT CA System

# O. System Descriptions:

# 1. Modes of Operation:

After the sample has been applied to the target plate and the spots have been manually identified in the software, organism identification is completely automated. A mass spectrum is acquired by the MALDI Biotyper CA System from the unknown organisms and is transformed into a numerical list of peak intensity and mass to charge ratio. Using an algorithm, the peak list is compared to reference peak lists of organisms in the reference library (database) and a log(score) value between 0.00 and 3.00 is calculated. The log(score) value ranges reflect the probability of organism identification. Results should be reviewed by a trained microbiologist and final organism identification should be based on all relevant information available. This information includes, but is not limited to, Gram staining, colony morphology, growth characteristics, sample matrix, or other factors that might impact organism identification.

The workflow to perform the test is summarized in the table below. The User is responsible for all sample preparation steps prior to inserting the target plate into the mass spectrometer. Sample preparation steps are described in the "MALDI Biotyper CA System System Package Insert Reference Library." After the user has prepared the target plate according to the package insert the Sample-IDs are manually entered into the system software which links a target plate spot position with a sample ID. After these steps are complete the sample analysis proceeds automatically through Step 14 (below) when the results report is presented to the user.

# Workflow table for organism identification using the MBT-CA System

0. Stat ic 1. Computer starts 2.ST-a 3. MBT-CAsho 4. Previde   
5.Run Definition   
Window opens   
6. Provide 7. Sample Table 8. Select 9. The OK Button 10. Close Run   
Sample-IDs Is populated QC Position Becomes Active Window with OK   
1 Run is saved to Database, Run Window is closed   
Main-SW-GUI shows defined Run   
12 Clickck Btton 13. Measurement 14 lik Button 15. Result Report 1. T-Ce

Before any MALDI Biotyper CA System analysis is started a quality control (QC) step is mandatory. This QC is executed using a sample preparation of an IVD Bacterial Test Standard (IVD BTS). The respective IVD BTS mass spectrum contains eight well known peaks which are used for an automatic calibration and for checking whether the instrument acquires spectra of sufficient quality.

Spectrum and data processing schematic Spratum PrepAles sing Pesk MS Iden ntification

2. Software:

FDA has reviewed applicant's Hazard Analysis and software development processes for this line of product types:

Yes _X__ or No

Level of Concern:

Moderate

Software Description:

The MALDI Biotyper CA System software runs together with the flexControl software on the same computer which is referred to as the instrument control computer. Accordingly the link between the MALDI Biotyper CA System and the flexControl software occurs as a communication of processes running on the same computer. The applied communication

protocol is COM where flexControl provides the COM server while MALDI Biotyper CA System acts as COM client. As shown in the figure below:

![](images/7e43878ba344e40341b75d4964f831948407cc8c6a8e52fefaa54c0a6dcd93ed.jpg)

The MALDI Biotyper CA System software has a three layer architecture. The components of the single layers run independently on the same computer and exchange information via various interfaces. The appropriate protocols used in the interfaces are selected according to special requirements of the participating sides.

The interface between the MALDI Biotyper CA System application and the MALDI Biotyper CA System server uses language independent protocols. The interface between the MALDI Biotyper CA System server and the MBT-DB server uses a standard protocol for accessing relational databases.

![](images/82a2780440fa7ec197471f9c4a4f9b209696b434ecaa66cdcbe4461cc629a6b6.jpg)

In the MALDI Biotyper CA System software the user creates a run by assigning the identifier of the prepared test organisms to the appropriate target plate positions. The run name automatically contains the unique target plate ID. After acquisition and analysis, the MALDI Biotyper CA System client outputs a report with the analysis results.

The flexControl software is used to control the microflex LT/SH mass spectrometer and the automated acquisition of mass spectra and includes the embedded firmware for the electronics in the microflex LT/SH instrument. Control involves setting and monitoring parameters such as high voltages, moving the sample carrier, docking and loading and saving acquisition methods. Automated tasks include loading the specified acquisition method, moving the sample carrier to the defined position and acquiring and storing a spectrum from that position on the target. In the MALDI Biotyper CA System workflow, the flexControl software is controlled by the MBT software and does not display a user interface. A functional diagram of the flexControl software is shown below:

![](images/c62eda921b46da804f100deef5c3ddeef8858c2f13c0ca49ad0f13f443ee844b.jpg)

Architecture Design Chart: Acceptable

The MALDI Biotyper CA System consists of the following main components and the functional relationship between these components is llustrated in the architecture design chart below:

microflex LT/SH L T/SH mass spectrometer   
Computer running the MBT -CA software   
MSP 48 target polished steel plates (Target)   
US IVD matrix HCCA (on prepared target spots)   
US IVD Bacterial Test Standard (US IVD BTS) (on a portion of the prepared target spots)

![](images/12a0e116a9e4b1a0c110d17942cba0c153150f31a2da8658a7c2e24358a22d49.jpg)

# Software Requirements Specifications (SRS): Acceptable

MALDI Biotyper CA System SRS

The MALDI Biotyper CA System SRS defines the requirements for the MALDI Biotyper CA System software package. This package consists of several components designed according to a client / server structure. Only the client component is directly accessible by the end user via a graphical user interface (GUI). The MALDI Biotyper CA System client communicates with the mass spectrometer control and acquisition software (flexControl) as well as with its server components. Product requirements, interface requirements, performance requirements, design constraints, and a number of system attributes are included in the SRS.

# Flexcontrol SRS

This document defines the requirements for the flexControl software from the MALDI Biotyper CA System perspective. The flexControl software is used to control the microflex LT/SH mass spectrometer and the automated acquisition of mass spectra. Product requirements, interface requirements, performance requirements, design constraints, and a number of system attributes are included in the SRS.

# MTB-CA SDS

This document describes the implementation of the requirements as outlined in the SRS. This document describes the design and function of the MALDI Biotyper CA software as it is used within the MALDI Biotyper CA workflow. Chapter 2 specifies the general design of the MALDI Biotyper CA software and its interfaces which constitutes the main part of this design document. Chapter 3 contains several specific requirements and particularly the risk analysis based design of the MALDI Biotyper CA software (section 3.7). The SDS uses the same section numbers to describe the design as applied in the SRS document.

# Flexcontrol SDS

This document describes the implementation of the requirements as outlined in the SRS. This document describes the design and function of the flexControl software as it is used within the MALDI Biotyper CA System workflow.

Development Environment: Acceptable

The flexControl software package is developed using the Microsoft Visual Studio 2008 development environment. The firm implements formal source code archiving and bug tracking systems. The source code is organized in a modular layout and follows an internal coding style guide. The modules are written using the $\mathbf { C } { \mathrel { + { + } } }$ or $\mathbf { C } \#$ programming languages.

The development environment for the MALDI Biotyper CA System client side is the Microsoft programming environment Visual Studio, the business logic tier in a Java based application server platform and the database tier in a standard relational database management system.

Traceability Matrix: Acceptable

Two traceability matrices were provided which link SRS items to a corresponding DHA line item. Traceability for the flexControl and MALDI Biotyper CA System software link user and patient hazard analysis items to product requirements, product functions, design constraints, and risk analysis identified requirements.

# Device Hazard Analysis (DHA): Acceptable

The risk analysis document describes the risk management plan for the MALDI Biotyper CA System and capture management activities, as outlined in ISO 14971. The risk management plan addresses the risk management for the MALDI Biotyper CA System from Bruker Daltonics Inc. throughout the entire product life cycle. This risk management plan describes how risk management is considered from the early device design phase through post market surveillance including regular updates based on risk relevant product information. The plan includes considerations for risk analysis, risk evaluation, risk control and risk acceptability along with acceptable risk assessments of false-positive and false-negative identifications of bacteria. The hazard severity of false results is stratified into three groups from low to high pathogenic species and other clinical considerations.

The DHA lists specific risks and hazards related to system functions and identifies the hazard, root cause, hazardous effects, risk mitigation, implementation and verification method. All hazard line items are listed as having acceptable risks according to the acceptance criteria in the risk management plan.

# Verification and Validation Testing: Acceptable

The following verification and validation documents were reviewed:

06_Appendix I_MBT-CA System 3.2_Verification Test_filled • R06_Appendix II_MBT-CA System 3.2_Verification Test_ $2 ^ { \mathsf { n d } } .$ _step_filled • R06_Appendix III_MBT-CA System_Verification_Summary R06_Appendix IV_FC-MBT-CA System_Verification_Test R06_Appendix V_FC-MBT-CA System_Verification_Summary R12_Appendix I_MBT-CA System Initial Verification Test Server_3.2.8.2

System verification has been done with software version MALDI Biotyper CA System 3.2.3. The verification procedure checked the basic functionality of the MALDI Biotyper CA System application in 197 single verification cases. The procedure started with the installation of the appropriate software which contains the completely configured software that is delivered to customers. The verification procedure was executed over a continuous time period and all results observed were acceptable in verifying the main software performance. The completed verification procedure was provided.

# Revision Level History: Acceptable

A summary revision level history for the MALDI Biotyper CA System software was acceptable as provided. Revision levels were given for each phase of verification, validation and the proposed final release of the MALDI Biotyper CA System and the flexcontrol softwares. A summary list of the minor changes between versions indicated that a controlled process is in place to track and describe feature changes with respect to version number. The final software revisions which were reviewed appear in the table below.

<table><tr><td rowspan=1 colspan=1>Software Item</td><td rowspan=1 colspan=1>Reviewed Revision</td></tr><tr><td rowspan=1 colspan=1>MBT-CA System Software</td><td rowspan=1 colspan=1>3.2.10.0</td></tr><tr><td rowspan=1 colspan=1>MBT-CA Server</td><td rowspan=1 colspan=1>4.0.14.0</td></tr><tr><td rowspan=1 colspan=1>flexControl</td><td rowspan=1 colspan=1>3.4.127.0</td></tr></table>

# Unresolved Anomalies: Acceptable

Descriptions of the unresolved anomalies in the final release were summarized and tracking information was provided. Bugs appear to have been prioritized and evaluated with respect to severity and remaining anomalies all have minor classification or otherwise cannot be reproduced.

The system for bug tracking was acceptable as described. Each bug or feature request is entered into the system and receives a unique ID. A product development team reviews the bug or feature request and decides whether to address the item under review. If the item passes this review, the project lead schedules bug fixes and feature implementations for a certain target software release and assigns the entry to a software developer. The software developer implements the fix or feature and sets the entry to fixed and specifies the build number containing the fix. Once a new software product build is available, the software test team tests the fixed entries and verifies their functionality. If acceptable, the entries are set to closed in the tracking system otherwise they are reopened and reassigned to the developer.

Off the Shelf Software (OTS): Acceptable

A table in the SDS lists the Open Source software components used in the software architecture. The source code of all applied Open Source software is archived in the appropriate version. The description, specifications, end-user actions, verification, version control and hazard analysis for all OTS software components were acceptable as provided. The user has been restricted from modifying and/or does not interact directly with OTS elements. Windows updates are not expected to alter the performance of the system and a method is provided to restored the factory state if needed. All other OTS will stay at the version provided at the time of MALDI Biotyper CA System installation.

The MALDI Biotyper CA System operates without remote access software; it can also operate without public network access. For remote maintenance an operationally well-tried and widely available third party service is used. Only the instrument computer can be connected to the internet for remote maintenance. The mass spectrometer and its built in electronics have no internet connection. To manipulate the mass spectrometer through the internet, one must have physical control over the computer. The end-user has to grant the support person the right to view or control programs or the desktop or to copy files. When a support session is closed, the end-user's computer is no longer accessible through this service (i.e. the connection is user initiated and not persistent). Thus, as long as the instrument computer is physically safe, the mas spectrometer cannot be manipulated via the internet.

The instrument computer runs a modern standard operating system (Windows 7). Patches are recommended to have known security issues solved as soon as possible. The manual includes language explaining that Bruker does not supply anti-virus software or spyware protection software with the installed system.

# EMC and Electrical Testing:

The MALDI BIOTYPER CA System has been tested and complies with IEC 61326-1 (2005- 12).

Electrical Safety of MALDI Biotyper CA System was also evaluated in accordance with IEC 61010-1:2004 by a third party vendor. The microflex LT/SH LT/SH mass spectrometer is an In vitro diagnostic equipment which does not fall within the definition of a Medical Electrical Equipment per IEC 60601-1 (per IEC 60601-1:2006, MOD. $\ S 3 . 6 3 )$ therefore, the electrical safety evaluation of the microflex LT/SH LT/SH mass spectrometer has been performed per IEC

# 3. Specimen Identification:

The user manually enters the specimen identification information into the MALDI Biotyper CA System. The user first defines active sample positions (see section 4.2.1 of User Manual) and US IVD BTS control positions (see section 4.2.2 of User Manual). The defined sample positions are required to have a user entered and valid sample identifier in the Id column along with an optional description. All inoculated positions must contain a valid ID before the run can proceed. After all sample positions have been input, at least two US IVD BTS control positions must be defined. It is recommended that the cross-joint positions on the target are used as quality control positions.

# 4. Specimen Sampling and Handling:

After incubation of bacteria on recommended isolation media for $1 8 { - } 2 4 \mathrm { h }$ at $( 3 7 ^ { \circ } \mathbf { C } \pm 2 ^ { \circ } \mathbf { C } )$ colonies are stable for up to $^ { 1 2 \mathrm { ~ h ~ } }$ when held at room temperature. If testing is not done within a total of 36 hours, subculture the test organism prior to testing on the MALDI Biotyper CA System.

Using a sterile colony transfer device, smear isolated colonies of bacteria as a thin film directly onto a sample position on a cleaned target. Inoculating an appropriate amount of the test organism onto the target is important. Excessive or insufficient amounts of inoculum may impact organism identification. The dynamic range to identify Gram-negative bacteria using the MALDI Biotyper CA System is estimated to be $1 0 ^ { \overline { { 4 } } }$ to $1 0 ^ { 7 }$ genome equivalents on the target position. The User Manual visually illustrates suitable and unsuitable inoculum amounts of Gram-negative bacteria on target. After samples and US IVD BTS are dried, US IVD HCCA portioned must be added within 1 h or the target must be cleaned and the inoculation of samples and US IVD BTS must be performed again.

Each of the sample positions and US IVD BTS control positions are overlayed with $1 \mu \mathrm { L }$ US IVD HCCA portioned solution. Use a new pipette tip to add matrix to each inoculated sample position. Dry the inoculated plate at room temperature. The inoculated MALDI target plate is now ready for use. An inoculated target with IVD HCCA overlay must be processed within 24 hours of preparation.

If the MALDI BIOTYPER CA System identification of the test organism does not result in a bacterial identification with a log(score) value of $\geq 2 . 0$ , repeat testing using the extraction procedure in Section 3.6 of the User Manual. Briefly, the extraction of a sample from an isolated colony consists of multiple washing cycles of ethanol and water followed by addition of formic acid and acetonitrile followed by a centrifugation step.After centrifugation, a sample of the supernatant is applied to the target plate, dried and loaded into the MALDI Biotyper CA System for analysis.

# 5. Calibration and Quality Control:

During system start-up, an auto-calibration is performed using a US IVD BTS sample. The sample is analyzed to confirm that the US IVD BTS control meets all defined specifications. The status of the auto-calibration is displayed in the status bar at the bottom of the MALDI Biotyper CA System window.

The US IVD BTS contains a manufactured extract of Escherichia coli DH5 alpha that demonstrates a characteristic peptide and protein profile mass spectrum, when tested on the MALDI Biotyper CA System . US IVD BTS is spiked with two additional proteins (RNAse A and Myoglobin) in order to test the entire mass range of the system from 3.6 to $1 7 \ k D \mathbf { a }$ Each US IVD BTS tube contains enough material to inoculate approximately 40 US IVD 48 Spot Target locations.

In order to process a control the US IVD BTS material is inoculated on the target at two or more positions. The "in specification" functionality of the MALDI Biotyper CA System System and the microflex LT/SH LT/SH mass spectrometer is confirmed with each run and at least one of the prepared US IVD BTS control positions is automatically checked multiple times before a run can proceed. The $\mathbf { m } / \mathbf { z }$ of the calibration peaks and the spectrum baseline values are part of the verification of functionality. A number of quality parameters are combined to provide an overall quality value.

When the "in specification" check is successful, the US IVD BTS will be reported as Escherichia coli during the subsequent database matching process.

If the "in specification" check is not successful, the run is stopped and the message "QC on spot BTS(x) was not successful!" is displayed on the status bar; $\mathbf { \rho } ( \mathbf { x } )$ represents a number from 0 to 4. In this case, the user repeats the control process using the second (unused) US IVD BTS control position on the same target. If the control using the second US IVD BTS control position fails, the user contacts Bruker Technical Support.

After successfully confirming performance of the US IVD BTS sample, patient sample positions can be analyzed.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The "Performance Characteristics" Section above:

# Q. Proposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10, 21 CFR 801.109, and special controls of regulation 21 CFR 866. 3361.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.