# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k)Num ber:

k130894

# B. PurposeofSubm ission:

The Xpert MRSA/SA Blood Culture Assay was previously FDA-cleared in two 510(k) submission k082140 and special 510(k) k101879. The purpose of this submission is to establish performance to support proposed changes to the current assay and changes in the Intended Use Statement.

# C. M easurand:

Target DNA sequences for the staphylococcal protein A (spa), for methicillin/oxacillin resistance (mecA), and for the staphylococcal chromosomal cassette (SCCmec) insertion event into the staphylococcus aureus chromosomal attB site.

# D. TypeofTest:

Nucleic Acid Amplification Test, DNA, Methicillin-resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA)

# E. Applicant:

Cepheid®

F. Proprietary and Established Nam es: Xpert® MRSA/SA Blood Culture Assay

# G .Regulatory Inform ation:

1. Regulation section: 21 CFR section 866.1640, Antimicrobial susceptibility test powder

2. Classification: Class II

# Page - 2/40

3. Product code:

NQX

4. Panel: Microbiology (83)

# H .Intended Use:

1. Intended use(s):

The Cepheid Xpert® MRSA/SA Blood Culture Assay, performed on the GeneXpert® Instrument Systems, is a qualitative in vitro diagnostic test intended for the detection of Staphylococcus aureus (SA) and methicillin-resistant Staphylococcus aureus (MRSA) DNA directly from positive blood cultures. The assay utilizes automated real-time polymerase chain reaction (PCR) for the amplification of MRSA/SA specific DNA targets and fluorogenic target-specific hybridization probes for the real-time detection of the amplified DNA. The assay is performed directly on positive blood culture samples from BD BACTEC™ Plus Aerobic/F, BacT/ALERT® SA (Standard Aerobic) or VersaTREK REDOX $1 ^ { \mathfrak { B } }$ (aerobic) blood culture bottles that are determined by Gram stain as Gram Positive Cocci in Clusters (GPCC) or as Gram Positive Cocci in singles (GPC). The Xpert MRSA/SA Blood Culture Assay is indicated for use in conjunction with other laboratory tests, such as culture, and clinical data available to the clinician as an aid in the detection of MRSA/SA from positive blood cultures. Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing or for epidemiological typing. The Cepheid Xpert MRSA/SA Blood Culture Assay is not intended to monitor treatment for MRSA/SA infections.

2. Indication(s) for use: N/A

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

To be used with the GeneXpert DX, GeneXpert Infinity-48, GeneXpert Infinity48s, and GeneXpert Infinity-80 systems

# I. DeviceDescription:

The Xpert MRSA/SA Blood Culture Assay system performs real-time, multiplex polymerase chain reaction (PCR) for detection of Staphylococcus aureus and methicillin-resistant Staphylococcus aureus DNA. The assay is performed directly on positive blood culture specimens using Becton Dickinson BD BACTEC Plus Aerobic/F, bioMérieux BacT/ALERT SA (Standard Aerobic), or Trek Diagnostic VersaTREK REDOX 1 (aerobic) blood cultures bottles that are determined as GramPositive Cocci in Clusters (GPCC) or as Gram-Positive Cocci in singles (GPC) by Gram stain. The primers and probes specifically detect nucleic acid sequences of the staphylococcal protein A (spa), the gene for methicillin/oxacillin resistance (mecA), and staphylococcal cassette chromosome (SCCmec) inserted into the SA chromosomal attB site. The test includes a sample processing control (SPC) to control for adequate processing of the target bacteria and to monitor the presence of inhibitor(s) in the PCR assay to reduce false-negative results. The SPC also indicates whether the PCR reaction conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. The Probe Check Control (PCC) verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability.

The specimen for testing with the Xpert MRSA/SA Blood Culture Assay is prepared from an aliquot taken from a positive blood culture bottle. Using one of the disposable fixed $5 0 ~ \mu \mathrm { L }$ volume transfer pipettes provided with the test kit, an aliquot of the positive blood culture is transferred into a single-use tube of Elution Reagent, also provided with the kit. The Elution Reagent is briefly vortexed and the entire content is transferred to the “ $\mathbf { \vec { S } } ^ { \ 3 }$ chamber of the disposable fluidic Xpert MRSA/SA Blood Culture Assay cartridge, after which the cartridge is ready to be place on the GeneXpert Instrument System.

The GeneXpert Instrument Systems perform sample preparation by mixing the sample with the sample preparation control (Bacillus globigii in the form of a dry spore cake within the cartridge) and treatment reagents, capturing the bacterial cells on a filter, lysing the cells using glass beads and an ultrasonic horn, then eluting the released DNA. The DNA solution is then mixed with dry PCR reagents and transferred into the PCR tube for real-time PCR and detection. In this platform, sample preparation, amplification, and real-time detection are all fully-automated. Summary and detailed test results are obtained in approximately 60 minutes and are displayed in tabular and graphic formats.

# J. SubstantialEquivalenceInform ation:

1. Predicate device name(s): BD GeneOhm StaphSR Assay

2. Predicate 510(k) number(s): k071026

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>New DeviceXpert MRSA/SABlood Culture Assay</td><td rowspan=1 colspan=1>Predicate DeviceBD GeneOhmStaphSR Assay(510(k) #K071026)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Rapid detection of MRSAand SA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Indication for Use</td><td rowspan=1 colspan=1>Identification of MRSAand SA colonization</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Positive Blood Culture</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technological Principles</td><td rowspan=1 colspan=1>Fully-automated nucleicacid amplification(DNA); real-time PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DNA Target Sequence</td><td rowspan=1 colspan=1>Sequence incorporatingthe insertion site (attB) ofStaphylococcal CassetteChromosome mec(SCCmec) for detectionof MRSA.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Clinical ComparisonResults</td><td rowspan=1 colspan=1>Xpert MRSA/SA BloodCulture AssayPerformance vs.Reference Culture :MRSA:Positive % Agreement:98.1%Negative % Agreement:99.6%SA:Positive % Agreement:99.6%Negative % Agreement:99.5%</td><td rowspan=1 colspan=1>BD GeneOhmTStaphSR AssayPerformance vs.Reference CulturemethodsMRSA:Positive % Agreement:100.0Negative % Agreement:98.2 - 100.0SA:Positive % Agreement:98.8 - 100.0Negative % Agreement:96.5 - 100.0[Data obtained from theBD GeneOhm StaphSRAssay 510(k) Summary]</td></tr></table>

Page - 5/40   

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>New DeviceXpert MRSA/SABlood Culture Assay</td><td rowspan=1 colspan=1>Predicate DeviceBD GeneOhmStaphSR Assay(510(k) #K071026)</td></tr><tr><td rowspan=1 colspan=1>Sample Preparation</td><td rowspan=1 colspan=1>Self-contained andautomated after mixedspecimen is added tocartridge. All otherreagents are contained inthe cartridge.</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>Probes</td><td rowspan=1 colspan=1>TaqMan Probes</td><td rowspan=1 colspan=1>Molecular Beacons</td></tr><tr><td rowspan=1 colspan=1>Internal Controls</td><td rowspan=1 colspan=1>Sample processingcontrol (SPC) and probecheck control (PCC).</td><td rowspan=1 colspan=1>One internal reagentcontrol and externalpositive and negativecontrols required per run.</td></tr><tr><td rowspan=1 colspan=1>DNA Target Sequence</td><td rowspan=1 colspan=1>Sequence specific toStaphylococcus aureusspecies (spa gene)</td><td rowspan=1 colspan=1>Sequence specific toStaphylococcus aureusspecies (nuc gene)</td></tr><tr><td rowspan=1 colspan=1>Users</td><td rowspan=1 colspan=1>Operators with no clinicallab experience toexperienced clinicallaboratory technologists.</td><td rowspan=1 colspan=1>CLIA High ComplexityLaboratory Users</td></tr><tr><td rowspan=1 colspan=1>Ability to identify&quot;Empty CassetteVariants&quot;</td><td rowspan=1 colspan=1>Yes, sequence specific toStaphylococcus aureusspecies (mecA gene)</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>~ 50 min.</td><td rowspan=1 colspan=1>~ 60-75 min.</td></tr></table>

# K .Standard/G uidanceDocum entReferenced (ifapplicable):

Draft Guidance for Industry and Food and Drug Administration Staff - Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA)

# L. TestPrinciple:

The primers and probes in the Xpert MRSA/SA Blood Culture Assay are designed to detect proprietary gene sequences for the staphylococcal protein A $( s p a )$ , for methicillin/oxacillin resistance (mecA), and for the staphylococcal chromosomal cassette (SCCmec) insertion event into the Staphylococcus aureus chromosomal attB site. The Staphylococcal protein A (spa) gene codes for a monomeric cell wall component of Staphylococcus aureus. Methicillin resistance in SA is caused by the acquisition of an exogenous gene, mecA that encodes an altered beta lactam-resistant penicillin-binding protein (PBP), termed PBP2a or PBP2′. The mecA gene is carried on a large heterologous mobile genetic element – the staphylococcal chromosomal cassette or SCCmec. SCCmec DNA integrates at a specific attachment site (attB) in the methicillin susceptible SA (MSSA) chromosome located at the $3 ^ { \prime }$ end of an open reading frame (ORF), orfX, of unknown function. SCCmec types have been identified and designated into a number of sub-types.

The firm identified twelve primer and three probe sequences using in silico design tools to develop specific complimentary sequences for spa, mecA and SCCmec targets. In silico experiments simulated all secondary structures for targets (optimal and suboptimal), primers (optimal and suboptimal), homodimers, and target and primer heterodimers, given specified conditions. Values for Tm, dG, percent bound, and concentrations for all species were calculated. Using various software tools, unwanted predicted interactions between oligonucleotides and non-Staphylococcus aureus targets were evaluated thermodynamically and minimized.

The test begins when a blood culture bottle becomes positive. Routine processing of the bottle involves withdrawal of an aliquot from the bottle to perform a Gram stain and to inoculate solid media. For positive blood culture that are determined to be Gram-positive Cocci in clusters (GPCC) or Gram-positive cocci in singles (GPC), an aliquot is collected and transported to the GeneXpert System area. The user uses a firm supplied fixed volume transfer pipette, $5 0 ~ \mu \mathrm { L }$ of positive blood culture is transferred to the tube containing $2 . 0 \mathrm { m L }$ elution reagent. Following a 10 second vortex, the eluted material is transferred to the “ $\mathbf { \vec { S } } ^ { \ 3 }$ chamber of the cartridge. The user initiates a test from the system user interface and the instrument signals the user where to place the cartridge. The cartridge is manually placed into the indicated module in the GeneXpert Dx System Instrument, or onto a conveyor belt on the GeneXpert Infinity System. The GeneXpert Infinity System then transports the cartridge to the appropriate GeneXpert module or to the holding area for later transport to the appropriate GeneXpert module. Instrument controlled fluidic movements transfer the sample and reagents to and from different chambers within the Xpert MRSA/SA Blood Culture Assay cartridge.

M .Perform anceCharacteristics(if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Precision

The precision of the Xpert MRSA/SA Blood Culture Assay was tested with an in-house blinded study using a precision panel consisting of 11 panel members. Samples were prepared using cultured material in a simulated matrix at high, moderate and low levels. The study design incorporated testing of two strains of MRSA and one strain of MSSA. The MRSA-1 strain was MRSA Type III and the MRSA-2 strain was a heterogeneous oxacillin resistant Staphylococcus aureus, Type IV (ATCC 43300). A total of 96

replicates of each of the 11 panel members were tested per instrument platform (Infinitity-80, Infinity-48 and Dx). The study design incorporated the following variance components per panel member:

· One reagent lot   
· One test site   
· Three instruments   
· Two operators   
12 non-consecutive days of testing Four runs per day

All negative samples (576/576) were correctly classified as MRSA NEGATIVE; SA NEGATIVE.

The MRSA and MSSA high negative samples were prepared to target a concentration below the LoD where 20 to $80 \%$ of samples are expected to be MRSA NEGATIVE. Overall $4 5 . 6 \%$ (131/287) of the valid high negative MRSA strain-1 samples and $7 9 . 5 \%$ (229/288) of the high negative MRSA strain-2 samples were correctly classified as MRSA NEGATIVE. Among the MSSA high negative samples, $7 6 . 4 \%$ (220/288) were corrected classified as MRSA NEGATIVE; SA NEGATIVE.

The low positive samples were prepared at a concentration of 1x LoD which was expected to produce a positive result ${ \sim } 9 5 \%$ of the time. Among the MRSA low positive samples (MRSA-1 and MRSA-2 combined), $9 9 . 1 \%$ (571/576) were correctly classified as MRSA POSITIVE; SA POSITIVE. Among the MSSA low positive samples, $9 8 . 6 \%$ (284/288) were correctly classified as MRSA NEGATIVE; SA POSITIVE.

Among the MRSA moderate positive samples (MRSA-1 and MRSA-2 combined), $9 9 . 7 \%$ (574/576) were correctly classified as MRSA POSITIVE; SA POSITIVE. Among the MSSA moderate positive samples, $1 0 0 . 0 \%$ (288/288) were correctly classified as MRSA NEGATIVE; SA POSITIVE. Among the MRSA low positive samples (MRSA-1 and MRSA-2 combined), $9 9 . 1 \%$ (571/576) were correctly classified as MRSA POSITIVE; SA POSITIVE. Among the MSSA low positive samples, $9 8 . 6 \%$ (284/288) were correctly classified as MRSA NEGATIVE; SA POSITIVE.

No statistically significant difference in assay performance based on instrument platform was observed during the study for the negative, low positive or moderate positive samples. The total standard deviation for all targets is in the range of $0 . 5 – 1 . 6 \mathrm { C t }$ . The precision data is acceptable as presented for this type of device.

# Reproducibility

The reproducibility of the Xpert MRSA/SA Blood Culture Assay was tested with a total 30 replicates of 11 unique panel members across three test sites. Samples were prepared using cultured material in a simulated matrix. The MRSA-1 strain was MRSA Type III. The MRSA-2 strain was heterogeneous oxacillin resistant Staphylococcus aureus, Type IV (ATCC 43300). Specimens were blinded with a Specimen ID, which was used to define the order in which the samples were tested and samples were tested in a different order throughout the study.

Quality control for the Xpert MRSA/SA Blood Culture Assay consisted of three samples negative, MRSA positive and MSSA positive. All controls were run on each instrument by each operator on each day that samples were tested. Correct results from all controls were required prior to testing any panel samples.

The study design incorporated the following variance components per panel member:

Two reagent lots   
· Three test sites   
· Three instruments, one unique instrument platform per site Two operators per site   
· 5 non-consecutive days of testing Four runs per day

There were a total of 90 tests per panel member, excluding control runs.

All negative samples (180/180) were correctly classified as MRSA NEGATIVE; SA NEGATIVE.

The MRSA and MSSA high negative samples were prepared to target a concentration below the LoD. At this concentration 20 to $80 \%$ of samples are expected to be negative. Overall $6 1 . 1 \%$ (55/90) of the high negative MRSA strain-1 samples and $5 5 . 6 \%$ (50/90) of the high negative MRSA strain-2 samples were classified as MRSA NEGATIVE. Among the MSSA high negative samples, $5 9 . 6 \%$ (53/89) were classified as MRSA NEGATIVE; SA NEGATIVE; one sample was mistakenly not run at site 2.

The low positive samples were prepared to target a positive result ${ \sim } 9 5 \%$ of the time. Among the MRSA low positive samples (MRSA-1 and MRSA-2 combined), $1 0 0 . 0 \%$ (180/180) were correctly classified as MRSA POSITIVE; SA POSITIVE. Among the MSSA low positive samples, $9 7 . 8 \%$ (88/90) were correctly classified as MRSA NEGATIVE; SA POSITIVE.

Among the valid MRSA moderate positive samples (MRSA-1 and MRSA-2 combined), $1 0 0 . 0 \%$ (179/179) were correctly classified as MRSA POSITIVE; SA POSITIVE. Among the MSSA moderate positive samples, $1 0 0 . 0 \%$ (90/90) were correctly classified as MRSA NEGATIVE; SA POSITIVE.

There were no statistically significant differences in assay performance between the three sites. Summary data is shown in Table 1.

Table 1: Summary of Reproducibility Data   

<table><tr><td rowspan=1 colspan=1>Table</td><td rowspan=1 colspan=3>I able T: Summary of Reproducibility Data</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Conc.</td><td rowspan=2 colspan=1>Agree/N</td><td rowspan=2 colspan=1>Agrmt(%)</td><td rowspan=2 colspan=3>MeanCt</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=1>Betw</td><td rowspan=1 colspan=1>Between-Run</td><td rowspan=1 colspan=1>With</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=11 colspan=1>spa</td><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>55/90</td><td rowspan=1 colspan=1>61.1</td><td rowspan=1 colspan=3>35.6</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.8</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>31.2</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>50/90</td><td rowspan=1 colspan=1>55.6</td><td rowspan=1 colspan=3>35.3</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.3</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>30.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>53/89</td><td rowspan=1 colspan=1>59.6</td><td rowspan=1 colspan=3>36.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=3>33.5</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>31.7</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>NEG-1</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=4></td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=5></td></tr><tr><td rowspan=1 colspan=1>NEG-2</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=11 colspan=1>mec</td><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>55/90</td><td rowspan=1 colspan=1>61.1</td><td rowspan=1 colspan=3>35.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.4</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>31.9</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>50/90</td><td rowspan=1 colspan=1>55.6</td><td rowspan=1 colspan=3>35.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.8</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>31.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>53/89</td><td rowspan=1 colspan=1>59.6</td><td rowspan=1 colspan=3></td><td rowspan=5 colspan=11>N/A</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8</td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Neg-1</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Neg-2</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td></td><td></td></tr><tr><td rowspan=11 colspan=1>SCC</td><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>55/90</td><td rowspan=1 colspan=1>61.1</td><td rowspan=1 colspan=3>37.2</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>34.5</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.0</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>50/90</td><td rowspan=1 colspan=1>55.6</td><td rowspan=1 colspan=3>36.8</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.7</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>53/89</td><td rowspan=1 colspan=1>59.6</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=5 colspan=9>N/A</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neg-1</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Neg-2</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=11 colspan=1>SPC</td><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>55/90</td><td rowspan=1 colspan=1>61.1</td><td rowspan=1 colspan=3>35.6</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>MRSA-1</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.0</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>50/90</td><td rowspan=1 colspan=1>55.6</td><td rowspan=1 colspan=3>33.1</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.9</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MRSA-2</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>32.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>high neg</td><td rowspan=1 colspan=1>53/89</td><td rowspan=1 colspan=1>59.6</td><td rowspan=1 colspan=3>32.8</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>low pos</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=3>32.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>mod pos</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=3>33.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>NEG-1</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=13>N/A</td></tr><tr><td rowspan=1 colspan=1>NEG-2</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Stability

Assay Reagent Stability

The shelf life of the Xpert MRSA/SA Blood Culture was tested in stability studies using real-time stability results and linear regression analysis with data from three final product lots. Stability testing is ongoing with three lots and the actual shelf life dating will be determined by the results of real time stability studies and approved by the firm in compliance with firm approved procedures. The stability of the product was evaluated at four temperatures $( 5 ^ { \circ } \pm 3 ^ { \circ } \mathrm { C }$ , $2 5 ^ { \circ } \pm 3 ^ { \circ } \mathrm { C }$ , $3 5 ^ { \circ } \pm 3 ^ { \circ } \mathrm { C }$ and $4 5 ^ { \circ } \pm 3 ^ { \circ } \mathrm { C } )$ at predefined time point intervals up to 36 months following the study plan schedule.

The data for Xpert MRSA/SA Blood Culture Lot 00101 are monitored according to the times and temperatures specified in the test plan and analyzed for $\mathrm { C t }$ values for SPC, spa, mecA, and SCCmec. Lot 00101 currently has 6 months real-time data and the predicted $\mathrm { C t }$ values for, spa, mecA, and SCCmec are within limits for 6 months at $2 { - } 8 ^ { \circ } \mathrm { C }$ after exposure to nonshipping and summer and winter shipping conditions. Lots 00401 and 00501 currently have 3 months of real-time data and are comparable to, and within stability limits set by, Lot 00101. Testing is ongoing on three lots for up to 36 months.

Package performance testing and summer and winter shipping simulation studies were performed. MRSA/SA Blood Culture Assay kits were subjected to package performance testing as outlined in the ASTM D4169-09 Test Protocol. Individual test inputs included initial manual handling, vehicle stacking, loose load vibration, low pressure, vehicle vibration, and final manual handling. Inspections of the packaging concluded that there was no unusual external physical damage to the protective packaging as a result of the test inputs. The physical damage sustained by the protective packaging was considered normal due to the nature of the test inputs. At the completion of the testing Cepheid examined the packaged assay kits and concluded that there was no unusual internal physical damage to the assay kits. The packaging was determined to be suitable for the MRSA/SA Blood Culture Assay.

Functional kit testing was performed in accordance with the Xpert MRSA/SA Stability plan. The Ct values for SPC, spa, mecA, and SCCmec were compared to data obtained from control cartridges stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and 22- $2 8 ^ { \circ } \mathrm { C }$ and not subjected to shipping simulations. Under the conditions of this study similar performance was observed between shipped samples (exposed to shipping simulation and summer/winter temperature profile conditions) and the non-shipped samples at ${ } ^ { \mathfrak { c } \mathfrak { c } } \mathrm { T } { = } 0$ , Non-shipped”.

Specimen Stability

Positive and negative samples were included in the specimen stability study. Positive samples consisted of each blood culture matrix bottle containing 10 mL of MRSA-negative whole blood inoculated with MRSA cells (N315,

SCCmec type II) near the analytical limit of detection (LoD) at 500 CFU/test and incubated at $3 5 \mathrm { { ^ \circ C } }$ for 18 hours. Following the 18 hour incubation at $3 5 \mathrm { ^ \circ C }$ , aliquots from each blood culture bottle type were stored at $2 ^ { \circ } \mathrm { C }$ and $8 \mathrm { { } ^ { \circ } C }$ to represent the extremes of the recommended refrigerated storage temperature range. Replicates of 8 positive and negative samples were evaluated at $\mathrm { T } = 0$ , $\mathrm { T } = 1$ day, $\mathrm { T } = 2$ days and $\mathrm { T } = 3$ days. Following the 18 hour incubation at $3 5 \mathrm { { ^ \circ C } }$ , aliquots from each blood culture bottle type were also stored at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 \mathrm { { } ^ { \circ } C }$ to represent the extremes of the recommended room temperature storage range. Replicates of 8 positive and negative specimens were evaluated at $\mathrm { T } = 0$ , $\mathrm { T } = 2$ hours, $\mathrm { T } = 4$ hours, $\mathrm { T } = 8$ hours, $\mathrm { T } =$ 12 hours, and $\mathrm { T } = 2 4$ hours. Following the 24 hour incubations at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 ^ { \circ } \mathrm { C } _ { \mathrm { i } }$ , the remaining samples were moved to $2 ^ { \circ } \mathrm { C } .$ , stored for an additional 3 days, and tested. Statistical significance (ANOVA) was determined by comparing cycle threshold $\mathrm { ( C t ) }$ values from tests run at scheduled time points and temperatures to $\mathrm { T } = 0$ .

Under the conditions of this study, all positive and negative specimens were correctly identified using the Xpert MRSA/SA Blood Culture Assay for all three blood culture bottle types at all storage temperatures. The data in Table 2 supports the recommended specimen storage conditions at room temperature $( 1 5 { - } 3 0 ^ { \circ } \mathrm { C } )$ up to 24 hours and the data in Table 3 supports refrigerated specimens $\left( 2 { - } 8 ^ { \circ } \mathrm { C } \right)$ up to three days until testing is performed on the GeneXpert for all three blood culture bottle types: Becton Dickinson BACTEC™ Plus Aerobic/F, BioMérieux BacT/ALERT SA (Standard Aerobic), and VersaTREK REDOX1 (aerobic). In positive samples, there were no statistically significant differences observed relative to $\mathrm { T } { = } 0$ (one-way ANOVA) using the Becton Dickinson, BACTEC™ Plus Aerobic/F or TREK, VersaTREK REDOX1 Aerobic blood culture bottles at all storage temperatures and times.

In positive samples using the BioMérieux, BacT/ALERT SA (Standard Aerobic) bottle, there were no statistically significant differences observed relative to $\mathrm { T } = 0$ at $2 \mathrm { { ^ \circ C } }$ , $8 \mathrm { ^ \circ C }$ and $1 5 ~ \mathrm { { ^ \circ C } }$ storage temperatures (p-value ${ \tt > } 0 . 0 5$ ). One-way ANOVA, however, indicated a statistically significant difference in spa, mecA, and SCCmec Cts at $3 0 \mathrm { { ^ \circ C } }$ storage. The $\mathrm { C t }$ values for all three targets (spa, mecA and SCCmec) in positive specimens were earlier than the T $= 0$ control for the BioMérieux, BacT/ALERT SA (Standard Aerobic) bottle (p-value $< 0 . 0 5$ ). This difference is not clinically significant since the reported results of the assay were not affected.

All negative samples tested in the study remained MRSA negative using the Xpert MRSA/SA Blood Culture Assay. In negative samples, there were no statistically significant differences observed in SPC Cts relative to $\mathrm { T } = 0$ using one-way ANOVA for all three blood culture bottle types at all storage temperatures and times.

Table 2 and Table 3 show the mean cycle threshold (Ct) values for the positive and negative specimens at each time point and temperature for each blood culture bottle type.

Table 2 : Room Temperature Stability Test Data MRSA Positive   

<table><tr><td rowspan=2 colspan=1>Time</td><td rowspan=2 colspan=1>Temp</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=3>Becton DickinsonBACTEC</td><td rowspan=1 colspan=3>BioMérieuxBacT/ALERT SA</td><td rowspan=1 colspan=3>TREKVersaTREKREDOX1</td></tr><tr><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td></tr><tr><td rowspan=1 colspan=1>T = 0</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>2 hrs.</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>14.2</td></tr><tr><td rowspan=1 colspan=1>4 hrs.</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>8 hrs.</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.7</td></tr><tr><td rowspan=1 colspan=1>12 hrs.</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>24 hrs.</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>T= 0</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>2 hrs.</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>4 hrs.</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.7</td></tr><tr><td rowspan=1 colspan=1>8 hrs.</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>12 hrs.</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.7</td></tr><tr><td rowspan=1 colspan=1>24 hrs.</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>2°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>14</td></tr></table>

Table 3 : Cold Storage Stability Test Data MRSA Positive Cold Storage   

<table><tr><td rowspan=2 colspan=1>Time</td><td rowspan=2 colspan=1>Temp</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=3>Becton DickinsonBACTEC</td><td rowspan=1 colspan=3>BioMérieuxBacT/ALERT SA</td><td rowspan=1 colspan=3>TREKVersaTREKREDOX1</td></tr><tr><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td><td rowspan=1 colspan=1>spa</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>SCC</td></tr><tr><td rowspan=1 colspan=1>T= 0</td><td rowspan=1 colspan=1>2°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>1 Day</td><td rowspan=1 colspan=1>2°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>2 Days</td><td rowspan=1 colspan=1>2°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>3 Days</td><td rowspan=1 colspan=1>2°</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>T = 0</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>1 Day</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>2 Days</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>3 Days</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>13.9</td></tr></table>

d. Detection limit:

The Limit of Detection (LoD) test included 3 methicillin-susceptible Staphylococcus aureus (MSSA) strains and 10 methicillin-resistant Staphylococcus aureus (MRSA) strains diluted in negative matrix. Negative matrix consisted of $1 0 ~ \mathrm { m L }$ of Staphylococcus aureus-free whole blood added to blood culture media (BD BACTEC Plus Aerobic/F, BioMérieux BacT/ALERT SA Standard Aerobic, or VersaTREK REDOX1 Aerobic), EDTA blood and MSSE (methicillin-susceptible Staphylococcus epidermidis) cells at $1 0 6 \mathrm { C F U / m L }$ . Negative replicates $( \mathrm { n } { = } 2 0 )$ ) consisted of the blood culture matrix only. Each stock concentration was quantified by plating in duplicate and additional plating was performed on each dilution for all strains included in the study. LoD was estimated using the BD BACTEC Plus Aerobic/F derived simulated negative matrix and was subsequently confirmed separately using simulated matrix using derived from all claimed blood culture bottle types.

In total, the LoD estimation and confirmation study consisted of 1813 unique instrument runs performed across the GeneXpert Dx R1 systems (GX-IV and GX-XVI), Infinity-48 and Infinity-80 instrument platforms with a total of 9 indeterminate results (instrument errors). External control runs yielded no indeterminate or unexpected results.

The LoD estimate and confidence intervals were determined using probit regression analysis with data over the range of organism concentrations expressed as CFU/test. The point estimates were calculated using the method of maximum likelihood estimates (MLE) from the probit model parameters. The numeric estimates of the confidence intervals (CI) were taken from the Minitab 16 probit function. The probit regression analyses are shown in Figures 18.3-1 through 18.3-13.

The LoD for MRSA was confirmed using 10 strains representing MRSA SCCmec types I, II, III, IVa, IVd, V, VII, and VII. Point estimates of the LoD for each MRSA SCCmec type along with the two-sided confidence interval are shown in Table 4. The LoD for MSSA was confirmed using three strains. Point estimates of the LoD and two-sided confidence interval values for each of the strains tested are shown in Table 5. The tabular results support the Sponsor’s performance claim that the Xpert MRSA/SA Blood Culture Assay will produce a positive MRSA result $9 5 \%$ of the time for a blood culture sample containing $4 0 0 \mathrm { C F U } / 5 0 \mu \mathrm { L }$ per aliquot and positive MSSA results $9 5 \%$ of the time for a blood culture sample containing $3 0 0 \mathrm { C F U } / 5 0 \mu \mathrm { L }$ per aliquot.

Table 4: MRSA LoD and Confidence Intervals (BD BACTEC Plus Aerobic/F)   

<table><tr><td rowspan=2 colspan=1>MRSAStrains</td><td rowspan=2 colspan=1>PFGE ID</td><td rowspan=2 colspan=1>ConfirmedLoD(CFU/test)[at least19/20positivel</td><td rowspan=1 colspan=3>LoD Estimate (ProbitRegressionAnalysis) (CFU/test)</td></tr><tr><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>LoDEstimate</td><td rowspan=1 colspan=1>Upper 95%CI</td></tr><tr><td rowspan=1 colspan=1>Type I(64/4176)</td><td rowspan=1 colspan=1>USA500</td><td rowspan=1 colspan=1>350 (19/20)</td><td rowspan=1 colspan=1>332.3</td><td rowspan=1 colspan=1>366.8</td><td rowspan=1 colspan=1>433.5</td></tr><tr><td rowspan=1 colspan=1>Type II(N315)</td><td rowspan=1 colspan=1>USA100b</td><td rowspan=1 colspan=1>175 (19/20)</td><td rowspan=1 colspan=1>113.7</td><td rowspan=1 colspan=1>137.0</td><td rowspan=1 colspan=1>178.1</td></tr><tr><td rowspan=1 colspan=1>Type III(11373)</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>225 (19/20)</td><td rowspan=1 colspan=1>191.9</td><td rowspan=1 colspan=1>222.6</td><td rowspan=1 colspan=1>273.9</td></tr><tr><td rowspan=1 colspan=1>Type IVa(MWMW2)</td><td rowspan=1 colspan=1>USA400b</td><td rowspan=1 colspan=1>350 (19/20)</td><td rowspan=1 colspan=1>313.1</td><td rowspan=1 colspan=1>356.1</td><td rowspan=1 colspan=1>427.0</td></tr><tr><td rowspan=1 colspan=1>Type V(ST59)c</td><td rowspan=1 colspan=1>USA1000b</td><td rowspan=1 colspan=1>250 (19/20)</td><td rowspan=1 colspan=1>218.2</td><td rowspan=1 colspan=1>243.1</td><td rowspan=1 colspan=1>282.3</td></tr><tr><td rowspan=1 colspan=1>Type VI(HDE288)a</td><td rowspan=1 colspan=1>USA800b</td><td rowspan=1 colspan=1>250 (19/20)</td><td rowspan=1 colspan=1>222.2</td><td rowspan=1 colspan=1>246.0</td><td rowspan=1 colspan=1>385.0</td></tr><tr><td rowspan=1 colspan=1>Type VII(JCSC6082)</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>300 (19/20)</td><td rowspan=1 colspan=1>264.1</td><td rowspan=1 colspan=1>288.0</td><td rowspan=1 colspan=1>347.1</td></tr><tr><td rowspan=1 colspan=1>Type VIII(WAMRSA- 16)</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>400 (19/20)</td><td rowspan=1 colspan=1>348.7</td><td rowspan=1 colspan=1>386.7</td><td rowspan=1 colspan=1>499.1</td></tr><tr><td rowspan=1 colspan=1>Type I(BK2464)</td><td rowspan=1 colspan=1>USA100</td><td rowspan=1 colspan=1>125 (19/20)</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>116.1</td><td rowspan=1 colspan=1>162.0</td></tr><tr><td rowspan=1 colspan=1>Type IVd(BK2529)a</td><td rowspan=1 colspan=1>USA500</td><td rowspan=1 colspan=1>200 (19/20)</td><td rowspan=1 colspan=1>120.8</td><td rowspan=1 colspan=1>148.8</td><td rowspan=1 colspan=1>202.5</td></tr></table>

(a) Heterogeneous Oxacillin-resistant isolates (b) K. Bonnstetter, et al., J Clin Micro 2007, p. 141-146; L. McDougal, et al., J Clin Micro 2003, p. 5113- 5120

Table 5: MSSA LoD and Confidence Intervals (BD BACTEC Plus Aerobic/F)   

<table><tr><td rowspan=2 colspan=1>MRSAStrains</td><td rowspan=2 colspan=1>PFGEID</td><td rowspan=2 colspan=1>Confirmed LoD(CFU/test) [atleast 19/20positivel</td><td rowspan=1 colspan=3>LoD Estimate (ProbitRegression Analysis)(CFU/test)</td></tr><tr><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>LoDEstimate</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>102-04a</td><td rowspan=1 colspan=1>USA1200</td><td rowspan=1 colspan=1>100 (19/20)</td><td rowspan=1 colspan=1>60.4</td><td rowspan=1 colspan=1>74.5</td><td rowspan=1 colspan=1>101.6</td></tr><tr><td rowspan=1 colspan=1>29213b</td><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>150 (19/20)</td><td rowspan=1 colspan=1>120.1</td><td rowspan=1 colspan=1>138.2</td><td rowspan=1 colspan=1>172.7</td></tr><tr><td rowspan=1 colspan=1>N7129a</td><td rowspan=1 colspan=1>USA900</td><td rowspan=1 colspan=1>300 (19/20)</td><td rowspan=1 colspan=1>224.2</td><td rowspan=1 colspan=1>255.2</td><td rowspan=1 colspan=1>314.8</td></tr></table>

Strain Source: (a) American Type Culture Collection (ATCC), Manassas, VA., USA (b) Centers for Disease Control and Prevention (CDC), Atlanta, GA., US

Potential effects on $\mathrm { C t }$ score as a function of blood culture bottle type were tested in an additional LoD confirmatory study. The three claimed blood culture bottle types were each tested with 10 MRSA strains representing MRSA SCCmec types I, II, III, IVa, IVd, V, VII, and VII and three representative MSSA strains. All testing was performed with organism concentrations near or at the confirmed LoD for each individual strain. The mean Ct score per bottle type for the internal positive control and the spa, mec and SCC targets for the MRSA strains are shown in Table 6and for the MSSA strains in Table 7.

Table 6: Summarized Mean Ct Scores for 10 MRSA strains at LoD using Three Blood Culture Bottle Types   

<table><tr><td rowspan=1 colspan=1>MRSA(SCCmecType)</td><td rowspan=1 colspan=1>Blood CultureMedium</td><td rowspan=1 colspan=1>CFU/test</td><td rowspan=1 colspan=1>Positives/20replicates*</td><td rowspan=1 colspan=1>SPCCt</td><td rowspan=1 colspan=1>spaCt</td><td rowspan=1 colspan=1>mecCt</td><td rowspan=1 colspan=1>SCCCt</td></tr><tr><td rowspan=3 colspan=1>I 64/4176</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>400</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>36.4</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>36.6</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>36.4</td></tr><tr><td rowspan=3 colspan=1>II N315</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>175</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>35.1</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=3 colspan=1>III 11373</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>225</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=2 colspan=1>Iva MW2</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=2 colspan=1>350</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.3</td></tr></table>

# Table 6: Summarized Mean Ct Scores for 10 MRSA strains at LoD using Three Blood Culture Bottle Types

<table><tr><td rowspan=1 colspan=1>MRSA(SCCmecType)</td><td rowspan=1 colspan=1>Blood CultureMedium</td><td rowspan=1 colspan=1>CFU/test</td><td rowspan=1 colspan=1>Positives/20replicates*</td><td rowspan=1 colspan=1>SPCCt</td><td rowspan=1 colspan=1>spaCt</td><td rowspan=1 colspan=1>mecCt</td><td rowspan=1 colspan=1>SCCCt</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=3 colspan=1>V ST59</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>250</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>35.9</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=3 colspan=1>VI HDE288^</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>250</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=3 colspan=1>VIIJCSC6082</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>300</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.7</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.8</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.9</td></tr><tr><td rowspan=3 colspan=1>VIII WAMRSA-16</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>400</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>36.6</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>36.5</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>36.8</td></tr><tr><td rowspan=3 colspan=1>II BK2464</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>125</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>35.3</td></tr><tr><td rowspan=3 colspan=1>IVd BK2529^</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>200</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.9</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>34.7</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>34.3</td></tr></table>

(\*) Based upon maximum valid $\mathrm { C t } = 3 6 . 0$ for spa, mecA and 38.0 for SCCmec (^) Heterogeneous oxacillin-resistant isolates

Table 7: Summarized Mean Ct Scores for Three MSSA Strains at LoD for Three Blood Culture Bottle Types   

<table><tr><td rowspan=1 colspan=1>MSSAStrains</td><td rowspan=1 colspan=1>Blood CultureMedium</td><td rowspan=1 colspan=1>CFU/test</td><td rowspan=1 colspan=1>Positives/20replicates*</td><td rowspan=1 colspan=1>SPC Ct</td><td rowspan=1 colspan=1>spa Ct</td></tr><tr><td rowspan=3 colspan=1>102-04</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>100</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>33.4</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>33.4</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.4</td></tr><tr><td rowspan=3 colspan=1>29213</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>150</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>34.5</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>34.2</td></tr></table>

Table 7: Summarized Mean Ct Scores for Three MSSA Strains at LoD for Three Blood Culture Bottle Types   

<table><tr><td rowspan=1 colspan=1>MSSAStrains</td><td rowspan=1 colspan=1>Blood CultureMedium</td><td rowspan=1 colspan=1>CFU/test</td><td rowspan=1 colspan=1>Positives/20replicates*</td><td rowspan=1 colspan=1>SPC Ct</td><td rowspan=1 colspan=1>spa Ct</td></tr><tr><td rowspan=3 colspan=1>N7129</td><td rowspan=1 colspan=1>BD BACTEC</td><td rowspan=3 colspan=1>300</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>34.3</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>34.3</td></tr></table>

(\*) Based upon maximum valid $\mathrm { C t } = 3 6 . 0$ for spa

e. Effect of Competing Amounts of SA on the Limit of Detection of MRSA

The potential competitive inhibitory effect of increasing amounts of MSSA (ATCC strain 29213) relative to MRSA at LoD was evaluated for ten (10) MRSA isolates [SCCmec types I, II (2 isolates), III, IVa, IVd, V, VI, VII, and VIII] including 2 heteroresistant strains. This analytical study was conducted to test MRSA isolates at the claimed Xpert MRSA/SA Blood Culture LoD concentration for each SCCmec type in the presence of MSSA at ten-fold increasing concentrations (i.e., MRSA to MSSA ratios of 1:1, 1:10, 1:1e2, 1:1e3, 1:1e4, 1:1e5, and 1:1e6). MRSA isolates at the claimed LoD concentrations in the absence of MSSA cells were run as controls. Cells used in this study were diluted into a blood culture background matrix. The blood matrix consists of $1 0 ~ \mathrm { m L }$ of whole blood negative for Staphylococcus aureus and 1 BD Bactec Bottle containing $3 0 ~ \mathrm { m L }$ of media. Dilutions were prepared daily and kept on ice prior to testing.

Replicates of 5 were run at MRSA to MSSA ratios of 1:1, 1:10, $1 { : } 1 \mathrm { x } 1 0 ^ { 2 }$ , 1:1 $\mathrm { { x } } 1 \dot { 0 } ^ { 3 }$ , $1 { : } 1 \mathrm { x } 1 0 ^ { 4 }$ , $1 { : } 1 \mathrm { x } 1 0 ^ { 5 }$ and replicates of 20 were run at MRSA to MSSA ratios of $1 { : } 1 \mathrm { x } 1 0 ^ { 6 }$ . Negative and positive controls were included in the study. One replicate of each external was tested.

One of the ten MRSA isolates tested (SCCmec type I, 64/4176) resulted in 18 of 20 positives at a MRSA to MSSA ratio of $1 { : } 1 \bar { \mathrm { ~ x ~ } } 1 0 ^ { 6 }$ . All other sets of 20 run at MRSA to MSSA ratios of $1 { : } 1 \mathrm { x } 1 0 ^ { 6 }$ resulted in 19 of 20 or 20 of 20 MRSA positive results. The observed positivity rates fall within the expected rate based on the assumption of a binomial distribution and are within the Sponsor’s acceptance criteria for the concentration of MRSA tested. The study data supports the performance claim that, under the conditions of this study, no significant competitive inhibitory effects were observed at the analytical LoD for MRSA SCCmec types I, II, III, IVa, IVd, V, VI, VII or VIII in the presence of competing MSSA cells at ratios up to $1 { : } 1 \mathrm { x } 1 0 ^ { 6 }$ .

# f. Analytical reactivity:

A total of 250 isolates (203 MRSA strains and 47 MSSA strains) were tested. Tested strains included known USA300 isolates (37), USA100 (53), USA200 (6), USA400 (1), USA500 (7), USA600 (2), USA700 (3), USA800 (9), USA1000 (8), USA1100 (2), IBERIAN isolates (2), novel mecA isolate MRSA LGA251 (1) and strains designated PFGE types A, B, and F. Oxacillin MIC data was presented for 119 strains including 9 MSSA isolates with values between 0.25 to $8 ~ \mu \mathrm { g / m L }$ and 110 MRSA isolates with values of 4 to $> 3 2 ~ \mu \mathrm { g / m L }$ .

Of the 203 MRSA strains, the following were represented: type I (6), type II (79), type III (1), type IV (91), type V (7), type VI (3), type VII (1), type VIII (1), and type XI (1). The SCCmec type was not known in 13 MRSA strains tested. Isolates were selected to represent the diversity present in the species S. aureus based on its phylogenetic structure as indicated by Cooper and Feil (J. Bacteriol., June 2003 vol., 185 no.11 3307-3316). Selections were made to broadly represent the primary lineages. Lineages that contain MRSA and MSSA, as well as those that contain exclusively MSSA were selected. Pandemic and epidemic strains are included. Isolated “singletons” of distinct clinical significance, such as ST59 and ST121 were also tested. In a number of STs, multiples of a given cassette type were tested; these isolates are not simply replicates of each other, as their spa-type demonstrates, these are distinct genetic backgrounds. To a significant extent as identified by the firm, isolates are reflective of the natural occurrence of these cassettes however not all lineages contain all cassette types.

250 Staphylococcus aureus strains were tested using the Xpert MRSA/SA Blood Culture Assay. Stock cultures were prepared by suspending the bacterial growth from an agar plate grown aerobically at $3 7 ^ { \circ } \mathrm { C }$ for 18 to 24 hours on Tryptic Soy Agar (TSA) in PBS buffer containing $1 5 \%$ glycerol. One mL buffer was added per $1 0 ~ \mathrm { m g }$ cells. Stocks prepared by this procedure contained $1 \mathrm { x } 1 0 ^ { 1 0 }$ to $1 \mathrm { { x } 1 0 ^ { 9 } \dot { C } F U / m L }$ . All strains were tested in triplicate using $1 0 ~ \mu \mathrm { L }$ of cell stock diluted one million-fold. Colony forming units per test were determined by plate counts of the same volume and dilution.

Three no template control replicates and one replicate of each external positive and negative control (Kwik-Stiks, MicroBiologics, St. Cloud, MN) were included in the study.

The Xpert MRSA/SA Blood Culture Assay correctly identified 249 of $2 5 0 S .$ aureus strains; 47 MSSA and 202 of 203 MRSA. Tested strains represent Cooper and Feil Groups 1A, 1B, and 2, 13 SCCmec types and subtypes (I, II, III, IV, IVa, IVb, IVc, IVd, V, VI, VII, VIII and XI), 23 sequence types (STs), 79 spa-types, 15 PFGE types, and 17 clonal complexes (CC). The Xpert MRSA/SA Blood Culture Assay incorrectly identified one (1) S. aureus strain (LGA251) as MSSA instead of MRSA. The S. aureus strain (LGA251) contains a novel mecA gene representing a divergent mecA homologue (mecALGA251 otherwise known as mecC) located in a novel staphylococcal chromosome mec element, designated SCCmec type XI. This was an expected outcome because the mecA primers and probes in the MRSA/SA Blood Culture Assay will not detect the mecC gene in this strain due to mutations in the primer/probe binding regions. The mecC gene in this strain is only $70 \%$ homologous to the mecA gene in other known MRSA strains. The Xpert MRSA/SA Blood Culture Assay does not contain a reverse primer with sufficient homology to amplify the SCCmec (type XI) segment in S. aureus LGA251 strain (i.e. mecC). All 759 runs provided valid GeneXpert results upon initial testing. GeneXpert Dx R1 systems (GX-IV) were used in this study. The no template controls and the external controls gave the expected GeneXpert test results.

Under the conditions of this study, the results of the analytical reactivity (inclusivity) study demonstrated that the Xpert MRSA/SA Blood Culture Assay identified S. aureus strains selected to represent the range of genetic diversity found in the species based on current known of phylogenetic structure, and those known to be currently circulating in the health community. Each of the 37 known USA300 isolates were correctly reported MRSA positive. Empty cassette variants (2), BORSA strains (7) and heteroresistant strains (4) were all correctly identified using the Xpert MRSA/SA Blood Culture Assay. A single MRSA strain containing the mecC genetic variant was incorrectly identified as MSSA as predicted based on sequence homology.

# g. Analytical specificity:

One hundred and one (101) strains phylogenetically related to Staphylococcus aureus or those potentially present in blood culture flora were tested in triplicate. Of the 101 strains tested, 91 were obtained from the American Type Culture Collection (ATCC), 1 was obtained from Culture Collection, University of Göteborg, Sweden (CCUG), 1 was obtained from Teruyo Ito, Juntendo University, Tokyo, Japan, 1 carbapenemase (KPC) producing Klebsiella pneumoniae strain was obtained from National Collection of Type Cultures (NCTC), UK, and 7 were obtained from the Network on Antimicrobial Resistance in Staphylococcus aureus (NARSA).

The organisms tested were identified as either Gram positive (74), Gram negative (24), or yeast (3). Methicillin-sensitive, coagulase negative Staphylococcus, MSCoNS (27) and methicillin- resistant, coagulase negative Staphylococcus, MRCoNS (12) were included. All replicates of the 12 MRCoNS are expected to provide valid mecA Ct values. The organisms were further classified as either aerobic (94) or anaerobic (7).

Stock cultures were prepared by suspending the bacterial growth from an agar plate in PBS buffer containing $1 5 \%$ glycerol. Each strain was tested using 50 $\mu \mathrm { L }$ of culture adjusted to $4 . 5 - 9 . 5 \mathrm { { x i } 0 ^ { \mathrm { { \bar { 8 } } } } \mathrm { { C F U / m L } } }$ or $1 . 7 - 3 . 2 \mathrm { M c F }$ arland units. Positive, negative, and no template controls were also included in the study.

Under the conditions of this study, all of the non-Staphylococcus aureus isolates were reported as MRSA negative; SA negative. All replicates of the 12 MRCoNS provided valid mecA Ct values as expected however the overall assay result was negative (e.g. MRSA negative; SA negative). All positive, negative and no template controls included in the study gave the expected GeneXpert test results.

# h. Assay cut-off:

# Lot Specific Parameters and Assay Settings

Lot specific assay settings are generated for every lot manufactured to account for slight variations in reagent production. The lot specific assay settings (LSP files) are incorporated into the barcode on each cartridge label and are transferred to the GeneXpert Instrument Systems via a barcode scanner prior to initiating the Xpert MRSA/SA Blood Culture Assay.

# General Assay Settings

The valid cycle range for two MRSA targets (spa and mecA) is 3 to 36 and the valid cycle range for the third MRSA target (SCCmec) is 3 to 38 cycles in the Xpert MRSA/SA Blood Culture Assay. The valid cycle range for the SPC is 3 to 40. A default setting of 3 is used as the valid minimum cycle for all targets and the sample processing control (SPC). A total of 40 PCR cycles are performed in the Xpert MRSA/SA Blood Culture Assay. The SPC is designed to be sensitive to deviations in the PCR reaction conditions (temperature and time) and to potentially inhibitory conditions. The SPC should provide an invalid test result in lieu of a believable but erroneous test result.

To obtain a valid SA positive test result, the spa Ct must be reported within the valid cycle range. To obtain a valid MRSA positive test result, the spa, mecA, and SCCmec Cts must be reported within the valid cycle range. To obtain a valid MRSA or SA negative test result, the spa, mecA, and SCCmec Cts must not be reported within the valid cycle range and the SPC Ct must be reported within its valid cycle range. If the SPC falls outside the valid cycle range, the test result is Invalid and the test must be repeated.

A tabular summary of the general assay settings which are fixed for all reagent lots are shown in Table 8.

Table 8: General Assay Settings   

<table><tr><td rowspan=1 colspan=1>Attribute</td><td rowspan=1 colspan=1>Setting</td></tr><tr><td rowspan=1 colspan=1>Background Subtraction</td><td rowspan=1 colspan=1>Always ON</td></tr><tr><td rowspan=1 colspan=1>Background Minimum Cycle</td><td rowspan=1 colspan=1>Default setting = 5</td></tr><tr><td rowspan=1 colspan=1>Background Maximum Cycle</td><td rowspan=1 colspan=1>Manual setting = 30</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (all targets and SPC)</td><td rowspan=1 colspan=1>Manual setting = 20</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis</td><td rowspan=1 colspan=1>Primary</td></tr><tr><td rowspan=1 colspan=1>Boxcar Average Cycles</td><td rowspan=1 colspan=1>Zero (OFF)</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (all targets and SPC)</td><td rowspan=1 colspan=1>Default setting = 3</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (SPC)</td><td rowspan=1 colspan=1>Manual setting = 40*</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (spa, mecA)</td><td rowspan=1 colspan=1>Manual setting = 36</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (SCCmec)</td><td rowspan=1 colspan=1>Manual setting = 38</td></tr></table>

# i. Interfering species

An interfering substances study was conducted with thirteen (13) substances that may be present in blood culture specimens with potential to interfere with the Xpert MRSA/SA Blood Culture Assay. Potentially interfering substances (IS) are listed in Table 9 and include anticoagulated whole blood with ACD, EDTA, Heparin, and Sodium Citrate, human plasma, three blood culture media bottles (Becton Dickinson BACTEC™ Plus Aerobic/F, BioMérieux BacT/ALERT SA Standard Aerobic, and TREK Diagnostics VersaTREK REDOX1 (Aerobic), bilirubin, $\gamma$ -globulin, hemoglobin, triglycerides, and sodium polyanetholesulfonate (SPS). Bilirubin, $\gamma$ -globulin, hemoglobin, and triglycerides were tested at concentrations approximately one log higher than reference levels (http://www.bloodbook.com/ranges.html). SPS was tested at a 10 fold higher concentration than found in blood culture media.

Positive and negative samples were included in this study. Negative samples $( \mathtt { n } = 8 )$ ) were tested per substance to determine the effect on the performance of the sample processing control (SPC). Positive samples $( \mathtt { n } = 8 )$ were tested per substance with two clinical isolates each of MSSA (29213 and 102–04) and MRSA (SCCmec types II and III) spiked near the analytical LoD determined for each isolate. This study was completed in two phases. Statistical significance was determined by comparing cycle threshold $\mathrm { ( C t ) }$ values from tests run in the presence of potentially inhibitory substances to positive and negative buffer controls (One-way ANOVA followed by Dunnett’s pairwise comparison method).

Negative and positive controls were included in the study. One replicate of a negative external control (MSSE) and one replicate of two positive external controls (MSSA and MRSA) were tested.

Table 9: Interfering substances   

<table><tr><td>Substance ID</td><td>Substance</td><td>Ingredient(s)</td></tr></table>

Table 9: Interfering substances   

<table><tr><td rowspan=1 colspan=1>SubstanceID</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Ingredient(s)</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>TET Buffer(control)</td><td rowspan=1 colspan=1>Control</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Whole Blood(ACD)</td><td rowspan=1 colspan=1>Anticoagulant (ACD)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Whole Blood(EDTA)</td><td rowspan=1 colspan=1>Anticoagulant (EDTA)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Whole Blood(Heparin)</td><td rowspan=1 colspan=1>Anticoagulant (Heparin)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Whole Blood(SodiumCitrate</td><td rowspan=1 colspan=1>Anticoagulant (Sodium Citrate)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>HumanPlasma</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>BacT/Alert SAAerobic</td><td rowspan=1 colspan=1>Supplemented Tryptic Soy,0.035% SPS</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>VersaTREKREDOX1 Aerobic</td><td rowspan=1 colspan=1>Supplemented peptone enrichedTryptic</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>BACTEC PlusAerobic/F</td><td rowspan=1 colspan=1>2.75% Soybean-Casein DigestBroth,</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>Bilirubin, 10 mg/dL</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Globulins</td><td rowspan=1 colspan=1>γ-Globulin, 35 mg/dL</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>Hemoglobin, 180 gm/dL</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>SPS</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>Triolein (Glycerol standard), 1.63</td></tr></table>

All negative samples were correctly reported “MRSA NEGATIVE; SA NEGATIVE” using the Xpert MRSA/SA Blood Culture Assay. No statistically significant inhibitory effects were observed on the performance of the SPC for all negative samples in the presence of each potentially interfering substance relative to buffer controls.

All positive replicates of MSSA strain 29213 (225 CFU/test) and MSSA strain 102–04 (225 CFU/test) were correctly reported MRSA Negative; SA Positive using the Xpert MRSA/SA Blood Culture Assay. No statistically significant inhibitory effects were observed on spa Cts in the presence of each potentially interfering substance relative to buffer controls.

All positive replicates were correctly reported MRSA Positive; SA Positive using the Xpert MRSA/SA Blood Culture Assay. A statistically significant inhibitory effect on some $\mathrm { C t }$ values were observed for MRSA strain N315 (300 CFU/test) with the VersaTREK REDOX1 blood culture bottle (p-value $=$ 0.0211) and bilirubin at $1 0 \mathrm { m g / d L }$ (p-value $= 0 . 0 2 8 5$ ) and for MRSA strain 11373 (500 CFU/test) in the presence of EDTA whole blood (p-values $=$ 0.0077 and 0.0039, respectively), bilirubin at $1 0 \mathrm { m g / d L }$ (p-values $= 0 . 0 0 7 2$ and 0.0318, respectively) and Heparin whole blood (p-value $= 0 . 0 0 5 3 $ ). All other Ct values for MRSA positive samples were showed no statistically significant difference from the control reaction. These observed statistically significant differences are not clinically significant because the difference in the cycle threshold (Ct) values relative to the buffer control was less than $1 \mathrm { C t }$ and therefore no false negative results were reported.

The Interfering Substances Study analytically demonstrated that substances found in additional blood culture bottle types did not interfere with the Xpert MRSA/SA Blood Culture Assay under the conditions of this study. However, all clinical study samples were obtained from BD BACTEC™ Plus Aerobic/F, BacT/ALERT® SA (Standard Aerobic), or VersaTREK REDOX $1 ^ { \mathfrak { B } }$ (aerobic) blood culture bottle types and therefore the Intended Use of this device will include only these bottle types.

# j. Carryover

The single–use GeneXpert cartridges contain multiple chambers for holding sample materials, a valve body composed of a plunger and syringe barrel, a rotary valve system for controlling the movement of fluids between chambers, an area for capturing, concentrating, washing, and lysing spores/cells, dry real–time PCR reagents, and an integrated PCR reaction tube that is automatically filled by the instrument. All samples and fluids including amplicons are contained within the disposable cartridge.

This carryover study consisted of a negative sample (elution reagent only) run immediately following a very high positive sample $( 6 \mathrm { x } 1 0 ^ { 7 } \mathrm { M S S A }$ or MRSA cells spiked into the elution reagent) in the same GeneXpert Dx System module. This was repeated 40 times between two GeneXpert Dx Systems (GX-IV) and 2 modules for each instrument. A total of 84 runs per strain were tested (40 positive samples per system per strain and 44 negative samples per system per strain). A negative sample was run on each module prior to the first high positive replicate and reported negative as expected.

Under the conditions of this study, no evidence of specimen or amplicon carry–over contamination was observed. All 40 high MSSA positive replicates and all 40 high MRSA positive replicates were correctly identified as expected. All 88 negative replicates were correctly reported MRSA NEGATIVE; SA NEGATIVE as expected. The external controls (1 replicate each of a negative and 2 positive controls) gave the expected GeneXpert test results.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Prospective clinical study:

Performance characteristics of the Xpert MRSA/SA Blood Culture Assay were determined in a multi-site prospective clinical validation study. Eight U.S. clinical sites participated in the study. All clinical sites were granted waivers of informed consent by their IRB for this study. Inclusion criteria included, but were not limited to: positive blood culture specimen taken from BD BACTEC™ Plus Aerobic/F, BacT/ALERT® SA (Standard Aerobic) or Versa TREK REDOX $1 ^ { \mathfrak { B } }$ (aerobic) blood culture bottles, and Gram Stain containing either Gram Positive Cocci in Clusters (GPCC) or Gram Positive Cocci in singles (GPC).

A total of 869 specimens were initially enrolled in the study, of which 848 were eligible for inclusion. Of the 848 eligible study participants, 792 were included in the final dataset used for the analyses of the Xpert MRSA/SA Blood Culture Assay. Of the 848 eligible study samples, 468 were collected from male and 380 from female participants. The average age was 58.6 years (range $= 1$ month to 96 years).

The first run success rate for the Xpert MRSA/SA Blood Culture assays was $9 6 . 1 \%$ (764/795). The indeterminate cases included 22 ERROR results, one INVALID result, and eight NO RESULT outcomes. Thirty of the 31 indeterminate cases were retested; one specimen was not retested. Twentyeight of the 30 indeterminate cases that were retested yielded valid results upon repeat assay. The overall rate of assay success was $9 9 . 6 \%$ (792/795).

Relative to culture with susceptibility testing, the Xpert MRSA/SA Blood Culture Assay demonstrated a Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) for MRSA of $9 8 . 1 \%$ and $9 9 . 6 \%$ , respectively and the PPA and NPA for SA were $9 9 . 6 \%$ and $9 9 . 5 \%$ , respectively. Performance data from all study sites are presented in the following table:

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Culture</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>SA+/MRSA-</td><td rowspan=1 colspan=1>Negative/NoGrowth</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>XPERTMRSA/SABlood</td><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>SA+/MRSA-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>132</td></tr><tr><td rowspan=1 colspan=1>SA-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>553</td><td rowspan=1 colspan=1>554</td></tr></table>

Page - 26/40   

<table><tr><td>Culture Assay</td><td>Total</td><td>105</td><td>131</td><td>556</td><td>792</td></tr><tr><td rowspan="3"></td><td colspan="3">MRSA: PPA: 103/105</td></tr><tr><td colspan="3">98.1% 95% CI: 93.3%-99.8% 95% CI: 98.7%-99.9%</td></tr><tr><td colspan="3">NPA: 684/687 99.6%</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3">SA:</td></tr><tr><td colspan="3">PPA: 235/236 99.6% 95% CI: 97.7%-100%</td></tr><tr><td colspan="3">NPA: 553/556 99.5% 95% CI: 98.4%-99.9%</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3"></td></tr></table>

4. Clinical cut-off:

A Receiver Operating Characteristics (ROC) analysis was performed on the clinical validation study data in order to validate the use of the published assay cut-offs. The objective was to validate the cut-off Ct values by calculating the maximum PPA (the highest priority) and NPA. PPA and NPA are inversely proportional for changing cut-off values, so as one increases the other will decrease. ROC analysis was used to support and validate the $3 6 \mathrm { C t }$ cut-off chosen for spa, mecA, and for the cut-off of 38 for SCC targets. The analysis demonstrated that the established cut off Ct values maximize PPA without NPA falling below $9 9 \%$ .

# 5. Expected values/Reference range:

Results from the clinical validation study were analyzed by age group. The following tables show the prevalence of false and true results for MRSA (Table 10) and SA (Table 11) analytes as determined by the XPERT MRSA/SA Blood Culture Assay. Results by age group were also analyzed for homogeneity using the Fisher’s Exact Test. The calculated p-values for all age groups are greater than 0.05 indicating no statistically significant difference in prevalence between age categories.

Table 10: MRSA Prevalence by Age Group   

<table><tr><td rowspan=2 colspan=1>Xpert MRSA/SA BloodCulture Assay</td><td rowspan=1 colspan=8>Age Group</td></tr><tr><td rowspan=1 colspan=1>0-20</td><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>51-60</td><td rowspan=1 colspan=1>61-70</td><td rowspan=1 colspan=1>&gt;70</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MRSA True Positives(Prevalence)</td><td rowspan=1 colspan=1>2(9%)</td><td rowspan=1 colspan=1>8(19%)</td><td rowspan=1 colspan=1>11(17%)</td><td rowspan=1 colspan=1>21(17%)</td><td rowspan=1 colspan=1>22(14%)</td><td rowspan=1 colspan=1>15(9%)</td><td rowspan=1 colspan=1>24(11%)</td><td rowspan=1 colspan=1>103(13%)</td></tr><tr><td rowspan=1 colspan=1>MRSA True Negative(Prevalence)</td><td rowspan=1 colspan=1>20(90%)</td><td rowspan=1 colspan=1>35(81%)</td><td rowspan=1 colspan=1>53(82%)</td><td rowspan=1 colspan=1>102(82%)</td><td rowspan=1 colspan=1>131(86%)</td><td rowspan=1 colspan=1>148(90%)</td><td rowspan=1 colspan=1>195(89%)</td><td rowspan=1 colspan=1>684(86%)</td></tr><tr><td rowspan=1 colspan=1>MRSA False Positive(Prevalence)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1(2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3(0.4%)</td></tr><tr><td rowspan=1 colspan=1>MRSA False Negatives(Prevalence)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1(0.8%)</td><td rowspan=1 colspan=1>1(0.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(0.3%)</td></tr><tr><td rowspan=1 colspan=1>Total Specimens Tested</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>792</td></tr></table>

Table 11: SA Prevalence by Age Group   

<table><tr><td rowspan=2 colspan=1>Xpert MRSA/SABlood CultureAssay</td><td rowspan=1 colspan=8>Age Group</td></tr><tr><td rowspan=1 colspan=1>0-20</td><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>51-60</td><td rowspan=1 colspan=1>61-70</td><td rowspan=1 colspan=1>&gt;70</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>SA True Positives(Prevalence)</td><td rowspan=2 colspan=1>7(32%)</td><td rowspan=2 colspan=1>10(23%)</td><td rowspan=2 colspan=1>25(38%)</td><td rowspan=2 colspan=1>45(36%)</td><td rowspan=1 colspan=1>49</td><td rowspan=2 colspan=1>46(28%)</td><td rowspan=2 colspan=1>53(24%)</td><td rowspan=2 colspan=1>235(30%)</td></tr><tr><td rowspan=1 colspan=1>(32%)</td></tr><tr><td rowspan=1 colspan=1>SA True Negative(Prevalence)</td><td rowspan=1 colspan=1>15(68%)</td><td rowspan=1 colspan=1>33(77%)</td><td rowspan=1 colspan=1>40(62%)</td><td rowspan=1 colspan=1>79(64%)</td><td rowspan=1 colspan=1>104(68%)</td><td rowspan=1 colspan=1>118(72%)</td><td rowspan=1 colspan=1>164(75%)</td><td rowspan=1 colspan=1>553(70%)</td></tr><tr><td rowspan=1 colspan=1>SA False Positive(Prevalence)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1(0.6%)</td><td rowspan=1 colspan=1>1(0.6%)</td><td rowspan=1 colspan=1>1(0.5%)</td><td rowspan=1 colspan=1>3(0.3%)</td></tr><tr><td rowspan=1 colspan=1>SA False Negatives(Prevalence)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1(0.5%)</td><td rowspan=1 colspan=1>1(0.1%)</td></tr><tr><td rowspan=1 colspan=1>Total SpecimensTested</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>792</td></tr></table>

# N. Instrum entNam es:

GeneXpert Dx GX-I GeneXpert Dx GX-II GeneXpert Dx GX-IV GeneXpert Dx GX-XVI GeneXpert Infinity-48 GeneXpert Infinity-48s GeneXpert Infinity-80

# O .System Descriptions:

1. Modes of Operation:

The Xpert MRSA/SA Blood Culture Assay is performed on the Cepheid GeneXpert Instrument Systems. The GeneXpert Instrument Systems automate and integrate sample purification, nucleic acid amplification and detection of the target sequences in samples using real-time PCR. The systems consist of an instrument, personal computer, and preloaded software for running the tests and viewing the results. The GeneXpert Instrument Systems require the use of singleuse disposable cartridges that hold the PCR reagents and host the PCR process. The cartridges are self-contained and therefore cross-contamination between samples is reduced. In these platforms, additional sample preparation, amplification, and real-time detection are fully-automated. Results of the Xpert MRSA/SA Blood Culture Assay are displayed in tabular and graphic formats.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development   
processes for this line of product types:   
Yes ___X_ _ or No

# Level of Concern: Moderate

Software Description:

The GeneXpert Instrument Systems consist of random access, closed-system, computer-based software and embedded firmware running dedicated microprocessor-controlled modules to integrate sample preparation, amplification and real-time detection in a single system. The software assigns access by predefined user types – basic, detail, and administrative. The user interface software is designed with a graphical user interface consisting of a monitor with a pointing device (mouse or touchpad) and keyboard. The GeneXpert Infinity also provides touch screen capability.

Once the Xpert MRSA/SA Blood Culture Assay cartridge is loaded into the instrument system, a computer system prepares the instructions to run a test (assay profile) and then downloads the assay profile to the GeneXpert module. The system integrates sample processing and real-time PCR amplification and detection in a single Xpert MRSA/SA cartridge. The Xpert MRSA/SA Blood Culture Assay completes sample preparation and real-time PCR in approximately two hours. A test result is provided when the test is completed. During the test, the software collects test data from the GeneXpert module periodically, analyzes the optical data, and computes the test result. After the test is completed, the result is shown on the user interface and a report can be generated.

The Xpertise software is the user interface for the Cepheid Infinity System which provides functionality for ordering tests as well as automation of loading and unloading of cartridges into GeneXpert modules within the system. The Xpertise user interface builds upon the existing core software functionality for handling GeneXpert modules for cartridge fluidics control, temperature control, optics control, and data analysis by the addition of automation handling for the robotic arm. Each of the GeneXpert Instrument Systems process data in the same manner using the same optics, are calibrated the same, and process signal the same.

The Xpert MRSA/SA Blood Culture Assay cartridges and the GeneXpert modules are the same for all the GeneXpert Instrument Systems and the Xpert MRSA/SA Blood Culture Assay is designed to perform on any of the GeneXpert Instrument System family models. Each GeneXpert module, regardless of the Instrument System, processes one sample at a time. Each module contains a syringe drive for dispensing fluids, an ultrasonic horn for lysing cells or spores, a valve drive for sample movement, and I-CORE thermocycler for performing real-time PCR and detection.

# Device Hazard Analysis:

The device hazard analysis (DHA) and failure mode effects and criticality analysis (FMECA) for the GeneXpert Dx was provided. An FMECA was conducted for the Xpert MRSA/SA Blood Culture Assay, which evaluated hazards associated with assay design, user, and production/manufacturing failure modes. The FMECA document includes the identification of the failure mode and cause, the hazard effect, initial risk assessment, mitigation suggestions and mitigations completed, and the hazard effect and risk remaining after mitigation, providing a conclusion about the acceptability of the residual risk.

False negative results, false positive results, and harm to user were considered the worst case hazard effects to the patient and operator. Risk Acceptability was identified and rated as intolerable (I), ALARP (A) (as low as reasonably practicable), or negligible (N)) in accordance with Cepheid’s Risk Management procedure. Based on the Risk Management Reviews, discussions, the FMECA and the mitigation actions taken, the Xpert MRSA/SA Blood Culture Assay Risk Management Core Team judged that all risks were acceptable (ALARP or negligible) following postrisk implementation of mitigation measures. Information on residual risks will be printed in the Package Insert outlining the warnings and limitations to the device.

Based on the FMECA and DHA conducted for the GeneXpert Instrument Systems, the residual risks identified are acceptable.

Architecture Design Chart:

GeneXpert Software Architecture Chart in Attachment II-14 is acceptable.

Software Requirements Specification (SRS):

The SRS documents for the GeneXpert Dx and GeneXpert Infinity Systems software define the general requirements and documents the detailed functional, performance, interface, design, developmental and other requirements, as well as describes how the SRS will be implemented. The SRS is the baseline for the Verification & Validation (V&V) testing for the software. The software requirements are written to a level of detail which are testable. The SRS documents for the GeneXpert Dx System software, GeneXpert Infinity-48 Xpertise software, and GeneXpert Infinity-80/Infinity-48s Xpertise software are provided as follows:

<table><tr><td rowspan=1 colspan=1>DocumentNumber</td><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>Document Title</td></tr><tr><td rowspan=1 colspan=1>D2828</td><td rowspan=1 colspan=1>GeneXpert DxSystems</td><td rowspan=1 colspan=1>Att II-11_D2828_GX_Dx_SRS</td></tr></table>

Page - 30/40   

<table><tr><td rowspan=1 colspan=1>DocumentNumber</td><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>Document Title</td></tr><tr><td rowspan=1 colspan=1>D8822</td><td rowspan=1 colspan=1>GeneXpert Infinity-48 System</td><td rowspan=1 colspan=1>Att II-12_D8822_Infinity-48_SRS</td></tr><tr><td rowspan=1 colspan=1>D12565</td><td rowspan=1 colspan=1>GeneXpert Infinity-80/48s System</td><td rowspan=1 colspan=1>Att II-13_D12565_GX_InfinityG2_SRS</td></tr></table>

The SRS documents are acceptable as presented.

Software Design Specification (SDS):

The SDS provides a detailed explanation of the design and structure of the software and is presented as a design output. The operation of the system is controlled by three functional units. The lowest level is the Infinity system which is composed of low level devices. The Infinity server runs on a real time platform and interfaces with the Infinity system. The Xpertise software runs the user interface on a “kiosk” pc which communicates with the Infinity server. Below is a list of features which have been defined:

· Common modules and code reuse   
· Superclass and sub-class   
· Client and server   
· Active object vs. user interface object   
· Managers   
· Event managers and even listeners   
· Attach and detach a test   
· Binary data handling   
· GUID

The approach the Firm has taken toward the SDS as a design output is novel. From a regulatory perspective this approach is acceptable since it provides a detailed description of how the software functions and the design principles.

# Traceability Analysis:

The traceability analysis chart links together the system requirements, software requirements, hazard analysis, test reference ID test criteria, and V&V test result. These documents show the relationship between design inputs, design outputs, and hazard analysis. Testing references were provided for the unit, integration and system level testing.

The traceability analysis report is acceptable as presented in document D14662, D6467, D11845.

Software Development Environment Description:

The Cepheid “Software Life Cycle and Development Process SOP” (Document #D0058) describes the overall framework in which product software is conceived, developed, and maintained. Each software project employs a structured life-cycle process, using the Cepheid Product Development Process (CPDP) as a framework. The process consists of a series of phases that are performed incrementally, consisting of activities that result in the definition, creation, or upgrade of specified products. Selection of the specific activities and deliverable products is driven by the application and customer user needs and project risk in accordance with the CPDP. The “Software Configuration Control SOP” defines the process used to establish and maintain configuration control of software products and software contained in company products. Software configuration control activities include both design control of software developmental, demonstration and validation releases, and formal change control of software production releases. Software version identification is maintained following Cepheid’s “Software Version Identification” SOP (Document #D0074) where major and minor versions are used for describing product software releases. Fix letters identify software update releases. The “Software Life Cycle and Development Process SOP” is provided Attachment II-1. The “Software Configuration Control SOP” and the “Software Version Identification SOP” are provided in Attachment II-19.

The software development environment description is acceptable as presented.

# Verification and Validation Testing:

As stated in Section 16.0, each of the GeneXpert Instrument Systems has distinct and instrument-specific software. The current versions of GeneXpert Dx Systems software is 4.4a, the Infinity-48 Xpertise software version 4.3, and the Infinity-80 and Infinity-48s Xpertise software version 6.0a were developed by Cepheid Software Engineering in accordance with the Cepheid “Software Lifecycle and Development Process SOP” (Volume IIa, Attachment II-1). Beginning in the design and development phase of the software project, the versions of each software item used in the product or system are identified, controlled, and tracked per Software Engineering’s “Software Configuration Control SOP” (Attachment II-19).

The purpose of the verification and validation of release versions of the GeneXpert Dx System software, the Infinity-48 Xpertise software, and the Infinity-80/Infinity-48s Xpertise software was to verify that the software performs according to its pre-determined specifications as defined in the respective SRS documents.

Verification activities were performed following Software Engineering’s “Software Lifecycle and Development Process SOP,” and for the GeneXpert Dx

R1 and R2 Systems software and the Infinity Systems Xpertise softwares the verification activities included:

Source code reviews   
· Unit testing   
· Functional testing Testing of implemented risk mitigations   
· Integration Testing Regression Testing

The GeneXpert software validations established, by examination and provision of objective evidence that the particular requirements for the specific intended use can be fulfilled. Software validation activities for the three Instrument Systems were performed following Software V&V’s “Product Software Verification & Validation SOP”, and included for each:

· Installation Qualification at the start of testing System performance testing Verification testing of Defect fixes Verification testing of Risk mitigations Data reduction (algorithms for reporting results) testing   
· Functional Testing   
· Installation Testing Regression Testing   
· Stress Testing GUI Testing Boundary Testing Data Driven Testing Exploratory Testing Gray Box Testing Error Handling Anomaly Testing Automated Testing   
· Simulated Testing (Infinity system and LIS Connectivity) Security and user privileges testing GeneXpert biofunctionality (instrument $^ +$ software $^ +$ reagents) testing Review of Labeling (e.g. installation instructions, operator manual)

The acceptance criteria required that the expected results be achieved and any required data printouts or screen prints of the test data be attached to the completed test protocol. Discrepancies from expected results were logged as Software Change Requests (SCRs). A Software Change Request (SCR) is the mechanism for tracking software defects (also known as “unresolved anomalies”) that are not fixed (or deferred) during a software project or found after a software release.

SCRs, and their corrective or preventative actions, are reviewed by a crossfunctional team to determine the impact on device performance. At the beginning of each software development project (new, updates and upgrades), the outstanding SCRs are reviewed to determine which ones need to be fixed for that software release.

Software may be released with open SCRs which have been acted upon appropriately (for example, covered in the Release Notes, covered in Operators Manuals, etc.) following Cepheid’s Software Defect Tracking SOP.

The GeneXpert Dx Software version 4.4a validation, the GeneXpert Infinity-48 Xpertise software version 4.3 validation, and the GeneXpert Infinity-80 and Infinity-48s Xpertise software version 6.0a validation, were successfully completed. Test failures were reviewed and determined to be due to software defects that can be prevented or corrected if they are encountered. The SCRs open at the end of the validations were appropriately identified and addressed (see Volume IIA, Section 16.11 “Unresolved Anomalies.”)

The test results for the software versions used with the Instrument Systems show that the software has been successfully tested as described above and have been shown to be acceptable for release to customers. Copies of the V&V Summary reports are provided as follows:

<table><tr><td rowspan=1 colspan=1>DocumentNumber</td><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>Document Title</td></tr><tr><td rowspan=1 colspan=1>D15278</td><td rowspan=1 colspan=1>GeneXpert Dx R1/R2Systems</td><td rowspan=1 colspan=1>GeneXpert Dx Software,version 4.4a, Verificationand Validation Summary</td></tr><tr><td rowspan=1 colspan=1>D12618</td><td rowspan=1 colspan=1>GeneXpert Infinity-48System</td><td rowspan=1 colspan=1>GeneXpert Infinity-48System Xpertise Software,version 4.x (4.3),Verification and ValidationSummary</td></tr><tr><td rowspan=1 colspan=1>D14638</td><td rowspan=1 colspan=1>GeneXpert Infinity-80/48sSystem</td><td rowspan=1 colspan=1>GeneXpert Infinity-80Software, version 6.0a,Verification &amp; ValidationSummary</td></tr></table>

The verification and validation test reports are acceptable as presented in the documents listed above.

# Revision Level History:

The current release version of the software for use with the Xpert MTB/RIF Assay is GeneXpert Dx System software version 4.4a, GeneXpert Infinity-48 Xpertise software version 4.3, and the GeneXpert Infinity-80 and Infinity-48s Xpertise software version 6.0a. For purposes of the 510(k) submission, data obtained with previous software revisions were recalculated using the latest revision.

# Unresolved Anomalies:

There are a number of unresolved anomalies associated with the released version of the software for all systems. Anomalies are captured and communicated to the user in the Software Release Notes found in Volume 2 Attachment 23-25. Anomalies are presented in a hazard analysis format including a description of the problem, potential impact on system performance, status and planned action. Although there are numerous anomalies, the risk of each anomaly has been mitigated to an acceptable level.

# EMC Testing:

The GeneXpert Instrument Systems comply with the electromagnetic compatibility (EMC), Low Voltage (LV) and electrical safety (ES) standards for laboratory equipment listed below. Certifications and test reports are on file with the Firm. Compliance with the standard has been confirmed either by the Firm or a third party vendor (TÜV Rheinland and Bay Area Compliance Lab).

The GeneXpert Dx R1 Systems (GX-I, GX-IV and GX-XVI) comply with the following standards:

· EMC (Electromagnetic Compatibility) Directive, 2004/108/EC   
· LVD (Low Voltage Directive) 2006/95/EC IEC 61010-1:2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” EN 61010-1:2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” UL 61010-1:2004 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” EN 61010-2-101:2002 “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” CAN-CSA $2 2 . 2 \mathrm { N o }$ . 61010-1: 2004 2nd Edition “Safety Requirements fo electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements”

· CAN-CSA 22.2 No. 61010-2-101: 2004 “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” WEEE Directive 2002/96/EC

The GeneXpert Dx R2 Systems (GX-I, GX-II, GX-IV and GX-XVI) comply with the following standards:

EMC (Electromagnetic Compatibility) Directive, 2004/108/EC $\mathrm { E N } 5 5 0 1 1 { \cdot } 2 0 0 7 + \mathrm { A } 1 { \cdot } 2 0 0 7$ “Industrial, scientific and medical (ISM) radiofrequency equipment - Electromagnetic disturbance characteristics - Limits an methods of measurements” EN 61326-1:2006 “Electrical Equipment for Measurement and Control and Laboratory Use – EMC Requirements” EN 61326-2-6:2006 “Electrical equipment for measurement, control and laboratory use-EMC requirements-Part 2-6: Particular Requirement for in vitro diagnostic (IVD) medical equipment” FCC Part 15 Rules and Regulations for Information Technology Equipment   
· FCC Part 18 Rules and Regulations for Information Technology Equipment   
· CISPR 11:2004 “Industrial, scientific and medical equipment - Radiofrequency disturbance characteristics - Limits and methods of measurement” (Class A Radiated Emission Requirements) CISPR 22:2006 “Information technology equipment –Radio disturbance characteristics –Limits and methods of measurement” (Class A Radiated Emission Requirements) LVD (Low Voltage Directive) 2006/95/EC IEC 61010-1:2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” EN 61010-1:2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” UL 61010-1:2004 R10.08 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” IEC 61010-2-101:2002 “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” EN 61010-2-101:2002 “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” CAN-CSA $2 2 . 2 \mathrm { N o }$ . 61010-1: 2004 +GI1(R2009) 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” CAN-CSA $2 2 . 2 \mathrm { N o }$ . 61010-2-101: 2004 “Safety Requirements for electric equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment”

WEEE Directive 2002/96/EC

The GeneXpert Instrument System: GeneXpert Infinity-48 (including configurations of 16, 24, 32, 40, and 48 modules) complies with the electromagnetic compatibility (EMC), Low Voltage (LV) and electrical safety (ES) standards for laboratory equipment listed below:

· EMC (Electromagnetic Compatibility) Directive, 2004/108/EC $\operatorname { E N } 5 5 0 1 1 ! . 1 9 9 8 + \mathrm { A } 1 ! . 1 9 9 9 + \mathrm { A } 2 ! 2 0 0 2$ “Industrial, scientific and medical (ISM) radio-frequency equipment - Electromagnetic disturbance characteristics - Limits and methods of measurements” EN 61326-1:2006 “Electrical Equipment for Measurement and Control and Laboratory Use – EMC Requirements” EN 61326-2-6:2006 “Electrical equipment for measurement, control and laboratory use-EMC requirements-Part 2-6: Particular Requirement for in vitro diagnostic (IVD) medical equipment”   
· FCC Part 18 Rules and Regulations for Information Technology Equipment   
· CISPR 22: 199 $7 + \mathrm { A } 1 { : } 2 0 0 0 + \mathrm { A } 2 { : } 2 0 0 3$ “Information technology equipment – Radio disturbance characteristics –Limits and methods of measurement” (Class A Radiated Emission Requirements)   
· LVD (Low Voltage Directive) 2006/95/EC   
· UL 61010-1:2004 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements”   
CAN-CSA $2 2 . 2 \mathrm { N o }$ . 61010-1: 2004 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” EN 61010-1: 2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” WEEE Directive 2002/96/EC

The GeneXpert Infinity-80 System and GeneXpert Infinitity-48s System complies with the electromagnetic compatibility (EMC), Low Voltage (LV) and electrical safety (ES) standards for laboratory equipment listed below:

EMC (Electromagnetic Compatibility) Directive, 2004/108/EC EN 61326-1:2006 “Electrical Equipment for Measurement and Control and Laboratory Use – EMC Requirements” EN 61326-2-6:2006 “Electrical equipment for measurement, control and laboratory use-EMC requirements-Part 2-6: Particular Requirement for in vitro diagnostic (IVD) medical equipment” FCC Part 15 Subparts A and B and CISPR 22:2006 “Information technology equipment –Radio disturbance characteristics –Limits and methods of measurement” (Class A Radiated Emission Requirements) · LVD (Low Voltage Directive) 2006/95/EC · CAN-CSA 22.2 No. 61010-1: 2004 $^ +$ GI1(R2009) 2nd Edition “Safety

Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” IEC 61010-1:2001 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” UL 61010-1:2004 R10.08 2nd Edition “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 1: General Requirements” · CAN-CSA $2 2 . 2 \mathrm { N o }$ . 61010-2-101: 2004 (R2009)“Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2- 101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” IEC 61010-2-101:2002 “Safety Requirements for electrical equipment for measurement, control, and laboratory use-Part 2-101: Particular Requirements for in vitro diagnostic (IVD) medical equipment” · WEEE Directive 2002/96/EC

# 3. Specimen Identification:

To perform a test, the user selects the ‘Create Test’ or ‘Orders’ icon, scans the cartridge barcode, enters or scans the sample ID barcode, selects the assay that has been ordered (for example, Xpert MRSA/SA), and loads the cartridge into the module (for the GeneXpert Dx Systems) or onto the conveyor belt (for the GeneXpert Infinity Systems) to start the test.

# 4. Specimen Sampling and Handling:

Specimens for the Xpert MRSA/SA Blood Culture Assay are obtained from aliquots of positive blood cultures collected in blood culture media bottles that have been confirmed as Gram-positive cocci (singles or clusters). The clinical study evaluated specimens from positive cultures obtained from BD BACTEC™ Plus Aerobic/F, BacT/ALERT SA (Standard Aerobic) blood, and VersaTrek Redox 1 (aerobic) blood cultures bottles. An aliquot of approximately $2 5 0 ~ \mu \mathrm { L }$ of the positive blood culture fluid is transferred to a standard microfuge vial for testing in the Xpert MRSA/SA Blood Culture Assay ( $5 0 \mu \mathrm { L }$ is needed for the test). If the aliquot will be tested within 24 hours, it can be stored at either $2 { - } 8 \mathrm { { ^ \circ C } }$ or at room temperature. If the specimen will not be tested within 24 hours, it can be refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ for up to three days until testing is performed.

The initial drawing and processing of blood culture specimens, performing and reading Gram stains, and withdrawing the blood culture fluid should be conducted according to each institution’s procedures. Once the aliquot is collected it should be transferred to the GeneXpert testing area for testing with the Xpert MRSA/SA Blood Culture Assay. All biological specimens, including used cartridges, should be treated as if capable of transmitting infectious agents, and according to the

institution’s safety procedures for working with chemicals and handling biological samples.

# 5. Calibration:

Optical and thermal calibration of the GeneXpert Instrument Systems is performed by Cepheid at the time of manufacture prior to installation and once yearly or after 2,000 runs per module. The user does not perform any serviceable functions on the instrument. An internal normalization function compensates for any optical degradation between calibrations.

Thermistors are located in the thermal reaction chamber and are calibrated to $\pm$ $0 . 5 0 ^ { \circ } \mathrm { C }$ using National Institute of Standards and Technology (NIST) – traceable standards. During the manufacturing process, the temperature of the heating system is measured at two temperatures: $6 0 \mathrm { { ^ \circ C } }$ and $9 5 \mathrm { { ^ \circ C } }$ . Calibration coefficients that correct for small errors in the raw thermistor readings of the heaters are stored in the memory of each I-CORE module.

The optical system is calibrated using standard concentrations of individual unquenched fluorescent dye conjugated DNA oligomers (dye-oligos). For each optical channel, the signal produced by a tube alone (the blank signal) is subtracted from the raw signal produced by the dye-oligo standard to determine the spectral characteristics. Using the individual spectral characteristics of the pure dye-oligos, signals from an unknown mixture of dye-oligos can be resolved into corrected signals for the individual dye-oligos in the mixture.

Following one year or 2000 runs, the Xpert Calibration cartridge can be used by the user to recalibrate the optical system of the modules on the 6-Color GeneXpert Instrument Systems. The Xpert Calibration kit includes cartridges with reagents for the optical recalibration and performance verification of the module. The 4- Color GeneXpert modules cannot run Xpert Calibration and must be calibrated by Cepheid Service.

# 6. Quality Control:

The Xpert MRSA/SA Blood Culture Assay includes internal reagent controls identified as Sample Processing Control (SPC) and internal system controls identified as Probe Check Control (PCC) and Max Pressure (PSI). An additional control is the System Control Check for Temperature. This check is designed to indicate that the GeneXpert Instrument Systems is operating within validated heating and cooling specifications.

# Internal controls

The SPC verifies that the procedural conditions for the processing of the target SA and MRSA bacteria have occurred within an acceptable range. The SPC consists of Bacillus globigii spores formulated into a dry reagent bead included in each Xpert MRSA/SA Blood Culture cartridge. The SPC verifies the effectiveness of each sample preparation step and reaction tube filling by providing an indication that all reaction components are present and functioning within an acceptable range. Additionally the SPC enables a method for detecting the presence of potential inhibitor(s) in the PCR assay.

Test results are reported INVALID if the SPC fails the valid minimum or maximum Ct specification. If an invalid test result is reported, the test is repeated using a new sample, a new cartridge, and a new reagent. During clinical testing, 1 INVALID test result was reported out of 826 total runs (795 initial tests $+ 3 1$ retests) for eligible subjects $\left( 0 . 1 \% \right)$ . 107 of 553 true negative SA test results reported SPC $\mathrm { C t } = 0$ , and 1 of 1 false negative SA test results reported SPC $\mathrm { C t } =$ 0. In each of these cases, either the spa, mecA, or SCCmec test result was positive and the amplification of the SPC target was competed out as expected.

After sample preparation, bead reconstitution, and reaction tube filling, but prior to thermal cycling, the GeneXpert Instrument System is programmed to perform an additional check that the amplification mixture is in an acceptable state. The PCC verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is considered to PASS if the fluorescence generated during the check routine meets the acceptance criteria. If the PCC fails for any SA or MRSA target or the SPC, a probe check error is reported and the test will not continue. If a probe check error is reported, the test is repeated using a new sample, a new cartridge, and a new reagent.

During clinical testing, 23 test results were reported as ERROR due to PCC failures out of 826 total runs for eligible subjects $( 2 . 8 \% )$ . Sixteen (16) runs failed the minimum PCC setting for the SPC.

In addition to the SPC and PCC controls, another system control monitors the cartridge internal pressure during all sample processing steps. If the maximum cartridge internal pressure setting of 120 psi is exceeded during any fluidic movements, the GeneXpert run is aborted. The maximum setting of 120 ensures the integrity of the cartridge and main valve body filter, preventing the potential for fluidic leaks either internal or external to the cartridge. If a pressure abort is reported, the test is repeated using a new sample, a new cartridge, and a new reagent. Failure rates related to internal pressure are expected to be near $0 . 1 \%$ .

# External controls

Commercially available, ready-to-use materials manufactured by MicroBioLogics® (St. Cloud, Minnesota) were tested for use as external control material with the Xpert MRSA/SA Blood Culture Assay. Per the manufacturer, each lyophilized microorganism is no more than four passages from traceable reference cultures. MicroBioLogics KWIK-STIK™ is a self-contained package including a lyophilized microorganism pellet, reservoir of hydrating fluid, and inoculating swab. Each KWIK- STIK unit has a peel-off identification label for easy documentation. The selected KWIK-STIK controls include a negative control containing methicillin-sensitive Staphylococcus epidermidis cells, methicillin-sensitive Staphylococcus aureus cells, and methicillin- resistant Staphylococcus aureus cells. Replicates of 20 each were run per the manufacturer’s instructions using the Xpert MRSA/SA Blood Culture Assay.

All MRSA positive control replicates were correctly reported MRSA POSITIVE; SA POSITIVE. The spa Cts ranged from 23.1 to 26.4 (mean $\mathrm { C t } = 2 4 . 6 )$ . The mecA Cts ranged from 23.8 to 27.3 (mean $\mathrm { C t } = 2 5 . 3 $ ). The SCCmec Cts ranged from 25.0 to 28.2 (mean $\mathrm { C t } = 2 6 . 3 $ . One of 20 replicates reported a Probe Check ERROR. This sample was repeated and resulted in the correct call.

All MSSA positive control replicates were correctly reported MRSA NEGATIVE; SA POSITIVE. The spa Cts ranged from 25.1 to 28.7 (mean $\mathrm { C t } =$ 26.7). There were 3 of the 20 replicates that resulted in mecA signal. Two samples reported late mecA Cts (37.4 and 38.9) that were above the maximum valid cycle of 36.0 and 1 sample reported a valid mecA Ct (35.7). There were no SCCmec Cts reported.

All negative controls (MSSE) were correctly reported MRSA NEGATIVE; SA NEGATIVE. Two of 20 replicates reported late Cts for mecA (both 37.3) that were above the maximum valid cycle of 36.0. No spa or SCCmec Cts were reported.

Of the total 60 runs, 1 provided an indeterminate GeneXpert result (ERROR). This sample was repeated such that 20 replicates were run per KWIK-STIK control. The KWIK-STIK external controls gave the expected results using the Xpert MRSA/SA Blood Culture Assay. The sponsor is recommending these external controls to the end users in the package insert.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered in the“Performance Characteristics” Section above: None

Q .Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.