# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number: k130914

# B. Purpose for Submission:

To obtain clearance for the FilmArray® Blood Culture Identification (BCID) Panel

# C. Measurand:

Enterococci, Listeria monocytogenes, Staphylococci (including specific differentiation of Staphylococcus aureus), Streptococci (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes), Acinetobacter baumannii, Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens), Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa, Candida albicans, Candida glabrata, Candida krusei, Candida parapsilosis, and Candida tropicalis.

# D. Type of Test:

A multiplexed nucleic acid-based test intended for use with the FilmArray® instrument for the qualitative in vitro detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants of antimicrobial resistance. The BCID assay is performed directly on positive blood culture samples that demonstrate the presence of organisms as determined by Gram stain.

# E. Applicant:

BioFire Diagnostics, Inc.

F. Proprietary and Established Names:

FilmArray® Blood Culture Identification (BCID) panel

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3365 - Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures

2. Classification: Class II

3. Product codes: PAM, PEN, PEO, OOI

4. Panel: 83 (Microbiology)

# H. Intended Use:

1. Intended use(s):

The FilmArray Blood Culture Identification (BCID) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with the FilmArray Instrument. The FilmArray BCID Panel is capable of simultaneous detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants of antimicrobial resistance. The BCID assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system that demonstrates the presence of organisms as determined by Gram stain.

The following gram-positive bacteria, gram-negative bacteria, and yeast are identified using the FilmArray BCID Panel: Enterococci, Listeria monocytogenes, commonly encountered Staphylococci (including specific differentiation of Staphylococcus aureus), commonly encountered Streptococci (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes), Acinetobacter baumannii, commonly encountered Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens), Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa, Candida albicans, Candida glabrata, Candida krusei, Candida parapsilosis, and Candida tropicalis.

The FilmArray BCID Panel also contains assays for the detection of genetic determinants of resistance to methicillin (mecA), vancomycin (vanA and vanB), and carbapenems $( b l a _ { \mathrm { K P C } } )$ to aid in the identification of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, and carbapenems exist.

FilmArray BCID is indicated as an aid in the diagnosis of specific agents of bacteremia and fungemia and results should be used in conjunction with other clinical and laboratory findings. Positive FilmArray results do not rule out co-infection with organisms not included in the FilmArray BCID Panel. FilmArray BCID is not intended to monitor treatment for bacteremia or fungemia.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the FilmArray BCID Panel, and for species determination of some Staphylococci, Enterococci, Streptococci, and Enterobacteriaceae that are not specifically identified by the FilmArray BCID Panel assays.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: For use with the FilmArray instrument

# I. Device Description:

The FilmArray Blood Culture Identification (BCID) Panel is a multiplex nucleic acid test designed to be used with the FilmArray Instrument. The FilmArray BCID pouch contains freeze-dried reagents to perform nucleic acid purification and nested, multiplex PCR with DNA melt analysis. The BCID Panel simultaneously tests a single blood culture sample for 24 different organisms and organism groups that cause bloodstream infections and three genetic markers that are known to confer antimicrobial resistance. Targeted organisms and resistance genes are listed in the following table.

<table><tr><td rowspan=1 colspan=1>Gram-Positive Bacteria</td><td rowspan=1 colspan=1>Gram-Negative Bacteria</td><td rowspan=1 colspan=1>Yeast</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>Candida glabrata</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Candida krusei</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Candida parapsilosis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Candida tropicalis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Antimicrobial resistance genes</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>mecA - methicillin resistance</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>vanA/B - vancomycin resistance</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>KPC - carbapenem resistance</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis (encapsulated)</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr></table>

A test is initiated by loading Hydration Solution and a positive blood culture sample mixed with the provided Sample Buffer into the FilmArray BCID pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is

prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray Instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed PCR reaction which includes all primers of the outer primer sets. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green® Plus+, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The $2 ^ { \mathrm { n d } }$ stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the $2 ^ { \mathrm { n d } }$ stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the $2 ^ { \mathrm { n d } }$ stage PCR captures fluorescent images of the PCR reactions and software interprets the data. The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism and antimicrobial resistance gene on the panel.

Test results for antimicrobial resistance genes are only reported when an associated organism (as shown in the following table) is detected in the same test.

Antimicrobial Resistance Genes and Associated Organisms   

<table><tr><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td><td rowspan=1 colspan=1>Associated Organism</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Staphylococcus</td></tr><tr><td rowspan=1 colspan=1>vanA/B*</td><td rowspan=1 colspan=1>Enterococcus</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>Any Enterobacteriaceae,A. baumannii, and/orP. aeruginosa</td></tr></table>

While many of the individual assays within the BCID Panel detect one specific organism, the following BCID assays can detect multiple organisms within a genus or family:

Enterococcus: Detects most species of Enterococcus (see analytical reactivity study below).

Staphylococcus: The BCID Panel contains three assays for the detection of Staphylococcus species. The Staphylococcus aureus assay and two multi-species assays (Staphylococcus1 and Staphylococcus2). The Saureus assay detects all strains of S. aureus and does not cross-react with other organisms, including other species of Staphylococcus. The multi-species assays detect the most prevalent coagulasenegative Staphylococcus (CoNS) species encountered in blood culture specimens and can also react with high levels of S. aureus. The FilmArray Software integrates the results of the three Staphylococcus assays into a final Staphylococcus test result. If all three assays are negative, the test result will be Staphylococcus Not Detected. If any of the three assays is positive, the result will be Staphylococcus Detected. Results for the Saureus assay (positive or negative) determine the Staphylococcus aureus test result (Detected or Not Detected, respectively).

Streptococcus: The BCID Panel contains four assays for the detection of Streptococcus species. Species-specific assays are included for the detection of Group A Strep (Spyogenes), Group B Strep (Sagalactiae), and Spneumoniae. The fourth assay is a multi-species assay (Streptococcus) designed to react with select Viridans group and other Streptococcus species encountered in blood culture specimens. However, the BCID Panel may not detect all Streptococcus species. The FilmArray Software integrates the results of all four Streptococcus assays into a final Streptococcus result as shown in the table below. If all of the assays are negative, the test result will be Streptococcus Not Detected. Alternatively, if any of the four assays are positive, the test result will be Streptococcus Detected. Results for each speciesspecific assay are also reported independently.

Enterobacteriaceae: The BCID Panel includes seven assays to detect members of the Enterobacteriaceae family. Six genus/species specific assays are included for the detection of Enterobacter cloacae (and other $E$ . cloacae complex species); Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus spp., and Serratia marcescens. A seventh assay (the Enteric assay) will react with some (not all) species detected by the other six assays; however, its primary function is to detect other less common, but clinically relevant members of the Enterobacteriaceae family. Combined, these seven assays will detect many, but not all Enterobacteriaceae (see Analytical Reactivity section below). A positive result for any of the seven Enterobacteriaceae-associated assays will generate an Enterobacteriaceae Detected result. Each specific genus/species assay result will also be reported independently. Results for the Enteric assay are not reported independently, but are incorporated into the Enterobacteriaceae test result. Negative results for all seven assays will generate an Enterobacteriaceae Not Detected result.

Enterobacter cloacae complex: The Enterobacter cloacae complex is comprised of six species (E. asburiae, E. cloacae, $E$ . hormaechei, $E .$ . kobei, $E .$ . ludwigii, and $E$ . nimipressuralis) that may all be identified as $E .$ . cloacae by phenotypic laboratory methods. Of the six species in the complex, the BCID Panel Ecloacae assay detects $E$ cloacae (subspecies cloacae and dissolvens), $E$ . asburiae, and $E .$ . hormaechei.

Proteus: The BCID Panel Proteus assay detects four of five characterized species within the genus $P .$ . mirabilis, $P$ . hauseri, $P .$ . penneri, and $P$ . vulgaris).

Materials provided in each kit:

Individually packaged FilmArray BCID Panel Pouches: The BCID pouches are used to test the patient samples. Each reagent pouch is packaged in a metal canister under vacuum.   
Single-use Sample Buffer vials: The Sample Buffer serves to inactivate RNases in the sample and promote binding of nucleic acids to the magnetic beads for extraction. Single-use Hydration Solution vials: Each single use vial is used to rehydrate the freeze-dried reagents contained in the FilmArray BCID pouch.   
Individually packaged transfer pipettes: The Transfer Pipettes are used to mix the blood culture sample with Sample Buffer. An alternative workflow is also described whereby the Transfer Pipette is used to transfer approximately $1 0 0 \mu \mathrm { L }$ of blood culture to the single use Sample Buffer vials prior to mixing.   
Individually packaged Sample Loading Syringes with attached cannula (red cap): Used for adding the patient sample/ buffer mixture to the pouch.   
Individually packaged Pouch Hydration Syringes with attached cannula (blue cap): Used for adding Hydration Solution to the pouch prior to testing.

# Materials required but not provided:

FilmArray® Instrument: The FilmArray System is composed of the FilmArray Instrument and a laptop computer loaded with FilmArray Software. The FilmArray Software controls the function of the instrument and collects, analyzes, and stores data generated by the instrument.   
FilmArray Pouch Loading Station: The FilmArray Pouch Loading Station is used for hydrating the BCID pouch and loading of the sample/Sample Buffer mixture. Syringes with a 28-gauge needle capable of measuring $0 . 1 \mathrm { m L }$ $( 1 0 0 \mu \mathrm { l } )$ sample volume.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Nanosphere Verigene® Gram-Positive Blood Culture Nucleic Acid Test

2. Predicate 510(k) number(s): k122514

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device: FilmArray BCID Panel</td><td colspan="1" rowspan="1">Predicate: NanosphereVerigeneRGram-PositiveBlood Culture Nucleic Acid Test</td></tr><tr><td colspan="1" rowspan="1">OrganismsandResistanceMarkersDetected</td><td colspan="1" rowspan="1">Enterococci, Staphylococci (includingspecific differentiation ofStaphylococcus aureus), Streptococci(with specific differentiation ofStreptococcus agalactiae,Streptococcus pneumoniae, andStreptococcus pyogenes) andresistance markers mecA, vanA, andvanB.</td><td colspan="1" rowspan="1">SameSee below for differences</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid-based</td><td colspan="1" rowspan="1">SameSee below for differences</td></tr><tr><td colspan="1" rowspan="1">SampleProcessingandPurification</td><td colspan="1" rowspan="1">Automated by instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in eachreagent pouch to control for sampleprocessing and both stages of PCRand melt analysis.</td><td colspan="1" rowspan="1">Two Internal Processing Controls(whole organism complete assaycontrol and single-stranded DNAHybridization control)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device: FilmArray BCID Panel</td><td colspan="1" rowspan="1">Predicate: NanosphereVerigeneGram-PositiveBlood Culture Nucleic Acid Test</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Positive blood culture samplescontaining gram-positive bacteria,gram-negative bacteria,and/or yeast.</td><td colspan="1" rowspan="1">Positive blood culture bottlescontaining gram-positive bacteria.</td></tr><tr><td colspan="1" rowspan="1">Organisms andResistanceMarkersDetected</td><td colspan="1" rowspan="1">Detection of additional targets:Listeria monocytogenes,Acinetobacter baumannii,Enterobacteriaceae (includingspecific differentiation ofEnterobacter cloacae complexspecies, Escherichia coli, Klebsiellaoxytoca, Klebsiella pneumoniae,Proteus, and Serratia marcescens),Haemophilus influenzae, Neisseriameningitidis, Pseudomonasaeruginosa, Candida albicans,Candida glabrata, Candida krusei,Candida parapsilosis, Candidatropicalis, and resistance markerblaKPC</td><td colspan="1" rowspan="1">Tests for gram-positive bacteria.Tests for Listeria spp. rather thanListeria monocytogenes. Includestesting for additional Staphylococcusspp.: Staphylococcus epidermidis,Staphylococcus lugdunensis, as wellas testing for specific Enterococcusspp.: Enterococcus faecalis,Enterococcus faecium. Includestesting for an additionalStreptococcus spp.: Streptococcusanginosus group. Does not includetesting for blaKPC. Specificallydifferentiates between vanA andvanB resistance markers.</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Nested multiplex PCR followed byhigh resolution melting analysis toconfirm identity of amplified product.</td><td colspan="1" rowspan="1">Qualitative, multiplexed test for thedetection of specific nucleic acidtargets in a microarray format usingcapture and mediator oligonucleotidesfor gold nanoparticle probe-basedendpoint detection.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray Instrument</td><td colspan="1" rowspan="1">Verigene Reader and Processor SP</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Less than 1 hour</td><td colspan="1" rowspan="1">2.5 hours</td></tr></table>

# K. Standard/Guidance Document Referenced

Draft Guidance for Industry and Food and Drug Administration Staff - Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices, (November 9, 2012) Draft Guidance for Industry and FDA Staff - Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of methicillinresistant Staphylococcus aureus (MRSA) for Culture Based Devices (June 15, 2011) Draft Guidance for Industry and Food and Drug Administration Staff - Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA) (January 5, 2011) Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, FDA Guidance Document (March 13, 2007) User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute (CLSI) Approved Guideline – Second Edition, EP12- A2 (January 2008)   
· Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline, MM3-A2 (February 2006)   
· Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005) Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005)   
Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999) General Principle of Software Validation; Final Guidance for Industry and FDA Staff (January 11, 2002)

# L. Test Principle:

The FilmArray BCID pouch is a closed system that houses all the chemistry required to isolate, amplify, and detect nucleic acid from multiple bloodstream pathogens within a single blood culture sample. The user of the FilmArray BCID Panel loads the sample into the FilmArray BCID pouch, places the pouch into the FilmArray Instrument, and starts the run and the following processes occur automatically during the FilmArray run:

Nucleic Acid Purification - Nucleic acid purification occurs in the first three blisters of the pouch. The sample is lysed by agitation (bead beating) and the liberated nucleic acid is captured, washed and eluted using magnetic bead technology. These steps require about ten minutes.

$\mathbf { 1 ^ { s t } }$ Stage Multiplex PCR – After nucleic acid purification, the purified nucleic acid solution is combined with a preheated master mix to initiate thermocycling for multiplex PCR. During this stage, the FilmArray performs a single, large volume, highly multiplexed PCR reaction which includes all primers of the outer primer sets.

The effect of $1 ^ { \mathrm { s t } }$ stage PCR is to enrich for the target nucleic acids present in the sample.

$2 ^ { \mathbf { n d } }$ Stage PCR - The products of $1 ^ { \mathrm { s t } }$ stage PCR are diluted and mixed with fresh PCR reagents containing a double stranded DNA binding dye (LCGreen® Plus, BioFire Diagnostics, Inc.). This solution is distributed over the $2 ^ { \mathrm { n d } }$ stage PCR array. The individual wells of the array contain primers for different assays (each present in triplicate) that target specific nucleic acid sequences from each of the pathogens detected, as well as control template material. These primers are ‘nested’ or internal to the specific products of the $1 ^ { \bar { \mathrm { s t } } }$ stage multiplex reaction, which enhances both the sensitivity and specificity of the reactions.

DNA Melting Analysis – After $2 ^ { \mathrm { n d } }$ stage PCR, the temperature is slowly increased and fluorescence in each well of the array is monitored and analyzed to generate a melt curve. The temperature at which a specific PCR product melts (melting temperature or Tm) is consistent and predictable and the FilmArray Software automatically evaluates the data from replicate wells for each assay to report results. The FilmArray Software controls the operation of the instrument, collects and analyzes data, and automatically generates a test report at the end of the run.

# M. Performance Characteristics:

1. Analytical performance:

a. Reproducibility:

A multicenter reproducibility study was performed to determine between-site and overall reproducibility of the BCID Panel. Reproducibility testing occurred at three test sites using a panel of six simulated blood culture specimens, each spiked with various combinations of two different organisms. Specimens were prepared in a matrix of human whole blood and blood culture media. In total, 12 organisms were evaluated to generate positive data for 17 of the possible 27 BCID Panel test results, including antimicrobial resistance genes.

To best represent the composition of specimens likely to be tested by the BCID Panel, half of the replicates were at a concentration consistent with the level of organism in a blood culture bottle at the time of positivity, and half were at a concentration similar to that observed in bottles eight hours after positivity. The concentrations used were determined based on the results of the Growth and Detection Studies (see below). Final organism dilutions were plated to confirm the concentration of organism in each sample. A negative blood culture panel member was also included in the study.

The study incorporated potential variances: seven different operators, three different pouch lots, and 10 different FilmArray Instruments. Over the course of four weeks, samples were tested on eight different days, for a total of 90 replicates per analyte and per concentration.

Valid results were attained for 540 of 547 runs $( 9 8 . 7 \% )$ . The seven remaining runs included four runs with Instrument Communication Errors $( 4 / 5 4 7 = 0 . 7 3 \% )$ and three runs with control failures $( 3 / 5 4 7 = 0 . 5 5 \% )$ . Expected positive (Detected) test results were obtained in all 540 completed test runs $( 1 8 0 0 / 1 8 0 0 = 1 0 0 \%$ agreement with the expected result). Expected negative (Not Detected) or N/A results were obtained in 539/540 pouch runs. Correct negative results were 12,776/12,780 or $9 9 . 9 7 \%$ agreement with the expected result. The four false positive results were generated from a single pouch run that was determined to be due to a pouch failure.

A summary of results (percent agreement with the expected result) for each analyte is provided in the following tables:

Summary of Reproducibility Results – Organism Assays   

<table><tr><td colspan="1" rowspan="2">BCID PanelTest Result</td><td colspan="1" rowspan="2">Organism TestedTest Concentration</td><td colspan="1" rowspan="2">Test Site</td><td colspan="2" rowspan="1">Results</td><td colspan="1" rowspan="2">% AgreementwithExpectedResult</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="12">Enterococcus</td><td colspan="1" rowspan="4">Enterococcus faecium[vanA]JMI4751.50E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="8">180/180100%[98.0% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis[vanB]JMI 3688.95E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="4">360/360100%[99.0% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/360</td><td colspan="1" rowspan="1">360/360</td></tr><tr><td colspan="1" rowspan="4">Listeriamonocytogenes</td><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="4">540/540100%[99.3% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td></tr><tr><td colspan="1" rowspan="8">Staphylococcus</td><td colspan="1" rowspan="4">Staphylococcus aureus[MRSA]ATCC BAA-17478.60E+06 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/90100%[96.0% -100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="4">449/450a99.8%[98.8% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">1/150 a</td><td colspan="1" rowspan="1">149/150</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">1/450</td><td colspan="1" rowspan="1">449/450</td></tr><tr><td colspan="1" rowspan="6">Staphylococcusaureus</td><td colspan="1" rowspan="4">Staphylococcus aureus[MRSA]        ATCCBAA-17478.60E+06 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/90100%[96.0% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="2">−</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="2">450/450100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="2">[99.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/450</td><td colspan="1" rowspan="1">450/450</td></tr><tr><td colspan="1" rowspan="8">Streptococcus</td><td colspan="1" rowspan="4">Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/9000%[96.0% -100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="2">450/450100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="2">[99.2% -100%]</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/450</td><td colspan="1" rowspan="1">450/450</td></tr><tr><td colspan="1" rowspan="4">Streptococcusagalactiae</td><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="2">540/540100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="2">[99.3% -10%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td></tr><tr><td colspan="1" rowspan="4">Streptococcuspneumoniae</td><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="4">540/540100%[99.3% -10%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td></tr><tr><td colspan="1" rowspan="8">Streptococcuspyogenes</td><td colspan="1" rowspan="4">Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="2">90/90100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="2">[96.0% -1 0%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="4">450/45000%[99.2%-100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/450</td><td colspan="1" rowspan="1">450/450</td></tr><tr><td colspan="1" rowspan="8">Acinetobacterbaumannii</td><td colspan="1" rowspan="4">Acinetobacter baumanniiATCC 99552.00E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/90100%[96.0%-0%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="4">450/450100%[99.2% -10 %</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/450</td><td colspan="1" rowspan="1">450/450</td></tr><tr><td colspan="1" rowspan="12">Enterobacteriacea</td><td colspan="1" rowspan="4">Klebsiella pneumoniae[KPC]JMI 7669.40E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="8">180/180100%[98%- 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="4">360/36000%[99.0%-1 0%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/360</td><td colspan="1" rowspan="1">360/360</td></tr><tr><td colspan="1" rowspan="3">Enterobactercloacae complex</td><td colspan="1" rowspan="3">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="3">540/540100%[99.3% -</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr></table>

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=2>Results</td><td rowspan=2 colspan=2>% AgreementwithExpectedResult</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>NotDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=2>100%</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=2 colspan=2>540/540100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=2 colspan=2>[99.3% -P10 0%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Klebsiellaoxytoca</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td><td rowspan=4 colspan=1>540/540100%[99.3% -100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td></td></tr><tr><td rowspan=8 colspan=1>Klebsiellapneumoniae</td><td rowspan=4 colspan=1>Klebsiella pneumoniae KPC]JMI 7669.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=2>90/90100%[96.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=2 colspan=2>450/450100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=2 colspan=2>[99.2% -10 %</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>Proteus</td><td rowspan=4 colspan=1>Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td><td rowspan=4 colspan=1>90/90100%[96.0% -10%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>[</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td></td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=2>450/450100%[99.2% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Serratiamarcescens</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>540/540100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=2 colspan=2>[99.3% -10 0%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Haemophilusinfluenzae</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td><td rowspan=4 colspan=1>539/540 a98.0%99.%-100 %</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/180 a</td><td rowspan=1 colspan=1>179/180</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/540</td><td rowspan=1 colspan=1>539/540</td><td></td></tr><tr><td rowspan=4 colspan=1>Neisseriameningitidis</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=2>540/540100%[99.3% -10%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>Pseudomonasaeruginosa</td><td rowspan=4 colspan=1>Pseudomonas aeruginosaATCC 278531.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td><td rowspan=4 colspan=1>90/90100%96.0%-10 0%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td></td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=2>450/450100%[99.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=4 colspan=1>Candida albicans</td><td rowspan=4 colspan=1>Candida albicansATCC 102313.10E+04</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=2>90/90100%[96.0%-100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr></table>

<table><tr><td rowspan=2 colspan=1>BCID PanelTest Result</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=2>Results</td><td rowspan=2 colspan=1>% AgreementwithExpectedResult</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>NotDetected</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Candida glabrata</td><td rowspan=4 colspan=1>Candida glabrataATCC 155452.00E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=8 colspan=1>Candida krusei</td><td rowspan=4 colspan=1>Candida kruseiATCC 908783.20E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=4 colspan=1>Candidaparapsilosis</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=4 colspan=1>539/540 a99.8%[99.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>1/180 a</td><td rowspan=1 colspan=1>179/180</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/540</td><td rowspan=1 colspan=1>539/540</td></tr><tr><td rowspan=8 colspan=1>Candidatropicalis</td><td rowspan=4 colspan=1>Candida tropicalisATCC 660299.70E+05</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90100%[96.0% -100%]</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450100%[99.2% -100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr></table>

a A single pouch run at Site B generated four false positive results: Staphylococcus, mecA , Haemophilus influenzae, and Candida parapsilosis.

Summary of Reproducibility Results: Antimicrobial Resistance Assays   

<table><tr><td colspan="1" rowspan="2">BCIDPanelTestResult</td><td colspan="1" rowspan="2">Organism TestedTest Concentration</td><td colspan="1" rowspan="2">TestSite</td><td colspan="3" rowspan="1">Results</td><td colspan="1" rowspan="2">% Agreementwith ExpectedTest Result</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="7">vanA/B</td><td colspan="1" rowspan="4">Enterococcus faecium[vanA]JMI4751.50E+08 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="7">180/180100%[98.0% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="3">Enterococcus faecalis[vanB]JMI 368</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">8.95E+08 CFU/mL</td><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">No Associated Organism</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="4">360/360100%[99.0% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">0/120</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">0/360</td><td colspan="1" rowspan="1">0/360</td><td colspan="1" rowspan="1">360/360</td></tr><tr><td colspan="1" rowspan="8">mecA</td><td colspan="1" rowspan="4">Staphylococcus aureus[MRSA]        ATCCBAA-17478.60E+06 CFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/90100%[96.0% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">No Associated Organism</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td><td colspan="1" rowspan="4">449/450a99.8%[98.8% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">1/150a</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">149/150</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">0/150</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">1/450</td><td colspan="1" rowspan="1">0/450</td><td colspan="1" rowspan="1">449/450</td></tr><tr><td colspan="1" rowspan="12">KPC</td><td colspan="1" rowspan="4">Klebsiella pneumoniae[KPC]JMI 7669.40E+08</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="4">90/90100%[96.0% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">0/30</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilisATCC 29906andPseudomonas aeruginosaATCC 27853</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="4">270/270100%[98.6% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">0/270</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">0/270</td></tr><tr><td colspan="1" rowspan="4">No Associated Organism</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="4">180/180100%[98.0% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td></tr></table>

a A single pouch run at Site B generated a false positive result for mecA.

The reproducibility of Tm values for each analyte was evaluated and a summary is provided in the following tables.

Summary of Tm Analysis for Positive Organism Assays   

<table><tr><td rowspan=2 colspan=1>BCID Panel Assay</td><td rowspan=2 colspan=2>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>TestSite</td><td rowspan=1 colspan=5>Reproducibility of Tm</td></tr><tr><td rowspan=1 colspan=1>TmMean</td><td rowspan=1 colspan=1>TmStd Dev</td><td rowspan=1 colspan=1>TmMinimum</td><td rowspan=1 colspan=1>TmMaximum</td><td rowspan=1 colspan=1>ObservedRange(Max-Min)</td></tr><tr><td rowspan=1 colspan=7>Gram-Positive Bacteria</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Enterococcus</td><td rowspan=3 colspan=2>Enterococcus faecium[vanA]JMI475</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>82.3</td><td rowspan=1 colspan=1>83.0</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>0.9</td></tr></table>

<table><tr><td rowspan=2 colspan=1>BCID Panel Assay</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>TestSite</td><td rowspan=1 colspan=5>Reproducibility of Tm</td></tr><tr><td rowspan=1 colspan=1>TmMean</td><td rowspan=1 colspan=1>TmStd Dev</td><td rowspan=1 colspan=1>TmMinimum</td><td rowspan=1 colspan=1>TmMaximum</td><td rowspan=1 colspan=1>ObservedRange(Max-Min</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>1.50E+08 CFU/mL</td><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>82.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=4 colspan=1>Enterococcus faecalisvanBJM1 3688.95E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>82.4</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>81.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1>82.4</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=4 colspan=1>Saureus</td><td rowspan=4 colspan=1>Staphylococcus aureus MRSA]       ATCCBA-17478.60E+06 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>77.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>76.6</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>77.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>76.8</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>76.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>76.5</td><td rowspan=1 colspan=1>77.5</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>77.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>76.5</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=4 colspan=1>Streptococcus</td><td rowspan=4 colspan=1>Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>83.6</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>82.3</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>83.6</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=4 colspan=1>Spyogenes</td><td rowspan=4 colspan=1>Streptococcus pyogenesATCC 196155.70E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>78.5</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.7</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.5</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.5</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>canmaGellowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>um-Negati</td><td rowspan=1 colspan=1>e Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Abaumannii</td><td rowspan=4 colspan=1>AcinetobacterbaumanniATC A 99552.00E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>80.4</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>80.9</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=4 colspan=1>Enteric</td><td rowspan=4 colspan=1>Klebsiella pneumoniaeKPC]JM 7669.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>88.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>89.1</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>88.6</td><td rowspan=1 colspan=1>89.2</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>88.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>88.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>89.2</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=4 colspan=1>Kpneumoniae</td><td rowspan=4 colspan=1>Klebsiella pneumoniae[KPC]JMI 7669.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>87.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>87.3</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>88.4</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=4 colspan=1>Proteus</td><td rowspan=4 colspan=1>Proteus mirabilisATCC 299069.20E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>80.7</td><td rowspan=1 colspan=1>81.6</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=4 colspan=1>Paeruginosa</td><td rowspan=4 colspan=1>PseudomonasaeruginosaATCC 278531.40E+08 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>87.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>87.3</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>88.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>89.1</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>88.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>87.3</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>2.2</td></tr></table>

Yeast

<table><tr><td rowspan=2 colspan=1>BCID Panel Assay</td><td rowspan=2 colspan=1>Organism TestedTest Concentration</td><td rowspan=2 colspan=1>TestSite</td><td rowspan=1 colspan=5>Reproducibility of Tm</td></tr><tr><td rowspan=1 colspan=1>TmMean</td><td rowspan=1 colspan=1>TmStd Dev</td><td rowspan=1 colspan=1>TmMinimum</td><td rowspan=1 colspan=1>TmMaximum</td><td rowspan=1 colspan=1>ObservedRange(Max-Min)</td></tr><tr><td rowspan=4 colspan=1>Calbicans</td><td rowspan=4 colspan=1>Candida albicansATCC 102313.10E+04</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>79.3</td><td rowspan=1 colspan=1>80.3</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>79.7</td><td rowspan=1 colspan=1>80.5</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>80.2</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>80.5</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=4 colspan=1>Cglabrata</td><td rowspan=4 colspan=1>Candida glabrataATCC 155452.00E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>75.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>74.7</td><td rowspan=1 colspan=1>76.1</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>75.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>74.9</td><td rowspan=1 colspan=1>76.4</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>75.4</td><td rowspan=1 colspan=1>76.1</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>75.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>74.7</td><td rowspan=1 colspan=1>76.4</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=4 colspan=1>Ckrusei</td><td rowspan=4 colspan=1>Candida kruseiATCC 908783.20E+07</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>84.1</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>84.3</td><td rowspan=1 colspan=1>85.3</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>85.8</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>84.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>84.1</td><td rowspan=1 colspan=1>85.8</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=4 colspan=1>Ctropicalis</td><td rowspan=4 colspan=1>Candida tropicalisATCC 660299.70E+05</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.6</td><td rowspan=1 colspan=1>80.1</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>79.6</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>79.3</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>79.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>78.6</td><td rowspan=1 colspan=1>80.1</td><td rowspan=1 colspan=1>1.6</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Controls: The following internal controls are included in each FilmArray BCID pouch:

DNA Process Control: The DNA Process Control assay targets DNA from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and is hydrated and introduced into the test when the sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, $\mathrm { i } ^ { \mathrm { s t } }$ stage PCR, dilution, $2 ^ { \mathrm { n d } }$ stage PCR, and DNA melting. A positive control result indicates that all steps carried out in the pouch were successful.

PCR2 Control: The PCR2 Control assay detects a DNA target that is dried into the wells of the array along with the corresponding primers. A positive result indicates that the $2 ^ { \mathrm { { \overset { \cdot } { n d } } } }$ stage PCR was successful.

Both internal control assays must be positive for the test run to pass. When either control fails, the Controls field of the test report will display "Failed" and all results will be listed as Invalid. If the controls fail, the user is instructed to repeat the test using a new pouch.

Recommended External Controls: External controls are not provided with the BCID Panel, but are recommended in the package insert. Uninoculated blood culture media can be used as an external negative control and previously characterized positive samples or samples spiked with well characterized organisms can be used as external positive controls. External controls should be used in accordance with the appropriate accrediting organization requirements, as applicable.

External Controls used in clinical studies:

External controls used in the clinical study consisted of six different organism mixes spiked into a human blood and blood culture matrix. All BCID targets were represented in at least one external control mix. Testing of positive external controls was rotated, with one of the six controls tested each day.

# Specimen Stability:

Testing with the FilmArray BCID Panel should be performed as soon as possible after a blood culture specimen is signaled as positive by a continuous monitoring instrument. As blood culture bottles are sometimes not removed from the instrument immediately (e.g., when microbiology laboratories are closed at night), testing of positive specimens is indicated up to eight hours after bottle ring. Testing with BCID Panel is indicated for both time frames. A "Growth and Detection" study (see below) was performed to validate the performance of the BCID Panel when testing immediately after bottle positivity as compared to testing after incubation for an additional eight hours.

# d. Growth and Detection Study

A study was performed to establish the range of expected organism concentrations in positive blood cultures. Testing with the BCID Panel is indicated immediately after a blood culture specimen is indicated as positive for growth by a continuously monitoring blood culture system and subsequent Gram stain or up to eight hours after bottle positivity. This study included evaluated the FilmArray BCID Panel using blood cultures at both time points. All organism growth and testing was performed using seeded blood culture bottles (BACTEC™ Plus Aerobic/F Medium) incubated in the BACTEC™ 9050 continuously monitoring blood culture instrument. Each microorganism was mixed with human whole blood and seeded directly into blood culture bottles for growth. At the time of positivity (and/or eight hours after positivity), the blood culture was removed from the instrument for

determination of organism concentration (CFU/mL using a plate count procedure) and FilmArray BCID testing. Three independent positive cultures (bottles) were evaluated for each organism at each time point and FilmArray testing was performed in triplicate for each bottle.

The following table summarizes the concentration of organism $\mathrm { ( C F U / m L ) }$ determined for a representative panel of 30 isolates. The number and percent of correct positive BCID Panel test results is provided for each isolate and overall $( \%$ Detected). A correct result means that both the correct organism and antimicrobial resistance gene (where applicable) were detected in the sample. The correct organism and antimicrobial resistance gene results were reported for all 540 samples tested (540/540, $100 \%$ ). In addition to the correct results, 5 false positive results (Streptococcus, Streptococcus agalactiae, Haemophilus influenzae, Neisseria meningitidis, and Candida krusei) were observed in a single run (1/540; $0 . 2 \%$ . The correct results were obtained when the sample was retested.

Summary of Organism Concentration $\mathbf { ( C F U / m L ) }$ in Positive Blood Cultures and Correct Detection of Organisms in Positive Blood Cultures by the FilmArray BCID Panel   

<table><tr><td colspan="1" rowspan="2">Species/Isolate(s) Tested</td><td colspan="3" rowspan="1">At Positivity</td><td colspan="3" rowspan="1">8 Hours After Positivity</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1">#Detected/Total(%Detected)</td><td colspan="1" rowspan="1">Per BottleCFU/mL</td><td colspan="1" rowspan="1">MeanCFU/mL</td><td colspan="1" rowspan="1">#Detected/Total(% Detected)</td></tr><tr><td colspan="7" rowspan="1">Gram-Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis [vanB]JMI 368</td><td colspan="1" rowspan="1">4.60E+081.80E+082.62E+08</td><td colspan="1" rowspan="1">3.01E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">7.25E+088.90E+081.07E+09</td><td colspan="1" rowspan="1">8.95E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium [vanA]JMI 475</td><td colspan="1" rowspan="1">1.47E+081.53E+081.59E+08</td><td colspan="1" rowspan="1">1.53E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.23E+081.64E+081.55E+08</td><td colspan="1" rowspan="1">1.81E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hiraeATCC 49135</td><td colspan="1" rowspan="1">1.26E+082.76E+083.25E+08</td><td colspan="1" rowspan="1">2.42E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.00E+086.60E+087.20E+08</td><td colspan="1" rowspan="1">7.27E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenesCDC F2380 (ATCC 43256)</td><td colspan="1" rowspan="1">4.50E+081.22E+091.18E+09</td><td colspan="1" rowspan="1">9.50E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.76E+092.31E+091.67E+09</td><td colspan="1" rowspan="1">1.91E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusATCC 11632</td><td colspan="1" rowspan="1">1.48E+082.00E+072.56E+07</td><td colspan="1" rowspan="1">6.45E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.75E+089.80E+081.21E+08</td><td colspan="1" rowspan="1">6.59E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus[MRSA/mecA]ATCC BAA-1747</td><td colspan="1" rowspan="1">1.41E+075.65E+066.05E+06</td><td colspan="1" rowspan="1">8.60E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">5.70E+073.85E+079.75E+07</td><td colspan="1" rowspan="1">6.43E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidisATCC 12228</td><td colspan="1" rowspan="1">1.38E+089.85E+071.16E+08</td><td colspan="1" rowspan="1">1.18E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.12E+083.95E+081.56E+09</td><td colspan="1" rowspan="1">7.22E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis[MRSE/mecA]ATCC 29887</td><td colspan="1" rowspan="1">3.60E+073.75E+071.56E+08</td><td colspan="1" rowspan="1">7.65E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.35E+096.80E+082.29E+09</td><td colspan="1" rowspan="1">1.44E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1">#Detected/Total(%Detected)</td><td colspan="1" rowspan="1">Per BottleCFU/mL</td><td colspan="1" rowspan="1">MeanCFU/mL</td><td colspan="1" rowspan="1">#Detected/Total(% Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiaeATCC 13813</td><td colspan="1" rowspan="1">4.50E+081.22E+089.15E+08</td><td colspan="1" rowspan="1">4.96E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">3.15E+085.80E+084.30E+08</td><td colspan="1" rowspan="1">4.42E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitisATCC 1591</td><td colspan="1" rowspan="1">1.57E+081.51E+096.90E+08</td><td colspan="1" rowspan="1">7.86E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.50E+092.03E+092.91E+09</td><td colspan="1" rowspan="1">2.15E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniaeATCC BAA-255</td><td colspan="1" rowspan="1">3.45E+082.67E+081.31E+09</td><td colspan="1" rowspan="1">6.41E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.03E+096.00E+081.37E+09</td><td colspan="1" rowspan="1">1.00E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenesATCC 19615</td><td colspan="1" rowspan="1">2.53E+082.44E+083.80E+08</td><td colspan="1" rowspan="1">2.92E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.38E+085.70E+088.90E+08</td><td colspan="1" rowspan="1">5.66E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram-Negative Bacteria</td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniiATCC 9955</td><td colspan="1" rowspan="1">2.17E+081.44E+082.45E+08</td><td colspan="1" rowspan="1">2.02E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">4.85E+083.85E+084.35E+08</td><td colspan="1" rowspan="1">4.35E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaeATCC 13047</td><td colspan="1" rowspan="1">4.20E+083.95E+081.50E+08</td><td colspan="1" rowspan="1">3.22E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.23E+091.46E+092.19E+09</td><td colspan="1" rowspan="1">1.96E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliATCC 43888</td><td colspan="1" rowspan="1">9.80E+076.10E+071.93E+08</td><td colspan="1" rowspan="1">1.17E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.17E+091.39E+097.70E+07</td><td colspan="1" rowspan="1">8.79E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytocaATCC 13182</td><td colspan="1" rowspan="1">7.40E+086.85E+083.85E+08</td><td colspan="1" rowspan="1">6.03E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">3.05E+091.86E+091.20E+09</td><td colspan="1" rowspan="1">2.04E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca [KPC]JMI 7818</td><td colspan="1" rowspan="1">6.15E+079.15E+073.05E+07</td><td colspan="1" rowspan="1">6.12E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.96E+092.00E+091.13E+09</td><td colspan="1" rowspan="1">1.70E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaeATCC 13883</td><td colspan="1" rowspan="1">4.35E+082.10E+089.15E+08</td><td colspan="1" rowspan="1">5.20E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.60E+091.65E+091.58E+09</td><td colspan="1" rowspan="1">1.61E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae [KPC]JMI 766</td><td colspan="1" rowspan="1">1.21E+082.50E+082.05E+08</td><td colspan="1" rowspan="1">1.92E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.14E+099.10E+087.70E+08</td><td colspan="1" rowspan="1">9.40E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilisATCC 29906</td><td colspan="1" rowspan="1">3.25E+071.04E+089.10E+07</td><td colspan="1" rowspan="1">7.58E+07</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.04E+099.80E+087.30E+08</td><td colspan="1" rowspan="1">9.17E+08</td><td colspan="1" rowspan="1">9/9a(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescensATCC 27137</td><td colspan="1" rowspan="1">8.35E+081.46E+094.90E+08</td><td colspan="1" rowspan="1">9.28E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.05E+091.37E+091.02E+09</td><td colspan="1" rowspan="1">1.15E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens [KPC]JMI 697</td><td colspan="1" rowspan="1">4.90E+083.90E+081.02E+08</td><td colspan="1" rowspan="1">3.27E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.19E+091.40E+092.42E+08</td><td colspan="1" rowspan="1">1.28E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (type b)ATCC 10211</td><td colspan="1" rowspan="1">2.80E+083.60E+082.23E+08</td><td colspan="1" rowspan="1">2.88E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">3.25E+093.35E+092.74E+09</td><td colspan="1" rowspan="1">3.11E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">2.07E+08</td><td colspan="1" rowspan="1">2.51E+08</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">6.65E+08</td><td colspan="1" rowspan="1">7.38E+08</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1">#Detected/Total(%Detected)</td><td colspan="1" rowspan="1">Per BottleC/mL</td><td colspan="1" rowspan="1">MeanCFU/mL</td><td colspan="1" rowspan="1">#Detected/Total(% Detected)</td></tr><tr><td colspan="1" rowspan="1">ATCC 43744</td><td colspan="1" rowspan="1">3.90E+081.55E+08</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">7.65E+087.85E+08</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosaATCC 27853</td><td colspan="1" rowspan="1">1.34E+081.76E+089.75E+07</td><td colspan="1" rowspan="1">1.36E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.35E+091.39E+081.76E+09</td><td colspan="1" rowspan="1">1.08E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicansATCC 10231</td><td colspan="1" rowspan="1">9.05E+038.00E+044.65E+03</td><td colspan="1" rowspan="1">3.12E+04</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8.80E+041.03E+051.00E+05</td><td colspan="1" rowspan="1">9.70E+04</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrataATCC 15545</td><td colspan="1" rowspan="1">1.26E+061.11E+061.97E+06</td><td colspan="1" rowspan="1">1.45E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.47E+072.65E+071.91E+07</td><td colspan="1" rowspan="1">2.01E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida kruseiATCC 90878</td><td colspan="1" rowspan="1">5.65E+062.47E+066.35E+06</td><td colspan="1" rowspan="1">4.82E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.68E+073.55E+073.25E+07</td><td colspan="1" rowspan="1">3.16E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosisATCC 90875</td><td colspan="1" rowspan="1">2.56E+063.60E+063.20E+06</td><td colspan="1" rowspan="1">3.12E+06</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">6.70E+073.80E+075.55E+07</td><td colspan="1" rowspan="1">5.35E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalisATCC 66029</td><td colspan="1" rowspan="1">1.50E+067.45E+056.65E+05</td><td colspan="1" rowspan="1">9.70E+05</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.10E+072.04E+079.45E+06</td><td colspan="1" rowspan="1">1.36E+07</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Overall Correct Detection(Organism and AntimicrobialResistance Genes)</td><td colspan="2" rowspan="1">At Positivity:</td><td colspan="1" rowspan="1">270/270(100%)</td><td colspan="2" rowspan="1">8 Hours After Positivity:</td><td colspan="1" rowspan="1">270/270(100%)</td></tr></table>

# e. Analytical Reactivity (Inclusivity):

The analytical reactivity of the FilmArray BCID Panel was evaluated with a collection of 303 bacterial and yeast isolates that represent the diversity of the FilmArray BCID Panel analytes, including antimicrobial resistance genes. Isolates were selected to represent relevant species or serotypes and selection with specific inclusion of more commonly encountered species and known human pathogens. When possible, in silico analysis of sequence data was used to make predictions of assay reactivity for less common species that were not tested but that may be detected by the FilmArray BCID Panel.

Each isolate was initially tested in blood culture matrix at a concentration consistent with the levels of organism enumerated from blood cultures at the time of positivity (see Growth and Detection section above). If an isolate was not detected initially, the sample was retested at 10-100 fold higher concentrations. If detected at the higher concentration(s), the species/isolate is indicated as detected with reduced

sensitivity and the concentration of organism that was detected is indicated. If not detected at the highest concentration the isolate is listed as not detected by the FilmArray BCID Panel. Results are provided below for each FilmArray BCID Panel test result.

Results of Enterococcus Inclusivity Testing   

<table><tr><td rowspan=1 colspan=2>Enterococcus Detected[~1x108 CFU/mL]</td><td rowspan=1 colspan=2>Enterococcus Detected withReduced Sensitivity[~1x10° CFU/mL</td><td rowspan=1 colspan=2>EnterococcusNot Detecteda</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>ATCC 49463</td><td rowspan=1 colspan=1>Enterococcussaccharolyticus</td><td rowspan=1 colspan=1>ATCC 43076</td><td rowspan=1 colspan=1>Enterococcuspseudoavium</td><td rowspan=1 colspan=1>ATCC 49372</td></tr><tr><td rowspan=1 colspan=1>Enterococcuscasseliflavus</td><td rowspan=1 colspan=1>ATCC 700668</td><td rowspan=1 colspan=1>Enterococcus dispar</td><td rowspan=1 colspan=1>ATCC 51266</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>ATCC 49427</td></tr><tr><td rowspan=1 colspan=1>Enterococcuscecorum</td><td rowspan=1 colspan=1>ATCC 43198</td><td rowspan=19 colspan=2></td><td rowspan=19 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Enterococcusdurans</td><td rowspan=1 colspan=1>ATCC 11576</td></tr><tr><td rowspan=6 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC 49532</td></tr><tr><td rowspan=1 colspan=1>ATCC 49533</td></tr><tr><td rowspan=1 colspan=1>JMI 12536</td></tr><tr><td rowspan=1 colspan=1>ATCC 51299</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td></tr><tr><td rowspan=1 colspan=1>JMI 368</td></tr><tr><td rowspan=6 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC 27270</td></tr><tr><td rowspan=1 colspan=1>ATCC 35667</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2127</td></tr><tr><td rowspan=1 colspan=1>JMI 536</td></tr><tr><td rowspan=1 colspan=1>ATCC 700221</td></tr><tr><td rowspan=1 colspan=1>JMI 475</td></tr><tr><td rowspan=1 colspan=1>Enterococcusflavescens</td><td rowspan=1 colspan=1>ATCC 49996</td></tr><tr><td rowspan=1 colspan=1>Enterococcusgallinarum</td><td rowspan=1 colspan=1>ATCC 49608</td></tr><tr><td rowspan=1 colspan=1>Enterococcushirae</td><td rowspan=1 colspan=1>ATCC 8043</td></tr><tr><td rowspan=1 colspan=1>Enterococcusmalodoratus</td><td rowspan=1 colspan=1>ATCC 43197</td></tr><tr><td rowspan=1 colspan=1>Enterococcusmundtii</td><td rowspan=1 colspan=1>ATCC 43187</td></tr></table>

a Not detected at the highest test concentrations $\sim 1 \mathrm { x } 1 0 ^ { 9 } – 1 \mathrm { x } 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ .

Results of Listeria monocytogenes Inclusivity Testing   

<table><tr><td colspan="3" rowspan="1">Listeria monocytogenes Detecteda</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">Isolate ID</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1/2a</td><td colspan="1" rowspan="1">FSL-C1-056b</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1/2a</td><td colspan="1" rowspan="1">FSL-J2-020 b</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1/2b</td><td colspan="1" rowspan="1">FSL-J2-064 b</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1/2b</td><td colspan="1" rowspan="1">HUM-2009042206c</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">4b</td><td colspan="1" rowspan="1">ATCC 43256</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">4b</td><td colspan="1" rowspan="1">ATCC 13932</td></tr></table>

a Estimated concentration in a positive blood culture is $\overline { { { \sim } 5 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ . b Isolates obtained from Cornell University. c Isolates obtained from the Colorado Department of Public Health (CDPH).

Results of Staphylococcus aureus Inclusivity Testing   

<table><tr><td colspan="2" rowspan="1">Staphylococcus/Staphylococcus aureus Detecteda</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td><td colspan="1" rowspan="1">PFGE Type</td></tr><tr><td colspan="1" rowspan="1">Methicillin-sensitive S. aureus (MSSA)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1749</td><td colspan="1" rowspan="1">96:308</td><td colspan="1" rowspan="1">USA 900</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1759</td><td colspan="1" rowspan="1">N7129</td><td colspan="1" rowspan="1">USA 900</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1765</td><td colspan="1" rowspan="1">102-04</td><td colspan="1" rowspan="1">USA 1200</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 12600</td><td colspan="1" rowspan="1">NCTC 8532 Type strain</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 11632</td><td colspan="1" rowspan="1">S13</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2419</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2420</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2421</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1060728</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ant1</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Lem8</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">MAL8134</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">MAQ</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Per2</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RAR</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">S313</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Sal3</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ver2</td><td colspan="1" rowspan="1">n/ac</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureusb</td><td colspan="1" rowspan="1">ATCC 10832</td><td colspan="1" rowspan="1">Wood 46</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureusb</td><td colspan="1" rowspan="1">ATCC 14154</td><td colspan="1" rowspan="1">Rose</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC 25923</td><td colspan="1" rowspan="1">Seattle/1945</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Borderline Oxacillin-resistant S. aureus (BORSA)</td><td colspan="1" rowspan="1">rus (BORSA)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN1d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN2d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN3d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN4d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN5d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">SUN6d</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Methicillin-resistant S. aureus (MRSA)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellob</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-38</td><td colspan="1" rowspan="1">E2125 Denmark</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">F-182 Kansas</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC 700698</td><td colspan="1" rowspan="1">Mu3 Japan/1996</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-1720</td><td colspan="1" rowspan="1">MRSA252 UK</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="4" rowspan="1">Staphylococcus/Staphylococcus aureus Detecteda</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td><td colspan="1" rowspan="1">PFGE Type</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-39</td><td colspan="1" rowspan="1">HUSA304 Hungary/1993</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS705</td><td colspan="1" rowspan="1">NY-12 New York/2005</td><td colspan="1" rowspan="1">USA 100</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS701</td><td colspan="1" rowspan="1">MN-082 Minn/2006</td><td colspan="1" rowspan="1">USA 200</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-1717</td><td colspan="1" rowspan="1">TCH1516 Texas</td><td colspan="1" rowspan="1">USA 300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS703</td><td colspan="1" rowspan="1">MN-095 Minn/2006</td><td colspan="1" rowspan="1">USA 300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS683</td><td colspan="1" rowspan="1">GA-298 Georgia/2005</td><td colspan="1" rowspan="1">USA 300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS662</td><td colspan="1" rowspan="1">CO-34 Colorado/2005</td><td colspan="1" rowspan="1">USA 300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS707</td><td colspan="1" rowspan="1">NY-155 New York/2005</td><td colspan="1" rowspan="1">USA 300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1707</td><td colspan="1" rowspan="1">MW2 N. Dakota/1998</td><td colspan="1" rowspan="1">USA 400</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS691</td><td colspan="1" rowspan="1">GA-62 Georgia/2005</td><td colspan="1" rowspan="1">USA 500</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS648</td><td colspan="1" rowspan="1">CA-347 California/2005</td><td colspan="1" rowspan="1">USA 600</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS689</td><td colspan="1" rowspan="1">GA-442 Georgia/2006</td><td colspan="1" rowspan="1">USA 700</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-42</td><td colspan="1" rowspan="1">HDE288 Portugal/1996</td><td colspan="1" rowspan="1">USA 800</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS668</td><td colspan="1" rowspan="1">CO-72 Colorado/2005</td><td colspan="1" rowspan="1">USA 800</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1747</td><td colspan="1" rowspan="1">94:1013 Vermont/1993</td><td colspan="1" rowspan="1">USA 1000</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS676</td><td colspan="1" rowspan="1">CT-19 Conn/2005</td><td colspan="1" rowspan="1">USA 1000</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSANRS745</td><td colspan="1" rowspan="1">CA-629 California/2006</td><td colspan="1" rowspan="1">USA 1000</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1764</td><td colspan="1" rowspan="1">7031 Alaska</td><td colspan="1" rowspan="1">USA 1100</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1691</td><td colspan="1" rowspan="1">HFH-30137Michigan/2003</td><td colspan="1" rowspan="1">Not 100-1100</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1700</td><td colspan="1" rowspan="1">HFH-33798 Illinois/2004</td><td colspan="1" rowspan="1">Not 100-1100</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">M10/0061 Ireland/2010</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">M10/0148 Ireland/2010</td><td colspan="1" rowspan="1">CC130</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (VRSA)</td><td colspan="1" rowspan="1">NARSA VRS5</td><td colspan="1" rowspan="1">HIP15178 Michigan/2005</td><td colspan="1" rowspan="1">Unknown</td></tr></table>

a Detected at the initial test concentration of $\overline { { 5 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ . b Initial test concentration was $5 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ . c Isolates obtained from University of Rennes, Laboratory of Microbiology and Immunology, France. d Isolates obtained from Sunnybrook Research Institute, affiliated with the University of Toronto. e Tested as a seeded blood culture at the time of positivity.

<table><tr><td rowspan=1 colspan=2>Staphylococcus Detected[~5x106 CFU/mL]</td><td rowspan=1 colspan=2>Staphylococcus Detected withReduced Sensitivity[~5x107 CFU/mL]</td><td rowspan=1 colspan=2>StaphylococcusNot  Dtectedb</td></tr><tr><td rowspan=1 colspan=6>Coagulase-positive staphylococci (non-S.aureus)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuslutrae</td><td rowspan=1 colspan=1>ATCC 700373</td><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1>Staphylococcusintermedius</td><td rowspan=1 colspan=1>ATCC 29663</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Staphylococcuspseudointermedius</td><td rowspan=1 colspan=1>ATCC 49444</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. coagulans</td><td rowspan=1 colspan=1>ATCC 49545</td></tr><tr><td rowspan=1 colspan=6>Coagulase-negative staphylococci (CoNS)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuscaprae</td><td rowspan=1 colspan=1>ATCC 51548</td><td rowspan=1 colspan=1>Staphylococcuscapitis subsp. capitis</td><td rowspan=1 colspan=1>ATCC 27842</td><td rowspan=1 colspan=1>Staphylococcusauricularis</td><td rowspan=1 colspan=1>Clinicalisoolated</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuscohnii</td><td rowspan=1 colspan=1>ATCC 29972</td><td rowspan=1 colspan=1>Staphylococcuspasteuri</td><td rowspan=1 colspan=1>ATCC 51127</td><td rowspan=1 colspan=1>Staphylococcuscarnosus</td><td rowspan=1 colspan=1>ATCC 51365</td></tr><tr><td rowspan=6 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>ATCC 12228</td><td rowspan=1 colspan=1>Staphylococcussaprophyticus</td><td rowspan=1 colspan=1>ATCC 15305</td><td rowspan=1 colspan=1>Staphylococcusentus</td><td rowspan=1 colspan=1>ATCC 700403</td></tr><tr><td rowspan=1 colspan=1>ATCC 29886</td><td rowspan=1 colspan=1>Staphylococcussimulans</td><td rowspan=1 colspan=1>Clinicalisolates</td><td rowspan=1 colspan=1>Staphylococcuspettenkoferi</td><td rowspan=1 colspan=1>5 clinicalisolates</td></tr><tr><td rowspan=1 colspan=1>ATCC 55133</td><td rowspan=1 colspan=1>Staphylococcuswarneri</td><td rowspan=1 colspan=1>ATCC 25614</td><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. schleiferi</td><td rowspan=1 colspan=1>ATCC 43808</td></tr><tr><td rowspan=1 colspan=1>ATCC 29887</td><td rowspan=8 colspan=2></td><td rowspan=1 colspan=1>Staphylococcussciuri</td><td rowspan=1 colspan=1>ATCC 29060</td></tr><tr><td rowspan=1 colspan=1>ATCC 51625</td><td rowspan=7 colspan=2></td></tr><tr><td rowspan=1 colspan=1>ATCC 35984</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusequorum</td><td rowspan=1 colspan=1>ATCC 43958</td></tr><tr><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>ATCC 29968</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hominisssp. hominis</td><td rowspan=1 colspan=1>ATCC 25615</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>ATCC 43809</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusxylosus</td><td rowspan=1 colspan=1>ATCC 29966</td></tr></table>

a All 54 S. aureus isolates (table above) received Staphylococcus Detected results. b Not detected when tested at a concentration of $\geq 5 \mathrm { x i 0 ^ { \bar { 8 } } C F U / m L }$ . c Isolates identified as Staphylococcus intermedius by automated identification systems were detected in two clinical specimens. d Staphylococcus auricularis was not tested in analytic studies, but was not detected in a clinical blood culture. e An isolate identified as Staphylococcus lentus by an automated identification system was detected in one clinical specimen. f Staphylococcus simulans was not tested in analytic studies, but was detected in three clinical blood cultures at unknown concentration.

Based on inclusivity testing results for staphylococci and in silico analysis of available sequences, the following predictions of reactivity (table below) are provided for less common CoNS species. Prediction of reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). Listed organisms were not tested by the FilmArray assay either in

# analytical or clinical testing. Performance of the FilmArray BCID Panel for these organisms has not been established.

In silico Predictions of Staphylococcus Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicteda</td><td rowspan=1 colspan=1>Detection Predicted withReduced Sensitivityb</td><td rowspan=1 colspan=1>Detection Not Predictedc</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum</td><td rowspan=1 colspan=1>Staphylococcus microti</td><td rowspan=1 colspan=1>Staphylococcus arlettae</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus kloosii</td><td rowspan=1 colspan=1>Staphylococcus simiae</td><td rowspan=1 colspan=1>Staphylococcus chromogenes</td></tr><tr><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus succinus</td><td rowspan=1 colspan=1>Staphylococcus condimenti</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus fleurettii</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus piscifermentans</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus pulvereri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus rostri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus saccharolyticus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus vitulinus</td></tr></table>

a Predicted result of Staphylococcus Detected when present in a blood culture sample at a concentration of ${ \bf \{ }  ^ { \bf { > } 5 { \bf { x } } 1 0 ^ { 6 } }$ CFU/mL. b Predicted result of Staphylococcus Detected when present in a blood culture sample at a concentration of $\geq 5 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . c Predicted result of Staphylococcus Not Detected at relevant concentrations.

# Results of Streptococcus Inclusivity Testing

<table><tr><td colspan="3" rowspan="1">Streptococcus Detecteda</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 19615</td><td colspan="1" rowspan="5">Group A (Pyogenic group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">PCMC 20100107CI02</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 49399</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12344</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813Type strain  Serotype 1a/c</td><td colspan="1" rowspan="5">Group B (Pyogenic group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">PCMC 20100107CI03Untyped clinical isolate</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12403Type III</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC BAA-611Serotype V</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">NCTC 8017Unknown serotype</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae ssp. equisimilis</td><td colspan="1" rowspan="1">ATCC 12388</td><td colspan="1" rowspan="1">Group C/G (Pyogenicgroup)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">ATCC 33317</td><td colspan="1" rowspan="2">Group D (Bovis group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equinis</td><td colspan="1" rowspan="1">ATCC 9812</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">ATCC 25175</td><td colspan="1" rowspan="1">Group E (Mutans group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 33397</td><td colspan="1" rowspan="3">Group F (Anginosus group)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">ATCC 10558</td><td colspan="1" rowspan="2">Mitis group</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">ATCC 31412</td></tr><tr><td colspan="3" rowspan="1">Streptococcus Detected</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">ATCC 10556</td><td colspan="1" rowspan="9">−</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 15914</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">ATCC 10557</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-960</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-255Strain R6 (no capsule)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 700672Serotype 14</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-334Serotype 4</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 700673Serotype 19A</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-341Serotype 5</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">Salivarius group</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">ATCC BAA-2069</td><td colspan="1" rowspan="1">Uncertain grouping</td></tr></table>

a Detected at the initial test concentration of $\overline { { { \sim } 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ .

Results of Streptococcus agalactiae Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus agalactiae (Group B) Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813Type strain  Serotype 1a/c</td><td rowspan=5 colspan=1>Group B (Pyogenic group)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>PCMC 20100107CI03Untyped clinical isolate</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 12403Type III</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC BAA-611Serotype V</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>NCTC 8017Unknown serotype</td></tr></table>

a Detected at the initial test concentration of $\overline { { { \sim } 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ .

Results of Streptococcus pneumoniae Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus pneumoniae Detecteda,b</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-255Strain R6 (no capsule)</td><td rowspan=5 colspan=1>Mitis group</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 700672Serotype 14</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-334Serotype 4</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 700673Serotype 19A</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-341Serotype 5</td></tr></table>

a Detected at the initial test concentration of $\overline { { { \sim } 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ . b Based on sequence analysis, the BCID Panel may not detect S. pneumoniae serotypes 11A and 19, or may detect these serotypes with reduced sensitivity compared to other serotypes.

Results of Streptococcus pyogenes Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Streptococcus/Streptococcus pyogenes (Group A) Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=5 colspan=1>Group A (Pyogenic group)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>PCMC20100107CI02</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 49399</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12344</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12384</td></tr></table>

a Detected at the initial test concentration of $\mathrm { \sim } 1 \mathrm { { x } 1 0 ^ { 8 } C F U / m L }$ .

Based on results of inclusivity testing and in silico analysis of available sequences, predictions of reactivity are provided in the table below for less common Streptococcus species. Prediction of reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). The analysis predicts that many species may be detected at concentrations expected in positive blood cultures $( 1 \dot { 0 } ^ { 8 } { - } \dot { 1 } 0 ^ { 9 } \mathrm { C F U / \dot { m } L } )$ ), and others (particularly Mutans group species) will likely not be detected due to sequence mismatches with the assay primers. Listed organisms were not tested by the FilmArray assay either in analytical or clinical testing. Performance of the FilmArray BCID Panel for these organisms has not been established.

In silico Predictions of Streptococcus Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicteda</td><td rowspan=1 colspan=1>Detection Predicted withReduced Sensitivityb</td><td rowspan=1 colspan=1>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Streptococcus australis</td><td rowspan=1 colspan=1>Streptococcus parauberis</td><td rowspan=1 colspan=1>Streptococcus cricetic</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equi</td><td rowspan=9 colspan=1></td><td rowspan=1 colspan=1>Streptococcus downeic</td></tr><tr><td rowspan=1 colspan=1>Streptococcus ictaluri</td><td rowspan=1 colspan=1>Streptococcus macacaec</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantis</td><td rowspan=1 colspan=1>Streptococcus porcinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantarius</td><td rowspan=1 colspan=1>Streptococcus urialis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pasteurianus</td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus perois</td></tr><tr><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus thermophilus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus vestibularis</td></tr></table>

a Predicted result of Streptococcus Detected when present in a blood culture sample at a concentration of $\overline { { \sim 1 \mathbf { x } 1 0 ^ { 8 } } }$ CFU/mL. b Predicted result of Streptococcus Detected when present in a blood culture sample at a concentration of ${ \geq } 1 \mathrm { x } 1 0 ^ { 9 }$ CFU/mL. c Mutans group streptococci.

Results of Acinetobacter baumannii Inclusivity Testing   

<table><tr><td colspan="2" rowspan="1">Acinetobacter baumannii Detecteda</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 9955</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC BAA-1605</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 17961</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19003</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC BAA-2093</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC15308</td></tr></table>

a Detected at the initial test concentration of $\overline { { { \sim } 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$

Results of Enterobacteriaceae Inclusivity Testing   

<table><tr><td rowspan=1 colspan=2>Enterobacteriaceae Detected[~5×107 CFU/mL or 1×108 CFU/mL]</td><td rowspan=1 colspan=2>Enterobacteriaceae Detected withReduced Sensitivity[~5×108-1×10° CFU/mL]</td><td rowspan=1 colspan=2>EnterobacteriaceaeNot Detecteda</td></tr><tr><td rowspan=1 colspan=1>Cedeceae davisiae</td><td rowspan=1 colspan=1>ATCC 43023</td><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>ATCC 15947</td><td rowspan=1 colspan=1>Morganella morganiisubsp. morganii</td><td rowspan=1 colspan=1>ATCC 25829</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ATCC 43864</td><td rowspan=1 colspan=1>Enterobactergergoviae</td><td rowspan=1 colspan=1>ATCC 33028</td><td rowspan=1 colspan=1>Pantoea (Enterobacter)agglomeransb</td><td rowspan=1 colspan=1>ATCC 2715</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>ATCC 29223</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>ATCC 51815</td><td rowspan=1 colspan=1>Providencia (Proteus)acalifaciens</td><td rowspan=1 colspan=1>ATCC 51902</td></tr><tr><td rowspan=1 colspan=1>Cronobacter muytjensii</td><td rowspan=1 colspan=1>ATCC 51329</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>SGSC 3041</td><td rowspan=1 colspan=1>Providencia (Proteus)rettgeri</td><td rowspan=1 colspan=1>ATCC 9250</td></tr><tr><td rowspan=1 colspan=1>Cronobacter(Enterobacter) sakazakii</td><td rowspan=1 colspan=1>ATCC 29544</td><td rowspan=1 colspan=1>Serratia fonticola</td><td rowspan=1 colspan=1>ATCC 29844</td><td rowspan=1 colspan=1>Providencia stuarti</td><td rowspan=1 colspan=1>ATCC 33672</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>ATCC 13048</td><td rowspan=1 colspan=1>Serratia odorifera</td><td rowspan=1 colspan=1>ATCC 33077</td><td rowspan=1 colspan=1>Rahnella aquatilis</td><td rowspan=1 colspan=1>ATCC 3307</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>ATCC 29751</td><td rowspan=1 colspan=1>Serratia rubidaea</td><td rowspan=1 colspan=1>ATCC 27593</td><td rowspan=1 colspan=1>Serratia liquefaciens</td><td rowspan=1 colspan=1>ATCC 27592</td></tr><tr><td rowspan=1 colspan=1>Enterobacter amnigenus</td><td rowspan=1 colspan=1>ATCC 51816</td><td rowspan=21 colspan=4>Serratia plymuthica        ATCC 183Tatumella ptyseos         ATCC 3330Yersinia enterocolitica     ATCC 6025</td></tr><tr><td rowspan=1 colspan=1>Enterobacterasburiae</td><td rowspan=1 colspan=1>ATCC 35953</td><td rowspan=1 colspan=1>Tatumella ptyseos</td><td rowspan=1 colspan=1>ATCC 3330</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>9 isolatesc</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 6025</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>ATCC 49162</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>ATCCBAA-260d</td></tr><tr><td rowspan=1 colspan=1>Enterobacternimipressuralis</td><td rowspan=1 colspan=1>ATCC 9912d</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>5 isolates</td></tr><tr><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>ATCC 35469</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermanii</td><td rowspan=1 colspan=1>ATCC 33650</td></tr><tr><td rowspan=1 colspan=1>Escherichia vulneris</td><td rowspan=1 colspan=1>ATCC 33821</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>11 isolates</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>10 isolates</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCCBAA-830</td></tr><tr><td rowspan=1 colspan=1>Kluyvera ascorbata</td><td rowspan=1 colspan=1>ATCC 33433</td></tr><tr><td rowspan=1 colspan=1>Kluyvera (Enterobacter)intermedius</td><td rowspan=1 colspan=1>ATCC 33110</td></tr><tr><td rowspan=1 colspan=1>Leclercia adecarboxylata</td><td rowspan=1 colspan=1>ATCC 23216</td></tr><tr><td rowspan=1 colspan=1>Proteus species</td><td rowspan=1 colspan=1>10 isolatesh</td></tr><tr><td rowspan=1 colspan=1>Raoultellaornithinolytica</td><td rowspan=1 colspan=1>ATCC 31898</td></tr><tr><td rowspan=1 colspan=1>Raoultella planticola</td><td rowspan=1 colspan=1>ATCC 31900</td></tr><tr><td rowspan=1 colspan=1>Raoultella terrigena</td><td rowspan=1 colspan=1>ATCC 33257</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica-cholerasius</td><td rowspan=1 colspan=1>ATCC 10708</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Enterobacteriaceae Detected[~5×107 CFU/mL or 1×108 CFU/mL]</td><td rowspan=1 colspan=1>Enterobacteriaceae Detected withReduced Sensitivity[~5×108-1×10° CFU/mL]</td><td rowspan=1 colspan=1>EnterobacteriaceaeNot Detecteda</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica-heidelberg</td><td rowspan=1 colspan=1>ATCC 8326</td><td rowspan=11 colspan=1></td><td rowspan=11 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica-paratyphi</td><td rowspan=1 colspan=1>SGSC 3222</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica-typhimurium</td><td rowspan=1 colspan=1>ATCC 13311</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>6 isolatesi</td></tr><tr><td rowspan=1 colspan=1>Serratiaentomophila</td><td rowspan=1 colspan=1>ATCC 43705</td></tr><tr><td rowspan=1 colspan=1>Serratiaficaria</td><td rowspan=1 colspan=1>ATCC 33105</td></tr><tr><td rowspan=1 colspan=1>Shigellaboydi</td><td rowspan=1 colspan=1>ATCC 8700</td></tr><tr><td rowspan=1 colspan=1>Shigelladysenteriae</td><td rowspan=1 colspan=1>PHM-2004008089</td></tr><tr><td rowspan=1 colspan=1>Shigellaflexneri</td><td rowspan=1 colspan=1>ATCC 12022</td></tr><tr><td rowspan=1 colspan=1>Shigellasonnei</td><td rowspan=1 colspan=1>ATCC 11060</td></tr><tr><td rowspan=1 colspan=1>Yokenellaregensburgei</td><td rowspan=1 colspan=1>ATCC 35313</td></tr></table>

a Not Detected at the highest test concentration of $1 { \times } 1 0 ^ { 9 } – 1 { \times } 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ . b Not Detected in this study, but Pantoea agglomerans was detected by the BCID Panel in a clinical blood culture. c See Enterobacter cloacae complex table. d Tested as purified nucleic acid at a concentration of $0 . 6 3 \mu \mathrm { g / m L }$ (equivalent to $\sim 1 . 0 { \times } 1 0 ^ { 8 } \mathrm { C F U / m L ) }$ . e See Escherichia coli table. f See Klebsiella oxytoca table. g See Klebsiella pneumoniae table. h See Proteus table. i See Serratia marcescens table. j Tested as a seeded blood culture within 1 hour of positivity.

Based on results of inclusivity testing and in silico analysis of available sequences, predictions of reactivity are provided in the following table for less common members of the Enterobacteriaceae that were not tested. Prediction of reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). Listed organisms were not tested by the FilmArray assay either in the analytical or clinical testing. The performance of the FilmArray BCID Panel for these organisms has not been established.

In silico Predictions of Enterobacteriaceae Reactivity   

<table><tr><td rowspan=1 colspan=1>Detection Predicted withReduced Sensitivitya</td><td rowspan=1 colspan=1>Detection Not Predicted</td><td rowspan=1 colspan=1>Unknown Reactivityb</td></tr><tr><td rowspan=1 colspan=1>Brenneria spp.</td><td rowspan=1 colspan=1>Photorhabdus spp.</td><td rowspan=1 colspan=1>Buttiauxella spp.</td></tr><tr><td rowspan=1 colspan=1>Dickeya spp.</td><td rowspan=1 colspan=1>Serratia grimesii</td><td rowspan=1 colspan=1>Ewingella americana</td></tr><tr><td rowspan=1 colspan=1>Erwinia spp.</td><td rowspan=1 colspan=1>Serratia proteamaculans</td><td rowspan=1 colspan=1>Leminorella spp.</td></tr><tr><td rowspan=1 colspan=1>Pectobacterium spp.</td><td rowspan=1 colspan=1>Xenorhabdus spp.</td><td rowspan=1 colspan=1>Moellerella spp.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Yersinia spp.</td><td rowspan=1 colspan=1></td></tr></table>

a Predicted result of Enterobacteriaceae Detected when present in a blood culture sample at a concentration of $\geq 1 \mathrm { x } 1 0 ^ { 8 }$ CFU/mL b Sequence data not available for in silico reactivity predictions

Results of Enterobacter cloacae complex Inclusivity Testing   

<table><tr><td rowspan=1 colspan=2>Enterobacter cloacae complex Detected[~1×108 CFU/mL]</td><td rowspan=1 colspan=2>Enterobacter cloacae complexNot Detecteda</td></tr><tr><td rowspan=1 colspan=1>Enterobacter asburiae</td><td rowspan=1 colspan=1>ATCC 35953</td><td rowspan=1 colspan=1>Enterobacter nimipressuralisb</td><td rowspan=1 colspan=1>ATCC 9912</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. cloacae</td><td rowspan=1 colspan=1>ATCCBAA-1143</td><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>ATCCBAA-260</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. cloacae</td><td rowspan=1 colspan=1>ATCC 13047</td><td rowspan=5 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. cloacae</td><td rowspan=1 colspan=1>NCTC 10005</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. cloacae</td><td rowspan=1 colspan=1>ATCC 49141</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaesubsp. dissolvensb</td><td rowspan=1 colspan=1>ATCC 23373</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>ATCC 49162</td></tr></table>

a Not Detected at highest test concentration of $\overline { { 1 \times 1 0 ^ { 1 0 } \mathrm { C F U / m L } } }$ . b Tested as purified nucleic acid at a concentration of $0 . 6 3 \mu \mathrm { g / m L }$ (equivalent to $\sim 1 { \times } 1 0 ^ { 8 } \mathrm { C F U / m L } )$ ).

Results of Escherichia coli Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Escherichia coli Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 43888</td><td rowspan=1 colspan=1>CDC B6914-MS1 serotype O157:H7</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 49105</td><td rowspan=1 colspan=1>7482-1-1 serotype O15</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 25922</td><td rowspan=1 colspan=1>FDA-Seattle1946</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>H10407serotype O78:H11</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC BAA-201</td><td rowspan=1 colspan=1>Produces ESBL TEM-3</td></tr></table>

a Detected at the initial test concentration of $\overline { { 5 { \times } 1 0 ^ { 7 } \ : \mathrm { C F U / m L } } }$ .

Results of Klebsiella oxytoca Inclusivity Testing   

<table><tr><td colspan="3" rowspan="1">Klebsiella oxytoca Detecteda</td><td colspan="3" rowspan="1">Klebsiella oxytoca Not Detected</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Info</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Info</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">Klebsiella oxytocab.c</td><td colspan="1" rowspan="1">JMI10678</td><td colspan="1" rowspan="1">MY/2011</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">n/a</td><td colspan="3" rowspan="3">cYanmaGYelowb</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 700324</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43086</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">n/a</td><td colspan="3" rowspan="6"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 14611</td><td colspan="1" rowspan="1">AR/2011</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 12707</td><td colspan="1" rowspan="1">MA/2011</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 7818</td><td colspan="1" rowspan="1">AR/2004</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 2661</td><td colspan="1" rowspan="1">NY/2003</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 2523</td><td colspan="1" rowspan="1">n/a</td></tr></table>

a Detected at the initial test concentration of $\overline { { 5 { \times } 1 0 ^ { 7 } \ : \mathrm { C F U / m L } } }$ . b Detected as Enterobacteriaceae at the initial test concentration of $5 { \times } 1 0 ^ { 7 } \mathrm { C F U / m L }$ but Not Detected for Klebsiella oxytoca at the highest test concentration of $1 \times 1 0 ^ { 1 0 }$ CFU/mL. c Sequence analysis confirmed this isolate as a variant $K .$ . oxytoca that will not be detected by the FilmArray BCID Panel Koxytoca assay.

# Results of Klebsiella pneumoniae Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Klebsiella pneumoniae Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>StrainInformation</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1706</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. pneumoniae</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. ozaenae</td><td rowspan=1 colspan=1>ATCC 11296</td><td rowspan=1 colspan=1>NCTC 5050</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ssp. rhinoscleromatis</td><td rowspan=1 colspan=1>ATCC 13884</td><td rowspan=1 colspan=1>NCTC 5046Type strain</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 328</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 8091</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>JMI 438</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicolab</td><td rowspan=1 colspan=1>ATCC BAA-830</td><td rowspan=1 colspan=1>F2R9/ 2001Type strain</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ . b Identical sequence to $K .$ . pneumoniae variant 342. Both $K .$ pneumoniae variant 342 and Klebsiella variicola have been recovered from clinical specimens and will be identified by the BCID Panel and most standard laboratory methods as Klebsiella pneumoniae.

# Results of Proteus Inclusivity Testing

<table><tr><td rowspan=1 colspan=2>Proteus Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td></tr><tr><td rowspan=5 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>ATCC 29906</td></tr><tr><td rowspan=1 colspan=1>JMI 10793</td></tr><tr><td rowspan=1 colspan=1>ATCC 25933</td></tr><tr><td rowspan=1 colspan=1>ATCC 33583</td></tr><tr><td rowspan=1 colspan=1>ATCC7002</td></tr><tr><td rowspan=2 colspan=1>Proteus hauseri</td><td rowspan=1 colspan=1>ATCC13315</td></tr><tr><td rowspan=1 colspan=1>ATCC 700826</td></tr><tr><td rowspan=1 colspan=1>Proteus penneri</td><td rowspan=1 colspan=1>ATCC33519</td></tr><tr><td rowspan=2 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>ATCC 33420</td></tr><tr><td rowspan=1 colspan=1>ATCC 27973</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 7 } \mathrm { C F U / m L } } }$ .

Results of Serratia marcescens Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Serratia marcescens Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 13880</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC14756</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 27137</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 43297</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>CT/2009</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>JMI 8089</td><td rowspan=1 colspan=1>TX/2004</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 8 } } } \mathrm { C F U / m L }$ .

Results of Inclusivity Testing for Haemophilus influenzae   

<table><tr><td rowspan=1 colspan=3>Haemophilus influenzae Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Information</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 33929</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 51907</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae ssp. aegyptus</td><td rowspan=1 colspan=1>ATCC 11116</td><td rowspan=1 colspan=1>Non-typeable</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 9006</td><td rowspan=1 colspan=1>Type a</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 31512</td><td rowspan=1 colspan=1>Type b</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCCb10211</td><td rowspan=1 colspan=1>Type b</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 49699</td><td rowspan=1 colspan=1>Type c</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 9008</td><td rowspan=1 colspan=1>Type d</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 8142</td><td rowspan=1 colspan=1>Type e</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 700223</td><td rowspan=1 colspan=1>Type f</td></tr></table>

a Detected as seeded positive blood cultures tested within 1 hour of positivity. The concentration of $H .$ influenzae in a positive blood culture at the time of positivity is estimated to be $\sim 1 { \times } 1 0 ^ { 8 } \mathrm { C F U / m L }$

Results of Neisseria meningitidis Inclusivity Testing   

<table><tr><td rowspan=1 colspan=3>Neisseria meningitidis Detecteda</td><td rowspan=1 colspan=3>Neisseria meningitidis Not Detectedb,c</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Serogroup</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Serogroup</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC43744</td><td rowspan=1 colspan=1>W135</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinicalisolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC13077</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinicalisolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC13090</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinicalisolatec</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC13102</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Neisseria meningitidis(unencapsulated)</td><td rowspan=1 colspan=1>Clinicalisolate</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC13113</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Clinicalisolated</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>ATCC35561</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=3>cYanmaGellowb</td></tr></table>

a Detected in a seeded blood culture tested within 1 hour of positivity (estimated concentration $\overline { { \sim 1 \times 1 0 ^ { 8 } } }$ CFU/mL). b Not Detected in a seeded blood culture tested 1-5 hours after positivity. c Clinical isolates of unencapsulated $N .$ meningitidis were tested from seeded positive blood cultures to confirm that they would not be detected by the BCID Panel. d DNA from a clinical isolate with a variant ctrA gene was tested and not detected at a concentration

equivalent to $2 . 5 { \times } 1 0 ^ { 9 } \mathrm { C F U / m L }$ .

Results of Pseudomonas aeruginosa Inclusivity Testing   

<table><tr><td rowspan=1 colspan=2>Pseudomonas aeruginosa Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 27853</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 10145</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 19429</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 25619</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC BAA-1744</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 35554</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 8 } \mathrm { C F U / m L } } }$

# Results of Candida albicans Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Candida albicans Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=1>Serotype A - 3147</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC MYA-427</td><td rowspan=1 colspan=1>A39 [DUMC 136.97]</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC MYA-2876</td><td rowspan=1 colspan=1>SC5314</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 11651</td><td rowspan=1 colspan=1>171D</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 22972</td><td rowspan=1 colspan=1>M 97</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 90028</td><td rowspan=1 colspan=1>NCCLS 11</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 4 } ~ } } \overline { { \mathrm { C F U / m L } } }$ .

# Results of Candida glabrata Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Candida glabrata Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15545</td><td rowspan=1 colspan=1>NRRL YB-4025</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 32554</td><td rowspan=1 colspan=1>26247-1</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 2001</td><td rowspan=1 colspan=1>CBS138</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15126</td><td rowspan=1 colspan=1>CBS15126</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC MYA-2950</td><td rowspan=1 colspan=1>n/a</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ .

# Results of Candida krusei Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Candida krusei Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 90878</td><td rowspan=1 colspan=1>B74</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 201748</td><td rowspan=1 colspan=1>89-08-008</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 14243</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Candida krusei/Issatchenkia orientalisb</td><td rowspan=1 colspan=1>ATCC 28870</td><td rowspan=1 colspan=1>CBS 2052</td></tr><tr><td rowspan=1 colspan=1>Issatchenkia orientalisb</td><td rowspan=1 colspan=1>ATCC 6258</td><td rowspan=1 colspan=1>NRRL Y-413</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ . b Issatchenkia orientalis and Pichia kudriavzevii are anamorphs of $C .$ . krusei.

# Results of Candida parapsilosis Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Candida parapsilosis Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 90875</td><td rowspan=1 colspan=1>B78</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 34136</td><td rowspan=1 colspan=1>ST-89</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 96142</td><td rowspan=1 colspan=1>MCO462[UTHSC R-648]</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 96138</td><td rowspan=1 colspan=1>MC0433</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 22019</td><td rowspan=1 colspan=1>CBS604</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ .

# Results of Candida tropicalis Inclusivity Testing

<table><tr><td rowspan=1 colspan=3>Candida tropicalis Detecteda</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Strain Info</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 66029</td><td rowspan=1 colspan=1>AmMS 227</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 750</td><td rowspan=1 colspan=1>Type Strain</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 90874</td><td rowspan=1 colspan=1>B79</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC MYA-2734</td><td rowspan=1 colspan=1>508-12.1</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 201380</td><td rowspan=1 colspan=1>API 9001 105(Vitek QC)</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 5 } \mathrm { C F U / m L } } }$ . b Target concentration was $5 { \times } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , final test concentration was $1 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L }$

# Results of mecA Inclusivity Testing

<table><tr><td colspan="4" rowspan="1">mecA Detecteda,b</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Strain Information</td><td colspan="1" rowspan="1">SCCmecType</td></tr><tr><td colspan="4" rowspan="1">Methicillin-sensitive S. aureus (MSSA) with SCCmec cassette (mecA positive)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2419</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2420</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2421</td><td colspan="1" rowspan="1">Mass/2010</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Methicillin-resistant S. epidermidis (MRSE) (mecA positive)</td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 29887</td><td colspan="1" rowspan="1">255-01B</td><td colspan="1" rowspan="3">Unknown</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 51625</td><td colspan="1" rowspan="1">CCF 15990</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">RP62A</td></tr><tr><td colspan="4" rowspan="1">Methicillin-resistant S. aureus (MRSA) (mecA positive)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-38</td><td colspan="1" rowspan="1">E2125 Denmark</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">F-182 Kansas</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC 700698</td><td colspan="1" rowspan="1">Mu3 Japan/1996</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-1720</td><td colspan="1" rowspan="1">MRSA252 UK</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS705</td><td colspan="1" rowspan="1">NY-12 New York/2005</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS701</td><td colspan="1" rowspan="1">MN-082 Minn/2006</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS648</td><td colspan="1" rowspan="1">CA-347 California/2005</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-39</td><td colspan="1" rowspan="1">HUSA304 Hungary/1993</td><td colspan="1" rowspan="1">III 3A&amp;5</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS703</td><td colspan="1" rowspan="1">MN-095 Minnesota/2006</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS683</td><td colspan="1" rowspan="1">GA-298 Georgia/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS662</td><td colspan="1" rowspan="1">CO-34 Colorado/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS707</td><td colspan="1" rowspan="1">NY-155 New York/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1707</td><td colspan="1" rowspan="1">MW2 N. Dakota/1998</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS691</td><td colspan="1" rowspan="1">GA-62 Georgia/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS689</td><td colspan="1" rowspan="1">GA-442 Georgia/2006</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS668</td><td colspan="1" rowspan="1">CO-72 Colorado/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1747</td><td colspan="1" rowspan="1">94:1013 Vermont/1993</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS676</td><td colspan="1" rowspan="1">CT-19 Conn/2005</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1764</td><td colspan="1" rowspan="1">7031 Alaska</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1691</td><td colspan="1" rowspan="1">HFH-30137 Mich/2003</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1700</td><td colspan="1" rowspan="1">HFH-33798 Illinois/2004</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-1717</td><td colspan="1" rowspan="1">TCH1516 Texas</td><td colspan="1" rowspan="1">IVa</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NARSA NRS745</td><td colspan="1" rowspan="1">CA-629 California/2006</td><td colspan="1" rowspan="1">V</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus ssp. aureus</td><td colspan="1" rowspan="1">ATCC BAA-42</td><td colspan="1" rowspan="1">HDE288 Portugal/1996</td><td colspan="1" rowspan="1">VI</td></tr><tr><td colspan="4" rowspan="1">Methicillin-resistant S. aureus with mecALGA251/mecC variant</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">M10/0061 Ireland/2010</td><td colspan="1" rowspan="1">XI</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">M10/0148 Ireland/2010</td><td colspan="1" rowspan="1">XI</td></tr></table>

a Detected at the initial test concentration of $\overline { { 5 \times 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ . b Staphylococcus Detected and/or Staphylococcus aureus Detected results also reported, as appropriate. c Initial test concentration was $5 \mathrm { { x } 1 0 ^ { 5 } \mathrm { { C F U } / m L } }$ .

# Results of vanA/B Inclusivity Testing

<table><tr><td rowspan=1 colspan=4>vanA/B Detecteda,b</td></tr><tr><td rowspan=1 colspan=2>Species</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>StrainInformation</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>JMI 536</td><td rowspan=1 colspan=1>TX/2006</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=1 colspan=1>Connecticut</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecium [vanA]</td><td rowspan=1 colspan=1>JMI 475</td><td rowspan=1 colspan=1>IN/2003</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecalis [vanA]</td><td rowspan=1 colspan=1>JMI 12536</td><td rowspan=1 colspan=1>Mass/2002</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>Missouri</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>Missouri/1987</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis [vanB]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>JMI 368</td><td rowspan=1 colspan=1>VA/2003</td></tr></table>

a Detected at the initial test concentration of $\overline { { 1 \times 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ . b Enterococcus Detected results also reported.

# Results of KPC Inclusivity Testing

<table><tr><td colspan="4" rowspan="1">KPC Detecteda, b</td></tr><tr><td colspan="1" rowspan="1">Speciesc</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">KPC Type</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">BAA-2341</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">1101152</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">BAA-2082</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">BAA-2340</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">1101362</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 2523</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 7818</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">AR/2004</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 328</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1705</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">Modified Hodge TestControl</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">JMI 697</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">CT/2009</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">KPC Type</td><td colspan="1" rowspan="1">Strain Information</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 2661</td><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">NY/2003</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 766</td><td colspan="1" rowspan="1">KPC-4</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 8091</td><td colspan="1" rowspan="1">KPC-4</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 438</td><td colspan="1" rowspan="1">KPC-4</td><td colspan="1" rowspan="1">n/a</td></tr></table>

a Detected at the initial test concentration of $\overline { { 5 { \times } 1 0 ^ { 7 } \ : \mathrm { C F U / m L } } }$ for $K .$ . oxytoca isolates and $\overline { { 1 \times 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ for $K .$ . pneumoniae and S. marcescens isolates. Detected in a seeded blood culture tested within 1 hour of positivity for Enterobacter spp. and $E .$ . coli. b Enterobacteriaceae and corresponding species specific Detected results also reported. c Other isolates which carry the KPC gene (i.e. Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacteriaceae other than those listed above) were not evaluated.

Isolates of KPC-carrying organisms other than those listed in the above table (e.g., Pseudomonas aeruginosa, Acinetobacter baumanii, Citrobacter spp., Salmonella spp., Enterobacter spp. (other than $E$ . cloacae)) were not evaluated in the inclusivity or clinical studies. In silico analysis of available KPC sequences of these organisms known to contain the KPC resistance marker demonstrated $100 \%$ homology with the BCID primers. Therefore detection of the KPC gene is predicted but has not been demonstrated for these organisms.

# f. Analytical specificity (Exclusivity):

The potential for cross-reactivity between assays contained in the BCID Panel was evaluated by testing blood culture samples with concentrations equal to or greater than the level of organism estimated to be in a blood culture sample eight hours after positivity (approximately $1 0 ^ { 9 } – 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ for bacteria and $1 0 ^ { 7 } { \cdot } 1 0 ^ { 8 } \mathrm { C F U / m L }$ for yeast), or the highest concentration possible based on the organism stock. Organisms were tested as either seeded blood cultures or contrived specimens at known concentrations. For blood culture samples, test organisms were seeded into blood culture bottles and grown to positivity on a continuous monitoring blood culture instrument. Bottles were removed from the instrument and tested with the BCID Panel after eight hours of being called positive. Contrived samples were prepared by spiking the organism into a simulated blood culture matrix (human whole blood in blood culture medium), and incubated on the blood culture instrument for ${ \sim } 2 4$ hours to the final desired concentration. Target concentrations were confirmed by plating. Frozen quantified stocks were used to test the following organisms at the highest concentrations possible: Mycoplasma hominis $( 3 . 1 6 { \times } 1 0 ^ { 7 } \mathrm { C F U / m L } )$ ), Ureaplasma urealyticum $( 1 . 5 7 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ), Mycobacterium tuberculosis $( 7 . 3 3 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ , and Legionella pneumophilia $( 2 . 6 3 { \times } 1 0 ^ { 8 } \mathrm { C F U / m L } )$ ). In addition, extracted DNA was used for a vancomycin-resistant $E .$ . faecium isolate carrying the vanM gene ( $\mathrm { 5 n g / m L }$ ; ${ \sim } 1 { \times } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ ).

The selection of organisms focused on species that may be found in positive blood cultures (clinically relevant) and/or those that are closely related to target organisms (nearest neighbors). Organisms were also selected based on antimicrobial resistance phenotypes and the presence or absence of the antimicrobial resistance genes identified by the BCID Panel. The tested organisms were divided into two categories:

on-panel organisms and off-panel organisms. On-panel exclusivity testing included a total of 141 isolates of gram-positive bacteria, gram-negative bacteria, and yeast representing 29 genera and 98 individual species. Off-panel organisms were expected to have negative test results for all of the assays on the FilmArray BCID Panel (or positive organism results but negative results for the antimicrobial resistance genes detected by the FilmArray BCID Panel). Off-panel testing included gram-positive bacteria, gram-negative bacteria, yeast, viruses and Mycoplasmataceae.

The following tables present both on-panel and off-panel organisms that were tested and yielded the expected negative FilmArray BCID Panel test results for all BCID targets (other than the targets for which the on-panel organism were expected to be positive).

# Non-Cross Reactive Gram Positive Organisms

<table><tr><td rowspan=1 colspan=5>ON PANEL  Gram Positive Organisms</td></tr><tr><td rowspan=1 colspan=1>Enterococcus Species</td><td rowspan=1 colspan=2>Staphylococcus aureus</td><td rowspan=1 colspan=1>Coagulase-NegativeStaphylococci</td><td rowspan=1 colspan=1>Streptococcus Species</td></tr><tr><td rowspan=3 colspan=1>E. aviumE. casseliflavus(2 isolates)E. cecorumE. disparE. duransE. faecalis (3 isolates)E. faecium (2 isolates)aE. gallinarium (2 isolates)E. hiraeE. raffinosus</td><td rowspan=1 colspan=2>MSSA (18 isolates)Resistant S. aureus —BORSA (6 isolates)MRSA (mecA)VRSA (mecA, vanA)</td><td rowspan=5 colspan=1>S. capitis ssp. capitisS. capraeS. cohniiS. epidermidis(2 isolates)S. haemolyticusS. hominisS. lugdunensisS. pasteuriS. saprophyticusS. schleiferi ssp.schleiferiS. sciuriS. warneriS. xylosus</td><td rowspan=5 colspan=1>S. agalactiaeS. anginosusS. bovisS. dysgalactiaeS. gallolyticusS. mitisS. mutansS. parasanguinisS. pneumoniaeS. pseudopneumoniaeS. pyogenesS. salivarius</td></tr><tr><td rowspan=1 colspan=2>Coagulase-PositiveStaphylococci</td></tr><tr><td rowspan=3 colspan=2>S. intermediusS. lutraeS. pseudointermediusS. schleiferi ssp.coagulans</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>L. monocytogenes</td></tr><tr><td rowspan=1 colspan=5>OFF PANEL</td></tr><tr><td rowspan=1 colspan=1>Gram-positive Cocci</td><td rowspan=1 colspan=1>Gram-positive Bacilli</td><td rowspan=1 colspan=2>Listeria Species</td><td rowspan=1 colspan=1>Gram-positive Anaerobes</td></tr><tr><td rowspan=1 colspan=1>Granulicatella adiacensGemella morbillorumLactococcus lactisMacrococcuscaseolyticusMicrococcus luteusVagococcus fluvialis</td><td rowspan=1 colspan=1>ActinomycesodontolyticusBacillus cereusCorynebacteriumjeikeiumLactobacillus acidophilusMycobacteriumtuberculosisRhodococcus equiRothia mucilaginosa</td><td rowspan=1 colspan=2>L. (murrayi) grayiL. innocuaL. ivanoviissp. londoniensisL. seeligeriL. welshimeri</td><td rowspan=1 colspan=1>Clostridium perfringensPeptostreptococcusanaerobiusPropionibacterium acnes</td></tr></table>

a One isolate was tested at a concentration of $5 \mathrm { n g / m L }$ Extracted DNA; $\overline { { \sim 1 \times 1 0 ^ { 6 } } }$ CFU/mL. $^ { \mathrm { ~ b ~ } } \mathrm { A }$ false positive Streptococcus result was observed in the initial test of this isolate. The expected negative results were observed in multiple subsequent tests. No cross-reactivity between $G$ . adiacens and the BCID Panel Streptococcus assays is predicted by sequence analysis.

c Tested at a concentration of $7 . 3 3 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L }$ . d In silico analysis predicts that cross-reactivity between the Lmonocytogenes assay and some atypical strains of $L$ . innocua is possible, however, no cross-reactivity was observed in this testing.

# Non-Cross Reactive Gram-Negative Organisms

<table><tr><td colspan="4">ON PANEL</td></tr><tr><td>Acinetobacter baumannii</td><td colspan="3">Enterobacteriaceae Isolatesa Providencia acalifaciens</td></tr><tr><td>A. baumannii (2 isolates)</td><td colspan="3">Cedeceae davisiae Citrobacter freundii Escherichia hermanii Citrobacter koseri Escherichia vulneris Hafnia alvei Klebsiella oxytoca (3 isolates)</td></tr><tr><td>Haemophilus</td><td colspan="3">Cronobacter muytjensi Cronobacter sakazakii Enterobacter amnigenus Enterobacter asburiae Enterobacter cancerogenus Enterobacter cloacae Enterobacter hormaechei</td></tr><tr><td>influenzae</td><td colspan="3">Klebsiella pneumoniae Kluyvera ascorbata</td></tr><tr><td>H. influenzae (type b)</td><td colspan="3">Kluyvera intermedius</td></tr><tr><td>Neisseria meningitidis</td><td colspan="3">Leclercia adecarboxylata Enterobacter gergoviae Morganella morganii Escherichia coli (2 Pantoea agglomeransa isolates)</td></tr><tr><td>N. meningitidis</td><td colspan="3">Serratia plymuthica Proteus mirabilis Tatumella ptyseos Proteus penneri Yersinia enterocolitica</td></tr><tr><td>Pseudomonas aeruginosa</td><td colspan="3">Proteus vulgaris Yokenella regensburgei</td></tr></table>

OFF PANEL   

<table><tr><td>Acinetobacter Species A. calcoaceticus</td><td>Haemophilus Species</td><td>Pseudomonas Species</td><td>Gram-negative Bacilli</td></tr><tr><td>A. haemolyticus A. johnsonii A. junii A. lwoffii A. radioresistens</td><td>H. parahaemolyticus H. parainfluenzae H. parasuis H. somnus Neisseria Species</td><td>P. fluorescens P. luteola P. nitroreducens P. oryzihabitans P. pertucinogena P. stutzeri</td><td>Aeromonas hydrophila Brevundimonasdiminuta Moraxella catarrhalis (3 isolates) Stenotrophomonas maltophila Vibrio parahaemolyticus</td></tr><tr><td rowspan="3">A. schindleri A. ursingii A. nosocomialis (genomospecies 13TU; 2 isolates )</td><td>N. sicca N. elongate N. perflava</td><td></td><td>Gram-negative Coccobacilli</td></tr><tr><td>N. mucosa N. lactamica</td><td>Gram-negative Anaerobes Bacteroides fragilis</td><td>Bordetella pertussis</td></tr><tr><td></td><td>Veillonella parvula</td><td>Campylobacter fetus Chlamydia trachomatis Legionella pneumophiliab</td></tr></table>

aIn silico analysis indicates that cross-reactivity between the Enterobacter cloacae complex assay and Pantoea (Enterobacter) agglomerans may be possible. However, no cross-reactivity was observed in this study. b Tested at a concentration of $2 . 6 3 { \times } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .

Non-Cross-Reactive Fungi   

<table><tr><td>ON PANEL</td><td colspan="3">OFF PANEL</td></tr><tr><td>Candida Species</td><td colspan="2">Candida Species</td><td>Non-Candida Fungi</td></tr><tr><td>C. albicans C. glabrata C. krusei</td><td rowspan="3">C. dubliniensis C. lusitaniae C. metapsilosis C. multigemmisa</td><td>C. sojae C. viswanathii C. guilliermondii</td><td>Aspergillus fumigatus Debaryomyces hansenii</td></tr><tr><td>C. parapsilosis C. tropicalis</td><td>Schizosaccharomyces pombe</td><td>Kluyveromyces lactis Saccharomyces cerevisiae</td></tr></table>

a In silico analysis predicts that cross-reactivity between the Cparapsilosis assay and C. multigemmis is possible, however, no cross-reactivity was observed in this testing.

# Non-Cross-Reactive Viruses and Mycoplasmataceae

<table><tr><td rowspan=1 colspan=2>OFF PANEL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmataceae Isolates</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis (3.16×107 CFU/mL )Ureaplasma urealyticum (1.57×106 CFU/mL)</td><td rowspan=1 colspan=1>Cytomegalovirus (1.67×104TCID50/mL)Epstein Barr Virus (1.00×105 TCID50/mL)Herpes Simplex Virus - Type 1 (1:30 dilution of stock)Varicella Zoster Virus (8.17×103 TCID50/mL)</td></tr></table>

Non-cross-reactive with Antimicrobial Resistance Gene Assays   

<table><tr><td colspan="2">ON PANEL</td><td>OFF PANELa</td></tr><tr><td colspan="3">mecA</td></tr><tr><td colspan="2">Methicillin Resistant Staphylococci (mecA)</td><td>Borderline Oxacillin Resistant S. aureus</td></tr><tr><td rowspan="2">Staphylococcus epidermidis- MRSE Staphylococcus aureus-MRSA</td><td rowspan="2">mecA mecA mecA/vanA</td><td>(BORSA) Staphylococcus aureus-BORSA (6 isolates)</td></tr><tr><td>Methicillin Sensitive Staphylococci Staphylococcus aureus-MSSA (18 isolates)b</td></tr><tr><td>Staphylococcus aureus-VRSA</td><td></td><td>Staphylococcus epidermidis-MRSE (1 isolate) Staphylococcus spp. (16 isolates)</td></tr><tr><td colspan="2">vanA/B Vancomycin Resistant Enterococci (vanA/B)</td><td>Vancomycin Resistant Enterococci (non-vanA/B)</td></tr><tr><td rowspan="4">Enterococcus faecalis Enterococcus faecium</td><td>vanB vanA</td><td rowspan="2">Enterococcus casseliflavus vanC Enterococcus casseliflavus vanC</td></tr><tr><td rowspan="5"></td></tr><tr><td>Enterococcus gallinarium vanC Enterococcus gallinarium vanC</td></tr><tr><td>Vancomycin Sensitive Enterococci Enterococcus spp. (8 isolates)</td></tr><tr><td rowspan="2"></td></tr><tr><td rowspan="2">Carbapenem Resistant Enterobacteriaceae (KPC)</td></tr><tr><td>Carbapenem Resistant Enterobacteriaceae (non- KPC-2</td></tr><tr><td rowspan="8">Klebsiella oxytoca Klebsiella pneumoniae Serratia marcescens</td><td rowspan="2">KPC)</td><td rowspan="2">Unknown</td></tr><tr><td>Klebsiella pneumoniae Klebsiella pneumoniae NDM</td></tr><tr><td>KPC-4 KPC-2 Isolates</td><td>Carbapenem Sensitive/Beta-lactam Resistant</td></tr><tr><td>Klebsiella pneumoniae Klebsiella pneumoniae</td><td>AmpC SHV</td></tr><tr><td>Escherichia coli Acinetobacter baumannii</td><td>TEM-3/CTX-1 blaOXA</td></tr><tr><td>Moraxella catarrhalis</td><td>blaOXA</td></tr><tr><td>Moraxella catarrhalis</td><td>BRO-1(bla)/orf3</td></tr><tr><td>Carbapenem Sensitive Isolates</td><td></td></tr><tr><td>ON PANEL</td><td>OFF PANEL</td></tr><tr><td></td><td>Enterobacteriaceae (51 Isolates) Acinetobacter baumannii (1 isolate) Pseudomonas aeruginosa (2 isolates)</td></tr></table>

a Off-panel refers to the antimicrobial resistance gene. Organisms may be positive for organism assay(s). b Ten Isolates known to harbor remnants of SCCmec cassette (empty cassette strains)

The following table includes organisms for which cross-reactivity was observed with one or more of the BCID Panel assays as well as any predicted cross-reactivity as determined by in silico analysis.

# Predicted and Observed Cross-Reactivity with On-Panel or Off-Panel

Organisms   

<table><tr><td rowspan=1 colspan=1>BCID Panel Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism(s)/Isolate(s)/Gene</td></tr><tr><td rowspan=1 colspan=2>Gram-positive Bacteria</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Some coagulase-negative staphylococcia</td></tr><tr><td rowspan=1 colspan=2>Gram-negative Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii (ACB) complex species:Acinetobacter calcoaceticus (ssp. anitratus)Acinetobacter pitti (formerly genomospecies 3) b</td></tr><tr><td rowspan=1 colspan=1>Escherichia colilEnterobacteriaceae</td><td rowspan=1 colspan=1>Shigella species:Shigella boydiiShigella dysenteriaeShigella flexneriShigella sonneiEscherichia fergusonnii</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae/Enterobacteriaceae</td><td rowspan=1 colspan=1>Klebsiella variicola (aka Klebsiella pneumoniae variant 342)Enterobacter aerogenesRaoultella ornithinolytica</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens/Enterobacteriaceae</td><td rowspan=1 colspan=1>Serratia species (S. entomophila, S. ficaria, S. odorifera, and S.rubidaea)Raoultella ornithinolyticaPseudomonas aeruginosa (ATCC 25619)fPseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Haemophilus haemolyticusg</td></tr><tr><td rowspan=1 colspan=2>Yeast</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Candida orthopsilosis (Group III Candida parapsilosis) h</td></tr><tr><td rowspan=1 colspan=2>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>vanA/B</td><td rowspan=1 colspan=1>vanM i</td></tr></table>

a Cross-reactivity was not observed in this study but is predicted by in silico analysis to occur only with some species (i.e. S. epidermis, $S _ { \cdot }$ capitis and $S .$ haemolyticus) when present in a sample at very high levels. This cross-reactivity was observed infrequently in pre-analytical studies and the clinical evaluation (estimated occurrence of ${ \sim } 0 . 2 5 \%$ of all Staphylococcus positive patient samples). b Acinetobacter calcoaceticus-baumannii (ACB) complex species are often mis-identified as $A$ . baumannii by automated and manual microbial identification methods. c Cross-reactivity was not observed when ATCC 31898 was tested in the inclusivity study at a concentration ${ \sim } 1 { \bf { x } } 1 0 ^ { 8 } \mathrm { { C F U / m L } }$ , but cross-reactivity was observed in clinical cultures containing $R$ . ornithinolytica. d Cross-reactivity was observed only at high organism concentration $( \geq 1 0 ^ { 9 } \mathrm { C F U / m L } )$ ; rare human pathogens.

# g. Assay cut-off:

The BCID Melt Detector software determines whether a FilmArray BCID assay result is positive or negative using a predefined algorithm that includes Tm values, fluorescence values, and analysis of melting curves. For each BCID target sequence there is a mean Tm at which PCR2 products from the most similar isolates will melt. The range surrounding that mean is where amplicon from more diverse isolates are expected to melt. The location and width of each Tm range is assay specific and is used in the FA analysis software to ensure specificity in target detection. Due to the number of species, rarity of some strains, and variability of strain sequences detected by BCID assays, thousands of clinical samples would need to be tested from isolates spanning geography and time in order to estimate the Tm ranges for each target experimentally.

The Melt Detector software uses a mathematical model for Tm prediction that was developed and verified using database sequences, beta site testing, analytical studies, and reference runs. Melting ranges for all 31 pathogen, antibiotic resistance gene targets, and control assays were initially determined and subsequently validated. The validation of the BCID Melt Detector software was performed by comparing FilmArray BCID test results obtained from well-characterized clinical and analytical samples to expert annotation (review of melt curves and assay calls made by the Biofire software team). For individual melt curves, the observed sensitivity and specificity of the melt detector software as compared to expert annotation are greater than $9 8 . 5 \%$ and $9 9 . 9 \%$ , respectively. For the Analysis Software, the observed sensitivity and specificity as compared to expert annotation, of the assay calls are greater than $9 7 . 0 \%$ and $9 9 . 8 5 \%$ , respectively. The validation results surpassed the predefined acceptance criteria of $59 5 \%$ accuracy as compared to expert annotation.

# h. Fresh versus Frozen Study:

In order to utilize frozen clinical blood culture samples in the evaluation of FilmArray BCID Panel, an analytical study was conducted to demonstrate that preservation of samples by freezing at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ does not affect the accuracy of test results compared to freshly collected or freshly prepared samples.

A total of 62 FilmArray runs were attempted in this evaluation of 60 frozen specimens, 60 of which were completed. There was one run failure each for a software error and an instrument communication error. Of the 60 completed runs, no control failures occurred. Specimens had been stored at $\le 7 0 ^ { \circ } \mathrm { C }$ for an average of 44 days before re-testing for this study. The median age was 47 days (Range: 9-83 days). All of the analytes that were originally detected in the specimens when tested fresh were also detected after storage at $\le 7 0 ^ { \circ } \mathrm { C }$ . Further, no additional analytes were detected in frozen specimens that had not been previously detected when they were tested fresh.

Analyte Detections in Frozen Specimens compared to Fresh Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>PPA</td><td rowspan=1 colspan=3>Frozen/Fresh</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>vanA/B</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td></tr></table>

Additionally, A total of four (4) co-infections were identified when the specimens were originally tested fresh, all of which were also detected when the specimens were re-tested from frozen aliquots.

Co-infection Analysis   

<table><tr><td rowspan=1 colspan=1>Coinfections</td><td rowspan=1 colspan=1>Frozen/Fresh</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>E. coli + Enterococcus + vanA/B</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>S. aureus + mecA + S. agalactiae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>S. aureus + mecA + P. aeruginosa + Enterococcus</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus + K. pneumoniae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td></tr></table>

An analysis of Cp and Tm values was conducted to compare the performance of the BCID individual assays in the original, freshly tested specimens and the frozen specimens. The difference in average and median $\mathrm { C p }$ values between the samples when tested fresh and frozen were typically within 2 cycles and the delta $\mathrm { C p }$ were observed in both directions (both earlier and later relative to fresh testing. The largest variations were consistent with run-to-run variation in $\mathrm { C p }$ values that were observed in the Reproducibility Study when the same samples were tested multiple times over several days. Similarly, the delta Tm values ranged from 0 to $\mathrm { T m } 0 . 9 ^ { \circ } \mathrm { C }$ (median 0.4), which is consistent with the variation of Tm values observed in the Reproducibility Study. Tm changes were also observed in both directions.

# i. Interference:

Substances that could be present in blood culture samples or introduced during sample handling were evaluated for potential interference. Potentially interfering substances were added to simulated positive blood culture samples which contained simulated blood culture matrix (human whole blood that had been incubated in a blood culture bottle) and one of six different organism mixes. Each organism mix contained two live pathogens at a concentration equivalent to the level determined to be present when a blood culture bottle is detected as positive by the blood culture instrument as shown in the following table. Twelve targeted organisms and four targeted resistance genes were evaluated.

Organism Mixes and Targeted Test Concentrations   

<table><tr><td rowspan=1 colspan=1>Mixture ID</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>IBR Level*</td></tr><tr><td rowspan=2 colspan=1>Mix 1</td><td rowspan=1 colspan=1>Enterococcus faecium (vanA)</td><td rowspan=1 colspan=1>JMI</td><td rowspan=1 colspan=1>475</td><td rowspan=1 colspan=1>1.53×10CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-255</td><td rowspan=1 colspan=1>6.41×108CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Mix 2</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9955</td><td rowspan=1 colspan=1>2.02×10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae (KPC)</td><td rowspan=1 colspan=1>JMI</td><td rowspan=1 colspan=1>766</td><td rowspan=1 colspan=1>1.92×108 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Mix 3</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>66029</td><td rowspan=1 colspan=1>9.70×103 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>90878</td><td rowspan=1 colspan=1>4.82×10CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Mix 4</td><td rowspan=1 colspan=1>Staphylococcus aureus (MRSA)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1747</td><td rowspan=1 colspan=1>8.60×10CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>29906</td><td rowspan=1 colspan=1>7.58×10 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Mix 5</td><td rowspan=1 colspan=1>Enterococcus faecalis (vanB)</td><td rowspan=1 colspan=1>JMI</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>3.01×10CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>10231</td><td rowspan=1 colspan=1>3.12×104 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Mix 6</td><td rowspan=1 colspan=1>Haemophilus influenzae (type b)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>10211</td><td rowspan=1 colspan=1>2.88×108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>15545</td><td rowspan=1 colspan=1>1.45×10CFU/mL</td></tr></table>

Potentially interfering test substances were spiked at levels predicted to be above the concentration of the substance likely to be found in a blood culture specimen. For most substances the concentration tested was up to three times the level expected to be found in patient blood/culture samples. The following tables list the test substances and final test concentrations used in this study.

Endogenous Substances   

<table><tr><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>0.20 mg/mL</td></tr><tr><td rowspan=1 colspan=1>γ-globulin</td><td rowspan=1 colspan=1>60 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Human genomic DNA</td><td rowspan=1 colspan=1>0.2, 2, and 20 ng/μL</td></tr></table>

# Exogenous Substances

<table><tr><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>75 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>103 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>10 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin sulfate</td><td rowspan=1 colspan=1>10 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>900 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin/Clavulanate</td><td rowspan=1 colspan=1>75 μg/mL/6.9 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>966 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>15 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanetholesulfonate (SPS)</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>3 Units/mL</td></tr><tr><td rowspan=1 colspan=1>Bleach</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>7%</td></tr></table>

On each day of testing, one specimen per organism mix was evaluated without any interfering substance to serve as a positive control (no interference) to which the test specimens were compared. For each endogenous and exogenous test substance, one specimen per organism mix was spiked with the appropriate amount of test substance.

Testing of specimens with the above described potential interfering substances produced the expected positive and negative results indicating that none of the endogenous or exogenous test substances compete or interfere with obtaining accurate test results with the FilmArray BCID system. In addition, there were no consistent trends for either Cp or Tm values when comparing the presence and absence of each endogenous or exogenous substance. In summary, study data suggests that higher than expected levels of the all evaluated substances will not interfere with obtaining

accurate test results with the FilmArray BCID Panel.

# j. Mixed Culture Study (Microbial interference):

A study was performed to evaluate whether the presence of high levels of organisms will interfere with detection of representative organisms at "bottle ring" concentrations. Three bacteria (Staphylococcus epidermidis, Escherichia coli, and Streptococcus mitis) that can be detected by the FilmArray BCID Panel were tested at high levels. These organisms were selected for their high prevalence in mixed blood culture specimens. Five bacteria (Propionibacterium acnes, Corynebacterium jeikeium, Bacillus cereus, Micrococcus luteus, and Clostridium perfringens) that are not detected by assays in the FilmArray BCID Panel, but that can be found in blood cultures as contaminants, were also tested to determine if their presence interferes with the ability of the FilmArray BCID Panel to detect organisms targeted by the BCID Panel.

Testing was performed by spiking a high concentration of each potentially interfering bacteria $( { \sim } 1 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 1 0 } \mathrm { C F U / \dot { m L } } )$ into blood culture specimens that each contained 2 BCID organisms (Mix 1-6 described above for interference testing) at "bottle ring" concentrations. Study results demonstrated that high concentrations of the on-panel BCID microorganisms spiked into blood culture samples produced positive results for the relevant assays on the BCID Panel but did interfere with any expected results for other analytes. High concentrations of off-panel BCID microorganisms also showed no interference with the detection of any FilmArray BCID organism with no unexpected false negative or false positive results observed.

# k. Testing of Additional Blood Culture Bottle Types

Fourteen different bottle types from three different blood culture systems (BacT/Alert, BACTEC and VersaTREK) were evaluated analytically with the FilmArray BCID panel. Blood culture bottles/media were tested with the recommended ratio of blood to media. The six organism mixes described above (see interference section) were separately spiked into each bottle type with final concentrations tested at "bottle ring" levels. Results were evaluated to determine if the bottle type provided the expected positive results. A negative control (simulated blood culture matrix) was also tested for each bottle type. The test results for the negative control (NC) and the positive mixes (which give negative results for many organisms on the panel) were evaluated to determine if the bottle type posed a risk of false positive test results due to presence of non-viable organism or organism nucleic acids in the culture media. The 14 bottle types listed in the table below were evaluated and study results demonstrated correct positive and negative FilmArray BCID results with each bottle type.

<table><tr><td colspan="2" rowspan="1">Blood Culture Bottle Types</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert SA Standard Aerobic</td><td colspan="1" rowspan="1">BACTEC Plus Aerobic/F</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert SN Standard Anaerobic</td><td colspan="1" rowspan="1">BACTEC Plus Anaerobic/F</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert FA Aerobic FAN</td><td colspan="1" rowspan="1">BACTEC Pediatric Plus</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert FN Anaerobic FAN</td><td colspan="1" rowspan="1">BACTEC Lytic/10 Anaerobic/F</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert PF Pediatric FAN</td><td colspan="1" rowspan="1">VersaTREK REDOX 1</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Aerobic</td><td colspan="1" rowspan="1">VersaTREK REDOX 2</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Anaerobic</td><td colspan="1" rowspan="1">BacT/ALERT FA Plus Aerobic</td></tr></table>

False positive test results were observed during development and beta testing of the FilmArray BCID panel when using BacT/ALERT FA $\mathrm { \dot { F } A N } ^ { \mathfrak { B } }$ Aerobic media. An investigation determined that these bottle types contain nucleic acids and/or non-viable organism that can be detected by FilmArray BCID Panel (multiple organisms could be detected in different test runs and media lots). It was confirmed that the charcoal used in FAN media can have a relatively high bio-burden; therefore false results could be generated from the media itself. Because of the potential for false positive results associated with charcoal-containing media, BacT/ALERT FAN media should not be used with the FilmArray BCID Panel. This information is presented in the package insert.

# l. Carryover Study:

The potential for run-to-run carryover was evaluated by determining whether a high positive sample tested in one pouch would cause false positive results in subsequently tested pouches. Each high positive sample contained ${ \sim } 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ of a unique onpanel bacteria and each negative sample contained a unique off-panel organism or no organism. For each set of samples the negative sample was loaded in the same workspace, using the same Pouch Loading Station and tested directly after the high positive sample using the same FilmArray instrument. No false positive results were observed during consecutive testing of 10 high positive samples alternating with negative samples, demonstrating that recommended sample handling and testing practices are effective in preventing false positive results due to carryover or crosscontamination between samples.

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison: N/A

3. Clinical studies:

The clinical performance of the FilmArray BCID Panel was established during a two armed clinical study which was conducted at eight U.S. clinical sites over an eight month time period. The study included a prospective residual blood culture arm and a seeded

blood culture arm. In the prospective arm, 1635 prospectively-collected residual blood culture samples (pediatric and adult) were initially included in the study. Only blood cultures collected in BD BACTEC Plus Aerobic/F blood culture bottles were included in the study. Sixty-seven (67) specimens were excluded from the study. The most common reasons for exclusion were when specimens were ${ > } 8$ hours past positivity, incomplete reference/comparator data were provided, or the specimen was from a subject who had a previous specimen included in the study. In the seeded culture arm, analytes proven to be of low prevalence in the prospective arm were evaluated by seeding previously characterized isolates into blood culture bottles and incubating until positivity. A total of 716 seeded cultures were initiated for the study. Seeded specimens were also prepared in BD BACTEC Plus Aerobic/F blood culture bottles. Seventy-seven (77) cultures were excluded from the study. The most common reasons for exclusion were that the specimens were ${ > } 8$ hours past positivity, the seeded culture was not called positive by the automated blood culture system, or the culture was contaminated or inconsistent with the intended seed organism. The final specimen set consisted of 2207 blood cultures (1568 prospective and 639 seeded). Specimens were tested by the FilmArray BCID Panel either from freshly positive blood cultures or from frozen blood culture aliquots. The following table provides a summary of demographic information for the 1568 specimens included in the prospective arm of the study.

Demographic Summary for Prospective Arm of FilmArray BCID Clinical Evaluation   

<table><tr><td rowspan=1 colspan=2>Prospective Study Specimens: Total Specimens - 1568</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>917 (58%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>651 (42%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>≤ 1 year</td><td rowspan=1 colspan=1>57 (4%)</td></tr><tr><td rowspan=1 colspan=1>1 - 17 years</td><td rowspan=1 colspan=1>92 (6%)</td></tr><tr><td rowspan=1 colspan=1>18 - 44 years</td><td rowspan=1 colspan=1>281 (18%)</td></tr><tr><td rowspan=1 colspan=1>45 - 64 years</td><td rowspan=1 colspan=1>583 (37%)</td></tr><tr><td rowspan=1 colspan=1>65 - 84 years</td><td rowspan=1 colspan=1>442 (28%)</td></tr><tr><td rowspan=1 colspan=1>≥ 85 years</td><td rowspan=1 colspan=1>113 (7%)</td></tr></table>

Positive blood cultures (prospective and seeded) were tested with the FilmArray BCID Panel. The performance of FilmArray BCID was evaluated by comparing each FilmArray BCID Panel result with the appropriate comparator/reference methods shown in the following table.

<table><tr><td rowspan=1 colspan=1>Test Result</td><td rowspan=1 colspan=1>Reference/Comparator Method(s)</td></tr><tr><td rowspan=1 colspan=1>All organism detectionsexcept Acinetobacter baumannii</td><td rowspan=1 colspan=1>Standard manual and automated microbiological/biochemicalidentification methods a</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii detection</td><td rowspan=1 colspan=1>Standard manual and automated microbiological/biochemicalidentification methodsPlus16S PCR with bi-directional sequencing of all A. calcoaceticus-baumannii complex isolates for characterization as A.baumannii or non-A. baumannii a</td></tr><tr><td rowspan=1 colspan=1>Antimicrobial resistance gene detectionsin specimens in which an associated organismwas detected(mecA from Staphylococcus; vanA/B fromEnterococcus, KPC from Enterobacteriaceae,Acinetobacter baumannii, and Pseudomonasaeruginosa)</td><td rowspan=1 colspan=1>Method 1: PCR with bi-directional sequencing for specificresistance gene direct from blood culturebMethod 2: PCR with bi-directional sequencing for specificresistance gene from appropriate cultured isolatesbInformational: Standard manual and automated phenotypicantimicrobial susceptibility testing of appropriate culturedisolates (methicillin resistance, vancomycin resistance, andcarbapenem resistance (and/or carbapenemase production)according to current CLSI criteria)</td></tr></table>

A total of 2207 specimens were tested in the clinical evaluation, of which $9 9 \%$ (2185/2207) were successful on the first test. Six (6) initial tests were incomplete due to instrument/software errors (5 tests) or a user aborted run (1 test). Sixteen (16) of the completed runs were invalid due to a pouch control failure. Valid results were achieved after a single retest for the 22 incomplete/invalid specimens, resulting in a final successful testing rate of $100 \%$ .

Clinical sensitivity or positive percent agreement (PPA) was calculated as $100 \%$ x $\left( \mathrm { T P / T P + F N } \right)$ . True positive (TP) indicates that both FilmArray BCID and the reference/comparator method had a positive result for a specific analyte, and false negative (FN) indicates that the FilmArray BCID result was negative while the reference/comparator method was positive. Clinical specificity or negative percent agreement (NPA) was calculated as $100 \%$ x $( \mathrm { T N } / \mathrm { T N } + \mathrm { F P } )$ . True negative (TN) indicates that both FilmArray BCID and the reference/comparator method had a negative result for a specific analyte, and false positive (FP) indicates that the FilmArray BCID result was positive while the reference/comparator method was negative. The exact binomial twosided $9 5 \%$ confidence interval was calculated. The results are summarized in the following tables.

FilmArray BCID Clinical Performance Summary – Gram-Positive Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="2" rowspan="2">Gram-Positive Bacteria</td><td colspan="2">Sensitivity/PPA a</td><td colspan="3">Specificity/NPA </td></tr><tr><td>TP/TP + FN</td><td>% 95% CI</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="5">Enterococcus</td><td>Prospective Fresh</td><td>55/55</td><td>100 93.5-100</td><td>762/766</td><td>99.5</td><td>98.7-99.9</td></tr><tr><td>Prospective Frozen</td><td>43/46 93.5</td><td>82.1-98.6</td><td>701/701</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>12/12 100</td><td>73.5-100</td><td>407/407</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>17/17 100</td><td>80.5-100</td><td>203/203</td><td>100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>127/130 97.7</td><td>93.4-99.5</td><td>2073/2077 b</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Listeria monocytogenes</td><td>Prospective Fresh</td><td>0/0</td><td>- -</td><td>821/821</td><td>100</td><td>99.6-100</td></tr><tr><td>Prospective Frozen</td><td>0/0 -</td><td>-</td><td>747/747</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>23/23</td><td>100 85.2-100</td><td>396/396</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>13/13</td><td>100 75.3-100</td><td>207/207</td><td>100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>36/36 100</td><td>90.3-100</td><td>2171/2171</td><td>100</td><td>99.8-100</td></tr><tr><td rowspan="5">Staphylococcus</td><td>Prospective Fresh</td><td>405/418</td><td>96.9 94.7-98.3</td><td>401/403</td><td>99.5</td><td>98.2-99.9</td></tr><tr><td>Prospective Frozen</td><td>364/379</td><td>96.0 93.6-97.8</td><td>359/368</td><td>97.6</td><td>95.4-98.9</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>- -</td><td>418/419</td><td>99.8</td><td>98.7-100</td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100 2.5-100</td><td>219/219</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>770/798</td><td>96.5 95.0-97.7</td><td>1397/1409 c</td><td>99.1</td><td>98.5-99.6</td></tr><tr><td rowspan="5">Staphylococcus aureus</td><td>Prospective Fresh</td><td>133/136</td><td>97.8 93.7-99.5</td><td>685/685</td><td>100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>120/121</td><td>99.2 95.5-100</td><td>622/626</td><td>99.4</td><td>98.4-99.8</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td>419/419</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td>- -</td><td>220/220</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>253/257 d</td><td>- 98.4 96.1-99.6</td><td>1946/1950 d</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Streptococcus</td><td>Prospective Fresh</td><td>73/77</td><td>94.8</td><td>87.2-98.6 740/744</td><td>99.5</td><td>98.6-99.9</td></tr><tr><td>Prospective Frozen</td><td>63/64</td><td>98.4 91.6-100</td><td>683/683</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>18/18</td><td>81.5-100</td><td>401/401</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>44/44</td><td>100</td><td></td><td></td><td></td></tr><tr><td>Overall</td><td>198/203</td><td>100 92.0-100 94.3-99.2</td><td>175/176 1999/2004</td><td>99.4 99.8</td><td>96.9-100</td></tr><tr><td rowspan="5">Streptococcus agalactiae (Group B)</td><td>Prospective Fresh</td><td>8/8</td><td>97.5 100</td><td>63.1-100</td><td>813/813 100</td><td>99.4-99.9 99.5-100</td></tr><tr><td>Prospective Frozen</td><td>10/10</td><td>100 69.2-100</td><td>737/737</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>3/3</td><td>100 29.2-100</td><td>416/416</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>15/15</td><td>100</td><td></td><td></td><td></td></tr><tr><td>Overall</td><td></td><td>78.2-100 90.3-100</td><td>205/205 2171/2171</td><td>100</td><td>98.2-100</td></tr><tr><td rowspan="5">Streptococcus pneumoniae</td><td>Prospective Fresh</td><td>36/36 15/15</td><td>100 100 78.2-100</td><td></td><td>100</td><td>99.8-100</td></tr><tr><td>Prospective Frozen</td><td>10/10</td><td>100</td><td>805/806</td><td>99.9</td><td>99.3-100 99.5-100</td></tr><tr><td>Seeded Fresh</td><td>4/5</td><td>69.2-100 28.4-99.5</td><td>737/737 413/414</td><td>100</td><td>98.7-100</td></tr><tr><td>Seeded Frozen</td><td></td><td>80.0</td><td></td><td>99.8</td><td></td></tr><tr><td>Overall</td><td>7/7 36/37</td><td>100 59.0-100</td><td>213/213</td><td>100</td><td>98.3-100</td></tr><tr><td rowspan="5">Streptococcus pyogenes Group A)</td><td>Prospective Fresh</td><td>5/5</td><td>97.3</td><td>85.8-99.9 2168/2170</td><td>99.9</td><td>99.7-100</td></tr><tr><td>Prospective Frozen</td><td></td><td>100 47.8-100</td><td>815/816</td><td>99.9</td><td>99.3-100</td></tr><tr><td></td><td>2/2</td><td>100 15.8-100</td><td>745/745</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>9/9</td><td>100 66.4-100</td><td>410/410</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen Overall</td><td>22/22 38/38</td><td>100 84.6-100 100 90.7-100</td><td>198/198 2168/2169</td><td>100 99.9</td><td>98.2-100 99.7-100</td></tr></table>

aSensitivity and Specificity refer to performance with the prospective specimens only; Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) refer to performance with the seeded specimens. $^ { \mathsf { b } } 3 / 4$ false positive Enterococcus specimens contained Staphylococcus; the false positive results may be due to crossreactivity. c Isolates from 16/28 false negative Staphylococcus specimens were identified as the newly described species S.

pettenkoferi by bi-directional sequencing. Bidirectional sequencing confirmed the presence of Staphylococcus in 10/12 false positive specimens; 2 were S. aureus, 6 were S. epidermidis, and 1 was S. haemolyticus.

d Bidirectional sequencing identified 2 isolates from S. aureus false negative specimens as $S .$ . hominis and $S .$ . epidermidis; they were not S. aureus. Bidirectional sequencing confirmed the presence of $S _ { \cdot }$ . aureus in 1/4 false positive specimens. One false positive and one false negative S. aureus were consecutively tested specimens and may be due to sample mix-up.

e Bidirectional sequencing confirmed the presence of S. mitis in 1/5 false positive Streptococcus specimens.

FilmArray BCID Clinical Performance Summary – Gram-Negative Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification plus 16S Sequencing for Speciation for A. baumannii)   

<table><tr><td colspan="2"></td><td colspan="2">Sensitivity/PPA a</td><td colspan="3">Specificity/NPA a</td></tr><tr><td colspan="2">Gram-Negative Bacteria</td><td colspan="2">TP/TP + FN % 95% CI</td><td colspan="3">TP/TP + FP %</td></tr><tr><td rowspan="5">Acinetobacter baumannii</td><td>Prospective Fresh</td><td>7/7</td><td>100 59.0-100</td><td>813/814</td><td>99.9</td><td>95% CI 99.3-100</td></tr><tr><td>Prospective Frozen</td><td>7/7</td><td>100 59.0-100</td><td>739/740</td><td>99.9</td><td>99.2-100</td></tr><tr><td>Seeded Fresh</td><td>20/20</td><td>100 83.2-100</td><td>397/399</td><td>99.5</td><td>98.2-99.9</td></tr><tr><td>Seeded Frozen</td><td>17/17</td><td>100 80.5-100</td><td>202/203</td><td>99.5</td><td>97.3-100</td></tr><tr><td>Overall</td><td>51/51</td><td>100 93.0-100</td><td>2151/2156 b</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Enterobacteriaceae</td><td>Prospective Fresh</td><td>153/156</td><td>98.1 94.5-99.6</td><td></td><td>665/665 100</td><td>99.4-100</td></tr><tr><td>Prospective Frozen</td><td>150/154</td><td>97.4 93.5-99.3</td><td>589/593</td><td>99.3</td><td>98.3-99.8</td></tr><tr><td>Seeded Fresh</td><td>93/93</td><td>100 96.1-100</td><td>326/326</td><td>100</td><td>98.9-100</td></tr><tr><td>Seeded Frozen</td><td>94/95</td><td>98.9 94.3-100</td><td>125/125</td><td>100</td><td>97.1-100</td></tr><tr><td>Overall</td><td>490/498</td><td>98.4 96.9-99.3</td><td>1705/1709</td><td>99.8</td><td>99.4-99.9</td></tr><tr><td rowspan="5">Enterobacter cloacae complex</td><td>Prospective Fresh</td><td>10/11</td><td>90.9</td><td>58.7-99.8</td><td>809/810 99.9</td><td>99.3-100</td></tr><tr><td>Prospective Frozen</td><td>11/11</td><td>100 71.5-100</td><td></td><td>734/736 99.7</td><td>99.0-100</td></tr><tr><td>Seeded Fresh</td><td>8/8</td><td>100 63.1-100</td><td>411/411</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>9/9</td><td>100 66.4-100</td><td>211/211</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>38/39</td><td>97.4 86.5-99.9</td><td>2165/2168</td><td>99.9</td><td>99.6-100</td></tr><tr><td rowspan="5">Escherichia coli</td><td>Prospective Fresh</td><td>77/79</td><td>97.5</td><td>91.2-99.7</td><td>742/742 100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>68/69</td><td>98.6 92.2-100</td><td></td><td>674/678 99.4</td><td>98.5-99.8</td></tr><tr><td>Seeded Fresh</td><td>4/4</td><td>100 39.8-100</td><td>414/415</td><td>99.8</td><td>98.7-100</td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100 2.5-100</td><td></td><td>219/219 100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>150/153 </td><td>98.0 94.4-99.6</td><td>2049/2054</td><td>99.8</td><td>99.4-99.9</td></tr><tr><td rowspan="5">Klebsiella oxytoca</td><td>Prospective Fresh</td><td>4/4</td><td>100</td><td>39.8-100</td><td>817/817 100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>1/2</td><td>50 1.3-98.7</td><td></td><td>744/745 99.9</td><td>99.3-100</td></tr><tr><td>Seeded Fresh</td><td>32/36</td><td>88.9 73.9-96.9</td><td>383/383</td><td>100</td><td>99.0-100</td></tr><tr><td>Seeded Frozen</td><td>22/22</td><td>100 84.6-100</td><td></td><td>198/198 100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>59/64</td><td>92.2 82.7-97.4</td><td>2142/2143</td><td>99.9</td><td>99.7-100</td></tr><tr><td rowspan="5">Klebsiella pneumoniae</td><td>Prospective Fresh</td><td>33/34</td><td>97.1</td><td>84.7-99.9</td><td>786/787 99.9</td><td>99.3-100</td></tr><tr><td>Prospective Frozen</td><td>35/37</td><td>94.6 81.8-99.3</td><td>705/710</td><td>99.3</td><td>98.4-99.8</td></tr><tr><td>Seeded Fresh</td><td>13/13</td><td>100 75.3-100</td><td>403/406</td><td>99.3</td><td>97.9-99.8</td></tr><tr><td>Seeded Frozen</td><td>21/21</td><td>100 83.9-100</td><td></td><td>199/199 100</td><td>98.2-100</td></tr><tr><td>Overall</td><td>102/105</td><td>97.1 91.9-99.4</td><td>2093/2102f</td><td>99.6</td><td>99.2-99.8</td></tr><tr><td rowspan="5">Proteus</td><td>Prospective Fresh</td><td>11/11</td><td>100</td><td>71.5-100</td><td>810/810 100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>11/11</td><td>100 71.5-100</td><td></td><td>736/736 100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>2/2</td><td>100 15.8-100</td><td>417/417</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>15/15</td><td>100 78.2-100</td><td></td><td></td><td>98.2-100</td></tr><tr><td>Overall</td><td>39/39</td><td>100 91.0-100</td><td>205/205 2168/2168</td><td>100 100</td><td>99.8-100</td></tr><tr><td rowspan="5">Serratia marcescens</td><td>Prospective Fresh</td><td>14/14</td><td>100</td><td>76.8-100</td><td>807/807</td><td>100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>8/8</td><td>100</td><td>63.1-100</td><td>739/739</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>28/28</td><td>100</td><td>87.7-100</td><td>390/391</td><td>99.7</td><td>98.6-100</td></tr><tr><td>Seeded Frozen</td><td>26/27</td><td>96.3</td><td>81.0-99.9</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>193/193</td><td>100</td><td>98.1-100</td></tr><tr><td colspan="2" rowspan="2">Gram-Negative Bacteria</td><td colspan="3">Sensitivity/PPA a</td><td colspan="3">Specificity/NPA </td></tr><tr><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TP/TP + FP</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="5">Haemophilus</td><td>Overall</td><td>76/778</td><td>98.7</td><td>93.0-100</td><td>2129/2130 8</td><td>99.9</td><td>99.7-100</td></tr><tr><td>Prospective Fresh</td><td>5/5</td><td>100</td><td>47.8-100</td><td>816/816</td><td>100</td><td>99.5-100</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100</td><td>29.2-100</td><td>744/744</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>29/29</td><td>100</td><td>88.1-100</td><td>390/390</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>6/6</td><td>100</td><td>54.1-100</td><td>214/214</td><td>100</td><td>98.3-100</td></tr><tr><td rowspan="5">Neisseria meningitidis</td><td>Overall Prospective Fresh</td><td>43/43 1/1</td><td>100</td><td>91.8-100</td><td>2164/2164</td><td>100</td><td>99.8-100</td></tr><tr><td>Prospective Frozen</td><td>0/0</td><td>100</td><td>2.5-100</td><td>820/820</td><td>100</td><td>99.6-100</td></tr><tr><td>Seeded Fresh</td><td>30/30</td><td>-</td><td>-</td><td>747/747</td><td>100</td><td>99.5-100</td></tr><tr><td></td><td></td><td>100</td><td>88.4-100</td><td>389/389</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen Overall</td><td>5/5 36/36</td><td>100 100</td><td>47.8-100 90.3-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td rowspan="5">Pseudomonas aeruginosa</td><td>Prospective Fresh</td><td>19/19</td><td>100</td><td>82.4-100</td><td>2171/2171 802/802</td><td>100 100</td><td>99.8-100 99.5-100</td></tr><tr><td>Prospective Frozen</td><td>32/33</td><td>97</td><td>84.2-99.9</td><td>713/714</td><td>99.9</td><td>99.2-100</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td>-</td><td>419/419</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td>-</td><td></td><td>220/220</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>51/52h</td><td>98.1</td><td>89.7-100</td><td>2154/2155</td><td>99.9</td><td>99.7-100</td></tr></table>

a Sensitivity and Specificity refer to performance with the prospective specimens only; Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) refer to performance with the seeded specimens.

b Bidirectional sequencing identified isolates from 4 false positive specimens as $A$ . pittii (genomospecies 3); this species cross-reacts with the A. baumannii assay. These four isolates were identified as A. baumannii by phenotypic methods. 6 other isolates originally identified as A. baumannii by phenotypic methods were identified by bidirectional sequencing as $A$ . nosocomialis (genomospecies 13; 4 isolates), A. bereziniae, and $A$ . radioresistens; these 6 isolates did not cross-react with the A. baumannii assay.

c One false positive and one false negative Enterobacteriaceae were consecutively tested specimens and may be due to sample mix-up. One isolate from another false negative specimen, identified as $E$ . coli by phenotypic methods, was identified as Pasteurella, and not $E$ . coli, by bidirectional sequencing.

d One false positive and one false negative $E$ . coli were consecutively tested specimens and may be due to sample mix-up.

e Bidirectional sequencing identified 4/5 isolates from false negative $K .$ oxytoca specimens as the closely related species, Raoultella ornithinolytica, and not $K .$ . oxytoca. The misidentification is a known limitation of phenotypic testing methods for this species.

f The isolate from one false negative $K .$ . pneumoniae specimen was identified as the closely related organism, Raoultella planticola and not $K .$ . pneumoniae. $6 / 9$ false positive $K .$ . pneumoniae results appear to be due to crossreactivity with Enterobacter aerogenes and Raoultella ornithinolytica (misidentified as $K$ . oxytoca by phenotypic methods).

g Bidirectional sequencing identified the isolate from the one false negative $S .$ . marcescens specimen as being in the S. proteomaculans/grimesii group and not S. marcescens. The one false positive $S .$ marcescens result appears to be due to cross-reactivity with Raoultella ornithinolytica (misidentified as $K .$ . oxytoca by phenotypic methods).

h Bidirectional sequencing identified the isolate from the one false negative $P .$ . aeruginosa specimen as the closely related species Pseudomonas stutzeri and not $P$ . aeruginosa.

FilmArray BCID Clinical Performance Summary – Yeast Organism Results (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="2" rowspan="2">Yeast</td><td colspan="3">Sensitivity/PPA a</td><td colspan="3">Specificity/NPA a</td></tr><tr><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="5">Candida albicans</td><td>Prospective Fresh Prospective Frozen</td><td>12/12</td><td>100</td><td>73.5-100</td><td>808/809</td><td>99.9</td><td>99.3-100</td></tr><tr><td></td><td>4/4</td><td>100</td><td>39.8-100</td><td>740/743</td><td>99.6</td><td>98.8-99.9</td></tr><tr><td>Seeded Fresh</td><td>47/47</td><td>100</td><td>92.5-100</td><td>372/372</td><td>100</td><td>99.0-100</td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100</td><td>2.5-100</td><td>219/219</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>64/64</td><td>100</td><td>94.4-100</td><td>2139/2143</td><td>99.8</td><td>99.5-99.9</td></tr><tr><td rowspan="5">Candida glabrata</td><td>Prospective Fresh Prospective Frozen</td><td>6/6</td><td>100</td><td>54.1-100</td><td>813/815</td><td>99.8</td><td>99.1-100</td></tr><tr><td></td><td>6/6</td><td>100</td><td>54.1-100</td><td>741/741</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>32/32</td><td>100</td><td>89.1-100</td><td>387/387</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>49/49</td><td>100</td><td>92.7-100</td><td>2156/2158</td><td>99.9</td><td>99.7-100</td></tr><tr><td rowspan="5">Candida krusei</td><td>Prospective Fresh</td><td>2/2</td><td>100</td><td>15.8-100</td><td>819/819</td><td>100</td><td>99.6-100</td></tr><tr><td>Prospective Frozen</td><td>2/2</td><td>100</td><td>15.8-100</td><td>745/745</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>28/28</td><td>100</td><td>87.7-100</td><td>391/391</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>37/37</td><td>100</td><td>90.5-100</td><td>2170/2170</td><td>100</td><td>99.8-100</td></tr><tr><td rowspan="5">Candida parapsilosis</td><td>Prospective Fresh</td><td>3/3</td><td>100</td><td>29.2-100</td><td>818/818</td><td>100</td><td>99.6-100</td></tr><tr><td>Prospective Frozen</td><td>4/4</td><td>100</td><td>39.8-100</td><td>742/743</td><td>99.9</td><td>99.3-100</td></tr><tr><td>Seeded Fresh</td><td>47/49</td><td>95.9</td><td>86.0-99.5</td><td>370/370</td><td>100</td><td>99.0-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>214/215</td><td>99.5</td><td>97.4-100</td></tr><tr><td>Overall</td><td>59/61 b</td><td>96.7</td><td>88.7-99.6</td><td>2144/2146</td><td>99.9</td><td>99.7-100</td></tr><tr><td rowspan="5">Candida tropicalis</td><td>Prospective Fresh</td><td>0/0</td><td>-</td><td>-</td><td>821/821</td><td>100</td><td>99.6-100</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100</td><td>29.2-100</td><td>744/744</td><td>100</td><td>99.5-100</td></tr><tr><td>Seeded Fresh</td><td>31/31</td><td>100</td><td>88.8-100</td><td>388/388</td><td>100</td><td>99.1-100</td></tr><tr><td>Seeded Frozen</td><td>5/5</td><td>100</td><td>47.8-100</td><td>215/215</td><td>100</td><td>98.3-100</td></tr><tr><td>Overall</td><td>39/39</td><td>100</td><td>91.0-100</td><td>2168/2168</td><td>100</td><td>99.8-100</td></tr></table>

a Sensitivity and Specificity refer to performance with the prospective specimens only; Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) refer to performance with the seeded specimens. b Bidirectional sequencing identified the isolates from the two false negative C. parapsilosis specimens as being the closely related species C. metapsilosis. This misidentification is a known limitation of phenotypic identification methods.

Comparator testing for the antimicrobial resistance genes was performed with both the blood culture sample and with isolates recovered after subculture of the blood culture media. The results are presented in the tables below. The NPA for mecA and vanA/B are lower when comparing to PCR/sequencing from bacterial isolates than to PCR/sequencing directly from blood culture primarily due to the reference culture methods not isolating a resistant clone of an applicable organism. This may be due to heterogeneous resistance within a population of cultured organisms or co-culturing of multiple indistinguishable applicable organisms with different resistance profiles (e.g., mixed culture of a resistant Staphylococcus with sensitive Staphylococcus).

FilmArray BCID Clinical Performance Summary – Antimicrobial Resistance Genes (Comparator Method: PCR/Sequencing Direct from Blood Culture).   

<table><tr><td colspan="2" rowspan="2">Antimicrobial Resistance Genes</td><td colspan="3">Sensitivity /PPA a</td><td colspan="3">Specificity /NPA a</td></tr><tr><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td colspan="8">mecA - Methicillin Resistance Gene</td></tr><tr><td rowspan="5">mecA All Staphylococcus Detected</td><td>Prospective Fresh</td><td>253/257</td><td>98.4%</td><td>96.1-99.6%</td><td>147/150</td><td>98.0%</td><td>94.3-99.6%</td></tr><tr><td>Prospective Frozen</td><td>233/237</td><td>98.3%</td><td>95.7-99.5%</td><td>134/136</td><td>98.5%</td><td>94.8-99.8%</td></tr><tr><td>Seeded Fresh</td><td>1/1</td><td>100%</td><td>n/a</td><td>0/0</td><td></td><td></td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100%</td><td>n/a</td><td>0/0</td><td>'</td><td></td></tr><tr><td>Overall</td><td>488/496</td><td>98.4%</td><td>96.8-99.3%</td><td>281/286</td><td>98.3%</td><td>96.0-99.4%</td></tr><tr><td rowspan="2">mecA Staphylococcus</td><td>Prospective Fresh</td><td>67/69</td><td>97.1%</td><td>89.9-99.6%</td><td>64/64</td><td>100%</td><td>94.4-100%</td></tr><tr><td>Prospective Frozen</td><td>70/70</td><td>100%</td><td>94.9-100%</td><td>54/54</td><td>100%</td><td>93.4-100%</td></tr><tr><td colspan="2">Antimicrobial Resistance Genes</td><td colspan="3">Sensitivity /PPA a</td><td colspan="3">Specificity /NPA a</td></tr><tr><td rowspan="3">Detected; S. aureus Detected</td><td>Seeded Fresh</td><td>TP/TP + FN</td><td>%</td><td>95% CI</td><td>TN/TN + FP</td><td>%</td><td>95% CI</td></tr><tr><td>Seeded Frozen</td><td>0/0 0/0</td><td>-</td><td>-</td><td>0/0</td><td>-</td><td>-</td></tr><tr><td>Overall</td><td>137/139</td><td>- 98.6%</td><td>94.9-99.8%</td><td>0/0 118/118</td><td>- 100%</td><td>- 96.9-100%</td></tr><tr><td>mecA</td><td>Prospective Fresh</td><td>186/188</td><td>98.9%</td><td>96.2-99.9%</td><td>83/86</td><td>96.5%</td><td>90.1-99.3%</td></tr><tr><td>Staphylococcus</td><td>Prospective Frozen</td><td>163/167</td><td>97.6%</td><td>94.0-99.3%</td><td>80/82</td><td>97.6%</td><td>91.5-99.7%</td></tr><tr><td>Detected;</td><td>Seeded Fresh</td><td>1/1</td><td>100%</td><td>n/a</td><td>0/0</td><td>-</td><td>-</td></tr><tr><td>S. aureus</td><td>Seeded Frozen</td><td>1/1</td><td>100%</td><td>n/a</td><td>0/0</td><td>-</td><td></td></tr><tr><td>Not Detected</td><td>Overall</td><td>351/357</td><td>98.3%</td><td>96.4-99.4%</td><td>163/168</td><td>97.0%</td><td>93.2-99.0%</td></tr><tr><td colspan="6">vanA/B - Vancomycin Resistance Genes</td><td></td><td></td></tr><tr><td rowspan="5">vanA/B Enterococcus Detected</td><td>Prospective Fresh</td><td>23/23</td><td>100%</td><td>85.2-100%</td><td>36/36</td><td>100%</td><td>90.3-100%</td></tr><tr><td>Prospective Frozen</td><td>13/13</td><td>100%</td><td>75.3-100%</td><td>30/30</td><td>100%</td><td>88.4-100%</td></tr><tr><td>Seeded Fresh</td><td>12/12</td><td>100%</td><td>73.5-100%</td><td>0/0</td><td>-</td><td>-</td></tr><tr><td>Seeded Frozen</td><td>16/16</td><td>100%</td><td>79.4-100%</td><td>1/1</td><td>100%</td><td>n/a</td></tr><tr><td>Overall</td><td>64/64</td><td>100%</td><td>94.4-100%</td><td>67/67</td><td>100%</td><td>94.6-100%</td></tr><tr><td colspan="6">KPC - Carbapenem Resistance Gene (Carbapenemase)</td><td></td><td></td></tr><tr><td rowspan="5">KPC Enterobacteriaceae and/or A. baumannii and/or P.</td><td>Prospective Fresh</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>177/177</td><td>100%</td><td>97.9-100%</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>187/187</td><td>100%</td><td>98.0-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10</td><td>100%</td><td>69.2-100%</td><td>105/105</td><td>100%</td><td>96.5-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100%</td><td>85.2-100%</td><td>89/89</td><td>100%</td><td>95.9-100%</td></tr><tr><td>Overall</td><td>39/39</td><td>100%</td><td>91.0-100%</td><td>558/558</td><td>100%</td><td>99.3-100%</td></tr><tr><td rowspan="5">KPC Enterobacteriaceae Detected</td><td>Prospective Fresh</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>150/150</td><td>100%</td><td>97.6-100%</td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>151/151</td><td>100%</td><td>97.6-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10</td><td>100%</td><td>69.2-100%</td><td>83/83</td><td>100%</td><td>95.7-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100%</td><td></td><td></td><td></td><td></td></tr><tr><td>Overall</td><td>39/39</td><td></td><td>85.2-100%</td><td>71/71</td><td>100%</td><td>94.9-100%</td></tr><tr><td rowspan="5">KPC Enterobacteriaceae Not Detected; A. baumannii</td><td>Prospective Fresh</td><td>0/0</td><td>100%</td><td>91.0-100%</td><td>455/455</td><td>100%</td><td>99.2-100%</td></tr><tr><td></td><td></td><td>-</td><td>-</td><td>27/27</td><td>100%</td><td>87.4-100%</td></tr><tr><td>Prospective Frozen Seeded Fresh</td><td>0/0</td><td></td><td></td><td>36/36</td><td>100%</td><td>90.3-100%</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td></td><td></td><td>22/22</td><td>100%</td><td>84.6-100%</td></tr><tr><td></td><td>0/0</td><td></td><td></td><td>18/18</td><td>100%</td><td>81.5-100%</td></tr><tr><td>and/or P. aeruginosa Detected</td><td>Overall</td><td>0/0</td><td></td><td></td><td>103/103</td><td>100%</td><td>96.5-100%</td></tr></table>

a Sensitivity and Specificity refer to performance with the prospective specimens only; Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) refer to performance with the seeded specimens.

FilmArray BCID Clinical Performance Summary – Antimicrobial Resistance Genes (Comparator Method: PCR/Sequencing of Cultured Isolates).   

<table><tr><td colspan="2">ANTIMICROBIAL RESISTANCE GENES</td><td colspan="3">Positive Percent Agreement 95% CI</td><td colspan="3">Negative Percent Agreementa % 95% CI</td></tr><tr><td colspan="2"></td><td colspan="3">TP/TP + FN % mecA - Methicillin Resistance Gene</td><td colspan="3">TN/TN + FP</td></tr><tr><td rowspan="5">mecA All Staphylococcus</td><td>Prospective Fresh</td><td>234/236</td><td>99.2%</td><td>97.0-99.9%</td><td>149/172</td><td>86.7%</td><td>80.6-91.3%</td></tr><tr><td>Prospective Frozen</td><td>219/222</td><td>98.6%</td><td>96.1-99.7%</td><td>135/151</td><td>89.4%</td><td>83.4-93.8%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td>-</td><td>0/0</td><td>-</td><td></td></tr><tr><td>Seeded Frozen</td><td>1/1</td><td>100%</td><td>n/a</td><td>0/0</td><td></td><td>-</td></tr><tr><td>Overall</td><td>454/459</td><td>98.9%</td><td>97.5-99.6%</td><td>284/323</td><td>- 87.9%</td><td>-</td></tr><tr><td rowspan="2">mecA Staphylococcus</td><td>Prospective Fresh</td><td>64/65</td><td>98.5%</td><td>91.7-100%</td><td></td><td></td><td>83.9-91.3%</td></tr><tr><td>Prospective Frozen</td><td></td><td>100%</td><td>94.6-100%</td><td>65/68</td><td>95.6%</td><td>87.6-99.1%</td></tr><tr><td>Detected;</td><td>Seeded Fresh</td><td>66/66</td><td></td><td></td><td>54/58</td><td>93.1%</td><td>83.3-98.1%</td></tr><tr><td>S. aureus Detected</td><td>Seeded Frozen</td><td>0/0 0/0</td><td>- -</td><td>- -</td><td>0/0 0/0</td><td>- -</td><td>-</td></tr><tr><td colspan="2">ANTIMICROBIAL RESISTANCE GENES</td><td colspan="3">Positive Percent Agreement</td><td colspan="3">Negative Percent Agreementa</td></tr><tr><td></td><td>Overall</td><td>TP/TP + FN 130/131</td><td>% 99.2%</td><td>95% CI 95.8-100%</td><td>TN/TN + FP 119/126</td><td>% 94.4%</td><td>95% CI 88.9-97.7%</td></tr><tr><td rowspan="5">mecA Staphylococcus</td><td>Prospective Fresh</td><td>170/171</td><td>99.4%</td><td>96.8-100%</td><td>84/104</td><td>80.8%</td><td>71.9-87.8%</td></tr><tr><td>Prospective Frozen</td><td>153/156</td><td>98.1%</td><td>94.5-99.6%</td><td>81/93</td><td>87.1%</td><td>78.6-93.2%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td>-</td><td></td><td>0/0</td><td></td><td></td></tr><tr><td></td><td>1/1</td><td>100%</td><td>-</td><td></td><td>-</td><td></td></tr><tr><td>Seeded Frozen Overall</td><td>324/328</td><td>98.8%</td><td>n/a 96.9-99.7%</td><td>0/0 165/197</td><td>- 83.8%</td><td>- 77.9-88.6%</td></tr><tr><td colspan="8">vanA/B - Vancomycin Resistance Genes</td></tr><tr><td rowspan="5">vanA/B Enterococcus Detected</td><td>Prospective Fresh</td><td>20/20</td><td>100%</td><td>83.2-100%</td><td>36/39</td><td>92.3%</td><td>79.1-98.4%</td></tr><tr><td>Prospective Frozen</td><td>12/12</td><td>100%</td><td>73.5-100%</td><td>30/31</td><td>96.8%</td><td>83.3-99.9%</td></tr><tr><td>Seeded Fresh</td><td>12/12</td><td>100%</td><td>73.5-100%</td><td>0/0</td><td></td><td></td></tr><tr><td>Seeded Frozen</td><td>16/16</td><td>100%</td><td>79.4-100%</td><td>1/1</td><td>-</td><td>-</td></tr><tr><td>Overall</td><td>60/60</td><td>100%</td><td>94.0-100%</td><td>67/71</td><td>100% 94.4%</td><td>n/a 86.2-98.4%</td></tr><tr><td colspan="8">KPC - Carbapenem Resistance Gene (</td></tr><tr><td>KPC</td><td>Prospective Fresh</td><td>3/3</td><td>100%</td><td>(Carbapenemase) 29.2-100%</td><td>177/177</td><td>100%</td><td>97.9-100%</td></tr><tr><td>Enterobacteriaceae</td><td>Prospective Frozen</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>187/187</td><td>100%</td><td>98.1-100%</td></tr><tr><td>and/or</td><td>Seeded Fresh</td><td>10/10</td><td>100%</td><td>69.2-100%</td><td>105/105</td><td>100%</td><td>96.5-100%</td></tr><tr><td>A. baumannii and/or</td><td>Seeded Frozen</td><td>23/23</td><td>100%</td><td>85.2-100%</td><td>89/89</td><td>100%</td><td>95.9-100%</td></tr><tr><td>P. aeruginosa Detected</td><td>Overall</td><td>39/39</td><td>100%</td><td>91.0-100%</td><td>558/558</td><td>100%</td><td>99.3-100%</td></tr><tr><td rowspan="5">KPC Enterobacteriaceae Detected</td><td>Prospective Fresh</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>151/151</td><td></td><td></td></tr><tr><td>Prospective Frozen</td><td>3/3</td><td>100%</td><td>29.2-100%</td><td>152/152</td><td>100% 100%</td><td>97.6-100% 97.6-100%</td></tr><tr><td>Seeded Fresh</td><td>10/10</td><td>100%</td><td>69.2-100%</td><td>83/83</td><td>100%</td><td>95.7-100%</td></tr><tr><td>Seeded Frozen</td><td>23/23</td><td>100%</td><td>85.2-100%</td><td>71/71</td><td>100%</td><td>94.9-100%</td></tr><tr><td>Overall</td><td>39/39</td><td>100%</td><td>91.0-100%</td><td>457/457</td><td>100%</td><td>99.2-100%</td></tr><tr><td rowspan="4">KPC Enterobacteriaceae Not Detected;</td><td>Prospective Fresh</td><td>0/0</td><td>-</td><td></td><td>26/26</td><td>100%</td><td>86.8-100%</td></tr><tr><td>Prospective Frozen</td><td>0/0</td><td></td><td>-</td><td>35/35</td><td>100%</td><td>90.0-100%</td></tr><tr><td>Seeded Fresh</td><td>0/0</td><td></td><td></td><td>22/22</td><td>100%</td><td>84.6-100%</td></tr><tr><td>Seeded Frozen</td><td>0/0</td><td></td><td></td><td>18/18</td><td>100%</td><td></td></tr><tr><td>A. baumannii and/or P. aeruginosa Detected</td><td>Overall</td><td>0/0</td><td></td><td></td><td>101/101</td><td>100%</td><td>81.5-100% 96.4-100%</td></tr></table>

a Isolates for 12 Staphylococci, 4 Enterococci, and 7 Enterobacteriaceae/A. baumannii/P. aeruginosa did not grow from the subcultured blood culture and could therefore not be tested with the PCR/bi-directional sequencing comparator method. These blood cultures were considered negative for the antimicrobial resistance genes by comparator method, and FilmArray performance has been calculated as True Negative (when FilmArray is negative for the analyte) or False Positive (when FilmArray is positive for the analyte) for each of these isolates.

The performance of FilmArray BCID panel as compared to phenotypic antimicrobial susceptibility testing (AST) results was calculated for informational purposes. Results stratified by AST method are presented in the following three tables. Positive percent agreement is sometimes lower when comparing results from bacterial isolates than when comparing to PCR/sequencing directly from blood culture because phenotypic AST testing is capable of detecting antimicrobial resistance due to mechanisms other than acquisition of mecA, vanA/B, or KPC. AST results were not provided for all isolates.

mecA Performance – Comparison to Phenotypic Cefoxitin (Methicillin/Oxacillin Susceptibility) AST Methods.   

<table><tr><td colspan="2">PHENOTYPIC METHODS Cefoxitin Disc</td><td colspan="2">Positive Percent Agreement</td><td colspan="2">Negative Percent Agreement</td></tr><tr><td colspan="2"></td><td>TP/TP + FN 22/22</td><td>% (95%CI) 100%</td><td>TN/TN + FP 15/15</td><td>% (95%CI) 100%</td></tr><tr><td rowspan="3">Prospective All Staphylococcus</td><td>Diffusion Chromogenic Agar Automated</td><td>42/46</td><td>91.3%</td><td>25/32</td><td>78.1%</td></tr><tr><td>Antimicrobial Susceptibility</td><td>366/380</td><td>96.3%</td><td>226/262</td><td>86.3%</td></tr><tr><td>Testing All Methods</td><td>430/448</td><td>96.0% (93.7 - 97.6%)</td><td>266/309</td><td>86.1% (81.7 -</td></tr><tr><td rowspan="2">Prospective Staphylococcus, S. aureus Detected</td><td>Chromogenic Agar Automated Antimicrobial</td><td>10/11</td><td>90.9%</td><td>8/8</td><td>89.7%) 100%</td></tr><tr><td>Susceptibility Testing All Methods</td><td>117/119</td><td>98.3% 97.7%</td><td>108/112</td><td>96.4% 96.7%</td></tr><tr><td>Seeded Staphylococcus</td><td>Automated Antimicrobial Susceptibility Testing</td><td>127/130 1/1</td><td>(93.4 - 99.5%) 100%</td><td>116/120 0/0</td><td>(91.7 - 99.1%)</td></tr></table>

vanA/B Performance – Comparison to Phenotypic Vancomycin AST Methods.   

<table><tr><td colspan="2">PHENOTYPIC METHODS</td><td colspan="2">Positive Percent Agreement TP/TP + FN % (95%CI)</td><td colspan="2">Negative Percent Agreement TN/TN + FP % (95%CI)</td></tr><tr><td rowspan="4">Prospective Enterococcus</td><td>Vancomycin Screen Agar Vancomycin Disc</td><td>3/3</td><td>100%</td><td>5/5</td><td>100%</td></tr><tr><td>Diffusion Automated Antimicrobial</td><td>0/1</td><td>0.0%</td><td>-</td><td>-</td></tr><tr><td>Susceptibility Testing</td><td>29/30</td><td>96.7% 94.1%</td><td>55/58</td><td>94.8% 95.2%</td></tr><tr><td>All Methods</td><td>32/34 </td><td>(80.3- 99.3%)</td><td>60/63</td><td>(86.7- 99.0%)</td></tr><tr><td rowspan="3">Seeded Enterococcus</td><td>Vancomycin Disc Diffusion</td><td>14/14</td><td>100%</td><td>1/1</td><td>100%</td></tr><tr><td>Vancomycin Screen Agar</td><td>14/14</td><td>100% 100%</td><td>-</td><td></td></tr><tr><td>All Methods</td><td>28/28</td><td>(87.7 - 100%)</td><td>1/1</td><td>100% (n/a)</td></tr><tr><td>Combined Prospective and Seeded Enterococcus</td><td>All Methods</td><td>60/62 a</td><td>96.8% (88.8 - 99.6%)</td><td>61/64</td><td>95.3% (86.9 - 99.0%</td></tr></table>

a Two isolates (one $E$ . gallinarum and one $E .$ . faecalis) that were vancomycin resistant by phenotypic AST testing were negative for the vanA $/ B$ genes by bi-directional sequence analysis.

# KPC Performance – Comparison to Phenotypic Carbapenem AST Methods.

AST results were not provided for several isolates.   
· Acinetobacter baumannii and Pseudomonas aeruginosa are commonly resistant to carbapenems due to mechanisms other than acquisition of the KPC gene $( b l a _ { \mathrm { K P C } } )$ . These bacteria very rarely carry the KPC gene.

<table><tr><td colspan="2">PHENOTYPIC METHODS</td><td colspan="2">Positive Percent Agreement TP/ % (95%CI)</td><td colspan="2">Negative Percent Agreement TN/ % (95%CI)</td></tr><tr><td>Prospective A. baumannii</td><td>Automated Antimicrobial</td><td>TP + FN 0/10</td><td>0%</td><td>TN + FP 4/4</td><td>100%</td></tr><tr><td>Seeded A. baumannii</td><td>Susceptibility Testing Meropenem Disc Diffusion</td><td>0/30</td><td>0%</td><td>9/9</td><td>100%</td></tr><tr><td colspan="2">A. baumannii  All Methods</td><td>0/40</td><td>0% (n/a)</td><td>13/13</td><td>100% (75.3-100%)</td></tr><tr><td>Prospective P. aeruginosa</td><td>Automated Antimicrobial Susceptibility Testing Meropenem Disc Diffusion Meropenem/Ertapenem</td><td></td><td>0/10 0% - - 0/1 0%</td><td>32/32 6/6</td><td>100% 100%</td></tr><tr><td colspan="2">P. aeruginosa  All Methods</td><td>Disc Diffusion</td><td>0/11 0% (n/a)</td><td>2/2 40/40</td><td>100% 100% (91.2-100%)</td></tr><tr><td colspan="2">Prospective Automated K. pneumoniae</td><td>Antimicrobial Susceptibility Testing</td><td>6/6 100%</td><td>64/64</td><td>100%</td></tr><tr><td rowspan="2">Seeded K. pneumoniae</td><td>Meropenem Disc Diffusion Modified Hodge Test</td><td>19/19</td><td>100%</td><td>1/1</td><td>100%</td></tr><tr><td>(Meropenem) K. pneumoniae  All Methods</td><td>11/11</td><td>100% 100%</td><td>1/1</td><td>100% 100%</td></tr><tr><td colspan="2">Prospective Automated</td><td></td><td>36/36 (90.3-100%)</td><td>66/66</td><td>(94.6-100%)</td></tr><tr><td colspan="2">E. cloacae</td><td>Antimicrobial Susceptibility Testing Automated Antimicrobial</td><td></td><td>22/22</td><td>100%</td></tr><tr><td rowspan="3">Seeded E. cloacae</td><td>Susceptibility Testing Meropenem Disc Diffusion</td><td>- 0/1</td><td>- 0%</td><td>3/3 -</td><td>100% -</td></tr><tr><td>Modified Hodge Test (Meropenem)</td><td>2/2</td><td>100%</td><td>11/11</td><td>100% 100%</td></tr><tr><td>E. cloacae  All Methods Automated</td><td>2/3a</td><td>66.7% (9.4-99.2%)</td><td>36/36</td><td>(90.3-100%)</td></tr><tr><td>Prospective E. coli</td><td>Antimicrobial Susceptibility Testing</td><td></td><td>- -</td><td>144/144</td><td>100%</td></tr><tr><td>Seeded E. coli</td><td>Modified Hodge Test (Meropenem)</td><td>1/1</td><td>100%</td><td>4/4</td><td>100%</td></tr><tr><td colspan="2">PHENOTYPIC METHODS</td><td colspan="2">Positive Percent Agreement TP/</td><td colspan="2">Negative Percent Agreement TN/</td></tr><tr><td colspan="2">E. coli  All Methods</td><td>TP + FN 1/1</td><td>% (95%CI) 100% (n/a)</td><td>TN + FP 148/148</td><td>% (95%CI) 100%</td></tr><tr><td>Prospective P. mirabilis</td><td>Automated Antimicrobial</td><td>-</td><td></td><td>21/21</td><td>(97.5-100%) 100%</td></tr><tr><td>Seeded P. mirabilis</td><td>Susceptibility Testing Meropenem Disc Diffusion Modified Hodge Test</td><td>-</td><td>-</td><td>4/4</td><td>100%</td></tr><tr><td>P. mirabilis  All Methods</td><td>(Meropenem)</td><td>0/1 0/1a</td><td>0% 0% (n/a)</td><td>11/11 36/36</td><td>100% 100%</td></tr><tr><td colspan="2">Prospective Automated All Other</td><td></td><td></td><td></td><td>(90.3-100%)</td></tr><tr><td rowspan="3">Enterobacteriaceae Seeded All Other Enterobacteriaceae</td><td>Antimicrobial Susceptibility Testing Automated Antimicrobial</td><td></td><td></td><td>43/43 42/42</td><td>100% 100%</td></tr><tr><td>Susceptibility Testing Meropenem Disc Diffusion</td><td></td><td></td><td>13/13</td><td>100%</td></tr><tr><td>Modified Hodge Test (Meropenem)</td><td></td><td></td><td>61/61</td><td>100%</td></tr><tr><td colspan="2">All Other Enterobacteriaceae  All Methods</td><td></td><td></td><td>159/159</td><td>100% (97.7-100%)</td></tr></table>

a Two isolates (one $E$ . cloacae and one $P$ . mirabilis) that were carbapenem resistant by phenotypic AST testing were negative for the KPC gene by bi-directional sequence analysis.

The FilmArray BCID Panel reports genus or family level results for Enterococcus, Staphylococcus, Streptococcus, Enterobacteriaceae, and Proteus. Standard identification methods identified various genera/species within each of these groups during the clinical evaluation. Stratification of performance by species within the groups is presented below.

Stratification of Enterococcus Clinical Performance by Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td rowspan=2 colspan=1>Enterococcus species</td><td rowspan=1 colspan=2>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>E. avium</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. casseliflavus</td><td rowspan=1 colspan=1>1/2 (50%)</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. durans</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>55/56 (98.2%)</td><td rowspan=1 colspan=1>8/8 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. faecalis + E. faecium</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>36/37 (97.3%)</td><td rowspan=1 colspan=1>9/9 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. gallinarum</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus sp. (not speciated)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Enterococcus</td><td rowspan=1 colspan=1>98/101 (97.0%)95%CI = 91.6-99.4%</td><td rowspan=1 colspan=1>29/29 (100%)95%CI = 88.1-100%</td></tr></table>

Stratification of Staphylococcus Clinical Performance by Species (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td rowspan=2 colspan=1>Staphylococcus species</td><td rowspan=1 colspan=2>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>256/257 (99.6%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. auricularis</td><td rowspan=1 colspan=1>0/1 (0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. capitis</td><td rowspan=1 colspan=1>15/17 (88.2%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. capitis + S. epidermidis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. capitis + S. hominis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. capitis + S. lugdunensis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. carnosus</td><td rowspan=1 colspan=1>0/1 (0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. cohnii</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. cohnii + S. hominis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>200/201 (99.5%)</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis + S. hominis</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis + Staphylococcussp. (not speciated)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus</td><td rowspan=1 colspan=1>19/19 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus + S. hominis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>65/65 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. hominis + Staphylococcus sp.(not speciated)</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. intermedius</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. intermedius + Staphylococcussp. (not speciated)</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. lentus</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. saprophyticus</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. sciuri</td><td rowspan=1 colspan=1>0/1 (0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. simulans</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. warneri</td><td rowspan=1 colspan=1>4/5 (80%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus sp. (notspeciated)a</td><td rowspan=1 colspan=1>180/200 (90%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall Staphylococcus</td><td rowspan=1 colspan=1>769/797 (96.5%)95%CI = 95.0-97.7%</td><td rowspan=1 colspan=1>1/1 (100%)95%CI = n/a</td></tr></table>

a Of the 20 unspeciated staphylococci not detected by FilmArray BCID, 16 were identified as $S _ { \cdot }$ . pettenkoferi, 2 as $S _ { \star }$ epidermidis, 1 as $S _ { \cdot }$ . capitis, and 1 as $S .$ . caprae by 16S sequence analysis. The 180 unspeciated Staphylococci that were detected by FilmArray BCID were not sequenced.

Stratification of Streptococcus Clinical Performance by Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="1" rowspan="2">Streptococcus species</td><td colspan="2" rowspan="1">Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="3" rowspan="1">Group A (Pyogenic)</td></tr><tr><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1">7/7 (100%)</td><td colspan="1" rowspan="1">31/31 (100%)</td></tr><tr><td colspan="3" rowspan="1">Group B (Pyogenic)</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">18/18 (100%)</td><td colspan="1" rowspan="1">18/18 (100%)</td></tr><tr><td colspan="3" rowspan="1">Group C/G (Pyogenic)</td></tr><tr><td colspan="1" rowspan="1">S. canis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. equi/S. dysgalactiae</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus group C</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus group G</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Group D (Bovis Group)</td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">S. bovis</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. equinus</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Group F (Anginosus Group)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">4/4 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. intermedius</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. constellatus</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Group H (Mitis Group)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. gordonii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. mitis</td><td colspan="1" rowspan="1">8/9 (88.9%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. mitis + viridans streptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. mitis/S. oralis</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. mitis/S. oralis + viridansstreptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. oralis</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis + viridansstreptococci</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">12/12 (100%)</td></tr><tr><td colspan="1" rowspan="1">S. sanguinis</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salivarius Group</td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">S. salivarius</td><td colspan="1" rowspan="1">1/2 (50%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. salivarius + S. sanguinis group</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">S. vestibularis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridans streptococci(not further speciated)</td><td colspan="1" rowspan="1">40/43 (93.0%)</td><td colspan="1" rowspan="1">1/1 (100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sp. (not speciated)</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Overall Streptococcus</td><td colspan="1" rowspan="1">136/141 (96.5)95%CI = 91.9-98.8%</td><td colspan="1" rowspan="1">62/62 (100%)95%CI = 94.2-100%</td></tr></table>

Stratification of Enterobacteriaceae Clinical Performance by Genus/Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td colspan="1" rowspan="2">Enterobacteriaceaegenus/species</td><td colspan="2" rowspan="1">Positive Agreement</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii +Escherichia coli</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1/2 (50%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">2/2 (100%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes +Klebsiella oxytoca</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">19/19 (100%)</td><td colspan="1" rowspan="1">17/17 (100%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter gergoviae</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter sakasakii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter sp.</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">141/144 (98%)</td><td colspan="1" rowspan="1">5/5 (100%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli + Klebsiellapneumoniae</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli + Providenciastuartii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">58/58 (100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">67/68 (99%)</td><td colspan="1" rowspan="1">34/34 (100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae + Pantoeaagglomerans</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Leclercia adacarboxylata</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii + Proteusmirabilis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pantoea sp.</td><td colspan="1" rowspan="1">0/2 (0%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">21/21 (100%)</td><td colspan="1" rowspan="1">15/15 (100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella group B</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella group C</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella sp.</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhi</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">22/22 (100%)</td><td colspan="1" rowspan="1">54/55 (98%)</td></tr><tr><td colspan="1" rowspan="1">Overall Enterobacteriaceae</td><td colspan="1" rowspan="1">303/310 (97.7%)95%CI = 95.4-99.1%</td><td colspan="1" rowspan="1">187/188 (99.5%)95%CI = 97.1-100%</td></tr></table>

Stratification of Proteus Clinical Performance by Species. (Comparator Method: Standard Manual/Automated Microbiological/Biochemical Identification)   

<table><tr><td rowspan=2 colspan=1>Proteus species</td><td rowspan=1 colspan=2>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>22/22 (100%)</td><td rowspan=1 colspan=1>15/15 (100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Proteus</td><td rowspan=1 colspan=1>22/22 (100%)95%CI = 84.6-100%</td><td rowspan=1 colspan=1>17/17 (100%)95%CI = 80.5-100%</td></tr></table>

FilmArray BCID reported a total of 81 prospective specimens with discernible multiple organism detections $5 . 2 \%$ of all prospective specimens; 81/1568). The majority of multiple detections (74/81; $9 1 . 3 \% )$ ) contained two discernible organisms, while $6 . 2 \%$ (5/81) contained three discernible organisms, and $2 . 5 \%$ (2/81) contained four discernible organisms. The most prevalent multiple detection was Enterococcus with Staphylococcus (S. aureus not detected) ( $1 . 3 \%$ of all specimens; 20/1568). Out of the 81 polymicrobial specimens, 29 contained one or more analytes that had not been detected with the reference/comparator methods, i.e., discrepant result/false positive by FilmArray BCID.

<table><tr><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Il ay</td><td></td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detection Combinationsas Determined by FilmArray BCID</td><td colspan="1" rowspan="2">Soeee Tns</td><td colspan="1" rowspan="2">GscmnSeens</td><td colspan="1" rowspan="2">DiscrepantResult(s)(Organism Not Detected byReferenceMethod</td></tr><tr><td colspan="1" rowspan="1">Organism 1Results</td><td colspan="1" rowspan="1">Organism 2Results</td><td colspan="1" rowspan="1">Organism 3Results</td><td colspan="1" rowspan="1">Organism 4RResults</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacae complex,Enterobacteriaceae</td><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1">Klebsiellaoxytoca,Enterobacteriaceae</td><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae, E.coli, K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. albicans</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candidaparapsilosis</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. parapsilosis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Proteus,Enterobacteriaceae</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcusagalactiae,Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus,Enterobacteriaceae</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus,S. aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1">Streptococcusagalactiae,Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1">Streptococcuspneumoniae,Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus,S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus,S. aureus</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. albicans</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus,Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriace</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobacteriaceae, E. coli</td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detection Combinationsas Determined by FilmArray BCID</td><td colspan="1" rowspan="2">Soeee Cn</td><td colspan="1" rowspan="2">CscesSeens</td><td colspan="1" rowspan="2">DiscrepantResult(s)(Organism NotDetected byReferenceMethod</td></tr><tr><td colspan="1" rowspan="1">Organism 1Results</td><td colspan="1" rowspan="1">Organism 2Results</td><td colspan="1" rowspan="1">Organism 3esults</td><td colspan="1" rowspan="1">Organism 4esults</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">A. baumannii</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacae complex</td><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacaecomplex</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae,Enteric</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">E. coli,K. pneumoniae(2)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Proteus,Enterobacteriaceae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. glabrata</td></tr><tr><td colspan="1" rowspan="1">Proteus,Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Staphylococcus(3), Enterococcus(3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Enterobacteriaceae</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae,Enterobacteriaceae</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. glabrata</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. albicans</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detection Combinationsas Determined by FilmArray BCID</td><td colspan="1" rowspan="2">Soec Cns</td><td colspan="1" rowspan="2">CscemSees</td><td colspan="1" rowspan="2">DiscrepantResult(s)(Organism NotDetected byReferenceMethod)</td></tr><tr><td colspan="1" rowspan="1">Organism 1Results</td><td colspan="1" rowspan="1">Organism 2Results</td><td colspan="1" rowspan="1">Organism 3Results</td><td colspan="1" rowspan="1">Organism 4Results</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="4" rowspan="1">Total Specimens with Multiple Detections</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1"></td></tr></table>

The following table lists 86 additional specimens with multiple species identified by the reference method. Of these additional 86 mixed cultures, 16 had organisms positive by the reference culture method but not detected (false negative) by FilmArray BCID.

# Additional Specimens with Multiple Isolates Identified by Reference/Comparator Methods

ote: Organisms shaded gray are not targeted by FilmArray BCID (i.e., off-panel organisms).

This list does not include multiple detection combinations already represented in the previous table of multiple detections by Film Array BCID.

<table><tr><td colspan="4" rowspan="1">Distinct Multiple Detections by Reference/Comparator methods</td><td colspan="1" rowspan="2">Toe Cns</td><td colspan="1" rowspan="2">CscemtSeens</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganismsNot Detected byFilmArray BCID)</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">Pantoeaagglomerans</td><td colspan="1" rowspan="1">Pantoeaagglomerans</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">Flavobacteriumspecies</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus,Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Neisseria species</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi</td><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coryneformbacterium species</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Streptococcusoralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Streptococcusmitis/oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pantoea species</td><td colspan="1" rowspan="1">Staphylococcusintermedius</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobacteriaceae</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Streptococcusparasanguis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcusmitis/oralis</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detections by Reference/Comparator methods</td><td colspan="1" rowspan="2">oo Ct</td><td colspan="1" rowspan="2">CrcemSeecns</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganismsNot Detected byFilmArray BCID)</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">Abiotrophiadefectiva</td><td colspan="1" rowspan="1">Staphyloccocusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii (seq. =A.nosocomialis/calcoaceticus)</td><td colspan="1" rowspan="1">Acinetobacterbaumannii (seq. =A.nosocomialis/calcoaceticus)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Aerococcusviridans</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae,Enterobacteriaceae</td></tr><tr><td colspan="1" rowspan="1">Aerococcusspecies</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Bacillus pumilus</td><td colspan="1" rowspan="1">Pseudomonasfluorescens/putida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Brevundimonasdiminuta</td><td colspan="1" rowspan="1">Weeksella virosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candidaparapsilosis</td><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacterfreundii</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumjeikeium</td><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Micrococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumspecies</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Diphtheroids</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detections by Reference/Comparator methods</td><td colspan="1" rowspan="2">ooe Ce</td><td colspan="1" rowspan="2">CircemSeemns</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganismsNot Detected byFilmArray BCID)</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Pasteurellamultocida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli,Enterobacteriaceae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Pantoeaagglomerans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">Streptococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcusspecies</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria species</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rhodococcusspecies</td><td colspan="1" rowspan="1">Staphylococcuswarneri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcuscaprae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Staphylococcusapitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscohnii</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepiddermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Multiple Detections by Reference/Comparator methods</td><td colspan="1" rowspan="2">Toe Cte</td><td colspan="1" rowspan="2">cscecstSeens</td><td colspan="1" rowspan="2">Discrepant Result(s)(Targeted OrganismsNot Detected byFilmArray BCID)</td></tr><tr><td colspan="1" rowspan="1">Isolate 1</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">Isolate 3</td><td colspan="1" rowspan="1">Isolate 4</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">Streptococcussanguis group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1">Viridansstreptococci</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1"></td></tr></table>

The reference method detected 201 off-panel organism isolates (i.e., those not targeted by FilmArray BCID) from the 1568 prospective cultures. The majority of these isolates belong to groups of organisms commonly considered to be blood culture contaminants (49 Corynebacterium/Diphtheroids, 33 Bacillus sp., and 27 Micrococcus sp., among others). Occurrence of off-panel organisms in the prospective arm of the clinical evaluation is presented in the following table.

Occurrence of Off-Panel Organisms as Determined by Reference/Comparator Methods   

<table><tr><td colspan="1" rowspan="1">Off-Panel Organism</td><td colspan="1" rowspan="1">NumberIdentified</td><td colspan="1" rowspan="1">Off-Panel Organism</td><td colspan="1" rowspan="1">NumberIdentified</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia sp. or Granulicatella sp.(formerly nutritionally-deficientStreptococci)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Flavobacterium species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xylosoxidans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Fusarium species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter sp. (not A. baumannii)</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Actinomyces species</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Lactobacillus species</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Aerococcus species</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Micrococcus luteus/lylae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Micrococcus species</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Bacillus pumilus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Bacillus species</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Moraxella species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium species</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Mycobacterium fortuitum complex</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium ensei</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Mycobacterium species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Brevundimonas diminuta</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Neisseria species</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Brevundimonas vesicularis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Paenibacillus species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia complex</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Pasteurella multocida</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Pasteurella species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Capnocytophaga species</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Propionibacterium species</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Chryseobacterium meningosepticum(Elizabethkingia/Flavobacterium)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Pseudomonas fluorescens/putida</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Chryseobacterium indologenes</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Pseudomonas species</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Chryseomonas luteola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhizobium radiobacter</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rothia (Stomatococcus)mucilaginosa</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium mucifaciens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Sphingomonas mucosissima</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium species/Diphtheroids</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Weeksella virosa</td><td colspan="1" rowspan="1">1</td></tr></table>

External Control testing in the Clinical Study:

Six frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ control mixes were provided to the study sites for daily testing. Five control mixes consisted of pooled blood and blood culture media containing whole bacteria/yeast at levels expected at culture positivity (bacteria at approximately $1 0 ^ { 8 }$ $\mathrm { C F U / m L }$ and yeast at approximately $1 0 ^ { 6 } \mathrm { C F U / m L }$ ). Combined, the five mixes covered all panel analytes. A sixth mix was negative for all panel members and only contained pooled blood and blood culture media. The operators were required to complete a valid control mix run with correct results obtained on each day of fresh specimen testing. A total of 403 control mix runs were attempted; two runs did not complete and two runs had failed internal control(s). Of the remaining 399 control runs, 396 $( 9 9 . 2 \% )$ were successful while 3 $( 0 . 8 \% )$ did not return the correct organism results either due to the detection of an extra analyte and/or the failure to detect one or more analytes. For each day that a control run was invalid or produced incorrect results, another control run was performed and valid results were required prior to testing any study specimens.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range: In the prospective arm of the FilmArray BCID clinical study, 1568 eligible blood cultures were collected and tested at seven of eight study sites across the United States over eight months. The number and percentage of positive results as determined by FilmArray BCID, stratified by study site or age group, are presented in the following tables. Overall, FilmArray BCID detected at least one organism in $8 8 . 1 \%$ (1382/1568) of prospective positive blood cultures.

Expected Value (as Determined by FilmArray BCID) Summary by Study Site for the Prospective Arm of the Clinical Evaluation   

<table><tr><td colspan="1" rowspan="1">FilmArray BCIDResult</td><td colspan="1" rowspan="1">Site 1(n =94)</td><td colspan="1" rowspan="1">Site 2(n =611)</td><td colspan="1" rowspan="1">Site 4(n =225</td><td colspan="1" rowspan="1">Site 5(n =193</td><td colspan="1" rowspan="1">Site 6(n =122</td><td colspan="1" rowspan="1">Site 7(n =178</td><td colspan="1" rowspan="1">Site 8(n=$145</td><td colspan="1" rowspan="1">Total(n =1568</td></tr><tr><td colspan="9" rowspan="1">Gram-Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">4(4%)</td><td colspan="1" rowspan="1">32(5%)</td><td colspan="1" rowspan="1">17(8%)</td><td colspan="1" rowspan="1">15(8%)</td><td colspan="1" rowspan="1">6(5%)</td><td colspan="1" rowspan="1">16(9%)</td><td colspan="1" rowspan="1">12(8%)</td><td colspan="1" rowspan="1">102(7%)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">47(50%)</td><td colspan="1" rowspan="1">314(51%)</td><td colspan="1" rowspan="1">108(48%)</td><td colspan="1" rowspan="1">87(45%)</td><td colspan="1" rowspan="1">74(61%)</td><td colspan="1" rowspan="1">85(48%)</td><td colspan="1" rowspan="1">65(45%)</td><td colspan="1" rowspan="1">780(50%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">19(20%)</td><td colspan="1" rowspan="1">84(14%)</td><td colspan="1" rowspan="1">34(15%)</td><td colspan="1" rowspan="1">35(18%)</td><td colspan="1" rowspan="1">30(25%)</td><td colspan="1" rowspan="1">34(19%)</td><td colspan="1" rowspan="1">21(14%)</td><td colspan="1" rowspan="1">257(16%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">13(14%)</td><td colspan="1" rowspan="1">51(8%)</td><td colspan="1" rowspan="1">22(10%)</td><td colspan="1" rowspan="1">12(6%)</td><td colspan="1" rowspan="1">7(6%)</td><td colspan="1" rowspan="1">14(8%)</td><td colspan="1" rowspan="1">21(14%)</td><td colspan="1" rowspan="1">140(9%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">5(1%)</td><td colspan="1" rowspan="1">4(2%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">18(1%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">8(1%)</td><td colspan="1" rowspan="1">5(2%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">4(2%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">26(2%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2(2%)</td><td colspan="1" rowspan="1">3(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">8(1%)</td></tr><tr><td colspan="9" rowspan="1">Gram-Negative Bacteria</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">9(1%)</td><td colspan="1" rowspan="1">3(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">16(1%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">14(15%)</td><td colspan="1" rowspan="1">120(20%)</td><td colspan="1" rowspan="1">33(15%)</td><td colspan="1" rowspan="1">51(26%)</td><td colspan="1" rowspan="1">22(18%)</td><td colspan="1" rowspan="1">36(20%)</td><td colspan="1" rowspan="1">31(21%)</td><td colspan="1" rowspan="1">307(20%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">3(3%)</td><td colspan="1" rowspan="1">9(1%)</td><td colspan="1" rowspan="1">3(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">24(2%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">8(9%)</td><td colspan="1" rowspan="1">53(9%)</td><td colspan="1" rowspan="1">17(8%)</td><td colspan="1" rowspan="1">21(11%)</td><td colspan="1" rowspan="1">15(12%)</td><td colspan="1" rowspan="1">22(12%)</td><td colspan="1" rowspan="1">13(9%)</td><td colspan="1" rowspan="1">149(10%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">2(2%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">6(0%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">30(5%)</td><td colspan="1" rowspan="1">7(3%)</td><td colspan="1" rowspan="1">16(8%)</td><td colspan="1" rowspan="1">2(2%)</td><td colspan="1" rowspan="1">7(4%)</td><td colspan="1" rowspan="1">11(8%)</td><td colspan="1" rowspan="1">74(5%)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">14(2%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">22(1%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">8(1%)</td><td colspan="1" rowspan="1">4(2%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">4(3%)</td><td colspan="1" rowspan="1">22(1%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3(3%)</td><td colspan="1" rowspan="1">2(0%)</td><td colspan="1" rowspan="1">1(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">8(1%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(0%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3(3%)</td><td colspan="1" rowspan="1">19(3%)</td><td colspan="1" rowspan="1">8(4%)</td><td colspan="1" rowspan="1">11(6%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">6(3%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">52(3%)</td></tr><tr><td colspan="9" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="1">FilmArray BCIDResult</td><td colspan="1" rowspan="1">Site 1(n =94)</td><td colspan="1" rowspan="1">Site 2(n =611)</td><td colspan="1" rowspan="1">Site 4(n =225)</td><td colspan="1" rowspan="1">Site 5(n =193)</td><td colspan="1" rowspan="1">Site 6(n =122)</td><td colspan="1" rowspan="1">Site 7(n =178)</td><td colspan="1" rowspan="1">Site 8(n =145)</td><td colspan="1" rowspan="1">Total(n =1568)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">7(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">3(2%)</td><td colspan="1" rowspan="1">20(1%)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(0%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">7(4%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">14(1%)</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">4(0%)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">5(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">8(1%)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">3(0%)</td></tr><tr><td colspan="9" rowspan="1">Antimicrobial Resistance Genes</td></tr><tr><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">28(30%)</td><td colspan="1" rowspan="1">201(33%)</td><td colspan="1" rowspan="1">70(31%)</td><td colspan="1" rowspan="1">56(29%)</td><td colspan="1" rowspan="1">43(35%)</td><td colspan="1" rowspan="1">56(32%)</td><td colspan="1" rowspan="1">37(26%)</td><td colspan="1" rowspan="1">491(31%)</td></tr><tr><td colspan="1" rowspan="1">vanA/B</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">13(2%)</td><td colspan="1" rowspan="1">8(4%)</td><td colspan="1" rowspan="1">4(2%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">5(3%)</td><td colspan="1" rowspan="1">6(4%)</td><td colspan="1" rowspan="1">36(2%)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(&lt;1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">1(1%)</td><td colspan="1" rowspan="1">2(1%)</td><td colspan="1" rowspan="1">6(&lt;1%)</td></tr></table>

Expected Value (as Determined by FilmArray BCID) Summary by Age Group for the Prospective Arm of the Clinical Evaluation   

<table><tr><td colspan="1" rowspan="1">FilmArray BCIDResult</td><td colspan="1" rowspan="1">&lt;1(n = 57)</td><td colspan="1" rowspan="1">1-17(n = 92)</td><td colspan="1" rowspan="1">18-44(n = 281)</td><td colspan="1" rowspan="1">45-64(n = 583)</td><td colspan="1" rowspan="1">65-84(n = 442)</td><td colspan="1" rowspan="1">85+(n = 113)</td></tr><tr><td colspan="7" rowspan="1">Gram-Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">1 (2%)</td><td colspan="1" rowspan="1">4 (4%)</td><td colspan="1" rowspan="1">17 (6%)</td><td colspan="1" rowspan="1">42 (7%)</td><td colspan="1" rowspan="1">30 (7%)</td><td colspan="1" rowspan="1">8 (7%)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">34 (60%)</td><td colspan="1" rowspan="1">40 (43%)</td><td colspan="1" rowspan="1">141 (50%)</td><td colspan="1" rowspan="1">304 (52%)</td><td colspan="1" rowspan="1">201 (45%)</td><td colspan="1" rowspan="1">60 (53%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">7(12%)</td><td colspan="1" rowspan="1">18 (20%)</td><td colspan="1" rowspan="1">43 (15%)</td><td colspan="1" rowspan="1">110 (19%)</td><td colspan="1" rowspan="1">62 (14%)</td><td colspan="1" rowspan="1">17 (15%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">8 (14%)</td><td colspan="1" rowspan="1">13 (14%)</td><td colspan="1" rowspan="1">33 (12%)</td><td colspan="1" rowspan="1">44 (8%)</td><td colspan="1" rowspan="1">31 (7%)</td><td colspan="1" rowspan="1">11 (10%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae(GBS)</td><td colspan="1" rowspan="1">2 (4%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">8 (1%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">2 (2%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">3 (3%)</td><td colspan="1" rowspan="1">5 (2%)</td><td colspan="1" rowspan="1">9 (2%)</td><td colspan="1" rowspan="1">5 (1%)</td><td colspan="1" rowspan="1">4 (4%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes (GAS)</td><td colspan="1" rowspan="1">1 (2%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">2 (0%)</td><td colspan="1" rowspan="1">2 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="7" rowspan="1">Gram-Negative Bacteria</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">6 (1%)</td><td colspan="1" rowspan="1">6 (1%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">13 (23%)</td><td colspan="1" rowspan="1">14 (15%)</td><td colspan="1" rowspan="1">50 (18%)</td><td colspan="1" rowspan="1">112 (19%)</td><td colspan="1" rowspan="1">102 (23%)</td><td colspan="1" rowspan="1">16 (14%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">2 (4%)</td><td colspan="1" rowspan="1">2 (2%)</td><td colspan="1" rowspan="1">6 (2%)</td><td colspan="1" rowspan="1">8 (1%)</td><td colspan="1" rowspan="1">6 (1%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">10 (18%)</td><td colspan="1" rowspan="1">6 (7%)</td><td colspan="1" rowspan="1">25 (9%)</td><td colspan="1" rowspan="1">50 (9%)</td><td colspan="1" rowspan="1">48 (11%)</td><td colspan="1" rowspan="1">10 (9%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">1 (0%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">1 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">5 (5%)</td><td colspan="1" rowspan="1">11 (4%)</td><td colspan="1" rowspan="1">32 (5%)</td><td colspan="1" rowspan="1">23 (5%)</td><td colspan="1" rowspan="1">3 (3%)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">9 (2%)</td><td colspan="1" rowspan="1">7 (2%)</td><td colspan="1" rowspan="1">4(4%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">9 (2%)</td><td colspan="1" rowspan="1">9 (2%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1 (2%)</td><td colspan="1" rowspan="1">2 (2%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (0%)</td><td colspan="1" rowspan="1">2 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">4 (4%)</td><td colspan="1" rowspan="1">9 (3%)</td><td colspan="1" rowspan="1">15 (3%)</td><td colspan="1" rowspan="1">18 (4%)</td><td colspan="1" rowspan="1">6 (5%)</td></tr><tr><td colspan="7" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">11 (2%)</td><td colspan="1" rowspan="1">1 (0%)</td><td colspan="1" rowspan="1">4 (4%)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">7 (1%)</td><td colspan="1" rowspan="1">4 (1%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (0%)</td><td colspan="1" rowspan="1">1 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">3 (1%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (1%)</td><td colspan="1" rowspan="1">1 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="7" rowspan="1">Antimicrobial Resistance Genes</td></tr><tr><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">24 (42%)</td><td colspan="1" rowspan="1">22 (24%)</td><td colspan="1" rowspan="1">97 (35%)</td><td colspan="1" rowspan="1">175 (30%)</td><td colspan="1" rowspan="1">133 (30%)</td><td colspan="1" rowspan="1">40 (35%)</td></tr><tr><td colspan="1" rowspan="1">vanA/B</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">9 (3%)</td><td colspan="1" rowspan="1">14 (2%)</td><td colspan="1" rowspan="1">10 (2%)</td><td colspan="1" rowspan="1">3 (3%)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">1 (&lt;1%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">5(1%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr></table>

# N. Instrument Name:

FilmArray® Instrument

# O. System Descriptions:

1. Modes of Operation: See Device Description (Section I) above

2. Software

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _X___ or No

3. Specimen Identification:

The Sample ID can be entered manually or scanned in by using the FilmArray barcode scanner.

4. Specimen Sampling and Handling:

N/A

5. Calibration:

N/A

6. Quality Control: See Section M (1c) above

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.