# APR 0 2 2013

March 28, 2013

# BD Diagnostics BD MAX™ Cdiff Assay

# Submitted by:

GeneOhm Sciences Canada Inc. (BD Diagnostics)   
2555 Boul. Parc-Technologique   
Quebec (Quebec), Canada   
G1P 4S5

# Contact:

Patricia Dionne, Ph.D.

# Device:

510(k) Number:

K130470

BD MAX™ Cdiff Assay

Clostridium difficile tcdB detection assay

Type of Test:

Qualitative Nucleic Acid Amplification Test for C. difficile toxin B gene from liquid or soft stool specimens

# Classification:

II   
Clostridium difficile Toxin Gene Amplification Assay. 866.3130   
OZN, 00I   
Microbiology (83)   
BD GeneOhm™ Ciff Assay

# Panel:

Predicate Device:

Predicate 510(k) number: K081920

# Intended Use:

The BD MAX™ Cdiff Assay performed on the BD MAX™ System is an automated in vitro diagnostic rhedqualitativedetectin eClostrid ifficietoxi giumanu or soft stool specimens from patients suspected of having C. difficile infection (CDl). The test, performed directly on the specimen, utilizes real-time polymerase chain reaction (PCR) for the liiationfici xan recizatin the detection f the amplified DNA. The BD MAX™ Cdf Assay is intended to aid in the diagnosis o CDI.

# Indication for Use:

The BD MAX™ Ci Assay peformed on the BD MAX™System is anautated in vi diasc alCo or soft stool specimens from patients suspected of having C. difficile infection (CDl). The test,

# BD MAX ™ Cdiff Assay PreMarket Notification

performed directly on the specimen, utilizes real-time polymerase chain reaction (PR) for the amplification of C. difficile toxin B gene DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD MAX™ Cdiff Assay is intended to aid in the diagnosis of CDI.

Special Conditions for Use Statement:

For prescription use

Special Instrument Requirements:

The BD MAX™ System

# Device Description:

The BD MAX™ System and the BD MAX™M Cdiff Assay are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes. The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances.The BD MAX™ System software automatically interprets test results. A test result may be called as NEG (negative), POS (positive) or UNR (unresolved) based on the amplification status of the target and of the Sample Processing Control. IND (indeterminate) or INC (incomplete) results are due to BD MAX™ System failure.

# Test Principle:

The BD MAX™ CdiffAssay performed on the BD MAX™ System is an automated in vitro diagnostic o he irequalitativedetectin Clostrid difficile toxi giuan or soft stool specimens from patients suspected of having C. difficile infection (CDl).

A liquid or soft stool specimen is collected and transported to the laboratory. For testng,a disposable 10uL inoculating loop is dipped into the stool material and the contents dispersed into a BD MAX™ Cdiff Sample Buffer Tube. The Sample Buffer Tube is closed with a septum cap and voeed. A worklist is creatd and he Sample Bufr Tube, the BD MAX Ciff nitizd rent strip and the BD MAX ™ PCR Cartridge are loaded onto the BD MAX ™ System.

Following enzymatic cel lysis, the released nucleic acids are captured on magnetic beads. The beads, with the bound nucleic acids, are washed using Wash Buffer and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized using Neutralization Buffer and transferred to a Master Mix to rehydrate PCR reagents. After reconstitution, the BD MAX ™ System dispenses a fixed volume of PCR-ready solution containing extracted nucleic acids into the BD MAX PCR Cartridge. Microvalves in he BD MAX™ PCR Carridge are sealed by the system prior to initiating PCR to contain the amplification mixture, thus preventing evaporation and contamination.

The amplified DNA targets are detected using hydrolysis TaqMan®) probes, labeled at one nd labeled with different fluorophores are used to detect tcdB and SPC amplicons in two different

# BD MAX ™ Cdiff Assay PreMarket Notification

optical channels of the BD MAX™m System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ^ { \prime } { - } 3 ^ { \prime }$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the two optical channels used for the BD MAX™ Cdiff Assay is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX™ System monitors these signals at each cycle of the PCR and interprets the data at the end of the program to provide a final result.

# Substantial Equivalence:

Table 1 shows the similarities and differences between the BD MAX™M Cdiff Assay and the predicate device.

Table 1: Substantial Equivalence Information   

<table><tr><td rowspan=1 colspan=1>.Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>BD MAX™ Cdiff Assay</td><td rowspan=1 colspan=1>BD GeneOhm ™ Cdiff Assay (K081920)</td></tr><tr><td rowspan=1 colspan=2>SIMILARITIES</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The BD MAX™ Cdiff Assay performedon the BD MAX ™ System is anautomated in vitro diagnostic test forthe direct, qualitative detection of theClostridium difficile toxin B gene (tcdB)in human liquid or soft stool specimensfrom patients suspected of having C.difficile infection (CDI). The test,performed directly on the specimen,utilizes real-time polymerase chainreaction (PCR) for the amplification ofC. difficile toxin B gene DNA andfluorogenic target-specifichybridization probes for the detectionof the amplified DNA. The BD MAX™Cdiff Assay is intended to aid in thediagnosis of CDI.</td><td rowspan=1 colspan=1>The BD GeneOhm ™ Cdiff Assay is arapid in vitro diagnostic test for thedirect, qualitative detection of C.difficile toxin B gene (tcdB) in humanliquid or soft stool specimens frompatients suspected of havingClostridium difficile-associated disease(CDAD). The test, based on real-timePCR, is intended for use as an aid indiagnosis of CDAD. The test isperformed directly on the specimen,utilizing polymerase chain reaction(PCR) for the amplification of specifictargets and fluorogenic target-specifichybridization probes for the detectionof the amplified DNA.</td></tr><tr><td rowspan=1 colspan=1>Mode ofdetection fortoxin B</td><td rowspan=1 colspan=1>Presence of the toxin B (tcdB) gene</td><td rowspan=1 colspan=1>Presence of the toxin B (tcdB) gene</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Liquid and soft stool specimens</td><td rowspan=1 colspan=1>Liquid and soft stool specimens</td></tr><tr><td rowspan=1 colspan=1>Assay Format</td><td rowspan=1 colspan=1>Amplification: PCRDetection: Fluorogenic target-specifichybridization</td><td rowspan=1 colspan=1>Amplification: PCRDetection: Fluorogenic target-specifichybridization</td></tr><tr><td rowspan=1 colspan=1>Interpretationof testresults</td><td rowspan=1 colspan=1>Automated (Diagnostic software of BDMAX™ system)</td><td rowspan=1 colspan=1>Automated (Diagnostic software ofSmartCycler® system)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>DIFFERENCES</td></tr><tr><td rowspan=1 colspan=1>AnalysisPlatform</td><td rowspan=1 colspan=1>BD MAX™ System</td><td rowspan=1 colspan=1>SmartCycler® System</td></tr><tr><td rowspan=1 colspan=1>PCR SamplePreparation</td><td rowspan=1 colspan=1>Automated by the BD MAX™ System</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>DetectionProbes</td><td rowspan=1 colspan=1>TaqMan® Probe</td><td rowspan=1 colspan=1>Molecular Beacon Probe</td></tr></table>

# BD MAX™ Cdiff Assay PreMarket Notification

<table><tr><td>Assay Controls</td><td>Specimen Processing Control (SPC)</td><td>Positive (DNA from Clostridium difficile bearing the tcdB gene ATCC 43255) Negative (DNA from Escherichia coli ATCC 25922) Internal procedural control</td></tr></table>

# Analytical Performance:

# Precision

Within-laboratory precision was evaluated for the BD MAX™ Cdiff Assay at one (1) site. The Precision panel consisted of 5 sample categories in simulated sample matrix as follows:

Moderate Positive (MP): $2 - 5 x$ LoD   
Low Positive (LP): 1- 2 x LoD   
High Negative 1:10 (HN1:10): 10-fold dilution of 1 x LoD   
High Negative 1:100 (HN1:100): 100-fold dilution of 1 x LoD   
True Negative (Neg)

Testing was performed in duplicate, over 12 days, with 2 runs per day, by 2 technologists. Precision study results for Neg, LP and MP samples demonstrated $100 \%$ agreement. Precision study results for HN1:100 and HN1:10 demonstrated agreement of $9 5 . 8 \%$ and $5 8 . 3 \%$ , respectively. Precision performance was acceptable for the LP, MP, and Neg sample categories. No specific acceptance criteria were defined for either of the high negative sample categories.

# Reproducibility

Reproducibility was evaluated with a panel that consisted of the same sample categories described f   syp  ay npla c wherein each day 2 panels were tested by 2 technologists, at 3 clinical sites using 1 lot of reagents Site-to-Site).One f these clinical sites participated in an extended study where  additional lots reagents were tested (Lot-to-Lot). Results are shown for each sample category.

ForSi-to-S eproducibiliy, he verall percent agreeent was $100 \%$ for MP, LP and Neg categories. The Site-to-Site Reproducibility results demonstrated $9 2 . 2 \%$ and $50 . 0 \%$ negative agreement for HN1:100 and HN1:10 categories, respectively (Table 2).

For Lot-to-Lot Reproducibility, the overall percent agreement was $100 \%$ for MP, LP and Neg categories. The Lot-to-Lot Reproducibility results demonstrated $9 6 . 7 \%$ and $6 4 . 4 \%$ negative agreement for HN1:100 and HN1:10 categories, respectively (Table 3).

S Lo-L el  aab   LPMP sl catos.oe aptane rita wi e h iv sample categories.

Lot-to-Lot and Site-to-Site quantitative reproducibility study results for positive and negative samples were stratified as follows (see Tables 2 to 5):

Within run • Between runs within day Between days within lot

# BD MAX ™ Ciff Assay PreMarket Notification

Between lots

<table><tr><td></td><td rowspan=2 colspan=3>EP</td><td rowspan=2 colspan=2>Within Run</td><td rowspan=2 colspan=2>Between Run within Day</td><td rowspan=2 colspan=2>Between Day within Lot</td><td rowspan=2 colspan=2>Between Lot</td><td rowspan=2 colspan=2>Overall</td><td></td></tr><tr><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>316.1</td><td rowspan=1 colspan=1>101.89</td><td rowspan=1 colspan=1>32.2%</td><td rowspan=1 colspan=1>53.21</td><td rowspan=1 colspan=1>16.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>114.95</td><td rowspan=1 colspan=1>36.4%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>748.6</td><td rowspan=1 colspan=1>167.05</td><td rowspan=1 colspan=1>22.3%</td><td rowspan=1 colspan=1>74.75</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>21.14</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>64.47</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>195.18</td><td rowspan=1 colspan=1>26.1%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>975.6</td><td rowspan=1 colspan=1>201.66</td><td rowspan=1 colspan=1>20.7%</td><td rowspan=1 colspan=1>47.85</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>.41.24</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>58.00</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>219.14</td><td rowspan=1 colspan=1>22.5%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td></td></tr><tr><td></td><td rowspan=1 colspan=2>SDPA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>2.8%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.4%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.5%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td></td><td rowspan=1 colspan=3>SDPH</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>37.3%</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>39.4%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>2.34</td><td rowspan=1 colspan=1>24.8%</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>9.4%</td><td rowspan=1 colspan=1>2.67</td><td rowspan=1 colspan=1>28.3%</td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>2.96</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>10.6%</td><td rowspan=1 colspan=1>3.38</td><td rowspan=1 colspan=1>26.3%</td><td></td></tr><tr><td rowspan=1 colspan=15>Vals shown are those btained or the C.difficile target in the samples that gave a positive esult</td></tr></table>

5   

<table><tr><td rowspan=1 colspan=1>′</td><td rowspan=1 colspan=3>EP</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>$D</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>5361.3</td><td rowspan=1 colspan=1>510.44</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>219.72</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>455.55</td><td rowspan=1 colspan=1>8.5%</td><td rowspan=1 colspan=1>718.58</td><td rowspan=1 colspan=1>13.4%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>5296.7</td><td rowspan=1 colspan=1>656.51</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>432.84</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>361.57</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>865.50</td><td rowspan=1 colspan=1>16.3%</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>5432.3</td><td rowspan=1 colspan=1>543.30</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>148.59</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>498.52</td><td rowspan=1 colspan=1>9.2%</td><td rowspan=1 colspan=1>752.18</td><td rowspan=1 colspan=1>13.8%</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>SDPA</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=3>SDPH</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>69.5</td><td rowspan=1 colspan=1>6.58</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4.10</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>8.00</td><td rowspan=1 colspan=1>11.5%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>68.9</td><td rowspan=1 colspan=1>7.72</td><td rowspan=1 colspan=1>11.2%</td><td rowspan=1 colspan=1>5.04</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.83</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>9.64</td><td rowspan=1 colspan=1>14.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69.5</td><td rowspan=1 colspan=1>6.56</td><td rowspan=1 colspan=1>9.4%</td><td rowspan=1 colspan=1>2.09</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4.52</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>8.23</td><td rowspan=1 colspan=1>11.8%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=3>EP</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>279.7</td><td rowspan=1 colspan=1>56.11</td><td rowspan=1 colspan=1>20.1%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>186.31</td><td rowspan=1 colspan=1>66.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>194.58</td><td rowspan=1 colspan=1>69.6%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>314.7</td><td rowspan=1 colspan=1>151.79</td><td rowspan=1 colspan=1>48.2%</td><td rowspan=1 colspan=1>67.47</td><td rowspan=1 colspan=1>21.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>12.53</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>166.58</td><td rowspan=1 colspan=1>52.9%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>948.8</td><td rowspan=1 colspan=1>149.65</td><td rowspan=1 colspan=1>15.8%</td><td rowspan=1 colspan=1>89.99</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>361.55</td><td rowspan=1 colspan=1>38.1%</td><td rowspan=1 colspan=1>401.51</td><td rowspan=1 colspan=1>42.3%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1217.4</td><td rowspan=1 colspan=1>224.30</td><td rowspan=1 colspan=1>18.4%</td><td rowspan=1 colspan=1>150.77</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>374.72</td><td rowspan=1 colspan=1>30.8%</td><td rowspan=1 colspan=1>462.02</td><td rowspan=1 colspan=1>37.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>SDPA</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD.</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>2.17</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>5.53</td><td rowspan=1 colspan=1>16.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5.94</td><td rowspan=1 colspan=1>17.8%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>SDPH         .</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.99</td><td rowspan=1 colspan=1>76.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.99</td><td rowspan=1 colspan=1>76.7%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>2.34</td><td rowspan=1 colspan=1>55.8%</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>23.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>60.4%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>2.37</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5.28</td><td rowspan=1 colspan=1>40.4%</td><td rowspan=1 colspan=1>5.94</td><td rowspan=1 colspan=1>45.5%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>3.83</td><td rowspan=1 colspan=1>21.8%</td><td rowspan=1 colspan=1>2.42</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5.50</td><td rowspan=1 colspan=1>31.3%</td><td rowspan=1 colspan=1>7.13</td><td rowspan=1 colspan=1>40.5%</td></tr><tr><td rowspan=1 colspan=3>Values shown are these obtained for</td><td rowspan=1 colspan=10>Valsshon ethose tined or the difc target in te samples that gav positi sult</td></tr></table>

<table><tr><td rowspan=1 colspan=3>EP</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>5020.4</td><td rowspan=1 colspan=1>631.46</td><td rowspan=1 colspan=1>12.6%</td><td rowspan=1 colspan=1>162.27</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>417.35</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>774.12</td><td rowspan=1 colspan=1>15.4%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>5006.5</td><td rowspan=1 colspan=1>524.23</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>250.03</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>421.67</td><td rowspan=1 colspan=1>8.4%</td><td rowspan=1 colspan=1>717.73</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>5166.0</td><td rowspan=1 colspan=1>554.66</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>204.88</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>543.70</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>803.26</td><td rowspan=1 colspan=1>15.5%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3>SDPA</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>SDPH</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run within Day</td><td rowspan=1 colspan=2>Between Day within Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN1:100</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>64.8</td><td rowspan=1 colspan=1>9.84</td><td rowspan=1 colspan=1>15.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4.65</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>10.89</td><td rowspan=1 colspan=1>16.8%</td></tr><tr><td rowspan=1 colspan=1>HN1:10</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>64.5</td><td rowspan=1 colspan=1>7.47</td><td rowspan=1 colspan=1>11.6%</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4.43</td><td rowspan=1 colspan=1>6.9%</td><td rowspan=1 colspan=1>8.94</td><td rowspan=1 colspan=1>13.9%</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>67.0</td><td rowspan=1 colspan=1>6.51</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>2.40</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>6.55</td><td rowspan=1 colspan=1>9.8%</td><td rowspan=1 colspan=1>9.54</td><td rowspan=1 colspan=1>14.2%</td></tr><tr><td rowspan=1 colspan=13>Calculated for the Specimen Processing Control of negative samples</td></tr></table>

# BD MAX ™ Cdiff Assay PreMarket Notification

Secn Derivativ Peak Abscissa DPA), an internal criteriaused to determine a fnal assay result, was selected as an additional means of assessing assay reproducibility. Overall mean SDPA values with variance components (SD and $\% 0 \%$ ) are shown in Tables 6 and 7.

ab -oi ebty Su Rsutsig e ot A   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>SITE</td><td rowspan=3 colspan=2>Overall PercentAgreement</td><td rowspan=2 colspan=3>SDPA Values1</td></tr><tr><td rowspan=2 colspan=2>Site 1PercentAgreement</td><td rowspan=2 colspan=2>Site 2PercentAgreement</td><td rowspan=2 colspan=2>Site 3PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg&#x27;</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>HN1:10012</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>25/30</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.2%</td><td rowspan=1 colspan=1>(95% CI: 84.8%, 96.2%)</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>HN1:1012</td><td rowspan=1 colspan=1>17/30</td><td rowspan=1 colspan=1>56.7%</td><td rowspan=1 colspan=1>8/30</td><td rowspan=1 colspan=1>26.7%</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>(95% CI: 39.9%, 60.1%)</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.6</td></tr></table>

difficile target.

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>LOT</td><td rowspan=3 colspan=2>Overall PercentAgreement</td><td rowspan=2 colspan=3>SDPA Values1</td></tr><tr><td rowspan=2 colspan=2>Lot 1PercentAgreement</td><td rowspan=2 colspan=2>Lot 2PercentAgreement</td><td rowspan=2 colspan=2>Lot 3PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg&#x27;</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>HN1:10012</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>(95% CI: 90.7%, 98.9%)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>HN1:1012</td><td rowspan=1 colspan=1>17/30</td><td rowspan=1 colspan=1>56.7%</td><td rowspan=1 colspan=1>21/30</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>64.4%</td><td rowspan=1 colspan=1>(95% C1: 54.2%, 73.6%)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95% CI: 95.9%, 100.0%)</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.5</td></tr></table>

difficile target.

# Sample Storage

Collected specimens can be stored at $2 . 2 5 ^ { \circ } \mathsf { C }$ for a maximum of 48 hours or at $2 . 8 \textdegree$ for a maximum of 120 hours (5 days) before testing. In case of repeat testing from the Sample Buffer Tube, the following storage conditions apply:

up to 5h after the end of the initial run when stored at $2 5 \pm 2 ^ { \circ } \mathsf C$ or up to 120h after the end of the initial run when stored at $2 . 8 ^ { \circ } \mathsf { C }$ .

# Controls

External Control materials ae ot provided b B. Various types  External Controls a recommended to allow the user to select the most appropriate for their laboratory quality control program:

Coercialy vailablecontrol ateals ATCc   Ciil anc  oeifc as positive and negative controls, respectively); • Suspensions bacterial strains characterized by theuser s toxigenicr non-toxigenc

# BD MAX™ Cdiff Assay PreMarket Notification

Previously characterized clinical specimens known to be positive or negative for toxigenic C. difficile.

The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances.

# Analytical Sensitivity

The analytical sensitivity (Limit of Detection or LoD) for the BD MAX™ Cdiff Assay was determined as follows: individual inoculating loops were dipped into C. difficile bacterial suspensions at different concentrations, prepared and quantified from cultures of four C. difficile strains representing three toxinotypes (0, III, Vill). Nine bacterial concentrations ranging between 0 and 64,000 CFU/mL were tested for each toxinotype. Each loop was then transferred to a SBT, already containing fecal matrix negative for toxigenic C. difficile. Each C. difficile strain was tested in replicates of 24 per concentration, by two different operators, using three different production lots of the BD MAX™ Cdiff Assay. The results were analyzed using a statistical linear logistic model. Briefly, the method models the positive response (expressed in percentage) as a function of Log (CFU/mL). The logistic model equation for the fitted curve allows the computation of the LoD by inverse prediction using the parameter estimates and their $9 5 \%$ 'confidence interval. The LoD was defined as the lowest concentration at which $9 5 \%$ of all replicates tested positive. The LoD ranged from 125 to 265 CFU per loop for each strain (Table 8).

Table 8: Limit of Detection of the BD MAX™ Cdiff Assay   

<table><tr><td rowspan=1 colspan=1>C. difficile Strain</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>LoD in CFU per loop</td></tr><tr><td rowspan=1 colspan=1>ATCC® 43255</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>265 (95% Cl: 140, 502)</td></tr><tr><td rowspan=1 colspan=1>ATCC® 9689</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>156 (95% CI: 82, 298)</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1805</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>205(95%Cl: 102, 412)</td></tr><tr><td rowspan=1 colspan=1>ATCC® 43598</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>125 (95% CI: 66, 235)</td></tr></table>

# Analytical Inclusivity

A variety of toxigenic C. difficile strains were included in this study taking into account geographic origin, toxinotype, NAP1 outbreaks and temporal diversity. Sixty-four (64) strains including 23 toxinotypes and representing 21 countries were tested, including strains from public collections and well-characterized clinical isolates (see Table 9). The assay correctly identified 62 of the 64 toxigenic C. diffcile strains which were tested at \~3xLoD. Two 2 strains produced low signal results and were false-negative in one out of five replicates.

Table 9: Strains Tested in the Inclusivity Study of the BD MAX™ Cdiff Assay.   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Toxinotype</td></tr><tr><td rowspan=1 colspan=1>ATCC 700792 (14797-2)</td><td rowspan=22 colspan=1>ND</td><td rowspan=1 colspan=1>DIl1</td><td rowspan=5 colspan=1>ND1</td><td rowspan=1 colspan=1>M30761</td><td rowspan=1 colspan=1>V1</td></tr><tr><td rowspan=1 colspan=1>ATCC 17858 (1253)</td><td rowspan=1 colspan=1>QI1</td><td rowspan=1 colspan=1>57267</td><td rowspan=1 colspan=1>VII</td></tr><tr><td rowspan=1 colspan=1>ATCC 51695 (BDMS 18AN)</td><td rowspan=1 colspan=1>MI23</td><td rowspan=1 colspan=1>ATCC 43598</td><td rowspan=1 colspan=1>VIII</td></tr><tr><td rowspan=1 colspan=1>ATCC 43600 (2149)</td><td rowspan=1 colspan=1>Ell1</td><td rowspan=1 colspan=1>51680</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>ATCC 43599 (2022)</td><td rowspan=1 colspan=1>PI2</td><td rowspan=1 colspan=1>M46722</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>ATCC 43596 (545)</td><td rowspan=1 colspan=1>M14614</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>M40767</td><td rowspan=1 colspan=1>IX</td></tr><tr><td rowspan=1 colspan=1>ATCC 43594 (W1194)</td><td rowspan=1 colspan=1>M39177</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8864</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>ATCC 17857 (870)</td><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>M26479</td><td rowspan=1 colspan=1>XII</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1382 (630)</td><td rowspan=1 colspan=1>ATCC 9689</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>IS25</td><td rowspan=1 colspan=1>$X1}</td></tr><tr><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>M16256</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>R9367</td><td rowspan=1 colspan=1>XIII</td></tr><tr><td rowspan=1 colspan=1>CD11</td><td rowspan=1 colspan=1>AC008</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>R10870</td><td rowspan=1 colspan=1>XIV</td></tr><tr><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>M14473</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>R9385</td><td rowspan=1 colspan=1>$xV{}$</td></tr><tr><td rowspan=1 colspan=1>. CD 246</td><td rowspan=1 colspan=1>M28681</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>SUC36</td><td rowspan=1 colspan=1>XVI</td></tr><tr><td rowspan=1 colspan=1>NCTC 11382</td><td rowspan=1 colspan=1>M37992</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>J9965</td><td rowspan=1 colspan=1>XVII</td></tr><tr><td rowspan=1 colspan=1>NCTC 11204</td><td rowspan=1 colspan=1>M41124</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>K095</td><td rowspan=1 colspan=1>XVIII</td></tr><tr><td rowspan=1 colspan=1>Ni1</td><td rowspan=1 colspan=1>M1137</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>TR13</td><td rowspan=1 colspan=1>XIX</td></tr><tr><td rowspan=1 colspan=1>CI3</td><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>CH6223</td><td rowspan=1 colspan=1>XXI</td></tr><tr><td rowspan=1 colspan=1>F19</td><td rowspan=1 colspan=1>SE844</td><td rowspan=1 colspan=1>Illa</td><td rowspan=1 colspan=1>M13883</td><td rowspan=1 colspan=1>XXI</td></tr><tr><td rowspan=1 colspan=1>LI6</td><td rowspan=1 colspan=1>55767</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>8785</td><td rowspan=1 colspan=1>XXIII</td></tr><tr><td rowspan=1 colspan=1>KI3</td><td rowspan=1 colspan=1>M25097</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>5978</td><td rowspan=1 colspan=1>XXIV</td></tr><tr><td rowspan=1 colspan=1>Gll1</td><td rowspan=1 colspan=1>SE881</td><td rowspan=1 colspan=1>V</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>HI1</td><td rowspan=1 colspan=1>51377</td><td rowspan=1 colspan=1>VI</td></tr></table>

${ \mathsf { N D } } =$ Not Determined. from SBT and in an additional test performed with 3 replicates. Overall, $4 / 5$ test results were found positive.

# Analytical Specificity

The BD MAX™ Cdiff Assay was performed on samples containing phylogenetically related species (Clostridium other than toxigenic C. difficile) and other organisms (bacteria, viruses) likely to be found in stool specimens (Table 10).

Six (6) out of six C. difficile strains not bearing the tcdB gene, tested at a concentration ≥ 1 X $\dot { 1 } 0 ^ { 8 }$ CFU/mL, produced negative results with the BD MAX™ Cdiff Assay; Thirty (30) out of 30 Clostridium strains other than C.difficile, including four strains of C. sordelli, tested at a concentration ≥ 1 X $1 0 ^ { 8 }$ CFU/mL, produced iv u  e  Ai Ninety-five (95) out of 98 other bacterial strains, including 93 species and subspecies, were tested at a concentration ≥ 1 X $1 0 ^ { 8 }$ CFU/mL $( 0 r \sim 1 \times 1 0 ^ { 8 }$ genomic DNA cp/mL or 1 X $1 0 ^ { 8 }$ elementary bodies/mL) and produced negative results with the BD MAX™" Cdiff Assay. The following strains gave a positive result:

Bacteroides stercoris (ATCC 43183) Gemella morbillorum (ATCC 27824)

# BD MAX ™ Cdiff Assay PreMarket Notification

# Peptoniphilus asaccharolyticus (ATCC 14963)

An investigation was conducted and determined that the false-positive results were due to contamination. Overall, out of 5 replicates, the three strains each gave an initial positive result due to contamination. ATCC 27824 gave negative results for the 4 other replicates. However, ATCC 43183 and ATCC 27824 gave another positive result due to instrument issue and the 3 last replicates were found negative.

Seven (7) out of seven viruses, tested at a concentration $\geq 1 \times 1 0 ^ { 5 }$ PFU/mL, produced negative results with the BD MAX™ Cdiff Assay.

Table 10: Species Tested for the Cross-Reactivity Study   

<table><tr><td rowspan=1 colspan=3>Clostridium species ( 1x10 CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Clostridium beijerinckii</td><td rowspan=1 colspan=1>Clostridium histolyticum</td><td rowspan=1 colspan=1>Clostridium septicum</td></tr><tr><td rowspan=1 colspan=1>Clostridium bifermentans</td><td rowspan=1 colspan=1>Clostridium innocuum</td><td rowspan=1 colspan=1>Clostridium sordellii</td></tr><tr><td rowspan=1 colspan=1>Clostridium bolteae</td><td rowspan=1 colspan=1>Clostridium nexile</td><td rowspan=1 colspan=1>Clostridium lavalense</td></tr><tr><td rowspan=1 colspan=1>Clostridium butyricum</td><td rowspan=1 colspan=1>Clostridium novyi</td><td rowspan=1 colspan=1>Clostridium sphenoides</td></tr><tr><td rowspan=1 colspan=1>Clostridium chauvoei</td><td rowspan=1 colspan=1>Clostridium orbiscindens</td><td rowspan=1 colspan=1>Clostridium spiroforme</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(non-toxigenic strains)</td><td rowspan=1 colspan=1>Clostridium paraputrificum</td><td rowspan=1 colspan=1>Clostridium sporogenes</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(Xla (A-B-tox bin+) and XIb)</td><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Clostridium symbiosum</td></tr><tr><td rowspan=1 colspan=1>Clostridium fallax</td><td rowspan=1 colspan=1>Clostridium ramosum</td><td rowspan=1 colspan=1>Clostridium tertium</td></tr><tr><td rowspan=1 colspan=1>Clostridium haemolyticum</td><td rowspan=1 colspan=1>Clostridium scindens</td><td rowspan=1 colspan=1>Clostridium tetani</td></tr><tr><td rowspan=1 colspan=3>Other Bacterial Species ( 1x10 CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia defectiva</td><td rowspan=1 colspan=1>Enterococcus faecium VanA</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1>Porphyromonasasaccharolytica</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwoffi</td><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis subsp.faecalis</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Proteus penneri</td></tr><tr><td rowspan=1 colspan=1>Anaerococcus tetradius</td><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>Providencia alcalifaciens</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>Providencia rettgeri</td></tr><tr><td rowspan=1 colspan=1>Bacteroides caccae</td><td rowspan=1 colspan=1>Fusobacterium varum</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Bacteroides stercoris</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium longum</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>Ruminococcus bromii</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Helicobacter cinaedi</td><td rowspan=1 colspan=1>Salmonella enterica subsp.enterica</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp.jejuni</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Salmonella enterica subsp.arizonae</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Serratia liquefaciens</td></tr><tr><td rowspan=1 colspan=1>Candida catenulata</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Cedecea davisae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp.pneumoniae</td><td rowspan=1 colspan=1>Shigella boydii</td></tr><tr><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Shigella dysenteriae</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>Lactococcus lactis subsp. lactis</td><td rowspan=1 colspan=1>Staphylococcus aureussubsp. aureus</td></tr><tr><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>Leminorella grimontii</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Collinsella aerofaciens</td><td rowspan=1 colspan=1>Listeria grayi</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td></tr></table>

# BD MAX™ Cdiff Assay PreMarket Notification

<table><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Desulfovibrio piger</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Streptococcus dysgalactiaesubsp. dysgalactiae</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>Mitsuokella multacida</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Eggerthella lenta</td><td rowspan=1 colspan=1>Mobiluncus curtisii subsp.holmesii</td><td rowspan=1 colspan=1>Streptococcus uberis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Moellerella wisconsensis</td><td rowspan=1 colspan=1>Trabulsiella guamensis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp.cloacae</td><td rowspan=1 colspan=1>Morganella morganii subsp.morganii</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>Neisseria gononhoeae</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus cecorum</td><td rowspan=1 colspan=1>Parabacteroides merdae</td><td rowspan=1 colspan=1>Yersinia bercovieri</td></tr><tr><td rowspan=1 colspan=1>Enterococcus dispar</td><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Yersinia rohdei</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intracellular bacteria(1x10 elementary bodies/mL)</td><td rowspan=1 colspan=1>Vibrio cholerae species~1x10^ cp/mL)</td><td rowspan=1 colspan=1>Human sample~1x10³ cp/mL)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Homo sapiens</td></tr><tr><td rowspan=1 colspan=3>virus (1x10^ PFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 14</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Cytomegalovirus, Strain AD-169</td></tr><tr><td rowspan=1 colspan=1>Rotavirus, stain WA</td><td rowspan=1 colspan=1>Echovirus 19 (Strain Burke)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1> Norovirus group 1</td><td rowspan=1 colspan=1>Coxsackie virus B1 (Com-5)</td><td rowspan=1 colspan=1></td></tr></table>

# Interfering Substances

Twenty-five (25) biological and chemical substances occasionally used or found in perianal, rectal and/or stool specimens were evaluated for potential interference with the BD MAX™M Cdiff Assay. Two (2) organisms (E. coli ATCC 25922 and non-toxigenic C. difficile ATCC 700057) were also tested at high loads in order to assess bacterial interference. Toxigenic C. difficile negative specimens and toxigenic C. dificile positive specimens at 2-3X LoD were tested with the highest amount of each compound likely to be found in the specimens or with interfering organisms (1 X $1 0 ^ { 8 }$ CFU/mL of each strain). Potentially interfering substances include calcium carbonate (Tums®) as well as magnesium and aluminum hydroxide (Maalox $\textsuperscript { \textregistered }$ liquid). Results demonstrated no reportable interference with any other tested substance except for Mesalamine rectal suspension enema and Gynol $1 1 \textcircled { \times }$ that both showed slight inhibition (delay of Second Derivative Peak Abscissa) in the BD MAX™ Cdiff Assay, however, expected assay results were still obtained (Table 11).

Table 11: Endogenous and Commercial Exogenous Substances tested with the BD MAX™ Cdiff Assay   

<table><tr><td rowspan=1 colspan=1>Brand Name or Description</td><td rowspan=1 colspan=1>Weight orolumeTested/SBT</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Brand Name or Description</td><td rowspan=1 colspan=1>Weight orVolumeTested/SBT</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Pepto Bismol ™</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>HydermHydrocortisone(cream)</td><td rowspan=1 colspan=1>0.0246 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Ex-Lax</td><td rowspan=1 colspan=1>0.0134 g</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Glycerin Suppositories</td><td rowspan=1 colspan=1>0.0129 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>Ni</td></tr><tr><td rowspan=1 colspan=1>Ihle&#x27;s Paste</td><td rowspan=1 colspan=1>0.0388 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Anusol Plus</td><td rowspan=1 colspan=1>0.0123 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Polysporin</td><td rowspan=1 colspan=1>0.0240 g</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Preparation H® with Bio-Dyne (cream)</td><td rowspan=1 colspan=1>0.0222 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Naproxen</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Major Prep withPhenylephrine</td><td rowspan=1 colspan=1>0.0111 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Tucks® PersonalCleansing Pads</td><td rowspan=1 colspan=1>3 mm^{2}}$</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Tums</td><td rowspan=1 colspan=1>0.0395 g</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Triglyceride Mix (C2-C10)</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Maalox (liquid)</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Palmitic Acid</td><td rowspan=1 colspan=1>25 mg</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Mesalamine RectalSuspension Enema</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>Stearic Acid</td><td rowspan=1 colspan=1>10 mg</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Fleet® Mineral Oil Enema</td><td rowspan=1 colspan=1>0.0182 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Gynol Il® VaginalContraceptive(with Nonoxynol-9)</td><td rowspan=1 colspan=1>0.0262 g</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Imodium AD</td><td rowspan=1 colspan=1>0.0062 g</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>E. coli + non-toxigenicC. difficile</td><td rowspan=1 colspan=1>10 μL</td><td rowspan=1 colspan=1>NI</td></tr></table>

I: Interference with the BD MAX™ Cdiff Assay. N: No reportable interference with the BD Cdiff Assay. in the BD MACdif Assay, however, expected assay results were sill btaid

# Carryover / Cross-Contamination

A study was conducted to investigate within-run carryover and between-run carryover while processing specimens with high bacterial load of toxigenic C. difficile in the BD MAX™ Cdiffassay. A panel made of one high positive member and one negative member was used to prepare numerous samples. A Clostridium difficile strain (Tox 0, ATCC 9689) was used for the high positive C. difficile panel member $( \sim 3 \times 1 0 ^ { 8 } \mathsf { C F U / m L } )$ . The negative member.did not contain any target analyte. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run by alternating negative and positive samples. Three (3) operators performed 3 consecutive uns for a total  runs   samples.There were o false positive results due o carryover contamination.

# Clinical Performance Studies

Clinical performance characteristics of the BD MAX™ Cdiff Assay were determined in a multi-site prospective investigational study. Six (6) investigational centers participated in the study. To be enroled in the study, specimens had to be from patients suspected of having C. difficile infection for which diagnostic tests were indicated and ordered. Only soft or liquid stools, and only one specimen per patient, were included.

The Comparative Reference Method consisted of direct culture complemented by enriched culture. Enriched culture analysis was completed for all specimens that were negative for

# BD MAX ™ Cdiff Assay PreMarket Notification

toxigenic C. difficile by direct culture. The anaerobic culture was used to isolate C. difficile, i present. This was followed by confirmation of the isolate identification and a Tissue Culture Cytotoxicity Assay to determine the toxigenicity of the isolate.

Of the 2071 soft or liquid stool specimens compliant with the Reference Method (Direct and Enriched Culture), 1819 gave compliant and reportable results with the BD MAX TMCdiff Assay. In comparison to the Reference Method, the BD MAX ™ Cdiff Assay identified $8 7 . 7 \%$ of the toxigenic C. difficile positive specimens and $9 6 . 8 \%$ of the toxigenic C. difficile negative specimens (Tables 12 and 13).

Table 12: Results Obtained with the BD MAX ™ Cdiff Assay in Comparison to the Reference Method   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXT CdiffAssay</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>481</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>371</td><td rowspan=1 colspan=1>1469</td><td rowspan=1 colspan=1>1506</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>302</td><td rowspan=1 colspan=1>1517</td><td rowspan=1 colspan=1>1819</td></tr><tr><td rowspan=1 colspan=5>Sensitivity: 87.7% (265/302) (95% Cl: 83.6%, 91.0%)Specificity: 96.8% (1469/1517) (95% Cl: 95.8%, 97.6%)PPV: 83.5% (95% CI: 79.4%, 87.1%)NPV: 97.7% (95% CI: 97.0%, 98.4%)</td></tr></table>

Further investigation was performed on specimens with discordant results between the Reference Method and the BD MAX ™ Cdiff Assay.

27 of 48 False Positive BD MAX™ Cdiff specimens were also found to be positive using another commercially available FDA-cleared RT-PCR assay targeting the C. difficile tcdB gene. 32 of 37 False Negative BD MAX™ Cdiff specimens, were also found to be negative using another commercially available FDA-cleared RT-PCR assay targeting the C. difficile tcdB gene.

Table 13. Results Obtained by Site using the BD MAX™ Cdiff Assay in Comparison to the Reference Method   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>90.0% (18/20)(69.9%, 97.2%)</td><td rowspan=1 colspan=1>95.3% (202/212)(91.5%, 97.4%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>84.4%(38/45)(71.2%,92.3%)</td><td rowspan=1 colspan=1>95.3% (205/215)(91.7%,,97.5%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>94.7%(36/38)(82.7%,98.5%)</td><td rowspan=1 colspan=1>97.9%(275/281)(95.4%,99%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>83.8%(31/37)(68.9%,, 92.3%)</td><td rowspan=1 colspan=1>95.8% (322/336)(93.1%, 97.5%)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>96.1% (49/51)(86.8%, 98.9%)</td><td rowspan=1 colspan=1>98.4% (186/189)(95.4%, 99.5%)</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>83.8% (93/111)(75.8%, 89.5%)</td><td rowspan=1 colspan=1>98.2% (279/284)(95.9%, 99.2%)</td></tr><tr><td rowspan=1 colspan=1>Overall Study</td><td rowspan=1 colspan=1>87.7% (265/302)(83.6%, 91.0%)</td><td rowspan=1 colspan=1>96.8% (1469/1517)(95.8%, 97.6%)</td></tr></table>

'Numbers in parentheses express the $95 \%$ confidence interval boundaries

In comparison to direct culture, the BD MAX™ Cdiff Assay identified $9 6 . 5 \%$ of the toxigenic C. difficile positive specimens and $9 2 . 7 \%$ of the toxigenic C. difficile negative specimens (Tables 14 and 15).

Table 14: Results Obtained with the BD MAX ™ Cdiff Assay in Comparison to Direct Culture   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>Direct Culture</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAX™ CdiffAssay</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>312</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1507</td><td rowspan=1 colspan=1>1514</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>1625</td><td rowspan=1 colspan=1>1826</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement: 96.5% (194/201) (95% Cl: 93.0%, 98.3%)Negative Percent Agreement: 92.7% (1507/1625) (95% Cl: 91.4%, 93.9%)</td></tr></table>

Table 5. Results Obtained by Site using the BD MAX™ Cdiff Assay in Comparison to Direc Culture   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>SensitivityPositivePercent Agreement</td><td rowspan=1 colspan=1>Negative PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>92.9% (13/14)(68.5%, 98.7%)1</td><td rowspan=1 colspan=1>93.1% (203/218)(89.0%, 95.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>100.0%(26/26)(87.1%,100%)</td><td rowspan=1 colspan=1>90.6% (213/235)(86.2%, 93.7%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>96.0% (24/25)(80.5%,, 99.3%)</td><td rowspan=1 colspan=1>93.9% (276/294)(90.5%, 96.1%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>93.8% (15/16)(71.7%, 98.9%)</td><td rowspan=1 colspan=1>91.7% (330/360)(88.4%, 94.1%)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>95.7% (45/47)(85.8%, 98.8%)</td><td rowspan=1 colspan=1>96.4% (188/195)(92.8%, 98.3%)</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>97.3% (71/73)(90.5%, 99.2%)</td><td rowspan=1 colspan=1>92.0% (297/323)(88.5%, 94.4%)</td></tr><tr><td rowspan=1 colspan=1>Overall Study</td><td rowspan=1 colspan=1>96.5% (194/201)(93.0%, 98.3%)</td><td rowspan=1 colspan=1>92.7% (1507/1625)(91.4%, 93.9%)</td></tr></table>

'Numbers in parentheses express the $9 5 \%$ confidence interval boundaries

O 860  qu o se  wite B MAX ssy8 3%) were initially reported as unresolved. 42 of those were repeated and 32 were resolved upon repeat testing. Overall, $0 . 5 \%$ remainedunresolved afterrepeat Tabl.

Table 16: Unresolved Rate   

<table><tr><td>Initial Unresolved Rate</td><td>Unresolved Rate After Repeat</td></tr><tr><td>3.1% (58/1860)* (95% CI: 2.4%, 4.0%)</td><td>0.5% (10/1844) (95% Cl: 0.3%, 1.0%)</td></tr></table>

# BD MAX™ Cdiff Assay PreMarket Notification

\* Total number based on compliant specimens and BD MAX ™ Cdiff Assay results.

Out of 1916 soft or liquid stool specimens tested with the BD MAX™ Cdiff Assay, 21 (1.1%) were initially reported as indeterminate. Out of a total of 1844 compliant results, no result remained Indeterminate upon repeat (considering valid results and compliant repeats).

Out of 1916 soft or liquid stool specimens tested with the BD MAX ™ Cdiff Assay, 28 $( 1 . 5 \% )$ EY were initially reported as incomplete. Out of a total of 1844 compliant results, no result remained Incomplete upon valid repeat (considering valid results and compliant repeats).

# Comparison Studies

In addition to the multi-site investigational study, a comparison study was performed using the BD MAX™ Cdiff assay and two (2) commercially available FDA-cleared molecular assays for the detection of the C. difficile tcdB gene. Testing was performed on 2013 specimens at two external sites.

In comparison to one FDA-cleared molecular test (Table 10a), the PPA and NPA of the BD MAX Cdiff Assay were $9 9 . 1 \%$ (CI: $9 4 . 9 \% - 9 9 . 8 \%$ and $9 7 . 4 \%$ (CI: $9 5 . 7 \% . 9 8 . 4 \% )$ respectively.

In comparison to a second FDA-cleared molecular test (Table 10b), the PPA and NPA of the BD MAX™ Cdiff Assay were $9 5 . 5 \%$ (CI: $9 2 . 1 \% 9 7 . 5 \%$ and $9 8 . 8 \%$ (CI: $9 7 . 9 \% 9 9 . 3 \%$ respectively.

Table 17a: Results Obtained with the BD MAX™ CdiffAssay in Comparison to a Commercially Available FDA-cleared Molecular Assay for C. difficile.   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>FDA-cleared Molecular Assay 1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAX™ CdiffAssay</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>526</td><td rowspan=1 colspan=1>527</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>647</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement: 99.1% (95% Cl: 94.9%, 99.8%)Negative Percent Agreement: 97.4% (95% Cl: 95.7%, 98.4%)</td></tr></table>

Table 17b: Results Obtained with the BD MAX™ Cdiff Assay in Comparison to a second Commercially Available FDA-cleared Molecular Assay for C. difficile.   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>FDA-cleared Molecular Assay 21y2</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXT CdiffAssay</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>247</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1108</td><td rowspan=1 colspan=1>1119</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>244</td><td rowspan=1 colspan=1>1122</td><td rowspan=1 colspan=1>1366</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement: 95.5% (95% Cl: 92.1%, 97.5%)Negative Percent Agreement: 98.8% (95% Cl: 97.9%, 99.3%)</td></tr></table>

# Expected Values

In the BD MAX ™ Cdiff Assay clinical study a total of 1834 reportable results, from specimens compliant at the specimen and PCR levels, were obtained from 6 geographically diverse sites and compared with Direct and Enriched culture. The study population was grouped into in-patient, out-patient and unknown categories. The number and percentage of positive cases, as determined by the BD MAX™ Cdiff Assay, are presented in Table 18 below.

Table 18: Expected Values   

<table><tr><td></td><td></td><td rowspan=1 colspan=2>BD MAXT Cdiff Assay</td><td rowspan=2 colspan=1>Positive Percentage</td></tr><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Total Number ofSpecimens</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>NumberNegative</td></tr><tr><td rowspan=1 colspan=1>In-patient</td><td rowspan=1 colspan=1>1249</td><td rowspan=1 colspan=1>184J</td><td rowspan=1 colspan=1>1065</td><td rowspan=1 colspan=1>14.7% (184/1249)</td></tr><tr><td rowspan=1 colspan=1>Out-patient</td><td rowspan=1 colspan=1>457</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>343</td><td rowspan=1 colspan=1>24.9% (114/457)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>13.3% (17/128)</td></tr><tr><td rowspan=1 colspan=1>Total1</td><td rowspan=1 colspan=1>1834</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>1519</td><td rowspan=1 colspan=1>17.2% (315/1834)</td></tr></table>

Total number of specimens based on compliant PCR results

PATRICIA DIONNE   
DIRECTOR, REGULATORY AFFAIRS   
GENEOHM SCIENCES CANADA, INC. (BD DIAGNOSTICS)   
2555 BOUL. DU PARC-TECHNOLOGIQUE   
QUEBEC (QUEBEC) G1P 4S5   
CANADA

Re: K130470 Trade/Device Name: Bd MAX Cdiff Assay Regulation Number: 21 CFR 866.3130 Regulation Name: Clostridium difficile Toxin Gene Amplification Assay Regulatory Class: II Product Code: OZN, OOI Dated: February 22, 2013 Received: February 27, 2013

Dear Dr. Dionne:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or classIII (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be f in h Cod  Federal Regulations, Ti art 80 to In dition, Fy publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

Page 2—Dr. Dionne

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostics and Radiological Health at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

# Uwe Scherf -S for

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics and Radiological Health Center for Devices and Radiological Health

Enclosure

# BD MAX T Cdiff Assay PreMarket Notification

# 1.2 Indications for Use Statement

510(k) Number (if known): K 130470

Device Name: _ BD MAX T Cdiff Assay

Intended Use:

The BD MAX ™M Cdiff Assay performed on the BD MAX ™ System is an automated in vitro diagnostic test for the direct, qualitative detection of the Clostridium difficile toxin B gene (tcdB) in human liquid or soft stool specimens from patients suspected of having C. difficile infection (CDl). The test, performed directly on the specimen, utilizes real-time polymerase chain reaction (PCR) for the amplification of C. difficile toxin B gene DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD MAX ™ Cdiff Assay is intended to aid in the diagnosis of CDI.

Prescription   
Use_ XXX   
(Per 21 CFR 801.109)

Over-The-Counter Use (Optional Format 1-2-96)

# (PLEASE DO NOT WRITE BELOW THIS LINE - CONTINUE ON ANOTHER PAGE IF EDED)

Concurrence of CDRH, Office of In Vitro Diagnostic Devices Evaluation and Safety (IVD)