# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K131728 B. Purpose for Submission:

This is a traditional $5 1 0 ( \mathrm { k } )$ submission requesting market clearance for the Quide $^ \mathrm { \textregistered }$ Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay, an in vitro diagnostic device, using the Life Technologies QuantStudio™ Dx Real-Time PCR Instrument.

The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay has been reviewed and 510(k) cleared for use on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx instrument under K112172 and on the Cepheid SmartCycler® II instrument under K113777.

# C. Measurand:

Influenza A RNA sequence within the matrix protein gene and influenza B viral RNA sequence within the neuraminidase gene.

# D. Type of Test:

The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ is a multiplex real time reverse transcriptionpolymerase chain reaction (RT-PCR) in vitro assay for qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs. The test utilizes NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ System (bioMérieux) for nucleic acid isolation and purification followed by amplification and detection using the Life Technologies QuantStudio™ Dx Real-Time PCR Instrument.

# E. Applicant:

Quidel Corporation

F. Proprietary and Established Names: Quidel $^ \mathrm { \textregistered }$ Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code: OZE, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Quidel $^ \mathrm { \textregistered }$ Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay is a multiplex Real Time RT-PCR assay for the in vitro qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2011 and 2013 influenza seasons when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

The assay can be performed using either the Life Technologies QuantStudio™ Dx, the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx (ABI 7500), or the Cepheid SmartCycler $^ \mathrm { \textregistered }$ II.

2. Indication(s) for use: Same as the Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

4. Special instrument requirements:

bioMérieux NucliSENS easyMAG System (software version 2.0) Life Technologies QuantStudio™ Dx Real Time PCR Instrument (software version 1.0)

K112172 for Applied Biosystems (ABI) 7500 FAST Dx (software version 1.4), and

K113777 for Cepheid SmartCycler $\textsuperscript { \textregistered }$ II (software version 3.0b)

# I. Device Description:

The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay detects viral nucleic acids that have been extracted from a patient sample using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ automated extraction platform. The extraction of the nucleic acids is performed according to the manufacturer’s instructions (Quidel). The Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ Assay contains specific primers and probes for influenza A and influenza B virus and for the Process Control (PRC). The PRC is an inactivated and stabilized MS2 bacteriophage containing an RNA genome and is added to each sample aliquot prior to extraction. The PRC controls for the assay procedure, reverse transcriptase and polymerase activity, and for the presence of inhibitors in the sample that may interfere with the assay.

The Master Mix (MM) is provided in a lyophilized state and must be rehydrated prior to testing. The MM contains oligonucleotide primers, fluorophore and quencher-labeled probes targeting highly conserved regions of the influenza A and influenza B viruses as well as the PRC sequence. The primers are complementary to highly specific and conserved regions in the genome of these viruses. The probes are dual labeled with a reporter dye attached to the $5 ^ { \circ }$ -end and a quencher attached to the $3 ^ { \circ }$ -end. The assay uses the following reporter dyes.

<table><tr><td rowspan=1 colspan=3>QuantStudio Dx Dye Filters</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Reporter DyeFilter</td><td rowspan=1 colspan=1>Quencher Dye</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Process Control</td><td rowspan=1 colspan=1>CY5</td><td rowspan=1 colspan=1>None</td></tr></table>

The amplification and detection is performed on the Life Technologies QuantStudio $^ \mathrm { \textregistered }$ Dx Real Time PCR Instrument using 96-well PCR plates. A multiplex real-time RT-PCR reaction is carried out under optimized conditions in a single tube generating amplicons for each of the target viruses present in the sample. The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay is based on TaqMan $^ \mathrm { \textregistered }$ chemistry, and uses an enzyme with reverse transcriptase, DNA polymerase, and $5 ' - 3 '$ exonuclease activities. The user pipettes $1 5 ~ \mu \mathrm { L }$ of the rehydrated MM and $5 ~ \mu \mathrm { L }$ of extracted

nucleic acids (containing the PRC). Once the plate is inserted into the instrument, the assay protocol is initiated. During DNA amplification, an enzyme cleaves the probe bound to the complementary DNA sequence, separating the quencher dye from the reporter dye. This step generates an increase in fluorescent signal upon excitation by a light source of the appropriate wavelength. With each cycle, additional dye molecules are separated from their quenchers resulting in additional signal. If sufficient fluorescence is achieved a Ct (threshold cycle) value is reported for a given target and the sample is considered positive for the detected target sequence.

<table><tr><td rowspan=1 colspan=4>Interpretation of the Quidel Molecular Influenza A+B Assay Results onLife Technologies QuantStudio Dx Real-Time PCR Instrument</td></tr><tr><td rowspan=1 colspan=1>Detector:Influenza A</td><td rowspan=1 colspan=1>Detector:Influenza B</td><td rowspan=1 colspan=1>Detector:ProcessControl</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>Ct-valueReported</td><td rowspan=1 colspan=1>Negative - No Influenza A or Influenza Bviral RNA detected; PRC detected</td></tr><tr><td rowspan=1 colspan=1>Ct-valueReported</td><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza A Positive - Influenza A viralRNA detected</td></tr><tr><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>Ct-valueReported</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza B Positive - Influenza B viralRNA detected</td></tr><tr><td rowspan=1 colspan=1>Ct-valueReported</td><td rowspan=1 colspan=1>Ct-valueReported</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza A and B Positive - Influenza Aand Influenza B viral RNA detected**</td></tr><tr><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>No Ct-valueReported</td><td rowspan=1 colspan=1>Invalid - No Influenza A or Influenza Band no PRC viral RNA detected; invalidtest. Retest the same purified sample. Ifthe test is also invalid, re-extract and retestanother aliquot of the same sample orobtain a new sample and retest.</td></tr></table>

$\ast \mathrm { N o \ C t }$ value is required for the Process control to make a positive call. \*\* Dual infections are rare. Repeat testing using the same purified sample. If the retest confirms this result, collect and test a new specimen. Contact Quidel if multiple samples provide this result.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Prodesse ProFlu+ Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ on the Cepheid SmartCycler II platform

2. Predicate 510(k) number(s): K092500, K112171, K113777

3. Comparison with predicates:

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Quidel MolecularInfluenza A+BK131728</td><td colspan="1" rowspan="1">Predicate K092500</td><td colspan="1" rowspan="1">PredicatesK112171,K113777</td></tr><tr><td colspan="1" rowspan="1">Format</td><td colspan="1" rowspan="1">Multiplex real time RT-PCR</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Detects anddifferentiates influenzaA and influenza B viralRNA from nasal andnasopharyngeal swabs.</td><td colspan="1" rowspan="1">Detects anddifferentiatesinfluenza A,influenza B andRespiratory SyncytialVirus (RSV) viralRNA from nasal andnasopharyngealswabs.</td><td colspan="1" rowspan="1">Detects anddifferentiatesinfluenza Aand influenzaB viral RNAfrom nasal andnasopharyngealswabs.</td></tr><tr><td colspan="1" rowspan="1">Extraction Method</td><td colspan="1" rowspan="1">bioMérieux easyMAGAutomated MagneticExtraction Reagents</td><td colspan="1" rowspan="1">bioMérieuxeasyMAGAutomated MagneticExtraction Reagents,orRoche MagNA PureLC Total NucleicAcid Isolation Kit</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Principle</td><td colspan="1" rowspan="1">PCR-based system fordetecting the presenceor absence of viral RNAin clinical specimens</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Multiplex assay usingdifferent reporter dyesfor each target</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Viral Targets</td><td colspan="1" rowspan="1">Influenza A: Matrixgene;Influenza B: Conservedsequence within theneuraminidase gene</td><td colspan="1" rowspan="1">Influenza A: MatrixGene;Influenza B: Non-structural NS1 andNS2</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Quidel MolecularInfluenza A+BK131728</td><td colspan="1" rowspan="1">Predicate K092500</td><td colspan="1" rowspan="1">Predicates K112171,K113777</td></tr><tr><td colspan="1" rowspan="1">Detects</td><td colspan="1" rowspan="1">Influenza A andInfluenza B</td><td colspan="1" rowspan="1">Influenza A,Influenza B andRespiratorySyncytial Virus(RSV).</td><td colspan="1" rowspan="1">No Difference</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasal andnasopharyngealswabs</td><td colspan="1" rowspan="1">Nasopharyngealswabs</td><td colspan="1" rowspan="1">No Difference</td></tr><tr><td colspan="1" rowspan="1">PCRInstrument</td><td colspan="1" rowspan="1">QuantStudioDx</td><td colspan="1" rowspan="1">CepheidSmartCycler II</td><td colspan="1" rowspan="1">Cepheid SmartCycler II orABI 7500</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

1. Guidance for Industry and FDA Staff - Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (July 15, 2011) - http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc m079171.htm   
2. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9, 2009) - http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc m180307.htm   
3. Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) http://www.fda.gov/cdrh/oivd/guidance/1588.pdf.   
4. Guidance for Industry and Food and Drug Administration Staff - eCopy Program for Medical Device (December 2012) http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo cuments/UCM313794.pdf   
5. Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999) - http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc m073778.htm

The following documents were referenced in one or more of the Guidance Documents above and were used in analytical study designs:

1. CLSI EP17-A: Guidance for Protocols for Determination of Limits of Detection and Limits of Quantitation (Vol. 2, No. 34) (Oct 2004).

2. CLSI MM13-A: Guidance for the Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (Vol. 25, No. 31) (Dec 2005).   
3. CLSI EP7-A2: Guidance for Interference Testing in Clinical Chemistry (Vol. 25, No.27 Second Ed) (Nov 2005).   
4. CLSI EP12-A: Guidance for User Protocol for Evaluation of Qualitative Test Performance (Vol. 22, No. 14) (Sept 2002).   
5. CLSI MM6-A: Guidance for the Quantitative Molecular Methods for Infectious Diseases (Vol. 23, No.28) (Oct 2003).   
6. CLSI EP5-A2: Guidance for Evaluation of Precision Performance of Quantitative Measurement Methods (Vol. 24, No. 25 Second Ed.) (Aug 2004).

# L. Test Principle:

Detection of the viral targets and the Process Control (PRC) is based on three processes: nucleic acid isolation, reverse transcription, and real time PCR amplification/detection. Human respiratory specimens (nasal swabs and nasopharyngeal swabs) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. Identification of influenza A occurs by the use of target specific primers and a fluorescent-labeled probe that hybridizes to a conserved influenza A sequence within the matrix protein gene. Identification of influenza B occurs by the use of target specific primers and fluorescent-labeled probes that will hybridize to a conserved influenza B sequence within the neuraminidase gene. The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction.

After initial reverse transcription of RNA into complementary DNA (cDNA), amplification procedes during which the probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This generates an increase in fluorescent signal upon excitation from a light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time. Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real-time PCR instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Internal Repeatability Study

An internal precision study was conducted in-house over a 12 day period with 2 operators and one instrument. Sample panel members were prepared using grown and titered strains of influenza A (A/Mexico/4108/2009) and influenza B (B/Florida/04/2006) spiked into negative nasal matrix. The target concentrations were relative to the established LoD for each virus, and a panel consisted of two aliquots at three concentrations of the virus (as shown below), a negative sample and a Positive and Negative Influenza $\mathrm { A } / \mathrm { B }$ control.

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>TCID50Titer(5X LoD)</td><td rowspan=1 colspan=1>TCID50Titer(2X LoD)</td><td rowspan=1 colspan=1>TCID50Titer(0.3LoD)</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>9.75E+01</td><td rowspan=1 colspan=1>3.90E+01</td><td rowspan=1 colspan=1>5.85E+00</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>5.00E+02</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>3.00E+01</td></tr></table>

Each operator performed extraction and PCR once a day for each sample (aliquot). The results are presented in Ct values, with an average, SD and $\% \mathrm { C V }$ calculated for each panel member per operator for each virus (influenza A and influenza B). The percent agreement with the expected result is also presented.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="10" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">Operator 1</td><td colspan="5" rowspan="1">Operator 2</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Detection Rate</td><td colspan="1" rowspan="1">%Agreement withExpected Result</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">STD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Detection Rate</td><td colspan="1" rowspan="1">%Agreemet withExpected Result</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">STD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">5xLoD (Positive)</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">2.62</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">2.91</td></tr><tr><td colspan="1" rowspan="1">2xLoD (LowPositive</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">3.81</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">3.56</td></tr><tr><td colspan="1" rowspan="1">0.3xLoD (HighNegative</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">3/24</td><td colspan="1" rowspan="1">87.5%</td><td colspan="1" rowspan="1">39.2</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">10/24</td><td colspan="1" rowspan="1">58.3%</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">1.53</td></tr><tr><td colspan="1" rowspan="1">Negative (Matrix)</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">A/B Pos. Control</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">29.8</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">2.68</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">29.1</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">3.09</td></tr><tr><td colspan="1" rowspan="1">A/B Neg. Control</td><td colspan="1" rowspan="1">$\fac{2 }$</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="10" rowspan="1">Flu B</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">Operator 1</td><td colspan="5" rowspan="1">Operator 2</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">%Agreement withExpected Result</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">STD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">%Agreement withExpected Result</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">STD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">5xLoD (Positive)</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.90</td></tr><tr><td colspan="1" rowspan="1">2xLoD (LowPositive</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">1.70</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.41</td></tr><tr><td colspan="1" rowspan="1">0.3xLoD (HighNegative)</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">7/24</td><td colspan="1" rowspan="1">70.8%</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">8/24</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">Negative (Matrix)</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">A/B Pos. Control</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">27.9</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.43</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">27.7</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">1.08</td></tr><tr><td colspan="1" rowspan="1">A/B Neg Control</td><td colspan="1" rowspan="1">~</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# Reproducibility Study

The reproducibility of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay was evaluated at three laboratory sites. Reproducibility was assessed using a panel of eight simulated samples which included 2 positive samples (above LoD), a high negative $0 . 3 { \mathrm { x } }$ LoD) sample, and a negative (non-spiked) sample for each virus, influenza A (A/Mexico/4108/2009) and influenza B (B/Florida/04/2006). The panel also included daily quality controls. Panels and controls were tested at each site by two operators for five days, each sample tested in 3 replicates, for a total of 90 results per level for each virus (2 operators x 5 days x 3 sites x 3 replicates). Each panel and controls were extracted using the bioMérieux easyMAG system and tested on the Life Technologies QuantStudio Dx platform.

<table><tr><td rowspan=1 colspan=13>Reproducibility Results - Life Technologies QuantStudioTM Dx</td></tr><tr><td rowspan=2 colspan=1>PanelMemberID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Combined Site Data</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA HighNgative(0.3 xLoD</td><td rowspan=1 colspan=1>25/30</td><td rowspan=1 colspan=1>37.4*</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>38.2*</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>4/28**</td><td rowspan=1 colspan=1>37.5*</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>42/88**</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>InfluenzaAPositive1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>29/29**</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>89/89**</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>InfluenzaAPositive2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>3.9</td></tr><tr><td rowspan=1 colspan=1>InfluenzaANegative</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB HighNegative(0.3  xLoD</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>37.7*</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>37.8*</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>14/28**</td><td rowspan=1 colspan=1>37.7*</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>65/88**</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>InfluenzaBPositive1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>25.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>89/89**</td><td rowspan=1 colspan=1>25.7</td><td rowspan=1 colspan=1>5.2</td></tr><tr><td rowspan=1 colspan=1>InfluenzaBPositive2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>5.5</td></tr><tr><td rowspan=1 colspan=1>InfluenzaBNegative</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>InfluenzaAPositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>InfluenzaBPositive CControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>NegativeTControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\*Average Ct of positive results only \*\* One or more of the replicates was invalid due to non-detection of the PRC

Upon review of the reproducibility data it was determined that the concentration of the positive samples was higher than expected. A supplemental study was conducted internally using a near-LoD specimen. In this study, a sample with a low positive concentration (2x LoD) for influenza A and for influenza B and a negative sample were each extracted on three bioMérieux easyMAG systems and then tested on three QuantStudio Dx platforms. The two samples and controls were tested by two operators per instrument for five days, each sample tested in 3 replicates, for a total of 90 results per sample for each virus for each instrument (2 operators $_ { \textrm { X } 5 }$ days $_ { \textrm { X 3 } }$ instruments x 3 replicates). This data is presented in a table below as a supplemental study.

<table><tr><td rowspan=1 colspan=12>Reproducibility Results - Life Technologies QuantStudioTM Dx  Supplemental Study</td></tr><tr><td rowspan=1 colspan=3>QuantStudioTM #1</td><td rowspan=1 colspan=3>QuantStudio #2</td><td rowspan=1 colspan=3>QuantStudioTM #3</td><td rowspan=1 colspan=3>Combined Instrument Data</td></tr><tr><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

b. Linearity/assay reportable range:

N/A

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Reagent Stability

Reagent real time stability at $2 ^ { \circ } C - 8 ^ { \circ } C$ was conducted with three lots of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay kits. The data collected supports two years of reagent shelf life stability when stored at $2 ^ { \circ } C - 8 ^ { \circ } C$ .

# Master Mix Stability after Rehydration

An evaluation of master mix stability after rehydration was carried out over a five day period using various storage conditions, including ${ } _ { - 2 0 } \mathrm { { \circ C } }$ , $4 ^ { \circ } \mathrm { C }$ , and $2 5 \mathrm { { ^ \circ C } }$ (room temperature). Mock specimens at 100X, 10X, and 3X LoD were used in the analysis. The data was reviewed under K112171. The data showed that the rehydrated master mix can be stored for up to 24-hours at room temperature ( $2 0 ^ { \circ }$ to $2 5 ^ { \circ } \mathrm { C }$ ) or at $\leq - 2 0 ^ { \circ } \mathrm { C }$ for up to two weeks.

# Extracted Specimen Stability

An evaluation of the stability of extracted specimens was carried out using the easyMAG extraction system followed by real-time RT-PCR analysis. Parallel aliquots of samples that were influenza A (3X LoD) and influenza B (3X LoD) positive were used for this study. Samples were analyzed at time 0 (immediately after extraction), after 2h at room temperature, after 8h at $2 { - } 8 ^ { \circ } \mathrm { C }$ , after 96 hours with up to three freeze thaw cycles, and after one month at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ . The data was reviewed under K112171.The data showed that extracted nucleic acids may be stored at room

temperature ( $2 0 ^ { \circ }$ to $2 5 ^ { \circ } \mathrm { C }$ ) for 2 hours, at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ for 8 hours and for one month at $- 2 0 ^ { \circ }$ to $- 7 0 \mathrm { { } ^ { \circ } C }$ .

# d. Detection limit:

A Limit of Detection for the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay when used on the QuantStudio Dx instrument was established for three (3) strains of Influenza A and three (3) strains of Influenza B. A stock was prepared for each set of viruses in negative nasal matrix that contained both influenza A and influenza B at a concentration approximately $1 0 X$ LoD (based on previous results on the 7500 Fast Dx platform, see K112172). Each stock was diluted to 3X LoD, 1X LoD and 0.3X LoD. Additional concentrations were prepared as needed. Each dilution was split into 20 aliquots, where each aliquot was extracted separately using the NucliSENS easyMAG (bioMérieux) instrument. The LoD was defined as the lowest concentration at which $\ge 9 5 \%$ of all replicates tested positive.

The final LoD for the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ when used on the QuantStudio Dx instrument for each influenza strain tested is shown below.

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Final LOD(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009(H1N1)</td><td rowspan=1 colspan=1>2.0E+01</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007(H1N1)</td><td rowspan=1 colspan=1>1.00E+02</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane(H3N2)</td><td rowspan=1 colspan=1>1.00E+02</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane&#x27;09-&#x27;10Vaccinestrain</td><td rowspan=1 colspan=1>1.00E+02</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>1.00E+02</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/25/06/04</td><td rowspan=1 colspan=1>1.00E+00</td></tr></table>

# e. Analytical Reactivity

The analytical reactivity of the Quidel Influenza $\mathbf { A } { + } \mathbf { B }$ assay with various strains of influenza A and influenza B viruses was evaluated on the previously cleared Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform, K112172); the data was reviewed at that time. Since the LoDs of the two instrument platforms are considered to be similar, these studies were not repeated for this submission.

The analytical reactivity, as evaluated on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform (each organism was extracted using the NucliSENS easyMAG (bioMérieux) instrument and tested in triplicate), is shown below.

<table><tr><td rowspan=1 colspan=1>Influenza A Viruses</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1</td><td rowspan=1 colspan=1>H1N1 A/California/07/2009</td><td rowspan=1 colspan=1>1.45E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>1.12E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>3.80E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>5.89E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>1.26E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/FM/1/47</td><td rowspan=1 colspan=1>3.80E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1</td><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>1.40E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A1/Mal/302/54</td><td rowspan=1 colspan=1>4.19E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>3.39E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>7.24E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/Solomon Islands/3/06</td><td rowspan=1 colspan=1>1.41E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>1.15E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>7.24E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>4.17E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>4.57E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>9.83E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Uruguay/7/16/2007</td><td rowspan=1 colspan=1>1.03E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>2.19E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>4.17E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Influenza B Viruses</td></tr><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>cYanmaGellowb</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>B/HongKong/5/72</td><td rowspan=1 colspan=1>6.67E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>1.02E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/2006</td><td rowspan=1 colspan=1>3.16E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>3.80E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/2004</td><td rowspan=1 colspan=1>1.26E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/25/06/04</td><td rowspan=1 colspan=1>3.41E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>1.15E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>4.17E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>1.51E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>2.19E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Mass/3/66</td><td rowspan=1 colspan=1>1.38E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>1.95E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>6.30E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr></table>

The following avian strains were tested with the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ assay as cultured isolates in a BS-3 facility.

<table><tr><td rowspan=1 colspan=5>Restricted Influenza A Viruses</td></tr><tr><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>Results</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/WI/629-2/2008 (H3N2)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/WI/629-S7(D02473)/2009 (H1N1pdm)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/WI/629-S5 (D02312)/2009 (H1N1pdm)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Mallard/NY/6750/78 (H2N2)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H7N3</td><td rowspan=1 colspan=1>A/Chicken/NJ/15086-3/94 (H7N3)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H9N2</td><td rowspan=1 colspan=1>A/Chicken/NJ/12220/97 (H9N2)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H4N8</td><td rowspan=1 colspan=1>A/Mallard/OH/338/86 (H4N8)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H6N2</td><td rowspan=1 colspan=1>A/Chicken/CA/431/00 (H6N2)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H8N4</td><td rowspan=1 colspan=1>A/Blue Winged Teal/LA/B174/86 (H8N4)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG5</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>A/GWT/LA/169GW/88 (H10N7)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H11N9</td><td rowspan=1 colspan=1>A/Chicken/NJ/15906-9/96 (H11N9)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H12N5</td><td rowspan=1 colspan=1>A/Duck/LA/188D/87 (H12N5)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H13N6</td><td rowspan=1 colspan=1>A/Gull/MD/704/77 (H13N6)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H14N5</td><td rowspan=1 colspan=1>A/Mallard/GurjevRussia/262/82 (H14N5)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H15N9</td><td rowspan=1 colspan=1>A/Shearwater/Australia/2576/79 (H15N9)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H16N3</td><td rowspan=1 colspan=1>A/Shorebird/DE/172/2006(H16N3)</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

Additional analytical reactivity studies were performed with two new influenza A variants, H3N2v and H7N9, which emerged between 2012 and 2013.

a. Six isolates of the H3N2v strain were obtained from CDC. The viruses were cultured and titered by Quidel. Each virus was diluted in transport media to reach a $\mathrm { T C I D } _ { 5 0 }$ range of $4 \mathrm { E } { + } 0 1$ to $6 \mathrm { E } { + } 0 1$ , based on the limit of detection $\mathrm { T C I D } _ { 5 0 }$ value of the H3N2 strain evaluated for this submission. Each sample was extracted using the NucliSENS easyMAG instrument and tested in triplicate on two instrument platforms, the Life Technologies QuantStudio Dx and the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx. All six isolates of the influenza A H3N2v virus were detected by the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay using both instruments, at the concentrations shown below.

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strains</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=2>Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Indiana/10/11</td><td rowspan=1 colspan=1>5.90E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Kansas/13/9</td><td rowspan=1 colspan=1>4.90E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Pennsylvania/14/10</td><td rowspan=1 colspan=1>4.80E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Victoria/361/11</td><td rowspan=1 colspan=1>5.20E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Minnesota/11/10</td><td rowspan=1 colspan=1>4.60E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/West Virginia/6/11</td><td rowspan=1 colspan=1>5.00E+01</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

b. To determine the reactivity of the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ assay with the Avian Influenza A H7N9 virus, an inactivated virus was obtained from CDC. The virus was titered by the CDC using Egg Infective Dose $( \mathrm { E I D } ) _ { 5 0 }$ methodology, and was subsequently inactivated with $\beta$ -propiolactone (BPL). The virus was serially diluted (10 fold) in universal transport media (UTM) to reach a $\mathrm { T C I D } _ { 5 0 }$ range of $1 \mathrm { E } { + } 0 2$ to $1 \mathrm { E } { + } 0 3$ . Each dilution was tested in replicates of 10 to estimate the lowest concentration that provided $2 9 5 \%$ detection. Further, two fold dilutions were performed, ranging from $8 . 0 0 \mathrm { E } { + } 0 1$ $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ to $8 . 0 0 \mathrm { E } \mathrm { + } 0 6 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ to determine the LoD. Each dilution was extracted using the NucliSENS easyMAG instrument and tested in 10 to 20 replicates on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx. Based on 20 replicates, the LoD was determined to be $3 . 9 5 \mathrm { E } { + 0 3 }$ $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ when tested using the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ Assay on ABI 7500.

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=2>Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>H7N9</td><td rowspan=1 colspan=1>A/ANHUI/1/2013</td><td rowspan=1 colspan=1>3.95E+03*</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

\* Inactivated virus – relative EID50 Titer/mL

# f. Analytical specificity:

Interference

The effect of potentially interfering substances that may be present in nasopharyngeal specimens on the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay was evaluated on the previously cleared Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform, K112172); the data was reviewed at that time. Since the LoDs of the two instrument platforms are considered to be similar, these studies were not repeated for this submission. The interference studies conducted at that time utilized influenza A (A/Mexico/4108/2009) and influenza B (B/Florida/04/2006) strains at concentrations of 3x LoD on the ABI 7500 platform. No interference was observed when tested in the presence of the following substances at the concentrations shown.

<table><tr><td colspan="1" rowspan="1">Substance Name</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">InfluenzaA (3xLoD)</td><td colspan="1" rowspan="1">Influenza B(3x LoD)</td></tr><tr><td colspan="1" rowspan="1">Mucin (Bovine SubmaxillaryGland, type I-S)</td><td colspan="1" rowspan="1">60 μg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Blood (human), EDTAanticoagulated</td><td colspan="1" rowspan="1">2% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Zicam Homeopathic Non-Drowsy Allergy Relief No Drip</td><td colspan="1" rowspan="1">5% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Liquid Nasal Gel</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">15% (vol/vol) ofdose</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges</td><td colspan="1" rowspan="1">0.68g/mL; 1/18 drop,crushed; activeingredients: 1.7mg/mL menthol</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3 to 5 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4.0 μg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">6.6 to 10 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir phosphate</td><td colspan="1" rowspan="1">7.5 to 25 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr></table>

# Cross-reactivity

The potential for cross-reactivity of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay with microorganisms commonly found in respiratory passages or those causing other respiratory tract infections were evaluated on the previously cleared Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform, K112172); the data was reviewed at that time. Since the LoDs of the two instrument platforms are considered to be similar, these studies were not repeated for this submission. The cross-reactivity studies performed at that time included 24 bacteria, 26 viruses and one yeast strain. Bacteria and yeast were tested at concentrations between $1 0 ^ { 5 }$ and $1 0 ^ { 1 0 } \mathrm { C F U / m L }$ . Viruses were tested at concentrations between ${ { 1 0 } ^ { 3 } }$ to $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Samples were extracted using the NucliSENS easyMAG instrument and tested in triplicate with the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay on the ABI 7500 platform. No cross-reactivity with the organisms tested was observed. The results of that study are presented below.

<table><tr><td colspan="4" rowspan="1">Cross Reactivity</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">FinalConc.</td><td colspan="1" rowspan="1">InfluenzaA Result</td><td colspan="1" rowspan="1">InfluenzaB Result</td></tr><tr><td colspan="1" rowspan="1">hMPV A1</td><td colspan="1" rowspan="1">3.70E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">hMPV B1</td><td colspan="1" rowspan="1">2.37E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV Long</td><td colspan="1" rowspan="1">4.40E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV Washington</td><td colspan="1" rowspan="1">1.75E+03</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid71</td><td colspan="1" rowspan="1">5.67E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus229E</td><td colspan="1" rowspan="1">1.70E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">CoronavirusOC43</td><td colspan="1" rowspan="1">1.67E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.43E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">CoxsackievirusB5/10/2006</td><td colspan="1" rowspan="1">2.28E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">8.76E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">5.38E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">1.50E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.05E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.50E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.68E+03</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.66E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">5,000cp/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 Maclnytrestrain</td><td colspan="1" rowspan="1">1.95E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">3.67E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Rubeola</td><td colspan="1" rowspan="1">3.78E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">8.43E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">2.50E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">2.20E+04</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">9.10E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">9.57E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">7.50E+02</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.04E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetellabronchiseptica</td><td colspan="1" rowspan="1">2.55E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2.10E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">2.05E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumintracellulare</td><td colspan="1" rowspan="1">6.90E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">6.60E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.36E+10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">5.90E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">5.15E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">2.65E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">2.75E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">2.15E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.85E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1.85E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">3.30E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6.80E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.85E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">6.0E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.03E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">4.5E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.05E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">2.50E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">2.6E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">5.15E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.07E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr></table>

# g. Assay cut-off:

The assay cutoff was established to optimize the sensitivity of the assay without compromising the specificity for either influenza A or influenza B detection. The threshold “cutoff value” represents the fluorescent intensity signal (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel. Multiple positive and negative clinical specimens as well as contrived specimens were used to establish instrument baseline and threshold settings. A total of 14 influenza A positive specimens and 12 influenza B positive specimens were evaluated for their Ct ranges to determine the appropriate cutoff for the assay. Based on the data, a cutoff at Ct 40 was chosen.

The distribution of the Ct values for positive clinical samples in this study is shown below.

![](images/d6046e6cd544a501798ddb6a9f49298d6e66f690995b2c2f16ad5ce557f706e0.jpg)

The assay cutoff was validated in the clinical study.

# h. Carryover Contamination

A study was performed to evaluate a potential for carryover cross-contamination with viral RNA when testing positive specimens with a high viral content using the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ assay during extraction on the bioMérieux easyMAG followed by amplification on the QuantStudio Dx platform. The following influenza strains were re-grown and stocks were prepared at the given starting concentration.

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>TCID50Titer</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>2.11E+07</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>1.67E+07</td></tr></table>

The study was performed with two dilutions that were prepared in negative nasal matrix that contained both Influenza A and Influenza B at a high positive concentration $( \mathrm { T C I D } _ { 5 0 }$ $2 . 1 1 \mathrm { E } { + } 0 5$ and $1 . 6 7 \mathrm { E } { + } 0 5$ , respectively) and high negative concentration $( \mathrm { T C I D } _ { 5 0 }$ 2.11E-01 and 1.67E-01, respectively). Viral RNA was extracted and purified using the easyMAG instrument according to the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ instructions for use. The study was conducted over five days with a separate plate prepared for each of the influenza A and B strains. Each day of testing, 24 Positive and 24 Negative replicates were extracted and purified. The extracted samples were aliquoted into the wells of a PCR plate, alternating positive and negative samples for a total of 48 positive and 48 negative samples on each plate. There was no carryover or cross-contamination observed for this assay when using the QuantStudi Dx platform. The summary of results is presented below.

Carryover Study Results Summary   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Total HighPositiveConcentrationDetected asPositive</td><td rowspan=1 colspan=1>CtAve</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AcceptanceCriteria forPositives</td><td rowspan=1 colspan=1>AcceptanceCriteriaMet?</td><td rowspan=1 colspan=1>Total HighNegativeConcentrationDetected asNegative</td><td rowspan=1 colspan=1>AcceptanceCriteria</td><td rowspan=1 colspan=1>AcceptanceCriteriaMet?</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>17.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>≥95%</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>20.6</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>≥95%</td><td rowspan=1 colspan=1>Yes</td></tr></table>

# i. Transport Media

Analytical performance of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay was evaluated with six different transport media in a study submitted in support of the previously cleared Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform, K112172; the data was reviewed at that time. No differences between the tested transport media were observed.

2. Comparison studies:

a. Method comparison with predicate device:

See Clinical studies section below.

b. Matrix comparison:

N/A

3. Clinical studies:

The Quidel Molecular Influenza A and B assay was evaluated in a clinical study using nasal swabs (NS) and nasopharyngeal swab (NPS) specimens. The study utilized fresh, non-frozen specimens prospectively collected between November 2012 and March 2013 from individuals with signs and symptoms of respiratory tract infection. The samples were collected and tested at three geographically diverse clinical sites. The performance of the assay was compared to the test results obtained with a commercially available high-performing, FDA-cleared, Influenza A and B molecular test.

Specimens were collected from 631 patients. Each specimen was extracted with the bioMérieux easyMAG and tested with Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ Assay using the Life Technologies QuantStudio Dx Real-Time PCR Instrument. Each specimen was also tested by the comparator method. Twelve specimens were removed from analysis due to invalid results (four specimens were invalid on initial and repeat testing with the Quidel device $( 0 . 6 3 \%$ , $9 5 \%$ CI: 0.25-1.62) and eight specimens were invalid on initial and repeat testing on the comparator device $( 1 . 2 7 \%$ , $9 5 \%$ CI: $0 . 6 4 \substack { - 2 . 4 8 \% }$ ). Among the 619 subjects with evaluable results, 204 $( 3 3 . 0 \% )$ were positive for influenza A and 107 $( 1 7 . 3 \% )$ were positive for influenza B. Among the 311 subjects infected with influenza $( 5 0 . 2 \%$ of all subjects tested), 11 tested positive for both influenza A and influenza B $( 3 . 5 \% )$ .

Performance characteristics for influenza A were established when influenza A/H3 and influenza A 2009 H1N1 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

The following table summarizes the gender and age distribution of all patients enrolled in the evaluation.

<table><tr><td rowspan=1 colspan=3>Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;5 years</td><td rowspan=1 colspan=1>124 (36.6%)</td><td rowspan=1 colspan=1>103 (35.3%)</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>90 (26.5%)</td><td rowspan=1 colspan=1>103 (35.3%)</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>67 (19.8%)</td><td rowspan=1 colspan=1>47 (16.1%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 60 years</td><td rowspan=1 colspan=1>58 (17.1%)</td><td rowspan=1 colspan=1>39 (13.4%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EPY</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>339</td><td rowspan=1 colspan=1>292</td></tr></table>

The clinical performance of the Quidel Influenza $\mathbf { A } + \mathbf { B }$ Assay when used on the QuantStudio Dx instrument across the three sites combined is shown below.

<table><tr><td rowspan=1 colspan=4>Influenza A</td></tr><tr><td rowspan=1 colspan=4>Comparator: FDA-cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecularon QuantStudio Dx</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>382</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>619</td></tr></table>

Positive Percent Agreement: $100 \%$ (204/204) $9 5 \%$ CI: $9 8 . 2 \%$ to $100 \%$ Negative Percent Agreement: $9 2 . 0 \%$ (382/415) $9 5 \%$ CI: $8 9 . 0 \%$ to $9 4 . 3 \%$

<table><tr><td rowspan=1 colspan=4>Influenza B</td></tr><tr><td rowspan=1 colspan=4>Comparator: FDA-cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecularon QuantStudio Dx</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>116</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>502</td><td rowspan=1 colspan=1>503</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>512</td><td rowspan=1 colspan=1>619</td></tr></table>

Positive Percent Agreement: $9 9 . 1 \%$ (106/107) $9 5 \%$ CI: $9 4 . 9 \%$ to $9 9 . 8 \%$ Negative Percent Agreement: $9 8 . 0 \%$ (502/512) $9 5 \%$ CI: $9 6 . 4 \%$ to $9 8 . 9 \%$

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

In the study conducted during the winter of 2013 (January 2013 to March 2013) with 631 patients (with 619 evaluable specimens) reporting with symptoms of respiratory tract infection, the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay when used with the Life Technologies QuantStudio Dx Real-Time PCR Instrument System detected influenza A in $3 8 . 3 \%$ of patients and influenza B in $1 8 . 7 \%$ of patients. Testing was performed with prospective specimens received from three different regions in the United States. The number and percentage of the influenza A and influenza B RNA positive cases, calculated by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=1 colspan=3>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Numberof Patients</td><td rowspan=1 colspan=1>Number ofPositives</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>NumberofPatients</td><td rowspan=1 colspan=1>NumberofPositives</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>≤ 5 year</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>37.0%</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>16.0%</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>52.3%</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>31.1%</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>28.3%</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>15.0%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>24.5%</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>38.3%</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>18.7%</td></tr></table>

# N. Instrument Name:

QuantStudio™ Dx Real-Time PCR Instrument System

# O. System Descriptions:

The QuantStudio™ Dx instrument is a bench top Real-Time PCR instrument that uses fluorescent-based polymerase chain reaction (PCR) reagents to provide qualitative or quantitative detection of target nucleic acid sequences (targets) using real-time analysis from a 96-well (or 384-well) plate. Samples are prepared for nucleic acid testing manually using specific reagents indicated by the assay.

# 1. Modes of Operation:

The QuantStudio™ Dx Real-Time PCR instrument has two modes of operation: IVD and Research Use Only (RUO), which cannot be executed simultaneously. The protocols that are performed on each instrument for IVD assays depend upon the assay-specific, closedmode application specifications that are installed on the system.

# 2. Software:

The embedded software on the QuantStudio™ Dx manages instrument operations and enables the user to monitor instrument status. The QuantStudio™ Dx instrument software is installed on a separate computer and communicates with the instrument to control the instrument and collect data. The QuantStudio™ Dx instrument software performs data analysis and outputs analyzed results.

The diagnostic test parameters for the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay are predefined by Quidel in a Test Definition Document (TDD) that is imported into the QuantStudio™ Dx instrument software. The TDD file is locked and the parameters cannot be changed by the user.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

3. Specimen Identification: Performed manually

4. Specimen Sampling and Handling:

There is no automated sample processing offered in conjunction with the QuantStudio™ Dx instrument. Patient specimens are prepared (RNA extraction) using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ System (bioMérieux ). The extracted samples are added manually to the 96- well microplate.

# 5. Calibration:

The instrument’s optics (fluorescence parameters) are calibrated upon installation. After installation of the instrument, the user can perform all required regular maintenance and calibration using IVD-labeled calibration plates.

6. Quality Control:

Quality control is addressed for each sample by addition of an in-process control during sample processing and amplification in the assay. Additionally, there are external positive and negative controls available that may be used in accordance with the user lab standards.

# P. Other Supportive Instrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

The QuantStudio Dx software was reviewed in detail under K123955.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.