# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# C. Measurand:

# D. Type of Test:

# E. Applicant:

# G. Regulatory Information:

1. Regulation section: 866.3980 - Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code:

OCC, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay performed on the Abbott $m 2 0 0 0$ System is a qualitative in vitro diagnostic test designed for the detection of influenza A, influenza B, and RSV RNA directly from nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. The assay detects RNA from influenza A, influenza B, and RSV (A and B) using real-time, reverse transcription polymerase chain reaction (rRT-PCR) and fluorogenic target specific hybridization for unique sequences in the viral target genomes. The IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay is intended for use as an aid in diagnosing influenza A and/or influenza B and/or RSV infection.

Negative results for influenza A, influenza B, or RSV do not preclude influenza virus or RSV infection and should not be used as the sole basis for diagnosis, treatment, or patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent(s) detected may not be the definite cause of disease. The use of additional laboratory testing and clinical presentation must be considered when diagnosing respiratory viral infection.

Performance characteristics for influenza A were established during the 2011-2012 and 2012-2013 influenza seasons when Influenza A/2009 H1N1 and Influenza A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use:

Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Abbott $^ \mathrm { \textregistered }$ $\mathfrak { d } m 2 0 0 0 ^ { \mathrm { T M } }$ System

# I. Device Description:

The IMDx Flu $\mathrm { A } / \mathrm { B }$ , and RSV for Abbott $m 2 0 0 0$ assay uses nucleic acid extraction and purification technology, performed on the Abbott $m 2 0 0 0$ Sample Preparation System $( m 2 0 0 0 { \mathrm { s p } } )$ , combined with rRT-PCR, performed on the Abbott PCR analyzer (m2000rt), to generate and detect amplified products from influenza A, influenza B, and RSV RNA that is isolated from clinical specimens. The presence of a viral RNA target sequence is indicated by the fluorescent signal generated through the use of fluorescently labeled oligonucleotide probes on the Abbott $m 2 0 0 0 \mathrm { r t }$ instrument. The probes do not generate a signal unless they are specifically bound to the amplified product. The amplification cycle at which fluorescent signal is detected by the Abbott $m 2 0 0 0 \mathrm { r t }$ is inversely proportional to the viral RNA target concentration present in the original sample.

The IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay consists of two reagent kits: • IMDx Flu A/B and RSV for Abbott m2000 Amplification Reagent Kit IMDx Flu A/B and RSV for Abbott m2000 Control Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene $^ \mathrm { \textregistered }$ Respiratory Virus Plus Nucleic Acid Test on the Verigene $^ \mathrm { \textregistered }$ System $\left( \mathrm { R V + } \right)$

2. Predicate 510(k) number(s): K103209

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The IMDx Flu A/B and RSV forAbbott m2000 assay performedon the Abbott m2000 System is aqualitative in vitro diagnostic test</td><td colspan="1" rowspan="1">The Verigene® Respiratory VirusPlus Nucleic Acid Test (RV+)on the Verigene® System is aqualitative nucleic acid</td></tr><tr><td colspan="3"></td></tr><tr><td>Item</td><td>Similarities Device</td><td>Predicate multiplex test intended to</td></tr><tr><td></td><td>designed for the detection of influenza A, influenza B, and RSV RNA directly from nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. The assay detects RNA from influenza A, influenza B, and RSV (A and B) using real-time, reverse transcription polymerase chain reaction (rRT-PCR) and fluorogenic target specific hybridization for unique sequences in the viral target genomes. The IMDx Flu A/B and for influenza A were established during the 2011- 2012 and 2012-2013</td><td>simultaneously detect and identify multiple respiratory virus nucleic acids in nasopharyngeal (NP) swab specimens from individuals with signs and symptoms of respiratory tract infection. The following virus types and subtypes are identified using the RV+: Influenza A, Influenza A subtype H1, Influenza A subtype H3, 2009 H1N1, Influenza B, be the definite cause of disease. The use of additional laboratory</td></tr><tr><td>infection.</td><td>RSV for Abbott m2000 assay is intended for use as an aid in diagnosing influenza A and/or influenza B and/or RSV Negative results for influenza A, influenza B, or RSV do not preclude influenza virus or RSV infection and should not be used as the sole basis for diagnosis, treatment, or patient management decisions. Conversely, positive</td><td>Respiratory Syncytial Virus (RSV) subtype A, and RSV subtype B. The test is not intended to detect Influenza C virus. Detecting and identifying specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection, if used in conjunction with other clinical</td></tr><tr><td>other viruses. The agent(s) cause of disease. The use of clinical presentation must be considered when diagnosing respiratory viral infection. Performance characteristics</td><td>results do not rule-out bacterial infection or co-infection with detected may not be the definite additional laboratory testing and</td><td>and laboratory findings. Negative results for influenza A, influenza B, or RSV do not preclude influenza virus or RSV infection and should not be used as the sole basis for diagnosis, treatment, or patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A/2009 H1N1 andInfluenza A/H3 were thepredominant influenza Aviruses in circulation. Whenother influenza A viruses areemerging, performancecharacteristics may vary.If infection with a novelinfluenza A virus is suspectedbased on current clinical andepidemiological screeningcriteria recommended by publichealth authorities, specimensshould be collected withappropriate infection controlprecautions for novel virulentinfluenza viruses and sent to stateor local health department fortesting. Viral culture should notbe attempted in these casesunless a BSL 3+ facility isavailable to receive and culturespecimens.</td><td colspan="1" rowspan="1">respiratory viral infection.Performance characteristics forinfluenza A Virus were establishedwhen influenza A/H3, A/H1, and2009 H1N1 were the predominantinfluenza A viruses circulating.These characteristics may varywhen other influenza A viruses areemerging.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiologicalscreening criteria recommended bypublic health authorities, specimensshould be collected with appropriateinfection control precautions usedspecifically for novel virulentinfluenza viruses and sent to state orlocal health department for testing.Viral culture should not beattempted in these cases unless aBSL 3+ facility is available toreceive and culture specimens.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">SamplePreparation</td><td colspan="1" rowspan="1">Automated extraction of nucleicacids</td><td colspan="1" rowspan="1">Automated extraction of nucleicAcids</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Real-time, reverse transcriptionpolymerase chain reaction(rRT- PCR) DNAamplification</td><td colspan="1" rowspan="1">Real-time, reverse transcriptionpolymerase chain reaction(RT- PCR) DNAamplification</td></tr><tr><td colspan="1" rowspan="1">TargetsDetected</td><td colspan="1" rowspan="1">influenza Ainfluenza BRSV A/B</td><td colspan="1" rowspan="1">influenza Ainfluenza BRSV ARSV B</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Positive ControlNegative ControlProcess Control</td><td colspan="1" rowspan="1">Positive ControlNegative ControlInhibition ControlInternal Control</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Abbott® m2000TM System(K092705)</td><td colspan="1" rowspan="1">Verigene® System(K070597)</td></tr><tr><td colspan="1" rowspan="1">Throughput</td><td colspan="1" rowspan="1">Batch</td><td colspan="1" rowspan="1">Single use cassette</td></tr><tr><td colspan="1" rowspan="1">Viral Sub-Typing</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">Yes</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff: Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay. Issued October 9, 2009 Center for Devices and Radiological Health, FDA/HHS

Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses. Issued July 15, 2011. Center for Devices and Radiological Health, FDA/HHS

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable - Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff

FDA document #337, Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices - Guidance for Industry and FDA Staff

FDA document #1638, Establishing the Performance Characteristics of IVDs for the Detection and Differentiation of Influenza Viruses, February 15, 2008

FDA document #1594, In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path, May 2, 2007

Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems, March 10, 2005

CLSI EP07-A2; Interference Testing in Clinical Chemistry; Approved Guideline – Second Edition, 2005

CLSI EP05-A2; Evaluation of Precision Performance of Quantitative Measurement Methods;   
Approved Guideline – Second Edition.

# L. Test Principle:

IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay enables detection of influenza A Virus, influenza B Virus, RSV A/B, and Process Control through the following workflow:

# Sample Preparation

The Abbott $m 2 0 0 0 s p$ reagents lyse the virus, capture the nucleic acids with magnetic microparticles, and wash the particles to remove unbound sample components. The bound nucleic acids are eluted and transferred to an Abbott 96-Deep-Well Plate. The nucleic acids are then ready for amplification. The Process Control is introduced into the sample preparation procedure and is processed along with the controls and specimens.

# Reagent Preparation and Reaction Plate Assembly

The Abbott $m 2 0 0 0 s p$ combines the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ Amplification Reagent components (IMDx Flu A/B and RSV Amplification Reagent and IMDx PCR Reagent-B). It then dispenses the resulting Master Mix into the Abbott 96-Well Optical Reaction Plate along with aliquots of the nucleic acid samples prepared by the Abbott $m 2 0 0 0 s p$ . After manual application of the Abbott Optical Adhesive Cover, the plate is ready for transfer to the Abbott $m 2 0 0 0 r t$ instrument.

# Amplification on the Abbott $m 2 0 0 0 r t$ instrument

During the reverse-transcription reaction, viral target RNA is reverse-transcribed into cDNA by the reverse-transcriptase activity of rTth polymerase enzyme. During the amplification/detection reaction, the target cDNA is amplified by the DNA polymerase activity of the rTth enzyme, in the presence of deoxynucleotide triphosphates (dNTPs) and manganese. The IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ Amplification Reagent contains specific sets of amplification primers for influenza A, influenza B, RSV A, RSV B and Process Control. During PCR amplification, high temperature is used to separate the strands of double stranded DNA. When the reaction is cooled to a temperature at which DNA annealing can again occur, the analyte-specific, single-stranded DNA oligonucleotide primers bind to the analyte DNA. The primers are extended by DNA polymerase, thereby making an exact copy of a short stretch of the analyte DNA target region.

During each round of thermal cycling, amplification products dissociate to single strands at high temperature, allowing primer annealing and extension as the temperature is lowered. Exponential amplification of the target is achieved through repeated cycling between higher and lower temperatures. Amplification of the influenza A, influenza B, RSV A, RSV B, and Process Control RNA targets takes place simultaneously in the same reaction.

# Detection

During each round of PCR amplification, the fluorescent probes anneal to the amplified target DNA. The probes are labeled with different fluorescent molecules allowing influenza A, influenza B, RSV and Process Control RNA targets to be distinguished from each other. RSV A and RSV B are both detected using the same fluorophore and are not discriminated from one another. The probes are single-stranded, linear DNA oligonucleotides modified with a fluorescent moiety covalently linked to one end of the probe and a quenching moiety to the other end. When the probe binds to its complementary sequence in the target, the fluorophore is released, allowing fluorescent emission and detection.

Since this fluorescence occurs during every cycle, the PCR reaction can be read in real-time. The amplification cycle at which fluorescent signal is detected by the Abbott $m 2 0 0 0 r t$ is inversely proportional to the viral RNA target concentration present in the original sample.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Within-Laboratory Repeatability:

For within-laboratory repeatability study, a sixteen-member panel was tested twice a day for twelve days. The panel was prepared using five viral strains; two influenza A strains (Influenza A/Swine/NY/02/09 (H1N1) and Influenza A/Brisbane/10/07 (H3N2)), one influenza B strain (Influenza B/Florida/04/2006), and two RSV strains (RSV A (RSV Type A) and RSV B (RSV B CH93-(18)-18)). Each strain was spiked into viral transport medium targeting three concentration levels: Moderate Positive ( $2 { - } 3 \mathbf { x }$ LoD), Low Positive (1x LoD) and High Negative $( 0 . 2 \mathrm { x } - 0 . 8 \mathrm { x }$ LoD). Panel members were tested in replicates of three for each run (for a total of 1152 data points for the 24 runs). Percent CV $( \% C V )$ was calculated for all replicates based on CN (cycle number) values. Percent agreement was calculated based on the expected result (detected or not detected) for each panel member. Moderate and low positive samples were expected to be reported as detected. The high negative was represented by a near cut-off sample since a $\mathrm { C } _ { 5 }$ sample (results are detected ${ \sim } 5 \%$ of the time) is difficult to obtain for an ultrasensitive test. The result with such a high negative was expected to be reported as detected only $20 \%$ of the time, and negative samples were expected to be reported as not detected.

Within-Laboratory Precision/Repeatability   

<table><tr><td colspan="1" rowspan="2">Panel member</td><td colspan="3" rowspan="2">Flu A/B and RSV CNAVG  SD  % CV</td><td colspan="3" rowspan="1">Process control CN</td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="2">95% CI</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">AVG</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">Agreement withexpected results</td></tr><tr><td colspan="1" rowspan="1">Flu A H1N1 HighNegative</td><td colspan="1" rowspan="1">37.60</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">1.84%</td><td colspan="1" rowspan="1">28.63</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.72%</td><td colspan="1" rowspan="1">61.97% (44/71*)</td><td colspan="1" rowspan="1">50.34-72.37</td></tr><tr><td colspan="1" rowspan="1">Flu A H1N1 LowPositive</td><td colspan="1" rowspan="1">35.17</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">1.83%</td><td colspan="1" rowspan="1">28.65</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.66%</td><td colspan="1" rowspan="1">98.57% (69/70*)</td><td colspan="1" rowspan="1">92.34-99.75</td></tr><tr><td colspan="1" rowspan="1">FLU A H1N1Moderate Positive</td><td colspan="1" rowspan="1">34.04</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.62%</td><td colspan="1" rowspan="1">28.69</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.74%</td><td colspan="1" rowspan="1">100.00% (72/72)</td><td colspan="1" rowspan="1">94.93-100.00</td></tr><tr><td colspan="1" rowspan="1">FLU A H3N2 HighNegative</td><td colspan="1" rowspan="1">37.53</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.41%</td><td colspan="1" rowspan="1">28.80</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.56%</td><td colspan="1" rowspan="1">76.39% (55/72)</td><td colspan="1" rowspan="1">65.40-84.70</td></tr><tr><td colspan="1" rowspan="1">FLU A H3N2 LowPositive</td><td colspan="1" rowspan="1">35.80</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.37%</td><td colspan="1" rowspan="1">28.81</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.55%</td><td colspan="1" rowspan="1">98.61% (71/72)</td><td colspan="1" rowspan="1">92.54-99.75</td></tr><tr><td colspan="1" rowspan="1">FLU A H3N2Moderate Positive</td><td colspan="1" rowspan="1">34.51</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.35%</td><td colspan="1" rowspan="1">28.83</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.48%</td><td colspan="1" rowspan="1">100.00% (71/71*)</td><td colspan="1" rowspan="1">94.87-100</td></tr><tr><td colspan="1" rowspan="1">FLU B HighNegative</td><td colspan="1" rowspan="1">36.68</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.35%</td><td colspan="1" rowspan="1">28.79</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.59%</td><td colspan="1" rowspan="1">9.72% (7/72)</td><td colspan="1" rowspan="1">4.79-18.74</td></tr><tr><td colspan="1" rowspan="1">FFLU B LowPositive</td><td colspan="1" rowspan="1">35.34</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.80%</td><td colspan="1" rowspan="1">28.79</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.54%</td><td colspan="1" rowspan="1">97.22% (70/72)</td><td colspan="1" rowspan="1">90.43-99.23</td></tr><tr><td colspan="1" rowspan="1">FLU B ModeratePositive</td><td colspan="1" rowspan="1">33.29</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.89%</td><td colspan="1" rowspan="1">28.75</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.56%</td><td colspan="1" rowspan="1">100.00% (72/72)</td><td colspan="1" rowspan="1">94.93-100.00</td></tr><tr><td colspan="1" rowspan="1">RSVA High Neegative</td><td colspan="1" rowspan="1">36.80</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.13%</td><td colspan="1" rowspan="1">28.79</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.69%</td><td colspan="1" rowspan="1">31.94% (23/72)</td><td colspan="1" rowspan="1">22.33-43.39</td></tr><tr><td colspan="1" rowspan="1">SVA LowPositive</td><td colspan="1" rowspan="1">35.72</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">1.94%</td><td colspan="1" rowspan="1">28.77</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.54%</td><td colspan="1" rowspan="1">100.00% (72/72)</td><td colspan="1" rowspan="1">94.93-100.00</td></tr><tr><td colspan="1" rowspan="1">RSVA ModeratePositive</td><td colspan="1" rowspan="1">34.95</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">2.37%</td><td colspan="1" rowspan="1">28.71</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.62%</td><td colspan="1" rowspan="1">97.22% (70/72)</td><td colspan="1" rowspan="1">90.43-99.23</td></tr><tr><td colspan="1" rowspan="1">RSVB High Negative</td><td colspan="1" rowspan="1">38.40</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">1.51%</td><td colspan="1" rowspan="1">28.77</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.73%</td><td colspan="1" rowspan="1">36.11% (26/72)</td><td colspan="1" rowspan="1">25.98-47.65</td></tr><tr><td colspan="1" rowspan="1">RSVB LowPositive</td><td colspan="1" rowspan="1">36.98</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.22%</td><td colspan="1" rowspan="1">28.75</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.84%</td><td colspan="1" rowspan="1">94.44% (68/72)</td><td colspan="1" rowspan="1">86.57-97.82</td></tr><tr><td colspan="1" rowspan="1">RSVB ModeratePositive</td><td colspan="1" rowspan="1">33.81</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">1.28%</td><td colspan="1" rowspan="1">28.78</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.64%</td><td colspan="1" rowspan="1">100.00% (72/72)</td><td colspan="1" rowspan="1">94.93-100.00</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">28.77</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.69%</td><td colspan="1" rowspan="1">100.00% (72/72)</td><td colspan="1" rowspan="1">94.93-100.00</td></tr></table>

\*Samples for which less than 72 replicates were run were the result of instrument or assay processing errors for individual samples. Those samples were withdrawn from the study.

# Site-to-Site Reproducibility

Site-to-site reproducibility was conducted at three sites, by two operators at each site, with each operator performing one run each day. The same sixteen-member panel that was used in the within-laboratory repeatability study was tested in the site-to-site reproducibility study. Each panel member was tested in replicates of three, twice a day for five days, for a total of 10 experiment runs.

Site-to-Site Reproducibility Data   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">Site 1</td><td colspan="2" rowspan="1">Site 2</td><td colspan="2" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All 3 Sites</td><td colspan="1" rowspan="1">tes</td></tr><tr><td colspan="1" rowspan="1">SpecificPanelMember</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">%Agreement(Agreement</td><td colspan="1" rowspan="1">Avg.CN(%CV)</td><td colspan="1" rowspan="1">%Agreement(Agreement</td><td colspan="1" rowspan="1">Avg.CN(%CV)</td><td colspan="1" rowspan="1">%Agreement(Agreement</td><td colspan="1" rowspan="1">Avg.CN(%CV)</td><td colspan="1" rowspan="1">%Agreement(95% CI)</td><td colspan="1" rowspan="1">Avg.CN(%CV)</td></tr><tr><td colspan="1" rowspan="3">INFA(H1N1)</td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">33.52(2.46)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">34.28(2.17)</td><td colspan="1" rowspan="1">93.33(28/30)</td><td colspan="1" rowspan="1">33.76(2.58)</td><td colspan="1" rowspan="1">97.78(94.73 100.00)</td><td colspan="1" rowspan="1">33.85(2.28)</td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">34.95(1.67)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">35.83(2.41)</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">35.16(3.21)</td><td colspan="1" rowspan="1">97.78(94.73 100.00)</td><td colspan="1" rowspan="1">35.31(2.35)</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">36.67(11/30)</td><td colspan="1" rowspan="1">36.92(4.67)</td><td colspan="1" rowspan="1">33.33(10/30)</td><td colspan="1" rowspan="1">37.88(1.99)</td><td colspan="1" rowspan="1">33.33(10/30)</td><td colspan="1" rowspan="1">37.43(3.31)</td><td colspan="1" rowspan="1">34.44(24.63 44.26)</td><td colspan="1" rowspan="1">37.38(3.04)</td></tr><tr><td colspan="1" rowspan="2">INFA(H3N2)</td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">33.26(1.77)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">33.65(1.37)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">33.31(1.59)</td><td colspan="1" rowspan="1">100.00(100.00 -100.00)</td><td colspan="1" rowspan="1">33.41(1.53)</td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">34.61(1.29)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">34.85(1.22)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">35.04(1.96)</td><td colspan="1" rowspan="1">100.00(100.00 -100.00)</td><td colspan="1" rowspan="1">34.83(1.50)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">6.67(2/30)</td><td colspan="1" rowspan="1">36.53(1.41)</td><td colspan="1" rowspan="1">16.67(5/30)</td><td colspan="1" rowspan="1">37.31(1.81)</td><td colspan="1" rowspan="1">20.00(6/30)</td><td colspan="1" rowspan="1">37.28(1.88)</td><td colspan="1" rowspan="1">14.44(7.18 -21.71)</td><td colspan="1" rowspan="1">37.04(1.72)</td></tr><tr><td colspan="1" rowspan="3">INFB</td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">32.25(1.37)</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">32.18(1.33)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">32.22(0.79)</td><td colspan="1" rowspan="1">98.89(96.72 100.00)</td><td colspan="1" rowspan="1">32.22(1.13)</td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">33.91(0.81)</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">33.89(1.21)</td><td colspan="1" rowspan="1">73.33(22/30)</td><td colspan="1" rowspan="1">33.89(1.00)</td><td colspan="1" rowspan="1">88.89(82.40 95.38)</td><td colspan="1" rowspan="1">33.90(1.01)</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">30.00(9/30)</td><td colspan="1" rowspan="1">34.53(1.07)</td><td colspan="1" rowspan="1">36.67(11/30)</td><td colspan="1" rowspan="1">34.74(0.90)</td><td colspan="1" rowspan="1">53.33(16/30)</td><td colspan="1" rowspan="1">34.50(1.07)</td><td colspan="1" rowspan="1">40.00(29.88 50.12)</td><td colspan="1" rowspan="1">34.53(0.93)</td></tr><tr><td colspan="1" rowspan="3">RSVA</td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">32.62(3.52)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">32.10(4.82)</td><td colspan="1" rowspan="1">93.33(28/30)</td><td colspan="1" rowspan="1">32.47(2.60)</td><td colspan="1" rowspan="1">97.78(94.73 100.00)</td><td colspan="1" rowspan="1">32.39(3.64)</td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">33.63(2.15)</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">33.57(2.47)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">33.15(3.21)</td><td colspan="1" rowspan="1">97.78(94.73 100.00)</td><td colspan="1" rowspan="1">33.45(2.52)</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">6.67(2/30)</td><td colspan="1" rowspan="1">34.93(2.80)</td><td colspan="1" rowspan="1">6.67(2/30)</td><td colspan="1" rowspan="1">35.03(1.90)</td><td colspan="1" rowspan="1">23.33(7/30)</td><td colspan="1" rowspan="1">34.50(3.07)</td><td colspan="1" rowspan="1">12.22(5.46 18.99)</td><td colspan="1" rowspan="1">34.82(2.48)</td></tr><tr><td colspan="1" rowspan="3">RSVB</td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">96.67(29/30)</td><td colspan="1" rowspan="1">32.68(1.90)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">32.39(2.14)</td><td colspan="1" rowspan="1">90.00(27/30)</td><td colspan="1" rowspan="1">31.78(2.54)</td><td colspan="1" rowspan="1">95.56(91.30 99.81)</td><td colspan="1" rowspan="1">32.29(2.18)</td></tr><tr><td colspan="1" rowspan="1">LowPossitive</td><td colspan="1" rowspan="1">86.67(26/30)</td><td colspan="1" rowspan="1">35.20(1.91)</td><td colspan="1" rowspan="1">86.67(26/30)</td><td colspan="1" rowspan="1">35.21(1.50)</td><td colspan="1" rowspan="1">70.00(21/30)</td><td colspan="1" rowspan="1">34.63(2.14)</td><td colspan="1" rowspan="1">81.11(73.02 89.20)</td><td colspan="1" rowspan="1">35.02(1.83)</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">70.00(21/30)</td><td colspan="1" rowspan="1">36.78(0.95)</td><td colspan="1" rowspan="1">86.67(26/30)</td><td colspan="1" rowspan="1">36.55(1.20)</td><td colspan="1" rowspan="1">80.00(24/30)</td><td colspan="1" rowspan="1">36.06(1.76)</td><td colspan="1" rowspan="1">78.89(70.46 87.32)</td><td colspan="1" rowspan="1">36.23(0.83)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">-1.00(0.00)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">-1.00(0.00)</td><td colspan="1" rowspan="1">100.00(30/30)</td><td colspan="1" rowspan="1">-1.00(0.00)</td><td colspan="1" rowspan="1">100.00(100.00 100.00)</td><td colspan="1" rowspan="1">-1.00(0.00)</td></tr></table>

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Specimen Stability at $2 { - } 8 ^ { \circ } \mathrm { C }$

This study was performed using contrived specimens containing one strain each of influenza A, influenza B, and RSV. Samples were tested prior to refrigeration (Day 0) to provide a baseline value and then tested on days 1, 4, 7, and 8. Samples were tested in replicates of 20 on Day 0, and replicates of 3 at all other test time points. All samples showed $100 \%$ detection for all time points. A summary of the average CN values for each time point, virus, and medium tested for storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ is shown in three tables below.

Average CN Values and ΔCN Compared to Day 0 for Influenza A Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Day 0</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Day 7</td><td rowspan=1 colspan=1>Day 8</td></tr><tr><td rowspan=1 colspan=5>Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>34.24 (0.24)</td><td rowspan=1 colspan=1>33.40 (0.60)</td><td rowspan=1 colspan=1>33.96 (0.04)</td><td rowspan=1 colspan=1>33.32 (0.68)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>34.31 (0.45)</td><td rowspan=1 colspan=1>35.03 (1.17)</td><td rowspan=1 colspan=1>33.92 (0.06)</td><td rowspan=1 colspan=1>33.72 (0.14)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>34.11 (0.16)</td><td rowspan=1 colspan=1>33.85 (0.42)</td><td rowspan=1 colspan=1>33.93 (0.34)</td><td rowspan=1 colspan=1>34.31 (0.04)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>34.61 (0.41)</td><td rowspan=1 colspan=1>34.04 (0.16)</td><td rowspan=1 colspan=1>33.63 (0.57)</td><td rowspan=1 colspan=1>33.80 (0.40)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.65</td><td rowspan=1 colspan=1>34.07 (0.58)</td><td rowspan=1 colspan=1>33.58 (1.07)</td><td rowspan=1 colspan=1>33.93 (0.72)</td><td rowspan=1 colspan=1>33.82 (0.83)</td></tr></table>

Average CN Values and ΔCN Compared to Day 0 for Influenza B Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td>Influenza B</td><td>Day 0 |</td><td>Day 1</td><td>| Day 4</td><td>| Day 7</td><td>Day 8</td></tr><tr><td>M4</td><td>31.43</td><td>31.09 (0.33)</td><td>Average CN (∆CN) 31.09 (0.33)</td><td>32.20 (0.77)</td><td>31.23 (0.19)</td></tr><tr><td>M4RT</td><td>31.67</td><td>31.59(0.09)</td><td>31.59 (0.09)</td><td>32.11 (0.43)</td><td>31.62 (0.06)</td></tr><tr><td>M5</td><td>31.66</td><td>31.87 (0.20)</td><td>31.87 (0.20)</td><td>31.79 (0.12)</td><td>31.64 (0.02)</td></tr><tr><td>M6</td><td>31.86</td><td>32.40 (0.54)</td><td>31.98 (0.12)</td><td>31.71 (0.15)</td><td>31.84 (0.01)</td></tr><tr><td>UTM</td><td>31.86</td><td>32.46 (0.60)</td><td>31.90 (0.04)</td><td>32.28 (0.41)</td><td>31.93 (0.07)</td></tr></table>

Average CN Values and ΔCN Compared to Day 0 for RSV Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>Day 0</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Day 7</td><td rowspan=1 colspan=1>Day 8</td></tr><tr><td rowspan=1 colspan=5>Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>34.91 (0.84)</td><td rowspan=1 colspan=1>33.64 (0.43)</td><td rowspan=1 colspan=1>34.65 (0.58)</td><td rowspan=1 colspan=1>34.11 (0.05)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>34.72</td><td rowspan=1 colspan=1>35.74 (1.02)</td><td rowspan=1 colspan=1>34.50 (0.22)</td><td rowspan=1 colspan=1>34.88 (0.16)</td><td rowspan=1 colspan=1>34.09 (0.63)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>34.50 (0.13)</td><td rowspan=1 colspan=1>34.66 (0.29)</td><td rowspan=1 colspan=1>34.82 (0.45)</td><td rowspan=1 colspan=1>34.56 (0.20)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>35.30 (0.77)</td><td rowspan=1 colspan=1>34.92 (0.39)</td><td rowspan=1 colspan=1>34.83 (0.30)</td><td rowspan=1 colspan=1>34.73 (0.21)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.26</td><td rowspan=1 colspan=1>34.45 (0.19)</td><td rowspan=1 colspan=1>33.38 (0.88)</td><td rowspan=1 colspan=1>34.43 (0.17)</td><td rowspan=1 colspan=1>33.65 (0.61)</td></tr></table>

# Specimen Stability at $- 2 0 ^ { \circ } \mathrm { C }$

Samples and media controls were tested prior to freezing (Day 0) to provide a baseline value and then tested on days 4, 8, 14, and 35. Samples were tested in replicates of 20 for Day 0, and 3 replicates for all other time points. The specimens were stored in a freezer with a target temperature of ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ . Temperatures were recorded throughout the study duration, and the temperature readings were

between ${ } _ { - 2 3 } \mathrm { { } ^ { \circ } C }$ and ${ } _ { - 1 4 ^ { \circ } \mathrm { C } }$ . All samples showed $100 \%$ detection for all time points.

Average CN Values and ΔCN Compared to Day 0 for Influenza A Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Day 0</td><td rowspan=1 colspan=1>Day4      ]</td><td rowspan=1 colspan=1>Day8    |</td><td rowspan=1 colspan=1>Day14</td><td rowspan=1 colspan=1>Day35</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>34.15 (0.15)</td><td rowspan=1 colspan=1>33.57 (0.43)</td><td rowspan=1 colspan=1>33.90 (0.10)</td><td rowspan=1 colspan=1>33.55 (0.45)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>34.46 (0.60)</td><td rowspan=1 colspan=1>33.35 (0.51)</td><td rowspan=1 colspan=1>34.38 (0.52)</td><td rowspan=1 colspan=1>33.95 (0.09)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>34.98 (0.71)</td><td rowspan=1 colspan=1>33.30 (0.97)</td><td rowspan=1 colspan=1>34.04 (0.23)</td><td rowspan=1 colspan=1>34.60 (0.33)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>35.04 (0.84)</td><td rowspan=1 colspan=1>33.49(0.71)</td><td rowspan=1 colspan=1>33.96 (0.24)</td><td rowspan=1 colspan=1>34.14 (0.06)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.65</td><td rowspan=1 colspan=1>33.69 (0.96)</td><td rowspan=1 colspan=1>33.73 (0.92)</td><td rowspan=1 colspan=1>33.95 (0.70)</td><td rowspan=1 colspan=1>33.93 (0.72)</td></tr></table>

Average CN Values and ΔCN Compared to Day 0 for Influenza B Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=2 colspan=1>Day 0</td><td rowspan=1 colspan=1>Day4     I</td><td rowspan=1 colspan=1>Day8     |</td><td rowspan=1 colspan=1>Day14</td><td rowspan=1 colspan=1>Day35</td></tr><tr><td rowspan=1 colspan=4>Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>31.43</td><td rowspan=1 colspan=1>31.61 (0.18)</td><td rowspan=1 colspan=1>31.69 (0.26)</td><td rowspan=1 colspan=1>32.17 (0.74)</td><td rowspan=1 colspan=1>31.89 (0.46)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>31.67</td><td rowspan=1 colspan=1>31.57 (0.10)</td><td rowspan=1 colspan=1>31.68 (0.01)</td><td rowspan=1 colspan=1>33.21 (1.54)</td><td rowspan=1 colspan=1>31.37 (0.30)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>31.66</td><td rowspan=1 colspan=1>32.15(0.49)</td><td rowspan=1 colspan=1>31.77(0.11)</td><td rowspan=1 colspan=1>32.33(0.67)</td><td rowspan=1 colspan=1>31.28 (0.38)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>31.87(0.01)</td><td rowspan=1 colspan=1>32.05(0.19)</td><td rowspan=1 colspan=1>32.81(0.95)</td><td rowspan=1 colspan=1>31.47 (0.39)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>32.15 (0.29)</td><td rowspan=1 colspan=1>31.69(0.17)</td><td rowspan=1 colspan=1>33.02 (1.16)</td><td rowspan=1 colspan=1>31.90 (0.03)</td></tr></table>

Average CN Values and ΔCN Compared to Day 0 for RSV Specimens at Given Time Points in Various Viral Transport Media   

<table><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=2 colspan=1>Day 0</td><td rowspan=1 colspan=1>Day4     </td><td rowspan=1 colspan=1>Day8      /</td><td rowspan=1 colspan=1>Day14     I</td><td rowspan=1 colspan=1>Day35</td></tr><tr><td rowspan=1 colspan=4>/Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>34.41 (0.34)</td><td rowspan=1 colspan=1>33.34 (0.73)</td><td rowspan=1 colspan=1>36.91 (2.84)</td><td rowspan=1 colspan=1>34.20 (0.14)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>34.72</td><td rowspan=1 colspan=1>33.60 (1.12)</td><td rowspan=1 colspan=1>34.62 (0.10)</td><td rowspan=1 colspan=1>36.50 (1.79)</td><td rowspan=1 colspan=1>34.56 (0.16)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>34.57 (0.20)</td><td rowspan=1 colspan=1>34.77 (0.40)</td><td rowspan=1 colspan=1>35.94 (1.57)</td><td rowspan=1 colspan=1>34.63 (0.26)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>34.81 (0.28)</td><td rowspan=1 colspan=1>34.70 (0.17)</td><td rowspan=1 colspan=1>36.72 (2.19)</td><td rowspan=1 colspan=1>34.71 (0.18)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.26</td><td rowspan=1 colspan=1>34.13 (0.13)</td><td rowspan=1 colspan=1>33.97 (0.29)</td><td rowspan=1 colspan=1>34.78 (0.52)</td><td rowspan=1 colspan=1>32.93 (1.33)</td></tr></table>

# d. Detection limit:

The Limit of Detection (LoD) was determined by limiting dilution studies using recultured and re-titered viral stocks provided by an outside vendor. Two strains of each viral type: influenza A, influenza B, and RSV, were tested. Six 10-fold dilutions of each strain were prepared using viral transport medium. The concentration generating $2 9 5 \%$ positivity is determined and further tested in replicates of 20. LoD was defined as the lowest concentration to yield 19/20 positive replicates.

LoD for each strain tested   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/Swine/NY/02/2009 (H1N1)</td><td rowspan=1 colspan=1>3.90 x 100</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/Brisbane/10/2007 (H3N2)</td><td rowspan=1 colspan=1>1.51 x 101</td></tr><tr><td rowspan=1 colspan=1>Influenza B B/Florida/04/2006 (YA/88)</td><td rowspan=1 colspan=1>2.82 x 10-2</td></tr><tr><td rowspan=1 colspan=1>Influenza B B/Malaysia/2506/2004 (VI/87)</td><td rowspan=1 colspan=1>3.62 x 10-1</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A RSVA Type A</td><td rowspan=1 colspan=1>4.17 x 100</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B RSVB CH93-(18)-78</td><td rowspan=1 colspan=1>1.65 x 100</td></tr></table>

# e. Analytical reactivity:

A panel of 55 strains was tested for their ability to be detected by the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay. Each strain was diluted in viral transport media and tested in triplicate.

Strains tested in Reactivity Studies   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Concentration Detected</td></tr><tr><td colspan="1" rowspan="1">A/California/7/2009</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">1.07 x 102 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">2.62 x 101 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">6.19 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">1.56 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Canada/6294/09</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">6.57 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">2.49 x 101 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/NJ/8/76</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">5.91 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/NWS/33</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">4.05 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/WS/33</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">4.88 x 10-1 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/FM/1/47</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">1.20 x 101 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mal/302/54</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">1.86 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Denver/1/57</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">7.48 x 101 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Virginia/ATCC2/09</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">5.91 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">4.85 x 101 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">FLU A MRC 2</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">1.56 x 102 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/26</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">2.85 x 102 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">8.87 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">3.64 x 102 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Perth/16/09</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">3.79 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">5.10 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Rhode Island/01/2010</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">1.50 x 10' TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/New York/55/2004</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">2.58 x 10- TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Uruguay/716/2007</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">9.79 x 102 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Florida/2/2006</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">2.28 x 102 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/361/2011</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">3.84 x 102 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/10/2011</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">4.61 x 102 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Texas/71/2007</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">1.56 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/08/2011</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">2.60 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Mass/3/66</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.78 x 10' CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.00 x 104 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.00 x 10' CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.00 x 104 CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.82 x 10' TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Florida/02/2006</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">3.43 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/HONGKONG/5/72</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.19 x 10' CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/RUSSIA/69</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">9.99 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/TAIWAN/2/62 (93-02)</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.00 x 10- CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.00 x 10°CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/01/2010</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">3.20 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Santiago/4360/2007</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.61 x 10° CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Texas/39/2006</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">5.91 x 10' CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Ohio/01/2005</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.19 x 10* CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">B/Brisbane/60/08</td><td colspan="1" rowspan="1">influenza B</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">2.01 x 10-1 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV/A2</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">7.42 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVA/Long</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">7.08 x 10- TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVA 1998/12-21</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">2.10 x 10' TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVA 1998/3-2</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">6.97 x 10-1 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVA 2001/2-20</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">4.09 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVA 2001/3-12</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">9.97 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVB/WASH/18537/62</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.97 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVB/9320</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">8.39 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVB/WV/14617/85</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">1.79 x 10° TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Hubei/1/2010</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">17 pg/μL</td></tr><tr><td colspan="1" rowspan="1">A/duck/Pennsylvania/10218/84</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H5N2</td><td colspan="1" rowspan="1">23 pg/μL</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/33982/2009</td><td colspan="1" rowspan="1">influenza A</td><td colspan="1" rowspan="1">H9N2</td><td colspan="1" rowspan="1">57 pg/μL</td></tr></table>

# f. Analytical specificity:

Cross reactivity study was performed using a panel of 36 test organisms (bacterial, fungal or viral) or human genomic DNA. Bacteria or fungal strains were tested at a concentration $\mathrm { { o f } \geq 1 x 1 0 ^ { 6 } \bar { C F U } / m L }$ , and viruses were tested at a concentration $1 \mathrm { x } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Human genomic DNA was tested at ${ \geq } 1 . 0 \mathrm { x } 1 0 ^ { 4 }$ genome copies $\mathrm { \Delta } / \mathrm { m L }$ . None of the organisms showed any cross reactivity in the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay.

Cross Reactivity Results Summary   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="4" rowspan="2">Reactivity: Detected (D)/ NotConcentration tested          Detected (ND)(CFU. copies, or TCID50/mL)Flu A   Flu B   RSV</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 1</td><td colspan="1" rowspan="1">7 0 x 105 TCID50/mI</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 7A</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ulcerans</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Human Herpes Virus 6 (HHV6),Z29</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Human Herpes Virus 7 (HHV7), SB</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">1.0 x 104genome copies/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus Z048</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumoniae</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">2.24 x 105 cFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">7.0 x 106 cFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MRSA</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MSSA</td><td colspan="1" rowspan="1">7.0 x 106 cFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis MRSE</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 3 type B1</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 9 type A1</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">7.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr></table>

g.

# Microbial interference:

The same organisms that were tested in the cross reactivity study were evaluated in the microbial interference study. They were added to sample tubes containing one of the six target organisms; two influenza A, two influenza B and two RSV strains in viral transport medium. Assay analytes were present in the samples at concentrations ${ \sim } 2 { - } 3 \mathbf { x }$ LoD. Each of the six strains was tested against a panel of 36 test organisms in triplicate using the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay. The data showed no interference with the organisms tested.

# h. Competitive interference:

Competitive Interference of the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay was evaluated using simulated samples containing pairs of target viruses (influenza A and influenza B, influenza A and RSV, influenza B and RSV) at two different concentrations. One of the concentrations was near the LoD (2-3X LoD) while the other concentration was $2 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $\mathrm { ^ { 1 0 ^ { 4 } - 1 0 ^ { 6 } X } }$ LoD). There was no observed interference in the competitive interference study; high concentration targets do not interfere with the detection of a low concentration target that was tested at 2-3X LoD.

# i. Interference:

A panel of potentially interfering substances (listed in the table below) and two strains of each viral type at a final concentration of approximately $2 { - } 3 \mathbf { x }$ LoD were evaluated in the interference study. None of the substances showed inhibition when tested at concentrations presented in the table below. In addition to these substances, FluMist® Influenza vaccine was tested for its ability to interfere with the IMDx Flu A/B and RSV for Abbott $\mathrm { m } 2 0 0 0$ assay. Initial testing of FluMist (that contains live attenuated reassortants of each of the three strains: A/California/07/2009 (H1N1), A/Perth/16/2009 (H3N2), and B/Brisbane/60/2008) alone without any target virus strain gave detected signals for influenza A and B at a concentration of $1 0 ^ { 6 . 5 }$ to $1 0 ^ { 7 . 5 }$ units, but the influenza A, and influenza B CN values were higher than expected, indicating that the assay detected the viruses present in the vaccine. The vaccine was diluted further to ${ \sim } 1 \mathrm { ~ x ~ i 0 } ^ { 8 }$ till no vaccine virus strains could be detected. When FluMist $^ \mathrm { \textregistered }$ was tested at this dilution in the presence of the target virus strains using the IMDx FluA/B and RSV for Abbott $m 2 0 0 0$ assay, influenza A, influenza B, and RSV were detected as expected.

# Interfering Substances

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Active Ingredient(s) inSubstance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=3 colspan=1>NasalSprays</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Sodium Chloride (with preservatives)</td><td rowspan=1 colspan=1>0.325% v/v</td></tr><tr><td rowspan=1 colspan=1>Nasa1 Gel</td><td rowspan=1 colspan=1>LuffaOpperculataGalphimia</td><td rowspan=1 colspan=1>0.5x0.5x3.3x</td></tr><tr><td rowspan=7 colspan=1>Nasal corticosteroids</td><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>50 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mometasone furoate</td><td rowspan=1 colspan=1>2.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>55 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>5.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>16 μg/mL</td></tr><tr><td rowspan=3 colspan=1>NSAID</td><td rowspan=1 colspan=1>Aspirin</td><td rowspan=1 colspan=1>16.2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>40 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Naproxen</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Acetominophen</td><td rowspan=1 colspan=1>Acetominophen</td><td rowspan=1 colspan=1>0.1 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Rele</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antibacterial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>40.0 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Benzo</td><td rowspan=1 colspan=1>Benzocain</td><td rowspan=1 colspan=1>2.5% soln</td></tr><tr><td rowspan=1 colspan=1>Antibiotic nasal ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>0.15 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Allergy medicine</td><td rowspan=1 colspan=1>Histamine hydrochloricum</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>Mucin protein, type 1-S</td><td rowspan=1 colspan=1>19 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Blood (Human)</td><td rowspan=1 colspan=1>Whole Blood with EDTA</td><td rowspan=1 colspan=1>5% v/v</td></tr></table>

# $j .$ . Carryover contamination:

The purpose of this study was to determine the incidence of target carryover and cross contamination between samples when using the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay. A minimum of five extraction runs were performed with alternating high positive and negative samples. The total number of samples run across the 5 extractions was 470, of which 235 were positive and 235 negative. The carryover rate for the three targets was $2 . 1 \%$ , $0 . 9 \%$ and $0 . 9 \%$ for the influenza A, influenza B and RSV targets respectively. Therefore, the overall carryover rate of $1 . 3 \%$ was reported for the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay.

# k. Fresh Vs. Frozen:

A fresh vs. frozen specimen study was performed by comparing samples that were stored unfrozen (Day 0) to samples frozen at $\sim - 2 0 ^ { \circ } \mathrm { C }$ for 35 days. Samples were tested prior to freezing (Day 0) to provide a baseline value and Day 0 values were compared to values obtained after freezing. Samples were tested in triplicates after freezing. Assay results were assessed for a change in result call (positive to negative, negative to positive, or either positive or negative to invalid) over the storage time period. In addition, the CN averages for each target and viral transport medium were monitored. All samples showed $100 \%$ detection after freezing for 35 days.

Average Influenza A CN Value Comparison of Fresh and Frozen Specimens in Various Transport Medium   

<table><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>| Day 0 CN</td><td rowspan=1 colspan=1>Day 35 CN</td><td rowspan=1 colspan=1>ΔCN</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>33.55</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>33.95</td><td rowspan=1 colspan=1>0.09</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>34.60</td><td rowspan=1 colspan=1>0.33</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>34.14</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.65</td><td rowspan=1 colspan=1>33.93</td><td rowspan=1 colspan=1>0.72</td></tr></table>

Average Influenza B CN Value Comparison of Fresh and Frozen Specimens in Various Transport Medium   

<table><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>| Day 0 CN</td><td rowspan=1 colspan=1>| Day 35 CN</td><td rowspan=1 colspan=1>ΔCN</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>31.43</td><td rowspan=1 colspan=1>31.89</td><td rowspan=1 colspan=1>0.46</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>31.67</td><td rowspan=1 colspan=1>31.37</td><td rowspan=1 colspan=1>0.30</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>31.66</td><td rowspan=1 colspan=1>31.28</td><td rowspan=1 colspan=1>0.38</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>31.47</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>31.90</td><td rowspan=1 colspan=1>0.04</td></tr></table>

Average RSV CN Value Comparison of Fresh and Frozen Specimens in Various Transport Medium   

<table><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Day 0 CN</td><td rowspan=1 colspan=1>| Day 35 CN</td><td rowspan=1 colspan=1>∆CN</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>0.13</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>34.72</td><td rowspan=1 colspan=1>34.56</td><td rowspan=1 colspan=1>0.16</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>34.63</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>34.71</td><td rowspan=1 colspan=1>0.18</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.26</td><td rowspan=1 colspan=1>33.58</td><td rowspan=1 colspan=1>0.68</td></tr></table>

A second study was performed using frozen clinical specimens (12 influenza A, 7 influenza B, 14 RSV) that had been tested prior to freezing by an FDA-cleared molecular method. These samples were frozen at $\mathrm { - 7 0 ^ { \circ } C }$ for up to 24 months. These frozen clinical samples were tested with one lot of IMDx Flu $\mathrm { A } / \mathrm { B }$ and RSV for Abbott $m 2 0 0 0$ assay reagents. $100 \%$ concordance was observed between fresh and frozen results.

# Freeze-Thaw stability:

The freeze/thaw cycle study was conducted with 5 viral transport media: M4, M4RT, M5, M6, and UTM with three viral strains representative of the assay targets, influenza A - Influenza A/Swine/NY/02/2009, influenza B - Influenza B/Florida/04/06, and RSV - RSV A Type A strain. Aliquots of specimens were removed from the freezer and allowed to come to ambient temperature for 1 to 3 hours, then returned to the $\sim - 2 0 ^ { \circ } \mathrm { C }$ freezer for 1 to 3 hours. Samples were tested in replicates of three after 1, 2, and 3 freeze/thaw cycles. The freeze thaw study demonstrated that samples were stable in all media types for up to 3 successive freeze/thaw cycles.

Average Influenza A CN Value Comparison of Freeze Thaw cycles Specimens in Various Transport Medium   

<table><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=2 colspan=1>Day 0  |</td><td rowspan=1 colspan=1>F/T-1</td><td rowspan=1 colspan=1>F/T-2</td><td rowspan=2 colspan=1>F/T-3</td></tr><tr><td rowspan=1 colspan=2>Average CN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>33.57 (0.43)</td><td rowspan=1 colspan=1>33.45 (0.55)</td><td rowspan=1 colspan=1>33.16 (0.84)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>33.35 (0.92)</td><td rowspan=1 colspan=1>33.51(0.76)</td><td rowspan=1 colspan=1>33.57 (0.70)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>33.30(0.90)</td><td rowspan=1 colspan=1>34.22(0.02)</td><td rowspan=1 colspan=1>33.70(0.50)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>33.49 (1.16)</td><td rowspan=1 colspan=1>34.33 (0.32)</td><td rowspan=1 colspan=1>33.57 (1.08)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.65</td><td rowspan=1 colspan=1>33.73 (0.13)</td><td rowspan=1 colspan=1>34.21 (0.35)</td><td rowspan=1 colspan=1>33.30 (0.56)</td></tr></table>

Average Influenza B CN Value Comparison of Freeze Thaw cycles Specimens in Various Transport Medium   

<table><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">Day 0</td><td colspan="1" rowspan="1">F/T-1</td><td colspan="1" rowspan="1">F/T-2</td><td colspan="1" rowspan="1">F/T-3</td></tr><tr><td colspan="4" rowspan="1">AverageCN (∆CN)</td></tr><tr><td colspan="1" rowspan="1">M4</td><td colspan="1" rowspan="1">31.43</td><td colspan="1" rowspan="1">31.69 (0.26)</td><td colspan="1" rowspan="1">32.00 (0.57)</td><td colspan="1" rowspan="1">31.19 (0.24)</td></tr><tr><td colspan="1" rowspan="1">M4RT</td><td colspan="1" rowspan="1">31.67</td><td colspan="1" rowspan="1">31.68 (0.02)</td><td colspan="1" rowspan="1">32.31 (0.65)</td><td colspan="1" rowspan="1">31.48 (0.18)</td></tr><tr><td colspan="1" rowspan="1">M5</td><td colspan="1" rowspan="1">31.66</td><td colspan="1" rowspan="1">31.68 (0.20)</td><td colspan="1" rowspan="1">32.70 (0.84)</td><td colspan="1" rowspan="1">32.16 (0.30)</td></tr><tr><td colspan="1" rowspan="1">M6</td><td colspan="1" rowspan="1">31.86</td><td colspan="1" rowspan="1">32.05 (0.38)</td><td colspan="1" rowspan="1">32.57 (0.90)</td><td colspan="1" rowspan="1">32.05 (0.38)</td></tr><tr><td colspan="1" rowspan="1">UTM</td><td colspan="1" rowspan="1">31.86</td><td colspan="1" rowspan="1">31.69 (0.17)</td><td colspan="1" rowspan="1">32.26 (0.40)</td><td colspan="1" rowspan="1">32.18 (0.32)</td></tr></table>

Average RSV CN Value Comparison of Freeze Thaw cycles Specimens in Various Transport Medium   

<table><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>Day 0</td><td rowspan=1 colspan=1>F/T-1</td><td rowspan=1 colspan=1>F/T-2</td><td rowspan=1 colspan=1>F/T-3</td></tr><tr><td rowspan=1 colspan=4>AverageCN (∆CN)</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>33.34(0.73)</td><td rowspan=1 colspan=1>33.99(0.08)</td><td rowspan=1 colspan=1>34.43(0.36)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>34.62(0.25)</td><td rowspan=1 colspan=1>34.23(0.14)</td><td rowspan=1 colspan=1>33.94 (0.43)</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>34.77(0.24)</td><td rowspan=1 colspan=1>33.72(0.81)</td><td rowspan=1 colspan=1>33.95(0.58)</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.72</td><td rowspan=1 colspan=1>34.70 (0.02)</td><td rowspan=1 colspan=1>34.34 (0.38)</td><td rowspan=1 colspan=1>34.40 (0.32)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>32.50</td><td rowspan=1 colspan=1>33.97 (1.47)</td><td rowspan=1 colspan=1>34.20 (1.70)</td><td rowspan=1 colspan=1>34.27 (1.77)</td></tr></table>

# l. Assay cut-off:

The initial maxRatio (MR) threshold parameters set for the IMDx Flu A/B and RSV for Abbott $\mathrm { m } 2 0 0 0$ assay were evaluated using known influenza A, influenza B, and RSV-positive and negative samples. The sample set for analysis consisted of 479 nasopharyngeal swabs samples that had 110 Flu A positive, 84 Flu B positive, 159 RSV positive, and 126 negative results.

MR values for influenza A, influenza B, and RSV positive and negative samples were plotted individually and the mean and standard deviation were calculated for each component. The MR threshold was established to be at least 5 standard deviations above the average MR values of the negative sample cohort.

Thresholds were set as follows: influenza A was set to 0.020, influenza B was set to 0.020 and RSV was set to 0.010. The final threshold values used in the assay are provided in the table below:

# Observed MR Value Data for Influenza A Specimens

<table><tr><td rowspan=1 colspan=1>Influenza A Component</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MR Mean</td><td rowspan=1 colspan=1>MR StdDev</td><td rowspan=1 colspan=1>MR Range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.002</td><td rowspan=1 colspan=1>0.000 - 0.014</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0.123</td><td rowspan=1 colspan=1>0.016</td><td rowspan=1 colspan=1>0.071 - 0.159</td></tr></table>

![](images/2a1e3dc2fef7042fdb0b5777d1a8102505b0d4ae3b2b2a8ebd03bafe34433c2d.jpg)  
MR values of positive influenza A samples relative to threshold

Observed MR Value Data for Influenza B Specimens   

<table><tr><td rowspan=1 colspan=1>Influenza B Component</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MR mean</td><td rowspan=1 colspan=1>MR SD</td><td rowspan=1 colspan=1>MR range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.002 - 0.007</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0.097</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.030 - 0.113</td></tr></table>

![](images/cfe9f92ea2ab560574e43aee92ea06476123fbe4407586f4e208a014c9f51622.jpg)  
MR values of positive influenza B samples relative to threshold

# Observed MR Value Data for RSV Specimens

<table><tr><td rowspan=1 colspan=1>RSV Component</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MR mean</td><td rowspan=1 colspan=1>MR SD</td><td rowspan=1 colspan=1>MR range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.000 - 0.007</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>0.094</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.039 - 0.107</td></tr></table>

![](images/94f159b0dd722862aee21e1a643213db1ccc9e4f029fce846b8119cf4b779103.jpg)  
MR values of positive RSV samples relative to threshold

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison:

Equivalency between different viral transport media The purpose of this study was to demonstrate equivalency among a variety of different transport media (Remel: M4, M4RT, M5, M6, Copan: UTM). For each transport medium being tested, twenty (20) replicates each of influenza A, influenza B, and RSV (60 replicates total) were formulated to provide organisms in transport medium that were at concentrations of approximately 2-3X LoD.

Table below shows the detection rates and CN values obtained for all viral transport media test sets. The maximum CN differences across transport media were calculated for each virus tested. All the average CN values were within $0 . 8 \mathrm { C N }$ of each other.

Viral Transport Media Equivalency Study   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=1 colspan=3>Influenza B</td><td rowspan=1 colspan=3>RSV</td></tr><tr><td rowspan=1 colspan=1>FluACN</td><td rowspan=1 colspan=1>PCCN</td><td rowspan=1 colspan=1># Det</td><td rowspan=1 colspan=1>FluBCN</td><td rowspan=1 colspan=1>PCCN</td><td rowspan=1 colspan=1># Det</td><td rowspan=1 colspan=1>RSVCN</td><td rowspan=1 colspan=1>PCCN</td><td rowspan=1 colspan=1># Det</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>28.53</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.43</td><td rowspan=1 colspan=1>28.27</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>28.36</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>28.66</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.66</td><td rowspan=1 colspan=1>28.28</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>28.48</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>28.76</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>28.50</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>28.61</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>28.62</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.67</td><td rowspan=1 colspan=1>28.29</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.72</td><td rowspan=1 colspan=1>28.50</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>34.65</td><td rowspan=1 colspan=1>28.72</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.86</td><td rowspan=1 colspan=1>28.44</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.26</td><td rowspan=1 colspan=1>28.54</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>MaximumCNDifference</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>N/A</td></tr></table>

FluA $=$ influenza A, FluB $=$ influenza B, PC $=$ Process control, # Det $=$ number of replicates detected, Maximum CN Difference $=$ span between highest and lowest average CN values observed

Equivalency between negative clinical matrix and viral transport medium The purpose of this study was to show that the negative specimen matrix in viral transport medium was equivalent to viral transport medium without matrix when testing the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay.

Contrived specimens were prepared by diluting two of the target viruses (influenza A, and RSV) in viral transport medium containing negative specimen matrix, and in viral transport medium alone at ${ \sim } 3 \mathrm { X }$ LoD. Comparable CN values were obtained across matrices; the $\Delta C \mathrm { N }$ values observed between matrix types, for both virus targets and process controls, ranged from 0.05 to 0.49.

CN comparisons and CN differences (ΔCN) between Matrices   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=2>RSV</td></tr><tr><td rowspan=1 colspan=1>AverageInfluenza ACN</td><td rowspan=1 colspan=1>AverageProcessControl CN</td><td rowspan=1 colspan=1>AverageRSV CN</td><td rowspan=1 colspan=1>AverageProcessControl CN</td></tr><tr><td rowspan=1 colspan=1>Negative specimenmatrix in viraltransport medium</td><td rowspan=1 colspan=1>32.32</td><td rowspan=1 colspan=1>27.68</td><td rowspan=1 colspan=1>32.32</td><td rowspan=1 colspan=1>27.38</td></tr><tr><td rowspan=1 colspan=1>Viral transportmedium</td><td rowspan=1 colspan=1>31.91</td><td rowspan=1 colspan=1>27.48</td><td rowspan=1 colspan=1>31.83</td><td rowspan=1 colspan=1>27.33</td></tr><tr><td rowspan=1 colspan=1>∆CN</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.05</td></tr></table>

3. Clinical studies:

The performance of the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay was assessed during the course of two influenza seasons (2011-2012 and 2012-2013). For the 2011 - 2012 season, four geographically diverse test sites within the United States prospectively collected influenza A/B and RSV samples. Samples enrolled for this study were Nasopharyngeal Swabs (562 specimens) collected for routine influenza testing. A total of seven (7) samples yielded an error message and were categorized as “unresolved errors/invalid” yielding an overall invalid rate of $1 . 2 4 \%$ . In addition, 58 samples were withdrawn and were not included in the final data set. Of these 58 samples, 41 had incomplete reference method (viral culture) testing therefore no viral culture result available for interpretation, 9 samples had insufficient volume for IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ testing, 7 samples were excluded because the site utilized a sample dilution procedure outside of manufacturer’s instructions for use, and one sample had reference method contamination, no viral culture result available for interpretation. A total of four hundred and ninety seven (497) valid specimens were included in the final data set and analyzed for product performance.

For the 2012-2013 season, three geographically diverse laboratory test sites participated in the study. All samples enrolled during the 2012-2013 influenza season were NPS received by the laboratory for routine influenza testing. A total of five hundred and twenty seven (527) NPS samples were enrolled. Three samples yielded an error message and were categorized as “unresolved errors”. These samples were considered to be invalid $0 . 5 7 \%$ invalid rate). In addition, 89 samples were withdrawn due to protocol deviations/delayed shipping and were not included in the final data set. For the 2012- 2013 season, a total of 435 valid NPS samples were included in the final data set and analyzed for product performance.

Between the two influenza seasons, a total of $9 3 2 \mathrm { N P S }$ samples were tested and analyzed for performance. The gender and age demographics of samples included in the study are presented below.

<table><tr><td rowspan=1 colspan=5>Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=2>Female</td><td rowspan=1 colspan=2>Male</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2011-2012</td><td rowspan=1 colspan=1>2012-2013</td><td rowspan=1 colspan=1>2011-2012</td><td rowspan=1 colspan=1>2012-2013</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>84 (30.1%)</td><td rowspan=1 colspan=1>71 (31.8%)</td><td rowspan=1 colspan=1>76 (34.7%)</td><td rowspan=1 colspan=1>101 (47.6%)</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>38 (13.6%)</td><td rowspan=1 colspan=1>29 (13.0%)</td><td rowspan=1 colspan=1>45 (20.5%)</td><td rowspan=1 colspan=1>28 (13.2%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>96 (34.4%)</td><td rowspan=1 colspan=1>76 (34.1%)</td><td rowspan=1 colspan=1>63 (28.9%)</td><td rowspan=1 colspan=1>44 (20.7%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>61 (21.9%)</td><td rowspan=1 colspan=1>47 (21.1%)</td><td rowspan=1 colspan=1>34 (15.5%)</td><td rowspan=1 colspan=1>39 (18.4%)</td></tr><tr><td rowspan=1 colspan=1>Season Totals</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>212</td></tr><tr><td rowspan=1 colspan=1>Overall Totals</td><td rowspan=1 colspan=2>502</td><td rowspan=1 colspan=2>430</td></tr></table>

The performance of the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay was compared to viral cell culture followed by DFA. Results are shown in the three tables below. Discordant samples were tested using the Nanosphere Verigene $^ \mathrm { \textregistered }$ Respiratory Virus Plus Nucleic Acid Test $\left( \mathrm { R V + } \right)$ on the Verigene $^ \mathrm { \textregistered }$ System and subsequent results are documented in the footnotes.

Influenza A Clinical Agreement Summary   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Viral Culture</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx Flu A/B and RSVfor Abbott m2000 Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.6% (94% -99%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>731</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.7% (94% -97%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>764</td><td rowspan=1 colspan=1>932</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1Of the 33 influenza A false positive results observed, twenty-five (25) were confirmed as influenza A positives by FDA cleared molecular assay. Eight (8) remained discrepant. $^ 2 \mathrm { O f }$ the four (4) influenza A false negative results observed, all four (4) were confirmed as negatives by FDA cleared molecular assay.

Influenza B Clinical Agreement Summary   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Viral Culture</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx Flu A/B and RSVfor Abbott m2000 Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.1% (90% -99%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>839</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.2% (96% -98%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>932</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

3Of the 24 influenza B false positive results observed, twenty (20) were confirmed as influenza B positives by FDA cleared molecular assay. Four (4) remained discrepant. 4The two (2) influenza B false negative results remained discrepant.

RSV Clinical Agreement Summary   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Viral Culture</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx Flu A/B and RSVfor Abbott m2000 Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>585$</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.2% (92% -99%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>770</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>93.0% (91% -95%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>932</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

5Of the 58 RSV false positive results observed, thirty-nine (39) were confirmed as RSV positives by FDA cleared molecular assay. Nineteen (19) remained discrepant. $^ { 6 } \mathrm { O f }$ the three (3) RSV false negative results observed, one (1) was confirmed as negative by FDA cleared molecular assay. Two (2) remained discrepant.

Assay positive and negative quality controls were run with each IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay run. No quality control failures were observed during the course of the method comparison study.

# 4. Clinical cut-off:

MR threshold parameters were validated using data compiled at three test sites from 533 (188 positive, 345 negative) nasopharyngeal swab samples compared to the reference method. The table below shows the observed MR results for influenza A/B and RSV. The observed MR ranges for positive and negative samples were consistent with those observed for internal assay cut-off evaluation study, thus indicating that threshold values for Flu A/B and RSV were set appropriately for the assay.

Influenza A, Influenza B, and RSV MR Data for Method Comparison Study   

<table><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MR mean</td><td rowspan=1 colspan=1>MR StdDev</td><td rowspan=1 colspan=1>MR range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>345</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.002</td><td rowspan=1 colspan=1>0.000 - 0.019</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>0.117</td><td rowspan=1 colspan=1>0.029</td><td rowspan=1 colspan=1>0.022 - 0.146</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MR mean</td><td rowspan=1 colspan=1>MR SD</td><td rowspan=1 colspan=1>MR range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>345</td><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.001 - 0.006</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.069</td><td rowspan=1 colspan=1>0.026</td><td rowspan=1 colspan=1>0.023 - 0.106</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MR mean</td><td rowspan=1 colspan=1>MR SD</td><td rowspan=1 colspan=1>MR range</td></tr><tr><td rowspan=1 colspan=1>Negative Samples</td><td rowspan=1 colspan=1>345</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>0.001 - 0.008</td></tr><tr><td rowspan=1 colspan=1>Positive Samples</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0.088</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.010 - 0.112</td></tr></table>

# 5. Expected values/Reference range:

In the multi-site method comparison study conducted using the IMDx Flu $\mathrm { A } / \mathrm { B }$ and RSV for Abbott $m 2 0 0 0$ assay, a total of 932 samples collected during the 2011-2012 and 2012- 2013 seasons were analyzed. The prevalence of influenza A was $1 7 . 6 \%$ , the prevalence of influenza B was $7 . 2 \%$ and the prevalence of RSV was $1 1 . 1 \%$ across the 2011-2012

and 2012-2013 seasons combined.

No dual infections were detected by viral culture, but the IMDx Flu A/B and RSV for Abbott $\mathrm { m } 2 0 0 0$ assay found $1 . 9 \%$ (18 specimens) of the total infections to be dual infections. Of these 18 specimens, 1 was positive for influenza A and influenza B, 5 were positive for influenza A and RSV, and 12 were positive for influenza B and RSV. The IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay also detected one triple infection, positive for influenza A, influenza B and RSV.

N. Instrument Name: Abbott $^ \mathrm { { \textregistered } }$ $\mathfrak { d } m 2 0 0 0 ^ { \mathrm { T M } }$ system

# O. System Descriptions:

1. Modes of Operation:

The Abbott $m 2 0 0 0$ System is comprised of:

· An automated sample preparation instrument system (m2000sp),   
· A real-time polymerase chain reaction (PCR) thermal cycler/reader instrument system (m2000rt)   
· Independent workstations for each system above Separate software for the m2000sp and the $m 2 0 0 0 \mathrm { r t }$ systems

The Abbott $m 2 0 0 0$ System processes up to 96 specimens, controls, and calibrators in batch mode by parameters that are contained in individual assay application specifications. The Abbott $m 2 0 0 0$ system used with the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay is a closed platform, with proprietary, closed software; users can neither modify assay application specifications nor change software specifications used to generate results. Instrument system software has been validated by Abbott Molecular.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: Patient ID/Sample ID can be entered manually or barcoded

4. Specimen Sampling and Handling: Not Applicable

5. Calibration:

Optical calibration of the Abbott $m 2 0 0 0 r t$ instrument is required for the accurate measurement and differentiation of dye fluorescence during the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ assay. The following Abbott $m 2 0 0 0 r t$ Optical Calibration Plates are used to calibrate the Abbott $m 2 0 0 0 r t$ instrument for the IMDx Flu A/B and RSV for $m 2 0 0 0$ assay:

FAM™ Plate (Carboxyfluorescein): Part No. 50-805035 $\mathrm { V I C } ^ { \mathfrak { B } }$ Plate (Proprietary dye): Part No. 50-805045   
· ROX Plate (Carboxy-X-rhodamine): Part No. 50-805015 $\mathrm { C Y 5 ^ { r M } }$ Plate (Proprietary dye): Part No. 50-805040   
· TAMRA Plate (5(6)-carboxytetramethylrhodamine, succinimidyl ester): Part No. 50- 805005

6. Quality Control:

Positive and negative controls:

Positive control contains a mixture of synthetic influenza A, influenza B, RSV-A, and RSV-B RNA in buffer.

Negative control (IMDx Negative Control-B) is viral transport medium negative for assay target nucleic acids. When the negative control tube is loaded onto the m2000sp, an equal volume of lysis buffer containing the inactivated MS2 RNA bacteriophage process control is added before sample preparation.

One positive control and one negative control must be included in each run to monitor run validity. If either the positive control or negative control is out of predetermined range, an error message is generated and no results for the plates are reported. All of the specimens and controls from that run must be reprocessed, beginning with sample preparation.

The presence of influenza A, influenza B, and RSV targets must not be detected in the IMDx Negative Control-B. Detection of influenza A, influenza B, and/or RSV RNA targets in the IMDx Negative Control-B is indicative of contamination by other samples or by amplified product introduced during sample preparation or during preparation of the Abbott 96-Well Optical Reaction Plate.

# Process control:

The IMDx Process Control-B is composed of an RNA bacteriophage unrelated to influenza and RSV viruses. Prior to sample preparation, a defined, consistent quantity of the IMDx Process Control-B is dispensed into lysis buffer (mLysisDNA). It is then added to each sample during the processing of each specimen and control, RT-PCR amplified by reagents included in the Amplification Reagent mix, and measured on the Abbott $m 2 0 0 0 r t$ instrument to demonstrate proper sample processing and assay validity.

IMDx Process Control-B CN and MR assay validity parameters are encoded in the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ application files installed on the Abbott $m 2 0 0 0 s p$ and Abbott $m 2 0 0 0 r t$ systems from the IMDx Flu A/B and RSV for Abbott $m 2 0 0 0$ System ROW Combined Application CD-ROM. An error is displayed when a specimen or control fails to meet these validity specifications. The user must refer to the Abbott $m 2 0 0 0 r t$ Operations Manual for an explanation of the corrective actions for the error code. Samples for which the IMDx Process Control-B CN value falls outside of the established range must be retested using the original specimen.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.