# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K131813

# B. Purpose for Submission:

The purpose of this submission is to show that the Quidel® Molecular $\mathrm { R S V + h M P V }$ Assay is substantially equivalent to the Prodesse ProFlu™ Assay and Pro hMPV $^ +$ Assay when used on the Life Technologies QuantStudio™ Dx Instrument.

# C. Measurand:

This assay detects the presence of respiratory syncytial virus (RSV) and human metapneumovirus (hMPV) ribonucleic acid (RNA) using reverse transcription real-time polymerase chain reaction (RT-PCR). The RT-PCR primers are developed to bind to conserved regions of the L viral polymerase and NS2 genes for RSV and RNA polymerase gene for hMPV.

# D. Type of Test:

This is a nucleic acid based test using RT-PCR. The negative sense strands of the RSV and hMPV genomes are reverse transcribed and then amplified. The presence of RSV or hMPV RNA is then detected through sequence-specific labeled probe that is cleaved during PCR amplification releasing the fluorescent reporter dye from the quencher dye.

# E. Applicant:

Quidel Corporation

F. Proprietary and Established Names:

Quidel Molecular RSV $^ +$ hMPV Assay

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

# 2. Classification:

Class II

3. Product code:

OEM, OCC

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Quidel Molecular $\mathrm { R S V + h M P V }$ Assay is a multiplex Real-Time PCR (RT-PCR) assay for the qualitative detection and identification of respiratory syncytial virus (RSV) and human metapneumovirus (hMPV) ribonucleic acid (RNA) extracted from nasal and nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection. This in vitro diagnostic test is intended to aid in the differential diagnosis of RSV and hMPV infections in humans in conjunction with clinical and epidemiological risk factors. This test is not intended to differentiate the two subtypes of RSV or the four genetic sub-lineages of hMPV.

Negative results do not preclude RSV infection and/or hMPV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease. The use of additional laboratory testing and clinical presentation must be considered in order to obtain the final diagnosis of respiratory viral infection.

The Quidel Molecular $\mathrm { R S V + h M P V }$ Assay can be performed using either the Life Technologies QuantStudio™ Dx RT-PCR Instrument, the Applied Biosystems® 7500 Fast Dx RT-PCR Instrument, or the Cepheid SmartCycler® II System.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): Prescription only.

4. Special instrument requirements:

bioMérieux NucliSENS® easyMAG® System (software version 2.0)

Life Technologies QuantStudio Dx RT-PCR Instrument (software version 1.0)

Related submission:

K122189 for Applied Biosystems 7500 Fast Dx RT-PCR Instrument (software version 1.4) and the Cepheid SmartCycler II System (software version 3.0b)

# I. Device Description:

The Quidel Molecular $\mathrm { { } R S V + h M P V }$ Assay detects viral nucleic acids that have been extracted from a patient sample using the NucliSENS easyMAG automated extraction platform. A multiplex RT-PCR reaction is carried out under optimized conditions in a single tube generating amplicons for each of the target viruses present in the sample. This reaction is performed utilizing either the Cepheid SmartCycler II, the Applied Biosystems 7500 Fast Dx, or the Life Technologies QuantStudio Dx. Identification of RSV and hMPV and the Process Control (PRC) occurs by the use of target-specific primers and fluorescent-labeled probes that hybridize to conserved regions in the genomes of RSV and hMPV and the PRC. Results are analyzed by the Cepheid SmartCycler II, Applied Biosystems 7500 Fast Dx RTPCR, or QuantStudio Dx proprietary software (version 3.0b, 1.4,1.0, or higher, respectively) and then reported to the user.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

ProFlu Assay and Pro hMPV $^ +$ Assay

2. Predicate 510(k) number(s): K092500 and K082688

3. Comparison with predicate:

<table><tr><td>Item</td><td>Subject Device Quidel Molecular RSV +hMPV Assay</td><td>Predicate Device Prodesse ProFlu+ (K092500)</td><td>Predicate Device Prodesse Pro hMPV+ (K082688)</td></tr><tr><td>Intended Use</td><td>The Quidel Molecular RSV+ hMPV Assay is a multiplex Real- Time PCR (RT- PCR) assay for the qualitative detection</td><td>The ProFluTM+ Assay is a multiplex Real-Time PCR (RT-PCR) in vitro diagnostic test for the rapid and qualitative detection</td><td>The Pro hMPV+ Assay is a Real-Time RT-PCR in vitro diagnostic test for the qualitative detection of human Metapneumovirus</td></tr><tr><td>Item</td><td>Subject Device Quidel Molecular RSV +hMPV Assay and identification of respiratory syncytial virus (RSV) and human metapneumovirus (hMPV) ribonucleic acid (RNA) extracted from nasal</td><td>Predicate Device Prodesse ProFlu+ (K092500) and discrimination of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids</td><td>Predicate Device Prodesse Pro hMPV+ (K082688) (hMPV) nucleic acid isolated and purified from nasopharyngeal swab (NP) specimens</td></tr><tr><td>and nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection. This in vitro diagnostic test is intended to aid in the differential diagnosis of RSV and hMPV infections in humans in conjunction with clinical and epidemiological risk factors. This test is not intended to differentiate the two subtypes of RSV or the four genetic sub- lineages of hMPV. Negative results do not preclude RSV infection and/or hMPV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.</td><td>isolated and purified from nasopharyngeal (NP) swab specimens obtained from symptomatic patients. This test is intended for use to aid in the differential diagnosis of Influenza A, Influenza B and RSV viral infections in humans and is not intended to detect Influenza C. Negative results do not preclude influenza or RSV virus infection and should not be used as the sole basis for treatment or other management decisions. Conversely, positive results do not rule- out bacterial infection or co- infection with other viruses. The agent detected may not be Conversely, positive the definite cause of</td><td>intended to of hMPV. decisions.</td><td>obtained from individuals exhibiting signs and symptoms of acute respiratory infection. This assay targets a highly conserved region of the Nucleocapsid gene of hMPV. The detection of hMPV nucleic acid from symptomatic patients aids in the diagnosis of human respiratory hMPV infection if used in conjunction with other clinical and laboratory findings. This test is not differentiate the four genetic sub-lineages Negative results do not preclude hMPV infection and should not be used as the sole basis for diagnosis, treatment or other management</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="2" rowspan="1">Subject DeviceQuidel MolecularRSV +hMPV Assay</td><td colspan="4" rowspan="1">Predicate DeviceProdesse ProFlu+(K092500)</td><td colspan="1" rowspan="1">Predicate DeviceProdesse Pro hMPV+(K082688)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="2" rowspan="2">results do not rule-out bacterialinfection or co-infection with otherviruses. The agentdetected may not bethe definite cause ofdisease. The use ofadditional laboratorytesting and clinicalpresentation must beconsidered in orderto obtain the finaldiagnosis ofrespiratory viralinfection.The QuidelMolecular RSV +hMPV Assay can beperformed usingeither the Life</td><td colspan="4" rowspan="3">disease. The use ofadditional laboratorytesting and clinicalpresentation must beconsidered in orderto obtain the finaldiagnosis ofrespiratory viralinfection.Performancecharacteristics forInfluenza A Viruswere establishedwhen InfluenzaA/H3 and A/H1were thepredominantInfluenza A virusesin circulation (2006-2007 respiratoryseason).Performancecharacteristics forInfluenza A wereconfirmed whenInfluenza A/H1,Influenza A/H3, andInfluenza A/2009H1N1 were thepredominantInfluenza A virusesin circulation (2008and 2009). Whenother Influenza Aviruses areemerging,performancecharacteristics mayvary.If infection with anovel Influenza A</td><td colspan="2" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">TechnologiesQuantStudio™ DxRT-PCR Instrument,the AppliedBiosystems® 7500Fast Dx RT-PCRInstrument, or theCepheidSmartCycler® I1System.</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject DeviceQuidel MolecularRSV +hMPV Assay</td><td colspan="1" rowspan="1">Predicate DeviceProdesse ProFlu+(K092500)</td><td colspan="1" rowspan="1">Predicate DeviceProdesse Pro hMPV+(K082688)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">virus is suspectedbased on currentclinical andepidemiologicalscreening criteriarecommended bypublic healthauthorities,specimens should becollected withappropriate infectioncontrol precautionsfor novel virulentInfluenza virusesand sent to state orlocal healthdepartment fortesting. Viral cultureshould not beattempted in thesecases unless a BSL3+ facility isavailable to receiveand culturespecimens.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">RSV, hMPV</td><td colspan="1" rowspan="1">Influenza A virus,influenza B virus,RSV</td><td colspan="1" rowspan="1">hMPV</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasal swab,nasopharyngealswab</td><td colspan="1" rowspan="1">Nasopharyngealswab</td><td colspan="1" rowspan="1">Nasopharyngeal swab</td></tr><tr><td colspan="1" rowspan="1">Instrument/AssayPlatform</td><td colspan="1" rowspan="1">Life TechnologiesQuantStudio Dx RT-PCR Instrument, theApplied Biosystems7500 Fast Dx RT-PCR Instrument, orthe CepheidSmartCycler IISystem</td><td colspan="1" rowspan="1">CepheidSmartCycler IISystem</td><td colspan="1" rowspan="1">Cepheid SmartCyclerII System</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">An internal RNAcontrol is provided</td><td colspan="1" rowspan="1">Influenza A,Influenza B, RSV A,RSV B positive</td><td colspan="1" rowspan="1">hMPV positive RNAtranscript control andan internal RNA</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">RNA transcriptcontrols and aninternal RNAcontrol are provided</td><td colspan="1" rowspan="1">control are provided</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethods</td><td colspan="1" rowspan="1">bioMérieuxNucliSENSeasyMAG System</td><td colspan="1" rowspan="1">Roche MagNA PureLC Total NucleicAcid Isolation Kit orthe bioMérieuxNucliSENSeasyMAG System</td><td colspan="1" rowspan="1">Roche MagNA PureLC Total NucleicAcid Isolation Kit orthe bioMérieuxNucliSENS easyMAGSystem</td></tr><tr><td colspan="1" rowspan="1">AssayMethodology</td><td colspan="1" rowspan="1">RT-PCR-basedsystem for detectingthe presence orabsence of viralRNA in clinicalspecimens</td><td colspan="1" rowspan="1">RT-PCR-basedsystem for detectingthe presence orabsence of viralRNA in clinicalspecimens</td><td colspan="1" rowspan="1">RT-PCR-based systemfor detecting thepresence or absence ofviral RNA in clinicalspecimens</td></tr><tr><td colspan="1" rowspan="1">Viral Targets</td><td colspan="1" rowspan="1">RSV: L viralpolymerase and NS2geneshMPV: RNApolymerase gene</td><td colspan="1" rowspan="1">Influenza A: MatrixgeneInfluenza B: Non-structural NS1 andNS2</td><td colspan="1" rowspan="1">Nucleocapsid gene</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

# L. Test Principle:

This is a nucleic acid based test using reverse transcription polymerase chain reaction (RTPCR). Viral infection is detected through the use of PCR to amplify and detect viral RNA. The negative sense RNA genomes of RSV and hMPV are reverse-transcribed and then amplified during the RT-PCR reaction. The presence of RSV or hMPV RNA is then detected through a sequence-specific labeled probe that is cleaved during PCR amplification releasing the fluorescence reporter dye from the quencher dye.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility

Reproducibility studies were performed on contrived samples extracted using the bioMérieux easy MAG system and tested on the Life Technologies QuantStudio Dx platform. Reproducibility was assessed at three separate sites (two external, one in-house) where two users at each site tested each virus sample in triplicate over five days. The hMPV-A2 or RSV A viruses were tested at high negative (0.3X LoD for RSV A and 0.15X LoD for hMPV-A2), low positive/close to the limit of detection (2X LoD), and medium positive (5X LoD) concentrations. The viruses were diluted in a pooled negative nasal matrix. The RSV and hMPV negative samples (RSV Negative and hMPV Negative) consisted of RSV and hMPV negative nasal matrix. Negative and positive controls included in the study were from the Quidel Molecular $\mathrm { R S V + h M P V }$ Control Set.

Table 1. Results of Reproducibility Study   

<table><tr><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalResults</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt*</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt*</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt*</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>RSV High Negative0.3X LoD</td><td rowspan=1 colspan=1>15/30</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1/30</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>39/90</td></tr><tr><td rowspan=1 colspan=1>RSV Low Positive2X LoD</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>89/90</td></tr><tr><td rowspan=1 colspan=1>RSV Med Positive5X LoD</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>RSV Negative</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>RSV NegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>RSV PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>hMPV HighNegative 0.15X LoD</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>11/30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>21/30</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>52/90</td></tr><tr><td rowspan=1 colspan=1>hMPV Low Positive2X LoD</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>hMPV Med Positive5X LoD</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>hMPV Negative</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>hMPV NegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=1>hMPV PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>90/90</td></tr></table>

\* Average Ct values only on positive results. Ct values of 0 were left out of this calculation.

The data demonstrates good reproducibility as it shows $100 \%$ detection of hMPV and $9 9 \%$ detection of RSV at low positive/close to the detection limit (2X LoD) and $100 \%$ detection of hMPV and RSV at medium positive levels (5X LoD). Detection was lower than $50 \%$ for the high negative samples, which is expected and therefore acceptable for real-time PCR tests.

$b$ . Linearity/assay reportable range:

N/A

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The package insert for this assay includes a reference to an optional unassayed control set (the Quidel Molecular RSV +hMPV Control Set, #M107).

# Fresh vs. Frozen Study

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

A viral panel was created and tested before freezing and then after storage frozen at - $7 0 \mathrm { { } ^ { \circ } C }$ for 22-56 days. Data collected from these samples showed that storage for up to 56 days did not impact the ability of the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay to detect either RSV or hMPV RNA near the cut-off. In a similar study, clinical samples were tested within 72 hours of collection and once again after seven days at $- 7 0 \mathrm { { } ^ { \circ } C }$ . These samples did not show any significant change in Ct values over time when stored at $- 7 0 \mathrm { { } ^ { \circ } C }$

# Master Mix Stability after Rehydration

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

An evaluation of master mix stability after rehydration was carried out over a five-day period. The data showed that the rehydrated master mix was stable at room temperature and at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 48 hours and for five days at ${ - 2 0 ^ { \circ } C }$ .

# Extracted Specimen Stability

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

An evaluation of the stability of extracted specimens was carried out using the bioMérieux NucliSENS® easyMAG® System. Studies showed that extracted hMPV and RSV RNA was stable for six hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ , four hours at room temperature, one month at ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ , and after 96 hours at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ with up to three freeze/thaw cycles.

# $d .$ . Detection limit:

# Limit of Detection (LoD)

The LoD was determined using four genetic sub-lineages for hMPV and two subtypes for RSV. Each virus sub-lineage/subtype was diluted and each dilution was extracted and run 20 times using the Life Technologies QuantStudio Dx. The LoD was determined to be the lowest titer at which the virus was detected greater than $9 5 \%$ of the time. Virus was diluted in a pooled negative nasal matrix. The positive controls were from the Quidel

Molecular $\mathrm { R S V + h M P V }$ Control Set. The negative control was negative nasal matrix (RSV and hMPV negative).

Table 2: Results of LoD Study   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD QuantStudio Dx(TCID50/mL)</td><td rowspan=1 colspan=1>Previous LoD ABI 7500 FastDx (TCID50/mL) - K122189</td></tr><tr><td rowspan=1 colspan=1>hMPV-A1</td><td rowspan=1 colspan=1>8.73E+00</td><td rowspan=1 colspan=1>2.91E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV-A2</td><td rowspan=1 colspan=1>2.91E+00</td><td rowspan=1 colspan=1>2.91E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV-B1</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>2.25E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV- B2</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>2.25E+00</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>6.29E-01</td><td rowspan=1 colspan=1>6.29E-01</td></tr><tr><td rowspan=1 colspan=1>RSV-B</td><td rowspan=1 colspan=1>2.25E-01</td><td rowspan=1 colspan=1>7.50E-01</td></tr></table>

The data demonstrates that the LoD values for the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay on the new instrument (QuantStudio Dx) is within 3X of the LoD values determined for the assay on the previously cleared instrument (ABI 7500 Fast Dx) in K122189.

# e. Analytical specificity:

# Interference (Other Microorganisms)

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

This study was conducted to evaluate the potential for interference when other common respiratory pathogens are present in a RSV or hMPV low positive sample at high concentrations. The study found that there was no interference seen. All samples positive for RSV or hMPV tested positive in the presence of other respiratory pathogens.

# Cross-Reactivity

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

This study was conducted to evaluate the potential for cross-reactivity when other microorganisms (viruses, bacteria, yeast) are present in a sample at a high concentration. All samples negative for RSV or hMPV tested negative, indicating that there was no cross-reactivity seen between any of the organisms tested.

# Competitive Interference (RSV vs. hMPV)

The competitive interference study was conducted to assess if a high concentration of one virus could affect detection of another virus present at low levels in the same sample. Samples containing both RSV A and hMPV B2 were tested with the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay on the QuantStudio Dx. The low concentration virus was created by spiking either RSV or hMPV into negative nasopharyngeal swab matrix at 2X the LoD. The high concentration virus was spiked into the low level specimen. These

concentrations ranged from 10X LoD to 100,000X LoD. Any test sample in which detection of RSV or hMPV was not observed was considered inhibitory. Virus was diluted in a pooled negative nasal matrix. The positive controls were from the Quidel Molecular $\mathrm { R S V + h M P V }$ Control Set. The negative control was negative nasal matrix (RSV and hMPV negative).

Table 3: Results of Competitive Inhibition Study   

<table><tr><td>Test Analyte Concentration</td><td>Competitor Analyte Concentration (Multiples of LoD)</td><td>Test Analyte Mean Ct Value</td><td>All Replicates of Test Analyte Detected (yes/no)</td><td>Competitive Interference (yes/no)</td></tr><tr><td rowspan="5">2X LoD RSV</td><td>0 (Control)</td><td>32.2</td><td>Yes</td><td>No</td></tr><tr><td>2X</td><td>32.2</td><td>Yes</td><td>No</td></tr><tr><td>10X</td><td>32.7</td><td>Yes</td><td>No</td></tr><tr><td>100X</td><td>35.9</td><td>Yes</td><td>No</td></tr><tr><td>1,000X</td><td>37.3</td><td>Yes</td><td>No</td></tr><tr><td rowspan="6">2X LoD hMPV</td><td>10,000X</td><td>Neg 30.8</td><td>No (0/3) Yes</td><td>Yes No</td></tr><tr><td>0 (Control) 2X</td><td>31.5</td><td>Yes</td><td>No</td></tr><tr><td>10X</td><td>30.7</td><td>Yes</td><td>No</td></tr><tr><td>100X</td><td>32</td><td>Yes</td><td>No</td></tr><tr><td>1,000X</td><td>32.6</td><td>Yes</td><td>No</td></tr><tr><td>10,000X</td><td>Neg</td><td>No (0/3)</td><td>Yes</td></tr><tr><td></td><td>100,000X</td><td>34.7</td><td>No (1/3)</td><td>Yes</td></tr></table>

Results indicate that, at 10,000X LoD, the presence of RSV can affect detection of hMPV near the cut-off (2X LoD) and at 10,000X LoD, the presence of hMPV can affect detection of RSV near the cut-off (2X LoD). Additionally, at, 1,000X LoD, the presence of hMPV may affect detection of RSV near the cut-off (2X LoD) as indicated by a shift in $\mathrm { C t } > 5 . 0$ compared to the control. This has been addressed as a limitation in the product labeling.

# Carry-Over

The carry-over study was designed to uncover the presence of contamination in negative specimens due to carry-over of virus during nucleic acid extraction and PCR amplification. The study was designed by alternately placing high positive $( 2 . 5 7 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ RSV B and $3 . 1 6 \times 1 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 } ^ { - } / \mathrm { m L }$ hMPV A1, each diluted in negative nasopharyngeal matrix) and negative specimens in each well of the PCR plate. A total of 48 high positives and 48 negatives were tested in each run, with a total of five runs, each run on a separate day $4 8 \times 5 = 2 4 0$ samples total for each group in the study). Virus was diluted in a pooled negative nasal matrix. The negative control was negative nasal matrix (RSV and hMPV negative).

Table 4: Results of Carry-Over Study   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>TotalHighPositiveDetectedasPositive</td><td rowspan=1 colspan=1>CtAve</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AcceptanceCriteria forHighPositives</td><td rowspan=1 colspan=1>AcceptanceCriteria Met?</td><td rowspan=1 colspan=1>TotalNegativesDetected asNegative</td><td rowspan=1 colspan=1>AcceptanceCriteria</td><td rowspan=1 colspan=1>AcceptanceCriteria Met?</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>100%positive</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>≥95%negative</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>100%positive</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>≥95%negative</td><td rowspan=1 colspan=1>Yes</td></tr></table>

All 240 replicates of negative sample were “Not Detected” for both RSV and hMPV, therefore, no evidence of carry-over was observed.

f. Assay cut-off:

# Baseline

Initial baseline assessment was conducted using contrived samples from the reproducibility study panel. Testing indicated that the use of the auto baseline resulted in the detection of false positives for RSV in the negative specimens (negative nasal matrix), so the baseline was changed to a manual baseline (starting at $\mathrm { C t } 3 . 0$ and ending at Ct 15.0). This resolved the false positive RSV detection and had no effect on detection of hMPV or the PRC. Clinical specimens were tested to confirm that the change to a manual baseline reduced RSV false positives but did not affect RSV true positives. The data from this study is included below.

Table 5: Determination of Baseline   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>RSV Ct  Auto Baseline</td><td rowspan=1 colspan=1>RSV Ct  Manual Baseline (start atCt 3.0 and end at Ct 15.0)</td></tr><tr><td rowspan=5 colspan=1>RSV Clinical Negative</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>43.6</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>43.9</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=6 colspan=1>RSV Clinical Positive</td><td rowspan=1 colspan=1>17.8</td><td rowspan=1 colspan=1>17.7</td></tr><tr><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>38.8</td></tr><tr><td rowspan=1 colspan=1>17.0</td><td rowspan=1 colspan=1>17.1</td></tr><tr><td rowspan=1 colspan=1>21.8</td><td rowspan=1 colspan=1>22.1</td></tr><tr><td rowspan=1 colspan=1>21.7</td><td rowspan=1 colspan=1>21.8</td></tr></table>

Background fluorescence for the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay on the Life Technologies QuantStudio Dx was established as cycles 3.0-15.0.

# Threshold

Threshold analysis was conducted using contrived samples from the reproducibility study panel. Initial testing indicated that the use of the threshold settings previously established on the ABI 7500 Fast Dx platform (K122189) resulted in the detection of RSV false positives in the negative specimens (negative nasal matrix). The threshold for RSV was adjusted from $8 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ to $2 . 5 0 \times 1 0 ^ { 5 } \mathrm { \ : T C I D _ { 5 0 } / m L }$ and clinical specimens were tested to confirm that the change did not affect RSV true positives. The results of this study are included below.

Table 6: Determination of Threshold   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>RSV Ct Threshold 8.00x104 TCID50/mL</td><td rowspan=1 colspan=1>RSV Ct Threshold2. 50 x 105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV Clinical Negative</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=6 colspan=1>RSV Clinical Positive</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1>19.2</td></tr><tr><td rowspan=1 colspan=1>15.3</td><td rowspan=1 colspan=1>16.4</td></tr><tr><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>39.5</td></tr><tr><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>18.6</td></tr><tr><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>23.3</td></tr><tr><td rowspan=1 colspan=1>21.8</td><td rowspan=1 colspan=1>23.3</td></tr></table>

Analysis of contrived and clinical specimens indicated that there was no need to change the auto threshold settings for the hMPV and PRC targets from the original settings previously established on the ABI 7500 Fast Dx platform (K122189).

# Ct Cut-Off

A Ct cut-off study was conducted prior to the initiation of the clinical study. The clinical specimens used for this study were archival specimens from the original submission (K122189). A total of 63 RSV positive and $3 7 \mathrm { h M P V }$ positive specimens were evaluated for their Ct value range to determine the appropriate cut-off for the assay. For RSV 62/63 positives were detected at values less than $3 0 . 0 \mathrm { C t } .$ . For hMPV, 34/37 positives were detected at values less than $3 5 . 0 \mathrm { C t }$ .

The Ct cut-off was set at 40.0 cycles because this value will not decrease sensitivity for detection of positive clinical specimens.

# 2. Comparison studies:

# a. Method comparison with predicate device:

Method comparison was based on the results from the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay on the Life Technologies QuantStudio Dx compared to the results obtained form two FDA-cleared molecular tests (one test for RSV and a different test for hMPV). The testing description and data are listed in the Clinical Studies section (below).

# b. Matrix comparison:

This data was reviewed and determined to be satisfactory in K122189. The results of the study are described briefly below:

A transport media study was conducted to show compatibility of five different types of transport media: UTM, M4, M4-RT, M5, and M6 to the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay. RSV and hMPV were spiked into the media, extracted, and detected using the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay. No differences between the tested transport media were observed.

3. Clinical studies:

# a. Clinical Sensitivity and Specificity

The clinical study for this device was performed during the 2013 respiratory virus season (January-March 2013). One specimen per patient was collected at three US study sites. All 713 nasal or nasopharyngeal swab specimens were collected prospectively from patient samples with a physician’s order to perform testing for respiratory viruses. Samples were extracted within 72 hours using the bioMerieux NucliSENS easyMag extraction technology and then stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ until the time of testing. Performance on the QuantStudio Dx instrument was compared to performance of two FDA-cleared assays: a molecular assay for RSV and a molecular assay for hMPV. Sites 1 and 2 performed their own comparator testing. Aliquots of each specimen from site 3 were sent to site 1 for testing with the comparator devices.

There was an appropriate distribution of subjects from all ages and both sexes.

Table 7: Gender Distribution for Clinical Study   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>375 (52.6%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>338 (47.4%)</td></tr></table>

Table 8: Gender Distribution for Clinical Study   

<table><tr><td rowspan=1 colspan=1>Age (Years)</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>286 (40.1%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>200 (28.1%)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>116 (16.3%)</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>111 (15.6%)</td></tr></table>

Seven hundred and thirteen (713) nasal or nasopharyngeal swab specimen extracts were tested by both the Quidel assay and comparator device for RSV viral RNA. A total of 13 invalid specimens were removed from the analysis. Two of these specimens were invalid on initial and repeat testing with the subject device $( 0 . 3 \% )$ . Eleven specimens were invalid on initial and repeat testing on the comparator device $( 1 . 5 \% )$ . The table below details the results for the remaining 700 specimens.

Table 9: Clinical Study - Combined Site RSV Data   

<table><tr><td rowspan=1 colspan=4>Comparator: FDA-cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecular RSV +</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>hMPV Assay          Positive</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>116</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>584</td></tr><tr><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>588</td><td rowspan=1 colspan=1>700</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>105/112</td><td rowspan=1 colspan=1>93.8%</td><td rowspan=1 colspan=1>87.7% to96.9%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>577/588</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>96.7% to99.0%</td></tr></table>

Seven hundred and thirteen (713) nasal or nasopharyngeal swab specimens were tested by both the subject and comparator device for hMPV viral RNA. A total of six invalid specimens were removed from the analysis. Two of these specimens were invalid on initial and repeat testing with the subject device $( 0 . 3 \% )$ . Four specimens were invalid on initial and repeat testing on the comparator device $( 0 . 6 \% )$ . The table below details the results for the remaining 707 specimens.

Table 10: Clinical Study - Combined Site hMPV Data   

<table><tr><td rowspan=1 colspan=4>Comparator: FDA-cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecular RSV +</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>hMPV Assay          Positive</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>647</td><td rowspan=1 colspan=1>648</td></tr><tr><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>651</td><td rowspan=1 colspan=1>707</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>55/56</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>90.6% to 99.7%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>647/651</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.4% to 99.8%</td></tr></table>

Analysis of individual site data did not show any noteworthy differences between sites for either analyte. Therefore, pooling of clinical site data to generate the summary tables above was appropriate.

4. Clinical cut-off:

N/A

# 5. Expected values/Reference range:

The expected values for the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay on the QuantStudio Dx was $1 6 . 3 \%$ and $8 . 3 \%$ . Expected values were highest for both viruses in children under one year of age. The table below shows the expected values for all sites segregated by age group.

Table 11: Expected Values   

<table><tr><td rowspan=1 colspan=7>Expected Values for QuantStudio Dx (Winter 2013)</td></tr><tr><td rowspan=1 colspan=1>AgeGroup(Years)</td><td rowspan=1 colspan=1>Total (n)</td><td rowspan=1 colspan=1>Total # RSVPositive</td><td rowspan=1 colspan=1>RSV DetectionRate</td><td rowspan=1 colspan=1>Total (n)</td><td rowspan=1 colspan=1>Total # hMPVPositive</td><td rowspan=1 colspan=1>hMPV DetectionRate</td></tr><tr><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>37.2%</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>16.3%</td></tr><tr><td rowspan=1 colspan=1>1 to 5</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>29.0%</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>13.0%</td></tr><tr><td rowspan=1 colspan=1>6 to 10</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>11 to 15</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>16 to 21</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>&gt;21</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>7.5%</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>711*</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>16.3%</td><td rowspan=1 colspan=1>711*</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>8.3%</td></tr></table>

\* Two specimens were Invalid.

# N. Instrument Name:

QuantStudio Dx Real-Time PCR (RT-PCR) Instrument

# O. System Descriptions:

The QuantStudio Dx RT-PCR Instrument is a bench top real-time PCR instrument that uses fluorescent-based polymerase chain reaction (PCR) reagents to provide qualitative or quantitative detection of target nucleic acid sequences (targets) using real-time analysis from a 96-well (or 384-well) plate. Samples are prepared for nucleic acid testing manually using specific reagents indicated by the assay.

# 1. Modes of Operation:

The QuantStudio Dx RT-PCR Instrument has two modes of operation: IVD and Research Use Only (RUO), which cannot be executed simultaneously. The protocols that are performed on each instrument for IVD assays depend upon the assay-specific, closedmode application specifications that are installed on the system.

# 2. Software:

The software embedded on the QuantStudio Dx RT-PCR Instrument manages instrument operations and enables the user to monitor instrument status. The QuantStudio Dx RTPCR Instrument software is installed on a separate computer and communicates with the instrument to control the instrument and collect data. The QuantStudio Dx RT-PCR Instrument software performs data analysis and outputs analyzed results.

The diagnostic test parameters for the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay are predefined by Quidel in a Test Definition Document (TDD) that is imported into the QuantStudio Dx RT-PCR Instrument software. The TDD file is locked and the parameters cannot be changed by the user.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

3. Specimen Identification: Performed manually

4. Specimen Sampling and Handling:

There is no automated sample processing offered in conjunction with the QuantStudio Dx RT-PCR Instrument. Patient specimens are prepared (RNA extraction) using the bioMérieux NucliSENS easyMAG System. The extracted samples are added manually to the 96-well microplate.

# 5. Calibration:

The instrument’s optics (fluorescence parameters) are calibrated upon installation. After installation of the instrument, the user can perform all required regular maintenance and calibration using IVD-labeled calibration plates.

6. Quality Control:

Quality control is addressed for each sample by addition of an in-process control during sample processing and amplification in the assay. Additionally, there are external positive and negative controls available that may be used in accordance with the user lab standards.

# P. Other Supportive Instrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

The QuantStudio Dx RT-PCR Instrument software was reviewed in detail under K123955.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.