# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N

A. 510(k) Number: K132235

# B. Purpose for Submission:

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay was submitted to obtain a substantial equivalence determination for the IMDx C. difficile assay on the Abbott $m 2 0 0 0$ platform using unpreserved (i.e., fresh) or Cary-Blair preserved stool specimens.

# C. Measurand:

DNA target sequences encoding the tcdA and tcdB genes which are associated with toxigenic strains of Clostridium difficile.

# D. Type of Test:

Qualitative nucleic acid amplification test using real-time PCR technology to amplify and detect the tcdA and tcdB genes.

# E. Applicant:

Intelligent Medical Devices, Inc.

F. Proprietary and Established Names: IMDx C. difficile for Abbott m2000

# G. Regulatory Information:

1. Regulation section: 21 CFR §866.3130 - C. difficile Nucleic Acid Amplification Test Assay

2. Classification: Class II

3. Product code: OZN - Amplification assay for the detection of Clostridium difficile toxin genes from stool specimens of symptomatic patients OOI - Real-Time Nucleic Acid Amplification System

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is an in vitro diagnostic assay that uses real-time polymerase chain reaction (PCR) amplification for the qualitative detection of nucleic acids encoding the toxin A gene (tcdA) and toxin B gene (tcdB) sequences of toxigenic strains of Clostridium difficile in human liquid or soft stool specimens collected from patients suspected of having symptoms of Clostridium difficile infection.

The assay is intended to be performed on the Abbott m2000 System (which comprises the Abbott $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments) and is indicated for use as an aid in the diagnosis of Clostridium difficile infection. The test is intended to be used directly on liquid or soft stool specimens (unpreserved stool, or stool preserved in Cary Blair transport medium). Negative results do not preclude toxigenic C. difficile infection and should not be used as the sole basis for treatment or other patient management decisions. The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is intended for professional use. The device is not intended for point-of-care use.

# 2. Indication(s) for use:

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is an in vitro diagnostic assay that uses real-time polymerase chain reaction (PCR) amplification for the qualitative detection of nucleic acids encoding the toxin A gene (tcdA) and toxin B gene (tcdB) sequences of toxigenic strains of Clostridium difficile in human liquid or soft stool specimens collected from patients suspected of having symptoms of Clostridium difficile infection.

The assay is intended to be performed on the Abbott m2000 System (which comprises the Abbott $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments) and is indicated for use as an aid in the diagnosis of Clostridium difficile infection. The test is intended to be used directly on liquid or soft stool specimens (unpreserved stool, or stool preserved in Cary Blair transport medium). Negative results do not preclude toxigenic C. difficile infection and should not be used as the sole basis for treatment or other patient management decisions. The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is intended for professional use. The device is not intended for point-of-care use.

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: Abbott m2000sp: sample preparation and reagent mixing

Abbott $m 2 0 0 0 r t$ : amplification reaction and detection

# I. Device Description:

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is a qualitative, real-time PCR-based assay that targets the genes tcdA and tcdB; these genes are associated toxigenic strains of $C .$ . difficile. These toxin A and B genes can be found in human liquid or soft stool specimens from patients with symptoms of $C .$ . difficile associated disease. The assay is intended for use directly on patient samples and is intended to aid in the diagnosis of C. difficile-associated disease. Specimen preparation and reagent mixing occurs on the Abbott $m 2 0 0 0 s p$ instrument, while amplification and detection occurs on the Abbott $m 2 0 0 0 r t$ instrument. The Abbott $m 2 0 0 0 r t$ application software monitors the fluorescence emitted by each fluorescent probe, interprets all data, and provides a final result at the end of the cycling program. Differentiation of tcdA from tcdB is achieved by labeling the oligonucleotide probes with different colored fluorescent dyes. Assay results are reported as either the presence or absence of the tcdA and/or tcdB targets

A specimen processing control is introduced into each specimen during the sample extraction. This control is composed of an inactivated Gram-positive bacterium unrelated toxigenic C. difficile that is introduced in a defined quantity into each sample during processing; it is co-extracted with the specimen and co-amplified in the same PCR reaction as the tcdA and tcdB targets and serves to demonstrate that the entire assay process has proceeded within specification. Positive and negative controls are included with each run to ensure the integrity of the system.

# IMDx C. difficile for Abbott m2000 Kit

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay consists of two reagent kits packaged together: (1) a separate box for the Amplification Reagent Kit, and (2) a separate box for the Control Kit:

<table><tr><td>1) IMDx C. difficile for Abbott m2000 Amplification Reagent Kit</td></tr><tr><td>Each Amplification Reagent Kit contains two types of items:</td></tr><tr><td>a) IMDx Process Control-A:</td></tr><tr><td>4 vials, 0.6 mL per vial of inactivated bacteria in a buffered solution.</td></tr><tr><td> IMDx Amplification Reagent Packs:</td></tr><tr><td>• 4 Amplification Reagent Packs, 24 tests/pack. Each Amplification Reagent Pack contains:*</td></tr><tr><td>1 vial (0.408 mL) IMDx C. difficile for Abbott m2000 Amplification Reagent</td></tr><tr><td>consisting of synthetic oligonucleotides in a buffered solution, located in position 1 of the reagent pack. 1 vial (0.192 mL) IMDx PCR Reagent-A (DNA polymerase and dNTPs in a buffered solution with ROX™&quot; reference dye), located in position 3 of the</td></tr></table>

\*Each Amplification Reagent Pack, although capable of containing up to three reagent vials, contains only two reagent vials in each pack. These vials are located in positions 1 and 3, as marked on the Reagent Pack; position 2, in the middle of the pack, is empty.

<table><tr><td>2) IMDx C. difficile for Abbott m2000 Control Kit</td></tr><tr><td>Each Control Kit contains two types of items:</td></tr><tr><td>IMDx Negative Control-A:</td></tr><tr><td>•6 tubes (2.5 mL per tube) containing a buffered solution with carrier DNA isolated from</td></tr><tr><td>Bacteroides sp.</td></tr><tr><td>b) IMDx C. difficile for Abbott m2000 Positive Control</td></tr><tr><td>6 tubes (2.5 mL per tube) containing a mixture of synthetic C. difficile DNA in buffer.</td></tr></table>

# Interpretation of Results

The decision algorithm for the IMDx C. difficile for Abbott $m 2 0 0 0$ assay is embedded in the Abbott $m 2 0 0 0 r t$ application software (v3.0.338.0). The interpretation of assay results is provided by analyte results and the process control result.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=5>Reported Target Output</td></tr><tr><td rowspan=1 colspan=1>tcdA</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>tcdB</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>ProcessControl</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected / NotDetected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Result Call</td><td rowspan=1 colspan=1>tcdA detected</td><td rowspan=1 colspan=1>tcdB detected</td><td rowspan=1 colspan=1>tcdA and tcdBdetected</td><td rowspan=1 colspan=1>tcdA and tcdBnot detected</td><td rowspan=1 colspan=1>Invalid;no resultreported</td></tr><tr><td rowspan=1 colspan=1>Indicatespresence oftoxigenic C.</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): Quidel AmpliVue™ C. difficile Assay

2. Predicate 510(k) number(s): K123355

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">IMDx C. difficile forAbbott m2000(K132235)</td><td colspan="1" rowspan="1">Quidel AmpliVueC.difficile Assay(K123355)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The IMDx C. difficile forAbbott m2000 assay is an invitro diagnostic assay that usesreal-time polymerase chainreaction (PCR) amplification forthe qualitative detection ofnucleic acids encoding the toxinA gene (tcdA) and toxin B gene(tcdB) sequences of toxigenicstrains of Clostridium difficilein human liquid or soft stoolspecimens collected frompatients suspected of havingsymptoms of Clostridiumdifficile infection.The assay is intended to beperformed on the Abbott m2000System (which comprises theAbbott m2000sp and m2000rtinstruments) and is indicated foruse as an aid in the diagnosis ofClostridium difficile infection.The test is intended to be useddirectly on liquid or soft stoolspecimens (unpreserved stool,or stool preserved in Cary Blairtransport medium). Negativeresults do not precludetoxigenic C. difficile infectionand should not be used as thesole basis for treatment or otherpatient management decisions.The IMDx C. difficile forAbbott m2000 assay is intendedfor professional use. Thedevice is not intended for point-of-care use.</td><td colspan="1" rowspan="1">The AmpliVue™ C. difficileAssay is an in vitro diagnostictest for the direct, qualitativedetection of the Clostridiumdifficile Toxin A gene (tcdA)in unformed stool specimensof patients suspected ofhaving Clostridium difficile-associated disease (CDAD).The AmpliVue™ C. difficileAssay is intended for use asan aid in diagnosis of CDAD.The assay utilizes helicase-dependent amplification(HDA) for the amplificationof a highly conservedfragment of the Toxin A genesequence and a self-containeddisposable amplicon detectiondevice that allows for manualevaluation of assay results.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Soft or liquid stool(unpreserved stool, or stool</td><td colspan="1" rowspan="1">Unformed stool</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">IMDx C. difficile forAbbott m2000(K132235)</td><td colspan="1" rowspan="1">Quidel AmpliVue™C.difficile Assay(K123355)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">preserved in Cary Blairtransport medium).</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Real-time PCR DNAamplification</td><td colspan="1" rowspan="1">Isothermal nucleic acidamplification</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Toxin A gene (tcdA)Toxin B genes (tcdB, includingvariant tcdBv)</td><td colspan="1" rowspan="1">Toxin A gene (tcdA)</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Positive Control, NegativeControl and Process Controlincluded in the kit</td><td colspan="1" rowspan="1">Process Control included inthe kit.Positive and NegativeControls not included in kit;separate control kitcommercially available</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>IMDx C. difficile forAbbott m2000(K132235)</td><td rowspan=1 colspan=1>Quidel AmpliVue&#x27;™&quot; C. difficileAssay(K123355)</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Assay uses the Abbott m2000System</td><td rowspan=1 colspan=1>Self-contained, disposablecassette with an ampliconcartridge and detection chamber</td></tr><tr><td rowspan=1 colspan=1>Interpretation ofTest Results</td><td rowspan=1 colspan=1>Abbott m2000rt applicationsoftware.</td><td rowspan=1 colspan=1>Visual read</td></tr><tr><td rowspan=1 colspan=1>SamplePreparation</td><td rowspan=1 colspan=1>Automated by Abbott m2000sp</td><td rowspan=1 colspan=1>Manual</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile – Draft Guidance for Industry and FDA Staff (issued on November 29, 2010).

# L. Test Principle:

The IMDx C. difficile for Abbott $m 2 0 0 0$ assay is a qualitative, in vitro diagnostic assay containing reagents for the real-time PCR amplification, detection, and differentiation of nucleic acids from C. difficile bacteria harboring tcdA and/or tcdB genes. Detection of the tcdA and tcdB genes is measured by the presence of fluorescently-labeled oligonucleotide probes that generate a fluorescent signal when specifically bound to amplified tcdA and/or tcdB PCR products; differentiation of tcdA from tcdB is attained by labeling the

oligonucleotide probes with different colored fluorescent dyes. The IMDx C. difficile for Abbott $m 2 0 0 0$ assay includes inactivated bacteria as a full process control; this inactivated bacterium is unrelated to $C .$ . difficile and introduced into each specimen during sample preparation such that it is co-extracted and co-amplified with each specimen. This full process control serves to demonstrate that the entire assay process has proceeded within specification for individual specimens. The Abbott $m 2 0 0 0$ system consists of two instruments: the Abbott $m 2 0 0 0 s p$ (for sample preparation) and the Abbott $m 2 0 0 0 r t$ (for realtime PCR amplification and detection). The assay is intended to be used directly on fresh (i.e., unpreserved), or Cary-Blair preserved stool specimens collected from patients to aid in the diagnosis of Clostridium difficile infection.

Fresh/raw or Cary-Blair preserved stool specimens are initially subjected to sample preparation on the Abbott $m 2 0 0 0 s p$ , an automated sample preparation system. Sample preparation lyses bacteria present in the sample, including any toxigenic $C .$ . difficile, to make the tcdA and tcdB target nucleic acids accessible for amplification and to remove potential amplification inhibitors. At the completion of the sample processing procedure, the resulting bacterial lysate is transferred to an Abbott 96-Deep-Well. The Abbott $m 2 0 0 0 s p$ also combines the IMDx C. difficile for Abbott $m 2 0 0 0$ Amplification Reagent components and dispenses the resulting Master Mix into the Abbott 96-Well Optical Reaction Plate. After manual application of the Abbott Optical Adhesive Cover, the plate is ready for transfer to the Abbott $m 2 0 0 0 r t$ .

Amplification/detection takes place on the Abbott $m 2 0 0 0 r t$ instrument using real-time PCR techniques; during each round of PCR amplification the fluorescent probes anneal to the amplified target DNA, if present. The probes are labeled with different fluorescent molecules allowing tcdA, tcdB and the IMDx Process Control-A targets to be distinguished from each other. The probes are single-stranded, linear DNA oligonucleotides modified with a fluorescent moiety covalently linked to one end of the probe and a quenching moiety to the other end. When the probe binds to its complementary sequence in the target during amplification, the fluorophore separates from the quenchers, thus allowing fluorescent emission and detection. Since this fluorescence occurs during every cycle, the PCR reaction can be read in real-time. Positive and negative controls are included with each run to ensure the integrity of the system.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Assay precision was measured in four independent studies:

user-to-user reproducibility; · lot-to-lot reproducibility; · instrument-to-instrument reproducibility, and within-laboratory repeatability.

A seven-member precision panel was comprised of intact organisms from two strains of C. difficile: a NAP-1 strain (ATCC BAA-1870), and the tcdB-variant 1470 (ATCC 43598). Each strain was formulated at three target levels:

i. positive (corresponding to a concentration associated with ${ \sim } 2 { - } 3 \mathbf { x }$ LoD), ii. low positive (1x LoD), and iii. high negative $_ { ( \sim 0 . 0 5 \mathrm { ~ x ~ } }$ LoD, estimated to have a 20 to $80 \%$ positivity rate).

Panel members were made fecal matrix buffer to simulate clinical specimens. The seventh panel member was a true negative sample that contained TE Buffer plus fecal matrix alone, without $C .$ . difficile organism added.

# Site-to-Site Reproducibility

For the reproducibility study the panel members were randomized and blinded. Each panel member was tested in replicates of three, for six days, at three study sites, by two technologists at each site for a total of 36 experimental runs; each operator performed one run each day. The entire study was conducted using one instrument system $m 2 0 0 0 s p$ and $ { \mathbf Ḋ } 2 0 0 0 r t )$ at each site and one reagent lot of the IMDx $C .$ . difficile for Abbott $m 2 0 0 0$ assay. The results and percent agreement (observed/tested) at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>SITE</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Site #1</td><td rowspan=1 colspan=2>Site #2</td><td rowspan=1 colspan=2>Site #3</td></tr><tr><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598High Negative</td><td rowspan=1 colspan=1>32/36</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>27/36</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27/34</td><td rowspan=1 colspan=1>79%</td><td rowspan=1 colspan=1>86/106</td><td rowspan=1 colspan=1>81%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Low Positive</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Positive</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870High Negative</td><td rowspan=1 colspan=1>31/35</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>31/36</td><td rowspan=1 colspan=1>86%</td><td rowspan=1 colspan=1>30/35</td><td rowspan=1 colspan=1>86%</td><td rowspan=1 colspan=1>92/106</td><td rowspan=1 colspan=1>87%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Low Positive</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>106/107</td><td rowspan=1 colspan=1>99%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Positive</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>241/249</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>237/251</td><td rowspan=1 colspan=1>94.4%</td><td rowspan=1 colspan=1>236/249</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>714/749*</td><td rowspan=1 colspan=1>95.3%</td></tr></table>

$\ast _ { \mathrm { O f } }$ the 758 samples tested 9 samples resulted in an instrument error and were withdrawn from the study. The overall invalid rate was $1 \%$ .

# Lot-to Lot-Reproducibility

Lot-to-lot reproducibility was assessed using three lots of the IMDx C. difficile for Abbott $m 2 0 0 0$ assay. One experiment was run for each of the three lots (for a total of three runs). A single operator performed the study using a single instrument pair $( m 2 0 0 0 s p$ and $m 2 0 0 0 r t )$ . Panel members were tested in replicates of six for each run. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>IMDx C. difficile for Abbott m2000 Lot</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Lot #1</td><td rowspan=1 colspan=2>Lot #2</td><td rowspan=1 colspan=2>Lot #3</td></tr><tr><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598High Negative</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>15/18</td><td rowspan=1 colspan=1>83%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870High Negative</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>4/5*</td><td rowspan=1 colspan=1>80%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>12/17</td><td rowspan=1 colspan=1>71%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>39/42</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>39/41</td><td rowspan=1 colspan=1>95.1%</td><td rowspan=1 colspan=1>39/42</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>117/125</td><td rowspan=1 colspan=1>93.6%</td></tr></table>

$\ast _ { \mathrm { O f } }$ the 126 samples tested, one ATCC BAA-1870 sample was invalidated due to an instrument error and was withdrawn from the study. The overall invalid rate was ${ < } 1 \%$ .

# Instrument-to-Instrument Reproducibility

Instrument-to-instrument reproducibility was determined independently for both the Abbott $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments. To measure $m 2 0 0 0 s p$ instrument variability, the test panel was run on three different $m 2 0 0 0 s p$ instruments. Panel members were run in replicates of six. Runs were performed in succession by the same operator, using a single IMDx $C .$ . difficile for Abbott $m 2 0 0 0$ assay lot. Once the master mix assembly portion of the protocol was executed by the different $m 2 0 0 0 s p$ instruments, the assembled reaction plates were run using a single $m 2 0 0 0 r t$ instrument. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>Abbott m2000sp Instrument-to-Instrument Reproducibility</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Instrument #1</td><td rowspan=1 colspan=2>Instrument #2</td><td rowspan=1 colspan=2>Instrument #3</td></tr><tr><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598High Negative</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870High Negative</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>12/18</td><td rowspan=1 colspan=1>67%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>5/5*</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>39/42</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>40/42</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>39/41</td><td rowspan=1 colspan=1>95.1%</td><td rowspan=1 colspan=1>118/125</td><td rowspan=1 colspan=1>94.4%</td></tr></table>

$\ast _ { \mathrm { O f } }$ the 126 samples tested, one ATCC BAA-1870 sample was invalidated due to an instrument error and was withdrawn from the study. The overall invalid rate was ${ < } 1 \%$ .

To measure $m 2 0 0 0 r t$ variability, a single $m 2 0 0 0 s p$ instrument was used in one experiment. Six replicates of each panel member were prepared. The eluted nucleic acids from the sample extractions were automatically transferred to an Abbott 96- well Deep Well Plate by the $m 2 0 0 0 s p$ per normal operational procedures. Aliquots from one deep well plate were then used to assemble three separate, identical Abbott 96-well Optical Reaction Plates, using the same lot of the IMDx $C .$ difficile for Abbott $m 2 0 0 0$ assay. Each of the three identical Abbott 96-Well Optical Reaction plates were then run on three separate Abbott $m 2 0 0 0 r t$ instruments, by the same operator. The expected results and percent agreement at each site, as well as the overall agreement were reported as follows:

<table><tr><td rowspan=3 colspan=1>Sample ID</td><td rowspan=1 colspan=6>Abbott m2000rt Instrument-to-Instrument Reproducibility</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>Instrument #1</td><td rowspan=1 colspan=2>Instrument #2</td><td rowspan=1 colspan=2>Instrument #3</td></tr><tr><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td><td rowspan=1 colspan=1>#observedresults /#tested</td><td rowspan=1 colspan=1>%Agreem&#x27;t</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598High Negative</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>16/18</td><td rowspan=1 colspan=1>89%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870High Negative</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>13/18</td><td rowspan=1 colspan=1>72%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Low Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Positive</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>39/42</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>40/42</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>40/42</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>119/126</td><td rowspan=1 colspan=1>94.4%</td></tr></table>

Negative samples showed an overall negativity rate of $100 \%$ ; no false positive results were observed. High Negative samples were expected to provide a negativity rate of $20 \mathrm { - } 8 0 \%$ ; the observed negativity rate was $8 1 . 2 \%$ , and $8 4 . 1 \%$ for the ATCC BAA1870 and ATCC 43598 strains, respectively. Low Positive samples were expected to provide a positivity rate of $2 9 5 \%$ ; the observed positivity rate was $9 8 \%$ . Positive samples were expected to provide a positivity rate of $100 \%$ ; the observed positivity rate was $100 \%$ . There were 11 instrument errors out of the 1,136 samples tested in these studies $( < 1 \% )$ .

# Within-Laboratory Repeatability

For the repeatability study, the seven-member panel was tested twice a day for a total of 12 days. Panel members were tested in replicates of three for each run (for a total of 504 data points for the 24 runs). The entire study was conducted by one technician using one instrument pair $( m 2 0 0 0 s p$ and $m 2 0 0 0 r t )$ and one reagent lot of the IMDx $C .$ . difficile for Abbott $m 2 0 0 0$ assay.

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1># expectedresults/# tested</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>95% Confidence Interval</td></tr><tr><td rowspan=3 colspan=1>C. difficile NAP-1(ATCC BAA-1870)</td><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>57/70</td><td rowspan=1 colspan=1>81%</td><td rowspan=1 colspan=1>[0.71, 0.89]</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>70/71</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>[0.92, 1.00]</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>[0.94, 1.00]</td></tr><tr><td rowspan=3 colspan=1>C. difficile1470 tcdB-variant(ATCC 43598)</td><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>61/72</td><td rowspan=1 colspan=1>85%</td><td rowspan=1 colspan=1>[0.75, 0.91]</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>71/72</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>[0.92, 1.00]</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>[0.94, 1.00]</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>[0.94, 1.00]</td></tr></table>

$\ast _ { \mathrm { O f } }$ the 504 samples tested 3 samples resulted in an instrument error and were withdrawn from the study. The overall invalid rate was $1 \%$ .

The details regarding the average (Avg), standard deviation (SD), and coefficient of variation $( \% C V )$ in cycle number (CN) for tcdA, tcdB and the Internal Control (IC; Process Control-A) for the repeatability study were reported below. The IMDx $C .$ . difficile for Abbott $m 2 0 0 0$ assay provided reproducible results.

<table><tr><td rowspan=1 colspan=1>cYanmaGYellowb</td><td rowspan=1 colspan=3>tcdA CN</td><td rowspan=1 colspan=3>tcdB CN</td><td rowspan=1 colspan=3>IC CN</td><td rowspan=1 colspan=3>% Agreement</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Expect</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870High Negative</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>42.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>81%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Low Positive</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>99%</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870Positive</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598High Negative</td><td rowspan=1 colspan=1>40.2</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>41.8</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598 LowPositive</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>40.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>99%</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598Positive</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>40.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>-1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The IMDx Process Control-A is composed of inactivated Gram-positive bacteria unrelated to C. difficle and is introduced into each sample during specimen processing; therefore, an IMDx Process Control-A result is generated with each specimen result. Positive and negative Quality Controls are run with each IMDx $C .$ . difficile for Abbott m2000 assay run.

There were a total of 5,722 replicates run during analytical verification; a total of 56 $( 0 . 9 8 \% )$ errors were observed. Of these 56 errors, 54 were due to instrument failures. There were a total 1,583 evaluable specimens in the clinical study, and 1,718 assay results were generated during testing (including both initial and repeat runs). The reported overall error rate during the clinical study was $3 . 4 \%$ (59/1,718); the majority of these $( > 8 2 \%$ [47/59]) were due to IMDx Process Control-A (i.e., the internal control) failures. Assay Positive and Negative Controls were tested with each assay run during the trial. Throughout the course of the study there were a total of 90 runs performed; only three runs failed due to quality control failures (in two of these three runs both the positive and negative controls failed). Therefore, the overall QC runfailure rate was $3 . 3 \%$ (3/90), while the total QC failure rate was $0 . 3 \%$ (5/1,718). The remaining failures $( 0 . 5 \% )$ were due to instrument issues (7/1,718). Eighteen (18) samples yielded an unresolved error message after repeat testing and were eventually categorized as “Invalid.” The rate of failure was not excessive; the assay control strategy adequately evaluated the integrity of the result.

# Specimen Stability Studies

The following specimen types were tested:

Unpreserved (fresh/raw) stool Cary-Blair preserved stool

The data provided support that each specimen type was stable when stored under the following conditions:

Unpreserved (fresh/raw) stool: o 7 days at $2 \mathrm { { } ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ . $\bigcirc$ 60 days at ${ - 3 0 ^ { \circ } C }$ to ${ } _ { - 1 0 ^ { \circ } \mathrm { C } }$ .   
Cary-Blair preserved stool: $\bigcirc$ 4 days at $2 \mathrm { { } ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ . $\bigcirc$ 60 days at ${ - 3 0 ^ { \circ } C }$ to ${ } _ { - 1 0 ^ { \circ } \mathrm { C } }$ .   
Specimens (both unpreserved and preserved) were stable for one (1) freeze   
thaw cycle.

# d. Detection limit:

The limit of detection (LoD) of the IMDx C. difficile for Abbott m2000 assay was determined using three strains of toxigenic C. difficile. These included VPI10463 (ATCC 43255, toxinotype 0), 1470 (ATCC 43598, toxinotype VIII, tcdB-variant), and 4118 (ATCC BAA-1870, toxinotype III, BI/NAP1/027). A range finding study was initially performed, and then six serial 2-fold dilutions around the appropriate concentration were tested. A total of 60 replicates of each dilution were tested to determine the LoD of raw/fresh stool, whereas a total of 20 replicates were tested to determine the LoD of Cary-Blair preserved stool specimens. All dilutions were confirmed by colony counting, and three lots of the IMDx C. difficile for Abbott $m 2 0 0 0$ assay were used in the raw/fresh LoD determinations (20 replicates for each lot). The concentration of each strain at which $2 9 5 \%$ of the replicates was detected is as follows:

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Raw/Fresh Stool</td><td rowspan=1 colspan=1>Cary-Blair Preserved Stool</td></tr><tr><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>337 CFU/mL</td><td rowspan=1 colspan=1>463 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598</td><td rowspan=1 colspan=1>256CFU/mL</td><td rowspan=1 colspan=1>861 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA 1870</td><td rowspan=1 colspan=1>67 CFU/mL</td><td rowspan=1 colspan=1>134 CFU/mL</td></tr></table>

The LoD for fresh/raw stool specimens was determined to be:

337 CFU/mL.

The LoD for Cary-Blair preserved stool specimens was determined to be:

861 CFU/mL.

# Analytical Reactivity

Thirty-one (31) C. difficile strains and/or clinical isolates were tested for reactivity with the IMDx C. difficile for Abbott m2000 assay. Strains were reported to originate from at least nine states and four countries (USA, UK, Belgium, and Switzerland); 11 toxinotypes were represented (0, II, III, IV, V, VIII, IX, X, XII, XXI, and XXII). Each strain was diluted in buffer plus fecal matrix at 2-3X LoD; three replicates were tested. All strains were detected by the assay, demonstrating that the IMDx $C .$ . difficile Assay for Abbott $m 2 0 0 0$ can detect a broad range of toxinotypes.

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Toxintype</td><td rowspan=1 colspan=1>Origin</td></tr><tr><td rowspan=1 colspan=1>ATCC 17857</td><td rowspan=1 colspan=1>870</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC 17858</td><td rowspan=1 colspan=1>1253</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC43594</td><td rowspan=1 colspan=1>W1194</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human feces; Belgium</td></tr><tr><td rowspan=1 colspan=1>ATCC43596</td><td rowspan=1 colspan=1>545</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human feces;; Belgium</td></tr><tr><td rowspan=1 colspan=1>ATCC43599</td><td rowspan=1 colspan=1>2022</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human feces;Belgium</td></tr><tr><td rowspan=1 colspan=1>ATCC43600</td><td rowspan=1 colspan=1>2149</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Humanfeces;Belgium</td></tr><tr><td rowspan=1 colspan=1>ATCC 51695</td><td rowspan=1 colspan=1>BDMS 18 AN</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Baltimore, MD, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC 700792</td><td rowspan=1 colspan=1>14797-2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human feces; Michigan, USA,</td></tr><tr><td rowspan=1 colspan=1>ATCC 9689</td><td rowspan=1 colspan=1>90556-M6S</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1382</td><td rowspan=1 colspan=1>630</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Switzerland</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1871</td><td rowspan=1 colspan=1>4111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Human; New Jersey, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1872</td><td rowspan=1 colspan=1>4206</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Human; Maine, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1873</td><td rowspan=1 colspan=1>5283</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Human; New York, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1874</td><td rowspan=1 colspan=1>4205</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Human; Oregon, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1875</td><td rowspan=1 colspan=1>5325</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>Human; Georgia,,USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2155</td><td rowspan=1 colspan=1>LBM 0801058</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human; New Mexico, USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2156</td><td rowspan=1 colspan=1>LBM 0801040</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Human; Cambridge, UK</td></tr><tr><td rowspan=1 colspan=1>CCUG 20309</td><td rowspan=1 colspan=1>8864</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Birmingham, UK</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix NAP-1</td><td rowspan=1 colspan=1>Loyola-02</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>278 (II) BZ1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>464 (IV) AN1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>4092 (VIII) CF2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>5572 (VIII) CF4</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>3430 (IX) AH1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IX</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>1753 (XII) AL1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>5090 (XXI) V2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>XXI</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>3130 (XXII) BW1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>Illinois,USA</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1806</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1808</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1811</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Unknown</td></tr></table>

# e. Analytical specificity:

Cross reactivity was performed using a panel of 120 test organisms to evaluate any cross-reactivity with the IMDx $C .$ . difficile for Abbott $m 2 0 0 0$ assay. Bacteria were obtained from commercial sources, checked for purity and tested from fresh cultures at a concentration $\mathrm { { \Delta ) f \geq 1 } \times 1 0 ^ { 6 } \ C F U / m L }$ . Viruses were acquired commercially and tested from frozen stocks at a concentration $\mathrm { \Delta \mathrm { { 9 f } \geq 1 } \times 1 0 ^ { 5 } \mathrm { { T C I D 5 0 / m L } } }$ . Nucleic acids were tested at genomic equivalents $1 \times 1 0 ^ { 6 }$ copies $/ \mathrm { m L }$ for bacteria, and ${ \ge } 1 \times 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses. Samples were initially evaluated in singlicate; if cross reactivity was observed, the organism was retested in replicates of six. All samples

were prepared by diluting microorganisms or DNA into buffer plus fecal matrix. Cross reactivity of Clostridium botulinum was analyzed in silico.   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC19606</td><td colspan="1" rowspan="1">Enterococcus faecium vanA</td><td colspan="1" rowspan="1">ATCC 700221</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">ATCC17925</td><td colspan="1" rowspan="1">Enterococcus gallinarum vanC</td><td colspan="1" rowspan="1">ATCC 49573</td></tr><tr><td colspan="1" rowspan="1">Adenovirus (Type 40)</td><td colspan="1" rowspan="1">ZMC 0810084CF</td><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 8043</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ZMC 0601715</td><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49427</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp.Faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">Enterovirus (Type 71)</td><td colspan="1" rowspan="1">ZMC 0810047CF</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">ATCC 35098</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 11778</td><td colspan="1" rowspan="1">Escherichia coli O157</td><td colspan="1" rowspan="1">ZMC 0801622</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 13472</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">ATCC 43185</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">ATCC 33650</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">ATCC 43183</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">ATCC 8501</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">ATCC 15703</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">ATCC 14019</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43479</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni</td><td colspan="1" rowspan="1">ATCC 33292</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 13337</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">ZMC 0601486;Z040</td></tr><tr><td colspan="1" rowspan="1">Candida catenulate</td><td colspan="1" rowspan="1">ATCC 10565</td><td colspan="1" rowspan="1">Homo sapiens</td><td colspan="1" rowspan="1">ATCC MGC-15492</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 33496</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">ZMC D-UW3; Z054</td><td colspan="1" rowspan="1">Klebsiella pneumonia subsp.pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">ATCC 25405</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ZMC 0601540</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">ATCC 23272</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ZMC 0601745</td><td colspan="1" rowspan="1">Lactococcus lactis subsp. lactis</td><td colspan="1" rowspan="1">ATCC 11454</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ATCC 51115</td><td colspan="1" rowspan="1">Leminorela grimontii</td><td colspan="1" rowspan="1">ATCC 33999</td></tr><tr><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">ATCC 8260</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">ATCC 19120</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">ATCC 638</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">ATCC 33090</td></tr><tr><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">ATCC BAA-613</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ZMC 0801543</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">ATCC 19398</td><td colspan="1" rowspan="1">Norovirus (Type II)</td><td colspan="1" rowspan="1">ZMC 0810087CF</td></tr><tr><td colspan="1" rowspan="1">Clostridium chauvoei</td><td colspan="1" rowspan="1">ATCC 11957</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td><td colspan="1" rowspan="1">ATCC 14963</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43593</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43601</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Clostridium fallax</td><td colspan="1" rowspan="1">ATCC 19400</td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">ATCC 9656</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">ATCC 19401</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">ATCC 14501</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ZMC 0601589</td></tr><tr><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">ATCC 27757</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">ATCC 19402</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9250</td></tr><tr><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">ATCC 49531</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">Clostridium paraputrificum</td><td colspan="1" rowspan="1">ATCC 25780</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ZMC 0601585</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ZMC 0601722</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">ATCC 25582</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">ZMC MA-104</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">ATCC 35704</td><td colspan="1" rowspan="1">Ruminococcus bromii</td><td colspan="1" rowspan="1">ATCC 27255</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">ATCC 9714</td><td colspan="1" rowspan="1">Salmonella choleraesuis subsp.choleraesuis</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="1" rowspan="1">Clostridium sphenoides</td><td colspan="1" rowspan="1">ATCC 19403</td><td colspan="1" rowspan="1">Salmonella enterica subsp.</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">enterica</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium spiroforme</td><td colspan="1" rowspan="1">ATCC 29900</td><td colspan="1" rowspan="1">Salmonella enterica subsp.arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">ATCC 15579</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Clostridium symbiosum</td><td colspan="1" rowspan="1">ATCC 14940</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Clostridium tertium</td><td colspan="1" rowspan="1">ATCC 14573</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">ATCC 19406</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ZMC 0601609</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">ATCC 25986</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">ATCC 33798</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ZMC 0601675</td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus (Type B4)</td><td colspan="1" rowspan="1">ZMC 0810075CF</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (AD-169)</td><td colspan="1" rowspan="1">ZMC 0810003CF</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13637</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">ATCC 29098</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ZMC 0601545</td></tr><tr><td colspan="1" rowspan="1">Echovirus (Type 11)</td><td colspan="1" rowspan="1">ZMC 0810023CF</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">ATCC 43078</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">ATCC 25559</td><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">ATCC 19436</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">ATCC 10790</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">Vibrio cholera</td><td colspan="1" rowspan="1">ATCC 25870</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ZMC 0601565</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">ATCC 43198</td><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">ATCC 43970</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">ATCC 51266</td><td colspan="1" rowspan="1">Yersinia rohdei</td><td colspan="1" rowspan="1">ATCC 43380</td></tr></table>

There were two instances where the initial testing resulted in a detected call: nontoxigenic C. difficile (ATCC 43601) and Proteus penneri (ZMC 0801589). In both cases, none of the additional six replicates tested resulted in a detected call.

In silico analysis showed that the IMDx C. difficile for Abbott $\mathrm { m } 2 0 0 0$ assay had no predicted cross-reactivity for C. botulinum. The data indicates that there is no cross reactivity when using the organisms tested.

# Microbial Interference

To assess if there was any microbial interference with the IMDx C. difficile for Abbott $m 2 0 0 0$ assay, the same panel used to assess analytical specificity was added to tubes, each containing one of three strains of toxigenic C. difficile in buffer plus fecal matrix. Each strain was tested in triplicate for each panel member; strains were present in the samples at a concentration corresponding to 2-3x LoD. If interference was observed, the organism was retested in replicates of six. The three strains utilized included:

§ NAP-1 (ATCC BAA-1870) § VPI 10463 (ATCC 43255, toxinotype 0), and 1470 (ATCC 43598, toxinotype VIII)

In the microbial interference study 118 of the tested strains showed no obvious interference with the initial test. Two organisms, Clostridium scindens and Porphyromonas asaccharolytica, required retesting. For Clostridium scindens, one of the three replicates was not detected for the C. difficile ATCC 43598, and there were two instrument errors in the initial run for C. difficile ATCC 43255. When the test was repeated with an additional six replicates using Clostridium scindens in

combination with the three C. difficile test strains, all replicates were detected. For Porphyromonas asaccharolytica, two of three replicates were not detected for $C .$ . difficile ATCC BAA-1870 and one of three replicates were not detected with the $C .$ . difficile ATCC 43598. When the test was repeated with an additional six replicates, all replicates were detected.

In silico analysis predicted that interfering PCR products would not be produced by the inclusion of $C .$ . botulinum organisms in a reaction. In summary, no evidence of microbial interference was observed for any of the 118 test organisms included in the analysis.

# Potentially Interfering Substances

To evaluate the potential of substances typically found in fecal specimens to interfere with the IMDx C. difficile for Abbott m2000 assay, three strains of C. difficile were tested in the presence of such substances. The strains used included:

§ NAP-1 (ATCC BAA-1870) § VPI 10463 (ATCC 43255, toxinotype 0), and § 1470 (ATCC 43598, toxinotype VIII)

Substances were diluted in buffer plus fecal matrix to concentrations that would either replicate or exceed the highest concentration expected to be found in a clinical sample. Each of the three C. difficile strains were tested in triplicate at $2 { - } 3 \mathbf { x } \ \mathrm { L o D }$ in the presence of each substance. No interference was observed with any of the substances tested. The test panel consisted of the 23 substances shown in the table below:

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Active Ingredient(s) inSubstance</td><td colspan="1" rowspan="1">Concentration of SubstanceTested</td></tr><tr><td colspan="1" rowspan="1">Miconazole nitrate cream</td><td colspan="1" rowspan="1">Miconazole Nitrate</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Preparation H®</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">Zinc oxide</td><td colspan="1" rowspan="1">40% w/w paste</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">Hemorrhoid gel</td><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Witch Hazel</td><td colspan="1" rowspan="1">N/A*</td></tr><tr><td colspan="1" rowspan="2">Gaviscon®</td><td colspan="1" rowspan="1">Aluminum hydroxide</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Magnesium carbonate</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">TUMS</td><td colspan="1" rowspan="1">Calcium carbonate</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tagamet®</td><td colspan="1" rowspan="1">Cimetidine</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Prilosec® (delayed release)</td><td colspan="1" rowspan="1">Omeprazole magnesium</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="1">Condoms</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">7% v/v</td></tr><tr><td colspan="1" rowspan="1">Imodium</td><td colspan="1" rowspan="1">Loperamide HCI</td><td colspan="1" rowspan="1">0.00667 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Pepto Bismol®</td><td colspan="1" rowspan="1">Bismuth Subsalicylate</td><td colspan="1" rowspan="1">0.87 mg/mL</td></tr><tr><td colspan="1" rowspan="1">ExLax</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Vancomycin HCl</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">12.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Aleve</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Moist Towelettes</td><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">0.12% w/v</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">Glucose, hormones, enzymes,iron, ions, etc</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid (fecal fat)</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Stearic acid (fecal fat)</td><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">4 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">5 mg/mL</td></tr></table>

$^ { * } \mathrm { { N } } / \mathrm { { A } } =$ not applicable

# Target Carryover Study

Five assay runs were performed with alternating high positive and negative samples to assess the potential for cross-over or carryover contamination. A total of 470 specimens were run: 235 high positive and 235 negative specimens. One strains of $C .$ . difficile was used in this study. A positive signal for both tcdA and tcdB was detected in one run for a single negative sample; the carryover rate was $0 . 4 \%$ (1/235).

# f. Assay cut-off:

The cut-off for the IMDx C. difficile for Abbott $m 2 0 0 0$ assay was determined using Abbott’s proprietary maxRatio algorithm for real-time PCR data analysis. The maxRatio method identifies a consistent point within or very near the exponential region of the PCR signal. Compared to other analysis techniques that generate only a cycle number, the maxRatio method generates several measurements of amplification including cycle numbers (CN), relative measures of amplification efficiency and curve shape. By using these values, the maxRatio method can achieve a reliable reactive/nonreactive determination along with quantitative evaluation. For a result to be considered reactive, the fluorescence generated must cross a reactive threshold value (reactive threshold settings are based on “maxRatio values”). Initial maxRatio threshold parameters were based on analysis of results of characterized strains of toxigenic C.difficile. A third set included tcdB-variant toxigenic strains; however, since tcdB and tcdBv are labeled with the same fluorescent, the assay cannot differentiate tcdB from tcdBv. The assay design allows a positive result call to be made if a signal is detected for either the tcdA or tcdB/tcdBv component or both. Validation of threshold parameters came from analysis of 1,107 results (191 positive, and 916 negative) from the clinical study, compiled from five test sites. The results of this analysis are presented below:

<table><tr><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=2>tcdA</td><td rowspan=1 colspan=2>tcdB</td></tr><tr><td rowspan=1 colspan=1>Negativen=916</td><td rowspan=1 colspan=1>Positiven = 191</td><td rowspan=1 colspan=1>Negativen=916</td><td rowspan=1 colspan=1>Positiven = 191</td></tr><tr><td rowspan=1 colspan=1>maxRatio mean</td><td rowspan=1 colspan=1>0.0016</td><td rowspan=1 colspan=1>0.217</td><td rowspan=1 colspan=1>0.0037</td><td rowspan=1 colspan=1>0.093</td></tr><tr><td rowspan=1 colspan=1>maxRatio St Dev</td><td rowspan=1 colspan=1>0.0009</td><td rowspan=1 colspan=1>0.046</td><td rowspan=1 colspan=1>0.0027</td><td rowspan=1 colspan=1>0.020</td></tr><tr><td rowspan=1 colspan=1>maxRatio Range</td><td rowspan=1 colspan=1>0.000 - 0.009</td><td rowspan=1 colspan=1>0.035- 0.280</td><td rowspan=1 colspan=1>0.000-0.015</td><td rowspan=1 colspan=1>0.016- 0.124</td></tr><tr><td rowspan=1 colspan=1>Threshold set</td><td rowspan=1 colspan=2>0.010</td><td rowspan=1 colspan=2>0.015</td></tr><tr><td rowspan=1 colspan=1># standard deviations of thresholdfrom mean of negative samples</td><td rowspan=1 colspan=2>17.2</td><td rowspan=1 colspan=2>4.8</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: See clinical studies below.   
b. Matrix comparison: Not Applicable.

3. Clinical studies:

The performance of the IMDx C. difficile for Abbott $m 2 0 0 0$ assay was evaluated using an IRB approved protocol at seven (7) geographically diverse locations within the United States. Specimens were received as either fresh stool or in Cary Blair transport media (preserved). Results from the IMDx C. difficile for Abbott $m 2 0 0 0$ assay were compared to results obtained from direct culture followed by testing using an FDA cleared cell cytotoxicity assay. Reference testing was conducted at two sites.

A total of 1,583 specimens, consisting of 1,204 fresh stool (i.e., unpreserved) stool specimens, and 379 specimens Cary Blair transport media (i.e., preserved), were included in the final data set and analyzed for product performance. Eighteen (18) samples yielded an unresolved error message after repeat testing and were categorized as “Invalid;” the assay invalid rate was $1 . 1 \%$ (18/1,583). Therefore, a total of 1,565 valid specimens were included in the final analyses: 1,186 fresh stool specimens and 379 preserved stool specimens were evaluable. Among fresh stool specimens there were 118 concordant positive and 970 concordant negative results; there were also 79 false positive, and 19 false negative results. Among Cary-Blair preserved stool specimens there were 21 concordant positive and 333 concordant negative results; there were also 23 false positive, and 2 false negative results. Discordant specimens were subjected to further analysis by bi-directional sequencing.

There were a total of 123 discordant specimens. Ninety-one (91) of those specimens (73 false positives, 18 false negatives,) were tested additionally to resolve the discordances. Of the 73 false positives, bidirectional sequencing results confirmed the PCR products present in 52 cases to be toxigenic C. difficile, 17 samples remained discrepant, and four were indeterminate. Of the 18 false negatives, bidirectional sequencing confirmed the absence of

PCR products in 14 samples, and four remained discrepant. The clinical validation study results are summarized below:

<table><tr><td rowspan=1 colspan=5>Fresh/Raw (unpreserved) Stool SpecimensIMDx C. difficile for Abbott m2000 assayVS.DirectCulture.</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>79b</td><td rowspan=1 colspan=1>197</td></tr><tr><td></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>19a</td><td rowspan=1 colspan=1>970</td><td rowspan=1 colspan=1>989</td></tr><tr><td></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>1,049</td><td rowspan=1 colspan=1>1,186</td></tr></table>

$^ { \mathrm { a } } 1 6$ samples were sequenced: 13 were resolved as negative while 3 remained discrepant. $^ { \mathsf { b } } 5 3$ samples were sequenced: 40 were resolved as positive, 9 remained discrepant, and 4 had indeterminate results.

<table><tr><td rowspan=1 colspan=5>Direct Culture</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>23d</td><td rowspan=1 colspan=1>44</td></tr><tr><td></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>335</td></tr><tr><td></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>379</td></tr></table>

$^ \mathrm { c } _ { 2 }$ samples were sequenced: 1 was resolved as negative while 1 remained discrepant.   
${ ^ { \mathrm { d } } } _ { 2 0 }$ samples were sequenced: 12 were resolved as positive and 8 remained discrepant.

a. Clinical Sensitivity: See clinical performance studies above.

b. Clinical specificity: See clinical performance studies above.   
c. Other clinical supportive data (when a. and b. are not applicable): N/A

4. Clinical cut-off: See discussion in Assay Cut-off section above (Section M(1)(f)).

5. Expected values/Reference range: Expected values by specimen type among prospectively collected specimens:

<table><tr><td rowspan=1 colspan=7>Age and Gender Distribution of IMDx C. difficile for Abbott m2000 Assay Positive Results</td></tr><tr><td rowspan=3 colspan=1>Age Group</td><td rowspan=1 colspan=6>Specimen Type &amp; Gender*# Positive / # Enrolled(Prevalence [%])</td></tr><tr><td rowspan=1 colspan=3>Raw/Fresh (Unpreserved)</td><td rowspan=1 colspan=3>Cary-Blair (Preserved)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Total (%)</td></tr><tr><td rowspan=1 colspan=1>Unknown age</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/5$(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Infant(&lt;2 yrs)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>2/2(100.0%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Child(≥2 to &lt;12 yrs)</td><td rowspan=1 colspan=1>1/7(14.3%)</td><td rowspan=1 colspan=1>1/9(11.1%)</td><td rowspan=1 colspan=1>2/16(12.5%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>2/9(22.2%)</td></tr><tr><td rowspan=1 colspan=1>Adolescent(≥12 to &lt;18 yrs)</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>1/6(16.7%)</td><td rowspan=1 colspan=1>1/14(7.1%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Transitional Adolescent(≥18 to ≤21 yrs)</td><td rowspan=1 colspan=1>2/7(28.6%)</td><td rowspan=1 colspan=1>3/15(20.0%)</td><td rowspan=1 colspan=1>5/22(22.7%)</td><td rowspan=1 colspan=1>2/4(50.0%)</td><td rowspan=1 colspan=1>1/8(12.5%)</td><td rowspan=1 colspan=1>3/12(25.0%)</td></tr><tr><td rowspan=1 colspan=1>Adult(&gt;21 to 59 yrs)</td><td rowspan=1 colspan=1>38/273(13.9%)</td><td rowspan=1 colspan=1>40/297(13.5%)</td><td rowspan=1 colspan=1>78/571*(13.7%)</td><td rowspan=1 colspan=1>11/82(13.4%)</td><td rowspan=1 colspan=1>8/105(7.6%)</td><td rowspan=1 colspan=1>19/187(10.2%)</td></tr><tr><td rowspan=1 colspan=1>Sr. Adult(&gt; 60 yrs)</td><td rowspan=1 colspan=1>47/242(19.4%)</td><td rowspan=1 colspan=1>62/310(20.0%)</td><td rowspan=1 colspan=1>109/552(19.7%)</td><td rowspan=1 colspan=1>9/68(13.2%)</td><td rowspan=1 colspan=1>11/94(11.7%)</td><td rowspan=1 colspan=1>20/162(12.3%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>88/542(16.2%)</td><td rowspan=1 colspan=1>109/640(17.0%)</td><td rowspan=1 colspan=1>197/1,186(16.6%)</td><td rowspan=1 colspan=1>24/165(14.5%)</td><td rowspan=1 colspan=1>20/214(9.3%)</td><td rowspan=1 colspan=1>44/379(11.6%)</td></tr></table>

\*Prevalence based on $C .$ . difficile positives with the IMDx C. difficile for Abbott $m 2 0 0 0$ assay. §The gender of three individuals in this age group was not known. †The gender of one individual in this age group was not known.

# N. Instrument Name:

Abbott m2000 System running C. difficile (cDiff) Application Specification (App Spec) Parameter Definitions v1.15:

· Abbott m2000sp: sample preparation and reagent mixing (version 3.0.21.0) · Abbott $m 2 0 0 0 \mathrm { r t }$ : amplification reaction and detection (version 3.0.338.0).

# O. System Descriptions:

1. Modes of Operation:

The Abbott m2000 System is an instrument platform that automates steps to perform nucleic acid amplification assays from sample processing through amplification, detection, and data reduction. The Abbott $m 2 0 0 0$ System comprises the $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ instruments, which are operated with separate System Control Center (SCC) workstations. Each instrument contains an independent software application; one for the $m 2 0 0 0 s p$ and a second for the $m 2 0 0 0 r t$ . The $m 2 0 0 0 s p$ instrument is a floor standing, automated sample preparation system. The $m 2 0 0 0 r t$ instrument is a real-time PCR thermal cycler/reader instrument system. Abbott Molecular is the manufacturer of the $m 2 0 0 0$ System. The principal hardware components that comprise the $m 2 0 0 0 s p$ and $m 2 0 0 0 r t$ were developed by Original Equipment Manufacturer vendors, Tecan Schweiz AG, Mannedorf, Switzerland, and Applied Biosystems, Foster City, CA, respectively. Abbott Molecular developed the software that is uniquely for use with the $m 2 0 0 0$ System. The Abbott m2000 System software processes sample preparation and amplification/detection protocols based on pre-determined, assay-specific parameters that are contained in individual assay application specification (App Spec) files that are installed on the SCC. The Abbott $m 2 0 0 0 s p$ reads and processes bar coded primary sample tubes and processes up to 96 specimens, controls, and calibrators in batch mode. The m2000 System is capable of processing samples from various matrices, depending on the specific assay application. At the completion of the automated sample preparation protocol, the operator seals and manually transfers the PCR plate to the Abbott $m 2 0 0 0 r t$ for nucleic acid detection. Bar code and $m 2 0 0 0 s p$ data is transferred to the $m 2 0 0 0 r t$ electronically via removable media (i.e., a CD).

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

3. Specimen Identification:

Specimen identification occurs on the Abbott $m 2 0 0 0 s p$ instrument when it is loaded with specimen tubes.

4. Specimen Sampling and Handling:

Fresh/raw (i.e., unpreserved) human liquid or soft stool specimens collected and transported to the laboratory are used for testing. Cary-Blair (i.e, preserved) or frozen human liquid or soft stool samples collected and transported to the laboratory in a sterile container and sampled for testing. Specimen processing is automated on the Abbott $m 2 0 0 0 s p$ .

# 5. Calibration:

There is no calibration procedure associated with the IMDx C. difficile for Abbott $m 2 0 0 0$ assay per se. Optical calibration of the Abbott $m 2 0 0 0 r t$ instrument is required for the accurate measurement and discrimination of dye fluorescence during the IMDx $C$ . difficile for Abbott m2000 assay; the Calibration Procedures section in the Abbott $m 2 0 0 0 r t$ Operations Manual describes how to perform an optical calibration.

6. Quality Control:

N/A

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

1. The submitted information in this premarket notification is complete and supports a substantial equivalence decision.