# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K132508

# B. Purpose for Submission:

The CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel (K111507, K123905, and K130551) has been modified with the addition of a new assay within the panel for the determination of the genetic lineage of human influenza B virus as B/Victoria or B/Yamagata lineage. An additional positive control was also added to the panel for use with the new assay.

# C. Measurand:

Influenza virus nucleic acids target sequences. Influenza types, subtypes and genetic lineages detected and differentiated: Influenza A, Influenza A/H1, Influenza A/H3, Influenza A/H5 (Asian lineage), Influenza A/H1pdm09, Influenza B, Influenza B/Victoria, and Influenza B/Yamagata.

# D. Type of Test:

A panel of oligonucleotide primers and dual-labeled hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes to be used in rRT-PCR assays for the in vitro qualitative detection and differentiation of influenza virus type, subtype, and genetic lineage target sequences in respiratory specimens from human patients with signs or symptoms of respiratory infection and/or from viral culture using manual or automated nucleic acid isolation, and amplification/detection on the ABI 7500 Fast Dx Real-Time PCR instrument with Sequence Detection Software version 1.4.

# E. Applicant:

Centers for Disease Control and Prevention (CDC)

# F. Proprietary and Established Names:

CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel

# G. Regulatory Information:

1. Regulation section: 866.3332 Reagents for detection of specific novel influenza A viruses

2. Classification: Class II

3. Product code(s): OQW, NSU, NXD, OEP

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel is intended for use in real-time RT-PCR (rRT-PCR) assays on an Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR instrument in conjunction with clinical and epidemiological information:

For qualitative detection of influenza virus type A or B from viral RNA in upper respiratory tract clinical specimens (including nasopharyngeal swabs [NPS], nasal swabs [NS], throat swabs [TS], nasal aspirates [NA], nasal washes [NW] and dual nasopharyngeal/throat swabs [NPS/TS]), and lower respiratory tract specimens (including bronchoalveolar lavage [BAL], bronchial wash [BW], tracheal aspirate [TA], sputum, and lung tissue) from human patients with signs and symptoms of respiratory infection and/or from viral culture;   
For determination of the subtype of seasonal human influenza A viruses as seasonal A/H1, A/H3, and/or A/H1pdm09 from viral RNA in upper respiratory tract clinical specimens (including NPS, NS, TS, NA, NW, and NPS/TS) and lower respiratory tract specimens (including BAL, BW, TA, sputum and lung tissue) from human patients with signs and symptoms of respiratory infection and/or from viral culture; For the determination of the genetic lineage of human influenza B viruses as B/Victoria or B/Yamagata lineage from viral RNA in upper respiratory tract clinical specimens (including NPS, NS, TS, NA, NW, and NPS/TS) from human patients with signs and symptoms of respiratory infection and/or from viral culture;   
For the presumptive identification of virus in patients who may be infected with influenza A subtype A/H5 (Asian lineage) from viral RNA in human respiratory specimens and viral culture in conjunction with clinical and epidemiological risk factors;   
To provide epidemiological information for surveillance of circulating influenza viruses.

Performance characteristics for influenza were established during a season when seasonal influenza viruses A/H1 and A/H3 were the predominant influenza A viruses in circulation and during a season when the A/H1pdm09 influenza virus was the predominant influenza A virus in circulation. Performance characteristics may vary with other emerging influenza A viruses.

Testing with the influenza H5a and H5b primer and probe sets should not be performed unless the patient meets the most current U.S. Department of Health and Human Services (DHHS) clinical and epidemiological criteria for testing suspect A/H5 specimens. The definitive identification of influenza A/H5 (Asian lineage) either directly from patient specimens or from virus cultures requires additional laboratory testing, along with clinical and epidemiological assessment in consultation with national influenza surveillance experts.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions. Conversely, positive results do not rule out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted unless a BSL $3 +$ facility is available to receive and culture specimens.

All users, analysts, and any person reporting results from use of this device should be trained to perform and interpret the results from this procedure by a competent instructor prior to use. CDC Influenza Division will limit the distribution of this device to only those users who have successfully completed a training course provided by CDC instructors or designees.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR Instrument with Sequence Detection Software version 1.4.

# I. Device Description:

The CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel is used in rRT-PCR assays on the ABI 7500 Fast Dx Real-Time PCR Instrument. It consists of oligonucleotide primers, dual-labeled hydrolysis $( \mathrm { T a q M a n @ } )$ ) probes, and positive controls for the in vitro qualitative detection and characterization of the human influenza virus RNA in respiratory tract specimens from human patients presenting with signs and symptoms of respiratory infections. These reagents are used for detection of influenza A and B virus and characterization of influenza A viruses as seasonal A/H1, A/H3, A/H1pdm09, or A/H5 (Asian lineage), and influenza B viruses as B/Victoria or B/ Yamagata lineage.

Primers and probes for Influenza A, 2009 Influenza A (swine origin), and B viruses were selected from highly conserved regions of specific gene targets within the matrix protein (M), nucleoprotein (NP), and non-structural protein (NS), respectively. Primers and probes for detection and differentiation of seasonal influenza A/H1, A/H3, A/H1pdm09, and A/H5 viruses are targeted for conserved regions of their respective hemagglutinin (HA) genes. Primers and probes for detection and differentiation of influenza B/Victoria and B/Yamagata, are also targeted for conserved regions of their respective hemagglutinin (HA) genes.

The components of the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel include the following:

# Primer and probe sets

· InfA detects all influenza A viruses, but does not detect influenza B viruses   
· InfB detects all influenza B viruses, but does not detect influenza A viruses   
· H1 specifically detects influenza A, seasonal H1 subtype   
· H3 specifically detects influenza A, seasonal H3 subtype   
· H5a and H5b specifically detects influenza A, H5 Asian lineage   
· pdm InfA detects classical swine influenza viruses, triple reassortant swine influenza viruses, and A/H1pdm09 influenza virus   
· pdm H1 specifically detects the A/H1pdm09 influenza virus   
· VIC specifically detects influenza B/Victoria lineage viruses   
· YAM specifically detects influenza B/Yamagata lineage viruses   
· RNase P (RP) detects human RNase P and is used with human clinical specimens to indicate that adequate isolation of nucleic acid resulted from the extraction of the clinica specimen.

# Positive Controls

# · Pooled Influenza Positive Control (PIPC)

For use as a positive control with the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel procedure to ensure the detection of seasonal influenza virus A/H1, A/H3, A/H1pdm09 and influenza B. The PIPC contains noninfectious positive control materials supplied as a liquid, $5 0 0 \mu \mathrm { l }$ per vial, suspended in $0 . 0 1 { \bf M }$ phosphate buffer saline (PBS) at pH 7.2–7.4. PIPC consists of four different beta-propiolactone treated influenza viruses (influenza A/H1, A/H3, A/H1pdm09, and influenza B) suspended in cultured human cells (A549). PIPC will yield a positive result with the following primer and probe sets: InfA, InfB, H1, H3, pdm InfA, pdm H1, and RP.

# · Influenza Virus A/H5N1 Positive Control (H5VC)

For use as a positive control with the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel procedure to ensure proper detection of Influenza A/H5 virus (Asian lineage). Noninfectious (beta-propiolactone treated) positive control material supplied as a liquid, $5 0 0 \mu \mathrm { l }$ per vial, suspended in 0.01 M PBS at $\mathrm { p H } 7 . 2 – 7 . 4$ . The H5VC control consists of a reassortant human vaccine candidate virus (A/Vietnam/1203/04 x PR/8/34) that was generated by reverse genetics in a suspension of cultured human cells (A549). The H5VC will yield a positive result with the following primer and probe sets: InfA, H5a, H5b, and RP.

# · Influenza B Positive Control (IBPC)

For use as a positive control with the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel procedure to ensure the detection and differentiation of seasonal influenza B viruses of both the B/Yamagata and B/Victoria lineages. The IBPC contains noninfectious positive control materials supplied as a liquid, $5 0 0 \mu \mathrm { l }$ per vial, suspended in $0 . 0 1 { \bf M }$ phosphate buffer saline (PBS) at $\mathrm { p H } 7 . 2 – 7 . 4$ . IBPC consists of two different betapropiolactone treated influenza viruses, representing influenza B/Yamagata and B/Victoria lineages, suspended in cultured human cells (A549). IBPC will yield a positive result with the following primer and probe sets: InfB, VIC or YAM, and RP.

# · Human Specimen Control (HSC)

For use as a RNA extraction procedural control with the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel to demonstrate successful recovery of RNA, as well as extraction reagent integrity. Purified RNA from the HSC material should yield a positive result with the RP primer and probe set and negative results with all influenza specific markers. The HSC consists of noninfectious (beta-propiolactone treated) cultured human cell material (A549) supplied as a liquid suspended in 0.01 M PBS at pH 7.2–7.4.

Materials Required But Not Provided (Ancillary Reagents)

# 1. rRT-PCR Mastermix Options

· Invitrogen SuperScript™ III Platinum $^ \mathrm { \textregistered }$ One-Step Quantitative RT-PCR System   
(without Rox)   
· Invitrogen SuperScript™ III Platinum $^ \mathrm { \textregistered }$ One-Step Quantitative RT-PCR System (with   
Rox)   
· Quanta BioSciences qScript™ One-Step qRT- PCR Kit, Low Rox

Specific lots of enzyme systems will be qualified for use with the CDC Human Influenza Virus Real- Time RT-PCR Diagnostic Panel by the CDC’s reagent qualification program. A supplemental list of qualified enzyme system lots will be made available to users.

# 2. RNA Extraction Options

· Roche MagNA Pure LC 2.0, Total Nucleic Acid Kit -External Lysis   
· Roche MagNA Pure Compact, Nucleic Acid Isolation Kit I – External   
· Roche MagNA Pure Compact, RNA Isolation Kit – RNA_Tissue_V3_1 Protocol   
· Qiagen QIAamp $^ \mathrm { \textregistered }$ , DSP Viral RNA Mini Kit   
· QIAGEN QIAcube, DSP Viral RNA Mini Kit   
· bioMérieux NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$

# 3. Equipment and Consumables Required But Not Provided

· Plasticware and consumables   
· Rnase/Dnase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes   
· $100 \%$ Ethanol (EtOH)   
· Disposable gloves   
· Molecular Grade Water (RNase/DNase Free)   
· $- 7 0 \mathrm { { } ^ { \circ } C }$ and $- 2 0 \mathrm { { ^ \circ C } }$ Freezer(s)   
· $4 \mathrm { { ^ \circ C } }$ Refrigerator   
$\bullet$ 96-well cold block   
· Micropipettors $( 1 - 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { - } 1 0 0 0 \mu \mathrm { L } )$   
· Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument (Applied Biosystems, Foster City, CA)   
· Applied Biosystems 7500 Fast Sequence Detection Consumables (Applied Biosystems, Foster City, CA).   
· ABI MicroAmp™ Fast 8-tube strip $0 . 1 \mathrm { m l }$ , cat #4358293 (required), or   
· ABI MicroAmp™ Optical 8-cap strip, cat #4323032 (required)   
· ABI MicroAmp™ Fast Optical 96-Well Reaction Plate with Barcode, $0 . 1 ~ \mathrm { m l }$ ,   
part#4346906, 4346907, or part #4366932 (alternate to 8-strip tubes)   
· Benchtop Microcentrifuge

# J. Substantial Equivalence Information:

1. Predicate device name(s): CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel

2. Predicate 510(k) number(s): K130551

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CDC Human Influenza VirusReal-Time RT-PCR DiagnosticPanel with Influenza BLineage Genotyping Assay(K132508)</td><td colspan="1" rowspan="1">CDC Human Influenza VirusReal-Time RT-PCR DiagnosticPanel (K130551)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Same as the predicate, except forthe following additional</td><td colspan="1" rowspan="1">The CDC Human Influenza VirusReal-Time RT-PCR Diagnostic</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="2">Similarities and Differences</td><td>Predicate</td></tr><tr><td>Item</td><td>Device indication: •For the determination of the genetic lineage of human</td><td>Panel is intended for use in real- time RT-PCR (rRT-PCR) assays on an Applied Biosystems (ABI)</td></tr><tr><td rowspan="2"></td><td>influenza B viruses as B/Victoria or B/Yamagata lineage from viral RNA in upper respiratory tract clinical specimens (including NPS, NS, TS, NA, NW, and NPS/TS) from human patients with signs and symptoms of respiratory infection and/or from viral culture;</td><td>7500 Fast Dx Real-Time PCR instrument in conjunction with clinical and epidemiological information: For qualitative detection of influenza virus type A or B from viral RNA in upper respiratory tract clinical specimens (including nasopharyngeal swabs [NPS], nasal swabs [NS], throat swabs [TS], nasal aspirates [NA], nasal washes [NW] and dual nasopharyngeal/throat swabs [NPS/TS]), and lower respiratory tract specimens (including bronchoalveolar lavage [BAL], bronchial wash [BW], tracheal aspirate [TA], sputum, and lung tissue) from human patients with signs and symptoms of respiratory infection and/or from viral culture; For determination of the subtype of seasonal human influenza A viruses as seasonal A/H1, A/H3, and/or A/H1pdm09 from viral RNA in upper respiratory tract clinical specimens (including NPS, NS, TS, NA, NW, and NPS/TS) and lower respiratory tract specimens (including BAL, BW, TA, sputum and lung tissue) from human patients with signs and symptoms of respiratory infection and/or</td></tr><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">identification of virus inpatients who may be infectedwith influenza A subtype A/H5(Asian lineage) from viralRNA in human respiratoryspecimens and viral culture inconjunction with clinical andepidemiological risk factors;To provide epidemiologicalinformation for surveillance ofcirculating influenza viruses.</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected andDifferentiated</td><td colspan="1" rowspan="1">Same as the predicate, except forthe following additional viruslineages:Influenza B/Yamagata andB/Victoria lineages</td><td colspan="1" rowspan="1">Universal influenza A viruses(animal and human), Swine-origininfluenza A viruses, Influenza Bviruses, and Influenza Asubtypes: seasonal A/H1, A/H3,A/H1pdm09, and A/H5</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Same as the predicate, except forinfluenza B genetic lineagedetermination; only upperrespiratory tract clinicalspecimens are claimed.</td><td colspan="1" rowspan="1">Nasopharyngeal swabs, nasalswabs, throat swabs, nasalaspirates, nasal washes and dualnasopharyngeal/throat swabs,bronchoalveolar lavages,bronchial aspirates, bronchialwashes, tracheal aspirates,sputum, and lung tissue and virusculture.</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtractionRequired</td><td colspan="1" rowspan="1">Same as the predicate</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">RNA ExtractionOptions</td><td colspan="1" rowspan="1">Same as the predicate</td><td colspan="1" rowspan="1">Roche MagNA Pure LC 2.0,Total Nucleic Acid Kit -ExternalLysis; orRoche MagNA Pure Compact,Nucleic Acid Isolation Kit I External; orRoche MagNA Pure Compact,RNA Isolation Kit -RNA_Tissue_V3_1 Protocol; orQiagen QIAamp®, DSP ViralRNA Mini Kit; orQIAGEN QIAcube, DSP ViralRNA Mini Kit; or</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">bioMérieux NucliSENS®easyMAG®</td></tr><tr><td colspan="1" rowspan="1">Enzyme MasterMix Options</td><td colspan="1" rowspan="1">Same as the predicate</td><td colspan="1" rowspan="1">Invitrogen SuperScript™ IIIPlatinum® One-StepQuantitative RT-PCR Kit (withor without ROX); orQuanta BioSciences qScript™MOne-Step qRT-PCR Kit, LowROX</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document:   
Reagents for Detection of Specific Novel Influenza A Viruses, March 22, 2006   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments   
/ucm078583.htm   
Guidance for Industry and FDA Staff, In Vitro Diagnostic Devices to Detect Influenza A   
Viruses: Labeling and Regulatory Path, May 1, 2007   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments   
/ucm078538.htm   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document:   
Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex   
Assays October 9, 2009   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments   
/ucm180307.htm   
Guidance for Industry and FDA Staff - Establishing Performance Characteristics of In   
Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza   
Viruses, July 15, 2011   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments   
/ucm079171.htm

# L. Test Principle:

The CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel is based on technology which is used in many molecular diagnostic assays. rRT-PCR assays are one-tube assays that first reverse-transcribe specific regions of RNA into cDNA copies. The cDNA then serves as a template for a polymerase chain reaction that utilizes thermocyclic heating and cooling of the reaction to logarithmically amplify a specific region of DNA. The probe anneals to a specific internal target sequence located between the target loci of the forward and reverse primers. During the extension phase of the PCR cycle, the $5 "$ exonuclease activity of Taq polymerase degrades any probe molecules hybridized to amplified target sequence, causing the reporter dye to separate from the quencher dye, and generating a fluorescent signal. With each cycle, additional reporter dye molecules are cleaved from their respective probes, increasing the fluorescence intensity. Fluorescence intensity is monitored at each PCR cycle. Amplification of each target is reflected by logarithmic increase in fluorescence over time in comparison to the background signal.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Please refer to previously FDA-cleared 510(k) Premarket Notifications, K080570 and K101564 for precision/reproducibility performance.

An additional internal precision study was performed to assess the precision of the new VIC and YAM primer and probe sets. Each primer and probe set was tested with one influenza B strain for which it was designed to detect and differentiate (i.e., Influenza B/Nevada/03/2011, Victoria lineage and Influenza B/Wisconsin/01/2010, Yamagata lineage) using one extraction method (i.e., Roche MagNA Pure Compact RNA Kit) and one enzyme kit (i.e., Invitrogen SuperScript III Platinum). Three panel members of each influenza B lineage (i.e., moderate positive, low positive and high negative) were tested by two operators who individually performed two separate runs per testing day. Each panel member was tested in three replicates per run. Testing was performed over 12 days (not necessarily consecutive) to generate a total of 96 data points per panel member (i.e., 2 replicates/run x 2 runs/day/operator $_ { \textrm { X 2 } }$ operators x $1 2 \mathrm { d a y s } = 9 6 \mathrm { \Omega }$ .

The precision study summary results are presented in the table below:

Precision Study Results   

<table><tr><td rowspan=2 colspan=1>Panel Sample</td><td rowspan=2 colspan=1>Primer/Probe Set</td><td rowspan=1 colspan=3>Operator 1</td><td rowspan=1 colspan=3>Operator 2</td><td rowspan=2 colspan=1>AgreementTotal</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>B/VictoriaModerate</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>23.86</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>23.96</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>26.36</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>25.98</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=2 colspan=1>B/VictoriaLow**</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>33.16*</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>45/48</td><td rowspan=1 colspan=1>33.38*</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>91/96</td><td rowspan=1 colspan=1>94.8 (88.4-97.8)</td></tr><tr><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>31/48</td><td rowspan=1 colspan=1>36.41*</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>38/48</td><td rowspan=1 colspan=1>36.00*</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>69/96</td><td rowspan=1 colspan=1>71.9 (62.2 - 79.9)</td></tr><tr><td rowspan=2 colspan=1>B/VictoriaHigh Negative</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>44/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>91/96</td><td rowspan=1 colspan=1>94.8 (88.4-97.8)</td></tr><tr><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=2 colspan=1>B/YamagataModerate</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>26.01</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>25.70</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>YAM</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>26.70</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>26.14</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=2 colspan=1>B/YamagataLow</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.46</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>94/96</td><td rowspan=1 colspan=1>97.9 (92.7 - 99.4)</td></tr><tr><td rowspan=1 colspan=1>YAM</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>32.72</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.21</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>94/96</td><td rowspan=1 colspan=1>97.9 (92.7 - 99.4)</td></tr><tr><td rowspan=2 colspan=1>B/YamagataHigh Negative</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0 (96.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>YAM</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>99.0 (94.3 - 99.8)</td></tr></table>

$\mathrm { N A } = \mathrm { N o t }$ Applicable \* Average Ct calculations were based on the positive sample Cts only \*\*Using information from the LoD range finding study, this low positive precision study sample

was prepared using a concentration lower than the reported LoD in order to target a concentration as close to the $\mathrm { C } _ { 9 5 }$ as possible. However, since the assay is very close to its analytical endpoint at this concentration, several replicates between both operators produced no measureable fluorescence above background or measurements above the cutoff.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

There are no changes to the internal positive control, the human RNase P; Human Specimen Control (HSC); Influenza virus A/H5N1 Positive Control (H5VC), Pooled Influenza Positive Control (PIPC), and No Template Control (NTC) for the modified device. Please refer to previously FDA-cleared 510(k) Premarket Notifications K080570, K101564, K111507, and K130551 for further information.

An additional positive control, the Influenza B Positive Control (IBPC), was included in the modified device. The IBPC is for use as a positive control with the CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel procedure to ensure the detection and differentiation of seasonal influenza B viruses of both the B/Yamagata and B/Victoria lineages.

# d. Detection limit:

Analytical sensitivity of the Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets and the previously FDA-cleared InfB primer and probe set) was demonstrated by determining the Limit of Detection (LoD) using both the Quanta qScript™ and the Invitrogen SuperScript™ enzyme kits.

Characterized viruses of known $50 \%$ infectious dose titers $\mathrm { ( E I D 5 0 / m L ) }$ ) were extracted, and the RNA was serially diluted and tested ( $\mathrm { \Delta } \mathrm { n } { = } 3$ to 5 replicates) in order to determine an apparent endpoint range. The LoD for each primer and probe set was confirmed by testing extraction replicates $( \mathrm { n } { = } 2 0 )$ ) of the highest virus dilution where $2 9 5 \%$ of all replicates tested positive. Virus dilutions were prepared in virus transport medium containing human A549 cells to emulate clinical specimen matrix. The lowest concentration where the InfB and VIC or InfB and YAM primer and probe sets both met the LoD criteria was reported as the LoD.

The LoD study results are presented in the table below:

LoD Study Results   

<table><tr><td rowspan=2 colspan=1>Influenza B VirusLineage</td><td rowspan=2 colspan=1>Virus Designation</td><td rowspan=1 colspan=2>LoD (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Invitrogen SuperScriptT</td><td rowspan=1 colspan=1>Quanta qScriptT</td></tr><tr><td rowspan=1 colspan=1>B/Victoria</td><td rowspan=1 colspan=1>B/Navada/03/2011</td><td rowspan=1 colspan=1>102.</td><td rowspan=1 colspan=1>101.4</td></tr><tr><td rowspan=1 colspan=1>B/Yamagata</td><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>103.5</td><td rowspan=1 colspan=1>102.8</td></tr></table>

# e. Analytical reactivity:

Analytical reactivity (inclusivity) testing was conducted to demonstrate the capability of the oligonucleotide primers and probes in the Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets and the previously FDAcleared InfB primer and probe set) to detect multiple strains of each influenza B virus lineage at low concentrations. Ten influenza B/Victoria lineage and ten influenza B/Yamagata lineage viruses were grown to high titer and harvested. Each virus was serially diluted to near the assay LoD in virus transport medium containing human A549 cells to emulate clinical specimen matrix. The diluted influenza B/Victoria lineage and B/Yamagata lineage viruses were extracted and tested in triplicate with the corresponding primer and probe set to demonstrate analytical reactivity. Inclusivity of the Influenza B Lineage Genotyping Assay was evaluated with both enzyme systems (i.e. Invitrogen SuperScript™ and Quanta qScript™).

The analytical reactivity study results are presented in the two tables below:

Analytical Reactivity Study Results – Influenza B/Victoria Lineage Assay   

<table><tr><td rowspan=2 colspan=1>Virus Designation</td><td rowspan=2 colspan=1>Lineage</td><td rowspan=2 colspan=1>EID50/mL</td><td rowspan=1 colspan=2>InvitrogenSuperScriptM</td><td rowspan=1 colspan=2>Quanta qScriptM</td></tr><tr><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td></tr><tr><td rowspan=1 colspan=1>B/Bolivia/1526/2010</td><td rowspan=10 colspan=1>B/Vic</td><td rowspan=1 colspan=1>10 1.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/33/2008</td><td rowspan=1 colspan=1>10 2.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>101.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Fujian Gulou/1272/2008</td><td rowspan=1 colspan=1>102.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Georgia/07/2010</td><td rowspan=1 colspan=1>103.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/230/2009</td><td rowspan=1 colspan=1>101.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/HongKong/259/2010</td><td rowspan=1 colspan=1>103.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>101.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>101.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Texas/26/2008</td><td rowspan=1 colspan=1>103.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Analytical Reactivity Study Results – Influenza B/Yamagata Lineage Assay   

<table><tr><td colspan="2" rowspan="2">Virus Designation</td><td colspan="2" rowspan="2">Lineage</td><td colspan="1" rowspan="2">EID50/mL</td><td colspan="2" rowspan="1">InvitrogenSuperScriptM</td><td colspan="2" rowspan="1">Quanta qScriptTM</td></tr><tr><td colspan="1" rowspan="1">InfB</td><td colspan="1" rowspan="1">YAM</td><td colspan="1" rowspan="1">InfB</td><td colspan="1" rowspan="1">YAM</td></tr><tr><td colspan="2" rowspan="1">B/Wisconsin/1/2010</td><td colspan="2" rowspan="4">B/Yam</td><td colspan="1" rowspan="1">10 2.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="2" rowspan="1">B/Bangladesh/5972/2007</td><td colspan="1" rowspan="1">101.1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="2" rowspan="1">B/Bangladesh/7110/2007</td><td colspan="1" rowspan="1">101.6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="2" rowspan="1">B/Chongqingyongchuan/18/200</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">101.3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="2" rowspan="1">B/Finland/39/2010</td><td></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">101.9</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="2" rowspan="1">B/Brisbane/3/2007</td><td></td><td colspan="1" rowspan="1">101.4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B/Hubei-Wujiagang/158/2009</td><td colspan="1" rowspan="4">−</td><td colspan="1" rowspan="1">101.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B/Pennsylvania/7/2007</td><td colspan="1" rowspan="1">10 2.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B/Santiago/4364/2007</td><td colspan="1" rowspan="1">1022.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B/Texas/06/2011</td><td colspan="1" rowspan="1">101.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# f. Analytical specificity:

An exclusivity study was performed to demonstrate the specificity of each primer and probe set of the Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets and the previously FDA-cleared InfB primer and probe set) when tested with influenza B viruses of the other lineage, with influenza A viruses, and with common non-influenza human respiratory viruses, respiratory bacteria, and commensal organisms of the human respiratory tract. All organisms used in the study were propagated, titered, and characterized to confirm identity prior to testing. Nucleic acids were purified from ten influenza B/Victoria lineage and ten influenza B/Yamagata lineage viruses, eight influenza A viruses of various subtypes that circulate in humans or from animal origin that infect humans, and 35 non-influenza organisms (16 viruses, 18 bacteria, and 1 yeast) representing common respiratory pathogens or flora commonly present in specimens collected from the human nasopharynx region. High titer preparations of bacteria and yeast, generally greater than or equal to $\mathrm { \bar { 1 0 ^ { 6 } C F U / m L } }$ were tested with the Influenza B Lineage Genotyping Assay. Similarly, influenza and non-influenza respiratory virus preparations at concentrations greater than $1 0 ^ { 6 } \mathrm { T C I D 5 0 / m L }$ were tested (except in cases where production of high titer virus stock was not possible, e.g. parainfluenza virus type 2). The Influenza B Lineage Genotyping Assay was evaluated with both enzyme kits (i.e. Invitrogen SuperScript™ and Quanta qScript™).

No cross-reactivity was observed. The analytical specificity study results are presented in the tables below:

Analytical Specificity Study Results – Influenza B/Yamagata Lineage Assay   

<table><tr><td rowspan=2 colspan=1>Virus Designation</td><td rowspan=2 colspan=1>Lineage</td><td rowspan=2 colspan=1>EID50/mL</td><td rowspan=1 colspan=2>InvitrogenSuperScript</td><td rowspan=1 colspan=2>Quanta qScriptTM</td></tr><tr><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>YAM</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>YAM</td></tr><tr><td rowspan=1 colspan=1>B/Bolivia/1526/2010</td><td rowspan=10 colspan=1>B/Vic</td><td rowspan=1 colspan=1>107.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/33/2008</td><td rowspan=1 colspan=1>10 7.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>107.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Fujian Gulou/1272/2008</td><td rowspan=1 colspan=1>107.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Georgia/07/2010</td><td rowspan=1 colspan=1>10 7.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/230/2009</td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/259/2010</td><td rowspan=1 colspan=1>10 8.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>107.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Texas/26/2008</td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr></table>

Analytical Specificity Study Results – Influenza B/Victoria Lineage Assay   

<table><tr><td rowspan=2 colspan=1>Virus Designation</td><td rowspan=2 colspan=3>Lineage</td><td rowspan=2 colspan=1>EID50/mL</td><td rowspan=1 colspan=2>InvitrogenSuperScriptM</td><td rowspan=1 colspan=2>Quanta qScript</td></tr><tr><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/2010</td><td></td><td rowspan=8 colspan=2>B/Yam</td><td rowspan=1 colspan=1>10 8.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/5972/2007</td><td></td><td rowspan=1 colspan=1>106.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/7110/2007</td><td></td><td rowspan=1 colspan=1>104.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Chongqingyongchuan/18/200</td><td></td><td rowspan=1 colspan=1>107.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Finland/39/2010</td><td></td><td rowspan=1 colspan=1>106.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/3/2007</td><td></td><td rowspan=1 colspan=1>107.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Hubei-Wujiagang/158/2009</td><td></td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Pennsylvania/7/2007</td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>B/Santiago/4364/2007</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>106.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>10 6.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr></table>

Analytical Specificity Study Results - Influenza A Viruses Infecting Humans   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Virus Designation</td><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=1>EID50/mLorTCID50/mL</td><td rowspan=1 colspan=3>InvitrogenSuperScriptM</td><td rowspan=1 colspan=3>Quanta qScript ™M</td></tr><tr><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>YAM</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>YAM</td></tr><tr><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>108.4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>(H1N1)pdm09</td><td rowspan=1 colspan=1>10 8.4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>108.2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Swine</td><td rowspan=1 colspan=1>A/Minnesota/19/2011</td><td rowspan=1 colspan=1>H1N2v</td><td rowspan=1 colspan=1>107.1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Swine</td><td rowspan=1 colspan=1>A/Indiana/10/2011</td><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>1010.2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>A/chicken/Vietnam/NCVD-016/2008</td><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>10 9.1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>A/Egypt/NO3072/2010</td><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>10 9.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>A/Bangladesh/0994/2011</td><td rowspan=1 colspan=1>H9N2</td><td rowspan=1 colspan=1>1010.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

Analytical Specificity Study Results - Respiratory Pathogens and Flora   

<table><tr><td colspan="3" rowspan="1">Organism Tested</td><td colspan="3" rowspan="1">InvitrogenSuperScriptTM</td><td colspan="3" rowspan="1">Quanta qScript</td></tr><tr><td colspan="1" rowspan="1">Bacteria and Yeast</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">Inf B</td><td colspan="1" rowspan="1">VIC</td><td colspan="1" rowspan="1">YAM</td><td colspan="1" rowspan="1">InfB</td><td colspan="1" rowspan="1">VIC</td><td colspan="1" rowspan="1">YAM</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">10 8.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">2001-21-196</td><td colspan="1" rowspan="1">10 8.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">40 IFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1010</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">K12</td><td colspan="1" rowspan="1">109.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">M15709</td><td colspan="1" rowspan="1">106.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">108.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">107.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">M15757</td><td colspan="1" rowspan="1">10 9.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis2</td><td colspan="1" rowspan="1">H37Rv</td><td colspan="1" rowspan="1">95 ng/μL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">MI-29</td><td colspan="1" rowspan="1">107.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 8.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M2578</td><td colspan="1" rowspan="1">107.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1010.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1010.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1010.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">249-06(Thailand)</td><td colspan="1" rowspan="1">10 6.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">7790-06</td><td colspan="1" rowspan="1">10 7.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">SS1672</td><td colspan="1" rowspan="1">10 8.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">Inf B</td><td colspan="1" rowspan="1">VIC</td><td colspan="1" rowspan="1">YAM</td><td colspan="1" rowspan="1">InfB</td><td colspan="1" rowspan="1">VIC</td><td colspan="1" rowspan="1">YAM</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Echo 6</td><td colspan="1" rowspan="1">10 6.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus, type</td><td colspan="1" rowspan="1">Ad.71</td><td colspan="1" rowspan="1">10 9.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus, type 7a</td><td colspan="1" rowspan="1">S-1058</td><td colspan="1" rowspan="1">10 7.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus virus2</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">50.4 ng /μL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus virus2</td><td colspan="1" rowspan="1">299E</td><td colspan="1" rowspan="1">31.6 ng /μL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus A</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">10 3.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza 1 virus2</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3.0 ng/μL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza 2 virus</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">10 3.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza 3 virus</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">10 7.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial virus</td><td colspan="1" rowspan="1">CH93-18b</td><td colspan="1" rowspan="1">10 6.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus</td><td colspan="1" rowspan="1">KOS</td><td colspan="1" rowspan="1">10 8.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Varicella-zoster Virus</td><td colspan="1" rowspan="1">AV92-3</td><td colspan="1" rowspan="1">10 4.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus2</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">1.7 ng/μL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">105.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">Enders</td><td colspan="1" rowspan="1">107.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">AD-169</td><td colspan="1" rowspan="1">10 6.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

1 Organism quantified by Infectious Forming Units (IFU) 2 Organism quantified by spectrophotometry $( \mathrm { n g / \mu L } )$

# g. Assay cut-off:

The assay cut-offs are unchanged. When all controls exhibit the expected performance, a specimen is considered positive for influenza if the influenza marker (InfA, InfB, H1, H3, pdm InfA, pdm H1, VIC, and YAM) cycle threshold $\mathrm { ( C t ) }$ growth curve crosses the threshold line within 38.00 cycles $\left( < 3 8 . 0 0 \mathrm { C t } \right)$ . Please refer to previously FDA-cleared 510(k) Premarket Notifications, K080570, K101564, K111507, and K130551 for more details.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Performance of the new Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets and the previously FDA-cleared InfB primer and probe set) was evaluated against the comparator method of RT-PCR followed by sequencing.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Study:

A prospective clinical study was conducted during the 2011-2012 influenza season to evaluate the performance of the new CDC Human Influenza rRT-PCR Diagnostic Panel- Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets). Residual material from a total of 1,002 respiratory specimens from patients who were symptomatic for influenza-like illness (ILI) was collected and tested at six clinical sites. The study population and specimen types are summarized by age range and specimen type in the table below:

Prospective Clinical Study Population and Specimen Type Summary   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Number of Clinical Specimensby Type</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Totals</td></tr><tr><td rowspan=1 colspan=1>Age Range</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NW</td><td rowspan=1 colspan=1>NS</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>LR</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>0-16</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>337</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>434</td></tr><tr><td rowspan=1 colspan=1>17-54</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>328</td></tr><tr><td rowspan=1 colspan=1>≥55</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Totals</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1002</td></tr></table>

NA=nasal aspirate, $\mathrm { N W } =$ nasal wash, ${ \bf N S } { = }$ nasal swab, ${ \mathrm { N P S } } { = }$ nasopharyngeal swab, NPS/TS ${ \bf \Phi } _ { 1 } = { \bf \Phi } _ { }$ dual nasopharyngeal and throat swab, $\mathrm { T S } { = } 1$ throat swab, LR $\varXi$ lower respiratory specimens, including bronchoalveolar lavage, bronchial wash, tracheal aspirate, sputum, or lung tissue.

The performance of each Influenza B Lineage Genotyping Assay (i.e., the new VIC and YAM primer and probe sets) was assessed at the clinical sites using both the Invitrogen Superscript™ and Quanta qScript™ enzyme kits against a comparator method of RT-PCR followed by bidirectional nucleic acid sequence analysis of a region of the influenza B hemagglutinin (HA) gene. Specimens collected in the study were tested with the InfB and RP primer and probe sets from the FDA-cleared CDC Human Influenza Real-Time RT-PCR Diagnostic Panel to establish the presence of influenza B viral RNA in upper respiratory specimens and confirm adequate specimen collection. The comparator method was then performed on each specimen containing influenza B viral RNA as determined by the FDA-cleared CDC Human Influenza Virus Real-Time RT-PCR Diagnostic Panel.

A brief description of the comparator method is provided below:

A 221-bp fragment containing the selected variable region of the HA gene was amplified by RT-PCR from RNA isolated from each clinical specimen. Each strand of the DNA amplicon was sequenced to obtain 4-fold coverage of an 82-bp region with Phred20 quality sequence data ( $9 9 \%$ accuracy). Sequence data of each sample were aligned to B/Victoria and B/Yamagata reference virus sequences of the HA gene and the percent homology calculated. A homology exceeding $9 5 \%$ to one lineage reference sequence while showing less than $9 5 \%$ to the other lineage reference sequence determines the lineage identity of the influenza B virus.

The reference virus sequences that were used were submitted by the WHO Collaborating Center for Influenza at CDC. Each reference virus was published with a serological identification to a vaccine reference virus and consequently identified as either Victoria or Yamagata lineage. Using the 82-bp region, a percent homology between contemporary B/Yamagata lineage viruses and B/Victoria lineage reference viruses does not exceed approximately $9 3 \%$ . The criteria of exceeding $9 5 \%$ homology to one lineage while showing less than $9 5 \%$ homology to the other lineage sets a significant threshold that is valid for available reference sequences of each lineage (i.e. no cases of a virus showing greater than $9 5 \%$ homology to both lineages is seen).

The following enrolled clinical specimens were excluded from the clinical performance data analysis due to deviations from the clinical study protocol: four specimens of unknown specimen type; five lower respiratory specimens; one specimen of inconclusive results; and 131 specimens due to technician error (excluded from Quanta qScript™ runs only). After exclusion of these specimens, there were a total of 992 specimens tested using the Invitrogen SuperScriptTM enzyme kits and 861 specimens tested using the Quanta qScript™ enzyme kits were included in the performance data analysis.

The prospective study results of the Influenza B Lineage Genotyping Assay used with the Invitrogen SuperScript™ enzyme kits are summarized in the tables below:

VIC Assay using Invitrogen SuperScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives1</td><td rowspan=1 colspan=1>% Positive Agreement(95% CI)</td><td rowspan=1 colspan=1># of Negatives1</td><td rowspan=1 colspan=1>% Negative Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0 (34.2 100.0)</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100.0 ( 90.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100.0 (92.1−100.0)</td><td rowspan=1 colspan=1>862/863</td><td rowspan=1 colspan=1>99.9 (99.3 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100.0 (75.8- 100.0)</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0 ( 86.7  100.0)</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0 ( 67.6 - 100.0)</td></tr></table>

1Proportion of true positives or true negatives correctly identified against the comparator method 2Not applicable; no positive samples of this specimen type were obtained

YAM Assay using Invitrogen SuperScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives</td><td rowspan=1 colspan=1>% Positive Agreement(95% CI)</td><td rowspan=1 colspan=1># of Negatives1</td><td rowspan=1 colspan=1>% Negative Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>100.0 (91.0100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0 (83.1 - 100.0)</td><td rowspan=1 colspan=1>889/889</td><td rowspan=1 colspan=1>100.0 (99.6 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0 (34.2 -100.0)</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100.0 ( 90.1 -100.0)</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA{2</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0 ( 67.6 - 100.0)</td></tr></table>

1Proportion of true positives or true negatives correctly identified against the comparator method 2Not applicable; no positive samples of this specimen type were obtained

The prospective study results of the Influenza B Lineage Genotyping Assay used with the Quanta qScript™ enzyme kits are summarized in the tables below:

VIC Assay using Quanta qScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives1</td><td rowspan=1 colspan=1>% Positive Agreement(95% CI)</td><td rowspan=1 colspan=1># of Negatives1</td><td rowspan=1 colspan=1>% Negative Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0 (34.2100.0)</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100.0 (85.13100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100.0 (92.0100.0)</td><td rowspan=1 colspan=1>749/752</td><td rowspan=1 colspan=1>99.6 (98.8−-99.9)</td></tr><tr><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7 (64.6 -98.5)</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100.0 ( 84.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0 ( 67.6 - 100.0)</td></tr></table>

1Proportion of true positives or true negatives correctly identified against the comparator method 2Not applicable; no positive samples of this specimen type were obtained

YAM Assay using Quanta qScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives</td><td rowspan=1 colspan=1>% Positive Agreement(95% CI)</td><td rowspan=1 colspan=1># of Negatives</td><td rowspan=1 colspan=1>% Negative Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA^{2</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0 (86.2- 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0 (83.2 --100.0)</td><td rowspan=1 colspan=1>775/777</td><td rowspan=1 colspan=1>99.7 (99.1- 99.9)</td></tr><tr><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0 (34.2 -100.0)</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100.0(89.0- 100.0)</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0 (67.6 - 100.0)</td></tr></table>

1Proportion of true positives or true negatives correctly identified against the comparator method 2Not applicable; no positive samples of this specimen type were obtained

Please also refer to previously FDA-cleared, 510(k) Premarket Notifications, K080570, K101564, K111507, and K130551, for details on other prospective clinical studies.

# b. Retrospective Study:

The prevalence of influenza B viruses within the prospective clinical study population was approximately $8 \%$ with the B/Victoria lineage outnumbering B/Yamagata lineage by a factor of 3 to 1. To augment the number of B/Yamagata lineage viruses in the prospective clinical study, additional clinical specimens were selected from routine influenza surveillance samples collected between September 2012 and January 2013 that had been initially screened for influenza A and B with the FDA-cleared CDC Human Influenza Real-Time RT-PCR Diagnostic Panel. Fifty-one (51) specimens identified as influenza B positive were selected for the retrospective study and tested with the new VIC and YAM primer and probe sets. The same comparator method as described in the Prospective Study section above was carried out in the retrospective study to determine influenza B lineage for each specimen.

The retrospective study results of the Influenza B Lineage Genotyping Assay used with the Invitrogen SuperScript™ enzyme kits are summarized in the tables below:

VIC Assay using Invitrogen SuperScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives</td><td rowspan=1 colspan=1>% Positive Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0 (20.7 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0 ( 78.5 - 100.0)</td></tr></table>

1Proportion of true positives correctly identified against the comparator method

YAM Assay using Invitrogen SuperScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives</td><td rowspan=1 colspan=1>% Positive Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0 ( 43.9 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100.0 ( 88.3 - 100.0)</td></tr></table>

1Proportion of true positives correctly identified against the comparator method

The retrospective study results of the Influenza B Lineage Genotyping Assay used with the Quanta qScript™ enzyme kits are summarized in the tables below:

# VIC Assay using Quanta qScript™

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives1</td><td rowspan=1 colspan=1>% Positive Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0 ( 20.7  100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0 (78.5 - 100.0)</td></tr></table>

1Proportion of true positives correctly identified against the comparator method

YAM Assay using Invitrogen SuperScript™   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1># of Positives</td><td rowspan=1 colspan=1>% Positive Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA, NW</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0 ( 43.9 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>28/29</td><td rowspan=1 colspan=1>96.6 (82.8  99.4)</td></tr></table>

1Proportion of true positives correctly identified against the comparator method

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: The expected values are derived from the clinical studies performed during the 2006-

During February 25, 2012, to May 19, 2012, World Health Organization and National Respiratory and Enteric Virus Surveillance System (NREVSS) collaborating laboratories in the United States tested 47,281 respiratory specimens for influenza viruses. Of these, 9,415 $( 1 9 . 9 \% )$ were positive: $8 5 \%$ of the positive specimens were positive for influenza A viruses and $1 5 \%$ were positive for influenza B viruses. Among the 5,071 influenza A viruses for which subtyping was performed, 3,680 $( 7 2 . 6 \% )$ were influenza $\mathrm { A } / \mathrm { H } 3$ viruses and 1,391 $( 2 7 . 4 \% )$ were 2009 H1N1 influenza viruses.

From August 30, 2009, through March 27, 2010, World Health Organization (WHO) and National Respiratory and Enteric Virus Surveillance System (NREVSS) collaborating laboratories in the United States tested 422,648 specimens. Of these, 89,585 $( 2 1 . 1 \% )$ were positive: 89,298 $( 9 9 . 7 \% )$ were positive for influenza A and 287 $( 0 . 3 \% )$ were positive for influenza B. Among 66,978 influenza A viruses for which subtyping was performed, almost all (66,589 $[ 9 9 . 4 \% ] $ ) were 2009 H1N1 viruses. Of the 37,260 specimens reported during February 14 - March 27, 2010, a total of 2,020 $( 5 . 4 \% )$ tested positive for influenza, of which 1,999 $( 9 8 . 9 \% )$ were positive for influenza A and 21 $( 1 . 0 \% )$ were positive for influenza B. Of the 1,510 influenza A viruses reported since mid-February for which subtyping was performed, almost all (1,506 $[ 9 9 . 7 \% ] ,$ ) were 2009 H1N1 viruses. No seasonal influenza A (H1) viruses and only three influenza A (H3) viruses were reported. During February 14 - March 27, states in the Southeast (Alabama, Florida, Georgia, Kentucky, Mississippi, North Carolina, South Carolina, and Tennessee) accounted for approximately $5 5 \%$ of the influenza positives reported but only $20 \%$ of the specimens tested.

During October 1, 2006--May 19, 2007, World Health Organization and National Respiratory and Enteric Virus Surveillance System (NREVSS) collaborating laboratories in the United States tested 179,268 respiratory specimens for influenza viruses; 23,753 $( 1 3 . 2 \% )$ were positive. Of these, 18,817 $( 7 9 . 2 \% )$ were influenza A viruses and 4,936 $( 2 0 . 8 \% )$ were influenza B viruses. Among the influenza A viruses, 6,280 $( 3 3 . 4 \% )$ were subtyped; 3,912 $( 6 2 . 3 \% )$ were influenza A/H1 viruses and 2,368 $( 3 7 . 7 \% )$ were influenza A/H3 viruses (http://www.cdc.gov/mmwr/preview/mmwrhtml/mm5631a2.htm). In the rRT-PCR Flu Panel multicenter prospective clinical study during the 2006-2007 influenza season, the prevalence as determined by virus culture was as follows: influenza A/H1 $( 7 . 0 \% )$ , influenza A/H3 $( 2 3 . 6 \% )$ , and influenza B $( 9 . 9 \% )$ .

# N. Instrument Name:

Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR

# O. System Descriptions:

1. Modes of Operation:

The Applied Biosystems 7500 Fast Dx Real-Time PCR instrument integrates a thermal cycler, a fluorimeter, and application specific software. The instrument houses the thermal cycler and the fluorimeter, while the application software is run on a PC that is attached to the instrument. Samples are placed in a tube strip or 96-well low-head space plate that is moved to a Peltier-based thermal block and positioned relative to the optics using a tray loading mechanism.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification: User manually enters Patient ID/Sample ID.

4. Specimen Sampling and Handling: Not applicable.

5. Calibration: Not applicable.

6. Quality Control: Quality control is addressed for each specific assay to be run on the instrument (separately cleared).

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.