# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

# B. Purpose for Submission:

Substantial equivalence determination for the Verigene® Gram Negative Blood Culture (BCGN) Nucleic Acid Test on the Verigene® System.

# C. Measurand:

Target DNA sequences of the following gram negative bacteria and resistance markers (RM): Acinetebacter spp., Citrobacter spp, Enterobacter spp. Proteus spp., Escherichia coli, Klebsiella pneumoniae, Klebsiella oxytoca, Pseudomonas aeruginosa, CTX-M (blaCTX-M),KPC $( b l a _ { \mathrm { K P C } } )$ , NDM (blaNDM), VIM (blaVIM), and IMP $\left( b l a _ { \mathrm { I M P } } \right)$ and OXA (blaOXA).

# D. Type of Test:

Qualitative, in vitro diagnostic test for the detection of specific nucleic acid targets in a microarray format using capture and mediator oligonucleotides for gold nanoparticle probebased endpoint detection.

# E. Applicant:

Nanosphere, Inc.

F. Proprietary and Established Names:

Verigene Gram-Negative Blood Culture Nucleic Acid Test (BC-GN)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3365 - Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures

2. Classification:

Class II

3. Product code:

PEN: Gram-Negative Bacteria and Associated Resistance Markers NSU: Instrumentation for Clinical Multiplex Test Systems

4. Panel:

83 (Microbiology)

# H. Intended Use:

1. Intended use(s):

The Verigene® Gram Negative Blood Culture Nucleic Acid Test (BC-GN), performed using the sample-to-result Verigene System, is a qualitative multiplexed in vitro diagnostic test for the simultaneous detection and identification of selected gram-negative bacteria and resistance markers. BC-GN is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system and which contain gramnegative bacteria as determined by gram stain.

BC-GN detects and identifies the following:

<table><tr><td>Bacterial Genera and Species</td><td>Resistance Markers</td></tr><tr><td>Acinetobacter spp. Citrobacter spp. Enterobacter spp. Proteus spp. Escherichia colil Klebsiella pneumoniae Klebsiella oxytoca Pseudomonas aeruginosa</td><td>CTX-M (blacTx-M) KPC (blaκPc) NDM (blanDM) VIM (blavIM) IMP (blaMP) OXA (blaoXA)</td></tr></table>

1BC-GN will not distinguish Escherichia coli from Shigella spp. (S. dysenteriae, S. flexneri, S. boydii, and S. sonnei)

BC-GN is indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor these infections. Sub-culturing of positive blood cultures is necessary to recover organisms for antimicrobial susceptibility testing (AST), for identification of organisms not detected by BC-GN, to detect mixed infections that may not be detected by BC-GN, for association of antimicrobial resistance marker genes to a specific organism, or for epidemiological typing.

2. Indication(s) for use:

Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

For use with the Verigene System which consists of a bench top work station consisting of two modules: the Verigene Processor SP and the Verigene Reader.

# I. Device Description:

BC-GN is performed using the Verigene System, which is a bench-top sample-to-result molecular diagnostics workstation consisting of two modules: the Verigene Processor $S P$ and the Verigene Reader. The Verigene Processor $S P$ automates the BC-GN sample analysis steps including: (i) Specimen Preparation - cell lysis and magnetic bead-based nucleic acid extraction from positive blood culture specimens and (ii) Hybridization—bacterial DNA hybridization to target specific capture DNA in a microarray format and mediator and gold-nanoparticle probe hybridization to captured bacterial nucleic acids. Silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that are assessed optically by the Verigene Reader. The Verigene Reader also serves as the user interface and central control unit for the Verigene System, storing and tracking information throughout the assay process.

The Verigene Processor $S P$ utilizes single-use disposables to perform BC-GN, including an Extraction Tray, Utility Tray, and Verigene Test Cartridge. A separate Tip Holder Assembly contains two pipette tips that are used to transfer and mix reagents during the assay. The user tests a specimen by loading the single-use disposables into the Verigene Processor $S P$ , pipetting the prepared specimen into the Extraction Tray, and initiating the protocol on the Verigene Reader by scanning or entering the Test Cartridge ID and specimen information. Following assay completion, the user inserts the Test Cartridge into the Verigene Reader for optical analysis and generation of BC-GN test results.

The Verigene BC-GN Kit contains:

· Verigene BC-GN Test Cartridges: Each Test Cartridge comes preloaded with all required reaction solutions, including wash solutions, oligonucleotide probe solution and signal amplification solutions. Verigene BC-GN Extraction Trays (with Tip Holder Assemblies): Each Extraction Tray comes preloaded with all required solutions, including lysis/binding buffer, digestion enzymes, wash solutions, and buffer solutions necessary to extract nucleic acids.

The Verigene BC-GN Utility Kit contains Verigene BC-GN Utility Trays. Each Utility Tray comes preloaded with an Internal Processing Control.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

FilmArray Blood Culture Identification Panel (BCID) Verigene Gram-Positive Blood Culture Nucleic Acid Test (BC-GP)

2. Predicate 510(k) number(s): K130914, K122514

3. Comparison with predicates:

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Verigene BC-GN Test</td><td colspan="1" rowspan="1">Predicate: FilmArrayBCID Panel (K130914)</td><td colspan="1" rowspan="1">Predicate: VerigeneBC-GP Test(K122514)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative in vitrodiagnostic test fordetection andidentification ofmicroorganisms andresistance marker nucleicacids</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">OrganismsandResistanceMarkersDetected</td><td colspan="1" rowspan="1">Detects Klebsiellapneumoniae, Klebsiellaoxytoca, Escherichiacoli, Proteus spp,Pseudomonasaeruginosa and the KPCgene (See Differencestable below foradditional organisms andresistance markersdetected)</td><td colspan="1" rowspan="1">Same (See Differencestable below for additionalorganisms and resistancemarkers detected)</td><td colspan="1" rowspan="1">See Differences table</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid-based</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SampleProcessingandPurification</td><td colspan="1" rowspan="1">Automated byInstrument</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretationof Results</td><td colspan="1" rowspan="1">Software DecisionAlgorithm</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two internal processingcontrols (whole organismcomplete assay control andsingle-stranded DNAHybridization control)</td><td colspan="1" rowspan="1">Two internal controls(DNA Process Control tocontrol for entire testprocess and PCR2Control to control for the2nd stage PCR process)</td><td colspan="1" rowspan="1">Same as BC-GN</td></tr><tr><td colspan="4">Differences</td></tr><tr><td>Item</td><td>Verigene BC-GN Test</td><td>Predicate: FilmArray BCID Panel</td><td>Predicate: Verigene BC-GP Test</td></tr><tr><td>Organisms and Resistance Markers Detected</td><td>Detection of the following additional bacteria and resistance gene targets: Acinetobacter spp, Citrobacter spp, Enterobacter species, CTX-M, NDM, VIM, IMP, OXA</td><td>Detection of the following additional bacterial identification and resistance gene targets: Acinetobacter baumanii, Enterobacteriaceae, Enterobacter cloacae complex, Haemophilus influenzae, Neisseria meningitis (encapsulated), Serratia marcescens,</td><td>Detection of multiple gram positive bacteria and mecA, vanA, and vanB.</td></tr><tr><td>Technological Principles</td><td>Qualitative, multiplexed test for the detection of specific nucleic acid targets in a microarray format using capture and mediator oligonucleotides for gold nanoparticle probe- based endpoint detection. Does not use</td><td>positive bacteria and Candida species, mecA and vanA/vanB. Nested multiplex PCR followed by high resolution melting analysis to confirm identity of amplified product.</td><td>Same as BC-GN</td></tr><tr><td>Instrumentation</td><td>amplification. Verigene Reader and Processor SP</td><td>FilmArray Instrument</td><td>Verigene Reader and Processor SP</td></tr><tr><td>Time to Result</td><td>~2.0 hours</td><td>Less than 1 hour</td><td>~2.0 hours</td></tr></table>

# K. Standard/Guidance Documents Referenced

CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline - Second Edition   
· CLSI EP7-A2; Interference Testing in Clinical Chemistry Approved Guideline – Second Edition   
· CLSI EP12-A; User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition   
C CLSI MM3-A2; Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline - Second Edition CLSI EP15-A2 - User Verification of Performance for Precision and Trueness; Approved Guideline – second edition CLSI EP9-A2 - Method Comparison and Bias Estimation Using Patient Samples; Approved Guideline – second edition Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable (April 25, 2006)   
· Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices (May 11, 2005)   
· Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005) Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices – Draft Guidance for Industry and FDA Staff (November 9, 2012) Establishing the Performance Characteristics of Nucleic Acid-Based In Vitro Diagnostic Devices for the Detection and Differentiation of MRSA and SA – Draft Guidance for Industry and FDA Staff (January 5, 2011) In Vitro Diagnostic (IVD) Device Studies – Frequently Asked Questions (June 25, 2010)   
· Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests (March 13, 2007)

# L. Test Principle:

The Verigene Gram Negative Blood Culture Nucleic Acid Test (BC-GN) is a molecular assay which relies on detection of specific nucleic acid targets in a microarray format. The Verigene Processor $S P$ executes the test procedure, automating the steps of (1) Sample Preparation– cell lysis and magnetic bead-based bacterial DNA isolation from blood culture samples, and (2) Hybridization–detection and identification of bacterial-specific DNA in a microarray format by using gold nanoparticle probe-based technology. Once the specimen is loaded by the operator, all other fluid transfer steps are performed by an automated pipette that transfers reagents between wells of the trays and loads the specimen into the Test Cartridge for hybridization. Single-use disposable test consumables and a self-contained Verigene Test Cartridge are utilized for each sample tested with the BC-GN test.

For each of the bacterial DNA sequences detected by the BC-GN test, unique Capture and Mediator oligonucleotides are utilized. The Capture oligonucleotides are covalently bound to the microarray substrate and hybridize to a specific portion of the nucleic acid targets. The Mediator oligonucleotides have regions which bind to a different portion of the same nucleic acid targets and also have sequences which allow binding of gold nanoparticle probes. Specific silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light, allowing detection of target capture.

To obtain the test results after processing is complete, the user removes the Test Cartridge from the Processor $S P$ , and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader and intensities from the microarray spots are used to make a determination regarding the presence (Detected) or absence (Not Detected) of a bacterial nucleic acid sequence/analyte. This determination is made by means of software-based decision algorithm resident in the Reader.

# M. Performance Characteristics:

1. Analytical performance:

a. Reproducibility:

A reproducibility study was performed at three external clinical testing sites with test panels consisting of samples prepared with eight different targeted organisms tested at two concentrations: the concentration at bottle positivity and eight hours after bottle positivity. All BC-GN targeted resistance genes were represented in one or more of the organisms tested. Two negative samples were also tested, one containing negative human whole blood and blood culture media only and one containing Morganella morganii, an organism not detected by BC-GN. Samples were prepared by spiking organisms at 5-30 CFU/bottle into blood culture bottles containing blood culture media and negative human whole blood. Bottles were incubated on the BACTEC 9050 System and removed at bottle positivity and at eight hours past bottle positivity. Single-use aliquots were frozen at $\le 7 0 ^ { \circ } \mathrm { C }$ and shipped to testing sites. The 16 panel members were tested in triplicate twice daily by two operators for five non-consecutive days generating 90 results per panel member. The panel members are provided in the following table.

BC-GN Reproducibility Sample Panel Composition   

<table><tr><td rowspan=2 colspan=1>SampleNo.</td><td rowspan=2 colspan=1>Organism/Sample</td><td rowspan=2 colspan=1>Resistance Marker 1</td><td rowspan=2 colspan=1>ResistanceMarker 2</td><td rowspan=2 colspan=1>Source No.</td><td rowspan=1 colspan=2>Concentration</td></tr><tr><td rowspan=1 colspan=1>IncubationTime</td><td rowspan=1 colspan=1>CFU/mL</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative (Bloodculture media andnegative human</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5 Days</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 25830</td><td rowspan=1 colspan=1>BP*</td><td rowspan=1 colspan=1>4.70 x107</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Acinetobacter**baumannii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 128307</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.46 x 1085.65x 108</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 549813</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>2.33 x108</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 550287</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>3.15 x108</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 449261</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>4.55 x108</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>JMI 18518</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>6.65 x108</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 683079</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.15 x109</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 12453</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.45 x108</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Pseudomonas</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 576602</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>3.35 x108</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 128307</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>7.00 x108</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 549813</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>2.58 x109</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 550287</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.80 x109</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 449261</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.44 x109</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>JMI 18518</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>2.80 x108</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 683079</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.29 x109</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 12453</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>9.55 x108</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 576602</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.24 x108</td></tr></table>

\* BP – ‘bottle positivity’ or evidence of bacterial growth in the automated blood culture monitoring system \*\* Two preparations of A. baumanii were used in the study.

A total of 1620 initial tests were conducted. There were nine pre-analysis errors (pre-AE) for an initial pre-AE rate of $0 . 5 \%$ (9/1652). All nine were repeated and yielded valid results. There were 23 initial No Calls for an initial No Call rate of 23/1652 or $1 . 4 \%$ . Repeat testing was performed for each of the 23 samples with all yielding valid results. The final call rate (number of valid tests/total tests conducted) was $100 \%$ (1620/1620) with all replicates yielding the expected result for each panel member. Test results as shown in the following table indicate that the Verigene BC-GN Test yielded reproducible results in the multi-center study.

Reproducibility Test Results   

<table><tr><td rowspan=1 colspan=3>Sample</td><td rowspan=2 colspan=1>ExpectedCal(s)</td><td rowspan=1 colspan=2>Bottle Positivity</td><td rowspan=1 colspan=2>Bottle Positivity + 8hours</td></tr><tr><td rowspan=1 colspan=1>Organism/Specimen</td><td rowspan=1 colspan=1>ResistanceMarker(s)</td><td rowspan=1 colspan=1>SourceNo.</td><td rowspan=1 colspan=1>FinalCallRate</td><td rowspan=1 colspan=1>Accuracy</td><td rowspan=1 colspan=1>Final CallRate</td><td rowspan=1 colspan=1>Accuracy</td></tr><tr><td rowspan=1 colspan=1>Negative Control -Blood Culture MediaOnly</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC25830</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaumanii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>IHMA128307</td><td rowspan=1 colspan=1>Acinetobacterspp. &amp; OXA</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>IHMA549813</td><td rowspan=1 colspan=1>Citrobacterspp. &amp; VIM</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>IHMA550287</td><td rowspan=1 colspan=1>Enterobacterspp. &amp; KPC</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>IHMA449261</td><td rowspan=1 colspan=1>Escherichiacoli &amp; NDM</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>OXA, CTX-M</td><td rowspan=1 colspan=1>JMI18518</td><td rowspan=1 colspan=1>Klebsiellapneumoniae&amp; OXA &amp;CTX-M</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>IHMA683079</td><td rowspan=1 colspan=1>Klebsiellaoxytoca &amp;CTX-M</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC12453</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>IHMA576602</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa &amp;IMP</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td><td rowspan=1 colspan=1>100%90/90(96.0-100)</td></tr></table>

# $b$ . Precision

The precision of the BC-GN test was determined by conducting an internal precision study using an 18 member sample panel. Samples were prepared with eight different targeted organisms tested at two concentrations, bottle positivity and eight hours after bottle positivity. All BC-GN targeted resistance genes were represented in one or more of the organisms tested. In addition, two negative samples were tested, one containing negative human whole blood and blood culture media only and one containing Halnia alvei, an organism not detected by BC-GN. Tested was performed in duplicate twice daily by two operators for 12 non-consecutive days. Samples were prepared by spiking organisms at 5-30 CFU/bottle into blood culture bottles containing blood culture media and negative human whole blood. Bottles were grown on the BACTEC 9050 System and removed at bottle positivity and at eight hours past bottle positivity. Panel members were tested twice daily by two operators over 12 non-consecutive days for a total of 48 replicates per panel member. The panel composition is provided in the following table.

BC-GN Test Precision Sample Panel Composition   

<table><tr><td rowspan=2 colspan=1>SampleNo.</td><td rowspan=2 colspan=1>Organism/Sample</td><td rowspan=2 colspan=1>ResistanceMarker 1</td><td rowspan=2 colspan=1>ResistanceMarker 2</td><td rowspan=2 colspan=1>Source No.</td><td rowspan=1 colspan=2>Concentration</td></tr><tr><td rowspan=1 colspan=1>IncubationTime</td><td rowspan=1 colspan=1>CFU/mL</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative (Blood culturemedia and negativehuman blood)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5 Days</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 13337</td><td rowspan=1 colspan=1>BP*</td><td rowspan=1 colspan=1>4.95 x107</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 128307</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.46 x108</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 549813</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>2.33 x108</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 550287</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>3.15 x108</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 449261</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>4.55 x108</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>JMI 18518</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>6.65 x108</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 683079</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.15 x109</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 12453</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>1.45 x108</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 576602</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>3.35 x108</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 128307</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>7.00 x108</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 549813</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>2.58 x109</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 550287</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.80 x109</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 449261</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.44 x109</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>JMI 18518</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>2.80 x108</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 683079</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.29 x109</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC 12453</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>9.55 x108</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>IHMA 576602</td><td rowspan=1 colspan=1>BP+8 hrs.</td><td rowspan=1 colspan=1>1.24 x108</td></tr></table>

${ \bf \mathrm { * } } _ { \mathrm { B P - } }$ ‘bottle positivity’ or evidence of bacterial growth in the automated blood culture monitoring system

Testing generated 48 results per panel member for a total of 864 test results. All replicates for each panel member yielded concordant results with the exception of one false positive result for the $K .$ . oxytoca target in one replicate of the $K .$ . pneumoniae panel member. Evaluation by in silico analysis suggests a low likelihood of cross-reactivity for $K .$ . pneumoniae with the BC-GN gene target for $K .$ . oxytoca. As a result of the observed cross-reactivity and in silico investigation, the draft BC-GN Package Insert contains the following limitation:

Based on sequence homology analysis and analytical testing, a low likelihood of cross-reactivity exists between BC-GN probes which detect Klebsiella pneumoniae and the gene target for Klebsiella oxytoca. Therefore, on rare occasions, both a “K. oxytoca Detected” result and a “K. pneumoniae Detected” result may be obtained when Klebsiella pneumoniae is present in the specimen.

Results for the Precision study are provided in the following table.

Precision Study Panel Composition and Test Results   

<table><tr><td colspan="3" rowspan="1">Sample</td><td colspan="1" rowspan="2">ExpectedCal(s)</td><td colspan="2" rowspan="1">Bottle Positivity</td><td colspan="2" rowspan="1">Bottle Positivity + 8hours</td></tr><tr><td colspan="1" rowspan="1">Organism/Specimen</td><td colspan="1" rowspan="1">ResistanceMarker(s)</td><td colspan="1" rowspan="1">SourceNo.</td><td colspan="1" rowspan="1">FinalCallRate</td><td colspan="1" rowspan="1">Accuracy</td><td colspan="1" rowspan="1">FinalCallRate</td><td colspan="1" rowspan="1">Accuracy</td></tr><tr><td colspan="1" rowspan="1">Negative (Bloodculture media andnegative humanblood</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC13337</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">97.9%(47/48)88.9-100</td><td colspan="1" rowspan="1">100%(47/47)92.5-100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumanii</td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">IHMA128307</td><td colspan="1" rowspan="1">Acinetobacterspp. &amp; OXA</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">IHMA549813</td><td colspan="1" rowspan="1">Citrobacterspp. &amp; VIM</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">IHMA550287</td><td colspan="1" rowspan="1">Enterobacterspp. &amp; KPC</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">IHMA449261</td><td colspan="1" rowspan="1">Escherichiacoli &amp; NDM</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">OXA, CTX-M</td><td colspan="1" rowspan="1">JMI18518</td><td colspan="1" rowspan="1">Klebsiellapneumoniae&amp; OXA &amp;CTX-M</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">97.9%(47/48)88.9-100</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">IHMA683079</td><td colspan="1" rowspan="1">Klebsiellaoxytoca &amp;CTX-M</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC12453</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td><td colspan="1" rowspan="1">100%(48/48)92.6-100</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">IHMA576602</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa &amp;</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(48/48)</td></tr><tr><td></td><td></td><td></td><td>IMP</td><td>92.6- 100</td><td>92.6-100</td><td>92.6-100</td><td>92.6-100</td></tr></table>

c. Linearity/assay reportable range:

Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Controls: BC-GN contains two sets of internal controls to ensure proper fluid control, hybridization, and signal detection.

Internal Control 1 (INT CTL 1) is a processing control which detects the presence (hybridization and signal enhancement) of an artificial DNA oligonucleotide construct and mediator oligonucleotide contained within the Sample buffer on the Extraction Tray. This material serves to monitor hybridization inhibition or test cartridge reagent failures. If the signals from INT CTL 1 are not valid the test should be repeated.

Internal Control 2 (INT CTL 2) is comprised of an intact non-target, gramnegative bacterium (Shewanella oneidensis) contained within the BC-GN utility tray and added to the sample prior to the nucleic acid extraction step. INT CTL2 functions as a complete assay control, the primary purpose of which is to monitor failures attributable to the specimen preparation step (i.e., lysis and nucleic acid isolation) it also functions as a hybridization/detection control. If the signals from INT CTL2 are not valid the test should be repeated.

When detecting a negative sample, both the internal processing controls INT CTL 1 and INT CTL 2 must be present for a valid “Not Detected” call for all targets to be reported. If INT CTL 1 or INT CTL 2 is “Not Detected,” a “No Call – INT CTL 1” or a “No Call – INT CTL $2 ^ { \circ }$ result, respectively, is generated. If both INT CTL 1 and INT CTL 2 are “Not Detected”, a “NO CALL – INT CTL” result is generated. These internal controls are not utilized for the detection of positive samples.

External Controls: External controls are not included with the BC-GN Test but are recommended for routine testing on a rotating basis using 3-4 smaller groups of organisms, and/or under the following circumstances:

Instrument installation, test validation, and when troubleshooting is necessary During performance verification for receipt of a new set/lot of consumables C When the integrity of consumables or the device is in question.

External positive controls may consist of frozen aliquots of positive blood cultures containing organisms targeted by BC-GN. External quality control testing should be performed in conformance with local, state, and federal regulations or accreditations organizations as applicable and should follow the user’s laboratory standard quality control procedures.

System Controls: Each cartridge has a number of internal controls to confirm proper test performance, including: imaging controls, background controls, negative controls, extraction controls and hybridization controls. In addition, the Verigene System continuously monitors system performance while the assay is running. Failures of any instrument controls will result in procedure termination (Pre-Analytical Error, Pre-AE).

# Specimen Stability:

Positive blood culture specimens can be tested with the BC-GN Test immediately after the blood culture is signaled as positive and up to 24 hours post bottle ring when stored at $2 { - } 3 7 ^ { \circ } \mathrm { C }$ . Studies to support these claims included testing at the low, middle and high end of the temperature range over a 36-hour period. A panel of eight contrived specimens, representing the full panel of bacterial and resistance targets was tested. BACTEC Plus Aerobic blood culture bottles containing negative human whole blood were inoculated with low concentrations of organisms and incubated on an automated blood culture system until “bottle positivity”. Study results presented in the following table demonstrated that all analytes were detected in all replicates after storage in the claimed storage conditions.

# BC-GN Test Results for Specimen Stability

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">BC-GN TestExpected Target(s)</td><td colspan="1" rowspan="2">StorageCondition</td><td colspan="3" rowspan="1">BC-GN Call Accuracy</td></tr><tr><td colspan="1" rowspan="1">Positivityt=0 hr</td><td colspan="1" rowspan="1">t=24hours</td><td colspan="1" rowspan="1">t=36hours</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumanniiIHMA 128307</td><td colspan="1" rowspan="3">Acinetobacter spp.OXA</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Citrobacter freundiiIHMA 549813</td><td colspan="1" rowspan="3">Citrobacter spp.VIM</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacaeIHMA 550287</td><td colspan="1" rowspan="3">Enterobacter spp.KPC</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Escherichia coliIHMA 449261</td><td colspan="1" rowspan="3">E. coliNDM</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-24</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniaeJMI 18518</td><td colspan="1" rowspan="3">K. pneumoniaeOXA 48CTX-M</td><td colspan="1" rowspan="1">2-8 °</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 °</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="2">K. oxytoca</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">IHMA 683079</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Proteus mirabilisATCC 12453</td><td colspan="1" rowspan="3">Proteus spp.</td><td colspan="1" rowspan="1">2-8 °</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosaIHMA 576602</td><td colspan="1" rowspan="3">P. aeruginosaIMP</td><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18-26 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33-37 </td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# e. Limit of Detection

The analytical sensitivity of the BC-GN Test was evaluated by testing bacterial strains representative of BC-GN targets. A sub-culture of each strain was inoculated into BACTEC Plus bottles spiked with $1 0 ~ \mathrm { m L }$ of fresh, anticoagulated whole blood, and grown to bottle positivity on the BACTEC blood culture system. To provide the most challenging matrix, each positive blood culture sample was then used to prepare a dilution series using a simulated matrix consisting of BacT/ALERT FA Culture bottle media (containing charcoal), negative human whole blood, and Staphyloccoccus epidermidis (a common skin contaminant. ) The Staphylococcus epidermidis was present in the matrix at a minimum concentration of $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \dot { \mathrm { C F U / m L } }$ .

A total of 12 organisms were tested in the study with four replicates of each sample tested at multiple dilutions in order to determine the preliminary LoD. Confirmation testing was performed and the final LoD determined as the concentration for which 19/20 or 20/20 replicates were detected. For $C .$ . freundii, C. koseri, $P .$ . mirabilis, $P .$ . vulgaris and $E .$ . coli, confirmation testing did not yield a detection rate of at least $9 5 \%$ ; therefore confirmatory testing for these organisms was repeated at the next highest concentration.

Study results demonstrated that the LoD for each organism is lower than the concentration present at bottle positivity. A summary of the LoD confirmation testing for each organism and resistance marker is shown in the following table.

Final Limit of Detection Results for Organism and Resistant Marker (RM) Detection   

<table><tr><td colspan="1" rowspan="2">Genus</td><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="2">ResistanceMarker(s)(RM)</td><td colspan="3" rowspan="1">Number of Detected Calls</td><td colspan="1" rowspan="2">CFU/mL atLoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">RM(1)</td><td colspan="1" rowspan="1">RM(2)</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">OXA-23</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4.61x106</td></tr><tr><td colspan="1" rowspan="1">calcoaceticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4.01x105</td></tr><tr><td colspan="1" rowspan="2">Citrobacter</td><td colspan="1" rowspan="1">freundii</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.26x107</td></tr><tr><td colspan="1" rowspan="1">koseri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.94x106</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">aerogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.06x107</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">cloacae</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1">4.07x106</td></tr><tr><td colspan="1" rowspan="2">Proteus</td><td colspan="1" rowspan="1">mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.72x105</td></tr><tr><td colspan="1" rowspan="1">vulgaris</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.91x105</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">CTX-M,OXA-48</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">1.19x107</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">oxytoca</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.99x107</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.74x106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.32x107</td></tr></table>

# f. Analytical Reactivity (Inclusivity):

The analytical reactivity of the BC-GN Test was evaluated using a panel of 195 strains for 45 different bacterial species. Samples were prepared by inoculating each organism into a blood culture bottle containing negative anticoagulated human whole blood. Bottles were then incubated on the BACTEC blood culture instrument until bottle positivity followed by subculturing to agar media for purity checks and colony counts.

Organisms were selected to cover the genetic diversity of each BC-GN target and antibiotic resistance marker. At least 10 strains for each species target and resistance marker target were evaluated, with additional strains evaluated for genus level targets. Testing was performed in triplicate and included the following for the BC-GN species targets: 10 strains of K. oxytoca, 25 strains of $K$ . pneumoniae, 13 strains of $P .$ . aeruginosa, eight species of Shigella sp. and 17 strains of E. coli. Additional replicates were tested for any organism that yielded discordant results. Organisms evaluated for the genus level targets are presented in the following table:

Organisms tested for Inclusivity: BC-GN Test Genus level Bacterial Targets   

<table><tr><td rowspan=2 colspan=1>BC-GN Target</td><td rowspan=2 colspan=1>Total No. ofStrainsTested</td><td rowspan=1 colspan=2>Species Tested</td></tr><tr><td rowspan=1 colspan=1>Name (No. of Strains)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.*</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>A. baylyi (2), A. baumannii (8), A. bereziniae (1), A.calcoaceticus (5), A. guillouiae (1), A. haemolyticus (5),A. johnsonii (3), A. junii (3), A. Iwoffi (3), A.radioresistens (3), A. schindleri (1), and A. ursingii (1)</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Citrobacter spp.</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>C. amalonaticus (2), C. braakii (5) , C. farmeri (2), C.freundii (5), C. gillenii (3), C. koseri (5), C. murliniae(3), C. rodentium (5), C. sedlakii (3), C. werkmanii (3),and C. youngae (5)</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Enterobacter spp*.</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>E. aerogenes (5), E. amnigenus (2), E. asburiae (4), E.cancerogenus (5), E. cloacae (8), E. hormaechei (3), E.ludwigii (1), and E. nimipressuralis/E. orzae (1)</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>P. hauseri (2), P. mirabilis (6), P. myxofaciens (1), P.penneri (2), and P. vulgaris (5)</td><td rowspan=1 colspan=1>5</td></tr></table>

\* BC-GN does not detect Acinetobacter tartarogenes, Enterobacter gergoviae, Enterobacter kobei, and Enterobacter pyrinus

Inclusivity testing yielded the expected results for bacterial and resistance targets with the following exceptions:

In two strains of A. radioresistans (out of thirteen total strains of Acinetobacter sp. containing the OXA gene), 4 of 12 replicates for one strain and 5 of 12 replicates for a second strain yielded false negative results for the OXA target. The Acinetobacter spp. target was   
correctly detected in all replicates.   
In 2 of 9 replicates for one strain of $C .$ . amaloniticus, BC-GN gave false negative results for the Citrobacter spp. target.

An investigation to determine the cause for the false negative OXA results discovered that the OXA signals for these two strains were weaker than those observed for other organisms with this marker. The potential for false negative results for OXA lead to the inclusion of the following limitations in the package insert:

In rare instances for specimens with organisms carrying a resistance marker, BC-GN may not yield a positive result for the resistance marker when the organism(s) is detected.   
False negative results for OXA may occur in certain Acinetobacter radioresistens strains that result as “Acinetobacter Detected” with BC-GN.

One strain of Citrobacter amalonaticus (ATCC 25405) yielded false negative results for 2 of 9 replicates; however another strain (ATCC 25407) yielded the expected results in 9/9 replicates. In silico analysis based on probe sequence homology mismatches suggests that BC-GN could potentially yield a false negative result for $C .$ amalonaticus and therefore the following limitation is included in the package insert:

In silico sequence analysis indicated sequence homology mismatches of C. amalonaticus that may yield false negative results, as demonstrated during BC-GN analytical testing, which resulted in detection of “Citrobacter” a total of 16 out of 18 tests $( 8 8 . 9 \% )$ for these organisms.

As predicted by in silico analysis, all replicates for Shigella boydii, S. dysenteriae, S. flexneri, and S. sonnei were detected with the BC-GN test, and reported as E. coli. The following limitation has been added to the BC-GN package insert.

BC-GN will not distinguish Escherichia coli from Shigella spp. including S. dysenteriae, S. flexneri, S. boydii, and S. sonnei.

Of the 195 strains tested in the inclusivity studies, 79 contained one or more resistance markers associated with 11 different bacterial species including a total of 38 strains containing CTX-M, 16 containing OXA, 12 containing IMP, 10 containing VIM, 10 containing KPC, and 9 containing NDM. Separately, in silico analysis was performed by aligning the assay probes for each of the strains/organisms against available GenBank sequence entries. The following table includes the various resistance marker subgroups and types that were evaluated by wet-testing and by in silico analysis.

Summary of RMs Detected by Wet testing or Predicted to be Detected based on in silico Analysis   

<table><tr><td rowspan=2 colspan=1>Merrrr</td><td rowspan=2 colspan=1>TraditionalSubgroups</td><td rowspan=1 colspan=2>Detected by Wet Testing</td><td rowspan=1 colspan=2>Predicted to Be Detected based on In Silico Analysis2</td></tr><tr><td rowspan=1 colspan=1>No. ofSamples</td><td rowspan=1 colspan=1>Type Tested</td><td rowspan=1 colspan=1>Types With Identical Probe BindingSites to Wet Tested Types1</td><td rowspan=1 colspan=1>Types with in SilicoData Only</td></tr><tr><td rowspan=5 colspan=1>C-XLO</td><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>1, 3, 12, 15, 22,28, 30, 55, 79</td><td rowspan=1 colspan=1>11, 23, 29, 32, 33, 36, 37, 42, 52, 54, 57,58, 60, 61, 62, 66, 69, 71, 72, 80, 82, 88,96, 101, 107, 109, 114, 116, 117, 133</td><td rowspan=1 colspan=1>10, 34, 53, 68, 108,123, 132</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2,31</td><td rowspan=1 colspan=1>4, 5, 6, 7, 20, 43, 44, 56, 59, 77, 92, 95,97, 124, 131</td><td rowspan=1 colspan=1>74, 75, 76</td></tr><tr><td rowspan=1 colspan=1>CTX-M-8</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40,63</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>9, 14, 24, 27, 45</td><td rowspan=1 colspan=1>102, 104, 105, 106, 110, 111, 112, 113,121, 122, 126, 13, 134, 16, 17, 18, 19, 21,38, 46, 47, 48, 49, 50, 51, 64, 65, 67, 81,83, 84, 85, 86, 87, 90, 93, 98, 99</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CTX-M-25</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>25, 26, 41, 89, 91, 94, 100</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1, 4, 7, 8, 13, 15,16, 18, 26, 27</td><td rowspan=1 colspan=1>2, 5, 6, 10, 11, 19, 20, 21, 24, 25, 28, 29,30, 33, 37, 38, 40, 41, 42</td><td rowspan=1 colspan=1>3, 9, 12, 14, 22, 32,34, 35</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>2, 3, 4, 5, 11</td><td rowspan=1 colspan=1>1, 6, 7, 8, 9, 10, 12, 13, 14</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1, 4,6</td><td rowspan=1 colspan=1>2, 3, 5,7</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=4 colspan=1>O</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>27, 49, 73, 133, 146, 165, 166, 167, 168,169, 170, 171, 225, 239</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>24, 40</td><td rowspan=1 colspan=1>25, 26, 33, 72, 139, 160, 207</td><td rowspan=1 colspan=1>143, 182, 231</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48, 162</td><td rowspan=1 colspan=1>163, 181, 199, 204, 232</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>96, 97, 164</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>1, 2, 4, 5, 7, 26,27, 28, 33</td><td rowspan=1 colspan=1>3, 6, 8, 9, 10, 11, 12, 14, 15, 16, 17, 18,19, 20, 23, 24, 25, 29, 30, 31, 32, 34, 35,36, 37</td><td rowspan=1 colspan=1>13</td></tr></table>

(1) The BC-GN probe binding sites in these types are identical to those in the wet tested types; therefore, BCGN performance for these types is expected to be the same as those that were wet tested. (2) These specific resistance marker types and subtypes were evaluated by in silico analysis only.

The following two tables provide resistance marker types associated with the bacterial species that were evaluated by wet-testing during the Clinical and Analytical Studies, respectively.

Summary of Resistance Marker Types Linked to Organisms(1) Wet Tested – Clinical Study   

<table><tr><td rowspan=2 colspan=1>BC-GN-DetectedLinkedOrganism</td><td rowspan=1 colspan=2>CTX-M</td><td rowspan=1 colspan=2>OXA</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>IMP</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types(#)(1)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types(#)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types (#)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types(#)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types(#)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Types (#)</td></tr><tr><td rowspan=1 colspan=1>A. baumanii(2)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1(2), 4</td></tr><tr><td rowspan=1 colspan=1>A. lwoffii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A.radioresistens</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>C. braakii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1(2), 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1(2)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. cloacae 2)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3, 14,15(3)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1(7), 4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1(5)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>1,15(22),27, 55</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>48(16)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2(2)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1(11),6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. oyxtoca</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48(2)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K.pneumoniae</td><td rowspan=1 colspan=1>n6</td><td rowspan=1 colspan=1>15(38),27(3)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>48(21)</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>2(22),3(4),11(2)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1(16),26(3),33</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1(16)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>26(3)</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>27(2)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>7(2), 13</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1(3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1(4)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1(4)</td><td rowspan=1 colspan=1>-</td></tr></table>

(1) Only accounts for specimens for which specific resistance marker type identification information was available (2) (3) (4) Includes organisms identified as A. baumanii complex or $E .$ . cloacae complex, respectively Escherichia coli and Acinetobacter baumannii complex and Enterococcus spp. Klebsiella pneumoniae and Enterobacter cloacae complex

Summary of Resistance Marker Types Linked to Organisms(1) Wet Tested – Analytical Studies   

<table><tr><td colspan="1" rowspan="2">LinkedOrganism</td><td colspan="2" rowspan="1">CTX-M</td><td colspan="2" rowspan="1">OXA</td><td colspan="2" rowspan="1">KPC</td><td colspan="2" rowspan="1">VIM</td><td colspan="2" rowspan="1">NDM</td><td colspan="2" rowspan="1">IMP</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)(2)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Types (#)</td></tr><tr><td colspan="1" rowspan="1">A. baumanii(3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">23(4),24/40,58(3)</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. lwoffi</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A.radioresistens</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">23(3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2,3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. cloacae(3)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2, 9, 12, 15,30</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">.</td></tr><tr><td colspan="1" rowspan="1">E.hormaechei</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">1(2), 2, 3,8(2), 14,15(5), 24,27, 28, 55</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">48(2)</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1(2), 4(2),6</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">K. oyxtoca</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">14, 31, 34</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K.pneumoniae</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">1(4), 8, 12,14, 15(3),</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">48(2), 162</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2(2), 3, 4,11(2)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1, 26, 27</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1(2)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8, 26(2)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22, 39, 79</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1, 2, 7, 28</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1,7, 15</td></tr></table>

The following table provides a summary of the organisms tested in the inclusivity study that contained single and dual resistance markers detected by BC-GN.

Organisms Tested for Inclusivity amongst the BC-GN Test Resistance Marker Targets   

<table><tr><td rowspan=1 colspan=2>BC-GN ResistanceMarker Target</td><td rowspan=1 colspan=1>Total No.StrainsTested</td><td rowspan=1 colspan=1>Species Tested Containing the ResistanceMarker</td></tr><tr><td rowspan=6 colspan=1>Presence ofa SingleResistanceMarker</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Citrobacter freundii (1), Enterobacter cloacae (4),Escherichia coli (9), Klebsiella oxytoca (3), Klebsiellapneumoniae (5)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Acinetobacter baumannii (7), Acinetobacter Iwoffi (1),Acinetobacter radioresistens(3)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (3), Proteus mirabilis (1),Pseudomonas aeruginosa (6)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Acinetobacter baumannii (1), Citrobacter freundii (1),Enterobacter cloacae (1), Klebsiella pneumoniae (3),Pseudomonas aeruginosa (4)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Enterobacter cloacae (1), Enterobacter hormaechei (1),Citrobacter freundii (2)i, Klebsiella pneumoniae (5)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Escherichia coli</td></tr><tr><td rowspan=4 colspan=1>Presence ofDualResistanceMarkers</td><td rowspan=1 colspan=1>NDM/CTX-M</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Enterobacter cloacae (1), Escherichia coli (4), Klebsiellapneumoniae (3)</td></tr><tr><td rowspan=1 colspan=1>IMP/CTX-M</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Escherichia coli (1), Klebsiella pneumoniae (1)</td></tr><tr><td rowspan=1 colspan=1>OXA/CTX-M</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Escherichia coli (2), Klebsiella pneumoniae (3)</td></tr><tr><td rowspan=1 colspan=1>KPC/CTX-M</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (1)</td></tr></table>

# g. Analytical specificity (Exclusivity):

Analytical specificity was evaluated using organisms phylogenetically related to panel organisms detected by the BC-GN test, common bloodborne pathogens, organisms without gene sequence information, as well as organisms potentially present as contaminants in blood culture specimens.

Strains were grown in BACTEC blood culture bottles to positivity, checked for purity, quantitated to ensure concentrations of greater than $1 0 ^ { 6 } \mathrm { C F U / m L }$ , and assayed with the BC-GN test in triplicate. For safety reasons, Neisseria meningitidis (ATCC 53414D-5) and Cryptococcus neoformans (ATCC 66031D-5) were tested using genomic DNA ${ { ( 2 . 4 \times 1 0 } ^ { 7 } }$ and $3 . 6 \mathrm { x } 1 0 ^ { 6 }$ copies/assay, respectively). In cases where some degree of cross-reactivity was observed in the initial testing, an additional six replicates were tested.

Samples were divided into two distinct panels of organisms. The first panel consisted of 195 “BC-GN panel” organisms, which in total comprised the analytical inclusivity study samples. There were no false positive results observed for any of the samples tested in this panel.

The second panel consisted of 172 “non-BC-GN panel” organisms including 94 strains of gram negative bacterial strains (88 gram negative rods, six gram-negative cocci), 71 gram positive bacterial strains, and seven yeast strains. The following tables present the non-BC-GN panel organisms that were evaluated.

Gram Negative Organisms Tested   

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter</td><td rowspan=2 colspan=1>tartarogenes</td><td rowspan=1 colspan=1>Leclercia</td><td rowspan=1 colspan=1>adecarboxylata</td></tr><tr><td rowspan=2 colspan=1>Leminorella</td><td rowspan=1 colspan=1>grimontii</td></tr><tr><td rowspan=1 colspan=1>Aggregatibacter</td><td rowspan=1 colspan=1>aphrophilus</td><td rowspan=1 colspan=1>richardii</td></tr><tr><td rowspan=4 colspan=1>Bacteroides</td><td rowspan=1 colspan=1>fragilis</td><td rowspan=1 colspan=1>Morganella</td><td rowspan=1 colspan=1>morganii</td></tr><tr><td rowspan=1 colspan=1>ovatus</td><td rowspan=1 colspan=1>Pantoea</td><td rowspan=1 colspan=1>agglomerans</td></tr><tr><td rowspan=1 colspan=1>uniformis</td><td rowspan=2 colspan=1>Parabacteroides</td><td rowspan=1 colspan=1>distasonis</td></tr><tr><td rowspan=1 colspan=1>thetaiotamicron</td><td rowspan=1 colspan=1>merdae</td></tr><tr><td rowspan=2 colspan=1>Brevundimonas</td><td rowspan=1 colspan=1>diminuta</td><td rowspan=2 colspan=1>Pasteurella</td><td rowspan=1 colspan=1>aerogenes</td></tr><tr><td rowspan=1 colspan=1>vesicularis</td><td rowspan=1 colspan=1>multocida</td></tr><tr><td rowspan=1 colspan=1>Burkholderia</td><td rowspan=1 colspan=1>cepacia</td><td rowspan=1 colspan=1>Plesiomonas</td><td rowspan=1 colspan=1>shigelloides</td></tr><tr><td rowspan=1 colspan=1>Buttiauxella</td><td rowspan=1 colspan=1>gaviniae</td><td rowspan=4 colspan=1>Prevotella</td><td rowspan=1 colspan=1>bivia</td></tr><tr><td rowspan=1 colspan=1>Capnocytophaga</td><td rowspan=1 colspan=1>ochracea</td><td rowspan=1 colspan=1>melaninogenica</td></tr><tr><td rowspan=1 colspan=1>Cardiobacterium</td><td rowspan=1 colspan=1>hominis</td><td rowspan=1 colspan=1>buccae</td></tr><tr><td rowspan=3 colspan=1>Cedecea</td><td rowspan=1 colspan=1>davisae</td><td rowspan=1 colspan=1>denticola</td></tr><tr><td rowspan=1 colspan=1>lapagei</td><td rowspan=3 colspan=1>Providencia</td><td rowspan=1 colspan=1>alcalifaciens</td></tr><tr><td rowspan=1 colspan=1>neteri</td><td rowspan=1 colspan=1>rettgeri</td></tr><tr><td rowspan=1 colspan=1>Comamonas</td><td rowspan=1 colspan=1>testosteroni</td><td rowspan=1 colspan=1>stuartii</td></tr><tr><td rowspan=2 colspan=1>Cronobacter</td><td rowspan=1 colspan=1>sakazakii</td><td rowspan=16 colspan=1>Pseudomonas</td><td rowspan=1 colspan=1>species (genus)</td></tr><tr><td rowspan=1 colspan=1>muytjensii</td><td rowspan=1 colspan=1>alcaligenes</td></tr><tr><td rowspan=1 colspan=1>Delftia</td><td rowspan=1 colspan=1>acidovorans</td><td rowspan=1 colspan=1>chloraphis</td></tr><tr><td rowspan=1 colspan=1>Eikenella</td><td rowspan=1 colspan=1>corrodens</td><td rowspan=1 colspan=1>fragi</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella</td><td rowspan=1 colspan=1>tarda</td><td rowspan=1 colspan=1>fulva</td></tr><tr><td rowspan=1 colspan=1>Enteric group 137</td><td rowspan=1 colspan=1>Enteric group 137</td><td rowspan=1 colspan=1>fluorescens</td></tr><tr><td rowspan=4 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>gergoviae</td><td rowspan=1 colspan=1>fluorescens(2nd srain)</td></tr><tr><td rowspan=1 colspan=1>gergoviae (2nd strain)</td><td rowspan=1 colspan=1>luteola</td></tr><tr><td rowspan=1 colspan=1>kobei</td><td rowspan=1 colspan=1>mendocina</td></tr><tr><td rowspan=1 colspan=1>pyrinus</td><td rowspan=1 colspan=1>mucidolens</td></tr><tr><td rowspan=5 colspan=1>Escherichia</td><td rowspan=1 colspan=1>albertii</td><td rowspan=1 colspan=1>nitroreducens</td></tr><tr><td rowspan=1 colspan=1>blattae</td><td rowspan=1 colspan=1>pertucinogena</td></tr><tr><td rowspan=1 colspan=1>fergusonii</td><td rowspan=1 colspan=1>pseudoalcaligenes</td></tr><tr><td rowspan=1 colspan=1>hermannii</td><td rowspan=1 colspan=1>putida</td></tr><tr><td rowspan=1 colspan=1>vulneris</td><td rowspan=1 colspan=1>putida(2nd strain)</td></tr><tr><td rowspan=1 colspan=1>Elizabethkingia</td><td rowspan=1 colspan=1>meningoseptica</td><td rowspan=1 colspan=1>stutzeri</td></tr></table>

# Gram Negative Organisms Tested

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>Ewingella</td><td rowspan=1 colspan=1>americana</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>veronii</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium</td><td rowspan=1 colspan=1>necrophorum</td><td rowspan=2 colspan=1>Raoultella</td><td rowspan=1 colspan=1>ornithinolytica</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium</td><td rowspan=1 colspan=1>nucleatum</td><td rowspan=1 colspan=1>planticola</td></tr><tr><td rowspan=1 colspan=1>Hafnia</td><td rowspan=1 colspan=1>alvei</td><td rowspan=3 colspan=1>Salmonella</td><td rowspan=1 colspan=1>bongori</td></tr><tr><td rowspan=2 colspan=1>Haemophilus</td><td rowspan=1 colspan=1>influenzae</td><td rowspan=1 colspan=1>enterica subsp entericaserovar Bareilly</td></tr><tr><td rowspan=1 colspan=1>parainfluenzae</td><td rowspan=1 colspan=1>enterica subsp entericaserovar Typhimurium</td></tr><tr><td rowspan=1 colspan=1>Herbaspirillum</td><td rowspan=1 colspan=1>huttiense</td><td rowspan=4 colspan=1>Serratia</td><td rowspan=1 colspan=1>fonticola</td></tr><tr><td rowspan=1 colspan=1>Kingella</td><td rowspan=1 colspan=1>kingae</td><td rowspan=1 colspan=1>liquefaciens complex</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>variicola</td><td rowspan=1 colspan=1>marcescens</td></tr><tr><td rowspan=3 colspan=1>Kluyvera</td><td rowspan=1 colspan=1>ascorbata</td><td rowspan=1 colspan=1>odorifera</td></tr><tr><td rowspan=1 colspan=1>cryocrescens</td><td rowspan=1 colspan=1>Stenotrophomonas</td><td rowspan=1 colspan=1>maltophilia</td></tr><tr><td rowspan=1 colspan=1>georgiana</td><td></td><td></td></tr></table>

# Gram Positive Organisms Tested

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>Aerococcus</td><td rowspan=1 colspan=1>viridans</td><td rowspan=1 colspan=1>Listeria</td><td rowspan=1 colspan=1>monocytogenes</td></tr><tr><td rowspan=2 colspan=1>Arcanobacterum</td><td rowspan=1 colspan=1>bernardiae</td><td rowspan=1 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>luteus</td></tr><tr><td rowspan=1 colspan=1>heamolyticum</td><td rowspan=1 colspan=1>Parvimonas</td><td rowspan=1 colspan=1>micra</td></tr><tr><td rowspan=5 colspan=1>Bacillus</td><td rowspan=1 colspan=1>cereus</td><td rowspan=2 colspan=1>Pediococcus</td><td rowspan=1 colspan=1>acidilactici</td></tr><tr><td rowspan=1 colspan=1>licheniformis</td><td rowspan=1 colspan=1>pentosaceus</td></tr><tr><td rowspan=1 colspan=1>sphaericus</td><td rowspan=1 colspan=1>Peptostreptococcus</td><td rowspan=1 colspan=1>anaerobius</td></tr><tr><td rowspan=1 colspan=1>subtilis</td><td rowspan=2 colspan=1>Planococcus</td><td rowspan=1 colspan=1>citreus</td></tr><tr><td rowspan=1 colspan=1>thuringiensis</td><td rowspan=1 colspan=1>kocurii</td></tr><tr><td rowspan=1 colspan=1>Cellulosimicrobium</td><td rowspan=1 colspan=1>cellulans</td><td rowspan=1 colspan=1>Propionibacterium</td><td rowspan=1 colspan=1>acnes</td></tr><tr><td rowspan=1 colspan=1>Cellumonas</td><td rowspan=1 colspan=1>Turbata</td><td rowspan=1 colspan=1>Rothia</td><td rowspan=1 colspan=1>dentocariosa</td></tr><tr><td rowspan=5 colspan=1>Clostridium</td><td rowspan=1 colspan=1>bifermentans</td><td rowspan=1 colspan=1>Rothia (Stomatococcus)</td><td rowspan=1 colspan=1>mucilaginosa</td></tr><tr><td rowspan=1 colspan=1>clostridioforme</td><td rowspan=9 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>aureus</td></tr><tr><td rowspan=1 colspan=1>perfringens</td><td rowspan=1 colspan=1>aureus (2nd strain)</td></tr><tr><td rowspan=1 colspan=1>septicum</td><td rowspan=1 colspan=1>caprae</td></tr><tr><td rowspan=1 colspan=1>tertium</td><td rowspan=1 colspan=1>epidermidis</td></tr><tr><td rowspan=10 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>bovis</td><td rowspan=1 colspan=1>haemolyticus</td></tr><tr><td rowspan=1 colspan=1>diphtheriae</td><td rowspan=1 colspan=1>hominis</td></tr><tr><td rowspan=1 colspan=1>flavescens</td><td rowspan=1 colspan=1>intermedius</td></tr><tr><td rowspan=1 colspan=1>genitalium</td><td rowspan=1 colspan=1>lugdunensis</td></tr><tr><td rowspan=1 colspan=1>glutamicum</td><td rowspan=1 colspan=1>schleiferi</td></tr><tr><td rowspan=1 colspan=1>jeikeium</td><td rowspan=7 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>agalactiae</td></tr><tr><td rowspan=1 colspan=1>renale</td><td rowspan=1 colspan=1>anginosus</td></tr><tr><td rowspan=1 colspan=1>species</td><td rowspan=1 colspan=1>constellatus</td></tr><tr><td rowspan=1 colspan=1>striatum</td><td rowspan=1 colspan=1>equinus</td></tr><tr><td rowspan=1 colspan=1>urealyticum</td><td rowspan=1 colspan=1>intermedius</td></tr><tr><td rowspan=7 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>avium</td><td rowspan=1 colspan=1>pneumoniae</td></tr><tr><td rowspan=1 colspan=1>casseliflavis</td><td rowspan=1 colspan=1>pyogenes</td></tr><tr><td rowspan=1 colspan=1>durans</td><td rowspan=5 colspan=2></td></tr><tr><td rowspan=1 colspan=1>faecalis</td></tr><tr><td rowspan=1 colspan=1>faecium</td></tr><tr><td rowspan=1 colspan=1>flavescens</td></tr><tr><td rowspan=1 colspan=1>gallinarum</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=8>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>hirae</td><td rowspan=17 colspan=9></td></tr><tr><td rowspan=1 colspan=1>mundtii</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>raffinosus</td></tr><tr><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>Erysipelothrix</td><td rowspan=3 colspan=1>rhusiopathiae</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Finegoldia</td><td rowspan=1 colspan=1>magna</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Kocuria</td><td rowspan=1 colspan=1>kristinae</td></tr><tr><td rowspan=1 colspan=1>Kytococcus</td><td rowspan=1 colspan=1>sedentarius</td></tr><tr><td rowspan=3 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>acidophilus</td><td rowspan=1 colspan=3>b</td></tr><tr><td rowspan=1 colspan=1>crispatus</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>rhamnosus</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>Leuconostoc</td><td rowspan=1 colspan=1>carnosum</td></tr><tr><td rowspan=2 colspan=1>mesenteroids</td></tr><tr></tr></table>

# Gram Negative Cocci Tested

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>Moraxella</td><td rowspan=1 colspan=1>catarrhalis</td></tr><tr><td rowspan=4 colspan=1>Neisseria</td><td rowspan=1 colspan=1>lactamica</td></tr><tr><td rowspan=1 colspan=1>mucosa</td></tr><tr><td rowspan=1 colspan=1>sicca</td></tr><tr><td rowspan=1 colspan=1>Meningitidis*</td></tr><tr><td rowspan=1 colspan=1>Veillonella</td><td rowspan=1 colspan=1>parvula</td></tr></table>

\*Genomic DNA Tested

Yeast Tested   

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=5 colspan=1>Candida</td><td rowspan=1 colspan=1>albicans</td></tr><tr><td rowspan=1 colspan=1>glabrata</td></tr><tr><td rowspan=1 colspan=1>krusei</td></tr><tr><td rowspan=1 colspan=1>parapsilosis</td></tr><tr><td rowspan=1 colspan=1>tropicalis</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus</td><td rowspan=1 colspan=1>neoformans*</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces</td><td rowspan=1 colspan=1>cerevisiae</td></tr></table>

\*Genomic DNA Tested

Of the non-BC-GN panel strains tested with BC-GN, 159 demonstrated no cross-reactivity while the organisms listed in the table below were determined to cross-react with BC-GN analytes.

BC-GN will not distinguish E. coli from Shigella spp. including S. dysenteriae, S. flexneri, S. boydii, and S. sonnei. Specimens containing Shigella spp. or E. coli will be reported as “E. coli detected". This information has been included as a limitation in the package insert.

# Cross-Reactive Organisms

<table><tr><td colspan="1" rowspan="1">BC-GN Target for Which Cross ReactivityObserved</td><td colspan="1" rowspan="1">Cross ReactiveOrganism/Resistance Markers</td></tr><tr><td colspan="1" rowspan="2">Citrobacter spp.</td><td colspan="1" rowspan="1">Buttiauxella gaviniae</td></tr><tr><td colspan="1" rowspan="1">Enteric group 137</td></tr><tr><td colspan="1" rowspan="2">Enterobacter spp.</td><td colspan="1" rowspan="1">Klebsiella variicola</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td></tr><tr><td colspan="1" rowspan="5">Escherichia coli*</td><td colspan="1" rowspan="1">Escherichia albertii</td></tr><tr><td colspan="1" rowspan="1">S. dysenteriae</td></tr><tr><td colspan="1" rowspan="1">S. flexneri</td></tr><tr><td colspan="1" rowspan="1">S. boydii</td></tr><tr><td colspan="1" rowspan="1">S. sonnei</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Kluyvera ascorbata</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td></tr><tr><td colspan="1" rowspan="7">CTX-M</td><td colspan="1" rowspan="1">Kluyvera georgiana**</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raftinosus</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td></tr><tr><td colspan="1" rowspan="1">blaKLUA</td></tr><tr><td colspan="1" rowspan="1">blaKLUG</td></tr><tr><td colspan="1" rowspan="1">blaKLUY</td></tr></table>

\*Shigella spp. were tested in inclusivity study but listed here due to cross-reactivity with the $E$ . coli target. \*\*Organism confirmed by bi-directional sequencing to contain CTX-M

# h. Assay cut-off:

For the BC-GN Test, the presence or absence of each target analyte is determined by the mean intensity of target capture spots relative to the Signal Detection Threshold. The capture and mediator oligonucleotides in the BCGN test are designed to eliminate sequence-related cross-reactivity, thereby ensuring that the non- specific target signal intensities at capture spots are similar to the microarray background signal. At cell concentrations of positive culture samples, target hybridization to complementary capture and mediator probes is expected to give signals that are well-separated from negative capture spots. When reading a test slide, multiple images of each array are taken at increasing exposures times and the final target group mean intensity value for an analyte is assigned at the shortest exposure at which the value exceeds the Signal Detection Threshold. If none of the target signals exceeds the threshold for any exposure, the mean spot intensity is evaluated at the longest exposure taken. In order to verify the cutoffs (signal thresholds) for the BC-GN Test, the target mean intensity values were examined for a panel of 18 contrived specimens, representing all of the BC-GN test analytes as well as two additional organisms not detected by the BC-GN test, which served as negative organism controls. In all, 2460 data points were compiled for the threshold evaluation, using logistic fit and ROC statistics.

# i. Fresh versus Frozen Study:

In order to utilize frozen clinical blood culture samples in the clinical study and frozen contrived samples in analytical studies, an analytical study was conducted to demonstrate if multiple cycles of freezing and thawing of specimens will affect BC-GN Test results. The study included multiple freeze/thaw cycles; however all samples tested in the precision, reproducibility and clinical studies were subjected to only one freeze/thaw cycle.

Testing was performed with panel members consisting of eight contrived samples representing all of the BC-GN analytes and two additional organisms, Hafnia alvei and Staphylococcus epidermidis (a common skin contaminant) which are not detected by the BC-GN test and served as negative controls. Organisms were inoculated into blood culture bottles containing $1 0 ~ \mathrm { m L }$ of negative human whole blood and incubated on a blood culture instrument until positivity. Aliquots of each blood culture were stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ for 24 hours before testing with BC-GN. Additional aliquots were subjected to an additional freeze/thaw cycle with storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ for at least 24 hours between cycles. Study results demonstrated the expected positive and negative results for all BC-GN analytes for freshly tested samples as well as for samples that were tested after one or two freeze/thaw cycles and therefore supports the use of frozen specimens in the clinical, precision and reproducibility studies.

Fresh versus Frozen Study Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=1>Count(CFU/mL)</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Frozen/Thawed1x</td><td rowspan=1 colspan=1>Frozen/Thawed2x</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>3.4x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>1.0x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>6.2x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1.5x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>OXA,CTX-M</td><td rowspan=1 colspan=1>3.2x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>5.0x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.5x108</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>7.8x107</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Hafnia Alvei</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.2x107</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3.7x107</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Negative Blood Matrix</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# j. Interference:

Potential inhibitory effects of substances that may be encountered in blood and associated with the blood culturing process were evaluated for potential interference of the BC-GN Test. Samples were prepared by growing organisms in blood culture bottles on a blood culture instrument until bottle positivity followed by spiking each potential interferent into the positive blood culture sample. Control samples containing no interferents were also tested. All samples were tested in triplicate with BC-GN. Testing included the following representative strains: A. baumannii (OXA), C. freundii (VIM), E. cloacae (KPC), E. coli (NDM), K. pneumoniae (OXA, CTX-M), K. oxytoca (CTX-M),

$P$ . mirabilis, and $P$ . aeruginosa (IMP). Each potential interferent and concentration evaluated is provided in the following table. Study results demonstrated no evidence of interference for any of the samples tested.

List of Interfering Substances and Additives   

<table><tr><td rowspan=1 colspan=1>InterferingSubstance/Additive</td><td rowspan=1 colspan=1>ReferenceLevel(adult humanblood)</td><td rowspan=1 colspan=1>TestConcentration</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>1-2 g/L</td><td rowspan=1 colspan=1>14 g/L</td></tr><tr><td rowspan=1 colspan=1>Triglyceride (Intralipid)</td><td rowspan=1 colspan=1>Normal &lt;150 mg/dLHigh 300-500 mg/dLVery high &gt; 500</td><td rowspan=1 colspan=1>3000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Conjugated Bilirubin</td><td rowspan=1 colspan=1>0.1-0.4 mg/dL</td><td rowspan=1 colspan=1>20 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Unconjugated Bilirubin</td><td rowspan=1 colspan=1>0.1-0.8 mg/dL</td><td rowspan=1 colspan=1>20 mg/dL</td></tr><tr><td rowspan=1 colspan=1>γ -Globulin</td><td rowspan=1 colspan=1>0.7-1.7 g/dL</td><td rowspan=1 colspan=1>6 g/dL</td></tr><tr><td rowspan=1 colspan=1>Sodium PolyanetholSulfonate</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.25% w/v</td></tr></table>

# k. Mixed Culture Study (Competitive Inhibition):

A competitive inhibition study was conducted to evaluate the impact of mixed cultures on BC-GN performance. Combinations of eight organisms with resistance markers representing all BC-GN panel targets (for a total of 28 paired combinations) were co-inoculated into individual blood culture bottles at clinically-relevant starting concentrations, and incubated on a blood culture instrument until positivity. BC-GN correctly detected the bacteria and resistance marker(s) for four of the eight target organisms together with their associated resistance marker present in co-inoculated blood culture bottles $\ b { A }$ . baumannii/OXA, C. freundii/VIM, K. pneumoniae/OXA and CTX-M, and $P .$ . mirabilis), demonstrating that these organisms are not subject to competitive inhibition at concentrations expected in routine clinical practice.

For samples containing E. cloacae, E. coli, K. oxytoca, and $P .$ . aeruginosa mixed with other targeted organisms, at least one of the expected bacterial or resistance marker targets was not detected in one or more replicates. Because the study design included growing paired organisms in a blood culture bottle until positivity, it was thought that the cause for false negative results was likely due to differences in growth rates between organisms (i.e., At positivity, the slower growing organism may be present at a concentration below LoD).

A second competitive inhibition study was conducted to further evaluate representative combinations of the four organisms for which at least one false negative result was observed during the first study. In this study, each sample was prepared to contain both organisms at “bottle positivity” concentrations. The following nine organism combinations were evaluated in this second study:

<table><tr><td rowspan=1 colspan=1>Primary Organism</td><td rowspan=1 colspan=1>Secondary Organism</td><td rowspan=1 colspan=1>Combination No.</td></tr><tr><td rowspan=1 colspan=1>E. coli (NDM)</td><td rowspan=1 colspan=1>K. pneumoniae (CTX-M/OXA)</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=4 colspan=1>K. oxytoca (CTX-M)</td><td rowspan=1 colspan=1>A. baumanii (OXA)</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>E. coli (NDM)</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae (CTX-M/OXA)</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=4 colspan=1>P. aeruginosa (IMP)</td><td rowspan=1 colspan=1>A. baumanii (OXA)</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>E. cloacae (KPC)</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>E. coli (NDM)</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca (CTX-M)</td><td rowspan=1 colspan=1>9</td></tr></table>

Eight of the nine organism combinations evaluated in the second study yielded the expected results for all analytes. For one combination ( $K .$ oxytoca/CTX-M and $E$ . coli/NDM), 2/9 replicates yielded false negative results for the $K .$ oxytoca target and correct results (9/9) for the CTX-M, $E .$ . coli, and NDM targets. This second study demonstrated that except for mixed cultures of $K .$ . oxytoca and $E$ . coli, growth rate and not competitive inhibition was the likely contributing factor to observed false negative results in the initial competitive inhibition study. The following limitation is included in the package insert:

· In mixed cultures containing gram-negative bacteria and other organisms, BC-GN may not identify all the detectable organisms in the specimen.

l. Universal Blood Culture Bottle Validation

The performance of BC-GN was evaluated for thirteen 13 types of blood culture media using three different automated blood culture monitoring systems. The following bottle types and representative organisms/resistance markers were evaluated:

<table><tr><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>VersaTREK</td></tr><tr><td rowspan=1 colspan=1>Plus/Aerobic/F</td><td rowspan=1 colspan=1>SA Standard Aerobic</td><td rowspan=1 colspan=1>REDOX 1 EZDraw®Aerobic</td></tr><tr><td rowspan=1 colspan=1>Plus/Anaerobic/F</td><td rowspan=1 colspan=1>FA FAN Aerobic</td><td rowspan=1 colspan=1>REDOX 2 EZDraw Anarobic</td></tr><tr><td rowspan=1 colspan=1>Standard/10 Aerobic/F</td><td rowspan=1 colspan=1>PF Pediatric FAN</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Peds Plus/F</td><td rowspan=1 colspan=1>SN Anaerobic</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Standard Anaerobic/F</td><td rowspan=1 colspan=1>FN FAN® Anaerobic</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lytic/10 Anaerobic/F</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ResistanceMarker(s)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>OXA</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CTX-M &amp; OXA</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>IMP</td></tr><tr><td rowspan=1 colspan=1>None  Negative Blood</td><td rowspan=1 colspan=1>None</td></tr></table>

Bottles were inoculated with organism suspensions into blood culture bottles containing $1 0 ~ \mathrm { m L }$ of negative human whole blood. Bottles were placed on the appropriate culture system until ring positivity. Additionally, a single negative blood culture specimen was created for each bottle type by adding $1 0 ~ \mathrm { m L }$ of negative whole blood and incubating for a minimum of 5 days prior to testing with BC-GN. The study evaluated 300 bottles (12 bottle types x 3 replicate bottles per organism $\textbf { \em X } 8$ organisms $+ \ 1 2$ negative samples).

Upon reaching bottle positivity, an aliquot of each sample was tested in triplicate with BC-GN. To assess specimen stability in the various bottles, one bottle of each organism was then stored for 36 hours at each of three temperature conditions: $2 { - } 8 ~ \mathrm { { } ^ { \circ } C }$ (refrigerated), 20 - $2 5 . 5 \mathrm { { ^ \circ C } }$ (ambient), and $3 4 { - } 3 7 ^ { \circ } \mathrm { C }$ (on the blood culture system), after which BC-GN tests (one test/bottle) were performed.

As expected, the obligate aerobes, A. baumanii and $P$ . aeruginosa, did not grow in any of the anaerobic bottle types and therefore was not evaluated in these bottles. In addition, A. baumanii repeatedly did not grow in one aerobic bottle (VersaTREK REDOX 1 EZ Draw/Aerobic) and therefore the performance of BC-GN with this organism and bottle type is unknown. The following limitation is included in the package insert:

The performance of BC-GN for the detection of A. baumanii in the VersaTREK REDOX 1 EZ Draw/Aerobic bottle is unknown.

All replicates yielded expected positive result for targeted analyes present in the sample with the exception of two A. baumannii samples grown in two separate BACTEC Standard Aerobic/F bottles. For these samples, the Acinetobacter target was detected in all replicates; however the BC-GN Test failed to detect the OXA target in one of three replicates for each bottle. In order to assess if the false negative results for OXA were due to the use of BACTEC Standard Aerobic/F bottles, an additional 10 bottles containing the same strain of A. baumanii were tested. All replicates from this additional testing yielded the expected result for the Acinetobacter and OXA targets. Therefore, it was determined that the cause for the initial two false negative OXA results was not due to the bottle type.

With the exception of the bottle/organism combinations that did not grow, the study results demonstrated that the 13 blood culture bottle types are appropriate for use with BC-GN and that specimens are stable in those bottles at refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ , ambient $( 1 8 - 2 4 ^ { \circ } \mathrm { C } )$ , and culture system (34- $3 7 ^ { \circ } \mathrm { C } )$ storage conditions for up to 36 hours after reaching bottle positivity.

# m. Carryover Study/Cross-Contamination Study:

A study was performed using 12 Verigene $S P$ instruments to assess the potential for carryover/cross-contamination with BC-GN by alternately running “high positive” samples followed by negative samples. Representative strains of A. baumannii (OXA), C. freundii (VIM), C. sedlakii, C. koseri, E. cloacae (KPC), $E .$ . aerogenes, $E .$ . coli (NDM), $K .$ . pneumoniae (CTX-M, OXA), $K .$ . oxytoca, $P .$ . mirabilis, and $P .$ . aeruginosa (IMP) were used to prepare the high positive samples at concentrations at least as high as those found at eight hours past bottle positivity. Six runs of alternating high positive and negative samples were run on each of the 12 Verigene SP instruments for a total of 72 BC-GN test runs.

The study demonstrated that all of the high positive samples yielded the expected “Detected” results for the expected bacterial and resistance marker analytes and “Not Detected” results for the other analytes. In addition, all negative samples gave a ‘Not Detected” result for all analytes. The study demonstrated that the BC-GN Test does not exhibit carryover or crosscontamination that could result in a false positive test result.

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison:

N/A

c. Clinical studies:

A method comparison study was conducted at multiple external clinical study sites to evaluate the performance of BC-GN. Subjects included individuals whose routine care called for blood culture testing and whose blood culture specimens were determined to be positive for growth by an automated blood culturing

system and subsequently determined to contain gram-negative organisms by Gram stain. Specimens were tested with BC-GN as well as sub-cultured to obtain bacterial isolates. BC-GN results for bacterial analytes were compared to standard of care biochemical identification of blood culture isolates. BC-GN results for resistance markers were compared to PCR followed by confirmatory bidirectional sequencing for specimens with positive PCR results.

External control testing was performed daily at each study testing site with positive and negative controls. Testing of external controls included a rotation of nine different positive controls representing all BC-GN analytes. The negative control sample was prepared with $8 { \cdot } 1 0 \mathrm { m L }$ of negative whole blood in BACTEC Aerobic/F bottles and incubating on the blood culture instrument for 48 hours. Positive control samples were prepared with negative whole blood and organisms and were incubated on the blood culture instrument until bottle positivity. The controls were divided into single-use aliquots, stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ and distributed to the testing sites until thawed for testing.

Testing included 604 prospectively collected and tested fresh specimens, 272 prospectively collected and retrospectively tested frozen specimens (all comers), 239 retrospectively tested selected frozen specimens, and 297 simulated frozen specimens.

Simulated specimens were prepared by spiking organisms into blood culture bottles containing $8 { \cdot } 1 0 \mathrm { m L }$ of negative human blood and incubated on an automated blood culture instrument until bottle positivity, after which specimen aliquots were frozen and distributed to clinical testing sites.

There were 1434 evaluable specimens enrolled in the clinical trial; 62 specimens had an initial BC-GN “No Call” result $( 4 . 3 \% )$ and 11 specimens had initial PreAnalytical Errors $( 0 . 8 \% )$ for an initial valid test rate of $9 4 . 9 \%$ . Of the 62 initial No-Calls, 41 yielded a valid test result upon retesting and of the 11 initial PreAEs, 10 yielded a valid test result upon repeat and one was classified as a final Pre-AE. The final “No Call” rate was $1 . 5 \%$ (21/1434 speicmens) and the final Pre-Analysis Error rate was $0 . 8 \%$ (12/1507 tests run) for a total final valid test rate of $9 7 . 7 \%$ . The 22 specimens that did not yield a final valid result were not included in the analysis. Thus, 1412 specimens were analyzed in this clinical evaluation to establish clinical performance of the test.

Summaries of the clinical performance of BC-GN for detection of the 14 targeted organism and resistant markers is summarized below in the following three tables:

Summary of Clinical Test Performance for Acinetobacter spp, Citrobacter spp., Enterobacter spp., and Proteus spp: Comparison to Reference Methods (Culture and Conventional Biochemical and Automated Phenotypic Identification)   

<table><tr><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=1>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr></table>

Summary of Clinical Test Performance for E. coli, K. pneumoniae, K. oxytoca, and P. aeruginosa: Comparison to Reference Methods (Culture and Conventional Biochemical and Automated Phenotypic Identification)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=5 colspan=1>Ee tcc</td><td rowspan=2 colspan=1>eesd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>100%283/283(98.7-100)</td><td rowspan=1 colspan=1>99.1%318/321(97.3-99.8)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>bopesd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>97.1%67/69(89.9-99.7)</td><td rowspan=1 colspan=1>100%535/535(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>99.3%142/143(96.2-100)</td><td rowspan=1 colspan=1>99.2%128/129(95.8-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>91.7%11/12(61.5-99.8)</td><td rowspan=1 colspan=1>100%260/260(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>smese</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>100%42/42(91.6-100)</td><td rowspan=1 colspan=1>99.5%196/197(97.2-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>100%19/19(82.4-100)</td><td rowspan=1 colspan=1>100%220/220(98.3-100)</td></tr><tr><td rowspan=1 colspan=1>smese</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>100%50/50(92.9-100)</td><td rowspan=1 colspan=1>100%247/247(98.5-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>100%27/27(87.2-100)</td><td rowspan=1 colspan=1>100%270/270(98.6-100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1412</td><td rowspan=1 colspan=1>99.8%517/518(98.9-100)</td><td rowspan=1 colspan=1>99.4%889/894f(98.7-99.8)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1412</td><td rowspan=1 colspan=1>97.6%124/127j(93.3-99.5)</td><td rowspan=1 colspan=1>100%1285/1285(99.7-100)</td></tr><tr><td rowspan=5 colspan=1>R peece</td><td rowspan=2 colspan=1>resnd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>95.7%22/23(78.1-99.9)</td><td rowspan=1 colspan=1>98.2%576/581(95.9-99.4)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>boed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>88.0%88/100(80.0-93.6)</td><td rowspan=1 colspan=1>100%504/504(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>100%9/9(66.4-100)</td><td rowspan=1 colspan=1>99.6%262/263(97.9-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>87.0%40/46(73.7-95.1)</td><td rowspan=1 colspan=1>100%226/226(98.4-100)</td></tr><tr><td rowspan=1 colspan=1>Seeee</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>92.6%25/27(75.7-99.1)</td><td rowspan=1 colspan=1>100%212/212(98.3-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sece</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>94.7%36/38(82.3-99.4)</td><td rowspan=1 colspan=1>100%201/201(98.2-100)</td></tr><tr><td rowspan=1 colspan=1>hie</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>60.0%3/5(14.7-94.7)</td><td rowspan=1 colspan=1>100%292/292(98.7-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>sie</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>99.2%121/122(95.5-100)</td><td rowspan=1 colspan=1>100%175/175(97.9-100)</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1412</td><td rowspan=1 colspan=1>92.2%59/64h(82.7-97.4)</td><td rowspan=1 colspan=1>99.6%1342/1348(99.0-99.8)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1412</td><td rowspan=1 colspan=1>93.1%285/306g(89.7-95.7)</td><td rowspan=1 colspan=1>100%1106/1106(99.7-100)</td></tr></table>

<table><tr><td colspan="1" rowspan="1">−e.−</td><td colspan="1" rowspan="1">−−1−</td><td colspan="1" rowspan="1">"Enterobacter"</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E.cloacae</td></tr><tr><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="1">−1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E..coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">. pneumoniae"E. coli"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Shigella spp.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"E. coli"</td><td colspan="1" rowspan="1">K.oxytoca</td><td colspan="1" rowspan="1">E. coli</td></tr><tr><td colspan="1" rowspan="1">−1</td><td colspan="1" rowspan="1">K. oxytoca""E. coli"</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. oxytoca and E. coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Enterobacter"E. coli"</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">Shigella spp.</td></tr><tr><td colspan="1" rowspan="14">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1"> K. pneumoniae</td><td colspan="1" rowspan="1">Unable to differentiate: Similar match to both K.variicola and K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K¯ pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Klebsiella spp.</td></tr><tr><td></td><td colspan="1" rowspan="3">"Not Detected"</td><td colspan="1" rowspan="3">K. pneumoniae</td><td colspan="1" rowspan="2">Unable to differentiate: Similar match to both K.</td></tr><tr><td colspan="1" rowspan="7"></td></tr><tr><td colspan="1" rowspan="1">variicola and K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1"> "Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">E"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1"> "Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Unable to differentiate: Similar match to both K.variicola and K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1"> "Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1"> "Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. varicola</td></tr><tr><td colspan="1" rowspan="1">"Not Detected"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"E. coli"</td><td colspan="1" rowspan="1">E. coli, K. pneumoniae and S.dysgalactiae</td><td colspan="1" rowspan="1">K. pneumoniae (E. coli not tested</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC"</td><td colspan="1" rowspan="1">K. pneumoniae and KPC</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">−1</td><td colspan="1" rowspan="1">E. coli"</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Klebsiella spp.</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Not Detected"</td><td colspan="1" rowspan="1">K.oxytoca</td><td></td></tr><tr><td colspan="1" rowspan="1">−1</td><td colspan="1" rowspan="1">Not Detected"</td><td colspan="1" rowspan="1">K.oxytoca</td><td colspan="1" rowspan="1">K. variicola</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">E. coli""CTX-M"</td><td colspan="1" rowspan="1">K. oxytoca, E. coli, CTX-MGp. 1</td><td colspan="1" rowspan="1">K. oxytoca and E. coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E. coli"</td><td colspan="1" rowspan="1">K oxytoca and E_ coli</td><td colspan="1" rowspan="1">K._oxytoca and E_ coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">"KPC"</td><td colspan="1" rowspan="1">K. oxytoca and KPC</td><td colspan="1" rowspan="1">K.oxytoca</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">K. oxytoca"</td><td colspan="1" rowspan="1">R.planticola</td><td colspan="1" rowspan="1">R.planticola</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">R.planticola</td><td colspan="1" rowspan="1">R. planticola</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">K. oxytoca"</td><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">R. planticola</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">K. oxytoca""K. oxytoca</td><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">R. planticola</td></tr><tr><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">R. planticola</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">. pneumoniae""K. oxytoca"</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="3">j</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">K. pneumoniae"CTX-M"</td><td colspan="1" rowspan="1">P.aeruginosa, K. pneumoniae andCTX-M</td><td colspan="1" rowspan="1">P. aeruginosa (K. pneumoniae not tested)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">P.aeruginosa and K. pneumoniae</td><td colspan="1" rowspan="1">P.aeruginosa</td></tr><tr><td colspan="2" rowspan="1">"E. coli"</td><td colspan="1" rowspan="1">P.aeruginosa and E. coli</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr></table>

Summary of Clinical Test Performance for Resistance Marker Targets CTX-M, OXA, KPC, VIM, NDM, and IMP: Comparison to Reference Methods (PCR Amplification/Bi Directional Sequencing)   

<table><tr><td colspan="1" rowspan="2">−</td><td colspan="2" rowspan="2">SpecimenType</td><td colspan="1" rowspan="2">n=</td><td colspan="2" rowspan="1">% Agreement (95% CI)</td><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="2">Specimen Type</td><td colspan="1" rowspan="2">n=</td><td colspan="2" rowspan="1">% Agreement (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="2">reedsnd</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1">100 %5/5(47.8-100)</td><td colspan="1" rowspan="1">100%532/532(99.3-100)</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">oeesd</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1">97.5%39/40(86.8-99.9)</td><td colspan="1" rowspan="1">100%497/497(99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%236/236(98.5-100)</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1">91.2%11/12(61.5-99.8)</td><td colspan="1" rowspan="1">100%224/224(98.4-100)</td></tr><tr><td colspan="1" rowspan="1">Seceee</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">50.0%2/4(6.8-93.2)</td><td colspan="1" rowspan="1">100%202/202(98.2-100)</td><td colspan="1" rowspan="1">W-XXO</td><td colspan="1" rowspan="1">Sececee</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%203/203(98.2-100)</td></tr><tr><td colspan="1" rowspan="1">Sie</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">98.3%54/55(90.8-100)</td><td colspan="1" rowspan="1">99.6%231/232(97.6-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Sie</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">100%98/98(96.3-100)</td><td colspan="1" rowspan="1">99.5%188/189(97.1-100)</td></tr><tr><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">95.3%61/64m(86.9-99.0)</td><td colspan="1" rowspan="1">99.9%1201/1202"(99.5-100)</td><td colspan="1" rowspan="1">′</td><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">98.7%151/153 k(95.4-99.8)</td><td colspan="1" rowspan="1">99.9%1112/1113(99.5-100)</td></tr><tr><td colspan="1" rowspan="2">M</td><td colspan="1" rowspan="2">oeed</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%535/535(99.3-100)</td><td colspan="1" rowspan="2">M</td><td colspan="1" rowspan="2">oeend</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%536/536(99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%235/235(98.5-100)</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100%236/236(98.5-100)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Seceee</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%206/206(98.2-100)</td><td colspan="1" rowspan="1">′</td><td colspan="1" rowspan="1">Seccce</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%206/206(98.2-100)</td></tr><tr><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">100%48/48(92.6-100)</td><td colspan="1" rowspan="1">100%239/239(98.5-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">he</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">100%40/40(91.2-100)</td><td colspan="1" rowspan="1">100%247/247(98.5-100)</td></tr><tr><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">100%51/51(93.1-100)</td><td colspan="1" rowspan="1">100%1215/1215(99.7-100)</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">100%41/41(91.4-100)</td><td colspan="1" rowspan="1">100%1225/1225(99.7-100)</td></tr><tr><td colspan="1" rowspan="5">A</td><td colspan="1" rowspan="2">poedsod</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%537/537(99.3-100)</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">oeed</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">537</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%537/537(99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%236/236(98.5-100)</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%236/236(98.5-100)</td></tr><tr><td colspan="1" rowspan="1">secccd</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%206/206(98. 2-100)</td><td colspan="1" rowspan="1">M</td><td colspan="1" rowspan="1">Secd</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%206/206(98.2-100)</td></tr><tr><td colspan="1" rowspan="1">tie</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">100%48/48(92.6-100)</td><td colspan="1" rowspan="1">100%239/239(98.5-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pid</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">100%41/41(91.4-100)</td><td colspan="1" rowspan="1">100%246/246(98.5-100)</td></tr><tr><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">100%48/48(92.6-100)</td><td colspan="1" rowspan="1">100%1218/1218(99.7-100)</td><td colspan="1" rowspan="1">b</td><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">1266</td><td colspan="1" rowspan="1">100%41/41(91.4-100)</td><td colspan="1" rowspan="1">100%1225/1225(99.7-100)</td></tr></table>

No. Identified by BC-GN test as: Identified by Reference Method(s) as: k. “E. coli” “E. coli” and “Enterobacter” Escherichia coli and CTX-M Gp1Escherichia coli and CTX-M Gp1 “Citrobacter”, “CTX-M” and “OXA” C. braakii and OXA Gp48 m. “Acinetobacter” “Acinetobacter” Acinetobacter baumannii and OXA Gp23 Acinetobacter radioresistans and OXA Gp23 n. “Acinetobacter” and “OXA” Acinetobacter baumannii

The following table lists 22 specimens that yielded positive BC-GN results for more than one organism. Two organisms were detected in 21 specimens and three organisms were detected in one specimen. Nine of the 22 specimens yielded discordant results for BC-GN as compared to the reference method (positive for a BC-GN analyte that was not detected by the reference method).

Clinical Specimens with Multiple Organisms Detected by BC-GN   

<table><tr><td colspan="4" rowspan="1">Multiple Organism Combinations Detected by BC-GN 1</td><td colspan="3" rowspan="1">Reference Test</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">ResistanceMarker</td><td colspan="1" rowspan="1">TotalSpecimens</td><td colspan="1" rowspan="1">DiscrepantSpecimens</td><td colspan="1" rowspan="1">DiscrepantAnalyte(s)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Acinetobacterspp.</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterspp.</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterspp.</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">Enterobacter spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Escherichiacoli, CTX-M</td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">E. asburaieEnterobacterspp.</td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IMP/VIM</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobacterspp.</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Escherichiacoli</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Escherichiacoli</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. oxytoca,Enterobactercloacaecomplex</td></tr><tr><td colspan="2" rowspan="1">TOTAL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1"></td></tr></table>

1 Defined as an analyte that was detected by the BC-GN test, but not detected by the reference methods.

The following table lists 35 clinical study specimens that were positive with multiple organisms by the reference culture method. Two organisms were isolated in 29 specimens and three organisms were isolated in six specimens. Nine of the 35 specimens yielded discordant results (positive by reference culture or PCR/bidirectional sequencing and negative by BC-GN).

Clinical Mixed Specimen Combinations Detected by Reference Methods   

<table><tr><td colspan="4" rowspan="1">Multiple Organism Combinations by Reference Test1</td><td colspan="3" rowspan="1">Detected by BC-GN</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">ResistanceMarker</td><td colspan="1" rowspan="1">TotalSpecimens</td><td colspan="1" rowspan="1">DiscrepantSpecimens</td><td colspan="1" rowspan="1">DiscrepantAnalyte(s)1</td></tr><tr><td colspan="1" rowspan="1">A. baumanniicomplex</td><td colspan="1" rowspan="1">Enterococcusspp</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacaecomplex</td><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterobactercloacaecomplex</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Streptococcusdysgalactiae</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Proteusmirabilis</td><td colspan="1" rowspan="1">Sphingomonaspaucimobilis</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Pseudomonasstutzeri</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Pseudomonasputida</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumanniicomplex</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aeromonascaviae</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Aeromonashydrophila</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium spp.</td><td colspan="1" rowspan="1">Sphingomonaspaucimobilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cupriaviduspauculus</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Delftiaacidovorans</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Enterococcusspp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Pantoea spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CTX-M</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. oxytoca(2)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NONE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasputida</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Sphingomonaspaucimobilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">Achromobacterxylosoxidans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratiaodorifera</td><td colspan="1" rowspan="1">Pantoea spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">TOTAL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">canmaGYellowb</td></tr></table>

Defined as an analyte that was detected by the reference methods which should have been detected by BC-GN but was

BC-GN reports the genus only for Acinetobacter spp., Citrobacter spp, Enterobacter spp, Proteus spp. The following table includes the performance of BC-GN (positive percent agreement) for each genus stratified by species for all specimens tested in the clinical and analytical (inclusivity) studies. The clinical study data includes all clinical study specimens combined (prospective, archived, and simulated.)

Summary of Genus/Group-level Test Performance versus Reference Method(s) – Stratified by Species.   

<table><tr><td colspan="4" rowspan="1">Acinetobacter Genus</td><td colspan="4" rowspan="1">Citrobacter Genus</td><td colspan="4" rowspan="1">Enterobacter Genus</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="1">Clinical</td><td colspan="2" rowspan="1">Analytical (2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Clinica1</td><td colspan="2" rowspan="1">Analytical (2)</td><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="1">Clinical</td><td colspan="2" rowspan="1">Analytical (2)</td></tr><tr><td colspan="2" rowspan="1">% POSAgreement(95% CI)</td><td colspan="1" rowspan="1">No.ofStrains(3)</td><td colspan="1" rowspan="1">Organism</td><td colspan="2" rowspan="1">% POS Agreement(95% CI)</td><td colspan="1" rowspan="1">No.ofStrains(3)</td><td colspan="2" rowspan="1">% POS Agreement(95% CI)</td><td colspan="1" rowspan="1">No.ofStrains(3)</td></tr><tr><td colspan="1" rowspan="1">CombinedAcinetobacter</td><td colspan="1" rowspan="1">98.2%5/56(90.5-10)</td><td colspan="1" rowspan="1">96.3%104/108(90.8-99.0)</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">CombinedCitrobacter</td><td colspan="1" rowspan="1">100%9/49(92.8-100)</td><td colspan="1" rowspan="1">100%123/123(97.1-100)</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">CombinedEnterobacter</td><td colspan="1" rowspan="1">97.5%119/122(93.0-99.5</td><td colspan="1" rowspan="1">100%87/87(95.9-100)</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterspecies</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">NT(1)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">amalonaticus</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">cloacae</td><td colspan="1" rowspan="1">96.3%26/27(81.0-99.9)</td><td colspan="1" rowspan="1">100%24/24(85.8-100)</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">baylyi</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%6/6(54.1-100)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">braakii</td><td colspan="1" rowspan="1">100%10/10(69.2-100)</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">cloacaecomplex</td><td colspan="1" rowspan="1">100%74/76(90.8-99.7)</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">lwoffii</td><td colspan="1" rowspan="1">100%5/5(47.8-10</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">freundii</td><td colspan="1" rowspan="1">100%22/22(84.5-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">aerogenes</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">100%30/30(88.4-100)</td><td colspan="1" rowspan="1">100%24/24(85.8-100)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">koseri</td><td colspan="1" rowspan="1">100%12/12(73.5-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">asburaie</td><td colspan="1" rowspan="1">100%4/4(40.0-100)</td><td colspan="1" rowspan="1">100%112/12(73.5-100)</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">baumanniicomplex</td><td colspan="1" rowspan="1">93.3%14/15(68.1-99.8)</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">youngae</td><td colspan="1" rowspan="1">2/2100%(15.8-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">amnigenus</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ursingii</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">farmeri</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%6/6(54.1-100)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">cancerogenus</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">bereziniae</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">gillenii</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">hormaechei</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">calcoaceticus</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">murliniae</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ludwigii</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">guillouiae</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">rodentium</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">nimipressuralis oryzae</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">haemolyticus</td><td colspan="1" rowspan="2">NT</td><td colspan="1" rowspan="2">100%15/15(78.2-100)</td><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="2">sedlakii</td><td colspan="1" rowspan="2">NT</td><td colspan="1" rowspan="2">100%9/9(66.4-100)</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Proteus Genus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CombinedProteus</td><td colspan="1" rowspan="1">100%58/58(93.8-100)</td><td colspan="1" rowspan="1">100%48/48(92.6-100)</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">johnsonii</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">werkmanii</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">mirabilis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9(66.4-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9(66.4-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">58/58(93.8-100)</td><td colspan="1" rowspan="1">15/15(78.2-100)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">junii</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">3</td><td colspan="4" rowspan="4"></td><td colspan="1" rowspan="1">hauseri</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%6/6(54.1-100)</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">schindleri</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">myxofaciens</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">radioresistenS</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">55.6%5/9(21.2-86.3)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">penneri</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%6/6(54.1-100)</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">vulgaris</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">5</td></tr></table>

(1) “Not Tested” (2) Analytical Reactivity (Inclusivity) (3) Each strain was tested in triplicate in the Inclusivity study.

The performance of BC-GN for detection of CTX-M, OXA, KPC, VIM, NDM, and IMP stratified by organism is provided in the following two tables. The data presented includes all clinical study specimens combined (prospective, archived, and simulated.)

Detection of CTX-M, OXA, and KPC Resistance Markers Stratified by Organism, as determined by reference method (PCR/Bi-directional sequencing)   

<table><tr><td colspan="3" rowspan="1">CTX-M</td><td colspan="2" rowspan="1">OXA</td><td colspan="2" rowspan="1">KPC</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Percent Agreement</td><td colspan="2" rowspan="1">Percent Agreement</td><td colspan="2" rowspan="1">Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">Positive(95% CI)</td><td colspan="1" rowspan="1">Negative(95% CI)</td><td colspan="1" rowspan="1">Positive(95% CI)</td><td colspan="1" rowspan="1">Negative(95% CI)</td><td colspan="1" rowspan="1">Positive(95% CI)</td><td colspan="1" rowspan="1">Negative(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%52/52(93.2-100)</td><td colspan="1" rowspan="1">85.0%17/20(c(62.1-96.8)</td><td colspan="1" rowspan="1">96.9%31/32(83.8-99.9)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%52/52(93.21-100)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">98.0%49/50(89.4-100)</td><td colspan="1" rowspan="1">100%1/1(d)(2.5-100)</td><td colspan="1" rowspan="1">100%49/49(92.8-100)</td><td colspan="1" rowspan="1">100%1/1 ()(2.5-100)</td><td colspan="1" rowspan="1">100%49/49(92.8-100)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">100%9/9 (a)(66.4-100)</td><td colspan="1" rowspan="1">100%111/111(96.7-100)</td><td colspan="1" rowspan="1">100%2/2()(15.8-100)</td><td colspan="1" rowspan="1">100%118/118(96.9-100)</td><td colspan="1" rowspan="1">100%1/1(h)(2.5-100)</td><td colspan="1" rowspan="1">100%119/119(97.0-100)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.8%85/86(93.7-100)</td><td colspan="1" rowspan="1">100%424/424(99.1-100)</td><td colspan="1" rowspan="1">100%16/16(79.4-100)</td><td colspan="1" rowspan="1">100%494/494(99.3-100)</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%508/508(99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%56/56(93.6-100)</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">100%56/56(93.6-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%56/56(93.6-100)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%124/124(97.1-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%124/124(97.1-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%123/123(97.1-100)</td></tr><tr><td colspan="1" rowspan="1">K. oyxtoca</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%59/59(93.9-100)</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%58/58(93.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%59/59(93.9-100)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">100%217/217</td><td colspan="1" rowspan="1">100%22/22</td><td colspan="1" rowspan="1">100%251/251</td><td colspan="1" rowspan="1">100%45/45</td><td colspan="1" rowspan="1">100%228/228</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(93.6-100)</td><td colspan="1" rowspan="1">(98.3-100)</td><td colspan="1" rowspan="1">(84.6-100)</td><td colspan="1" rowspan="1">(98.5-100)</td><td colspan="1" rowspan="1">(92.1-100)</td><td colspan="1" rowspan="1">(98.4-100)</td></tr><tr><td colspan="1" rowspan="1">Polymicrobialsamples</td><td colspan="1" rowspan="1">0%0/1((0-97.5)</td><td colspan="1" rowspan="1">100%20/20(83.2-100)</td><td colspan="1" rowspan="1">0%1/1()(2.5-100)</td><td colspan="1" rowspan="1">100%20/20(83.2-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%21/21(83.9-100)</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">98.7%151/153(95.4-99.8)</td><td colspan="1" rowspan="1">99.9%1112/1113(99.5-100)</td><td colspan="1" rowspan="1">95.3%61/64(86.9-99.0)</td><td colspan="1" rowspan="1">99.9%1201/1202(99.5-100)</td><td colspan="1" rowspan="1">100%51/51(93.1-100)</td><td colspan="1" rowspan="1">100%1215/1215(99.7-100)</td></tr></table>

a. Eight (8) Enterobacter cloacae complex and one (1) Enterobacter cloacae b. One (1) Escherichia coli and Enterobacter aerogenes c. Eleven (11) Acinetobacter baumannii and six (6) Acinetobacter baumannii complex and three (3) Acinetobacter radioresistens d. One (1) Citrobacter braakii e. Two (2) Enterobacter cloacae f. One (1) Escherichia coli and Acinetobacter baumannii complex and Enterococcus spp. g. One (1) Citrobacter freundii h. One (1) Enterobacter cloacae complex

Detection of VIM, NDM, and IMP Resistance Markers Stratified by Organism, as determined by reference method (PCR/Bi-directional sequencing)   

<table><tr><td rowspan=1 colspan=3>VIM</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>IMP</td></tr><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=2>Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>100%52/52(93.2-100)</td><td rowspan=1 colspan=1>100%1/1 d)(2.5-100)</td><td rowspan=1 colspan=1>100%51/51(93.0-100)</td><td rowspan=1 colspan=1>100%5/5 ()(47.8-100)</td><td rowspan=1 colspan=1>100%47/47(92.5-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>100%3/3 (a)(29.2-100)</td><td rowspan=1 colspan=1>100%47/47(92.5-100)</td><td rowspan=1 colspan=1>100%2/2 (e)(15.8-100)</td><td rowspan=1 colspan=1>100%48/48(92.6-100)</td><td rowspan=1 colspan=1>100%1/1 (h)(2.5-100)</td><td rowspan=1 colspan=1>100%49/49(92.8-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>100%10/10 ((69.2-100)</td><td rowspan=1 colspan=1>100%110/110(96.7-100)</td><td rowspan=1 colspan=1>100%6/6()(54.7-100)</td><td rowspan=1 colspan=1>100%114/114(96.8100)</td><td rowspan=1 colspan=1>100%6/6 ()(54.1-100)</td><td rowspan=1 colspan=1>100%114/114(96.8-100)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%509/509(99.3-100)</td><td rowspan=1 colspan=1>100%15/15(78.2-100)</td><td rowspan=1 colspan=1>100%495/495(99.3-100)</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%509/509(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%56/56(93.6-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%56/56(93.6100)</td><td rowspan=1 colspan=1>100%2/2(15.8-100)</td><td rowspan=1 colspan=1>100%54/54(93.4-100)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>100%2/2(15.8-100)</td><td rowspan=1 colspan=1>100%122/122(97.0-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%124/124(97.1-100)</td><td rowspan=1 colspan=1>100%22/22(84.6-100)</td><td rowspan=1 colspan=1>100%102/102(96.5-100)</td></tr><tr><td rowspan=1 colspan=1>K. oyxtoca</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%60/60(94.0-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%60/60(94.0-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%60/60(94.0-100)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>100%24/24(85.8-100)</td><td rowspan=1 colspan=1>100%249/249(98.5-100)</td><td rowspan=1 colspan=1>100%17/17(80.5-100)</td><td rowspan=1 colspan=1>100%256/256(98.6-100)</td><td rowspan=1 colspan=1>100%10/10(69.2-100)</td><td rowspan=1 colspan=1>100%263/263(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Polymicrobialsamples</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>100%20/20(83.2-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%21/21(83.9-100)</td><td rowspan=1 colspan=1>0%1/1 (k)(2.5-100)</td><td rowspan=1 colspan=1>100%20/20(83.2-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>100%41/41(91.4-100)</td><td rowspan=1 colspan=1>100%1225/1225(99.7-100)</td><td rowspan=1 colspan=1>100%41/41(91.4-100)</td><td rowspan=1 colspan=1>100%1225/1225(99.7-100)</td><td rowspan=1 colspan=1>100%48/48(92.6-100)</td><td rowspan=1 colspan=1>100%1218/1218(99.7-100)</td></tr></table>

The following table includes a listing of organisms detected in clinical study specimens that contained single or dual resistance markers. The data presented includes all clinical study specimens combined (prospective, archived, and simulated.)

Summary of Organisms Containing Single and Dual Resistance Markers   

<table><tr><td rowspan=1 colspan=2>BC-GN ResistanceMarker Target</td><td rowspan=1 colspan=1>No. ofStrainsTested</td><td rowspan=1 colspan=1>Species Containing the Resistance Marker</td></tr><tr><td rowspan=7 colspan=1>Presence ofa SingleResistanceMarker</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>Enterobacter cloacae (3), Escherichia coli (56), Klebsiellapneumoniae (11)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Acinetobacter baumannii (12), Acinetobacter radioresistens(2)Escherichia coli (3), Klebsiella oxytoca (1), Klebsiella pneumoniae(2),</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (4), Proteus mirabilis (2), Pseudomonasaeruginosa (22), Citrobacter braakii (1), Acinetobacter Iwoffi (1),Acinetobacter baumannii (1), Enterobacter cloacae (5),</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Citrobacter freundii (3), Enterobacter cloacae (9), Klebsiellapneumoniae (19), Pseudomonas aeruginosa (2)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>Enterobacter cloacae (1), Klebsiella oxytoca (1), Citrobacter freundii(l), Klebsiella pneumoniae (39),Pseudomonas aeruginosa (1)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Escherichia coli (2), Citrobacter freundii (2), Enterobacter cloacae(2), Klebsiella pneumoniae (3)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>211(70%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=9 colspan=1>Presence ofDualResistanceMarkers</td><td rowspan=1 colspan=1>NDM/CTX-M</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Enterobacter cloacae (3), Escherichia coli (12), Klebsiellapneumoniae (14)</td></tr><tr><td rowspan=1 colspan=1>IMP/CTX-M</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Enterobacter cloacae (1), Escherichia coli (1), Klebsiellapneumoniae (6)</td></tr><tr><td rowspan=1 colspan=1>OXA/CTX-M</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Escherichia coli (13), Klebsiella pneumoniae (20), Klebsiella oxytoca(1), Enterobacter cloacae (1), Citrobacter braakii (I),</td></tr><tr><td rowspan=1 colspan=1>KPC/CTX-M</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Escherichia coli (2), Klebsiella pneumoniae (3)</td></tr><tr><td rowspan=1 colspan=1>VIM/CTX-M</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Escherichia coli (1), Enterobacter cloacae (1), Klebsiellapneumoniae (2)</td></tr><tr><td rowspan=1 colspan=1>VIM/KPC</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>OXA/IMP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td></tr><tr><td rowspan=1 colspan=1>OXA/NDM</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Acinetobacter baumannii (1), Enterobacter cloacae (1)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>90(30%)</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=2>GRAND TOTAL</td><td rowspan=1 colspan=1>301</td></tr></table>

For informational purposes, the performance of BC-GN was evaluated for detection of KPC, OXA, NDM, VIM, and IMP as compared to phenotypic antimicrobial susceptibility testing (AST) results. Meropenem agar gradient diffusion was performed for 993 specimens of the 1266 specimen dataset used for the resistance marker performance tables presented above. MIC results were interpreted using current CLSI breakpoints.

The percent positive agreement (PPA) for this comparison was calculated as $100 \%$ $\mathrm { \nabla { \mit \ x } \left( T P / T P + F N \right) }$ . A true positive (TP) is positive by BC-GN for KPC, OXA, NDM, VIM, and/or IMP and resistant (R) or intermediate (I) to Meropenem. A false negative (FN) is negative by BC-GN for KPC, OXA, NDM, VIM, and/or IMP and resistant (R) or intermediate (I) to Meropenem. The negative percent agreement (NPA) for this comparison was calculated as $1 0 0 \% \mathbf { x }$ $( \mathrm { T N } / \mathrm { T N } + \mathrm { F P } )$ . A true negative (TN) is negative by BC-GN for KPC, OXA, NDM, VIM, and/or IMP and susceptible (S) to Meropenem. A false positive (FP) is positive by BC-GN for KPC, OXA, NDM, VIM, and/or IMP and susceptible (S) to Meropenem.

Results stratified by organism and resistance marker are presented in the following table. It is noted that “false negative” results can occur when carbapenem resistance is caused by mechanisms other than by the presence of the five targeted resistance genes. In addition, it is noted that “false positive results” can occur because the presence of a resistance marker may not always infer resistance to meropenem. The following limitations/statements are included in the package insert.

Carbapenem resistance in the organisms detected by BC-GN can be due to mechanisms other than acquisition of the KPC (blaIMP), OXA (blaIMP), NDM (blaIMP), VIM (blaIMP), or IMP (blaIMP) gene(s).   
Detecton of KPC, OXA, NDM, VIM, or IMP resistance markers may not always infer resistance to carbapenems.

BC-GN Performance for KPC, OXA, NDM, VIM, and IMP and all Resistance Markers Combined as Compared to Phenotypic Antimicrobial Susceptibility Testing using Meropenem Agar Gradient Diffusion   

<table><tr><td colspan="1" rowspan="3">BC-GN PanelAnalyteDetected</td><td colspan="1" rowspan="3">n=</td><td colspan="12" rowspan="1">Positive and Negative Percent Agreement (PPA and NPA Detection of Resistance Markers VersusMeropenem Agar Gradient Diffusion (Resistant (R) or Intermediate (I))</td></tr><tr><td colspan="2" rowspan="1">KPC</td><td colspan="2" rowspan="1">OXA</td><td colspan="2" rowspan="1">NDM</td><td colspan="2" rowspan="1">VIM</td><td colspan="2" rowspan="1">IMP</td><td colspan="2" rowspan="1">"Combined Result" DataAnalysis Algorithm (KPC,OXA, NDM, VIM, and/orIMP detected)</td></tr><tr><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">PPA(a)</td><td colspan="1" rowspan="1">NPA(b)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterspp.</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0%0/8</td><td colspan="1" rowspan="1">100%200/20</td><td colspan="1" rowspan="1">100%8/8</td><td colspan="1" rowspan="1">90.0%18/20</td><td colspan="1" rowspan="1">12.5%1/8</td><td colspan="1" rowspan="1">100%20/20</td><td colspan="1" rowspan="1">0%0/8</td><td colspan="1" rowspan="1">100%20/20</td><td colspan="1" rowspan="1">37.5%3/8</td><td colspan="1" rowspan="1">100%20/20</td><td colspan="1" rowspan="1">100%8/8(63.1-100)</td><td colspan="1" rowspan="1">90.0%18/20(68.3-98.8)</td></tr><tr><td colspan="1" rowspan="1">Citrobacterspp.</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">33.3%1/3</td><td colspan="1" rowspan="1">100%28/28</td><td colspan="1" rowspan="1">0%0/3</td><td colspan="1" rowspan="1">100%28/28</td><td colspan="1" rowspan="1">33.3%1/3</td><td colspan="1" rowspan="1">100%28/28</td><td colspan="1" rowspan="1">33.3%1/3</td><td colspan="1" rowspan="1">92.9%26/28</td><td colspan="1" rowspan="1">0%0/3</td><td colspan="1" rowspan="1">100%28/28</td><td colspan="1" rowspan="1">100%3/3</td><td colspan="1" rowspan="1">92.9%26/28</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(29.2-100)</td><td colspan="1" rowspan="1">(76.5-99.1)</td></tr><tr><td colspan="1" rowspan="1">Enterobacterspp.</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">0%0/5</td><td colspan="1" rowspan="1">100%82/82</td><td colspan="1" rowspan="1">0%0/5</td><td colspan="1" rowspan="1">100%82/82</td><td colspan="1" rowspan="1">40.0%2/5</td><td colspan="1" rowspan="1">100%82/82</td><td colspan="1" rowspan="1">60.0%3/5</td><td colspan="1" rowspan="1">96.3%79/82</td><td colspan="1" rowspan="1">0%0/5</td><td colspan="1" rowspan="1">96.3%79/82</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td><td colspan="1" rowspan="1">97.7%76/82(84.8-97.3)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%48/48</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%48/48</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%48/48</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%48/48</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%48/48</td><td colspan="1" rowspan="1">0%0/1(0-75.5)</td><td colspan="1" rowspan="1">100%48/48(92.6-100)</td></tr><tr><td colspan="1" rowspan="1">Escherichiacoli</td><td colspan="1" rowspan="1">435</td><td colspan="1" rowspan="1">3.7%1/27</td><td colspan="1" rowspan="1">99.8%407/408</td><td colspan="1" rowspan="1">40.7%11/27</td><td colspan="1" rowspan="1">99.5%406/408</td><td colspan="1" rowspan="1">51.9%14/27</td><td colspan="1" rowspan="1">100%408/408</td><td colspan="1" rowspan="1">0%0/27</td><td colspan="1" rowspan="1">100%408/408</td><td colspan="1" rowspan="1">0%0/27</td><td colspan="1" rowspan="1">100%408/408</td><td colspan="1" rowspan="1">96.3%26/27(81.0-99.9)</td><td colspan="1" rowspan="1">99.3%405/408(97.9-99.9)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">225</td><td colspan="1" rowspan="1">47.4%36/76</td><td colspan="1" rowspan="1">98.7%147/149</td><td colspan="1" rowspan="1">5.3%4/76</td><td colspan="1" rowspan="1">97.3%145/149</td><td colspan="1" rowspan="1">19.7%15/76</td><td colspan="1" rowspan="1">100%149/149</td><td colspan="1" rowspan="1">28.3%20/76</td><td colspan="1" rowspan="1">99.3%148/149</td><td colspan="1" rowspan="1">4.0%93/76</td><td colspan="1" rowspan="1">97.9%146/149</td><td colspan="1" rowspan="1">98.7%75/76(92.9-99.9)</td><td colspan="1" rowspan="1">93.3%139/149(88.0-96.7)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">100%1/1</td><td colspan="1" rowspan="1">100%49/49</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%49/49</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%49/49</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%49/49</td><td colspan="1" rowspan="1">0%0/1</td><td colspan="1" rowspan="1">100%49/49</td><td colspan="1" rowspan="1">1000%1/1(2.5-100)</td><td colspan="1" rowspan="1">100%49/49(92.8-100)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">3.1%1/32</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">0%0/32</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">0%0/32</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">6.3%2/32</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">3.1%1/32</td><td colspan="1" rowspan="1">100%56/56</td><td colspan="1" rowspan="1">12.5%4/32(3.5-29.0)</td><td colspan="1" rowspan="1">100% 6/56(93.6-100)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">993</td><td colspan="1" rowspan="1">26.1%40/153</td><td colspan="1" rowspan="1">99.6%837/840</td><td colspan="1" rowspan="1">15.0%23/153</td><td colspan="1" rowspan="1">99.1%832/840</td><td colspan="1" rowspan="1">21.6%33/153</td><td colspan="1" rowspan="1">100%840/840</td><td colspan="1" rowspan="1">17.0%26/153</td><td colspan="1" rowspan="1">99.3%834/840</td><td colspan="1" rowspan="1">4.6%7/153</td><td colspan="1" rowspan="1">99.3%834/840</td><td colspan="1" rowspan="1">79.7%122/153(71.5-85.8)</td><td colspan="1" rowspan="1">97.1%816/840(95.8-98.2)</td></tr></table>

a) Carbepenem resistance in the organisms detected by BC-GN can be due to mechanisms other than acquisition of the KPC $( \mathrm { b l a } _ { \mathrm { K P C } } )$ , OXA $\mathrm { ( b l a _ { O X A } ) }$ , NDM $\left( \mathsf { b l a } _ { \mathrm { N D M } } \right)$ ), VIM $\left( \mathsf { b l a } _ { \mathsf { V I M } } \right)$ , or IMP $\bf ( b l a _ { I M P } )$ gene(s). b) Detection of KPC, OXA, NDM, VIM, or IMP resistance markers may not always infer resistance to carbapenems.

For informational purposes, the performance of BC-GN was evaluated for detection of CTX-M as compared to phenotypic antimicrobial susceptibility testing (AST) results. Agar gradient diffusion using ceftazadime and ceftriaxone was performed for 993 specimens of the 1266 specimen dataset used for the performance table presented above for the CTX-M target. MIC results were interpreted using current CLSI breakpoints.

The percent positive agreement (PPA) for this comparison was calculated as $100 \%$ x $\left( \mathrm { T P / T P + F N } \right)$ . A true positive (TP) is positive by BC-GN for CTX-M and resistant/intermediate to Ceftazadime only or resistant/intermediate to Ceftazadime and/or Ceftriaxone. A false negative (FN) is negative by BC-GN for CTX-M and resistant/intermediate to Ceftazadime only or resistant/intermediate to Ceftazadime and/or Ceftriaxone. The negative percent agreement (NPA) for this comparison was calculated as $100 \% \mathbf { x }$ $\mathrm { T N } / \mathrm { T N } + \mathrm { F P } )$ . A true negative (TN) is negative by BCGN for CTX-M and susceptible to Ceftazadime only or susceptible to Ceftazidime and Ceftriaxone. A false positive (FP) is positive by BC-GN for CTX-M and susceptible to Ceftazadime only or susceptible to Ceftazidime and Ceftriaxone.

Results for detection of the CTX-M resistance marker as compared to ASTf ceftazidime and/or ceftriaxone are provided in the table below. It is noted that “false negative” results can occur when resistance is caused by other mechanisms than by the presence of the CTX-M gene. In addition, it is noted that “false positive results” can occur because the presence of CTX-M may not always infer resistance to ceftazidime or ceftriaxone. The following limitation/statements are included in the package insert.

· Cephalosporin resistance in organisms detected by BC-GN can be due to mechanisms other than acquisition of the CTX-M (blaCTX-M) gene.

· In vitro resistance to Ceftazadime and/or ceftriaxone is not always demonstrated for specimens containing CTX-M.

BC-GN Performance for Detection of CTX-M as Compared to Phenotypic Antimicrobial Susceptibility Testing using Ceftazadime and Ceftriaxone Agar Gradient Diffusion   

<table><tr><td rowspan="2">BC-GN Panel Analyte Detected</td><td rowspan="2">n= (a)</td><td colspan="4">Positive and Negative Percent Agreement (PPA and NPA) Detection of Resistance Markers Versus Ceftazadime Only or Ceftazidime and/or Ceftriaxone Agar Gradient Diffusion</td></tr><tr><td colspan="2">Ceftazadime (R) or (I)</td><td colspan="2">Ceftazadime (R) or (I) and/or Ceftriaxone (R) or (I)</td></tr><tr><td rowspan="2">Acinetobacter</td><td rowspan="2">28</td><td>PPA (c)</td><td>NPA d)</td><td>PPA (c)</td><td>NPA (d)</td></tr><tr><td>0%</td><td>100%</td><td>0% 0/25</td><td>100%</td></tr><tr><td rowspan="2">spp.</td><td rowspan="2"></td><td>0/10</td><td>18/18 (81.5-100)</td><td>(0-13.7)</td><td>3/3</td></tr><tr><td>(0-30.9) 0%</td><td>100%</td><td>0%</td><td>(29.2-100) 100%</td></tr><tr><td rowspan="2">Citrobacter spp.</td><td rowspan="2">31</td><td>0/10</td><td>21/21</td><td>0/11</td><td>20/20</td></tr><tr><td>(0-30.9)</td><td>(83.9-100)</td><td>(0-28.5)</td><td>(83.2-100)</td></tr><tr><td rowspan="2">Enterobacter spp.</td><td rowspan="2">87</td><td>10.0%</td><td>100%</td><td>12.1%</td><td>98.2%</td></tr><tr><td>3/30 (2.1-26.5)</td><td>57/57 (93.7-100)</td><td>4/33 (3.4-28.2)</td><td>53/54</td></tr><tr><td rowspan="2">Proteus spp. (b)</td><td rowspan="2">49</td><td>0%</td><td>100%</td><td>0%</td><td>(90.1-100) 100%</td></tr><tr><td>0/5</td><td>44/44</td><td>0/5</td><td>44/44</td></tr><tr><td rowspan="2">Escherichia coli</td><td rowspan="2">435</td><td>(0-52.2)</td><td>(92.0-100)</td><td>(0-52.2)</td><td>(92.0-100)</td></tr><tr><td>81.2%</td><td>96.5% 353/366</td><td>82.1%</td><td>100%</td></tr><tr><td rowspan="2">Klebsiella</td><td rowspan="2">225</td><td>56/69 (69.9-90.0)</td><td>(94.0-98.1)</td><td>69/84 (72.3-89.7)</td><td>351/351</td></tr><tr><td>33.7%</td><td>98.4%</td><td>34.6%</td><td>(99.0-100) 100%</td></tr><tr><td rowspan="2">pneumoniae</td><td rowspan="2"></td><td>34/101</td><td>122/124</td><td>36/104</td><td>121/121</td></tr><tr><td>(24.6-43.8)</td><td>(94.3-99.8)</td><td>(25.6-44.6)</td><td>(97.0-100)</td></tr><tr><td rowspan="2">Klebsiella oxytoca</td><td rowspan="2">50</td><td>0%</td><td>100%</td><td>0%</td><td>100%</td></tr><tr><td>0/7</td><td>43/43</td><td>0/7</td><td>43/43</td></tr><tr><td rowspan="2">Pseudomonas aeruginosa (b)</td><td rowspan="2">88</td><td>(0-41.0)</td><td>(91.8-100) 100%</td><td>(0-41.0) 0%</td><td>(91.8-100)</td></tr><tr><td>0% 0/24</td><td>64/64</td><td>0/24</td><td>100%</td></tr><tr><td rowspan="2"></td><td rowspan="2"></td><td>(0.0-14.3)</td><td>(94.4-100)</td><td>(0.0-14.3)</td><td>64/64</td></tr><tr><td>36.3%</td><td>98.0%</td><td>37.2%</td><td>(94.4-100)</td></tr><tr><td rowspan="2">Total</td><td rowspan="2">993</td><td>93/256</td><td>722/737</td><td>109/293</td><td>99.9%</td></tr><tr><td>(30.4-42.6)</td><td>(96.7-98.9)</td><td>(31.7-43.0)</td><td>699/700 (99.2-100)</td></tr></table>

(a) Ceftazadime only and Ceftazadime and/or Ceftriaxone Agar Gradient Diffusion results are provided for 993 of the 1266 total isolates available. (b) Only Ceftazidime results are calculated for Proteus spp. and Pseudomonas aeruginosa--Ceftriaxone results are not applicable per CLSI M100-S22. (c) Ceftazadime only and/or ceftriaxone resistance in the organisms detected by BC-GN can be due to mechanisms other than acquisition of the CTX-M (blaCTX-M) gene. (d) In vitro resistance to Ceftazadime or Ceftriaxone is not always demonstrated for specimens containing CTX-M.

3. Clinical cut-off:

Not applicable

4. Expected values/Reference range:

In the BC-GN clinical study, 876 prospectively-collected fresh and frozen blood culture specimens were obtained from twelve medium to large-sized healthcare institutions geographically distributed across the United States. The positivity rate as determined by the BC-GN Test stratified by geographic region for each of the organisms and antimicrobial resistance markers detected by the BC-GN test are presented in the table below. Overall, the BC-GN test detected at least one organism in $9 0 . 4 \%$ (792/876) and one resistance marker in $6 . 7 \%$ (59/876) of prospectivelycollected specimens.

Prevalence of Organisms Detected by BC-GN – Clinical Study Observations   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="12" rowspan="1">US Geographic Region/Division*</td><td colspan="1" rowspan="4">Total</td></tr><tr><td colspan="1" rowspan="4">Organism</td><td colspan="1" rowspan="1">Region</td><td colspan="6" rowspan="1">Midwest</td><td colspan="2" rowspan="1">South</td><td colspan="1" rowspan="1">Northeast</td><td colspan="3" rowspan="1">West</td></tr><tr><td colspan="1" rowspan="1">Division</td><td colspan="2" rowspan="1">WestNorthCentral</td><td colspan="4" rowspan="1">East North Central</td><td colspan="1" rowspan="1">SouthAtlantic</td><td colspan="1" rowspan="1">W.South Central</td><td colspan="1" rowspan="1">MiddleAtlantiC</td><td colspan="2" rowspan="1">Pacific</td><td colspan="1" rowspan="1">Mountain</td></tr><tr><td colspan="1" rowspan="1">State</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">MN</td><td colspan="1" rowspan="1">MI</td><td colspan="1" rowspan="1">WI</td><td colspan="1" rowspan="1">IL</td><td colspan="1" rowspan="1">OH</td><td colspan="1" rowspan="1">MD</td><td colspan="1" rowspan="1">TX</td><td colspan="1" rowspan="1">NY</td><td colspan="1" rowspan="1">CA</td><td colspan="1" rowspan="1">WA</td><td colspan="1" rowspan="1">UT</td></tr><tr><td colspan="1" rowspan="1">Total$n=$</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">145</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">876</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="2">Citrobacter</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="2">Enterobacter</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">8.7</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">4.6</td><td colspan="1" rowspan="1">9.5</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">9.0</td><td colspan="1" rowspan="1">12.2</td><td colspan="1" rowspan="1">7.6</td><td colspan="1" rowspan="1">4.6</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">7.6</td></tr><tr><td colspan="1" rowspan="2">Proteus</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">8.7</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">3.7</td></tr><tr><td colspan="1" rowspan="2">E. coli</td><td colspan="1" rowspan="1">POSn=$</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">429</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">47.8</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">55.6</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">56.3</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">25.6</td><td colspan="1" rowspan="1">58.6</td><td colspan="1" rowspan="1">59.1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">41.8</td><td colspan="1" rowspan="1">53.4</td><td colspan="1" rowspan="1">49.0</td></tr><tr><td colspan="1" rowspan="2">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">128</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">26.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16.7</td><td colspan="1" rowspan="1">21.6</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">19.4</td><td colspan="1" rowspan="1">23.3</td><td colspan="1" rowspan="1">13.8</td><td colspan="1" rowspan="1">9.1</td><td colspan="1" rowspan="1">11.5</td><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">14.6</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">POSn=$</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="1">oxytoca</td><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8.3</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">6.8</td><td colspan="1" rowspan="1">4.2</td></tr><tr><td colspan="1" rowspan="2">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">78</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19.4</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">13.5</td><td colspan="1" rowspan="1">4.7</td><td colspan="1" rowspan="1">9.0</td><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">6.2</td><td colspan="1" rowspan="1">10.2</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">16.4</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">8.9</td></tr><tr><td colspan="1" rowspan="1">ResistanceMarker(linked)</td><td colspan="1" rowspan="1">Totaln=</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">136</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">773</td></tr><tr><td colspan="1" rowspan="2">NDM</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="2">KPC</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="2">CTX-M</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8.6</td><td colspan="1" rowspan="1">7.1</td><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">8.9</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">11.0</td><td colspan="1" rowspan="1">6.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">13.6</td><td colspan="1" rowspan="1">7.3</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">6.5</td></tr><tr><td colspan="1" rowspan="2">VIM</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">IMP</td><td colspan="1" rowspan="1">POSn=$</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">OXA</td><td colspan="1" rowspan="1">POSn=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">%Prev.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.6</td></tr></table>

# N. Instrument Name:

Verigene Processor SP Verigene Reader

# O. System Descriptions:

1. Modes of Operation:

The Reader is the Verigene System’s central control unit and user interface, and, with a touch-screen control panel and barcode scanner, guides the user through test processing, imaging, and test result generation. The Verigene Processor $S P$ executes the test procedure, automating the steps of (1) Sample Preparation– cell lysis and magnetic bead-based bacterial DNA isolation from blood culture samples, and (2) Hybridization– detection and identification of bacterial-specific DNA in a microarray format by using gold nanoparticle probe-based technology. Once the specimen is loaded by the operator, all other fluid transfer steps are performed by an automated pipette that transfers reagents between wells of the trays and loads the specimen into the Test Cartridge for

hybridization. Single-use disposable test consumables and a self-contained Verigene Test Cartridge are utilized for each sample tested with the BC-GN test.

To obtain the test results after processing is complete, the user removes the Test Cartridge from the Processor $S P$ , and inserts the substrate holder into the Reader for analysis. Determination of the presence or absence of each targeted analyte is made by means of software-based decision algorithm resident in the Reader.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X___ or No

3. Specimen Identification:

Specimen cartridges are labeled with a Barcode. The Processor SP and Reader detect the specimen ID and the results are printed with this specimen identifier.

4. Specimen Sampling and Handling:

The user manually pipettes the initial blood culture sample into the Sample Loading Well in the Extraction Tray after which all specimen handling is performed by the automated Verigene System.

5. Calibration: Not Required

6. Quality Control: See Section M (1d) above

P. O therSupportiveInstrum entPerform anceCharacteristicsData NotCovered In The“Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.