# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K132822

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for a new 510(k) application for the BD MAX™ StaphSR assay on the BD MAX™ System.

# C. Measurand:

Target DNA sequences for

(1) SCCmec/orfX junction area of methicillin-resistant Staphylococcus aureus (i.e., MREJ for SCCmec Right Extremity Junction). The BD MAXTM StaphSR assay is designed to detect MREJ types i, ii, iii, iv, v, vi, vii, ix, xiii, xiv, and xxi;

(2) mecA and mecC genes for methicillin resistance; and (3) nuc gene encoding a thermostable nuclease of Staphylococcus aureus.

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for the amplification and detection of methicillin-resistant Staphylococcus aureus (MRSA) DNA and Staphylococcus aureus (SA) DNA.

# E. Applicant:

GeneOhm Sciences Canada, Inc. (BD Diagnostics)

F. Proprietary and Established Names:

BD MAX™ StaphSR BD MAX™ System

# G. Regulatory Information:

1. Regulation section: 21 CFR section 866.1640, Antimicrobial Susceptibility Test Powder

2. Classification: Class II

3. Product code(s):

NQX - System, Nucleic acid amplification test, DNA, methicillin resistant   
Staphylococcus aureus, Direct specimen   
OOI - Real time nucleic acid amplification system

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD MAX™ StaphSR assay performed on the BD MAX™ System is an automated qualitative in vitro diagnostic test for the direct detection and differentiation of Staphylococcus aureus (SA) and methicillin-resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization. The test utilizes realtime polymerase chain reaction (PCR) for the amplification of MRSA/SA DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD MAX™ StaphSR assay is intended to aid in the prevention and control of MRSA and SA infections in healthcare settings. It is not intended to diagnose MRSA or SA infections nor guide or monitor treatment for MRSA/SA infections. A negative result does not preclude nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: The BD MAX™ System

# I. Device Description:

The BD MAX™ System and the BD MAX™ StaphSR assay are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, real-time PCR master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes. The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX™ System software automatically interprets test results.

The real-time PCR master mixes include 13 different primers (nine primers for SCCmec/orfX junction, two primers for mecA, and two primers for mecC) for the detection of methicillinresistant Staphylococcus aureus (MRSA) DNA, and two different primers for the detection of Staphylococcus aureus (SA) DNA.

# Brief Explanation of the Procedure

A nasal specimen is collected and transported to the laboratory using a recommended swab collection and transport system. The swab is placed in a BD MAX™ StaphSR Sample Buffer Tube. The Sample Buffer Tube is vortexed to release cells from the swab into the buffer. The Sample Buffer Tube is placed into the BD MAX™ System and the following automated procedures occur: The bacterial cells are lysed, DNA is extracted on magnetic beads and concentrated, and then an aliquot of the eluted DNA is added to PCR reagents which contain the SA- and MRSA- specific primers used to amplify the genetic targets, if present. The assay also includes a Sample Processing Control (SPC). The SPC is present in the Extraction Tube and undergoes the extraction, concentration and amplification steps to monitor for inhibitory substances as well as process inefficiency due to instrument or reagent failure. No operator intervention is necessary once the clinical sample and reagent strip are loaded into the BD MAX™ System. The BD MAX™ System automates sample lysis, DNA extraction and concentration, reagent rehydration, nucleic acid amplification and detection of the target nucleic acid sequence using real-time polymerase chain reaction (PCR). Amplified targets are detected with hydrolysis probes labeled with quenched fluorophores. The amplification, detection and interpretation of the signals are done automatically by the BD MAX™ System.

# Reagents

<table><tr><td rowspan=1 colspan=1>Contents</td><td rowspan=1 colspan=1>Quantity</td></tr><tr><td rowspan=1 colspan=1>BD MAXTM StaphSR Master Mix (B7)Dried PCR Master Mix containing polymerase, nucleotides and specific molecular probes andprimers along with Sample Processing Control-specific molecular probe.</td><td rowspan=1 colspan=1>24 tests</td></tr><tr><td rowspan=1 colspan=1>BD MAXTM StaphSR Reagent StripUnitized Reagent strip containing all liquid reagents and disposable pipette tips necessary forspecimen processing and DNA extraction.</td><td rowspan=1 colspan=1>24 tests</td></tr><tr><td rowspan=1 colspan=1>BD MAXTM StaphSR Extraction Tube (B8)Dried extraction reagent containing DNA magnetic affinity beads, Achromopeptidase and SampleProcessing Control</td><td rowspan=1 colspan=1>24 tests</td></tr><tr><td rowspan=1 colspan=1>BD MAXTM StaphSR Sample Buffer Tube(with 25 septum caps)</td><td rowspan=1 colspan=1>24 tests</td></tr></table>

# Equipment and Materials Required But Not Provided

BBL™ CultureSwab™ Liquid Stuart single or double swab (Becton Dickinson catalogue no. 220099 or 220109), Copan (Venturi) Transystem™ Liquid Stuart single or double swab (Copan, catalogue no. 141C or 139C)

· VWR Multi-Tube Vortexer (VWR catalog no. 58816-115)   
· NALGENE® Cryogenic Vial Holder (VWR, catalog no. 66008-783)   
· Disposable gloves, powderless   
· Sterile scissors (optional)   
· Sterile Gauze   
· Stopwatch or timer   
· BD MAX™ PCR Cartridges (BD Diagnostic Systems catalogue no. 437519)

# Quality Control

Quality control procedures monitor the performance of the assay. Laboratories must establish the number, type and frequency of testing control materials according to guidelines or requirements of local, provincial, state and/country regulations or accreditation organizations.

1. An External Positive Control is intended to monitor for substantial reagent failure while an External Negative Control is used to detect reagent or environmental contamination (or carry-over) from other specimens or SA or MRSA amplicons. External Control materials are not provided by BD. Various types of External Controls are recommended to allow the user to select the most appropriate control for their laboratory quality control program:

Commercially available control materials [e.g., a reference MRSA strain (ATCC 43300), and methicillin-susceptible Staphylococcus aureus (MSSA) strain (e.g., ATCC 29213) can be used as positive controls. Staphylococcus epidermidis strain (e.g., ATCC 12228) can be used as negative control.].

· Previously characterized specimens known to be positive or negative for S. aureus or MRSA.

2. One (1) External Positive Control and one (1) External Negative Control should be run daily until adequate process validation is achieved on the BD MAX™ System. The MRSA and the MSSA control strains should be tested alternately; each of them being tested every other day. Reduced frequency of control testing should be based on a protocol and data as determined by the individual laboratory.

3. An External Negative Control that yields a positive test result is indicative of a specimen handling and/or contamination problem. Review the specimen handling technique to avoid mix-up and/or contamination.

4. An External Positive Control that yields a negative result is indicative of a specimen handling/preparation problem. Review the specimen handling/preparation technique.

5. An External Control that yields an Unresolved, Indeterminate or Incomplete test result is indicative of a reagent or a BD MAX™ System failure. Check the BD MAX™ System and monitor for any error messages. If the problem persists, use reagents from an unopened pouch or use a new BD MAX™ StaphSR assay kit.

Note: External Positive and Negative Controls are not used by the BD MAX™ System software for the purpose of sample test result interpretation. External Controls are treated as if they were patient samples.

6. Each BD MAX™ StaphSR assay Extraction Tube contains a Sample Processing Control (SPC) which is a plasmid containing a synthetic target DNA sequence. The SPC will be extracted, eluted and amplified along with any DNA present in the processed specimen.

The SPC monitors the efficiency of DNA capture, washing and elution during the sample processing steps, as well as the efficiency of DNA amplification and detection during PCR analysis. If the SPC result fails to meet the acceptance criteria for a negative specimen, the result will be reported as Unresolved. An Unresolved result is indicative of specimen-associated inhibition or reagent failure. Repeat any specimen reported as “Unresolved”.

# Results Interpretation

The BD MAX™ System software automatically interprets test results. A test result may be called as [SA NEG, MRSA NEG (negative)], [SA POS, MRSA POS (MRSA positive)], [SA POS, MRSA NEG (SA positive)] or [SA UNR, MRSA UNR (Unresolved)] based on the amplification status of the targets and of the Sample Processing Control. IND (Indeterminate) or INC (Incomplete) results are due to BD MAX™ System failure. Results interpretation is based on the following decision algorithm:

<table><tr><td rowspan=1 colspan=1>Assay Result Reported</td><td rowspan=1 colspan=1>Interpretation of Result</td></tr><tr><td rowspan=1 colspan=1>SA POSMRSA POS</td><td rowspan=1 colspan=1>MRSA DNA detected</td></tr><tr><td rowspan=1 colspan=1>SA POSMRSA NEG</td><td rowspan=1 colspan=1>SA DNA detected;No MRSA DNA detected</td></tr><tr><td rowspan=1 colspan=1>SA NEGMRSA NEG</td><td rowspan=1 colspan=1>No SA and no MRSA DNA detected</td></tr><tr><td rowspan=1 colspan=1>SA UNRMRSA UNR</td><td rowspan=1 colspan=1>No target amplification; no SPCamplification</td></tr><tr><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>Indeterminate due to BD MAXTM Systemfailure(with Warning or Error Codes )</td></tr><tr><td rowspan=1 colspan=1>INC</td><td rowspan=1 colspan=1>Incomplete Run(with Warning or Error Codes )</td></tr></table>

# SA POS, MRSA POS (MRSA DNA detected)

Fluorescence signal is detected for both MREJ (S. aureus specific) and mecA/mecC targets; and,   
The nuc gene target may or may not be detected since it has been shown that, in rare instances, the nuc gene may be absent for MRSA; and,   
The SPC is ignored since MRSA target amplification overrides this control.

SA POS, MRSA NEG (SA DNA detected; No MRSA DNA detected)

Fluorescence signal is detected for the nuc gene target only (indicative of an SA   
strain); or,   
Fluorescence signal is detected for the nuc gene and mecA/mecC targets in the   
absence of MREJ sequences (indicative of an SA strain present with co-colonization   
of a non-SA methicillin-resistant bacterial strain); or,   
Fluorescence signal is detected for the nuc gene and MREJ targets (indicative of an   
empty cassette variant); or,   
Fluorescence signal is detected for the MREJ target only (indicative of a S. aureus   
empty cassette variant. MREJ is specific to S. aureus species and thus is indicative of

an SA strain. The presence of the nuc gene target is or is not detected for an empty cassette variant); SPC is ignored since SA target amplification overrides this control.

# SA NEG, MRSA NEG (No SA and MRSA DNA detected)

Fluorescence signal is not detected by the BD MAX™ StaphSR assay for any target (nuc, mecA/mecC and MREJ targets) and fluorescence signal is detected for the SPC; or,   
Fluorescence signal is detected for the mecA/mecC gene only [the mecA and mecC genes are not unique to S. aureus species and can be found in other bacterial genera (e.g., S. epidermidis)].

# SA UNR, MRSA UNR (Unresolved result, No Target and SPC Amplification)

· Fluorescence signal not detected for nuc gene, mecA/mecC, or MREJ targets; and, · Fluorescence signal not detected for the SPC (Inhibitory specimen or reagent failure).

IND (Indeterminate result) · BD MAX™ System failure with Warning or Error Codes.   
INC (Incomplete run) · BD MAX™ System failure with Warning or Error Codes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX™ MRSA Assay and BD GeneOhm™ StaphSR Assay   
2. Predicate 510(k) number(s): K071026 and K120138   
3. Comparison with predicates:

<table><tr><td colspan="4" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">BD MAXTM StaphSR</td><td colspan="1" rowspan="1">BD GeneOhm™M StaphSR Assay(K071026)</td><td colspan="1" rowspan="1">BD MAXTM MRSA Assay(K120138)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD MAXTM StaphSR assayperformed on the BD MAXTMSystem is an automated qualitativein vitro diagnostic test for the directdetection and differentiation ofStaphylococcus aureus (SA) andmethicillin-resistant Staphylococcusaureus (MRSA) DNA from nasalswabs in patients at risk for nasalcolonization. The test utilizes real-</td><td colspan="1" rowspan="1">The BD GeneOhmTM StaphSRAssay is a qualitative in vitrodiagnostic test for the rapiddetection of Staphylococcusaureus (SA) and methicillin-resistant Staphylococcus aureus(MRSA) directly from positiveblood culture. The assay utilizespolymerase chain reaction (PCR)for the amplification of specific</td><td colspan="1" rowspan="1">The BD MAXTM MRSA Assayperformed on the BD MAXTMSystem is an automated qualitativein vitro diagnostic test for thedirect detection of methicillin-resistant Staphylococcus aureus(MRSA) DNA from nasal swabs inpatients at risk for nasalcolonization. The test utilizes real-time polymerase chain reaction</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">time polymerase chain reaction(PCR) for the amplification ofMRSA/SA DNA and fluorogenictarget-specific hybridization probesfor the detection of the amplifiedDNA. The BD MAXTM StaphSRassay is intended to aid in theprevention and control of MRSAand SA infections in healthcaresettings. It is not intended todiagnose MRSA or SA infectionsnor guide or monitor treatment forMRSA/SA infections. A negativeresult does not preclude nasalcolonization. Concomitant culturesare necessary to recover organismsfor epidemiological typing or forfurther susceptibility testing.</td><td colspan="1" rowspan="1">targets and fluorogenic target-specific hybridization probes forthe real-time detection of theamplified DNA. The assay isperformed on Gram positivecocci, identified by Gram stain,from positive blood cultures. TheBD GeneOhm™M StaphSR Assayis not intended to monitortreatment for MRSA/SAinfections. Subculturing ofpositive blood cultures isnecessary for furthersusceptibility testing.</td><td colspan="1" rowspan="1">(PCR) for the amplification ofMRSA DNA and fluorogenictarget-specific hybridization probesfor the detection of the amplifiedDNA. The BD MAXTM MRSAAssay is intended to aid in theprevention and control of MRSAinfections in healthcare settings. Itis not intended to diagnose, guideor monitor MRSA infections. Anegative result does not precludenasal colonization. Concomitantcultures are necessary to recoverorganisms for epidemiologicaltyping or for further susceptibilitytesting.</td></tr><tr><td colspan="1" rowspan="1">Specimen type</td><td colspan="1" rowspan="1">Nasal swabs</td><td colspan="1" rowspan="1">Positive blood culture</td><td colspan="1" rowspan="1">Nasal swabs</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCRDetection: Fluorogenic target-specific hybridization</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Mode of Detection forMethicillin Resistancein S. aureus</td><td colspan="1" rowspan="1">Presence of SCCmec cassette atorfX junction (specific to S. aureus)and mecA or mecC genes</td><td colspan="1" rowspan="1">Presence of SCCmec cassette atorfX junction (specific to S.aureus)</td><td colspan="1" rowspan="1">Presence of SCCmec cassette atorfX junction (specific to S.aureus)</td></tr><tr><td colspan="1" rowspan="1">Mode of Detection forSA</td><td colspan="1" rowspan="1">Presence of nuc gene specific for S.aureus</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Not detected</td></tr><tr><td colspan="1" rowspan="1">Interpretation of TestResults</td><td colspan="1" rowspan="1">Automated (Diagnostic software ofBD MAX™M System)</td><td colspan="1" rowspan="1">Automated (Diagnostic softwareof SmartCycler® System)</td><td colspan="1" rowspan="1">Automated (Diagnostic software ofBD MAXTM System)</td></tr><tr><td colspan="1" rowspan="1">Analysis Platform</td><td colspan="1" rowspan="1">BD MAXTM System</td><td colspan="1" rowspan="1">SmartCycler® System</td><td colspan="1" rowspan="1">BD MAX™M System</td></tr><tr><td colspan="1" rowspan="1">PCR SamplePreparation</td><td colspan="1" rowspan="1">Automated by the BD MAXTMSystem</td><td colspan="1" rowspan="1">Manual</td><td colspan="1" rowspan="1">Automated by the BD MAXTMSystem</td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">TaqMan® Probe</td><td colspan="1" rowspan="1">Molecular Beacon Probe</td><td colspan="1" rowspan="1">TaqMan® Probe</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Specimen Processing Control (SPC)</td><td colspan="1" rowspan="1">Positive PCR control (DNA fromS.aureus ATCC 43300).Negative control (DNA fromS.epidermidis ATCC 14990).Internal Control</td><td colspan="1" rowspan="1">Specimen Processing Control(SPC)</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA). Draft Guidance for Industry and Food and Drug Administration Staff. Document issued on: January 5, 2011

# L. Test Principle:

The BD MAX™ System uses a combination of lytic and extraction reagents to perform cell lysis and DNA extraction. Following enzymatic cell lysis at elevated temperature, the released nucleic acids are captured by magnetic affinity beads. The beads with the bound nucleic acids are washed and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized with Neutralization Buffer and transferred to the Master Mix Tube to rehydrate the PCR reagents. The reconstituted amplification reagent is dispensed into the BD MAX™ PCR Cartridge. Microvalves in the BD MAX™ PCR Cartridge are sealed by the system prior to initiating PCR to prevent evaporation and amplicon contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan®) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect a specific amplicon in the SCCmec right- extremity junction (MREJ), the genes for methicillin resistance mecA and mecC, the nuc gene encoding a thermostable nuclease of S. aureus and SPC amplicons in four different optical channels of the BD MAX™ System: MREJ amplicons are detected in the FAM channel, mecA and mecC amplicons are detected in the ROX channel, nuc amplicons are detected in the VIC channel and SPC amplicons are detected in the $\mathrm { C y } 5 . 5$ channel. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the four optical channels used for the BD MAX™ StaphSR assay is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX™ System measures these signals at the end of each amplification cycle, and interprets the data to provide a result.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Precision Study

Within-laboratory precision was evaluated for the BD MAX™ StaphSR assay at one site. The precision panel consisted of six categories near the LoD (i.e., Moderate Positive MRSA, Low Positive MRSA, Low Positive MSSA, High Negative MRSA, High Negative MSSA, and True Negative). Each specimen contained simulated nasal matrix $2 \%$ EMD or $5 \%$ Sigma-Aldrich w/v Mucin, $5 \%$ v/v Whole Blood, $0 . 8 \%$ v/v NaCl, Saline, and $0 . 0 0 0 0 \%$ w/v Human Genomic DNA. Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details). The MSSA strain used in the precision panels was Staphylococcus aureus ATCC 29213 and the MRSA strains used were Staphylococcus aureus ATCC 43300 (MREJ type ii) and a Staphylococcus aureus strain obtained from a private collection (MREJ type vii). The negative strain used in the panels was Staphylococcus epidermidis ATCC 14990). Bacterial strains were suspended in the simulated nasal matrix described above at the targeted concentrations. Bacterial cell numbers in the suspensions were verified by colony counts. The bacterial suspensions were divided into $8 2 . 5 { \mathrm { u l } }$ aliquots and frozen at $- 8 0 ^ { 0 } \mathrm { C }$ prior to testing. At the testing site, aliquots were thawed on a cooling block, vortexed, and spun at low speed. The contents of each aliquot were completely absorbed onto a sterile dry swab. The swab was then transferred directly into a Sample Buffer Tube (SBT) and processed according to the BD MAX™ StaphSR assay instructions for use. Appropriate external controls (ECs) were included in each run. A reference MRSA strain (ATCC 43300) or a methicillin susceptible S. aureus (MSSA) strain (ATCC 25923) were used alternatively as a positive EC, and a reference strain of Staphylococcus epidermidis (ATCC 14990) was used as a negative EC.

The MRSA and MSSA strains were tested as follows:

Moderate Positive (MP) MRSA (MREJ ii): $\geq 2$ and $\le 5 \mathrm { \bf ~ X }$ LoD · Low Positive (LP) MRSA (MREJ ii): $\geq 1$ and $< 2 \mathrm { ~ x ~ }$ LoD Low Positive (LP) MRSA (MREJ vii): $\geq 1$ and $< 2 \mathrm { ~ x ~ }$ LoD Low Positive (LP) MSSA: $\geq 1$ and $< 2 \mathrm { \Delta x }$ LoD High Negative (HN) MRSA (MREJ ii): $< 1 \mathrm { \Delta x }$ LoD High Negative (HN) MSSA: $< 1 \mathrm { \Delta x }$ LoD True negative (TN): Negative specimens (no target)

Testing was performed in duplicate, over 12 days, with two runs per day, by two different technologists (2 replicates/run/day x 2 runs/day by two different technologists $\mathrm { \Delta x 1 2 d a y s = 4 8 }$ data points for each panel member). Precision study results for TN, MP, LP, and HN MRSA samples demonstrated $100 \%$ (48/48, $9 5 \%$ CI, $9 2 . 6 \% - 1 0 0 \%$ , $100 \%$ (48/48, $9 5 \%$ CI, $9 2 . 6 \% - 1 0 0 \%$ , $9 7 . 9 \%$ (94/96, $9 5 \%$ CI, $9 2 . 7 \% - 9 9 . 4 \% )$ and $2 7 . 1 \%$ (13/48, $9 5 \%$ CI, $1 6 . 6 \% - 4 1 . 0 \% )$ agreement with expected result (i.e., negative for TN and HN, positive for MP and LP), respectively. Precision study results for LP and HN MSSA samples demonstrated $100 \%$ (48/48, $9 5 \%$ CI, $9 2 . 6 \% - 1 0 0 \%$ , and $5 6 . 2 \%$ (27/48, $9 5 \%$ CI, $4 2 . 3 \% - 6 9 . 3 \% )$ agreement with expected result (i.e., negative for TN and HN), respectively.

The precision study results met the pre-defined acceptance criteria for LP, MP, and TN samples (i.e., LP overall correct percentage of approximately $9 5 \%$ with $9 5 \%$ CI; MP overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI; TN overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI). No specific acceptance criteria were defined for the HN samples.

# Reproducibility Study

The reproducibility study was performed using the same sample categories and testing procedures as defined above for the Precision Study.

Samples in each category were tested in triplicate, on five distinct days, wherein each day two runs were tested by two different technologists, at three clinical sites using one lot of reagents (Site-to-Site) (3 replicates/run/day x 2 runs/day by two different technologists/per site x 5 days x 3 sites $= 9 0$ data points for each panel member). One of these three clinical sites participated in an extended study where two additional lots of reagents were tested in 10 additional days (Lot-to-Lot) (3 replicates/run/day $_ { \textrm { X 2 } }$ runs/day by two different technologists/per site x 5 days/per Lot x 3 Lots $= 9 0$ data points for each panel member).

For Site-to-Site Reproducibility, the overall percent agreement with expected result (i.e., negative for TN and HN, positive for MP and LP) was $100 \%$ (90/90, $9 5 \%$ CI, $9 5 . 9 \% - 1 0 0 \% )$ for MRSA MP and TN categories; $9 6 . 7 \%$ (174/180, $9 5 \%$ CI, $9 2 . 9 \%$ - $9 8 . 5 \% )$ and $9 7 . 8 \%$ (88/90, $9 5 \%$ CI, $9 2 . 3 \% - 9 9 . 4 \% )$ for MRSA LP and MSSA LP, respectively; and $3 6 . 7 \%$ (33/90, $9 5 \%$ CI, $2 7 . 4 \% - 4 7 . 0 \% )$ and $3 0 . 0 \%$ (27/90, $9 5 \%$ CI, $2 1 . 5 \% - 4 0 . 1 \%$ for MRSA HN and MSSA HN, respectively.

Site-To-Site Reproducibility Study Qualitative Results Using One Lot of the BD MAX™ StaphSR Assay   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>SITE</td><td rowspan=3 colspan=2>Overall PercentAgreement and 95% CI</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td></tr><tr><td rowspan=1 colspan=1>HN1 MSSA</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>5/30</td><td rowspan=1 colspan=1>23.3%</td><td rowspan=1 colspan=1>7/30</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>15/30</td><td rowspan=1 colspan=1>30.0%(27/90)</td><td rowspan=1 colspan=1>(21.5%, 40.1%)</td></tr><tr><td rowspan=1 colspan=1>HN MRSA</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>12/30</td><td rowspan=1 colspan=1>26.7%</td><td rowspan=1 colspan=1>8/30</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>36.7%(33/90)</td><td rowspan=1 colspan=1>(27.4%, 47.0%)</td></tr><tr><td rowspan=1 colspan=1>LP MSSA</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>97.8%(88/90)</td><td rowspan=1 colspan=1>(92.3%, 99.4%)</td></tr><tr><td rowspan=1 colspan=1>LP MRSA</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%(174/180)</td><td rowspan=1 colspan=1>(92.9%, 98.5%)</td></tr><tr><td rowspan=1 colspan=1>MP MRSA</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

End Point (EP) and Second Derivative Peak Abscissa (SDPA), two underlying numerical values used to determine a final assay result, were selected as additional means of assessing assay reproducibility.

Site-to-Site Reproducibility Study Underlying Numerical Results Across Sites, Days, Runs, and Replicates Using One Lot, for MREJ Target with MREJ Types Pooled (FAM Channel)   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Category</td><td colspan="1" rowspan="1">HN MRSA</td><td colspan="1" rowspan="1">LP MRSA</td><td colspan="1" rowspan="1">MP MRSA</td></tr><tr><td colspan="1" rowspan="8">EP</td><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">638.9</td><td colspan="1" rowspan="1">964.5</td><td colspan="1" rowspan="1">992.9</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">309.82</td><td colspan="1" rowspan="1">344.38</td><td colspan="1" rowspan="1">398.36</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">48.5%</td><td colspan="1" rowspan="1">35.7%</td><td colspan="1" rowspan="1">40.1%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">166.88</td><td colspan="1" rowspan="1">63.09</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">17.3%</td><td colspan="1" rowspan="1">6.4%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="17">SDPA</td><td colspan="1" rowspan="1">Between Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">208.85</td><td colspan="1" rowspan="1">348.35</td><td colspan="1" rowspan="1">366.40</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">32.7%</td><td colspan="1" rowspan="1">36.1%</td><td colspan="1" rowspan="1">36.9%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">373.64</td><td colspan="1" rowspan="1">517.49</td><td colspan="1" rowspan="1">544.90</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">58.5%</td><td colspan="1" rowspan="1">53.7%</td><td colspan="1" rowspan="1">54.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">30.7</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">0.47</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.1%</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">1.5%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.49</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Between Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.21</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">0.7%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">0.71</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.2%</td><td colspan="1" rowspan="1">3.4%</td><td colspan="1" rowspan="1">2.3%</td></tr></table>

1Values shown are those obtained for the MREJ target in the samples that gave a SA+. MRSA+ result

Site-to-Site Reproducibility Study Underlying Numerical Results Across Sites, Days, Runs, and Replicates Using One Lot, for mecA/mecC Target with MREJ Types Pooled (ROX Channel)1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>HN MRSA</td><td rowspan=1 colspan=1>LP MRSA</td><td rowspan=1 colspan=1>MP MRSA</td></tr><tr><td rowspan=25 colspan=1>EPSDPA</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>1722.9</td><td rowspan=1 colspan=1>2582.8</td><td rowspan=1 colspan=1>2702.9</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>456.44</td><td rowspan=1 colspan=1>621.52</td><td rowspan=1 colspan=1>723.06</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>26.5%</td><td rowspan=1 colspan=1>24.1%</td><td rowspan=1 colspan=1>26.8%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>143.99</td><td rowspan=1 colspan=1>240.76</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>8.4%</td><td rowspan=1 colspan=1>9.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>199.57</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>11.6%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>510.53</td><td rowspan=1 colspan=1>558.35</td><td rowspan=1 colspan=1>647.66</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>29.6%</td><td rowspan=1 colspan=1>21.6%</td><td rowspan=1 colspan=1>24.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>727.70</td><td rowspan=1 colspan=1>869.49</td><td rowspan=1 colspan=1>970.71</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>42.2%</td><td rowspan=1 colspan=1>33.7%</td><td rowspan=1 colspan=1>35.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>31.1</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.52</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>2.5%</td></tr></table>

1Values shown are those obtained for the mecA/mecC target in the samples that gave a $\mathrm { S A ^ { + } }$ . MRSA+ result

# Site-to-Site Reproducibility Study Underlying Numerical Results Across Sites, Days, Runs, and Replicates Using One Lot, for nuc Target (VIC Channel)1

<table><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Category</td><td colspan="1" rowspan="1">HN MREJ Type ii</td><td colspan="1" rowspan="1">HN MSSA</td><td colspan="1" rowspan="1">LP MSSA</td></tr><tr><td colspan="1" rowspan="10">EP</td><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">88</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">631.7</td><td colspan="1" rowspan="1">1165.4</td><td colspan="1" rowspan="1">1664.8</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">359.53</td><td colspan="1" rowspan="1">451.70</td><td colspan="1" rowspan="1">280.37</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">56.9%</td><td colspan="1" rowspan="1">38.8%</td><td colspan="1" rowspan="1">16.8%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">280.15</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">333.08</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">44.3%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">20.0%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Between Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">425.40</td><td colspan="1" rowspan="1">550.34</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">36.5%</td><td colspan="1" rowspan="1">33.1%</td></tr><tr><td colspan="1" rowspan="15">SDPA</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">455.79</td><td colspan="1" rowspan="1">620.48</td><td colspan="1" rowspan="1">701.73</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">72.1%</td><td colspan="1" rowspan="1">53.2%</td><td colspan="1" rowspan="1">42.1%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">88</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">32.0</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">1.61</td><td colspan="1" rowspan="1">1.41</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">4.6%</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">2.6%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">2.3%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.2%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Between Site</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.11</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">1.78</td><td colspan="1" rowspan="1">1.69</td><td colspan="1" rowspan="1">1.12</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">5.1%</td><td colspan="1" rowspan="1">4.9%</td><td colspan="1" rowspan="1">3.5%</td></tr></table>

1Values shown are those obtained for the nuc target in the samples that gave a SA+. MRSA- result

Site-to-Site Reproducibility Study Underlying Numerical Results Across Sites, Days, Runs, and Replicates Using One Lot, Sample Processing Control Value (Cy5.5 Channel)1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>HN MREJ Type ii</td><td rowspan=1 colspan=1>HN MSSA</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=25 colspan=1>EPSDPA</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>3065.0</td><td rowspan=1 colspan=1>2854.6</td><td rowspan=1 colspan=1>2968.1</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>700.52</td><td rowspan=1 colspan=1>366.78</td><td rowspan=1 colspan=1>901.46</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>22.9%</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>30.4%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>922.55</td><td rowspan=1 colspan=1>688.75</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>30.1%</td><td rowspan=1 colspan=1>24.1%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>181.03</td><td rowspan=1 colspan=1>409.70</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>14.4%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1172.43</td><td rowspan=1 colspan=1>881.34</td><td rowspan=1 colspan=1>901.46</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>38.3%</td><td rowspan=1 colspan=1>30.9%</td><td rowspan=1 colspan=1>30.4%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>30.2</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.51</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.13</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>0.35</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>0.63</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>2.1%</td></tr></table>

1Calculated for the Specimen Processing Control of the samples that gave a SA-. MRSA- result

For Lot-to-Lot Reproducibility, the overall percent agreement with expected result (i.e., negative for TN and HN, positive for MP and LP) was $100 \%$ (90/90, $9 5 \%$ CI, $9 5 . 9 \% - 1 0 0 \% )$ for MRSA MP and TN categories; $9 6 . 7 \%$ (174/180, $9 5 \%$ CI, $9 2 . 9 \%$ - $9 8 . 5 \% )$ and $9 6 . 7 \%$ (87/90, $9 5 \%$ CI, $9 0 . 7 \% - 9 8 . 9 \% )$ for MRSA LP and MSSA LP, respectively; and $4 0 . 0 \%$ (36/90, $9 5 \%$ CI, $3 0 . 5 \% - 5 0 . 3 \% )$ and $4 4 . 4 \%$ (40/90, $9 5 \%$ CI, $3 4 . 6 \% - 5 4 . 7 \% )$ for MRSA HN and MSSA HN, respectively.

Lot-To-Lot Reproducibility Study Results using Three Lots of the BD MAX™ StaphSR Assay   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>LOT</td><td rowspan=3 colspan=2>Overall Percent Agreementand 95% CI</td></tr><tr><td rowspan=1 colspan=2>Lot 1</td><td rowspan=1 colspan=2>Lot 2</td><td rowspan=1 colspan=2>Lot 3</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td></tr><tr><td rowspan=1 colspan=1>HN&#x27; MSSA</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>15/30</td><td rowspan=1 colspan=1>36.7%</td><td rowspan=1 colspan=1>11/30</td><td rowspan=1 colspan=1>46.7%</td><td rowspan=1 colspan=1>14/30</td><td rowspan=1 colspan=1>44.4%</td><td rowspan=1 colspan=1>(34.6%, 54.7%)</td></tr><tr><td rowspan=1 colspan=1>HN MRSA</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>(30.5%, 50.3%)</td></tr><tr><td rowspan=1 colspan=1>LP MSSA</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>(90.7%, 98.9%)</td></tr><tr><td rowspan=1 colspan=1>LP MRSA</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>(92.9%, 98.5%)</td></tr><tr><td rowspan=1 colspan=1>MP MRSA</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

End Point (EP) and Second Derivative Peak Abscissa (SDPA), two underlying numerical values used to determine a final assay result, were selected as additional means of assessing assay reproducibility.

Lot-to-Lot Reproducibility Study Underlying Numerical Results Across Lots, Days, Runs, and Replicates at One Site, for MREJ Target with MREJ Types Pooled (FAM Channel)1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>HN MRSA</td><td rowspan=1 colspan=1>LP MRSA</td><td rowspan=1 colspan=1>MP MRSA</td></tr><tr><td rowspan=25 colspan=1>EPSDPA</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>345.5</td><td rowspan=1 colspan=1>580.7</td><td rowspan=1 colspan=1>570.7</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>131.14</td><td rowspan=1 colspan=1>200.66</td><td rowspan=1 colspan=1>196.71</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>38.0%</td><td rowspan=1 colspan=1>34.6%</td><td rowspan=1 colspan=1>34.5%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>50.91</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8.8%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>34.69</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>5.22</td><td rowspan=1 colspan=1>52.86</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>9.3%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>135.65</td><td rowspan=1 colspan=1>207.08</td><td rowspan=1 colspan=1>203.69</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>39.3%</td><td rowspan=1 colspan=1>35.7%</td><td rowspan=1 colspan=1>35.7%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>30.8</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.34</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.13</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.37</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>1.2%</td></tr></table>

1Values shown are those obtained for the MREJ target in the samples that gave a $\mathrm { S A ^ { + } }$ . MRSA+ result

Lot-to-Lot Reproducibility Study Underlying Numerical Results Across Lots, Days, Runs, and Replicates at One Site, for mecA/mecC Target with MREJ Types Pooled (ROX Channel)1   

<table><tr><td></td><td>Category</td><td></td><td>HN MRSA</td><td>LP MRSA</td><td>MP MRSA</td></tr><tr><td colspan="1" rowspan="25">EPSDPA</td><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">1219.7</td><td colspan="1" rowspan="1">2041.2</td><td colspan="1" rowspan="1">2032.9</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">473.47</td><td colspan="1" rowspan="1">567.63</td><td colspan="1" rowspan="1">617.89</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">38.8%</td><td colspan="1" rowspan="1">27.8%</td><td colspan="1" rowspan="1">30.4%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">131.35</td><td colspan="1" rowspan="1">97.16</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">10.8%</td><td colspan="1" rowspan="1">4.8%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Lot</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">154.33</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">114.79</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">12.7%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">5.6%</td></tr><tr><td colspan="1" rowspan="1">Between Lot</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">276.63</td><td colspan="1" rowspan="1">78.75</td><td colspan="1" rowspan="1">111.11</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">22.7%</td><td colspan="1" rowspan="1">3.9%</td><td colspan="1" rowspan="1">5.5%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">584.61</td><td colspan="1" rowspan="1">581.25</td><td colspan="1" rowspan="1">638.21</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">47.9%</td><td colspan="1" rowspan="1">28.5%</td><td colspan="1" rowspan="1">31.4%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">N</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">31.1</td></tr><tr><td colspan="1" rowspan="1">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">0.50</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Between Run within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Between Day within Lot</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.14</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Between Lot</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">1.7%</td></tr></table>

1Values shown are those obtained for the mecA/mecC target in the samples that gave a $\mathrm { S A ^ { + } }$ . MRSA+ result

Lot-to-Lot Reproducibility Study Underlying Numerical Results Across Lots, Days, Runs, and Replicates at One Site, for nuc Target (VIC Channel)1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>HN MREJ Type ii</td><td rowspan=1 colspan=1>HN MSSA</td><td rowspan=1 colspan=1>LP MSSA</td></tr><tr><td rowspan=25 colspan=1>EPSDPA</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>412.4</td><td rowspan=1 colspan=1>698.1</td><td rowspan=1 colspan=1>1013.5</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>316.90</td><td rowspan=1 colspan=1>246.87</td><td rowspan=1 colspan=1>256.22</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>76.8%</td><td rowspan=1 colspan=1>35.4%</td><td rowspan=1 colspan=1>25.3%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>72.08</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>87.17</td><td rowspan=1 colspan=1>41.84</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>316.90</td><td rowspan=1 colspan=1>261.81</td><td rowspan=1 colspan=1>269.44</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>76.8%</td><td rowspan=1 colspan=1>37.5%</td><td rowspan=1 colspan=1>26.6%</td></tr><tr><td rowspan=1 colspan=1>cYanmaGellob</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>32.0</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.73</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>2.3%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>0.29</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.85</td><td rowspan=1 colspan=1>1.57</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>5.2%</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>2.4%</td></tr></table>

1Values shown are those obtained for the nuc target in the samples that gave a $\mathrm { S A ^ { + } }$ . MRSA- result

Lot-to-Lot Reproducibility Study Underlying Numerical Results Across Lots, Days, Runs, and Replicates at One Site, Sample Processing Control Value (Cy5.5 Channel)1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>HN MREJ Type ii</td><td rowspan=1 colspan=1>HN MSSA</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=25 colspan=1>EPSDPA</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>2758.0</td><td rowspan=1 colspan=1>2546.5</td><td rowspan=1 colspan=1>2877.1</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>565.84</td><td rowspan=1 colspan=1>339.82</td><td rowspan=1 colspan=1>555.97</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>20.5%</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>19.3%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>99.28</td><td rowspan=1 colspan=1>523.38</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>221.49</td><td rowspan=1 colspan=1>77.48</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>2.7%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>574.48</td><td rowspan=1 colspan=1>662.17</td><td rowspan=1 colspan=1>561.34</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>20.8%</td><td rowspan=1 colspan=1>26.0%</td><td rowspan=1 colspan=1>19.5%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>30.0</td></tr><tr><td rowspan=1 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>Between Run within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Between Day within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>1.5%</td></tr></table>

1Calculated for the Specimen Processing Control of the samples that gave a SA-. MRSA- result

The reproducibility study results met the pre-defined acceptance criteria for LP, MP, and TN samples (i.e., LP overall correct percentage of approximately $9 5 \%$ with $9 5 \%$ CI; MP overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI; TN overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI). No specific acceptance criteria were defined for the HN samples.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Positive and Negative External Controls

An External Positive Control is intended to monitor for substantial reagent failure while an External Negative Control is used to detect reagent or environmental contamination (or carry-over) from other specimens or SA or MRSA amplicons. External Control materials are not provided by BD in the BD $\mathbf { M A X T M }$ StaphSR assay. However, during the clinical trial, external controls were freshly prepared and tested per package insert with each group of samples tested on the BD MAX™ instrument. For each BD MAX™ StaphSR assay run, one positive and one negative external control were included. The positive control was one of the following two strains,

alternated daily (testing days): Reference MRSA strain, ATCC 43300, and methicillin-susceptible Staphylococcus aureus (MSSA) strain, ATCC 29213. The negative external control was prepared using Staphylococcus epidermidis strain, ATCC 12228. A failed control invalidated the entire run. In the case of a failure of either or both external controls, the testing of all samples included in the run was repeated from the appropriately stored SBTs along with a new set of external controls. In cases where external controls failed again, no additional repeat testing was performed.

During the prospective clinical study, a total of 140 external positive controls and 140 external negative controls were tested. Seven external positive controls (7/140, $5 . 0 \%$ , $9 5 \%$ CI, $2 . 4 \% \mathrm { ~ - ~ } 1 0 . 0 \%$ and eight external negative controls (8/140, $5 . 7 \%$ , $9 5 \%$ CI, $2 . 9 \% - 1 0 . 9 \% )$ failed initially, but succeeded on repeat testing.

Internal Sample Processing Control (SPC)

The Sample Processing Control incorporated into each Unitized Reagent Strip (URS) is intended to monitor PCR inhibition as well as reagent integrity for each sample tested. A failed SPC renders a sample result “Unresolved” (UNR). The clinical sites were instructed to repeat all UNR samples on the next run.

During the prospective clinical study, 15 specimens out of 2,399 total specimens (15/2399, $0 . 6 \%$ , $9 5 \%$ CI, $0 . 4 \% \mathrm { ~ - ~ } 1 . 0 \% )$ ) were reported as UNR after initial testing. Upon repeat testing, one specimen out of 2,399 total specimens (1/2399, $0 . 0 4 \%$ , $9 5 \%$ CI, $0 . 0 \% - 0 . 2 \%$ ) remained UNR.

# Nasal Matrix and Simulated Nasal Matrix Equivalency Study

Negative natural clinical nasal matrix is difficult to obtain due to the high prevalence of mecA/mecC and nuc genes present in normal bacterial nasal flora, in a proportion of $60 \%$ and $20 \mathrm { - } 3 0 \%$ , respectively. In order to compensate for the limited availability of negative natural clinical nasal matrix, a simulated nasal matrix was developed for the verification and validation studies of the BD MAX™ StaphSR assay.

An analytical study was carried out to demonstrate the equivalency between the natural clinical nasal matrix and simulated nasal matrix developed.

The composition of the simulated nasal matrix is the following:

Composition of the Simulated Nasal Matrix   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Quantity</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>2% (EMD) or 5% (Sigma-Aldrich) w/v</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Saline (0.85% NaCl)</td><td rowspan=1 colspan=1>0.8% v/v NaCl(95% Saline)</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>0.00002% w/v (200pg/ul)</td></tr></table>

Note that bovine mucin from two suppliers (EMD and Sigma-Aldrich) was used in the formulation of the simulated nasal matrix. During product development, SigmaAldrich was unable to supply mucin due to placing the product on backorder. Thus, the supplier was replaced by EDM. The suitable mucin concentration was determined to be $2 \%$ for the mucin from EMD and $5 \%$ for the mucin from Sigma- Aldrich. The simulated matrix containing $5 \%$ mucin from EMD was found to be inhibitory at $5 \%$ compared to the clinical nasal matrix.

An MRSA MREJ Type ii bacterial panel (a bacterial suspension corresponding to a target concentration of 2-3X LoD, 256 CFU/swab) was used in this study. Twelve replicates of negative natural clinical nasal matrix and simulated nasal matrix (mucin from EMD or Sigma-Aldrich) in SBT were tested using five BD MAX™ instruments.

Statistical analysis was performed. The Two One-Sided Test (TOST) procedure of Schuirmann (1987) was used for assessing the equivalence of the SDPA means. Rejection of the non-equivalence hypothesis in favor of the equivalence hypothesis at a significance level of $5 \%$ occurred only if the $90 \%$ confidence interval of the mean difference was contained completely within the equivalence margins which are listed in the table below:

Analytical Acceptability Criteria for BD MAX™ StaphSR Assay   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Analytical Acceptability Criteria</td></tr><tr><td rowspan=4 colspan=1>With and WithoutSimulated Nasal Matrix</td><td rowspan=1 colspan=1>FAM SDPA (Pos)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>ROX SDPA (Pos)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>VIC SDPA (Pos)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>Cy5.5 SDPA(Neg)</td><td rowspan=1 colspan=1>± 4%</td></tr></table>

Data from this study were used to perform TOST equivalency analyses for FAM and VIC mean SDPA values between clinical matrix and simulated matrix $5 \%$ and $2 \%$ mucin supplied by Sigma-Aldrich and EMD, respectively). Data from the ROX channel were not considered for the clinical matrix condition comparison, as the mecA/mecC genes (detected in ROX channel) are present in a normal bacterial nasal flora. However, the ROX channel was considered in the comparison of the simulated matrix composition.

The mean SDPA obtained for the simulated and natural clinical matrices in the FAM and VIC channels ( $5 \%$ Sigma-Aldrich mucin and $2 \%$ EMD mucin) are equivalent based on meeting the Analytical Acceptability Criteria described above. The comparison of the FAM, VIC and ROX mean SDPA of the two simulated nasal matrix composition showed that the simulated matrix containing $2 \%$ EMD mucin is also equivalent to the simulated matrix containing $5 \%$ Sigma-Aldrich mucin based on meeting the Analytical Acceptability Criteria described above.

In conclusion, this study demonstrated that both simulated nasal matrices $5 \%$ SigmaAldrich and $2 \%$ EMD mucin, respectively) developed for the BD MAX™ StaphSR assay are equivalent to each other and are equivalent to negative natural clinical nasal matrix. Therefore, the simulated nasal matrices are appropriate and adequate to be used in place of natural nasal matrix for the verification and validation studies of the BD MAX™ StaphSR assay.

# Preservation of DNA on Collection Device

The preservation of amplifiable DNA on Liquid Stuart swabs was evaluated in an analytical study.

Twelve (12) methicillin-resistant $S .$ . aureus (MRSA) strains, 18 methicillin-sensitive S. aureus (MSSA) strains, and 30 methicillin-susceptible Staphylococcus sp. (MSSsp) strains were used in this study. The MRSA strains were of diverse geographical origin and included MREJ types i, ii, iii, iv, v, vi, vii, ix, xiii, xiv, and xxi. For MRSA and MSSA strains, a mix of $7 5 \mu \mathrm { L }$ of simulated matrix and 7.5 ul of a bacterial suspension corresponding to a target concentration of 2-3X LoD was used to determine the preservation of amplifiable DNA on Liquid Stuart swabs. For MSSsp strains, a mix of $7 5 \mathrm { u l }$ of simulated matrix and $7 . 5 { \mathrm { u l } }$ of a bacterial suspension corresponding to a theoretical concentration of $1 . 3 0 \mathrm { E } { + } 0 6 $ CFU/mL was tested.

Each collection (Liquid Stuart swab) was tested with 12 MRSA, 18 MSSA, and 30 MSSsp strains at baseline and under four different storage conditions using five different instruments. The Liquid Stuart swabs were used to completely absorb 82.5 $\mu \mathrm { L }$ of the relevant bacterial suspension; the swabs were then inserted into their respective transport tubes and stored under the following conditions:

room temperature for at least 5 minutes $( \mathrm { T } { = } 0 )$ )   
$2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 48 hours   
$2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 60 hours   
$2 { - } 8 ^ { \circ } \mathrm { C }$ for 120 hours (5 days)   
$2 { - } 8 ^ { \circ } \mathrm { C }$ for 150 hours

Samples were then analyzed with the BD MAX™ StaphSR assay.

To be considered acceptable, the percentage of conforming results had to be equal to or greater than $9 5 \%$ .

The percentage of conforming results met the acceptance criteria $( \geq 9 5 \% )$ , under all storage conditions, for all strains tested.

In conclusion, this analytical study demonstrated that DNA from MRSA and MSSA strains absorbed on Liquid Stuart swabs is stable up to 60 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ and up to 150 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ . DNA from MSSsp strains absorbed on Liquid Stuart swabs gave the expected negative result up to 60 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ and up to 150 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# Lysis Efficiency

The lysis efficiency characterization was presented in the BD MAX™ MRSA Assay 510(k) (K120138). A lysis efficiency of $\geq 9 7 \%$ was obtained for the BD MAX™ MRSA Assay. Since the lysis strategy has not changed for the BD MAX™ StaphSR assay, no new lysis efficiency analytical study was performed.

# Preservation of DNA in Sample Buffer Tube (SBT)

The preservation of amplifiable DNA in Sample Buffer Tube (SBT) was evaluated in an analytical study.

Twelve (12) methicillin-resistant S. aureus (MRSA) strains, 18 methicillin-sensitive S. aureus (MSSA) strains, and 30 methicillin-susceptible Staphylococcus sp. (MSSsp) strains were used in this study. The MRSA strains were of diverse geographical origin and included MREJ types i, ii, iii, iv, v, vi, vii, ix, xiii, xiv, and xxi. For MRSA and MSSA strains, a bacterial suspension corresponding to a target concentration of 2-3X LoD was used to determine the preservation of amplifiable DNA in the SBT. For MSSsp strains, a bacterial suspension corresponding to a theoretical concentration of $1 . 3 0 \mathrm { E } { + } 0 6 $ CFU/mL was tested.

The study was performed with 12 MRSA, 18 MSSA, and 30 MSSsp strains at baseline and under four different storage conditions using five different instruments. The Liquid Stuart swabs were used to completely absorb $7 5 \mu \mathrm { L }$ of the relevant bacterial suspension; the swabs were then inserted into their respective transport tubes for five minutes. The swabs were then broken off into SBTs (three lots) containing simulated nasal matrix. The SBTs containing the swabs were stored under the following conditions:

room temperature for at least 5 minutes $( \mathrm { T } { = } 0 )$ $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 36 hours $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 45 hours · $2 { - } 8 ^ { \circ } \mathrm { C }$ for 120 hours (5 days) $2 { - } 8 ^ { \circ } \mathrm { C }$ for 150 hours

Samples were then analyzed with the BD MAX™ StaphSR assay.

To be considered acceptable, the percentage of conforming results had to be equal to or greater than $9 5 \%$ . Results were calculated for each time point separately.

The percentage of conforming results met the acceptance criteria $( \geq 9 5 \% )$ , under all storage conditions, for all strains tested.

In conclusion, this analytical study demonstrated that DNA from MRSA and MSSA strains is stable up to 45 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ and up to 150 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ after sample elution in SBT. DNA from MSSsp strains gave the expected negative result up to 45 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ and up to 150 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ after sample elution in SBT.

# Recovery of Methicillin-Resistant S. aureus Cells in Sample Buffer Tube

An analytical study was carried out to demonstrate that MRSA remains viable when stored 36 to 45 hours after a BD $\mathbf { M A X T M }$ run (3 hours at $2 5 ^ { \circ } \mathrm { C }$ ) followed by incubation at $2 { - } 8 ^ { \circ } \mathrm { C }$ in the Sample Buffer Tube (SBT).

Three simulated nasal matrix lots were used in combination with three different lots of Sample Buffer Tubes (SBT). Three MRSA reference strains representing MREJ Type ii, iiv, and ii (PFGE USA300) were used in this study.

Cultures were prepared and diluted in order to obtain suspensions consistent with 2- 3X LoD of the BD MAX™ StaphSR assay. The concentration was confirmed by bacterial count. For each strain, $7 5 \mu \mathrm { L }$ of cell suspension were distributed in $2 \mathrm { m L }$ tubes and absorbed with a swab.

Six swabs were spiked by absorbing a bacterial suspension of $7 5 \mu \mathrm { L }$ (3 for 36 hours and 3 for 45 hours). After the complete volume of sample was absorbed with a collection swab, each swab was reinserted into its storage device and incubated for at least five minutes at room temperature. Each swab was then broken in its respective SBT (three lots) containing simulated nasal matrix. Tubes were closed and vortexed for 1 minute. For each sample and incubation time (36 and 45 hours), a time zero point was established using $6 0 0 \mu \mathrm { L }$ of the suspension dispensed in $2 \mathrm { m L }$ tubes preincubation. All samples were then incubated for 3 hours at $2 5 \mathrm { { ^ \circ C } }$ followed by 36 or 45 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

To determine viability, two culture methods were used; direct culture and enriched culture. The direct culture was performed by plating $5 0 ~ \mu \mathrm { L }$ of samples on each of the six CHROMagar™ MRSA plates; mauve-colored colonies were considered as MRSA positive. If plates were MRSA negative after 24 hours, they were incubated for another 24 hours. The enriched culture was executed by adding $3 0 0 \mu \mathrm { L }$ of samples in a TSB NaCl $6 . 5 \%$ tube. Cultures with growth were then processed by following the same steps used for the direct culture. If no growth was observed, tubes were incubated for another 24 hours at $3 5 \mathrm { { ^ \circ C } }$ followed by the same procedure. If MRSA is isolated from three Direct Culture plates, the Enriched Culture may be halted.

MRSA viability was determined by the presence of MRSA on a minimum of one out of six CHROMagar™ MRSA plates per strain by either direct or enriched culture method.

Acceptance criteria were met all storage conditions. Growth was observed on at least one out of six CHROMagar™ MRSA plates.

In conclusion, this analytical study demonstrated that MRSA (at 2-3X LoD of the BD MAX™ StaphSR assay) remains viable after storage of 3 hours at $2 5 \mathrm { { ^ \circ C } }$ followed by 36 to 45-hours incubation at $2 { - } 8 ^ { \circ } \mathrm { C }$ in the SBT.

# Reagents Shelf Life Determination

The stability of the BD MAX™ StaphSR assay stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 \pm 2 ^ { \circ } \mathrm { C }$ was evaluated in real-time. Testing was performed to determine the stability of the BD MAX™ StaphSR kit and the lysis efficiency of the extraction tube over time.

Three lots of kit components (strips, Master Mix, Sample Buffer and Extraction Tubes) were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 \pm 2 ^ { \circ } \mathrm { C }$ . A subset of the first lot was subjected to simulated shipping conditions prior to storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 \pm 2 ^ { \circ } \mathrm { C }$ . Components were removed from storage at pre-determined time points for testing. Trend of the PCR metric curves (in each channel) over time was analyzed by a linear regression model and used to determine a stability breaking point. A stability breaking point is defined as time point when the confidence interval (CI) of the linear regression crosses the Allowable Drift Limits (ADL). Tolerance intervals with $9 5 \%$ confidence and $9 9 . 7 3 \%$ coverage of the means were used to set the ADL empirically. The tolerance interval assessment was based on a total sample size of 1088 for the POS PCR curve parameters (SDPA and LogEP) and on a total sample size of 978 for the NEG PCR curve parameters (SDPA and LogEP) tested at baseline $( \mathrm { T } { = } 0 )$ ). If no significant degradation (i.e., no stability breaking point is identified) occurred over time, the kit stability corresponds to the last tested time point.

Three lots of kit Extraction Tubes were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 \pm 2 ^ { \circ } \mathrm { C }$ . A subset of the first lot was subjected to simulated shipping conditions prior to storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 \pm 2 ^ { \circ } \mathrm { C }$ . Components were removed from storage at pre-determined time points for testing. The extent of the lysis was measured by optical density and the lysis percentage was calculated. This percentage was used to determine a stability breaking point. A stability breaking point is reached when at least one out of three lots shows failed results for two consecutive time points.

Shipping simulation consisted of storage in extreme conditions variations for up to 11 days. Temperature ranged from $- 1 0 \mathrm { { ^ \circ C } }$ to $4 0 \%$ and humidity varied from $10 \%$ to 70- $7 5 \%$ RH.

BD MAX™ StaphSR kit stability and Extraction Tubes lysis efficiency tests were taken into account to determine the shelf-life but were analyzed independently. Overall kit shelf life claim is based on the least stable lot tested for the kit stability or for the lysis efficiency (i.e., lot of BD MAX™ StaphSR or lot of Extraction Tube).

The stability study is currently ongoing and available real-time data supports a shelf life of up to five months at $2 \mathrm { - } 8 ^ { \circ } \mathrm { C }$ and/or $2 5 \pm 2 ^ { \circ } \mathrm { C }$ when Master Mix and Extraction Tubes are stored in sealed bags. After bag opening, real-time data supports a shelf life of seven days when Master Mix and Extraction Tubes are stored at $2 \mathrm { - } 8 ^ { \circ } \mathrm { C }$ or at $2 5 \pm$

$2 \mathrm { { } ^ { \circ } C }$ . All other kit components are stored unsealed as per kit configuration. Simulated shipping does not impact shelf life.

# Un-reconstituted Reagents Stability outside of Protective Pouch

An analytical study was carried out to demonstrate the stability of the sealed reagents (Master Mix and Extraction Tube) when stored for five hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ or $2 5 \pm 2 ^ { \circ } \mathrm { C }$ outside of the protective pouch.

One methicillin-resistant $S$ .aureus (MREJ type ii) strain and one methicillin-sensitive $S$ .aureus strain were used in this study. Bacterial suspensions corresponding to a target concentration of 3X LoD were tested in this study. The study was performed using more than five different instruments.

The Master Mix and the Extraction Tube were stored under the following conditions:

$2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 5 hours $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 6.25 hours $2 { - } 8 ^ { \circ } \mathrm { C }$ for 5 hours $2 { - } 8 ^ { \circ } \mathrm { C }$ for 6.25 hours

The swabs were used to absorb $7 5 \mu \mathrm { L }$ of the relevant bacterial suspension and then broken off into SBTs containing simulated nasal matrix. Samples were then analyzed with the BD MAX™ StaphSR assay.

To be considered acceptable, the percentage of conforming results had to be equal to or greater than $9 5 \%$ .

The percentage of conforming results met the acceptance criteria $( \geq 9 5 \% )$ , under all storage conditions, for all strains tested.

In conclusion, this analytical study demonstrated that un-reconstituted sealed reagents (Master Mix and Extraction Tube) outside of the protective pouch are stable when stored for up to 5 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ or at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# d. Detection limit:

Limit of Detection (LoD) for the BD MAX™ StaphSR assay was determined in an analytical study. Positive specimens were prepared by soaking swabs in a wide range of MRSA or MSSA bacterial suspensions prepared and quantified from cultures. The tested strains included 11 MRSA strains representing 11 MREJ genotypes (i, ii, iii, iv, v, vi, vii, ix, xiii, xiv and xxi) corresponding to five SCCmec types (I, II, III, IV and XI), as well as two MSSA strains. The swabs were then eluted in simulated nasal matrix (Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details). Each MRSA and MSSA strain was tested in replicates of 24 per concentration by two different operators using three different production lots of the BD MAX™

StaphSR assay. LoDs, defined as the lowest concentration at which $9 5 \%$ of all replicates tested positive, were determined using a statistically based methodology. This methodology allows determination of the LoD with a $9 5 \%$ confidence interval. Specifically, the LoD results were determined using a statistical linear logistic model that describes the relationship between the probability of the response and the bacterial concentration. This method models the positive response (expressed in percentage) as a function of Log (CFU/swab). The logistic model equation for the fitted curve allows the computation of the LoD by inverse prediction using the parameter estimates and their $9 5 \%$ confidence interval.

Limit of Detection of MRSA Genotypes by the BD MAX™ StaphSR Assay   

<table><tr><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>MREJ Genotype</td><td rowspan=1 colspan=1>SCCmec type</td><td rowspan=1 colspan=1>LoD Concentration [CFU/swab (95% C14)]</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Type i</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>84(49, 142)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Type ii</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>103(64, 167)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Type iii</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>160 (93, 278)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Type iv</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>68(42, 109)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Type v</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>128(73, 225)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Type vi</td><td rowspan=1 colspan=1>ND2</td><td rowspan=1 colspan=1>343 (186, 632)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Type vii</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>219 (110, 439)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Type ix</td><td rowspan=1 colspan=1>ND2</td><td rowspan=1 colspan=1>144(82, 255)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Type xiii</td><td rowspan=1 colspan=1>ND2</td><td rowspan=1 colspan=1>64(36, 114)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Type xiv</td><td rowspan=1 colspan=1>ND{2</td><td rowspan=1 colspan=1>78(48, 127)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Type xxi3</td><td rowspan=1 colspan=1>XI</td><td rowspan=1 colspan=1>112(64, 197)</td></tr></table>

1SCCmec type does not correlate to the MREJ type as these are two different typing methods. ${ } ^ { 2 } \mathrm { N D } =$ not determined 3mecC-containing MRSA strains (Also known as $m e c A _ { L G A 2 5 I }$ strain) 4Confidence Interval

Limit of Detection of MSSA by the BD MAX™ StaphSR Assay   

<table><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Concentration [CFU/swab (95% C1) 1]</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 29213</td><td rowspan=1 colspan=1>174 (89, 341)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 19095</td><td rowspan=1 colspan=1>211 (105, 428)</td></tr></table>

1Confidence Interval

# e. Analytical reactivity:

An analytical inclusivity study was performed using a variety of MRSA and MSSA strains, taking into account geographic origin, MREJ genotype (wild type and mutant), SCCmec type, Pulsed-Field Gel Electrophoresis (PFGE) type, temporal diversity and susceptibility pattern. Seventy-seven (77) MRSA strains from 27 countries and 51 MSSA strains from 16 countries were tested at a concentration equivalent to 2-3X LoD in this study, including strains from public collections and from well-characterized clinical isolates, including Vancomycin-Resistant Staphylococcus aureus (VRSA) and Vancomycin Intermediate Staphylococcus aureus (VISA) strains.

<table><tr><td colspan="1" rowspan="1">Collection</td><td colspan="1" rowspan="1">ReferenceNumber</td><td colspan="1" rowspan="1">Geographic Origin</td><td colspan="1" rowspan="1">MREJ Type</td><td colspan="1" rowspan="1">SCCmecTyping/ PFGEType</td><td colspan="1" rowspan="1">Year of theStrain'sIsolation/YearofFreezing</td><td colspan="1" rowspan="1">Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td colspan="1" rowspan="8">ATCC</td><td colspan="1" rowspan="1">ATCC BAA-</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">USA1000</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-42</td><td colspan="1" rowspan="1">Portugal (Lisbon)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">VI</td><td colspan="1" rowspan="1">1996</td><td colspan="1" rowspan="1">SA+, MRSA-1</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-38</td><td colspan="1" rowspan="1">Denmark (Gentofte)</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1960's</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-41</td><td colspan="1" rowspan="1">USA (NY city)</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1994</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-39</td><td colspan="1" rowspan="1">Hungary (Dunaújváros)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-40</td><td colspan="1" rowspan="1">Portugal (Lisbon)</td><td colspan="1" rowspan="1">iv</td><td colspan="1" rowspan="1">III</td><td colspan="1" rowspan="1">1994</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">USA, Kansas</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ATCC 33592</td><td colspan="1" rowspan="1">USA (NY city)</td><td colspan="1" rowspan="1">iv</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">1979</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="4">HarmonyCollection ofEuropeanEpidemicMRSA</td><td colspan="1" rowspan="1">62305</td><td colspan="1" rowspan="1">Finland (NK)</td><td colspan="1" rowspan="1">ii mut36</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">1990</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">97S99</td><td colspan="1" rowspan="1">Belgium (Brussels)</td><td colspan="1" rowspan="1">ii mut45</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">1992</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">3717</td><td colspan="1" rowspan="1">Greece (Athens)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">1994</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">9805-01937</td><td colspan="1" rowspan="1">Sweden (Orebro)</td><td colspan="1" rowspan="1">iii mut45</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">1998</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="2">LSPQ</td><td colspan="1" rowspan="1">ID-61882</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">III / CMRSA-</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">ID-61880</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">II / CMRSA-1</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="17">NARSA</td><td colspan="1" rowspan="1">NRS383</td><td colspan="1" rowspan="1">USA (North Carolina)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">II / USA200</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS385</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV /USA500</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS715</td><td colspan="1" rowspan="1">USA (New York)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">II/USA600</td><td colspan="1" rowspan="1">2006</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS386</td><td colspan="1" rowspan="1">USA (Louisiana)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA700</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS686</td><td colspan="1" rowspan="1">USA (Georgia)</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">IV/IBERIAN</td><td colspan="1" rowspan="1">2006</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS234</td><td colspan="1" rowspan="1">USA (California)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">VRS53</td><td colspan="1" rowspan="1">USA (Michigan)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS14</td><td colspan="1" rowspan="1">Japan</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">¤</td><td colspan="1" rowspan="1">1996</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS44</td><td colspan="1" rowspan="1">USA (New Jersey)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">Ⅱ</td><td colspan="1" rowspan="1">1997</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">VRS23</td><td colspan="1" rowspan="1">USA (Pennsylvanie)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">VRS43</td><td colspan="1" rowspan="1">USA (Michigan)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA-1</td></tr><tr><td colspan="1" rowspan="1">NRS382</td><td colspan="1" rowspan="1">USA (Ohio)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">II/USA100</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS384</td><td colspan="1" rowspan="1">USA (Mississipi)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA300</td><td colspan="1" rowspan="1">1998</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS387</td><td colspan="1" rowspan="1">USA (Washington)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV /USA800</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS484</td><td colspan="1" rowspan="1">USA (Alaska)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA1100</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS645</td><td colspan="1" rowspan="1">USA (California )</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV/IBERIAN</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NRS123</td><td colspan="1" rowspan="1">USA (North Dakota)</td><td colspan="1" rowspan="1">ii mut36</td><td colspan="1" rowspan="1">IV / USA400</td><td colspan="1" rowspan="1">1998</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="11">WellCharacterizedClinical Strains</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Baltimore)</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">II / USA 100</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Baltimore)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">II /USA 100</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">II /USA100</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV /USA300</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA300</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA300</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA300</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV / USA300</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">ii mut36</td><td colspan="1" rowspan="1">IV /USA400</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Baltimore)</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">IV /USA 800</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA}$</td><td colspan="1" rowspan="1">France, Aix en Provence</td><td colspan="1" rowspan="1">xxi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2006</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA$</td><td colspan="1" rowspan="1">Denmark</td><td colspan="1" rowspan="1">xxi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (Miami)</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">VII</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA, Washington,</td><td colspan="1" rowspan="1">xiii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA, Delaware,</td><td colspan="1" rowspan="1">xiv</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Mexico</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Israel</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">South Africa</td><td colspan="1" rowspan="1">ii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+ 2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">United Kingdom</td><td colspan="1" rowspan="1">ii mut36</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+ 2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Netherlands (Den Bosch)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA (New Jersey)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Italy</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">China (Hong Kong)</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Turkey</td><td colspan="1" rowspan="1">iv</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Denmark (Copenhague)</td><td colspan="1" rowspan="1">iv</td><td colspan="1" rowspan="1">III</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Canada (Toronto)</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">1996</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Australia</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">vi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Norway</td><td colspan="1" rowspan="1">vi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Brazil</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+ 2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Singapore</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Canada (Toronto)</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Albania (Tirana)</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">vi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Poland</td><td colspan="1" rowspan="1">ix</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">xiii</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA5</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">xxi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2008</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Germany, Tübingen</td><td colspan="1" rowspan="1">xiv</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2008</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Denmark</td><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA\}$</td><td colspan="1" rowspan="1">United Kingdom,</td><td colspan="1" rowspan="1">xxi</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA5</td><td colspan="1" rowspan="1">United Kingdom, England,Somerset</td><td colspan="1" rowspan="1">xxi</td><td colspan="1" rowspan="1">XI</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">iii</td><td colspan="1" rowspan="1">III</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Canada (Québec)</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">United Kingdom</td><td colspan="1" rowspan="1">vii</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">2002</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Austria</td><td colspan="1" rowspan="1">ii mut36</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA+</td></tr></table>

1The initial result was negative for MRSA but positive for SA. Both samples were repeated from the SBT and assay results are conforming $( \mathbf { S } \mathbf { A } ^ { + } ;$ , ${ \mathrm { M R S A ^ { + } } }$ ). 2These are the results for the repeats as the initial run gave an IND result due to a PCR heater warning. 3VRSA strains 4VISA strains 5mecC variant strains

Analytical Reactivity Study Results Testing MSSA Strains at 2-3X LOD   

<table><tr><td colspan="1" rowspan="1">Genera and Species</td><td colspan="1" rowspan="1">Geographic Origin[Country (City)]</td><td colspan="1" rowspan="1">Year of theStrain'sIsolation/Yearof Freezing</td><td colspan="1" rowspan="1">Assay ResultObtained withBD MAXTMStaphSR Assay</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Halifax)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Montréal)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Quebec)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Quebec)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Longueuil)</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Ste-Foy)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Edmonton)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Toronto)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Vancouver)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Longueuil)</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Montréal)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Montréal)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Rimouski)</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Regina)</td><td colspan="1" rowspan="1">2005</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Quebec)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Edmonton)</td><td colspan="1" rowspan="1">1994</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Ottawa)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Halifax)</td><td colspan="1" rowspan="1">1993</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada (Toronto)</td><td colspan="1" rowspan="1">2001</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Germany</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Germany</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Greece</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ireland</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ireland</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Israel</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Italy</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Italy</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Japan</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Netherland (Rotterdam)</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Poland</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Spain</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Turkey</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">UK</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Iowa)</td><td colspan="1" rowspan="1">1986</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Chicago)</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Atlanta)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Connecticut)</td><td colspan="1" rowspan="1">2004</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Wisconsin)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (New York)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (St-Louis)</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">USA (Washington)</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(empty cassette variant )</td><td colspan="1" rowspan="1">Germany</td><td colspan="1" rowspan="1">2008</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr></table>

In conclusion, 51 out of 51 MSSA strains tested with the BD $\mathbf { M A X T M }$ StaphSR assay were detected at 2-3X LoD. The assay correctly identified 75 of the 77 MRSA strains tested with the BD MAX™ StaphSR assay at 2-3X LoD. Two strains produced initially false negative MRSA result and were found to be MRSA positive after repeat testing from the SBT.

The assay detected:

MRSA including: - MREJ types i, ii, iii, iv, v, vi, vii, ix, xiii, xiv, xxi (wild and mutant) - mecA/mecC variant gene - SCCmec types I, II, III, IV, V, VI, VII, VIII, XI - PFGE MRSA USA 100 to 800, 1000 and 1100 - Strains isolated over 49 years (ranging from 1960 - 2009) representing temporal diversity - Strains representing 27 countries, geographical diversity - Strains displaying additional resistance to Vancomycin (VISA and VRSA)   
MSSA including: - mecA empty cassette variant strains - Strains isolated over 22 years (ranging from 1986 - 2008) representing temporal diversity - Strains representing 16 countries, geographical diversity

# f. Challenge Study:

An additional analytical study was carried out to evaluate the analytical performance of the BD MAX™ StaphSR assay using a well characterized challenge strain panel. The objective of this study was to determine the assay results obtained with a challenge strain panel containing MRSA strains with high and low oxacillin MICs, including PFGE types USA 100 to 800, 1000, PFGE type IV/IBERIAN and mecC variant LGA251 strain, BORSA, MSSA and methicillin-resistant Staphylococcus epidermidis (MRSE) strains using the BD MAX™ StaphSR assay.

The challenge strain panel used in this study was composed of 17 MRSA, four BORSA, one MRSE and five MSSA strains. All these strains were tested with FDAcleared methods for determination of the MIC. The MRSA strains were all tested using the BD MAX™ StaphSR assay at 2-3X LoD. The MSSA, BORSA and MRSE (non-MRSA) strains were all tested at high concentrations (approximately $1 0 ^ { 6 }$ CFU/swab).

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>PFGE Type</td><td rowspan=1 colspan=1>MIC Level ofResistance</td><td rowspan=1 colspan=1>Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td rowspan=17 colspan=1>MRSA</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>USA 100</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>USA 200</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=5 colspan=1>USA 300</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>USA 400</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>USA 500</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>USA 600</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>USA 700</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>USA 800</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>USA 1000</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>IBERIAN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr><tr><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SA+, MRSA+</td></tr></table>

1mecC variant LGA251 strain

Challenge Study Results Testing MRSE, MSSA, and BORSA Strains at $1 0 ^ { 6 }$ CFU/swab or Higher

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td rowspan=1 colspan=1>MRSE</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=5 colspan=1>MSSA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=4 colspan=1>BORSA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr></table>

In conclusion, all strains tested as part of the Challenge Study exhibited the expected results with the BD $\mathbf { M A X T M }$ StaphSR assay. All MRSA strains tested as part of the Challenge Study with the BD MAX™ StaphSR assay exhibited $\mathrm { S A ^ { + } }$ , ${ \mathrm { M R S A ^ { + } } }$ results when tested at 2-3X LoD concentrations. The MRSE strain tested exhibited SA-, MRSA- and all BORSA and MSSA strains tested exhibited $\mathrm { S A ^ { + } } _ { \mathrm { i } }$ , MRSA- results when tested at high concentrations.

# g. Analytical Specificity:

The BD MAX™ StaphSR assay was performed on samples containing high levels of non-target organisms and MSSA strains, using the BD MAX™ System, to demonstrate the specificity of the assay for detection of MRSA and SA. The analytical specificity study testing included the following:

Fifteen (15) empty cassette variant MSSA strains tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / s w a l }$ b Fifty-seven (57) strains of various non-staphylococcal species tested at a concentration of $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ (except for Cryptococcus neoformans which was tested at $3 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U } / \mathrm { s w } \mathrm { : }$ ab, the highest concentration achieved) Forty-five (45) Coagulase-Negative staphylococcal strains (CoNS) and Coagulase-Positive staphylococcal strains (CoPS) representing 28 species were tested at a concentration of $0 . 5 \mathrm { M c F }$ arland with the BD MAX™ StaphSR assay.   
· Fifty (50) MSSA strains tested at high concentrations $( \geq 1 0 ^ { 6 }$ CFU/swab)   
· Seventeen (17) viruses representing 12 different viral species tested at $\geq 1 0 ^ { 5 }$ PFU/mL

BD MAX™ StaphSR Assay Analytical Specificity Study Results with Empty CassetteVariant MSSA Strains   
BD MAX™ StaphSR Assay Analytical Specificity Study Results with Various Non-Staphylococcal Species   

<table><tr><td rowspan=1 colspan=1>Genera and Species</td><td rowspan=1 colspan=1>City/State of Origin</td><td rowspan=1 colspan=1>Country ofOrigin</td><td rowspan=1 colspan=1>MREJType</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>New Haven, CT</td><td rowspan=1 colspan=1>USA</td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Charlottesville, Virginia</td><td rowspan=1 colspan=1>USA</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Providence, RhodeIsland</td><td rowspan=1 colspan=1>USA</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Geneva</td><td rowspan=1 colspan=1>Switzerland</td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Seattle, WA</td><td rowspan=1 colspan=1>USA</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Terre Haute, IN</td><td rowspan=1 colspan=1>USA</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Greifswald</td><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Leipzig</td><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>vii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Leipzig</td><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>xiii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Wurtzbürg</td><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Australia</td><td rowspan=1 colspan=1>vii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Bolton</td><td rowspan=1 colspan=1>UK</td><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Barnet, Herts</td><td rowspan=1 colspan=1>UK</td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Barnet, Herts</td><td rowspan=1 colspan=1>UK</td><td rowspan=1 colspan=1>vii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>China</td><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SA+, MRSA-</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Genera and species</td><td colspan="1" rowspan="1">Gram Stain</td><td colspan="1" rowspan="1">Assay Result Obtainedwith BD MAXTMStaphSR Assay</td></tr><tr><td colspan="1" rowspan="1">Acinetobater baumannii</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Acinetobater haemolyticus</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium aquaticus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Enterrococcus gallinarum</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA--</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus suis</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sp.</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. Enterica</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr></table>

1Gram Stain Result: $\Nu =$ Gram Negative, $\mathrm { P } =$ Gram Positive, $\mathrm { N A } = \mathrm { N o t }$ Available

BD MAX™ StaphSR Assay Analytical Specificity Study Results with Various CoPS Species   

<table><tr><td rowspan=1 colspan=1>Genera and Species</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Country of Origin</td><td rowspan=1 colspan=1>Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus delphini</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lutrae</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lutrae</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuspseudointermedius</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi subspcoagulans</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr></table>

BD MAX™ StaphSR Assay Analytical Specificity Study Results with Various CoNS Species   

<table><tr><td colspan="1" rowspan="1">Genera and Species</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">Country of Origin</td><td colspan="1" rowspan="1">Assay ResultObtained with BDMAXTM StaphSRAssay</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii subsp.urealyticum</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">China</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus felis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Japan</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Belgium</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">China</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Denmark</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Denmark</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis subsp.hominis</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus kloosii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pulvereri</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus sciuri</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">China</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">UK</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">SA-, MRSA-</td></tr></table>

BD MAX™ StaphSR Assay Analytical Specificity Study Results with MSSA Strains   

<table><tr><td colspan="1" rowspan="1">Genera and Species</td><td colspan="1" rowspan="1">City/State oforigin</td><td colspan="1" rowspan="1">Country ofOrigin</td><td colspan="1" rowspan="1">Assay Result Obtainedwith BD MAXTMStaphSR Assay</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Halifax</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Montreal</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Quebec</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Quebec</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Longueuil</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ste-Foy</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Edmonton</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Vancouver</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Longueuil</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Montreal</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Rimouski</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Regina</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Québec</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Edmonton</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Ottawa</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Halifax</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Germany</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Germany</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Greece</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Ireland</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Ireland</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Israel</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Italy</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Italy</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Japan</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Roterdam</td><td colspan="1" rowspan="1">Netherland</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Poland</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Spain</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">=</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Turkey</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">UK</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Iowa</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Atlanta</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Connecticut</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Wisconsin</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">New York</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">St-Louis</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Washington</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Chicago</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Montreal</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">SA+, MRSA-</td></tr></table>

BD MAX™ StaphSR Assay Analytical Specificity Study Results with Various Viral Strains   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Type/Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Assay Result Obtainedwith BD MAXTMStaphSR Assay</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>1x106.622 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 7A</td><td rowspan=1 colspan=1>1x108.73 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>1x107.933 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>1x107.777 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>AD-169</td><td rowspan=1 colspan=1>1x106.98TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3.1x105 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>B95-8</td><td rowspan=1 colspan=1>3.00x10 cp/mL1</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human influenza virus</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>4.6x105 PFU/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human influenza virus</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>1.7x105 PFU/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>1x107.06 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza</td><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>1x107.29 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza</td><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>4.27x105 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3.2x107 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1x106.10 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1x109.44 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus</td><td rowspan=1 colspan=1>Type B</td><td rowspan=1 colspan=1>5.37x105 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1A</td><td rowspan=1 colspan=1>1x106.10 TCID50 units/mL</td><td rowspan=1 colspan=1>SA-, MRSA-</td></tr></table>

1The Epstein-Barr virus usually produces lysogenic infection and was therefore quantified in DNA cp/mL.

In conclusion, all organisms tested by the BD $\mathbf { M A X T M }$ StaphSR assay in this Analytical Specificity Study at the concentrations indicated showed expected results. The observed analytical specificity in this study was $100 \%$ .

# h. Potentially Interfering Substances:

An analytical study was performed to assess the potential inhibitory effects of biological and chemical materials that may be present on nasal swab specimens on the BD MAX™ StaphSR assay.

Biological and chemical substances occasionally used in the nares or found on nasal swab specimens were evaluated and are listed in the table below:

Substances and Organisms Tested in the Potentially Interfering Substances Study   

<table><tr><td colspan="1" rowspan="1">Brand Name or Description</td><td colspan="1" rowspan="1">State</td><td colspan="1" rowspan="1">Purpose</td><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">Quantity or Percentageof Active Substance perSwab</td></tr><tr><td colspan="1" rowspan="1">Mucin, from bovine submaxillaryglands</td><td colspan="1" rowspan="1">Liquid (oncerehydrated)</td><td colspan="1" rowspan="1">Simulation of nasalsecretion/mucus</td><td colspan="1" rowspan="1">EMB(Calbiochem)</td><td colspan="1" rowspan="1">4.65 x10-3 g</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone Sodium PhosphateOphthalmic Solution USP, 0.1%Dexamethasone Phosphate Equivalent(Sterile)</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">Ophtalmic drops1</td><td colspan="1" rowspan="1">Bausch &amp; LombIncorporated</td><td colspan="1" rowspan="1">7.5x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Chloraseptic</td><td colspan="1" rowspan="1">Lozenge (liquidonce dissolved)</td><td colspan="1" rowspan="1">Oral anesthetic/analgesiclozenges</td><td colspan="1" rowspan="1">Manufactured forPrestige Brands,Inc.</td><td colspan="1" rowspan="1">Benzocaine: 9.0x10-6 gMenthol: 1.5x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Taro-Mupirocin, MupirocinOintment USP, 2%</td><td colspan="1" rowspan="1">Gel</td><td colspan="1" rowspan="1">Topical nasal antibiotic</td><td colspan="1" rowspan="1">TaroPharmaceuticalsInc.</td><td colspan="1" rowspan="1">2.7x10-2 g ofMupirocin USP 2%</td></tr><tr><td colspan="1" rowspan="1">Long Lasting DristanNasal Mist</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Temporary relief of nasalcongestion</td><td colspan="1" rowspan="1">WyethConsumerHealthcare Inc.</td><td colspan="1" rowspan="1">3.8x10-5g</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine®</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Nasal decongestant</td><td colspan="1" rowspan="1">BayerHealthcare LLC</td><td colspan="1" rowspan="1">75 μL ofPhenylephrine-HCI0.5%</td></tr><tr><td colspan="1" rowspan="1">Equate® Nasal SprayDecongestant</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">NasalDecongestant</td><td colspan="1" rowspan="1">Equate®</td><td colspan="1" rowspan="1">75 μL ofXylometazoline 0.1%</td></tr><tr><td colspan="1" rowspan="1">Beconase AQ®</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of the nasalsymptoms of seasonal orperennial rhinitis</td><td colspan="1" rowspan="1">GSK</td><td colspan="1" rowspan="1">3.6x10-3 g</td></tr><tr><td colspan="1" rowspan="1">Flunisolide Nasal SolutionUSP, 0.025%</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of the nasalsymptoms of seasonal orperennial rhinitis</td><td colspan="1" rowspan="1">Bausch &amp; LombIncorporated</td><td colspan="1" rowspan="1">1.9x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Nasacort® AQ</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of the nasalsymptoms of seasonal orperennial rhinitis</td><td colspan="1" rowspan="1">Sanofi-AventisU.S. LLC</td><td colspan="1" rowspan="1">4.4x10-3 g</td></tr><tr><td colspan="1" rowspan="1">Rhinocort aqua®</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of the nasalsymptoms of seasonal orperennial rhinitis</td><td colspan="1" rowspan="1">AstraZeneca LP</td><td colspan="1" rowspan="1">4.3x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Nasonex</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of the nasalsymptoms of seasonal orperennial rhinitis</td><td colspan="1" rowspan="1">ScheringCorporation</td><td colspan="1" rowspan="1">4.1x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Fluticasone Propionate</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Treatment of nasal allergysymptoms and nonallergicrhinitis</td><td colspan="1" rowspan="1">Hi-TechPharmacal Co.,Inc.</td><td colspan="1" rowspan="1">3.8x10-5 g</td></tr><tr><td colspan="1" rowspan="1">Luffeel®</td><td colspan="1" rowspan="1">Nasal spray(liquid)</td><td colspan="1" rowspan="1">Homeopathic preparationused to treat hay fever</td><td colspan="1" rowspan="1">Heel CanadaInc.</td><td colspan="1" rowspan="1">Galphimia glauca andLuffa operculata:7.5x10-3 g of eachdilutionHistaminum and Sulfur:3.8x10-3 g of eachdilution</td></tr><tr><td colspan="1" rowspan="1">Zicam® No-Drip Liquid® Nasal GelTMExtreme Congestion Relief</td><td colspan="1" rowspan="1">Nasal gel</td><td colspan="1" rowspan="1">Relief of congestion dueto the common cold, hayfever and upperrespiratory allergies</td><td colspan="1" rowspan="1">Zicam LLC</td><td colspan="1" rowspan="1">3.7x10-2 g ofoxymethazoline HCl0.05%</td></tr><tr><td colspan="1" rowspan="1">Relenza</td><td colspan="1" rowspan="1">Liquid (afterresuspended)</td><td colspan="1" rowspan="1">Treatment andprophylaxis of influenza</td><td colspan="1" rowspan="1">GSK</td><td colspan="1" rowspan="1">6.4x10-3 g</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Liquid (afterrehydrated)</td><td colspan="1" rowspan="1">Respiratory antibiotic</td><td colspan="1" rowspan="1">Sigma-AldrichCorporation</td><td colspan="1" rowspan="1">4.5x10-3 g</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">FluMist®</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">Influenza VaccineLive, Intranasal</td><td colspan="1" rowspan="1">MedImmune</td><td colspan="1" rowspan="1">102.4-2.8 FFU (fluorescentfocus units) of liveattenuated influenzavirus reassortants</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="9">NA</td><td colspan="1" rowspan="9">Simulation of potentialmicrobial interference</td><td colspan="1" rowspan="9">NA</td><td colspan="1" rowspan="9">1x105 CFU each</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium flavescens</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhali</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis subsp.hominis</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr></table>

Most of the substances in the table above were available in a ready-to-use liquid or gel format and were used full strength in this study. Certain substances (e.g., Mucin, Chloraseptic®, Relenza®, Tobramycin) were rehydrated, dissolved or resuspended at a biologically or medically relevant concentration prior to use in this study. Fresh blood was obtained from a blood donor center.

For liquid substances, the volume tested was $7 5 ~ \mu \mathrm { L }$ since this volume represents the maximum volume that could theoretically be absorbed by a swab. For gel substances, swabs were coated with the substance. In both cases, the weight of each substance was considered as the maximum weight likely to be present on a nasal swab specimen. For substances available in a ready-to-use format, the actual amount of active ingredient used $\mathrm { ( g 0 r \% ) }$ ) was calculated based on documentation provided by the individual manufacturers.

MRSA negative samples and MRSA positive samples at $2 { \cdot } 3 \ \mathbf { x }$ LoD in simulated nasal matrix (Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details) were tested with the highest amount of each substance or organisms likely to be found at the sampling site or on the nasal swab sample.

For each potentially interfering substance and for potential microbial interference assessment, at least $9 5 \%$ conforming assay results had to be obtained. In addition, equivalence had to be demonstrated between the conditions tested and the controls based on the statistical analyses as described below:

The two one-sided tests (TOST) procedure of Schuirmann (1987) was used to assess the equivalence of the SDPA means. Rejection of the non-equivalence hypothesis in favor of equivalence hypothesis at a significance level of $5 \%$ occurred only if the $90 \%$ confidence interval of the mean difference was contained completely within predetermined the equivalence margins below:

Equivalence Margins for BD MAX™ StaphSR Assay   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Equivalence Margin</td></tr><tr><td rowspan=4 colspan=1>With and Without PotentiallyInterfering Substance</td><td rowspan=1 colspan=1>FAM SDPA (MRSA)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>ROX SDPA (MRSA)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>VIC SDPA (MSSA)</td><td rowspan=1 colspan=1>± 6%</td></tr><tr><td rowspan=1 colspan=1>Cy5.5 SDPA (Neg)</td><td rowspan=1 colspan=1>± 4%</td></tr></table>

Testing results that did not comply with these acceptance criteria were indicative of a potentially interfering substance.

The Potentially Interfering Substances Study results demonstrated no reportable interference (based on the acceptance criteria described above) with any microorganisms or chemical substances tested, except for Tobramycin, which showed inhibition in the BD MAX™ StaphSR assay when tested at a concentration of $4 . 5 \mathrm { { x } }$ $1 0 ^ { - 3 } \ \mathrm { g / s w a b }$ .

In conclusion, none of the potentially interfering microorganisms or chemical substances at the concentrations tested in this study interfered with the BD MAX™ StaphSR assay, except for Tobramycin at $4 . 5 \mathrm { ~ x ~ } 1 0 ^ { - 3 } \mathrm { ~ g / s w a b }$ . Tobramycin is reported as an interfering substance in the product package insert.

Low Loads of MRSA or MSSA Co-Spiked with MRSE

An analytical study was performed to assess any potential competitive inhibitory effects of increasing concentrations of MRSE when co-spiked with low loads of MRSA or MSSA.

The bacterial stocks used in this study is listed the table below.

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Stock Concentration(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ Type ii</td><td rowspan=1 colspan=1>9.90 x 107</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ Type iv</td><td rowspan=1 colspan=1>2.10 x 108</td></tr><tr><td rowspan=1 colspan=1>MSSA (ATCC 29213)</td><td rowspan=1 colspan=1>2.20 x 108</td></tr><tr><td rowspan=1 colspan=1>MRSE</td><td rowspan=1 colspan=1>4.64 x 109</td></tr></table>

The MRSA MREJ Type ii, MRSA MREJ Type iv, and MSSA strains were tested at 1-2X LoD in simulated nasal matrix (Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details), with increasing concentrations of a co-spiked MRSE strain. The experimental testing conditions of the study were summarized in the three tables below:

Testing Conditions for Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MRSA MREJ Type ii   
1Highest concentration achieved   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>MRSA StrainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MRSE strainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MRSA to MRSERatio</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=8 colspan=1>MRSA MREJ Type ii</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1:0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+02</td><td rowspan=1 colspan=1>1:1 x 100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>1:1 x 101</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+04</td><td rowspan=1 colspan=1>1:1 x 102</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+05</td><td rowspan=1 colspan=1>1:1 x 103</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+06</td><td rowspan=1 colspan=1>1:1 x 104</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>2.06E+07</td><td rowspan=1 colspan=1>1:1 x 105</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2.06E + 02</td><td rowspan=1 colspan=1>1.74E+081</td><td rowspan=1 colspan=1>1:8.45 x 105</td></tr></table>

Testing Conditions for Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MRSA MREJ Type iv   

<table><tr><td colspan="1" rowspan="1">Condition #</td><td colspan="1" rowspan="1">MRSA Strain</td><td colspan="1" rowspan="1">MRSA StrainConcentration inCFU/Swab</td><td colspan="1" rowspan="1">MRSE strainConcentration inCFU/Swab</td><td colspan="1" rowspan="1">MRSA to MRSERatio</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="6">MRSA MREJ Type iv</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1:0</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+02</td><td colspan="1" rowspan="1">1:1 x 100</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+03</td><td colspan="1" rowspan="1">1:1 x 101</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+04</td><td colspan="1" rowspan="1">1:1 x 102</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+05</td><td colspan="1" rowspan="1">1:1 x 103</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+06</td><td colspan="1" rowspan="1">1:1 x 104</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+07</td><td colspan="1" rowspan="1">1:1 x 105</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.35E + 02</td><td colspan="1" rowspan="1">1.35E+08</td><td colspan="1" rowspan="1">1:1 x 106</td></tr></table>

Testing Conditions for Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MSSA   
1Highest concentration achieved   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>MSSA Strain</td><td rowspan=1 colspan=1>MSSA StrainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MRSE strainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MSSA to MRSERatio</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=8 colspan=1>MSSA</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1:0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+02</td><td rowspan=1 colspan=1>1:1 x 100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+03</td><td rowspan=1 colspan=1>1:1 x 101</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+04</td><td rowspan=1 colspan=1>1:1 x 102</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+05</td><td rowspan=1 colspan=1>1:1 x 103</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+06</td><td rowspan=1 colspan=1>1:1 x 104</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>3.47E+07</td><td rowspan=1 colspan=1>1:1 x 105</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.47E + 02</td><td rowspan=1 colspan=1>1.74E+081</td><td rowspan=1 colspan=1>1:5 x 10^</td></tr></table>

To be considered to be “no competitive inhibitory effect observed” for this study, $\ge 9 5 \%$ of expected results had to be obtained when MRSA or MSSA strains were tested at or near their respective LoD in the presence of MRSE, the potential microbial competitor.

The study results are presented in the tables below:

Results of Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MRSA MREJ Type ii   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>SA+,MRSA+/SA+,MRSA-/ SA-,MRSA-/UNR/Total</td><td rowspan=1 colspan=1>Conforming/Total</td><td rowspan=1 colspan=1>% ConformingResult</td><td rowspan=1 colspan=1>IND Excluded</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7/0/0/0/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6/0/0/0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7/0/1/0/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/0/0/0/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr></table>

1Sample(s) were excluded due to an “Indeterminate” (IND) result (i.e., Heater Warning or Error in Full Fill Check)

Results of Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MRSA MREJ Type iv   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>SA+,MRSA+/SA+,MRSA-/ SA-,MRSA- /UNR/Total</td><td rowspan=1 colspan=1>Conforming/Total</td><td rowspan=1 colspan=1>% ConformingResult</td><td rowspan=1 colspan=1>IND Excluded</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/0/3/0/8</td><td rowspan=1 colspan=1>5/8</td><td rowspan=1 colspan=1>62.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7/1/0/0/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/0/1/0/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8/0/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr></table>

Results of Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MRSA (MREJ types combined)   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>SA+,MRSA+/SA+,MRSA-/ SA-,MRSA- /UNR/Total</td><td rowspan=1 colspan=1>Conforming/Total</td><td rowspan=1 colspan=1>% ConformingResult</td><td rowspan=1 colspan=1>IND Excluded</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16/0/0/0/16</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>16/0/0/0/16</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/0/0/0/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>14/0/0/0/14</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>13/0/3/0/16</td><td rowspan=1 colspan=1>13/16</td><td rowspan=1 colspan=1>81.3%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14/1/1/0/16</td><td rowspan=1 colspan=1>14/16</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14/0/1/0/15</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16/0/0/0/16</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr></table>

1Sample(s) were excluded due to an “Indeterminate” (IND) result (i.e., Heater Warning or Error in Full Fill Check)

Results of Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MRSE when Co-spiked with Low loads of MSSA   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>SA+,MRSA+/SA+,MRSA-/SA-,MRSA- /UNR/Total</td><td rowspan=1 colspan=1>Conforming/Total</td><td rowspan=1 colspan=1>% ConformingResult</td><td rowspan=1 colspan=1>INDExcluded</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/8/0/0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0/6/2/0/8</td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0/7/1/0/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr></table>

In conclusion, competitive inhibitory effects were observed for an $\mathrm { M R S A } : \mathrm { M R S E }$ ratio of $1 : \geq 1 \mathrm { ~ x ~ } 1 \hat { 0 } ^ { 3 }$ and an MSSA $\underline { { \boldsymbol { 1 } } }$ MRSE ratio of $1 : \geq 1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ . These observed competitive inhibitory effects are reported in the product package insert.

# Low Loads of MRSA Co-Spiked with MSSA

An analytical study was performed to assess any potential competitive inhibitory effects of increasing concentrations of MSSA when co-spiked with low loads of MRSA.

MRSA MREJ Type iv, an MREJ type with one of the lowest LoDs of the assay, was tested at 1-2X LoD in simulated nasal matrix (Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details), with increasing concentrations of a co-spiked MSSA strain. The experimental testing conditions of the study were summarized in the table below:

Testing Conditions for Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MSSA when Co-spiked with Low loads of MRSA MREJ Type iv   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>MRSA StrainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MSSA strainConcentration inCFU/Swab</td><td rowspan=1 colspan=1>MRSA to MRSERatio</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=8 colspan=1>MRSA MREJ Type iv</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1:0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+02</td><td rowspan=1 colspan=1>1:1 x 100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+03</td><td rowspan=1 colspan=1>1:1 x 101</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+04</td><td rowspan=1 colspan=1>1:1 x 102</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+05</td><td rowspan=1 colspan=1>1:1 x 103</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+06</td><td rowspan=1 colspan=1>1:1 x 104</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+07</td><td rowspan=1 colspan=1>1:1 x 105</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.35E + 02</td><td rowspan=1 colspan=1>1.35E+08</td><td rowspan=1 colspan=1>1:1 x 106</td></tr></table>

To be considered to be “no competitive inhibitory effect observed” for this study, $\ge 9 5 \%$ of expected results had to be obtained when the MRSA strain was tested near the LoD in the presence of MSSA, the potential microbial competitor.

The study results are presented in the table below:

Results of Assessing Potential Competitive Inhibitory Effects of Increasing Concentrations of MSSA when Co-spiked with Low loads of MRSA   

<table><tr><td rowspan=1 colspan=1>Condition #</td><td rowspan=1 colspan=1>SA+,MRSA+/SA+,MRSA-/ SA-,MRSA- /UNR/Total</td><td rowspan=1 colspan=1>Conforming/Total</td><td rowspan=1 colspan=1>% ConformingResult</td><td rowspan=1 colspan=1>INDExcluded</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24/0/0/0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>23/1/0/0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24/0/0/0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>23/1/0/0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23/1/0/0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>21/3/0/0/24</td><td rowspan=1 colspan=1>21/24</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>24/0/0/0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>22/1/0/1/24</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>1</td></tr></table>

1One excluded sample due to an atypical curve in all channels caused by a bubble inside the PCR cartridge.

In conclusion, competitive inhibitory effect was observed for a MRSA $\boldsymbol { : }$ MSSA ratio of $1 : \geq 1 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ . This observed competitive inhibitory effect is reported in the product package insert.

# $j .$ . Carry-Over Study:

A study was conducted to evaluate the risk of carry-over contamination with the BD MAX™ StaphSR assay on the BD MAX™ instrument.

The testing panels used in this study were made of high positive MRSA members and negative members with or without simulated nasal matrix (Refer to the “Nasal Matrix and Simulated Nasal Matrix Equivalency Study” section of this decision summary for details).

Carry-Over without Simulated Nasal Matrix

The MREJ type with the lowest LoD in absence of simulated nasal matrix (MREJ Type iv MRSA strain) was used as the high positive panel member $\mathrm { ( 8 } \mathrm { X } 1 0 ^ { 7 }$ CFU/swab). ID Broth was used as the negative panel member. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run; high positive and negative panel members were alternated sequentially in each run. Three operators performed three consecutive runs for a total of nine runs of 24 samples per run (i.e., 108 expected MRSA negative results).

# Carry-Over with Simulated Nasal Matrix

The MREJ type with the lowest LoD in the presence of simulated nasal matrix (MREJ Type xiii MRSA strain) was used as the high positive panel member (5.3 X $\mathrm { i } 0 ^ { 7 }$ CFU/swab). ID Broth was used as the negative panel member. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run; high positive and negative panel members were alternated sequentially in each run. Three operators performed three consecutive runs for a total of nine runs of 24 samples per run (i.e., 108 expected MRSA negative results).

The pre-determined acceptance criterion for this study is less than $2 \%$ false positive rate (point estimate) due to MRSA contamination.

Two MRSA false positive results were obtained during the carry-over study in absence of simulated nasal matrix. One MRSA false positive result was obtained during the carry-over study in presence of simulated nasal matrix.

Overall, from 203 reportable results, three false MRSA positive results were obtained (3/203, $1 . 5 \%$ ; $9 5 \%$ CI, $0 . 5 \% - \ 4 . 3 \% )$ in this study due to carry-over contamination, which met the pre-determined acceptance criterion for this study.

# k. Assay cut-off:

BD MAX™ StaphSR assay cut-off values (see the table below) were determined in verification experiments.

MAX™ StaphSR Assay Cut-off Values   

<table><tr><td rowspan=1 colspan=1>Cut-off</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>ROX</td><td rowspan=1 colspan=1>VIC</td><td rowspan=1 colspan=1>Cy5.5</td></tr><tr><td rowspan=1 colspan=1>SDPA Min</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>SDPA Max</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>EP Min</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>236</td></tr></table>

The cut-offs of the BD $\mathbf { M A X T M }$ StaphSR assay were initially established through a series of internal analytical studies and verified through a clinical simulation as described below:

Cut-offs were initially set using an approach based on (but not limited to) the Process performance index (Ppk). The main idea behind cut-off determination was to vary the threshold values (cut-offs) considering the appropriate distribution for the PCR variables (EP and SDPA). The $\mathrm { P p k }$ (the estimated proportion of false positive / negative results) was calculated in order to maximize sensitivity while minimizing non-specific amplification detection.

Briefly, the EP values of negative and positives samples were modeled using spiked simulated biological matrix samples. The interfering fluorescence from nearby channels (“cross-talk”) was also evaluated. Finally, the cut-offs were repositioned within a certain range and the final choice was based on expected false positive and negative proportions. These proportions were calculated using the distribution properties to ensure balanced and optimal sensitivity/specificity performance.

A positive result is obtained in FAM, VIC or ROX channel when the fluorescence signal is higher than the EP cut-off and between the SDPA lower and upper cut-offs. However, only specific gene(s) required to correctly identify MRSA or SA must be detected in corresponding channels. Moreover, cut-offs were set in order to allow the Sample Processing Control (SPC) to be predictive of Polymerase Chain Reaction (PCR) inhibition to avoid false-negative assay results.

Variability was included in the design of experiments to ensure that cut-offs values are suitable for true clinical specimens. Multiple instruments, reagents and simulated nasal matrix lots were therefore included in the study design.

The assay cut-offs selected were then verified with an experiment with 100 negative clinical specimens spiked with a low charge of MRSA MREJ Type ii and adjusted accordingly.

The final step of cut-offs determination was to verify whether cross-talk of amplification signal occurred.

Following these steps, preliminary cut-off values were refined for each channel.

Finally, a clinical simulation study involving 150 clinical samples was performed to verify the adequacy of the preliminary cut-offs.

These pre-determined preliminary cut-off values were validated again using the data collected during the clinical performance study.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Performance of BD MAX™ StaphSR assay was evaluated against the reference method of culture in a prospective study.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Study:

Clinical performance characteristics of the BD MAX™ StaphSR assay were determined in a multi-site prospective study. Three investigational sites participated in the study. To be enrolled in the study, patients had to be eligible for MRSA or SA testing according to institutional policies. Eligibility requirements for targeted screening as per clinical site policies included, but were not limited to: patients admitted into the particular healthcare system; patients admitted to the Intensive Care Unit; patients transferred to the Intensive Care Unit; pre-elective surgery patients; and patients being admitted from long-term care facilities. Specimens from patients previously enrolled in the study were excluded from the enrollment.

The Comparative Reference Method consisted of direct culture complemented by enriched culture. Enriched culture analysis was completed for all specimens that were negative for MRSA or SA by direct culture. Presumptive S. aureus colonies observed on selective (S. aureus) chromogenic medium were subcultured onto Blood Agar (BA). Identification was confirmed with an agglutination test, while methicillin resistance was confirmed by Cefoxitin disk $( 3 0 \mu \mathrm { g } )$ diffusion susceptibility testing. Enrichment in Trypticase Soy Broth with $6 . 5 \%$ NaCl (TSB $6 . 5 \%$ NaCl) was completed in the event that MRSA or SA was not confirmed by the initial direct culture method. Turbid TSB $6 . 5 \%$ NaCl broth was used to inoculate additional chromogenic medium and BA plates; MRSA confirmation was performed as described above.

Performance against the Reference Method (Direct/Enriched Culture)

A total of 2451 specimens were enrolled in the study. Of those, 94 specimens were regarded as noncompliant per protocol criteria, and three fully compliant per protocol criteria specimens gave final non-reportable PCR results. A total of 2354 specimen results were used to determine the clinical performance of the BD MAX™ StaphSR assay.

Compared to the Reference Method (Direct/Enriched Culture), the BD MAX™ StaphSR assay identified $9 3 . 1 \%$ of the MRSA positive specimens and $9 7 . 5 \%$ of the MRSA negative specimens. For the population tested, this resulted in a Negative Predictive Value (NPV) of $9 9 . 5 \%$ and a Positive Predictive Value (PPV) of $7 3 . 2 \%$ .

Results Obtained for MRSA with the BD MAX™ StaphSR Assay in Comparison to the Reference Method   

<table><tr><td rowspan=2 colspan=1>All Sites</td><td rowspan=1 colspan=4>Reference Method</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXTM StaphSR Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>203</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>11b</td><td rowspan=1 colspan=1>2140</td><td rowspan=1 colspan=1>2151</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>2194</td><td rowspan=1 colspan=1>2354</td></tr><tr><td rowspan=1 colspan=5>Sensitivity: 93.1% (149/160) (95% CI c: 88.1%, 96.1%)Specificity: 97.5% (2140/2194) (95% CI c: 96.8%, 98.1%)PPV: 73.2% (95% CI : 67.8%, 78.3%)NPV: 99.5% (95% CI: 99.1%, 99.7%)</td></tr></table>

a 12 of 54 MRSA False Positive BD MAX™ StaphSR specimens were also found to be positive after repeat of Reference Method $^ { \mathsf { b } } 5$ of 11 MRSA False Negative BD MAX™ StaphSR specimens were also found to be negative after repeat of Reference Method. c Confidence Interval

# Site by Site Performance Obtained for MRSA with the BD MAX™ StaphSR Assay in Comparison to the Reference Method

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence a</td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>4.3% (41/960)</td><td rowspan=1 colspan=1>92.7% (38/41)(80.6%, 97.5%)</td><td rowspan=1 colspan=1>98.9% (908/918)(98.0%, 99.4%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>5.8% (38/650)</td><td rowspan=1 colspan=1>86.8% (33/38)(72.7%, 94.2%)</td><td rowspan=1 colspan=1>98.5% (583/592)(97.1%, 99.2%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>10.6% (81/765)</td><td rowspan=1 colspan=1>96.3% (78/81)(89.7%, 98.7%)</td><td rowspan=1 colspan=1>94.9% (649/684)(93.0%, 96.3%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>6.7% (160/2375 c)</td><td rowspan=1 colspan=1>93.1% (149/160)(88.1%, 96.1%)</td><td rowspan=1 colspan=1>97.5% (2140/2194)(96.8%, 98.1%)</td></tr></table>

a Prevalence based on reference method only b Confidence interval $^ { \mathrm { c } } 2 3 7 5$ specimens were reference method compliant.

Compared to the Reference Method (Direct/Enriched Culture), the BD MAX™ StaphSR assay identified $9 2 . 0 \%$ of the SA positive specimens and $9 3 . 1 \%$ of the SA negative specimens. For the population tested, this resulted in a NPV of $9 6 . 8 \%$ and a PPV of $8 3 . 4 \%$ .

Results Obtained for SA with the BD MAX™ StaphSR Assay in Comparison to the Reference Method   

<table><tr><td rowspan=2 colspan=1>All Sites</td><td rowspan=1 colspan=4>Reference Method</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXTM StaphSR Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>599</td><td rowspan=1 colspan=1>118a</td><td rowspan=1 colspan=1>717</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>52b</td><td rowspan=1 colspan=1>1585</td><td rowspan=1 colspan=1>1637</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>651</td><td rowspan=1 colspan=1>1703</td><td rowspan=1 colspan=1>2354</td></tr><tr><td rowspan=1 colspan=5>Sensitivity: 92.0% (599/651) (95% CI c: 89.7%, 93.9%)Specificity: 93.1% (1585/1703) (95% CI c: 91.8%, 94.2%)PPV: 83.4% (95% CI : 81.9%, 85.8%)NPV: 96.8% (95% CI : 96.0%, 97.6%)</td></tr></table>

a 28 of 118 SA False Positive BD MAX™ StaphSR specimens were also found to be positive after repeat of Reference Method

Site by Site Performance Obtained for SA with the BD MAX™ StaphSR Assay in Comparison to the Reference Method   

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence a</td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>27.2% (261/960)</td><td rowspan=1 colspan=1>90.0% (235/261)(85.8%, 93.1%)</td><td rowspan=1 colspan=1>96.3% (672/698)(94.6%, 97.4%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>27.5% (179/650)</td><td rowspan=1 colspan=1>91.5% (162/177)(86.5%, 94.8%)</td><td rowspan=1 colspan=1>90.9% (412/453)(88.0%, 93.3%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>27.8% (213/765)</td><td rowspan=1 colspan=1>94.8% (202/213)(91.0%, 97.1%)</td><td rowspan=1 colspan=1>90.8% (501/552)(88.1%, 92.9%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>27.5% (653/2375 c)</td><td rowspan=1 colspan=1>92.0% (599/561)(89.7%, 93.9%)</td><td rowspan=1 colspan=1>93.1% (1585/1703)(91.8%, 94.2%)</td></tr></table>

a Prevalence based on reference method only b Confidence interval c 2375 specimens were reference method compliant

# Performance against Direct Culture

A total of 2451 specimens were enrolled in the study. Of those, 2397 nasal swab specimens were found compliant at the specimen, direct culture and BD MAX™ StaphSR assay level (i.e., 54 specimens were regarded as noncompliant per protocol criteria) and four fully compliant per protocol criteria nasal swab specimens gave non reportable results. A total of 2393 specimen results were used to determine the positive and negative percent agreement of the BD MAX™ StaphSR assay against direct culture.

Compared to the direct culture method, the BD MAX™ StaphSR assay identified $9 6 . 5 \%$ of the MRSA positive specimens and $9 6 . 9 \%$ of the MRSA negative specimens.

Results Obtained for MRSA with the BD $\mathbf { M A X T M }$ StaphSR Assay in Comparison to the Direct Culture Method   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>Direct Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXTM StaphSR Assay</td><td rowspan=2 colspan=1>PositiveNegative</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>206</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2182</td><td rowspan=1 colspan=1>2187</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>2251</td><td rowspan=1 colspan=1>2393</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement: 96.5% (137/142) (95% CI a: 92.0%, 98.5%)Negative Percent Agreement: 96.9% (2182/2251) (95% CI a: 96.1%, 97.6%)</td></tr></table>

a Confidence interval

Site by Site Performance Obtained for MRSA with the BD MAX™ StaphSR Assay in Comparison to the Direct Culture Method   

<table><tr><td colspan="1" rowspan="1">Clinical Sites</td><td colspan="1" rowspan="1">Positive Percent Agreementwith 95% CI a</td><td colspan="1" rowspan="1">Negative Percent Agreementwith 95% CI a</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">100% (35/35)(90.1%, 100%)</td><td colspan="1" rowspan="1">98.6% (911/924)(97.6%, 99.2%)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">93.5% (29/31)(79.3%, 98.2%)</td><td colspan="1" rowspan="1">97.8% (586/599)(96.3%, 98.7%)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">96.1% (73/76)(89.0%, 98.6%)</td><td colspan="1" rowspan="1">94.1% (685/728)(92.1%, 95.6%)</td></tr><tr><td>Overall</td><td>96.5% (137/142) (92.0%, 98.5%)</td><td>96.9% (2182/2251) (96.1%, 97.6%)</td></tr></table>

a Confidence interval

Compared to the direct culture method, the BD $\mathbf { M A X T M }$ StaphSR assay identified $9 5 . 1 \%$ of the SA positive specimens and $9 0 . 9 \%$ of the SA negative specimens.

Results Obtained for SA with the BD MAX™ StaphSR Assay in Comparison to the Direct Culture Method   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>Direct Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>BD MAXTM StaphSR Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>564</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>728</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1636</td><td rowspan=1 colspan=1>1665</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>593</td><td rowspan=1 colspan=1>1800</td><td rowspan=1 colspan=1>2393</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement: 95.1% (564/593) (95% CI a: 93.1%, 96.6%)Negative Percent Agreement: 90.9% (1636/1800) (95% CI a: 89.5%, 92.1%)</td></tr></table>

a Confidence interval

# Site by Site Performance Obtained for SA with the BD MAX™ StaphSR Assay in Comparison to the Direct Culture Method

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Positive Percent Agreementwith 95% CI a</td><td rowspan=1 colspan=1>Negative Percent Agreementwith 95% CI a</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>93.0% (213/229)(89.0%, 95.7%)</td><td rowspan=1 colspan=1>93.4% (682/730)(91.4%, 95.0%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>94.9% (149/157)(90.3%, 97.4%)</td><td rowspan=1 colspan=1>88.6% (419/473)(85.4%, 91.1%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>97.6% (202/207)(94.5%, 99.0%)</td><td rowspan=1 colspan=1>89.6% (535/597)(86.9, 91.8%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>95.1% (564/593)(93.1%, 96.6%)</td><td rowspan=1 colspan=1>90.9% (1636/1800)(89.5%, 92.1%)</td></tr></table>

a Confidence interval

Unresolved, Indeterminate, Incomplete Rates

Out of 2399 nasal swab specimens compliant at the specimen and PCR level, tested with the BD $\mathbf { M A X T M }$ StaphSR assay, 37 $( 1 . 5 \% )$ were reported as Unresolved, Indeterminate, or Incomplete after initial testing. Fifteen (15) $( 0 . 6 \% )$ were reported as Unresolved initially. The Unresolved Rate after repeat testing is $0 . 0 4 \%$ (1/2399). Fourteen (14) $( 0 . 6 \% )$ were initially reported as Indeterminate. No result remained Indeterminate upon repeat (two specimens were not retested since they had already been tested twice). Eight (8) $( 0 . 3 \% )$ were initially reported as Incomplete. No result remained Incomplete upon repeat (one specimen was not retested since they had already been tested twice).

Unresolved, Indeterminate, Incomplete Rates with $9 5 \%$ Confidence Intervals by Site   

<table><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Initial Unresolved Rates</td><td colspan="1" rowspan="1">95% CI a</td><td colspan="1" rowspan="1">Final Unresolved Rates</td><td colspan="1" rowspan="1">95% CI a</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.6% (6/960)</td><td colspan="1" rowspan="1">(0.3%, 1.4%)</td><td colspan="1" rowspan="1">0% (0/960)</td><td colspan="1" rowspan="1">(0%, 0.4%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6% (4/635)</td><td colspan="1" rowspan="1">(0.2%, 1.6%)</td><td colspan="1" rowspan="1">0.2% (1/635)</td><td colspan="1" rowspan="1">(0%, 0.9%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.6% (5/804)</td><td colspan="1" rowspan="1">(0.3%, 1.4%)</td><td colspan="1" rowspan="1">0% (0/804)</td><td colspan="1" rowspan="1">(0%, 0.5%)</td></tr><tr><td colspan="1" rowspan="1">Overall Study</td><td colspan="1" rowspan="1">0.6% (15/2399)</td><td colspan="1" rowspan="1">(0.4%, 1.0%)</td><td colspan="1" rowspan="1">0.04% (1/2399)</td><td colspan="1" rowspan="1">(0%, 0.2%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Initial Indeterminate Rates</td><td colspan="1" rowspan="1">95% CI a</td><td colspan="1" rowspan="1">Final Indeterminate Rates</td><td colspan="1" rowspan="1">95% CI a</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.7% (7/960)</td><td colspan="1" rowspan="1">(0.4%, 1.5%)</td><td colspan="1" rowspan="1">0.1% (1/960)</td><td colspan="1" rowspan="1">(0%, 0.6%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.3% (2/635)</td><td colspan="1" rowspan="1">(0.1%, 1.1%)</td><td colspan="1" rowspan="1">0.2% (1/635)</td><td colspan="1" rowspan="1">(0%, 0.9%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.6% (5/804)</td><td colspan="1" rowspan="1">(0.3%, 1.4%)</td><td colspan="1" rowspan="1">0% (0/804)</td><td colspan="1" rowspan="1">(0%, 0.5%)</td></tr><tr><td colspan="1" rowspan="1">Overall Study</td><td colspan="1" rowspan="1">0.6% (14/2399)</td><td colspan="1" rowspan="1">(0.3%, 1.0%)</td><td colspan="1" rowspan="1">0.1% (2/2399)</td><td colspan="1" rowspan="1">(0%, 0.3%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Initial Incomplete Rates</td><td colspan="1" rowspan="1">95% CI a</td><td colspan="1" rowspan="1">Final Incomplete Rates</td><td colspan="1" rowspan="1">95% CI a</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.8% (8/960)</td><td colspan="1" rowspan="1">(0.4%, 1.6%)</td><td colspan="1" rowspan="1">0% (0/960)</td><td colspan="1" rowspan="1">(0%, 0.4%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0% (0/635)</td><td colspan="1" rowspan="1">(0%, 0.6%)</td><td colspan="1" rowspan="1">0.2% (1/635)</td><td colspan="1" rowspan="1">(0%, 0.9%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0% (0/804)</td><td colspan="1" rowspan="1">(0%, 0.5%)</td><td colspan="1" rowspan="1">0% (0/804)</td><td colspan="1" rowspan="1">(0%, 0.5%)</td></tr><tr><td colspan="1" rowspan="1">Overall Study</td><td colspan="1" rowspan="1">0.3% (8/2399)</td><td colspan="1" rowspan="1">(0.2%, 0.7%)</td><td colspan="1" rowspan="1">0.04 (1/2399)</td><td colspan="1" rowspan="1">(0%, 0.2%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1.5% (37/2399)</td><td colspan="1" rowspan="1">(1.1%, 2.1%)</td><td colspan="1" rowspan="1">0.2% (4/2399)</td><td colspan="1" rowspan="1">(0.1%, 0.4%)</td></tr></table>

a Confidence interval

b. Retrospective Study: Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values:

In the BD MAX™ StaphSR assay prospective clinical study a total of 2395 reportable results, from specimens compliant at the specimen and PCR levels, were obtained from three geographically diverse sites and compared with Direct and Enriched culture. The study population was grouped into in-patient and out-patient categories. The number and percentage of MRSA and SA positive cases, as determined by the BD MAX™ StaphSR assay, are presented in the table below:

Expected Values   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>BD MAXTM StaphSR Assay</td><td rowspan=2 colspan=1>PositiveMRSAPercentage</td><td rowspan=2 colspan=1>Positive SAPercentage</td></tr><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Total Numberof Specimens1</td><td rowspan=1 colspan=1>Number ofMRSA Positive</td><td rowspan=1 colspan=1>Number of SAPositive</td></tr><tr><td rowspan=1 colspan=1>In-patient</td><td rowspan=1 colspan=1>1685</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>548</td><td rowspan=1 colspan=1>10.6%(178/1685)</td><td rowspan=1 colspan=1>32.5%(548/1685)</td></tr><tr><td rowspan=1 colspan=1>Out-patient</td><td rowspan=1 colspan=1>710</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>3.9%(28/710)</td><td rowspan=1 colspan=1>25.6%(182/710)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2395</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>730</td><td rowspan=1 colspan=1>8.6%(206/2395)</td><td rowspan=1 colspan=1>30.5%(730/2395)</td></tr></table>

1Total specimens based on compliant PCR results.

N. Instrument Name: BD MAX™ System

# O. System Descriptions:

1. Modes of Operation:

The 2nd generation BD MAX™ System fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification: Specimens are labeled with a Barcode.

4. Specimen Sampling and Handling:

Not applicable. The nasal swab is manually inserted in the sample buffer tube, vortexed and placed into the system.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control: Quality control is addressed for each specific assay to be run on the instrument (separately cleared).

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.