# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# E. Applicant:

F. Proprietary and Established Names: ICEPlex® C. difficile Assay

# G. Regulatory Information:

. Regulation section: 21 CFR 866.3130, Clostridium difficile toxin gene amplification assay

2. Classification: II

3. Product code: OZN, NSU

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The PrimeraDx ICEPlex® C. difficile Assay is for the qualitative detection of the Clostridium difficile toxin B gene (tcdB gene) in nucleic acids purified from unpreserved liquid or soft human stool specimens from patients suspected of having C. difficile infection (CDI).

The ICEPlex C. difficile Assay is intended to be used only on the ICEPlex® System, which integrates PCR-based amplification with capillary electrophoresis (CE) for the detection of amplification products. The assay is intended to aid in the diagnosis of CDI. Results should be considered in conjunction with patient clinical history.

2. Indication(s) for use:

The PrimeraDx ICEPlex® C. difficile Assay is for the qualitative detection of the Clostridium difficile toxin B gene (tcdB gene) in nucleic acids purified from unpreserved liquid or soft human stool specimens from patients suspected of having C. difficile infection (CDI).

The ICEPlex C. difficile Assay is intended to be used only on the ICEPlex® System, which integrates PCR-based amplification with capillary electrophoresis (CE) for the detection of amplification products. The assay is intended to aid in the diagnosis of CDI. Results should be considered in conjunction with patient clinical history.

3. Special conditions for use statement(s): For prescription use

4. Special instrument requirements: ICEPlex® System bioMérieux NucliSENS easyMAGTM System

# I. Device Description:

The ICEPlex System combines two functional modules: an amplification module – PCR (Polymerase Chain Reaction) thermal cycler – and an analysis module – CE (Capillary Electrophoresis) system with fluorescent detection. Individual fluorescent PCR products from multiplexed PCR reactions are analyzed by CE through direct electrokinetic injection into the separating capillaries. The labeled amplicons are separated by size and the dyes are excited by two lasers within the system.

The ICEPlex C. difficile Assay kit contains sufficient reagents to perform 100 tests and includes the following components:

• 2X PCR Buffer: Tris buffer, KCl, dNTPs, MgCl2, betaine, BSA, DTT, glycerol and Proclin.

• PCR Enzyme: Hot-start DNA polymerase

• 25X Primers Mix: Oligonucleotide primers specific to the tcdB gene of $C .$ . difficile, Internal Control and Calibration Controls. Each primer is either not labeled or has FAM or TYE-665 fluorescent label. The mix also contains salmon sperm DNA, BSA and Proclin.

• 25X Calibrators Mix: A liquid concentrate containing DNA template for Calibration Controls.

• Internal Control: Non-infectious synthetic DNA.

• 10X Injection Buffer: A liquid used by diluting 1:10 to fill those wells in the PCR plate which are not being used for samples.

• Positive Control: A liquid concentrate containing DNA template for PCR Positive Controls.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX C. diff Assay

2. Predicate 510(k) number(s): K130470

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceICEPlex® C. difficile Assay</td><td rowspan=1 colspan=1>PredicateBD MAX™ C.diff Assay(K130470)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The PrimeraDx ICEPlex® C.difficile Assay is for thequalitative detection of theClostridium difficile toxin Bgene (tcdB gene) in nucleicacids purified fromunpreserved liquid or softhuman stool specimens frompatients suspected of having C.difficile infection (CDI). TheICEPlex C. difficile Assay isintended to be used only on theICEPlex® System, whichintegrates PCR-basedamplification with capillaryelectrophoresis (CE) for thedetection of amplificationproducts. The assay is intendedto aid in the diagnosis of CDI.Results should be considered inconjunction with patientclinical history.</td><td rowspan=1 colspan=1>The BD MAX C. diff Assayperformed on the BD MAXSystem is an automated invitro diagnostic test for thedirect, qualitative detection ofthe Clostridium difficile toxinB gene (tcdB) in human liquidor soft stool specimens frompatients suspected of having C.difficile infection (CDI). Thetest, performed directly on thespecimen, utilizes real-timepolymerase chain reaction(PCR) for the amplification ofC. difficile toxin B gene DNAand fluorogenic target-specifichybridization probes for thedetection of the amplifiedDNA. The BD MAX C. diffAssay is intended to aid in thediagnosis of CDI.</td></tr><tr><td rowspan=1 colspan=1>Measureand</td><td rowspan=1 colspan=1>Clostridium difficile toxin Bgene (tcdB)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Unformed (liquid or soft) stool</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Principle</td><td rowspan=1 colspan=1>Real time PCR</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceICEPlex® C. difficile Assay</td><td colspan="1" rowspan="1">PredicateBD MAXTM C.diff Assay(K130470)</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">ICEPlex System</td><td colspan="1" rowspan="1">BD MAX System</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction</td><td colspan="1" rowspan="1">bioMérieux NucliSENSeasyMAG</td><td colspan="1" rowspan="1">Integrated</td></tr><tr><td colspan="1" rowspan="1">Assay Amplification/Detection Format</td><td colspan="1" rowspan="1">Real time PCR amplificationwith Capillary Electrophoresisand direct laser-inducedfluorescent detection and sizebased detection of target</td><td colspan="1" rowspan="1">Real time PCR withfluorescent detection of targetspecific labeled TaqManprobes</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceICEPlex® C. difficile Assay</td><td colspan="1" rowspan="1">PredicateBD MAX™ C.diff Assay(K130470)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">specific labeled primers</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">Four hours for 48 reactions</td><td colspan="1" rowspan="1">Under three hours for 24reactions</td></tr><tr><td colspan="1" rowspan="1">Results Interpretation</td><td colspan="1" rowspan="1">Positive, Negative, Invalidwith error code</td><td colspan="1" rowspan="1">Positive, Negative,Unresolved, Indeterminatewith error code, IncompleteRun with error code</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Calibration Control, InternalControl, and Positive PCRControl provided. NegativeControl and PositiveExtraction Control supplied byuser</td><td colspan="1" rowspan="1">Sample Processing Controlprovided. Positive andNegative Controlsrecommended supplied by user</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

ISO 15233-1:2007, Medical devices – Symbols to be used with medical device labels, labeling and information to be supplied

ISO 14971 second edition 2007-0301, Medical devices – Application of risk management to medical devices

AAMI / ANSI / IEC 62304:2006, Medical device software – Software life cycle processes

EN 61326-1:2006 Electrical equipment for measurement, control and laboratory use. EMC requirements.

UL 61010-1:2004, 2nd Ed. Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory User Part 1

CAN/CSA-C22.2 No. 61010-1-04:2001 Safety Requirements for Electrical Eq.1- for Measurement, Control, and Lab. Use Part 1

CAN/CSA-C22.2 No. 61010-2-101-04:2002 Safety Requirements for Electrical Equipment for Measurement, Control, and Lab. Use

EN 55011:2007, Industrial, scientific and medical (ISM) radio-frequency equipment – Electromagnetic disturbance characteristics

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile – Draft Guidance for Industry and FDA Staff (issued November 29, 2010).

# L. Test Principle:

The ICEPlex C. difficile Assay is a molecular diagnostic test for the qualitative detection of toxigenic C. difficile nucleic acids isolated and purified from liquid or soft stool specimens. This test targets the C. difficile toxin B encoding gene (tcdB).

The PrimeraDx ICEPlex system uses Scalable Target Analysis Routine (STAR) technology. STAR employs sequential sampling and analysis of fluorescently end-labeled PCR products (amplicons) by means of Capillary Electrophoresis (CE). Separation of the amplicons is facilitated by appropriate primer design allowing for discrimination by size and fluorescent signal after CE separation. In a molecular diagnostic assay performed on an ICEPlex system, all targets (including sample template, controls and internal calibration standards) are independently assessed within a single reaction well.

Real-time sampling of the amplicons allows for the construction of amplification curves and calculation of threshold cycle (Ct) similar to other real-time PCR methods.

To run the ICEPlex C. difficile Assay, nucleic acids are first extracted from stool specimens with the bioMérieux NucliSENS easyMAG system. An ICEPlex system PCR plate is assembled combining PCR buffer, PCR enzyme, primer mix containing fluorescently endlabeled oligonucleotide primers for assay controls and targets, calibration controls mix, and purified DNA from the clinical sample spiked with internal control. The PCR plate is placed in the thermal cycler module of the ICEPlex system and, after inputting sample and assay information into the integrated software, the run is initiated.

At specific PCR cycles, the capillaries and electrodes of capillary electrophoresis module are introduced into the PCR reaction. The ICEPlex system applies voltage for a predetermined time to force negatively charged DNA molecules to enter the capillary ─ a process known as electrokinetic injection. The capillaries and electrodes are then moved and are immersed in a CE buffer, voltage is applied, and capillary electrophoresis separation is performed. The PCR cycling and CE separation are timed to match one complete CE separation within two PCR cycles. All amplicons (including internal calibration controls) are separated, detected and quantified by CE using laser induced fluorescence detection. At the end of the PCR amplification, capillaries and electrodes are chemically decontaminated to remove the residual amplified PCR fragments and to recondition the capillary array for a new assay run.

The ICEPlex system software automatically processes the raw data and reports positive, negative, or invalid results for the detection of the C. difficile tcdB gene target.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The within-laboratory precision was estimated on a panel of samples prepared by spiking an appropriate amount of $C .$ . difficile culture strain BAA-1805 into a pooled clinical negative stool matrix. The blinded panel members included: low positive sample (near assay limit of detection, expected positive $59 5 \%$ of the time), moderately positive sample (expected to be positive approximately $100 \%$ of the time), negative sample (expected to be negative approximately $100 \%$ of the time), and a C20-80 sample (expected to be positive $20 - 8 0 \%$ of the time). The panel also included positive and negative controls.

The precision study consisted of two operators; three lots of ICEPlex C. difficile Assay kits; and three ICEPlex instruments. The precision study was run over 12 nonconsecutive days with two runs per day and two replicates of each sample per run. One negative sample (out of a total of 144 observations) and one low positive sample (out of a total of 144 observations) produced invalid results during the study. Overall results are summarized in the following table.

<table><tr><td rowspan=2 colspan=1>SampleConcentration</td><td rowspan=1 colspan=2>Overall Agreement</td><td rowspan=1 colspan=3>Ct Value</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>142/143</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>C20-80 2</td><td rowspan=1 colspan=1>105/144</td><td rowspan=1 colspan=1>73%</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>6.2</td></tr></table>

1 Ct values for negative samples calculated using Internal Control   
2 C20-80 sample percent agreement given as percent positive results

<table><tr><td rowspan=3 colspan=1>Sample</td><td rowspan=1 colspan=5>Lot 1</td><td rowspan=1 colspan=5>Lot2</td><td rowspan=1 colspan=5>Lot 3</td></tr><tr><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=3>Ct Value</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=3>Ct Value</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=3>Ct Value</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>SD %CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>84/84</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>83/83</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.9</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>84/84</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>s 100</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>C20-80</td><td rowspan=1 colspan=1>58/84</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>33/36</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>14/24</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.2</td></tr></table>

Between lot, between operator, between instrument, and between day repeatability were acceptable.

# Reproducibility:

Reproducibility was evaluated at three independent laboratory sites using a study panel with the same strain and concentrations as described for the precision study. Panel samples were tested at each independent laboratory site by two operators for five days with two runs per day and three replicates of each panel member per run. One low positive sample (out of a total of 90 observations) and one moderate positive

sample (out of a total of 90 observations) produced invalid results during the study.   
Study results are summarized in the tables below.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.98 to 100</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.98 to 100</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.98 to 100</td></tr><tr><td rowspan=1 colspan=1>C20-80*</td><td rowspan=1 colspan=1>62/90</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>58.26 to 78.23</td></tr></table>

Site to site reproducibility

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Site A</td><td rowspan=1 colspan=2>Site B</td><td rowspan=1 colspan=2>Site C</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C20-80*</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>16/30</td><td rowspan=1 colspan=1>53</td></tr></table>

Ct values   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Site A</td><td rowspan=1 colspan=3>Site B</td><td rowspan=1 colspan=3>Site C</td></tr><tr><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>C20-80</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>6.1</td></tr></table>

Between site, between operator, between run, and between day reproducibility results were acceptable.

b. Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

In-reaction controls:

Calibration control: A group of ICEPlex specific elements used to align electropherograms and assign identities of the target peaks. It also controls for the integrity of the kit reagents and the presence of PCR inhibitors.

Internal Control: Non-target nucleic acid that is co-extracted and co-amplified with the tcdB target. It controls for nucleic acid extraction efficiency, for the integrity of the reagents and for the presence of PCR inhibitors in a given sample. Detection of the Internal Control is not required for a positive result. The Internal Control needs to be spiked into each sample before extraction.

# External controls:

Negative Control: Substitute Stool Transport and Recovery (S.T.A.R., Roche Diagnostics Coporation) buffer for the clinical specimen and process normally through the easyMAG extraction system and on the ICEPlex Instrument.

Positive Control: This control is used for the PCR stage only. It should not be processed for nucleic acids extraction. In the positive control reaction, substitute extracted sample with provided Positive Control. Note: Provided Positive Control must be diluted 1:10 prior to use to ensure target concentration is at the appropriate level.

Known Positive Sample: It is also required to include a previously characterized positive sample or simulated sample with every easyMAG extraction run and include it in the subsequent ICEPlex Instrument run to verify successful lysis.

# Sample stability:

Sample stability studies were conducted using $C .$ . difficile strains spiked into negative stool matrix at low positive, moderate positive, and below LoD concentrations. Sample stability was evaluated for fresh sample storage at $2 - 8 ^ { \circ } \mathrm { C }$ , for fresh versus frozen storage at $- 7 0 ^ { \circ } \mathrm { C } + / - 1 0 ^ { \circ } \mathrm { C } .$ , and for multiple freeze/thaw cycles at $- 7 0 ^ { \circ } \mathrm { C } + / .$ - $1 0 ^ { \circ } \mathrm { C }$ . The results support sample storage at $2 - 8 ^ { \circ } \mathrm { C }$ for 48 hours, at $- 7 0 ^ { \circ } \mathrm { C } + / - 1 0 ^ { \circ } \mathrm { C }$ for 15 months, and for one freeze thaw cycle at $- 7 0 ^ { \circ } \mathrm { C } + / - 1 0 ^ { \circ } \mathrm { C }$ . More than one freeze thaw cycle is not recommended as it may lead to a higher rate of invalid results.

# d. Detection limit:

Analytical sensitivity of the ICEPlex $C .$ . difficile Assay was determined using a twofold serial dilution of two C. difficile strains that were spiked into qualified negative stool and processed according to ICEPlex C. difficile Assay Instructions for Use.

Twenty replicates at each concentration level were tested on three different ICEPlex instruments. Analytical sensitivity of the assay was defined as the lowest concentration at which at least $9 5 \%$ of all replicates were reported positive. The LoD was validated by testing an additional 20 replicates at the target concentration.

LoD of the ICEPlex C. difficile assay determined for:

• C. difficile strain 43255 (630) (Toxinotype 0): 8CFU/rxn • C. difficile strain BAA-1805 (Toxinotype III): 2CFU/rxn

LoD study results are acceptable.

# Analytical reactivity/ inclusivity

To assess the analytical inclusivity of the ICEPlex C. difficile Assay, a set of 20 additional toxigenic strains of $C .$ . difficile were spiked into pooled negative matrix at a level approximately three times above the assay LoD. Spiked samples were processed in accordance with the ICEPlex C. difficile Assay Instructions for Use. Two samples out of a total of 81 samples tested produced invalid assay results. The results of the study are as follows:

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>BAA-1382 (630)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1871 (4111)</td><td rowspan=1 colspan=1>0, A+B+binary-, NAP5</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>9689 (90556-M6S)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>700792 (14797-2)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1875 (5325)</td><td rowspan=1 colspan=1>V, A+B+, NAP7</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>51695 (BDMS 18 AN)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43598 (1470)</td><td rowspan=1 colspan=1>VIII, A-B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43600 (2149)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43599(2022)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43597</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43594(W1194)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43596(545)</td><td rowspan=1 colspan=1>I, A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>17858 (1253)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>17857 (870)</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1808</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1806</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1803</td><td rowspan=1 colspan=1>III A+B+, NAP1</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1870(4118)</td><td rowspan=1 colspan=1>III, binary+, NAP1</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>BAA-1873 (5283)</td><td rowspan=1 colspan=1>0, A+,B+,binary-</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Note: Strain BAA-1814 (Toxinotype XXII) was determined to be non-viable. PrimeraDx cannot claim inclusivity to this strain.

The inclusivity study results are acceptable.

e. Analytical specificity:

Cross reactivity:

A microbial cross reactivity study was performed with the ICEPlex C. difficile Assay using a panel of samples spiked into pooled negative clinical matrix or contrived low positive $C .$ . difficile samples. Three samples produced invalid results on the initial run. The following potential cross-reactive bacteria and viruses were tested at $\boldsymbol { 1 } \mathbf { x } \boldsymbol { 1 } 0 ^ { 7 }$ organisms $/ \mathrm { m L }$ and $\mathrm { 1 x 1 0 ^ { 6 } T C I D } _ { 5 0 } / \mathrm { m L }$ , respectively:

Abiotrophia defectiva, Acinetobacter baumannii, Aeromonas hydrophila, Bacillus cereus, Bacteroides fragilis, Bifidobacterium adolescentis, Campylobacter coli, Campylobacter jejuni subsp. jejuni,Candida albicans, Citrobacter freundii, Clostridium beijerinckii, Clostridium bifermentans, Clostridium chauvoei, Clostridium difficile 43593, Clostridium difficile 43601, Clostridium difficile 43602, Clostridium difficile 700057, Clostridium difficile BAA-1801, Clostridium haemolyticum, Clostridium histolyticum, Clostridium novyi, Clostridium orbiscindens, Clostridium perfringens, Clostridium scindens, Clostridium septicum, Clostridium sordellii, Clostridium sporogenes, Clostridium symbiosum, Clostridium tetani, Edwardsiella tarda, Enterococcus dispar, Enterobacter cloacae, Entamoeba histolytica, Enterococcus faecium vanA, Enterococcus faecalis vanB, Escherichia coli, Escherichia coli (serotype O157:H7), Escherichia fergusonii. Escherichia hermannii, Helicobacter pylori, Klebsiella pneumoniae, Lactobacillus acidophilus, Lactococcus lactis, Listeria grayi, Listeria monocytogenes, Peptostreptococcus anaerobius, Plesiomonas shigelloides, Porphyromonas asaccharolytica, Proteus mirabilis, Providencia alcalifaciens, Pseudomonas aeruginosa, Salmonella enterica serovar Typhimurium, Salmonella enterica subsp. arizonae, Salmonella enterica subsp. enterica, Serratia liquefaciens, Serratia marcescens, Shigella boydii, Shigella dysenteriae, Shigella sonnei, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus agalactiae, Vibrio cholerae, Vibrio parahaemolyticus, Yersinia enterocolitica, Adenovirus, Rotavirus, Norovirus, Enterovirus, Echovirus, Coxsackie virus, and Cytomegalovirus.

Cross reactivity was observed with Clostridium sordellii as expected due to the high sequence similarity between the C. sordellii toxin gene and the C. difficile tcdB.

The non-toxigenic C. difficile strain 43601 tested positive in the ICEPlex $C .$ . difficile Assay at a value that was just above the assay cutoff in one out of three replicates.

None of the other tested organisms reacted with the ICEPlex C. difficile Assay. None of the tested organisms interfered with the detection of low positive toxigenic $C .$ . difficile samples. The cross reactivity study and microbial interference study results are acceptable.

Interference study:

A chemical interference study was conducted using a panel of contrived samples consisting of pooled negative clinical matrix and matrix spiked with one of two strains of $C .$ . difficile (ATCC BAA-1805 or 43255) at 9 or 21 CFU/reaction, respectively (approximately three times the assay LoD). The following potentially interfering substances were tested:

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Anti-Fungal/ Anti-Itch Vaginal</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Creams/ Ointments/Suppositories</td><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Anti-HemorrhoidCreams/ Ointments</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Antacids</td><td rowspan=1 colspan=1>Calcium Carbonate/Aluminum Hydroxide/Magnesium Hydroxide</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Enemas</td><td rowspan=1 colspan=1>Mesalazine/ Mineral Oil</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>SpermicidalLubricant</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Anti-DiarrhealMedication</td><td rowspan=1 colspan=1>LoperamideHydrochloride/ BismuthSubsalicylate</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Laxatives</td><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Antibiotic</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>12.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antibiotic</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>12.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Non-Steroidal Anti-InflammatoryMedications</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>12.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Moist Towelettes</td><td rowspan=1 colspan=1>Benzalkonium Chloride,Ethanol</td><td rowspan=1 colspan=1>0.1% (v/v),1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Fecal Fat</td><td rowspan=1 colspan=1>Lipids, etc.</td><td rowspan=1 colspan=1>40% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>Glucose, Hormones,Enzymes, Ions, Iron, etc.</td><td rowspan=1 colspan=1>40% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>Mucin protein</td><td rowspan=1 colspan=1>3.5% (w/v)</td></tr></table>

None of the substances produced a false positive or false negative result with the ICEPlex C. difficile Assay at the concentrations tested. Results of the study indicated that samples with high levels of fecal fat may produce elevated rates of invalid results.

The chemical interference study results are acceptable.

Well-to-well Cross Contamination and Run-to-run Carryover Contamination:

Samples for Well-to-well Cross Contamination and Run-to-run Carryover Contamination study were prepared by extracting a series of high positive samples (with analyte concentration exceeding the concentration found in $9 5 \%$ positive samples in the intended use population) alternating with negative samples. Location of positive and negative samples on the extraction instrument was altered run-to-run. On the ICEPlex system, high positive and negative samples were run in a checkerboard fashion. In the consecutive run, the plate map was inverted to allow wells and capillaries that were running negative samples to run positive samples. The study included 6 runs: each run had 24 high positive and 24 negative samples for a total of 144 samples for all runs. The overall invalid rate was $2 \%$ . The study identified no well-to-well or run-to run contamination in the negative samples (0 false positives out of 141 valid negatives).

The well-to-well cross contamination and run-to-run carryover contamination study results are acceptable.

# f. Assay cut-off:

A panel of $4 3 ~ C .$ . difficile positive and 45 negative clinical samples from the intended use population were collected and tested with the ICEPlex C. difficile Assay. Numerical readout data was extracted from the ICEPlex system, and Receiver Operator Curve (ROC) analysis was applied to the data. An ROC specificity/ sensitivity decision plot was used to determine the cut-off level with optimal assay sensitivity and specificity. The cut-off value was determined at 12 copies per reaction.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

A prospective clinical study was conducted at three independent sites to compare performance of the ICEPlex C. difficile Assay Kit on the ICEPlex System to toxigenic C. difficile direct culture. This study tested specimens from patients suspected of gastrointestinal tract infection with toxigenic strains of C. difficile bacteria. Each of the sites tested the ICEPlex C. difficile Assay on specimens collected at the site. The specimens collected from all three sites were tested by toxigenic C. difficile direct culture at a single reference laboratory site. A total of 1103 (806 fresh, 297 frozen) samples were collected and enrolled. 97 frozen samples were excluded from the analysis due to improper storage conditions and temperature excursions. An additional 37 samples were excluded due to study protocol deviations. 969 specimens were compliant and met all protocol requirements. 952 out of 969 $( 9 8 . 2 \% )$ gave reportable results and were included in the statistical analysis. 68 out of 969 compliant samples were invalid upon initial testing $( 7 . 0 \%$ initial invalid rate), and 17 of these 68 samples remained invalid upon retest ( $1 . 8 \%$ invalid rate). Three out of the 17 unresolved invalids were reported positive by direct culture. Results were reported in percent agreements as performance was compared to direct culture. The Clopper-Pearson exact method was used to calculate confidence intervals.

Age demographics of the study population:

<table><tr><td rowspan=1 colspan=1>Age Group (years)</td><td rowspan=1 colspan=1>n (%)</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>77 (8.1%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>261(27.4%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59</td><td rowspan=1 colspan=1>283(29.7%)</td></tr><tr><td rowspan=1 colspan=1>&gt;=60</td><td rowspan=1 colspan=1>331(34.8%)</td></tr></table>

The ICEPlex $C .$ difficile Assay demonstrated the following performance characteristics:

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Direct Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>ICEPlex C.difficile Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>20a</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>17b</td><td rowspan=1 colspan=1>762</td><td rowspan=1 colspan=1>779</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>782</td><td rowspan=1 colspan=1>952</td></tr></table>

Positive Percent Agreement, PPA $( 9 5 \% \mathrm { C I } )$ 90.0% (84.5 - 94.1) Negative Percent Agreement, NPA $( 9 5 \%$ CI) 97.4% (96.1 - 98.4)

Discordant testing was performed for samples where ICEPlex C. difficile Assay and toxigenic C. difficile direct culture reported results in disagreement. Discordant analysis included microbiological isolation and PCR targeting of three appropriate regions of the toxin B gene (different recognition sites than the ones used in the ICEPlex C. difficile assay) with bi-directional DNA sequencing.

$^ \mathrm { a } _ { \ 6 }$ of 20 reported positive by ICEPlex C. difficile Assay were reported positive by discordant analysis.

b 14 of 17 reported negative by ICEPlex C. difficile Assay samples were reported

positive by discordant analysis.

Comparison by Site

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>49/ 55 = 89.1( 77.8 - 95.9)</td><td rowspan=1 colspan=1>56/ 64 = 87.5( 76.8 - 94.4)</td><td rowspan=1 colspan=1>48/ 51 = 94.1( 83.8 - 98.8)</td></tr><tr><td rowspan=1 colspan=1>NPA(95% CI)</td><td rowspan=1 colspan=1>281/ 288 = 97.6( 95.1 - 99.0)</td><td rowspan=1 colspan=1>256/ 262 = 97.7( 95.1 - 99.2)</td><td rowspan=1 colspan=1>225/ 232 = 97.0( 93.9 - 98.8)</td></tr></table>

The clinical performance study results are acceptable.

b. Clinical specificity: See section M3a   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: See section M1f

5. Expected values/Reference range:

The ICEPlex C. difficile Assay clinical study was conducted at three different US clinical site locations with a total of 952 reportable results. The number and percentage of positives by ICEPlex C. difficile assay is listed in the following table.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>ICEPlex C. difficile Assay</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Groups</td><td rowspan=1 colspan=1>Total Numberof Specimens</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>NumberNegative</td><td rowspan=1 colspan=1>PositivePercentage</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>458</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>372</td><td rowspan=1 colspan=1>18.78% (86/458)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>494</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>407</td><td rowspan=1 colspan=1>17.61% (87/494)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>952</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>779</td><td rowspan=1 colspan=1>18.17% (173/952)</td></tr></table>

# N. Instrument Name:

PrimeraDx ICEPlex system

# O. System Descriptions:

1. Modes of Operation:

To perform the test, liquid or soft stool is collected in a standard sterile container that can be sealed. Samples can be stored cold for up to 48 hours prior to testing or frozen at - ${ . 7 0 ^ { \circ } C }$ or below if not processed within 48 hours. Sample extraction is performed using bioMérieux NucliSENS easyMAG System as per the ICEPlex $C .$ . difficile Assay instruction for use. PCR reagent master mix is prepared using primer mix, PCR enzyme, PCR buffer and calibrators. In a designated well of the PCR plate, $4 0 ~ \mu \mathrm { L }$ of ICEPlex $C .$ . difficile Master Mix is aliquoted and $1 0 ~ \mu \mathrm { L }$ of extracted sample is added. Negative and positive controls are added in designated wells of the PCR plate. After the PCR plate is prepared, it is placed in the ICEPlex System to execute an instrument run for the detection of the Clostridium difficile toxin B gene (tcdB gene).

The PrimeraDx ICEPlex system employs sequential sampling and analysis of fluorescently labeled PCR products (amplicons) by means of Capillary Electrophoresis (CE). Separation of the amplicons is facilitated by appropriate primer design allowing for accurate discrimination by size and fluorescent signal after CE separation. This analysis method supports multiplexing in a single reaction well since all targets (including sample template, controls and internal calibration standards) can be detected from a single reaction well.

Real-time sampling of the amplicons is used for the construction of amplification curves and calculations of threshold cycle (Ct) similar to other real-time PCR methods.

To run an assay on the ICEPlex system, a PCR plate is assembled and placed in the thermal cycler module of the ICEPlex system and, after manually inputting sample and assay information into the integrated software, the run is initiated.

At specific PCR cycles, the capillaries and electrodes of the capillary electrophoresis module are introduced into the PCR reaction. The ICEPlex system applies voltage for a predetermined time to force negatively charged DNA molecules to enter the capillary ─ a process commonly known as electrokinetic injection. The capillaries and electrodes are then moved and are immersed in a CE buffer, voltage is applied, and capillary electrophoresis separation is performed. The PCR cycling and CE separation are timed to match one complete CE separation within two PCR cycles. All amplicons (including internal calibration controls) are separated, detected and quantified by CE using laser induced fluorescence detection. At the end of the PCR amplification, capillaries and electrodes are chemically decontaminated to remove the residual amplified PCR fragments and to recondition the capillary array for a new assay run.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Moderate

# Software Description:

The ICEPlex System is a dual mode system with Combined Functionality that can be run in IVD mode or OPEN mode. The sponsor states that the software architecture, user interface, and instrument design prevent interference of the OPEN functionality with the IVD functionality.

The software designed and distributed by PrimeraDx for operation of the ICEPlex instrument is broken down into 4 core components:

User Application (UA): customer facing software responsible for converting user requests into hardware control. Instrument Control: responsible for all communication with the hardware. initiating complex automated operations · Embedded Software: responsible for controlling hardware. · Data Analysis: responsible for converting raw data into results.

All software runs natively on a PC embedded within the ICEPlex hardware. This PC runs an OEM copy of Windows XP SP3 with at least 3GB of memory. The software is developed with Windows XP, Java 6, and Microsoft Visual Studio and $\mathrm { C } { + + }$ compiler. The off-the-shelf software runs on an ICEPlex instrument containing a Core 2 quad processor, running at 2.66 GHZ with PC10600 2GB, DDR3, 1333MHZ RAM, and a 320GB, SATA $2 . 5 "$ disk drive, with a 2x Intel gigabit Ethernet board, and a Critical Link Camera- S7031-1007/28. The instrument peripherals consist of a 104 keys keyboard, an optical mouse, a $1 9 "$ LCD touchscreen display, and barcode scanner. The end-user is prevented from making any changes to the ICEPlex Instrument software.

The software description is acceptable.

Device Hazard Analysis (DHA): Acceptable

PrimeraDx provided an acceptable DHA for the ICEPlex Analyzer instrument software. Before mitigation, the sponsor identified 10 hazards: one was considered high (potential of serious injury or death); eight were considered medium (potential of injury), and one was considered to be low (little or no potential of injury). All 10 hazards describe appropriate mitigations. The one hazard with a high degree of risk was originally described as having a low probability of occurrence; following mitigation, this was considered very low. The sponsor provided an acceptable tabular description of identified hardware and software hazards, including severity assessment and mitigations.

Architecture Design Chart: Acceptable

PrimeraDx provided acceptable flow charts showing detailed depiction of functional units and software modules.

Software Specifications: Acceptable

Software Requirements Specifications

PrimeraDx provided SRS documentation which includes a software architecture chart that describes the required software structure and content of each software module implemented. A total of 1,422 requirements were identified. These descriptions included Hardware Requirements, Interface Requirements, Software Performance and Functional Requirements for the ICEPlex Analyzer Instrument. The submitted description is acceptable.

# Software Design Specifications

PrimeraDx design specification and software development environment are documented. The detailed software description, hardware platform, development environment, and detailed information for software component are acceptable. The sponsor provided adequate SDS descriptions to show how the requirements in the SRS are implemented. The submitted description is acceptable.

Development Environment: Acceptable

The software was developed using Windows XP, Java 6, Microsoft Visual Studio and $\mathrm { C } { + + }$ compiler. The sponsor also described the non-modifiable libraries which are used for the GUI/program interface and object management and creation.

Traceability Matrix: Acceptable

The detailed Software Trace Matrix of the ICEPlex C. difficile was provided as well as the System Risk Assessment establishing a relationship between identified software risks and their mitigation actions. PrimeraDx provided comprehensive information showing traceability between Requirements, Design, and Software Validation.

Verification and Validation Testing: Acceptable

PrimeraDx provided sufficient documentation for V&V testing, including all executed software verification procedures and reports for the 10.7.3 software version. The System Level Functioning Testing protocol and report were also provided. Software Validation Activities were completed per System Level Functioning Testing, in accordance with General Principles of Software Validation; Final Guidance for Industry and FDA Staff. These pass/fail criteria and results are acceptable.

PrimeraDx provided an overview of their RLH, including the details of the RLH. Version 10.5 was first released under Design Control, and version 10.7.3 was released for clinical trial validation. The submitted description is acceptable.

Unresolved Anomalies: Acceptable

PrimeraDx provided an adequate list of Unresolved Anomalies for review. The sponsor reports 10 unresolved anomalies in version 10.7.3; all 10 unresolved anomalies have a severity of 1 (“little or no potential injury”), none of which impact the device safety or performance. The sponsor outlined the impact and mitigation required for each; no plans or timeframes for correcting the problems were indicated.

Off the Shelf Software (OTS): Acceptable

The sponsor described 24 OTS software and firmware applications along with the OS including descriptions of the version, manufacturer, part number, function, testing performed, known defects, reasons for acceptability, how the OTS links to other software and any maintenance information relevant to updates. User driven software updating is disabled by the Windows Steady state configuration. The firmware is installed by the manufacturer and updates are effectively disabled due to the need for special equipment to interface with the instrument. No user updating of third-party executables incorporated into the ICEPlex operational software are possible. Upgrading any third party software requires evaluation and testing at the manufacturer before incorporation into the released software image.

EMC and Electrical Testing: Acceptable

The instrument was tested and evaluated to demonstrate compliance with the following standards:

European EMC Directive 2004/108/EC in accordance with emissions product specific standard · En 55011:2007 immunity product specific standard EN 61326-1:2006. · UL 61010-1 :2004 Safety Requirements for Electrical Equipment for Measurement, · Control, and Laboratory Use- Part 1: General Requirements · CAN/CSA C22.2 No. 61010-2-101-04 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use -Part 2-101: Particular Requirements for · In Vitro Diagnostics (IVD) Medical Equipment

CAN/CSA C22.2 No. 61010-1-04 Safety Requirements for Electrical Equipment for

3. Specimen Identification:

The user first must verify that the onboard consumables are adequate for the run. Adequate supplies are indicated by the appearance of the buttons shown in the status bar, or by touching the System navigation menu to display the Run Status screen. The Run Status screen provides status information about the current run, the onboard consumables, and instrument messages. The user then registers an IVD assay kit from a barcode label that is included with a PrimeraDx assay IVD kit. The barcode scanner is used to register the IVD assay kit and make a corresponding assay definition available on the instrument. The assay definition needs to be updated for every new lot of the IVD assay kit. The user manually defines the following types of wells:

Sample, a sample well containing a clinical specimen or test sample;   
· Negative control;   
· Positive control;   
· Buffer, a well without any sample containing only 1X Injection buffer;   
· Buffer/Lock, a well without any sample containing only 1X Injection buffer that has been locked by the system due to a well-specific quality evaluation of the installed cartridge.

In the software UI, the user selects a plate map location, for example A1, and enters the sample ID or Accession number or designates plate map locations as Negative, Positive control or Buffer. Each sample or control well must have a unique accession number/identification number.

4. Specimen Sampling and Handling:

Appropriate Specimen type for testing

This assay is intended for use with liquid or soft raw stool specimens. Do not use on wellformed stools or on other types of specimens.

# Collecting the specimen

Standard stool collection and handling procedures are appropriate to obtain raw stool.   
Obtained sample must be placed in a sterile container that can be adequately sealed.

# Storing Specimens

Samples should be tested as soon as possible. It is possible to store specimens refrigerated $\left( 2 { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 48 hours prior to testing. If sample cannot be processed within 48 hours, store frozen at $- 7 0 ^ { \circ } \mathrm { C }$ or below.

5. Calibration:

Each individual PCR reaction run for the ICEPlex C. difficile Assay is internally controlled by utilizing the included controls. In-reaction Controls (provided as a separate kit component or included as a part of reagent) consist of the following:

Calibration Control – a group of ICEPlex specific elements used to align electropherograms and assign identities of the target peaks. It also controls for the integrity of the kit reagents and the presence of PCR inhibitors in a given sample. Internal Control – Non-target nucleic acid that is co-extracted and co-amplified with the tcdB target. It controls for nucleic acid extraction efficiency, for the integrity of the reagents and for the presence of PCR inhibitors in a given sample. The Internal Control needs to be spiked into each sample before extraction.

6. Quality Control:

The ICEPlex system reports the reason for a sample or control result to be called “Invalid”. If a control sample is invalid due to a reason unrelated to its function as a control, it is excluded from consideration. To monitor run validity, the user is required to include the following external controls (additional control PCR reactions) with the run (duplicates are recommended):

Negative Control, substitute S.T.A.R. buffer for the clinical specimen and process normally through the extraction system and on the ICEPlex Instrument. · Positive Control, this control is used for the PCR stage only, it should not be processed for nucleic acids extraction. In the positive control reaction, substitute extracted sample with provided Positive Control. · Known Positive Sample, is required to include previously characterized positive sample or simulated sample with every easyMAG extraction run and include it in the subsequent ICEPlex Instrument run to verify successful lysis.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.