# 510(k) SUMMARY

# General Information

Submitted by:

QIAGEN GmbH Qiagen Strasse 1 Hilden, Germany D-40724

# Contact Person:

Kimberly Mapp, Ph.D. Manager, Regulatory Affairs 1201 Clopper Road Gaithersburg, MD 20878

Phone: 301.944.7817   
Fax: 240.686.3847   
Email: kimberly.mapp@qiagen.com

Date Prepared: April 4, 2014

Device Name: artus® C. difficile QS-RGQ MDx Kit

Trade Name: artus® C. difficile QS-RGQ MDx Kit Common Name: C. difficile nucleic acid amplification test assay Classification: Class I

# Predicate Device

# Manufacturer Quidel

Product Name Quidel Molecular Direct C. difficile Assay

# Device Description

The artus C. difficile QS-RGQ MDx Kit assay uses PCR to generate an amplified product from the tcdA and tcdB/tcdBv genes of toxigenic C. difficile DNA in clinical specimens. Samples are extracted and prepared using the QlAsymphony SP instrument with the QIAsymphony DSP Virus/Pathogen Mini Kit, followed by assay setup on the QlAsymphony AS. Amplification and detection are carried out using the artus C. difficile QS-RGQ MDx Kit with the Rotor-Gene Q MDx (RGQ MDx) and Rotor-Gene AssayManager software. The presence of a toxigenic C. difficile target sequence is indicated by the fluorescent signal generated through the use of fluorescently labeled oligonucleotide probes. The probes do not generate a signal unless they are specifically bound to the amplified product. The amplification cycle at which fluorescent signal is detected by the RGQ MDx is inversely proportional to the toxigenic C. difficile DNA target concentration present in the original specimen. A bacterial species unrelated to toxigenic C. difficile is introduced into each specimen during sample preparation to serve as an internal control. The internal control bacteria are lysed simultaneously with toxigenic C. difficile in the specimen, and amplified in the same reaction as the C. difficile targets using $\mathrm { P C R } ,$ and serve to demonstrate that the entire assay process has proceeded correctly for each specimen.

# Intended Use

The artus C. difficile QS-RGQ MDx Kit is an in vitro polymerase chain reaction (PCR) assay for use on the QlAsymphony RGQ MDx system for the qualitative detection of toxigenic Clostridium difficile toxin A and toxin B genes in human liquid or soft stool specimens from patients suspected of having Clostridium difficile associated disease. The test is intended to be used directly on patient samples.

The artus C. difficile QS-RGQ MDx Kit is intended to be used to aid in diagnosis of Clostridium difficile infection.

# Comparison of the artus® C. difficile QS-RGQ MDx Kit and the Predicate Device

The artus® C. diffcile QS-RGQ MDx Kit is substantially equivalent to the predicate device:

K123998: Quidel Molecular Direct C. difficile Assay

Similarities and differences between the artus® C. difficile QS-RGQ MDx Kit (72) and the predicate device are shown in Table 1.

Table 1: Comparison of the artus® C. difficile QS-RGQ MDx Kit with the predicate device.   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">arlus C. difficile QS-RGQ MDxKit</td><td colspan="1" rowspan="1">Quidel Molecular Direct C. difficileAssay</td></tr><tr><td colspan="1" rowspan="1">510(k) No.</td><td colspan="1" rowspan="1">TBD</td><td colspan="1" rowspan="1">K123998</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.3130</td><td colspan="1" rowspan="1">866.3130</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OZN</td><td colspan="1" rowspan="1">OZN</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class H</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="3" rowspan="1"> Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The artus C. difficile QS-RGQMDxKit isan  invitropolymerase chain reaction (PCR)assay   for   use   on   theQIAsymphony   RGQ   MDxsystem  for  the  qualitativedetection       of      toxigenicClostridium difficile toxin A andtoxin B genes in human liquid orsoft stool specimens from patientssuspected of having Clostridiumdifficile associated disease. Thetest is intended to be used directlyon patient samples.The artus C. difficile QS-RGQMDx Kit is intended to be used toaid in diagnosis of Clostridiumdifficile infection.</td><td colspan="1" rowspan="1">The Quidel Molecular DirectC.difficile Assay is aqualitative, multiplexed in vitrodiagnostic test for the directdetection of toxin A gene(tcdA) or toxin B gene (tcdB)sequences of toxigenic strainsof Clostridium difficile fromunformed (liquid or soft) stoolspecimens  collected  frompatients suspected of havingClostridium            difficile-Associated Disease (CDAD).The Quidel Molecular DirectC. difficile Assay is a real-timePCR   test   and   utilizes.proprietary sample preparationwith  fluorescentlylabeledprimers and probes. The assaycan be performed using eitherthe    Life    TechnologiesQuantStudio® Dx; the AppliedBiosystems 7500 Fast Dx, orthe Cepheid SmartCycler II, todetect the toxin gene sequencesassociated     with     toxin-producing C. difficile strains.The assay is intended to beperformed directly on CDAD-suspected stool specimens, andis indicated for use as an aid inthe diagnosis of CDAD.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Liquid or soft stool</td><td colspan="1" rowspan="1">Unformed (liquid or soft) stool</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Toxin A gene (icdA)Toxin B gene (tcdB and tcdBv)</td><td colspan="1" rowspan="1">Toxin A gene (tcdA)Toxin B gene (tcdB)</td></tr><tr><td colspan="1" rowspan="1">Amplificationand DetectionTechnology</td><td colspan="1" rowspan="1">Real-time       PCR        DNAamplification</td><td colspan="1" rowspan="1">Real-time       PCR       DNAamplification</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Positive    Control,   NegativeControl and Internal Controlincluded in the kit.</td><td colspan="1" rowspan="1">Process Control included in thekit.Positive and Negative Controlsnot included in kit: separatecontrol kit available for sale.</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction andAssay Setup</td><td colspan="1" rowspan="1">Assay uses the QlAsymphonySP/AS for automated samplepreparation and assay setup.</td><td colspan="1" rowspan="1">Assay uses proprietary samplepreparation buffer and manualassay setup.</td></tr><tr><td colspan="1" rowspan="1">Amplificationand DetectionInstrumentSystem</td><td colspan="1" rowspan="1">Assay uses the Rotor-Gene QMDx.</td><td colspan="1" rowspan="1">Assay can be performed usingeither the:Life TechnologiesQuantStudio DxApplied Biosystems 7500 FastDxCepheid SmartCycler II</td></tr></table>

The differences between the artus C. difficile QS-RGQ MDx Kit and Quidel Molecular Direct C. difficile assays are primarily centered around the control strategy and instrumentation. These differences do not affect substantial equivalency of the artus C. difficile QS-RGQ MDx Kit and the Quidel Molecular Direct C. difficile Assay. Both assays detect toxigenic C. difficile, and the assays have comparable intended uses. The differences noted above do not change the intended use and do not raise questions of safety and effectiveness.

# Performance Characteristics - Non-Clinical Studies

# Analytical Sensitivity (Limit of detection)

The limit of detection (LoD) was assessed for the arius C. difficile QS-RGQ MDx Kit using 3 toxigenic Clostridium difficile strains: NAP-1/BI/027 strain, toxinotype III $\mathbf { A } { + } \mathbf { B } { + }$ $( \mathbf { A } \mathbf { T } \mathbf { C } \mathbf { C } \mathbf { \textcircled { 8 } }$ BAA-1870); 1470 strain, toxinotype VIII A- $\mathbf { B } +$ (ATCC 43598): and VPI 10463 strain, toxinotype 0 $\mathbf { A } { \ + } \mathbf { B } { \ + }$ (ATCC 43255). The LoD is defined as the toxigenic $C .$ difficile bacterial titer $\scriptstyle ( \mathrm { C F U / m L } )$ detected with a probability of $9 5 \%$ or greater and was determined by probit analysis. The results, representative of the analytical sensitivity of the artus C. difficile QS-RGQ MDx Kit, are summarized in Table 2.

Table 2: Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD (95%CI)</td></tr><tr><td rowspan=1 colspan=1>C. difficile NAP1ATCC BAA-1870, strain:4118</td><td rowspan=1 colspan=1>7.9 CFU/mL(6.1-15.0)</td></tr><tr><td rowspan=1 colspan=1>C. difficile 1470ATCC 43598, strain: 1470</td><td rowspan=1 colspan=1>11.2 CFU/mL (8.7-16.8)</td></tr><tr><td rowspan=1 colspan=1>C. difficile 10463ATCC 43255, strain:10463</td><td rowspan=1 colspan=1>2.8 CFU/mL (2.1-4.2)</td></tr></table>

# Analytical Reactivity

The analytical reactivity of the artus C. difficile QS-RGQ MDx Kit was assessed to determine whether the kit could detect a broad range of toxigenic C. difficile strains representing temporal and geographical diversity. A total of 27 strains and characterized clinical isolates were diluted in TE buffer to $2 { - } 3 \mathbf { x }$ LOD of the reference strain and tested with the artus C. difficile QS-RGQ MDx Kit. C. difficile target was detected in all strains tested Table 3.

Table 3: Strains Tested in Analytical Reactivity   

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Toxinotype</td><td colspan="1" rowspan="1">Origin</td></tr><tr><td colspan="1" rowspan="1">ATCC 17857</td><td colspan="1" rowspan="1">870</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC 17858</td><td colspan="1" rowspan="1">1253</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC 43594</td><td colspan="1" rowspan="1">W1194</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43596</td><td colspan="1" rowspan="1">545</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43599</td><td colspan="1" rowspan="1">2022</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43600</td><td colspan="1" rowspan="1">2149</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 51695</td><td colspan="1" rowspan="1">BDMS18AN</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Becton DickinsonMicrobiology Systems,Johns Hopkins Univ.Hosp. Lab</td></tr><tr><td colspan="1" rowspan="1">ATCC 700792</td><td colspan="1" rowspan="1">14797-2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces;Michigan, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC 9689</td><td colspan="1" rowspan="1">90556-M6S</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1382</td><td colspan="1" rowspan="1">630</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Switzerland</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1805</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">III</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1871</td><td colspan="1" rowspan="1">4111</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">Human; New Jersey,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1872</td><td colspan="1" rowspan="1">4206</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">Human; Maine, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1873</td><td colspan="1" rowspan="1">5283</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">Human; New York,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1874</td><td colspan="1" rowspan="1">4205</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">Human; Oregon, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1875</td><td colspan="1" rowspan="1">5325</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">Human; Georgia, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2155</td><td colspan="1" rowspan="1">LBM0801058</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human; New Mexico,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2156</td><td colspan="1" rowspan="1">LBM0801040</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human; CambridgeUK</td></tr><tr><td colspan="1" rowspan="1">CCUG 20309</td><td colspan="1" rowspan="1">8864</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Birmingham, UK</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate278</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate464</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">4092</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate5572</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate3430</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IX</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate1753</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate5090</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XXI</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate3130</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XXII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr></table>

# Cross-Reactivity and Microbial Interference

A panel of microorganisms that may be present in patient specimens was tested to determine whether these microorganisms interfered with the detection of tcdA or tcdB targets or were cross-reactive with the artus C. difficile QS-RGQ MDx Kit. Organisms were tested at a target concentration of approximately $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { ~ C F U / m l }$ for bacteria and fungi $\sigma \mu \textbf { \em x } 1 0 ^ { 5 }$ units/ml for viruses separately in the presence of $2 { - } 3 \mathbf { x }$ LOD of each of three C. difficile strains: NAP-1/BI/027 strain, 1470 strain, and VPI 10463 strain. None of the potential interfering organisms cross-reacted or interfered with the detection of any of the $3 ~ C .$ difficile strains by the artus C. difficile QS-RGQ MDx Kit (Table 4). Crossreactivity for Clostridium botulinum was analyzed in silico and predicted no cross reactivity or microbial interference for the artus C. difficile QS-RGQ MDx Kit.

Table 4: Organisms Tested in Cross Reactivity and Microbial Interference   

<table><tr><td colspan="1" rowspan="1">Organism Tested</td><td colspan="1" rowspan="1">Source ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC 7966</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 13472</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ZMC 0601533</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43479</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 33559</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp. jejuni</td><td colspan="1" rowspan="1">ATCC 33292</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">ATCC 638</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">ATCC 19398</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">ATCC 9650</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">ATCC 19402</td></tr><tr><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">ATCC 49531</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ATCC 13124</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">ATCC 35704</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">ATCC 12464</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">ATCC 9714</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43593</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43601</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">ATCC 15579</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanB)</td><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">ATCC 700927</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori DNA</td><td colspan="1" rowspan="1">ATCC 43504D-5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 33496</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ZMC 0801534</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Salmonella choleraesuis (Typhimurium)</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ATCC 11835</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 27541</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">ZMC 08101 10 CF</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">ZMC0810041 CF</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">ZMC0810086 CF</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">ZMC0810047 CF</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">ZMC0810023 CF</td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus</td><td colspan="1" rowspan="1">ZMC0810075 CF</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ZMC 0810003 CF</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">Promega G3041</td></tr></table>

# Precision

The precision of the artus C. difficile QS-RGQ Kit was assessed using a 7-member panel consisting of $2 \ C .$ difficile strains: NAP-1/Bl/027 strain, toxinotype III $\mathbf { A } { + } \mathbf { B } { + }$ (ATCC BAA-1870) and 1470 strain, toxinotype VIII $\mathbf { A } { \cdot } \mathbf { B } { + }$ (ATCC 43598). Panel members were initially diluted in TE buffer then tested in Buffer ATL containing negative stool matrix with a single strain present (NAP-1/Bl/027 or 1470) at 3 concentrations; positive (approximately 23x LoD), low positive (Ix LoD), and high negative (<lx LoD). A seventh panel member (negative) was prepared using TE buffer only and also tested in Buffer ATL containing negative stool matrix. The data obtained were used to determine the mean $\mathbb { C } _ { \mathtt { T } }$ ,standard deviation (SD) and the coefficient of variation $( \% C V )$ for each target and the internal control.

For the within laboratory repeatability study, the seven-member panel was tested in replicates of three, once a day for a total of twelve days (for a total of 262 data points for the 12 runs). The testing was conducted by two operators using one instrument (QS/AS RGQ MDx) and one reagent kit lot (Table 5).

Table 5: Within Laboratory Repeatability Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Internal Control</td><td rowspan=1 colspan=3>tcdA</td><td rowspan=1 colspan=3>tcdB</td></tr><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>NAPI Positive</td><td rowspan=1 colspan=1>29.78</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.64%</td><td rowspan=1 colspan=1>30.84</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.52%</td><td rowspan=1 colspan=1>33.58</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.91%</td></tr><tr><td rowspan=1 colspan=1>NAPI LowPositive</td><td rowspan=1 colspan=1>30.07</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.82%</td><td rowspan=1 colspan=1>32.19</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>3.08%</td><td rowspan=1 colspan=1>34.92</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>1.94%</td></tr><tr><td rowspan=1 colspan=1>NAP1 HighNegative</td><td rowspan=1 colspan=1>29.99</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.41%</td><td rowspan=1 colspan=1>34.21</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.66%</td><td rowspan=1 colspan=1>36.18</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.64%</td></tr><tr><td rowspan=1 colspan=1>1470 Positive</td><td rowspan=1 colspan=1>29.85</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.73%</td><td rowspan=1 colspan=1>30.95</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.78%</td><td rowspan=1 colspan=1>33.72</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.33%</td></tr><tr><td rowspan=1 colspan=1>1470 LowPositive</td><td rowspan=1 colspan=1>29.89</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.39%</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.03%</td><td rowspan=1 colspan=1>35.13</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.80%</td></tr><tr><td rowspan=1 colspan=1>1470 HighNegative</td><td rowspan=1 colspan=1>29.92</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.76%</td><td rowspan=1 colspan=1>34.14</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.92%</td><td rowspan=1 colspan=1>36.39</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.07%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30.00</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.46%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

To measure site-to-site reproducibility, the 7-member panel was run by 2 users at each of 3 sites (IMDx and 2 external sites). Each of the 2 users performed 5 runs on alternating testing days. Panel members were tested in replicates of 3 that were randomized and blinded to the user. A single QlAsymphony RGQ instrument system and one lot of the artus C. difficile QS-RGQ Kit were used at each site to conduct the study (Table 6).

Table 6: Site-to-Site Reproducibility Study Results   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Internal Control</td><td colspan="3" rowspan="1">tcdA</td><td colspan="3" rowspan="1">tcdB</td></tr><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">PanelMember</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="12">NAP-1/B1/027</td><td colspan="1" rowspan="4">Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.61</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">31.85</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">34.32</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">2.02</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.58</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.65</td><td colspan="1" rowspan="1">32.16</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">34.43</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">2.05</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.55</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">31.97</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">2.65</td><td colspan="1" rowspan="1">34.87</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">2.25</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.58</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">32.00</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">2.49</td><td colspan="1" rowspan="1">34.53</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">2.19</td></tr><tr><td colspan="1" rowspan="4">LowPositive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.63</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">33.38</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">2.14</td><td colspan="1" rowspan="1">35.58</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.55</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.73</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.94</td><td colspan="1" rowspan="1">33.27</td><td colspan="1" rowspan="1">0.81</td><td colspan="1" rowspan="1">2.45</td><td colspan="1" rowspan="1">35.01</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.52</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.86</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">33.07</td><td colspan="1" rowspan="1">0.84</td><td colspan="1" rowspan="1">2.53</td><td colspan="1" rowspan="1">35.45</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">1.96</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.74</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.78</td><td colspan="1" rowspan="1">33.24</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">2.38</td><td colspan="1" rowspan="1">35.36</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">1.76</td></tr><tr><td colspan="1" rowspan="4">HighNegative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.71</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">34.21</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">35.88</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.59</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">1.09</td><td colspan="1" rowspan="1">34.21</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">35.92</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.64</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.53</td><td colspan="1" rowspan="1">34.17</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">1.68</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.65</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">34.19</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">35.91</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.43</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Internal Control</td><td colspan="3" rowspan="1">tcdA</td><td colspan="3" rowspan="1">•          tcdB</td></tr><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">PanelMember</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="12">1470</td><td colspan="1" rowspan="4">Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.67</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">32.22</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">2.67</td><td colspan="1" rowspan="1">34.74</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.76</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.69</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">31.74</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">2.99</td><td colspan="1" rowspan="1">34.66</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.14</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.86</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">31.81</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">2.45</td><td colspan="1" rowspan="1">34.95</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.01</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.74</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.35</td><td colspan="1" rowspan="1">31.92</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">2.77</td><td colspan="1" rowspan="1">34.78</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">1.99</td></tr><tr><td colspan="1" rowspan="4">LowPositive•</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.74</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">33.17</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">2.83</td><td colspan="1" rowspan="1">35.61</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.38</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.51</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">33.06</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">35.62</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.7</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">33.41</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">3.22</td><td colspan="1" rowspan="1">35.77</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.29</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.65</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">35.65</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.48</td></tr><tr><td colspan="1" rowspan="4">HighNegative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.76</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">34.64</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">36.41</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.55</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.46</td><td colspan="1" rowspan="1">34.15</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">35.82</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.44</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">34.63</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.58</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.49</td><td colspan="1" rowspan="1">34.49</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">36.12</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.16</td></tr><tr><td colspan="2" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">30.68</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30.66</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.55</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30.67</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">33.58</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">overall</td><td colspan="1" rowspan="1">30.67</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">33.58</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# Target Carryover Study

Absence of carryover between samples for the entire workflow was proven by performing 5 runs with alternating high positive (formulated at a concentration which exceeded that of organisms found in $9 5 \%$ of specimens from diseased patients in the intended use population) and negative samples. All samples were detected correctly, generating a carryover rate of $0 . 0 \%$ .

# Interfering Substances

A panel of 23 substances that may be present in patient specimens (Table 7) was tested to determine whether these substances interfered with the performance of the artus C. difficile QS-RGQ MDx Kit. Three toxigenic $C .$ difficile strains: NAP-1/BI/027 strain, 1470 strain, and VPI 10463 strain, were diluted to approximately $_ { 2 - 3 \mathbf { x } }$ LoD and spiked with each potentially inhibitory substance. None of the substances showed an inhibitory effect on the detection of C. difficile by the artus C. difficile QS-RGQ MDx Kit.

Table 7: Potentially Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>ConcentrationTested*</td></tr><tr><td rowspan=1 colspan=1>Anti-fungal</td><td rowspan=1 colspan=1> Miconazolenitrate cream</td><td rowspan=1 colspan=1>Miconazole nitrate</td><td rowspan=1 colspan=1>2% w/v</td></tr><tr><td rowspan=3 colspan=1>Cream/Suppositories</td><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>Hydrocortizone</td><td rowspan=1 colspan=1>2% w/v</td></tr><tr><td rowspan=1 colspan=1>Zinc oxide</td><td rowspan=1 colspan=1>Zinc oxide</td><td rowspan=1 colspan=1>40% w/v</td></tr><tr><td rowspan=1 colspan=1>Vaseline</td><td rowspan=1 colspan=1>Petroleum jelly</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Anti-hemorrhoidcreams</td><td rowspan=1 colspan=1>Hemorrhoid gel</td><td rowspan=1 colspan=1>Phenylephrinehydrochloride</td><td rowspan=1 colspan=1>2% w/v</td></tr><tr><td rowspan=1 colspan=1>Condoms</td><td rowspan=1 colspan=1>Condoms</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>MoistTowelettes</td><td rowspan=1 colspan=1>Moist Towelettes</td><td rowspan=1 colspan=1>BenzalkoniumChloride</td><td rowspan=1 colspan=1>0.12% w/v</td></tr><tr><td rowspan=4 colspan=1>Antacids</td><td rowspan=1 colspan=1>Gaviscon</td><td rowspan=1 colspan=1>Aluminum hydroxide,Magnesium carbonate</td><td rowspan=1 colspan=1>0.1 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Tums</td><td rowspan=1 colspan=1>Ca carbonate</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Tagamet</td><td rowspan=1 colspan=1>Cimetidine</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Prilosec (delayedrelease</td><td rowspan=1 colspan=1>Omeprazolemagnesium</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Enemas</td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=2 colspan=1>Anti-DiarrhealMedication</td><td rowspan=1 colspan=1>Imodium</td><td rowspan=1 colspan=1>Loperamide HCI</td><td rowspan=1 colspan=1>0.00667 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Pepto Bismol</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>0.87 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Laxative</td><td rowspan=1 colspan=1>ExLax</td><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>0.1 mg/mL</td></tr><tr><td rowspan=2 colspan=1>Antibiotics</td><td rowspan=1 colspan=1>Vancomycin HCI</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>12.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>14 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Anti-inflammatory</td><td rowspan=1 colspan=1>Naproxen Sodium(Aleve)</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>14 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>Glucose, hormones,enzymes, iron, etc.,</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=3 colspan=1>FecalComponents</td><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Palmitic acid</td><td rowspan=1 colspan=1>Palmitic acid</td><td rowspan=1 colspan=1>2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Stearic acid</td><td rowspan=1 colspan=1>Stearic acid</td><td rowspan=1 colspan=1>4 mg/mL</td></tr><tr><td rowspan=1 colspan=1>MRI ContrastAgents</td><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>5 mg/mL</td></tr></table>

\* Represents physiologically relevant concentrations of substances

# Performance Characteristics - Clinical Studies

The performance of the artus C. difficile QS-RGQ MDx Kit was evaluated at 3 external testing sites using C. difficile patient samples from 5 geographically diverse locations within the United States in 2013. The artus C. difficile QS-RGQ MDx Kit was compared to direct and/or enriched culture. The tables below present the data from these studies.

# Assay vs. Direct and Enriched Culture Comparison

A total of 759 liquid or soft stool specimens were collected from patients suspected of having Clostridium difficile-associated disease. Ten (10) specimens were missing artus and culture results and were withdrawn from the study. Of the 749 specimens with artus results reported, sixteen (16) specimens reported an invalid result $( 2 . 1 4 \% )$ with the artus C. difficile QS-RGQ MDx Kit. Of these samples, 8 returned valid results upon retesting (all negative) and 8 reported a final invalid result for the artus C. difficile QS-RGQ MDx Kit $( 1 . 0 7 \% )$ . The remaining 741 specimens with valid artus results were included in the final data set and analyzed for product performance.

Assay vs. Enriched Culture Comparison

A total of 741 specimens were tested by both the artus C. difficile QS-RGQ MDx Kit and enriched toxigenic culture. The overall assessment of sensitivity and specificity versus enriched toxigenic culture is shown in Table 8.

Table 8: Clinical Performance of artus C. difficile QS-RGQ MDx Kit vs. Enriched Toxigenic Culture   

<table><tr><td rowspan=1 colspan=5>Combined Sites - Combined Ages</td></tr><tr><td rowspan=5 colspan=1>artus C. difficileQS-RGQ MDxKit</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Enriched Toxigenic Culture</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>17*</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>13**</td><td rowspan=1 colspan=1>597</td><td rowspan=1 colspan=1>610</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>741</td></tr></table>

Sensitivity 90% 83% - 94% Specificity 97% 96% - 98% Positive Predictive Value 87% 80% - 92% Negative Predictive Value 98% 96% - 99% Prevalence 17% 15% - 20% $\yen 17$ discordant specimens (arlus C. difficile QS-RGQ MDx Positive, Enriched Toxigenic Culture Negative) reported were analyzed by alternative PCR followed by bi-directional sequcncing and the result was that 12 out of 17 were positive or toxigenic C. difficile, agrecing with the artus C. difficile QS-RGQ MDx result.

\*\* 12 discordant spccimens (artus C. difficile QS-RGQ MDx Negativc, Enriched Toxigenic Culture Posiive) reported were analyzed by alternative PCR followed by bi-directional sequencing and the result was that 10 out of 12 were ncgative for toxigenic C. dificile. agreeing with the artus C. difficile QS-RGQ MDx result. The remaining 1 discordant specimen was unavailable for testing.

# Assay vs. Direct Toxigenic Culture Comparison

There were a total of 699 specimens for which a direct toxigenic culture result was available. The overall assessment of sensitivity and specificity versus direct toxigenic culture is shown in Table 9.

Table 9: Clinical Performance of artus C. difficile QS-RGQ MDx Kit vs. Direct Toxigenic Culture   

<table><tr><td rowspan=1 colspan=5>Combined Sites - Combined Ages</td></tr><tr><td rowspan=5 colspan=1>artus C. difficileQS-RGQ MDxKit</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Direct Toxigenic Culture</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>21*</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1**</td><td rowspan=1 colspan=1>593</td><td rowspan=1 colspan=1>594</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>*        614       </td><td rowspan=1 colspan=1>699</td></tr></table>

Sensitivity 99% 94% - 100% Specificity 97%. 95% - 98% Positive Predictive Value 80% 71% - 87% Negative Predictive Value 100% 99% - 100% Prevalence 12% 10% - 15% $^ { \ddagger 1 9 }$ discordant specimens (arfus C. difficile QS-RGQ MDx Positive. Direct Toxigenic Culture Negative) repored were analyzed by alternative PCR followed by bi-directional sequencing and the result was that 14 out of 19 were positive for toxigenic C. difficile, agrecing with the artus C. diflicile QS-RGQ MDx result. The remaining 2 specimens were unavailable for testing.

\*\*The 1 discordant specimen (artus C. difficile QS-RGQ MDx Negative. Direct Toxigenic Culture Positive was unavailable or testing.

# Conclusions

The artus® C. difficile QS-RGQ MDx Kit is substantially equivalent to the legally marketed Quidel Molecular Direct C. difficile Assay.

QIAGEN GMBH   
KIMBERLY MAPP, Ph.D   
MANAGER, REGULATORY AFFAIRS   
1201 CLOPPER ROAD   
GAITHERSBURG MD 20878

Re: K133936 Trade/Device Name: artus C. difficile QS-RGQ MDx Kit Regulation Number: 21 CFR 866.3130 Regulation Name: Clostridium diffcile toxin gene amplification assay Regulatory Class: II Product Code: OZN, 001 Dated: January 6, 2014 Received: January 6, 2014

Dear Dr. Mapp:

We have reviewed your Section ${ \mathfrak { s l o } } ( { \mathbf k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638 2041or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part .    v n 1 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Offce of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

# Uwe Scherf -S for

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K133936

Device Name artus® C. difficile QS-RGQ MDx Kit

The rt C.ifficile QS-RGQ MDx Kt is a n vi polyee chai acin (PCR)assa rus e QAymphony RGQ MDx syste or the qualitative detection  toxigenic Clostridium difficile toxi Aand toxin B The test is intended to be used directly on patient samples.

Th ru C.ifficil QS-RGQ MDx Kiintend ese iini Clostriiffein

# This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.'

The burden tmeor hiscollection  nomation is stated averagehours per esponse,includ time to reviewinstructions, search existing data sources, gather and maintain the data needed and coplet and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

'An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."