# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

# B. Purpose for Submission:

To obtain a substantial equivalent determination for a premarket notification for the illumigene Pertussis DNA Amplification Assay and illumigene Pertussis External Control

# C. Measurand:

DNA target sequence: IS481 Insertion Sequence

# D. Type of Test:

The illumigene Pertussis DNA Amplification Assay is a qualitative in vitro diagnostic device for the direct detection of Bordetella pertussis DNA in human nasopharyngeal swab (NP) samples. The assay uses isothermal loop-mediated DNA amplification (LAMP) technology that targets the IS481 insertional element of the Bordetella pertussis genome.

# E. Applicant:

Meridian Bioscience Inc.

F. Proprietary and Established Names: illumigene® Pertussis DNA Amplification Assay illumigene® Pertussis External Control

# G. Regulatory Information:

1. Regulation section: 866.3980: Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: II

3. Product code: OZZ, Bordetella pertussis DNA Assay System

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended Use:

The illumigene $^ \mathrm { \textregistered }$ Pertussis DNA Amplification Assay, performed on the illumipro- $. I O ^ { \mathrm { T M } }$ , is a qualitative in vitro diagnostic test for the direct detection of Bordetella pertussis in human nasopharyngeal swab samples taken from patients suspected of having respiratory tract infection attributable to Bordetella pertussis.

The illumigene Pertussis assay utilizes loop-mediated isothermal DNA amplification (LAMP) technology to detect Bordetella pertussis by targeting the IS481 insertional element of the Bordetella pertussis genome. The IS481 insertional element can also be found in Bordetella holmesii and Bordetella bronchiseptica strains. Respiratory infection with Bordetella pertussis, Bordetella holmesii or Bordetella bronchiseptica may yield positive test results in IS481 assays. B. holmesii infection may cause clinical illness similar to $B$ . pertussis, and mixed outbreaks involving both B. pertussis and B. holmesii infection have been reported. Additional testing should be performed if necessary to differentiate $B _ { \cdot }$ . holmesii and $B$ . pertussis. B. bronchiseptica is a rare cause of infection in humans. When clinical factors suggest that $B$ . pertussis may not be the cause of respiratory infection, other clinically appropriate investigation(s) should be carried out in accordance with published guidelines.

Negative results for the illumigene $^ \mathrm { \textregistered }$ Pertussis DNA Amplification Assay do not preclude Bordetella pertussis infection and positive results do not rule out co-infection with other respiratory pathogens. Results from the illumigene Pertussis assay should be used in conjunction with information obtained during the patient’s clinical evaluation as an aid in diagnosis of Bordetella pertussis infection and should not be used as the sole basis for treatment or other patient management decisions.

illumigene Pertussis is intended for use in hospital, reference or state laboratory settings.   
The device is not intended for point-of-care use.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For Prescription Use Only

4. Special instrument requirements: illumipro $\mathbf { \nabla } \cdot I O ^ { \mathrm { T M } }$ Automated Isothermal Amplification and Detection System

# I. Device Description:

The illumigene Molecular Diagnostic Test System is comprised of the illumigene $^ \mathrm { \textregistered }$ Pertussis DNA Amplification Assay Test Kit, the illumigene Pertussis External Control Kit and the illumipro‐10 Automated Isothermal Amplification and Detection System.

The illumigene Pertussis assay utilizes loop‐mediated isothermal amplification (LAMP) technology to detect the presence of Bordetella pertussis in human nasopharyngeal swab specimens. The assay targets a 198 base pair sequence of the Bordetella pertussis genome residing in a region of the IS481 insertional element sequence. Each illumigene Pertussis assay is completed using an illumigene Assay Control/Negative Control Reagent containing Control material, an illumigene Sample Buffer tube, an illumigene Pertussis Test Device and Mineral Oil. Nasopharyngeal swab specimens are eluted with illumigene Sample Buffer. Eluted sample is added to an Assay/Control/Negative Control Tube, vortexed and heated at $9 5 \mathrm { { ^ \circ C } }$ for 10 minutes. The heat‐treated Specimen/Control sample is added to the illumigene Test Device. Mineral oil is added to the illumigene Test Device to prevent evaporation. DNA amplification occurs in the illumigene Test Device.

The illumipro‐10 heats each illumigene Pertussis Test Device containing prepared Sample and Control material, facilitating amplification of target DNA. When B. pertussis is present in the specimen, target DNA is amplified and magnesium pyrophosphate is generated, forming a precipitate in the reaction mixture.

The illumipro ‐10 monitors the absorbance characteristics of the reaction solutions at the assay Run Start $\mathrm { ( S i g n a l _ { i n i t i a l } , \ S _ { i } }$ and at the assay Run End $\mathrm { ( S i g n a l _ { f i n a l } , \Omega \Omega }$ $\mathrm { S _ { f } } )$ . The illumipro ‐10 calculates the r a t i o o f $\mathrm { S _ { f } }$ to $\mathrm { S _ { i } }$ and compares the ratio to an established cut‐off value. The illumipro ‐10 software calculates this ratio for both the TEST chamber and the CONTROL chamber.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Respiratory Panel, Biofire Diagnostics, Inc.

2. Predicate 510(k) number(s): K123620

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>illumigene Pertussis DNAAmplification Assay</td><td rowspan=1 colspan=1>FilmArray RespiratoryPanel</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative detection of aBordetella pertussis DNAsequence fromnasopharyngeal specimens</td><td rowspan=1 colspan=1>Qualitative detection of aBordetella pertussis DNAsequence fromnasopharyngeal specimens(see below for differences)</td></tr><tr><td rowspan=1 colspan=1>Detection</td><td rowspan=1 colspan=1>Self Contained andAutomated</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Internal Control Provided</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>illumigene Pertussis DNAAmplification Assay</td><td rowspan=1 colspan=1>FilmArray RespiratoryPanel</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Multiple viral analytes aswell as Bordetellapertussis, Mycoplasmapneumoniae, andChlamydophilapneumoniae</td></tr><tr><td rowspan=1 colspan=1>External Controls</td><td rowspan=1 colspan=1>External positive controlincluded in illumigenePertussis External ControlKit</td><td rowspan=1 colspan=1>External controls notprovided</td></tr><tr><td rowspan=1 colspan=1>Test Format</td><td rowspan=1 colspan=1>DNA Amplification; Loop-Mediated IsothermalAmplification (LAMP)</td><td rowspan=1 colspan=1>Nested Multiplex PCRAmplification followed bymelt curve analysis</td></tr><tr><td rowspan=1 colspan=1>Assay Target</td><td rowspan=1 colspan=1>Bordetella pertussis IS481Insertional Element</td><td rowspan=1 colspan=1>Bordetella pertussis DNA,promoter region of toxingene</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>illumipro-10</td><td rowspan=1 colspan=1>FilmArray Instrument</td></tr><tr><td rowspan=1 colspan=1>Sample Processing</td><td rowspan=1 colspan=1>Manual extraction</td><td rowspan=1 colspan=1>Automated extraction</td></tr><tr><td rowspan=1 colspan=1>Reading Method</td><td rowspan=1 colspan=1>Visible Light Transmission</td><td rowspan=1 colspan=1>Fluorescence Emissions</td></tr></table>

# K. Standard/Guidance Document Referenced:

· EP12-A2: User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute

EP15-A2: User Verification of Performance for Precision and Trueness, Clinical and Laboratory Standards Institute

EP7-A2: Interference Testing in Clinical Chemistry, Clinical and Laboratory Standards Institute

MM06-A2: Quantitative Molecular Methods for Infectious Diseases, Clinical and Laboratory Standards Institute

# L. Test Principle:

The illumigene Mycoplasma assay is based on loop mediated isothermal amplification technology (LAMP). Loop mediated amplification of DNA is accomplished by the use of specially designed primers that provide specific and continuous isothermal amplification. Magnesium pyrophosphate is produced as a by-product of LAMP amplification. The magnesium pyrophosphate forms a white precipitate in the reaction solution, giving the reaction solution a turbid appearance. Change in sample absorbance created by precipitation of magnesium pyrophosphate indicates the presence of target DNA and is considered a positive reaction. The absence of target DNA results in no detectable change in sample absorbance and is considered a negative reaction.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

A Reproducibility study performed at three independent laboratories included 10- member test panels consisting of moderately positive $\scriptstyle ( \ n = 3 )$ , low positive $( \mathrm { n } { = } 3 )$ , high negative $( \mathrm { n } { = } 3 )$ and negative $\scriptstyle ( \mathrm { n = 1 }$ ) samples. Samples were prepared by inoculating quantitated suspensions of $B$ . pertussis strain BA-589 into a negative matrix of flocked nylon NP swabs containing pooled negative nasal wash specimen. Prior to sample preparation, the nasal wash matrix was determined to be negative for $B$ . pertussis both by culture and by the illumigene Pertussis Assay. Final concentrations of $B$ . pertussis for each panel member are shown in the following table.

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Final Concentration(CFU/mL)</td><td rowspan=1 colspan=1>Final Concentration(CFU/Test)</td><td rowspan=1 colspan=1>ExpectedResult</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>1.31 x 104</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>4.89 x 103</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>0.004</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td></tr></table>

Samples were randomly sorted within each panel to mask sample identities. Panels were tested two times per day by two operators at each testing site for five testing days. Three lots of illumigene Pertussis kits and six illumipro ‐ $\mathbf { \nabla } \cdot \mathbf { \nabla } l 0$ instruments were used in the study. Qualitative and quantitative reproducibility study results are

shown in the following table:

<table><tr><td rowspan=2 colspan=1>SampleType</td><td rowspan=2 colspan=2>PercentAgreement</td><td rowspan=2 colspan=2>PercentAgreement</td><td rowspan=2 colspan=2>PercentAgreement</td><td rowspan=1 colspan=5>Total</td></tr><tr><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=1>AverageSf:Si</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>57.51</td><td rowspan=1 colspan=1>4.14</td><td rowspan=1 colspan=1>7.20%</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>86/90</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>58.16</td><td rowspan=1 colspan=1>10.02</td><td rowspan=1 colspan=1>17.23%</td></tr><tr><td rowspan=1 colspan=1>HighNegativ ve</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>76.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>78/90</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>96.19</td><td rowspan=1 colspan=1>11.47</td><td rowspan=1 colspan=1>11.92%</td></tr><tr><td rowspan=1 colspan=1>Negative2</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>100.29</td><td rowspan=1 colspan=1>4.75</td><td rowspan=1 colspan=1>4.73%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>56.83</td><td rowspan=1 colspan=1>4.83</td><td rowspan=1 colspan=1>8.50%</td></tr><tr><td rowspan=1 colspan=1>NegativeL Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.70</td><td rowspan=1 colspan=1>3.04</td><td rowspan=1 colspan=1>3.01%</td></tr></table>

1 $\mathrm { S f . S i }$ information is not available to the end‐ user. The illumipro ‐10 reports and stores final results for the end‐user as POSITIVE, NEGATIVE, or INVALID. 2 Numbers represent samples yielding negative results.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

Internal Control:

Each patient specimen is added to an illumigene Assay Control tube containing a trisbuffered solution with internal control sequence (formalin-treated $E$ . coli harboring a plasmid with a specific sequence of Staphylococcus aureus DNA). The heat-treated specimen (with internal control) is added to both the test and control wells of the illumigene test device. The internal control sequence is separately amplified in the control well. Stringent cut-off criteria are applied to the internal control chamber reaction in order to assure amplification is not inhibited, reagents are performing as intended, and sample processing was performed correctly.

External positive $B$ . pertussis control material is sold separately in the Pertussis External Control kit. The control includes tris-buffered solution containing noninfectious plasmid DNA with $B _ { \cdot }$ . pertussis target DNA inserts. The test procedure involves pipetting $5 0 ~ \mu \mu \mu$ of positive control material into a Control/Negative Control Reagent tube, vortexing, heat treating for 10 minutes, followed by addition of the heat-treated control sample to the illumigene test device and testing in the same manner as a patient specimen. The illumigene Assay Control/Negative Control Reagent (without patient specimen added) is used as an external negative control. This reagent is heat treated and added to an illumigene test device in the same manner as a patient specimen. External controls are recommended to aid in detection of reagent deterioration, adverse environmental or test conditions, or variance in operator performance that may lead to test errors. External control testing is recommended with new reagent lots and shipments of illumigene Pertussis kits and should be performed thereafter in accordance with appropriate federal, state, and local guidelines. The illumigene Pertussis DNA Amplification Assay should not be used for testing patient specimens testing if external controls do not produce the expected results. External positive and negative controls were tested a minimum of once per day during the clinical and analytical studies performed for the evaluation of the illumigene Pertussis DNA Amplification Assay.

# Specimen Stability:

A sample hold time study was performed to support claimed specimen transport and storage conditions. The study included testing of unpreserved NP swabs (without transport media), swabs stored in Liquid Amies transport media (without Charcoal) and swabs stored in Liquid Stuart transport media. Samples were prepared using flocked nylon NP swabs and nasal wash matrix determined to be negative for $B$ . pertussis. Positive swabs were spiked with $B$ . pertussis, strain BAA-589 to a final concentration near the assay LoD $( 3 , 2 6 5 \mathrm { C F U / m L }$ of nasal wash or 1.48 CFU/test). Testing included three positive and three negative swab samples tested in triplicate for each storage condition and transport media type (including unpreserved swabs, no transport media).

Samples were stored at $3 2 ^ { \circ } \mathrm { C }$ (worst case room temperature) and at $2 { - } 8 ^ { \circ } \mathrm { C }$ prior to testing with the illumigene assay. Study results yielded the expected results for all positive and negative sample replicates stored in the following conditions stated in the package insert:

NP swab samples stored unpreserved, with Liquid Amies (without charcoal), or with Liquid Stuart’s media may be held at room temperature $( 2 1 { - } 3 0 ^ { \circ } \mathrm { C } )$ for up to 5 days prior to testing with the illumigene Pertussis DNA   
Amplification Assay.   
NP samples stored unpreserved or with both transport medias may be stored refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ for up to 7 days prior to t e s t i n g .

# Eluted Sample Stability:

A study was performed to evaluate acceptable storage conditions for NP swabs eluted into each of the following elution fluids:

· Tris-EDTA (illumigene Pertussis Sample Buffer)   
· Molecular grade water   
· $0 . 8 5 \%$ saline   
· PBS

Samples evaluated in each elution fluid included polyester swabs (aluminum shaft) stored in transport media (both Liquid Amies without charcoal and Liquid Stuart) and flocked nylon swabs stored unpreserved. Low positive samples were prepared by spiking each swab/transport media combination with a negative nasal wash matrix containing Bordetella pertussis strain BAA-589 at approximately $2 \mathbf { x }$ LoD (3265 $\mathrm { C F U / m L }$ or 1.48 CFU/test). Negative samples were prepared for each swab/transport media combination with matrix only. The baseline sample type was flocked nylon swabs stored unpreserved and eluted in Tris-EDTA.

Diluted sample hold time was evaluated by storing eluted samples at refrigeration temperature $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ and at the upper limit (worst case) of room temperature $( 3 3 . 5 ^ { \circ } \mathrm { C } )$ . Testing of positive and negative samples was performed in triplicate for all storage conditions and for each swab type/transport media/elution fluid combination.

All positive replicates for each potential combination of swab/transport media/elution fluid produced the expected positive results by the illumigene Pertussis Assay. For negative samples, only one replicate of one swab type (polyester swab in Liquid Stuarts, eluted in molecular grade water) yielded a false positive result when stored at $3 2 \mathrm { { } ^ { \circ } C }$ for 25 hours. Repeat testing of 20 additional negative replicates of the same swab type yielded the correct negative results for the same storage conditions.

In summary, the study demonstrated that polyester and flocked swabs in either Amies or Liquid Stuart’s transport media, and eluted in Tris-EDTA, $8 5 \%$ saline, PBS, or molecular grade water are stable for the claimed storage conditions of $2 1 { - } 3 0 ^ { \circ } \mathrm { C }$ for 48 hours or $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 7 days.

# d. Detection limit:

The limit of detection (LoD) of the illumigene Pertussis Assay was determined in an analytical study using $B$ . pertussis strain BAA-589. Samples were prepared using serially-diluted and quantitated organism suspensions added to a negative nasal wash matrix (flocked nylon swabs inoculated with nasal wash determined to be negative for B. pertussis). Organism concentrations were confirmed by colony count.

The preliminary LoD was established for each of three illumigene Pertussis kit lots by testing 20 individually prepared replicates at several serial dilutions. Testing was

performed until a dilution for which 19 of 20 replicates produced positive results was identified. The final limit of detection was confirmed by testing an additional 60 replicates for each kit lot with testing performed by two operators on six illumipro ‐10 instruments. The LoD was determined for each lot as the lowest concentration of $B$ . pertussis for which $59 5 \%$ of replicates were positive.

Confirmed Analytical Limit of Detection by Lot   

<table><tr><td rowspan=2 colspan=1>KitLot</td><td rowspan=2 colspan=1>StrainDescription</td><td rowspan=2 colspan=1>CFU/mL</td><td rowspan=2 colspan=1>CFU/Test</td><td rowspan=1 colspan=2>Replicate Sumary</td><td rowspan=2 colspan=1>PercentPositive(%)</td></tr><tr><td rowspan=1 colspan=1>npos</td><td rowspan=1 colspan=1>n</td></tr><tr><td rowspan=1 colspan=1>280750C001</td><td rowspan=1 colspan=1>ATCCBAA-589</td><td rowspan=1 colspan=1>3265</td><td rowspan=1 colspan=1>1.48</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>280750C002</td><td rowspan=1 colspan=1>ATCCBAA-589</td><td rowspan=1 colspan=1>1633</td><td rowspan=1 colspan=1>0.742</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>98.3%</td></tr><tr><td rowspan=1 colspan=1>280750D004</td><td rowspan=1 colspan=1>ATCCBAA-589</td><td rowspan=1 colspan=1>1633</td><td rowspan=1 colspan=1>0.742</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>98.3%</td></tr></table>

npos = number of positive replicates, $n =$ total of number of replicates

# e. Analytical Reactivity:

An analytical inclusivity study was performed using well-characterized strains of $B$ . pertussis. Samples were prepared at concentrations near the previously determined LoD. Stocks of each organism were diluted in saline and dilutions were spiked into negative matrix (flocked nylon NP swabs with negative nasal wash) to obtain final organism concentrations of 3,265 CFU/mL or 1.48 CFU/test. Three replicates were tested for each strain.

The following eight $B$ . pertussis strains gave positive results for all replicates tested:

<table><tr><td rowspan=1 colspan=1>B. pertussis ATCC12743</td><td rowspan=1 colspan=1>B. pertussis ATCC53894</td></tr><tr><td rowspan=1 colspan=1>B. pertussisATCC8478</td><td rowspan=1 colspan=1>B. pertussisATCC10380</td></tr><tr><td rowspan=1 colspan=1>B. pertussisATCC8467</td><td rowspan=1 colspan=1>B. pertussisATCC12742</td></tr><tr><td rowspan=1 colspan=1>B. pertussisATCC9797</td><td rowspan=1 colspan=1>B. pertussis A639</td></tr></table>

The following two $B .$ . pertussis strains initially gave positive results for two of three replicates tested at organism concentrations of 3,265 $\mathrm { C F U / m L }$ . Additional testing at a higher concentration of $3 { , } 5 0 0 \mathrm { C F U / m L }$ yielded $100 \%$ detection of sample replicates.

<table><tr><td></td><td>B. per</td></tr></table>

In order to demonstrate the analytical specificity of the illumigene Pertussis DNA Amplification Assay, testing was performed with samples containing high concentrations of non-target organisms that may be found in NP specimens as well as human genomic DNA. Samples were prepared by inoculating flocked nylon NP swabs with negative nasal wash matrix and each potential cross-reacting organism at the following concentrations: $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and fungi and $1 \mathrm { ~ \overline { { x } } ~ } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses and $2 0 0 \mathrm { n g / \mu L }$ for human genomic DNA. Samples containing the following organisms as well as Negative Sample Matrix and Dilution Controls were tested in triplicate.

Acinetobacter baumannii, Acinetobacter calcoaceticus, Acinetobacter lwoffii, Actinomyces odontolyticus, Arcanobacterium haemolyticum, Bacillus subtilis, Bacteroides fragilis, Bordetella avium, Bordetella hinzii, Bordetella parapertussis, Bordetella petrii, Bordetella trematum, Burkholderia cepacia, Candida albicans, Candida glabrata, Chlamydia pneumoniae, Chlamydia trachomatis, Citrobacter freundii, Clostridium difficile, Corynebacterium diphtheriae, Corynebacterium pseudodiphtheriticum, Enterobacter cloacae, Enterococcus faecalis, Escherichia coli, Escherichia coli (ESBL), Fusobacterium nucleatum, Haemophilus influenzae, Haemophilus parainfluenzae, Klebsiella oxytoca, Klebsiella pneumoniae (KPC), Lactobacillus acidophilus, Lactobacillus plantarum, Legionella jordanis, Legionella longbeachae, Legionella micdadei, Legionella pneumophila, Listeria monocytogenes, Moraxella catarrhalis, Mycobacterium tuberculosis, Mycoplasma genitalium, Mycoplasma hominis, Mycoplasma pneumoniae, Neisseria cinerea, Neisseria elongata, Neisseria gonorrhoeae, Neisseria meningitidis, Nocardia asteroides, Peptostreptococcus anaerobius, Proteus mirabilis, Proteus vulgaris, Pseudomonas aeruginosa, Pseudomonas fluorescens, Serratia liquefaciens, Staphylococcus aureus, Staphylococcus epidermidis, Stenotrophomonas maltophilia, Streptococcus anginosus (Group F), Streptococcus bovis (Group D), Streptococcus canis (Group G), Streptococcus dysgalactiae ssp. dysgalactiae, Streptococcus dysgalactiae ssp. equisimilis Streptococcus intermedius, Streptococcus mitis, Streptococcus mutans, Streptococcus pneumoniae, Streptococcus pyogenes, Streptococcus salivarius, Streptococcus suis, Ureaplasma urealyticum, Adenovirus, Coronavirus, Coxsackievirus, Cytomegalovirus, Epstein Barr Virus, Herpes Simplex Virus 1, Herpes Simplex Virus 2, Human Metapneumovirus, Influenza A, Influenza B, Measles virus, Mumps virus, Parainfluenza virus 1, Parainfluenza virus 2, Parainfluenza virus 3, Respiratory syncytial virus A, Respiratory syncytial virus B, Rhinovirus, and human genomic DNA.

Study results yielded expected negative results for all replicates of each potentially cross-reacting organism and human genomic DNA with the following exceptions:

Strains of Bordetella holmesii and Bordetella bronchiseptica gave positive results in all replicates tested with the illumigene Pertussis DNA Amplification Assay. Samples containing these organisms should be positive because they are known to carry the IS481 gene sequence targeted by the illumigene Pertussis assay. This information is presented in the package insert in the Intended Use Statement, the limitations section, and the test Interpretation section.

Unexpected results were initially observed for $B$ . hinzii and H. parainfluenzae. False positive results were obtained for one of three replicates containing Bordetella hinzii and for three of three replicates containing Haemophilus parainfluenzae. For each organism, additional testing yielded the expected negative results for 20 of 20 replicates, demonstrating that $B _ { \cdot }$ . hinzii and $H .$ . parainfluenzae are not considered cross-reactive with the illumigene Pertussis DNA Amplification Assay.

Invalid results were obtained for two of three replicates for samples containing Mycoplasma genitalium. Additional testing yielded the expected negative result for 10/10 tests. Mycoplasma genitalium is not thought to interfere with detection of $B$ . pertussis.

In conclusion, all organisms evaluated in the cross-reactivity study are not considered to cross-react with the illumigene Pertussis DNA Amplification Assay except for $B$ . holmesii and B. bronchiseptica.

# g. Microbial Interference:

The same organisms evaluated in the cross-reactivity study listed in $^ { \circ } 1 f ^ { \prime }$ above were evaluated for potential interference with the detection of $B$ . pertussis. For this study, samples containing $B$ . pertussis ATCC strain BAA-589 with concentrations near the assay LoD were mixed with each potentially interfering organism at high concentrations $\mathrm { ( > 1 x ~ 1 0 ^ { 6 } ~ C F U / m L }$ for bacteria and $> 1 \mathrm { \Delta x \ 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$ for viruses). Samples as well as Positive Sample Matrix and Dilution Controls were tested in triplicate.

Expected positive results for $B$ . pertussis were obtained for all organisms evaluated with the exception of the following initially invalid results.

· Three of three samples containing $H .$ . parainfluenzae mixed with low-positive $B$ . pertussis gave invalid results. Repeat testing of 10 additional replicates yielded the expected positive results for $B$ . pertussis. As the initial results were not confirmed, $H .$ parainfluenzae does not appear to interfere with detection of $B$ . pertussis.

In summary, none of the organisms evaluated at high concentrations appear to interfere with detection of the $B$ . pertussis target in low-positive $B$ . pertussis samples.

h. Interfering Substances

An analytical study was performed to assess the potential inhibitory effects of biological and chemical materials that may be present in NP specimens on the performance of the illumigene Pertussis DNA Amplification Assay.

Negative samples were prepared using inoculated flocked nylon NP swabs with negative nasal wash matrix. Positive samples were prepared using the same swab type and natural matrix with the addition of Bordetella pertussis strain BAA-589 at a concentration near the assay LoD. Both negative and positive samples were stored unpreserved prior to elution in sample buffer. Each potentially interfering substance was added to eluted samples at final concentrations of $0 . 1 \mathrm { m g / m L }$ , $1 \%$ v/v, $1 \% \mathrm { w / v }$ , or greater. All samples were tested in triplicate with additional testing of any substance producing unexpected results.

The following biological substances did not interfere with the illumigene Pertussis test results: Mucin $( 1 \% \ \mathrm { w / v ) }$ , Human DNA $( 2 0 0 ~ \mathrm { n g / \mu L } )$ , and Whole blood $( 1 \% \mathrm { w / v } )$

The following chemical substances, at the saturated solvent/diluents concentrations indicated were evaluated and gave the expected results for both negative and positive samples: Acetaminophen $\mathrm { { 1 0 \ m g / m L } ) }$ , Afrin $^ \mathrm { \textregistered }$ Decongestant Nasal Spray [Oxymetazoline hydrochloride $( 0 . 0 0 0 5 \% \ \mathrm { ~ w / v ) ] }$ , Albuterol Sulfate [Salbutamol sulfate $\left( 1 \% \quad \mathrm { w / v } \right) ]$ , Coricidin $^ \mathrm { \textregistered }$ HBP Cold/Flu tablets [Acetominophen (3.26 $\mathrm { m g / m L ) }$ , Chlorpheniramine maleate $( 0 . 0 2 \mathrm { m g / m L } ) _ { . } ^ { - }$ ], Diphenhydramine HCl (0.25 $\mathrm { m g / m L ) }$ , Erythromycin $( 2 \% \ \mathrm { w / v ) }$ , Mupirocin $( 2 \% \ \mathrm { w / v ) }$ , Petroleum Jelly [White Petrolatum $\left( 1 \% \ \mathrm { w / v } \right) ]$ , Robitussin $^ \mathrm { \textregistered }$ Cough+Chest Congestion DM Cough Syrup [Dextromethorphan HBr $( 0 . 1 ~ \mathrm { \ m g / m L } )$ , Guaifenesin $\left( 1 . 0 \ : \ : \mathrm { \ m g / m L } \right) ]$ , Suphedrine PE [Phenylephrine HCl $( 0 . 3 \mathrm { m g / m L } ) ]$ , Saline Nasal Spray [Sodium chloride $( 0 . 0 0 6 5 \% \ \mathrm { w / v } ) ]$ , Smokeless Tobacco (snuff) $( 1 \% \mathrm { w } / \mathrm { v } )$ , Tobramycin $( 0 . 6 ~ \mathrm { m g / m L } )$ , Vicks $^ \mathrm { \textregistered }$ VaporRub $^ \mathrm { \textregistered }$ [Camphor $( 0 . 4 8 \% \mathrm { w / v } )$ , Eucalyptus oil $( 0 . 1 2 \% \mathrm { w / v }$ ), Menthol $( 0 . 2 6 \% \mathrm { w } / \mathrm { v } ) ]$ .

Ibuprofen $\mathrm { { 1 0 m g / m L } ) }$ produced invalid results for $3 / 3$ replicates during initial testing of positive $B _ { \cdot }$ . pertussis specimens; however additional testing gave the correct positive result for $1 0 / 1 0$ replicates. As the initial results were not confirmed, Ibuprofen ( $1 0 \mathrm { m g / m L } )$ ) is not considered to interfere with the illumigene Pertussis Assay.

Aspirin at concentrations greater than $5 \mathrm { m g / m L }$ was found to interfere with the illumigene Pertussis Assay and this information is included in the product package insert.

# i. Fresh/Frozen Studies:

The clinical study included testing of both prospectively collected and tested fresh specimens as well as archived frozen swab specimens that were eluted into either Tris-EDTA, water, or saline prior to freezing. A fresh versus frozen study was performed to demonstrate equivalence of eluted frozen specimens and freshly tested swab specimens for testing with the illumigene Pertussis DNA Amplification Assay.

Samples were prepared with aluminum shafted polyester-tipped NP swabs which are the indicated NP swabs with the lowest absorption for the illumigene Pertussis Assay. A panel of contrived positive samples $\mathrm { ( n } { = } 3 5 \mathrm { ) }$ ) and negative samples $\mathrm { ( n } { = } 3 5 \mathrm { ) }$ was prepared consisting of NP swabs inoculated with 35 unique negative nasal wash specimens (negative matrix). Positive samples were spiked with $B$ . pertussis strain BAA-589 at low positive concentrations (1.5x LoD). The positive and negative NP swabs were eluted by vortexing in illumigene Pertussis Sample Buffer Reagent (Tris EDTA). All samples were tested at $\mathrm { T } _ { 0 }$ prior to storing at both ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ and $- 8 0 ^ { \circ } \mathrm { C }$ . Samples were frozen for a minimum of four hours prior to thawing. The panel of positive and negative samples was tested after each of three freeze/thaw cycles for both freezer temperatures. One positive sample gave a false negative result after the first freeze/thaw cycle at -20C storage; however this result was not repeated for an additional 20 test replicates of the same eluted specimen or subsequent freeze/thaw cycles for the same sample after storage at either ${ } _ { - 2 0 } \mathrm { { \circ C } }$ or $8 0 \mathrm { { } ^ { \circ } C }$ storage conditions. In summary, study results demonstrated that frozen eluted swab samples give equivalent results to freshly tested NP swabs and are acceptable for inclusion in the clinical studies and determination of the illumigene Pertussis DNA Amplification Assay performance.

# j. Swab Type and Transport Media Equivalence Study:

Flocked nylon, Polyester, and Rayon swab types were evaluated in an analytical study. Each swab type was further evaluated either unpreserved, or in Liquid Amies medium (without charcoal) or Liquid Stuart non-nutritive transport media. Negative samples were prepared by inoculating sterile NP swabs with screened negative nasal wash. Positive samples were prepared by inoculating sterile NP swabs with negative nasal wash spiked with diluted Bordetella pertussis (strain ATCC BAA‐589) to concentrations near the limit of detection for the illumigene Pertussis kit. Positive and negative samples for all swab types and transport media combinations were tested in triplicate.

The study yielded the expected results for all swab types and non-nutritive transport media (and unpreserved swabs) combinations. Results confirmed equivalence of the three swab types (Flocked Nylon, Polyester and Rayon) with the three transport conditions (unpreserved, Liquid Amies (without Charcoal), and Liquid Stuart).

# k. Assay cut-off:

The illumigene Pertussis assay is manufactured with fixed cut‐off values. The product is designed around a pre‐selected cut‐off value and amplification reagent concentrations are optimized to ensure appropriate reactions are obtained. Development optimization includes evaluation of characterized positive and negative clinical specimens.

Cut‐off values are applied in the following manner:

The illumipro ‐10 calculates the ratio of the Run End (Signal final or $\mathrm { S _ { f } }$ ) reads with the Run Start (Signal Initial or Si) reads and compares the ratio to an established cut‐off value. The illumipro ‐ $\mathbf { \nabla } \cdot \mathbf { \nabla } l 0$ performs this ratio calculation to both the TEST chamber and the CONTROL chamber. Numerical results are not available to the user.

Fixed cut‐off values for the TEST chamber are used to report sample results. TEST chamber Sf:Si ratios less than $82 \%$ are reported as ‘POSITIVE’; TEST chamber $\mathrm { S _ { f } . S _ { i } }$ ratios greater than or equal to $82 \%$ are reported as ‘NEGATIVE’.

Fixed cut‐off values for the CONTROL chamber are used to determine validity of specimen test results. CONTROL chamber $\mathrm { S _ { f } { : } S _ { i } }$ ratios less than $90 \%$ are considered valid and allow for reporting of TEST chamber results (POSITIVE, NEGATIVE). Specimens for which CONTROL chamber $\mathrm { S _ { f } { : } S _ { i } }$ ratios are greater than or equal to $90 \%$ are reported as invalid and test chamber results are not reported. More stringent cut‐off criteria are applied to the Control chamber reaction to ensure amplification is not inhibited, reagents are performing as intended and sample processing was performed appropriately.

Assay cutoffs for the illumigene Pertussis Assay were initially evaluated using contrived specimens. Samples were prepared by spiking PBS buffer with dilutions $( 2 0 \mathrm { C F U / m L }$ or $4 0 \mathrm { C F U / m L }$ ) of Bordetella pertussis, ATCC strain 8467. Twenty replicates at each concentration were tested. Results of the contrived sample study confirmed that the previously established fixed cut‐off values are appropriate $82 \%$ for the test well and $90 \%$ for the control well).

Additional confirmation of the assay cutoffs was performed using known positive and negative clinical specimens from two clinical testing sites. Samples were characterized as positive or negative by the standard of care $B$ . pertussis testing available at each clinical site. Finally, results from the pivotal clinical study were used to validate the fixed assay cutoffs. In summary, illumigene test results for clinical specimens demonstrated a clear separation for test well results between positive and negative specimens, thus validating that the fixed illumigene Pertussis cutoff is appropriate.

# l. CarryOver Study:

A study was conducted to evaluate the risk of carry-over contamination with the illumigene Pertussis DNA Amplification Assay test devices and the illumipro-10 instrument. Samples tested in this study consisted of alternating high positive $B$ . pertussis samples and negative samples both prepared using flocked nylon NP swabs inoculated with negative nasal wash matrix confirmed to be negative for $B$ . pertussis. Positive samples were inoculated with $B$ . pertussis strain BAA-589. Negative samples contained negative matrix only. Alternating positive and negative samples were processed in the same heat block followed by tested on the illumipro-10 incubator/reader. A total of six runs were performed using three heat blocks and three illumipro-10

instruments. A minimum of 10 positive and 10 negative alternating samples were run on each illumipro- $I 0$ instrument.

Results from the study showed no evidence of carryover or cross-contamination with the illumigene Pertussis DNA Amplification Assay on the illumipro-10 system.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison:

# Matrix Equivalency Study:

An analytical study was performed to demonstrate equivalent assay performance for flocked nylon swabs inoculated with a negative nasal wash matrix as compared to flocked nylon NP swab specimen matrix. Study samples were prepared using 40 paired matrix specimens collected from the same individual (nasal washes and NP swabs). All matrices were confirmed negative for $B$ . pertussis by both culture and the illumigene Pertussis assay. For each matrix, testing included 20 positive and 20 negative samples. Positive samples were spiked with quantified $B$ . pertussis cultures (ATCC BAA-589) into eluted sample preparations at concentrations near the assay LoD. Negative samples contained matrix only. Study results demonstrated $100 \%$ correct positive and negative results for samples prepared with both NP swab and nasal wash matrices. The study results support the use of a nasal wash matrix as a substitute for NP swab specimens for preparation of analytical study samples.

# Sample Buffer (Elution buffer) Equivalence Study:

In the clinical study, three different sample preparation solutions were used for elution of NP swabs; Tris-EDTA, Saline $( 0 . 8 5 \% )$ or Molecular Grade Water. An analytical study was performed to demonstrate equivalent assay performance for different elution solutions. Samples were prepared using NP Flocked Nylon swabs with positive samples containing organism concentrations near the assay LoD and negative samples containing matrix only. Testing was performed using unpreserved swabs, swabs preserved in Liquid Amies (without charcoal) and swabs preserved in Liquid Stuart transport media. Samples were eluted in $0 . 5 ~ \mathrm { m L }$ of Tris-EDTA, Phosphate-Buffered Saline, Saline $( 0 . 8 5 \% )$ or Molecular Grade Water and tested with the illumigene Pertussis DNA Amplification assay. Positive and negative samples for each elution solution/transport media combination were evaluated in triplicate. The study yielded $100 \%$ correct positive and negative results for all replicates tested with each elution fluid evaluated, thus confirming equivalence of the four sample elution materials (Tris-EDTA, Phosphate Buffered Saline, Saline $( 0 . 8 5 \% )$ and Molecular Grade Water).

3. Clinical studies:

The clinical study for the illumigene Pertussis DNA Amplification Assay run on the illumipro‐10 was conducted from December 2012 to July 2013. A total of 729 qualified nasopharyngeal (NP) swab specimens collected from patients suspected of having Bordetella pertussis infection were evaluated. All specimens included in the study were originally submitted to a testing laboratory by an ordering physician specifically for $B .$ . pertussis testing, were presumed from symptomatic patients, and were leftover and de‐identified prior to inclusion in the study. No restrictions were placed on age, gender, or antibiotic status. Four independent clinical test sites located in the Midwestern, Northern, Southern, and Eastern regions of the United States participated in the device evaluation.

A total of 851 patient specimens were originally included in the clinical study of which 122 specimens were excluded from the final performance calculations due to improper specimen storage or incomplete comparator assay testing. After exclusions, the dataset includes 729 eligible specimens including 508 $( 6 9 . 7 \% )$ prospectively tested fresh specimens, 184 $( 2 5 . 2 \% )$ retrospective specimens (all comers), and 37 $( 5 . 1 \% )$ retrospective selected specimens.

The study population included 413 $( 5 6 . 7 \% )$ female patients and 308 $( 4 2 . 2 \% )$ male patients. Gender was unknown for eight $( 1 . 1 \% )$ of the study participants. Patient age information was known for 723 $( 9 9 . 2 \% )$ of patient specimens and ranged from 1 month to 88 years. Thirty-eight $( 5 . 2 \% )$ patients were less than 1 year of age; 13 $( 1 . 8 \% )$ were between 1 and 2 years old; 296 $( 4 0 . 6 \% )$ were between 2 and up to 12 years, 157 $( 2 1 . 5 \% )$ were between 12 and up to 21 years, 190 $( 2 6 . 0 \% )$ were above 21 but below 65 years, and the remaining 29 $( 4 . 0 \% )$ patients were above 65 years of age.

NP swabs were collected as unpreserved dry swabs or in a non-nutritive transport medium (i.e., Liquid Stuart or Liquid Amies without charcoal). Swabs were eluted into $0 . 5 \mathrm { m L }$ of Tris-EDTA elution buffer, saline, or molecular grade water prior to testing with the illumigene assay. Retrospective specimens were eluted and stored at $\mathrm { \leq - 2 0 ^ { \circ } C }$ prior to testing. After testing with the illumigene Pertussis DNA Amplification Assay, an aliquot of each eluted specimen was sent to an internal reference laboratory at Meridian BioScience for comparator PCR testing.

Performance of the illumigene Pertussis DNA Amplification Assay was evaluated by comparison to a composite of two validated real-time PCR assays followed by confirmation of positive PCR amplification product with bi-directional sequencing. Both comparator PCR assays targeted unique sequences within the IS481 insertional element and neither of the comparator assay target sequences overlapped with the illumigene Pertussis target sequence.

The comparator PCR assays included internal controls for each specimen tested as well as external controls for each run. Validation of the comparator PCR assays included LoD and Inclusivity studies. LoD studies demonstrated that the each PCR assay had a similar analytical sensitivity to that of the illumigene assay and Inclusivity studies demonstrated that the comparator assays detected ten well-characterized strains of $B$ . pertussis.

Comparator PCR testing was performed at Meridian Bioscience, Inc. and bi-directional sequencing was performed by a third party reference laboratory for specimens positive by one or both comparator PCR assays. Only samples with bi-directional sequencing results meeting pre-defined quality scores (PHRED Scores of 20 and E-values of $< 1 0 ^ { - 3 0 }$ ) were considered true positives. Specimens were characterized as positive for $B$ . pertussis if one or both comparator PCR assays was positive (Ct values ${ < } 4 0$ ) and confirmed by bidirectional sequencing. Specimens were characterized as negative if both comparator PCR assays were negative (Ct values $\geq 4 0$ ).

Invalid illumigene results were initially obtained for 13 specimens $( 1 . 8 \% )$ and after repeat testing, two specimens remained invalid and were not included in assay performance calculations. Two prospective and two retrospective specimens were also excluded from the analysis due to indeterminate results for one or both comparator PCR assays.

Performance for the illumigene Pertussis assay as compared to the composite reference testing is shown in the following table for prospective, retrospective (all comers) and retrospective (selected) specimens.

Assay Performance for the illumigene Pertussis DNA Amplification Assay   

<table><tr><td rowspan=2 colspan=1>SpecimenDescription</td><td rowspan=1 colspan=3>Positive Specimens</td><td rowspan=1 colspan=3>Negative Specimens</td><td rowspan=2 colspan=1>InvalidResultsb</td></tr><tr><td rowspan=1 colspan=1>illumigeneVS.Comparator</td><td rowspan=1 colspan=1>PPAa</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>illumigeneVS.Comparator</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=8>Composite Method Comparator, All Comers</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>39/45</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=1>73.8 -93.7%</td><td rowspan=1 colspan=1>447/459</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>95.5-98.5%</td><td rowspan=1 colspan=1>2 (13)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0 -100.0%</td><td rowspan=1 colspan=1>176/178</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>96.0 99.7%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total:</td><td rowspan=1 colspan=1>43/49</td><td rowspan=1 colspan=1>87.8%</td><td rowspan=1 colspan=1>75.8 -94.3%</td><td rowspan=1 colspan=1>623/637</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>96.3 98.7%</td><td rowspan=1 colspan=1>2 (13)</td></tr><tr><td rowspan=1 colspan=8>Composite Method Comparator, Selected Specimens</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>19/21</td><td rowspan=1 colspan=1>90.5%</td><td rowspan=1 colspan=1>71.1-97.3%</td><td rowspan=1 colspan=1>14/16</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>64.0 -96.5%</td><td rowspan=1 colspan=1>0</td></tr></table>

a Eight specimens produced false-negative illumigene results when compared to the Composite Comparator Method. Six of the eight specimens produced detectable levels of DNA between 35 and 40 comparator assay amplification cycles and were confirmed positive by bi-directional sequencing. Three of these six specimens gave positive results in only one of the two comparator PCR/Sequencing assays. $^ { \mathrm { ~ b ~ } } 1 1 / 1 3$ initial invalid specimens produced valid results upon repeat testing.

False-negative illumigene results were individually evaluated at the conclusion of clinical testing. The Cycle threshold (Ct) values produced during comparator assay testing were above 35 for one or both comparator PCR/Bi-directional sequencing assays for six of the eight specimens evaluated. High Ct values in PCR assays may indicate that low levels of

DNA are present. False-negative illumigene results and corresponding Composite Comparator data are shown in the following table:

False-Negative illumigene Specimens, Comparator Assay Results   

<table><tr><td rowspan=2 colspan=1>SpecimenDesignation</td><td rowspan=2 colspan=1>SpecimenStatus</td><td rowspan=1 colspan=2>PCR1</td><td rowspan=1 colspan=2>PCR2</td></tr><tr><td rowspan=1 colspan=1>Ct Value</td><td rowspan=1 colspan=1>Bi-DirectionalSequencingResult</td><td rowspan=1 colspan=1>Ct Value</td><td rowspan=1 colspan=1>Bi-DirectionalSequencingResult</td></tr><tr><td rowspan=1 colspan=1>1-19</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>34.90</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>33.85</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>1-29</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>37.69</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>1-259</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>35.09</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>1-269</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>35.72</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>1-275</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>39.13</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>3-33</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>39.28</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>35.80</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>4-710</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>36.87</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>36.06</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>4-712</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>37.44</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>38.41</td><td rowspan=1 colspan=1>+</td></tr></table>

Performance data by clinical site is shown in the following table:

illumigene Pertussis Assay Performance by Clinical Test Site, Composite Reference Method (CRM)   

<table><tr><td rowspan=2 colspan=1>SpecimenDescription</td><td rowspan=1 colspan=3>Positive Specimens</td><td rowspan=1 colspan=3>Negative Specimens</td><td rowspan=2 colspan=1>InvalidResults</td></tr><tr><td rowspan=1 colspan=1>illumigeneVS.Comparator</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>illumigeneVS.Comparator</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Composite Method Comparator, All Comers</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>35/40</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>73.9 -94.5%</td><td rowspan=1 colspan=1>440/450</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>96.0 -98.8%</td><td rowspan=1 colspan=1>2 (13)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0 -100.0%</td><td rowspan=1 colspan=1>67/69</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>90.0 -99.2%</td><td rowspan=1 colspan=1>0 (2)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0 -79.3%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>64.6 100.0%</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>51.0 -100.0%</td><td rowspan=1 colspan=1>109/111</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>93.7 99.5%</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=8>Composite Method Comparator, Selected Specimens</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>79.6 -100.0%</td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>40.9 -92.9%</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>30.0 -90.3%</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>67.6 -100.0%</td><td rowspan=1 colspan=1>0 (0)</td></tr></table>

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

Overall incidence of $B$ . pertussis as detected by the illumigene Pertussis Assay in prospectively and retrospectively collected, non-selected specimens (all comers) during the period of this study was $8 . 2 \%$ (57/692).

# N. Instrument Name:

illumipro ‐ ${ \mathbf { } } l { \mathbf { } } 0 ^ { { \mathrm { T M } } }$ Automated Isothermal Amplification and Detection System

# O. System Descriptions:

1. Modes of Operation:

The illumipro-10 instrument heats each illumigene Pertussis Test Device containing prepared samples and Control Reagent, facilitating amplification of target DNA. When $B$ . pertussis is present in a nasopharyngeal specimen, a portion of the IS481 Insertional Element is amplified and magnesium pyrophosphate is generated forming a precipitate in the reaction mixture. The illumipro-10 instrument detects the change in light transmission through the reaction mixture created by the precipitating magnesium pyrophosphate. Specimen results are reported as Positive or Negative based on the detected change in light transmission.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

Specimens are identified by position. Default Sample Identification is based on Block and Well position (e.g., Block A, Well 1). The user may input Sample Identification information using the keypad or the optional external keyboard.

4. Specimen Sampling and Handling:

Specimens are processed manually prior to inoculation of test cartridges which are then placed into the illumipro- $\mathbf { \nabla } _ { J 0 }$ instrument for automated amplification and detection.

5. Calibration: Calibration of the illumipro-10 is not required.

6. Quality Control: See description above for internal and external controls for the illumigene Pertussis DNA Amplification Assay.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.