# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number: K133936   
B. Purpose for Submission: To obtain substantial equivalence for artus® C. difficile QS-RGQ MD   
C. Measurand: C. difficile Toxin A gene (tcdA) and Toxin B gene (tcdB and tcdBv)   
D. Type of Test: Real-time PCR with DNA amplification   
E. Applicant: Qiagen GmbH   
F. Proprietary and Established Names: artus® C. difficile QS-RGQ MDx kit

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3130

2. Classification: Class II

3. Product codes: OZN – C. difficile toxin gene amplification assay OOI – Real time nucleic acid amplification system

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The artus C. difficile QS-RGQ MDx Kit is an in vitro polymerase chain reaction (PCR) assay for use on the QIAsymphony RGQ MDx system for the qualitative detection of toxigenic Clostridium difficile toxin A and toxin B genes in human liquid or soft stool specimens from patients suspected of having Clostridium difficile associated disease. The test is intended to be used directly on patient samples.

The artus C. difficile QS-RGQ MDx Kit is intended to be used to aid in diagnosis of Clostridium difficile infection.

# 2. Indication(s) for use:

The artus C. difficile QS-RGQ MDx Kit is an in vitro polymerase chain reaction (PCR) assay for use on the QIAsymphony RGQ MDx system for the qualitative detection of toxigenic Clostridium difficile toxin A and toxin B genes in human liquid or soft stool specimens from patients suspected of having Clostridium difficile associated disease. The test is intended to be used directly on patient samples.

The artus C. difficile QS-RGQ MDx Kit is intended to be used to aid in diagnosis of Clostridium difficile infection.

3. Special conditions for use statement: For Prescription Use   
4. Special instrument requirements: QIA symphony RGQ MDx System

# I. Device Description:

The contents of the artus C. difficile QS-RGQ MDx Kit are sufficient for 72 tests in one to three batches of 24 reactions on the QIAsymphony RGQ MDx system. Kit components are dispensed in $2 \mathrm { m L }$ tubes with color-coded caps. Each component is provided in 3 separate aliquots sufficient to perform up to 24 reactions each, and the components are provided in a single kit carton. The list of materials included in the kit is provided in Table 1.

Table 1: Materials provided with the artus C. difficile QS-RGQ MDx Kit   

<table><tr><td rowspan=1 colspan=1>Color of tube</td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Number of tubesprovided in kit</td><td rowspan=1 colspan=1>Volume pertube</td></tr><tr><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>C. difficile Master A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>330 μL</td></tr><tr><td rowspan=1 colspan=1>Violet</td><td rowspan=1 colspan=1>C. difficile Master B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>600 μL</td></tr><tr><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>C. difficile Internal Control</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>540 μL</td></tr><tr><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>C. difficile Positive Control</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>330 μL</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>C. difficile Negative Control</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>330 μL</td></tr></table>

C. difficile Master A: A mixture containing thermostable DNA polymerase, buffer, dNTPs, and proprietary PCR enhancers and stabilizers.

C. difficile Master B: A buffered solution containing C. difficile and IC-specific primers and probes. It contains primer sets for the amplification of conserved regions of C. difficile tcdA , tcdB, and tcdBv genes. In addition, the mix contains primers for the amplification of a conserved region of the Geobacillus stearothermophilus (GeoB) genome contained in an intact, chemically inactivated organism (NATtrol). The mix also contains specific fluorophore labeled probes to allow for the detection and differentiation of the amplification products.

C. difficile Internal Control: Consists of intact inactivated Geobacillus stearothermophilus in a soy peptone buffer.

C. difficile Positive Control: Consists of plasmids that encode the tcdA and tcdB genes in a buffered solution.   
C. difficile Negative Control: Consists of Bacteroides Thetaiotaomicron DNA in $1 0 \mathrm { m M }$ Tris / 1 mM EDTA, $\mathsf { p H 8 . 0 }$ buffer.

# J. Substantial Equivalence Information:

1. Predicate device name: Quidel Molecular Direct C. difficile assay

2. Predicate 510(k) number: K123998

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Intended for the qualitativedetection of toxigenic C. difficiletoxin A and toxin B genes in stoolspecimens from patients suspectedof having CDAD</td><td rowspan=1 colspan=1>Similar</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Real -time PCR; DNAamplification</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Procedure</td><td rowspan=1 colspan=1>Assay uses the QIAsymphonySP/AS for automated samplepreparation and assay setup</td><td rowspan=1 colspan=1>Assay uses proprietarysample preparation bufferand manual assay setup</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>Assay uses the Rotor-Gene QMDx</td><td rowspan=1 colspan=1>Assay can be performedusing either the:Life TechnologiesQuantStudio DxApplied Biosystems7500 fast DxCepheid Smart Cycler II</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>All controls required to run theassay (Internal Control, PositiveControl, Negative Control) areincluded in the kit.</td><td rowspan=1 colspan=1>Assay includes a processcontrol and recommends theuse of external controls</td></tr></table>

# K. Standard/Guidance Document Referenced:

Draft Guidance for Industry and Food and Drug Administration Staff –“ Establishing the Performance Characteristics of In Vitro diagnostic Devices for the Detection of Clostridium difficile”, issued November 29, 2010.

# L. Test Principle:

The artus C. difficile QS-RGQ MDx Kit assay uses PCR to generate an amplified product from the tcdA and tcdB/tcdBv genes of toxigenic C. difficile DNA in clinical specimens. Samples are extracted and prepared using the QIAsymphony SP instrument with the QIAsymphony DSP Virus/Pathogen Mini Kit, followed by assay setup on the QIAsymphony AS. Amplification and detection are carried out using the artus C. difficile QS-RGQ MDx Kit with the Rotor-Gene Q MDx (RGQ MDx) and Rotor-Gene AssayManager software. The presence of a toxigenic C. difficile target sequence is indicated by the fluorescent signal generated through the use of fluorescently labeled oligonucleotide probes. The probes do not generate a signal unless they are specifically bound to the amplified product. The amplification cycle at which fluorescent signal is detected by the RGQ MDx is inversely proportional to the toxigenic C. difficile DNA target concentration present in the original

specimen. A bacterial species unrelated to toxigenic C. difficile is introduced into each specimen during sample preparation to serve as an internal control. The internal control bacteria are lysed simultaneously with toxigenic C. difficile in the specimen, and amplified in the same reaction as the C. difficile targets using PCR, and serve to demonstrate that the entire assay process has proceeded correctly for each specimen.

# M. Performance Characteristics:

1. Analytical performance:

# a. Precision/Reproducibility:

The precision of the artus C. difficile QS-RGQ Kit was assessed using a 7-member panel consisting of $2 C .$ . difficile strains: NAP-1/BI/027 strain, toxinotype III $\mathbf { A } { + } \mathbf { B } { + }$ (ATCC BAA1870) and 1470 strain, toxinotype VIII $\mathrm { \mathbf { A } } \mathrm { - } \mathbf { B } +$ (ATCC 43598). Panel members were initially diluted in TE buffer then tested in Buffer ATL containing negative stool matrix with a single strain present (NAP-1/BI/027 or 1470) at 3 concentrations; positive (approximately 2–3x LoD), low positive (1x LoD), and high negative $_ { < 1 \mathrm { x } }$ LoD). A seventh panel member (negative) was prepared using TE buffer only and also tested in Buffer ATL containing negative stool matrix. The data obtained were used to determine the mean $\mathrm { C _ { T } }$ , standard deviation (SD) and the coefficient of variation $( \% C V )$ for each target and the internal control.

For the within laboratory repeatability study, the seven-member panel was tested in replicates of three, once a day for a total of twelve days (for a total of 262 data points for the 12 runs). The testing was conducted by two operators using one instrument (QS/AS RGQ MDx) and one reagent kit lot (Table 2).

Table 2: Within Laboratory Repeatability Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Internal Control</td><td rowspan=1 colspan=3>tcdA</td><td rowspan=1 colspan=3>tcdB</td></tr><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>NAP1 Positive</td><td rowspan=1 colspan=1>29.78</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.64%</td><td rowspan=1 colspan=1>30.84</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.52%</td><td rowspan=1 colspan=1>33.58</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.91%</td></tr><tr><td rowspan=1 colspan=1>NAP1 LowPositive</td><td rowspan=1 colspan=1>30.07</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.82%</td><td rowspan=1 colspan=1>32.19</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>3.08%</td><td rowspan=1 colspan=1>34.92</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>1.94%</td></tr><tr><td rowspan=1 colspan=1>NAP1 HighNegative</td><td rowspan=1 colspan=1>29.99</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.41%</td><td rowspan=1 colspan=1>34.21</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.66%</td><td rowspan=1 colspan=1>36.18</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.64%</td></tr><tr><td rowspan=1 colspan=1>1470 Positive</td><td rowspan=1 colspan=1>29.85</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.73%</td><td rowspan=1 colspan=1>30.95</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.78%</td><td rowspan=1 colspan=1>33.72</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.33%</td></tr><tr><td rowspan=1 colspan=1>1470 LowPositive</td><td rowspan=1 colspan=1>29.89</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.39%</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.03%</td><td rowspan=1 colspan=1>35.13</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.80%</td></tr><tr><td rowspan=1 colspan=1>1470 HighNegative</td><td rowspan=1 colspan=1>29.92</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.76%</td><td rowspan=1 colspan=1>34.14</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.92%</td><td rowspan=1 colspan=1>36.39</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.07%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30.00</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.46%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

To measure site-to-site reproducibility, the 7-member panel was run by 2 users at each of 3 sites (IMDx and 2 external sites). Each of the 2 users performed 5 runs on alternating testing days. Panel members were tested in replicates of 3 that were randomized and blinded to the user. A single QIAsymphony RGQ MDx system and one lot of the artus $C .$ . difficile QS-RGQ MDx Kit were used at each site to conduct the study (Table 3). The reproducibility study results are acceptable.

Table 3: Site-to-Site Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=12 colspan=1>NAP-1/BI/027</td><td rowspan=4 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.61</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>31.85</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>34.32</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.02</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.58</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>32.16</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>34.43</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.05</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.55</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>31.97</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.65</td><td rowspan=1 colspan=1>34.87</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.25</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.58</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.49</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>2.19</td></tr><tr><td rowspan=4 colspan=1>LowPositive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.63</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>33.38</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.14</td><td rowspan=1 colspan=1>35.58</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.55</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.73</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>33.27</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.45</td><td rowspan=1 colspan=1>35.01</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.52</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.86</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33.07</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>35.45</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>1.96</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.74</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>33.24</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>35.36</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.76</td></tr><tr><td rowspan=4 colspan=1>HighNegative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.71</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>34.21</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>35.88</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.59</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>34.21</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>35.92</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.64</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>34.17</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.68</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.65</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>34.19</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.32</td><td rowspan=1 colspan=1>35.91</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.43</td></tr><tr><td rowspan=12 colspan=1>1470</td><td rowspan=4 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.67</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>32.22</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.67</td><td rowspan=1 colspan=1>34.74</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.76</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.69</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.32</td><td rowspan=1 colspan=1>31.74</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>2.99</td><td rowspan=1 colspan=1>34.66</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.14</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.86</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>31.81</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.45</td><td rowspan=1 colspan=1>34.95</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.01</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.74</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>31.92</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>2.77</td><td rowspan=1 colspan=1>34.78</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>1.99</td></tr><tr><td rowspan=4 colspan=1>LowPositive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.74</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>33.17</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>2.83</td><td rowspan=1 colspan=1>35.61</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.38</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.51</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>33.06</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>35.62</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>33.41</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>3.22</td><td rowspan=1 colspan=1>35.77</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.29</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.65</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>35.65</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.48</td></tr><tr><td rowspan=4 colspan=1>HighNegative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.76</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>34.64</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>36.41</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.55</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>34.15</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>35.82</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.44</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>34.63</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.58</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.49</td><td rowspan=1 colspan=1>34.49</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>36.12</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.16</td></tr><tr><td rowspan=4 colspan=2>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30.68</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30.66</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30.67</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>33.58</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>overall</td><td rowspan=1 colspan=1>30.67</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>33.58</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

b. Linearity/assay reportable range: N/A   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

For controls: See # I - Device Description above

Traceability for the QIAsymphony Instrument, Firmware, and Software is maintained between requirements, specifications, testing, and risk mitigations. Product requirements that are defined at the highest system level (QS-RGQ) are translated into product requirements for the SP and AS subsystem, and translated further into specifications for hardware, firmware, software, and disposables.

# d. Detection limit:

The limit of detection (LOD) was assessed for the artus C. difficile QS-RGQ MDx Kit using 3 toxigenic Clostridium difficile strains: NAP-1/BI/027 strain, toxinotype III $\mathbf { A } { + } \mathbf { B } { + }$ ( $\mathbf { A T C C } \textsuperscript { \textregistered }$ BAA-1870); 1470 strain, toxinotype VIII $\mathrm { \mathbf { A } } \mathrm { - } \mathrm { \mathbf { B } } +$ (ATCC 43598); and VPI 10463 strain, toxinotype 0 $\mathrm { \bf A } { + } \mathrm { \bf B } { + }$ (ATCC 43255). The LOD is defined as the toxigenic C. difficile bacterial titer $\mathrm { ( C F U / m L ) }$ ) detected with a probability of $9 5 \%$ or greater and was determined by probit analysis. The results, representative of the analytical sensitivity of the artus C. difficile QS-RGQ MDx Kit, are summarized in Table 4.

Table 4: Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD (95%CI)</td></tr><tr><td rowspan=1 colspan=1>C. difficile NAP1ATCCBAA-1870, strain:4118</td><td rowspan=1 colspan=1>7.9 CFU/mL (6.1- 15.0)</td></tr><tr><td rowspan=1 colspan=1>C.difficile 1470ATCC43598,, strain:1470</td><td rowspan=1 colspan=1>11.2 CFU/mL (8.7- 16.8)</td></tr><tr><td rowspan=1 colspan=1>C.difficile 10463ATCC 43255, strain:10463</td><td rowspan=1 colspan=1>2.8 CFU/mL (2.1- 4.2)</td></tr></table>

# e. Analytical specificity:

# Cross-Reactivity and Microbial Interference

A panel of microorganisms that may be present in patient specimens was tested to determine whether these microorganisms interfered with the detection of tcdA or tcdB targets or were cross-reactive with the artus $C .$ . difficile QS-RGQ MDx Kit. Organisms were tested at a target concentration of approximately $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/ml for bacteria and fungi or ${ \geq } 1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ units $/ \mathrm { m l }$ for viruses separately in the presence of $2 { - } 3 \mathbf { x }$ LOD of each of three C. difficile strains: NAP-1/BI/027 strain, 1470 strain, and VPI 10463 strain. None of the potential interfering organisms cross-reacted or interfered with the detection of any of the $3 ~ C .$ . difficile strains by the artus C. difficile QS-RGQ MDx Kit (Table 5). Cross- reactivity for Clostridium botulinum was analyzed in silico and predicted no cross reactivity or microbial interference for the artus C. difficile QS-RGQ MDx Kit.

Table 5: Organisms Tested in Cross Reactivity and Microbial Interference   

<table><tr><td colspan="1" rowspan="1">Organism Tested</td><td colspan="1" rowspan="1">Source ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC 7966</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 13472</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ZMC 0601533</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43479</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 33559</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp. jejuni</td><td colspan="1" rowspan="1">ATCC 33292</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">ATCC 638</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">ATCC 19398</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">ATCC 9650</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">ATCC 19402</td></tr><tr><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">ATCC 49531</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ATCC 13124</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">ATCC 35704</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">ATCC 12464</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">ATCC 9714</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43593</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43601</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">ATCC 15579</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanB)</td><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">ATCC 700927</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori DNA</td><td colspan="1" rowspan="1">ATCC 43504D-5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 33496</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ZMC 0801534</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Salmonella choleraesuis (Typhimurium)</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ATCC 11835</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC43300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 27541</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">ZMC 0810110 CF</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">ZMC 0810041 CF</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">ZMC 0810086 CF</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">ZMC 0810047 CF</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">ZMC 0810023 CF</td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus</td><td colspan="1" rowspan="1">ZMC 0810075 CF</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ZMC 0810003 CF</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">Promega G3041</td></tr></table>

# Analytical Reactivity

The analytical reactivity of the artus C. difficile QS-RGQ MDx Kit was assessed to determine whether the kit could detect a broad range of toxigenic C. difficile strains representing temporal and geographical diversity. A total of 27 strains and characterized clinical isolates were diluted in TE buffer to $2 { - } 3 \mathbf { x }$ LOD of the reference strain and tested with the artus C. difficile QS-RGQ MDx Kit. C. difficile target was detected in all strains tested (Table 6).

Table 6: Strains Tested in Analytical Reactivity   

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Toxinotype</td><td colspan="1" rowspan="1">Origin</td></tr><tr><td colspan="1" rowspan="1">ATCC 17857</td><td colspan="1" rowspan="1">870</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC17858</td><td colspan="1" rowspan="1">1253</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC43594</td><td colspan="1" rowspan="1">W1194</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43596</td><td colspan="1" rowspan="1">545</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43599</td><td colspan="1" rowspan="1">2022</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 43600</td><td colspan="1" rowspan="1">2149</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces; Belgium</td></tr><tr><td colspan="1" rowspan="1">ATCC 51695</td><td colspan="1" rowspan="1">BDMS18AN</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Becton DickinsonMicrobiology Systems,Johns Hopkins Univ.Hosp. Lab</td></tr><tr><td colspan="1" rowspan="1">ATCC 700792</td><td colspan="1" rowspan="1">14797-2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human feces;Michigan, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC 9689</td><td colspan="1" rowspan="1">90556-M6S</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1382</td><td colspan="1" rowspan="1">630</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Switzerland</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1805</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">III</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1871</td><td colspan="1" rowspan="1">4111</td><td colspan="1" rowspan="1">O</td><td colspan="1" rowspan="1">Human; New Jersey,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1872</td><td colspan="1" rowspan="1">4206</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Human; Maine, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1873</td><td colspan="1" rowspan="1">5283</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Human; New York,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1874</td><td colspan="1" rowspan="1">4205</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Human; Oregon, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1875</td><td colspan="1" rowspan="1">5325</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">Human; Georgia, USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2155</td><td colspan="1" rowspan="1">LBM0801058</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human; New Mexico,USA</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2156</td><td colspan="1" rowspan="1">LBM0801040</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Human; CambridgeUK</td></tr><tr><td colspan="1" rowspan="1">CCUG 20309</td><td colspan="1" rowspan="1">8864</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Birmingham, UK</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate278</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate464</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate4092</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate5572</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate3430</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IX</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate1753</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate5090</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XXI</td><td colspan="1" rowspan="1">Illinois, USA</td></tr><tr><td colspan="1" rowspan="1">Illinois VAHospital isolate3130</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XXII</td><td colspan="1" rowspan="1">Illinois, USA</td></tr></table>

# Interfering Substances

A panel of 23 substances that may be present in patient specimens was tested to determine whether these substances interfered with the performance of the artus C. difficile QS-RGQ MDx Kit. Three toxigenic $C .$ . difficile strains: NAP-1/BI/027 strain, 1470 strain, and VPI 10463 strain, were diluted to approximately $2 { - } 3 \mathbf { x }$ LOD and spiked with each potentially inhibitory substance. None of the substances showed an inhibitory effect on the detection of C. difficile by the artus C. difficile QS-RGQ MDx Kit (Table 7).

Table 7: Potentially Interfering Substances Tested   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Potential Interferent</td><td colspan="1" rowspan="1">ConcentrationTested*</td></tr><tr><td colspan="1" rowspan="1">Anti-fungal</td><td colspan="1" rowspan="1">Miconazolenitrate cream</td><td colspan="1" rowspan="1">Miconazole nitrate</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="3">Cream/Suppositories</td><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">Hydrocortizone</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Zinc oxide</td><td colspan="1" rowspan="1">Zinc oxide</td><td colspan="1" rowspan="1">40% w/v</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">Petroleum jelly</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Anti-hemorrhoidcreams</td><td colspan="1" rowspan="1">Hemorrhoid gel</td><td colspan="1" rowspan="1">Phenylephrinehydrochloride</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Condoms</td><td colspan="1" rowspan="1">Condoms</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">7%</td></tr><tr><td colspan="1" rowspan="1">MoistTowelettes</td><td colspan="1" rowspan="1">Moist Towelettes</td><td colspan="1" rowspan="1">BenzalkoniumChloride</td><td colspan="1" rowspan="1">0.12% w/v</td></tr><tr><td colspan="1" rowspan="4">Antacids</td><td colspan="1" rowspan="1">Gaviscon</td><td colspan="1" rowspan="1">Aluminum hydroxide,Magnesium carbonate</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tums</td><td colspan="1" rowspan="1">Ca carbonate</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tagamet</td><td colspan="1" rowspan="1">Cimetidine</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Prilosec (delayedrelease)</td><td colspan="1" rowspan="1">Omeprazolemagnesium</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Enemas</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="2">Anti-DiarrhealMedication</td><td colspan="1" rowspan="1">Imodium</td><td colspan="1" rowspan="1">Loperamide HCl</td><td colspan="1" rowspan="1">0.00667 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Pepto Bismol</td><td colspan="1" rowspan="1">Bismuth Subsalicylate</td><td colspan="1" rowspan="1">0.87 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Laxative</td><td colspan="1" rowspan="1">ExLax</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="2">Antibiotics</td><td colspan="1" rowspan="1">Vancomycin HCl</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">12.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-inflammatory</td><td colspan="1" rowspan="1">Naproxen Sodium(Aleve)</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">14 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">Glucose, hormones,enzymes, iron, etc.,</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="3">FecalComponents</td><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">4 mg/mL</td></tr><tr><td colspan="1" rowspan="1">MRI ContrastAgents</td><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">5 mg/mL</td></tr></table>

\*Represents physiologically relevant concentrations of substances

# f. Assay cut-off:

In order to determine the CT cutoff values for tcdA and tcdB targets in $C .$ . difficile positive samples, the following studies were performed: (1) Serial dilutions of contrived C. difficile specimens and- (2) Testing of in-house $C .$ . difficile positive and negative clinical specimens.

Based on the studies conducted to establish the cutoff values for the artus C. difficile QS-RGQ MDx Kit, the following parameters were established (Table 8).

Table 8: CT threshold and cutoff parameters   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>CT cutoff</td></tr><tr><td rowspan=1 colspan=1>tcdA (Cycling Orange)</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>&lt; 38.3</td></tr><tr><td rowspan=1 colspan=1>tcdB (Cycling Green)</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>&lt;36.5</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>&lt;35.9</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison: N/A

# a. Clinical Sensitivity:

The performance of the artus C. difficile QS-RGQ MDx Kit was evaluated at 3 external testing sites using patient samples from patients suspected of C. difficile infection. Patient samples were collected from 5 geographically diverse locations within the United States. The artus C. difficile QS-RGQ MDx Kit was compared to direct and/or enriched culture. For direct culture, the stool specimen was swabbed to a CCFA plate and streaked for isolation. For enriched culture, the same swab was used to inoculate enrichment broth (CCMB-TAL). At 36-48 hours, the enrichment broth was subcultured to a CCFA plate which was streaked for isolation. Suspicious colonies were identified and the cytotoxin assay was performed from a cell free supernatant. The tables below present the data from these studies.

# Assay vs. Enriched Culture Comparison

A total of 741 specimens were tested by both the artus C. difficile QS-RGQ MDx Kit and enriched toxigenic culture. The overall assessment of sensitivity and specificity versus enriched toxigenic culture is shown below (Table 9).

Table 9: Clinical Performance of artus C. difficile QS-RGQ MDx Kit vs. Enriched Toxigenic Culture   

<table><tr><td rowspan=1 colspan=5>Combined Sites - Combined Ages</td></tr><tr><td rowspan=5 colspan=1>artus C. difficileQS-RGQ MDxKit</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Enriched Toxigenic Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>17*</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>13**</td><td rowspan=1 colspan=1>597</td><td rowspan=1 colspan=1>610</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>741</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>95% CI (%)Sensitivity  90%                 83-94Specificity  97%                 96-98PPV          87%                 80-92NPV          98%                 96-99Prevalence  17%                 15-20</td></tr></table>

$^ { * } 1 7$ discordant specimens (artus C. difficile QS-RGQ MDx Positive, Enriched Toxigenic Culture Negative) reported were analyzed by alternative PCR followed by bi-directional sequencing and the result was that 12 out of 17 were positive for toxigenic C. difficile, agreeing with the artus C. difficile QS-RGQ MDx result.

$^ { * * } \ 1 2$ discordant specimens (artus C. difficile QS-RGQ MDx Negative, Enriched Toxigenic Culture Positive) reported were analyzed by alternative PCR followed by bi-directional sequencing and the result was that 10 out of 12 were negative for toxigenic C. difficile, agreeing with the artus C. difficile QS-RGQ MDx result. The remaining 1 discordant specimen was unavailable for testing.

Assay vs. Direct Toxigenic Culture Comparison

A direct toxigenic culture result was available for 699 specimens. The overall assessment of sensitivity and specificity versus direct toxigenic culture is shown below (Table 10).

Table 10: Clinical Performance of artus C. difficile QS-RGQ MDx Kit vs. Direct Toxigenic Culture   

<table><tr><td rowspan=1 colspan=5>Combined Sites - Combined Ages</td></tr><tr><td rowspan=5 colspan=1>artus C. difficileQS-RGQ MDxKit</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Direct Toxigenic Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>21*</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1**</td><td rowspan=1 colspan=1>593</td><td rowspan=1 colspan=1>594</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>699</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>95% CI (%)Sensitivity   99%                94-100Specificity    97%                95-98PPV            89%                71-87NPV            100%               99-100Prevalence    12%                10-15</td></tr></table>

$^ { * } 1 9$ discordant specimens (artus C. difficile QS-RGQ MDx Positive, Direct Toxigenic Culture Negative) reported were analyzed by alternative PCR followed by bi-directional sequencing and the result was that 14 out of 19 were positive for toxigenic C. difficile, agreeing with the artus C. difficile QS-RGQ MDx result. The remaining 2 specimens were unavailable for testing.

$^ { * * }$ The 1 discordant specimen (artus C. difficile QS-RGQ MDx Negative, Direct Toxigenic Culture Positive) was unavailable for testing.

b. Clinical specificity: See 3 a. above for specificity results.   
c. Other clinical supportive data (when a. and b. are not applicable): N/A

4. Clinical cut-off: N/A

5. Expected values/Reference range: The prevalence for C. difficile observed during a multi-center clinical trial was estimated as $1 8 . 2 \%$ (135/741) when tested using the artus C. difficile QS-RGQ MDx Kit. Of the patient population included in the study, the majority of patients were senior adults $( \ge 6 0$ years) and the prevalence of $C .$ . difficile in this age group was found to be $1 9 \%$ (75/395). The next largest age group was adults (age ${ > } 2 1$ to $\leq 5 9$ years) and the prevalence was found to be $1 6 \%$ (52/324). The remaining patient population included one infant ( $\cdot < 2$ years), 6 children ${ \ge } 2$ to $^ { < 1 2 }$ years), 9 adolescents ${ \ge } 1 2$ to ${ < } 1 8$ years), and 6 transitional adolescents $( \ge 1 8$ to ${ \leq } 2 1$ years).

# N. Instrument Name:

QIAsymphony RGQ MDx system

# O. System Descriptions:

1. Modes of Operation:

The QIAsymphony RGQ MDx system consists of 3 subsystems: the QIAsymphony SP (QSSP) for sample preparation, the QIAsymphony AS (QSAS) for assay set-up and the Rotor-Gene Q MDx (RGQ MDx) for target amplification and detection.

2. Software:

The QIAsymphony SP and QIAsymphony AS are used in combination with the QIAsymphony software (version 4.0) and the RGQ MDx is used in combination with the Rotor-Gene AssayManager analysis software (version 1.0). As part of the QIAsymphony RGQ MDx system, cycler control, data analysis and reporting with the Rotor-Gene MDx instrument are done by the Rotor-Gene AssayManager 1.0 software using plug-ins, which contain specific analysis algorithms and configurations.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification:

For specimen identification, switch on the QIAsymphony SP/AS, and wait until the “Sample Preparation” screen appears and the initialization procedure has finished. Then log in to the instrument and load the sample drawer with the positive and negative controls. Next load the “Sample” drawer with the samples and using the “Integrated run” setup on the QIAsymphony touchscreen, enter the required information for each batch of samples to be processed. Then define the assay to run, define the QIAsymphony AS batch, load the “Sample” drawer with the internal control mixture, define the internal control positions and start the run.

4. Specimen Sampling and Handling:

The QIAsymphony SP instrument is designed to perform automated purification of nucleic acids in combination with QIAsymphony DSP Kits. An assay is developed for use with a specific QIAsymphony DSP Kit based on the application and sample type. The QIAsymphony AS instrument is designed to perform automated assay setup and is physically connected to the QIAsymphony SP. The eluate transfer between QIAsymphony SP and QIAsymphony AS is done automatically. After the assay setup is complete, the user needs to remove the assay rack from the QIAsymphony AS. The Rotor-Gene strip tubes must be sealed with lids, transferred to the Rotor-Gene Q MDx and secured with a locking ring. The Rotor-Gene Q MDx instrument is a real-time nucleic acid amplification and detection system.

5. Calibration:

N/A. The QS-RGQ MDx system does not require calibration.

6. Quality Control:

A bacterial species unrelated to toxigenic C. difficile is introduced into each specimen during sample preparation to serve as an internal control. The internal control bacteria are lysed simultaneously with toxigenic C. difficile in the specimen, and amplified in the same reaction as the C. difficile targets using PCR, and serve to demonstrate that the entire assay process has proceeded correctly for each specimen. Kit also contains a positive and negative control for use in the assay run.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.