# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K140111

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for the BD MAX Enteric Bacterial Panel on the BD MAX™ System.

C. Measurand: Target DNA sequences for:

Salmonella spp. - (SpaO gene target)   
• Campylobacter spp.- (C. jejuni, C. coli) - (Campylobacter specific tuf gene target)   
• Shigella spp./Enteroinvasive Escherichia coli (EIEC) - (ipaH gene target) Shiga Toxin 1/2 (stx1/stx2 gene targets)

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for the amplification and detection of DNA from Salmonella spp., Shigella spp., and Campylobacter spp., as well as the toxin genes stx1/and stx2 found in Shiga-toxin producing Escherichia coli (STEC).

# E. Applicant:

BD Diagnostics (BD GenOhm Sciences Canada, Inc.)

# F. Proprietary and Established Names:

BD MAXTM Enteric Bacterial Panel BD MAXTM System

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 – Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: Class II

3. Product code: PCI, PCH, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use:

The BD MAX™ Enteric Bacterial Panel performed on the BD MAX™ System is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric bacterial pathogens. The BD MAX Enteric Bacterial Panel detects nucleic acids from:

• Salmonella spp.   
• Campylobacter spp. (jejuni and coli)   
• Shigella spp. / Enteroinvasive $E .$ . coli (EIEC) Shiga toxin 1 $( s t x 1 )$ / Shiga toxin 2 $( s t x 2 )$ genes (found in Shiga toxin-producing E. coli [STEC]) as well as Shigella dysenteriae, which can possess a Shiga toxin gene $( s t x )$ that is identical to the stx1 gene of STEC.

Testing is performed on unpreserved soft to diarrheal stool specimens or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of SpaO, a Campylobacter specific tuf gene sequence, ipaH and $s t x 1 / s t x 2$ . The test utilizes fluorogenic sequence-specific hybridization probes for detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Salmonella, Shigella/EIEC, Campylobacter and Shiga toxin-producing $E$ . coli (STEC) infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For Prescription Use Only

4. Special instrument requirements:

The assay is run on the BD MAXTM System.

# I. Device Description:

The BD MAX System and the BD MAX Enteric Bacterial Panel are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, real-time PCR master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes (SBT). The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using realtime PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX System software automatically interprets test results.

# Brief Explanation of the Procedure

A soft to diarrheal stool is collected and transported to the laboratory. After the stool has been homogenized, a disposable inoculating loop is used to collect a $1 0 ~ \mu \mathrm { L }$ aliquot of the stool material and the contents of the loop are dispensed into a SBT. The SBT is closed with a septum cap and vortexed. A worklist is created and the SBT, the BD MAX Enteric Bacterial Panel Unitized Reagent Strip (URS) and the BD MAX PCR cartridge are loaded onto the BD MAX System. The BD MAX System automates sample preparation including cell lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. Amplified targets are detected with hydrolysis probes labeled with quenched fluorophores. The amplification, detection and interpretation of the signals are done automatically by the BD MAX System.

# Reagents Provided with the BD MAX Enteric Bacterial Panel

BD MAX Enteric Bacterial Panel Master Mix: Oven-dried PCR Master Mix containing TaqMan $\textsuperscript { \textregistered }$ specific molecular probe and primers along with Sample Processing Controlspecific Taqman probe and primers. BD MAX Enteric Bacterial Panel Reagent Strip: Unitized reagent strip containing all liquid reagents and disposable pipette tips necessary for specimen processing and DNA extraction. BD MAX Enteric Bacterial Panel Extraction Tube: Oven-dried DNA magnetic affinity beads, Oven-dried protease reagents. • BD MAX Enteric Bacterial Panel Sample Buffer Tube (with septum caps)

Equipment and Materials Required But Not Provided

• VWR Multi-Tube Vortexer (VWR catalog no. 58816-115)   
• NALGENE $\textsuperscript { \textregistered }$ Cryogenic Vial Holder (VWR, catalog no. 66008-783) Disposable gloves, powderless Sterile scissors (optional)   
• Sterile Gauze

• Stopwatch or timer BD MAX PCR Cartridges (BD Diagnostic Systems catalogue no. 437519)

# J. Substantial Equivalence Information:

1. Predicate device name(s): ProGastro SSCS Assay, Gen-Probe Prodesse, Inc.

2. Predicate 510(k) number(s): K123274

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD MAX Enteric BacterialPanel</td><td rowspan=1 colspan=1>Hologic ProdesseProGastro SSCS(K123274)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Multiplex real-time PCRassay for detection ofnucleic acids from bacterialenteric pathogens and toxingenes</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Organisms/toxin genesdetected</td><td rowspan=1 colspan=1>Salmonella spp., Shigellaspp./EIEC, Campylobacterspp. (jejuni and coli only),and STEC (stx1/stx2)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Format</td><td rowspan=1 colspan=1>Amplification: real-timePCRDetection: fluorogenictarget-specific hybridization</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection Probes</td><td rowspan=1 colspan=1>TaqMan Probe</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Interpretation of TestResults</td><td rowspan=1 colspan=1>Automated (BD MAXSystem diagnostic software)</td><td rowspan=1 colspan=1>Automated (CepheidSmartCycler II)</td></tr><tr><td rowspan=1 colspan=1>Shigella target</td><td rowspan=1 colspan=1>Presence of ipaH genespecific for Shigellaspp./EIEC</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Unpreserved and Cary-Blairpreserved stool</td><td colspan="1" rowspan="1">Stool in Cary-Blairpreserved or Para-Pak C &amp;S transport medium</td></tr><tr><td colspan="1" rowspan="1">Campylobacter target</td><td colspan="1" rowspan="1">tuf gene specific for</td><td colspan="1" rowspan="1">glyA gene specific for</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Camyplobacter jejuni andcadF gene specific for C.coli</td></tr><tr><td colspan="1" rowspan="1">Salmonella target</td><td colspan="1" rowspan="1">SpaO gene specific forSalmonella</td><td colspan="1" rowspan="1">orgC gene specific forSalmonella</td></tr><tr><td colspan="1" rowspan="1">Shiga-toxin target</td><td colspan="1" rowspan="1">stx1/stx2 genes specific toshiga-toxin producingorganisms. Positive reportdoes not distinguishbetween stx1 and stx2</td><td colspan="1" rowspan="1">stx1/stx2 genes specific toshiga-toxin producingorganisms. Positive reportdoes distinguish betweenstx1 and stx2.</td></tr><tr><td colspan="1" rowspan="1">PCR SamplePreparation/Extraction</td><td colspan="1" rowspan="1">BD MAX System</td><td colspan="1" rowspan="1">bioMerieux NucliSENSeasyMAG</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample Processing Control(SPC)</td><td colspan="1" rowspan="1">Internal Control</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

# L. Test Principle:

The BD MAX Enteric Bacterial Panel detects target DNA from unpreserved soft to diarrheal stool specimens or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis.

A stool specimen is collected and transported to the laboratory in a dry, clean container (for unpreserved specimens) or in Cary-Blair transport media. The specimen is vortexed for 15 seconds and then a $1 0 ~ \mu \mathrm { L }$ loop is used to inoculate a SBT. The SBT is closed with a septum cap and vortexed. A worklist is created and the SBT, the BD MAX Enteric Bacterial Panel unitized reagent strip (URS) and the BD MAX PCR Cartridge are loaded onto the BD MAX System.

Following enzymatic cell lysis, the released nucleic acids are captured on magnetic beads. The beads, with the bound nucleic acids, are washed using Wash Buffer and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized using Neutralization Buffer and transferred to a Master Mix to rehydrate PCR reagents. After reconstitution, the BD MAX System dispenses a fixed volume of PCR-ready solution containing extracted nucleic acids into the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed prior to initiating PCR to contain the amplification mixture, thus preventing evaporation and contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes, labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect amplicons for the following enteric bacterial targets:

Campylobacter spp.: The assay detects the Campylobacter-specific tuf gene sequence of Campylobacter jejuni and Campylobacter coli; however the assay does not distinguish which species is present. Species other than $C .$ . jejuni and $C .$ . coli are not detected by the assay. Salmonella spp.: The assay detects the SpaO gene of Salmonella spp.; however the assay does distinguish which Salmonella species is present. All species of Salmonella should be detected by the assay. Shigella spp.: The assay detects the ipaH gene of Shigella spp. or Enteroinvasive E. coli (EIEC). The assay will give a positive result for Shigella spp./EIEC when S. boydii, S. flexneri, S. sonnei, S. dysenteriae or EIEC are present in a specimen; however the assay report does not indicate if the detected organism is Shigella spp. or EIEC. In addition the assay does not distinguish which Shigella species is present. • $S t x 1$ and stx2 genes: The assay detects stx1 and stx2 gene sequences that are present in STEC and S. dysenteriae; however the assay does not distinguish which stx gene is present.

The four described targets as well as the SPC are detected in five different optical channels of the BD MAX System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the optical channels used for the BD MAX Enteric Bacterial Panel is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX System measures these signals at the end of each amplification cycle, and interprets the data to provide a result.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

Site-to-site Reproducibility

Site-to-Site reproducibility was evaluated for the BD MAX Enteric Bacterial Panel in a study that included three external clinical testing sites using a single lot of reagents. Ten panels, each consisting of 12 samples were evaluated over five days with two panels tested per day, each by one of two technologists resulting in a total of 90 data points per panel member (3 replicates/run x 2 runs/day by two different technologists/site x 5 days $_ { \textrm { X 3 } }$ sites). Positive panel members were prepared as organism mixes with cultures of Salmonella typhimurium (ATCC 14028), Shigella sonnei (ATCC 9290), Campylobacter coli (ATCC 43134) and Escherichia coli $( s t x 1 )$ (ATCC 43890). Each positive panel member contained targeted organisms at varying concentrations: moderate positive (MP) ${ \sim } 3 \mathbf { x }$ LoD (Limit of Detection), low positive (LP) at ${ \sim } 1 . 5 \mathbf { x }$ LoD, and high negative (HN) at $\mathrm { C } _ { 2 0 - 8 0 } \mathrm { L o D }$ . True negative (TN) panel members contained sample buffer only.

Composition of the Site to Site Reproducibility Test Panel Mixes   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>stx1</td></tr><tr><td rowspan=1 colspan=1>01R, 03R, 09R</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td></tr><tr><td rowspan=1 colspan=1>02R, 08R, 12R</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>04R, 06R, 10R</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td></tr><tr><td rowspan=1 colspan=1>05R, 07R, 11R</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TN</td></tr></table>

TN: True negative, no target LP: Low positive, ${ \geq } 1$ and ${ \le } 2$ X LoD MP: Moderate positive, ${ \geq } 2$ and ${ \le } 5 \mathrm { X }$ LoD HN: High negative, appropriate dilution of the organism to produce a negative result $2 0 \%$ to $8 0 \%$ of the time TN: Sample Buffer only

Three replicates of each positive mix and true negative sample were included in each run. Panel members were prepared by spiking SBTs with organism mixes. Testing panels were shipped to testing sites and prior to testing, the user expressed a $1 0 \mu 1$ loop of pooled stool matrix (previously determined to be negative for all analytes by the BD MAX Enteric Bacterial Panel) into each sample.

The overall Site-to-Site Reproducibility percent agreement with expected results (i.e., negative for TN and HN, positive for MP and LP) was $100 \%$ for the TN category for all targets, and ranged from $4 1 . 1 \%$ to $7 7 . 8 \%$ , $9 6 . 7 \%$ to $100 \%$ and $9 8 . 9 \%$ to $100 \%$ for the HN, LP and MP categories, respectively. The reproducibility study results met the pre-defined acceptance criteria for LP, MP, HN and TN samples of overall expected results: approximately $9 5 \%$ detection for LP samples, approximately $100 \%$ detection for MP samples, between $20 \%$ detection for HN samples and $100 \%$ negative results for TN samples.

Site-to-Site Overall Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Campylobacter(coli and jejuni)[n], (95% CI)</td><td rowspan=1 colspan=1>Salmonella spp.[n], (95% CI)</td><td rowspan=1 colspan=1>Shigella spp.[n], (95% CI)</td><td rowspan=1 colspan=1>Shiga toxins(stx1 and stx2)[n], (95% CI)</td></tr><tr><td rowspan=1 colspan=1>TN*</td><td rowspan=1 colspan=1>100.0%, [90/90],95.9%, 100.0%)</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>HN*</td><td rowspan=1 colspan=1>77.8%, [70/90],(68.2%, 85.1%)</td><td rowspan=1 colspan=1>44.4%, [40/90],(34.6%, 54.7%)</td><td rowspan=1 colspan=1>41.1%, [37/90],(31.5%, 51.4%)</td><td rowspan=1 colspan=1>50.0%, [45/90],(39.9%, 60.1%)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td><td rowspan=1 colspan=1>96.7%, [87/90],(90.7%, 98.9%)</td><td rowspan=1 colspan=1>97.8%, [88/90],(92.3%, 99.4%)</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td><td rowspan=1 colspan=1>98.9%, [89/90],(94.0%, 99.8%)</td><td rowspan=1 colspan=1>100.0%, [90/90],(95.9%, 100.0%)</td><td rowspan=1 colspan=1>98.9%, [89/90],(94.0%, 99.8%)</td></tr></table>

\* For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results. HNs are dilutions of the LoD designed to produce results that are negative for $2 0 \%$ and $8 0 \%$ of replicates.

The following tables included qualitative and quantitative results from the site-to-site reproducibility study for the Campylobacter spp., Salmonella spp., Shigella spp. and $s t x 1 / s t x 2$ targets, respectively. Qualitative analyses of study results are presented by testing site as percent agreement to the expected result. Quantitative analyses of study results are presented as a numerical analysis of Ct values.

Campylobacter Site-to-Site Qualitative Reproducibility Across Sites   

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=4 colspan=1>Concentration</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td><td rowspan=1 colspan=4>5</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct.</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>5 CFU/mL</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>73.3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>22.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>≥1 and &lt;2x LoD</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>≥2 and ≤5x LoD</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Campylobacter Site-to-Site Reproducibility: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>Within RunWithinn Day</td><td rowspan=1 colspan=2>BetweenRunWithin Day</td><td rowspan=1 colspan=2>BetweenDayWithinSite</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.9%</td></tr></table>

Salmonella Site-to-Site Qualitative Reproducibility Across Sites   

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=4 colspan=1>Concentration</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td><td rowspan=1 colspan=4>5</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct.</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>75 CFU/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>53.3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>46.7</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>46.7</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>53.3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>44.4</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>55.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>≥1 and &lt;2x LoD</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>≥2 and ≤5x LoD</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1</td></tr></table>

Salmonella Site-to-Site Reproducibility: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>Within RunWithin Day</td><td rowspan=1 colspan=2>BetweenRunWithin Day</td><td rowspan=1 colspan=2>BetweenDayWithinSite</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.4%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.6%</td></tr></table>

# Shigella Site-to-Site Qualitative Reproducibility Across Sites

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=4 colspan=1>Concentration</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td><td rowspan=1 colspan=4>5</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct.</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>9 CFU/mL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>60.0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>43.3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>56.7</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>60.0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>58.9</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>≥1 and &lt;2x LoD</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>≥2 and ≤5x LoD</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Shigella Site-to-Site Reproducibility: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>Within RunWithin Day</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between DayWithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>3.2%</td></tr></table>

# Shiga toxin Site-to-Site Qualitative Reproducibility Across Sites

<table><tr><td colspan="1" rowspan="4">Category</td><td colspan="1" rowspan="4">Concentration</td><td colspan="12" rowspan="1">SITE</td><td colspan="4" rowspan="2">Total</td></tr><tr><td colspan="4" rowspan="1">2</td><td colspan="4" rowspan="1">3</td><td colspan="4" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Correct</td><td colspan="2" rowspan="1">Incorrect</td><td colspan="2" rowspan="1">Correct.</td><td colspan="2" rowspan="1">Incorrect</td><td colspan="2" rowspan="1">Correct</td><td colspan="2" rowspan="1">Incorrect</td><td colspan="2" rowspan="1">Correct</td><td colspan="2" rowspan="1">Incorrect</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">Blank</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">100 CFU/mL</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">53.3</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">46.7</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">46.7</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">53.3</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">50.0</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">≥1 and &lt;2x LoD</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">≥2 and ≤5x LoD</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.1</td></tr></table>

Shiga toxin (stx1/stx2) Site-to-Site Reproducibility: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>Within RunWithin Day</td><td rowspan=1 colspan=2>BetweenRunWithin Day</td><td rowspan=1 colspan=2>Between DayWithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.3%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>2.2%</td></tr></table>

# Lot-to-lot Reproducibility:

For evaluation of lot-to lot reproducibility, two users each completed a single run of 12 panel members on a single instrument for each of two lots of reagents over a 5-day period for a total of 10 runs. The same panels used for the site-to-site reproducibility study were tested in this study. The overall Lot-to-Lot reproducibility percent agreement was $100 \%$ for the TN category for all targets, and ranged from $1 3 . 3 3 \%$ to $6 2 . 2 2 \%$ , $9 5 . 5 6 \%$ to $100 \%$ and $9 7 . 7 8 \%$ to $100 \%$ for the HN, LP and MP categories, respectively as shown in the following table.

Lot-to-Lot Reproducibility Study: Qualitative Results   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=2>95% CILowerCI     UpperCI</td></tr><tr><td rowspan=4 colspan=1>stx1/stx2</td><td rowspan=1 colspan=1>TN**</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>HN*</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>30.00%</td><td rowspan=1 colspan=1>21.51%</td><td rowspan=1 colspan=1>40.13%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>98.89%</td><td rowspan=1 colspan=1>93.97%</td><td rowspan=1 colspan=1>99.80%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=4 colspan=1>Campy</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>62.22%</td><td rowspan=1 colspan=1>51.90%</td><td rowspan=1 colspan=1>71.54%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>97.78%</td><td rowspan=1 colspan=1>92.26%</td><td rowspan=1 colspan=1>99.39%</td></tr><tr><td rowspan=4 colspan=1>Shig</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>16.67%</td><td rowspan=1 colspan=1>10.37%</td><td rowspan=1 colspan=1>25.69%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>95.56%</td><td rowspan=1 colspan=1>89.12%</td><td rowspan=1 colspan=1>98.26%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>98.89%</td><td rowspan=1 colspan=1>93.97%</td><td rowspan=1 colspan=1>99.80%</td></tr><tr><td rowspan=4 colspan=1>Sal</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>13.33%</td><td rowspan=1 colspan=1>7.79%</td><td rowspan=1 colspan=1>21.87%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>98.89%</td><td rowspan=1 colspan=1>93.97%</td><td rowspan=1 colspan=1>99.80%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.91%</td><td rowspan=1 colspan=1>100.00%</td></tr></table>

\*HNs are dilutions of the LoD designed to produce results that are negative for $20 \%$ and $80 \%$ of replicates. As such, $\%$ Correct” correlates to the percent of negative results. \*\* TNs contained no organisms. $\%$ Correct” correlates to the percent of negative results.

Ct Values were evaluated as an additional means to assess lot-to-lot reproducibility as shown in the following table.

Lot-Lot Reproducibility: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>LevelN</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Within Runwithin Day</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Mean CtSD</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>stx1/stx2</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>6335.26</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>4.15%</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.05%</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.74%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>4.34%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>8931.79</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.18%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.04%</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.42%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>9030.69</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.38%</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.51%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.65%</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.61%</td></tr><tr><td rowspan=3 colspan=1>CampylobacterLP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>3435.8</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>3.71%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>1.81%</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>1.94%</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>1.634.56%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>9032.44</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.53%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.61%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.36%</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.692.14%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>8931.79</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.02%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.74%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.17%</td></tr><tr><td rowspan=3 colspan=1>Shigella</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>7534.49</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>4.41%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>4.48%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>8831.97</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.33%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.15%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>0.752.33%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>8931.11</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.41%</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.72%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.22%</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.60%</td></tr><tr><td rowspan=3 colspan=1>Salmonella</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>7835.48</td><td rowspan=1 colspan=1>4.17</td><td rowspan=1 colspan=1>11.74%</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>3.12%</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.44%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>4.34</td><td rowspan=1 colspan=1>4.3412.23%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>8933.2</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.08%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.31%</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.10%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>9032.04</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.63%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.62%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.93%</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.98%</td></tr></table>

# Precision

Within-laboratory precision was evaluated for the BD MAX Enteric Bacterial Panel at one internal testing site. Testing was performed over 12 days, with 2 runs per day (one each by 2 technologists), for a total of 24 runs using the same panel tested in the reproducibility studies. The following tables include a qualitative and quantitative analysis of assay precision.

Precision Study Qualitative results   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=5>Percent Agreement by Analyte</td></tr><tr><td rowspan=1 colspan=1>E. coli stx 1</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Expected Values</td></tr><tr><td rowspan=4 colspan=1>TN1HN1LPMP</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>27.78%</td><td rowspan=1 colspan=1>25.00%</td><td rowspan=1 colspan=1>30.56%</td><td rowspan=1 colspan=1>54.17%</td><td rowspan=1 colspan=1>20% to 80%</td></tr><tr><td rowspan=1 colspan=1>98.61%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.61%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>≥ 95.00%</td></tr><tr><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.61%</td><td rowspan=1 colspan=1>98.61%</td><td rowspan=1 colspan=1>100.00%</td></tr></table>

1 For the Negative and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Precision Study Results: Numerical analysis of PCR Ct values   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>BetweenBRun</td><td rowspan=1 colspan=2>Between</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Optic</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Shiga-toxin</td><td rowspan=1 colspan=1>$Cy_\q}$</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>5.41%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.87%</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>5.72%</td></tr><tr><td rowspan=1 colspan=1>$Cy\q}$</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>31.81</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.71%</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.25%</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.21%</td><td rowspan=1 colspan=1>1.32</td><td rowspan=1 colspan=1>4.15%</td></tr><tr><td rowspan=1 colspan=1>Cy_\}$</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>30.69</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.39%</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.40%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>3.39%</td></tr><tr><td rowspan=3 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>35.24</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>2.96%</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>3.46%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.55%</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>32.49</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.30%</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.19%</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.21%</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>3.87%</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>32.52</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>3.35%</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.22%</td><td rowspan=1 colspan=1>1.56</td><td rowspan=1 colspan=1>4.79%</td></tr><tr><td rowspan=3 colspan=1>Shigella</td><td rowspan=1 colspan=1>ROX</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>1.23</td><td rowspan=1 colspan=1>3.54%</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.27%</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.46%</td><td rowspan=1 colspan=1>1.69</td><td rowspan=1 colspan=1>4.87%</td></tr><tr><td rowspan=1 colspan=1>ROX</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>32.69</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>4.07%</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.60%</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.47%</td><td rowspan=1 colspan=1>1.51</td><td rowspan=1 colspan=1>4.61%</td></tr><tr><td rowspan=1 colspan=1>ROX</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>31.33</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.88%</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.09%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>3.56%</td></tr><tr><td rowspan=3 colspan=1>Salmonella</td><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>14.02%</td><td rowspan=1 colspan=1>1.23</td><td rowspan=1 colspan=1>3.45%</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>2.85%</td><td rowspan=1 colspan=1>5.25</td><td rowspan=1 colspan=1>14.72%</td></tr><tr><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>33.49</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>2.76%</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.28%</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.63%</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.45%</td></tr><tr><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>32.22</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.04%</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.67%</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.48%</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>3.02%</td></tr></table>

b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Positive and Negative External Controls

External Positive Controls are intended to monitor for substantial reagent failure and External Negative Controls are used to detect reagent or environmental contamination (or carry-over) from positive specimens or nucleic acids (amplicon). External controls are not provided with the BD MAX Enteric Bacterial Panel; however recommendations for control preparation and testing are provide in the package insert.

During the clinical study, external positive and negative controls were included in each run and in the case of a failure of either or both external controls, testing of all samples included in the run was repeated from the stored SBTs. Negative external controls consisted of saline. Positive controls were prepared as organism suspensions using cultures of Salmonella enterica subsp. enterica serovar Typhimurium (ATCC 14028), Shigella sonnei (ATCC 9290), Campylobacter jejuni subsp. jejuni (ATCC 33291), and Escherichia coli $( s t x 1 )$ (ATCC 43890). Each positive control was tested on a rotating basis.

During the clinical study, a total of 476 external positive controls and 476 external negative controls were tested resulting in valid expected results for 456 $( 9 5 . 8 \% )$ and 450 $( 9 4 . 5 \% )$ for positive and negative controls respectively. Testing of positive controls yielded three $( 0 . 6 \% )$ valid but incorrect results, four $( 0 . 8 \% )$ Unresolved results, five $( 1 . 1 \% )$ Incomplete results, and eight $( 1 . 7 \% )$ Indeterminate results. Testing of negative controls yielded three $( 0 . 6 \% )$ false negative results, ten $( 2 . 1 \% )$ Unresolved results, seven $( 1 . 5 \% )$ Incomplete results, and six $( 1 . 3 \% )$ Indeterminate results.

Internal Specimen Processing Control:

Each BD MAX Enteric Bacterial Extraction Tube contains a Sample Processing Control (SPC) which is a plasmid containing a synthetic target DNA sequence. The SPC monitors the efficiency of DNA capture, washing and elution during the sample processing steps, as well as the efficiency of DNA amplification and detection during PCR analysis. If the SPC result fails to meet the acceptance criteria, the test result of will be reported as Unresolved (UNR). An Unresolved result is indicative of sample-associated inhibition or reagent failure. The user is instructed to repeat any sample reported as Unresolved.

Sample Processing Control Effectiveness Study

A Sample Processing control effectiveness study was performed to evaluate the effectiveness of the SPC to monitor for substantial reagent or process failure. This study was designed to test for failure in the extraction process, failure in PCR amplification and/or failure due to the presence of PCR inhibitors.

For the SPC effectiveness study, the following conditions were tested:

Failure in PCR amplification was tested by replacing the BD MAX Enteric Bacterial Panel Master Mix tube with an empty Snap Tube (expected result $=$ Indeterminate (IND)).   
Failure in the Extraction process was tested by replacing the BD MAX Enteric Bacterial Panel Extraction tube with an empty Snap Tube (expected result $=$ UNR).   
Failure due to the presence of PCR inhibitors was tested by placing $1 0 0 \mathrm { m M }$ EDTA solution into an empty Snap Tube of the BD MAX System test rack prior to beginning a run (expected result ${ \bf \Phi } = \mathrm { U N R } { \bf \Phi }$ .

A minimum of 24 replicates were tested for each test condition for which the assay performed as expected, producing the expected failures as shown in the following table.

Results of Sample Processing Control Effectiveness Study   

<table><tr><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=1>Number of Reps</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>UNR</td><td rowspan=1 colspan=1>IND</td></tr><tr><td rowspan=1 colspan=1>Empty Tx Tube</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Empty MM Tube</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>EDTA in Snap Tube</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Study #1: Specimen stability testing included testing of clinical specimens previously determined positive for Salmonella spp., Shigella spp., Campylobacter spp, or $E$ . coli $( s t x 1 )$ . Specimens were mixed together to obtain baseline $\mathrm { C t }$ values near the LoD for each targeted analyte (i.e., Ct values between 29 and 32). Testing included 24 replicates for each specimen at baseline, day 1 and day 2 for storage at $2 5 ^ { \circ } \mathrm { C }$ , and day 2 and day 5 for storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ . Acceptance criteria for each analyte included positive results for $59 5 \%$ of replicates tested with no more than a mean increase of 2 Cts as compared to baseline testing.

For unpreserved specimens, initial testing yielded acceptable results for all four targeted analytes for specimens stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ up to five days. For unpreserved specimens stored at $2 5 ^ { \circ } \mathrm { C }$ , the study yielded acceptable results for Salmonella spp, Shigella spp, and $E$ . coli $( s t x 1 )$ positive specimens; however study results did not pass acceptance criteria for detection of Campylobacter spp. At Day 1 of storage at 25C, $8 7 \%$ of replicates for Campylobacter spp. were positive with an increase in mean Ct values of 2.82. At Day 2 of storage at $2 5 \mathrm { { ^ \circ C } }$ , $100 \%$ of replicates for Campylobacter spp. were positive with an increase in mean Ct values of 2.41.

For Cary-Blair preserved specimens, initial testing yielded acceptable results for all analytes when specimens were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ . For Cary-Blair specimens stored at $2 5 ^ { \circ } \mathrm { C }$ , $E .$ . coli $( s t x 1 )$ did not pass acceptance criteria for samples stored at day 1 or day 2 and Shigella spp. did not pass acceptance criteria for samples stored at $2 5 ^ { \circ } \mathrm { C }$ at day 2. No additional testing was performed for Cary-Blair specimens positive with Shigella spp. because the final stability claim for Cary-Blair specimens stored at $2 5 ^ { \circ } \mathrm { C }$ is 24 hours.

Study #2: It was determined that mixing of analytes and stool matrices may be the reason that initial study results did not meet the acceptance criteria for $2 5 \mathrm { { ^ \circ C } }$ storage for positive unpreserved stool specimens containing Campylobacter spp. and positive Cary-Blair specimens containing $E$ . coli $( s t x I )$ . Additional testing for these two analytes was performed on individual positive specimens and the following results were obtained:

Unpreserved Specimens: Campylobacter spp. - $100 \%$ of replicates positive with difference in mean Ct of $^ { < 2 }$ Ct.   
Cary-Blair Specimens: $E .$ . coli $( s t x I ) - 1 0 0 \%$ of replicates positive with difference in mean Ct of $^ { < 2 }$ Ct.

In summary, the specimen stability studies support that targeted DNA is stable in both Unpreserved and Cary-Blair preserved stool specimens when stored for up to one day at $2 5 ^ { \circ } \mathrm { C }$ or five days or at $2 { - } 8 ^ { \circ } \mathrm { C }$ prior to testing with the BD MAX Enteric Bacterial Panel.

Stability of specimen in SBT (Sample Buffer Tube)

An analytical study was performed to evaluate the stability of target DNA in SBTs for both unpreserved and Cary-Blair specimens. Organism mixes were prepared with Salmonella, Campylobacter, Shigella and $E$ . coli $( s t x 1 / s t x 2 ) $ ) cultures spiked into SBTs with each organism near the assay LoD (approximately $1 . 5 \mathrm { x }$ LoD). SBTs were also inoculated with individual negative unpreserved or Cary-Blair stool specimens (negative matrix). Testing included 24 replicates for each specimen type (unpreserved and CaryBlair) at baseline , after storage at 24 and 48 hours at $2 5 { \pm } 2 \mathrm { C } ^ { \circ }$ , and after storage at 48 and 120 hours at $5 \pm 3 ~ ^ { \circ } \mathrm { C }$ .

Acceptance criteria included a minimum of 20 valid replicate results with $59 5 \%$ of replicates positive with a mean $\mathrm { C t }$ increase of less than 2 as compared to baseline testing results for each storage condition evaluated.

For Cary-Blair specimens, SBT stability for all targets met the acceptance criteria for all targets at all storage conditions evaluated. For Unpreserved stood specimens, two of the four specimen mixes failed to meet baseline acceptance criteria (i.e., there was less than $9 5 \%$ detection at baseline) and therefore two new sets of specimen mixes were prepared. Testing of the final four specimen mixes yielded acceptable baseline results and passed the acceptance criteria for all targets and storage conditions tested $5 \pm 3 ~ ^ { \circ } \mathrm { C }$ and $2 5 \pm 2$ $^ { \circ } \mathrm { C } $ .

In summary, this analytical study supports the storage claims for Unpreserved and CaryBlair stools in SBT $( 2 5 \pm 2 \ ^ { \circ } \mathrm { C }$ for up to 48 hours and $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 5 days).

# d. Detection limit:

Limit of Detection (LoD) for each targeted organism was determined in an analytical study. Samples were prepared in two individual organism mixes containing representative strains of organisms detected by the BD MAX Enteric Bacterial Panel. Target Mix 1 consisted of Salmonella typhimurium (ATCC 14028), E. coli O157 (stx1 & stx2) (ENF 10513), Campylobacter coli (ATCC 43134) and Shigella sonnei (ATCC 10523). Target mix 2 consisted of Salmonella enteritidis (ATCC 13076), E. coli O157 $( s t x 2 )$ (ATCC 43889), Campylobacter jejuni (ATCC 43429) and Shigella flexneri (ATCC 700930). An additional E.coli strain containing stx1 (ATCC 43890) was also tested individually. Samples were prepared in unpreserved stool matrix as well as in stool/CaryBlair matrix. Prior to sample preparation, pooled matrices were determined to be negative for all analytes by the BD MAX Bacterial Enteric Bacterial Panel. Multiple dilutions of each strain were tested with the BD MAX Enteric Bacterial Panel with up to 36 replicates tested per dilution. Testing was performed using three reagent lots and three BD MAX Systems.

A minimum of 30 valid results per dilution were used to determine the LoD for each organism strain using a statistically-based methodology which allows for the determination of LoD with a $9 5 \%$ confidence interval. Each LoD was calculated using a linear logistic model that assesses the relationship between the probability of the response and the bacterial concentration. The LoD, defined as the lowest concentration at which greater than $9 5 \%$ of all replicates tested positive, ranged from 10 to 653 CFU/mL (in SBT) and 1,500 to 97,950 CFU/mL (in stool) for preserved Cary-Blair specimens and

42 to 910 CFU/mL (in SBT) and 6,300 to 136,500 CFU/ mL (in stool) for unpreserved specimens.

Results of the study are shown in the following table.

BD MAX Enteric Panel Limit of Detection   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Unpreserved</td><td colspan="1" rowspan="1">Cary-Blair Preserved</td></tr><tr><td colspan="3" rowspan="1">Salmonella typhimurium (ATCC 14028)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">296 [233  376]</td><td colspan="1" rowspan="1">193 [142  263]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">44,400 [34,950  56,400]</td><td colspan="1" rowspan="1">28,950 [21,300  39,450]</td></tr><tr><td colspan="3" rowspan="1">Salmonella enteriditis (ATCC 13076)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">620 [403  954]</td><td colspan="1" rowspan="1">502 [345 - 729]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">|93,000 [60,450  143,100]75,300 [51,750  109,350]</td><td colspan="1" rowspan="1">|93,000 [60,450  143,100]75,300 [51,750  109,350]</td></tr><tr><td colspan="3" rowspan="1">Campylobacter coli (ATCC 43134)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">95 [70  128]</td><td colspan="1" rowspan="1">55 [41  76]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">14,250 [10,500  19,200]</td><td colspan="1" rowspan="1">8,250 [6,150  11,400]</td></tr><tr><td colspan="3" rowspan="1">Campylobacter jejuni (ATCC 43429)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">42 [36  49]</td><td colspan="1" rowspan="1">10 [9 - 10]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">6,300 [5,400  7,350]</td><td colspan="1" rowspan="1">1,500 [1,350 - 1,500]</td></tr><tr><td colspan="3" rowspan="1">Shigella flexneri (ATCC 700930)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">374 [249  561]</td><td colspan="1" rowspan="1">229 [151  347]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">56,100 [37,350  84,150]</td><td colspan="1" rowspan="1">34,350 [22,650  52,050]</td></tr><tr><td colspan="3" rowspan="1">Shigella sonnei (ATCC 10523)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval</td><td colspan="1" rowspan="1">84 [59 - 118]</td><td colspan="1" rowspan="1">124 [67  229]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">12,600 [8,850  17,700]</td><td colspan="1" rowspan="1">18,600 [10,050  34,350]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E. coli stx1 (ATCC 43890)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">255 [195 - 332]</td><td colspan="1" rowspan="1">223 [167  299]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)[95% confidence interval]</td><td colspan="1" rowspan="1">38,202 [29,259  49,865]</td><td colspan="1" rowspan="1">33,495 [25,026  44,817]</td></tr><tr><td colspan="3" rowspan="1">E. coli stx1/stx2 (BD ENF 10513)</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">910 [550 - 1,505]</td><td colspan="1" rowspan="1">653 [384  1111]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)</td><td colspan="1" rowspan="1">136,500 [82,50,25,7501</td><td colspan="1" rowspan="1">97,950 [57,600 166,650</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E. coli stx2 (ATCC 43889)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in SBT)[95% confidence interval]</td><td colspan="1" rowspan="1">722 [519  1006]</td><td colspan="1" rowspan="1">599 [291  1231]</td></tr><tr><td colspan="1" rowspan="1">LoD (CFU/mL in stool)</td><td colspan="1" rowspan="1">108,300 [7,850  10,9001</td><td colspan="1" rowspan="1">89,850 [43,650  184,650]</td></tr></table>

# e. Analytical Reactivity (Inclusivity):

Inclusivity was assessed through testing of a wide range of clinically relevant GI pathogen strains, genotypes, serotypes and clinical isolates. Samples were prepared using 121 well-characterized clinical strains and strains from public collections.

Inclusivity testing included 30 strains of Campylobacter spp. (C. jejuni and C. coli), 30 strains of Salmonella spp. (S. enterica and S. bongori), 31 strains of Shigella spp./Enteroinvasive $E .$ . coli and 35 organism strains positive for Shiga-toxin Types 1 and/or 2 (including $3 0 E .$ . coli strains of which 20 were non-O157, and $5 S .$ . dysenteriae strains). Testing consisted of 35 organism mixes containing three or four targeted organisms in each organism mix with individual organism concentrations at 1x LoD for each organism. Shigella dysenteriae and Shiga-toxin producing $E .$ . coli were tested individually because they both carry Shiga-toxin genes. All samples were prepared in SBTs using unpreserved stool matrix. Testing included 20-24 replicates for each organism mix or strain (8 replicates each with three different reagent lots).

Acceptance criteria for a strain to be considered inclusive required an $8 5 \%$ positivity rate at 1x LoD with a minimum of 20 valid replicates. Additional testing was performed at higher organism concentrations for targets that did not meet the acceptance criteria.

The BD MAX Enteric Bacterial Panel correctly identified 120 of the 121 strains tested at the LOD. One strain of Shigella sonnei (ENF 15987) demonstrated $7 9 . 1 7 \%$ positivity at a concentration of $5 6 . 1 \mathrm { C F U / m L }$ of SBT. The isolate was further evaluated and yielded $100 \%$ positivity at a higher concentration of $4 0 5 \ : \mathrm { C F U / m L }$ of SBT. Seven other strains of Shigella sonnei were evaluated during the analytical inclusivity study and met the study acceptance criteria at a concentration of 56.1 $\mathrm { C F U / m L }$ .

In summary, the BD MAX Enteric Bacterial Panel demonstrated acceptable performance for detection of a wide variety of targeted strains. The tables below include a listing of the 121 strains, serovars, and subspecies that were included in the study.

Additional in silico analysis was performed to determine the expected detection of stx2 subtypes a-g. The analysis supports the predicted detection of all $s t x 2$ subtypes with the exception of subtype $f$ for which the targeted sequence contains several mismatches with BD MAX Enteric Bacterial Panel primers. A limitation is included in package insert regarding this subtype.

Salmonella spp: Strains Evaluated for Inclusivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>GeographicOrigin</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>GeographicOrigin</td></tr><tr><td rowspan=1 colspan=1>Salmonella agona</td><td rowspan=1 colspan=1>BD ENF15960</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella typhi</td><td rowspan=1 colspan=1>ATCC10749</td><td rowspan=1 colspan=1>UK</td></tr><tr><td rowspan=1 colspan=1>Salmonella anatum</td><td rowspan=1 colspan=1>BD ENF15961</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella virchow</td><td rowspan=1 colspan=1>ATCC51955</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Salmonella braenderup</td><td rowspan=1 colspan=1>BD ENF15962</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella bareilly</td><td rowspan=1 colspan=1>ATCC 9115</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Salmonellacholeraesuis</td><td rowspan=1 colspan=1>ATCC 7001</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Salmonella thompson</td><td rowspan=1 colspan=1>BD ENF15968</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Salmonella hadar</td><td rowspan=1 colspan=1>ATCC 51956</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Salmonellaschwarzengrund</td><td rowspan=1 colspan=1>BD ENF7452</td><td rowspan=1 colspan=1>California</td></tr><tr><td rowspan=1 colspan=1>Salmonella heidelberg</td><td rowspan=1 colspan=1>BD ENF15963</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>ATCC43975</td><td rowspan=1 colspan=1>Paris</td></tr><tr><td rowspan=1 colspan=1>Salmonella infantis</td><td rowspan=1 colspan=1>ATCC 51741</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>BD ENF16009</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Salmonella iaviana</td><td rowspan=1 colspan=1>BD ENF13330</td><td rowspan=1 colspan=1>NorthCarolina</td><td rowspan=1 colspan=1>Salmonella entericasubsp. arizonae</td><td rowspan=1 colspan=1>ATCC13314</td><td rowspan=1 colspan=1>UK</td></tr><tr><td rowspan=1 colspan=1>Salmonella montevideo</td><td rowspan=1 colspan=1>BD ENF15964</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella entericasubsp. diarizonae</td><td rowspan=1 colspan=1>ATCC29226</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Salmonella muenchen</td><td rowspan=1 colspan=1>BD ENF 8388</td><td rowspan=1 colspan=1>Maryland</td><td rowspan=1 colspan=1>Salmonella entericasubsp. diarizonae</td><td rowspan=1 colspan=1>ATCC43973</td><td rowspan=1 colspan=1>UK</td></tr><tr><td rowspan=1 colspan=1>Salmonella newport</td><td rowspan=1 colspan=1>BD ENF15965</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella entericasubsp. houtenae</td><td rowspan=1 colspan=1>ATCC15788</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Salmonellaoiranienburg</td><td rowspan=1 colspan=1>BD ENF 7482</td><td rowspan=1 colspan=1>California</td><td rowspan=1 colspan=1>Salmonella entericasubsp. houtenae</td><td rowspan=1 colspan=1>ATCC43974</td><td rowspan=1 colspan=1>Paris</td></tr><tr><td rowspan=1 colspan=1>Salmonella paratyphi A</td><td rowspan=1 colspan=1>ATCC 9150</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Salmonella entericasubsp. indica</td><td rowspan=1 colspan=1>ATCC43976</td><td rowspan=1 colspan=1>Paris</td></tr><tr><td rowspan=1 colspan=1>Salmonella paratyphi B</td><td rowspan=1 colspan=1>ATCC 51962</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Salmonella entericasubsp. indica</td><td rowspan=1 colspan=1>ATCCBAA-1576</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Salmonella saintpaul</td><td rowspan=1 colspan=1>BD ENF15967</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Salmonella entericasubsp. salamae</td><td rowspan=1 colspan=1>ATCC43972</td><td rowspan=1 colspan=1>Paris</td></tr></table>

BD NH and BD ENF are designations for BD internal strains

Campylobacter spp: Strains Evaluated for Inclusivity   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">GeographicOrigin</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">GeographicOrigin</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43483</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. doylei</td><td colspan="1" rowspan="1">ATCC49349</td><td colspan="1" rowspan="1">Australia</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43484</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. doylei</td><td colspan="1" rowspan="1">ATCC BAA-1458</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43133</td><td colspan="1" rowspan="1">Illinois</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. doylei</td><td colspan="1" rowspan="1">BD NH 450</td><td colspan="1" rowspan="1">Australia</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43135</td><td colspan="1" rowspan="1">Illinois</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. doylei</td><td colspan="1" rowspan="1">BD NH 451</td><td colspan="1" rowspan="1">California</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43136</td><td colspan="1" rowspan="1">Illinois</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. doylei</td><td colspan="1" rowspan="1">BD NH 452</td><td colspan="1" rowspan="1">California</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43472</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC33292</td><td colspan="1" rowspan="1">Colorado</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43473</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC33560</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43478</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC35918</td><td colspan="1" rowspan="1">Illinois</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43481</td><td colspan="1" rowspan="1">Colorado</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC29428</td><td colspan="1" rowspan="1">Paris</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43482</td><td colspan="1" rowspan="1">Toronto</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC43434</td><td colspan="1" rowspan="1">Ottawa,Ontario</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 43485</td><td colspan="1" rowspan="1">Atlanta</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC43435</td><td colspan="1" rowspan="1">Toronto</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 49941</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC43449</td><td colspan="1" rowspan="1">Toronto,Ontario</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">BD NH 422</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC43503</td><td colspan="1" rowspan="1">Ottawa</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">BD NH 423</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">BD NH 544</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">BD NH 424</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC700819</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# Shigella spp: Strains Evaluated for Inclusivity

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">GeographicOrigin</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">GeographicOrigin</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC12028</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC25931</td><td colspan="1" rowspan="1">Paris</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 8700</td><td colspan="1" rowspan="1">Virginia</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF5704</td><td colspan="1" rowspan="1">California</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td><td colspan="1" rowspan="1">Virginia</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF8063</td><td colspan="1" rowspan="1">Washington St.</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">BD ENF15975</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF15986</td><td colspan="1" rowspan="1">Canada</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">BD ENF15976</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF15987</td><td colspan="1" rowspan="1">Canada</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">ATCC29903</td><td colspan="1" rowspan="1">Virginia</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF15988</td><td colspan="1" rowspan="1">Canada</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">ATCC33948</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC29930</td><td colspan="1" rowspan="1">Virginia</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF2900</td><td colspan="1" rowspan="1">WashingtonSt.</td><td colspan="1" rowspan="1">Escherichia coli (EIEC)</td><td colspan="1" rowspan="1">BD ENF15626</td><td colspan="1" rowspan="1">Norway</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF7419</td><td colspan="1" rowspan="1">California</td><td colspan="1" rowspan="1">Escherichia coli0124:NM (EIEC)</td><td colspan="1" rowspan="1">ATCC43893</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">ATCC12022</td><td colspan="1" rowspan="1">Virginia</td><td colspan="1" rowspan="1">Escherichia coli029:NM (EIEC)</td><td colspan="1" rowspan="1">ATCC43892</td><td colspan="1" rowspan="1">Virginia</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF15983</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">Shigella dysenteriae*</td><td colspan="1" rowspan="1">ATCC11835</td><td colspan="1" rowspan="1">Virginia</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF15984</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">Shigella dysenteriae*</td><td colspan="1" rowspan="1">ATCC13313</td><td colspan="1" rowspan="1">Virginia</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">Shigella dysenteriae*</td><td colspan="1" rowspan="1">ATCC 9361</td><td colspan="1" rowspan="1">Virginia</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">15985</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF15428</td><td colspan="1" rowspan="1">California</td><td colspan="1" rowspan="1">Shigella dysenteriae*</td><td colspan="1" rowspan="1">BD ENF2932</td><td colspan="1" rowspan="1">Washington St.</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">BD ENF2903</td><td colspan="1" rowspan="1">WashingtonSt.</td><td colspan="1" rowspan="1">Shigella dysenteriae*</td><td colspan="1" rowspan="1">BD ENF15977</td><td colspan="1" rowspan="1">Canada</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">BD ENF7140</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

\*Shigella dysenteriae strains positive for the Shigella spp. target as well as for stx1

Shiga-toxin Producing Organisms (stx1 and/or stx2): Strains Evaluated for Inclusivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>GeographicOrigin</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>GeographicOrigin</td></tr><tr><td rowspan=1 colspan=6>Strains containing stx 1/2</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0111:H8</td><td rowspan=1 colspan=1>ATCC BAA-179</td><td rowspan=1 colspan=1>Alabama</td><td rowspan=1 colspan=1>Escherichia coli0157:H7</td><td rowspan=1 colspan=1>ATCC43894</td><td rowspan=1 colspan=1>Michigan</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O103:H8</td><td rowspan=1 colspan=1>BD ENF15804</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Escherichia coli091:H21</td><td rowspan=1 colspan=1>ATCC51434</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliO113:H21</td><td rowspan=1 colspan=1>ATCC BAA-177</td><td rowspan=1 colspan=1>New Mexico</td><td rowspan=1 colspan=1>Escherichia coli0157:H7</td><td rowspan=1 colspan=1>ATCC35150</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=6>Strains containing stx 2</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0104:H21</td><td rowspan=1 colspan=1>ATCC BAA178</td><td rowspan=1 colspan=1>Montana</td><td rowspan=1 colspan=1>Escherichia coli091:H21</td><td rowspan=1 colspan=1>ATCC 51435</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0145:H28</td><td rowspan=1 colspan=1>ATCC BAA-2129</td><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>Escherichia coliOX3:H21</td><td rowspan=1 colspan=1>BD ENF15816</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157</td><td rowspan=1 colspan=1>BDENF13568</td><td rowspan=1 colspan=1>WashingtonSt.</td><td rowspan=1 colspan=1>Escherichia coli0121:H19</td><td rowspan=1 colspan=1>ATCC BAA-2219</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157</td><td rowspan=1 colspan=1>BDENF13604</td><td rowspan=1 colspan=1>WashingtonSt.</td><td rowspan=1 colspan=1>Escherichia coli0145:H25</td><td rowspan=1 colspan=1>ATCC BAA-2211</td><td rowspan=1 colspan=1>Minnesota</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0157:NM</td><td rowspan=1 colspan=1>BDENF10301</td><td rowspan=1 colspan=1>Wyoming</td><td rowspan=1 colspan=1>Escherichia coli0145:H48</td><td rowspan=1 colspan=1>ATCC BAA-1652</td><td rowspan=1 colspan=1>Belgium</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0145:NM</td><td rowspan=1 colspan=1>BDENF15812</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Escherichia coliO111:H8</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>Missouri</td></tr><tr><td rowspan=1 colspan=6>Strains containing stx 1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O103:H2</td><td rowspan=1 colspan=1>BDENF15805</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Escherichia coli0111:NM</td><td rowspan=1 colspan=1>BDENF15809</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O111:H8</td><td rowspan=1 colspan=1>ATCC BAA-184</td><td rowspan=1 colspan=1>Virginia</td><td rowspan=1 colspan=1>Escherichia coli0157:NM</td><td rowspan=1 colspan=1>ATCC700376</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0145:NM</td><td rowspan=1 colspan=1>BDENF15811</td><td rowspan=1 colspan=1>Canada</td><td rowspan=1 colspan=1>Escherichia coli0145:NM</td><td rowspan=1 colspan=1>ATCC BAA-2222</td><td rowspan=1 colspan=1>Minnesota</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157</td><td rowspan=1 colspan=1>BDENF13581</td><td rowspan=1 colspan=1>WashingtonSt.</td><td rowspan=1 colspan=1>Escherichia coli0103:H25</td><td rowspan=1 colspan=1>ATCC BAA-2213</td><td rowspan=1 colspan=1>Virginia</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157</td><td rowspan=1 colspan=1>BD ENF7582</td><td rowspan=1 colspan=1>Newfoundland</td><td rowspan=1 colspan=1>Escherichia coli0103:H11</td><td rowspan=1 colspan=1>ATCC BAA-2215</td><td rowspan=1 colspan=1>Idaho</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157:H7</td><td rowspan=1 colspan=1>BDENF13579</td><td rowspan=1 colspan=1>Washingon St.</td><td rowspan=1 colspan=1>Escherichia coli0103:H2</td><td rowspan=1 colspan=1>ATCC BAA-2210</td><td rowspan=1 colspan=1>Wisconsin</td></tr></table>

# f. Analytical specificity:

A study was performed with samples containing high concentrations of non-target organisms to evaluate the analytical specificity of the BD MAX Enteric Bacterial Panel. The study included organisms phylogenetically related to targeted organisms as well as other bacteria, yeast, parasites and viruses that may be found in stool specimens. Samples were prepared using quantified organism preparations and pooled negative unpreserved stool matrix.

The study included 106 bacterial strains, three parasites, two Candida spp., and 15 viruses. Included were nine Campylobacter spp. strains (non jejuni or coli) as well as six non Shiga-toxin producing $E$ . coli strains. Each organism was initially tested in triplicate and if a positive result was obtained, an additional 20 replicates were tested to confirm cross-reactivity. The following non-targeted organisms were evaluated for potential cross-reactivity with the BD MAX Enteric Bacterial Panel:

Nine Campylobacter strains (Campylobacter species other than $C .$ . jejuni or $C .$ . coli), tested at a concentration $\geq \dot { 1 } \dot { \mathrm { ~ x ~ } } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT. Six $E$ . coli strains other than Shiga toxin-producing strains, tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT.   
• Ninety-nine other bacterial strains (including 53 species and subspecies), tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT $( \mathrm { o r } \tilde { \sim } 1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ genomic DNA copies $/ \mathrm { m L }$ or $1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ elementary bodies $\mathrm { \Delta } / \mathrm { m L }$ of SBT).   
Fifteen different viruses tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { ~ P F U / m L }$ of SBT. Three different parasites tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ cysts/mL of SBT.

Analytical Specificity: Bacterial Strains Evaluated   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">49176</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">12014</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">19606</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">8739</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">17925</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">10536</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">49847</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">33605</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">8750</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">35469</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">35098</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">33650</td></tr><tr><td colspan="1" rowspan="1">Arcobacter butzleri</td><td colspan="1" rowspan="1">49616</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">33821</td></tr><tr><td colspan="1" rowspan="1">Arcobacter cryaerophilus</td><td colspan="1" rowspan="1">43157</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">27725</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">49064</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">14019</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">43185</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">11604</td></tr><tr><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">43184</td><td colspan="1" rowspan="1">Helicobacter fennelliae</td><td colspan="1" rowspan="1">35683</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">43183</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">43504</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">15706</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">13182</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">15707</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">33495</td></tr><tr><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">CCUG 17580</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">4355</td></tr><tr><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">CCUG 47528</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">23272</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus subsp. fetus</td><td colspan="1" rowspan="1">27374</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">15346</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus subsp. venerealis</td><td colspan="1" rowspan="1">19438</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">33999</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">33236</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">19120</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hominis</td><td colspan="1" rowspan="1">BAA-381</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">33090</td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">43675</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">19115</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">33238</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">25830</td></tr><tr><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">49815</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td><td colspan="1" rowspan="1">14963</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">33431</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">27337</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">VR-879</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">14029</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">25405</td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">25260</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">33128</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">25845</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">27156</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">29906</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">51115</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">35198</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">17858</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">13315</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">43598</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">27971</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">CCUG 8864-9689</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">29944</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">43255</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">33672</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">BAA-1805</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">27853</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">43593</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">13525</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">10543</td><td colspan="1" rowspan="1">Ruminococcus bromii</td><td colspan="1" rowspan="1">27255</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">35085</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">35551</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">33030</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">13880</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">29098</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">25923</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">15947</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">12228</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">25559</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">13637</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">13048</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">13813</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">35030</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">43078</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">49605</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">27335</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">43198</td><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">19436</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">51266</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td><td colspan="1" rowspan="1">49490</td></tr><tr><td colspan="1" rowspan="1">Enterococus faecalis</td><td colspan="1" rowspan="1">29212</td><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">10790</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">49032</td><td colspan="1" rowspan="1">Vibrio cholera</td><td colspan="1" rowspan="1">13498</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">49573</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">17802</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">49612</td><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">43970</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">49427</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">9610</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">25922</td><td colspan="1" rowspan="1">Yersinia rohdei</td><td colspan="1" rowspan="1">43380</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">35520</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

Analytical Specificity: Yeast, Parasites, and Viruses Evaluated   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>24433</td><td rowspan=1 colspan=1>Coxsackie B1</td><td rowspan=1 colspan=1>VR-687</td></tr><tr><td rowspan=1 colspan=1>Candida catenulate</td><td rowspan=1 colspan=1>18821</td><td rowspan=1 colspan=1>HHV-5 Cytomegalovirus</td><td rowspan=1 colspan=1>AD-169</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>87712</td><td rowspan=1 colspan=1>Enterovirus type 69</td><td rowspan=1 colspan=1>VR-785</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30458</td><td rowspan=1 colspan=1>Human Papillomavirus Type 16</td><td rowspan=1 colspan=1>45113</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50137</td><td rowspan=1 colspan=1>Human Papillomavirus Type 18</td><td rowspan=1 colspan=1>45152</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 2</td><td rowspan=1 colspan=1>VR-680</td><td rowspan=1 colspan=1>Herpes Simplex Virus I</td><td rowspan=1 colspan=1>VR-539</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 14</td><td rowspan=1 colspan=1>VR-15</td><td rowspan=1 colspan=1>Herpes Simplex Virus II</td><td rowspan=1 colspan=1>VR-734</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 40</td><td rowspan=1 colspan=1>VR-931</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 41</td><td rowspan=1 colspan=1>VR-930</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>VR-2274</td></tr></table>

Analytical specificity testing produced expected negative results for all organisms evaluated with the exception of Enterobacter aerogenes, Aeromonas hydrophila, and Abiotrophia defectiva which each yielded unexpected positive results for one of three replicates. For these three organisms, additional testing of 20 replicates gave no

additional false positive results. Therefore these organisms were deemed to be non-crossreactive with the assay.

The following organisms which are expected to be detected by the assay were evaluated to ensure the expected positive and negative results occur in the appropriate optical channels on the BD MAX system:

Three Campylobacter spp.; one $C .$ . coli, one $C .$ . jejuni, subsp. doylei and one C. jejuni, subsp. jejuni at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT.   
Four E. coli; two O157 and two non-O157 strains containing stx1/stx2 genes tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT.   
Five Salmonella strains tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT.   
Three Shigella spp.; one each of $S .$ sonnei, S. boydii, S. flexneri, and S. dysentariae tested at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT.

Testing of these targeted organisms produced the expected positive and negative results with the exception of one strain of Shigella boydii (ATCC 12028) which unexpectedly gave a positive result for $s t x 1 / s t x 2$ in one of three replicates. Additional testing of this strain produced positive results with 8 out of 20 replicates for the presence of $s t x 1 / s t x 2$ . Four additional strains of Shigella boydii tested in Inclusivity studies did not give positive results for $s t x 1 / s t x 2$ .

In conclusion, all organisms tested by the BD MAX Enteric Bacterial panel in this Analytical Specificity study are not considered cross-reactants with the exception of Shigella boydii (ATCC 12028).

g. Matrix equivalence Study

N/A

# h. Interference Study

A study was performed to evaluate potential interference of biological and chemical substances that may be present in stool specimens in samples tested with the BD MAX Enteric Bacterial Panel. Of these substances, a pool of eight different antibiotics was tested with each antibiotic at a concentration that might be excreted in stool. Testing included exogenous substances at $50 \%$ concentrations (1:1 substance/stool ratio) and endogenous substances at varying concentrations. Samples were prepared in unpreserved stool as a worst case representative specimen matrix due to the presence of higher concentrations of endogenous interfering substances in unpreserved stool as compared to Cary-Blair preserved stool.

Samples were prepared by inoculating SBTs with each potentially interfering substance, unpreserved stool matrix, as well as organism suspensions at approximate concentrations of 1.5 X LoD for each targeted organism. The strains used included Salmonella typhimurium (ATCC 14028), Campylobacter coli (ATCC 43134), Shigella sonnei (ATCC 10523) and $E$ . coli O157:H7 $( s t x 1 )$ (BD ENF 10513). Testing included 24 positive and 12 negative samples for each potential interfering substance. If any interference was seen, additional testing was performed with lower concentrations of the interfering substance.

Testing demonstrated no interference for all substances tested with the exception of Nystatin cream, spermicidal lubricant, and Vagisil. Initial testing showed assay interference for samples containing Nystatin cream and spermicidal lubricant at $50 \%$ concentrations (1:1 substance/stool ratio); however it was determined that these substances were tested at concentrations that could not form a uniform homogenous mixture. Repeat testing was performed using lower concentrations of each substance and study results yielded no interference for Nystatin cream or spermicidal lubricant at concentrations of $31 \%$ and $34 \%$ respectively. Interference was seen for samples prepared with Vagisil at concentrations greater than $9 . 2 \%$ and this substance is described in the package insert as a potential interferent for the assay. The following table lists all substances and concentrations evaluated in the interference study.

Endogenous and Exogenous Substances Evaluated   

<table><tr><td colspan="1" rowspan="1">Potential Interferent</td><td colspan="1" rowspan="1">Initial Level Tested</td><td colspan="1" rowspan="1">Level at which aPassing Result wasobtained</td></tr><tr><td colspan="1" rowspan="1">Fecal Fat</td><td colspan="1" rowspan="1">7%</td><td colspan="1" rowspan="1">7%</td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">0.1% (1mg/mL)</td><td colspan="1" rowspan="1">0.1%</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Whole human blood</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Hydrocortisone Cream</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Antiseptic Towelettes</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Enema</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Hemorrhoidal Gel</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Nystatin Cream</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">31%</td></tr><tr><td colspan="1" rowspan="1">Topical Antibiotic</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Spermicidal Lubricant</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">34%</td></tr><tr><td colspan="1" rowspan="1">Diaper Rash Cream</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">50%</td></tr><tr><td colspan="1" rowspan="1">Vagisil</td><td colspan="1" rowspan="1">50% (1:1)</td><td colspan="1" rowspan="1">9.2%</td></tr><tr><td colspan="1" rowspan="1">Laxatives</td><td colspan="1" rowspan="1">4.7 % (47 mg/mL)</td><td colspan="1" rowspan="1">4.7%</td></tr><tr><td colspan="1" rowspan="1">Anti-Diarrheal</td><td colspan="1" rowspan="1">5.9% (59 mg/mL)</td><td colspan="1" rowspan="1">5.9%</td></tr><tr><td colspan="1" rowspan="1">Anti-Diarrheal (pill)</td><td colspan="1" rowspan="1">0.38% (3.75 mg/mL)</td><td colspan="1" rowspan="1">0.38%</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">1.6% (15.8 mg/mL)</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Sulfamethoxazole</td><td colspan="1" rowspan="1">8.0%b (80.0 mg/mL)</td><td colspan="1" rowspan="1">8.0%</td></tr><tr><td colspan="1" rowspan="1">Tetracycline HCI</td><td colspan="1" rowspan="1">1.6%(16.0 mg/mL)</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Amoxicillin</td><td colspan="1" rowspan="1">6.4%(64.0 mg/mL)</td><td colspan="1" rowspan="1">6.4%</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">6.0%(60.8 mg/mL)</td><td colspan="1" rowspan="1">6.0%</td></tr><tr><td colspan="1" rowspan="1">Erythromycin</td><td colspan="1" rowspan="1">1.5% (14.0 mg/mL)</td><td colspan="1" rowspan="1">1.5%</td></tr><tr><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">5.4% (5.4 mg/mL)</td><td colspan="1" rowspan="1">5.4%</td></tr><tr><td colspan="1" rowspan="1">Trimethoprim</td><td colspan="1" rowspan="1">1.6%(16.0mg/mL)</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Antacids</td><td colspan="1" rowspan="1">3.1%(31mg/mL)</td><td colspan="1" rowspan="1">3.1%</td></tr><tr><td colspan="1" rowspan="1">Non-Steroidal Anti-Inflammatory (NSAID)</td><td colspan="1" rowspan="1">8.1% (81mg/mL)</td><td colspan="1" rowspan="1">8.1%</td></tr></table>

# i. Fresh versus Frozen Study:

A fresh versus frozen study was performed to support inclusion of frozen archived specimens in the clinical study. The study was designed to assess the effect on assay performance after one freeze/thaw cycle. Sixty individual preserved (Cary-Blair) and 60 unpreserved negative stool specimens were used to prepare samples. Culture isolates of Salmonella typhimurium, Shigella sonnei, $E$ . coli $( s t x 1 )$ and Campylobacter coli were prepared as a target mix and spiked into two aliquots from each negative stool specimen.

Organism Concentrations/Replicates per Targeted Analyte   

<table><tr><td rowspan=1 colspan=1>Target Level</td><td rowspan=1 colspan=1>Number ofSpecimens Tested per Matrix</td></tr><tr><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>4X LoD</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>600X</td><td rowspan=1 colspan=1>21</td></tr></table>

One aliquot of each sample was tested on the day of preparation and one aliquot was stored for a minimum of 18 hours at $- 7 0 \mathrm { { } ^ { \circ } C }$ . Study results demonstrated greater than $9 5 \%$ positive results for both fresh and frozen samples for both unpreserved and Cary-Blair preserved specimens.

Fresh vs. Frozen Results: Cary-Blair Specimens   

<table><tr><td rowspan=1 colspan=1>Fresh/Frozen Analyte</td><td rowspan=1 colspan=1>Analyt</td><td rowspan=1 colspan=1>NumberPositive/NumberTested</td><td rowspan=1 colspan=1>PercentPositive ofExpected</td><td rowspan=1 colspan=1>Mean CtValue</td></tr><tr><td rowspan=4 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>29.19</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30.79</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>27.83</td></tr><tr><td rowspan=1 colspan=1>E. coli stx1/2</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>30.35</td></tr><tr><td rowspan=4 colspan=1>Frozen</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>28.46</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.96</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.81</td></tr><tr><td rowspan=1 colspan=1>E. coli stx1/2</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.86</td></tr></table>

Fresh vs Frozen Results: Unpreserved Samples   

<table><tr><td rowspan=1 colspan=1>Fresh/Frozen Analyte</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NumberPositive/NumberTested</td><td rowspan=1 colspan=1>PercentPositive ofExpected</td><td rowspan=1 colspan=1>Mean CtValue</td></tr><tr><td rowspan=4 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>28.89</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>30.32</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>28.78</td></tr><tr><td rowspan=1 colspan=1>E. coli stx1/2</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>30.48</td></tr><tr><td rowspan=4 colspan=1>Frozen</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>28.37</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>29.66</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>28.36</td></tr><tr><td rowspan=1 colspan=1>E. coli stx1/2</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>29.90</td></tr></table>

In summary, results of the fresh versus frozen study demonstrate that one freeze/thaw cycle should not significantly affect the performance of the BD MAX Enteric Bacterial Panel as compared to testing of fresh specimens.

# j. Carryover / Cross-Contamination

A study was conducted to investigate within-run carryover and between-run carryover while processing specimens with high bacterial loads of Salmonella enterica (ATCC 13076), Shigella sonnei (ATCC 10523), Campylobacter jejuni (ATCC 29428) and Shigatoxin producing Escherichia coli (stx1 and stx2) (ENF 10513) in the BD MAX Enteric Bacterial Panel. Positive samples were prepared in SBT with an unpreserved stool matrix and an organism mix containing with all four targeted organisms, each present at ${ \sim } 1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/mL. The negative member did not contain any target analyte. Negative samples were prepared in SBT with negative unpreserved stool matrix. Testing included 16 runs containing alternating positive and negative samples tested on three BD MAX instruments.

Study results included 167 valid replicates for all negative samples tested of which one sample yielded false positive results for all four targets. The overall contamination carry-over/cross-contamination rate for each targeted organism was $0 . 6 \%$ (1/167; $9 5 \%$ CI, $0 . 1 1 \% - 3 . 3 1 \%$ ).

# k. Mixed Infection Study:

A mixed infection/competitive interference study was performed to evaluate the ability of the BD MAX Enteric Bacterial Panel to detect low positive results in the presence of other targets at high concentrations. Each of the targeted organisms was evaluated at low concentrations (1.5X LoD) when present in samples containing mixes of other targeted organisms at high concentrations $\mathrm { \dot { ( \ge 1 x 1 0 ^ { 6 } C F U / m L ) } }$ ). Samples were prepared with unpreserved stool matrix. A total of 24 replicates were tested for each of the sample mixes described in the table below.

Mixed infection Target Combinations   

<table><tr><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Sample Mix Low Positive Target (1.5X LoDin SBT)</td><td rowspan=1 colspan=1>High Positive Target Mix( Each organisms &gt;1x106 CFU/mL inSBT)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Salmonella typhimurium (ATCC14028)</td><td rowspan=1 colspan=1>Shigella sonnei (ATCC 15023),Campylobacter coli (ATCC 43134),E. coli O157:H7 [stx-1] (BD ENF10513)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Shigella sonnei (ATCC 15023)</td><td rowspan=1 colspan=1>Salmonella typhimurium (ATCC14028), Campylobacter coli (ATCC43134), E. coli O157:H7 [stx-1] (BDENF 10513)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Campylobacter coli (ATCC43134)</td><td rowspan=1 colspan=1>Salmonella typhimurium (ATCC14028), Shigella sonnei (ATCC15023), E. coli O157:H7 [stx-1] (BDENF 10513)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>E. coli, O157:H7 stx-1a (BD ENF10513)</td><td rowspan=1 colspan=1>Salmonella typhimurium (ATCC14028), Shigella sonnei (ATCC15023), Campylobacter coli (ATCC43134)</td></tr></table>

Study results showed no evidence of competitive inhibition in samples with low concentrations of each targeted organism when combined with high concentrations of other targeted organisms.

# l. Assay cut-off:

Assay cut-offs for the BD MAX Enteric Bacterial Panel were pre-determined in analytical verification experiments and then subsequently validated using data from the multi-site clinical study. The PCR metrics of EP (Endpoint Fluorescence), FDPH (First Derivative Peak Height), and Ct Value (Cycle Threshold) from the clinical study were graphically and statistically analyzed as compared to results from the reference method for each targeted analyte. ROC curve analysis was performed separately for each PCR metric demonstrating that use of the pre-determined cutoffs yielded optimal positive percent agreement for all analytes as compared to the reference method. The analysis did suggest however that a change in the Campylobacter spp. cutoff could improve the negative percent agreement for the assay. Given the importance of detecting these pathogens in stool specimens, it was deemed inappropriate to apply any change to the decision algorithm for Campylobacter spp. and the cutoff was left unchanged. In conclusion, results from the assay cutoff validation demonstrated that the initial cutoffs determined prior to the clinical studies are acceptable.

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison: N/A

3. Clinical Study:

Performance characteristics of the BD MAX Enteric Bacterial Panel were determined in a multi-site investigational study. The study involved a total of eight geographically diverse clinical centers where specimens were prospectively collected and tested as part of routine patient care, and the excess de-identified specimens were then tested with the BD MAX Enteric Bacterial Panel. Specimens were also collected at an additional four sites and shipped for testing at a central location. Specimens were obtained from pediatric and adult patients suspected of acute bacterial gastroenteritis, enteritis or colitis, for which stool culture had been ordered by a healthcare provider. Unpreserved or Cary-Blair preserved specimens were included. Stool specimens were required to have a soft or diarrheal consistency and only one specimen was enrolled per patient. Additional testing was performed with preselected retrospective (frozen) specimens known to be positive for analytes targeted by the BD MAX Enteric Bacterial Panel.

Reference testing for prospective (fresh) specimens was standard direct stool culture followed by biochemical tests to finalize identification of colonies suspected to be Salmonella, Shigella, and Campylobacter. The reference method for detection of Shiga toxin 1 and 2 included broth enrichment followed by enzyme immunoassay. For retrospective (frozen) specimens, historical results were confirmed using an alternate PCR assay followed by bi-directional sequencing in order to confirm the presence of target DNA.

A total of 3457 prospective specimens (2112 Cary-Blair preserved and 1345 unpreserved) and 785 retrospective specimens (464 Cary-Blair preserved and 321 unpreserved) were enrolled in the clinical evaluation. A total of 104 retrospective specimens for which the historical results were not confirmed by an alternate PCR and bi-directional sequencing were not included in performance calculations. The following table describes the number of compliant specimens enrolled by patient age and specimen type.

Compliant Specimens Tested by Patient Age   

<table><tr><td colspan="1" rowspan="1">AgeGroup</td><td colspan="1" rowspan="1">Cary-BlairPreserved</td><td colspan="1" rowspan="1">Unpreserved</td><td colspan="1" rowspan="1">Combined</td></tr><tr><td colspan="1" rowspan="1">&lt;1</td><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">153</td></tr><tr><td colspan="1" rowspan="1">1-4</td><td colspan="1" rowspan="1">302</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">430</td></tr><tr><td colspan="1" rowspan="1">5-12</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">479</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">Cary-BlairPreserved</td><td colspan="1" rowspan="1">Unpreserved</td><td colspan="1" rowspan="1">Combined</td></tr><tr><td colspan="1" rowspan="1">13-18</td><td colspan="1" rowspan="1">271</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">439</td></tr><tr><td colspan="1" rowspan="1">19-65</td><td colspan="1" rowspan="1">1222</td><td colspan="1" rowspan="1">799</td><td colspan="1" rowspan="1">2021</td></tr><tr><td colspan="1" rowspan="1">Over 65</td><td colspan="1" rowspan="1">388</td><td colspan="1" rowspan="1">249</td><td colspan="1" rowspan="1">637</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td></tr></table>

The following tables include the BD MAX Enteric Bacterial Panel clinical study results stratified by prospective (fresh) and retrospective (frozen) unpreserved specimens as well as prospective and retrospective Cary-Blair preserved specimens. Assay performance for each targeted analyte is calculated as compared to the reference method (RM) for prospective specimens and to the historical result (confirmed by alternate PCR and bi-directional sequencing) for retrospective specimens.

Campylobacter spp. – Assay Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BDMAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>23^{2$</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1751</td><td rowspan=1 colspan=1>1752</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1774</td><td rowspan=1 colspan=1>1800</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 96.2% (81.1%, 99.3%)NPA (95% CI): 98.7% (98.1%, 99.1%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>153</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>217</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 97% (89.6%, 99.2%)NPA (95% CI): 100% (97.5%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1185</td><td rowspan=1 colspan=1>1185</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1216</td><td rowspan=1 colspan=1>1238</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (85.1%, 100%)NPA (95% CI): 97.5% (96.4%, 98.2%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(rozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>221</td><td rowspan=1 colspan=1>223</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>290</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 97% (89.8%, 99.2%)NPA (95% CI): 99.1% (96.8%, 99.8%)</td></tr></table>

1 This specimen was also tested using an alternate PCR assay followed by bi-directional sequencing and gave a negative result.

Salmonella spp. – Assay Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BDMAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-BBlair</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1791</td><td rowspan=1 colspan=1>1794</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1808</td><td rowspan=1 colspan=1>1828</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 85% (64%, 94.8%)NPA (95% CI): 99.1% (98.5%, 99.4%)</td></tr><tr><td rowspan=3 colspan=1>Cary-BBlair</td><td rowspan=3 colspan=1>RetrospectiveFrozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>319</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 99.1% (94.8%, 99.8%)NPA (95% CI): 100% (98.2%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1202</td><td rowspan=1 colspan=1>1204</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1215</td><td rowspan=1 colspan=1>1239</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 91.7% (74.2%, 97.7%)NPA (95%CI):: 98.9% (98.2%, 99.4%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>299</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (94.1%, 100%)NPA (95% CI): 99.6% (97.7%, 99.9%)</td></tr></table>

1 These three (3) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing and gave a negative result. 2 These seventeen (17) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; eleven (11) of seventeen (17) gave a positive result. 3 These thirteen (13) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; eleven (11) of thirteen (13) gave a positive result.

Shigella spp. / EIEC – Overall Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOOrigin</td><td rowspan=2 colspan=1>BDMAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1804</td><td rowspan=1 colspan=1>1804</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1809</td><td rowspan=1 colspan=1>1828</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (83.2%, 100%)NPA (95% CI): 99.7% (99.4%, 99.9%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>188</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>PPA (95% CI): 98% (89.7%, 99.7%)NPA (95% CI): 100% (98%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1212</td><td rowspan=1 colspan=1>1212</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1219</td><td rowspan=1 colspan=1>1241</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>PPA (95% CI): 100% (85.1%, 100%)NPA (95% CI): 99.4% (98.8%, 99.7%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>RetrospectiveFrozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>264</td><td rowspan=1 colspan=1>264</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>264</td><td rowspan=1 colspan=1>305</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (91.4%, 100%)NPA (95% CI): 100% (98.6%, 100%)</td></tr></table>

1 These five (5) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; all five (5) specimens gave a positive result. 2 These seven (7) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; six (6) of seven (7) gave a positive result.

Shiga toxins (stx1/stx2) – Overall Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BDMAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Prospective(Fresh</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1768</td><td rowspan=1 colspan=1>1770</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1781</td><td rowspan=1 colspan=1>1789</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 75% (40.9%, 92.9%)NPA (95% CI): 99.3% (98.8%, 99.6%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>79</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (91.4%, 100%)NPA (95% CI): 100% (95.4%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>704</td><td rowspan=1 colspan=1>704</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>711</td><td rowspan=1 colspan=1>713</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (34.2%, 100%)NPA (95% CI): 99% (98%, 99.5%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (86.7%, 100%)NPA (95% CI): 100% (74.1%, 100%)</td></tr></table>

1 These two (2) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing and gave a negative result. 2 These thirteen (13) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; seven (7) of thirteen (13) gave a positive result. 3 These seven (7) specimens were also tested using an alternate PCR assay followed by bi-directional sequencing; three (3) of seven (7) gave a positive result.

# Performance of the BD MAX Enteric Bacterial Panel by species/toxin type

Species/toxin type identification was obtained for each positive specimen either from the culture and identification portion of the reference method testing, EIA result, or from sequencing performed for the confirmation of retrospective specimen historical results. While the BD MAX Enteric Bacterial Panel is designed to detect the species and toxin types described below, the panel does not report results to the species or toxin level. The following tables describe the assay performance stratified by species and Shiga-toxin type.

Campylobacter spp. – Assay performance stratified by species   

<table><tr><td colspan="3">Campylobacter spp.</td><td colspan="2">PPA</td></tr><tr><td>Specimen Type</td><td>Specimen Origin</td><td>Species</td><td>Estimate</td><td>95% CI</td></tr><tr><td rowspan="5">Cary-Blair Preserved</td><td>Prospective</td><td>jejunil</td><td>95.8% (23/24)</td><td>(79.8%, 99.3%)</td></tr><tr><td>(Fresh)</td><td>Untyped</td><td>100.0% (2/2)</td><td>(34.2%, 100.0%)</td></tr><tr><td>Retrospective</td><td>coli</td><td>100.0% (2/2)</td><td>(34.2%, 100.0%)</td></tr><tr><td>(Frozen)</td><td>jejuni</td><td>96.9% (62/64)</td><td>(89.3%, 99.1%)</td></tr><tr><td>Prospective</td><td>jejuni</td><td>100.0% (19/19)</td><td>(83.2%, 100.0%)</td></tr><tr><td rowspan="4">Unpreserved</td><td>(Fresh)</td><td>jejuni or coli</td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td></td><td>Untyped</td><td>100.0% (2/2)</td><td>(34.2%, 100.0%)</td></tr><tr><td></td><td>coli</td><td>100.0% (5/5)</td><td>(56.6%, 100.0%)</td></tr><tr><td>Retrospective</td><td>jejuni</td><td>96.8% (60/62)</td><td>(89.0%, 99.1%)</td></tr></table>

Of these specimens, one (1) prospective specimen was also tested using a validated PCR assay followed by bi-directional equencing and gave a negative result.

Shigella spp.-Assay performance stratified by species type   

<table><tr><td colspan="3">Shigella spp.</td><td colspan="2">PPA</td></tr><tr><td>Specimen Type</td><td>Specimen Origin</td><td>Species</td><td>Estimate</td><td>95% CI</td></tr><tr><td rowspan="4">Cary-Blair Preserved</td><td>Prospective</td><td>flexneri</td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td>(Fresh)</td><td>sonnei</td><td>100.0% (18/18)</td><td>(82.4%, 100.0%)</td></tr><tr><td>Retrospective (Frozen)</td><td>sonnei</td><td>98.0% (50/51)</td><td>(89.7%, 99.7%)</td></tr><tr><td>Prospective</td><td>flexneri</td><td>100.0% (2/2)</td><td>(34.2%, 100.0%)</td></tr><tr><td rowspan="4">Unpreserved</td><td>(Fresh)</td><td>sonnei</td><td>100.0% (20/20)</td><td>(83.9%, 100.0%)</td></tr><tr><td rowspan="2">Retrospective (Frozen)</td><td>flexneri</td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td>sonnei</td><td>100.0% (40/40)</td><td></td></tr><tr><td></td><td></td><td></td><td>(91.2%, 100.0%)</td></tr></table>

Stx1/stx2 - Assay performance stratified by toxin type   

<table><tr><td colspan="3">Shiga toxins</td><td colspan="2">PPA</td></tr><tr><td>Specimen Type</td><td>Specimen Origin</td><td>Toxin Type</td><td>Estimate</td><td>95% CI</td></tr><tr><td rowspan="4">Cary-Blair Preserved</td><td rowspan="4">Prospective (Fresh) Retrospective</td><td>stx1</td><td>100.0% (4/4)</td><td>(51.0%, 100.0%)</td></tr><tr><td>stx2</td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td>stx1 and stx21</td><td>33.3% (1/3)</td><td>(6.1%, 79.2%)</td></tr><tr><td>stx1</td><td>100.0% (28/28)</td><td>(87.9%, 100.0%)</td></tr><tr><td rowspan="7">Unpreserved</td><td>(Frozen)</td><td>stx2</td><td>100.0% (6/6)</td><td>(61.0%, 100.0%)</td></tr><tr><td rowspan="4">Prospective (Fresh)</td><td>stx1 and stx2</td><td>100.0% (7/7)</td><td>(64.6%, 1 , 100.0%)</td></tr><tr><td>stx1</td><td></td><td></td></tr><tr><td></td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td>stx1 and stx2</td><td>100.0% (1/1)</td><td>(20.7%, 100.0%)</td></tr><tr><td rowspan="3">Retrospective (Frozen)</td><td>stx1</td><td>100.0% (5/5)</td><td>(56.6%, 100.0%)</td></tr><tr><td>stx2</td><td>100.0% (6/6)</td><td>(61.0%, 100.0%)</td></tr><tr><td>stx1 and stx2</td><td>100.0% (14/14)</td><td>(78.5%, 100.0%)</td></tr></table>

Two (2) prospective specimens were also tested using a validated PCR assay followed by bi-directional sequencing and gave a negative result.

# Co-infections Observed in the Prospective Clinical Study

The table below shows the co-infections detected by the BD MAX Enteric Bacterial Panel during the prospective segment of the clinical study. There were no co-infections detected by the reference method in the prospective study.

Co-infections observed during prospective clinical study   

<table><tr><td rowspan=1 colspan=2>Distinct Co-infection CombinationsDetected by BD MAX Enteric BacterialAssay</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-Infections</td><td rowspan=2 colspan=1>Discrepant Analyte(s)1</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>stx1/2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>stx1/2 2</td></tr><tr><td rowspan=1 colspan=1>stx1/2</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>stx1/2^3</td></tr><tr><td rowspan=1 colspan=1>stx1/2</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>stx1/2 (2) and Salmonella (1)4</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Campylobacter (2), Salmonella (1)5</td></tr></table>

1 A discrepant co-infection or discrepant analyte was defined as one that was detected by the BD MAX assay but not detected by the reference method.   
2 One (1) discrepant $s t x 1 / 2$ was investigated using an alternate method; bi-directional sequence analysis identified the analyte in $_ { 0 / 1 }$ cases.   
3 One (1) discrepant $s t x 1 / 2$ was investigated using an alternate method; bi-directional sequence analysis identified the analyte in 1/1 cases.   
4 Two (2) discrepant $s t x 1 / 2$ were investigated using an alternate method; bi-directional sequence analysis identified the analyte in 0/2 cases. One (1) discrepant Salmonella was investigated using an alternate method; bi-directional sequence analysis identified the analyte in $1 / 1$ cases.   
5 Two (2) discrepant Campylobacter were investigated using an alternate method; bi-directional sequence analysis identified the analyte in 0/2 cases. One (1) discrepant Salmonella was investigated using an alternate method; bi-directional sequence analysis identified the analyte in 0/1 cases.

Of the 3183 prospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $4 . 0 \%$ of the Cary-Blair preserved and $7 . 8 \%$ of the unpreserved specimens were initially reported as Unresolved. Following a valid repeat test, $0 . 1 \%$ of the Cary-Blair preserved and $1 . 0 \%$ of the unpreserved specimens remained Unresolved. Of the 783

retrospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $2 . 2 \%$ of the Cary-Blair preserved and $4 . 1 \%$ of the unpreserved specimens were initially reported as Unresolved. Following a valid repeat test, $0 . 2 \%$ of the Cary-Blair preserved and $0 . 6 \%$ of the unpreserved specimens remained Unresolved. Unresolved rates seen in the clinical study are shown in the following table.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Unresolved Rates</td><td rowspan=1 colspan=2>Unresolved Rates After Repeat</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Specimen Origin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Cary-Blair</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>4.0% (77/1905)</td><td rowspan=1 colspan=1>(3.2%, 5.0%)</td><td rowspan=1 colspan=1>0.1% (2/1897)</td><td rowspan=1 colspan=1>(0.0%, 0.4%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>2.2% (10/464)</td><td rowspan=1 colspan=1>(1.2%, 3.9%)</td><td rowspan=1 colspan=1>0.2% (1/463)</td><td rowspan=1 colspan=1>(0.0%, 1.2%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>7.8% (100/1278)</td><td rowspan=1 colspan=1>(6.5%, 9.4%)</td><td rowspan=1 colspan=1>1.0% (13/1251)</td><td rowspan=1 colspan=1>(0.6%, 1.8%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>4.1% (13/319)</td><td rowspan=1 colspan=1>(2.4%, 6.8%)</td><td rowspan=1 colspan=1>0.6% (2/317)</td><td rowspan=1 colspan=1>(0.2%, 2.3%)</td></tr></table>

Of the 3183 prospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $1 . 7 \%$ of the Cary-Blair preserved and $1 . 6 \%$ of the unpreserved specimens were initially reported as Indeterminate. Following a valid repeat test, $0 \%$ of the Cary-Blair preserved and $0 . 2 \%$ of the unpreserved specimens remained Indeterminate. Of the 783 retrospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $1 . 5 \%$ of the Cary-Blair preserved and $1 . 9 \%$ of the unpreserved specimens were initially reported as Indeterminate. Following a valid repeat test, $0 \%$ of the Cary-Blair preserved and $0 \%$ of the unpreserved specimens remained Indeterminate. Indeterminate rates seen in the clinical study are shown in the following table.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Indeterminate Rates</td><td rowspan=1 colspan=2>Final Indeterminate Rates AfterRepeat</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Specimen Origin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Cary-Blair</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>1.7% (33/1905)</td><td rowspan=1 colspan=1>(1.2%, 2.4%)</td><td rowspan=1 colspan=1>0.0% (0/1897)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>1.5% (7/464)</td><td rowspan=1 colspan=1>(0.7%, 3.1%)</td><td rowspan=1 colspan=1>0.0% (0/463)</td><td rowspan=1 colspan=1>(0.0%, 0.8%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>1.6% (20/1278)</td><td rowspan=1 colspan=1>(1.0%, 2.4%)</td><td rowspan=1 colspan=1>0.2% (2/1251)</td><td rowspan=1 colspan=1>(0.0%, 0.6%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>1.9% (6/319)</td><td rowspan=1 colspan=1>(0.9%, 4.0%)</td><td rowspan=1 colspan=1>0.0% (0/317)</td><td rowspan=1 colspan=1>(0.0%, 1.2%)</td></tr></table>

Of the 3183 prospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $1 . 3 \%$ of the Cary-Blair preserved and $2 . 0 \%$ of the unpreserved specimens initially reported as Incomplete. Following a valid repeat test, $0 \%$ of the Cary-Blair preserved and $0 \%$ of the unpreserved specimens remained Incomplete. Of the 783 retrospective specimens initially evaluated with the BD MAX Enteric Bacterial Panel, $1 . 3 \%$ of the Cary-Blair preserved and $0 \%$ of the unpreserved specimens initially reported as Unresolved. Following a valid repeat test, $0 \%$ of the Cary-Blair preserved specimens remained Incomplete. Incomplete rates seen in the clinical study are shown in the following table.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Incomplete Rates</td><td rowspan=1 colspan=2>Final Incomplete Rates AfterRepeat</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Specimen Origin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Cary-Blair</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>1.3% (24/1905)</td><td rowspan=1 colspan=1>(0.8%, 1.9%)</td><td rowspan=1 colspan=1>0.0% (0/1897)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>1.3% (6/464)</td><td rowspan=1 colspan=1>(0.6%, 2.8%)</td><td rowspan=1 colspan=1>0.0% (0/463)</td><td rowspan=1 colspan=1>(0.0%, 0.8%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>2.0% (26/1278)</td><td rowspan=1 colspan=1>(1.4%, 3.0%)</td><td rowspan=1 colspan=1>0.0% (0/1251)</td><td rowspan=1 colspan=1>(0.0%, 0.3%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>0.0% (0/319)</td><td rowspan=1 colspan=1>(0.0%, 1.2%)</td><td rowspan=1 colspan=1>0.0% (0/317)</td><td rowspan=1 colspan=1>(0.0%, 1.2%)</td></tr></table>

4. Clinical cut-off:

N/A

5. Expected values/Reference range:

In the BD MAX Enteric Bacterial Panel clinical study, reportable results from compliant specimens were obtained from eight geographically diverse sites and compared to the reference methods. The study population was grouped based on specimen type. The number and percentage of positive cases by target, as determined by the BD MAX Enteric Bacterial Panel during the prospective segment of the clinical trial, are presented in the following table.

Prevalence Values Observed during the Clinical Trial   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="4" rowspan="1">Prevalence</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Shiga toxins</td></tr><tr><td colspan="1" rowspan="4">Cary-BlairPreserved</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0% (0/186)</td><td colspan="1" rowspan="1">0.0% (0/186)</td><td colspan="1" rowspan="1">1.1% (2/188)</td><td colspan="1" rowspan="1">0.0% (0/185)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8% (3/377)</td><td colspan="1" rowspan="1">0.3% (1/377)</td><td colspan="1" rowspan="1">1.6% (6/368)</td><td colspan="1" rowspan="1">0.8% (3/391)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9% (5/548)</td><td colspan="1" rowspan="1">0.2% (1/548)</td><td colspan="1" rowspan="1">0.8% (4/528)</td><td colspan="1" rowspan="1">0.2% (1/551)</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3.9% (6/152)</td><td colspan="1" rowspan="1">11.2% (17/152)</td><td colspan="1" rowspan="1">2.0% (3/152)</td><td colspan="1" rowspan="1">0.0% (0/135)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.3% (1/339)</td><td colspan="1" rowspan="1">0.0% (0/339)</td><td colspan="1" rowspan="1">1.5% (5/340)</td><td colspan="1" rowspan="1">0.3% (1/320)</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.4% (6/431)</td><td colspan="1" rowspan="1">0.0% (0/431)</td><td colspan="1" rowspan="1">1.9% (8/431)</td><td colspan="1" rowspan="1">0.7% (3/411)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1.0% (21/2033)</td><td colspan="1" rowspan="1">0.9% (19/2033)</td><td colspan="1" rowspan="1">1.4% (28/2007)</td><td colspan="1" rowspan="1">0.4% (8/1993)</td></tr><tr><td colspan="1" rowspan="5">Unpreserved</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.6% (6/376)</td><td colspan="1" rowspan="1">0.3% (1/376)</td><td colspan="1" rowspan="1">0.8% (3/376)</td><td colspan="1" rowspan="1">0.0% (0/176)</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.6% (5/305)</td><td colspan="1" rowspan="1">0.0% (0/305)</td><td colspan="1" rowspan="1">2.0% (6/304)</td><td colspan="1" rowspan="1">0.0% (0/229)</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.4% (4/284)</td><td colspan="1" rowspan="1">0.0% (0/284)</td><td colspan="1" rowspan="1">1.1% (3/284)</td><td colspan="1" rowspan="1">0.4% (1/265)</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2.9% (9/314)</td><td colspan="1" rowspan="1">6.7% (21/314)</td><td colspan="1" rowspan="1">3.5% (11/314)</td><td colspan="1" rowspan="1">0.4% (1/266)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1.9% (24/1279)</td><td colspan="1" rowspan="1">1.7% (22/1279)</td><td colspan="1" rowspan="1">1.8% (23/1278)</td><td colspan="1" rowspan="1">0.2% (2/936)</td></tr></table>

# N. Instrument Name:

BD MAX System

# O. System Descriptions:

1. Modes of Operation:

The BD MAX System fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification: Specimens are labeled with a barcode.

4. Specimen Sampling and Handling:

A disposable inoculating loop is used to place $1 0 ~ \mu \mu 1$ of the unpreserved or Cary-Blair stool specimen into a SBT which is then vortexed and placed onto the system.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control: See Section M.1c above.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.