# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K140198   
B. Purpose for Submission: Clearance of New Device

C. Measurand: Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2)

E. Applicant: Intelligent Medical Devices, Inc.

F. Proprietary and Established Names: IMDx HSV-1/2 for Abbott $m 2 0 0 0$

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay is an in vitro diagnostic test for the direct, qualitative detection and differentiation of herpes simplex virus type 1 (HSV-1) and type 2 (HSV-2) DNA from male and female skin lesions from anogenital or oral sites. The test is intended for use as an aid in the diagnosis of HSV infection in symptomatic patients. The assay is intended to be run on the Abbott $m 2 0 0 0$ instrument system.

Warning: The IMDx HSV-1/2 for Abbott m2000 assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay is not intended for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

Abbott® $m 2 0 0 0 ^ { \mathrm { T M } }$ instrument system which is comprised of the Abbott $m 2 0 0 0 s p$ for sample preparation and reagent mixing and the Abbott $m 2 0 0 0 r t$ for amplification reaction and detection.

# I. Device Description:

The IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay uses nucleic acid extraction and purification technology, performed on the Abbott $m 2 0 0 0$ Sample Preparation System $( m 2 0 0 0 { \mathrm { s p } } )$ , combined with real-time polymerase chain reaction (PCR), performed on the Abbott PCR analyzer $( m 2 0 0 0 \mathrm { r t } )$ , to generate and detect amplified products from HSV-1 and HSV-2 DNA that is isolated from clinical specimens. The presence of HSV-1 and/or HSV-2 target DNA is indicated by the fluorescent signal generated through the use of fluorescently labeled oligonucleotide probes on the Abbott $m 2 0 0 0 r t$ instrument. The probes do not generate a signal unless they are specifically bound to the amplified product. The amplification cycle at which fluorescent signal is detected by the Abbott $m 2 0 0 0 r t$ is inversely proportional to the HSV-1 and/or HSV-2 DNA target concentration present in the original sample.

The IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay consists of two reagent kits packaged together:

• IMDx HSV-1/2 for Abbott $m 2 0 0 0$ Amplification Reagent Kit IMDx HSV-1/2 for Abbott m2000 Control Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s):

2. Predicate 510(k) number(s): K100336

3. Comparison with predicate:

K. Standard/Guidance Document Referenced (if applicable): N/A   

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>IMDx HSV-1/2 for Abbott m2000(New Device)</td><td rowspan=1 colspan=1>Eragen Biosciences MultiCode®-RTxHerpes Simplex Virus 1 &amp; 2 Kit(K100336)(Predicate Device)</td></tr><tr><td rowspan=1 colspan=1>Intended use</td><td rowspan=1 colspan=1>The IMDx HSV-1/2 for Abbott m2000assay is an in vitro diagnostic test for thedirect,qualitative   detection  anddifferentiation of herpes simplex virustype 1 (HSV-1) and type 2 (HSV-2)DNA from male and female skin lesionsfrom anogenital or oral sites. The test isintended for use as an aid in thediagnosis of HSV infection insymptomatic patients. The assayisintended to be run on the Abbott m2000instrument system.Warning: The IMDx HSV-1/2 forAbbott m2000 assay is not FDA clearedfor use with cerebrospinal fluid (CSF).The assay is not intended for prenatalscreening.</td><td rowspan=1 colspan=1>The MultiCode®-RTx Herpes SimplexVirus 1 &amp; 2 Kit is a polymerase chainreaction (PCR)-based qualitative in vitrodiagnostic test for the detection and typingof herpes simplex virus (HSV1&amp;2) DNAin vaginal lesions. It is indicated for use inthe detection and typing of HSV-1 orHSV-2 in vaginal lesion swab specimensfrom symptomatic female patients as anaid in the diagnosis of genital herpesinfection.Warning: The device is not FDA cleared forthe use with cerebrospinal fluid (CSF) or anylesions other than vaginal. The assay is notintended to be used for male penilespecimens, for prenatal screening, or femalesunder the age of 18 years.</td></tr><tr><td rowspan=1 colspan=1>Test Principle</td><td rowspan=1 colspan=1>Real-time PCR DNA amplification</td><td rowspan=1 colspan=1>Real-time PCR DNA amplification</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Qualitative detection and differentiationof HSV-1 and HSV-2 DNA</td><td rowspan=1 colspan=1>Qualitative detection and differentiation ofHSV-1 and HSV-2 DNA</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>IMDx HSV-1/2 for Abbott m2000(New Device)</td><td rowspan=1 colspan=1>Eragen Biosciences MultiCode®-RTxHerpes Simplex Virus 1 &amp; 2 Kit(K100336)</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>Sample extraction and real-time PCRamplification/detection using the Abbottm2000 system.</td><td rowspan=1 colspan=1>Sample extraction using Roche MagNA PureSystem or bioMérieux NucliSENS system.Real-time PCR amplification/ detectionusing the Roche LightCycler 1.2 instrument.</td></tr><tr><td rowspan=1 colspan=1>DetectionMethod</td><td rowspan=1 colspan=1>Double-labeled (fluorophore andquencher) hydrolysis probes. Measuresincrease in assay fluorescence with eachPCR cycle..</td><td rowspan=1 colspan=1>Pairs fluorescent-labeled primers withquencher labeled nucleotides. Measuresdecrease in assay fluorescence with eachPCR cycle.</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Male and female skin lesions fromanogenital or oral sites</td><td rowspan=1 colspan=1>Female vaginal lesions</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td></tr></table>

# L. Test Principle:

The IMDx HSV-1/2 for Abbott m2000 assay enables detection and differentiation of HSV-1 and HSV-2 DNA from clinical specimens and Internal Control through the following workflow:

# Sample Preparation

HSV-1 and HSV-2 DNA is extracted from the clinical specimens by the Abbott $m 2 0 0 0 s p$ , an automated sample preparation system designed to use magnetic microparticles for the purification of nucleic acids. The Abbott mSample Preparation SystemDNA $( 4 \mathrm { ~ x ~ } 2 4 \mathrm { ~ P r e p s } )$ reagents lyse the HSV and the released DNA is then captured on magnetic microparticles that are subsequently washed to remove unbound sample components. The bound DNA is eluted and transferred to an Abbott 96-Deep-Well plate. The DNA is then ready for amplification. The Internal Control is introduced into each specimen or control prior to the sample preparation procedure and is processed along with the controls and specimens.

# Reagent Preparation and Reaction Plate Assembly

The Abbott $m 2 0 0 0 s p$ combines the IMDx HSV-1/2 for Abbott m2000 Amplification Reagent Pack components (IMDx HSV-1/2 for Abbott m2000 Amplification Reagent and IMDx PCR Reagent-A) to prepare the Master Mix. The Abbott $m 2 0 0 0 s p$ dispenses the resulting Master Mix into the Abbott 96-Well Optical Reaction Plate along with aliquots of the nucleic acid samples prepared by the Abbott $m 2 0 0 0 s p$ . After manual application of the Abbott Optical Adhesive Cover, the plate is ready for transfer to the Abbott $m 2 0 0 0 r t$ . Up to 96 tests can be performed during each run, inclusive of a positive and negative control.

# Amplification

During the amplification/detection reaction on the Abbott $m 2 0 0 0 r t$ instrument, the target DNA is amplified by DNA polymerase in the presence of deoxynucleotide triphosphates (dNTPs) and magnesium. The IMDx HSV-1/2 for Abbott $m 2 0 0 0$ Amplification Reagent contains specific amplification primers for the HSV-1 Glycoprotein D gene, HSV-2 UL30 gene, and the IMDx Internal Control-B targets. During PCR amplification, high temperature is used to separate the strands of double-stranded DNA. When the reaction is cooled to a temperature at which DNA annealing can again occur, the analyte-specific, single-stranded DNA oligonucleotide primers bind to the analyte DNA. The primers are extended by DNA polymerase, thereby making an exact copy of a short stretch of the analyte DNA target region. The DNA polymerase enzyme is a thermophilic enzyme that has been modified in its active site by a molecule that renders it inactive. When the enzyme is heated prior to the initiation of PCR, the inhibitory molecule dissociates from the enzyme allowing it to regain its activity.

During each round of thermal cycling, amplification products dissociate to single strands at high temperature, allowing primer annealing and extension as the temperature is lowered. Exponential amplification of the target is achieved through repeated cycling between higher and lower temperatures. Amplification of the HSV-1 Glycoprotein D gene, HSV-2 UL30 gene, and the IMDx Internal Control-B targets takes place simultaneously in the same reaction.

# Detection

During each round of PCR amplification, the fluorescent probes anneal to the amplified target DNA, if present. The probes are labeled with different fluorescent molecules allowing the HSV-1 Glycoprotein D gene, HSV-2 UL30 gene, and IMDx Internal Control-B targets to be distinguished from each other. The probes are single-stranded, linear DNA oligonucleotides modified with a fluorescent moiety covalently linked to one end of the probe and a quenching moiety linked to the other end. When the probe binds to its complementary sequence of the target during amplification, the fluorophore is released, allowing fluorescent emission and detection.

Since this fluorescence occurs during every cycle, the PCR reaction can be read in real-time. The amplification cycle at which fluorescent signal is detected by the Abbott $m 2 0 0 0 r t$ is inversely proportional to the HSV-1 Glycoprotein D gene and HSV-2 UL30 gene DNA target concentration present in the original sample.

# Result Calling:

HSV-1 and HSV-2: Results are reported independently for HSV-1, HSV-2 and IMDx Internal Control-B.   
Positive and Negative Controls: If the IMDx HSV-1/2 for Abbott m2000 Positive Control and IMDx Negative Control-B are outside of predetermined ranges, an error code is generated and no results for the plate are reported.   
Internal Control: If the IMDx Internal Control-B is outside its pre-determined range, an IC flag is generated. If no HSV-1 and HSV-2 targets are detected in a sample where the IMDx Internal Control-B is outside its predetermined range, an error message is generated, the sample is invalidated, and no HSV-1 or HSV-2 target results are reported.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=5>Result Reported</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>InternalControl</td><td rowspan=1 colspan=1>Detected/NotDetected</td><td rowspan=1 colspan=1>Detected/NotDetected</td><td rowspan=1 colspan=1>Detected/NotDetected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>HSV-1 DNADetected</td><td rowspan=1 colspan=1>HSV-2 DNADetected</td><td rowspan=1 colspan=1>HSV-1 andHSV-2 DNAdetected</td><td rowspan=1 colspan=1>HSV-1 andHSV-2 DNAnot detected</td><td rowspan=1 colspan=1>Invalid;No resultreported</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Reproducibility:

The reproducibility of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay was evaluated at three sites using a panel consisting of seven members. The panel members were formulated with a single target present (HSV-1 MacIntyre strain or HSV-2 MS strain) at three concentrations: 2-3 X LoD (Positive), 1 X LoD (Low Positive), and ${ < } 1 \mathrm { ~ X ~ }$ LoD (High Negative). A true negative sample, where no HSV was added, was also prepared using M4RT viral transport medium. Each panel member was tested in replicates of three, for five days, at three study sites. Testing at each site was performed by two operators and each operator ran the panel once a day. The study was conducted using one instrument system (Abbott $m 2 0 0 0 s p$ and Abbott $m 2 0 0 0 r t )$ and one reagent lot of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay at each site. The results from the reproducibility study for the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay are presented in the table below.

Reproducibility   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All 3 Sites</td></tr><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>Avg.CN*(%CV)</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>Avg.CN(%CV)</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>Avg.CN(%CV)</td><td rowspan=1 colspan=1>%Agreement(5% CI)</td><td rowspan=1 colspan=1>Avg.CN(%CV)</td></tr><tr><td rowspan=1 colspan=1>HSV-1Positive</td><td rowspan=1 colspan=1>2-3XLoD</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.01(1.3%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.12(1.1%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>36.67(1.1%)</td><td rowspan=1 colspan=1>100%(100 -100%)</td><td rowspan=1 colspan=1>36.93(1.3%)</td></tr><tr><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>1XLoD</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>38.50(1.6%)</td><td rowspan=1 colspan=1>96.67%(29/30)</td><td rowspan=1 colspan=1>38.63(2.5%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>38.13(1.5%)</td><td rowspan=1 colspan=1>98.89%(94.11 -100%)</td><td rowspan=1 colspan=1>38.42(1.9%)</td></tr><tr><td rowspan=1 colspan=1>HSV-1HighNegative</td><td rowspan=1 colspan=1>&lt;1XLoD</td><td rowspan=1 colspan=1>53.33%(16/30)</td><td rowspan=1 colspan=1>39.79(1.5%)</td><td rowspan=1 colspan=1>50.00%(15/30)</td><td rowspan=1 colspan=1>39.91(2.0%)</td><td rowspan=1 colspan=1>36.67%(11/30)</td><td rowspan=1 colspan=1>39.75(2.2%)</td><td rowspan=1 colspan=1>46.67%(24.77 -68.57%)</td><td rowspan=1 colspan=1>39.81(1.9%)</td></tr><tr><td rowspan=1 colspan=1>HSV-2Positive</td><td rowspan=1 colspan=1>2-3XLoD</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.70(1.5%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.87(1.0%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.61(1.1%)</td><td rowspan=1 colspan=1>100%(100 -100%)</td><td rowspan=1 colspan=1>37.73(1.2%)</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>1XLoD</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>39.26(1.9%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>39.52(2.0%)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>39.05(1.5%)</td><td rowspan=1 colspan=1>100%(100 -100%)</td><td rowspan=1 colspan=1>39.28(1.9%)</td></tr><tr><td rowspan=1 colspan=1>HSV-2HighNegative</td><td rowspan=1 colspan=1>&lt; 1XLoD</td><td rowspan=1 colspan=1>53.33%(16/30)</td><td rowspan=1 colspan=1>41.45(2.0%)</td><td rowspan=1 colspan=1>50.00%(15/30)</td><td rowspan=1 colspan=1>41.95(2.5%)</td><td rowspan=1 colspan=1>63.33%(19/30)</td><td rowspan=1 colspan=1>41.69(3.3%)</td><td rowspan=1 colspan=1>55.56%(38.32 -72.79%)</td><td rowspan=1 colspan=1>41.70(2.6%)</td></tr><tr><td rowspan=1 colspan=1>HSVNegative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(100 -100%</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\*CN Cycle Number

# b. Precision (Within-Laboratory Repeatability)

The repeatability of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay was evaluated using the same panel described in the reproducibility study above. The seven member panel was tested twice a day for a total of twelve days. Panel members were tested in replicates of three for each run (for a total of 504 data points for the 24 runs). The entire study was conducted by one trained technician using one instrument pair (Abbott $m 2 0 0 0 s p$ and Abbott $m 2 0 0 0 r t \rrangle$ ) and one reagent lot of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. Results of the repeatability study for the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay performed at one sites are presented in the table below.

Precision (Within-Laboratory Repeatability)   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Agreement withexpected results</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td><td rowspan=1 colspan=1>Avg. CN</td><td rowspan=1 colspan=1>SD CN</td><td rowspan=1 colspan=1>Avg. CN(%CV)</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Positive</td><td rowspan=1 colspan=1>2-3 X LoD</td><td rowspan=1 colspan=1>100.00% (72/72)</td><td rowspan=1 colspan=1>100.00% - 100.00%</td><td rowspan=1 colspan=1>36.86</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.19%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 LowPositive</td><td rowspan=1 colspan=1>1 X LoD</td><td rowspan=1 colspan=1>100.00% (72/72)</td><td rowspan=1 colspan=1>100.00% - 100.00%</td><td rowspan=1 colspan=1>38.35</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.62%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 HighNegative</td><td rowspan=1 colspan=1>&lt;1 X LoD</td><td rowspan=1 colspan=1>44.440% (32/72)</td><td rowspan=1 colspan=1>33.54% - 55.91%</td><td rowspan=1 colspan=1>40.00</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.27%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Positive</td><td rowspan=1 colspan=1>2-3 X LoD</td><td rowspan=1 colspan=1>100.00% (72/72)</td><td rowspan=1 colspan=1>100.00% - 100.00%</td><td rowspan=1 colspan=1>37.58</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.66%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 LowPositive</td><td rowspan=1 colspan=1>1 X LoD</td><td rowspan=1 colspan=1>100.00% (72/72)</td><td rowspan=1 colspan=1>100.00% - 100.00%</td><td rowspan=1 colspan=1>39.16</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.52%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 HighNegative</td><td rowspan=1 colspan=1>&lt;1 X LoD</td><td rowspan=1 colspan=1>34.72% (25/72)</td><td rowspan=1 colspan=1>24.75% - 46.24%</td><td rowspan=1 colspan=1>41.48</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>3.42%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.00% (72/72)</td><td rowspan=1 colspan=1>100.00% - 100.00%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

c. Linearity/assay reportable range: N/A d. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Control-B

The Internal Control-B consists of synthetic plasmid DNA, unrelated to HSV-1 and HSV-2. An internal control is introduced into each specimen during sample preparation to serve as an internal control. The internal control is amplified in the same reaction as the HSV-1 and HSV-2 DNA targets, and serves to demonstrate that the sample preparation and amplification processes have proceeded correctly for each sample. The internal control primer and probe are pre-mixed in the Amplification Reagent.

# External Assay Controls

The positive control consists of a preparation of intact, inactivated HSV-1 and HSV-2 and is run as a separate control to demonstrate that the HSV-1/2 PCR reagents are functional. In addition, the positive control functions as a process control to demonstrate that sample preparation has proceeded correctly during the run. A negative control consisting of M4RT viral transport medium is included in each run to independently verify the absence of contaminating target material in assay reagents.

# e. Detection limit:

A Limit of Detection (LoD) study was performed to evaluate the analytical sensitivity of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay using two representative strains of HSV-1 (McIntyre & Clinical Isolate 1) and two representative strains of HSV-2 (MS & Clinical Isolate 1). To narrow the range for LoD analysis, a series of six 10-fold dilutions of virus in M4RT viral transport medium was tested with the IMDx HSV1/2 for Abbott $m 2 0 0 0$ assay in replicates of six. From this preliminary study, a series of six 2-fold dilutions of the same strains of HSV-1 (McIntyre & Clinical Isolate 1) and HSV-2 (MS & Clinical Isolate 1) was tested with three kit lots with 20 replicates per dilution for each kit lot. The results from the three kit lots were combined to provide a total of 60 results for each level of an HSV strain and each LoD was determined using probit analysis. The final LoDs are presented in the table below.

Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Limitof Detection(95%CI)</td></tr><tr><td rowspan=1 colspan=1>HSV-1 MacIntyre</td><td rowspan=1 colspan=1>13.47 TCID50/mL(10.5220.11)</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Clinical Isolate 1</td><td rowspan=1 colspan=1>7.63 TCID50/mL(5.89 - 10.90)</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>0.68 TCID50/mL(0.52  0.97)</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Clinical Isolate 1</td><td rowspan=1 colspan=1>219.41 TCID50/mL(178.45  305.37)</td></tr></table>

Analytical Reactivity: In addition, forty (40) clinical isolates $2 0 \mathrm { H S V - 1 }$ and $2 0 \mathrm { H S V } .$ 2) were tested for reactivity with the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. The titered stocks of frozen isolates were obtained from the supplier. Each isolate was diluted to $3 ~ \mathrm { X } \ \mathrm { L o D }$ in M4RT viral transport medium and was tested in triplicate. All strains were detected by the assay, demonstrating that the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay can detect a broad range of both HSV-1 and HSV-2 isolates.

# f. Analytical specificity:

A study was performed to evaluate the performance of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay in the presence of fifty (50) microorganisms and human DNA that might be found in anogenital or oral skin lesion specimens. The panel members were obtained from suppliers as purified genomic DNA (GD) or quantified cultures (QC), or prepared in house (IHC) by growing each organism and quantifying the culture. Bacteria were tested at $1 0 ^ { 6 } \mathrm { c f u / m l }$ or higher for bacteria and $\mathrm { i } 0 ^ { 5 } \mathrm { p f u / m l }$ or higher for viruses. For bacteria that were difficult to obtain or grow, purified DNA was used in the place of the intact microorganism, and tested at a concentration of ${ \geq } 1 ~ \times ~ 1 0 ^ { 6 }$ genome copies/mL. Human DNA was tested at a concentration of $1 \times 1 0 ^ { 5 }$ genome copies/mL. All samples were prepared by diluting microorganisms or DNA into M4RT viral transport medium prior to testing for cross-reactivity. No strains tested were positive for HSV-1 or HSV-2 using the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. Similarly, no cross-reactivity was observed with human DNA.

To assess microbial interference, each test microorganism from the panel was added to a sample tube containing one of four strains of HSV (HSV-1 MacIntyre, HSV-1 Clinical Isolate 1, HSV-2 MS, or HSV-2 Clinical Isolate 1) at $2 { - } 3 \ \mathrm { X }$ LoD in M4RT viral transport medium. Each potentially interfering microorganism was tested in three (3) replicates at the same levels used in the cross-reactivity study described above. The IMDx HSV-1/2 for Abbott $\mathrm { m } 2 0 0 0$ assay was challenged with microorganisms and the results of each test run were assessed for a change in result call from detected to not detected for the HSV-1 or HSV-2 target. No evidence of microbial interference was observed for any of the 50 test microorganisms included in the analysis. Similarly, no evidence of interference was observed for human DNA.

Cross Reactivity & Microbial Panel   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus var. anitratus(IHC)</td><td rowspan=1 colspan=1>Lactobacillus acidophilus (QC)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffii (QC)</td><td rowspan=1 colspan=1>Mobiluncus curtisii, V125 [DSM 2711](QC)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 2 (QC)</td><td rowspan=1 colspan=1>Mobiluncus mulieris, BV 64-5 (QC)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis (QC)</td><td rowspan=1 colspan=1>Moraxella catarrhalis (QC)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (QC)</td><td rowspan=1 colspan=1>Mycoplasma hominis, PG21 (GD)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata (QC)</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae [GD]</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii (QC)</td><td rowspan=1 colspan=1>Neisseria meningitidis (QC)</td></tr><tr><td rowspan=1 colspan=1>Candida krusei (QC)</td><td rowspan=1 colspan=1>Prevotella melaninogenica (QC)</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae (IHC)</td><td rowspan=1 colspan=1>Rubella virus (QC)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis (QC)</td><td rowspan=1 colspan=1>Simian Virus type 40 (SV40) PML-1 (EK)(GD)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis (QC)</td><td rowspan=1 colspan=1>Staphylococcus agalactiae, Serotype III(IHC)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis, LGV-II434(QC/GD)</td><td rowspan=1 colspan=1>Staphylococcus agalactiae, Serotype V(IHC)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis, UW-3/Cx (GD)</td><td rowspan=1 colspan=1>Staphylococcus aureus (MRSA) (IHC)</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus, AD-169 (QC)</td><td rowspan=1 colspan=1>Staphylococcus aureus (IHC)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (QC)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis (IHC)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Type 71 (QC)</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus (IHC)</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr Virus (QC)</td><td rowspan=1 colspan=1>Streptococcus mitis, clinical isolate (QC)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (QC)</td><td rowspan=1 colspan=1>Streptococcus mutans (QC)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum, VPI 4355 (IHC)</td><td rowspan=1 colspan=1>Streptococcus pneumoniae (QC/GD)</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis (QC)</td><td rowspan=1 colspan=1>Streptococcus pyogenes (QC)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus ducreyi, Class I (GD)</td><td rowspan=1 colspan=1>Streptococcus salivarius (IHC)</td></tr><tr><td rowspan=1 colspan=1>Human Herpesvirus 6 (HHV-6) [QC]</td><td rowspan=1 colspan=1>Toxoplasma gondii (QC)</td></tr><tr><td rowspan=1 colspan=1>Human Herpesvirus 7 (HHV-7) (QC)</td><td rowspan=1 colspan=1>Trichomonas vaginalis Donne (GD)</td></tr><tr><td rowspan=1 colspan=1>Human papillomavirus 16 (HPV-16) (GD)</td><td rowspan=1 colspan=1>Varicella-Zoster Virus (HHV-3) Ellen (GD)</td></tr><tr><td rowspan=1 colspan=1>Human papillomavirus 18 (HPV-18) (QC)</td><td rowspan=1 colspan=1>Human DNA (GD)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumonia (QC)</td><td rowspan=1 colspan=1>Y</td></tr></table>

GD: Purified genomic DNA; QC: quantitated cultures from external source; IHC: culture prepared and quantitated by IMDx

This study was performed to evaluate potential interference with the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay with a panel of twenty-eight (28) biological and chemical substances. All of the potentially interfering substances were tested at concentrations at or above physiological levels or typical usage levels with two HSV-1 strains (MacIntyre and Clinical Isolate 1), and two HSV-2 strains (MS and Clinical Isolate 1). The study was carried out in the presence of HSV-1 and HSV-2 at $2 { - } 3 ~ \mathrm { X }$ LoD to evaluate potential interference with the detection of the HSV-1 and HSV-2 targets. The study was also carried out in the absence of HSV to evaluate potential interference with the detection of the internal control of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. Each potentially interfering substance was tested in triplicate. No interference was observed with any of the substances tested.

Interfering Substance Panel   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Potential Inhibitor</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Whole blood with EDTA</td><td colspan="1" rowspan="1">Heme, DNA, proteases,nucleases</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Female Urine</td><td colspan="1" rowspan="1">Non-specific PCRinhibitors</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Male Urine</td><td colspan="1" rowspan="1">Non-specific PCRinhibitors</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">Acycloguanosine</td><td colspan="1" rowspan="1">7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">K-Y® Brand Jelly</td><td colspan="1" rowspan="1">Glycerin, Cellulose</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Douche</td><td colspan="1" rowspan="1">Decyl Glucoside;Octoxynol-9</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Condom</td><td colspan="1" rowspan="1">Non-oxynol-9</td><td colspan="1" rowspan="1">0.07% v/v</td></tr><tr><td colspan="1" rowspan="1">YeastGard</td><td colspan="1" rowspan="1">Phosphoricum Acidum</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Monistat® 1</td><td colspan="1" rowspan="1">Miconazole nitrate cream</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Monistat® 3</td><td colspan="1" rowspan="1">Miconazole nitrate cream</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Vagisil® Cream</td><td colspan="1" rowspan="1">Benzocaine, Resorcinol</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Triconazole 1</td><td colspan="1" rowspan="1">Tioconazole</td><td colspan="1" rowspan="1">6.5%</td></tr><tr><td colspan="1" rowspan="1">Balneol®HygienicCleansing Lotion</td><td colspan="1" rowspan="1">Mineral oil, Fatty acids</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole 3 VaginalCream</td><td colspan="1" rowspan="1">Clotrimazole</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">CVS Anti-Itch Cream</td><td colspan="1" rowspan="1">Benzocaine;Benzalkonium Chloride</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">ListerineAntiseptic</td><td colspan="1" rowspan="1">Ethanol, Menthol,</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Mouth Wash</td><td colspan="1" rowspan="1">Thymol, Eucalyptol</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Abreva</td><td colspan="1" rowspan="1">Docosanol 10%</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">CarmexCold Sore LipBalm</td><td colspan="1" rowspan="1">Menthol, Camphor,Phenol</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Releev® cold soretreatment</td><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Lip clear Lysine+®</td><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Surfactants, fluorides,antibacterials</td><td colspan="1" rowspan="1">10 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Buffy coat</td><td colspan="1" rowspan="1">Heme, DNA, proteases,nucleases</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Mineral oil</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Diaper Rash Ointment</td><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">1% v/v</td></tr></table>

# h. Specimen Stability in Viral Transport Media

The performance of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay and the specimen stability were assessed with the following viral transport media: Remel M4, Remel M4RT, Remel M5, Remel M6, and BD Universal Viral Transport (UVT). Each transport medium was spiked with HSV-1 MacIntyre strain and HSV-2 MS strain at 2-3 X LoD and stored and tested at the following temperatures and intervals: Refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C } ) / \mathrm { D a y } 0 , 2 , 4 , 7$ , and 8; Frozen ( ${ } ^ { - 3 0 ^ { 0 } \dot { \mathrm { { C } } } }$ to $- 1 0 ^ { \circ } \mathrm { C }$ )/ Day 0, 2, 4, 7, 8, 11, 21, 28, 35, 64, 81, and 177. The viral transport media were also tested in the absence of HSV-1 and HSV-2 to determine if the viral transport media interfered with the detection of the internal control in negative samples.

For the specimen freeze-thaw study, stability was assessed by alternately freezing specimens at ${ } _ { - 3 0 } { } ^ { 0 } \mathrm { C }$ to $\mathrm { - 1 0 ^ { 0 } C }$ for a minimum of 2 hours and thawing at room temperature for a minimum of 2 hours. Each specimen was tested in replicates of three (3) for the stability studies at different conditions.

There was no interference observed with the Remel M4, Remel M4RT, Remel M5, Remel M6, and BD Universal Viral Transport (UVT) media for the detection of HSV-1 and HSV- 2 target or the internal control. The study data supported the stability claims for HSV-1 and HSV-2 specimens collected in Remel M4, Remel M4RT, Remel M5, Remel M6, and BD Universal Viral Transport (UVT) viral transport media refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ) for 7 days and frozen $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ ) for 6 months. The data also supported the claim that the specimens may undergo three (3) freeze/thaw cycles.

# i. Competitive Inhibition

Competitive inhibition of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay was evaluated to assess the potential for interference in HSV-1/2 target detection when both viruses are present in a sample. Two strains of HSV-1 (MacIntyre and Clinical Isolate 1) and two strains of HSV-2 (MS and Clinical Isolate 1) were used for the study. Contrived samples were made to mimic HSV-1 and HSV-2 co-infections, where one target was present at LoD and the second target was present at a higher concentration. No interference was seen in the detection of both HSV-1 strains in the presence of either HSV-2 strain at concentrations of up to $1 . 0 ~ \mathrm { ~ x ~ } ~ 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The highest concentration of HSV-1 that could be present while maintaining detection of the HSV-2 Clinical Isolate-1 target at 1X LoD was 100 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The highest concentration of HSV-1 that could be present while maintaining detection of the HSV-2 MS target at 1 X LoD was $5 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

# h. Carry-over/Cross Contamination

Five assay runs were performed with alternating high positive and negative samples using two contrived HSV-positive samples: one prepared using intact HSV-1 and HSV-2 at cycle number (CN) values of 26.6 (HSV-1) and 25.5 (HSV-2), and a second using HSV-1 and HSV-2 plasmid at cycle number (CN) values of 18.2 (HSV1) and 19.8 (HSV-2). No carryover events (0/138) were observed with the contrived HSV samples formulated with intact virus. A carryover rate of $1 . 4 \%$ (2/144) was observed with the plasmid-based contrived HSV samples.

j. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

The clinical performance evaluation was performed against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

b. Matrix comparison: N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A b. Clinical Specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

The performance of the AmpliVue $\textsuperscript { \textregistered }$ HSV 1&2 Assay was compared with the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance

The performance of the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay was evaluated at four geographically diverse locations within the United States. A total of 954 prospective specimens (807 anogenital and 147 oral) were tested by the IMDx HSV1/2 for Abbott $m 2 0 0 0$ assay and were compared to results obtained from the ELVIS® (Enzyme Linked Virus Inducible System) HSV ID and ${ \boldsymbol { \mathrm { D } } } ^ { 3 }$ Typing Test System (Diagnostic Hybrids, Athens, OH). The reference ELVIS viral culture used in this study is unable to detect co-infected specimens and cannot identify HSV-1 if HSV-2 is identified first. Consequently, if a specimen was positive for HSV-2, it was removed from the calculation of the HSV-1 clinical performance.

Prospective Studies: One hundred and sixty one (161) anogenital prospective specimens identified as HSV-2 positive by ELVIS viral culture were removed from the initial 807 anogenital specimens for the calculation of the HSV-1 clinical performance. Due to low prevalence of HSV-2 in oral specimens, only two oral specimens identified as HSV-2 positive by ELVIS viral culture were removed from the initial 147 oral specimens for the calculation of the HSV-1 clinical performance.

Retrospective Studies: A total of 54 retrospective specimens (27 anogenital and 27 oral) were tested with the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay and results were compared to historical results from the ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System. Twelve (12) anogenital specimens identified as HSV-2 positive by ELVIS viral culture were removed from the initial 27 anogenital specimens for the calculation of the HSV-1 clinical performance. There was no HSV-2 detected by the IMDx HSV1/2 for Abbott $m 2 0 0 0$ in the 27 oral specimens in agreement with the historical culture results.

Results from the prospective and retrospective studies are presented in the tables below.

HSV-1 Results for Anogenital Specimens (Prospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>20a</td><td rowspan=1 colspan=1>121</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>524</td><td rowspan=1 colspan=1>525</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>544</td><td rowspan=1 colspan=1>646</td></tr><tr><td rowspan=1 colspan=2>Sensitivity;95% CI</td><td rowspan=1 colspan=3>99.0% (101/102); 95% CI (94.7% - 99.8%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>96.3% (524/544); 95% (CI (94.4% - 97.6%)</td></tr></table>

a Discordant analysis was performed for 17 of the 20 specimens identified as HSV-1 positive by the IMDx HSV1/2 for Abbott $m 2 0 0 0$ assay. HSV-1 was detected in 6 of the 17 specimens. The remaining 11 specimens remained discordant (HSV-1 was not detected). b Discordant analysis was performed using bidirectional sequencing for the single specimen identified as HSV-1 negative by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-2, but not HSV-1, was detected in this specime.n

HSV-2 Results for Anogenital Specimens (Prospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-2</td><td rowspan=1 colspan=3>ELVIS HSV ID and D3 Typing</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>68a</td><td rowspan=1 colspan=1>225</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>4b}$</td><td rowspan=1 colspan=1>578</td><td rowspan=1 colspan=1>582</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>646</td><td rowspan=1 colspan=1>807</td></tr><tr><td rowspan=1 colspan=2>Sensitivity; 95% CI</td><td rowspan=1 colspan=3>97.5% (157/161); 95% (CI (93.8% - 99.0%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>89.5% (578/646); 95% CI (86.9% - 91.6%)</td></tr></table>

a Discordant analysis was performed using bidirectional sequencing for 62 of the 68 specimens identified as HSV-2 positive by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-2 was detected in 55 of the 62 specimens. The remaining 7 specimens remained discordant (HSV-2 was not detected). b Discordant analysis was performed using bidirectional sequencing for 2 of the 4 specimens identified as HSV2 negative by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-2 was not detected in either specimen..

HSV-1 Results for Oral Specimens (Prospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>24a</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=2>Sensitivity; 95% CI</td><td rowspan=1 colspan=3>100.0%(37/37); 95% CI (90.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>77.8% (84/108); 95% CI (69.1% - 84.6%)</td></tr></table>

a Discordant analysis was performed using bidirectional sequencing for the 24 specimens identified as HSV-1 positive by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-1 was detected in 14 of the 24 specimens. The remaining 10 specimens remained discordant (HSV-1 was not detected).

HSV-2 Results for Oral Specimens (Prospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>147</td></tr><tr><td rowspan=1 colspan=2>Sensitivity; 95% CI</td><td rowspan=1 colspan=3>0.0% (0/2); 95% CI (0.0% - 65.8%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>98.6% (143/145); 95% CI (95.1% - 99.6%)</td></tr></table>

a Discordant analysis was performed using bidirectional sequencing for the 2 specimens identified as HSV-2 positive by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-2 was detected in both specimens.

b Discordant analysis was performed using bidirectional sequencing for the 2 specimens identified as HSV-2 negative by the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. HSV-2 was not detected in either specimen. HSV-1 was detected in both specimens.

Note: Due to low prevalence of HSV-2 in oral specimens, there was no HSV-2 positive specimen detected in oral specimens.

HSV-1 Results for Anogenital Specimens (Retrospective Study)   

<table><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=1 colspan=4>Reference Method</td></tr><tr><td rowspan=1 colspan=2>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=2>Sensitivity; 95% CI</td><td rowspan=1 colspan=3>93.3% (14/15); 95% CI (70.2% - 98.8%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>N/A</td></tr></table>

HSV-2 Results for Anogenital Specimens (Retrospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-2</td><td rowspan=1 colspan=3>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>Sensitivity;95% CI</td><td rowspan=1 colspan=3>100.0%(12/12); 95%CI (75.7 - 100%)</td></tr><tr><td rowspan=1 colspan=2>Specificity;95% CI</td><td rowspan=1 colspan=3>93.3% (14/15); 95% CI (70.2 - 98.8%)</td></tr></table>

HSV-1 Results for Oral Specimen Results (Retrospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>Sensitivity; 95% CI</td><td rowspan=1 colspan=3>100.0% (27/27); 95%CI (87.5 % - 100%)</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>N/A</td></tr></table>

HSV-2 Results for Oral Specimen Results (Retrospective Study)   

<table><tr><td rowspan=2 colspan=2>HSV-2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>Sensitivity;95%CI</td><td rowspan=1 colspan=3>N/A</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>100.0% (27/27); 95%CI (87.5 % - 100%)</td></tr></table>

HSV-2 Oral Contrived Specimen Study: A contrived specimen study was performed to provide additional performance data for detection of HSV-2 in oral samples. HSVnegative oral samples (culture negative and PCR negative) used for the contrived study were remainders from the clinical specimens used for the method comparison study. Thirty (30) contrived HSV-2 positive oral samples were prepared by spiking HSV-2 virus into HSV-negative oral samples. HSV-2 was spiked in HSV-negative oral samples at concentrations $2 { - } 3 ~ \mathrm { X }$ LoD, $1 0 \mathrm { ~ X ~ }$ LoD, 100 X LoD, 1,000 X LoD and 10,000 X LoD. In addition, fifteen (15) HSV-1 positive oral and fifteen (15) HSVnegative oral samples remaining from the clinical specimens used for the method comparison study were also tested. All samples were randomized and blinded to the operator prior to testing. HSV-2 was detected in all contrived samples at all concentrations tested.

4. Clinical cut-off: N/A

# 5. Expected values/Reference range:

Prevalence: The prevalence of HSV-1 and HSV-2 observed during the multi-center clinical study was calculated for the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay. The prevalence rates for HSV-1 were individually established as $1 8 . 7 \%$ (121/646) for anogenital samples and $4 2 . 1 \%$ (61/145) for oral samples. The prevalence rates for HSV-2 were individually established as $2 7 . 9 \%$ (225/807) for anogenital samples and $1 . 4 \%$ (2/147) for oral samples.

Gender and Age Distribution for Anogenital Specimens   

<table><tr><td rowspan=2 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>&lt;1 to 17</td><td rowspan=1 colspan=1>5/25(20.0%)</td><td rowspan=1 colspan=1>0/10(0.0%)</td><td rowspan=1 colspan=1>5/35(14.3%)</td><td rowspan=1 colspan=1>7/31(22.6%)</td><td rowspan=1 colspan=1>0/10(0.0%)</td><td rowspan=1 colspan=1>7/41(17.1%)</td></tr><tr><td rowspan=1 colspan=1>18 to 30</td><td rowspan=1 colspan=1>63/220(28.6%)</td><td rowspan=1 colspan=1>12/54(22.2%)</td><td rowspan=1 colspan=1>75/274(27.4%)</td><td rowspan=1 colspan=1>86/292(29.5%)</td><td rowspan=1 colspan=1>14/63(22.2%)</td><td rowspan=1 colspan=1>100/355(28.2%)</td></tr><tr><td rowspan=1 colspan=1>31 to 40</td><td rowspan=1 colspan=1>20/108(18.5%)</td><td rowspan=1 colspan=1>1/20(5.0%)</td><td rowspan=1 colspan=1>21/128(16.4%)</td><td rowspan=1 colspan=1>28/124(22.6%)</td><td rowspan=1 colspan=1>16/34(47.1%)</td><td rowspan=1 colspan=1>44/158(27.8%)</td></tr><tr><td rowspan=1 colspan=1>41 to 50</td><td rowspan=1 colspan=1>8/78(10.3%)</td><td rowspan=1 colspan=1>2/14(14.3%)</td><td rowspan=1 colspan=1>10/92(10.9%)</td><td rowspan=1 colspan=1>29/95(30.5%)</td><td rowspan=1 colspan=1>3/16(18.8%)</td><td rowspan=1 colspan=1>32/111(28.8%)</td></tr><tr><td rowspan=1 colspan=1>51 to 60</td><td rowspan=1 colspan=1>5/50(10.0%)</td><td rowspan=1 colspan=1>2/10(20.0%)</td><td rowspan=1 colspan=1>7/60(11.7%)</td><td rowspan=1 colspan=1>14/59(23.7%)</td><td rowspan=1 colspan=1>2/12(16.7%)</td><td rowspan=1 colspan=1>16/71(22.5%)</td></tr><tr><td rowspan=1 colspan=1>61 to 70</td><td rowspan=1 colspan=1>2/30(6.7%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>2/35(5.7%)</td><td rowspan=1 colspan=1>15/36(41.7%)</td><td rowspan=1 colspan=1>4/9(44.4%)</td><td rowspan=1 colspan=1>19/45(42.2%)</td></tr><tr><td rowspan=1 colspan=1>71 to 80</td><td rowspan=1 colspan=1>1/10(10.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>1/16(6.3%)</td><td rowspan=1 colspan=1>3/12(25.0%)</td><td rowspan=1 colspan=1>2/8(25.0%)</td><td rowspan=1 colspan=1>5/20(25.0%)</td></tr><tr><td rowspan=1 colspan=1>81 to 95</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>104/525(19.8%)</td><td rowspan=1 colspan=1>17/121(14.0%)</td><td rowspan=1 colspan=1>121/646(18.7%)</td><td rowspan=1 colspan=1>183/653(28.0%)</td><td rowspan=1 colspan=1>42/154(27.3%)</td><td rowspan=1 colspan=1>225/807(27.9%)</td></tr></table>

Gender and Age Distribution for Oral Specimens   

<table><tr><td rowspan=2 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>&lt;1 to 17</td><td rowspan=1 colspan=1>15/33(45.5%)</td><td rowspan=1 colspan=1>12/26(46.2%)</td><td rowspan=1 colspan=1>27/59(45.8%)</td><td rowspan=1 colspan=1>0/33(0.0%)</td><td rowspan=1 colspan=1>0/26(0.0%)</td><td rowspan=1 colspan=1>0/59(0.0%)</td></tr><tr><td rowspan=1 colspan=1>18 to 30</td><td rowspan=1 colspan=1>3/12(25.0%)</td><td rowspan=1 colspan=1>5/11(45.5%)</td><td rowspan=1 colspan=1>8/23(34.8%)</td><td rowspan=1 colspan=1>0/13(0.0%)</td><td rowspan=1 colspan=1>0/11(0.0%)</td><td rowspan=1 colspan=1>0/24(0.0%)</td></tr><tr><td rowspan=1 colspan=1>31 to 40</td><td rowspan=1 colspan=1>5/8(62.5%)</td><td rowspan=1 colspan=1>3/4(75.0%)</td><td rowspan=1 colspan=1>8/12(66.7%)</td><td rowspan=1 colspan=1>1/9(11.1%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td></tr><tr><td rowspan=1 colspan=1>41 to 50</td><td rowspan=1 colspan=1>7/12(58.3%)</td><td rowspan=1 colspan=1>2/4(50.0%)</td><td rowspan=1 colspan=1>9/16(56.3%)</td><td rowspan=1 colspan=1>0/12(0.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/16(0.0%)</td></tr><tr><td rowspan=1 colspan=1>51 to 60</td><td rowspan=1 colspan=1>4/10(40.0%)</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>4/18(22.2%)</td><td rowspan=1 colspan=1>0/10(0.0%)</td><td rowspan=1 colspan=1>1/8(12.5%)</td><td rowspan=1 colspan=1>1/18(5.6%)</td></tr><tr><td rowspan=1 colspan=1>61 to 70</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>1/10(10.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>0/10(0.0%)</td></tr><tr><td rowspan=1 colspan=1>71 to 80</td><td rowspan=1 colspan=1>1/1(100.0%)</td><td rowspan=1 colspan=1>2/4(50.0%)</td><td rowspan=1 colspan=1>3/5(60.0%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td></tr><tr><td rowspan=1 colspan=1>81 to 95</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/1(0.0%)</td></tr><tr><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>1/1(100.0%)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>1/1(100.0%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/1(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>37/82(45.1%)</td><td rowspan=1 colspan=1>24/63(38.1%)</td><td rowspan=1 colspan=1>61/145(42.1%)</td><td rowspan=1 colspan=1>1/84(1.2%)</td><td rowspan=1 colspan=1>1/63(1.6%)</td><td rowspan=1 colspan=1>2/147(1.4%)</td></tr></table>

Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the IMDx HSV-1/2 for Abbott $m 2 0 0 0$ assay are shown below. These calculations are based on hypothetical prevalence and overall sensitivity and specificity per specimen type as determined in the clinical trial.

For HSV-1, these calculations are based upon an overall sensitivity and specificity of $9 9 . 0 \%$ and $9 6 . 3 \%$ , respectively, for anogenital swabs and $1 0 0 . 0 \%$ and $7 7 . 8 \%$ , respectively, for oral swabs. For HSV-2, these calculations are based upon an overall sensitivity and specificity of $9 7 . 5 \%$ and $8 9 . 5 \%$ , respectively, for anogenital swabs and $0 . 0 \%$ and $9 8 . 6 \%$ , respectively, for oral swabs.

PPV was calculated using:   
(Sensitivity x Prevalence)/([Sensitivity x Prevalence] $^ +$ [1 - Specificity] x [1 - Prevalence]). NPV was calculated using:   
(Specificity x [1 - Prevalence])/([1 - Sensitivity] x Prevalence $^ +$ Specificity x [1 - Prevalence]).

# Prevalence vs. Hypothetical Predictive Values

<table><tr><td rowspan=3 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=4>Anogenital Swabs</td><td rowspan=1 colspan=4>Oral Swabs</td></tr><tr><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>15.9</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>98.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>58.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>94.9</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>74.8</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>50.8</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>89.9</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>87.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>69.9</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>53.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>79.8</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>92.0</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>79.9</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>65.9</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>69.7</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>75.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>59.7</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>90.3</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>49.6</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.