# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K140448   
B. Purpose for Submission: The purpose of this submission is to migrate the BD ProbeTec Neisseria $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay from the previously cleared (K081825) BD Vi operating in extracted mode to the new BD Viper LT System.   
C. Measurand: Neisseria gonorrhoeae DNA   
D. Type of Test: Strand displacement nucleic acid amplification (SDA) assay   
E. Applicant: Becton, Dickinson and Company   
F. Proprietary and Established Names: BD ProbeTec™ Neisseria gonorrhoeae (GC) $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay   
G. Regulatory Information: 1. Regulation section: 21 CFR 866.3390, Neisseria spp. direct serological test reagents 2. Classification: Class II 3. Product code: LSL 4. Panel: Microbiology (83)   
H. Intended Use:

1. Intended use(s):

The BD ProbeTec Neisseria gonorrhoeae (GC) $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay, when tested with either the BD Viper™ System in Extracted Mode or the BD Viper™ LT System, uses Strand

Displacement Amplification technology for the direct, qualitative detection of Neisseria gonorrhoeae DNA in clinician-collected female endocervical and male urethral swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and male and female urine specimens (both UPT and Neat). The assay is also intended for use with gynecological specimens collected in BD SurePath™ Preservative Fluid or PreservCyt™ Solution using an aliquot that is removed prior to processing for either the BD SurePath or ThinPrep™ Pap test. The assay is indicated for use with asymptomatic and symptomatic individuals to aid in the diagnosis of gonococcal urogenital disease.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): Prescription only

4. Special instrument requirements:

The BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay uses SDA technology on either the BD Viper System or the BD Viper™ LT System.

# I. Device Description:

The BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay uses SDA technology for the detection of GC DNA. The BD Viper LT System is a table-top instrument that is designed to be fully contained on a standard laboratory bench-top. The system performs automated extraction of nucleic acids from multiple specimen types in addition to amplification and detection of target nucleic acid sequences when utilized with legally marketed in vitro diagnostic assays.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD ProbeTec™ Neisseria gonorrhoeae (GC) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay on the BD Viper System

2. Predicate 510(k) number(s): K081825

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Predicate Device:GCQ Assay on the BD ViperSystem - K081825</td><td colspan="1" rowspan="1">Subject Device:GCQ Assay on the BD ViperLT System − K140448</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD ProbeTec GC Qx</td><td colspan="1" rowspan="1">The BD ProbeTec Neisseria</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Predicate Device:GCQ Assay on the BD ViperSystem - K081825</td><td colspan="1" rowspan="1">Subject Device:GCQ Assay on the BD ViperLT System - K140448</td></tr><tr><td colspan="1" rowspan="12"></td><td colspan="1" rowspan="10">Amplified DNA Assay,when tested with either theBD ViperTM System inExtracted Mode, usesStrand DisplacementAmplification technologyfor the direct, qualitativedetection of Neisseriagonorrhoeae DNA inclinician-collected femaleendocervical and maleurethral swab specimens,patient-collected vaginalswab specimens (in aclinical setting), and maleand female urinespecimens. The assay isindicated for use withasymptomatic andsymptomatic individuals toaid in the diagnosis of</td><td colspan="1" rowspan="12">gonorrhoeae (GC) QxAmplified DNA Assay,when tested with either theBD ViperTM System inExtracted Mode or the BDViperTM LT System, usesStrand DisplacementAmplification technologyfor the direct, qualitativedetection of Neisseriagonorrhoeae DNA inclinician-collected femaleendocervical and maleurethral swab specimens,patient-collected vaginalswab specimens (in aclinical setting), and maleand female urine specimens.The assay is also intendedfor use with gynecologicalspecimens collected in BDSurePathTM PreservativeFluid or PreservCytTMSolution using an aliquotthat is removed prior toprocessing for either the BDSurePath or ThinPrepT Paptest. The assay is indicatedfor use with asymptomaticand symptomaticindividuals to aid in thediagnosis of gonococcalurogenital disease.</td></tr><tr><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">swab specimens (in aclinical setting), and male</td></tr><tr><td colspan="1" rowspan="1">and female ur</td></tr><tr><td colspan="1" rowspan="2">The assay i</td></tr><tr><td colspan="1" rowspan="1">y IS d</td></tr><tr><td colspan="1" rowspan="1">uith e</td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">gonococcal urogenital</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">disease.</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Strand DisplacementAmplification (SDA)</td><td colspan="1" rowspan="1">Strand DisplacementAmplification (SDA)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Female specimens:Endocervical swabPatient-collected vaginal swabNeat urineUPT urine</td><td colspan="1" rowspan="1">Female specimens:Endocervical swabPatient-collected vaginal swabNeat urineUPT urineLBC specimens collected in</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Predicate Device:GCQ Assay on the BD ViperSystem - K081825</td><td colspan="1" rowspan="1">Subject Device:GCQ Assay on the BD ViperLT System − K140448</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Male Specimens:Urethral swabNeat urineUPT urine</td><td colspan="1" rowspan="1">SurePath preservative fluidLBC specimens collected inPreservCyt SolutionMale Specimens:Urethral swabNeat urineUPT urine</td></tr><tr><td colspan="1" rowspan="1">PrimingMicrowell</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Primers</td><td colspan="1" rowspan="1">Target a sequence within theGC pilin gene inverting proteinhomologue</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Detector</td><td colspan="1" rowspan="1">Linear DetectorFlourescein (fluorophore)Dabcyl (quencher)</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Nucleotides</td><td colspan="1" rowspan="1">4 of 4 nucleotides required forSDA</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Non-specificreagents andcofactors</td><td colspan="1" rowspan="1">Buffering components,magnesium ions, salt, andstabilizing reagents</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">AmplificationMicrowell</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Restrictionenzyme</td><td colspan="1" rowspan="1">BsoBI restriction enzyme</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Polymerase</td><td colspan="1" rowspan="1">Bst DNA polymerase</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Nucleotides</td><td colspan="1" rowspan="1">0 of 4 nucleotides required forSDA</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Non-specificreagents andcofactors</td><td colspan="1" rowspan="1">Buffering components,magnesium ions, salt, andstabilizing reagents</td><td colspan="1" rowspan="1">Same as predicate</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition. Sections 5 and 7 – Protocols for Evaluation of the Limit of Blank & Limit of Detection and Verification of Detection Capability Claims. CLSI document EP 17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

2. CLSI. Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline. Sections 5, 6, and 8 – Specimen Collection, Specimen Transport and Storage, & Sample Storage. CLSI document MM13-A. Wayne, PA:

Clinical and Laboratory Standards Institute; 2005.

3. CLSI. Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline – Second Edition. Sections 10 and 13 – Establishment and Evaluation of Performance Characteristics of Molecular Diagnostic Tests & Controlling False-Positive Nucleic Acid Target Amplification Reactions. CLSI document MM3-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2006.

4. FDA. Assay Migration Studies for in vitro Diagnostic Devices. Guidance for Industry and FDA Staff; 2013.

# L. Test Principle:

The BD ProbeTec GC $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides, and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper LT System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of Neisseria gonorrhoeae DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescence Units (Max RFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified N. gonorrhoeae target DNA, a second fluorescently-labeled oligonucleotide is incorporated into each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the N. gonorrhoeae-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper LT instrument and an automated algorithm is applied to both the EC and N. gonorrhoeae-specific signals to report specimen results as positive, negative, or EC failure.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility

To create the panel for the reproducibility study, CT (serovar H) and GC (ATCC strain 19424) were spiked into female UPT urine, PreservCyt LBC fluid or vaginal matrix in $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ dilutent at four levels: negative, high negative $\left( \mathrm { C } _ { 2 0 } – \mathrm { C } _ { 8 0 } \right)$ , low positive (1.5x LoD), and moderate positive (3x LoD). The reproducibility panel was comprised of four levels of analyte for each of the three matrix types (4 levels x 3 matrices $= 1 2$ panel members). Each panel member was tested in duplicate on three BD Viper LT instruments over eight days with two operators testing a panel per day (2 replicates $_ { \textrm { X 3 } }$ instruments $\textbf { \em X } 8$ days x 2 operators $= 9 6$ tests per panel member). Testing was conducted at two external sites (each had one BD Viper LT) and one internal site (one BD Viper LT and three BD Viper Instruments). For the BD Viper System, two operators tested the same number of panel members on three instruments over a period of four days. Results were tabulated by day, by operator, by replicate, across days within a site, and across sites for each sample level. Positive and negative percent agreement was calculated along with $9 5 \%$ confidence intervals for each assay by specimen type and level over all sites and days.

Table 1: Reproducibility of the BD ProbeTec GCQ Assay on the BD Viper LT   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SpecimenPType</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>% Expectedesults*</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Meanof MaxRFU</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>PreservCytLBC</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>280.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>65.4</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>287.6</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>20.8% (20/96)</td><td rowspan=1 colspan=1>(13.9 - 30.0%)</td><td rowspan=1 colspan=1>560.2</td><td rowspan=1 colspan=1>425.0</td><td rowspan=1 colspan=1>75.9</td><td rowspan=1 colspan=1>49.0</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>427.8</td><td rowspan=1 colspan=1>76.4</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1415.9</td><td rowspan=1 colspan=1>231.4</td><td rowspan=1 colspan=1>16.3</td><td rowspan=1 colspan=1>172.0</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>289.7</td><td rowspan=1 colspan=1>20.5</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0%(94/94*)</td><td rowspan=1 colspan=1>(96.1 - 100.0%)</td><td rowspan=1 colspan=1>1631.9</td><td rowspan=1 colspan=1>169.7</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>70.9</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>206.4</td><td rowspan=1 colspan=1>12.6</td></tr><tr><td rowspan=4 colspan=1>VaginalSwab</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.0% (95/96)</td><td rowspan=1 colspan=1>(94.3 - 99.8%)</td><td rowspan=1 colspan=1>41.6</td><td rowspan=1 colspan=1>180.1</td><td rowspan=1 colspan=1>432.6</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>180.6</td><td rowspan=1 colspan=1>433.8</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>13.5% (13/96)</td><td rowspan=1 colspan=1>(8.1 - 21.8%)</td><td rowspan=1 colspan=1>871.5</td><td rowspan=1 colspan=1>562.4</td><td rowspan=1 colspan=1>64.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>88.2</td><td rowspan=1 colspan=1>10.1</td><td rowspan=1 colspan=1>569.2</td><td rowspan=1 colspan=1>65.3</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>100.0%(95/95*)</td><td rowspan=1 colspan=1>(96.1 - 100.0%)</td><td rowspan=1 colspan=1>1687.5</td><td rowspan=1 colspan=1>297.7</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>299.7</td><td rowspan=1 colspan=1>17.8</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1819.2</td><td rowspan=1 colspan=1>163.3</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>48.2</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>43.3</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>73.3</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>190.3</td><td rowspan=1 colspan=1>10.5</td></tr><tr><td rowspan=4 colspan=1>FemaleUPT</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>221.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>221.8</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>18.8% (18/96)</td><td rowspan=1 colspan=1>(12.2 - 27.7%)</td><td rowspan=1 colspan=1>766.6</td><td rowspan=1 colspan=1>502.1</td><td rowspan=1 colspan=1>65.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>75.8</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>508.0</td><td rowspan=1 colspan=1>66.3</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1593.6</td><td rowspan=1 colspan=1>224.9</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>86.6</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>243.8</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1741.5</td><td rowspan=1 colspan=1>126.1</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>86.2</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>158.2</td><td rowspan=1 colspan=1>9.1</td></tr></table>

\* Two invalid LBC results and one invalid swab result occurred due to extraction transfer errors. \*\* The results for Negative panel members calculated according to an expected result of ‘negative for GC’. All other panel members calculated according to an expected result of ‘positive for GC’.

Table 2: Reproducibility of the BD ProbeTec GCQ Assay on the BD Viper   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>% ExpectedResults*</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean ofMaxRFU</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>PreservCytLBC</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>438.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>245.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>128.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>518.5</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>18.8% (18/96)</td><td rowspan=1 colspan=1>(12.2 - 27.7%)</td><td rowspan=1 colspan=1>1047.1</td><td rowspan=1 colspan=1>672.7</td><td rowspan=1 colspan=1>64.2</td><td rowspan=1 colspan=1>205.0</td><td rowspan=1 colspan=1>19.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>703.2</td><td rowspan=1 colspan=1>67.2</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>99.0% (95/96)</td><td rowspan=1 colspan=1>(94.3 - 99.8%)</td><td rowspan=1 colspan=1>1590.9</td><td rowspan=1 colspan=1>356.3</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>356.3</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0%(96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1943.2</td><td rowspan=1 colspan=1>110.8</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>111.5</td><td rowspan=1 colspan=1>5.7</td></tr><tr><td rowspan=4 colspan=1>Vaginal Swab</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>379.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>61.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>55.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>68.3</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>394.4</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>35.4% (34/96)</td><td rowspan=1 colspan=1>(26.6 - 45.4%)</td><td rowspan=1 colspan=1>594.5</td><td rowspan=1 colspan=1>622.7</td><td rowspan=1 colspan=1>104.7</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>108.7</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1>637.4</td><td rowspan=1 colspan=1>107.2</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>99.0%(95/96)</td><td rowspan=1 colspan=1>(94.3 - 99.8%)</td><td rowspan=1 colspan=1>1589.4</td><td rowspan=1 colspan=1>433.1</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>137.5</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>457.5</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1666.6</td><td rowspan=1 colspan=1>337.3</td><td rowspan=1 colspan=1>20.2</td><td rowspan=1 colspan=1>76.6</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>58.5</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>350.8</td><td rowspan=1 colspan=1>21.1</td></tr><tr><td rowspan=4 colspan=1>FemaleUPT</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>468.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>133.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>486.8</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>8.3% (8/96)</td><td rowspan=1 colspan=1>(4.3 - 15.6%)</td><td rowspan=1 colspan=1>1127.9</td><td rowspan=1 colspan=1>673.0</td><td rowspan=1 colspan=1>59.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>673.0</td><td rowspan=1 colspan=1>59.7</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1806.2</td><td rowspan=1 colspan=1>256.7</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>256.7</td><td rowspan=1 colspan=1>14.2</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>100.0% (96/96)</td><td rowspan=1 colspan=1>(96.2 - 100.0%)</td><td rowspan=1 colspan=1>1937.9</td><td rowspan=1 colspan=1>143.3</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>143.3</td><td rowspan=1 colspan=1>7.4</td></tr></table>

\* The results for Negative panel members calculated according to an expected result of ‘negative for GC’. All other panel members calculated according to an expected result of ‘positive for GC’.

The data demonstrates good reproducibility as it shows $\ge 9 5 \%$ detection of GC at low positive/close to the detection limit and moderate positive levels on both instruments. Detection was $\leq 2 1 \%$ for the high negative samples on the BD Viper LT, which is acceptable and typical for SDA assays. The total percent variance for each of the specimen types across both the old and the new system are approximately equivalent.

$b$ . Linearity/assay reportable range: N/A   
c. Traceability, Stability, Expected values (controls, calibrators, or   
methods):

# Post Pre-Warm Specimen Stability

The existing workflow previously cleared for the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay on the BD Viper includes a pre-warm step for swab and urine samples. The workflow proposed in K140448 for the BD Viper LT includes a pre-warm step for LBC specimens in SurePath or PreservCyt media so all sample types (urine, swabs, and LBC specimens) can be tested on the same instrument run. For this reason, a post pre-warm specimen stability study was conducted to support the storage and transport stability claims for LBC specimens in SurePath or PreservCyt LBC media.

Mini-pools were created by diluting CT and NG negative LBC specimens in either SurePath or PreservCyt LBC media at a ratio of $1 { : } 3 . 4 ~ \mathrm { m L }$ (12 mini-pools were created for each LBC media). Each mini-pool was split equally. Half were spiked with CT serovar H and NG strain ATCC 19424 (at $9 0 ~ \mathrm { E B / m L }$ and $3 0 0 \ { \mathrm { c e l l s / m L } }$ , respectively) and half were not spiked (negative samples). From these split mini-pools, $2 . 2 \mathrm { m L }$ aliquots were removed and dispensed into BD LBC Specimen Dilution Tubes (15 samples per group; 30 samples total/mini-pool). Fourteen of these tubes were pre-warmed in the BD Viper LT heat block at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ ; one tube was read at “baseline” (no warming).

After the pre-warm step, the seven tubes were stored at the following conditions: two tubes at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7 days, two tubes at $3 0 \mathrm { { } ^ { \circ } C }$ for 7 days, and two tubes at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for 90 days (one tube was available for testing and one was available for retesting, if required). At each storage condition/time point, one negative and one positive tube from each of the 12 mini-pools was removed and tested in a dual dispensing mode, which generated two replicate test results (1 tube x 2 replicates x 12 mini-pools $= 2 4$ measurements/storage condition/time point for each LBC media type). Four lots of extraction reagents, 2-3 lots of amplification reagents, three lots of the control set for the BD ProbeTec CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assays, and three BD Viper LT Instruments were included in this study. Results are summarized below.

Table 3: PreservCyt Specimens Post Pre-Warm Stability Results   

<table><tr><td rowspan=1 colspan=1>Temperature</td><td rowspan=1 colspan=1>Timepoint</td><td rowspan=1 colspan=1>Positives</td><td rowspan=1 colspan=1>Negatives</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>2-8 </td><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>30 </td><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>2-8 °C</td><td rowspan=1 colspan=1>Day 7</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>30 </td><td rowspan=1 colspan=1>Day 7</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>-20 °</td><td rowspan=1 colspan=1>Day 90</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)</td></tr><tr><td rowspan=1 colspan=1>-20 °</td><td rowspan=1 colspan=1>Day 90</td><td rowspan=1 colspan=1>24/24 (Pass)</td><td rowspan=1 colspan=1>0/24 (Pass)*</td></tr></table>

\*Data generated by Out of Specification (OOS) testing after contaminated environment was deemed clean.

Table 4: SurePath Specimens Post Pre-Warm Stability Results   

<table><tr><td colspan="1" rowspan="1">Temperature</td><td colspan="1" rowspan="1">Timepoint</td><td colspan="1" rowspan="1">Positives</td><td colspan="1" rowspan="1">Negatives</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Baseline</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">2-8 </td><td colspan="1" rowspan="1">Day 3</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">30 </td><td colspan="1" rowspan="1">Day 3</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">2-8 °</td><td colspan="1" rowspan="1">Day 7</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">30 °</td><td colspan="1" rowspan="1">Day 7</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">-20 </td><td colspan="1" rowspan="1">Day 90</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr><tr><td colspan="1" rowspan="1">-20 °</td><td colspan="1" rowspan="1">Day 90</td><td colspan="1" rowspan="1">24/24 (Pass)</td><td colspan="1" rowspan="1">0/24 (Pass)</td></tr></table>

The data indicates that, for all conditions and time points tested, the percent positive rate for pre-warmed GC positive and percent negative rate for GC negative LBC samples was $100 \%$ for samples stored in either PreservCyt or SurePath LBC media.

# Run Control Failure Rate

This analysis was conducted to determine the failure rate for positive and negative controls across four different studies. One positive and negative control result was recorded for each run for each of the following:

Clinical agreement study $= 5 6$ runs   
Carry-over study (in K140448, referred to as “contamination validation study”) $= 6 3$ runs   
LoD/LoB validation studies $= 7 1$ runs   
Reproducibility study $= 5 1$ runs

A total of 241 results were collected for each control type. There were no false negatives found in the positive control samples assessed $( 0 / 2 4 1 \mathrm { ~ x ~ } 1 0 0 = 0 \%$ failure rate). There were 2 false positives found in the negative control samples assessed $( 2 / 2 4 1 \mathrm { ~ x ~ } 1 0 0 =$ $0 . 8 3 \%$ failure rate).

# Environmental Study

The environmental study was conducted to assess the performance of the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay on the BD Viper LT when the instrument is exposed to variations in environmental temperature and humidity. One BD Viper LT instrument was placed in an environmental chamber and equilibrated to either 1) $1 8 ^ { \circ } \mathrm { C }$ with $20 \%$ humidity (low temperature, low humidity) or 2) $2 7 ^ { \circ } \mathrm { C }$ with $8 5 \%$ humidity (high temperature, high humidity). Three lots of positive and negative controls from the control set for the BD ProbeTec CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assays were reconstituted with Viper QC Wash Buffer and run as samples. For the negative controls, 30 “samples” were prepared from each lot with one run per lot. An extraction control (EC) was included in each of the negative controls to show that the negative result was valid (30 negative samples/ECs x 3 lots $= 9 0$ measurements). For the positive controls, 90 “samples” were prepared from each lot with 3 runs/lot (90 positive samples/lot x 3 lots $= 2 7 0$ measurements).

Each rack was loaded with all positive or all negative controls (30/rack) that were logged in as “samples”, and one pair of positive and negative controls that were run normally

(e.g., reconstituted, extracted, and amplified by the BD Viper LT). This study required 12 runs total with one lot of extraction and amplification reagents for testing with the assay. The results are summarized in the table below.

Table 5 - Environmental System Testing Summary   

<table><tr><td rowspan=2 colspan=1>Conditions/Parameter</td><td rowspan=1 colspan=3>GCQ Percent Correct (N)</td><td rowspan=2 colspan=1>Pass/Fail ≥97%</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>1: 18°C / 20% RH</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100% (270/270)</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>2: 27°C / 85% RH</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100% (270/270)</td><td rowspan=1 colspan=1>Pass</td></tr></table>

There were no false negatives or false positives detected in this study, indicating that the performance of the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay is not affected by the temperature and humidity tested in this study.

# d. Detection limit:

# Limit of Detection (LoD)

Sample mini-pools were prepared from pre-screened CT/GC negative vaginal swabs in $\mathrm { Q } ^ { \mathrm { x } }$ diluent, urine in UPT, endocervical swabs in SurePath LBC diluent, and endocervical swabs in PreservCyt LBC diluent. Four mini-pools were prepared for each matrix to be tested.

Four mini-pools of matrix were used to create panels with six analyte levels per minipool and six replicates per analyte level (4 minipools x 6 levels $_ { \textrm { X 6 } }$ replicates $= 1 4 4$ total tubes). The panels were tested across six instruments. The table below demonstrates the setup of a rack on one instrument:

Table 6 – LoD Testing Rack Layout   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Level 5MP 1</td><td rowspan=1 colspan=1>Level 4MP 1</td><td rowspan=1 colspan=1>Level 3MP 1</td><td rowspan=1 colspan=1>Level 2MP 1</td><td rowspan=1 colspan=1>Level 1MP 1</td><td rowspan=1 colspan=1>nullMP 1</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Level 5MP 2</td><td rowspan=2 colspan=1>Level 4MP 2</td><td rowspan=2 colspan=1>Level 3MP 2</td><td rowspan=1 colspan=1>Level 2</td><td rowspan=2 colspan=1>Level 1MP 2</td><td rowspan=2 colspan=1>nullMP 2</td></tr><tr><td rowspan=1 colspan=1>MP 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Level 5MP 3</td><td rowspan=1 colspan=1>Level 4MP 3</td><td rowspan=1 colspan=1>Level 3MP 3</td><td rowspan=1 colspan=1>Level 2MP 3</td><td rowspan=1 colspan=1>Level 1MP 3</td><td rowspan=1 colspan=1>nullMP 3</td></tr><tr><td rowspan=2 colspan=1>Pos Cont</td><td rowspan=2 colspan=1>Neg Cont</td><td rowspan=2 colspan=1>Level 5MP 4</td><td rowspan=2 colspan=1>Level 4MP 4</td><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>null</td></tr><tr><td rowspan=1 colspan=1>MP 4</td><td rowspan=1 colspan=1>MP 4</td><td rowspan=1 colspan=1>MP 4</td><td rowspan=1 colspan=1>MP 4</td></tr></table>

(Level $=$ analyte level; $\begin{array} { r } { \mathbf { M P } = \operatorname* { m i n i } \mathbf { \bar { \Pi } } \mathbf { \Theta } } \end{array}$

Each rack was run twice on the same instrument (4 replicates x 2 runs x 3 instruments $=$ 24 measurements).

Spiking levels for the sets consisted of the following concentrations of GC: 0, 5, 20, 50, 100, or $2 0 0 \mathrm { c e l l s / m L }$ . The LoD concentrations for each instrument were determined for two strains of GC by identifying the lowest concentration of GC cells/mL which provided a detection rate of at least $9 5 \%$ . The table below summarizes the LoD for the two strains of GC in each matrix type as detected by the two instruments:

Table 7: LoD comparison between BD Viper LT and BD Viper   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>BD ViperLT</td><td rowspan=1 colspan=1>BD Viper</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>GC Strain</td><td rowspan=1 colspan=1>LoD (GCcells/mL)</td><td rowspan=1 colspan=1>LoD (GCcells/mL)</td></tr><tr><td rowspan=2 colspan=1>VaginalMatrix</td><td rowspan=1 colspan=1>19424</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>49226</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=2 colspan=1>UPT Urine</td><td rowspan=1 colspan=1>19424</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>49226</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=2 colspan=1>Surepathmatrix inLBCDiluent</td><td rowspan=1 colspan=1>19424</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>49226</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=2 colspan=1>PreservCytmatrix inLBCDiluent</td><td rowspan=1 colspan=1>19424</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>49226</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td></tr></table>

The difference in LoD as determined on the BD Viper LT compared to the BD Viper System ranged from $0 . 5 \mathrm { x }$ to $2 . 5 \mathrm { x }$ GC cells/mL. The observed ratio is acceptable as ‘approximately equivalent’.

# Limit of Blank (LoB)

Data from GC negative specimens (Level 0) from the analytical LoD studies described above were combined to calculate an overall LoB for the BD ProbeTec GC $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Assay on the BD Viper LT. Data from all specimen matrices were compared and the worst (or highest) LoB estimate among all the specimen matrices was selected as the overall LoB (70 MaxRFU). Because this value is below the assay threshold of 125 MaxRFU, this demonstrates that the assay has an acceptable separation between background fluorescence obtained from GC negative specimens and the assay cut-off for a positive result.

e. Analytical specificity:

# Carry-Over/System Contamination

The carry-over study was conducted to look for the presence of contamination in negative specimens due to carry-over of DNA during nucleic acid extraction and PCR amplification in the BD Viper LT. Each run consisted of 15 high positive and 15 negative samples arranged in alternating positive or negative rows. The CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was selected as the representative analyte test due to better analytical sensitivity of CT. For information on the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay, see K140446. To create high positive samples, $1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ $\mathrm { E B / m L }$ of CT was spiked into either one of two diluents (listed below). These diluents were selected because they represent two most challenging settings that can cause sample contamination.

1. 3 mL of Qx Swab Diluent: This volume was selected to represent a maximum fill volume, thereby maximizing the amount of fluid adhering to the outside of the pipette tip.   
2. PreservCyt LBC media diluted in BD LBC Specimen Dilution tube diluent (1.7 mL LBC diluent $^ +$ 0.5 mL PreservCyt $= 2 . 2$ mL; “LBC Specimen Matrix”): This LBC media was selected because it has a high alcohol content and is therefore more likely to drip from the pipette tips during sample transfer due to low surface tension.

Each run consisted of 30 samples. Seven runs were conducted per instrument with 3 BD Viper LT Systems for a total of (30 samples/run x 7 runs $_ { \textrm { X 3 } }$ instruments) 630 measurements for each diluent. Runs were conducted by three operators over a period of three days. Two lots of extraction reagents, one lot of amplification reagents, and two lots of the BD CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Control Set were used in this study.

Table 8 – Carry Over on Viper LT   

<table><tr><td rowspan=2 colspan=1>BD ViperLTSystem</td><td rowspan=1 colspan=3>Qx Sample Diluent</td><td rowspan=1 colspan=3>LBC Specimen Matrix</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.48%</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.48%</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>630</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.32%</td><td rowspan=1 colspan=1>630</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr></table>

The overall rate of carry-over was $0 . 3 2 \%$ (2/630) for CT spiked into the $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent and $0 . 0 \%$ (0/630) for CT spiked into PreservCyt media diluted in BD LBC Specimen Dilution tube diluent. These results are acceptable as they are comparable to the levels of carry-over cleared for the BD ProbeTec GCQ Assay on the BD Viper in K081825.

f. Assay cut-off:

Assay cut-off was established in K081825

2. Comparison studies:

a. Method comparison with predicate device:

# Clinical Comparison Study

A clinical comparison study was performed to compare results obtained with the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay tested on the BD Viper LT system and the BD Viper system run in extracted mode. Clinical samples were collected between October 2012 and April 2013 from four geographically diverse clinical sites in North America (OB/GYN, sexually transmitted disease and family planning clinics). For each female subject, specimens were collected in the following order: (1) a first void urine specimen, (2) five patient-collected vaginal swab specimens, and (3) a SurePath liquid based cytology (LBC) specimen and a PreservCyt LBC specimen. The LBC specimen collection order was randomized throughout the study. For each male subject, a first void urine specimen was collected. Each urine specimen (male and female) was aliquoted into five UPTs (BD urine Preservative Transport for the $\mathbf { Q } ^ { \mathrm { x } }$ Amplified DNA Assays).

A total of 823 subjects were enrolled, 170 males and 653 females. Specimens were collected from subjects presenting with symptoms of chlamydia and/or gonorrhoeae (symptomatic) and shipped to BD on cold packs for screening, aliquoting, and panel assembly. Following exclusion of non-compliant subjects and non-evaluable specimens, a total of 617 compliant female subjects and 167 compliant male subjects were available for the clinical comparison panel assembly.

To create the clinical comparison panel, for each subject, a UPT specimen was tested on the BD Viper System operating in extracted mode (BD Viper). These results were used to confirm that the target number of CT and NG positive panel members had been met. The clinical comparison panel consisted of randomly selected positive and negative specimens with four aliquots created from each specimen type. The number of GC positive and negative specimens chosen for the clinical comparison panel was based on minimums suggested by FDA in previous correspondence (minimum of 25 GC positive and 40 negative). The positive and negative specimens were randomized within the panel and the testing sites were blinded to the results for each panel member. Panels were identical across all testing sites. Three external sites tested the panels on the BD Viper LT instrument and compared the results (positive and negative percent agreement) to the results obtained on the BD Viper. The BD Viper panel testing was conducted internally at BD.

Table 9 – Clinical Comparison Study Results – Female Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=2>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>VaginalSw</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%, 100.0%)</td><td rowspan=1 colspan=1>94.9% (75/79)</td><td rowspan=1 colspan=1>(87.7%, 98.0%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.7%, 99.3%)</td><td rowspan=1 colspan=1>96.2% (76/79)</td><td rowspan=1 colspan=1>(89.4%, 98.7%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.7%, 99.3%)</td><td rowspan=1 colspan=1>96.2% (76/79)</td><td rowspan=1 colspan=1>(89.4%, 98.7%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>97.5%(79/81)</td><td rowspan=1 colspan=1>(92.6%, 100.0%)</td><td rowspan=1 colspan=1>95.8%(227/237)</td><td rowspan=1 colspan=1>(92.0%, 98.7%)</td></tr><tr><td rowspan=4 colspan=1>UPT</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.7%, 99.3%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.7%, 99.3%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>97.5%(79/81)</td><td rowspan=1 colspan=1>(92.6%, 100.0%)</td><td rowspan=1 colspan=1>100.0%(237/237)</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>SurePath</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>96.4%(27/28)</td><td rowspan=1 colspan=1>(82.3%, 99.4%)</td><td rowspan=1 colspan=1>100.0% (78/78)</td><td rowspan=1 colspan=1>(95.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>96.4%(27/28)</td><td rowspan=1 colspan=1>(82.3%, 99.4%)</td><td rowspan=1 colspan=1>100.0% (78/78)</td><td rowspan=1 colspan=1>(95.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>96.4%(27/28)</td><td rowspan=1 colspan=1>(82.3%, 99.4%)</td><td rowspan=1 colspan=1>98.7% (77/78)</td><td rowspan=1 colspan=1>(93.1%, 99.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>96.4%(81/84)</td><td rowspan=1 colspan=1>(89.3%, 100.0%)</td><td rowspan=1 colspan=1>99.6%(233/234)</td><td rowspan=1 colspan=1>(98.7%, 100.0%)</td></tr><tr><td rowspan=4 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (79/79)</td><td rowspan=1 colspan=1>(95.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>100.0%(81/81)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(237/237)</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table 10 – Clinical Comparison Study Results – Male Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=2>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>UPT</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>(91.2%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (73/73)</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>(91.2%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (73/73)</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>(91.2%, 100.0%)</td><td rowspan=1 colspan=1>98.6% (72/73)</td><td rowspan=1 colspan=1>(92.6%, 99.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>100.0%(120/120)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.5%(218/219)</td><td rowspan=1 colspan=1>(98.6%, 100.0%)</td></tr></table>

b. Matrix comparison:

N/A

3. Clinical studies: a. Clinical Sensitivity and Specificity N/A

4. Clinical cut-off: N/A

5. Expected values/Reference range:

N/A

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.