# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K140377

# B. Purpose for Submission:

To unmask three analytes that were not cleared in the original K121454 xTAG Gastrointestinal Pathogen Panel (Adenovirus 40/41, Entamoeba histolytica (E. histolytica), and Vibrio cholerae (V. cholerae) cholera toxin gene (ctx)) as well as add a claim for human stool in Cary Blair media.

# C. Measurand:

# Viruses

· Adenovirus 40/41 · Norovirus GI/GII · Rotavirus A

Bacteria

· Campylobacter (C. jejuni, C. coli and C. lari only)   
· Clostridium difficile (C. difficile) toxin A/B   
· Escherichia coli (E. coli) O157   
· Enterotoxigenic Escherichia coli (ETEC) LT/ST   
· Salmonella   
· Shiga-like Toxin producing E. coli (STEC) stx 1/stx 2 Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae) Vibrio cholerae (V. cholerae) cholera toxin gene (ctx)

# Parasites

· Cryptosporidium (C. parvum and C. hominis only) Entamoeba histolytica (E. histolytica) Giardia (G. lamblia only - also known as $G .$ . intestinalis and G. duodenalis) in raw human stool samples and human stool samples in Cary Blair media.

# D. Type of Test:

Qualitative nucleic acid multiplex test

# E. Applicant:

Luminex Molecular Diagnostics, Inc., Toronto, Ontario, Canada

F. Proprietary and Established Names: xTAG® Gastrointestinal Pathogen Panel (GPP)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 – Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: Class II

3. Product code: PCH, NSU, JJH

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The xTAG® Gastrointestinal Pathogen Panel (GPP) is a multiplexed nucleic acid test intended for the simultaneous qualitative detection and identification of multiple viral, bacterial and parasitic nucleic acids in human stool specimens or human stool in Cary Blair media from individuals with signs and symptoms of infectious colitis or gastroenteritis. The following pathogen types, subtypes and toxin genes are identified using the xTAG GPP:

# Viruses:

Adenovirus 40/41 Norovirus GI/GII Rotavirus A

# Bacteria:

Campylobacter (C. jejuni, C. coli and C. lari only) Clostridium difficile (C. difficile) toxin A/B

Escherichia coli (E. coli) O157   
Enterotoxigenic E. coli (ETEC) LT/ST   
Salmonella   
Shiga-like Toxin producing E. coli (STEC) stx 1/stx 2   
Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae)   
Vibrio cholerae (V. cholerae) cholera toxin gene (ctx)

# Parasites:

Cryptosporidium (C. parvum and C. hominis only)   
Entamoeba histolytica (E. histolytica)   
Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis)

The detection and identification of specific gastrointestinal microbial nucleic acid from individuals exhibiting signs and symptoms of gastrointestinal infection aids in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation, laboratory findings and epidemiological information. A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

# $\mathbf { x T A G } ^ { ( \mathbf { \widehat { B } } ) }$ GPP positive results are presumptive and must be confirmed by FDAcleared tests or other acceptable reference methods.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out coinfection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative xTAG® Gastrointestinal Pathogen Panel results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

# xTAG GPP is not intended to monitor or guide treatment for C. difficile infections.

The xTAG GPP is indicated for use with the Luminex ${ } ^ { \textregistered } 1 0 0 / 2 0 0 ^ { \mathrm { r m } }$ instrument with xPONENT® software.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): For prescription use only. Manufacturer must provide device-specific user training to facilities prior to using the device.

4. Special instrument requirements: Extraction: Biomerieux NucliSens® EasyMag® instrument

Analysis: Luminex $\textsuperscript { \textregistered } 1 0 0 / 2 0 0 ^ { \mathrm { \tiny T M } }$ instruments with xPONENT® software.

# I. Device Description:

The Luminex Molecular Diagnostics xTAG GPP consists of kit reagents and software. The reagents in conjunction with a thermal cycler are used to perform nucleic acid amplification (reverse transcription‐polymerase chain reaction, or RT‐PCR/PCR), and the protocol configuration file is used to generate results while the data analysis software (TDAS GPP (US)) is used to analyze the results from the Luminex Corporation Luminex 100/200 instrument system (which includes the xPONENT core software).

The components of the xTAG GPP kit are contained within 2 boxes (one that is frozen, and one that is refrigerated). The kit is shipped with the xTAG GPP CD which contains the xTAG GPP T‐A (LX) protocol configuration file and the TDAS GPP (US) software. The instrument is shipped with the xPONENT software.

The xTAG Gastrointestinal Pathogen Panel (xTAG GPP) incorporates multiplex reverse transcription and polymerase chain reaction (RT‐PCR / PCR) with Luminex's proprietary universal tag sorting system on the Luminex platform. The assay also detects an internal control (bacteriophage MS2) that is added to each sample prior to extraction. Each sample is pre‐treated prior to extraction and is then put through extraction using the Biomerieux NucliSens EasyMag kit (product code JJH, class I, an IVD‐labeled automated system for nucleic acid extraction).

Post‐extraction, for each sample, $1 0 ~ \mu \mathrm { L }$ of extracted nucleic acid is amplified in a single multiplex RT‐PCR/PCR reaction. Each target or internal control in the sample results in PCR amplicons ranging from 58 to 202 bp (not including the 24‐mer tag). A five $\mu \mathrm { L }$ aliquot of the RT‐PCR product is then added to a hybridization/detection reaction containing bead populations coupled to sequences from the Universal Array ("antitags"), streptavidin, R‐phycoerythrin conjugate. Each Luminex bead population detects a specific microbial target or control through a specific tag/anti‐tag hybridization reaction. Following the incubation of the RT‐PCR products with the xTAG GPP Bead Mix and xTAG Reporter Buffer, the Luminex instrument sorts and reads the hybridization/detection reactions.

A signal, or median fluorescence intensity (MFI), is generated for each bead population. These fluorescence values are analyzed to establish the presence or absence of bacterial, viral or parasitic targets and/or controls in each sample. A single multiplex reaction identifies all targets.

The xTAG Data Analysis Software for the Gastrointestinal Pathogen Panel (TDAS GPP (US)) analyzes the data to provide a report summarizing which pathogens are present. Before data are analyzed, a user has the option to select a subset of the targets from the intended use of the xTAG GPP (for each sample). Consequently the remaining target results are masked and cannot be retrieved.

Target results above or equal to the cutoff are considered positive, while target results below the cutoff are considered negative. For each sample analyzed by TDAS GPP (US), there are individual results for each of the targets and the internal control (bacteriophage MS2).

# J. Substantial Equivalence Information:

1. Predicate device name(s): xTAG GPP

2. Predicate 510(k) number(s): K121454

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Luminex Molecular Diagnostics</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Extraction Method</td><td rowspan=1 colspan=1>Biomérieux NucliSENS®EasyMag®</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Kit Reagents</td><td rowspan=1 colspan=1>xTAG® GPP Primer Mix,xTAG® OneStep Enzyme Mix,xTAG® OneStep Buffer,xTAG® RNase-Free Water,xTAG® BSA, xTAG® MS2,xTAG® GPP Bead Mix,xTAG® Reporter Buffer,xTAG® 0.22 SAP</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test Format</td><td rowspan=1 colspan=1>Multiplex MAGPLEX bead-based universal array</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection Method</td><td rowspan=1 colspan=1>Fluorescence based</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Quality Control</td><td rowspan=1 colspan=1>Internal Control (MS2), rotatinganalyte controls and negativecontrol (RNAse-free water)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrument SoftwareSystem</td><td rowspan=1 colspan=1>Luminex 100/200 withxPONENT Software</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Human stool specimens and human stool inCary-Blair media</td><td rowspan=1 colspan=1>Human stool specimens</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>Updated assay protocol to acquire and showdata for additional 3 analytes: Adenovirus40/41, Entamoeba histolytica (E.histolytica), and Vibrio cholerae (V.cholerae) cholera toxin gene (ctx).xPONENT 3.1 software and higher</td><td rowspan=1 colspan=1>Assay protocol fileexcludes analytesAdenovirus 40/41,Entamoeba histolytica (E.histolytica), and Vibriocholerae (V. cholerae)cholera toxin gene (ctx)</td></tr></table>

<table><tr><td colspan="3"></td><td></td></tr><tr><td colspan="3">Differences</td><td>Predicate</td></tr><tr><td>Item</td><td colspan="2">Device The xTAG® Gastrointestinal</td><td>xPONENT 3.1 software</td></tr><tr><td>Intended Use</td><td colspan="2">Pathogen Panel (GPP) is a multiplexed nucleic acid test intended for the simultaneous qualitative detection and identification of multiple viral, bacterial and parasitic nucleic acids in human stool specimens or human stool in Cary Blair media from individuals with signs and symptoms of infectious colitis or gastroenteritis. The following pathogen types, subtypes and toxin genes are identified using the</td><td>The xTAG® Gastrointestinal Pathogen Panel (GPP) is a multiplexed nucleic acid test intended for the simultaneous qualitative detection and identification of multiple viral, parasitic, and bacterial nucleic acids in human stool specimens from individuals with signs and symptoms of infectious colitis or gastroenteritis. The following pathogen types, subtypes and toxin</td></tr><tr><td colspan="3"></td></tr><tr><td>Viruses:</td><td colspan="2">xTAG GPP:</td></tr><tr><td>•</td><td>Adenovirus 40/41 Norovirus GI/GII Rotavirus A</td><td>the xTAG® GPP: Campylobacter (C. jejuni, C. coli and C. lari only</td></tr><tr><td>Bacteria:</td><td></td><td>Clostridium difficile (C. difficile) toxin A/B</td></tr><tr><td></td><td>Campylobacter (C. jejuni, C. coli and C. lari</td><td>Cryptosporidium (C. parvum and C.</td></tr><tr><td></td><td>only) Clostridium difficile (C.</td><td>hominis only) Escherichia coli (E.</td></tr><tr><td></td><td>difficile) toxin A/B Escherichia coli (E. coli)</td><td>coli) 0157 Enterotoxigenic</td></tr><tr><td></td><td>0157 Enterotoxigenic E. coli</td><td>Escherichia coli (ETEC) LT/ST</td></tr><tr><td></td><td>(ETEC) LT/ST Salmonella</td><td>Giardia (G. lamblia only - also known as</td></tr><tr><td></td><td>Shiga-like Toxin producing E. coli</td><td>G. intestinalis and G. duodenalis)</td></tr><tr><td></td><td>(STEC) stx 1/stx 2 Shigella (S. boydii, S. sonnei, S. flexneri and S.</td><td>Norovirus GI/GII Rotavirus A</td></tr><tr><td></td><td>dysenteriae)</td><td>Salmonella Shiga-like Toxin</td></tr><tr><td></td><td>Vibrio cholerae (V.</td><td>producing E. coli (STEC) stx 1/stx 2</td></tr><tr><td></td><td>cholerae) cholera toxin gene (ctx)</td><td>Shigella (S. boydii, S. sonnei, S. flexneri</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td colspan="2">Device Predicate Parasites: and S. dysenteriae)</td></tr><tr><td></td><td>Cryptosporidium (C. • parvum and C. hominis only Entamoeba histolytica (E. histolytica) Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis) The detection and identification of specific gastrointestinal microbial nucleic acid from individuals exhibiting signs and symptoms of gastrointestinal infection aids in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation, laboratory findings and epidemiological information. A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context positive results are presumptive and must be confirmed by FDA- cleared tests or other acceptable reference methods.The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative xTAG® Gastrointestinal Pathogen Panel results in the setting of clinical illness compatible with</td><td>The detection and identification of specific gastrointestinal microbial nucleic acid from individuals exhibiting signs and symptoms of gastrointestinal infection aids in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation, laboratory findings and epidemiological information. A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks. xTAG® GPP positive results are presumptive and must be confirmed by FDA-cleared tests or other acceptable reference methods. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative xTAG Gastrointestinal Pathogen Panel results in the setting of clinical illness</td></tr><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td>Device infection by pathogens that are not</td><td>Predicate gastroenteritis may be due</td></tr><tr><td>Targets Reported</td><td>detected by this test or non- infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease. xTAG GPP is not intended to monitor or guide treatment for C. difficile infections. The xTAG GPP is indicated for use with the Luminex® 100/200™ instrument with xPONENT® software. Adenovirus 40/41, Campylobacter (C.</td><td>to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease. xTAG® GPP is not intended to monitor or guide treatment for C. difficile infections. The xTAG® GPP is indicated for use with the Luminex® 100/200TM instrument.</td></tr><tr><td></td><td>jejuni, C. coli and C. lari only), Clostridium difficile (C. difficile) toxin A/B, Cryptosporidium (C. parvum and C. hominis only), Escherichia coli (E. coli) 0157, Enterotoxigenic Escherichia coli (ETEC) LT/ST, Entamoeba histolytica (E. histolytica), Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis), Norovirus GI/GII, Rotavirus A, Salmonella, Shiga-like Toxin producing E. coli (STEC) stx 1/stx 2, Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae), Vibrio cholerae (V. cholerae) cholera toxin gene (ctx)</td><td>Campylobacter (C. jejuni, C. coli and C. lari only), Clostridium difficile (C. difficile) toxin A/B, Cryptosporidium (C. parvum and C. hominis only), Escherichia coli (E. coli) O157, Enterotoxigenic Escherichia coli (ETEC) LT/ST, Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis), Norovirus GI/GII, Rotavirus A, Salmonella, Shiga-like Toxin producing E. coli (STEC) stx 1/stx 2, Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae)</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance Documents

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Title</td><td colspan="1" rowspan="1">Date</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Establishing the Performance Characteristics of In Vitro DiagnosticDevices for the Detection of Clostridium difficile</td><td colspan="1" rowspan="1">Nov. 29, 2010</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Class II Special Controls Guidance Document: Norovirus SerologicalReagents</td><td colspan="1" rowspan="1">Mar. 9, 2012</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Class II Special Controls Guidance Document: Instrumentation forClinical Multiplex Test Systems - Guidance for Industry and FDA Staff</td><td colspan="1" rowspan="1">Mar. 10, 2005</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Guidance for the Content of Premarket Submissions for SoftwareContained in Medical Devices</td><td colspan="1" rowspan="1">May 11, 2005</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Guidance document for Format for Traditional and Abbreviated 510(k)s</td><td colspan="1" rowspan="1">Aug. 12, 2005</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Guidance on the CDRH Premarket Notification Review Program, 510(k)Memorandum #K86-3</td><td colspan="1" rowspan="1">June 30, 1986</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">The New 510(k) Paradigm - Alternate Approaches to DemonstratingSubstantial Equivalence in Premarket Notifications - Final Guidance</td><td colspan="1" rowspan="1">Mar. 20, 1998</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">The 510(k) Program: Evaluating Substantial Equivalence in PremarketNotifications [510(k)]</td><td colspan="1" rowspan="1">Dec. 27, 2011</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Guidance for Industry and Food and Drug Administration Staff - eCopyProgram for Medical Device Submissions</td><td colspan="1" rowspan="1">Oct. 10, 2013</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Guidance for Industry and Food and Drug Administration Staff - FDAand Industry Actions on Premarket Notification (510(k)) Submissions:Effect on FDA Review Clock and Goals</td><td colspan="1" rowspan="1">Oct. 15, 2012</td></tr></table>

Standards   

<table><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>StandardNo.</td><td rowspan=1 colspan=1>RecognitionNumber(FDA)</td><td rowspan=1 colspan=1>Standards Title</td><td rowspan=1 colspan=1>Date</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>EP05-A2</td><td rowspan=1 colspan=1>7-110</td><td rowspan=1 colspan=1>Evaluation of Precision Performance of Quantitativemeasurement Methods (2nd ed.)</td><td rowspan=1 colspan=1>10/31/2005</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>EP07-A2</td><td rowspan=1 colspan=1>7-127</td><td rowspan=1 colspan=1>Interference Testing in Clinical Chemistry (2ndedition)</td><td rowspan=1 colspan=1>05/21/2007</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>EP12-A2</td><td rowspan=1 colspan=1>7-152</td><td rowspan=1 colspan=1>User Protocol for Evaluation f Qualitative TestPerformance (2nd edition)</td><td rowspan=1 colspan=1>09/09/2008</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>EP14-A2</td><td rowspan=1 colspan=1>7-143</td><td rowspan=1 colspan=1>Evaluation of Matrix Effects (2nd edition)</td><td rowspan=1 colspan=1>03/16/2012</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>EP15-A2</td><td rowspan=1 colspan=1>7-153</td><td rowspan=1 colspan=1>User Verification of Performance for Precision andTrueness (2nd edition)</td><td rowspan=1 colspan=1>09/09/2008</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>EP17-A</td><td rowspan=1 colspan=1>7-194</td><td rowspan=1 colspan=1>Protocol for Determination of Limits of Detectionand Limits of Quantitation(NOTE: Original studies included this standard)</td><td rowspan=1 colspan=1>03/28/2009</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>EP17-A2</td><td rowspan=1 colspan=1>7-233</td><td rowspan=1 colspan=1>Evaluation of Detection Capability for ClinicalLaboratory Measurement Procedures</td><td rowspan=1 colspan=1>01/15/2013</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ISO 14971</td><td rowspan=1 colspan=1>5-40</td><td rowspan=1 colspan=1>Application of Risk Management to Medical Devices</td><td rowspan=1 colspan=1>08/20/2012</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>MM03-A2</td><td rowspan=1 colspan=1>7-132</td><td rowspan=1 colspan=1>Molecular Diagnostic Methods for InfectiousDiseases (2nd edition)</td><td rowspan=1 colspan=1>09/09/2008</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>MM13-A</td><td rowspan=1 colspan=1>7-191</td><td rowspan=1 colspan=1>Collection, Transport, Preparation and Storage ofSpecimens</td><td rowspan=1 colspan=1>03/18/2009</td></tr></table>

# L. Test Principle:

Human stool samples are pretreated and then subjected to nucleic acid extraction. For each sample, $1 0 \mu \mathrm { L }$ of extracted nucleic acid is amplified in a single multiplex RT‐PCR/PCR reaction. Each target or internal control in the sample results in PCR amplimers ranging from 58 to 202 bp (not including the 24‐mer tag). A five $\mu \mathrm { L }$ aliquot of the RT‐PCR product is then added to a hybridization/detection reaction containing bead populations coupled to sequences from the Universal Array ("antitags"), streptavidin, R‐phycoerythrin conjugate. Each Luminex bead population detects a specific microbial target or control through a specific tag/anti‐tag hybridization reaction. Following the incubation of the RT‐PCR products with the xTAG GPP Bead Mix and xTAG Reporter Buffer, the Luminex instrument sorts and reads the hybridization/detection reactions. A signal or median fluorescence intensity (MFI) is generated for each bead population. These fluorescence values are analyzed to establish the presence or absence of bacterial, viral or parasitic targets and/or controls in each sample. A single multiplex reaction identifies all targets.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

Site-to-site reproducibility was assessed for each of the additional targets and for mixed analyte samples (representing co-infected samples). Original study results for the other analytes were presented in submission k121454. Replicates of simulated samples were tested across 3 sites by 2 operators at each site. One exception was made for testing of the Vibrio cholerae samples at Site 3, where due to operator illness the runs for the second operator were performed by two individuals. All sample replicates tested were prepared through serial dilutions of stock material (pretreated negative stool spiked with a pathogen or positive stool) containing a microbial target from the intended use. Each sample replicate assayed in the study contained either a single microbial target or 2 microbial targets detected by xTAG $^ \mathrm { \textregistered }$ GPP in addition to the internal control (bacteriophage MS2). For single analyte samples, dilutions tested fell into 1 of the following 3 categories:

1. High Negative (HN): microbial target concentrations which generate MFI values not lower than $20 \mathrm { - } 3 0 \%$ below the cut-off MFI for the indicated analyte   
2. Low Positive (LP): microbial target concentrations which generated MFI values that were 1-5X the cut-off MFI for the indicated analyte   
3. Moderate Positive (MP): microbial target concentrations which generated MFI values 7-10X the cut-off MFI for the indicated analyte

For those samples prepared to simulate co-infections, one microbial target was present at the LP level defined above and the other at a High Positive (HP) level. HP levels were defined as follows:

High Positive (HP) viral cultures were prepared to a concentration of $1 0 ^ { 5 } \mathrm { P F U / m L }$ $( 1 \bar { 0 ^ { 5 } } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ or higher; High Positive (HP) bacterial cultures were prepared to a concentration of $1 { \bar { 0 } } ^ { 6 } \mathrm { C F U / m L }$ or higher.

Each sample replicate underwent a single pre-treatment and extraction step. All samples were extracted using the NucliSens® EasyMAG™ extraction method. Extracted material was kept frozen at $\mathring { - } 7 0 ^ { \circ } \mathrm { C }$ until testing. A total of 90 replicates were tested for each single analyte and dual analyte sample (3 replicates per run x 5 runs per operator $_ { \textrm { X 2 } }$ operators per site $_ { \textrm { X 3 } }$ sites $= 9 0$ replicates). Reproducibility was assessed both in terms of calls and MFI values.

# Single Analyte Results

For single analyte samples prepared at the MP level, depending on the microbial target, 89/90 $( 9 9 \% )$ to 90/90 $( 1 0 0 \% )$ replicates generated a positive result (after allowable re-runs). For LP dilutions, depending on the microbial target, the correct positive call was made in 85/90 $( 9 4 \% )$ to 90/90 $( 1 0 0 \% )$ replicates tested. For HN dilutions, depending on the target, the correct negative call was generated in as few as 67/90 $( 7 4 \% )$ replicates to as many as 90/90 $( 1 0 0 \% )$ . Greater variability in the HN dilution, compared to the LP and MP dilution, is expected based on the fact that a target is present in these samples at levels sufficient to generate MFI values $20 { - } 3 0 \%$ below the cut-off MFI, and based on the stochastic nature of end-point PCR in the presence of low levels of targeted analytes. Accordingly, percent variability, measured as the coefficient of variation (CV) for MFI values were lowest at the MP dilution and highest at the HN dilution.

# Dual Analyte Results

For dual analyte samples tested for the additional targets, all targets generated a positive call when present as a HP dilution. When present at the LP concentration, 3 of the 4 target combinations tested generated a positive call in 90/90 $( 1 0 0 \% )$ replicates tested. The 4 combinations were:

Rotavirus (HP) / Adenovirus (LP) Adenovirus (HP) / Rotavirus (LP) C. difficile (HP) / Adenovirus (LP) Adenovirus (HP)/ C. difficile (LP)

C. difficile has two probes resulting in a call for this target, (if either is positive, the target is positive). The following was observed for the remaining target present at LP concentration in the sample containing a second target at HP concentration:

1/90 replicates of the C. difficile (HP) /Adenovirus (LP) sample generated a negative call for Adenovirus

Although the C. difficile LP sample was 89/90 for probe 1, probe 2 made all the calls for the LP sample.

Reproducibility of Overall Total Raw Median MFI values for Three New Targets in xTAG GPP after Reruns   

<table><tr><td rowspan=24 colspan=2>Site 1Site 2Site 3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Panel Member ID</td><td rowspan=1 colspan=1>Adenovirus40/41Low Positive</td><td rowspan=1 colspan=1>Adenovirus40/41MediumPositive</td><td rowspan=1 colspan=1>Adenovirus40/41High Negative</td><td rowspan=1 colspan=1>EntamoebahistolyticaLow Positive</td><td rowspan=1 colspan=1>EntamoebahistolyticaMediumPositive</td><td rowspan=1 colspan=1>EntamoebahistolyticaHigh Negative</td><td rowspan=1 colspan=1>VibriocholeraeLow Positive</td><td rowspan=1 colspan=1>VibriocholeraeMediumPositive</td><td rowspan=1 colspan=1>VibriocholeraeHighNegative</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>1.45x101TCID5o/mL</td><td rowspan=1 colspan=1>5.8x101TCID50/mL</td><td rowspan=1 colspan=1>1.81 TCID50/mL</td><td rowspan=1 colspan=1>1.44x101Cells/mL</td><td rowspan=1 colspan=1>5.76x101Cells/mL</td><td rowspan=1 colspan=1>2.25x10-1Cells/mL</td><td rowspan=1 colspan=1>9.37x106CFU/mL</td><td rowspan=1 colspan=1>3.75x107CFU/mL</td><td rowspan=1 colspan=1>5.86x10^}$CFU/mL</td></tr><tr><td rowspan=5 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>22/3073.3%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>25th Percentile MFI</td><td rowspan=1 colspan=1>634.0</td><td rowspan=1 colspan=1>1333.3</td><td rowspan=1 colspan=1>110.5</td><td rowspan=1 colspan=1>573.0</td><td rowspan=1 colspan=1>1270.0</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>579.5</td><td rowspan=1 colspan=1>1206.0</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>678.8</td><td rowspan=1 colspan=1>1392.0</td><td rowspan=1 colspan=1>133.0</td><td rowspan=1 colspan=1>614.8</td><td rowspan=1 colspan=1>1410.3</td><td rowspan=1 colspan=1>41.8</td><td rowspan=1 colspan=1>690.0</td><td rowspan=1 colspan=1>1269.0</td><td rowspan=1 colspan=1>57.3</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>727.5</td><td rowspan=1 colspan=1>1428.0</td><td rowspan=1 colspan=1>151.5</td><td rowspan=1 colspan=1>775.0</td><td rowspan=1 colspan=1>1516.0</td><td rowspan=1 colspan=1>50.5</td><td rowspan=1 colspan=1>763.0</td><td rowspan=1 colspan=1>1364.0</td><td rowspan=1 colspan=1>74.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>12.26</td><td rowspan=1 colspan=1>6.34</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>23.47</td><td rowspan=1 colspan=1>14.51</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20.16</td><td rowspan=1 colspan=1>10.06</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=5 colspan=1>Site 2</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>15/3050%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>21/3070%</td></tr><tr><td rowspan=1 colspan=1>25th Percentile MFI</td><td rowspan=1 colspan=1>770.0</td><td rowspan=1 colspan=1>1623.0</td><td rowspan=1 colspan=1>124.0</td><td rowspan=1 colspan=1>319.0</td><td rowspan=1 colspan=1>1111.0</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>860.5</td><td rowspan=1 colspan=1>1612.5</td><td rowspan=1 colspan=1>62.0</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>929.0</td><td rowspan=1 colspan=1>1706.3</td><td rowspan=1 colspan=1>151.0</td><td rowspan=1 colspan=1>422.5</td><td rowspan=1 colspan=1>1370.5</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>1327.3</td><td rowspan=1 colspan=1>1897.8</td><td rowspan=1 colspan=1>111.5</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>1178.5</td><td rowspan=1 colspan=1>1903.0</td><td rowspan=1 colspan=1>255.5</td><td rowspan=1 colspan=1>659.0</td><td rowspan=1 colspan=1>1661.0</td><td rowspan=1 colspan=1>48.0</td><td rowspan=1 colspan=1>1574.0</td><td rowspan=1 colspan=1>2294.0</td><td rowspan=1 colspan=1>162.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>28.63</td><td rowspan=1 colspan=1>11.45</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>41.15</td><td rowspan=1 colspan=1>25.39</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>45.08</td><td rowspan=1 colspan=1>30.80</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=5 colspan=1>Site 3</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>25/3083.3%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td></tr><tr><td rowspan=1 colspan=1>25th Percentile MFI</td><td rowspan=1 colspan=1>245.0</td><td rowspan=1 colspan=1>618.5</td><td rowspan=1 colspan=1>54.0</td><td rowspan=1 colspan=1>277.0</td><td rowspan=1 colspan=1>749.0</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>270.0</td><td rowspan=1 colspan=1>757.0</td><td rowspan=1 colspan=1>37.0</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>275.3</td><td rowspan=1 colspan=1>715.3</td><td rowspan=1 colspan=1>64.0</td><td rowspan=1 colspan=1>364.3</td><td rowspan=1 colspan=1>986.3</td><td rowspan=1 colspan=1>47.0</td><td rowspan=1 colspan=1>326.3</td><td rowspan=1 colspan=1>924.0</td><td rowspan=1 colspan=1>47.5</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>340.0</td><td rowspan=1 colspan=1>860.0</td><td rowspan=1 colspan=1>77.0</td><td rowspan=1 colspan=1>446.5</td><td rowspan=1 colspan=1>1135.0</td><td rowspan=1 colspan=1>55.0</td><td rowspan=1 colspan=1>522.5</td><td rowspan=1 colspan=1>1031.0</td><td rowspan=1 colspan=1>56.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>24.57*</td><td rowspan=1 colspan=1>41.66*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.84</td><td rowspan=1 colspan=1>37.32</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48.47</td><td rowspan=1 colspan=1>24.88</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Total Agreement withExpected Result</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>67/9074.4%</td><td rowspan=1 colspan=1>85/9094.4%</td><td rowspan=1 colspan=1>89/9098.9%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>80/9088.9%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>64.6%- 82.3%</td><td rowspan=1 colspan=1>87.6%- 97.6%</td><td rowspan=1 colspan=1>94.0%- 99.8%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>80.7%-93.9%</td></tr><tr><td rowspan=1 colspan=1>Overall 25 Percentile MFI</td><td rowspan=1 colspan=1>340.0</td><td rowspan=1 colspan=1>860.0</td><td rowspan=1 colspan=1>72.5</td><td rowspan=1 colspan=1>330.5</td><td rowspan=1 colspan=1>1007.0</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>420.5</td><td rowspan=1 colspan=1>1006.0</td><td rowspan=1 colspan=1>47.0</td></tr><tr><td rowspan=1 colspan=1>Overall Median MFI Value</td><td rowspan=1 colspan=1>672.5</td><td rowspan=1 colspan=1>1394.0</td><td rowspan=1 colspan=1>112.5</td><td rowspan=1 colspan=1>489.3</td><td rowspan=1 colspan=1>1240.5</td><td rowspan=1 colspan=1>43.5</td><td rowspan=1 colspan=1>672.8</td><td rowspan=1 colspan=1>1258.0</td><td rowspan=1 colspan=1>58.5</td></tr><tr><td rowspan=1 colspan=1>Overall 75th Percentile MFI</td><td rowspan=1 colspan=1>824.0</td><td rowspan=1 colspan=1>1659.5</td><td rowspan=1 colspan=1>151.5</td><td rowspan=1 colspan=1>631.0</td><td rowspan=1 colspan=1>1473.5</td><td rowspan=1 colspan=1>51.0</td><td rowspan=1 colspan=1>975.5</td><td rowspan=1 colspan=1>1612.5</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Overall % CV</td><td rowspan=1 colspan=1>51.30*</td><td rowspan=1 colspan=1>35.00*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40.94</td><td rowspan=1 colspan=1>31.45</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>66.46</td><td rowspan=1 colspan=1>42.16</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\* This $\% C V$ value includes the re-run

Reproducibility of Overall Total Raw Median MFI values for Mixed Analytes in xTAG GPP after Reruns   

<table><tr><td rowspan=4 colspan=1></td><td rowspan=3 colspan=1>Panel Member ID</td><td rowspan=1 colspan=2>Rotavirus A Low Positive/Adenovirus 40/41 High Positive</td><td rowspan=1 colspan=2>Rotavirus A High Positive/Adenovirus 40/41 Low Positive</td><td rowspan=1 colspan=3>Adenovirus 40/41 Low Positive/C. difficile High Positive</td><td rowspan=1 colspan=3>Adenovirus 40/41 High Positive/C. difficile Low Positive</td></tr><tr><td rowspan=2 colspan=1>Rotavirus ALow Positive</td><td rowspan=2 colspan=1>Adenovirus40/41 HighPositiv</td><td rowspan=2 colspan=1>Rotavirus AHigh Positive</td><td rowspan=2 colspan=1>Adenovirus40/41Low Positive</td><td rowspan=1 colspan=1>Adenovirus40/41 Low</td><td rowspan=1 colspan=2>C. difficileHigh Positive</td><td rowspan=1 colspan=1>Adenovirus40/41 High</td><td rowspan=1 colspan=2>C. difficileLow Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Probe 1</td><td rowspan=1 colspan=1>Probe 2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Probe 1</td><td rowspan=1 colspan=1>Probe 2</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Indeterminate*</td><td rowspan=1 colspan=1>9.28x10^{2TCID5o/mL</td><td rowspan=1 colspan=1>Indeterminate*</td><td rowspan=1 colspan=1>2.17x101TCID5o/mL</td><td rowspan=1 colspan=1>2.17x101TCID5o0/mL</td><td rowspan=1 colspan=1>6.00x10&#x27;CFU/mL</td><td rowspan=1 colspan=1>6.00x10CFU/mL</td><td rowspan=1 colspan=1>9.28x10^{2TCIDso/mL</td><td rowspan=1 colspan=1>7.50x10CFU/mL</td><td rowspan=1 colspan=1>7.50x10CFU/mL</td></tr><tr><td rowspan=5 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%%</td></tr><tr><td rowspan=1 colspan=1>25t Percentile MFI</td><td rowspan=1 colspan=1>409.0</td><td rowspan=1 colspan=1>1883.0</td><td rowspan=1 colspan=1>1269.0</td><td rowspan=1 colspan=1>466.0</td><td rowspan=1 colspan=1>485.0</td><td rowspan=1 colspan=1>1818.0</td><td rowspan=1 colspan=1>2596.5</td><td rowspan=1 colspan=1>1932.0</td><td rowspan=1 colspan=1>426.5</td><td rowspan=1 colspan=1>1193.5</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>899.0</td><td rowspan=1 colspan=1>1955.5</td><td rowspan=1 colspan=1>1557.0</td><td rowspan=1 colspan=1>512.3</td><td rowspan=1 colspan=1>552.0</td><td rowspan=1 colspan=1>2196.8</td><td rowspan=1 colspan=1>2702.0</td><td rowspan=1 colspan=1>2005.5</td><td rowspan=1 colspan=1>551.5</td><td rowspan=1 colspan=1>1380.5</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>1410.0</td><td rowspan=1 colspan=1>2051.0</td><td rowspan=1 colspan=1>1833.5</td><td rowspan=1 colspan=1>564.0</td><td rowspan=1 colspan=1>639.0</td><td rowspan=1 colspan=1>2409.5</td><td rowspan=1 colspan=1>2766.0</td><td rowspan=1 colspan=1>2213.0</td><td rowspan=1 colspan=1>766.0</td><td rowspan=1 colspan=1>1604.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>73.05</td><td rowspan=1 colspan=1>5.86</td><td rowspan=1 colspan=1>29.42</td><td rowspan=1 colspan=1>17.82</td><td rowspan=1 colspan=1>21.03</td><td rowspan=1 colspan=1>17.79</td><td rowspan=1 colspan=1>6.39</td><td rowspan=1 colspan=1>9.21</td><td rowspan=1 colspan=1>46.02</td><td rowspan=1 colspan=1>25.12</td></tr><tr><td rowspan=5 colspan=1>Site 2</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/301100%</td></tr><tr><td rowspan=1 colspan=1>25th Percentile MFI</td><td rowspan=1 colspan=1>478.5</td><td rowspan=1 colspan=1>2064.5</td><td rowspan=1 colspan=1>1607.5</td><td rowspan=1 colspan=1>413.0</td><td rowspan=1 colspan=1>441.0</td><td rowspan=1 colspan=1>2233.0</td><td rowspan=1 colspan=1>3001.0</td><td rowspan=1 colspan=1>2369.0</td><td rowspan=1 colspan=1>587.0</td><td rowspan=1 colspan=1>1764.5</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>828.0</td><td rowspan=1 colspan=1>2438.5</td><td rowspan=1 colspan=1>1854.8</td><td rowspan=1 colspan=1>510.8</td><td rowspan=1 colspan=1>558.0</td><td rowspan=1 colspan=1>2514.5</td><td rowspan=1 colspan=1>3185.8</td><td rowspan=1 colspan=1>2521.8</td><td rowspan=1 colspan=1>687.5</td><td rowspan=1 colspan=1>1992.5</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>1670.0</td><td rowspan=1 colspan=1>2616.0</td><td rowspan=1 colspan=1>2160.5</td><td rowspan=1 colspan=1>606.0</td><td rowspan=1 colspan=1>720.0</td><td rowspan=1 colspan=1>2705.0</td><td rowspan=1 colspan=1>3394.0</td><td rowspan=1 colspan=1>2709.0</td><td rowspan=1 colspan=1>957.0</td><td rowspan=1 colspan=1>2276.5</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>80.25</td><td rowspan=1 colspan=1>11.69</td><td rowspan=1 colspan=1>26.10</td><td rowspan=1 colspan=1>26.82</td><td rowspan=1 colspan=1>34.32</td><td rowspan=1 colspan=1>19.83</td><td rowspan=1 colspan=1>11.54</td><td rowspan=1 colspan=1>9.97</td><td rowspan=1 colspan=1>45.91</td><td rowspan=1 colspan=1>18.23</td></tr><tr><td rowspan=5 colspan=1>Site 3</td><td rowspan=1 colspan=1>Agreement with ExpectedResult</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>25th Percentile MFI</td><td rowspan=1 colspan=1>430.5</td><td rowspan=1 colspan=1>1452.0</td><td rowspan=1 colspan=1>1093.0</td><td rowspan=1 colspan=1>206.0</td><td rowspan=1 colspan=1>222.0</td><td rowspan=1 colspan=1>1088.0</td><td rowspan=1 colspan=1>2118.0</td><td rowspan=1 colspan=1>1431.0</td><td rowspan=1 colspan=1>274.5</td><td rowspan=1 colspan=1>1088.0</td></tr><tr><td rowspan=1 colspan=1>Median MFI Value</td><td rowspan=1 colspan=1>689.5</td><td rowspan=1 colspan=1>1527.5</td><td rowspan=1 colspan=1>1538.0</td><td rowspan=1 colspan=1>264.5</td><td rowspan=1 colspan=1>235.5</td><td rowspan=1 colspan=1>1510.5</td><td rowspan=1 colspan=1>2404.5</td><td rowspan=1 colspan=1>1550.0</td><td rowspan=1 colspan=1>414.3</td><td rowspan=1 colspan=1>1247.5</td></tr><tr><td rowspan=1 colspan=1>75th Percentile MFI</td><td rowspan=1 colspan=1>1067.0</td><td rowspan=1 colspan=1>1645.0</td><td rowspan=1 colspan=1>1726.0</td><td rowspan=1 colspan=1>292.0</td><td rowspan=1 colspan=1>269.5</td><td rowspan=1 colspan=1>1837.0</td><td rowspan=1 colspan=1>2613.5</td><td rowspan=1 colspan=1>1630.0</td><td rowspan=1 colspan=1>523.0</td><td rowspan=1 colspan=1>1425.0</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>64.25</td><td rowspan=1 colspan=1>11.30</td><td rowspan=1 colspan=1>35.50</td><td rowspan=1 colspan=1>23.76</td><td rowspan=1 colspan=1>22.74</td><td rowspan=1 colspan=1>32.71</td><td rowspan=1 colspan=1>15.68</td><td rowspan=1 colspan=1>13.12</td><td rowspan=1 colspan=1>48.98</td><td rowspan=1 colspan=1>24.47</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>Total Agreement withExpected Result</td><td rowspan=1 colspan=1>90/90 100%</td><td rowspan=1 colspan=1>90/9010%</td><td rowspan=1 colspan=1>90/9010%</td><td rowspan=1 colspan=1>90/9010%</td><td rowspan=1 colspan=1>89/9098.9%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>89/9098.9%</td><td rowspan=1 colspan=1>90/9010%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9%- 100.0%</td><td rowspan=1 colspan=1>95.9%- 100.0%</td><td rowspan=1 colspan=1>95.9%- 100.0%</td><td rowspan=1 colspan=1>95.9%- 100.0%</td><td rowspan=1 colspan=1>94.0%- 99.8%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>94.0%-99.8%</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall 25t Percentile FI</td><td rowspan=1 colspan=1>430.5</td><td rowspan=1 colspan=1>1645.0</td><td rowspan=1 colspan=1>1361.0</td><td rowspan=1 colspan=1>292.0</td><td rowspan=1 colspan=1>269.5</td><td rowspan=1 colspan=1>1648.0</td><td rowspan=1 colspan=1>2483.0</td><td rowspan=1 colspan=1>1630.0</td><td rowspan=1 colspan=1>406.0</td><td rowspan=1 colspan=1>1219.0</td></tr><tr><td rowspan=1 colspan=1>Overall MedianMFI Value</td><td rowspan=1 colspan=1>741.8</td><td rowspan=1 colspan=1>1948.8</td><td rowspan=1 colspan=1>1614.8</td><td rowspan=1 colspan=1>444.3</td><td rowspan=1 colspan=1>447.0</td><td rowspan=1 colspan=1>2122.3</td><td rowspan=1 colspan=1>2706.3</td><td rowspan=1 colspan=1>2004.8</td><td rowspan=1 colspan=1>575.8</td><td rowspan=1 colspan=1>1452.8</td></tr><tr><td rowspan=1 colspan=1>Overall 5 Percentile FI</td><td rowspan=1 colspan=1>1251.5</td><td rowspan=1 colspan=1>2128.5</td><td rowspan=1 colspan=1>1884.5</td><td rowspan=1 colspan=1>543.0</td><td rowspan=1 colspan=1>603.0</td><td rowspan=1 colspan=1>2422.0</td><td rowspan=1 colspan=1>3023.0</td><td rowspan=1 colspan=1>2377.5</td><td rowspan=1 colspan=1>766.0</td><td rowspan=1 colspan=1>1849.0</td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>75.04</td><td rowspan=1 colspan=1>20.54</td><td rowspan=1 colspan=1>32.39</td><td rowspan=1 colspan=1>37.71</td><td rowspan=1 colspan=1>44.38</td><td rowspan=1 colspan=1>30.51</td><td rowspan=1 colspan=1>16.62</td><td rowspan=1 colspan=1>23.01</td><td rowspan=1 colspan=1>53.59</td><td rowspan=1 colspan=1>30.19</td></tr></table>

\*Real-time PCR failed to return a meaningful result. The amount of Rotavirus added to this sample is the same as the amount used in equivalent Rotavirus dilutions used in the Repeatability study.

Overall, adequate site-to-site reproducibility has been established for all targets that xTAG $^ \mathrm { \textregistered }$ GPP has been designed to detect (also see results in k121454).

# Repeatability

As in the original study results presented for k121454, repeatability was assessed for each target by testing 20 replicates of each of two different analyte concentrations: a very low positive sample (at the LoD) and a moderate positive dilution level $( 5 \mathbf { x } - 1 0 \mathbf { x }$ above the cut-off MFI). All replicates for each dilution level were examined starting from sample extraction with the bioMérieux NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ system followed by xTAG $\mathrm { G P P \mathbb { R } }$ in a single run. For each set of 20 replicates, the same operator performed the testing on the same instrument system, using the same lot of extraction kit and xTAG $^ \mathrm { \textregistered }$ GPP reagents. Results of testing were as follows:

Assay Repeatability   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Dilution Level</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>xTAG GPPCalls</td><td rowspan=1 colspan=1>MeanMFIValue</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>Adenovirus40/41</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>5.80x101TCID50/mL</td><td rowspan=1 colspan=1>20 of 20 POS</td><td rowspan=1 colspan=1>1355</td><td rowspan=1 colspan=1>9.22%</td></tr><tr><td rowspan=1 colspan=1>Low Positive/LoD</td><td rowspan=1 colspan=1>1.45x101TCID50/mL</td><td rowspan=1 colspan=1>20 of 20 POS</td><td rowspan=1 colspan=1>548</td><td rowspan=1 colspan=1>34.09%</td></tr><tr><td rowspan=2 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>5.76x101cells/mL</td><td rowspan=1 colspan=1>20 of 20 POS</td><td rowspan=1 colspan=1>889</td><td rowspan=1 colspan=1>7.83%</td></tr><tr><td rowspan=1 colspan=1>Low Positive/LoD</td><td rowspan=1 colspan=1>2.88x101 cells/mL</td><td rowspan=1 colspan=1>20 of 20 POS</td><td rowspan=1 colspan=1>883</td><td rowspan=1 colspan=1>16.96%</td></tr><tr><td rowspan=2 colspan=1>Vibriocholerae</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>4.68x10 CFU/mL</td><td rowspan=1 colspan=1>20 of 20 POS</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>15.91%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.34x106 CFU/mL</td><td rowspan=1 colspan=1>19 of 20 POS</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>23.62%</td></tr></table>

The correct qualitative result was obtained for $\geq 1 9$ of 20 replicates at the low positive level and for 20 of 20 replicates at the moderate positive level for each analyte tested at these concentrations.

b. Linearity/assay reportable range: Not applicable, qualitative assay.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Before using the Luminex system to read samples prepared by the xTAG assay, prepare and calibrate the Luminex instrument system following the procedures in the appropriate system user manual.

Negative Controls - Negative controls are defined as either RNase-free water added to the RT-PCR/PCR step (amplification/detection negative control) or lysis buffer that has undergone the entire assay procedure (pretreatment/extraction/amplification/detection negative control). At least one negative control that underwent extraction process must be included in each batch of specimens run on xTAG GPP. The recommended number of negative controls to be included in a batch is dependent on batch size. For batches of 1-30 samples, one negative control must be included. For batches of 31-61 samples, two negative controls are recommended. For batches of 62-92 samples, three negative controls are recommended. When running multiple negative controls disperse the controls throughout the batch.

NOTE: Users will need to identify all the negative controls (including extraction controls) from the TDAS software before the test data is analyzed. If a negative control has a significant signal detected for an analyte, the TDAS software will generate a ‘no call’ for the samples that were positive for the specific analyte and they will need to be retested.

External Positive Controls - Known strains or positive clinical samples with known results for the targeted viruses, bacteria or parasites should be included in routine quality control procedures ("external controls") as positive controls for the assay. At least one of these external controls are analyte positive controls and should be included with each batch of patient specimens and controls positive for different targets should be rotated from batch to batch. External controls should be prepared, extracted and tested in the same manner as patient samples. Results from external controls should be examined before the results from the patient samples. The interpretation of the correct positive control results is performed by the user and not the data analysis software (TDAS). If a given analyte control does not perform as expected, all results for that analyte in the batch of samples should be examined to determine if a re-run is required. If any unexpected calls occur where one or more analytes with signal exceeding the thresholds are detected in any of the positive controls (i.e. non-specific positive signals) for a given run then samples that were positive for the specific analyte(s) that triggered a control failure will need to be rerun. At least one positive control per PCR run must pass, i.e. all expected calls made in order to report any results from the plate.

Internal Control - Bacteriophage MS2 is the internal control for the assay. This internal positive control is added to each patient specimen prior to extraction. This internal control allows the user to ascertain whether the assay is functioning properly. Failure to generate a PRES (present) call for the MS2 control indicates a failure at the extraction step, and/or the reverse-transcription step, and/or the PCR step, and may be indicative of the presence of amplification inhibitors, which can lead to false negative results.

# d. Detection limit:

As in the original study results presented for k121454, the LoD was assessed by analyzing serial dilutions of simulated samples made from high-titre stocks of commercial strains or high-titre clinical specimens (when commercial strains were not available). All simulated specimens were prepared in negative clinical matrix (stool). The data from serial dilutions were confirmed in at least 20 replicates of the selected dilution for each analyte target. Results of testing for the three additional analytes were as follows:

Summary of Limit of Detection (LoD) for Additional Analytes   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Titre (correspondingto the estimatedLoD)</td><td rowspan=1 colspan=1>AverageMFIValue</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>Adenovirus40/41</td><td rowspan=1 colspan=1>Adenovirus 40, 0810084CF (Dugan)</td><td rowspan=1 colspan=1>1.45x101TCID5o/mL</td><td rowspan=1 colspan=1>548</td><td rowspan=1 colspan=1>34.09%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41, 0810085CF (Tak)</td><td rowspan=1 colspan=1>7.69 TCID5o/mL</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>22.04%</td></tr><tr><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>Entamoeba histolytica, 30890</td><td rowspan=1 colspan=1>2.88x101 cells/mL</td><td rowspan=1 colspan=1>883</td><td rowspan=1 colspan=1>16.96%</td></tr><tr><td rowspan=1 colspan=1>Vibriocholerae</td><td rowspan=1 colspan=1>Vibrio cholerae, 14101 (Serovar0:1)</td><td rowspan=1 colspan=1>2.34x106 CFU/mL</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>23.62%</td></tr></table>

The data summarized above establish a limit of detection for each indicated analyte.

# Stool in Cary Blair Media Limit of Detection Study Results

The purpose of this analytical study was to evaluate the equivalency in the limit of detection (LoD) between the two sample types: raw stool (sample type from k121454) and stool in Cary-Blair transport medium (additional sample type commonly collected) in a representative sub-set of the xTAG GPP targets. One analyte from each of three pathogen classes (bacterial, parasitic, and viral) was examined in the form of simulated stool samples and simulated stool samples in Cary-Blair media. The three representative analytes tested in this study were: Clostridium difficile, Giardia lamblia and Norovirus GII.The simulated samples were prepared as a dilution series using high titre stocks.

In the first part of this study, serial dilution curves for each analyte target were made for both stool and stool in Cary-Blair sample types. These curves were generated by assessing 3 replicates per sample type of each dilution level, starting from the sample extraction step. The dilutions for both sample types were prepared in parallel and analyzed with the xTAG GPP assay on the same plate to minimize variation.

From part 1 of the study, a dilution for each target in each of the sample types was selected for further confirmation testing. Confirmation of LoD was achieved through testing of 20 replicates of the selected dilutions starting from sample extraction. In general, the dilution level corresponding to the lowest concentration of the analyte in which 3/3 replicates generated positive calls by xTAG GPP was selected for LoD confirmation testing for that sample type. LoD was considered as confirmed if the selected dilution level gave POSITIVE calls for ${ \geq } 1 9 / 2 0 $ of the replicates.

Summary of the Limit of Detection (LoD) for GPP analytes in stool and stool in Cary-Blair media   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Strain ID</td><td rowspan=1 colspan=2>Raw Stool</td><td rowspan=1 colspan=2>Stool in Cary-Blair</td><td rowspan=2 colspan=1>LoD Differencebetween Stool andStool in Cary-Blair</td></tr><tr><td rowspan=1 colspan=1>Titre atlimit ofdetection</td><td rowspan=1 colspan=1>AverageMFIValue(n=20)</td><td rowspan=1 colspan=1>Titre atlimit ofdetection</td><td rowspan=1 colspan=1>AverageMFI Value(n=20)</td></tr><tr><td rowspan=1 colspan=1>C.difficileToxinA/B</td><td rowspan=1 colspan=1>Clostridium difficile,BAA-1805(toxinotype Ill A+B+)</td><td rowspan=1 colspan=1>4.69x105CFU/mL</td><td rowspan=1 colspan=1>Probe 1= 383Probe 2= 862</td><td rowspan=1 colspan=1>4.69x105CFU/mL</td><td rowspan=1 colspan=1>Probe 1 =460Probe 2 =1128</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>Giardialamblia,PRA-243</td><td rowspan=1 colspan=1>2.20x10²cells/mL</td><td rowspan=1 colspan=1>1069</td><td rowspan=1 colspan=1>2.20x102cells/mL</td><td rowspan=1 colspan=1>1008</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>NorovirusGII,Clinicalsample,sourceToronto</td><td rowspan=1 colspan=1>4.75x102copies/mL(Ct =32.23)</td><td rowspan=1 colspan=1>1466</td><td rowspan=1 colspan=1>4.75x102copies/mL(Ct =32.23)</td><td rowspan=1 colspan=1>2299</td><td rowspan=1 colspan=1>None</td></tr></table>

The data summarized above demonstrate that raw stool samples and stool samples in Cary-Blair media have equivalent limits of detection.

# e. Analytical specificity:

# Analytical Reactivity

Analytical reactivity was assessed through empirical testing of a wide range of clinically relevant GI pathogen strains, genotypes, serotypes and isolates representing temporal and geographical diversity for each analyte. Through testing of unique samples covering the additional intended use pathogens, reactivity was established at concentrations 2 to 3 times the limit of detection.

Adenovirus - The Limit of Detection (LoD) using Adenovirus 40, Zeptometrix 0810084CF (Dugan) and Adenovirus 41, Zeptometrix 0810085CF (Tak) were found to be $1 . 4 5 \mathrm { E } { + 0 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (or $4 . 8 9 \mathrm { E } { + 0 6 }$ Copies/mL) and $7 . 6 9 \mathrm { E } { + } 0 0$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (or $1 . 4 8 \mathrm { E } { + } 0 7$ Copies/mL), respectively (see LoD section above). The following two samples were tested at the Centers for Disease Control and Prevention (CDC) (Atlanta, Georgia, USA). Note: these samples were different isolates of the strains used in the LoD study. The amount of the viral target DNA for GP-093 and GP-094 was measured by real-time PCR and the $\mathrm { C t }$ values generated were used to calculate the DNA copy number. The lowest reactivity titers for GP-093 and GP-094, were found to be at $3 \mathbf { x }$ and 1x multiple of LoD level, respectively.

Adenovirus Reactivity List   

<table><tr><td rowspan=1 colspan=1>Run Batch ID</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain or Serotype</td><td rowspan=1 colspan=1>ReactivityTitre(Copies/mL)</td><td rowspan=1 colspan=1>Results Summary</td></tr><tr><td rowspan=1 colspan=1>Analytical reactivity_II_LX200</td><td rowspan=1 colspan=1>Adenovirus40</td><td rowspan=1 colspan=1>CDC - GP-093</td><td rowspan=1 colspan=1>DuganpCMK2Gr10, 9/23/91</td><td rowspan=1 colspan=1>1.49E+07</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>Analytical reactivity_II_LX200</td><td rowspan=1 colspan=1>Adenovirus41</td><td rowspan=1 colspan=1>CDC - GP-094</td><td rowspan=1 colspan=1>TakHeLa2Gr10, 9/23/91</td><td rowspan=1 colspan=1>1.43E+07</td><td rowspan=1 colspan=1>POS</td></tr></table>

Furthermore, in sequencing analysis of clinical specimens tested as part of the multisite clinical study of xTAG GPP, 9 Adenovirus 40 and 28 Adenovirus 41 positive samples were detected by the assay and sequencing.

Adenovirus Clinical Specimen Positive by the xTAG GPP   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Clinical Sample ID</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-092B, GPP03-099B, GPP03-101B, GPP03-102B, GPP03-103B, GPP03-106B, GPP03-109B,GPP03-300B, GPP03-240B</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-001B, GPP03-003B, GPP03-007B, GPP03-013B, GPP03-014B, GPP03-019B, GPP03-020B,GPP03-022B, GPP03-025B, GPP03-026B, GPP03-028B, GPP03-029B, GPP03-033B, GPP03-035B,GPP03-036B, GPP03-037B, GPP03-038B, GPP03-039B, GPP03-048B, GPP03-055B, GPP03-060B,GPP03-095B, GPP03-229B, GPP03-313B, GPP04-159, GPP04-174, GPP02-129, GPP02-192</td></tr></table>

Entamoeba histolytica - The LoD for this pathogen was not confirmed prior to this Analytical Reactivity study initiation; thus, serial dilutions of the samples were prepared and tested. The LoD using Entamoeba histolytica, ATCC 30890 was later found to be $2 . 8 8 \mathrm { E + } 0 1 \mathrm { C e l l s / m L }$ , equivalent to $4 . 3 0 \mathrm { E } { + } 0 2 $ Copies/mL (see LoD section above). For E.histolytica, ATCC 50007, 50481, 50738 and 50454, the titer information expressed in Cells/mL could not be obtained. To standardize the quantification units for all E.histolytica strains, in this Analytical Reactivity study the amount of target DNA was measured by real-time PCR and the Ct values generated were used to calculate the DNA copy numbers. The reactivity titers for most of the strains were in the range of $0 . 4 \mathbf { x }$ to $6 . 7 \mathrm { x }$ multiple of LoD level for E.histolytica. The reactivity titer for ATCC 50738 (Rahman) was found to be $0 . 2 \mathbf { x }$ multiple of LoD level.

<table><tr><td rowspan=1 colspan=1>Run Batch ID</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain or Serotype</td><td rowspan=1 colspan=1>Reactivity Titre(Cells orCopies/mL)</td><td rowspan=1 colspan=1>ResultsSummary</td></tr><tr><td rowspan=1 colspan=1>20120216_JF_GPP_Reactivity_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30015</td><td rowspan=1 colspan=1>(HK-9, colonic biopsyfrom adult humanmale with amebicdysentery, Korea);frozen</td><td rowspan=1 colspan=1>2.86E+00 Cells/mLor 1.82E+02Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120216_JF_GPP_Reactivity_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30190</td><td rowspan=1 colspan=1>(HB-301:NIH, fecesfrom adult humanmale with amebicdysentery, Burma,1960); test tube</td><td rowspan=1 colspan=1>1.07E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120216_JF_GPP_Reactivity_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30457</td><td rowspan=1 colspan=1>(HU-21:AMC, colonicbiopsy from malechild with amebicdysentery, LittleRock, AR, 1970); testtube</td><td rowspan=1 colspan=1>1.68E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120216_JF_GPP_Reactivity_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30458</td><td rowspan=1 colspan=1>(200:NIH); frozen</td><td rowspan=1 colspan=1>1.83E+02 Cells/mLor 2.42E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120216_JF_GPP_Reactivity_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30459</td><td rowspan=1 colspan=1>(HM-1:IMSS [ABRM];feces from adulthuman male,asymptomatic cystpasser, England,1972); test tube</td><td rowspan=1 colspan=1>1.83E+02 Cells/mLor 1.10E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120314_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30889</td><td rowspan=1 colspan=1>(H-458:CDC[ATCC30217], fecesfrom human adultfemale with amebicdysentery, Asia (?),(patient in U.S. fortreatment), 1971);test tube</td><td rowspan=1 colspan=1>8.78E+02Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120411_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30923</td><td rowspan=1 colspan=1>(HU-2:MUSC)</td><td rowspan=1 colspan=1>4.98E+02Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120207_JF_GPP_Reactivity</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 30925</td><td rowspan=1 colspan=1>(HU-1:CDC, feces offemale child,asymptomatic, sero-negative cyst passer,Cherokee, NC, 1978)</td><td rowspan=1 colspan=1>1.89E+02Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120411_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 50007</td><td rowspan=1 colspan=1>DKB</td><td rowspan=1 colspan=1>2.88E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120411_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 50481</td><td rowspan=1 colspan=1>SD157</td><td rowspan=1 colspan=1>1.36E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120411_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 50738</td><td rowspan=1 colspan=1>Rahman</td><td rowspan=1 colspan=1>8.90E+01Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>20120411_JF_GPP_React_LX</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>ATCC 50454</td><td rowspan=1 colspan=1>HB-301:NIH</td><td rowspan=1 colspan=1>1.08E+03Copies/mL</td><td rowspan=1 colspan=1>POS</td></tr></table>

Vibrio cholerae - The LoD using Vibrio cholerae Pacini ATCC 14101 (serovar O:1) was found to be $2 . 3 4 \mathrm { E } { + 0 6 }$ CFU/mL. For this Analytical Reactivity study $3 \mathrm { x L o D } { = } 7 . 0 2 \mathrm { E } { + } 0 6 \mathrm { C F U } / \mathrm { m L }$ was used for initial reactivity testing. In addition to toxinogenic strains, (i.e. O1 and O139), the xTAG GPP assay also detects any non:O1 Vibrio cholerae strains that do express ct toxin gene (xTAG GPP Vibrio cholerae primers target gene), but not the non:O1 strains that may cause clinical symptoms such as diarrhea by expressing a different virulence factor, which is likely the case for sample ATCC 14374 and other non:O1 strains in this table. Both non-O1 ATCC 25872 and non-O1 ATCC 25873 strains, were tested in sequencing assays and confirmed to contain the ctx gene with well conserved xTAG GPP Vibrio cholerae primer binding regions.

Vibrio cholerae Reactivity List   

<table><tr><td colspan="1" rowspan="1">Run Batch ID</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Strain or Serotype</td><td colspan="1" rowspan="1">Reactivity Titre(CFU/mL)</td><td colspan="1" rowspan="1">ResultsSummary</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">NCTC 30</td><td colspan="1" rowspan="1">Non-O:1, ATCC4735;MARTIN 1</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 4714</td><td colspan="1" rowspan="1">Non-O:1, Isolatedfrom pilgrims in ElTor quarantine camp,El Tor 34-D 19</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 7260</td><td colspan="1" rowspan="1">0:1, EGYPT 117</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 11500</td><td colspan="1" rowspan="1">Non-O:1, VL 7050</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 11507</td><td colspan="1" rowspan="1">Non-O:1, VL 1941</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 11510</td><td colspan="1" rowspan="1">O:1, VL 01211</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 12945</td><td colspan="1" rowspan="1">0:139 (Non-O:1(NAG) - referencestrain for 0:139serovar</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120827-JX-V cholera-AR-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">NCTC 12946</td><td colspan="1" rowspan="1">0:139 (Non-O:1(NAG))</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120406-JX-AnaReact-Vibrio2-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14033</td><td colspan="1" rowspan="1">0:1, El Tor DO1930;CN 5774;R.Hugh 1092, SerotypeInaba, Non-toxinogenic</td><td colspan="1" rowspan="1">1.50E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraeasiaticae (Trevisan)Pfeiffer</td><td colspan="1" rowspan="1">ATCC 14035</td><td colspan="1" rowspan="1">0:1, Serotype Ogawa[7787]</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14101</td><td colspan="1" rowspan="1">0:1, SerotypeOgawa, clinicalspecimen - human([185754] choleraepidemic circa 1960,Calcutta) CalcuttaIndia</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120406-JX-AnaReact-Vibrio2-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14374</td><td colspan="1" rowspan="1">Non-O:1 (NAG),5035; R. Hugh 1513</td><td colspan="1" rowspan="1">1.50E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120921-MB-VibrioAnalytical-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14730</td><td colspan="1" rowspan="1">Non-O:1 (Serovar0:2), biovar El Tor,Subgroup III of</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gardner andVenkatraman, NCTC4711, NANKING32/123</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">20120921-MB-VibrioAnalytical-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14731</td><td colspan="1" rowspan="1">Non-O:1, (Serovar0:3), biovar El Tor,Subgroup V ofGardner andVenkatraman, NCTC4715, El Tor 34-D23;CN 3426</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120921-MB-VibrioAnalytical-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14732</td><td colspan="1" rowspan="1">Non-O:1 (Serovar0:4), biovar El Tor,Subgroup VI ofGardner andVenkatraman, NCTC4716, KASAULI 73</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120921-MB-VibrioAnalytical-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 14733</td><td colspan="1" rowspan="1">Non-O:1 (Serovar0:7), biovar El Tor,Subgroup II ofGardner andVenkatraman, NCTC8042, NANKING32/124</td><td colspan="1" rowspan="1">6.00E+08</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 25870</td><td colspan="1" rowspan="1">0:1, Serotype Inaba</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 25872</td><td colspan="1" rowspan="1">Non-O:1 (NAG),Isolated from apatient with clinicalcholera</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 25873</td><td colspan="1" rowspan="1">Non-O:1 (NAG),Isolated from apatient with clinicalcholera</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 51394</td><td colspan="1" rowspan="1">0:139 (Non-O:1[NAG]), Cholerapatient, Madras,India</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio choleraePacini</td><td colspan="1" rowspan="1">ATCC 51395</td><td colspan="1" rowspan="1">0:139 (non O:1[NAG]), clinicalspecimen - human(cholera patient,Madras, India)</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">20120404-JX-AnaReact-Vibrio-LX</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">ATCC BAA-2163</td><td colspan="1" rowspan="1">0:1, Isolated from apatient in ArtiboniteDepartment, Haiti,October 2010,Serotype Ogawa,Biogroup El Torcholera toxin positiveCDC Isolate 2010 EL-1786</td><td colspan="1" rowspan="1">7.02E+06</td><td colspan="1" rowspan="1">POS</td></tr></table>

The table below summarizes the samples reactive with xTAG GPP.

Reactivity of Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>ATCC / OtherReference</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>ATCC / OtherReference</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>CDC - GP-093</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-095B</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-092B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-229B</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-099B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-313B</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-101B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP04-159</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-102B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP04-174</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-103B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP02-129</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-106B</td><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP02-192</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-109B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30015</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-240B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30190</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>GPP03-300B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30457</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>CDC - GP-094</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30458</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-001B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30459</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-003B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30889</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-007B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30923</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-013B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30925</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-014B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 50007</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-019B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 50481</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-020B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 50738</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-022B</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 50454</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-025B</td><td rowspan=1 colspan=1>Vibrio cholerae, serovar O:1</td><td rowspan=1 colspan=1>NCTC 7260</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-026B</td><td rowspan=1 colspan=1>Vibrio cholerae, serovar O:1</td><td rowspan=1 colspan=1>NCTC 11510</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-028B</td><td rowspan=1 colspan=1>Vibrio cholerae, serovar O:139 (Non-O:1(NAG)) - reference strain for O:139 serovar</td><td rowspan=1 colspan=1>NCTC 12945</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-029B</td><td rowspan=1 colspan=1>Vibrio cholerae, serovar O:139 (Non-O:1(NAG))</td><td rowspan=1 colspan=1>NCTC 12946</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-033B</td><td rowspan=1 colspan=1>Vibrio cholerae asiaticae (Trevisan) Pfeiffer,serovar O:1, serotype Ogawa</td><td rowspan=1 colspan=1>ATCC 14035</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-035B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar O:1, SerotypeOgawa</td><td rowspan=1 colspan=1>ATCC 14101</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-036B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar O:1, SerotypeInaba</td><td rowspan=1 colspan=1>ATCC 25870</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-037B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar Non-O:1(NAG)</td><td rowspan=1 colspan=1>ATCC 25872</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-038B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar Non-O:1(NAG)</td><td rowspan=1 colspan=1>ATCC 25873</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-039B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar O:139 (Non-0:1 [NAG])</td><td rowspan=1 colspan=1>ATCC 51394</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-048B</td><td rowspan=1 colspan=1>Vibrio cholerae Pacini, serovar O:139 (Non-0:1 [NAG])</td><td rowspan=1 colspan=1>ATCC 51395</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-055B</td><td rowspan=1 colspan=1>Vibrio cholera, serovar O:1, serotype Ogawa,biovar El Tor, cholera toxin positive</td><td rowspan=1 colspan=1>ATCC BAA-2163</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>GPP03-060B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Vibrio cholerae strains that did not react with xTAG GPP   

<table><tr><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">ATCC / OtherReference</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">ATCC / OtherReference</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae Pacini, SerovarNon-O:1 (NAG)</td><td colspan="1" rowspan="1">NCTC 30</td><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar Non-O:1 (NAG)</td><td colspan="1" rowspan="1">ATCC 14374</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae, Serovar Non-O:1</td><td colspan="1" rowspan="1">NCTC 4714</td><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar O:2, biovar ElTor, Subgroup III of Gardner andVenkatraman</td><td colspan="1" rowspan="1">ATCC 14730</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae, Serovar Non-O:1</td><td colspan="1" rowspan="1">NCTC 11500</td><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar O:3, biovar</td><td colspan="1" rowspan="1">ATCC 14731</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ElTor, Subgroup V of Gardner andVenkatraman</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae, Serovar Non-O:1</td><td colspan="1" rowspan="1">NCTC 11507</td><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar O:4, biovar ElTor; Subgroup VI of Gardner andVenkatraman</td><td colspan="1" rowspan="1">ATCC 14732</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar O1,biotype El Tor, serotype Inaba,non-toxinogenic</td><td colspan="1" rowspan="1">ATCC 14033</td><td colspan="1" rowspan="1">Vibrio cholerae Pacini, Serovar O:7, biovar ElTor; Subgroup II of Gardner and Venkatraman</td><td colspan="1" rowspan="1">ATCC 14733</td></tr></table>

# Analytical Specificity and Potential Interfering Agents

Analytical specificity was assessed with respect to the following:

1. Propensity for cross-reactivity leading to false positive results: Potential cross reactivity with pathogens (viruses, bacteria and parasites) associated with gastrointestinal (GI) infections that are not probed by the assay. Potential cross reactivity was also assessed for commensal flora and non-microbial agents. Organisms were tested at high positive titres.

2. Propensity for interference leading to false negative results: Potential interference by pathogens (viruses, bacteria and parasites) associated with gastrointestinal (GI) infections that are not probed by the assay. Potential interference by commensal flora was also assessed. Panel analytes were tested at low positive concentrations in the presence of highly concentrated non-panel organisms.

3. Propensity for competitive interference leading to false negative results: Potential interference by GI pathogens that are detected by the assay was evaluated by testing one microbial target prepared at a concentration near the assay cut-off (LP) in the presence of a second microbial target prepared at a very high concentration (HP), and vice-versa. The combinations of analytes tested were selected based on the frequency of co-infections reported in the literature.

Results for the 3 categories of testing outlined above were detailed in the decision summary presented for submission k12454 which are still applicable for the additional 3 analytes.

The following additions relevant to results for the additional 3 analytes are presented below:

Astrovirus was used as a representative interfering pathogen associated with gastrointestinal (GI) infections that are not probed by the assay (See table below). The xTAG GPP analyte, in this case Adenovirus 40/41, was also run without a second analyte present. No interference was seen.

Non-panel interference with common commensal bacteria, yeast and parasites was evaluated for each target in the xTAG GPP assay. Low positive samples of each analyte target in the assay were tested in the presence of a high positive sample of the potential interfering microorganism. All non-panel bacteria and yeast were tested at a concentration of $6 \times 1 0 ^ { \wedge } 8$ cfu/mL except for Blastocystis hominis (ATCC 50587 - concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { \land } 6$ cells/mL and ATCC 50608 - concentration $2 . 0 0 \times 1 0 ^ { \wedge } 7$ cells/mL). There was no interference found with the xTAG GPP analytes Adenovirus, Entamoeba histolytica and Vibrio cholera.

However, cross-reactivity was observed with a false positive call for one Entamoeba dispar strain. PRA‐353, tested at $3 . 0 \mathrm { E } { + } 0 5$ cells/mL (highest available stock concentration), produced a low positive call for $E$ . histolytica with the average MFI of 419.5. Tested at a fourfold dilution of the stock $( 7 . 5 \mathrm { E } + 0 4 \mathrm { c e l l s } / \mathrm { m L } )$ , this strain was negative for $E$ . histolytica (average MFI 149.8) and all other GPP analytes. In addition, five different $E$ . dispar strains (including PRA-353) were sequenced at LMD with primers flanking the xTAG GPP kit $E$ . histolytica primer binding region. All five of the $E .$ . dispar sequences were identical in the $E$ . histolytica GPP kit amplicon region. The forward primer was a perfect match to the $E$ . dispar sequences, whereas the reverse primer had multiple mismatches, most notably, a 2‐nt contiguous mismatch on the $3 ^ { \circ }$ end. These mismatches in the reverse primer would likely cause a significant decrease in amplification efficiency, and, therefore, result in a low risk of obtaining a false‐positive xTAG GPP result for $E$ . histolytica. However, as the xTAG GPP testing demonstrated, a false‐positive call is possible when $E$ . dispar is present at a very high concentration, $3 . 0 \mathrm { E } { + } 0 5$ cells/mL $\mathrm { o r } > 1 0 ^ { 4 }$ times LoD for $E$ . histolytica) or higher. Therefore, this information will be included in product labeling.

Interference with Non-Panel Gastrointestinal Pathogens   

<table><tr><td rowspan=1 colspan=1>xTAG GPP Analyte (concentration)</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Potentially InterferingOrganism (concentration)</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>InterferenceYes (Y) /No (N)</td></tr><tr><td rowspan=2 colspan=1>Adenovirus serotypes 40 (LP)(1.49 x 10^7 copies/mL)</td><td rowspan=2 colspan=1>CDC</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Astrovirus (High-titer)(6.00 x 10^10 copies/mL)</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=2 colspan=1>Adenovirus serotypes 41 (LP)(1.43 x 10^7 copies/mL)</td><td rowspan=2 colspan=1>CDC</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Astrovirus (High-titer)(6.00 x 10^10 copies/mL)</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N</td></tr></table>

Common commensal bacteria, yeast and parasites tested for interference   

<table><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y) / No (N)</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectivat</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 17906</td><td colspan="1" rowspan="1">1.64 x 10^7 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Actinomyces naeslundii</td><td colspan="1" rowspan="1">ATCC 12104</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Akkermansia muciniphila</td><td colspan="1" rowspan="1">ATCC BAA-835</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y)/ No (N)</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">ATCC 35098</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Atopobium vaginae</td><td colspan="1" rowspan="1">ATCC BAA-55</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacillus btilis ubs.</td><td colspan="1" rowspan="1">ATCC 6633</td><td colspan="1" rowspan="1">1.9 x 10^7 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacillus ubtilis subsp. sutilis</td><td colspan="1" rowspan="1">ATCC 6051</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">ATCC 43185</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">ATCC 43183</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacteroides thetaiotaomicron</td><td colspan="1" rowspan="1">ATCC 29148</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bacteroides vulgatus</td><td colspan="1" rowspan="1">ATCC 8482</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">ATCC 15703</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifidum</td><td colspan="1" rowspan="1">ATCC 29521</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longum subsp.longum</td><td colspan="1" rowspan="1">ATCC 15707</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">ATCC 50587</td><td colspan="1" rowspan="1">≥ 10^6 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">ATCC 50608</td><td colspan="1" rowspan="1">2 x 10^7 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">ATCC 33237</td><td colspan="1" rowspan="1">3.11 x 10^5 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">ATCC 35224</td><td colspan="1" rowspan="1">1.71 x 10^5 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">ATCC 33236</td><td colspan="1" rowspan="1">1.41 x 10^5 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">ATCC 51209</td><td colspan="1" rowspan="1">4.64 x 10^7 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hominis</td><td colspan="1" rowspan="1">ATCC BAA-381</td><td colspan="1" rowspan="1">6.61 x 10^3 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">ATCC 33238</td><td colspan="1" rowspan="1">1.18 x 10^5 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">ATCC 51146</td><td colspan="1" rowspan="1">2.49 x 10^3 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Campylobacter sputorum biovarsputorum</td><td colspan="1" rowspan="1">ATCC 35980</td><td colspan="1" rowspan="1">1.56 x 10^6 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Candida catenulata</td><td colspan="1" rowspan="1">ATCC 10565</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Capnocytophaga gingivalis</td><td colspan="1" rowspan="1">ATCC 33624</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Chryseobacterium gleum</td><td colspan="1" rowspan="1">ATCC 35910</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Zeptometrix0801718</td><td colspan="1" rowspan="1">1.35 x 10^10 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">1.3 x 10^8bacteria/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReeference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y)/ No (N)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27028</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">$N^}$</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ATCC 51115</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">ATCC 8260</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">ATCC 638</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">ATCC BAA-613</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">ATCC 19398</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium chauvoei</td><td colspan="1" rowspan="1">ATCC 11957</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43593</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43601</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 700057</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium fallax</td><td colspan="1" rowspan="1">ATCC 19400</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">ATC 9650</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">ATCC 19401</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">ATCC 14501</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">ATCC 43829</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">ATCC 27757</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">ATCC 3540</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium paraputrificum</td><td colspan="1" rowspan="1">ATCC 25780</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">ATCC 25582</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">ATCC 35704</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium sphenoides</td><td colspan="1" rowspan="1">ATCC 19403</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">ATCC 3584</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Clostridium symbiosum</td><td colspan="1" rowspan="1">ATCC 14940</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">ATCC 33030</td><td colspan="1" rowspan="1">3.53 x 10^7 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium glutamicum</td><td colspan="1" rowspan="1">ATCC 13032</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">ATCC 29098</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">ci (strai:(Migula)Castellaniand Chalmers) strain Crooks</td><td colspan="1" rowspan="1">ATCC 8739</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">E. coli (strain: (Migula) Castellaniand Chalmers) serotype026:K60(B6)</td><td colspan="1" rowspan="1">ATCC 12795</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1"> cli (strai:(Migula) Castellaniand Chalmers) O Group 26</td><td colspan="1" rowspan="1">ATCC 11840</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y) / No (N)</td></tr><tr><td colspan="1" rowspan="1">E coli (strain: (Migula) Castellaniand Chalmers) serotype 0103:K:H8</td><td colspan="1" rowspan="1">ATCC 23982</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1"> ci (strai:(Migula)Castellaniand Chalmers) serotype 0111:NM</td><td colspan="1" rowspan="1">Zeptometrix0801747</td><td colspan="1" rowspan="1">1.05 x 10^10 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">E. coli (strain: (Migula) Castellaniand Chalmers) - feces, human (fecesfrom a healthy human), strainHGH21</td><td colspan="1" rowspan="1">ATCC BAA-97</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1"> ci (strai:(Migula)Castellaniand Chalmers) - adult, humanNewYork, strain ECOR2</td><td colspan="1" rowspan="1">ATCC 35321</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1"> ci (strai:(Migula)Castellaniand Chalmers) - adult, humanSweden, ECOR 9 (reference strain)</td><td colspan="1" rowspan="1">ATCC 35328</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">E. coli (strain: (Migula) Castellaniand Chalmers) - adult, humanTonga, ECOR 41 (reference strain)</td><td colspan="1" rowspan="1">ATCC 35360</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">ATCC 25559</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Entamoeba dispar</td><td colspan="1" rowspan="1">ATCC PRA-260</td><td colspan="1" rowspan="1">6.80 x 10^6 copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Entamoeba dispar</td><td colspan="1" rowspan="1">ATCC PRA-353</td><td colspan="1" rowspan="1">3.00 x 10^5 cells/mL</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Entamoeba dispar</td><td colspan="1" rowspan="1">ATCC PRA-353</td><td colspan="1" rowspan="1">7.50 x 10^4 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Entamoeba dispar</td><td colspan="1" rowspan="1">ATCC PRA-368</td><td colspan="1" rowspan="1">7.00 x 10^4 cells/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Entamoeba moshkovskii</td><td colspan="1" rowspan="1">ATCC 50004</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 35028</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp. cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ATCC 25788</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">ATCC 43198</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">ATCC 51266</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 19433</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">1.1 x 10^9bacteria/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 19434</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium vanA</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 49573</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 8043</td><td colspan="1" rowspan="1">5.8 x 10^9 bacteria/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y)/ No (N)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49427</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Eubacterium rectale</td><td colspan="1" rowspan="1">ATCC 33656</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Faecalibacterium prausnitzii(formerly Fusobacterium prausnitzii)</td><td colspan="1" rowspan="1">ATCC 27766</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">ATCC 8501</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 13337</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Helicobacter fennelliae</td><td colspan="1" rowspan="1">ATCC 35683</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Homo sapiens</td><td colspan="1" rowspan="1">ATCC MGC-15492</td><td colspan="1" rowspan="1">Titer not available;used from stock</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp.pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">ATCC 393</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">ATCC 23272</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis subsp. lactis</td><td colspan="1" rowspan="1">ATCC 11454</td><td colspan="1" rowspan="1">9 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">ATCC 33999</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">ATCC 33090</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma fermentans</td><td colspan="1" rowspan="1">ATCC 19989</td><td colspan="1" rowspan="1">Titer not available;used from stock</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td><td colspan="1" rowspan="1">ATCC 14963</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas levii</td><td colspan="1" rowspan="1">ATCC 29147</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td><td colspan="1" rowspan="1">3.2 x 10^7bacteria/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 4630</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35198</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ATCC 47054</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Ruminococcus bromii</td><td colspan="1" rowspan="1">ATCC 27255</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Salmonella subterranea**</td><td colspan="1" rowspan="1">ATCC BAA-836</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">ywith Shigella</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus subsp.aureus strain FDA 209</td><td colspan="1" rowspan="1">ATCC 6538</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Commensal Flora</td><td colspan="1" rowspan="1">ATCC/OtherReference</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-Reactive Yes(Y) / No (N)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus subsp.aureus, Cowan's serotype 1(contains a protein A)</td><td colspan="1" rowspan="1">ATCC 12598</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 7073</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sp.</td><td colspan="1" rowspan="1">ATCC 12973</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">ATCC 19436</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Trabulsiella guamensis</td><td colspan="1" rowspan="1">ATCC 49490</td><td colspan="1" rowspan="1">1.84 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Veillonella atypica</td><td colspan="1" rowspan="1">ATCC 12641</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">ATCC 10790</td><td colspan="1" rowspan="1">6 x 10^8 cfu/mL</td><td colspan="1" rowspan="1">N</td></tr></table>

Note: Streptococcus faecalis is another name for Enterococcus faecalis. Therefore, only one of the two (Enterococcus faecalis) were tested. $\dagger$ Added following release of the C. difficile FDA guidance document Nov. 29, 2010. \*\*Salmonella subterranea is closely related to Escherichia hermanii and does not belong to the genus Salmonella. ^ One of eight replicates cross-reacted with Shigella . ¥ As these targets are not part of the three analytes’ performance description included in this decision summary, for details of the cross-reactivity refer to the original k121454.

Potential interference with GI pathogens that are a part of the assay (competitive interference) was evaluated with one target prepared at a concentration near the assay cut-off (LP) and the other target prepared at a very high concentration (HP) and vice versa. In each case, xTAG GPP Analyte 1 was also run without a second analyte present. Results (interference in making the appropriate calls) are shown in the table below. There was no competitive interference observed between pathogens probed by xTAG GPP when testing was carried out with the mixed analyte samples described below.

Competitive Interference with Panel Pathogens   

<table><tr><td colspan="1" rowspan="1">xTAG GPP Analyte #1</td><td colspan="1" rowspan="1">xTAG GPP Analyte #2</td></tr><tr><td colspan="1" rowspan="4">Adenovirus serotype 40(HP)(3.80 x 10^6 TCID5o/mL)</td><td colspan="1" rowspan="1">No Analyte #2</td></tr><tr><td colspan="1" rowspan="1">Norovirus (LP)(160x dilution of stock)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovar Typhimurium (LP)(8.78 x 10^4 cfu/mL)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni (LP)(2.93 x 10^5 cfu/mL)</td></tr><tr><td colspan="1" rowspan="3">Adenovirus serotype 40(LP)(5.25 x 10^1 TCID5/mL)</td><td colspan="1" rowspan="1">No Analyte #2</td></tr><tr><td colspan="1" rowspan="1">Norovirus (HP)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovar Typhimurium (HP)</td></tr><tr><td rowspan="3"></td><td>(6.00 x 10^8 cfu/mL)</td></tr><tr><td>Campylobacter jejuni (HP)</td></tr><tr><td>(6.00 x 10^8 cfu/mL)</td></tr></table>

The pathogens listed in the table below were not attainable. However, an in silico analysis was performed to assess the potential for non-specific cross-reactivity of these microbial pathogens with the primers used in xTAG GPP (BLAST results located in the design history file). These pathogens do not exhibit sufficient sequence homology against the xTAG GPP primer sequences, and therefore would not be expected to cross-react with the exception of Entamoeba coli and Taenia saginata.

In silico evaluation of pathogens for potential cross‐reactivity   

<table><tr><td rowspan=1 colspan=1>Pathogen</td></tr><tr><td rowspan=1 colspan=1>Ascaris lumbricoides (roundworm)</td></tr><tr><td rowspan=1 colspan=1>Chilomastix mesnili</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium canis</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium felis</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td></tr><tr><td rowspan=1 colspan=1>DF-3 - Dysgonomonas capnocytophagoides</td></tr><tr><td rowspan=1 colspan=1>Dientamoeba fragilis</td></tr><tr><td rowspan=1 colspan=1>Diphyllobothrium species</td></tr><tr><td rowspan=1 colspan=1>Endolimax nana</td></tr><tr><td rowspan=1 colspan=1>Entamoeba coli</td></tr><tr><td rowspan=1 colspan=1>Entamoeba hartmanni</td></tr><tr><td rowspan=1 colspan=1>Entamoeba polecki</td></tr><tr><td rowspan=1 colspan=1>Enterobius vermicularis (pinworm)</td></tr><tr><td rowspan=1 colspan=1>Enteromonas hominis</td></tr><tr><td rowspan=1 colspan=1>Hymenolepis nana (the dwarf tapeworm)</td></tr><tr><td rowspan=1 colspan=1>Idamoeba buetschlii</td></tr><tr><td rowspan=1 colspan=1>Isospora belli</td></tr><tr><td rowspan=1 colspan=1>Strongyloides stercoralis</td></tr><tr><td rowspan=1 colspan=1>Taenia sp.</td></tr><tr><td rowspan=1 colspan=1>Trichuris trichiura</td></tr></table>

From the in silico analysis, Entamoeba coli may cross-react with xTAG GPP primers based on the strong forward primer alignment of E_histolytica-FR_RVM77 (16 bp contig. on the $3 ^ { \circ }$ end) and reverse primer E_coli_stx1-Rev_Biosg_2 (10 bp contig. on the $3 ^ { \circ }$ end), as well as an amplimer size $( 1 3 8 \mathrm { b p } )$ which is well within the design of the kit. To further elucidate, a thermal melting temperature (Tm) analysis was performed using the DINAMelt (Di-Nucleic Acid hybridization and melting prediction) program available at http://mfold.rna.albany.edu/?q=DINAMelt. Sequences of Entamoeba coli that aligned to the xTAG primers were analyzed to see if they would form a stable interaction with the xTAG primers which could possibly result in cross reactivity with the xTAG GPP kit. Mismatches would negatively impact the Tm of the primers and Entamoeba coli. At the xTAG GPP reaction temperature of $5 8 \mathrm { { } ^ { \circ } C }$ , the Entamoeba coli sequences would bind to the $E$ . histolytica forward primer with approximately $6 4 . 4 \%$ of the Entamoeba coli sequences bound to the primer sequence, compared to binding of the forward primer to its target sequence without any mismatches $( 9 8 . 3 \% )$ . However, binding of the reverse $E .$ . coli stx1 primer to Entamoeba coli would be reduced to $0 . 1 \%$ compared to this primer binding to its target sequence without any mismatches $( 8 1 . 8 \% )$ . Therefore, Entamoeba coli is not likely to cross-react with the analytes in the xTAG GPP assay.

# Carry-over Contamination

The likelihood of carry-over contamination events was initially assessed and presented in k121454 by testing 2 representative pathogens (a bacteria and a parasite): C. difficile, and Giardia respectively. In this study, a representative virus (Adenovirus 40) was tested. This analyte was examined in the form of simulated samples prepared at concentrations just below the assay cut-off (High Negative, HN) and well above the assay cut-off (High Positive, HP). The target was examined in a set of 6 independent extractions. Each extraction was assayed in duplicate arranged in a checkerboard manner on a 96-well plate using xTAG GPP.

As with the results in k121454 for the representative bacteria (C. difficile) and parasite (Giardia), results with the virus (Adenovirus 40) showed that all 144 high negative samples remained negative when run on the Luminex $^ \mathrm { \textregistered }$ $1 0 0 / 2 0 0 ^ { \mathrm { { T M } } }$ instrument for all three targets ( $100 \%$ HN). In addition, results for Adenovirus 40 showed that all 144 high positive samples remained positive when run on the Luminex $^ \mathrm { \textregistered }$ $1 0 0 / 2 0 0 ^ { \mathbf { T M } }$ instrument ( $100 \%$ HP), as with the targets previously tested. Therefore a lack of carryover contamination has been demonstrated.

# f. Assay cut-off:

The description of the cut-off determination process was initially presented in k121454. The table below details the final cutoff values selected for each of the 3 additional targets probed by the xTAG GPP assay.

xTAG GPP Additional Analyte Cutoff Values   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Final Cut-off (MFI) for LX 100/200</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>≥ 150 (POS)</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>≥250 (POS)</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>≥ 150 (POS)</td></tr></table>

# Fresh vs. Frozen

The purpose of this evaluation was to generate data to support the hypothesis that no significant difference in the performance of xTAG GPP would be observed between specimens tested from the “fresh” state (i.e., unfrozen) and specimens that were tested after being stored frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ to $- 8 0 ^ { \circ } \mathrm { C }$ . The results of this study will be used to support (or reject) the inclusion of frozen clinical specimens in the multi-site method comparison clinical evaluation of xTAG GPP. The description of this evaluation was initially presented in k121454. The tables below detail the results for each of the 3 additional targets, Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae probed by the xTAG GPP assay.

# One Month Stability Results

Positive agreement between fresh and frozen un-extracted specimens was $\ge 9 5 \%$ with a lower bound of the $9 5 \%$ (two-sided) confidence interval exceeding $8 5 \%$ for Adenovirus 40/41 and Vibrio cholerae.

Positive agreement between fresh and frozen pre-treated specimens was $\ge 9 5 \%$ with a lower bound of the $9 5 \%$ (two-sided) confidence interval exceeding $8 5 \%$ for Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae.

Positive agreement between fresh and frozen extracted specimens was $\ge 9 5 \%$ with a lower bound of the $9 5 \%$ (two-sided) confidence interval exceeding $8 5 \%$ for Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae.

Adenovirus 40/41 and Vibrio cholerae met the 1-month stability acceptance criteria, and the MFIs generated on HP, MP and LP replicates of frozen un-extracted, extracted and extracted specimens were generally close to those generated at baseline. However, the un-extracted specimen stability of Entamoeba histolytica did not meet the acceptance criteria.

# Three Month Stability Results

Positive agreement between fresh and frozen un-extracted specimens was $\ge 9 5 \%$ with a lower bound of the $9 5 \%$ (two-sided) confidence interval exceeding $8 5 \%$ for Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae.

Positive agreement between fresh and frozen extracted specimens was $\ge 9 5 \%$ with a lower bound of the $9 5 \%$ (two-sided) confidence interval exceeding $8 5 \%$ for

Adenovirus 40/41 and Vibrio cholerae.

The 3-month stability results for Entamoeba histolytica are of particular interest as they do not reflect the 1-month stability results. That is study criteria were met for the un-extracted specimen at 3-month stability time point but not at the 1-month time point. The 3-month stability data supports the stability of un-extracted Entamoeba histolytica frozen at $- 7 0 ^ { \circ } \mathrm { C }$ to $- 8 0 \mathrm { { } ^ { \circ } C }$ for 1 month. Study criteria for Entamoeba histolytica nucleic acid stability were met at the 1-month time point but not at the 3- month time point. Overall, the data supports the stability of un-extracted and extracted Entamoeba histolytica specimens frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ to ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for 1 month.

Supplemental Stability Results - Entamoeba histolytica (un-extracted) Additional data to support the stability of un-extracted Entamoeba histolytica specimens was also generated by analyzing LP and MP results obtained at site 1 (LMD) during the multi-site reproducibility study as well as testing LP and MP remnants at a later date. These results also suggest that un-extracted Entamoeba histolytica specimens are stable for at least 1-month when stored frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ to - $8 0 \mathrm { { } ^ { \circ } C }$ .

Results are summarized for the un-extracted, pre-treated and extracted sample stability for the additional analytes in the following table.

Summary of Stability Results Additional Analytes xTAG GPP (also see k121454)   

<table><tr><td rowspan=1 colspan=1>Analyte Target</td><td rowspan=1 colspan=1>Un-extracted1 month</td><td rowspan=1 colspan=1>Un-extracted3 months</td><td rowspan=1 colspan=1>Pre-Treated1 month</td><td rowspan=1 colspan=1>Extracted1 month</td><td rowspan=1 colspan=1>Extracted3 months</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>√</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>√</td><td rowspan=1 colspan=1>√</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>$\^r$</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>√</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1>v</td></tr></table>

^Based on supplemental testing results, possible titer or extraction issue with sample rather than stability failure

The results generated support the inclusion of frozen clinical specimens positive for all three targets, Adenovirus 40/41, Entamoeba histolytica and Vibrio cholerae, in the multi-site clinical evaluation of the xTAG GPP.

# Comparator Assays Analytical Validation Studies

PCR followed by bi-directional sequencing assays (PCR/sequencing) are used as a comparator method and to resolve discordant results to establish analyte identity during the clinical evaluation of xTAG assays. They are validated to evaluate certain performance characteristics including analytical sensitivity (limit of detection), analytical reactivity and specificity (cross-reactivity).

The primers were chosen to perform sequencing as a comparator method for Adenovirus 40/41 and Entamoeba histolytica targets of the xTAG Gastrointestinal Pathogen Panel (xTAG GPP). The comparator assays analytical validation of the primers for Campylobacter, Enterotoxigenic Escherichia Coli (ETEC) LT and ST, and Rotavirus A targets is described in k121454.

To the extent possible, the sequencing primers were designed to amplify regions of the genomic sequence that are not covered by the xTAG GPP kit primers. The second set of sequencing primers designed for ETEC LT and ETEC ST targets were designed to flank the GPP kit amplicon. Bi-directional (both forward and reverse sequences of the produced amplicon) Sanger dideoxy - sequencing method and BLAST analysis were used to confirm sequence identity.

Sequencing primers were validated using samples from the following sources:

1. Representative Clinical Sample: Wherever possible, known positive clinical samples were tested with the sequencing primers to evaluate detection from an extracted clinical stool sample.

2. Limit of Detection (LoD): Serial dilutions of the target analytes were tested to establish the lower limit of primer sensitivity. Samples tested for “Evaluation of the Limit of Detection and Repeatability of xTAG Gastrointestinal Pathogen Panel (FDA),” study were used here.

3. Cross-reactivity: For the xTAG GPP panel targets, samples representing all the targtes in the xTAG GPP panel, were tested at the highest available titres. For the xTAG GPP non-panel cross-reactivity targets, BLAST analysis was performed with each sequencing primer. If both the forward and reverse sets contained an 11 base pair match up to the 3' end (Kwok S, 1994) of the primer with any of the non-panel cross-reactivity species, then a representative sample for that strain was tested to evaluate crossreactivity.

4. Reactivity: Various strains for each target were analyzed to evaluate the strain coverage of the sequencing primers. Samples tested for “Evaluation of Analytical Reactivity of the xTAG Gastrointestinal Pathogen Panel (FDA)” study were used here.

Detailed descriptions of the types of samples tested are listed below:

Clinical Sample: Pre-characterized target-specific clinical samples for Adenovirus were tested with the sequencing primers. For Entamoeba histolytica clinical samples could not be identified; therefore, the evaluation of the primers with a clinical sample was not possible.

Limit of Detection Study: The same sample sets prepared for the Evaluation of the Limit of Detection and Repeatability of xTAG GPP study, were used for this Sequencing Primer Validation study. Briefly, stock solutions were diluted to a starting concentration and dilution series were prepared by making sequential 4-fold dilutions to about 10 dilution levels. Sample dilutions were prepared and tested in triplicates.

· Cross-reactivity: To test for cross-reactivity of the sequencing primers the following studies were conducted.

o For the xTAG GPP panel targets, samples representing all the targets in the xTAG GPP panel, were tested at the highest available titers, except for Vibrio, where the sample was not available.   
o For the xTAG GPP non-panel cross-reactivity targets, BLAST analysis was performed with each sequencing primer. If both the forward and reverse sets contained an 11 base pair match up to the $3 ^ { \circ }$ end (Kwok S, 1994) of the primer with any of the non-panel cross-reactivity species, then a representative sample for that strain was tested to evaluate crossreactivity.

Reactivity: A variety of strains for Entamoeba histolytica used in the Analytical Reactivity study were tested with each sequencing primer set. For Adenovirus, no additional strains to those used in the LoD study could be sourced.

# Categorizing Sequencing Results

Positive – Samples were considered positive by sequencing if the following criteria were met:

o The generated sequences, from bidirectional sequencing, should be at least 200 bases of an acceptable quality, defined as a minimum of $90 \%$ of the total bases (20 bases per 200bp read) with PHRED quality score of 20 or higher (accuracy of base call $\mathrm { i s } \geq 9 9 \%$ )   
o For sequences containing ambiguous base calls such as “N”s, the total number of ambiguous bases in the acceptable quality sequences generated using bidirectional sequencing should not exceed $5 \%$ of total bases (or 10 bases per 200 bp read).   
o Blast analysis of the acceptable quality sequences generated by bidirectional sequencing should have at least $9 5 \%$ query coverage compared to reference and at least $9 5 \%$ identity to reference.   
o Sequence matches the reference or sequence generates an Expected Value $\mathrm { ( E - V a l u e ) } < 1 0 ^ { - 3 0 }$ for the specific target following a BLAST search in GenBank (http://www.ncbi.nlm.nih.gov/Genbank/).

Negative – Samples were considered negative by sequencing if any one of the above criteria were not met.

# Acceptance Criteria

Clinical Sample: The clinical sample of known identity, if available, must be positive by sequencing for the expected target.   
Limit of Detection Study: At least, 2 of the 3 extraction replicates must be positive by sequencing at the equivalent or lower titer than the established limit of detection recorded for the xTAG GPP analyte.   
· Cross-reactivity Study: All samples tested should generate no sequencing reactions of acceptable quality.   
· Reactivity: Strains,genotypes and serotypes should generate positives results with their respective sequencing primers.

# Conclusion

All sequencing primers met the acceptance criteria for all studies.

Summary Comparator Validation Table for Adenovirus 40/41, E.histolytica and V.cholerae   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Entamoeba</td><td rowspan=1 colspan=1>Vibrio cholerae*</td></tr><tr><td rowspan=1 colspan=1>Sequencing primer /Study</td><td rowspan=1 colspan=1>Outside 101</td><td rowspan=1 colspan=1>Outside 103</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Limit of Detection</td><td rowspan=1 colspan=1>More sensitive than kit</td><td rowspan=1 colspan=1>Equal to kit</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Cross-Reactivity</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Reactivity</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>9/9 strains reacted</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\*Sequencing was not a comparator method for the V.cholerae analyte.

# Summary of negative control failures and sample re-run rates for analytical performance studies

Including all analytes in the xTAG GPP test intended use, there were a total of 278 xTAG GPP runs performed over the course of analytical performance studies. Each xTAG run has at least one no template negative control depending on batch size. Of the 278 runs, 12 $( 4 . 3 2 \% )$ had one or more negative control (NC) failures. These are summarized in the table below.

Summary of Negative Control Failures for Analytical Performance Studies   

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Total # of runs(includingallowable re-runs)</td><td rowspan=1 colspan=1>Total # of runswith at least oneNC failure</td><td rowspan=1 colspan=1>% totalruns withat leastone NCfailure</td><td rowspan=1 colspan=1>Total No. ofNCs includedin runs andallowable re-runs</td><td rowspan=1 colspan=1>Total No.of NCfailures</td><td rowspan=1 colspan=1>% total NC s includedwhich failed in xTAG runs/ allowable re-runs</td></tr><tr><td rowspan=1 colspan=1>Multi-site reproducibility</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6.32%</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.82%</td></tr><tr><td rowspan=1 colspan=1>Matrix equivalence</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Limit of detection</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Carry-over contamination</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Analytical specificity andinterference</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.00%</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.99%</td></tr><tr><td rowspan=1 colspan=1>Analytical reactivity</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.56%</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.42%</td></tr><tr><td rowspan=1 colspan=1>Evaluation of fresh vs.frozen stool</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.17%</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.39%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>278</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.32%</td><td rowspan=1 colspan=1>922</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.52%</td></tr></table>

Included in the 278 xTAG runs summarized above were 14960 specimens. Of these, $9 9 . 8 1 \%$ (14931/14960) yielded valid results on the first attempt. The remaining 29 specimens generated valid results following allowable re-runs. Sample rerun rates are summarized in the table below.

Summary of Sample Re-Run Rates for Analytical Performance Studies   

<table><tr><td rowspan=1 colspan=1>Studies</td><td rowspan=1 colspan=1>Total # ofspecimenstested</td><td rowspan=1 colspan=1>Total # ofinvalid resultsprior to re-run</td><td rowspan=1 colspan=1>% invalidresults priorto re-run</td><td rowspan=1 colspan=1>Invalid resultsafter re-run</td><td rowspan=1 colspan=1>% invalid resultsafter re-run</td></tr><tr><td rowspan=1 colspan=1>Multi-site reproducibility</td><td rowspan=1 colspan=1>5065</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0.49%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Matrix equivalence</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Limit of detection</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Carry-over contamination</td><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Analytical specificity andinterference</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Analytical reactivity</td><td rowspan=1 colspan=1>2225</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.04%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Evaluation of fresh vs.frozen stool</td><td rowspan=1 colspan=1>4162</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.05%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>14960</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0.19%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies section of this document.   
b. Matrix comparison: See raw stool and Cary-Blair study above.

3. Clinical studies:

# Microbial Detection in Asymptomatic Volunteers

In order to determine baseline levels for each analyte included in xTAG GPP for individuals who are not exhibiting signs and symptoms of infectious gastroenteritis, 200 clinical stool samples were collected from healthy, asymptomatic donors. Asymptomatic donors from various age groups were included in this study.

Demographic information for the asymptomatic donors is shown in the table below.

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>92 (46%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>108 (54%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0 - 1</td><td rowspan=1 colspan=1>5 (2.5%)</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>7 (3.5%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>43 (21.5%)</td></tr><tr><td rowspan=1 colspan=1>22 - 60</td><td rowspan=1 colspan=1>111 (55.5%)</td></tr><tr><td rowspan=1 colspan=1>≥61</td><td rowspan=1 colspan=1>34 (17.0%)</td></tr></table>

PCR inhibition, as determined by results for the internal control used with xTAG GPP (bacteriophage MS2), was observed in 23 of the 200 samples tested $( 1 1 . 5 \% )$ . After re-running these specimens in accordance with the instructions for use, PCR inhibition was still observed in eight samples $( 4 \% )$ . The absence of a detectable internal control signal in these samples meant that negative results for the indicated microbial targets could not be reported. Therefore, the final data analysis was conducted on 192 of the 200 samples collected for this study.

A total of 14 samples that were positive by xTAG GPP were sequenced. Two (2) out of 14 samples were positive by sequencing (C. Difficile Toxin A/B), while 12 of 14 samples were not positive by sequencing.

These results are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Percent Negative Results by xTAGGPP for all samples</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B</td><td rowspan=1 colspan=1>98.4% (189/192)1</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>99.5% (191/192)2</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>ETEC LT/ST</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>99.0% (190/192) 3</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>98.4% (189/192)4</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>97.4% (187/192) 5</td></tr><tr><td rowspan=1 colspan=1>STEC stx1/stx2</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>100.0% (192/192)</td></tr></table>

NOTE: Sample 216 was positive by xTAG GPP for both Norovirus GII and C. Difficile 1 Two (2) out of 3 xTAG GPP C. Difficile positive samples were confirmed as positive by sequencing analysis. 2 The one (1) xTAG GPP $E$ . histolytica positive sample was not confirmed as positive by sequencing analysis. 3 None of the 2 xTAG GPP Giardia positive samples was confirmed as positive by sequencing analysis. 4 None of the $3 \mathrm { { x T A G } }$ GPP Noroviris GI/GII positive samples was confirmed as positive by sequencing analysis. 5 None of the 5 xTAG GPP Salmonella positive samples was confirmed as positive by sequencing analysis.

Samples (at the specimen level) that were positive by xTAG GPP but negative by sequencing were considered false positives (12/192, $6 . 3 \%$ ). These samples had MFI values that were relatively close to the cut-offs. Two samples at the specimen level that were called positive by xTAG GPP were also positive by sequencing analysis for $C$ . difficile. These two samples positive for $C .$ . difficile by both xTAG GPP and sequencing may represent asymptomatic infections.

# Prospective Clinical Study

The clinical performance of the xTAG GPP was evaluated during prospective studies at six clinical laboratories in North America (four sites in the U.S. and two sites in Canada). Stool specimens were collected and tested at the six clinical laboratories (Sites 1, 2, 3, 4, 5, and 6) during June 2011 thru February 2012. Clinical study sites were selected based on the types of patients usually referred (e.g. pediatrics, adults), conditions often treated (e.g. C. difficile colitis), as well as the geographical prevalence of particular targeted pathogens.

Six geographically separated clinical study sites participated in the clinical evaluation of the xTAG GPP. The study sites were located in East-Central Canada (Toronto, Ontario and Hamilton, Ontario), and Southeast (Nashville, TN), Southwest (Temple, TX and Tucson, AZ), and Midwest (St Louis, MN) of the U.S. Each study location was

representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

The table below summarized the total number of patients recruited at each site.

<table><tr><td rowspan=1 colspan=1>Site #</td><td rowspan=1 colspan=1># PatientsRecruited</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>461</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>449</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>188</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>295</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1534</td></tr></table>

Patient specimens (one specimen from each of the recruited patients) that met all of the following characteristics were eligible for the study.

1. An exemption from the requirement for Informed Consent had been granted by the site IRB to include the left-over stool specimen in the study.   
2. The specimen was from a pediatric or adult, male or female subject who was either hospitalized, admitted to a hospital emergency department, visiting an outpatient clinic or resident of a long-term care facility.   
3. The specimen was from a patient for whom a requisition had been made for testing of microbial pathogens suspected of gastrointestinal tract infections.   
4. The specimen was from a patient exhibiting clinical signs and symptoms of infectious colitis (including C. difficile colitis) or gastroenteritis (including traveler's diarrhea), such as diarrhea, nausea and vomiting, loss of appetite, fever, abdominal pain and tenderness, cramping, bloating, flatulence, bloody stools, fainting and weakness.   
5. The volume of the specimen was $\ge 8 . 5$ ML or $\geq 6 \mathrm { ~ g ~ }$ .

Patient specimens with any one of the following characteristics was not eligible for study entry:

1. The specimen was collected at a site which was not covered under the study IRB.   
2. The specimen was a preserved stool, stool in Cary-Blair media or rectal swab.   
3. The specimen was from an individual who did not exhibit clinical signs and symptoms of infectious colitis or gastroenteritis.   
4. Based on available clinical information, the specimen was from an individual with known and documented non-infectious conditions such as ulcerative colitis, irritable bowel syndrome and/or Crohn’s disease.   
5. The specimen was not properly collected, transported, processed or stored according to the instructions provided by the sponsor.   
6. The specimen could not be tested by the relevant comparator assays within 72 hours of collection.

Of the 1534 stool specimens, 127 were excluded from the study. The reasons for exclusion are summarized in the table below.

Summary of Excluded Specimens $( \mathbf { N } { = } 1 2 7 )$ )   

<table><tr><td rowspan=1 colspan=1>Reason for Specimen Exclusion</td><td rowspan=1 colspan=1>Exclusion Criteria #</td><td rowspan=1 colspan=1># Excluded Specimens</td></tr><tr><td rowspan=1 colspan=1>The specimen was collected from a site not coveredunder the study IRB</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5 (0.3%)</td></tr><tr><td rowspan=1 colspan=1>The specimen was from an individual with knownand documented non-infectious conditions such asulcerative colitis, irritable bowel syndrome and/or Crohn&#x27;sdisease</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>67 (4.3%)</td></tr><tr><td rowspan=1 colspan=1>The specimen was not properly collected,transported, processed or stored according to the instructionsprovided by the sponsor</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50 (3.2%)</td></tr><tr><td rowspan=1 colspan=1>The specimen could not be tested by the relevant comparatorassays within 72 hours of collection</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4 (0.2%)</td></tr><tr><td rowspan=1 colspan=1>Other: multiple extraction failures</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1 (0.05%)</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>127</td></tr></table>

The following table provides a summary of demographic information for the 1407 subjects whose stool specimens were included in the prospective study.

General Demographic Details for the Prospective Data Set $\bf { ( N { = } 1 4 0 7 ) }$ .   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>632 (44.9%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>775 (55.1%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>6 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>20 (1.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>25 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>&gt;12-21</td><td rowspan=1 colspan=1>51 (3.6%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>879 (62.5%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>426 (30.3%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1>Subject Status</td><td></td></tr><tr><td rowspan=1 colspan=1>Outpatients</td><td rowspan=1 colspan=1>421 (29.9%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>804 (57.1%)</td></tr><tr><td rowspan=1 colspan=1>Emergency Department</td><td rowspan=1 colspan=1>118 (8.4%)</td></tr><tr><td rowspan=1 colspan=1>Long Term Care Facility</td><td rowspan=1 colspan=1>18 (1.3%)</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>46 (3.3%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1>Immune Status</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Immuno-compromised</td><td rowspan=1 colspan=1>493 (35.0%)</td></tr><tr><td rowspan=1 colspan=1>Immuno-competent</td><td rowspan=1 colspan=1>758 (53.9%)</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>156 (11.1%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1407</td></tr></table>

In addition to patients’ demographic details, every effort was made to ensure that information on clinical signs and symptoms of infectious colitis or gastroenteritis was available on all subjects enrolled in the prospective study. This information was collected by way of chart reviews. Chart reviews were conducted by an individual at the sites who was not directly involved in the study (e.g. research nurse) so that information was collected in a manner that did not make the specimen source identifiable to the investigator or any other individual involved in the investigation including the Sponsor. Local IRB approval for the study was obtained prior to study start. If available, the following information was also collected:

· Stool consistency (based on Bristol Stool Scale)   
· Clinical signs and symptoms of infectious colitis or gastroenteritis such as diarrhea, nausea and vomiting, loss of appetite, fever, abdominal pain and tenderness, cramping, bloating, flatulence, bloody stools, fainting and weakness   
· Duration and severity of symptoms prior to enrolment   
· Method of transmission (e.g. food-borne outbreak or close contact method)   
· Prior and concomitant medications including dose, type, frequency and duration.   
· Other orally ingested substances (e.g. fiber, stool bulking agents), including dose, type, frequency and duration   
· Other laboratory results (e.g. viral/bacterial culture, gram positive/negative infection, hematology and serum chemistry etc.)

Wherever available in the medical charts, the duration and severity of each specific sign or symptom was also recorded.

Stool consistency (based on the Bristol Stool Form Scale) was recorded for each clinical specimen included in the prospective clinical study. A summary of this information is provided in the table below.

Stool consistency $\bf { ( N { = } 1 4 0 7 ) }$   

<table><tr><td rowspan=1 colspan=1>Stool Consistency</td><td rowspan=1 colspan=1># Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Type 1 Separate hard lumps</td><td rowspan=1 colspan=1>8 (0.5%)</td></tr><tr><td rowspan=1 colspan=1>Type 2 Sausage-shaped but lumpy</td><td rowspan=1 colspan=1>24 (1.7%)</td></tr><tr><td rowspan=1 colspan=1>Type 3 Like a sausage but with cracks</td><td rowspan=1 colspan=1>26 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>Type 4 Like sausage/snake, smooth, soft</td><td rowspan=1 colspan=1>77 (5.5%)</td></tr><tr><td rowspan=1 colspan=1>Type 5 Soft blobs with clear-cut edges</td><td rowspan=1 colspan=1>160 (11.4%)</td></tr><tr><td rowspan=1 colspan=1>Type 6 Fluffy pieces with ragged edges</td><td rowspan=1 colspan=1>354 (25.2%)</td></tr><tr><td rowspan=1 colspan=1>Type 7 Watery, no solid pieces</td><td rowspan=1 colspan=1>758 (53.9%)</td></tr></table>

Information on clinical signs and symptoms of infectious colitis or gastroenteritis were available on 918 patients $( 6 5 . 2 \% )$ . A summary of the findings from the patient medical charts is provided in the table below.

Summary of Clinical Signs and Symptoms $\scriptstyle ( \mathbf { N = 9 1 8 } )$ )   

<table><tr><td colspan="1" rowspan="1">Clinical Signs and Symptoms</td><td colspan="1" rowspan="1"># Events Reported (%)</td><td colspan="1" rowspan="1">Duration Reported</td></tr><tr><td colspan="1" rowspan="1">Diarrhea</td><td colspan="1" rowspan="1">807 (87.9%)</td><td colspan="1" rowspan="1">1 day to 6 months</td></tr><tr><td colspan="1" rowspan="1">Nausea</td><td colspan="1" rowspan="1">327 (35.6%)</td><td colspan="1" rowspan="1">1 day to 6 months</td></tr><tr><td colspan="1" rowspan="1">Vomiting</td><td colspan="1" rowspan="1">228 (24.8%)</td><td colspan="1" rowspan="1">1 to 30 days</td></tr><tr><td colspan="1" rowspan="1">Loss of appetite</td><td colspan="1" rowspan="1">179 (19.4%)</td><td colspan="1" rowspan="1">1 day to 2 months</td></tr><tr><td colspan="1" rowspan="1">Fever</td><td colspan="1" rowspan="1">170 (18.5%)</td><td colspan="1" rowspan="1">1 day to 2 weeks</td></tr><tr><td colspan="1" rowspan="1">Abdominal pain</td><td colspan="1" rowspan="1">284 (30.9%)</td><td colspan="1" rowspan="1">1 day to 6 months</td></tr><tr><td colspan="1" rowspan="1">Tenderness</td><td colspan="1" rowspan="1">118 (12.8%)</td><td colspan="1" rowspan="1">1 day to 4 months</td></tr><tr><td colspan="1" rowspan="1">Cramping</td><td colspan="1" rowspan="1">101 (11.0%)</td><td colspan="1" rowspan="1">1 day to 4 months</td></tr><tr><td colspan="1" rowspan="1">Bloating</td><td colspan="1" rowspan="1">62 (6.7%)</td><td colspan="1" rowspan="1">1 day to 6 months</td></tr><tr><td colspan="1" rowspan="1">Flatulence</td><td colspan="1" rowspan="1">50 (5.4%)</td><td colspan="1" rowspan="1">1 day to 3 months</td></tr><tr><td colspan="1" rowspan="1">Bloody stool</td><td colspan="1" rowspan="1">89 (9.7%)</td><td colspan="1" rowspan="1">1 day to 4 months</td></tr><tr><td colspan="1" rowspan="1">Weakness</td><td colspan="1" rowspan="1">159 (17.3%)</td><td colspan="1" rowspan="1">1 day to 4 months</td></tr><tr><td colspan="1" rowspan="1">Other (e.g. Constipation)</td><td colspan="1" rowspan="1">87 (9.5%)</td><td colspan="1" rowspan="1">1 to 25 days</td></tr></table>

All prospective clinical specimens were submitted fresh to the sites and were processed according to their routine algorithm and as ordered by the referring physician. Upon receipt at the laboratory, any left-over stool specimen that met the study inclusion / exclusion criteria was placed into the following six containers.

1. Meridian sterile, leak-proof, wide-mouthed empty container (unpreserved stools)   
2. Meridian container containing Cary-Blair holding medium (Para- $\mathbf { \cdot P a k } ^ { \mathbb { \otimes } }$ C&S)   
3. Meridian container containing PVA fixative (Para-Pak® LV-PVA Fixative)   
4. Meridian container containing formalin (Para- $\mathbf { \cdot P a k } ^ { \mathbb { \left( B \right) } }$ $10 \%$ Buffered Neutral Formalin)   
5. Container containing ACTD medium (swab)   
6. Sterile container for xTAG GPP testing (unpreserved stools)

The time from collection to processing into the appropriate containers was kept to a minimum ${ < } 2 4$ hours). Prior to study initiation, processing instructions as well as shipping details were provided to each clinical site by the central laboratories carrying out reference and comparator method testing. Specimens were shipped to the central laboratories within 24 hours of processing. Prospective clinical specimens were then processed for both comparator testing and xTAG GPP testing as described below.

For all prospective specimens, reference and comparator method testing was performed at central laboratories independent of xTAG GPP testing sites. Reference/comparator testing was performed for all analytes on all prospectively collected specimens. In the event that comparator results were not available for all targets on a given specimen, then the specimen in question was excluded from performance calculations of xTAG GPP.

Reference and comparator methods for each analyte target are listed in the table below.

Reference/Comparator Methods and Shipping Requirements   

<table><tr><td colspan="1" rowspan="1">xTAG GPPAnalytes</td><td colspan="1" rowspan="1">Reference/Comparator Method</td><td colspan="1" rowspan="1">ShippingRequirements</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">Composite comparator consistingof PremierRotaclone EIA (MeridianBioScience k852969) directly onthe stool specimen and one PCR/sequencing assay directly fromclinical specimen</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tubes</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Composite comparator consistingof CDC real-time PCR andconventional PCR followed by bi-directional sequencing assaysdirectly from clinical specimen1</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tubes</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile Toxin A/B</td><td colspan="1" rowspan="1">Bartels CytotoxicityAssay for Clostridium difficileToxin (Bartels k833447) usingdiluted stool filtrate processeddirectly from clinical specimen</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tubes</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Bacterial culture</td><td colspan="1" rowspan="1">Stool in Cary-Blair holding medium</td></tr><tr><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Bacterial culture</td><td colspan="1" rowspan="1">Stool in Cary-Blair holding medium</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Bacterial culture(A PCR/Sequencing assay wasalso performed directly on clinicalspecimens that were testedpositive by culture for speciesidentification only)</td><td colspan="1" rowspan="1">Stool in Cary- Blair holdingmedium</td></tr><tr><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">Bacterial culture</td><td colspan="1" rowspan="1">Stool in Cary-Blair holding medium</td></tr><tr><td colspan="1" rowspan="1">Shiga-Like Toxin Producing E.coli (STEC)</td><td colspan="1" rowspan="1">Broth enrichment followed byImmunoCard STAT EHEC(Meridian BioScience, k062546)</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tube</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E. coli (ETEC)LT/ST</td><td colspan="1" rowspan="1">Composite comparator consistingof PCR/sequencing directly fromclinical specimen using fourPCR/sequencing assays, twoassays each for the LT and the STgene</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tube</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">Microscopy</td><td colspan="1" rowspan="1">Preserved stool in 10% Formalin</td></tr><tr><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1">Microscopy</td><td colspan="1" rowspan="1">Preserved stool in PVA fixative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 40/41</td><td colspan="1" rowspan="1">Composite comparator consistingof Premier Adenoclone Type40/41 EIA (Meridian Bioscience,K881894) directly on the stoolspecimen and one PCR/sequencing assay directly fromclinical specimen1</td><td colspan="1" rowspan="1">Unpreserved stool in sterile tube</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">Microscopy (A PCR/Sequencingassay was also performed directlyon clinical specimens that weretested positive by microscopy forspecies identification only)</td><td colspan="1" rowspan="1">Preserved stool in PVA fixative</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">Bacterial culture</td><td colspan="1" rowspan="1">Swab in ACTD transport medium</td></tr></table>

Refer to more detailed descriptions below.

Performance of the xTAG GPP detecting ETEC-LT and ETEC-ST was compared to a composite comparator method consisted of four separate analytically validated PCR followed by bi-directional sequencing assays (two for ETEC-LT and two for ETEC-ST). “True” ETEC positives were considered as any sample that was tested positive for LT or ST by any of the four PCR/sequencing assays. “True” ETEC negatives were considered as any sample that was tested negative for LT and ST by all four PCR/sequencing assays. PCR/sequencing assays were performed on nucleic acid extracted directly from clinical specimens using primers that targeted different genomic regions from the ones probed by xTAG GPP. Generated sequence results were analyzed as follows:

· For a given base from the consensus sequence generated from bi-directional sequencing, the PHRED score was calculated by averaging the PHRED quality score from the forward and reverse sequencing.   
· The generated sequence should be at least 200 bases of an acceptable quality, defined   
as a minimum of $90 \%$ of the total bases with PHRED quality score of 20 or higher.   
· Blast analysis of the consensus sequence generated by bi-directional sequencing should have at least $9 5 \%$ query coverage compare to reference, at least $9 5 \%$ identity to   
reference and an Expected Value (E-Value) 1of at least 10-30.   
· For sequences containing “N”s, the consensus generated using bi-directional sequencing should correspond to the strand including the high quality base instead of the strand including the “N” called base. In addition, the total number of N's should not exceed $5 \%$ of total bases (or 10 bases per 200 bp read).

Performance of the xTAG GPP detecting rotavirus or adenovirus 40/41 was compared to a composite comparator method consisted of an FDA cleared EIA test and one analytically validated PCR followed by bi-directional sequencing assay. “True” rotavirus or adenovirus 40/41 positives were considered as any sample that was tested positive for rotavirus or adenovirus 40/41 by the EIA and/or the PCR/sequencing assay. “True” rotavirus or adenovirus 40/41 negatives were considered as any sample that was tested negative for rotavirus or adenovirus 40/41 by both the EIA and the PCR/sequencing assay. PCR/sequencing was performed on nucleic acid extracted directly from clinical specimens using primers that targeted different genomic regions from the ones probed by xTAG GPP. Generated sequence results were analyzed described above.

Performance of the xTAG GPP for norovirus was assessed by comparing test results to the “patient norovirus infected status” of each specimen. The “patient norovirus infected status” was determined using a composite comparator method consisting of the CDC norovirus real-time Taqman RT-PCR assay and the CDC Conventional RT-PCR (Region-C and D primers) followed by bi-directional sequencing assays. The following interpretation algorithm was used to determine the “patient norovirus infected status”:

Composite Comparator Algorithm for Norovirus   

<table><tr><td rowspan=1 colspan=1>CDC Norovirus Real- TimeTaqman RT-PCR Result</td><td rowspan=1 colspan=1>CDC ConventionalRT-PCR Result (Region C)Followed by Bi-DirectionalSequencing</td><td rowspan=1 colspan=1>CDC ConventionalRT-PCR Result (Region D)Followed by Bi-DirectionalSequencing</td><td rowspan=1 colspan=1>Final CompositeComparator Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td></tr></table>

Clinical runs and re-runs (per the instructions provided in the product package insert) using xTAG GPP were carried out on left-over clinical specimens that had been extracted from the fresh or frozen state using the NucliSENS EasyMAG method (BioMérieux, Inc., Durham, NC) according to the manufacturer’s instructions. Total extracted nucleic acid material was stored at $\mathrm { - 7 0 ^ { 0 } C }$ prior to testing with xTAG GPP.

PCR negative (water blanks, NTC) control and external rotating positive controls (RC) representing analytes probed by the assay were also included with each xTAG GPP run. The external positive controls used in the study are listed in the table below and, for the most part (except for Cryptosporidium), consisted of chemically-inactivated bacteria, viruses and parasites from ZeptoMetrix. These controls were used to control the entire assay process including nucleic acid extraction, amplification, and detection. The external positive controls contained low organism copy numbers and were designed to mimic patient specimens. These were run as separate samples, concurrently with patient specimens. External positive controls were included in each assay plate in a rotating manner.

External Positive Controls   

<table><tr><td rowspan=1 colspan=1>External Positive Control</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Dilution Factor</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>Stock*</td></tr><tr><td rowspan=1 colspan=1>C difficile Toxin A/B</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/100</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>Pooled clinical specimens</td><td rowspan=1 colspan=1>Stock**</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157 / STEC</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/100</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>PRA-243 (ATCC)</td><td rowspan=1 colspan=1>Stock</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/100</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/1000</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/1000</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>Stock</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>Stock</td></tr></table>

\* Stock material was used as MFI signals generated for campylobacter in the initial clinical runs using1/10 dilution of the stock were too close to the assay cut-off. $^ { * * }$ Pooled clinical specimens positive for Crytopsoridium hominis were used as positive control for this target. MFI values generated were however close to the assay cut-off and, in a number of clinical runs were below the threshold for a positive call.

Clinical specimens were tested in accordance with the package insert for xTAG GPP assay and were tested by a single operator at each of the clinical sites.

The xTAG GPP assay includes an internal control (MS2 bacteriophage) that is added to each sample prior to extraction. In the event that none of the pathogen targets probed by xTAG GPP were detected in a clinical specimen and the MS2 call in that specimen was “Absent”, a 1/10 dilution of the nucleic acid remnant (from the original extraction) was prepared and tested by xTAG GPP. Two outcomes of running a 1/10 dilution were addressed in the following manner:

If the MS2 call was “Present” following a 1/10 dilution of the original extract, it is likely that the original result was due to PCR inhibition. All additional positive results generated in this scenario were reported as “Positive” in the calculation of sensitivity and specificity (or positive and negative agreement). Negative results generated in this scenario were reported as “inhibited” and excluded from the calculation of sensitivity and specificity (or positive and negative agreement) for the targets in question. However, inhibited results are presented in the performance tables as “invalid” for each microbial target. C If the MS2 signal was “Absent” following a 1/10 dilution of the original extract and none of the pathogen targets were detected, then the sample was re-tested with xTAG GPP, starting from the extraction step. If MS2 signal was “Present” after re-testing from the extraction step, it is likely that the original result was due to sub-optimal extraction. Negative and positive results generated in this allowable re-run were included in the calculation of sensitivity and specificity (or positive / negative agreement) for each individual target. If MS2 signal was still “Absent” after re-testing from the extraction step and none of the pathogen targets were detected, then the sample was coded as “inhibited” and was

excluded from the calculation of sensitivity and specificity (or positive and negative agreement) for the targets in question. However, inhibited results are presented in the performance tables as “invalid” for each microbial target.

In the event that an unexpected positive call was made in any of the assay controls included in the xTAG GPP run (negative or external positive control), then all clinical specimens that tested positive for the analyte(s) in question were re-tested by xTAG GPP. Negative and positive results generated in this allowable re-run were included in the calculation of sensitivity and specificity (or positive and negative percent agreements) for each individual target.

Discrepant results between the xTAG GPP and the reference methods were also evaluated using analytically validated PCR/sequencing assays or FDA cleared molecular assays (i.e., for C. difficile Toxin), and results are footnoted in the performance tables below.

The prospective performance data (all sites combined) are presented in the following tables by analyte:

Adenovirus 40/41   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1154</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1155</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>235</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>1400</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>37.5% - 96.4%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>98.1% - 99.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1The one specimen that was positive for Adenovirus 40/41 by reference but negative by xTAG GPP was positive by bidirectional sequencing only (i.e., FDA-cleared EIA negative)

Campylobacter   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1155</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>228</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1404</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>97.3% - 98.8%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.2%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1Sequencing results from these specimens revealed that all three were campylobacter jejuni. $^ 2 \mathrm { A }$ total of six Campylobacter xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

Clostridium difficile Toxin A/B   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>1051</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>220³3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>922</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>991</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>196</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>1197</td><td rowspan=1 colspan=1>952}$</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>87.9% - 97.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>89.8%</td><td rowspan=1 colspan=1>87.8% - 91.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1A total of 48 C. difficile Toxin A/B xTAG GPP positive specimens that were negative by the comparator method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP, or FDA cleared $C$ . difficile Toxin molecular assays .

$^ 2 \mathrm { A } ^ { \mathrm { \check { \Phi } } }$ total of 95 specimens generated a “Nonspecific reaction, not characteristic of Clostridium difficile toxin”. A titration test was performed on all 95 specimens and it was determined that in each case, the cytotoxicity reaction was not typical of $C$ . difficile toxin. This finding is consistent with the expected values for invalid results noted in the package insert of the Bartels Cytotoxicity Assay for Clostridium difficile Toxin.

$^ 3 \mathrm { A }$ total of 151 (151/220, $6 8 . 7 \%$ ) C. difficile Toxin $\mathrm { A } / \mathrm { B }$ xTAG GPP positive specimens were positive for both the Toxin A and B gene targets by the xTAG GPP Test. A total of 57 (57/220, $2 5 . 9 \%$ ) C. difficile Toxin A/B xTAG GPP positive specimens were positive for the Toxin B target and 12 (12/220, $5 . 4 \%$ ) were positive for the Toxin A target.

Cryptosporidium   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>53^2}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1131</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1132</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>210</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>1394</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>66.7% - 98.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>94.2% - 96.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>14.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1All 13 Cryptosporidium reference positive specimens were collected during a single outbreak which occurred at Site 2 and were typed as Cryptosporidium hominis.

$^ 2 \mathrm { A }$ total of eight Crytosporidium xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

Entamoeba histolytica   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1149</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1149</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>239</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.5% - 99.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>17.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

E. coli O157   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1158</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1158</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1405</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.5% - 99.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1A total of four $E$ . coli O157 xTAG GPP positive specimens that were negative by the comparator method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP. 2 Both reference positive $E$ . coli 0157 specimens were also positive for STEC by xTAG GPP. Only one was positive for STEC by the reference culture and EIA.

ETEC   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1156</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1162</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>239</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1398</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>7.1% - 59.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.1% - 99.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>17.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 One sample was positive for LT by both ETEC-LT PCR/sequencing assays. The other sample was positive for ST by both ETEC-ST PCR/sequencing assays. 2 ETEC performances were calculated against a composite comparator consisting of four well-characterized PCR/bidirectional sequencing assays, two ETEC-LT PCR/sequencing assays and two ETEC-ST PCR/sequencing assays. All six specimens were positive by only one of the four PCR/sequencing assays.

Giardia   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1132</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1132</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>232</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1403</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>51.0% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>95.5% - 97.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Norovirus GI/GII   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>1023</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1027</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>210</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>782</td><td rowspan=1 colspan=1>1329</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>94.9%</td><td rowspan=1 colspan=1>87.5% - 98.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>91.4%</td><td rowspan=1 colspan=1>89.6% - 92.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>14.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 All four xTAG GPP false negative Norovirus specimens were Norovirus GII. 2 Five of the 78 Norovirus comparator positive specimens were typed as GI at the CDC by sequencing, and 73 of the 78 Norovirus comparator positive specimens were typed as GII at the CDC by sequencing.

Rotavirus A   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1162</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1162</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>241</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1405</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>17.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Salmonella

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1143</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1143</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>236</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>236</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>1397</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72.2% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.6% - 99.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.8%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Cultured isolates from all 10 salmonella reference positive clinical specimens were typed at the Ontario Public Health Laboratory in Toronto. Three specimens were typed as Salmonella enterica subsp. enterica, Typhimurium; one specimen as Salmonella enterica subsp. enterica, Typhi; one specimen as Salmonella enterica subsp. enterica, Salamae; one specimen as Salmonella enterica subsp. enterica, Javiana; one specimen as Salmonella enterica subsp. enterica, Bredeney; one specimen as Salmonella enterica subsp. enterica, Mississippi; one specimen as Salmonella enterica subsp. enterica, Heidelberg; one specimen as Salmonella enterica subsp. enterica, Muenchen.

$^ 2 \mathrm { A }$ total of two salmonella xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

Shiga-Like Toxin Producing E. coli (STEC) stx1/stx2   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16{2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1153</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1153</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1406</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>97.8% - 99.2%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 This STEC reference positive specimen was typed a Shiga-like toxin 2 using the ImmunoCard STAT EHEC. $^ 2 \mathrm { A }$ total of one STEC xTAG GPP positive specimen that was negative by the reference method was confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

Shigella   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1154</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1154</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>234</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1405</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.7% - 99.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>16.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Two clinical specimens tested positive for shigella by bacterial culture; one was reported as Shigella flexneri while the other one was reported as Shigella sonnei. $^ 2 \mathrm { A }$ total of two shigella xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

Vibrio cholerae   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Reference</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1166</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1166</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>240</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1407</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>17.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The prospective performance data (all sites combined) are presented in the following table by organism:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8% - 100%</td><td rowspan=1 colspan=1>1155/11761</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>97.3% - 98.8%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>66.7% - 98.6%</td><td rowspan=1 colspan=1>1131/11842</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>94.2% - 96.6%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>1158/11673</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.5% - 99.6%</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1149/1168</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.5% - 99.0%</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>51.0% - 100%</td><td rowspan=1 colspan=1>1132/1171</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>95.5% - 97.6%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72.2% - 100%</td><td rowspan=1 colspan=1>1143/11614</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.6% - 99.0%</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7% - 100%</td><td rowspan=1 colspan=1>1153/11695</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>97.8% - 99.2%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>1154/11716</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.7% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1166/1167</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Positive PercentAgreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>Negative PercentAgreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>4/57</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>37.5% - 96.4%</td><td rowspan=1 colspan=1>1154/1167</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>98.1% - 99.3%</td></tr><tr><td rowspan=1 colspan=1>C. difficile Toxin A/B</td><td rowspan=1 colspan=1>107/114</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>87.9% - 97.0%</td><td rowspan=1 colspan=1>922/10278</td><td rowspan=1 colspan=1>89.8%</td><td rowspan=1 colspan=1>87.8% -91.5%</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>2/8</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>7.1% - 59.1%</td><td rowspan=1 colspan=1>1156/1160</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.1% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>74/78</td><td rowspan=1 colspan=1>94.9%</td><td rowspan=1 colspan=1>87.5% - 98.0%</td><td rowspan=1 colspan=1>1023/1119</td><td rowspan=1 colspan=1>91.4%</td><td rowspan=1 colspan=1>89.6% - 92.9%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>1162/1164</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr></table>

1 A total of six Campylobacter xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

$^ 2 \mathrm { A }$ total of eight Crytosporidium xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

$^ 3 \mathrm { \ A }$ total of four $E .$ . coli O157 xTAG GPP positive specimens that were negative by the comparator method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

$^ 4 \mathrm { A }$ total of two Salmonella xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

$^ 5 \mathrm { A }$ total of one STEC xTAG GPP positive specimen that was negative by the reference method was confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

$^ 6 \mathrm { A }$ total of two Shigella xTAG GPP positive specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP.

7The one specimen that was positive for Adenovirus 40/41 by reference but negative by xTAG GPP was positive by bi-directional sequencing only (i.e., FDA-cleared EIA negative).

$^ 8 \mathrm { A }$ total of $4 8 \ : C$ . difficile Toxin A/B xTAG GPP positive specimens that were negative by the comparator method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP, or FDA cleared C. difficile Toxin molecular assay.

# Prospective Clinical Study Mixed Infection Analysis

xTAG GPP detected a total of 98 mixed infections in the prospective clinical evaluation. This represents $2 0 . 2 \%$ of the total number of xTAG GPP positive specimens (98/486). Sixty two (62) (62/98; $6 3 . 3 \% $ were double infections, 23 (23/98; $2 3 . 5 \% )$ were triple infections, eight (8/98; $8 . 2 \%$ ) were quadruple infections, two (2/98; $2 . 0 \%$ were quintuple infections, one $( 1 / 9 8 ; 1 . 0 \%$ was sextuple infection, one was septuple infection (1/98; $1 . 0 \% )$ ), and one was octuplet infection (1/98; $1 . 0 \% )$ ). The single most common co-infections (20/98; $2 0 . 4 \% )$ was Norovirus GI/GII with $C .$ . difficile Toxin A/B. Out of the $9 8 \ : \mathrm { c o }$ -infections, 93 contained one or more analytes that had not been detected with the reference/comparator methods, i.e. discrepant coinfections. Distinct co-infection combinations detected by xTAG GPP in the prospective clinical study are summarized in the table below.

Distinct Co-infection Combinations Detected by the xTAG GPP in the Prospective Clinical Trial   

<table><tr><td colspan="8" rowspan="1">Distinct Co-infection CombinationsDetected by xTAG GPP</td><td colspan="1" rowspan="2">CootnsTot</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infections</td><td colspan="1" rowspan="2">Discrepant Analyte(s) a</td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1">Analyte 4</td><td colspan="1" rowspan="1">Analyte 5</td><td colspan="1" rowspan="1">Analyte 6</td><td colspan="1" rowspan="1">Analyte 7</td><td colspan="1" rowspan="1">Analyte 8</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C. diff. (x1);</td></tr><tr><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">STEC (x1);</td></tr><tr><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E coli O157 (x1); STEC (x1);</td></tr><tr><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">16b</td><td colspan="1" rowspan="1">Norovirus (x12); C diff. (x9);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Norovirus (x8); Crypt. (x5);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Norovirus (x3); Giardia (x6);</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2b</td><td colspan="1" rowspan="1">Salmonella (x2); C diff. (x1);</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">E. histo. (x2);</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">STEC (x1);</td></tr><tr><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Salmonella (x1); Shigella (x1);</td></tr><tr><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1b</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Campyl. (x1); C diff. (x1);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Norovirus (x2); C diff. (x2); E.histo. (x3);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ETEC (x1); Giardia (x1);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">STEC (x1);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">STEC (x1); Giardia (x1);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Norovirus (x1); Shigella (x1);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Norovirus (x1); Shigella (x1);</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C diff. (x1); E. coli O157 (x1);</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">V. cholerae</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">Adeno 40/41</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">E. coli0157</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adeno 40/41 (1x); Shigella (x1);Campyl. (x1); ETEC (x1); E. col0157 (x1); Crypto (1x);</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">Campyl.</td><td colspan="1" rowspan="1">C. diff.</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">E.hist..</td><td colspan="1" rowspan="1">Crypto</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Norovirus (x1); Shigella (x1);Campyl. (x1); ETEC (x1); STEC); Ehisx); Cypt);</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Quadruple infections</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Quintuple infections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Number of sextuple infections</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Number of septuplet infections</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Total Number of Octuple infections</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by the xTAG GPP but not detected by the reference/comparator methods. b One Norovirus /C. difficile Tox A/B, one Norovirus $/ E$ . coli 0157/STEC and one Salmonella/C.difficile Tox A/B xTAG GPP reported co-infected specimens that were negative by the reference method were confirmed as positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the xTAG GPP

Additional Distinct Co-infection Combinations Detected by the Reference/Comparator Methods, But Not Detected by the xTAG GPP in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=2>Distinct Co-infection Combinations a</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=2 colspan=1>Discrepant Analyte(s) b</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>cooun-sC0tL</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>C. diff.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C. diff.</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ETEC (x2)</td></tr></table>

a This table includes only distinct co-infections that were detected by the reference/comparator method but not by the xTAG GPP; the remaining co-infections detected by the reference methods are already represented in the table above. b Discrepant analyte is defined as one that is detected by the reference/comparator but not detected by the xTAG GPP.

Of the 1407 clinical specimens included in the data analysis, 98 $( 7 . 0 \% )$ were identified as positive for more than one target by xTAG GPP. In most cases, bacteria presented with viruses $\mathrm { N } { = } 2 8$ , $2 8 . 6 \%$ ), followed by bacteria $^ +$ viruses $^ +$ parasites $( \mathrm { N } { = } 2 3$ , $2 3 . 5 \% )$ , viruses $^ +$ parasites $\mathrm { N } { = } 2 0$ ; $2 0 . 4 \%$ ), bacteria $^ +$ parasites $( \mathrm { N } { = } 1 7$ , $1 7 . 3 \%$ ), bacteria $^ +$ bacteria $( \mathrm { N } { = } 9$ , $9 . 2 \%$ , and parasite $^ +$ parasite $( \mathrm { N } { = } 1 , 1 . 0 \% )$ . All enteric pathogens probed by xTAG GPP were implicated in co-infections. Results for co- infections are summarized in the table below.

Summary of co-infected samples $\mathbf { ( N { = } 9 8 ) }$   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Number Implicatedin Co-Infections</td><td rowspan=1 colspan=1>Percent of Total Co-InfectedSpecimens</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11.2%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16.3%</td></tr><tr><td rowspan=1 colspan=1>C. difficile</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>49.0%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>34.7%</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11.2%</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21.4%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>65.3%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15.3%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15.3%</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td></tr></table>

# Prospective Clinical Study Per Specimen/Patient Summary Results

Prospective study results were also analyzed on a per sample/patient basis. Results of this analysis are summarized in the table below both without taking into consideration the discrepant analysis by PCR/bi-directional sequencing or FDA cleared molecular assays

(Primary Reference/Comparator) and taking into consideration this discrepant analysis (After Discrepant Investigation).

Per Sample/Patient Summary Results – Prospective Sample Set $\scriptstyle ( \mathbf { N } = \mathbf { 1 4 0 7 } )$   

<table><tr><td rowspan=1 colspan=1>Analyses</td><td rowspan=1 colspan=1>PrimaryReference/Comparator</td><td rowspan=1 colspan=1>After Discrepant Investigation</td></tr><tr><td rowspan=1 colspan=1># Specimens with at least one pathogenpositive by xTAG GPP</td><td rowspan=1 colspan=1>486</td><td rowspan=1 colspan=1>486</td></tr><tr><td rowspan=1 colspan=1># Specimens with at least one pathogenpositive by xTAG GPP and confirmed byreference/comparator</td><td rowspan=1 colspan=1>217</td><td rowspan=1 colspan=1>286</td></tr><tr><td rowspan=1 colspan=1># Specimens with at least one pathogenpositive by xTAG GPP but none confirmedby reference/comparator</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1># Specimens with at least one pathogenpositive by reference/comparator but nonewas positive by xTAG GPP</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td></tr></table>

# Prospective Clinical Study Contaminated Runs

Unexpected positive call(s) in negative (NTC) or external rotating positive control(s) (RC) were reported in 10 out of 49 xTAG GPP runs (10/49, $2 0 . 4 \% )$ during the prospective clinical study. A total of 49 clinical specimens included in these contaminated runs tested positive for analytes that were unexpectedly present in assay controls $( 4 9 / 1 4 0 7 ; 3 . 5 \% )$ .

# Retrospective Clinical Study 1 - Pre-Selected Clinical Specimens

Due to low prevalence observed for most of the xTAG GPP analytes in the prospective clinical study (see above), xTAG GPP performance detecting the following microbial targets was further evaluated in a retrospective clinical study testing pre-selected clinical specimens.

<table><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni, C. coli and C. lari only)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium (C. parvum and C. hominis only)</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) LT/ST</td></tr><tr><td rowspan=1 colspan=1>Giardia</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E. coli (STEC) stx1/stx2</td></tr><tr><td rowspan=1 colspan=1>Shigella</td></tr></table>

Pre-selected stool specimens were collected at multiple sites in North America and Europe. Demographic information (age and gender) was collected on all pre-selected specimens for which these data were available and is summarized in the table below.

General Demographic Details for the Pre-Selected Data Set $( \mathbf { N } { = } 2 \mathbf { 0 } 7 )$   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>107 (51.7%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>86 (41.5%)</td></tr><tr><td rowspan=1 colspan=1>Not known</td><td rowspan=1 colspan=1>14 (6.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>38 (18.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>26 (12.5%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>13 (6.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;12-21</td><td rowspan=1 colspan=1>11 (5.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21 - 65</td><td rowspan=1 colspan=1>91 (44.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>14 (6.8%)</td></tr><tr><td rowspan=1 colspan=1>Not known</td><td rowspan=1 colspan=1>14 (6.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>207</td></tr></table>

The table below outlines the number of pre-selected positive specimens included in the retrospective clinical study for each analyte target as well as the characterization method used.

Pre-selected Specimen Information $( \mathbf { N } { = } 2 \mathbf { 0 } 7 )$   

<table><tr><td colspan="1" rowspan="1">Pre-selected Target</td><td colspan="1" rowspan="1"># Specimens Included</td><td colspan="1" rowspan="1">Characterization Method (Comparator)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PCR/sequencing directly from clinical specimen usingone PCR/sequencing assay</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Bacterial culture</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">13 (9 Cryptosporidium parvumand 4 Cryptosporidiumhominis)</td><td colspan="1" rowspan="1">FDA cleared DFA or microscopy</td></tr><tr><td colspan="1" rowspan="1">E. histolitica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Microscopy (A PCR/Sequencing assay using the sameanalytically validated primers as those used in theProspective Clinical Study was also performed directlyon clinical specimens that were tested positive bymicroscopy for species identification only)</td></tr><tr><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">Bacterial culture</td></tr><tr><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">PCR/sequencing directly from clinical specimen usingfour PCR/sequencing assays (two for LT and two forST)</td></tr><tr><td colspan="1" rowspan="1">Giardia</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">FDA cleared DFA or microscopy</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">FDA cleared EIA or PCR followed bybi-directional sequencing using the same analyticallyvalidated primers as those used in the ProspectiveClinical Study</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Bacterial culture</td></tr><tr><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">FDA cleared EIA</td></tr><tr><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Bacterial culture</td></tr></table>

1 All eight $E$ . coli 0157 clinical specimens were also assessed by PCR followed by bi-directional sequencing for STEC. 2 All 10 STEC clinical specimens were also assessed by PCR followed by bi-directional sequencing for $E$ . coli 0157.

These pre-selected positive specimens were tested with xTAG GPP at three clinical sites along with 273 “negative” clinical specimens in a randomized, blinded fashion. The “negative” designation for these 273 specimens was based on the routine algorithms used at the clinical site (e.g. bacterial culture, EIA, microscopy, in-house real time PCR). These algorithms did not test for all pathogen targets probed by xTAG GPP.

The table below summarizes the positive percent agreement between comparator and xTAG GPP for all pre-selected targets evaluated.

Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Positive PercentAgreement</td><td rowspan=2 colspan=1>95%CI for PositivePercentAgreement</td><td rowspan=2 colspan=1>Number of &quot;Invalid&quot;xTAG GPP Results</td></tr><tr><td rowspan=1 colspan=1>TP /(TP+FN)</td><td rowspan=1 colspan=1>percent</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>40/41</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>87.4% - 99.6%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.7% - 100%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2.5% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>E. coli O157 1</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>78.5% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>38/39</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>86.8% - 99.5%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>15/16</td><td rowspan=1 colspan=1>93.7%</td><td rowspan=1 colspan=1>71.7% - 98.9%</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.9% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>71.9% - 96.1%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>STEC 2</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9% - 100%</td><td rowspan=1 colspan=1>0</td></tr></table>

1 Eight (8)/8 E. coli 0157 were also positive for STEC by xTAG GPP. Sample remnants of all $8 E .$ . coli 0157 specimens were tested for the presence of $s t x I$ and stx 2 genes by bi-directional sequencing and the results added to those obtained for STEC. 2 Six $( 6 ) / 1 0$ STEC were also positive for $E .$ . coli 0157 by xTAG GPP. Sample remnants of all 10 STEC specimens were assessed by bi-directional sequencing for $E$ . coli 0157 and the results added to those obtained for $E .$ . coli 0157.

Nucleic acid amplification followed by bi-directional sequencing using analytically validated primers was also performed on all available pre-selected clinical specimens that were positive by xTAG GPP for other analytes. More specifically, confirmatory testing was performed for those analytes that were positive by xTAG GPP but not pre-selected at the banking site in order to determine whether these additional positive calls represented True Positive (TP) or False Positive (FP) clinical results. To the extent possible, sequencing primers targeted genomic regions distinct from those of the kit primers. xTAG GPP

generated 114 additional positive calls (after allowable re-runs) for analytes that were not pre-selected at the banking site. A summary of these additional calls and confirmatory testing results are provided in the tables below.

Adenovirus 40/41   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>402</td><td rowspan=1 colspan=1>402</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>469</td><td rowspan=1 colspan=1>477*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>62.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * 3 }$ specimens were pre-selected for Adenovirus 40/41. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

Campylobacter   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>369</td><td rowspan=1 colspan=1>369</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>435</td><td rowspan=1 colspan=1>439*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } 4 1$ specimens were pre-selected for Campylobacter. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

# C. Difficile Toxin A/B

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>251</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>394</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>455</td><td rowspan=1 colspan=1>480</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>64.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1A total of 17 (17/25, $6 8 . 0 \%$ C. difficile Toxin $\mathrm { A } / \mathrm { B }$ xTAG GPP positive specimens were positive for both the Toxin A and B gene targets by the xTAG GPP Test. A total of 7 (7/25, $2 8 . 0 \%$ $C .$ . difficile Toxin $\mathrm { A } / \mathrm { B }$ xTAG GPP positive specimens were positive for the Toxin B target and $( 1 / 2 5 , 4 . 0 \% )$ were positive for the Toxin A target.

Cryptosporidium   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>401</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>466</td><td rowspan=1 colspan=1>467*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } 1 3$ specimens were pre-selected for Cryptosporidium. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

E. histolytica   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>403</td><td rowspan=1 colspan=1>403</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>471</td><td rowspan=1 colspan=1>479*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\*1 specimen was pre-selected for $E$ . histolytica. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

E. coli o157   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>397</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>466*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } 1 4$ specimens were pre-selected for $E$ . coli O157. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

ETEC   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>369</td><td rowspan=1 colspan=1>369</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>433</td><td rowspan=1 colspan=1>441*</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>50%</td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.3%</td></tr></table>

$^ { \ast } 3 9$ specimens were pre-selected for ETEC. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

Giardia   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>395</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>458</td><td rowspan=1 colspan=1>463*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } 1 7$ specimens were pre-selected for Giardia. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

Norovirus   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>396</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>471</td><td rowspan=1 colspan=1>480</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>11.8%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Rotavirus   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>379</td><td rowspan=1 colspan=1>379</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>446</td><td rowspan=1 colspan=1>452*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { \ast } 2 8$ specimens were pre-selected for Rotavirus. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

Salmonella   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>382</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>443</td><td rowspan=1 colspan=1>453*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { \ast } 2 7$ specimens were pre-selected for Salmonella. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

STEC   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>390</td><td rowspan=1 colspan=1>390</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>456</td><td rowspan=1 colspan=1>462*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } 1 8$ specimens were pre-selected for STEC. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

Shigella   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>379</td><td rowspan=1 colspan=1>379</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>447</td><td rowspan=1 colspan=1>460*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>78.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate (N=480)</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$\ast _ { 2 0 }$ specimens were pre-selected for Shigella. Results are presented in the “Positive Percent Agreement of xTAG GPP in the Pre-Selected Data Set” table.

V. cholerae   

<table><tr><td colspan="1" rowspan="1">xTAG GPP</td><td colspan="3" rowspan="1">PCR/Bi-directional Sequencing</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Not Done</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">412</td><td colspan="1" rowspan="1">412</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">68</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">480</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Invalid Rate (N=480)</td><td colspan="1" rowspan="1">14.2%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

# Retrospective Clinical Study 1 (Pre-Selected Clinical Specimens) Contaminated Runs

Unexpected positive call(s) in negative (NTC) or external rotating positive control(s) (RC) were reported in three out of 15 pre-selected xTAG GPP runs (3/15, $2 0 . 0 \%$ ). A total of 21 clinical specimens included in these runs tested positive by xTAG GPP for analytes that were unexpectedly present in assay controls (21/480; $4 . 4 \%$ .

# Supplemental Clinical Study – Botswana Pediatric Stool Specimens

The clinical performance of xTAG GPP for Adenovirus 40/41, Rotavirus, ETEC, Cryptosporidium and Gardia was also evaluated in a set of pediatric stool specimens $\scriptstyle ( \mathrm { N } = 3 1 3$ ) prospectively collected between February 2011 and January 2012 from symptomatic pediatric patients admitted to two referral hospitals in Botswana, Africa. All pediatric patients included in this evaluation presented with diarrhea and/or vomiting. General demographic details for these patients are summarized in the table below.

General demographic details of Botswana Sample Set $\scriptstyle ( \mathbf { N } = 3 \mathbf { 1 } 3 )$ )   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>186 (59.4%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>127(40.6%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>231 (73.8%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>62 (19.8%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11 (3.5%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 (0.9%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3 (0.9%)</td></tr><tr><td rowspan=1 colspan=1>&gt;4</td><td rowspan=1 colspan=1>3 (0.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>313</td></tr></table>

All specimens were shipped frozen to one of the study sites in Ontario, Canada for xTAG GPP testing. Stools were extracted by the Biomerieux NucliSENS EasyMag and tested using the xTAG GPP per the instructions provided in the product package insert.

Comparator testing by nucleic acid amplification followed by bi-directional sequencing using analytically validated primers was performed on samples positive for Adenovirus 40/41, Rotavirus, ETEC, Cryptosporidium and Giardia by xTAG GPP. In order to minimize bias, a random subset of the 313 Botswana specimens that tested negative by xTAG GPP was also assessed by the same nucleic acid amplification followed by bidirectional sequencing method for Adenovirus 40/41, Rotavirus, ETEC, Cryptosporidium and Giardia. In the case of Cryptosporidium and Giardia, the number of xTAG GPP negative specimens assessed was equal to or greater than the number of specimens identified as positive by xTAG GPP. In the case of ETEC, the number of xTAG GPP negative specimens assessed was slightly less than the number of specimens identified as positive by xTAG GPP. Since 178 of 313 specimens tested positive by xTAG GPP for Rotavirus, the number of negative Rotavirus specimens tested by nucleic acid amplification followed by sequencing was less than the number of positive Rotavirus specimens tested by this comparator method. Comparator testing by nucleic acid amplification followed by bidirectional sequencing using analytically validated primers was performed on a total of 91, 308, 56, 24, and 20 specimens for Adenovirus 40/41, Rotavirus, ETEC, Cryptosporidium, and Giardia, respectively.

In addition, all available residual clinical specimens $\scriptstyle ( \mathrm { N } = 3 1 1$ ) were assessed for Adenovirus 40/41 using the same FDA-cleared EIA as that used in the prospective study (Premier Adenoclone Type 40/41 EIA, Meridian Bioscience, K881894).

The Botswana Study performance data are presented in the following tables by analyte:

Adenovirus 40/41   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator (PCR/Bi-directional sequencing and/or FDAcleared EIA)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34{2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>273</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>260 3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>65.4%</td><td rowspan=1 colspan=1>51.8% - 76.8%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.5% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate 4</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 All 18 specimens that were positive for Adenovirus 40/41 by comparator but negative by xTAG GPP were positive by bidirectional sequencing only (i.e. FDA-cleared EIA negative). All these 18 specimens were assessed by real-time PCR for Adenovirus (all sub-types) at the laboratory testing site. The mean Ct value for these 18 specimens was 32.9; indicating low viral titer in these specimens, which is less clinically relevant.

2 All these 34 specimens were also assessed by real-time PCR for Adenovirus (all sub-types) at the laboratory testing site In contrast to the 18 specimens in footnote 1 above, the mean $\mathrm { C t }$ value for the 34 adenovirus samples positive by the PCR/Bidirectional sequencing assay and detected by xTAG GPP in this cohort was 22.81; indicating higher viral titer in these specimens, which is more clinically relevant.

$^ 3 2 2 2$ of the comparator negative Adenovirus 40/41 specimens were assessed by FDA-cleared EIA only.   
4 Six out of a total of 313 samples tested by the xTAG GPP generated an “invalid” result for Adenovirus 40/41.

Rotavirus A   

<table><tr><td colspan="1" rowspan="1">xTAG GPP</td><td colspan="3" rowspan="1">Comparator (PCR/Bi-directional sequencing)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">175</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">178</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">193</td><td colspan="1" rowspan="1">115</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">308</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Positive Percent Agreement</td><td colspan="1" rowspan="1">90.7%</td><td colspan="1" rowspan="1">85.7% - 94.0%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative Percent Agreement</td><td colspan="1" rowspan="1">97.3%</td><td colspan="1" rowspan="1">92.4% - 99.1%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Invalid Rate</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

1 Four out of a total of 313 samples tested by the xTAG GPP generated an “invalid” result for Rotavirus A.

ETEC   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator (PCR/Bi-directional sequencing)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>81.7% - 99.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>89.7%</td><td rowspan=1 colspan=1>73.6% - 96.4%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Five out of a total of 313 samples tested by the xTAG GPP generated an “invalid” result for ETEC

Cryptosporidium   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator (PCR/Bi-directional sequencing)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>64.6% - 98.5%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.7% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Five out of a total of 313 samples tested by the xTAG GPP generated an “invalid” result for Cryptosporidium. Giardia

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>Comparator (PCR/Bi-directional sequencing)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>70.1% - 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>90.9%</td><td rowspan=1 colspan=1>62.3% - 98.4%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Five out of a total of 313 samples tested by the xTAG GPP generated an “invalid” result for Giardia.

The table below summarizes the positive and negative agreement (PPA and NPA) between comparator results and xTAG GPP for Adenovirus 40/41, Rotavirus, Cryptosporidium and Giardia.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>34/52</td><td rowspan=1 colspan=1>65.4%</td><td rowspan=1 colspan=1>51.8% - 76.8%</td><td rowspan=1 colspan=1>255/255</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>175/193</td><td rowspan=1 colspan=1>90.7%</td><td rowspan=1 colspan=1>85.7% - 94.0%</td><td rowspan=1 colspan=1>108/111</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>92.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>26/27</td><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>81.7% - 99.3%</td><td rowspan=1 colspan=1>26/29</td><td rowspan=1 colspan=1>89.7%</td><td rowspan=1 colspan=1>73.6% - 96.4%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>64.6% - 98.5%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.7% - 100%</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>70.1% - 100%</td><td rowspan=1 colspan=1>10/11</td><td rowspan=1 colspan=1>90.9%</td><td rowspan=1 colspan=1>62.3% - 98.4%</td></tr></table>

Nucleic acid amplification followed by bi-directional sequencing using analytically validated primers was also performed on all available clinical specimens that were positive by xTAG GPP for other analytes (i.e., Campylobacter, C. difficile Toxin A/B, $E .$ . coli O157, Norovirus, Salmonella, Shigella, and STEC) in order to determine whether these additional positive calls represented True Positive (TP) or False Positive (FP) clinical results. The tables below summarize the confirmed xTAG GPP positive rate (i.e., confirmed xTAG GPP positives/all xTAG GPP positives) by PCR/bi-directional sequencing for Campylobacter, C. difficile Toxin A/B, E. histolytica, E. coli O157, Norovirus, Salmonella, Shigella, STEC, and V. cholerae.

Campylobacter   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>49</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>258</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# C. Difficile Toxin A/B

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>151</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>292</td><td rowspan=1 colspan=1>292</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1A total of 9 (9/15, $6 0 . 0 \%$ ) C. difficile Toxin $\mathbf { A } / \mathbf { B }$ xTAG GPP positive specimens were positive for both the Toxin A and B gene targets by the xTAG GPP Test. A total of 3 (3/15, $2 0 . 0 \%$ ) $C .$ . difficile Toxin $\mathrm { A } / \mathrm { B }$ xTAG GPP positive specimens were positive for the Toxin B target and 3 (3/15, $2 0 . 0 \%$ ) were positive for the Toxin A target.

E. histolytica   

<table><tr><td colspan="1" rowspan="1">xTAG GPP</td><td colspan="3" rowspan="1">PCR/Bi-directional Sequencing</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Not Done</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">307</td><td colspan="1" rowspan="1">307</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">313</td><td colspan="1" rowspan="1">313</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Invalid Rate</td><td colspan="1" rowspan="1">1.9%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

E. coli O157   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>303</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Norovirus   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>65.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Salmonella   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>290</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>35.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Shigella   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>271</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

STEC   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>302</td><td rowspan=1 colspan=1>302</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

V. cholerae   

<table><tr><td rowspan=1 colspan=1>xTAG GPP</td><td rowspan=1 colspan=3>PCR/Bi-directional Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>306</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Confirmed xTAG GPPPositives/All xTAG GPPPositives</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Invalid Rate</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Supplemental Clinical Study (Botswana Pediatric Stool Specimens) Contaminated Runs

Unexpected positive call(s) in negative (NTC) or external rotating positive control(s) (RC) were reported in 2 out of 5 Botswana xTAG GPP runs $( 4 0 \% )$ . A total of 5 clinical specimens included in these runs tested positive by xTAG GPP for analytes that were unexpectedly present in assay controls (5/313; $1 . 6 \%$ .

# Supplemental Study – Contrived Stool Specimens

The performances of xTAG GPP for $E$ . histolytica and V. cholera were further assessed using contrived specimens.

The low prevalence of Entamoeba histolytica made it very difficult to source available retrospective specimens. Many efforts were made by the sponsor to source Entamoeba histolytica positive clinical specimens. However, the sponsor could not acquire any usable Entamoeba histolytica clinical specimens for testing with the GPP assay. For Vibrio cholerae the major limitation to obtaining retrospective samples was the import/export restrictions placed around this pathogen. These import/export restrictions are applied at the molecular level preventing the sponsor from obtaining inactivated material or even purified nucleic acids from Vibrio cholerae. The prevalence of Vibrio cholerae is very low in Canada and the United States and the sponsor was unable to find a Canadian or US source for retrospective samples.

A panel of contrived specimens was made for each of these two rare analytes, as well as a panel of un-spiked contrived negative specimens. The contrived specimens consisted of unique source (individual donor) stool matrix that was used as-is for the negative specimen panel or was spiked with reference strain culture material for Entamoeba histolytica and Vibrio cholerae. A total of 50 unique specimens were made and tested for each panel.

This study was performed at three North American sites:

· Site A – Saint Joseph’s Hospital (SJH), Hamilton, Ontario · Site B – Mount Sinai Hospital (MSH), Toronto, Ontario · Site C – Luminex Molecular Diagnostics (LMD), Toronto, Ontario.

The stool specimens used to create individual negative clinical matrix were obtained from anonymized donors. Prior to being used as matrix, the stools were screened with xTAG GPP to ensure they were negative for all xTAG GPP analytes and that they had a present call for the spiked-in internal control (MS2). Stool specimens which did not meet these criteria were not used for this study. A total of 50 unique lots of stool matrix were created from 50 raw stool samples, with each lot of stool matrix obtained from a unique donor. The 50 lots of stool matrix were then utilized to make 50 Entamoeba histolytica contrived samples, 50 Vibrio cholerae contrived samples and 50 negative samples.

The Entamoeba histolytica contrive samples were created by spiking stool matrix with high titer culture material from four different strains of Entamoeba histolytica obtained from ATCC.

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Titer of Stock(Cells/mL)</td><td rowspan=1 colspan=1>Titer of ContrivedSpecimens(Cells/mL)</td><td rowspan=1 colspan=1>Number ofContrivedSpecimens</td><td rowspan=1 colspan=1>Multiples ofLoD(approximatedbased on real-time PCRassay)</td></tr><tr><td rowspan=6 colspan=1>ATCC30890</td><td rowspan=6 colspan=1>HM-3:IMSS</td><td rowspan=6 colspan=1>1.47 x 104</td><td rowspan=1 colspan=1>5.76 x 10</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>1.23 x 102</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4X</td></tr><tr><td rowspan=1 colspan=1>3.96 x 102</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14X</td></tr><tr><td rowspan=1 colspan=1>1.23 x 103</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>43X</td></tr><tr><td rowspan=1 colspan=1>1.23 x 104</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>430X</td></tr><tr><td rowspan=1 colspan=1>1.65 x 104</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>570X</td></tr><tr><td rowspan=1 colspan=1>ATCC30459</td><td rowspan=1 colspan=1>HM-1:MISS [ABRM]</td><td rowspan=1 colspan=1>1.40 x 106</td><td rowspan=1 colspan=1>4.00 x 104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1400X</td></tr><tr><td rowspan=4 colspan=1>ATCC30458</td><td rowspan=4 colspan=1>200:NIH</td><td rowspan=4 colspan=1>1.27 x 107</td><td rowspan=1 colspan=1>4.00 x 104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1400X</td></tr><tr><td rowspan=1 colspan=1>1.20 x 105</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4200X</td></tr><tr><td rowspan=1 colspan=1>4.00 x 105</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14000X</td></tr><tr><td rowspan=1 colspan=1>4.00 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>140000X</td></tr><tr><td rowspan=3 colspan=1>ATCC30015</td><td rowspan=3 colspan=1>HK-9</td><td rowspan=3 colspan=1>9.60 x 106</td><td rowspan=1 colspan=1>4.00 x 104</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1400X</td></tr><tr><td rowspan=1 colspan=1>1.20 x 105</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4200X</td></tr><tr><td rowspan=1 colspan=1>4.00 x 105</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14000X</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1></td></tr></table>

The Vibrio cholerae contrive samples were created by spiking stool matrix with high titer culture material from two different strains of Vibrio cholerae obtained from NCTC.

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Titer of Stock(Cells/mL)</td><td rowspan=1 colspan=1>Titer of ContrivedSpecimens(Cells/mL)</td><td rowspan=1 colspan=1>Number ofContrivedSpecimens</td><td rowspan=1 colspan=1>Multiples ofLoD(approximatedbased on real-time PCRassay)</td></tr><tr><td rowspan=6 colspan=1>NCTC12945</td><td rowspan=6 colspan=1>0139</td><td rowspan=6 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 107</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>43X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 108</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>130X</td></tr><tr><td rowspan=1 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>260X</td></tr><tr><td rowspan=6 colspan=1>NCTC7260</td><td rowspan=6 colspan=1>01</td><td rowspan=6 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>13X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 108</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>43X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>130X</td></tr><tr><td rowspan=1 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>260X</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1></td></tr></table>

The 50 sample contrived specimen panels for each of Entamoeba histolytica, Vibrio cholerae and the negative stool specimens were de-identified so that the sample identity and pathogen concentration was not determinable by the site investigator, operator or any other individual associated with the study. After de-identification the samples were randomized and distributed between the three study sites. Each contrived specimen was processed and analyzed following the assay kit instructions.

All 50 contrived negative stool specimens produced the expected negative result for all analytes. The agreement with expected negative results is $100 \%$ (50/50) with a $9 5 \%$ confidence interval from $9 2 . 9 \%$ to $100 \%$ .

The table below summarizes agreements with expected positive results for $E$ . histolytica and $V .$ . cholerae.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=2>Concentration (Cells/mL)</td><td rowspan=1 colspan=1>Multiples of LoD(approximatedbased on real-timePCR assay)</td><td rowspan=1 colspan=1>Agreement withExpected PositiveResult</td><td rowspan=1 colspan=1>95% ConfidenceInterval (CI)</td></tr><tr><td rowspan=10 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>5.76 x 10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.23 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4X</td><td rowspan=1 colspan=1>100% (1/1)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3.96 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>14X</td><td rowspan=1 colspan=1>100% (1/1)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.23 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>43X</td><td rowspan=1 colspan=1>100% (2/2)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.23 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>430X</td><td rowspan=1 colspan=1>100% (2/2)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.65 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>570X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>4.00 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1400X</td><td rowspan=1 colspan=1>100% (4/4)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.20 x 105</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4200X</td><td rowspan=1 colspan=1>100% (4/4)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>4.00 x 105</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>14000X</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>4.00 x 106</td><td rowspan=1 colspan=1>140000X</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>E. histolytica Overall</td><td rowspan=1 colspan=1>100% (50/50)</td><td rowspan=1 colspan=1>92.9% - 100%</td></tr><tr><td rowspan=6 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4X</td><td rowspan=1 colspan=1>80% (4/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>3.00 x 107</td><td rowspan=1 colspan=1>13X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>1.00 x 108</td><td rowspan=1 colspan=1>43X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>3.00 x 108</td><td rowspan=1 colspan=1>130X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>6.00 x 108</td><td rowspan=1 colspan=1>260X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>V. cholera Overall</td><td rowspan=1 colspan=1>98.0% (49/50)</td><td rowspan=1 colspan=1>89.5% - 99.7%</td></tr></table>

# Stool Specimens in Cary-Blair Media

# Prospective Clinical Study

Performance of xTAG GPP testing stool specimens in Cary-Blair media was evaluated on all available prospectively collected, leftover stool specimens stored in Cary-Blair medium from the prospective study testing raw unpreserved stool specimens as described in the “Prospective Clinical Study” section of this decision summary.

The prospective specimens in the Cary-Blair medium were de-identified by an individual who was not involved in the study, so that the identity of the subject was not readily ascertained by the site operator. These Cary-Blair stool remnants were distributed to at least three clinical sites and tested with xTAG GPP starting from pre-treatment and extractions steps. All specimens were distributed to the sites in a frozen state. No sample preparation was done for prospectively collected specimens in Cary-Blair medium.

Sensitivity/positive percentage agreement of xTAG GPP on stool in Cary-Blair medium is summarized for each individual target in the table below. For comparison purposes, performance results generated from the unpreserved stool as part of the prospective study as described in the “Prospective Clinical Study” section of this decision summary are also presented alongside of the performance results generated from the Cary-Blair preserved stool specimens.

<table><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=6>Sensitivity</td></tr><tr><td rowspan=1 colspan=3>Unpreserved Stool</td><td rowspan=1 colspan=3>Stool in Cary-Blair Media</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>3/(3+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9% - 100%</td><td rowspan=1 colspan=1>3/(3+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>12/(12+1)</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>66.7% - 98.6%</td><td rowspan=1 colspan=1>12/(12+1)</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>66.7% - 98.6%</td></tr><tr><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=3>n/a</td><td rowspan=1 colspan=3>n/a</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>2/(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>2/(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>4(4+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0% - 100%</td><td rowspan=1 colspan=1>4(4+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>74/(74+4)</td><td rowspan=1 colspan=1>94.9%</td><td rowspan=1 colspan=1>87.5% - 98.0%</td><td rowspan=1 colspan=1>70(70+3)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>88.6% - 98.6%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>10(10+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72.2% - 100%</td><td rowspan=1 colspan=1>10(10+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72.2% -100%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>2(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>2(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholera</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>n/a</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=6>Positive Agreement</td></tr><tr><td rowspan=1 colspan=3>Unpreserved Stool</td><td rowspan=1 colspan=3>Stool in Cary-Blair Media</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/411</td><td rowspan=1 colspan=1>4/(4+1)</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>37.5% - 96.4%</td><td rowspan=1 colspan=1>2/(2+3)</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>11.8% - 76.9%</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficiletoxin A/B</td><td rowspan=1 colspan=1>107/(107+7)</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>87.9% - 97.0%</td><td rowspan=1 colspan=1>98/(98+9)</td><td rowspan=1 colspan=1>91.6%</td><td rowspan=1 colspan=1>84.8% - 95.5%</td></tr><tr><td rowspan=1 colspan=1>ETEC LT/ST2</td><td rowspan=1 colspan=1>2/(2+6)</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>7.1% - 59.1%</td><td rowspan=1 colspan=1>2/(2+7)</td><td rowspan=1 colspan=1>22.2%</td><td rowspan=1 colspan=1>6.3% - 54.7%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>2/(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>2/(2+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>1/(1+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>20.7% - 100%</td><td rowspan=1 colspan=1>1/(1+0)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>20.7% - 100%</td></tr></table>

1In the case of Adenovirus 40/41, one of the clinical specimens that was concordant positive in the original GPP runs performed on raw stool yielded a negative result when tested in Cary-Blair (sample $\# 0 2 1 2 9$ ). MFI generated on in the original stool run were close to the assay cut off (195) suggesting a low titer specimen. Another specimen that was inhibited in the original stool runs performed on raw stool yielded a negative result in the Cary-Blair runs (sample $\# 0 1 3 6 6$ ). Lastly, one specimen that was positive for Adenovirus 40/41 by composite comparator was unavailable for re-testing in the Cary-Blair study (sample $\# 0 2 1 9 2 )$ ). For these reasons, positive agreement of xTAG GPP for Adenovirus 40/41 dropped from $8 0 \%$ (4/5) in the raw stool study to $40 \%$ (2/5) in the Cary-Blair evaluation. Refer to the results of further evaluation testing contrived samples close to the limit of detection (LoD) for Adenovirus 40/41 described in detail in the “Supplemental Study – Contrived Stool Specimens” section.

2ETEC comparator results were calculated against a composite consisting of four well characterized nucleic acid amplification tests (NAATs) followed by bi-directional sequencing. All specimens that were false negative by xTAG GPP for ETEC were positive by only one out of four comparator NAATs. Repeat sequencing of these specimens were negative by all four NAAT, except for one sample which was positive by one NAAT.

Overall, sensitivity/positive agreements generated in the stool in Cary-Blair study were comparable to those generated in the original clinical study performed on raw stool specimens.

Specificity/negative percentage agreement of xTAG GPP on stool in Cary-Blair medium is summarized for each individual target in the table below. For comparison purposes, performance results generated from the unpreserved stool as part of the prospective study as described in the “Prospective Clinical Study” section of this decision summary are also presented alongside of the performance results generated from the Cary-Blair preserved stool specimens.

<table><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=6>Specificity</td></tr><tr><td rowspan=1 colspan=3>Unpreserved Stool</td><td rowspan=1 colspan=3>Stool in Cary-Blair Media</td></tr><tr><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>1155/(1155+21)</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>97.3% - 98.8%</td><td rowspan=1 colspan=1>1268/(1268+9)</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.7% - 99.6%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>1131/(1131+53)</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>94.2% - 96.6%</td><td rowspan=1 colspan=1>1253/(1253+21)</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.5% - 98.9%</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>1149/(1149+19)</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.5% - 99.0%</td><td rowspan=1 colspan=1>1264/(1264+20)</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.6% - 99.0%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>1158/(1158+9)</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.5% - 99.6%</td><td rowspan=1 colspan=1>1269/(1269+9)</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.7% - 99.6%</td></tr><tr><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>1132(1132+39)</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>95.5% - 97.6%</td><td rowspan=1 colspan=1>1259(1259+22)</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>97.4% - 98.9%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>1023/(1023+96)</td><td rowspan=1 colspan=1>91.4%</td><td rowspan=1 colspan=1>89.6% - 92.9%</td><td rowspan=1 colspan=1>1144(1144+66)</td><td rowspan=1 colspan=1>94.5%</td><td rowspan=1 colspan=1>93.1% - 95.7%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>1143(1143+18)</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.6% - 99.0%</td><td rowspan=1 colspan=1>1237(1237+36)</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>96.1% -98.0%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>1154(1154+17)</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.7% - 99.1%</td><td rowspan=1 colspan=1>1275(1275+4)</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.2% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholera</td><td rowspan=1 colspan=1>1166/(1166+1)</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100%</td><td rowspan=1 colspan=1>1279/(1279+1)</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.6% -100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=6>Negative Agreement</td></tr><tr><td rowspan=1 colspan=3>Unpreserved Stool</td><td rowspan=1 colspan=3>Stool in Cary-Blair Media</td></tr><tr><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>1154(1154+13)</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>98.1% - 99.3%</td><td rowspan=1 colspan=1>1272/(1272+3)</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficiletoxin A/B</td><td rowspan=1 colspan=1>922/(922+105)</td><td rowspan=1 colspan=1>89.8%</td><td rowspan=1 colspan=1>87.8% - 91.5%</td><td rowspan=1 colspan=1>1020/(1020+89)</td><td rowspan=1 colspan=1>92.0%</td><td rowspan=1 colspan=1>90.2% - 93.4%</td></tr><tr><td rowspan=1 colspan=1>ETEC LT/ST</td><td rowspan=1 colspan=1>1156/(1156+4)</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.1% - 99.9%</td><td rowspan=1 colspan=1>1266/(1266+5)</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.1% - 99.8%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1162/(1162+2)</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4% - 100%</td><td rowspan=1 colspan=1>1277/(1277+1)</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>1153/(1153+16)</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>97.8% - 99.2%</td><td rowspan=1 colspan=1>1273/(1273+7)</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.9% - 99.7%</td></tr></table>

Overall, lower false positive results were observed in the stool in Cary-Blair study compared to the original clinical study performed on raw stool specimens. It is believed that this is mainly due to the fact that Cary-Blair clinical runs were conducted in accordance with the risk mitigations procedures aimed at preventing contamination requested by FDA during the initial review of xTAG GPP (k121454).

Results generated in this prospective clinical study demonstrate that the performance of xTAG GPP tested on stool stored in Cary-Blair media was equivalent to that of tested on unpreserved stools.

# Retrospective Clinical Study - Pre-Selected Clinical Specimens in Cary-Blair

In this study, all pre-selected Cary-Blair specimens were prepared from frozen stool mixed proportionally with Cary-Blair medium (at a ratio of 1:3, stool vs. Cary-Blair). These frozen specimens were remnants from the retrospective xTAG GPP clinical study as described in the “Retrospective Clinical Study 1 – Pre-Selected Clinical Specimens” section of this decision summary. A total of 81 specimens were included, including 40 Campylobacter, two E.coli O157, 26 Salmonella, and 13 Shigella positive specimens. All were characterized by bacterial culture. Although a smaller sample set was used for this study comparing to the original retrospective xTAG GPP clinical study (described above), positive agreement between comparator and xTAG GPP results was $100 \%$ for all preselected targets tested in this study.

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=2 colspan=1>95% ConfidenceInterval (CI)</td><td rowspan=2 colspan=1>Number of InvalidResults</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percentage</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.3% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.1% - 100%</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>77.2% - 100%</td><td rowspan=1 colspan=1>0</td></tr></table>

# Supplemental Study – Contrived Stool Specimens in Cary-Blair

Adenovirus 40/41

In order to assess whether Cary-Blair prospective clinical study results are an accurate representation of the performance of the assay for the Adenovirus 40/41 target, contrived specimens made from individual negative Stool specimens in Cary-Bair, were prepared at concentration spanning the analytical detection range of the assay and tested in a randomized fashion with negative specimens. Both Adenovirus 40 and 41 cultured isolates were tested and $50 \%$ of the samples were prepared at a concentration of 2XLoD. Results of this evaluation are presented in the table below.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Titer(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD(approximatedbased on real-time PCRassay)</td><td rowspan=1 colspan=1>Number ofContrivedSamples</td><td rowspan=1 colspan=1>AgreementwithExpectedPositiveResults</td><td rowspan=1 colspan=1>95%ConfidenceInterval (CI)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus40</td><td rowspan=3 colspan=1>ATCC</td><td rowspan=3 colspan=1>Type 40(Dugan)</td><td rowspan=1 colspan=1>2.90 x 101</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>100%(13/13)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2.32 x 102</td><td rowspan=1 colspan=1>16X</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>9.28 x 102</td><td rowspan=1 colspan=1>64X</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5>Adenovirus 40 Overall</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100%(25/25)</td><td rowspan=1 colspan=1>86.7% -100%</td></tr><tr><td rowspan=3 colspan=1>Adenovirus41</td><td rowspan=3 colspan=1>Zeptometrix</td><td rowspan=3 colspan=1>Type 41(Tak)</td><td rowspan=1 colspan=1>1.54 x 101</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>100%(12/12)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.23 x 102</td><td rowspan=1 colspan=1>16X</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>4.92 x 102</td><td rowspan=1 colspan=1>64X</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5>Adenovirus 40 Overall</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100%(25/25)</td><td rowspan=1 colspan=1>86.7% -100%</td></tr><tr><td rowspan=1 colspan=5>Adenovirus 40/41 Overall</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100%(50/50)</td><td rowspan=1 colspan=1>92.9% -100%</td></tr></table>

The results of this evaluation suggest that the addition of Cary-Blair does not impact the performance of xTAG GPP for Adenovirus 40/41near the limit of detection (LoD).

# E. histolytica and V. cholera

The performances of xTAG GPP for $E$ . histolytica and V. cholera were further assessed using contrived specimens in Cary-Blair.

The stool specimens in Cary-Blair used to create the contrived samples were obtained from anonymized donors. Prior to being used for matrix, the stools were screened with xTAG GPP to ensure they were negative for all xTAG GPP analytes, and that they had a present call for the spiked-in internal control (MS2). Stool specimens which did not meet these criteria were not used for this study.

A total of 50 unique specimens of stool in Cary-Blair were obtained from individual unique donors. The 50 stool in Cary-Blair specimens were then utilized to make 50 Entamoeba histolytica contrived samples, 50 Vibrio cholerae contrived samples and 50 negative samples.

The Entamoeba histolytica contrive samples were created by spiking the stool in CaryBlair with high titer culture material from three different strains of Entamoeba histolytica obtained from ATCC. The stock culture information and the concentrations used for the contrived samples are found in the table below.

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Titer of Stock(Cells/mL)</td><td rowspan=1 colspan=2>Titer of ContrivedSpecimens(Cells/mL)</td><td rowspan=1 colspan=1>Number ofContrivedSpecimens</td><td rowspan=1 colspan=1>Multiples ofLoD(approximatedbased on real-time PCRassay)</td></tr><tr><td rowspan=4 colspan=1>ATCC30890</td><td rowspan=4 colspan=1>HM-3:IMSS</td><td rowspan=4 colspan=1>1.34 x 104</td><td rowspan=1 colspan=1>5.76 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>4.61 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>16X</td></tr><tr><td rowspan=1 colspan=1>9.22 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>32X</td></tr><tr><td rowspan=1 colspan=2>1.84 x 103</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>64X</td></tr><tr><td rowspan=2 colspan=1>ATCC30459</td><td rowspan=2 colspan=1>HM-1:MISS [ABRM]</td><td rowspan=1 colspan=1>1.40 x 106</td><td rowspan=1 colspan=1>1.00 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>320X</td></tr><tr><td rowspan=1 colspan=1>1.27 x 107</td><td rowspan=1 colspan=1>3.00 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>960X</td></tr><tr><td rowspan=1 colspan=1>ATCC30458</td><td rowspan=1 colspan=1>200:NIH</td><td rowspan=1 colspan=1>1.27 x 107</td><td rowspan=1 colspan=1>1.00 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>320X</td></tr><tr><td rowspan=1 colspan=1>ATCC30015</td><td rowspan=1 colspan=1>200:NIH</td><td rowspan=1 colspan=1>9.60 x 106</td><td rowspan=1 colspan=1>3.00 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>960X</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1></td></tr></table>

The Vibrio cholerae contrive samples were created by spiking the stool in Cary-Blair with high titer culture material from two different strains of Vibrio cholerae obtained from NCTC. The stock culture information and the concentrations used for the contrived samples can be found in table below.

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Titer of Stock(Cells/mL)</td><td rowspan=1 colspan=1>Titer of ContrivedSpecimens(Cells/mL)</td><td rowspan=1 colspan=1>Number ofContrivedSpecimens</td><td rowspan=1 colspan=1>Multiples ofLoD(approximatedbased on real-time PCRassay)</td></tr><tr><td rowspan=5 colspan=1>NCTC12945</td><td rowspan=5 colspan=1>0139</td><td rowspan=5 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 107</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>43X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 108</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>130X</td></tr><tr><td rowspan=5 colspan=1>NCTC7260</td><td rowspan=5 colspan=1>01</td><td rowspan=5 colspan=1>6.00 x 108</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 107</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13X</td></tr><tr><td rowspan=1 colspan=1>1.00 x 108</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>43X</td></tr><tr><td rowspan=1 colspan=1>3.00 x 108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>130X</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1></td></tr></table>

The 50 sample contrived specimen panels for each of Entamoeba histolytica, Vibrio cholerae and the negative stool specimens were de-identified so that the sample identity and pathogen concentration was not determinable by the investigator, operator or any other individual associated with the study. After de-identification the samples were randomized and distributed to one study site. Each contrived specimen was processed and analyzed following the assay kit instructions.

The 50 negative stool in Cary-Blair contrived specimens produced the expected negative result for Entamoeba histolytica and Vibrio cholerae in 50/50 samples. The agreement with expected negative results is $100 \%$ (50/50) with a $9 5 \%$ confidence interval from $9 2 . 9 \%$ to $100 \%$ . On a per analyte bases the expected negative call for all analytes was obtained in 947/950 analytes. The three positive results obtained for the negative samples were for the targets Norovirus GII, Enterotoxigenic $E$ . coli (ETEC LT toxin) and Shiga-like toxin producing $E .$ . coli (stx2 toxin). Although the source Cary-Blair stool samples used to create this contrived sample set originally screened negative for xTAG GPP analytes, it appears that these three source samples contain low level analytes for these targets. This assessment is based on the observation that the three individual lots of stool in question reproducibly generated the unexpected positive call (i.e., Norovirus GII, ETEC LT toxin, and stx2 toxin in the Entamoeba histolytica and Vibrio cholera positive sample sets.

The table below summarizes agreements with expected positive results for $E$ . histolytica and V. cholerae.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Concentration (Cells/mL)</td><td rowspan=1 colspan=1>Multiples of LoD(approximated basedon real-time PCRassay)</td><td rowspan=1 colspan=1>Agreement withExpected PositiveResult</td><td rowspan=1 colspan=1>95% ConfidenceInterval (CI)</td></tr><tr><td rowspan=6 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>5.76 x 101</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>92.0% (22/24&#x27;)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>4.61 x 102</td><td rowspan=1 colspan=1>16X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>9.22 x 102</td><td rowspan=1 colspan=1>32X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.84 x 103</td><td rowspan=1 colspan=1>64X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.00 x 104</td><td rowspan=1 colspan=1>320X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3.00 x 104</td><td rowspan=1 colspan=1>960X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>E. histolytica Overall</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>96.0% (47/491)</td><td rowspan=1 colspan=1>86.3% - 98.9%</td></tr><tr><td rowspan=5 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>4.86 x 106</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.00 x 107</td><td rowspan=1 colspan=1>4X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3.00 x 107</td><td rowspan=1 colspan=1>13X</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1.00 x 108</td><td rowspan=1 colspan=1>43X</td><td rowspan=1 colspan=1>100% (9/9)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3.00 x 108</td><td rowspan=1 colspan=1>130X</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>V. cholera Overall</td><td rowspan=1 colspan=1>100.0% (50/50)</td><td rowspan=1 colspan=1>92.9% - 100%</td></tr></table>

1One Entamoeba histolytica contrived sample produced a $^ { \mathfrak { c } } \mathrm { N o }$ Call” as a result of the internal control failing to generate a present call.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Expected Value (As Determined by the xTAG GPP) Summary by Site for the xTAG GPP Prospective Clinical Evaluation (June 2011 – February 2012)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall (n=1407)</td><td rowspan=1 colspan=2>Site 1 (n=434)</td><td rowspan=1 colspan=2>Site 2 (n=428)</td><td rowspan=1 colspan=2>Site 3 (n=155)</td><td rowspan=1 colspan=2>Site 4 (n=260)</td><td rowspan=1 colspan=2>Site 5 (n=88)</td><td rowspan=1 colspan=2>Site 6 (n=42)</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>11.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>ETEC LT/ST</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>STEC (1/ 2)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>C. difficile Toxin A/B</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>15.6%</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>13.1%</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>14.7%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>18.1%</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>16.2%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>23.9%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>21.4%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>12.1%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>17.8%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>15.8%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>15.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Expected Value (As Determined by the xTAG GPP) Summary by Age Group for the xTAG GPP Prospective Clinical Evaluation (June 2011 – February 2012)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall (n=1407)</td><td rowspan=1 colspan=2>0-1 year (n=6)</td><td rowspan=1 colspan=2>&gt;1-5 years (n=20)</td><td rowspan=1 colspan=2>&gt;5-21 years (n=76)</td><td rowspan=1 colspan=2>&gt;21-65 years (n=879)</td><td rowspan=1 colspan=2>&gt;65 years (n=426)</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedVValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>5.2%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>ETEC LT/ST</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3.5%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>STEC (1/s 2)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>C. difficile Toxin A/B</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>15.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>17.1%</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>19.5%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>12.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>14.5%</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>11.5%</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>12.0%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

# N. Instrument Name:

Luminex 100/200

# O. System Descriptions:

1. Modes of Operation: Batch

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X___ or No

3. Specimen Identification:

Users must fill in Batch Information by providing a unique batch Name, Description and Creator. Users have to enter appropriate patient information, i.e. number of samples, and sample IDs.

4. Specimen Sampling and Handling:

DNA is extracted using the Biomerieux NucliSens EasyMag system. Samples are manually prepared for amplification according to assay package insert and, once amplified, are transferred to a 96-well microtiter plate for analysis on the Luminex system.

5. Calibration:

xMAP Calibrator Microspheres, Classification (CAL1) and Reporter (CAL2) serve as system calibrators for Luminex xMAP technology based detectors and are intended to normalize the settings for both the classification channel (CL1, CL2), the doublet discriminator channel (DD), and the reporter channel (RP1). They are not intended to be used as calibrators for a given assay.

6. Quality Control:

xMAP Control Microspheres, Classification (CON1) and Reporter (CON2) are intended to verify the calibration and optical integrity for the Luminex 100/200 System. Classification Control Microspheres verify both classification channels and the doublet discriminator channel (DD). Reporter Control Microspheres verify the reporter channel. They are not intended to be used as controls for a given assay which are described in the specific assay package insert.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.