# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K140029 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2)

# D. Type of Test:

An in vitro molecular diagnostic test for the direct, qualitative detection and differentiation of HSV-1 and HSV-2 DNA in cutaneous or mucocutaneous lesion specimens

# E. Applicant:

AmpliVue® HSV $1 + 2$ Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s): The AmpliVue® HSV $1 + 2$ Assay is an in vitro diagnostic test for the direct, qualitative detection and differentiation of Herpes Simplex Virus 1 (HSV-1) and Herpes Simplex

Virus 2 (HSV-2) DNA in cutaneous or mucocutaneous lesion specimens from symptomatic patients. The test is intended for use as an aid in diagnosis of HSV infection in symptomatic patients.

Warning: The AmpliVue® HSV $1 + 2$ Assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay is not intended for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: N/A

# I. Device Description:

The AmpliVue® HSV $1 + 2$ Assay is an in vitro diagnostic test for the qualitative detection and differentiation of HSV-1 and HSV-2 DNA isolated from cutaneous or mucocutaneous lesion specimens collected with swabs from symptomatic patients. The assay utilizes helicasedependent amplification (HDA) for the amplification of highly conserved HSV-1 and HSV-2 gene sequences and a self-contained disposable amplification detection device that allows for visual evaluation of assay results without analytical instrumentation.

The AmpliVue® HSV $1 + 2$ Assay consists of three major steps: 1) specimen preparation, 2) amplification of target gene sequences specific to HSV-1 and HSV-2 by HDA, and 3) detection of the amplified DNA by target-specific hybridization probes by a colorimetric reaction on a lateral-flow strip which is embedded in a self-contained disposable cassette to prevent amplicon contamination.

The AmpliVue® HSV $1 + 2$ Assay kit contains 16 tests per kit. The components are provided in a single box and are: 16 each of Dilution Tubes, Reaction Tubes, Detection Cassettes, and Amplicon Cartridges.

# J. Substantial Equivalence Information:

1. Predicate device name(s): IsoAmp® HSV Assay (Biohelix)

# Reference Method:

ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) for clinical evaluation (K971662)

2. Predicate 510(k) number(s): K111951

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Features</td><td colspan="1" rowspan="1">Quidel CorporationAmpliVue® HSV 1+2Assay (K140029)</td><td colspan="1" rowspan="1">BioHelix CorporationIsoAmp® HSV Assay(K111951)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The AmpliVue® HSV 1+2Assay is an in vitrodiagnostic test for the direct,qualitative detection anddifferentiation of HerpesSimplex Virus 1 (HSV-1)and Herpes Simplex Virus 2(HSV-2) DNA in cutaneousor mucocutaneous lesionspecimens fromsymptomatic patients. Thetest is intended for use as anaid in diagnosis of HSVinfection in symptomaticpatients.</td><td colspan="1" rowspan="1">The IsoAmp® HSV Assayis an in vitro diagnostic testfor the direct, qualitativedetection of the HerpesSimplex Virus (HSV-1 &amp;HSV-2) DNA in male andfemale genital and orallesions. The test is intendedfor use as an aid indiagnosis of HSV infectionin symptomatic patients.</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative/ Visual coloredband</td><td colspan="1" rowspan="1">Qualitative/ Visual coloredband</td></tr><tr><td colspan="1" rowspan="1">Detection of HSV-1 &amp;HSV-2</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Methodology</td><td colspan="1" rowspan="1">HDA (Helicase-DependentAmplification)</td><td colspan="1" rowspan="1">HDA (Helicase-DependentAmplification)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Features</td><td colspan="1" rowspan="1">Quidel CorporationAmpliVue® HSV 1+2Assay (K140029)</td><td colspan="1" rowspan="1">BioHelix CorporationIsoAmp® HSV Assay(K111951)</td></tr><tr><td colspan="1" rowspan="1">Typing of HSV-1 &amp;HSV-2</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Packaging</td><td colspan="1" rowspan="1">The product is supplied as akit in one box; 16 tests perkit1. Amplification-related KitComponents (ARKC)2. Non-amplification relatedKit Components (NKC)</td><td colspan="1" rowspan="1">The product is supplied astwo separate labeled boxes;50 tests per/kit1. Amplification-relatedKit Components (ARKC)2. Non-amplificationrelated Kit Components(NKC)</td></tr><tr><td colspan="1" rowspan="1">Kit Reagent StorageConditions</td><td colspan="1" rowspan="1">ARKC: 2°C to 8°CNKC: 2°C to 30°C</td><td colspan="1" rowspan="1">ARKC: &lt;-15°CNKC: 15-30°C</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

The AmpliVue® HSV $1 + 2$ Assay consists of three major steps: 1) specimen preparation, 2) amplification of target gene sequences specific to HSV-1 and HSV-2 by HDA, and 3) detection of the amplified DNA by target-specific hybridization probes via a colorimetric reaction on a lateral-flow strip which is embedded in a self-contained disposable cassette to prevent amplicon contamination.

Specimen preparation involves one dilution step in which specimens in viral transport medium are diluted 80-fold in Dilution Tubes. The diluted samples are transferred into a 0.2 mL Amplification Tube containing lyophilized HDA reagents. Incubation at $6 4 \mathrm { { } ^ { \circ } C }$ for 45 minutes results in the release of the HSV DNA and subsequent isothermal amplification of the target sequence. The amplified DNA is detected by a set of specific detection probes included in the Amplification Tube: HSV-1 target hybridizes to two specific probes, one labeled with Biotin (BioTEG) and the other with Digoxigenin (DIG), while HSV-2 target hybridizes to two specific probes labeled with Biotin (BioTEG) and 6-carboxyfluorescein (6- FAM). A competitive internal control (IC) is included in the Amplification Tube to monitor for inhibition, reagent failure or device failure. The IC target is amplified by HSV-2 specific primers and hybridizes to the biotin-labeled HSV-2 probe and an IC specific probe labeled with 2, 4-dinitrophenyl (DNP-TEG).

Detection of the amplified DNA with specific probes is achieved by AmpliVue cassettes. The self-contained AmpliVue cassettes carry lateral-flow DNA detection strips coated with antiDNP antibodies (C line), anti-DIG antibodies (T1 line) and anti-FAM antibodies (T2 line). HSV-1 amplicon complexed with BioTEG and DIG-labeled probes is captured by anti-DIG antibodies at the T1 line and HSV-2 amplicon complexed with BioTEG and FAM-labeled probes is captured by anti-FAM antibodies at the T2 line, while the IC amplicon complexed with BioTEG and DNP-labeled probes is captured by anti-DNP antibodies at the C line. The biotin in the amplicon-probe complexes captures the streptavidin-conjugated color particles for visualization and the test result is shown as colored lines that are visually read.

# Interpretation of the assay results:

· Positive: Always read the Test lines (T1 and T2) first.

o When T1 line is visible, report the assay result as “HSV-1 DNA detected”. o When T2 line is visible, report the assay result as “HSV-2 DNA detected”. o When T1 and T2 are visible, report the assay result as “HSV-1 and HSV-2 Positive: HSV-1 and HSV-2 DNA detected

Negative: When no visible T line is present (T1-/T2-), a visible C line indicates that the Internal Control DNA has been amplified and detected, eliminating the possibility of a false negative due to failure of amplification or device, and thus the assay result should be reported as negative - “no HSV DNA detected”. The C line intensity may vary with each test. Any pink to red colored visible line in the control signifies a valid test.

Invalid: If T and C lines are not present (T1- /T2-/C-), then the assay is invalid and the test needs to be repeated.

# Visual Interpretation of Assay Results

![](images/f33507fa75bb388e6923d71469a23bd0cbca6e35e4dd2a1a8fc87723ff31466f.jpg)

The interpretation of the assay results is carried out according to the following criteria:

<table><tr><td rowspan=1 colspan=1>Test line 1(T1) Reading</td><td rowspan=1 colspan=1>Test line 2(T2) Reading</td><td rowspan=1 colspan=1>Control line(C) Reading</td><td rowspan=1 colspan=1>Interpretation of result</td></tr><tr><td rowspan=1 colspan=1>T1+</td><td rowspan=1 colspan=1>T2-</td><td rowspan=1 colspan=1>C+ or C-</td><td rowspan=1 colspan=1>HSV-1 Positive: HSV-1 DNA detected</td></tr><tr><td rowspan=1 colspan=1>T1-</td><td rowspan=1 colspan=1>T2+</td><td rowspan=1 colspan=1>C+ or C-</td><td rowspan=1 colspan=1>HSV-2 Positive: HSV-2 DNA detected</td></tr><tr><td rowspan=1 colspan=1>T1+</td><td rowspan=1 colspan=1>T2+</td><td rowspan=1 colspan=1>C+ or C-</td><td rowspan=1 colspan=1>HSV-1 and HSV-2 Positive: HSV-1 andHSV-2 DNA detected</td></tr><tr><td rowspan=1 colspan=1>T1-</td><td rowspan=1 colspan=1>T2-</td><td rowspan=1 colspan=1>C+</td><td rowspan=1 colspan=1>Negative: No HSV-1 or HSV-2 DNAdetected</td></tr><tr><td rowspan=1 colspan=1>T1-</td><td rowspan=1 colspan=1>T2-</td><td rowspan=1 colspan=1>C-</td><td rowspan=1 colspan=1>Invalid: Failure due to inhibitory specimen,reagent failure, or device failure. Repeat testwith original specimen.</td></tr><tr><td rowspan=1 colspan=4>Note: The absence of a C line (control) in conjunction with a positive test line (T1, T2 or T1and T2) means that target material was successfully amplified. This occurs because of the overabundance of amplicons that generates competition with the internal control target.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility: The reproducibility of the AmpliVue® HSV $1 + 2$ Assay was evaluated at three test sites using a panel consisting of four panel members: HSV $1 + 2$ High Negative; HSV-1 Low Positive; HSV $1 + 2$ Moderate Positive; and HSV-2 Low Positive members. The HSV-1 Low Positive member served as a HSV-2 Negative member and the HSV-2 Low Positive member served as a HSV-1 Negative member. The panel members were prepared in HSV Negative Matrix that consisted of a pool of HSV negative cheek swabs in M4 medium. HSV Negative Matrix was spiked with quantified HSV-1 and HSV-2 stocks at pre-determined $\mathrm { T C I D } _ { 5 0 }$ concentrations. The HSV-1 and HSV-2 stocks were diluted in the HSV Negative Matrix to three (3) different concentration levels, defined as High Negative ( $0 . 3 { \bf x }$ LoD), Low Positive (1 x LoD) and Moderate Positive ( $3 \mathrm { ~ x ~ }$ LoD level).

Each run tested the four member panel in triplicate and also included three each of HSV-1 positive control, HSV-2 positive control and negative control. The external positive controls used were HSV-1 and HSV-2 plasmid DNA. Two (2) operators per test site each carried out one run of the four member panel plus controls per test day for five (5) days using one lot of the AmpliVue® HSV $1 + 2$ Assay.

Results of the reproducibility study for the AmpliVue® HSV $1 + 2$ Assay performed at three sites are presented in the tables below.

<table><tr><td rowspan=1 colspan=10>Reproducibility Study Summary for HSV-1</td></tr><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>Site</td><td rowspan=3 colspan=1>Rate ofDetection</td><td rowspan=3 colspan=1>OverallPercentAgreement</td><td rowspan=3 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=2>Site #1</td><td rowspan=1 colspan=2>Site #2</td><td rowspan=1 colspan=2>Site #3</td></tr><tr><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2HighNegative</td><td rowspan=1 colspan=1>16/30</td><td rowspan=1 colspan=1>47%</td><td rowspan=1 colspan=1>9/30</td><td rowspan=1 colspan=1>70%</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>45/90</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>40% - 60%</td></tr><tr><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>94% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2ModeratePositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr></table>

Reproducibility Study Summary for HSV-2   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>Site</td><td rowspan=3 colspan=1>Rate ofDetection</td><td rowspan=3 colspan=1>OverallPercentAgreement</td><td rowspan=3 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=2>Site #1</td><td rowspan=1 colspan=2>Site #2</td><td rowspan=1 colspan=2>Site #3</td></tr><tr><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2HighNegative</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>17/30</td><td rowspan=1 colspan=1>43%</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>57%</td><td rowspan=1 colspan=1>50/90</td><td rowspan=1 colspan=1>44%</td><td rowspan=1 colspan=1>35% - 55%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2ModeratePositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96% -100%</td></tr></table>

Precision:

The repeatability study used the same four member panel as used for the reproducibility study. As for the reproducibility study, the HSV-1 Low Positive member served as a HSV-2 Negative member and the HSV-2 Low Positive member served as a HSV-1 Negative member. The HSV-1 and HSV-2 stocks were diluted in the HSV Negative Matrix to three (3) different concentration levels, defined as High Negative ( $0 . 3 { \mathrm { ~ x ~ } }$ LoD), Low Positive (1 x LoD), and Moderate Positive ( $3 \mathrm { \Delta x }$ LoD level).

Each run tested the panel of four members in triplicate by two (2) operators, twice a day (2X) for twelve (12) days on all three instruments (triplicate testing $_ { \textrm { X 2 } }$ operators $\mathrm { ~ x ~ } 1 2 \mathrm { ~ d a y s = 7 } 2$ results per level for each virus). Positive and negative controls were run in triplicate with each test run. The external positive controls used were HSV-1 and HSV-2 plasmid DNA. Results of the Repeatability study for the AmpliVue® HSV $1 + 2$ Assay performed at one sites are presented in the tables below.

<table><tr><td rowspan=1 colspan=4>Repeatability Study Summary for HSV-1</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>OverallPercentAgreement</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2 High Negative</td><td rowspan=1 colspan=1>35/72</td><td rowspan=1 colspan=1>51%</td><td rowspan=1 colspan=1>40 - 63%</td></tr><tr><td rowspan=1 colspan=1>HSV-1Low Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2 Moderate Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2Low Positive</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Positive Control</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr></table>

b. Linearity/assay reportable range: N/A   

<table><tr><td rowspan=1 colspan=4>Repeatability Study Summary for HSV-2</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>OverallPercentAgreement</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2 High Negative</td><td rowspan=1 colspan=1>43/72</td><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>30- 52%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Low Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr><tr><td rowspan=1 colspan=1>HSV 1+2 Moderate Positive</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Low Positive</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Positive Control</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95- 100%</td></tr></table>

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Internal Control

The competitive internal control (IC) consists of plasmid DNA. The IC target sequence is amplified using the same primer set that amplifies the HSV-2 target sequence. The internal sequence of the IC target is different from the HSV target sequences and is detected by an IC-specific probe. After amplification, the IC amplicon-probe complexes are detected as a visible Control line by anti-DNP antibodies. The IC DNA and probes are pre-mixed in the Amplification Reagent.

# External Controls

The assay Positive Controls are commercially available external positive HSV-1 and HSV-2 from patient specimens. The controls are intended to monitor reagent and cassette failure. Previously characterized positive HSV-1 and HSV-2 specimens may be used in lieu of commercial HSV-1 and HSV-2 controls. The assay Negative Control consists of blank viral transport medium or previously characterized negative specimen and is used to detect reagent or environmental contamination (or carryover) by either HSV DNA or amplicons.

Note: For the analytical studies, HSV-1 and HSV-2 plasmid DNA were used as the positive external controls. The plasmid DNA contains HSV-1 or HSV-2 target sequences. For the clinical studies, positive HSV-1 and HSV-2 patient specimens were used as the external controls.

# d. Detection limit:

A Limit of Detection (LoD) study was performed to evaluate the analytical sensitivity of the AmpliVue® HSV $1 + 2$ Assay using two representative viral strains of HSV-1 (McIntyre & HF) and two representative strains of HSV-2 (G & MS). Quantified $\mathrm { ( T I C D _ { 5 0 } / m L ) }$ cultures of the HSV-1 and HSV-2 strains were serially diluted to five (5) concentrations in HSV-negative matrix pools and tested in replicates of ten (10) with three reagent lots. The observed LoD of a HSV strain was determined as the lowest concentration level that had a positivity rate of $2 9 5 \%$ . The observed LoDs for HSV-1 and HSV-2 were determined to be $1 . 1 \mathrm { \dot { x } } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , respectively.

Results from HSV LoD Panels   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>ConcentrationTCID50/mL</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>Positivityrate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=5 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=2>1.00 x106</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.33 x106</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>1.1 x10^{}$</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.70 x104</td><td rowspan=1 colspan=1>24/30</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>62.69%</td><td rowspan=1 colspan=1>90.50%</td></tr><tr><td rowspan=1 colspan=2>1.23 x104</td><td rowspan=1 colspan=1>8/30</td><td rowspan=1 colspan=1>26.7%</td><td rowspan=1 colspan=1>14.18%</td><td rowspan=1 colspan=1>44.45%</td></tr><tr><td rowspan=5 colspan=1>HSV-1HF</td><td rowspan=1 colspan=2>1.00 x106</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.33 x10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>1.11 x105</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.70 x104</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>59.07%</td><td rowspan=1 colspan=1>88.21%</td></tr><tr><td rowspan=1 colspan=2>1.23 x104</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>16.66%</td><td rowspan=1 colspan=1>47.88%</td></tr><tr><td rowspan=5 colspan=1>HSV-2G</td><td rowspan=1 colspan=2>1.00 x105</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.33 x104</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>1.11 x104</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.30%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.70 x103</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>78.68%</td><td rowspan=1 colspan=1>98.15%</td></tr><tr><td rowspan=1 colspan=2>1.23 x103</td><td rowspan=1 colspan=1>24/30</td><td rowspan=1 colspan=1>73.3%</td><td rowspan=1 colspan=1>55.55%</td><td rowspan=1 colspan=1>85.82%</td></tr><tr><td rowspan=5 colspan=1>HSV-2MS</td><td rowspan=1 colspan=2>1.00 x10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.33 x104</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.11 x104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.70 x103</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>1.23 x10^3</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>74.38%</td><td rowspan=1 colspan=1>96.54%</td></tr></table>

The LoDs determined in the initial study were confirmed with the same two HSV-1 and two HSV-2 reference strains diluted to the observed LODs $( 1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV-1 and $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV-2) and tested with twenty (20) replicates using three (3) lots of the AmpliVue HSV $1 + 2$ Assay. Since all HSV-1 and HSV-2 strains showed positivity rates of $2 9 5 \%$ with all three (3) validation lots, the observed LoDs were confirmed for both HSV-1 and HSV-2.

Clinical Isolates Testing: In addition, twenty (20) HSV-1 and twenty (20) HSV-2 clinical isolates were cultured and quantified in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Each isolate was diluted to the corresponding LoD in HSV-negative matrix and tested in triplicate. The AmpliVue HSV $1 + 2$ Assay was able to detect all 20 HSV-1 and 20 HSV-2 clinical isolates at their corresponding LoD.

Assay LoD: The final assay LoD claim is $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV-1 and $1 . 1 \mathrm { { x } }$ $1 0 ^ { 4 } \dot { \mathrm { T C I D } } _ { 5 0 } / \mathrm { m L }$ for HSV-2.

e. Analytical specificity:

A study was performed to evaluate the performance of the AmpliVue® HSV $1 + 2$ Assay in the presence of eighty-nine (89) microorganisms that might be found in cutaneous or mucocutaneous lesion specimens. The panel members were obtained from suppliers as purified genomic DNA (GD) or quantified cultures (QC), or prepared in house (IHC) by growing each organism and quantifying the culture. Each potentially interfering or cross-reactive microorganism was tested in three (3) replicates in negative matrix. Clinically relevant levels of viruses and bacteria are typically $1 0 ^ { 6 } \mathrm { c f u / m l }$ or higher for bacteria and $1 0 ^ { 5 } \mathrm { p f u / m l }$ or higher for viruses. Purified and quantified DNA was used for nine (9) of the microorganisms. For these microorganisms, $1 0 ^ { 6 }$ copies per ml $\left( \mathrm { c p / m l } \right)$ or higher was used. None of the eightynine (89) microorganisms showed cross-reactivity with the assay.

Cross Reactivity Panel   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Member Type(GD, QC , IHC )</td><td colspan="1" rowspan="1">Test concentration</td></tr><tr><td colspan="3" rowspan="1">Bacteria (N=52)</td></tr><tr><td colspan="1" rowspan="1">Acholeplasma laidlawi PG8</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">7.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.80 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">4.55 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis Z029</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">8.8 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.17 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis E431</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.73 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis VR-347</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.00 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis D-UW3</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">7.83 x 107 IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatisLGV-II 434 DNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.5 x 107 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniaeDNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.6 x 106 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile NAP1</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.77 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens Type A</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.06 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.44 x 106cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae Z101</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">5.70 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis VSE</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">8.60 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli ATCC 43895</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.13 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">8.05 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.20 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilis influenzae type A</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.00 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi Class IDNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">2.97 x 106 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.75 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus Z048</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.00 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.42 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisii V125 [DSM2711] ATCC 43063</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.2 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus mulieris ATCC35240</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.76 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella cartarrhalis Ne11</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.90 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis LBD-4</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.30 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hyorhinis BTS-7</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.6 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale CH 19299</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.08 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae M129</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.16 x 106 CCU/mL(Color Changing Units)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma salivarium H110</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.67 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae Z017</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">5.73 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidisSerogroupA</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">7.07 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenicaATCC 25845</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">7.3 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.19 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.32 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Salmonella enteriditis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">5.40 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.60 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MRSA</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">7.52 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MSSA</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">7.02 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidisMRSE</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.75 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.00 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.20 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.43 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans Z072</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.17 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.8 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes ATCC9898</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.38 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">2.75 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.6 x 106 tachyzoites/mL</td></tr><tr><td colspan="1" rowspan="1">Treponema pallidum Nichols</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.0 x 106Treponema pallidum/mL</td></tr><tr><td colspan="1" rowspan="1">Trichomonas vaginalis Z070</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.65 x 106trophzoites/mL</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma uralyticum NCTC10177 DNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.23 x 106 cp/mL</td></tr><tr><td colspan="3" rowspan="1">Yeast (N=7)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.00 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata Z007</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.73 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii Z008</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.96 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida krusei Z009</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">5.33 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae Z010</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.56 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis Z011</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.24 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis Z012</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 cfu/mL</td></tr><tr><td colspan="3" rowspan="1">Virus (N=30)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mexico/4108/2009H1N1</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.08 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 2</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.02 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7 VR-7</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.58 x 103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43 VR-1558</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.42 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.08 x 103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus AD-169</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.55 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11 ODH-37285</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.14 x 103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus B95-8</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">2.22 x 105 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Hong Kong VR-791</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">9.53 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Hepatitis B Virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.44 x 105 IU/mL</td></tr><tr><td colspan="1" rowspan="1">Hepatitis C Virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">7.58 x 105 IU/mL</td></tr><tr><td colspan="1" rowspan="1">HHV-8</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.26 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">HIV-1 Subtype B RNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.14 x 105 cp/mL</td></tr><tr><td colspan="1" rowspan="1">hMPV (Italy) A1</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.66 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 6 virus Z29strain</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.95 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 7 virus SB strain</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.15 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human papilloma virus 16 DNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">4.3 x 103 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Human papilloma virus 18 DNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.8 - 3.6 x 105 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.95 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">5.89 x 105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1 #2</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.97 x 105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaType 2</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.15 x10}$TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3 NY14</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">2.36 x 105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4B VR-1377</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.37 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV A Long VR-26</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.36 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV B Washington VR-1401</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.43 x 10TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rubella virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.17 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Simian Virus type 40 Pa-57ATCC strain VR-239</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">3.16 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">VZV DNA</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.5 x 105 cp/mL</td></tr></table>

# $f .$ Interference Studies

This study was performed to evaluate potential interference with the AmpliVue® HSV $1 + 2$ Assay with a panel of thirty-three (33) substances, five different viral transport media, and the 87 microorganisms from the cross reactivity panel that may be present in clinical specimens. The study was carried out in the presence of HSV-1 and HSV-2 at 3 times the limit of detection $( 3 \mathrm { ~ x ~ L o D } )$ to evaluate potential interference with the detection of the HSV targets. The study was also carried out in the absence of HSV to evaluate potential interference with the detection of the internal control of the AmpliVue® HSV $1 + 2$ Assay. Each potential interfering substance was tested in triplicate.

Interfering Substances: The analytical performance of the AmpliVue® HSV 1+2 Assay was characterized in the presence of interfering substances at the potentially highest (“the worst case”) concentrations to evaluate the susceptibility of the HSV assay to interference. Each interfering substance was introduced into the assay by directly wetting a clean, dry Remel M4 kit swab with the substance and placing the swab directly in transport medium. Calculated concentrations are based on an estimated volume of $2 0 0 \mu \mathrm { L }$ of substance introduced by the swab. Each panel member was tested in triplicate spiked with HSV-1 HF and HSV-2 G strains separately at $3 \mathrm { ~ x ~ }$ LoD. The panel was also tested in triplicate in the absence of HSV transport medium to see if the potentially interfering substances interfere with the detection of the internal control. No interference was observed in the presence of the potential interfering substances tested.

# Interfering Substance Panel

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Test conc.</td></tr><tr><td rowspan=1 colspan=1>Seminal Fluid</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Cornstarch</td><td rowspan=1 colspan=1>1.25 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Acetamidophenol</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Feces</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Acetylsalicylic Acid</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Chlorpheniramine</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Dextromethorphan</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Whole blood with EDTA</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Female Urine</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Male Urine</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Acyclovir (Acycloguanosine)</td><td rowspan=1 colspan=1>7 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>3.3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>7 mg/mL</td></tr><tr><td rowspan=1 colspan=1>K-Y Brand Jelly</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Douche</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Monistat 1</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Monistat 3</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Tioconazole 1</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Lanacane</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Listerine</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Abreva</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Carmex Cold Sore Lip Balm</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Releev cold sore treatment</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Crest</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Mucin (Bovine Submaxillary Gland, type I-S)</td><td rowspan=1 colspan=1>60 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Buffy coat</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>YeastGard</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Vagisil Crème</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Lip clear Lysine+</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole 3 Vaginal Cream</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Balneol Hygienic Cleansing Lotion</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Ortho Options Gynol II Extra Strength VaginalContraceptive Jelly</td><td rowspan=1 colspan=1>7%</td></tr></table>

Viral Transport Media: The performance of the AmpliVue® HSV $1 + 2$ Assay was assessed with the following viral transport media: Remel M4RT, Remel M5, Bartels VTM, and BD Universal Viral Transport (UVT)/Copan UTM. Remel M4 had previously been assessed and found to not interfere with the assay. Each transport medium was tested after spiking with HSV-1 HF and HSV-2 G strain to a final concentration of approximately $3 \mathrm { ~ x ~ }$ LOD to determine if the viral transport medium interferes with the detection of HSV targets in positive samples. The media were tested in the absence of HSV-1 and HSV-2 (transport medium only) to see if the viral transport media interfere with the detection of the internal control in negative samples.

There was no interference observed with the Remel M4RT, Remel M5, Bartels VTM, and BD UVT/Copan UTM media for the detection of HSV-1 and HSV- 2 target or the internal control. Remel M4RT, Remel M5, Bartels VTM, and BD UVT/Copan UTM did not interfere with the detection of HSV-1 and HSV-2 target or the internal control.

Cross-Reactivity Panel Members: The performance of the AmpliVue® HSV $1 + 2$ Assay was characterized by testing the eighty-nine (89) microorganisms that were evaluated for cross reactivity in the presence of HSV-1 HF and HSV-2 G at $3 \mathrm { ~ x ~ }$ LoD separately to see if the presence of any of these 89 organisms interferes with the detection of HSV targets. Each panel member was tested in triplicate. None of the cross reactivity panel members interfered with the detection of HSV-1 and HSV-2 target.

# g. Specimen Stability

A study was performed to assess the stability of HSV-1 and HSV-2 in viral transport media in accordance with recommended storage and handling specifications of each medium tested. Based on the supporting data, HSV-1 and HSV-2 were stable in all five (5) media for 7 days at $2 - 8 ^ { \circ } \mathrm { C } _ { \mathrm { i } }$ , and for 34 days at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

# h. Competitive Inhibition

The performance of the $\mathrm { A m p l i V u e } ^ { \mathbb { B } } \mathrm { H S V } 1 + 2$ Assay was assessed for competitive interference using simulated samples in two studies mimicking co-infections. The first study used simulated samples with one target at a concentration near the LoD (3 x LOD) and the other target at higher concentrations ( $3 0 \mathrm { x }$ LOD to $3 0 0 0 \mathrm { x }$ LOD). The second study used simulated samples that had equal concentrations of HSV-1 virus and HSV-2 virus $3 \mathrm { ~ x ~ L O D }$ to $3 0 0 0 \mathrm { { x } }$ LOD).

In the first study, competitive inhibition was not observed with simulated samples containing one target at a concentration near the LOD $( 3 \mathrm { ~ x ~ L O D } )$ and the other target at up to $3 0 0 \mathrm { ~ x ~ L O D }$ . However, competitive inhibition was observed for both HSV-1 and HSV-2 with simulated samples containing one target at a concentration near the LOD $( 3 \mathrm { ~ x ~ L O D } )$ and the other target at $3 0 0 0 \mathrm { x }$ LOD.

In the second study, competitive inhibition was not observed with simulated samples containing equal concentrations of HSV-1 virus and HSV-2 virus, from $3 \mathrm { ~ x ~ }$ LOD to $3 0 0 0 \mathbf { x }$ LOD.

# i. Carry-over/Cross Contamination

Test results confirm that carry-over/cross contamination does not occur with the AmpliVue® HSV $1 + 2$ Assay. High HSV-1 (HSV-2) positive samples were tested in series alternating with negative samples. In order to challenge the device, cultured and quantified viral stock served as high positive sample. The HSV-1 HF ${ 7 . 9 6 \times 1 0 ^ { 8 } }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) and HSV-2 G $( 2 . 2 7 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ stocks were used directly without dilution, for the highest concentration available. Remel M4 viral transport medium was used as the negative sample. Ten (10) replicates of negative sample together with assay controls were run by two (2) operators to confirm that negative samples (Remel M4 viral transport medium) generate a negative result $100 \%$ of the time. Five (5) replicates of high-concentration positive and negative samples were tested in a series with alternating sample types. All HSV-1 and HSV-2 high positive samples gave positive results and all the negative samples gave HSV negative results.

# 2. Comparison Studies:

# a. Method comparison with predicate device:

The method comparison/clinical performance evaluation was done against a gold standard/reference method using the FDA cleared, ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.), i.e., cell culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies. The testing description and data are listed in the Clinical Studies Section below (Section 3).

b. Matrix comparison: N/A

3. Clinical Studies:

a. Clinical Sensitivity: N/A b. Clinical Specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

The performance of the AmpliVue® HSV $1 + 2$ Assay was compared with the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) which is a gold standard reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance

The performance of the $\mathrm { A m p l i V u e } ^ { \mathbb { B } } \mathrm { H S V } 1 + 2$ Assay was evaluated at five geographically diverse locations within the United States. A total of one thousand three hundred and fifty-five (1,355) specimens from symptomatic male and female patients was tested. Three (3) cultures $( 0 . 2 \% )$ were unable to be read due to gross microbial contamination (there was insufficient quantity of each sample to repeat the culture). Four (4) specimens $( 0 . 3 \% )$ were positive for HSV in the ELVIS culture system, but the virus could not be identified as HSV-1 or HSV-2. Ten (10) specimens remained invalid upon repeat testing by the AmpliVue® HSV $1 + 2$ Assay, and two (2) specimens were unavailable for re-testing. These nineteen (19) specimens were removed from performance analysis. The ages of the patient population ranged from 5 years to 60 years. The swab specimens were categorized as cutaneous (e.g., skin lesion, genital - penis), or mucocutaneous (e.g., anorectal, genital – vaginal/cervical, nares, ocular, oral lesion and urethral).

The reference ELVIS viral culture used in this study is unable to detect co-infected specimens and cannot identify HSV-1 if HSV-2 is identified first. Consequently, if a specimen was positive for HSV-2, it was removed from the calculation of the HSV-1 clinical performance. Two hundred and eleven (211) specimens identified as HSV-2 positive by ELVIS were removed from the initial 1,336 specimens. The tables below show 1,125 HSV-1 specimens from cutaneous and mucocutaneous lesions.

<table><tr><td rowspan=1 colspan=9>Combined Sites  HSV-1 Cutaneous Lesions (N=340)</td></tr><tr><td rowspan=1 colspan=6>ReferenceMethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=4 colspan=1>AmpliVueHSV 1&amp;2Assay</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>98.5%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>301</td><td rowspan=2 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>340</td></tr></table>

<table><tr><td rowspan=1 colspan=9>Combined Sites - HSV-1 Mucocutaneous Lesions (N=785)</td></tr><tr><td rowspan=1 colspan=6>ReferenceMethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=4 colspan=1>AmpliVueHSV 1&amp;2Assay</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>94.9%</td><td rowspan=1 colspan=1>90.3%</td><td rowspan=1 colspan=1>97.4%</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>97.7%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>606</td><td rowspan=1 colspan=1>614</td><td rowspan=2 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>628</td><td rowspan=1 colspan=1>785</td></tr></table>

The tables below show the HSV-2 results for 1,336 specimens from cutaneous and mucocutaneous lesions.

<table><tr><td rowspan=1 colspan=9>Combined Sites - HSV-2 Cutaneous Lesions (N=399)</td></tr><tr><td rowspan=1 colspan=6>ReferenceMethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=4 colspan=1>AmpliVueHSV 1&amp;2Assay</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.0%</td><td rowspan=1 colspan=1>99.7%</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.6%</td><td rowspan=1 colspan=1>92.8%</td><td rowspan=1 colspan=1>97.3%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>325</td><td rowspan=1 colspan=1>326</td><td rowspan=2 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>399</td></tr></table>

<table><tr><td rowspan=1 colspan=9>Combined Sites - HSV-2 Mucocutaneous Lesions (N=937)</td></tr><tr><td rowspan=1 colspan=6>ReferenceMethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=4 colspan=1>AmpliVueHSV 1&amp;2Assay</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>93.4%</td><td rowspan=1 colspan=1>99.0%</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>94.2%</td><td rowspan=1 colspan=1>97.0%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>752</td><td rowspan=1 colspan=1>756</td><td rowspan=2 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>785</td><td rowspan=1 colspan=1>937</td></tr></table>

4. Clinical cut-off: N/A

5. Expected values/Reference range:

The prevalence of HSV-1 and HSV-2 with the AmpliVue® HSV $1 + 2$ Assay in cutaneous (e.g., skin lesion, genital - penis) or mucocutaneous (e.g., anorectal, genital – vaginal/cervical, nares, ocular, oral lesion and urethral) specimens was estimated for the 1,343 specimens with valid AmpliVue® HSV $1 + 2$ Assay results. Seven of the 1,343 specimens that had valid AmpliVue® HSV $1 + 2$ Assay results were not included in the performance analysis due to contamination or invalid results by the reference method; these seven samples were included in the calculations of prevalence

The prevalence of HSV-1 and HSV-2 with the AmpliVue® HSV $1 + 2$ Assay was calculated for the combined sites based on the age of the patient and the specific source of specimen and the results are presented below.

<table><tr><td rowspan=1 colspan=7>Combined Study  Cutaneous Prevalence by Age</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>HSV- 1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.7%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>19.1%</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>8.8%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>21.8%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>25.7%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositivePredictiveValue</td><td rowspan=1 colspan=1>76.9%</td><td rowspan=1 colspan=2>88.6% to 100%</td><td rowspan=1 colspan=1>79.5%</td><td rowspan=1 colspan=2>68.8% to 87.1%</td></tr><tr><td rowspan=1 colspan=1>NegativePredictiveValue</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=2>94.6% to 98.5%</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=2>98.3% to 99.9%</td></tr></table>

<table><tr><td colspan="7" rowspan="1">Combined Study  Cutaneous Prevalence by Specific Source</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">HSV- 1</td><td colspan="3" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Total #</td><td colspan="1" rowspan="1">TotalPositive</td><td colspan="1" rowspan="1">Prevalence</td><td colspan="1" rowspan="1">Total#</td><td colspan="1" rowspan="1">TotalPositive</td><td colspan="1" rowspan="1">Prevalence</td></tr><tr><td colspan="1" rowspan="1">Skin lesion</td><td colspan="1" rowspan="1">271</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">10.0%</td><td colspan="1" rowspan="1">271</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">17.7%</td></tr><tr><td colspan="1" rowspan="1">Genital - penis</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">19.1%</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">20.2%</td></tr><tr><td colspan="7" rowspan="1">Combined Study  Mucocutaneous Prevalence by Age</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">HSV- 1</td><td colspan="3" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">Age</td><td colspan="1" rowspan="1">Total #</td><td colspan="1" rowspan="1">TotalPositive</td><td colspan="1" rowspan="1">Prevalence</td><td colspan="1" rowspan="1">Total #</td><td colspan="1" rowspan="1">TotalPositive</td><td colspan="1" rowspan="1">Prevalence</td></tr><tr><td colspan="1" rowspan="1">≤ 5 years</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">25.6%</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2.6%</td></tr><tr><td colspan="1" rowspan="1">6 to 21 years</td><td colspan="1" rowspan="1">190</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">22.1%</td><td colspan="1" rowspan="1">190</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">17.9%</td></tr><tr><td colspan="1" rowspan="1">22 - 59 years</td><td colspan="1" rowspan="1">606</td><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">17.2%</td><td colspan="1" rowspan="1">606</td><td colspan="1" rowspan="1">132</td><td colspan="1" rowspan="1">21.8%</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">15.9%</td></tr><tr><td colspan="1" rowspan="1">Not provided</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%</td><td colspan="2" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%</td><td colspan="2" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">PositivePredictiveValue</td><td colspan="1" rowspan="1">87.1%</td><td colspan="2" rowspan="1">81.3% to 91.3%</td><td colspan="1" rowspan="1">81.8%</td><td colspan="2" rowspan="1">75.5% to 86.7%</td></tr><tr><td colspan="1" rowspan="1">NegativePredictiveValue</td><td colspan="1" rowspan="1">98.7%</td><td colspan="2" rowspan="1">97.5% to 99.3%</td><td colspan="1" rowspan="1">99.5%</td><td colspan="2" rowspan="1">98.6% to 99.8%</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Combined Study  Mucocutaneous Prevalence by Specific Source</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>HSV- 1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Total#</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>Anorectal</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>22.9%</td></tr><tr><td rowspan=1 colspan=1>Genital vaginal/cervical</td><td rowspan=1 colspan=1>691</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>15.9%</td><td rowspan=1 colspan=1>691</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>24.3%</td></tr><tr><td rowspan=1 colspan=1>Nasal</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>31.3%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>12.5%</td></tr><tr><td rowspan=1 colspan=1>Ocular</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td></tr><tr><td rowspan=1 colspan=1>Oral lesion</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>32.7%</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>Urethral</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16.7%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.