# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k)Num ber:

K140083

# B. PurposeofSubm ission:

The purpose of the submission is the premarket notification for the Enteric Pathogens Nucleic Acid Test which is intended to be run on the Verigene System.

# C. M easurand:

The Verigene Enteric Pathogens Nucleic Acid Test is performed on the automated Nanosphere Verigene System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific gastrointestinal microbial nucleic acid gene sequences:

· Campylobacter Group (comprised of C. coli, C. jejuni, and C. lari)   
· Salmonella species   
· Shigella species (including S. dysenteriae, S. boydii, S. sonnei, and S. flexneri) Vibrio Group (comprised of V. cholerae and V. parahaemolyticus)   
· Yersinia enterocolitica   
· Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers

# D. TypeofTest:

The Verigene Enteric Pathogens Nucleic Acid Test is a molecular assay which relies on detection of specific nucleic acid targets in a microarray format. For each of the bacterial nucleic acid sequences detected by Enteric Pathogens Nucleic Acid Test, unique Capture and Mediator oligonucleotides are utilized, with gold nanoparticle probe-based endpoint detection. The Capture oligonucleotides are covalently bound to the microarray substrate and hybridize to a specific portion of the nucleic acid targets. The Mediator oligonucleotides have a region which bind to a different portion of the same nucleic acid targets and also have a sequence which allows binding of a gold nanoparticle probe. Catalytic enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency and provide accurate detection of target capture.

# E. Applicant:

Nanosphere, Inc.

# F. Proprietary and Established Nam es:

Enteric Pathogens Nucleic Acid Test (EP)

# G .Regulatory Inform ation:

1. Regulation section:

21 CFR section 866.3990, Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification:

Class II

3. Product code:

PCH, PCI, OOI

4. Panel:

Microbiology (83)

# H .Intended Use:

1. Intended use(s):

The Verigene Enteric Pathogens Nucleic Acid Test (EP) is a multiplexed, qualitative test for simultaneous detection and identification of common pathogenic enteric bacteria and genetic virulence markers from liquid or soft stool preserved in Cary-Blair media, collected from individuals with signs and symptoms of gastrointestinal infection. The test is performed on the automated Nanosphere Verigene System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific gastrointestinal microbial nucleic acid gene sequences associated with the following pathogenic bacteria:

• Campylobacter Group (comprised of C. coli, C. jejuni, and C. lari)   
• Salmonella species   
• Shigella species (including S. dysenteriae, S. boydii, S. sonnei, and S. flexneri)   
• Vibrio Group (comprised of V. cholerae and V. parahaemolyticus)   
• Yersinia enterocolitica

In addition, EP detects the Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers. Shiga toxin producing $E$ . coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2.

EP is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness, in conjunction with other clinical, laboratory, and epidemiological information; however, is not to be used to monitor these infections. EP also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

Due to the limited number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Yersinia enterocolitica, Vibrio Group and Shigella species were primarily established with contrived specimens.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

EP results should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative EP results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use:

Same as Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

4. Special instrument requirements:

Verigene System

# I. DeviceDescription:

The Enteric Pathogens Nucleic Acid Test is performed on the Verigene System which is a fully automated, bench-top molecular diagnostics workstation. The System enables automated nucleic acid extraction from unformed stool specimens (liquid or soft) preserved in Cary-Blair media and detection of bacterial-specific target DNA. The Verigene System consists of two components: the Verigene Reader and the Verigene Processor $S P$ .

The Reader is the Verigene System’s user interface, which serves as the central control unit for all aspects of test processing, imaging, and result generation using a touch-screen control panel and a barcode scanner. The Verigene Processor $S P$ executes the test procedure, automating the steps of sample preparation, target amplification hybridization to the detection microarray. Detection and identification of bacterial-specific DNA is performed in a microarray format by using gold nanoparticle probe-based technology. Once the specimen is loaded by the operator, all other fluid transfer steps are performed by an automated pipette that transfers reagents between wells of the trays and finally

loads the specimen into the Test Cartridge for hybridization. Single-use disposable test consumables and a self-contained Verigene Test Cartridge are utilized for each sample tested with the Enteric Pathogens Nucleic Acid Test assay.

To obtain the test results after test processing is complete, the user removes the Test Cartridge from the Processor $S P$ , and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader and intensities from the microarray spots are used to make a determination regarding the presence (Detected) or absence (Not Detected) of a bacterial nucleic acid sequence/analyte. This determination is made by means of software-based decision algorithm resident in the Reader.

# J. SubstantialEquivalenceInform ation:

1. Predicate device name(s):

xTAG Gastrointestinal Pathogen Panel (GPP) by Luminex Molecular Diagnostics, Inc.

2. Predicate 510(k) number(s):

K121894

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:Enteric Pathogens Nucleic Acid Test(EP)K140083</td><td rowspan=1 colspan=1>Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The Verigene Enteric Pathogens NucleicAcid Test (EP) is a multiplexed,qualitative test for simultaneous detectionand identification of common pathogenicenteric bacteria and genetic virulencemarkers from liquid or soft stoolpreserved in Cary-Blair media, collectedfrom individuals with signs andsymptoms of gastrointestinal infection.The test is performed on the automatedNanosphere Verigene System utilizingreverse transcription (RT), polymerasechain reaction (PCR), and arrayhybridization to detect specificgastrointestinal microbial nucleic acidgene sequences associated with thefollowing pathogenic bacteria:•Campylobacter Group (comprised of C.coli, C. jejuni, and C. lari)Salmonella speciesShigella species (including S.dysenteriae, S. boydii, S. sonnei, and S.flexneri)</td><td rowspan=1 colspan=1>The xTAG® GastrointestinalPathogen Panel (GPP) is a multiplexednucleic acid test intended for thesimultaneous qualitative detection andidentification of multiple viral,parasitic, and bacterial nucleic acids inhuman stool specimens fromindividuals with signs and symptomsof infectious colitis or gastroenteritis.The following pathogen types,subtypes and toxin genes are identifiedusing the xTAG® GPP:Campylobacter (C. jejuni, C. coliand C. lari only)Clostridium difficile (C. difficile)toxin A/BCryptosporidium (C. parvum and C.hominis only)Escherichia coli (E. coli) O157Enterotoxigenic Escherichia coli(ETEC) LT/ST• Giardia (G. lamblia only - also</td></tr></table>

Page - 5/45   

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:Enteric Pathogens Nucleic Acid Test(EP)K140083</td><td rowspan=1 colspan=1>Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Vibrio Group (comprised of V. choleraeand V. parahaemolyticus)Yersinia enterocoliticaIn addition, EP detects the Shiga toxin 1gene and Shiga toxin 2 gene virulencemarkers. Shiga toxin producing E. coli(STEC) typically harbor one or bothgenes that encode for Shiga Toxins 1 and2.EP is indicated as an aid in the diagnosisof specific agents of gastrointestinalillness, in conjunction with other clinical,laboratory, and epidemiologicalinformation; however, is not to be used tomonitor these infections. EP also aids inthe detection and identification of acutegastroenteritis in the context of outbreaks.Due to the limited number of positivespecimens collected for certain organismsduring the prospective clinical study,performance characteristics for Yersiniaenterocolitica, Vibrio Group and Shigellaspecies were primarily established withcontrived specimens.Concomitant culture is necessary fororganism recovery and further typing ofbacterial agents.EP results should not be used as the solebasis for diagnosis, treatment, or otherpatient management decisions.Confirmed positive results do not rule outco-infection with other organisms that arenot detected by this test, and may not bethe sole or definitive cause of patientillness. Negative EP results in the settingof clinical illness compatible withgastroenteritis may be due to infection bypathogens that are not detected by thistest or non-infectious causes such asulcerative colitis, irritable bowelsyndrome, or Crohn&#x27;s disease.</td><td rowspan=2 colspan=1>known as G. intestinalis and G.duodenalis)Norovirus GI/GIIRotavirus ASalmonellaShiga-like Toxin producing E. coli(STEC) stx 1/stx 2Shigella (S. boydii, S. sonnei, S.flexneri and S. dysenteriae)The detection and identification ofspecific gastrointestinal microbialnucleic acid from individualsexhibiting signs and symptoms ofgastrointestinal infection aids in thediagnosis of gastrointestinal infectionwhen used in conjunction with clinicalevaluation, laboratory findings andepidemiological information. Agastrointestinal microorganismmultiplex nucleic acid-based assayalso aids in the detection andidentification of acute gastroenteritisin the context of outbreaks.xTAG® GPP positive results arepresumptive and must be confirmedby FDA cleared tests or otheracceptable reference methods.The results of this test should not beused as the sole basis for diagnosis,treatment, or other patientmanagement decisions. Confirmedpositive results do not rule outcoinfection with other organisms thatare not detected by this test, and maynot be the sole or definitive cause ofpatient illness. Negative xTAGGastrointestinal Pathogen Panelresults in the setting of clinical illnesscompatible with gastroenteritis may bedue to infection by pathogens that arenot detected by this test or non-infectious causes such as ulcerativecolitis, irritable bowel syndrome, orCrohn&#x27;s disease. xTAG GPP is notintended to monitor or guide treatmentfor C. difficile infections.The xTAG GPP is indicated for use</td></tr><tr><td rowspan=1 colspan=1>test, and </td></tr></table>

Page - 6/45   

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:Enteric Pathogens Nucleic Acid Test(EP)K140083</td><td rowspan=1 colspan=1>Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>with the Luminex MAGPIXinstrument.</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Human Stool sample</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DNAAmplification</td><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Organisms/NATargets Detected</td><td rowspan=1 colspan=1>Campylobacter Group(C. coli, C. jejuni, and C. lari)Salmonella speciesShigella species(S. dysenteriae, S. boydii, S. sonnei, and S.flexneri)Yersinia enterocoliticaShiga toxin 1 gene and Shiga toxin 2 genevirulence markers</td><td rowspan=1 colspan=1>Same with additional analytes</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:Enteric Pathogens Nucleic Acid Test(EP)</td><td rowspan=1 colspan=1>Predicate:</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>~ 2 hours</td><td rowspan=1 colspan=1>5 hours</td></tr><tr><td rowspan=1 colspan=1>Sample prep</td><td rowspan=1 colspan=1>On-board, automated NA extraction andamplification</td><td rowspan=1 colspan=1>Off-line NA Extraction andamplification</td></tr><tr><td rowspan=1 colspan=1>DetectionMethod</td><td rowspan=1 colspan=1>Gold/Silver nanoparticle probe detectionof bacterial-specific DNA oncomplementary oligo- microarray</td><td rowspan=1 colspan=1>Specific microbial target or controlbead populations coupled to sequencesfrom Universal Array streptavidin, R-phycoerythrin conjugate</td></tr><tr><td rowspan=1 colspan=1>OpticalDetection</td><td rowspan=1 colspan=1>Image analysis of visible light imagefrom CCD</td><td rowspan=1 colspan=1>Multi-color fluorescence</td></tr></table>

# K .Standard/G uidanceDocum entReferenced:

CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline - Second Edition

CLSI EP12-A2; User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition

CLSI MM3-A2; Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline - Second Edition

CLSI EP15-A2 - User Verification of Performance for Precision and Trueness; Approved Guideline – second edition

CLSI EP9-A2-IR - Method Comparison and Bias Estimation Using Patient Samples; Approved Guideline – second edition (Interim Revision)

CLSI EP17-A - Protocols for Determination of Limits of Detection and Limits of Quantitation; Approved Guideline

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices

Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium Difficile – Draft Guidance for Industry and FDA Staff

In Vitro Diagnostic (IVD) Device Studies – Frequently Asked Questions

Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests

# L. TestPrinciple:

The Enteric Pathogens Nucleic Acid Test is performed on samples of stool preserved in Cary-Blair media, collected from individuals suspected of having acquired an enteric infection. The provided flocked swab is dipped into the specimen until flocked tip is fully immersed. Once evenly coated, the swab is transferred to the Stool Prep Buffer tube and broken at the pre-formed scored breakpoint. The Stool Prep Buffer tube is sealed with a screw cap and thoroughly vortexed and centrifuged to ensure a proper mixture of the specimen. Approximately $2 0 0 \mu \mathrm { L }$ of the prepared Stool Prep Buffer (SLB) is pipetted into the Specimen Well of the Extraction Tray and the Drawer Assembly is closed to initiate the test. An automated pipettor performs all further fluidic transfers during the Sample Extraction and Amplification steps.

Sample processing and analysis occurs in two phases: (i) Sample Preparation – Cells are lysed and purified by a magnetic bead-based bacterial DNA isolation procedure. The Sample Preparation steps occur within the Processor SP’s Extraction Tray, which is preloaded with all reagents required to extract purified nucleic acid from the blood culture specimen. (ii) Target Amplification – Purified nucleic acid is amplified using a multiplex PCR based amplification to generate target-specific amplicons. Amplification of the extracted DNA occurs on the Amplification Tray. At the end of these processes, the extracted and amplified DNA is mixed with sample buffer and transferred to the Test Cartridge.

The Test Cartridge consists of two parts: a Reagent Pack containing reservoirs preloaded with reagents required for various steps of a specific test in the Processor $S P$ , and a Substrate Holder which houses the microarray. Within the Test Cartridge, the bacterial DNA hybridizes to complementary sequence-specific DNA oligonucleotides, known as capture oligonucleotides, arrayed on the surface of a microarray within the Substrate Holder. The “captured DNA” also hybridizes to a second DNA oligonucleotide, known as a mediator oligonucleotide, which is contained in the test-specific sample buffer. The mediator oligonuceotide contains two sequence domains: one domain is complementary to the genomic DNA target and a second domain is complementary to a common oligonucleotide attached to a signal generating gold nanoparticle probe. After washing away any DNA not hybridized to the captures, the probe is exposed to the captured mediator/target complex. Presence of the gold nanoparticle probes at a particular location on the substrate is then further amplified with silver enhancement reagents. All the reagents required for the above steps are contained within the Reagent Pack and are sequentially added to the microarray and transferred out to the waste well. Once the test processing is complete, the Test Cartridge is removed from the Processor $S P$ and the Reagent Pack is removed and discarded. The remaining Test Substrate is now ready for imaging and analysis on the Reader. The high detection sensitivity derived from evanescence-induced light scatter after the electrocatalytic enhancement of the bound nanoparticle probes is measured by optical methods. The relative intensity of light scatter detected from each arrayed test site is measured. After background subtraction, the signal intensities are compared to a signal threshold for a decision regarding the presence or absence of target. A final ‘Detected’ or ‘Not Detected’ result is provided for each analyte in the test. These results are linked to the test and patient information entered at the beginning of each test session to provide a results report.

# M .Perform anceCharacteristics(if/when applicable):

1. Sample Preparation and Stability

Simulated Samples for Analytical Validation Studies

For the analytical studies, target organisms were spiked into a Negative Stool Matrix (NSM) pool, whereas the clinical simulated specimens were prepared by spiking target organism directly into individual negative stool specimens preserved in Cary-Blair media. With the exception of the specimen stability study that used fresh clinical samples, all of the analytical studies described below involved testing of simulated samples using a Negative Stool Matrix (“NSM”); preparation of this matrix is described below. Simulated samples are prepared by suspending cultured and quantified strains of target bacterial organisms in the NSM.

The NSM was prepared by pooling individual de-identified residual stool specimens preserved in Cary-Blair medium. These individual “Cary- Blair specimens” were originally prepared by diluting unpreserved stool specimens (obtained from individuals in the intended use population) in Cary-Blair medium such that the stool comprises $2 5 \%$ by volume of the resulting specimen. Each individual Cary-Blair specimen was confirmed negative for target organisms by testing the specimen in duplicate with the Enteric Pathogens Nucleic Acid Test (EP) test. Obtaining duplicate negative results for all panel analytes qualified the specimen for inclusion in the negative pool. NSM was formulated by pooling the negative Cary-Blair specimens, combining with Enteric Stool Prep Buffer in a 1:5 ratio, and clarifying by centrifugation and collection of the supernatant. The NSM pool was again tested with the Enteric Pathogens Nucleic Acid Test in replicates of 20 and confirmed as negative.

To prepare the simulated samples for analytical testing, the target bacterial organism was diluted and added to the NSM to achieve the appropriate organism concentration. These simulated specimens were either used immediately (within $3 0 \mathrm { { m i n } }$ of preparation) or stored frozen at ${ < } . 7 0 { } ^ { \circ } \mathrm { C }$ until use.

# Simulated Samples for Clinical Validation Study

Simulated samples used during the clinical validation study were prepared in a similar manner to the samples used for analytical testing with one difference. During the clinical validation study, target organisms were spiked directly into individual stool specimens preserved in Cary-Blair media which were confirmed negative by the Enteric Pathogens Nucleic Acid Test prior to entry into the study.

# Frozen Simulated Sample Stability

The purpose of this study was to establish the performance of frozen simulated samples for use during clinical and analytical validation. This study tested six representative bacterial organisms (see Table 1). For each organism five concentrations were prepared in a 3-fold serial dilution series using NSM. A sufficient starting volume of each simulated sample was prepared to make at least eight individual single-use vials at each dilution, each containing approximately $3 0 0 \mu \mathrm { L }$ of sample. Baseline testing of the freshly-prepared samples was performed in replicates of four; the four remaining vials were stored frozen at $\leq - 7 0 ^ { \circ } \mathrm { C }$ then thawed and tested. The starting concentration of each sample was chosen to approximately bracket the Limit of Detection (LoD) determined for each of the analytes (see Section 2). Initial testing was performed at a concentration $3 \mathbf { x }$ higher than the established analytical LoD with 20 replicates. If a 20/20 detection rate was obtained, the previously established analytical LoD level was tested in replicates of 20 to test whether the detection rate at this level was less than 19/20. If the initial results yielded 19/20 detection rate, no further testing was conducted. If the initial results yielded less than 19/20 correct results, higher concentrations (6x and $9 \mathbf { x }$ the analytical LoD) were tested until the ${ \geq } 1 9 / 2 0 $ detection rate was met. The results of this study as shown in Table 1 are acceptable to establish the performance of frozen simulated samples for use in the clinical and analytical validation studies.

<table><tr><td rowspan=1 colspan=5>Table 1: Summary Results for Fresh Versus Frozen Study</td></tr><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>DetectedTarget</td><td rowspan=1 colspan=2>LoD (CFU/mL)</td><td rowspan=2 colspan=1>ReferencetoAnalyticalLoD</td></tr><tr><td rowspan=1 colspan=1>FreshSample</td><td rowspan=1 colspan=1>FrozenSample</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp jejuni</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp enterica serovarTyphi</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>9.99x105</td><td rowspan=1 colspan=1>3x</td></tr></table>

Page - 10/45   

<table><tr><td rowspan=1 colspan=5>Table 1: Summary Results for Fresh Versus Frozen Study</td></tr><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>DetectedTarget</td><td rowspan=1 colspan=2>LoD (CFU/mL)</td><td rowspan=2 colspan=1>ReferencetoAnalyticalLoD</td></tr><tr><td rowspan=1 colspan=1>FreshSample</td><td rowspan=1 colspan=1>FrozenSample</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae(Stx 1)</td><td rowspan=1 colspan=1>Shigella;Shiga Toxin 1</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>2.22x105</td><td rowspan=1 colspan=1>6x</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Y.enterocolitica</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>6.66x105</td><td rowspan=1 colspan=1>6x</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli/ Stx2</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>1x</td></tr></table>

2. Analytical performance:

# Precision:

The precision of the Enteric Pathogens Nucleic Acid Test was established by conducting an internal study. The study was conducted for four days and included the following sources of variability: operators, days, consumable lots, and runs. Fourteen unique samples were tested daily in duplicate by two operators for four nonconsecutive days for a total of 16 tests per sample (2 operators / day x 2 replicates / operator $\mathrm { _ { x } 4 d a y s = 1 6 }$ tests per sample). Positive and negative controls were run once per combination of consumable lots received. The 14 sample panel for the precision study was comprised of six unique strains at two concentrations (12 positive samples) and two negative samples (Negative Stool Matrix and Clostridium difficile), as listed in Table 2. Previously characterized and quantified pure Tris stocks were inoculated into screened negative stool matrix (NSM) at a Moderate Positive (defined as approximately $2 { - } 5 \mathrm { x } \ \mathrm { L o D } $ ) and a Low Positive (defined as approximately $1 - 2 \mathbf { x }$ LoD) concentrations. Each unique specimen was divided into single-use aliquots of $3 0 0 \mu \mathrm { L }$ each and frozen at $\leq - 7 0 ^ { \circ } \mathrm { C }$ .

<table><tr><td rowspan=1 colspan=4>Table 2: Precision and reproducibility test panelcomposition</td></tr><tr><td rowspan=2 colspan=1>Description</td><td rowspan=2 colspan=3>ConcentrationCFU/mL     MultiplesLabel(estimated)  of LoD</td></tr><tr><td rowspan=1 colspan=1>CFU/mL(estimated)</td></tr><tr><td rowspan=1 colspan=1>Negative StoolMatrix</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Escherichiacoli/Stx2</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>5.50x105</td><td rowspan=1 colspan=1>2-5x</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.20x105</td><td rowspan=1 colspan=1>1-2x</td></tr><tr><td rowspan=2 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>5.50x105</td><td rowspan=1 colspan=1>2-5x</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.20x10</td><td rowspan=1 colspan=1>1-2x</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>1.85x105</td><td rowspan=1 colspan=1>2-5x</td></tr></table>

Page - 11/45   

<table><tr><td rowspan=1 colspan=4>Table 2: Precision and reproducibility test panelcomposition</td></tr><tr><td rowspan=2 colspan=1>Description</td><td rowspan=2 colspan=3>ConcentrationCFU/mL     MultiplesLabel(estimated)   of LoD</td></tr><tr><td rowspan=1 colspan=1>CFU/mL(estimated)</td></tr><tr><td rowspan=1 colspan=1>dysenteriae/Stx1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>7.40x104</td><td rowspan=1 colspan=1>1-2x</td></tr><tr><td rowspan=2 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>5.55x105</td><td rowspan=1 colspan=1>2-5x</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.22x105</td><td rowspan=1 colspan=1>1-2x</td></tr><tr><td rowspan=2 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>5.55x10{}$</td><td rowspan=1 colspan=1>2-5x</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.22x105</td><td rowspan=1 colspan=1>1-2x</td></tr><tr><td rowspan=2 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>1.85x105</td><td rowspan=1 colspan=1>2-5x</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>7.40x104</td><td rowspan=1 colspan=1>1-2x</td></tr></table>

The initial call rate for the Precision Study was $9 8 . 2 \%$ (220/224). There were four initial “No Call” results. These samples were re-tested according to the retesting rules found in the package insert and all yielded a valid test result upon retesting. The final study results for the negative panel members, moderate positive samples and low positive samples agreed $9 9 . 6 \%$ with the expected results. The summary results of the Precision Study as shown in Table 3 establish acceptable within-laboratory precision of the Enteric Pathogens Nucleic Acid Test.

<table><tr><td rowspan=1 colspan=3>Table 3: Precision Study Summary Results</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Final Percent Agreementwith Expected Result(95 % CI)</td></tr><tr><td rowspan=2 colspan=1>Escherichiacoli/Stx2</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=2 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>93.8%15/16***(69.8%-99.8%)</td></tr><tr><td rowspan=2 colspan=1>Shigelladysenteriae/Stx1</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=2 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr></table>

Page - 12/45   

<table><tr><td rowspan=1 colspan=3>Table 3: Precision Study Summary Results</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Final Percent Agreementwith Expected Result(95 % CI)</td></tr><tr><td rowspan=2 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=2 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative StoolMatrix</td><td rowspan=2 colspan=1>N/A</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Clostridiumdifficile (NegativeControl)</td><td rowspan=1 colspan=1>100%16/16(79%-100%)</td></tr></table>

\*\*\*One sample expected to call “Salmonella” called “Salmonella” and “Stx2”.

# Reproducibility:

The inter-laboratory reproducibility of the Enteric Pathogens Nucleic Acid Test was established by a three external site study. The study included five days of testing incorporating the following sources of imprecision: sites, operators, days, consumable lots and runs. Fourteen unique samples were tested daily in triplicate by two operators for five non-consecutive days at three sites for a total of 90 tests per sample (3 sites x 2 operators / site x 3 replicates / operator $\mathrm { \bf ~ x } 5 \mathrm { d a y s } = 9 0 $ tests per sample). Positive and negative controls were run at each site according to the study protocol.

The 14 sample panel for this study was the same panel previously described in the precision study and was comprised of six unique strains at two concentrations (12 positive samples) and two negative samples (Negative Stool Matrix and Clostridium difficile), as listed in Table 2. For each strain, the panel included a “Low Positive” sample, which would be expected to produce a positive result approximately $9 5 \%$ of the time, and a “Moderate Positive” sample, which would be expected to yield a positive result approximately $9 9 \%$ of the time.

The results of the reproducibility study are summarized in Table 4. This table shows the agreement between the expected results and the obtained results for each sample tested. A sample which yielded an initial ‘No Call’ test result with the Enteric Pathogens Nucleic Acid Test was repeated from the original sample (new aliquot) per the package insert instructions and the repeat result was considered the final result. A sample which yielded a Pre-Analysis Error was also repeated using the original

sample (new aliquot) per the package insert instructions and the repeat results were considered the final result.

<table><tr><td rowspan=1 colspan=3>Table 4: Reproducibility Study Summary Data</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration(Sample ID #)</td><td rowspan=1 colspan=1>Final Percent Agreementwith Expected Result (95 %CI</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli/Stx2</td><td rowspan=1 colspan=1>Moderate(#2)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=1 colspan=1>Low(#3)</td><td rowspan=1 colspan=1>89/9098.9%(94-100)</td></tr><tr><td rowspan=2 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>Moderate(#4)</td><td rowspan=1 colspan=1>88/9097.8%(92.2-99.7)</td></tr><tr><td rowspan=1 colspan=1>Low(#5)</td><td rowspan=1 colspan=1>86/9095.6%(89-98.8)</td></tr><tr><td rowspan=2 colspan=1>Shigella dysenteriae/Stx1</td><td rowspan=1 colspan=1>Moderate(#6)</td><td rowspan=1 colspan=1>88/9097.8%(92-99.7)</td></tr><tr><td rowspan=1 colspan=1>Low(#7)</td><td rowspan=1 colspan=1>86/9095.6%(89-98.8)</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Moderate(#8)</td><td rowspan=1 colspan=1>89/9098.9%(94-100)</td></tr><tr><td rowspan=1 colspan=1>Low(#9)</td><td rowspan=1 colspan=1>80/9088.9%(80.5-94.5)</td></tr><tr><td rowspan=2 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Moderate(#10)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=1 colspan=1>Low(#11)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=2 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Moderate(#16)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=1 colspan=1>Low(#17)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=1 colspan=1>Negative StoolMatrix</td><td rowspan=3 colspan=1>N/A</td><td rowspan=2 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=2 colspan=1>Clostridium difficile(Negative Control)</td></tr><tr><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr></table>

The percent agreement across all sites was $9 9 . 1 \%$ . There were 24 discordant calls 19 of which were observed at the low positive concentration. The final study results for the negative panel members were $100 \%$ agreement with the expected results. For the moderate positive samples, it is expected that the target(s) present in the sample will be detected approximately $9 9 \%$ of the time; in this study, the targets were detected at an acceptable rate of $9 9 . 1 \%$ . Undetected targets were observed with five moderate samples No. 4 (Salmonella), No. 6 (Shigella/Stx1), and No. 8 (Y. enterocolitica) at two different sites. In the case of sample No. 6, only the Stx1 target of the sample was not detected. These results suggest that the observations could be explained as random events, likely related to variations in the concentration, characteristics, and stability of the prepared samples. Concentration of the sample was surely a factor for the low positive samples. It was expected that the target(s) in the low positive sample would be detected approximately $\ge 9 5 \%$ ; in this study, combined, the low positive sample targets were detected at an acceptable rate of $9 6 . 7 \%$ . Undetected targets were observed a total of 18 times with low samples No. 3 ( $E .$ . coli / Stx2), No. 5 (Salmonella), No. 7 (Shigella/Stx1), and No. 9 (Y. enterocolitica) at three different sites. In the case of sample No. 7, only the Stx1 target of the sample was not detected once and only the Shigella target of the sample was not detected twice. Unexpectedly on one occasion, Sample No. 7 also generated one false positive result, with Stx2 being detected in addition to Shigella and Stx1.

Linearity/assay reportable range:

Not applicable.

Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Processing Controls

Enteric Pathogens Nucleic Acid Test (EP) is a ‘specimen-to-result’ detection system wherein nucleic acids are isolated and identified by an automated process. In order to decrease the likelihood of false results due to processing errors, all reagents are prepackaged in single-use disposables which are actuated by the analysis instrument. The following controls are built into the system to so that processing failures are identified and the appropriate actions are taken.

An artificial DNA construct (INTL CTL 1 see Table 5) serves as the internal hybridization control or “positive control for detection.” It is a synthetic DNA construct present in the sample buffer and is hybridized in parallel with every test sample. For a valid Negative test result, INTL CTL 1 needs to be “Detected”. If the INTL CTL 1 is not valid, a no call result will be returned and the test should be repeated according to the instructions in the package insert.

An MS2 Phage serves as a specimen isolation and amplification control and is referred to as the Internal Processing Control 2 (INT CTL 2 see Table 5). This control is added to each specimen prior to the extraction step. For a valid Negative test result, both IC1 and IC2 must be “Detected.” If IC1 or IC2 are not detected a “No Call –

INT CTL 1” or a “No Call – INT CTL $2 ^ { \circ }$ is provided respectively. If both IC1 and IC2 are not detected, a “NO CALL – INT CTL” result is provided. The recommended action for any “No Call” decision is to repeat the test according to the instructions package insert.

The Imaging Control (IC) is a set of oligonucleotide spots on the Test Substrate that act as a quality check for proper fluid control and movement between the Test Cartridge and the Reagent Pack. Inability to detect the imaging controls results in a “No Call – No Grid.” The Imaging Control signal is generated when a signal probe hybridizes to an oligonucleotide on the substrate. Signal at these spots indicates that the steps of probe hybridization and signal enhancement were performed as intended.

The detection algorithm requires that all controls be valid before decisions regarding the presence or absence of any other target on the panel can be reported. If all controls are not within the predefined specification, a no call result will be obtained and further action should be taken according to the product labeling.

<table><tr><td rowspan=1 colspan=3>Table 5: Internal Processing Controls</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Function</td></tr><tr><td rowspan=1 colspan=1>Internal ProcessControl (INT CTL 1)</td><td rowspan=1 colspan=1>Artificial DNA constructwith detectionoligonucleotides.</td><td rowspan=1 colspan=1>Controls for target hybridization-related issues</td></tr><tr><td rowspan=1 colspan=1>Internal ProcessControl (INT CTL 2)</td><td rowspan=1 colspan=1>Intact MS2 Phage alongwith primers and detectionoligonucleotides. Addedto each test specimen.</td><td rowspan=1 colspan=1>Controls for lysis, extraction andtarget amplification.</td></tr></table>

# Sample stability

This study involved testing six representative bacterial organisms (see Table 6). For each organism five concentrations were prepared in a 3-fold serial dilution series using NSM. One freeze-thaw cycle was required and therefore only one cycle was evaluated. A sufficient starting volume of each simulated sample was prepared to make at least eight individual single-use vials at each dilution, each containing approximately $3 0 0 \mu \mathrm { L }$ of sample. Baseline testing of the freshly-prepared samples was performed in replicates of four; the four remaining vials were stored frozen at $< -$ $7 0 \ \mathrm { { ^ \circ C } }$ . The frozen samples were subsequently thawed and tested in replicates of four.

The starting concentration of each sample was chosen to bracket the Limit of Detection (LoD) determined for each of the analytes. The acceptance criteria was that the LoD determined for the frozen samples differed by no more than plus or minus $3 \mathbf { x }$ from the fresh LoD.

The test results in Table 6 demonstrate that the lowest concentrations where all replicates were detected were identical for two (Salmonella enterica and Yersinia enterocolitica) of the six organisms and within $3 \mathbf { x }$ of the fresh LoD for the remaining four organisms. The frozen LoD for Campylobacter jejuni, Escherichia coli and Vibrio parahaemolyticus were no more than $3 \mathbf { x }$ lower than the fresh LoD determination, and frozen Shigella dysenteriae was detected at 3x higher concentration than the corresponding fresh samples. All the LoD values were within one log of the strain LoD determined reported in the Detection Limit study.

A total of 256 initial tests were performed during the study. The initial call rate was $9 9 . 6 \%$ (255/256) with one No Call-Int Ctl 1, which was repeated successfully for a final call rate of $100 \%$ (256/256). The study supports the use of samples subjected to a single freeze/thaw cycle interchangeably with fresh/unfrozen samples in the analytical and clinical studies.

<table><tr><td rowspan=1 colspan=5>Table 6: Results from Fresh/Frozen Study</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>PreviouslyEstablishedLoD</td><td rowspan=1 colspan=2>Lowest Concentration where allReplicates wereDetected&quot; (CFU/mL)Fresh            Frozen</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>ATCC43429</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.23x104</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp.enterica (serovar Typhi)</td><td rowspan=1 colspan=1>ATCC9993</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae, Stx1</td><td rowspan=1 colspan=1>ATCC29026</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli, Stx2</td><td rowspan=1 colspan=1>ATCCBAA-176</td><td rowspan=1 colspan=1>1.11x10</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC23715</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC49398</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.23x104</td><td rowspan=1 colspan=1>3.70x104</td></tr></table>

# Detection Limit:

The Limit of Detection (LoD) study tested representative strains of bacteria detected by the Enteric Pathogens Nucleic Acid Test as shown in Table 7. In a range finding study, quantified preparations of bacterial strains were diluted in Negative Stool Matrix. Each concentration in the dilution series was then tested in replicates of four. The preliminary LoD concentration level for a specific organism was the lowest concentration level whereby all four replicates tested positive. The LoD for a specific organism was confirmed by testing 20 additional replicates at the preliminary LoD concentration to establish that the strain was detected in at least $9 5 \%$ of the measurements. If the detection rate was $100 \%$ (20/20), 20 replicates were tested at the next lower concentration.

Page - 17/45   

<table><tr><td rowspan=1 colspan=5>Table 7: List of Representative Bacterial Species/Strains used During the LoD Study</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species / Genogroup</td><td rowspan=1 colspan=1>Strain Designation</td><td rowspan=1 colspan=1>ATCCSource ID</td><td rowspan=1 colspan=1>ExpectedVerigeneDetected Call</td></tr><tr><td rowspan=3 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>jejuni subsp jejuni</td><td rowspan=1 colspan=1>MK5-S7630</td><td rowspan=1 colspan=1>43429</td><td rowspan=3 colspan=1>Campylobacter</td></tr><tr><td rowspan=1 colspan=1>coli</td><td rowspan=1 colspan=1>VanH13</td><td rowspan=1 colspan=1>43482</td></tr><tr><td rowspan=1 colspan=1>lari</td><td rowspan=1 colspan=1>NCTC 11457[A20/81, WRI921/79]</td><td rowspan=1 colspan=1>35222</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>enterica subsp enterica serovarTyphi</td><td rowspan=1 colspan=1>AMC 42-A-63</td><td rowspan=1 colspan=1>9993</td><td rowspan=2 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>enterica subsp arizonae</td><td rowspan=1 colspan=1>NCTC 8297[CIP 8230]</td><td rowspan=1 colspan=1>13314</td></tr><tr><td rowspan=4 colspan=1>Shigella</td><td rowspan=1 colspan=1>dysenteriae  Shiga Toxin 1(Stx1)</td><td rowspan=1 colspan=1>CDC 3044-74</td><td rowspan=1 colspan=1>29026</td><td rowspan=1 colspan=1>Shigella,Shiga Toxin 1</td></tr><tr><td rowspan=1 colspan=1>flexneri</td><td rowspan=1 colspan=1>NCDC 1235-66</td><td rowspan=1 colspan=1>25929</td><td rowspan=3 colspan=1>Shigella</td></tr><tr><td rowspan=1 colspan=1>sonnei</td><td rowspan=1 colspan=1>CDC 4447-74</td><td rowspan=1 colspan=1>29030</td></tr><tr><td rowspan=1 colspan=1>boydii</td><td rowspan=1 colspan=1>CDC C-2770-51</td><td rowspan=1 colspan=1>12035</td></tr><tr><td rowspan=2 colspan=1>Vibrio</td><td rowspan=1 colspan=1>cholerae</td><td rowspan=1 colspan=1>N16961</td><td rowspan=1 colspan=1>39315</td><td rowspan=2 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>parahaemolyticus</td><td rowspan=1 colspan=1>954</td><td rowspan=1 colspan=1>49398</td></tr><tr><td rowspan=2 colspan=1>Yersinia</td><td rowspan=2 colspan=1>enterocolitica</td><td rowspan=1 colspan=1>NCTC 11175</td><td rowspan=1 colspan=1>700822</td><td rowspan=2 colspan=1>Y. enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Billups-1803-68</td><td rowspan=1 colspan=1>23715</td></tr><tr><td rowspan=3 colspan=1>Escherichia</td><td rowspan=3 colspan=1>coli</td><td rowspan=1 colspan=1>CDC C984[CDC 3526-87]</td><td rowspan=1 colspan=1>43890</td><td rowspan=1 colspan=1>Shiga Toxin 1</td></tr><tr><td rowspan=1 colspan=1>CDC 2001-3004</td><td rowspan=1 colspan=1>BAA-176</td><td rowspan=1 colspan=1>Shiga Toxin 2</td></tr><tr><td rowspan=1 colspan=1>CDC EDL 933</td><td rowspan=1 colspan=1>43895</td><td rowspan=1 colspan=1>Shiga Toxin 1Shiga Toxin 2</td></tr></table>

The Detection Limit study demonstrated that the LoD for the EP test organisms ranged from $4 . 1 \mathrm { x } 1 0 ^ { 3 }$ to $3 . { \dot { 3 } } 3 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ stool. Individual organism results and LoD ranges are shown in Table 8 and Table 9.

<table><tr><td rowspan=1 colspan=2>Table 8: Established LoD per Organism Tested</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>LoD (CFU/mL Stool)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp jejuni</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp enterica serovar Typhi</td><td rowspan=1 colspan=1>3.33x10</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp arizonae</td><td rowspan=1 colspan=1>3.33x10</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Stx 1)</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica (ATCC 70082 )</td><td rowspan=1 colspan=1>3.33x105</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica (ATCC 23175)</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli  Shiga Toxin 1 (ATCC 43890)</td><td rowspan=1 colspan=1>4.10x10^3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli  Shiga Toxin 2 (ATCC BAA-176)</td><td rowspan=1 colspan=1>1.11x105</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli  Stx1 and Stx2 (ATCC 43895)</td><td rowspan=1 colspan=1>3.70x104</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Table 9: Established LoD Range per Claimed Analyte</td></tr><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=2>LoD (CFU/mL Stool)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter Group</td><td rowspan=1 colspan=1>3.70x10</td><td rowspan=1 colspan=1>3.70x104 - 1.11x105</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>3.33x10</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=1>3.70x10</td><td rowspan=1 colspan=1>3.70x104 - 1.11x105</td></tr><tr><td rowspan=1 colspan=1>Vibrio Group</td><td rowspan=1 colspan=1>3.70x10</td><td rowspan=1 colspan=1>3.70x104 - 1.11x105</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1.11x10</td><td rowspan=1 colspan=1>1.11x10 -3.33x105</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin 1 (stx1)</td><td rowspan=1 colspan=1>4.10x10</td><td rowspan=1 colspan=1>4.10x103 - 3.70x104</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin 2 (stx2)</td><td rowspan=1 colspan=1>3.70x10</td><td rowspan=1 colspan=1>3.70x104 - 1.11x105</td></tr></table>

# Analytical Reactivity/Specificity:

# Reactivity:

Alignment of GenBank nucleic acid sequences of the proposed gene targets was the first step toward generating and selecting the appropriate probes for the assay and to confirm inclusivity by in silico analysis. Table 10 lists the targets, specific genes, and the total number of GenBank nucleic acid sequence entries used to confirm inclusivity of each gene target. Acceptability of the target was based on the evaluation of the Minimum Percent Oligo Identity (e.g. the percent match between a proposed probe set and the GenBank consensus sequence for each target).

<table><tr><td rowspan=1 colspan=3>Table 10: Enteric Pathogens Nucleic Acid Test (EP) Gene Targets</td></tr><tr><td rowspan=1 colspan=1>Analyte / Target</td><td rowspan=1 colspan=1>Detected Gene</td><td rowspan=1 colspan=1>Total No. of GenBank SequenceEntries Used to Develop Targets</td></tr><tr><td rowspan=1 colspan=1>Campylobacter group</td><td rowspan=1 colspan=1>fusA</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>rpoD</td><td rowspan=1 colspan=1>256</td></tr><tr><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=1>ipaH</td><td rowspan=1 colspan=1>377</td></tr><tr><td rowspan=2 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>rfbL</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>trkH</td><td rowspan=1 colspan=1>352</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>tnaA</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>recN</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin 1</td><td rowspan=1 colspan=1>Stx1</td><td rowspan=1 colspan=1>264</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin 2 (variant Stx 2fexcluded from analysis)</td><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>362a</td></tr></table>

An analytical reactivity study for the Enteric Pathogens Nucleic Acid Test was designed to wet test reactivity towards clinically relevant microorganisms that represent temporal, geographical, and phylogenic diversity for each claimed target at concentrations at two to three times the LoD. Analytical reactivity of the Enteric Pathogens Nucleic Acid Test was demonstrated with a panel of 111 bacterial strains shown in summary in Table 11. To prepare the samples, bacterial stocks were

cultured, titered, and diluted in liquid media. This material was then added to Negative Stool Matrix and tested at levels no greater than 3-fold higher than the determined LoD for each analyte then tested in triplicate.

<table><tr><td rowspan=1 colspan=3>Table 11: Organisms Tested for Reactivity with the Enteric Pathogens Nucleic AcidTest (EP) Test</td></tr><tr><td rowspan=2 colspan=1>ReportableTarget</td><td rowspan=2 colspan=1>Total Number ofOrganisms/StrainsTested</td><td rowspan=1 colspan=1>Species Tested</td></tr><tr><td rowspan=1 colspan=1>Name(No. of Strains)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>C. coli (5), C. jejuni subsp jejuni (4), C. jejunisubsp doylei (1), C. lari (5)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>S. bongori (1), S. enterica subsp various (5),S. enterica subsp enterica serovar various (25)</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>S. boydii (5), S. dysenteriae (5)a, S. flexneri (5),S. sonnei (5)</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>V. cholerae (5), V. parahaemolyticus (5)</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Y. enterocolitica (7)</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin 1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>S. dysenteriae (2)b, E. coli (17)c</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin 2</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>E. coli (16) c</td></tr></table>

a Seven strains also contain Stx1 and/or Stx2, therefore, some strains have been counted twice. b Two (2) strains contain Stx1. c Five (5) strains contain both Stx1 and Stx2.

All tested strains generated the expected calls when tested in triplicate at a concentration of three times the LoD. These test results establish the reactivity claims of the Enteric Pathogens Nucleic Acid Test test.

Shiga toxin 2 variants Stx2c, Stx2d, Stx2e, and $\operatorname { S t x } 2 \mathrm { g }$ were tested for reactivity based on in silico analysis only. These four variants were aligned with three unique GenBank sequences per variant. Overall, only one mismatch was found in the alignment of Stx2d. The remaining alignment results showed no mismatches with the selected GenBank sequences. This analysis is acceptable to establish reactivity toward the Stx2c, Stx2d, Stx2e and $\operatorname { S t x } 2 \mathrm { g }$ variants. Shiga toxin 2 variant Stx2f was determined via in silico analysis to be non-reactive with the test based on the high degree of mismatches (maximum of $71 \%$ identity) between the selected GenBank gene targets and test probes/primers.

# Specificity:

A specificity study was designed to assess potential unintended reactivity of the Enteric Pathogens Nucleic Acid Test with clinically relevant levels of enteric pathogens and microorganisms that may be present in stool specimens but not detectable by the test. A panel of 161 organisms listed in Table 12 through Table 14 was selected to establish test specificity. The panels were comprised of 135 bacterial organisms, 21 viruses, four parasites and one human cell line.

Samples were tested with the Enteric Pathogens Nucleic Acid Test in triplicate and prepared as follows.

# Bacteria

The bacterial organisms were added to Negative Stool Matrix at a concentration of at least $1 0 ^ { 7 } \mathrm { C F U / m L }$ . If an organism could not be accurately titered or was unavailable, genomic DNA at an estimated genome equivalent copy number to $1 0 ^ { 7 } \mathrm { C F U / m L }$ was added directly to the Enteric Pathogens Nucleic Acid Test Amplification Tray, bypassing the DNA Extraction step of the Enteric Pathogens Nucleic Acid Test procedure. The organisms tested in this study were obtained from commercial sources and the identities of the organisms confirmed using the following procedures:

· Aerobic organisms were identified using the Gram Negative (GN) and Gram Positive (GP) identification cards of the BioMérieux Vitek 2 system whenever possible.   
· Anaerobic organisms identified using the Remel RapID ANA II system.   
· Microaerophilic organisms and Bacillus cereus (aerobic) were identified using standard microbiological methods involving growth on selective culture media, biochemical spot tests, and Gram stains.

Seven organisms, including Astrovirus and Sapovirus (two strains), and all four parasites (Blastocystis hominis, Cryptosporidium parvum, Entamoeba histolytica, and Giardia lamblia) were only commercially available as genomic DNA or RNA. Additionally, Campylobacter hominis was unable to be successfully grown and therefore genomic DNA/RNA was tested. For each of these, genomic DNA/RNA was tested directly utilizing the amplification-hybridization steps of the test. To prepare the samples, negative stool matrix was processed through the extraction step using the Verigene Processor $S P$ to generate a stool-containing eluent, to which DNA/RNA was added (equivalent to $\bar { \geq } 1 0 ^ { 7 }$ copies/mL in stool). This mixture was then tested using the amplification hybridization steps of the test.

# Viruses

The viral species listed in Table 13 were added to Negative Stool Matrix to final concentrations of ${ > } 1 0 ^ { 6 } \mathrm { P F U / m L }$ stool. For the viruses that could not be accurately titered or were unavailable (i.e., Astrovirus and Sapovirus), genomic RNA at a genome equivalent copy number of $1 0 ^ { 7 }$ copies/ml was added directly to the Enteric Pathogens Nucleic Acid Test (EP) Amplification Tray such that amplification and hybridization could be performed without the DNA extraction step.

# Parasites

As described above, the four parasites listed in Table 13 were tested using commercially purchased genomic DNA. In these test cases, the equivalent of ${ \sim } 1 0 ^ { 6 }$ copies of genomic DNA was added directly to the Enteric Pathogens Nucleic Acid Test Amplification Tray such that amplification and hybridization could be performed without the DNA extraction step.

Human Cell Line

The human cell line was obtained from commercial sources and was evaluated by adding an aliquot to the Negative Stool Matrix at a concentration of ${ \sim } 1 0 ^ { 5 }$ cells/mL stool.

<table><tr><td rowspan=1 colspan=6>Table 12 Exclusivity Test Organisms  Bacterial Non-Test Panel Members (tested at 1x107 CFU/mL unlessotherwise noted)</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC SourceID</td><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC SourceID</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia</td><td rowspan=1 colspan=1>defectiva</td><td rowspan=1 colspan=1>49176</td><td rowspan=2 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>faecalis</td><td rowspan=1 colspan=1>51299</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter</td><td rowspan=1 colspan=1>baumannii</td><td rowspan=1 colspan=1>19606</td><td rowspan=1 colspan=1>faecium</td><td rowspan=1 colspan=1>700221</td></tr><tr><td rowspan=1 colspan=1>lwoffi</td><td rowspan=1 colspan=1>17925</td><td rowspan=10 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli</td><td rowspan=1 colspan=1>23511</td></tr><tr><td rowspan=2 colspan=1>Acrobacter</td><td rowspan=1 colspan=1>butzleri</td><td rowspan=1 colspan=1>49942</td><td rowspan=2 colspan=1>coli(non-pathogenic)</td><td rowspan=1 colspan=1>25922</td></tr><tr><td rowspan=1 colspan=1>cryaerophilus</td><td rowspan=1 colspan=1>43157</td><td rowspan=1 colspan=1>35218</td></tr><tr><td rowspan=13 colspan=1>Aeromonas</td><td rowspan=1 colspan=1>allosaccharophila</td><td rowspan=1 colspan=1>35942</td><td rowspan=1 colspan=1>coli (EAEC)</td><td rowspan=1 colspan=1>23501</td></tr><tr><td rowspan=1 colspan=1>bestiarum</td><td rowspan=1 colspan=1>BAA-231</td><td rowspan=1 colspan=1>coli (EPEC)</td><td rowspan=1 colspan=1>BAA-1704</td></tr><tr><td rowspan=1 colspan=1>caviae</td><td rowspan=1 colspan=1>13136</td><td rowspan=1 colspan=1>coli (EPEC)</td><td rowspan=1 colspan=1>12014</td></tr><tr><td rowspan=1 colspan=1>encheleia</td><td rowspan=1 colspan=1>51929</td><td rowspan=1 colspan=1>coli (ETEC)</td><td rowspan=1 colspan=1>43886</td></tr><tr><td rowspan=1 colspan=1>enteropelogenes</td><td rowspan=1 colspan=1>49803</td><td rowspan=1 colspan=1>coli (ETEC)</td><td rowspan=1 colspan=1>23519</td></tr><tr><td rowspan=1 colspan=1>eucrenophila</td><td rowspan=1 colspan=1>23309</td><td rowspan=1 colspan=1>fergusonii</td><td rowspan=1 colspan=1>35469</td></tr><tr><td rowspan=1 colspan=1>hydrophilia</td><td rowspan=1 colspan=1>7966</td><td rowspan=1 colspan=1>hermannii</td><td rowspan=1 colspan=1>33650</td></tr><tr><td rowspan=1 colspan=1>jandaei</td><td rowspan=1 colspan=1>49568</td><td rowspan=1 colspan=1>Fusobacterium</td><td rowspan=1 colspan=1>varium</td><td rowspan=1 colspan=1>8501</td></tr><tr><td rowspan=2 colspan=1>salmonicidasubsp masoucida</td><td rowspan=2 colspan=1>27013</td><td rowspan=5 colspan=1>Helicobacter</td><td rowspan=1 colspan=1>hepaticus*</td><td rowspan=1 colspan=1>51449</td></tr><tr><td rowspan=1 colspan=1>pylori</td><td rowspan=1 colspan=1>43504</td></tr><tr><td rowspan=2 colspan=1>salmonicidasubspsalmonicida</td><td rowspan=1 colspan=1>14174</td><td rowspan=1 colspan=1>pylori</td><td rowspan=1 colspan=1>700392</td></tr><tr><td rowspan=1 colspan=1>33658</td><td rowspan=1 colspan=1>pylori</td><td rowspan=1 colspan=1>49503</td></tr><tr><td rowspan=1 colspan=1>veronii</td><td rowspan=1 colspan=1>9071</td><td rowspan=1 colspan=1>pylori</td><td rowspan=1 colspan=1>51652</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes</td><td rowspan=1 colspan=1>faecalis</td><td rowspan=1 colspan=1>15554</td><td rowspan=2 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>oxytoca</td><td rowspan=1 colspan=1>43165</td></tr><tr><td rowspan=1 colspan=1>Bacillus</td><td rowspan=1 colspan=1>cereus</td><td rowspan=1 colspan=1>10702</td><td rowspan=1 colspan=1>pneumoniae</td><td rowspan=1 colspan=1>13883</td></tr><tr><td rowspan=4 colspan=1>Bacteroides</td><td rowspan=1 colspan=1>caccae</td><td rowspan=1 colspan=1>43185</td><td rowspan=3 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>acidophilus</td><td rowspan=1 colspan=1>11975</td></tr><tr><td rowspan=1 colspan=1>fragilis</td><td rowspan=1 colspan=1>25285</td><td rowspan=1 colspan=1>reuteri</td><td rowspan=1 colspan=1>23272</td></tr><tr><td rowspan=1 colspan=1>merdae</td><td rowspan=1 colspan=1>43184</td><td rowspan=1 colspan=1>rhamnosus</td><td rowspan=1 colspan=1>53103</td></tr><tr><td rowspan=1 colspan=1>stercoris</td><td rowspan=1 colspan=1>43183</td><td rowspan=1 colspan=1>Lactococcus</td><td rowspan=1 colspan=1>lactis</td><td rowspan=1 colspan=1>11454</td></tr><tr><td rowspan=1 colspan=1>Candida</td><td rowspan=1 colspan=1>albicans</td><td rowspan=1 colspan=1>10231</td><td rowspan=1 colspan=1>Leminorela</td><td rowspan=1 colspan=1>grimontii</td><td rowspan=1 colspan=1>33999</td></tr><tr><td rowspan=1 colspan=1>Cedecea</td><td rowspan=1 colspan=1>davisae</td><td rowspan=1 colspan=1>33431</td><td rowspan=2 colspan=1>Listeria</td><td rowspan=1 colspan=1>grayi</td><td rowspan=1 colspan=1>19120</td></tr><tr><td rowspan=3 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>amalonaticus</td><td rowspan=1 colspan=1>25407</td><td rowspan=1 colspan=1>monocytogenes</td><td rowspan=1 colspan=1>7644</td></tr><tr><td rowspan=1 colspan=1>freundii</td><td rowspan=1 colspan=1>8090</td><td rowspan=1 colspan=1>Morganella</td><td rowspan=1 colspan=1>morganii</td><td rowspan=1 colspan=1>25830</td></tr><tr><td rowspan=1 colspan=1>sedlakii</td><td rowspan=1 colspan=1>51115</td><td rowspan=1 colspan=1>Peptostreptococcus</td><td rowspan=1 colspan=1>anaerobius</td><td rowspan=1 colspan=1>27337</td></tr><tr><td rowspan=14 colspan=1>Clostridium</td><td rowspan=1 colspan=1>bifermentans</td><td rowspan=1 colspan=1>638</td><td rowspan=1 colspan=1>Plesiomonas</td><td rowspan=1 colspan=1>shigelloides</td><td rowspan=1 colspan=1>14029</td></tr><tr><td rowspan=1 colspan=1>bolteae</td><td rowspan=1 colspan=1>BAA-613</td><td rowspan=1 colspan=1>Porphyromonas</td><td rowspan=1 colspan=1>asaccharoluticus</td><td rowspan=1 colspan=1>25260</td></tr><tr><td rowspan=1 colspan=1>butyricum</td><td rowspan=1 colspan=1>19398</td><td rowspan=1 colspan=1>Prevotella</td><td rowspan=1 colspan=1>melaniogenica</td><td rowspan=1 colspan=1>25845</td></tr><tr><td rowspan=1 colspan=1>difficile</td><td rowspan=1 colspan=1>BAA-1805</td><td rowspan=3 colspan=1>Proteus</td><td rowspan=1 colspan=1>mirabilis</td><td rowspan=1 colspan=1>25933</td></tr><tr><td rowspan=1 colspan=1>difficile</td><td rowspan=1 colspan=1>17857</td><td rowspan=1 colspan=1>vulgaris</td><td rowspan=1 colspan=1>29905</td></tr><tr><td rowspan=1 colspan=1>difficile, non-tox</td><td rowspan=1 colspan=1>BAA-1801</td><td rowspan=1 colspan=1>penneri</td><td rowspan=1 colspan=1>35198</td></tr><tr><td rowspan=1 colspan=1>haemolyticum</td><td rowspan=1 colspan=1>9650</td><td rowspan=3 colspan=1>Providencia</td><td rowspan=1 colspan=1>stuartii</td><td rowspan=1 colspan=1>33672</td></tr><tr><td rowspan=1 colspan=1>methylpentosum</td><td rowspan=1 colspan=1>43829</td><td rowspan=1 colspan=1>alcalifaciens</td><td rowspan=1 colspan=1>9886</td></tr><tr><td rowspan=1 colspan=1>nexile</td><td rowspan=1 colspan=1>27757</td><td rowspan=1 colspan=1>rettgeri</td><td rowspan=1 colspan=1>9250</td></tr><tr><td rowspan=1 colspan=1>noyvi</td><td rowspan=1 colspan=1>19402</td><td rowspan=4 colspan=1>Pseudomonas</td><td rowspan=1 colspan=1>aeruginosa</td><td rowspan=1 colspan=1>35554</td></tr><tr><td rowspan=1 colspan=1>orbiscindens</td><td rowspan=1 colspan=1>49531</td><td rowspan=1 colspan=1>fluroescenes</td><td rowspan=1 colspan=1>13525</td></tr><tr><td rowspan=1 colspan=1>perfringens</td><td rowspan=1 colspan=1>13124</td><td rowspan=1 colspan=1>putida</td><td rowspan=1 colspan=1>12633</td></tr><tr><td rowspan=1 colspan=1>scindens</td><td rowspan=1 colspan=1>35704</td><td rowspan=1 colspan=1>aeruginosa</td><td rowspan=1 colspan=1>27853</td></tr><tr><td rowspan=1 colspan=1>septicum</td><td rowspan=1 colspan=1>12464</td><td rowspan=1 colspan=1>Ruminococcus</td><td rowspan=1 colspan=1>bromii</td><td rowspan=1 colspan=1>27255</td></tr></table>

Page - 22/45   

<table><tr><td rowspan=1 colspan=6>Table 12 Exclusivity Test Organisms  Bacterial Non-Test Panel Members (tested at 1x107 CFU/mL unlessotherwise noted)</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC SourceID</td><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC SourceID</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>sordellii</td><td rowspan=1 colspan=1>9714</td><td rowspan=2 colspan=1>Serratia</td><td rowspan=1 colspan=1>liquefacians</td><td rowspan=1 colspan=1>27592</td></tr><tr><td rowspan=1 colspan=1>spiroforme</td><td rowspan=1 colspan=1>29899</td><td rowspan=1 colspan=1>marcescens</td><td rowspan=1 colspan=1>13880</td></tr><tr><td rowspan=1 colspan=1>sporogenes</td><td rowspan=1 colspan=1>15579</td><td rowspan=2 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>aureus</td><td rowspan=1 colspan=1>43300</td></tr><tr><td rowspan=1 colspan=1>Colinsella</td><td rowspan=1 colspan=1>aerofaciens</td><td rowspan=1 colspan=1>25986</td><td rowspan=1 colspan=1>epidermidis</td><td rowspan=1 colspan=1>700583</td></tr><tr><td rowspan=1 colspan=1>Desulfovibrio</td><td rowspan=1 colspan=1>piger</td><td rowspan=1 colspan=1>29098</td><td rowspan=3 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>agalactiae,O90R</td><td rowspan=1 colspan=1>12386</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella</td><td rowspan=1 colspan=1>tarda</td><td rowspan=1 colspan=1>15947</td><td rowspan=1 colspan=1>dysgalactiae</td><td rowspan=1 colspan=1>12394</td></tr><tr><td rowspan=2 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>aerogenes</td><td rowspan=1 colspan=1>13048</td><td rowspan=1 colspan=1>mutans</td><td rowspan=1 colspan=1>25175</td></tr><tr><td rowspan=1 colspan=1>cloacae</td><td rowspan=1 colspan=1>29006</td><td></td><td></td><td></td></tr></table>

<table><tr><td rowspan=1 colspan=4>Table 13: Exclusivity Test Organisms  Viral, Parasites and Human DNA.</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain #/UniqueIdentifier/Source</td><td rowspan=1 colspan=1>Serovars/ Groups</td><td rowspan=1 colspan=1>Titer Tested</td></tr><tr><td rowspan=11 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>0810050CF</td><td rowspan=1 colspan=1>Type 1/Group C</td><td rowspan=9 colspan=1>1.0x106 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>0810110CF</td><td rowspan=1 colspan=1>Type 2/Group C</td></tr><tr><td rowspan=1 colspan=1>0810062CF</td><td rowspan=1 colspan=1>Type 3/Group B1</td></tr><tr><td rowspan=1 colspan=1>0810070CF</td><td rowspan=1 colspan=1>Type 4/Group E</td></tr><tr><td rowspan=1 colspan=1>0810020CF</td><td rowspan=1 colspan=1>Type 5/Group C</td></tr><tr><td rowspan=1 colspan=1>0810108CF</td><td rowspan=1 colspan=1>Type 14/Group B2</td></tr><tr><td rowspan=1 colspan=1>0810117CF</td><td rowspan=1 colspan=1>Type 26/Group D</td></tr><tr><td rowspan=1 colspan=1>0810073CF</td><td rowspan=1 colspan=1>Type 31/Group A</td></tr><tr><td rowspan=1 colspan=1>0810119CF</td><td rowspan=1 colspan=1>Type 37/Group D</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-931</td><td rowspan=1 colspan=1>Type 40/Group F</td><td rowspan=1 colspan=1>1.58x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>VR-1572D</td><td rowspan=1 colspan=1>Human 4</td><td rowspan=1 colspan=1>1.0x106 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>P#711/24/08</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>ATCC VR-184</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3.16x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>0810003-CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>7.24x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>0810023-CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0x 106 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>VR-213</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0x106 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Norovirus</td><td rowspan=1 colspan=1>2012792142</td><td rowspan=1 colspan=1>Genogroup GI</td><td rowspan=1 colspan=1>1.0x108 viralparticles/mL</td></tr><tr><td rowspan=1 colspan=1>2012792012</td><td rowspan=1 colspan=1>Genogroup GII</td><td rowspan=1 colspan=1>3.0x106 viralparticles/mL</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>VR-2551</td><td rowspan=1 colspan=1>Genogroup A</td><td rowspan=1 colspan=1>1.58x106TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>2008729730</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>1x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>2009726567</td></tr><tr><td rowspan=1 colspan=4>Human Cell Line</td></tr><tr><td rowspan=1 colspan=1>Colon epithelial cells</td><td rowspan=1 colspan=1>ATCC CCL-218</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0x105cells/mL</td></tr></table>

Page - 23/45   

<table><tr><td rowspan=1 colspan=4>Table 13: Exclusivity Test Organisms  Viral, Parasites and Human DNA.</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain #/UniqueIdentifier/Source</td><td rowspan=1 colspan=1>Serovars/ Groups</td><td rowspan=1 colspan=1>Titer Tested</td></tr><tr><td rowspan=1 colspan=1>(colorectaladenocarcinoma)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Parasites</td></tr><tr><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>ATCC 50608-D</td><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1>1x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidiumparvum</td><td rowspan=1 colspan=1>PRA-67-D</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30459-D</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>ATCC 50803-D</td></tr></table>

The 35 bacterial organisms/strains listed in Table 14 are organisms which are not intended to be detected by the Enteric Pathogens Nucleic Acid Test panel for the group and species level bacterial targets Campylobacter, Vibrio, and Yersinia enterocolitica, respectively. Testing of these targets was performed in the same conditions as the organisms found in Table 12.

<table><tr><td rowspan=1 colspan=4>Table 14: Exclusivity Test Organisms  Non-detected Strains of Test Panel Members</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Designation</td><td rowspan=1 colspan=1>Titer tested(CFU/mL)</td></tr><tr><td rowspan=13 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>concisus</td><td rowspan=1 colspan=1>ATCC BAA-1457</td><td rowspan=1 colspan=1>5.5x106</td></tr><tr><td rowspan=1 colspan=1>curvus</td><td rowspan=1 colspan=1>ATCC BAA-1459</td><td rowspan=1 colspan=1>6.1x106</td></tr><tr><td rowspan=1 colspan=1>fetus</td><td rowspan=1 colspan=1>ATCC 25936</td><td rowspan=1 colspan=1>3.9x107</td></tr><tr><td rowspan=1 colspan=1>gracilis</td><td rowspan=1 colspan=1>ATCC 33236</td><td rowspan=1 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>hominis (gDNA)</td><td rowspan=1 colspan=1>ATCC BAA-381D-5</td><td rowspan=1 colspan=1>1.0x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>hyointestinalis</td><td rowspan=1 colspan=1>ATCC 35217</td><td rowspan=7 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>insulaenigrae</td><td rowspan=1 colspan=1>CCUG48653</td></tr><tr><td rowspan=1 colspan=1>lanienae</td><td rowspan=1 colspan=1>CCUG44467</td></tr><tr><td rowspan=1 colspan=1>mucosalis</td><td rowspan=1 colspan=1>ATCC49352</td></tr><tr><td rowspan=1 colspan=1>rectus</td><td rowspan=1 colspan=1>ATCC 33238</td></tr><tr><td rowspan=1 colspan=1>showae</td><td rowspan=1 colspan=1>ATCC 51146</td></tr><tr><td rowspan=1 colspan=1>sputorum</td><td rowspan=1 colspan=1>ATCC 35980</td></tr><tr><td rowspan=1 colspan=1>upsaliensis</td><td rowspan=1 colspan=1>ATCC BAA-1059</td><td rowspan=1 colspan=1>7.5x106</td></tr><tr><td rowspan=12 colspan=1>Vibrio</td><td rowspan=1 colspan=1>alginolyticus</td><td rowspan=1 colspan=1>ATCC 17749</td><td rowspan=14 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>campbellii*</td><td rowspan=1 colspan=1>ATCC 25920</td></tr><tr><td rowspan=1 colspan=1>cincinnatiensis</td><td rowspan=1 colspan=1>ATCC 35912</td></tr><tr><td rowspan=1 colspan=1>fluvialis</td><td rowspan=1 colspan=1>ATCC 33809</td></tr><tr><td rowspan=1 colspan=1>furnissii</td><td rowspan=1 colspan=1>ATCC11218</td></tr><tr><td rowspan=1 colspan=1>harveyi</td><td rowspan=1 colspan=1>ATCC14126</td></tr><tr><td rowspan=1 colspan=1>metschnikovii</td><td rowspan=1 colspan=1>ATCC 7708</td></tr><tr><td rowspan=1 colspan=1>mimicus</td><td rowspan=1 colspan=1>ATCC 33653</td></tr><tr><td rowspan=1 colspan=1>tubiashii*</td><td rowspan=1 colspan=1>ATCC 18106</td></tr><tr><td rowspan=3 colspan=1>vulnificus</td><td rowspan=1 colspan=1>ATCC BAA-86</td></tr><tr><td rowspan=1 colspan=1>ATCC 27562</td></tr><tr><td rowspan=1 colspan=1>ATCC 33815</td></tr><tr><td rowspan=2 colspan=1>Yersinia</td><td rowspan=1 colspan=1>aldovae</td><td rowspan=1 colspan=1>ATCC 35236</td></tr><tr><td rowspan=1 colspan=1>aleksiciae</td><td rowspan=1 colspan=1>CCUG 52872</td></tr></table>

Page - 24/45   

<table><tr><td rowspan=1 colspan=4>Table 14: Exclusivity Test Organisms  Non-detected Strains of Test Panel Members</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Designation</td><td rowspan=1 colspan=1>Titer tested(CFU/mL)</td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=1 colspan=1>bercovieri</td><td rowspan=1 colspan=1>ATCC 43970</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>frederiksenii</td><td rowspan=1 colspan=1>ATCC 33644</td></tr><tr><td rowspan=1 colspan=1>intermedia</td><td rowspan=1 colspan=1>ATCC 33647</td></tr><tr><td rowspan=1 colspan=1>kristensenii</td><td rowspan=1 colspan=1>ATCC 33639</td></tr><tr><td rowspan=1 colspan=1>mollaretii</td><td rowspan=1 colspan=1>ATCC43969</td></tr><tr><td rowspan=1 colspan=1>pseudotuberculosis</td><td rowspan=1 colspan=1>ATCC 29910</td></tr><tr><td rowspan=1 colspan=1>ruckeri</td><td rowspan=1 colspan=1>ATCC 29473</td></tr><tr><td rowspan=1 colspan=1>rohdei</td><td rowspan=1 colspan=1>ATCC 43380</td></tr></table>

\* Species not associated with human infections.

There are forty-four species of Vibrio that are not associated with infections in humans. Two Vibrio strains were wet tested and the 15 species were analyzed in silico to establish exclusivity. The remaining 27 species were not tested due to the low prevalence of these species in human stool and a lack of sequence information in public databases. The test package insert states this with the following product limitations:

The following 15 species of Vibrio, each of which are NOT associated with infections in humans and therefore unlikely to be encountered in human stool, were shown NOT to be detected by EP based upon in silico analysis only: V. anguillarum, V. brasiliensis, V. coralliilyticus, V. crassostreae, V. cyclitrophicus, V. ichthyoenteri, V. kanaloae, V. nigripulchritudo, V. ordalii, V. orientalis, V. rotiferianus, V. rumoiensis, V. scophthalmi, V. splendidus, and V. tasmaniensis.

The following 27 species of Vibrio, each of which are NOT associated with infections in humans and therefore unlikely to be encountered in human stool, were not evaluated for exclusivity by wet testing or in silico analysis due to a lack of genome sequence information: V. aerogenes, V. aestuarianus, V. chagasii, V. diabolicus, V. diazotrophicus, V. ezurae, V. fortis, V. gallicus, V. gazogenes, V. gigantis, V. halioticoli, V. hepatarius, V. hispanicus, V. litoralis, V. mediterranei, V. mytili, V. natriegens, V. navarrensis, V. neonatus, V. nereis, V. pacinii, V. pectenicida, V. pomeroyi, V. proteolyticus, V. ruber, V. superstes, and V. xuii

One hundred and sixty samples of the 161 tested yielded negative results for all three replicates performed. One enteric organism, Campylobacter insulaenigrae, generated one “Campylobacter Detected” result of the three replicates tested. Repeat testing of six replicates did not yield a positive result, therefore, the initial results suggesting cross reactively were not reproducible. To further investigate the anomalous result, bi-directional sequencing of the fusA gene region of Campylobacter insulaenigrae, which encompasses the region detected by the EP test, was performed. The percent homologies between the Campylobacter detection oligos and the Campylobacter insulaenigrae sequences indicate the potential for the test to amplify and detect

Campylobacter insulaenigrae. As this is a novel species that has limited references attributing this organism to disease in humans, a limitation will be placed in the product labeling. Therefore, the package insert contains the following statement regarding the potential for cross-reactivity with this species:

In rare instances, Campylobacter insulaenigrae may yield a false positive “Campylobacter detected” result.

In summary, this study is acceptable to establish the specificity of the test to 160 enteric organisms under the stated test conditions

# Assay cut-off:

The presence or absence of each target analyte is determined by the mean intensity of target capture spots relative to the Signal Detection Threshold. The capture, mediator, and PCR primer oligonucleotides in the Enteric Pathogens Nucleic Acid Test are designed to eliminate sequence-related cross-reactivity, resulting in non-specific target signal intensities at capture spots that are similar to the microarray background signal. In contrast, target amplicon hybridization to complementary capture and mediator probes are expected to give signals that are well-separated from negative capture spots. When reading a test slide, multiple images of each array are taken at increasing exposures times and the final target group mean intensity value for an analyte is assigned at the shortest exposure at which the intensity exceeds the Signal Detection Threshold. If none of the target signal exceeds the threshold for any exposure, the mean spot intensity is evaluated at the longest exposure taken. With this imaging and analysis design, a signal detection threshold of 30,000 was established to generate a “Detected” call for the eight bacterial target spot groups and two controls of Enteric Pathogens Nucleic Acid Test test.

In order to demonstrate the appropriateness of the cut-off value for this threshold, the target mean intensity values observed with the Enteric Pathogens Nucleic Acid Test were examined for the final confirmatory tests of the sixteen simulated bacterial samples tested as part of the Limit of Detection Study. In addition, the cut-off data set included the test results of three viral samples which represent negative control samples. In this case, to better represent negative samples, the target intensity values at the maximum exposure (1280 msec) were analyzed, thereby negating any potential impact of the exposure cap limit upon detection of a masked viral target. With replicates of 20 for each sample and ten target spot groups evaluated per test, a total of 3800 data points (1120 expected positive) were assessed in the study.

A logistic fit analysis of Expected Results by Target Mean Intensity for the chosen threshold shows that expected positive signals are well separated from the expected negative target signals therefore the chosen threshold value distinguishes the “True Positives” from the “True Negatives”. Acceptability of the chosen cut-off is established by the LoD study data and the clinical validation data.

Interfering species

Microbial interference

Analytical testing was performed to establish that nonpathogenic microorganisms present in stool do not interfere with the ability of the Enteric Pathogens Nucleic Acid Test to detect enteric pathogens. Two representative bacterial organisms detected by the Enteric Pathogens Nucleic Acid Test shown in Table 15 are Campylobacter jejuni and Escherichia coli (Shiga toxin 1). These test panel members were evaluated for potential interference in the presence of 14 normal flora microorganisms listed in Table 16. These microorganisms represent highly prevalent bacterium known to be present in the human colon as determined through literature review. This study was conducted using simulated specimens in NSM prepared as previously described. Each of representative panel organisms were added to NSM at a concentration of three times the LoD determined for the organism (see Table 15). The normal flora bacteria were spiked individually into each prepared specimen at a concentration of $1 0 ^ { 7 }$ $\mathrm { C F U / m L }$ with the exception of the parasites Blastocystis hominis and Entamoeba histolytica which, due to titer constraints, were spiked into the NSM at concentrations of $9 \mathrm { { x i } 0 ^ { 6 } \mathrm { { c e l l s } / \mathrm { { m L } } } }$ and $7 \mathrm { { x } 1 0 ^ { 5 } \mathrm { { c e l l s } / \mathrm { { m L } } } }$ respectively. The resulting specimens were tested in triplicate.

<table><tr><td rowspan=1 colspan=5>Table 15: Representative Test Organisms</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain Number</td><td rowspan=1 colspan=1>Expected Verigene Result</td><td rowspan=1 colspan=1>EP Test LoD(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>jejuni subsp jejuni</td><td rowspan=1 colspan=1>ATCC 43429</td><td rowspan=1 colspan=1>Campylobacter Detected</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli (stx1)</td><td rowspan=1 colspan=1>ATCC 43890</td><td rowspan=1 colspan=1>Shiga Toxin 2 Detected</td><td rowspan=1 colspan=1>4.1 x103</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Table 16: Organisms tested</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>Bacteroides</td><td rowspan=1 colspan=1>fragilis</td></tr><tr><td rowspan=1 colspan=1>Prevotella</td><td rowspan=1 colspan=1>oralis</td></tr><tr><td rowspan=1 colspan=1>Prevotella</td><td rowspan=1 colspan=1>melaninogenicus</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium</td><td rowspan=1 colspan=1>bifidum</td></tr><tr><td rowspan=1 colspan=1>Clostridium</td><td rowspan=1 colspan=1>perfringens</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>aerogenes</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>faecalis</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>pneumonia</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>acidophilus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>aureus</td></tr><tr><td rowspan=1 colspan=1>Blastocystis</td><td rowspan=1 colspan=1>hominis</td></tr><tr><td rowspan=1 colspan=1>Entamoeba</td><td rowspan=1 colspan=1>histolytica</td></tr><tr><td rowspan=1 colspan=1>Candida</td><td rowspan=1 colspan=1>albicans</td></tr></table>

The Enteric Pathogens Nucleic Acid Test results for all 28 samples, tested in triplicate, correctly identified the expected analytes for the two representative enteric pathogens evaluated. These results establish that, under the conditions of this study, microorganisms that may be present in stool samples do not interfere with the ability of the Enteric Pathogens Nucleic Acid Test to detect enteric pathogens in a sample.

# Interfering substances

An interfering substances study was performed to assess the potential inhibitory effect of endogenous and exogenous substances that can commonly be found in clinical stool specimens. Two organisms (see Table 17) representative of the target analytes detected by the test (i.e.; one Gram-negative bacteria and one Shiga-toxin releasing bacteria), were individually challenged with 22 potentially interfering substances at high, medically relevant concentrations. The organisms were tested at 3x the LoD determined for each organism.

The panel of interferents tested is provided in Table 18. Each sample was prepared by adding the appropriate mass or volume of the interfering substance to a volume of Negative Stool Matrix to achieve the desired interferent/stool ratio. An individual culture of each of two representative organisms was then added to the NSM/interferent mixture to achieve a concentration of $3 \mathbf { x }$ LoD for the organism. Each sample was tested in replicates of three. NSM/interferent samples without bacterial organisms were also tested as controls.

<table><tr><td rowspan=1 colspan=5>Table 17: Representative Test Organisms for the Interfering Substances Study</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain Number</td><td rowspan=1 colspan=1>Expected VerigeneResult</td><td rowspan=1 colspan=1>EP Test LoD(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>jejuni subsp jejuni</td><td rowspan=1 colspan=1>ATCC 43429</td><td rowspan=1 colspan=1>Campylobacter Detected</td><td rowspan=1 colspan=1>3.70x104</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli (Stx1)</td><td rowspan=1 colspan=1>ATCC 43890</td><td rowspan=1 colspan=1>Stx1 Detected</td><td rowspan=1 colspan=1>4.1 x103</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Table 18: Potentially Interfering Substances Tested</td></tr><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>InterferentConcentration instool</td></tr><tr><td rowspan=1 colspan=1>Intralipid</td><td rowspan=1 colspan=1>Triglyceride (Fecal Fat)</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>Cholesterol (Fecal Fat)</td><td rowspan=1 colspan=1>5% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>Glucose, Hormones, Enzymes,Ions, Iron etc.</td><td rowspan=1 colspan=1>40% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucus (Nasopharyngeal swab sample inUTM)</td><td rowspan=1 colspan=1>Immunoglobulins, Lysozyme,Polymers</td><td rowspan=1 colspan=1>40% w/v</td></tr><tr><td rowspan=1 colspan=1>Nystatin Suspension</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Preparation H® Anti-itch Hydrocortisone 1%</td><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Desitin Maximum Strength Original Paste</td><td rowspan=1 colspan=1>Zinc Oxide</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Preparation H® Hemorrhoidal Ointment</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>30% w/v</td></tr></table>

Page - 28/45   

<table><tr><td rowspan=1 colspan=3>Table 18: Potentially Interfering Substances Tested</td></tr><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>InterferentConcentration instool</td></tr><tr><td rowspan=1 colspan=1>Options Conceptrol®Vaginal ContraceptiveGel</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Wet Ones®Antibacterial Hand Wipes</td><td rowspan=1 colspan=1>Benzalkonium Chloride, Ethanol</td><td rowspan=1 colspan=1>30% v/v</td></tr><tr><td rowspan=1 colspan=1>K-Y®Personal Lubricant Jelly</td><td rowspan=1 colspan=1>Glycerin</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Vaseline Original 100% Pure Petroleum Jelly</td><td rowspan=1 colspan=1>Petroleum</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Tums Antacid with Calcium Extra Strength750</td><td rowspan=1 colspan=1>Calcium Carbonate</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Gaviscon Extra Strength Liquid Antacid</td><td rowspan=1 colspan=1>Aluminum Hydroxide,Magnesium Hydroxide</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Mesalazine</td><td rowspan=1 colspan=1>S. Amino Salicylic Acid</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Immodium® AD Anti-Diarrheal</td><td rowspan=1 colspan=1>Loperamide Hydrochloride</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol Max Strength</td><td rowspan=1 colspan=1>Bismuth subsalicylate</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>MetronidazoleTopical Cream (0.75%)</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Mucin from bovine submaxillary glands,Type I-S (Dehydrated)</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin (Antibiotic)</td><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Control (no interferent)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Testing results correctly detected all the bacterial target organisms with one exception. In the Campylobacter jejuni sample with Hydrocortizone, two of the three replicates gave the expected calls while one replicate reported detection of Stx1 in addition to the detection of Campylobacter jejuni. The additional call for Stx1 target was determined to be a result of sample to sample contamination or error in sample preparation since both Campylobacter jejuni and $E$ . coli (Stx1) stocks were being handled during sample prep. The Stx1 signal was similar with other Stx1 samples in the study, supporting the conclusion that a sample preparation issue caused the false positive. The correct call was made for Campylobacter jejuni therefore the results support the conclusion that Hydrocortizone did not interfere with the test.

# Competitive inhibition

In order to assess competitive inhibition in the EP test, binary combinations of all six of the EP test panel organisms (see Table 19) representing all possible dual infections, were evaluated. Contrived samples were prepared in Negative Stool Matrix (NSM), with one panel organism present at a Low Positive titer $( 3 \mathrm { x } \ \mathrm { L o D } )$ and a second organism present at a High Positive titer $\mathrm { ( > 1 0 ^ { 6 } C F U / m L }$ stool). The performance of the EP test was evaluated with each of the 30 unique sample combinations tested in replicates of three.

<table><tr><td rowspan=1 colspan=5>Table 19: Target Strains and Titers used to Assess Competitive Inhibition.</td></tr><tr><td rowspan=1 colspan=1>Organism(Source/Strain)</td><td rowspan=1 colspan=1>Expected Calls</td><td rowspan=1 colspan=1>LoD (CFU/mL)</td><td rowspan=1 colspan=2>Test Concentrations(CFU/mL Stool)Low Positive   High Positive</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli(ATCC 43482)</td><td rowspan=1 colspan=1>Campylobacter Detected</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.11x107</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica(ATCC 13314)</td><td rowspan=1 colspan=1>Salmonella Detected</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.00x106</td><td rowspan=1 colspan=1>1.00x107</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae(Stx1) (ATCC 29026)</td><td rowspan=1 colspan=1>Shigella sp. and Shiga Toxin 1Detected</td><td rowspan=1 colspan=1>3.70x104</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.70x106</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (ATCC39315)</td><td rowspan=1 colspan=1>Vibrio Detected</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.11x107</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica(ATCC 23715)</td><td rowspan=1 colspan=1>Y. enterocolitica Detected</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.11x107</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (Stx2)(ATCC BAA-176)</td><td rowspan=1 colspan=1>Shiga Toxin 2 Detected</td><td rowspan=1 colspan=1>1.11x105</td><td rowspan=1 colspan=1>3.33x105</td><td rowspan=1 colspan=1>1.11x107</td></tr></table>

<table><tr><td rowspan=1 colspan=9>Table 20: Results from Competitive Inhibition Study</td></tr><tr><td rowspan=3 colspan=1>Organism at low titers(3x LoD)</td><td rowspan=1 colspan=6>Organisms at High Titer(&gt; 106 CFU/mL stool)</td><td rowspan=2 colspan=2>Total DetectionRate</td></tr><tr><td rowspan=2 colspan=1>cco mpqe</td><td rowspan=2 colspan=1>Se pce</td><td rowspan=2 colspan=1>she mieMA</td><td rowspan=2 colspan=1>i ca</td><td rowspan=2 colspan=1>R u</td><td rowspan=2 colspan=1>Ts tc</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coliATC 43482</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2*(6/6)</td><td rowspan=1 colspan=1>14/15(20/21)*</td><td rowspan=1 colspan=1>93.3%(95.2%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericaATCC 13314</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Stx1)ATCC 29026</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Vibrio choleraeATCC 39315</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocoliticaATCC 23715</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (Stx2)ATCC BAA-176</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td></tr></table>

\*In one of three replicates, Campylobacter was not detected; E. coli (Stx2) was correctly identified

Examination of the image intensity values for the single aberrant test showed that signal was present for Campylobacter coli, but the detection threshold was not met at the 320 msec exposure cap limit within the call algorithm. As a result of the false negative result for Campylobacter, an additional six (6) replicate tests of the Low Titer Campylobacter coli and High Titer E. coli sample was performed. All the additional tests yielded correct calls for both organisms, suggesting that the errant result was not indicative of systemic competitive inhibition. Across the different binary combinations, Campylobacter coli at $3 \mathbf { x }$ LoD was correctly detected in 20 of 21 tests, for a call accuracy rate of $9 5 . 2 \%$ .

# Carryover/Cross-contamination

The carryover/cross-contamination study was performed across six Processor $S P s$ over six separate test runs. Six samples, each containing a High Positive level $( 5 \mathrm { x } 1 0 ^ { 6 }$ CFU/mL) of one of the test panel members spiked into the Negative Stool Matrix were evaluated. The same Negative Stool Matrix was utilized as the negative sample.

<table><tr><td rowspan=1 colspan=7>Table 21: Testing Layout of the Carryover Study</td></tr><tr><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=6>Verigene Processor SP</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>Shigella, Stx1</td><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>Sal.</td><td rowspan=1 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>Shigella, Stx1</td><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>Sal.</td><td rowspan=1 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>Shigella, Stx1</td><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>Sal.</td><td rowspan=1 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr></table>

All of the negative samples yielded “Not Detected” calls for all the analytes, with the exception of a single sample (Run 2, SP5) which generated a “No Call – INT CTL1” result. For this sample, analysis of capture spot intensities showed no residual Salmonella signal from the previously run sample and the “No Call” result was not attributable to a carry-over event. Following the “No Call” result, both the High titer Salmonella enterica sample and the Negative sample were repeat-tested, with the negative sample giving a valid “Not Detected” call. In addition, all the High Positive samples yielded the expected “Detected” results for the intended organism and “Not Detected” results for the other analytes. These results are acceptable to establish the carry-over/cross-contamination of the Enteric Pathogens Nucleic Acid Test.

3. Comparison studies:

Method comparison with predicate device: Not applicable Matrix comparison: Not applicable

4. Clinical studies:

A method comparison study was conducted at seven external clinical study testing sites: Children’s Hospital Central California, Children’s Hospital Los Angeles, Laboratory Alliance of Central New York, Medical College of Wisconsin, Southern California Permanente Medical Group, University of Texas Health Science Center at Houston, and Washington University. The study was conducted under IRB supervision at each testing and specimen acquisition site.

The Verigene System, including the Verigene Processor $S P$ and the Verigene Reader, the Verigene EP Test Kits (EP Test Cartridges and EP Extraction Trays) and EP Amplification Trays used in the method comparison studies were identical to the devices intended for commercialization with respect to their functional features and reagent composition. Instrument and kit components were manufactured in compliance with the applicable sections of the Quality System Regulations. The testing for this study was performed using multiple lots of Test Cartridges, Extraction Trays, and Amplification Trays. Verigene Processor SPs and Verigene Readers used in these studies were standard production instruments. The software versions were validated for use according to documented test protocols and developed according to Standard Operating Procedures. Each site utilized one Verigene Reader and multiple Verigene Processor $S P$ instruments, depending on each site’s specimen enrollment and testing volume.

De-identified specimens tested in this study were enrolled from individuals receiving routine care and were recommended for enteric pathogen testing. A portion of each leftover residual unformed stool specimen in Cary-Blair media was obtained for testing. The study inclusion and exclusion criteria are listed below:

# Inclusion Criteria

Specimen must be collected from subjects/patients which meet the institute’s criteria for a suspected enteric pathogen, such as individuals presenting with symptoms of gastrointestinal infection (e.g., significant diarrhea—new onset of 3 or more unformed or; watery stools per 24 hour period, abdominal pain, foul stool odor, etc.);   
· Specimen will consist of unformed (liquid or soft) stool specimens preserved in Cary- Blair (or equivalent); Specimen will be residual and de-identified; Specimen must be of sufficient volume $\geq 5 \mathrm { m L }$ fresh stool preserved in Cary-Blair media or equivalent; Fresh, unpreserved specimen must be transferred to Cary-Blair, or equivalent, within 2 to 4 hours of specimen collection;   
· Cary-Blair media specimen must have been stored according to manufacturer’s directions before enrollment in the study;   
· Upon enrollment in the study, store the specimen at Room Temperature; Specimens must be shipped for Reference Testing within 60 hours of stool specimen collection;   
· Specimens must be prepared into Stool Prep Buffer within 48 hours of collection;   
· Specimens must have initial EP test performed within 24 hours of preparation of Stool Prep Buffer;   
· Best efforts to avoid enrolling multiple specimens from the same patient will be made however, it is recognized that due to the specimen de-identification process this cannot be guaranteed.

# Exclusion Criteria

Inability to perform EP on a specimen due to a protocol deviation (e.g., insufficient specimen, residual extracted nucleic acid not retained, incorrect shipping, handling or storage); Nucleic acid not recovered from Extraction Tray;   
· Any fresh, preserved specimen not tested within 48 hours of collection;   
· Any specimen not tested within 24 hours of Stool Prep Buffer preparation;   
· Hard stool (not unformed, liquid, or soft);   
· Specimen contaminated with urine or water; Leaking container; Insufficient volume; Specimens received unpreserved which are put into Cary-Blair, or equivalent, ${ > } 4$ hours from collection; Specimen containing interfering substances such as castor oil, bismuth, Metamucil, barium, Vaseline, or other cream contaminants (refer to institution specimen collection criteria);   
· Specimens collected in diapers; Rectal swab specimens; Specimens arriving to lab already in preservative collection media other than Cary-Blair (or equivalent);   
· Specimens not collected according to manufacturer’s instructions (under-filled or overfilled specimens); Cary-Blair media is a yellowish color.

The study utilized four categories of specimens to establish performance:

1) Fresh: Prospectively-collected fresh Cary- Blair specimens enrolled and tested at the study test sites;   
2) Frozen: Prospectively-collected frozen Cary-Blair specimens enrolled and tested at the study test sites;   
3) Selected: De-identified archived frozen specimens collected and stored in CaryBlair media were obtained from the specimen acquisition sites. These specimens were shipped frozen from each specimen acquisition site to the Sponsor, blinded and sent to the method comparison study testing sites for testing. In parallel, each specimen was tested by analyte-specific PCR amplification and bi-directional sequencing (BDS), following a previously approved protocol. This testing was performed to confirm the original analyte identification. If the original analyte identification could not be confirmed, the specimen was excluded from the method comparison study.

4) Simulated: Simulated frozen specimens were seeded Cary-Blair specimens using glycerol stocks and shipped to/tested at the study test sites. Each simulated specimen was made using a unique strain of the intended organism.

For all fresh and frozen prospective specimens bacterial identification reference testing was performed at a central laboratory. Each specimen was cultured and suspected EP colonies were isolated and run for Phenotype Identification on an FDA Cleared Automated System. All samples were enriched in MacConkey Broth for suspected Shiga toxin producing organisms followed by EHEC EIA testing. In addition, bi-directional sequencing (BDS) systems were developed to confirm identity of bacterial target analytes. Overall, these BDS systems were used in the EP test Method Comparison study in various capacities: (i) for species-level identification of clinical positives, if applicable, (ii) typing of virulence markers (Stx1, Stx2), and (iii) discordant troubleshooting between the EP test and the reference method identification. An overview of comparator methods performed for each analyte are listed in Table 22.

<table><tr><td rowspan=1 colspan=2>Table 22: Comparator Methods Used During the Clinical Validation Study</td></tr><tr><td rowspan=1 colspan=1>EP Analyte</td><td rowspan=1 colspan=1>Comparator Methods</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=5 colspan=1>Bacterial Culture and Automated Phenotype Identification using FDA ClearedMethods</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>Shigella</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Stx1</td><td rowspan=1 colspan=1>MacConkey Broth Enrichment followed by EHEC EIA and PCR amplification//bi-directional sequencing for Confirmation and Typing</td></tr><tr><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>MacConkey Broth Enrichment followed by EHEC EIA and PCR amplification/bi-directional sequencing for Confirmation and Typing</td></tr></table>

# Method Comparison Study Results

From the original number of enrolled samples in the study, fifty-nine specimens were excluded as indicated in Table 23. The final dataset is composed of 1243 Prospectivelycollected fresh Cary-Blair specimens enrolled and tested at the study test sites, 34 Prospectively-collected frozen Cary-Blair specimens enrolled and tested at the study test sites, 166 Selected frozen specimens collected by sample acquisition sites, and 409 Simulated frozen specimens as summarized in Table 24.

Page - 34/45  

<table><tr><td rowspan=1 colspan=8>Table 23: Non-evaluable Clinical Specimens and Reasons for Exclusion from Data</td></tr><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>No.Excluded</td><td rowspan=1 colspan=6>Reason for Exclusion from the Validation Dataset</td></tr><tr><td rowspan=1 colspan=1>TimeInclusionCriteria NotMet</td><td rowspan=1 colspan=1>OperatorError</td><td rowspan=1 colspan=1>ShippingIssue</td><td rowspan=1 colspan=1>InvalidQC</td><td rowspan=1 colspan=1>No/InconclusiveReferenceResult</td><td rowspan=1 colspan=1>InvalidSpecimen</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr></table>

<table><tr><td rowspan=1 colspan=6>Table 24: Number of Valid Specimens Tested per Specimen Category</td></tr><tr><td rowspan=1 colspan=1>CategoryNo.</td><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Collected/Preparedby</td><td rowspan=1 colspan=1>Stored</td><td rowspan=1 colspan=1>Tested at</td><td rowspan=1 colspan=1>ValidSpecimens</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Study test sites</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=4 colspan=1>Study testsites</td><td rowspan=1 colspan=1>1243</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Study test sites</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Sample AcquisitionSites</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>166</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>Nanosphere</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>409</td></tr><tr><td rowspan=1 colspan=5>TOTAL</td><td rowspan=1 colspan=1>1852</td></tr></table>

Table 25 shows a summary of demographic information for 1262 of the 1277 prospectively collected specimens in the valid dataset (age was not recorded for 15 specimens).

<table><tr><td rowspan=1 colspan=3>Table 25: Prospectively Collected Specimens Stratified by Patient Age (n=1262)</td></tr><tr><td rowspan=1 colspan=1>Age Range</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>4.8%</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>6.7%</td></tr><tr><td rowspan=1 colspan=1>&gt;12-21</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>11.0%</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>609</td><td rowspan=1 colspan=1>48.3%</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>25.5%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1262</td><td rowspan=1 colspan=1>100%</td></tr></table>

The initial call rate for the Method Comparison study was $9 6 . 0 \%$ . Of the 76 initial NoCalls, 51 yielded an evaluable test result upon retesting for a final call rate of $9 8 . 7 \%$ . The revised initial Pre-Analysis Error rate was $0 . 9 \%$ , and the final Pre-Analysis Error rate was $0 \%$ . The 25 specimens which yielded a final “No Call” result were not included in the evaluable dataset utilized in the comparative test result data analysis, since it was not

possible to collect and test a second specimen from the patient, as per the study protocol instructions. Therefore, 1852 specimens were analyzed to establish clinical performance of the test, 1243 of which were prospectively-collected fresh specimens, 34 of which were prospectively-collected frozen specimens, 166 selected samples, and 409 of which were simulated frozen specimens.

The following clinical performance tables provide a summary of the clinical performance of the Enteric Pathogens Nucleic Acid Test for the detection of five bacterial targets and the Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers, compared to the comparator methods noted above.

Table 26: Summary of the Clinical Performance for Campylobacter spp. and Salmonella spp.   

<table><tr><td rowspan=8 colspan=1>dsqup</td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>peoece-ceesed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>90.5%19/21a, q(69.6-98.8)</td><td rowspan=1 colspan=1>98.8%1207/1222d, q(98.0-99.3)</td><td rowspan=4 colspan=1>Ss op</td><td rowspan=4 colspan=1>poec-Cceced</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>85.7%18/21f(63.7-97.0)</td><td rowspan=1 colspan=1>99.4%1215/1222h(98.8-99.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%2/2(15.8-100)</td><td rowspan=1 colspan=1>100%32/32(89.1-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>97.0%32/33(84.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>97.5%39/40c(86.8-99.9)</td><td rowspan=1 colspan=1>99.2%125/126(95.7-100)</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>98.2%53/54g(90.1-100)</td><td rowspan=1 colspan=1>99.1%111/112j(95.1-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>95.2%60/63(86.7-99.0)</td><td rowspan=1 colspan=1>98.8%1364/1380(98.1-99.3)</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>94.7%72/76(87.1-98.6)</td><td rowspan=1 colspan=1>99.3%1358/1367(98.8-99.7)</td></tr><tr><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>98.5%67/68b(92.1-100)</td><td rowspan=1 colspan=1>100%341/341(98.9-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%67/67(94.6-100)</td><td rowspan=1 colspan=1>100%342/342(98.9-100)</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>97.0%127/131(92.4-99.2)</td><td rowspan=1 colspan=1>99.1%1705/1721(98.5-99.5)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>97.2%139/143(93.0-99.2)</td><td rowspan=1 colspan=1>99.5%1700/1709(99.0-99.8)</td></tr></table>

Table 27: Summary of the Clinical Performance for Shigella spp. and Vibrio spp.   

<table><tr><td rowspan=8 colspan=1>dds ps</td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>Ppeoec-Cccec</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>66.7%2/3k,r(9.4-99.2)</td><td rowspan=1 colspan=1>98.7%1224/12401(97.9-99.3)</td><td rowspan=4 colspan=1>dds o</td><td rowspan=4 colspan=1>poec-Ccecced</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1242</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%1242/1242(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>97.1%33/34m(84.7-99.9)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%33/33(89.4-100)</td></tr><tr><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>100%6/6(54.1-100)</td><td rowspan=1 colspan=1>99.4%159/160n(96.6-100)</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%165/165(97.8-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>88.9%8/9(51.8-99.7)</td><td rowspan=1 colspan=1>98.7%1416/1434(98.0-99.3)</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>100%3/3(29.2-100)</td><td rowspan=1 colspan=1>100%1440/1440(99.7-100)</td></tr><tr><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%50/50(92.9-100)</td><td rowspan=1 colspan=1>100%359/359(99.0-100)</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>91.1%51/56°(80.4-97.0)</td><td rowspan=1 colspan=1>99.7%352/353P(98.4-100)</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>98.3%58/59(90.9-100)</td><td rowspan=1 colspan=1>99.0%1775/1793(98.4-99.4)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>91.5%54/59 (81.3-97.2)</td><td rowspan=1 colspan=1>99.9%1792/1793(99.7-100)</td></tr></table>

<table><tr><td rowspan=1 colspan=6>Table 28: Summary of the Clinical Performance for Y.enterocolitica</td></tr><tr><td rowspan=8 colspan=1>R</td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>poecc-Ccec</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%1243/1243(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td></tr><tr><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%165/165(97.8-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%1442/1442(99.7-100)</td></tr><tr><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%59/59(93.9-100)</td><td rowspan=1 colspan=1>100%350/350(99.0-100)</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>100%60/60(94.0-100)</td><td rowspan=1 colspan=1>100%1792/1792(99.8-100)</td></tr></table>

<table><tr><td rowspan=1 colspan=6>Table 29: Footnoted Information for Table 26 through Table 28</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Specimen Type(concentration)</td><td rowspan=1 colspan=1>EP TestResult</td><td rowspan=1 colspan=1>Reference Method Result(s)</td><td rowspan=1 colspan=1>PCR Amplification and BDS Results</td></tr><tr><td rowspan=2 colspan=1>a.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C. jejuni subsp. jejuni</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C. jejuni subsp. jejuni &amp; Proteus spp.</td><td rowspan=1 colspan=1>Negative for Campylobacter spp.</td></tr><tr><td rowspan=1 colspan=1>b.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C. lari</td><td rowspan=1 colspan=1>Positive for Campylobacter lari</td></tr><tr><td rowspan=1 colspan=1>c.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Low-Level Positive for Campylobacterjejuni (at LoD; Negative upon repeat))</td></tr><tr><td rowspan=15 colspan=1>d.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter coli</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>M. morganii subsp. morganii &amp; N. cinerea</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>M. morganii subsp. morganii</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni.</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>C. braakii &amp; E. cloacae subsp dissolvens &amp; N.cinerea</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter spp.</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter spp.</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>e</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>CampylobacterandSalmonella</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni andSalmonella enterica</td></tr><tr><td rowspan=3 colspan=1>f.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>g.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Low-Level Positive for Salmonellaenterica (at LoD; Negative upon repeat)</td></tr><tr><td rowspan=5 colspan=1>h.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for Salmonella spp.</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>C. freundii &amp; Proteus spp.</td><td rowspan=1 colspan=1>Negative for Salmonella spp.</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr></table>

Page - 38/45   

<table><tr><td rowspan=1 colspan=6>Table 29: Footnoted Information for Table 26 through Table 28</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Specimen Type(concentration)</td><td rowspan=1 colspan=1>EP TestResult</td><td rowspan=1 colspan=1>Reference Method Result(s)</td><td rowspan=1 colspan=1>PCR Amplification and BDS Results</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>P. alcalifaciens</td><td rowspan=1 colspan=1>Negative for Salmonella spp.</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>i.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>j.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>k.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=16 colspan=1>l.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>A.hydrophila/cavieae &amp; P. putida</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp.</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp.</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp.</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>m.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>P. rettgeri</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>n.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Shiga toxin</td><td rowspan=1 colspan=1>Positive for Shiga toxin 1</td></tr><tr><td rowspan=5 colspan=1>0.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Positive for V. parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Positive for V. cholerae</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Simulated(40X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Negative for V. cholerae</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Simulated(40X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Negative for V. parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Simulated(40X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Positive for V. cholerae</td></tr><tr><td rowspan=1 colspan=1>p.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Simulated(30X)</td><td rowspan=1 colspan=1>Campylobacterand Vibrio</td><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=1 colspan=1>q.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=4>  result of a sample mix-up.</td></tr><tr><td rowspan=1 colspan=1>r.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=4>   mix-up.</td></tr></table>

<table><tr><td rowspan=1 colspan=12>Table 30: Summary of the Clinical Performance or Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers.</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=6 colspan=1>A</td><td rowspan=4 colspan=1>peoe-cecsd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>100%4/4(39.8-100)</td><td rowspan=1 colspan=1>99.7%1236/1239a(99.2-99.9)</td><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1>poce-cetec</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>100%6/6(54.1-100)</td><td rowspan=1 colspan=1>99.8%1235/1237d(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td></tr><tr><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>100%9/9(66.4-100)</td><td rowspan=1 colspan=1>99.4%156/157b(96.5-100)</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>100%9/9(66.4-100)</td><td rowspan=1 colspan=1>100%157/157(97.7-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>100%13/13(75.3-100)</td><td rowspan=1 colspan=1>99.7%1426/1430(99.3-99.9)</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1443</td><td rowspan=1 colspan=1>100%15/15(78.2-100)</td><td rowspan=1 colspan=1>99.9%1426/1428(99.5-100)</td></tr><tr><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%51/51(93.0-100)</td><td rowspan=1 colspan=1>99.4%356/358c(98.0-99.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>96.7%58/60d(88.5-99.6)</td><td rowspan=1 colspan=1>99.7%348/349(98.4-100)</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>100%64/64(94.4-100)</td><td rowspan=1 colspan=1>99.7%1782/1788(99.3-99.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1852</td><td rowspan=1 colspan=1>97.3%73/75(90.7-99.7)</td><td rowspan=1 colspan=1>99.8%1774/1777(99.5-100)</td></tr></table>

<table><tr><td rowspan=1 colspan=6>Table 31: Footnote Information for Table 30</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Fresh, Frozen orSimulated (xLoD)</td><td rowspan=1 colspan=1>Identified by EP test as:</td><td rowspan=1 colspan=1>Identified by Reference Method(s)as:</td><td rowspan=1 colspan=1>PCR Amp/BD SequencingResults (if applicable)</td></tr><tr><td rowspan=3 colspan=1>a.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 andNorovirus</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 and ShigaToxin 2</td><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>Positive for Stx 1 gene andStx 2 gene</td></tr><tr><td rowspan=1 colspan=1>b.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Shiga Toxin 1 andCampylobacter</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=2 colspan=1>c.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Simulated (13X)</td><td rowspan=1 colspan=1>Shiga Toxin 1andSalmonella</td><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica</td><td rowspan=1 colspan=1>Negative for Stx 1 gene andStx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Simulated (30X)</td><td rowspan=1 colspan=1>Shiga Toxin 1andCampylobacter</td><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=2 colspan=1>d.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 and ShigaToxin 2</td><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>Positive for Stx 1 gene andStx 2 gene</td></tr><tr><td rowspan=1 colspan=1>e.</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Simulated (31X)</td><td rowspan=1 colspan=1>Shiga Toxin 2 and Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Negative for Stx 1 gene andStx 2 gene</td></tr></table>

<table><tr><td rowspan=1 colspan=10>Table 32: Summary of Genus/Group-level Test Performance vs. Reference Methods(s) - Stratified by Species</td></tr><tr><td rowspan=1 colspan=5>Campylobacter Genus</td><td rowspan=1 colspan=5>Shigella Genus</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ProspectiveFresh/Frozen</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>Analytical</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ProspectiveFresh/Frozen</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>Analytical</td></tr><tr><td rowspan=1 colspan=1>CombinedCampylobacter</td><td rowspan=1 colspan=1>91.3%21/23(72.0-99.0)</td><td rowspan=1 colspan=1>97.5%39/40(86.8-99.9)</td><td rowspan=1 colspan=1>98.5%67/68(86.3-100)</td><td rowspan=1 colspan=1>100%115/15(78.2-100)</td><td rowspan=1 colspan=1>CombinedShigella</td><td rowspan=1 colspan=1>66.7%2/3(9.4-99.2)</td><td rowspan=1 colspan=1>100%6/6(54.1-100)</td><td rowspan=1 colspan=1>100%0/50(92.9-100)</td><td rowspan=1 colspan=1>100%20/20(83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Campylobactercoli</td><td rowspan=1 colspan=1>100%3/3 (29.2-100)</td><td rowspan=1 colspan=1>100%3/3 (29.2-100)</td><td rowspan=1 colspan=1>100%18/18(81.5-100)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100)</td><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%14/14(76.8-100)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100)</td></tr><tr><td rowspan=1 colspan=1>Campylobacterejuni</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>97.3%36/37(85.8-99.9)</td><td rowspan=1 colspan=1>100%9/9 (66.4-1100)</td><td rowspan=1 colspan=1>100%4/4 (40.0-100)</td><td rowspan=1 colspan=1>Shigelladysenteriae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%9/9 (66.4-1100)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100)</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni subsp.doylei</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%5/5 (47.8-100</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%16/16(79.4-100)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni subsp.jejuni</td><td rowspan=1 colspan=1>89.5%17/19(66.9-98.7)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%21/21(83.9-100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>100%2/2(15.8-100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%11/11(71.5-100)</td><td rowspan=1 colspan=1>100%5/5(47.8-100)</td></tr><tr><td rowspan=1 colspan=1>Campylobacterlari</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>93.3%14/15(68.1-99.8)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100)</td><td rowspan=1 colspan=1>Shigella spp notientified</td><td rowspan=1 colspan=1>0%0/1(0-97.5)</td><td rowspan=1 colspan=1>100%6/6 (54.1-100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>alm</td><td rowspan=1 colspan=1>Salmonella Genus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Vibrio Genus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ProspectiveFresh/Frozen</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>Analytical</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ProspectiveFresh/Frozen</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>Analytical</td></tr><tr><td rowspan=1 colspan=1>CombinedSalmonella</td><td rowspan=1 colspan=1>86.4%19/22 (65.1-97.1)</td><td rowspan=1 colspan=1>98.2%553/54(90.1-100)</td><td rowspan=1 colspan=1>100%67/67(94.6-100)</td><td rowspan=1 colspan=1>100%31/31(88.8-100)</td><td rowspan=1 colspan=1>CombinedVibrio</td><td rowspan=1 colspan=1>100%2/2 (15.8-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>91.1%51/256(80.4-97.0)</td><td rowspan=1 colspan=1>100%10/10(69.2-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella&quot;non-typhi&quot;</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%2/2 (15.8-10)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>84.2%16/19(60.4-96.6)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100</td></tr><tr><td rowspan=1 colspan=1>Salmonellabongori</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%2/2 (15.8-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>100%2/2 (15.8-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td><td rowspan=1 colspan=1>94.6%35/37(81.8-99.3)</td><td rowspan=1 colspan=1>100%5/5 (47.8-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>97.6%40/41(87.1-99.9</td><td rowspan=1 colspan=1>100%1/1 (2.5-100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=8 colspan=5></td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.arizonae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.diarizonae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%3/3 (29.2-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.enterica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%52/52(93.2-100)</td><td rowspan=1 colspan=1>100%25/25(86.3-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.houtenae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%2/2 (15.8-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.indica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%3/3 (29.2-10</td><td rowspan=1 colspan=1>100%1/1 (2.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica subsp.salamae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%2/2 (15.8-100)</td><td rowspan=1 colspan=1>100%1/1 (2.5-100</td></tr><tr><td rowspan=1 colspan=1>Salmonellaspp notidentified</td><td rowspan=1 colspan=1>86.4%19/22(65.1-97.1)</td><td rowspan=1 colspan=1>100%13/13(75.3-100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The clinical study protocol specified running QC samples each day of testing, utilizing one external negative control and one of eleven external positive controls (tested on a rotating basis) representing all target analytes. Summary data for quality control test runs are provided in Table 33. Since the Enteric Pathogens Nucleic Acid Test is a single-use device, QC failures do not occur on the same test run during which a clinical specimen result is generated. Therefore, an initial QC failure would not lead to the exclusion of a clinical specimen result, provided a valid QC result was obtained upon repeat testing during the same day as the initial QC failure.

<table><tr><td rowspan=1 colspan=14>Table 33: Summary of QC Test Run Results during the Clinical Validation Study</td></tr><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>TotalQCTestRuns</td><td rowspan=2 colspan=1>NegativeQCTests</td><td rowspan=1 colspan=11>Positive QC Tests(11 Samples on a Rotating Basis)</td></tr><tr><td rowspan=1 colspan=1>Cr</td><td rowspan=1 colspan=1>Ssoe</td><td rowspan=1 colspan=1>shm</td><td rowspan=1 colspan=1>Shs sI xI</td><td rowspan=1 colspan=1>Z x us</td><td rowspan=1 colspan=1>Pooeoh</td><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>GNoN</td><td rowspan=1 colspan=1>SNN</td><td rowspan=1 colspan=1>Aaau</td><td rowspan=1 colspan=1>Shomn</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>552</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Additional</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr></table>

Specimens co-infected with multiple test targets are shown in the tables below. As shown in Table 34, a total of 19 specimens were determined by the Enteric Pathogens Nucleic Acid Test to be co-infected by multiple targets ( $1 . 0 \%$ of evaluable specimens 19/1852). The majority of multiple target specimens contained two separate targets (18/19) while 1 specimen contained three separate targets.

<table><tr><td rowspan=1 colspan=7>Table 34: Multiple Target Combinations Detected by EP</td></tr><tr><td rowspan=1 colspan=3>Multiple Target Combinations Detected by EP</td><td rowspan=1 colspan=3>Reference Test</td><td rowspan=2 colspan=1>No. ofSamplesDiscrepant</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>DiscrepantSpecimens</td><td rowspan=1 colspan=1>DiscrepantIdentification</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=2 colspan=1>Shiga Toxin 1</td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>Stx 1 gene</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 gene</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 gene</td><td rowspan=1 colspan=1>1</td></tr></table>

Page - 42/45   

<table><tr><td rowspan=1 colspan=7>Table 34: Multiple Target Combinations Detected by EP</td></tr><tr><td rowspan=1 colspan=3>Multiple Target Combinations Detected by EP</td><td rowspan=1 colspan=3>Reference Test</td><td rowspan=2 colspan=1>No. ofSamplesDiscrepant</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>DiscrepantSpecimens</td><td rowspan=1 colspan=1>DiscrepantIdentification</td></tr><tr><td rowspan=2 colspan=1>Shiga Toxin 1</td><td rowspan=2 colspan=1>Shiga Toxin 2</td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=1>10</td><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 geneStx 2 gene</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=2></td></tr></table>

Table 35 shows the number of co-infected specimens which were detected by the comparator method.

<table><tr><td rowspan=1 colspan=6>Table 35: Multiple Target Combinations by Comparator Method Test</td></tr><tr><td rowspan=1 colspan=3>Multiple Target Combinations by ComparatorMethod</td><td rowspan=1 colspan=3>Detected by EP</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>DiscrepantSpecimens</td><td rowspan=1 colspan=1>DiscrepantTargets</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

5. Clinical cut-off:

Not Applicable.

6. Expected values:

In the Verigene Enteric Pathogens Nucleic Acid Test Method Comparison study, 1277 prospectively collected fresh and frozen specimens were obtained from seven medium to large-sized healthcare institutions geographically distributed across the United States. The number and percentage of positive cases (positivity rate) determined by the Enteric Pathogens Nucleic Acid Test stratified by geographic region for each of the organisms detected are presented in Table 36. Overall, the Enteric Pathogens Nucleic Acid Test detected at least one target in $7 . 7 \%$ of prospectively-collected specimens. In routine practice, prevalence rates may vary depending on the institution, geographical location, and patient population.

<table><tr><td rowspan=1 colspan=8>Table 36: Prevalence of Organisms Detected by the Enteric Pathogens Nucleic Acid Test (EP) Clinical Study Observations</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>US Geographic Region/Division*</td><td rowspan=4 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=1>Region</td><td rowspan=1 colspan=2>Midwest</td><td rowspan=1 colspan=1>South</td><td rowspan=1 colspan=1>Northeast</td><td rowspan=1 colspan=1>West</td></tr><tr><td rowspan=1 colspan=1>Division</td><td rowspan=1 colspan=1>WestNorthCentral</td><td rowspan=1 colspan=1>EastNorthCentral</td><td rowspan=1 colspan=1>W. SouthCentral</td><td rowspan=1 colspan=1>MiddleAtlantic</td><td rowspan=1 colspan=1>Pacific</td></tr><tr><td rowspan=1 colspan=1>State</td><td rowspan=1 colspan=1>MO</td><td rowspan=1 colspan=1>WI</td><td rowspan=1 colspan=1>TX</td><td rowspan=1 colspan=1>NY</td><td rowspan=1 colspan=1>CA</td></tr><tr><td rowspan=1 colspan=1>Totaln=</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>1277</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Y. entercolitica</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Stx1</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>POS n=</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.6</td></tr></table>

\*Geographic Areas Reference Manual (US Census Bureau).Chapter 6

N. Instrum entNam es: Verigene System

# O .System Descriptions:

1. Modes of Operation:

The Verigene System has one mode of operation available to the User. Prior to initiating a test run the User places the sample into the sample loading well, loads the consumable, and enters identifying information into the system. After the Processor SP run has completed, the test cartridge is removed from the instrument and the reagent pack is manually separated from the Substrate Holder. The barcode on the substrate holder is scanned prior insertion into the Reader. The Reader images the Substrate and automatically determines the test result without further user intervention.

1. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_ ___ or No

2. Specimen Identification:

All tests must be ordered through the Verigene Reader. No tests can be processed on the Verigene Processor $S P$ without the user entering the Test Cartridge ID and Sample ID to the Verigene Reader.

The User enters the Test Cartridge ID by scanning the barcode using the barcode scanner attached to the Reader. The user may manually enter in the Test Cartridge ID by selecting MENU and ‘Enter Barcode’ and then keying in the Test Cartridge ID number with the Reader’s keyboard. The User has the option to scan the Test Cartridge Cover’s 2D barcode using a barcode gun-style scanner to display the Test Cartridge’s Reference Number, Expiration Date, and Lot Number on reports. The User enters the Sample ID by scanning or manually enter the Sample ID using the Reader’s touch-screen keyboard then confirming the Sample ID in the software.

3. Specimen Sampling and Handling:

Inadequate or inappropriate specimen collection, storage, or transport may yield falsenegative results. Due to the importance of specimen quality, training of personnel in the correct manner to perform specimen collection and handling is highly recommended.

Collect stool preserved in Cary-Blair media by using the media manufacturer’s recommended collection procedure or collect unpreserved and unformed (liquid or soft) stool specimens and place as soon as possible into the Cary-Blair media by using the following collection procedure.

1. Put on fresh gloves.   
2. For each Cary-Blair preserved specimen to be tested, place one sterile flocked swab and one uncapped Stool Prep Buffer tube (place the cap to the side for recapping later) into a biological safety cabinet (BSC).   
3. Wipe down the outside of the specimen vial with a lint-free decontaminating wipe.   
4. Invert the vial containing the Cary-Blair preserved specimen twice and vortex the specimen for 5-10 seconds to ensure homogeneity.   
5. To prepare the Stool Prep Buffer tube, dip the provided flocked swab into either the primary Cary-Blair preserved specimen vial or the secondary tube until the flocked tip is fully immersed in specimen. Once evenly coated, transfer the swab to the Stool Prep Buffer tube and break swab at the preformed scored breakpoint. Leave the swab in the Stool Prep Buffer tube and screw the cap finger tight on to Stool Prep Buffer tube.   
6. Recap the original, primary Cary-Blair preserved specimen container and set aside.   
7. Repeat steps 1-6 for each specimen, changing gloves between each specimen.   
8. Vortex each Stool Prep Buffer tube for 15-20 seconds.

9. Spin all prepared Stool Prep Buffer tubes in the Mini Centrifuge for 30-35 seconds at a MAXIMUM of 2200 rcf. 10. Pipette $2 0 0 \mu \mathrm { L }$ of the prepared supernatant into the Extraction Tray.

It is recommended that Cary-Blair preserved specimens be stored refrigerated at 2- $8 \mathrm { { } ^ { \circ } C }$ until testing is completed (for up to 48 hours after collection). For repeat testing, prepare the stored specimen in a new Stool Prep Buffer as described in the Specimen Processing section (see Section B) of the product labeling.

4. Calibration:   
There is no user calibration.   
5. Quality Control:   
See Section 2 for a description of the Quality Control material included with the test.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q .Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.