# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K140407

# B. Purpose for Submission:

To obtain clearance for the FilmArray® Gastrointestinal (GI ) Panel microorganism multiplex nucleic acid-based assay.

# C. Measurand:

Campylobacter (C. jejuni/C. coli/C. upsaliensis ), Clostridium difficile (C. difficile) toxin A/B , Plesiomonas shigelloides, Salmonella, Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio cholerae,Yersinia enterocolitica, Enteroaggregative Escherichia coli (EAEC), Enteropathogenic Escherichia coli (EPEC), Enterotoxigenic Escherichia coli (ETEC) lt/st, Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli O157 serogroup within STEC), Shigella/ Enteroinvasive Escherichia coli (EIEC), Cryptosporidium, Cyclospora cayetanensis, Entamoeba histolytica, Giardia lamblia (also known as G. intestinalis and G. duodenalis), Adenovirus F 40/41, Astrovirus, Norovirus GI/GII, Rotavirus A, Sapovirus (Genogroups I, II, IV, and V)

# D. Type of Test:

A multiplexed nucleic acid-based test intended for use with the FilmArray $^ \mathrm { \textregistered }$ instrument for the qualitative in vitro detection and identification of multiple bacteria, viruses, and parasites. The FilmArray GI Panel assay is performed directly from stool samples in Cary Blair transport media.

# E. Applicant:

BioFire Diagnostics, LLC.

F. Proprietary and Established Names:

FilmArray Gastrointestinal (GI) Panel

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 – Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: Class II

3. Product code: PCH, OOI

4. Panel: 83 (Microbiology)

# H. Intended Use:

1. Intended use(s):

The FilmArray Gastrointestinal (GI) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with the FilmArray Instrument. The FilmArray GI Panel is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites directly from stool samples in Cary Blair transport media obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following bacteria (including several diarrheagenic $E$ . coli/Shigella pathotypes), parasites, and viruses are identified using the FilmArray GI Panel:

· Campylobacter (C. jejuni/C. coli/C. upsaliensis)   
· Clostridium difficile (C. difficile) toxin A/B   
· Plesiomonas shigelloides   
· Salmonella   
· Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio cholerae   
· Yersinia enterocolitica   
· Enteroaggregative Escherichia coli (EAEC)   
· Enteropathogenic Escherichia coli (EPEC)   
· Enterotoxigenic Escherichia coli (ETEC) lt/st   
· Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli O157 serogroup within STEC)   
· Shigella/ Enteroinvasive Escherichia coli (EIEC)   
· Cryptosporidium   
· Cyclospora cayetanensis   
· Entamoeba histolytica   
· Giardia lamblia (also known as G. intestinalis and G. duodenalis)   
· Adenovirus F 40/41   
· Astrovirus   
· Norovirus GI/GII   
· Rotavirus A   
· Sapovirus (Genogroups I, II, IV, and V)

p. 2

The FilmArray GI Panel is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the FilmArray GI Panel. The agent detected may not be the definite cause of the disease.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

This device is not intended to monitor or guide treatment for C. difficile infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for $E$ . coli O157, Plesiomonas shigelloides, Yersinia enterocolitica, Astrovirus, and Rotavirus A were established primarily with retrospective clinical specimens.

Performance characteristics for Entamoeba histolytica, and Vibrio (V. parahaemolyticus, V. vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens.

Negative FilmArray GI Panel results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

2. Indication(s) for use:

Same as Intended Use

Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: For use with the FilmArray instrument

# I. Device Description:

The FilmArray Gastrointestinal (GI) Panel is a multiplex nucleic acid test designed to be used with the FilmArray Instrument. The FilmArray GI pouch contains freeze-dried reagents to perform nucleic acid purification and nested, multiplex PCR with DNA melt analysis. The FilmArray Gastrointestinal (GI) Panel simultaneously conducts 22 tests for the identification of GI pathogens from stool specimens collected in Cary Blair transport medium (see table below). Results from the FilmArray GI Panel test are available within about one hour.

Table – Organisms Detected by the FilmArray GI Panel   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni/C. coli/C. upsaliensis)</td><td rowspan=1 colspan=1>Adenovirus F 40/41</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (toxin A/B)</td><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Norovirus GI/GII</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Rotavirus A</td></tr><tr><td rowspan=1 colspan=1>Vibrio (V. parahaemolyticus/V. vulnificus/V. cholerae)</td><td rowspan=1 colspan=1>Sapovirus (Genogroups I, II, IV, and V)</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholera</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Diarrheagenic E. coli/Shigella</td><td rowspan=1 colspan=1>Parasites</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>Cryptosporidium</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>Cyclospora cayetanensis</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin-producing E. coli (STEC) stx1/stx2</td><td rowspan=1 colspan=1>Giardia lamblia</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1></td></tr></table>

A test is initiated by loading Hydration Solution into one port of the FilmArray pouch and a stool sample (in Cary Blair transport medium) mixed with the provided Sample Buffer into the other port of the FilmArray GI pouch and placing it in the FilmArray Instrument. The pouch contains all of the reagents required for specimen testing and analysis in a freezedried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray Instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green® Plus, BioFire). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR

products generated during the first stage PCR reaction. The $2 ^ { \mathrm { n d } }$ stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the $2 ^ { \mathrm { n d } }$ stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the $2 ^ { \mathrm { n d } }$ stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

Materials provided in each kit:

· Individually packaged FilmArray GI Panel pouches   
· Single-use $( 1 . 0 \mathrm { m L } )$ Sample Buffer ampoules   
· Single-use pre-filled $( 1 . 5 \mathrm { m L }$ ) Hydration Injection Vials (blue) Single-use Sample Injection Vials (red) Individually packaged Transfer Pipettes

Materials required but not provided: FilmArray System including:

FilmArray Instrument   
FilmArray Pouch Loading Station compatible with the use of the FilmArray   
Injection Vials

Note: Previous versions of Pouch Loading Station should not be used with the FilmArray Injection Vials.

For many organisms detected by the FilmArray GI Panel, the organism is considered to be detected if a single corresponding assay is positive. For example, Plesiomonas shigelloides will have a result of “Plesiomonas shigelloides Detected” if at least two of the three replicates of the one Plesiomonas shigelloides assay have similar positive melt peaks with Tm values that are within the assay-specific Tm range. The following organisms are detected using a single assay: toxigenic C. difficile, P. shigelloides, Salmonella, Y. enterocolitica, EAEC, Shigella/EIEC, Adenovirus F 40/41, Astrovirus, Sapovirus (Genogroups I, II, IV, and V), C. cayetanensis, E. histolytica and G. lamblia.

In contrast, the test results for several organisms rely on the combination of multiple assays. These include Campylobacter (C. jejuni/C. coli/C. upsaliensis), Vibrio (V. parahaemolyticus/ V. vulnificus/V. cholerae) and Vibrio cholerae, Cryptosporidium, Norovirus GI/GII, and Rotavirus A. The test results for several Diarrheagenic E. coli(s) include multiple assays for genetic markers to identify various classic pathotypes of $E$ . coli including EPEC, ETEC, and STEC (including O157), (as well EAEC and Shigella/EIEC included above). Interpretation rules for these assays are described below. Also included are summary descriptions of the assays’ expected reactivity; for a full description of assay reactivity see Inclusivity.

NOTE: As polymicrobial results with four or more distinct organisms in a single sample are unusual based upon data from the prospective clinical study, confirmation of this result is recommended to rule out any unexpected error, either caused by the user’s handling of the sample or the test system. Polymicrobial results of four or more organisms were detected in less than $1 \%$ (12/1556) of the FilmArray GI prospective study specimens.

# Bacteria

Campylobacter (C. jejuni/C. coli/C. upsaliensis): The FilmArray GI Panel contains two assays (Campy 1 and Campy 2) designed to together detect, but not differentiate, the most common Campylobacter species associated with human gastrointestinal illness: $C .$ . jejuni, C. coli, and C. upsaliensis. These are the same three species that are identified using standard clinical laboratory practices. Other Campylobacter species will not be identified by the FilmArray GI Panel. Empirical testing and in silico sequence analysis indicates reduced sensitivity for a less common subspecies of C. jejuni (C. jejuni subsp. doylei). A positive result for one or both assays will give a Campylobacter Detected test result.

Clostridium difficile toxin A/B: The FilmArray GI Panel contains a single multiplexed assay (Cdiff) for the identification of toxigenic C. difficile which targets both the toxin A gene (tcdA) and the toxin B gene (tcdB). Typical toxigenic strains produce both toxins, but the presence of either is indicative of a pathogenic strain. Empirical testing and in silico sequence analysis support that the assay will detect all toxinotypes and the epidemic BI/NAP1/027 hypervirulent strain, although these will not be specifically differentiated by the assay. Detection of either or both toxin genes by this assay gives a test result for Clostridium difficile toxin A/B Detected. As rates of asymptomatic carriage of $C .$ . difficile can be high in very young children and hospitalized patients, the detection of toxigenic C. difficile should be interpreted within the context of guidelines developed by the testing facility or other experts (e.g., guidelines/policy statements published by The American Academy of Pediatrics or the Society for Healthcare Epidemiology of America and the Infectious Disease Society of America).

Plesiomonas shigelloides: The FilmArray GI Panel contains a single assay (Pshig) for detection of P. shigelloides, the only known species in the genus Plesiomonas.

Salmonella: The FilmArray GI Panel contains a single assay (Salm) designed to detect both species of Salmonella; S. enterica and S. bongori. Empirical testing and in silico sequence analysis supports the ability to detect all subspecies and serovars of Salmonella. Cross-reactivity may occur with certain $E$ . coli strains containing variants of the cryptic ETT2 type-III secretion system (see Inclusivity for additional information).

Vibrio (V. parahaemolyticus/V. vulnificus/V. cholerae) and Vibrio cholerae: The FilmArray GI Panel contains a single assay (Vibrio) for detection of Vibrio species most commonly implicated in gastroenteritis (V. parahaemolyticus, V. vulnificus, and $V .$ cholerae). Empirical testing and in silico sequence analysis indicate that the assay may also react with some less common Vibrio species (i.e., V. alginolyticus, V. fluvialis, and $V .$ . mimicus). The Vibrio assay does not indicate which species has been detected and the Vibrio assay is not expected to detect the rarer V. cincinnatiensis, V. furnissii and $V .$ .

metschnikovii. A second assay (Vchol) is also included for the specific detection of Vibrio cholerae. A Vibrio cholerae Detected result will only be reported when the $V .$ . choleraespecific assay is positive, while a positive result for either assay will give a Vibrio Detected test result (see table below).

Table – Possible Assay Results and Corresponding Vibrio Test Results   

<table><tr><td rowspan=1 colspan=1>Film Array GI Interpretation</td><td rowspan=1 colspan=1>Vibrio(Vibrio Assay)</td><td rowspan=1 colspan=1>V.cholerae(Vchol Assay)</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Vibrio: Not DetectedVibrio cholerae: Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Vibrio species detected</td></tr><tr><td rowspan=1 colspan=1>Vibrio: DetectedVibrio cholerae: Not Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Vibrio species detected(not V. cholerae)</td></tr><tr><td rowspan=1 colspan=1>Vibrio: DetectedVibrio cholerae: Detected</td><td rowspan=1 colspan=1>Any Results</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Vibrio cholerae detectedORVibrio cholerae and one ormore other Vibrio speciesdetected</td></tr></table>

Yersinia enterocolitica: The FilmArray GI Panel contains a single assay (Yent) designed to detect all known serotypes/biotypes of Y. enterocolitica. Empirical testing and in silico sequence analysis indicate a potential for cross-reactivity with Y. kristensenii and Y. frederiksenii when present at high levels $\mathrm { ( > 1 0 ^ { 8 } C F U / m L ) }$ ). These two species are in the Y. enterocolitica group and are difficult to differentiate from Y. enterocolitica by culture methods; both are suspected human pathogens.

# Diarrheagenic E. coli

The FilmArray GI Panel contains multiple assays designed to detect genetic determinants associated with classic diarrheagenic $E .$ . coli/Shigella pathotypes. Horizontal transfer of these genes between organisms has been documented; therefore, Detected results for multiple diarrheagenic $E$ . coli/Shigella may be due to the presence of multiple pathotypes or a single strain containing the characteristic determinants of multiple pathotypes. An example of this is the $2 0 1 1 E$ . coli O104:H4 outbreak strain that contains determinants of both Shiga-like toxin-producing $E$ . coli (STEC) and Enteroaggregative $E$ . coli (EAEC).

Enteroaggregative E. coli (EAEC): The FilmArray GI Panel contains a single multiplexed assay (EAEC) for the identification of two gene targets typically associated with enteroaggregative E. coli; the aggR regulatory gene and the putative outer membrane protein, aatA, both located on the partially-conserved pAA plasmid. pAA is not present in all strains phenotypically identified as EAEC, and not all pAA plasmids carry aggR and aatA genes; therefore the FilmArray GI Panel will not detect all members of this diverse pathotype, but is likely to detect most pathogenic strains (including $E$ . coli O104:H4, which was responsible for recent outbreaks in Europe).

Enterotoxigenic (ETEC) heat-labile (lt) and heat-stable (st) Enterotoxins: The FilmArray GI Panel contains three assays (ETEC 1, ETEC 2, and ETEC 3) for the detection of enterotoxins found in Enterotoxigenic E. coli (ETEC). The assays are designed for the detection of heat-labile (LT) enterotoxin $( l t A )$ and two heat-stable (ST) enterotoxin variants (st1a, also known as STp; and $s t I b$ , also known as STh). The reported results do not indicate which of these toxin(s) have been detected. A positive result for any combination of the three assays will give an Enterotoxigenic $E .$ . coli (ETEC) lt/st Detected test result. The variant LT-II toxin (structurally similar to LT) and the STB/ST2 toxin (structurally dissimilar to ST1) are not targeted by the ETEC assays and have not been established as important in human disease. Empirical testing and in silico sequence analysis indicates the potential for cross-reactivity with certain strains of Hafnia alvei, C. koseri, C. sedlakii, and Cedecea davisae.

Enteropathogenic E. coli (EPEC): The FilmArray GI Panel contains a single assay (Ec eae) for the detection of eae, the gene encoding the adhesin intimin. Both typical and atypical EPEC will be detected, but not differentiated. The LEE pathogenicity island, which includes the eae gene, is also found in some Shiga-like toxin producing $E .$ . coli (STEC; O157 and non-O157 strains). Therefore, the results of the eae assay (positive or negative) are only reported when STEC is not detected. When STEC is detected, Enteropathogenic $E$ . coli (EPEC) will be reported as N/A (Not Applicable), regardless of the EPEC assay result (see table below). Consequently, the FilmArray GI Panel cannot distinguish between STEC containing the eae gene and a co-infection of EPEC and STEC.

Shiga-like toxin-producing $E .$ . coli (STEC) Shiga-like toxin genes 1 and 2 $( s t x I / s t x 2 )$ : The FilmArray GI Panel contains two assays (STEC 1 and STEC 2) for the detection of Shigalike toxin 1 $( s t x I )$ and Shiga-like toxin 2 (stx2) sequences. The reported results do not indicate which of these toxin(s) have been detected. A positive result for either or both of these assays will give a Shiga-like toxin-producing $E$ . coli (STEC) $s t x I / s t x 2$ detected test result (see table below). Shiga toxin (stx; identical to stx1 of STEC) is found in Shigella dysenteriae; therefore, a FilmArray GI Panel report with positive test results for Shiga-like toxin-producing $E$ . coli (STEC) stx1/stx2 and Shigella/Enteroinvasive $E$ . coli (EIEC) in the same sample may indicate the presence of S. dysenteriae.

E. coli O157: To aid in the identification of STEC of the O157 serotype, the FilmArray GI Panel contains a single assay (Ec O157) to detect a gene target that is specific to this serotype. Strains of $E .$ . coli O157 that do not carry the Shiga-like toxin genes have also been identified. However, as the pathogenicity of these non-STEC strains remains undefined, the E. coli O157 assay result is not reported unless a Shiga-like toxin gene is also detected (STEC detected). Detection of STEC stx1/stx2 and the $E .$ . coli O157 target results in a reporting of $E$ . coli O157 as a qualifier to the positive STEC result. If STEC stx1/stx2 is Not Detected, the result for $E$ . coli O157 is indicated as N/A (Not Applicable). The FilmArray GI Panel cannot distinguish between infections with a single toxigenic STEC O157 or rare co-infections of STEC (non-O157) with a stx1/stx2-negative E. coli O157 (see table below).

Table - Possible Assay Results and Corresponding Test Results for Enteropathogenic E.coli (EPEC) and Shiga-like toxin-producing E. coli (STEC) stx1/stx   

<table><tr><td rowspan=1 colspan=1>FilmArray GI Results</td><td rowspan=1 colspan=1>EPEC (Eceae)Assay</td><td rowspan=1 colspan=1>STEC stx1/2(STEC 1/STEC 2)Assays</td><td rowspan=1 colspan=1>E. coli0157(Ec O157)Assay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC): NotDetectedShiga-like toxin-producing E. coli(STEC) stx1/stx2: Not DetectedE. coli O157: N/A</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>AnyResult</td><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) notdetected and Shiga-like toxin-producing E. coli (STEC) stx1/stx2 notdetectedE. coli O157 result is not applicablewhen STEC is not detected</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC):DetectedShiga-like toxin-producing E. coli(STEC) stx1/stx2: Not DetectedE. coli O157: N/A</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>AnyResult</td><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)detectedShiga-like toxin-producing E. coli(STEC) stx1/stx2 not detectedE. coli O157 result is not applicablewhenSTEC is not detected</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC): N/AShiga-like toxin-producing E. coli(STEC) stx1/stx2: DetectedE. coli O157: Not Detected</td><td rowspan=1 colspan=1>AnyResult</td><td rowspan=1 colspan=1>Positive a</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>EPEC result is not applicable(detection cannot be differentiatedfrom eae-containing STEC)Shiga-like toxin-producing E. coli(STEC) stx1/stx2 detected, 0157serotype not detected</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC): N/AShiga-like toxin-producing E. coli(STEC) stx1/stx2: DetectedE. coli O157: Detected</td><td rowspan=1 colspan=1>AnyResult</td><td rowspan=1 colspan=1>Positive a</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>EPEC result is not applicable(detection cannot be differentiatedfrom eae-containing STEC)Shiga-like toxin-producing E. coli(STEC) stx1/stx2 detected, 0157serotype detected b</td></tr></table>

a Positive results for the STEC assay(s) and the Shigella/Enteroinvasive E. coli (EIEC) assay may indicate the presence of Shigella dysenteriae. b O157 determinant may be from the STEC or may be due to the rare possibility of a shiga-like toxin-negative $E .$ . coli O157 being in the same specimen with a non-O157 STEC.

Shigella/Enteroinvasive $E$ . coli (EIEC): The FilmArray GI Panel contains a single assay (Shig) for the detection of ipaH, a gene specifically found in all Shigella species as well as Enteroinvasive $E .$ . coli (EIEC). It is not possible to differentiate Shigella from EIEC using this method, and detection of ipaH will result in a Shigella/Enteroinvasive E. coli (EIEC) Detected test result. Shiga toxin $( s t x ;$ identical to stx1 of STEC) is found in Shigella dysenteriae, therefore a FilmArray GI Panel report with positive test results for Shiga-like toxin-producing $E$ . coli (STEC) stx1/stx2 with Shigella/Enteroinvasive E. coli (EIEC) in the same sample may indicate the presence of S. dysenteriae.

# Parasites

Cryptosporidium: The FilmArray GI Panel contains two assays (Crypt 1 and Crypt 2) for detection of Cryptosporidium species. Empirical testing and in silico sequence analysis support detection of approximately 23 different Cryptosporidium, including the most common species of human clinical relevance (i.e., C. hominis and C. parvum), as well as several less common species (e.g., C. meleagridis, C. felis, C. canis, C. cuniculus, C. muris, and C. suis). The assays do not differentiate between species and the very rare species $C$ . bovis, C. ryanae and C. xiaoi may not be detected. A positive result for either or both assays will give a Cryptosporidium Detected test result.

Cyclospora cayetanensis: The FilmArray GI Panel contains a single assay (Ccayet) for the detection of $C .$ . cayetanensis, the only Cyclospora species implicated in human disease.

Entamoeba histolytica: The FilmArray GI Panel contains a single assay (Ehist) for the detection of $E$ . histolytica, the only Entamoeba species implicated in gastroenteritis. This assay may cross-react with the closely related $E$ . dispar when present at higher levels (approximately $1 0 ^ { 5 }$ oocysts $/ \mathrm { m L }$ or greater).

Giardia lamblia: The FilmArray GI Panel contains a single assay (Glam) designed to detect $G$ . lamblia (aka $G .$ . intestinalis, G. duodenalis), the only Giardia species infectious to humans. A very low frequency of cross-reactivity with commensal microorganisms (i.e., Bifidobacterium and Ruminococcus) was observed in the clinical evaluation.

Viruses

Adenovirus F40/41: The FilmArray GI Panel contains a single multiplexed assay (AdenoF) for the specific detection of both Adenovirus F40 and F41 (i.e., will not cross-react with respiratory non-40/41 Adenovirus species when shed in the stool). The reported results do not indicate which serotype (40 or 41) has been detected. The assay will not detect other adenovirus species, such as species B, C, and E, which are associated with respiratory infections.

Astrovirus: The FilmArray GI Panel contains a single assay (Astro) designed to detect eight subtypes (HAstV1-8) of human Astrovirus. The assay is not predicted to detect newlyidentified astroviruses of the MLB and VA clades.

Norovirus GI/GII: The FilmArray GI Panel contains two assays (Noro 1 and Noro 2) that together target the Norovirus genogroups most commonly associated with human infections (GI and GII). Neither assay will detect genogroup GIV, non-human genogroups, or closely related Caliciviruses such as Sapovirus. The reported results do not indicate which genogroup(s) (GI and/or GII) have been detected. A positive result for either or both assays will produce test result of Norovirus GI/GII Detected.

Rotavirus A: The FilmArray GI Panel contains two separate Rotavirus A assays (RotaA 1 and RotaA 2) to be inclusive of all strains of Rotavirus A. In silico sequence analysis indicates that these assays will not cross-react with Rotavirus B and C, which are less

common in human disease, or Rotavirus D, E, and F, which have not been found in humans. Empirical testing has demonstrated that these assays will detect recombinant viruses included in Rotavirus vaccines. A FilmArray GI Panel test result of Rotavirus A Detected is reported if either or both assays are positive.

Sapovirus (Genogroups I, II, IV, and V): The FilmArray GI Panel contains a single assay (Sapo) designed to detect, but not differentiate, Sapovirus genogroups identified in human infections (I, II, IV and V). Genogroup III, a porcine pathogen will not be detected.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Luminex xTAG $^ \mathrm { \textregistered }$ Gastrointestinal Pathogen Panel (GPP)

2. Predicate 510(k) number(s): K121454

3. Comparison with predicate:

The Luminex xTAG $^ \mathrm { \textregistered }$ Gastrointestinal Pathogen Panel (GPP) is a qualitative, multiplexed in vitro diagnostic assay intended to simultaneously detect and identify microorganism nucleic acids from human stool samples. Testing is performed on pre-treated human stool samples. The tables below outline the similarities and the differences between the two systems.

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">FilmArray GI Panel</td><td colspan="1" rowspan="1">Luminex xTAG® GastrointestinalPathogen Panel (GPP)</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Campylobacter, toxigenic Clostridiumdifficile toxA/B, Salmonella, NorovirusGI/GII, Rotavirus A, Cryptosporidium,Giardia lamblia, E. coli O157, Shiga toxin-producing E. coli (STEC), EnterotoxigenicE. coli (ETEC), and Shigella/EnteroinvasiveE. coli (EIEC).</td><td colspan="1" rowspan="1">SameSee below for differences</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td><td colspan="1" rowspan="1">SameSee below for differences</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">FilmArray GI Panel</td><td colspan="1" rowspan="1">Luminex xTAG® GastrointestinalPathogen Panel (GPP)</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Human stool sample collected in CaryBlair transport media.</td><td colspan="1" rowspan="1">Pre-treated human stool sample.</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Detects the following Campylobacterspecies: C. jejuni/C. coli/C. upsaliensis.Also detects additional Cryptosporidiumspecies, Plesiomonas shigelloides, Vibrio(V. parahaemolyticus/V. vulnificus/V.cholerae), V. cholerae, Yersiniaenterocolitica, Adenovirus F40/41,Astrovirus, Sapovirus (Genogroups I, II,IV, and V), Cyclospora cayetanensis,Entamoeba histolytica, EnteropathogenicE. coli (EPEC), Enteroinvasive E. coli(EIEC), and Enteroaggregative E. coli(EAEC).</td><td colspan="1" rowspan="1">Detects the following Campylobacterspecies: C. jejuni, C. coli, and C. lari. Onlydetects the following Cryptosporidiumspecies: C. parvum and C. hominis.</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followed byhigh resolution melting analysis to confirmidentity of amplified product.</td><td colspan="1" rowspan="1">Multiplex RT-PCR and multiplex TSPEfollowed by Fluorescence-activated sortingof labeled beads coupled to streptavidin-conjugated biotinylated products.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray Instrument</td><td colspan="1" rowspan="1">Nucleic Acid Purification SystemPCR ThermocyclerLuminex® 100/200TM or MAGPIXinstruments</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Less than 1 hour</td><td colspan="1" rowspan="1">Approximately 5 hours</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Room temperature</td><td colspan="1" rowspan="1">Reagents stored at 4°C and -20°C.</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample Processing is automated in theFilmArray Instrument.</td><td colspan="1" rowspan="1">Up front sample processing is required toextract nucleic acid.</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Semi-automated test interpretation. Usermust review all "no call" results todetermine cause and retesting strategy.</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td colspan="1" rowspan="1">Internal control added to each sample.External control processed with each batchof samples.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Draft Guidance for Industry and Food and Drug Administration Staff - Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices, (November 9, 2012) · Class II Special Controls Guidance Document: Norovirus Serological Reagents. Document issued on: March 9, 2012. Document number 1767.

Draft Guidance – Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile. Document issued on November 29, 2010. Document number 1715. Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, FDA Guidance Document (March 13, 2007) User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute (CLSI) Second Edition, EP12-A2 (January 2008) Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005) · EN ISO 14971:2012, ‘Medical devices – Application of risk management to medical devices’

# Software Specific

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005)   
Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999)   
General Principle of Software Validation; Final Guidance for Industry and FDA Staff (January 11, 2002)   
ISO 62304:2006, ‘Medical device software – Software life-cycle processes’.

# Labeling

Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use, FDA Guidance Document (November 30, 2004) 21 CFR 809.10, Labeling for in vitro diagnostic products ISO 15223-1:2012, ‘Medical Devices – Symbols to be used with medical device labels, labeling and information to be supplied – Part 1: General requirements’. C EN ISO 18113-1:2011, ‘In vitro diagnostic medical devices – Information supplied by the manufacturer (labeling) – Part 1: Terms, definition and general requirements’. EN ISO 18113-2:2011, ‘In vitro diagnostic medical devices – Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use’.

# L. Test Principle:

The FilmArray GI pouch is a closed system disposable that houses all the chemistry required to isolate, amplify and detect nucleic acid from multiple gastrointestinal pathogens within a single stool specimen collected in Cary Blair. The rigid plastic component (fitment) of the FilmArray GI pouch contains reagents in freeze-dried form. The flexible plastic portion of the pouch is divided into discrete segments (blisters) where the required chemical processes are carried out. The user of the FilmArray GI Panel loads the sample into the FilmArray GI pouch, places the pouch into the FilmArray Instrument, and starts the run. All other operations are automated.

The following is an overview of the operations and processes that occur during a FilmArray run:

Nucleic Acid Purification - Nucleic acid purification occurs in the first three blisters of the pouch. The sample is lysed by a combination of chemical and mechanical (bead beating)

mechanisms and the liberated nucleic acid is captured, washed and eluted using magnetic bead technology. These steps require about ten minutes, and the bead-beater apparatus can be heard as a high-pitched whine during the first few minutes of operation.

Reverse Transcription and 1st Stage Multiplex PCR - Since the GI Panel includes RNA viruses, a reverse transcription (RT) step is performed to convert the viral RNA into cDNA prior to amplification. The purified nucleic acid solution is combined with a preheated master mix to initiate the RT step and subsequent thermocycling for multiplex PCR. The effect of 1st stage PCR is to enrich for the target nucleic acids present in the sample.

2nd Stage PCR - The products of 1st stage PCR are diluted and mixed with fresh PCR reagents containing an intercalating fluorescent DNA dye (LCGreen $^ \mathrm { \textregistered }$ Plus, BioFire Diagnostics). This solution is distributed over the 2nd stage PCR array. The individual wells of the array contain primers for different assays (each present in triplicate) that target specific nucleic acid sequences from each of the pathogens detected, as well as control template material. These primers are ‘nested’ or internal to the specific products of the 1st stage multiplex reaction, which enhances both the sensitivity and specificity of the reactions.

DNA Melting Analysis – After 2nd stage PCR, the temperature is slowly increased and fluorescence in each well of the array is monitored and analyzed to generate a melt curve. The temperature at which a specific PCR product melts (melting temperature or Tm) is consistent and predictable and the FilmArray Software automatically evaluates the data from replicate wells for each assay to report results

The FilmArray Software controls the operation of the instrument, collects and analyzes data, and automatically generates a test report at the end of the run.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility:

A multicenter reproducibility study was performed to determine between-site and overall reproducibility of the FilmArray GI Panel. Reproducibility testing occurred at three test sites using a panel of contrived stool samples, each spiked with various combinations of four different GI Panel analytes. Each analyte was evaluated at three different concentrations (Negative, Low Positive and Moderate Positive).

The study incorporated a range of potential variation introduced by 13 different operators, 4 different pouch lots, and 16 different FilmArray Instruments. Samples were stored refrigerated $( 4 ^ { \circ } \mathrm { C } )$ or frozen $( { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C } )$ prior to testing. Frozen samples were tested on five different days at three testing sites for 90 data points per sample and refrigerated samples were tested on four different days at three testing sites for 108 data points per sample. A summary of results (percent $( \% )$ agreement with the expected result) for each analyte (by site and overall) and the reproducibility of Tm for each positive assay is provided in the tables below.

Table - Reproducibility of the FilmArray GI Panel Test Results   

<table><tr><td colspan="1" rowspan="2">Organism Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">% Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95% ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="3">CampylobacterjejuniATCC BAA-1234</td><td colspan="1" rowspan="1">ModeratePositive3xLoD1.2x105 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD4x104 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">ClostridiumdifficileaATCC 9689</td><td colspan="1" rowspan="1">ModeratePositive3xLoD1.2x106 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD4x105 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli(EPEC)E2348/69(STEC Center,MSU)</td><td colspan="1" rowspan="1">ModeratePositive3xLoD3x10³3 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD1x10³3 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192b100%</td><td colspan="1" rowspan="1">192/192b100%</td><td colspan="1" rowspan="1">192/192b100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">SalmonellaentericaaSarC1 (SGSC)</td><td colspan="1" rowspan="1">ModeratePositive3xLoD1.5x104 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD5x10³ CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli(STEC) 0157ATCC 43895</td><td colspan="1" rowspan="1">ModeratePositive3xLoD3x104 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD1x104 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="2">Organism Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="2" rowspan="1">% Agreemen</td><td colspan="2" rowspan="1">% Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95% ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">Shigella sonneiATCC 29930</td><td colspan="1" rowspan="1">ModeratePositive3xLoD3x10²2 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD1x10²2 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">VibrioparahaemolyticusaATCC 17802</td><td colspan="1" rowspan="1">ModeratePositive3xLoD2.4x105 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD8x104 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="3">CryptosporidiumparvumWaterborneP102C</td><td colspan="1" rowspan="1">ModeratePositive3xLoD1.5x104oocysts/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD5x103 oocysts/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">Giardiaintestinalisa(syn. GiardialambliaATCC 30957</td><td colspan="1" rowspan="1">ModeratePositive3x LoD150 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD50 cells/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/3683.3%</td><td colspan="1" rowspan="1">30/3683.3%</td><td colspan="1" rowspan="1">31/3686.1%</td><td colspan="1" rowspan="1">91/10884.3%(77.0 - 91.0%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%(96.6 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F41ATTC VR-930</td><td colspan="1" rowspan="1">ModeratePositive3x LoD300 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="2">Organism Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">% Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95% ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Low Positive1xLoD100 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">Astrovirus(Type 8)NCPV 1003071v</td><td colspan="1" rowspan="1">ModeratePositive3xLoD150 FFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD50 FFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr><tr><td colspan="1" rowspan="3">Norovirus GIClinical Specimen</td><td colspan="1" rowspan="1">ModeratePositive3xLoD3x104 copies/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive1xLoD1x104 copies/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">28/3093.3%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">87/9096.7%(96.0 - 100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">192/192100%</td><td colspan="1" rowspan="1">576/576100%(99.4 - 100%)</td></tr></table>

a Reproducible, but suboptimal $( < 9 5 \% )$ detection was observed at one or both concentrations in frozen contrived samples. Data presented are from contrived samples stored at ${ \sim } 4 ^ { \circ } \mathrm { C }$ for up to 4 days prior to testing. b Includes $\mathrm { N } / \mathrm { A }$ results for 60 samples (180 for all sites) spiked with STEC O157. When an STEC is detected, N/A is reported for the EPEC test result, regardless of the status of the EPEC assay.

The reproducibility of Tm for each positive assay was also evaluated and a summary is provided in the following TABLE:

Table - Reproducibility of Tm for Positive FilmArray GI Panel Assays   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=2>Assay</td><td rowspan=2 colspan=2>Test Level</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=5>Tm Reproducibility</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDevTm</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>(Max -Min)</td></tr><tr><td rowspan=1 colspan=11>Bacteria and (Including Diarrheagenic E. coli)</td></tr><tr><td rowspan=8 colspan=1>CampylobacterjejuniATCC BAA-1234</td><td rowspan=8 colspan=2>Campy 1</td><td rowspan=4 colspan=2>Moderate Positive3xLoD1.2x105 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.38</td><td rowspan=1 colspan=1>± 0.27</td><td rowspan=1 colspan=1>77.86</td><td rowspan=1 colspan=1>79.01</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.28</td><td rowspan=1 colspan=1>± 0.21</td><td rowspan=1 colspan=1>77.87</td><td rowspan=1 colspan=1>78.59</td><td rowspan=1 colspan=1>0.72</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.04</td><td rowspan=1 colspan=1>± 0.29</td><td rowspan=1 colspan=1>77.60</td><td rowspan=1 colspan=1>78.59</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.23</td><td rowspan=1 colspan=1>± 0.30</td><td rowspan=1 colspan=1>77.60</td><td rowspan=1 colspan=1>79.01</td><td rowspan=1 colspan=1>1.41</td></tr><tr><td rowspan=4 colspan=2>Low Positive1xLoD4x104 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.60</td><td rowspan=1 colspan=1>± 0.33</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>79.47</td><td rowspan=1 colspan=1>1.74</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.65</td><td rowspan=1 colspan=1>± 0.19</td><td rowspan=1 colspan=1>78.28</td><td rowspan=1 colspan=1>79.01</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.21</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>78.72</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.48</td><td rowspan=1 colspan=1>± 0.33</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>79.47</td><td rowspan=1 colspan=1>1.74</td></tr><tr><td rowspan=16 colspan=1>ClostridiumdifficileAAC 9689</td><td rowspan=16 colspan=1>Cdiffr</td><td rowspan=8 colspan=1>Tm 1</td><td rowspan=4 colspan=2>Moderate Positive3xLoD1.2x106 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>76.01</td><td rowspan=1 colspan=1>± 0.34</td><td rowspan=1 colspan=1>75.30</td><td rowspan=1 colspan=1>76.99</td><td rowspan=1 colspan=1>1.69</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>75.79</td><td rowspan=1 colspan=1>±0.40</td><td rowspan=1 colspan=1>74.71</td><td rowspan=1 colspan=1>76.59</td><td rowspan=1 colspan=1>1.88</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.60</td><td rowspan=1 colspan=1>± 0.34</td><td rowspan=1 colspan=1>75.02</td><td rowspan=1 colspan=1>77.09</td><td rowspan=1 colspan=1>2.07</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>75.80</td><td rowspan=1 colspan=1>± 0.39</td><td rowspan=1 colspan=1>74.71</td><td rowspan=1 colspan=1>77.09</td><td rowspan=1 colspan=1>2.38</td></tr><tr><td rowspan=4 colspan=2>Low Positive1xLoD4x105 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>76.18</td><td rowspan=1 colspan=1>± 0.43</td><td rowspan=1 colspan=1>75.45</td><td rowspan=1 colspan=1>77.15</td><td rowspan=1 colspan=1>1.70</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>75.94</td><td rowspan=1 colspan=1>±0.43</td><td rowspan=1 colspan=1>75.09</td><td rowspan=1 colspan=1>76.74</td><td rowspan=1 colspan=1>1.65</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.73</td><td rowspan=1 colspan=1>± 0.28</td><td rowspan=1 colspan=1>75.29</td><td rowspan=1 colspan=1>76.45</td><td rowspan=1 colspan=1>1.16</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>75.95</td><td rowspan=1 colspan=1>± 0.43</td><td rowspan=1 colspan=1>75.09</td><td rowspan=1 colspan=1>77.15</td><td rowspan=1 colspan=1>2.06</td></tr><tr><td rowspan=8 colspan=1>Tm 2</td><td rowspan=4 colspan=2>Moderate Positive3xLoD1.2x106 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.84</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>78.44</td><td rowspan=1 colspan=1>79.56</td><td rowspan=1 colspan=1>1.12</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.61</td><td rowspan=1 colspan=1>± 0.30</td><td rowspan=1 colspan=1>77.86</td><td rowspan=1 colspan=1>79.17</td><td rowspan=1 colspan=1>1.31</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.40</td><td rowspan=1 colspan=1>± 0.22</td><td rowspan=1 colspan=1>78.01</td><td rowspan=1 colspan=1>79.02</td><td rowspan=1 colspan=1>1.01</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.62</td><td rowspan=1 colspan=1>± 0.32</td><td rowspan=1 colspan=1>77.86</td><td rowspan=1 colspan=1>79.56</td><td rowspan=1 colspan=1>1.70</td></tr><tr><td rowspan=4 colspan=2>Low Positive1xLoD4x105 cells/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.94</td><td rowspan=1 colspan=1>±0.31</td><td rowspan=1 colspan=1>78.45</td><td rowspan=1 colspan=1>79.61</td><td rowspan=1 colspan=1>1.16</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.67</td><td rowspan=1 colspan=1>± 0.30</td><td rowspan=1 colspan=1>78.02</td><td rowspan=1 colspan=1>79.17</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.48</td><td rowspan=1 colspan=1>±0.24</td><td rowspan=1 colspan=1>78.02</td><td rowspan=1 colspan=1>79.02</td><td rowspan=1 colspan=1>1.00</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.70</td><td rowspan=1 colspan=1>± 0.34</td><td rowspan=1 colspan=1>78.02</td><td rowspan=1 colspan=1>79.61</td><td rowspan=1 colspan=1>1.59</td></tr><tr><td rowspan=8 colspan=1>Escherichia coli(EPEC)E2348/69(STEC Center,MSU)</td><td rowspan=8 colspan=2>Ec eae</td><td rowspan=4 colspan=2>Moderate Positive3xLoD3x10³3 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.53</td><td rowspan=1 colspan=1>± 0.24</td><td rowspan=1 colspan=1>80.16</td><td rowspan=1 colspan=1>81.04</td><td rowspan=1 colspan=1>0.88</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.39</td><td rowspan=1 colspan=1>± 0.20</td><td rowspan=1 colspan=1>79.86</td><td rowspan=1 colspan=1>80.74</td><td rowspan=1 colspan=1>0.88</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.38</td><td rowspan=1 colspan=1>±0.17</td><td rowspan=1 colspan=1>80.01</td><td rowspan=1 colspan=1>80.61</td><td rowspan=1 colspan=1>0.60</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.43</td><td rowspan=1 colspan=1>± 0.22</td><td rowspan=1 colspan=1>79.86</td><td rowspan=1 colspan=1>81.04</td><td rowspan=1 colspan=1>1.18</td></tr><tr><td rowspan=4 colspan=2>Low Positive1xLoD1x10³3 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.59</td><td rowspan=1 colspan=1>± 0.24</td><td rowspan=1 colspan=1>80.15</td><td rowspan=1 colspan=1>81.18</td><td rowspan=1 colspan=1>1.03</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.46</td><td rowspan=1 colspan=1>± 0.20</td><td rowspan=1 colspan=1>79.87</td><td rowspan=1 colspan=1>80.73</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.42</td><td rowspan=1 colspan=1>±0.14</td><td rowspan=1 colspan=1>80.15</td><td rowspan=1 colspan=1>80.72</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.49</td><td rowspan=1 colspan=1>± 0.21</td><td rowspan=1 colspan=1>79.87</td><td rowspan=1 colspan=1>81.18</td><td rowspan=1 colspan=1>1.31</td></tr><tr><td rowspan=3 colspan=1>SalmonellaentericaSarC1 (SGSC)</td><td rowspan=3 colspan=2>Salm</td><td rowspan=3 colspan=2>Moderate Positive3xLoD1.5x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.17</td><td rowspan=1 colspan=1>±0.20</td><td rowspan=1 colspan=1>81.86</td><td rowspan=1 colspan=1>82.59</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.88</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>81.30</td><td rowspan=1 colspan=1>82.32</td><td rowspan=1 colspan=1>1.02</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>81.78</td><td rowspan=1 colspan=1>± 0.25</td><td rowspan=1 colspan=1>81.44</td><td rowspan=1 colspan=1>82.17</td><td rowspan=1 colspan=1>0.73</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Assay</td><td rowspan=2 colspan=1>Test Level</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=5>Tm Reproducibility</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDevTm</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>(Max -Min)</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=5 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>81.95</td><td rowspan=1 colspan=1>± 0.29</td><td rowspan=1 colspan=1>81.30</td><td rowspan=1 colspan=1>82.59</td><td rowspan=1 colspan=1>1.29</td></tr><tr><td rowspan=4 colspan=1>Low Positive1xLoD5x10³3 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.21</td><td rowspan=1 colspan=1>±0.27</td><td rowspan=1 colspan=1>81.74</td><td rowspan=1 colspan=1>82.77</td><td rowspan=1 colspan=1>1.03</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.96</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>82.39</td><td rowspan=1 colspan=1>1.08</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>81.83</td><td rowspan=1 colspan=1>±0.25</td><td rowspan=1 colspan=1>81.45</td><td rowspan=1 colspan=1>82.31</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.00</td><td rowspan=1 colspan=1>±0.30</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>82.77</td><td rowspan=1 colspan=1>1.46</td></tr><tr><td rowspan=24 colspan=1>Escherichia coli(STEC) 0157ATCC 43895</td><td rowspan=8 colspan=1>0157</td><td rowspan=4 colspan=1>Moderate Positive3xLoD3x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.23</td><td rowspan=1 colspan=1>±0.22</td><td rowspan=1 colspan=1>82.58</td><td rowspan=1 colspan=1>83.77</td><td rowspan=1 colspan=1>1.19</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.20</td><td rowspan=1 colspan=1>± 0.19</td><td rowspan=1 colspan=1>82.85</td><td rowspan=1 colspan=1>83.60</td><td rowspan=1 colspan=1>0.75</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.96</td><td rowspan=1 colspan=1>±0.29</td><td rowspan=1 colspan=1>82.59</td><td rowspan=1 colspan=1>83.44</td><td rowspan=1 colspan=1>0.85</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.13</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>82.58</td><td rowspan=1 colspan=1>83.77</td><td rowspan=1 colspan=1>1.19</td></tr><tr><td rowspan=4 colspan=1>Low Positive1xLoD1x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.26</td><td rowspan=1 colspan=1>± 0.24</td><td rowspan=1 colspan=1>82.80</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>1.08</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.20</td><td rowspan=1 colspan=1>± 0.20</td><td rowspan=1 colspan=1>82.73</td><td rowspan=1 colspan=1>83.59</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.01</td><td rowspan=1 colspan=1>±0.29</td><td rowspan=1 colspan=1>82.46</td><td rowspan=1 colspan=1>83.60</td><td rowspan=1 colspan=1>1.14</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.16</td><td rowspan=1 colspan=1>±0.26</td><td rowspan=1 colspan=1>82.46</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>1.42</td></tr><tr><td rowspan=8 colspan=1>STEC 1</td><td rowspan=4 colspan=1>Moderate Positive3xLoD3x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.85</td><td rowspan=1 colspan=1>±0.25</td><td rowspan=1 colspan=1>82.16</td><td rowspan=1 colspan=1>83.48</td><td rowspan=1 colspan=1>1.32</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.80</td><td rowspan=1 colspan=1>±0.19</td><td rowspan=1 colspan=1>82.28</td><td rowspan=1 colspan=1>83.17</td><td rowspan=1 colspan=1>0.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.52</td><td rowspan=1 colspan=1>± 0.28</td><td rowspan=1 colspan=1>82.16</td><td rowspan=1 colspan=1>83.02</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.72</td><td rowspan=1 colspan=1>± 0.28</td><td rowspan=1 colspan=1>82.16</td><td rowspan=1 colspan=1>83.48</td><td rowspan=1 colspan=1>1.32</td></tr><tr><td rowspan=4 colspan=1>Low Positive1xLoD1x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.89</td><td rowspan=1 colspan=1>±0.24</td><td rowspan=1 colspan=1>82.44</td><td rowspan=1 colspan=1>83.31</td><td rowspan=1 colspan=1>0.87</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.78</td><td rowspan=1 colspan=1>±0.18</td><td rowspan=1 colspan=1>82.44</td><td rowspan=1 colspan=1>83.17</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.55</td><td rowspan=1 colspan=1>±0.28</td><td rowspan=1 colspan=1>82.03</td><td rowspan=1 colspan=1>83.17</td><td rowspan=1 colspan=1>1.14</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.74</td><td rowspan=1 colspan=1>± 0.27</td><td rowspan=1 colspan=1>82.03</td><td rowspan=1 colspan=1>83.31</td><td rowspan=1 colspan=1>1.28</td></tr><tr><td rowspan=8 colspan=1>STEC 2</td><td rowspan=4 colspan=1>Moderate Positive3xLoD3x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.99</td><td rowspan=1 colspan=1>±0.22</td><td rowspan=1 colspan=1>84.44</td><td rowspan=1 colspan=1>85.49</td><td rowspan=1 colspan=1>1.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.90</td><td rowspan=1 colspan=1>±0.19</td><td rowspan=1 colspan=1>84.43</td><td rowspan=1 colspan=1>85.31</td><td rowspan=1 colspan=1>0.88</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.68</td><td rowspan=1 colspan=1>±0.30</td><td rowspan=1 colspan=1>84.30</td><td rowspan=1 colspan=1>85.16</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>84.86</td><td rowspan=1 colspan=1>± 0.27</td><td rowspan=1 colspan=1>84.30</td><td rowspan=1 colspan=1>85.49</td><td rowspan=1 colspan=1>1.19</td></tr><tr><td rowspan=4 colspan=1>Low Positive1xLoD1x104 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.98</td><td rowspan=1 colspan=1>±0.22</td><td rowspan=1 colspan=1>84.58</td><td rowspan=1 colspan=1>85.45</td><td rowspan=1 colspan=1>0.87</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.92</td><td rowspan=1 colspan=1>± 0.19</td><td rowspan=1 colspan=1>84.45</td><td rowspan=1 colspan=1>85.30</td><td rowspan=1 colspan=1>0.85</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.72</td><td rowspan=1 colspan=1>±0.28</td><td rowspan=1 colspan=1>84.31</td><td rowspan=1 colspan=1>85.32</td><td rowspan=1 colspan=1>1.01</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>84.88</td><td rowspan=1 colspan=1>± 0.26</td><td rowspan=1 colspan=1>84.31</td><td rowspan=1 colspan=1>85.45</td><td rowspan=1 colspan=1>1.14</td></tr><tr><td rowspan=8 colspan=1>Shigella sonneiATCC 29930</td><td rowspan=8 colspan=1>Shig</td><td rowspan=4 colspan=1>Moderate Positive3xLoD3x10² CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.58</td><td rowspan=1 colspan=1>±0.25</td><td rowspan=1 colspan=1>86.01</td><td rowspan=1 colspan=1>87.05</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.38</td><td rowspan=1 colspan=1>±0.19</td><td rowspan=1 colspan=1>85.87</td><td rowspan=1 colspan=1>86.61</td><td rowspan=1 colspan=1>0.74</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.44</td><td rowspan=1 colspan=1>± 0.17</td><td rowspan=1 colspan=1>86.16</td><td rowspan=1 colspan=1>86.75</td><td rowspan=1 colspan=1>0.59</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>86.47</td><td rowspan=1 colspan=1>± 0.22</td><td rowspan=1 colspan=1>85.87</td><td rowspan=1 colspan=1>87.05</td><td rowspan=1 colspan=1>1.18</td></tr><tr><td rowspan=4 colspan=1>Low Positive1xLoD1x10²2 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.57</td><td rowspan=1 colspan=1>± 0.22</td><td rowspan=1 colspan=1>86.29</td><td rowspan=1 colspan=1>87.18</td><td rowspan=1 colspan=1>0.89</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.52</td><td rowspan=1 colspan=1>±0.24</td><td rowspan=1 colspan=1>86.02</td><td rowspan=1 colspan=1>87.01</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.26</td><td rowspan=1 colspan=1>±0.24</td><td rowspan=1 colspan=1>85.87</td><td rowspan=1 colspan=1>86.73</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>86.45</td><td rowspan=1 colspan=1>± 0.27</td><td rowspan=1 colspan=1>85.87</td><td rowspan=1 colspan=1>87.18</td><td rowspan=1 colspan=1>1.31</td></tr></table>

p. 19

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Assay</td><td colspan="1" rowspan="2">Test Level</td><td colspan="1" rowspan="2">Test Site</td><td colspan="5" rowspan="1">Tm Reproducibility</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">StDevTm</td><td colspan="1" rowspan="1">Min</td><td colspan="1" rowspan="1">Max</td><td colspan="1" rowspan="1">(Max -Min)</td></tr><tr><td colspan="1" rowspan="8">VibrioparahaemolyticusATCC 17802</td><td colspan="1" rowspan="8">Vibrio</td><td colspan="1" rowspan="4">Moderate Positive3xLoD2.4x105 cells/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">81.96</td><td colspan="1" rowspan="1">±0.23</td><td colspan="1" rowspan="1">81.59</td><td colspan="1" rowspan="1">82.42</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.69</td><td colspan="1" rowspan="1">± 0.24</td><td colspan="1" rowspan="1">81.02</td><td colspan="1" rowspan="1">82.03</td><td colspan="1" rowspan="1">1.01</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">81.57</td><td colspan="1" rowspan="1">±0.27</td><td colspan="1" rowspan="1">81.17</td><td colspan="1" rowspan="1">82.16</td><td colspan="1" rowspan="1">0.99</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">81.74</td><td colspan="1" rowspan="1">± 0.30</td><td colspan="1" rowspan="1">81.02</td><td colspan="1" rowspan="1">82.42</td><td colspan="1" rowspan="1">1.40</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD8x104 cells/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.03</td><td colspan="1" rowspan="1">± 0.17</td><td colspan="1" rowspan="1">81.73</td><td colspan="1" rowspan="1">82.42</td><td colspan="1" rowspan="1">0.69</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.74</td><td colspan="1" rowspan="1">± 0.23</td><td colspan="1" rowspan="1">81.29</td><td colspan="1" rowspan="1">82.17</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">81.60</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">81.30</td><td colspan="1" rowspan="1">82.02</td><td colspan="1" rowspan="1">0.72</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">81.79</td><td colspan="1" rowspan="1">± 0.28</td><td colspan="1" rowspan="1">81.29</td><td colspan="1" rowspan="1">82.42</td><td colspan="1" rowspan="1">1.13</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Protozoa</td></tr><tr><td colspan="1" rowspan="16">CryptosporidiumparvumWaterborneP102C</td><td colspan="1" rowspan="8">Crypt 1</td><td colspan="1" rowspan="4">Moderate Positive3xLoD1.55x1040ocysts/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.99</td><td colspan="1" rowspan="1">± 0.23</td><td colspan="1" rowspan="1">78.58</td><td colspan="1" rowspan="1">79.46</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.95</td><td colspan="1" rowspan="1">± 0.24</td><td colspan="1" rowspan="1">78.29</td><td colspan="1" rowspan="1">79.58</td><td colspan="1" rowspan="1">1.29</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">±0.15</td><td colspan="1" rowspan="1">78.57</td><td colspan="1" rowspan="1">79.16</td><td colspan="1" rowspan="1">0.59</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.92</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">78.29</td><td colspan="1" rowspan="1">79.58</td><td colspan="1" rowspan="1">1.29</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD5x10³ oocysts/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.00</td><td colspan="1" rowspan="1">± 0.26</td><td colspan="1" rowspan="1">78.59</td><td colspan="1" rowspan="1">79.61</td><td colspan="1" rowspan="1">1.02</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.94</td><td colspan="1" rowspan="1">± 0.21</td><td colspan="1" rowspan="1">78.29</td><td colspan="1" rowspan="1">79.31</td><td colspan="1" rowspan="1">1.02</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.88</td><td colspan="1" rowspan="1">± 0.18</td><td colspan="1" rowspan="1">78.43</td><td colspan="1" rowspan="1">79.17</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.95</td><td colspan="1" rowspan="1">± 0.23</td><td colspan="1" rowspan="1">78.29</td><td colspan="1" rowspan="1">79.61</td><td colspan="1" rowspan="1">1.32</td></tr><tr><td colspan="1" rowspan="8">Crypt 2</td><td colspan="1" rowspan="4">Moderate Positive3xLoD1.5x104oocysts/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">71.75</td><td colspan="1" rowspan="1">± 0.28</td><td colspan="1" rowspan="1">71.29</td><td colspan="1" rowspan="1">72.31</td><td colspan="1" rowspan="1">1.02</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">71.74</td><td colspan="1" rowspan="1">± 0.20</td><td colspan="1" rowspan="1">71.15</td><td colspan="1" rowspan="1">72.15</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">71.50</td><td colspan="1" rowspan="1">± 0.20</td><td colspan="1" rowspan="1">71.28</td><td colspan="1" rowspan="1">72.15</td><td colspan="1" rowspan="1">0.87</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">71.67</td><td colspan="1" rowspan="1">± 0.26</td><td colspan="1" rowspan="1">71.15</td><td colspan="1" rowspan="1">72.31</td><td colspan="1" rowspan="1">1.16</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD5x103 oocysts/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">71.81</td><td colspan="1" rowspan="1">± 0.35</td><td colspan="1" rowspan="1">71.29</td><td colspan="1" rowspan="1">72.43</td><td colspan="1" rowspan="1">1.14</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">71.81</td><td colspan="1" rowspan="1">± 0.16</td><td colspan="1" rowspan="1">71.43</td><td colspan="1" rowspan="1">72.16</td><td colspan="1" rowspan="1">0.73</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">71.59</td><td colspan="1" rowspan="1">± 0.21</td><td colspan="1" rowspan="1">71.28</td><td colspan="1" rowspan="1">72.14</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">71.74</td><td colspan="1" rowspan="1">± 0.27</td><td colspan="1" rowspan="1">71.28</td><td colspan="1" rowspan="1">72.43</td><td colspan="1" rowspan="1">1.15</td></tr><tr><td colspan="1" rowspan="8">Giardiaintestinalis(syn. G. lamblia)ATCC 30957</td><td colspan="1" rowspan="8">Glam</td><td colspan="1" rowspan="4">Moderate Positive3xLoD150 cells/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">91.52</td><td colspan="1" rowspan="1">± 0.24</td><td colspan="1" rowspan="1">91.04</td><td colspan="1" rowspan="1">92.08</td><td colspan="1" rowspan="1">1.04</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">91.19</td><td colspan="1" rowspan="1">±0.25</td><td colspan="1" rowspan="1">90.47</td><td colspan="1" rowspan="1">91.59</td><td colspan="1" rowspan="1">1.12</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">91.12</td><td colspan="1" rowspan="1">± 0.29</td><td colspan="1" rowspan="1">90.62</td><td colspan="1" rowspan="1">91.74</td><td colspan="1" rowspan="1">1.12</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">91.28</td><td colspan="1" rowspan="1">± 0.31</td><td colspan="1" rowspan="1">90.47</td><td colspan="1" rowspan="1">92.08</td><td colspan="1" rowspan="1">1.61</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD50 cells/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">91.57</td><td colspan="1" rowspan="1">± 0.21</td><td colspan="1" rowspan="1">91.17</td><td colspan="1" rowspan="1">91.91</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">91.24</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">90.75</td><td colspan="1" rowspan="1">91.62</td><td colspan="1" rowspan="1">0.87</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">91.10</td><td colspan="1" rowspan="1">±0.30</td><td colspan="1" rowspan="1">90.60</td><td colspan="1" rowspan="1">91.61</td><td colspan="1" rowspan="1">1.01</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">91.30</td><td colspan="1" rowspan="1">± 0.31</td><td colspan="1" rowspan="1">90.60</td><td colspan="1" rowspan="1">91.91</td><td colspan="1" rowspan="1">1.31</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="3">Adenovirus F41ATTC VR-930</td><td colspan="1" rowspan="3">AdenoF</td><td colspan="1" rowspan="3">Moderate Positive3xLoD300 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.71</td><td colspan="1" rowspan="1">± 0.23</td><td colspan="1" rowspan="1">86.01</td><td colspan="1" rowspan="1">87.35</td><td colspan="1" rowspan="1">1.34</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">86.61</td><td colspan="1" rowspan="1">± 0.18</td><td colspan="1" rowspan="1">86.28</td><td colspan="1" rowspan="1">87.03</td><td colspan="1" rowspan="1">0.75</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.36</td><td colspan="1" rowspan="1">± 0.31</td><td colspan="1" rowspan="1">85.87</td><td colspan="1" rowspan="1">86.87</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.56</td><td colspan="1" rowspan="1">± 0.28</td><td colspan="1" rowspan="1">85.87</td><td colspan="1" rowspan="1">87.35</td><td colspan="1" rowspan="1">1.48</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD100 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.85</td><td colspan="1" rowspan="1">± 0.27</td><td colspan="1" rowspan="1">86.37</td><td colspan="1" rowspan="1">87.48</td><td colspan="1" rowspan="1">1.11</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">86.70</td><td colspan="1" rowspan="1">± 0.20</td><td colspan="1" rowspan="1">86.30</td><td colspan="1" rowspan="1">87.16</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.47</td><td colspan="1" rowspan="1">± 0.29</td><td colspan="1" rowspan="1">86.02</td><td colspan="1" rowspan="1">87.03</td><td colspan="1" rowspan="1">1.01</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.67</td><td colspan="1" rowspan="1">± 0.30</td><td colspan="1" rowspan="1">86.02</td><td colspan="1" rowspan="1">87.48</td><td colspan="1" rowspan="1">1.46</td></tr><tr><td colspan="1" rowspan="8">Astrovirus(Type 8)NCPV 1003071v</td><td colspan="1" rowspan="8">Astro</td><td colspan="1" rowspan="4">Moderate Positive3xLoD150 FFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.62</td><td colspan="1" rowspan="1">± 0.25</td><td colspan="1" rowspan="1">85.17</td><td colspan="1" rowspan="1">86.06</td><td colspan="1" rowspan="1">0.89</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.48</td><td colspan="1" rowspan="1">± 0.18</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">85.88</td><td colspan="1" rowspan="1">0.87</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.51</td><td colspan="1" rowspan="1">± 0.21</td><td colspan="1" rowspan="1">85.02</td><td colspan="1" rowspan="1">85.90</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.54</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">86.06</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD50 FFU/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.67</td><td colspan="1" rowspan="1">± 0.26</td><td colspan="1" rowspan="1">85.17</td><td colspan="1" rowspan="1">86.19</td><td colspan="1" rowspan="1">1.02</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.54</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">86.01</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.55</td><td colspan="1" rowspan="1">± 0.16</td><td colspan="1" rowspan="1">85.29</td><td colspan="1" rowspan="1">85.89</td><td colspan="1" rowspan="1">0.60</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.59</td><td colspan="1" rowspan="1">± 0.22</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">86.19</td><td colspan="1" rowspan="1">1.18</td></tr><tr><td colspan="1" rowspan="8">Norovirus GIClinical Specimen</td><td colspan="1" rowspan="8">Noro 1</td><td colspan="1" rowspan="4">Moderate Positive3xLoD3x104 copies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.69</td><td colspan="1" rowspan="1">± 0.23</td><td colspan="1" rowspan="1">83.14</td><td colspan="1" rowspan="1">84.07</td><td colspan="1" rowspan="1">0.93</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.46</td><td colspan="1" rowspan="1">± 0.20</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">83.76</td><td colspan="1" rowspan="1">0.84</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.43</td><td colspan="1" rowspan="1">± 0.20</td><td colspan="1" rowspan="1">83.02</td><td colspan="1" rowspan="1">83.87</td><td colspan="1" rowspan="1">0.85</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">83.52</td><td colspan="1" rowspan="1">± 0.24</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">84.07</td><td colspan="1" rowspan="1">1.15</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xLoD1x104 copies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.62</td><td colspan="1" rowspan="1">±0.24</td><td colspan="1" rowspan="1">83.22</td><td colspan="1" rowspan="1">84.15</td><td colspan="1" rowspan="1">0.93</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.59</td><td colspan="1" rowspan="1">± 0.21</td><td colspan="1" rowspan="1">83.18</td><td colspan="1" rowspan="1">83.98</td><td colspan="1" rowspan="1">0.80</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.30</td><td colspan="1" rowspan="1">± 0.24</td><td colspan="1" rowspan="1">82.93</td><td colspan="1" rowspan="1">83.79</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">83.50</td><td colspan="1" rowspan="1">± 0.27</td><td colspan="1" rowspan="1">82.93</td><td colspan="1" rowspan="1">84.15</td><td colspan="1" rowspan="1">1.22</td></tr></table>

a A characteristic double melt profile is observed when both C. difficile toxin genes (tcdA and tcdB) are present in a sample and two different Tm values are reported (Tm1 and $\mathrm { T m } 2$ ).

$b$ . Linearity/assay reportable range: Not applicable, qualitative assay.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Controls:

The following internal controls are included in each FilmArray GI Panel pouch:

DNA Process Control: The DNA Process Control assay targets DNA from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and is hydrated and introduced into the test when the sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, 1st stage PCR, dilution, 2nd stage PCR, and DNA melting. A positive control result indicates that all steps carried out in the pouch were successful.

PCR2 Control: The PCR2 Control assay detects a DNA target that is dried into the wells of the array along with the corresponding primers. A positive result indicates that the 2nd stage PCR was successful.

Both internal control assays must be positive for the test run to pass. When either control fails, the Controls field of the test report will display "Failed" and all results will be listed as Invalid. If the controls fail, the user is instructed to repeat the test using a new pouch. Of the nine pouch control failures observed in the prospective clinical study, seven were attributed to the RNA process control and two were due to the PCR2 control. There were no instances of both RNA process and PCR2 control failure in the same pouch.

Recommended External Controls:

External controls are not provided with the FilmArray GI Panel, but are recommended in the package insert. Cary Blair media can be used as an external negative control. Previously characterized positive stool samples or negative samples spiked with well characterized organisms can be used as external positive controls. External controls should be used in accordance with the appropriate accrediting organization requirements, as applicable.

External Controls used in clinical studies:

Daily external control mixes (ECMs) and weekly swab testing for contamination monitoring was performed during the clinical evaluation of the FilmArray GI Panel. External controls used in the clinical study consisted of four different organism mixes and a negative control provided to the clinical sites as frozen single use aliquots. (Three control mixes consisted of Cary Blair media containing whole organisms, one contained nucleic acid template (for organisms that could not be obtained in sufficient quantities), and one contained Cary Blair media alone. A valid ECM was required for each day of testing. All FilmArray GI Assay targets were represented by the four external control mixes. Testing of external controls was rotated, with one of the five controls tested each day. No external contamination was found when following assay instructions (i.e., prior to processing samples, both the work area and the FilmArray Pouch Loading Station were thoroughly cleaned using a suitable cleaner such as freshly prepared $10 \%$ bleach or a similar disinfectant. To avoid residue build-up and potential PCR inhibition, wipe disinfected surfaces with water.)

A total of 234 control mix runs were attempted, all of which completed. Of these, 225 $( 9 6 . 2 \%$ ; 225/234) were successful while nine $( 3 . 8 \% )$ did not return the correct organism results either due to the detection of an extra analyte $\scriptstyle ( \mathtt { n } = 3 )$ ) and/or the failure to detect one or more spiked analytes $( \mathrm { n } { = } 6 )$ ).

# PCR Comparator Controls:

An inhibition control was performed as a separate assay (primers $^ +$ template) on the nucleic acid extract of each specimen prior to running any other assay. There were no inhibition control failures.

# Specimen Stability:

To determine whether FilmArray GI Panel test results are obtained from stool samples in Cary-Blair stored under the time and temperature conditions recommended for use with transport media (e.g. up to 96 hours (4 days) at room temperature or refrigerated (4 days), 10 different stool-Cary Blair specimens were spiked with representative panel organisms. Samples were prepared by spiking organism mixes at concentrations approximately 3-fold higher $( 3 \times )$ than the established LoD for each individual organism into 10 unique stool samples in Cary Blair collected from asymptomatic donors. Each mix contained four different representative panel organisms for a total of 12 organisms.

‘Fresh’ or non-stored sample test results (Day 0) were established by testing samples with the FilmArray GI Panel within an hour of when the samples were prepared. The remaining sample was aliquoted and stored at the desired storage temperatures (room temperature or under refrigeration). At designated time-points (i.e., days 1-4), an aliquot from each sample was tested with the FilmArray GI Panel.

Transport and storage conditions were considered acceptable for use with the FilmArray GI Panel if accurate test results (equivalent to the non-stored samples (Day 0)) were obtained for at least nine of the ten aliquots evaluated at that condition. Negative results were compiled from specimens that had not been spiked with a particular organism/analyte, such that for every storage temperature/time-point, 20-30 negative FilmArray test results were expected. All analytes with unexpected negative test results were investigated and resolved by testing two additional aliquots of the same sample at the same condition.

Table - Test Results for Contrived Samples Stored at Room Temperature for 0-4 Days   

<table><tr><td rowspan=2 colspan=1>Test Result</td><td rowspan=1 colspan=2>Day 0</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Day 2</td><td rowspan=1 colspan=2>Day 3</td><td rowspan=1 colspan=2>Day 4</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29/30a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29/30a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29/30a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/41</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr></table>

For refrigerated samples, the expected results (at least 9/10 positive) were observed for each spiked analyte of Clostridium difficile on Day 4 (8/10 positive). All analytes with unexpected negative test results were investigated and resolved by testing two additional aliquots of the same sample at the same condition.

Table - Test Results for Contrived Samples Stored Refrigerated for 0-4 Days   

<table><tr><td rowspan=2 colspan=1>Test Result</td><td rowspan=1 colspan=2>Day 0</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Day 2</td><td rowspan=1 colspan=2>Day 3</td><td rowspan=1 colspan=2>Day 4</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Aeromonas</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>V. cholerae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/41</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td></tr></table>

The results of this study support the recommendations for sample handling and storage. Stool specimens in Cary Blair can be stored for up to four days at room temperature or in the refrigerator without affecting the accuracy of FilmArray GI Panel test results.

A total of 325 runs were initiated throughout the course of this study and 318 completed with valid results $3 1 8 / 3 2 5 = 9 7 . 8 \%$ . A total of 5 runs (5/325; $1 . 5 \%$ were invalid due to a control failure, one run did not complete due to a software error $( 1 / 3 2 5 ; 0 . 3 \% )$ and another one run $( 1 / 3 2 5 ; 0 . 3 \% )$ did not complete due to an instrument error.

# d. Detection limit:

A study was performed to determine the analytical sensitivity, or limit of detection (LoD), of the FilmArray GI Panel for each test result included in the panel. LoD (or $\mathrm { L o D } _ { 9 5 }$ ) is defined as the lowest concentration of organism that can be consistently detected $( \geq 9 5 \%$ of samples test positive) in the defined sample type (stool in Cary Blair transport medium). The LoD for each organism was estimated with limiting dilutions as single-spiked and multi-spiked samples (up to four organisms per mix), to provide an estimated LoD concentration, and to determine whether assay sensitivity is affected by the presence of

multiple panel organisms in a single sample. Confirmation of LoDs was performed by spiking organism (single or multi-spike) at the LoD estimate determined by the dilutions series, into 20 independent stool samples. LoD was confirmed when the correct organism/assay results were obtained from at least 19 of the 20 samples ( $1 9 / 2 0 = 9 5 \%$ ) tested.

Table - Confirmed Limit of Detection (LoD) for GI Panel Analytes   

<table><tr><td rowspan=1 colspan=1>GI Panel Test Result</td><td rowspan=1 colspan=1>Species/Isolate Tested</td><td rowspan=1 colspan=1>Confirmed LoDConcentration</td><td rowspan=1 colspan=1>Detection at LoDConcentration</td></tr><tr><td rowspan=1 colspan=1>BACTERIA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Campylobacter coliATCC 33559</td><td rowspan=3 colspan=1>4 x 104 cells/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuniATCC BAA-1234</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensisATCC BAA-1059</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Clostridium difficile (toxin A/B)</td><td rowspan=1 colspan=1>Clostridium difficileToxinotype 0 A+B+ATCC 9689</td><td rowspan=1 colspan=1>4 x 105 cells/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (NAP1)Toxinotype III A+B+Zeptometrix #801619</td><td rowspan=1 colspan=1>4 x 104 cells/mL</td><td rowspan=1 colspan=1>19/2095%</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Plesiomonas shigelloidesATCC 14029</td><td rowspan=1 colspan=1>1 x 103³ CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonella bongori O66:H1z41:H2-SGSC RKS#3041 SarC11</td><td rowspan=1 colspan=1>1 x 104 cFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica ssp. enterica SerovarTyphimurium01,4,[5],12:H1i:H21,2SGSC RKS#4194 SarC1</td><td rowspan=1 colspan=1>5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Vibrio andVibrio cholerae</td><td rowspan=1 colspan=1>Vibrio choleraeOgawa serotype O:1ATCC 14035</td><td rowspan=1 colspan=1>8 x 103 cells/mL</td><td rowspan=1 colspan=1>20/2000%</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticusATCC 17802</td><td rowspan=1 colspan=1>8 x 104 cells/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocoliticaATCC 9610Biovar1 serogroup O:8</td><td rowspan=1 colspan=1>5 x 104 cFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>DIARRHEAGENIC E. coli/Shigella</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>Escherichia coli O92:H33STEC Center # JM221</td><td rowspan=1 colspan=1>1 x 104 cFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>Escherichia coli E2348/69 O127:H6STEC Center</td><td rowspan=1 colspan=1>1 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Escherichia coli H10407 O78:H11ATCC 35401</td><td rowspan=1 colspan=1>1 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Shiga-like toxin-producing E. coli (STEC)stx1/tx2E. coli O157</td><td rowspan=1 colspan=1>Escherichia coli O25:H11ATCC BAA-2196</td><td rowspan=1 colspan=1>1 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157:H7ATCC 43895</td><td rowspan=1 colspan=1>1 x 104 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1>Escherichia coli O29:NMATCC 43892</td><td rowspan=1 colspan=1>5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Shigella sonneiATCC 29930</td><td rowspan=1 colspan=1>100 CFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>PARASITES</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium a</td><td rowspan=1 colspan=1>Cryptosporidium parvumIowa isolate (Harley Moon)Waterborne, Inc. P102C</td><td rowspan=2 colspan=1>5 x 103 oocysts/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominisClinical Specimen</td><td rowspan=1 colspan=1>20/20100%</td></tr></table>

p. 25

<table><tr><td rowspan=1 colspan=1>GI Panel Test Result</td><td rowspan=1 colspan=1>Species/Isolate Tested</td><td rowspan=1 colspan=1>Confirmed LoDConcentration</td><td rowspan=1 colspan=1>Detection at LoDConcentration</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>Cyclospora cayetanensisClinical Specimen</td><td rowspan=1 colspan=1>180 genomeequivalents(GE)/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Entamoeba histolytica HM-1:IMSSATCC 30459</td><td rowspan=1 colspan=1>2 x 103 cells/mL</td><td rowspan=1 colspan=1>19/2095%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Giardia intestinalis (aka G. lamblia)ATCC 30957</td><td rowspan=1 colspan=1>50 cells/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>VIRUSES</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1>Adenovirus F40ATCC VR-931</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F41ATCC VR-930</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>Astrovirus - Type 8NCPV#1003071v</td><td rowspan=1 colspan=1>50 FFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Norovirus GIClinical Specimen</td><td rowspan=2 colspan=1>1 x 104RNAcopies/mL</td><td rowspan=1 colspan=1>19/2095%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GIIClinical Specimen</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Rotavirus A - Type G4 [P6]NCPV#0904053v</td><td rowspan=1 colspan=1>1 x 105 FFU/mL</td><td rowspan=1 colspan=1>20/20100%</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>Sapovirus (Genogroup I)Clinical Specimen</td><td rowspan=1 colspan=1>1.1 x 107 RNAcopies /mL</td><td rowspan=1 colspan=1>20/20100%</td></tr></table>

a Limited testing with a clinical specimen containing Cryptosporidium meleagridis indicates that the LoD for this species is similar to that of $C .$ . parvum and $C$ . hominis.

# e. Inclusivity

The analytical reactivity (inclusivity) of the FilmArray GI Panel was evaluated with a collection of 270 isolates that represent the diversity of the FilmArray GI Panel analytes. Isolates were selected to represent relevant subspecies or serotypes and selection was biased toward more common species and known human pathogens. When possible, in silico analysis of sequence data was used to make predictions of assay reactivity for less common species, strains, serovars, or serotypes that were not tested.

Organisms were tested at concentrations near the limit of detection (LoD). If a sample containing a particular strain was positive (detected) at the initial test level, no further testing was required. If a strain was not detected, the strain was retested at the same level (up to five additional times) and if necessary, additional testing was performed at 10- and 100-fold higher concentrations to determine if the strain can be detected by the GI Panel. Based upon predicted assay reactivity, a few select isolates were initially tested at a high concentration, followed by evaluation at lower concentrations if detection was observed. Results are provided below for each FilmArray GI Panel test result.

Table - FilmArray Campylobacter Inclusivity Results (C. coli/C. jejuni/C. upsaliensis)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=5 colspan=1>Campylobacter colia</td><td rowspan=1 colspan=1>ATCC BAA-1061</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>BEI HM-296</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC43485</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43478</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 33559b</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=3 colspan=1>Campylobacter jejuni subsp. doylei</td><td rowspan=1 colspan=1>ATCC 49349</td><td rowspan=1 colspan=1>4.0 x 106</td><td rowspan=1 colspan=1>Not Detectedc</td></tr><tr><td rowspan=1 colspan=1>ATCC 49351</td><td rowspan=1 colspan=1>4.0 x 106</td><td rowspan=1 colspan=1>100×LoDc</td></tr><tr><td rowspan=1 colspan=1>ATCC 49350</td><td rowspan=1 colspan=1>4.0 x 106</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=4 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>ATCC 43430</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1062</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1234b</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>BEI NR-128</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=6 colspan=1>Campylobacter upsaliensis</td><td rowspan=1 colspan=1>ATCC BAA-1059</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>CCUG 24191</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43953</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43954d</td><td rowspan=1 colspan=1>4.0 x 106</td><td rowspan=1 colspan=1>Not Detectedd</td></tr><tr><td rowspan=1 colspan=1>ATCC 49815</td><td rowspan=1 colspan=1>1.2 x 10 5</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>BEI HM-297</td><td rowspan=1 colspan=1>1.2 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a In silico analysis indicates primer mismatches that might lead to reduced assay sensitivity or lack of reactivity with $1 1 / 1 3 8 C .$ coli sequences. b Isolate was used to establish the LoD for this assay. c In silico analysis indicates primer mismatches that might lead to reduced assay sensitivity for this subspecies. d Sequencing under the primers identified an insertion/deletion in the primer binding region of the target

Table - FilmArray Clostridium difficile toxin A/B Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=9 colspan=1>Clostridium difficile</td><td rowspan=9 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 9689a</td><td rowspan=1 colspan=1>4.0 x 105</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1382</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 17857</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 17858</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43594</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43596</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43599</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 43600</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

p. 27

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=9 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>ATCC 51695</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 700792</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=2 colspan=1>III A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1805(NAP1)</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix#0801619 (NAP1)a</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>VA+B+</td><td rowspan=1 colspan=1>ATCC BAA-1875</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>VIII A-B+</td><td rowspan=1 colspan=1>ATCC 43598</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>XA-B+</td><td rowspan=1 colspan=1>CCUG 8864</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>XII A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1812</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>XXII A+B (unknown)</td><td rowspan=1 colspan=1>ATCC BAA-1814</td><td rowspan=1 colspan=1>1.2 x 106</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a This isolate was used to establish the LoD for this assay.

Table - FilmArray Plesiomonas shigelloides Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>GeographicIsolation</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected (cells/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=6 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>CDC 3085-55</td><td rowspan=1 colspan=1>ATCC 14029a</td><td rowspan=1 colspan=1>1.0 x 103</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>CDC 16408</td><td rowspan=1 colspan=1>ATCC 14030</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Dakar, Senegal</td><td rowspan=1 colspan=1>ATCC 51572</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC 51903</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Colorado</td><td rowspan=1 colspan=1>CDPH HUM-2011019465</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Czech Republic</td><td rowspan=1 colspan=1>NIPH-Czech Republic 6300</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a This isolate was used to establish the LoD for this assay. The organism was quantified in CFU/mL by plate enumeration.

Table FilmArray Salmonella Inclusivity Results   

<table><tr><td colspan="1" rowspan="1">Organism(species, subspecies, and serovar)</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">ConcentrationDetected(cells/mL)</td><td colspan="1" rowspan="1">Multiple ofLoD Detected</td></tr><tr><td colspan="1" rowspan="3">Salmonella bongori</td><td colspan="1" rowspan="1">SGSCRKS 3041a</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">1xLoD</td></tr><tr><td colspan="1" rowspan="1">NCTC 10946</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">SGSCRKS 3044</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. salamae II</td><td colspan="1" rowspan="1">SGSCRKS 2985</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae IIIa</td><td colspan="1" rowspan="1">SGSCRKS 2980</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. diarizonae IIIb</td><td colspan="1" rowspan="1">SGSCRKS 2978</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. houtenae IV</td><td colspan="1" rowspan="1">SGSCRKS 3027</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="2" rowspan="1">Organism(species, subspecies, and serovar)</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">ConcentrationDetected(cells/mL)</td><td colspan="1" rowspan="1">Multiple ofLoD Detected</td></tr><tr><td colspan="2" rowspan="1">Salmonella enterica subsp. indica VI</td><td colspan="1" rowspan="1">SGSCRKS 2995</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="20">Salmonellaentericasubsp. entericab</td><td colspan="1" rowspan="1">Typhimurium</td><td colspan="1" rowspan="1">SGSCRKS 4194a</td><td colspan="1" rowspan="1">5.0 x 103</td><td colspan="1" rowspan="1">1xLoD</td></tr><tr><td colspan="1" rowspan="1">Enteritidis</td><td colspan="1" rowspan="1">ATCC BAA-708</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Newport</td><td colspan="1" rowspan="1">ATCC 27869</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Javiana</td><td colspan="1" rowspan="1">ATCC 10721</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Heidelberg</td><td colspan="1" rowspan="1">ATCC 8326</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Montevideo</td><td colspan="1" rowspan="1">ATCC BAA-710</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">1 4,[5],12:i:-</td><td colspan="1" rowspan="1">Cornell CU0580</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Oranienburg</td><td colspan="1" rowspan="1">ATCC 9239</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Saintpaul</td><td colspan="1" rowspan="1">ATCC 9712</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Muenchen</td><td colspan="1" rowspan="1">ATCC 8388</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Braenderup</td><td colspan="1" rowspan="1">ATCC 700136</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Infantis</td><td colspan="1" rowspan="1">ATCCBAA-1675</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Thompson</td><td colspan="1" rowspan="1">ATCC 8391</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Mississippi</td><td colspan="1" rowspan="1">Cornell CU0633</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Paratyphi B var. L(+)tartrate+(formerly java)</td><td colspan="1" rowspan="1">CCUG 9561</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Typhi (Purified DNA)b</td><td colspan="1" rowspan="1">ATCC700931D-5</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Agona</td><td colspan="1" rowspan="1">ATCC 51957</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Schwarzengrund</td><td colspan="1" rowspan="1">CCUG 21280</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Bareilly</td><td colspan="1" rowspan="1">ATCC 9115</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Hadar</td><td colspan="1" rowspan="1">ATCC51956</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3×LoD</td></tr></table>

a This isolate was used to establish the LoD for this assay. The organism was quantified in $\overline { { \mathrm { C F U / m L } } }$ by plate enumeration. b Purified DNA was quantified in $\mathrm { G E / m L }$ by spectrophotometer.

Note: In addition to those evaluated in this study, in silico sequence analysis indicates the FilmArray Salmonella assay should react with all species and subspecies of Salmonella, including all serotypes of S. enterica subsp. enterica.

Table - FilmArray Vibrio (V. parahaemolyticus/V. vulnificus/V. cholerae) and Vibrio cholerae Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Organismspecies, biotype and serotype)</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=6 colspan=1>Vibriocholerae</td><td rowspan=1 colspan=1>O:1 Ogawa</td><td rowspan=1 colspan=1>ATCC 14035a</td><td rowspan=1 colspan=1>8.0 x 103</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>O:1 Inaba, Biotype El Tor</td><td rowspan=1 colspan=1>BEI NR-147</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>O:1 Ogawa, Biotype El Tor</td><td rowspan=1 colspan=1>BEI NR-148</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>non-O:1,non-O:139 (0:2)</td><td rowspan=1 colspan=1>BEI NR-149</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>non-O:1,non-O:139 (0:7)</td><td rowspan=1 colspan=1>BEI NR-152</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>O:1 Inaba, Biotype El Tor</td><td rowspan=1 colspan=1>ATCC 25870</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=6 colspan=2>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802a</td><td rowspan=1 colspan=1>8.0 x 104</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-242</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 27969</td><td rowspan=1 colspan=1>2.4 x 10}$</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 33845</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>BEI NR-21990</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>BEI NR-21992</td><td rowspan=1 colspan=1>2.4 x 10 5$</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=5 colspan=2>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 29306</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 33817</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-88</td><td rowspan=1 colspan=1>2.4 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 27562</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-86</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>0.3xLoD</td></tr></table>

a Isolate was used to establish the LoD for this assay.

Note: In the clinical evaluation, a Vibrio carrying a variant toxR sequence was not detected by the Vchol assay and very rare strains of pathogenic $V .$ . cholerae that do not carry that toxR gene will also not be detected by the Vchol assay.

Table - FilmArray Yersinia enterocolitica Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source/IsolateID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multipleoff LoDDetected</td></tr><tr><td rowspan=7 colspan=1>Yersiniaenterocolitica</td><td rowspan=3 colspan=1>0:8</td><td rowspan=1 colspan=1>ATCC 9610a</td><td rowspan=1 colspan=1>5.0 x 104</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>ATCC 23715</td><td rowspan=1 colspan=1>1.5 x 105$</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>BEI NR-207</td><td rowspan=1 colspan=1>1.5 x 105}$</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>0:5,27</td><td rowspan=1 colspan=1>NCTC 10463</td><td rowspan=1 colspan=1>1.5 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=2 colspan=1>0:3</td><td rowspan=1 colspan=1>ATCC 700822</td><td rowspan=1 colspan=1>1.5 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>BEI NR-212</td><td rowspan=1 colspan=1>1.5 x 10 5</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>0:9</td><td rowspan=1 colspan=1>ATCC 55075</td><td rowspan=1 colspan=1>1.5 x 105</td><td rowspan=1 colspan=1>3xLoD</td></tr></table>

a Isolate was used to establish the LoD for this assay. The organism was quantified in CFU/mL by plate enumeration.

Note: In addition to those evaluated in this study, in silico sequence analysis indicates the FilmArray Yersinia enterocolitica assay should react with all strains/serotypes of Y. enterocolitica (including O:1, 2a, 3; O:2a,3; O:4,32; O:12,25; O:13a,13b; O:19; O:20; and O:21).

Table - FilmArray Enteroaggregative E. coli (EAEC) Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multipleof LoDDetected</td></tr><tr><td rowspan=10 colspan=1>Enteroaggregative E. coli(EAEC)</td><td rowspan=1 colspan=1>092:H33</td><td rowspan=1 colspan=1>STEC CenterJM221a</td><td rowspan=1 colspan=1>1.0 x 104</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>0162:NM</td><td rowspan=1 colspan=1>Penn State 92.0148</td><td rowspan=1 colspan=1>3.0 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>017:H6</td><td rowspan=1 colspan=1>Penn State 92.0142</td><td rowspan=1 colspan=1>3.0 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>04:H7</td><td rowspan=1 colspan=1>Penn State 92.0144</td><td rowspan=1 colspan=1>3.0 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>051:H11</td><td rowspan=1 colspan=1>Penn State 92.0143</td><td rowspan=1 colspan=1>3.0 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>068:NM</td><td rowspan=1 colspan=1>Penn State 92.0154</td><td rowspan=1 colspan=1>3.0 x 104</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>07:NM</td><td rowspan=1 colspan=1>Penn State 92.0151</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>044:H18</td><td rowspan=1 colspan=1>STEC Center O42</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>0104:H4(Purified DNA)b</td><td rowspan=1 colspan=1>2011 EuropeanOutbreak strain</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>Ond:H10d</td><td rowspan=1 colspan=1>STEC Center 101-1</td><td rowspan=1 colspan=1>1.5 x 108</td><td rowspan=1 colspan=1>NotDetectedd</td></tr></table>

a Isolate was used to establish the LoD for this assay. The organism was quantified in CFU/mL by plate enumeration. b Purified DNA was quantified in $\mathrm { G E / m L }$ by spectrophotometer. c Isolate has genetic characteristics consistent with STEC and EAEC. d Phenotypic EAEC but known to not carry the marker(s) detected by the FilmArray GI Panel EAEC assay.

able - FilmArray Enteropathogenic E. coli (EPEC) Inclusivity Result   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Typical/Atypical</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=10 colspan=1>EnteropathogenicE. coli (EPEC)</td><td rowspan=1 colspan=1>0127:H6</td><td rowspan=1 colspan=1>Typical</td><td rowspan=1 colspan=1>STEC CenterE2348/69a</td><td rowspan=1 colspan=1>1.0 x 103</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>O128:H2</td><td rowspan=1 colspan=1>Atypical</td><td rowspan=1 colspan=1>STEC Center DEC11a</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>111a:NM</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STEC Center Stoke W</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0142:H6</td><td rowspan=1 colspan=1>Typical</td><td rowspan=1 colspan=1>STEC Center E851/71</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>O55:H7</td><td rowspan=1 colspan=1>Atypical</td><td rowspan=1 colspan=1>STEC Center DEC5A</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>O114:H2</td><td rowspan=1 colspan=1>Typical</td><td rowspan=1 colspan=1>STEC Center 3448-87</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0119:H+</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STEC CenterRN410/1</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>096:H</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STEC Center HSP19/4</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>086:Hnm</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STEC Center E990</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>055:H-</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STEC CenterMA551/1</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay. The organism was quantified in CFU/mL by plate enumeration.

able-FilmArray Enterotoxigenic E. coli (ETEC) lt/st Inclusivity Result   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>ST/LT</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(cells/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=13 colspan=1>EnterotoxigenicE. coli (ETEC)</td><td rowspan=1 colspan=1>078:H11</td><td rowspan=1 colspan=1>STA (+)/LT (+)</td><td rowspan=1 colspan=1>ATCC 35401a</td><td rowspan=1 colspan=1>1.0 x 103</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>O175:H15</td><td rowspan=1 colspan=1>STA (-)/LT (+)</td><td rowspan=1 colspan=1>Penn State 6.0671</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0149:H5</td><td rowspan=1 colspan=1>STA (-)/LT (+)</td><td rowspan=1 colspan=1>Penn State 6.1182</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>084:H28</td><td rowspan=1 colspan=1>STA (-)/LT (+)b</td><td rowspan=1 colspan=1>Penn State 7.1493</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>Not Detectedb</td></tr><tr><td rowspan=1 colspan=1>H5</td><td rowspan=1 colspan=1>STA (+)/LT (-)</td><td rowspan=1 colspan=1>Penn State 10.0049</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0168</td><td rowspan=1 colspan=1>STA (+)/LT (-)</td><td rowspan=1 colspan=1>Penn State 9.1809</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0145:H25</td><td rowspan=1 colspan=1>STA (+)/LT (-)</td><td rowspan=1 colspan=1>Penn State 10.0136</td><td rowspan=1 colspan=1>1.0 x 104</td><td rowspan=1 colspan=1>100xLoDc</td></tr><tr><td rowspan=1 colspan=1>078</td><td rowspan=1 colspan=1>STA (+)/LT (+)</td><td rowspan=1 colspan=1>Penn State 2.1507</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>019:H5</td><td rowspan=1 colspan=1>STA (+)/LT (+)</td><td rowspan=1 colspan=1>Penn State 5.0038</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>H14</td><td rowspan=1 colspan=1>STA (+)/LT (-)</td><td rowspan=1 colspan=1>Penn State 10.045</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>0141</td><td rowspan=1 colspan=1>STA (+)/LT (+)</td><td rowspan=1 colspan=1>Penn State 93.0045</td><td rowspan=1 colspan=1>3.0 x 103</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STB (+)dSTA(-)/LT(-)</td><td rowspan=1 colspan=1>Penn State 8.2425</td><td rowspan=1 colspan=1>1.5 x 109</td><td rowspan=1 colspan=1>Not Detectedd</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>STB (+)dSTA(-)/LT(-)</td><td rowspan=1 colspan=1>Penn State 9.1179</td><td rowspan=1 colspan=1>1.5 x 109</td><td rowspan=1 colspan=1>Not Detectedd</td></tr></table>

a Isolate was used to establish the LoD for this assay. The organism was quantified in CFU/mL by plate enumeration. b Secondary PCR assay could not confirm the presence of the target gene(s) – plasmid/gene loss suspected. c Sequencing of the target gene(s) identified sequence variation leading to reduced sensitivity for STA in this isolate. d The FilmArray GI Panel will not detect phenotypic ETEC that that express only heat-stable toxin ST2/STB or heat-labile toxin LT-II.

Table -FilmArray Shiga-like toxin producing E. coli (STEC) stx1/stx2 and E. coli O157 Inclusivity Results   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">stx1/stx2</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">ConcentrationDetected(cells/mL)</td><td colspan="1" rowspan="1">Multipleof LoDDetectedSTEC</td><td colspan="1" rowspan="1">Multiple ofLoDDetected0157</td></tr><tr><td colspan="1" rowspan="10">Shiga-liketoxinproducingE. coli(STEC)</td><td colspan="6" rowspan="1">STEC (non-O157)</td></tr><tr><td colspan="1" rowspan="1">O25:H11</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">ATCCBAA-2196a</td><td colspan="1" rowspan="1">1.0 x 103</td><td colspan="1" rowspan="1">1×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0113:H21</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">ATCCBAA-177</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">045:H2</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">STEC Center DEC11C</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0103:H2</td><td colspan="1" rowspan="1">+/Unknown</td><td colspan="1" rowspan="1">STEC Center 107-226</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0104:H21</td><td colspan="1" rowspan="1">-/+</td><td colspan="1" rowspan="1">STEC Center G5506</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0111:NM</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">STEC Center95-3208</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">O111:H2</td><td colspan="1" rowspan="1">-/+</td><td colspan="1" rowspan="1">STEC Center RD8</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0111:H8</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">STEC Center DEC8B</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0121:H19</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">STEC Center F6173</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="13"></td><td colspan="1" rowspan="1">026:NM</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">STEC CenterDA-22</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">026:H11</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">STEC CenterH19</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0145:NM</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">STEC CenterGS G5578620</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1">0104:H4b(PurifiedDNA)c</td><td colspan="1" rowspan="1">-/+</td><td colspan="1" rowspan="1">ATCCBAA-2326D-5b</td><td colspan="1" rowspan="1">3.0 x 103c</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">NotDetected</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">STEC O157</td></tr><tr><td colspan="1" rowspan="1">0157:NM</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">STEC CenterDA-26</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">0.3×LoD</td></tr><tr><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">-/+</td><td colspan="1" rowspan="1">STEC Center E32511</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">0.3×LoD</td></tr><tr><td colspan="1" rowspan="1">0157:HNT</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">STEC CenterDA-74</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">3×LoD</td><td colspan="1" rowspan="1">0.3×LoD</td></tr><tr><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">ATCC 43895a</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">10xLoD</td><td colspan="1" rowspan="1">1xLoD</td></tr><tr><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">STEC Center A8993-CS2</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">30×LoD</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="3" rowspan="1">Non-STEC O157</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">-/-</td><td colspan="1" rowspan="1">ATCC 43888</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">N/Ad</td></tr><tr><td colspan="1" rowspan="1">0157:H45</td><td colspan="1" rowspan="1">-/-</td><td colspan="1" rowspan="1">STEC Center SC373/2</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">N/Ad</td></tr></table>

a Isolate was used to establish the LoD. The organism was quantified in CFU/mL by plate enumeration. b 2011 European Outbreak Strain. Isolate has genetic characteristics consistent with STEC and EAEC. c Purified DNA was quantified in $\mathrm { G E / m L }$ by spectrophotometer. $^ { \mathrm { d } } E .$ . coli $\mathrm { O } 1 5 7 \mathrm { N } / \mathrm { A }$ results reported due to lack of positive results for STEC.

Note: Based on in silico analysis, stx2 subtypes e and f are predicted to be detected with reduced sensitivity or not detected by the FilmArray GI Panel STEC assays.

Table - FilmArray Shigella/Enteroinvasive E. coli (EIEC) Inclusivity Results   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotype(Temporal/Geographic Isolation)</td><td colspan="1" rowspan="1">Source/Isolate ID</td><td colspan="1" rowspan="1">ConcentrationDetected(cells/mL)</td><td colspan="1" rowspan="1">Multiple ofLoD Detected</td></tr><tr><td colspan="1" rowspan="6">EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">029:NM</td><td colspan="1" rowspan="1">ATCC 43892a</td><td colspan="1" rowspan="1">5.0 x 103</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">029:HNM (1977)</td><td colspan="1" rowspan="1">STEC Center 1885-77</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">0124:HNM (1978)</td><td colspan="1" rowspan="1">STEC Center 929-78</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">O29:H27(USA, VA; 1979)</td><td colspan="1" rowspan="1">STEC Center 1827-79</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">Not Detectedb</td></tr><tr><td colspan="1" rowspan="1">O28:H-(Brazil, 1983)</td><td colspan="1" rowspan="1">STEC Center LT-15</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">0136:H-(Bangladesh, 1983)</td><td colspan="1" rowspan="1">STEC Center LT-41Strain 1111-55</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="2">Shigella boydii(Serogroup C)</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">ATCC 8700</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 4</td><td colspan="1" rowspan="1">CDPH HUM-2010029296</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">ATCC 9207</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 20</td><td colspan="1" rowspan="1">ATCC BAA-1247</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 10</td><td colspan="1" rowspan="1">ATCC 12030</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="5">Shigelladysenteriae(Serogroup A)</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">BEI NR-520</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoDc</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">CDPH PHM-2004008089</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 13</td><td colspan="1" rowspan="1">ATCC 49555</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">ATCC 29028</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 12</td><td colspan="1" rowspan="1">ATCC 49551</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="6">Shigella flexneri(Serogroup B)</td><td colspan="1" rowspan="1">Type 2a</td><td colspan="1" rowspan="1">ATCC 700930</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type la</td><td colspan="1" rowspan="1">ATCC 9199</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 6</td><td colspan="1" rowspan="1">CDPH PHM-2006004043</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 2b</td><td colspan="1" rowspan="1">ATCC 12022</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Type 2a</td><td colspan="1" rowspan="1">ATCC 29903</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">STEC Center VA-6</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="6">Shigella sonnei(Serogroup D)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC 29930</td><td colspan="1" rowspan="1">1.0 x 102a</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC 11060</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">CDPH HUM-2010027998</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC 29031</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC 25931</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ATCC 9290</td><td colspan="1" rowspan="1">3.0 x 102</td><td colspan="1" rowspan="1">3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay. The organism was quantified in $\mathrm { C F U / m L }$ by plate enumeration. b Secondary PCR assay could not confirm the presence of the target gene(s) – plasmid/gene loss suspected. c This isolate gave the expected STEC Detected and Shigella/EIEC Detected results due to the presence of $s t x$ in Shigella dysenteriae.

Table - FilmArray Cryptosporidium Inclusivity Results   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Geographic Source/Isolate</td><td colspan="1" rowspan="1">ConcentrationDetected(cells/mL)</td><td colspan="1" rowspan="1">Multiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium canis</td><td colspan="1" rowspan="1">Peru Clinical Sample</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">&lt;LoDa</td></tr><tr><td colspan="1" rowspan="4">Cryptosporidium hominis</td><td colspan="1" rowspan="1">Scotland Clinical Sampleb</td><td colspan="1" rowspan="1">2.1 x 103 b</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">Scotland Clinical Sample</td><td colspan="1" rowspan="1">6.4 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Scotland Clinical Sample</td><td colspan="1" rowspan="1">6.4 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">BEI NR-2520 (PurifiedDNA Isolate TU502)</td><td colspan="1" rowspan="1">6.4 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium meleagridis</td><td colspan="1" rowspan="1">BEI NR-2521 (PurifiedDNA Isolate TU1867)</td><td colspan="1" rowspan="1">1.8 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium muris</td><td colspan="1" rowspan="1">Waterborne P104</td><td colspan="1" rowspan="1">1.5×104oocycts/mL</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="4">Cryptosporidium parvum</td><td colspan="1" rowspan="1">Waterborne P102Cc</td><td colspan="1" rowspan="1">6.0 x 102 c</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">Scotland Clinical Sample</td><td colspan="1" rowspan="1">1.8 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">Scotland Clinical Sample</td><td colspan="1" rowspan="1">1.8 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="1">BEI NR-2519 (PurifiedDNA Isolate Iowa)</td><td colspan="1" rowspan="1">1.8 x 103</td><td colspan="1" rowspan="1">3×LoD</td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium ubiquitum</td><td colspan="1" rowspan="1">Scotland Purified DNAfrom Clinical Sample</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">&lt;LoDa</td></tr><tr><td colspan="1" rowspan="1">Scotland Purified DNAfrom Clinical Sample</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">&lt;LoDa</td></tr></table>

a Quantification by qPCR indicated that these purified samples have an analyte concentration that is lower than the assay LoD. b This C. hominis sample was used to establish the LoD for $C .$ . hominis (LoD of $5 . 0 \times 1 0 ^ { 3 }$ oocysts/mL was determined to be equivalent to $2 . 1 \times 1 0 ^ { 3 }$ copies/mL). c This C. parvum isolate was used to establish the LoD for $C .$ . parvum (LoD of $5 . 0 \times 1 0 ^ { 3 }$ oocysts/mL was determined to be equivalent to $6 . 0 \times 1 0 ^ { 2 }$ copies/mL).

Note: In silico sequence analysis indicates the FilmArray Cryptosporidium assay(s) should react with approximately 23 different Cryptosporidium species (including those evaluated in this study) as well as sequences not assigned to specific species. In silico analysis predicts that the Cryptosporidium assay(s) may not react with the rare or non-human species C. bovis, $C .$ . ryanae and $C .$ . xiaoi.

Table - FilmArray Cyclospora cayetanensis Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Location/Sample</td><td rowspan=1 colspan=1>ConcentrationDetected(GE/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=7 colspan=1>Cyclosporacayetanensis</td><td rowspan=3 colspan=1>Nebraska</td><td rowspan=1 colspan=1>Clinical Specimena</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=4 colspan=1>Peru</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay.

Table - FilmArray Entamoeba histolytica Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainDesignation</td><td rowspan=1 colspan=1>Location/Yearof Isolation</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=6 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>HM-1:IMSS</td><td rowspan=1 colspan=1>Mexico City1967</td><td rowspan=1 colspan=1>ATCC 30459a</td><td rowspan=1 colspan=1>~1.2 x 105</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>EntaHB-301:NIH</td><td rowspan=1 colspan=1>Burma 1960</td><td rowspan=1 colspan=1>BEI NR-176</td><td rowspan=1 colspan=1>3.6 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Rahman</td><td rowspan=1 colspan=1>England 1972</td><td rowspan=1 colspan=1>BEI NR-179</td><td rowspan=1 colspan=1>3.6 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>HU-21:AMC</td><td rowspan=1 colspan=1>Arkansas 1970</td><td rowspan=1 colspan=1>BEI NR-2589</td><td rowspan=1 colspan=1>3.6 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>IP:1182:2</td><td rowspan=1 colspan=1>Honduras 1982</td><td rowspan=1 colspan=1>BEI NR-20088</td><td rowspan=1 colspan=1>3.6 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>SAW 408 RR,Clone A</td><td rowspan=1 colspan=1>Mexico</td><td rowspan=1 colspan=1>BEI NR-20090</td><td rowspan=1 colspan=1>3.6 x 105</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay (LoD of $\overline { { 2 . 0 { \times } 1 0 ^ { 3 } } }$ cells/mL was determined to be equivalent to ${ \sim } 1 . 2 { \times } 1 0 ^ { 5 }$ copies/mL).

Table - FilmArray Giardia lamblia Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Location/Year ofIsolation</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected (cells/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=6 colspan=1>Giardialamblia(aka G.intestinalis orG. duodenalis)</td><td rowspan=1 colspan=1>New Orleans, LA 1985</td><td rowspan=1 colspan=1>ATCC 50137</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Portland, OR 1971</td><td rowspan=1 colspan=1>ATCC 30888</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Bethesda, MD 1979</td><td rowspan=1 colspan=1>ATCC 30957a</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Waterborne P101</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>ATCC PRA-243</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>United States</td><td rowspan=1 colspan=1>ATCC PRA-247</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay.

Table - FilmArray Adenovirus F 40/41 Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=4 colspan=1>Adenovirus F 40</td><td rowspan=1 colspan=1>ATCC VR-931a</td><td rowspan=1 colspan=1>~2.8×105</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample E239</td><td rowspan=1 colspan=1>8.4×105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>NCPV 0101141v(Dugan)</td><td rowspan=1 colspan=1>8.4×105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810084CF</td><td rowspan=1 colspan=1>8.4×105</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=6 colspan=1>Adenovirus F 41</td><td rowspan=1 colspan=1>ATCC VR-930 (Tak)a</td><td rowspan=1 colspan=1>~3.0×104</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix#0810085CF (Tak)b</td><td rowspan=1 colspan=1>9.0×104</td><td rowspan=1 colspan=1>10×LoDb</td></tr><tr><td rowspan=1 colspan=1>UIRF/Zeptometrix305571</td><td rowspan=1 colspan=1>9.0×104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample 762</td><td rowspan=1 colspan=1>9.0×104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample 976</td><td rowspan=1 colspan=1>9.0×104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample Chn81</td><td rowspan=1 colspan=1>9.0×104</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay. For ATCC VR-9310, the LoD of $1 ~ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ was determined to be equivalent to $2 . 8 { \times } 1 0 ^ { 5 } \mathrm { c o p i e s / m L }$ and for ATCC VR-930, the LoD of $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ was determined to be equivalent to $3 . 0 \times 1 0 ^ { 4 }$ copies/mL. b Same strain as ATCC VR-930 (which was detected at $1 \times \mathrm { L o D } $ ) but obtained from a different source.

Table - FilmArray Astrovirus Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Location/Source/Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=10 colspan=1>HumanAstrovirus</td><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>China Clinical Sample</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>10×LoD</td></tr><tr><td rowspan=1 colspan=1>China Clinical Sample</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>USA Clinical Sample</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>University of Barcelona Spain</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NCPV #1002072v</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=2 colspan=1>5</td><td rowspan=1 colspan=1>USA Clinical Sample</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>USA Clinical Sample</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>University of Barcelona Spain</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>University of Barcelona Spain</td><td rowspan=1 colspan=1>3.9×107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>NCPV #1003071va</td><td rowspan=1 colspan=1>~1.3×107</td><td rowspan=1 colspan=1>1×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay (LoD of 50 FFU/mL was determined to be equivalent to $\overline { { 1 . 3 \times 1 } } 0 ^ { 7 }$ copies/mL).

Table - FilmArray Norovirus GI/GII Inclusivity Results   

<table><tr><td colspan="2" rowspan="1">NorovirusGenogroup/Genotype</td><td colspan="1" rowspan="1">Isolate ID(Clinical Samples)</td><td colspan="1" rowspan="1">ConcentrationDetected(copies/mL)</td><td colspan="1" rowspan="1">Multiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="11">NorovirusGI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Norol_036a</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Norol_002</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="3">3</td><td colspan="1" rowspan="1">Norol_003</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">Norol_012</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">Norol_030</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Norol_031</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Norol_021</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">10×LoD</td></tr><tr><td colspan="1" rowspan="3">7</td><td colspan="1" rowspan="1">Norol_009</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">20×LoDc</td></tr><tr><td colspan="1" rowspan="1">Norol_029</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">Noro1_034</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Noro G1.8b</td><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">6×LoD</td></tr><tr><td colspan="1" rowspan="10">NorovirusGII</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Noro2_013a</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">1×LoD</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">NoroII.2b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">China-5</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">SGB_038</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="5">4</td><td colspan="1" rowspan="1">GI-PILOT-SPDRL-077</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">20×LoDc</td></tr><tr><td colspan="1" rowspan="1">Noro2_004</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">20×LoDc</td></tr><tr><td colspan="1" rowspan="1">Noro2_032</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">20×LoDc</td></tr><tr><td colspan="1" rowspan="1">PCMC__025 (Sydney)</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">PCMC__031 (Sydney)</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">NYH-A</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">NoroII.7b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">NoroII.8b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">NoroII.12b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">NoroII.16b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr><tr><td colspan="1" rowspan="2">20</td><td colspan="1" rowspan="1">NoroII.20cb</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">20×LoDc</td></tr><tr><td colspan="1" rowspan="1">NoroII.20b</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">0.6×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay. b Isolate obtained as RNA extract from a clinical sample. Genotype provided by the source laboratory. c Noroviruses are genetically diverse. In silico analysis predicts that most strains of all genotypes will be detected, though some variant strains may be detected with reduced sensitivity or may not be detected due to inefficient amplification or exclusion by melt analysis.

Table - FilmArray Rotavirus A Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain Designation(Serotype)</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=6 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ST3 (G4P6)</td><td rowspan=1 colspan=1>NCPV 0904053va</td><td rowspan=1 colspan=1>3.9 x 103</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>RV4 (G1P8)</td><td rowspan=1 colspan=1>NCPV 0904052v</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>69M (G8P5)</td><td rowspan=1 colspan=1>NCPV 0904055v</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>P (G3P1A[8])</td><td rowspan=1 colspan=1>NCPV 0904056v</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Wa (G1P1A[8])</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>DS-1 (G2P1B[4])</td><td rowspan=1 colspan=1>ATCC VR-2550</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Isolate was used to establish the LoD for this assay (LoD of $\overline { { 1 . 0 { \times } 1 0 ^ { 5 } \ : \mathrm { F F U / m L } } }$ was determined to be equivalent to $3 . 9 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ copies/mL).

Note: The Rotavirus A assay will also detect reassortant viruses used in vaccine production.

Table - FilmArray Sapovirus Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Genogroup</td><td rowspan=1 colspan=1>Isolate ID(Clinical Samples)</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=11 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>AB_SaV_14a</td><td rowspan=1 colspan=1>5.0 x 106</td><td rowspan=1 colspan=1>1×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>China_56</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>AB_SaV_03</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>PCMC_54</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SPDRL-006</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SPDRL-099</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>SGB-MP-11</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>Sapo_03b</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>Sapo__06b</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>GIV</td><td rowspan=1 colspan=1>Sapo_09b</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr><tr><td rowspan=1 colspan=1>GV</td><td rowspan=1 colspan=1>Sapo_02b</td><td rowspan=1 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>3×LoD</td></tr></table>

a Clinical Sample was used to establish the LoD for this assay. b Isolate obtained as RNA extract from a clinical sample, genogroup information provided by source laboratory.

# f. Analytical specificity (Exclusivity):

The potential for cross-reactivity between assays contained in the FilmArray GI Panel was evaluated by testing high concentrations of analyte in contrived stool samples. The organisms/viruses tested consisted of on-panel (identified by the GI Panel assays) and offpanel (not identified by the GI Panel assays) organisms/viruses.

On-panel organisms were tested to verify that they only react with the appropriate assays on the panel. On-panel exclusivity testing included 28 analytes were selected to evaluate the potential for cross-reactivity with other panel assays. This group of organisms was chosen such that at least one positive result would be obtained for each assay in the FilmArray GI Panel. Each organism was tested at a high concentration to show analytical specificity with all FilmArray GI Panel assays.

Organisms for off-panel testing were selected based on a combination of several factors including (1) relatedness to specific species detected by the GI Panel (near-neighbors), (2) clinical relevance, (3) likelihood of being present in stool specimens and (4) genetic similarity to GI Panel assay primers, as determined by in silico analyses during assay design. When empirical testing of these organisms was not performed, a separate organismspecific in silico analysis of whole genome sequence(s) directed against all GI Panel primers was attempted for reactivity predictions.

Results are presented for all organisms/viruses that were tested and received the expected FilmArray GI Panel test result(s) (no cross-reactivity, table below)

<table><tr><td colspan="3" rowspan="1">BACTERIA</td></tr><tr><td colspan="3" rowspan="1">Wet Tested</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectivia</td><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">Megamonas hypermegale</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Megasphaeara elsdenii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">Methanobrevibacter smithii</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Arcobacter butzleri</td><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">Photobacterium damselae</td></tr><tr><td colspan="1" rowspan="1">Arcobacter cryaerophilus</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Diffusely adherent E.coli</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Bacteroides thetaiotaomicron</td><td colspan="1" rowspan="1">Escherichia blattae</td><td colspan="1" rowspan="1">Proteus penneri</td></tr><tr><td colspan="1" rowspan="1">Bacteroides vulgatus</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentisa</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">Provedencia alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifiduma</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longuma</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Ruminococcus bromiia</td></tr><tr><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">Egglerthella lenta</td><td colspan="1" rowspan="1">Ruminococcus flavefaciensa</td></tr><tr><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Ruminococcus obeuma</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Selenomonas ruminantium</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hominis</td><td colspan="1" rowspan="1">Eubacterium cylindroides</td><td colspan="1" rowspan="1">Shewanella algae</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">Eubacterium rectale</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">Faecalibacterium prausnitzii</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Campylobacter ureolyticus</td><td colspan="1" rowspan="1">Hafnia alveib</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisaeb</td><td colspan="1" rowspan="1">Helicobacter fennelliae</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Yersinia bercovieri</td></tr><tr><td colspan="1" rowspan="1">Clostridium acetobutylicum</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Yersinia frederiksenit</td></tr><tr><td colspan="1" rowspan="1">Clostridium botulinum</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">Yersinia intermedia</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile non-toxigenicc</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">Yersinia mollaretii</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">Yersinia pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Yersinia rohdei</td></tr></table>

a Though not observed in this study, cross-reactivity of the Giardia lamblia assay with one or more Bifidobacterium and Ruminococcus species was observed in the clinical evaluation. Bifidobacterium and Ruminococcus species are listed as potential cross-reacting organisms. b Though not observed in this study, possible cross-reactivity of the ETEC 2 assay with Hafnia alvei and Cedeceae davisiae was observed in the clinical evaluation or predicted by in silico analysis. Halfnia alvei and Cedeceae davisiae are also listed as potentially cross-reactive organisms. c Two isolates of this species were tested for analytical specificity. d Though not observed in this study, in silico analysis indicates that cross-reactivity between Yersinia fredricksenii and the Yersinia entercolitica assay is possible at high concentrations. Y. fredricksenii is also listed as potentially cross-reactive organism.

Table - No Cross-Reactivity with FilmArray GI Panel Assays Observed (Tested) or Predicted by in silico Analysis   

<table><tr><td rowspan=1 colspan=2>PROTOZOA/PARASITES</td><td rowspan=1 colspan=1>FUNGI</td></tr><tr><td rowspan=1 colspan=1>Wet Tested</td><td rowspan=1 colspan=1>In silico Analysis Only</td><td rowspan=1 colspan=1>Tested</td></tr><tr><td rowspan=1 colspan=1>Babesia microtiBlastocystis hominisConidiobolus lachnodesConidiobolus lobatusEncephalitozoon hellemEncephalitozoon intestinalisEntamoeba gingivalisEntamoeba moshkovskiiGiardia murisPentatrichomonas hominisSchistosoma mansoniToxoplasma gondiiTrichomonas tenax</td><td rowspan=1 colspan=1>Ancylostoma duodenaleAscaris lumbricoidesBalantidium coliChilomastix mesniliDientamoeba fragilisEndolimax nanaEntamoeba coliEntamoeba hartmanniEntamoeba poleckiEnterobius vermicularisEnteromonas hominisIsospora belliNecator americanus</td><td rowspan=1 colspan=1>Aspergillus fumigatusCandida albicansCandida catenulatePenicillium marneffeiSaccharomyces boulardiSaccharomyces cerevisiae</td></tr></table>

Table - No Cross-Reactivity with FilmArray GI Panel Assays Observed (Tested) or Predicted by in silico Analysis   

<table><tr><td colspan="3">VIRUSES</td></tr><tr><td colspan="2">Wet Tested</td><td>In silico Analysis Only</td></tr><tr><td>Adenovirus A:31</td><td>Coronavirus 229E</td><td>Adenovirus G52</td></tr><tr><td>Adenovirus B:34</td><td>Coxsackievirus B3</td><td>Norovirus GIV</td></tr><tr><td>Adenovirus C:2</td><td>Cytomegalovirus (CMV)</td><td>Rotavirus B</td></tr><tr><td>Adenovirus D-37</td><td>Enterovirus 6</td><td>Rotavirus C</td></tr><tr><td>Adenovirus E:4a</td><td>Enterovirus 68</td><td></td></tr><tr><td>Astrovirus variant VA1</td><td>Hepatitis A</td><td></td></tr><tr><td>Astrovirus variant MLB</td><td>Herpes Simplex Type 2</td><td></td></tr><tr><td>Bocavirus Type 1</td><td>Rhinovirus 1A</td><td></td></tr></table>

Table - Observed or Predicted Cross-Reactivity between GI Panel Assays and Off-Panel Organisms   

<table><tr><td rowspan=1 colspan=1>FilmArray GI Panel Test Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism(s)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Entamoeba dispar</td></tr><tr><td rowspan=2 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Bifidobacterium spp.a</td></tr><tr><td rowspan=1 colspan=1>Ruminococcus spp.a</td></tr><tr><td rowspan=4 colspan=1>Enterotoxigenic E.coli (ETEC) It/st[ETEC 2 assay]</td><td rowspan=1 colspan=1>Citrobacter koseri</td></tr><tr><td rowspan=1 colspan=1>Citrobacter sedlakii</td></tr><tr><td rowspan=1 colspan=1>Hafnia alveia</td></tr><tr><td rowspan=1 colspan=1>Cedeceae davisiae</td></tr><tr><td rowspan=1 colspan=1>Salmonella b</td><td rowspan=1 colspan=1>E. coli with variant type III secretion proteinb</td></tr><tr><td rowspan=1 colspan=1>Vibrio(V. parahaemolyticus/V. vulnificus/V. cholerae)</td><td rowspan=1 colspan=1>Vibrio alginolyticusVibrio fluvialisVibrio mimicusGrimontia (formerly Vibrio) hollisae</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia frederikseniia,d</td></tr><tr><td rowspan=1 colspan=1>Yersinia kristenseniid</td></tr></table>

a Cross-reactivity was not observed when tested at high concentration $\overline { { { ( 1 . 5 { \times } 1 0 ^ { 9 } \mathrm { c e l l s } / \mathrm { m L } } } }$ ). However, cross reactivity was suspected or confirmed in clinical specimens and/or the potential for cross-reactivity is supported by in silico predictions. b Cross-reactivity resulting in false positive Salmonella results has not been observed in analytic or clinical testing. However, non-specific amplification products with Tm values close to the assay specific Tm range have been observed and the potential for false positive Salmonella test results exists. c Detected at concentrations near the Vibrio assay LoD. $^ { \mathrm { ~ d ~ } } Y .$ . kristensenii and Y. fredericksenii are difficult to distinguish from Y. entercolitica by standard laboratory method

# g. Assay cut-off:

The FilmArray GI Panel Melt Detector software determines whether a FilmArray GI assay result is positive or negative using a predefined algorithm that includes Tm values, fluorescence values, and analysis of melting curves.

The FilmArray GI Melt Detector software performs a set of basic calculations on the melt data to determine if a PCR reaction occurred in each well. If the melt profile indicates that a PCR product is present, then the analysis software calculates one or two Tm values, depending on the number of melt curves present in the data, and the Tm values are compared against an expected melt range for the associated assay. If the software determines that the melt is positive and the melt curve falls inside the assay’s specific melt range, then the curve is called positive. If the software determines that the melt is negative or that it is not in the appropriate range,

then the curve is called negative. All assays have three replicates. The melt curve for each replicate is given a positive or negative melt call by the Melt Detector.

For individual melt curves, the observed sensitivity and specificity, as compared to expert annotation, of the Melt Detector is greater than $9 8 . 9 2 \%$ and $9 9 . 9 9 \%$ , respectively. For the Analysis Software, the observed sensitivity and specificity, as compared to expert annotation, of the assay calls are greater than $9 9 . 3 4 \%$ and $9 9 . 9 9 \%$ , respectively.

# h. Fresh versus Frozen Study:

This study was performed to support the use of frozen specimens in the FilmArray GI Panel archived specimen and reproducibility testing; the test is not intended for use on frozen samples.

Stool specimens in Cary Blair were stored at ${ < } . 7 0 { } ^ { \circ } \mathrm { C }$ . The test panel contained at least one specimen for every organism on the panel (with the exception of Entamoeba histolytica). Specimens from all four prospective clinical study sites were included in the study. The median time interval between testing fresh and retesting from frozen was 60 days (Range: 10-90 days). Detection was $100 \%$ for most analytes but $9 0 { - } 9 4 \%$ for five analytes (Campylobacter, C. difficile, $P .$ . shigelloides, EPEC, and Cryptosporidium), and $54 \%$ for four analytes (Shigella/EIEC, G. lamblia, Norovirus) Based on study results, specific analytes for study testing were performed with fresh or frozen samples as appropriate.

A total of 203 FilmArray runs were attempted in this evaluation of 201 frozen specimens, 201 of which were completed $9 9 \%$ ; 201/203). There was one run failure each for a software error $( 0 . 5 \% )$ and a control failure $( 0 . 5 \% )$ .

# i. Interference

Substances that could be present in stool samples (preserved in Cary Blair transport medium) or introduced during sample handling were evaluated for their potential to interfere with assay performance. A potentially interfering substance (see tables below) was added to a contrived stool sample by spiking representative GI Panel organisms into negative sample matrix (individual or pooled donor stool in transport medium). Each contrived sample contained a mix of four different organisms, each present at approximately three times $( 3 \times )$ the limit of detection (LoD). Contrived samples without added potentially interfering substances served as positive controls.

Of the endogenous and exogenous substances tested, only the bovine-derived mucin gave unexpected results (EPEC was reported in samples that were not spiked with EPEC). An investigation found bacterial nucleic acid in the bovine-derived mucin used as the test substance, and it was determined that the unexpected results were due to EPEC contamination in the commercially prepared mucin.

Table - Potentially Interfering Endogenous and Exogenous Substances Tested   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=2>Exogenous Substances (including laboratory disinfectants)</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>Bacitracin</td><td rowspan=1 colspan=1>Glycerin</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>Doxycycline</td><td rowspan=1 colspan=1>Hydrocortisone</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>Loperamide hydrochloride</td></tr><tr><td rowspan=1 colspan=1>Fatty acids (palmitic acid)</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>Magnesium hydroxide</td></tr><tr><td rowspan=1 colspan=1>Fatty acids (stearic acid)</td><td rowspan=1 colspan=1>Naproxen sodium</td><td rowspan=1 colspan=1>Mineral oil</td></tr><tr><td rowspan=1 colspan=1>Bovine Mucin a</td><td rowspan=1 colspan=1>Bisacodyl</td><td rowspan=1 colspan=1>Phenylephrine hydrochloride</td></tr><tr><td rowspan=1 colspan=1>Human Bile</td><td rowspan=1 colspan=1>Bismuth subsalicylate</td><td rowspan=1 colspan=1>Sodium phosphate</td></tr><tr><td rowspan=1 colspan=1>Human Urine</td><td rowspan=1 colspan=1>Calcium carbonate</td><td rowspan=1 colspan=1>Nonoxynol-9</td></tr><tr><td rowspan=1 colspan=1>Human stool (overfill of Cary</td><td rowspan=1 colspan=1>Docusate sodium</td><td rowspan=1 colspan=1>Bleach</td></tr><tr><td rowspan=1 colspan=1>Blair vial)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Ethanol</td></tr></table>

a Unexpected EPEC detected results reported. The presence of EPEC nucleic acid in test material was confirmed by independent PCR assays, indicating the unexpected results were caused by contamination of the mucin with EPEC.

No inhibition or unexpected test results were obtained in the presence of high concentrations of potentially competing microorganisms (on-panel or off-panel organisms; see table below). However, Rotavirus A Detected results were reported when Rotavirus A reassortant strains used in the manufacturing of Rotavirus A vaccines were tested (see table below). Rotavirus A vaccine may be shed in stool following oral administration and Rotavirus A will be detected by the FilmArray GI Panel if vaccine is present in the test sample.

Table- Potentially Interfering or Competing Organisms and Vaccine Material Tested   

<table><tr><td rowspan=1 colspan=1>Off-Panel Organisms</td><td rowspan=1 colspan=1>On-Panel Organisms</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Adenovirus F41</td></tr><tr><td rowspan=1 colspan=1>Bacteroides vulgatus</td><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC)</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium bifidum</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus 87</td></tr><tr><td rowspan=1 colspan=1>Non-pathogenic E. coli</td><td rowspan=1 colspan=1>RotaTeq Rotavirus A Vaccine Components</td></tr><tr><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Rotavirus reassortant WC3:2-5, R574(9) [ATCC VR-2195]a</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces boulardii</td><td rowspan=1 colspan=1>Rotavirus reassortant WI79-4,9 [ATCC VR-2415]a</td></tr></table>

a Reactivity with the FilmArray Rotavirus A assay expected.

Contrived stool samples prepared in various enteric and fixative-containing transport media, including Cary Blair (see table below), were evaluated for the potential of different media to interfere with the accuracy of FilmArray GI Panel test results. No interference was observed for samples collected in Protocol Cary Blair or other brands of enteric transport media (Para-Pak Enteric Plus and Para-Pak C&S media; performance of the FilmArray GI Panel has not been established in these media). However, accurate detection of analytes was impaired (false negative results) for samples prepared in media containing fixatives, particularly those containing formalin.

Table Transport Media Tested   

<table><tr><td colspan="3">Enteric Transport Media  No Interference Observed</td></tr><tr><td>PROTOCOLTM Cary Blair Para-Pak Enteric Plusa</td><td colspan="2">Para-Pak C&amp;Sa</td></tr><tr><td>Fixative-containing Transport Media - Interference Observeda</td><td colspan="2"></td></tr><tr><td>Modified (Cu) PVA Fixative Para-Pak ECOFIX Fixative</td><td>Para-Pak 10% Formalin Fixativeb Para-Pak LV-PVA Fixative</td><td>Para-Pak SAF Fixativea Para-Pak Zn-PVA Fixative</td></tr></table>

aPerformance has not been established in these media bImpaired detection of analytes (false negative results) observed in formalin containing media.

# j. Carryover Study:

To demonstrate sample-to-sample carryover does not occur when the recommended testing and cleaning procedures are followed, five sets of samples were tested with the FilmArray GI Panel over several days such that testing of one positive, high analyte level sample (Plesiomonas shigelloides, Escherichia coli (STEC), Yersinia enterocolitica, Giardia intestinalis, or Adenovirus F41) was followed directly by testing of a negative sample (containing no analyte).

In accordance with recommended procedures, each sample was prepared and loaded into a GI pouch one at a time using the provided pouch loading implements. Samples were loaded in a work area (biosafety cabinet or hood) that was separated from the location of the FilmArray instrument(s) and the loading block was cleaned between each pouch. For each set of samples, the negative sample was loaded in the same workspace, using the same Pouch Loading Station and tested directly after the high positive sample using the same FilmArray instrument. No false positive results were observed during testing of five sets of a high positive sample followed directly by a negative sample

# k. Analysis of Pouch Hydration

The FilmArray GI pouch is manufactured to contain a vacuum that draws in the required amount of Hydration Solution and Sample/Buffer Mix when the seals are broken by the cannula-tipped cartridges in each of the pouch ports. If either the Hydration Solution or the Sample/Buffer Mix is not drawn into the pouch, the operator is instructed to discard the faulty pouch and obtain a new pouch to test the specimen. A total of 1591 pouches were used to achieve successful results for the 1556 valid specimens (note: this includes 14 pouches used for retests after initial pouch/instrument failures). Twenty two (22; $1 . 4 \% )$ of 1591 pouches failed to hydrate or pull in sample/buffer mix and were discarded; successful hydration was achieved on the subsequent pouch. Overall, $9 8 . 6 \%$ (1569/1591) of pouches used to test study specimens loaded successfully. No specimens were lost due to pouch hydration failures.

Table - Analysis of pouch hydration   

<table><tr><td rowspan=1 colspan=1>TotalDistinctPouches Used</td><td rowspan=1 colspan=1>LoadedSuccessfully</td><td rowspan=1 colspan=1>Failure toHydrate/PullSample</td></tr><tr><td rowspan=2 colspan=1>1591</td><td rowspan=1 colspan=1>1569</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>1.4%</td></tr></table>

p. 44

2. Comparison studies:

a. Method comparison with predicate device: N/A b. Matrix comparison: N/A

3. Clinical studies:

The clinical performance of the FilmArray GI Panel was established during a multi-center study conducted at four geographically distinct U.S. study sites (Pacific, North Central, Great Lakes, and Northeast regions) between May and September, 2013. A total of 1578 prospective residual stool specimens in Cary Blair transport media were acquired for the clinical study; 22 of these were excluded. The most common reasons for exclusion were that a valid external control was not completed on the day of testing, that the specimen was not plated to all of the appropriate bacterial culture media required for the reference method, or that the specimen was beyond four days from the date of collection. The final data set consisted of 1556 specimens.

The tables below provides a summary of the number of clinician ordered tests per site, the types of test ordered per site and the demographic information for the 1556 specimens included in the prospective study.

Table - Clinician Ordering Practices: number of tests ordered\*   

<table><tr><td rowspan=1 colspan=1>Number ofadditional tests</td><td rowspan=1 colspan=1>Site 1(n=253)</td><td rowspan=1 colspan=1>Site 2(n=366)</td><td rowspan=1 colspan=1>Site 3(n=793)</td><td rowspan=1 colspan=1>Site 4(n=144)</td><td rowspan=1 colspan=1>Total(n=1556)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>362</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>492</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>478</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>157</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5</td></tr></table>

\*Excludes standard testing: Yersinia at Site 3 and Shiga toxin at sites 1, 3, and 4.

Table - Clinician ordering practices: types of tests ordered\*   

<table><tr><td rowspan=1 colspan=1>Test Ordered</td><td rowspan=1 colspan=1>Site 1(n=253)</td><td rowspan=1 colspan=1>Site 2(n=366)</td><td rowspan=1 colspan=1>Site 3(n=793)</td><td rowspan=1 colspan=1>Site 4(n=144)</td><td rowspan=1 colspan=1>Total(n=1556)</td></tr><tr><td rowspan=1 colspan=1>Vibrio Culture</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Yersinia Culture</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>O&amp;P Exam</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>548</td></tr><tr><td rowspan=1 colspan=1>Crypto/Giardia DFA</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>236</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>656</td></tr><tr><td rowspan=1 colspan=1>Virology</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>708</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>105</td></tr></table>

\*Excludes standard testing: Yersinia at Site 3 and Shiga toxin at sites 1, 3, and 4.

Table - Demographic Summary for Prospective FilmArray GI Panel Clinical Evaluation   

<table><tr><td rowspan=1 colspan=2>Prospective Study Specimens</td></tr><tr><td rowspan=1 colspan=1>Total Specimens</td><td rowspan=1 colspan=1>1556</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>718 (46%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>838 (54%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>121 (8%)</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>418 (27%)</td></tr><tr><td rowspan=1 colspan=1>6-12 years</td><td rowspan=1 colspan=1>193 (12%)</td></tr><tr><td rowspan=1 colspan=1>13-21| years</td><td rowspan=1 colspan=1>240 (15%)</td></tr><tr><td rowspan=1 colspan=1>22-64 years</td><td rowspan=1 colspan=1>411 (26%)</td></tr><tr><td rowspan=1 colspan=1>65+ years</td><td rowspan=1 colspan=1>173 (11%)</td></tr><tr><td rowspan=1 colspan=1>Status</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>1350 (87%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>164 (11%)</td></tr><tr><td rowspan=1 colspan=1>Emergency</td><td rowspan=1 colspan=1>42 (3%)</td></tr></table>

The performance of the FilmArray GI Panel was evaluated by comparing the FilmArray GI Panel test result for each member of the panel with the appropriate comparator/reference methods shown in the table below. For each comparator PCR assay, each extracted sample was first tested with an inhibition control.

Table -FilmArray GI Panel Study Comparator/Reference Methods   

<table><tr><td rowspan=1 colspan=1>FilmArray Test Results</td><td rowspan=1 colspan=1>Reference/Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=6 colspan=1>Stool cultureb(Blood agar, Blood agar with Ampicillin,MacConkey agar, Sorbitol-MacConkey agar, GNbroth + Hektoen enteric agar, Campylobacter agar,Cefsulodin-IrgasanTM-Novobiocin agar, andThiosulfate Citrate Bile Salts agar) with standardmanual and automated microbiological/biochemicalidentification methods</td></tr><tr><td rowspan=1 colspan=1>E. coli O157a</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>b</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Vibrio and V. cholerae</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>STEC (stx1/2)</td><td rowspan=13 colspan=1>PCR with Bi-directional Sequencing</td></tr><tr><td rowspan=1 colspan=1>ETEC</td></tr><tr><td rowspan=1 colspan=1>EPECc</td></tr><tr><td rowspan=1 colspan=1>EIEC/Shigellad</td></tr><tr><td rowspan=1 colspan=1>EAEC</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GIIe</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile toxin A/B</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td></tr></table>

<table><tr><td rowspan=1 colspan=1>FilmArray Test Results</td><td rowspan=1 colspan=1>Reference/Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td></tr></table>

a The $E .$ . coli O157 comparator method data were only used to determine the accuracy of the FilmArray determination of $E .$ . coli O157 detected or not detected for specimens in which FilmArray detected STEC.

b Any bacteria isolated from stool culture that could not be identified to the species level by laboratory methods were sequenced using an assay capable of providing species information (e.g., 16S).

c A result for EPEC is only reported in the absence of STEC (same algorithm as FilmArray).

dShigella may be identified by routine culture methods; however, culture detection will be reported for informational purposes only.

e CDC Calicinet assays (non-sequenceable) were used for the comparator method for Norovirus.

f Sapovirus comparator assays consisted of one well-validated, sequenceable assay and one published assay that was not sequenceable.

g G. lamblia comparator assays consisted of one well-validated, sequenceable assay and one published assay that was not sequenceable.

h PCR assays were designed to amplify different sequences than those targeted by FilmArray GI. Positive results for sequenceable assays required a sequencing result of adequate quality to match a sequence of the expected organism/gene deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with an acceptable E-value.

A total of 1556 specimens were evaluated in this study. Of these specimens, 832 $( 5 3 . 5 \% )$ were positive for at least one analyte. Clinical sensitivity or positive percent agreement (PPA) was calculated as $100 \%$ x $\left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the FilmArray GI Panel and reference/comparator method had a positive result for this specific analyte, and false negative (FN) indicates that the FilmArray result was negative while the comparator result was positive. Specificity or negative percent agreement (NPA) was calculated as $100 \% \mathbf { x }$ $\left( \mathrm { T N } / \left( \mathrm { T N } + \mathrm { F P } \right) \right)$ . True negative (TN) indicates that both the FilmArray GI Panel and the reference/comparator method had negative results, and a false positive (FP) indicates that the FilmArray GI Panel result was positive but the comparator result was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. The results are summarized in the table below.

Table - FilmArray GI Clinical Performance Summary   

<table><tr><td rowspan=2 colspan=2>Bacteria</td><td rowspan=1 colspan=4>Sensitivity/PPAa</td><td rowspan=1 colspan=3>Specificity/NPAa</td></tr><tr><td rowspan=1 colspan=2>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=2>Campylobacter (C. jejuni/C. coli/C.upsaliensis)</td><td rowspan=1 colspan=2>34/35b</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.1-99.9</td><td rowspan=1 colspan=1>1497/1521b</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>97.7-99.0</td></tr><tr><td rowspan=1 colspan=2>Clostridium difficile toxin A/Ba</td><td rowspan=1 colspan=2>163/165c</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>95.7-99.9</td><td rowspan=1 colspan=1>1350/1391c</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>96.0-97.9</td></tr><tr><td rowspan=1 colspan=2>Plesiomonas shigelloides</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29.2-100</td><td rowspan=1 colspan=1>1538/1553d</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>98.4-99.5</td></tr><tr><td rowspan=1 colspan=2>Salmonella</td><td rowspan=1 colspan=2>31/31</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.8-100</td><td rowspan=1 colspan=1>1519/1525</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.1-99.9</td></tr><tr><td rowspan=1 colspan=2>Vibrio (V. parahaemolyticus/V.vulnificus/V. cholerae)</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1554/1556</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=2>Vibrio cholerae</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1555/15568</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=2>Yersinia enterocolitica</td><td rowspan=1 colspan=2>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1555/1555</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=2 colspan=2>Diarrheagenic E. coli/Shigella</td><td rowspan=1 colspan=4>Positive Percent Agreement (PPA)a</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)a</td></tr><tr><td rowspan=1 colspan=2>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=2>Enteroaqggregative E. coli (EAEC)</td><td rowspan=1 colspan=2>82/83</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>93.5-100</td><td rowspan=1 colspan=1>1446/1473h</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>97.3-98.8</td></tr><tr><td rowspan=1 colspan=2>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=2>314/317</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.3-99.8</td><td rowspan=1 colspan=1>1167/1201</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>96.1-98.0</td></tr><tr><td rowspan=1 colspan=2>Enterotoxigenic E. coli (ETEC) lt/</td><td rowspan=1 colspan=2>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>84.6-100</td><td rowspan=1 colspan=1>1525/1534</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.9-99.7</td></tr><tr><td rowspan=1 colspan=2>Shiga-like toxin-producing E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=2>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.4-100</td><td rowspan=1 colspan=1>1518/1523k</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.2-99.9</td></tr><tr><td rowspan=1 colspan=2>E. coli O157a</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29.2-100</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.1-99.9</td></tr><tr><td rowspan=1 colspan=2>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=2>47/49</td><td rowspan=1 colspan=1>95.9</td><td rowspan=1 colspan=1>86.0-99.5</td><td rowspan=1 colspan=1>1505/1507</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=2 colspan=1>Parasites</td><td rowspan=1 colspan=5>Positive Percent Agreement (PPA)a</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)a</td></tr><tr><td rowspan=1 colspan=2>TP/(TP + FN)</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=2>18/18</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>81.5-100</td><td rowspan=1 colspan=1>1532/1538m</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.2-99.9</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=2>19/19</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>1537/1537</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1556/1556</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=2>20/20</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>83.2-100</td><td rowspan=1 colspan=1>1529/1536&quot;</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>99.1-99.8</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Viruses</td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)a</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)a</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1>42/44°</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>84.5-99.4</td><td rowspan=1 colspan=1>1499/1512°</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>98.5-99.5</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>59.0-100</td><td rowspan=1 colspan=1>1548/1549P</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>52/59</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>84.9-98.9</td><td rowspan=1 colspan=1>1483/15019</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.1-99.3</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.1-100</td><td rowspan=1 colspan=1>1538/1550&quot;</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.7-99.6</td></tr><tr><td rowspan=1 colspan=1>Sapovirus (Genogroups I, II, IV, andV)</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.3-100</td><td rowspan=1 colspan=1>1497/1510s</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>98.5-99.5</td></tr></table>

a C. difficile performance is reported as positive percent agreement and negative percent agreement, and $E$ . coli O157 performance is reported as sensitivity/specificity, in contrast to the headings of their respective sections. The performance measures of sensitivity and specificity only refer to those analytes for which the gold-standard bacterial culture was used as the reference method; Campylobacter, $E$ . coli O157, Plesiomonas shigelloides, Salmonella, Vibrio, Vibrio cholerae, and Yersinia enterocolitica. Performance measures of positive percent agreement (PPA) and negative percent agreement (NPA) refer to all other analytes, for which PCR/sequencing assays were used as comparator methods.

b Campylobacter jejuni subsp. doylei was identified in the single false negative specimen using bi-directional sequence analysis. Campylobacter was detected in 19/24 false positive specimens using bi-directional sequence analysis.

c C. difficile was detected in 1/2 false negative specimens and 41/41 false positive specimens using bi-directional sequence analysis.

$^ { \mathrm { ~ d ~ } } P .$ . shigelloides was detected in 15/15 false positive specimens using bi-directional sequence analysis.

monella was detected in 6/6 false positive specimens using bi-directional sequence analysis.

f Vibrio was detected in 2/2 false positive specimens using bi-directional sequence analysis.

i EPEC was detected in 23/34 false positive specimens using bi-directional sequence analysis.

j ETEC was detected in 6/9 false positive specimens using bi-directional sequence analysis. The three remaining false positive results were determined to have been caused by cross-reactivity with Citrobacter koseri (2 instances), and Hafnia alvei (1 instance). These bacteria contain a variant of the fliP gene with sequence similarity to assay primers. k STEC was detected in $5 / 5$ false positive specimens using bi-directional sequence analysis.

li O157 was detected in the single false positive specimen using bi-directional sequence analy

Cryptosporidium was detected in $6 / 6$ false positive specimens using bi-directional sequence analysis.

n G. lamblia was detected in $4 / 7$ false positive specimens using bi-directional sequence analysis. Two false positive results appear to be caused by cross-reactivity with Bifidobacterium longum and Ruminococcus callidus.

o Adenovirus was detected in 1/2 false negative specimens and 11/13 false positive specimens using bi-directional sequence analysis

Astrovirus was detected in the single false positive specimen using bi-directional sequence analysis.

The FilmArray GI system detected Norovirus in 1/3 false negative specimens when retested. Norovirus was detected in 1/2 remaining false negative specimens and 8/18 false positive specimens using bi-directional sequence analysis.

r Rotavirus A was detected in 11/12 false positive specimens using bi-directional sequence analysis.

s Sapovirus was detected in 12/13 false positive specimens using bi-directional sequence analysis.

standard methods did not provide a species identification, bi-directional sequencing was used to identify the species of the isolate. Stratification of performance by species/serovar is presented below.

For Vibrio, no organisms were isolated by the culture methods; however, bi-directional sequencing from the original specimens identified one $V .$ . parahaemolyticus and one $V .$ . cholerae.

Table –Campylobacter Clinical Performance Stratified by Species   

<table><tr><td rowspan=1 colspan=1>Campylobacter speciesa</td><td rowspan=1 colspan=1>Sensitivity</td></tr><tr><td rowspan=1 colspan=1>C. jejunib</td><td rowspan=1 colspan=1>31/31 (100%)</td></tr><tr><td rowspan=1 colspan=1>C. coli</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=1>C. jejuni subsp. Doylei</td><td rowspan=1 colspan=1>0/1 (0%)</td></tr><tr><td rowspan=1 colspan=1>C. upsaliensis</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Campylobacter</td><td rowspan=1 colspan=1>34/35 (97.1%)95%CI = 81.3-99.3%</td></tr></table>

a Fifteen (15) Campylobacter were not speciated by the source laboratory and were subject to sequencing of the cadF gene. This method identified $1 1 ~ C .$ . jejuni, two $C$ . coli, one $C .$ . jejuni subsp. doylei, and one $C .$ upsaliensis. b Two $C .$ . jejuni were originally identified by the source lab as “Campylobacter species”. Sequencing of the isolates provided by the laboratory identified them as $C .$ . jejuni. However, molecular testing of the specimen from which the isolates were obtained also detected the presence of $C .$ . upsaliensis, representing co-infection by these two species.

Table - Salmonella Clinical Performance Stratified by Species/Serovar   

<table><tr><td rowspan=1 colspan=1>Salmonella species/serovar</td><td rowspan=1 colspan=1>Sensitivity</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Enteritidis</td><td rowspan=1 colspan=1>7/7 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Typhimurium (i:-)</td><td rowspan=1 colspan=1>7/7 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Typhimurium</td><td rowspan=1 colspan=1>3/3 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Javiana</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Newport</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Agbeni</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Berta</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Ealing</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Gaminara</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Infantis</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Mbandaka</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Miami</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Muenchen</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Paratyphi B var L-Tartrate</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. enterica ser. Thompson</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Overall Salmonella</td><td rowspan=1 colspan=1>31/31 (100%)95%CI = 88.8-100%</td></tr></table>

The FilmArray GI Panel reported multiple organism detections (i.e., mixed infections) for a total of 262 specimens. This represents $3 1 . 5 \%$ of positive specimens (262/832) and $1 6 . 8 \%$ of all specimens (262/1556). The majority of multiple detections $( 1 9 9 / 2 6 2 ; 7 6 . 0 \% )$ . contained two organisms, while $1 9 . 1 \%$ (50/262) contained three organisms, $3 . 4 \%$ (9/262)

contained four organisms, $1 . 1 \%$ (3/262) contained five organisms, and $0 . 4 \%$ (1/262) contained six organisms. The three organisms that were most prevalent in co-infections were also the three most prevalent organisms in the study as a whole (i.e., EPEC, $C .$ . difficile, and EAEC). Out of the 262 specimens with multiple detections, 144 specimens $( 5 5 . 0 \%$ ; 144/262) were concordant with the reference methods. One hundred eighteen specimens $( 4 5 . 0 \%$ ; 118/262) contained one or more organisms that had not been detected by the reference/comparator methods (i.e., 139 false positive results); however, bidirectional sequence analysis confirmed the presence of the analyte for $8 8 . 5 \%$ (123/139) of the discordant results.

Table - Prevalence of Analytes in Mixed Infections as determined by the FilmArray GI Panel   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Prevalence in Mixed InfectionsN = 262</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>11.5%</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile toxin A/B</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>41.6%</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6.1%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=3>Diarrheagenic E. coli/Shigella</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>25.6%</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>60.7%</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>9.9%</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin-producing E. coli (STEC)stx1/stx2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Shigella / Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=2>Parasites</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.2%</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=3>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>13.0%</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>16.4%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>12.6%</td></tr></table>

The most prevalent mixed infection was $C .$ . difficile toxinA/B with EPEC $( 2 . 0 \%$ of all specimens; 32/1556) followed by EAEC with EPEC ( $1 \%$ of all specimens; 15/1556); as previously stated

these were the most prevalent organisms detected in the study. Mixed infections were observed for all combinations of analyte classes (e.g. bacteria with viruses, diarrheagenic $E$ . coli/Shigella with parasites) and co-infections were observed within classes (e.g. three diarrheagenic $E$ . coli/Shigella combined; ETEC, EAEC, and STEC).

Table - Most Prevalent Multiple Detection Combinations $2 5$ instances) as Determined by the FilmArray GI Panel   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B + EPEC</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>EAEC + EPEC</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + EPEC</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>EPEC + Sapovirus</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + EPEC</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>EPEC + Norovirus GI/GII</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B + EAEC</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B + Norovirus GI/GII</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B + STEC stx1/stx2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>EPEC + ETEC lt/st</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>EPEC + G. lamblia</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>EPEC + Shigella/EIEC</td><td rowspan=1 colspan=1>5</td></tr></table>

The overall success rate for initial specimen tests in the prospective study was $9 9 . 4 \%$ (1544/1557). Four tests were incomplete due to software errors (3) or a user aborted run (1), and nine tests were invalid due to pouch control failures. Of the nine pouch control failures observed in the prospective clinical study, seven were attributed to the RNA process control and two were due to the PCR2 control. There were no instances of both RNA process and PCR2 control failure in the same pouch. All specimens but one were retested within four days of specimen collection and were successful after a single retest, for a final success rate of $9 9 . 9 \%$ (1556/1557).

# Testing of Preselected Archived Specimens

Several analytes were either not encountered or had a low prevalence in the clinical study. To supplement the results of the prospective clinical study, an evaluation of 222 preselected archived specimens was performed. These specimens were archived clinical specimens that were selected because they had previously tested positive for one of the following analytes: $E$ . coli O157, $P$ . shigelloides, Y. enterocolitica, Vibrio, Astrovirus, Rotavirus, and $E$ . histolytica, or had been negative in previous laboratory testing. Prior to testing with the FilmArray GI Panel, the presence (or absence for negative specimens) of the expected analytes was verified in each specimen using analyte-specific PCR followed by bi-directional sequencing.

The specimens were organized into “test panels” and randomized such that the users performing the FilmArray GI Panel testing were blinded as to the expected test result. A summary of the available demographic information of the tested samples and the results of the FilmArray GI testing are presented in the tables below

Table - Demographic Summary for Preselected Archived Specimens   

<table><tr><td rowspan=1 colspan=2>Preselected Archived Specimens</td></tr><tr><td rowspan=1 colspan=1>Total Specimens</td><td rowspan=1 colspan=1>222</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>57 (25.7%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>48 (21.6%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>117 (52.7%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>12 (5.4%)</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>36 (16.2%)</td></tr><tr><td rowspan=1 colspan=1>6-12 years</td><td rowspan=1 colspan=1>15 (6.8%)</td></tr><tr><td rowspan=1 colspan=1>13-21 years</td><td rowspan=1 colspan=1>11 (5%)</td></tr><tr><td rowspan=1 colspan=1>22-64 years</td><td rowspan=1 colspan=1>18 (8.1%)</td></tr><tr><td rowspan=1 colspan=1>65+ years</td><td rowspan=1 colspan=1>4 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>126 (56.8%)</td></tr></table>

Table - FilmArray GI Panel Archived Specimen Performance Data Summary   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>73.5-100</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.6-100</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1-100</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>63.1-100</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9-100</td></tr><tr><td rowspan=1 colspan=7>Diarrheagenic E. coli/Shigella</td></tr><tr><td rowspan=1 colspan=1>(STEC) E. coli O157a</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=7>Parasites</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>82.8-99.9</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.6-100</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>15.8-100</td><td rowspan=1 colspan=1>123/123</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.8-100</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.6-100</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>31/32</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>83.8-99.9</td><td rowspan=1 colspan=1>91/91</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.0-100</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.1-100</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.5-100</td></tr></table>

a No non-O157 STEC were included in the data set; therefore, negative percent agreement (NPA) could not be calculated for $E$ . coli O157.

# Testing of Contrived Specimens

Several analytes, such as Entamoeba histolytica, are so rare that both prospective and archived testing efforts were insufficient to demonstrate system performance. To supplement the prospective and archived data, an evaluation of contrived specimens was performed. Surrogate clinical specimens were prepared using residual specimens from the prospective clinical study that had previously tested negative for all GI panel analytes by FilmArray GI and comparator methods. Specimens were spiked at clinically relevant levels using five different quantified strains for each organism (or unspiked; at least 50 of each). The analyte status of each contrived specimen was blinded to the users analyzing the specimens, and the specimens were randomized before testing. The results of the FilmArray GI testing are presented in the table below:

Table - FilmArray GI Panel Performance using Contrived Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>44/50</td><td rowspan=1 colspan=1>88.0</td><td rowspan=1 colspan=1>75.7-95.5</td><td rowspan=1 colspan=1>75/75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.2-100</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>70/70</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.9-100</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.5-100</td></tr><tr><td rowspan=1 colspan=1>Vibrioa</td><td rowspan=1 colspan=1>112/115</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>92.6-99.5</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>V. choleraeb</td><td rowspan=1 colspan=1>55/65</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>73.5-92.4</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7-100</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.5-100</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7-100</td></tr></table>

a Includes $6 4 / 6 5 ~ V .$ . cholerae (five different strains were used in spiking; one specimen spiked near the assay limit of detection was not detected) and 48/50 non- $V$ . cholerae (four $V .$ parahaemolyticus strains and one $V .$ . vulnificus strain were used in spiking; two specimens spiked with $V .$ . parahaemolyticus near the assay limit of detection were not detected).

b Ten (10) of these specimens were spiked with an isolate which was found to have a highly divergent toxR gene that was not present in the NCBI database and non-reactive with the FilmArray GI Panel V. cholerae assay. The FilmArray GI Panel Vibrio assay was positive for nine of these specimens.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

The prevalence of detected analytes has been stratified in the tables below.

Table - Expected Values for E. coli O157 in presence of STEC   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall (n=38)</td><td rowspan=1 colspan=2>&lt;1(n=1)</td><td rowspan=1 colspan=2>1-5(n=24)</td><td rowspan=1 colspan=2>6-12(n=2)</td><td rowspan=1 colspan=2>13-21(n=4)</td><td rowspan=1 colspan=2>22-64(n=5)</td><td rowspan=1 colspan=2>65+(n=2)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>E. coli O157in presence of STEC</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Table - Expected Values for EPEC in absence of STEC   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall(n=1518)</td><td rowspan=1 colspan=2>&lt;1(n=120)</td><td rowspan=1 colspan=2>1-5(n=394)</td><td rowspan=1 colspan=2>6-12(n=191)</td><td rowspan=1 colspan=2>13-21(n=236)</td><td rowspan=1 colspan=2>22-64(n=406)</td><td rowspan=1 colspan=2>65+(n=171)</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>EPEC inabsence ofSTEC</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>22.9%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>39.3%</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>23.6%</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>19.5%</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>13.5%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>9.9%</td></tr></table>

Table - Prevalence of Detected Analytes Stratified by Age Group ( $\mathbf { n } \# =$ Number; $\mathbf { E V } =$ Expected Value)   

<table><tr><td rowspan=1 colspan=2>Overall(n=1556)</td><td rowspan=1 colspan=2>&lt;1(n=121)</td><td rowspan=1 colspan=2>1-5(n=418)</td><td rowspan=1 colspan=2>6-12(n=193)</td><td rowspan=1 colspan=2>13-21(n=240)</td><td rowspan=1 colspan=2>22-64(n-411)</td><td rowspan=1 colspan=2>65+(n=173)</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>3.70%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>6.20%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.50%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>4.60%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>5.20%</td></tr><tr><td rowspan=1 colspan=1>C. difficile toxin A/B</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>13.10%</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>40.50%</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>15.80%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>9.30%</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>13.80%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>7.10%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>5.20%</td></tr><tr><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.10%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.70%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2.40%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4.10%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.10%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.70%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.30%</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Vibrio Cholerae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.20%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E.coli (EAEC)</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>7.00%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>7.40%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>8.10%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>10.40%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>7.10%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6.10%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.30%</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E.coli (EPEC)</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>22.40%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24.80%</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>37.10%</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>23.30%</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>19.20%</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>13.40%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>9.80%</td></tr><tr><td rowspan=1 colspan=1>Entertoxigenic E.coli (ETEC)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>2.00%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.60%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.10%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.20%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.30%</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin-producing E. coli(STEC)</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2.40%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5.70%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.00%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.20%</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.70%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Shigella |EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>3.10%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>7.40%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.60%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.10%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.20%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.60%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.50%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.60%</td></tr><tr><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.20%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3.50%</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.40%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.20%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3.50%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9.90%</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>8.60%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.00%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>4.50%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12.40%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>7.40%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.60%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.90%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.20%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.70%</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9.10%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.60%</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>3.80%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9.90%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>7.40%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.60%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.70%</td></tr></table>

A study of 100 asymptomatic volunteers was conducted. The FilmArray GI Panel gave a negative result for 67 $( 6 7 \% )$ of specimens. Of the 33 positive specimens, 27 $( 8 1 . 8 \% )$ were positive for one analyte and six $( 1 8 . 2 \% )$ were positive for two analytes.

Table - $9 5 \%$ Confidence Intervals for Prevalence Rates in the Prospective Clinical Study and Asymptomatic Volunteers Stratified   

<table><tr><td rowspan="2">Observation</td><td colspan="3">Prospective Study (n = 1556)</td><td colspan="3">Asymptomatic Volunteers (n = 100)</td></tr><tr><td>Obs %</td><td>CI95 lower bound</td><td>CI95 upper bound</td><td>Obs %</td><td>CI95 lower bound</td><td>CI95 upper bound</td></tr><tr><td>Total Positives</td><td>54.3</td><td>51.8</td><td>56.8</td><td>33</td><td>23.9</td><td>43.1</td></tr><tr><td>Clostridium difficile toxin A/B</td><td>13.1</td><td>11.5</td><td>14.9</td><td>5</td><td>1.6</td><td>11.3</td></tr><tr><td>Plesiomonas shigelloides</td><td>1.2</td><td>0.7</td><td>1.8</td><td>1</td><td>0.02</td><td>5</td></tr><tr><td>EAEC</td><td>7</td><td>5.8</td><td>8.4</td><td>7</td><td>2.8</td><td>13.9</td></tr><tr><td>EPEC*</td><td>22.4</td><td>20.3</td><td>24.5</td><td>18</td><td>11</td><td>27</td></tr><tr><td>Giardia lamblia</td><td>1.7</td><td>1.2</td><td>2.5</td><td>3</td><td>0.06</td><td>8.5</td></tr><tr><td>Adenovirus F 40/41</td><td>3.5</td><td>2.7</td><td>4.6</td><td>2</td><td>0.02</td><td>7</td></tr><tr><td>Rotavirus A</td><td>1.2</td><td>0.7</td><td>1.8</td><td>1</td><td>0.02</td><td>5</td></tr><tr><td>Sapovirus</td><td>3.8</td><td>2.9</td><td>4.9</td><td>2</td><td>0.02</td><td>7</td></tr></table>

\* Asymptomatic carriage of EPEC has also been documented with some studies reporting similar rates to symptomatic individuals

Table – FilmArray GI Detections in Asymptomatic Volunteers Stratified by Age Range   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>1-5</td><td rowspan=1 colspan=1>6-12</td><td rowspan=1 colspan=1>13-21</td><td rowspan=1 colspan=1>22-64</td><td rowspan=1 colspan=1>65+</td></tr><tr><td rowspan=1 colspan=1>All Negative</td><td rowspan=1 colspan=1>5 (62.5%)</td><td rowspan=1 colspan=1>17 (58.6%)</td><td rowspan=1 colspan=1>11 (73.3%)</td><td rowspan=1 colspan=1>8 (57.1%)</td><td rowspan=1 colspan=1>20 (74.1%)</td><td rowspan=1 colspan=1>6 (85.7%)</td></tr><tr><td rowspan=1 colspan=1>Single Infection</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>11 (37.9%)</td><td rowspan=1 colspan=1>3 (20%)</td><td rowspan=1 colspan=1>5 (35.7%)</td><td rowspan=1 colspan=1>7 (25.9%)</td><td rowspan=1 colspan=1>1 (14.3%)</td></tr><tr><td rowspan=1 colspan=1>Co-Infections</td><td rowspan=1 colspan=1>3 (37.5%)</td><td rowspan=1 colspan=1>1 (3.4%)</td><td rowspan=1 colspan=1>1 (6.7%)</td><td rowspan=1 colspan=1>1 (7.1%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile toxin A/B</td><td rowspan=1 colspan=1>1 (12.5%)</td><td rowspan=1 colspan=1>2 (6.9%)</td><td rowspan=1 colspan=1>1 (6.7%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.7%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (6.7%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>1 (12.5%)</td><td rowspan=1 colspan=1>2 (6.9%)</td><td rowspan=1 colspan=1>1 (6.7%)</td><td rowspan=1 colspan=1>1 (7.1%)</td><td rowspan=1 colspan=1>1 (3.7%)</td><td rowspan=1 colspan=1>1 (14.3%)</td></tr><tr><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>1 (12.5%)</td><td rowspan=1 colspan=1>6 (20.7%)</td><td rowspan=1 colspan=1>2 (13.3%)</td><td rowspan=1 colspan=1>4 (28.6%)</td><td rowspan=1 colspan=1>5 (18.5%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.4%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>2 (14.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F 40/41</td><td rowspan=1 colspan=1>2 (25%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1 (12.5%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>2 (6.9%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

# N. Instrument Name:

FilmArray® Instrument

# O. System Descriptions:

1. Modes of Operation: See Device Description (Section I) above

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification: The Sample ID can be entered manually or scanned in by using the FilmArray barcode scanner.

4. Specimen Sampling and Handling: See Section I Device Description.

5. Calibration: N/A

6. Quality Control: See Analytical Section M.1c -Traceability, Stability, Expected values (controls, calibrators, or methods)

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10 and 21 CFR 801.109(b)(1).

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.