# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY ONLY TEMPLATE

A. 510(k) Number: K141338   
B. Purpose for Submission: To obtain a substantial equivalence determination for the Liat™ Strep A Assay performed on the Liat™ Analyzer for the detection of Streptococcus pyogenes (Group A Streptococcus).   
C. Measurand: Group A Streptococcus DNA   
D. Type of Test: Real-time PCR assay for qualitative detection of Group A Streptococcus DNA in throat swab specimens.   
E. Applicant: IQuum, Inc.   
F. Proprietary and Established Names: Liat™ Strep A Assay   
G. Regulatory Information:

1. Regulation section:   
21 CFR 866.2680

2. Classification: Class II

3. Product code: PGX

4. Panel:

83 - Microbiology

# H. Intended Use:

1. Intended use(s):

The Liat™ Strep A Assay, performed on the Liat™ Analyzer, is a qualitative in vitro diagnostic test for the detection of Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus, Strep A) in throat swab specimens from patients with signs and symptoms of pharyngitis.

The Liat™ Strep A Assay utilizes nucleic acid purification and polymerase chain reaction (PCR) technology to detect Streptococcus pyogenes by targeting a segment of the Streptococcus pyogenes genome.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: IQuum Liat™ Analyzer

# I. Device Description:

The Liat™ Strep A Assay, performed on the Liat™ Analyzer, is a rapid, automated in vitro diagnostic test for the qualitative detection of Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus, Strep A) DNA in throat swab specimens in Amies medium.

The Liat™ Strep A Assay targets a conserved region of the S. pyogenes genome. An Internal Process Control (IPC) is also included to monitor the adequacy of process steps involved in nucleic acid extraction and amplification/detection, as well as for the presence of inhibitors. In order for a sample to be called negative for S. pyogenes, the IPC must be detected. If the IPC is not detected, the result is reported as invalid and the operator is instructed to repeat the test. The time-to-result is approximately 15 minutes.

The assay employs a single-use disposable Liat™ Tube that holds the sample preparation and PCR reagents, and in which the nucleic acid extraction and amplification/detection processes take place. The reagents are housed in unit dose pre-packed tube segments separated by frangible seals that are ruptured sequentially by actuators within the Liat™ Analyzer to effect sample processing, including bacterial cell lysis, DNA recovery and removal of inhibitors, PCR amplification and detection.

To perform the Liat™ Strep A Assay the operator transfers an aliquot of a throat swab sample in Amies medium into a Liat™ Strep A Assay tube, scans the relevant tube and sample identification barcodes and then inserts the tube into the Liat™ Analyzer for automated processing and result interpretation. An embedded microprocessor coordinates the functions of the Liat™ Analyzer to move the sample from one segment of the tube to another, and to control the reaction volume, temperature and duration of each step.

Positive and Negative External Controls are provided separately from the assay reagents in the Liat™ Strep A Assay Quality Control Kit. The Positive Control comprises dried, inactivated S. pyogenes bacteria that are rehydrated by the operator with Amies medium. The Negative Control comprises Amies medium alone.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra™ Direct Strep Assay

2. Predicate 510(k) number(s): K133883

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">LiatT Strep A</td><td colspan="1" rowspan="1">LyraT Direct Strep</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The LiatTM Strep A Assay, performedon the LiatTM Analyzer, is a qualitativein vitro diagnostic test for the detectionof Streptococcus pyogenes (Group A β-hemolytic Streptococcus) in throat swabspecimens from patients with signs andsymptoms of pharyngitis.The Liat Strep A Assay utilizes nucleicacid purification and polymerase chainreaction (PCR) technology to detectStreptococcus pyogenes by targeting asegment of the Streptococcus pyogenesgenome.</td><td colspan="1" rowspan="1">The Lyra Direct Strep Assay is a Real-Time PCR in vitro diagnostic test forthe qualitative detection anddifferentiation of Group A β-hemolyticStreptococcus (Streptococcus pyogenes)and pyogenic Group C and G β-hemolytic Streptococcus nucleic acidsisolated from throat swab specimensobtained from patients with signs andsymptoms of pharyngitis, such as sorethroat. The assay does not differentiatebetween pyogenic Groups C and G β-hemolytic Streptococcus.All negative test results should beconfirmed by bacterial culture, becausenegative results do not preclude GroupA, C or G Strep infection and should not</td></tr><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">LiatTM Strep A</td><td colspan="1" rowspan="1">LyraTM Direct Strep</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">be used as the sole basis for treatment.The assay is intended for use inhospital, reference, or state laboratorysettings. The device is not intended forpoint-of-care use.</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.2690</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PGX</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Group A Streptococcus (S. pyogenes)</td><td colspan="1" rowspan="1">Group A, C/G Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Throat swab</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Strep A Target</td><td colspan="1" rowspan="1">Conserved region of Group AStreptococcus genome</td><td colspan="1" rowspan="1">Conserved regions within the genomesof Group A and C/G streptococci</td></tr><tr><td colspan="1" rowspan="1">Assay Method</td><td colspan="1" rowspan="1">PCR for detecting the presence /absence of bacterial DNA in clinicalspecimens</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnique</td><td colspan="1" rowspan="1">Different reporter dyes for targetanalyte and Internal Control</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Result</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">LiatTM Strep A</td><td colspan="1" rowspan="1">LyraTM Direct Strep</td></tr><tr><td colspan="1" rowspan="1">SampleProcessing</td><td colspan="1" rowspan="1">Automated bacterial lysis and silica-magnetic bead-based nucleic acidextraction and purification</td><td colspan="1" rowspan="1">Manual heat lysis without nucleic acidpurification</td></tr><tr><td colspan="1" rowspan="1">Bacterial Lysis</td><td colspan="1" rowspan="1">Chaotrope and enzymatic digestion</td><td colspan="1" rowspan="1">Heat</td></tr><tr><td colspan="1" rowspan="1">EquipmentRequired</td><td colspan="1" rowspan="1">Liat™ Analyzer</td><td colspan="1" rowspan="1">•ABI 7500 Fast Thermocycler•Plate centrifuge for 96 well plate• Heat block• Thermometer• Timer• Micropipette</td></tr><tr><td colspan="1" rowspan="1">AssayAutomation</td><td colspan="1" rowspan="1">Yes: computer controlled sampleprocessing and PCRamplification/detection</td><td colspan="1" rowspan="1">No: manual sample processing and PCRset-up</td></tr><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">LiatT Strep A</td><td colspan="1" rowspan="1">LyraTM Direct Strep</td></tr><tr><td colspan="1" rowspan="1">Reagents / KitComponents</td><td colspan="1" rowspan="1">Unitized LiatTM Strep A AssayTubeTransfer pipette</td><td colspan="1" rowspan="1">•  Unitized Process Buffer for heatlysis•  Bulk PCR Master Mix•  Bulk Rehydration Solution forMaster Mix</td></tr><tr><td colspan="1" rowspan="1">Reagent Format</td><td colspan="1" rowspan="1">Unitized ready for useManual reagent manipulation notrequired</td><td colspan="1" rowspan="1">−Bulk reagentsManual pipetting required</td></tr><tr><td colspan="1" rowspan="1">ResultInterpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Manual</td></tr><tr><td colspan="1" rowspan="1">Time-to-result</td><td colspan="1" rowspan="1">~15 minutes</td><td colspan="1" rowspan="1">&gt;70 minutes</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

To perform the Liat™ Strep A Assay a throat swab sample in Amies medium is first added to the Liat™ Tube using a transfer pipette. The tube is then loaded into the Liat™ Analyzer for automated processing, the first step of which is mixing of the sample with the Internal Process Control (IPC). Lysis of both the IPC and the S. pyogenes target organism, if present, are then induced by chaotropic and proteolytic reagents and the nucleic acids are recovered by binding to the surface of silica-coated magnetic beads. The beads and bound nucleic acid are captured using a magnetic field and the lysate is removed, after which a wash step is conducted to remove potential inhibitors and the captured nucleic acids are eluted under low-salt conditions into a small volume of buffer.

Target amplification and detection are performed using TaqMan (hydrolysis) probe-based realtime PCR. Primers and probes for both the S. pyogenes-specific target and the IPC are included in the reaction mixture. The fluorescence intensity for each optical channel is monitored and results are interpreted by an automated algorithm based on a combination of cycle threshold (Ct) and endpoint fluorescence values.

The sample preparation and real-time PCR amplification processes are conducted within the Liat™ Analyzer in approximately 15 minutes.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the Liat™ Strep A Assay was evaluated at 3 sites. Two operators at each site tested a 4 member panel in triplicate on 5 different days, for a total of 360 assay runs (4 panel members X 3 replicates X 2 operators X 5 days X 3 sites). Nine Liat™ Analyzers and 3 Liat™ Strep A Assay tube lots were included in the study. Each reproducibility panel comprised a negative, a high negative, a low positive and a medium positive sample prepared in throat swab matrix (Table 1).

Table 1. Reproducibility Panel Members   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Multipleof LOD</td><td rowspan=1 colspan=1>CFU/Test</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>ExpectedResult forS. pyogenes 1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=3 colspan=1>ATCCBAA-946</td><td rowspan=1 colspan=1>0.03X</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>3X</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Used in determination of percent agreement

The percent agreement with expected results for each panel member from each site is shown in Table 2 for the S. pyogenes target and in Table 3 for the Internal Process Control (IPC). The mean and coefficient of variation $( \% C V )$ for the $\mathrm { C t }$ and endpoint fluorescence values are also provided for all samples with valid $\mathrm { C t }$ scores (i.e., this analysis was not performed for the S. pyogenes target in Negative or High Negative samples).

All (90/90; $100 \%$ Moderate Positive and 89/90 $( 9 8 . 9 \% )$ Low Positive samples produced positive assay results. All (90/90; $100 \%$ ) Negative and High Negative samples produced negative assay results. Overall there was $9 9 . 7 \%$ (359/360) agreement with expected results.

At the Low Positive target level, the $\% \mathrm { C V }$ for the S. pyogenes Ct and endpoint fluorescence values ranged from 2.5 to $3 . 6 \%$ and from 29.2 to $4 3 . 8 \%$ , respectively depending on the site. For the IPC in S. pyogenes negative samples, the $\% \mathrm { C V }$ for $\mathrm { C t }$ ranged from 1.7 to $1 . 9 \%$ and for endpoint fluorescence between 12.2 and $1 3 . 3 \%$ . Overall, the PCR Ct and endpoint fluorescence values for both the $S .$ pyogenes target and IPC demonstrated acceptable precision.

Table 2. Reproducibility Study Results for S. pyogenes Stratified by Site   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=5>Site A</td><td rowspan=1 colspan=5>Site B</td><td rowspan=1 colspan=5>Site E</td><td rowspan=1 colspan=5>Total</td></tr><tr><td rowspan=2 colspan=1>Agmt%)</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>A (%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>A(%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30/30(100))</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>90/90(100))</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>29/30(96.7)</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>43.8</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>1.79</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>89/90(98.9)</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.69</td><td rowspan=1 colspan=1>35.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>3.15</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.79</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>3.19</td><td rowspan=1 colspan=1>7.86</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>3.04</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>119/120(99.2)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>120/120(101)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>120/120(10)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>359/360( 99.7)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

Table 3. Reproducibility Study Results for Internal Process Control Stratified by Site   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=5>Site A</td><td rowspan=1 colspan=5>Site B</td><td rowspan=1 colspan=5>Site E</td><td rowspan=1 colspan=5>Total</td></tr><tr><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td><td rowspan=1 colspan=1>Agmt</td><td rowspan=1 colspan=2>Ct</td><td rowspan=1 colspan=2>Endpoint</td></tr><tr><td rowspan=1 colspan=1>%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>A((%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>A(%)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.89</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>12.8</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>3.01</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2.72</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>15.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.95</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>2.62</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.84</td><td rowspan=1 colspan=1>11.6</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30(100))</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.69</td><td rowspan=1 colspan=1>11.6</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.66</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.23</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>90/90(100)</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.52</td><td rowspan=1 colspan=1>14.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120/120(10)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>120/120(10 0)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>120/120(10)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>360/360(101)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

Legend to Tables 3 and 4: NA: Not Applicable; Neg: Negative; HN: High Negative; LP: Low Positive; MP: Moderate Positive; Agmt: Agreement with expected result; $\% \mathrm { C V }$ : Percent Coefficient of Variation

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

The Liat™ Strep A Assay incorporates 3 controls:

(1) Internal Process Control (IPC);   
(2) External Positive Control;   
(3) External Negative Control.

Internal Process Control

The IPC is designed to avoid reporting of false negative results due to excessive sample inhibition or system operation outside the normal range. The IPC is comprised of an inactivated bacterium that is pre-packaged in each Liat™ Tube. When conducting an assay, the sample is first mixed with the IPC which then goes through all the test process steps to monitor both the sample processing and PCR performance. The IPC DNA is detected in a separate optical channel from the S. pyogenes target nucleic acid. If the IPC Ct and fluorescence endpoint values do not fall within specified limits and S. pyogenes target DNA is not detected, the test is reported as invalid and the operator is instructed to repeat the test.

External Positive and Negative Controls

Positive and Negative Controls are provided separately from the assay reagents in the Liat™ Strep A Assay Quality Control Kit and must be tested each time a new lot of reagents is used. Should either the Positive or Negative Control fail to produce the expected result during the “Add Lot” process the system software precludes use of the Liat™ tube lot until the cause of the failure is resolved.

The Positive Control and consists of dried, inactivated S. pyogenes bacteria. The target level for the Positive Control is designed to be close to the LOD of the assay.

The Negative Control consists of Amies medium without S. pyogenes target organisms and is provided in unit dose quantity.

Evaluation of Control Performance

The ability of the Liat™ Strep A IPC and Positive Control to monitor the performance of the assay was verified under various simulated failure modes (Table 3). All of the replicates tested under each simulated failure mode were reported as invalid thereby demonstrating that the IPC and Positive Control are effective in monitoring substantial failures in reagent and process integrity.

Table 3. Simulated failure modes used to evaluate assay controls   

<table><tr><td></td><td>Process Failure</td><td>Reagent Failure</td></tr><tr><td>Sample Preparation</td><td>Failure to capture silica magnetic beads during nucleic acid extraction</td><td>Premature rupture of frangible seal between Lysis and Wash Buffers</td></tr><tr><td>PCR Amplification and Detection</td><td>Deviation in PCR annealing temperature</td><td>Premature rupture of frangible seal between Wash and Elution Buffers</td></tr></table>

The ability to monitor for contamination was evaluated by spiking Negative Controls with low levels of $S .$ . pyogenes DNA. In each case the system software indicated that the control failed.

During the Clinical Study to validate the performance of the Liat™ Strep A Assay, 15 Positive and 15 Negative Controls were tested across 6 lots of Liat™ Tubes and 11 Liat™ Analyzers. All $( 1 0 0 \% )$ of the controls produced the expected results as did 20/20 $( 1 0 0 \% )$ Negative Controls and 22/22 $( 1 0 0 \% )$ Positive Controls or known positive samples that were used to monitor assay performance during the course of the in-house analytical studies.

# Sample Stability

Sample stability was evaluated by testing S. pyogenes positive and negative throat swabs after storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ , $2 5 \mathrm { { ^ \circ C } }$ or ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ . S. pyogenes positive samples were spiked with organisms at 3X LOD. At each test point, all of the samples produced the expected result, supporting the Package Insert claims for sample stability (48 hours at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ ).

# d. Analytical Sensitivity (Detection limit):

The Limit of Detection (LOD) of the Liat™ Strep A Assay was determined by limiting dilution studies using titered stocks of 4 strains of S. pyogenes that were spiked into throat swab matrix. The LOD was determined as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. $, 2 1 9 / 2 0$ replicates tested positive) and ranged from 5-20 CFU/mL (Table 4).

Table 4. Limits of Detection in Throat Swab Matrix   

<table><tr><td rowspan=1 colspan=1>S. pyogenes Strain</td><td rowspan=1 colspan=1>LOD (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-946</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>ATCC12370</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1066</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>ATCC700294</td><td rowspan=1 colspan=1>20</td></tr></table>

The analytical reactivity of the Liat™ Strep A Assay was evaluated with 5 strains of S. pyogenes in addition to those tested in the LOD study (Table 5). Titered stocks of each strain were diluted in throat swab matrix and tested in triplicate. All 5 strains produced positive results at or below a concentration of 80 CFU/mL.

Table 5. Analytical reactivity of the Liat™ Strep A Assay   

<table><tr><td rowspan=1 colspan=1>S. pyogenesStrain</td><td rowspan=1 colspan=1>Reactivity TiterCFU/mL</td></tr><tr><td rowspan=1 colspan=1>ATCC700949</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>ATCC21548</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>ATCC10403</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>ATCC700497</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>ATCC700499</td><td rowspan=1 colspan=1>40</td></tr></table>

BLAST analysis also showed the target region for the Liat™ Strep A Assay to be conserved across different strains of S. pyogenes.

# e. Analytical specificity:

# Cross-Reactivity Study

The analytical specificity of the Liat™ Strep A Assay was evaluated by testing a panel of 72 potentially cross-reactive organisms and viruses (Table 6). Except where noted, bacteria and yeast were tested at $\overline { { 1 } } . 1 \substack { - 4 . 4 \mathrm { ~ x ~ } 1 0 } ^ { 6 }$ Colony Forming Units/mL and viruses were tested at $1 . 0 0 { - } 4 . 4 5 \times 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } .$ . Each virus or species of organism was tested in triplicate in S. pyogenes negative throat swab matrix. In each case, all three assay replicates produced negative results indicating no evidence of crossreaction with the Liat™ Strep A Assay.

BLAST analysis of the Liat™ Strep A primer, probe and amplicon sequences also showed no homology with other organisms or viruses that is likely to impact assay performance.

Table 6. Organisms and Viruses Tested for Potential Cross-reaction or Interference with the Liat™ Strep A Assay   

<table><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Streptococcus canis</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Streptococcus constellatus</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td></tr><tr><td colspan="1" rowspan="1">Bacteroides oralis</td><td colspan="1" rowspan="1">Moraxella catarrhalis (2 strains)</td><td colspan="1" rowspan="1">Streptococcus equi</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Moraxella lacunata</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae 3</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae 3</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Streptococcus oralis</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae 1</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis 2</td><td colspan="1" rowspan="1">Neisseria subflava</td><td colspan="1" rowspan="1">Streptococcus sanguis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Treponema denticola 3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium pseudodiphtheriticum</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">Adenovirus, Type 1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Adenovirus, Type 7</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Cytomegalovirus</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Epstein-Barr Virus 4</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Hepatitis B Virus 4</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumonia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Human Papilloma Virus, Type 11 4</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Human Papilloma Virus, Type 6 4</td></tr><tr><td colspan="1" rowspan="1">Legionella jordanis</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr></table>

Concentration tested (per mL): $^ { 1 } 1 . 4 0 \times 1 0 ^ { 5 }$ T $\overline { { \mathrm { \ C I D } _ { 5 0 } } } .$ ; $^ 2 1 . 2 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ Elementary Bodies; $^ 3 1 . 2 5 { - } 1 . 6 3 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genomic copies; $\overline { { ^ 4 4 . 4 5 } } _ { \mathrm { ~ X ~ } }$ $1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 }$ ; $^ 5 2 . 1 5 { - } 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genomic copies

# Cross-Contamination Study

In order to evaluate the risk of false-positive results due to contamination between samples (run-to-run) and between analyzers (instrument-to-instrument), 20 high positive ( $3 . 1 3 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ) and $2 0 S .$ . pyogenes negative samples were tested on each of two Liat™ Analyzers (i.e., 40 high positive and 40 negative samples in total). All 40 high positive and all 40 negative samples produced the expected results, indicating that the risk of contamination between runs and instruments is acceptably low.

# f. Assay cut-off:

The Liat™ Strep A Assay result algorithm analyses the characteristics of the fluorescence amplification curves for the S. pyogenes target and Internal Process Control (IPC) to disposition samples as “Strep A Detected,” “Strep A Not Detected,” “Indeterminate” or “Invalid.”

The algorithm employs cut-offs for both the cycle threshold (Ct) value and endpoint amplitude in addition to other parameters. The cut-offs were determined through analysis of a combination of negative clinical samples and samples that were spiked with different strains at the LOD target level.

In order for a sample to be called negative for S. pyogenes (“Strep A Not Detected”), the IPC must be detected. If the IPC is not detected, the result is reported as “Assay Invalid. Repeat Assay.” In cases in which S. pyogenes is detected but the curve shape is abnormal, the result is reported as “Indeterminate.”

As indicated in the Package Insert, if the test result is “Indeterminate” or “Invalid”, the assay should be repeated with the same patient specimen or, if possible, a new specimen from the same patient. Specimens that have repeat “Indeterminate” or “Invalid” results should be sent for confirmatory testing by another method.

# g. Assay Interference:

# Interfering Substances

The Liat™ Strep A was evaluated with 28 substances that may be encountered in throat swabs (Table 7). Testing was performed in triplicate with medically and/or physiologically relevant concentrations of each potential interfering substance in throat swab matrix and in the presence and absence of S. pyogenes ATCC 700294 at 60 CFU/mL (i.e., 3X LOD). In all cases the expected results were obtained, although with bovine mucin at the concentration tested all three assay replicates exhibited delayed Ct values and lower endpoint fluorescence for the S. pyogenes target. This is reflected in the Package Insert in a footnote to the list of potential interfering substances that were evaluated.

Table 7. Interfering Substance Panel   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen (Tylenol)</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Adult Robitussin Peak Cold, Maximum strength, Cough+Chest</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Adult Robitussin Peak Cold, Nighttime, Multi-symptom cold</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Blood (human)</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Brompheniramine Maleate</td><td rowspan=1 colspan=1>60 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Cepacol Sore Throat</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Cepacol Ultra Sore Throat Spray</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Children&#x27;s Dimetapp Cold &amp; Cough</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Children&#x27;s Robitussin Cough &amp; Cold</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Children&#x27;s Dimetapp Nighttime Cold &amp; Congestion</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic Max</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Chlorpheniramine Maleate</td><td rowspan=1 colspan=1>25 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Cool Mint Listerine, antiseptic</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Crest Pro-Health</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Dextromethorphan HBr</td><td rowspan=1 colspan=1>20 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Diphenhydramine HCl</td><td rowspan=1 colspan=1>350 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Doxylamine Succinate</td><td rowspan=1 colspan=1>300 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Erythromycin</td><td rowspan=1 colspan=1>15 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Guaifenesin (Guaiacol glyceryl)</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Halls Mentho-lyptus Cherry</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Halls Mentho-lyptus Sugar Free</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen (Advil)</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mucin: bovine submaxillary gland, type I-S</td><td rowspan=1 colspan=1>25 mg/mL 1</td></tr><tr><td rowspan=1 colspan=1>Penicillin G</td><td rowspan=1 colspan=1>1.2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Sucrets Complete</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Tussin Adult Chest Congestion</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Tylenol Cold Sore Throat</td><td rowspan=1 colspan=1>5% v/v</td></tr></table>

Delayed Ct and reduced endpoint fluorescence for S. pyogenes

# Interfering Microorganisms

The organisms and viruses listed in Table 6, above, were evaluated for the potential to interfere with the Liat™ Strep A Assay by testing them at the concentrations shown in the table in the presence of a low level of S. pyogenes ATCC 700294 (i.e., $6 0 \mathrm { C F U / m L } = 3 \mathrm { X }$ LOD). Each organism or virus was tested in triplicate. All assay replicates produced positive results and there was therefore no evidence of microbial interference with the Liat™ Strep A Assay.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

The performance of the Liat™ Strep A Assay was evaluated in a study that was conducted at six clinical sites in the U.S. between December 2013 and April 2014. Specimens were collected from subjects who were 3 years of age or older and who presented with symptoms that are characteristic of pharyngitis (i.e., sore throat plus at least one of the following: pharyngeal redness, pharyngeal or tonsillar exudate, tonsillar swelling, tender cervical lymphadenopathy or fever). Two or three throat swabs were collected from each subject. One swab for use with the Liat™ Strep A Assay was placed in 1mL Amies transport medium. The additional swabs were used for standard of care diagnosis. The order of swab collected was at the discretion of the clinical sites.

The swabs in Amies medium were tested with the Liat™ Strep A Assay at the clinical sites on the day of collection. Residual specimen was then shipped to a central laboratory for reference culture where an aliquot of the transport medium was plated on Trypticase Soy Agar with $5 \%$ Sheep Blood and incubated at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ in an atmosphere of $5 \text{‰}$ $\mathrm { C O } _ { 2 }$ . Isolated colonies that exhibited $\beta$ -hemolysis were typed by latex agglutination (Remel Strepex®). Culture plates that did not exhibit $\beta$ -hemolytic colonies after 48 hours were recorded as negative.

Specimens that produced discordant results between the reference culture and the Liat™ Strep A Assay were subject to additional characterization by an alternative PCR method, followed by bi-directional sequencing.

Throat swabs were collected from a total of 799 subjects during the course of the study. Two hundred and twenty-nine (229) specimens were excluded from the analysis of

performance due to failure to comply with inclusion criteria (2), delayed reference culture (16), absence of a Liat™ Strep A result (1), use of the incorrect swab type (130), incorrect Liat™ Strep assay procedure (20) or protocol deviation (60). A total of 570 specimens was therefore included in the analysis of performance (Table 8).

Table 8. Liat™ Strep A Assay Clinical Performance vs Reference Culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Reference Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>LiatT Strep A</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>193</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3^2$</td><td rowspan=1 colspan=1>374</td><td rowspan=1 colspan=1>377</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>570</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>170/173=98.3%(95%CI: 95.0-99.4%)</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>374/397= 94.2%(95%CI: 91.5-96.1%)</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>170/193= 88.1%(95%CI:82.8-91.9%)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>374/377 = 99.2% (95% CI: 97.7-99.7%)</td></tr></table>

1 23/23 positive by alternative PCR and bi-directional sequencing $^ 2 3 / 3$ positive by alternative PCR and bi-directional sequencing and upon repeat testing with Liat™ Strep A

Of the 570 unique specimens included in the analysis, 7 initially produced invalid Liat™ Strep A test results (invalid rate $= 7 / 5 7 7 = 1 . 2 \%$ ). All 7 produced valid results on retest and those results are included in the analysis of performance.

The performance of the LiatStrep A Assay stratified by clinical site is shown in Table 9.

Table 9. Liat™ Strep A Assay performance stratified by site   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Samples(%)</td><td rowspan=2 colspan=1>Culture Positive(% Prevalence)</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>246(43.2)</td><td rowspan=1 colspan=1>75(30.5)</td><td rowspan=1 colspan=1>96.0(88.9-98.6)</td><td rowspan=1 colspan=1>91.2(86.0-94.6)</td><td rowspan=1 colspan=1>82.8(73.5-89.3)</td><td rowspan=1 colspan=1>98.1(94.6-99.4)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>85(14.9)</td><td rowspan=1 colspan=1>32(37.6)</td><td rowspan=1 colspan=1>100(89.3-100)</td><td rowspan=1 colspan=1>94.3(84.6-98.1)</td><td rowspan=1 colspan=1>91.4(77.6-97.0)</td><td rowspan=1 colspan=1>100(92.9-100)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>52(9.1)</td><td rowspan=1 colspan=1>11(21.2)</td><td rowspan=1 colspan=1>100(74.1-100)</td><td rowspan=1 colspan=1>100(91.4-100)</td><td rowspan=1 colspan=1>100(74.1-100)</td><td rowspan=1 colspan=1>100(91.4-100)</td></tr><tr><td rowspan=1 colspan=1>E1</td><td rowspan=1 colspan=1>4(0.7)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100(51.0-100)</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>40(7.0)</td><td rowspan=1 colspan=1>11(27.5)</td><td rowspan=1 colspan=1>100(74.1-100)</td><td rowspan=1 colspan=1>93.1(78.0-98.1)</td><td rowspan=1 colspan=1>84.6(57.8-95.7)</td><td rowspan=1 colspan=1>100(87.5-100)</td></tr><tr><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>143(25.1)</td><td rowspan=1 colspan=1>44(30.8)</td><td rowspan=1 colspan=1>100(92.0-100)</td><td rowspan=1 colspan=1>97.0(91.5-99.0)</td><td rowspan=1 colspan=1>93.6(82.8-97.8)</td><td rowspan=1 colspan=1>100(96.2-100)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>570(100)</td><td rowspan=1 colspan=1>173(30.4)</td><td rowspan=1 colspan=1>98.3(95.0-99.4)</td><td rowspan=1 colspan=1>94.2(91.5-96.1)</td><td rowspan=1 colspan=1>88.1(82.8-91.9)</td><td rowspan=1 colspan=1>99.2(97.7-99.7)</td></tr></table>

1 Enrollment at Site E was low but the samples were collected according to the study protocol and are therefore included in the analysis.

b. Clinical specificity:

Refer to Section $3 a$ , above.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The clinical study included 570 specimens from six U.S. sites that were collected between December 2013 and April 2014. The overall prevalence of S. pyogenes as determined by culture was $3 0 . 4 \%$ (173/570), and as determined by the Liat™ Strep A Assay the overall prevalence was $3 3 . 9 \%$ (193/570). The prevalence by age and sex of the subjects is shown in Table 10.

Table 10. Prevalence of S. pyogenes stratified by Age and Gender   

<table><tr><td rowspan=2 colspan=1>Age/Gender</td><td rowspan=1 colspan=2>LiatTM Strep A</td><td rowspan=2 colspan=1>Total</td><td rowspan=2 colspan=1>Prevalence (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>41.8</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>32.4</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>16.0</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>37.3</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>302</td><td rowspan=1 colspan=1>30.8</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.