# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K141931

# B. Purpose for Submission:

The purpose of this submission is to show that performance of the Quidel® Lyra™ Adenovirus Assay on the Applied Biosystems® 7500 Fast Dx Real-Time PCR Instrument is substantially equivalent to the performance of the Adenovirus R-gene® US Assay.

# C. Measurand:

This assay detects the presence of adenoviral deoxyribonucleic acid (DNA) using real-time polymerase chain reaction (PCR). The PCR primers are developed to bind to the penton base gene.

# D. Type of Test:

This is a nucleic acid based test that uses PCR to detect the presence of adenoviral DNA with fluorescently-labeled sequence-specific probes.

# E. Applicant:

Quidel Corporation

F. Proprietary and Established Names:

Lyra™ Adenovirus Assay

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code:

OCC

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Lyra™ Adenovirus Assay is a real-time polymerase chain reaction (PCR) in vitro diagnostic test for the qualitative detection of human adenovirus (HAdV) viral DNA isolated from nasal and nasopharyngeal swab specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infections. The intended use of this test is to aid in the diagnosis of HAdV in conjunction with other clinical and laboratory findings.

The test detects, but does not differentiate HAdV species (A, B, C, D, E, and F) or serotypes (HAdV 1-52).

Negative results do not preclude HAdV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

2. Indication(s) for use:

Same as intended use.

3. Special conditions for use statement(s): Prescription only.

4. Special instrument requirements: bioMérieux NucliSENS® easyMAG® System (software version 2.0)

Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR Instrument (software version 1.4)

# I. Device Description:

The Lyra Adenovirus Assay detects viral nucleic acid that has been extracted from a patient sample using the NucliSENS easyMAG automated extraction platform. The assay includes a Process Control (PRC) that is added prior to the extraction procedure. The PRC acts as an extraction and amplification control.

Real-time PCR is used to generate and detect amplicons for the target virus and the PRC. This reaction is performed using the ABI 7500 Fast Dx Real-Time PCR Instrument (“ABI 7500 Fast Dx”).

During amplification, a DNA polymerase cleaves the sequence-specific probes that have hybridized to the adenoviral and PRC DNA. This enzymatic step generates a fluorescent signal as, when the probe is cleaved, the fluorescent dye is separated from the quencher dye. The fluorescent signal intensity grows as the reaction proceeds and the number of cleaved probe molecules increases with each thermal cycle. A positive signal for human Adenovirus (HAdV) and the PRC are achieved when the fluorescent signals reach a certain level above the threshold. The results of the Lyra Adenovirus Assay are recorded and analyzed by the ABI 7500 Fast Dx Software (version 1.4 or higher) and then reported to the user.

# J. Substantial Equivalence Information:

1. Predicate device name:

Adenovirus R-gene US Assay

2. Predicate 510(k) number: K121942

3. Comparison with predicate:

<table><tr><td>Item Intended Use</td><td>Subject Device Lyra Adenovirus Assay</td><td>Predicate Device Adenovirus R-gene US Assay (K121942)</td></tr><tr><td></td><td>The LyraTM Adenovirus Assay is a real-time polymerase chain reaction (PCR) in vitro diagnostic test for the qualitative detection of human adenovirus (HAdV) viral DNA isolated from nasal and nasopharyngeal swab specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infections. The intended use of this test is to aid in the diagnosis of HAdV in conjunction with other clinical</td><td>Adenovirus R-gene US is a Real-Time PCR in vitro diagnostic test for the rapid and qualitative detection of Adenovirus viral DNA isolated and purified from nasopharyngeal specimens (swab or wash/aspirate) obtained from individuals exhibiting signs and symptoms of acute respiratory infection. The intended use for this test is to aid in the diagnosis of Adenovirus infections in humans.</td></tr><tr><td></td><td>and laboratory findings.</td><td>Negative results do not preclude</td></tr><tr><td></td><td>The test detects, but does not differentiate HAdV species (A, B, C, D, E, and F) or serotypes (HAdV 1-52).</td><td>Adenovirus infection and should not be used as the sole basis for treatment or other management decisions.</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject DeviceLyra Adenovirus Assay</td><td colspan="1" rowspan="1">Predicate DeviceAdenovirus R-gene US Assay(K121942)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative results do notpreclude HAdV infection andshould not be used as the solebasis for diagnosis, treatment orother patient managementdecisions.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Real-time polymerase chainreaction (PCR)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Viral Target</td><td colspan="1" rowspan="1">Penton Base Gene</td><td colspan="1" rowspan="1">Hexon Gene</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasal and nasopharyngealswabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs andNasopharyngealaspirates/washes</td></tr><tr><td colspan="1" rowspan="1">Amplification</td><td colspan="1" rowspan="1">Self-contained and automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionTechniques</td><td colspan="1" rowspan="1">Self-contained and automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtractionMethod</td><td colspan="1" rowspan="1">bioMérieux NucliSENSeasyMAG System</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Collection andTransport Media</td><td colspan="1" rowspan="1">Universal Transport Medium(Copan/DHI)MicroTest M4, M4-RT, M5,and M6 (Remel)</td><td colspan="1" rowspan="1">Universal Transport Medium(DHI)MicroTest M4RT Transport(Remel)</td></tr><tr><td colspan="1" rowspan="1">Instrument/AssayPlatform</td><td colspan="1" rowspan="1">ABI 7500 Fast Dx Instrument</td><td colspan="1" rowspan="1">Cepheid SmartCycler II</td></tr><tr><td colspan="1" rowspan="1">ControlsIncluded withAssay</td><td colspan="1" rowspan="1">Internal process control(MS2 Bacteriophage)</td><td colspan="1" rowspan="1">Positive control plasmid DNANeg control (molecular gradewater)Internal control (IC2) phageparticle</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Guidance for Industry and FDA Staff - Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests. Issued: March 13, 2007. http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/Guidanc eDocuments/UCM071287.pdf   
2. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay. Issued: October 9, 2009. Guidance Documents (Medical Devices and Radiation-Emitting Products) $>$ Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay   
3. Guidance for Sponsors, Institutional Review Boards, Clinical Investigators, and FDA Staff - Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable. Issued: April 25, 2006. http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/Guidanc eDocuments/ucm071265.pdf   
4. Guidance for Industry and Food and Drug Administration Staff – eCopy Program for Medical Device Submissions. Issued: October 10, 2013. http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/Guidanc eDocuments/UCM313794.pdf

# L. Test Principle:

This is a nucleic acid based test using real-time PCR. Infection with HAdV is detected through the use of PCR to amplify and detect viral DNA. The DNA sequence for the viral penton base gene is amplified and detected using a sequence-specific probe that is cleaved during PCR amplification, resulting in a signal that occurs when the fluorescent reporter dye is released from the quencher.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility

The reproducibility study was performed with contrived samples extracted using the bioMérieux NucliSENS easyMAG System and tested on the ABI 7500 Fast Dx Instrument. Reproducibility was assessed at three separate sites (two external, one inhouse) where two users at each site tested each virus sample in triplicate over five nonconsecutive days (2 operators x 3 aliquots x 5 days x 3 sites $= 9 0$ replicates total for each panel member). The four member reproducibility panel consisted of a negative sample (HAdV Negative Nasal Matrix), a high negative (0.5X LoD), low positive/close to the limit of detection (2X LoD), and moderate positive (5X LoD) sample. To create the positive panel members, HAdV species A (serotype 31) was diluted in a pooled negative nasal matrix. The positive and negative controls used in this study were from the Lyra Adenovirus Control Set (HAdV Positive Control and HAdV Negative Control, respectively).

The reproducibility panel members were randomized, blinded, stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ , and delivered to the testing sites frozen. Each day, the samples were extracted using the bioMérieux NucliSENS easyMAG System. The extracted nucleic acids were tested with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx. A single lot of kits was used to test reproducibility at each of the three sites.

The data (summarized in Table 1 below) demonstrates good reproducibility as overall percent agreement for the Lyra Adenovirus Assay ranged from 97 to $100 \%$ with overall $\%$ CV across all sites for all samples and controls ranging from $4 . 8 \%$ to $1 2 . 8 \%$ depending on the concentration tested.

Table 1 - Reproducibility Study Results   

<table><tr><td rowspan=2 colspan=1>PanelMember ID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=3>Combined Site Data</td></tr><tr><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Rate ofDetection</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HAdVHighNegative(0.5x LoD)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>26/30*</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>86/90*</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>6.2</td></tr><tr><td rowspan=1 colspan=1>HAdVLowPositive(2x LoD)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>5.9</td></tr><tr><td rowspan=1 colspan=1>HAdVModeratePositive(5x LoD)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>25.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>4.8</td></tr><tr><td rowspan=1 colspan=1>HAdVNegativeNasalMatrix</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HAdVPositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>20.5</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.2</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>18.6</td><td rowspan=1 colspan=1>8.5</td></tr><tr><td rowspan=1 colspan=1>HAdVNegativeControl</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

One aliquot was removed from the analysis because it had an invalid PRC result. Three aliquots (from the same replicate) tested negative.

# Precision

Precision was assessed at a single site (in-house) where two users performed extraction and PCR on contrived samples once a day for 12 non-consecutive days. Each virus sample was tested in triplicate (2 operators $_ { \textrm { X 3 } }$ aliquots $\mathrm { \textbf { x } } 1 2 \mathrm { \ d a y s } = 7 2$ replicates total for each panel member). The four member precision panel consisted of a negative sample (HAdV negative nasal matrix), a high negative (0.5X LoD), low positive/close to the limit of detection (2X LoD), and moderate positive (5X LoD) sample. To create the positive panel members, HAdV species A (serotype 31) was diluted in a pooled negative nasal matrix. Panel members were stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ until the time of extraction and testing.

The positive and negative controls used in this study were from the Lyra Adenovirus Control Set.

The data (summarized in Table 2 below) demonstrates good reproducibility as overall percent agreement for the Lyra Adenovirus Assay was $100 \%$ with overall $\%$ CV across operators for all samples and controls ranging from $1 . 0 \%$ to $3 . 4 \%$ depending on the concentration tested.

Table 2 - Precision Study Results   

<table><tr><td rowspan=2 colspan=1>Platform</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=6>Average Ct Value</td></tr><tr><td rowspan=1 colspan=1>Pos.Control</td><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>0.5XLoD</td><td rowspan=1 colspan=1>Neg.Matrix</td><td rowspan=1 colspan=1>Neg.Control</td></tr><tr><td rowspan=3 colspan=1>ABI 7500Fast Dx</td><td rowspan=1 colspan=1>Operator 1 Avg Ct</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>Operator 2 Avg Ct</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>25.4</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td></tr></table>

b. Linearity/assay reportable range:

N/A

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

1. The kit contains an internal process control (PRC), consisting of the MS2 Bacteriophage. It is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification. The PRC serves as a general process control to ensure that each step of the procedure was performed correctly and that general reagents are working.

2. The package insert for this assay includes a reference to an optional external control set (the Lyra Adenovirus Control Set, #M110).

# Fresh vs. Frozen Study

Five viral panels were created by diluting HAdV species A (type 31) in pooled negative nasal matrix. Panel 1 was extracted and tested on the day of preparation. Panels 2 and 3 were stored at $4 \mathrm { { } ^ { \circ } C }$ and then extracted and tested on days 7 and 8, respectively. Panels 4 and 5 were stored at ${ - 2 0 ^ { \circ } \mathrm { C } }$ and then extracted and tested on days 7 and 8, respectively.

Each panel consisted of 30 low positive samples (2X LoD), 30 moderate positive (5X LoD) samples, and 10 negative samples. Each panel member was extracted and tested with the Lyra Adenovirus Assay in singlet. The negative control consisted of HAdV negative nasal matrix. The positive control included in the study was from the Lyra Adenovirus Control Set.

Data collected from these samples met the acceptance criteria o $f \geq 9 5 \%$ agreement between results obtained on date of preparation and after storage. This indicates that storage of samples at either $4 ^ { \circ }$ or ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ for up to 8 days does not impact the ability of the Lyra Adenovirus Assay to detect HAdV in low and moderate positive samples.

# Master Mix Stability after Rehydration

An evaluation of master mix stability was carried out using a single lot of master mix that was rehydrated and stored at ${ - 2 0 ^ { \circ } \mathrm { C } }$ , $4 \mathrm { { } ^ { \circ } C }$ , and at room temperature. After storage at time points ranging from 1 to 76 hours, the master mix was used to test samples that were low positive (2X LoD) for HAdV species E (type 4). A single working stock of HAdV

positive sample was created by diluting HAdV type 4 in negative nasal matrix, then extracting the viral DNA and storing it at $- 7 0 \mathrm { { } ^ { \circ } C }$ until the time of testing.

Three vials of master mix were tested at each time point and temperature combination. Each vial was tested in triplicate. The negative control consisted of HAdV negative nasal matrix. The positive control included in the study was from the Lyra Adenovirus Control Set.

Data collected from these samples met the acceptance criteria of $\leq 3$ Cts between results obtained at time 0 versus after storage. This indicates that storage of rehydrated master mix at room temperature for up to 4 hours or at $4 ^ { \circ }$ or ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for up to 3 days does not impact the ability of the Lyra Adenovirus Assay to detect HAdV in low positive samples.

# Extracted Specimen Stability

Stability of extracted specimens was tested with samples that were low positive (3X LoD) for HAdV species E (type 4). A single working stock of HAdV positive samples was created by diluting HAdV type 4 in negative nasal matrix, then extracting the viral DNA. Aliquots were stored at various temperatures and then tested with the Lyra Adenovirus Assay in triplicate at the time points described in the table below.

Table 3 – Time Points Tested in Extracted Specimen Stability Study   

<table><tr><td rowspan=2 colspan=1>Temperature</td><td rowspan=1 colspan=4>Hour</td><td rowspan=1 colspan=9>Day</td><td rowspan=1 colspan=3>Freeze/ThawCycle</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>RoomTemperature</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>4°C</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>:</td></tr><tr><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>:</td></tr></table>

All samples tested in this study met the acceptance criteria of $< 3$ Cts between results obtained at time 0 versus results after storage. This indicates that the following storage conditions are acceptable for extracted samples: 1) storage at room temperature, $4 ^ { \circ }$ , ${ } _ { - 2 0 } \mathrm { { \circ C } }$ , or ${ } _ { - 8 0 } \mathrm { { } ^ { \circ } C }$ for up to 31 days and 2) up to three freeze/thaw cycles.

# d. Detection limit:

# Limit of Detection (LoD)

The LoD was determined using one type from each of the six HAdV serotypes (A-F). For each virus serotype 2-fold serial dilutions were prepared. Twenty aliquots were extracted from each dilution and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ until testing with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument. The LoD was determined to be the lowest titer at which the virus was detected greater than $9 5 \%$ of the time. For confirmation purposes, at least one dilution higher than the LoD was included in the study for each serotype.

Virus was diluted in a pooled negative nasal matrix. The negative control consisted of HAdV negative nasal matrix. The positive control included in the study was from the Lyra Adenovirus Control Set. Three lots of master mix were used in this study.

Results of the study are shown in Table 4.

Table 4 - Results from LoD Study   

<table><tr><td rowspan=1 colspan=1>Species/ Serotype</td><td rowspan=1 colspan=1>LoD (TCID50/mL)</td><td rowspan=1 colspan=1>Avg. Ct Value</td></tr><tr><td rowspan=1 colspan=1>A/31</td><td rowspan=1 colspan=1>8.00 x10-2</td><td rowspan=1 colspan=1>28.5</td></tr><tr><td rowspan=1 colspan=1>B/3</td><td rowspan=1 colspan=1>8.00 x10-2</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>C/1</td><td rowspan=1 colspan=1>8.00 x10-2</td><td rowspan=1 colspan=1>28.1</td></tr><tr><td rowspan=1 colspan=1>D/19</td><td rowspan=1 colspan=1>1.61 x101</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=1 colspan=1>E/4</td><td rowspan=1 colspan=1>1.00 x100</td><td rowspan=1 colspan=1>27.7</td></tr><tr><td rowspan=1 colspan=1>F/41</td><td rowspan=1 colspan=1>3.20 x10-2</td><td rowspan=1 colspan=1>28.9</td></tr></table>

# e. Analytical specificity:

# Microbial Interference

This study was conducted to evaluate the potential for false negative results when nontarget pathogens are present at high concentrations in a HAdV low positive sample. A panel of 57 microorganisms (30 viruses, 26 bacteria, and 1 yeast) representing common respiratory pathogens or flora commonly present in the nasopharynx was evaluated. This 57 member panel was created and tested three times, once in the presence of HAdV type 4, once in the presence of HAdV type 31, and once in the absence of adenoviruses (negative nasal matrix).

For the 57 member panel, the level of each microorganism was determined by growing and titering the microorganism listed; the exceptions were Coronavirus NL63, Epstein Barr Virus, Measles, Mumps virus, Parainfluenza Type 4A, Chlamydophila pneumonia, and Mycoplasma pneumoniae for which the original titer provided by the supplier was used. Each potential cross reactant was individually spiked at a high level into a negative swab matrix and then added to either a HAdV type 4 or HAdV type 31 working stock (20X LoD virus diluted in negative nasal matrix). In the final solution, the concentration was 2X LoD for HAdV.

Bacteria and yeast were tested at concentrations of $1 0 ^ { 6 }$ to $1 0 ^ { 8 } \mathrm { C F U / m L }$ ; the exceptions were Chlamydia pneumonia ( $1 9 . 8 ~ \mathrm { n g / m L }$ ) and Mycoplasma pneumonia $( 2 . 8 2 \mathrm { ~ x ~ i 0 } ^ { 6 }$ Color Changing Units/mL). Viruses were tested at concentrations of $1 0 ^ { 3 }$ to $1 0 ^ { 8 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ; the exception was Epstein Barr Virus $\left( 8 . 3 4 \mathrm { ~ x ~ } 1 0 \right) ^ { 7 }$ copies/mL). Microorganism DNA was extracted from these solutions and stored at ${ - 8 0 ^ { \circ } C }$ until samples were tested in triplicate with the Lyra Adenovirus Assay.

The negative control used in this study consisted of HAdV negative nasal matrix. The positive control consisted of either HAdV type 4 or type 31 diluted in negative nasal matrix at a concentration of 2X LoD with no other microorganisms present.

The results (below in Table 5) showed that all HAdV positive samples remained positive, indicating that there was no interference with the results of the Lyra Adenovirus Assay when the microorganisms tested were present at high concentrations.

# Cross-Reactivity

This study was conducted to evaluate the potential for false positive results when nontarget pathogens are present at high concentrations in a HAdV negative sample. A panel 57 microorganisms (30 viruses, 26 bacteria, and one yeast) was evaluated.

For the 57 member panel, the level of each microorganism was determined by growing and titering the microorganism listed; the exceptions were Coronavirus NL63, Epstein Barr Virus, Measles, Mumps virus, Parainfluenza Type 4A, Chlamydophila pneumonia, and Mycoplasma pneumoniae for which the original titer provided by the supplier was used. Each potential cross reactant was individually spiked at a high level into a negative swab matrix. Bacteria and yeast were tested at concentrations of $1 { \overset { \vartriangle } { 0 } } ^ { 6 }$ to $1 0 ^ { 9 } \mathrm { C F U / m L }$ ; the exceptions were Chlamydia pneumonia $\mathrm { 0 . 1 1 n g / m L ) }$ and Mycoplasma pneumonia (3.16 $\mathrm { ~ x ~ } 1 0 ^ { \tilde { 6 } }$ Color Changing Units $ { ' } \mathrm { m L }$ ). Viruses were tested at concentrations of $1 0 ^ { 4 }$ to $1 0 ^ { 8 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ; the exception was Epstein Barr Virus $9 . 2 7 \times 1 0 ^ { 7 }$ copies/mL).

Microorganism DNA was extracted with the bioMérieux NucliSENS easyMAG System and then stored at ${ } _ { - 8 0 } \mathrm { { } ^ { \circ } C }$ until samples were tested in triplicate with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument.

The negative control used in this study consisted of HAdV negative nasal matrix. The positive control consisted of HAdV type 4 diluted in negative nasal matrix at a concentration of 5X LoD with no other microorganisms present.

The results (below in Table 5) showed that all HAdV negative samples remained negative, indicating that the Lyra Adenovirus Assay did not cross-react with any of the microorganisms tested.

Table 5 – Microbial Interference and Cross-Reactivity Study Results   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Concentration tested(TCID50/mL orCFU/mL)</td><td colspan="2" rowspan="1">Lyra Adenovirus Assay Result</td></tr><tr><td colspan="1" rowspan="1">Interference(+ HAdV)</td><td colspan="1" rowspan="1">Cross-reactivity(- HAdV)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">9.08E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">5.40E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumonia</td><td colspan="1" rowspan="1">0.11 ng/μL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2.10E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.42E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellualre</td><td colspan="1" rowspan="1">1.53E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">9.30E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">3.18E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">3.16E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">4.00E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.32E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">6.53E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.19E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.40E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.29E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1.61E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">9.75E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.13E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.26E+09</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">3.44E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">3.18E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.43E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">6.38E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">5.40E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">9.23E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">6.08E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albican</td><td colspan="1" rowspan="1">9.70E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">2.46E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1.41E+04</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">2.42E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.00E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">3.62E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">2.14E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">7.60E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">4.45E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">2.17E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">2.17E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">2.41E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.03E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">9.27E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="1" rowspan="1">5.89E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 Strain G</td><td colspan="1" rowspan="1">1.96E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (A1)</td><td colspan="1" rowspan="1">3.66E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 45</td><td colspan="1" rowspan="1">2.94E+04</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 52</td><td colspan="1" rowspan="1">2.63E+04</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mexico/4108/2009</td><td colspan="1" rowspan="1">4.08E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">3.55E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/04/2006</td><td colspan="1" rowspan="1">1.54E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.95E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">2.75E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">3.97E+06</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">3.15E+08</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">2.36E+07</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4A</td><td colspan="1" rowspan="1">1.04E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV A (Long)</td><td colspan="1" rowspan="1">4.36E+04</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV B (Wash/18537/62)</td><td colspan="1" rowspan="1">3.43E+05</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">1.11E+04</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr></table>

# Interfering Substances

An interfering substances study was carried out to examine whether a panel of 11 endogenous and exogenous potential PCR inhibitors affected the performance of the Lyra Adenovirus Assay. The panel consisted of substances such as blood, mucin, or medications (prescription and over-the-counter) for relief of congestion, sore throat, allergy, and asthma symptoms (concentrations tested are listed in Table 6 below).

Table 6 – Concentrations of Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Substance Name</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Mucin (Bovine Submaxillary Gland,type I-S)</td><td rowspan=1 colspan=1>60 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Blood (human), EDTAanticoagulated</td><td rowspan=1 colspan=1>2% (vol/vol)</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine</td><td rowspan=1 colspan=1>15% (vol/vol)</td></tr><tr><td rowspan=1 colspan=1>Afrin Nasal Spray</td><td rowspan=1 colspan=1>15% (vol/vol)</td></tr><tr><td rowspan=1 colspan=1>Zicam Homeopathic Non-DrowsyAllergy Relief No Drip Liquid NasalGel</td><td rowspan=1 colspan=1>5% (vol/vol)</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>15% (vol/vol) of dose</td></tr><tr><td rowspan=1 colspan=1>Throat Lozenges</td><td rowspan=1 colspan=1>0.68g/mL; 1/18 drop, crushed;active ingredients: 1.7 mg/mLmenthol</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>3.3-5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4.0 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>6.6-10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir phosphate</td><td rowspan=1 colspan=1>7.5-25 mg/mL</td></tr></table>

Each of the 11 panel members were diluted individually in negative nasal matrix and then tested three times, once in the presence of HAdV type 4, once in the presence of HAdV type 31, and once in negative matrix only (no virus added). In the HAdV positive samples, the virus concentration tested was 2X LoD. Microorganism DNA was extracted with the bioMérieux NucliSENS easyMAG System and then stored at ${ - 8 0 ^ { \circ } C }$ until samples were tested in triplicate with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument.

The negative control used in this study consisted of HAdV negative nasal matrix. The positive controls consisted of HAdV type 4 or type31 diluted in negative nasal matrix at a concentration of 2X LoD with no other substances present. The results of this study are shown in Table 7 below.

Table 7 – Results of Interfering Substances Study   

<table><tr><td rowspan=3 colspan=1>Substance Name</td><td rowspan=1 colspan=6>HAdV</td><td rowspan=2 colspan=1>Inhibition(yes/no)</td></tr><tr><td rowspan=1 colspan=2>Species ESerotype 4</td><td rowspan=1 colspan=2>Species ASerotype 31</td><td rowspan=1 colspan=2>No Analyte</td></tr><tr><td rowspan=1 colspan=1>Ct Avg.</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Ct Avg.</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Ct Avg.</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Mucin (Bovine SubmaxillaryGland, type I-S)</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Blood (human), EDTAanticoagulated</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Afrin Nasal Spray</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Zicam Homeopathic Non-Drowsy Allergy Relief NoDrip Liquid Nasal Gel</td><td rowspan=1 colspan=1>25.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>OTC Throat Lozenges:Ricola Action Cherry</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir phosphate</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No</td></tr></table>

The results of this study showed that all HAdV negative samples remained negative and all HAdV positive samples remained positive, indicating that there was no assay interference from any of the substances tested.

# Inclusivity

Reactivity of the Lyra Adenovirus Assay was established in the LoD study with one type from each of the six HAdV serotypes (A-F). An inclusivity study was conducted to test reactivity of the Lyra Adenovirus Assay with the 52 other known HAdV types.

Forty nine (49) HAdV types were tested initially at a concentration of 2X the serotype LoD (established in the LoD study above). One sample was generated for each HAdV type by spiking cultured and titered virus into negative nasal matrix. Viral DNA was extracted with the bioMérieux NucliSENS easyMAG System. Each extract was tested in triplicate. Higher concentrations (range of 6-250 X LoD) were tested if the organism was not detected at the LoD. The negative control used in this study consisted of HAdV negative nasal matrix. The positive controls consisted of the six HAdV types from the LoD panel diluted in negative nasal matrix and tested individually at a concentration of 1X LoD.

The multiple of LoD at which the serotypes were detected is listed in the Table 8 below.

Table 8 – Inclusivity Study Results   

<table><tr><td colspan="1" rowspan="1">1 able 0</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">ConcentrationTested(TCID50/mL)</td><td colspan="1" rowspan="1">Multiple ofLoDDetected</td></tr><tr><td colspan="1" rowspan="2">A</td><td colspan="1" rowspan="1">HAdV-12</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-18</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="8">B</td><td colspan="1" rowspan="1">HAdV-7</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-11</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-14</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-16</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-21</td><td colspan="1" rowspan="1">2.00 x101</td><td colspan="1" rowspan="1">250x LoDa</td></tr><tr><td colspan="1" rowspan="1">HAdV-34</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-35</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-50</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="3">C</td><td colspan="1" rowspan="1">HAdV-2</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-5</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-6</td><td colspan="1" rowspan="1">1.60 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">HAdV-40</td><td colspan="1" rowspan="1">6.40 x10-2</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="16">D</td><td colspan="1" rowspan="1">HAdV-8</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-9</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-10</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-13</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-15</td><td colspan="1" rowspan="1">9.66 x101</td><td colspan="1" rowspan="1">6x LoDb</td></tr><tr><td colspan="1" rowspan="1">HAdV-17</td><td colspan="1" rowspan="1">9.66 x101</td><td colspan="1" rowspan="1">6x LoDc</td></tr><tr><td colspan="1" rowspan="1">HAdV-20</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-22</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-23</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-24</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-25</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-26</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-27</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-28</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-29</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-30</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">ConcentrationTested(TCID50/mL)</td><td colspan="1" rowspan="1">Multiple ofLoDDetected</td></tr><tr><td colspan="1" rowspan="13">b</td><td colspan="1" rowspan="1">HAdV-32</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-33</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-36</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-37</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-39</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-43</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-44</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-45</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-46</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-47</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-48</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-49</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">HAdV-51</td><td colspan="1" rowspan="1">3.22 x10-1</td><td colspan="1" rowspan="1">2x LoD</td></tr></table>

a HAdV-21 required repeat testing and was ultimately found to be detected at 250X LoD. b HAdV-15 required repeat testing and was ultimately found to be detected at 6X LoD. c HAdV-17 required repeat testing and was ultimately found to be detected at 6X LoD.

All of the 49 HAdV types tested (above) were detected by the Lyra Adenovirus Assay. Three types were not detected at 2X LoD. These types were subjected to repeat analysis and were ultimately detected at 250X LoD (ADV-21/Serotype B) and 6X LoD (ADVADV-15 and ADV-17/Serotype D) for their respective serotypes.

HAdV Species D serotypes 38 and 42 and Species G serotype 52 (often associated with conjunctivitis and gastroenteritis) were not evaluated for inclusivity as the sponsor could not obtain these strains. Instead, the manufacturer conducted an in silico analysis where they aligned their probe and primers with the target regions from a single sequence (obtained from GenBank) for each of these adenovirus types. The alignments showed a $100 \%$ agreement of the forward primer with types 38, 42, and 52. For the reverse primer, three mismatches were detected in the sequences for types 38 and 42 $84 \%$ homology) and two mismatches with the sequence for type 52 ( $8 9 . 5 \%$ homology). For the probe, there was $100 \%$ agreement with types 38 and 42 while three mismatches were observed with the sequence for type 52 ( $8 5 \%$ homology).

# Carry-Over

The carry-over study was designed to uncover the presence of contamination in negative specimens due to carry-over of virus during nucleic acid extraction and PCR amplification. The study was conducted by alternately placing high positive $( 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ HAdV type 4 diluted in negative nasal matrix) and negative specimens (negative nasal matrix) in each well of the PCR plate. All samples were extracted and tested fresh.

A total of 48 high positives and 48 negatives were tested in each run, with a total of five runs, each run on a separate day $4 8 \times 5 = 2 4 0$ samples total). All 240 replicates of negative sample were “Not Detected,” therefore, no evidence of carry-over was observed.

f. Assay cut-off:

# Baseline

The initial baseline assessment was conducted with a manual baseline setting (starting at $\mathrm { C t } 3 . 0$ and ending at Ct 15.0). A total of 34 positive and 31 negative clinical specimens were used for the analysis. Testing indicated that the use of the manual baseline resulted in decreased sensitivity for HAdV positive specimens as three out of five culture negative specimens (with direct fluorescence antigen confirmation) were detected as positive with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument. Additional testing showed that four of the culture negative specimens were detected as positive by the Prodesse ProAdeno $^ +$ Assay on the Cepheid SmartCycler II. As a result, the baseline was changed to an auto baseline to resolve these results. The clinical specimens were tested again to verify that the change had no effect on the detection of the PRC and HAdV culture positive specimens.

# Threshold

For the ABI 7500 Fast Dx Instrument, the threshold is defined as the level of fluorescence above which the ABI 7500 Fast Dx software will assign a “positive” call. The software uses the baseline and threshold values to calculate the threshold cycle (Ct).

When the panel of clinical specimens (described above) was tested with the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument using the auto baseline setting, one of the adenovirus positive specimens did not cross the threshold and was therefore reported as negative. This indicated that the use of the initial threshold setting $( 8 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ baseline-corrected normalized reporter units; $\Delta \mathrm { R n }$ ) could result in decreased sensitivity for HAdV positive specimens. An analysis was conducted to compare the results of the initial threshold setting with a new, lower threshold setting $( 6 . 5 0 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 4 } \Delta \mathrm { R n } )$ for the ABI 7500 Fast Dx Instrument. Retesting of the clinical specimens confirmed that the HAdV positive specimen crossed the new threshold and was reported as positive. This change did not affect the detection of the PRC.

# Ct Cut-Off

A Ct cut-off study was conducted prior to the initiation of the clinical study. The clinical specimens from the baseline and threshold analyses were evaluated for their range in Ct values to determine the appropriate cut-off for the assay. The majority of samples were detected around or before 25 cycles. The highest Ct value detected was 33. The cut-off was set at 35 cycles because an earlier cut-off could result in decreased sensitivity for HAdV positive specimens.

2. Comparison studies:

# a. Method comparison with predicate device:

Method comparison was based on the results from the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument compared to the results obtained from direct specimen fluorescent antibody (DSFA) and cell culture with direct fluorescent antigen (DFA). The testing description and data are listed in the Clinical Studies section (below).

# b. Matrix comparison:

A transport media study was conducted to show compatibility of five different types of viral transport media (VTM) with the Lyra Adenovirus Assay. The five VTM tested were Remel MicroTest M4, M4-RT, M5, and M6, and Universal Transport Medium (UTM) from Copan or Diagnostics Hybrids Inc.

Low positive (2X LoD) HAdV samples were created by diluting HAdV type 4 or type 31 in transport media. All samples were extracted and then tested immediately afterwards. A blank extraction control (VTM with no virus added) was included for each type of transport media. For each virus and VTM combination, three separate extractions were conducted. Each extract was tested in duplicate (3 extractions x 2 replicates $= 6$ replicates tested for each virus and media combination). Results are shown in Table 9.

Table 9 – Results of Comparison Study for Viral Transport Media   

<table><tr><td rowspan=1 colspan=1>Transport Media</td><td rowspan=1 colspan=1>Avg CtHAdV4</td><td rowspan=1 colspan=1>HAdV4%CV</td><td rowspan=1 colspan=1>Avg CtHAdV31</td><td rowspan=1 colspan=1>HAdV31%CV</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>7.0%</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>Theoretical Average CtValue*</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>3.2%</td></tr></table>

\*The “Theoretical Average Ct Value” represents the average Ct value obtained in other analytical studies from HAdV tested in nasal swab matrix only (no transport media).

The five VTM sample types demonstrated comparable performance, indicating that the Lyra Adenovirus Assay is compatible with Remel MicroTest M4, M4-RT, M5, and M6, and Universal Transport Medium (UTM) from Copan or Diagnostics Hybrids Inc.

3. Clinical studies:

a. Clinical Sensitivity and Specificity

Clinical performance of the Lyra Adevnovirus Assay was established in a prospective study with fresh samples collected at three sites in the United States during the winter of 2013 (January 2013) and the winter of 2014 (December 2013 until February 2014). Due to the overall low prevalence of HAdV, these samples were supplemented with retrospective frozen samples collected at one site in the United States on dates ranging from May 2012 until February 2014.

For the prospective study, performance of the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument was compared to an FDA-cleared direct detection kit for respiratory specimens. The kit is a composite method of viral culture with direct fluorescent antigen staining (CDFA) followed by direct fluorescent antibody staining of the specimen (SDFA). A specimen was called positive if either the CDFA or the SDFA were positive. For a specimen to be called negative, it must be negative for both CDFA and SDFA.

All samples used for this study were remnants of nasal and nasopharyngeal swab specimens that were collected from symptomatic individuals for routine analysis and that otherwise would have been discarded. All clinical sites were granted waivers of informed consent by their IRBs for this study. Inclusion criteria included: samples were submitted to the testing laboratory by an ordering physician from patients with signs and symptoms of acute respiratory illness. Exclusion criteria included: inadequate specimen volume, specimen inappropriately stored or shipped.

Sites 2 and 3 performed the extraction and testing for all samples collected at those sites. Specimens collected at site 1 were sent to an in-house testing laboratory where they were extracted and tested using the Lyra Adenovirus Assay. The samples were extracted within 72 hours of collection using the bioMérieux NucliSENS easyMag System and then tested using the Lyra Adenovirus Assay run on the ABI 7500 Fast Dx Instrument.

All reference testing with the comparator method was conducted at an in-house testing laboratory. Specimens were shipped to this location daily with cold packs and were cultured within 72 hours of collection.

Tables 10 and 11 (below) show the age and gender distribution of subjects from the prospective study.

Table 10 - Gender Distribution for Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>651 (52.5%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>590 (47.5%)</td></tr></table>

Table 11 - Age Distribution for Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Age (Years)</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>525 (42.3%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>279 (22.5%)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>266 (21.4%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60</td><td rowspan=1 colspan=1>169 (13.6%)</td></tr></table>

A total of 1241 nasal and nasopharyngeal swab specimen extracts were tested by both the Lyra Adenovirus Assay and comparator method. A total of two invalid specimens were removed from the analysis. These specimens were invalid on initial and repeat testing with the subject device. Table 12 below details the results for the remaining 1239 specimens. Analysis of individual site data did not show any noteworthy differences between sites. Therefore, pooling of clinical site data to generate the summary table below was appropriate.

Table 12 – Performance of Prospectively Collected Clinical Samples   

<table><tr><td rowspan=1 colspan=4>Combined Site Data</td></tr><tr><td rowspan=2 colspan=1>Lyra AdenovirusAssay</td><td rowspan=1 colspan=3>Comparator: CDFA with SDFA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>54*</td><td rowspan=1 colspan=1>89</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1150</td><td rowspan=1 colspan=1>1150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>1204</td><td rowspan=1 colspan=1>1239</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.1% to 100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>1150/1204</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>94.2 % to 96.5%</td></tr></table>

\* Forty-five (45) of the fifty-four (54) positives were positive by an additional FDA cleared PCR assay. Four (4) of the fifty-four (54) positives were negative by an additional FDA cleared PCR assay. Two (2) of the fifty-four (54) positives were invalid by an additional FDA cleared PCR assay. Three (3) of the fiftyfour (54) positives had insufficient volume for additional testing.

For the retrospectively selected samples, one hundred five (105) nasopharyngeal swab specimens were obtained from a pediatric hospital in the southern United States. These specimens were selected based on a qualitative result previously generated by an FDAcleared respiratory panel. All specimens were stored frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ until they were shipped to an in-house testing lab where they were extracted and tested with the Lyra Adenovirus Assay and the comparator, an FDA-cleared PCR assay. The operator performing the testing was blinded to the identity of these specimens. Table 13 shows the results obtained from these specimens.

Table 13 – Performance of Retrospectively Collected Clinical Samples   

<table><tr><td rowspan=2 colspan=1>Lyra Adenovirus Assay</td><td rowspan=1 colspan=3>Comparator: FDA-Cleared PCR Assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.5% to 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1% to 99.8%</td></tr></table>

\* One (1) positive sample was positive by a third FDA-cleared PCR assay.

4. Clinical cut-off:

N/A

5. Expected values/Reference range:

The overall expected value (positives as determined by the Lyra Adenovirus Assay on the ABI 7500 Fast Dx Instrument) was $7 . 2 \%$ , while the overall prevalence (positives as determined by the reference method, CDFA and SDFA) was $2 . 8 \%$ . Expected value and prevalence was highest in children between 1 to 5 years of age ( $1 3 . 3 \%$ and $5 . 3 \%$ , respectively). The table below shows the expected values and prevalence for all sites, stratified by age group.

Table 14 - Expected Values for the Winter of 2013 and 2014 (Combined)   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1># Tested</td><td rowspan=1 colspan=1>LyraAdenovirusPositive</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>TotalPositive byReferenceMethod</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>1 to 5</td><td rowspan=1 colspan=1>452</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>6 to 10</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>11 to 15</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=1 colspan=1>16 to 21</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>&gt; 21</td><td rowspan=1 colspan=1>437</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.7%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1241</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>2.8%</td></tr></table>

# N. Instrument Name:

ABI 7500 Fast Dx Instrument with the SDS Software version 1.4

# O. System Descriptions:

The ABI 7500 Fast Dx Instrument with the SDS Software version 1.4 is a real-time nucleic acid amplification and detection system that measures fluorescence and converts them to comparative quantitative readouts using fluorescent detection of dual-labeled hydrolysis probes. The 7500 Fast Dx is to be used only by technologists trained in laboratory techniques, procedures and on use of the analyzer.

# 1. Modes of Operation:

Instructions for programming the Lyra Adenovirus Assay, using the Version 1.4 software, are included in the package insert. The computer system is locked and controlled by ABI. No additional software can be loaded by the end-user.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X__ or No

Level of Concern: Moderate

Software Description:

The Sequence Detection Software (SDS) version 1.4 for the ABI 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-by-cycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamplife monitoring and other instrument maintenance information. The software runs as an application on a Windows operating system.

3. Specimen Identification: Performed manually

4. Specimen Sampling and Handling:

There is no automated sample processing offered in conjunction with the ABI 7500 Fast Dx Instrument. Patient specimens are prepared (DNA extraction) using the bioMérieux NucliSENS easyMAG System. The extracted samples are added manually to the 96-well microplate.

5. Calibration:

The instrument’s optics (fluorescence parameters) are calibrated upon installation. After installation of the instrument, the user can perform all required regular maintenance and calibration using IVD-labeled calibration plates.

6. Quality Control:

Quality control is addressed for each sample by addition of an in-process control during sample processing and amplification in the assay. Additionally, there are external positive and negative controls available that may be used in accordance with the user lab standards.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.