# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K141927

To obtain a Substantial Equivalence Determination for a new 510(k) application for Lyra™ Parainfluenza Virus Assay

# C. Measurand:

The Lyra™ Parainfluenza Virus Assay detects viral RNA from Parainfluenza 1, 2, and 3 viruses. The primer and probe sets amplify and detect the Parainfluenza type 1 nuclear protein gene, Parainfluenza type 2 phosphate protein gene, and Parainfluenza type 3 phosphate protein gene, using real time reverse transcriptase polymerase chain reaction.

# D. Type of Test:

The Lyra™ Parainfluenza Virus Assay is a multiplex nucleic acid detection assay that uses real time polymerase chain reaction.

# E. Applicant:

Quidel Corporation   
12544 High Bluff Drive, Suite 200   
San Diego, California 92130   
Telephone: 858-552-7910   
Fax: 858-646-8045

F. Proprietary and Established Names: Lyra™ Parainfluenza Virus Assay

# G. Regulatory Information:

1. Regulation section:

21CFR 866.3980 - Respiratory viral panel multiplex nucleic acid assay

2. Classification:

Class II

3. Product code:

OOU

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Lyra™ Parainfluenza Virus Assay is a Real‐Time PCR assay for the qualitative detection and identification of human parainfluenza virus types 1, 2 and 3 viral RNA from nasal and nasopharyngeal swab specimens from symptomatic patients. It is intended for use as an aid in the differential diagnosis of parainfluenza virus types 1, 2 and 3. This test is not intended to detect Parainfluenza 4a or Parainfluenza 4b viruses.

Negative results do not preclude parainfluenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

# 2. Indication(s) for use:

Same as intended use.

3. Special conditions for use statement(s):

This is a prescription only test. The assay is intended for use in hospital, reference, or state laboratory settings. The device is not intended for point-of-care use.

# 4. Special instrument requirements:

The assay can be performed using the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx.

# I. Device Description:

The Lyra™ Parainfluenza Virus Assay detects viral RNA of Parainfluenza 1, 2 and 3 viruses in nasal and nasopharyngeal swab specimens. First the nucleic acid is extracted from the patient sample using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ System. The extracted nucleic acid is then introduced into the master mix which contains the primers, probes, revise transcriptase, and TaqMan $^ \mathrm { \textregistered }$ chemistry and the reaction is loaded into the Applied Biosystems $^ \mathrm { \textregistered }$ (ABI) 7500 Fast Dx instrument. First, the reverse transcriptase generates a complimentary DNA strand from the viral RNA. The primer sets hybridize to the Parainfluenza type 1 nuclear protein gene, Parainfluenza type 2 phosphate protein gene, and Parainfluenza type 3 phosphate protein gene, and transcription and DNA amplification of the target sequence occur using real time reverse transcriptase polymerase chain reaction. Fluorescent-labeled probes bind to the target sequences and the dye is released from the probe containing quencher via the TaqMan $^ \mathrm { \textregistered }$ polymerase $5 ' - 3 '$ exonuclease activity. The un-quenched dye is detected in the ABI 7500 Fast Dx and if sufficient fluorescence is detected the result is reported as positive.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Prodesse ProParaflu $\mathbf { T M } +$ Assay

2. Predicate 510(k) number(s): K091053

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subject Device:Lyra Parainfluenza Virus Assay</td><td colspan="1" rowspan="1">Predicate Device:Prodesse ProParafluTM+ AssayK091053</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The LyraTM Parainfluenza VirusAssay is a Real-Time RT-PCRassay for the qualitativedetection and identification ofhuman parainfluenza virustypes 1, 2 and 3 viral RNA fromnasal and nasopharyngeal swabspecimens from symptomaticpatients. It is intended for useas an aid in the differentialdiagnosis of parainfluenzavirus types 1, 2 and 3. This testis not intended to detectParainfluenza 4a orParainfluenza 4b viruses.Negative results do not precludeparainfluenza virus infectionand should not be used as thesole basis for treatment or otherpatient management decisions.</td><td colspan="1" rowspan="1">The ProParaflu+ Assay is amultiplex Real TimeRT-PCR in vitro diagnostic testfor the qualitative detection anddiscrimination of Parainfluenza IVirus, Parainfluenza 2 Virus andParainfluenza 3 Virus (HPIV-1,HPIV-2 and HPIV-3) nucleic acidsisolated and purified fromnasopharyngeal (NP) swabspecimens obtained fromindividuals exhibiting signs andsymptoms of respiratory tractinfections. This assay targets theconserved regions of theHemagglutinin- Neuraminidase(HN) gene of HPIV-1, HPIV-2 andHPIV-3, respectively. Thedetection and discrimination ofHPIV- I, HPIV-2 and HPIV-3nucleic acids from symptomaticpatients aid in the diagnosis ofhuman respiratory tractparainfluenza infections if used inconjunction with other clinical andlaboratory findings. This test is notintended to detect Parainfluenza 4aor Parainfluenza 4b Viruses.Negative test results arepresumptive and should beconfirmed by cell culture.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative results do not precludeParainfluenza 1, 2 or 3 virusinfections and should not be usedas the sole basis for treatment orother management decisions.</td></tr><tr><td colspan="1" rowspan="1">DNA AmplificationTechnology</td><td colspan="1" rowspan="1">Real time polymerase chainreaction</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target SequenceDetected</td><td colspan="1" rowspan="1">Parainfluenza type 1 nuclearprotein gene, Parainfluenza type 2phosphate protein gene,Parainfluenza type 3 phosphateprotein gene</td><td colspan="1" rowspan="1">Conserved regions of theHemagglutinin-Neuraminidase(HN) gene of HPIV-1, HPIV-2 andHPIV-3</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasal and Nasopharyngeal swabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Extraction</td><td colspan="1" rowspan="1">NucliSENS® easyMAGTM(bioMérieux)</td><td colspan="1" rowspan="1">MagNA Pure LC System (Roche),NucliSENS® easyMAGTM(bioMérieux)</td></tr><tr><td colspan="1" rowspan="1">DetectionTechniques</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Fast Dx</td><td colspan="1" rowspan="1">Cepheid SmartCycler II</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay

# L. Test Principle:

The assay detects viral nucleic acids that have been extracted from a patient respiratory sample. A multiplex Real-time RT-PCR reaction is carried out under optimized conditions in a single tube generating amplicons for PIV-1, PIV-2, PIV-3, and the Process Control (PRC). Identification of PIV-1, PIV-2, PIV-3, and the PRC occurs by the use of target-specific primers and fluorescent-labeled probes that hybridize to conserved regions in the genomes of PIV-1, PIV-2, PIV-3, and the PRC.

<table><tr><td rowspan=1 colspan=2>LyraT Probe Targets</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Target</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>NuclearProteinggene</td></tr><tr><td rowspan=1 colspan=1>PIV-2</td><td rowspan=1 colspan=1>PhosphateProteingene</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>Phosphate Protein gene</td></tr></table>

<table><tr><td rowspan=1 colspan=2>LyraTM Probe Labels</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Dye</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>PIV-2</td><td rowspan=1 colspan=1>JOE</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>Tex Red</td></tr><tr><td rowspan=1 colspan=1>PRC</td><td rowspan=1 colspan=1>CY5</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility studies were conducted over 14 days and testing was performed on 5 non-consecutive days at two clinical sites and one in-house laboratory. Each site was provided with 5 complete panels, which were stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ until used. Operators were blinded to the sample identity, which was randomized each day. Testing was conducted following the package insert instructions for use. One kit lot was used to perform the study. One strain of each parainfluenza virus was used at 3 different concentrations, 5x LoD, 2x LoD and $0 . 5 \mathrm { x }$ LoD for the Moderate Positive, Low Positive and High Negative samples respectively. The following strains were used; Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, Parainfluenza-3 strain C243.

The following table shows the summary of results for the reproducibility study. The results show that the assay had appropriate reproducibility at both the external clinical sites and the in-house laboratory. The reproducibility for this device is acceptable. The instances where the expected result differed from the actual result are within what is reasonable for $C _ { 5 }$ and $\mathrm { C } _ { 9 5 }$ samples.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="3" rowspan="1">Combined</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rate ofDetection</td><td colspan="1" rowspan="1">Ave.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Rate ofDetection</td><td colspan="1" rowspan="1">Ave.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Rate ofDetection</td><td colspan="1" rowspan="1">Ave. Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Rate ofDetection</td><td colspan="1" rowspan="1">Ave.Ct</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">PIV-1HighN egative</td><td colspan="1" rowspan="1">18/30</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">7/30</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">11/30</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">4.9</td><td colspan="1" rowspan="1">36/90</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">PIV-1LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">26.3</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27.1</td><td colspan="1" rowspan="1">9.1</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">26.9</td><td colspan="1" rowspan="1">6.3</td></tr><tr><td colspan="1" rowspan="1">PIV-1ModeratePositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">24.5</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.8</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">24.2</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">24.2</td><td colspan="1" rowspan="1">3.3</td></tr><tr><td colspan="1" rowspan="1">PIV-1Negative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/29</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/89</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">PIV-2HighNe gative</td><td colspan="1" rowspan="1">24/30</td><td colspan="1" rowspan="1">30.2</td><td colspan="1" rowspan="1">6.7</td><td colspan="1" rowspan="1">2/30</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6/30</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">7.1</td><td colspan="1" rowspan="1">32/90</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">7.7</td></tr><tr><td colspan="1" rowspan="1">PIV-2LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">4.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27.9</td><td colspan="1" rowspan="1">9.4</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">26.8</td><td colspan="1" rowspan="1">7.7</td></tr><tr><td colspan="1" rowspan="1">PIV-2ModeratePositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">22.8</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.7</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">24.5</td><td colspan="1" rowspan="1">7.6</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">23.7</td><td colspan="1" rowspan="1">5.7</td></tr><tr><td colspan="1" rowspan="1">PIV-2Negative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/29</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/89</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">PIV-3HighNegative</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.1</td><td colspan="1" rowspan="1">5.8</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">14/30</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">3.6</td><td colspan="1" rowspan="1">73/90</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">6.7</td></tr><tr><td colspan="1" rowspan="1">PIV-3LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">25.5</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">26.1</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">26.9</td><td colspan="1" rowspan="1">8.2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">26.2</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">PIV-3ModeratePositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">22.8</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">24.3</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">23.2</td><td colspan="1" rowspan="1">4.6</td></tr><tr><td colspan="1" rowspan="1">PIV-3Negative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/29</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/89</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">PIV-1PositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.6</td><td colspan="1" rowspan="1">9.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">22.6</td><td colspan="1" rowspan="1">8.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.3</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">11.2</td></tr><tr><td colspan="1" rowspan="1">PIV-2Positive CControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">17.2</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.4</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">19.1</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">PIV-3PositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">16.7</td><td colspan="1" rowspan="1">7.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">8.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">18.5</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">18.3</td><td colspan="1" rowspan="1">11.5</td></tr><tr><td colspan="1" rowspan="1">Negative Control</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

\* One (1) replicate had an invalid PRC value and was removed for analysis.

# b. Baseline, Threshold and Cut-off:

# Baseline:

The baseline was determined using 38 parainfluenza virus negative clinical specimens. Next, known positive samples were tested and analyzed using two different baseline settings, manual and auto baseline on the ABI 7500 Fast Dx. When using the manual baseline background noise was reduced for all channels but the Ct values for all samples were changed, in some cases more than 2Cts. This resulted in a false positive for PIV-1 in a PIV-2 sample. This did not occur with the auto baseline setting. Therefore the auto baseline setting was used for all future analysis and is the chosen baseline setting in the package insert.

# Signal Threshold:

For the ABI 7500 Fast Dx Instrument, the threshold is defined as the level of fluorescence above which the ABI 7500 Fast Dx software will assign a “positive” call. The software uses the baseline and threshold values to calculate the threshold cycle $\mathrm { ( C t ) }$ .

On the ABI 7500 Fast Dx Platform, PIV-1 and PIV-3 targets required no threshold changes from the default threshold of $1 . 0 0 \mathrm { E } { + } 0 5$ baseline-corrected normalized reporter units (∆Rn). However, the initial PIV-2 target threshold of $1 . 0 0 \mathrm { E } { + } 0 5$ ∆Rn resulted in a loss of one positive specimen previously determined positive by culture with DFA confirmation. After re-analyzing the PIV-2 target using a lower threshold $( 8 . 0 0 \mathrm { E } \mathrm { + } 0 4 ~ \Delta \mathrm { R } \mathrm { n } )$ , this specimen had a detectable Ct value. Therefore, the data indicated that lowering the threshold for this target would allow for maximum sensitivity to detect late positive specimens, but did not decrease specificity (no false positives were detected with the lower threshold). The threshold for

PIV-3 $1 . 6 0 \mathrm { E } { + 0 5 } \Delta \mathrm { R n } )$ was raised to compensate for excess background fluorescence noise resulting from the auto baseline setting in this channel. These threshold levels are the levels used for all future analysis and are the chosen threshold settings in the package insert.

# Cut-Off:

The cut-off was determined using the threshold for PIV-1 $( 1 . 0 0 \mathrm { E } ^ { + 0 5 } )$ , PIV-2 $( 8 . 0 0 \mathrm { E } \substack { + 0 4 } )$ and PIV-3 $( 1 . 6 0 \mathrm { E } { + } 0 5 )$ . Auto baseline was used for all targets. These conditions follow the package insert directions. Most Ct values were recorded between 18-27 cycles; however, at least 15 percent of samples were recorded after 30 cycles. The figure below shows the Ct Distribution for all three PIV targets from the ABI 7500 Fast Dx. The highest Ct value was detected at 44 cycles. The $\mathrm { C t }$ range on this platform was from 17 cycles to 44 cycles, indicating that the cutoff should be at 45 cycles. Forty five cycles is the cut-off indicated in the package insert.

![](images/f9e11f59c15a2b0449e35182340911d6a7b0100ce127c420f75fd37ad7268b6c.jpg)

# c. Stability:

# Kit Stability:

Three kit lots were used to determine the real-time stability of the kit reagents when being stored at $4 \mathrm { { } ^ { \circ } C }$ . PIV-1, PIV-2 and PIV-3 samples were tested at $2 \mathbf { x }$ LoD for each time point. Testing was conducted on days 0, 5, 7, 10, 15, 21, 38, 56, 76 and 90 (3 months), testing was then conducted monthly from months 4-11 and will continue through month 25. The following strains were used; Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, Parainfluenza-3 strain C243. Testing was conducted according to the directions in the package insert. Testing showed there was no change in Ct value greater than $3 \ : \mathrm { C t } ,$ , which would indicate a significant change in detection and loss of stability, during the first 11 months of storage. Stability of the kit will be listed as 12 months in the package insert, and will be modified as more real time stability data is available.

# Rehydrated Master Mix Stability:

The stability of rehydrated master mix, when stored at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ , $4 \mathrm { { } ^ { \circ } C }$ , and at room temperature, was determined in this study. Testing was conducted on the ABI $7 5 0 0 \mathrm { D x }$ according to the package instructions. Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, Parainfluenza-3 strain C243 were tested at $2 \mathbf { x }$ LoD. Testing for the $4 \mathrm { { } ^ { \circ } C }$ and room

temperature conditions was performed at 1, 2, 4, 8, 9, 24, 28, 48, 52, 72 and 76 hours. Testing for the ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ condition was performed at 24, 28, 48, 52, 72 and 76 hours. Viral RNA was extracted only once (a minimum of 24 extractions), the RNA was then pooled, aliquoted and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ until used. Each strain was tested in triplicate, with three separate vials of rehydrated master mix, for every time and temperature condition. At the end of each incubation period an aliquot of viral RNA was added to the rehydrated Master Mix and loaded into the ABI $7 5 0 0 \mathrm { D x }$ . The time point selected for stability claims in the package insert is the time point one point earlier than where all replicates were not detected. The table below shows the stability of the rehydrated Master Mix.

<table><tr><td rowspan=1 colspan=1>StorageTemperature</td><td rowspan=1 colspan=1>StabilityTime Limit</td></tr><tr><td rowspan=1 colspan=1>RoomTemperature</td><td rowspan=1 colspan=1>2 hours</td></tr><tr><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>4 hours</td></tr><tr><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>52 hours</td></tr></table>

# Extracted Specimen Stability:

The stability of extracted samples was tested at Room Temperature, $4 \mathrm { { } ^ { \circ } C }$ , ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ , and $- 7 0 \mathrm { { } ^ { \circ } C }$ at various time points. Testing was conducted on the ABI $7 5 0 0 \mathrm { D x }$ according to the package instructions. Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, and Parainfluenza-3 strain C243 were prepared at $3 \mathrm { { X L o D } }$ in negative nasal matrix and used in this study. Sufficient volume was prepared to extract RNA for testing all time points. Samples stored at ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ and - $7 0 \mathrm { { } ^ { \circ } C }$ were also tested for stability after 1, 2, and 3 freeze/thaw cycles. The table below shows the different time points tested for each temperature. Testing showed that there was no change in Ct value greater than $3 \ : { \mathrm { C t } } .$ , which would indicate a significant change in detection and loss of stability, at any of the time points tested. The package insert will list the stability of extracted RNA as stable if stored at Room Temperature, $4 ^ { \circ } \mathrm { C }$ , $- 2 0 ^ { \circ } \mathrm { C }$ and - $7 0 \mathrm { { } ^ { \circ } C }$ for up to 30 days.

<table><tr><td rowspan=1 colspan=1>Temperature</td><td rowspan=1 colspan=1>Hours post extraction</td><td rowspan=1 colspan=1>Days post extraction</td></tr><tr><td rowspan=1 colspan=1>Room Temperature</td><td rowspan=1 colspan=1>4, 8 and 9</td><td rowspan=1 colspan=1>1, 2, 3, 7, 14, 21, 28, 30, 31</td></tr><tr><td rowspan=1 colspan=1>4C</td><td rowspan=1 colspan=1>4, 8 and 9</td><td rowspan=1 colspan=1>1, 2, 3, 7, 14, 21, 28, 30, 31</td></tr><tr><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1, 2, 3, 7, 14, 21, 28, 30, 31</td></tr><tr><td rowspan=1 colspan=1>-70°</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1, 2, 3, 7, 14, 21, 28, 30, 31</td></tr></table>

d. Expected values (controls, calibrators, or methods):

No external controls are provided with this assay.

There is an internal control called the ‘process control’ (PRC). It is the MS2 bacteriophage (Zeptometrix) which is a single stranded RNA virus that can be extracted, amplified, and detected by primers and probes included in the master mix. Detection of the PRC indicates that assay conditions allowed proper sample extraction, amplification, and detection.

e. Limit of Detection:

The limit of detection (LOD) (defined as the lowest $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ that is detected $2 9 5 \%$ of the time) of the Lyra Parainfluenza Virus Assay was determined on the ABI 7500 Fast Dx platform using three separate validation lots of lyophilized master mix and limiting dilutions of viral stocks. Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, and Parainfluenza-3 strain C243 were used in this study. Twenty replicates for each dilution of each parainfluenza virus type were prepared and tested. Viral stocks were diluted in negative nasal matrix and then extracted and stored for up to one month at $- 7 0 \mathrm { { } ^ { \circ } C }$ . Testing was then conducted according to the package insert. Data showing the results for each parainfluenza virus strain using each validation lot was provided at one concentration above and below the determined LoD. The table below shows the LoD determined for each parainfluenza virus type.

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>LoD on ABI 7500 Fast Dx</td><td rowspan=1 colspan=1>Average Ct Value at LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza-1, strain C-35</td><td rowspan=1 colspan=1>2.50 TCID50/mL</td><td rowspan=1 colspan=1>38.0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza-2, strain Greer</td><td rowspan=1 colspan=1>250 TCID50/mL</td><td rowspan=1 colspan=1>37.0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza-3, strain C243</td><td rowspan=1 colspan=1>80 TCID50/mL</td><td rowspan=1 colspan=1>38.1</td></tr></table>

# f. Analytical Inclusivity:

In silico analysis to demonstrate inclusivity for all three parainfluenza virus types was performed. No laboratory studies were performed because there are a very limited number of characterized strains of any human parainfluenza virus type. In silico analysis was performed comparing the sequence of the chosen target region for each parainfluenza virus type with the NCBI database for human parainfluenza virus sequences. Three hundred and eight human parainfluenza virus type 1, 458 human parainfluenza virus type 2, and 509 parainfluenza virus type 3 full-genome sequences were analyzed. Primers and probes were designed against consensus sequences found for each virus type and have $100 \%$ homology to all sequences analyzed but do not have homology with other parainfluenza virus types. Therefore, in silico analysis and primer and probe design demonstrates that the assay should detect all human parainfluenza viruses of types 1, 2, and 3.

# g. Analytical Specificity/Cross-Reactivity:

Cross Reactivity:

This study was designed to determine potential cross-reactivity of the assay with nonparainfluenza respiratory viruses, yeast, and bacteria. Organisms were sourced from DHI, Zeptometrix, or ATCC. Bacteria were tested at $1 . 0 0 \mathrm { E } { + } 6 \mathrm { c f u } / \mathrm { m l }$ or higher and viruses were tested at $1 . 0 0 \mathrm { E } { + } 0 5$ TCID50.mL or higher, stock concentrations permitting. Stocks were diluted in negative nasal matrix and the samples were extracted and amplified according to the package insert instructions. All samples were tested in triplicate starting from the extraction step. The results were analyzed for any cross-reactivity for all three parainfluenza virus types and are listed below. No Cross-reactivity was observed in this study.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Vendor</td><td colspan="1" rowspan="1">Final ConcentrationTCID50/mL orCFU/mL</td><td colspan="1" rowspan="1">Result(Avg. Ct)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 1</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">7.05E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4:ODH-42385</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5:ODH-594484</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">7.60E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">9.27E+07 Copies/mL</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">5.89E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (A1)</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 45</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">2.94E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 52</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">2.63E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mexico/4108/2009</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/04/2006</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">1.95E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">2.75E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV A (Long)</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain (Wash/18537/62)</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilia pneumonia</td><td colspan="1" rowspan="1">ATC</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila trachomatis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">3.16E+06 Color ChangingUnit/mL</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aereus</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4A</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">1.04E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaType 1</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">18.0 for PIV1</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">13.0 for PIV2</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">15.2 for PIV3</td></tr><tr><td colspan="1" rowspan="1">Negative Matrix Control</td><td colspan="1" rowspan="1">DHI</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Negative</td></tr></table>

Microbial Interference:

Parainfluenza virus types 1, 2 and 3 were spiked with clinically relevant concentrations of viruses, bacteria or yeast in order to evaluate possible interference from organisms not targeted by the assay. Parainfluenza virus types were used at a 2X LoD concentration; all dilutions were made in negative nasal matrix. Clinically relevant levels of viruses and bacteria were used when stock concentrations permitted. Samples were extracted and amplified according to the package insert instructions. All samples were tested in triplicate starting from the extraction step. The results were analyzed for any interference with any of the three parainfluenza virus types. The final concentration tested and results are listed in the table below. No interference was observed in the study.

<table><tr><td colspan="1" rowspan="1">Organism Name</td><td colspan="1" rowspan="1">ConcentrationTested(TCID50/ml orCFU/mL)</td><td colspan="1" rowspan="1">PIV-1Avg. Ct</td><td colspan="1" rowspan="1">PIV-2Avg. Ct</td><td colspan="1" rowspan="1">PIV-3Avg. Ct</td><td colspan="1" rowspan="1">Interference(Yes/No)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1.51E+05</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">2.46E+06</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1.41E+04</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">2.42E+06</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.00E+06</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">No</td></tr><tr><td></td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoxsackievirusB5/10/2006</td><td colspan="1" rowspan="1">3.62E+05</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">2.14E+05</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.52E+08</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">4.58E+05</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">2.17E+06</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">2.17E+05</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">2.41E+05</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.03E+05</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">9.27E+07copies/mL</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyreStrain</td><td colspan="1" rowspan="1">5.89E+06</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 Strain G</td><td colspan="1" rowspan="1">1.96E+05</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus (A1)</td><td colspan="1" rowspan="1">3.66E+05</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 45</td><td colspan="1" rowspan="1">1.47E+04</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 52</td><td colspan="1" rowspan="1">1.31E+04</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/Mexico/4108/2009</td><td colspan="1" rowspan="1">4.08E+05</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Influenza A/PortChalmers</td><td colspan="1" rowspan="1">3.55E+07</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Florida/04/2006</td><td colspan="1" rowspan="1">1.54E+05</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">1.95E+06</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">2.75E+08</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">RSV A (Long)</td><td colspan="1" rowspan="1">4.36E+04</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">RSV B(Wash/18537/62)</td><td colspan="1" rowspan="1">3.43E+05</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">1.11E+04</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">9.08E+07</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Bordetellabronchiseptica</td><td colspan="1" rowspan="1">5.40E+07</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">22 ng/mL (DNA)</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2.10E+06</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">1.42E+08</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumintracellualre</td><td colspan="1" rowspan="1">1.53E+08</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">9.30E+06</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">3.18E+08</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">3.16E+06color changingunits</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">4.00E+07</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1.32E+08</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">6.53E+07</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.19E+08</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.40E+08</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.29E+07</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1.61E+08</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">9.75E+07</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.13E+08</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.26E+08</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">3.44E+07</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusplantarum</td><td colspan="1" rowspan="1">3.18E+07</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1.43E+07</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">6.38E+07</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">5.40E+07</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">9.23E+07</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">6.08E+07</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">9.70E+07</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4A</td><td colspan="1" rowspan="1">4.17E+04</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Negative Nasal Matrix</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">N/A</td></tr></table>

Competitive Interference:

The purpose of this study is to determine whether competitive interference exists between analytes when more than one parainfluenza type is present in the same reaction. All samples were tested in triplicate starting from the extraction step. The results were analyzed for any competitive interference with any of the three parainfluenza virus types. The final concentration tested and results are listed in the table below.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Analyte ConcentrationTCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Test Analyte(at 2xLoD)</td><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>Replicates of TestAnalyte Detected</td><td rowspan=1 colspan=1>CompetitiveInterference(Yes/No)</td></tr><tr><td rowspan=13 colspan=1>PIV1</td><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>1.25E+03</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>1.25E+05</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>1.25E+06</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>1.25E+07</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.00E+02</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.00E+03</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.00E+04</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.00E+05</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.00E+06</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=12 colspan=1>PIV2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+01</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+04</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>4.00E+02</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>4.00E+03</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>4.00E+04</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>4.00E+05</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>4.00E+06</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=12 colspan=1>PIV3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+01</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2.50E+04</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.50E+02</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.25E+03</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.25E+05</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.25E+06</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.25E+07</td><td rowspan=1 colspan=1>8.00E+01</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Yes</td></tr></table>

The results indicate that competitive interference exists for PIV-1 when combined with PIV-2 at a concentration $1 . 2 \dot { 5 } \times 1 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 }$ or greater and at $4 . 0 \mathrm { x } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 }$ or greater when combined with PIV-3. When evaluating PIV-2, it was determined that interference existed when combined with PIV-3 at a concentration $4 . 0 \mathrm { x } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 }$ or greater. No interference was seen when PIV-2 was combined with PIV-1. For PIV-3, there was competitive interference when combined with PIV-2 at a concentration of $1 . 2 5 \mathrm { x } 1 0 ^ { 7 }$ $\mathrm { T C I D } _ { 5 0 }$ or greater. No interference was seen when PIV-3 was combined with PIV-1.

For all virus combinations, detectable interference was seen at concentrations equal to or greater than $4 . 0 \mathrm { x } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 }$ . These titers are high and not likely to be found in clinical samples; it is therefore highly unlikely that competitive interference will be seen in the clinical settings. A limitation indicating competitive interference at high concentrations will be included in the package insert.

# $h$ . Interfering Substances:

A study was conducted to demonstrate that the Lyra Parainfluenza Virus Assay specifically detects parainfluenza virus types 1, 2, and 3 in the presence of clinically relevant concentrations of potentially interfering substances. To prepare samples for testing, parainfluenza virus at 4x LoD in negative nasal matrix was diluted 1:1 with each test substance. The final concentration after dilution for each substance is listed in the table below and the final concentration of parainfluenza virus was 2x LoD. Each mixed sample was then extracted and tested according to the package insert.

Ct values for all testing were reported and there was no significant change from the control.   
No interference was detected in the presence of any of the substances tested.

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ActiveIngredients</td><td colspan="1" rowspan="1">Concentrationto be Tested</td><td colspan="1" rowspan="1">Testing LevelRationale</td></tr><tr><td colspan="1" rowspan="1">Mucin (BovineSubmaxillaryGland, type I-S)</td><td colspan="1" rowspan="1">Purified mucinprotein</td><td colspan="1" rowspan="1">60μg/mL</td><td colspan="1" rowspan="1">1000xmaximum levelpresent inserum</td></tr><tr><td colspan="1" rowspan="1">Blood (human),EDTAanticoagulated</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2% (vol/vol)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">PhenylephrineHC1</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">OxymetazolineHydrochloride</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr><tr><td colspan="1" rowspan="1">ZicamHomeopathic Non-Drowsy AllergyRelief No DripLiquid Nasal Gel</td><td colspan="1" rowspan="1">LuffaOperculata,GalphimiaGlauca,HistaminumHydrochloricum,Sulphur</td><td colspan="1" rowspan="1">5% (vol/vol)</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray:</td><td colspan="1" rowspan="1">Sodium chloridewithpreservatives</td><td colspan="1" rowspan="1">15% (vol/vol)of dose</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr><tr><td colspan="1" rowspan="1">OTC ThroatLozenges: LNegensWild Cherry</td><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">25%(mass/vol)</td><td colspan="1" rowspan="1">Same menthollevel as otherproducts</td></tr><tr><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3-5mg/mL(target4mg/mL)</td><td colspan="1" rowspan="1">10% of totalspray dose</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4.0μg/mL</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">6.6-10mg/mL(target8mg/mL)</td><td colspan="1" rowspan="1">10% of totalrecommendeddose inMupirocinointment</td></tr><tr><td colspan="1" rowspan="1">Oseltamivirphosphate(Tamiflu)</td><td colspan="1" rowspan="1">Oseltamivirphosphate</td><td colspan="1" rowspan="1">7.5-25mg/mL(target16mg/mL)</td><td colspan="1" rowspan="1">10% of totalrecommendeddose</td></tr></table>

# i. Carry-Over/Cross-Contamination:

Simulated parainfluenza high positive samples were analyzed in series alternating with negative samples (checkerboard pattern) in order to determine whether carry-over/cross-contamination occurs with the use of the Lyra Parainfluenza Virus Assay. The high positive samples in this study were at concentration of $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in negative nasal matrix. Parainfluenza-1, Parainfluenza-2, and Parainfluenza-3 analytes were combined into one sample for this study. Samples were processed and tested according to the package insert. Testing was performed on 5 separate days where each 96-well plate contained 48 high positive samples alternating with 48 negative samples (negative nasal matrix only). No cross-contamination was observed during the study. All negative samples tested had negative results and all high positive samples had all three parainfluenza viruses detected.

2. Comparison studies:

a. Method comparison with predicate device: N/A

# b. Matrix comparison:

This study was designed to verify that the assay does not have variable performance when samples are transported in various viral transport media. Parainfluenza-1 strain C35, Parainfluenza-2 strain Greer, Parainfluenza-3 strain C243 were used in this study. Viruses were diluted in the transport media to 2x LoD and testing was performed using six replicates per virus/media combination.

<table><tr><td rowspan=1 colspan=1>TransportMedia Name</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>PIV-1Avg Ct</td><td rowspan=1 colspan=1>PIV-2Avg Ct</td><td rowspan=1 colspan=1>PIV-3Avg Ct</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>34.4</td></tr><tr><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>33.5</td></tr><tr><td rowspan=1 colspan=1>M4-RT</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.8</td></tr><tr><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>34.3</td></tr></table>

Varying the transport media did not affect the performance of this assay, with no observed changes in the $\mathrm { C t }$ value. The data demonstrates that all transport media included in the study are suitable for use with the Lyra Parainfluenza Virus assay.

# c. Fresh vs. Frozen sample comparison:

Parainfluenza virus stocks were diluted to 2X and 5X LoD using negative nasal matrix. The five panels created contained 2X LoD (combined PIV-1, PIV-2 and PIV-3), 5X LoD (combined PIV-1, PIV-2 and PIV-3) and negative samples. Each panel consisted of 30 aliquots of 5X LoD, 30 aliquots of 2X LoD and 10 aliquots of negative nasal matrix. Panel 1 was extracted on the day of preparation. Panels 2 and 3 were stored at $4 \mathrm { { } ^ { \circ } C }$ and then extracted on day 7 and 8, respectively. Panels 4 and 5 were stored at ${ - 2 0 ^ { \circ } \mathrm { C } }$ and then extracted on day 7 and 8, respectively. A single replicate of each panel member was extracted and tested according to the instructions in the package insert.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>5X LoD</td><td rowspan=1 colspan=3>2X LoD</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Panel#</td><td rowspan=1 colspan=1>Conditions</td><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>Neg.</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Day 0</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Day 7, 4°C</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Day 8, 4°C</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Day 7, -20°C</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Day 8, -20°C</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>NEG</td></tr></table>

The average $\mathrm { C t }$ values for all samples at 5x LoD and $2 \mathbf { x }$ LoD are substantially equivalent among all conditions. The data presented demonstrates that the performance of this test with frozen samples is equal to the performance with fresh samples.

# 3. Clinical studies:

The evaluation of the Lyra™ Parainfluenza Virus Assay occurred in two separate studies: a prospective multi-center study using 1241 fresh specimens from the upper respiratory tract; and a retrospective study using 105 frozen specimens from the upper respiratory tract. In both studies the bioMérieux NucliSENS® easyMag® was used at all sites for the extraction of nucleic acids from the clinical specimens. The Applied Biosystems® 7500 Fast Dx Real-Time PCR Instrument was used with the Lyra™ Parainfluenza Virus Assay for the amplification and detection of the target nucleic acids.

The prospective specimens were also tested with direct specimen fluorescent antibody (DSFA) and cell culture with DFA (CCFA). Specimens for DSFA and CCFA were shipped to central location daily with cold packs, and were cultured within 72- hours of collection. The DSFA and CCFA of the specimens were performed at a central location, DHI reference laboratory (Diagnostic Hybrids, Athens, Ohio).

The study utilized fresh and frozen specimens from the 2013 and 2014 respiratory seasons. Clinical samples were collected at three geographically diverse locations, all clinical laboratories

# Prospective Study:

A prospective study was conducted with 1241 fresh specimens from male and female patients with age range of ${ \le } 2$ to 102 years. The table below shows the age and gender distribution of the patients in this study.

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>651</td><td rowspan=1 colspan=1>590</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt; 2 years</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>158</td></tr><tr><td rowspan=1 colspan=1>2 to 5 years</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>108</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>69</td></tr></table>

Similar performance was observed at the 3 clinical sites participating in the study during the 2013 and 2014 respiratory seasons. Therefore, performance results from all sites were pooled and the summary data for PIV-1, PIV-2 and PIV-3 are presented in the tables below.

<table><tr><td rowspan=1 colspan=4>PIV-1</td></tr><tr><td rowspan=3 colspan=1>LyraTMParainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: DSFA and Culture with DFA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1228</td><td rowspan=1 colspan=1>1228</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1231</td><td rowspan=1 colspan=1>1241</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72.2% to 100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>1228/1231</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3 % to 99.9%</td></tr></table>

\* Two (2) of the three (3) samples positive by Lyra but negative by the comparator method were positive when tested by an additional RT-PCR assay.

<table><tr><td rowspan=1 colspan=4>PIV-2</td></tr><tr><td rowspan=3 colspan=1>LyraTMParainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: DSFA and Culture with DFA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>1241</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>1246</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6% to100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>1236/1236</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7% to 100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>PIV-3</td></tr><tr><td rowspan=3 colspan=1>LyraTMParainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: DSFA and Culture with DFA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1219</td><td rowspan=1 colspan=1>1219</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1224</td><td rowspan=1 colspan=1>1241</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>81.6% to 100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>1219/1224</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0% to 99.8%</td></tr></table>

\* Five (5) of the five (5) positive by Lyra but negative by the comparator method, were positive when tested by an additional RT-PCR assay.

Retrospective Study:

A Retrospective study was also conducted at site 1 because of the low prevalence of parainfluenza virus seen at all clinical sites during the study period. One hundred five frozen specimens from the upper respiratory tract were tested concurrently with the Lyra™

Parainfluenza Virus Assay and the Prodesse ProParaFlu $^ +$ assay (K091053).

<table><tr><td rowspan=1 colspan=4>PIV-1</td></tr><tr><td rowspan=3 colspan=1>LyraTM Parainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: Prodesse ProParaFlu+ assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.2% to 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>80/81</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>93.3% to 99.8%</td></tr></table>

\* One (1) sample positive by Lyra but negative by the comparator method, was positive when tested by an additional RT-PCR assay.

<table><tr><td rowspan=1 colspan=4>PIV-2</td></tr><tr><td rowspan=3 colspan=1>LyraTM Parainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: Prodesse ProParaFlu+ assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85.1% to 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>78/83</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>86.7% to 97.4%</td></tr></table>

\* Five (5) of five (5) positive samples were positive by Lyra but negative by the comparator method, were positive when tested by an additional RT-PCR assay.

<table><tr><td rowspan=1 colspan=4>PIV-3</td></tr><tr><td rowspan=3 colspan=1>LyraT Parainfluenza VirusAssayPositive</td><td rowspan=1 colspan=3>Comparator: Prodesse ProParaFlu+ assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=2 colspan=1>NegativeTotal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.2% to100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.5% to 100%</td></tr></table>

When performed on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx the Lyra™ Parainfluenza Virus Assay yielded good sensitivity and specificity when compared to direct specimen fluorescent antibody (DSFA) and cell culture with DFA (CCFA).

When performed on the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx the Lyra™ Parainfluenza

Virus Assay yielded good Positive Percent Agreement and Negative Percent Agreement when compared to the Prodesse ProParaFlu $^ +$ assay.

Based on the performance data presented from the clinical study this device performs as well as the predicate device, Prodesse ProParaFlu $^ +$ (K0910533).

# 4. Clinical cut-off:

Please refer to the “Baseline, Threshold, Cut-off” sections of this document.

# 5. Expected values/Reference range:

The prevalence (positives as determined by the reference method) of PIV-1, PIV-2 and PIV-3 during the study (2013 and 2014 respiratory seasons) is presented in the table below. The highest prevalence of PIV1 was observed among adults 60 years or older $( 2 . 4 \% )$ and of PIV3 was observed among children younger than 2 years of age $( 3 . 7 \% )$ .

<table><tr><td rowspan=1 colspan=10>Combined Study  Prevalence (N=1241)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>PIV-1</td><td rowspan=1 colspan=3>PIV-2</td><td rowspan=1 colspan=3>PIV-3</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Total#</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total#</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total#</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>&lt; 2 years</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.70%</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.70%</td></tr><tr><td rowspan=1 colspan=1>2 to 5 years</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.10%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.70%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.90%</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.80%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.40%</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# N. Instrument Name:

Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx Real-Time PCR Instrument with the SDS Software version 1.4

# O. System Descriptions:

The Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument with the SDS Software version 1.4 is a real-time nucleic acid amplification and detection system that measures fluorescence and converts the signal to comparative quantitative readouts using fluorescent detection of dual-labeled hydrolysis probes. The 7500 Fast Dx is to be used only by technologists trained in laboratory techniques, procedures and on use of the analyzer.

# 1. Modes of Operation:

Instructions for programming the Lyra Direct Strep Assay, using the Version 1.4 software, are included in the package insert. The computer system is locked and controlled by ABI.

No additional software can be loaded by the end-user.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

Level of Concern: Moderate

Software Description:

The Sequence Detection Software (SDS) version 1.4 for the 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-bycycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamp-life monitoring and other instrument maintenance information. The software runs as an application on a Windows operating system.

3. Specimen Identification:

Specimen identification is described in the “Test Principle” and “Baseline, Threshold, Cutoff” sections of this document.

4. Specimen Sampling and Handling:

All specimen sampling and handling is manual.

5. Calibration: N/A

6. Quality Control:

An internal control ‘process control’ (PRC) is included in the device. It is the MS2 bacteriophage (Zeptometrix) which is a single stranded RNA virus that can be extracted, amplified and detected by primers and probes included in the master mix. Detection of the PRC indicates that assay conditions allowed proper sample extraction, amplification and detection.

No external controls are provided with this assay, but they are available from Quidel. External controls for PIV-1, PIV-2 and PIV-3 from the Lyra Parainfluenza Control Set #M1XX are recommended to serve as external positive controls. Laboratories may use previously characterized specimens that are negative for PIV-1, PIV-2 and PIV-3 as negative controls.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.