# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K141520

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for a new 510(k) application for the Alere™ i Influenza A&B assay on the Alere™ i Instrument.

# C. Measurand:

Target RNA sequences for the highly conserved regions of the PB2 gene (encodes an RNA polymerase – polymerase basic protein 2) segment of Influenza A virus and the PA gene (encodes another RNA polymerase – polymerase acidic protein) segment of Influenza B virus.

# D. Type of Test:

Qualitative isothermal nucleic acid amplification assay for the amplification and detection of specific Influenza A and B RNA gene targets.

# E. Applicant:

Alere Scarborough, Inc.

F. Proprietary and Established Names:

Alere™ i Influenza A&B Alere™ i Instrument

# G. Regulatory Information:

1. Regulation section: 21 CFR section 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code(s):

OCC - Respiratory virus panel nucleic acid assay system OZE - Influenza A and influenza B multiplex nucleic acid assay OOI - Real time nucleic acid amplification system

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Alere™ i Influenza A & B assay performed on the Alere™ i Instrument is a rapid molecular in vitro diagnostic test utilizing an isothermal nucleic acid amplification technology for the qualitative detection and discrimination of influenza A and B viral RNA in nasal swabs from patients with signs and symptoms of respiratory infection. It is intended for use as an aid in the differential diagnosis of influenza A and B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay is not intended to detect the presence of influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2012-2013 influenza season when influenza A/H3 and A/H1N1 pandemic were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Alere™ i Instrument

# I. Device Description:

# Overview

The Alere™ i Influenza A & B assay system utilizes an isothermal nucleic acid amplification technology and is comprised of:

Sample Receiver – single use, disposable containing the elution buffer Test Base – single use, disposable comprising two sealed reaction tubes, each containing a lyophilized pellet · Transfer Cartridge – single use, disposable for transfer of the eluted sample to the Test Base · Alere™ i Instrument – repeat use reader, and · Alere™ i Influenza A & B positive and negative external controls – for quality control purposes

Nasal Swabs – sterile swabs for use with the Alere™ i Influenza A & B Test.

The first reaction tube (Tube 1) in the Test Base contains the reagents required for amplification of the influenza A target viral nucleic acid, and the second reaction tube (Tube 2) in the Test Base contains reagents required for amplification of the influenza B target viral nucleic acid and an internal control synthetic RNA sequence. Alere™ i Influenza A & B utilizes pairs of templates (similar to primers in a PCR reaction) for the specific isothermal amplification of target RNA sequences from influenza A (highly conserved region of the PB2 gene), influenza B (highly conserved region of the PA gene), and the internal control, and fluorescently labeled molecular beacons designed to specifically identify the amplified products. Alere™ i Influenza A & B is performed within the confinement of the Test Base, and no other part of the Alere™ i Instrument has contact with the sample during the amplification process.

To perform the assay, the Sample Receiver and the Test Base are inserted into the Alere™ i Instrument and the elution buffer is automatically heated by the instrument. The sample is added to the Sample Receiver and transferred via the Transfer Cartridge to the Test Base, resuspending the lyophilized pellet contained within the Test Base and initiating target amplification. Heating, mixing and detection by fluorescence are provided by the instrument, with results automatically reported.

Results (positive, negative, or invalid) are displayed by the Alere™ i Instrument separately for influenza A and influenza B. Results are also stored in an on-board archive and are assigned to a sample ID that has been entered into the Alere™ i Instrument by the operator, and the date/time the test was performed. Data can be retrieved and downloaded by the operator at any time after testing. An external Alere™ Universal Printer can be attached via USB to the Alere™ i Instrument to print test results.

# Alere™ i Influenza A & B System Components

# Sample Receiver

The Sample Receiver is a foil sealed, single-use, disposable plastic component containing $2 . 5 \mathrm { m l }$ of an elution/lysis buffer. Rapid elution of influenza virus from swabs and virus lysis occur through the action of the elution/lysis buffer. The elution/lysis buffer contains salts at the appropriate concentrations that are required for the test reaction, which upon transfer of the sample using the Transfer Cartridge to the Test Base, is used to reconstitute the lyophilized pellets. The elution/lysis buffer also contains a chelating agent to remove any heavy metals found in the sample, as they are potential inhibitors of the test reaction.

# Transfer Cartridge

The Transfer Cartridge is a single-use, disposable plastic component for transfer of the eluted sample to the Test Base. The aspiration of the sample is achieved by placing the Transfer Cartridge over the Sample Receiver and pressing downwards until a click is heard and the orange indicator on the top of the Transfer Device is up. The dispense of the sample is achieved by placing the Transfer Cartridge into the Test Base and pressing on the Transfer Cartridge until the orange indicator on the top of the Transfer Cartridge descends. When the indicator descends, so does the plunger, thus driving the sample out of the two nozzles and into the reaction tubes in the Test Base and mixing with the lyophilized reagent in the reaction tubes.

# Test Base

The Test Base is a single-use, disposable plastic component comprising two sealed reaction tubes, the first reaction tube (Tube 1) is for the influenza A reaction and the second tube (Tube 2) is for the influenza B and the Internal Control reactions. Each reaction tube contains a lyophilized pellet which provides all of the necessary reagents to drive the respective reaction(s) except for salts. Tube 2 also contains an internal

control synthetic RNA sequence. The salts that are required for the test reactions are provided at their appropriate concentrations within the elution/lysis buffer contained in the Sample Receiver. Upon transfer of the sample from the Sample Receiver to the Test Base using the Transfer Cartridge, the sample in elution/lysis buffer is used to reconstitute the lyophilized pellets. The Alere™ i Influenza A & B reaction is performed within the Test Base. As soon as the sample is transferred, it comes into contact with the lyophilized pellets, reconstitutes the lyophilized pellets and initiates the assay reaction. Each Test Base contains a barcode with assay specific information (assay ID, expiration date and lot number). The barcode is read automatically from the Test Base upon insertion into the Alere™ i Instrument, providing the information necessary to run the assay.

# Alere™ i Instrument

The Alere™ i Instrument is a portable bench-top unit designed to perform in laboratory and point-of care environments. Alere™ i Influenza A & B components are color-coded to match the corresponding holder on the instrument and designed to only fit in the corresponding holder in one direction. The Alere™ i Instrument uses image analysis to automatically detect the presence of the Test Base and to check the test type and expiry date on the barcode present on the Test Base. This triggers the test sequence. Heating, mixing, and detection by fluorescence are provided by the instrument. The Alere™ i Instrument is equipped with two optical fluorescence modules, FAM™ and ROX™. Each optical module consists of a LED excitation source and photodiode receiver configured to the fluorescent channel. Fluorescence signals are optically filtered by a confocal lens and mirror arrangement and processed by software running on an onboard computer. The results are calculated automatically and presented on the instrument graphical user interface and available for export in JSON format or printing.

# Alere™ i Influenza A & B External Controls

Alere™ i Influenza A & B External Controls are designed for use with Alere™ i Influenza A & B. The Positive Control swab is coated with inactivated influenza A virus and inactivated influenza B virus dried onto a swab. The Negative Control swab is coated with inactivated Streptococcus Group C dried onto a swab.

# Nasal Swabs

Sterile swabs for use with the Alere™ i Influenza A & B test are provided as a part of the Alere™ i Influenza A & B System.

# Alere™ i Influenza A & B Workflow

An overview of the Alere™ i Influenza A & B workflow is summarized below:

1. User ID and Patient ID are entered via the touch screen prior to initiating the testing sequence.

by using the Instrument touch screen by using a barcode reader

2. The Alere™ i Influenza A & B Test Base barcode contains assay information (test type, expiry date, and lot number). Upon proper insertion into the Test Base holder of the instrument, using image analysis the instrument automatically detects the presence of the Test Base and checks the test type, expiry date, and lot number encoded in the barcode. Insertion of an expired Test Base will generate an error message on the instrument screen and the user is not able to proceed with testing. Upon confirmation of the Test Base insertion the user is instructed to insert the Sample Receiver.

3. The Alere™ i Instrument detects the insertion of the Sample Receiver into the Sample Receiver holder and automatically progresses to the warm-up state. The instrument software implements temperature control of both the Test Base and the Sample Receiver holders (heater blocks). Each of the heater blocks is managed independently with high precision temperature monitoring and control.

4. Upon completion of the warm-up state, the Alere™ i Instrument instructs the user to remove the foil seal from the Sample Receiver, insert (direct nasal swab specimen) sample to the elution buffer contained in the Sample Receiver, and mix as described on the screen.

5. The Alere™ i Instrument then instructs the user to transfer the sample in the Sample Receiver to the Test Base using the Transfer Cartridge.

6. The Alere™ i Instrument uses image analysis to confirm that the Transfer Cartridge is present in the Test Base, instructs the user to close the lid, and automatically proceeds to the measurement state upon detection of test bay lid closure. The lid operates using a magnetic switch system that detects lid closure and an electrical signal triggers testing commencement. Heating, rotation and detection by fluorescence is provided by the instrument.

7. Upon completion of the testing sequence, results are displayed on the screen and are presented to the user as positive, negative, or invalid separately for influenza A and influenza B. Upon acknowledgement of the test results, the user is instructed to dispose of the used test components.

The turnaround time for testing a sample is approximately 15 minutes.

# Quality Control

# Internal Control (Procedural Control)

Alere™ i Influenza A & B contains an internal control (IC) that was designed to control for sample inhibition, amplification, and assay reagent function for both influenza A and influenza B. The IC is an RNA oligonucleotide contains $5 '$ and $3 ^ { \circ }$ ends that are complementary to the influenza B target template set’s recognition regions, but with a spacer region (the region between the two recognition regions) that differs from the influenza B target’s spacer region. The IC is present in the second reaction tube (Tube 2) of the Test Base along with all of the necessary reagents to drive the influenza B and the IC reactions except for salts. Detection of the IC occurs via a molecular beacon that specifically detects the amplified product generated from the IC RNA oligonucleotide. The two molecular beacons present in the second reaction tube of the Test Base are labeled with fluorophores with distinct excitation/emission profiles (Influenza B – FAM and IC – ROX), enabling detection of both beacons simultaneously using a two-color detector.

In positive samples where influenza target amplification is strong, the IC result is ignored. The influenza B target amplification in Tube 2 of the Test Base also serves as the “Control” to confirm that the clinical sample was not inhibitory and that influenza A reagent performance in Tube 1 was robust. At a low frequency, clinical samples can contain inhibitors that may generate invalid results.

# Procedural Controls

In addition to the test specific Internal Control described previously, the Alere™ i instrument also employs additional “Procedural Controls” to ensure that a test is performed appropriately. Each Test Base has a barcode which identifies the test type – the instrument automatically detects the Test Base and asks the user to confirm the test type. Each of the additional steps (sample heating, sample transfer, and closing the lid, etc.) are monitored by the instrument and if the user takes too long to complete a certain step, the instrument will time out, generate an error and a result is not reported.

“Procedural Control Valid” displayed on the instrument screen indicates that the assay reagents maintained their functional integrity and the sample did not significantly inhibit assay performance.

# External Positive and Negative Controls

Good laboratory practice suggests the use of positive and negative controls to ensure that test reagents are working and that the test is correctly performed. Alere™ i Influenza A & B kits contain Positive and Negative Control Swabs. These swabs monitor the entire assay. Test these swabs once with each new shipment received and once for each new operator. Further controls may be tested in order to conform to local, state and/or federal regulations, accrediting groups, or laboratory’s standard Quality Control procedures.

# Results Interpretation

The Alere™ i Instrument software automatically interprets test results. Alere™ i Influenza A & B assay operates in the Test Base with two separate reaction tubes, Tube 1 and Tube 2, for influenza A and influenza B plus IC respectively. The assay uses the following three fluorescent channels: Flu A (FAM) in Tube 1, Flu B (FAM) in Tube 2, and IC (ROX) in Tube 2. The ROX channel in Tube 1 is not used. Each channel of each reaction tube is first individually assessed using a defined channel decision algorithm and assigned a channel result as “Asserted”, “Not Asserted” or “Indeterminate”. The individual channel results are then combined to arrive at the influenza A and influenza B results. All nine possible assay result combinations and the respective underlying assay decision logic rules are presented in Table 1 below:

Table 1: All Possible Assay Result Combinations and Respective Underlying Assay Decision Logic Rules   

<table><tr><td rowspan=1 colspan=1>Assay Result Reported</td><td rowspan=1 colspan=3>Underlying Assay Decision Logic Rules</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A Channel(FAM) in Tube 1</td><td rowspan=1 colspan=1>Flu B Channel(FAM) in Tube 2</td><td rowspan=1 colspan=1>IC Channel (ROX)in Tube 2</td></tr><tr><td rowspan=1 colspan=1>Flu A Positive/ Flu B Negative</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=5 colspan=1>Flu A Positive/ Flu B Invalid</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>Flu A Positive/ Flu B Positive</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>Flu A Negative/ Flu B Positive</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Invalid</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Negative</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=3 colspan=1>Flu A Invalid/Flu B Positive</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Flu A Invalid/Flu B Negative</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=9 colspan=1>Flu A Invalid/Flu B Invalid</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr></table>

All possible assay result combinations and the respective interpretations and follow-up actions are presented

in Table 2 below:

Table 2: All Possible Assay Result Combinations and Respective Interpretations and Follow-up Actions   

<table><tr><td rowspan=1 colspan=1>Assay Result Reported</td><td rowspan=1 colspan=1>Interpretation of Results and Follow-up Actions</td></tr><tr><td rowspan=1 colspan=1>Flu A Positive/ Flu B Negative</td><td rowspan=1 colspan=1>Flu A Viral RNA Detected; Flu B Viral RNA Not Detected.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype.</td></tr><tr><td rowspan=1 colspan=1>Flu A Positive/ Flu B Invalid</td><td rowspan=1 colspan=1>Flu A Viral RNA Detected; The presence or absence of Flu BViral RNA can not be determined.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype.</td></tr><tr><td rowspan=1 colspan=1>Flu A Positive/ Flu B Positive</td><td rowspan=1 colspan=1>Flu A Viral RNA Detected; Flu B Viral RNA Detected.Dual infections of Flu A and Flu B are rare. Repeat testingusing new test components. Contact Technical Support duringnormal business hours if multiple samples provide this result.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype or specificinfluenza B virus lineage.</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Positive</td><td rowspan=1 colspan=1>Flu A Viral RNA Not Detected; Flu B Viral RNA Detected.This result does not rule out co-infections with other pathogensor identify any specific influenza B virus lineage.</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Invalid</td><td rowspan=1 colspan=1>Flu A Viral RNA Not Detected; The presence or absence of FluB Viral RNA can not be determined.Infection due to Flu B can not be ruled out. Repeat testing ofthe sample using new test components. If repeated Flu BInvalid results are obtained, results should be confirmed byanother method prior to reporting the results.</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Negative</td><td rowspan=1 colspan=1>Flu A Viral RNA Not Detected; Flu B Viral RNA NotDetected.</td></tr><tr><td rowspan=1 colspan=1>Flu A Invalid/Flu B Positive</td><td rowspan=1 colspan=1>The presence or absence of Flu A Viral RNA can not bedetermined. Flu B Viral RNA Detected;This result does not rule out co-infections with other pathogensor identify any specific influenza B virus lineage.</td></tr><tr><td rowspan=1 colspan=1>Flu A Invalid/Flu B Negative</td><td rowspan=1 colspan=1>The presence or absence of Flu A Viral RNA can not bedetermined. Flu B Viral RNA Not Detected;Infection due to Flu A can not be ruled out. Repeat testing ofthe sample using new test components. If repeated Flu AInvalid results are obtained, results should be confirmed byanother method prior to reporting the results.</td></tr><tr><td rowspan=1 colspan=1>Flu A Invalid/Flu B Invalid</td><td rowspan=1 colspan=1>The presence or absence of Flu A and Flu B Viral RNAs cannot be determined.Repeat testing of the sample using new test components. Ifrepeated Flu A and Flu B Invalid results are obtained, resultsshould be confirmed by another method prior to reporting theresults.</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): IQuum Liat™ Influenza A/B Assay

2. Predicate 510(k) number: K111387

3. Comparison with predicates:

<table><tr><td rowspan=1 colspan=3>Similarities and Differences</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>AlereTM i Influenza A &amp; B (K141520)</td><td rowspan=1 colspan=1>IQuum LiatTM Influenza A/B Assay (K111387)</td></tr><tr><td rowspan=1 colspan=1>Assay Analyte</td><td rowspan=1 colspan=1>Influenza A and Influenza B</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The AlereTM i Influenza A &amp; B assay performed onthe Alere™ i Instrument is a rapid molecular in vitrodiagnostic test utilizing an isothermal nucleic acidamplification technology for the qualitative detectionand discrimination of influenza A and B viral RNAin nasal swabs from patients with signs andsymptoms of respiratory infection. It is intended foruse as an aid in the differential diagnosis of influenzaA and B viral infections in humans in conjunctionwith clinical and epidemiological risk factors. Theassay is not intended to detect the presence ofinfluenza C virusNegative results do not preclude influenza virusinfection and should not be used as the sole basis fordiagnosis, treatment or other patient managementdecisions.Performance characteristics for influenza A wereestablished during the 2012-2013 influenza seasonwhen influenza A/H3 and A/H1N1 pandemic werethe predominant influenza A viruses in circulation.When other influenza A viruses are emerging,performance characteristics may vary.If infection with a novel influenza A virus issuspected based on current clinical andepidemiological screening criteria recommended bypublic health authorities, specimens should becollected with appropriate infection controlprecautions for novel virulent Influenza viruses andsent to state or local health department for testing.Viral culture should not be attempted in these casesunless a BSL 3+ facility is available to receive andculture specimens.</td><td rowspan=1 colspan=1>The IQuum LiatTM Influenza A/B Assay performed onthe LiatτM Analyzer is an automated multiplex real-time RT-PCR assay for the rapid in vitro qualitativedetection and discrimination of influenza A virus andinfluenza B virus RNA in nasopharyngeal swabspecimens from patients with signs and symptoms ofrespiratory infection in conjunction with clinical andepidemiological risk factors. The test is intended foruse as an aid in the differential diagnosis of influenza Aand influenza B in humans and is not intended to detectinfluenza C.Negative results do not preclude influenza virusinfection and should not be used as the sole basis fortreatment or other patient management decisions.Conversely, positive results do not rule-out bacterialinfection or co-infection with other viruses. The agentdetected may not be the definite cause of disease.Performance characteristics for influenza A wereestablished when influenza A/H1 and A/H3 were thepredominant influenza A viruses in circulation. Whenother influenza A viruses are emerging, performancecharacteristics may vary.If infection with a novel influenza A virus is suspectedbased on current clinical and epidemiological screeningcriteria recommended by public health authorities,specimens should be collected with appropriateinfection control precautions for novel virulentInfluenza viruses and sent to state or local healthdepartment for testing. Viral culture should not beattempted in these cases unless a BSL 3+ facility isavailable to receive and culture specimens.</td></tr><tr><td rowspan=1 colspan=1>IntendedEnvironment forUse</td><td rowspan=1 colspan=1>Professional use, in a medical laboratory or point-of-care</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>Alere i Instrument</td><td rowspan=1 colspan=1>Liat Analyzer</td></tr><tr><td rowspan=1 colspan=1>Self-contained System</td><td rowspan=1 colspan=1>Integrated PC, Software, and Touch Screen Display</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Automated Assay</td><td rowspan=1 colspan=1>Yes. Sample preparation, amplification, detectionand result interpretation</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Direct Nasal Swab</td><td rowspan=1 colspan=1>Nasopharyngeal Swab in UTM</td></tr><tr><td rowspan=1 colspan=1>Influenza A ViralTarget</td><td rowspan=1 colspan=1>PB2 gene segment</td><td rowspan=1 colspan=1>Matrix Gene</td></tr><tr><td rowspan=1 colspan=1>Influenza B ViralTarget</td><td rowspan=1 colspan=1>PA gene segment</td><td rowspan=1 colspan=1>Non-Structural Protein (NSP) Gene</td></tr><tr><td rowspan=1 colspan=1>Amplification andDetection Technology</td><td rowspan=1 colspan=1>Isothermal nucleic acid amplification and detectionof specific amplification products using molecularbeacon probes</td><td rowspan=1 colspan=1>Real-time RT-PCR using TaqMan probes</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Same</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None

# L. Test Principle:

Individual reactions for influenza A and B and IC, provided as lyophilized pellets contained within the Test Base, specifically amplify unique target regions within the RNA genomes of these two viruses and the IC. Alere™ i Influenza A & B employs fluorescently-labeled molecular beacon probes for real-time detection of amplified products. Alere™ i Influenza A & B amplification technology, Nicking and Extension Amplification Reaction (NEAR) is isothermal and does not require a heat denaturation step to generate a single strand target for amplification. Instead a thermostable, strand-displacing DNA polymerase, a thermostable nicking endonuclease, and two oligonucleotides (typically known as primers, but are referred to as templates in this technology) are utilized. Reverse transcriptase is also present in the reaction. The products of Alere™ i Influenza A & B are two complementary oligonucleotides 30 – 40 nucleotides in length. A product of this size is of sufficient length to be highly unique amongst genomes. The exact length of the products depends on the specific placement of the templates relative to one another and relative to their complementary sequences within the target genome. Alere™ i Influenza A & B assay specificity is achieved through two levels of sequence-specific interrogation, template annealing to complementary target sequence at an elevated temperature to drive target-specific amplification, and amplification product detection as the molecular beacon anneals to Alere™ i Influenza A & B amplification product in a sequence-specific fashion.

# Alere™ i Influenza A & B Isothermal Amplification Mechanism

Alere™ i Influenza A & B isothermal amplification, NEAR, is driven by a thermostable strand displacing DNA polymerase, thermostable nicking endonuclease, two oligonucleotide templates, plus a reverse transcriptase. The Alere™ i Influenza A & B oligonucleotide template (primer) consists of three distinct regions. The $5 ^ { \circ }$ -end of the template is known as the stabilizing region (SR) which is not involved in targetspecific complementary sequence-dependent annealing, but rather in stabilizing the double stranded products generated during amplification. The central region of each template maintains the nicking endonuclease binding site (NEBS) and nicking endonuclease cut site (NECS), which also are not complementary to the target sequence, but provide the sequence required for single strand nicking activity by the nicking endonuclease. At the $3 ^ { \circ }$ -end of each template is the recognition region (RR), which provides target-specific, sequence-dependent annealing. Typically, the RR is 11 – 12 nucleotides in length on each template. As shown in Figure 1 below, step 1a, the recognition region (RR) of template T2 (labeled as Y), anneals to a complimentary sequence present in the single strand DNA target (labeled as $\mathrm { Y } '$ ). In step 2a, a DNA polymerase extends T2 from its $3 ^ { \circ }$ -OH group generating sequence X which is complementary to target sequence $X ^ { \prime }$ . In this step only the $3 ^ { \circ }$ end of T2 anneals to a target sequence in a complementary fashion. The SR, NEBS and NECS regions of the template are not complementary to the target sequence. In step 3a, a second T2 template displaces the original, partially annealed T2. DNA polymerase extension from this second T2 displaces the first synthesized strand. T1 recognizes, anneals to and extends from sequence X of the displaced strand (step 4a). This leaves a product containing a double strand NEBS and NECS (step 5a). A nicking endonuclease will then bind to its recognition site and nick the structure at its cut site. This generates an accessible $3 ^ { \circ }$ -OH that can be extended by the DNA polymerase, resulting in the release of a single strand DNA (containing regions X and Y) and generation of a double strand DNA product containing NEBS and NECS on both ends (step 6a), known as the Amplification Duplex. Figure 1 depicts the linear phase of the mechanism, essentially the generation of the NEAR Amplification Duplex.

![](images/05b7e6e91b84d8bdcd0b6770f29889623891ffd04a533afa6868fa0059512905.jpg)  
Figure 1 – Alere™ i Influenza A & B Mechanism – Linear Phase

In Figure 2 below, the geometric amplification phase is depicted. The NEAR Amplification Duplex is a double strand DNA structure containing two complete nicking endonuclease binding and cut sites (1b). As shown in 2b, nicking of the top and bottom strands occurs, providing the DNA polymerase two $3 ^ { \circ }$ -OH groups for initiating strand displacement extension. This results in the release of $\mathbf { \boldsymbol { X } } ^ { \prime } \mathbf { \boldsymbol { Y } } ^ { \prime }$ (specific product 1) and X Y (specific product 2) products, and the generation of two double strand DNA moieties containing single nicking endonuclease binding and cut sites (3b). The two moieties undergo reiterative nicking and extension, generating additional specific products 1 and 2 (3b). Specific products 1 and 2 are substrates for T2 and T1 (4b). Extension of these complexes (4b) generates additional double strand DNA moieties containing single nicking endonuclease binding and cut sites (3b), with the result of this reiterative nicking and extension being geometric amplification.

![](images/e2dba3bb6b6f7f38ddb2c438dfd5f649110ffa9948ab82a27318687b743374bd.jpg)  
Figure 2 – Alere™ i Influenza A & B Amplification Duplex – Geometric

The intentional skewing of templates leading to asymmetric amplification product formation is necessary for optimal performance of the real-time fluorescence-based molecular beacon detection system coupled to the Alere™ i Influenza A & B amplification technology. Molecular beacons are single-stranded DNA sequences that are able to form hairpin loop structures. The hairpin is formed by complementary $5 ^ { \circ }$ and $3 ^ { \circ }$ - ends that are typically 4 – 7 nucleotides in length. Molecular beacons contain a fluorophore and a quencher, typically at opposite ends of the molecular beacon’s linear nucleotide sequence. When the molecular beacon forms a hairpin structure, the fluorophore and quencher are in close proximity, and fluorescence is quenched. Upon opening of the hairpin, which can occur when a complementary DNA or RNA species anneals to the molecular beacon’s loop region, the distance between the fluorophore and quencher is lengthened, reducing the ability of the quencher to quench the fluorescence signal. As a result, fluorescence is emitted and can be measured using an optical reader. The amount of fluorescence released is directly proportional to the amount of molecular beacon that has been opened, and concomitantly, the amount of complementary DNA or RNA species present in the sample. The result of the Alere™ i Influenza A & B NEAR amplification reaction is two specific products which are perfect complements to one another. These two specific products form a very stable double strand DNA structure at assay temperature. In order to combine the use of the Alere™ i Influenza A & B NEAR technology and molecular beacons, the concentrations of the two templates present in the Alere™ i Influenza A & B assay are skewed relative to one another to drive asymmetric product formation, i.e., one product is made to a larger extent than the other. The product that is generated in excess is then targeted by the molecular beacon. Annealing of the loop region of the molecular beacon to the Alere™ i Influenza A & B product generated in excess results in opening of the hairpin stem, separation of the fluorophore and quencher, and ultimately fluorescence signal release.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility Study

Reproducibility performance was evaluated for the Alere™ i Influenza A & B at three testing sites. Six Alere™ i instruments (per study site) were utilized at Site 1 and Site 2, and four Alere™ i instruments were utilized at Site 3. Three operators (Operator 1, 2, and 3) participated in the study at each site. Two Alere™ i Influenza A & B reagent lots were used in the Reproducibility Study.

The reproducibility panel consisted of seven samples at virus concentrations near the respective LoD (i.e., Moderate Positive Influenza A, Low Positive Influenza A, High Negative Influenza A, Moderate Positive Influenza B, Low Positive Influenza B, High Negative Influenza B, and True Negative). Each sample was prepared using the influenza A and B strains (described above) spiked into simulated nasal swab matrix, the Universal Transport Medium (UTM). The influenza A strain used in the reproducibility study was Influenza A/Puerto Rico/8/34 (Lot J1215A, inactivated), and the influenza B strain used in the study was Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated). The targeted concentrations for the Moderate Positive (MP) samples were approximately 2 to $3 \mathrm { ~ X ~ }$ the respective LoD, the targeted concentrations for the Low Positive (LP) samples were approximately 1 X the respective LoD $\mathrm { C } _ { 9 5 }$ concentration), and the targeted concentrations for the High Negative (HN) samples were less than the respective LoD $C _ { 5 }$

concentration). The True Negative samples contain no influenza virus.

Contrived nasal swab samples were prepared using the influenza A and B strains (described above) dilutions in UTM to the targeted concentrations. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents $ { / \mathrm { m L } }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilution onto the swab. Swab coating was done by lightly scratching the swab surface with the pipette tip as the dilution was pipetted to ensure liquid was absorbed into the swab tip. Once all solution was absorbed, swabs were allowed to dry and assembled into blinded panels.

Samples in each of the seven categories were tested twice by each of the three operators per study site, on five distinct days, at three clinical sites, using two lots of reagents (Site-to-Site: 2 tests/day/operator x 3 operators/per site $\texttt { X 5 }$ days x 3 sites $= 9 0$ data points for each sample category).

The influenza A and influenza B strains were tested as follows:

Moderate Positive (MP) influenza A: $1 . 4 7 \times { { 1 0 } ^ { 4 } }$ TCID50/Swab $( 2 . 3 8 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $1 . 4 7 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ $2 . 3 8 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL) virus dilution coated onto a swab] Low Positive (LP) influenza A: 4.91 x 103 TCID50/Swab $( 7 . 9 5 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $4 . 9 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $7 . 9 5 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab] High Negative (HN) influenza $\mathrm { A } : 6 . 8 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ T $\mathrm { C I D } _ { 5 0 ^ { \prime } }$ /Swab $( 1 . 1 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $6 . 8 0 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $1 . 1 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents/mL) virus dilution coated onto a swab] Moderate Positive (MP) influenza B: $5 . 0 4 \times 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { S w a b }$ $( 6 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $5 . 0 4 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ ${ \bf \dot { 6 } } . 0 0 { \bf x } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab] Low Positive (LP) influenza B: $1 . 6 8 \times 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { S w a b } ( 2 . 0 0 \times 1 0 ^ { 2 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $1 . 6 8 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { \bar { 4 } }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab] High Negative (HN) influenza B: $4 . 2 0 \times 1 0 ^ { - 1 }$ T $\mathrm { \ C I D } _ { 5 0 }$ /Swab $( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 0 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $4 . 2 0 \times 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/mL) virus dilution coated onto a swab] True negative (TN): Negative samples (no virus)

All samples were tested using the Alere™ i Influenza A & B on the Alere™ i instruments, according to the product instructions. Each operator tested one external positive and one external negative control swab on each instrument used for the study, on each day of testing prior to performing the study testing.

Operator 2 at Site 2 generated a single invalid result for the external negative control swab tested on Day 1 of the Reproducibility Study. The expected negative result was obtained upon repeat testing per the product instructions. All other external positive and negative control swabs tested in the Reproducibility Study generated the expected results each day of testing at all three sites.

On Day 1 of the Reproducibility Study, Operator 2 at Site 1 generated a “Flu A Invalid/Flu B

Negative” result for a TN sample, and a “Flu A Invalid/Flu B Invalid” result for a HN influenza B and a MP influenza B sample. Upon repeat testing per the product instructions, valid expected results were obtained for all three samples. No other invalid results were obtained. These repeated test results were included in the Reproducibility Study results presented in Table 3 and Table 4 below.

For the Site-to-Site Reproducibility Study, the overall percent agreement with expected result (i.e., negative for TN and HN samples, positive for MP and LP samples) was $100 \%$ (90/90, $9 5 \%$ CI, $9 5 . 9 \% - 1 0 0 \% )$ for the influenza A MP, the influenza A LP, the influenza $\mathbf B \mathbf M \mathbf P$ , and the TN categories; $9 2 . 2 \%$ (83/90, $9 5 \%$ CI, $8 4 . 8 \% - 9 6 . 2 \% )$ for the influenza B LP category; and $7 0 . 0 \%$ (63/90, $9 5 \%$ CI, $5 9 . 9 \% - 7 8 . 5 \% )$ and $9 0 . 0 \%$ (81/90, $9 5 \%$ CI, $8 2 . 1 \% - 9 4 . 6 \% )$ for the influenza A HN and the influenza B HN categories, respectively.

The Reproducibility Study site-to-site qualitative results (agreements with expected results) are presented in Table 3 below:

Table 3: Reproducibility Study Site-To-Site Qualitative Results   

<table><tr><td rowspan=3 colspan=1>SampleCategory</td><td rowspan=1 colspan=6>SITE</td><td rowspan=3 colspan=2>Overall PercentAgreement and 95% CI</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td></tr><tr><td rowspan=1 colspan=1>HN1Influenza A</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>24/30</td><td rowspan=1 colspan=1>63.3%</td><td rowspan=1 colspan=1>19/30</td><td rowspan=1 colspan=1>70.0%(63/90)</td><td rowspan=1 colspan=1>(59.9%, 78.5%)</td></tr><tr><td rowspan=1 colspan=1>LPInfluenza A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100%)</td></tr><tr><td rowspan=1 colspan=1>MPInfluenza A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100%)</td></tr><tr><td rowspan=1 colspan=1>HN1Influenza B</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>25/30</td><td rowspan=1 colspan=1>90.0%(81/90)</td><td rowspan=1 colspan=1>(82.1%, 94.6%)</td></tr><tr><td rowspan=1 colspan=1>LPInfluenza B</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>92.2%(83/90)</td><td rowspan=1 colspan=1>(84.8%, 96.2%)</td></tr><tr><td rowspan=1 colspan=1>MPInfluenza B</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>(95.9%, 100%)</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

The Reproducibility Study operator-to-operator qualitative results (agreements with expected results) are presented in Table 4 below:

Table 4: Reproducibility Study Operator-To-Operator Qualitative Results   

<table><tr><td colspan="2" rowspan="1">Sample Category</td><td colspan="1" rowspan="1">HN1Influenza A</td><td colspan="1" rowspan="1">LPInfluenza A</td><td colspan="1" rowspan="1">MPInfluenza A</td><td colspan="1" rowspan="1">HN1Influenza B</td><td colspan="1" rowspan="1">LPInfluenza B</td><td colspan="1" rowspan="1">MPInfluenza B</td><td colspan="1" rowspan="1">TN</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Operator</td><td colspan="1" rowspan="1">Count (%AAgrement)</td><td colspan="1" rowspan="1">Count (%Agreement)</td><td colspan="1" rowspan="1">Count (%Agrement)</td><td colspan="1" rowspan="1">Count (%Agrement)</td><td colspan="1" rowspan="1">Count (%Agrement</td><td colspan="1" rowspan="1">Count (%Agreement)</td><td colspan="1" rowspan="1">Count (%Agrement)</td><td colspan="1" rowspan="1">Count (%Agrment)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Site 1</td><td colspan="1" rowspan="2">Operator 1</td><td colspan="1" rowspan="2">6/10(60.0%)</td><td colspan="1" rowspan="2">10/10(100%)</td><td colspan="1" rowspan="2">10/10(100%)</td><td colspan="1" rowspan="2">9/10(90.0%)</td><td colspan="1" rowspan="2">9/10(90.0%)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="2">10/10(100%)</td><td colspan="1" rowspan="2">64/70(91.4%)</td><td colspan="1" rowspan="2">82.5% -96.0%</td></tr><tr><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">Operator 2</td><td colspan="1" rowspan="1">6/10(60.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">9/10(90.0%)</td><td colspan="1" rowspan="1">9/10(90.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">64/70(91.4%)</td><td colspan="1" rowspan="1">82.5% -96.0%</td></tr><tr><td colspan="1" rowspan="1">Operator 3</td><td colspan="1" rowspan="1">8/10(80.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">8/10(80.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">66/70(94.3%)</td><td colspan="1" rowspan="1">86.2% -97.8%</td></tr><tr><td colspan="1" rowspan="2">Site 2</td><td colspan="1" rowspan="1">Operator 1</td><td colspan="1" rowspan="1">8/10(80.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">9/10(90%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">67/70(95.7%)</td><td colspan="1" rowspan="1">88.1% -98.5%</td></tr><tr><td colspan="1" rowspan="1">Operator 2</td><td colspan="1" rowspan="1">6/10(60.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">7/10(70%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">63/70(90.0%)</td><td colspan="1" rowspan="1">80.8% -95.1%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Operator 3</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">70/70(100%)</td><td colspan="1" rowspan="1">94.8% -100%</td></tr><tr><td colspan="1" rowspan="3">Site 3</td><td colspan="1" rowspan="1">Operator 1</td><td colspan="1" rowspan="1">5/10(50%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">8/10(80.0%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">63/70(90.0%)</td><td colspan="1" rowspan="1">80.8% -95.1%</td></tr><tr><td colspan="1" rowspan="1">Operator 2</td><td colspan="1" rowspan="1">8/10(80%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">8/10(80.0%)</td><td colspan="1" rowspan="1">9/10(90%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">65/70(92.9%)</td><td colspan="1" rowspan="1">84.3% -96.9%</td></tr><tr><td colspan="1" rowspan="1">Operator 3</td><td colspan="1" rowspan="1">6/10(60%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">9/10(90%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">65/70(92.9%)</td><td colspan="1" rowspan="1">84.3% -96.9%</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

The Reproducibility Study instrument-to-instrument qualitative results (agreements with expected results) are presented in Table 5 below:

Table 5: Reproducibility Study Instrument-To-Instrument Qualitative Results   

<table><tr><td rowspan=1 colspan=2>Sample Category</td><td rowspan=1 colspan=1>HNInfluenza A</td><td rowspan=1 colspan=1>LPInfluenza A</td><td rowspan=1 colspan=1>MPInfluenza A</td><td rowspan=1 colspan=1>HNTInfluenza B</td><td rowspan=1 colspan=1>LPInfluenza B</td><td rowspan=1 colspan=1>MPInfluenza B</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Count (%Agreement</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>Count (%Agreement</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>Count (%Agreement)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>Site1</td><td rowspan=1 colspan=1>Instrument 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>51.0%-00%</td></tr><tr><td rowspan=1 colspan=1>Instrument 2</td><td rowspan=1 colspan=1>3/9 (33.3%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>7/8 (87.5%)</td><td rowspan=1 colspan=1>4/5 (90%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>50/58 (86.2%)</td><td rowspan=1 colspan=1>75.1%-9/8%</td></tr><tr><td rowspan=1 colspan=1>Instrument 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%-00%</td></tr><tr><td rowspan=1 colspan=1>Instrument 4</td><td rowspan=1 colspan=1>7/8 (87.5%)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>9/10 (90%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>63/65 (96.9%)</td><td rowspan=1 colspan=1>89.5% -99.2%</td></tr><tr><td rowspan=1 colspan=1>Instrument 5</td><td rowspan=1 colspan=1>7/10 (70%)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>7/9 (77.8%</td><td rowspan=1 colspan=1>3/4 (75%)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>53/59 (89.8%)</td><td rowspan=1 colspan=1>79.5% -95.3%</td></tr><tr><td rowspan=1 colspan=1>Instrument 6</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>19/19 (100%)</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=6 colspan=1>Site2</td><td rowspan=1 colspan=1>Instrument 1</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>24/26 (92.3%)</td><td rowspan=1 colspan=1>75.9%-97.9%</td></tr><tr><td rowspan=1 colspan=1>Instrument 2</td><td rowspan=1 colspan=1>4/6 (66.7)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>7/7 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>10/10(10%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>44/46 (95.7%)</td><td rowspan=1 colspan=1>85.5%-98.8%</td></tr><tr><td rowspan=1 colspan=1>Instrument 3</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>14/16 (87.5%)</td><td rowspan=1 colspan=1>64.0%-966.5%</td></tr><tr><td rowspan=1 colspan=1>Instrument 4</td><td rowspan=1 colspan=1>11/13(84.6%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>47/49 (95.9%)</td><td rowspan=1 colspan=1>86.3%-9.9%</td></tr><tr><td rowspan=1 colspan=1>Instrument 5</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>4/6 (66.7%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>25/27 (92.6%)</td><td rowspan=1 colspan=1>76.6%-97..9%</td></tr><tr><td rowspan=1 colspan=1>Instrument 6</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>7/7 (100%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>7/7 (100%)</td><td rowspan=1 colspan=1>47/47(100%)</td><td rowspan=1 colspan=1>92.4%-100%</td></tr><tr><td rowspan=4 colspan=1>Site3</td><td rowspan=1 colspan=1>Instrument 1</td><td rowspan=1 colspan=1>6/8 (75%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>6/8 (75%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/74 (94.6%)</td><td rowspan=1 colspan=1>86.9%-97.9%</td></tr><tr><td rowspan=1 colspan=1>Instrument 2</td><td rowspan=1 colspan=1>7/12 (58.3%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>10/12(83.3%)</td><td rowspan=1 colspan=1>15/16(93.8%)</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>68/76 (89.5%)</td><td rowspan=1 colspan=1>80.6% -946%</td></tr><tr><td rowspan=1 colspan=1>Instrument 3</td><td rowspan=1 colspan=1>4/6 (66.7%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>28/30 (93.3%)</td><td rowspan=1 colspan=1>78.7% -98.2%</td></tr><tr><td rowspan=1 colspan=1>Instrument 4</td><td rowspan=1 colspan=1>2/4 (50%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>3/4 (75%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>27/30 (90.0%)</td><td rowspan=1 colspan=1>74.4% -96.5%</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

Due to the fact that evaluating reagent kit lot to lot reproducibility was not addressed by the Reproducibility Study design, although two reagent kit lots were included in the Reproducibility Study, most of the data points were generated using Lot 1. To assess reagent kit lot to lot reproducibility of the Alere™ i Influenza A & B, the results of the “Near the Cut Off” study were utilized since the same sample categories that were tested in the Reproducibility Study were also tested in the “Near the Cut Off” study. The reproducibility assessment lot-to-lot qualitative results (agreements with expected results) are presented in Table 6 below:

Table 6: Reproducibility Study Lot-To-Lot Qualitative Results   

<table><tr><td rowspan=3 colspan=1>SampleCategory</td><td rowspan=1 colspan=6>Reagent Lot</td><td rowspan=3 colspan=1>Overall Countand PercentAgreement</td><td rowspan=3 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=2>Lot 1</td><td rowspan=1 colspan=2>Lot 2</td><td rowspan=1 colspan=2>Lot 3</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td></tr><tr><td rowspan=1 colspan=1>HN1Influenza A</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>63/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>46.7%</td><td rowspan=1 colspan=1>28/60</td><td rowspan=1 colspan=1>91/150(60.7%)</td><td rowspan=1 colspan=1>52.7% - 68.1%</td></tr><tr><td rowspan=1 colspan=1>LPInfluenza A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>150/150(100%)</td><td rowspan=1 colspan=1>97.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>MPInfluenza A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>HN1Influenza B</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>81/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>55/60</td><td rowspan=1 colspan=1>136/150(90.7%)</td><td rowspan=1 colspan=1>84.9% - 94.4%</td></tr><tr><td rowspan=1 colspan=1>LPInfluenza B</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>60/62</td><td rowspan=1 colspan=1>82.1%</td><td rowspan=1 colspan=1>23/28</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>141/150(94.0%)</td><td rowspan=1 colspan=1>89.0% - 96.8%</td></tr><tr><td rowspan=1 colspan=1>MPInfluenza B</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>150/150(100%)</td><td rowspan=1 colspan=1>97.5% - 100%</td></tr></table>

1Percent Agreement correlates to the percent of negative results.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

External Positive and Negative Controls

Alere™ i Influenza A & B kits contain Positive and Negative Control Swabs. These swabs monitor the entire assay. The Alere™ i Influenza A & B Instructions for Use requires testing these control swabs once with each new shipment received and once for each new operator. Further controls may be tested in order to conform to local, state and/or federal regulations, accrediting groups, or laboratory’s standard Quality Control procedures.

During the prospective clinical study, control testing using external positive and negative controls was performed prior to sample testing each day and on each instrument Alere™ i Influenza A & B testing was performed, at all eight study sites. A total of 335 external positive controls and 335 external negative controls were tested during the prospective clinical study. Seven external positive controls (7/335, $2 . 1 \%$ , $9 5 \%$ CI, $1 . 0 \% - 4 . 2 \%$ and nine external negative controls (9/335, $2 . 7 \%$ , $9 5 \%$ CI, $1 . 4 \% - 5 . 0 \%$ ) failed initially, but succeeded on repeat testing per the product instructions.

# Natural Nasal Swab Matrix and Simulated Nasal Swab Matrix Equivalency

Negative natural clinical nasal swab matrix is difficult to obtain in large volumes to be used as diluent in various analytical studies and the Reproducibility Study. In order to compensate for the limited availability of negative natural clinical nasal swab matrix, a simulated nasal swab matrix, the Universal Transport Medium (UTM), was used as diluent in the Reproducibility Study and in a number of analytical studies (i.e., Carryover Contamination Study, Swab Type Validation Study, Reagent Shelf Life Study, Interfering Substances Study, Competitive Interference with NonInfluenza Respiratory Viruses Study, and Competitive Interference with Influenza Viruses Study).

Limit of Detection Studies (UTM vs. natural clinical nasal swab matrix in UTM were carried out to demonstrate the equivalency between the natural clinical nasal swab matrix and the simulated nasal swab matrix.

The anchoring influenza strains used in the LoD studies were Influenza A/Puerto Rico/8/34 (Lot D1002B - inactivated, and Lot G1112B), and Influenza B/Malaysia/2506/2004 (Lot B1012C - inactivated, and Lot C1320E).

For the LoD study using UTM as the diluent, dilutions of each of the two anchoring influenza strains (described above) were prepared in UTM. $1 0 ~ \mu \mathrm { l }$ of each dilution was dispensed onto a swab, using the pipette tip to scratch and rub liquid onto the swab tip. The swab sample was then tested per the product instructions. A total of 20 replicates were tested at the estimated LoD level.

For the LoD study using natural clinical nasal swab matrix in UTM as the diluent, presumed negative natural clinical nasal swabs were eluted in $3 ~ \mathrm { m L }$ of UTM. Swab eluates were combined and mixed thoroughly to create a swab eluate pool. Dilutions of each of the two anchoring influenza strains (described above) were prepared in this swab eluate pool. $1 0 ~ \mu \mathrm { l }$ of each dilution was dispensed onto each swab, using the pipette tip to scratch and rub liquid onto the swab tip. The swab sample was then tested per the product instructions. A total of 20 replicates were tested at the estimated LoD level.

The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method by the venders and confirmed by the sponsor (except for the inactivated virus lots, where the $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ values were determined by standard virologic method by the venders prior to inactivation). The concentration for each dilution (in genome equivalents/mL) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

The estimated LoD concentration (defined as the lowest concentration at which $\ge 9 5 \%$ of all replicates tested positive), presented in both calculated $\mathrm { T C I D } _ { 5 0 }$ /reaction and genome equivalents/reaction, per virus strain tested, is presented in Table 7 below.

Table 7: Natural Nasal Swab Matrix and Simulated Nasal Swab Matrix Equivalency Demonstration   

<table><tr><td rowspan=1 colspan=1>A/Puerto Rico/8/34 (H1N1)</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>TCID50/Reaction</td><td rowspan=1 colspan=1>GenomeEquivalents/Reaction</td></tr><tr><td rowspan=1 colspan=1>LoD Study -Clinical Matrix</td><td rowspan=1 colspan=1>G1112B</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>1688</td></tr><tr><td rowspan=1 colspan=1>LoD Study Without Clinical Matrix</td><td rowspan=1 colspan=1>D1002B (inactivated)</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>304</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>TCID50/Reaction</td><td rowspan=1 colspan=1>GenomeEquivalents/Reaction</td></tr><tr><td rowspan=1 colspan=1>LoD Study -Clinical Matrix</td><td rowspan=1 colspan=1>C1320E</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>LoD Study Without Clinical Matrix</td><td rowspan=1 colspan=1>B1012C (inactivated)</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>8</td></tr></table>

This analysis demonstrated that the Alere™ i Influenza A & B assay LoDs using simulated nasal swab matrix (i.e., UTM) are similar to those using natural nasal swab matrix. Therefore, the simulated nasal swab matrix is appropriate and adequate to be used in place of natural nasal swab matrix for the analytical studies, including the Reproducibility Study.

# Specimen Stability Study

To provide data supporting the specimen storage recommendations stated in the product package insert, an analytical study was carried out to evaluate specimen stability.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated) were used in this study to prepare contrived samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive nasal swab samples were prepared using the influenza A and B strains (described above) dilutions in saline with targeted concentrations near the respective LoD levels (approximately 2 to 3 X the respective LoD). Mock negative nasal swab samples were prepared using saline. Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilution or saline onto the swab. Swab coating was done by lightly scratching the swab surface with the pipette tip as the dilution was pipetted to ensure liquid was absorbed into the swab tip. All prepared swabs were tested in triplicates at each of the following three time points/conditions: Time 0, Hour 3 (after being stored at approximately1 $8 { - } 2 2 ^ { \circ } \mathrm { C }$ for 3 hours), and Hour 25 (after being stored at approximately $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ for 3 hours , and then placed at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 22 hours of storage).

The contrived influenza A and influenza B positive swabs were tested at each time point as follows:

· Contrived influenza A swab: $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ T $\mathrm { C I D } _ { 5 0 } /$ Swab $( 2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ ( $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL) virus dilution coated onto a swab]   
· Contrived influenza B swab: $4 . 2 0 \times 1 0 ^ { 1 }$ T $\mathrm { C I D } _ { 5 0 }$ /Swab ${ ( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $4 . 2 0 \times 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab]

Direct nasal swab testing results are presented in Table 8 below.

Table 8: Direct Nasal Swab Specimen Stability Study Results   

<table><tr><td rowspan=1 colspan=1>Time Point</td><td rowspan=1 colspan=2>Mock Negative Swab</td><td rowspan=1 colspan=2>Contrived Flu A Positive Swab</td><td rowspan=1 colspan=2>Contrived Flu B Positive Swab</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=1 colspan=1>Time 0</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr><tr><td rowspan=1 colspan=1>Hour 3 (approximately18-22°C</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr><tr><td rowspan=1 colspan=1>Hour 25 (22 hrs at 2-8°C)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr></table>

This analytical study data supports Alere™ i Influenza A & B product insert sample storage recommendations for direct nasal swab samples of up to 3 hours at $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ , or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 25 hours (22 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ and 3 hours at $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ ). This analytical study data also supports Alere™ i Influenza A & B product insert sample storage recommendations for nasal swab of up to 2 hours at $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ , or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 24 hours.

# Product Shelf Life Determination

Alere™ i Influenza A & B Assay Components Stability

A product stability study was conducted (and currently on-going) to generate real-time stability data to support the expiry dating assigned to Alere™ i Influenza A & B Kit.

Two lots of Alere™ i Influenza A & B were included in this stability study. All test components (Sample Receiver, Test Base, and Transfer Cartridge) were placed at room temperature $( 2 8 \small { - } 3 2 ^ { \circ } \mathrm { C } )$ , $2 { - } 8 ^ { \circ } \mathrm { C }$ and “Customer Storage Condition” (i.e., Test Base at $2 { - } 8 ^ { \circ } \mathrm { C }$ , Sample Receiver and Transfer Cartridge at $2 8 \mathrm { - } 3 2 ^ { \circ } \mathrm { C }$ ), and held at these controlled temperatures for the duration of the stability study.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated, and A1012A) were used in this study to prepare contrived positive samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents/mL) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays. Elution buffer, Alere™ i Influenza A & B Negative Control Swab (polyester swab hand coated with approximately $\mathrm { 1 x 1 0 ^ { 7 } }$ Streptococcus dysgalactiae (ATCC 12388) organisms), and presumed negative natural nasal swabs (collected from asymptomatic donors) were tested as negative samples in this study.

Contrived positive nasal swab samples were prepared using the influenza A and B strains (described above) diluted in UTM with targeted concentrations near the respective LoD levels. $5 0 \mu \mathrm { l }$ of each dilution was added to the Sample Receiver and pipetted up and down to mix. The contrived influenza A and influenza B positive samples were tested in triplicates at each specified time point for each stability condition as follows:

· Contrived influenza A #1 (tested at Time 0, Month 4, Month 7, Month 10, and Month 13):   
$3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ T $\mathrm { C I D } _ { 5 0 }$ $( 2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of   
$6 . 5 9 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { \dot { m } L }$ ( $4 . 0 0 \times \mathbf { \bar { 1 0 } } ^ { 5 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution added to the Sample   
Receiver]   
· Contrived influenza A $\# 2$ (tested at Time 0, Month 7, and Month 13): $6 . 6 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ $\mathrm { T C I D } _ { 5 0 }$ (4.00   
$\mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $1 . 3 2 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { \ : T C I D } _ { 5 0 } / \mathrm { m L }$ (8.01   
$\mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution added to the Sample Receiver]   
· Contrived influenza B #1A (tested at Time 0 and Month 4): $2 . 1 \mathrm { \bar { 1 0 } \times 1 0 ^ { 2 } \mathrm { T C I D } } _ { 5 0 }$ $( 2 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $4 . 2 0 \mathrm { x }$ $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { \bar { m L } }$ ( $5 . 0 0 { \dot { \mathrm { ~ x ~ } } } 1 0 ^ { 4 }$ genome equivalents $\mathrm { \Delta } m \mathrm { \Delta } L$ ) virus dilution added to the Sample   
Receiver] Contrived influenza B #2A (tested at Time 0): $4 . 2 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 }$ $( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $8 . 4 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution added to the Sample Receiver] Contrived influenza B #1B (tested at Month 7): $1 . 2 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ T $\mathrm { C I D } _ { 5 0 }$ $( 1 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome   
equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $2 . 3 9 \times 1 0 ^ { 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $( 3 . { \overline { { 0 0 } } } \mathrm { ~ x ~ } 1 0 ^ { 4 }$   
genome equivalents/mL) virus dilution added to the Sample Receiver] Contrived influenza $\mathrm { ~ B ~ } \# 2 \mathrm { B }$ (tested at Month 7): $2 . 3 9 \mathrm { \times 1 0 ^ { 2 } T C I D } _ { 5 0 }$ ( $3 . \mathrm { { 0 0 } \times 1 0 ^ { 3 } }$ genome

equivalents) in the Sample Receiver; [Note: $5 0 ~ \mu \mathrm { l }$ of $4 . 7 8 \times 1 0 ^ { 3 }$ TCID50/mL (6.00 x 104 genome equivalents $/ \mathrm { m L }$ ) virus dilution added to the Sample Receiver]

Contrived influenza B #1C (tested at Time 10 and Month 13): $4 . 2 0 \mathrm { ~ x ~ } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 }$ $( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 ~ \mu \mathrm { l }$ of $8 . 4 0 \mathrm { x }$ $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $1 . 0 0 { \overset { \cdot } { \mathrm { ~ x ~ } } } 1 0 ^ { 4 }$ genome equivalents/mL) virus dilution added to the Sample Receiver] Contrived influenza $\textrm { B } \# 2 \mathsf { C }$ (tested at Month 13): $8 . 4 0 \times 1 0 ^ { 1 }$ T $\mathrm { \ Y D } _ { 5 0 }$ $( 1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 ~ \mu \mathrm { l }$ of $1 . 6 8 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $2 . { \overset { \sim } { 0 } } 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution added to the Sample Receiver]

To perform testing of the Elution Buffer, no sample was added to the Sample Receiver. The Elution Buffer contained in the Sample Receiver was transferred to the Test Base using the Transfer Cartridge when prompted by the instrument according to the product instructions. For testing the Alere™ i Influenza A & B Negative Control Swabs and the presumed negative natural nasal swabs, the swabs were tested according to the product instructions.

The Elution Buffer and the Alere™ i Influenza A & B Negative Control Swab sample were tested in triplicates at Time 0, Month 3, Month 4, Month 7, Month10, and Month13, for each stability condition. A total of 20 presumed negative natural nasal swabs were tested at Time 0, Month 8, and Month13, for each stability condition.

The Alere™ i Influenza A & B assay components were considered “stable” as long as they continued to produce the expected influenza A and B positive results for the positive A & B samples and expected negative results for the Elution Buffer, the Alere™ i Influenza A & B Negative Control Swabs, and the presumed negative natural nasal swabs.

The stability study at $2 \cdot 8 ^ { \circ } \mathrm { C }$ and at the “Customer Storage Condition” (i.e., Test Base at $2 { - } 8 ^ { \circ } \mathrm { C }$ , Sample Receiver and Transfer Cartridge at $2 8 \mathrm { - } 3 2 ^ { \circ } \mathrm { C }$ ) is currently ongoing, and available real-time data supports a shelf life of up to 12 months at $2 \mathrm { - } 8 ^ { \circ } \mathrm { C }$ and at the “Customer Storage Condition”. The stability study for the entire kit at room temperature $( 2 8 - 3 2 ^ { \circ } \mathrm { C } )$ was discontinued after four months due to test results not meeting the acceptance criteria. No stability claim for the entire kit at room temperature $( 2 8 - 3 2 ^ { \circ } \mathrm { C } )$ can be supported by real-time data.

Alere™ i Influenza A & B External Positive and Negative Control Swabs Stability

The Alere™ i Influenza A & B Positive Control Swab is the same swab control currently used in the FDA cleared BinaxNOW® Influenza A & B test kit. The Alere™ i Influenza A & B Negative Control Swab is the same swab control currently used in the FDA cleared BinaxNOW® Strep A test kit. An analytical study was carried out to confirm the stability claim for the Alere™ i Influenza A & B Positive Control and Negative Control Swab, expiry dating of 2 years when stored at room temperature (approximately $1 8 \mathrm { - } 2 2 ^ { \circ } \mathrm { C }$ ), which was established based on real-time stability testing data in the context of the FDA cleared BinaxNOW® Influenza A & B test kit and the BinaxNOW® Strep A test kit.

For the Alere™ i Influenza A & B Positive Control Swab, 20 Positive Control swabs were tested 25 months postdate of manufacture (i.e., 1 month post expiry) according to the product instructions. Expected Influenza A and B dual positive results were obtained for all 20 Positive Control swabs tested.

For the Alere™ i Influenza A & B Positive Control Swab, 20 Negative Control swabs were tested 16 months postdate of manufacture according to the product instructions. Expected Influenza A and B negative results were obtained for all 20 Negative Control swabs tested.   
This part of the study is on-going and 20 Negative Control swabs will be tested again at expiry (i.e., December 2014).

# Swab Type Compatibility Study

To assess the compatibility of various swabs to be used with the Alere™ i Influenza A & B, an analytical study was conducted testing the following swabs:

Puritan PurFlock® Ultra flocked swab (standard tip), Puritan PurFlock® Ultra flocked swab (mini tip), Puritan PurFlock® Ultra flocked swab (large tip), HydraFlock® Flocked swab (standard tip), Foam swabs, Polyester swabs, Rayon swabs, Puritan Calcium Alginate swabs, and Copan Regular Flocked swabs.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated) were used in this study to prepare contrived swab samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents/mL) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive nasal swab samples for each swab type were prepared using the influenza A and B strains (described above) dilutions in UTM with targeted concentrations near the respective LoD levels (approximately 2 to $3 \mathrm { ~ X ~ }$ the respective LoD). Mock negative nasal swab samples for each swab type were prepared using UTM. Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilution or UTM onto the swab. Swab coating was done by lightly scratching the swab surface with the pipette tip as the dilution was pipetted to ensure liquid was absorbed into the swab tip. All prepared swabs were tested in triplicates according to the product instructions for testing direct swabs.

The influenza A and influenza B positive swab samples with concentration approximately 2 to $3 \mathrm { ~ X ~ }$ times the respective LoD were tested for each swab type as follows:

· Contrived influenza A swab: $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ T $\mathrm { \ Y D } _ { 5 0 } /$ Swab $( 2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL) virus dilution coated onto a swab]   
· Contrived influenza B swab: $4 . 2 0 \times 1 0 ^ { 1 }$ $\Gamma \mathrm { C I D } _ { 5 0 ^ { \prime } }$ /Swab $( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $4 . 2 0 \times 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab]

A swab type was considered to be compatible for use with the Alere™ i Influenza A & B assay if expected results for positive and negative samples were achieved for all replicates tested.

Of the nine swab types evaluated in this study, all three Puritan PurFlock® Ultra swab types (with standard, mini, or large tips) and the Calcium Alginate swabs were found to be not compatible or appropriate for use with Alere™ i Influenza A & B. The following statement is included in the “Sample Collection and Handling” section of the Product Insert: “Calcium Alginate and Puritan Purflock® Ultra flocked swabs are not suitable for use in this assay.”

The other five swab types evaluated in this study, HydraFlock® Flocked swab (standard tip), Foam swabs, Polyester swabs, Rayon swabs, and Copan Regular Flocked swabs were found to be compatible for use with Alere™ i Influenza A & B. The following statements have been added to the “Sample Collection and Handling” section of the Product Insert: “For optimal performance, use the swabs provided in the test kit. Alternatively, rayon, foam, HydraFlock® Flocked Swab (standard tip), Copan Regular Flocked Swab or polyester nasal swabs can be used to collect nasal swab samples.”

LoD Study – With Natural Clinical Matrix

An analytical study was carried out to determine the LoD for each Alere™ i Influenza A & B assay targeted influenza viruses in natural nasal swab matrix. Three strains of influenza A virus representing each of the three common currently or recently circulating influenza A subtypes (i.e., A/H1N1, A/H3N2 seasonal, and A/H1N1 pandemic (pdm)) and two strains of influenza B virus representing each of the two influenza B genetic lineages (i.e., Victoria and Yamagata) were included in this study.

Presumed negative nasal swab specimens were eluted in $3 ~ \mathrm { m L }$ of UTM. Swab elutes were combined and mixed thoroughly by gentle vortex to create a clinical matrix pool, and this clinical matrix pool was used as the diluent throughout this study. Vender provided stocks of Influenza A/Puerto Rico/8/34 (A/H1N1, Lot G1112B), A/Perth/16/2009 (A/H3N2, Lot A1121B), A/California/7/2009 (A/H1N1 pdm, Lot E1020A1), Influenza B/Malaysia/2506/2004 (Victoria lineage, Lot C1320E), and B/Bangladesh/3333/2007 (Yamagata lineage, Lot H1012A) were diluted in the natural clinical nasal swab matrix pool as described previously to generate virus dilutions for testing. The vender provided virus strains were re-titered and the concentrations (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived swab specimens were prepared by coating $1 0 ~ \mu \mathrm { l }$ of virus dilution onto each swab. Swab coating was carried out by lightly scratching the swab surface with the pipette tip as virus dilution was pipetted to ensure liquid was absorbed into the swab tip. The contrived swab samples were tested according to the product instructions for testing direct swabs.

A range finding experiment was conducted initially testing triplicates per dilution per strain. If $100 \%$ positive results were obtained with the starting dilution, a series of 2-fold dilutions beginning with the dilution that produced $100 \%$ positive results were prepared and tested in triplicates until a negative result was obtained. 20 replicates of the lowest 2-fold dilution that produced $100 \%$ (3/3) positive results were then tested for the confirmation of the estimated limit of detection (LoD). If $2 9 5 \%$ positive results were achieved, the level was reported as the confirmed LoD. If $- 9 5 \%$ positive results were obtained, 20 replicates of the next 2-fold dilution up from the dilution that produced a $- 9 5 \%$ positive results were tested.

The confirmed LoD in natural nasal swab matrix, defined as the lowest virus level that was detected $2 9 5 \%$ of the time (i.e., virus level at which at least 19 out of 20 replicates tested positive), for each influenza strain tested, were presented in Table 9 below:

Table 9: Limit of Detection (LoD) Study Results – Natural Nasal Swab Matrix   

<table><tr><td rowspan=1 colspan=1>InfluenzaStrain</td><td rowspan=1 colspan=1>Influenza ASubtype orInfluenza BGenetic Lineage</td><td rowspan=1 colspan=1>LoD(TCID50/mL)</td><td rowspan=1 colspan=1>LoD(TCID50/Swab)*</td><td rowspan=1 colspan=1>LoD (GenomeEquivalents/mL)</td><td rowspan=1 colspan=1>LoD (GenomeEquivalents/Swab)*</td></tr><tr><td rowspan=1 colspan=1>A/Puerto Rico/8/34</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>1.88 x 105</td><td rowspan=1 colspan=1>1.88 x 103</td><td rowspan=1 colspan=1>4.22 x 106</td><td rowspan=1 colspan=1>4.22 x 104</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>8.60 x 102</td><td rowspan=1 colspan=1>8.60 x 100</td><td rowspan=1 colspan=1>7.91 x 104</td><td rowspan=1 colspan=1>7.91 x 102</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>1.25 x 104</td><td rowspan=1 colspan=1>1.25 x 102</td><td rowspan=1 colspan=1>5.20 x 106</td><td rowspan=1 colspan=1>5.20 x 104</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>B Victoria lineage</td><td rowspan=1 colspan=1>1.90 x 103</td><td rowspan=1 colspan=1>1.90 x 101</td><td rowspan=1 colspan=1>7.24 x 104</td><td rowspan=1 colspan=1>7.24 x 102</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/3333/2007</td><td rowspan=1 colspan=1>B Yamagatalineage</td><td rowspan=1 colspan=1>5.55 x 102</td><td rowspan=1 colspan=1>5.55 x 100</td><td rowspan=1 colspan=1>7.36 x 104</td><td rowspan=1 colspan=1>7.36 x 102</td></tr></table>

\*Note: 10 ul of each virus dilution was coated onto a swab

An additional analytical study was also carried out to determine the LoD for each Alere™ i Influenza A & B assay targeted influenza viruses in simulated nasal swab matrix, UTM. Two strains of influenza A virus (A/H1N1) and two strains of influenza B virus were included in this study.

Vender provided stocks of Influenza A/Puerto Rico/8/34 (A/H1N1, Lot D1002B, inactivated), A/Solomon Islands/3/2006 (A/H1N1, Lot G0931A1), Influenza B/Malaysia/2506/2004 (Victoria lineage, Lot B1012C, inactivated), and B/Brisbane/60/2008 (Victoria lineage, Lot D1029C1) were diluted in UTM to generate virus dilutions for testing.

The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) for A/Puerto Rico/8/34 and B/Malaysia/2506/2004 were determined by standard virologic method prior to inactivation by the vender. For B/Brisbane/60/2008, the vender provided virus strain was re-titered and the concentration (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) was determined by standard virologic method. For A/Solomon Islands/3/2006, the vender provided concentration (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) was not confirmed by re-titering due to unavailability of residual viral stock. The concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived swab specimens were prepared by coating $1 0 ~ \mu \mathrm { l }$ of virus dilution onto each swab. Swab coating was carried out by lightly scratching the swab surface with the pipette tip as virus dilution was pipetted to ensure liquid was absorbed into the swab tip. The contrived swab samples were tested according to the product instructions for testing direct swabs.

A range finding experiment was conducted initially testing triplicates per dilution per strain. If $100 \%$ positive results were obtained with the starting dilution, a series of 2-fold dilutions beginning with the dilution that produced $100 \%$ positive results were prepared and tested in triplicates until a negative result was obtained. 20 replicates of the lowest 2-fold dilution that produced $100 \%$ (3/3) positive results were then tested for the confirmation of the estimated limit of detection (LoD). If $2 9 5 \%$ positive results were achieved, the level was reported as the confirmed LoD. If $- 9 5 \%$ positive results were obtained, 20 replicates of the next 2-fold dilution up from the dilution that produced a $- 9 5 \%$ positive results were tested.

The confirmed LoD in simulated nasal swab matrix (UTM), defined as the lowest virus level that was detected $2 9 5 \%$ of the time (i.e., virus level at which at least 19 out of 20 replicates tested positive), for each influenza strain tested, were presented in Table 10 below:

Table 10: Limit of Detection (LoD) Study Results – Simulated Nasal Swab Matrix   

<table><tr><td rowspan=1 colspan=1>Influenza Strain</td><td rowspan=1 colspan=1>Influenza ASubtype orInfluenza BGenetic Lineage</td><td rowspan=1 colspan=1>LoD(TCID50/mL)</td><td rowspan=1 colspan=1>LoD(TCID50/Swab)*</td><td rowspan=1 colspan=1>LoD (GenomeEquivalents/mL)</td><td rowspan=1 colspan=1>LoD (GenomeEquivalents/Swab)*</td></tr><tr><td rowspan=1 colspan=1>A/Puerto Rico/8/34</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>1.25 x 10</td><td rowspan=1 colspan=1>1.25 x 103</td><td rowspan=1 colspan=1>7.59 x 105</td><td rowspan=1 colspan=1>7.59 x 103</td></tr><tr><td rowspan=1 colspan=1>A/SolomonIslands/3/2006</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>2.34 x 103</td><td rowspan=1 colspan=1>2.34 x 101</td><td rowspan=1 colspan=1>1.12 x 105</td><td rowspan=1 colspan=1>1.12 x 103</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>B Victoria lineage</td><td rowspan=1 colspan=1>1.64 x 103</td><td rowspan=1 colspan=1>1.64 x 101</td><td rowspan=1 colspan=1>1.95 x 104</td><td rowspan=1 colspan=1>1.95 x 102</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>B Victoria lineage</td><td rowspan=1 colspan=1>3.98 x 102</td><td rowspan=1 colspan=1>3.98 x 100</td><td rowspan=1 colspan=1>8.08 x 104</td><td rowspan=1 colspan=1>8.08 x 102</td></tr></table>

\*Note: 10 ul of each virus dilution was coated onto a swab

# e. Analytical Reactivity:

An analytical reactivity (inclusivity) study was performed to determine whether the Alere™ i Influenza A & B assay is able to detect a variety of influenza A and B strains that represent temporal and geographic diversity.

Vender provided stocks of influenza A and B strains were diluted in UTM to generate virus dilutions for testing. The concentration (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ ) for each was determined by standard virologic method. The concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived swab specimens were prepared by coating $1 0 ~ \mu \mathrm { l }$ of virus dilution onto each swab. Swab coating was carried out by lightly scratching the swab surface with the pipette tip as virus dilution was pipetted to ensure liquid was absorbed into the swab tip. The contrived swab samples were tested according to the product instructions.

The starting dilution concentration selected for testing in this study was higher than the established LoDs in the Limit of Detection studies. Each starting dilution per strain was tested in triplicates initially. If the initial testing concentration tested positive for all three replicates, the strain was further diluted 10-fold and tested in triplicates until at least one out three replicates generated a negative result. When a negative result was obtained, additional 2-fold dilutions were tested, starting from the highest dilution that produced $100 \%$ (3/3) positive results. A concentration level was considered “reactive/positive” in this study for all but one strain tested (i.e., B/Texas/06/2011 – see footnote C under Table 11 below) if all three replicates generated a positive result for the expected influenza virus. A concentration level was considered “non-reactive/negative” for all but one strain tested (i.e., B/Texas/06/2011 – see footnote C under Table 11 below) in this study if at least one out of the three replicates generated a negative result for the expected influenza virus.

The Alere™ i Influenza A & B assay was evaluated with 17 Influenza A and nine Influenza B strains. Influenza A strains included four pre-2009 pandemic seasonal A/H1N1 strains, eight seasonal H3N2 strains, three 2009 A/H1N1 pandemic strains, one A/H3N2v strain, and one avian influenza A/H7N9 virus (detected in China in 2013). Influenza B strains included four Victoria lineage strains and five Yamagata lineage strains. The Alere™ i Influenza A & B assay detected all strains tested at the concentrations indicated in Table 11 below:

Table 11: Analytical Reactivity Study Results   

<table><tr><td rowspan=2 colspan=1>Influenza Strain</td><td rowspan=2 colspan=1>Influenza ASubtype orInfluenza BGenetic Lineage</td><td rowspan=2 colspan=5>Test Concentration (in TCID50 or Genome Equivalents ,unless indicated otherwise)</td><td rowspan=1 colspan=2>Flu AResult</td><td rowspan=3 colspan=1>Flu BResult(n=3, unlessindicateotherwise)</td></tr><tr><td rowspan=2 colspan=2>(n=3,unlessnndicateotherwise)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>TCID50/Swab*</td><td rowspan=1 colspan=1>GenomeEquivalents/mL</td><td rowspan=1 colspan=1>GenomeEquivalents/Swab*</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999 a</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>9.19 x 105</td><td rowspan=1 colspan=2>9.19 x 103</td><td rowspan=1 colspan=1>4.09 x 106</td><td rowspan=1 colspan=1>4.09 x 104</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76 a</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>3.41 x 101</td><td rowspan=1 colspan=1>3.41 x 10-1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.52 x 105</td><td rowspan=1 colspan=1>1.52 x 103</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007 a</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>2.11 x 104</td><td rowspan=1 colspan=1>2.11 x 10^2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.39 x 10</td><td rowspan=1 colspan=1>3.39 x 103</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/WSN/33 a</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>2.11 x 102</td><td rowspan=1 colspan=1>2.11 x 100</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.43 x 105</td><td rowspan=1 colspan=1>2.43 x 103</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>4.22 x 104</td><td rowspan=1 colspan=1>4.22 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.31 x 106</td><td rowspan=1 colspan=1>1.31 x 104</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>7.03 x 100</td><td rowspan=1 colspan=2>7.03 x 10-2</td><td rowspan=1 colspan=1>2.70 x 10$</td><td rowspan=1 colspan=1>2.70 x 103</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>2.08 x 10</td><td rowspan=1 colspan=1>2.08 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.47 x 106</td><td rowspan=1 colspan=1>7.47 x 104</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>3.68 x 103</td><td rowspan=1 colspan=1>3.68 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.39 x 106</td><td rowspan=1 colspan=1>3.39 x 104</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>6.81 x 104</td><td rowspan=1 colspan=2>6.81 x 102</td><td rowspan=1 colspan=1>2.57 x 106</td><td rowspan=1 colspan=1>2.57 x 104</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td></tr></table>

<table><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=2>3.16 x 102</td><td rowspan=1 colspan=2>3.16 x 100</td><td rowspan=1 colspan=1>3.37 x 105</td><td rowspan=1 colspan=1>3.37 x 10^³3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012b</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>2.50 x 100</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>2.50 x 10-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6.35 x 10^3</td><td rowspan=1 colspan=1>6.35 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.56 x 101</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1.56 x 10-1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.53 x 10</td><td rowspan=1 colspan=2>3.53 x 10^{3</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/California/4/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=2>1.47 x 104</td><td rowspan=1 colspan=1>1.47 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.07 x 106</td><td rowspan=1 colspan=2>1.07 x 104</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Maryland/04/2011</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=2>7.88 x 104</td><td rowspan=1 colspan=1>7.88 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.81 x 106</td><td rowspan=1 colspan=2>3.81 x 104</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/New York/18/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=2>1.25 x 102</td><td rowspan=1 colspan=2>1.25 x 100</td><td rowspan=1 colspan=1>9.16 x 105</td><td rowspan=1 colspan=2>9.16 x 103</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2013(Inactivated)a</td><td rowspan=1 colspan=1>A/H7N9(Detected inChina in 2013)</td><td rowspan=1 colspan=2>4.00 x 106EID50/mL</td><td rowspan=1 colspan=2>4.00 x 104EID50/Swab</td><td rowspan=1 colspan=1>1.72 x 106</td><td rowspan=1 colspan=2>1.72 x 104</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/10/2011a</td><td rowspan=1 colspan=1>A/H3N2v</td><td rowspan=1 colspan=2>2.00 x 108EID50/mL</td><td rowspan=1 colspan=1>2.00 x 106EID50/Swab</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.00 x 106</td><td rowspan=1 colspan=2>2.00 x 104</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=2>5.00 x 101CED50/mL</td><td rowspan=1 colspan=2>5.00 x 10-1CEID50/Swab</td><td rowspan=1 colspan=1>5.40 x 104</td><td rowspan=1 colspan=2>5.40 x 102</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Victoria/504/2000</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=2>1.19 x 103</td><td rowspan=1 colspan=1>1.19 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.29 x 104</td><td rowspan=1 colspan=2>8.29 x 102</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=2>1.75 x 10^³3</td><td rowspan=1 colspan=1>1.75 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.13 x 10</td><td rowspan=1 colspan=2>1.13 x 103</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Montana/05/2012</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=2>9.00 x 101</td><td rowspan=1 colspan=1>9.00 x 10-1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.55 x 104</td><td rowspan=1 colspan=2>2.55 x 102</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>8.51 x 102</td><td rowspan=1 colspan=2>8.51 x 10</td><td rowspan=1 colspan=1>1.13 x 105</td><td rowspan=1 colspan=2>1.13 x 103</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Russia/69c</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>4.44 x 101</td><td rowspan=1 colspan=2>4.44 x 10-1</td><td rowspan=1 colspan=1>2.96 x 106</td><td rowspan=1 colspan=2>2.96 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/01/2010d</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>3.68 x 104</td><td rowspan=1 colspan=2>3.68 x 102</td><td rowspan=1 colspan=1>1.16 x 106</td><td rowspan=1 colspan=2>1.16 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>6.25 x 101</td><td rowspan=1 colspan=2>6.25 x 10-1</td><td rowspan=1 colspan=1>2.28 x 105</td><td rowspan=1 colspan=2>2.28 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011d</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>2.89 x 105</td><td rowspan=1 colspan=2>6.25 x 103</td><td rowspan=1 colspan=1>2.00 x 106</td><td rowspan=1 colspan=2>2.00 x 104</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr></table>

\*Note: 10 ul of each virus dilution was coated onto a swab

a Although this test has been shown to detect A/H1N1 (pre-2009 pandemic), A/H7N9 (detected in China in 2013) and A/H3N2v viruses cultured from positive human respiratory specimens, the performance characteristics of this device with clinical specimens that are positive for the A/H1N1 (pre-2009 pandemic), A/H7N9 (detected in China in 2013) and A/H3N2v viruses have not been established.

b Influenza A/Texas/50/2012 lowest level in which 3/3 replicates were positive is approximately 0.01 to $0 . 0 6 \times$ the LoD (as comparing to the Genome Equivalents/Swab values generated in the LoD with simulated matrix study testing A/Puerto Rico/8/34 and A/Solomon Islands/3/2006, respectively). Based on in silico analysis, no polymorphism within the target sequence was reveled that can account for the observed increased reactivity. A plausible cause for the observation of increased reactivity is the impact that polymorphisms found outside of the target region can have on amplification and detection. Influenza virus mutates at a very high frequency, and as a result conservation across the entirety of the Influenza genome is very limited. The Influenza genome is RNA, and RNA readily folds into complex secondary and tertiary structures based on the primary sequence of the nucleic acid. Polymorphisms introduced into the primary structure can result in changes in secondary/tertiary structure, potentially making specific regions more or less accessible to nucleic acid binding moieties. The Alere™ i Influenza A & B test requires access to the target nucleic acid sequence in order to initiate amplification and detection, therefore it is plausible that specific changes to the genome’s primary structure (polymorphisms) may result in secondary/tertiary structural changes that could potentially increase (more accessible to nucleic acid binding moieties) or decrease (less accessible to nucleic acid binding moieties) the test’s analytical reactivity.

c Influenza B/Russia/69 lowest level in which 3/3 replicates were positive is approximately 40 to $1 5 0 \mathrm { x }$ the LoD (as comparing to the Genome Equivalents/Swab values generated in the LoD with simulated clinical matrix study testing B/Malaysia/2506/2004 and B/Brisbane/60/2008, respectively). A polymorphism within segment PA of the Influenza B genome was identified at a position which is 4 nucleotides from the $_ { 3 } \mathrm { \cdot }$ -end of template 2. This G to A polymorphism results in a $\mathrm { G } / \mathrm { C }$ (product/template) match to an $\mathbf { A } / \mathbf { C }$ (product/template) mismatch. An $\mathbf { A } / \mathbf { C }$ mismatch is determined to be moderately destabilizing, and coupled to its position only 4 nucleotides from the $_ 3 \mathrm { \cdot }$ -end of the template 2 recognition region, its impact on annealing is potentially great. The frequency of this G to A polymorphism is determined to be very low. In analyzing the strains present in the NCBI Influenza Virus Resource database from 2/2005 to 3/2014 $\scriptstyle \mathrm { N = } 9 8 6$ ), no strains contained this polymorphism, suggesting that it has not been circulating for an extended period of time.

d Influenza B/Wisconsin/01/2010 lowest level in which $3 / 3$ replicates were positive is approximately 15 to $6 0 \mathrm { x }$ the LoD, and Influenza B/Texas/06/2011 lowest level in which at least 1/3 replicates were positive is approximately 25 to $1 0 0 \mathrm { { x } }$ the LoD (as comparing to the Genome Equivalents/Swab values generated in the LoD with simulated clinical matrix study testing B/Malaysia/2506/2004 and B/Brisbane/60/2008, respectively). A single G to A polymorphism within segment PA of the Influenza B genome was identified at a position which is 5 nucleotides from the $_ { 3 } \mathrm { \cdot }$ -end of the molecular beacon annealing region in both strains. The G to A polymorphism results in a C/G match to a C/A mismatch between the molecular beacon and product 1. The C/A mismatch is determined to be moderately destabilizing that can significantly reduce assay sensitivity. An assessment of what impact this polymorphism would have on the melting temperature (Tm) of the molecular beacon/product 1 annealing was performed and the results showed a Tm drop from $6 2 . 3 ^ { \circ } \mathrm { C }$ to $5 5 . 6 \mathrm { } ^ { \circ } \mathrm { C }$ , just below the assay running temperature. This suggests that annealing would occur, but at a greatly reduced level, with a concomitant loss of assay sensitivity. The frequency of this G to A polymorphism is found at a frequency of approximately $5 \%$ within the NCBI Influenza Virus Resource database covering the time frame from 2/2005 through 3/2014.

In addition to laboratory testing, bioinformatics resources were also used to predict reactivity of additional influenza A strains of human, swine, or avian origin with the Alere™ i Influenza A & B.

Predicted reactivity of the Alere™ i Influenza A&B assay with additional influenza A strains of human, avian, and swine origin was generated using a bioinformatics approach. Assay template and probe sequences were aligned with GenBank sequences corresponding to the appropriate gene targets and reactivity were predicated based on the number and location of mismatches in the targeted region. For each subtype, multiple GenBank IDs were evaluated, corresponding to the gene segments targeted by the Alere™ i Influenza A&B assay (PB2 gene for Flu A and PA gene for Flu B). The strains listed in Table 12 below are predicted to react with the Alere™ i Influenza A&B assay as indicated:

Table 12: Predicted Alere i Influenza A&B Reactivity for Influenza A   

<table><tr><td rowspan=1 colspan=1>Origin</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Predicted Alere i Influenza A&amp;BReactivity</td></tr><tr><td rowspan=22 colspan=1>Human</td><td rowspan=1 colspan=1>H1N1pdm</td><td rowspan=1 colspan=1>A/Washington/24/2012(H1N1pdm)</td><td rowspan=1 colspan=1>EPI439208</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/Fujian-Gulou/1896/2009(H1N1)</td><td rowspan=1 colspan=1>EPI244244</td><td rowspan=1 colspan=1>Influenza A (100% sequencehmology)</td></tr><tr><td rowspan=8 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Pennsylvania/09/2011(H3N2v)</td><td rowspan=1 colspan=1>JN655532</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Pennsylvania/13/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI392732</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/123/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI397375</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/99/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI397401</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/114/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI397910</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/21/2013(H3N2v)</td><td rowspan=1 colspan=1>EPI467215</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/Ohio/13/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI381854</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/Ohio/74/2012(H3N2v)</td><td rowspan=1 colspan=1>EPI393779</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=9 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Japan/305-MA12/1957(H2N2)</td><td rowspan=1 colspan=1>CY087807</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Montevideo/2208/1967(H2N2)</td><td rowspan=1 colspan=1>CY125829</td><td rowspan=1 colspan=1>Influenza A (100% sequencehmology)</td></tr><tr><td rowspan=1 colspan=1>A/Ann Arbor/7/1967(H2N2)</td><td rowspan=1 colspan=1>CY125845</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Ann Arbor/6/60(H2N2)</td><td rowspan=1 colspan=1>AY209938</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Albany/1/65(H2N2)</td><td rowspan=1 colspan=1>AY209942</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Berkeley/1/66(H2N2)</td><td rowspan=1 colspan=1>AY209946</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/Albany/3/1958(H2N2)</td><td rowspan=1 colspan=1>CY021076</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/Ann Arbor/6/60(H2N2)</td><td rowspan=1 colspan=1>M23970</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=1 colspan=1>A/North Carolina/1/1968(H2N2)</td><td rowspan=1 colspan=1>CY031594</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology</td></tr><tr><td rowspan=3 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Cambodia/Q0321176/2006(H5N1)</td><td rowspan=1 colspan=1>HQ200422</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Shanghai/1/2006(H5N1)</td><td rowspan=1 colspan=1>CY098631</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2006(H5N1)</td><td rowspan=1 colspan=1>CY098665</td><td rowspan=1 colspan=1>Influenza A (100% sequence</td></tr></table>

<table><tr><td colspan="1" rowspan="12"></td><td colspan="1" rowspan="6">′</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">A/Guangdong/1/2008(H5N1)</td><td colspan="1" rowspan="1">CY098699</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Beijing/1/2009(H5N1)</td><td colspan="1" rowspan="1">CY098706</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/481/97(H5N1)</td><td colspan="1" rowspan="1">AF115290</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/481/97(H5N1)</td><td colspan="1" rowspan="1">AF258837</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/V0606311/2011(H5N1)</td><td colspan="1" rowspan="1">JN588929</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="2">H7N7</td><td colspan="1" rowspan="1">A/Netherlands/219/03 (H7N7)</td><td colspan="1" rowspan="1">AY342413</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/England/AV877/1996 (H7N7)</td><td colspan="1" rowspan="1">GU053116</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="4">H9N2</td><td colspan="1" rowspan="1">A/Hong Kong/1073/99 (H9N2)</td><td colspan="1" rowspan="1">AF258835</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/1073/99 (H9N2)</td><td colspan="1" rowspan="1">AJ404630</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/1074/1997 (H9N2)</td><td colspan="1" rowspan="1">GU053185</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/33982/2009 (H9N2)</td><td colspan="1" rowspan="1">KF188313</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch-moderate impact on reactivity)</td></tr><tr><td colspan="1" rowspan="22">Avian</td><td colspan="1" rowspan="2">H2N2</td><td colspan="1" rowspan="1">A/duck Hong Kong/319/1978(H2N2)</td><td colspan="1" rowspan="1">CY005545</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/mallard New York/6750/1978(H2N2)</td><td colspan="1" rowspan="1">CY130005</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch -strong impact on reactivity)</td></tr><tr><td colspan="1" rowspan="2">H5N1</td><td colspan="1" rowspan="1">A/Brown-headed Gull Qinghai/3/05(H5N1)</td><td colspan="1" rowspan="1">DQ095756</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/mallard California/2531P/2011(H5N1)</td><td colspan="1" rowspan="1">CY133956</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="9">H5N2</td><td colspan="1" rowspan="1">A/wild bird Minnesota/460613/2006(H5N2)</td><td colspan="1" rowspan="1">GQ923532</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/mallard Arkansas/473507-9/2006(H5N2)</td><td colspan="1" rowspan="1">GQ923348</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/northern pintail Utah/469648/2006(H5N2)</td><td colspan="1" rowspan="1">GQ923300</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/turkey MB/FAV11/2010(H5N2)</td><td colspan="1" rowspan="1">JX570826</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/mallard MD/865/2002(H5N2)</td><td colspan="1" rowspan="1">EU980493</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/American black-duckIllinois/080S2688/2008(H5N2)</td><td colspan="1" rowspan="1">CY079459</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/spur-winged goose Nigeria/2/2008(H5N2)</td><td colspan="1" rowspan="1">FR771844</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/tundra swanShimane/3211A001/2011(H5N2)</td><td colspan="1" rowspan="1">AB830625</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Muscovy duck New York/62095-1/2006(H5N2)</td><td colspan="1" rowspan="1">CY036041</td><td colspan="1" rowspan="1">Influenza A (sequence mismatchstrong impact on reactivity)</td></tr><tr><td colspan="1" rowspan="8">H7N2</td><td colspan="1" rowspan="1">A/Chicken New York/6777-3/97(H7N2)</td><td colspan="1" rowspan="1">EU084916</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/guinea fowl New York/23165-12/2005(H7N2)</td><td colspan="1" rowspan="1">CY031730</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/chicken NY/8030-2/1997(H7N2)</td><td colspan="1" rowspan="1">EU742937</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Guinea fowl New York/11646-3/2005(H7N2)</td><td colspan="1" rowspan="1">CY035961</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken New York/88291-14/2005(H7N2)</td><td colspan="1" rowspan="1">CY037054</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/turkey New York/88291-7/2005(H7N2)</td><td colspan="1" rowspan="1">CY095131</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken Hebei/1/2002(H7N2)</td><td colspan="1" rowspan="1">AY724251</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch minimal impact on reactivity)</td></tr><tr><td colspan="1" rowspan="1">A/goose NY/8600-3/1998(H7N2)</td><td colspan="1" rowspan="1">EU743272</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch strong impact on reactivity)</td></tr><tr><td colspan="1" rowspan="1">H7N3</td><td colspan="1" rowspan="1">A/turkey Italy/3620/2003 (H7N3)</td><td colspan="1" rowspan="1">CY021364</td><td colspan="1" rowspan="1">Influenza A (100% sequence</td></tr><tr><td colspan="2" rowspan="34">H7N7H9N2H10N7</td><td></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">homology)</td></tr><tr><td colspan="1" rowspan="1">A/turkey Italy/2685/2003 (H7N3)</td><td colspan="1" rowspan="1">CY028675</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/ruddy turnstone NJ/65/1985 (H7N3)</td><td colspan="1" rowspan="1">CY004412</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/pheasant Minnesota/917/1980 (H7N3)</td><td colspan="1" rowspan="1">CY005895</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American black duck NB/2538/2007(H7N3)</td><td colspan="1" rowspan="1">EU500849</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/ruddy turnstone Delaware Bay/121/2007(H7N3)</td><td colspan="1" rowspan="1">CY036790</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/turkey Minnesota/1138/1980 (H7N3)</td><td colspan="1" rowspan="1">CY041838</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/shorebird Delaware Bay/552/2006 (H7N3)</td><td colspan="1" rowspan="1">CY103022</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American green-winged tealMississippi/110S250/2011 (H7N3)</td><td colspan="1" rowspan="1">CY133676</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/ruddy turnstone New Jersey/175/2006(H7N3)</td><td colspan="1" rowspan="1">GU051674</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="7">H7N7</td><td colspan="1" rowspan="1">A/northern shovelerMississippi/09OS643/2009 (H7N7)</td><td colspan="1" rowspan="1">CY079419</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/ruddy turnstone Delaware/892/2006(H7N7)</td><td colspan="1" rowspan="1">GQ257395</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/wild bird Korea/A72/10 (H7N7)</td><td colspan="1" rowspan="1">JN244116</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/American black duck NewBrunswick/00344/2010 (H7N7)</td><td colspan="1" rowspan="1">CY138776</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/common teal Hong Kong/MPL634/2011(H7N7)</td><td colspan="1" rowspan="1">KF260797</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken Wenzhou/299/2013 (H7N7)</td><td colspan="1" rowspan="1">KF260930</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch strong impact on reactivity)</td></tr><tr><td colspan="1" rowspan="1">A/chicken Wenzhou/645/2013 (H7N7)</td><td colspan="1" rowspan="1">KF260919</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch strong impact on reactivity)</td></tr><tr><td colspan="1" rowspan="7">H9N2</td><td colspan="1" rowspan="1">A/avian Saudi Arabia/910135/2006 (H9N2)</td><td colspan="1" rowspan="1">GU050294</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken Guangdong/TS/2004 (H9N2)</td><td colspan="1" rowspan="1">JQ639775</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/bird Guangxi/A1/2006 (H9N2)</td><td colspan="1" rowspan="1">EU086301</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Bewick's swan Netherlands/5/2007 (H9N2)</td><td colspan="1" rowspan="1">CY041281</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/Rock Pigeon Vietnam/6/2009 (H9N2)</td><td colspan="1" rowspan="1">AB753198</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/chicken Bangladesh/5209/2009 (H9N2)</td><td colspan="1" rowspan="1">KC757995</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken China/Guangxi/14/2000 (H9N2)</td><td colspan="1" rowspan="1">DQ485213</td><td colspan="1" rowspan="1">Influenza A (sequence mismatch -moderate impact on reactivity)</td></tr><tr><td colspan="1" rowspan="1">A/mallard Ohio/99/1989 (H10N7)</td><td colspan="1" rowspan="1">CY017788</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/chicken Germany/n/1949 (H10N7)</td><td colspan="1" rowspan="1">CY014678</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/mallard Minnesota/g-00056/2007(H10N7)</td><td colspan="1" rowspan="1">CY064080</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology</td></tr><tr><td colspan="1" rowspan="1">A/American green-winged tealIllinois/08OS2713/2008(H10N7)</td><td colspan="1" rowspan="1">CY079475</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American black duck NewBrunswick/00906/2010(H10N7)</td><td colspan="1" rowspan="1">CY139392</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American black duck NewBrunswick/00906/2010(H10N7)</td><td colspan="1" rowspan="1">CY139392</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/mallard Alberta/289/2007(H10N7)</td><td colspan="1" rowspan="1">CY137721</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/mallard California/2954/2012(H10N7)</td><td colspan="1" rowspan="1">CY157237</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American green-winged tealMississippi/120S405/2012(H10N7)</td><td colspan="1" rowspan="1">CY167247</td><td colspan="1" rowspan="1">Influenza A (100% sequencehomology)</td></tr><tr><td colspan="1" rowspan="1">A/American green-winged teal</td><td colspan="1" rowspan="1">CY166193</td><td colspan="1" rowspan="1">Influenza A (100% sequence</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Wisconsin/11OS3499/2011(H10N7)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>homology)</td></tr><tr><td rowspan=1 colspan=1>A/green-winged tealCalifornia/10197/2008(H10N7)</td><td rowspan=1 colspan=1>CY093893</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>Swine</td><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/swine Fujian/1/2003(H5N1)</td><td rowspan=1 colspan=1>AY747616</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr><tr><td rowspan=1 colspan=1>Swine</td><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/swine KU/16/2001(H7N2)</td><td rowspan=1 colspan=1>CY067683</td><td rowspan=1 colspan=1>Influenza A (100% sequencehomology)</td></tr></table>

# $f .$ Analytical Specificity:

The Alere™ i Influenza A & B was performed on samples containing high levels of non-target organisms using the Alere™ i instrument, to demonstrate the specificity of the assay for detection of influenza A and B. The analytical specificity study testing included 37 bacteria, 15 viruses and 1 yeast common to the nasal cavity/nasopharynx. All bacterial stocks and the yeast stock were obtained from ATCC, with the exception of Chlamydia pneumoniae (obtained from the University of Washington). All viral stocks were obtained from ATCC, with the exception of human metapneumovirus (obtained from the Zeptometrix Corporation).

The majority of the bacteria and yeast were re-grown by Alere Scarborough, Inc. using standard microbiological culture techniques, reconstituted into saline dilution buffer and then quantified by spectrophotometer (cells/ml) versus a saline dilution buffer blank. The Mycoplasma pneumoniae received from the vendor was resuspended by Alere Scarborough, Inc. per vendor instructions and tested. Re-growth did not yield enough for harvesting and reconstitution in saline. The Chlamydia pneumonia was supplied by the University of Washington as an irradiated stock and it was, therefore, not re-grown and titered by Alere Scarborough, Inc. prior to testing. The titers of the quantified viruses indicated on certificates of analysis from vendors were not confirmed by Alere Scarborough, Inc. prior to testing.

Swab samples were prepared using the bacterial, viral and yeast strains at the stock concentration indicated in Table 13 below. $7 5 \mu \mathrm { l }$ of each organism was dispensed into each of three $1 . 5 \mathrm { m l }$ tubes. A swab was placed into each tube and stirred to absorb all the liquid into the swab head. The samples were tested in triplicate in Alere™ i Influenza A & B, according to product instructions for testing direct swabs.

Table 13: Alere™ i Influenza A & B Analytical Specificity Study Results   

<table><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Concentration (Cells/mL,Unless IndicatedOtherwise</td><td colspan="1" rowspan="1">Concentration(Cells/Reaction,Unless IndicatedOtherwise</td><td colspan="3" rowspan="1">Influenza A Result</td><td colspan="3" rowspan="1">Influenza B Result</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">2.00 x 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">4.00 x 109</td><td colspan="1" rowspan="1">1.20 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">9.00 x 108 1FU/ml</td><td colspan="1" rowspan="1">2.70 x 106 IFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheria</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">2.00 X 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">3.00 X 1010</td><td colspan="1" rowspan="1">9.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2.00 X 109</td><td colspan="1" rowspan="1">6.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">7.00 X 109</td><td colspan="1" rowspan="1">2.10 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">5.00 X 109</td><td colspan="1" rowspan="1">1.50 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">7.00 X 109</td><td colspan="1" rowspan="1">2.10 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella/Branhamella catarrhalis</td><td colspan="1" rowspan="1">2.00 X 109</td><td colspan="1" rowspan="1">6.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.00 X 109</td><td colspan="1" rowspan="1">3.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="1" rowspan="1">1.00 X 109</td><td colspan="1" rowspan="1">3.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00 X 108</td><td colspan="1" rowspan="1">3.00 x 105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.50 x 103 CFU/ml</td><td colspan="1" rowspan="1">4.50 x 10° CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">2.00 X 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">8.00 x 109</td><td colspan="1" rowspan="1">2.40 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria subflava</td><td colspan="1" rowspan="1">2.00 X 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">7.00 X 108</td><td colspan="1" rowspan="1">2.10 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00 X 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3.00 X 109</td><td colspan="1" rowspan="1">9.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">3.00 X 1010</td><td colspan="1" rowspan="1">9.00 x 107</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">5.00 x 1010</td><td colspan="1" rowspan="1">1.50 x 108</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus, Group A</td><td colspan="1" rowspan="1">3.00 x 109</td><td colspan="1" rowspan="1">9.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus, Group B</td><td colspan="1" rowspan="1">2.00 x 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus, Group C</td><td colspan="1" rowspan="1">1.00 x 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus, Group F</td><td colspan="1" rowspan="1">3.00 x 109</td><td colspan="1" rowspan="1">9.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus, Group G</td><td colspan="1" rowspan="1">2.00 x 1010</td><td colspan="1" rowspan="1">6.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.00 x 1010</td><td colspan="1" rowspan="1">3.00 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">2.00 X 109</td><td colspan="1" rowspan="1">6.00 x 106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivaris</td><td colspan="1" rowspan="1">6.00 X 109</td><td colspan="1" rowspan="1">1.80 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguis</td><td colspan="1" rowspan="1">6.00 X 109</td><td colspan="1" rowspan="1">1.80 x 107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1">Concentration (Cells/mL,Unless IndicatedOtherwise)</td><td colspan="1" rowspan="1">Concentration(Cells/Reaction,Unless IndicatedOtherwise</td><td colspan="3" rowspan="1">Influenza A Result</td><td colspan="3" rowspan="1">Influenza B Result</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">2.00 X 108</td><td colspan="1" rowspan="1">6.00 x 105</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Concentration(TCID50/mL, UnlessIndicated Otherwise</td><td colspan="1" rowspan="1">Concentration(TCID50/Reaction,Unless IndicatedOtherwise)</td><td colspan="3" rowspan="1">Influenza A Result</td><td colspan="3" rowspan="1">Influenza B Result</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1.58 X 107</td><td colspan="1" rowspan="1">4.47 X 104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7</td><td colspan="1" rowspan="1">2.81 X 106</td><td colspan="1" rowspan="1">8.43 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">8.89 x 105</td><td colspan="1" rowspan="1">2.67 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">2.81 x 104</td><td colspan="1" rowspan="1">8.43 X 101</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">2.81 x 105</td><td colspan="1" rowspan="1">8.43 X 102</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Coxsackievirus B4</td><td colspan="1" rowspan="1">2.81 x 107</td><td colspan="1" rowspan="1">8.43 X 104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus*</td><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">'</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">2.19 x 106</td><td colspan="1" rowspan="1">6.57 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">1.58 x 108CEID50/mL</td><td colspan="1" rowspan="1">4.74 X 10CEID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">2.81 x 106</td><td colspan="1" rowspan="1">8.43 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">2.81 x 107</td><td colspan="1" rowspan="1">8.43 X 104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Measles (Edmonston)</td><td colspan="1" rowspan="1">1.58 x 104</td><td colspan="1" rowspan="1">4.74 X 101</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps (Enders)</td><td colspan="1" rowspan="1">1.58 x 106</td><td colspan="1" rowspan="1">4.74 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus Type B</td><td colspan="1" rowspan="1">8.89 x 105</td><td colspan="1" rowspan="1">2.67 X 103</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">1.58 x 107</td><td colspan="1" rowspan="1">4.74 X 104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

\* Epstein Barr Virus was received from ATCC and the titer was listed as N/A on the Certificate of Analysis (CoA).

None of the bacteria, yeast, or viruses tested cross-reacted in the Alere™ i Influenza A & B at these concentrations.

# g. Potentially Interfering Substances:

An analytical study was performed to assess the potential interference effects of 25 substances naturally present in respiratory specimens or that may be artificially introduced into the nasal cavity/nasopharynx.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated), Influenza A/Solomon Islands/3/2006 (Lot D1326A), Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated), and Influenza B/Brisbane/60/2008 (Lot D1326B) were used in this study to prepare contrived positive samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents/mL) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above) diluted to a concentration approximately 2 to 3 times the respective limit of detection (LoD) levels and the various interfering substances at the concentrations specified in Table 14 below. (Note: During preparation of the B/Brisbane/60/2008 dilution a calculation error was made. This strain was inadvertently diluted to a concentration approximately 5 to 10 times the LoD level.) Each influenza strain, combined with each interfering substance, was evaluated separately. Negative samples were prepared using UTM and the various interfering substances at the concentrations specified in Table 10 below.

$5 0 \mu \mathrm { l }$ of each positive and negative sample with interfering substance was added to the Sample Receiver and pipetted up and down to mix. Each sample was tested in triplicates per the product instructions.

The influenza A and influenza B positive samples were tested as follows:

· Influenza A/Puerto $\mathrm { R i c o } / 8 / 3 4 \colon 3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 }$ $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $6 . 5 9 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { \dot { m } L }$ ( $4 . 0 0 \times \mathrm { ~ i ~ } 0 ^ { 5 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution contain an interfering substance added to the Sample Receiver]   
· Influenza A/Solomon Islands $\langle 3 / 2 0 0 6 ; 4 . 1 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 }$ b $( 2 . 8 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $8 . 1 9 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ (5.60 $\mathrm { ~ x ~ } \mathrm { 1 0 ^ { 4 } }$ genome   
equivalents $/ \mathrm { m L }$ ) virus dilution contain an interfering substance added to the Sample Receiver] · Influenza B/Malaysia $/ 2 5 0 6 / 2 0 0 4 \colon 2 . 7 0 \mathrm { ~ x ~ } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 }$ ${ \bf \zeta } 5 . 0 0 { \bf x } 1 0 ^ { 2 }$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $8 . 4 0 \times 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $( 1 . 0 0 ~ \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution contain an interfering substance added to the Sample Receiver]   
· Influenza B/Brisbane $/ 6 0 / 2 0 0 8 \colon 7 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 }$ $\left( 4 . 0 \bar { 6 } \mathrm { ~ x ~ } 1 0 ^ { 3 } \right.$ genome equivalents) in the Sample Receiver; [Note: $5 0 \mu \mathrm { l }$ of $1 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ $8 . 1 2 ~ \mathrm { ~ x ~ i 0 ^ { 4 } }$ genome equivalents/mL) virus dilution contain an interfering substance added to the Sample Receiver]

The Interfering Substances Study Results are presented in Table 14 below.

Table 14: Interfering Substances Study Results   

<table><tr><td colspan="1" rowspan="2">Substance Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="2" rowspan="1">Negative Samples</td><td colspan="4" rowspan="1">Influenza Positive Samples</td></tr><tr><td colspan="1" rowspan="1">Flu AResult</td><td colspan="1" rowspan="1">Flu BResult</td><td colspan="1" rowspan="1">Flu AResult(A/PR/8/34)</td><td colspan="1" rowspan="1">Flu A ResultA/SolomonIslands/3/2006)</td><td colspan="1" rowspan="1">Flu B Result(B/Malaysia/2506/2004</td><td colspan="1" rowspan="1">Flu B Result(B/Brisbane/6/2008)</td></tr><tr><td colspan="1" rowspan="3">Mucin</td><td colspan="1" rowspan="3">20 μg /mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Whole Blood</td><td colspan="1" rowspan="3">50 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Sinus Buster NasalSpray</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">NeoSynephrine Cold &amp;Sinus Extra StrengthSpray</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Zicam ExtremeCongestion Relief</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">4-acetamidophenol</td><td colspan="1" rowspan="3">200 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Acetylsalicylic acid</td><td colspan="1" rowspan="3">650 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Chlorpheniramine</td><td colspan="1" rowspan="3">145 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Dexamethasone</td><td colspan="1" rowspan="3">0.80 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Dextromethorphan</td><td colspan="1" rowspan="3">1 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Diphenhydramine</td><td colspan="1" rowspan="3">5 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Doxylamine Succinate</td><td colspan="1" rowspan="3">236 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Ephedrine</td><td colspan="1" rowspan="3">273 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Flunisolide</td><td colspan="1" rowspan="3">6.8 ng/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Guaiacol glycerol ether</td><td colspan="1" rowspan="3">3.5 ng/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Mupirocin</td><td colspan="1" rowspan="3">12 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Oxymetazoline</td><td colspan="1" rowspan="3">0.6 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Phenylephrine</td><td colspan="1" rowspan="3">12 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Rebetol</td><td colspan="1" rowspan="3">4.5 μg /mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="2">Relenza</td><td colspan="1" rowspan="2">282 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Rimantadine</td><td colspan="1" rowspan="3">282 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Tamiflu</td><td colspan="1" rowspan="3">1.1 μg /mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Tobramycin</td><td colspan="1" rowspan="3">2.43 mg/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Triamcinolone</td><td colspan="1" rowspan="3">40 μg /mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr></table>

None of the 25 potential interference substances tested produced false positive or false negative test results in Alere™ i Influenza A & B at the concentrations specified.

One important substance, FluMistTM was not tested in this study. The following statements have been added to the “Limitations” section of the Product Insert: “Potential interference effects from FluMistTM have not been evaluated. Individuals who have received nasally administered influenza vaccine reportedly may test positive in commercially available influenza diagnostic tests for up to three days after vaccination.”

h. Potential Microbial Interference Study:

Low Loads of Influenza A or Influenza B Co-Spiked with Selected Non-influenza Respiratory Viruses

An analytical study was performed to assess any potential interference of high concentrations of selected non-influenza respiratory viruses when co-spiked with low loads of influenza A or influenza B.

The non-influenza respiratory viral stocks used in this study are listed in Table 15 below.

Table 15: Selected Non-Influenza Respiratory Viral Stocks Tested   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>ATCC Strain</td><td rowspan=1 colspan=1>Lot Number</td><td rowspan=1 colspan=1>Stock Concentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>VR-1</td><td rowspan=1 colspan=1>59772750</td><td rowspan=1 colspan=1>1.58 x 107</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>VR-1559</td><td rowspan=1 colspan=1>57685054</td><td rowspan=1 colspan=1>1.58 x 107</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus(RSV) Type B, Strain 18537</td><td rowspan=1 colspan=1>VR-1580</td><td rowspan=1 colspan=1>58214862</td><td rowspan=1 colspan=1>8.89 x 105</td></tr></table>

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot A1012A) were used in this study to prepare contrived positive swab samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above) diluted to a concentration approximately 2 to 3 times the respective limit of detection (LoD) levels. Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilution onto the swab.

Swab coating was done by lightly scratching the swab surface with the pipette tip as the solution was pipetted to ensure liquid was absorbed into the swab tip.

The contrived influenza A and influenza B positive swabs were prepared as follows:

· Contrived influenza A swab: $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ T $\mathrm { C I D } _ { 5 0 } /$ Swab ( $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $3 . 3 0 \mathrm { \times 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$ $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL) virus dilution coated onto a swab]   
· Contrived influenza B swab: $5 . 1 9 \mathrm { ~ x ~ } 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } /$ Swab ${ ( 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $5 . 1 9 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ ( $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab]

For each of the three non-influenza respiratory viruses, $7 5 ~ \mu \mathrm { l }$ of each viral stock (described in Table 15 above) was dispensed into each of six $1 . 5 { \mathrm { - m l } }$ Eppendorf tubes. One contrived influenza A or influenza B swab (prepared as described above) was placed into each tube. Swabs were stirred to absorb all of the liquid into the swab head. A total of three mock co-infection swabs per noninfluenza virus were prepared using the influenza A or influenza B contrived positive swabs. The mock co-infection swabs were tested immediately according to the product instructions. Three contrived influenza A positive swabs and three contrived influenza B swabs without co-infection were also tested as positive experimental controls.

The Potential Microbial Interference – Non-Influenza Respiratory Viruses Study Results are presented in Table 16 below.

Table 16: Microbial Interference Study Results – Selected Non-Influenza Respiratory Viruses   

<table><tr><td rowspan=2 colspan=1>Non-InfluenzaRespiratory Virus</td><td rowspan=2 colspan=1>Non-InfluenzaRespiratory VirusConcentrationTested</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveSwab</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveSwab</td></tr><tr><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=3 colspan=1>None</td><td rowspan=3 colspan=1>None</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=3 colspan=1>1.19 x 106TCID50/Swab</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus</td><td rowspan=3 colspan=1>1.19 x 106TCID50/Swab</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=3 colspan=1>6.67 x 104TCID50/Swab</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr></table>

No competitive inhibitory effect from co-infecting non-influenza respiratory virus was observed for this study.

Low Loads of Influenza A or Influenza B Co-Spiked with Influenza Virus

An analytical study was performed to assess any potential competitive inhibitory effects of high concentrations of influenza virus when co-spiked with low loads of influenza A or influenza B.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot A1012A) were used in this study to prepare contrived positive swab samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above) diluted to a concentration approximately 2 to 3 times the respective limit of detection (LoD) levels. Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilution onto the swab. Swab coating was done by lightly scratching the swab surface with the pipette tip as the solution was pipetted to ensure liquid was absorbed into the swab tip.

The contrived influenza A and influenza B positive swabs were prepared as follows:

· Contrived influenza A swab: $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ T $\mathrm { \ C I D } _ { 5 0 }$ Swab $2 . 0 0 { \overset { - } { \mathbf { x } } } 1 0 ^ { 4 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $3 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL) virus dilution coated onto a swab]   
· Contrived influenza B swab: $5 . 1 9 \times 1 0 ^ { 1 }$ $\Gamma \mathrm { C I D } _ { 5 0 ^ { \prime } }$ /Swab ${ 5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $5 . 1 9 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab]

To prepare the co-infection swabs, $1 0 ~ \mu \mathrm { l }$ of influenza A virus dilution $( 1 . 6 5 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or 1.00 $\mathrm { ~ x ~ } 1 \dot { 0 } ^ { 7 }$ genome equivalents/mL) was added to the contrived influenza B positive swabs prepared above by lightly scratching the swab surface with the pipette tip as the solution was pipetted to ensure liquid was absorbed into the swab tip $( 1 . 6 5 \mathrm { ~ x ~ } \dot { 1 0 } ^ { 4 } \mathrm { T C I D } \dot { 5 } 0 / \mathrm { s w } \dot { \mathsf { c } }$ ab or $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ influenza A genome equivalents/Swab). Likewise, $1 0 ~ \mu \mathrm { l }$ of influenza B virus dilution $( 1 . 5 9 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\bar { 2 } . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents $ { / \mathrm { m L } }$ ) was added to the contrived influenza A positive swabs prepared above using the same method $( 1 . 5 9 \mathrm { { x } \ 1 0 ^ { 3 } \mathrm { { T C I D } _ { 5 0 } / s } }$ wab or $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ influenza B genome equivalents/Swab).

A total of three mock co-infection swabs per influenza virus were prepared using the influenza A or influenza B contrived positive swabs. The mock co-infection swabs were tested immediately according to the product instructions. Three contrived influenza A positive swabs and three contrived influenza B swabs without co-infection were also tested as positive experimental controls.

The Potential Microbial Interference – Influenza Viruses Study Results are presented in Table 17 below.

Table 17: Microbial Interference Study Results –Influenza Viruses   

<table><tr><td rowspan=2 colspan=1>Co-Infecting InfluenzaVirus</td><td rowspan=2 colspan=1>Co-Infecting InfluenzaVirus ConcentrationTested</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveSwab</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveSwab</td></tr><tr><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=3 colspan=1>None</td><td rowspan=3 colspan=1>None</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=3 colspan=1>1.00 x 105 genomeequivalents/Swab</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=3 colspan=1>2.00 x 104 genomeequivalents/Swab</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr></table>

Note: ND $=$ Not Done

No competitive inhibitory effect from co-infecting influenza virus was observed for this study.

i. Carry-Over Study:

A study was conducted to evaluate the risk of carry-over contamination with the Alere™ i Influenza A & B assay on the Alere™ i instrument.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B, inactivated) and Influenza B/Malaysia/2506/2004 (Lot B1012C, inactivated) were used in this study to prepare contrived samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents $\mathrm { \Delta } m \mathrm { L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive nasal swab samples were prepared using the influenza A and B strains (described above) dilutions in UTM with targeted concentrations of a minimum of $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Mock negative nasal swab samples were prepared using UTM. Swabs were prepared by coating $1 0 ~ \mu \mathrm { l }$ of the virus dilutions or UTM onto the swab. Swab coating was done by lightly scratching the swab surface with the pipette tip as the dilution was pipetted to ensure liquid was absorbed into the swab tip. Alere™ i Influenza A & B Negative Control swabs were tested as negative swab samples.

The contrived influenza A and influenza B positive swabs were tested as follows:

· Contrived influenza A and B positive swab: o For Flu A: $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ T $\mathrm { \bar { C I D } } _ { 5 0 } /$ Swab ( $6 . 0 7 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $6 . 0 7 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ genome equivalents/mL) virus dilution coated onto a swab] o For Flu B: $1 . 0 0 \mathrm { { x } 1 0 ^ { 4 } \ T C I D _ { 5 0 } / S w a b ( 1 . 1 9 \mathrm { { x } 1 0 ^ { 5 } } }$ genome equivalents/Swab); [Note: $1 0 ~ \mu \mathrm { l }$ of $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ ( $1 . 1 9 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ genome equivalents $/ \mathrm { m L }$ ) virus dilution coated onto a swab]

Positive and negative swab samples were tested per the product instructions. Testing between positive and negative samples alternated for a total of 15 rounds (15 positives and 15 negatives) on one Alere™ i instrument.

The Carry-over Study results are presented in Table 18 below.

Table 18: Carry-Over Study Results   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1># Tested</td><td rowspan=1 colspan=2>Influenza A and B Positive Sample</td><td rowspan=1 colspan=2>Influenza A and B Negative Sample</td></tr><tr><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=1 colspan=1>Direct Nasal Swab</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr></table>

No carry-over/contamination event was observed in this study. This data demonstrated that there is low risk of carryover/contamination when testing positive and negative samples on the Alere™ i instrument when the product insert instructions are followed.

# j. Assay cut-off:

Alere™ i Influenza A & B consists of three sets of fluorescence response curves for Flu A, Flu B, and IC. The response curves are analyzed within a decision algorithm to form a channel result. There are nine critical parameters (Table 19 below) associated with each channel in the decision algorithm. Most important among these parameters is the gradient threshold (parameter – GradThreshold). For each response curve, the gradient values within the threshold window (parameter –

ThresholdWindow) are compared to the gradient threshold (parameter – GradThreshold) and the results of this comparison are what ultimately determine the channel result, “Asserted” or “Not Asserted”. In addition, if the calculated NormAve in a NormWindow is not between NormAveMax and NormAveMin, the channel result is “Indeterminate”.

The values of the nine parameters used within the channel decision algorithm are listed in Table 19 below.

Table 19: Channel Decision Algorithm Parameter Values   

<table><tr><td rowspan=1 colspan=1>Parameter Name</td><td rowspan=1 colspan=1>Value</td></tr><tr><td rowspan=1 colspan=1>NormWindow (s)</td><td rowspan=1 colspan=1>40  110 (measurement blocks 5  12)</td></tr><tr><td rowspan=1 colspan=1>ThresholdWindow (s)</td><td rowspan=1 colspan=1>120  End of acquisition (measurement blocks 13  58)</td></tr><tr><td rowspan=1 colspan=1>GradWindow</td><td rowspan=1 colspan=1>5 consecutive data points (50s, 5 measurement blocks)</td></tr><tr><td rowspan=1 colspan=1>NormAveMin (mV)</td><td rowspan=1 colspan=1>Flu A: 1, Flu B: 1, IC: 1</td></tr><tr><td rowspan=1 colspan=1>NormAveMax (mV)</td><td rowspan=1 colspan=1>Flu A:1200, Flu B: 1200, IC: 1200</td></tr><tr><td rowspan=1 colspan=1>GradThreshold (mV/s)</td><td rowspan=1 colspan=1>Flu A: 1.5, Flu B: 1.5, IC: 1.5</td></tr><tr><td rowspan=1 colspan=1>GradPlateau (mV/s)</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>GradCount</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>AmpMAx (mV)</td><td rowspan=1 colspan=1>4000</td></tr></table>

The gradient thresholds (parameter – GradThreshold) (cut-offs) in the channel decision algorithm were derived through analysis of clinical testing data (20 Flu A positives and 105 Flu A negatives for the Flu A GradThreshold ; 9 Flu B positives and 117 Flu B negatives for the Flu B GradThreshold ) collected during the 2011-2012 respiratory season. The gradients of the true positive and true negative clinical samples (as determined by viral culture and confirmed by PCR when there are discrepancies) were analyzed.

These thresholds were subsequently verified during the prospective clinical study conducted during the 2012-2013 respiratory season.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Performance of Alere™ i Influenza A & B was evaluated against the reference method of viral culture in a prospective clinical study.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Study:

Clinical performance characteristics of the Alere™ i Influenza A & B were evaluated in a multi-site prospective study during the 2012-2013 flu season in the U.S. A total of eight investigational sites throughout the U.S. participated in the study. To be enrolled in the study, patients had to be

presenting at the participating study centers with flu-like symptoms. Direct nasal swab specimens (without elution in VTM) from patients with flu-like symptoms were collected and tested using the Alere™ i Influenza A & B at the eight study sites. Viral culture performed according to standard virology culture procedures, was utilized as the reference method for this study.

Two nasal swabs were collected from one nostril from each subject using standard collection methods. At all sites, one nasal swab was tested directly on Alere™ i Influenza A & B, according to product instructions, without prior elution in VTM. The other nasal swab was eluted in 3-mLof viral transport media (VTM). Six of the eight sites (Site 1, Site 4, Site 8, Site 10, Site 11, and Site 12) shipped nasal swab samples in VTM to a central testing laboratory for viral culture testing. This central testing laboratory was located at Site 1, which also participated as a sample collection and Alere™ i Influenza A & B testing site. The nasal swab samples in VTM from Site 2 and Site 9 were cultured on site by a local laboratory.

External control testing, using Alere™ i Influenza A & B Positive and Negative Controls, was performed prior to sample testing each day and on each Alere™ i instrument the testing was performed, at all study sites.

All specimens generating discrepant Alere™ i Influenza A & B and viral culture results were investigated by testing using an FDA cleared Influenza RT-PCR assay at a central testing laboratory located at Site 1.

Performance - Direct Nasal Swab (Without Elution in VTM)

A total of 612 nasal swab specimens were enrolled in this study. Of those, 27 direct nasal swab specimens did not meet eligibility criteria (i.e., five samples failed to meet inclusion/exclusion criteria, eight samples were not transported to the viral culture testing laboratory per the conditions required by the clinical study protocol, two samples for which site did not perform external QC controls testing before sample testing, nine samples were observed with blood, and three samples had indeterminate viral culture results). A total of 585 direct nasal swab specimens were considered evaluable. Patient age and gender distribution for the 585 specimens is presented in Table 20 below.

Table 20: Age and Gender Distribution – Direct Nasal Swab Study   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>275</td></tr></table>

Of the evaluable 585 specimens, the Alere™ i Influenza A & B generated influenza A invalid results for 14 specimens and influenza B invalid results for 16 specimens, resulting in a total of 571 specimens for influenza A performance analysis and 569 specimens for influenza B performance analysis.

Compared to the viral culture reference method, the performance of Alere™ i Influenza A & B for influenza A and influenza B are presented in Table 21 and Table 22 below, respectively.

Table 21: Alere™ i Influenza A & B Influenza A Direct Nasal Swab Performance against Viral Culture   

<table><tr><td rowspan=2 colspan=1>AlereTInfluenza A &amp; B- Flu A</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>66a</td><td rowspan=1 colspan=1>158</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>411</td><td rowspan=1 colspan=1>413</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>477</td><td rowspan=1 colspan=1>571</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:92/94      97.9% (95%CI: 92.6%-99.4%)</td></tr><tr><td rowspan=1 colspan=4>Specificity:411/477   86.2% (95%CI: 82.8%-89.0%)</td></tr></table>

a Flu A nucleic acid was detected in 58/66 False Positive specimens using an FDA-cleared molecular test b Flu A nucleic acid was not detected in 1/2 False Negative specimens using an FDA-cleared molecular test

Table 22: Alere™ i Influenza A & B Influenza B Direct Nasal Swab Performance against Viral Culture   

<table><tr><td rowspan=2 colspan=1>AlereTInfluenza A &amp; B- Flu B</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>17a</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6b</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>478</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>569</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:74/80      92.5% (95%CI: 84.6%-96.5%)</td></tr><tr><td rowspan=1 colspan=4>Specificity:472//489   96.5% (95%CI: 94.5%-97.8%)</td></tr></table>

a Flu B nucleic acid was detected in 15/17 False Positive specimens using an FDA-cleared molecular test b Flu B nucleic acid was not detected in 4/6 False Negative specimens using an FDA-cleared molecular test

Site to site performance of Alere™ i Influenza A & B for influenza A and influenza B comparing to viral culture are presented in Table 23 and Table 24 below, respectively.

Table 23: Site by Site Direct Nasal Swab Performance Obtained for Influenza A with the Alere™ i Influenza A & B in Comparison to Viral Culture   

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence </td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0.0% (0/58)</td><td rowspan=1 colspan=1>0.0% (0/0)(NA)</td><td rowspan=1 colspan=1>89.7% (52/58) c(79.2%, 95.2%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15.5% (9/58)</td><td rowspan=1 colspan=1>100% (9/9)(70.1%, 100%)</td><td rowspan=1 colspan=1>91.8% (45/49) d(80.8%, 96.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>3.2% (1/31)</td><td rowspan=1 colspan=1>100% (1/1)(20.7%, 100%)</td><td rowspan=1 colspan=1>93.3% (28/30)(78.7%, 98.2%)</td></tr><tr><td rowspan=1 colspan=1>Site 8</td><td rowspan=1 colspan=1>3.4% (2/58)</td><td rowspan=1 colspan=1>100% (2/2)(34.2%, 100%)</td><td rowspan=1 colspan=1>67.9% (38/56)*(54.8%, 78.6%)</td></tr><tr><td rowspan=1 colspan=1>Site 9</td><td rowspan=1 colspan=1>27.4% (81/296)</td><td rowspan=1 colspan=1>98.8% (80/81)(93.3%, 99.8%)</td><td rowspan=1 colspan=1>85.6% (184/215) g(80.3%, 89.7%)</td></tr><tr><td rowspan=1 colspan=1>Site 10</td><td rowspan=1 colspan=1>0.0% (0/33)</td><td rowspan=1 colspan=1>0.0% (0/0)(NA)</td><td rowspan=1 colspan=1>97.0% (32/33)(84.7%, 99.5%)</td></tr><tr><td rowspan=1 colspan=1>Site 11</td><td rowspan=1 colspan=1>0.0% (0/15)</td><td rowspan=1 colspan=1>0.0% (0/0)(NA)</td><td rowspan=1 colspan=1>93.3% (14/15)h(70.2%, 98.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 12</td><td rowspan=1 colspan=1>4.5% (1/22)</td><td rowspan=1 colspan=1>0.0% (0/1)j(NA)</td><td rowspan=1 colspan=1>85.7% (18/21)(65.4%, 95.0%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>16.5% (94/571)</td><td rowspan=1 colspan=1>97.9% (92/94)(92.6%, 99.4%)</td><td rowspan=1 colspan=1>86.2% (411/477)k(82.8%, 89.0%)</td></tr></table>

a Prevalence based on the reference method only b Confidence interval c Flu A nucleic acid was detected in $6 / 6$ False Positive specimens using an FDA-cleared molecular test d Flu A nucleic acid was detected in 2/4 False Positive specimens using an FDA-cleared molecular test e Flu A nucleic acid was detected in 2/2 False Positive specimens using an FDA-cleared molecular test f Flu A nucleic acid was detected in 16/18 False Positive specimens using an FDA-cleared molecular test g Flu A nucleic acid was detected in 28/31 False Positive specimens using an FDA-cleared molecular test h Flu A nucleic acid was detected in 1/1 False Positive specimen using an FDA-cleared molecular test i Flu A nucleic acid was detected in 3/3 False Positive specimens using an FDA-cleared molecular test j Flu A nucleic acid was not detected in this False Negative specimen using an FDA-cleared molecular test k Flu A nucleic acid was detected in 58/66 False Positive specimens using an FDA-cleared molecular test

Table 24: Site by Site Direct Nasal Swab Performance Obtained for Influenza B with the Alere™ i Influenza A & B in Comparison to Viral Culture   

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence </td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>1.7% (1/58)</td><td rowspan=1 colspan=1>100% (1/1)(20.7%, 100%)</td><td rowspan=1 colspan=1>98.2% (56/57) c(90.7%, 99.7%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15.8% (9/57)</td><td rowspan=1 colspan=1>100% (9/9)(70.1%, 100%)</td><td rowspan=1 colspan=1>95.8% (46/48) d(86.0%, 98.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>19.4% (6/31)</td><td rowspan=1 colspan=1>100% (6/6)(61.0%, 100%)</td><td rowspan=1 colspan=1>92.0% (23/25)(75.0%, 97.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 8</td><td rowspan=1 colspan=1>10.3% (6/58)</td><td rowspan=1 colspan=1>100% (6/6)(61.0%, 100%)</td><td rowspan=1 colspan=1>94.2% (49/52) (84.8%, 98.0%)</td></tr><tr><td rowspan=1 colspan=1>Site 9</td><td rowspan=1 colspan=1>15.9% (47/295)</td><td rowspan=1 colspan=1>87.2% (41/47) g(74.8%, 94.0%)</td><td rowspan=1 colspan=1>98.4% (244/248)(95.9%, 99.4%)</td></tr><tr><td rowspan=1 colspan=1>Site 10</td><td rowspan=1 colspan=1>27.3% (9/33)</td><td rowspan=1 colspan=1>100% (9/9)(70.1%, 100%)</td><td rowspan=1 colspan=1>95.8% (23/24)h(79.8%, 99.3%)</td></tr><tr><td rowspan=1 colspan=1>Site 11</td><td rowspan=1 colspan=1>13.3% (2/15)</td><td rowspan=1 colspan=1>100% (2/2)(34.2%, 100%)</td><td rowspan=1 colspan=1>92.3% (12/13)1(66.7%, 98.6%)</td></tr><tr><td rowspan=1 colspan=1>Site 12</td><td rowspan=1 colspan=1>0.0% (0/22)</td><td rowspan=1 colspan=1>0.0% (0/0)(NA)</td><td rowspan=1 colspan=1>86.4% (19/22)j(66.7%, 95.3%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>14.1% (80/569)</td><td rowspan=1 colspan=1>92.5% (74/80)(84.6%, 96.5%)</td><td rowspan=1 colspan=1>96.5% (472/489)k(94.5%, 97.8%)</td></tr></table>

a Prevalence based on the reference method only b Confidence interval c Flu B nucleic acid was detected in 1/1 False Positive specimen using an FDA-cleared molecular test d Flu B nucleic acid was detected in 2/2 False Positive specimens using an FDA-cleared molecular test e Flu B nucleic acid was detected in 3/3 False Positive specimens using an FDA-cleared molecular test ${ } ^ { \mathrm { f } } \mathrm { F l u } \mathrm { B }$ nucleic acid was detected in 4/4 False Positive specimens using an FDA-cleared molecular test g Flu B nucleic acid was not detected in 4/6 False Negative specimens using an FDA-cleared molecular test h Flu B nucleic acid was detected in 1/1 False Positive specimen using an FDA-cleared molecular test i Flu B nucleic acid was detected in 1/1 False Positive specimen using an FDA-cleared molecular test j Flu B nucleic acid was detected in 3/3 False Positive specimens using an FDA-cleared molecular test k Flu A nucleic acid was detected in 15/17 False Positive specimens using an FDA-cleared molecular test l Flu A nucleic acid was not detected in 4/6 False Negative specimens using an FDA-cleared molecular test

Performance of Alere™ i Influenza A & B for the detection of influenza A and influenza B versus culture is presented in Table 25 below stratified by patient age.

Table 25: Nasal Swab Performance Obtained for Influenza A and Influenza B with the Alere™ i Influenza A & B in Comparison to Viral Culture – Stratified by Patient Age   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>≤ 5 Years of Age(n = 332)</td><td rowspan=1 colspan=2>6 - ≤ 21 Years of Age(n = 162)</td><td rowspan=1 colspan=2>≥ 22 Years of Age(n = 77)</td></tr><tr><td rowspan=1 colspan=1>Influenza Type</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity9% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity9% CI</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>98.3%(58/59)91.0% - 99.7%</td><td rowspan=1 colspan=1>89.0%(243/273)84.7% - 92.2%</td><td rowspan=1 colspan=1>100%(31/31)89.0% - 100%</td><td rowspan=1 colspan=1>85.5%(112/131)78.5% - 90.5%</td><td rowspan=1 colspan=1>75.0%(3/4)30.1% - 95.4%</td><td rowspan=1 colspan=1>76.7%(56/73)65.8% - 84.9%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>88.9%(32/36)74.7% - 95.6%</td><td rowspan=1 colspan=1>98.0%(288/294)95.6% - 99.1%</td><td rowspan=1 colspan=1>94.4%(34/36)81.9% - 98.5%</td><td rowspan=1 colspan=1>96.8%(122/126)92.1% - 98.8%</td><td rowspan=1 colspan=1>100%(8/8)67.6% - 100%</td><td rowspan=1 colspan=1>89.9%(62/69)80.5% - 95.0%</td></tr></table>

The Alere™ i Influenza A & B detected one mixed influenza A and B infections in the prospective clinical evaluation. This sample tested positive for influenza B only by viral culture, but tested positive for influenza A only by an FDA cleared Influenza RT-PCR assay. Based on this result, the following have been added to the “Result Interpretation” section of the Product Insert: “Dual infections of Flu A and Flu B are rare. Repeat testing using new test components. Contact Technical Support during normal business hours if multiple samples provide this result.”

During the prospective clinical study, the initial invalid rate (before repeat testing per the product instructions) was $5 . 8 \%$ (34/585) ( $9 5 \%$ CI: $4 . 2 \%$ to $8 . 0 \%$ ) for Flu A, and $3 . 6 \%$ (21/585) $9 5 \%$ CI: $2 . 4 \%$ to $5 . 4 \%$ ) for Flu B. After repeat testing per the product instructions, the invalid rate was $2 . 4 \%$ (14/585) $9 5 \%$ CI: $1 . 4 \%$ , $4 . 0 \%$ ) for Flu A, and $2 . 7 \%$ (16/585) ( $9 5 \%$ CI: $1 . 7 \%$ , $4 . 4 \%$ ) for Flu B.

$b$ . Retrospective Study: None.

4. Clinical cut-off: Not applicable.

5. Expected values:

In the Alere™ i Influenza A & B prospective clinical study (described in the “Clinical Studies” section above), a total of 585 direct nasal swab specimens were determined to be evaluable. The number and percentage of influenza A and influenza B positive cases per specified age group, as determined by the Alere™ i Influenza A & B assay, are presented in Table 26 and Table 27 below:

Table 26: Influenza A Positives by the Alere™ i Influenza A & B Assay per Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofDirect Nasal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza A Positives</td><td rowspan=1 colspan=1>Influenza A Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20.1%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>28.8%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>34.3%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>29.3%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>25.4%</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22.2%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>585</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>27.0%</td></tr></table>

Table 27: Influenza B Positives by the Alere™ i Influenza A & B Assay per Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofDirect Nasal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza B Positives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9.9%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>11.9%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>25.5%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>17.1%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22.7%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>19.7%</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11.1%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>585</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>15.6%</td></tr></table>

# N. Instrument Name:

Alere™ i Instrument

# O. System Descriptions:

1. Modes of Operation:

The Alere™ i Instrument is a portable bench-top unit designed to perform in laboratory and point-of care environments. Alere™ i Influenza A & B components are color-coded to match the corresponding holder on the instrument and designed to only fit in the corresponding holder in one direction. Once properly placed into the instrument, the Alere™ i Instrument uses image analysis to automatically detect the presence of the Test Base and to check the test type and expiry date on the barcode present on the Test Base. This triggers the test sequence. Heating, mixing, and detection by fluorescence are provided by the instrument. The Alere™ i Instrument is equipped with two optical fluorescence modules, FAM™ and $\mathrm { R O X ^ { T M } }$ . Each optical module consists of a LED excitation source and photodiode receiver configured to the fluorescent channel. Fluorescence signals are optically filtered by a confocal lens and mirror arrangement and processed by software running on an on-board computer. The results are calculated automatically and presented on the instrument graphical user interface and available for export in JSON format or printing.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification: Specimen ID is entered using on screen keyboard or barcode scanner.

4. Specimen Sampling and Handling: Not applicable. The specimens are manually inserted in the Sample Receiver in the instrument.

5. Calibration:

The Alere™ i Instrument is factory calibrated and does not require any further calibration and verification at user site. However, if the instrument was transported or moved, a performance check using Alere™ i Positive and Negative Controls is recommended to ensure proper functionality by the manufacture.

# 6. Quality Control:

Quality control is addressed for each specific FDA-cleared assay to be run on the instrument (separately cleared).

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.