# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K142422   
B. Purpose for Submission: To obtain a substantial equivalence determination for a new de   
C. Measurand: tcdB gene of toxigenic Clostridium difficile   
D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) assay   
E. Applicant: Roche Molecular Systems, Inc.   
F. Proprietary and Established Names: cobas ® Cdiff Test

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130, Clostridium difficile toxin gene amplification assay 21 CFR 862.2570 - Instrumentation for clinical multiplex test systems

2. Classification: II

3. Product code: OZN, OOI

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The cobas® Cdiff Test on the cobas® 4800 system is an automated, qualitative in vitro diagnostic test, that utilizes real-time polymerase chain reaction (PCR), for the direct detection of the toxin B (tcdB) gene of toxigenic Clostridium difficile (C. difficile) in unformed (liquid or soft) stool specimens obtained from patients suspected of having $C .$ . difficile infection (CDI). The cobas® Cdiff Test is intended for use as an aid in the diagnosis of CDI in humans in conjunction with clinical and epidemiological risk factors.

# 2. Indication(s) for use:

The cobas® Cdiff Test on the cobas® 4800 system is an automated, qualitative in vitro diagnostic test, that utilizes real-time polymerase chain reaction (PCR), for the direct detection of the toxin B (tcdB) gene of toxigenic Clostridium difficile (C. difficile) in unformed (liquid or soft) stool specimens obtained from patients suspected of having $C .$ . difficile infection (CDI). The cobas® Cdiff Test is intended for use as an aid in the diagnosis of CDI in humans in conjunction with clinical and epidemiological risk factors.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: cobas® 4800 System

# I. Device Description:

The Roche Molecular Systems (RMS) cobas Cdiff Test utilizes real-time polymerase chain reaction (PCR) for the detection of toxigenic Clostridium difficile (C. difficile) DNA from unformed (liquid or soft) stool specimens to aid in the diagnosis of C. difficile infections in humans.

The cobas Cdiff Test contains two major processes: (1) automated sample preparation to extract nucleic acids from the unformed stool specimens; (2) PCR amplification of target DNA sequences using C. difficile specific primers and real-time detection of cleaved fluorescent-labeled C. difficile specific oligonucleotide detection probes. An Internal Control (IC), containing unrelated randomized DNA sequence, is added to all samples prior to automated sample preparation and is amplified and detected simultaneously with each sample to monitor the entire process.

The RMS cobas Cdiff Test is designed to be performed on the cobas 4800 System. The cobas 4800 utilizes the cobas $\tt X 4 8 0$ Instrument for automated sample preparation, and the cobas $\mathbf { z 4 8 0 }$ Analyzer for automated amplification and detection. The cobas 4800 system software integrates the sample preparation with nucleic acid amplification and detection to generate test results

Materials and reagents provided:

· cobas 4800 Cdiff Amplification/Detection Kit   
· cobas 4800 Cdiff Controls and Cofactor Kit   
· cobas 4800 System Wash Buffer Kit cobas 4800 System Lysis Kit 1 cobas 4800 System Internal Control Kit 1 cobas 4800 System Sample Preparation Kit

Additional materials required but not provided:

C CORE Tips, $1 0 0 0 \mu \mathrm { L }$ , rack of 96   
· $5 0 ~ \mathrm { m L }$ Reagent Reservoir   
· $2 0 0 ~ \mathrm { m L }$ Reagent Reservoir   
· cobas 4800 System Extraction (deep well) Plate   
$\bullet$ cobas 4800 System AD (microwell) Plate $0 . 3 ~ \mathrm { m L }$ and Sealing Film   
· Sealing foil applicator 24-position carrier   
· Solid waste bag   
· Hamilton STAR Plastic Chute   
· cobas PCR Media and Swab Sample Kit Disposable gloves, powderless Vortex Mixer (single tube) Centrifuge equipped with a swinging bucket rotor with minimum RCF of 1

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAXTM Cdiff Assay

2. Predicate 510(k) number(s): K130470

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Devicecobas Cdiff Test</td><td rowspan=1 colspan=1>PredicateBD MAX C.diff Assay(K130470)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The cobas Cdiff Test on thecobas 4800 system is anautomated, qualitative invitro diagnostic test, thatutilizes real-timepolymerase chain reaction(PCR), for the directdetection of the toxin B(tcdB) gene of toxigenicClostridium difficile (C.difficile) in unformed (liquidor soft) stool specimensobtained from patientssuspected of having C.difficile infection (CDI).The cobas Cdiff Test isintended for use as an aid inthe diagnosis of CDI inhumans in conjunction withclinical and epidemiologicalrisk factors.</td><td rowspan=1 colspan=1>The BD MAX C. diffAssay performed on theBD MAX System is anautomated in vitrodiagnostic test for thedirect, qualitative detectionof the Clostridium difficiletoxin B gene (tcdB) inhuman liquid or soft stoolspecimens from patientssuspected of having C.difficile infection (CDI).The test, performed directlyon the specimen, utilizesreal-time polymerase chainreaction (PCR) for theamplification of C. difficiletoxin B gene DNA andfluorogenic target-specifichybridization probes for thedetection of the amplifiedDNA. The BD MAX C.diff Assay is intended toaid in the diagnosis of CDI.</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=1>C. difficile toxin B gene(tcdB)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>unformed (liquid or soft)stool specimens</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Amplification technology</td><td rowspan=1 colspan=1>Real-time PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection Technology</td><td rowspan=1 colspan=1>TaqMan® probes</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample extraction</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Devicecobas Cdiff Test</td><td colspan="1" rowspan="1">PredicateBD MAX C.diff Assay(K130470)</td></tr><tr><td colspan="1" rowspan="1">Instrument platform</td><td colspan="1" rowspan="1">cobas 4800 System</td><td colspan="1" rowspan="1">BD MAX System</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample processing control(IC)Positive and negative</td><td colspan="1" rowspan="1">Sample processing control(IC)Positive and negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">control</td><td colspan="1" rowspan="1">control (optional)</td></tr><tr><td colspan="1" rowspan="1">Sample transfer device</td><td colspan="1" rowspan="1">Polyester swab</td><td colspan="1" rowspan="1">10μL inoculating loop</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

Sample preparation for the cobas Cdiff Test is automated with the use of the cobas $\tt X 4 8 0$ instrument. Unformed stool specimens are transferred into cobas PCR Media by using a swab. Organisms are lysed with chaotropic agent, proteinase K, and SDS reagents. Released nucleic acids, along with added Internal Control DNA, are bound by magnetic glass particles. They are washed and then eluted into a small volume of buffer. The instrument then takes an aliquot of the eluted material and sets up the PCR reaction with an activated Master Mix.

The PCR cycling steps and detection of target signal occurs in the cobas $\mathbf { z 4 8 0 }$ Analyzer. The Master Mix reagent contains primer pairs and probes for two targets: $C .$ . difficile toxin B gene and Internal Control. If the target nucleic acid sequences are present, amplification with the corresponding primers will occur by a thermostable DNA polymerase, generating PCR products (amplicons). These products are detected by specific TaqMan probes containing a fluorescent dye and a quencher. If the PCR product is present, the probe hybridizes to the product and gets cleaved by the $5 '$ to $3 ^ { \circ }$ nuclease activity of the polymerase. This reaction allows the fluorescence to be emitted from the dye, and the signal is recorded in real time during each PCR cycle by the cobas $\mathbf { z 4 8 0 }$ analyzer. The signal is interpreted by the cobas 4800 System Software and reported as final results.

The cobas 4800 System uses the cobas $\tt X 4 8 0$ Instrument for sample preparation, and the cobas $\mathbf { z 4 8 0 }$ Analyzer for amplification and detection. Both the cobas $\tt X 4 8 0$ Instrument and the cobas $\mathbf { z 4 8 0 }$ Analyzer are controlled by a computer workstation running the cobas 4800 System Software.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Precision

An in-house precision study was conducted using a panel composed of C. difficile cultures diluted into negative stool matrix in cobas PCR Media to concentration levels below Limit of Detection (LOD), near LOD and above LOD of the cobas Cdiff Test. A negative level composed of only the stool suspension in cobas PCR Media was also tested. The study used three unique lots of cobas Cdiff Test reagents and three instruments for a total of 36 runs over 12 days. A description of the precision panels and the study summary is shown in Table 1. Analysis of the variance components (Table 2) suggested that most variability of target $\mathrm { C t }$ values was attributed to within run and lot to lot factors $6 0 . 0 \%$ and $2 5 . 3 \%$ , respectively) for concentrations at or around LOD. For concentration levels above LOD, most of the $\mathrm { C t }$ value variability was attributed to within run and instrument to instrument factors $7 2 . 5 \%$ and $2 4 . 7 \%$ , respectively). Overall percent coefficients of variation (CV) for panel members at LOD and above LOD were $1 . 5 \%$ and $1 . 1 \%$ , respectively (Table 3).

Table 1. Precision study positive rate analysis   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=2 colspan=1>NTested</td><td rowspan=2 colspan=1>NPositive</td><td rowspan=2 colspan=1>Positive Rate</td><td rowspan=1 colspan=2>95% CL</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>&lt; 1 x LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>29.2%</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>41.1%</td></tr><tr><td rowspan=1 colspan=1>~ 1 x LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>~ 3 x LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

$\mathrm { L O D = }$ Limit of Detection

Table 2. Variance components analysis for precision panel members   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=5>Variance of Components/Percent Contribution to Total</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Day</td><td rowspan=1 colspan=1>Within Run</td></tr><tr><td rowspan=2 colspan=1>~ 1 x LOD</td><td rowspan=2 colspan=1>38.5</td><td rowspan=1 colspan=1>0.0789</td><td rowspan=1 colspan=1>0.0189</td><td rowspan=1 colspan=1>0.0001</td><td rowspan=1 colspan=1>0.0270</td><td rowspan=1 colspan=1>0.1875</td><td rowspan=1 colspan=1>0.3123</td></tr><tr><td rowspan=1 colspan=1>25.3%</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>~ 3 x LOD</td><td rowspan=2 colspan=1>37.5</td><td rowspan=1 colspan=1>0.0047</td><td rowspan=1 colspan=1>0.0404</td><td rowspan=1 colspan=1>0.0000</td><td rowspan=1 colspan=1>0.0000</td><td rowspan=1 colspan=1>0.1188</td><td rowspan=1 colspan=1>0.1638</td></tr><tr><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>24.7%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>72.5%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

$\mathrm { L O D = }$ Limit of Detection

Table 3. Standard deviations and percent CV for precision panel members   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=5>SD of Components/Percent CV</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Day</td><td rowspan=1 colspan=1>Within Run</td></tr><tr><td rowspan=2 colspan=1>~ 1 x LOD</td><td rowspan=2 colspan=1>38.5</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.56</td></tr><tr><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=2 colspan=1>∼ 3 x LOD</td><td rowspan=2 colspan=1>37.5</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>1.1%</td></tr></table>

$\mathrm { L O D = }$ Limit of Detection

In-house precision study results were acceptable.

# Reproducibility

The reproducibility of the cobas Cdiff Test on the cobas 4800 System was established in a multi-site investigation using simulated clinical samples evaluated across lot, site/instrument, operator, day and within-run.

Reproducibility test panels of four specimens, with three replicates each, were prepared at varying concentrations of $C .$ . difficile strain ATCC 43255 (Negative, Below LOD, $1 \mathrm { ~ x ~ L O D }$ , and $3 \mathrm { ~ x ~ L O D }$ ) into pooled, C. difficile-negative, unformed stool matrix in cobas PCR Media and tested at three sites by two operators/day for five days/lot over two lots for an overall total of 720 tests (4 specimens x 3 replicates $_ { \textrm { X 3 } }$ sites x 2 operators/site x 5 days/lot x 2 lots) or 180 tests/panel member or 90 tests/panel member/lot.

Overall, 60 runs were performed; all external controls and all runs were valid. Of the 720 test performed across all panel members, there were 712 $( 9 8 . 9 \% )$ valid results; Seven failed results were due to clot detection or pipetting errors, and one invalid result was due to an invalid internal control result. Only valid test results were included in percent agreement analyses.

Table 4 summarizes the $\mathrm { C t }$ values and the overall percent agreement (two-sided $9 5 \%$ exact CI) by site and panel member. The SD and CV $( \% )$ for $\mathrm { C t }$ values across positive panel members ranged from 0.64 to 0.71 and 1.7 to $1 . 9 \%$ , respectively. The positive percent agreements for the $C .$ . difficile positive panel members Below LOD, $1 \mathrm { ~ x ~ L O D }$ , and $3 \mathrm { ~ x ~ L O D }$ were $6 6 . 1 \%$ $9 5 \%$ CI: $5 8 . 7 \%$ to $7 3 . 0 \%$ ), $1 0 0 . 0 \%$ ( $9 5 \%$ CI: $9 8 . 0 \%$ to $1 0 0 . 0 \% )$ , and $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 7 . 9 \%$ to $1 0 0 . 0 \%$ ), respectively. The negative percent agreement for negative panel member was $1 0 0 . 0 \%$ ( $9 5 \%$ CI: $9 7 . 9 \%$ to $1 0 0 . 0 \%$ ).

Table 4. Summary of reproducibility results: $\mathrm { C t }$ values and percent agreement by site and panel member   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>ValidTestsResults(n)</td><td rowspan=1 colspan=3>Ct</td><td rowspan=1 colspan=3>Percent Agreement by Site(n/N)</td><td rowspan=1 colspan=2>Total Agreement</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Percent(n/N)</td><td rowspan=1 colspan=1>(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(54/54)</td><td rowspan=1 colspan=1>100.0%(174/174)</td><td rowspan=1 colspan=1>(97.9%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>BelowLOD</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>71.7(43/60)</td><td rowspan=1 colspan=1>68.3(41/60)</td><td rowspan=1 colspan=1>58.3(35/60)</td><td rowspan=1 colspan=1>66.1%(119/180)</td><td rowspan=1 colspan=1>(58.7%, 73.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0%(180/180)</td><td rowspan=1 colspan=1>(98.0%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>3 x LOD</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(60/60)</td><td rowspan=1 colspan=1>100.0(58/58)</td><td rowspan=1 colspan=1>100.0%(178/178)</td><td rowspan=1 colspan=1>(97.9%, 100.0%)</td></tr></table>

Table 5. Summary of reproducibility results: percent agreement by variable and panel member   

<table><tr><td rowspan=1 colspan=2>cYanmaGYellowb</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Below LOD</td><td rowspan=1 colspan=1>1 x LOD</td><td rowspan=1 colspan=1>3 x LOD</td></tr><tr><td rowspan=2 colspan=1>Lot</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0% (85/85)</td><td rowspan=1 colspan=1>65.6% (59/90)</td><td rowspan=1 colspan=1>100.0% (90/90)</td><td rowspan=1 colspan=1>100.0% (90/90)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0% (89/89)</td><td rowspan=1 colspan=1>66.7% (60/90)</td><td rowspan=1 colspan=1>100.0% (90/90)</td><td rowspan=1 colspan=1>100.0% (88/88)</td></tr><tr><td rowspan=3 colspan=1>Site/Instrument</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0% (60/60)</td><td rowspan=1 colspan=1>71.7% (43/60)</td><td rowspan=1 colspan=1>100.0% (60/60)</td><td rowspan=1 colspan=1>100.0% (60/60)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0% (60/60)</td><td rowspan=1 colspan=1>68.3% (41/60)</td><td rowspan=1 colspan=1>100.0% (60/60)</td><td rowspan=1 colspan=1>100.0% (60/60)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0% (54/54)</td><td rowspan=1 colspan=1>58.3% (35/60)</td><td rowspan=1 colspan=1>100.0% (60/60)</td><td rowspan=1 colspan=1>100.0% (58/58)</td></tr><tr><td rowspan=6 colspan=1>Operator</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>76.7% (23/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>66.7% (20/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>66.7% (20/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>70.0% (21/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100.0% (24/24)</td><td rowspan=1 colspan=1>53.3% (16/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (29/29)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>63.3% (19/30)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>100.0% (29/29)</td></tr><tr><td rowspan=5 colspan=1>Day</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0% (35/35)</td><td rowspan=1 colspan=1>69.4% (25/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td><td rowspan=1 colspan=1>100.0% (35/35)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0% (35/35)</td><td rowspan=1 colspan=1>61.1% (22/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0% (34/34)</td><td rowspan=1 colspan=1>58.3% (21/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100.0% (35/35)</td><td rowspan=1 colspan=1>63.9% (23/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td><td rowspan=1 colspan=1>100.0% (35/35)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100.0% (35/35)</td><td rowspan=1 colspan=1>77.8% (28/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td><td rowspan=1 colspan=1>100.0% (36/36)</td></tr></table>

\* For the negative panel member, percent agreement $=$ (number of negative results/total valid results) $\textbf { \em X } 1 0 0$ ; for the positive panel members, percent agreement $=$ (number of positive results/total valid results) $\textbf { \em X } 1 0 0$ . $\mathrm { C I } =$ confidence interval; $\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; $\mathrm { L O D = }$ limit of detection; $\mathrm { { N / A } = }$ not applicable; $\mathrm { S D } =$ standard deviation.

Table 6. Overall Mean, SD, and $\mathrm { C V } ( \% )$ for valid results from positive panel members   

<table><tr><td rowspan=1 colspan=3>cYanmaGllow</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Site/Inst.</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Below LOD</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>1 x LOD</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>3 x LOD</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.9%</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; LOD $=$ limit of detection; $\mathrm { S D } =$ standard deviation.

Reproducibility study results were acceptable.

b. Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Specimen stability

Unformed stool specimen stability was demonstrated by testing ten C. difficile positive and two negative clinical specimens on the cobas 4800 system after consecutive storage at $3 0 \pm 1 ^ { \circ } \mathrm { C }$ for two days, followed by $2 { - } 8 ^ { \circ } \mathrm { C }$ for five days, followed by ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for 60 days. The results supported the storage of unformed stool specimens at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ for two days, or $2 { - } 8 ^ { \circ } \mathrm { C }$ for seven days, and at ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ for 60 days.

The stability of stool specimens mixed with the cobas PCR media was demonstrated by testing the specimen panel mixed in the cobas PCR media after storage at $3 0 \pm 1 ^ { \circ } \mathrm { C }$ for seven days and at $2 { - } 8 ^ { \circ } \mathrm { C }$ for one and two months. The results supported the storage of stool specimen mixed with cobas PCR media at $3 0 \mathrm { { } ^ { \circ } C }$ for seven days or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 60 days.

# Controls

The cobas Cdiff Test uses Positive $( + )$ and Negative (-) Controls (cobas 4800 Cdiff Controls and Cofactor Kit), and an Internal Control (cobas 4800 System Internal Control Kit 1). One set of Positive and Negative Controls are included in each run. For any run, valid results must be obtained for both the Positive and Negative Control for the cobas 4800 Software to display the reportable cobas Cdiff Test results from that run.

Positive control. The Cdiff $( + )$ Control contains non-infectious DNA plasmids of $C .$ . difficile. The Cdiff $( + )$ Control monitors nucleic acid extraction, amplification, and detection steps in a given run of the test. The Cdiff $( + )$ Control result must be ‘Valid’.

Negative control. The (-) Control result must be ‘Valid’.

Internal control. The Internal Control is a recombinant bacteriophage lambda that contains randomized sequences and targets for internal control-specific primers and probe. The Internal Control is added to all specimens and the Positive and Negative Controls during sample preparation on the cobas $\times 4 8 0$ instrument. The Internal Control monitors nucleic acid extraction, amplification, and detection steps for a given specimen. The Internal Control is also required for validation of the run controls.

# d. Detection limit:

The analytical sensitivity (Limit of Detection or LOD) for the cobas Cdiff Test was determined by analyzing quantified cultures of seven toxigenic C. difficile strains diluted into pooled negative stool specimen matrix in cobas PCR Media. All levels were analyzed using cobas Cdiff Test across three unique reagent lots. At least 21 replicates per reagent lot were tested at each level. The LOD of the test was defined as defined as the target concentration which was detected as positive i $12 \%$ of the replicates tested, based on results generated by the worst performing reagent lot. The seven C. difficile strains tested are shown in Table 7.

Table 7. cobas Cdiff Test Limit of Detection   

<table><tr><td rowspan=2 colspan=1>Strain ID</td><td rowspan=2 colspan=1>Toxinotype</td><td rowspan=2 colspan=1>REA*Type</td><td rowspan=2 colspan=1>PFG†Type</td><td rowspan=2 colspan=1>Ribotype</td><td rowspan=2 colspan=1>Phenotype</td><td rowspan=1 colspan=2>LOD (CFU/swab)</td></tr><tr><td rowspan=1 colspan=1>ByPositiveRate</td><td rowspan=1 colspan=1>By Probitnalysis(95% CI)</td></tr><tr><td rowspan=1 colspan=1>ATCC 43255 (VPI 10463)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>087</td><td rowspan=1 colspan=1>A+B+CDT-</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>90 (66, 311)</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1382 (630)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>R 23</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>012</td><td rowspan=1 colspan=1>A+B+CDT-</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>83 (62, 145)</td></tr><tr><td rowspan=1 colspan=1>CDC 204118</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>BI 8</td><td rowspan=1 colspan=1>NAP1</td><td rowspan=1 colspan=1>027</td><td rowspan=1 colspan=1>A+B+CDT+</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>42 (30, 129)</td></tr><tr><td rowspan=1 colspan=1>R12087 (CD196)</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>BI</td><td rowspan=1 colspan=1>NAP1</td><td rowspan=1 colspan=1>027</td><td rowspan=1 colspan=1>A+B+CDT+</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54 (39, 126)</td></tr><tr><td rowspan=1 colspan=1>2748-06</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>078</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>45 (33, 113)</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598 (1470)</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>017</td><td rowspan=1 colspan=1>A-B+</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>130 (96, 228)</td></tr><tr><td rowspan=1 colspan=1>F15</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>59 (43, 117)</td></tr></table>

\* Restriction endonuclease analysis; † Pulse Field Gel

# Inclusivity/reactivity

The sensitivity of the cobas Cdiff Test was determined for 28 additional strains of toxigenic C. difficile tested at a minimum of three concentrations and 40 replicates per concentration level. The lowest levels that had at least a $9 5 \%$ hit rate are summarized in Table 8. All 28 strains were detected as positive $\mathrm { i n } \geq 9 5 \%$ of replicates at concentrations ranging from 77.9 to 460 CFU/swab.

Table 8. Inclusivity/reactivity   

<table><tr><td colspan="1" rowspan="1">C. difficileStrain</td><td colspan="1" rowspan="1">Toxinotype</td><td colspan="1" rowspan="1">Ribotype</td><td colspan="1" rowspan="1">Conc. (CFU/swab)</td><td colspan="1" rowspan="1">Positive Rate</td></tr><tr><td colspan="1" rowspan="1">EX 623</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">95.0%</td></tr><tr><td colspan="1" rowspan="1">AC 008</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">95.0%</td></tr><tr><td colspan="1" rowspan="1">SE 844</td><td colspan="1" rowspan="1">IIIa</td><td colspan="1" rowspan="1">080</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">55767</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">023</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">SE 881</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">045</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">51377</td><td colspan="1" rowspan="1">VI</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">57267</td><td colspan="1" rowspan="1">VII</td><td colspan="1" rowspan="1">063</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">51680</td><td colspan="1" rowspan="1">IX</td><td colspan="1" rowspan="1">019</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">8864</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">036</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">R 9367</td><td colspan="1" rowspan="1">XIII</td><td colspan="1" rowspan="1">070</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">R 10870</td><td colspan="1" rowspan="1">XIV</td><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">R 9385</td><td colspan="1" rowspan="1">XV</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">SUC36</td><td colspan="1" rowspan="1">XVI</td><td colspan="1" rowspan="1">078</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">J9965</td><td colspan="1" rowspan="1">XVII</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">460</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">K095</td><td colspan="1" rowspan="1">XVIII</td><td colspan="1" rowspan="1">014</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">95.0%</td></tr><tr><td colspan="1" rowspan="1">TR13</td><td colspan="1" rowspan="1">XIX</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">TR14</td><td colspan="1" rowspan="1">XX</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">CH6223</td><td colspan="1" rowspan="1">XXI</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">CD07-468</td><td colspan="1" rowspan="1">XXII</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">8785</td><td colspan="1" rowspan="1">XXIII</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">95.0%</td></tr><tr><td colspan="1" rowspan="1">597B</td><td colspan="1" rowspan="1">XXIV</td><td colspan="1" rowspan="1">131</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">7325</td><td colspan="1" rowspan="1">XXV</td><td colspan="1" rowspan="1">027</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">7459</td><td colspan="1" rowspan="1">XXVI</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">95.0%</td></tr><tr><td colspan="1" rowspan="1">KK2443-2006</td><td colspan="1" rowspan="1">XXVII</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">CD08-070</td><td colspan="1" rowspan="1">XXVIII</td><td colspan="1" rowspan="1">126</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">CD07-140</td><td colspan="1" rowspan="1">XXIX</td><td colspan="1" rowspan="1">056</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">ES 130</td><td colspan="1" rowspan="1">XXX</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">234</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">WA 151</td><td colspan="1" rowspan="1">XXXI</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">460</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

# e. Analytical specificity:

Cross reactivity:

Cross reactivity was evaluated with a panel that consisted of 103 bacteria, fungi, and viruses that may be found in stool specimens and one human cell line (Table 9). In addition, 28 Clostridium genus organisms including non-toxigenic C. difficile were tested (Table 10).

Bacteria were quantified as colony forming units $\mathrm { ( C F U ) / m L }$ , human cells were quantified as cells $\mathrm { \Delta m L }$ , and viruses were quantified as plaque forming units $( \mathrm { P F U } ) / \mathrm { m L }$ , except for the following microorganisms: Chlamydia trachomatis was quantified as elementary bodies $\mathrm { ( E B ) / m L }$ , and Cytomegalovirus, Human Echovirus, and Human Enterovirus were quantified as $\mathrm { I U / m L }$ .

All bacteria and human cells were spiked to $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { U n i t s / m L }$ , and all viruses were spiked to $1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ Units/mL, except for Adenovirus Type 40, Cytomegalovirus (HHV5), and Human Rotavirus, which were spiked to lower concentrations due to stock concentration limitations. Testing was performed with the organisms alone or with two C. difficile isolates present individually at $3 \mathrm { ~ x ~ }$ Limit of Detection (LOD) of the cobas Cdiff Test. Results indicated that none of these organisms interfered with detection of the intended $C .$ . difficile targets. None produced false positive results when there was no intended $C .$ . difficile target present.

Clostridium botulinum cross reactivity was evaluated in silico using BLAST and Fuzznuc programs against the GenBank nucleotide sequence database to mimic PCR amplicon generation and probe detection steps. In silico analysis indicated that cross reactivity with $C .$ . botulinum is highly unlikely.

Table 9. Microorganisms and human cells tested   

<table><tr><td rowspan=1 colspan=1>Abiotrophia defectiva</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Acinetobacter lwoffii</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Alcaligenes faecalis subsp. Faecalis</td><td rowspan=1 colspan=1>Anaerococcus tetradius</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Bacteroides caccae</td></tr><tr><td rowspan=1 colspan=1>Bacteroides merdae</td><td rowspan=1 colspan=1>Bacteroides stercoris</td><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium longum</td><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida catenulata</td><td rowspan=1 colspan=1>Cedecea davisae</td></tr><tr><td rowspan=1 colspan=1>Chlamydia Trachomatis Serovar L2</td><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>Citrobacter freundii</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>Collinsella aerofaciens</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Desulfovibrio piger</td><td rowspan=1 colspan=1>Edwardsiella tarda</td></tr><tr><td rowspan=1 colspan=1>Eggerthella lenta</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Enterobacter cloacae</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>Enterococcus cecorum</td><td rowspan=1 colspan=1>Enterococcus dispar</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Enterococcus gallinarum</td></tr><tr><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>Escherichia coli</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>Escherichia hermannii</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Gemella morbillorum</td></tr><tr><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>HCT-15 Human Cells</td><td rowspan=1 colspan=1>Helicobacter fennelliae</td></tr><tr><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>Lactococcus lactis</td></tr><tr><td rowspan=1 colspan=1>Leminorella grimontii</td><td rowspan=1 colspan=1>Listeria grayi</td><td rowspan=1 colspan=1>Listeria innocua</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Mitsuokella multacida</td><td rowspan=1 colspan=1>Mobiluncus curtisii</td></tr><tr><td rowspan=1 colspan=1>Moellerella wisconsensis</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Porphyromonas asaccharolytica</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Proteus penneri</td><td rowspan=1 colspan=1>Providencia alcalifaciens</td><td rowspan=1 colspan=1>Providencia rettgeri</td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Ruminococcus bromii</td><td rowspan=1 colspan=1>Salmonella choleraesuis subsp.choleraesuis</td><td rowspan=1 colspan=1>Salmonella enterica subsp. arizonae (f.k.a.Salmonella choleraesuis ssp. arizonae)</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. entericaserovar Choleraesuis</td><td rowspan=1 colspan=1>Serratia liquefaciens</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Streptococcus uberis</td><td rowspan=1 colspan=1>Trabulsiella guamensis</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Yersinia bercovieri</td></tr><tr><td rowspan=1 colspan=1>Yersinia rohdei</td><td rowspan=1 colspan=1>Cytomegalovirus (HHV5)*</td><td rowspan=1 colspan=1>Human Adenovirus 40*</td></tr><tr><td rowspan=1 colspan=1>Human Coxsackievirus A 10</td><td rowspan=1 colspan=1>Human Echovirus 11</td><td rowspan=1 colspan=1>Human Enterovirus 71</td></tr><tr><td rowspan=1 colspan=1>Human Rotavirus*</td><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>-</td></tr></table>

\* Cytomegalovirus (HHV5) at $2 . 0 \times 1 0 ^ { 3 } \mathrm { I U / m L }$ , Human Adenovirus Type 40 was spiked at $2 . 2 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { \ P F U / m L } ,$ and Human Rotavirus at $9 . 8 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ PFU/mL for testing.

Table 10. Clostridium genus organisms tested, including non-toxigenic C. difficile   

<table><tr><td rowspan=1 colspan=1>Clostridium beijerinckii</td><td rowspan=1 colspan=1>Clostridium bifermentans</td><td rowspan=1 colspan=1>Clostridium bolteae</td></tr><tr><td rowspan=1 colspan=1>Clostridium botulinum*</td><td rowspan=1 colspan=1>Clostridium butyricum</td><td rowspan=1 colspan=1>Clostridium chauvoei</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficileSerogroup B (non-toxigenic)</td><td rowspan=1 colspan=1>Clostridium difficileSerogroup I (non-toxigenic)</td><td rowspan=1 colspan=1>Clostridium fallax</td></tr><tr><td rowspan=1 colspan=1>Clostridium haemolyticum</td><td rowspan=1 colspan=1>Clostridium histolyticum</td><td rowspan=1 colspan=1>Clostridium innocuum</td></tr><tr><td rowspan=1 colspan=1>Clostridium methylpentosum</td><td rowspan=1 colspan=1>Clostridium nexile</td><td rowspan=1 colspan=1>Clostridium novyi</td></tr><tr><td rowspan=1 colspan=1>Clostridium orbiscindens (re-named Flavonifractor plautii)</td><td rowspan=1 colspan=1>Clostridium paraputrificum</td><td rowspan=1 colspan=1>Clostridium perfringens</td></tr><tr><td rowspan=1 colspan=1>Clostridium ramosum</td><td rowspan=1 colspan=1>Clostridium scindens</td><td rowspan=1 colspan=1>Clostridium septicum</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellii</td><td rowspan=1 colspan=1>Clostridium sphenoides</td><td rowspan=1 colspan=1>Clostridium spiroforme</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>Clostridium symbiosum</td><td rowspan=1 colspan=1>Clostridium tertium</td></tr><tr><td rowspan=1 colspan=1>Clostridium tetani</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

\* Based on in silico analysis

# Interference

Twenty-six commonly used medications, as well as fecal fat, whole blood, and mucin, were tested for potential interference effects with the cobas Cdiff Test. The amount of potentially interfering substance is expressed as the concentration in the primary stool specimen. The effects of the potentially interfering substances were evaluated in the presence and absence of two toxigenic C. difficile isolates spiked to approximately three times the LOD of the cobas Cdiff Test as determined for those strains in the LoD study (54 and 113 CFU/mL). No interference was observed for the exogenous substances tested. For fecal fat, no interference was observed up to $2 8 \%$ , for whole blood, no interference was observed up to $50 \%$ , and for mucin, no interference was observed up to $2 5 \%$ . Mucin at $50 \%$ interfered with the detection of the toxigenic C. difficile isolates. These results are summarized in Table 11.

Table 11. Potentially interfering substances tested   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Results</td></tr><tr><td colspan="1" rowspan="1">Fecal Fat</td><td colspan="1" rowspan="1">4 ~ 28 % (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">25, 50 % (v/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">25, 50* % (w/v)</td><td colspan="1" rowspan="1">No interference upto 25% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Tums</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Imodium AD®</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Stool Softener</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol® (Procter &amp; Gamble)</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Nystatin Ointment USP</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Preparation H® with Bio-Dyne®Cream (Wyeth)</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">GYNOL II</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Vagisil® Anti-itch cream</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Anusol® Plus</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Sunscreen</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Monistat® 7</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">VaselineT</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">SAB-Dimenhydrinate® Suppositories (SABEX®)</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Equate Natural Vegetable Laxative</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Dulcolax®</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Fleet® (CB Fleet Company)</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">K-Y Jelly/Gelée®(McNeil-PPC)</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Afrin Original Nasal Spray</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Witch hazel</td><td colspan="1" rowspan="1">Liquid from 1wipe/swab</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">E-Z-HDTM High Density Barium Sulfate forsuspension (E-Z-EM Canada)</td><td colspan="1" rowspan="1">20% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Aleve</td><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">No interference</td></tr></table>

\*Mucin at $50 \%$ $\mathbf { \widetilde { V } } / \mathbf { \widetilde { V } } )$ concentration interfered with the detection of toxigenic $C .$ difficile isolates.

# Cross Contamination and Carryover Contamination:

The cross contamination rate was assessed by testing a series of high positive toxigenic C. difficile samples and negative samples in a checkerboard configuration on the cobas 4800 system. High positive samples were prepared by spiking pooled negative stool matrix with C. difficile culture to generate a Ct of the ${ 9 } { 5 } ^ { \mathrm { t h } }$ percentile of the clinical specimen population. A total of 94 samples were included per run.

A total of nine runs were performed using three cobas 4800 systems with alternating checkerboard configurations to assess the cross contamination rate. A total of three runs of negative samples following checkerboard runs were performed to assess the carry-over contamination rate. In the nine checkerboard runs, one out of 423 negative samples produced a positive result for a cross contamination rate of $0 . 2 4 \%$ . None of the 282 negative samples from the carry-over contamination runs produced a positive result.

f. Assay cut-off:

Data from pre-clinical testing of stool specimens as well as limit of detection, inclusivity, and within-laboratory repeatability studies were compiled and analyzed to verify the preliminary cut-off value. None of the negative results exhibited a Ct value before the cut-off. The cut-off was validated in the prospective clinical study which demonstrated an adequate separation between the cut-off and the highest Ct value observed for a positive specimen.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the cobas Cdiff Test was established in an IRB-approved, prospective, multi-site investigation comparing the results with direct and enriched toxigenic culture using leftover, de-identified, unformed stool samples from subjects suspected of having CDI. Specimens were collected at five geographically diverse sites across the US from symptomatic eligible male and female subjects. The toxigenic culture was performed at a single reference laboratory and the cobas Cdiff Test was performed at one of three designated sites. A specimen was considered positive for toxigenic C. difficile if C. difficile was recovered from stool either by direct or enriched toxigenic culture and if the recovered isolates tested positive by cytotoxicity testing. If toxigenic C. difficile was isolated from the direct culture, the enrichment culture was not further processed. Specimens were classified as negative for toxigenic C. difficile only if they tested negative by both direct, and repeat direct and enrichment culture. The sensitivity, specificity, and PPV and NPV values were calculated by comparing cobas Cdiff Test results with the combined results of direct and enrichment toxigenic culture. In addition, the positive percent agreement (PPA) and negative percent agreement (NPA) was determined comparing the cobas Cdiff Test with the initial direct culture results. Discrepant analysis was performed using an FDA-cleared nucleic acid amplification test (NAAT) on all samples with discordant results between the cobas Cdiff Test and toxigenic culture.

Specimens were collected from 683 evaluable subjects: 306 males $( 4 4 . 8 \% )$ and 377 females $( 5 5 . 2 \% )$ with a mean age of 56 years (range 3 to 99). Specimens from all 683 subjects had valid results for both direct toxigenic culture but one sample lacked

sufficient volume for repeat direct and enrichment culture and was not included in the statistical analysis. Of the 683 specimens, 113 were positive for toxigenic $C .$ . difficile during the initial direct toxigenic culture and 141 of 682 were positive for toxigenic C. difficile using the combined results of direct and enrichment toxigenic culture, for a prevalence rate of $2 0 . 7 \%$ for the study.

In a total of 49 cobas Cdiff Test runs in the clinical study, all external positive and negative controls were valid. Two failed runs $( 4 . 1 \% )$ were observed in the study due to one hardware error and one operational incident. Out of 683 specimens, the initial cobas Cdiff Test invalid rate was $0 . 3 \%$ (2/683) and the final invalid rate following one retest as per the labelled instructions for use was $0 \%$ (0/683). The initial cobas Cdiff Test failed rate was $0 . 7 \%$ (5/683) and the final failed rate following one retest was $0 \%$ (0/683).

# Comparison with combined direct and enrichment culture

The clinical performance of the cobas Cdiff Test compared with the combined direct culture and enrichment culture results is shown in Table 12. The sensitivity and specificity of the cobas Cdiff Test were $9 2 . 9 \%$ (131/141; $9 5 \%$ CI: $8 7 . 4 \%$ to $9 6 . 1 \%$ ) and $9 8 . 7 \%$ (534/541; $9 5 \%$ CI: $9 7 . 4 \%$ to $9 9 . 4 \%$ ), respectively; and the PPV and NPV were $9 4 . 9 \%$ $9 5 \%$ CI: $8 9 . 9 \%$ to $9 7 . 5 \%$ ) and $9 8 . 2 \%$ $9 5 \%$ CI: $9 6 . 6 \%$ to $9 9 . 0 \%$ ), respectively.

Table 12. Comparison of cobas Cdiff Test with combined direct culture and enrichment culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Combined Direct and Enrichment Culturea</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>cobas CdiffTest</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>7c</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10b</td><td rowspan=1 colspan=1>534</td><td rowspan=1 colspan=1>544</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>541</td><td rowspan=1 colspan=1>682</td></tr><tr><td rowspan=1 colspan=5>Sensitivity:  92.9% (131/141; 95% CI = 87.4% to 96.1%)Specificity: 98.7% (534/541; 95% CI = 97.4% to 99.4%)PPV: 94.9% (95% CI = 89.9% to 97.5%)NPV: 98.2% (95% CI = 96.6% to 99.0%)</td></tr></table>

a Includes combined results from an initial direct culture and a repeat direct and enrichment culture performed on all initial direct culture-negative samples. One specimen with an initial direct culture-negative result had insufficient specimen volume to perform repeat direct culture and enrichment culture and was excluded from the analysis. Thirty-six (36) specimens with initial direct culture-negative results had their combined direct and enrichment culture results based on repeat culture that used three culture plate media (CCFA, CCFA-HB, CCFA-VA) in combination with enrichment culture. Of these 36 specimens, 21 were culture positive.

b Of the 10 specimens with false-negative cobas Cdiff Test results relative to combined direct and enrichment culture, all 10 were negative by a second NAAT c Of the seven specimens with false-positive cobas Cdiff Test results relative to combined direct and enrichment culture, three were positive and four were negative by that second NAAT method.

Comparison with direct culture

The performance of the cobas Cdiff Test compared to initial direct culture is shown in Table 13. The PPA and NPA of the cobas Cdiff Test compared to the initial direct culture for all 683 subjects were $9 7 . 3 \%$ (110/113) and $9 4 . 9 \%$ (541/570), respectively. Of the three specimens with false-negative cobas Cdiff Test results relative to direct culture, all three were negative by a second NAAT method.

Table 13. Comparison of cobas Cdiff Test with direct culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Direct Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>cobas CdiffTest</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>29b</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>541</td><td rowspan=1 colspan=1>544</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>570</td><td rowspan=1 colspan=1>683</td></tr><tr><td rowspan=1 colspan=5>Positive Percent Agreement:  97.3% (110/113; 95% CI = 92.5% to 99.1%)Negative Percent Agreement:  94.9% (541/570; 95% CI = 92.8% to 96.4%)</td></tr></table>

a Of the three specimens with false negative cobas Cdiff Test results relative to direct culture, all three were negative by a second NAAT method. b Of the 29 specimens with false positive cobas Cdiff Test results relative to direct culture, 15 were positive, 13 were negative by a second NAAT method, and one sample was not further tested because of insufficient specimen volume.

b. Clinical specificity: See section M3a   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the prospective clinical study, specimens were collected from 683 subjects suspected of CDI, with 306 males $( 4 4 . 8 \% )$ and 377 females $( 5 5 . 2 \% )$ and a mean age of 56 years (range 3 to 99). The percentage of positive results observed with the cobas Cdiff Test in this population was $2 0 . 4 \%$ (139/683).

# N. Instrument Name:

cobas 4800 System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X___ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No ___X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

3. Specimen Identification: Specimens are identified using barcodes

4. Specimen Sampling and Handling:

Specimens are placed on the cobas $\times 4 8 0$ instrument as open tubes and specimen processing is fully automated. After completion of specimen processing, the user transfers the plate carrier to the cobas $\boldsymbol { z } 4 8 0$ instrument for automated amplification and detection.

5. Calibration:

No calibration is required by the user. Roche technicians perform calibration periodically as required.

6. Quality Control:

Positive and Negative Controls are included in every run. An Internal control is introduced for each Control and Specimen during sample preparation for testing on the cobas $\times 4 8 0$ instrument.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.