# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K142033

# B. Purpose for Submission

The purpose of the submission is for clearance of an additional two analytes, Norovirus GI/GII and Rotavirus A, to the already cleared Verigene Enteric Pathogens Nucleic Acid Test (EP) (k140083) which is run on the Verigene System.

# C. Measurand

The Verigene Enteric Pathogens Nucleic Acid Test is performed on the automated Nanosphere Verigene System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific gastrointestinal microbial nucleic acid gene sequences for the following organisms:

· Campylobacter Group (comprised of C. coli, C. jejuni, and C. lari)   
· Shigella species (including S. dysenteriae, S. boydii, S. sonnei, and S. flexneri)   
· Vibrio Group (comprised of V. cholerae and V. parahaemolyticus) Yersinia enterocolitica   
· Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers   
· Norovirus GI/GII   
· Rotavirus A

# D. Type of Test:

The Verigene Enteric Pathogens Nucleic Acid Test is a qualitative molecular assay which relies on detection of specific nucleic acid targets in a microarray format. For each of the bacterial nucleic acid sequences detected by Enteric Pathogens Nucleic Acid Test, unique Capture and Mediator oligonucleotides are utilized, with gold nanoparticle probe-based endpoint detection. The Capture oligonucleotides are covalently bound to the microarray substrate and hybridize to a specific portion of the nucleic acid targets. The Mediator oligonucleotides have a region which bind to a different portion of the same nucleic acid targets and also have a sequence which allows binding of a gold nanoparticle probe. Catalytic enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency and provide accurate detection of target capture.

# E. Applicant:

Nanosphere, Inc.

# F. Proprietary and Established Names:

Verigene® Enteric Pathogens Nucleic Acid Test (EP)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 – Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: Class II

3. Product code: PCH, PCI, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended Use:

The Verigene Enteric Pathogens Nucleic Acid Test (EP) is a multiplexed, qualitative test for simultaneous detection and identification of common pathogenic enteric bacteria, viruses and genetic virulence markers from liquid or soft stool preserved in Cary-Blair medium, collected from individuals with signs and symptoms of gastrointestinal infection. The test is performed on the automated Nanosphere Verigene System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific gastrointestinal microbial nucleic acid gene sequences associated with the following pathogenic bacteria and viruses:

· Campylobacter Group (comprised of C. coli, C. jejuni, and C. lari)   
· Salmonella species   
· Shigella species (including S. dysenteriae, S. boydii, S. sonnei, and S. flexneri)   
· Vibrio Group (comprised of V. cholerae and V. parahaemolyticus)   
· Yersinia enterocolitica   
· Norovirus GI/GII   
· Rotavirus A

In addition, EP detects the Shiga toxin 1 gene and Shiga toxin 2 gene virulence markers. Shiga toxin producing E. coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2.

EP is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness, in conjunction with other clinical, laboratory, and epidemiological information; however, is not to be used to monitor these infections. EP also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

Due to the limited number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Yersinia enterocolitica, Vibrio Group and Shigella species were primarily established with contrived specimens.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

EP results should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative EP results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For Prescription Use Only

4. Special instrument requirements: The assay is run on the Verigene System.

# I. Device Description:

The Enteric Pathogens Nucleic Acid Test is performed on the Verigene System which is a fully automated, bench-top molecular diagnostics workstation. The System enables automated nucleic acid extraction from unformed stool specimens (liquid or soft) preserved in Cary-Blair media and detection of bacterial and viral-specific target DNA. The Verigene System consists of two components: the Verigene Reader and the Verigene Processor $S P$ .

The Reader is the Verigene System’s user interface, which serves as the central control unit for all aspects of test processing, imaging, and result generation using a touch-screen control panel and a barcode scanner. The Verigene Processor $S P$ executes the test procedure,

automating the steps of sample preparation, target amplification hybridization to the detection microarray. Detection and identification of bacterial and viral-specific DNA is performed in a microarray format by using gold nanoparticle probe-based technology. Once the specimen is loaded by the operator, all other fluid transfer steps are performed by an automated pipette that transfers reagents between wells of the trays and finally loads the specimen into the Test Cartridge for hybridization. Single-use disposable test consumables and a self-contained Verigene Test Cartridge are utilized for each sample tested with the Enteric Pathogens Nucleic Acid Test assay.

To obtain the test results after test processing is complete, the user removes the Test Cartridge from the Processor $S P$ , and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader and intensities from the microarray spots are used to make a determination regarding the presence (Detected) or absence (Not Detected) of a bacterial or viral nucleic acid sequence/analyte. This determination is made by means of software-based decision algorithm resident in the Reader.

# J. Substantial Equivalence Information:

1. Predicate device name(s): xTAG Gastrointestinal Pathogen Panel (GPP) by Luminex Molecular Diagnostics, Inc.

2. Predicate 510(k) number(s): K121894/K121454

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:Enteric Pathogens Nucleic Acid Test(EP)</td><td rowspan=1 colspan=1>Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The Verigene Enteric Pathogens NucleicAcid Test (EP) is a multiplexed,qualitative test for simultaneousdetection and identification of commonpathogenic enteric bacteria, viruses andgenetic virulence markers from liquid orsoft stool preserved in Cary-Blairmedium, collected from individuals withsigns and symptoms of gastrointestinalinfection. The test is performed on theautomated Nanosphere Verigene Systemutilizing reverse transcription (RT),polymerase chain reaction (PCR), andarray hybridization to detect specificgastrointestinal microbial nucleic acidgene sequences associated with thefollowing pathogenic bacteria and</td><td rowspan=1 colspan=1>The xTAG® GastrointestinalPathogen Panel (GPP) is a multiplexednucleic acid test intended for thesimultaneous qualitative detection andidentification of multiple viral,parasitic, and bacterial nucleic acids inhuman stool specimens fromindividuals with signs and symptomsof infectious colitis or gastroenteritis.The following pathogen types,subtypes and toxin genes are identifiedusing the xTAG® GPP:Campylobacter (C. jejuni, C.coli and C. lari only)Clostridium difficile (C.</td></tr></table>

<table><tr><td></td><td>(EP)</td><td>Panel (GPP)</td><td>K121894</td></tr><tr><td rowspan="11">Due to the limited number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Yersinia enterocolitica, Vibrio Group and Shigella species were primarily established with contrived specimens. Concomitant culture is necessary for organism recovery and further typing of</td><td>viruses: Campylobacter Group •</td><td></td><td>difficile) toxin A/B</td></tr><tr><td>and C. lari)</td><td>(comprised of C. coli, C. jejuni,</td><td>Cryptosporidium (C. parvum and C. hominis only)</td></tr><tr><td>Salmonella species</td><td></td><td>Escherichia coli (E. coli) 0157</td></tr><tr><td>Shigella species (including S. dysenteriae, S. boydii, S. sonnei, and S. flexneri)</td><td></td><td>Enterotoxigenic Escherichia coli (ETEC) LT/ST</td></tr><tr><td>Vibrio Group (comprised of V. cholerae and V. parahaemolyticus)</td><td></td><td>Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis)</td></tr><tr><td>Yersinia enterocolitica</td><td></td><td>Norovirus GI/GII</td></tr><tr><td>Norovirus GI/GII</td><td></td><td>Rotavirus A</td></tr><tr><td>Rotavirus A</td><td>•</td><td>Salmonella</td></tr><tr><td>In addition, EP detects the Shiga toxin 1 gene and Shiga toxin 2 gene virulence</td><td></td><td>Shiga-like Toxin producing E.</td></tr><tr><td>markers. Shiga toxin producing E. coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2.</td><td></td><td>coli (STEC) stx 1/stx 2 Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae)</td></tr><tr><td>EP is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness, in conjunction with other clinical, laboratory, and epidemiological information; however, is not to be used to monitor these infections. EP also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.</td><td>The detection and identification of specific gastrointestinal microbial nucleic acid from individuals exhibiting signs and symptoms of gastrointestinal infection aids in the evaluation, laboratory findings and epidemiological information. A gastrointestinal microorganism</td><td>diagnosis of gastrointestinal infection when used in conjunction with clinical</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">New Device:Enteric Pathogens Nucleic Acid Test(EP)</td><td colspan="1" rowspan="1">Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">basis for diagnosis, treatment, or otherpatient management decisions.Confirmed positive results do not rule outco-infection with other organisms that arenot detected by this test, and may not bethe sole or definitive cause of patientillness. Negative EP results in the settingof clinical illness compatible withgastroenteritis may be due to infection bypathogens that are not detected by thistest or non-infectious causes such asulcerative colitis, irritable bowelsyndrome, or Crohn's disease.</td><td colspan="1" rowspan="1">treatment, or other patientmanagement decisions. Confirmedpositive results do not rule outcoinfection with other organisms thatare not detected by this test, and maynot be the sole or definitive cause ofpatient illness. Negative xTAGGastrointestinal Pathogen Panelresults in the setting of clinical illnesscompatible with gastroenteritis may bedue to infection by pathogens that arenot detected by this test or non-infectious causes such as ulcerativecolitis, irritable bowel syndrome, orCrohn's disease. xTAG GPP is notintended to monitor or guide treatmentfor C. difficile infections.The xTAG GPP is indicated for usewith the Luminex MAGPIXinstrument.</td></tr><tr><td colspan="1" rowspan="1">DNAAmplification</td><td colspan="1" rowspan="1">PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Organisms/NATargets Detected</td><td colspan="1" rowspan="1">Campylobacter Group(C. coli, C. jejuni, and C. lari)Salmonella speciesShigella species(S. dysenteriae, S. boydii, S. sonnei, and S.flexneri)Yersinia enterocoliticaNorovirus GI/GIIRotavirus AShiga toxin 1 gene and Shiga toxin 2 genevirulence markers</td><td colspan="1" rowspan="1">Same with additional analytes(excluding Vibrio Group and Yersiniaenterocolitica)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">New Device:Enteric Pathogens Nucleic Acid Test(EP)</td><td colspan="1" rowspan="1">Predicate:xTAG® Gastrointestinal PathogenPanel (GPP)K121894</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">~ 2 hours</td><td colspan="1" rowspan="1">5 hours</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Stool in Cary Blair Media</td><td colspan="1" rowspan="1">Unpreserved Stool</td></tr><tr><td colspan="1" rowspan="1">Sample prep</td><td colspan="1" rowspan="1">On-board, automated NA extraction andamplification</td><td colspan="1" rowspan="1">Off-line NA Extraction andamplification</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">Gold/Silver nanoparticle probe detectionof bacterial-specific DNA oncomplementary oligo- microarray</td><td colspan="1" rowspan="1">Specific microbial target or controlbead populations coupled to sequencesfrom Universal Array, streptavidin, R-phycoerythrin conjugate</td></tr><tr><td colspan="1" rowspan="1">OpticalDetection</td><td colspan="1" rowspan="1">Light scatter</td><td colspan="1" rowspan="1">Multi-color fluorescence</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline - Second Edition

CLSI EP12-A2; User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition

CLSI MM3-A2; Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline - Second Edition

CLSI EP15-A2 - User Verification of Performance for Precision and Trueness; Approved Guideline – second edition

CLSI EP9-A2-IR - Method Comparison and Bias Estimation Using Patient Samples; Approved Guideline – second edition (Interim Revision)

CLSI EP17-A - Protocols for Determination of Limits of Detection and Limits of Quantitation; Approved Guideline

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices

Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems

Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium Difficile – Draft Guidance for Industry and FDA Staff

In Vitro Diagnostic (IVD) Device Studies – Frequently Asked Questions

# L. Test Principle:

The Enteric Pathogens Nucleic Acid Test is performed on samples of stool preserved in Cary-Blair media, collected from individuals suspected of having acquired an enteric infection. The provided flocked swab is dipped into the specimen until flocked tip is fully immersed. Once evenly coated, the swab is transferred to the Stool Prep Buffer tube and broken at the pre-formed scored breakpoint. The Stool Prep Buffer tube is sealed with a screw cap and thoroughly vortexed and centrifuged to ensure a proper mixture of the specimen. A $2 0 0 \mu \mathrm { L }$ aliquot of the prepared Stool Prep Buffer (SLB) is pipetted into the Specimen Well of the Extraction Tray and the Drawer Assembly of the Verigene SP is closed to initiate the test. An automated pipettor performs all further fluidic transfers during the Sample Extraction and Amplification steps.

The steps automated by the Verigene System include:

Sample Preparation—Cell lysis and magnetic bead-based bacterial and viral NA isolation from liquid or soft stool samples preserved in Cary-Blair media   
Target Amplification—Multiplex PCR-based amplification to generate specific amplicons   
Hybridization and Detection—Hybridization of analyte-specific NA to capture oligonucleotides on a microarray, using gold nanoparticle probe-based technology to aid detection.

For each of the bacterial or viral nucleic acid sequences/analytes detected by the EP test, two sets of oligonucleotides are required for the endpoint gold nanoparticle probe-based process: (i) Capture oligonucleotides (or captures) and (ii) Mediator oligonucleotides (or mediators). The Capture oligonucleotides are printed on the Test Substrate (or the microarray) and are designed to specifically bind to one part of the analyte-specific target oligonucleotide (or target). The Mediator oligonucleotides bind to a complementary portion of the same target which is now hybridized to the capture and enable binding of gold nanoparticle probes. Silver enhancement of the bound gold nanoparticle probes at the capture sites results in goldsilver aggregates that scatter light with high efficiency. Light scatter from the capture spots is imaged by the Verigene Reader and intensities from the microarray spots are processed by a decision algorithm to make calls regarding the presence (“Detected”) or absence (“Not Detected”) of a nucleic acid sequence/analyte.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

Refer to K140083 for Precision/Reproducibility Study results for previously cleared EP

bacterial analytes.

Preparation of Samples for Precision/Reproducibility Studies (and other analytical studies):

For preparation of samples for analytical studies, organisms were spiked into a Negative Stool Matrix (NSM) pool. The NSM was prepared by pooling individual de-identified residual stool specimens preserved in Cary-Blair medium which were determined to be negative by standard culture methods as well as by the EP assay. These individual “CaryBlair specimens” were originally prepared by diluting unpreserved stool specimens (obtained from individuals in the intended use population) in Cary-Blair medium such that the stool comprises $2 5 \%$ by volume of the resulting specimen. NSM was formulated by pooling the negative Cary-Blair specimens, combining with Enteric Stool Prep Buffer in a 1:5 ratio, and clarifying by centrifugation and collection of the supernatant. The NSM pool was again tested with the Enteric Pathogens Nucleic Acid Test in replicates of 20 and confirmed as negative.

Analytical samples were prepared using Rotavirus strains obtained from the American Type and Culture Collection (ATCC) and Norovirus strains obtained from the Center for Disease Control and Prevention (CDC). Viral strains were characterized and quantified and then diluted and added to the NSM to achieve appropriate organism concentrations. These simulated specimens were either used immediately (within $3 0 \mathrm { { m i n } }$ of preparation) or stored frozen at ${ < } . 7 0 ^ { \circ } \mathrm { C }$ until use.

# Precision Study

A Precision Study for the Norovirus and Rotavirus targets was conducted as part of a full 20-sample EP study (bacteria and viruses) previously performed for K140083. The panel was composed of simulated specimens prepared with three characterized and quantified viral strains as presented in the table below. In addition, two negative samples were also run (Negative Stool Matrix and Clostridium difficile). For each EP viral strain the panel included, a “Low Positive” sample at $1 - 2 \mathbf { x }$ the Limit of Detection (LoD), which would be expected to produce a positive result approximately $9 5 \%$ of the time, and a “Moderate Positive” sample at $2 { \cdot } 5 \mathrm { x }$ LoD, which would be expected to yield a positive result approximately $9 9 \%$ of the time. Panel members were tested daily in duplicate by two operators for four non-consecutive days for a total of 16 tests per sample (2 operators/day x 2 replicates/operator $\mathrm { ~ x ~ 4 ~ d a y s = 1 6 ~ }$ tests per sample). Positive and negative controls were run once per combination of consumable lots received.

Precision and Reproducibility Test Panel Composition   

<table><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Strain ID.</td><td rowspan=1 colspan=1>LOD*</td><td rowspan=1 colspan=1>Label</td><td rowspan=1 colspan=1>Concentration*</td><td rowspan=1 colspan=1>ExpectedResult</td></tr><tr><td rowspan=2 colspan=1>Rotavirus</td><td rowspan=2 colspan=1>VR-2550</td><td rowspan=2 colspan=1>1.11x103</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>5.55x103</td><td rowspan=1 colspan=1>Rotavirus</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2.22x10^{</td><td rowspan=1 colspan=1>Rotavirus</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI</td><td rowspan=2 colspan=1>CDC 2142</td><td rowspan=2 colspan=1>4.12x10</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>2.06x106</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1.03x106</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=2 colspan=1>Norovirus GII</td><td rowspan=2 colspan=1>D17219</td><td rowspan=2 colspan=1>1.67x106</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>8.35x106</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>3.34x106</td><td rowspan=1 colspan=1>Norovirus</td></tr></table>

The Precision Study yielded the expected results for all 96 samples containing the targeted viral analytes. With the inclusion of the Norovirus and Rotavirus targets to the EP test, the results of the positive bacterial as well as negative samples tested in the full Precision Study performed for K140083 were assessed with the viral target results unmasked. Across all of the 224 bacterial and negative control samples tested, there were no cases of false positive results for Norovirus or Rotavirus and therefore the Precision study results presented in K140083 are unchanged with the addition of the viral targets.

Two samples returned initial No Calls, giving an initial call rate for viral samples of $9 7 . 9 \%$ (94/96). In addition, three samples returned pre-analysis (pre-AE) errors for a rate of $3 \%$ (3/101 tests). All initial no-calls and pre-AE errors gave the expected results upon repeat testing.

Study results as shown in the following table demonstrated acceptable withinlaboratory precision for the EP test.

<table><tr><td rowspan=1 colspan=3>Summary of Precision Study Results</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Final Percent Agreementwith Expected Result(95 % CI)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>16/16100%(79.4-100)</td></tr><tr><td rowspan=1 colspan=1>Negative StoolMatrix</td><td rowspan=2 colspan=1>Negative forTargetedAnalytes</td><td rowspan=1 colspan=1>100%a16/16(79%-100%)</td></tr><tr><td rowspan=1 colspan=1>Clostridiumdifficile (NegativeControl</td><td rowspan=1 colspan=1>100%a16/16(79%-100%)</td></tr></table>

a) Performance is for Negative results

The inter-laboratory reproducibility of the EP test for Norovirus and Rotavirus was evaluated in a study performed for the initial clearance of the EP test (K140083) at three external clinical sites. The viral panel members for the Reproducibility study were the same as described above for the internal Precision Study. Samples were prepared in NSM with each strain at two concentrations; low positive and moderate positive. In addition two negative samples were evaluated (NSM and Clostridium difficile). The study included five days of testing incorporating the following sources of potential variance: sites, operators, days, consumable lots and runs. Samples were tested daily in triplicate by two operators for five non-consecutive days at three sites for a total of 90 tests per sample (3 sites $_ { \textrm { X 2 } }$ operators / site $_ { \textrm { X 3 } }$ replicates / operator x 5 days $= 9 0$ tests per sample). Positive and negative controls were run at each site according to the study protocol. Samples which yielded an initial ‘No Call’ or Pre-Analysis Error test result with the Enteric Pathogens Nucleic Acid Test were repeated from the original sample (new aliquot) per the package insert instructions and the repeat result was considered the final result.

The results of the Reproducibility Study for the Norovirus and Rotavirus targets are summarized in the tables below. In addition to the results presented, a single false positive Salmonella result occurred for a Rotavirus sample.

Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Total Final PercentAgreement with ExpectedResult(95 % CI)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>90/90100%(96-100)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>27/3090.0%(73.5-97.9)</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>87/9096.7%(90.6-99.3)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>26/2892.9%(76.5-99.1)</td><td rowspan=1 colspan=1>86/8897.7%(92.0-99.7)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>28/3093.3%(77.9-99.2)</td><td rowspan=1 colspan=1>28/3093.3% (77.9-99.2)</td><td rowspan=1 colspan=1>86/9095.6%(89.0-98.8)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>28/28100%(87.7-100)</td><td rowspan=1 colspan=1>30/30100% (88.4-100)</td><td rowspan=1 colspan=1>28/29100% (82.2-99.9)</td><td rowspan=1 colspan=1>86/8798.9%(93.8-100)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>30/30100%(88.4-100)</td><td rowspan=1 colspan=1>29/29100%(88.1-100)</td><td rowspan=1 colspan=1>89/89100%(95.9-100)</td></tr></table>

Of the 540 tests performed for the viral targets, there were 25 initial No Call results $( 4 . 6 \% )$ and after repeat testing, six samples yielded a final No Call result $( 1 . 1 \% )$ . In addition, three samples gave initial pre-AE errors $( 0 . 3 \% )$ all of which gave the expected result after repeat testing.

Initial No Call Rates Observed by Type   

<table><tr><td rowspan=1 colspan=1>Initial No Call Type</td><td rowspan=1 colspan=1>Observed Rate</td></tr><tr><td rowspan=1 colspan=1>No Call - INTCTL</td><td rowspan=1 colspan=1>1.5%(8/540)</td></tr><tr><td rowspan=1 colspan=1>NoCall - INTCTL 1</td><td rowspan=1 colspan=1>1.1%(6/540)</td></tr><tr><td rowspan=1 colspan=1>NoCall - INTCTL 2</td><td rowspan=1 colspan=1>0.9%(5/540)</td></tr><tr><td rowspan=1 colspan=1>No Call - VARIATION</td><td rowspan=1 colspan=1>1.1%(6/540)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4.6%(25/540)</td></tr></table>

With the inclusion of the Norovirus and Rotavirus targets to the EP test, the results of the positive bacterial as well as negative samples tested in the full Reproducibility Study performed for K140083 were assessed with the viral target results unmasked. Across all samples tested in the original reproducibility study, there were no false positive results for Norovirus or Rotavirus and therefore the Reproducibility Study results in K140083 are unchanged with the addition of the viral targets.

b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Assay Controls

Enteric Pathogens Nucleic Acid Test (EP) is a ‘specimen-to-result’ detection system wherein nucleic acids are isolated and identified by an automated process. In order to decrease the likelihood of false results due to processing errors, all reagents are prepackaged in single-use disposables. The following controls are built into the system so that processing failures are identified and the appropriate actions are taken.

INTL CTL 1: An artificial DNA construct serves as the internal hybridization control or “positive control for detection.” It is a synthetic DNA construct present in the sample buffer and is hybridized in parallel with every test sample.

INTL CTL 2: An MS2 Phage serves as a specimen isolation and amplification control and is referred to as the Internal Processing Control 2. This control is added to each specimen prior to the extraction step.

For a valid Negative test result, both IC1 and IC2 must be “Detected.” If IC1 or IC2 are not detected a “No Call – INT CTL 1” or a “No Call – INT CTL $2 ^ { \circ }$ is provided respectively. If both IC1 and IC2 are not detected, a “NO CALL – INT CTL” result is provided. The recommended action for any “No Call” decision is to repeat the test

according to the instructions package insert. These internal controls are not utilized for the detection of positive samples.

# Imaging Controls:

These controls comprise a set of oligonucleotide spots on the Test Substrate that act as a quality check for proper fluid control and movement between the Test Cartridge and the Reagent Pack. Inability to detect the imaging controls results in a “No Call – No Grid.” The Imaging Control signal is generated when a signal probe hybridizes to an oligonucleotide on the substrate. Signal at these spots indicates that the steps of probe hybridization and signal enhancement were performed as intended.

In summary, the detection algorithm requires that all internal controls are valid before decisions regarding the presence or absence of any other target on the panel can be reported. If all controls are not within the predefined specification, a no call result will be obtained and further action should be taken according to the product labeling.

Positive and Negative External Controls

External controls are not provided with the EP test; however the package insert indicates that external quality control testing should be performed in conformance with local, state, and federal regulations or accreditation organizations as applicable and users should follow their laboratory’s standard quality control procedures. Refer to clinical the study section below for results of external quality control testing performed during the clinical study.

# d. Detection limit:

Refer to K140083 for Limit of Detection (LoD) Study results for the previously cleared EP bacterial analytes.

The LoD for the viral analytes was evaluated using samples prepared with previously characterized and quantified preparations of Norovirus and Rotavirus strains serially diluted in Negative Stool Matrix (NSM). Each concentration in the dilution series was tested in replicates of four and the preliminary LoD for each targeted organism was estimated as the concentration level with the lowest concentration level for which all four replicates were positive. The LoD was then confirmed by testing 20 additional replicates at the preliminary LoD concentration to establish that the strain was detected in at least $9 5 \%$ of the measurements. If an organism was detected in all replicates (20/20), the next lower concentration was also tested to verify a detection rate below $9 5 \%$ (19/20). If the initial results yielded a 19/20 correct result, no further testing was conducted and the level was confirmed as the LoD. The validated LoD for the EP test Norovirus and Rotavirus targets is summarized in the following tables:

LoD Established for Rotavirus and Norovirus Strains   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">LoD/mL</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI (CDC 2142)</td><td colspan="1" rowspan="1">Clinical Sample (CDC)</td><td colspan="1" rowspan="1">4.12×105copies</td></tr><tr><td colspan="1" rowspan="1">NorovirusGII (D17219)</td><td colspan="1" rowspan="1">Clinical Sample (CDC)</td><td colspan="1" rowspan="1">1.67×106 copies</td></tr><tr><td colspan="1" rowspan="1">Rotavirus DS1 (VR-2550)</td><td colspan="1" rowspan="1">VR-2550 (ATCC)</td><td colspan="1" rowspan="1">1.11×103 TCID50</td></tr><tr><td colspan="1" rowspan="1">Rotavirus - W161 (VR-2551)</td><td colspan="1" rowspan="1">VR-2551 (ATCC)</td><td colspan="1" rowspan="1">3.70×102TCID50</td></tr></table>

Final LoD Established for the EP Test Viral Analytes   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>LoD/mL</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>4.12×105- 1.67×106 copies</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>3.70×102-1.11×103 TCID50</td></tr></table>

# e. Analytical Reactivity (Inclusivity):

Refer to K140083 for Analytical Reactivity Study results for previously cleared EP bacterial analytes.

An Analytical Reactivity study was performed to determine whether the EP test is able to detect a variety of organisms that represent the genetic diversity of Norovirus and Rotavirus stains. This study included different strains that were used to prepare samples at low concentrations (near the LoD). The study included a panel of 41 viral strains, representing temporal, geographical, and phylogenetic diversity of the organisms. As propagating Noroviruses in vitro is challenging, the Norovirus strains evaluated in the study were patient stool samples, characterized and quantitated by the CDC using a validated realtime RT- PCR reference assay. The Norovirus samples were prepared by diluting the stool samples to intermediate concentrations in viral transport media and spiked into NSM to yield the desired concentration. Rotavirus samples were prepared using titered stocks diluted in viral transport media and then subsequently added to NSM to give the appropriate sample concentration. All viral strains were tested in triplicate.

All tested strains of Rotavirus and Norovirus GI were detected at a concentration of 3x LoD. All strains of Norovirus GII were detected; however for some strains, increased organism concentrations (10x and 50x LoD) were required to achieve detection of all replicates. The strains tested are presented in the following table:

Inclusivity Results for Norovirus and Rotavirus   

<table><tr><td rowspan=1 colspan=1>ReportableTarget</td><td rowspan=1 colspan=1>Strain #/UniqueIdentifier</td><td rowspan=1 colspan=1>Inclusive Organisms</td><td rowspan=1 colspan=1>Inclusivity Titer</td><td rowspan=1 colspan=1>Multiple ofLoD</td></tr><tr><td rowspan=11 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>CDC GI.1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.65×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.2-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×1066 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.2-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.3-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.3-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.4-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.4-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.5</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.6</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.7-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI.7-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>CDC GI.8-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>1.24×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GI 8-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>1.24×106copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=16 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>CDC GII.1-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>1.67×107copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>1.67×107 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.3-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>5.01×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.3-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×10</td><td rowspan=1 colspan=1>1.67×107 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.3-3</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×10</td><td rowspan=1 colspan=1>1.67×107 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.4-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×107</td><td rowspan=1 colspan=1>copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.4-3</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×107</td><td rowspan=1 colspan=1>copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.4-4</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>5.01×106</td><td rowspan=1 colspan=1>copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.4-5</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×10</td><td rowspan=1 colspan=1>1.67×107 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.5</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>5.01×10</td><td rowspan=1 colspan=1>5.01×106 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.7</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>8.35×10</td><td rowspan=1 colspan=1>8.35×107 copies/mL</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.10</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×10</td><td rowspan=1 colspan=1>1.67×107 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.12-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>5.01×106bcopies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.12-2</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.67×107</td><td rowspan=1 colspan=1>7copies/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.16</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>8.35×107</td><td rowspan=1 colspan=1>copies/mL</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=1 colspan=1>CDC GII.17-1</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=2>8.35×107 copies/mL</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=12 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>VR-2272</td><td rowspan=1 colspan=1>Group A, G1</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>VR-2275</td><td rowspan=1 colspan=1>Group A, G1</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>VR-2417</td><td rowspan=1 colspan=1>Group A, G2P[3]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>VR-2550</td><td rowspan=1 colspan=1>Group A, G2P[1]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>VR-2551</td><td rowspan=1 colspan=1>Group A, G9P[1]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0810041CF</td><td rowspan=1 colspan=1>Group A, G1P[1]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G6P14 #528</td><td rowspan=1 colspan=1>Group A, G6P[14]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G8P14 #511</td><td rowspan=1 colspan=1>Group A, G8P[14]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G12P4 #262</td><td rowspan=1 colspan=1>Group A, G12P[4]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G1P8 #102</td><td rowspan=1 colspan=1>Group A, G1P[8]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G2P4 #95</td><td rowspan=1 colspan=1>Group A, G2P[4]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MCRI G3P8 #XX 434</td><td rowspan=1 colspan=1>Group A, G3P[8]</td><td rowspan=1 colspan=2>3.33×103 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# In silico Results for Norovirus:

Based on in silico analysis, the majority of genotypes of Norovirus GII are expected to be detected by the EP test with the exception of GII.11 which is not expected to be detected by EP. Additionally, GII.6 and GII.13 are expected to either be not detected or to be detected with reduced sensitivity (i.e., detected only at higher concentrations). Norovirus GII.2, GII.3, and GII.12 genotypes are predicted to be detected with reduced sensitivity. Norovirus strains GII.9, GII.14, and GIV.1 are predicted to be detected by EP and Norovirus strains GII.8 could not be evaluated due to lack of available sequence information.

The need for higher concentrations for detection of some Norovirus GII strains with the EP test is likely a reflection of the high genetic diversity within Norovirus GII genotypes resulting in reduced sensitivity due to inefficient amplification or hybridization or both.

The following limitations are included in the draft EP test Package Insert to address the detection or predicted detection of various Norovirus genogroups/genotypes:

EP is expected to be inclusive to most strains of the Norovirus GI, GII, and GIV genotypes known to cause disease in humans based on empirical testing and supplemented by in silico analysis. However, due to the high genetic diversity within Noroviruses, some strains may not be detected or may be detected with reduced sensitivity by EP. Refer to the Analytical Sensitivity (Inclusivity) section and Table 16 for details.   
· EP inclusivity to Norovirus strains GII.9, GII.14, and GIV.1 was evaluated by in silico analysis only. Rare Norovirus genotypes GII.6 and GII.13 were determined to either be detected at reduced sensitivity or predicted to not be detected by EP based on in silico analysis. For GII.8, EP inclusivity is unknown as, in the absence of sequence information, in silico analysis could not be performed.   
· Norovirus GII.11 is not expected to be detected by EP based on in silico analysis.

In silico Results for Rotavirus:

Rotavirus genotypes G4, G5, G10, G11, and G15 were not wet-tested but are predicted to be detected based on in silico data. Sequence data regarding genotypes G7, G21, and G24 was not available and therefore it is not known if these strains would be detected by the EP test. The following limitation is included in the EP test package insert.

EP inclusivity to Rotavirus A genotypes, G4, G5, G10, G11, and G15 was evaluated based on in silico analysis only. Inclusivity of EP to Rotavirus A genotypes G7, G21, and G24 is unknown; representative strains were not available for empirical testing and in the absence of sequence information, in silico analysis could not be performed.

f. Analytical specificity (exclusivity):

Refer to K140083 for Analytical Specificity Study results for previously cleared EP bacterial analytes.

The specificity of the Norovirus and Rotavirus probes in the EP Test was addressed through the “unmasking” of the viral target results generated in the original analytical specificity study performed for K140083. In the study, a panel of 158 organisms comprised of 34 bacterial organisms, 18 viruses, four parasites, one fungal organism ( $C .$ . albicans) and one human cell line were evaluated. Samples were prepared by spiking NSM with bacterial strains and Candida albicans at $1 0 ^ { 7 } \dot { \mathrm { C F U / m L } }$ and viral strains at $1 0 ^ { 6 } \mathrm { P F U / m L }$ . If an organism could not be accurately titered or was unavailable in whole organism form, genomic DNA was tested at approximately $1 0 ^ { 7 } \mathrm { C F U / m L }$ . All samples were tested in triplicate. Results for the Norovirus and Rotavirus targets yielded a “Not Detected” results in all three replicates for all organisms evaluated demonstrating no cross-reactivity with the EP test. The organisms evaluated in the analytical specificity study are presented in the following tables:

<table><tr><td colspan="6" rowspan="1">Bacteria and Yeast (tested at 1x10' CFU/mL unless otherwise noted)</td></tr><tr><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">ATCC SourceID</td><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">ATCC SourceID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia</td><td colspan="1" rowspan="1">defectiva</td><td colspan="1" rowspan="1">49176</td><td colspan="1" rowspan="2">Enterococcus</td><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">51299</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">19606</td><td colspan="1" rowspan="1">faecium</td><td colspan="1" rowspan="1">700221</td></tr><tr><td colspan="1" rowspan="1">lwoffi</td><td colspan="1" rowspan="1">17925</td><td colspan="1" rowspan="10">Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="1" rowspan="1">23511</td></tr><tr><td colspan="1" rowspan="2">Acrobacter</td><td colspan="1" rowspan="1">butzleri</td><td colspan="1" rowspan="1">49942</td><td colspan="1" rowspan="2">coli(non-pathogenic)</td><td colspan="1" rowspan="1">25922</td></tr><tr><td colspan="1" rowspan="1">cryaerophilus</td><td colspan="1" rowspan="1">43157</td><td colspan="1" rowspan="1">35218</td></tr><tr><td colspan="1" rowspan="13">Aeromonas</td><td colspan="1" rowspan="1">allosaccharophila</td><td colspan="1" rowspan="1">35942</td><td colspan="1" rowspan="1">coli (EAEC)</td><td colspan="1" rowspan="1">23501</td></tr><tr><td colspan="1" rowspan="1">bestiarum</td><td colspan="1" rowspan="1">BAA-231</td><td colspan="1" rowspan="1">coli (EPEC)</td><td colspan="1" rowspan="1">BAA-1704</td></tr><tr><td colspan="1" rowspan="1">caviae</td><td colspan="1" rowspan="1">13136</td><td colspan="1" rowspan="1">coli (EPEC)</td><td colspan="1" rowspan="1">12014</td></tr><tr><td colspan="1" rowspan="1">encheleia</td><td colspan="1" rowspan="1">51929</td><td colspan="1" rowspan="1">coli (ETEC)</td><td colspan="1" rowspan="1">43886</td></tr><tr><td colspan="1" rowspan="1">enteropelogenes</td><td colspan="1" rowspan="1">49803</td><td colspan="1" rowspan="1">coli (ETEC)</td><td colspan="1" rowspan="1">23519</td></tr><tr><td colspan="1" rowspan="1">eucrenophila</td><td colspan="1" rowspan="1">23309</td><td colspan="1" rowspan="1">fergusonii</td><td colspan="1" rowspan="1">35469</td></tr><tr><td colspan="1" rowspan="1">hydrophilia</td><td colspan="1" rowspan="1">7966</td><td colspan="1" rowspan="1">hermannii</td><td colspan="1" rowspan="1">33650</td></tr><tr><td colspan="1" rowspan="1">jandaei</td><td colspan="1" rowspan="1">49568</td><td colspan="1" rowspan="1">Fusobacterium</td><td colspan="1" rowspan="1">varium</td><td colspan="1" rowspan="1">8501</td></tr><tr><td colspan="1" rowspan="2">salmonicidasubsp masoucida</td><td colspan="1" rowspan="2">27013</td><td colspan="1" rowspan="5">Helicobacter</td><td colspan="1" rowspan="1">hepaticus*</td><td colspan="1" rowspan="1">51449</td></tr><tr><td colspan="1" rowspan="1">pylori</td><td colspan="1" rowspan="1">43504</td></tr><tr><td colspan="1" rowspan="2">salmonicidasubspsalmonicida</td><td colspan="1" rowspan="1">14174</td><td colspan="1" rowspan="1">pylori</td><td colspan="1" rowspan="1">700392</td></tr><tr><td colspan="1" rowspan="1">33658</td><td colspan="1" rowspan="1">pylori</td><td colspan="1" rowspan="1">49503</td></tr><tr><td colspan="1" rowspan="1">veronii</td><td colspan="1" rowspan="1">9071</td><td colspan="1" rowspan="1">pylori</td><td colspan="1" rowspan="1">51652</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes</td><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">15554</td><td colspan="1" rowspan="2">Klebsiella</td><td colspan="1" rowspan="1">oxytoca</td><td colspan="1" rowspan="1">43165</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">cereus</td><td colspan="1" rowspan="1">10702</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">13883</td></tr><tr><td colspan="1" rowspan="4">Bacteroides</td><td colspan="1" rowspan="1">caccae</td><td colspan="1" rowspan="1">43185</td><td colspan="1" rowspan="3">Lactobacillus</td><td colspan="1" rowspan="1">acidophilus</td><td colspan="1" rowspan="1">11975</td></tr><tr><td colspan="1" rowspan="1">fragilis</td><td colspan="1" rowspan="1">25285</td><td colspan="1" rowspan="1">reuteri</td><td colspan="1" rowspan="1">23272</td></tr><tr><td colspan="1" rowspan="1">merdae</td><td colspan="1" rowspan="1">43184</td><td colspan="1" rowspan="1">rhamnosus</td><td colspan="1" rowspan="1">53103</td></tr><tr><td colspan="1" rowspan="1">stercoris</td><td colspan="1" rowspan="1">43183</td><td colspan="1" rowspan="1">Lactococcus</td><td colspan="1" rowspan="1">lactis</td><td colspan="1" rowspan="1">11454</td></tr><tr><td colspan="1" rowspan="1">Candida</td><td colspan="1" rowspan="1">albicans</td><td colspan="1" rowspan="1">10231</td><td colspan="1" rowspan="1">Leminorela</td><td colspan="1" rowspan="1">grimontii</td><td colspan="1" rowspan="1">33999</td></tr><tr><td colspan="1" rowspan="1">Cedecea</td><td colspan="1" rowspan="1">davisae</td><td colspan="1" rowspan="1">33431</td><td colspan="1" rowspan="2">Listeria</td><td colspan="1" rowspan="1">grayi</td><td colspan="1" rowspan="1">19120</td></tr><tr><td colspan="1" rowspan="3">Citrobacter</td><td colspan="1" rowspan="1">amalonaticus</td><td colspan="1" rowspan="1">25407</td><td colspan="1" rowspan="1">monocytogenes</td><td colspan="1" rowspan="1">7644</td></tr><tr><td colspan="1" rowspan="1">freundii</td><td colspan="1" rowspan="1">8090</td><td colspan="1" rowspan="1">Morganella</td><td colspan="1" rowspan="1">morganii</td><td colspan="1" rowspan="1">25830</td></tr><tr><td colspan="1" rowspan="1">sedlakii</td><td colspan="1" rowspan="1">51115</td><td colspan="1" rowspan="1">Peptostreptococcus</td><td colspan="1" rowspan="1">anaerobius</td><td colspan="1" rowspan="1">27337</td></tr><tr><td colspan="1" rowspan="16">Clostridium</td><td colspan="1" rowspan="1">bifermentans</td><td colspan="1" rowspan="1">638</td><td colspan="1" rowspan="1">Plesiomonas</td><td colspan="1" rowspan="1">shigelloides</td><td colspan="1" rowspan="1">14029</td></tr><tr><td colspan="1" rowspan="1">bolteae</td><td colspan="1" rowspan="1">BAA-613</td><td colspan="1" rowspan="1">Porphyromonas</td><td colspan="1" rowspan="1">asaccharoluticus</td><td colspan="1" rowspan="1">25260</td></tr><tr><td colspan="1" rowspan="1">butyricum</td><td colspan="1" rowspan="1">19398</td><td colspan="1" rowspan="1">Prevotella</td><td colspan="1" rowspan="1">melaniogenica</td><td colspan="1" rowspan="1">25845</td></tr><tr><td colspan="1" rowspan="1">difficile</td><td colspan="1" rowspan="1">BAA-1805</td><td colspan="1" rowspan="3">Proteus</td><td colspan="1" rowspan="1">mirabilis</td><td colspan="1" rowspan="1">25933</td></tr><tr><td colspan="1" rowspan="1">difficile</td><td colspan="1" rowspan="1">17857</td><td colspan="1" rowspan="1">vulgaris</td><td colspan="1" rowspan="1">29905</td></tr><tr><td colspan="1" rowspan="1">difficile, non-tox</td><td colspan="1" rowspan="1">BAA-1801</td><td colspan="1" rowspan="1">penneri</td><td colspan="1" rowspan="1">35198</td></tr><tr><td colspan="1" rowspan="1">haemolyticum</td><td colspan="1" rowspan="1">9650</td><td colspan="1" rowspan="3">Providencia</td><td colspan="1" rowspan="1">stuartii</td><td colspan="1" rowspan="1">33672</td></tr><tr><td colspan="1" rowspan="1">methylpentosum</td><td colspan="1" rowspan="1">43829</td><td colspan="1" rowspan="1">alcalifaciens</td><td colspan="1" rowspan="1">9886</td></tr><tr><td colspan="1" rowspan="1">nexile</td><td colspan="1" rowspan="1">27757</td><td colspan="1" rowspan="1">rettgeri</td><td colspan="1" rowspan="1">9250</td></tr><tr><td colspan="1" rowspan="1">noyvi</td><td colspan="1" rowspan="1">19402</td><td colspan="1" rowspan="4">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">35554</td></tr><tr><td colspan="1" rowspan="1">orbiscindens</td><td colspan="1" rowspan="1">49531</td><td colspan="1" rowspan="1">fluorescens</td><td colspan="1" rowspan="1">13525</td></tr><tr><td colspan="1" rowspan="1">perfringens</td><td colspan="1" rowspan="1">13124</td><td colspan="1" rowspan="1">putida</td><td colspan="1" rowspan="1">12633</td></tr><tr><td colspan="1" rowspan="1">scindens</td><td colspan="1" rowspan="1">35704</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">27853</td></tr><tr><td colspan="1" rowspan="1">septicum</td><td colspan="1" rowspan="1">12464</td><td colspan="1" rowspan="1">Ruminococcus</td><td colspan="1" rowspan="1">bromii</td><td colspan="1" rowspan="1">27255</td></tr><tr><td colspan="1" rowspan="1">sordellii</td><td colspan="1" rowspan="1">9714</td><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">liquefacians</td><td colspan="1" rowspan="1">27592</td></tr><tr><td colspan="1" rowspan="1">spiroforme</td><td colspan="1" rowspan="1">29899</td><td colspan="1" rowspan="1">marcescens</td><td colspan="1" rowspan="1">13880</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">sporogenes</td><td colspan="1" rowspan="1">15579</td><td colspan="1" rowspan="2">Staphylococcus</td><td colspan="1" rowspan="1">aureus</td><td colspan="1" rowspan="1">43300</td></tr><tr><td colspan="1" rowspan="1">Colinsella</td><td colspan="1" rowspan="1">aerofaciens</td><td colspan="1" rowspan="1">25986</td><td colspan="1" rowspan="1">epidermidis</td><td colspan="1" rowspan="1">700583</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio</td><td colspan="1" rowspan="1">piger</td><td colspan="1" rowspan="1">29098</td><td colspan="1" rowspan="3">Streptococcus</td><td colspan="1" rowspan="1">agalactiae,O90R</td><td colspan="1" rowspan="1">12386</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella</td><td colspan="1" rowspan="1">tarda</td><td colspan="1" rowspan="1">15947</td><td colspan="1" rowspan="1">dysgalactiae</td><td colspan="1" rowspan="1">12394</td></tr><tr><td colspan="1" rowspan="2">Enterobacter</td><td colspan="1" rowspan="1">aerogenes</td><td colspan="1" rowspan="1">13048</td><td colspan="1" rowspan="1">mutans</td><td colspan="1" rowspan="1">25175</td></tr><tr><td colspan="1" rowspan="1">cloacae</td><td colspan="1" rowspan="1">29006</td><td></td><td></td><td></td></tr></table>

<table><tr><td colspan="4" rowspan="1">Viral, Parasites and Human DNA.</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Strain #/UniqueIdentifier/Source</td><td colspan="1" rowspan="1">Serovars/ Groups</td><td colspan="1" rowspan="1">Titer Tested</td></tr><tr><td colspan="1" rowspan="11">Adenovirus</td><td colspan="1" rowspan="1">0810050CF</td><td colspan="1" rowspan="1">Type 1/Group C</td><td colspan="1" rowspan="9">1.0x106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">0810110CF</td><td colspan="1" rowspan="1">Type 2/Group C</td></tr><tr><td colspan="1" rowspan="1">0810062CF</td><td colspan="1" rowspan="1">Type 3/Group B1</td></tr><tr><td colspan="1" rowspan="1">0810070CF</td><td colspan="1" rowspan="1">Type 4/Group E</td></tr><tr><td colspan="1" rowspan="1">0810020CF</td><td colspan="1" rowspan="1">Type 5/Group C</td></tr><tr><td colspan="1" rowspan="1">0810108CF</td><td colspan="1" rowspan="1">Type 14/Group B2</td></tr><tr><td colspan="1" rowspan="1">0810117CF</td><td colspan="1" rowspan="1">Type 26/Group D</td></tr><tr><td colspan="1" rowspan="1">0810073CF</td><td colspan="1" rowspan="1">Type 31/Group A</td></tr><tr><td colspan="1" rowspan="1">0810119CF</td><td colspan="1" rowspan="1">Type 37/Group D</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-931</td><td colspan="1" rowspan="1">Type 40/Group F</td><td colspan="1" rowspan="1">1.58x105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">VR-1572D</td><td colspan="1" rowspan="1">Human 4</td><td colspan="1" rowspan="1">1.0x106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">P#711/24/08</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.0x107 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">ATCC VR-184</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3.16x105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">0810003-CF</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.24x105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">0810023-CF</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">VR-213</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0x106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">Sapovirus</td><td colspan="1" rowspan="1">2008729730</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">1x107 copies/mL</td></tr><tr><td colspan="1" rowspan="1">2009726567</td></tr><tr><td colspan="3" rowspan="1">Human Cell Line</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Colon epithelial cells(colorectaladenocarcinoma)</td><td colspan="1" rowspan="1">ATCC CCL-218</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0x105cells/mL</td></tr><tr><td colspan="4" rowspan="1">Parasites</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">ATCC 50608-D</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">1x107 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidiumparvum</td><td colspan="1" rowspan="1">PRA-67-D</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">ATCC 30459-D</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">ATCC 50803-D</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

<table><tr><td rowspan=1 colspan=4>Species of Campylobacter, Vibrio, and Yersinia not detected by EP</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Designation</td><td rowspan=1 colspan=1>Titer tested(CFU/mL)</td></tr><tr><td rowspan=13 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>concisus</td><td rowspan=1 colspan=1>ATCC BAA-1457</td><td rowspan=1 colspan=1>5.5x106</td></tr><tr><td rowspan=1 colspan=1>curvus</td><td rowspan=1 colspan=1>ATCC BAA-1459</td><td rowspan=1 colspan=1>6.1x106</td></tr><tr><td rowspan=1 colspan=1>fetus</td><td rowspan=1 colspan=1>ATCC 25936</td><td rowspan=1 colspan=1>3.9x107</td></tr><tr><td rowspan=1 colspan=1>gracilis</td><td rowspan=1 colspan=1>ATCC 33236</td><td rowspan=1 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>hominis (gDNA)</td><td rowspan=1 colspan=1>ATCC BAA-381D-5</td><td rowspan=1 colspan=1>1.0x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>hyointestinalis</td><td rowspan=1 colspan=1>ATCC 35217</td><td rowspan=7 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>insulaenigrae</td><td rowspan=1 colspan=1>CCUG 48653</td></tr><tr><td rowspan=1 colspan=1>lanienae</td><td rowspan=1 colspan=1>CCUG 44467</td></tr><tr><td rowspan=1 colspan=1>mucosalis</td><td rowspan=1 colspan=1>ATCC 49352</td></tr><tr><td rowspan=1 colspan=1>rectus</td><td rowspan=1 colspan=1>ATCC 33238</td></tr><tr><td rowspan=1 colspan=1>showae</td><td rowspan=1 colspan=1>ATCC 51146</td></tr><tr><td rowspan=1 colspan=1>sputorum</td><td rowspan=1 colspan=1>ATCC 35980</td></tr><tr><td rowspan=1 colspan=1>upsaliensis</td><td rowspan=1 colspan=1>ATCC BAA-1059</td><td rowspan=1 colspan=1>7.5x106</td></tr><tr><td rowspan=12 colspan=1>Vibrio</td><td rowspan=1 colspan=1>alginolyticus</td><td rowspan=1 colspan=1>ATCC 17749</td><td rowspan=22 colspan=1>1.0x107</td></tr><tr><td rowspan=1 colspan=1>campbellii</td><td rowspan=1 colspan=1>ATCC 25920</td></tr><tr><td rowspan=1 colspan=1>cincinnatiensis</td><td rowspan=1 colspan=1>ATCC 35912</td></tr><tr><td rowspan=1 colspan=1>fluvialis</td><td rowspan=1 colspan=1>ATCC 33809</td></tr><tr><td rowspan=1 colspan=1>furnissii</td><td rowspan=1 colspan=1>ATCC 11218</td></tr><tr><td rowspan=1 colspan=1>harveyi</td><td rowspan=1 colspan=1>ATCC 14126</td></tr><tr><td rowspan=1 colspan=1>metschnikovii</td><td rowspan=1 colspan=1>ATCC 7708</td></tr><tr><td rowspan=1 colspan=1>mimicus</td><td rowspan=1 colspan=1>ATCC 33653</td></tr><tr><td rowspan=1 colspan=1>tubiashii*</td><td rowspan=1 colspan=1>ATCC 18106</td></tr><tr><td rowspan=3 colspan=1>vulnificus</td><td rowspan=1 colspan=1>ATCC BAA-86</td></tr><tr><td rowspan=1 colspan=1>ATCC 27562</td></tr><tr><td rowspan=1 colspan=1>ATCC 33815</td></tr><tr><td rowspan=10 colspan=1>Yersinia</td><td rowspan=1 colspan=1>aldovae</td><td rowspan=1 colspan=1>ATCC 35236</td></tr><tr><td rowspan=1 colspan=1>aleksiciae</td><td rowspan=1 colspan=1>CCUG 52872</td></tr><tr><td rowspan=1 colspan=1>bercovieri</td><td rowspan=1 colspan=1>ATCC 43970</td></tr><tr><td rowspan=1 colspan=1>frederiksenii</td><td rowspan=1 colspan=1>ATCC 33644</td></tr><tr><td rowspan=1 colspan=1>intermedia</td><td rowspan=1 colspan=1>ATCC 33647</td></tr><tr><td rowspan=1 colspan=1>kristensenii</td><td rowspan=1 colspan=1>ATCC 33639</td></tr><tr><td rowspan=1 colspan=1>mollaretii</td><td rowspan=1 colspan=1>ATCC 43969</td></tr><tr><td rowspan=1 colspan=1>pseudotuberculosis</td><td rowspan=1 colspan=1>ATCC 29910</td></tr><tr><td rowspan=1 colspan=1>ruckeri</td><td rowspan=1 colspan=1>ATCC 29473</td></tr><tr><td rowspan=1 colspan=1>rohdei</td><td rowspan=1 colspan=1>ATCC 43380</td></tr></table>

Potential “within panel” cross-reactivity was evaluated using representative targeted viral and bacterial analytes tested in the Carryover/Cross-Contamination Study performed for the initial clearance of the EP Test (K140083). The study yielded no EP false positive results, demonstrating that cross-reactivity does not occur between EP targets. The strains evaluated are presented in the following table.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Expected EPTest</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica subsp. enterocolitica</td><td rowspan=1 colspan=1>ATCC 23715</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>5×106 cFU/mL</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae, Stx1</td><td rowspan=1 colspan=1>ATCC 29026</td><td rowspan=1 colspan=1>Shigellaspp., Shiga</td><td rowspan=1 colspan=1>5×106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli, Stx2</td><td rowspan=1 colspan=1>ATCC BAA-176</td><td rowspan=1 colspan=1>Shiga toxin 2</td><td rowspan=1 colspan=1>5×106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica serovar Typhi</td><td rowspan=1 colspan=1>ATCC 7251</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>5×106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>ATCC 43429</td><td rowspan=1 colspan=1>Campylobacter spp.</td><td rowspan=1 colspan=1>5×106 cFU/mL</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>ATCC 39315</td><td rowspan=1 colspan=1>Vibrio spp.</td><td rowspan=1 colspan=1>5×106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>CDC 2142</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>1.0×108copies/mL</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>CDC 2012</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>3.0×106copies/mL</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>VR-2551</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>1.6×106TCID50/mL</td></tr></table>

For viral strains that were difficult to obtain, the analytical specificity of the EP Test was evaluated via in silico analysis. This analysis demonstrated that Norovirus genotype GIV.2, Norovirus genogroups GIII, GV and Rotavirus genogroups B, C, D, and NADRV (Novel Adult Diarrheal Rotavirus) are not predicted to cross-react with the EP test. The following product limitations are included in the package insert:

Norovirus GIII, GIV.2, and GV are not expected to be detected based on in silico analysis.   
Rotavirus genogroups B, D, and NADRV are not expected to be detected by EP based on in silico analysis. In addition, Rotavirus genogroup C strains are not expected to be detected by EP, with the exception of porcine strains within this genogroup.

g. Matrix equivalence Study

N/A

h. Interference Study

Refer to K140083 for Interference Study results for previously cleared EP bacterial analytes.

A study was performed to assess the potential inhibitory effect of endogenous and exogenous substances commonly found in clinical stool specimens. Two representative strains for both Norovirus and Rotavirus were evaluated as shown in the following table:

Viral strains tested Interfering Substance Study

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source/Strain</td><td rowspan=1 colspan=1>Expected Calls</td><td rowspan=1 colspan=1>LoD in Stool</td></tr><tr><td rowspan=1 colspan=1>Norovirus (GI)</td><td rowspan=1 colspan=1>CDC 2142</td><td rowspan=1 colspan=1>Norovirus Detected</td><td rowspan=1 colspan=1>4.12×105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2550</td><td rowspan=1 colspan=1>Rotavirus Detected</td><td rowspan=1 colspan=1>1.11×10³ TCID50/mL</td></tr></table>

Samples were prepared with each potentially interfering substance at high, medically relevant concentrations in NSM with Norovirus and Rotavirus strains at $3 \mathbf { x }$ the LoD. Samples containing NSM/interferent only were tested as controls. Each sample was tested in replicates of three. The potential interfering substances that were evaluated are provided in the following table.

Potentially Interfering Substances Evaluated   

<table><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>InterferentConcentration instool</td></tr><tr><td rowspan=1 colspan=1>Intralipid</td><td rowspan=1 colspan=1>Triglyceride (Fecal Fat)</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>Cholesterol (Fecal Fat)</td><td rowspan=1 colspan=1>5% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>Glucose, Hormones, Enzymes, Ions,Iron etc.</td><td rowspan=1 colspan=1>40% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucus (Nasopharyngeal swab sample in UTM)</td><td rowspan=1 colspan=1>Immunoglobulins, Lysozyme,Polymers</td><td rowspan=1 colspan=1>40% w/v</td></tr><tr><td rowspan=1 colspan=1>Nystatin Suspension</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Preparation H® Anti-itch Hydrocortisone 1%</td><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Desitin Maximum Strength Original Paste</td><td rowspan=1 colspan=1>Zinc Oxide</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Preparation H® Hemorrhoidal Ointment</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Options Conceptrol®Vaginal Contraceptive Gel</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Wet Ones®Antibacterial Hand Wipes</td><td rowspan=1 colspan=1>Benzalkonium Chloride, Ethanol</td><td rowspan=1 colspan=1>30% v/v</td></tr><tr><td rowspan=1 colspan=1>K-Y®Personal Lubricant Jelly</td><td rowspan=1 colspan=1>Glycerin</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Vaseline Original 100% Pure Petroleum Jelly</td><td rowspan=1 colspan=1>Petroleum</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Tums Antacid with Calcium Extra Strength 750</td><td rowspan=1 colspan=1>Calcium Carbonate</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Gaviscon Extra Strength Liquid Antacid</td><td rowspan=1 colspan=1>Aluminum Hydroxide, MagnesiumHydroxide</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Mesalazine</td><td rowspan=1 colspan=1>S. Amino Salicylic Acid</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Immodium® AD Anti-Diarrheal</td><td rowspan=1 colspan=1>Loperamide Hydrochloride</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol Max Strength</td><td rowspan=1 colspan=1>Bismuth subsalicylate</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>MetronidazoleTopical Cream (0.75%)</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Mucin from bovine submaxillary glands, Type I-S(Dehydrated)</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin (Antibiotic)</td><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Control (no interferent)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The EP test correctly detected Rotavirus and Norovirus for all samples; however the following two samples yielded false positive results:

One of three replicates of low-titer Rotavirus with Loperamide HCl gave a false positive result for Norovirus; however the Rotavirus was detected correctly for this sample.   
One of three replicates of low titer Rotavirus with Zinc Oxide gave a false positive for Salmonella; however the Rotavirus was detected correctly for this sample.

Six additional replicates were tested for both of these samples and the additional testing yielded no further false positive results and therefore Loperamide HCl and Zinc Oxide are not considered to interfere with the EP test. In summary, the Interference Study results indicate that the substances evaluated should not affect the performance of the EP test for detection of Norovirus and Rotavirus.

# i. Fresh versus Frozen Study

Refer to K140083 for Fresh versus Frozen Study results for previously cleared EP bacterial analytes.

A fresh versus frozen equivalency study was performed to support inclusion of frozen samples in analytical studies as well as inclusion of frozen archived specimens in the clinical study. The study included samples prepared with three viral strains at five concentrations prepared in a 3-fold serial dilution series in NSM. Baseline testing of the freshly-prepared samples was performed in replicates of four; the four remaining vials were stored frozen at $\le 7 0 \mathrm { { ^ \circ C } }$ then thawed and tested. For demonstration of equivalence, the study acceptance criteria required detection of each targeted analyte in fresh and frozen samples with no higher than a 3-fold difference in the lowest concentration detected in fresh versus frozen samples.

Study results for both Norovirus strains demonstrated that the lowest concentration for which all replicates were detected was identical for fresh and frozen samples. For Rotavirus, frozen samples were detected at a 3-fold higher concentration than fresh samples, thereby meeting the predefined acceptance criteria. Therefore the fresh versus frozen study results shown in the table below in combination with study results for the bacterial analytes (K140083) support the use of simulated samples subjected to a single freeze/thaw cycle interchangeably with fresh/unfrozen simulated samples in the analytical studies. Additionally, these study results support the use of archived frozen specimens in the clinical study.

Summary of Results for Fresh Versus Frozen Study – NSM Samples   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Established LoD</td><td colspan="2" rowspan="1">Lowest Concentration where allReplicateswere "Detected" (/mL)</td></tr><tr><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">Frozen</td></tr><tr><td colspan="1" rowspan="1">Norovirus, GI</td><td colspan="1" rowspan="1">CDC 2142</td><td colspan="1" rowspan="1">4.12×105 copies/mL</td><td colspan="1" rowspan="1">1.37×105copies</td><td colspan="1" rowspan="1">1.37×105 copies</td></tr><tr><td colspan="1" rowspan="1">Norovirus, GII</td><td colspan="1" rowspan="1">D17219</td><td colspan="1" rowspan="1">1.67×106 copies/mL</td><td colspan="1" rowspan="1">1.67×106copies</td><td colspan="1" rowspan="1">1.67×106 copies</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">ATCC VR-2550</td><td colspan="1" rowspan="1">1.11×103 TCID50/mL</td><td colspan="1" rowspan="1">3.70×102 TCID50</td><td colspan="1" rowspan="1">1.11×103 TCID50</td></tr></table>

j. Carryover/Cross-Contamination

Refer to K140083 for Carryover/Cross-Contamination Study results for previously cleared EP bacterial analytes.

In order to assess the EP assay for potential carryover or cross-contamination from high positive Norovirus and Rotavirus samples, positive samples (100x LoD) prepared in NSM were tested in an alternating pattern with negative samples comprised of NSM only. The following Norovirus and Rotavirus strains were evaluated on three different Verigene Processor SP instruments as shown in the following tables.

Organisms Tested for the Carryover/Cross-Contamination Study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Expected EP Test Result</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>CDC 2142</td><td rowspan=1 colspan=1>4.12×107 copies/mL</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>D17219</td><td rowspan=1 colspan=1>1.67×108 copies/mL</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>ATCC VR-2550</td><td rowspan=1 colspan=1>1.11×105 TCID50/mL</td><td rowspan=1 colspan=1>Rotavirus</td></tr></table>

Sample Testing Order for the Carryover/Cross-Contamination Study   

<table><tr><td rowspan=2 colspan=1>Run No.</td><td rowspan=2 colspan=3>Verigene Processor SP1                            2                             3</td></tr><tr><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr></table>

The study results demonstrated that all nine negative samples yielded the expected “Not Detected” call for all EP analytes. These results combined with the initial carryover study results for the EP bacterial analytes (K140083) demonstrate that the carryover/cross contamination rate for the EP test is acceptable.

# k. Competitive Inhibition Study:

Refer to K140083 for Competitive Inhibition Study results for previously cleared EP bacterial analytes.

Within Panel Competitive Inhibition

In order to assess the EP test for potential competitive inhibition for the Norovirus and Rotavirus targets, samples were tested that contained various combinations of each viral target with other EP bacterial and viral EP test panel organisms. Contrived samples were prepared in NSM with Norovirus and Rotavirus present at approximately $3 \mathbf { x }$ LoD mixed with high concentrations of each potentially interfering organism $( > 1 0 ^ { 6 } \mathrm { \dot { C } F U / m L } )$ ). Testing was performed in triplicate for each unique sample combination. The EP test correctly detected both expected targets for all sample combinations with one exception in which one of three replicates of the low titer Rotavirus/high titer Y. enterocolitica sample gave a false negative result for Rotavirus. Testing of an additional six replicates of this combination yielded correct calls for both organisms suggesting that high concentrations of Y. enterocolitica should not interfere with the detection of Rotavirus by EP.

Evaluation of Potential Interference from Non-targeted Microorganisms

A study was performed to evaluate whether 14 microorganisms commonly present in stool may interfere with the EP test for detection of Norovirus and Rotavirus. Samples for this study were prepared in NSM with Norovirus GI and Rotavirus A strains at $3 \mathbf { x }$ LoD mixed with each potentially interfering organism at ${ > } 1 0 ^ { 7 } \mathrm { C F U / m L }$ for bacteria, and $9 \times 1 0 ^ { 6 }$ cells/mL and $7 \times 1 0 ^ { 5 }$ cells/mL respectively for parasites Blastocystis hominis and Entamoeba histolytica. Additional samples containing only Norovirus or Rotavirus were included as control samples. Testing was performed in triplicate with the organisms listed in the following table.

<table><tr><td rowspan=1 colspan=2>Organisms tested</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>Bacteroides</td><td rowspan=1 colspan=1>fragilis</td></tr><tr><td rowspan=1 colspan=1>Prevotella</td><td rowspan=1 colspan=1>oralis</td></tr><tr><td rowspan=1 colspan=1>Prevotella</td><td rowspan=1 colspan=1>melaninogenicus</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium</td><td rowspan=1 colspan=1>bifidum</td></tr><tr><td rowspan=1 colspan=1>Clostridium</td><td rowspan=1 colspan=1>perfringens</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>aerogenes</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>faecalis</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>pneumonia</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>acidophilus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>aureus</td></tr><tr><td rowspan=1 colspan=1>Blastocystis</td><td rowspan=1 colspan=1>hominis</td></tr><tr><td rowspan=1 colspan=1>Entamoeba</td><td rowspan=1 colspan=1>histolytica</td></tr><tr><td rowspan=1 colspan=1>Candida</td><td rowspan=1 colspan=1>albicans</td></tr></table>

The EP test correctly identified the expected analyte for all replicates of each organism combination, demonstrating that high concentrations of non-targeted organisms did not interfere with the ability of the EP test to detect Norovirus and Rotavirus.

# l. Assay cut-off:

The presence or absence of each EP target analyte is determined by the mean intensity of target capture spots relative to the Signal Detection Threshold. The capture, mediator, and PCR primer oligonucleotides in the EP test are designed to eliminate sequencerelated cross-reactivity, resulting in non-specific target signal intensities at capture spots that are similar to the microarray background signal. In contrast, target amplicon hybridization to complementary capture and mediator probes are expected to give signals that are well-separated from negative capture spots. When reading a test slide, multiple images of each array are taken at increasing exposures times and the final target group mean intensity value for an analyte is assigned at the shortest exposure at which the intensity exceeds the Signal Detection Threshold. If none of the target signal exceeds the threshold for any exposure, the mean spot intensity is evaluated at the longest exposure taken. With this imaging and analysis design, a signal detection threshold of 30,000 was established to generate a “Detected” call for the eight bacterial target spot groups, four virus target spot groups, and two controls of the EP test.

In order to demonstrate the appropriateness of the cut-off value for the selected threshold, the target mean intensity values observed with the EP test were examined for the final LoD confirmatory tests for the bacterial and viral targets. Data from 16 bacterial strains, four viral strains, and two negative samples were evaluated. With replicates of 20 for each of 22 samples and 14 unique target spot groups evaluated per test, a total of 6160 data points (1320 expected positive) were assessed in the study.

A logistic fit analysis of Expected Results by Target Mean Intensity for the chosen threshold demonstrated that expected positive signals are well separated from the expected negative target signal and therefore the chosen threshold value distinguishes the “True Positives” from the “True Negatives”. The selected threshold was then validated in the pivotal clinical study.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to Clinical Study Section of this document.   
b. Matrix comparison: Not applicable

3. Clinical study

Refer to K140083 for additional details regarding Clinical Study results for previously cleared EP bacterial analytes.

The clinical evaluation of the EP test for Norovirus and Rotavirus was performed concurrently with the clinical evaluation of EP bacterial targets for the initial clearance of the EP test for bacterial analytes (K140083). Due to insufficient positive specimens for Norovirus and Rotavirus, the viral targets were not included for the initial clearance of the EP test and therefore the original clinical study was extended at one clinical site in

order to obtain additional performance data for these viral analytes. For this submission the Norovirus and Rotavirus results from the original clinical study were “unmasked” and the performance data provided below includes study results from the original study combined with results from the additional testing.

The clinical study was conducted at eight clinical study testing sites and was conducted under IRB supervision at each testing and specimen acquisition site. De-identified CaryBlair specimens were enrolled from individuals for whom standard of care testing was ordered. A portion of each leftover residual unformed stool specimen in Cary-Blair media was obtained for testing.

The clinical study utilized the following four categories of specimens to establish performance for the EP bacterial and viral targets:

Fresh: Prospectively-collected fresh Cary-Blair specimens enrolled and tested at the study test sites; Frozen: Prospectively-collected frozen Cary-Blair specimens enrolled and tested at the study test sites; Selected: De-identified archived frozen specimens collected and stored in CaryBlair media were obtained from the specimen acquisition sites. These specimens were shipped frozen from each specimen acquisition site to the Sponsor, blinded and sent to clinical testing sites. In parallel, each specimen was tested by analytespecific PCR amplification and bi-directional sequencing (BDS) to confirm the original result. If the original analyte identification could not be confirmed, the specimen was excluded from the study dataset. · Contrived specimens (for selected bacterial analytes)

Reference/Comparator methods for Bacterial Analytes: Each specimen was cultured and suspected colonies were isolated and identified using an FDA Cleared Automated System. All samples were enriched in MacConkey Broth for suspected Shiga toxin producing organisms followed by EHEC EIA testing. In addition, bi-directional sequencing (BDS) assays were developed to confirm the identity of targeted analytes. Overall, these BDS systems were used in the EP test Clinical Study in various capacities: (i) for species-level identification of clinical positives, if applicable, (ii) typing of virulence markers (Stx1, Stx2), and (iii) discordant analysis between the EP test and the reference method identification. An overview of comparator methods performed for the bacterial analytes are listed in the following table:

Comparator Methods for Bacterial Analytes   

<table><tr><td>EP Analyte</td><td>Comparator Methods</td></tr><tr><td>Campylobacter</td><td>Bacterial Culture and Automated Phenotypic Identification using FDA Cleared</td></tr><tr><td>Salmonella</td><td>Methods</td></tr><tr><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Vibrio</td></tr><tr><td colspan="1" rowspan="1">Y. enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Stx1</td><td colspan="1" rowspan="1">MacConkey Broth Enrichment followed by EHEC EIA and PCR amplification//bi-directional sequencing for Confirmation and Typing</td></tr><tr><td colspan="1" rowspan="1">Stx2</td><td colspan="1" rowspan="1">MacConkey Broth Enrichment followed by EHEC EIA and PCR amplification//bi-directional sequencing for Confirmation and Typing</td></tr></table>

Comparator Methods for Viral Analytes: For both the Norovirus and Rotavirus targets, the comparator methods used for prospectively collected fresh and frozen specimens were a composite of one real-time PCR and two conventional PCR assays. Positive conventional PCR tests were confirmed with bi-directional sequencing. Specimens that were positive by real-time PCR and negative by the first conventional PCR assay were tested with a second conventional PCR assay and bi-directional sequencing, if applicable. Specimens were characterized as negative when both the real-time RT-PCR assay and the first conventional PCR assay were negative. The algorithm used to determine the composite comparator result is shown in the following table:

Composite Test Algorithm for Norovirus and Rotavirus   

<table><tr><td rowspan=1 colspan=1>Real-Time RT-PCRResult</td><td rowspan=1 colspan=1>Conventional PCR1w/BDS Result</td><td rowspan=1 colspan=1>Conventional PCR2w/BDS Result</td><td rowspan=1 colspan=1>Final CompositeComparator Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td></tr></table>

The Norovirus comparator assays included three well-established, validated and published assays from the CDC CalciNet Laboratory: a real-time PCR assay and two conventional PCR assays. These assays were also used to quantitate and characterize all Norovirus GI and GII strains used for LoD and analytical reactivity testing. Validation of the three assays was performed by CDC and the acceptability of these assays for use in the Norovirus composite comparator algorithm was supported by literature references provided by the sponsor.

For the Rotavirus comparator assays, primers for the real-time PCR and one of the conventional PCRs were obtained from published protocols. The second conventional PCR was developed and validated by Nanosphere. None of the primers used for each of three reference assays overlap with the EP test primers. Validation of the three comparator Rotavirus assays included evaluation of each assay’s LoD and as well as inclusivity testing.

# Clinical Study Results

The clinical study included a total of 1940 evaluable specimens comprised of 1328 prospectively collected fresh and frozen specimens, 203 selected archived specimens and 409 contrived specimens.

Following the clearance of K140083, confirmatory PCR sequencing results were received for an additional nine selected archived specimens positive for EP bacterial analytes that were not included in the original submission. These specimens included one specimen for $S t x I / S t x 2$ , two specimens positive for Shigella spp., and six specimens positive for Salmonella spp. For this submission, EP test results for these nine specimens have been added to the valid clinical dataset evaluated for K140083.

A total of 2102 specimens were tested with the EP test at the eight study sites. Excluded specimens included 60 fresh prospectively-collected, 73 selected specimens, and three simulated specimens. Of the remaining 1966 specimens, 26 specimens had a final No Call resulting in 26 indeterminate specimens. The following tables show the number of clinical specimens, arranged by site, determined to be valid, indeterminate, and excluded and additional information regarding the reasons for exclusion of 136 specimens.

Breakdown of Clinical Specimens by Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>EnrolledSpecimens</td><td rowspan=1 colspan=1>ExcludedSpecimens</td><td rowspan=1 colspan=1>IndeterminateSpecimens</td><td rowspan=1 colspan=1>Valid Specimens</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>114</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>858</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>839</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>380</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>351</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>348</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>85</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2102</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1940</td></tr></table>

Non-evaluable Clinical Specimens and Reasons for Exclusion from Dataset   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>No.Excluded</td><td rowspan=1 colspan=1>Time InclusionCriteria Not Met</td><td rowspan=1 colspan=1>OperatorError</td><td rowspan=1 colspan=1>ShippingIssue</td><td rowspan=1 colspan=1>InvalidQC</td><td rowspan=1 colspan=1>No/InconclusiveReference Result</td><td rowspan=1 colspan=1>InvalidSpecimens</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>14</td></tr></table>

The initial No Call rate for the clinical study was $4 \%$ (78/1940 specimens). Of the 78 initial “No Calls,” 52 yielded a valid test result upon retesting for a final No-call rate of $1 . 3 \%$ (26/1946). Additionally, there were an initial 16 specimens that gave Pre-AE results and three specimens that gave Pre-AE results upon repeat testing, for a total of 19 Pre-AE occurrences in the Method Comparison study (19/2063 tests ru $1 1 = 0 . 9 \%$ pre-AE rate). Combining the No Call and Pre-AE results showed an initial invalid test rate of $4 . 9 \%$ (104/1940) and a final invalid test rate of $1 . 3 \%$ (26/1940). The 26 specimens yielding a final No Call result were not included in the valid dataset.

The following table provides a summary of patient age for prospectively collected specimens (patient age was unknown for 15 specimens):

Valid Specimens Stratified by Patient Age $\scriptstyle ( \mathbf { n } = 1 3 1 3 )$ )   

<table><tr><td rowspan=1 colspan=1>Age Range</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>4.8%</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=1>&gt;12-21</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>11.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>636</td><td rowspan=1 colspan=1>48.4%</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>25.4%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1313</td><td rowspan=1 colspan=1>100%</td></tr></table>

The following clinical performance tables provide a summary of the clinical performance of the EP test for the detection of the five bacterial targets, the Shiga toxin 1 gene and Shiga toxin 2 gene targets, and the Norovirus GI/GII and Rotavirus A targets.

Summary of Bacterial Target Clinical Test Performance $\scriptstyle ( \mathbf { n } = \mathbf { 1 } 9 4 \mathbf { 0 } )$ - Compared to Reference Methods (Culture and Conventional Biochemical and Automated Phenotypic Identification   

<table><tr><td rowspan=6 colspan=1>de aecqe.</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cii Cican</td><td rowspan=2 colspan=1>Pdpocc</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>90.9%20/22 a(79.8-98.9)</td><td rowspan=1 colspan=1>98.7%1255/1272 d,q(97.9-99.2)</td><td rowspan=3 colspan=1>Ss cp</td><td rowspan=3 colspan=1>Cii Cican</td><td rowspan=2 colspan=1> </td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>86.4%19/22(65.1-97.1)</td><td rowspan=1 colspan=1>99.4%1265/1272h(98.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%2/2(15.8-100)</td><td rowspan=1 colspan=1>100%32/32(89.1-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>97.0%32/33(84.2-99.9)</td></tr><tr><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>97.5%39/40c(86.8-99.9)</td><td rowspan=1 colspan=1>99.4%162/163(96.6-100)</td><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>98.3%58/59(90.9-100)</td><td rowspan=1 colspan=1>99.3%143/144j(96.2-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>98.5%67/68b(92.1-100)</td><td rowspan=1 colspan=1>100%341/341(98.9-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%67/67(94.6-100)</td><td rowspan=1 colspan=1>100%342/342(98.9-100)</td></tr><tr><td rowspan=2 colspan=1>dds ks</td><td rowspan=2 colspan=1>SeecmnCiic</td><td rowspan=2 colspan=1>Ppoe</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>66.7%2/3k,r(9.4-99.2)</td><td rowspan=1 colspan=1>98.8%1275/1291(98.0-99.3)</td><td rowspan=2 colspan=1>dds o</td><td rowspan=2 colspan=1>SeecmnsCii.</td><td rowspan=2 colspan=1>Prpoecce</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%1293/1293(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>97.1%33/34 m(84.7-99.9)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%33/33(89.4-100)</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100%8/8(63.1-100)</td><td rowspan=1 colspan=1>99.5%194/195n(97.2-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%202/202(98.2-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%50/50(92.9-100)</td><td rowspan=1 colspan=1>100%359/359(99.0-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>91.1%51/56°(80.4-97.0)</td><td rowspan=1 colspan=1>99.7%352/353P(98.4-100)</td></tr><tr><td rowspan=4 colspan=1>R</td><td rowspan=3 colspan=1>Cts Cinn</td><td rowspan=2 colspan=1>Prepoct</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%1294/1294(99.7-100)</td><td rowspan=4 colspan=6></td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td></tr><tr><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100%1/1(2.5-100)</td><td rowspan=1 colspan=1>100%202/202(98.2-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%59/59(93.9-100)</td><td rowspan=1 colspan=1>100%350/350(99.0-100)</td></tr></table>

# Footnoted Information for Previous Table

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>oted</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0r ITevlous Iar</td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Type_</td><td rowspan=1 colspan=1>EP Test Result</td><td rowspan=1 colspan=1>Reference Method Result(s)</td><td rowspan=1 colspan=2>PCR Amp/BDS Results</td></tr><tr><td rowspan=2 colspan=1>a.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C. jejuni subsp. jejuni</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C&amp;.</td><td rowspan=1 colspan=1>Negative for Campylobacter spp</td><td rowspan=1 colspan=1>E</td></tr><tr><td rowspan=1 colspan=1>b.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>C. lari</td><td rowspan=1 colspan=1>Positive for Campylobacter Tari</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>c.</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Select</td><td rowspan=2 colspan=1>Not Detected</td><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=2 colspan=1>upon repeat))</td><td rowspan=1 colspan=1>cter jejuni (at LoD; Negative</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=17 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter coli</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>FFresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>M. morganii subsp. morganii &amp; N. cinerea</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Fesh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negave</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>M. morganii subsp. morganii</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>C.braakii &amp; E.cloace subsp dissolvens&amp;N.cinerea</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter spp.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter spp.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Fesh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Negative_</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Positive for Campylobacter coli</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>M.morganii subsp. morganii</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Campylobacter andSalmonella</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni and Salmonella enterica</td><td rowspan=1 colspan=1>Positive for Campylobacter jejuni and Salmonella enterica</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FFresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td><td rowspan=1 colspan=1>-------------</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>FFresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>g.</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Select</td><td rowspan=2 colspan=1>Not Detected</td><td rowspan=2 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Low-Level Positive for Salmonell</td><td rowspan=2 colspan=1>enterica (at LD; Negative</td></tr><tr><td rowspan=1 colspan=1>upon repeat)</td></tr><tr><td rowspan=4 colspan=1>h.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for Salmonella spp.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td><td rowspan=1 colspan=1>------------</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Postive for Salmonella enterica</td><td rowspan=1 colspan=1> - - - - - - - - - - - - -</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>C.freundi&amp;Proteus spp</td><td rowspan=1 colspan=1>Negative for Salmonella spp</td><td rowspan=1 colspan=1>-------------</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Salmonella enterica</td><td rowspan=1 colspan=1> - - - - - - - - - - - - -</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>P. alcalifaciens_</td><td rowspan=1 colspan=1>Negative for Salmonella spp</td><td rowspan=1 colspan=1>-------------</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Ngative</td><td rowspan=1 colspan=2>Positive for Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=2>Positive or Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=2>Positive for Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=2>Positive for Shigella/EIEC</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N Negative</td><td rowspan=1 colspan=2>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fesh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N Ngative</td><td rowspan=1 colspan=2>Positive for Shigella/IEC</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N Ngative</td><td rowspan=1 colspan=2>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N Ngative</td><td rowspan=1 colspan=2>Positive for Shigella/IEC</td></tr><tr><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>NNgative</td><td rowspan=1 colspan=2>Positive for Shigella/EIEC</td></tr></table>

<table><tr><td rowspan=10 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2> Shhigella</td><td rowspan=1 colspan=1>A.hydrophila/caviae &amp; P. putida</td><td rowspan=1 colspan=1>Positive for Shigella/IEC</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=1>N Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1> Shhigella</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SShhigella</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>SShhigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2>Sigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2>SShhigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella spp</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=3>Fresh</td><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>m.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Frozen</td><td rowspan=1 colspan=2>Sigella</td><td rowspan=1 colspan=1>P.retgeri</td><td rowspan=1 colspan=1>Positive for Shigella/EIEC</td></tr><tr><td rowspan=1 colspan=1>n.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Select</td><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=1>Shigatoxin</td><td rowspan=1 colspan=1>Positive for Shiga toxin 1</td></tr><tr><td rowspan=5 colspan=1>o.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Simulated (2X)</td><td rowspan=1 colspan=2>Not Detected</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Positive for V. parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=3>Simulated (2X)</td><td rowspan=1 colspan=2>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Positive for V. cholera</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=3>Simulated (40X)</td><td rowspan=1 colspan=2>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Negative for V. cholera</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=3>Simulated (40X)</td><td rowspan=1 colspan=2>Not Detected</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Negative for V. parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=3>Simulated (40X)</td><td rowspan=1 colspan=2>Not Detected</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>Positive for V. cholerae</td></tr><tr><td rowspan=1 colspan=1>p.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Simulated (30X)</td><td rowspan=1 colspan=2>Campylobacter andVibrio</td><td rowspan=1 colspan=1>! Campylobacter lari</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=1 colspan=1>q.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7> .</td></tr><tr><td rowspan=1 colspan=1>r.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7> .</td></tr></table>

Summary of Clinical Test Performance $\scriptstyle \mathbf { n } = \mathbf { 1 9 4 0 }$ - Compared to Broth Enrichment/EHEC EIA). Stx Combined   

<table><tr><td rowspan=8 colspan=3>Footnoted Information</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=2></td><td rowspan=3 colspan=3>Specimen Type</td><td rowspan=3 colspan=2>n</td><td rowspan=2 colspan=3>% Agreement (95%CI</td><td></td></tr><tr><td rowspan=6 colspan=1>for Previous Table</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Negative</td></tr><tr><td rowspan=4 colspan=2>Suqn xm</td><td rowspan=3 colspan=1>Cite Ccenn</td><td rowspan=2 colspan=1>Prpocecc</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=2>1294</td><td rowspan=1 colspan=1>100%7/7(59.0-100)</td><td rowspan=1 colspan=2>99.7%1283/1287a, e(99.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=2>34</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>100%34/34(89.7-100)</td></tr><tr><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=2>203</td><td rowspan=1 colspan=1>100%13/13(75.3-100)</td><td rowspan=1 colspan=2>99.5%189/190 b(97.1-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=2>409</td><td rowspan=1 colspan=1>98.2%107/109d(93.5-99.8</td><td rowspan=1 colspan=2>98.7%296/300 c, f(96.6-99.6)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=2>Fresh, Frozen orSimulated (xLoD)</td><td rowspan=1 colspan=5>Identified by EP test as:</td><td rowspan=1 colspan=3>Identified by ReferenceMethod(s) as:</td><td rowspan=1 colspan=2>PCR Amp/BD Sequencing Results (if applicable)</td></tr><tr><td rowspan=3 colspan=1>a.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=5>Shiga Toxin 1</td><td rowspan=1 colspan=3>Negative</td><td rowspan=1 colspan=2>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=5>Shiga Toxin 1 and Norovirus</td><td rowspan=1 colspan=3>Escherichia coli</td><td rowspan=1 colspan=2>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=5>Shiga Toxin 1 and Shiga Toxin 2</td><td rowspan=1 colspan=3>Citrobacter youngae</td><td rowspan=1 colspan=2>Positive for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>b.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Select</td><td rowspan=1 colspan=5>Shiga Toxin 1 and Campylobacter</td><td rowspan=1 colspan=3>Campylobacter</td><td rowspan=1 colspan=2>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>Simulated (13X)</td><td rowspan=2 colspan=5>Shiga Toxin 1and Salmonella</td><td rowspan=2 colspan=3>ISalmonella enterica subsp.enterica</td><td rowspan=2 colspan=2>Negative for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1>c.</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>Simulated (30X)</td><td rowspan=1 colspan=5>Shiga Toxin 1and Campylobacter</td><td rowspan=1 colspan=3>Campylocereusjejuni</td><td rowspan=1 colspan=2>Not performed</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>Simulated (2X)</td><td rowspan=1 colspan=5>Shiga Toxin 1, Shiga Toxin 2 andYersinia enterocolitica</td><td rowspan=1 colspan=3>Yersinia enterocolitica</td><td rowspan=1 colspan=2>Positive for Stx 1 gene &amp; Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1>d.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Simulated (30X)</td><td rowspan=1 colspan=5>Not Detected</td><td rowspan=1 colspan=3>Escherichia coli, Shiga Toxin 2</td><td rowspan=1 colspan=2>Positive for Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>Simulated (2X)</td><td rowspan=1 colspan=5>Not Detected</td><td rowspan=1 colspan=3>Escherichia coli, Shiga Toxin 2</td><td rowspan=1 colspan=2>Positive for Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1>e.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=5>Shiga Toxin 2</td><td rowspan=1 colspan=3>Negative</td><td rowspan=1 colspan=2>Positive for Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>Fresh</td><td rowspan=1 colspan=5>Shiga Toxin 1 and Shiga Toxin 2</td><td rowspan=1 colspan=3>Citrobacter youngae</td><td rowspan=1 colspan=2>Positive for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Simulated (31X)</td><td rowspan=1 colspan=5>Shiga Toxin 2 and Yersiniaenterocolitica</td><td rowspan=1 colspan=3>Yersinia enterocolitica</td><td rowspan=1 colspan=2>Negative for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>Simulated (2X)</td><td rowspan=1 colspan=5>Shiga Toxin 1, Shiga Toxin 2 andYersinia enterocolitica</td><td rowspan=1 colspan=3>IYersinia enterocolitica</td><td rowspan=1 colspan=2>Positive for Stx 1 gene &amp; Stx 2 gene</td></tr></table>

Summary of Clinical Test Performance $\scriptstyle \mathbf { n } = \mathbf { 1 9 4 0 }$ - Compared to Broth Enrichment/EHEC EIA and Stx 1 and Stx 2 typing).   

<table><tr><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>A</td><td rowspan=3 colspan=1>Cit Cen</td><td rowspan=2 colspan=1>PrdCoect</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>100%4/4(39.8-100)</td><td rowspan=1 colspan=1>99.8%1287/1290 a(99.3-100</td><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1>Cii Cicn</td><td rowspan=2 colspan=1>Pedpoec</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1294</td><td rowspan=1 colspan=1>100%6/6(54.1-100)</td><td rowspan=1 colspan=1>99.8%1286/1288(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%34/34(89.7-100)</td></tr><tr><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100%9/9(66.4-100)</td><td rowspan=1 colspan=1>99.5%193/194b(97.2-100)</td><td rowspan=1 colspan=2>Selected</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100%10/10(69.2-100)</td><td rowspan=1 colspan=1>100%193/193(98.1-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>100%50/50(92.9-100)</td><td rowspan=1 colspan=1>99.2%356/359c(97.6-99.8)</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>96.6%57/59d(88.3-99.6)</td><td rowspan=1 colspan=1>99.4%348/350f(98.0-99.9)</td></tr></table>

# Footnoted Information for Previous Table

<table><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Fresh, Frozen orSimulated (xLoD)</td><td rowspan=1 colspan=1>Identified by EP test as:</td><td rowspan=1 colspan=1>Identified byReference/ComparatorMethod(s) as:</td><td rowspan=1 colspan=1>PCR Amp/BD Sequencing Results (if applicable)</td></tr><tr><td rowspan=3 colspan=1>a.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 and Norovirus</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 and Shiga Toxin 2</td><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>Positive for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>Shiga Toxin 1 and Campylobacter</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Positive for Stx 1 gene</td></tr><tr><td rowspan=3 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Simulated (13X)</td><td rowspan=1 colspan=1>Shiga Toxin 1and Salmonella</td><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica</td><td rowspan=1 colspan=1>Negative for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Simulated (30X)</td><td rowspan=1 colspan=1>Shiga Toxin 1 and Campylobacter</td><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>Not performed</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Shiga Toxin 1, Shiga Toxin 2 andYersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Positive for Stx 1 gene &amp; Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1>d.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Simulated (30X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Escherichia coli, Shiga Toxin 2</td><td rowspan=1 colspan=1>Positive for Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Escherichia coli, Shiga Toxin 2</td><td rowspan=1 colspan=1>Positive for Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1>e.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive for Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Shiga Toxin 1 and Shiga Toxin 2</td><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>Positive for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Simulated (31X)</td><td rowspan=1 colspan=1>Shiga Toxin 2 and Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Negative for Stx 1 gene and Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Simulated (2X)</td><td rowspan=1 colspan=1>Shiga Toxin 1, Shiga Toxin 2 andYersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Positive for Stx 1 gene &amp; Stx 2 gene</td></tr></table>

Summary of Viral Target Clinical Test Performance $\scriptstyle ( \mathbf { n } = \mathbf { 1 9 4 0 } )$ ) - Compared to Composite Comparator Method (real-time RT-PCR and two conventional PCRs with bi-directional sequencing)   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="3" rowspan="2">Specimen Type</td><td colspan="1" rowspan="2">n</td><td colspan="2" rowspan="1">% Agreement (95% CI)</td><td colspan="1" rowspan="2">−</td><td colspan="3" rowspan="2">Specimen Type</td><td colspan="1" rowspan="2">n</td><td colspan="2" rowspan="1">% Agreement (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="3">II CI</td><td colspan="1" rowspan="3">Cit Cican</td><td colspan="1" rowspan="2">Pdpoce</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1294</td><td colspan="1" rowspan="1">94.9%37/39a(82.7-99.4)</td><td colspan="1" rowspan="1">99.6%1250/1255d(99.0-99.9)</td><td colspan="1" rowspan="3">V RtnS</td><td colspan="1" rowspan="3">Ci Cicann</td><td colspan="1" rowspan="2">Prdpoec</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">66.7%2/3(9.4-99.2)</td><td colspan="1" rowspan="1">99.9%1290/1291g(99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0%0/1 b(0.0-97.5)</td><td colspan="1" rowspan="1">100%33/33(89.7-100)</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%34/34(89.7-100)</td></tr><tr><td colspan="2" rowspan="1">Selected</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">100%18/18(81.5-100)</td><td colspan="1" rowspan="1">99.5%184/185c(97.0-100)</td><td colspan="2" rowspan="1">Selected</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">98.0%50/51f(89.6-100)</td><td colspan="1" rowspan="1">100%152/152(97.6-100)</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">409</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%409/409(99.1-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">409</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%409/409(99.1-100)</td></tr></table>

Footnoted Information for Previous Table   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>No.</td><td rowspan=2 colspan=1>SID</td><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>EP test result</td><td rowspan=2 colspan=1>Composite Comparator MethodResult*:</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0611271</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0611381</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>065602</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Norovirus GI</td></tr><tr><td rowspan=1 colspan=1>c</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>973241</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Rotavirus, Norovirus</td><td rowspan=1 colspan=1>Rotavirus,</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>052401</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0610451</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>d</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0611191</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>066181</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>069051</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>NorovirusShiga Toxin 1,</td><td rowspan=1 colspan=1>Stx1</td></tr><tr><td rowspan=1 colspan=1>e</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>067641</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Rotavirus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>970381</td><td rowspan=1 colspan=1>Selected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Rotavirus</td></tr><tr><td rowspan=1 colspan=1>g</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>067631</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Negative</td></tr></table>

\*Composite Comparator method positive for Norovirus and Rotavirus: RT- PCR (endpoint PCR) with subsequent bi-directional sequencing must be positive.

Summary of Genus/Group-level Test Performance versus Reference Method(s) – Stratified by Species.   

<table><tr><td colspan="5" rowspan="1">Campylobacter Genus</td><td colspan="5" rowspan="1">Shigella Genus</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ProspectiveFresh/Frozen</td><td colspan="1" rowspan="1">Selected</td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">Analytical</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ProspectiveFresh/Frozen</td><td colspan="1" rowspan="1">Selected</td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">Analyticai</td></tr><tr><td colspan="1" rowspan="1">CombinedCampylobacter</td><td colspan="1" rowspan="1">91.7%22/24(72.0-99.0)</td><td colspan="1" rowspan="1">97.5%39/40(86.8-99.9)</td><td colspan="1" rowspan="1">98.5%67/68(86.3-100)</td><td colspan="1" rowspan="1">100%15/15(78.2-100)</td><td colspan="1" rowspan="1">CombinedShigella</td><td colspan="1" rowspan="1">66.7%2/3(9.4-99.2)</td><td colspan="1" rowspan="1">100%8/8(54.1-100)</td><td colspan="1" rowspan="1">100%50/50(92.9-100)</td><td colspan="1" rowspan="1">100%20/20(83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobactercoli</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%18/18(81.5-100)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%14/14(76.8-100)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobacterjejuni</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">97.3%36/37(85.8-99.9)</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">100%4/4(40.0-100)</td><td colspan="1" rowspan="1">Shigelladysenteriae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%9/9(66.4-100)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobacterjejuni subsp.doylei</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">Shigellaflexneri</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%16/16(79.4-100)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobacterjejuni subsp.jejuni</td><td colspan="1" rowspan="1">90.0%18/20(68.3-99.8)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%21/21(83.9-100)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%11/11(71.5-100)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobacterlari</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">93.3%14/15(68.1-99.8)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td><td colspan="1" rowspan="1">Shigella sppnot identified</td><td colspan="1" rowspan="1">0%0/1(0-97.5)</td><td colspan="1" rowspan="1">100%6/6(54.1-100)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Salmonella Genus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Vibrio Genus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ProspectiveFresh/Frozen</td><td colspan="1" rowspan="1">Selected</td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">Analytical</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ProspectiveFresh/Frozen</td><td colspan="1" rowspan="1">Selected</td><td colspan="1" rowspan="1">Simulated</td><td colspan="1" rowspan="1">Analyticai</td></tr><tr><td colspan="1" rowspan="1">CombinedSalmonella</td><td colspan="1" rowspan="1">87.0%20/23(66.4-97.2)</td><td colspan="1" rowspan="1">98.3%58/59(90.9-100)</td><td colspan="1" rowspan="1">100%67/67(94.6-100)</td><td colspan="1" rowspan="1">100%31/31(88.8-100)</td><td colspan="1" rowspan="1">CombinedVibrio</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">91.1%51/56(80.4-97.0))</td><td colspan="1" rowspan="1">100%10/10(69.2-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonella"non-typhi"</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">84.2%16/19(60.4-96.6)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellabongori</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">Vibrioparahaemolyticus</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">94.6%35/37(81.8-99.3)</td><td colspan="1" rowspan="1">100%5/5(47.8-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">98.3%58/59(90.9-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td><td colspan="1" rowspan="1">N/A</td><td colspan="5" rowspan="8"></td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.arizonae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.diarizonae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.enterica</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%52/52(93.2-100)</td><td colspan="1" rowspan="1">100%25/25(86.3-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.houtenae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.indica</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%3/3(29.2-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica subsp.salamae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%2/2(15.8-100)</td><td colspan="1" rowspan="1">100%1/1(2.5-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonella sppnot identified</td><td colspan="1" rowspan="1">87.0%20/23(66.4-97.2)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

The following two tables list the distinct mixed specimen combinations observed during the clinical study. In summary, there were a total of 26 mixed specimens that were detected by either the EP test or the reference methods or by both. All mixed specimens that were detected by the reference methods were detected by the EP test.

Clinical Mixed Specimen Combinations Detected by EP   

<table><tr><td rowspan=1 colspan=3>Multiple Target Combinations Detected by EP</td><td rowspan=1 colspan=3>Reference/Comparator Test</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>DiscrepantSpecimens</td><td rowspan=1 colspan=1>DiscrepantIdentification</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 gene, Stx 2gene</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Stx 1 gene</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=2 colspan=1>Salmonella</td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Campylobacter</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Vibrio</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 2 gene</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 geneStx 2 gene</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Shiga Toxin 1</td><td rowspan=2 colspan=1>Norovirus</td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stx 1 gene</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus</td></tr><tr><td rowspan=1 colspan=2>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1></td></tr></table>

Clinical Mixed Specimen Combinations Detected by Reference/Comparator Methods   

<table><tr><td rowspan=1 colspan=3>Multiple Target Combinations by Reference/Comparator Test</td><td rowspan=1 colspan=3>Detected by EP</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>DiscrepantSpecimens</td><td rowspan=1 colspan=1>DiscrepantTargets</td></tr><tr><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

External Quality Control Testing: The clinical study protocol specified running QC samples each day of testing, utilizing one external negative control and one of eleven external positive controls (tested on a rotating basis) representing all target analytes. The following table summarizes the positive and negative control test runs performed during the clinical study.

Summary of QC Performance during the Clinical Study   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Call Rate</td><td rowspan=1 colspan=1>Mis-calls</td><td rowspan=1 colspan=1>Accuracy</td><td rowspan=1 colspan=1>Pre-AE</td><td rowspan=1 colspan=1>Pre-AERate</td></tr><tr><td rowspan=1 colspan=1>N01</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>254</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>P01</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P02</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P03</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P04</td><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P05</td><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P06</td><td rowspan=1 colspan=1>Shigella + ShigaToxin 1</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>P07</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P08</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>93.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P09</td><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P10</td><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>P11</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>509</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.6%</td></tr></table>

4. Clinical cut-off:

N/A

5. Expected values/Reference range:

In the Verigene EP Methods Comparison study, 1328 prospectively collected fresh and frozen specimens were obtained from multiple medium to large-sized healthcare institutions geographically distributed across the United States. The number and percentage of positive cases (positivity rate) determined by Verigene EP stratified by geographic region for each of the organisms detected by the test are presented in the table below. Overall, EP detected at least one target in $1 1 . 2 \%$ (149/1328) of prospectively collected specimens. In routine practice, prevalence rates may vary depending on the institution, geographical location, and patient population.

Prevalence of Organisms Detected by Verigene EP – Clinical Study Observations   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>US Geographic Region/Division*</td><td rowspan=4 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=1>Region</td><td rowspan=1 colspan=2>Midwest</td><td rowspan=1 colspan=1>South</td><td rowspan=1 colspan=1>Northeast</td><td rowspan=1 colspan=1>West</td></tr><tr><td rowspan=1 colspan=1>Division</td><td rowspan=1 colspan=1>West NorthCentral</td><td rowspan=1 colspan=1>East NorthCentral</td><td rowspan=1 colspan=1>W. SouthCentral</td><td rowspan=1 colspan=1>MiddleAtlantic</td><td rowspan=1 colspan=1>Pacific</td></tr><tr><td rowspan=1 colspan=1>State</td><td rowspan=1 colspan=1>MO</td><td rowspan=1 colspan=1>WI</td><td rowspan=1 colspan=1>TX</td><td rowspan=1 colspan=1>NY</td><td rowspan=1 colspan=1>CA</td></tr><tr><td rowspan=1 colspan=1>Total n</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>196</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>771</td><td rowspan=1 colspan=1>1328</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Stx1</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Stx2</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>POS n</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Prev</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td></tr></table>

# N. Instrument Name:

Verigene® System

# O. System Descriptions:

1. Modes of Operation:

The Verigene® System has one mode of operation available to the User. Prior to initiating a test run the User places the sample into the sample loading well, loads the consumable, and enters identifying information into the system. After the Processor SP run has completed, the test cartridge is removed from the instrument and the reagent pack is manually separated from the Substrate Holder. The barcode on the substrate holder is scanned prior insertion into the Reader. The Reader images the Substrate and automatically determines the test result without further user intervention.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X_ or No

3. Specimen Identification:

All tests must be ordered through the Verigene Reader. No tests can be processed on the Verigene Processor $S P$ without the user entering the Test Cartridge ID and Sample ID into the Verigene Reader.

The User enters the Test Cartridge ID by scanning the barcode using the barcode scanner attached to the Reader. The user may manually enter in the Test Cartridge ID by selecting MENU and ‘Enter Barcode’ and then keying in the Test Cartridge ID number with the Reader’s keyboard. The User has the option to scan the Test Cartridge Cover’s 2D barcode using a barcode gun-style scanner to display the Test Cartridge’s Reference Number, Expiration Date, and Lot Number on reports. The User enters the Sample ID by scanning or manually entering the Sample ID using the Reader’s touch-screen keyboard then confirming the Sample ID in the software.

# 4. Specimen Sampling and Handling:

The user is instructed to perform the following steps after which all specimen handling is performed by the automated Verigene System:

· Put on fresh gloves. For each Cary-Blair preserved specimen to be tested, place one sterile flocked swab and one uncapped Stool Prep Buffer tube (place the cap to the side for recapping later) into a biological safety cabinet (BSC).   
· Wipe down the outside of the specimen vial with a lint-free decontaminating wipe.   
· Invert the vial containing the Cary-Blair preserved specimen twice and vortex the specimen for 5-10 seconds to ensure homogeneity. To prepare the Stool Prep Buffer tube, dip the provided flocked swab into either the primary Cary-Blair preserved specimen vial or the secondary tube until the flocked tip is fully immersed in specimen. Once evenly coated, transfer the swab to the Stool Prep Buffer tube and break swab at the pre-formed scored breakpoint. Leave the swab in the Stool Prep Buffer tube and screw the cap finger tight on to Stool Prep Buffer tube.   
· Recap the original, primary Cary-Blair preserved specimen container and set aside. Repeat steps 1-6 for each specimen, changing gloves between each specimen.   
Vortex each Stool Prep Buffer tube for 15-20 seconds.

Spin all prepared Stool Prep Buffer tubes in the Mini Centrifuge for 30-35 seconds at a MAXIMUM of 2200 rcf. Pipette $2 0 0 \mu \mathrm { L }$ of the prepared supernatant into the Extraction Tray.

5. Calibration: There is no calibration required by the user.

6. Quality Control: See Section M.1c above.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.