# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K142156 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2).

# D. Type of Test:

An in vitro molecular diagnostic test for the qualitative detection and differentiation of HSV1 and HSV-2 DNA from female skin lesions from anogenital sites.

# E. Applicant:

AnyplexTM HSV-1/2 Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Anyplex™ II HSV-1/2 Assay is a real-time polymerase chain reaction (PCR)-based in vitro diagnostic test intended for the qualitative detection and differentiation of Herpes Simplex Virus Type-1 (HSV-1) and Herpes Simplex Virus Type-2 (HSV-2) DNA from

female skin lesions from anogenital sites. The test is intended for use as an aid in the diagnosis of anogenital HSV infection in symptomatic patients.

Warning: Anyplex™ II HSV-1/2 Assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay is not intended to be used for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Cepheid SmartCycler® II DX System (software version 3.0b).

# I. Device Description:

The Anyplex $\mathbf { I M }$ II HSV-1/2 Assay uses PCR to generate amplified product from HSV-1 and HSV-2 DNA present in clinical specimens. The presence of HSV-1 and/or HSV-2 target DNA is indicated by the fluorescent signal generated through the use of fluorescently-labeled oligonucleotide probes (duplex Catcher) on the Cepheid SmartCycler® II Dx instrument. The probes do not generate a signal unless they are specifically bound to the amplified product. A preparation of HSV-1 and HSV-2 plasmids is included as the positive control in the Anyplex™ II HSV-1/2 Assay. The positive control serves to demonstrate that the HSV-1/2 PCR reagents are functional and to check the validity of the run. An internal control (IC) is also included in the assay kit. The IC is added to each sample specimen during sample preparation, and is also used to create a Blank Negative Control by adding a set amount to viral transport media to serve as an extraction control. In addition, RNase-free water is used to create the Negative Control by adding a set volume to the prepared master mix. Users are instructed to include all three controls, Positive, Negative, and Blank Negative Control with each test run.

The AnyplexTM II HSV-1/2 Assay detects and differentiates HSV-1 and HSV-2 genomic DNA in a single assay utilizing Seegene’s TOCE™ technology. Clinical samples are spiked with an Internal Control and DNA is extracted utilizing a standard QIAGEN QIAamp® DNA Mini Kit. Extracted DNA is then mixed with the Seegene reagents and processed in a Cepheid SmartCycler® II Dx System. In approximately 2.5 hours, results can be available, including validity and acceptance criteria for the sample.

The one reagent kit for the Anyplex $\mathbf { \Delta } ^ { \mathrm { T M } }$ II HSV-1/2 Assay is sufficient for 50 reactions. The kit contains:

• 10X HSV-1/2 TOM - consisting of TOCE Oligo Mix (TOM), amplification and detection reagents

2X AnyplexTM II PCR Master Mix (with UDG) - consisting of DNA polymerase, Uracil-DNA glcosylase, and buffer containing dNTPs • HSV-1/2 Positive Control (PC) - containing a mixture of Cloned DNA HSV-1/2 Internal Control (IC) - containing exogenous DNA RNase-free Water - Ultrapure PCR-grade quality

# J. Substantial Equivalence Information:

1. Predicate device name(s): IMDx HSV-1/2 for Abbott m2000 Reference Method: ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc.) for clinical evaluation (K971662)

2. Predicate 510(k) number(s): K140198

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Characteristic</td><td>Seegene AnyplexTM II HSV-1/2 Assay (New Device)</td><td>IMDx HSV-1/2 for Abbott m2000 (Predicate)</td></tr><tr><td>Intended use</td><td>The AnyplexTM II HSV-1/2 Assay is a real-time polymerase chain reaction (PCR)-based in vitro diagnostic test intended for the qualitative detection and differentiation of Herpes Simplex Virus Type-1 (HSV-1) and Herpes Simplex Virus Type-2 (HSV-2) DNA from female skin lesions from anogenital sites. The test is intended as an aid in the diagnosis of anogenital HSV infection in symptomatic patients. Warning: The AnyplexTM II HSV-1/2 Assay is not indicated for use with cerebrospinal fluid (CSF). The assay is not intended to be used for pre-natal screening.</td><td>The IMDx HSV-1/2 for Abbott m2000 assay is an in vitro diagnostic test for the direct, qualitative detection and differentiation of Herpes Simplex Virus Type 1 (HSV-1) and Type 2 (HSV-2) DNA from male and female skin lesions from anogenital or oral sites. The test is intended for use as an aid in the diagnosis of HSV infection in symptomatic patients. The assay is intended to be run on the Abbott m2000 instrument system. Warning: The IMDx HSV-1/2 for Abbott m2000 assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay is not intended for prenatal screening.</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Seegene Anyplex™M II HSV-1/2 Assay(New Device)</td><td colspan="1" rowspan="1">IMDx HSV-1/2 for Abbott m2000(Predicate)</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Real-time PCR DNA amplification</td><td colspan="1" rowspan="1">Real-time PCR DNA amplification</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detection and differentiationof HSV-1 and HSV-2 DNA</td><td colspan="1" rowspan="1">Qualitative detection and differentiation ofHSV-1 and HSV-2 DNA</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Seegene AnyplexTM II HSV-1/2 Assay(New Device)</td><td colspan="1" rowspan="1">IMDx HSV-1/2 for Abbott m2000(Predicate)</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Real-time PCR amplification/detectionusing the Cepheid SmartCycler II DXsystem.</td><td colspan="1" rowspan="1">Sample extraction and real-time PCRamplification/detection using the Abbottm2000 system.</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethod</td><td colspan="1" rowspan="1">Manual extraction using the QIAGENQIAamp® DNA Mini Kit</td><td colspan="1" rowspan="1">Automated on Abbott m2000 system</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Female skin lesions from anogenital sites</td><td colspan="1" rowspan="1">Male and female skin lesions fromanogenital or oral sites</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">Double-labeled (fluorophore andquencher) duplex Catcher. Measuresincrease in assay fluorescence with eachPCR cycle.</td><td colspan="1" rowspan="1">Double-labeled (fluorophore and quencher)hydrolysis probes. Measures increase inassay fluorescence with each PCR cycle.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

The Anyplex™ II HSV-1/2 Assay is based on real-time PCR technology combined with Seegene’s TOCE™ (Tagging Oligonucleotide Cleavage and Extension) technology on the SmartCycler II Dx instrument system. A typical real-time PCR consists of denaturation of purified DNA, primer annealing and extension. The detection of target by the TOCE™ technology depends on two novel oligonucleotides, the Pitcher and the Catcher. When the PCR reaction initiates, Dual Priming Oligonucleotides $( \mathrm { D P O ^ { \mathrm { T M } } } )$ specifically hybridize to the Glycoprotein D gene of HSV-1 and HSV-2 and the portion of the Pitcher at the $_ 3 \cdot$ -end binds between the priming oligonucleotides. During PCR extension, the unbound $5 '$ portion (Tag) of the Pitcher oligonucleotide is cleaved off by Taq DNA polymerase exonuclease. Subsequently, the released Tag anneals to a dual-labeled and single-stranded artificial template, called the Catcher. Extension of the Tag on the Catcher template results in a Duplex Catcher causing the physical distance of the fluorescent reporter from the quencher to increase, thereby generating signal. Thus, fluorescent signal is proportional to the concentration of Glycoprotein D gene of HSV-1 and HSV-2 in the original specimen, and is displayed as the value of threshold cycle, Ct. The threshold cycle is the cycle number at which the fluorescence signal in the reaction crosses above the background of fluorescence.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility:

Reproducibility studies were performed at three sites using three lots of reagent kits. Each site tested one lot of the reagent kit. The test panel of seven members was prepared randomized and blinded and tested for five (5) days non-consecutive days by two operators with each operator running the panel, in triplicate, once a day for a total of 90 results per panel member. The panel was formulated using intact virus diluted in M4 Viral Transport Media. Panel members were prepared with a single target present (HSV1 or HSV-2) at three concentrations: ${ \sim } 3 \mathrm { X }$ LoD (High Positive), 1X LoD (Low Positive), and ${ \sim } 0 . 1 2 5 \mathrm { X }$ LoD (High Negative). A seventh panel member (HSV Negative) was prepared using M4 medium without HSV present. Reproducibility testing was carried out using panels that had been randomized and blinded prior to providing to the test site. The table below summarizes the results from each site as well as the combined results.

# Reproducibility

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All 3 sitesCombined</td></tr><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>%Agreementgreementwith expectedresult</td><td rowspan=1 colspan=1>Avg.Ct(%CV)</td><td rowspan=1 colspan=1>%AgreementAgrementwith expectedresult</td><td rowspan=1 colspan=1>Avg.Ct(%CV)</td><td rowspan=1 colspan=1>%AgreementAgrementwith expectedresult</td><td rowspan=1 colspan=1>Avg.Ct(%CV)</td><td rowspan=1 colspan=1>%AgreementAagrmentwith expectedreesult</td><td rowspan=1 colspan=1>Avg.Ct(%CV)</td></tr><tr><td rowspan=1 colspan=1>HSV-1Hig- ossitive*</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>36.4(2.0%)</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>35.9(2.6%)</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>36.1(2.9%)</td><td rowspan=1 colspan=1>100.0%90/90</td><td rowspan=1 colspan=1>36.1(2.6%)</td></tr><tr><td rowspan=1 colspan=1>HSV-1Low-Positive*</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>39.9(2.2%)</td><td rowspan=1 colspan=1>90.0%27/30</td><td rowspan=1 colspan=1>39.8(3.3%)</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>39.2(1.7%)</td><td rowspan=1 colspan=1>96.7%87/90</td><td rowspan=1 colspan=1>39.6(2.6%)</td></tr><tr><td rowspan=1 colspan=1>HSV-1High-Negative**</td><td rowspan=1 colspan=1>&lt;1X LoD</td><td rowspan=1 colspan=1>6.7%2/30</td><td rowspan=1 colspan=1>42.9(3.2%)</td><td rowspan=1 colspan=1>43.3%13/30</td><td rowspan=1 colspan=1>42.6(3.9%)</td><td rowspan=1 colspan=1>3.3%1/30</td><td rowspan=1 colspan=1>42.4(2.4%)</td><td rowspan=1 colspan=1>17.8%16/90</td><td rowspan=1 colspan=1>42.6(3.1%)</td></tr><tr><td rowspan=1 colspan=1>HS-2High-Positive**</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>36.5(1.6%)</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>35.7(2.2%)</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>36.4(1.7%)</td><td rowspan=1 colspan=1>100.0%90/90</td><td rowspan=1 colspan=1>36.2(2.0%)</td></tr><tr><td rowspan=1 colspan=1>HSV-2Low-Positive*</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>100.0%30/30</td><td rowspan=1 colspan=1>39.8(2.3%)</td><td rowspan=1 colspan=1>93.3%28/30</td><td rowspan=1 colspan=1>39.6(2.4%)</td><td rowspan=1 colspan=1>100.030/30</td><td rowspan=1 colspan=1>39.9(2.4%)</td><td rowspan=1 colspan=1>97.8%88/90</td><td rowspan=1 colspan=1>39.8(2.3%)</td></tr><tr><td rowspan=1 colspan=1>HSV-2HiighNegative**</td><td rowspan=1 colspan=1>&lt;1X LoD</td><td rowspan=1 colspan=1>30.0%9/30</td><td rowspan=1 colspan=1>42.3(2.8%)</td><td rowspan=1 colspan=1>46.7%14/30</td><td rowspan=1 colspan=1>42.1(4.0%)</td><td rowspan=1 colspan=1>16.7%5/30</td><td rowspan=1 colspan=1>42.2(2.5%)</td><td rowspan=1 colspan=1>31.1%28/90</td><td rowspan=1 colspan=1>42.2(3.0%)</td></tr><tr><td rowspan=1 colspan=1>HSVNegative**</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.0%60/60</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>98.3%59/60</td><td rowspan=1 colspan=1>42.7</td><td rowspan=1 colspan=1>100.0%60/60</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>99.4%179/180</td><td rowspan=1 colspan=1>42.7</td></tr></table>

\*Expected result is positive. $\% \mathrm { C V }$ calculated from results with non-zero Ct values. \*\* Expected result is negative. $\% \mathrm { C V }$ calculated from results with non-zero Ct values.

# b. Precision:

The precision studies were conducted in-house using a five membered panel tested twice a day by a single operator using one lot of reagents for twenty days. Panel members were tested in triplicate for each run (for a total of 600 data points for the 40 runs). The panel was prepared with a single target present (HSV-1 or HSV-2) at two concentrations:

HSV-1 Low Positive (1X LoD), HSV-1 High Positive (10X LoD), HSV-2 Low Positive (1X LoD), HSV-2 High Positive (10X LoD), and HSV-1/2 Negative. The HSV-1/2 Negative member was prepared using M4 medium without HSV present. The table below shows the Percent $( \% )$ Agreement for each panel member tested for the precision study.

# Precision Study Average Ct Values

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>% Agreement withexpected results</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td><td rowspan=1 colspan=1>Avg. Ct</td><td rowspan=1 colspan=1>SD Ct</td><td rowspan=1 colspan=1>%CV Ct</td></tr><tr><td rowspan=1 colspan=1>HSV-1 LowPositive</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>100.00%(120/120)</td><td rowspan=1 colspan=1>96.90% - 100.00%</td><td rowspan=1 colspan=1>42.93</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>1.74%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 HighPositive</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>100.00%(120/120)</td><td rowspan=1 colspan=1>96.90% - 100.00%</td><td rowspan=1 colspan=1>40.32</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.76%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 LowPositive</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>100.00%(120/120)</td><td rowspan=1 colspan=1>96.90% - 100.00%</td><td rowspan=1 colspan=1>41.31</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>2.56%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 HighPositive</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>100.00%(120/120)</td><td rowspan=1 colspan=1>96.90% - 100.00%</td><td rowspan=1 colspan=1>38.19</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.34%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.00%(120/120)</td><td rowspan=1 colspan=1>96.90% - 100.00%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

c. Linearity/assay reportable range: N/A d. Traceability, Stability, Expected values (controls, calibrators, or methods):

Assay Controls

The Anyplex™ II HSV-1/2 Assay Positive Control, Negative Control, and Blank Negative Control are included in each run to evaluate run validity.

• The Anyplex™ II HSV-1/2 Positive Control is constructed from plasmids with gDNA extracts from HSV-1 HF (VR-260) and HSV-2 G (VR-734). The Positive Control serves to demonstrate that the HSV-1/2 PCR reagents are functional, and to check the validity of the run.   
• The Anyplex™ II HSV-1/2 Negative Control is constructed of RNase-free water added to the Master Mix prior to PCR. The presence of HSV-1 and/or HSV-2 must not be detected in the Negative Control.   
• The Anyplex™ II HSV-1/2 Blank Negative Control is comprised of Internal Control added to blank VTM. The Blank Negative Control is carried through sample preparation alongside the clinical specimens.

# e. Detection limit:

The purpose of this study was to evaluate the Limit of Detection (LoD) of the Anyplex™ II HSV-1/2 Assay. The study was performed with two representative strains of HSV-1 (MacIntyre and HF) and two representative strains of HSV-2 (MS and G). A preliminary study was performed for each HSV-1 and HSV-2 strain. A series of seven (7) 10-fold dilutions of virus was made in pooled negative matrix, then tested with the Anyplex™ II HSV-1/2 Assay in replicates of three (3). Results from the preliminary LoD estimation study were used to determine the range for the

LoD analysis. The last dilution with a $100 \%$ detection rate was used to prepare a series of five (5) 2-fold dilutions using negative matrix as a diluent. These five (5) dilutions were tested on three (3) Anyplex™ II HSV-1/2 Assay kit lots in replicates of 20. Results from the three (3) kit lots were combined and the detection rate was calculated for each dilution. From this study, the LoD was determined to be the lowest concentration of target that was detected in at least $9 5 \%$ of the replicates.

The LoD of the Anyplex™ II HSV-1/2 Assay was determined to be $3 . 7 5 \mathrm { ~ X ~ } 1 0 ^ { 2 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV-1 (MacIntyre), $1 . 8 7 5 \mathrm { ~ X ~ i 0 ~ ^ { 2 } ~ T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV-1 (HF), and $3 . 7 5 \mathrm { X } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for both MS and G strains of HSV-2.

# Limit of Detection

HSV1-MacIntyre   

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Positive/total</td><td rowspan=1 colspan=1>Positivity rate (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>1.5 X 103</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>7.5 X 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>3.75 X 102</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>95.00</td><td rowspan=1 colspan=1>86.30</td><td rowspan=1 colspan=1>98.29</td></tr><tr><td rowspan=1 colspan=1>1.875 X 102</td><td rowspan=1 colspan=1>49/60</td><td rowspan=1 colspan=1>81.67</td><td rowspan=1 colspan=1>70.08</td><td rowspan=1 colspan=1>89.44</td></tr><tr><td rowspan=1 colspan=1>9.375 X 101</td><td rowspan=1 colspan=1>17/60</td><td rowspan=1 colspan=1>28.33</td><td rowspan=1 colspan=1>18.51</td><td rowspan=1 colspan=1>40.77</td></tr></table>

HSV1- HF   

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Positive/total</td><td rowspan=1 colspan=1>Positivity rate (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>1.5 X 103</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>7.5 X 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>3.75 X 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>1.875 X 102</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>95.00</td><td rowspan=1 colspan=1>86.30</td><td rowspan=1 colspan=1>98.29</td></tr><tr><td rowspan=1 colspan=1>9.375 X 101</td><td rowspan=1 colspan=1>46/60</td><td rowspan=1 colspan=1>76.66</td><td rowspan=1 colspan=1>64.56</td><td rowspan=1 colspan=1>85.56</td></tr></table>

HSV2-MS   

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Positive/total</td><td rowspan=1 colspan=1>Positivity rate (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>1.5 X 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>7.5 X 101</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>3.75 X 101</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.33</td><td rowspan=1 colspan=1>91.15</td><td rowspan=1 colspan=1>99.71</td></tr><tr><td rowspan=1 colspan=1>1.875 X 101</td><td rowspan=1 colspan=1>54/60</td><td rowspan=1 colspan=1>90.00</td><td rowspan=1 colspan=1>79.85</td><td rowspan=1 colspan=1>95.34</td></tr><tr><td rowspan=1 colspan=1>9.375 X 100</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>66.67</td><td rowspan=1 colspan=1>54.06</td><td rowspan=1 colspan=1>77.27</td></tr></table>

HSV2-G   

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Positive/total</td><td rowspan=1 colspan=1>Positivity rate (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>1.5 X 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>7.5 X 101</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>93.98</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>3.75 X 101</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.33</td><td rowspan=1 colspan=1>91.15</td><td rowspan=1 colspan=1>99.71</td></tr><tr><td rowspan=1 colspan=1>1.875 X 101</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>66.67</td><td rowspan=1 colspan=1>54.06</td><td rowspan=1 colspan=1>77.27</td></tr><tr><td rowspan=1 colspan=1>9.375 X 100</td><td rowspan=1 colspan=1>33/60</td><td rowspan=1 colspan=1>55.00</td><td rowspan=1 colspan=1>42.49</td><td rowspan=1 colspan=1>66.91</td></tr></table>

Analytical Reactivity/LoD Confirmation: In addition, an LoD confirmation study using titered stocks of cultured clinical isolates obtained from a supplier was conducted in house to confirm the observed LoD values for HSV-1 and HSV-2. A total of 42 clinical isolates (21 HSV-1 and 21 HSV-2) were diluted to the observed LoD in the negative matrix pool. Each HSV-1 and HSV-2 clinical isolate was extracted and tested in 3 replicates using the three different lots of kit reagents (9 replicates for each isolate). All strains were detected by the assay, demonstrating that the AnyplexTM II HSV-1/2 Assay can detect a broad range of HSV-1 and HSV-2 isolates.

# f. Analytical specificity:

A study was performed to evaluate cross reactivity of the Anyplex™ II HSV-1/2 Assay with fifty (50) microorganisms that might be found in anogenital lesion specimens. The panel members were obtained from suppliers as quantified cultures (CF), or prepared in house (IHC) by growing each organism and quantifying the culture. Bacteria were tested at $1 0 ^ { 6 } \mathrm { c f u / m L }$ or higher and viruses were tested at $1 0 ^ { 5 } \mathrm { p f u / m L }$ or higher. For bacteria that were difficult to obtain or grow, purified DNA was used in the place of the intact microorganism and tested at a concentration of $\geq 1 \times 1 0 ^ { 6 }$ genome copies $ { \mathrm { / m L } }$ . Human DNA was tested at a concentration of $1 \times 1 0 ^ { 5 }$ genome copies/mL. All samples were prepared by diluting microorganisms or DNA into M4RT viral transport medium prior to testing for cross-reactivity. No microorganisms tested positive for HSV-1 or HSV-2 using the Anyplex™ II HSV-1/2 Assay indicating no cross reactivity with these microorganisms.

To assess microbial interference, each test microorganism from the cross reactivity panel was added to a sample tube with two representative strains of HSV-1 (MacIntyre and HF), and HSV-2 (MS and G) at 3X LoD in M4RT viral transport medium. Each potentially interfering microorganism was tested in three (3) replicates at the same levels used in the cross-reactivity study described above. No evidence of microbial interference was observed for any of the 50 test microorganisms included in the analysis.

Cross Reactivity & Microbial Interference Panel   

<table><tr><td colspan="1" rowspan="1">NO.</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">Origin*</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Atopobium vaginae</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1 X 10 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Candida glabrata Z007</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Candida guilliemondii Z008</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Candida krusei Z009</td><td colspan="1" rowspan="1">1 X 106 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Candida lusitaniae Z010</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Candida parapsilosis Z011</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Candida tropicalis Z012</td><td colspan="1" rowspan="1">1 X 106 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Chlamydia trachomatis(-UW3)</td><td colspan="1" rowspan="1">1 X 108 IFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype E)</td><td colspan="1" rowspan="1">1 X 106 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype F)</td><td colspan="1" rowspan="1">1 X 106 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype G)</td><td colspan="1" rowspan="1">1 X 104 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype H)</td><td colspan="1" rowspan="1">1 X 106 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype I)</td><td colspan="1" rowspan="1">1 X 107 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype J)</td><td colspan="1" rowspan="1">5 X 10³ IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Chlamydia trachomatis(serotype K)</td><td colspan="1" rowspan="1">1 X 107 IFU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Cytomegalovirus (AD169)</td><td colspan="1" rowspan="1">1 X 106 1IU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">5 X 108 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1 X 10 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1 X 10 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">1 X 10 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Enterovirus (Type 71)</td><td colspan="1" rowspan="1">1 X 106TCID50/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Epstein-Barr virus(B95-8 strain)</td><td colspan="1" rowspan="1">5 X 108 copies/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Human Herpes 6B virus(Z29 strain)</td><td colspan="1" rowspan="1">1 X 105TCID50/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Human Herpes 7 virus(SB strain)</td><td colspan="1" rowspan="1">5X10TCID50/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Human Papilloma virus-16(Caski)</td><td colspan="1" rowspan="1">1,000 cell/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Human Papilloma virus-18(Hela)</td><td colspan="1" rowspan="1">5,000 cell/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Klebsiella pneumoniae Z026</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Lactobacillus acidophlus</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">1 X 106 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Moraxella catarrhalis Ne 11</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">5.27 X 10^{3CCU/mL(1.00 X 106copies/mL)</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Rubella virus</td><td colspan="1" rowspan="1">1 X 105TCID50/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1 X 109 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Streptococcus mutans Z072</td><td colspan="1" rowspan="1">1 X 10 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Streptococcus pneumoniae 19F</td><td colspan="1" rowspan="1">1 X 108 CFU/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Streptococcus pyogenesRosenbach</td><td colspan="1" rowspan="1">1 X 107 CFU/mL</td><td colspan="1" rowspan="1">IHC</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1 X 106tachyzoites/mL</td><td colspan="1" rowspan="1">CF</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">5X 10}$parasites/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">5 X 107 cCU/mL</td><td colspan="1" rowspan="1">OPZ</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Varicella Zoster virus</td><td colspan="1" rowspan="1">1 X 108 Copies/mL</td><td colspan="1" rowspan="1">CF</td></tr></table>

\* CF – Culture Fluid; OPZ – Organism Propagation from ZeptoMetrix; IHC – In-House Culture

This study was performed to evaluate potential interference with the Anyplex™ II HSV-1/2 Assay with a panel of twenty-two (22) biological and chemical substances. All of the potentially interfering substances were tested at concentrations at or above physiological levels or typical usage levels with HSV-1 (MacIntyre) and HSV-2 (MS) at 3X LoD to evaluate potential interference with the detection of the HSV-1 and HSV-2 targets. The study was also carried out in the absence of HSV to evaluate potential interference with the detection of the internal control of the Anyplex™ II HSV-1/2 Assay. Each potentially interfering substance was tested in triplicate. No interference was observed with any of the substances tested.

Interfering Substance Panel   

<table><tr><td colspan="3" rowspan="1">llterlerig Substalce I alel</td></tr><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Potential Inhibitor</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Whole blood with EDTA</td><td colspan="1" rowspan="1">Heme, DNA, proteases,nucleases</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Female Urine</td><td colspan="1" rowspan="1">Non-specific PCR inhibitors</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Male Urine</td><td colspan="1" rowspan="1">Non-specific PCR inhibitors</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Acyclovir Cream 50mg/g</td><td colspan="1" rowspan="1">Acycloguanosine</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">10 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">K-Y® Brand Jelly</td><td colspan="1" rowspan="1">Glycerin, Cellulose</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Douche</td><td colspan="1" rowspan="1">Decyl Glucoside;Octoxynol-9</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Monistat® 1</td><td colspan="1" rowspan="1">Miconazole nitrate cream2%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Monistat® 3</td><td colspan="1" rowspan="1">Miconazole nitrate cream4%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Vagisil Cream</td><td colspan="1" rowspan="1">Benzocaine, Resorcinol</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Triconazole 1</td><td colspan="1" rowspan="1">Tioconazole 6.5%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Balneol®HygienicCleansing Lotion</td><td colspan="1" rowspan="1">Mineral oil, Fatty acids</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">CVS Anti-Itch Cream</td><td colspan="1" rowspan="1">Benzocaine; BenzalkoniumChloride</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Listerine® AntisepticMouth Wash</td><td colspan="1" rowspan="1">Ethanol, Menthol, Thymol,Eucalyptol</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Abreva®</td><td colspan="1" rowspan="1">Docosanol 10%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Cold Sore LipBalm</td><td colspan="1" rowspan="1">Menthol 0.7%, Camphor1.7%, Phenol 0.4%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Releev® cold soretreatment</td><td colspan="1" rowspan="1">Benzalkonium Chloride0.13%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Potential Inhibitor</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Lip clear Lysine+®</td><td colspan="1" rowspan="1">Zinc Oxide 1.2%</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Surfactants, fluorides,antibacterials</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Contraceptive Jelly</td><td colspan="1" rowspan="1">Non-oxynol-9</td><td colspan="1" rowspan="1">7% w/v</td></tr><tr><td colspan="1" rowspan="1">Vaginal ContraceptiveFoam</td><td colspan="1" rowspan="1">Non-oxynol-9</td><td colspan="1" rowspan="1">7% w/v</td></tr></table>

# h. Specimen Stability in Viral Transport Media

The performance of the Anyplex™ II HSV-1/2 Assay was evaluated with respect to the specimen stability in viral transport media such as Remel M4, Remel M4RT, Remel M5, Remel M6, Copan Universal Transport Media (UTM), and BD Universal Viral Transport (UVT). Each transport medium was spiked with HSV-1 or HSV-2 positive clinical specimens at 3X LoD and stored and tested at the following temperatures and intervals: Refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C } ) / \mathrm { D a y } 0 , 3 , 5$ , and 7; Frozen $( - 2 0 ^ { \circ } \mathrm { C } ) / \mathrm { D a y } \ : 0$ , 15, 30, and 35. All samples were run in triplicate for all time points. Testing consisted of removing specimens from storage temperature, thawing (if frozen), extracting the DNA, and then running the Anyplex $\mathbf { \partial } _ { \cdot } \mathbf { T M }$ II HSV-1/2 Assay.

There was no interference observed with any of the viral transport media listed above for the detection of HSV-1 and HSV- 2 target or the internal control. The study data supported the stability claims for HSV-1 and HSV-2 specimens collected in Remel M4, Remel M4RT, Remel M5, Remel M6, Copan Universal Transport Media (UTM), and BD Universal Viral Transport (UVT) refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ) for 7 days and frozen $\left( - 2 0 ^ { \circ } \mathrm { C } \right)$ for 30 days.

# i. Competitive Inhibition

Competitive inhibition of the Anyplex™ II HSV-1/2 Assay was evaluated to assess the potential for interference in HSV-1/2 target detection when both HSV-1 and HSV-2 are present in a sample. The study was conducted by using simulated samples with equal or varying concentrations of HSV-1 and HSV-2 (3X, 10X, 100X and 1000X LoD). The results showed that competitive inhibition was not observed when the concentration of HSV1 and HSV2 strains are equal or varying. There was no influence of competitive inhibition between HSV-1 and HSV-2 on the performance of the AnyplexTM II HSV-1/2 Assay.

# h. Carry-over/Cross Contamination

The analytical carry-over/cross contamination study was performed to assess the potential for carry-over contamination of HSV target from a high-positive sample into a negative sample during the Anyplex™ II HSV-1/2 Assay run. The carry-over/cross contamination study was performed using HSV-1 MacIntyre and HSV-2 MS highpositive (\~100,000X LoD) samples in M4 medium. Alternating high positive samples and negative samples (M4 medium) were extracted, placed in sequential wells of the SmartCycler® II Dx and run. The run was repeated five (5) times.

No carry-over (0/100) or cross contamination (0/100) events were observed between positive and negative samples.

j. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

The clinical performance evaluation was performed against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

b. Matrix comparison: N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A b. Clinical Specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

The performance of the Anyplex™ II HSV-1/2 Assay was compared with the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc.) which is a gold standard/reference method, i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance

The performance of the Anyplex™ II HSV-1/2 Assay was evaluated at three geographically diverse locations within the United States. A total of 656 prospective specimens were tested by the Anyplex™ II HSV-1/2 Assay and compared to results obtained with the ${ \mathrm { E L V I S } } ^ { \mathrm { \textregistered } }$ (Enzyme Linked Virus Inducible System) HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System (Diagnostic Hybrids, Athens, OH). The reference ELVIS viral culture method used in this study is unable to detect co-infected specimens and cannot identify HSV-1 if HSV-2 is identified first. Consequently, if a specimen was positive for HSV-2, it was removed from the calculation of the HSV-1 clinical performance. Quality Controls (positive, negative, blank negative, and internal control) were run with each Anyplex™ II HSV-1/2 Assay run. The internal control was introduced into each specimen and the blank negative control prior extraction.

Results from the prospective studies are presented in the tables below.

HSV-1 Results for Female Anogenital Specimens   

<table><tr><td rowspan=2 colspan=2>HSV-1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Seegene</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>29a</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>429</td><td rowspan=1 colspan=1>430</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>458</td><td rowspan=1 colspan=1>550</td></tr><tr><td rowspan=1 colspan=2>Sensitivity;95% CI</td><td rowspan=1 colspan=3>98.9%(91/92);95% CI [94.1%-99.8%]</td></tr><tr><td rowspan=1 colspan=2>Specificity;95% CI</td><td rowspan=1 colspan=3>93.7% (429/458); 95% CI [91.0%-95.6%]</td></tr></table>

a Discordant analysis (bidirectional sequencing) was performed on all 29 samples identified as HSV-1 positive by the Anyplex™ II HSV-1/2 Assay. HSV-1 was detected in 18 of the 29 samples. The remaining 11 specimens remained discordant (HSV-1 was not detected). b Discordant analysis (bidirectional sequencing) was performed on the one sample identified as negative by the Anyplex™ II HSV-1/2 Assay. HSV-1 was detected in this sample.

HSV-2 Results for Female Anogenital Specimens   

<table><tr><td rowspan=2 colspan=2>HSV-2</td><td rowspan=1 colspan=3>ELVIS HSV ID and D3 Typing</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IMDx</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>35c</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3d</td><td rowspan=1 colspan=1>515</td><td rowspan=1 colspan=1>518</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>550</td><td rowspan=1 colspan=1>656</td></tr><tr><td rowspan=1 colspan=2>Sensitivity;95% CI</td><td rowspan=1 colspan=3>97.2%(103/106);95%CI [92.0%-99.0%]</td></tr><tr><td rowspan=1 colspan=2>Specificity; 95% CI</td><td rowspan=1 colspan=3>93.6% (515/550); 95% CI [91.3%-95.4%]</td></tr></table>

c Discordant analysis (bidirectional sequencing) was performed on all 35 discordant specimens identified as HSV-2 positive by the AnyplexTM II HSV-1/2 Assay. HSV-2 was detected in 21 of the 35 specimens. The remaining 14 remained discordant (HSV-2 was not detected). d Discordant analysis (bidirectional sequencing) was conducted for the three specimens identified as HSV-2 negative by the Anyplex $\mathbf { \Pi } ^ { \mathrm { T M } }$ II HSV-1/2 Assay. HSV-2 was not detected in any specimen.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

Prevalence: The prevalence of HSV-1 and HSV-2 observed during the multi-center clinical trial was calculated for the AnyplexTM II HSV-1/2 Assay. The prevalence rate for HSV-1 was established as $2 1 . 8 \%$ (120/550) for anogenital samples. The prevalence rate for HSV-2 was established as $2 1 . 0 \%$ (138/656) for anogenital samples.

Age Distribution for Female Anogenital Specimens   

<table><tr><td rowspan=1 colspan=1>Age (Years)</td><td rowspan=1 colspan=1>HSV-1 Positive</td><td rowspan=1 colspan=1>HSV-2 Positive</td></tr><tr><td rowspan=1 colspan=1>&lt;1 to 17</td><td rowspan=1 colspan=1>5/34(14.7%)</td><td rowspan=1 colspan=1>2/35(5.7%)</td></tr><tr><td rowspan=1 colspan=1>18 to 30</td><td rowspan=1 colspan=1>67/237(28.3%)</td><td rowspan=1 colspan=1>65/287(22.6%)</td></tr><tr><td rowspan=1 colspan=1>31 to 40</td><td rowspan=1 colspan=1>21/110(19.1%)</td><td rowspan=1 colspan=1>25/131(19.1%)</td></tr><tr><td rowspan=1 colspan=1>41 to 50</td><td rowspan=1 colspan=1>10/73(13.7%)</td><td rowspan=1 colspan=1>18/88(20.4%)</td></tr><tr><td rowspan=1 colspan=1>51 to 60</td><td rowspan=1 colspan=1>8/55(14.5%)</td><td rowspan=1 colspan=1>15/68(22.1%)</td></tr><tr><td rowspan=1 colspan=1>61 to 70</td><td rowspan=1 colspan=1>9/29(31.0%)</td><td rowspan=1 colspan=1>12/35(34.3%)</td></tr><tr><td rowspan=1 colspan=1>71 to 80</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>1/9(11.1%)</td></tr><tr><td rowspan=1 colspan=1>81 to 90</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td></tr><tr><td rowspan=1 colspan=1>91 to 95</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120/550(21.8%)</td><td rowspan=1 colspan=1>138/656(21.0%)</td></tr></table>

Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the AnyplexTM II HSV-1/2 Assay are shown below. These calculations are based on hypothetical prevalence and overall sensitivity and specificity per specimen type as determined in the clinical trial.

For HSV-1, these calculations are based upon an overall sensitivity and specificity of $9 8 . 9 \%$ and $9 3 . 7 \%$ , respectively, for anogenital swabs.

For HSV-2, these calculations are based upon an overall sensitivity and specificity of $9 7 . 2 \%$ and $9 3 . 6 \%$ , respectively, for anogenital swabs.

PPV was calculated using: (Sensitivity x Prevalence)/(Sensitivity x Prevalence $^ +$ [1 - Specificity] x [1 - Prevalence]).

NPV was calculated using: (Specificity x [1 - Prevalence])/([1 - Sensitivity] x Prevalence $^ +$ Specificity x [1 - Prevalence]).

# Prevalence vs. Hypothetical Predictive Values

<table><tr><td rowspan=3 colspan=1>Prevalence (%)</td><td rowspan=1 colspan=4>Female Anogenital Swabs</td></tr><tr><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV (%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>25.2</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>44.3</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>46.5</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>62.6</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>64.8</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>79.1</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>80.5</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>86.6</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>91.0</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>97.5</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.