# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K142721

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for the cobas® MRSA/SA Test.

# C. Measurand:

Target DNA sequences for:   
1. SCCmec/orfX junction area of methicillin-resistant Staphylococcus aureus (i.e., MREJ for SCCmec Right Extremity Junction).   
2. . capsular polysaccharide enzyme CAP5N (cpe) gene of Staphylococcus aureus.

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for the amplification and detection of methicillin-resistant Staphylococcus aureus (MRSA) DNA and Staphylococcus aureus (SA) DNA.

# E. Applicant:

Roche Molecular Systems, Inc.

F. Proprietary and Established Names: cobas® MRSA/SA Test

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NQX - System, nucleic acid amplification test, DNA, methicillin resistant Staphylococcus aureus, direct specimen

OOI – Real-time nucleic acid amplification system

4. Panel:

83- Microbiology

# H. Intended Use:

1. Intended use(s):

The cobas® MRSA/SA Test on the cobas® 4800 system, is a qualitative in vitro diagnostic real-time PCR assay, for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) and S. aureus (SA) DNA from nasal swabs to aid in the prevention and control of MRSA and SA infections in healthcare settings. The cobas® MRSA/SA Test is not intended to diagnose, guide or monitor treatment for MRSA or SA infections, or provide results of susceptibility to methicillin. A negative result does not preclude MRSA/SA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiology typing or for further susceptibility testing.

2. Indication(s) for use: Same as the intended use.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: cobas® 4800 System

# I. Device Description:

The cobas® 4800 System uses the cobas $\times 4 8 0$ Instrument for sample preparation, and the cobas $z 4 8 0$ Analyzer for amplification and detection. Both the cobas $\times 4 8 0$ Instrument and the cobas $z 4 8 0$ Analyzer are controlled by a computer workstation running the cobas $^ { \textregistered } 4 8 0 0$ System Software.

The Roche Molecular Systems (RMS) cobas® MRSA/SA Test utilizes real-time polymerase chain reaction (PCR) for the detection of Methicillin resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA) DNA from nasal swab specimens collected in MSwab medium to aid in the prevention and control of MRSA and SA infections in healthcare settings.

The cobas® MRSA/SA Test contains two major processes: (1) automated sample preparation to extract nucleic acids from the nasal swab specimens; (2) PCR amplification of target DNA sequences using MRSA and SA specific primers, and real-time detection of cleaved fluorescent-labeled MRSA and SA specific oligonucleotide detection probes. An Internal Control (IC), containing unrelated randomized DNA sequence, is added to all samples prior to automated sample preparation and is amplified and detected simultaneously with each sample to monitor the entire process.

The MSwab Collection, Transport and Preservation System (Copan Flock Technologies S.r.l.) is used for specimen collection, transportation and storage of specimen for the cobas® MRSA/SA Test.

The cobas® MRSA/SA Test utilizes six reagent kits:

1. cobas® 4800 MRSA/SA Amplification/Detection Kit   
2. cobas® 4 800 MRSA/SA Controls and Cofactor Kit   
3. cobas® 4800 System Wash Buffer Kit   
4. cobas® 4800 System Lysis Kit 1   
5. cobas® 4800 System Internal Control Kit 1   
6. cobas® 4800 System Sample Preparation Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lightcycler® MRSA Advanced Test

2. Predicate 510(k) number(s): k091409

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cobas® MRSA/SA Test(k142721)</td><td colspan="1" rowspan="1">Lightcycler® MRSAAdvanced Test(k091409)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The cobas® MRSA/SA Teston the cobas® 4800 system,is a qualitative in vitrodiagnostic real-time PCRassay, for the directdetection of methicillin-resistant Staphylococcusaureus (MRSA) andS.aureus (SA) DNA fromnasal swabs to aid in the</td><td colspan="1" rowspan="1">The LightCycler® MRSAAdvanced Test is aqualitative in vitrodiagnostic test for the directdetection of nasalcolonization withmethicillin-resistant(MRSA) to aid in theprevention and control ofMRSA infections inhealthcare settings. The test</td></tr><tr><td colspan="1" rowspan="1">′</td><td colspan="1" rowspan="1">prevention and control ofMRSA and SA infections inhealthcare settings. Thecobas® MRSA/SA Test isnot intended to diagnose,guide or monitor treatmentfor MRSA or SA infections,or provide results ofsusceptibility to methicillin.A negative result does notpreclude MRSA/SA nasalcolonization. Concomitantcultures are necessary torecover organisms forepidemiology typing or forfurther susceptibility testing.</td><td colspan="1" rowspan="1">is performed on theLightCycler® 2.0 Instrumentwith nasal swab specimensfrom patients suspected ofcolonization, uses swabextraction and mechanicallysis for specimenpreparation followed bypolymerase chain reaction(PR) for the amplificationof MRSA DNA, andfluorogenic target specifichybridization probes for thedetection of the amplifiedDNA.The LightCycler® MRSAAdvanced Test is notintended to diagnose, guideor monitor treatment forMRSA infections.Concomitant cultures arenecessary to recoverorganisms for epidemiologytyping or for furthersusceptibility testing.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasal swab</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Amplification Technology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Mechanism</td><td colspan="1" rowspan="1">Paired target-specifichybridization probes usingFörster resonance energytransfer (FRET)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">MRSA Analyte Target</td><td colspan="1" rowspan="1">SCCmec cassette RightExtremity (RE) junction ofmethicillin-resistantStaphylococcus aureus(MRSA)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure</td><td colspan="1" rowspan="1">Semi-automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cobasMRSA/SA Test(k142721)</td><td colspan="1" rowspan="1">Lightcycler® MRSAAdvanced Test(k091409)</td></tr><tr><td colspan="1" rowspan="1">SA Analyte Target</td><td colspan="1" rowspan="1">Capsular polysaccharideenzyme CAPN (CPE)</td><td colspan="1" rowspan="1">No SA target detected</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">PCR cycle threshold (Ct)analysis</td><td colspan="1" rowspan="1">Melting peak analysis</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Draft Guidance for Industry and Food and Drug Administration Staff. Establishing the Performance Characteristics of Nucleic Acid-Based In Vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA). Document issued on June 15, 2011.

# L. Test Principle:

Sample preparation for the cobas® MRSA/SA Test is automated with the use of the cobas x 480 instrument. Organisms in nasal swab specimens collected in MSwab medium are lysed with chaotropic agent, proteinase K, and SDS reagents. Released nucleic acids, along with added Internal Control DNA, are bound by magnetic glass particles. They are washed and then eluted into a small volume of buffer. The instrument then takes an aliquot of the eluted material and sets up the PCR reaction with an activated Master Mix.

# Target Selection

The well-characterized Right Extremity (RE) junction between the Staphylococcus aureus orfX gene and SCCmec cassette carrying the mecA drug-resistant gene was chosen to be the target to specifically detect MRSA. The capsular polysaccharide enzyme CAP5N (CPE) gene was chosen as the target for Staphylococcus aureus identification. This gene is conserved in Staphylococcus aureus and present in both methicillin resistant and methicillin sensitive organisms. Internal Control (IC) DNA sequence is unique and unrelated to either MRSA or SA target sequences.

# Target Amplification and Detection

The PCR cycling steps and detection of target signal occurs in the cobas $z 4 8 0$ Analyzer. The Master Mix reagent contains primer pairs and probes for orfX (MRSA), CPE (SA) and IC targets. If the target nucleic acid sequences are present, amplification with the corresponding primers will occur by a thermostable DNA polymerase, generating PCR products (amplicons). These products are detected by specific TaqMan probes containing a fluorescent dye and a quencher. Normally, the quencher suppresses the fluorescence of the dye.

However, if the PCR product is present, the probe hybridizes to the product and gets cleaved by the $5 ^ { \circ }$ to $3 ^ { \circ }$ nuclease activity of the polymerase. This reaction allows the fluorescence to be emitted from the dye, and the signal is recorded in real time during each PCR cycle by the cobas $\boldsymbol { z } 4 8 0$ analyzer. The signal is interpreted by the cobas® 4800 System Software and reported as final results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Precision Study

Within-laboratory precision was evaluated for the cobas® MRSA/SA Test at one (1) site with four (4) panel members:

1) Negative,   
2) High Negative (HN), $< 1 . 0 \mathrm { x }$ LoD,   
3) Low Positive (LP), 1 x LoD and   
4) Moderate Positive (MP), $3 . 0 \mathrm { x }$ LoD.

The study was conducted over twelve (12) days using three (3) reagent lots of two (2) kit sizes on three (3) cobas $^ { \textregistered } 4 8 0 0$ systems. Each run contained two (2) replicates of each of the panel members (12 days $_ { \textrm { X 3 } }$ lots x 2 replicates $= 7 2$ replicates/panel member/strain).

The results for the study are summarized in Table I, below. The upper limit of the 2- sided exact binomial $9 5 \%$ confidence interval (CI) of the detection for panel members at or near the limit of detection was $100 \%$ for MRSA and SA culture isolates. Positive rates for all other panel members were as expected.

<table><tr><td rowspan=1 colspan=7>Table I: Precision Study</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>MRSA-SA-</td><td rowspan=1 colspan=1>MRSA-SA+</td><td rowspan=1 colspan=1>MRSA+SA+</td><td rowspan=1 colspan=1>%Detected</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>No Analyte</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>(0.0%, 5.0%)</td></tr><tr><td rowspan=3 colspan=1>MRSA10364(NARSA384)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>15/72</td><td rowspan=1 colspan=1>57/72</td><td rowspan=1 colspan=1>79.2%</td><td rowspan=1 colspan=1>(68.0%, 87.8%)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr><tr><td rowspan=3 colspan=1>MRSA8065(ATCC300)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>9/72</td><td rowspan=1 colspan=1>63/72</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>(77.6%, 94.1%)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%,,100.0%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%, 1, 100.0%)</td></tr><tr><td rowspan=3 colspan=1>SA 10851(NARSA164</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>4/71*</td><td rowspan=1 colspan=1>67/71*</td><td rowspan=1 colspan=1>0/71*</td><td rowspan=1 colspan=1>94.4%</td><td rowspan=1 colspan=1>(86.2%, 98.4%)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.0%, 100.0%)</td></tr></table>

\*One (1) of seventy-two (72) samples tested could not be processed due to sample pipetting error on the instrument.

The overall coefficient of variation $( \mathrm { C V \% } )$ of the $\mathrm { C _ { t } }$ values for cobas® MRSA/SA Test was less than or equal to $1 . 3 \%$ indicating good reproducibility. $\mathrm { C _ { t } }$ value variability of the cobas® MRSA/SA Test is mostly attributed to random factors with contributions of reagent lot, kit size, run/day and instrument/operator being substantially (several-fold) lower or negligible across the detected targets.

The precision study results met the pre-defined acceptance criteria for LP, MP, and HN samples. The cobas® MRSA/SA Test reproducibly detected both targets at concentrations around and above the LOD of the test. Reproducibility has been demonstrated across multiple runs/days, reagent lots, kit sizes and instruments/operators.

The results of this study are acceptable and these results are described in labeling.

# Reproducibility Study

The reproducibility of the cobas® MRSA/SA Test on the cobas 4800 System was established in a multi-center study. The same strains and panel members were tested as listed in the precision study above (Table I). Panels were tested at three (3) sites by two (2) operators per site with one (1) run per operator per day, for five (5) days per lot, over two (2) lots for a total of one-thousand eight hundred (1,800) tests (180 tests/panel member or 90 tests/panel member/lot). Overall, sixty (60) runs were performed per site, per condition and all were valid runs. There were no failed/invalid tests.

The MRSA/SA reproducibility test panels were prepared by seeding MRSA strains NRS384 (MRSA-384) and ATCC 43300 (MRSA-43300), or SA strain RMSCC 10851 into contrived sample matrix (simulated clinical MSwab nasal specimens with mucin and human epithelial cells) at 1 of 3 concentrations $( < 1 \mathrm { ~ x ~ L o D }$ , $1 \times \mathrm { L o D }$ , and 3 $\times \mathrm { L o D } )$ ; a MRSA/SA-negative panel member was included as a panel member control. In all, there were 10 members per test panel with 3 replicates per panel member included in each run. The results of the reproducibility study are summarized in Table II below.

<table><tr><td colspan="9" rowspan="1">Table II: Reproducibility Study</td></tr><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="3" rowspan="1">Ct</td><td colspan="3" rowspan="1">Percent Agreement by Site(n/N)</td><td colspan="2" rowspan="1">Total Agreement</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Percent(n/N)</td><td colspan="1" rowspan="1">(95%C</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100.0(120/120)</td><td colspan="1" rowspan="1">100.0(120/120)</td><td colspan="1" rowspan="1">100.0(120/120)</td><td colspan="1" rowspan="1">100.0%(60/360)</td><td colspan="1" rowspan="1">(98.0%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1">HNNARSA-384</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">95.0 7/60)</td><td colspan="1" rowspan="1">83.3 50/60)</td><td colspan="1" rowspan="1">78.3(47/60)</td><td colspan="1" rowspan="1">85.6%(154/180)</td><td colspan="1" rowspan="1">(79.6%,90.3%</td></tr><tr><td colspan="1" rowspan="1">LPNARSA-384</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0%(180/180)</td><td colspan="1" rowspan="1">(98.0%,10.0%</td></tr><tr><td colspan="1" rowspan="1">MPNARSA-384</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">100.0 (60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0%(180/180)</td><td colspan="1" rowspan="1">(98.0%, 10.0%</td></tr><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="3" rowspan="1">Ct</td><td colspan="3" rowspan="1">Percent Agreement by Site/N)</td><td colspan="2" rowspan="1">Total Agreement</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Percent/N)</td><td colspan="1" rowspan="1">(95%C</td></tr><tr><td colspan="1" rowspan="1">HNATCC-43300</td><td colspan="1" rowspan="1">40.4</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">91.7(55/60)</td><td colspan="1" rowspan="1">81.7(49/60)</td><td colspan="1" rowspan="1">88.3(5/60)</td><td colspan="1" rowspan="1">87.2%(157/180)</td><td colspan="1" rowspan="1">(81.4%,91%%</td></tr><tr><td colspan="1" rowspan="1">LPATCC-43300</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.060/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0%180/180)</td><td colspan="1" rowspan="1">(98.0%, 100.0%</td></tr><tr><td colspan="1" rowspan="1">MPATCC-43300</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">100.0 60/60)</td><td colspan="1" rowspan="1">100.0 (60/60)</td><td colspan="1" rowspan="1">100.0 0/60)</td><td colspan="1" rowspan="1">100.0%180/180)</td><td colspan="1" rowspan="1">(98.0%, 100.0%</td></tr><tr><td colspan="1" rowspan="1">HNSA(NARSA 164)</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">23.3(14/60)</td><td colspan="1" rowspan="1">60.0(36/60)</td><td colspan="1" rowspan="1">66.7(40/60)</td><td colspan="1" rowspan="1">50.0%(90/180)</td><td colspan="1" rowspan="1">(42.5%,57.5%</td></tr><tr><td colspan="1" rowspan="1">LPSA(NARSA 164)</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">98.3(59/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">99.4%(179/180)</td><td colspan="1" rowspan="1">(96.9%,100.0%</td></tr><tr><td colspan="1" rowspan="1">MPSA(NARSA 164)</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0(60/60)</td><td colspan="1" rowspan="1">100.0%(180/180)</td><td colspan="1" rowspan="1">(98.0%,100.0%)</td></tr></table>

These results are acceptable and they are described in labeling.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

One set of cobas $^ \mathrm { \textregistered }$ MRSA/SA Test Positive and Negative Controls are included in each run. For any run, valid results must be obtained for both the Positive and Negative Control for the cobas® 4800 Software to display the reportable cobas® MRSA/SA Test results from that run.

# Internal Control

The Internal Control is a lambda phage molecule that contains randomized sequences and targets for internal control-specific primers and probe. The Internal Control is added to all specimens and the Positive and Negative Controls during sample preparation on the cobas $\times 4 8 0$ instrument. The Internal Control monitors nucleic acid amplification and detection steps for a given specimen. The Internal Control is also required for validation of the run controls.

# External Controls:

Whole-cell positive external controls are not provided with this assay to control for lysis. However, commercially available control materials, MSSA strain ATTC 29213 and MRSA strain ATTC 43300, can be used as positive controls, in addition to the other controls provided. During the clinical trial testing was conducted using the external Positive and Negative Controls listed below.

One set of cobas® MRSA/SA Test Positive and Negative Controls are included in each run. For any run, valid results must be obtained for both the Positive and Negative Control for the cobas® 4800 Software to display the reportable cobas® MRSA/SA Test results from that run.

# Positive Control

The MRSA $( + )$ Control which is provided with this assay contains non-infectious DNA plasmids of both MRSA and Staphylococcus aureus. The MRSA/SA $( + )$ Control monitors the nucleic acid amplification, and detection steps in a given run of the test. The MRSA/SA positive control result must be ‘Valid’.

# Negative Control

The negative control, which is provided with this assay, contains Poly rA RNA.   
The negative control result must be ‘Valid’.

During the clinical study, the positive and negative external control isolates were tested each day of testing. All MRSA/SA positive controls were detected accurately $100 \%$ , 116/116). All negative controls were detected accurately ( $100 \%$ , 116/116).

# Kit stability:

Many components of assay stability were previously assessed during previous submissions for the cobas HSV 1 and 2, C. diff and CT/NG tests, as components of the kit are identical.

Three production lots of new reagent components of the kits required for cobas® MRSA/SA testing were entered into the stability program to establish shelf life. The expiration date for each test kit was defined by the shortest dated component in that kit. These reagents include cobas® MRSA/SA Test specific reagent kits: the cobas® MRSA/SA Amplification/Detection Kit, the cobas® MRSA/SA Controls & Cofactor Kit; and the cobas® 4800 IC Kit 1, and Lysis Buffer 1 from the cobas® 4800 System Lysis Kit 1. Stability testing of the MRSA/SA specific reagents was carried out using the cobas® MRSA/SA Test. Stability of the cobas® 4800 IC Kit 1 was carried out using cobas® MRSA/SA Test, and stability of Lysis Buffer 1 was assessed using chemical/analytical tests.

The stability of these existing generic reagents demonstrated stability from 0-25 months in real-time experiments with the exception of Lysis Buffer-1, as was previously established. To date, the $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ storage real time testing stability results for three (3) lots of cobas® 4800 System Lysis Buffer-1 supports a shelf life of fifteen (15) months. Testing will continue, and the initial dating will be extended when data becomes available. Accelerated stability $2 5 ^ { \circ } \mathrm { C }$ storage) testing results predicts a shelf life of at least 19 months for cobas $^ { \textregistered } 4 8 0 0$ System Lysis Buffer-1 at the storage temperature of $2 { - } 8 \mathrm { { \% } }$ .

# Specimen stability:

The study was performed to determine the stability of MRSA and SA nasal swab specimens in MSwab medium (Copan MSwab™ Collection, Preservation and Transport System) after storage under various storage conditions for use with the cobas® MRSA/SA Test.

A total of twenty-six (26) nasal swab specimens (12 MRSA, 10 SA, 2 MRSA/SA and 2 negative) were included in the stability testing. Of these twenty-six (26) samples, six (6) MRSA positive, six (6) SA positive and two (2) MRSA/SA negative samples were prepared by pooling clinical specimens to create enough volume to test at all scheduled time-points. The remaining samples were prepared by spiking MRSA culture isolates (10364 or 8065) and/or SA culture isolate (10851) at $3 \mathrm { ~ x ~ }$ LoD into negative clinical specimen pool.

All positive and negative specimens generated positive and negative results, respectively, at all time-points of up to 40 days of cumulative storage.

The results indicated that nasal clinical specimens collected in Mswab medium were stable for:

1. up to 2 days at room temperature $( 1 5 . 3 0 ~ ^ { \circ } \mathrm { C } )$ followed by 5 days stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ ;   
2. up to 4 days at room temperature $( 1 5 . 3 0 \ ^ { \circ } \mathrm { C } )$ followed by 5 days stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$

followed by 1 month frozen at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ before testing with the cobas® MRSA/SA Test.

There is at least a two (2) hour interval from the completion of sample preparation and amplification/detection plate setup to the start of amplification/detection on the cobas® $\boldsymbol { z } 4 8 0$ Analyzer for the cobas® MRSA/SA Test.

# d. Detection limit:

The limit of detection (LoD) of the cobas® MRSA/SA Test was determined using contrived stocks of one Staphylococcus aureus (SA) and two methicillin resistant Staphylococcus aureus (MRSA) strains per CLSI guideline EP17-A2.

The titer of each culture isolates was determined by counting colonies from plates of serial dilutions. Each of the culture isolates was diluted in collection and transport medium, then spiked into a flocked swab, which was then immersed in simulated nasal swab matrix to produce a panel consisting of nine (9) concentration levels and one (1) negative level (simulated matrix only). The swab type and medium used for these assays are components of the Copan MSwab Collection, Transport and Preservation System designated in the labeling for use with the cobas® MRSA/SA Test.

Three lots of reagents and two different batch sizes were used for this study. For each reagent lot, a minimum of three independent panel dilutions were prepared. A minimum of 19 valid replicates per testing level were generated from multiple

instruments and runs for each reagent lot, yielding a minimum of 61 valid replicates for each testing level with all three lots of reagents.

The LoD of the cobas® MRSA/SA Test is the concentration where at least $9 5 \%$ observed positive rate for which all higher concentrations had an observed positive rate above $9 5 \%$ . The LoD study results are shown in Table III below.

<table><tr><td rowspan=1 colspan=5>Table III: LoD for MRSA and SA</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>SCCmec Type</td><td rowspan=1 colspan=1>spa Type</td><td rowspan=1 colspan=1>PFGE Type</td><td rowspan=1 colspan=1>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>MRSA 10364(NARSA 384)</td><td rowspan=1 colspan=1>IVa</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300-0114</td><td rowspan=1 colspan=1>1.53 x 105</td></tr><tr><td rowspan=1 colspan=1>MRSA 8065(ATCC 43300)</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>t007</td><td rowspan=1 colspan=1>Sac-15</td><td rowspan=1 colspan=1>1.23 x 106</td></tr><tr><td rowspan=1 colspan=1>SA(NARSA 164)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>t084</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.71 x 105</td></tr></table>

These study results are acceptable and they are described in labeling.

# e. Analytical reactivity:

The analytical inclusivity of the cobas® MRSA/SA Test was determined for 35 additional MRSA culture isolates and 5 additional SA culture isolates. The panel included a variety of strains, taking into account geographic origin, SCCmec type, RE type, SPA type, and Pulsed-Field Gel Electrophoresis (PFGE) type. The panels of at least three concentrations per isolate were used to determine the LoD for each strain in simulated nasal swab matrix. The LoD was calculated as the lowest concentration level with $\geq 9 5 ~ \%$ positive rate for which all higher concentration levels show $\geq 9 5 ~ \%$ positive rate.

For the five (5) SA isolates (Table IV) and thirty-five (35) MRSA isolates (Table V) tested the LoDs range from $\mathrm { i } . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ and $1 . 2 3 \mathrm { { \dot { x } } 1 0 ^ { 6 } \mathrm { { \dot { C F U } } / m l } }$ . The cobas MRSA/SA Test detected the right extremity (RE) types 1, 2, 3, 4, 6, 9, 11, 14, 24 and 25. The cobas® MRSA/SA Test detected MRSA staphylococcal cassette chromosome mec (SCCmec) types I, II, III, IV, V, VI and VIII, as well as MRSA pulsed-field gel electrophoresis (PFGE) types USA 100 to 1000.

<table><tr><td rowspan=1 colspan=2>Table IV: Analytical Reactivity (SA)</td></tr><tr><td rowspan=1 colspan=1>SA Isolate #</td><td rowspan=1 colspan=1>spa Type</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>t238</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>t018</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>t008</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>t002</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>t088</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table V: Analytical Reactivity (MRSA)</td></tr><tr><td rowspan=1 colspan=1>MRSAIsolate #</td><td rowspan=1 colspan=1>RE Type</td><td rowspan=1 colspan=1>SCCmecType</td><td rowspan=1 colspan=1>spa Type</td><td rowspan=1 colspan=1>PFGETye</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>new</td><td rowspan=1 colspan=1>t002</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>t242</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9/11</td><td rowspan=1 colspan=1>new</td><td rowspan=1 colspan=1>t024</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>t003</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t216</td><td rowspan=1 colspan=1>USA100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>t037</td><td rowspan=1 colspan=1>USA200</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t1578</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t002</td><td rowspan=1 colspan=1>USA100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA800</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t064</td><td rowspan=1 colspan=1>USA500</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t148</td><td rowspan=1 colspan=1>USA700</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t688</td><td rowspan=1 colspan=1>USA800</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t688</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t042</td><td rowspan=1 colspan=1>USA100</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t018</td><td rowspan=1 colspan=1>USA200</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t5576</td><td rowspan=1 colspan=1>USA800</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t004</td><td rowspan=1 colspan=1>USA600</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t216</td><td rowspan=1 colspan=1>USA1000</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t064</td><td rowspan=1 colspan=1>Iberian</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>t266</td><td rowspan=1 colspan=1>USA600</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t008</td><td rowspan=1 colspan=1>USA300</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>t002</td><td rowspan=1 colspan=1>USA800</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>t242</td><td rowspan=1 colspan=1>USA1000</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>new</td><td rowspan=1 colspan=1>t476</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>t149</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>t030</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td></tr></table>

In addition to thirty-seven (37) MRSA isolates and six (6) SA isolates tested in the LoD and analytical sensitivity studies shown above, two hundred and eighty-one (281) MRSA isolates from locations across Europe, United States, Japan and Australia and eighty-five (85) SA isolates from locations across the United States were tested. The MRSA collection contained MRSA isolates of different SCCmec types (I, II, III, IV, IVa, V, VI, VII, and new), and seventy-one (71) spa types. Of the eighty-five (85) SA isolates from the United States 75 spa types were represented. All of the SA isolates were detected by the cobas® MRSA/SA Test. Of the two hundred eighty-one (281) MRSA isolates, two hundred seventy-seven (277, $9 8 . 6 \% $ of the MRSA isolates were detected. Sequencing results suggest that the target regions for the four (4) MRSA isolates not detected by the cobas® MRSA/SA Test contained sequences not recognized by the primers and probes in the assay. One of the four isolates was a mecALGA251 strain (also known as mecC). All other results were as expected.

These study results are acceptable and they are described in labeling.

# f. Analytical specificity:

Microbial cross-reactivity and interference:

The cobas® MRSA/SA Test was examined for analytical specificity by testing nonMRSA/SA microorganisms, including coagulase-negative (CoNS) and methicillinresistant CoNS (MR-CoNS) in negative nasal matrix in the presence or absence of MRSA and SA targets.

Organisms were spiked at high concentration into the simulated nasal swab matrix, without (cross-reactivity) or with the targets (interference) at approximately $3 \mathrm { ~ x ~ L o D }$ All bacteria and fungi were tested at $1 . 0 \dot { 0 } \propto 1 \dot { 0 } ^ { 6 } \mathrm { C F U / m l }$ . All viruses except for Adenovirus type 1 $\mathrm { \bar { 1 } . 0 0 \times 1 0 ^ { 4 } P F U / m l ) }$ were tested at $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { P F U / m l }$ . The same two MRSA and one SA isolates used in the LoD study were used. The list of strains used for the cross-reactivity and microbial interference studies is listed below in Tables VI and VII.

<table><tr><td colspan="3" rowspan="1">Table VI: Staphylococcal Strains Included in Analytical Cross-Reactivity andMicrobial Interference</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Organisms</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">ATCC43957</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="2">ATCC33753</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="2">ATCC35538</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Staphylococcus captis</td><td colspan="1" rowspan="1">ATCC35661</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">ATCC51365</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">ATCC43764</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">ATCC35662</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">MayoClinicH18859</td></tr><tr><td colspan="1" rowspan="2">9</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC14990</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC35547</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC35983</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(Methicillin-resistant)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="2">12</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC35984</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="2">13</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC51624</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="2">14</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC51625</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC700583</td></tr><tr><td colspan="1" rowspan="2">16</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="2">ATCC27676</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">ATCC43958</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Staphylococcus felis</td><td colspan="1" rowspan="1">ATCC49168</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">ATCC35539</td></tr><tr><td colspan="1" rowspan="2">20</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="2">ATCC29968</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC29970</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC43252</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC25615</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC35982</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Staphylococcus hominis subsp. Hominis</td><td colspan="1" rowspan="1">ATCC27844</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Staphylococcus hominis subsp. Hominis</td><td colspan="1" rowspan="1">ATCC27845</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC29663</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Staphylococcus kloosii</td><td colspan="1" rowspan="1">ATCC43959</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">ATCC29070</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC49576</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">ATCC51129</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Staphylococcus pseudointermedius</td><td colspan="1" rowspan="1">DSMZ21284**</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Staphylococcus pulvereri</td><td colspan="1" rowspan="1">ATCC51698</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">ATCC15305</td></tr><tr><td colspan="1" rowspan="2">35</td><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="2">ATCC43808</td></tr><tr><td colspan="1" rowspan="1">(subspecies coagulans)</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Staphylococcus sciuri</td><td colspan="1" rowspan="1">ATCC49575</td></tr><tr><td colspan="1" rowspan="2">37</td><td colspan="1" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="2">ATCC27848</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="1">ATCC11631</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">ATCC27836</td></tr><tr><td colspan="1" rowspan="2">40</td><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="2">ATCC27839</td></tr><tr><td colspan="1" rowspan="1">(Methicillin-resistant)</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">RMSCC 1224</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">ATCC35663</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">ATCC29971</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Table VII: Staphylococcal Strains Included in Analytical Cross-Reactivity andMicrobial Interference</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Organisms</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC19606</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC17906</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC13472</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Bordetella bronchioseptica</td><td colspan="1" rowspan="1">ATCC19395</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">ATCC15311</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC9797</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC25416</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC10231</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC2001</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC22019</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC750</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">CDC-CWL-011 strain</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC8090</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC27028</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">ATCC49368</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">ATCC7715</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">ATCC10340</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">ATCC33030</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Corynebacterium glutamicum</td><td colspan="1" rowspan="1">ATCC13032</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">ATCC43734</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC32719</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">ATCC23834</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC13048</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC13047</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">ATCC49996</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC49573</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC8043</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC11775</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Finegoldia magna</td><td colspan="1" rowspan="1">RMSCC 974</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Haemophilus aphrophilus</td><td colspan="1" rowspan="1">ATCC19415</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC33391</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">ATCC33392</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Issatchenkia orientalis</td><td colspan="1" rowspan="1">ATCC6258</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC33496</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Klebsiella pneumoniae (KPC producing)</td><td colspan="1" rowspan="1">ATCC700603</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Klebsiella pneumoniae (KPC producing)</td><td colspan="1" rowspan="1">ATCC BAA1900</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">ATCC33820</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Lactobacillus delbrueckii</td><td colspan="1" rowspan="1">ATCC12315</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC33152</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Leifsonia aquatica (formerly Corynebacteriumaquaticum)</td><td colspan="1" rowspan="1">ATCC14665</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC15313</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Microbacterium testaceum</td><td colspan="1" rowspan="1">ATCC15829</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC4698</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC8176</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis avirulent</td><td colspan="1" rowspan="1">ATCC25177</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC15531</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Mycoplasma salivarium</td><td colspan="1" rowspan="1">ATCC23064</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC13077</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Parvimonas micra</td><td colspan="1" rowspan="1">ATCC33270</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">ATCC27883</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">Planococcus maritimus</td><td colspan="1" rowspan="1">RMSCC11454**</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC29906</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">RMSCC204**</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC22914</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC33584</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC11250</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">ATCC6939</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">ATCC25296</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Enterica (formerlySalmonella typhimurium)</td><td colspan="1" rowspan="1">RMSCC374*</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC8100</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC29930</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">RMSCC983**</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC12395</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC33399</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">ATCC25175</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC33400</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC12344</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC7073</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">ATCC10556</td></tr><tr><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">Streptococcus suis</td><td colspan="1" rowspan="1">ATCC43765</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">ATCC9610</td></tr><tr><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">VR-7</td></tr><tr><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">Dugan (VR-40)</td></tr><tr><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">VR-740</td></tr><tr><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">VR-1558</td></tr><tr><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td></tr><tr><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">B95-8</td></tr><tr><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">HSV 1</td><td colspan="1" rowspan="1">MacIntyre (VR-539)</td></tr><tr><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">Human Adenovirus type 1</td><td colspan="1" rowspan="1">VR-1</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">Human enterovirus 71</td><td colspan="1" rowspan="1">VR-1775</td></tr><tr><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">Peru6-2003</td></tr><tr><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">A/PR/8/34 (VR-95)</td></tr><tr><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">Influenza A/H3N2 A/HongKong/8/68</td><td colspan="1" rowspan="1">H3N2</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">Enders (VR-106)</td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">C-35 (VR-94)</td></tr><tr><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Greer (VR-92)</td></tr><tr><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">C-243 (VR-93)</td></tr><tr><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">Rhinovirus type 1A</td><td colspan="1" rowspan="1">VR-1559</td></tr><tr><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">VR-1540</td></tr><tr><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">VR-1400</td></tr><tr><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1">HCT-15 cells (human genomic DNA)</td><td colspan="1" rowspan="1">RMSCC3515</td></tr></table>

None of the organisms tested above cross-reacted or interfered with the assay at the concentrations tested.

These study results are acceptable and these results are described in labeling.

# Co-infection Studies:

To test for competitive interference with the MRSA target, SA organisms including borderline oxacillin-resistant S. aureus (BORSA) strains and empty cassette variants (mecA drop-outs) were tested at high concentration to examine any cross-reactivity and/or interference with MRSA detection at $3 \mathrm { ~ x ~ }$ LoD using the two (2) MRSA strains used in the LoD study. Both MRSA strains were correctly identified in every case. Ten (10) BORSA strains and sixteen (16) empty-cassette (mecA drop out) MRSA strains were also tested in the absence of MRSA. The cobas® MRSA/SA test correctly detected 2/2 MSSA, 10/10 BORSA and 13/16 mecA drop out isolates as positive for SA and negative for MRSA. Three (3) of the mecA drop out isolates were detected as positive for SA and MRSA. The risk of incorrect identification of MRSA from mecA drop-outs is mitigated in the labeling by requiring follow-up testing for results indicating the presence of MRSA. A limitation is also included in labeling indicating that false positive MRSA result may occur if an “empty cassette variant” Staphylococcus aureus is present.

Additionally, two (2) methicillin-sensitive (MSSA) and two (2) methicillin-resistant Staphylococcus epidermidis (MRSE) isolates were spiked at increasing concentrations to determine if there is any competitive inhibition effect on the detection of the MRSA and SA targets (SA strains were tested against MRSA targets only). In the presence of increasing concentration of SA or MRSE, low positive MRSA targets were consistently detected as positive. This demonstrates that there is no competitive inhibition by these organisms.

In consideration of microbial specificity overall, none of the 135 closely related Staphylococcus species or microorganisms commonly found in the nasal flora at tested levels yielded false positive or false negative results. The cobas® MRSA/SA Test detected all 10 BORSA isolates, 13 of 16 empty cassette variants and 2 SA isolates correctly as SA only. Increasing concentration of MSSA did not compete with the detection of low level MRSA targets and increasing concentrations of MRSE did not compete with the detection of MRSA and SA targets.

# g. Interfering substances:

Twenty-seven (27) substances (Table VIII) that may potentially be present in the sampling area including human whole blood, mucin, nasal sprays, medications or moisturizers were evaluated in this study.

<table><tr><td rowspan=1 colspan=1>Table VIII: Interfering Substances</td></tr><tr><td rowspan=1 colspan=1>Substance</td></tr><tr><td rowspan=1 colspan=1>Whole blood</td></tr><tr><td rowspan=1 colspan=1>Mucin</td></tr><tr><td rowspan=1 colspan=1>Afrin Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>Beconase Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>Bepanthen® nasal ointment</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic Max Sore Throat Lozenges</td></tr><tr><td rowspan=1 colspan=1>Fluticasone Propionate (50 mcg) Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>FluMist® (Afluria, Influenza virus vaccine)</td></tr><tr><td rowspan=1 colspan=1>Flunisolide Nasal Solution USP, 0.025%</td></tr><tr><td rowspan=1 colspan=1>Mupirocin Ointment</td></tr><tr><td rowspan=1 colspan=1>Dristan Nasal Mist</td></tr><tr><td rowspan=1 colspan=1>Luffee1M</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone Acetonide Nasal spray</td></tr><tr><td rowspan=1 colspan=1>NasalCrom Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>Nasonex Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine</td></tr><tr><td rowspan=1 colspan=1>Otrivine Nasal Spray</td></tr><tr><td rowspan=1 colspan=1>Relenza</td></tr><tr><td rowspan=1 colspan=1>Budesonide Inhalation Suspension 0.25 mg/2 ml</td></tr><tr><td rowspan=1 colspan=1>Azelastin HCL Nasal Solution</td></tr><tr><td rowspan=1 colspan=1>Equate Saline Nasal Moisturizing Spray</td></tr><tr><td rowspan=1 colspan=1>Rhinaris® Nasal gel</td></tr><tr><td rowspan=1 colspan=1>Tobramycin and Dexamethasone OphthalmicSolution</td></tr><tr><td rowspan=1 colspan=1>Releev™M (for cold sores)</td></tr><tr><td rowspan=1 colspan=1>Zicam Nasal Gel</td></tr><tr><td rowspan=1 colspan=1>QVAR (40 mcg) Inhalation Aerosol</td></tr><tr><td rowspan=1 colspan=1>Nostrilla</td></tr></table>

For blood and mucin, three levels of each interferent were spiked into samples constructed with a simulated nasal swab matrix at $3 \mathrm { ~ x ~ L o D }$ for reach strain used in the LoD studies. Interferents were added into the samples in the amounts corresponding to a percentage of volume/volume $\mathbf { \Pi } ( \mathbf { v } / \mathbf { v } )$ or weight/volume $\mathbf { \Pi } ( \mathbf { w } / \mathbf { v } )$ based on the maximum capacity of the flocked swab used with the cobas® MRSA/SA Test.

Ten (10) replicates were prepared this way for each interferent at each level, for each of the three (3) strains. Samples with interferents but no targets were also tested. At $1 0 0 \% \ \mathrm { v / v }$ whole blood level and at $2 0 \% \mathrm { w / v }$ mucin level, false negative results were observed for target positive samples, as listed in Table VI below.

For exogenous substance testing, substances were tested in concentrations specified per the manufacturer’s labeling. Each product was spiked into simulated nasal swab matrix and tested in the absence or presence of MRSA and SA targets used in the LoD studies at approximately $3 \mathrm { ~ x ~ }$ LoD. Each substance was introduced in the

amounts corresponding to $100 \%$ of the swab capacity, except for Relenza®, which was tested at the maximum prescription dosage. For substances that interfered with the cobas® MRSA/SA Test at initial testing concentration, lower concentrations were tested to identify the level that is tolerated by the test. Each condition was tested in triplicate.

The performance of the cobas® MRSA/SA Test was not affected by 23 out of 25 exogenous substances tested. Rhinaris® Nasal gel and ReleevTM interfered with the performance of the cobas® MRSA/SA Test when present in the amounts above $1 5 \%$ (Rhinaris® Nasal gel) and above $2 5 \%$ (ReleevTM) of the swab capacity.

The minimum concentration that caused interference for each substance in the conditions tested us listed in Table IX below.

<table><tr><td rowspan=1 colspan=1>Table IX: Interfering Substances</td></tr><tr><td rowspan=1 colspan=1>Strains</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood   100% v/v</td></tr><tr><td rowspan=1 colspan=1>Type II Mucin            20% w/v</td></tr><tr><td rowspan=1 colspan=1>RhinarisNasal gel      15% v/v</td></tr><tr><td rowspan=1 colspan=1>ReleevTM                    25% v/v</td></tr></table>

These study results are acceptable. A limitation is included in labeling indicating that false negative or invalid results may occur due to interference from these various substances.

# h. Carry-Over:

A cross-contamination study was conducted for the cobas® MRSA/SA Test. MRSA and negative samples were processed in a checkerboard configuration on the cobas® 4800 system. High titer samples were prepared by spiking MRSA culture to simulated nasal swab matrix to generate a cycle threshold $\mathrm { ( C _ { t } ) }$ that exceeded $9 5 \%$ of signal from specimens of infected patients of the clinical specimen population.

Five (5) runs were performed on each of the three (3) cobas® 4800 systems (3 instruments x $5 \mathrm { \ r u n s } = 1 5$ runs total). The first run on each system contained only the negative samples to confirm the instrument was clean. The three (3) subsequent runs on each system had alternating positive and negative samples in checkerboard configurations to assess the cross contamination rate (3 instruments $_ { \textrm { X 3 } }$ checkerboard runs $= 9$ checkerboard runs total). The last run on each system contained only the negative samples to assess the carry-over contamination rate.

There were no cross-contamination events in any of the nine checkerboard runs across three cobas $^ { \textregistered } 4 8 0 0$ systems (a total of 423 MRSA negative samples) for an observed cross-contamination rate of $0 \%$ . All results in the last three (3) runs containing only the negative samples were negative, suggesting that there was no carry-over run-to-run contamination.

i. Assay cut-off:

The $\mathrm { C _ { t } }$ cut-off values for the cobas® MRSA/SA Test was determined to achieve the balanced performance against the established reference culture method. The initial $\mathrm { C _ { t } }$ cut-off values were determined using 533 nasal swab samples collected at two sites in the United States for a pre-clinical feasibility study. These samples were tested on the cobas ${ } ^ { \mathbb { R } } 4 8 0 0$ System using two lots of reagents. Direct and enrichment MRSA and SA culture procedures were also conducted on these specimens. Final combined direct and enrichment culture results were designated as reference culture results.

The initial cut-off values were further evaluated in an interim analysis of the United States clinical utility study data with first 1,643 subject results. The results of this analysis and final cut-off values are listed in Table X below.

<table><tr><td rowspan=1 colspan=6>Table X: Assay Cutoff</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Analyte</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Reference Culture</td></tr><tr><td rowspan=16 colspan=1>cobas®MRSA/SATest</td><td rowspan=8 colspan=2>SA(Ct 39.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>402</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1148</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>(91.2%-95.8%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>94.5%</td><td rowspan=1 colspan=1>(93.1%-95.6%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>(82.3%-88.6%)</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>(96.8%-98.5%)</td></tr><tr><td rowspan=8 colspan=2>MRSA(Ct 41.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1503</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>93.1%</td><td rowspan=1 colspan=1>(86.4%-96.6%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>(96.6%-98.1%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>70.7%</td><td rowspan=1 colspan=1>(62.4%-77.7%)</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>(99.0%-99.8%)</td></tr></table>

The final $\mathrm { C _ { t } }$ cut-off values were determined to be 39.0 for SA and 41.0 for MRSA in order to assure balanced performance of the cobas® MRSA/SA Test against the defined culture reference method.

The results of this study are acceptable.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

All analytical studies were conducted in a simulated negative matrix. A comparison study was conducted between negative clinical matrix and a contrived negative matrix in order to validate the use of the contrived negative matrix in place of a clinical negative matrix for the analytical studies in section M1 above. Contrived negative matrix was constructed to mimic challenging clinical specimens, consisting of mucin, human cells and an isotonic (MSwab) media.

The performance results of the simulated matrix compared to the natural nasal swab specimen was demonstrated using two different MRSA isolates spiked into each matrix at $1 \mathrm { \Delta x }$ LoD. The $\mathrm { C _ { t } }$ values between the two matrices demonstrate that the simulated matrix is comparable to a natural specimen. All specimens were detected $( 1 0 0 \% )$ . The results of this matrix comparison study are listed in Table XI below.

<table><tr><td rowspan=1 colspan=6>Table XI: Matrix Comparison Study</td></tr><tr><td rowspan=2 colspan=1>Panel ID</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>Pooled Negative ClinicalMatrix</td><td rowspan=1 colspan=2>Contrived NegativeMatrix</td></tr><tr><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>MRSA 10364(NARSA 384)</td><td rowspan=1 colspan=1>orfX</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>38.8 - 39.5</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>38.7 - 39.5</td></tr><tr><td rowspan=1 colspan=1>IC</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>37.7 - 38.3</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>37.9 - 38.7</td></tr><tr><td rowspan=2 colspan=1>MRSA 10364(NARSA 384)</td><td rowspan=1 colspan=1>orfX</td><td rowspan=1 colspan=1>39.2</td><td rowspan=1 colspan=1>38.9 - 39.5</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>37.6 - 38.4</td></tr><tr><td rowspan=1 colspan=1>IC</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>37.6 - 38.4</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>37.7 - 38.5</td></tr></table>

The results of this study are acceptable.

# 3. Clinical studies:

This was a prospective multi-site study to evaluate the performance of the cobas® MRSA/SA Test compared with direct culture and enrichment culture in the screening of patients for nasal colonization with methicillin-resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA). Six (6) collection sites participated in this study and three (3) sites performed testing with the cobas® MRSA/SA Test. Reference testing using direct and enrichment culture was performed at a central reference laboratory that specializes in culture and molecular detection of MRSA and SA.

Swabs were collected from patients that met the inclusion criteria. Each subject had one (1) Standard-of-Care (SOC) swab collected, for SOC testing if applicable, and one (1) study swab collected using the Copan MSwab Collection, Transport and Preservation System. The study swab was processed using the cobas® MRSA/SA Test and used for direct and enrichment culture as the reference method.

On the day of collection, the swab was expressed into MSwab media and discarded. An aliquot of the eluted specimen was transported to a reference laboratory at $2 { - } 8 ^ { \circ } \mathrm { C }$ to be cultured within 48 hours of collection. The remainder of the swab-expressed media in the MSwab tube is tested with the cobas® MRSA/SA Test.

At the reference laboratory, the aliquoted swab-expressed media was divided into three (3) even fractions. Two fractions are applied to HardyCHROM™ MRSA, HhardyCHROM™ Staph aureus for direct culture. If growth was observed within twentyfour (24) hours, presumptive colonies were subcultured onto blood agar. Confirmed colonies are reported as MRSA or SA, as appropriate. The third fraction was incubated in tryptic soy broth (TSB) with $6 . 5 \%$ w/v sodium chloride for up to fourty-eight (48) hours, subcultured on blood agar then confirmed if direct culture results did not detect MRSA. Presumptive MRSA isolates from the MRSA plate were confirmed using the cefoxitin disk screening test for methicillin resistance.

Clinical performance was based on comparison of the cobas® MRSA/SA Test results to those obtained by a composite culture of directly plated patients’ nasal swabs and culture of the transport fluid material at a central location.

# a. Clinical Sensitivity:

Of the two thousand five hundred twenty-eight (2528) specimens collected (1 specimen collected/enrolled patient) in the clinical study, twelve (12) were excluded due to inclusion/exclusion criteria, withdrawal or consent errors. Twelve (12) were non-evaluable due to errors in sample handling or invalid cobas® MRSA/SA Test results. During culture enrichment, four (4) cultures were confounded due to Proteus overgrowth, excluding them from evaluation for enrichment. One of the four was positive for SA from direct culture evaluation.

Table XII below shows the comparison of the cobas® MRSA/SA Test with combined direct and enrichment culture results for the 2,500 evaluable specimens for MRSA and 2,501 evaluable specimens for SA. MRSA sensitivity and specificity relative to combined direct and enrichment culture were $9 3 . 1 \%$ and $9 7 . 5 \%$ , respectively. The overall PPV and NPV of the cobas $^ \mathrm { \textregistered }$ MRSA/SA Test for MRSA were $7 1 . 6 \%$ and $9 9 . 5 \%$ , respectively. Similarly, SA sensitivity and specificity relative to combined direct and enrichment culture were $9 3 . 9 \%$ and $9 4 . 2 \%$ , respectively. The overall PPV and NPV of the cobas® MRSA/SA Test for SA were $8 5 . 3 \%$ and $9 7 . 7 \%$ , respectively. Across test sites, MRSA sensitivity and specificity ranged from $9 1 . 4 \%$ to $9 4 . 5 \%$ and from $9 7 . 0 \%$ to $9 7 . 9 \%$ , respectively. The corresponding ranges for SA were from $9 1 . 6 \%$ to $9 6 . 1 \%$ and from $9 2 . 9 \%$ to $9 5 . 4 \%$ .

<table><tr><td colspan="4" rowspan="1">Table XII: Clinical Performance Data for the cobas®MRSA/SA Test vs. Direct and Enrichment Culture forSA and MRSA</td></tr><tr><td colspan="4" rowspan="1">MRSA Overall</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">149</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">208</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">2281</td><td colspan="1" rowspan="1">2292</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">160</td><td colspan="1" rowspan="1">2340</td><td colspan="1" rowspan="1">2500</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 93.1% (149/160) 95% CI (88.1%-96.1%)Specificity: 97.5% (2281/2340) 95% CI (96.8%-98.0%)</td></tr><tr><td colspan="4" rowspan="1">MRSA Site1</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">74</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">963</td><td colspan="1" rowspan="1">968</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">984</td><td colspan="1" rowspan="1">1042</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 91.4% (53/58) 95% CI (81.4%, 96.3%)Specificity: 97.9% (963/984) 95% CI (96.8%, 98.6%)</td></tr><tr><td colspan="4" rowspan="1">MRSA Site 2</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">73</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">768</td><td colspan="1" rowspan="1">771</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">789</td><td colspan="1" rowspan="1">844</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 94.5% (52/55) 95% CI (85.1%, 98.1%)Specificity: 97.3% (768/789) 95% CI (96.0%, 98.3%)</td></tr><tr><td colspan="4" rowspan="1">MRSA Site 3</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">550</td><td colspan="1" rowspan="1">553</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">567</td><td colspan="1" rowspan="1">614</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 93.6% (44/47) 95% CI (82.8%, 97.8%)Specificity: 97.0% (550/567) 95% CI (95.3%, 98.1%)</td></tr><tr><td colspan="4" rowspan="1">SA Overall</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">620</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">727</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">1734</td><td colspan="1" rowspan="1">1774</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">660</td><td colspan="1" rowspan="1">1841</td><td colspan="1" rowspan="1">2501</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 93.9% (620/660) 95% CI (91.9%-95.5%)Specificity: 94.2% (1734/1841) 95% CI (93.0%-95.2%)</td></tr><tr><td colspan="4" rowspan="1">SA Site 1</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">246</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">301</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">725</td><td colspan="1" rowspan="1">742</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">263</td><td colspan="1" rowspan="1">780</td><td colspan="1" rowspan="1">1043</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 93.5% (246/263) 95% CI (89.9%, 95.9%)Specificity: 92.9% (725/780) 95% CI (90.9%, 94.5%)</td></tr><tr><td colspan="4" rowspan="1">SA Site 2</td></tr><tr><td colspan="1" rowspan="2">cobas®MRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">249</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">586</td><td colspan="1" rowspan="1">595</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">230</td><td colspan="1" rowspan="1">614</td><td colspan="1" rowspan="1">844</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 96.1% (221/230) 95% CI (92.7%, 97.9%)Specificity: 95.4% (586/614) 95% CI (93.5%, 96.8%)</td></tr><tr><td colspan="4" rowspan="1">SA Site 3</td></tr><tr><td colspan="1" rowspan="2">cobasMRSA/SA Test</td><td colspan="3" rowspan="1">Direct and Enrichment Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">153</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">177</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">423</td><td colspan="1" rowspan="1">437</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">447</td><td colspan="1" rowspan="1">614</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 91.6% (153/167) 95% CI (86.4%, 94.9%)Specificity: 94.6% (423/447) 95% CI (92.1%, 96.4%)</td></tr></table>

The positive and negative external control isolates were tested each day during the clinical studies. All MRSA/SA positive controls were detected accurately $100 \%$ ,

116/116). All negative controls were detected accurately ( $100 \%$ , 116/116).

These study results are acceptable.

b. Clinical specificity: See table above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

During the multi-center clinical trials, based on combined direct and enrichment culture, the overall incidence of MRSA in nasal swab specimens during these studies was $6 . 4 \%$ (160/2500). The overall incidence of SA in nasal swab specimens during these studies was $2 6 . 4 \%$ (660/2501).

N. Instrument Name: cobas® 4800 System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ____ X___ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X___ or No

3. Specimen Identification: Specimens are identified using barcodes on specimen vials.

4. Specimen Sampling and Handling:

Specimens are placed on the cobas $\times 4 8 0$ instrument as open tubes and specimen processing is fully automated. After completion of specimen processing, the user transfers the plate carrier to the cobas $\boldsymbol { z } 4 8 0$ instrument for automated amplification and detection. Specimens can be processed directly from primary collection vials or as aliquots of the specimen in secondary vials. See section I for more information on specimen handling.

5. Calibration:

No calibration is required by the user. Roche technicians perform calibration periodically as required.

6. Quality Control:

See section M.1.c for information on internal and external controls.   
See section M.3.a for information on external control performance during clinical trials.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.