# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

To obtain a substantial equivalence determination for a new device

# C. Measurand:

Target nucleic acids from:

· Giardia lamblia   
· Cryptosporidium (C. hominis and C. parvum only) Entamoeba histolytica

D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) assay

# E. Applicant:

BD Diagnostics Systems Becton, Dickinson and Company

F. Proprietary and Established Names: BD MAX™ Enteric Parasite Panel (EPP) BD MAXTM System

# G. Regulatory Information:

1. Regulation section: 866. 3990 - Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: II

3. Product code:

PCH, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD MAX™ Enteric Parasite Panel performed on the BD MAX™ System is an automated in vitro diagnostic test for the direct qualitative detection of enteric parasitic pathogens. The BD MAX Enteric Parasite Panel detects nucleic acids from:

• Giardia lamblia   
• Cryptosporidium (C. hominis and $C .$ . parvum only)   
• Entamoeba histolytica

Testing is performed on unpreserved or $10 \%$ formalin-fixed stool specimens from symptomatic patients with suspected gastroenteritis, enteritis or colitis. The assay is intended to aid in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation and other laboratory findings. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of specific targets. The test utilizes fluorogenic gene-specific hybridization probes for detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Giardia lamblia, Cryptosporidium hominis and $C .$ . parvum, as well as Entamoeba histolytica infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decision. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis and/or colitis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

BD MAXTM System

# I. Device Description:

The BD MAX™ System and the BD MAX™ Enteric Parasite Panel are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes. The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The instrument includes a pre-warm heater as part of the sample preparation. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX™ System software automatically interprets test results.

A test result may be called as NEG (negative), POS (positive) or UNR (unresolved) based on the amplification status of the target and of the Sample Processing Control. IND (indeterminate) or INC (incomplete) results are due to BD MAX™ System failure.

Reagents provided with the BD MAX Enteric Parasite Panel

BD MAX Enteric Parasite Master Mix: Oven-dried PCR Master Mix containing TaqMan $^ \mathrm { \textregistered }$ specific molecular probe and primers along with Sample Processing Control-specific Taqman probe and primers. BD MAX Enteric Parasite Reagent Strip: Unitized reagent strip containing all liquid reagents and disposable pipette tips necessary for specimen processing and DNA extraction. BD MAX Enteric Parasite Extraction Tube: Oven-dried DNA magnetic affinity beads, protease reagents, and Sample Processing Control · BD MAX Enteric Bacterial Panel Sample Buffer Tube (with septum caps)

Equipment and materials required but not provided

· BD Pre-warm Heater   
· BD MAX PCR Cartridges   
· VWR Multi-Tube Vortex Mixer or equivalent   
· Vortex Genie 2 or equivalent   
· Nalgene $^ \mathrm { \textregistered }$ Cryogenic Vial Holder or equivalent   
· Disposable gloves, powderless $1 0 ~ \mu \mathrm { L }$ loops (BD Catalog no. 220216)

For unpreserved stool specimen types:

Dry, clean containers for the collection of stool specimens For $10 \%$ formalin-fixed stool specimen types: $10 \%$ formalin transport ( $[ 1 5 \mathrm { m L }$ )

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Gastrointestinal (GI) Panel Kit, BioFire Diagnostics, LLC

2. Predicate 510(k) number(s): K140407

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device:</td><td rowspan=1 colspan=1>Predicate:</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD MAX Enteric ParasitePanel</td><td rowspan=1 colspan=1>FilmArray GastrointestinalPanel Kit</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Detects nucleic acids ofenteric parasitic pathogensfrom the stool samples ofpatients with symptoms ofgastrointestinal infection asan aid in the diagnosis ofgastrointestinal illness.</td><td rowspan=1 colspan=1>Same(See below for targetorganism differences)</td></tr><tr><td rowspan=1 colspan=1>Target organism DNAdetected</td><td rowspan=1 colspan=1>Giardia lamblia,Cryptosporidium (C.hominis and C. parvumonly),Entamoeba histolytica</td><td rowspan=1 colspan=1>Same(See below for noteddifferences)</td></tr><tr><td rowspan=1 colspan=1>Test Interpretation</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Nucleic acids</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Multiplex nucleic acidamplification and detection</td><td rowspan=1 colspan=1>Same(See below for noteddifferences)</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BD MAX Enteric ParasitePanel</td><td colspan="1" rowspan="1">FilmArray GastrointestinalPanel Kit</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">BD MAX Instrument</td><td colspan="1" rowspan="1">FilmArray Instrument</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Unpreserved or 10%formalin-fixed stoolspecimens</td><td colspan="1" rowspan="1">Cary Blair stool specimens</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Giardia lamblia,Cryptosporidium (C.</td><td colspan="1" rowspan="1">Campylobacter (C.jejuni/C. coli/C.</td></tr><tr><td></td><td></td><td colspan="1" rowspan="1">upsaliensis), Clostridiumdifficile (C. difficile) toxinA/B, Plesiomonasshigelloides, Salmonella,Vibrio (V.parahaemolyticus/V.vulnificus/ V. cholerae),including specificidentification of Vibriocholera, Yersiniaenterocolitica,EnteroaggregativeEscherichia coli (EAEC),EnteropathogenicEscherichia coli (EPEC),EnterotoxigenicEscherichia coli (ETEC)lt/st, Shiga-like toxin-producing Escherichia coli(STEC) stx1/stx2(including specificidentification of the E. coli0157 serogroup withinSTEC), Shigella/Enteroinvasive Escherichiacoli (EIEC), Cyclosporacayetanensis,Cryptosporidium (genusclaim), Adenovirus F40/41,Astrovirus,NorovirusGI/GII, Rotavirus A,Sapovirus (Genogroups I,II, IV, and V)</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Amplification: RT-PCRDetection: fluogenic targetspecific hybridization</td><td colspan="1" rowspan="1">Amplification: Nestedmultiplex PCRDetection: high resolutionmelting analysis</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

Stool specimens are collected from patients and transported to the laboratory in a clean container, unpreserved or $10 \%$ formalin-fixed. The specimen is vortexed and a $1 0 \mu \mathrm { L }$ loop is inserted to the depth of the loop into the specimen, and expressed using a swirling motion into a BD MAX Sample Buffer Tube. The Sample Buffer Tube is closed with a septum cap and then heated on the BD Pre-warm Heater to facilitate lysis of the parasite organisms. The Sample Buffer Tube is then vortexed and transferred to the BD MAX System. Once the work list is generated and the sample is loaded on the BD MAX instrument with a BD MAX Enteric Parasite Panel Unitized Reagent Strip and PCR Cartridge, the run is started and no further operator intervention is required. The BD MAX System automates sample preparation, including target organism lysis, DNA extraction and concentration, reagent rehydration, target nucleic acid sequence amplification and detection using real-time PCR. The interpretation of the signal is performed automatically by the BD MAX System. The assay also includes a Sample Processing Control that is provided in the Extraction Tube and subjected to extraction, concentration and amplification steps. The Sample Processing Control is incorporated into the lysis, extraction, concentration and amplification steps to monitor for the presence of potential inhibitory substances as well as system or reagent failures.

Following enzymatic cell lysis at an elevated temperature, the released nucleic acids are captured on magnetic affinity beads. The beads, with the bound nucleic acids, are washed and the nucleic acids are eluted by heat and high $\mathsf { p H }$ in Elution Buffer. Eluted DNA is neutralized and transferred to the Master Mix Tube to rehydrate the PCR reagents. After rehydration, the BD MAX System dispenses a fixed volume of PCR-ready solution into the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed by the system prior to initiating PCR to contain the amplification mixture thus preventing evaporation and contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes, labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect amplicons for enteric parasite targets and the Sample Processing Control in four different optical channels of the BD MAX System: Giardia lamblia target amplicons are detected in the FAM channel, Cryptosporidium parvum/hominis target amplicons are detected in the ROX channel, Entamoeba histolytica target amplicons are detected in the VIC channel and Sample Processing Control amplicons are detected in the $\mathrm { C y } 5 . 5$ channel. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at each cycle and interprets the data at the end of the program to report the final results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Site-to-site reproducibility

Site-to-site reproducibility was evaluated for the BD MAX Enteric Parasite Panel at three clinical sites by two operators at each site for eight days. Each panel member was tested in triplicate for each run resulting in a total of 144 data points per panel member. The panel members were prepared as organism mixes of $G$ . lamblia cysts, C. parvum oocysts, and $E .$ . histolytica trophozoites spiked at varying concentrations per target in negative unpreserved stool matrix:

Moderate Positive (MP): 2 to ${ 5 } \mathrm { x }$ LOD Low Positive (LP): 0.6 to 1.2x LOD High Negative (HN): 0.1 to 0.5x LOD True Negative (TN): No Target

Three replicates of each positive mix and true negative sample were included in each run. Panel members were prepared by spiking SBTs with organism mixes. Testing panels were shipped to testing sites and prior to testing, the user expressed a $1 0 \mu \mathrm { l }$ loop of pooled stool matrix (previously determined to be negative for all analytes by the BD MAX Enteric Bacterial Panel) into each sample.

Site-to-site reproducibility results are presented as percent agreement with expected result $( \% )$ , (ratio of count with expected result/total count), and $[ 9 5 \%$ confidence interval] in Table 1.

Table 1. Site-to-Site Reproducibility Overall Results   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Cryptosporidiumparvum</td><td rowspan=1 colspan=1>Entamoebahistolytica</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>48.6% (70/144)[40.6%, 56.7%]</td><td rowspan=1 colspan=1>38.2% (55/144)[30.7%, 46.3%]</td><td rowspan=1 colspan=1>47.2% (68/144)[39.2%, 55.3%]</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>97.2% (140/144)[93.1%, 98.9%]</td><td rowspan=1 colspan=1>97.2% (140/144)[93.1%, 98.9%]</td><td rowspan=1 colspan=1>98.6% (142/144)[95.1%, 99.6%]</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td><td rowspan=1 colspan=1>100.0% (144/144)[97.4%, 100.0%]</td></tr></table>

The qualitative and quantitative reproducibility across sites and by target is presented below in Tables 2 through 7. Ct.Score is an internal criterion used to determine final assay results and was selected as an additional means of assessing assay reproducibility.

Table 2. Giardia lamblia Site-to-Site Qualitative Reproducibility   

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>1</td><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>47.9</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>52.1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>54.2</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>52.1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>47.9</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>48.6</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>51.4</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 3. Giardia lamblia Site-to-Site Quantitative Reproducibility   

<table><tr><td rowspan=1 colspan=3>Ct.score</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>4.5%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.8%</td></tr></table>

Table 4. C. parvum Site-to-Site Qualitative Reproducibility   

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>1</td><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>64.6</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>62.5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>41.7</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>58.3</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>61.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 5. C. parvum Site-to-Site Quantitative Reproducibility   

<table><tr><td rowspan=1 colspan=3>Ct.score</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>2.9%</td></tr></table>

Table 6. E. histolytica Site-to-Site Qualitative Reproducibility   

<table><tr><td rowspan=4 colspan=1>Category</td><td rowspan=1 colspan=12>SITE</td><td rowspan=2 colspan=4>Total</td></tr><tr><td rowspan=1 colspan=4>1</td><td rowspan=1 colspan=4>2</td><td rowspan=1 colspan=4>3</td></tr><tr><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=2>Incorrect</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>54.2</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>54.2</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>41.7</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>58.3</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>47.2</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>52.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 7. E. histolytica Site-to-Site Quantitative Reproducibility   

<table><tr><td rowspan=1 colspan=3>Ct.score</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>2.34</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>2.80</td><td rowspan=1 colspan=1>9.2%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>23.5</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.9%</td></tr></table>

# Lot to Lot Reproducibility

Lot-to-Lot reproducibility was evaluated with three replicates of each panel member and three production lots at a single site, with two user/runs per day performed over five days. The panels used were the same as described for Site-to-Site reproducibility. Results from five days of the Site-to-Site study were used to comprise data for one lot of reagents for the Lot-to-Lot study.

The overall Lot-to-Lot reproducibility percent agreement was $100 \%$ for the TN and MP categories for all targets, and ranged from $4 8 . 9 \%$ to $5 5 . 6 \%$ and $9 7 . 8 \%$ to $9 8 . 9 \%$ for the HN and LP categories, respectively. Lot-to-Lot reproducibility is presented in Tables 8 and 9 below.

Table 8. Lot-to-Lot Qualitative Reproducibility   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>Correct</td><td rowspan=2 colspan=1>Total</td><td rowspan=2 colspan=1>% Correct</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LowerCI</td><td rowspan=1 colspan=1>UpperCI</td></tr><tr><td rowspan=4 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>TN*</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>HN**</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>48.9%</td><td rowspan=1 colspan=1>38.8%</td><td rowspan=1 colspan=1>59.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=4 colspan=1>C. parvum</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>55.6%</td><td rowspan=1 colspan=1>45.3%</td><td rowspan=1 colspan=1>65.4%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=4 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>39.9%</td><td rowspan=1 colspan=1>60.1%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

\* TNs contained no organisms. $\%$ Correct” correlates to the percent of negative results.

\*\* HNs are dilutions of the LoD designed to produce results that are negative for $20 \%$ to $80 \%$ of replicates. $\%$ Correct” correlates to the percent of negative results.

Table 9. Lot-to-Lot Quantitative Reproducibility   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>Ct.score</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>Between Daywithin Lot</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>4.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>2.3%</td></tr><tr><td rowspan=3 colspan=1>C. parvum</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.59</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>2.25</td><td rowspan=1 colspan=1>6.1%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>5.2%</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>2.31</td><td rowspan=1 colspan=1>7.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>2.52</td><td rowspan=1 colspan=1>7.9%</td></tr><tr><td rowspan=3 colspan=1>E.histolytica</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>2.70</td><td rowspan=1 colspan=1>8.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>2.83</td><td rowspan=1 colspan=1>9.2%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>5.6%</td></tr></table>

# Precision

Within-laboratory precision was evaluated at one site over 12 days with two runs per day (one each by two technologists) using the same panel as described for the site-tosite reproducibility study. Within-laboratory precision results are presented in Tables 10 and 11 below.

Table 10. Precision Study Qualitative Results   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>G. lamblia %, (n),[95% CI]</td><td rowspan=1 colspan=1>C. parvum %, (n),[95% CI]</td><td rowspan=1 colspan=1>E. histolytica %, (n),[95% CI]</td></tr><tr><td rowspan=1 colspan=1>TN*</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0%]</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0% ]</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0% ]</td></tr><tr><td rowspan=1 colspan=1>HN*</td><td rowspan=1 colspan=1>48.6% (35/72)[37.4%,,59.9%</td><td rowspan=1 colspan=1>41.7% (30/72)[31.0%, 53.2% ]</td><td rowspan=1 colspan=1>37.5% (27/72)[27.2%,49.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>98.6% (71/72)[92.5%, 99.8% ]</td><td rowspan=1 colspan=1>98.6% (71/72)[92.5%, 99.8% ]</td><td rowspan=1 colspan=1>98.6% (71/72)[92.5%, 99.8% ]</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0% ]</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0% ]</td><td rowspan=1 colspan=1>100.0% (72/72)[94.9%, 100.0% ]</td></tr></table>

\*For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Table 11. Precision Study Quantitative Results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>Ct.score</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithin Day</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=3 colspan=1>C. parvum</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>5.5%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=3 colspan=1>E.histolytica</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>2.56</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>8.6%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>3.2%</td></tr></table>

b. Linearity/assay reportable range: Not applicable

Positive and Negative External Controls:

External control materials are not provided by BD; however, Quality Control strains and procedures are recommended in the package insert. External positive controls are intended to monitor for substantial reagent failure, and External negative controls are used to detect reagent or environmental contamination (or carry-over) by target nucleic acids. External positive and negative controls are not used by the BD MAX System software for the purpose of sample test result interpretation. External controls are treated as patient samples. Various types of external controls are recommended to allow the user to select the most appropriate for their laboratory quality control program. Due to lack of commercial availability of organism oocysts/cysts for all targets and burdensome culture methods for the trophozoite form of the target organisms, BD recommends genomic DNA preparations according to the listing below:

Commercially available external positive control materials Giardia intestinalis (Lambl) (ATCC 3088D) $( 2 , 4 7 5 ~ \mathrm { p g / m L }$ gDNA) Cryptosporidium parvum (ATCC PRA-67D) $( 1 1 , 0 2 5 \ : \mathrm { p g / m L }$ gDNA) Entamoeba histolytica (ATCC 30459D) $( 1 5 0 \mathrm { p g / m L }$ gDNA)

Recommended external negative control - A non-inoculated BD MAX™ Enteric Parasite Panel SBT

During the clinical study, the above mentioned external positive and negative controls were included in each run. A failed control invalidated the entire run. In the case of a failed control, the testing was repeated for all samples in the run from the appropriately stored SBTs, along with a new set of controls. A total of 253 external positive controls and 253 external negative controls were tested resulting in valid expected results for 242 $( 9 5 . 6 \% )$ and 240 $( 9 4 . 9 \% )$ of positive controls and negative controls, respectively. Three runs were invalidated due to Incomplete results $( 1 . 2 \% )$ , and one run was invalidated due to an Indeterminate result $( 7 . 4 \% )$ . Positive control testing resulted in five valid but incorrect results $( 2 . 0 \% )$ and two Unresolved results $( 0 . 8 \% )$ . Negative control testing resulted in three Unresolved results $( 1 . 2 \% )$ .

Internal Sample Processing Control:

Each BD MAX Enteric Bacterial Extraction Tube contains a Sample Processing Control (SPC) which is a plasmid containing a synthetic target DNA sequence. The SPC monitors the efficiency of DNA capture, washing and elution during the sample processing steps, as well as the efficiency of DNA amplification and detection during PCR analysis. If the SPC result fails to meet the acceptance criteria, the test result of will be reported as Unresolved (UNR). An Unresolved result is indicative of sampleassociated inhibition or reagent failure. The user is instructed to repeat any sample

reported as Unresolved.

Sample Processing Control Effectiveness Study

A Sample Processing control effectiveness study was performed to evaluate the effectiveness of the SPC to monitor for substantial reagent or process failure. This study was designed to test for failure in the extraction process, failure in PCR amplification and/or failure due to the presence of PCR inhibitors.

Twenty-four replicates were tested for each of the following conditions:

Failure in PCR amplification was tested by replacing the BD MAX Enteric Bacterial Panel Master Mix tube with an empty Snap Tube (expected result $=$ IND).   
Failure in the Extraction process was tested by replacing the BD MAX Enteric Bacterial Panel Extraction tube with an empty Snap Tube (expected result $=$ UNR).   
Failure due to the presence of PCR inhibitors was tested by placing $1 0 0 \mathrm { m M }$ EDTA solution into an empty Snap Tube of the BD MAX System test rack prior to beginning a run (expected result ${ \bf \Phi } = \mathrm { U N R } { \bf \Phi }$ ).   
Valid SPC control (expected result $=$ positive SPC)

Each test condition produced the expected SPC detection or failures for all replicates.

# Specimen Stability Study:

Stability studies were performed to demonstrate that target DNA is stable in both unpreserved and $10 \%$ formalin-fixed stool specimens prior to testing with the BD MAX EPP. The claimed storage conditions for either unpreserved or $10 \%$ formalinfixed stool are at $2 - 2 5 ^ { \circ } \mathrm { C }$ for a maximum of 48 hours and at $2 - 8 \ ^ { \circ } \mathrm { C }$ for up to 120 hours (five days). The claimed storage conditions for samples inoculated into Sample Buffer Tubes are at $2 - 2 5 \ ^ { \circ } \mathrm { C }$ for a maximum of 48 hours and at $2 - 8 \ ^ { \circ } \mathrm { C }$ for up to 120 hours.

C. parvum, G. lamblia, and $E$ . histolytica target organisms were spiked into negative unpreserved and fixed stool matrices at a final target concentration of 2X LoD. Contrived specimens were also inoculated into SBTs and stability was evaluated either before or after the pre-warm step of the BD MAX EPP procedure. Testing included 24 replicates for each test condition at baseline, 24, and 48 hours for storage claims at $2 5 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ , and at 48 hours and 120 hours for storage claims at $5 ^ { \circ } C \pm 3 ^ { \circ } C$

The specimen stability studies support the claimed storage times and conditions for target DNA in both unpreserved and $10 \%$ formalin fixed stool. The SBT stability studies support the claimed storage times and conditions for target DNA in SBT either before or after the pre-warm step.

# d. Detection limit:

Limit of Detection (LoD) was determined in an analytical study: Stock organisms were purified and quantified for inclusion in this study. Individual inoculating loops were dipped into each organism preparation and were then transferred to a Sample Buffer Tube, already containing fecal matrix (unpreserved or $10 \%$ formalin-fixed) that was pre-determined to be negative for all the targets detected by the BD MAX Enteric Parasite Panel. Each organism was tested with a minimum of 30 replicates per specimen type (unpreserved or $10 \%$ formalin-fixed), by a single operator, using three different production lots of the BD MAX Enteric Parasite Panel kits. Each LOD, defined as the lowest concentration at which greater than $9 5 \%$ of all replicates are expected to test positive, ranged from 10.67 to 160.17 organisms per milliliter (in SBT) and 1,601 to 24,026 organisms per milliliter (in stool) for unpreserved specimens, and 10.04 to 154.25 organisms per milliliter (in SBT) and 6,024 to 92,550 (in stool) for $10 \%$ formalin-fixed specimens.

Results are shown in the following table.

Table 12. BD MAX Enteric Parasite Panel Limit of Detection   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>10% formalin fixed</td></tr><tr><td rowspan=1 colspan=3>Cryptosporidium parvum</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in SBTa)[95% confidence interval]</td><td rowspan=1 colspan=1>160.17 [96.93  264.22]</td><td rowspan=1 colspan=1>154.25 [100.46  236.57]</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in stool)[95% confidence interval]</td><td rowspan=1 colspan=1>24,026 [14,540  39,633]</td><td rowspan=1 colspan=1>92,550 [60,276  141,942]</td></tr><tr><td rowspan=1 colspan=3>Giardia lamblia</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in SBT)[95% confidence interval]</td><td rowspan=1 colspan=1>10.67 [5.69  15.66]</td><td rowspan=1 colspan=1>10.04 [4.83 - 15.25]</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in stool)[95% confidence interval]</td><td rowspan=1 colspan=1>1,601 [854  2,349]</td><td rowspan=1 colspan=1>6,024 [2,898  9,150]</td></tr><tr><td rowspan=1 colspan=3>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in SBT)[95% confidence interval]</td><td rowspan=1 colspan=1>16.79 [11.97  23.42]</td><td rowspan=1 colspan=1>15.52 [10.99  21.76]</td></tr><tr><td rowspan=1 colspan=1>LoD (organism/mL in stool)[95% confidence interval]</td><td rowspan=1 colspan=1>2,519 [1,796  3,570]</td><td rowspan=1 colspan=1>9,300 [6,600  13,080]</td></tr></table>

a SBT: Sample Buffer Tube

# Inclusivity/reactivity

Inclusivity was assessed through testing clinically relevant and geographically diverse organism isolates for each of the assay target organisms. Twenty-two isolates representing the three assay targets were tested (11 isolates of Giardia lamblia and Giardia intestinalis, 10 isolates of Entamoeba histolytica, and one isolate of Cryptosporidium hominis). The isolates were tested at 2X the LOD in unpreserved stool matrix either as target pools containing two assay targets each or individually (Table 13). The BD MAX EPP detected all isolates.

Genomic DNA extracts from 13 individual human clinical stool specimens (nine $C .$ . hominis, three C. parvum, and one $C .$ . ubiquitum) collected during routine clinical testing or from a Cryptosporidiosis outbreak in 2005 were also tested (Table 14). The assay identified all 13 genomic DNA extracts as being positive for $C .$ . hominis/parvum.

Additionally, genomic DNA and prepared genomic DNA libraries were also evaluated. One genomic DNA representative of each BD MAX Enteric Parasite Panel target was tested at 2X the LOD for the assay (Table 15). Due to the lack of availability of Cryptosporidium parvum and Cryptosporidium hominis organisms, commercially available genomic DNA libraries prepared from nine distinct Cryptosporidium parvum isolates were also evaluated as supplementary information. The BD MAX EPP detected all samples at the concentrations tested.

Table 13. Isolates Evaluated for Inclusivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ATCC ID</td><td rowspan=1 colspan=1>Strain/Designation</td><td rowspan=1 colspan=1>Geographic Origin</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>PRA-242</td><td rowspan=1 colspan=1>CM</td><td rowspan=1 colspan=1>Florida</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>PRA-244</td><td rowspan=1 colspan=1>Mario</td><td rowspan=1 colspan=1>U.S.</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>PRA-247</td><td rowspan=1 colspan=1>DAN</td><td rowspan=1 colspan=1>U.S.</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>PRA-249</td><td rowspan=1 colspan=1>BE-1</td><td rowspan=1 colspan=1>Canada</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>30888</td><td rowspan=1 colspan=1>Portland-1</td><td rowspan=1 colspan=1>Oregon</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>30957</td><td rowspan=1 colspan=1>WB</td><td rowspan=1 colspan=1>Maryland</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50114</td><td rowspan=1 colspan=1>KS</td><td rowspan=1 colspan=1>Pennsylvania</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50137</td><td rowspan=1 colspan=1>New Orleans-1</td><td rowspan=1 colspan=1>Louisiana</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50581</td><td rowspan=1 colspan=1>GS clone H7</td><td rowspan=1 colspan=1>Alaska</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50584</td><td rowspan=1 colspan=1>JH</td><td rowspan=1 colspan=1>West Virginia</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>50585</td><td rowspan=1 colspan=1>AB</td><td rowspan=1 colspan=1>Peru</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30190</td><td rowspan=1 colspan=1>HB-301:NIH</td><td rowspan=1 colspan=1>Burma</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>50541</td><td rowspan=1 colspan=1>HK-9 Clone 1</td><td rowspan=1 colspan=1>Korea</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>PRA-358</td><td rowspan=1 colspan=1>F22</td><td rowspan=1 colspan=1>U.S.</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>50007</td><td rowspan=1 colspan=1>DKB</td><td rowspan=1 colspan=1>London</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30890</td><td rowspan=1 colspan=1>HM-3:IMSS</td><td rowspan=1 colspan=1>Mexico</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30889</td><td rowspan=1 colspan=1>H-458:CDC</td><td rowspan=1 colspan=1>Asia</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30458</td><td rowspan=1 colspan=1>200:NIH</td><td rowspan=1 colspan=1>U.S.</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30459</td><td rowspan=1 colspan=1>HM-1:IMSS</td><td rowspan=1 colspan=1>Mexico</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>PRA-357</td><td rowspan=1 colspan=1>IP:1182:2</td><td rowspan=1 colspan=1>Honduras</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>50738</td><td rowspan=1 colspan=1>Rahman</td><td rowspan=1 colspan=1>U.K.</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>502 (Tufts)</td><td rowspan=1 colspan=1>TU</td><td rowspan=1 colspan=1>U.S.</td></tr></table>

Table 14. Clinical Specimen DNA Extracts: Cryptosporidium spp. Positive   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Geographic Origin (Year)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Routine Clinical (2006)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Routine Clinical (2010)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Routine Clinical (2011)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Routine Clinical (2012)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IeA11G3T3</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>IbA10G2</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium ubiquitum*</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Spray Park (2005)</td></tr></table>

\* C. ubiquitum has been previously identified in the literature as $C$ . parvum genotype cervine and in 2010 was reclassified as C. ubiquitum (Fayer, et al., 2010; Ong, et al., 2002).

Table 15. Genomic DNA Evaluated for Inclusivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ATCC ID</td><td rowspan=1 colspan=1>Isolate/Strain</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>PRA-67D</td><td rowspan=1 colspan=1>Iowa</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>30888D</td><td rowspan=1 colspan=1>Oregon</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>30459D</td><td rowspan=1 colspan=1>Mexico</td></tr></table>

e. Analytical specificity:

Cross reactivity:

The BD MAX Enteric Parasite Panel was performed on samples containing phylogenetically related species and other organisms (bacteria, viruses, parasites and yeast) likely to be found in stool specimens. Potentially cross-reacting organisms are listed in Tables 16 - 18. Results are summarized below:

Six (6) out of 6 Entamoeba spp. (Entamoeba species other than $E$ . histolytica) produced negative results with the BD MAX Enteric Parasite Panel. The organisms were tested directly from stock at a 1:10 dilution to obtain the highest possible concentration in the Sample Buffer Tube, with concentrations ranging from $4 . 0 0 \mathrm { x } 1 0 ^ { 3 }$ organisms/mL to $\dot { 2 } . 9 0 \mathrm { x } 1 0 ^ { 5 }$ organisms/mL in the Sample Buffer Tube. $E .$ . barretti (ATCC 30996) was provided as non-titered stock. · One (1) out of 1 Cryptosporidium meleagridis strain tested at a concentration $\geq 1 \mathrm { x } 1 0 ^ { 5 }$ cysts/mL in the Sample Buffer Tube, produced positive results with the BD MAX Enteric Parasite Panel. $C .$ . meleagridis has been documented in

symptomatic human infection. (Table $1 8 ~ ^ { * }$ ). C. meleagridis will be listed as a potential cross reactant with the $C .$ . parvum/C. hominis assay target.   
· One hundred thirteen (113) out of 113 bacterial strains, tested at a concentration $\geq 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ in the Sample Buffer Tube, produced negative results with the BD MAX Enteric Parasite Panel. Fifteen (15) out of 15 viruses, produced negative results with the BD MAX Enteric Parasite Panel. Thirteen (13) were tested directly from stock at a 1:10 dilution to obtain the highest possible concentration in the Sample Buffer Tube, with concentrations ranging from $1 . 6 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } - 8 . 9 \mathrm { x } \dot { 1 } 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 . }$ . Human Papillomavirus was tested as plasmid in Escherichia coli and Rotavirus was tested as a high titer qualitative standard. Five (5) out of 5 phylogenetically unrelated parasites, tested at a concentration $\geq 1 \mathrm { x } 1 0 ^ { 5 }$ organisms $/ \mathrm { m L }$ in the Sample Buffer Tube, produced negative results with the BD MAX Enteric Parasite Panel.   
Two (2) out of 2 Candida spp. tested at a concentration $\geq 1 \mathrm { x } 1 0 ^ { 6 }$ organisms/mL in the Sample Buffer Tube, produced negative results with the BD MAX Enteric Parasite Panel.   
· Three (3) enteric organisms representing each target of the BD MAX Enteric Parasite Panel were tested, with results as follows: $\bigcirc$ One (1) of 1 Cryptosporidium spp.; Cryptosporidium parvum tested at a concentration $\geq \mathrm { i } \mathbf { x } 1 0 ^ { 5 }$ cysts/mL in the Sample Buffer Tube, produced positive results for Cryptosporidium and negative results for all other targets with the BD MAX Enteric Parasite Panel. o One (1) of 1 Giardia spp.; Giardia lamblia tested at a concentration $\geq$ $\mathrm { 1 x 1 0 ^ { 5 } }$ cysts/mL in the Sample Buffer Tube, produced positive results for Giardia and negative results for all other targets with the BD MAX Enteric Parasite Panel. o One (1) of 1 Entamoeba spp.; Entamoeba histolytica tested at a concentration $\geq 1 \mathrm { x } 1 0 ^ { 5 }$ cysts/mL in the Sample Buffer Tube, produced positive results for Entamoeba and negative results for all other targets with the BD MAX Enteric Parasite Panel.

Table 16. Potential Cross-reactant Bacteria and Yeast   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="3">Genus</td><td colspan="1" rowspan="3">Species</td><td colspan="1" rowspan="3">ID</td><td colspan="2" rowspan="3"></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="2"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia</td><td colspan="1" rowspan="1">defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="7">Escherichia</td><td colspan="1" rowspan="4">coli</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td><td></td><td></td><td colspan="1" rowspan="1">ATCC 8739</td></tr><tr><td colspan="1" rowspan="1">Iwoffii</td><td colspan="1" rowspan="1">ATCC 17925</td><td></td><td></td><td colspan="1" rowspan="1">ATCC 10536</td></tr><tr><td colspan="1" rowspan="1">Aeromonas</td><td colspan="1" rowspan="1">hydrophila</td><td colspan="1" rowspan="1">ATCC 49847</td><td></td><td></td><td colspan="1" rowspan="1">ATCC 33605</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes</td><td colspan="1" rowspan="1">faecalis subsp.faecalis</td><td colspan="1" rowspan="1">ATCC 8750</td><td></td><td></td><td colspan="1" rowspan="1">fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus</td><td colspan="1" rowspan="1">tetradius</td><td colspan="1" rowspan="1">ATCC 35098</td><td></td><td></td><td colspan="1" rowspan="1">hermannii</td><td colspan="1" rowspan="1">ATCC 33650</td></tr><tr><td colspan="1" rowspan="1">Arcobacter</td><td colspan="1" rowspan="1">butzleri</td><td colspan="1" rowspan="1">ATCC 49616</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td></tr><tr><td colspan="1" rowspan="1">Arcobacter</td><td colspan="1" rowspan="1">cryaerophilus</td><td colspan="1" rowspan="1">ATCC 43157</td><td></td><td></td><td colspan="1" rowspan="1">Fusobacterium</td><td colspan="1" rowspan="1">varium</td><td colspan="1" rowspan="1">ATCC 27725</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">cereus</td><td colspan="1" rowspan="1">ATCC 49064</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Gardnerella</td><td colspan="1" rowspan="1">vaginalis</td><td colspan="1" rowspan="1">ATCC 14019</td></tr><tr><td colspan="1" rowspan="3">Bacteroides</td><td colspan="1" rowspan="1">caccae</td><td colspan="1" rowspan="1">ATCC 43185</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Gemella</td><td colspan="1" rowspan="1">morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td></tr><tr><td colspan="1" rowspan="1">merdae</td><td colspan="1" rowspan="1">ATCC 43184</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Hafnia</td><td colspan="1" rowspan="1">alvei</td><td colspan="1" rowspan="1">ATCC 11604</td></tr><tr><td colspan="1" rowspan="1">stercoris</td><td colspan="1" rowspan="1">ATCC 43183</td><td colspan="2" rowspan="2">b</td><td colspan="1" rowspan="2">Helicobacter</td><td colspan="1" rowspan="1">fennelliae</td><td colspan="1" rowspan="1">ATCC 35683</td></tr><tr><td colspan="1" rowspan="2">Bifidobacterium</td><td colspan="1" rowspan="1">adolescentis</td><td colspan="1" rowspan="1">ATCC 15706</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">pylori</td><td colspan="1" rowspan="1">ATCC 43504</td></tr><tr><td colspan="1" rowspan="1">longum</td><td colspan="1" rowspan="1">ATCC 15707</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="2">b</td><td colspan="1" rowspan="2">Klebsiella</td><td colspan="1" rowspan="1">oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td></tr><tr><td colspan="1" rowspan="11">Camplylobacter</td><td colspan="1" rowspan="1">coli</td><td colspan="1" rowspan="1">ATCC 43134</td><td></td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">ATCC 33495</td></tr><tr><td colspan="1" rowspan="1">concisus</td><td colspan="1" rowspan="1">CCUG 17580</td><td colspan="2" rowspan="2">b</td><td colspan="1" rowspan="2">Lactobacillus</td><td colspan="1" rowspan="1">acidophilus</td><td colspan="1" rowspan="1">ATCC 4355</td></tr><tr><td colspan="1" rowspan="1">curvus</td><td colspan="1" rowspan="1">CCUG 47528</td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="1">reuteri</td><td colspan="1" rowspan="1">ATCC 23272</td></tr><tr><td colspan="1" rowspan="1">fetus subsp.fetus</td><td colspan="1" rowspan="1">ATCC 27374</td><td colspan="1" rowspan="2">Lactococcus</td><td colspan="1" rowspan="2">lactis</td><td colspan="1" rowspan="1">ATCC 15346</td></tr><tr><td colspan="1" rowspan="1">fetus subsp.venerealis</td><td colspan="1" rowspan="1">ATCC 19438</td><td colspan="1" rowspan="1">ATCC 49032</td></tr><tr><td colspan="1" rowspan="1">gracilis</td><td colspan="1" rowspan="1">ATCC 33236</td><td></td><td></td><td colspan="1" rowspan="1">Leminorella</td><td colspan="1" rowspan="1">grimontii</td><td colspan="1" rowspan="1">ATCC 33999</td></tr><tr><td colspan="1" rowspan="1">hominis</td><td colspan="1" rowspan="1">ATCC BAA-381</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">Listeria</td><td colspan="1" rowspan="1">grayi</td><td colspan="1" rowspan="1">ATCC 19120</td></tr><tr><td colspan="1" rowspan="1">jejuni</td><td colspan="1" rowspan="1">ATCC 43429</td><td></td><td colspan="1" rowspan="1">innocua</td><td colspan="1" rowspan="1">ATCC 33090</td></tr><tr><td colspan="1" rowspan="1">lari</td><td colspan="1" rowspan="1">ATCC 43675</td><td></td><td colspan="1" rowspan="1">monocytogenes</td><td colspan="1" rowspan="1">ATCC 19115</td></tr><tr><td colspan="1" rowspan="1">rectus</td><td colspan="1" rowspan="1">ATCC 33238</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Morganella</td><td colspan="1" rowspan="1">morganii</td><td colspan="1" rowspan="1">ATCC 25830</td></tr><tr><td colspan="1" rowspan="1">upsaliensis</td><td colspan="1" rowspan="1">ATCC 49815</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Peptoniphilus</td><td colspan="1" rowspan="1">asaccharolyticus</td><td colspan="1" rowspan="1">ATCC 14963</td></tr><tr><td colspan="1" rowspan="2">Candida</td><td colspan="1" rowspan="1">albicans</td><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Peptostreptococcus</td><td colspan="1" rowspan="1">anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">catenulate</td><td colspan="1" rowspan="1">ATCC 18821</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Plesiomonas</td><td colspan="1" rowspan="1">shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Cedecea</td><td colspan="1" rowspan="1">davisae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Porphyromonas</td><td colspan="1" rowspan="1">asaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Chlamydia</td><td colspan="1" rowspan="1">trachomatis</td><td colspan="1" rowspan="1">ATCC VR-879</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Prevotella</td><td colspan="1" rowspan="1">melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">amalonaticus</td><td colspan="1" rowspan="1">ATCC 25405</td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="3">Proteus</td><td colspan="1" rowspan="1">mirabilis</td><td colspan="1" rowspan="1">ATCC 29906</td></tr><tr><td colspan="1" rowspan="1">fruendii</td><td colspan="1" rowspan="1">ATCC 33128</td><td colspan="1" rowspan="1">penneri</td><td colspan="1" rowspan="1">ATCC 35198</td></tr><tr><td colspan="1" rowspan="1">koseri</td><td colspan="1" rowspan="1">ATCC 27156</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">vulgaris</td><td colspan="1" rowspan="1">ATCC 13315</td></tr><tr><td colspan="1" rowspan="1">sedlakii</td><td colspan="1" rowspan="1">ATCC 51115</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">Providencia</td><td colspan="1" rowspan="1">alcalifaciens</td><td colspan="1" rowspan="1">ATCC 27971</td></tr><tr><td colspan="1" rowspan="7">Clostridium</td><td colspan="1" rowspan="6">difficile</td><td colspan="1" rowspan="1">ATCC 17858</td><td></td><td colspan="1" rowspan="1">rettgeri</td><td colspan="1" rowspan="1">ATCC 29944</td></tr><tr><td colspan="1" rowspan="1">ATCC 43598</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">CCUG 8864- ATCC9689</td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="2">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td></tr><tr><td colspan="1" rowspan="1">ATCC 43255</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1805</td><td></td><td></td><td colspan="1" rowspan="1">Ruminococcus</td><td colspan="1" rowspan="1">bromii</td><td colspan="1" rowspan="1">ATCC 27255</td></tr><tr><td colspan="1" rowspan="1">ATCC 43593</td><td colspan="2" rowspan="2">b</td><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">typhimurium</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">perfringens</td><td colspan="1" rowspan="1">ATCC 10543</td><td colspan="1" rowspan="1">enteriditis</td><td colspan="1" rowspan="1">ATCC 13076</td></tr><tr><td colspan="1" rowspan="1">Collinsella</td><td colspan="1" rowspan="1">aerofaciens</td><td colspan="1" rowspan="1">ATCC 35085</td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">liquefaciens</td><td colspan="1" rowspan="1">ATCC 35551</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">genitalium</td><td colspan="1" rowspan="1">ATCC 33030</td><td colspan="1" rowspan="1">marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio</td><td colspan="1" rowspan="1">piger</td><td colspan="1" rowspan="1">ATCC 29098</td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="2">Shigella</td><td colspan="1" rowspan="1">sonnei</td><td colspan="1" rowspan="1">BD ENF 7142</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella</td><td colspan="1" rowspan="1">tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">flexneri</td><td colspan="1" rowspan="1">ATCC 700930</td></tr><tr><td colspan="1" rowspan="1">Eggerthella</td><td colspan="1" rowspan="1">lenta</td><td colspan="1" rowspan="1">ATCC 25559</td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="2">Staphylococcus</td><td colspan="1" rowspan="1">aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="2">Enterobacter</td><td colspan="1" rowspan="1">aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td></tr><tr><td colspan="1" rowspan="1">cloacae</td><td colspan="1" rowspan="1">ATCC 35030</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Stenotrophomonas</td><td colspan="1" rowspan="1">maltophilia</td><td colspan="1" rowspan="1">ATCC 13637</td></tr><tr><td colspan="1" rowspan="7">Enterococcus</td><td colspan="1" rowspan="1">casseliflavus</td><td colspan="1" rowspan="1">ATCC 49605</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">Streptococcus</td><td colspan="1" rowspan="1">agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td></tr><tr><td colspan="1" rowspan="1">cecorum</td><td colspan="1" rowspan="1">ATCC 43198</td><td colspan="1" rowspan="1">dysgalactiae</td><td colspan="1" rowspan="1">ATCC 43078</td></tr><tr><td colspan="1" rowspan="1">dispar</td><td colspan="1" rowspan="1">ATCC 51266</td><td colspan="1" rowspan="1">intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">uberis</td><td colspan="1" rowspan="1">ATCC 19436</td></tr><tr><td colspan="1" rowspan="1">gallinarum</td><td colspan="1" rowspan="1">ATCC 49573</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Trabulsiella</td><td colspan="1" rowspan="1">guamensis</td><td colspan="1" rowspan="1">ATCC 49490</td></tr><tr><td colspan="1" rowspan="1">hirae</td><td colspan="1" rowspan="1">ATCC 49612</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Veillonella</td><td colspan="1" rowspan="1">parvula</td><td colspan="1" rowspan="1">ATCC 10790</td></tr><tr><td colspan="1" rowspan="1">raffinosus</td><td colspan="1" rowspan="1">ATCC 49427</td><td colspan="1" rowspan="2">b</td><td colspan="1" rowspan="2">Vibrio</td><td colspan="1" rowspan="1">cholerae</td><td colspan="1" rowspan="1">BD ENF 13503</td></tr><tr><td colspan="1" rowspan="3">Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="1" rowspan="1">ATCC 25922</td><td colspan="1" rowspan="1">parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1">coli 0157 stx 1</td><td colspan="1" rowspan="1">BD RD012313-01</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">Yersinia</td><td colspan="1" rowspan="1">bercovieri</td><td colspan="1" rowspan="1">ATCC 43970</td></tr><tr><td colspan="1" rowspan="1">coli O157 stx 2</td><td colspan="1" rowspan="1">BD RD092612-01</td><td colspan="1" rowspan="1">enterocolitica</td><td colspan="1" rowspan="1">ATCC 9610</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">rohdei</td><td colspan="1" rowspan="1">ATCC 43380</td></tr></table>

Table 17. Potential Cross-reactant Viruses   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 2</td><td rowspan=1 colspan=1>ATCC VR-680</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 14</td><td rowspan=1 colspan=1>ATCC VR-15</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 40</td><td rowspan=1 colspan=1>ATCC VR-931</td></tr><tr><td rowspan=1 colspan=1>Adenovirus type 41</td><td rowspan=1 colspan=1>ATCC VR-930</td></tr><tr><td rowspan=1 colspan=1>Coxsackie A9</td><td rowspan=1 colspan=1>ATCC VR-186</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B1</td><td rowspan=1 colspan=1>ATCC VR-687</td></tr><tr><td rowspan=1 colspan=1>HHV-5 Cytomegalovirus</td><td rowspan=1 colspan=1>ATCC VR-538</td></tr><tr><td rowspan=1 colspan=1>Enterovirus type 69</td><td rowspan=1 colspan=1>ATCC VR-785</td></tr><tr><td rowspan=1 colspan=1>Human Papillomavirus Type 16</td><td rowspan=1 colspan=1>ATCC 45113</td></tr><tr><td rowspan=1 colspan=1>Human Papillomavirus Type 18</td><td rowspan=1 colspan=1>ATCC 45152</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus I</td><td rowspan=1 colspan=1>ATCC VR-539</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus II</td><td rowspan=1 colspan=1>ATCC VR-734</td></tr><tr><td rowspan=1 colspan=1>Norovirus I</td><td rowspan=1 colspan=1>0810086CF</td></tr><tr><td rowspan=1 colspan=1>Norovirus II</td><td rowspan=1 colspan=1>0810087CF</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>NATROTA</td></tr></table>

Table 18. Potential Cross-reactant Parasites   

<table><tr><td rowspan=1 colspan=1>Parasite</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>ATCC 50608</td></tr><tr><td rowspan=1 colspan=1>Encephalitozoon intestinalis</td><td rowspan=1 colspan=1>ATCC 50651</td></tr><tr><td rowspan=1 colspan=1>Encephalitozoon hellum</td><td rowspan=1 colspan=1>ATCC 50504</td></tr><tr><td rowspan=1 colspan=1>Encephalitozoon cuniculi</td><td rowspan=1 colspan=1>ATCC 50602</td></tr><tr><td rowspan=1 colspan=1>Pentatrichomonas hominis</td><td rowspan=1 colspan=1>ATCC 30098</td></tr><tr><td rowspan=1 colspan=1>Entamoeba barretti</td><td rowspan=1 colspan=1>ATCC 30996</td></tr><tr><td rowspan=1 colspan=1>Entamoeba dispar</td><td rowspan=1 colspan=1>ATCC PRA-260</td></tr><tr><td rowspan=1 colspan=1>Entamoeba gigivalis</td><td rowspan=1 colspan=1>ATCC 30927</td></tr><tr><td rowspan=1 colspan=1>Entamoeba invadens</td><td rowspan=1 colspan=1>ATCC 30994</td></tr><tr><td rowspan=1 colspan=1>Entamoeba moshkovskii</td><td rowspan=1 colspan=1>ATCC 30041</td></tr><tr><td rowspan=1 colspan=1>Entamobea ranarum</td><td rowspan=1 colspan=1>ATCC 50389</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium meleagridis*</td><td rowspan=1 colspan=1>1867</td></tr></table>

# Interference:

Twenty-two (22) biological and chemical substances occasionally used or found in stool specimens were evaluated for potential interference with the BD MAX Enteric Parasite Panel near the LOD for each particular target. Antibiotics were included as a combination of 8 different antibiotics tested simultaneously, with each antibiotic at a concentration that may be excreted in a stool specimen. Three of the substances exhibited potential interference with the BD MAX Enteric Parasite Panel (refer to Table 19). Vagisil cream demonstrated potential interference at concentrations greater than $9 \%$ in stool. Whole human blood demonstrated potential interference at concentrations greater than $2 5 \%$ in stool. Additional testing with grossly bloody clinical stool specimens showed potential interference in one out of a total of 12 specimens tested. Substances that demonstrated interference may result in unresolved (UNR), indeterminate (IND), or false negative assay results.

Table 19. Endogenous and Commercial Exogenous Substances Tested   

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Fecal Fat</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Spermicidal Lubricant</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Diaper Rash Cream</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Vagisil (&gt;9%)</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Whole Human Blood (&gt;25%)</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Laxatives</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone Cream</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Anti-Diarrheal (liquid)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Antiseptic Towelettes</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Anti-Diarrheal (pill)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Enema</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antibiotics Mixture</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hemorrhoidal Gel</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antacids</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Nystatin Cream</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Non-Steroidal Anti-Inflammatory (NSAID)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>10% Buffered Formalin</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Topical Antibiotic</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>20% Buffered Formalin</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Grossly Bloody Stool</td><td rowspan=1 colspan=1>I</td></tr></table>

I: Interference with the BD MAX Enteric Parasite Panel. NI: No reportable interference with the BD MAX Enteric Parasite Panel.

# Fresh versus Frozen Study

A fresh versus frozen study was performed to support the use of frozen unpreserved stool specimens in the clinical study; the test is not intended for use on frozen samples. Seven individual negative clinical unpreserved stool specimens were used to prepare a panel of sixty-six positive samples spiked with each of the three target organisms at various concentrations. The panel included low positive (2X LoD; 24 replicates), moderate positive (4X LoD; 24 replicates), as well as high positive samples (10X, 50X, and 100X LoD; 6 replicates each). Panels were tested with the BD MAX EPP at baseline, stored at ${ } - 2 0 \ { } ^ { \circ } \mathrm { C }$ , and then exposed to a total of 10 freeze/thaw cycles. The samples were tested again after every other freeze/thaw cycle. One C. parvum replicate and one $E .$ . histolytica replicate produced Unresolved results at baseline testing and were excluded from analysis. All replicates with positive baseline results were positive with the BD MAX EPP for up to 10 freeze/thaw cycles. The results indicate that up to 10 freeze/thaw cycles with unpreserved stool should not significantly affect the performance of the BD MAX EPP as compared to fresh specimen testing.

# Carryover/Cross-Contamination

A study was conducted to investigate within-run carryover and between-run carryover while processing specimens with a high load of Giardia lamblia, Cryptosporidium parvum, and Entamoeba histolytica in the BD MAX Enteric Parasite Panel. A panel was prepared with one high positive member containing all three target organisms and one negative member. Isolates of Giardia lamblia, Entamoeba histolytica and Cryptosporidium parvum were spiked at $1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ cysts/trophozoites per mL to prepare the high positive panel member. The negative member did not contain any target analyte. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run by alternating negative and positive samples. Two (2) operators performed a total of 15 runs with each run containing 24 samples.

Carryover contamination was assessed for each target in the BD MAX Enteric Parasite Panel. A total of 180 Sample Buffer Tubes, each containing the three BD MAX Enteric Parasite Panel targets, were assessed in the carryover contamination study. All of the 180 spiked Sample Buffer Tubes produced the expected positive results for all three target organisms. A total of 180 Sample Buffer Tubes, each negative for all three BD MAX Enteric Parasite Panel targets, were also assessed in the carryover contamination study. One hundred and seventy-eight (178) of the 180 spiked Sample Buffer Tubes produced the expected negative results. One expected negative result was positive for Giardia lamblia and the second was positive for both Giardia lamblia and Entamoeba histolytica in a single Sample Buffer Tube. The overall carryover/cross-contamination rate was $1 . 1 \%$ .

# Mixed Infection/ Competitive Interference

The mixed infection/competitive interference study was designed to evaluate the ability of the BD MAX Enteric Parasite Panel to detect low positive results in the presence of other targets at high concentrations. Four (4) organisms (Giardia lamblia, Cryptosporidium parvum and two preparations of Entamoeba histolytica) were individually prepared at 2X their respective LOD to serve as a low concentration target in the BD MAX Enteric Parasite Panel Sample Buffer Tube. A high concentration target mix of the other two BD MAX Enteric Parasite Panel analytes was spiked into the Sample Buffer Tube at a final concentration of $\mathbf { \bar { \rho } } > 1 \mathbf { x } 1 0 ^ { 5 }$ organisms/mL along with $1 0 ~ \mu \mathrm { L }$ of unpreserved stool and tested to simulate mixed infections. The second target mix for Entamoeba histolytica also contained Entamoeba dispar prepared as just described for the other high concentration targets. All four low concentration target organisms were successfully detected by the BD MAX Enteric Parasite Panel when combined with other organisms at high concentrations to simulate mixed infections.

# f. Assay cut-off:

Assay cut-offs for the BD MAX Enteric Parasite Panel were determined in analytical verification experiments and then subsequently validated using data from the multi-site clinical study. PCR metrics from the clinical study were graphically and statistically

analyzed as compared to results from the reference method for each targeted analyte. ROC curve analysis was performed separately for each PCR metric to confirm the optimal cutoffs for all analytes as compared to the reference method.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

A multicenter clinical study was conducted to assess the performance of the BD MAX Enteric Parasite Panel for the identification of Giardia lamblia, Cryptosporidium parvum/hominis and Entamoeba histolytica, from unpreserved or $10 \%$ formalin-fixed stool specimens from symptomatic patients suspected of gastroenteritis, enteritis and/or colitis. The study evaluated results obtained with the BD MAX Enteric Parasite Panel compared to those obtained with the reference method.

The study involved a total of five (5) clinical sites where specimens were collected and tested on the BD MAX Enteric Parasite Panel, as well as seven (7) sites which served as collection sites. Clinical centers were selected for the study based on a number of criteria, such as investigator and site personnel availability, number of specimens of interest, target prevalence, routinely collected specimen types, and familiarity with PCR methodology. Additionally, an internal site was involved as a clinical center to perform BD MAX Enteric Parasite Panel testing on specimens supplied by collection sites. Samples tested at BD were obtained from all collection sites. Specimens consisted of a mix of $10 \%$ formalin-fixed and unpreserved specimens as well as a mix of prospective and retrospective specimens. Only excess, de-identified patient specimens were used.

A total of 2204 prospective specimens ( $1 1 2 8 ~ 1 0 \%$ formalin-fixed, 1058 unpreserved and 18 non-compliant) and 411 retrospective specimens $148 1 0 \%$ formalin-fixed, 251 unpreserved and 12 non-compliant) were enrolled in the clinical evaluation. Retrospective unpreserved specimens were stored at ${ } - 2 0 \ { } ^ { \circ } \mathrm { C }$ or colder. Out of 978 compliant prospective unpreserved specimens, 287 were stored at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ or colder. A total of 128 retrospective specimens were not included in the performance calculations as the historical results were not confirmed by an alternate PCR and bi-directional sequencing.

Table 20. Compliant clinical trial enrollment summary by age group and specimen type   

<table><tr><td rowspan=2 colspan=1>Age Group</td><td rowspan=1 colspan=3>Specimen Type</td></tr><tr><td rowspan=1 colspan=1>Formalin 10%</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0-1 month</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>1 month to 2 years</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>162</td></tr><tr><td rowspan=1 colspan=1>2-12 years</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>294</td></tr><tr><td rowspan=1 colspan=1>13-18 years</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>19-21 years</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>Over 21 years</td><td rowspan=1 colspan=1>723</td><td rowspan=1 colspan=1>782</td><td rowspan=1 colspan=1>1505</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>264</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1273</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>2495</td></tr></table>

For the $10 \%$ formalin fixed specimen type, the BD MAX Enteric Parasite Panel identified $9 5 . 5 \%$ and $9 9 . 7 \%$ of the Giardia lamblia prospective positive and negative specimens, respectively, and $100 \%$ and $100 \%$ of the retrospective positive and negative specimens, respectively. For the unpreserved specimen type, the BD MAX Enteric Parasite Panel identified $9 4 . 4 \%$ and $100 \%$ of the Giardia lamblia prospective positive and negative specimens, respectively, and $9 8 . 6 \%$ and $9 4 . 9 \%$ of the retrospective positive and negative specimens, respectively (Table 21).

Table 21. Giardia lamblia - Clinical Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAXEPP</td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=8 colspan=1>10%FormalinFixed</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>32$</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>996</td><td rowspan=1 colspan=1>997</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>999</td><td rowspan=1 colspan=1>1021</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY (95% CI): 95.5% (78.2%,,99.2%)SPECIFICITY(95%CI): 99.7%b (99.1%,, 99.9%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=4>PPA (95% CI): 100.0% (93.5%,,100.0%)NPA (95%CI): 100.0% (94.9%,,100.0%)</td></tr><tr><td rowspan=8 colspan=1>Unpreserved</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>655</td><td rowspan=1 colspan=1>656</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>655</td><td rowspan=1 colspan=1>673</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY(95%CI): 94.4%6 (74.2%,99.0%)SPECIFICITY (95% CI): 100.0% (99.4%, 1100.0%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>74,5</td><td rowspan=1 colspan=1>79</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>209</td></tr><tr><td rowspan=1 colspan=4>PPA (95% CI): 98.6% (92.6%, 99.8%)NPA (95% CI): 94.9% (89.8%, 97.5%)</td></tr></table>

The alternate PCR and bi-directional sequencing component of the reference method was negative for this specimen, and the DFA component was positive. Discrepant repeat testing with alternate PCR/ sequencing and DFA were performed and both produced negative results. Discrepant repeat testing with the BD MAX™ Enteric Parasite Panel was performed in twelve (12) replicates of this specimen and gave all negative results (0/12). Discrepant testing with a Giardia antigen EIA also gave a negative result.

2 Discrepant testing with a Giardia antigen EIA gave a negative result for one (1) specimen, and discrepant repeat testing with the BD MAX™ Enteric Parasite Panel in six (6) replicates of this specimen and gave a positive result for one replicate (1/6). No discrepant testing was performed for the other two (2) specimens.

3 The alternate PCR and bi-directional sequencing component of the reference method was negative for this specimen, and the DFA component was positive. Discrepant repeat testing with the alternate PCR/sequencing gave a negative result, and discrepant repeat testing with DFA gave a positive result. Discrepant repeat testing with the BD MAX™ Enteric Parasite Panel was performed in six (6) replicates of this specimen and gave all negative results (0/6). Discrepant testing with a Giardia antigen EIA also gave a negative result.

4 Discrepant testing on one (1) specimen with a Giardia antigen EIA and a commercially-available molecular assay gave positive results for both. Discrepant repeat testing with the BD MAX™ Enteric Parasite Panel performed in six (6) replicates of this specimen gave all positive results $( 6 / 6 )$ .

5 Discrepant testing on one (1) specimen with a Giardia antigen EIA gave a positive result, and discrepant testing with a commercially-available molecular assay gave a negative result. Discrepant repeat testing with the BD MAX™ Enteric Parasite Panel was done in eleven (11) replicates of this specimen and gave 5 positive results (5/11). No discrepant testing was performed for the other five (5) specimens.

For the $10 \%$ formalin fixed specimen type, the BD MAX Enteric Parasite Panel identified $9 0 . 3 \%$ and $9 9 . 8 \%$ of the Cryptosporidium parvum/hominis prospective positive and negative specimens, respectively, and $93 \%$ and $100 \%$ of the retrospective positive and negative specimens, respectively. For the unpreserved specimen type, the BD MAX Enteric Parasite Panel identified $100 \%$ and $9 9 . 5 \%$ of the Cryptosporidium parvum/hominis prospective positive and negative specimens, respectively, and $9 7 . 7 \%$ and $9 8 . 4 \%$ of the retrospective positive and negative specimens, respectively (Table 22). Because DFA identifies Cryptosporidium to the genus level, DFA-positive specimens identified by bi-directional sequencing as other than $C .$ . hominis or $C .$ . parvum were considered reference method negative.

Table 22. Cryptosporidium hominis/parvum - Clinical Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAXEPP</td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=8 colspan=1>10%FormalinFixed</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>58</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>61,2</td><td rowspan=1 colspan=1>9514</td><td rowspan=1 colspan=1>957</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>953</td><td rowspan=1 colspan=1>1015</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY: 90.3% (80.5%, 95.5%)SPECIFICITY: 99.8% (99.2%, 99.9%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>121</td></tr><tr><td rowspan=1 colspan=4>PPA: 93% (81.4%, 97.6%)NPA: 100% (95.3%, 100%)</td></tr><tr><td rowspan=8 colspan=1>Unpreserved</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>625</td><td rowspan=1 colspan=1>625</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>628</td><td rowspan=1 colspan=1>663</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY: 100% (90.1%, 100%)SPECIFICITY: 99.5% (98.6%, 99.8%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>182</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>228</td></tr><tr><td rowspan=1 colspan=4>PPA: 97.7% (88.2%, 99.6%)NPA: 98.4% (95.3%, 99.4%)</td></tr></table>

All six specimens were positive by the DFA component of the reference method. One specimen sequenced as $C$ . parvum, three (3) specimens were negative, and two (2) were non-reportable by the alternate PCR and bi-directional sequencing components of the composite reference method.

2 Discrepant repeat testing with the alternate PCR and bi-directional sequencing was performed on all six (6) specimens. One (1) specimen sequenced as Cryptosporidium parvum, one (1) specimen sequenced as Cryptosporidium felis and the remaining four (4) were PCR negative by discrepant repeat testing. Discrepant testing was also performed using an antigen detecting EIA that does not distinguish between Cryptosporidium and Giardia. Two specimens were EIA negative and four specimens were EIA positive, of which two were positive for Giardia by other test methods.

3 One DFA-positive specimen was classified as reference method negative based on alternate PCR and bi-directional sequencing results that identified Cryptosporidium meleagridis.

Six DFA-positive specimens were classified as reference method negative based on alternate PCR and bi-directional sequencing results that identified (4) Cryptosporidium canis, (1) C. meleagridis and (1) Cryptosporidium spp. (undefined).

Discrepant repeat testing was performed with BD MAX Enteric Parasite Panel in twelve (12) replicates per specimen. One specimen was positive for five (5) replicates (5/12) and one specimen was positive for two (2) replicates (2/12).

6 Discrepant repeat testing was performed with BD MAX Enteric Parasite Panel in six (6) replicates per specimen. One specimen was positive for five (5) of six (6) replicates and one specimen was positive for three (3) of six (6) specimens. A third specimen was negative in six (6) of six (6) replicates.

7 Discrepant repeat testing was performed with BD MAX Enteric Parasite Panel in six (6) replicates per specimen. Two specimens were negative for six (6) of six (6) replicates.

For the $10 \%$ formalin fixed specimen type, the BD MAX Enteric Parasite Panel identified $100 \%$ of the Entamoeba histolytica negative specimens for both the prospective and retrospective specimens. There were no prospective or retrospective $10 \%$ formalin fixed Entamoeba histolytica positive specimens found during the clinical evaluation. For the unpreserved specimen type, the BD MAX Enteric Parasite Panel identified $100 \%$ Entamoeba histolytica prospective negative specimens and $100 \%$ and $100 \%$ of the retrospective positive and negative specimens, respectively (Table 23. There were no prospective unpreserved Entamoeba histolytica positive specimens found during the clinical evaluation.

Table 23. Entamoeba histolytica - Clinical Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAXEPP</td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=8 colspan=1>10%FormalinFixed</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>827</td><td rowspan=1 colspan=1>827</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>827</td><td rowspan=1 colspan=1>827</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY(95%CI): No data for calculationSPECIFICITY(95% CI): 100% (99.5%, 100%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=4>PPA(95%CI): No data for calculationNPA (95%CI): 100% (93.4%, 100%)</td></tr><tr><td rowspan=8 colspan=1>Unpreserved</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>577</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>577</td></tr><tr><td rowspan=1 colspan=4>SENSITIVITY (95% CI): No data for calculationSPECIFICITY (95% CI): 100% (99.3%, 100%)</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>191</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>202</td></tr><tr><td rowspan=1 colspan=4>PPA (95% CI):: 100% (74.1%, 100%)NPA (95%CI):100% (98.0%,100%)</td></tr></table>

As Entamoeba histolytica is a rare analyte, both prospective and retrospective testing efforts were unable to demonstrate the positive percent agreement of the BD MAX Enteric Parasite Panel. To supplement the prospective and retrospective data, an evaluation of contrived specimens was performed. Surrogate clinical specimens were prepared using residual specimens that had previously tested negative for all BD MAX

Enteric Parasite Panel targets. Specimens were spiked at clinically relevant levels at various concentrations of the limit of detection for each specimen type. Users analyzing the specimens were blinded to the specimen status.

For both the $10 \%$ formalin fixed and unpreserved specimen types, the BD MAX Enteric Parasite Panel correctly identified $100 \%$ of both the positive and negative specimens. The contrived study results obtained with the BD MAX Enteric Parasite Panel were compared to the expected results and are summarized in Table 24.

Table 24. Entamoeba histolytica – Contrived Specimen Performance   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>BD MAXEPP</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=4 colspan=1>Formalin10%</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=4>PPA(95%CI):100%(92.9%,,100%)NPA (95%CI):100%(92.9%, 100%)</td></tr><tr><td rowspan=4 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=4>PPA (95%CI):100%(92.9%, 100%)NPA (95%CI):100% (92.9%, 100%)</td></tr></table>

Performance of the BD MAX Enteric Parasite Panel by Cryptosporidium hominis and Cryptosporidium parvum species identification as observed during the clinical trial is presented below in Table 25. The species identification was obtained from the alternate PCR and bi-directional sequencing segment of the composite reference method. While the BD MAX Enteric Parasite Panel is designed to detect the species described below, the panel does not report results to the species level.

Table 25. Cryptosporidium PPA per species observed during the clinical trial   

<table><tr><td rowspan=1 colspan=3>Cryptosporidium</td><td rowspan=1 colspan=2>PPA</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Specimen OriginSpecies</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>%Point Estimate(n/N)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>10% Formalin Fixed</td><td rowspan=2 colspan=1>Prospective</td><td rowspan=1 colspan=1>hominis</td><td rowspan=1 colspan=1>100.0% (17/17)</td><td rowspan=1 colspan=1>(81.6%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>parvum</td><td rowspan=1 colspan=1>97.4% (37/38)</td><td rowspan=1 colspan=1>(86.5%, 99.5%)</td></tr><tr><td rowspan=2 colspan=1>Retrospective</td><td rowspan=1 colspan=1>hominis</td><td rowspan=1 colspan=1>95.0% (19/20)</td><td rowspan=1 colspan=1>(76.4%, 99.1%)</td></tr><tr><td rowspan=1 colspan=1>parvum</td><td rowspan=1 colspan=1>91.3% (21/23)</td><td rowspan=1 colspan=1>(73.2%, 97.6%)</td></tr><tr><td rowspan=4 colspan=1>Unpreserved</td><td rowspan=2 colspan=1>Prospective</td><td rowspan=1 colspan=1>hominis</td><td rowspan=1 colspan=1>100.0% (22/22)</td><td rowspan=1 colspan=1>(85.1%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>parvum</td><td rowspan=1 colspan=1>100.0% (11/11)</td><td rowspan=1 colspan=1>(74.1%, 100.0%)</td></tr><tr><td rowspan=2 colspan=1>Retrospective</td><td rowspan=1 colspan=1>hominis</td><td rowspan=1 colspan=1>96.2% (25/26)</td><td rowspan=1 colspan=1>(81.1%, 99.3%)</td></tr><tr><td rowspan=1 colspan=1>parvum</td><td rowspan=1 colspan=1>100.0% (18/18)</td><td rowspan=1 colspan=1>(82.4%, 100.0%)</td></tr></table>

There were twenty-three (23) co-infections detected by the BD MAX Enteric Parasite Panel. Table 26 below shows the co-infections detected by the BD MAX Enteric Parasite Panel during the clinical trial.

Table 26. Co-infections observed during the BD MAX Enteric Parasite Panel clinical trial   

<table><tr><td rowspan=1 colspan=2>Co-Infections</td><td rowspan=2 colspan=1>Number ofCo-InfectionsObserved</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-Infections</td><td rowspan=2 colspan=1>DiscrepantTarget</td><td rowspan=2 colspan=1>UnavailableReference MethodResult forComparison</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>C. parvum/hominis</td><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C. parvum/hominis</td><td rowspan=1 colspan=1>1 C. parvum/ hominis</td></tr><tr><td rowspan=1 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>C. parvum/hominis</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>C. parvum/hominis</td><td rowspan=1 colspan=1>3 G. lamblia and4 C. parvum/ hominis</td></tr><tr><td rowspan=1 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>G. lamblia</td><td rowspan=1 colspan=1>1 G. lamblia and3 E. histolytica</td></tr></table>

1 All four (4) are retrospective specimens with unconfirmed historical results.

Of the 2226 specimens initially evaluated with the BD MAX Enteric Parasite Panel, $1 . 5 \%$ of the $10 \%$ formalin fixed and $4 . 7 \%$ of the unpreserved specimens were initially reported as Unresolved. Following a valid repeat test, $0 \%$ of the $10 \%$ formalin fixed and $1 . 2 \%$ of the unpreserved specimens remained Unresolved. The total numbers provided in Table 27 are based on compliant specimens and BD MAX Enteric Parasite Panel results.

Table 27. Unresolved Rates   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Unresolved Rate with95% CI</td><td rowspan=1 colspan=2>Final Unresolved Rate withValid Repeat with 95% CI</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SpecimenOrigin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Formalin 10%</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1.5% (16/1084)</td><td rowspan=1 colspan=1>(0.9%, 2.4%)</td><td rowspan=1 colspan=1>0.0% (0/1084)</td><td rowspan=1 colspan=1>(0.0%, 0.4%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1.4% (2/146)</td><td rowspan=1 colspan=1>(0.4%, 4.9%)</td><td rowspan=1 colspan=1>0.0% (0/146)</td><td rowspan=1 colspan=1>(0.0%, 2.6%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>5.6% (42/752)</td><td rowspan=1 colspan=1>(4.2%, 7.5%)</td><td rowspan=1 colspan=1>1.5% (11/747)</td><td rowspan=1 colspan=1>(0.8%, 2.6%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2.0% (5/244)</td><td rowspan=1 colspan=1>(0.9%, 4.7%)</td><td rowspan=1 colspan=1>0.4% (1/244)</td><td rowspan=1 colspan=1>(0.1%, 2.3%)</td></tr></table>

Of the 2226 specimens initially evaluated with the BD MAX Enteric Parasite Panel, $0 . 3 \%$ of the $10 \%$ formalin fixed and $0 . 1 \%$ of the unpreserved specimens were initially reported as Indeterminate. Following a valid repeat test, $0 \%$ of both the $10 \%$ formalin fixed and the unpreserved specimens remained Indeterminate. The total numbers provided in Table 28 are based on compliant specimens and BD MAX Enteric Parasite Panel results.

Table 28. Indeterminate Rates   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Indeterminate Ratewith 95% CI</td><td rowspan=1 colspan=2>Final Indeterminate Rate withValid Repeat with 95% CI</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SpecimenOrigin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Formalin 10%</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0.4% (4/1084)</td><td rowspan=1 colspan=1>(0.1%, 0.9%)</td><td rowspan=1 colspan=1>0.0% (0/1084)</td><td rowspan=1 colspan=1>(0.0%, 0.4%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0.0% (0/146)</td><td rowspan=1 colspan=1>(0.0%, 2.6%)</td><td rowspan=1 colspan=1>0.0% (0/146)</td><td rowspan=1 colspan=1>(0.0%, 2.6%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0.1% (1/752)</td><td rowspan=1 colspan=1>(0.0%, 0.7%)</td><td rowspan=1 colspan=1>0.0% (0/747)</td><td rowspan=1 colspan=1>(0.0%, 0.5%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0.0% (0/244)</td><td rowspan=1 colspan=1>(0.0%, 1.5%)</td><td rowspan=1 colspan=1>0.0% (0/244)</td><td rowspan=1 colspan=1>(0.0%, 1.5%)</td></tr></table>

Of the 2226 specimens initially evaluated with the BD MAX Enteric Parasite Panel, $0 . 6 \%$ of the $10 \%$ formalin fixed and $0 . 4 \%$ of the unpreserved specimens were initially reported as Incomplete. Following a valid repeat test, $0 \%$ of both the $10 \%$ formalin fixed and the unpreserved specimens remained Incomplete. The total numbers provided in Table 29 are based on compliant specimens and BD MAX Enteric Parasite Panel results.

Table 29. Incomplete Rates   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Initial Incomplete Rate with95% CI</td><td rowspan=1 colspan=2>Final Incomplete Rate withValid Repeat with 95% CI</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SpecimenOrigin</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Formalin 10%</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0.6% (6/1084)</td><td rowspan=1 colspan=1>(0.3%, 1.2%)</td><td rowspan=1 colspan=1>0.0% (0/1084)</td><td rowspan=1 colspan=1>(0.0%, 0.4%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0.7% (1/146)</td><td rowspan=1 colspan=1>(0.1%, 3.8%)</td><td rowspan=1 colspan=1>0.0% (0/146)</td><td rowspan=1 colspan=1>(0.0%, 2.6%)</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0.0% (0/752)</td><td rowspan=1 colspan=1>(0.0%, 0.5%)</td><td rowspan=1 colspan=1>0.0% (0/747)</td><td rowspan=1 colspan=1>(0.0%,0.5%)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1.6% (4/244)</td><td rowspan=1 colspan=1>(0.6%, 4.1%)</td><td rowspan=1 colspan=1>0.0% (0/244)</td><td rowspan=1 colspan=1>(0.0%, 1.5%)</td></tr></table>

In the event of an unsuccessful pre-warm step with the BD MAX Pre-Warm heater, the BD MAX System will display an error message. There were a total of 211 pre-warm runs during the BD MAX EPP clinical study. A single BD MAX Pre-Warm failure was observed for a failure rate of $0 . 5 \%$ (1/211).

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the BD MAX Enteric Parasite Panel clinical study, reportable results from compliant specimens were obtained from clinical sites and compared to a composite reference method. The study population was grouped based on specimen type and age. The number and percentage of positive cases by target, as determined by the BD MAX Enteric Parasite Panel during the prospective segment of the clinical trial, are presented below in Table 30. Table 30 includes data from all clinical trial sites, including a state reference laboratory that enrolled specimens that were presumptively positive for Cryptosporidium as well as specimens collected outside of the US.

Table 30. BD MAX Enteric Parasite Panel Expected Values   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=6>BD MAX Enteric Parasite Panel Positive Rate</td></tr><tr><td rowspan=1 colspan=2>Giardia lamblia</td><td rowspan=1 colspan=2>Cryptosporidiumparvum/hominis</td><td rowspan=1 colspan=2>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>NumberofSpecimens</td><td rowspan=1 colspan=1>Expected Value</td><td rowspan=1 colspan=1>Number ofSpecimens</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number ofSpecimens</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=7 colspan=1>Formalin10%</td><td rowspan=1 colspan=1>1 month to 2 years</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>3.9% (4/103)</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>4.9% (5/103)</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>0.0% (0/103)</td></tr><tr><td rowspan=1 colspan=1>2-12</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>5.4% (10/185)</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>8.6% (16/185)</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>0.0% (0/185)</td></tr><tr><td rowspan=1 colspan=1>13-18</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0.0% (0/105)</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>6.7% (7/105)</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0.0% (0/105)</td></tr><tr><td rowspan=1 colspan=1>19-21</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3.1% (1/32)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>9.4% (3/32)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0.0% (0/32)</td></tr><tr><td rowspan=1 colspan=1>Over 21</td><td rowspan=1 colspan=1>658</td><td rowspan=1 colspan=1>1.7% (11/658)</td><td rowspan=1 colspan=1>658</td><td rowspan=1 colspan=1>4.1% (27/658)</td><td rowspan=1 colspan=1>658</td><td rowspan=1 colspan=1>0.0% (0/658)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0% (0/1)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0% (0/1)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0% (0/1)</td></tr><tr><td rowspan=1 colspan=1>Overall study</td><td rowspan=1 colspan=1>1084</td><td rowspan=1 colspan=1>2.4% (26/1084)</td><td rowspan=1 colspan=1>1084</td><td rowspan=1 colspan=1>5.4% (58/1084)</td><td rowspan=1 colspan=1>1084</td><td rowspan=1 colspan=1>0.0% (0/1084)</td></tr><tr><td rowspan=7 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>month to 2 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0% (0/2)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0% (0/2)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0% (0/2)</td></tr><tr><td rowspan=1 colspan=1>2-12</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>2.4% (1/41)</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0.0% (0/41)</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0.0% (0/41)</td></tr><tr><td rowspan=1 colspan=1>13-18</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0.0% (0/59)</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0.0% (0/59)</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0.0% (0/59)</td></tr><tr><td rowspan=1 colspan=1>19-21</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0.0% (0/19)</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0.0% (0/19)</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0.0% (0/19)</td></tr><tr><td rowspan=1 colspan=1>Over 21</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>1.0% (5/499)</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0.2% (1/498)</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0.0% (0/498)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>11.2% (13/116)</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>34.5% (40/116)</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>0.0% (0/116)</td></tr><tr><td rowspan=1 colspan=1>Overall study</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>2.6% (19/736)</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>5.6% (41/735)</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>0.0% (0/735)</td></tr><tr><td rowspan=7 colspan=1>Combined</td><td rowspan=1 colspan=1>month to 2 years</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>3.8% (4/105)</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>4.8% (5/105)</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0.0% (0/105)</td></tr><tr><td rowspan=1 colspan=1>2-12</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>4.9% (11/226)</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>7.1% (16/226)</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>0.0% (0/226)</td></tr><tr><td rowspan=1 colspan=1>13-18</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>0.0% (0/164)</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>4.3% (7/164)</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>0.0% (0/164)</td></tr><tr><td rowspan=1 colspan=1>19-21</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>2.0% (1/51)</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>5.9% (3/51)</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>0.0% (0/51)</td></tr><tr><td rowspan=1 colspan=1>Over 21</td><td rowspan=1 colspan=1>1157</td><td rowspan=1 colspan=1>1.4% (16/1157)</td><td rowspan=1 colspan=1>1156</td><td rowspan=1 colspan=1>2.4% (28/1156)</td><td rowspan=1 colspan=1>1156</td><td rowspan=1 colspan=1>0.0% (0/1156)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>11.1% (13/117)</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>34.2% (40/117)</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0.0% (0/117)</td></tr><tr><td rowspan=1 colspan=1>Overall study</td><td rowspan=1 colspan=1>1820</td><td rowspan=1 colspan=1>2.5% (45/1820)</td><td rowspan=1 colspan=1>1819</td><td rowspan=1 colspan=1>5.4% (99/1819)</td><td rowspan=1 colspan=1>1819</td><td rowspan=1 colspan=1>0.0% (0/1819)</td></tr></table>

# N. Instrument Name:

BD MAX System

# O. System Descriptions:

1. Modes of Operation:

The BD MAX System fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ____X___ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No ___X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X___ or No

3. Specimen Identification: Specimens are labeled with a barcode.

4. Specimen Sampling and Handling:

A disposable inoculating loop is used to place $1 0 ~ \mu \mathrm { L }$ of the unpreserved or $10 \%$ formalin stool specimen into a SBT which is then vortexed and placed onto the system.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control: See section M.1c above.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.