# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K143651

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the Simplexa™ Group A Strep Direct and Simplexa™ Group A Strep Positive Control Pack from throat specimens.

# C. Measurand:

Group A $\beta$ -hemolytic Streptococcus (GAS; Streptococcus pyogenes) nucleic acids.

# D. Type of Test:

The Simplexa™ Group A Strep Direct and Simplexa™ Group A Strep Positive Control Pack (Simplexa™ Group A Strep Direct) assay is a Real-Time PCR in vitro diagnostic test for the rapid and qualitative detection and differentiation of Group A $\beta$ -hemolytic Streptococcus (Streptococcus pyogenes) nucleic acids isolated from throat swab specimens obtained from symptomatic patients.

# E. Applicant:

Focus Diagnostics, Inc.

F. Proprietary and Established Names:

Simplexa™ Group A Strep Direct and Simplexa™ Group A Strep Positive Control Pack

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.2680 - Streptococcus spp. Nucleic Acid-Based Assay

2. Classification: Class II

3. Product code:

PGX – Groups A, C and G Beta-Hemolytic Streptococcus Nucleic Acid Amplification System

OOI – Real-Time Nucleic Acid Amplification System

4. Panel: 83- Microbiology

# H. Intended Use:

1. Intended use(s):

Simplexa™ Group A Strep Direct   
The Focus Diagnostics Simplexa™ Group A Strep Direct assay is intended for use on the 3M Integrated Cycler for the in vitro qualitative detection of Group A Streptococcus (GAS) from throat swabs collected from human patients with signs and symptoms of pharyngitis, such as sore throat. This test is intended for use as an aid in the diagnosis of GAS infection. The assay is intended for use in hospital, reference, or state laboratory settings. The device is not intended for point-of-care use. Simplexa™ Group A Strep Positive Control Pack   
Focus Diagnostics’ Simplexa™ Group A Strep Positive Control Pack is intended to be used as a control with Simplexa™ Group A Strep Direct. This control is not intended for use with other assays or systems.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: 3M Integrated Cycler

# I. Device Description:

The Simplexa™ Group A Strep Direct assay system is a real-time PCR system that enables the direct amplification and qualitative detection of Group A Strep bacterial DNA from throat swabs that have not undergone a nucleic acid extraction. The system consists of the Simplexa™ Group A Strep Direct assay, the 3M Integrated Cycler (with Integrated Cycler Studio Software), the Direct Amplification Disc (DAD) and associated accessories.

In the Simplexa™ Group A Strep Direct assay, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify Group A Strep bacterial DNA and the Internal Control (DNA IC). The assay targets a conserved region of Group A Strep (pyrogenic exotoxin B gene) to identify this bacteria in the specimen. The DNA IC is used to detect PCR failure and/or inhibition.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra™ Direct Strep Assay

2. Predicate 510(k) number(s): k133883

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">SimplexaTM Group A StrepDirect</td><td colspan="1" rowspan="1">LyraTM Direct Strep Assay(k133883)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">SimplexaTM Group A StrepDirectThe Focus DiagnosticsSimplexa™M Group A StrepDirect assay is intended for useon the 3M Integrated Cycler forthe in vitro qualitative detectionof Group A Streptococcus(GAS) from throat swabscollected from human patientswith signs and symptoms ofpharyngitis, such as sore throat.This test is intended for use as anaid in the diagnosis of GASinfection. The assay is intendedfor use in hospital, reference, orstate laboratory settings. Thedevice is not intended for point-of-care use.SimplexaTM Group A StrepPositive Control PackFocus Diagnostics' SimplexaTMGroup A Strep Positive ControlPack is intended to be used as a</td><td colspan="1" rowspan="1">The Lyra Direct Strep Assay is aReal-Time PCR in vitrodiagnostic test for the qualitativedetection and differentiation ofGroup A β-hemolyticStreptococcus (Streptococcuspyogenes) and pyogenic Group Cand G β-hemolytic Streptococcusnucleic acids isolated from throatswab specimens obtained frompatients with signs and symptomsof pharyngitis, such as a sorethroat. The assay does notdifferentiate between pyogenicGroups C and G β-hemolyticStreptococcus.All negative test results should beconfirmed by bacterial culture,because negative results do notpreclude Group A, C or G Strepinfection and should not be usedas the sole basis for treatment.The assay is intended for use in</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">SimplexaT Group A StrepDirect</td><td colspan="1" rowspan="1">LyraTM Direct Strep Assay(k133883)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">control with Simplexa™M GroupA Strep Direct. This control isnot intended for use with otherassays or systems.</td><td colspan="1" rowspan="1">hospital, reference, or statelaboratory settings. The device isnot intended for point-of-care use.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Throat swab</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Testing Time</td><td colspan="1" rowspan="1">60 minutes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DNA AmplificationTechnology</td><td colspan="1" rowspan="1">Real-time polymerase chainreaction</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Techniques</td><td colspan="1" rowspan="1">Automatically detectsfluorescence after dissociation offluorophore from quencherduring amplification</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Extraction</td><td rowspan=1 colspan=1>Self-contained and automated</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>Target SequenceDetected</td><td rowspan=1 colspan=1>Well conserved region of theexotoxin B gene (speB)</td><td rowspan=1 colspan=1>Group A  99bp product in theputative competence (comX1.1)geneGroups C/G  188bp product inthe tagatose-6-phosphate kinase(lacC) gene</td></tr><tr><td rowspan=1 colspan=1>Reagents/Components</td><td rowspan=1 colspan=1>SimplexaTM Group A StrepDirect Reaction MixSimplexaTM Group A StrepPositive Control PackDirect Amplification Disks</td><td rowspan=1 colspan=1>LyraTM Direct Strep Master Mix,Process Buffer, and RehydrationSolutionABI 7500 Fast Dx 96-well PCRPlate, optical plate films andplate centrifugeDry heating block</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>3M Integrated Cycler</td><td rowspan=1 colspan=1>ABI 7500 Fast DX Thermocycler</td></tr><tr><td rowspan=1 colspan=1>PerformanceCharacteristics</td><td rowspan=1 colspan=1>GAS* Sensitivity:97.4%[95% CI: 93.6% - 99.0%]GAS* Specificity:95.2%[95% CI: 93.9% - 96.3%]</td><td rowspan=1 colspan=1>GAS* Sensitivity:96.5%[95% CI: 91.3% - 98.6%]GAS* Specificity:98.0%[95% CI: 97.0% - 98.6%]Pyo GCS/GGS* Sensitivity:95.7%[95% CI: 88.1% - 98.5%]Pyo GCS/GGS* Specificity:98.3%[95% CI: 97.4% - 98.9%]</td></tr></table>

\*GAS = Group A Streptococcus; Pyo GCS/GGS $=$ Pyogenic Group C/G Streptococcus

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable.

# L. Test Principle:

The Simplexa™ Group A Strep Direct assay system is a real-time PCR system that enables the direct amplification and qualitative detection of Group A Strep bacterial DNA from throat swabs that have not undergone a nucleic acid extraction. The system consists of the Simplexa™ Group A Strep Direct assay, the 3M Integrated Cycler (with Integrated Cycler Studio Software), the Direct Amplification Disc (DAD) and associated accessories. In the Simplexa™ Group A Strep Direct assay, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify Group A Strep bacterial DNA and the Internal Control (DNA IC). The assay targets a conserved region of Group A Strep (pyrogenic exotoxin B gene) to identify this bacteria in the specimen. The DNA IC is used to detect PCR failure and/or inhibition.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Within-laboratory, inter-lot precision was evaluated for the Simplexa™ Group A Strep Direct assay at one (1) site using two (2) strains of Group A Streptococci (GAS) representative of serotypes M1 and M3 with four (4) panel members:

1) Negative   
2) Low Positive (LP), 1 x LoD and   
3) Moderate Positive (MP), 3.0 x LoD.   
4) Positive Control (PC)

Contrived specimens were spiked from freshly-grown bacterial stocks into a simulated negative matrix with target GAS at the final intended target concentration (confirmed by titration and plate count). Cell preparations were frozen at $- 7 0 ^ { \circ } \mathrm { C }$ after preparation until testing. The LoD values were based on the values obtained in the LoD study. See section ${ \bf M } . 2 . b$ below for details on the matrix equivalency and freezethaw study.

The study was conducted with three (3) different Reaction Mix lots. Each panel member was run in duplicate (2), twice (2) per day for three (3) non-consecutive days by one (1) operator at one (1) site (2 replicates x twice per day x 3 days x 3 lots $= 3 6$ replicates/panel member/strain). The results are shown in Table I.

<table><tr><td rowspan=1 colspan=6>Table I: Inter Lot Precision Study</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1># Detected/# Tested</td><td rowspan=1 colspan=1>%Detected</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>No Analyte</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>(0.0%, 10.6%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>GAS M1</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(90.4%, 100.0%)</td><td rowspan=1 colspan=1>6.8 x 102</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(90.4%,100.0%)</td><td rowspan=1 colspan=1>2.4 x 10^3</td></tr><tr><td rowspan=2 colspan=1>GAS M3</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(90.4%, ,100.0%)</td><td rowspan=1 colspan=1>2.0 x 10^3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(90.4%,100.0%)</td><td rowspan=1 colspan=1>7.1 x 103</td></tr><tr><td rowspan=1 colspan=1>PositiveT Control</td><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(90.4%, 100.0%)</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The precision study results met the pre-defined acceptance criteria for LP, MP, and PC specimens. The results of this study are acceptable and these results are described in labeling.

# Reproducibility Study

The reproducibility of the Simplexa™ Group A Strep Direct assay was established in a multi-center study. The same strains and panel members were prepared and tested as listed in the precision study above (Table I). Panel members were tested in triplicate (3) in two (2) runs per day each by a different operator at three (3) sites for five (5) non-consecutive days (3 replicates x 2 operators runs $_ { \textrm { X 3 } }$ sites $\mathrm { ~ x ~ } 5 ~ \mathrm { d a y s } = 9 0$ replicates/panel member). These studies were conducted with a single (1) lot of positive control and reagents across the three (3) sites. The results are shown in Table II.

<table><tr><td rowspan=1 colspan=6>Table II: Reproducibility Study</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1># Detected/# Tested</td><td rowspan=1 colspan=1>%Detected</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>No Analyte</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1/90*</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>(0.2%, 6.0%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>GAS M1</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>84/90</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>(86.2%,96.9%)</td><td rowspan=1 colspan=1>6.8 x 102</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>88/90*</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>(92.3%,99.4%)</td><td rowspan=1 colspan=1>2.4 x 103</td></tr><tr><td rowspan=2 colspan=1>GAS M3</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>85/90</td><td rowspan=1 colspan=1>94.4%</td><td rowspan=1 colspan=1>(87.6%,97.6%)</td><td rowspan=1 colspan=1>2.0 x 10^3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>87/90*</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>(90.7%,98.9%)</td><td rowspan=1 colspan=1>7.1 x 10^{3</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.9%, 100.0%)</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\* All observed unexpected results came from runs performed by the same operator at the same site during the first two days of the study.

These results met the predefined acceptance criteria for the various panels and are described in labeling.

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The Simplexa™ Group A Strep Direct and Simplexa™ Group A Strep Positive Control Pack have been determined to be stable for at least 14 months of real-time stability over a temperature range from $\boldsymbol { \mathbf { \ell } } _ { - 1 4 } \ : ^ { \mathrm { { o } } } \boldsymbol { \mathrm { { C } } }$ to ${ } _ { - 2 8 } { } ^ { \circ } \mathrm { C }$ . The products are labeled with a 12 month expiration date.

To assess the room-temperature stability of positive control aliquots of positive control were thawed and kept at room temperature for 8 hours and 24 hours and tested to determine room temperature stability. The positive control is stable up to 24 hours when stored at room temperature.

To assess the room-temperature stability of Simplexa™ Group A Strep Reaction Mix, the Reaction Mix was thawed from ${ \displaystyle - 2 0 ^ { \circ } C }$ and kept at room temperature for 30 and 60 minutes. Positive Control (PC) was tested in quadruplicate (4) with singlet (1) of unspiked Aimes medium as a negative sample. The Reaction Mix stability was established and the labeling instructs the end user to use within 30 minutes from thawing.

Clinical studies demonstrate that swab specimens are stable for over $^ { 2 4 \mathrm { ~ h ~ } }$ .

# d. Detection limit:

The limit of detection (LoD) of the Simplexa™ Group A Strep Direct assay was determined using contrived stocks of two (2) GAS serotypes which were prepared per the protocol described in the precision study section (see M.1.a above). The LoD study was conducted using two (2) lots on four (4) instruments. The LoD was defined as the point at which at least $9 5 \%$ of all replicates tested positive $( \mathbf { C } _ { 9 5 } )$ . Eight (8) dilutions from each serotype were screened in triplicate (3). The lowest dilution at which all three (3) replicates were detected was confirmed using twenty (20) additional replicates for each serotype at that concentration. Two invalid replicates were observed due to internal control failure. The LoD study results are shown in Table III below.

<table><tr><td rowspan=1 colspan=3>Table III: LoD for Group A β-hemolytic Streptococcus</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>Group A Streptococcal strain 1(Streptococcus pyogenes)</td><td rowspan=1 colspan=1>M1</td><td rowspan=1 colspan=1>6.82 x 102</td></tr><tr><td rowspan=1 colspan=1>Group A Streptococcal strain 2(Streptococcus pyogenes)</td><td rowspan=1 colspan=1>M3</td><td rowspan=1 colspan=1>2.35 x 103</td></tr></table>

These LoD values are described in the labeling and they were used in determining spiking levels for all subsequent analytical studies.

e. Analytical reactivity: Inclusivity studies were conducted for the Simplexa™ Group A Strep Direct assay with twenty-one (21) GAS strains, in addition to those tested in the LoD studies, in replicates of three (3) per strain near at $2 { \cdot } 3 \ \mathbf { x }$ LoD $( 1 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 3 } )$ using the LoD determined for the M1 serotype $\mathrm { ( L o D = 6 8 2 C F U / m l ) }$ as the basis. If detection failure was experienced, the concentration of the particular strain was increased until all three (3) replicates were positively detected.

This study was performed with one (1) reagent lot on two (2) different instruments, performed by two (2) operators. Each cultured strain was prepared per the protocol described above in M.1.a above. The inclusivity study results and the final organism concentrations tested are shown in Table IV below. All GAS strains were correctly detected by the assay at the concentrations indicated.

<table><tr><td rowspan=1 colspan=4>Table IV: Group A β-hemolytic Streptococcus Inclusivity</td></tr><tr><td rowspan=1 colspan=1>Group A StreptococcusStrain #</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=2>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>M2</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>M4</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>M5</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>M9</td><td rowspan=1 colspan=2>3.00 x 103</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>M12</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>M13</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>M14</td><td rowspan=1 colspan=1>1.50 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>M18</td><td rowspan=1 colspan=1>5.00 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>M22</td><td rowspan=1 colspan=1>1.50 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>M27</td><td rowspan=1 colspan=1>3.00 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>M28</td><td rowspan=1 colspan=1>1.50 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>M29</td><td rowspan=1 colspan=1>1.50 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>M49</td><td rowspan=1 colspan=1>1.50 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>M73</td><td rowspan=1 colspan=1>3.00 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>M75</td><td rowspan=1 colspan=1>3.00 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>M77</td><td rowspan=1 colspan=1>3.00 x 103</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>M78</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>M82</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>M87</td><td rowspan=1 colspan=2>1.50 x 103</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>M89</td><td rowspan=1 colspan=2>3.00 x 103</td></tr></table>

These study results are acceptable and they are described in the labeling.

# i. Microbial cross-reactivity

An in silico BLAST analysis of primers used in the Simplexa™ Group A Strep Direct assay against the NCBI database did not show evidence of cross-reactivity with organisms that might be found in throat swab specimens.

A study was performed to evaluate the performance of the Simplexa™ Group A Strep Direct assay in the presence of sixty-four (64) potentially cross-reacting bacteria and viruses commonly found in throat specimens (see Table V below). Each organism was tested in triplicate in the presence of clinically relevant levels of viruses $( 1 0 ^ { 5 } \mathrm { P F U / m l } )$ and bacteria $( 1 0 ^ { 6 } \mathrm { C F U / m l } )$ or higher. All strains were spiked into simulated negative matrix and evaluated using one (1) lot and two (2) instruments by two (2) operators.

<table><tr><td colspan="3" rowspan="1">Table V: Strains Included in Cross-Reactivity</td></tr><tr><td colspan="3" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA),ATCC 43300</td><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE), ATCC 29887</td><td colspan="1" rowspan="1">Streptococcus uberis</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ovatus</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Streptococcus vestibularis</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Treponema denticola</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">Streptococcus canis</td><td colspan="1" rowspan="1">Adenovirus 1</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Streptococcus constellatus subsp.constellatus</td><td colspan="1" rowspan="1">Adenovirus 7A</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Streptococcus cristatus</td><td colspan="1" rowspan="1">Coronavirus 229E</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">Streptococcus equi subsp.zooepidemicus</td><td colspan="1" rowspan="1">Enterovirus 71</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Streptococcus equinus</td><td colspan="1" rowspan="1">Epstein-Barr virus (B95-8)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">HSV-1 McIntyre</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">HSV-2 G</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">Influenza A/Hong Kong/8/68H3N2</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Influenza B/Panama/45/90</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">Metapneumovirus-9</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">Parainfluenza 1</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">Parainfluenza 2</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Parainfluenza 3</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus micros</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">Rhinovirus 1A</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">RSV-B 9320</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

None of the sixty-four (64) bacteria and viruses tested that might be found in throat swab specimens cross-react with the assay.

These study results are acceptable and these results are described in labeling.

# ii. Microbial interference

Interference studies were conducted with each of the sixty-four (64) microorganisms listed above in Table V. Each organism was tested in triplicate in the presence of clinically relevant levels of viruses $( 1 0 ^ { 5 } \mathrm { P F U / m l } )$ and bacteria ${ ( 1 0 ^ { 6 } }$ $\mathrm { C F U / m l }$ ) or higher. All strains were spiked into simulated negative matrix along with $2 { \cdot } 4 \mathrm { ~ x ~ L o D }$ Group A Streptococcus serotypes M1 and M3 and evaluated using one (1) lot and two (2) instruments by two (2) operators. None of the organisms interfered with the detection of the Group A Streptococcal strains.

# iii. Interfering substances

Twenty-nine (29) chemical and biological substances likely to be found in throat swab specimens were evaluated for potential to interfere with the Simplexa™ Group A Strep Direct assay, including blood $( 1 0 \% \mathrm { v / v ) }$ , mucin $( 6 0 \mu \mathrm { g / m l } )$ and saliva $( 5 0 \mu \mathrm { l / s w a b ) }$ . Each substance was tested in triplicate using two serotypes of Streptococcus pyogenes (M1 and M3), tested near $3 \mathrm { ~ x ~ }$ LoD at medically relevant concentrations. None of the substances tested were found to interfere with the Simplexa™ Group A Strep Direct assay.

# iv. Carryover/Cross-contamination

A study to assess carryover contamination was conducted with another previously cleared kit (k120413) and the results were determined to be acceptable. Due to the similarities of risks posed by the previous device and the Simplexa™ Group A Strep Direct device, no carryover studies were conducted for this assay.

# g. Assay cut-off:

A Receiver Operating Characteristic (ROC) analysis was conducted to determine the cycle threshold (Ct) cutoff for the Simplexa™ Group A Strep Direct assay. Using ROC analysis and the Ct spread from the LoD as well as the clinical study, the Ct cutoff was set at 45. This value was used to determine assay results in the analytical and clinical studies.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable.

b. Matrix comparison:

A comparison study was conducted between negative clinical matrix and a contrived negative matrix in order to validate the use of the contrived negative matrix in place of a clinical negative matrix for the analytic studies in section M1 above. Contrived negative matrix was comprised of Amies medium with $9 \%$ w/v mucin and $1 \%$ v/v whole blood, constructed to mimic challenging clinical specimens. The matrix comparison study results are shown in Table VI below.

<table><tr><td rowspan=1 colspan=6>Table VI: Matrix Comparison Study</td></tr><tr><td rowspan=2 colspan=2>Panel ID</td><td rowspan=1 colspan=2>SimulatedNegative Matrix</td><td rowspan=1 colspan=2>Swab NegativeClinical Matrix</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=3 colspan=1>Group A StreptococcusSerotype M1</td><td rowspan=1 colspan=1>10 x LoD</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2 x LoD</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Group A StreptococcusSerotype M3</td><td rowspan=1 colspan=1>10 x LoD</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2 x LoD</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

These studies demonstrate that the contrived negative matrix is produces results that are equivalent to those obtained with a clinical matrix. These study results are acceptable.

# 3. Clinical studies:

Performance characteristics of the Simplexa™ Group A Strep Direct assay were established during a prospective study using fresh left-over throat swabs collected from and tested at four (4) clinical sites across the United States between May 6, 2014 and October 28, 2014. The assay was evaluated for the qualitative detection and differentiation of Group A $\beta$ -hemolytic Streptococcus nucleic acids in comparison to culture. Discrepant analysis was conducted using bi-directional sequencing.

One thousand three hundred and ninety-seven (1397) specimens collected during clinical trials, of which one thousand three hundred and ninety-six (1396) were tested with the Simplexa™ Group A Strep Direct. Of these, one thousand three hundred and ninety-two (1392) specimens provided valid results from the Simplexa™ Group A Strep Direct and one thousand three hundred and fifty-seven (1357) were evaluable by culture. Of these, one thousand three hundred and fifty-two (1352) specimens were evaluable by both the test and reference method and could be used to evaluate the Simplexa™ Group A Strep Direct. The invalid rate observed during the clinical prospective study for this device was

$0 . 5 7 \%$ (8/1396). These results are shown in Table VII below for all sites combined and for each study site separately.

a. Clinical Sensitivity:   

<table><tr><td colspan="4" rowspan="1">Table VII: Clinical Performance Data for theSimplexaTM Group A Strep Direct vs. CompositeCultures for Group A β-hemolytic Streptococcus</td></tr><tr><td colspan="4" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="2">SimplexaTMGroup AStrep Direct</td><td colspan="3" rowspan="1">Composite Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">152</td><td colspan="1" rowspan="1">57*</td><td colspan="1" rowspan="1">209</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4**</td><td colspan="1" rowspan="1">1139</td><td colspan="1" rowspan="1">1143</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="1">1196</td><td colspan="1" rowspan="1">1352</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 97.4% (152/156) 95% CI (93.6% - 99.0%)Specificity: 95.2% (1139/1196) 95% CI (93.9% - 96.3%)</td></tr><tr><td colspan="4" rowspan="1">* Of the 57 discordant specimens, 46 were positive forGAS when tested with bi-directional sequencing, 9 werenegative and 2 were indeterminate.** Of the 4 discordant specimens, 2 were GAS negativewhen tested with bi-directional sequencing.</td></tr><tr><td colspan="4" rowspan="1">Site 1</td></tr><tr><td colspan="1" rowspan="2">SimplexaTMGroup AStrep Direct</td><td colspan="3" rowspan="1">Composite Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">10*</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1**</td><td colspan="1" rowspan="1">144</td><td colspan="1" rowspan="1">145</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">154</td><td colspan="1" rowspan="1">183</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 96.6% (28/29) 95% CI (82.8% - 99.4%)Specificity: 93.5% (144/154) 95% CI (88.5% - 96.4%)</td></tr><tr><td colspan="4" rowspan="1">* Of the 10 discordant specimens, 7 were positive for GASwhen tested with bi-directional sequencing, 3 werenegative.** The 1 discordant specimen was GAS positive whentested with bi-directional sequencing.</td></tr><tr><td colspan="4" rowspan="1">Site 2</td></tr><tr><td colspan="1" rowspan="2">SimplexaTMGroup AStrep Direct</td><td colspan="3" rowspan="1">Composite Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">14*</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1**</td><td colspan="1" rowspan="1">435</td><td colspan="1" rowspan="1">436</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">449</td><td colspan="1" rowspan="1">497</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 97.9% (47/48) 95% CI (89.1% - 99.6%)Specificity: 96.9% (435/449) 95% CI (94.8%- 98.1%)</td></tr><tr><td colspan="4" rowspan="1">* Of the 14 discordant specimens, 11 were positive forGAS when tested with bi-directional sequencing, 2 werenegative and 1 was indeterminate.** The 1 discordant specimen was GAS negative whentested with bi-directional sequencing.</td></tr><tr><td colspan="4" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="2">SimplexaTMGroup AStrep Direct</td><td colspan="3" rowspan="1">Composite Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">23*</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">215</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">282</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100.0% (44/44) 95% CI (92.0% to 100.0%)Specificity: 90.3% (215/238) 95% CI (85.9% - 93.5%)</td></tr><tr><td colspan="4" rowspan="1">* Of the 23 discordant specimens, 19 were positive forGAS when tested with bi-directional sequencing, 3 werenegative and 1 was indeterminate.</td></tr><tr><td colspan="4" rowspan="1">Site 4</td></tr><tr><td colspan="1" rowspan="2">SimplexaTMGroup AStrep Direct</td><td colspan="3" rowspan="1">Composite Culture</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">10*</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2**</td><td colspan="1" rowspan="1">345</td><td colspan="1" rowspan="1">347</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">355</td><td colspan="1" rowspan="1">390</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 94.3% (33/35) 95% CI (81.4% - 98.4%)Specificity: 97.2% (345/355) 95% CI (94.9% - 98.5%)</td></tr><tr><td colspan="4" rowspan="1">* Of the 10 discordant specimens, 9 were positive for GASwhen tested with bi-directional sequencing.** Of the 2 discordant specimens, 1 was GAS negativewhen tested with bi-directional sequencing.</td></tr></table>

The positive control for this assay is an inactivated GAS strain. The negative control is a no-template control. The positive and negative external quality control isolates provided were tested at least once per day during the clinical studies for each instrument used in the study for that day. One positive control lot was used throughout the clinical trials. All valid Group A Streptococcus positive controls were detected accurately $( 1 0 0 \%$ , 203/203). All valid no-template controls were detected accurately ( $100 \%$ , 204/204). During these studies, there were four (4) invalid negative controls and five (5) invalid positive controls. In each instance, the controls were repeated successfully during the same day on the same instrument.

b. Clinical specificity: See table above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

The overall incidence of Group A $\beta$ -hemolytic Streptococcus in patients tested during this study was $1 1 . 2 \%$ (152/1352) based on culture results. All clinical specimens collected during this study were collected between May 6, 2014 and October 28, 2014.

# N. Instrument Name:

3M Integrated Cycler

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes _ or No __ _X

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No ___ X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____ X___ or No

3. Specimen Identification: Specimens are identified by scanning or typing in the sample identifier.

4. Specimen Sampling and Handling:

Swab specimens are expressed in transport medium. $5 0 \mu \mathrm { l }$ of the expressed specimen are transferred to the Sample Well of the Direct Amplification Disk and the disk is loaded into the Integrated Cycler for automated extraction, amplification and detection. See section I above for more information.

5. Calibration:

End-user calibration for the Integrated Cycler is not necessary. Calibration of the optical modules (excitation and emission gain settings) is performed during the manufacturing process and the values are stored in the instrument firmware.

6. Quality Control:

See section M.1.c for information on internal and external controls.   
See section ${ \bf M } . 3 . a$ for information on external control performance during clinical trials.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.