# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K143653   
B. Purpose for Submission: New device   
C. Measurand: Adenovirus, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza B, Parainfluenza virus 1, Parainfluenza virus 2, Parainfluenza virus 3, Parainfluenza virus 4, Rhinovirus, Respiratory Syncytial Virus A, Respiratory Syncytial Virus B, Bordetella pertussis, Bordetella holmesii, and Bordetella parapertussis/Bordetella bronchiseptica nucleic acid target sequences.   
D. Type of Test: A multiplexed nucleic acid test intended for use with the automated Verigene System for the qualitative in vitro detection and identification of multiple respiratory pathogen nucleic acids in nasopharyngeal swabs (NPS) collected in viral transport media and obtained from individuals suspected of respiratory tract infections.   
E. Applicant: Nanosphere, Inc.   
F. Proprietary and Established Names: Verigene® Respiratory Pathogens Flex Nucleic Acid Test (RP Flex) Common Name: Verigene® RP Flex

G. Regulatory Information:

<table><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Regulation Section</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class Ⅱ</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class Ⅱ</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZE</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZZ</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">001</td><td colspan="1" rowspan="1">Class Ⅱ</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation forClinical Multiplex TestSystems</td><td colspan="1" rowspan="1">Clinical Chemistry(75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The Verigene® Respiratory Pathogens Flex Nucleic Acid Test (RP Flex) is a multiplexed qualitative test intended for the simultaneous detection and identification of multiple viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infection. The test is performed on the automated Verigene System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and microarray hybridization to detect gene sequences of the following organism types and subtypes:

<table><tr><td>Viruses</td><td>Bacteria</td></tr><tr><td>Adenovirus Human Metapneumovirus</td><td>Bordetella parapertussis/bronchiseptica Bordetella holmesii Bordetella pertussis</td></tr><tr><td>Influenza A (Subtype H1) Influenza A (Subtype H3) Influenza B</td><td></td></tr><tr><td>Parainfluenza 1</td><td></td></tr><tr><td>Parainfluenza 2</td><td></td></tr><tr><td>Parainfluenza 3</td><td></td></tr><tr><td>Parainfluenza 4</td><td></td></tr><tr><td>Respiratory Syncytial Virus A</td><td></td></tr><tr><td></td><td></td></tr><tr><td>Respiratory Syncytial Virus B</td><td></td></tr><tr><td>Rhinovirus</td><td></td></tr></table>

Detecting and identifying specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or patient management decisions.

Negative results in the presence of a respiratory illness do not preclude respiratory infection and may be due to infection with pathogens that are not detected by this test or lower respiratory tract infection that is not detected by an NPS specimen. Conversely, positive results do not rule-out infection or co-infection with organisms not detected by RP Flex. The agent(s) detected may not be the definite cause of disease. The use of additional laboratory testing and clinical presentation may be necessary to establish a final diagnosis of respiratory infection.

Clinical evaluation indicates a lower sensitivity specific to RP Flex for the detection of Rhinovirus. If infection with Rhinovirus is suspected, negative samples should be confirmed using an alternative method.

Performance characteristics for Influenza A were established when Influenza A/H1 (2009 Pandemic) and A/H3 were the predominant Influenza A viruses in circulation. RP Flex may not detect novel Influenza A strains. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions used specifically for novel virulent influenza viruses and sent to appropriate health authorities for testing. Viral culture should not be attempted in these cases unless a biosafety level (BSL) $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Verigene® System

# I. Device Description:

The VerigeneRP Flex is performed on the Verigene System. The Verigene System consists of a Verigene Reader and multiple Verigene Processor $S P$ modules or units. Each Processor $S P$ module processes one sample at a time under the control of the Verigene Reader. The RP Flex components required to perform the test include the following single-use, disposables:

RP Flex Test Cartridge   
• RP Flex Extraction Tray (with Tip Holder Assembly)   
• RP Flex Amplification Tray Verigene Sample Well Caps

Prior to initiating a test on the Verigene Processor $S P$ , a $2 0 0 \mu \mathrm { L }$ aliquot of NPS in Viral Transport Media (VTM) is pipetted into the Sample Loading Well within the Extraction Tray and a Sample Well Cap is used to cover the Sample Loading Well. The Extraction Tray, Tip Holder, and Amplification Tray are loaded into the Verigene Processor $S P$ . Next, the barcode located on the RP Flex Test Cartridge is entered via the scanner attached to the Verigene Reader, and the associated sample information is entered either using the barcode-scanner or the Verigene Reader touch-screen interface. This links specific patient information to a specific Test Cartridge number. The RP Flex Test Cartridge is then inserted into the Processor $S P$ . The Drawer Assembly is closed to initiate the test. The Processor $S P$ identifies the Test Cartridge via an internal barcode scanner and communicates with the Verigene Reader to receive test instructions. Once the Processor $S P$ module completes processing (about 2 hours), the RP Flex Test Cartridge is removed and inserted into the Verigene Reader for automated identification of the gene-specific nucleic acids.

The RP Flex detects a total of 17 nucleic acid targets as listed in Table 1 below. For each intended RP Flex target, four sequence components, referred to as oligonucleotides (Oligos), consisting of one or more Capture probe(s), Mediator probe(s), forward primer(s), and reverse primer(s), are required.

Table 1: A summary of the RP Flex Targets   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target Gene(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Hexon</td></tr><tr><td rowspan=2 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L) for SpeciesA</td></tr><tr><td rowspan=1 colspan=1>Nucleoprotein (NP) for Species B</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Matrix Protein (M)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Non-Structural Protein (NS)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Fusion Protein (F)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Nucleoprotein (NP)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Phosphoprotein (P)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>5&#x27;- UTR</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>Fusion Protein (F)</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis/Bordetellabronchiseptica</td><td rowspan=1 colspan=1>gidA</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>fumC</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Toxin Promotor Region</td></tr></table>

# Materials Provided with the Verigene RP Flex Kit:

Verigene RP Flex Test Kit (Catalog # 20-005-024)

20 Verigene RP Flex Test Cartridges   
Each Test Cartridge comes preloaded with all required reaction reagents, including wash solutions, oligonucleotide probe solution and signal amplification solutions required to generate a test result. The Test Cartridges are contained within a carrier labeled as “RP; 20-006-024”.

20 Verigene RP Flex Test Extraction Trays (with Tip Holder Assemblies) Each Extraction Tray comes preloaded with all required reagents, including lysis/binding buffer, wash solutions, and buffer solutions necessary to extract nucleic acids and generate a test result. The Extraction Trays are contained within a carrier labeled as “RP; 20-009-024”.

20 Verigene Sample Well Caps   
The Caps come packaged in strips of 5 Caps and are contained within a plastic bag. The bag is labeled as “40-001-001”.

Verigene RP Flex Amplification Kit (Catalog # 20-012-024)

20 Verigene RP Flex Amplification Trays   
Each Amplification Tray comes preloaded with all required reagents, including enzymes and buffers necessary to amplify nucleic acids and generate a test result as well as an amplification tube. The Amplification Trays are contained within a carrier labeled as “RP; 20-011-024”.

# Materials Needed but Not Provided with the Verigene RP Flex Kit:

Instruments and Equipment

• Verigene Reader (Catalog # 10-0000-02)   
• Verigene Processor $S P$ (Catalog # 10-0000-07)   
• Barcode Scanner $2 { - } 8 ^ { \circ } \mathrm { C }$ Refrigerator $\leq - 2 0 ^ { \circ } \mathrm { C }$ Freezer   
• $\le - 7 0 ^ { \circ } \mathrm { C }$ Freezer (Optional)   
- Micro-pipettors & filtered tips   
• Vortex   
• Decontamination wipes/spray or comparable sanitizer Biological Safety Cabinet (BSC) Verigene Extraction Tray Holder (Catalog # 421-00019-01) (Optional) Test Cartridge Cover Opener (Optional)

# Interpretation of Results

Verigene RP Flex provides a qualitative result for the presence (Detected) or absence (Not Detected) of the Verigene RP Flex target genes. The image analysis of the Substrate provides light signal intensities from the target-specific capture spots as well as the internal processing controls, negative control, background, and imaging control spots. The mean signal intensity of a target is compared to the assay’s signal detection threshold to make a determination. Table 2 below lists the possible test results generated by Verigene RP Flex representing identification of viral and bacterial nucleic acid sequences/targets.

Table 2: RP Flex Calls for Valid Tests   

<table><tr><td rowspan=1 colspan=1>Test Result Reported as &quot;Detected&quot;</td><td rowspan=1 colspan=1>Target Genes</td></tr><tr><td rowspan=1 colspan=2>Viral Targets</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Hexon</td></tr><tr><td rowspan=2 colspan=1>hMPV</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L) for Species A</td></tr><tr><td rowspan=1 colspan=1>Nucleoprotein (NP) for Species B</td></tr><tr><td rowspan=1 colspan=1>Influenza A*</td><td rowspan=1 colspan=1>Matrix Protein (M)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1**</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3**</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Non-Structural Protein (NS)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Fusion Protein (F)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Nucleoprotein (NP)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Phosphoprotein (P)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>5&#x27;- UTR</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>Polymerase/Large Protein (L)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>Fusion Protein (F)</td></tr><tr><td rowspan=1 colspan=2>Bacterial Targets</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis/bronchiseptica***</td><td rowspan=1 colspan=1>gidA</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>fumC</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Toxin Promotor Region</td></tr><tr><td rowspan=1 colspan=2>Test Result Reported as &quot; Not Detected&quot;</td></tr><tr><td rowspan=1 colspan=2>All Analytes Not Detected</td></tr></table>

\*Detection of Influenza A without an Influenza A/H1 or Influenza A/H3 subtype may occur at low titer of the virus in the specimen or may indicate a false positive due to contamination. The result could also indicate a novel Influenza A strain. In these cases, the sample should be retested. If an Influenza A detected result is obtained without detection of an Influenza A/H1 or A/H3 subtype upon retesting, contact local or state public health authorities for confirmatory testing.

$^ { \ast \ast }$ Detection of Influenza A/H1 or Influenza A/H3 subtypes without an Influenza A “Detected” result may occur at low titer of the virus in the specimen or may indicate a false positive due to contamination. The result could also indicate potential genetic mutations in the Matrix protein gene among circulating seasonal Influenza A viruses. In these cases, the sample should be retested. If an Influenza A/H1 or A/H3 subtype detected result is obtained again without detection of Influenza A upon repeat testing, further investigations may be warranted.

\*\*\*Since the RP Flex Bordetella parapertussis/bronchiseptica probes also detect Bordetella pertussis, if a Bordetella pertussis “Detected” result is obtained, the results for Bordetella parapertussis/bronchiseptica are not to be considered as they do not indicate the presence or absence of Bordetella parapertussis / Bordetella bronchiseptica. The result of Bordetella parapertussis/bronchiseptica is reported as “N/A” upon a “Detected” result for Bordetella pertussis.

Calls related to an invalid Verigene RP Flex test are listed in Table 3 below, together with the appropriate recourse which should be taken by the user.

Table 3: RP Flex Invalid Calls and Recourse   

<table><tr><td rowspan=1 colspan=1>Call</td><td rowspan=1 colspan=1>Reason</td><td rowspan=1 colspan=1>Recourse</td></tr><tr><td rowspan=1 colspan=1>No Call  INT CTL 1</td><td rowspan=1 colspan=1>Internal Control 1 not detected,indicating a target hybridizationissue.</td><td rowspan=1 colspan=1>Repeat Verigene RP Flex.</td></tr><tr><td rowspan=1 colspan=1>No Call  INT CTL 2</td><td rowspan=1 colspan=1>Internal Control 2 not detected,indicating a lysis, extraction,amplification issue, or a targethybridization issue.</td><td rowspan=1 colspan=1>Repeat Verigene RP Flex.</td></tr><tr><td rowspan=1 colspan=1>No Call  INT CTL</td><td rowspan=1 colspan=1>Internal Control 1 and Control 2 notdetected, indicating a lysis,extraction, amplification issue,and/or a target hybridization issue.</td><td rowspan=1 colspan=1>Repeat Verigene RP Flex.</td></tr><tr><td rowspan=1 colspan=1>No Call  NO GRID</td><td rowspan=1 colspan=1>Reader unable to image Substrate.</td><td rowspan=1 colspan=1>Ensure Substrate is seated properlyin the Substrate Holder. Repeatimage analysis by selecting &quot;Menu&quot;and &quot;Enter Barcode&quot; and thenscanning the Substrate Holderbarcode. If the No Call persists,repeat Verigene RP Flex.</td></tr><tr><td rowspan=1 colspan=1>No Call - VARIATION</td><td rowspan=3 colspan=1>Reader unable to obtain resultbecause of high variability in thetarget-specific signals.</td><td rowspan=3 colspan=1>Repeat Verigene RP Flex.</td></tr><tr><td rowspan=1 colspan=1>No Call - BKGD</td></tr><tr><td rowspan=1 colspan=1>No Call  NEG CTL</td></tr><tr><td rowspan=1 colspan=1>Processing Error</td><td rowspan=1 colspan=1>Pre-Analysis Error (PAE): Internalchecks within the Processor SPdetected an unexpected event.</td><td rowspan=1 colspan=1>Power cycle the Processor SP andrepeat RP Flex.</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray® Respiratory Panel (RP) System (BioFire Diagnostics, Inc.)

2. Predicate K number(s):

K143080   
K123620   
K120267   
K110764   
K103175

3. Comparison with predicate(s):

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">Verigene RP Flex</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel (RP) System</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Verigene® Respiratory Pathogens FlexNucleic Acid Test (RP Flex) is amultiplexed qualitative test intended for the</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">simultaneous detection and identification ofmultiple viral and bacterial nucleic acids innasopharyngeal swabs (NPS) obtained fromindividuals suspected of respiratory tractinfection.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Adenovirus, Rhinovirus, HumanMetapneumovirus, Influenza A, InfluenzaA (subtype H1), Influenza A (subtype H3),Influenza B, Parainfluenza 1,Parainfluenza 2, Parainfluenza 3,Parainfluenza 4, Respiratory SyncytialVirus A, Respiratory Syncytial Virus B,and Bordetella pertussis</td><td colspan="1" rowspan="1">Same(See below for differences)</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid amplification</td><td colspan="1" rowspan="1">Same(See below for differences)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Nasopharyngeal swabs eluted in VTM</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">Verigene RP Flex</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel (RP) System</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Cannot distinguish Influenza A subtype2009 H1 from Influenza A subtype H1.•Can distinguish Respiratory SyncytialVirus Type A from Respiratory SyncytialVirus Type B.Detects Rhinovirus, but not Enterovirus.Detects and distinguishes Bordetellaparapertussis/bronchiseptica andBordetella holmesii from Bordetellapertussis.</td><td colspan="1" rowspan="1">• Can distinguish Influenza A subtype 2009H1 from Influenza A subtype H1.Cannot distinguish Respiratory SyncytialVirus Type A from Respiratory SyncytialVirus Type B.•Cannot distinguish Rhinovirus fromEnterovirus.Detects Coronavirus 229E, CoronavirusHKU1, Coronavirus NL63, CoronavirusOC43, Chlamydophila pneumoniae,Mycoplasma pneumoniae and Enterovirus.</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex RT-PCR followed by ampliconshybridization to target specific captureoligonucleotides in a microarray format andmediator and gold-nanoparticle probehybridization to captured amplicons.</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followed by highresolution melting analysis to confirmidentity of amplified product.</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">Gold nanoparticle with silver enhancementprobe-based endpoint detection</td><td colspan="1" rowspan="1">Real-time fluorescence detection during thesecond stage of the nested RT-PCR followedby endpoint high resolution melting analysisto confirm identity of amplified product.</td></tr><tr><td colspan="1" rowspan="1">OpticalDetection</td><td colspan="1" rowspan="1">Light scattering</td><td colspan="1" rowspan="1">Fluorescence</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Verigene System</td><td colspan="1" rowspan="1">FilmArray instrument</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">About 2 hours</td><td colspan="1" rowspan="1">About 1 hour</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample Processing is automated in theVerigene System.</td><td colspan="1" rowspan="1">Sample Processing is automated in theFilmArray instrument.</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Some reagents need to be stored atrefrigerated or freezing temperatures.</td><td colspan="1" rowspan="1">Reagents are stored at room temperature.</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

• CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline - Second Edition   
• CLSI EP12-A2; User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition   
• CLSI MM3-A2; Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline - Second Edition   
• CLSI EP15-A2 - User Verification of Performance for Precision and Trueness; Approved Guideline – second edition   
• CLSI EP9-A2 - Method Comparison and Bias Estimation Using Patient Samples; Approved Guideline – second edition CLSI EP17-A2 - Protocols for Determination of Limits of Detection and Limits of Quantitation; Approved Guideline FDA guidance document issued on April 25, 2006, titled “Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable” FDA guidance document issued on May 11, 2005, titled “Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices” FDA guidance document issued on March 10, 2005, titled “Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems” FDA guidance document issued on June 25, 2010, titled “In Vitro Diagnostic (IVD) Device Studies – Frequently Asked Questions”   
• FDA guidance document issued on August 27, 2014, titled “Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices” FDA guidance document issued on March 13, 2007, titled “Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests” FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay” FDA guidance document issued on July 15, 2011, titled “Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses” FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays” FDA guidance document issued on October 9, 2009, titled “Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays”

# L. Test Principle:

The Verigene RP Flex test is a multiplexed molecular assay with automated isolation, amplification, and detection of unique genomic sequences of target pathogens. Verigene RP Flex is performed using the Verigene System, which is a bench-top sample-to-result molecular diagnostics workstation consisting of two modules: the Verigene Processor $S P$ and the Verigene Reader. The Verigene Processor $S P$ automates the Verigene RP Flex sample analysis steps including: (1) Specimen Extraction—Magnetic bead-based RNA/DNA extraction from nasopharyngeal swab specimens obtained from symptomatic patients; (2) Target Amplification--Multiplex RT-PCR- and PCR-based amplification of the extracted nucleic acids to generate target-specific amplicons; (3) Hybridization—Amplicon hybridization to target specific capture DNA in a microarray format and mediator and goldnanoparticle probe hybridization to captured amplicons. Silver enhancement of the gold nanoparticle probes bound at the capture sites results in gold-silver aggregates that are imaged optically with high efficiency by the Verigene Reader. The Verigene Reader also serves as the user interface and central control unit for the Verigene System, storing and tracking information throughout the assay process.

The Verigene Processor $S P$ utilizes single-use consumables to perform Verigene RP Flex test, including an Extraction Tray, Amplification Tray and Test Cartridge. A separate Tip Holder Assembly contains two pipette tips that are used to transfer and mix reagents during the assay. The user tests a specimen by loading the single-use consumables into the Verigene Processor $S P$ , pipetting the prepared specimen into the Extraction Tray, and initiating the protocol on the Verigene Reader by scanning or entering the Test Cartridge ID and specimen information. Following assay completion, the user inserts the Substrate Holder portion of the Test Cartridge into the Verigene Reader for optical analysis and generation of Verigene RP Flex test results.

# Extraction

The isolation consists of an initial lysis of the viral or bacterial cells using proteases and chaotropic agents followed by the binding of the released nucleic acids to magnetic beads. The nucleic acid-bound magnetic beads are washed to eliminate cell debris, lysis buffers, and potential interferents, following which, the nucleic acids are eluted off the beads and made available for amplification.

# Amplification

The nucleic acid amplification consists of a first step reverse transcription (RT) to convert any viral genomic RNA to complimentary DNA (cDNA) amenable to subsequent multiplexed PCR amplification. The multiplexed RT-PCR step is conducted in a single closed tube that contains both the reverse transcriptase and the PCR polymerase enzymes along with the target-specific primers, dNTP nucleotides, and buffers.

# Contamination Control

The RP Flex test has multiple mitigations in place to handle potential contamination of PCR. First, each test is performed with a set of single-use consumables that inherently minimize contamination due to carryover and cross-contamination. Second, the RP Flex test includes a Uracil DNA Glycosylase (UDG) enzyme-based strategy to eliminate amplicon contamination.

Briefly, the dNTP formulation contains deoxyuridine triphosphate (dUTP) in place of the standard deoxythymidine triphosphate (dTTP), and during the multiplexed RT-PCR step dUTP is incorporated into the amplicons. Prior to the start of an amplification step, the UDG enzyme renders any dUTP-containing amplicons non-amplifiable by selectively hydrolyzing at the uracil base. The RP Flex test uses a thermolabile version of UDG enzyme which is inactivated by heat prior to the RT step and does not interfere with the newly-generated cDNA and/or the amplicon from the test. While the UDG-based strategy mitigates false positive risk due to lab-based carryover and cross-contamination, incomplete hydrolysis of uracilcontaining amplicons may lead to amplification and detection of a contaminant. Additionally, this strategy does not address genomic contamination during the preparation of the samples. Strict adherence to the prescribed handling/preparation of samples and laboratory/system cleaning protocols and careful disposal of the used consumables can reduce the likelihood of contamination from user-based sources.

# End-Point Detection and Analysis

The target-specific amplicon is detected in an endpoint assay that utilizes a microarray format. For each of the bacterial or viral nucleic acid sequences/analytes detected by the RP Flex test, two types of oligonucleotides are required for the endpoint gold nanoparticle probe-based detection: (1) Capture oligonucleotides (or captures) and (2) Mediator oligonucleotides (or mediators). The Capture oligonucleotides are printed on the Test Substrate (the microarray) and are designed to specifically bind to one part of the analyte-specific target amplicon. The Mediator oligonucleotides bind to a different portion of the same amplicon and enable binding of gold nanoparticle probes. Notably, in a multiplexed detection system, numerous unique target-specific mediators can coexist and form unique hybridizations at the different captures on the microarray. Since all the mediators have a target specific region and a poly-A tail region, a single, universal gold nanoparticle poly-T probe is sufficient for target/mediator labeling. Silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency. Light scatter from the capture spots is imaged by the Verigene Reader and intensities from the microarray spots are processed by a decision algorithm to make calls regarding the presence (Detected) or absence (Not Detected) of a nucleic acid sequence/analyte.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision Study and Reproducibility Study :

Precision Study

A study was conducted to evaluate the precision (repeatability) of the RP Flex. RP Flex performance was assessed across several sources of variability including operators, days, consumable lots, and Verigene instruments. The Precision Study used three lots of each of the consumables (cartridges, extraction trays and amplification trays). All precision testing was performed at a single laboratory site with one Verigene reader and 12 Verigene Processor $S P s$ .

In this study, a representative test panel was tested daily in duplicate by two operators for 12 non-consecutive days for a total of 48 tests per sample. The test panel, representing all the RP Flex analytes except for $B$ . parapertussis and B. bronchiseptica, consisted of two negative samples (one negative simulated NPS matrix and one Staphylococcus aureus spiked in negative simulated NPS matrix), as well as seven positive mixed samples at two different concentrations for a total of 16 unique samples. The composition and concentration of each sample is provided in Table 4 below. Samples were prepared by spiking previously characterized and quantified organism stocks into simulated NPS matrix at Moderate Positive (5x LoD) and Low Positive (2x LoD) concentrations. Each unique sample was divided into single-use aliquots of 300 $\mu \mathrm { L }$ each and frozen at $\leq - 7 0 ^ { \circ } \mathrm { C }$ until ready for use.

Table 4: Precision/Reproducibility Test Panel for the Verigene RP Flex   

<table><tr><td colspan="1" rowspan="1">able 4: r reclsloll/kep</td><td></td><td colspan="1" rowspan="1">on tle v enlgele Nr rt</td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #1</td><td colspan="1" rowspan="1">Sample #2</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">ATCC VR-94</td><td colspan="1" rowspan="1">9.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">ATCC VR-99</td><td colspan="1" rowspan="1">3.30E+00TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #3</td><td colspan="1" rowspan="1">Sample #4</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">ATCC VR-92</td><td colspan="1" rowspan="1">1.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">3.30E+00TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #5</td><td colspan="1" rowspan="1">Sample #6</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">Brisbane 59/2007</td><td colspan="1" rowspan="1">3.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">Wash/ 18537/62</td><td colspan="1" rowspan="1">3.70E-01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #7</td><td colspan="1" rowspan="1">Sample #8</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Florida/ 02/2006</td><td colspan="1" rowspan="1">3.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">ATCC VR-340</td><td colspan="1" rowspan="1">1.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #9</td><td colspan="1" rowspan="1">Sample#10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">ATCC VR-1378</td><td colspan="1" rowspan="1">2.70E+02TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">9A</td><td colspan="1" rowspan="1">3.00E+01TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #11</td><td colspan="1" rowspan="1">Sample#12</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus3</td><td colspan="1" rowspan="1">0810016CF</td><td colspan="1" rowspan="1">3.30E+00TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">0810070CF</td><td colspan="1" rowspan="1">4.00E-02TCID50/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #13</td><td colspan="1" rowspan="1">Sample#14</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">8.10E+02 CFU/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">ATCC 51541</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td><td colspan="1" rowspan="1">2 x LoD</td><td colspan="1" rowspan="1">5 x LoD</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #15</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Limit of Detection(LoD)</td><td colspan="1" rowspan="1">Sample #16</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">ATCC 12600</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td></tr></table>

The RP Flex results obtained during the Precision Study are presented for each RP Flex target evaluated in Table 5 below. The positive percent agreement for each analyte was determined for the low and moderate positive samples. The negative percent agreement for each analyte was assessed for the negative samples, as well as all samples that did not contain the particular target organism.

Table 5: Precision Study Results   

<table><tr><td colspan="1" rowspan="2">RP Flex Target</td><td colspan="2" rowspan="1">Positive Percent Agreement(95% CI)</td><td colspan="1" rowspan="1">Negative PercentAgreement (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">Moderate Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">95.846/48 (86.0-98.8)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">99.9%670/671 (99.2-100)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">93.8%45/48 (83.2-97.9)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">100%96/96 (96.2-100)</td><td colspan="1" rowspan="1">100%96/96 (96.2-100)</td><td colspan="1" rowspan="1">100%575/575 (99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">97.9%47/48 (89.1-99.6)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">99.9%670/671 (99.2-100)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%671/671 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">100%48/48 (92.6-100)</td><td colspan="1" rowspan="1">99.7%669/671 (98.9 -99.9)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">97.9%47/48 (89.1-99.6)</td><td colspan="1" rowspan="1">100%47/47 (92.4-100)</td><td colspan="1" rowspan="1">100%672/672 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">97.9%47/48 (89.1-99.6)</td><td colspan="1" rowspan="1">100%47/47 (92.4-100)</td><td colspan="1" rowspan="1">100%672/672 (99.4-100)</td></tr></table>

# Reproducibility Study

A reproducibility study was also conducted at three testing sites to evaluate the interlaboratory reproducibility of the RP Flex. RP Flex performance was assessed across several sources of variability including locations, operators, days, sample replicates, consumable lots, and Verigene instruments. The RP Flex reproducibility study encompassed five cartridges lots, six extraction tray lots and four amplification tray lots. A total of three Verigene Readers and 39 Verigene Processor $S P$ systems were utilized across the three testing sites (one internal and two external laboratory sites), with each site using one Verigene Reader with 12 to $1 4 \mathrm { V }$ erigene Processor $S P s$ .

In this reproducibility study, the same representative test panel of samples as presented in Table 2 previously was tested daily, in triplicate, by two operators for five nonconsecutive days at three sites for a total of 90 tests per sample (3 sites x 2 operators / site x 3 replicates / operator x 5 days $= 9 0$ tests per sample).

The RP Flex results obtained during the Reproducibility Study are presented for each RP Flex target evaluated in Table 6 below. The positive percent agreement for each analyte was determined for the low and moderate positive samples. The negative percent agreement for each analyte was assessed for the negative samples, as well as all samples that did not contain the particular target organism.

Table 6: Reproducibility Study Results   

<table><tr><td colspan="1" rowspan="2">RP Flex Target</td><td colspan="2" rowspan="1">Positive Percent Agreement(95% CI)</td><td colspan="1" rowspan="1">Negative PercentAgreement (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">Moderate Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%1258/1258 (99.7 -100)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">100%89/89 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.8%1256/1259 (99.3 -99.9)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%1258/1258 (99.7 -100)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%89/89 (95.9-100)</td><td colspan="1" rowspan="1">100%1259/1259 (99.7 -100)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">98.9%89/90 (94.0-99.8)</td><td colspan="1" rowspan="1">97.8%88/90 (92.3-99.4)</td><td colspan="1" rowspan="1">100%1258/1258 (99.7 -100)</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.9%1257/1258 (99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">100%179/179 (97.9-100)</td><td colspan="1" rowspan="1">100%180/180 (97.9-100)</td><td colspan="1" rowspan="1">100%1079/1079 (99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.8%1256/1258 (99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">98.9%88/89 (93.9-99.8)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.6%1254/1259 (99.1-99.8)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.8%1255/1258 (99.3 -99.9)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.9%1257/1258 (99.6-100)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%89/89 (95.9-100)</td><td colspan="1" rowspan="1">99.9%1258/1259 (99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.8%1255/1258 (99.3 -99.9)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">96.7%87/90 (90.7-98.9)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.9%1257/1258 (99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">100%90/90 (95.9-100)</td><td colspan="1" rowspan="1">99.9%1257/1258 (99.6-100)</td></tr></table>

The RP Flex results obtained during the Reproducibility Study are presented for each RP Flex target evaluated stratified by testing sites in Table 7 below.

Table 7: Reproducibility Study Results Stratified by Testing Sites   

<table><tr><td colspan="1" rowspan="2">RP Flex Target</td><td colspan="1" rowspan="2">Site</td><td colspan="2" rowspan="1">Positive Percent Agreement (95% CI)</td><td colspan="1" rowspan="1">Negative PercentAgreement* (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 1</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 2</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/42099.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">29/29100% (88.3-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/42099.5% (98.3-99.9)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 3</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 4</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">29/29100% (88.3-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">RSV A</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">29/3096.7% (83.8-99.4)</td><td colspan="1" rowspan="1">29/3096.7% (83.8-99.4)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">29/3096.7% (83.8-99.4)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">RSVB</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="3">Influenza A</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">360/360100% (98.9-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">359/359100% (98.9-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">29/29100% (88.3-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">360/360100% (98.9-100)</td></tr><tr><td colspan="1" rowspan="3">Influenza A/H1</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">417/41999.5% (98.3-99.9)</td></tr><tr><td colspan="1" rowspan="3">Influenza A/H3</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">29/3096.7% (83.8-99.4)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/42099.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">415/41999.0% (97.6-99.6)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">29/29100% (88.3-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/42099.5% (98.3-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">Rhinovirus</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="3">hMPV</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">29/29100% (88.3-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/42099.5% (98.3-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="3">B. pertussis</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">29/3096.7% (83.8-99.4)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">420/420100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">28/3093.3% (78.7-98.2)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">418/41999.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="3">B. holmesii</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/42099.8% (98.7-99.9)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">30/30100% (88.6-100)</td><td colspan="1" rowspan="1">419/419100% (99.1-100)</td></tr></table>

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The RP Flex microarray contains two sets of oligonucleotide spots that are used as process controls to check for proper fluid control, extraction, amplification, hybridization and signal detection. The IC1 (hybridization control) internal processing control probes detect the presence (hybridization and signal enhancement) of a DNA oligonucleotide and mediator oligo contained within the Sample Buffer on the Extraction Tray. The IC2 (extraction control) control probes verify the presence of an amplicon for Bacteriophage MS2, which is added to the sample prior to the nucleic acid extraction step. Both internal processing controls IC1 and IC2 must be present for a valid respiratory pathogen Not Detected call for all targets to be reported. If IC1 or IC2 is Not Detected, a No Call – INT CTL 1 or a No Call – INT CTL 2 result, respectively, is generated. If both IC1 and IC2 are Not Detected, a No Call – INT CTL result is generated. The IC2 is not utilized for the detection of positive samples. The IC1 is required for the detection of positive samples.

A set of regions on microarray which do not contain capture oligonucleotides is used as a control for background signal levels. If the signal in these background regions is too high, a No Call - BKGD is generated. Additionally, negative control oligomer spots are included on the array to ensure the hybridization stringency was sufficient. If the negative control signal is too high a No Call – NEG CTL is generated.

Each capture probe has six separate oligonucleotide spots on the microarray in six different areas. Mathematical algorithms, called spot filters, compare the array spot signals for a particular target as a set. Any individual spot signal in the target set that is determined to be an outlier is discarded. After applying the spot filters, the probe set must include four or more capture spots for a Call decision to be made for a target. Additionally, the coefficient of variation of spot intensities in the set must be less than or equal to $70 \%$ . If these requirements are not met a No Call - VARIATION results. The spot signal comparison algorithms protect against some hybridization failures and also against fluidics failures such as the partial filling of the hybridization chamber.

External controls are not provided with the RP Flex. However, five external control mixes (see Table 8 below) were provided to the clinical study sites for daily testing during the prospective clinical study. External controls were tested on each day of testing, utilizing one external negative control and one of four external positive controls (tested on a rotating basis) representing all of the RP Flex targets.

Table 8: External Controls Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>External Control</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>RPNC10</td><td rowspan=1 colspan=1>Negative (Not Detected)</td></tr><tr><td rowspan=1 colspan=1>RPPC11</td><td rowspan=1 colspan=1>Influenza B, Parainfluenza 4, hMPV</td></tr><tr><td rowspan=1 colspan=1>RPPC12</td><td rowspan=1 colspan=1>RSV A, Parainfluenza 1, Adenovirus, Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>RPPC13</td><td rowspan=1 colspan=1>Influenza A, Influenza A/H1, RSV B, Parainfluenza 2, B. holmesii,</td></tr><tr><td rowspan=1 colspan=1>RPPC14</td><td rowspan=1 colspan=1>Influenza A, Influenza A/H3, Parainfluenza 3, hMPV, Bordetellaparapertussis/Bordetella bronchiseptica, B. pertussis</td></tr></table>

The sponsor is also recommending the following in the product package insert: “Good laboratory practice recommends running external positive and negative controls regularly. Viral transport medium may be used as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms may be used as external Positive Controls. Regardless of the choice of quality control materials, external controls should be used in accordance with local, state, federal accrediting organizations, as applicable.”

# Specimen Stability

An analytical study was performed to establish the recommended transport and storage conditions for nasopharyngeal swab (NPS) specimens eluted in VTM that will be tested using the RP Flex.

A total of 14 viral and bacterial strains in negative natural clinical NPS in VTM were evaluated. These strains are representative of all of the 16 RP Flex targets. Each strain was prepared at Low Positive (2x LoD) and Moderate Positive (5x LoD) concentrations in pooled negative natural clinical NPS in VTM. The negative natural clinical NPS (collected in VTM) used for this study consisted of nasopharyngeal swab in VTM specimens that were previously screened to be negative for any respiratory organisms in the RP Flex panel. Each sample was tested once with the RP Flex test and generated negative results for all RP Flex panel analytes.

Initial testing was performed to establish the baseline time point $( \mathfrak { t } = 0 )$ for the study, and additional aliquots of the samples were stored at each of the following temperature conditions: (1) $2 0 – 2 5 ^ { \circ } \mathrm { C }$ , (2) $2 { - } 8 ^ { \circ } \mathrm { C }$ , and (3) Frozen $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ .

At the designated time points, shown in Table 9 below, one aliquot of each strain at each concentration was tested with the RP Flex test in replicates of three. The $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 0 – 2 5 ^ { \circ } \mathrm { C }$ samples were tested immediately, while the frozen aliquots were thawed at room temperature for 10-30 minutes prior to testing.

Table 9: Overview of Specimen Stability Storage Conditions and Time Points   

<table><tr><td rowspan=2 colspan=1>Test Time Point</td><td></td><td rowspan=1 colspan=5>Storage Temperature</td></tr><tr><td></td><td rowspan=1 colspan=2>20-25°C</td><td rowspan=1 colspan=2>2-8°C</td><td rowspan=1 colspan=1>&lt;-70°C</td></tr><tr><td rowspan=1 colspan=1>Baseline (0)</td><td></td><td rowspan=1 colspan=2>X</td><td rowspan=1 colspan=2>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td></td><td rowspan=1 colspan=2>X</td><td rowspan=2 colspan=2>N/A</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>6 hours</td><td></td><td rowspan=1 colspan=2>X</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td></td><td rowspan=4 colspan=2></td><td rowspan=1 colspan=2>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>72 hours</td><td></td><td rowspan=1 colspan=2>X</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>75 hours</td><td></td><td rowspan=1 colspan=2>X</td></tr><tr><td rowspan=1 colspan=1>15 days</td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>30 days</td><td rowspan=2 colspan=3>N/A</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=2>N/A</td></tr><tr><td rowspan=1 colspan=1>35 days</td><td></td><td rowspan=1 colspan=1>X</td></tr></table>

X indicates the testing time point for each target N/A indicates the time point/storage condition was not part of the testing protocol.

The results of this specimen stability study support the stability claim for RP Flex testing of clinical NPS specimens preserved in VTM at the following storage conditions: 4 hours at $2 0 – 2 5 ^ { \circ } \mathrm { C }$ , 72 hours at $2 { \cdot } 8 ^ { \circ } \mathrm { C }$ , and 30 days at ${ < } . 7 0 { } ^ { \circ } \mathrm { C }$ .

# Simulated Nasopharyngeal Swabs in VTM Sample Matrix (Simulated NPS)

The RP Flex test is intended for use with nasopharyngeal swabs specimens (NPS) collected in Viral Transport Media (VTM). However, the collection, and screening of negative clinical natural NPS collected in VTM is burdensome. Additionally, such clinical specimens are likely to contain low concentrations of RP Flex targeted organisms which could lead to unexpected positive test results. Consequently, a simulated nasopharyngeal swab (Simulated NPS) matrix was developed and used to contrive samples for the majority of the analytical studies.

The Simulated NPS matrix was formulated to resemble the content of a clinical NPS specimen collected in VTM as closely as possible, such that the matrix would not artificially alter the performance of the test. A matrix comprised of $2 . 0 0 \mathrm { E } { + } 0 3$ HeLa cells/mL in Universal Transport Media (UTM) was selected as the composition of Simulated NPS. Analytical study demonstrated that $2 . 0 0 \mathrm { E } { + } 0 3$ HeLa cells/mL in UTM had similar DNA content as observed in NPS specimens collected in VTM from human donors. Accordingly, Simulated NPS matrix for the contrived samples used for analytical testing was UTM containing HeLa cells at a concentration of $2 . 0 0 \mathrm { E } { + } 0 3$ cells/mL.

# Equivalence of Simulated and Natural Clinical NPS

This study was designed to demonstrate the equivalence of Simulated NPS used in analytical studies, with clinically-derived natural NPS. As described in the “Simulated NPS” section previously, the Simulated NPS tested in this study was UTM containing HeLa cells at a concentration of $2 . 0 0 \mathrm { E } { + } 0 3$ cells/mL. The negative natural clinical NPS (collected in VTM) used for this study consisted of nasopharyngeal swab in VTM specimens that were previously screened to be negative for any respiratory organisms in the RP Flex panel.

Sixteen (16) of the 28 strains described in the “Detection Limit” section representing all of the RP Flex targets, were evaluated for their LoD performance in negative natural clinical NPS (collected in VTM). Freshly prepared samples in negative natural clinical NPS were tested in replicates of 20 at the LoD established for fresh Simulated NPS as detailed in the “LoD Studies” section. If all 20 replicates of the negative natural clinical NPS sample were detected $( 1 0 0 \% )$ , 3-fold lower concentrations were tested until a detection rate $\leq 9 5 \%$ was obtained. If the initial detection rate was less than $9 5 \%$ , 3-fold higher concentrations were tested until the detection rate of the intended analyte was $\geq$ $9 5 \%$ . The performance of the test with samples prepared in negative natural clinical NPS and in Simulated NPS was considered to be equivalent if the LoD for the two sample types are within one dilution (i.e. 3-fold).

Confirmation of the LoD for the 16 organisms contrived in the negative natural clinical NPS is summarized in Table 10 below. In all cases, the LoDs for strains contrived in negative natural clinical NPS were determined to be equivalent to the LoDs for the strains contrived in Simulated NPS; all LoDs were within one dilution $( + / - 3$ -fold) of the fresh Simulated NPS samples, meeting the criterion for equivalency.

Table 10: Comparison of LoDs of Fresh Simulated NPS Samples vs. Negative Natural Clinical NPS Samples   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Target Detected</td><td colspan="2" rowspan="1">LoD (TCID50 or CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Fresh in SimulatedNPS</td><td colspan="1" rowspan="1">Fresh in Negative NaturalNPS</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4 (E)</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">4.10E-02 TCID50/mL</td><td colspan="1" rowspan="1">4.10E-02 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 39 (A)</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 9 (A1)</td><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 8 (B2)</td><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">3.33E+00 TCID50/mL</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">Influenza A/Brisbane/59/2007 (H1N1)</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05 (H3N2)</td><td colspan="1" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1">3.33E+00 TCID50/mL</td><td colspan="1" rowspan="1">1.11E+00 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/2006</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">9.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">9.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">3.33E+00 TCID50/mL</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4a</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">2.70E+02 TCID50/mL</td><td colspan="1" rowspan="1">2.70E+02 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV A (A2)</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">3.33E+00 TCID50/mL</td><td colspan="1" rowspan="1">1.11E+00 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV B (Wash/18537/62)</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">3.67E-01 TCID50/mL</td><td colspan="1" rowspan="1">3.67E-01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">B. parapertussis/B.bronchiseptica</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">8.10E+02 CFU/mL</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td></tr></table>

# Fresh vs. Frozen Study

In order to utilize frozen banked clinical samples in the evaluation of RP Flex to supplement the prospective clinical study data, and to use frozen simulated samples in analytical studies, an analytical study was conducted to demonstrate that preservation of samples (by freezing at $\le - 7 0 ^ { \circ } \mathrm { C }$ ) does not affect the accuracy of the test results compared to freshly collected or freshly prepared samples.

The study evaluated the analytical sensitivity (LoD) of frozen samples of organisms prepared in Simulated NPS in comparison to LoD results of fresh samples prepared in Simulated NPS as described in the “Equivalence of Simulated and Natural Clinical NPS” study above. A panel of 16 strains, representing all of the RP Flex targets, was prepared in Simulated NPS and frozen and stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ until used in testing.

Each of these samples was tested in replicates of 20 at the same LoD established for fresh Simulated NPS as described in the “Equivalence of Simulated and Natural Clinical NPS” study above. If all 20 replicates of the frozen sample were detected $( 1 0 0 \% )$ , 3- fold lower concentrations were tested until a detection rate of $\leq 9 5 \%$ was obtained. If the initial detection rate was less than $9 5 \%$ , 3-fold higher concentrations were tested until the detection rate of the intended analyte wa $: \geq 9 5 \%$ . The frozen Simulated NPS samples were considered to be equivalent to the fresh Simulated NPS samples, if their LoDs were within one dilution (i.e. 3-fold) of each other.

Confirmation of the LoD for the 16 frozen viral and bacterial samples contrived in the Simulated NPS is summarized in Table 11 below. All LoDs for the frozen Simulated NPS samples were found to be within one dilution $( + / - 3$ -fold) of the fresh Simulated NPS samples, meeting the criterion for equivalency.

Table 11: Comparison of LoDs of Fresh vs. Frozen Viral and Bacterial Samples Contrived in the Simulated NPS (in VTM)   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Target Detected</td><td rowspan=1 colspan=2>LoD (TCID50 or CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Fresh in SimulatedNPS</td><td rowspan=1 colspan=1>Frozen in SimulatedNPS</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 4 (E)</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>4.10E-02 TCID50/mL</td><td rowspan=1 colspan=1>4.10E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 39 (A)</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 9 (A1)</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 8 (B2)</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A/Brisbane/59/2007 (H1N1)</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Wisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/02/2006</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>9.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td><td rowspan=1 colspan=1>1.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4a</td><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>2.70E+02 TCID50/mL</td><td rowspan=1 colspan=1>9.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A (A2)</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td><td rowspan=1 colspan=1>1.11E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B (Wash/18537/62)</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>3.67E-01 TCID50/mL</td><td rowspan=1 colspan=1>1.10E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>B. parapertussis/B.bronchiseptica</td><td rowspan=1 colspan=1>2.43E+03 CFU/mL</td><td rowspan=1 colspan=1>2.43E+03 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>B. holmesii</td><td rowspan=1 colspan=1>2.43E+03 CFU/mL</td><td rowspan=1 colspan=1>7.29E+03 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>8.10E+02 CFU/mL</td><td rowspan=1 colspan=1>2.43E+03 CFU/mL</td></tr></table>

This fresh vs. frozen study demonstrated that freezing and thawing of samples did not have a significant impact on the sensitivity of the RP Flex detection of the target organisms. This supports the use of the frozen NPS samples in the RP Flex analytical and clinical validation studies.

# d. Detection limit:

Limit of detection (LoD) studies were carried out with freshly-prepared samples in Simulated NPS matrix designed to resemble a natural clinical NPS specimen, as described in the “Simulated Nasopharyngeal Swabs in VTM Sample Matrix (Simulated NPS)” section previously. An equivalence study was performed which confirmed that the simulated matrix was equivalent to the natural clinical NPS matrix and does not impact RP Flex test performance. Refer to the “Equivalence of Simulated and Natural Clinical NPS” section for details of the study.

The LoDs of the RP Flex were established with the 28 representative strains of organisms listed in Table 12 below. Identities of all viral and bacterial strains used in LoD testing were confirmed by PCR and bi-directional sequencing (BDS) using validated PCR systems for each RP Flex target.

Table 12: Viral and Bacterial Strains for LoD Testing   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>C (AdV-1)</td></tr><tr><td rowspan=1 colspan=1>B (AdV-3)</td></tr><tr><td rowspan=1 colspan=1>E (AdV-4)</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>A (Rhinovirus 39)</td></tr><tr><td rowspan=1 colspan=1>B (Rhinovirus 14)</td></tr><tr><td rowspan=1 colspan=1>C (Rhinovirus C41)</td></tr><tr><td rowspan=4 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Metapneumovirus 9 (A1)</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 27 (A2)</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 3 (B1)</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 8 (B2)</td></tr><tr><td rowspan=5 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Brisbane/59/2007 (H1N1)</td></tr><tr><td rowspan=1 colspan=1>California/04/2009pdm09 (H1N1)</td></tr><tr><td rowspan=1 colspan=1>Wisconsin/67/05 (H3N2)</td></tr><tr><td rowspan=1 colspan=1>Port Chalmers/1/73 (H3N2)</td></tr><tr><td rowspan=1 colspan=1>Victoria/361/2011 (H3N2)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Florida/02/2006</td></tr><tr><td rowspan=1 colspan=1>Brisbane/60/2008</td></tr><tr><td rowspan=1 colspan=1>Massachusetts/02/2012</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>ATCC VR-94</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>ATCC VR-92</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Zeptometrix #0810016CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4a</td><td rowspan=1 colspan=1>ATCC VR-1378</td></tr><tr><td rowspan=2 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>RSV A (A2)</td></tr><tr><td rowspan=1 colspan=1>RSV B (Wash/18537/62)</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>ATCC 15311</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>ATCC 786</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>ATCC 51541</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td></tr></table>

To determine the LoDs, these previously characterized and quantitated bacterial and viral strains were used to prepare a three-fold dilution series in Simulated NPS, and each concentration was tested in replicates of four. The preliminary LoDs were assessed to be the lowest concentration in which expected analytes were detected in all four replicates. The preliminary LoD for a specific strain was then confirmed by preparing and testing 20 additional replicates at that preliminary LoD concentration and demonstrating that the intended analyte is detected $2 9 5 \%$ of the time. If the detection rate was $100 \%$ (20/20), a further 20 replicates were tested at the next lowest concentration. In the event that a detection rate below $9 5 \%$ was observed $( < 1 9 / 2 0 )$ a 3- fold higher concentration was tested.

The RP Flex results for the preliminary (upper row for each organism) and confirmatory (lower row for each organism) LoD tests are summarized in Table 13 and Table 14 for viruses and in Table 15 for bacteria. The preliminary LoD levels are presented in bold, and the final confirmed LoD concentrations are identified by the gray shaded cells.

Table 13: Preliminary and Confirmatory LoD Results for Viral Targets   

<table><tr><td></td><td colspan="1" rowspan="1">a Comrmat</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">TargetDetected</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">8.10E+02</td><td colspan="1" rowspan="1">2.70E+02</td><td colspan="1" rowspan="1">9.00E+01</td><td colspan="1" rowspan="1">3.00E+01</td><td colspan="1" rowspan="1">1.00E+01</td><td colspan="1" rowspan="1">3.33E+00</td><td colspan="1" rowspan="1">1.11E+00</td><td colspan="1" rowspan="1">3.70E-01</td><td colspan="1" rowspan="1">1.23E-01</td><td colspan="1" rowspan="1">4.10E-02</td><td colspan="1" rowspan="1">1.40E-02</td><td colspan="1" rowspan="1">4.00E-03</td></tr><tr><td colspan="1" rowspan="2">Adenovirus 1 (C)</td><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Adenovirus 3 (B)</td><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Adenovirus 4 (E)</td><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Rhinovirus 39 (A)</td><td colspan="1" rowspan="2">Rhinovirus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Rhinovirus 14 (B)</td><td colspan="1" rowspan="2">Rhinovirus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Metapneumovirus 9 (A1)</td><td colspan="1" rowspan="2">hMPV</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Metapneumovirus 27 (A2)</td><td colspan="1" rowspan="2">hMPV</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Metapneumovirus 3 (B1)</td><td colspan="1" rowspan="2">hMPV</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Metapneumovirus 8 (B2)</td><td colspan="1" rowspan="3">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/Brisbane/59/2007 (H1N1)</td><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">A/H1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/California/04/2009pdm09(H1N1)</td><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">A/H1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Influenza A/PortChalmers/1/73 (H3N2)</td><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">A/H3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/Victoria/361/2011 (H3N2)</td><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">A/H3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Influenza A/Wisconsin/67/05</td><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">(3N2)</td><td colspan="1" rowspan="2">A/H3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">InfluenzaB/Brisbane/60/2008</td><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Influenza B/Florida/02/2006</td><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">InfluenzaB/Massachusetts/02/2012</td><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">1/4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Parainfluenza 1</td><td colspan="1" rowspan="2">Parainfluenza1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Parainfluenza 2</td><td colspan="1" rowspan="2">Parainfluenza2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Parainfluenza 3</td><td colspan="1" rowspan="2">Parainfluenza3</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Parainfluenza 4a</td><td colspan="1" rowspan="2">Parainfluenza4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">RSV A (A2)</td><td colspan="1" rowspan="2">RSV A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">RSV B (Wash/18537/62)</td><td colspan="1" rowspan="2">RSV B</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

Table 14: Preliminary and Confirmatory LoD Results for Rhinovirus C41 (C)   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TargetDetected</td><td rowspan=1 colspan=7>Concentration (PFU/mL)*</td></tr><tr><td rowspan=1 colspan=1>6.56E+04</td><td rowspan=1 colspan=1>2.19E+04</td><td rowspan=1 colspan=1>7.29E+03</td><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9.00E+01</td></tr><tr><td rowspan=2 colspan=1>Rhinovirus C41 (C)</td><td rowspan=2 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* As there is no susceptible cell line to grow Rhinovirus C, the strain was cloned into a plasmid vector and transfected into WisL cells (primary human lung fibroblasts). Identity of the clone was confirmed by sequencing. The titer was established by qPCR using serial dilutions of Rhinovirus 16 as a surrogate to provide actual $\mathrm { P F U / m L }$ values for the standard curve. Therefore, it has been assumed that Rhinovirus 16 has similar virulence rates to Rhinovirus C.

Table 15: Preliminary and Confirmatory LoD Results for Bacterial Analytes   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Target Detected</td><td rowspan=1 colspan=9>Concentration (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>1.97E+05</td><td rowspan=1 colspan=1>6.56E+04</td><td rowspan=1 colspan=1>2.19E+04</td><td rowspan=1 colspan=1>7.29E+03</td><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3.00E+01</td></tr><tr><td rowspan=2 colspan=1>Bordetella parapertussis</td><td rowspan=2 colspan=1>Bordetellaparapertussis/bronchiseptica</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Bordetellabronchiseptica</td><td rowspan=2 colspan=1>Bordetellaparapertussis/bronchiseptica</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Bordetella holmesii</td><td rowspan=3 colspan=1>Bordetellaholmesii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>1/4</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Bordetella pertussis</td><td rowspan=2 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The final confirmed LoDs of the RP Flex targets determined in the LoD study for a panel of 28 strains are summarized in Table 16 and Table 17 for viral and bacterial organisms respectively. The final confirmed LoD values for each RP Flex target are shown in Table 18. In cases where multiple strains were tested, the final RP Flex LoD for a given target was assessed to be the highest LoD of the various strains.

Table 16: Confirmed RP Flex LoD Results for Viral Strains   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Confirmed LoD Titer(TCID50/mL)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1 (C)</td><td colspan="1" rowspan="1">1.23E-01</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 3 (B)</td><td colspan="1" rowspan="1">1.11E+00</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4 (E)</td><td colspan="1" rowspan="1">4.10E-02</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 39 (A)</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 14 (B)</td><td colspan="1" rowspan="1">9.00E+01</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus C14 (C)</td><td colspan="1" rowspan="1">2.43E+03*</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 9 (A1)</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 27 (A2)</td><td colspan="1" rowspan="1">1.11E+00</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 3 (B1)</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 8 (B2)</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/2007 (H1N1)</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/04/2009pdm09 (H1N1)</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers/1/73 (H3N2)</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/361/2011 (H3N2)</td><td colspan="1" rowspan="1">1.23E-01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05 (H3N2)</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Brisbane/60/2008</td><td colspan="1" rowspan="1">1.23E-01</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/2006</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Massachusetts/02/2012</td><td colspan="1" rowspan="1">1.23E-01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">9.00E+01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4a</td><td colspan="1" rowspan="1">2.70E+02</td></tr><tr><td colspan="1" rowspan="1">RSV A (A2)</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">RSV B (Wash/18537/62)</td><td colspan="1" rowspan="1">3.70E-01</td></tr></table>

\* PFU/mL

Table 17: Confirmed RP Flex LoD Results for Bacterial Strains   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Confirmed LoD Titer (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>2.43E+03</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>2.43E+03</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>2.43E+03</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>8.10E+02</td></tr></table>

Table 18: Confirmed RP Flex LoD Results for Each Analyte   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Confirmed LoD Titer(TCID50/mL or CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.11E+00</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (A/B)</td><td colspan="1" rowspan="1">9.00E+01</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (C)</td><td colspan="1" rowspan="1">2.43E+03*</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/ H1N1</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Influenza A/ H3N2</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3.00E+01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">9.00E+01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">1.00E+01</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">2.70E+02</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">3.33E+00</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">3.70E-01</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis/ bronchiseptica</td><td colspan="1" rowspan="1">2.43E+03</td></tr><tr><td>Bordetella holmesii</td><td>2.43E+03</td></tr><tr><td>Bordetella pertussis</td><td>8.10E+02</td></tr></table>

\* PFU/mL

# e. Analytical Reactivity:

The analytical reactivity of the RP Flex test was demonstrated with a comprehensive panel of 108 strains (see Table 19 below) representing temporal, evolutionary, and geographic diversity for each of the RP Flex panel organisms. The selection of strains to test was based on those detected by previously cleared assays including the Nanosphere $\mathrm { R V + }$ Nucleic Acid Test (K103209) and BioFire FilmArray Respiratory Panel (K123620, K120267, K110764, and K103175), and on FDA recommendations during the presubmission review process. Together with the 28 strains evaluated as part of the Limit of Detection Study, a total of 136 strains were evaluated for analytical inclusivity to RP Flex through wet testing.

Table 19: Organisms/Strains Tested in the Analytical Reactivity Study and the LoD Study   

<table><tr><td colspan="1" rowspan="2">Virus/Bacteria</td><td colspan="1" rowspan="2">Species/Subtype</td><td colspan="2" rowspan="1">No. of Strains Tested</td></tr><tr><td colspan="1" rowspan="1">AnalyticalReactivity</td><td colspan="1" rowspan="1">LoD</td></tr><tr><td colspan="1" rowspan="7">Adenovirus</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H2N2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H2N3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H5N3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H7N2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H7N7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H7N9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H9N2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="4">HumanMetapneumovirus</td><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="5">Parainfluenza Virus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4a</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4b</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="3">Rhinovirus</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">holmesii</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pertussis</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">parapertussis</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">bronchiseptica</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="2" rowspan="1">Total Number</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">28</td></tr></table>

The organisms in the inclusivity panel were prepared in Simulated NPS. Thirteen (13) strains of Influenza A (subtypes H2N2, H2N3, H5N1, H5N3, H7N2, H7N7, H7N9, H9N2 & H10N7) were prepared and tested at a BSL 3 laboratory. Each sample was tested with the RP Flex in triplicate at an initial concentration 3-fold higher than the LoD determined for each analyte. In cases where the expected targets were not detected in one or more replicates, concentrations at a 3-fold higher level were evaluated.

RP Flex demonstrated analytical reactivity to all 108 strains tested, with some strains requiring higher titers for detection. The individual strains and concentrations at which positive test results were obtained for all three replicates are presented by target organism in Table 20 though Table 28 below.

Table 20: Adenovirus Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>AdenovirusSpecies</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Strain #</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0810073CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>VR-7</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>VR-1099</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>VR-12</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0810108CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>VR-716</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>VR-718</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>9x</td></tr><tr><td rowspan=3 colspan=1>C</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>111010</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>0810020CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>729x</td></tr><tr><td rowspan=1 colspan=1>6*</td><td rowspan=1 colspan=1>0810111CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>243x</td></tr><tr><td rowspan=2 colspan=1>D</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0810117CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0810119CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0810084CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0810085CF</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>1x</td></tr></table>

\*Based on in silico analysis, the oligonucleotide identities of all the tested Adenovirus C subtypes have very similar ranges. Based on the investigation of viral stocks titers using a quantitative TaqMan real-time PCR developed at Nanosphere that is specific for all Adenovirus species (note: the primers for the TaqMan assay are not the same primers used in the RP Flex), it appears that the amplifiable genome equivalents available in these two adenovirus viral stocks are significantly reduced comparing to that of the other adenovirus stocks tested in the study.

Table 21: Influenza A Inclusivity Results   

<table><tr><td rowspan=2 colspan=1>Influenza ASubtype</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Source</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=2>A/H1 or A/H3</td></tr><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple ofLoD</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=9 colspan=1>H1N1</td><td rowspan=1 colspan=1>A/California/07/2009pdm09</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>9x</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>9x</td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/ 34</td><td rowspan=1 colspan=1>Charles RiverLabs</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A1/Denver/1/57</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A1/FM/1/47</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/ Solomon Islands/3/2006</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/2001</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>27x</td></tr><tr><td rowspan=4 colspan=1>H3N2</td><td rowspan=1 colspan=1>A/ Aichi/ 68</td><td rowspan=1 colspan=1>Charles RiverLabs</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>&lt;1x</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/ Hong Kong/ 8/ 68</td><td rowspan=1 colspan=1>Charles RiverLabs</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/ Victoria/ 3/ 75*</td><td rowspan=1 colspan=1>Charles RiverLabs</td><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>81x</td><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>729x</td></tr><tr><td rowspan=1 colspan=1>A/Ohio/02/2012</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>81x</td></tr><tr><td rowspan=3 colspan=1>H3N2v</td><td rowspan=1 colspan=1>A/Indiana/08/2011</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>&lt;1x</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Minnesota/11/2010**</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>81x</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>27x</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/10/2011</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>&lt;1x</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>9x</td></tr><tr><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>Japan/305/1957</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>H2N3</td><td rowspan=1 colspan=1>Mallard/Albert79/03</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Duck/Hunan/795/02</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Korea/IS/2006</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Scaly-breasted Munia/HongKong/2006</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>A/Duck/Singapore/645/1997</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>27x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/New York/107/2003</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>H7N7</td><td rowspan=1 colspan=1>A/Netherlands/219/2003</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Equine-1/Prague/1956</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>H7N9</td><td rowspan=1 colspan=1>Anhui/01/2013</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>H9N2</td><td rowspan=1 colspan=1>Hong Kong/1073/99</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Chicken/Hong Kong/G9/97</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>Chick/Germany/n/1949</td><td rowspan=1 colspan=1>MRI</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

\*Based on in silico analysis, the oligonucleotide identities of all the tested Influenza A/H3N2 strains have very similar ranges. Based on the investigation of viral stocks titers using a quantitative TaqMan real-time PCR developed at Nanosphere that is specific for Influenza A/H3 strains (note: the primers for the TaqMan assay are not the same primers used in the RP Flex), it appears that the amplifiable genome equivalents available in this Influenza A/H3N2 viral stock are significantly reduced comparing to that of the other Influenza A/H3N2 stocks tested in the study.

$^ { \ast \ast }$ Based on in silico analysis, the oligonucleotide identities to this strain have slightly lower ranges than the other two H3N2v strains tested.

Table 22: Influenza B Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiplesof</td></tr><tr><td rowspan=10 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/ Allen/45</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/2004</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/ Lee/40</td><td rowspan=1 colspan=1>Charles River Lab</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 23: Human Metapneumovirus Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofD</td></tr><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810163CF</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810158CF</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810157CF</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Zeptometrix 0810162CF</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 24: Parainfluenza 1-4 Inclusivity Results   

<table><tr><td rowspan=1 colspan=2>Type</td><td rowspan=1 colspan=1>Source/Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=1 colspan=2>Parainfluenza 1</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>Parainfluenza 2</td><td rowspan=1 colspan=1>Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=2>Parainfluenza 3</td><td rowspan=1 colspan=1>ATCC VR-93*</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>81x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>9x</td></tr><tr><td rowspan=1 colspan=1>TriCore (ATCC VR-1782)</td><td rowspan=1 colspan=1>9.00E+01</td><td rowspan=1 colspan=1>27x</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>b</td><td rowspan=1 colspan=1>VR-1377</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810060BCF</td><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

\*For Parainfluenza 3, the extracted eluate from the three strains tested in the inclusivity study were each evaluated with PCR/bi-directional sequencing, and the sequence information were used to assess the homology to the RP Flex oligos. Based on the in silico analysis, the three strains have the identical homology to the RP Flex oligos, indicating that the apparent difference in sensitivity was not due to sequence diversity in the gene targeted by the RP Flex. The apparent variation in the sensitivity of the RP Flex test for these strains is likely attributable to inconsistencies in the quantification of the viral stocks.

Table 25: RSV Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Source/Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=2 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810040ACF</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>Zeptometrix 0810040CF</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>1.11E+00</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>3.33E+00</td><td rowspan=1 colspan=1>9x</td></tr></table>

Table 26: Rhinovirus A and B Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Rhinovirus Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=8 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810012CFN</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-482</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-283</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-507</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>Rhinovirus B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC VR-1663</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ATCC VR-1137</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>ATCC VR-338</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>ATCC VR-1193</td><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 27: Rhinovirus C Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>Rhinovirus Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(PFU/mL)*</td><td rowspan=1 colspan=1>Multiples ofLoD</td></tr><tr><td rowspan=2 colspan=1>Rhinovirus C</td><td rowspan=1 colspan=1>C2</td><td rowspan=1 colspan=1>UW-Madison</td><td rowspan=1 colspan=1>7.29E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>C15</td><td rowspan=1 colspan=1>UW-Madison</td><td rowspan=1 colspan=1>7.29E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

\*As there is no susceptible cell line to grow Rhinovirus C, the strains were cloned into a plasmid vector and transfected into WisL cells (primary human lung fibroblasts). All were sequenced to confirm identity. The titers were established by qPCR using serial dilutions of Rhinovirus 16 as a surrogate to provide actual $\mathrm { P F U / m L }$ values for the standard curve. Therefore, it has been assumed that Rhinovirus 16 has similar virulence rates to Rhinovirus C.

Table 28: Bordetella Species Inclusivity Results   

<table><tr><td colspan="1" rowspan="1">Bordetella Species</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">RP FlexTarget</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">Multiples ofLoD</td></tr><tr><td colspan="1" rowspan="8">B. pertussis</td><td colspan="1" rowspan="1">ATCC 51445</td><td colspan="1" rowspan="8">B. pertussis</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 10380</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 9340</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-589</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1335</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 53894</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 9306</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 8467</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">9x</td></tr><tr><td colspan="1" rowspan="5">B. parapertussis</td><td colspan="1" rowspan="1">ATCC 15237</td><td colspan="1" rowspan="5">BordetellaParapertussis/bronchiseptica</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 9305</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-587</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 15989</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix0801461</td><td colspan="1" rowspan="1">2.19E+04</td><td colspan="1" rowspan="1">9x</td></tr><tr><td colspan="1" rowspan="6">B. bronchiseptica</td><td colspan="1" rowspan="1">ATCC 4617</td><td colspan="1" rowspan="6">BordetellaParapertussis/bronchiseptica</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 7773</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 785</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 14064</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 10580</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC 19395</td><td colspan="1" rowspan="1">7.29E+03</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="3">B. holmesii</td><td colspan="1" rowspan="1">Zeptometrix0801464</td><td colspan="1" rowspan="3">B. holmesii</td><td colspan="1" rowspan="1">2.19E+04</td><td colspan="1" rowspan="1">9x</td></tr><tr><td colspan="1" rowspan="1">ATCC 700053</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">ATCC 700052</td><td colspan="1" rowspan="1">2.43E+03</td><td colspan="1" rowspan="1">1x</td></tr></table>

# Supplemental Adenovirus Reactivity Information (in silico analyses):

In silico analysis predicts that the RP Flex will detect Adenovirus species $\mathbf { G }$ as the oligo sequence match to the target sequence for Adenovirus G is identical to that for Adenovirus A.

f. Analytical Specificity/Cross-reactivity Evaluation:

An analytical specificity study was carried out to assess the potential for false positive results due to cross-reactivity between RP Flex assays and in-panel or non-panel bacterial, viral, and fungal organisms known or expected to be present in the upper respiratory system and that may be present in NPS.

The RP Flex analytical specificity study assessed the performance of the RP Flex against organisms using contrived and clinical samples of strains that are:

1) Phylogenetically-related to RP Flex target organisms,   
2) Clinically relevant (colonize the upper respiratory tract or cause respiratory symptoms),   
3) Common skin flora or laboratory contaminants, or   
4) Microorganisms with which much of the human population may have been infected (e.g. Herpes Simplex Virus), and   
5) In-panel organisms (tested for cross-reactivity to other RP Flex targets).

The viral and bacterial samples were contrived in Simulated NPS at high concentrations $\mathrm { ( 1 . 0 0 E + 0 5 ~ T C I D _ { 5 0 } / m L }$ for viral targets and at $1 . 0 0 \mathrm { E } { + 0 6 }$ CFU/mL for bacterial and fungal targets, except for Mumps virus which was tested at the highest available concentration of $1 . 6 0 \mathrm { E } { + 0 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ). Four (4) organisms which were not available as titered stocks were evaluated using genomic DNA at $1 . 0 0 \mathrm { E } { + 0 6 }$ copies/mL. All samples were tested in triplicate with the RP Flex.

A total of 107 organisms were tested at high titer for analytical specificity (exclusivity), including 46 bacteria/fungi (Table 29), 26 viruses (Table 30), 22 in-panel organisms from the LoD Study, and 13 additional influenza A virus strains with other hemagglutinin (HA) types (Table 31).

Any RP Flex positive result with an exclusivity panel organism was tested additional six times to further assess any potential reactivity with the RP Flex probes. Additionally, it was anticipated based on feasibility studies that some Enterovirus strains may cross-react with Rhinovirus probes, and in cases of crossreactivity lower concentrations were assessed.

Table 29: Bacterial and Fungal Organisms Tested for RP Flex Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain Number</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">avium</td><td colspan="1" rowspan="1">ATCC 35086</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">hinzii</td><td colspan="1" rowspan="1">ATCC 51784</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">petrii</td><td colspan="1" rowspan="1">ATCC BAA-461</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">trematum</td><td colspan="1" rowspan="1">ATCC 700309</td></tr><tr><td colspan="1" rowspan="1">Candida</td><td colspan="1" rowspan="1">albicans</td><td colspan="1" rowspan="1">ATCC 18804</td></tr><tr><td colspan="1" rowspan="1">Candida</td><td colspan="1" rowspan="1">glabrata</td><td colspan="1" rowspan="1">ATCC 38326</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">ATCC VR-1360</td></tr><tr><td colspan="1" rowspan="1">Chlamydia</td><td colspan="1" rowspan="1">trachomatis Serovar D</td><td colspan="1" rowspan="1">ATCC VR-885</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">pseudodiphtheriticum</td><td colspan="1" rowspan="1">ATCC 10700</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">diphtheriae</td><td colspan="1" rowspan="1">ATCC 14779</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">striatum</td><td colspan="1" rowspan="1">ATCC BAA-1293</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">Haemophilus</td><td colspan="1" rowspan="1">influenzae</td><td colspan="1" rowspan="1">ATCC 49144</td></tr><tr><td colspan="1" rowspan="1">Haemophilus</td><td colspan="1" rowspan="1">parainfluenzae</td><td colspan="1" rowspan="1">ATCC 9796</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">pneumoniae subsp. pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">acidophilus</td><td colspan="1" rowspan="1">Zeptometrix 0801540</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">plantarum</td><td colspan="1" rowspan="1">ATCC BAA-793</td></tr><tr><td colspan="1" rowspan="1">Legionella</td><td colspan="1" rowspan="1">pneumophilia</td><td colspan="1" rowspan="1">ATCC 33152</td></tr><tr><td colspan="1" rowspan="1">Legionella</td><td colspan="1" rowspan="1">longbechiae</td><td colspan="1" rowspan="1">ATCC 33462</td></tr><tr><td colspan="1" rowspan="1">Legionella</td><td colspan="1" rowspan="1">micdadei</td><td colspan="1" rowspan="1">ATCC 33204</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">innocua</td><td colspan="1" rowspan="1">ATCC 51742</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">monocytogenes serotype 4b</td><td colspan="1" rowspan="1">ATCC 19115</td></tr><tr><td colspan="1" rowspan="1">Moraxella (Branhamella)</td><td colspan="1" rowspan="1">catarrhalis</td><td colspan="1" rowspan="1">ATCC 43617</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium</td><td colspan="1" rowspan="1">tuberculosis</td><td colspan="1" rowspan="1">ATCC BAA-2237D-2 a</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma</td><td colspan="1" rowspan="1">genitalium</td><td colspan="1" rowspan="1">ATCC 49123 a</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma</td><td colspan="1" rowspan="1">hominis</td><td colspan="1" rowspan="1">ATCC 27545-TTR</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">ATCC 15531-TTR</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">elongata subsp. elongata</td><td colspan="1" rowspan="1">ATCC 25295</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 31426</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">meningitidis</td><td colspan="1" rowspan="1">ATCC 53415D-5 a</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">mucosa</td><td colspan="1" rowspan="1">ATCC 49233</td></tr><tr><td colspan="1" rowspan="1">Neisseria</td><td colspan="1" rowspan="1">sicca</td><td colspan="1" rowspan="1">ATCC 29256</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis</td><td colspan="1" rowspan="1">jirovecii</td><td colspan="1" rowspan="1">Erasme-Belgium-Clinical Sample</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">marcescens</td><td colspan="1" rowspan="1">ATCC 29021</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">aureus subsp. aureus</td><td colspan="1" rowspan="1">ATCC 12600</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">haemolyticus</td><td colspan="1" rowspan="1">ATCC 29970</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">agalactiae</td><td colspan="1" rowspan="1">ATCC 12386</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="1" rowspan="1">ATCC 6303</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">pyogenes</td><td colspan="1" rowspan="1">ATCC 14289</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma</td><td colspan="1" rowspan="1">urealyticum</td><td colspan="1" rowspan="1">ATCC 27618 a</td></tr></table>

a Genomic DNA tested at $1 . 0 0 \mathrm { E } { + 0 6 }$ copies/mL

Table 30: Viral Organisms Tested for RP Flex Analytical Specificity   

<table><tr><td rowspan=1 colspan=1>Virus Name</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Source/Strain Number</td></tr><tr><td rowspan=1 colspan=1>Bocavirus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>Zeptometrix 0810229CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>Zeptometrix 0810024CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>LIJ-Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC VR-977</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>Type 71</td><td rowspan=1 colspan=1>Zeptometrix 0810047CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>Coxsackievirus A2</td><td rowspan=1 colspan=1>ATCC VR-1550</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>Coxsackievirus A10</td><td rowspan=1 colspan=1>Zeptometrix 0810106CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix 0810017CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>ATCC VR-184</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Coxsackievirus B5</td><td rowspan=1 colspan=1>ATCC VR-185</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix 0810076CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix 0810077CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix 0810023CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Echovirus 30</td><td rowspan=1 colspan=1>Zeptometrix 0810078CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>Zeptometrix 0810235CF</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC VR-1662</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>Poliovirus 2 (attenuated)</td><td rowspan=1 colspan=1>ATCC VR-301</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>Poliovirus 3 (attenuated)</td><td rowspan=1 colspan=1>ATCC VR-193</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D</td><td rowspan=1 colspan=1>Type 68</td><td rowspan=1 colspan=1>ATCC VR-561</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Zeptometrix 0810008CF</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex virus</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>Zeptometrix 0810005CF</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC VR-24</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC VR-106</td></tr><tr><td rowspan=1 colspan=1>Varicella-Zoster virus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Zeptometrix 0810026CF</td></tr></table>

Table 31: In-Panel RP Flex Organisms (Viruses and Bacteria) and Additional Influenza A Virus Strains with Other Hemagglutinin (HA) Types Tested for Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Bacteria/Virus Name</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Source/Strain Number</td></tr><tr><td colspan="1" rowspan="1">Adenovirus A</td><td colspan="1" rowspan="1">Type 31</td><td colspan="1" rowspan="1">Zeptometrix ×810073CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus D</td><td colspan="1" rowspan="1">Type 26</td><td colspan="1" rowspan="1">Zeptometrix 0810117CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus D</td><td colspan="1" rowspan="1">Type 37</td><td colspan="1" rowspan="1">Zeptometrix 0810119CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F</td><td colspan="1" rowspan="1">Type 40</td><td colspan="1" rowspan="1">Zeptometrix 0810084CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F</td><td colspan="1" rowspan="1">Type 41</td><td colspan="1" rowspan="1">Zeptometrix 0810085CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus E</td><td colspan="1" rowspan="1">Type 4</td><td colspan="1" rowspan="1">Zeptometrix 0810070CF</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 51541</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 9797</td></tr><tr><td colspan="1" rowspan="1">Influenza A /Brisbane/59/2007</td><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">TriCore</td></tr><tr><td colspan="1" rowspan="1">Influenza A /Wisconsin/67/05</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">Zeptometrix N/A</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/California/04/2009pdm09</td><td colspan="1" rowspan="1">H1N1 - pandemic</td><td colspan="1" rowspan="1">TriCore</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/361/2011</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">Zeptometrix 0810240CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H2N2 a</td><td colspan="1" rowspan="1">Japan/305/1957</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H5N1 a</td><td colspan="1" rowspan="1">A/Duck/Hunan/795/02</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H5N1a</td><td colspan="1" rowspan="1">A/Chicken/Korea/IS/2006</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H5N1a</td><td colspan="1" rowspan="1">Scaly-breastedMunia/HongKong/2006</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H7N2 a</td><td colspan="1" rowspan="1">New York/107/2003</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H7N7 a</td><td colspan="1" rowspan="1">Netherlands/219/2003</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H7N9 a</td><td colspan="1" rowspan="1">Anhui/01/2013</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H9N2 a</td><td colspan="1" rowspan="1">Hong Kong/1073/99</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H2N3 a</td><td colspan="1" rowspan="1">Mallard/Albert79/03</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H5N3 a</td><td colspan="1" rowspan="1">Duck/Singapore/645/1997</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H7N7 a</td><td colspan="1" rowspan="1">Equine-1/Prague/1956</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H9N2 a</td><td colspan="1" rowspan="1">Chicken/Hong Kong/G9/97</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H10N7 a</td><td colspan="1" rowspan="1">Chick/Germany/n/1949</td></tr><tr><td colspan="1" rowspan="1">Influenza B /Florida/02/2006</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">TriCore</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 9</td><td colspan="1" rowspan="1">Type A1</td><td colspan="1" rowspan="1">TriCore</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 8</td><td colspan="1" rowspan="1">Type B2</td><td colspan="1" rowspan="1">TriCore</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">TriCore VR-94</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">TriCore VR-92</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Zeptometrix 0810016CF</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4a</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">TriCore VR-1378</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">Type A2</td><td colspan="1" rowspan="1">TriCore VR-1540</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">Type B</td><td colspan="1" rowspan="1">TriCore VR-1580</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 14</td><td colspan="1" rowspan="1">Type B</td><td colspan="1" rowspan="1">TriCore</td></tr></table>

a Prepared and tested at a BSL 3 laboratory.

The 46 bacteria or fungi strains (as listed in Table 29 above) tested for analytical specificity on the RP Flex all returned “Not Detected” results at the concentrations tested, with the exception of Pneumocystis jirovecii (from a clinical sample), which gave “Rhinovirus detected” results in all three replicates.

In silico analysis determined that Pneumocystis jirovecii sequences have a maximum Oligo Identity to RP Flex targets of $67 \%$ and therefore Pneumocystis jirovecii was not predicted to be cross-reactive to RP Flex Rhinovirus probes. This suggests that the clinical sample may have been contaminated by Rhinovirus which is a common organism in clinical samples. Additional testing was performed using two other Pneumocystis jirovecii positive clinical samples obtained from the same source. Test results with these samples showed Rhinovirus in 1/3 and $3 / 3$ replicates, respectively. To investigate the Pneumocystis jirovecii clinical samples for potential contamination, extracted nucleic acids from all seven Rhinovirus positive tests were evaluated with an analytically validated PCR/BDS Rhinovirus assay. PCR/ BDS test results confirmed the presence of Rhinovirus in all seven samples.

The RP Flex results for the representative serotypes from Enterovirus A, B, C and D (a total of 15 strains listed in Table 30 above) are shown in Table 32 below. All Enterovirus A and B serotypes tested were exclusive to all RP Flex targets, with no cross reactivity observed at the concentrations tested. However, for three of the four Enterovirus C serotypes (Coxsackievirus A24, Poliovirus 2, and Poliovirus 3) tested at high titer, there was reported false detection of Rhinovirus. To further characterize the cross-reactivity, additional tests with these strains were performed at a lower titer. The test results indicated that, Poliovirus 2, which was detected in 2 of 3 replicates at high titer $( 1 . 0 0 \mathrm { E } + 0 5 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ was not detected at a $3 \mathbf { x }$ lower concentration ( $3 . 3 3 \mathrm { E } { + } 0 4$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ . With Coxsackievirus A24 and Poliovirus 3, cross-reactivity with Rhinovirus was detected in all three replicates at $9 \mathbf { x }$ ( $3 . 1 0 \mathrm { E } { + } 0 2$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ or $2 7 \mathbf { x }$ $( 2 . 4 3 \mathrm { E } + 0 3 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ the RP Flex Rhinovirus LoD ( $9 . 0 0 \mathrm { E } { + } 0 1$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ). RP Flex tests of Enterovirus 68 at high titer reported cross-reactivity with Rhinovirus in $2 / 3$ tests. Additional testing at $3 \mathbf { x }$ lower titer of $3 . 3 3 \mathrm { E } { + } 0 4$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , did not detect Rhinovirus in any of the three replicates.

Table 32: Analytical Specificity Testing for Enterovirus Strains   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain #</td><td rowspan=1 colspan=1>TestedConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>RhinovirusDetected</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A Type 71</td><td rowspan=1 colspan=1>Zeptometrix 0810047CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A Coxsackievirus A2</td><td rowspan=1 colspan=1>ATCC VR-1550</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A Coxsackievirus A10</td><td rowspan=1 colspan=1>Zeptometrix 0810106CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix 0810017CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Coxsackievirus B4</td><td rowspan=1 colspan=1>ATCC VR-184</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Coxsackievirus B5</td><td rowspan=1 colspan=1>ATCC VR-185</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix 0810076CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix 0810077CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix 0810023CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus B Echovirus 30</td><td rowspan=1 colspan=1>Zeptometrix 0810078CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C Coxsackievirus A21</td><td rowspan=1 colspan=1>Zeptometrix 0810235CF</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=3 colspan=1>Enterovirus C Coxsackievirus A24</td><td rowspan=3 colspan=1>ATCC VR-1662</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2.70E+02</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=2 colspan=1>Enterovirus C Poliovirus 2 (attenuated)</td><td rowspan=2 colspan=1>ATCC VR-301</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=1 colspan=1>3.33E+04</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=3 colspan=1>Enterovirus C Poliovirus 3 (attenuated)</td><td rowspan=3 colspan=1>ATCC VR-193</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2.43E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>8.10E+02</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=2 colspan=1>Enterovirus D Type 68</td><td rowspan=2 colspan=1>ATCC VR-561</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=1 colspan=1>3.33E+04</td><td rowspan=1 colspan=1>0/3</td></tr></table>

Based on in silico analyses, a number of Enterovirus strains have a relatively high homology to RP Flex Rhinovirus oligos, with percent identities to Rhinovirus RP Flex oligos of up to $84 \%$ . As a result, occasional cross- reactivity at high titer was expected.

The Enterovirus strains/subtypes which gave the highest homology to Rhinovirus probes $( 8 4 \% )$ are presented below in Table 33.

Table 33: Distribution of Enterovirus Sequences with Highest Percent Identity $( 8 4 \% )$ to RP Flex Rhinovirus Oligos   

<table><tr><td rowspan=1 colspan=1>Enterovirus Strains/Subtypes</td><td rowspan=1 colspan=1>Number of Sequences</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 1</td><td rowspan=1 colspan=1>142</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 2</td><td rowspan=1 colspan=1>123</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>49</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 3</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Human enterovirus 70</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A13</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A11</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A17</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A18</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A20</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C96</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A90</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A15</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Human enterovirus 94</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Human enterovirus C99</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A91</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Human Enterovirus Hangzhou13-02</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Human Enterovirus Ningbo3-02</td><td rowspan=1 colspan=1>1</td></tr></table>

The observed cross-reactivity of the RP Flex Rhinovirus oligos to select strains of Enterovirus and the potential cross-reactivity of the RP Flex Rhinovirus oligos to other strains of Enterovirus based on in silico analyses are indicated as a limitation in the product package insert: “Due to the genetic similarity between human Rhinovirus and Enterovirus, some strains of Enterovirus may be detected as Rhinovirus. Cross-reactivity with Human poliovirus 2, Human poliovirus 3, Enterovirus D68, and Coxsackievirus A24 was demonstrated through empirical testing.” ,,

Eleven (11) additional viruses (listed in Table 30 above) were also tested at high titer for exclusivity to the RP Flex. All were negative for RP Flex targets. A summary of these test results are presented in Table 34 below.

Table 34: Analytical Specificity Testing for Other Non-Panel Viral Strains   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source and Strain Information</td><td colspan="1" rowspan="1">TestedConcentration(TCID50/mL)</td><td colspan="1" rowspan="1">RP Flex TargetDetected</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Erasme-Belgium-Clinical Sample</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Zeptometrix 0810229CF</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Zeptometrix 0810228CF</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Zeptometrix 0810024CF</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">LIJ-Clinical Sample</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ATCC VR-977</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Zeptometrix 0810008CF</td><td colspan="1" rowspan="1">1.00E+06*</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex virus Type 1</td><td colspan="1" rowspan="1">Zeptometrix 0810005CF</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">ATCC VR-24</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">ATCC VR-106</td><td colspan="1" rowspan="1">1.60E+04</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Varicella-Zoster virus</td><td colspan="1" rowspan="1">Zeptometrix 0810026CF</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">0/3</td></tr></table>

\*Titer in genome equivalents/mL

The 22 in-panel organisms from the LoD study and the 13 Influenza A HA subtypes other than $\mathrm { A } / \mathrm { H } 1$ and $\mathrm { A } / \mathrm { H } 3$ (listed in Table 31 above) were tested in triplicate at $1 . 0 0 \mathrm { E } { + } 0 5$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for the viral targets and at $1 . 0 0 \mathrm { E + } 0 6 \mathrm { C F U / m L }$ for the bacterial and fungal targets. All strains listed in Table 31 generated the expected results for the high titer organisms in all replicates. Additionally, there were no false positive results generated at the concentrations tested.

# g. Assay cut-off:

The presence or absence of each target analyte is determined by the mean intensity of target capture spots relative to the Signal Detection Threshold. The forward and reverse PCR primer oligonucleotides as well as the capture and mediator oligonucleotides in the RP Flex test are designed to eliminate sequence-related cross-reactivity, thereby ensuring that the non-specific target signal intensities at capture spots are similar to the microarray background signal. By contrast, target amplicon hybridization to complementary capture and mediator probes are expected to give signals that are wellseparated from negative capture spots.

After completion of the RP Flex hybridization step, the microarray is placed into the Verigene Reader where a camera detects light scattered from the different analytespecific capture oligo spots. Each target capture is represented by a total of six replicate spots on the RP Flex microarray and the average intensity from these spots is used in the analysis. Multiple images of each array are taken at increasing exposures times and the final target group mean intensity value for an analyte is assigned at the shortest exposure at which the intensity exceeds the Signal Detection Threshold. If none of the target signal exceeds the threshold for any exposure, the mean spot intensity is evaluated at the longest exposure taken. With this imaging and analysis design, a signal detection threshold of 30,000 was established to generate a Detected call for the 15 viral target spot groups, three bacterial target spot groups, as well as two assay controls within the RP Flex (see Table 35 below).

Table 35: Target Spot Groups on the RP Flex   

<table><tr><td colspan="1" rowspan="1">Reported Target</td><td colspan="1" rowspan="1">Target Spot Group</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">Influenza A/H1</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">Influenza A/H3</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Influenza B</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">RSV A</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">RSVB</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">Adenovirus B/E</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td></tr><tr><td colspan="1" rowspan="2">hMPV</td><td colspan="1" rowspan="1">Human Metapneumovirus A</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus B</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">Parainfluenza 1</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Parainfluenza 2</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">Parainfluenza 3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">Parainfluenza 4</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Rhinovirus</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis/bronchiseptica</td><td colspan="1" rowspan="1">Bordetella</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">B. pertussis</td></tr><tr><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">B. holmesii</td></tr><tr><td colspan="1" rowspan="2">Internal Controls</td><td colspan="1" rowspan="1">Hybridization Control</td></tr><tr><td colspan="1" rowspan="1">Process Control</td></tr></table>

In order to demonstrate the appropriateness of the cut-off value of the RP Flex, a Cutoff Verification study utilizing image analysis data from the RP Flex LoD Study was conducted. Specifically the target mean intensity values from the final 20 confirmatory tests at the limit of detection for the 28 strains contrived in simulated NPS media and evaluated while fresh were used for this study. With replicates of 20 for each of the 28 samples and 20 unique target spot groups evaluated per test, a total of 11200 data points were assessed in this Cut-off Verification study. Of these, 1860 data points were expected to be positive. However, since the dataset is at the established strain LoD and some samples were detected in 19/20 replicates and these were expected to provide false negative results.

A logistic fit plot of Expected RP Flex Results by observed Target Mean Intensity is shown in Figure 1 for all of the target groups on the RP Flex panel. The signal detection threshold is represented by the vertical red line in the figure. It is evident that the positive signals are well separated from the negative target signals, and that the threshold value clearly distinguishes the “True Positives” (●) from the “True Negatives” $( \bullet )$ . As presented in Table 36 below, with this threshold there were 1844 True Positives, 9340 True Negatives, no discrepant false-positive result and 16 false negative results.

![](images/37b9fd2a6d7025a957b968951521adf35353932dbc4a2cb769006d56275d0b00.jpg)  
Figure 1: Logistic Fit of Expected Result by Target Mean Group (The Signal Detection Threshold at 30,000 is shown as the vertical red line)

Table 36: Call Assessment for Signal Detection Threshold   

<table><tr><td rowspan=1 colspan=1>Legend</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>True Positive</td><td rowspan=1 colspan=1>1844</td></tr><tr><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>9340</td></tr><tr><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>False Positive</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>False Negative</td><td rowspan=1 colspan=1>16</td></tr></table>

Thirteen (13) of the 16 False Negatives were expected given the LoD confirmatory testing results of 19/20 at the sample/target LoD. It is notable that the signal intensity in these cases is similar to the signals for the expected negative target spots (lower left corner of Figure 1). This suggests that the target organism at this low titer did not have significant amplification during the RT-PCR process.

Two (2) of the remaining three false negative results were for Adenovirus C capture targets with Adenovirus 4 (E) sample in which the Adenovirus B/E capture was detected. Both of these cases had relatively high signals (18643 and 27229), just below the threshold. These two results are readily observed in Figure 1. Note that for an

Adenovirus Detected call, detection at either target spot group is sufficient, thus a false negative call is not generated in these cases.

The final false negative result occurred with the Extraction Control target with Influenza B sample. Since the Influenza B target was detected, the extraction control result did not impact the expected target call.

It is apparent in Figure 1 that the provided Signal Detection Threshold of 30,000 should be appropriate to make a reliable ‘Detected’ or ‘Not Detected’ decision. The threshold is able to clearly identify the targets that have been amplified in the assay and is well separated from the background or negative target signals.

# h. Interfering Substances:

An analytical study was performed to assess the potential inhibitory effects of exogenous and endogenous substances that may be commonly found in nasopharyngeal specimens (NPS). Three RP Flex target organisms − one DNA virus (Adenovirus), one RNA virus (Influenza A), and one gram-negative bacterium (Bordetella pertussis) − were individually challenged with various exogenous substances at medically-relevant “worst-case” concentrations. The selected organisms for testing in this study and the concentrations tested are listed in Table 37 below.

Table 37: Representative Organisms/Source and Concentrations for the Interference Study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source/Strain</td><td rowspan=1 colspan=1>Concentration(3x LoD)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3 (B)</td><td rowspan=1 colspan=1>Zeptometrix #0810062CF</td><td rowspan=1 colspan=1>3.33E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A (H1N1)</td><td rowspan=1 colspan=1>TriCore Influenza A/Brisbane/59/2007</td><td rowspan=1 colspan=1>9.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>2.43E+03 CFU/mL</td></tr></table>

The full panel of exogenous and endogenous interferents tested in this study is shown in Table 38 below. In addition, the interference study evaluated materials that are used to collect (swabs) or store (viral transport medium) NPS specimens, as listed in Table 39.

Table 38: Potentially Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Interferents</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">Amount</td></tr><tr><td colspan="1" rowspan="1">Wal-Four® Nasal Spray</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Anefrin Nasal Spray</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">NaCl</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Similasan Sinus ReliefTM</td><td colspan="1" rowspan="1">Luffa operculata</td><td colspan="1" rowspan="1">1.0% v/v</td></tr><tr><td colspan="1" rowspan="1">Anbesol® (Anesthetic)</td><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">0.5% v/v of sample</td></tr><tr><td colspan="1" rowspan="1">Qvar® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Beclomethasone dipropionate</td><td colspan="1" rowspan="1">1.60E+01 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Dexacort® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">5.00E+01 μg/mL</td></tr><tr><td colspan="1" rowspan="1">AeroBid® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">5.80E+01 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Triamcinolone (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Triamcinolone acetonide</td><td colspan="1" rowspan="1">5.50E+00 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Pulmicort® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">2.50E+01 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Elocon® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Mometasone furoate</td><td colspan="1" rowspan="1">2.50E+00 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Flonase® (Nasal corticosteroid)</td><td colspan="1" rowspan="1">Fluticasone propinoate</td><td colspan="1" rowspan="1">5.00E+00 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Veramyst®</td><td colspan="1" rowspan="1">Fluticasone furoate</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Sulfur (Boiron®)</td><td colspan="1" rowspan="1">Sulfur</td><td colspan="1" rowspan="1">4.50E+00 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">5.00E-01 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">5.00E+00 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">1.50E-02 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, bovine submaxillary Type I-S</td><td colspan="1" rowspan="1">1.00E-01 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, porcine stomach Type II</td><td colspan="1" rowspan="1">1.00E-01 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, porcine stomach Type III</td><td colspan="1" rowspan="1">1.00E-01 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tamiflu® (Anti-viral)</td><td colspan="1" rowspan="1">Oseltamivir Phosphate</td><td colspan="1" rowspan="1">3.30E+01 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Galphimia Glauca (Boiron®)</td><td colspan="1" rowspan="1">Galphimia Glauca</td><td colspan="1" rowspan="1">1.15E+02 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Histaminum Hydrochloricum (Boiron®)</td><td colspan="1" rowspan="1">Histaminum Hydrochloricum</td><td colspan="1" rowspan="1">1.15E+02 μg/mL</td></tr><tr><td colspan="1" rowspan="1">RelenzaTM (Anti-viral)</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">1.00E+01 mg/mL</td></tr><tr><td colspan="1" rowspan="1">FluMist® Influenza Vaccine Live,Intranasal</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Single dose</td></tr><tr><td colspan="1" rowspan="1">Human Blood</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="3">Human DNA</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">2.00E-01 ng/μL</td></tr><tr><td colspan="1" rowspan="1">2.00E+01 ng/μL</td></tr><tr><td colspan="1" rowspan="1">2.00E+01 ng/μL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+05 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Ethyl Alcohol, Absolute 200 Proof</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Acetonitrile</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10% v/v</td></tr></table>

Table 39: Potentially Interfering Sample Collection and Storage Materials   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Material</td><td rowspan=1 colspan=1>Amount</td></tr><tr><td rowspan=7 colspan=1>Transport Media</td><td rowspan=1 colspan=1>Copan Universal Viral Transport Media (UTM)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M4</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M4-RT®</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M5®</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M6TM</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Copan Liquid Amies (eSwab)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>BD Regan Lowe Semi-Solid Transport Media</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=2 colspan=1>Swabs</td><td rowspan=1 colspan=1>Copan CLASSIQSwabsTM Rayon tipped swab -Aluminum applicator (Cat. No. 170KS01)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Copan FLOQSwabs Nylon Flocked swab (CatNo. 519CS01)</td><td rowspan=1 colspan=1>1 swab</td></tr></table>

On the day of RP Flex testing, the three representative organisms were prepared to a concentration of 3x LoD in the interfering substance samples. A control containing no interfering substance was also prepared and tested for each target organism. Aliquots of each sample were tested in triplicate with the RP Flex. Additionally, all potential interferents were tested in triplicate without RP Flex target organisms as negative controls. No false positive was observed for the negative controls, except for the MedImmune FluMist ® Influenza Vaccine Live, Intranasal Spray (2011-2012 Formula).

The FluMist ® Influenza Vaccine Live was obtained from ATCC and further tested in a serial dilution range from $0 . 2 \% - 0 . 0 0 0 0 2 \%$ in Simulated NPS. Positive results were obtained in at least one replicate for Influenza A, Influenza A/H1, and Influenza A/H3 down to $0 . 0 2 \%$ dilution. Similarly, Influenza B was detected down to a dilution of $0 . 0 0 2 \%$ .

The observed reactivity to the FluMist ® Vaccine is indicated as a limitation in the product package insert which states: “Recent vaccination with the intranasal influenza vaccine may produce false positive results for Influenza A and/or Influenza $B ^ { \prime \prime }$ .

None of the substances or microorganisms at the concentrations tested in this study showed any inhibitory effect on the detection of target respiratory pathogens using the RP Flex.

# i. Competitive Interference Study:

In order to assess competitive inhibition in the RP Flex test, binary combinations of 14 organisms, representing the 16 RP Flex targets, were evaluated as presented in Table 40 below. The samples were prepared in Simulated NPS, with one panel member organism at a high titer and a second organism at a low titer representing all possible dual infections. High-positive titer strains were tested at $1 . 0 0 \mathrm { E } { + 0 5 } \ : \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses and $1 . 0 0 \mathrm { E } { + } 0 6 $ CFU/mL for bacteria, while low-positive titer strains were tested at $3 \mathbf { x }$ LoD. A total of 182 unique combinations of high positive/low positive titer samples were prepared, and each was tested in triplicate in RP Flex.

Table 40: Viral and Bacterial Strains Used to Assess Competitive Interference   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">High TiterSource/Strain</td><td colspan="1" rowspan="1">High TiterConcentration</td><td colspan="1" rowspan="1">Low Titer Source/Strain</td><td colspan="1" rowspan="1">Low TiterConcentration(3xLoD)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1</td><td colspan="1" rowspan="1">ZeptometrixA/Brisbane/59/2007</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">ZeptometrixA/Brisbane/59/2007</td><td colspan="1" rowspan="1">9.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1">ZeptometrixA/Wisconsin/67/05a</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Zeptometrix0810252CF</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">ZeptometrixB/Florida/02/2006</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">ZeptometrixB/Florida/02/2006</td><td colspan="1" rowspan="1">9.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">Zeptometrix0810160CFa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore 80410</td><td colspan="1" rowspan="1">9.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">Zeptometrix0810040ACFa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore VR-540</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">Zeptometrix0810040CFa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore VR-1580</td><td colspan="1" rowspan="1">1.10E+00 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">Zeptometrix0810014CFa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore VR-94</td><td colspan="1" rowspan="1">2.70E+02 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Zeptometrix0810015CFa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore VR-92</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">Zeptometrix0810016CF</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Zeptometrix0810016CF</td><td colspan="1" rowspan="1">1.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">TriCore VR-1378</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TriCore VR-1378</td><td colspan="1" rowspan="1">8.10E+02 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Zeptometrix0810062CF</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Zeptometrix0810062CF</td><td colspan="1" rowspan="1">3.33E+00 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Zeptometrix0810012CFNa</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">TriCore VR-340</td><td colspan="1" rowspan="1">3.00E+01 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">ATCC 51541</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">ATCC 51541</td><td colspan="1" rowspan="1">2.43E+03 CFU/mL</td></tr></table>

The results of the Competitive Interference testing are summarized in Table 41 below. The expected targets of the organisms present at a high titer were detected in all of the 182 unique combinations $( 1 4 \times 1 3 )$ tested. For the low titer organisms, there were 10 cases in which a false negative result was obtained in one of the three replicates.

To further evaluate the cases in which the low titer organism was detected only in two of the three replicates, an additional six RP Flex tests per combination were performed. In all of the additional tests both the low and high titer analytes were accurately detected in all six replicate tests. Thus, the initial false negative results are likely reflective of the low titer (3x LoD) and not of systemic competitive inhibition in the RP Flex.

Table 41: Summary of Competitive Interference Testing Results   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=19>Low Positive Titer Strains (3x LoD)</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=3 colspan=3>IH V zunI</td><td rowspan=3 colspan=3>T V u3</td><td rowspan=3 colspan=2>I tzu</td><td rowspan=3 colspan=1>ANPY</td><td rowspan=3 colspan=1>VSSA</td><td rowspan=3 colspan=1>RRAA</td><td rowspan=3 colspan=1>rzu∃d</td><td rowspan=3 colspan=1>Z runnu∃d</td><td rowspan=3 colspan=1>Pzgu</td><td rowspan=3 colspan=1>zgud</td><td rowspan=3 colspan=1>Ar</td><td rowspan=3 colspan=1>RhkouuS</td><td rowspan=3 colspan=1>Ps</td><td rowspan=3 colspan=1>psumoeh</td></tr><tr><td rowspan=1 colspan=2>BinaryCombinations</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=29 colspan=1>Hh  d b</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>Influenza A H1</td><td rowspan=4 colspan=3>-</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=3>3/3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=2>b</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2>8/9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>8/9</td></tr><tr><td rowspan=4 colspan=1>Influenza A H3</td><td rowspan=4 colspan=1></td><td></td><td></td><td rowspan=4 colspan=2></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=3>-</td><td rowspan=1 colspan=2></td><td rowspan=4 colspan=2>3/3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td rowspan=2 colspan=2>b</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>8/9</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>8/9</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>Influenza B</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=3>3/3</td><td rowspan=4 colspan=3>3/3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=2>-</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>8/9</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td></tr><tr><td></td></tr><tr><td rowspan=2 colspan=1>hMPV</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>8/9</td></tr><tr><td rowspan=2 colspan=1>RSV A</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>RSV B</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza 1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza 2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza 3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza 4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=3>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

<table><tr><td rowspan=11 colspan=1></td><td rowspan=1 colspan=2>Adenovirus</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=2>Rhinovirus</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>8/9</td></tr><tr><td rowspan=4 colspan=2>B. pertussis</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td rowspan=4 colspan=1>3/3</td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=1>-</td><td></td><td></td></tr><tr><td></td><td rowspan=2 colspan=1>3/3</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=2>B. holmesii</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=2>3/3</td><td></td></tr><tr><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=2>Total DetectionRate of Low TiterStrain</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>44/45</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>44/45</td><td rowspan=1 colspan=1>49/51</td><td rowspan=1 colspan=1>39/3939/39</td><td rowspan=1 colspan=1>39/3939/39</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>49/51</td><td rowspan=1 colspan=1>44/45 39/39</td><td rowspan=1 colspan=2>44/45 39/39</td><td rowspan=1 colspan=1>54/57</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>100% 100%</td><td rowspan=1 colspan=1>100% 100%</td><td rowspan=1 colspan=1>98%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98%</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>100%100%1</td><td rowspan=1 colspan=1>100%100%1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>98%</td><td rowspan=1 colspan=2>100%</td><td rowspan=1 colspan=1>95%</td></tr></table>

# j. Carry-Over Contamination:

The RP Flex is designed such that all of the sample contact components for the test are single-use consumables that minimize the risk for carryover or crosscontamination. An analytical study was performed to assess potential carryover or cross-contamination in the RP Flex by alternately testing high positive and negative samples on the Verigene System. The high positive samples consisted of three respiratory pathogens, representing the three major types of panel analytes: a DNA virus (Adenovirus), an RNA virus (Influenza A), and a gram negative bacterium (Bordetella pertussis). Each strain was prepared in Simulated NPS at a high positive titer, $1 . 0 0 \mathrm { E } { + 0 5 } \ : \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viral targets, and $1 . 0 0 \mathrm { E } { + } 0 6 $ CFU/mL for bacterial targets. Specific strain information and the expected RP Flex results are shown in Table 42 below.

Table 42: Organisms Tested in the Carryover Contamination Study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source / ID</td><td rowspan=1 colspan=1>Expected Results</td><td rowspan=1 colspan=1>Titer</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3 (B)</td><td rowspan=1 colspan=1>Zeptometrix#0810062CF</td><td rowspan=1 colspan=1>Adenovirus Detected</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA/Wisconsin/67/05(H3N2)</td><td rowspan=1 colspan=1>Zeptometrix#0810252CF</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A/H3 Detected</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>Bordetellaparapertussis/bronchiseptica. &amp;Bordetella pertussis Detected</td><td rowspan=1 colspan=1>1.00E+06 CFU/mL</td></tr></table>

The study was performed using six Verigene $S P$ processors, with each processor $S P$ tested five times with alternative high positive and negative specimens, as summarized in Table 43 below.

Table 43: Overview of Carryover/Cross-contamination Testing Format   

<table><tr><td rowspan=2 colspan=1>Run</td><td rowspan=1 colspan=6>Verigene Processor SP</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td><td rowspan=1 colspan=1>High PositiveAdenovirus</td><td rowspan=1 colspan=1>High PositiveInfluenza A</td><td rowspan=1 colspan=1>High PositiveB. pertussis</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr></table>

All of the negative samples yielded “Not Detected” calls for all the analytes, with the exception of three tests that returned an initial “No Call” result (Influenza A Run 4, $S P 5$ ; Influenza A Run 8, SP5; Bordetella pertussis Run 10, $S P 6$ . For these tests, analysis of the capture spot intensities showed no elevated signal from the previously run high positive samples and the “No Call” was not attributable to a carryover event. For each “No Call” result, an additional test of the high titer sample and negative sample were performed. In all cases, the high positive samples yielded the expected “Detected” results and the subsequent negative samples returned “Not Detected” results. These results confirm that there is no evidence of carryover from samples tested with the RP Flex in this study.

During the study, a single false positive result was observed with a high positive $B _ { \cdot }$ . pertussis sample. In addition to the correct calls for Bordetella parapertussis/bronchiseptica and B. pertussis, the test also detected Influenza B. Evaluation of the eluted material from this sample with PCR/Bi- directional Sequencing did not identify the presence of Influenza B. Influenza B was not one of the organisms being tested as part of this study. This suggests that the observation of the false positive result is not an indication of cross-contamination from samples or reagents across the $S P$ systems in use during this study. Although the Verigene System and the RP Flex consumables have been designed to minimize nucleic acid contamination both from genomic and amplicon-based sources, the source of the Influenza B false positive was most likely due to sample contamination from the laboratory environment.

In the prospective clinical study for the Verigene RP Flex, the RP Flex results were compared to the results of an FDA-cleared multiplexed respiratory pathogens panel performed by eight of the nine comparator testing sites for the majority of the RP Flex analytes (Adenovirus, Influenza A, Influenza B, Influenza A/H1 and A/H3 subtypes, Parainfluenza 1, 2, 3, and 4, Human Metapneumovirus, Respiratory Syncytial Virus, Rhinovirus, and Bordetella pertussis). The remaining RP Flex analytes (Bordetella holmesii and Bordetella parapertussis/bronchiseptica) required two analytically validated PCR followed by bi-directional sequencing (PCR/BDS) comparator assays which were tested using the residual nucleic acids samples extracted from the NPS specimens by the Verigene RP Flex at one dedicated PCR/Sequencing comparator testing site. Since the FDA-cleared multiplex respiratory pathogen panel comparator reports a general RSV result without distinguishing RSV A from RSV B, following a positive result for RSV on the FDA-cleared comparator, residual nucleic acid samples extracted from the NPS specimens by the Verigene RP Flex were sent for RSV subtyping by an analytically validated PCR/BDS assay. In addition, due to the fact that the FDA-cleared multiplexed respiratory pathogen panel comparator reports a Rhinovirus/Enterovirus result without disguising Rhinovirus from Enterovirus, following a positive result for Rhinovirus/Enterovirus on the FDA-cleared comparator, residual nucleic acid samples extracted from the NPS specimens by the Verigene RP Flex were sent for confirmation of Rhinovirus by two analytically validated PCR/BDS assays.

Table 44: PCR/BDS Comparators Testing Scheme   

<table><tr><td rowspan=1 colspan=1>Organism/Virus</td><td rowspan=1 colspan=1>Comparator PCR/BDS Assays</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>2 PCR/BDS tests on extracted samples</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis/bronchiseptica</td><td rowspan=1 colspan=1>2 PCR/BDS tests on extracted samples</td></tr><tr><td rowspan=1 colspan=1>RSV A and B</td><td rowspan=1 colspan=1>1 PCR/BDS test on extracted samples</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>2 PCR/BDS tests on extracted samples</td></tr></table>

These analytically validated PCR/BDS assays have different primer/probe sequences from those used in the RP Flex. In some cases, the PCR/BDS amplicon overlaps with the RP Flex assay amplicon due to design limitations (availability of conserved regions). However, none of the PCR/BDS primers are designed to overlap with any of the RP Flex assay primers/probes (Table 45 below). This eliminates the possibility that the PCR/BDS amplicon is derived from the RP Flex assay amplicon.

Table 45: Targets for PCR/BDS Comparators and RP Flex   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">RP Flex Target</td><td colspan="1" rowspan="1">PCR/BDS Assay 1Target</td><td colspan="1" rowspan="1">PCR/BDS Assay 2Target</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">fumC</td><td colspan="1" rowspan="1">fumC</td><td colspan="1" rowspan="1">rpoB</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis/bronchiseptica</td><td colspan="1" rowspan="1">gidA</td><td colspan="1" rowspan="1">Toxin PromoterRegion</td><td colspan="1" rowspan="1">Toxin PromoterRegion</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">Polymerase LargeSubunit - L</td><td colspan="1" rowspan="2">Polymerase LargeSubunit - L</td><td colspan="1" rowspan="2">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">Fusion Protein (F)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">5'UTR</td><td colspan="1" rowspan="1">5'UTR</td><td colspan="1" rowspan="1">5'UTR</td></tr></table>

Validation studies included analytical sensitivity (LoD) study and analytical reactivity study. The results of these validation studies demonstrated that the analytical sensitivity and reactivity of the PCR/BDS comparators are either equivalent or better comparing to that of the RP Flex.

# l. Specimen Stability Study

An analytical study was carried out to support the RP Flex stability claim for clinical specimens preserved in viral transport media (VTM), and to demonstrate the long-term stability of frozen banked clinical samples (retrospective samples) that were used to supplement the prospective clinical study data.

A total of fourteen viral and bacterial strains in pooled negative natural clinical NPS were evaluated in this study. These strains are representative of all of the sixteen RP Flex targets and were also part of the LoD study. Each strain was prepared at Low Positive (2xLoD) and Moderate Positive (5xLoD) concentrations in pooled negative natural clinical NPS collected in VTM.

Initial testing was performed to establish the baseline time point $( { \mathfrak { t } } = 0 )$ ) for the study, and additional aliquots of the samples were stored at each of the following temperature conditions: (1) $2 0 – 2 5 ^ { \circ } \mathrm { C }$ , (2) $2 { - } 8 ^ { \circ } \mathrm { C }$ , and (3) Frozen $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ .

At the designated time points, shown in Table 46 below, one aliquot of each strain at each concentration was tested with the RP Flex in replicates of three. The $2 { - } 8 ^ { \circ } \mathrm { C }$ and 20- $2 5 ^ { \circ } \mathrm { C }$ samples were tested immediately, while the frozen aliquots were thawed at room temperature for 10-30 minutes prior to testing.

Table 46: Specimen Stability Storage Conditions and Time Points   

<table><tr><td rowspan=2 colspan=1>Test Time Point</td><td rowspan=1 colspan=7>Storage Temperature</td></tr><tr><td rowspan=1 colspan=2>20-25°C</td><td rowspan=1 colspan=4>2-8°C</td><td rowspan=1 colspan=1>&lt;-70°C</td></tr><tr><td rowspan=1 colspan=1>Baseline (0)</td><td rowspan=1 colspan=2>X</td><td rowspan=1 colspan=4>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=2>X</td><td rowspan=2 colspan=4>N/A</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>6 hours</td><td rowspan=1 colspan=2>X</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=6 colspan=2>N/A</td><td rowspan=1 colspan=4>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>72 hours</td><td rowspan=1 colspan=4>X</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>75 hours</td><td rowspan=1 colspan=4>X</td></tr><tr><td rowspan=1 colspan=1>15 days</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3></td><td rowspan=3 colspan=2>N/A</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>30 days</td><td></td><td></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>35 days</td><td></td><td></td><td rowspan=1 colspan=1>X</td></tr></table>

The results of this specimen stability study support the stability claim for RP Flex testing of clinical NPS specimens preserved in VTM at the following storage conditions: 4 hours at $2 0 – 2 5 ^ { \circ } \mathrm { C }$ , 72 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ , and 30 days at ${ < } . 7 0 { } ^ { \circ } \mathrm { C }$ .

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# Prospective Clinical Studies

The clinical performance of the RP Flex was evaluated during prospective studies at a total of five geographically separated U.S. testing sites. The study sites were located in the Midwest (Milwaukee, WI and Indianapolis, IN), the Mid-Atlantic (Baltimore, MD), the Northeast (Liverpool, NY), and the Southwest (Albuquerque, NM). Each study location was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

# Fresh Prospective Specimens (All-comers)

Fresh (never frozen) nasopharyngeal swab (NPS) specimens in viral transport media (VTM) that were prospectively collected at the respective clinical sites were tested at the five U.S. testing sites (Site 1, 2, 3, 4, and 5) during July 2014 thru November 2014. Additional fresh (never frozen) NP swab samples in VTM that were prospectively collected from two additional specimen collection sites (one in Mexico and the other one in the Midwest part of the U.S.) were also prospectively tested at two of the five testing sites (Site 2 and 5) during July 2014 thru November 2014. Prospective clinical studies testing fresh NPS specimens in viral transport media (VTM) were also conducted at two of the five testing sites (Site 1 and 4) during February 2015 thru March 2015.

Most of the fresh prospective specimens tested in these studies were de-identified and enrolled from residual specimens collected from individuals presenting with signs and symptoms of respiratory infection at the testing sites. These clinical specimens were deidentified during enrollment by the testing sites and assigned a unique study identification number. In some cases, a third-party collection site labeled residual fresh prospective clinical specimens with an external unique de-identification number and shipped at $4 \mathrm { { } ^ { \circ } C }$ overnight to testing sites, who then assigned a unique study identification number for enrollment.

Fresh specimens were residual, de-identified and consisting of $\ge 1 . 5 \mathrm { m L }$ NPS specimen collected in VTM. Specimens were aliquoted in accordance with the study protocol. One aliquot was used for Verigene RP Flex testing at the test sites. If RP Flex testing was not performed immediately, the sample was stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 48 hours before testing. Another aliquot was immediately stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ and subsequently shipped overnight on dry ice, to a third-party reference laboratory for comparator testing using an FDA-cleared multiplexed respiratory pathogens panel.

Frozen Archived Prospective Specimens (All-comers)

In addition, archived (previously frozen) NPS specimens in VTM that were prospectively collected and archived from September 2013 thru March 2014 at one U.S. clinical reference laboratory in the Midwest (Indianapolis, IN) were randomly distributed to, enrolled and tested at the same five U.S. testing sites (Site 1, 2, 3, 4, and 5) that tested the fresh prospective samples, from July 2014 thru December 2014.

Frozen specimens (all-comers) were collected at the collection site using an external unique de-identification number. Samples for this study included all positive and negative specimens submitted to the laboratory for routine respiratory pathogen testing.

Frozen specimens were residual, de-identified and consisted o $\mathrm { \Delta } \mathrm { H } \geq 1 . 0 \mathrm { m L }$ NPS specimen collected in VTM. Specimens were stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ and shipped frozen to the sponsor. Prior to sending specimens to the testing sites, the sponsor thawed, de-identified, and aliquoted the specimens for study use by assigning another unique study identification number. Specimens were randomized and shipped to the testing sites in accordance with the site’s availability to receive and test specimens. One aliquot was immediately stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ and subsequently shipped overnight on dry ice, to a third-party reference laboratory for comparator testing using an FDA-cleared multiplexed respiratory pathogens panel. Another aliquot was immediately stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ and shipped overnight on dry ice, to the testing sites for Verigene RP Flex testing.

Verigene RP Flex Testing of All Prospective Specimens

Fresh prospective specimens were tested within 48 hours of collection while frozen prospective samples were tested within 48 hours of sample thaw at the test sites. If a specimen test yielded a result of a) No Call or b) Pre-Analysis Error (PAE), the specimen was re-tested using $2 0 0 \mu \mathrm { L }$ from the remaining sample volume in accordance with study protocol. If a specimen could not be re-tested within 48 hours of collection, it was frozen until the repeat test could be performed. At the conclusion of each Verigene RP Flex test, the residual nucleic acids were collected from the Extraction Tray into a microcentrifuge tube and stored frozen at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ . The tube was shipped overnight on dry ice to the sponsor and was sent to a third party laboratory for PCR/BDS testing. In the event that a sample residual nucleic acids in the Extraction Tray was less than $5 0 ~ \mu \mathrm { L }$ (insufficient volume for required PCR/BDS testing), repeat RP Flex testing was performed to generate additional volume of nucleic acids. However, the first RP Flex test was used as the test of record for all performance calculations.

For all fresh and frozen prospective specimens, comparator testing was performed at one of the eight comparator testing sites that are different from the five RP Flex test sites. An FDA-cleared multiplexed respiratory pathogens panel was used as the primary comparator method for the majority of the targets on the RP Flex panel (i.e., Adenovirus, Influenza A, Influenza B, Influenza A/H1 and A/H3 subtypes, Parainfluenza 1, 2, 3, and 4, Human Metapneumovirus, Respiratory Syncytial Virus, Rhinovirus, and Bordetella pertussis). As the FDA-cleared multiplexed respiratory pathogens panel does not distinguish RSV types nor Rhinovirus and Enterovirus, the comparator method for RSV subtypes consisted of the FDA-cleared multiplexed respiratory pathogens panel followed by an analytically validated polymerase chain reaction (PCR) and bi-directional sequencing (BDS) assay, and the comparator method for Rhinovirus consisted of the FDA-cleared multiplexed respiratory pathogens panel followed by a composite of two analytically validated polymerase chain reaction (PCR) and bi-directional sequencing (BDS) assays. For the targets on the RP Flex panel but not detected by the FDA-cleared multiplexed respiratory pathogens panel (i.e., Bordetella parapertussis/bronchiseptica and Bordetella holmesii), a composite of two analytically validated PCR/BDS assays was used as the comparator method. The PCR/BDS testing for the prospective clinical studies was performed at an external third-party site.

An overview of the composite algorithm to determine comparator results is listed in Table 47 below.

Table 47: RP Flex Prospective Clinical Studies Comparator Methods   

<table><tr><td rowspan=1 colspan=1>RP Flex Target</td><td rowspan=1 colspan=1>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>Influenza A and B</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype: A/H1 and A/H3</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1, 2, 3, and 4</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>RSV A and Ba</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel andone PCR/BDS assay to distinguishRSV subtype (A and/or B)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirusb</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel andcomposite of two PCR/BDS assays to confirm Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>An FDA-cleared multiplexed respiratory pathogens panel</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Composite of two analytically validated PCR/BDS assays</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis/bronchiseptica</td><td rowspan=1 colspan=1>Composite of two analytically validated PCR/BDS assays</td></tr></table>

a The FDA-cleared multiplexed respiratory pathogens panel does not differentiate RSV A from RSV B b The FDA-cleared multiplexed respiratory pathogens panel does not differentiate Rhinovirus from Enterovirus c This target is not present on the FDA-cleared multiplexed respiratory pathogens panel

Following a detected result for RSV on the FDA-cleared multiplexed respiratory pathogens panel, samples (i.e., the residual nucleic acids collected from the Verigene Processor $S P$ Extraction Tray) were sent for PCR/BDS for RSV subtype identification. Samples that were negative by PCR/BDS were considered negative. The composite algorithm used to determine the comparator result for RSV is summarized in Table 48 below.

Table 48: Composite Comparator Method for RSV   

<table><tr><td rowspan=1 colspan=1>FDA-Cleared RespiratoryPathogens Panel RSV Result</td><td rowspan=1 colspan=1>PCR/BDS Result</td><td rowspan=1 colspan=1>Final Composite Comparator Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>RSV A Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>RSV B Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Nega</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Nega</td></tr></table>

Following a positive result for Rhinovirus/Enterovirus on the FDA-cleared multiplexed respiratory pathogens panel, samples (i.e., the residual nucleic acids collected from the Verigene Processor $S P$ Extraction Tray) were sent for confirmation of Rhinovirus by a composite of two PCR/BDS assays. If the first PCR/BDS assay failed to provide a valid positive Rhinovirus or Enterovirus result, a second PCR/BDS assay targeting a different region of the viral genome was performed. The result of the second PCR/BDS assay was used as the final result. Samples were considered positive for Rhinovirus only upon a positive detection by a PCR/BDS assay. Samples were considered negative for Rhinovirus with 1) a negative Rhinovirus/Enterovirus result on the FDA-cleared multiplexed respiratory pathogens panel; 2) negative Rhinovirus results from both PCR/BDS assays; 3) a positive Enterovirus result from one PCR/BDS assay. The composite algorithm used to determine the comparator result for Rhinovirus is summarized in Table 49 below.

Table 49: Composite Comparator Method for Rhinovirus   

<table><tr><td rowspan=1 colspan=1>FDA-Cleared RespiratoryPathogens PanelRhinovirus/EnterovirusResult</td><td rowspan=1 colspan=1>PCR/BDS 1 Result</td><td rowspan=1 colspan=1>PCR/BDS 2 Result</td><td rowspan=1 colspan=1>Final Composite ComparatorResult</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Rhinovirus Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Rhinovirus Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Rhinovirus Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Rhinovirus Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Rhinovirus Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Rhinovirus Negative</td></tr></table>

All samples were tested by two PCR/BDS assays for Bordetella parapertussis/bronchiseptica and Bordetella holmesii. Samples that were positive or negative by a first PCR/BDS assay were tested with a second PCR/BDS assay. With this approach, samples were considered positive if an analyte was detected by at least one of the PCR/BDS assays. Samples were considered negative when both the PCR/BDS assays were negative. The composite algorithm used to determine the comparator result for Bordetella parapertussis/bronchiseptica and Bordetella holmesii is summarized in Table 50 below.

Table 50: Composite Comparator Method for Bordetella parapertussis/bronchiseptica and Bordetella holmesii   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>PCR/BDS 1 Result</td><td rowspan=1 colspan=1>PCR /BDS 2 Result</td><td rowspan=1 colspan=1>Final CompositeComparator Result</td></tr><tr><td rowspan=4 colspan=1>Bordetellaparapertussis/bronchiseptica</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative or N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative or N/A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr></table>

# Quality Control Testing

Quality Control testing was performed on each day of RP Flex testing, using a single-use aliquot of external positive and negative control samples provided by the sponsor. Control samples were shipped on dry ice to the study testing sites where they were stored frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ until use. Controls were tested per the RP Flex IUO Package Insert instructions. If either control yielded a “PAE” or “No Call” test result, a new aliquot was tested per the study protocol.

# Discrepant Result Investigation Testing

Samples for which false positive and/or false negative results (i.e., discrepant result) were obtained when comparing the Verigene RP Flex result to the comparator method result were further tested. The discrepant investigation testing was performed using analytically validated PCR assays followed by BDS to confirm the identity of the specific target in the discordant sample. The PCR/BDS methods were developed and validated for the identification and confirmation of the panel analytes on RP Flex with PCR primers that are different from the RP Flex primers.

# Patient Demographics

A total of 2412 prospective specimens (fresh and frozen) were included in the prospective clinical studies for testing with the RP Flex. The following table (Table 51) provides a summary of demographic information for the 2412 prospectively collected specimens (fresh and frozen) that were enrolled in the prospective studies:

Table 51: Demographic Summary for RP Flex Prospective Clinical Studies   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Number of Specimens(Prospective Fresh)</td><td rowspan=1 colspan=2>Number of Specimens(Prospective Frozen)</td><td rowspan=1 colspan=2>Number of Specimens (ProspectiveCombined)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>14.0%</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>13.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>16.3%</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>28.7%</td><td rowspan=1 colspan=1>558</td><td rowspan=1 colspan=1>23.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>6.7%</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;12-21</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>5.8%</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>426</td><td rowspan=1 colspan=1>39.4%</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>20.7%</td><td rowspan=1 colspan=1>701</td><td rowspan=1 colspan=1>29.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>15.1%</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>11.7%</td><td rowspan=1 colspan=1>318</td><td rowspan=1 colspan=1>13.2%</td></tr><tr><td rowspan=1 colspan=1>Not Provided</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>14.1%</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>8.6%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1082</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1330</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2412</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Prospective Specimens Accountability

A total of 2412 prospective specimens (1082 fresh and 1330 frozen) were initially included in the prospective clinical studies for testing with the RP Flex.

Of the 1082 fresh prospective specimens, two specimens were excluded from performance analyses due to the lack of comparator testing results for all analytes; five specimens generated a “No Call” result initially by the RP Flex but were not retested per the product instructions, and were therefore excluded from performance analyses; six specimens generated a “No Call” result initially and a “No Call” result again upon retesting by the RP Flex, and were therefore excluded from the performance analyses. A total of 1069 fresh prospective specimens were included in the performance analyses of the RP Flex.

Of the 1330 frozen prospective specimens, five specimens were excluded from the performance analyses due to the lack of comparator testing results for all analytes; three specimens generated a “No Call” result initially by the RP Flex but were not retested per the product instructions, and were therefore excluded from performance analyses; one specimen was excluded from performance analyses due to uncertainty in determining the correct specimen ID; four specimens generated a “No Call” result initially and a “No Call” result again upon retesting by the RP Flex, and were therefore excluded from performance analyses. A total of 1317 frozen prospective specimens were included in the performance analyses of the RP Flex.

Of the total of 2386 prospective clinical specimens that were included in the performance analyses of the RP Flex, 189 specimens were excluded from the RP Flex performance analysis for Adenovirus due to the lack of Adenovirus comparator results; 264 specimens were excluded from the RP Flex performance analysis for Rhinovirus due to the lack of Rhinovirus comparator results; 193 specimens were excluded from the

RP Flex performance analysis for Influenza A due to the lack of Influenza A comparator results; 196 specimens were excluded from the RP Flex performance analysis for Influenza A/H1 due to the lack of Influenza A/H1 comparator results; 196 specimens were excluded from the RP Flex performance analysis for Influenza A/H3 due to the lack of Influenza A/H3 comparator results; 189 specimens were excluded from the RP Flex performance analysis for Influenza B due to the lack of Influenza B comparator results; 189 specimens were excluded from the RP Flex performance analysis for Parainfluenza 1 due to the lack of Parainfluenza 1 comparator results; 189 specimens were excluded from the RP Flex performance analysis for Parainfluenza 2 due to the lack of Parainfluenza 2 comparator results; 189 specimens were excluded from the RP Flex performance analysis for Parainfluenza 3 due to the lack of Parainfluenza 3 comparator results; 189 specimens were excluded from the RP Flex performance analysis for Parainfluenza 4 due to the lack of Parainfluenza 4 comparator results; 216 specimens were excluded from the RP Flex performance analysis for RSV A due to the lack of RSV A comparator results; 216 specimens were excluded from the RP Flex performance analysis for RSV B due to the lack of RSV B comparator results; 189 specimens were excluded from the RP Flex performance analysis for hMPV due to the lack of hMPV comparator results; 189 specimens were excluded from the RP Flex performance analysis for $B$ . pertussis due to the lack of $B$ . pertussis comparator results; 90 specimens were excluded from the RP Flex performance analysis for $B$ .

parapertussis/bronchiseptica due to the lack of B. parapertussis/bronchiseptica comparator results; 80 specimens were excluded from the RP Flex performance analysis for B. holmesii due to the lack of $B$ . holmesii comparator results.

# Prospective Clinical Studies Performance

The RP Flex prospective performance data in positive percent and negative percent agreements against the comparator methods (all sites combined) are presented in the following Table 52 by analyte:

Table 52: RP Flex Prospective Clinical Performance   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Specimen Type</td><td colspan="2" rowspan="1">PPA</td><td colspan="1" rowspan="1">95% CI</td><td colspan="2" rowspan="1">NPA</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">22/24a</td><td colspan="1" rowspan="1">91.7%</td><td colspan="1" rowspan="1">74.1% - 97.7%</td><td colspan="1" rowspan="1">1009/1028b</td><td colspan="1" rowspan="1">98.2%</td><td colspan="1" rowspan="1">97.1% - 98.8%</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">27/33c</td><td colspan="1" rowspan="1">81.8%</td><td colspan="1" rowspan="1">65.6% - 91.4%</td><td colspan="1" rowspan="1">1072/112d</td><td colspan="1" rowspan="1">96.4%</td><td colspan="1" rowspan="1">95.1% - 97.3%</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">49/57</td><td colspan="1" rowspan="1">86.0%</td><td colspan="1" rowspan="1">74.7% - 92.7%</td><td colspan="1" rowspan="1">2081/2140</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">96.5% - 97.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">214/249</td><td colspan="1" rowspan="1">85.9%</td><td colspan="1" rowspan="1">81.1% - 89.7%</td><td colspan="1" rowspan="1">719/751</td><td colspan="1" rowspan="1">95.7%</td><td colspan="1" rowspan="1">94.1% - 97.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">193/248g</td><td colspan="1" rowspan="1">77.8%</td><td colspan="1" rowspan="1">72.2% - 82.5%</td><td colspan="1" rowspan="1">859/874h</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">97.2% - 99.0%</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">407/497</td><td colspan="1" rowspan="1">81.9%</td><td colspan="1" rowspan="1">78.3% - 85.0%</td><td colspan="1" rowspan="1">1578/1625</td><td colspan="1" rowspan="1">97.1%</td><td colspan="1" rowspan="1">96.2% - 97.8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">75.7% - 100%</td><td colspan="1" rowspan="1">1030/1037</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.6% - 99.7%</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">88.9% - 99.6%</td><td colspan="1" rowspan="1">1091/1097k</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.8% - 99.7%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">58/59</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.0% - 99.7%</td><td colspan="1" rowspan="1">2121/2134</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">99.0% - 99.6%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A/H1</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1046/10481</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.3% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">45/46m</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7% - 99.6%</td><td colspan="1" rowspan="1">1092/1096n</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.1% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">45/46</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7% - 99.6%</td><td colspan="1" rowspan="1">2138/2144</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A/H3</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">75.7% - 100%</td><td colspan="1" rowspan="1">1032/1036°</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.0% - 99.8%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.6% - 100%</td><td colspan="1" rowspan="1">1141/1141</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">77.2% - 100%</td><td colspan="1" rowspan="1">2173/2177</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.5% -99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">49/50P</td><td colspan="1" rowspan="1">98.0%</td><td colspan="1" rowspan="1">89.5% - 99.6%</td><td colspan="1" rowspan="1">995/10029</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">98.6% - 99.7%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1144/1145r</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0%</td><td colspan="1" rowspan="1">89.5% - 99.6%</td><td colspan="1" rowspan="1">2139/2147</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.3% - 99.8%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1052/1052</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.6% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">27/30s</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">74.4% - 96.5%</td><td colspan="1" rowspan="1">1113/1115</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.3% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">27/30</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">74.4% - 96.5%</td><td colspan="1" rowspan="1">2165/2167</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74.1% - 100%</td><td colspan="1" rowspan="1">1038/1041</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99.2% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">1/2u</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">9.5% - 90.5%</td><td colspan="1" rowspan="1">1143/1143</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">66.7% - 98.6%</td><td colspan="1" rowspan="1">2181/2184</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.6% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGllow</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">10/12v</td><td colspan="1" rowspan="1">83.3%</td><td colspan="1" rowspan="1">55.2% - 95.3%</td><td colspan="1" rowspan="1">1037/1040w</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99.2% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">4/5*</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">37.5% - 96.4%</td><td colspan="1" rowspan="1">1140/1140</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">14/17</td><td colspan="1" rowspan="1">82.4%</td><td colspan="1" rowspan="1">59.0% - 93.8%</td><td colspan="1" rowspan="1">2177/2180</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.6% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGllow</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">43.8% - 100%</td><td colspan="1" rowspan="1">1049/1049</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.6% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">16/21y</td><td colspan="1" rowspan="1">76.2%</td><td colspan="1" rowspan="1">54.9% - 89.4%</td><td colspan="1" rowspan="1">1120/1124²</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.1% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">19/24</td><td colspan="1" rowspan="1">79.2%</td><td colspan="1" rowspan="1">59.3% - 90.8%</td><td colspan="1" rowspan="1">2169/2173</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.5% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial VirusA</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74.1% - 100%</td><td colspan="1" rowspan="1">1036/1038aa</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.3% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61.0% - 100%</td><td colspan="1" rowspan="1">1114/1115bb</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.6% - 100%</td><td colspan="1" rowspan="1">2150/2153</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.6% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial VirusB</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67.6% - 100%</td><td colspan="1" rowspan="1">1037/1041cc</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">96.7% - 98.6%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">165/165</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.7% - 100%</td><td colspan="1" rowspan="1">936/956dd</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">96.8% - 98.6%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">173/173</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td><td colspan="1" rowspan="1">1973/1997</td><td colspan="1" rowspan="1">98.8%</td><td colspan="1" rowspan="1">98.2% - 99.2%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72.2% - 100%</td><td colspan="1" rowspan="1">1037/1042ce</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">98.9% - 99.8%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.4% - 100%</td><td colspan="1" rowspan="1">1108/1109</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92.3% - 100%</td><td colspan="1" rowspan="1">2145/2151</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.6% - 100%</td><td colspan="1" rowspan="1">1050/1051gg</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">64.6% - 100%</td><td colspan="1" rowspan="1">1137/1138hh</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67.6% - 100%</td><td colspan="1" rowspan="1">2187/2189</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B.parapertussis/bronchiseptica</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2% - 100%</td><td colspan="1" rowspan="1">1039/1039</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.6% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1254/1255"</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">2/2i</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2% 0 100%</td><td colspan="1" rowspan="1">2290/2291</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.8% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">Fresh Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.6% - 100%</td><td colspan="1" rowspan="1">1042/1042</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.6% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Frozen Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1263/1263</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.7% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.6% - 100%</td><td colspan="1" rowspan="1">2305/2305</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.8% - 100%</td></tr></table>

a Adenovirus was detected in $\overline { { 1 / 2 } }$ false negative samples using PCR/BDS analysis. Discordant analysis was not performed on 1/2 false negative samples. b Adenovirus was not detected in 8/19 and detected in 2/19 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 9/19 false positive samples.   
c Adenovirus was not detected in $5 / 6$ false negative samples using PCR/BDS analysis. Discordant analysis was not performed on 1/6 false negative samples.   
d Adenovirus was not detected in $2 7 / 4 0$ and detected in $3 / 4 0$ false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 10/40 false positive samples. e, g Discordant analysis using PCR/BDS was not performed for the Rhinovirus false negative samples, as bi-directional sequencing is part of the reference method algorithm for this target. f Rhinovirus was not detected in 19/32 and detected in 12/32 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 1/32 false positive samples.   
Rhinovirus was not detected in 11/15 and detected in 2/15 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 2/15 false positive samples.   
i Influenza A was not detected in 1/7 and detected in 1/7 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on $5 / 7$ false positive samples.   
j Influenza A was not detected in 1/1 false negative samples using PCR/BDS analysis.   
k Influenza A was not detected in 2/6 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on $4 / 6$ false positive samples.   
l Influenza $_ \mathrm { A / H 1 }$ discordant analysis using PCR/BDS was not performed in $2 / 2$ false positive samples.   
m Influenza A/H1 was not detected in 1/1 false negative sample using PCR/BDS analysis. n Influenza $\mathrm { A } / \mathrm { H } 1$ was not detected in 2/4 and detected in 1/4 false positive samples using PCR/BDS analysis. Discordant analysis was not performed in $\%$ false positive samples.   
o Influenza $\mathrm { A } / \mathrm { H } 3$ was not detected in $3 / 4$ and detected in 1/4 false positive samples using PCR/BDS analysis.   
p Influenza B was not detected in false negative sample using PCR/BDS analysis.   
q Influenza B was not detected in $4 / 7$ and detected in $2 ^ { \cdot } / 7$ false positive samples using PCR/BDS analysis. Discordant analysis was not performed in 1/7 samples.   
r Influenza B was not detected in $1 / 1$ false positive sample using PCR/BDS analysis.   
s Parainfluenza 1 was not detected in $3 / 3$ false negative samples using PCR/BDS analysis.   
t Parainfluenza 2 was not detected in $2 / 3$ and detected in 1/3 false positive samples using PCR/BDS analysis.   
u Parainfluenza 2 was not detected in $1 / 1$ false negative sample using PCR/BDS analysis.   
Parainfluenza 3 was not detected inw Parainfluenza 3 was not detected in $2 / 2$ false negative samples using PCR/BDS analysis.  false positive samples using PCR/BDS analysis.   
$3 / 3$   
x Parainfluenza 3 was not detected in 1/1 false negative sample using PCR/BDS analysis.   
y Parainfluenza 4 was not detected in 2/5 and detected in $2 ^ { \cdot } / 5$ false negative samples using PCR/BDS analysis. Discordant analysis was not performed in 1/5 false negative samples.   
Parainfluenza 4 was not deteaa RSV A was not detected in bb RSV A was not detected incc RSV B was detected in 2/4 $1 / 2$ d in 3/4 and detected in 1/4 false positive samples using PCR/BDS analysis.  and detected in 1/2 false positive samples using PCR/BDS analysis.  false positive sample using PCR/BDS analysis.  not detected in 2/4 false positive samples using PCR/BDS analysis.   
dd RSV B was not detected in $1 6 / 2 0$ and detected in 2/20 false positive samples using PCR/BDS analysis. Discordant analysis was not performed in 2/20 false positive samples.   
ee hMPV was not detected in $3 / 5$ and detected in 2/5 false positive samples using PCR/BDS analysis.   
ff hMPV was detected in 1/1 false positive samples using bi-directional sequencing analysis. h Bordetella pertussis was not detected inepeat PCR/BDS analysis was performed. . parapertussis was identified by PCR/BD $1 / \dot { 1 }$ n e positive sample using PCR/BDS analysis. ertussis/bronchiseptica was not detected in 1/1 false positive sample.  specimens. ${ \ " } B$ $2 / 2$

# Prospective Clinical Studies Mixed Infection Analysis

The RP Flex detected a total of 91 mixed infections in the prospective clinical evaluations. Eighty-six (86) (86/91; $9 4 . 5 \%$ ) were double infections, four (4/91; $4 . 4 \% )$ were triple infections, and one (1/91; $1 . 1 \% )$ was quadruple infections. The single most common co-infection (27/91; $2 9 . 7 \% )$ was Rhinovirus with Adenovirus, followed by the second most common co-infection (22/91; $2 4 . 2 \% )$ of Rhinovirus with RSV B. Out of the 91 co-infections, 46 contained one or more analytes that had not been detected with the comparator method(s), i.e. discrepant co-infections.

Table 53: Distinct Co-infection Combinations Detected by the RP Flex in the Prospective Clinical Trial   

<table><tr><td colspan="4" rowspan="1">Distinct Co-infection CombinationsDetected by the RP Flex</td><td colspan="1" rowspan="2">ctun-sCotL</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infectionsa</td><td colspan="1" rowspan="2">Discrepant Analyte(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1">Analyte 4</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Adenovirus (16); Rhinovirus (3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Rhinovirus (4); RSV B (4)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1); RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus (3); RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Parainfluenza 4 (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Rhinovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1); Parainfluenza 3 (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Influenza A andA/H1</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1); Influenza A andA/H1 (1); Influenza B (1); RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1); RSV A (1)</td></tr><tr><td colspan="1" rowspan="1">Influenza Aand A/H1</td><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Influenza A and A/H1(1)</td></tr><tr><td colspan="1" rowspan="1">Influenza Aand A/H3</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Influenza A and A/H1(1);Parainfluenza 2</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">B. parapertussis/bronchiseptica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Influenza A andA/H1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Influenza A and</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Parainfluenza 2 (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="4" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Quadruple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by RP Flex but not detected by the comparator method(s).

Table 54: Additional Distinct Co-infection Combinations Detected by the Comparator Method(s), but not detected by the RP Flex in the Prospective Clinical Trial   

<table><tr><td colspan="4" rowspan="1">Distinct Co-infection CombinationsDetected by the Comparator Methods</td><td colspan="1" rowspan="2">coun-NA</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infectionsa</td><td colspan="1" rowspan="2">Discrepant Analyte(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1">Analyte 4</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Rhinovirus (9)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus (2); Rhinovirus (3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Rhinovirus (2); Parainfluenza 4 (2)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Parainfluenza 3 (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1); Parainfluenza 4 (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Parainfluenza 2 (1)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Influenza A andA/H1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Parainfluenza 1 (1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Parainfluenza 4 (1)</td></tr><tr><td colspan="4" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Quadruple Infections</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a This table includes only distinct co-infections that were detected by the comparator method(s) but not by RP Flex; the remaining co-infections detected by the comparator methods are already represented in Table 53 above.

# Retrospective Clinical Study

A retrospective clinical evaluation of the RP Flex was also performed to supplement the prospective evaluation data. In this study, retrospective pre-selected archived specimens were further confirmed by the same comparator methods described in the “Prospective Clinical Studies” section previously. The retrospective pre-selected archived specimens were randomly distributed to a total of six testing sites in the U.S., the same five testing sites as described in the “Prospective Clinical Studies” section previously plus one additional testing site located in the Southeast, and tested with the RP Flex from July 2014 thru March 2015.

# Frozen Archived Retrospective Specimens (Selected)

Nanosphere worked with four laboratories in the U.S., one in the Southeast (Atlanta, GA), one in the Midwest (South Bend, IN), one in the Southwest (Albuquerque, NM), and one in the Mid-Atlantic (Chapel Hill, NC), to source selected frozen archived retrospective specimens for analytes with lower prevalence. Frozen specimens were residual, de-identified and consisting of $\geq 1 . 0 \mathrm { m L }$ NPS specimens collected in VTM. Specimens were stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ and shipped frozen to the sponsor. Prior to sending specimens to the study testing sites, the sponsor thawed, split, and de-identified the specimens for study use by assigning a unique study identification number. Specimens were randomized and shipped to the testing sites in accordance with the site’s availability to receive and test specimens. One aliquot was immediately stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ and subsequently shipped overnight on dry ice, to a third-party reference laboratory for comparator testing using an FDA-cleared multiplexed respiratory pathogens panel. Another aliquot was immediately stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ and shipped overnight on dry ice, to the testing sites for Verigene RP Flex testing. Upon receipt, study testing sites stored the samples at $\le - 7 0 ^ { \circ } \mathrm { C }$ until ready to test.

# Verigene RP Flex Testing of Retrospective Specimens

Frozen selected retrospective samples were tested within 48 hours of sample thaw at the test sites. If a specimen test yielded a result of a) No Call or b) Pre-Analysis Error

(PAE), the specimen was re-tested using $2 0 0 \mu \mathrm { L }$ from the remaining sample volume in accordance with study protocol. If a specimen could not be re-tested within 48 hours of collection, it was frozen until the repeat test could be performed. At the conclusion of each Verigene RP Flex test, the residual nucleic acids were collected from the Extraction Tray into a microcentrifuge tube and stored frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ . The tube was shipped overnight on dry ice to the sponsor and was sent to a third party laboratory for PCR/BDS testing. In the event that a sample residual nucleic acids in the Extraction Tray was less than $5 0 ~ \mu \mathrm { L }$ (insufficient volume for required PCR/BDS testing), repeat RP Flex testing was performed to generate additional volume of nucleic acids. However, the first RP Flex test was used as the test of record for all performance calculations.

Frozen Archived Retrospective Specimens (Selected) Comparator Methods Testing

The “Retrospective Selected” specimens consisted of positive specimens (as identified by the acquisition site) collected and frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ . These specimens were tested using the same comparator methods described in the “Prospective Clinical Studies” section previously to confirm the results that were provided to Nanosphere by the original acquisition sites.

The comparator method(s) results (an FDA-cleared multiplexed respiratory pathogens panel or PCR/BDS) were compared to the results provided to Nanosphere by the original acquisition sites. If the expected results agreed, the specimen was enrolled into the study for Verigene RP Flex testing. If the expected results did not agree, the specimen was not enrolled into this study. In the event that the specimen was positive by the FDA-cleared multiplexed respiratory pathogens panel for the expected result and additional target(s) were detected, the specimen was enrolled as positive for the additional targets as well.

Table 55 shows the number of specimens acquired for each target and the percent for which the comparator method(s) agreed with the expected results. Fifty three (53) enrolled specimens were positive for the expected target and at least one additional target.

Table 55: Comparator Method(s) Testing Results of the Initially Acquired Selected Retrospective Specimens   

<table><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">Number of SpecimensAcquired</td><td colspan="1" rowspan="1">Number of Specimenswith Results Confirmedby ComparatorMethod(s)</td><td colspan="1" rowspan="1">Percent of Specimenswith Results Confirmedby ComparatorMethod(s)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">84.4%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">92.0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A*</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">97.6%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">87.7%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">62.5%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">97.0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">80.3%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RSV*</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">98.2%</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">55.6%</td></tr><tr><td colspan="1" rowspan="1">B.parapertussis/bronchiseptica</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">89.8%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">612</td><td colspan="1" rowspan="1">526</td><td colspan="1" rowspan="1">85.9%</td></tr></table>

\* These specimens were DFA positive and thus not subtyped before testing by the reference method(s)

# Quality Control Testing

Quality Control testing was performed on each day of RP Flex testing, using a single-use aliquot of external positive and negative control samples provided by the sponsor. Control samples were shipped on dry ice to the study testing sites where they were stored frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ until use. Controls were tested per the RP Flex IUO Package Insert instructions. If either control yielded a “PAE” or “No Call” test result, a new aliquot was tested per the study protocol.

# Patient Demographics

A total of 526 selected retrospective specimens with confirmed comparator method(s) testing results were included in this retrospective clinical study. The following table (Table 56) provides a summary of demographic information for the 526 selected retrospective that were enrolled in the retrospective study:

Table 56: Demographic Summary for RP Flex Retrospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>cYanmaGlb</td><td rowspan=1 colspan=2>Retrospective Selected Specimens</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>No. of Specimens</td><td rowspan=1 colspan=1>Percentage</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>18.3%</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;5-12</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>4.4%</td></tr><tr><td rowspan=1 colspan=1>&gt;12-18</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;19-64</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2.3%</td></tr><tr><td rowspan=1 colspan=1>≥65</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>Not Provided</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>68.4%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>526</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Retrospective Selected Specimens Accountability

A total of 526 retrospective selected specimens with confirmed comparator method(s) testing results were initially included in the retrospective clinical study for testing with the RP Flex.

Of the 526 retrospective selected specimens, two specimens generated a “No Call” result initially by the RP Flex but were not retested per the product instructions, and were

therefore excluded from performance analyses; four specimens generated a “No Call” result initially and a “No Call” result again upon retesting by the RP Flex, and were therefore excluded from performance analyses. A total of 520 retrospective selected specimens were included in the performance analyses of the RP Flex in this study.

Of the total of 520 retrospective clinical specimens that were included in performance analyses of the RP Flex, four specimens were excluded from the RP Flex performance analysis for Adenovirus due to the lack of Adenovirus comparator results; 11 specimens were excluded from the RP Flex performance analysis for Rhinovirus due to the lack of Rhinovirus comparator results; seven specimens were excluded from the RP Flex performance analysis for Influenza A due to the lack of Influenza A comparator results; eight specimens were excluded from the RP Flex performance analysis for Influenza A/H1 due to the lack of Influenza A/H1 comparator results; eight specimens were excluded from the RP Flex performance analysis for Influenza A/H3 due to the lack of Influenza A/H3 comparator results; four specimens were excluded from the RP Flex performance analysis for Influenza B due to the lack of Influenza B comparator results; four specimens were excluded from the RP Flex performance analysis for Parainfluenza 1 due to the lack of Parainfluenza 1 comparator results; four specimens were excluded from the RP Flex performance analysis for Parainfluenza 2 due to the lack of Parainfluenza 2 comparator results; four specimens were excluded from the RP Flex performance analysis for Parainfluenza 3 due to the lack of Parainfluenza 3 comparator results; four specimens were excluded from the RP Flex performance analysis for Parainfluenza 4 due to the lack of Parainfluenza 4 comparator results; 22 specimens were excluded from the RP Flex performance analysis for RSV A due to the lack of RSV A comparator results; 22 specimens were excluded from the RP Flex performance analysis for RSV B due to the lack of RSV B comparator results; four specimens were excluded from the RP Flex performance analysis for hMPV due to the lack of hMPV comparator results; four specimens were excluded from the RP Flex performance analysis for $B$ . pertussis due to the lack of $B$ . pertussis comparator results; 29 specimens were excluded from the RP Flex performance analysis for $B$ .

parapertussis/bronchiseptica due to the lack of $B$ . parapertussis/bronchiseptica comparator results, and 28 specimens were positive for $B .$ . pertussis by RP Flex and therefore were excluded from the RP Flex performance analysis for $B$ .

parapertussis/bronchiseptica per RP Flex product instructions; 30 specimens were excluded from the RP Flex performance analysis for $B$ . holmesii due to the lack of $B$ holmesii comparator results.

# Retrospective Clinical Study Performance

The RP Flex retrospective performance data in positive percent and negative percent agreements against the comparator methods (all sites combined) are presented in the following Table 57 by analyte:

Table 57: RP Flex Retrospective Clinical Performance   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38/39a</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>86.8% - 99.5%</td><td rowspan=1 colspan=1>469/477b</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>96.7% - 99.1%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>28/35c</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>64.1% - 90.0%</td><td rowspan=1 colspan=1>466/474d</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>96.7% - 99.1%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>122/123</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>95.5% - 99.9%</td><td rowspan=1 colspan=1>387/390</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>97.8% - 99.7%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>40/418</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>87.4% - 99.6%</td><td rowspan=1 colspan=1>469/471h</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.5% - 99.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>82/82</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.5% - 100%</td><td rowspan=1 colspan=1>428/430&#x27;</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.3% - 99.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.1% - 100%</td><td rowspan=1 colspan=1>488/490&#x27;</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.5% - 99.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>466/466</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.9% - 100%</td><td rowspan=1 colspan=1>487/488k</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.8% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89.0% - 100%</td><td rowspan=1 colspan=1>485/485</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.4% - 100%</td><td rowspan=1 colspan=1>473/475</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.5% - 99.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>55/58m</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>85.9% - 98.2%</td><td rowspan=1 colspan=1>437/440</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.0% - 99.8%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85.7% - 100%</td><td rowspan=1 colspan=1>468/475</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.0% - 99.3%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>25/27P</td><td rowspan=1 colspan=1>92.6%</td><td rowspan=1 colspan=1>76.6% - 97.9%</td><td rowspan=1 colspan=1>488/4899</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.8% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>28/29</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>82.8% - 99.4%</td><td rowspan=1 colspan=1>487/487</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B.parapertussis/bronchiseptica</td><td rowspan=1 colspan=1>5/7$</td><td rowspan=1 colspan=1>71.4%</td><td rowspan=1 colspan=1>35.9% -91.8%</td><td rowspan=1 colspan=1>455/456</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.8% - 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B. holmesii</td><td rowspan=1 colspan=1>1/2u</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>9.4% - 90.1%</td><td rowspan=1 colspan=1>488/488</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2% - 100%</td></tr></table>

a Discordant analysis was not performed on 1/1 false negative sample.

c Discordant analysis using PCR/BDS was not performed for the Rhinovirus false negative samples, as PCR/BDS is part of the reference method algorithm for this target.

d Rhinovirus was not detected in 5/8 and detected in 1/8 false positive samples using PCR/BDS analysis in. Discordant analysis was not performed on 2/8 false positive samples.

Influenza A was not detected in $1 / 1$ false negative sample using PCR/BDS analysis.

f Influenza A was not detected in 2/3 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 1/3 false positive samples.

Influenza $\mathrm { A } / \mathrm { H } 1$ was not detected in $1 / 1$ false negative sample using PCR/BDS analysis.

Influenza A/H3 was not detected in $2 / 2$ false positive samples using PCR/BDS analysis.

i Influenza B was not detected in 2/2 false positive samples using PCR/BDS analysis.

j Parainfluenza 2 was not detected in 1/1 false negative sample using PCR/BDS analysis.

k Parainfluenza 4 was not detected in 1/2 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 1/2 false positive samples.

l Discordant analysis using PCR/BDS was not performed for the RSV A false negative samples, as PCR/BDS is part of the reference method algorithm for this target.

m RSV A was not detected in 1/3 and detected in 1/3 false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 1/3 false positive samples.   
n RSV A was not detected in 1/3 and detected in 1/3 false positive samples using bi-directional sequencing analysis. Discordant analysis was not performed in 1/3 samples.   
${ } ^ { \mathrm { { \circ } } } \mathrm { { R S V B } }$ was not detected in $5 / 7$ false positive samples using PCR/BDS analysis. Discordant analysis was not performed on 2/7 false positive samples.   
${ } ^ { \mathrm { p } } \mathrm { h M P V }$ was not detected in 1/2 false negative samples using PCR/BDS analysis. Discordant analysis was not performed on $\%$ false negative samples.   
q Discordant analysis was not performed on 1/1 false positive hMPV sample.   
r Bordetella pertussis was not detected in 1/1 false negative sample using PCR/BDS analysis.   
s Repeat PCR/BDS analysis was performed. B. parapertussis/bronchiseptica was not detected in 1/2 and $B$ . parapertussis was detected in 1/2 false negative samples. No B. bronchiseptica was identified by PCR/BDS in all 7 comparator positive specimens.   
t Repeat PCR/BDS analysis was performed. B. parapertussis/bronchiseptica was not detected in 1/1 false positive sample. u Repeat PCR/BDS analysis was performed. B. holmesii was not detected in 1/2 false negative samples. Contamination from a strong positive contrived B. holmesii sample in a neighboring well during the original PCR/BDS analysis is highly suspected.

# Mock Clinical Study

Frozen contrived NPS in VTM specimens for three bacteria with very low prevalence in the prospective clinical studies and for which retrospective natural NPS specimens in VTM are difficult to acquire were prepared by spiking bacterial strains of known origin and concentration into unique previously-screened, negative clinical NPS specimens in VTM. Organisms used in these preparations were obtained from the following sources: American Type Culture Collection (ATCC), BEI Resources, Nationwide Children’s Hospital (NCH), and Zeptometrix.

A summary of contrived positive clinical specimens included in this study are provided in Tables 58 - 60. Contrived specimens were blinded and assigned Nanosphere identification numbers at Nanosphere, shipped to the three testing sites on dry ice and tested at the study testing sites. The testing sites stored the samples at $\le - 7 0 ^ { \circ } \mathrm { C }$ until ready for Verigene RP Flex testing, at which point the samples were thawed and tested. To avoid bias, a total of 201 negative NPS specimens were blinded and randomized and included with the positive spiked samples to be tested by testing sites.

Table 58: Contrived Clinical Specimens Using Bordetella holmesii Stocks $\mathbf { \left( n = 5 6 \right) }$ .   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Test Site</td><td rowspan=1 colspan=1>Multiples ofLoD</td><td rowspan=1 colspan=1>FinalConcentration(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=8 colspan=1>NCH</td><td rowspan=8 colspan=1>29573</td><td rowspan=10 colspan=1>Site 1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=2 colspan=1>NCH</td><td rowspan=2 colspan=1>29628</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr></table>

<table><tr><td rowspan=1 colspan=1>11</td><td rowspan=3 colspan=5></td><td rowspan=3 colspan=1></td><td rowspan=49 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>16</td><td></td><td></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=5 colspan=5>NCH</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=3 colspan=1>29547</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=3 colspan=5></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=8 colspan=5>NCH</td><td rowspan=8 colspan=1>29546</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=3 colspan=5>NCH</td><td rowspan=3 colspan=1>29613</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=5 colspan=5>NCH</td><td rowspan=5 colspan=1>29575</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=2 colspan=5></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=3 colspan=5>NCH</td><td rowspan=8 colspan=1>29574</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=2>b</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=5 colspan=5></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=3 colspan=1>47</td><td></td><td></td></tr><tr><td></td><td></td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=3 colspan=5></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>52</td><td rowspan=4 colspan=5>NCH</td><td rowspan=6 colspan=1>29572</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=2>b</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=2 colspan=1>54</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>56</td><td></td><td></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr></table>

Table 59: Contrived Clinical Specimens Using Bordetella bronchiseptica Stocks $( \mathbf { n } = 5 5 )$ .   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Test Site</td><td rowspan=1 colspan=1>Multiples ofLoD</td><td rowspan=1 colspan=1>Final Concentration(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>ATCC</td><td rowspan=3 colspan=1>4617</td><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr></table>

<table><tr><td rowspan=1 colspan=1>4</td><td rowspan=5 colspan=3></td><td rowspan=5 colspan=1></td><td rowspan=39 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=8 colspan=3>ATCC</td><td rowspan=8 colspan=1>7773</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=2 colspan=3></td><td rowspan=8 colspan=1>785</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=6 colspan=3>ATCC</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=5 colspan=3>ATCC</td><td rowspan=10 colspan=1>14064</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=2></td><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=3 colspan=1>31</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>32</td><td></td><td></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=6 colspan=3>ATCC</td><td rowspan=8 colspan=1>10580</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=5 colspan=3>ATCC</td><td rowspan=7 colspan=1>19395</td><td rowspan=7 colspan=1>Site 3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=2 colspan=3></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.86E+03</td></tr><tr><td rowspan=1 colspan=1>50</td><td></td><td></td><td rowspan=3 colspan=1>ATCC</td><td rowspan=8 colspan=1>786</td><td rowspan=8 colspan=1>Site 2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>51</td><td></td><td></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.29E+03</td></tr><tr><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22E+04</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=5 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.43E+04</td></tr><tr><td rowspan=1 colspan=1>54</td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr><tr><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.86E+04</td></tr></table>

Table 60: Contrived Clinical Specimens Using Bordetella parapertussis Stocks $( \mathbf { n } = 4 9 )$ )   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="4" rowspan="1">Source</td><td colspan="1" rowspan="1">Strain</td><td colspan="2" rowspan="1">Test Site</td><td colspan="1" rowspan="1">Multiples ofLoD</td><td colspan="1" rowspan="1">Final Concentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="4" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="3" rowspan="1">−</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="3">4</td><td colspan="1" rowspan="3">−</td><td></td><td colspan="2" rowspan="4">ATCC</td><td colspan="1" rowspan="7">15237</td><td colspan="2" rowspan="7">Site 3</td><td colspan="1" rowspan="3"></td><td></td><td></td></tr><tr><td></td><td colspan="1" rowspan="1"></td><td></td><td></td></tr><tr><td></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">−</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">7</td><td></td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">8</td><td></td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="4" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="41"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">12</td><td></td><td colspan="3" rowspan="2">ATCC</td><td colspan="1" rowspan="5">9305</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">15</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">16</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="4" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="4" rowspan="2">ATCC</td><td colspan="1" rowspan="5">BAA-587</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="4" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="4" rowspan="5">ATCC</td><td colspan="1" rowspan="8">15989</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">31</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">32</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="4" rowspan="8">ATCC</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="5">15311</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="4" rowspan="6">ATCC</td><td colspan="1" rowspan="6">29686</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.86E+03</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="2" rowspan="1">b</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.29E+03</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.22E+04</td></tr><tr><td colspan="1" rowspan="1">47</td><td></td><td></td><td colspan="2" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.43E+04</td></tr><tr><td colspan="1" rowspan="1">48</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr><tr><td colspan="1" rowspan="1">49</td><td></td><td></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">4.86E+04</td></tr></table>

The RP Flex mock clinical study (contrived samples) performance data in positive percent and negative percent agreements against the expected results (all sites combined) are presented in the following Table 61 by analyte. One negative contrived sample generated a “No Call” result initially and a “No Call” result again upon retesting per the product instructions by the RP Flex, and was therefore excluded from this performance analysis.

Table 61: RP Flex Mock Clinical Performance (Contrived Samples)   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="2" rowspan="1">PPA</td><td colspan="1" rowspan="1">95% CI</td><td colspan="2" rowspan="1">NPA</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">358/360</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.0% - 99.8%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">359/360</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A/H1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A/H3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">359/360</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">358/360</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.0% - 99.8%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">360/360</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92.7% - 100%</td><td colspan="1" rowspan="1">311/311</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. bronchiseptica</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.5% - 100%</td><td colspan="1" rowspan="1">305/305</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. holmesii</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.6% - 100%</td><td colspan="1" rowspan="1">304/304</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.6% - 100%</td></tr></table>

# Initial No Call Results and Pre-Analysis Errors (PAE) During Clinical Testing

For the total of 3299 specimens tested with the RP Flex in the clinical trials (i.e., 1082 fresh prospective specimens, 1330 frozen prospective specimens, 526 frozen selected retrospective specimens, and 361 frozen contrived specimens), there were 152 specimens that had an initial “No Call” result. Therefore, the initial No Call rate was $4 . 6 \%$ (152/3299) $9 5 \%$ CI: $3 . 9 \% - 5 . 4 \% )$ . Upon repeat testing per the product instructions, all of the specimens except 15 yielded a valid call, resulting in a final No Call rate of $0 . 5 \%$ (15/3299) $9 5 \%$ CI: $0 . 3 \% - 0 . 7 \%$ ).

For a total of 1082 fresh natural specimens, the initial No Call rate was $5 . 9 \%$ (64/1082) ( $9 5 \%$ CI: $4 . 7 \% - 7 . 5 \%$ ), and the final No Call rate upon repeat testing per the product instructions was $0 . 6 \%$ (6/1082) ( $9 5 \%$ CI: $0 . 3 \% \mathrm { ~ - ~ } 1 . 2 \%$ ). For the total of 1856 frozen natural specimens, the initial No Call rate was $4 . 5 \%$ (84/1856) ( $9 5 \%$ CI: $3 . 7 \% - 5 . 6 \% )$ , and the final No Call rate upon repeat testing per the product instructions was $0 . 4 \%$ (8/1856) $9 5 \%$ CI: $0 . 2 \% - 0 . 8 \%$ )

There were a total of 17 PAEs observed during the clinical studies for a PAE failure rate of $0 . 5 \%$ (17/3299) $9 5 \%$ CI: $0 . 3 \% - 0 . 8 \% )$ . Upon repeat testing per the product instructions, all of the specimens yielded a valid call.

The total initial No Call and PAE rate observed during the clinical studies was $5 . 1 \%$ (169/3299) $9 5 \%$ CI: $4 . 4 \% - 5 . 9 \%$ ). Upon repeat testing per the product instructions, the final No Call and PAE rate was $0 . 5 \%$ (15/3299) ( $9 5 \%$ CI: $0 . 3 \% \mathrm { ~ - ~ } 0 . 7 \%$ .

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Table 62: Expected Value (As Determined by RP Flex) Summary by Collection Site for the RP Flex Prospective Clinical Evaluation (Fresh Prospective Specimens) (July 2014 – November 2014)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall (n=815)</td><td rowspan=1 colspan=2>Site 1 (n=34)Mid-Atlantic</td><td rowspan=1 colspan=2>Site 2 (n=54)Northeast</td><td rowspan=1 colspan=2>Site 3 (n=248)Midwest</td><td rowspan=1 colspan=2>Site 4 (n=4)Midwest</td><td rowspan=1 colspan=2>Site 5 (n=437)Southwest</td><td rowspan=1 colspan=2>Site 6 (n=38)Mexico</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.9%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>28.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>29.1%</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>33.2%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>13.2</td></tr><tr><td rowspan=1 colspan=1>B.parapertussis/brohiseptica</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>B. holmesii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Table 63: Expected Value (As Determined by RP Flex) Summary by Collection Site for the RP Flex Prospective Clinical Evaluation (Fresh Prospective Specimens) (February 2015 – March 2015)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall(n=254)</td><td rowspan=1 colspan=2>Site 2 (n=147)Northeast</td><td rowspan=1 colspan=2>Site 7 (n=107)Midwest</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>6.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>6.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>17.3%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>21.2%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>12.1%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=1>B. parapertussis/bronchiseptica</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>B. holmesii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Table 64: Expected Value (As Determined by RP Flex) Summary by Collection Site for the RP Flex Prospective Clinical Evaluation (Frozen Prospective Specimens) (October 2013 – March 2014)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Site 3 (n=1317)Midwest</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Expected Value</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>3.9%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>14.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>5.2%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>15.7%</td></tr><tr><td rowspan=1 colspan=1>B. parapertussis/bronchiseptica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>B. holmesii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

N. Instrument Name: Verigene System

# O. System Descriptions:

1. Modes of Operation:

The Verigene Respiratory Pathogens Flex Nucleic Acid Test (RP Flex) is conducted on the Verigene System, which is a “sample-to-result”, fully automated, bench-top molecular diagnostics workstation. The Verigene System consists of two components: the Verigene Reader and the Verigene Processor SP. A single Verigene Reader may be connected to up to 32 Processor SP units. Each Processor SP unit can be accessed independently through the Reader to run different tests. Each test utilizes single-use, disposable test consumables and a self-contained Verigene Test Cartridge to analyze nucleic acid samples. The RP Flex test is run on the Verigene System, which incorporates automated nucleic acid extraction, amplification, and target detection of specific viral and bacterial nucleic acid gene sequences.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The RP Flex microarray contains two sets of oligonucleotide spots that are used as process controls to check for proper fluid control, extraction, amplification, hybridization and signal detection. The IC1 (hybridization control) internal processing control probes detect the presence (hybridization and signal enhancement) of a DNA oligonucleotide and mediator oligo contained within the Sample Buffer on the Extraction Tray. The IC2 (extraction control) control probes verify the presence of an amplicon for Bacteriophage MS2, which is added to the sample prior to the nucleic acid extraction step. Both the internal processing controls IC1 and IC2 must be present for a valid respiratory pathogen Not Detected call for all targets to be reported. If IC1 or IC2 is Not Detected, a No Call – INT CTL 1 or a No Call – INT CTL 2 result, respectively, is generated. If both IC1 and IC2 are Not Detected, a No Call – INT CTL result is generated. The IC2 is not utilized for the detection of positive samples. The IC1 is required for the detection of positive samples.

A set of regions on microarray which do not contain capture oligonucleotides is used as a control for background signal levels. If the signal in these background regions is too high, a No Call - BKGD is generated. Additionally, negative control oligomer spots are included on the array to ensure the hybridization stringency was sufficient. If the negative control signal is too high a No Call – NEG CTL is generated.

Each capture probe has six separate oligonucleotide spots on the microarray in six different areas. Mathematical algorithms, called spot filters, compare the array spot signals for a particular target as a set. Any individual spot signal in the target set that are determined to be outliers are discarded. After applying the spot filters, the probe set must include four or more capture spots for a Call decision to be made for a target. Additionally, the coefficient of variation of spot intensities in the set must be less than or equal to $70 \%$ . If these requirements are not met a No Call - VARIATION results. The spot signal comparison algorithms protect against some hybridization failures and also against fluidics failures such as the partial filling of the hybridization chamber.

External controls are not provided with the RP Flex. However, the sponsor is making the following recommendations regarding running external controls in the product package insert:

“Good laboratory practice recommends running external positive and negative controls regularly. As an example, viral transport medium may be used as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms may be used as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.”

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision