# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K150617 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2)

# D. Type of Test:

An in vitro molecular diagnostic test for the qualitative detection and differentiation of HSV-1 and HSV-2 DNA in clinician-collected, external anogenital lesion specimens from symptomatic male and female patients.

# E. Applicant:

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The cobas® HSV 1 and 2 Test on the cobas® 4800 system is an automated, qualitative in vitro diagnostic test, that utilizes real-time polymerase chain reaction (PCR), for the direct detection and differentiation of Herpes simplex virus 1 and 2 (HSV-1 and HSV-2)

DNA in clinician-collected, external anogenital lesion specimens from symptomatic male and female patients. The cobas® HSV 1 and 2 Test is intended for use as an aid in diagnosis of anogenital HSV-1 and HSV-2 infections in symptomatic patients.

Warning: The cobas® HSV 1 and 2 Test is not FDA cleared for use with cerebrospinal fluid (CSF) and is not intended to be used for prenatal screening or for individuals under the age of 18 years.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: cobas® 4800 System

# I. Device Description:

The cobas® HSV 1 and 2 Test on the cobas® 4800 system is a real-time polymerase chain reaction (PCR) for the direct detection and differentiation of HSV-1 and HSV-2 DNA in clinician-collected, external anogenital lesion specimens, collected in MSwab Collection, Transport and Preservation System from symptomatic patients.

The cobas® HSV 1 and HSV 2 Test is comprised of two major processes: (1) automated sample preparation to extract nucleic acids from swab specimens; (2) PCR amplification of target DNA sequences using HSV-1 and HSV-2 specific primers, and real-time detection of cleaved fluorescent-labeled HSV-1 and HSV-2 specific oligonucleotide detection probes. An internal control (IC), containing unrelated randomized DNA sequence, is added to all samples prior to automated sample preparation and is amplified and detected simultaneously with each sample to monitor the entire process.

The specimens are collected and stored in the MSwab Collection, Transport and Preservation System for the cobas® HSV 1 and HSV 2 Test. The cobas® HSV 1 and HSV 2 Test is performed using the following reagent kits:

1. cobas® 4800 HSV 1 and HSV 2 Amplification/Detection Kit   
2. cobas® 4800 HSV 1 and HSV 2 Controls and Cofactor Kit   
3. cobas® 4800 System Wash Buffer Kit   
4. cobas® 4800 System Lysis Kit 1   
5. cobas® 4800 System Internal Control Kit 1   
6. cobas® 4800 System Sample Preparation Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s):

BD ProbeTecTM Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays

Reference Method:

A composite reference method which includes a culture method, namely, the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc. (K971662)) and a validated PCR followed by bidirectional sequencing for clinical evaluation.

The clinical performance of the cobas® HSV 1/2 Test was also compared to the culture method alone using the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc. (K971662)).

2. Predicate 510(k) number(s):

K103798

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">BD ProbeTecT Herpes SimplexViruses (HSV 1 &amp; 2) Q" AmplifiedDNA Assays on the BD ViperSystem in Extracted Mode,(K103798)</td><td colspan="1" rowspan="1">Roche cobas® HSV 1 and 2 TestNew Device (K150617)</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The BD ProbeTecTM Herpes SimplexViruses (HSV 1 &amp; 2) Qx AmplifiedDNA Assays (HSV Qx Assays), whentested with the BD ViperTM System inExtracted Mode, use StrandDisplacement Amplificationtechnology for the direct, qualitativedetection and differentiation of HerpesSimplex virus type 1 (HSV1) andHerpes Simplex virus type 2 (HSV2)DNA in clinician-collected externalanogenital lesion specimens. Theassays are indicated for use withsymptomatic individuals to aid in thediagnosis of anogenital HSV1 andHSV2 infections.</td><td colspan="1" rowspan="1">The cobas® HSV 1 and 2 Test on thecobas® 4800 system is an automated,qualitative in vitro diagnostic test, thatutilizes real-time polymerase chainreaction (PCR), for the direct detectionand differentiation of Herpes simplexvirus 1 and 2 (HSV-1 and HSV-2) DNAin clinician-collected anogenital lesionspecimens from symptomatic male andfemale patients. The cobas® HSV 1 and2 Test is intended for use as an aid indiagnosis of anogenital HSV-1 andHSV-2 infections in symptomaticpatients.Warning: The cobas® HSV 1 and 2Test is not FDA cleared for use with</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Warning: The BD ProbeTecTMHerpes Simplex Viruses (HSV 1 &amp; 2)Qx Amplified DNA Assays (HSV QxAssays) are not FDA cleared for usewith cerebrospinal fluid (CSF). Theassays are not intended to be used forprenatal screening or for individualsunder the age of 17 years.</td><td colspan="1" rowspan="1">cerebrospinal fluid (CSF) and is notintended to be used for prenatalscreening or for individuals under theage of 18 years.</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">External anogenital lesions</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detection anddifferentiation of HSV-1 and HSV-2DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes usingfluorescence resonance energytransfer (FRET)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">BD ProbeTecTM Herpes SimplexViruses (HSV 1 &amp; 2) Q" AmplifiedDNA Assays on the BD ViperSystem in Extracted Mode,(K103798)</td><td colspan="1" rowspan="1">Roche cobas® HSV 1 and 2 TestNew Device (K150617)</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Strand Displacement Amplification</td><td colspan="1" rowspan="1">Real-time PCR</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationProcedure</td><td colspan="1" rowspan="1">Automated on BDTM ViperTMSystem in Extracted Mode</td><td colspan="1" rowspan="1">Automated on cobas® 4800 System</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

# Target Selection

The cobas® HSV 1and 2 Test utilizes real-time PCR technology to detect the conserved regions of HSV-1 Thymidine Kinase and HSV-1 DNA Polymerase as well as HSV-2 Thymidine Kinase and HSV-2 Glycoprotein B genes. Fluorogenic target-specific probes are used for the detection of the amplified HSV-1 and HSV-2 DNA as well as Internal Control. Since two HSV type targets are detected with different fluorescent dyes, the cobas® HSV 1 and 2 Test has the ability to simultaneously detect and differentiate HSV-1 and HSV-2. Primer and probe oligonucleotide sequences were designed to select HSV-1 and HSV-2 conserved sequences without cross reacting to other viruses, or other bacterial organisms commonly found in human genital areas.

# Sample Preparation

Sample preparation for the cobas® HSV 1 and 2 Test is automated with the use of the cobas® $\textbf { x } 4 8 0$ instrument. Organisms from swab specimens collected in MSwab medium are lysed with chaotropic agent, proteinase K, and SDS reagents. Released nucleic acids, along with added Internal Control DNA, are bound by magnetic glass particles. The particles are washed and the bound nucleic acids are then eluted into a small volume of buffer. The instrument then takes an aliquot of the eluted material and sets up the PCR reaction with an activated Master Mix.

# PCR Amplification and TaqMan® Detection

The PCR cycling steps and detection of target signal occurs in the cobas® $\textbf { z } 4 8 0$ Analyzer. The Master Mix reagent contains primer pairs and probes for HSV-1, HSV-2 and Internal Control targets. If the target nucleic acid sequences are present, amplification with the corresponding primers will occur by a thermostable DNA polymerase, generating PCR products (amplicons). These products are detected by specific TaqMan probes containing a fluorescent dye and a quencher. Normally, the quencher suppresses the fluorescence of the dye. However, if the PCR product is present, the probe hybridizes to the product and is subsequently cleaved by the $5 ^ { \prime }$ to $3 ^ { \prime }$ nuclease activity of the polymerase. This reaction allows the fluorescence to be emitted from the dye, and the signal is recorded in real time during each PCR cycle by the $\mathbf { c o b a s } \otimes _ { \mathbf { Z } } 4 8 0$ analyzer. The signal is interpreted by the cobas® 4800 System Software and reported as final results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Reproducibility:

The reproducibility of the cobas® HSV 1 and 2 Test on the cobas® 4800 System was established at three sites using contrived clinical samples evaluated across lot, site/instrument, operator, day, and run. HSV test panels were prepared by spiking HSV-1 (Maclntyre strain) and/or HSV-2 (G strain) into contrived sample matrix in transport media at one of three concentrations (Below LOD, $1 \times \mathrm { L O D }$ , and $3 \times \mathrm { L O D } )$ ; HSV-1 and HSV-2 negative panel members were included as panel member controls. In all, there were 6 members per test panel with 3 replicates per panel member included in each run, not including external positive and negative assay controls. Panels were tested at 3 sites by 2 operators per site with 1 valid run per operator per day, for 6 days per lot over 2 lots for 216 tests per panel member.

# HSV-1 reproducibility results

The table below summarizes the percent agreement (two-sided $9 5 \%$ exact Confidence Interval (CI) for the negative panel member and HSV-1 positive panel members.

# HSV-1 Percent agreement by panel member

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>HSV-1</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Number ofValidTest Results</td><td rowspan=1 colspan=1>Agreement (n/N)</td><td rowspan=1 colspan=1>95% CIa</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Below LOD (HSV-1/HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>63.4% (137/216)</td><td rowspan=1 colspan=1>(56.6%, 69.9%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>99.5% (215/216)</td><td rowspan=1 colspan=1>(97.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr></table>

a $9 5 \%$ CI $=$ Two-sided $9 5 \%$ exact binomial confidence interval. b Negative panel members for HSV-1: percent negative agreement was $9 9 . 8 \%$ (431/432) with $9 5 \% \mathrm { C I } ( 9 8 . 7 \% , 1 0 0 . 0 \% ) .$ c Panel members with $_ { 1 \mathrm { ~ X ~ } }$ LOD HSV-1: percent positive agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI $( 9 9 . 1 \%$ , $1 0 0 . 0 \%$ . Note: Results are included as agreement when a positive panel member has a valid positive result for that analyte or when the negative panel member has a valid negative result for both analytes. $\mathrm { C I } =$ confidence interval; $\mathrm { L O D = }$ limit of detection.

The table below presents the percent agreement (negative or positive) by lot, site/instrument, operator, and day for HSV1 test results for each panel.

HSV-1 Percent agreement by panel member for lot, site/instrument, operator, and day   

<table><tr><td colspan="4" rowspan="1"></td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="2">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="2">Lot</td><td colspan="2" rowspan="2">Site/Inst.</td><td colspan="2" rowspan="2">Operator</td><td colspan="2" rowspan="2">Day</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Below LOD(HSV-1/HSV-2)</td><td colspan="1" rowspan="1">41.1</td><td colspan="1" rowspan="1">1.41</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">60.2(65/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">56.9(41/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58.3(21/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">66.7(72/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">68.1(49/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">55.6(20/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">58.3(21/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">65.3(47/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">63.9(23/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">72.2(26/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">72.2(26/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">63.9(23/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">66.7(24/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">58.3(21/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD(HSV-1)</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="4" rowspan="1">canmaGlowb</td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="2">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="2">Lot</td><td colspan="2" rowspan="2">Site/Inst.</td><td colspan="2" rowspan="2">Operator</td><td colspan="2" rowspan="2">Day</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD(HSV-2)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">99.1(107/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.6(71/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.2(35/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.2(35/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1"></td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="2">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="2">Lot</td><td colspan="2" rowspan="2">Site/Inst.</td><td colspan="2" rowspan="2">Operator</td><td colspan="2" rowspan="2">Day</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr></table>

\* For the negative panel member, percent agreement $=$ (number of negative results/total valid results) $\textbf { \em X } 1 0 0$ ; for the positive panel members, percent agreement $=$ (number of positive results/total valid results) $\textbf { x } 1 0 0$ . $\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { { N / A = } }$ not applicable; $\mathrm { S D = }$ standard deviation.

The table below presents the SD and CV $( \% )$ of $\mathrm { C t }$ values for HSV-1 positive panel members overall and attributable to lot, site/instrument, operator, day, and within-run. Across HSV-1 positive panel members, the total SD ranged from 1.10 to 1.41, and the total CV $( \% )$ ranged from $3 . 0 \%$ to $3 . 4 \%$ .

HSV-1 Overall mean, standard deviations, and coefficients of variation $( \% )$ for Ct values from valid results for positive panel members   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=12>Standard Deviation and Percent Coefficients of Variation</td></tr><tr><td rowspan=1 colspan=2>Site/Inst.</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Below LOD(HSV-1/HSV-2)</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>3.0</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { S D } =$ standard deviation.

In summary, the positive percent agreement for the HSV-1 positive panel member “Below LOD (HSV-1/HSV-2)” was $6 3 . 4 \%$ $9 5 \%$ CI: $5 6 . 6 \%$ to $6 9 . 9 \%$ and the positive percent agreement for all other positive panel members was $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 8 . 3 \%$ to $1 0 0 . 0 \%$ ). For the negative panel members, negative percent agreement was $9 9 . 8 \%$ $9 5 \%$ CI: $9 8 . 7 \%$ to $1 0 0 . 0 \%$ . The total SD and total $\% C V$ across all panel members were $\leq 1 . 4 1$ and $\leq 3 . 4 \%$ , respectively.

# HSV-2 reproducibility results

The table below summarizes the percent agreement (two-sided $9 5 \%$ exact CI) for the negative panel member and HSV-2 positive panel members.

HSV-2 Percent agreement by panel member   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Number ofValidTest Results</td><td rowspan=1 colspan=1>Agreement (n/N)</td><td rowspan=1 colspan=1>95% CIa</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Below LOD (HSV-1/HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>56.5% (122/216)</td><td rowspan=1 colspan=1>(49.6%, 63.2%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)b</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0% (216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr></table>

$9 5 \% \mathrm { C I } = 9 5 \%$ exact binomial confidence interval. b Negative panel members for HSV-2: percent negative agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI $( 9 9 . 1 \%$ $1 0 0 . 0 \% )$ . c Panel members with $_ { 1 \mathrm { ~ x ~ } }$ LOD HSV-2: percent positive agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI of $( 9 9 . 1 \%$ , $1 0 0 . 0 \%$ ). Note: Results are included as agreement when a positive panel member has a valid result of positive for the at analyte or when the negative panel member has a valid result of negative for both analytes. $\mathrm { C I } =$ confidence interval; $\mathrm { L O D = }$ limit of detection.

The table below presents the percent agreement (negative and positive) by lot, site/instrument, operator, and day for HSV-2 test results for each panel member.

HSV-2 Percent agreement by panel member for lot, site/instrument, operator, and day   

<table><tr><td colspan="4" rowspan="1"></td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="2">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="2">Lot</td><td colspan="2" rowspan="2">Site/Inst.</td><td colspan="2" rowspan="2">Operator</td><td colspan="2" rowspan="2">Day</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Below LOD(HSV-1/HSV-2)</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">51.9(56/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">65.3(47/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">55.6(20/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">61.1(66/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50.0(36/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">72.2(26/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50.0(18/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">54.2(39/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">47.2(17/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">38.9(14/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">61.1(22/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">63.9(23/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">50.0(18/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">55.6(20/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr></table>

\* For the negative panel member, percent agreement $=$ (number of negative results/total valid results) $\textbf { \em X } 1 0 0$ ; for the positive panel members, percent agreement $=$ (number of positive results/total valid results) $\textbf { x } 1 0 0$ . $\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { { N / A = } }$ not applicable; $\mathrm { S D = }$ standard deviation.

HSV-2 Overall mean, standard deviations, and coefficients of variation $( \% )$ for Ct values from valid results for positive panel members   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=12>Standard Deviation and Percent Coefficient of Variation</td></tr><tr><td rowspan=1 colspan=2>Site/Inst.</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Below LOD(HSV-1/HSV-2)</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>40.3</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>1.9</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { S D = }$ standard deviation.

In summary, the positive percent agreement for the HSV-2 positive panel member “Below LOD (HSV-1/HSV-2)” was $56 . 5 \%$ $9 5 \%$ CI: $4 9 . 6 \%$ to $6 3 . 2 \%$ ), whereas the positive percent agreement for all other positive panel members was $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 8 . 3 \%$ to $1 0 0 . 0 \%$ ). For the HSV-2 negative panel member, negative percent agreement was $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 9 . 1 \%$ to $1 0 0 . 0 \%$ ). The total SD and total CV $( \% )$ across all panel members were $\leq 0 . 9 2$ and $\leq 2 . 3 \%$ , respectively.

# b. Precision:

The precision studies were conducted using three lots of reagents and three instrument for a total of 36 runs over 12 days. A six member study panel was prepared by spiking cultured HSV-1 (Maclntyre strain) and/or HSV-2 (G strain) virus into contrived sample matrix in transport media. The results from the precision study are presented in the tables below.

In-house precision study hit rate analysis   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=3>HSV-1 (N=72)</td><td rowspan=1 colspan=3>HSV-2 (N=72)</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>Hitrate%</td><td rowspan=1 colspan=1>95%2-Sided CI</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>Hitrate%</td><td rowspan=1 colspan=1>95%2-Sided CI</td></tr><tr><td rowspan=1 colspan=1>P1</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 - 5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 - 5%</td></tr><tr><td rowspan=1 colspan=1>P2</td><td rowspan=1 colspan=1>&lt;LOD</td><td rowspan=1 colspan=1>&lt;LOD</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>38 - 62%</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>43 - 67%</td></tr><tr><td rowspan=1 colspan=1>P3</td><td rowspan=1 colspan=1>∼LOD</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 - 5%</td></tr><tr><td rowspan=1 colspan=1>P4</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>~LOD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 - 5%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>93 - 100%</td></tr><tr><td rowspan=1 colspan=1>P5</td><td rowspan=1 colspan=1>~3 x LOD</td><td rowspan=1 colspan=1>~LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95 - 100%</td></tr><tr><td rowspan=1 colspan=1>P6</td><td rowspan=1 colspan=1>∼LOD</td><td rowspan=1 colspan=1>~3 x LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95 - 100%</td></tr></table>

Variance components analysis for precision panel at ${ \textbf { 3 x } }$ LOD (Limit of Detection)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>HSVLevel</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=6>Variance Components/Percent Contribution to Total</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Run/Day</td><td rowspan=1 colspan=1>Random</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=2 colspan=1>∼3xLOD</td><td rowspan=2 colspan=1>37.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.355</td><td rowspan=1 colspan=1>0.289</td><td rowspan=1 colspan=1>0.704</td></tr><tr><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>8.60%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>50.40%</td><td rowspan=1 colspan=1>41.10%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=2 colspan=1>∼3xLOD</td><td rowspan=2 colspan=1>38.2</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.102</td><td rowspan=1 colspan=1>0.345</td><td rowspan=1 colspan=1>0.53</td></tr><tr><td rowspan=1 colspan=1>6.50%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>9.10%</td><td rowspan=1 colspan=1>19.30%</td><td rowspan=1 colspan=1>65.00%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Standard deviations and coefficients of variation $( \% )$ analysis for precision panel at ${ \textbf { 3 x } }$ LOD (Limit of Detection)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=5>Standard Deviation Components/CV Percent</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Run/Day</td><td rowspan=1 colspan=1>Random</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=2 colspan=1>72</td><td rowspan=2 colspan=1>37.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.245</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.595</td><td rowspan=1 colspan=1>0.538</td><td rowspan=1 colspan=1>0.839</td></tr><tr><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.70%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1.60%</td><td rowspan=1 colspan=1>1.40%</td><td rowspan=1 colspan=1>2.20%</td></tr><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=2 colspan=1>72</td><td rowspan=2 colspan=1>38.2</td><td rowspan=1 colspan=1>0.186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.587</td><td rowspan=1 colspan=1>0.728</td></tr><tr><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.60%</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>1.50%</td><td rowspan=1 colspan=1>1.90%</td></tr></table>

c. Linearity/assay reportable range: N/A d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

Positive Control (PC): The Positive Control (PC) contains non-infectious DNA plasmids of both HSV-1 and HSV-2. The PC monitors the nucleic acid extraction, amplification, and detection steps in a given run of the test.

Negative Control (NC): The Negative Control (NC) contains a buffer solution. The NC is processed in each run that contains a batch of HSV specimens and should invalidate the run if there is contamination during the assay process that results in a positive signal in any of the assay target detection channels and/or if Internal Control signal is negative or invalid.

Internal Control: The Internal Control is a lambda phage molecule that contains randomized sequences and targets for IC-specific primers and probe. The IC is added to all specimens and the positive and negative Controls during sample preparation on the cobas® $\times 4 8 0$ instrument. The IC monitors nucleic acid extraction, amplification, and detection steps for a given specimen. The IC is also required for validation of the run controls.

# e. Analytical Sensitivity (Limit of Detection)

The analytical sensitivity (Limit of Detection or LOD) for the cobas® HSV 1 and 2 Test was determined by analyzing quantified HSV-1 and HSV-2 viral cultures diluted at multiple concentration levels into a simulated anogenital lesion swab matrix. The simulated matrix composed of mucin and human cells and mimics the effect of the clinical anogenital background for the cobas® HSV 1 and 2 Test. All levels were tested with at least 21 replicates using the full cobas® HSV 1 and 2 Test workflow across five lots of cobas® HSV 1 and 2 Test reagents. The LoD of the cobas® HSV 1 and 2 Test is defined as the target concentration which can be detected as positive in $\geq$ $9 5 \%$ of the replicates tested, based on results generated by the worst performing lot.

HSV-1 Maclntyre and HSV-2 G strains were tested in the analytical sensitivity study.   
The final LoDs of the cobas® HSV 1 and 2 Test are presented in the table below.

Limit of Detection of cobas® HSV 1 and 2 Test   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ATCC ID</td><td rowspan=1 colspan=1>LoD(TCID50 /mL)</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>Maclntyre</td><td rowspan=1 colspan=1>VR-539</td><td rowspan=1 colspan=1>0.479</td></tr><tr><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>VR-734</td><td rowspan=1 colspan=1>0.112</td></tr></table>

Analytical Inclusivity: In addition, four HSV-1 strains (VR-260, VR-733, VR-735 and VR-1493) and three HSV-2 strains (VR-1779, VR-1781 and VR-540) were tested for reactivity with the cobas® HSV 1 and 2 Test. These strains were obtained from ATCC and were cultured and quantified by Virapur, LLC (California, US). Each strain was diluted in a similar fashion as described in the Limit of Detection Section and was tested in 40 replicates near the LoD. All strains were detected by the assay, demonstrating that the cobas® HSV 1 and 2 Test can detect a broad range of both HSV-1 and HSV-2 strains.

# f. Analytical Specificity/Cross Reactivity and Microbial Panel:

The analytical specificity/cross reactivity of the cobas® HSV 1 and 2 Test was assessed by testing a panel of organisms that could be present in anogenital swab specimens. The panel consisted of bacteria, fungi viruses, human cells, and HSV-1 and HSV-2. Testing was performed with the organisms alone to determine the analytical specificity of the cobas® HSV 1 and 2 Test or in the presence of HSV-1 and HSV-2 at three times of the LoD to assess the potential interference of the organisms cells on detection of HSV-1 and HSV-2 by the cobas® HSV 1 and 2 Test. All samples were prepared by diluting microorganisms or DNA into M4RT viral transport medium prior to testing for cross-reactivity.

All organisms, human cells, HSV-1 and HSV-2 viruses were spiked to $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ Units/mL or higher except for Treponema pallidum; Chlamydia trachomatis serovar $H _ { ; }$ , and Mycoplasma genitalium which were spiked to lower concentrations due to stock concentration limitations.

All bacteria were quantified as Colony Forming Units (CFU) except Chlamydia trachomatis serovar $H$ and Chlamydia trachomatis serovar $I$ which were quantified as Inclusion Forming Units (IFU); Toxoplasma gondii and Treponema pallidum which were quantified as DNA copies and Trichomonas vaginalis which was quantified in cells.

Cytomegalovirus (HHV5), Human Herpes Virus 6B Strain Z29, Human Herpes Virus 7 Strain SB, Echovirus 11, Human enterovirus 71 and Rubella Virus were quantified as $\mathrm { T C I D } _ { 5 0 }$ units/mL; HHV-6A strain GS, HSV-1 and HSV-2 were quantified as viral particles, HIV-1 Strain IIIB and HBV were quantified as International Units (IU). HIV-2 Strain NIH-Z, Epstein-Barr Virus (HHV4), Varicella-Zoster Virus (HHV3) and

HPV plasmids (HPV11, HPV16, HPV18, HPV6) were quantified as DNA copies. Two sources of Human Herpes Virus 8 were used, one was quantified as DNA copies and the other was quantified in cells and estimated as 150 DNA copies per cell. Human Peripheral Blood Mononuclear Cells (PBMC) were quantified as number of cells.

No microorganisms tested positive for HSV-1 or HSV-2 using the cobas® HSV 1 and 2 Test when there was no HSV-1 and HSV-2 target present indicating no cross reactivity with these microorganisms. In addition, none of the organisms or a high concentration of human cells interfered with the detection of HSV-1 and HSV-2 targets. A high concentration of HSV-1 $( 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { v p / m L } )$ did not produce false positive HSV-2 results and a high concentration of HSV-2 $\mathrm { \dot { ( } 1 { x } 1 0 ^ { 6 } \ v p / m L ) }$ did not produce false positive HSV-1 results.

Cross Reactivity & Microbial Interference Panel   

<table><tr><td rowspan=1 colspan=1>Human Adenovirus type 7</td><td rowspan=1 colspan=1>Staphylococcus aureus (MRSA)</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (HHV5)</td><td rowspan=1 colspan=1>Staphylococcus aureus (MSSA)</td><td rowspan=1 colspan=1>Moraxella lacunata</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr Virus (HHV4)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td></tr><tr><td rowspan=1 colspan=1>Varicella-Zoster Virus (HHV3)</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Mycoplasma genitalium**</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 6A strain GS</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Mycoplasma hominis</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 6B Strain Z29</td><td rowspan=1 colspan=1>Chlamydia trachomatis serovar H**</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 7 Strain SB</td><td rowspan=1 colspan=1>Chlamydia trachomatis serovar I</td><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 8*</td><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>HBV</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>HIV-1 Strain IIIB</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>HIV-2 Strain NIH-Z</td><td rowspan=1 colspan=1>Enterococcus faecalis vanB</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>HPV11</td><td rowspan=1 colspan=1>Enterococcus faecium vanA</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>HPV16</td><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>HPV18</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>HPV6</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Rubella Virus</td><td rowspan=1 colspan=1>Haemophilus ducreyi</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Treponema pallidum**</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffii</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp. ozaenae</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>PBMC (human genomic DNA)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Herpes Simplex Virus-1</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Mobiluncus curtisii spp. curtisii</td><td rowspan=1 colspan=1>Herpes Simplex Virus-2</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>-</td></tr></table>

\* Two sources of Human Herpes Virus 8 were tested at $\mathbf { 1 . 0 \ X 1 0 ^ { 6 } }$ copies $\mathrm { \Delta m L }$ (HHV8 viral DNA and HHV8 containing human cell line BCP-1). \*\* Chlamydia trachomatis serovar H was tested at $\mathbf { 1 . 9 \times 1 0 ^ { 4 } }$ IFU/ $\mathrm { m L }$ ; Mycoplasma genitalium was tested at $\mathbf { 1 . 0 \times }$ 105 CFU/mL; Treponema pallidum was tested at $9 . 0 \mathrm { ~ X ~ } 1 0 ^ { 4 }$ copies/mL.

# g. Interfering Studies

This study was performed to evaluate potential interference with the cobas® HSV 1 and 2 Test with twenty commonly used over the counter (OTC) products and antiviral medications, as well as whole blood, human serum albumin, urine, feces, and mucin. All OTC products were tested at or above $2 0 \mathrm { m g }$ or $4 0 \mathrm { m g }$ per swab for solids and $100 \%$ of swab capacity for liquids) the levels that could be reasonably expected to be collected by a swab in an anogenital lesion specimen. Anti-viral medicine was tested at $3 \mathrm { ~ x ~ }$ Cmax (maximum concentration of the drug as defined in the drug’s labeling) in the collected specimen. HSV-1 and HSV-2 were spiked at $\sim 3 \mathrm { ~ x ~ }$ LOD (Limit of Detection) of the cobas® HSV 1 and 2 Test and used as targets in the tests.

No interference was observed for the OTC products except for Vagisil Crème (interference observed at $1 0 ~ \mathrm { m g }$ and above). For whole blood, no interference was observed up to $40 \%$ of the swab capacity; for mucin, no interference was observed up to $4 . 8 ~ \mathrm { m g }$ per swab: for urine, no interference was observed up to $100 \%$ of the swab capacity; for feces, no interference was observed up to $1 . 6 ~ \mathrm { m g }$ per swab and for human serum albumin, no interference was observed up to $1 6 ~ \mathrm { m g }$ per swab. The results are summarized in the table below.

Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Substance/ProductName</td><td colspan="1" rowspan="1">Composition</td><td colspan="1" rowspan="1">Testing Level/Swab</td></tr><tr><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">Human whole blood</td><td colspan="1" rowspan="1">40%, 50%</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin Type II from porcine stomach</td><td colspan="1" rowspan="1">4.8 mg, 8 mg, 12 mg,20 mg</td></tr><tr><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">Human urine</td><td colspan="1" rowspan="1">70%, 100%</td></tr><tr><td colspan="1" rowspan="1">Feces</td><td colspan="1" rowspan="1">Human feces</td><td colspan="1" rowspan="1">1.6 mg, 4 mg</td></tr><tr><td colspan="1" rowspan="1">Human Serum Albumin</td><td colspan="1" rowspan="1">Human serum albumin, fatty acid and globulin free</td><td colspan="1" rowspan="1">8 mg, 16 mg</td></tr><tr><td colspan="1" rowspan="1">K-Y Brand Jelly(Personal Lubricant)</td><td colspan="1" rowspan="1">Glycerin, Hydroxyethylcellulose, ChlorhexidineGluconate, Methylparaben, Sodium Hydroxide,Water</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Gynol II (Contraceptivejelly)</td><td colspan="1" rowspan="1">3% Nonoxynol-9, Lactic Acid, Methylparaben,Povidone, Propylene Glycol, Purified Water,Sodium Carboxymethulcellulose, Sorbic Acid,Sorbitol Solution</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">YeastGardSuppositories</td><td colspan="1" rowspan="1">Pulsatilla, Candida Parapsilosis, Candida Albicans,Bacillus Coagulans, Polyethylene Glycols</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Monistat 1</td><td colspan="1" rowspan="1">2% Miconazole nitrate, Benzoic Acid, CetylAlcohol, Isopropyl Myristate, Polysorbate 60,Potassium Hydroxide, Propylene Glycol, PurifiedWater, Stearyl Alcohol</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Monistat 3</td><td colspan="1" rowspan="1">4% Miconazole nitrate, Benzoic Acid, CetylAlcohol, Isopropyl Myristate, Polysorbate 60,</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Potassium Hydroxide, Propylene Glycol, PurifiedWater, Stearyl Alcohol</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VagiStat 1</td><td colspan="1" rowspan="1">6.5% Tioconazole, Butylated Hydroxyanisole,Magnesium Aluminium Silicate, White Petrolatum</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole vaginalcream</td><td colspan="1" rowspan="1">1% Clotrimazole, Benzyl Alcohol, CetostearylAlcohol, Cetyl Esters Wax, 2-Octyldodecanol,Polysorbate 60, Purified Water, Sodium PhosphateMonobasic, Sorbitan Monostearate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Preparation HHemorrhoidal cream</td><td colspan="1" rowspan="1">14.4% Glycerin, 0.25% Phenylephrine HCl, 1%Pramoxine HCl, 15% white Petrolatum, AloeBarbadensis Leaf Extract, Anhydrous Citric Acid,Butylated Hydroxyanisole, CarboxymethylcelluloseSodium, Cetyl Alcohol, Citric Acid Monohydrate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Abreva cold soretreatment</td><td colspan="1" rowspan="1">10% Docosanol, Benzyl Alcohol, Light MineralOil, Propylene Glycol, Purified Water, SucroseDistearate, Sucrose Stearate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Releev cold soretreatment</td><td colspan="1" rowspan="1">Benzalkonium Chloride, Purified Water, Viracea,Methyl Cellulose, Methyl Paraben, PotassiumSorbate, Propyl Paraben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Acyclovir Cream</td><td colspan="1" rowspan="1">5% Acyclovir, Polyethylene Glycol</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Vagisil Crème</td><td colspan="1" rowspan="1">20% Benzocaine, 3% Resorcinol, Water, MineralOil, Cetyl Alcohol, Propylene Glycol, GlycerylStearate, PEG-100 Stearate, Isopropyl Palmitate,Aloe Barbadensis Leaf Juice, Tocopherl Acetate,Retinyl Plamitate, Zea Mays Oil</td><td colspan="1" rowspan="1">2.5 mg, 5 mg, 10 mg,20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Balneol HygienicCleansing lotion</td><td colspan="1" rowspan="1">Water, Mineral Oil, Propylene Glycol, GlycerylStearate, PEG-100 Stearate, PEG-40 Stearate,Laureth 4, PEG-4 Dilaurate, Lanolin Oil, SodiumAcetate, Carbomer 934, Triethanolamine,Methylparaben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Vagicaine Anti-ItchCream</td><td colspan="1" rowspan="1">20% Benzocaine, 3% Resorcinol, Aloe barbadensisLeaf Extract, Carbomer Homopolymer Type C,Cetyl Alcohol, Cholecalciferol, Corn Oil, GlycerylMonostearte, Isopropyl Myristate, IsopropylPalmitate, Lanolin Alcohol, Methylparaben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">VH Essentials Douche</td><td colspan="1" rowspan="1">3% Povidone-iodine, Purified Water, USP,Octylphenoxypolyethoxyethanol</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Denavir</td><td colspan="1" rowspan="1">1% Penciclovir, Cetomacrogol 1000BP, CetosterylAlcohol, Mineral Oil, Propylene Glycol, PurifiedWater, White Petrolatum</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Famciclovir</td><td colspan="1" rowspan="1">Famciclovir, Hydroxypropyl Cellulose,Hydroxypropyl Methylcellulose, Lactose,Magnesium Stearate, Polyethylene Glycols,Sodium Starch Glycolate, Titanium Dioxide</td><td colspan="1" rowspan="1">0.016 mg</td></tr><tr><td colspan="1" rowspan="1">Valacyclovir</td><td colspan="1" rowspan="1">Valacyclovir Hydrochloride, Carnauba Wax,Colloidal Silicon Dioxide, Crospovidone,Hypromellose, Magnesium Stearate,Microcrystalline Cellulose, Polyethylene Glycol</td><td colspan="1" rowspan="1">0.027 mg</td></tr><tr><td colspan="1" rowspan="1">Cidofovir</td><td colspan="1" rowspan="1">Cidofovir, Sodium hydroxide, Sterile Water</td><td colspan="1" rowspan="1">0.552 mg</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">Acyclovir, Magnesium Stearate, MicrocrystallineCellulose, Povidone, Sodium Starch Glycolate</td><td colspan="1" rowspan="1">0.008 mg</td></tr></table>

# h. Competitive Inhibition

Competitive inhibition of the cobas® HSV 1 and 2 Test was evaluated to assess the potential interference in HSV-1/2 target detection when both HSV-1 and HSV-2 are present in a sample. The panels were constructed with HSV-1 at $\sim 3 \mathrm { ~ x ~ }$ LOD (Limit of Detection), and competing HSV-2 at $\sim 3 0 0 \mathrm { x }$ LOD of the cobas® HSV 1 and 2 Test; and vice versa. One hundred fold higher concentration of HSV-1 did not affect the detection of HSV-2 at approximately $3 \mathrm { ~ x ~ }$ LOD concentration and one hundred fold higher concentration of HSV-2 did not affect the detection of HSV-1 at approximately $3 \mathrm { ~ x ~ }$ LOD concentration.

# i. Carry-over/Cross Contamination

The “worst case use scenario” cross contamination rate for the cobas® HSV 1 and 2 Test was assessed by testing HSV-2 high positive and negative samples that were processed in a checkerboard configuration on cobas® 4800 system. High positive samples were prepared by adding HSV-2 plasmid (DNA) to MSwab medium to generate a Ct that exceeded the signal obtained in specimens from $9 5 \%$ or more of infected patients in the intended use population.

Five runs were performed on each of the three cobas® 4800 systems. The first run on each system contained only the negative samples to confirm that the instrument was clean. The three subsequent runs had alternating positive and negative samples in checkerboard configurations to assess the cross contamination rate. The last run contained only the negative samples to assess the carry-over contamination rate.

In the nine checkerboard runs, 5 out of $4 2 2 \mathrm { H S V }$ negative samples exhibited HSV-2 positive results, for an observed cross-contamination rate of $1 . 1 8 \%$ . All results in the last 3 runs containing only the negative samples were negative, suggesting that there was no carry-over run-to-run contamination.

j. Assay cut-off: Not applicable (N/A)

2. Comparison studies:

a. Method comparison with predicate device:

The performance of the cobas® HSV 1/2 Test was compared to the composite reference method comprised of culture using the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc.) and PCR followed by bidirectional sequencing.

b. Matrix comparison: N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A $b$ . Clinical Specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

The performance of the cobas® HSV 1 and 2 Test was compared to the composite reference method comprised of culture using the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrids, Inc.) and PCR followed by bidirectional sequencing.

The clinical performance of the cobas® HSV 1 and 2 Test was also compared to the culture method alone using the FDA cleared ELVIS® HSV ID/Typing Test System.

# Clinical Performance

The clinical performance of the cobas® HSV 1 and 2 Test was established in a prospective, multi-site, investigation using the combined results of culture and PCR followed by bi-directional Sanger sequencing as the reference method. In addition, the performance of the cobas® HSV 1 and 2 Test was also compared to the culture method alone using the ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System.

A total of 408 prospective specimens (205 female and 203 male) were tested by the cobas® HSV 1 and 2 Test and compared to the combined results of culture (ELVIS® HSV ID/Typing Test System) and PCR followed by bidirectional sequencing. Two external anogenital swab specimens were collected from symptomatic eligible male and female subjects 17 years of age or older attending family planning, OB/GYN and sexually transmitted disease clinics at eight geographically diverse sites (seven across the United States and one in the United Kingdom). The first swab was used for (a) culture by the ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test, (b) PCR followed by bidirectional Sanger sequencing for HSV-1 and HSV-2, and (c) discordant analysis by using an FDA cleared nucleic acid amplification test. The second swab was for the cobas® HSV 1 and 2 Test.

# Comparison with composite reference method (culture and Sanger sequencing)

A total of 408 prospective specimens collected from 205 female and 203 male subjects were evaluated in the study. There were 243 HSV positive subjects;

84 HSV-1 (51 female, 33 male) and 167 HSV-2 (85 female, 82 male) positive subjects, with 8 $( 2 \% )$ subjects positive for both HSV-1 and HSV-2. The clinical performance of the cobas® HSV 1 and 2 Test compared to the composite reference method is presented in the table below.

Comparison of cobas® HSV 1 and 2 Test with the composite reference method   

<table><tr><td rowspan="3"></td><td rowspan="3"></td><td colspan="6">Composite Reference Method</td></tr><tr><td colspan="3">HSV-1</td><td colspan="3">HSV-2</td></tr><tr><td>Positive</td><td>Negative</td><td>Total</td><td>Positive</td><td>Negative</td><td>Total</td></tr><tr><td rowspan="3">cobas® HSV 1 and 2 Test</td><td>Positive</td><td>78</td><td>4a</td><td>82</td><td>162</td><td>13c</td><td>175</td></tr><tr><td>Negative</td><td>6b</td><td>320</td><td>326</td><td>5d$</td><td>228</td><td>233</td></tr><tr><td>Total</td><td>84</td><td>324</td><td>408</td><td>167</td><td>241</td><td>408</td></tr><tr><td>92.9% (78/84) Sensitivity:</td><td colspan="3">HSV-1 (95% CI = 85.3% - 96.7%)</td><td>HSV-2 Sensitivity:</td><td colspan="3">97.0% 162/167) (95% CI = 93.2% - 98.7%)</td></tr><tr><td>Specificity:</td><td colspan="3">98.8% (320/324) (95% CI = 96.9% - 99.5%)</td><td>Specificity:</td><td colspan="3">94.6% (228/241) (95% CI = 91.0% - 96.8%)</td></tr><tr><td>PPV:</td><td colspan="3">95.1% (78/82) (95% CI = 88.1% - 98.1%)</td><td>PPV:</td><td colspan="3">92.6% (162/175) (95% CI = 87.7% - 95.6%)</td></tr><tr><td>NPV:</td><td colspan="3">98.2% (320/326) (95% CI = 96.0% - 99.2%)</td><td>NPV:</td><td colspan="3">97.9% (228/233) (95% CI = 95.1% - 99.1%)</td></tr></table>

a Of the 4 specimens with HSV-1 false-positive cobas® HSV 1 and 2 Test results relative to the Reference Method, 2 were HSV1 positive by a FDA-cleared nucleic acid amplification test. b Of the 6 specimens with HSV-1 false-negative cobas® HSV 1 and 2 Test results relative to the Reference Method, all 6 were HSV-1 negative by a FDA-cleared nucleic acid amplification test. c Of the 13 specimens with HSV-2 false-positive cobas® HSV 1 and 2 Test results relative to the Reference Method, 5 were HSV-2 positive by a FDA-cleared nucleic acid amplification test. d Of the 5 specimens with HSV-2 false-negative cobas® HSV 1 and 2 Test results relative to the Reference Method, all 5 were HSV-2 negative by a FDA-cleared nucleic acid amplification test.

# Comparison with culture

The clinical performance of the cobas® HSV 1 and 2 Test was compared to the culture method alone using the ELVIS® HSV ID and ${ \boldsymbol { \mathrm { D } } } ^ { 3 }$ Typing Test system. The ELVIS® HSV ID and ${ \boldsymbol { \mathrm { D } } } ^ { 3 }$ Typing Test system used in this study is unable to detect patients co-infected with HSV-1 and HSV-2 and cannot detect HSV-1 if HSV-2 is detected first. Consequently, if a specimen was positive for HSV-2, it was removed from the calculation of the HSV-1 clinical performance. Only HSV-2 negative anogenital specimens were detected for HSV-1. Therefore, the number of samples used for the calculation of HSV-1 clinical performance equals the total number of specimens (408) minus the number of samples positive for HSV-2 by culture (129).

The clinical performance of the cobas® HSV 1 and 2 Test compared to the culture reference method is presented in the table below.

Comparison of cobas® HSV 1 and 2 Test with culture   

<table><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=6>Culture Reference Methoda</td></tr><tr><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>cobasHSV 1and 2 Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>13b</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>47c</td><td rowspan=1 colspan=1>175</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>233</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>408</td></tr></table>

a The reference viral culture and typing method (ELVIS® HSV ID and ${ \boldsymbol { \mathrm { D } } } ^ { 3 }$ Typing Test system) used in this study is unable to detect co-infected patients. Only HSV-2 negative specimens can be typed for HSV-1. Therefore, the number of samples used for the calculation of HSV-1 clinical performance equals the total number of evaluable specimens (408) minus the number of samples positive for HSV-2 by culture (129) for 279 evaluable specimens.

b Of the 13 specimens with HSV 1 false-positive cobas® HSV 1 and 2 Test results relative to the culture and typing, 6 were HSV-1 positive by a FDA-cleared nucleic acid amplification test and 4 of which were also HSV-1 positive by Sanger sequencing; 5 additional samples were HSV-1 positive by Sanger sequencing alone.

c Of the 47 specimens with HSV 2 false-positive cobas® HSV 1 and 2 Test results relative to the culture and typing, 32 were HSV-2 positive by a FDA-cleared nucleic acid amplification test.

d The one specimen with a HSV 2 false-negative cobas® HSV 1 and 2 Test result relative to the culture and typing was HSV-2 negative by a FDA-cleared nucleic acid amplification test.

Note: $\mathrm { C I } =$ (Score) confidence interval, PPV $=$ positive predictive value, NPV $=$ negative predictive value.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

# Prevalence

The prevalence of HSV-1 and HSV-2 observed during the multi-center clinical trial was calculated for the cobas® HSV 1 and 2 Test. The prevalence rates for HSV-1 and HSV-2 were individually established as $2 0 . 6 \%$ (84/408) and $4 0 . 9 \%$ (167/408) for anogenital samples. The gender and age distribution from anogenital specimens and the expected prevalence values for the cobas® HSV 1 and 2 Test from anogenital specimens is shown by age for males and females in the tables below.

Gender and Age Distribution for the cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test from anogenital specimens $\mathbf { \Delta N = 4 0 8 }$ evaluable results)   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=1 colspan=2>Gender</td><td rowspan=2 colspan=1>Total (N)</td></tr><tr><td rowspan=1 colspan=1>Male (n)</td><td rowspan=1 colspan=1>Female (n)</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>18 to &lt;21</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>≥21</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>350</td></tr><tr><td rowspan=1 colspan=1>Total (N)</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>408</td></tr></table>

Expected Prevalence Values for the cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test for anogenital specimens from males by age $( \mathbf { n } = 2 \mathbf { 0 } 3$ evaluable results)   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>Total(N)</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Positive(n)</td><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Positive(n)</td><td rowspan=1 colspan=1>Prevalence(%)</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>18 to &lt;21</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>33.3</td></tr><tr><td rowspan=1 colspan=1>≥21</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>16.0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>41.0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>16.3</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>40.4</td></tr></table>

Expected Prevalence Values for the cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test for anogenital specimens from females by age $( \mathbf { n } = 2 \mathbf { 0 } 5$ evaluable results)   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>Total(N)</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Positive(n)</td><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Positive(n)</td><td rowspan=1 colspan=1>Prevalence(%)</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>18 to &lt;21</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>41.5</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>31.7</td></tr><tr><td rowspan=1 colspan=1>≥21</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>21.0</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>43.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>41.5</td></tr></table>

# Positive and Negative Predictive Value

Hypothetical positive and negative predictive values (PPV & NPV) for the cobas® HSV 1 and 2 Test are shown in the table below. These calculations are based on hypothetical prevalence and overall sensitivity and specificity for anogenital swab specimens as determined in the clinical trial.

For HSV-1, the calculations are based upon an overall sensitivity and specificity of $9 2 . 9 \%$ and $9 8 . 8 \%$ , respectively.

For HSV-2, these calculations are based upon an overall sensitivity and specificity of $9 7 . 0 \%$ and $9 4 . 6 \%$ , respectively, for anogenital swabs.

# Hypothetical Positive and Negative Predictive Values (PPV & NPV) for the cobas® HSV 1 and 2 Test from anogenital specimens

<table><tr><td rowspan=2 colspan=1>HypotheticalPrevalence (%)</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPVa (%)</td><td rowspan=1 colspan=1>NPVb (%)</td><td rowspan=1 colspan=1>PPVa (%)</td><td rowspan=1 colspan=1>NPVb (%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>48.6</td><td rowspan=1 colspan=1>99.8</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>89.3</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>66.6</td><td rowspan=1 colspan=1>99.6</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>76.0</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>99.2</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>99.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>98.7</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>95.4</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>97.9</td></tr></table>

a $\mathrm { P P V } \ =$ (Sensitivity x Prevalence) / (Sensitivity x Prevalence $^ +$ [1 - Specificity] x [1 - Prevalence]). ${ } ^ { \mathsf { b } } \mathrm { \ N P V = }$ (Specificity x [1 - Prevalence]) / ([1 - Sensitivity] x Prevalence $^ +$ Specificity x [1 - Prevalence]).

# N. Instrument Name

cobas® 4800 System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___ X___ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _ X ___ or No

3. Specimen Identification:

Specimens are identified using barcodes on specimen vials.

4. Specimen Sampling and Handling:

Specimens are placed on the cobas $\textbf { x } 4 8 0$ instrument as open tubes and specimen processing is fully automated. After completion of specimen processing, the user transfers the plate carrier to the cobas $\textbf { z } 4 8 0$ instrument for automated amplification and detection. Specimens can be processed directly from primary collection vials or as aliquots of the specimen in secondary vials. See section I for more information on specimen handling.

5. Calibration:

No calibration is required by the user. Roche technicians perform calibration periodically as required.

6. Quality Control:

See section M.1.d for information on internal and external controls.

# P. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# Q. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.