# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

To obtain a substantial equivalence determination for the hemoFISH Masterpanel

# C. Measurand:

rRNA from the following organisms on slides prepared from positive blood cultures that demonstrate the presence of organisms by Gram stain:

<table><tr><td>Gram Positive Organisms</td><td>Gram Negative Organisms</td></tr><tr><td>Staphylococcus species</td><td>Enterobacteriaceae</td></tr><tr><td>Staphylococcus aureus</td><td>Escherichia coli</td></tr><tr><td>Streptococcus species</td><td>Klebsiella pneumoniae</td></tr></table>

# D. Type of Test:

A qualitative fluorescence in situ hybridization (FISH) assay using fluorescently labeled Beacon probes

E. Applicant: miacom diagnostics GmbH

F. Proprietary and Established Names: hemoFISH Masterpanel

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.2660: Microorganism differentiation and identification device

2. Classification: Class I

3. Product code:

JSS: Kit, Identification, Enterobacteriaceae

MCS: DNA-Probe, Staphylococcus aureus MDK: DNA-Probe, Reagents, Streptococcal

4. Panel:

83: Microbiology

# H. Intended Use:

1. Intended use(s):

hemoFISH Masterpanel is a qualitative nucleic acid hybridization assay performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system that demonstrates the presence of organisms as determined by Gram stain and is intended for the identification of the following species / genera / families:

<table><tr><td>Gram Positive Organisms</td><td>Gram Negative Organisms</td></tr><tr><td>Staphylococcus spp.</td><td>Enterobacteriaceae</td></tr><tr><td>Staphylococcus aureus</td><td>Escherichia coli</td></tr><tr><td>Streptococcus spp.</td><td>Klebsiella pneumoniae</td></tr></table>

The hemoFISH Masterpanel is indicated as an aid in the diagnosis of specific agents of bacteremia and results should be used in conjunction with other clinical and laboratory findings. Positive hemoFISH Masterpanel results do not rule out co-infection with organisms not included in the hemoFISH Masterpanel. The hemoFISH Masterpanel is not intended to monitor treatment for bacteremia.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not identified by the hemoFISH Masterpanel, and for species determination of Staphylococcus spp., Streptococcus spp., and Enterobacteriaceae that are not identified by the hemoFISH Masterpanel.

2. Indication(s) for use: hemoFISH Masterpanel is indicated as an aid in the diagnosis of bacteremia.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

# Microscope

Fluorescence microscope equipped with a $1 0 0 \mathrm { x }$ oil immersion objective and 10x eyepiece (1000x magnification is required).

· Filterset A for ATTO550-label: absorption maximum $5 5 4 \mathrm { n m }$ /emission maximum 576nm   
· Filterset B for FAM-label: absorption maximum $4 9 4 \mathrm { n m }$ /emission maximum $5 2 0 \mathrm { n m }$   
· Illumination device comparable to 100W HBO lamp

# Hotplate

Hotplate that can maintain an operating temperature of $5 2 { \pm } 1 ^ { \circ } \mathrm { C }$

# I. Device Description:

The hemoFISH Masterpanel is a Fluorescence In Situ Hybridization (FISH)-based assay that uses DNA Beacon probes to detect specific bacterial rRNA sequences in intact cells that are fixed on microscope slides prepared from positive blood cultures. The assay identifies 3 groups each of Gram-positive and Gram-negative bacteria to family, genus or species level within 30 minutes.

# J. Substantial Equivalence Information: (Provided by sponsor)

1. Predicate device name(s): AdvanDx, Inc., Gram-Negative QuickFISH

2. Predicate 510(k) number(s): K123418

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device K150031</td><td>Predicate K123418</td></tr><tr><td>Intended Use</td><td>hemoFISH Masterpanel is a qualitative nucleic acid hybridization assay performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system that demonstrates</td><td>Gram-Negative QuickFISH BC is a multicolor, qualitative nucleic acid hybridization assay intended for the identification of Escherichia coli and/or Pseudomonas aeruginosa and/or Klebsiella pneumoniae on smears from</td></tr></table>

<table><tr><td colspan="2">Similarities</td></tr><tr><td>Item</td><td>Device K150031</td></tr><tr><td></td><td>the presence of organisms as determined by Gram stain and is intended for the identification of the following species / genera / families: Gram Positive Organisms Staphylococcus spp. Staphylococcus aureus Streptococcus spp. Gram Negative Organisms Enterobacteriaceae Escherichia coli Klebsiella pneumoniae The hemoFISH Masterpanel is indicated as an aid in the diagnosis of specific agents of bacteremia and results should be used in conjunction with other clinical and laboratory findings. Positive hemoFISH Masterpanel results do not rule out co- infection with organisms not included in the hemoFISH Masterpanel. The hemoFISH Masterpanel is not intended to monitor treatment for bacteremia. Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceK150031</td><td rowspan=1 colspan=1>PredicateK123418</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>determination ofStaphylococcus spp.,Streptococcus spp., andEnterobacteriaceae that arenot identified by thehemoFISH Masterpanel.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Fluorescence In SituHybridization (rRNA)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Positive blood cultures fromstandard automated bloodculture device, confirmed byGram stain</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Visual by qualitativefluorescence microscopy</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceK150031</td><td rowspan=1 colspan=1>PredicateK123418</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>Identifies the following tospecies/genus/family level:Staphylococcus speciesStaphylococcus aureusStreptococcus speciesEnterobacteriaceaeEscherichia coliKlebsiella pneumoniae</td><td rowspan=1 colspan=1>Identifies the following tospecies level:Escherichia coliPseudomonas aeruginosaKlebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Probe type</td><td rowspan=1 colspan=1>Self-reporting DNA probeslabeled with green (FAM)or red (ATTO550)fluorophores</td><td rowspan=1 colspan=1>Self-reporting PeptideNucleic Acid (PNA) probeslabeled with green (FAM)or red(tetramethylrhodamine)fluorophores</td></tr><tr><td rowspan=1 colspan=1>Probe HybridizationMixtures</td><td rowspan=1 colspan=1>Four (4)i.Eubacteriaii. Staphylococcus spp.-Enterobacteriaceaeiii. S. aureus-E. coliiv.Streptococcus spp.- K.pneumoniae</td><td rowspan=1 colspan=1>One (1)E. coliP. aeruginosaK. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Integral Eubacterial probeon each slide</td><td rowspan=1 colspan=1>Pre-fixed QuickFISHControl Slides</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

The hemoFISH Masterpanel is a Fluorescence In Situ Hybridization (FISH)-based assay that uses DNA Beacon probes to detect specific bacterial rRNA sequences on microscope slides prepared from positive blood cultures. The assay identifies the targeted Gram-positive and Gram-negative panel members to family, genus or species level within 30 minutes. Other “off-panel” species are also detected using a universal Eubacterial probe.

The method is based on classical fluorescence in situ hybridization (FISH) using fluorescently labeled molecular beacon probes. The molecular beacons consist of a DNA sequence folded into a hairpin-like structure that is linked to a fluorophore on the 5´end and to a quencher on the 3´end. A part of the DNA sequence on both ends forms a stem region through complementary base-pairing. This structure keeps the fluorophore and quencher in close proximity, suppressing the generation of fluorescent signal. The DNA sequence between the stem regions of the probe is complementary to an rRNA region that is specific to the targeted group of organisms. Because each bacterial cell includes more than 10,000 copies of rRNA, no amplification step is necessary. The DNA sequences of the beacons that are specific to the target region form a loop-like structure and are able to bind to their rRNA targets after passing through the bacterial cell wall and membrane. In a bound state, the fluorophore of the beacon is spatially separated from its quencher and fluoresces when excited with an appropriate light source.

Every rRNA copy with a bound beacon contributes to the overall fluorescent signal and cells containing hybridized probe can be detected under a fluorescence microscope. Unbound beacons do not fluoresce, so that no washing step is required.

To conduct the assay, solutions of fluorescently labeled molecular DNA beacons are dispensed onto fixed, permeablized cells that are prepared from a positive blood culture. Different wells of the microscope slide are used for each probe combination. Hybridization is carried out at $5 2 { \pm } 1 ^ { \circ } \mathrm { C }$ for 10 minutes. Slides are immersed in a bath containing a stop solution and rinsed in alcohol to end the reaction. Mounting Medium is then added to each well to prevent fading of fluorescence. After applying a cover slip, the slide is ready for examination by fluorescence microscopy.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the hemoFISH Masterpanel was assessed by testing slides prepared from positive blood cultures at 3 sites over a period 10 days. Two sites (A and B) used cultures grown in VersaTREK REDOX 1 medium and one site (C) used BD BACTEC Plus Aerobic/F Culture Vials. Cultures that were positive for 9 different bacterial species were included in the study. All slides were prepared at “bottle ring”, dried and stored until testing. Three replicate slides for each strain were tested on each day. There were two operators per site and each operator tested 45 slides (total $= 9 0$ slides per strain). The results are summarized in Tables 1 and 2. The study showed that the results of the hemoFISH Masterpanel were reproducible, with no important differences in performance between sites, type of culture medium, operators or days of testing.

Table 1. Summary of results from the Reproducibility Study stratified by site and overall   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>ExpectedhemoFISHResult</td><td rowspan=1 colspan=4>Number Correct/Number Tested (%)</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>Streptococci</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>Streptococci</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29/30 2(96.7)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>89/90(98.9)</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>Other 1</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>29/29 3(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>89/89(100)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>29/30 4(96.7)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>89/90(98.9)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>S. maltophilia</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29/29 2(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>89/89(100)</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>268/269(99.6)</td><td rowspan=1 colspan=1>268/269(99.6)</td><td rowspan=1 colspan=1>270/270(100)</td><td rowspan=1 colspan=1>806/808(99.8)</td></tr></table>

1 Other: Other (not Staphylocci, Streptococci or Enterobacteriaceae) 2 1 sample reported as Other (not Staphylocci, Streptococci or Enterobacteriaceae) due to failure to obtain signal from Streptococcus Beacon 3 1 sample reported as Fail due to green fluorescence in Field 1 (Negative Control) was excluded from calculation of percent agreement 4 1 sample reported positive for Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae)

Table 2. Summary of results from the Reproducibility Study by hemoFISH probe   

<table><tr><td rowspan=1 colspan=1>Beacon Probe</td><td rowspan=1 colspan=1>Positive Agreement (%)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>89/89 (100)</td></tr><tr><td rowspan=1 colspan=1>K.pneumoniae</td><td rowspan=1 colspan=1>89/90(98.9)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>178/179(99.4)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>90/90 (100)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>179/180(99.4)</td></tr><tr><td rowspan=1 colspan=1>Eubacteria</td><td rowspan=1 colspan=1>359/359 (100)</td></tr></table>

1 Samples that are positive by the Eubacterial probe and not by any other probe(s) in the hemoFISH panel are reported as “Other (not Staphylococci, Streptococci or Enterobacteriaceae)”

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

One well of each hemoFISH Masterpanel microscope slide is used for quality control. This well is used for hybridization of an ATTO550-labeled Eubacterial probe. In order for the results from hybridization of the hemoFISH probes in the other wells of the slide to be interpreted, cells in the control well with a morphology that is consistent with that observed under Gram stain should fluoresce red (Positive Control) and there should be no green fluorescent cells (Negative Control). In addition, in order to interpret positive results from E. coli, $K .$ . pneumoniae and $S .$ . aureus species-specific probes, the appropriate family or genus probe must also produce a positive result.

Table 3 summarizes the observed failure modes associated with “Invalid” or “Fail” test results during the Clinical Study described in Section M 3.

Table 3. Summary of failure modes associated with Invalid/Fail test results   

<table><tr><td rowspan=1 colspan=1>Failure Mode</td><td rowspan=1 colspan=1>Assay Result</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>In-house</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Beacon Probe</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Sub-Total</td><td rowspan=1 colspan=1>Fail/Invalid</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Total Samples Tested 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>590</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=2 colspan=2>Percent Invalid/Fail</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=4>2.6</td></tr></table>

1 Genus/family probe failed when the corresponding species probe produced a positive result 2 Sum of all samples tested in the study prior to exclusion of duplicates, etc. (refer to Section M 3(a), Table 16)

Additional Positive and Negative External Controls comprised of cultured isolates may be tested as appropriate. Instructions for preparation of External Positive and Negative Controls are included in the Package Insert. The recommended bacterial strains for use as External Controls and expected results are shown in Table 4.

Table 4. Recommended strains for use as External Controls and expected results   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>ATCC No.</td><td rowspan=1 colspan=7>Expected Result by hemoFISH Probe</td></tr><tr><td rowspan=1 colspan=1>EUB</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>KP</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>9144</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>14948</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>10145</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>12403</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>13883</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr></table>

EUB: Eubacteria; STA: Staphylococcus; ENT: Enterobacteriaceae; SA: S. aureus; EC: E. coli; STR: Streptococcus; KP: $K .$ . pneumoniae

During the clinical study, a total of 244 External Controls were tested ( $8 8 E .$ . coli, 75 S. aureus and 81 $P$ . aeruginosa), all $( 1 0 0 \% )$ of which produced the expected results.

Note: No External Controls were tested during the Clinical Study to monitor the performance of the Streptococcus or $K .$ . pneumoniae probes. However, the Eubacterial Control was included on every slide and served to monitor hybridization conditions and clinical performance for the detection of these analytes was determined to be acceptable.

# Specimen Stability

Studies were performed to evaluate the ability of the hemoFISH Masterpanel to produce the expected results with blood culture bottles that were held under different conditions after turning positive. Testing was conducted with BD BACTEC Plus Aerobic/F blood culture bottles inoculated with different species of bacteria representing each of the targeted analytes. After turning positive, the bottles were either tested with the hemoFISH Masterpanel immediately or held for 8 or 12 hours at either room temperature or $3 5 \mathrm { { ^ \circ C } }$ prior to testing. The appropriate results were obtained for each organism at each time point. The instructions in the Package Insert therefore indicate that testing with the hemoFISH Masterpanel may be conducted up to 12 hours after bottle ring.

# Fixed Slide Stability

The stability of dried microscope slides after heat fixation and prior to permeablization and hybridization was established by storing fixed slides in the dark for up to 14 days at ${ \sim } 2 5 ^ { \circ } \mathrm { C }$ prior to testing with the hemoFISH Masterpanel. The slides were prepared from BD BACTEC Plus Aerobic/F Culture Vials using the strains listed in Table 5. All results were as expected. The Package Insert indicates that fixed slides may be stored for up to 14 days in the dark prior to testing.

# Post Hybridization Slide Stability

The stability of microscope slides after hybridization and mounting was determined by reading slides prepared from positive blood cultures of different organisms after storage for up to 24 hours in the dark at ${ \sim } 2 5 ^ { \circ } \mathrm { C }$ . The bacterial strains and culture media used in the study are shown in Table 5. All results at each time point (0, 2, 4 and 24 hours) were as expected. The Package Insert indicates that slides may be stored for up to 12 hours in the dark after hybridization prior to reading.

Table 5. Bacterial strains and culture media used in determination of slide stability   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>ATTC No.</td><td rowspan=1 colspan=1>Fixed SlideStability</td><td rowspan=1 colspan=2>Post Hybridization Stability</td><td rowspan=2 colspan=1>ExpectedhemoFISHResult</td></tr><tr><td rowspan=1 colspan=1>BD BACTECPlus Aerobic/F</td><td rowspan=1 colspan=1>VersaTREKREDOX 11</td><td rowspan=1 colspan=1>BacT/ALERTPF PediatricFAN</td></tr><tr><td rowspan=1 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>12403</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>STR</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>9144</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>SA</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>19606</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>14948</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>EC</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>29212</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>35667</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>13883</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>KP</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>10145</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>29245</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>ENT</td></tr><tr><td rowspan=1 colspan=1>S. maltophilia</td><td rowspan=1 colspan=1>51331</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>6303</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>STR</td></tr><tr><td rowspan=1 colspan=1>S. aureus /K. pneumoniae</td><td rowspan=1 colspan=1>914413883</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>SA/KP</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosaS. maltophila</td><td rowspan=1 colspan=1>1014551331</td><td rowspan=1 colspan=1>Not Done</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Other</td></tr></table>

X: Condition tested; STR: Streptococci; SA. S. aureus; EC: $E$ . coli; KP: K. pneumoniae; ENT: Other Enterobacteriaceae (not E. coli or K. pneumoniae); Other: Other (not Staphylococci, Streptococci or Enterobacteriaceae) VersaTREK REDOX 1 was the most common culture medium in the Clinical Study used to evaluate the performance of the hemoFISH Masterpanel (refer to Section M 3(a), Table 28)

# d. Detection limit:

The limit of detection (LOD) of each probe in the hemoFISH Masterpanel was estimated by testing serial dilutions of different bacterial species prepared from positive BD BACTEC Plus Aerobic/F blood culture vials. The LOD was confirmed by testing 20 replicates at the estimated LOD target level. Target levels were determined by colony counts. The LOD was considered confirmed if ${ \geq } 1 9 / 2 0$ replicates produced a positive result for the hemoFISH probe(s) under evaluation (Table 6).

Table 6. Analytical sensitivity of the hemoFISH Masterpanel   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>ATCC Strain</td><td rowspan=1 colspan=2>hemoFISH Probe</td><td rowspan=2 colspan=1>LOD(CFU/mL)</td><td rowspan=2 colspan=1>hemoFISH% Positive (n = 20)</td></tr><tr><td rowspan=1 colspan=1>Family/Genus</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>14948</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>105$</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>13883</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>29245</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>105$</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>27956</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>9144</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>14990</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>6303</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT: Enterobacteriaceae; STA: Staphylococcus; STR: Streptococcus

# e. Analytical specificity:

The analytical specificity of the hemoFISH Masterpanel was evaluated using suspensions of different organisms prepared from colonies grown on solid media that were suspended in hemoFISH Clinical Sample Buffer. The results are summarized in Tables 7 to 11. The observed false positive and false negative results are noted in the Limitations Section of the Package Insert.

Table 7. Specificity of the hemoFISH Masterpanel for $E$ . coli, $K .$ . pneumoniae and S. aureus   

<table><tr><td rowspan=1 colspan=5>hemoFISH Masterpanel Results for E. coli, K. pneumoniae and S. aureus</td></tr><tr><td rowspan=1 colspan=2>Strains Correctly Identified to Species Level</td><td rowspan=1 colspan=3>False Positive for E. coli, K. pneumoniae or S. aureus 1</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>hemoFISH Result</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 14948</td><td rowspan=1 colspan=1>Escherichia albertii</td><td rowspan=1 colspan=1>DSM 17582</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 35218</td><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>ATCC 35469</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCCBAA-830</td><td rowspan=1 colspan=1>K. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaesubsp. ozaenae</td><td rowspan=1 colspan=1>ATCC 11296</td><td rowspan=1 colspan=1>Macrococcus caseolyticus</td><td rowspan=1 colspan=1>ATCC 35662</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp.pneumoniae</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>ATCC 9207</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp.rhinoscleromatis</td><td rowspan=1 colspan=1>ATCC 13884</td><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>ATCC 9199</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 11822</td><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. coagulans</td><td rowspan=1 colspan=1>ATCC 49545</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 17091</td><td rowspan=1 colspan=1>Staphylococcus schleiferisubsp. schleiferi</td><td rowspan=1 colspan=1>ATCC 43808</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 9144</td><td rowspan=3 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 43300</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 29213</td></tr></table>

As noted in the Limitations Section of the Package Insert

Table 8. Specificity of the hemoFISH Masterpanel for “Other Staphylococci (not S. aureus)”   

<table><tr><td rowspan=1 colspan=4>Species Correctly Identified as &quot;Other Staphylococci (not S. aureus)&quot; by hemoFISH Masterpanel</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus arlettae</td><td rowspan=1 colspan=1>ATCC 43957</td><td rowspan=1 colspan=1>Staphylococcus lutrae</td><td rowspan=1 colspan=1>DSM 10244</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus capitis</td><td rowspan=1 colspan=1>ATCC 35661</td><td rowspan=1 colspan=1>Staphylococcus microti</td><td rowspan=1 colspan=1>DSM 22147</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus chromogenes</td><td rowspan=1 colspan=1>ATCC 43764</td><td rowspan=1 colspan=1>Staphylococcus pasteuri</td><td rowspan=1 colspan=1>ATCC 51129</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus condimenti</td><td rowspan=1 colspan=1>DSM 11674</td><td rowspan=1 colspan=1>Staphylococcus piscifermentans</td><td rowspan=1 colspan=1>ATCC 51136</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 14990</td><td rowspan=1 colspan=1>Staphylococcus pseudointermedius</td><td rowspan=1 colspan=1>ATCC 49444</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus equorum subsp. equorum</td><td rowspan=1 colspan=1>ATCC 43958</td><td rowspan=1 colspan=1>Staphylococcus pulvereri (vitulinus)</td><td rowspan=1 colspan=1>ATCC 51145</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum</td><td rowspan=1 colspan=1>ATCC 700401</td><td rowspan=1 colspan=1>Staphylococcus rostri</td><td rowspan=1 colspan=1>DSM 21968</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus heamolyticus</td><td rowspan=1 colspan=1>ATCC 29970</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td><td rowspan=1 colspan=1>ATCC 15305</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hominis subsp. hominis</td><td rowspan=1 colspan=1>ATCC 27844</td><td rowspan=1 colspan=1>Staphylococcus sciuri</td><td rowspan=1 colspan=1>ATCC 29061</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>ATCC 29663</td><td rowspan=1 colspan=1>Staphylococcus simiae</td><td rowspan=1 colspan=1>DSM 17636</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus kloosii</td><td rowspan=1 colspan=1>ATCC 43959</td><td rowspan=1 colspan=1>Staphylococcus succinus subsp. succinus</td><td rowspan=1 colspan=1>ATCC 700337</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lentus</td><td rowspan=1 colspan=1>ATCC 29070</td><td rowspan=1 colspan=1>Staphylococcus xylosus</td><td rowspan=1 colspan=1>ATCC 35033</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>ATCC 49576</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=4>Species False-Negative for &quot;Other Staphylococci (not S. aureus)&quot; by hemoFISH Masterpanel 1</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>hemoFISH Result</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi subsp. coagulans</td><td rowspan=1 colspan=1>ATCC 49545</td><td rowspan=1 colspan=2>S. aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi subsp. schleiferi</td><td rowspan=1 colspan=1>ATCC 43808</td><td rowspan=1 colspan=2>S. aureus</td></tr></table>

As noted in the Limitations Section of the Package Insert

Table 9. Specificity of hemoFISH Masterpanel for “Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae)”   

<table><tr><td colspan="4" rowspan="1">Speci Correctly Identified as "Other Enterobacteriaceae (not Ecoli or K. pneumoniae) by hemoFH Masterpanel</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Buttiauxella gaviniae</td><td colspan="1" rowspan="1">ATCC 51604</td><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">ATCC 13315</td></tr><tr><td colspan="1" rowspan="1">Cedeceae davisiae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 29245</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 33519</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27156</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 8427</td></tr><tr><td colspan="1" rowspan="1">Cronobacter (Enterobacter) sakazakii</td><td colspan="1" rowspan="1">ATCC 29544</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6896</td></tr><tr><td colspan="1" rowspan="1">Cronobacter muytjensi</td><td colspan="1" rowspan="1">ATCC 51329</td><td colspan="1" rowspan="1">Providencia (Proteus) acalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Providencia (Proteus) rettgeri</td><td colspan="1" rowspan="1">ATCC 9250</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9259</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cancerogenus</td><td colspan="1" rowspan="1">ATCC 33241</td><td colspan="1" rowspan="1">Rahnella aquatilis</td><td colspan="1" rowspan="1">ATCC 33071</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp. cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">ATCC 33531</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp. dissolvens</td><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">ATCC 33257</td></tr><tr><td colspan="1" rowspan="1">Enterobacter gergoviae</td><td colspan="1" rowspan="1">ATCC 33028</td><td colspan="1" rowspan="1">Salmonella abony</td><td colspan="1" rowspan="1">NCTC 6017</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">ATCC 43975</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (Pluribacter) pyrinus</td><td colspan="1" rowspan="1">ATCC 49851</td><td colspan="1" rowspan="1">Salmonella choleraesuis</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="4" rowspan="1">Spec Correctly Identified as "Other Enterobacteriaceae (not E coli or K. pneumoniae" by hemoFIS Masterpanel</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Escherichia (Shimwellia) blattae</td><td colspan="1" rowspan="1">DSM 4481/ATCC 29907</td><td colspan="1" rowspan="1">Salmonella enterica-heidelberg</td><td colspan="1" rowspan="1">ATCC 8326</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermanii</td><td colspan="1" rowspan="1">ATCC 33650</td><td colspan="1" rowspan="1">Salmonella enterica-paratyphi</td><td colspan="1" rowspan="1">ATCC 9150</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">ATCC 25241</td></tr><tr><td colspan="1" rowspan="1">Ewingella americana</td><td colspan="1" rowspan="1">ATCC 33852</td><td colspan="1" rowspan="1">Salmonella vellore</td><td colspan="1" rowspan="1">ATCC 15611</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">ATCC 33433</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43862</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">ATCC 23216</td><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC 183</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25829</td><td colspan="1" rowspan="1">Serratia proteamaculans</td><td colspan="1" rowspan="1">ATCC 19323</td></tr><tr><td colspan="1" rowspan="1">Pantoea (Enterobacter) agglomerans</td><td colspan="1" rowspan="1">ATCC 27155</td><td colspan="1" rowspan="1">Yokenella regensburgei</td><td colspan="1" rowspan="1">ATCC 35313</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Specis False-Negative for "Other Enterobacteriaceae (not E. coli or K. pneumoniae)" by hemoFISH Masterpanel</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="2" rowspan="1">hemoFISH Result</td></tr><tr><td colspan="1" rowspan="1">Tatumella ptyseos</td><td colspan="1" rowspan="1">ATCC 33301</td><td colspan="2" rowspan="1">Other (not Staphylococci, Streptococci or Enterobacteriaceae)</td></tr></table>

As noted in the Limitations Section of the Package Insert

Table 10. Specificity of the hemoFISH Masterpanel for “Streptococci”   

<table><tr><td rowspan=1 colspan=5>Species Correctly Identified as &quot;Streptococci&quot; by hemoFISH Masterpanel</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>Species</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 12403</td><td rowspan=1 colspan=2>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 13770</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 27956</td><td rowspan=1 colspan=2>Streptococcus mutans</td><td rowspan=1 colspan=1>ATCC 25175</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 33397</td><td rowspan=1 colspan=2>Streptococcus oralis</td><td rowspan=1 colspan=1>ATCC 20627</td></tr><tr><td rowspan=1 colspan=1>Streptococcus australis</td><td rowspan=1 colspan=1>ATCC 700641</td><td rowspan=1 colspan=2>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>ATCC 15909</td></tr><tr><td rowspan=1 colspan=1>Streptococcus bovis</td><td rowspan=1 colspan=1>ATCC 33317</td><td rowspan=1 colspan=2>Streptococcus pasteurianus</td><td rowspan=1 colspan=1>ATCC 49133</td></tr><tr><td rowspan=1 colspan=1>Streptococcus constellatus</td><td rowspan=1 colspan=1>ATCC 27823</td><td rowspan=1 colspan=2>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 6305</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>ATCC 12394</td><td rowspan=1 colspan=2>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 10015</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equi</td><td rowspan=1 colspan=1>ATCC 9528</td><td rowspan=1 colspan=2>Streptococcus porcinus</td><td rowspan=1 colspan=1>ATCC 43138</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equinis</td><td rowspan=1 colspan=1>ATCC 9812</td><td rowspan=1 colspan=2>Streptococcus pseudopneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-960</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gallolyticus</td><td rowspan=1 colspan=1>ATCC 9809</td><td rowspan=1 colspan=2>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12344</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gordonii</td><td rowspan=1 colspan=1>ATCC 10558</td><td rowspan=1 colspan=2>Streptococcus sanguinis</td><td rowspan=1 colspan=1>ATCC 10556</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantarius</td><td rowspan=1 colspan=1>ATCC BAA-102</td><td rowspan=1 colspan=2>Streptococcus suis</td><td rowspan=1 colspan=1>ATCC 43765</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantis</td><td rowspan=1 colspan=1>DSM 12492</td><td rowspan=1 colspan=2>Streptococcus urinalis</td><td rowspan=1 colspan=1>DSM 16830</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 6249</td><td rowspan=1 colspan=2>Streptococcus vestibularis</td><td rowspan=1 colspan=1>ATCC 49124</td></tr><tr><td rowspan=1 colspan=3>Species False-Positive for &quot;Streptococci&quot;by hemoFISH Masterpanel 1</td><td rowspan=5 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=2>Strain</td></tr><tr><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=2>ATCC 11454</td></tr><tr><td rowspan=1 colspan=1>Leuconostoc carnosum</td><td rowspan=1 colspan=2>DSM 5576 / ATCC 49367</td></tr><tr><td rowspan=1 colspan=1>Leuconostoc mesenteroides</td><td rowspan=1 colspan=2>DSM 20343 / ATCC 8293</td></tr></table>

As noted in the Limitations Section of the Package Insert

Table 11. Specificity of the hemoFISH Masterpanel for “Other (not Staphylococci, Streptococci or Enterobacteriaceae)”   

<table><tr><td colspan="4" rowspan="1">Species Correctly Identified as "Other (not Staphylococci, Streptococci or Enterobacteriaceae)"by hemoFISH Masterpanel</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCIMB 12457</td><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 10541</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">BAA-747</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baylyi</td><td colspan="1" rowspan="1">DSM 14961</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">DSM 20772 /ATCC 27824</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter bereziniae</td><td colspan="1" rowspan="1">ATCC 17924</td><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">ATCC 43205</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">ATCC 23055</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 43065</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter guillouiae</td><td colspan="1" rowspan="1">ATCC 11171</td><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 10014</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 19002</td><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">DSM 8978 /ATCC 33392</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter johnsonii</td><td colspan="1" rowspan="1">ATCC 17909</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">ATCC 7536</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwofii</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">DSM 20584 /ATC33820</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter radioresistens</td><td colspan="1" rowspan="1">ATCC 43998</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC 33152</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter schindleri</td><td colspan="1" rowspan="1">DSM 16038</td><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">ATCC 35897</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">DSM 16037</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">ATCC 33090</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">ATCC 15468</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 13932</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC 7966</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 4698</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">ATCC 7901</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 25238</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">ATCC 8750</td><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">DSM 17712 /ATCC 25295</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium (Trueperella)bernardiae</td><td colspan="1" rowspan="1">DSM 9152</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td></tr><tr><td colspan="1" rowspan="1">Bacillus atropheus</td><td colspan="1" rowspan="1">ATCC 9372</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13077</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 10872</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 9913</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis</td><td colspan="1" rowspan="1">ATCC 12759</td><td colspan="1" rowspan="1">Pandorea apista</td><td colspan="1" rowspan="1">ATCC 16535</td></tr><tr><td colspan="1" rowspan="1">Bacillus pumilus</td><td colspan="1" rowspan="1">ATCC 14884</td><td colspan="1" rowspan="1">Pandorea pnomenusa</td><td colspan="1" rowspan="1">ATCC 16536</td></tr><tr><td colspan="1" rowspan="1">Bacillus spizizenii</td><td colspan="1" rowspan="1">ATCC 6633</td><td colspan="1" rowspan="1">Parabacteroides distasonis</td><td colspan="1" rowspan="1">ATCC-BAA 1295</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 11774</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">ATCC 27883</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 6051</td><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">ATCC 33316</td></tr><tr><td colspan="1" rowspan="1">Bacteroides uniformis</td><td colspan="1" rowspan="1">ATCC 8492</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Burkholderia multivorans</td><td colspan="1" rowspan="1">ATCC 13243</td><td colspan="1" rowspan="1">Planococcus citreus</td><td colspan="1" rowspan="1">DSM 20549 /ATCC 25536</td></tr><tr><td colspan="1" rowspan="1">Burkholderia vietnamiensis</td><td colspan="1" rowspan="1">ATCC 11319</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">DSM 7089 /ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Buttiauxella gaviniae</td><td colspan="1" rowspan="1">ATCC 51604</td><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">ATCC 11827</td></tr><tr><td colspan="1" rowspan="1">Cardiobacterium hominis</td><td colspan="1" rowspan="1">DSM 8339 /ATCC 15826</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 9027</td></tr><tr><td colspan="1" rowspan="1">Cellumonas turbata (Oerskovia turbata)</td><td colspan="1" rowspan="1">DSM 20577 /ATCC 25835</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">ATCC 9698</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 10145</td></tr><tr><td colspan="1" rowspan="1">Comamonas testosteroni</td><td colspan="1" rowspan="1">DSM 50244 /ATCCC 11996</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">ATCC 23834</td><td colspan="1" rowspan="1">Pseudomonas fulva</td><td colspan="1" rowspan="1">DSM 17717</td></tr><tr><td colspan="1" rowspan="1">Elizabethkingia meningoseptica</td><td colspan="1" rowspan="1">ATCC 13253</td><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">DSM 6975 /ATCC 43273</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">Pseudomonas mendocina</td><td colspan="1" rowspan="1">DSM 50017/ATCC 25411</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 35667</td><td colspan="1" rowspan="1">Pseudomonas mucidolens</td><td colspan="1" rowspan="1">DSM 19186</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 19433</td><td colspan="1" rowspan="1">Pseudomonas nitroreducens</td><td colspan="1" rowspan="1">DSM 14399 /ATCC 33634</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">Pseudomonas pertucinogena</td><td colspan="1" rowspan="1">DSM 18268 /ATCC 190</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavis</td><td colspan="1" rowspan="1">ATCC 25788</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ATCC 49128</td></tr><tr><td colspan="1" rowspan="1">Enterococcus mundtii</td><td colspan="1" rowspan="1">DSM 4338 /ATCC 3186</td><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">ATCC 6939</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pseudoavium</td><td colspan="1" rowspan="1">DSM 5632 /ATCC 49372</td><td colspan="1" rowspan="1">Rothia dentocariosa</td><td colspan="1" rowspan="1">DSM 43762 /ATCC 17931</td></tr><tr><td colspan="1" rowspan="1">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 43076</td><td colspan="1" rowspan="1">Stenotrophomonas maltophila</td><td colspan="1" rowspan="1">ATCC 49130</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">ATCC 14025</td><td colspan="1" rowspan="1">Uruburuella suis</td><td colspan="1" rowspan="1">ATCC 17474</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">ATCC 6056</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="2" rowspan="1">Species False-Positive for "Other (not Staphylococci,Streptococci or Enterobacteriaceae)"by hemoFISH Masterpanel</td><td colspan="2" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Tatumella ptyseos 2</td><td colspan="1" rowspan="1">ATCC 33301</td></tr><tr><td colspan="4" rowspan="1">As noted in the Limitations Section of the Package Insert</td></tr></table>

2 False Negative for “Other Enterobacteriaceae” (refer to Table 9)

Laboratory testing using suspecnsions of different organsims was supported by in silico analysis performed on bacterial 16S and 23rRNA sequences available through databases maintained at the National Center for Biotechnology Information (NCBI, US National Library of Medicine) and ARB/Silva (Max Plank Institute, Bremen).

Organisms whose potential reactivity with the hemoFISH Masterpanel was evaluated exclusively through in silico analysis are listed in Table 12, which shows the expected (taxonomic) and predicted (in silico) hemoFISH result for each species/strain. The potential for false results with Shigella spp. is noted in the device labeling (refer also to Table 13).

Table 12. Summary of in silico analysis of hemoFISH Masterpanel inclusivity/exclusivity   

<table><tr><td colspan="1" rowspan="2">Species/Strain</td><td colspan="2" rowspan="1">hemoFISH Result</td><td colspan="1" rowspan="2">Species/Strain</td><td colspan="2" rowspan="1">hemoFISH Result</td></tr><tr><td colspan="1" rowspan="1">Expected</td><td colspan="1" rowspan="1">In silicoPredicted</td><td colspan="1" rowspan="1">Expected</td><td colspan="1" rowspan="1">In silicoPredicted</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Escherichia coli O157-H7</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td></tr><tr><td colspan="1" rowspan="1">Acinetobactertartarogenes</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Escherichia coli O26-H11</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Escherichia coli O55-H7</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td></tr><tr><td colspan="1" rowspan="1">Aeromonas taiwanensis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Aeromonas veronii</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Haemophilus parasuis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Arcanobacteriumhaemolyticum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Haemophilus somnus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ovatus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Herbaspirillum huttiense</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Bacteroidesthetaiotamicron</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Kluyvera (Enterobacter)intermedius</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Brevundimonas diminuta</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Kluyvera cryocrescens</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Brevundimonasvesicularis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Kluyvera georgiana</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Kytococcus sedentarius</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Leminorella richardii</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Capnocytophagaochracea</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Cellulosimicrobiumcellulans</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumflavescens</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Neisseria perflava</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumgenitalium</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Parabacteroides merdae</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumglutamicum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Pediococcus acidilactici</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Planococcus kocurri</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium renale</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Prevotella bivia</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Prevotella buccae</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumurealyticum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Prevotella denticola</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">Pseudomonas chloraphis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Delftia acidovorans</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Pseudomonas fragi</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enteric group 137</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Pseudomonasoryzihabitans</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus(Lelliottia amnigena)</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Pseudomonaspseudoalcaligenes</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Pseudomonas stutzeri</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enterobacter ludwigii</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Pseudomonas veronii</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enterobacter nimipressuralis(Lelliottia nimipressuralis)</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Enterobacter oryzae(Kosakonia oryzae)</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Serratia entomophila</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli ETEC</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O103-H2</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">ENT</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O104-H4</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">E. coli</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O111-H11</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">E. coli</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O121-H19</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Vagococcus fluvialis</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O124</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Other</td></tr></table>

Expected: Expected result based on species taxonomy In silico predicted: Predicted result based on sequence alignment ENT: Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae) Other: Other (not Staphylococci, Streptococci or Enterobacteriaceae) Fail: Negative test result with Eubacterial probe

Bacterial species for which false-positive or -negative hemoFISH results were predicted by in silico analysis and which also underwent confirmatory laboratory testing are listed in Table 13. Those species for which the potential for false results was confirmed are listed in the device labeling.

Table 13. Summary of in silico-predicted false-positive and false-negative results for the hemoFISH Masterpanel   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>TaxonomicClassification</td><td rowspan=1 colspan=1>hemoFISHIn silico Prediction</td><td rowspan=1 colspan=1>FunctionalhemoFISH Result 1</td></tr><tr><td rowspan=1 colspan=1>Tatumella ptyseos</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=1 colspan=1>Macrococcus caseolyticus</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STR</td></tr><tr><td rowspan=1 colspan=1>Leuconostoc carnosum</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STR</td></tr><tr><td rowspan=1 colspan=1>Leuconostoc mesenteroides</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STR</td></tr><tr><td rowspan=1 colspan=1>Escherichia albertii</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td></tr><tr><td rowspan=1 colspan=1>S. schleiferi subsp. coagulans</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>S. schleiferi subsp. schleiferi</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td></tr><tr><td rowspan=1 colspan=1>S. succinus subsp. succinus 2</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>STA</td></tr></table>

ENT: Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae) Other: Other (not Staphylococci, Streptococci or Enterobacteriaceae) STA: Other Staphylococci (not S. aureus) STR: Streptococci 1 Obtained from functional testing 2 Same mismatches in rRNA target region with the S. aureus Beacon probe as S. schleiferi subsp. schleiferi and subsp. coagulans (both hemoFISH false-positive for S. aureus)

f. Assay cut-off:

Not applicable.

g. Assay Interference:

A study was conducted to determine the ability of the hemoFISH Masterpanel to report the correct results from mixed cultures of two species that are identified in the same microscope field. Testing was performed with BD BACTEC Plus Aerobic/F culture vials containing whole blood that was inoculated with different ratios of $S .$ . aureus $( 1 0 ^ { 3 } – 1 0 ^ { 5 } \mathrm { C F U / m L } )$ ) and $E$ . coli $( 1 0 ^ { 6 } – 1 0 ^ { 8 } \mathrm { C F U / m L } )$ or $S .$ . pneumoniae $( 1 0 ^ { 3 } – 1 0 ^ { 5 }$ $\mathrm { C F U / m L }$ ) and $K .$ . pneumoniae $( 1 0 ^ { 6 } – 1 0 ^ { 9 } \mathrm { C F U / m L } )$ . The blood culture vials were incubated until they turned positive and were then tested with the hemoFISH Masterpanel. All assay results were as expected, indicating that the hemoFISH Masterpanel is capable of correctly identifying mixed cultures of specific organisms that are detected in the same field when the target levels are above the limit of detection of the assay.

Note: Based on the design of the hemoFISH Masterpanel and the performance observed with mixed cultures observed in the Clinical Study (Section M 3), the following specific Limitations are included in the device labeling.

The hemoFISH assay does not differentiate species of Staphylococcus other than Staphylococcus aureus. Therefore, whether a sample contains one or multiple species of Staphylococcus (other than S. aureus), the result will be

reported as “Other staphylococci.” Samples that are reported as positive for $S _ { \cdot }$ . aureus may also contain other Staphyloccocus species. The blood culture bottle should be plated on solid medium to obtain isolated colonies for Staphyloccus species identification and/or susceptibility testing. The hemoFISH assay does not differentiate species of Streptococcus. Therefore, whether a sample contains one or multiple species of Streptococcus, the result will be reported as “Streptococci.” The blood culture bottle should be plated on solid medium to obtain isolated colonies for Streptococcus species identification and/or susceptibility testing. The hemoFISH assay does not differentiate species of Enterobacteriaceae other than $E .$ . coli, and $K .$ . pneumoniae. Therefore, whether a sample contains one or multiple species of Enterobacteriaceae (other than E. coli, or $K$ . pneumoniae), the result will be reported as “Other Enterobacteriaceae.” Samples that are reported as positive for $E .$ . coli, or $K .$ . pneumoniae may also contain other species of Enterobacteriaceae. The blood culture bottle should be plated on solid medium to obtain isolated colonies for species identification and/or susceptibility testing. · The hemoFISH assay may produce false-negative results with mixed cultures that contain one or more organisms below the limit of detection. If a mixed culture is suspected on the basis of Gram stain or if the hemoFISH Masterpanel result is positive for “Other” (not Staphylococci, Streptococci or Enterobacteriaceae), the blood culture bottle should be plated on solid medium to obtain isolated colonies for further identification.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

An analytical study was performed to evaluate the compatibility of the hemoFISH Masterpanel with 13 different types of culture media (Table 14). Testing was performed with blood cultures containing representative isolates of each species, genus or family in the hemoFISH Masterpanel. Bottles were inoculated at low levels and incubated at $3 5 \mathrm { { ^ \circ C } }$ for 20-23 hours prior to testing depending on the culture medium. The expected results were obtained with each of the culture medium in which growth occurred. No growth was obtained when $E$ . coli was inoculated into in VersaTREK REDOX 2 vials, although the ability of the hemoFISH Masterpanel to detect $E$ . coli in this culture medium was demonstrated in the Clinical Study (Section M 3, Table 28).

Table 14. Culture media shown to be compatible with the hemoFISH Masterpanel   

<table><tr><td rowspan=2 colspan=1>Blood CultureSystem</td><td rowspan=1 colspan=2>Culture Medium</td></tr><tr><td rowspan=1 colspan=1>Aerobic</td><td rowspan=1 colspan=1>Anaerobic</td></tr><tr><td rowspan=3 colspan=1>BD BACTEC</td><td rowspan=1 colspan=1>Plus Aerobic/F</td><td rowspan=1 colspan=1>Plus Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>Standard 10 Aerobic/F</td><td rowspan=1 colspan=1>Standard Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>Pediatric PEDS/Plus/F</td><td rowspan=1 colspan=1>Lytic/10 Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 1</td><td rowspan=1 colspan=1>REDOX 2</td></tr><tr><td rowspan=3 colspan=1>BacT/ALERT</td><td rowspan=1 colspan=1>SA Standard Aerobic</td><td rowspan=1 colspan=1>SN Standard Anaerobic</td></tr><tr><td rowspan=1 colspan=1>FA Aerobic FAN</td><td rowspan=1 colspan=1>FN Anaerobic FAN</td></tr><tr><td rowspan=1 colspan=1>PF Pediatric FAN</td><td rowspan=1 colspan=1></td></tr></table>

An additional analytical study was conducted to verify the tolerance of the hemoFISH Masterpanel to differences in the volume of blood inoculated into the culture medium. Testing was performed with a representative panel of organisms grown in BD BACTEC Plus Aerobic/F and Plus Anaerobic/F culture vials containing volumes of blood ranging from 5 to $1 0 \mathrm { m L }$ per vial. All results were as expected, with no difference in performance according to blood volume.

a. Clinical Sensitivity:

The performance of the hemoFISH Masterpanel was evaluated in a prospective Clinical Study that was conducted at three testing sites (two in the US and one exUS). Samples included in the study were residual blood culture vials that were called positive by an automated blood culture system and which were confirmed to contain bacteria by Gram stain.

The prospective study at the clinical sites was supplemented with in-house testing of additional contrived samples prepared by inoculating blood culture vials with whole human blood containing organisms and incubating until “bottle ring.” Table 15 shows the total number of samples included in the study.

Table 15. Sources of positive blood cultures included in determination of performance for the hemoFISH Masterpanel   

<table><tr><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>Total</td><td rowspan=2 colspan=1>Contrived</td><td rowspan=2 colspan=1>Routine</td><td rowspan=2 colspan=1>Culture</td><td rowspan=2 colspan=1>Excluded</td><td rowspan=1 colspan=3>Included in PerformanceDetermination</td></tr><tr><td rowspan=1 colspan=1>Sub-total</td><td rowspan=1 colspan=1>Mono-microbial</td><td rowspan=1 colspan=1>Poly-microbial</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>590</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>590</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>362</td><td rowspan=1 colspan=1>344</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Site C 1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>In-house</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1096</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>1027</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>371</td><td rowspan=1 colspan=1>725</td><td rowspan=1 colspan=1>670</td><td rowspan=1 colspan=1>55</td></tr></table>

Ex-US

The reasons for exclusion of samples from the analysis of performance are summarized in Table 16.

Table 16. Reasons for exclusion of samples from performance calculations   

<table><tr><td rowspan=1 colspan=10>Excluded Monomicrobial Cultures</td></tr><tr><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>Duplicate</td><td rowspan=1 colspan=3>Failure</td><td rowspan=2 colspan=1>ID</td><td rowspan=2 colspan=1>NotTested</td><td rowspan=2 colspan=1>Cont.</td><td rowspan=2 colspan=1>SampleStability</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>Genus /Family Probe</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>In-house</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>325</td></tr><tr><td rowspan=1 colspan=10>Excluded Polymicrobial Cultures</td></tr><tr><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>Duplicate</td><td rowspan=1 colspan=3>Failure</td><td rowspan=2 colspan=1>ID</td><td rowspan=2 colspan=1>NotTested</td><td rowspan=2 colspan=1>Cont.</td><td rowspan=2 colspan=1>SampleStability</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>Genus / FamilyProbe</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>In-house</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td></tr></table>

Duplicate: repeat sample from previously enrolled subject (or repeat culture of the same strain for contrived samples) Negative Control Failure: Green fluorescence in control well Positive Control Failure: Failure of Eubacterial probe; absence of red fluorescence from control well Genus/Family Probe Failure: hemoFISH species probe positive but genus/family probe negative ID: Reference identification not performed or method not FDA-cleared Not Tested: hemoFISH assay not performed Cont.: Sample contaminated Sample Stability: ${ > } 1 2$ hours between bottle ring and hemoFISH testing (protocol deviation) Note: An additional 6 samples at Site C were excluded because the result from the reference identification method was not available (5 samples) or subculture yielded no growth (1 sample)

# Monomicrobial Cultures

The performance of the hemoFISH Masterpanel with prospectively collected monomicrobial samples, contrived monomicrobial samples and both sample types combined is shown in Tables 17 to 22.

Table 17. Performance of the hemoFISH Masterpanel with prospective monomicrobial clinical samples $( \mathrm { n } = 6 0 9 _ { \cdot }$ )   

<table><tr><td colspan="5" rowspan="1">Identification of E. coli</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">91</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">514</td><td colspan="1" rowspan="1">518</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">515</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">90/94 = 95.7% (89.6, 98.3%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">514/515 = 99.8% (98.9, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">90/91 = 98.9%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">514/518 = 99.2%</td></tr><tr><td colspan="5" rowspan="1">Identification of K. pneumoniae</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">573</td><td colspan="1" rowspan="1">573</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">573</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">36/36 = 100% (90.4, 100%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">573/573 = 100% (99.3, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">36/36 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">573/573 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of S. aureus</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">538</td><td colspan="1" rowspan="1">558</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">540</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">69/69 = 100% (94.7, 100%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">538/540 = 99.6% (98.7, 99.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">69/71 = 97.2%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">538/538 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Enterobacteriaceae(species other than E. coli and K. pneumoniae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">566</td><td colspan="1" rowspan="1">566</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">568</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">41/41 = 100% (91.4, 100%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">566/568 = 99.6% (98.7, 99.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">41/43 = 95.3%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">566/566 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Staphylococci(species other than S. aureus)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">160</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">161</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">441</td><td colspan="1" rowspan="1">448</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">442</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">160/167 = 95.8% (91.6, 98.0%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">441/442 = 99.8% (98.7, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">160/161 = 99.4%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">441/448 = 98.4%</td></tr><tr><td colspan="5" rowspan="1">Identification of Streptococci</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">69</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">540</td><td colspan="1" rowspan="1">540</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">542</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">67/67 = 100% (94.6, 100%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">540/542 = 99.6% (98.7, 99.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">67/69 = 97.1%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">540/540 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Species(not Staphylococci, Streptococci or Enterobacteriaceae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">131</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">139</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">466</td><td colspan="1" rowspan="1">470</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">135</td><td colspan="1" rowspan="1">474</td><td colspan="1" rowspan="1">609</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">131/135 = 97.0% (92.6, 98.8%)</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="3" rowspan="1">466/474 = 98.3% (96.7, 99.1%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">131/139 = 94.2%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">466/470 = 99.1%</td></tr><tr><td colspan="5" rowspan="1">Includes 1 sample that was positive by the reference method for yeast. Based on in silico analysis, theepeed hemoFSH result wit cultures that are positive or yeast when o bacteria are present is "Fail"dueto the bsence of signal with the Eubacterial probe. As noted in the Package Insert, cultures identied asposive or "Other" not Staphylococci, Streptococcior Enterobacteriacee) require additional folow-u fordefinitive identification of the species present.</td></tr></table>

Table 18. Summary of performance of the hemoFISH Masterpanel with prospective clinical monomicrobial samples $( \mathrm { n } = 6 0 9 $   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=7>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=7 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>161</td></tr><tr><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>131^2</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>609</td></tr><tr><td rowspan=2 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>90/94</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>160/167</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>131/135</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>95.7%(89.6, 98.3%)</td><td rowspan=1 colspan=1>100%(90.4, 100%)</td><td rowspan=1 colspan=1>100%(94.7, 100%)</td><td rowspan=1 colspan=1>100%(91.4, 100%)</td><td rowspan=1 colspan=1>95.8%(91.6, 98.0%)</td><td rowspan=1 colspan=1>100%(94.6, 100%)</td><td rowspan=1 colspan=1>97.0%(92.6, 98.8%)</td></tr><tr><td rowspan=2 colspan=2>Specificity</td><td rowspan=1 colspan=1>514/515</td><td rowspan=1 colspan=1>573/573</td><td rowspan=1 colspan=1>538/540</td><td rowspan=1 colspan=1>566/568</td><td rowspan=1 colspan=1>441/442</td><td rowspan=1 colspan=1>540/542</td><td rowspan=1 colspan=1>466/474</td></tr><tr><td rowspan=1 colspan=1>99.8%(98.9, 100%)</td><td rowspan=1 colspan=1>100%(99.3, 100%)</td><td rowspan=1 colspan=1>99.6%(98.7, 99.9%)</td><td rowspan=1 colspan=1>99.6%(98.7, 99.9%)</td><td rowspan=1 colspan=1>99.8%(98.7, 100%)</td><td rowspan=1 colspan=1>99.6%(98.7, 99.9%)</td><td rowspan=1 colspan=1>98.3%(96.7, 99.1%)</td></tr><tr><td rowspan=2 colspan=2>PositivePredictive Value</td><td rowspan=1 colspan=1>90/91</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>69/71</td><td rowspan=1 colspan=1>41/43</td><td rowspan=1 colspan=1>160/161</td><td rowspan=1 colspan=1>67/69</td><td rowspan=1 colspan=1>131/139</td></tr><tr><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>95.3%</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>94.2%</td></tr><tr><td rowspan=2 colspan=2>NegativePredictive Value</td><td rowspan=1 colspan=1>514/518</td><td rowspan=1 colspan=1>573/573</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>566/566</td><td rowspan=1 colspan=1>441/448</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>466/470</td></tr><tr><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.1%</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT: Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not $S _ { \cdot }$ . aureus); STR: Streptococci; Other (not Staphylococci, Streptococci or Enterobacteriaceae) 1 1 sample reference positive for EC; hemoFISH positive for EC and STR 2 Includes 1 sample that was positive by the reference method for $C .$ . albicans. Based on in silico analysis, the expected hemoFISH result with blood cultures that are positive for Candida spp. is “Fail” due to the absence of signal from the Eubacterial probe (refer to Table 12)

Table 19. Performance of the hemoFISH Masterpanel with contrived monomicrobial samples $( \mathtt { n } = 6 1$ )   

<table><tr><td colspan="5" rowspan="1">Identification of E. coli</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">59</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">2/2 = 100% (34.2, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">59/59 = 100% (93.9, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">2/2 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">59/59 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of K. pneumoniae</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">63</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">2/2 = 100% (34.2, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">59/59 = 100% (93.9, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">2/2 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">59/59 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of S. aureus</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">56</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">5/5 = 100% (56.6, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">56/56 = 100% (93.6, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">5/5 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">56/56= 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Enterobacteriaceae(species other than E. coli and K. pneumoniae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">21/23 = 91.3% (73.2, 97.6%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">38/38 = 100% (90.8, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">21/21 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">38/40 = 95.0%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Staphylococci(species other than S. aureus)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">54</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">7/7 = 100% (64.6, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">54/54 = 100% (93.4, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">7/7 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">54/54= 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Streptococci</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">52</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">9/9 = 100% (77.1, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">52/52 = 100% (93.1, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">9/9 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">52/52 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Species(not Staphylococci, Streptococci or Enterobacteriaceae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">48</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">61</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">13/13 = 100% (77.2, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">46/48 = 95.8%(86.0, 98.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">13/15 = 86.7%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">46/46 = 100%</td></tr></table>

Table 20. Summary of performance of the hemoFISH Masterpanel with contrived monomicrobial samples $( \mathtt { n } = 6 1$   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=7>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=7 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=2 colspan=2>PositiveAgreement</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>21/23</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>13/13</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>100%(34.2, 100%)</td><td rowspan=1 colspan=1>100%(34.2, 100%)</td><td rowspan=1 colspan=1>100%(56.6, 100%)</td><td rowspan=1 colspan=1>91.3%(73.2, 97.6%)</td><td rowspan=1 colspan=1>100%(64.6, 100%)</td><td rowspan=1 colspan=1>100%(77.1, 100%)</td><td rowspan=1 colspan=1>100%(77.2, 100%)</td></tr><tr><td rowspan=2 colspan=2>NegativeAgreement</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>46/48</td></tr><tr><td rowspan=1 colspan=1>100%(93.9, 100%)</td><td rowspan=1 colspan=1>100%(93.9, 100%)</td><td rowspan=1 colspan=1>100%(93.6, 100%)</td><td rowspan=1 colspan=1>100%(90.8, 100%)</td><td rowspan=1 colspan=1>100%(93.4, 100%)</td><td rowspan=1 colspan=1>100%(93.1, 100%)</td><td rowspan=1 colspan=1>95.8%(86.0, 98.9%)</td></tr><tr><td rowspan=2 colspan=2>PositivePredictive Value</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>13/15</td></tr><tr><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.7%</td></tr><tr><td rowspan=2 colspan=2>NegativePredictive Value</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>46/46</td></tr><tr><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT: Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not $S _ { \cdot }$ . aureus); STR: Streptococci; Other (not Staphylococci, Streptococci or Enterobacteriaceae)

Table 21. Overall performance of the hemoFISH Masterpanel with prospective and contrived monomicrobial samples combined $( \mathrm { n } = 6 7 0 )$ )   

<table><tr><td colspan="5" rowspan="1">Identification of E. coli</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">573</td><td colspan="1" rowspan="1">577</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">574</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">92/96 = 95.8% (89.8, 98.4%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">573/574 = 99.8% (99.0, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">92/93 = 98.9%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">573/577 = 99.3%</td></tr><tr><td colspan="5" rowspan="1">Identification of K. pneumoniae</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">632</td><td colspan="1" rowspan="1">632</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">632</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">38/38 = 100% (90.8, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">632/632 = 100% (99.4, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">38/38 = 100%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">632/632 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of S. aureus</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">76</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">594</td><td colspan="1" rowspan="1">594</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">596</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">74/74 = 100% (95.1, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">594/596 = 99.7% (98.8, 99.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">74/76 = 97.4%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">594/594 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Enterobacteriaceae(species other than E. coli and K. pneumoniae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">64</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">604</td><td colspan="1" rowspan="1">606</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">606</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">62/64 = 96.9% (89.3, 99.1%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">604/606 = 99.7% (98.8, 99.1%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">62/64 = 97.0%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">604/606 = 99.7%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Staphylococci(species other than S. aureus)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">168</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">495</td><td colspan="1" rowspan="1">502</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">496</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">167/174 = 96.0% (91.9, 98.0%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">495/496 = 99.8% (98.9, 100%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">167/168 = 99.4%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">495/502 = 98.6%</td></tr><tr><td colspan="5" rowspan="1">Identification of Streptococci</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">78</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">592</td><td colspan="1" rowspan="1">592</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">594</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">76/76 = 100% (95.2, 100%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">592/594 = 99.7% (98.8, 99.9%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">76/78 = 97.4%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">592/592 = 100%</td></tr><tr><td colspan="5" rowspan="1">Identification of Other Species(not Staphylococci, Streptococci or Enterobacteriaceae)</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">hemoFISHMasterpanel</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">1441</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">154</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">512</td><td colspan="1" rowspan="1">516</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">148</td><td colspan="1" rowspan="1">522</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">144/148 = 97.3% (93.3, 98.9%)</td></tr><tr><td colspan="2" rowspan="1">Negative Percent Agreement</td><td colspan="3" rowspan="1">512/522 = 98.1% (96.5, 99.0%)</td></tr><tr><td colspan="2" rowspan="1">Positive Predictive Value</td><td colspan="3" rowspan="1">144/154 = 93.5%</td></tr><tr><td colspan="2" rowspan="1">Negative Predictive Value</td><td colspan="3" rowspan="1">512/516 = 99.2%</td></tr><tr><td colspan="5" rowspan="1">I  hat was posi y heeeo or yeasBas   analy, he  u      l wiheubcl probes otehe ack nser ulue en positiv r "the" present.</td></tr></table>

# Note:

Based on analysis of the prospectively clinical samples and contrived monomicrobial samples combined (Table 21):

1) The lower $9 5 \%$ confidence interval for identification of “Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae)” was $8 9 . 3 \%$ , which is below the acceptance criterion of $2 9 0 \%$ . However, with the prospective samples alone, the point estimate and lower $9 5 \%$ confidence interval for identification of “Other Enterobacteriaceae” were $100 \%$ and $9 1 . 4 \%$ , respectively (Table 18). In addition, according to the design of the hemoFISH Masterpanel, samples that are negative for “Other Enterobacteriaceae” and not positive for any of the other individual species or genera identified by the panel are reported as positive for “Other (not Staphylococci, Streptococci or Enterobacteriaceae).” A Limitation in the Package

Insert stipulates that samples reported positive for “Other (not Staphylococci, Streptococci or Enterobacteriaceae)” should be subcultured for further identification. The risk of reporting a false-negative result for “Other Enterobacteriaceae” is therefore mitigated.

2) The lower $9 5 \%$ confidence interval for identification of $E$ . coli was $8 9 . 8 \%$ . With rounding to the nearest integer, this meets the acceptance criterion of a lower $9 5 \%$ confidence interval $2 9 0 \%$ . All 4 samples that were reported by the hemoFISH Masterpanel as false-negative for $E$ . coli were from the ex-US clinical site. Three of the 4 were reported as positive for “Other (not Staphylococci, Streptococci or Enterobacteriaceae)” and 1 was reported as positive for Other Enterobacteriaceae (not E.coli or $K .$ . pneumoniae).”

Table 22. Summary of performance of the hemoFISH Masterpanel with prospective clinical and contrived samples combined $( \mathtt { n } = 6 7 0 )$   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=7>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=7 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>76</td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>154</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>670</td></tr><tr><td rowspan=2 colspan=2>PositiveAgreement</td><td rowspan=1 colspan=1>92/96</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>62/64</td><td rowspan=1 colspan=1>167/174</td><td rowspan=1 colspan=1>76/76</td><td rowspan=1 colspan=1>144/148</td><td rowspan=9 colspan=1></td></tr><tr><td rowspan=1 colspan=1>95.8%(89.8, 98.4%)</td><td rowspan=1 colspan=1>100%(90.8, 100%)</td><td rowspan=1 colspan=1>100%(95.1, 100%)</td><td rowspan=1 colspan=1>96.9%(89.3, 99.1%)</td><td rowspan=1 colspan=1>96.0%(91.9, 98.0%)</td><td rowspan=1 colspan=1>100%(95.2, 100%)</td><td rowspan=1 colspan=1>97.3%(93.3, 98.9%)</td></tr><tr><td rowspan=3 colspan=2>NegativeAgreement</td><td rowspan=1 colspan=1>573/574</td><td rowspan=1 colspan=1>632/632</td><td rowspan=1 colspan=1>594/596</td><td rowspan=1 colspan=1>604/606</td><td rowspan=1 colspan=1>495/496</td><td rowspan=1 colspan=1>592/594</td><td rowspan=1 colspan=1>512/522</td></tr><tr><td rowspan=2 colspan=1>99.8%(99.0, 100%)</td><td rowspan=2 colspan=1>100%(99.4, 100%)</td><td rowspan=2 colspan=1>99.7%(98.8, 99.9%)</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=2 colspan=1>99.8%(98.9, 100%)</td><td rowspan=2 colspan=1>99.7%(98.8, 99.9%)</td><td rowspan=2 colspan=1>98.1%(96.5, 99.0%)</td></tr><tr><td rowspan=1 colspan=1>(98.8, 99.1%)</td></tr><tr><td rowspan=2 colspan=2>PositivePredictive Value</td><td rowspan=1 colspan=1>92/93</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>74/76</td><td rowspan=1 colspan=1>62/64</td><td rowspan=1 colspan=1>167/168</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>144/154</td></tr><tr><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>97.0%</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>93.5%</td></tr><tr><td rowspan=2 colspan=2>NegativePredictive Value</td><td rowspan=1 colspan=1>573/577</td><td rowspan=1 colspan=1>632/632</td><td rowspan=1 colspan=1>594/594</td><td rowspan=1 colspan=1>604/606</td><td rowspan=1 colspan=1>495/502</td><td rowspan=1 colspan=1>592/592</td><td rowspan=1 colspan=1>512/516</td></tr><tr><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2%</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT: Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not $S _ { \cdot }$ . aureus); STR: Streptococci; Othe (not Staphylococci, Streptococci or Enterobacteriaceae) 1 1 sample reference positive for EC; hemoFISH positive for EC and STR 2 Includes 1 sample that was positive by the reference method for $C .$ . albicans. Based on in silico analysis, the expected hemoFISH result with blood cultures that are positive for Candida spp. is “Fail” due to the absence of signal from the Eubacterial probe (refer to Table 12)

The performance of the individual probes within the hemoFISH Masterpanel with monomicrobial samples (prospective clinical and contrived, combined) is summarized in Tables 23 to 26.

Table 23. Performance of individual probes within the hemoFISH Masterpanel for detection of Enterobacteriaceae in monomicrobial samples $( \mathrm { n } = 6 7 0 ^ { \circ }$ )   

<table><tr><td rowspan=1 colspan=3>Reference Identification Result</td><td rowspan=1 colspan=8>hemoFISH Result</td></tr><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>% Agmt</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>amalonaticus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>braakii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>farmeri</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>gillenii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>koseri</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>murliniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>rodentium</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>werkmanii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>youngae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Cronobacter</td><td rowspan=1 colspan=1>muytjensi</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>cloacae</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>coli</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.8</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>oxytoca</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>pneumoniae</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Morganella</td><td rowspan=1 colspan=1>morganii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>mirabilis</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>vulgaris</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>Providencia</td><td rowspan=1 colspan=1>stuartii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>abony</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>enterica</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Group B</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>typhimurium</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>vellore</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>marcescens</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>enterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>97.0</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae); STA: Other Staphylococci (not S. aureus); STR: Streptococci; OTHER: Other (not Staphylococci, Streptococci or Enterobacteriaceae); $\%$ Agmt: Percent Agreement 1 Sample also positive by hemoFISH assay for $E$ . coli

Table 24. Performance of individual probes within the hemoFISH Masterpanel for detection of Staphylococcus spp. in monomicrobial samples $( \mathtt { n } = 6 7 0 )$ )   

<table><tr><td rowspan=1 colspan=2>Reference Identification Result</td><td rowspan=1 colspan=8>hemoFISH Result</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>% Agmt</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. auricularis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. capitis</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>93.8</td></tr><tr><td rowspan=1 colspan=1>S. caprae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. carnosus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.9</td></tr><tr><td rowspan=1 colspan=1>S. gallinarum</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.1</td></tr><tr><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. pettenkoferi</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. saccharolyticus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. simulans</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. warneri</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. xylosus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>97.2</td></tr></table>

EC: E. coli; KP: $K .$ pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae); STA: Other Staphylococci (not S. aureus); STR: Streptococci: OTHER: Other (not Staphylococci, Streptococci or Enterobacteriaceae); $\%$ Agmt: Percent Agreement

Table 25. Performance of the hemoFISH Masterpanel for detection of Streptococcus spp. in monomicrobial samples $( \mathrm { n } = 6 7 0 )$ )   

<table><tr><td colspan="2" rowspan="1">Reference Identification Result</td><td colspan="8" rowspan="1">hemoFISH Result</td></tr><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">KP</td><td colspan="1" rowspan="1">SA</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">STA</td><td colspan="1" rowspan="1">STR</td><td colspan="1" rowspan="1">OTHER</td><td colspan="1" rowspan="1">% Agmt</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. alactolyticus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. bovis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. gallolyticus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. gordonii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Streptococcus Group G</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. intermedius</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. lutentiensis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. mitis</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. mutans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. oralis</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. salivarius</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. sanguinis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. thermophilus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S. vestibularis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not S. aureus); STR: Streptococci: OTHER: Other (not Staphylococci, Streptococci or Enterobacteriaceae); $\%$ Agmt: Percent Agreement

Table 26. Performance of the hemoFISH Masterpanel for detection of bacteria other than Staphylococcus spp., Streptococcus spp. or Enterobacteriaceae in monomicrobial samples $( \mathrm { n } = 6 7 0 )$   

<table><tr><td colspan="3" rowspan="1">Reference Identification Result</td><td colspan="8" rowspan="1">hemoFISH Result</td></tr><tr><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">KP</td><td colspan="1" rowspan="1">SA</td><td colspan="1" rowspan="1">ENT</td><td colspan="1" rowspan="1">STA</td><td colspan="1" rowspan="1">STR</td><td colspan="1" rowspan="1">OTHER</td><td colspan="1" rowspan="1">% Agmt</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia</td><td colspan="1" rowspan="1">defectiva</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">hemolyticus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">junii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">lwoffi</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">p.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Actinomyces</td><td colspan="1" rowspan="1">meyeri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Actinomyces</td><td colspan="1" rowspan="1">naeslundi</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Aerococcus</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">cereus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">circulans</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">coagulans</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">megaterium</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">subtilis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">thuringiensis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">caccae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">distasonis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">fragilis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">ovatus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">thetaiotaomicron</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Bacteroides</td><td colspan="1" rowspan="1">vulgatus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Candida</td><td colspan="1" rowspan="1">albicans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">bifermentas</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">amycolatum</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">matruchoti</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">xerosis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Elizabethkingia</td><td colspan="1" rowspan="1">meningoseptica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">96.3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">faecium</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">gallinarum</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">raffinosus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium</td><td colspan="1" rowspan="1">nucleatum</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Granulicatella</td><td colspan="1" rowspan="1">adjacens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Kocuria</td><td colspan="1" rowspan="1">kristinae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Kocuria</td><td colspan="1" rowspan="1">varians</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Kroppenstedtia</td><td colspan="1" rowspan="1">eburnea</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">luteus</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">80.0</td></tr><tr><td colspan="1" rowspan="1">Moraxella</td><td colspan="1" rowspan="1">catarrhalis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Pasteurella</td><td colspan="1" rowspan="1">multocida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Pediococcus</td><td colspan="1" rowspan="1">pentosaceus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus</td><td colspan="1" rowspan="1">asaccharolyticus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Prevotella</td><td colspan="1" rowspan="1">disiens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium</td><td colspan="1" rowspan="1">acnes</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium</td><td colspan="1" rowspan="1">sp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Rotha</td><td colspan="1" rowspan="1">dentocariosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas</td><td colspan="1" rowspan="1">acidominophila</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas</td><td colspan="1" rowspan="1">maltophilia</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cYanmaGaellowb</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">152</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">148</td><td colspan="1" rowspan="1">96.7</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae); STA: Other Staphylococci; STR: Streptococcus: OTHER: Other (not Staphylococci, Streptococci or Enterobacteriaceae); $\%$ Agmt: Percent Agreement 1 False-positive with Eubacterial probe

A summary of the performance of each probe in the hemoFISH Masterpanel with prospective clinical and contrived monomicrobial samples combined is shown in Table 27.

Table 27. Performance of hemoFISH Masterpanel Beacon probes with monomicrobial samples (prospective clinical and contrived, combined)   

<table><tr><td rowspan="2"></td><td colspan="6">hemoFISH Probe</td></tr><tr><td>EC</td><td>KP</td><td>SA</td><td>ENT</td><td>STA</td><td>STR</td></tr><tr><td>True Positive</td><td>92</td><td>38</td><td>74</td><td>193</td><td>243</td><td>76</td></tr><tr><td>False Positive</td><td>1</td><td>0</td><td>2</td><td>2</td><td>1</td><td>2</td></tr><tr><td>False Negative</td><td>4</td><td>0</td><td>0</td><td>5</td><td>5</td><td>0</td></tr><tr><td>True Negative</td><td>573</td><td>632</td><td>594</td><td>470</td><td>421</td><td>592</td></tr><tr><td>Total</td><td>670</td><td>670</td><td>670</td><td>670</td><td>670</td><td>670</td></tr><tr><td rowspan="2">Positive Agreement (95% CI)</td><td>92/96</td><td>38/38</td><td>74/74</td><td>193/198</td><td>243/248</td><td>76/76</td></tr><tr><td>95.8% (89.8, 98.4%)</td><td>100% (90.8, 100%)</td><td>100% (95.1, 100%)</td><td>97.5% (94.4, 98.9%)</td><td>98.0%</td><td>100%</td></tr><tr><td rowspan="2">Negative Agreement (95% CI)</td><td>573/574</td><td>632/632</td><td>594/596</td><td>470/472</td><td>(95.4, 99.1%) 421/422</td><td>(95.2-100%) 592/594</td></tr><tr><td>99.8% (99.0, 100%)</td><td>100% (99.4, 100%)</td><td>99.7% (98.8, 100%)</td><td>99.6% (98.5-99.9%)</td><td>99.8% (98.7, 100%)</td><td>99.7% (98.8, 99.9%)</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT; Enterobacteriaceae; STA: Staphylococcus; STR: Streptococcus; CI: Confidence Interval

The Clinical Study was conducted using a combination of different blood culture media according to each site’s standard laboratory practice. The performance of the hemoFISH Masterpanel with monomicrobial cultures in each type of medium is summarized in Table 28.

Table 28. Comparison of results with monomicrobial cultures in different media used in the hemoFISH Masterpanel Clinical Study   

<table><tr><td rowspan=2 colspan=1>Culture Medium</td><td rowspan=2 colspan=1>Number ofSamples</td><td rowspan=2 colspan=1>Agreement</td><td rowspan=1 colspan=7>hemoFISH Result (%)</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=3 colspan=1>VersaTREK REDOX 1</td><td rowspan=3 colspan=1>370</td><td rowspan=2 colspan=1>Positive</td><td rowspan=2 colspan=1>56/56(100)</td><td rowspan=2 colspan=1>29/29(100)</td><td rowspan=2 colspan=1>36/36(100)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87/88</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>79/82(96.3)</td></tr><tr><td rowspan=1 colspan=1>(100)</td><td rowspan=1 colspan=1>(98.9)</td><td rowspan=1 colspan=1>(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>314/314(100)</td><td rowspan=1 colspan=1>341/341(100)</td><td rowspan=1 colspan=1>333/334(99.7)</td><td rowspan=1 colspan=1>339/340(99.7)</td><td rowspan=1 colspan=1>281/282(99.6)</td><td rowspan=1 colspan=1>321/322(99.7)</td><td rowspan=1 colspan=1>287/288(99.7)</td></tr><tr><td rowspan=2 colspan=1>VersaTREK REDOX 2</td><td rowspan=2 colspan=1>173</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23/23(100)</td><td rowspan=1 colspan=1>5/5(100)</td><td rowspan=1 colspan=1>26/26(100)</td><td rowspan=1 colspan=1>7/7(100)</td><td rowspan=1 colspan=1>50/50(100)</td><td rowspan=1 colspan=1>19/19(100)</td><td rowspan=1 colspan=1>41/42(97.6)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>149/150(99.3)</td><td rowspan=1 colspan=1>168/168(100)</td><td rowspan=1 colspan=1>147/147(100)</td><td rowspan=1 colspan=1>166/166(100)</td><td rowspan=1 colspan=1>123/123(100)</td><td rowspan=1 colspan=1>154/154(100)</td><td rowspan=1 colspan=1>131/131(100)</td></tr><tr><td rowspan=2 colspan=1>BD BACTEC Plus Aerobic/F</td><td rowspan=2 colspan=1>1041</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>10/13(76.9)2$</td><td rowspan=1 colspan=1>3/3(100)</td><td rowspan=1 colspan=1>11/11(100)</td><td rowspan=1 colspan=1>23/25(92.0)</td><td rowspan=1 colspan=1>18/22(81.8)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>21/21(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>91/91(100)</td><td rowspan=1 colspan=1>101/101(100)</td><td rowspan=1 colspan=1>92/93(98.9)</td><td rowspan=1 colspan=1>79/79(100)</td><td rowspan=1 colspan=1>82/82(100)</td><td rowspan=1 colspan=1>95/95(100)</td><td rowspan=1 colspan=1>75/83(90.4)</td></tr><tr><td rowspan=2 colspan=1>BD BACTEC Plus Anaerobic/F</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2/3(66.7)</td><td rowspan=1 colspan=1>1/1(100)</td><td rowspan=1 colspan=1>1/1(100)</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>10/12(83.3)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>1/1(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>117/17(100)</td><td rowspan=1 colspan=1>19/19(100)</td><td rowspan=1 colspan=1>19/19(100)</td><td rowspan=1 colspan=1>17/18(94.4)</td><td rowspan=1 colspan=1>8/8(100)</td><td rowspan=1 colspan=1>20/20(100)</td><td rowspan=1 colspan=1>17/19(89.5)</td></tr><tr><td rowspan=2 colspan=1>BD BACTEC Peds Plus/F</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>1/1(100)</td><td rowspan=1 colspan=1>0/0(N/A)</td><td rowspan=1 colspan=1>1/1(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>1/1(100)</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>1/1(100)</td></tr></table>

EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT: Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not S. aureus); STR: Streptococci; Other: Other (not Staphylococci, Streptococci or Enterobacteriaceae); N/A: Not Applicable 1 Includes 61 contrived samples 2 All 3 false-negatiive results for $E$ . coli were obtained at a single ex-US clinical site

Note: The total number of samples included in the table is 670. There was on (1) monomicrobial sample from the Clinical Study for which no information was available regarding the type of medium used.

# Polymicrobial Cultures

The performance of the hemoFISH Master panel for identification of organisms in polymicrobial blood cultures is summarized in Tables 29 and 30. Results from 55 prospectively collected polymicrobial samples were included in the analysis. The hemoFISH Masterpanel correctly identified each of the organisms present in only 5/55 $( 9 . 1 \% )$ polymicrobial samples, all of which contained 2 bacterial species. For 49/50 $( 9 8 . 0 \% )$ of the remaining samples, the hemoFISH Masterpanel correctly identified at least one of the organisms present.

The performance of the hemoFISH Masterpanel with mixed cultures in part reflects the inability to discriminate between co-infecting species of the same genus (for Staphylococci and Streptococci) or family (Enterobacteriaceae). As noted in Section M 1(g), Limitations with respect to correct identification of organisms in polymicrobial cultures are included in the device labeling.

Table 29. Summary of hemoFISH results from 55 prospectively collected polymicrobial samples   

<table><tr><td rowspan=2 colspan=5>Reference Result</td><td rowspan=2 colspan=2>Number ofSamples</td><td rowspan=1 colspan=12>hemoFISH Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ID-1</td><td rowspan=1 colspan=1>ID-2</td><td rowspan=1 colspan=1>ID-3</td><td rowspan=1 colspan=1>ID-4</td><td rowspan=1 colspan=1>ID-5</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>OTHER</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 *</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>YEAST</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>OTHER</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr></table>

ID: Identification; EC: E. coli; KP: K. pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K$ . pneumoniae); STA: Other Staphylococci (not $S _ { \cdot }$ . aureus); STR: Streptococci; Other (Other (not Staphylococci, Streptococci or Enterobacteriaceae) Samples for which the hemoFISH assay agreed with the reference method are shaded Samples for which at least one organism identified by hemoFISH matched at least one of those identified by the reference method are shown in bold italics The one sample for which there was no agreement between the hemoFISH assay and reference method is denoted by \*

Table 30. Summary of hemoFISH results from polymicrobial samples by analyte   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=7>Reference Result</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>Other</td></tr><tr><td rowspan=8 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>KP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ENT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>STA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>STR</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>201</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>30</td></tr></table>

EC: $E$ . coli; KP: $K .$ pneumoniae; SA: S. aureus; ENT; Other Enterobacteriaceae (not $E$ . coli or $K .$ . pneumoniae); STA: Staphylococci (not $S .$ . aureus); STR: Streptococci; Other: Other (not Staphylococci, Streptococci or Enterobacteriaceae); Negative: hemoFISH negative for the species or group identified by the reference method $^ 1 2$ samples that were positive by the reference method for “Other (no Staphylococci, Streptococci or Enterobacteriaceae) were hemoFISH false-positive for $E$ . coli

b. Clinical specificity: Refer to Section M 3(a), above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The clinical study included 664 prospectively collected monomicrobial and polymicrobial samples from three (3) sites. The prevalence at each clinical site for each analyte as determined by the reference culture method and by the hemoFISH Masterpanel is shown in Table 31. The prevelance of each analyte in monomicrobial and polymicrobial cultures is shown in Table 32.

Table 31. Prevalence of hemoFISH analytes in prospective blood cultures stratified by site   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=8>Percent Prevalence (n)</td></tr><tr><td rowspan=1 colspan=2>Site A(332)</td><td rowspan=1 colspan=2>Site B(232)</td><td rowspan=1 colspan=2>Site C(70)</td><td rowspan=1 colspan=2>Overall(664)</td></tr><tr><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>14.6(53)</td><td rowspan=1 colspan=1>14.4(52)</td><td rowspan=1 colspan=1>17.2(40)</td><td rowspan=1 colspan=1>17.7(41)</td><td rowspan=1 colspan=1>22.9(16)</td><td rowspan=1 colspan=1>18.6(13)</td><td rowspan=1 colspan=1>16.4(109)</td><td rowspan=1 colspan=1>16.0(106)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>6.4(23)</td><td rowspan=1 colspan=1>6.4(23)</td><td rowspan=1 colspan=1>9.9(23)</td><td rowspan=1 colspan=1>9.5(22)</td><td rowspan=1 colspan=1>2.9(2)</td><td rowspan=1 colspan=1>2.9(2)</td><td rowspan=1 colspan=1>7.2(48)</td><td rowspan=1 colspan=1>7.1(47)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>14.1(51)</td><td rowspan=1 colspan=1>14.1(51)</td><td rowspan=1 colspan=1>6.0(14)</td><td rowspan=1 colspan=1>6.0(14)</td><td rowspan=1 colspan=1>10.0(7)</td><td rowspan=1 colspan=1>11.4(8)</td><td rowspan=1 colspan=1>10.8(72)</td><td rowspan=1 colspan=1>11.0(73)</td></tr><tr><td rowspan=1 colspan=1>OtherEnterobacteriaceae</td><td rowspan=1 colspan=1>6.1(22)</td><td rowspan=1 colspan=1>5.5(20)</td><td rowspan=1 colspan=1>11.6(27)</td><td rowspan=1 colspan=1>9.5(22)</td><td rowspan=1 colspan=1>8.6(6)</td><td rowspan=1 colspan=1>7.1(5)</td><td rowspan=1 colspan=1>8.3(55)</td><td rowspan=1 colspan=1>7.1(47)</td></tr><tr><td rowspan=2 colspan=1>Other Staphylococci</td><td rowspan=2 colspan=1>27.1(98)</td><td rowspan=2 colspan=1>27.1(98)</td><td rowspan=2 colspan=1>25.4(59)</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>44.3</td><td rowspan=2 colspan=1>34.3(24)</td><td rowspan=2 colspan=1>28.3(188)</td><td rowspan=2 colspan=1>26.4(175)</td></tr><tr><td rowspan=1 colspan=1>(53)</td><td rowspan=1 colspan=1>(31)</td></tr><tr><td rowspan=1 colspan=1>Streptococci</td><td rowspan=1 colspan=1>9.7(35)</td><td rowspan=1 colspan=1>9.9(36)</td><td rowspan=1 colspan=1>16.4(38)</td><td rowspan=1 colspan=1>16.8(39)</td><td rowspan=1 colspan=1>2.9(2)</td><td rowspan=1 colspan=1>1.4(1)</td><td rowspan=1 colspan=1>11.3(75)</td><td rowspan=1 colspan=1>11.4(76)</td></tr><tr><td rowspan=1 colspan=1>Other 3</td><td rowspan=1 colspan=1>25.4(92)</td><td rowspan=1 colspan=1>23.5(85)</td><td rowspan=1 colspan=1>25.9(60)</td><td rowspan=1 colspan=1>19.0(44)</td><td rowspan=1 colspan=1>18.6(13)</td><td rowspan=1 colspan=1>25.7(18)</td><td rowspan=1 colspan=1>24.8(165)</td><td rowspan=1 colspan=1>22.1(147)</td></tr></table>

1 Not E. coli or K. pneumoniae 2 Not S. aureus 3 Not Staphylococci, Streptococci or Enterobacteriaceae

Table 32. Prevalence of hemoFISH analytes in prospective monomicrobial and polymicrobial blood cultures   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=6>Percent Prevalence (n)</td></tr><tr><td rowspan=1 colspan=2>Monomicrobial(609)</td><td rowspan=1 colspan=2>Polymicrobial(55)</td><td rowspan=1 colspan=2>Overall(664)</td></tr><tr><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>hemoFISH</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>15.4(94)</td><td rowspan=1 colspan=1>14.9(91)</td><td rowspan=1 colspan=1>27.3(15)</td><td rowspan=1 colspan=1>27.3(15)</td><td rowspan=1 colspan=1>16.4(109)</td><td rowspan=1 colspan=1>16.0(106)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>5.9(36)</td><td rowspan=1 colspan=1>5.9(36)</td><td rowspan=1 colspan=1>21.8(12)</td><td rowspan=1 colspan=1>20.0(11)</td><td rowspan=1 colspan=1>7.2(48)</td><td rowspan=1 colspan=1>7.1(47)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>11.3(69)</td><td rowspan=1 colspan=1>11.7(71)</td><td rowspan=1 colspan=1>5.5(3)</td><td rowspan=1 colspan=1>3.6(2)</td><td rowspan=1 colspan=1>10.8(72)</td><td rowspan=1 colspan=1>11.0(73)</td></tr><tr><td rowspan=1 colspan=1>OtherEnterobacteriaceae</td><td rowspan=1 colspan=1>6.7(41)</td><td rowspan=1 colspan=1>7.1(43)</td><td rowspan=1 colspan=1>25.5(14)</td><td rowspan=1 colspan=1>7.3(④)</td><td rowspan=1 colspan=1>8.3(55)</td><td rowspan=1 colspan=1>7.1(47)</td></tr><tr><td rowspan=2 colspan=1>Other Staphylococci 2</td><td rowspan=2 colspan=1>27.4(167)</td><td rowspan=2 colspan=1>26.4(161)</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>28.3</td><td rowspan=2 colspan=1>26.4(175)</td></tr><tr><td rowspan=1 colspan=1>(21)</td><td rowspan=1 colspan=1>(14)</td><td rowspan=1 colspan=1>(188)</td></tr><tr><td rowspan=2 colspan=1>Streptococci</td><td rowspan=2 colspan=1>11.0(67)</td><td rowspan=2 colspan=1>11.3(69)</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>11.3</td><td rowspan=2 colspan=1>11.4(76)</td></tr><tr><td rowspan=1 colspan=1>(8)</td><td rowspan=1 colspan=1>(7)</td><td rowspan=1 colspan=1>(75)</td></tr><tr><td rowspan=1 colspan=1>Other 3</td><td rowspan=1 colspan=1>22.2(135)</td><td rowspan=1 colspan=1>22.8(139)</td><td rowspan=1 colspan=1>54.5(30)</td><td rowspan=1 colspan=1>14.5(8)</td><td rowspan=1 colspan=1>24.8(165)</td><td rowspan=1 colspan=1>22.1(147)</td></tr></table>

1 Not E. coli or $K .$ pneumoniae 2 Not S. aureus 3 Not Staphylococci, Streptococci or Enterobacteriaceae

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted in this premarket notification is complete and supports a substantial equivalence decision.