ROCHE MOLECULAR SYSTEMS, INC.   
DAVID W. GATES, PH.D.   
SENIOR DIRECTOR, REGULATORY AFFAIRS 4300 HACIENDA DRIVE   
PLEASANTON, CA 94588-2722

Re: K150617 Trade/Device Name: cobas® HSV 1 and 2 Test Regulation Number: 21 CFR 866.3305 Regulation Name: Herpes Simplex Virus Nucleic Acid Amplification Assay Regulatory Class: II Product Code: OQO Dated: March 9, 2015 Received: March 10, 2015

Dear Dr. Gates:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA’s issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act’s requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000-1050.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801), please contact the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, sMisbranding by reference to premarket notifications (21CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH’s Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

Stephen J. Lovell -S for

Sally A. Hojvat, M. Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

510(k) Number (if known) K150617

Device Name cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test

Indications for Use (Describe)

The cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test on the cobas $^ \mathrm { \textregistered }$ 4800 system is an automated, qualitative in vitro diagnostic test, that utilizes real-time polymerase chain reaction (PCR), for the direct detection and differentiation of Herpes simplex virus 1 and 2 (HSV-1 and HSV-2) DNA in clinician-collected, external anogenital lesion specimens from symptomatic male and female patients. The cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test is intended for use as an aid in diagnosis of anogenital HSV-1 and HSV-2 infections in symptomatic patients.

Box warning:

Warning: The cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test is not FDA cleared for use with cerebrospinal fluid (CSF) and is not intended to be used for prenatal screening or for individuals under the age of 18 years.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# cobas® HSV 1 and 2 Test 510(k) Summary

<table><tr><td rowspan=1 colspan=1>Submitter Name</td><td rowspan=1 colspan=1>Roche Molecular Systems, Inc.</td></tr><tr><td rowspan=1 colspan=1>Address</td><td rowspan=1 colspan=1>4300 Hacienda DrivePleasanton, CA 94588-2722</td></tr><tr><td rowspan=1 colspan=1>Contact</td><td rowspan=1 colspan=1>David GatesPhone: 925.730-8237</td></tr><tr><td rowspan=1 colspan=1>Date Prepared</td><td rowspan=1 colspan=1>February 27, 2015</td></tr><tr><td rowspan=1 colspan=1>Proprietary Name</td><td rowspan=1 colspan=1>cobas® HSV 1 and 2 Test</td></tr><tr><td rowspan=1 colspan=1>Common Name</td><td rowspan=1 colspan=1>HSV 1 and 2 Test</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>21 CFR 866.3305</td></tr><tr><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Class II</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>OQO: Herpes Simplex Virus Nucleic Acid Amplification Assay</td></tr><tr><td rowspan=1 colspan=1>Predicate Devices</td><td rowspan=1 colspan=1>BD ProbeTecTM Herpes Simplex Viruses (HSV 1 &amp; 2) QxAmplified DNA Assays (K103798)</td></tr><tr><td rowspan=1 colspan=1>EstablishmentRegistration</td><td rowspan=1 colspan=1>Branchburg: 2243471Pleasanton: 3004141078Indianapolis: 1823260</td></tr></table>

# 1. DEVICE DESCRIPTION

The Roche Molecular Systems (RMS) cobas® HSV 1 and 2 Test utilizes real-time polymerase chain reaction (PCR) for detection of HSV-1 and HSV-2 DNA in clinician-collected external anogenital lesion specimens, collected in MSwab medium from symptomatic patients.

The cobas® HSV 1 and HSV 2 Test contains two major processes: (1) automated sample preparation to extract nucleic acids from swab specimens; (2) PCR amplification of target DNA sequences using HSV-1 and HSV-2 specific primers, and real-time detection of cleaved fluorescent-labeled HSV-1 and HSV-2 specific oligonucleotide detection probes. An Internal Control (IC), containing unrelated randomized DNA sequence, is added to all samples prior to automated sample preparation and is amplified and detected simultaneously with each sample to monitor the entire process.

The MSwab Collection, Transport and Preservation System (Copan Flock Technologies) is used for specimen collection, transportation and storage of specimen for the cobas® HSV 1 and HSV 2 Test.

The cobas® HSV 1 and HSV 2 Test utilizes six reagent kits:

1) cobas® 4800 HSV 1 and HSV 2 Amplification/Detection Kit   
2) cobas® 4800 HSV 1 and HSV 2 Controls and Cofactor Kit   
3) cobas® 4800 System Wash Buffer Kit   
4) cobas® 4800 System Lysis Kit 1   
5) cobas® 4800 System Internal Control Kit 1   
6) cobas® 4800 System Sample Preparation Kit

# Test Principle

Target Selection: The cobas® HSV 1and 2 Test utilizes real-time PCR technology to detect the conserved regions of HSV-1 Thymidine Kinase and HSV-1 DNA Polymerase as well as HSV-2 Thymidine Kinase and HSV-2 Glycoprotein B genes. Fluorogenic target-specific probes are used for the detection of the amplified HSV-1 and HSV-2 DNA as well as Internal Control. Since two HSV type targets are detected with different fluorescent dyes, the cobas® HSV 1 and 2 Test has the ability to simultaneously detect and differentiate HSV-1 and HSV-2. Primer and probe oligonucleotide sequences were designed to select HSV-1 and HSV-2 conserved sequences without cross reacting to other viruses, or other bacterial organisms commonly found in human genital areas.

Sample Preparation: Sample preparation for the cobas® HSV 1 and 2 Test is automated with the use of the cobas ${ \textsuperscript { \textregistered } } _ { \textbf { X } 4 8 0 }$ instrument. Organisms from swab specimens collected in MSwab medium are lysed with chaotropic agent, proteinase K, and SDS reagents. Released nucleic acids, along with added Internal Control DNA, are bound by magnetic glass particles. They are washed and then eluted into a small volume of buffer. The instrument then takes an aliquot of the eluted material and sets up the PCR reaction with an activated Master Mix.

PCR Amplification and TaqMan® Detection: The PCR cycling steps and detection of target signal occurs in the cobas® $ { { ^ \circ } } _ {  { \mathbf { z } } } 4 8 0$ Analyzer. The Master Mix reagent contains primer pairs and probes for HSV-1, HSV-2 and IC targets. If the target nucleic acid sequences are present, amplification with the corresponding primers will occur by a thermostable DNA polymerase, generating PCR products (amplicons). These products are detected by specific TaqMan probes containing a fluorescent dye and a quencher. Normally, the quencher suppresses the fluorescence of the dye. However, if the PCR product is present, the probe hybridizes to the product and gets cleaved by the $5 ^ { \prime }$ to $3 ^ { \prime }$ nuclease activity of the polymerase. This reaction allows the fluorescence to be emitted from the dye, and the signal is recorded in real time during each PCR cycle by the $\mathbf { c o b a s } ^ { \textregistered } \mathbf { z } 4 8 0$ analyzer. The signal is interpreted by the cobas® 4800 System Software and reported as final results.

# Assay Controls

RMS provides Negative Control (NC), Positive Control (PC) and Internal Control (IC) as reagent components in the cobas® HSV 1 and 2 Test

NC – contains a buffer solution. The NC is processed in each run that contains a batch of HSV specimens and should invalidate the run if there is contamination during the assay process that results in a positive signal in any of the assay target detection channels and/or if internal control signal is negative or invalid.   
PC – contains one plasmid that contains the HSV-1 target sequence, and a second plasmid that contains the HSV-2 target sequence, in the same buffer as NC. One PC is processed in each run that contains a batch of HSV specimens. The PC monitors the overall reagent and process integrity and will invalidate the run if results are outside of allowable ranges in the assay target detection channels and/or internal control signal is negative or invalid.   
IC – contains recombinant lambda phage with a generic DNA sequence, which is amplified and detected along with HSV-1 and 2 targets but uses specific primers and probe that are different from the HSV-1 and 2 sequences. Since the IC DNA is contained in the phage capsule, it acts both as a specimen lysis control and as a control for PCR inhibition. The IC is added to all specimens, PC and NC in a run.

# cobas® 4800 System Description

The cobas® 4800 System is a diagnostic system designed for sample preparation and real time amplification and detection of nucleic acid targets from clinical samples. The system hardware is unchanged from that originally approved for IVD use in PMA P100020 (cobas® HPV Test, April 19, 2011). The software version has been updated to software release 2.1 in order to support the expanded test menu. The updated software was cleared for the current tests on the cobas® 4800 system per Special 510(k) (K140887).

# 2. INTENDED USE

The cobas® HSV 1 and 2 Test on the cobas® 4800 system is an automated, qualitative in vitro diagnostic test, that utilizes real-time polymerase chain reaction (PCR), for the direct detection and differentiation of Herpes simplex virus 1 and 2 (HSV-1 and HSV-2) DNA in cliniciancollected, external anogenital lesion specimens from symptomatic male and female patients. The cobas® HSV 1 and 2 Test is intended for use as an aid in diagnosis of anogenital HSV-1 and HSV-2 infections in symptomatic patients.

Warning: The cobas® HSV 1 and 2 Test is not FDA cleared for use with cerebrospinal fluid (CSF) and is not intended to be used for prenatal screening or for individuals under the age of 18 years.

# 3. TECHNOLOGICAL CHARACTERISTICS

The RMS cobas® HSV 1 and 2 Test is substantially equivalent in terms of its technological characteristics to the currently legally marketed predicate device, the BD ProbeTecTM Herpes Simplex Viruses (HSV 1 & 2) $\boldsymbol { \mathrm { Q } } ^ { x }$ Amplified DNA Assays on the BD Viper System in Extracted Mode, (K103798).

Differences reside in the amplification technology. Although the candidate test utilizes real-time PCR while the predicate uses strand displacement, both methods are using the same basic principle to amplify low copies of nucleic acids to amounts which can subsequently be detected.

Table 1: Similarities and Differences Between the cobas® HSV 1 and 2 Test and the Predicate Device   

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">BD ProbeTecTM Herpes SimplexViruses (HSV 1 &amp; 2) Qx AmplifiedDNA Assays on the BD ViperSystem in Extracted Mode,(K103798)</td><td colspan="1" rowspan="1">Roche cobas® HSV 1 and 2 TestNew Device (K150617)</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The BD ProbeTecTM Herpes SimplexViruses (HSV 1 &amp; 2) Qx AmplifiedDNA Assays (HSV Q" Assays), whentested with the BD ViperTM System inExtracted   Mode,    use    Strand</td><td colspan="1" rowspan="1">The cobasHSV 1 and 2 Test on thequalitative in vitro diagnostic test, thatutilizes real-time polymerase chainreaction (PCR), for the direct detection</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Displacement             Amplificationtechnology for the direct, qualitativedetection and differentiation of HerpesSimplex virus type 1 (HSV1) andHerpes Simplex virus type 2 (HSV2)DNA in clinician-collected externalanogenital lesion specimens. Theassays are indicated for use withsymptomatic individuals to aid in thediagnosis of anogenital HSV1 andHSV2 infections.Warning: The BD ProbeTecTM HerpesSimplex Viruses (HSV 1 &amp; 2) QxAmplified DNA Assays (HSV QxAssays) are not FDA cleared for usewith cerebrospinal fluid (CSF). Theassays are not intended to be used forprenatal screening or for individualsunder the age of 17 years.</td><td colspan="1" rowspan="1">and differentiation of Herpes simplexvirus 1 and 2 (HSV-1 and HSV-2) DNAin clinician-collected anogenital lesionspecimens from symptomatic male andfemale patients. The cobas® HSV 1 and2 Test is intended for use as an aid indiagnosis of anogenital HSV-1 andHSV-2 infections in symptomaticpatients.Warning: The cobas® HSV 1 and 2Test is not FDA cleared for use withcerebrospinal fluid (CSF) and is notintended to be used for prenatalscreening or for individuals under theage of 18 years.</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">External anogenital lesions</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detection anddifferentiation of HSV-1 and HSV-2DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes usingfluorescence resonance energytransfer (FRET)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">BD ProbeTec Herpes SimplexViruses (HSV 1 &amp; 2) Q" AmplifiedDNA Assays on the BD ViperSystem in Extracted Mode,(K103798)</td><td colspan="1" rowspan="1">Roche cobas® HSV 1 and 2 TestNew Device (K150617)</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Strand Displacement Amplification</td><td colspan="1" rowspan="1">Real-time PCR</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationProcedure</td><td colspan="1" rowspan="1">Automated on BDTM ViperTMSystem in Extracted Mode</td><td colspan="1" rowspan="1">Automated on cobas® 4800 System</td></tr></table>

# 4. NON-CLINICAL PERFORMANCE EVALUATION

# Analytical sensitivity (Limit of Detection)

The analytical sensitivity (Limit of Detection or LOD) for the cobas® HSV 1 and 2 Test was determined by analyzing quantified HSV-1 and HSV-2 viral cultures diluted at multiple concentration levels into a simulated anogenital lesion swab matrix. The simulated matrix composed of mucin and human cells and mimics the effect of the clinical anogenital background for the cobas® HSV 1 and 2 Test. All levels were tested with at least 21 replicates using the full cobas® HSV 1 and 2 Test workflow across five lots of cobas® HSV 1 and 2 Test reagents. LOD for this test is defined as the target concentration which can be detected as positive in $12 \%$ of the replicates tested, based on results generated by the worst performing lot.

HSV-1 Maclntyre and HSV-2 G strains were tested in the analytical sensitivity study. The cobas® HSV 1 and 2 Test LOD on these isolates is shown in Table 2.

Table 2: cobas® HSV 1 and 2 Test LOD (Limit of Detection)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ATCC ID</td><td rowspan=1 colspan=1>LOD (TCID50 /mL)</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>Maclntyre</td><td rowspan=1 colspan=1>VR-539</td><td rowspan=1 colspan=1>0.479</td></tr><tr><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>VR-734</td><td rowspan=1 colspan=1>0.112</td></tr></table>

# Analytical Inclusivity

Four HSV-1 strains (VR-260, VR-733, VR-735 and VR-1493) and three HSV-2 strains (VR-1779, VR-1781 and VR-540) were tested for reactivity with the cobas® HSV 1 and 2 Test. These strains were obtained from ATCC and were cultured and quantified by Virapur, LLC (California, US). Each strain was diluted in a similar fashion as in the Limit of Detection study and was tested in 40 replicates near the LoD. All strains were detected by the assay, demonstrating that the cobas® HSV 1 and 2 Test can detect a broad range of both HSV-1 and HSV-2 isolates.

# Precision

An in-house precision study was conducted with HSV-1 and HSV-2 viruses diluted in a simulated anogenital swab matrix to concentration levels below Limit of Detection (LOD), near LOD and above LOD of the cobas® HSV 1 and 2 Test. A negative level composed of only the simulated anogenital swab matrix was also tested. The study used three unique lots of cobas® HSV 1 and 2 Test reagents and three instruments for a total of 36 runs over 12 days. A description of the precision panels and the study results are shown in Table 3. An analysis of the variance of the Ct values from valid tests was performed on positive panel members at above LOD concentrations. The analysis yielded overall CV $( \% )$ of $2 . 2 \%$ for HSV-1 Ct and $1 . 9 \%$ for HSV-2 Ct (see Table 4 and Table 5).

Table 3: In-house precision study hit rate analysis   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=3>HSV-1 (N=72)</td><td rowspan=1 colspan=3>HSV-2 (N=72)</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>Hitrate</td><td rowspan=1 colspan=1>95% 2-Sided CI</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>Hitrate</td><td rowspan=1 colspan=1>95% 2-Sided CI</td></tr><tr><td rowspan=1 colspan=1>P1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0 - 5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0 - 5%</td></tr><tr><td rowspan=1 colspan=1>P2</td><td rowspan=1 colspan=1>&lt;LOD</td><td rowspan=1 colspan=1>&lt;LOD</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>38 - 62%</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>56%</td><td rowspan=1 colspan=1>43 - 67%</td></tr><tr><td rowspan=1 colspan=1>P3</td><td rowspan=1 colspan=1>~LOD</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0 - 5%</td></tr><tr><td rowspan=1 colspan=1>P4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>~LOD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0 - 5%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>93 - 100%</td></tr><tr><td rowspan=1 colspan=1>P5</td><td rowspan=1 colspan=1>~3xLOD</td><td rowspan=1 colspan=1>~ LOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr><tr><td rowspan=1 colspan=1>P6</td><td rowspan=1 colspan=1>~ LOD</td><td rowspan=1 colspan=1>~3xLOD</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr></table>

Table 4: Variance components analysis for precision panel at ${ \textbf { 3 x } }$ LOD (Limit of Detection)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>HSVLevel</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=6>Variance Components/Percent Contribution to Total</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Run/Day</td><td rowspan=1 colspan=1>Random</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=2 colspan=1>~3xLOD</td><td rowspan=2 colspan=1>37.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.355</td><td rowspan=1 colspan=1>0.289</td><td rowspan=1 colspan=1>0.704</td></tr><tr><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>8.60%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>50.40%</td><td rowspan=1 colspan=1>41.10%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=2 colspan=1>∼3 xLOD</td><td rowspan=2 colspan=1>38.2</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.102</td><td rowspan=1 colspan=1>0.345</td><td rowspan=1 colspan=1>0.53</td></tr><tr><td rowspan=1 colspan=1>6.50%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>9.10%</td><td rowspan=1 colspan=1>19.30%</td><td rowspan=1 colspan=1>65.00%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 5: Standard deviations and coefficients of variation $( \% )$ analysis for precision panel at $\mathbf { 3 \ x \ L O D }$ (Limit of Detection)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=5>Standard Deviation Components/CV Percent</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Kit Size</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Run/Day</td><td rowspan=1 colspan=1>Random</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=2 colspan=1>72</td><td rowspan=2 colspan=1>37.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.245</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.595</td><td rowspan=1 colspan=1>0.538</td><td rowspan=1 colspan=1>0.839</td></tr><tr><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.70%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1.60%</td><td rowspan=1 colspan=1>1.40%</td><td rowspan=1 colspan=1>2.20%</td></tr><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=2 colspan=1>72</td><td rowspan=2 colspan=1>38.2</td><td rowspan=1 colspan=1>0.186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.587</td><td rowspan=1 colspan=1>0.728</td></tr><tr><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.60%</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>1.50%</td><td rowspan=1 colspan=1>1.90%</td></tr></table>

# Competitive inhibition

Panels were constructed with HSV-1 at $\sim 3 \mathrm { \Delta x }$ LOD (Limit of Detection), and competing HSV-2 at $\sim 3 0 0 \mathrm { x }$ LOD of the cobas® HSV 1 and 2 Test; and vice versa. One hundred fold higher concentration of HSV-1 did not affect the detection of HSV2 at $\sim 3 \mathrm { ~ x ~ }$ LOD concentration and one hundred fold higher concentration of HSV-2 did not affect the detection of HSV1 at $\sim 3 \mathrm { ~ x ~ }$ LOD concentration.

# Analytical specificity/Cross-reactivity and Microbial Panel

The analytical specificity of the cobas® HSV 1 and 2 Test was assessed by testing a panel of organisms that could be present in anogenital swab specimens. The panel consists of 1) 71 bacteria, fungi and viruses that may be found in anogenital swab specimens, 2) human cells , and 3) high titer HSV-1 or HSV-2. Testing was performed with the organisms and human cells alone to determine the analytical specificity of the cobas® HSV 1 and 2 Test or in the presence of HSV-1 and HSV-2 at $\sim 3 \mathrm { ~ x ~ }$ LOD to assess the potential interference of the organisms and the human cells on detection of HSV-1 and HSV-2 by the cobas® HSV 1 and 2 Test.

All organisms, human cells, HSV-1 and HSV-2 viruses were spiked to $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ Units\*/mL or higher except for Treponema pallidum; Chlamydia trachomatis serovar $H ,$ and Mycoplasma genitalium which were spiked to lower concentrations due to stock concentration limitations. \*All bacteria were quantified as Colony Forming Units (CFU) except Chlamydia trachomatis serovar H and Chlamydia trachomatis serovar I which were quantified as Inclusion Forming Units (IFU); Toxoplasma gondii and Treponema pallidum which were quantified as DNA copies and Trichomonas vaginalis which was quantified in cells. Cytomegalovirus (HHV5), Human Herpes Virus 6B Strain Z29, Human Herpes Virus 7 Strain SB, Echovirus 11, Human enterovirus 71 and Rubella Virus were quantified as $\mathrm { T C I D } _ { 5 0 }$ units; HHV-6A strain GS, HSV-1 and HSV-2 were quantified in viral particles, HIV-1 Strain IIIB and HBV were quantified in International Units (IU). HIV-2 Strain NIH-Z, Epstein-Barr Virus (HHV4), Varicella-Zoster Virus (HHV3) and HPV plasmids (HPV11, HPV16, HPV18, HPV6) were quantified as DNA copies. Two sources of Human Herpes Virus 8 were used, one was quantified as DNA copies, and the other was quantified in cells and estimated as 150 DNA copies per cell. Human Peripheral Blood Mononuclear Cells (PBMC) were quantified as number of cells.

Results indicated that none of these organisms or high concentration of human cells produced false positive results when there was no HSV-1 and HSV-2 target present. None of these organisms or high concentration of human cells interfered with the detection of HSV-1 and HSV-2 targets. A high concentration of HSV-1 $( 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { v p / m L } )$ ) did not produce false HSV-2 positive results and a high concentration of HSV-2 $\mathrm { ( 1 x 1 0 ^ { 6 } \ v p / m L ) }$ ) did not produce false HSV-1 positive results.

Table 6: Cross Reactivity Panel   

<table><tr><td rowspan=1 colspan=1>Human Adenovirus type 7</td><td rowspan=1 colspan=1>Staphylococcus aureus(MRSA)</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (HHV5)</td><td rowspan=1 colspan=1>Staphylococcus aureus(MSSA)</td><td rowspan=1 colspan=1>Moraxella lacunata</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr Virus (HHV4)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td></tr><tr><td rowspan=1 colspan=1>Varicella-Zoster Virus (HHV3)</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Mycoplasma genitalium**</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 6A strainGS</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Mycoplasma hominis</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 6B StrainZ29</td><td rowspan=1 colspan=1>Chlamydia trachomatisserovar H**</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 7 StrainSB</td><td rowspan=1 colspan=1>Chlamydia trachomatisserovar I</td><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 8*</td><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>HBV</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>HIV-1 Strain IIIB</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>HIV-2 Strain NIH-Z</td><td rowspan=1 colspan=1>Enterococcus faecalis vanB</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>HPV11</td><td rowspan=1 colspan=1>Enterococcus faecium vanA</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>HPV16</td><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>HPV18</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>HPV6</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Rubella Virus</td><td rowspan=1 colspan=1>Haemophilus ducreyi</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Treponema pallidum**</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffii</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp.ozaenae</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>PBMC (human genomicDNA)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Herpes Simplex Virus-1</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Mobiluncus curtisii spp.curtisii</td><td rowspan=1 colspan=1>Herpes Simplex Virus-2</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>-</td></tr></table>

\* Two sources of Human Herpes Virus 8 were tested at $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ copies/mL (HHV8 viral DNA and HHV8 containing human cell line BCP-1) $^ { \ast \ast }$ Chlamydia trachomatis serovar H was tested at $1 . 9 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { \ : I F U / m L }$ ; Mycoplasma genitalium was tested at $1 . 0 \mathrm { x }$ $1 0 ^ { 5 } \mathrm { C F U / m L }$ ; Treponema pallidum was tested at $9 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies/mL

# Interference

Twenty commonly used over the counter (OTC) products and anti-viral medications, as well as whole blood, human serum albumin, urine, feces, and mucin, were tested for potential interference effects with the cobas® HSV 1 and 2 Test. All OTC products were tested at or above ( $2 0 \mathrm { m g }$ or $4 0 \mathrm { m g }$ per swab respectively for solids and $100 \%$ of swab capacity for liquids)

the levels that could be reasonably expected to be collected by a swab in an anogenital lesion specimen. Anti-viral medicine was tested at $3 \mathrm { ~ x ~ }$ Cmax in the collected specimen. HSV-1 and HSV-2 were spiked to $\sim 3 \mathrm { \Delta x }$ LOD (Limit of Detection) of the cobas® HSV 1 and 2 Test and used as targets in the tests.

No interference was observed for the OTC products except for Vagisil Crème (interference observed at $1 0 ~ \mathrm { m g }$ and above). For whole blood, no interference was observed up to $40 \%$ of the swab capacity; for mucin, no interference was observed up to $4 . 8 \mathrm { m g }$ per swab: for urine, no interference was observed up to $100 \%$ of the swab capacity; for feces, no interference was observed up to $1 . 6 \mathrm { m g }$ per swab and for human serum albumin, no interference was observed up to $1 6 ~ \mathrm { m g }$ per swab. These results are summarized in Table 7.

Table 7: Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Substance/ProductName</td><td colspan="1" rowspan="1">Composition</td><td colspan="1" rowspan="1">Testing Level/Swab</td></tr><tr><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">Human whole blood</td><td colspan="1" rowspan="1">40%, 50%</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin Type II from porcine stomach</td><td colspan="1" rowspan="1">4.8 mg, 8 mg, 12 mg,20 mg</td></tr><tr><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">Human urine</td><td colspan="1" rowspan="1">70%, 100%</td></tr><tr><td colspan="1" rowspan="1">Feces</td><td colspan="1" rowspan="1">Human feces</td><td colspan="1" rowspan="1">1.6 mg, 4 mg</td></tr><tr><td colspan="1" rowspan="1">Human Serum Albumin</td><td colspan="1" rowspan="1">Human serum albumin, fatty acid and globulin free</td><td colspan="1" rowspan="1">8 mg, 16 mg</td></tr><tr><td colspan="1" rowspan="1">K-Y Brand Jelly(Personal Lubricant)</td><td colspan="1" rowspan="1">Glycerin, Hydroxyethylcellulose, ChlorhexidineGluconate, Methylparaben, Sodium Hydroxide,Water</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Gynol II (Contraceptivejelly)</td><td colspan="1" rowspan="1">3% Nonoxynol-9, Lactic Acid, Methylparaben,Povidone, Propylene Glycol, Purified Water,Sodium Carboxymethulcellulose, Sorbic Acid,Sorbitol Solution</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">YeastGardSuppositories</td><td colspan="1" rowspan="1">Pulsatilla, Candida Parapsilosis, Candida Albicans,Bacillus Coagulans, Polyethylene Glycols</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Monistat 1</td><td colspan="1" rowspan="1">2% Miconazole nitrate, Benzoic Acid, CetylAlcohol, Isopropyl Myristate, Polysorbate 60,Potassium Hydroxide, Propylene Glycol, PurifiedWater, Stearyl Alcohol</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Monistat 3</td><td colspan="1" rowspan="1">4% Miconazole nitrate, Benzoic Acid, CetylAlcohol, Isopropyl Myristate, Polysorbate 60,Potassium Hydroxide, Propylene Glycol, PurifiedWater, Stearyl Alcohol</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">VagiStat 1</td><td colspan="1" rowspan="1">6.5% Tioconazole, Butylated Hydroxyanisole,Magnesium Aluminium Silicate, White Petrolatum</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole vaginalcream</td><td colspan="1" rowspan="1">1% Clotrimazole, Benzyl Alcohol, CetostearylAlcohol, Cetyl Esters Wax, 2-Octyldodecanol,Polysorbate 60, Purified Water, Sodium PhosphateMonobasic, Sorbitan Monostearate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Preparation HHemorrhoidal cream</td><td colspan="1" rowspan="1">14.4% Glycerin, 0.25% Phenylephrine HCl, 1%Pramoxine HCl, 15% white Petrolatum, AloeBarbadensis Leaf Extract, Anhydrous Citric Acid,Butylated Hydroxyanisole, CarboxymethylcelluloseSodium, Cetyl Alcohol, Citric Acid Monohydrate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Abreva cold soretreatment</td><td colspan="1" rowspan="1">10% Docosanol, Benzyl Alcohol, Light MineralOil, Propylene Glycol, Purified Water, SucroseDistearate, Sucrose Stearate</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Releev cold soretreatment</td><td colspan="1" rowspan="1">Benzalkonium Chloride, Purified Water, Viracea,Methyl Cellulose, Methyl Paraben, PotassiumSorbate, Propyl Paraben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Acyclovir Cream</td><td colspan="1" rowspan="1">5% Acyclovir, Polyethylene Glycol</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Vagisil Crème</td><td colspan="1" rowspan="1">20% Benzocaine, 3% Resorcinol, Water, MineralOil, Cetyl Alcohol, Propylene Glycol, GlycerylStearate, PEG-100 Stearate, Isopropyl Palmitate,Aloe Barbadensis Leaf Juice, Tocopherl Acetate,Retinyl Plamitate, Zea Mays Oil</td><td colspan="1" rowspan="1">2.5 mg, 5 mg, 10 mg,20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Balneol HygienicCleansing lotion</td><td colspan="1" rowspan="1">Water, Mineral Oil, Propylene Glycol, GlycerylStearate, PEG-100 Stearate, PEG-40 Stearate,Laureth 4, PEG-4 Dilaurate, Lanolin Oil, SodiumAcetate, Carbomer 934, Triethanolamine,Methylparaben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Vagicaine Anti-ItchCream</td><td colspan="1" rowspan="1">20% Benzocaine, 3% Resorcinol, Aloe barbadensisLeaf Extract, Carbomer Homopolymer Type C,Cetyl Alcohol, Cholecalciferol, Corn Oil, GlycerylMonostearte, Isopropyl Myristate, IsopropylPalmitate, Lanolin Alcohol, Methylparaben</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">VH Essentials Douche</td><td colspan="1" rowspan="1">3% Povidone-iodine, Purified Water, USP,Octylphenoxypolyethoxyethanol</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Denavir</td><td colspan="1" rowspan="1">1% Penciclovir, Cetomacrogol 1000BP, CetosterylAlcohol, Mineral Oil, Propylene Glycol, PurifiedWater, White Petrolatum</td><td colspan="1" rowspan="1">20 mg, 40 mg</td></tr><tr><td colspan="1" rowspan="1">Famciclovir</td><td colspan="1" rowspan="1">Famciclovir, Hydroxypropyl Cellulose,Hydroxypropyl Methylcellulose, Lactose,Magnesium Stearate, Polyethylene Glycols,Sodium Starch Glycolate, Titanium Dioxide</td><td colspan="1" rowspan="1">0.016 mg</td></tr><tr><td colspan="1" rowspan="1">Valacyclovir</td><td colspan="1" rowspan="1">Valacyclovir Hydrochloride, Carnauba Wax,Colloidal Silicon Dioxide, Crospovidone,Hypromellose, Magnesium Stearate,Microcrystalline Cellulose, Polyethylene Glycol</td><td colspan="1" rowspan="1">0.027 mg</td></tr><tr><td colspan="1" rowspan="1">Cidofovir</td><td colspan="1" rowspan="1">Cidofovir, Sodium hydroxide, Sterile Water</td><td colspan="1" rowspan="1">0.552 mg</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">Acyclovir, Magnesium Stearate, MicrocrystallineCellulose, Povidone, Sodium Starch Glycolate</td><td colspan="1" rowspan="1">0.008 mg</td></tr></table>

# Reproducibility

The reproducibility of the cobas® HSV 1 and 2 Test on the cobas® 4800 System was established in a multi-site, investigation using contrived clinical samples evaluated across lot, site/instrument, operator, day, and run.

HSV test panels were prepared by spiking HSV-1 (Maclntyre strain) and/or HSV-2 (G strain) virus into contrived sample matrix in transport media at one of three concentrations (Below LOD, $1 \times \mathrm { L O D }$ , and $3 \times \mathrm { L O D } )$ ; HSV-1 and HSV-2 negative panel members were included as panel member controls. In all, there were 6 members per test panel with 3 replicates per panel member included in each run, not including external positive and negative assay controls. Panels were tested at 3 sites by 2 operators per site with 1 valid run per operator per day, for 6 days per lot over 2 lots for a total of 1,296 valid tests (216 tests/panel member or 108 tests/panel member/lot).

Table 8 summarizes the percent agreement (two-sided $9 5 \%$ exact CI) for the negative panel member and HSV-1 positive panel members.

Table 8: Percent agreement by panel member - HSV-1   

<table><tr><td rowspan=1 colspan=2>EY</td><td rowspan=1 colspan=2>HSV-1</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Number ofValidTest Results</td><td rowspan=1 colspan=1>Agreement(n/N)</td><td rowspan=1 colspan=1>95% CIa</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Below LOD (HSV-1/HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>63.4% (137/216)</td><td rowspan=1 colspan=1>(56.6%, 69.9%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>99.5% (215/216)</td><td rowspan=1 colspan=1>(97.4%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr></table>

a $9 5 \%$ $/ _ { 0 } \mathrm { C I } = 9 5 \%$ exact binomial confidence interval. b Negative panel members for HSV-1: percent negative agreement was $9 9 . 8 \%$ (431/432) with $9 5 \%$ CI $9 8 . 7 \%$ , $1 0 0 . 0 \%$ . c Panel members with $_ { 1 \mathrm { ~ x ~ } }$ LOD HSV-1: percent positive agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI $( 9 9 . 1 \%$ , $1 0 0 . 0 \%$ . Note: Results are included as agreement when a positive panel member has a valid positive result for that analyte or when the negative panel member has a valid negative result for both analytes. $\mathrm { C I } =$ confidence interval; $\mathrm { L O D = }$ limit of detection.

Table 9 presents the percent agreement (negative or positive) by lot, site/instrument, operator, and day for HSV-1 test results for each panel member.

Table 9: Percent agreement by panel member for lot, site/instrument, operator, and day - HSV-1   

<table><tr><td colspan="4" rowspan="1">EY</td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="1">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="1">Lot</td><td colspan="2" rowspan="1">Site/Inst.</td><td colspan="2" rowspan="1">Operator</td><td colspan="2" rowspan="1">Day</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">C%</td><td colspan="8" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0 2/2)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Below LoD(HSV-1/HSV-2)</td><td colspan="1" rowspan="1">41.1</td><td colspan="1" rowspan="1">1.41</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">60.2(65/10)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">56.9(41/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">58.3(21/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">66.7(72/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">68.1(49/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">55.620/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">58.321/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">65.3(47/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">63.9(23/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">72.2(26/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">72.2(26/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">63.9(23/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">66.7(24/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">58.3(21/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD(HSV-1)</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0 36/36)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.06/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/6)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0 72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(336/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0 (36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.036/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">C%v</td><td colspan="8" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD(HSV-2)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">99.1(107/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.6 (71/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.2(35/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.2(35/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(2/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0 (36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0 (36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(336/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0 (336/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0 2/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0 2/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0 /72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0 72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0 (36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.036/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0 (336/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0 (336/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0 36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0 36/36)</td></tr></table>

\* For the negative panel member, percent agreement $=$ (number of negative results/total valid results) x 100; for the positive panel members, percent agreement $=$ (number of positive results/total valid results) $\textbf { \em X } 1 0 0$ . $\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { { N / A = } }$ not applicable; $\mathrm { S D = }$ standard deviation.

Table 10 presents the SD and CV $( \% )$ of Ct values for HSV-1 positive panel members overall and attributable to lot, site/instrument, operator, day, and within-run. Across HSV-1 positive panel members, the total SD ranged from 1.10 to 1.41, and the total CV $( \% )$ ranged from $3 . 0 \%$ to $3 . 4 \%$ .

Table 10: Overall mean, standard deviations, and coefficients of variation $( \% )$ for Ct values from valid results for positive panel members - HSV-1   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=10>Standard Deviation and Percent Coefficients ofVariation</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Site/Inst.</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Below LOD(HSV-1/HSV-2)</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>3.0</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst. $=$ instrument; LOD $=$ limit of detection; $\mathbf { S D = }$ standard deviation.

In summary, the positive percent agreement for the HSV-1 positive panel member “Below LOD (HSV-1/HSV-2)” was $6 3 . 4 \%$ $9 5 \%$ CI: $5 6 . 6 \%$ to $6 9 . 9 \%$ ) and the positive percent agreement for all other positive panel members was $1 0 0 . 0 \%$ b $9 5 \%$ CI: $9 8 . 3 \%$ to $1 0 0 . 0 \%$ ). For the negative panel members, negative percent agreement was $9 9 . 8 \%$ ( $9 5 \%$ CI: $9 8 . 7 \%$ to $1 0 0 . 0 \%$ ). The total SD and total $\% \mathrm { C V }$ across all panel members were $\leq 1 . 4 1$ and $\leq 3 . 4 \%$ , respectively.

# HSV-2 reproducibility results

Table 11 summarizes the percent agreement (two-sided $9 5 \%$ exact CI) for the negative panel member and HSV-2 positive panel members.

Table 11: Percent agreement by panel member - HSV-2   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Number ofValidTest Results</td><td rowspan=1 colspan=1>Agreement(n/N)</td><td rowspan=1 colspan=1>95% CIa</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Below LOD (HSV-1/HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>56.5% (122/216)</td><td rowspan=1 colspan=1>(49.6%, 63.2%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)b</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)/1 x LOD (HSV-2)c</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>100.0%(216/216)</td><td rowspan=1 colspan=1>(98.3%, 100.0%)</td></tr></table>

a $5 \% \mathrm { C I } = 9 5 \%$ exact binomial confidence interval. b Negative panel members for HSV-2: percent negative agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI $( 9 9 . 1 \%$ , $1 0 0 . 0 \%$ . c Panel members with $_ { 1 \mathrm { ~ x ~ } }$ LOD HSV-2: percent positive agreement was $1 0 0 . 0 \%$ (432/432) with $9 5 \%$ CI of $( 9 9 . 1 \%$ , $1 0 0 . 0 \%$ . Note: Results are included as agreement when a positive panel member has a valid result of positive for that analyte or when the negative panel member has a valid result of negative for both analytes. $\mathrm { C I } =$ confidence interval; $\mathrm { L O D = }$ limit of detection.

Table 12 presents the percent agreement (negative and positive) by lot, site/instrument, operator, and day for HSV-2 test results for each panel member.

Table 12: Percent agreement by panel member for lot, site/instrument, operator, and day - HSV-2   

<table><tr><td colspan="4" rowspan="1"></td><td colspan="8" rowspan="1">Percent Agreement (n/N)*</td></tr><tr><td colspan="1" rowspan="1">Panel Member</td><td colspan="3" rowspan="1">Ct</td><td colspan="2" rowspan="1">Lot</td><td colspan="2" rowspan="1">Site/Inst.</td><td colspan="2" rowspan="1">Operator</td><td colspan="2" rowspan="1">Day</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="8" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Below LOD(HSV-1/HSV-2)</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">51.9(56/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">65.3(47/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">55.6(20/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">61.1(66/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50.0(36/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">72.2(26/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50.0(18/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">54.2(39/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">58.3(21/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">47.2(17/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">38.9(14/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">66.7(24/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">61.1(22/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">63.9(23/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">50.0(18/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">55.6(20/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3 x LOD (HSV-1)/1 x LOD (HSV-2)</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(108/108)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(72/72)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0(36/36)</td></tr></table>

\* For the negative panel member, percent agreement $=$ (number of negative results/total valid results) $\textbf { \em X } 1 0 0$ ; for the positive panel members, percent agreement $=$ (number of positive results/total valid results) $\textbf { x } 1 0 0$ . $\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Inst $=$ instrument; $\mathrm { L O D = }$ limit of detection; $\mathrm { { N / A = } }$ not applicable; $\mathrm { S D = }$ standard deviation.

Table 13 presents the SD and CV $( \% )$ of Ct values for HSV-2 positive panel members overall and attributable to lot, site/instrument, operator, day, and within-run. Across HSV-2 positive panel members, the total SD ranged from 0.73 to 0.92, and the total CV $( \% )$ ranged from $1 . 9 \%$ to $2 . 3 \%$ .

Table 13 Overall mean, standard deviations, and coefficients of variation $( \% )$ for Ct values from valid results for positive panel members - HSV-2   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=10>Standard Deviation and Percent Coefficient ofVariation</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Site/Inst.</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Below LOD(HSV-1/HSV-2)</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>40.3</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>3 x LOD (HSV-1)1 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>1 x LOD (HSV-1)/3 x LOD (HSV-2)</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>1.9</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation;; Inst. $=$ instrument; LOD $=$ limit of detection; $\mathrm { S D = }$ standard deviation.

In summary, the positive percent agreement for the HSV-2 positive panel member “Below LOD (HSV-1/HSV-2)” was $56 . 5 \%$ $9 5 \%$ CI: $4 9 . 6 \%$ to $6 3 . 2 \%$ ), whereas the positive percent agreement for all other positive panel members was $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 8 . 3 \%$ to $1 0 0 . 0 \%$ ). For the HSV-2 negative panel member, negative percent agreement was $1 0 0 . 0 \%$ ( $9 5 \%$ CI: $9 9 . 1 \%$ to $1 0 0 . 0 \%$ ). The total SD and total CV $( \% )$ across all panel members were $\leq 0 . 9 2$ and $\leq 2 . 3 \%$ , respectively.

# 5. CLINICAL PERFORMANCE

The clinical performance of the cobas® HSV 1 and 2 Test was established in an IRB-approved, prospective, multi-site, investigation comparing to the combined results of culture and Sanger sequencing as the Reference Method using clinician collected external anogenital lesion swab specimens from patients with possible HSV infection.

Two external anogenital swab specimens were collected from symptomatic eligible male and female subjects 17 years of age or older attending family planning, OB/GYN and sexually transmitted disease clinics at eight geographically diverse sites (seven across the United States and one in the United Kingdom). The first swab was used for (a) culture by the ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test (Diagnostic Hybrids, Inc.), (b) PCR followed by bi-directional Sanger sequencing for HSV 1 and HSV 2, and (c) a second FDA-cleared nucleic acid amplification test The second swab was for the cobas® HSV 1 and 2 Test.

The clinical performance of the cobas® HSV 1 and 2 Test was established compared to a composite Reference Method derived from the combined results of culture (ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test) and Sanger sequencing using the “any positive rule”.

# Comparison with composite reference method (culture and Sanger sequencing)

There was a total of 408 specimens from 205 female and 203 male subjects evaluated in the study with 243 positive for HSV; 84 HSV-1 (51 female, 33 male) and 167 HSV-2 (85 female, 82 male) positive subjects, of which 8 $( 2 \% )$ were positive for both HSV-1 and HSV-2. The clinical performance of the cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test compared to the composite reference method is summarized in the table below.

Table 14: Comparison of cobas® HSV 1 and 2 Test with the composite reference method   

<table><tr><td rowspan="3" colspan="2"></td><td rowspan="3"></td><td colspan="6">Composite Reference Method</td></tr><tr><td colspan="3">HSV 1</td><td colspan="3">HSV 2</td></tr><tr><td>Positive</td><td>Negative</td><td>Total</td><td>Positive</td><td>Negative</td><td>Total</td></tr><tr><td rowspan="3">cobas® HSV 1 and 2 Test</td><td>Positive</td><td>78</td><td>4a</td><td>82</td><td>162</td><td>13c</td><td></td><td>175</td></tr><tr><td>Negative</td><td>6b</td><td>320</td><td>326</td><td>5d</td><td>228</td><td></td><td>233</td></tr><tr><td>Total</td><td>84</td><td>324</td><td>408</td><td>167</td><td>241</td><td></td><td>408</td></tr><tr><td colspan="7">HSV 1 HSV 2 92.9% (78/84) Sensitivity: Sensitivity: (95% CI = 85.3% - 96.7%)</td><td colspan="2">97.0% 162/167)</td></tr><tr><td>Specificity:</td><td colspan="4">98.8% (320/324) Specificity: (95% CI = 96.9% - 99.5%)</td><td>94.6% (228/241)</td><td colspan="3">(95% CI = 93.2% - 98.7%)</td></tr><tr><td>PPV:</td><td colspan="3">95.1% (78/82) PPV:</td><td></td><td colspan="4">(95% CI = 91.0% - 96.8%) 92.6% (162/175)</td></tr><tr><td></td><td colspan="3">(95% CI = 88.1% - 98.1%) 98.2% (320/326)</td><td></td><td colspan="4">(95% CI = 87.7% - 95.6%)</td></tr><tr><td>NPV:</td><td colspan="3">(95% CI = 96.0% - 99.2%)</td><td></td><td colspan="4">97.9% (228/233) NPV: (95% CI = 95.1% - 99.1%)</td></tr></table>

a Of the 4 specimens with HSV-1 false-positive cobas® HSV 1 and 2 Test results relative to the Reference Method, 2 were HSV1 positive by a FDA-cleared nucleic acid amplification test b Of the 6 specimens with HSV-1 false-negative cobas® HSV 1 and 2 Test results relative to the Reference Method, all 6 were HSV-1 negative by a FDA-cleared nucleic acid amplification test c Of the 13 specimens with HSV-2 false-positive cobas® HSV 1 and 2 Test results relative to the Reference Method, 5 were HSV-2 positive by a FDA-cleared nucleic acid amplification test d Of the 5 specimens with HSV-2 false-negative cobas® HSV 1 and 2 Test results relative to the Reference Method, all 5 were HSV-2 negative by a FDA-cleared nucleic acid amplification test

# Comparison with culture

The clinical performance of the cobas $^ \mathrm { \textregistered }$ HSV 1 and 2 Test compared to the ELVIS $\textsuperscript { \textregistered }$ HSV ID and D3 Typing Test system is summarized in Table 15. The ELVIS $\textsuperscript { \textregistered }$ HSV ID and D3 Typing Test system used in this study is unable to detect patients co-infected with HSV-1 and HSV-2 when HSV-2 is detected. Only HSV-2 negative anogenital specimens can be typed for HSV-1. Therefore, the number of samples used for the calculation of HSV-1 clinical performance equals the total number of specimens (408) minus the number of samples positive for HSV-2 by culture (129).

Table 15: Comparison of cobas® HSV 1 and 2 Test with culture   

<table><tr><td rowspan="3"></td><td rowspan="3"></td><td colspan="6">Culture Reference Methoda</td></tr><tr><td colspan="3">HSV 1</td><td colspan="3">HSV 2</td></tr><tr><td>Positive</td><td>Negative</td><td>Total</td><td>Positive</td><td>Negative</td><td>Total</td></tr><tr><td rowspan="3">cobas HSV 1 and 2 Test</td><td>Positive</td><td>67</td><td>13b</td><td>80</td><td>128</td><td>47c</td><td>175</td></tr><tr><td>Negative</td><td>0</td><td>199</td><td>199</td><td>$1}$</td><td>232</td><td>233</td></tr><tr><td>Total</td><td>67</td><td>212</td><td>279</td><td>129</td><td>279</td><td>408</td></tr><tr><td>HSV 1 100.0% (67/67) Sensitivity: (95% CI = 94.6% - 100.0%)</td><td colspan="5">Sensitivity:</td><td colspan="3">HSV 2 99.2% (128/129) (95% CI = 95.7% - 99.9%)</td></tr><tr><td>Specificity:</td><td colspan="5">93.9% (199/212) Specificity:</td><td colspan="3" rowspan="2">83.2% (232/279) (95% CI = 78.3% - 87.1%)</td></tr><tr><td>PPV:</td><td colspan="6">(95% CI = 89.8% - 96.4%) 83.8% (67/80) 73.1% (128/175)</td></tr><tr><td></td><td colspan="3">(95% CI = 74.2% - 90.3%)</td><td>PPV:</td><td colspan="3">(95% CI = 66.1% - 79.2%)</td></tr><tr><td>NPV:</td><td colspan="3">100.0% (199/199) NPV: (95% CI = 98.1% - 100.0%)</td><td></td><td colspan="4">99.6% (232/233) (95% CI = 97.6% - 99.9%)</td></tr></table>

# Summary of Ct values for test positive samples

Histogram plots of Ct values from cobas® HSV 1 and 2 Test positive samples are presented in Figure 1 and Figure 2 for HSV-1 and HSV-2, respectively.

![](images/8c660db980511efcf914b85668e522b224186211a6f71c80ca9a655a00d1d317.jpg)  
Figure 1: Frequency distribution of cycle threshold values for HSV-1 positive test results

![](images/1fbb7f82ed4be3b2eac06967a62fc20effc2197a1b0530f8e4f3dc3cb5cc6e32.jpg)  
Figure 2: Frequency distribution of cycle threshold values for HSV-2 positive test results

# 5.1. Summary

The cobas® HSV 1 and 2 Test exhibited performance comparable to the combined results of culture with the ELVIS® HSV ID and D3 Typing Test System plus Sanger sequencing (Reference Method) using clinician-collected external anogenital lesion swab specimens from symptomatic patients. The results support the intended use of the cobas® HSV 1 and 2 Test as an aid in diagnosis of anogenital HSV 1 and HSV 2 infections in symptomatic patients.

# 6. CONCLUSION

A comparison of the intended use, technological characteristics, and the results of non-clinical analytical and clinical performance studies demonstrate that the cobas $^ { \textregistered } \mathrm { H S V } \ 1$ and HSV 2 Test is substantially equivalent to the predicate device.