# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION Summary

A. 510(k) Number: K150962   
B. Purpose for Submission: Clearance of New Device   
C. Measurand: Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2)   
D. Type of Test: Real-time PCR system   
E. Applicant: Focus Diagnostics, Inc.   
F. Proprietary and Established Names: Simplexa™ HSV 1 & 2 Direct Simplexa™ HSV 1 & 2 Positive Control Pack

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

Simplexa™ HSV 1 & 2 Direct The Focus Diagnostics Simplexa™ HSV 1 & 2 Direct assay is intended for use on the 3M Integrated Cycler instrument for the qualitative detection and differentiation of herpes simples virus (HSV-1 and HSV-2) DNA present in genital lesion swabs samples from patients with signs and symptoms of HSV-1 or HSV-2 infection of the genitalia. This test is an aid in the differential diagnosis of HSV-1 and HSV-2 genital infections.

The assay is not intended for use as a screening test for the presence of HSV-1 and HSV2 in blood or blood products. The assay is for professional use only.

Simplexa™ HSV 1 & 2 Positive Control Pack   
The Simplexa™ HSV 1 & 2 Positive Control Pack is intended to be used as a control with the Simplexa™ HSV 1 & 2 Direct kit. This control is not intended for use with other assays or systems.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

To be used with the 3M Integrated Cycler with Integrated Cycler Studio Software version 6.0 or higher.

# I. Device Description:

The Simplexa™ HSV 1 & 2 Direct assay system is a real-time PCR that enables the direct amplification, detection and differentiation of HSV-1 and/or HSV-2 DNA from unprocessed genital swab samples without nucleic acid extraction. The system consists of the Simplexa™ HSV 1 & 2 Direct assay, the 3M Integrated Cycler (with 3M Integrated Cycler Studio Software), the Direct Amplification Disc and associated accessories.

In the Simplexa™ HSV 1 & 2 Direct assay, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify HSV-1, HSV-2 and internal control targets. Well conserved regions of the HSV-1 and HSV-2 DNA polymerase genes are

targeted to identify HSV-1 and HSV-2 DNA respectively in the sample. An internal control is used to detect PCR failure and/or inhibition.

# J. Substantial Equivalence Information:

1. Predicate device name(s): artus® HSV-1/2 QS-RGQ MDx Kit

2. Predicate 510(k) number(s): K142738

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device</td><td>Predicate</td></tr><tr><td>Intended Use</td><td>SimplexaTM HSV 1 &amp; 2 Direct The Focus Diagnostics SimplexaTM HSV 1 &amp; 2 Direct assay is intended for use on the 3M Integrated Cycler instrument for the qualitative detection and differentiation of herpes simplex virus (HSV-1 and HSV-2DNA present in genital lesion swabs samples from patients with signs and symptoms of HSV-1 or HSV-2 infection of the genitalia. This test is an aid in the differential diagnosis of HSV-1 and HSV-2 genital infections. The assay is not intended for use as a screening test</td><td>The artus HSV-1/2 QS- RGQ MDx Kit is an in vitro real-time PCR DNA amplification assay performed on the QIAsymphony RGQ MDx system for the direct qualitative detection and differentiation of herpes simplex virus (HSV-1 and HSV-2) DNA in genital or oral vesicular lesions from male and female patients suspected of HSV infection. The assay is intended for use as an aid in diagnosis of HSV infection in symptomatic patients. Warning: The artus HSV- 1/2 QS-RGQ MDx Kit is not FDA-cleared for use with cerebrospinal fluid</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">SimplexaTM HSV 1 &amp; 2Positive Control PackThe SimplexaTM HSV 1 &amp; 2Positive Control Pack isintended to be used as acontrol with the SimplexaTMHSV 1 &amp; 2 Direct kit.This control is not intendedfor use with other assays orsystems.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Detection Technology</td><td colspan="1" rowspan="1">Multiplex PCR based assayusing different reporter dyesfor each target.</td><td colspan="1" rowspan="1">Multiplex PCR based assayusing different reporterdyes for each target.</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">HSV-1 and HSV-2 DNA</td><td colspan="1" rowspan="1">HSV-1 and HSV-2 DNA</td></tr></table>

<table><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td>Device</td><td>Predicate</td></tr><tr><td>Sample types</td><td>Genital herpetic lesion samples</td><td>Genital and oral herpetic lesion swab samples</td></tr><tr><td>Nucleic acid Isolation</td><td>Targets are amplified and detected directly from the clinical sample; there is no separate nucleic acid isolation step.</td><td>The nucleic acid is isolated and prepared for the amplification method using the QIAsymphony DSP Virus/Pathogen Mini Kit.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff: Administrative Procedures for CLIA Categorization, May 7, 2008

Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k), August 12, 2005

Off-The-Shelf Software Use in Medical Devices, September 9, 1999

Cybersecurity for Networked Medical Devices Containing Off-The-Shelf (OTS) Software, July 18, 2007

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005

General Principles of Software Validation, January 11, 2002

# L. Test Principle:

The Simplexa™ HSV 1 & 2 Direct system is a real-time PCR for the direct amplification, detection and differentiation of HSV-1 and HSV-2 DNA from unprocessed genital lesion samples without nucleic acid extraction. The system consists of the Simplexa™ HSV 1 & 2 Direct, the 3M Integrated Cycler (with 3M Integrated Cycler Studio Software), the Direct Amplification Disc and associated accessories.

In the Simplexa™ HSV 1 & 2 Direct assay reaction, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify HSV-1, HSV-2 and internal control (IC) targets. Well conserved regions of the HSV-1 and HSV-2 DNA polymerase genes are targeted to identify HSV-1 and HSV-2 DNA, respectively, in the sample. The IC is used to detect PCR failure and/or inhibition. The principle of the test and the procedural steps are summarized below.

# Sample Collection:

The acceptable specimen type is a swab sample collected from a herpetic genital lesion and stored in BD UVT, Remel M4, Remel M4RT, Remel M5, Remel M6 or UTM transport media. The manufacturer’s package inserts should be followed for collection media and acceptable swab types. Calcium alginate swabs should not be used, as they may contain substances that inhibit PCR testing. Specimens should be transported on ice and stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ for up to 7 days post collection. If there is a delay greater than 7 days before processing of the specimen, store specimen at $\mathrm { - 7 0 ^ { \circ } C }$ .

# Real-Time PCR Instrument Setup:

The 3M Integrated Cycler Operator Manual provides details on how to configure the 3M Integrated Cycler Studio Software, to add an assay definition, and to set up and analyze runs on the 3M Integrated Cycler.

Direct Amplification Disc Loading and Real-Time PCR Amplification:

1. Select samples that need to be tested.   
2. Thaw Reaction Mix vials at room temperature (approximate range 18 to $2 5 ^ { \circ } \mathrm { C }$ ). Thaw one Reaction Mix vial for each sample or control to be tested.   
3. Scan the barcode on the Simplexa™ HSV 1 & 2 Direct Reaction Mix vial or barcode card.   
4. Scan the disc barcode on the Direct Amplification Disc (DAD).   
5. Scan or type in each sample identifier.   
6. For one wedge at a time, peel the adhesive foil back to expose the Sample (SAMPLE) and Reaction (R) wells without completely removing the adhesive foil cover (Figures 1 & 2). Avoid touching the under side of the foil that will be in contact with the wells and disc surface.   
7. Ensure that the Reaction Mix is completely thawed. Briefly spin down the tubes as needed. (Do not vortex the Reaction Mix).   
8. Use the fixed volume pipette to transfer $5 0 ~ \mu \mathrm { L }$ of the Reaction Mix into the Reaction (R) well.   
9. Use the fixed volume pipette to transfer $5 0 ~ \mu \mathrm { L }$ of sample or control; pipette sample or control into the Sample well (SAMPLE).   
10. Cover the wedge sealing the wells with the peeled adhesive foil, pressing down firmly near the edge of the wedge. If the original foil is torn, do not load the wells in the wedge. Instead load another wedge.   
11. Tear off the tab portion of the foil cover along the perforation.   
12. Repeat steps 6 to 11 for the next sample(s).   
13. Load the sealed Direct Amplification Disc into the 3M Integrated Cycler and start the run.

![](images/cfb364f31389c82dca889af599b5ecdccf6882f3e1867b033dab2bc903f6e9e8.jpg)

# Interpretation of Results:

Upon completion of the run, the Integrated Cycler Studio Software automatically calculates and displays results. The display presents the user with a separate report box for each of the analytes as well as the IC. The IC can be reported as “valid” or “invalid”.

The software reports one of four possible outcomes for each of HSV-1 and HSV-2 after a run is completed for each sample ID entered: “Detected”, “Not Detected”, “Invalid” or $^ { \mathrm { { e } } } \mathrm { { E } } \mathrm { { C } } 5 0 0 ^ { \mathrm { { , } } }$ .

a. “Detected” result indicates that HSV-1 and/or HSV-2 DNA were detected in the patient sample tested (whether the DNA IC was detected or not detected).   
b. “Not Detected” result indicates that HSV-1 and/or HSV-2 DNA were not detected in the patient sample tested whereas the DNA IC was detected.   
c. “Invalid” result points to the inability to detect presence or absence of HSV-1 and/or HSV-2 DNA in the patient sample. This result may be due to: 1. DNA Internal Control (DNA IC) failure, or 2. Failure to detect sufficient sample. If an invalid result occurs, the sample needs to be re-tested per the instructions provided in the device Package Insert.

d. $^ { \cdot } \mathrm { E C } 5 0 0 ^ { \cdot \gamma }$ result points to a data quality error for the particular viral analyte(s). The software was unable to determine a valid amplification for that analyte(s). The sample should be re-tested.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the Simplexa™ HSV 1 & 2 Direct assay was evaluated at three investigative sites by assessing the device's inter-site, inter-day and inter/intra-assay reproducibility. Each of the laboratories tested the positive control and a panel of five contrived sample pools including a low (approximately 1-2 times LoD) and medium positive (approximately 2-4 times LoD) for each analyte and a high negative. The high negative sample contained a small amount of HSV-1 and HSV-2, and it was designed to be negative approximately $9 5 \%$ of the time. The assays were performed in triplicate on five different days. Each site had two operators; each operator assayed the entire sample panel and positive control once per day, for a total of two sets of data per day. The combined results for all sites are presented in the tables below.

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementwithExpectedResuts</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.C</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpected Results</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=7 colspan=1>HSV-1Result</td><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>100.0%(0/90)</td><td rowspan=1 colspan=1>95.9 to10.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-1MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to10.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30/30)a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.7%(29/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>98.9%(89/90) a</td><td rowspan=1 colspan=1>94.0 to99.8%</td></tr><tr><td rowspan=1 colspan=1>HSV-2MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.7%(29/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>98.9%(89/90)a</td><td rowspan=1 colspan=1>94.0 to99.8%</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>96.7%(29/30)a</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>93.3%(28/30) a</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>90.0%(27/30) a</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>93.3%(84/90) a</td><td rowspan=1 colspan=1>86.2 to96.9%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>99.4% (179/180)</td><td rowspan=1 colspan=3>98.3% (177/180)</td><td rowspan=1 colspan=3>97.8% (175/179)</td><td rowspan=1 colspan=1>98.5%(531/539)</td><td rowspan=1 colspan=1>97.1 to99..2%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=5>n Hgh N                  1</td></tr></table>

a) Expected Results of HSV-2 Low Positive, HSV-2 Medium Positive and High Negative samples are “Negative” for HSV-1.

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementwithExpectedResults</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.C</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.C</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=7 colspan=1>HSV-2Result</td><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>100.0%(0/ /0))</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(0/30)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.7%(29/0)  b</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>98.9%(89/900) b</td><td rowspan=1 colspan=1>94.0 to99.8%</td></tr><tr><td rowspan=1 colspan=1>HSV-1MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/ 0)b</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30//0)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30/0 )b</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0% 90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>90.0%(27/30)</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>93.3%(28/30)</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>94.4%(85/90)</td><td rowspan=1 colspan=1>87.6 to97.6%</td></tr><tr><td rowspan=1 colspan=1>HSV-2MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>100.0%(90/0)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>96.7%(/ /30</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>86.7%(26/0) b</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>100.0%(0/ /0)b</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>94.4%(85/90) b</td><td rowspan=1 colspan=1>87.6 to97.6%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>99.4% (179/180)</td><td rowspan=1 colspan=3>96.1% (173/180)</td><td rowspan=1 colspan=3>98.9% (176/179)</td><td rowspan=1 colspan=1>98.0%(528/539)</td><td rowspan=1 colspan=1>96.4 to98.9%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=12>Expec Results o HSV-1 Low Positive, HSV- Mediu Positive and High Negative amples are &quot;Negative&quot;or HSV-</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementwithExpectedResults</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementwithExpectedesults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=6 colspan=1>DNAICResult</td><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%(0/90)</td><td rowspan=1 colspan=1>95.9 to10.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-1Mediumositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2MediumPositive</td><td rowspan=1 colspan=1>100.0%(0/30)</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(0/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>100.0%(30/3)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to10.0%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to10.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>100.0% (180/180)</td><td rowspan=1 colspan=1>100.0% (180/180)</td><td rowspan=1 colspan=1>100.0% (179/179)</td><td rowspan=1 colspan=1>100.0%(539/539)</td><td rowspan=1 colspan=1>96.4 to98.9%</td></tr></table>

b. Linearity/assay reportable range:

Not Applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The positive control for this product is a blend of inactivated HSV-1 and HSV-2 provided in single use aliquots frozen at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ . The positive control has been assigned an expiration date of 12 months at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ .

To assess the room temperature stability of the positive control, aliquots of the positive control were thawed and kept at room temperature for up to 24 hours, and were then evaluated. No significant change in detection of each target analyte was observed when positive control was stored on the bench top at room temperature for 24 hours. Based on these results, positive control is stable for at least 24 hours at room temperature.

# d. Assay cut-off:

The Ct cut-offs for the Simplexa™ HSV 1 & 2 Direct assay were determined during feasibility and verification studies. During verification, all studies were performed to a maximum of 45 cycles of amplification for informational use, but the data were calculated with a cut-off of $\mathrm { C t } { = } 4 0$ for the HSV-1 and HSV-2 targets. When the verification data were analyzed, there were some results that prompted the Ct cut-off for HSV-2 to be re-set from $\mathrm { C t } { = } 4 0$ to $\mathrm { C t } { = } 4 2$ . The LoD verification study detected HSV-2 between a $\mathrm { C t }$ of 40 and 41 for four replicates at HSV-2 concentrations below the established HSV-2 LoD concentration. No HSV-2 LoD replicates (at or below LoD concentration) were detected beyond $\mathrm { C t } { = } 4 1$ . The HSV-1 cut-off was not adjusted from $\mathrm { C t } { = } 4 0$ because no HSV-1 LoD replicates from the LoD verification study were detected at a $\mathrm { C t } { > } 4 0$ . Based on the results of the verification studies, the final cut-offs were set at $\mathrm { C t } { = } 4 0$ for HSV-1 and at $\mathrm { C t } { = } 4 2$ for HSV-2. The PCR cycling protocol was reduced to 42 cycles because all target cut-offs are at $\mathrm { C t } { \le } 4 2$ . The Ct cutoffs and cycling protocol is stored in the assay definition for the Simplexa™ HSV 1 & 2 Direct.

# e. Detection limit (LoD):

The Limit of Detection (LoD) was determined for the Simplexa™ HSV 1 & 2 Direct assay using quantified stocks of HSV-1 and HSV-2 serially diluted into negative genital swab matrix containing male and female genital swabs. The LoD was determined to be the lowest concentration that could be detected positive $> 9 5 \%$ of the time.

<table><tr><td rowspan=1 colspan=1>VirusStrain</td><td rowspan=1 colspan=1>LoDConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>Qualitative Results(#Detected/#Total)</td><td rowspan=1 colspan=1>MeanCt ± SD(fromDetectedReplicatesonly)</td></tr><tr><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>36.4 ± 1.16</td></tr><tr><td rowspan=1 colspan=1>HSV-1 HF</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>35.2 ± 1.03</td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>37.5 ± 1.08</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>31/32</td><td rowspan=1 colspan=1>37.9 ± 1.15</td></tr></table>

# f. Analytical Reactivity:

The analytical reactivity of the Simplexa™ HSV 1 & 2 Direct assay was evaluated using different strains of HSV-1 and HSV-2 that were not used in the determination of the limit of detection (LoD) for the assay. Quantified viral material was spiked into negative genital swab matrix containing male and female genital swabs using a single dilution and assayed in triplicate. The Simplexa™ HSV 1 & 2 Direct assay was able to detect other strains of HSV-1 and HSV-2 viruses.

<table><tr><td rowspan=2 colspan=1>HSVStrain/Isolate</td><td rowspan=2 colspan=1>Spiked Concentration[TCID50/mL]</td><td rowspan=1 colspan=2>Qualitative Result(#Detected/#Total)</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 KOS</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>HSV-1 F</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Isolate 1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Isolate 2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Isolate 3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# g. Analytical specificity:

# Cross-Reactivity

The analytical specificity of the Simplexa™ HSV 1 & 2 Direct assay was evaluated by testing the ability to exclusively identify HSV-1 and HSV-2 viruses with no crossreactivity to organisms that are closely related, or cause similar clinical symptoms or may be present on swabs of the genital region. Thirty six (36) potential crossreactants were spiked into negative genital swab matrix containing male and female genital swabs and assayed in triplicate. No cross-reactivity was observed.

<table><tr><td colspan="1" rowspan="2">No.</td><td colspan="1" rowspan="2">Potential Cross-Reactants</td><td colspan="1" rowspan="2">Tested Concentration</td><td colspan="2" rowspan="1">Qualitative Result(#Detected/#Total)</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">None (Baseline)</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">0/15</td><td colspan="1" rowspan="1">0/15</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Bacteroides ureolyticus*</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 X 106 IFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Cytomegalovirus AD169 strain</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">1/8</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Epstein Barr Virus (B95-8)</td><td colspan="1" rowspan="1">1.00 X 10° copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Escherichia coli O157H7</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Hepatitis B</td><td colspan="1" rowspan="1">1.00 X 105 IU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Hepatitis C</td><td colspan="1" rowspan="1">1.00 X 105 IU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">HHV-6 (Z29 Strain)</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">HHV-7 SB</td><td colspan="1" rowspan="1">1.00 X 103 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">HIV-1 IIIB</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">HIV-2 NIHZ**</td><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">HPV18 Recombinant</td><td colspan="1" rowspan="1">1.00 X 103 pfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Mycoplasma genitalium**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1.00 X 106 CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.00 X 10° cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Rubella</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA),ATCC 700699</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE), ATCC 29887</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Streptococcus pyogenes, M1</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1.00 X 106tachyzooites/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Treponema pallidum**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">1.00 X 106trophozoites/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1.00 X 106 CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">VZV</td><td colspan="1" rowspan="1">1.00 X 103 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="5" rowspan="1">* Microorganism was not available for testing; therefore in-silico NCBI BLAST analysis wasperformed and found no predicted cross reactivity.** Quantified material was not available to test; instead the vendor provided a culture fluid with aknown Ct value. The site was directed to dilute the stock to a relevant Ct value; 1:50 dilution factor.</td></tr></table>

# Interference

The performance of the Simplexa™ HSV 1 & 2 Direct assay was evaluated with potentially interfering substances that may be present on swabs of the genital region at the concentrations indicated in the table below. A total of 26 potentially interfering substances was tested in a low positive HSV-1 and HSV-2 sample (4 times LoD) in negative genital swab matrix containing male and female genital swabs and assayed in triplicate. No interference was observed.

<table><tr><td colspan="1" rowspan="2">Potential Interferent</td><td colspan="1" rowspan="2">InterferentConcentration</td><td colspan="3" rowspan="1">#Detected/#Total</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="1">Abreva cold sore treatment</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acyclovir Cream*</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">5/8</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Balneol Hygienic Cleansinglotion</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cidofovir</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole vaginal cream</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Denavir</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Douche</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Famciclovir</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Feces</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Gynol II (Contraceptive jelly)</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KY Jelly</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Monistat 1</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Monistat 3</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Preparation H Hemorrhoidcream</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Releev cold sore treatment</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Vagicaine Anti-Itch Cream</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Vagisil creme</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">VagiStat 1</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Valacyclovir</td><td colspan="1" rowspan="1">2.5 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">YeastGard Suppositories</td><td colspan="1" rowspan="1">7% w/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="5" rowspan="1">* One out of three (1/3) initial replicates and two out of five (2/5) confirmation replicates werenegative for HSV-2.</td></tr></table>

# Competitive Interference

Competitive interference was studied to evaluate the effects of clinically relevant coinfections with each of the analytes detected by the Simplexa™ HSV 1 & 2 Direct assay. The study assessed whether a high concentration of one virus in the sample could potentially affect the Simplexa™ HSV 1 & 2 Direct assay performance for another target present at low levels. A low positive sample was contrived at approximately 4 times LoD for each target (HSV-1 McIntryre strain and HSV-2 G strain), and a baseline Ct was determined for each sample. Each potential concomitant infecting virus was spiked into the low level sample and assayed in triplicate. Baseline sample results are also shown below. No competitive interference was observed as summarized in the following table.

<table><tr><td rowspan=1 colspan=2>Baseline (Low Level)</td><td rowspan=1 colspan=2>Competitive Interferent(High Concentration)</td><td rowspan=1 colspan=2>Qualitative Results(#Detected/#Total)</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>1.00 X 106</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>1.71 X 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# Inhibition by other Microorganisms

The Simplexa™ HSV 1 & 2 Direct assay was evaluated by testing the ability to identify HSV-1 and HSV-2 viruses when other potentially inhibitory organisms are present. The panel of thirty-six (36) potentially inhibitory organisms was individually spiked into a pool with a low concentration (approximately 4 times LoD) of HSV-1 and HSV-2 in genital swab matrix. Each microorganism sample was initially tested in triplicate and if any one of the replicates was “Not Detected” for either the HSV-1 or the HSV-2 targets then five additional replicates were tested to confirm if any inhibition was caused by the microorganism. If the majority $( > 4 / 8 )$ replicates were “Not Detected” then an inhibitory effect was determined. None of the microorganisms caused ${ > } 4 / 8$ of the replicates to be “Not Detected”. The results are summarized in the table below.

<table><tr><td colspan="1" rowspan="2">No.</td><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">TestedConcentration</td><td colspan="2" rowspan="1">Qualitative Result(#Detected/#Total)</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Baseline</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">15/15</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Bacteroides ureolyticus**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 X 106 IFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.00 X 105TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.00 X 105TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Epstein Barr Virus (B95-8)</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Escherichia coli O157H7</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Hepatitis B</td><td colspan="1" rowspan="1">1.00 X 105 IU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Hepatitis C</td><td colspan="1" rowspan="1">1.00 X 105 IU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">HHV-6 (Z29 Strain)</td><td colspan="1" rowspan="1">1.00 X 105TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">HHV-7 SB</td><td colspan="1" rowspan="1">1.00 X 105TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">HIV-1 IIIB</td><td colspan="1" rowspan="1">1.00 X 103 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">HIV-2 NIHZ*</td><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">HPV18 Recombinant</td><td colspan="1" rowspan="1">1.00 X 105 pfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Mycoplasma genitalium**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1.00 X 106 CCU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Rubella</td><td colspan="1" rowspan="1">1.00 X 105</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Staphylococcus aureus(MRSA), ATCC 700699</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE), ATCC 29887</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Streptococcus pyogenes, M1</td><td colspan="1" rowspan="1">1.00 X 106 cfu/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1.00 X 106tachyzooites/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Treponema pallidum**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">1.00 X 106trophozoites/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1.00 X 106 CCU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">VZV</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="5" rowspan="1">*Quantified material was not available to test; instead the vendor provided a culture fluid with a knownCt value. The site was directed to dilute the stock to a relevant Ct value; 1:50 dilution factor.**Microorganism was not available for testing.</td></tr></table>

# h. Sample stability studies:

Storage conditions were validated by spiking various transport media with HSV at concentrations ranging from 3 times LoD to 50 times LoD and testing at different storage temperatures and durations. The following transport media were evaluated: BD VTM, M4, M4RT, M5, M6, and UTM.

The studies indicated that samples should be transported on ice and can be stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ for up to 7 days post collection.

# i. Stability studies:

A real-time stability was completed over a 14 month time period and the data support a shelf-life claim of 12 months for the Simplexa™ HSV 1 & 2 Direct and the Simplexa™ HSV 1 & 2 Positive Control Pack.

# j. Carryover contamination:

The amplification carry-over for the Simplexa™ HSV 1 & 2 Direct assay was assessed from the Simplexa™ Flu A/B & RSV Direct (K120413) viral assay, and can be found on the FDA website. The study can be applied to the Simplexa™ HSV 1 &

2 Direct assays as the study is not analyte specific. In the Simplexa™ Flu A/B & RSV Direct (K120413), the amplification carry-over study searched for the presence of contamination in negative samples adjacent to strong positive samples. The study was designed by alternately placing high positive and negative samples on each disc. No evidence of carry-over contamination was observed.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies section of this document.

b. Matrix comparison: Not Applicable.

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable.

b. Clinical specificity: Not Applicable.

c. Other clinical supportive data (when a. and b. are not applicable):

Prospective Study

A total of 718 genital swab samples, was prospectively collected from patients with signs and symptoms of genital herpes simplex virus (HSV) infection from 6 geographically diverse locations. Of the 718 samples collected, 9 samples were removed from the analysis because they were either not tested or had invalid results with the 3 assays (Simplexa™ HSV 1 & 2 Direct, culture or bi-directional sequencing) that were used in the clinical study . Of the 709 remaining samples, 13 samples were removed from the analysis because there was insufficient volume to generate a final comparator result. A total of 696 samples, was used for the analysis. The clinical performance of the Simplexa HSV 1 & HSV 2 assay was evaluated by comparing the positive and negative percent agreement to a composite comparator algorithm consisting of culture, bi-directional sequencing and an FDA cleared NAAT. A positive result for HSV-1 and/or HSV-2 was determined by a positive test result in either the culture or the bi-directional sequencing. If both the culture and the bidirectional sequencing yielded positive results but disagreed in the differentiation of HSV-1 versus HSV-2, the results of the FDA cleared NAAT were used and a 2 out of 3 rule was followed to determine the type of the virus (e.g. if two of the methods were positive for HSV-1, the final comparator result was HSV-1 positive). All sites collected and tested the genital swab samples on the Simplexa™ HSV-1 and HSV-2 Direct and sent samples to a central lab for culture testing. For culture, each sample was tested for HSV- 2 first and, if positive for HSV-2, no further testing was performed. Samples that were HSV-2 culture negative were further tested for HSV-1 culture positivity. Dual positives could not be identified in the culture assay.

The available retained samples were sent to Focus Diagnostics and tested in a validated bi-directional sequencing assay. Of the 24 discordant samples that were positive for HSV-2 by the culture method but positive for HSV-1 by the bi-directional sequencing assay, 18 samples had valid results and 1 sample had an invalid result when tested on an FDA cleared NAAT. There were 5 samples that were not tested on the FDA cleared NAAT due to insufficient volume and therefore 6 out of 24 samples were excluded from the analysis. There were 2 samples that were positive for HSV-1 by the culture method but positive for HSV-2 by the bi-directional sequencing assay that were not tested on the FDA cleared NAAT for insufficient volume and therefore were excluded from analysis. The results for Simplexa™ HSV 1 & 2 Direct compared to the composite comparator algorithm are presented in the following tables.

Simplexa™ HSV 1 & 2 Direct Compared to Composite Comparator Result (HSV-1)   

<table><tr><td rowspan=2 colspan=1>Simplexa™ HSV 1 &amp; 2Direct Results (HSV-1)</td><td rowspan=1 colspan=3>Composite Comparator Result (HSV-1)</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>121</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>560</td><td rowspan=1 colspan=1>563</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>570</td><td rowspan=1 colspan=1>684</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.4% (111/114)95% CI: 92.5% - 99.1%</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>98.2% (560/570)95% CI: 96.8% - 99.0%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>91.7% (111/121)95% CI: 85.5% - 95.4%</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>99.5% (560/563)95% CI: 98.4% - 99.8%</td></tr></table>

Simplexa™ HSV 1 & 2 Direct Compared to Composite Comparator Result (HSV-2)   

<table><tr><td rowspan=2 colspan=1>SimplexaTM HSV 1 &amp; 2Direct Results (HSV-2)</td><td rowspan=1 colspan=3>Composite Comparator Result (HSV-2)</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>186</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>497</td><td rowspan=1 colspan=1>502</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>508</td><td rowspan=1 colspan=1>688</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.2% (175/180)95% CI: 93.7% - 98.8%</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.8% (497/508)95% CI: 96.2% - 98.8%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>94.1% (175/186)95% CI: 89.7% - 96.7%</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>99.0% (497/502)95% CI: 97.7% - 99.6%</td></tr></table>

# Retrospective Study

A total of 28 genital swab samples (14 positive HSV-1 and 14 positive HSV-2) was retrospectively collected from male patients with signs and symptoms of genital herpes simplex virus (HSV) infection and contained preselected positive and negative samples. The samples were tested at Focus Diagnostics using the Simplexa™ HSV 1 & 2 Direct and a validated bi-directional sequencing assay.

HSV-1 Validated Bi-Directional Sequencing Assay Results   

<table><tr><td rowspan=2 colspan=1>SimplexaTM HSV 1 &amp; 2Direct Results</td><td rowspan=1 colspan=3>Composite Comparator Result (HSV-1)</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>28</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>100.0% (14/14)95% CI: 78.5% -100.0%</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>92.9% (13/14)95% CI: 68.5% - 98.7%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>93.3% (14/15)95% CI: 70.2% - 98.8%</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>100.0% (13/13)95% CI: 77.2% - 100.0%</td></tr></table>

HSV-2 Validated Bi-Directional Sequencing Assay Results   

<table><tr><td rowspan=2 colspan=1>SimplexaTM HSV 1 &amp; 2Direct Results</td><td rowspan=1 colspan=3>Composite Comparator Result (HSV-2)</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>28</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>100.0% (14/14)95% CI: 78.5% - 100.0%</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>100.0% (14/14)95% CI: 78.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>100.0% (14/14)95% CI: 78.5% - 100.0%</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>100.0% (14/14)95% CI: 78.5% - 100.0%</td></tr></table>

4. Clinical cut-off:

Not Applicable.

5. Expected values/Reference range:

The observed expected values using the Simplexa™ HSV 1 & 2 Direct assay are presented below for patients 17 years of age or older from the prospective study population. The data is stratified by age, and gender and collection sites.

<table><tr><td colspan="1" rowspan="3">Gender</td><td colspan="1" rowspan="3">AgeGroup</td><td colspan="6" rowspan="1">Sample Demographics by SimplexaTM HSV 1 &amp; 2 Direct Assay</td></tr><tr><td colspan="1" rowspan="2">All</td><td colspan="5" rowspan="1">SimplexaTM HSV 1 &amp; 2 Direct Result</td></tr><tr><td colspan="1" rowspan="1">HSV-1 &amp;HSV-2 NotDetected</td><td colspan="1" rowspan="1">HSV-1Detected &amp;HSV-2 NotDetected</td><td colspan="1" rowspan="1">HSV-1 NotDetected &amp;HSV-2Detected</td><td colspan="1" rowspan="1">HSV-1 &amp;HSV-2 DualPositive</td><td colspan="1" rowspan="1">Not-Evaluable*</td></tr><tr><td colspan="1" rowspan="2">Female</td><td colspan="1" rowspan="1">17 Yearsof age to21 Yearsof age</td><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">41.8%(41/98)</td><td colspan="1" rowspan="1">25.5% (25/98)</td><td colspan="1" rowspan="1">30.6% (30/98)</td><td colspan="1" rowspan="1">0.0% (0/98)</td><td colspan="1" rowspan="1">2.0% (2/98)</td></tr><tr><td colspan="1" rowspan="1">More than21 Yearsof age</td><td colspan="1" rowspan="1">503</td><td colspan="1" rowspan="1">54.3%(273/503)</td><td colspan="1" rowspan="1">16.9% (85/503)</td><td colspan="1" rowspan="1">25.8%(130/503)</td><td colspan="1" rowspan="1">0.8% (4/503)</td><td colspan="1" rowspan="1">2.2% (11/503)</td></tr><tr><td colspan="1" rowspan="2">All</td><td colspan="5" rowspan="1">Simplexa™M HSV 1 &amp; 2 Direct Result</td></tr><tr><td colspan="1" rowspan="1">HSV-1 &amp;HSV-2 NotDetected</td><td colspan="1" rowspan="1">HSV-1Detected &amp;HSV-2 NotDetected</td><td colspan="1" rowspan="1">HSV-1 NotDetected &amp;HSV-2Detected</td><td colspan="1" rowspan="1">HSV-1 &amp;HSV-2 DualPositive</td><td colspan="1" rowspan="1">Not-Evaluable*</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">601</td><td colspan="1" rowspan="1">52.2%(314/601)</td><td colspan="1" rowspan="1">18.3%(110/601)</td><td colspan="1" rowspan="1">26.6%(160/601)</td><td colspan="1" rowspan="1">0.7% (4/601)</td><td colspan="1" rowspan="1">2.2% (13/601)</td></tr><tr><td colspan="1" rowspan="3">Male</td><td colspan="1" rowspan="1">17 Yearsof age to21 Yearsof age</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">42.9% (6/14)</td><td colspan="1" rowspan="1">21.4% (3/14)</td><td colspan="1" rowspan="1">35.7% (5/14)</td><td colspan="1" rowspan="1">0.0% (0/14)</td><td colspan="1" rowspan="1">0.0% (0/14)</td></tr><tr><td colspan="1" rowspan="1">More than21 Yearsof age</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">68.2%(45/66)</td><td colspan="1" rowspan="1">10.6% (7/66)</td><td colspan="1" rowspan="1">18.2% (12/66)</td><td colspan="1" rowspan="1">1.5% (1/66)</td><td colspan="1" rowspan="1">1.5% (1/66)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">63.8%(51/80)</td><td colspan="1" rowspan="1">12.5% (10/80)</td><td colspan="1" rowspan="1">21.3% (17/80)</td><td colspan="1" rowspan="1">1.3% (1/80)</td><td colspan="1" rowspan="1">1.3% (1/80)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="1">All</td><td colspan="1" rowspan="1">681</td><td colspan="1" rowspan="1">53.6%(365/681)</td><td colspan="1" rowspan="1">17.6%(120/681)</td><td colspan="1" rowspan="1">26.0%(177/681)</td><td colspan="1" rowspan="1">0.7% (5/681)</td><td colspan="1" rowspan="1">0.7%(5/681)**</td></tr><tr><td colspan="8" rowspan="1">Sle ot tes uratuexuri stbu valeultai utel failure, insufficient specimen volume, daily PC/NTC failure.**Excluded from this table are 35 samples from patients less than 7 years of age and samples where the the patient was not available.</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Hypothetical Prevalence</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PPV*</td><td rowspan=1 colspan=1>NPV**</td><td rowspan=1 colspan=1>PPV*</td><td rowspan=1 colspan=1>NPV**</td></tr><tr><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>35.3%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30.9%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>52.5%</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>62.6%</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>57.7%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>74.0%</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>69.9%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>83.1%</td><td rowspan=1 colspan=1>99.7%</td></tr><tr><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>90.5%</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>88.6%</td><td rowspan=1 colspan=1>99.5%</td></tr><tr><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>93.1%</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>99.3%</td></tr><tr><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>98.8%</td></tr><tr><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>97.2%</td></tr></table>

<table><tr><td>*Positive Predictive Value (PPV) was calculated using: (Sensitivity x Prevalence)/(Sensitivity x Prevalence + [1 - Specificity] x [1 - Prevalence]).</td></tr><tr><td>**Negative Predictive Value (NPV ) was calculated using:</td></tr><tr><td></td></tr><tr><td>(Specificity x [1 - Prevalence])/([1 - Sensitivity] x Prevalence + Specificity x [1 - Prevalence]).</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.