# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K151906 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequences from Herpes Simplex Virus type 1 (HSV 1) and Herpes Simplex Virus type 2 (HSV 2)

# D. Type of Test:

An in vitro molecular diagnostic test for the direct, qualitative detection and differentiation of HSV 1 and HSV 2 DNA in cutaneous or mucocutaneous lesion specimens.

E. Applicant: Luminex Corporation   
F. Proprietary and Established Names: ARIES® HSV 1&2 Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3309

2. Classification: Class II

3. Product code: PGI OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The ARIES® HSV 1&2 Assay is a real-time polymerase chain reaction (PCR) based qualitative in vitro diagnostic test for the direct detection and differentiation of Herpes Simplex Virus 1 and 2 (HSV 1 and HSV 2) DNA in cutaneous or mucocutaneous lesion specimens from symptomatic patients. The test is indicated for use as an aid in diagnosis of HSV infection in symptomatic patients. The ARIES® HSV 1&2 Assay is indicated for use on the ARIES® System.

Warning: The ARIES® HSV 1&2 Assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay is not intended to be used for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

The ARIES® HSV 1&2 Assay uses the ARIES® System that is capable of automated nucleic acid extraction and purification from a clinical sample, real-time PCR detection and differentiation of nucleic acid sequences, and data analysis.

The $\boldsymbol { \mathrm { A R I E S } } ^ { \mathbb { B } }$ System instrument uses the ARIES Software v1.0. The $\mathrm { \Delta A R I E S ^ { \mathrm { \textregistered } } }$ instrument software provides the interface between the ARIES Software v1.0 and the ARIES system hardware.

Note: Refer to K151917 for additional information regarding ARIES® System and related Software.

# I. Device Description:

The ARIES® HSV 1 & 2 Assay is a polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test for the direct detection and differentiation of herpes simplex virus (HSV) DNA from cutaneous and mucocutaneous lesion swab specimens.

The cutaneous and mucocutaneous lesion swab specimens are collected in Copan Universal Transport Medium and transported to the laboratory. The specimen is pipetted into a cassette specific to the ARIES® HSV 1&2 Assay. In the cassette, the specimen is lysed and nucleic acid is extracted using the HSV 1&2 Assay Kit and $\mathrm { \sf A R I E S } ^ { \otimes }$ System. An extractable sample processing control (SPC) target is present in the ARIES® HSV 1&2 assay cassette and is processed with the specimen. The Ct value of the SPC serves to verify proper specimen lysis and nucleic acid extraction, to identify PCR inhibition, if any, and verify proper function of the extraction system and real-time instrument. The Tm value of the SPC is used as a reference for determining the target Tm.

The extracted nucleic acid is transferred via magnetic beads to the ARIES® HSV 1 & 2 Kit lyophilized PCR reagents in the cassette that contain a primer pair specific to HSV 1 and HSV 2 and a primer pair specific to the SPC sequence. The specific primer pairs are labeled with distinct fluorophore labels. PCR amplification is performed and assay fluorescence is monitored on the ARIES® System. Incorporation of the quencher-labeled nucleotide causes a decrease in assay fluorescence. Following amplification, the reaction is slowly heated and fluorescence is monitored. The strands of the amplification products will separate at a specific melting temperature (Tm) that is determined by an increase in fluorescence as the strands are separated. The sequences between the PCR primer binding sites of the HSV 1 and HSV 2 amplicons have different base compositions that are distinguished by their different Tm values. The instrument fluorescence output is analyzed and test results are determined using the ARIES® HSV 1 & 2 Kit assay protocol file. Total assay time, including extraction and PCR cycling, takes approximately two hours. A printed results report is generated.

The ARIES® HSV 1&2 Assay consists of two kits:

ARIES® HSV 1&2 Assay Cassette Kit - contains 24 assay cassettes which contain the necessary reagents for sample extraction, nucleic acid purification and amplification. ARIES® HSV 1&2 Assay Protocol File Kit - contains the assay protocol file, package insert and Quick Guide which ship separately on a USB flash drive as part of the ARIES® HSV 1&2 Assay Protocol File Kit.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

illumigene® HSV 1&2 DNA Amplification Assay (Meridian Bioscience, Inc.)

Reference Method:

ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) for clinical evaluation (K971662)

2. Predicate 510(k) number(s): K151046

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">DeviceCharacteristic</td><td colspan="1" rowspan="1">ARIES® HSV 1 &amp; 2 Assay(New Device)</td><td colspan="1" rowspan="1">illumigene® HSV 1&amp;2 DNA AmplificationAssay(Predicate Device - K151046</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The ARIES® HSV 1&amp;2 Assay is a real-time polymerase chain reaction (PCR)based qualitative in vitro diagnostic testfor the direct detection and differentiationof Herpes Simplex Virus 1 and 2 (HSV 1and HSV 2) DNA in cutaneous ormucocutaneous lesion specimens fromsymptomatic patients. The test is indicatedfor use as an aid in diagnosis of HSVinfection in symptomatic patients. TheARIES® HSV 1&amp;2 Assay is indicated foruse on the ARIES® System.Warning: The ARIES® HSV 1&amp;2 Assayis not FDA cleared for use withcerebrospinal fluid (CSF). The assay isnot intended to be used for prenatalscreening.</td><td colspan="1" rowspan="1">TheillumigeneHSV 1&amp;2DNAamplification assay, performed on theillumipro-10Tm, is a qualitative in vitrodiagnostic test for the direct detection anddifferentiation of herpes simplex virustype 1 (HSV-1) and herpes simplex virustype 2 (HSV-2) DNA in cutaneous andmucocutaneous lesion specimens frommale and female patients suspected ofHerpetic infections.illumigene HSV 1&amp;2 utilizes loop-mediated isothermal DNA amplification(LAMP) technology to detect HSV-1 andHSV-2 by targeting segments of theherpes simplex virus 1 and herpes simplexvirus 2 genomes. Results from illumigeneHSV 1&amp;2 are used as an aid in thediagnosis of HSV  infection   insymptomatic patients.The assay is intended for use in hospital,reference or state laboratory settings.This device is not intended fornonlaboratory point-of-care use.Warning: illumigene HSV 1&amp;2 is notFDA cleared for use with cerebrospinalfluid (CSF) or to aid in the diagnosis ofHSV infections of the central nervoussystem (CNS). The device is not intendedfor prenatal screening.</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Male and female cutaneous andmucocutaneous lesion swab specimens</td><td colspan="1" rowspan="1">Male and female cutaneous andmucocutaneous lesion swab specimens</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">DNA amplification</td><td colspan="1" rowspan="1">DNA amplification</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detection and differentiationof HSV-1 and HSV-2 DNA</td><td colspan="1" rowspan="1">Qualitative detection and differentiation ofHSV-1 and HSV-2 DNA</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">DeviceCharacteristic</td><td colspan="1" rowspan="1">ARIESHSV 1 &amp; 2 Assay(New Device)</td><td colspan="1" rowspan="1">illumigene® HSV 1&amp;2 DNA AmplificationAssay(Predicate Device - K151046</td></tr><tr><td colspan="1" rowspan="1">Sampleextraction andAmplificationInstrumentation</td><td colspan="1" rowspan="1">Automated sample extraction; Real-timePCR amplification/detection using theLuminex ARIES® System</td><td colspan="1" rowspan="1">Manual sample preparation; isothermalLoop Mediated Amplification (LAMP)using the illumipro-10TM.</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">Pairs fluorescent-labeled primers withquencher labeled nucleotides. Measuresdecrease in assay fluorescence with eachPCR cycle.</td><td colspan="1" rowspan="1">Visible Light Transmission (Turbidity).</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

The ARIES® HSV 1&2 Assay chemistry is based on an expanded genetic alphabet technology, using the synthetic DNA base pair 2'-deoxy-5-methyl-isocytidine (iC): 2'-deoxyisoguanosine (iG). The isobases (iC and iG) pair specifically with each other and not with natural nucleotides. In addition, isobases are efficiently incorporated during PCR. During PCR amplification, a quencher-modified iGTP is incorporated by the polymerase opposite an iC and a fluorophore reporter attached to a PCR primer. If the target is present and is amplified, assay fluorescence decreases with every cycle as amplification product accumulates. The decrease in assay fluorescence is monitored in real time using the Luminex ARIES® Instrument. Following PCR, the amplification products are thermally denatured and assay fluorescence is monitored. The strands of the amplification products are separated and assay fluorescence increases, thus determining the melting temperature $\mathrm { ( T m ) }$ profile of the amplicon. The sequences between the PCR primer binding sites of the HSV-1 and HSV-2 amplicons have different base compositions that are distinguished by their different melting temperatures using the ARIES® HSV 1&2 Assay Analysis Software.

The specimen is added to the sample chamber of an ARIES® HSV 1&2 Assay cassette. The cassette is then placed into an ARIES® System magazine. A magazine can hold up to six cassettes. The magazine is inserted into an ARIES® System, which can process two magazines simultaneously. A barcode on top of the HSV 1&2 Assay cassette is automatically scanned by the ARIES® System, associating a preloaded ARIES® HSV 1&2 Assay protocol file with the cassette. The HSV 1&2 Assay protocol file contains the necessary parameters to run the cassette, analyze data, and generate reports.

Once a run is started, the Sample Processing Control (SPC) is automatically added to the sample chamber of the cassette to control for sample lysis, recovery of extracted nucleic acid, detection of inhibitory substances, and confirmation of PCR reagent integrity. Sample and SPC lysis, as well as isolation and purification of nucleic acids, are automated within the ARIES® System and the ARIES® HSV 1&2 Assay cassette. Purified nucleic acids are automatically transferred to the cassette’s PCR tube that contains the lyophilized HSV 1&2 Master Mix for the PCR amplification step. The HSV 1&2 Master Mix contains a primer pair specific to HSV 1 and HSV 2, and a second primer pair specific to the SPC sequence.

# Interpretation of Sample Results

The ARIES® analysis software determines results for the sample and the sample processing control (SPC) based on the amplification cycle (Ct) value, the melting temperature $\mathrm { ( T _ { m } ) }$ value, and $\mathrm { T } _ { \mathrm { m } }$ threshold values provided in the assay protocol file. All assay outcomes are described below.

Interpretation of Sample Results   

<table><tr><td rowspan=2 colspan=1>Example</td><td rowspan=1 colspan=2>SPC</td><td rowspan=1 colspan=3>HSV</td><td rowspan=2 colspan=1>Call</td></tr><tr><td rowspan=1 colspan=1>Ct Value</td><td rowspan=1 colspan=1>Tm Value</td><td rowspan=1 colspan=1>Ct Value</td><td rowspan=1 colspan=1>HSV 1Tm Value</td><td rowspan=1 colspan=1>HSV 2Tm Value</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>HSV 1&amp;2</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>HSV 1 Positive</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>HSV 2 Positive</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=1>HSV 1&amp;2Negative</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>&gt;a</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>&gt;a/-</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=4 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td></tr></table>

N/A: Not applicable. All possible outcomes will result in the same call. a Greater than the Ct cut-off value.

# Invalid Results

In case of an “Invalid” result, re-test the sample with a new assay cassette. If the problem is unresolved, Luminex Technical Support should be contacted.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility:

The reproducibility of the ARIES® HSV 1&2 Assay was evaluated by testing one lot of ARIES® HSV 1&2 Assay Cassettes on two ARIE $S ^ { \mathfrak { B } }$ instruments by two operators at each of three sites on five non-consecutive days.

The panel members were formulated with a single target present (HSV-1 MacIntyre strain or HSV-2 MS strain) at three concentrations: moderate positive (4 X LoD for both HSV 1 and HSV 2), low positive (1 X LoD for both HSV 1 and HSV 2), and high negative (0.1 X LoD for HSV 1 and 0.4 X LoD for HSV 2). A true negative sample, where no HSV was added, was also prepared using viral transport medium. Each panel member was tested in replicates of three, for five days, at three study sites. Testing at each site was performed by two operators and each operator ran the panel once a day. The results from the reproducibility study for the ARIES® HSV 1&2 Assay are presented in the table below.

# Reproducibility Panel Results

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>Site 1</td><td rowspan=1 colspan=4>Site 2</td><td rowspan=1 colspan=4>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedRescults</td><td rowspan=1 colspan=1>AvgTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>Avg TmDeflection</td><td rowspan=1 colspan=1>AgreementwiithExpectedResults</td><td rowspan=1 colspan=1>AvgTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>Avg TmDeflectionb</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AvgTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>Avg TmDeflection</td></tr><tr><td rowspan=1 colspan=1>HSV-1 ModerateT Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.25E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1>2.56E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>0.18%</td><td rowspan=1 colspan=1>2.72E+06</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96.0-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1LowTPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.04E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.24E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.45E+06</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96.0-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1High Negative</td><td rowspan=1 colspan=1>11/30</td><td rowspan=1 colspan=1>85.4</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>1.39E+06</td><td rowspan=1 colspan=1>9/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.20%</td><td rowspan=1 colspan=1>2.33E+06</td><td rowspan=1 colspan=1>9/30</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>2.06E+06</td><td rowspan=1 colspan=1>29/90 (32.2%)</td><td rowspan=1 colspan=1>22.8-42.9%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 ModeratePositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.9</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>2.17E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.52E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.15%</td><td rowspan=1 colspan=1>2.43E+06</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96.0-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.11%</td><td rowspan=1 colspan=1>1.95E+06</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>2.23E+06</td><td rowspan=1 colspan=1>30/31</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>2.04E+06</td><td rowspan=1 colspan=1>89/91 (97.8%)</td><td rowspan=1 colspan=1>92.3-99.7%</td></tr><tr><td rowspan=1 colspan=1>HSV-2HighNNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.19%</td><td rowspan=1 colspan=1>1.75E+06</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.14%</td><td rowspan=1 colspan=1>1.98E+06</td><td rowspan=1 colspan=1>23/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.15%</td><td rowspan=1 colspan=1>1.94E+06</td><td rowspan=1 colspan=1>83/90 (92.2%)</td><td rowspan=1 colspan=1>84.6-96.8%</td></tr><tr><td rowspan=1 colspan=1>HSV1&amp;2Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>76.4</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=1>2.76E+05</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>76.3</td><td rowspan=1 colspan=1>0.24%</td><td rowspan=1 colspan=1>3.11E+05</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>76.3</td><td rowspan=1 colspan=1>0.68%</td><td rowspan=1 colspan=1>3.34E+05</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96.0-100%</td></tr></table>

a Agreement with expected results for the HSV 1&2 negative reflects SPC positivity since no HSV 1 or HSV 2 was detected. Expected result for HSV 1 Moderate Positive target was $100 \%$ HSV 1 Positive; for HSV 1 Low Positive was approximately $9 5 \%$ HSV 1 Positive; for HSV 1 High Negative was $20 \%$ to $80 \%$ HSV 1 Positive; for HSV 2 Moderate Positive target was $100 \%$ HSV 2 Positive; for HSV 2 Low Positive was approximately $9 5 \%$ HSV 1 Positive; for HSV 2 High Negative was $20 \%$ to $80 \%$ Positive; and for HSV 1&2 Negative was $100 \%$ HSV 1&2 Negative.

b Average $\mathrm { T } _ { \mathrm { m } }$ deflection (RFU) was calculated using all of the positive replicates for that target type. Average $\mathrm { T } _ { \mathrm { m } }$ deflection for the HSV 1&2 Negative reflects SPC $\mathrm { T } _ { \mathrm { m } }$ deflection since no HSV 1 or HSV 2 was detected.

# b. Precision (Within-Laboratory Repeatability)

The repeatability of the ARIES® HSV 1&2 Assay was evaluated using the same panel described in the reproducibility study above. The seven member panel was tested by two operators performing testing across multiple ARIES® instruments using one lot of ARIES® HSV 1&2 Assay Cassettes. Testing was performed for 10 days and included a total of 216 replicates used in assessing repeatability. Results of the repeatability study for the ARIES® HSV 1&2 Assay performed at one sites are presented in the table below.

# Repeatability Panel Resultsa

<table><tr><td rowspan=1 colspan=1>Target Type</td><td rowspan=1 colspan=1>Agreementwith ExpectedResultsb</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td><td rowspan=1 colspan=1>AverageTm</td><td rowspan=1 colspan=1>%CoefficientofVariationTm</td><td rowspan=1 colspan=1>Average TmDeflection</td></tr><tr><td rowspan=1 colspan=1>HSV 1Moderate Positive</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0- 100%</td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>3.28E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 1Low Positive</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0- 100%</td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>0.13%</td><td rowspan=1 colspan=1>2.88E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 1High Negative</td><td rowspan=1 colspan=1>45.80% (33/72)</td><td rowspan=1 colspan=1>34.0 - 58.0%</td><td rowspan=1 colspan=1>85.4</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1>2.18E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 2Moderate Positive</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0- 100%</td><td rowspan=1 colspan=1>87.9</td><td rowspan=1 colspan=1>0.16%</td><td rowspan=1 colspan=1>3.16E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 2Low Positive</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0- 100%</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.15%</td><td rowspan=1 colspan=1>2.75E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 2High Negative</td><td rowspan=1 colspan=1>97.40% (76/78)</td><td rowspan=1 colspan=1>91.0 - 99.7%</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>2.39E+06</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2Negative</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0- 100%</td><td rowspan=1 colspan=1>76.5</td><td rowspan=1 colspan=1>0.66%</td><td rowspan=1 colspan=1>4.41E+05</td></tr></table>

a An overall invalid rate of $0 . 8 \%$ (4/514) was observed.

b Expected result for HSV 1 Moderate Positive target was $100 \%$ HSV 1 Positive; for HSV 1 Low Positive was approximately $9 5 \%$ HSV 1 Positive; for HSV 1 High Negative was $20 \%$ to $80 \%$ HSV 1 Positive; for HSV 2 Moderate Positive target was $100 \%$ HSV 2 Positive; for HSV 2 Low Positive was approximately $9 5 \%$ HSV 2 Positive; for HSV 2 High Negative was $20 \%$ to $80 \%$ HSV 2 Positive; and HSV 1&2 Negative was $100 \%$ HSV 1&2 Negative.

c Average $\mathrm { T } _ { \mathrm { m } }$ deflection (RFU) was calculated using all of the positive replicates for that target type. Average $\mathrm { T } _ { \mathrm { m } }$ deflection for the HSV 1&2 Negative reflects SPC $\mathrm { T } _ { \mathrm { m } }$ deflection since no HSV 1 or HSV 2 was detected.

c. Linearity/assay reportable range: N/A d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Quality Control

Each ARIES® assay cassette contains a Sample Process Control (SPC), which is processed with the sample and analyzed during the amplification reaction. The SPC verifies sample lysis, nucleic acid extraction, and proper reagent, cassette, ARIES® System, and assay protocol performance. The SPC has a known melting temperature (Tm) range and Ct range. Each time an assay is run, the system measures the temperature and fluorescence intensity of the SPC control to ensure the thermal and optical subsystems have remained in calibration.

# External Controls

External controls should be tested according to guidelines or requirements of local, provincial and/or federal regulations or accreditation organizations. A reference HSV 1 and HSV 2 strain or well characterized HSV 1 and HSV 2 clinical isolates may be used as positive controls. Universal Viral Transport Medium may be used as a negative control. The Luminex ARIES® HSV 1&2 Assay Cassette Kit does not include external positive and negative control.

# e. Detection limit:

A Limit of Detection (LoD) study was performed to evaluate the analytical sensitivity of the ARIES® HSV 1&2 Assay using two representative strains of HSV 1 (MacIntyre and F) and two representative strains of HSV 2 (MS & G). Preliminary LoD concentrations were determined by performing a six-point, five-fold dilution series in Copan Universal Transport Media of each quantified $\mathrm { ( T C I D 5 0 / m L ) }$ strain. The observed LoD of a HSV strain was determined as the lowest concentration that had a positivity rate of $\ge 9 5 \%$ .

The LoD concentrations determined in the preliminary study were confirmed with the same HSV 1 and HSV 2 reference strains diluted to the preliminary LoD concentrations and tested with twenty-four (24) replicates. The final LoDs are presented in the table below.

Limit of Detection of the ARIES® HSV 1&2 Assay   

<table><tr><td rowspan=1 colspan=1>HSV</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoDConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positivity</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>MacIntyre</td><td rowspan=1 colspan=1>7.11E+03</td><td rowspan=1 colspan=1>24/24 (100%)</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>16.5</td><td rowspan=1 colspan=1>23/24 (95.8%)</td></tr><tr><td rowspan=2 colspan=1>HSV 2</td><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>24/24 (100%)</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>24/24 (100%)</td></tr></table>

The final assay LoD claim is $7 . 1 1 \mathrm { E } { + 0 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV 1 (MacIntyre) and 2.7 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for HSV 2 (MS). The analytical studies were performed using these two strains based on the final assay LoD claim.

# f. Analytical specificity:

A study was performed to evaluate cross reactivity and interference of the ARIES® HSV 1&2 Assay with 61 microorganisms that might be found in cutaneous and mucocutaneous lesion specimens. The effect of potential cross reactivity or interference was evaluated by testing five replicates of each HSV 1, HSV 2 near the device’s Limit of Detection (LoD), and negative replicates (Copan UTM) spiked with 61 potential cross reacting organisms. Bacteria were tested at $1 0 ^ { 6 } \mathrm { c f u / m l }$ or higher for bacteria and $1 0 ^ { 5 } \mathrm { \ p f u / m l }$ or higher for viruses. At the tested concentrations of the organisms, the organisms do not cross react or interfere with the assay. All HSV positive results were $100 \%$ positive and all HSV negative results were $100 \%$ negative.

Cross-Reacting and Microbial Interference Panel   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">9.27 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">4.2 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.74 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">7.87 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.8 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">4.9 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (AD169 Strain)</td><td colspan="1" rowspan="1">1.15 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">7.43 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">4.57 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Type 71)</td><td colspan="1" rowspan="1">4.17 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus (B95-8 Strain)</td><td colspan="1" rowspan="1">9.27 x 107 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5.13 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">5.43 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Hepatitis A Virus</td><td colspan="1" rowspan="1">8.47 x 102 IU/mL</td></tr><tr><td colspan="1" rowspan="1">Hepatitis B Virus</td><td colspan="1" rowspan="1">5.62 x 108 IU/mL</td></tr><tr><td colspan="1" rowspan="1">HIV-1</td><td colspan="1" rowspan="1">1.05 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 6 virus (Z29 Strain)</td><td colspan="1" rowspan="1">4.17 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 7 virus (SB Strain)</td><td colspan="1" rowspan="1">1.15 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Papilloma virus</td><td colspan="1" rowspan="1">1.68 x 109 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">2.00 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">2.70 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">3.18 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella cartarrhalis</td><td colspan="1" rowspan="1">9.90 x 105 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">3.6 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">1.4 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma salivarium</td><td colspan="1" rowspan="1">4.7 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">5.73 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">3.7 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">2.10 x 108 cfu /mL</td></tr><tr><td colspan="1" rowspan="1">Rubella virus</td><td colspan="1" rowspan="1">1.26 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Salmonella enteritidis</td><td colspan="1" rowspan="1">2.08 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">4.07 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.42 x 109 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">3.47 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.60 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">6.60 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">8.67 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">6.6 x 105 tachyzoites/mL</td></tr><tr><td colspan="1" rowspan="1">Treponema pallidum</td><td colspan="1" rowspan="1">9.8 x 106 genome copies/mL</td></tr><tr><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">4.21 x 105 trophozoites/mL</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster virus</td><td colspan="1" rowspan="1">2.45 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffi</td><td colspan="1" rowspan="1">8.27 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza type B</td><td colspan="1" rowspan="1">5.33 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">6.28 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides Serogroup A</td><td colspan="1" rowspan="1">7.07 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">4.10 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">5.73 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">4.37 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">9.2 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">7.47 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">2.87 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">2.15 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">10 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 2</td><td colspan="1" rowspan="1">5.01 x 105 U/mL</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">1.78 x 107 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">6.3 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">1.42 x 108 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">N/Aa</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">2.05 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisii</td><td colspan="1" rowspan="1">&gt;103 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Simian Virus type 40</td><td colspan="1" rowspan="1">2.8 x 106 TCID50/mL</td></tr></table>

a Concentration information not available

# g. Interfering Studies

This study was performed to evaluate potential interference with the $\mathrm { \Delta A R I E S ^ { \otimes } }$ HSV 1&2 Assay with a panel of twenty-eight (28) potential interfering substances. All of the potentially interfering substances were tested at concentrations at or above physiological levels or typical usage levels with HSV 1 (MacIntyre) and HSV 2 (MS) strains. The study was carried out in the presence of HSV 1 and HSV 2 near the device’s limit of detection (LoD) to evaluate potential interference with the detection of the HSV 1 and HSV 2 targets. Each potentially interfering substance was tested in 5 replicates of each HSV 1, HSV 2, and negative samples (Copan UTM). All HSV positive results were $100 \%$ positive and all negative results were $100 \%$ negative. No interference was observed with any of the substances tested.

Interfering Substance Panel   

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1>Abreva (Docosanol)</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Acyclovir (Acycloguanosine)</td><td rowspan=1 colspan=1>2.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Buffy Coat</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Carmex Cold Sore Lip Balm</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>7.0 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole 3 Vaginal Cream</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Toothpaste</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Anti-itch cream(Benzalkonium Chloride)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Cidofovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Douche</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Foscarnet</td><td rowspan=1 colspan=1>2.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ganciclovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Valganciclovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Lip Clear Lysine+</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Listerine</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Male Urine</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Female Urine</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Monistat 1</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Monistat 3</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Releev Cold Sore Treatment</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>K-Y Brand Jelly</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Spermicide</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Tioconazole</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Vagisil Cream</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>YeastGard</td><td rowspan=1 colspan=1>1%</td></tr></table>

# Fresh Specimen Stability:

The performance of the ARIES® HSV 1&2 Assay was assessed for fresh specimen stability when stored at $2 - 8 ^ { \circ } \mathrm { C }$ . The stability was assessed by testing 6 replicates of each of 7 HSV target concentrations across 6 different time points. The contrived specimens were prepared by spiking cultured organism into the negative clinical matrix (Copan Universal Transport Medium) at different levels. The concentrations tested were a moderate positive, low positive and high negative for HSV 1 and HSV 2 as well as a negative concentration. Moderate positive specimens gave the expected result of $100 \%$ positivity, low positive specimens gave the expected result of approximately $9 5 \%$ positivity (HSV $1 ; 9 7 . 2 \%$ positive and HSV 2 $100 \%$ positive) and high negative specimens gave the expected result of $20 \text{‰}$ positivity (HSV 1: $2 5 . 0 \%$ positive and HSV 2: $6 3 . 8 \%$ positive). Negative specimens gave the expected result of $100 \%$ negativity. The data from this stability study support the claim in the package insert that the fresh specimens for the ARIES® HSV 1&2 Assay can be held at $2 - 8 ^ { \circ } \mathrm { C }$ for up to 15 days.

# Frozen Specimen Stability:

The objective of the frozen specimen stability was to evaluate the stability of specimens when stored at ${ } _ { - 6 5 } { } ^ { \circ } \mathrm { C }$ to ${ } _ { - 9 5 } { } ^ { \circ } \mathrm { C }$ . This was assessed by testing 6 replicates of each of 7 HSV contrived target concentrations in Copan Universal Transport Medium across 7 different time points extending out to 12 months. The concentrations used for testing were a moderate positive, low positive and high negative concentration for HSV 1 and HSV 2 as well as a negative specimen. The data up to 3 months has been collected with all targets yielding the expected result. Moderate positive specimens are $100 \%$ positive, low positive specimens are positive approximately $9 5 \%$ of the time, high negative specimens are positive $2 0 \mathrm { ~ - ~ } 8 0 \%$ of the time and negative specimens are negative $100 \%$ of the time. The specimens are stable for up to 3 months when stored at ${ } _ { - 6 5 } { } ^ { \circ } \mathrm { C }$ to ${ } _ { - 9 5 } { } ^ { \circ } \mathrm { C }$ .

Note: The package insert claims that the specimens can be stored refrigerated at $2 ^ { \circ } \mathrm { C }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 15 days from the date of collection; if specimens will be used after 15 days from the date of collection, they should be stored frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ .

# i. Reagent Stability:

The objective of ARIES $^ \mathrm { \textregistered }$ HSV 1&2 Assay Cassette real time stability testing was to evaluate the stability of ARIES $^ \mathrm { \textregistered }$ HSV 1&2 Assay Cassette in order to establish a shelf life. This was assessed by testing 4 replicates of HSV 1, 4 replicates of HSV 2 and 4 replicates of negative (Copan UTM) targets on three different lots of ARIES $^ \mathrm { \textregistered }$ HSV 1&2 Assay Cassettes stored at 2 different temperatures $\left( 2 - 8 ^ { \circ } \mathrm { C } \right.$ and $2 5 ^ { \circ } \mathrm { C }$ ) at 10 different time points extending out to 19 months. Data up to 3 months has been collected and to date all targets for all lots and all storage temperatures have given the expected result. HSV 1 replicates are $100 \%$ HSV 1 Positive, HSV 2 replicates are $100 \%$ HSV 2 Positive and negative replicates are $100 \%$ HSV 1&2 Negative. Therefore, ARIES $^ \mathrm { \textregistered }$ HSV 1&2 Assay Cassettes are stable for 3 months when stored at both $4 \mathrm { { } ^ { \circ } C }$ and Room Temperature $( 2 5 ^ { \circ } \mathrm { C } )$ .

# $j$ . Co-infection

A study was designed to evaluate the ability of the ARIES® HSV 1&2 Assay to detect HSV-1 and HSV-2 analytes when both are present in one specimen. Analytes were tested at high (200X LoD) and low concentrations (5X LoD) using 12 replicates. The ARIES® HSV 1&2 Assay may not detect a co-infection of HSV 1 and HSV 2 in cases where the two virus types are not equally represented in clinical specimens. Coinfections were only detected when both analytes were present at 200X LoD. An HSV analyte at 5X LoD was not detected in the presence of a different HSV analyte at 200X LoD.

# k. Carry-over/Cross Contamination

Carry-over and cross contamination for the ARIES® HSV 1&2 Assay was assessed by testing fifteen (15) high positive HSV 1 samples, 15 high positive HSV 2 samples and thirty (30) HSV negative samples (Copan UTM). Samples were tested in an alternating pattern with high positive samples run adjacent to negative samples across ten (10) consecutive runs. No carry-over and cross contamination was observed. The overall percent agreement was $100 \%$ for positive and negative samples.

l. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

The clinical performance evaluation was performed against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

b. Matrix comparison: N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A b. Clinical Specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable): The performance of the ARIES® HSV 1&2 Assay was compared with the ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance

The clinical performance of the ARIES® HSV 1&2 Assay was assessed at three (3) geographically diverse clinical sites in the United States. A total of 1963 left-over clinical specimens from symptomatic male and female patients were included in the clinical study. Of these, 1500 specimens were prospectively collected (all comers). The remaining 463 were pre-selected for cutaneous and mucocutaneous lesion types that were under-represented in the initial prospective sample set. Additional oral and nasal lesions specimen lesion were also tested. All of the pre-selected specimens were also prospectively collected. Of the 1963 specimens tested, fifty-five (55) specimens were lesion sources from anatomical sites that could not be determined, four (4) specimens remained invalid upon re-testing by the ARIES® HSV 1&2 Assay and three were unavailable for re-testing. All of these 62 specimens were excluded from the clinical performance analysis.

The reference/comparative method used to evaluate the clinical performance of the ARIES® HSV 1&2 Assay was the ELVIS® HSV-ID and $\mathrm { D } ^ { 3 }$ Typing Test System. Because the ELVIS method provides no information on HSV 1 patient infected status (positive or negative) in specimens that test positive for HSV 2, all specimens that were positive for HSV 2 by the ELVIS® HSV-ID and $\mathrm { D } ^ { 3 }$ Typing System were excluded from the analysis of HSV 1 clinical performance.

A total of 448 cutaneous lesions specimens were tested. One hundred and one (101) specimens that were positive for HSV 2 by the ELVIS® HSV-ID and D3 Typing System were excluded from the analysis of HSV 1 clinical performance.

A total of 1453 mucocutaneous lesions specimens were tested. Two hundred and sixty three (263) specimens that were positive for HSV 2 by the ELVIS® HSV-ID and D3 Typing System were excluded from the analysis of HSV 1 clinical performance.

The performance of ARIES® HSV 1&2 Assay when compared to ELVIS® viral culture is summarized for cutaneous and mucocutaneous lesions in the tables below.

Summary of HSV 1 Results for Cutaneous Lesions $\scriptstyle ( \mathbf { N } = 3 4 7 )$   

<table><tr><td rowspan=1 colspan=1>ARIESHSV 1&amp;2 Assay</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>274</td><td rowspan=1 colspan=1>279</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>291</td><td rowspan=1 colspan=1>347</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>91.1% (51/56)</td><td rowspan=1 colspan=1>80.4% - 97.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>94.2% (274/291)</td><td rowspan=1 colspan=1>90.8% - 96.6%</td><td rowspan=1 colspan=1></td></tr></table>

1Thirteen (13) HSV 1 ARIES positive specimens that were negative by the reference method were positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. The remaining four (4) false positive specimens were negative for both HSV 1 and HSV 2 by bi- directional sequencing.

2All five (5) HSV 1 ARIES negative specimens that were positive by the reference method were negative by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. One of these specimens was positive for HSV 2 by both ARIES and sequencing.

Summary of HSV 1 Results for Mucocutaneous Lesions $\scriptstyle ( \mathbf { N } = \mathbf { 1 } \mathbf { 1 9 0 } )$ )   

<table><tr><td rowspan=1 colspan=1>ARIESHSV 1&amp;2 Assay</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>421</td><td rowspan=1 colspan=1>304</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>878</td><td rowspan=1 colspan=1>886</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>920</td><td rowspan=1 colspan=1>1190</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.0% (262/270)</td><td rowspan=1 colspan=1>94.2% - 98.7%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.4% (878/920)</td><td rowspan=1 colspan=1>93.9% - 96.7%</td><td rowspan=1 colspan=1></td></tr></table>

1Nineteen (19) HSV 1 ARIES® positive specimens that were negative by the reference method were positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the $\mathrm { \Delta A R I E S ^ { ( B ) } }$ HSV 1&2 Assay. Twenty (20) false positive specimens were negative for both HSV-1 and HSV-2 by bi-directional sequencing. The remaining three (3) specimens were unavailable (QNS) for sequence analysis.

2Seven (7) HSV 1 ARIES® negative specimens that were positive by the reference method were negative by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the $\mathrm { \Delta A R I E S ^ { ( B ) } }$ HSV 1&2 Assay. One of these specimens was positive for HSV 2 by both ARIES® and sequencing. One (1) false negative specimen was positive for HSV-1 by bi-directional sequencing.

Summary of HSV 2 Results for Cutaneous Lesions $\scriptstyle ( \mathbf { N } = 4 4 8 )$   

<table><tr><td rowspan=1 colspan=1>ARIESHSV 1&amp;2 Assay</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>135</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>52}$</td><td rowspan=1 colspan=1>308</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>448</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95.0% (96/101)</td><td rowspan=1 colspan=1>88.8% - 98.4%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>88.8% (308/347)</td><td rowspan=1 colspan=1>85.0% - 91.9%</td><td rowspan=1 colspan=1></td></tr></table>

1Thirty five (35) HSV 2 ARIES positive specimens that were negative by the reference method were positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. The remaining four (4) false positive specimens were negative for both HSV-1 and HSV-2 by bi- directional sequencing.

2All five (5) HSV 2 ARIES negative specimens that were positive by the reference method were negative by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. Two of these specimens were positive for HSV 1 by both ARIES and sequencing.

Summary of HSV 2 Results for Mucocutaneous Lesions $\scriptstyle ( \mathbf { N } = \mathbf { 1 } 4 5 3 )$ )   

<table><tr><td rowspan=1 colspan=1>ARIESHSV 1&amp;2 Assay</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>811</td><td rowspan=1 colspan=1>340</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>1109</td><td rowspan=1 colspan=1>1113</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>1190</td><td rowspan=1 colspan=1>1453</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>98.5% (250/263)</td><td rowspan=1 colspan=1>96.2% - 99.6%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>93.2% (1109/1190)</td><td rowspan=1 colspan=1>91.6% - 94.6%</td><td rowspan=1 colspan=1></td></tr></table>

1Fifty eight (58) HSV 2 ARIES positive specimens that were negative by the reference method were positive by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. Twenty-one (21) false positive specimens were negative for both HSV-1 and HSV-2 by bi-directional sequencing. The remaining two (2) specimens were unavailable (QNS) for sequence analysis.

2All four (4) HSV 2 ARIES negative specimens that were positive by the reference method were negative by bi-directional sequencing analysis using analytically validated primers that targeted genomic regions distinct from the ARIES® HSV 1&2 Assay. Three (3) of these specimens were positive for HSV 1 by both ARIES and sequencing.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

Prevalence: The prevalence of HSV-1 and HSV-2 with the ARIES® HSV 1&2 Assay was calculated for cutaneous and mucocutaneous specimens and is summarized for the combined sample set per age group and by specimen source in the table below.

# Cutaneous Prevalence by Age

<table><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV 1</td><td rowspan=1 colspan=3>HSV 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>0 - 10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>30.2%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>17.0%</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>15.2%</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>31.2%</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>30.6%</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.9%</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>28.6%</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8.0%</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>32.0%</td></tr><tr><td rowspan=1 colspan=1>&gt;60</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>43.5%</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>15.2%</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>30.1%</td></tr></table>

Cutaneous Prevalence by Lesion Source   

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=3>HSV</td><td rowspan=1 colspan=3>HSV 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>Genital - Penis</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>32.5%</td></tr><tr><td rowspan=1 colspan=1>Skin Lesion</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>27.7%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>15.2%</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>30.1%</td></tr></table>

# Mucocutaneous Prevalence by Age

<table><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV 1</td><td rowspan=1 colspan=3>HSV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>0 - 10</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>33.5%</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>20.2%</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>463</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>24.6%</td><td rowspan=1 colspan=1>463</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>27.4%</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>23.7%</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>13.0%</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>27.1%</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>23.2%</td></tr><tr><td rowspan=1 colspan=1>&gt;60</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8.4%</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>28.4%</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1453</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>20.9%</td><td rowspan=1 colspan=1>1453</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>23.4%</td></tr></table>

Mucocutaneous Prevalence by Lesion Source   

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=3>HSV 1</td><td rowspan=1 colspan=3>HSV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>Anorectal</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8.1%</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>37.8%</td></tr><tr><td rowspan=1 colspan=1>GenitalVaginal /Cervical</td><td rowspan=1 colspan=1>688</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>688</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>27.2%</td></tr><tr><td rowspan=1 colspan=1>GenitalLabia /Vulva</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>18.8%</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>32.1%</td></tr><tr><td rowspan=1 colspan=1>Urethral</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16.0%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16.0%</td></tr><tr><td rowspan=1 colspan=1>Nasal</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11.1%</td></tr><tr><td rowspan=1 colspan=1>Ocular</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11.6%</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.0%</td></tr><tr><td rowspan=1 colspan=1>Oral</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>31.1%</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1453</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>20.9%</td><td rowspan=1 colspan=1>1453</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>23.4%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.