# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K151464

# B. Purpose for Submission:

To request an expansion of the Intended Use of the Alere™ i Influenza A&B assay on the Alere™ i Instrument, which was FDA-cleared under K141520, to also include testing of nasal and nasopharyngeal swab samples that have been eluted into Viral Transport Media (VTM).

# C. Measurand:

Target RNA sequences for the highly conserved regions of the PB2 gene (encodes an RNA polymerase – polymerase basic protein 2) segment of Influenza A virus and the PA gene (encodes another RNA polymerase – polymerase acidic protein) segment of Influenza B virus.

# D. Type of Test:

Qualitative isothermal nucleic acid amplification assay for the amplification and detection of specific Influenza A and B RNA gene targets.

# E. Applicant:

Alere Scarborough, Inc.

F. Proprietary and Established Names:

Alere™ i Influenza A&B Alere™ i Instrument

# G. Regulatory Information:

1. Regulation section: 21 CFR section 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code(s):

OCC - Respiratory virus panel nucleic acid assay system OZE - Influenza A and influenza B multiplex nucleic acid assay OOI - Real time nucleic acid amplification system

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Alere™ i Influenza A & B assay performed on the Alere™ i Instrument is a rapid molecular in vitro diagnostic test utilizing an isothermal nucleic acid amplification technology for the qualitative detection and discrimination of influenza A and B viral RNA in direct nasal swabs and nasal or nasopharyngeal swabs eluted in viral transport media from patients with signs and symptoms of respiratory infection. It is intended for use as an aid in the differential diagnosis of influenza A and B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay is not intended to detect the presence of influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2012-2013 and the 2014-2015 influenza seasons when influenza A/H3 and A/H1N1 pandemic were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

# Alere™ i Instrument

# I. Device Description:

# Overview

The Alere™ i Influenza A & B assay system utilizes an isothermal nucleic acid amplification technology and is comprised of:

• Sample Receiver – single use, disposable containing the elution buffer   
• Test Base – single use, disposable comprising two sealed reaction tubes, each containing a lyophilized pellet   
• Transfer Cartridge – single use, disposable for transfer of the eluted sample to the Test Base   
• Alere™ i Instrument – repeat use reader, and   
• Alere™ i Influenza A & B positive and negative external controls – for quality control purposes   
• Nasal Swabs – sterile swabs for use with the Alere™ i Influenza A & B Test.

The first reaction tube (Tube 1) in the Test Base contains the reagents required for amplification of the influenza A target viral nucleic acid, and the second reaction tube (Tube 2) in the Test Base contains reagents required for amplification of the influenza B target viral nucleic acid and an internal control synthetic RNA sequence. Alere™ i Influenza A & B utilizes pairs of templates (similar to primers in a PCR reaction) for the specific isothermal amplification of target RNA sequences from influenza A (highly conserved region of the PB2 gene), influenza B (highly conserved region of the PA gene), and the internal control, and fluorescently labeled molecular beacons designed to specifically identify the amplified products. Alere™ i Influenza A & B is performed within the confinement of the Test Base, and no other part of the Alere™ i Instrument has contact with the sample during the amplification process.

To perform the assay, the Sample Receiver and the Test Base are inserted into the Alere™ i Instrument and the elution buffer is automatically heated by the instrument. The sample is added to the Sample Receiver and transferred via the Transfer Cartridge to the Test Base, resuspending the lyophilized pellet contained within the Test Base and initiating target amplification. Heating, mixing and detection by fluorescence are provided by the instrument, with results automatically reported.

Results (positive, negative, or invalid) are displayed by the Alere™ i Instrument separately for influenza A and influenza B. Results are also stored in an on-board archive and are assigned to a sample ID that has been entered into the Alere™ i Instrument by the operator, and the date/time the test was performed. Data can be retrieved and downloaded by the operator at any time after testing. An external Alere™ Universal Printer can be attached via USB to the Alere™ i Instrument to print test results.

# Alere™ i Influenza A & B System Components

# Sample Receiver

The Sample Receiver is a foil sealed, single-use, disposable plastic component containing $2 . 5 \mathrm { m l }$ of an elution/lysis buffer. Rapid elution of influenza virus from swabs and virus lysis occur through the action of the elution/lysis buffer. The elution/lysis buffer contains salts at the appropriate concentrations that are required for the test reaction, which upon transfer of the sample using the Transfer Cartridge to the Test Base, is used to reconstitute the lyophilized pellets. The elution/lysis buffer also contains a chelating agent to remove any heavy metals found in the sample, as they are potential inhibitors of the test reaction.

# Transfer Cartridge

The Transfer Cartridge is a single-use, disposable plastic component for transfer of the eluted sample to the Test Base. The aspiration of the sample is achieved by placing the Transfer Cartridge over the Sample Receiver and pressing downwards until a click is heard and the orange indicator on the top of the Transfer Device is up. The dispense of the sample is achieved by placing the Transfer Cartridge into the Test Base and pressing on the Transfer Cartridge until the orange indicator on the top of the Transfer Cartridge descends. When the indicator descends, so does the plunger, thus driving the sample out of the two nozzles and into the reaction tubes in the Test Base and mixing with the lyophilized reagent in the reaction tubes.

# Test Base

The Test Base is a single-use, disposable plastic component comprising two sealed reaction tubes, the first reaction tube (Tube 1) is for the influenza A reaction and the second tube (Tube 2) is for the influenza B and the Internal Control reactions. Each reaction tube contains a lyophilized pellet which provides all of the necessary reagents to drive the respective reaction(s) except for salts. Tube 2 also contains an internal control synthetic RNA sequence. The salts that are required for the test reactions are provided at their appropriate concentrations within the elution/lysis buffer contained in the Sample Receiver. Upon transfer of the sample from the Sample Receiver to the Test Base using the Transfer Cartridge, the sample in elution/lysis buffer is used to reconstitute the lyophilized pellets. The Alere™ i Influenza A & B reaction is performed within the Test Base. As soon as the sample is transferred, it comes into contact with the lyophilized pellets, reconstitutes the lyophilized pellets and initiates the assay reaction. Each Test Base contains a barcode with assay specific information (assay ID, expiration date, and lot number). The barcode is read automatically from the Test Base upon insertion into the Alere™ i Instrument, providing the information necessary to run the assay.

# Alere™ i Instrument

The Alere™ i Instrument is a portable bench-top unit designed to perform in laboratory and point-of care environments. Alere™ i Influenza A & B components are color-coded to match the corresponding holder on the instrument and designed to only fit in the corresponding holder in one direction. The Alere™ i Instrument uses image analysis to automatically detect the presence of the Test Base and to check the test type and expiry date on the barcode present on the Test Base. This triggers the test sequence. Heating, mixing, and detection by fluorescence are provided by the instrument. The Alere™ i Instrument is equipped with two optical fluorescence modules, $\mathrm { F A M ^ { T M } }$ and $\mathrm { R O X T M }$ . Each optical module consists of a LED excitation source and photodiode receiver configured to the fluorescent channel. Fluorescence signals are optically filtered by a confocal lens and mirror arrangement and processed by software running on an on-board computer. The results are calculated automatically and presented on the instrument graphical user interface and available for export in JSON format or printing.

# Alere™ i Influenza A & B External Controls

Alere™ i Influenza A & B External Controls are designed for use with Alere™ i Influenza A & B. The Positive Control swab is coated with inactivated influenza A virus and inactivated influenza B virus dried onto a swab. The Negative Control swab is coated with inactivated Streptococcus Group C dried onto a swab.

# Nasal Swabs

Sterile swabs for use with the Alere™ i Influenza A & B test are provided as a part of the Alere™ i Influenza A & B System.

# Alere™ i Influenza A & B Workflow

The protocols for the two sample types, direct nasal swab and nasal or nasopharyngeal swab eluted in VTM, are slightly different with regard to the initial Sample Receiver warm-up time in the Sample Receiver holder in the Alere™ i Instrument. The warm-up time for testing direct nasal swab samples is three minutes, while the warm-up time for testing nasal or nasopharyngeal swab eluted in VTM samples is six minutes. Users with designated “administrator” rights specify the sample type or types to be tested at each testing location during initial Alere™ i Instrument set up.

An overview of the Alere™ i Influenza A & B workflow is summarized below:

1. User ID and Patient ID are entered via the touch screen prior to initiating the testing sequence. Patient information can be entered by two means: • by using the Instrument touch screen • by using a barcode reader

2. The Alere™ i Influenza A & B Test Base barcode contains assay information (test type, expiry date, and lot number). Upon proper insertion into the Test Base holder of the instrument, using image analysis the instrument automatically detects the presence of the Test Base and checks the test type, expiry date, and lot number encoded in the barcode. Insertion of an expired Test Base will generate an error

message on the instrument screen and the user is not able to proceed with testing. Upon confirmation of the Test Base insertion the user is instructed to insert the Sample Receiver.

3. The Alere™ i Instrument detects the insertion of the Sample Receiver into the Sample Receiver holder and automatically progresses to the warm-up state. The instrument software implements temperature control of both the Test Base and the Sample Receiver holders (heater blocks). Each of the heater blocks is managed independently with high precision temperature monitoring and control.

4. Upon completion of the warm-up state, the Alere™ i Instrument instructs the user to remove the foil seal from the Sample Receiver, insert sample to the elution buffer contained in the Sample Receiver, and mix as described on the screen.

5. The Alere™ i Instrument then instructs the user to transfer the sample in the Sample Receiver to the Test Base using the Transfer Cartridge.

6. The Alere™ i Instrument uses image analysis to confirm that the Transfer Cartridge is present in the Test Base, instructs the user to close the lid, and automatically proceeds to the measurement state upon detection of test bay lid closure. The lid operates using a magnetic switch system that detects lid closure and an electrical signal triggers testing commencement. Heating, rotation and detection by fluorescence is provided by the instrument.

7. Upon completion of the testing sequence, results are displayed on the screen and are presented to the user as positive, negative, or invalid separately for influenza A and influenza B. Upon acknowledgement of the test results, the user is instructed to dispose of the used test components.

The turnaround time for testing a sample is approximately 15 minutes.

# Quality Control

# Internal Control (Procedural Control)

Alere™ i Influenza A & B contains an internal control (IC) that was designed to monitor sample inhibition, amplification, and assay reagent function for both influenza A and influenza B. The IC is an RNA oligonucleotide contains $5 '$ and $_ { 3 } ,$ ends that are complementary to the influenza B target template set’s recognition regions, but with a spacer region (the region between the two recognition regions) that differs from the influenza B target’s spacer region. The IC is present in the second reaction tube (Tube 2) of the Test Base along with all of the necessary reagents to drive the influenza B and the IC reactions except for salts. Detection of the IC occurs via a molecular beacon that specifically detects the amplified product generated from the IC RNA oligonucleotide. The two molecular beacons present in the second reaction tube of the Test Base are labeled with fluorophores with distinct excitation/emission profiles (Influenza B – FAM and IC – ROX), enabling detection of both beacons simultaneously using a two-color detector.

In positive samples where influenza target amplification is strong, the IC result is ignored. The influenza B target amplification in Tube 2 of the Test Base also serves as the “Control” to confirm that the clinical sample was not inhibitory and that influenza A reagent performance in Tube 1 was robust. At a low frequency, clinical samples can contain inhibitors that may generate invalid results.

# Procedural Controls

In addition to the test specific Internal Control described previously, the Alere™ i instrument also employs additional “Procedural Controls” to ensure that a test is performed appropriately. Each Test Base has a barcode which identifies the test type – the instrument automatically detects the Test Base and asks the user to confirm the test type. Each of the additional steps (sample heating, sample transfer, and closing the lid, etc.) are monitored by the instrument and if the user takes too long to complete a certain step, the instrument will time out, generate an error, and a result is not reported.

“Procedural Control Valid” displayed on the instrument screen indicates that the assay reagents maintained their functional integrity and the sample did not significantly inhibit assay performance.

# External Positive and Negative Controls

Good laboratory practice suggests the use of positive and negative controls to ensure that test reagents are working and that the test is correctly performed. Alere™ i Influenza A & B kits contain Positive and Negative Control Swabs, and should be tested following the “Run QC Test” instructions on the Alere™ i Instrument. These swabs monitor the entire assay. Test these swabs once with each new shipment received and once for each new operator. Further controls may be tested in order to conform to local, state and/or federal regulations, accrediting groups, or laboratory’s standard Quality Control procedures.

# Results Interpretation

The Alere™ i Instrument software automatically interprets test results. The assay uses the following three fluorescent channels: Flu A (FAM) in Tube 1, Flu B (FAM) in Tube 2, and IC (ROX) in Tube 2. The ROX channel in Tube 1 is not used. Each channel of each reaction tube is first individually assessed using a defined channel decision algorithm and assigned a channel result as “Asserted”, “Not Asserted” or “Indeterminate”. The individual channel results are then combined to arrive at the influenza A and influenza B results. All nine possible assay result combinations and the respective underlying assay decision logic rules are presented in Table 1 below:

Table 1: All Possible Assay Result Combinations and Respective Underlying Assay Decision Logic Rules   

<table><tr><td rowspan=1 colspan=1>Assay Result Reported</td><td rowspan=1 colspan=3>Underlying Assay Decision Logic Rules</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A Channel(FAM) in Tube 1</td><td rowspan=1 colspan=1>Flu B Channel(FAM) in Tube 2</td><td rowspan=1 colspan=1>IC Channel (ROX)in Tube 2</td></tr><tr><td rowspan=1 colspan=1>Flu A Positive/ Flu B Negative</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=5 colspan=1>Flu A Positive/ Flu B Invalid</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>Flu A Positive/ Flu B Positive</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>Flu A Negative/ Flu B Positive</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Invalid</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Flu A Negative/ Flu B Negative</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=3 colspan=1>Flu A Invalid/Flu B Positive</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Flu A Invalid/Flu B Negative</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=9 colspan=1>Flu A Invalid/Flu B Invalid</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Not Asserted</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td></tr></table>

All possible assay result combinations and the respective interpretations and follow-up actions are presented in Table 2 below:

Table 2: All Possible Assay Result Combinations and Respective Interpretations and Follow-up Actions   

<table><tr><td colspan="1" rowspan="1">Assay Result Reported</td><td colspan="1" rowspan="1">Interpretation of Results and Follow-up Actions</td></tr><tr><td colspan="1" rowspan="1">Flu A Positive/ Flu B Negative</td><td colspan="1" rowspan="1">Flu A Viral RNA Detected; Flu B Viral RNA Not Detected.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype.</td></tr><tr><td colspan="1" rowspan="1">Flu A Positive/ Flu B Invalid</td><td colspan="1" rowspan="1">Flu A Viral RNA Detected; The presence or absence of Flu BViral RNA can not be determined.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype.</td></tr><tr><td colspan="1" rowspan="1">Flu A Positive/ Flu B Positive</td><td colspan="1" rowspan="1">Flu A Viral RNA Detected; Flu B Viral RNA Detected.Dual infections of Flu A and Flu B are rare. Repeat testingusing new test components. Contact Technical Support duringnormal business hours if multiple samples provide this result.This result does not rule out co-infections with other pathogensor identify any specific influenza A virus subtype or specificinfluenza B virus lineage.</td></tr><tr><td colspan="1" rowspan="1">Flu A Negative/ Flu B Positive</td><td colspan="1" rowspan="1">Flu A Viral RNA Not Detected; Flu B Viral RNA Detected.This result does not rule out co-infections with other pathogensor identify any specific influenza B virus lineage.</td></tr><tr><td colspan="1" rowspan="1">Flu A Negative/ Flu B Invalid</td><td colspan="1" rowspan="1">Flu A Viral RNA Not Detected; The presence or absence of FluB Viral RNA can not be determined.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Infection due to Flu B can not be ruled out. Repeat testing ofthe sample using new test components. If repeated Flu BInvalid results are obtained, results should be confirmed byanother method prior to reporting the results.</td></tr><tr><td colspan="1" rowspan="1">Flu A Negative/ Flu B Negative</td><td colspan="1" rowspan="1">Flu A Viral RNA Not Detected; Flu B Viral RNA NotDetected.</td></tr><tr><td colspan="1" rowspan="1">Flu A Invalid/Flu B Positive</td><td colspan="1" rowspan="1">The presence or absence of Flu A Viral RNA can not bedetermined. Flu B Viral RNA Detected;This result does not rule out co-infections with other pathogensor identify any specific influenza B virus lineage.</td></tr><tr><td colspan="1" rowspan="1">Flu A Invalid/Flu B Negative</td><td colspan="1" rowspan="1">The presence or absence of Flu A Viral RNA can not bedetermined. Flu B Viral RNA Not Detected;Infection due to Flu A can not be ruled out. Repeat testing ofthe sample using new test components. If repeated Flu AInvalid results are obtained, results should be confirmed byanother method prior to reporting the results.</td></tr><tr><td colspan="1" rowspan="1">Flu A Invalid/Flu B Invalid</td><td colspan="1" rowspan="1">The presence or absence of Flu A and Flu B Viral RNAs cannot be determined.Repeat testing of the sample using new test components. Ifrepeated Flu A and Flu B Invalid results are obtained, resultsshould be confirmed by another method prior to reporting theresults.</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): Alere™ i Influenza A & B

2. Predicate 510(k) number: K141520

3. Comparison with predicates:

<table><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">AlereTM i Influenza A &amp; B (K151464)</td><td colspan="1" rowspan="1">AlereTM i Influenza A &amp; B (K141520)</td></tr><tr><td colspan="1" rowspan="1">Assay Analyte</td><td colspan="1" rowspan="1">Influenza A and Influenza B</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The AlereTM i Influenza A &amp; B assay performed onthe AlereTM i Instrument is a rapid molecular in vitrodiagnostic test utilizing an isothermal nucleic acidamplification technology for the qualitative detectionand discrimination of influenza A and B viral RNAin direct nasal swabs and nasal or nasopharyngealswabs eluted in viral transport media from patientswith signs and symptoms of respiratory infection. Itis intended for use as an aid in the differentialdiagnosis of influenza A and B viral infections inhumans in conjunction with clinical andepidemiological risk factors. The assay is notintended to detect the presence of influenza C virus</td><td colspan="1" rowspan="1">The AlereTM i Influenza A &amp; B assay performed on theAlereTM i Instrument is a rapid molecular in vitrodiagnostic test utilizing an isothermal nucleic acidamplification technology for the qualitative detectionand discrimination of influenza A and B viral RNA innasal swabs from patients with signs and symptoms ofrespiratory infection. It is intended for use as an aid inthe differential diagnosis of influenza A and B viralinfections in humans in conjunction with clinical andepidemiological risk factors. The assay is not intendedto detect the presence of influenza C virusNegative results do not preclude influenza virusinfection and should not be used as the sole basis for</td></tr><tr><td colspan="1" rowspan="1">Alere™M i Influenza A &amp; B (K151464)</td><td colspan="1" rowspan="1">Alere™M i Influenza A &amp; B (K141520)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative results do not preclude influenza virusinfection and should not be used as the sole basis fordiagnosis, treatment or other patient managementdecisions.Performance characteristics for influenza A wereestablished during the 2012-2013 and the 2014-2015influenza seasons when influenza A/H3 and A/H1N1pandemic were the predominant influenza A virusesin circulation. When other influenza A viruses areemerging, performance characteristics may vary.If infection with a novel influenza A virus issuspected based on current clinical andepidemiological screening criteria recommended bypublic health authorities, specimens should becollected with appropriate infection controlprecautions for novel virulent Influenza viruses andsent to state or local health department for testing.Viral culture should not be attempted in these casesunless a BSL 3+ facility is available to receive andculture specimens.</td><td colspan="1" rowspan="1">diagnosis, treatment or other patient managementdecisions.Performance characteristics for influenza A wereestablished during the 2012-2013 influenza seasonwhen influenza A/H3 and A/H1N1 pandemic were thepredominant influenza A viruses in circulation. Whenother influenza A viruses are emerging, performancecharacteristics may vary.If infection with a novel influenza A virus is suspectedbased on current clinical and epidemiological screeningcriteria recommended by public health authorities,specimens should be collected with appropriateinfection control precautions for novel virulentInfluenza viruses and sent to state or local healthdepartment for testing. Viral culture should not beattempted in these cases unless a BSL 3+ facility isavailable to receive and culture specimens.</td></tr><tr><td colspan="1" rowspan="1">IntendedEnvironment forUse</td><td colspan="1" rowspan="1">CLIA waived for testing direct nasal swab samplesonly. Professional use, in a medical laboratory orpoint-of-care.</td><td colspan="1" rowspan="1">CLIA Waived.Professional use, in a medical laboratory or point-of-care.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Alere i Instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Self-contained System</td><td colspan="1" rowspan="1">Integrated PC, Software, and Touch Screen Display</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Automated Assay</td><td colspan="1" rowspan="1">Yes. Sample preparation, amplification, detectionand result interpretation</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Direct nasal swab and nasal or nasopharyngeal swabeluted in VTM</td><td colspan="1" rowspan="1">Direct nasal swab</td></tr><tr><td colspan="1" rowspan="1">Influenza A ViralTarget</td><td colspan="1" rowspan="1">PB2 gene segment</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Influenza B ViralTarget</td><td colspan="1" rowspan="1">PA gene segment</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Amplification andDetection Technology</td><td colspan="1" rowspan="1">Isothermal nucleic acid amplification and detectionof specific amplification products using molecularbeacon probes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">15 Minutes</td><td colspan="1" rowspan="1">Same</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None

# L. Test Principle:

Individual reactions for influenza A and B and IC, provided as lyophilized pellets contained within the Test Base, specifically amplify unique target regions within the RNA genomes of these two viruses and the IC. Alere™ i Influenza A & B employs fluorescently-labeled molecular beacon probes for real-time detection of amplified products. Alere™ i Influenza A & B amplification technology, Nicking and Extension Amplification Reaction (NEAR) is isothermal and does not require a heat denaturation step to generate a single strand target for amplification. Instead a thermostable, strand-displacing DNA polymerase, a thermostable nicking endonuclease, and two oligonucleotides (typically known as primers, but are referred to as templates in this technology) are utilized. Reverse transcriptase is also present in the reaction. The products of Alere™ i Influenza A & B are two complementary oligonucleotides 30 – 40 nucleotides in length. A product of this size is of sufficient length to be highly unique amongst genomes. The exact length of the products depends on the specific placement of the templates relative to one another and relative to their complementary sequences within the target genome. Alere™ i Influenza A & B assay specificity is achieved through two levels of sequence-specific interrogation, template annealing to complementary target sequence at an elevated temperature to drive target-specific amplification, and amplification product detection as the molecular beacon anneals to Alere™ i Influenza A & B amplification product in a sequence-specific fashion.

# Alere™ i Influenza A & B Isothermal Amplification Mechanism

Alere™ i Influenza A & B isothermal amplification, NEAR, is driven by a thermostable strand displacing DNA polymerase, thermostable nicking endonuclease, two oligonucleotide templates, plus a reverse transcriptase. The Alere™ i Influenza A & B oligonucleotide template (primer) consists of three distinct regions. The $_ 5 \cdot$ -end of the template is known as the stabilizing region (SR) which is not involved in target-specific complementary sequencedependent annealing, but rather in stabilizing the double stranded products generated during amplification. The central region of each template maintains the nicking endonuclease binding site (NEBS) and nicking endonuclease cut site (NECS), which also are not complementary to the target sequence, but provide the sequence required for single strand nicking activity by the nicking endonuclease. At the $_ { 3 } ,$ -end of each template is the recognition region (RR), which provides target-specific, sequence-dependent annealing. Typically, the RR is 11 – 12 nucleotides in length on each template. As shown in Figure 1 below, step 1a, the recognition region (RR) of template T2 (labeled as Y), anneals to a complimentary sequence present in the single strand DNA target (labeled as Y’). In step 2a, a DNA polymerase extends T2 from its $3 ^ { \circ } { - } \mathrm { O H }$ group generating sequence X which is complementary to target sequence $X ^ { \prime }$ . In this step only the $_ 3 \cdot$ end of T2 anneals to a target sequence in a complementary fashion. The SR, NEBS and NECS regions of the template are not complementary to the target sequence. In step 3a, a second T2 template displaces the

original, partially annealed T2. DNA polymerase extension from this second T2 displaces the first synthesized strand. T1 recognizes, anneals to, and extends from sequence X of the displaced strand (step 4a). This leaves a product containing a double strand NEBS and NECS (step 5a). A nicking endonuclease will then bind to its recognition site and nick the structure at its cut site. This generates an accessible $_ { 3 } ,$ -OH that can be extended by the DNA polymerase, resulting in the release of a single strand DNA (containing regions X and Y) and generation of a double stranded DNA product containing NEBS and NECS on both ends (step 6a), known as the Amplification Duplex. Figure 1 depicts the linear phase of the mechanism, essentially the generation of the NEAR Amplification Duplex.

![](images/feb53ffc152861ff744eac3028c6c52da0f46af4ff77b169cfcbb93fbd1ef1ca.jpg)  
Figure 1 – Alere™ i Influenza A & B Mechanism – Linear Phase

In Figure 2 below, the geometric amplification phase is depicted. The NEAR Amplification Duplex is a double strand DNA structure containing two complete nicking endonuclease binding and cut sites (1b). As shown in 2b, nicking of the top and bottom strands occurs, providing the DNA polymerase two $_ { 3 } ,$ -OH groups for initiating strand displacement extension. This results in the release of $\mathbf { \boldsymbol { X } ^ { \flat } \mathbf { \boldsymbol { Y } ^ { \flat } } }$ (specific product 1) and X Y (specific product 2) products, and the generation of two double stranded DNA moieties containing single nicking endonuclease binding and cut sites (3b). The two moieties undergo reiterative nicking and extension, generating additional specific products 1 and 2 (3b). Specific products 1 and 2 are substrates for T2 and T1 (4b). Extension of these complexes (4b) generates additional double stranded DNA moieties containing single nicking endonuclease binding and cut sites (3b), with the result of this reiterative nicking and extension being geometric amplification.

![](images/fe81b92158ea697e995f54a42f9ed532f0ee9353a3ad37073e8e37d1dfabad72.jpg)  
Figure 2 – Alere™ i Influenza A & B Amplification Duplex – Geometric Phase

# Alere™ i Influenza A & B Amplification Detection Mechanism

The intentional skewing of templates leading to asymmetric amplification product formation is necessary for optimal performance of the real-time fluorescence-based molecular beacon detection system coupled to the Alere™ i Influenza A & B amplification technology. Molecular beacons are single-stranded DNA sequences that are able to form hairpin loop structures. The hairpin is formed by complementary $_ 5 \cdot$ and $_ { 3 } ,$ -ends that are typically 4 – 7 nucleotides in length. Molecular beacons contain a fluorophore and a quencher, typically at opposite ends of the molecular beacon’s linear nucleotide sequence. When the molecular beacon forms a hairpin structure, the fluorophore and quencher are in close proximity, and fluorescence is quenched. Upon opening of the hairpin, which can occur when a complementary DNA or RNA species anneals to the molecular beacon’s loop region, the distance between the fluorophore and quencher is lengthened, reducing the ability of the quencher to quench the fluorescence signal. As a result, fluorescence is emitted and can be measured using an optical reader. The amount of fluorescence released is directly proportional to the amount of molecular beacon that has been opened, and concomitantly, the amount of complementary DNA or RNA species present in the sample. The result of the Alere™ i Influenza A & B NEAR amplification reaction is two specific products which are perfect complements to one another. These two specific products form a very stable double strand DNA structure at assay temperature. In order to combine the use of the Alere™ i Influenza A & B NEAR technology and molecular beacons, the concentrations of the two templates present in the Alere™ i Influenza A & B assay are skewed relative to one another to drive asymmetric product formation, i.e., one product is made to a larger extent than the other. The product that is generated in excess is then targeted by the molecular beacon. Annealing of the loop region of the molecular beacon to the Alere™ i Influenza A & B product generated in excess results in opening of the hairpin stem, separation of the fluorophore and quencher, and ultimately fluorescence signal release.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility Study

The purpose of this $5 1 0 ( \mathrm { k } )$ submission is to expand the Intended Use to allow the test to be used on nasal or nasopharyngeal swabs eluted in viral transport media (VTM) and the test reagents were not modified, therefore additional Reproducibility Study data are not necessary.

Please refer to the Reproducibility testing data submitted in K141520.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

External Positive and Negative Controls

Alere™ i Influenza A & B kits contain Positive and Negative Control Swabs. These swabs monitor the entire assay. The Alere™ i Influenza A & B Instructions for Use requires testing these control swabs once with each new shipment received and once for each new operator. Further controls may be tested in order to conform to local, state and/or federal regulations, accrediting groups, or laboratory’s standard Quality Control procedures.

During the prospective clinical study during the 2014 to 2015 influenza season, control testing using external positive and negative controls was performed prior to sample testing each day and on each instrument Alere™ i Influenza A & B testing was performed, at all three study sites. A total of 234 external positive controls and 234 external negative controls were tested during the prospective clinical study. Seven external positive controls (7/234, $3 . 0 \%$ , $9 5 \%$ CI, $1 . 5 \% - 6 . 0 \% )$ and five external negative controls (5/234, $2 . 1 \%$ , $9 5 \%$ CI, $0 . 9 \% - 4 . 9 \% )$ ) failed initially, but succeeded on repeat testing per the product instructions for use.

# VTM Specimen Stability Study

To provide data supporting the VTM specimen storage recommendations stated in the product package insert, an analytical study was carried out to evaluate specimen stability.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD) and Influenza B/Malaysia/2506/2004 (Lot C1320E1) were used in this study to prepare contrived samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) were determined by standard virologic method by the venders. The targeted concentration for each dilution (in genome equivalents $ { / \mathrm { m L } }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive VTM samples were prepared using the influenza A and B strains (described above) dilutions in universal transport media (UTM) with targeted concentrations near the respective LoD levels (approximately $3 \mathrm { ~ X ~ }$ the respective LoD). Mock negative VTM samples were prepared using UTM only. All prepared VTM samples were tested in triplicates at each of the following three time points/conditions: Time 0, Hour 9 (after being stored at approximately1 $. 8 \mathrm { - } 2 2 ^ { \circ } \mathrm { C }$ for 9 hours), and Hour 25 (after being stored at approximately $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ for 9 hours , and then placed at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 16 hours of storage).

The contrived influenza A and influenza B positive VTM samples were tested at each time point as follows:

Contrived influenza A VTM: $4 . 3 4 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ $( 4 . 5 9 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/mL); $3 . 2 0 \mathrm { ~ x ~ } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { I }$ Reaction $( 3 . 4 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Reaction) • Contrived influenza B VTM: $4 . 3 9 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { \ : T C I D } _ { 5 0 } / \mathrm { m L } ($ $( 2 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents $/ \mathrm { m L }$ ); $3 . 2 6 \times 1 0 ^ { 1 }$ TCID $5 0$ /Reaction $( 1 . 7 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Reaction)

Contrived VTM sample testing results are presented in Table 3 below.

Table 3: VTM Specimen Stability Study Results   

<table><tr><td rowspan=1 colspan=1>Time Point</td><td rowspan=1 colspan=2>Mock Negative VTM Sample</td><td rowspan=1 colspan=2>Contrived Flu A Positive VTMSample</td><td rowspan=1 colspan=2>Contrived Flu B Positive VTMSample</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=1 colspan=1>Time 0</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr><tr><td rowspan=1 colspan=1>Hour 9 (approximately18-22°)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr><tr><td rowspan=1 colspan=1>Hour 25 (16 hrs at 2-8°C)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Negative (3/3)</td><td rowspan=1 colspan=1>Positive (3/3)</td></tr></table>

This analytical study data supports Alere™ i Influenza A & B product insert sample storage recommendations for VTM samples of up to 8 hours at $1 8 { - } 2 2 ^ { \circ } \mathrm { C }$ , or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 24 hours.

# Viral Transport Media (VTM) Type Compatibility Validation Study

To assess the compatibility of various types of VTM to be used with the Alere™ i Influenza A & B, an analytical study was conducted testing the following VTMs:

Amies Transport Media, Starplex Multitrans Media, Brain Heart Infusion Broth, M4 Media, M4-RT Media, M5 Media, M6 Media, Liquid Stuart’s, Universal Viral Transport Media (UTM), Dulbecco’s Modified Eagles’ Medium (D-MEM), Tryptose Phosphate Broth, Hank’s Balanced Salt Solution, $0 . 9 \%$ Saline, Phosphate Buffered Saline, $2 . 5 \%$ Veal Infusion Broth, and Vircell Transport Media.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD) and Influenza B/Malaysia/2506/2004 (Lot C1320E1) were used in this study to prepare contrived VTM samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) were determined by standard virologic method by the venders. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive VTM samples for each VTM type were prepared using the influenza A and B strains (described above) dilutions in UTM, and $0 . 0 1 ~ \mathrm { m L }$ of the virus dilutions were added to $0 . 9 9 \mathrm { m L }$ of each of the VTMs to be tested, with the final targeted testing concentrations near the respective LoD levels (approximately $3 \mathrm { ~ X ~ }$ the respective LoD). Mock negative VTM samples for each VTM type were prepared using the respective VTM type under investigation. All prepared VTM samples were tested in triplicates.

The influenza A and influenza B positive VTM samples with concentration approximately 3 X times the respective LoD were tested for each VTM type as follows:

Contrived influenza A VTM: $4 . 3 4 \times 1 0 ^ { 3 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ $( 4 . 5 9 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome   
equivalents $/ \mathrm { m L }$ ); $3 . 2 0 \times 1 0 ^ { 1 }$ T $\mathrm { C I D } _ { 5 0 } .$ /Reaction $( 3 . 4 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome   
equivalents/Reaction)   
Contrived influenza B VTM: $4 . 3 9 \mathrm { \times 1 0 ^ { 3 } T C I D _ { 5 0 } / m L }$ $( 2 . 3 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome   
equivalents/mL); $3 . 2 6 \times 1 0 ^ { 1 }$ TCID $5 0$ /Reaction $( 1 . 7 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome   
equivalents/Reaction)

A VTM type was considered to be compatible for use with the Alere™ i Influenza A & B assay if expected results for positive and negative samples were achieved for all replicates tested.

Of the 16 VTM types evaluated in this study, the Brain Heart Infusion Broth was found to be not compatible or appropriate for use with Alere™ i Influenza A & B. The following statement is included in the “Sample Collection and Handling” section of the Product Insert: “Brain Heart Infusion Broth is not suitable for use in this assay.”

All other 15 VTM types evaluated in this study, Amies Transport Media, Starplex Multitrans Media, M4 Media, M4-RT Media, M5 Media, M6 Media, Liquid Stuart’s, Universal Viral Transport Media (UTM), Dulbecco’s Modified Eagles’ Medium (D

MEM), Tryptose Phosphate Broth, Hank’s Balanced Salt Solution, $0 . 9 \%$ Saline, Phosphate Buffered Saline, $2 . 5 \%$ Veal Infusion Broth, and Vircell Transport Media were found to be compatible for use with Alere™ i Influenza A & B.

# Flex Studies

To assess if Alere™ i Influenza A & B test performance testing direct nasal swab specimens (CLIA waived) is impacted if 1) a direct swab sample is tested using the Alere™ i Instrument Viral Transport Media setting; and 2) a direct swab sample is tested at different time points after the required Sample Receiver warm up time, but before the instrument times out, using the Alere™ i Instrument Viral Transport Media setting, two analytical flex studies were conducted.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot D1002B) and Influenza B/Malaysia/2506/2004 (Lot A1012A) were used in this study to prepare contrived swab samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) were determined by standard virologic method by the venders. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Positive samples were prepared using the influenza A and B strains (described above) diluted in UTM. Swab samples were prepared by scratching $1 0 \mu \mathrm { l }$ of the appropriate dilution onto a swab head. Swabs were inoculated immediately prior to testing.

The influenza A and influenza B positive swab samples with concentration approximately $2 . 5 \mathrm { X }$ times the respective LoD (as established in K141520) were tested as follows:

• Contrived influenza A swab: $2 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genome equivalents/mL; $8 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Reaction Contrived influenza B swab: $5 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome equivalents/mL; $2 . 0 0 \times 1 0 ^ { 1 }$ genome equivalents/Reaction

Negative control swabs included in the test kit were tested as the negative sample in the flex studies.

Negative control swabs, Flu A swabs, and Flu B swabs were first tested $( \mathtt { n } = 1 0 )$ according to the Alere™ i Influenza A & B VTM assay procedure, adding the swab when directed by the instrument to add the VTM sample.

All swabs samples tested using the Instrument’s VTM setting generated the correct Results. This data demonstrates that there is low risk of generating incorrect results if a user accidentally performs testing of a direct nasal swab sample using the Instrument’s VTM assay settings.

Negative control swabs, Flu A swabs, and Flu B swabs were then tested $( \mathtt { n } = 1 0 $ ) at the following conditions near the extremes of the time interval:

• Immediately after the warm up is complete (6 minutes after the Sample Receiver is added)   
• Immediately before the instrument times out (Note: As the instrument times out after 10 minutes, 9 minutes and 30 seconds after the Sample Receiver is added was used as this time point).

Negative control swabs, Flu A swabs, and Flu B swabs were also tested $\mathtt { n } = 5$ at the following conditions near the middle of the time interval:

• 7 minutes after the Sample Receiver was added $\mathtt { \le 3 0 }$ seconds).   
• 8 minutes after the Sample Receiver was added $\mathtt { \le 3 0 }$ seconds).   
• 9 minutes after the Sample Receiver was added $\mathtt { \le 3 0 }$ seconds).

All swab samples tested using the Instrument’s VTM setting generated the correct Results, except that two invalid results occurred due to a lack of internal control amplification. The first invalid result occurred when testing a negative control swab 7 minutes after the Sample Receiver was added. The second invalid result occurred when testing a negative control swab 9 minutes after the Sample Receiver was added. In both cases, the negative control swab was retested $\mathrm { n } { = } 5$ and expected results were obtained. This data demonstrates that there is low risk of generating incorrect results if a user accidentally performs testing of a direct nasal swab sample using the Instrument’s VTM assay settings and adds the swab sample at various time points after the Sample Receiver warm up but before the instrument times out.

# d. Detection Limit:

# LoD Study – With Natural Clinical Matrix

An analytical study was carried out to determine the LoD for each Alere™ i Influenza A & B assay targeted influenza virus in natural nasal swab matrix. Three strains of influenza A virus representing each of the three common currently or recently circulating influenza A subtypes (i.e., A/H1N1, A/H3N2 seasonal, and A/H1N1 pandemic (pdm)) and two strains of influenza B virus representing each of the two influenza B genetic lineages (i.e., Victoria and Yamagata) were included in this study.

Presumed negative nasal swab specimens were eluted in $3 ~ \mathrm { m L }$ of UTM. Swab eluates were combined and mixed thoroughly by gently vortexing to create a clinical matrix pool, and this clinical matrix pool was used as the diluent throughout this study. Vender provided stocks of Influenza A/Puerto Rico/8/34 (A/H1N1, Lot 152044RD), A/Perth/16/2009 (A/H3N2, Lot A1121B), A/California/7/2009 (A/H1N1 pdm, Lot 090794RD), Influenza B/Malaysia/2506/2004 (Victoria lineage, Lot C1320E), and B/Bangladesh/3333/2007 (Yamagata lineage, Lot H1012A) were diluted in the

natural clinical nasal swab matrix pool as described previously to generate virus dilutions for testing. The vender provided virus strains were re-titered and the concentrations (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ were determined by standard virologic method. The concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived swab specimens were prepared by coating $1 0 ~ \mu \mathrm { l }$ of virus dilution onto each swab. Swab coating was carried out by lightly scratching the swab surface with the pipette tip as virus dilution was pipetted to ensure liquid was absorbed into the swab tip. Each swab was eluted into a separate vial containing $3 ~ \mathrm { m l }$ of UTM. Two swab eluates were combined and mixed thoroughly by gently vortexing to prepare enough of a homogeneous sample for testing the required replicates in the study. Each swab eluate was tested on the Alere™ i Influenza A & B test according to the VTM assay procedure.

A range finding experiment was conducted initially testing triplicates per dilution per strain. If $100 \%$ positive results were obtained with the starting dilution, a series of 2- fold dilutions beginning with the dilution that produced $100 \%$ positive results were prepared and tested in triplicates until a negative result was obtained. 20 replicates of the lowest 2-fold dilution that produced $100 \%$ (3/3) positive results were then tested for the confirmation of the estimated limit of detection (LoD). If $2 9 5 \%$ positive results were achieved, the level was reported as the confirmed LoD. If ${ < } 9 5 \%$ positive results were obtained, 20 replicates of the next 2-fold dilution up from the dilution that produced a ${ < } 9 5 \%$ positive results were tested.

The confirmed LoD in the matrix of natural nasal swab eluted in VTM, defined as the lowest virus level that was detected $2 9 5 \%$ of the time (i.e., virus level at which at least 19 out of 20 replicates tested positive), for each influenza strain tested, are presented in Table 4 below:

Table 4: Limit of Detection (LoD) Study Results – Natural Nasal Swab Eluted in VTM Matrix   

<table><tr><td rowspan=2 colspan=1>Influenza Strain</td><td rowspan=2 colspan=1>Influenza ASubtype orInfluenza BGenetic Lineage</td><td rowspan=2 colspan=1>LoD(TCID 50/mL)</td><td rowspan=2 colspan=1>LoD(TCID 50/Swab)*</td><td rowspan=2 colspan=1>LoD(TCID 50/Reeaction)</td><td rowspan=1 colspan=1>LoD(Genome</td><td rowspan=2 colspan=1>LoD(GenomeEquivalents/Swab)*</td><td rowspan=2 colspan=1>LoD(Genomequuivalents/Reaction</td></tr><tr><td rowspan=1 colspan=1>Equivalents/mL)</td></tr><tr><td rowspan=1 colspan=1>A/Puerto Rico/8/34</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>4.20 x 105</td><td rowspan=1 colspan=1>4.20 x 103</td><td rowspan=1 colspan=1>1.04 x 101</td><td rowspan=1 colspan=1>4.59 x 106</td><td rowspan=1 colspan=1>4.59 x 104</td><td rowspan=1 colspan=1>1.13 x 102</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>9.82 x 103</td><td rowspan=1 colspan=1>9.82 x 101</td><td rowspan=1 colspan=1>2.40 x 10-1</td><td rowspan=1 colspan=1>1.25 x 106</td><td rowspan=1 colspan=1>1.25 x 104</td><td rowspan=1 colspan=1>3.10 x 101</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>5.20 x 10^}$</td><td rowspan=1 colspan=1>5.20 x 10^3</td><td rowspan=1 colspan=1>1.28 x 10</td><td rowspan=1 colspan=1>7.77 x 106</td><td rowspan=1 colspan=1>7.77 x 104</td><td rowspan=1 colspan=1>1.92 x 102</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>B Victoria lineage</td><td rowspan=1 colspan=1>1.05 x 10</td><td rowspan=1 colspan=1>1.05 x 103</td><td rowspan=1 colspan=1>2.66 x 10</td><td rowspan=1 colspan=1>2.29 x 106</td><td rowspan=1 colspan=1>2.29 x 104</td><td rowspan=1 colspan=1>5.70 x 10</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/3333/2007</td><td rowspan=1 colspan=1>B Yamagatalineage</td><td rowspan=1 colspan=1>1.34 x 104</td><td rowspan=1 colspan=1>1.34 x 102</td><td rowspan=1 colspan=1>3.30 x 10-1</td><td rowspan=1 colspan=1>1.98 x 106</td><td rowspan=1 colspan=1>1.98 x 104</td><td rowspan=1 colspan=1>4.90 x 101</td></tr></table>

As a direct comparison, the confirmed LoD in natural nasal swab matrix for each influenza strain tested in K141520, are presented in Table 5 below:

Table 5: Limit of Detection (LoD) Study Results – Natural Nasal Swab Matrix (K141520)   

<table><tr><td rowspan=2 colspan=1>Influenza Strain</td><td rowspan=2 colspan=1>Influenza ASubtype orIn fluenza BGenetic Lineage</td><td rowspan=2 colspan=1>LoD(TCID 50/mL</td><td rowspan=2 colspan=1>LoD(TCID 50/Swab)*</td><td rowspan=1 colspan=1>LoD(TCID 50/</td><td rowspan=2 colspan=1>LoD(GenomeEquivalents/mL)</td><td rowspan=2 colspan=1>LoD(GenomeEquivalents/Swab)*</td><td rowspan=2 colspan=1>LoD(GenomeEquuivalents/Reaction</td></tr><tr><td rowspan=1 colspan=1>Reaction)</td></tr><tr><td rowspan=1 colspan=1>A/Puerto Rico/8/34</td><td rowspan=1 colspan=1> /HIN1</td><td rowspan=1 colspan=1>1.88 x 105</td><td rowspan=1 colspan=1>1.88 x 103</td><td rowspan=1 colspan=1>7.52 x 101</td><td rowspan=1 colspan=1>4.22 x 106</td><td rowspan=1 colspan=1>4.22 x 104</td><td rowspan=1 colspan=1>1.69 x 103</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>8.60 x 102</td><td rowspan=1 colspan=1>8.60 x 100</td><td rowspan=1 colspan=1>3.44 x 10-1</td><td rowspan=1 colspan=1>7.91 x 104</td><td rowspan=1 colspan=1>7.91 x 102</td><td rowspan=1 colspan=1>3.16 x 101</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>1.25 x 104</td><td rowspan=1 colspan=1>1.25 x 102</td><td rowspan=1 colspan=1>5.00 x 100</td><td rowspan=1 colspan=1>5.20 x 106</td><td rowspan=1 colspan=1>5.20 x 104</td><td rowspan=1 colspan=1>2.08 x 103</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>B Victoria lineage</td><td rowspan=1 colspan=1>1.90 x 103</td><td rowspan=1 colspan=1>1.90 x 101</td><td rowspan=1 colspan=1>7.60 x 10-1</td><td rowspan=1 colspan=1>7.24 x 104</td><td rowspan=1 colspan=1>7.24 x 102</td><td rowspan=1 colspan=1>2.90 x 101</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/3333/2007</td><td rowspan=1 colspan=1>B Yamagatalineage</td><td rowspan=1 colspan=1>5.55 x 102</td><td rowspan=1 colspan=1>5.55 x 100</td><td rowspan=1 colspan=1>2.22 x 10-1</td><td rowspan=1 colspan=1>7.36 x 104</td><td rowspan=1 colspan=1>7.36 x 102</td><td rowspan=1 colspan=1>2.94 x 101</td></tr></table>

# e. Analytical Reactivity:

Analytical Reactivity Testing

An analytical reactivity (inclusivity) study was performed to determine whether the Alere™ i Influenza A & B assay is able to detect a variety of influenza A and B strains that represent temporal and geographic diversity.

Vender provided stocks of influenza A and B strains were diluted in UTM to generate virus dilutions for testing. The concentration (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ ) for each was determined by standard virologic method. The concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

The serial dilutions in UTM were tested on the Alere™ i Influenza A & B test according to the VTM assay procedure.

The starting dilution concentration selected for testing in this study was higher than the established LoDs in the Limit of Detection studies. Each starting dilution per strain was initially tested in triplicates. If the initial testing concentration tested positive for all three replicates, the strain was further diluted 10-fold and tested in triplicates until at least one out three replicates generated a negative result. When a negative result was obtained, additional 2-fold dilutions were tested, starting from the highest dilution that produced $100 \%$ (3/3) positive results. A concentration level was considered “reactive/positive” in this study if all three replicates generated a positive result for the expected influenza virus. A concentration level was considered “non-reactive/negative” in this study if at least one out of the three replicates generated a negative result for the expected influenza virus.

The Alere™ i Influenza A & B assay following the VTM assay procedure was evaluated with 16 Influenza A and 10 Influenza B strains. Influenza A strains included three pre-2009 pandemic seasonal A/H1N1 strains, eight seasonal H3N2 strains, three 2009 A/H1N1 pandemic strains, one A/H3N2v strain, and one avian influenza A/H7N9 virus (detected in China in 2013). Influenza B strains included five Victoria lineage strains and five Yamagata lineage strains. The Alere™ i

Influenza A & B assay following the VTM assay detected all strains tested at the concentrations indicated in Table 6 below:

Table 6: Analytical Reactivity Study Results   

<table><tr><td></td><td rowspan=1 colspan=1>Alaly tlcal</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Influenza Strain</td><td rowspan=1 colspan=1>InfluenzaA SubtypeorInfluenzaB  GeneticLineage</td><td rowspan=1 colspan=5>Test Concentration (in TCID50 or Genome Equivalents , unlessindicated otherwise)</td><td rowspan=1 colspan=1>Flu AResult(n=3,unlessindicateotherwise)</td><td rowspan=2 colspan=1>Flu BResult(n=3, unlessindicateotherwise)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>TCID50/Reaction</td><td rowspan=1 colspan=1>GenomeEquivalents/mL</td><td rowspan=1 colspan=2>GenomeEquivalents/Reaction</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>9.20 x 104</td><td rowspan=1 colspan=1>6.81 x 102</td><td rowspan=1 colspan=1>2.79 x 10 $</td><td rowspan=1 colspan=2>2.07 x 10^3</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>3.41 x 100</td><td rowspan=1 colspan=1>2.53 x 10-2</td><td rowspan=1 colspan=1>1.95 x 104</td><td rowspan=1 colspan=2>1.45 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>2.11 x 103</td><td rowspan=1 colspan=1>1.56 x 101</td><td rowspan=1 colspan=1>4.79 x 104</td><td rowspan=1 colspan=2>3.55 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>4.22 x 103</td><td rowspan=1 colspan=1>3.13 x 101</td><td rowspan=1 colspan=1>1.79 x 105</td><td rowspan=1 colspan=2>1.33 x 103</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.43 x 10-1</td><td rowspan=1 colspan=1>1.06 x 10-3</td><td rowspan=1 colspan=1>2.70 x 10</td><td rowspan=1 colspan=2>8.99 x 10</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>5.20 x 103</td><td rowspan=1 colspan=1>3.85 x 10</td><td rowspan=1 colspan=1>1.21 x 104</td><td rowspan=1 colspan=2>1.18 x 103</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>3.68 x 104</td><td rowspan=1 colspan=1>2.73 x 102</td><td rowspan=1 colspan=1>4.06 x 10</td><td rowspan=1 colspan=1>3.01 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>4.26 x 103</td><td rowspan=1 colspan=1>3.16 x 101</td><td rowspan=1 colspan=1>1.71 x 10</td><td rowspan=1 colspan=2>1.26 x 10^3</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.58 x 10</td><td rowspan=1 colspan=1>1.17 x 10-1</td><td rowspan=1 colspan=1>2.51 x 104</td><td rowspan=1 colspan=2>1.86 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.19 x 10-1</td><td rowspan=1 colspan=1>8.10 x 10-4</td><td rowspan=1 colspan=1>9.60 x 101</td><td rowspan=1 colspan=2>7.11 x 10-1</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.51 x 100</td><td rowspan=1 colspan=1>1.12 x 10-2</td><td rowspan=1 colspan=1>3.91 x 104</td><td rowspan=1 colspan=2>2.90 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/California/4/2009</td><td rowspan=1 colspan=1>A/H1N1(pdm)</td><td rowspan=1 colspan=1>3.14 x 102</td><td rowspan=1 colspan=1>2.33 x 100</td><td rowspan=1 colspan=1>1.07 x 104</td><td rowspan=1 colspan=2>7.96 x 101</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Maryland/04/2011</td><td rowspan=1 colspan=1>A/H1N1(pdm)</td><td rowspan=1 colspan=1>3.90 x 103</td><td rowspan=1 colspan=1>2.89 x 101</td><td rowspan=1 colspan=1>1.23 x 105</td><td rowspan=1 colspan=2>9.12 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/New York/18/2009</td><td rowspan=1 colspan=1>A/H1N1(pdm)</td><td rowspan=1 colspan=1>6.75 x 100</td><td rowspan=1 colspan=1>5.00 x 10-2</td><td rowspan=1 colspan=1>2.72 x 104</td><td rowspan=1 colspan=2>2.01 x 102</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2013(Inactivated)a</td><td rowspan=1 colspan=1>A/H7N9(Detectedin China in2013)</td><td rowspan=1 colspan=1>4.00 x 107EID 50/mL</td><td rowspan=1 colspan=1>2.96 x 105EID50/Reaction</td><td rowspan=1 colspan=1>1.07 x 107</td><td rowspan=1 colspan=2>7.95 x 104</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Indiana/10/2011a</td><td rowspan=1 colspan=1>A/H3N2v</td><td rowspan=1 colspan=1>3.16 x 107EID50/mL</td><td rowspan=1 colspan=1>2.34 x 105EID50/Reaction</td><td rowspan=1 colspan=1>6.70 x 102</td><td rowspan=1 colspan=2>4.96 x 100</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>VictoriaLineage</td><td rowspan=1 colspan=1>2.00 x 101CEID50/mL</td><td rowspan=1 colspan=1>1.48 x 10-1CEID50/Reaction</td><td rowspan=1 colspan=1>5.26 x 10^3</td><td rowspan=1 colspan=2>3.90 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Victoria/504/2000</td><td rowspan=1 colspan=1>VictoriaLineage</td><td rowspan=1 colspan=1>2.50 x 102</td><td rowspan=1 colspan=1>1.85 x 10°</td><td rowspan=1 colspan=1>1.25 x 104</td><td rowspan=1 colspan=2>9.23 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>VictoriaLineage</td><td rowspan=1 colspan=1>4.40 x 101</td><td rowspan=1 colspan=1>3.26 x 10-1</td><td rowspan=1 colspan=1>1.60 x 103</td><td rowspan=1 colspan=2>1.19 x 101</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Montana/05/2012</td><td rowspan=1 colspan=1>VictoriaLineage</td><td rowspan=1 colspan=1>5.00 x 100</td><td rowspan=1 colspan=1>3.70 x 10-2</td><td rowspan=1 colspan=1>2.62 x 103</td><td rowspan=1 colspan=2>1.94 x 101</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>1.36 x 103</td><td rowspan=1 colspan=1>1.01 x 101</td><td rowspan=1 colspan=1>1.08 x 105</td><td rowspan=1 colspan=2>8.01 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>7.90 x 101</td><td rowspan=1 colspan=1>5.58 x 10-1</td><td rowspan=1 colspan=1>6.19 x 105</td><td rowspan=1 colspan=2>4.59 x 103</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>3.68 x 103</td><td rowspan=1 colspan=1>2.73 x 101</td><td rowspan=1 colspan=1>1.36 x 105</td><td rowspan=1 colspan=2>1.01 x 103</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>6.25 x 100</td><td rowspan=1 colspan=1>4.63 x 10-2</td><td rowspan=1 colspan=1>4.22 x 104</td><td rowspan=1 colspan=2>3.12 x 102</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>3.00 x 103</td><td rowspan=1 colspan=1>2.22 x 101</td><td rowspan=1 colspan=1>1.39 x 105</td><td rowspan=1 colspan=2>1.03 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr></table>

a Although this test has been shown to detect A/H1N1 (pre-2009 pandemic), A/H7N9 (detected in China in 2013) and A/H3N2v viruses cultured from positive human respiratory specimens, the performance characteristics of this device with clinical specimens that are positive for the A/H1N1 (pre-2009 pandemic), A/H7N9 (detected in China in 2013) and A/H3N2v viruses have not been established.

The analytical reactivity reported in this $5 1 0 ( \mathrm { k } )$ (expressed in Genome Equivalents/Reaction) is very similar to what was reported in K141529. As a direct comparison, a side-by-side listing of the analytical reactivity submitted in K141529 and the analytical reactivity submitted in this $5 1 0 ( \mathrm { k } )$ submission (expressed in Genome Equivalents/Reaction) is presented in Table 7 below:

Table 7: Side-by-Side Comparison of Analytical Reactivity Submitted in K141529 vs. K151464   

<table><tr><td rowspan=2 colspan=1>Influenza Strain</td><td rowspan=2 colspan=1>Influenza ASubtype orInfluenza BGenetic Lineage</td><td rowspan=1 colspan=4>Test Concentration (in Genome Equivalents)</td></tr><tr><td rowspan=1 colspan=2>K141529 (Direct Swab)</td><td rowspan=1 colspan=2>K151464 (VTM)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>GenomeEquivalents/Reaction</td><td rowspan=1 colspan=2>GenomeEquivalents/Reaction</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>1.64 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.07 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>6.08 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.45 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>1.36 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.55 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>5.24 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.33 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.08 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.99 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>2.99 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.18 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.36 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.01 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.03 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.26 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.35 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.86 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>2.54 x 100</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.11 x 10-1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>1.41 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.90 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/California/4/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>4.28 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.96 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Maryland/04/2011</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>1.52 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9.12 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/New York/18/2009</td><td rowspan=1 colspan=1>A/H1N1 (pdm)</td><td rowspan=1 colspan=1>3.66 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.01 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2013(Inactivated)</td><td rowspan=1 colspan=1>A/H7N9(Detected inChina in 2013)</td><td rowspan=1 colspan=1>6.86 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.95 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Indiana/10/2011</td><td rowspan=1 colspan=1>A/H3N2v</td><td rowspan=1 colspan=1>2.38 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4.96 x 100</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=1>2.16 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.90 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Victoria/504/2000</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=1>2.50 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9.23 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=1>3.32 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.19 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Montana/05/2012</td><td rowspan=1 colspan=1>Victoria Lineage</td><td rowspan=1 colspan=1>1.02 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.94 x 101</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>4.52 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.01 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>1.18 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4.59 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>4.66 x 102</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.01 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>9.12 x 101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.12 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>YamagataLineage</td><td rowspan=1 colspan=2>8.00 x 102</td><td rowspan=1 colspan=1>1.03 x 103</td><td rowspan=1 colspan=1></td></tr></table>

Supplemental Reactivity Information for Influenza Strains of Human, Swine and Avian Origin (in silico analysis):

The purpose of this $5 1 0 ( \mathrm { k } )$ submission is to expand the Intended Use to allow the test to be used on nasal or nasopharyngeal swabs eluted in viral transport media (VTM) and the test reagents were not modified, therefore additional Reactivity Study data are not necessary.

Please refer to the Supplemental Reactivity Information submitted in K141520.

# f. Analytical Specificity:

The purpose of this submission is to expand the Intended Use to allow the test to be used on nasal or nasopharyngeal swabs eluted in viral transport media and the test reagents were not modified, therefore additional Cross-Reactivity testing data are not necessary.

Please refer to the Cross-Reactivity testing data submitted in K141520.

# g. Potentially Interfering Substances:

An analytical study was performed to assess the potential interference effects of 25 substances naturally present in respiratory specimens or that may be artificially introduced into the nasal cavity/nasopharynx.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD), and Influenza B/Malaysia/2506/2004 (Lot C1320E1) were used in this study to prepare contrived positive samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above) diluted to a concentration approximately 3 times the respective limit of detection (LoD) levels and the various interfering substances at the concentrations specified in Table 8 below. Specifically, $0 . 1 \mathrm { m L }$ of virus dilution in UTM (or just UTM for influenza negative samples) was added to $0 . 9 \mathrm { m L }$ of the interfering substance. Each influenza strain, combined with each interfering substance, was evaluated separately. Negative samples were prepared using UTM and the various interfering substances at the concentrations specified in Table 8 below.

$5 0 \mu \mathrm { L }$ of each influenza positive and negative sample with interfering substance was pipetted into a microcentrifuge tube. The solution in the tube was absorbed onto a foam swab. The swab was eluted into $3 ~ \mathrm { m L }$ of UTM and mixed well. 200 $\mu \mathrm { L }$ of the swab eluate was then tested according to the VTM assay procedure. Each sample was tested in triplicates per the product instructions.

The influenza A and influenza B positive samples were tested as follows:

Influenza A/Puerto Rico/8/34: $2 . 6 0 \mathrm { \times 1 0 ^ { 6 } T C I D _ { 5 0 } / m L }$ $( 2 . 7 5 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ genome equivalents $/ \mathrm { m L }$ ); $3 . 2 0 \times 1 0 ^ { 1 }$ TCID $5 0$ /Reaction $( 3 . 4 0 \mathrm { ~ x ~ i ~ } 0 ^ { 2 }$ genome

equivalents/Reaction)   
Influenza B/Malaysia/2506/2004: 2.65 x 106 TCID50/mL ( $1 . 3 8 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ genome   
equivalents $/ \mathrm { m L }$ ); $\mathbf { \dot { 3 } } . 2 7 \times 1 0 ^ { 1 }$ TCID $5 0$ /Reaction $1 . 7 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome   
equivalents/Reaction)

The Interfering Substances Study Results are presented in Table 8 below.

Table 8: Interfering Substances Study Results   

<table><tr><td colspan="1" rowspan="2">Substance Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="2" rowspan="1">Negative Samples</td><td colspan="2" rowspan="1">Influenza Positive Samples</td></tr><tr><td colspan="1" rowspan="1">Flu AResult</td><td colspan="1" rowspan="1">Flu BResult</td><td colspan="1" rowspan="1">Flu A Result(A/PR/8/34)</td><td colspan="1" rowspan="1">Flu B Result(B/Malaysia/2506/2004)</td></tr><tr><td colspan="1" rowspan="3">Mucin</td><td colspan="1" rowspan="3">20 μg /mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Whole Blood</td><td colspan="1" rowspan="3">50 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Zicam Extreme CongestionRelief</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">NeoSynephrine Cold &amp;Sinus Extra Strength Spray</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Afrin PumpMist Original</td><td colspan="1" rowspan="3">200 μl/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">4-acetamidophenol</td><td colspan="1" rowspan="3">200 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Acetylsalicylic acid</td><td colspan="1" rowspan="3">650 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+*</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Albuterol</td><td colspan="1" rowspan="3">400 ng/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Chlorpheniramine</td><td colspan="1" rowspan="3">145 ng/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Dexamethasone</td><td colspan="1" rowspan="3">0.80 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Dextromethorphan</td><td colspan="1" rowspan="3">1 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Diphenhydramine</td><td colspan="1" rowspan="3">5 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Doxylamine Succinate</td><td colspan="1" rowspan="3">236 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Ephedrine</td><td colspan="1" rowspan="3">273 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="2">Flunisolide</td><td colspan="1" rowspan="2">6.8 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Guaiacol glycerol ether</td><td colspan="1" rowspan="3">3.5 ng/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Mupirocin</td><td colspan="1" rowspan="3">12 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Oxymetazoline</td><td colspan="1" rowspan="3">0.6 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Phenylephrine</td><td colspan="1" rowspan="3">12 mg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Rebetol</td><td colspan="1" rowspan="3">4.5 μg /mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Relenza</td><td colspan="1" rowspan="3">282 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Rimantadine</td><td colspan="1" rowspan="3">282 ng/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Tamiflu</td><td colspan="1" rowspan="3">1.1 μg/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Tobramycin</td><td colspan="1" rowspan="3">2.43 mg/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="3">Triamcinolone</td><td colspan="1" rowspan="3">40 μg /mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr></table>

\*Sample was retested $\scriptstyle \mathrm { n = 6 }$ , with expected negative results obtained.

None of the 25 potential interference substances tested produced false positive or false negative test results in Alere™ i Influenza A & B at the concentrations specified, except for Acetylsalicylic acid at $6 5 0 ~ \mu \mathrm { g / m L }$ tested without influenza. This sample was retested six time and the expected negative results were obtained.

h. Potential Microbial Interference Study:

Low Loads of Influenza A or Influenza B Co-Spiked with Selected Non-influenza Respiratory Viruses

An analytical study was performed to assess any potential interference of high concentrations of selected non-influenza respiratory viruses when co-spiked with low loads of influenza A or influenza B.

The non-influenza respiratory viral stocks used in this study are listed in Table 9 below.

Table 9: Selected Non-Influenza Respiratory Viral Stocks Tested   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>ATCC Strain</td><td rowspan=1 colspan=1>Lot Number</td><td rowspan=1 colspan=1>Stock Concentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>VR-1</td><td rowspan=1 colspan=1>59772750</td><td rowspan=1 colspan=1>1.58 x 107</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>VR-1559</td><td rowspan=1 colspan=1>57685054</td><td rowspan=1 colspan=1>1.58 x 107</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus(RSV) Type B, Strain 18537</td><td rowspan=1 colspan=1>VR-1580</td><td rowspan=1 colspan=1>58214862</td><td rowspan=1 colspan=1>8.89 x 105</td></tr></table>

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD) and Influenza B/Malaysia/2506/2004 (Lot C1320E) were used in this study to prepare contrived positive samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents $ { ' } \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above). The contrived influenza A and influenza B positive VTM samples were prepared as follows:

• Contrived influenza A VTM Sample: $6 . 9 3 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } ( 7 . 3 $ ${ 7 . 3 4 \mathrm { ~ x ~ } 1 0 ^ { 4 } }$ genome equivalents/mL);   
• Contrived influenza B VTM Sample: $7 . 0 1 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L } ( 3 . 6 7 \mathrm { ~ x ~ } 1 $ genome equivalents/mL).

For each of the three non-influenza respiratory viruses, $3 0 0 \mu \mathrm { L }$ of each viral stock (described in Table 9 above) was combined with $5 0 0 \mu \mathrm { L }$ of the Flu A or Flu B dilutions in VTM. $2 0 0 \mu \mathrm { L }$ of each combined sample was then tested according to the VTM assay procedure. Each sample was tested in triplicates per the product instructions. Three contrived influenza A positive VTM samples and three contrived influenza B positive VTM samples without co-infection were also tested as positive experimental controls.

The final tested influenza A or influenza B concentration per sample was approximately 3 X the respective LoD, $3 . 2 1 \mathrm { x } 1 0 ^ { 1 }$ TCID50/Reaction $( 3 . 4 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Reaction) for influenza A and $3 . 2 5 \mathrm { x } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } \mathrm { / R }$ eaction (1.70 $\bar { \mathrm { ~ x ~ } } 1 0 ^ { 2 }$ genome equivalents/Reaction) for influenza B.

The Potential Microbial Interference – Non-Influenza Respiratory Viruses Study Results are presented in Table 10 below.

Table 10: Microbial Interference Study Results – Selected Non-Influenza Respiratory Viruses   

<table><tr><td rowspan=2 colspan=1>Non-InfluenzaRespiratory Virus</td><td rowspan=2 colspan=1>Non-InfluenzaRespiratory VirusConcentrationTested</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveVTM Sample</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveVTM Sample</td></tr><tr><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=3 colspan=1>None</td><td rowspan=3 colspan=1>None</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=3 colspan=1>9.48 x 106TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus</td><td rowspan=3 colspan=1>9.48 x 106TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=3 colspan=1>5.33 x 105TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr></table>

No competitive inhibitory effect from co-infecting non-influenza respiratory virus was observed for this study.

Low Loads of Influenza A or Influenza B Co-Spiked with Influenza Virus

An analytical study was performed to assess any potential competitive inhibitory effects of high concentrations of influenza virus when co-spiked with low loads of influenza A or influenza B.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD) and Influenza B/Malaysia/2506/2004 (Lot C1320E1) were used in this study to prepare contrived positive VTM samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were determined by standard virologic method. The targeted concentration for each dilution (in genome equivalents $/ \mathrm { m L }$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Using UTM as the diluent, positive samples were prepared using the influenza A and B strains (described above).

The contrived influenza A and influenza B positive VTM samples were prepared as follows:

Contrived High Positive influenza A VTM Sample: $8 . 6 7 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ (9.18 $\mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome equivalents/mL);   
• Contrived Low Positive influenza A VTM Sample: $8 . 6 7 \mathrm { { x } 1 0 ^ { 3 } \mathrm { { T C I D } _ { 5 0 } / m L \left( 9 . 1 8 \mathrm { { x } } \right. } }$ $1 0 ^ { 4 }$ genome equivalents/mL);   
• Contrived High Positive influenza B VTM Sample: $8 . 8 3 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ (4.62

x $1 0 ^ { 5 }$ genome equivalents/mL);

Contrived Low Positive influenza A VTM Sample: $8 . 8 3 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ $( 4 . 6 2 \mathrm { x }$ $1 0 ^ { 4 }$ genome equivalents/mL).

To prepare the co-infection VTM samples, $4 0 0 \mu \mathrm { L }$ of contrived high positive influenza A VTM sample was combined with $4 0 0 \mu \mathrm { L }$ of contrived low positive influenza B VTM sample. Likewise, $4 0 0 \mu \mathrm { L }$ of contrived high positive influenza B VTM sample was combined with $4 0 0 \mu \mathrm { L }$ of contrived low positive influenza A VTM sample. Each combined sample was tested in triplicates per the product instructions. Three contrived influenza A low positive VTM samples without co-infection and three contrived influenza B low positive samples without co-infection were also tested as positive experimental controls.

The final tested influenza A or influenza B low positive concentration per sample was approximately $3 \mathrm { X }$ the respective LoD, $3 . 2 1 \mathrm { x } 1 \bar { 0 } ^ { 1 }$ T $\mathrm { C I D } _ { 5 0 }$ /Reaction $( 3 . \dot { 4 } 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ genome equivalents/Reaction) for influenza A and $3 . 2 5 \mathrm { x } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { R }$ eaction (1.70 $\bar { \mathrm { ~ x ~ } } 1 0 ^ { 2 }$ genome equivalents/Reaction) for influenza B. The final tested influenza A or influenza B high positive concentration per sample (as potential interfering coinfection) was approximately $3 0 \mathrm { X }$ the respective LoD, $3 . 2 1 \mathrm { x } \ 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } \mathrm { \overline { { / } } }$ Reaction $( 3 . 4 0 \mathrm { ~ x ~ } 1 \mathrm { 0 } ^ { 3 }$ genome equivalents/Reaction) for influenza A and $3 . 2 5 \mathrm { x } 1 0 ^ { 2 }$ $\mathrm { T C I D } _ { 5 0 }$ /Reaction $( 1 . 7 \dot { 0 } \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents/Reaction) for influenza B.

The Potential Microbial Interference – Influenza Viruses Study Results are presented in Table 11 below.

Table 11: Microbial Interference Study Results –Influenza Viruses   

<table><tr><td rowspan=2 colspan=1>Co-Infecting InfluenzaVirus</td><td rowspan=2 colspan=1>Co-Infecting InfluenzaVirus ConcentrationTested</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveVTM Sample</td><td rowspan=1 colspan=2>Contrived Influenza A Positive Contrived Influenza B PositiveVTM Sample</td></tr><tr><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td><td rowspan=1 colspan=1>Flu A Result</td><td rowspan=1 colspan=1>Flu B Result</td></tr><tr><td rowspan=3 colspan=1>None</td><td rowspan=3 colspan=1>None</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=3 colspan=1>3.40 x 103 genomeequivalents/Reaction</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=3 colspan=1>1.70 x 103 genomeequivalents/Reaction</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr></table>

No competitive inhibitory effect from co-infecting influenza virus was observed for this study.

# i. Carry-Over Study:

A study was conducted to evaluate the risk of carry-over contamination with the Alere™ i Influenza A & B assay according to the VTM procedure on the Alere™ i instrument.

Dilutions of Influenza A/Puerto Rico/8/34 (Lot 012234RD) and Influenza B/Malaysia/2506/2004 (Lot C1320E1) were used in this study to prepare contrived samples for testing. The concentrations of the viral stocks (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) were determined by standard virologic method prior to inactivation by the venders. The targeted concentration for each dilution (in genome equivalents $\mathrm { \Delta } m \mathrm { \Delta } L$ ) was also assessed using laboratory developed and validated influenza A and influenza B quantitative real-time PCR assays.

Contrived positive samples were prepared using the influenza A and B strains (described above) dilutions in UTM with targeted concentrations of approximately 30 times respective LoD. Negative samples were prepared in UTM by eluting one negative control in $3 ~ \mathrm { m L }$ of UTM

The contrived influenza A and influenza B positive VTM samples were tested as follows:

Contrived influenza A and B positive VTM samples:

For Flu A: $3 . 2 1 \mathrm { x } \ 1 0 ^ { 2 } \ \mathrm { \bar { T } C I D } _ { 5 0 } / ]$ Reaction ( $\bar { 3 } . 3 9 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents/Reaction)   
For Flu B: $3 . 2 7 \mathrm { x } 1 0 ^ { 2 }$ TCID $5 0$ /Reaction $( 1 . 7 1 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ genome equivalents/Reaction)

Positive and negative samples were tested per the product instructions. Testing between positive and negative samples alternated for a total of 30 rounds (30 positives and 30 negatives) on one Alere™ i instrument.

No carry-over/contamination event was observed in this study except that one Flu B false positive result was generated on a negative sample. The elution buffer for this sample was retained and retested using a new Test Base, and the results were as expected. Additionally, the remaining elution buffer was tested via qPCR for presence of Flu B RNA but was not detected.

This data demonstrated that there is low risk of carryover/contamination when testing positive and negative samples on the Alere™ i instrument when the product insert instructions are followed.

# j. Assay cut-off:

The purpose of this $5 1 0 ( \mathrm { k } )$ submission is to expand the Intended Use to allow the test to be used on nasal or nasopharyngeal swabs eluted in viral transport media. As result of this change in Intended Use, there has been no change to the algorithm thresholds for making positive and negative calls. Please refer to the Cut-Off information submitted in K141520 for a description of the methods used to establish these thresholds.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Performance of Alere™ i Influenza A & B was evaluated against the comparator method of an FDA-cleared nucleic acid test in a prospective clinical study.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Study:

Clinical performance characteristics of the Alere™ i Influenza A & B testing nasal and nasopharyngeal swab specimens eluted in VTM were evaluated in a multi-site prospective study during the 2014-2015 influenza season in the U.S. A total of three investigational sites throughout the U.S. participated in the study. To be enrolled in the study, patients had to be presenting at the participating study centers with flu-like symptoms. De-identified residual nasal or nasopharyngeal swab eluted in VTM specimens from patients with flu-like symptoms that have not been previously frozen were enrolled for testing using the Alere™ i Influenza A & B at the three study sites. An FDA-cleared Influenza RT-PCR assay was utilized as the comparator method for this study.

After completion of standard laboratory testing for influenza, one aliquot of each enrolled VTM specimen $( 2 0 0 \mu \mathrm { l } )$ was tested with Alere™ i Influenza A & B, according to product instructions. The remaining sample in VTM was pipetted into aliquots of $2 0 0 - 3 0 0 { \mu } 1$ , labeled and stored frozen at $- 8 0 ^ { \circ } \mathrm { C }$ (or $- 2 0 ^ { \circ } \mathrm { C }$ if $- 8 0 ^ { \circ } \mathrm { C }$ storage was not possible) for later shipment to the laboratory for comparator testing.

Two of the three clinical study sites (Site 015A and Site 002A) sent their VTM samples to a central testing laboratory for comparator testing. This central testing laboratory was located at Site 002A, which also participated as a sample collection and Alere™ i Influenza A & B testing site. One of the three clinical study sites (Site 009A) sent their VTM samples to another laboratory for comparator testing. This laboratory did not participate as a sample collection and Alere i testing site.

All samples generating discrepant Alere™ i Influenza A & B and comparator test results were tested using an alternative FDA-cleared Influenza RT-PCR assay at Alere Scarborough, Inc. to confirm influenza status.

External control testing, using Alere™ i Influenza A & B Positive and Negative Controls, was performed prior to sample testing each day and on each Alere™ i instrument the testing was performed, at all study sites.

A total of 1273 nasal swab (NS) or nasopharyngeal swab (NPS) eluted in VTM specimens (1120 NPS in VTM samples and 153 NS in VTM samples) were enrolled in this study. Of those, three specimens did not meet eligibility criteria. A total of 1270 specimens were included in the study. Patient age and gender distribution for the 1270 specimens is presented in Table 12 below.

Table 12: Age and Gender Distribution –Nasal Swab or Nasopharyngeal Swab in VTM Study   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>282</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>636</td><td rowspan=1 colspan=1>634</td></tr></table>

A total of 27 specimens included in the study were invalid by the Alere™ i Influenza A & B but could not be retested further due to insufficient residual specimen volume, and were excluded from performance analyses. A total of 1243 samples (1096 NPS in VTM samples and 147 NS in VTM samples) were included in performance analyses.

Compared to the comparator method, the performance of Alere™ i Influenza A & B for influenza A and influenza B are presented in Table 13 and Table 14 below, respectively.

Table 13: Alere™ i Influenza A & B Influenza A VTM Sample Performance against the Comparator   

<table><tr><td rowspan=2 colspan=1>AlereTInfluenza A &amp; B- Flu A</td><td rowspan=1 colspan=3>Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>221</td><td rowspan=1 colspan=1>35a</td><td rowspan=1 colspan=1>256</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>982</td><td rowspan=1 colspan=1>987</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>1017</td><td rowspan=1 colspan=1>1243</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:221/226      97.8% (95%CI: 94.9%-99.1%)</td></tr><tr><td rowspan=1 colspan=4>Specificity:982/1017     96.6% (95%CI: 95.3%-97.5%)</td></tr></table>

a Flu A nucleic acid was detected in 31/35 False Positive specimens using an alternative FDA-cleared molecular test

Table 14: Alere™ i Influenza A & B Influenza B VTM Sample Performance against the Comparator   

<table><tr><td rowspan=2 colspan=1>AlereTMInfluenza A &amp; B Flu B</td><td rowspan=1 colspan=3>Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>19a</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1125</td><td rowspan=1 colspan=1>1132</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>1144</td><td rowspan=1 colspan=1>1243</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:92/99        92.9% (95%CI:86.1%-96.5%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1125/1144   98.3% (95%CI: 97.4%-98.9%)</td></tr></table>

a Flu B nucleic acid was detected in 3/19 False Positive specimens using an alternative FDA-cleared molecular test

Site to site performance of Alere™ i Influenza A & B for influenza A and influenza B comparing to the comparator are presented in Table 15 and Table 16 below, respectively.

Table 15: Site by Site VTM Sample Performance Obtained for Influenza A with the Alere™ i Influenza A & B in Comparison to the Comparator   

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence </td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1 (002A)</td><td rowspan=1 colspan=1>22.7% (79/348)</td><td rowspan=1 colspan=1>98.7% (78/79)(93.2%, 99.8)</td><td rowspan=1 colspan=1>96.3% (259/269)c(93.3%, 98.0%)</td></tr><tr><td rowspan=1 colspan=1>Site 2 (009A)</td><td rowspan=1 colspan=1>16.6% (120/724)</td><td rowspan=1 colspan=1>98.3% (118/120)(94.1%, 99.5%)</td><td rowspan=1 colspan=1>97.7% (590/604)d(96.1%, 98.6%)</td></tr><tr><td rowspan=1 colspan=1>Site 3 (015A)</td><td rowspan=1 colspan=1>15.8% (27/171)</td><td rowspan=1 colspan=1>92.6% (25/27)(76.6%, 97.9%)</td><td rowspan=1 colspan=1>92.4% (133/144)e(86.8%, 95.7%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>18.2% (226/1243)</td><td rowspan=1 colspan=1>97.8% (221/226)(94.9%, 99.1%)</td><td rowspan=1 colspan=1>96.6% (982/1017)(95.3%, 97.5%)</td></tr></table>

a Prevalence based on the comparator method only b Confidence interval c Flu A nucleic acid was detected in 10/10 False Positive specimens using an alternative FDA-cleared molecular test d Flu A nucleic acid was detected in 10/14 False Positive specimens using an alternative FDA-cleared molecular test e Flu A nucleic acid was detected in 11/11 False Positive specimens using an alternative FDA-cleared molecular test

Table 16: Site by Site VTM Sample Performance Obtained for Influenza B with the Alere™ i Influenza A & B in Comparison to Comparator   

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=1>Prevalence </td><td rowspan=1 colspan=1>Sensitivity with 95% CI b</td><td rowspan=1 colspan=1>Specificity with 95% CI b</td></tr><tr><td rowspan=1 colspan=1>Site 1 (002A)</td><td rowspan=1 colspan=1>1.72% (6/348)</td><td rowspan=1 colspan=1>100% (6/6)(61.0%, 100%)</td><td rowspan=1 colspan=1>99.4% (340/342)(97.9%, 99.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 2 (009A)</td><td rowspan=1 colspan=1>11.9% (86/724)</td><td rowspan=1 colspan=1>96.5% (83/86)(90.2%, 98.8%)</td><td rowspan=1 colspan=1>97.6% (623/638)c(96.2%, 98.6%)</td></tr><tr><td rowspan=1 colspan=1>Site 3 (015A)</td><td rowspan=1 colspan=1>4.10% (7/171)</td><td rowspan=1 colspan=1>42.9% (3/7)(15.8%, 75.0%)</td><td rowspan=1 colspan=1>98.8% (162/164)(95.7%, 99.7%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>7.96% (99/1243)</td><td rowspan=1 colspan=1>92.9% (92/99)(86.1%, 96.5%)</td><td rowspan=1 colspan=1>98.3% (1125/1144)(97.4%, 98.9%)</td></tr></table>

a Prevalence based on the comparator method only b Confidence interval c Flu B nucleic acid was detected in 3/15 False Positive specimen using an alternative FDA-cleared molecular test

Performance of Alere™ i Influenza A & B versus the comparator method is presented in Table 17 below stratified by patient age.

Table 17: Nasal and Nasopharyngeal Swab in VTM Performance Obtained for Influenza A and Influenza B with the Alere™ i Influenza A & B in Comparison to the Comparator – Stratified by Patient Age   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>≤ 5 Years of Age(n = 706)56.7% (706/1243)</td><td rowspan=1 colspan=2>6 - ≤ 21 Years of Age(n = 388)31.2% (388/1243)</td><td rowspan=1 colspan=2>≥ 22 Years of Age(n = 149)12.0% (149/1243)</td></tr><tr><td rowspan=1 colspan=1>Influenza Type</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>99.1%(105/106)94.8% - 99.8%</td><td rowspan=1 colspan=1>98.2%(589/600)96.7% - 99.0%</td><td rowspan=1 colspan=1>97.0%(98/101)91.6% - 99.0%</td><td rowspan=1 colspan=1>95.8%(275/287)92.8% - 97.6%</td><td rowspan=1 colspan=1>94.7%(18/19)75.4% - 99.1%</td><td rowspan=1 colspan=1>90.8%(118/130)84.6% - 94.6%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>100%(39/39)91.0% - 100%</td><td rowspan=1 colspan=1>98.2%(655/667)96.9% - 99.0%</td><td rowspan=1 colspan=1>94.2%(49/52)84.4% - 98.0%</td><td rowspan=1 colspan=1>98.5%(331/336)96.6% - 99.4%</td><td rowspan=1 colspan=1>50%(4/8)21.5% - 78.5%</td><td rowspan=1 colspan=1>98.6%(139/141)95.0% - 99.6%</td></tr></table>

The Alere™ i Influenza A & B detected six mixed influenza A and B infections in this prospective clinical evaluation. Three samples tested positive for influenza A only by the comparator and another FDA-cleared Influenza RT-PCR assay. One sample tested positive for influenza B only by the comparator and another FDAcleared Influenza RT-PCR assay. One sample tested negative by the comparator and another FDA-cleared Influenza RT-PCR assay. One sample tested negative by the comparator, but tested positive for influenza B only by another FDA-cleared Influenza RT-PCR assay. The following statement is   
included in the “Result Interpretation” section of the Product Insert: “Dual infections of Flu A and Flu $B$ are rare. Repeat testing using new test components. Contact Technical Support during normal business hours if multiple samples provide this result.”

During the prospective clinical study, the initial invalid rate (before repeat testing per the product instructions) was $4 . 3 \%$ (54/1270) ( $9 5 \%$ CI: $3 . 3 \%$ to $5 . 5 \%$ ). After repeat testing per the product instructions, the invalid rate was $2 . 1 \%$ (27/1270) ( $9 5 \%$ CI: $1 . 5 \% , 3 . 1 \% )$ .

$b$ . Retrospective Study: None.

4. Clinical cut-off: Not applicable.

5. Expected values:

In the Alere™ i Influenza A & B prospective clinical study testing nasal and nasopharyngeal swab in VTM (described in the “Clinical Studies” section above), a total of 1243 specimens were determined to be evaluable. The number and percentage of influenza A and influenza B positive cases per specified age group, as determined by the Alere™ i Influenza A & B assay, are presented in Table 18 and Table 19 below:

Table 18: Influenza A Positives by the Alere™ i Influenza A & B Assay per Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasal and NasopharyngealSwab in VTM Specimens</td><td rowspan=1 colspan=1>Number of Influenza A Positives</td><td rowspan=1 colspan=1>Influenza A Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>12.1%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>533</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>17.8%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>31.5%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>26.1%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>22.4%</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>17.2%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>20.6%</td></tr></table>

Table 19: Influenza B Positives by the Alere™ i Influenza A & B Assay per Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasal and NasopharyngealSwab in VTM Specimens</td><td rowspan=1 colspan=1>Number of Influenza B Positives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5.8%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>533</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>7.7%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>15.5%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>13.0%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 to 60 years</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.9%</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>8.9%</td></tr></table>

# N. Instrument Name:

Alere™ i Instrument

# O. System Descriptions:

1. Modes of Operation:

The Alere™ i Instrument is a portable bench-top unit designed to perform in laboratory and point-of care environments. Alere™ i Influenza A & B components are color-coded to match the corresponding holder on the instrument and designed to only fit in the corresponding holder in one direction. Once properly placed into the instrument, the Alere™ i Instrument uses image analysis to automatically detect the presence of the Test Base and to check the test type and expiry date on the barcode present on the Test Base. This triggers the test sequence. Heating, mixing, and detection by fluorescence are provided by the instrument. The Alere™ i Instrument is equipped with two optical fluorescence modules, $\mathrm { F A M ^ { T M } }$ and $\mathrm { R O X ^ { T M } }$ . Each optical module consists of a LED excitation source and photodiode receiver configured to the fluorescent channel. Fluorescence signals are optically filtered by a confocal lens and mirror arrangement and processed by software running on an on-board computer. The results are calculated automatically and presented on the instrument graphical user interface and available for export in JSON format or printing.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No

3. Specimen Identification:

Specimen ID is entered using on screen keyboard or barcode scanner.

4. Specimen Sampling and Handling:

Not applicable. The specimens are manually inserted in the Sample Receiver in the instrument.

5. Calibration:

The Alere™ i Instrument is factory calibrated and does not require any further calibration and verification at user sites. However, if the instrument was transported or moved, a performance check using Alere™ i Positive and Negative Controls is recommended to ensure proper functionality.

6. Quality Control:

Quality control is addressed for each specific FDA-cleared assay to be run on the instrument (separately cleared).

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.