# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K152614 B. Purpose for Submission:

To obtain a substantial equivalence determination for the Xpert® Carba-R Assay on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems) in the qualitative detection of the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I N P } }$ gene sequence from colonies.

# C. Measurand:

Target DNA sequence of the following genes:

blaKPC, blaNDM, blaVIM, blaOXA-48, and blaIMP

D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) assay

# E. Applicant:

Cepheid

F. Proprietary and Established Names:

Proprietary Name: Xpert® Carba-R

Common Name: Xpert Carba-R Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 (Antimicrobial susceptibility test powder)

2. Classification: Class II

3. Product code:

PMY- System, nucleic acid amplification test, DNA, carbapenem non-susceptible gram   
negative organism, colony   
OOI-Real-time nucleic acid amplification system

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The Xpert® Carba-R Assay, performed on the GeneXpert® Instrument Systems, is a qualitative in vitro diagnostic test for the detection and differentiation of the $b l a _ { \mathrm { K P C } }$ , blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I N P } }$ gene sequences associated with carbapenem-nonsusceptible pure colonies of Enterobacteriaceae, Acinetobacter baumannii, or Pseudomonas aeruginosa grown on blood agar or MacConkey agar. The test utilizes automated real-time polymerase chain reaction (PCR).

A negative Xpert Carba-R Assay result does not preclude the presence of other resistance mechanisms. The Xpert Carba-R Assay should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing. The Xpert Carba-R Assay is intended as an aid for infection control in detecting and differentiating genetic markers of resistance to monitor the spread of carbapenem-non-susceptible organisms in healthcare settings. The Xpert Carba-R Assay is not intended to guide or monitor treatment for carbapenem-non-susceptible bacterial infections.

2. Indication(s) for use: Same as the Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

Organisms should be identified and carbapenem non-susceptibility status should be determined prior to testing with the Xpert Carba-R Assay.

The Xpert Carba-R Assay detects $b l a _ { \mathrm { K P C } } , b l a _ { \mathrm { N D M } } , b l a _ { \mathrm { V I M } } , b l a _ { \mathrm { O X A - } 4 8 } .$ and $b l a _ { \mathrm { I M P } }$ from pure colonies and is not for bacterial identification.

The performance of the Xpert Carba-R Assay with bacteria other than Enterobacteriaceae, Pseudomonas aeruginosa or Acinetobacter baumannii has not been evaluated.

The Xpert Carba-R Assay is not a sub-typing tool and does not report variants of the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ genes.

The detection of other OXA-carbapenemase genes, besides blaOXA-48 and blaOXA-181, has not been evaluated in the study.

4. Special instrument requirements:

The Xpert Carba-R Assay uses PCR technology on the GeneXpert Instrument Systems, which extract, amplify, and detect the target DNA.

# I. Device Description:

The Xpert Carba-R Assay is an automated real-time polymerase chain reaction (PCR) in vitro diagnostic test for qualitative detection of the $b l a _ { \mathrm { K P C } }$ , blaNDM, blaVIM, blaOXA-48, and blaIMP gene sequences from pure colonies of carbapenem non-susceptible Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii. The Xpert Carba-R Assay is intended as an aid for infection control to monitor the spread of carbapenem-non-susceptible organisms in healthcare settings. The assay is performed on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems). The GeneXpert Instrument Systems utilize single-use, disposable cartridges (Xpert Carba-R cartridges) containing PCR reagents and allows for automated sample preparation, amplification, and real-time detection of gene targets in approximately 50 minutes. A Sample Processing Control (SPC) and a Probe Check Control (PCC) have been incorporated into the assay design to address key failure modes that could result in a false negative determination.

The GeneXpert Instrument Systems (GeneXpert Dx Systems and the GeneXpert Infinity Systems) have 1 to 80 randomly accessible modules, depending upon the instrument, that are each capable of performing separate sample processing and real-time PCR and RT-PCR tests. Because the cartridges are self-contained and specimens never come into contact with working parts of the instrument modules, cross-contamination between samples is minimized.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Xpert ® vanA Assay

2. Predicate 510(k) number(s): K092953

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device Xpert® Carba-R Assay (K152614)</td><td>Predicate Cepheid Xpert® vanA (K092953)</td></tr><tr><td>Intended Use</td><td>The Xpert® Carba-R Assay, performed on the GeneXpert® Instrument Systems, is a qualitative in vitro diagnostic test for the detection and differentiation of the blakpc, blaNDm, blavm, blaoxA-48, and blaMP gene sequences associated with carbapenem-non-susceptible pure colonies of Enterobacteriaceae, Acinetobacter baumannii, or Pseudomonas aeruginosa grown on blood agar or MacConkey agar. The test utilizes automated real-time polymerase chain reaction (PCR). A negative Xpert Carba-R Assay result does not preclude the presence of other resistance mechanisms. The Xpert Carba-R Assay should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing. The Xpert Carba-R Assay is intended as an aid for infection control in detecting and differentiating genetic markers of resistance to monitor the spread of carbapenem-non- susceptible organisms in healthcare settings. The Xpert Carba-R Assay is not intended to guide or monitor treatment for carbapenem-non- susceptible bacterial infections.</td><td>The Cepheid Xpert® vanA Assay performed in the GeneXpert® Dx System is a qualitative in vitro diagnostic test designed for rapid detection of the vanA gene sequence associated with vancomycin resistance in bacteria obtained from rectal swab specimens from patients at risk for intestinal colonization with vancomycin-resistant bacteria. The test utilizes automated real-time polymerase chain reaction (PCR) to detect the vanA gene that is frequently associated with vancomycin-resistant enterococci (VRE). The Xpert vanA Assay is intended to aid in the recognition, prevention, and control of vancomycin resistant organisms that colonize patients in healthcare settings. The Xpert vanA Assay is not intended to diagnose infections caused by vancomycin-resistant bacteria nor to guide or monitor treatment for vancomycin-resistant bacterial infections. Concomitant cultures are necessary to recover organisms for confirmatory identification of vancomycin- resistant bacteria, antimicrobial susceptibility testing, and for</td></tr><tr><td>Technological Principles</td><td>Fully-automated nucleic acid amplification (DNA); real-time</td><td>epidemiological typing. Same</td></tr><tr><td>Test Cartridge</td><td>PCR Disposable single-use, multi-chambered fluidic cartridge</td><td>Same</td></tr><tr><td>Detection Probes</td><td>TaqMan® Probes</td><td>Same</td></tr><tr><td>Controls</td><td>Internal sample processing control (SPC) and probe</td><td>Same</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceXpert® Carba-R Assay(K152614)</td><td colspan="1" rowspan="1">PredicateCepheid Xpert® vanA(K092953)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">check control (PCC)External controls available</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Time to obtaintest results</td><td colspan="1" rowspan="1">Approximately 50 minutes toresults</td><td colspan="1" rowspan="1">Approximately 45 minutes toresults</td></tr><tr><td colspan="1" rowspan="1">Interpretation oftest results</td><td colspan="1" rowspan="1">Diagnostic software</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Bacterial isolates from culture</td><td rowspan=1 colspan=1>Rectal swabs</td></tr><tr><td rowspan=1 colspan=1>Assay Targets</td><td rowspan=1 colspan=1>Detects blakpc, blandm, blavm,blaoxA-48, and blamp gene sequences</td><td rowspan=1 colspan=1>Detects gene sequences for thevanA encoded resistance tovancomycin/teicoplanin</td></tr><tr><td rowspan=1 colspan=1>Organism(s)</td><td rowspan=1 colspan=1>Enterobacteriaceae, Pseudomonasaeruginosa, Acinetobacter baumannii</td><td rowspan=1 colspan=1>Vancomycin-resistant bacteria(Enterococcus)</td></tr><tr><td rowspan=1 colspan=1>InstrumentSystem</td><td rowspan=1 colspan=1>GeneXpert Instrument System(includes GeneXpert Dx,Infinity-48, Infinity-48s, andInfinity-80)</td><td rowspan=1 colspan=1>GeneXpert Dx</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. ASTM D4169-09, Standard Practice for Performance Testing of Shipping Containers and Systems.   
2. CLSI EP5-A2, Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline—Second Edition, 2004   
3. CLSI EP15-A2, User Verification of Performance for Precision and Trueness; Approved Guideline–Second Edition, 2006   
4. CLSI M02-A11. Performance standards for Antimicrobial Disk Susceptibility Tests; Eleventh Edition, 2012   
5. CLSI M07-A9. Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; Approved standard—Ninth Edition, 2012   
6. CLSI M07-A10. Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; Approved standard—Tenth Edition, 2015   
7. CLSI M100-S24. Performance Standard for Antimicrobial Susceptibility Testing; Approved standard—Twenty-fourth Informational Supplement, 2014   
8. CLSI MM3-A2, Molecular Diagnostic Methods for Infectious Disease; Approved Guideline–Second Edition, 2006   
9. EN 13640, Stability Testing of in vitro Diagnostic Reagents, June 2002   
10. General Principles of Software Validation; Final Guidance for Industry and FDA Staff, issued January 11, 2002   
11. Guidance for Industry and FDA Staff–Format for Traditional and Abbreviated 510(k)s, issued August 12, 2005   
12. Guidance for Industry and FDA Staff-Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems, issued on March 10, 2005   
13. Guidance for Industry and FDA Staff–Content of Premarket Submissions for Management of Cybersecurity in Medical Devices, issued on October 2, 2014   
14. Guidance for Industry-Cybersecurity for Networked Medical Devices Containing Offthe-Shelf (OTS) Software, issued January 14, 2005   
15. Guidance for Industry and FDA Staff–Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, issued May 11, 2005   
16. Guidance for Industry, FDA Reviewers and Compliance on Guidance for Off-the-Shelf Software Use in Medical Devices; issued September 9, 1999   
17. Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff-Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable, issued April 25, 2006

# L. Test Principle:

The Xpert Carba-R Assay cartridges contain reagents for the detection of $b l a _ { \mathrm { K P C } } , b l a _ { \mathrm { N D M } } .$ , blaVIM, blaOXA-48, and blaIMP gene sequences from isolates of pure cultures of carbapenemnon-susceptible Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii. The bacterial isolates from culture are tested using a loopful of organism (equivalent to a 0.5 McFarland suspension) resuspended in $5 \mathrm { m l }$ of Sample Reagent. The sample is vortexed then transferred $( 1 . 7 \mathrm { m l } )$ to the sample chamber of a disposable Xpert Carba-R Assay cartridge. The user initiates a test from the system-user interface and places the cartridge into the GeneXpert instrument platform, which performs hands-off real-time, multiplex PCR for detection of target sequences.

Results of the assay run are interpreted by the GeneXpert Instrument System software from measured fluorescent signals and embedded calculation algorithms. The results are automatically generated at the end of the process in a report that can be viewed and printed. Basic users see test results reported as “red” highlighted for DETECTED results and “green” highlighted for NOT DETECTED results. Additional results that can be reported include: INVALID, ERROR, and NO RESULT.

# Interpretation of Results

The Xpert Carba-R Assay provides test results for the IMP, VIM, NDM, KPC, and OXA-48 target DNA sequences. A Sample Processing Control (SPC) and a Probe Check Control (PCC) have been designed for the assay as internal controls to enable the GeneXpert Instrument System to detect specific failure modes related to assay performance. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for any of the IMP, VIM, NDM, KPC, and OXA-48 targets, or SPC target, a probe check error is reported and the test will not continue. The assay also reports if the test has an INVALID, ERROR or NO RESULT. Under these conditions, the test will need to be repeated using a new sample, a new cartridge, and/or new reagents. Retest procedures are described in the Xpert Carba-R package insert. An interpretation table for test results is shown in Table 1.

Table 1. Interpretation of Test Results for the Xpert Carba-R Assay   

<table><tr><td rowspan=1 colspan=1>Result Report</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>DETECTED</td><td rowspan=1 colspan=1>Target(s): For a valid &quot;DETECTED&quot; test result, PCRamplification of the target DNA sequence gives Ct value(s)within the valid range and a fluorescence endpoint above thethreshold setting for IMP, VIM, NDM, KPC, and/or OXA-48;SPC: Not applicable (if at least one target detected); PCC: PASS;all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>NOT DETECTED</td><td rowspan=1 colspan=1>Target(s): For a valid &quot;NOT DETECTED&quot; test result, no validCt(s) are reported for the IMP, VIM, NDM, KPC, and/or OXA-48target DNA sequences; SPC: PASS, PCR amplification of theSPC DNA sequence gives a Ct value within the valid range and afluorescence endpoint above the threshold setting; PCC: PASS;all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>ERROR</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC: NORESULT; PCC: FAIL*, one or more of the probe check resultsfailed. *If the probe check passed, the error is caused by a systemcomponent failure.</td></tr><tr><td rowspan=1 colspan=1>INVALID</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC:FAIL, No PCR amplification of the SPC DNA sequence or theSPC Ct is not within valid range and the fluorescence endpoint isbelow threshold setting; PCC: PASS; all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>NO RESULT</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC: NORESULT; PCC: Not applicable.A &quot;NO RESULT&quot; indicates that insufficient data were collected.For example, the operator stopped a test that was in progress or apower failure occurred.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The reproducibility of the Xpert Carba-R Assay was established in a multi-center study. A panel of 13 bacterial samples included the following: 1) two different organisms for each of the five resistance genes detected by the Xpert Carba-R Assay, 2) two different organisms that harbored two gene targets, and 3) one organism negative for all five gene targets. A $0 . 5 \mathrm { M c F }$ arland equivalent of bacterial cell suspension was prepared for each sample. To measure site-to-site reproducibility, the (13)-member panel was tested in replicates of four each day at three (3) sites over a six day testing period. Three lots of Xpert Carba-R Assay cartridges were used at each testing site. A total of 1872 samples consisting of 144 replicates for the 13

different panel members were tested for the study using two operators at each site. Twenty-five (25) test runs from one instrument module were excluded, resulting in 1847 test runs included in the analyses. For the Xpert Carba-R Assay, $9 9 . 4 \%$ (1836/1847) of samples were successful and produced the expected result on the first attempt. Six (6) ERROR cases, two (2) INVALID results, and three (3) NO RESULT outcomes were reported. All eleven samples yielded valid results upon repeat testing. The results of the reproducibility study are summarized in Table 2 below.

Table 2. Reproducibility of the 13-Member Sample Panel   

<table><tr><td rowspan=2 colspan=1>Resistance Gene*(sample number)</td><td rowspan=1 colspan=3>Site 1(GeneXpert Dx)</td><td rowspan=1 colspan=3>Site 2(Infinity-80)</td><td rowspan=1 colspan=3>Site 3(Infinity-48)</td><td rowspan=2 colspan=1>% TotalAgreementby Sample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=2 colspan=1>KPC (S1)</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=2 colspan=1>100%(48/48)</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=2 colspan=1>100%(24/24)</td><td rowspan=2 colspan=1>100%(48/48)</td><td rowspan=2 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>(48/48)</td></tr><tr><td rowspan=1 colspan=1>KPC (S2)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(22/22)</td><td rowspan=1 colspan=1>100%(45/45)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>99.3%(140/141)</td></tr><tr><td rowspan=1 colspan=1>VIM (S1)</td><td rowspan=1 colspan=1>100%(22/22)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(45/45)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(141/141)</td></tr><tr><td rowspan=1 colspan=1>VIM (S2)</td><td rowspan=1 colspan=1>100%(22/2)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(46/46)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(142/142)</td></tr><tr><td rowspan=1 colspan=1>IMP (S1)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(47/47)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(143/143)</td></tr><tr><td rowspan=1 colspan=1>IMP (S2)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(46/46)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(142/142)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 (S1)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(46/46)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>98.6%14/142)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 (S2)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(22/22)</td><td rowspan=1 colspan=1>100%(45/45)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(141/141)</td></tr><tr><td rowspan=1 colspan=1>NDM (S1)</td><td rowspan=1 colspan=1>100%(22/22)</td><td rowspan=1 colspan=1>100%(21/21)</td><td rowspan=1 colspan=1>100%(43/43)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(139/139)</td></tr><tr><td rowspan=1 colspan=1>NDM (S2)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(46/46)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>98.6%(140/142)</td></tr><tr><td rowspan=1 colspan=1>OXA-48/NDM (S1)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(47/47)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(143/143)</td></tr><tr><td rowspan=1 colspan=1>OXA-48/NDM (S2)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(47/47)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(143/143)</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr></table>

\*S1=Sample 1 $S 2 =$ Sample 2

The reproducibility of the Xpert Carba-R Assay was also evaluated by assessing the fluorescent signal (expressed in Ct values) for each target detected. The mean, standard deviation (SD), and coefficient of variation (CV) between sites, days, and operators for each panel member are presented in Table 3 below.

Table 3. Reproducibility of the Fluorescent Signal   

<table><tr><td rowspan=2 colspan=2>Resistance Gene(sample number)</td><td rowspan=2 colspan=1>AssayChanel(Analyte)</td><td rowspan=2 colspan=1>Na</td><td rowspan=1 colspan=2>BetweenSite</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>WithinAssay</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>KPC (S1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>5.8</td></tr><tr><td rowspan=1 colspan=1>KPC (S2)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.9</td></tr><tr><td rowspan=1 colspan=2>VIM (S1)</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>6.7</td></tr><tr><td rowspan=1 colspan=2>VIM (S2)</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>5.1</td></tr><tr><td rowspan=1 colspan=2>IMP (S1</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=1 colspan=2>IMP (S2)</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>7.6</td></tr><tr><td rowspan=1 colspan=2>OXA-48 (S1)</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>5.2</td></tr><tr><td rowspan=1 colspan=2>OXA-48 (S2)</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>6.4</td></tr><tr><td rowspan=1 colspan=2>NDM (S1)</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=2>NDM (S2)</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>6.3</td></tr><tr><td rowspan=2 colspan=2>NDM/OXA-48 (S1)</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>6.8</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>7.7</td></tr><tr><td rowspan=2 colspan=2>NDM/OXA-48 (S2)</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>6.9</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>7.6</td></tr><tr><td rowspan=1 colspan=2>NEG</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr></table>

a Results with non-zero Ct values out of 144.

Two NDM samples and two OXA-48 samples were negative for all targets. One KPC sample was positive for both KPC and OXA-48 gene targets. The remaining panel samples were positive for the expected targets. All negative samples were correctly identified as negative (144/144). Agreement between sites, operators, and lots was evaluated using Fisher’s Exact test. The data presented in Table 2 and Table 3 demonstrated good reproducibility for the Xpert Carba-R Assay on the GeneXpert Instrument Systems.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Controls

Commercially-available external controls can also be run in accordance with local, state, and federal accrediting organizations, as applicable. External controls include the following bacteria harboring target genes:

Multivalent External Positive Control—External positive control (inactivated Escherichia coli carrying plasmid with KPC, NDM, VIM, IMP, and OXA-48 gene sequences)

Individual Positive Controls • K. pneumoniae KPC (ATCC BAA-1705) K. pneumoniae NDM (ATCC BAA-2146) K. pneumoniae VIM (NCTC 13439)   
• K. pneumoniae OXA-48 (NCTC 13442)

External Negative Control—Inactivated E. coli containing a plasmid with no resistance gene inserted.

# Internal Control (IC) Reaction Analysis

Internal controls enable the system to detect specific failure modes that could potentially result in an incorrect test result. Each Xpert Carba-R Assay includes a Sample Processing Control (SPC) and Probe Check Control (PCC) pre-loaded in the cartridge and provided with the assay.

# Sample Processing Control (SPC)

The SPC contains genomic DNA of Bacillus globigii that is included in each cartridge to verify adequate processing of the sample. The SPC verifies that lysis of bacteria has occurred if the organisms are present and verifies the effectiveness of each sample preparation step—reaction tube filling, reaction components are present and functioning, and monitoring for the presence of potential inhibitor(s) in the PCR assay. Test results are reported as INVALID if the SPC fails to meet the valid minimum or maximum Ct specification (range 3.0 to $4 0 . 0 \mathrm { C t }$ .

# Probe Check control (PCC)

The PCC verifies reagent rehydration, real-time PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for any of the IMP, VIM, NDM, KPC, and OXA-48 targets or SPC target, a probe check error is reported and the test will not continue. If a probe check error is reported, the test may be repeated using a new sample, new cartridge, and new reagents.

# Stability Studies

The objectives of the Xpert Carba-R Assay Stability and Shipping Study were to: 1) evaluate the shelf-life of the Xpert Carba-R Assay, 2) determine the open package stability to support claimed temperature/time limits for the assay, and 3) determine shipping stability for the assay.

# Shelf-life and Expiration Dating Study

Three lots of the Xpert Carba-R Assay are being evaluated for stability at four temperatures and predefined time point intervals. Each Xpert Carba-R lot has been tested with samples containing high and low levels of target strains and with a negative sample type (an organism that is not detectable by the assay). Analysis of lot performance was based on the studies of the Cts for IMP, VIM, NDM, KPC, and OXA-48 targets from both low and high positive samples, as well as the SPC target from negative samples. Currently, six months of real-time data are available for the three stability lots. Real-time stability studies for determining shelf-life and expiration dating are ongoing. The storage temperature range for the Xpert Carba-R Assay is $2 ^ { \circ } \mathrm { C } - 2 8 ^ { \circ } \mathrm { C }$ .

# Shipping Stability

The purpose of the shipping stability study was to validate the ability of the packaging systems to protect the Xpert Carba-R Assay test kits during transport. Kits were subjected to transport simulation conditions under pre-defined temperatures and humidity conditions. Packages were inspected and functional testing performed to establish shipped sample performance. The Xpert Carba-R Assay packaging was determined to be adequate in protecting the product from damage. Product performance testing was conducted on one assay lot.

# Sample Stability

A study was conducted to establish the sample stability of carbapenem-nonsusceptible bacterial isolates seeded into Xpert Carba-R Assay Sample Reagent. Table 4 shows the one negative and five positive bacterial strains tested. The five positive organisms comprised of carbapenemase-producing bacteria harboring each of the gene targets detected by the Xpert Carba-R Assay. The study was performed first by sub-culturing the bacteria onto blood agar plates and placing a meropenem disc in the first streak quadrant of each plate as a means to ensure that the bacterial isolate retained its non-susceptibility to carbapenem. The agar plates were incubated at $3 5 ^ { \circ } \mathrm { C }$ for 18-24 hours. A bacterial suspension equivalent to a $0 . 5 \mathrm { M c F }$ arland suspension was prepared in tryptic soy broth for each plate. An aliquot of bacterial suspension was mixed with Sample Reagent and tested with the Xpert Carba-R Assay at $\scriptstyle { \mathrm { t = 0 } }$ day. One sample test volume in Sample Reagent was stored at $4 \mathrm { { } ^ { \circ } C }$ for up to 8 days and tested on those days with the Xpert CarbaR Assay. A second test volume was stored at $3 0 ^ { \circ } \mathrm { C }$ for up to 8 days and similarly tested. A negative control consisting of a $0 . 5 \mathrm { M c F }$ arland suspension of a carbapenem non-susceptible organism (that did not produce carbapenemase) was also prepared and tested.

Table 4. Bacterial Strains for Sample Stability Testing   

<table><tr><td rowspan=1 colspan=1>Carba-R Target</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=1 colspan=1>blaκpc</td><td rowspan=1 colspan=1>Enterobacter cloacae</td></tr><tr><td rowspan=1 colspan=1>blaNDM</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>blavm</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>blaoxA-48</td><td rowspan=1 colspan=1>Escherichia coli</td></tr><tr><td rowspan=1 colspan=1>blaMP</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Enterobacter cloacae</td></tr></table>

All positive and negative samples tested at each storage condition/temperature were correctly identified using the Xpert Carba-R Assay. The data supports that carbapenem non-susceptible bacterial isolate cells are stable in Xpert Carba-R Sample Reagent for up to 8 days when stored at $4 ^ { \circ } \mathrm { C } - 3 0 ^ { \circ } \mathrm { C }$ prior to testing with the Xpert Carba-R Assay. Therefore, the study supports storage claim of $2 { - } 2 8 ^ { \circ } \mathrm { C }$ for up to 96 hours in Xpert Carba-R Sample Reagent for carbapenem nonsusceptible bacterial isolates.

All external positive and negative controls gave the expected GeneXpert test result. Data for this study was collected using GeneXpert Dx software version

4.4a on the GeneXpert Dx GX-IV instrument.

Cartridge Hold Time Study Samples prepared for testing with the Xpert Carba-R Assay and the GeneXpert Infinity System may not be available for testing for up to two hours after the cartridge is loaded onto the System (in a multiple-module system). In order to assess the stability of cartridge storage times, one lot of the Xpert Carba-R Assay was subjected to a sample hold time study where the sample was added to cartridges, held at three storage conditions (room temperature, $2 5 ^ { \circ } \mathrm { C } / 7 5 \%$ relative humidity, and $3 5 ^ { \circ } \mathrm { C }$ ), and processed at scheduled time intervals ranging from 0-5 hours. Positive and negative samples were tested under all storage conditions as a panel with 4 negative and 14 positive samples. Two types of external positive controls were employed in the study—multivalent external positive control and monovalent positive controls. Acceptable performance was observed for up to five hours of hold time between sample addition and cartridge processing for positive and negative samples under all conditions tested. The maximum hold time recommended from sample addition to the Xpert Carba-R Assay cartridges to processing on the GeneXpert Infinity System is 4.5 hours. All external positive and negative controls gave the expected GeneXpert test result.

d. Detection limit:

Not applicable e. Analytical reactivity:

An inclusivity study was conducted to test reactivity of the Xpert Carba-R Assay with a 71-member panel (See Table 5) of well-characterized bacterial isolates that included the following molecular resistance marker groups and representative sub-groups: (11) $b l a _ { \mathrm { K P C } }$ isolates, (13) $b l a _ { \mathrm { N D M } }$ isolates, (11) blaVIM isolates, (8) blaOXA-48 isolates, (5) bla NDM/OXA-181 isolates, (5) $b l a _ { \mathrm { O X A - 1 8 1 } }$ , (17) $b l a _ { \mathrm { I M P } }$ isolates, and (1) $b l a _ { \mathrm { K P C / V I M } }$ isolate. The study was also used to demonstrate equivalency in growth on blood agar and MacConkey agar plates for the detection of target genes by the Xpert Carba-R Assay.

Bacterial isolates were grown on blood agar and MacConkey agar plates and incubated at $3 5 \mathrm { { ^ \circ C } }$ for 18-24 hours. A meropenem disc was placed in the first streak quadrant of each plate as a means to ensure that the bacterial isolate retained its nonsusceptibility to carbapenem. For each bacterial strain tested, a $0 . 5 \mathrm { M c F }$ arland cell suspension was prepared in tryptic soy broth for each plate. A $1 0 ~ \mu \mathrm { l }$ loop of the suspension was diluted in 5 ml Xpert Carba-R Sample Reagent, vortexed briefly, then duplicate $1 . 7 \mathrm { m l }$ aliquots were tested with the Xpert Carba-R Assay. Each organism was tested in replicates of four (4)—two replicates per agar plate.

For each day of the study, an external negative control and two types of experimental positive controls were tested. The negative control consisted of Escherichia coli cells containing a plasmid with no resistance gene sequence. The two types of external positive controls employed in the study included:

multivalent external positive control (containing all five gene targets) two monovalent positive controls (containing individual carbapenemaseproducing bacteria, each harboring only one of the Xpert Carba-R target carbapenemase genes on a rotating basis).

On each testing day, one negative control, the five-gene construct positive control, and two of five individual bacterial controls (single gene target) were tested.

For a list of strains tested during the Analytical Reactivity Study, please refer to Table 5 below.

Table 5. Strains Tested in Analytical Reactivity   

<table><tr><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Confirmed GeneticResistance Marker</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC Isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">31551</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-4</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1705</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CFVL</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">KBM18</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">BM9</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">CGNC</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">PA3</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">164-3</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">KPC</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NDM Isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2146</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">34262</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">GEN</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">3047</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">7892</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CAN</td><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">EGY</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NDM-2</td></tr><tr><td colspan="1" rowspan="1">I5</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-4</td></tr><tr><td colspan="1" rowspan="1">405</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-5</td></tr><tr><td colspan="1" rowspan="1">CF-ABE</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">73999</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">39365</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">VIM Isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-10</td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">NCTC 13440</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">758</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM</td></tr><tr><td colspan="1" rowspan="1">Col1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">BM19</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">KOW7</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">DIH</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-19</td></tr><tr><td colspan="1" rowspan="1">MSH2014-3</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">VIM</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA Isolates</td><td colspan="1" rowspan="1">cYanmaGYellowb</td></tr><tr><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">OM11</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">501</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">DUW</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">OM22</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">BOU</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">TUR</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">11670</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">MSH2014-64</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">MSH2014-72</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">166643</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">42194</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IMP Isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13476</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">695</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">2340</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMPBMI</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">6852</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">Yonsei 1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">Yonsei 2</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">70450-1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">3994</td><td colspan="1" rowspan="1">Pseudomonas spp.</td><td colspan="1" rowspan="1">IMP-10</td></tr><tr><td colspan="1" rowspan="1">MKAM</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">5344</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-2</td></tr><tr><td colspan="1" rowspan="1">G029</td><td colspan="1" rowspan="1">Salmonella spp</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">3985</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-11</td></tr><tr><td colspan="1" rowspan="1">4032</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="1">3424</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-7</td></tr><tr><td colspan="1" rowspan="1">32443</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-13</td></tr><tr><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-14</td></tr><tr><td colspan="1" rowspan="1">Isolates</td><td colspan="2" rowspan="1">Isolates with more than one genetic marker target</td></tr><tr><td colspan="1" rowspan="1">GR-04/KP-69</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2/VIM</td></tr><tr><td colspan="1" rowspan="1">B108A</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td></tr><tr><td colspan="1" rowspan="1">KP-OMA3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td></tr><tr><td colspan="1" rowspan="1">1300920</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td></tr><tr><td colspan="1" rowspan="1">MSH2014-69</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td></tr><tr><td colspan="1" rowspan="1">C10192-DISCS</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">NDM/OXA-181</td></tr></table>

All organisms tested exhibited adequate growth on both blood agar and MacConkey agar plates. Xpert Carba-R Assay target genes were detected in 68 of the 71 bacterial strains with the Xpert Carba-R Assay. Target sequences were not detected in three strains (Table 6). One replicate sample for one of the (12) KPC-containing strains reported an indeterminate result (INVALID). The sample was successfully repeated and was correctly reported as “KPC DETECTED.”

Table 6. Summary of Variants Detected by Wet Testing or Predicted to be Detected Based on In Silico Analysis.   

<table><tr><td rowspan=2 colspan=1>Marker (orTraditionalSubgroup)</td><td rowspan=1 colspan=3>Wet testing</td><td rowspan=2 colspan=1>Not tested but predicted to bedetected based onin silico analysis</td></tr><tr><td rowspan=1 colspan=1>No. ofSampleswith Target</td><td rowspan=1 colspan=1>Type(s)Detected</td><td rowspan=1 colspan=1>Type(s) notDetected</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>KPC-2, 3, 4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KPC-5, 6, 7, 8, 9, 10, 11, 12, 13,14, 15, 16</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>NDM-1, 2, 4, 5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NDM-3, 6, 7, 8, 9</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>VIM-1, 2, 4, 10,19</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>VIM-5, 6, 7, 8, 9, 11, 12, 13, 14,15, 16, 17, 18, 20, 23, 24, 25, 26,27, 28, 29, 30, 31, 32, 33, 34, 35,36, 37, 38</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>OXA-48,OXA-181(OXA-48 variant)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-162, 163, 204, 232, 244,245, 247</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>IMP-1 (9 strains),IMP-2, 4, 6, 10,11</td><td rowspan=1 colspan=1>IMP-7^, 13b14a</td><td rowspan=1 colspan=1>IMP-3, 8, 9, 13b, 19, 20, 21, 22, 24,25, 27, 28, 30, 31, 33, 37, 40, 42</td></tr></table>

a IMP-7 and IMP-14 genes (Pseudomonas aeruginosa) were not detected by the assay and were not predicted to be detected by in silico analysis (Limitation in package insert). b IMP-13 gene (Klebsiella pneumoniae): although predicted to be detected by in silico analysis, the IMP-13 gene was not detected by the assay (Limitation in package insert).

# f. Analytical specificity:

The Xpert Carba-R Assay was examined for analytical specificity by testing a panel of 62 well-characterized carbapenem-susceptible bacteria or bacteria with carbapenem non-susceptibility due to genes or mechanisms other than the Xpert Carba-R target genes. Twenty-four commensal/enteric microorganisms were also evaluated in the study. All organisms were grown aerobically on blood agar and MacConkey agar plates. Bacterial suspensions tested were equivalent to a 0.5 McFarland cell suspension. A $1 0 0 \mu \mathrm { l }$ aliquot of each suspension was diluted into $5 \mathrm { m l }$ Sample Reagent to a final concentration o $\mathrm { f { > } 1 \ x \ } 1 0 ^ { 6 } \mathrm { C F { \dot { U } } / m l } .$ Each organism was tested a total of four times (from each type of plate). Table 7 and Table 8 list the organisms tested during the Analytical Specificity Study.

Table 7. Analytical Specificity Panel with Organisms Having a Resistance Mechanism other than Targets of the Xpert Carba-R Assay   

<table><tr><td colspan="2" rowspan="2">Organism                 Strain ID</td><td colspan="1" rowspan="2">Confirmed ResistanceMechanismsa</td><td colspan="3" rowspan="1">Carbapenemsusceptibility (S//R)</td></tr><tr><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">CTX-M (15)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13465</td><td colspan="1" rowspan="1">CTX-M (25)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">OmpC/OmpF deficient</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">TEM (WT+164S)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">kpn5</td><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">kpn12</td><td colspan="1" rowspan="1">TEM; SHV; CTX-M</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">eco1</td><td colspan="1" rowspan="1">TEM; CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">fer29</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">gut25</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Shigella flexnerii</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">PA_3</td><td colspan="1" rowspan="1">AmpC; CTX-M-15; TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">CTX-M (1, -type 15like); SHV</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">32598</td><td colspan="1" rowspan="1">CTX-M (-1, -type 15like); SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">33560</td><td colspan="1" rowspan="1">CTX-M (15); SHV-11;TEM-1</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">33603</td><td colspan="1" rowspan="1">SHV-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">SHV-27</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">SHV (-5, -55); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34430</td><td colspan="1" rowspan="1">SHV; TEM; CTX-M-15</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34680</td><td colspan="1" rowspan="1">TEM; CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34732</td><td colspan="1" rowspan="1">CTX-M (15); SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">PA_174</td><td colspan="1" rowspan="1">GX-/Culture+; SHV;TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">SHV (WT+238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="2" rowspan="2">Organism                 Strain ID</td><td colspan="1" rowspan="2">Confirmed ResistanceMechanismsa</td><td colspan="3" rowspan="1">Carbapenemsusceptibility (/R)b</td></tr><tr><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">STU 669</td><td colspan="1" rowspan="1">SHV (WT+238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">C3015</td><td colspan="1" rowspan="1">AmpC (CMY II); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">RI_100</td><td colspan="1" rowspan="1">AmpC (DHA); SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B4A</td><td colspan="1" rowspan="1">SHV (WT + 238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B13A</td><td colspan="1" rowspan="1">SHV (WT + 238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">RI_474</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">B71</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">DD82A</td><td colspan="1" rowspan="1">SHV (WT + 238S +240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B100</td><td colspan="1" rowspan="1">CTX-M (-1, type-15like); SHV (WT+238S);TTEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">135B</td><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B157</td><td colspan="1" rowspan="1">SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">T2914280</td><td colspan="1" rowspan="1">CTX-M (-1, -15); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">DD188</td><td colspan="1" rowspan="1">TEM (104K + 164S)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">DD189</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">B198B</td><td colspan="1" rowspan="1">CTX-M (-1, type -15like);; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">T3019989-1</td><td colspan="1" rowspan="1">CTXM (-1, type-15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">T3019989-2</td><td colspan="1" rowspan="1">CTX-M (-1, type-15like); SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ENC-THAI14</td><td colspan="1" rowspan="1">VEB-1, TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CB154006</td><td colspan="1" rowspan="1">CTX-M (9); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">S35766</td><td colspan="1" rowspan="1">AmpC(ACT/MIR)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">X1856910</td><td colspan="1" rowspan="1">AmpC (ACT/MIR); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">W3758164</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SHV; TEM.</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">X2135758</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">W3809535</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC0064</td><td colspan="1" rowspan="1">SPM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0099</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0121</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="2" rowspan="2">Organism                  Strain ID</td><td colspan="1" rowspan="2">Confirmed ResistanceMechanismsa</td><td colspan="3" rowspan="1">Carbapenemsusceptibility (S//R)</td></tr><tr><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0122</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0123</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0124</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0130</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0131</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaegroup</td><td colspan="1" rowspan="1">CDC0132</td><td colspan="1" rowspan="1">IMI</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">CDC0164</td><td colspan="1" rowspan="1">IMI</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr></table>

a Presence of these markers of resistance were determined by individual PCR assays, DNA sequence analysis, or by other research-based methods. ${ } ^ { b } S / I / R =$ Susceptible/Intermediate/Resistant; $E T P = .$ Ertapenem, $I M P =$ Imipenem, $M E M = .$ Meropenem

Table 8. Panel of Commensal and Other Enteric Microorganisms Tested in this Study   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 29212</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 700603</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 700621</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 9756</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC BAA-747</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 33128</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 49948</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 51331</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27028</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 49809</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">CCUG 29780 / ATCC 12401</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 51697</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">CCUG 34787</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">CCUG 418</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">CCUG 33629</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">CCUG 17874</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">CCUG 33548</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">CCUG 6325</td></tr></table>

Of the 86 potentially cross-reactive organisms tested, including organisms exhibiting antibiotic resistance mechanisms other than production of KPC, NDM, VIM, IMP and OXA-48, none were detected with the Xpert Carba-R Assay. Four runs were indeterminate [(1) NO RESULT and (3) INVALID)]. All four indeterminate runs were successfully repeated and were reported as NOT DETECTED for all five targets. All data were collected on the GeneXpert Dx (GX-IV) instrument using GeneXpert Dx software version 4.4a.

For each day of the study, an external negative control and two types of experimental positive controls were tested as described in the Analytical Reactivity Study above.

g. Interfering substances:

Not Applicable

# h. Carry-over:

The purpose of the Carry-over Study was to determine the carry-over rate of contamination in negative samples due to the nucleic acid extraction and amplification of high positive samples in the GeneXpert cartridge. In this study, a negative sample was tested in a GeneXpert module on a GeneXpert Dx GX-IV instrument immediately following the testing of a high titer positive sample ( $\mathrm { ~ I ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m l } _ { , }$ ) in the same GeneXpert module. Negative samples consisted of inactivated E. coli cells without target sequences diluted in Sample Reagent to $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m l }$ . The positive sample was composed of inactivated $E .$ . coli cells (with plasmid containing all five Xpert Carba-R target analyte genes—KPC, NDM, VIM, IMP and OXA-48 targets) diluted in Sample Reagent. After an initial negative sample test, sample testing proceeded by alternating negative and high positive sample runs a total of 50 times for two GeneXpert modules. For each day of the study, an external negative control and two types of experimental positive controls were tested as described in the Analytical Reactivity Study. There were 102 total runs where all 50 high positive samples were correctly reported as DETECTED for all five Xpert Carba-R Assay targets. All 52 negative samples reported NOT DETECTED results for all five Xpert Carba-R Assay targets as expected. There were no errors or invalid results reported. Study results indicated no evidence of sample or amplicon carry-over contamination in the GeneXpert Dx GX-IV modules.

# i. Assay cut-off:

Optimized lot specific parameters (LSP) and assay settings for the Xpert Carba-R Assay were determined using pre-clinical data. The pre-clinical study was conducted using patient rectal swabs to determine the performance of the Xpert Carba-R Assay relative to reference DNA sequence analysis of organisms growing on agar media that were non-susceptible to at least one carbapenem. Contrived samples, prepared by seeding carbapenemase-producing bacteria into stool matrix, were also tested with the Xpert Carba-R Assay. Pre-clinical study sensitivity and specificity values for different Ct cut-offs were evaluated for both prospective and contrived samples.

For IMP, VIM, NDM, KPC, and OXA-48 gene targets, the valid cycle threshold (Ct) range was 3.0 to 38.0. For the SPC, the valid Ct range was set from 3.0 to 40.0. A Ct value outside the valid range is reported as NOT DETECTED. The Ct cut-offs are included as automatic calculations in the assay definition file (ADF) provided with the Xpert Carba-R Assay. The lot specific parameters and assay settings described were confirmed in the clinical study, and clinical study results indicated that the cutoff values selected for detection of the five carbapenemase gene targets yielded acceptable performance.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

In a multi-center study, performance of the Xpert Carba-R Assay was evaluated with bacterial isolates grown from both blood agar and MacConkey agar relative to reference DNA amplification of the target DNA followed by bi-directional sequencing. Testing was performed from growth of the isolate on blood agar. To include isolates in the study, organism identification must have been determined prior to testing with the Xpert Carba-R Assay in order to confirm that isolates belonged to the Enterobacteriaceae family or were identified as Pseudomonas aeruginosa or Acinetobacter baumannii. Organism susceptibility status (susceptible, intermediate or resistant) to meropenem, ertapenem and/or imipenem was determined using CLSI standard test methods (M07-A9) and the interpretive criteria found in the FDA drug label and CLSI M100-S24.

For reference DNA amplification followed by sequencing, DNA from culture isolates was purified, quantified, and amplified using primers specific to all 5 target that amplified larger regions than the assay targets. Amplicons of the appropriate size were confirmed on the Agilent 2100 Bioanalyzer and sent for reference bi-directional sequencing analysis.

For Xpert Carba-R Assay testing, well-isolated colonies that grew on each of the agar types were diluted to a 0.5 McFarland standard equivalent suspension using the direct colony suspension method. If discordant results between the Xpert Carba-R Assay and reference (DNA amplification and sequencing) were observed, discrepant testing was performed using bi-directional sequencing on isolates from MacConkey agar plates. All sites performing Xpert Carba-R Assay testing used GeneXpert Dx instruments with GeneXpert Dx System software version 4.3.

Calculations of sensitivity and specificity took into account the ability of the Xpert Carba-R Assay to report the presence or absence of target genes in the pool of test isolates. Because carbapenem non-susceptibilty can be due to mechanisms other than the presence of Xpert Carba-R target DNA sequences, results from all nonsusceptible isolates may not coincide with the presence of target genes by the Xpert Carba-R Assay and may be included in the specificity calculation. In general, Enterobacteriaceae isolates included in sensitivity calculations were determined to be either intermediate or resistant to meropenem, ertapenem and/or imipenem per FDA drug label and CLSI M100-S24. Isolates of Pseudomonas aeruginosa or Acinetobacter baumannii must have been determined to be intermediate or resistant to either imipenem or meropenem (these organisms are known to be resistant to ertapenem). In specificity calculations, Enterobacteriaceae isolates may have been determined to be susceptible or intermediate/resistant to meropenem, ertapenem, and imipenem per FDA drug label and CLSI M100-S24. Pseudomonas aeruginosa and Acinetobacter baumannii isolates should have been susceptible to both imipenem and meropenem.

A total of 489 bacterial isolates (431 clinical stock isolates and 58 fresh isolates) were enrolled in the clinical study across four clinical sites. Four isolates that had been previously enrolled were excluded from the study, and two isolates were excluded because reference method testing was not performed. An additional sixteen samples were excluded because organisms were not identified as Enterobacteriaceae, Pseudomonas aeruginosa, or Acinetobacter baumannii. Thus, 467 compliant samples remained in the final analysis.

Isolates grown on both blood agar and MacConkey agar were evaluated with the Xpert Carba-R Assay for a total of 934 runs (467 each from blood agar and McConkey agar). Performance of the Xpert Carba-R Assay was assessed separately for each type of agar, resistance marker target, and overall (all 5 targets combined) relative to reference sequencing for each of the two agar types used to grow isolates. The study showed that $9 9 . 9 \%$ (933/934) of isolates yielded results on the first run. One isolate resulted in a NO RESULT outcome upon initial testing, but yielded a result upon repeat testing. The final invalid rate was reported as $0 \%$ .

When tested with isolates from blood agar, the Xpert Carba-R Assay demonstrated an overall sensitivity and specificity of $100 \%$ ( $9 5 \%$ CI: 99.0-100) and $9 8 . 1 \%$ ( $9 5 \%$ CI: 93.2-99.5), respectively, relative to the reference method performed from isolates grown on blood agar plates. Similar results were reported for isolates grown on

MacConkey agar, where the Xpert Carba-R Assay demonstrated an overall sensitivity and specificity of $100 \%$ $9 5 \%$ CI: 99.0-100) and $9 7 . 1 \%$ ( $9 5 \%$ CI: 91.8-99.0),

respectively. The combined performance was evaluated based on defining a result as positive for the Xpert Carba-R Assay if any of the targets were positive and negative for the assay if all the targets were negative.

Combined performance, performance by target, and performance by organism group for the Xpert Carba-R Assay with isolates grown on blood agar and MacConkey agar are shown in Table 9 (Combined Performance), Table 10 (Individual Targets), and Table 11 (Organism Group) below.

Table 9. Xpert Carba-R Assay (Blood agar and MacConkey agar) vs Reference Method—Combined   

<table><tr><td rowspan=1 colspan=1>Plate</td><td rowspan=1 colspan=1>TotalIsolates</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Sensitivity(95% CI)</td><td rowspan=1 colspan=1>Specificity(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Blood agar</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>364</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(99.0-100)</td><td rowspan=1 colspan=1>98.1%(93.2-99.5)</td></tr><tr><td rowspan=1 colspan=1>MacConkeyagar</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>364</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(99.0-100)</td><td rowspan=1 colspan=1>97.1%(91.8-99.0)</td></tr></table>

Table 10. Xpert Carba-R Assay (Blood agar and MacConkey agar) vs Reference Method—by Target   

<table><tr><td rowspan=1 colspan=1>Plate</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Sensitivity(95% CI)</td><td rowspan=1 colspan=1>Specificity(95% CI)</td></tr><tr><td rowspan=5 colspan=1>Blood agar</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>426</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(91.2-100)</td><td rowspan=1 colspan=1>99.8%(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.5-100)</td><td rowspan=1 colspan=1>99.7%(98.5-100)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>389</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.3-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.6-100)</td><td rowspan=1 colspan=1>99.7%(98.5-100)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.9-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=5 colspan=1>MacConkeyagar</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>la</td><td rowspan=1 colspan=1>426</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(91.2-100)</td><td rowspan=1 colspan=1>99.8%(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.5-100)</td><td rowspan=1 colspan=1>99.7%(98.5-100)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>388</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.3-100)</td><td rowspan=1 colspan=1>99.7%(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>383</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.6-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(95.9-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr></table>

a bi-directional DNA sequencing result for this false positive IMP isolate exhibited $9 2 . 9 5 \%$ sequence homology which was slightly below the $9 5 \%$ cut-off criteria. Discrepant testing was not performed.

b Discrepant testing results: 1 of 1 was VIM positive.

c This false positive sample was investigated and may be due to KPC cross-contamination at the level of sample preparation. Discrepant testing did not produce a sequence match with the KPC target. Discrepant testing produced a sequence match for the VIM target. This isolate was classified as a TP in the overall assessment presented in Table 9 above.

d The clinical site reported that in-house characterization of this false positive isolate prior to study testing resulted in a positive NDM gene target. Discrepant testing did not produce a sequence match for any of the 5 gene targets.

Table 11. Xpert Carba-R Assay Performance (Blood agar and MacConkey agar) vs Reference Method—by Organism Group   

<table><tr><td colspan="1" rowspan="1">Plate</td><td colspan="1" rowspan="1">Organism Group</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FP</td><td colspan="2" rowspan="1">TN  FN</td><td colspan="1" rowspan="1">Sensitivity(95 CI)</td><td colspan="1" rowspan="1">Specificity(95 CI)</td></tr><tr><td colspan="1" rowspan="18">Blood agar</td><td colspan="1" rowspan="6">Enterobacteriaceae</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">339</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(51.0-100)</td><td colspan="1" rowspan="1">100%(98.9-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">291</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(93.0-100)</td><td colspan="1" rowspan="1">99.7%(98.1-99.9)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.0-100)</td><td colspan="1" rowspan="1">100%(98.6-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">259</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.6-100)</td><td colspan="1" rowspan="1">99.6%(97.9-99.9)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">254</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.9-100)</td><td colspan="1" rowspan="1">100%(98.5-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">291a</td><td colspan="1" rowspan="1">1a</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(98.7-100)</td><td colspan="1" rowspan="1">98.1%(89.9-99.7)</td></tr><tr><td colspan="1" rowspan="6">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(80.6-100)</td><td colspan="1" rowspan="1">98.4%(91.7-99.7)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.0-100)</td><td colspan="1" rowspan="1">100%(92.7-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(92.6-100)</td><td colspan="1" rowspan="1">96.9%(84.3-99.5)</td></tr><tr><td colspan="1" rowspan="6">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(83.9-100)</td><td colspan="1" rowspan="1">100%(86.2-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(56.6-100)</td><td colspan="1" rowspan="1">100%(91.0-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(86.7-100)</td><td colspan="1" rowspan="1">100%(83.2-100)</td></tr><tr><td colspan="1" rowspan="7">MacConkeyagar</td><td colspan="1" rowspan="6">Enterobacteriaceae</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">339</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(51.0-100)</td><td colspan="1" rowspan="1">100%(98.9-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">291</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(93.0-100)</td><td colspan="1" rowspan="1">99.7%(98.1-99.9)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">269</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.0-100)</td><td colspan="1" rowspan="1">99.6%(97.9-99.9)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">260</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.6-100)</td><td colspan="1" rowspan="1">100%(98.5-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">254</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(95.9-100)</td><td colspan="1" rowspan="1">100%(98.5-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">291a</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(98.7-100)</td><td colspan="1" rowspan="1">96.2%(87.0-98.9)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(80.6-100)</td><td colspan="1" rowspan="1">98.4%(91.7-99.7)</td></tr><tr><td colspan="1" rowspan="1">Plate</td><td colspan="1" rowspan="1">Organism Group</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FP</td><td colspan="2" rowspan="1">TN   FN</td><td colspan="1" rowspan="1">Sensitivity(95 CI)</td><td colspan="1" rowspan="1">Specificity(95 CI)</td></tr><tr><td colspan="1" rowspan="11"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.0-100)</td><td colspan="1" rowspan="1">100%(92.7-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(95.4-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(92.6-100)</td><td colspan="1" rowspan="1">96.9%(84.3-99.5)</td></tr><tr><td colspan="1" rowspan="6">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(83.9-100)</td><td colspan="1" rowspan="1">100%(86.2-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(56.6-100)</td><td colspan="1" rowspan="1">100%(91.0-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(92.0-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(86.7-100)</td><td colspan="1" rowspan="1">100%(83.2-100)</td></tr></table>

a Multiple target results were observed for some isolates.

Ten isolates were identified where at least two Xpert Carba-R Assay targets were detected. These results are shown in Table 12 below.

Table 12. Isolates with Multiple Targets Detected   

<table><tr><td rowspan=1 colspan=1># of Samples withMultiple Targets</td><td rowspan=1 colspan=1>Agar Typea</td><td rowspan=1 colspan=1>Targets Detected byXpert Carba-R Assay</td><td rowspan=1 colspan=1>Targets Detected byReference Sequencing</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>BA, MC</td><td rowspan=1 colspan=1>NDM, OXA-48</td><td rowspan=1 colspan=1>NDM, OXA-48</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>BA</td><td rowspan=1 colspan=1>VIM, KPC</td><td rowspan=1 colspan=1>VIM</td></tr></table>

a BA=Blood Agar, MC=MacConkey Agar

External controls for the Xpert Carba-R Assay consisted of one negative sample, one sample positive for all (5) targets of the assay, and five different positive controls each containing a single target of the assay. The negative control and five-target positive controls were run on each day that study samples were tested, along with two of the single-target positive controls (on a rotating basis). Study samples were not run until correct results were obtained for each of the four controls. External control data was compiled across all sites and overall QC results were acceptable.

b. Clinical specificity: See comments in 3a above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

Of the total 451 isolates grown on blood agar plates and reported to be carbapenem nonsusceptible based on conventional phenotypic (AST) results, 356 isolates were determined to have one or more of the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ gene targets by the Xpert Carba-R Assay. Results from isolate growth on MacConkey agar plates showed similar results with 357 blaKPC, blaNDM, blaVIM, blaOXA-48, and/or $b l a _ { \mathrm { I N P } }$ gene targets detected in 451 carbapenem non-susceptible isolates. Table 13 and Table14 show the results of the Xpert Carba-R Assay compared to susceptibility test results for the 467 isolates included in the clinical study.

Table 13. Xpert Carba-R Assay (Blood agar) vs Susceptibility Test Report   

<table><tr><td rowspan=5 colspan=1>R-er adRASssA</td><td rowspan=1 colspan=4>Phenotypea</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS(non-susceptible)</td><td rowspan=1 colspan=1>S(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>GeneDetected</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>366</td></tr><tr><td rowspan=1 colspan=1>GeneNot Detected</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>451</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>467</td></tr></table>

Table 14. Xpert Carba-R Assay (MacConkey agar) vs Susceptibility Test Report   

<table><tr><td rowspan=5 colspan=1>P-ee cRAsssE</td><td rowspan=1 colspan=4>Phenotypea</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS(non-susceptible)</td><td rowspan=1 colspan=1>S(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Gene Detected</td><td rowspan=1 colspan=1>357</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>367</td></tr><tr><td rowspan=1 colspan=1>Gene Not Detected</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>451</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>467</td></tr></table>

a A non-susceptible phenotype means the isolate was intermediate or resistant to at least one carbapenem. A susceptible phenotype means the isolate was susceptible to imipenem, meropenem, and ertapenem.

# N. Instrument Name:

GeneXpert Instrument Systems

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X_ __ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __ _X____ or No

3. Specimen Identification: Similar to previously cleared system.

4. Specimen Sampling and Handling:

A bacterial suspension equivalent to a 0.5 McFarland suspension is prepared and a $1 0 ~ \mu \mathrm { l }$ loop of suspension is transferred to $5 \mathrm { m l }$ of Xpert Carba-R Sample Reagent. An aliquot of sample $( 1 . 7 \mathrm { m l } )$ is then transferred to the sample chamber of the disposable, single-use fluidic cartridge (Xpert Carba-R cartridge). The user initiates a test from the system user interface and places the cartridge into the GeneXpert Instrument System. Additional sample preparation, amplification, and real-time detection are all fully-automated and completed by the instrument system.

5. Calibration: No calibration is required by the user.

6. Quality Control:

Quality control is addressed for each separately cleared assay to be run on the instrument.   
See section M1(c) for information on internal and external controls.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.