# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k152470

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the Great Basin Staph ID/R Blood Culture Panel.

# C. Measurand:

Staphylococcus aureus, Staphylococcus lugdunensis and various Staphylococcus species to the genus level and the detection of the mecA gene for methicillin resistance.

# D. Type of Test:

A multiplexed nucleic acid-based test intended for use with the Great Basin PA500 Portrait Analyzer instrument for the qualitative in vitro detection and identification of multiple bacterial nucleic acids and select genetic determinants of antimicrobial resistance from patient positive blood culture specimens. The Great Basin Staph ID/R Blood Culture Panel assay is performed directly on positive blood culture specimens determined by Gram stain to contain gram-positive cocci in clusters (GPCC) or grampositive cocci in singles (GPC).

# E. Applicant:

Great Basin Scientific, Inc.

F. Proprietary and Established Names:

Great Basin Staph ID/R Blood Culture Panel

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3365 - Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures

2. Classification: Class II

3. Product codes: PAM- Gram-positive bacteria and their resistance markers OOI- Real time Nucleic acid amplification system

4. Panel: 83, (Microbiology)

# H. Intended Use:

1. Intended use(s):

The Great Basin Staph ID/R Blood Culture Panel is a qualitative, multiplex, nucleic acid-based in vitro diagnostic assay intended for the simultaneous identification of nucleic acid from Staphylococcus aureus, Staphylococcus lugdunensis and various Staphylococcus species to the genus level and the detection of the mecA gene for methicillin resistance directly from patient positive blood culture specimens. The test utilizes automated hot-start enabled polymerase chain reaction (PCR) for the amplification of specific DNA targets detected by hybridization probes immobilized on a silicon chip surface. The assay is performed directly on positive blood culture specimens identified as positive by continuous monitoring blood culture system that demonstrates the presence of organisms as determined by Gram stain to contain gram-positive cocci in clusters (GPCC) or gram-positive cocci in singles (GPC). The test may be performed using blood culture bottles. The Staph ID/R Blood Culture Panel identifies Staphylococcus aureus (SA), and Staphylococcus lugdunensis, and detects other Staphylococcus species without identification to species level.

The Portrait Staph ID/R Blood Culture Panel is indicated for use in conjunction with other clinical or laboratory findings to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor these infections. Subculturing positive blood cultures is necessary to recover viable organisms for further identification, susceptibility testing, or epidemiological typing to identify organisms in the blood culture that are not detected by the Great Basin Staph ID/R Blood Culture Panel. If detected, mecA may or may not be associated with Staphylococcus spp. detected or the agent responsible for the disease. Negative results for mecA antimicrobial resistance gene assays do not always indicate susceptibility, as other mechanisms of resistance to methicillin exist.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: For use with The Great Basin PA500 Portrait™ Analyzer System.

# I. Device Description:

The Great Basin Staph ID/R Blood Panel on the PA500 Portrait™ Analyzer System utilizes automated hot-start enabled polymerase chain reaction (PCR) amplification technology to amplify specific nucleic acid sequences that are detected using species specific Staphylococcal DNA hybridization probes immobilized on a modified silicon chip surface.

Target genomic DNA is extracted from microbial cells and diluted to reduce potential inhibitors of the PCR reaction. During the PCR process, double-stranded DNA is separated and target nucleic acid sequences are amplified by thermal cycling. Biotinlabeled primers direct amplification of specific nucleic acid sequences within a variable region of the tuf gene for identification of coagulase-negative Staphylococcus species, within a conserved region of the thermonuclease (nuc) gene for specific identification of Staphylococcus aureus, and the mecA gene for detecting oxacillin/methicillin resistance. Following the PCR process, biotin-labeled, amplified target DNA sequences are hybridized to an array of probes immobilized on the silicon chip surface, then incubated with anti-biotin antibody conjugated to the horseradish peroxidase enzyme (HRP). The unbound conjugate is removed by washing and tetramethylbenzidine (TMB) is added to produce a colored precipitate at the location of the probe/target sequence complex. The resulting signal is detected by the automated Portrait Optical Reader within the Portrait System.

The single-use test cartridge contains blister packs, fluidic channels, processing chambers, and the assay chip coated with an array of sequence-specific detection probes. All reagents are contained within the integrated blister packs with the exception of the amplification enzymes that are lyophilized and placed into the Amplification Chamber. A positive blood culture specimen is placed into a sample port of the test cartridge for processing. Multiple fluidic channels move reagents from integrated blister packs to chambers where reagent mixing and sample processing occur. A waste chamber, self-contained and segregated within the test cartridge, collects and stores reagent waste.

The Great Basin PA500 Portrait Analyzer System is a fully automated system that includes the Portrait Analyzer, single-use Staph ID/R Blood Culture Panel Test Cartridges, and the Portrait data analysis software. The PA500 Portrait Analyzer System is designed to perform automated sample preparation, PCR, and optical chipbased detection with integrated data analysis in approximately 110 minutes.

Materials provided in each kit:

Staph ID/R Blood Culture Panel Test Cartridge Kit. Each test cartridge   
includes: o Blister Pack 1: Dilution Buffer (Tris buffer, salts, surfactant, BSA (bovine serum-albumin), antibacterial agents) o Blister Pack 2: Extraction Buffer (Enzymes, salts) o Blister Pack 3: Wash Solution (Saline Sodium Citrate (SSC) buffer, surfactant, preservative) o Blister Pack 4: Hybridization Buffer (SSC buffer, surfactant, preservative) o Blister Pack 5: Conjugate (Sodium citrate buffer, salts, fetal bovine serum (FBS), peroxidase conjugated monoclonal mouse antibody, preservative) o Blister Pack 6: Conjugate (Tetramethylbenzidine (TMB)) o Chamber 1 (CC1): Stir bar, SPC (Lyophilized Bacillus subtilis) o Chamber 2 (CC2): Stir bar o Chamber 3 (Amp) Amplification Reagents (Tris buffer salts, sucrose surfactant, nucleotides, primers, DNA polymerase) o Chamber 3 (Detect): Silicon chip with immobilized DNA probes

Materials required but not provided:

• PA500 Portrait™ Analyzer System and Operator Manual • Calibrated, $5 0 ~ \mu \mathrm { L }$ , fixed-volume pipette • $2 0 0 \mu \mathrm { L }$ sterile, barrier filter pipette tips

# J. Substantial Equivalence Information:

1. Predicate device name(s): Nanosphere Verigene® Gram-Positive Blood Culture Nucleic Acid Test (BC-GP)

2. Predicate 510(k) number(s): k113450

3. Comparison with predicate:

<table><tr><td>Item</td><td>Device: Staph ID/R Blood Culture Panel</td><td>Predicate (k113450)</td></tr><tr><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">Great Basin Scientific, Inc.</td><td colspan="1" rowspan="1">Nanosphere, Inc.</td></tr><tr><td colspan="1" rowspan="1">Trade Name</td><td colspan="1" rowspan="1">PortraitTM Staph ID/R Blood CulturePanel</td><td colspan="1" rowspan="1">Verigene® Gram-PositiveBlood Culture Nucleic Acid Test (BC-GP)</td></tr><tr><td colspan="1" rowspan="1">510(k) Number</td><td colspan="1" rowspan="1">k152470</td><td colspan="1" rowspan="1">k113450</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications for Use</td><td colspan="1" rowspan="1">The Great Basin Staph ID/R Blood CulturePanel is a qualitative, multiplex, nucleic acid-based in vitro diagnostic assay intended for thesimultaneous identification of nucleic acidfrom Staphylococcus aureus, Staphylococcuslugdunensis and various Staphylococcusspecies to the genus level and the detection ofthe mecA gene for methicillin resistancedirectly from patient positive blood culturespecimens. The test utilizes automated hot-startenabled polymerase chain reaction (PCR) forthe amplification of specific DNA targetsdetected by hybridization probes immobilizedon a silicon chip surface. The assay isperformed directly on positive blood culturespecimens identified as positive by continuousmonitoring blood culture system thatdemonstrates the presence of organisms asdetermined by Gram stain to contain gram-positive cocci in clusters (GPCC) or gram-positive cocci in singles (GPC). The test maybe performed using blood culture bottles. TheStaph ID/R Blood Culture Panel identifiesStaphylococcus aureus (SA), andStaphylococcus lugdunensis, and detects otherStaphylococcus species without identificationto species level.The Portrait Staph ID/R Blood Culture Panel isindicated for use in conjunction with otherclinical or laboratory findings to aid in thediagnosis of bacterial bloodstream infections;however, it is not used to monitor theseinfections. Sub-culturing positive bloodcultures is necessary to recover viableorganisms for further identification,susceptibility testing, or epidemiological typingto identify organisms in the blood culture thatare not detected by the Great Basin Staph ID/RBlood Culture Panel. If detected, mecA may ormay not be associated with Staphylococcusspp. detected or the agent responsible for thedisease. Negative results for mecAantimicrobial resistance gene assays do notalways indicate susceptibility, as othermechanisms of resistance to methicillin exist.</td><td colspan="1" rowspan="1">The Verigene® Gram Positive Blood CultureNucleic Acid Test (BC-GP) performed using thesample-to-result Verigene System is a qualitative,multiplexed in vitro diagnostic test for thesimultaneous detection and identification ofpotentially pathogenic gram-positive bacteriawhich may cause bloodstream infection (BSI).BC-GP is indicated for use in conjunction withother clinical and laboratory findings, such asculture, to aid in the diagnosis of bacterialbloodstream infections; however, it is not used tomonitor bloodstream infections. BC-Gp detectsthe following bacterial genera and species:Staphylococcus spp., Staphylococcus aureus,Staphylococcus epidermidis, and Staphylococcuslugdunensis, Streptococcus spp., Streptococcuspneumonia, Streptococcus pyogenes,Streptococcus agalactiae, Streptococcusanginosus group, Entercoccus faecalis,Enterococcus faecium, Listeria spp. In addition,BC-GP detects the mecA resistance marker, andthe vanA and vanB resistance markers, inferringvanA/vanB mediated vancomycin resistance toeither E. faecalis or E. faecium, or the mecA-mediated methicillin resistance to either S. aureusor S. epidermidis. BC-GP is indicated for use inconjunction with other clinical or laboratoryfindings to aid in the diagnosis of bacterialbloodstream infections; however, it is not used tomonitor these infections. Sub-culturing of positiveblood cultures is necessary to recover organismsfor susceptibility testing, identification oforganisms not detected by BC-GP, differentiationof mixed growth, association of antimicrobialmarker genes to a specific organism, or forepidemiological testing.</td></tr><tr><td colspan="1" rowspan="1">Qualitative/Quantitative</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Single-Use TestCartridge</td><td colspan="1" rowspan="1">Disposable, single-use, self-containedfluidic test cartridge</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Multiplex nucleic acid array-baseddetection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Direct blood culture positive by Gramstain for GPCC or GPC</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">One Internal Processing Control (wholeorganism complete assay control)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Calibration</td><td colspan="1" rowspan="1">Not required</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications for Use</td><td colspan="1" rowspan="1">Identification of S. aureus, S.lugdunensis, detection of otherStaphylococcus spp. Detection of themecA gene for methicillin resistance inall Staphylococcus organisms.</td><td colspan="1" rowspan="1">Tests for same Staphylococcus targets.Detection of mecA gene for methicillinreistance in S. aureus and S. epidermidisonly. Tests for additional gram-positivebacteria including Streptococcus spp.,Streptococcus pneumonia, Streptococcuspyogenes, Streptococcus agalactiae,Streptococcus anginosus group, Entercoccusfaecalis, Enterococcus faecium,and Listeriaspp. Tests for additional resistance markersincluding the vanA and vanB resistancemarkers, inferring vanA/vanB mediatedvancomycin resistance to either E. faecalis orE. faecium.</td></tr><tr><td colspan="1" rowspan="1">TestPrinciple/Technology</td><td colspan="1" rowspan="1">Fully automated multiplex PCR anddetection by target-specific captureoligonucleotides immobilized in amacroarray format onto a chip surfacefor probe-based end-point.</td><td colspan="1" rowspan="1">Fully automated multiplex DNA detection ofspecific nucleic acid sequences in amicroarray format using target-specificcapture and mediator oligonucleotides forprobe-based end-point detection.</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">PA500 Portrait Analyzer</td><td colspan="1" rowspan="1">Verigene Reader and Processor SPTime</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">110 minutes</td><td colspan="1" rowspan="1">150 minutes</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and Food and Drug Administration Staff - Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices, (November 9, 2012 August 27, 2014) Draft Guidance for Industry and FDA Staff - Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of methicillin

resistant Staphylococcus aureus (MRSA) for Culture Based Devices (June 15, 2011) Draft Guidance for Industry and Food and Drug Administration Staff - Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA) (January 5, 2011) • Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, FDA Guidance Document (March 13, 2007) User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute (CLSI) Approved Guideline – Second Edition, EP12- A2 (January 2008) Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline, MM3-A2 (February 2006) Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005) Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005) Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999) General Principle of Software Validation; Final Guidance for Industry and FDA Staff (January 11, 2002)

# L. Test Principle:

The Great Basin Staph ID/R Blood Panel on the PA500 Portrait™ Analyzer System utilizes automated hot-start enabled polymerase chain reaction (PCR) amplification technology to amplify specific nucleic acid sequences that are detected using species specific Staphylococcal DNA hybridization probes immobilized on a modified silicon chip surface.

Target genomic DNA is extracted from microbial cells and diluted to reduce potential inhibitors of the PCR reaction. During the PCR process, double-stranded DNA is separated and target nucleic acid sequences are amplified by thermal cycling. Biotin-labeled primers direct amplification of specific nucleic acid sequences within a variable region of the tuf gene for identification of coagulase-negative Staphylococcus species, within conserved region of the thermonuclease (nuc) gene for specific identification of Staphylococcus aureus, and the mecA gene for detecting oxacillin/methicillin resistance. Following the PCR process, biotinlabeled, amplified target DNA sequences are hybridized to an array of probes immobilized on the silicon chip surface, then incubated with anti-biotin antibody conjugated to the horseradish peroxidase enzyme (HRP). The unbound conjugate is removed by washing and tetramethylbenzidine (TMB) is added to produce a colored precipitate at the location of the probe/target sequence complex. The resulting signal is detected by the automated Portrait Optical Reader within the Portrait System.

# M. Performance Characteristics:

1. Analytical performance: a. Analytical Sensitivity:

The limit of detection (LoD) of the Staph ID/R Blood Culture Panel to Staphylococcus with or without mecA was assessed and confirmed with 22 different strains. For these studies, overnight incubations of each bacterial stock in Tryptic Soy Broth (TSB) were measured by optical density to estimate the inoculum, serially diluted and spiked into a BACTEC Plus Aerobic/F blood bottle containing negative blood. Spiked bottles were incubated until alarm positivity in a BACTEC Blood Culture System. Alarm positive blood cultures were gram stained, diluted, plated on agar plates and colonies were counted the following day. The enumerated blood culture bottles were diluted to a $1 \mathrm { x } 1 0 ^ { 5 } – 1 0 ^ { 6 }$ targeted sample input CFU/mL range in BACTEC Plus Aerobic/F media containing negative blood, tested on device and plated on agar plates to enumerate the actual sample input $\mathrm { C F U / m L }$ . The estimations of the sample input CFU/mL were revised to reflect the correct CFU/mL by this colony counting method.

The initial serial dilutions and testing for three strains were used to estimate the ideal range to test for all 22 strains. Correct results were determined by the call reported vs expected, e.g. Staphylococcus aureus ATCC strain BAA-1682 with mecA present was correct if detected in a ‘Positive, Staphylococcus aureus, mecA Present” call. The results for the initial testing are summarized in the table below.

Performance of the Staph ID/R Blood Culture Panel on serial dilutions of three Staphylococcus strains for initial sensitivity study.   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC #</td><td rowspan=1 colspan=1>Sample Input(CFU/mL)</td><td rowspan=1 colspan=1>Correct Staph ID/RBloodCulture Panel</td></tr><tr><td rowspan=3 colspan=1>S. aureus</td><td rowspan=3 colspan=1>BAA-1682</td><td rowspan=1 colspan=1>1.3x104</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>4.1 x104</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>1.3 x105</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=3 colspan=1>S. epidermidis</td><td rowspan=3 colspan=1>700562</td><td rowspan=1 colspan=1>1.4 x104</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>4.4 x104</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=1>1.4 x105</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=3 colspan=1>S. lugdunensis</td><td rowspan=3 colspan=1>49576</td><td rowspan=1 colspan=1>1.4 x104</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>4.5 x104</td><td rowspan=1 colspan=1>2/4</td></tr><tr><td rowspan=1 colspan=1>1.4 x105</td><td rowspan=1 colspan=1>4/4</td></tr></table>

In order to confirm an estimated LoD based on colony counting, an additional minimum of 20 replicates were tested for $S .$ aureus BAA-1682, S. epidermidis 700562, and $S .$ lugdunensis 49576 based on the results from on the initial sensitivity study. The 20 replicates initially tested for the three strains resulted in less than 19/20 correct calls, and were subsequently retested at a half-log increased concentration, and resulted in $100 \%$ correct calls. These experiments identified a target sample input range of $1 \mathrm { x } 1 0 ^ { 5 }$ - $1 0 ^ { 6 } \mathrm { C F U / m L }$ for future LoD studies. A minimum of 19/20 valid runs was required to identify the LoD for each strain tested.

The LoDs for $6 S .$ aureus strains $\pm$ mecA are reported as $3 . 5 { - } 8 . 2 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ ; the LoD for each S. aureus strain is listed in below.

Performance of the Staph ID/R Blood Culture Panel on a minimum of 20 replicates of 6 Staphylococcus aureus strains $\pm$ mecA for establishing LoD.   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC #</td><td rowspan=1 colspan=1>Sample Input(CFU/mL)</td><td rowspan=1 colspan=1>Correct Staph ID/RBloodCulture Panel</td></tr><tr><td rowspan=3 colspan=1>S. aureus, mecA +</td><td rowspan=1 colspan=1>BAA-1680</td><td rowspan=1 colspan=1>3.5 x105</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>BAA-1682</td><td rowspan=1 colspan=1>4.0 x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>BAA-1684</td><td rowspan=1 colspan=1>8.2 x105</td><td rowspan=1 colspan=1>22/22</td></tr><tr><td rowspan=3 colspan=1>S. aureus, mecA-</td><td rowspan=1 colspan=1>25923</td><td rowspan=1 colspan=1>5.1 x105</td><td rowspan=1 colspan=1>21/21</td></tr><tr><td rowspan=1 colspan=1>6538</td><td rowspan=1 colspan=1>3.9 x105</td><td rowspan=1 colspan=1>22/22</td></tr><tr><td rowspan=1 colspan=1>11632</td><td rowspan=1 colspan=1>6.2 x105</td><td rowspan=1 colspan=1>20/20*</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run

The LoDs for $6 S .$ epidermidis strains $\pm$ mecA are reported as $2 . 2 – 7 . 1 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ ; the LoD for each $S .$ epidermidis strain is listed in the table below.

Performance of the Staph ID/R Blood Culture Panel on a minimum of 20 replicates of 6 Staphylococcus epidermidis strains $\pm$ mecA for establishing LoD.   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC #</td><td rowspan=1 colspan=1>Sample Input(CFU/mL</td><td rowspan=1 colspan=1>Correct Staph ID/RBlood</td></tr><tr><td rowspan=4 colspan=1>S. epidermidis,mecA +</td><td rowspan=1 colspan=1>35984</td><td rowspan=1 colspan=1>3.6 x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>51625</td><td rowspan=1 colspan=1>4.0 x105</td><td rowspan=1 colspan=1>22/22</td></tr><tr><td rowspan=1 colspan=1>700562</td><td rowspan=1 colspan=1>4.3 x105</td><td rowspan=1 colspan=1>27/27</td></tr><tr><td rowspan=1 colspan=1>700566</td><td rowspan=1 colspan=1>5.8 x105</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=2 colspan=1>S. epidermidis,mecA-</td><td rowspan=1 colspan=1>700583</td><td rowspan=1 colspan=1>2.2 x105</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>12228</td><td rowspan=1 colspan=1>7.1 x105</td><td rowspan=1 colspan=1>22/23</td></tr></table>

The LoDs for $3 \ : S .$ lugdunensis strains without mecA are reported as $2 . 8 – 4 . 7 \mathrm { x } 1 0 ^ { 5 }$ $\mathrm { C F U / m L }$ ;   
the LoD for each S. lugdunensis strain is listed in below. LoD’s for 7 Staphylococcus species $\pm$ mecA, excluding S. aureus, S. epidermidis, and S.   
lugdunensis, are reported as $2 . 0 { - } 5 . 3 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ . The species were selected for inclusion

Performance of the Staph ID/R Blood Culture Panel on a minimum of 20 replicates of 3 Staphylococcus lugdunensis strains $\pm$ mecA for establishing LoD.   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCCH</td><td rowspan=1 colspan=1>Sample Input(CFU/mL</td><td rowspan=1 colspan=1>Correct Staph ID/RBlood</td></tr><tr><td rowspan=3 colspan=1>S. lugdunensis,mecA-</td><td rowspan=1 colspan=1>43809</td><td rowspan=1 colspan=1>2.8 x105</td><td rowspan=1 colspan=1>22/23</td></tr><tr><td rowspan=1 colspan=1>49576</td><td rowspan=1 colspan=1>4.5 x105</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>7990</td><td rowspan=1 colspan=1>4.7 x105</td><td rowspan=1 colspan=1>23/23</td></tr></table>

in the LoD studies based on tuf sequence variation from in silico analysis of the target sequence. The LoD for each Staphylococcus strain is listed in in the table below.

Performance of the Staph ID/R Blood Culture Panel on a minimum of 20 replicates of 7 Staphylococcus strains $\pm$ mecA for establishing LoD.   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>Sample Input(CFU/mL</td><td rowspan=1 colspan=1>Correct Staph ID/RBlood</td></tr><tr><td rowspan=1 colspan=1>S. simulans, mecA -</td><td rowspan=1 colspan=1>27848</td><td rowspan=1 colspan=1>2.0x105</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus,</td><td rowspan=1 colspan=1>BAA-1693</td><td rowspan=1 colspan=1>3.1 x105</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>S. warneri, mecA -</td><td rowspan=1 colspan=1>27830</td><td rowspan=1 colspan=1>4.0 x10^5</td><td rowspan=1 colspan=1>22/22</td></tr><tr><td rowspan=1 colspan=1>S. sciuri, mecA -</td><td rowspan=1 colspan=1>29060</td><td rowspan=1 colspan=1>4.3 x10^5</td><td rowspan=1 colspan=1>21/21**</td></tr><tr><td rowspan=1 colspan=1>S. capitis, mecA -</td><td rowspan=1 colspan=1>35661</td><td rowspan=1 colspan=1>1.5 x10^5$</td><td rowspan=1 colspan=1>20/20*</td></tr><tr><td rowspan=1 colspan=1>S. pasteuri, mecA -</td><td rowspan=1 colspan=1>51129</td><td rowspan=1 colspan=1>5.3 x105</td><td rowspan=1 colspan=1>21/22</td></tr><tr><td rowspan=1 colspan=1>S. hominis, mecA -</td><td rowspan=1 colspan=1>27844</td><td rowspan=1 colspan=1>5.3 x105$</td><td rowspan=1 colspan=1>23/23*</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run \*\*This set of test runs also contained 2 "Invalid" runs

# b. Analytical Reactivity (Inclusivity):

The analytical reactivity of the Staph ID/R Blood Culture Panel was tested against 48 well characterized S. aureus strains from ATCC representing USA100-1200 and SCCmecA I-VI, XI types representative of temporal and geographical diversity. In addition, 104 untyped strains, representing S. aureus, S. epidermidis, S. lugdunensis, and other various Staphylococcus species were tested in the Staph ID/R Blood Culture Panel.

The Staphylococcus stock cultures were prepared in TSB broth, incubated overnight, and measured by optical density to estimate the inoculum concentration. Broth cultures were combined with BACTEC Plus Aerobic/F media containing negative blood previously incubated 24 hours in a BACTEC Blood Culture System without alarm positivity. The Staphylococcus cultures were diluted to a sample input target range of approximately 2-3X LoD $( 1 { - } 2 . 5 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ determined by the Staphylococcus strain LoD studies. The CFU concentrations for each strain were estimated by optical density measurements and then confirmed by colony counting. This method led to some strains being tested at 4-7X LOD $( 2 . 5 { - } 7 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ).

The Staph ID/R Blood Culture Panel correctly detected all of the additional Staphylococcus strains, mecA present or absent with a sample input range of $3 . 2 \mathrm { x } 1 0 ^ { 5 }$ to $6 . { \dot { 7 } } { \mathrm { x } } { \dot { 1 0 } } ^ { 6 } \mathrm { C F U / m L }$ listed in the table below.

Analytical Reactivity (Inclusivity) Panel: Staphylococcus strains and results for inclusivity by the Staph ID/R Blood Culture Panel.   

<table><tr><td>Staphylococcus species; ATCC, CCUG, NRS, Clinical #</td><td>SCCmec, PFGE type / source</td><td>Staphylococcus species; ATCC, CCUG, NRS, Clinical #</td><td>SCCmec, PFGE type / source</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-38</td><td colspan="1" rowspan="1">I, Denmark</td><td colspan="1" rowspan="1">S. caprae 35538</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-44</td><td colspan="1" rowspan="1">I, Iberian</td><td colspan="1" rowspan="1">S. caprae 51548</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus700698</td><td colspan="1" rowspan="1">II, Japan</td><td colspan="1" rowspan="1">S. chromogenes 43764</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-41</td><td colspan="1" rowspan="1">II, USA100</td><td colspan="1" rowspan="1">S. chohnii subsp. cohnii</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1681</td><td colspan="1" rowspan="1">II, USA100</td><td colspan="1" rowspan="1">.chonii subs urealyics</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1682</td><td colspan="1" rowspan="1">II USA100</td><td colspan="1" rowspan="1">S. condimenti 4753</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1761</td><td colspan="1" rowspan="1">II, USA100</td><td colspan="1" rowspan="1">S. carnosus 51365</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS660</td><td colspan="1" rowspan="1">II, USA100</td><td colspan="1" rowspan="1">S. delphini 49171</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1720</td><td colspan="1" rowspan="1">II, USA200</td><td colspan="1" rowspan="1">S. epidermidis 12228</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1750</td><td colspan="1" rowspan="1">II, USA200</td><td colspan="1" rowspan="1">S. epidermidis 35983</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1760</td><td colspan="1" rowspan="1">II, USA200</td><td colspan="1" rowspan="1">S. epidermidis 35984</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS651</td><td colspan="1" rowspan="1">II USA200</td><td colspan="1" rowspan="1">S. epidermidis 51625</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-39</td><td colspan="1" rowspan="1">III, Hungary</td><td colspan="1" rowspan="1">S. epidermidis 700562</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus 35592</td><td colspan="1" rowspan="1">III, ST239</td><td colspan="1" rowspan="1">S. epidermidis 700563</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1680</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700565</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1717</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700566</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS643</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700567</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS662</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700568</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS688</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700576</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS716</td><td colspan="1" rowspan="1">IV, USA300</td><td colspan="1" rowspan="1">S. epidermidis 700583</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1683</td><td colspan="1" rowspan="1">IV, USA400</td><td colspan="1" rowspan="1">S. equorum 43958</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1696</td><td colspan="1" rowspan="1">IV, USA400</td><td colspan="1" rowspan="1">S. felis 49163</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1707</td><td colspan="1" rowspan="1">IV, USA400</td><td colspan="1" rowspan="1">S. fleuretti BAA-274</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1752</td><td colspan="1" rowspan="1">IV, USA400</td><td colspan="1" rowspan="1">S. gallinarum 33539</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1684</td><td colspan="1" rowspan="1">IV, USA500</td><td colspan="1" rowspan="1">S. gallinarum 49148</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1689</td><td colspan="1" rowspan="1">IV, USA500</td><td colspan="1" rowspan="1">S. haemolyticus BAA-1693</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1763</td><td colspan="1" rowspan="1">IV, USA500</td><td colspan="1" rowspan="1">S. haemolyticus 43253</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS685</td><td colspan="1" rowspan="1">IV, USA500</td><td colspan="1" rowspan="1">S. haemolyticus 29968</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1754</td><td colspan="1" rowspan="1">IV, USA600</td><td colspan="1" rowspan="1">S. haemolyticus 29970</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1755</td><td colspan="1" rowspan="1">IV, USA700</td><td colspan="1" rowspan="1">S. haemolyticus 700564</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1758</td><td colspan="1" rowspan="1">IV, USA800</td><td colspan="1" rowspan="1">S. hominis 25615</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1768</td><td colspan="1" rowspan="1">IV, USA800</td><td colspan="1" rowspan="1">S. hominis 27844</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS692</td><td colspan="1" rowspan="1">IV, USA800</td><td colspan="1" rowspan="1">S. hominis 51624</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS675</td><td colspan="1" rowspan="1">IV, USA800</td><td colspan="1" rowspan="1">S. hominis 700586</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1747</td><td colspan="1" rowspan="1">IV, USA1000</td><td colspan="1" rowspan="1">S. hominis subsp.</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS483</td><td colspan="1" rowspan="1">IV, USA1000</td><td colspan="1" rowspan="1">S. ihtermedius 298637</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS730</td><td colspan="1" rowspan="1">IV, USA1000</td><td colspan="1" rowspan="1">S. intermedius 49052</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1764</td><td colspan="1" rowspan="1">IV, USA1100</td><td colspan="1" rowspan="1">S. intermedius 51874</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus NRS484</td><td colspan="1" rowspan="1">IV, USA1100</td><td colspan="1" rowspan="1">S. kloosii 43959</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1766</td><td colspan="1" rowspan="1">V, USA700</td><td colspan="1" rowspan="1">S. lentus 29070</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-2094</td><td colspan="1" rowspan="1">V, WA-MRSA</td><td colspan="1" rowspan="1">S. lugdunensis 4436</td><td colspan="1" rowspan="1">CCM, Czech</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-42</td><td colspan="1" rowspan="1">VI, USA800</td><td colspan="1" rowspan="1">S. lugdunensis 7990</td><td colspan="1" rowspan="1">ATCC, NCTC</td></tr><tr><td colspan="1" rowspan="1">S. aureus (mecC) BAA-2313</td><td colspan="1" rowspan="1">XI, CC130</td><td colspan="1" rowspan="1">S. lugdunensis 48413</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1751</td><td colspan="1" rowspan="1">Untyped, USA600</td><td colspan="1" rowspan="1">S. lugdunensis 49576</td><td colspan="1" rowspan="1">CCUG, Sweden</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1771</td><td colspan="1" rowspan="1">Untyped, USA800</td><td colspan="1" rowspan="1">S. lugdunensis 700328</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1718</td><td colspan="1" rowspan="1">NA, USA300</td><td colspan="1" rowspan="1">S. lutrae 700373</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1749</td><td colspan="1" rowspan="1">NA, USA900</td><td colspan="1" rowspan="1">S. massiliensis 7895</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1765</td><td colspan="1" rowspan="1">NA, USA1200</td><td colspan="1" rowspan="1">S. muscae 49912</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-40</td><td colspan="1" rowspan="1">Untyped, Lisbon</td><td colspan="1" rowspan="1">S. nepalensis 48992</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1685</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. pasteuri 51129</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-1708</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. pettenkoferi 36</td><td colspan="1" rowspan="1">Indianapolis, Denys Lab</td></tr><tr><td colspan="1" rowspan="1">S. aureus BAA-1721</td><td colspan="1" rowspan="1">Untyped, UK</td><td colspan="1" rowspan="1">S. piscifermentans 51183</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus6538</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. pulvereri 33938</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus11632</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. pseudintermidus 49444</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus12600</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. saccharolyticus 14953</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus13150</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. saprophyticus BAA-750</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus14775</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. saprophyticus 15305</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus14776</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. schleiferi subsp.</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus14993</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1"> 43808</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus25923</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. sciuri 29061</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus29213</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. sciuri 29060</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus29247</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. sciuri 700013</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus43300</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. sciuri subsp. carnaticus700058</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus 700699</td><td colspan="1" rowspan="1">Untyped, ATCC</td><td colspan="1" rowspan="1">S. sciuri subsp. rodentium700063</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BORSA MCW1</td><td colspan="1" rowspan="1">Untyped, Wisconsin, Ledeboer Lab</td><td colspan="1" rowspan="1">S. simulans 27841</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BORSA MCW2</td><td colspan="1" rowspan="1">Untyped, Wisconsin, Ledeboer Lab</td><td colspan="1" rowspan="1">S. simulans 27848</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BORSA 23737</td><td colspan="1" rowspan="1">Untyped, New Jersey, Kreiswirth Lab</td><td colspan="1" rowspan="1">S. simiae 7213</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus BORSA 23739</td><td colspan="1" rowspan="1">Untyped, New Jersey, Kreiswirth Lab</td><td colspan="1" rowspan="1">S. succinus subsp. succinus700337</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus Empty Mec Cassette</td><td colspan="1" rowspan="1">Untyped, Iowa, Diekema lab</td><td colspan="1" rowspan="1">S. vitulinus 51162</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus Empty Mec Cassette</td><td colspan="1" rowspan="1">Untyped, Iowa, Diekema lab</td><td colspan="1" rowspan="1">S. warneri 10209</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus Empt Mec Cassette</td><td colspan="1" rowspan="1">Untyped, Iowa, Diekema lab</td><td colspan="1" rowspan="1">S. warneri 25614</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. aureus Empty Mec Cassette4</td><td colspan="1" rowspan="1">Untyped, Iowa, Diekema lab</td><td colspan="1" rowspan="1">S. warneri 27836</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. auricularis 33751</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">S. warneri 49454</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. auricularis 33753</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">S. xylosus 35633</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. capitis subsp. capitis 35661</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">S. xylosus 49148</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">S. capitis subsp. ureolyticus</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

In addition to these studies, a subset of (8) S. aureus strains representing SCCmecA

subtypes I-V, (1) mecC strain, (4) Borderline Oxacillin Resistant S. aureus (BORSA),   
(4)   
Empty Cassette S. aureus variants, and multiple S. epidermidis and S. lugdunensis   
strains   
were selected to be part of a “Challenge Panel”. The challenge panel was tested for   
oxacillin MIC using BD Phoenix ID. The results from the MIC determination and card   
results are listed in in the table below:

Analytical Reactivity (Inclusivity) Challenge Panel: Staphylococcus strains tested for oxacillin MIC and for inclusivity by the Staph ID/R Blood Culture Panel.   

<table><tr><td colspan="1" rowspan="1">Staphylococcusspecies</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">SCCmecType</td><td colspan="1" rowspan="1">PFGE/TypeStrain</td><td colspan="1" rowspan="1">Significance</td><td colspan="1" rowspan="1">OxacillinMIC (μg/mL)</td><td colspan="1" rowspan="1">Staph ID/R Blood CulturePanel Result</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-38</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">700699</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">Genome sequenced</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-1682</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-1681</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">33592</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">ST239</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-1680</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-1684</td><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">USA500</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">&gt;2</td><td colspan="1" rowspan="1">S. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-2094</td><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">WA-MRSA</td><td colspan="1" rowspan="1">MRSA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">. aureus, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">BAA-2313</td><td colspan="1" rowspan="1">XI (mecC )</td><td colspan="1" rowspan="1">CC130</td><td colspan="1" rowspan="1">mecC - MRSA</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">20723.046</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Empty Cassette</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">20723.051</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Empty Cassette</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">20723.045</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Empty Cassette</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">20723.050</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Empty Cassette</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">23739</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">23737</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">MCW1</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">MCW2</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">12600</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">serotype 3</td><td colspan="1" rowspan="1">MSSA</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">14993</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSA</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">11632</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSA</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">6538</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSA</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">25923</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSA</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. aureus, mecA Abscent</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">35984</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Genome sequenced</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">35983</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700562</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700565</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700567</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700566</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700576</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">51625</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">12228</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSE</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Absent</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">700583</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MSSE</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Absent</td></tr><tr><td colspan="1" rowspan="1">S. hominis</td><td colspan="1" rowspan="1">700586</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MR Staph ssp</td><td colspan="1" rowspan="1">&gt;1</td><td colspan="1" rowspan="1">Staphylococcus speciesOTHER than S. aureus or S.lugdunensis, mecA Present</td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">49576</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MS - Lug</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. lugdunensis mecA Absent</td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">700328</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MS - Lug</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. lugdunensis mecA Absent</td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">NCTC 7990</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MS - Lug</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. lugdunensis mecA Absent</td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">43809</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">MS - Lug</td><td colspan="1" rowspan="1">≤0.25</td><td colspan="1" rowspan="1">S. lugdunensis mecA Absent</td></tr></table>

All strains showed expected oxacillin MIC results, including the BORSA strains, which showed a range of oxacillin resistance $( 0 . 2 5 { - } 2 \mu \mathrm { g / m L } )$ as expected for strains resistant by alternative mechanisms other than mecA. In addition, all empty cassette strains, which lack mecA, were sensitive to oxacillin $( 0 . 5 { \mathrm { - } } 0 . 2 5 ~ \mu \mathrm { g / m L } )$ . Samples were correctly detected as mecA Present or Absent in the Staph ID/R Blood Culture Panel. The Staph ID/R did not detect mecA for mecC strain BAA-2313, empty cassette and BORSA strains as expected.

# c. Analytical Specificity (Exclusivity):

Studies were performed to assess the potential cross-reactivity of the Staph ID/R Blood Culture Panel with 116 off-panel microflora (bacterial, yeast and mollicute strains). BACTEC Plus Aerobic/F or Anaerobic/F media (for anaerobic strains) containing negative blood were inoculated with isolates and incubated in a BACTEC Blood Culture System until alarm positivity. Alarm positive samples were incubated for additional time in the Blood Culture System consistent with specimen stability studies to obtain a target microorganism bottle load $\geq 1 0 ^ { 8 } \mathrm { C F U / m L }$ . The positive alarm bottles were Gram stained, diluted, plated and counted to confirm all organisms were tested at $1 \mathrm { { x } 1 0 ^ { 8 } \mathrm { { C F U / m L } } }$ or higher. Alarm positive blood cultures were not obtained for a subset of organisms. For 2 organisms, alarm positive bottles samples were substituted with genomic DNA at a final concentration of ${ \geq } 1 0 ^ { 8 }$ copies/mL. Genomic DNA was spiked into a matrix of negative blood and BACTEC Plus Aerobic/F media.

A minimum of 2 replicates was tested in the Staph ID/R Blood Culture Panel for each of the bacterial and fungal strains evaluated and these data are summarized in the table below.

Analytical Specificity (Exclusivity) Panel: Non-Staphylococcus microorganisms or DNA from micro-organisms tested for exclusivity by the Staph ID/R Blood Culture Panel   

<table><tr><td>Exclusivity Species</td><td>Strain (ATT, CCUG,</td><td>Exclusivity Species</td><td>Strain (ATT, CCUG, Clinical)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Clinical)</td><td></td><td></td></tr><tr><td colspan="4" rowspan="1">Gram Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus (1)</td><td colspan="1" rowspan="1">17929</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">15313</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">49176</td><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">35967</td></tr><tr><td colspan="1" rowspan="1">Aerococcus urinae</td><td colspan="1" rowspan="1">51268</td><td colspan="1" rowspan="1">Macrococcus caseolyticus (1,2)</td><td colspan="1" rowspan="1">13548</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum (2)</td><td colspan="1" rowspan="1">BAA-1784</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">10240</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">14579</td><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">27567</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">11051</td><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">700898</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">43734</td><td colspan="1" rowspan="1">Pediococcus damnosus</td><td colspan="1" rowspan="1">29358</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">14025</td><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">33316</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">700327</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">27337</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">6056</td><td colspan="1" rowspan="1">Planococcus citreus</td><td colspan="1" rowspan="1">14404</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">29212</td><td colspan="1" rowspan="1">Planococcus kocurii</td><td colspan="1" rowspan="1">43650</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">19433</td><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">11827</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis, van A</td><td colspan="1" rowspan="1">1MC</td><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">6939</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis, van B</td><td colspan="1" rowspan="1">51575</td><td colspan="1" rowspan="1">Rothia dentocariosa</td><td colspan="1" rowspan="1">BAA-907</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">19434</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">49040</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">6057</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">BAA-611</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium, van A</td><td colspan="1" rowspan="1">700221</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">13813</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarium</td><td colspan="1" rowspan="1">700425</td><td colspan="1" rowspan="1">Streptococcus angunosis</td><td colspan="1" rowspan="1">NCTC 10713</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarium</td><td colspan="1" rowspan="1">49573</td><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">27823</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">8043</td><td colspan="1" rowspan="1">Streptococcus dysagalactiae</td><td colspan="1" rowspan="1">43078</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">49464</td><td colspan="1" rowspan="1">Streptococcus equi</td><td colspan="1" rowspan="1">9528</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">27824</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">9809</td></tr><tr><td colspan="1" rowspan="1">Globicatella sanguinis</td><td colspan="1" rowspan="1">51174</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">49475</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">BAA-752</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">6249</td></tr><tr><td colspan="1" rowspan="1">Kocuria rosea</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">25175</td></tr><tr><td colspan="1" rowspan="1">Kytococcus schroeteri</td><td colspan="1" rowspan="1">BAA-2410</td><td colspan="1" rowspan="1">Streptococcus mutans (2,3)</td><td colspan="1" rowspan="1">35668</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">4356</td><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">15909</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">11454</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ARUP</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">40932</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">49399</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroidessubsp. mesenteroides</td><td colspan="1" rowspan="1">8293</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">12344</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroidessubsp. mesenteroides</td><td colspan="1" rowspan="1">19254</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">4543</td></tr><tr><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">19120</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">10556</td></tr><tr><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">33090</td><td colspan="1" rowspan="1">Streptococcus thoraltensis</td><td colspan="1" rowspan="1">700865</td></tr><tr><td colspan="1" rowspan="1">Listeria ivanovii</td><td colspan="1" rowspan="1">19119</td><td colspan="1" rowspan="1">Streptococcus uberis (1)</td><td colspan="1" rowspan="1">9927</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">19606</td><td colspan="1" rowspan="1">Kluyvera intermedia (1)</td><td colspan="1" rowspan="1">33421</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">23055</td><td colspan="1" rowspan="1">Moraxella catarrhalis (1)</td><td colspan="1" rowspan="1">23246</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">19002</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">25829</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">17925</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">19424</td></tr><tr><td colspan="1" rowspan="1">Bacteriodes fragilis</td><td colspan="1" rowspan="1">23745</td><td colspan="1" rowspan="1">Neisseria meningitides (1)</td><td colspan="1" rowspan="1">13077</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">9797</td><td colspan="1" rowspan="1">Neisseria subflava (1)</td><td colspan="1" rowspan="1">49275</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">25416</td><td colspan="1" rowspan="1">Oligella urethralis</td><td colspan="1" rowspan="1">17960</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">25405</td><td colspan="1" rowspan="1">Proteus mirabilis (1,4)</td><td colspan="1" rowspan="1">25933</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">8090</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">6896</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">27156</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">9250</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">15038</td><td colspan="1" rowspan="1">Providencia rustigianii</td><td colspan="1" rowspan="1">13159</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">13047</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">10145</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (1)</td><td colspan="1" rowspan="1">BAA-199</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">49128</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">4157</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">14028</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">25586</td><td colspan="1" rowspan="1">Salmonella typhimurium (1)</td><td colspan="1" rowspan="1">13311</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">33390</td><td colspan="1" rowspan="1">Serratia liquefaciens (2,5)</td><td colspan="1" rowspan="1">27592</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">13337</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">13880</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">13182</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">29930</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">700603</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">13637</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">BAA-1705</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">9610</td></tr><tr><td colspan="3" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">18804</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">14054</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">66032</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ARUP 2</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">24210</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">90112</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">Mollicutes</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumonia (6)</td><td colspan="1" rowspan="1">15531</td><td></td><td></td></tr></table>

(1) (2) (3) (4) This set of test runs also contained 1 “invalid” run This set of test runs contained 1 false positive result This set of test runs also contained 10 “invalid” runs This set of test runs contained 2 false positive results (5) This set of test runs also contained 5 “invalid” runs (6) This set of test runs also contained 2 “invalid” runs

The vast majority of strains tested ‘Staphylococcus NEGATIVE,’ indicating no crossreactivity or interference with internal controls. The exceptions were 27 ‘invalid’ calls and 6 ‘Staphylococcus POSITIVE’ calls, all noted in the table above.

One ‘invalid’ call out of 2 tests was observed for a single strain of the following species: Actinomyces odontolyticus, Escherichia coli, Kluyvera intermedia, Moraxella catarrhalis, Neisseria meningitides, Neisseria subflava, Proteus mirabilis, Salmonella typhimurium, and Streptococcus uberis. Each invalid case resolved upon re-testing as ‘Staphylococcus NEGATIVE’.

Two ‘invalid’ calls out of 2 tests were observed for Mycoplasma pneumoniae upon initial testing. Two valid calls out of 2 tests were obtained upon re-testing as ‘Staphylococcus NEGATIVE’.

One ‘Staphylococcus POSITIVE’ call out of 2 tests was observed for a single strain of the following species: Arcanobacterium haemolyticum, Macrococcus caseolyticus, Streptococcus mutans, Proteus mirabilis, Serratia liquefaciens. Each positive result resolved upon re-testing as ‘Staphylococcus NEGATIVE’ with a minimum of 6 repeat tests, indicating the positive results previously obtained were likely a single contamination event in one card. One or more invalid calls were observed upon re-testing for the following species: Macrococcus caseolyticus, Streptococcus mutans and Serratia liquefaciens.

# d. Microbial Interference:

Off-Panel Microbial Interference: As a follow up to the previous exclusivity and inclusivity studies, the Staph ID/R Blood Culture Panel was further evaluated for the ability to detect low level Staphylococcus species in the presence of 14 “off-panel” microorganism strains that should not be detected. The “off-panel” strains represent Gram Positive, Gram Negative, Yeast and likely skin contaminants. BACTEC Plus Aerobic/F or Anaerobic/F Bottles containing blood were inoculated with competing “off-panel” microorganisms. The “off-panel” strains were grown to high concentrations by incubating 8 hours past bottle ring ‘On-board’, consistent with incubation time and temperatures tested in the specimen stability. The bottle contents were confirmed by Gram stain and serial dilutions plated on agar and counted the following day to confirm a concentration of ${ > } 1 0 ^ { 8 } \mathrm { C F U / m L }$ . Bottles were stored at $4 \mathrm { { } ^ { \circ } C }$ and tested within 72 hours in combination with Staphylococcus TSB cultures at approximately $1 { - } 2 . 5 { \mathrm { x } } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for each strain. The concentration of the Staphylococcus strains were verified by plating serial dilutions on agar and performing colony counts the following day.

For the ‘valid’ runs tested, the potentially interfering ‘off-panel’ microorganisms did not interfere with the detection of the Staphylococcus strains, resulting in ‘POSITIVE’ calls as expected. In some cases, a miscall was observed, and the low-target Staphylococcus strains were re-tested at a higher concentration and resulted in a positive result as expected. The re-tested concentrations are included in the table and were within the 2-3X LoD range for each species. Results are shown in the table below:

Microbial Interference Panel (Off-panel): Non-Staphylococcus microbial strains tested for

microbial interference in detecting 5 Staphylococcus strains by the Staph ID/R Blood Culture Panel.

<table><tr><td colspan="1" rowspan="2">"Off-Panel" Microorganisms Species,Sample Input ≥ 108 CFU/mL;TCC/NCTC strain #</td><td colspan="5" rowspan="1">Species; ATCC Strain #; Sample Input (CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">S. aureusBAA-16820.7-1.2x106</td><td colspan="1" rowspan="1">S. aureus116320.6-1.6x106</td><td colspan="1" rowspan="1">S.epidermidis7005620.8-0.9x106</td><td colspan="1" rowspan="1">S.epidermidis7005830.9-2.2x106</td><td colspan="1" rowspan="1">S.lugdunensis495761-1.2x106</td></tr><tr><td colspan="1" rowspan="1">Gram Positive Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium 43734</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis 19433</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium 19434</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes 19115</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus 10240</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnes 11827</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptoccocus agalactiae 13813</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2*</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus 10713</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae 27336</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes 49399</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli 4157</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2"</td><td colspan="1" rowspan="1">2/2*</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae 700603</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa 10145</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans 18804</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">2/2</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run "This set of test runs initially miscalled, but called correctly with a higher $\mathrm { C F U / m L }$ input of the low level

Staphylococcus Microbial Interference: 12 Staphylococcus species expected to be codetected with the low level Staphylococcus species. BACTEC Plus Aerobic/F bottles containing blood were inoculated with Staphylococcus isolates. The bacteria were grown to high concentrations by incubating 8 hours past bottle ring ‘On-board’, consistent with incubation time and temperatures tested in the specimen stability study. The bottle contents were confirmed by Gram stain and serial dilutions plated on agar and counted the following day to confirm a concentration of ${ > } 1 0 ^ { 8 } \mathrm { C F U / m L }$ for competing Staphylococcus strains. Bottles were stored at $4 \mathrm { { } ^ { \circ } C }$ and tested within 72 hours in combination with Staphylococcus TSB cultures at approximately $1 { - } 2 . 5 { \mathrm { x } } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for each strain. The concentration of the Staphylococcus strains were verified by plating serial dilutions on agar and performing colony counts the following day.

The studies assessed the detection of 5 ATCC Staphylococcus test strains: S. aureus $( m e c A { + } )$ BAA-1682 $( 0 . 7 { - } 1 . 2 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } ) S$ . aureus (mecA-) 11632 $( 0 . 6 \mathrm { - } 1 . 6 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ ), $S .$ . epidermidis (mecA+) 700562 (0.2- $\phantom { 0 } { . 9 \mathrm { x } 1 0 } ^ { 6 }$ CFU/mL), S. epidermidis (mecA-) strain 700583 $( 0 . 9 { - } 2 . 2 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ , and $S .$ . lugdunensis (mecA-) strain 49576 (1 $- 1 . 2 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ ). A minimum of 2 replicate Staph ID/R Blood Culture Panels were performed for each combination of competing and test organisms.

Staphylococcus interference was observed for S. aureus with $S .$ . epidermidis and S. caprae at initial concentrations tested $( 5 . 9 \mathrm { { x } 1 0 ^ { 5 } \mathrm { { C F U } / m L ) } }$ , but the interference was resolved upon re-testing at higher concentrations $( 1 . 5 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ , within 2-3X LoD). There were 13 cases of interference with S. epidermidis at initial concentrations $( 2 . 2 – 2 . 5 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L } )$ when tested against $S .$ . aureus, $S .$ epidermidis, S. lugdunensis, S. capitis, S. hominis, $S .$ . haemolyticus, and S. simulans. Higher concentrations of S. epidermidis resolved the interference $( 8 . 5 { \cdot } 9 . 4 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , within 2-3X LoD). Two cases of interference were observed with $S .$ lugdunensis: one case with $S .$ epidermidis, one case with S. hominis. Both cases resolved at higher concentrations of $S .$ . lugdunensis $( 1 . 2 \mathrm { x } 1 0 ^ { 6 }$ , within 2-3X LoD). Results are shown in the table below:

# Microbial Interference Panel (Staphylococcus): Staphylococcus microbial strains tested for microbial interference in detecting 5 different low level Staphylococcus strains by the Staph ID/R Blood Culture Panel.

<table><tr><td rowspan=2 colspan=1>Microbial InterferenceStaphylococcus species,Sample Input ≥ 108CFU/mL; ATCC,Clinical #</td><td rowspan=1 colspan=5>Species; ATCC Strain #; Sample Input (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>S. aureusBAA-16820.7-1.2x106</td><td rowspan=1 colspan=1>S. aureus116320.6-1.6x106</td><td rowspan=1 colspan=1>S. epidermidis7005620.2-0.9x106</td><td rowspan=1 colspan=1>S. epidermidis S. lugdunensis7005830.9-2.2x106</td><td rowspan=1 colspan=1>S. epidermidis S. lugdunensis495761-1.2x106</td></tr><tr><td rowspan=1 colspan=1>S. aureus BAA-1682, mecA+</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. aureus 11632, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis 700562, mecA +</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis 700583, mecA-</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>2/2&quot;</td></tr><tr><td rowspan=1 colspan=1>S. lugdunensis 49576, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>S. capitis 35661, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. caprae 35538, mecA -</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>1/2*</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. hominis 27844, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus BAA-1693, mecA +</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2*</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. pettenkoferii Denys 38, mecA +</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. simulans 27848, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2&quot;</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>S. warneri 27830, mecA -</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run "This set of test runs initially miscalled, but called correctly with a higher $\mathrm { C F U / m L }$ input of the low level target

# e. Interfering Substances:

The Staph ID/R Blood Culture Panel was evaluated for interference by a panel of 16 different substances. Substances were spiked into BACTEC Plus Aerobic/F (with resin) or Standard Aerobic/F (without resin) media containing negative blood incubated 24 hours in a BACTEC Blood Culture Device. Target Staphylococcus cells were combined with the substances at low positive concentrations at approximately 2-3X LoD ( $\mathrm { 1 - 2 . 5 x 1 0 ^ { 6 } }$ CFU/mL). The CFU concentrations for each strain were estimated by optical density measurements and then confirmed by colony counting. The studies assessed the detection of the same 10 Staphylococcus ATCC strains used for analytical sensitivity and microbial interference: S. aureus, mecA $^ { 1 + }$ strains BAA-1680 and BAA-1682, S. aureus, mecA+ strains 11632 and 6538, S. epidermidis, mecA+ strains 700562 and 51625, S. epidermidis, mecA- strains 700583 and 12228, and S. lugdunensis, mecA- strains 49576 and 43809. ATCC strain, $E .$ . faecalis, mecA- 29212, an off-panel “Negative” in the Staph ID/R Blood Culture Panel, was also included in the study to assess chemical interference with the sample processing control and all downstream detection steps. A minimum of two replicate assays were performed for each Staphylococcus strain using each substance in a background of BACTEC Plus Aerobic bottles (with resin) or Standard Aerobic bottles (without resin), see tables below.

# Interfering Substances Panel (BACTEC Plus with Resin)

<table><tr><td rowspan=3 colspan=1>Substance InputConcentrationinto Plus AerobicMedia (with resin)</td><td rowspan=1 colspan=11>Species, ATCC strain, Sample Input (CFU/mL)</td></tr><tr><td rowspan=1 colspan=4>S. aureus, mecA +     S. aureus, mecA-</td><td rowspan=1 colspan=2>S. lugdunensis, mecA -</td><td rowspan=1 colspan=2>S. epidermidis, mecA +</td><td rowspan=1 colspan=2>S. epidermidis, mecA-</td><td rowspan=1 colspan=1>E. faecalis (Neg)</td></tr><tr><td rowspan=1 colspan=1>BAA-16800.9-1.8x106</td><td rowspan=1 colspan=2>BAA-1682116321.4-1.6x1061.2-2.2x106</td><td rowspan=1 colspan=1>65381.8-2.2x106</td><td rowspan=1 colspan=1>438091.1-1.7x10</td><td rowspan=1 colspan=1>495760.9-1.8x106</td><td rowspan=1 colspan=1>516250.3-1.5x106</td><td rowspan=1 colspan=1>7005620.3-0.8x106</td><td rowspan=1 colspan=1>122280.9-1.2x106</td><td rowspan=1 colspan=1>7005830.3-1.9x106</td><td rowspan=1 colspan=1>292122.4x107</td></tr><tr><td rowspan=1 colspan=1>Whole Blood in ACD(≥35% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inEDTA(&gt;40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inHeparin (≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inSodium Citrate(≥35% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Human Plasma(≥40%, v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>SodiumPolyanetholsulfonate(≥0.20% w/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin(10 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>γ-Globulin(≥40 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Triglycerides(1mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2*</td></tr><tr><td rowspan=1 colspan=1>White Blood Cells(≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Platelets(≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>UnconjugatedBilirubin (0.075mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Conjugated Bilirubin(≥0.075 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Vancomycin(≥50 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin(≥7.5 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Cefoxitin(≥125 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr></table>

# Interfering Substances Panel (BACTEC Standard without Resin)

<table><tr><td rowspan=3 colspan=1>Substance InputConcentration intoStandard AerobicMedia (withoutresin)</td><td rowspan=1 colspan=11>Species, ATCC strain, Sample Input (CFU/mL)</td></tr><tr><td rowspan=1 colspan=2>S. aureus, mecA +</td><td rowspan=1 colspan=2>S. aureus, mecA-</td><td rowspan=1 colspan=2>S. lugdunensis, mecA -</td><td rowspan=1 colspan=2>S. epidermidis, mecA +</td><td rowspan=1 colspan=2>S. epidermidis, mecA-</td><td rowspan=1 colspan=1>E. faecalis (Neg)</td></tr><tr><td rowspan=1 colspan=1>BAA-16800.9-1.8x106</td><td rowspan=1 colspan=1>BAA-16821.4-1.6x106</td><td rowspan=1 colspan=1>11632[1.2-2.2x106</td><td rowspan=1 colspan=1>65381.8-2.2x106</td><td rowspan=1 colspan=1>43809[1.1-1.7x106</td><td rowspan=1 colspan=1>495760.9-1.8x106</td><td rowspan=1 colspan=1>516250.3-1.5x106</td><td rowspan=1 colspan=1>7005620.3-0.8x106</td><td rowspan=1 colspan=1>122280.9-1.2x106</td><td rowspan=1 colspan=1>7005830.3-1.9x106</td><td rowspan=1 colspan=1>292122.4x107</td></tr><tr><td rowspan=1 colspan=1>Whole Blood in ACD(≥35% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inEDTA(≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inHeparin (≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Whole Blood inSodium Citrate (35%v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Human Plasma(≥40%, v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>SodiumPolyanetholsulfonate(≥0.20% w/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin(≥10 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>γ-Globulin(≥40 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Triglycerides(≥10 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>White Blood Cells(≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Platelets(≥40% v/v)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>UnconjugatedBilirubin (0.075mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Conjugated Bilirubin(≥0.075 mg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2*</td></tr><tr><td rowspan=1 colspan=1>Vancomycin(≥50 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin(≥7.5 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Cefoxitin(≥125 μg/mL)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run

Testing of specimen with the above described potential interfering substances produced the expected positive and negative results indicating that none of the

endogenous or exogenous test substances compete or interfere with obtaining accurate test results with the Staph ID/R Blood Culture Panel.

# f. Carry-over/Cross-Contamination Study:

A study was performed to assess the cross-contamination of the Staph ID/R Blood Culture Panel by alternatively testing high titer S. aureus, mecA $^ +$ ATCC strain BAA1682 and off-target negative $E .$ . faecalis ATCC strain 29212. BACTEC Plus Aerobic/F blood bottle containing negative blood were inoculated with strain isolates and incubated until alarm positivity in a BACTEC Blood Culture System. Alarm positive samples were incubated past positivity consistent with timeframes used during the specimen stability studies to obtain a high titer. Alarm positive blood cultures were Gram stained, diluted, plated on agar plates and colonies were counted the following day to confirm target concentrations ${ \dot { > } } 1 0 ^ { 7 }$ for $S .$ aureus $( 3 . 0 { \mathrm { x } } 1 0 ^ { 7 } \mathrm { C F U / m L } )$ and ${ > } 1 0 ^ { \ \overline { { 8 } } } \ E$ . faecalis $( 5 . 8 \mathrm { x } 1 0 ^ { 9 } \dot { \mathrm { C F U / m L } } )$ ). Aliquots from the high titer blood culture bottles were stored at $- 2 0 ^ { \circ } \mathrm { C }$ until testing. Carry-over/cross-contamination was tested by running a series of alternating runs of high titer positive and negative samples on multiple Portrait Analyzers.

No false positive results were observed during consecutive testing of high positive samples alternating with negative samples, demonstrating that recommended sample handling and testing practices are effective in preventing false positive results due to carryover or cross-contamination between samples.

# g. Reproducibility:

A multicenter, blinded, reproducibility study was performed to determine reproducibility of the Staph ID/R Blood Culture Panel. Testing occurred at three sites using a panel of seven simulated blood culture specimens, each spiked with a single organism. Specimens were prepared in a matrix of whole blood and blood culture media. Half of the replicates for the three Staphylococcus positive samples were consistent with the level of organism present at the time of positivity (low) and half were at a concentration similar to that observed after 8 hours of positivity (high). For the one off-panel organism (E. faecalis; Staph ID/R negative) the concentration was “high.”

The study incorporated several variables including six different operators at three sites (two operators/site), five different cartridge lots, and 89 different Portrait Analyzers (15 at site 1, and 12 at site 3, 62 at site 4). Over the course of 10 weeks, samples were tested on 12 different days, for a total of 90 replicates per analyte per concentration.

Valid results were attained for 630 of 642 $( 9 8 . 1 \% )$ runs. For the detection of Staphylococcus positivity (Test result “Positive”), expected positive results were obtained for 540/540 runs $( 1 0 0 \% )$ , and expected Staphylococcus negative results (Test result “Negative”) were obtained in 87/90 runs $( 9 6 . 7 \% )$ . For the detection of specific Staphylococcus species (S. aureus, $S .$ epidermidis, and S. lugdunensis), expected

positive results were obtained for 534/540 runs $( 9 8 . 8 \% )$ and expected negative results were obtained for 1345/1350 $( 9 9 . 6 \% )$ results.

A summary of results (percent agreement with the expected result) for each analyte is provided in the following tables:

Summary of Reproducibility Study

<table><tr><td rowspan=1 colspan=1>Staph ID/RBlood CulturePanel Result</td><td rowspan=1 colspan=1>Species, Bacteria Load (Low or High),Sample Input (CFU/mL)</td><td rowspan=1 colspan=1>Test Site</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td></tr><tr><td rowspan=12 colspan=1>S. aureus,mecA present</td><td rowspan=4 colspan=1>S. aureus (mecA +), Low;4.0x106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>S. aureus (mecA +) High;4.0x107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/150 (1)</td><td rowspan=1 colspan=1>149/150</td><td rowspan=4 colspan=1>448/450; 99.6%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1/150 (2)</td><td rowspan=1 colspan=1>149/150</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2/450</td><td rowspan=1 colspan=1>448/450</td></tr><tr><td rowspan=12 colspan=1>Staphylococcusspecies OTHERthan S. aureus orS. lugdunensis,mecA present</td><td rowspan=4 colspan=1>S. epidermidis (mecA +), Low;8.5x106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>S. epidermidis (mecA +), High;7.0x107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2/150 (3)</td><td rowspan=1 colspan=1>148/150</td><td rowspan=4 colspan=1>446/450; 99.1%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2/150 (4)</td><td rowspan=1 colspan=1>148/150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4/450</td><td rowspan=1 colspan=1>446/450</td></tr><tr><td rowspan=12 colspan=1>S. lugdunensis,mecA absent</td><td rowspan=4 colspan=1>S. lugdunensis (mecA - ), Low;6.0x107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>28/30 (3)</td><td rowspan=1 colspan=1>2/30</td><td rowspan=4 colspan=1>88/90; 97.8%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>2/90</td></tr><tr><td rowspan=4 colspan=1>S. Iugdunensis (mecA - ), High;5.1x108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2/150 (5,6)</td><td rowspan=1 colspan=1>148/150</td><td rowspan=4 colspan=1>447/450; 99.3%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1/150 (5)</td><td rowspan=1 colspan=1>149/150</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3/450</td><td rowspan=1 colspan=1>447/450</td></tr><tr><td rowspan=4 colspan=1>StaphylococcusPositive</td><td rowspan=4 colspan=1>S. aureus , S. epidermidis , S. lugdunensis,Low and High</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=4 colspan=1>540/540; 100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>0/540</td></tr><tr><td rowspan=4 colspan=1>StaphylococcusNegative</td><td rowspan=4 colspan=1>E. faecalis, (mecA - ), High;1.1x10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/30 (6)</td><td rowspan=1 colspan=1>29/30</td><td rowspan=4 colspan=1>87/90; 96.7%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2/30 (4)</td><td rowspan=1 colspan=1>28/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3/90</td><td rowspan=1 colspan=1>87/90</td></tr></table>

(1) Sample detected as "Staphylococcus aureus in mixed Staph infection (NOT S. lugdunensis )" (2) One S. lugdunensis specimen additionally detected S. aureus (3) Two S. lugdunensis specimens detected as Staphylococcus OTHER than S. aureus or S. lugdunensis (4) Two E. faecalis specimens detected as Staphylococcus OTHER than S. aureus or S. lugdunensis (5) One specimen detected S. aureus correctly, but additionally detected S. lugdunensis (6) One E. faecalis specimen detected as S. lugdunensis

Summary of mecA results from Reproducibility Studies   

<table><tr><td rowspan=1 colspan=1>Staph ID/RBlood CulturePanel Result</td><td rowspan=1 colspan=1>Species, Bacteria Load (Low orHigh), Sample Input (CFU/mL)</td><td rowspan=1 colspan=1>Test Site</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td></tr><tr><td rowspan=12 colspan=1>S. aureus, mecApresent</td><td rowspan=4 colspan=1>S. aureus (mecA+), Low;4.0x106</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>S. aureus (mec A+) High;4.0x107</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=12 colspan=1>Staphylococcusspecies OTHERthan S. aureus orS. lugdunensis,mecA present</td><td rowspan=4 colspan=1>S. epidermidis (mec A+), Low;8.5x106</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=4 colspan=1>90/90; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=4 colspan=1>S. epidermidis (mec A+), High;7.0x107</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>29/30 (1)</td><td rowspan=1 colspan=1>1/30</td><td rowspan=4 colspan=1>89/90; 98.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>1/90</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td><td rowspan=4 colspan=1>450/450; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>0/150</td><td rowspan=1 colspan=1>150/150</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0/450</td><td rowspan=1 colspan=1>450/450</td></tr><tr><td rowspan=4 colspan=1>mecA Present;no organismassociated</td><td rowspan=4 colspan=1>S. aureus (mec A+),S. epidermidis (mecA +)Low and High</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>119/120</td><td rowspan=1 colspan=1>1/120</td><td rowspan=4 colspan=1>359/360; 99.7%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>0/120</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>0/120</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>359/360</td><td rowspan=1 colspan=1>1/360</td></tr><tr><td rowspan=4 colspan=1>mecA Absent; noorganismassociated</td><td rowspan=4 colspan=1>S. lugdunensis (mecA - ),E. faecalis (mecA -)Low and High</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td><td rowspan=4 colspan=1>270/270; 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0/270</td><td rowspan=1 colspan=1>270/270</td></tr><tr><td rowspan=1 colspan=2>(1) One Stanhyle           ocies.OTHER the  C</td><td rowspan=1 colspan=4>aec4absont</td></tr></table>

e Staphyloco spec OTHER thanS.ures rS.ugdunens, me present rpor  me absnt

For detection of the mecA gene with no associated organism, positive result (mecA Present) were detected in 359/360 $( 9 9 . 7 \% )$ runs and negative results (mecA absent) were detected in 270/270 $( 1 0 0 . 0 \% )$ runs. For $S .$ . aureus $( m e c \mathbf { A } { + } )$ , positive mecA results were obtained for 180/180 $( 1 0 0 . 0 \% )$ runs and negative mecA results were obtained for 450/450 $( 1 0 0 . 0 \% )$ runs. For $S .$ . epidermidis $( m e c \mathbf { A } { + } )$ , positive mecA results were obtained for 179/180 $( 9 9 . 4 \% )$ runs and negative mecA results were obtained for 450/450 $( 1 0 0 . 0 \% )$ runs.

Summary of discrepant results from Reproducibility Studies   

<table><tr><td rowspan=1 colspan=1>Correct Result</td><td rowspan=1 colspan=1>Staph ID/R Blood Culture Panel Result</td><td rowspan=1 colspan=1>Discre pantResult</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>TestDay</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staph. species</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staph. species</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative, mecA N/A</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis, mecA absent</td><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus, mecA present</td><td rowspan=1 colspan=1>Staphylococcus aureus;Staphylococcus lugdunensis,mecA present</td><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>mecA absent</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staphylococcus aureus in mixed Staph infection(NOT S. lugdunensis ), mecA present</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus, mecA present</td><td rowspan=1 colspan=1>Staphylococcus aureus; Staphylococcus lugdunensis,mecA present</td><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staphylococcus aureus; Staphylococcus lugdunensis,mecA absent</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative, mecA N/A</td><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staph. species</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative, mecA N/A</td><td rowspan=1 colspan=1>Staphylococcus OTHER than S. aureus orS. lugdunensis, mecA absent</td><td rowspan=1 colspan=1>Staph. species</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr></table>

h. Linearity/assay reportable range:

Not applicable

i. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Controls: To ensure that the reported Portrait Analyzer result is correct, controls have been built into the system to minimize the risk of errors during assay processing. These controls are designed to monitor reagent performance and to indicate correct performance of each Test Cartridge. Each Test Cartridge has integrated control features that are automatically performed with every Staph ID/R Blood Culture Panel. The following internal controls are included:

Fiducial Controls: Establish orientation of the silicon chip for image analysis. Amplification Threshold Control (AC): The AC is used to threshold amplification such that potentially contaminating levels of Staphylococci are not amplified in the test. Two synthetic double-stranded DNA templates or plasmids are added to the test. One has homology with the primer regions specific for the tuf gene used in the Staph ID/R Blood Culture Panel, but with unique intervening sequence so that the amplified product is not recognized by probes on the chip surface. The shared primer binding site allows for the tuf gene specific primers to be used for both the amplification of Staphylococci present in a positive blood culture and the AC. The second AC has homology with the primer region for the nuc gene used in the Staph

ID/R Blood Culture Panel, but with unique intervening sequence and works similarly to the AC directed at the tuf gene. The ACs are lyophilized within a small chamber preceding the Extraction Chamber of the Staph ID/R Blood Culture Panel Test Cartridge. Upon sample loading the AC is rehydrated and during the PCR step is amplified. The ACs can compete with Staphylococcal genomic DNA for binding to the tuf and nuc gene primers, but has been optimized to do so only at levels below alarm positive blood cultures. At high Staphylococcal concentrations, the AC may be negative due to competitive inhibition of the amplification reagents. Note that the AC control is not detected by any probe(s) on the chip surface.

Sample Processing Control (SPC): The SPC controls for all analytical steps in the procedure, including DNA extraction from organisms present in the specimen, amplification of target DNA sequences, hybridization, and detection on the chip surface. The SPC is a strain of Bacillus subtilis cells that has been genetically engineered with an integration vector designed to contain shared sequence with the tuf gene primers used in the Staph ID/R Blood Culture Panel, but with unique intervening sequence. The shared primer binding site allows for the tuf gene specific primers to be used for both the amplification of Staphylococci present in a positive blood culture and the SPC. The SPC is lyophilized within a small chamber preceding the Extraction Chamber of the Staph ID/R Blood Culture Panel Test Cartridge. After sample loading the SPC is extracted, amplified and the unique intervening sequence is detected by a target-specific capture probe on the chip surface. At high staphylococcal concentrations, the SPC may be negative due to competitive inhibition of the amplification reagents.

Recommended External Controls: External controls are not provided with the Staph ID/R Blood Culture Panel, but are recommended in the package insert. External positive and negative controls are intended to monitor for correct procedural technique and reagent integrity. The External Positive Control is intended to monitor for substantial reagent failure. The External Negative Control is intended to confirm non-reactivity. External controls should be used in accordance with the appropriate accrediting organization requirements, as applicable.

External Controls used in clinical studies: External positive and negative controls were incorporated in the clinical trial to monitor for correct procedural technique and reagent integrity. The external positive controls were intended to monitor for substantial reagent failure. The external negative control using a nonStaphylococcus Gram-positive cocci (E. faecalis) was intended to confirm nonreactivity. Each species in the daily QC blood culture panel were grown to bottle positivity in 2 separate preparations and aliquoted into single use frozen aliquots for investigational use during the prospective and analytical studies.

Daily QC testing was performed by dividing the strains into 2 panels and each panel was tested on alternating days. Daily QC testing was performed every day a

prospective, reproducibility or low prevalence study sample was tested with a Staph ID/R Blood Culture Panel Test Cartridge. Valid, correct results were required from the daily QC panel to include the test sample data for that day. In cases with an invalid or erroneous result, repeat testing with a valid, correct result was required to include the test sample data for that day. The daily QC panel results from all prospective sites from prospective, reproducibility and low prevalence studies are listed in the table below:

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Site #1</td><td rowspan=1 colspan=1>Site #2</td><td rowspan=1 colspan=1>Site #3</td><td rowspan=1 colspan=1>Site #4</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>S. aureus, mecA+</td><td rowspan=1 colspan=1>32/33 (1)</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>71/72</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis, mecA-</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1>S. lugdunensis, mecA-</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>66/66</td></tr><tr><td rowspan=1 colspan=1>S.epidermidis, mecA +</td><td rowspan=1 colspan=1>29/29 (2)</td><td rowspan=1 colspan=1>19/19 (2)</td><td rowspan=1 colspan=1>16/16 (2)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>65/65</td></tr><tr><td rowspan=1 colspan=1>E. faecalis, mecA- (Negative)</td><td rowspan=1 colspan=1>61/62 (1,2)</td><td rowspan=1 colspan=1>40/41 (1,3)</td><td rowspan=1 colspan=1>32/32 (2)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>136/138</td></tr></table>

(1) This set of test run also contained 1 miscall (2) This set of test runs a l s o contained 1 "Invalid" run (3) This set of test runs also contained 3 “invalid” runs

Eight (8) invalid test runs and 3 miscalls for a total of 11 discrepant results, were obtained in the Daily QC Panel from all clinical sites during prospective, reproducibility and low prevalence studies.

# Specimen Stability:

A specimen stability study was conducted to support the claims of the sample storage on the bench for alarm positive blood culture bottles. Conditions include detection immediately after blood culture “bottle ring” and after incubation “Onboard” up to 8 hours post alarm on the BACTEC Blood Culture System $( 3 5 ^ { \circ } { - } 3 7 ^ { \circ } \mathrm { C } )$ . Additionally testing included, incubation once the bottle is removed from the blood culture system, termed “Off-board”, for up to 18 hours at room temperature ( $1 5 ^ { \circ } { \mathrm { . } }$ - $3 0 ^ { \circ } \mathrm { C } )$ , up to 72 hours at $2 ^ { \circ } { - } 8 ^ { \circ } \mathrm { C }$ and the combination of room temperature for up to 12 hours followed by up to 72 hours at $2 ^ { \circ } { - } 8 \mathsf { o C }$

For sample storage on the bench studies, 4 different Staphylococcus strains were tested: methicillin resistant $S .$ aureus ATCC BAA-1682, methicillin resistant $S .$ . epidermidis ATCC 700562, methicillin sensitive S. lugdunensis ATCC 49576, methicillin resistant S. haemolyticus BAA-1693. An off-panel strain (Staph ID/R Blood Culture Panel “Negative”), $E .$ . faecalis ATCC 29212, was also included in the study. All strains were tested in three bottles in triplicate at each time point for a total of 9 replicates per condition.

For preparation of these samples, the strain cultures were prepared in TSB broth, incubated overnight, and the CFU concentrations for each strain were estimated by optical density measurements. Three BACTEC Plus Aerobic/F blood bottles

containing negative blood were inoculated with each respective strain at approximately 30-100 CFU/bottle and incubated until alarm positivity in a BACTEC Blood Culture System. Alarm positive bottles were pulled at bottle ring or incubated on the Blood Culture System past positivity for 8 hours $- / + 0 . 5$ hour “On-board”. Samples were removed from each bottle and stored “Off-board” at the various temperatures for the times indicated in table below. All samples were Gram stained, serially diluted and plated for colony counts to determine the $\mathrm { C F U / m L }$ concentration at each time point and storage condition. Three tests were performed for each of 3 bottles, resulting in 9 total test runs for each condition. The data for each strain and time point are shown in the table below.

Specimen Stability Study Sample Input   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Off-board</td><td rowspan=1 colspan=5>Species tested with Correct Staph ID/R Blood Panel</td></tr><tr><td rowspan=1 colspan=1>On-board</td><td rowspan=1 colspan=1>Temperature</td><td rowspan=1 colspan=1>Time (hours)</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>S. haemolyticus</td></tr><tr><td rowspan=5 colspan=1>Bottle Ring</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>≤7</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>≤1</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>≤1</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>30, then 2-8</td><td rowspan=1 colspan=1>&lt;12, then &lt;72</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=5 colspan=1>Bottle Ring+ 8 Hours</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9**</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>≤7</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>≤1</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>≤1</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>30, then 2-8</td><td rowspan=1 colspan=1>&lt;12, then &lt;72</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run \*\*This set of test runs also contained 2 "Invalid" runs

The Staph ID/R Blood Culture Panel results for all the conditions and time points are in $100 \%$ concordance with the expected results. The stability studies resulted in 8 ‘invalid’ calls out of a total of 458 cards, for a rate of $2 \%$ , consistent with the prospective study invalid rate. At each time point, under the three storage conditions, S. aureus ATCC BAA-1682, S. epidermidis ATCC 700562, S. lugdunensis ATCC 49576, S. haemolyticus BAA-1693 and $E .$ . faecalis ATCC 29212 resulted in the expected ‘Positive’ or ‘Negative’ call, demonstrating sufficient stability of Staphylococcus in the BACTEC Plus Aerobic/F bottles in support of the claims of the sample stability for up to 18 hours at room temperature $( 1 5 ^ { \circ } { - } 3 0 ^ { \circ } \mathrm { C } )$ , 8 hours at $3 5 ^ { \circ }$ - $3 7 ^ { \circ } \mathrm { C }$ and 72 hours at $2 ^ { \circ } { - } 8 ^ { \circ } \mathrm { C }$ .

The study was also informative in determining the conditions that result in the highest and lowest $\mathrm { C F U / m L }$ bacterial load in each bottle based on colony counts. As expected, bottle ring resulted in the lowest bacterial load across all strains tested: $7 . 2 \mathrm { x } 1 0 ^ { 6 }$ – $7 . 7 { \times } 1 0 ^ { 8 } \ : \mathrm { { C F U / m L } }$ . Also as expected, 8 hours incubation past bottle ring on the BACTEC Blood Culture Device increased bacterial load levels $( 2 . 3 \mathrm { x } 1 \mathrm { \overline { { 0 } } ^ { 7 } } - 3 . 4 \mathrm { x } 1 0 ^ { 9 }$ $\mathrm { C F U / m L }$ ), with Bottle $\mathrm { R i n g } + 8$ hours “On-board” $+ 1 8$ hours $( 3 0 ^ { \circ } \mathrm { C } )$ “Off-board” the consensus highest bacterial load condition for all strains: $1 . 1 \mathrm { x } 1 0 ^ { 8 } - 3 . 1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U } / \mathrm { m L }$ .

$j$ . Evaluation of Blood Culture Bottle Types:

To determine the effect of blood culture bottle type on Staph ID/R Blood Culture Panel, 13 unique bottle types were tested in the presence of target and non-target organisms. Staphylococcus isolates or $E .$ . faecalis (negative) at bottle ring load levels $( 2 \mathrm { { x } 1 0 ^ { 6 } - 1 \mathrm { { x } 1 0 ^ { 8 } \dot { C F U } / m L ) } }$ consistent with the specimen stability studies. All bottles were tested with the highest volume of negative blood recommended by the manufacturer (e.g., if the recommended blood volume was $8 { \cdot } 1 0 \mathrm { m L }$ , $1 0 ~ \mathrm { m L }$ of blood was spiked into the bottle). Bottles with blood were pre-incubated 18 hours at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ in a BACTEC Blood Culture System or in a shaking incubator prior to testing. Bacteria isolates were incubated ${ > } 1 8$ hrs in TSB, and the CFU concentrations for each strain were estimated by optical density measurements, confirmed by serial dilution and colony counting.

The studies assessed the detection of 7 Staphylococcus ATCC strains used for analytical sensitivity: S. aureus, mecA $^ +$ BAA-1680, S. aureus, mecA $^ +$ 1682, S. aureus, mecA- 11632, S. epidermidis, mecA $^ +$ 51625, S. epidermidis, mecA- 12228, S. lugdunensis, mecA- 49576, S. capitis, mecA- 35661. The studies also assessed performance of the Staph ID/R Blood Culture Panel in the presence of E. faecalis 29212 (Negative).

The following bottle types were tested in the study: BACTEC (Standard 10 Aerobic/F, Standard Anaerobic 10/F, Plus Aerobic/F, Plus Anaerobic/F, Lytic 10/F, and PEDS Plus/F), BacT/Alert (SA Standard Aerobic, SN Standard Anaerobic, FA FAN Aerobic, FN FAN Anaerobic, and PF Pediatric FAN), and Versa Trek (Redox 1 and Redox 2). Three bottles of each bottle type were used for each strain, and the samples tested three times in the Staph ID/R Blood Culture Panel for a total of 9 runs for each strain and bottle type combination.

For the ‘valid’ runs tested, all of the potential blood bottle types were compatible with the   
Staph ID/R Blood Culture Panel, with no false negative results. The results are shown in the table below:

Evaluation of Blood Culture Types:   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="8" rowspan="1">Staphylococcus Species, ATCC Strain; Sample Input (CFU/mL), Correct Staph ID/R Blood Panel</td></tr><tr><td colspan="1" rowspan="2">Bottle Type</td><td colspan="1" rowspan="2">S. aureus(mecA +)BA-16801.9x107</td><td colspan="1" rowspan="2">S. aureus(mecA +)BAA-16821.2x107</td><td colspan="1" rowspan="2">S. aureus(mecA -)116327.5x107</td><td colspan="1" rowspan="2">S. lugdunensis(mecA -)495761.9x108</td><td colspan="1" rowspan="1">S. epidermidis(mecA -)12228</td><td colspan="1" rowspan="1">S. epidermidis(mecA +)51625</td><td colspan="1" rowspan="2">S. capitis(mecA- )356612.3x106</td><td colspan="1" rowspan="2">E. faecal(mecA-)292122.1-4.5x1</td></tr><tr><td colspan="1" rowspan="1">2.9x107</td><td colspan="1" rowspan="1">6.5x107</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard 10 Aerobic/F</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Anaerobic</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BACTEC Plus Aerobic/F</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BACTEC Plus Anaerobic/F</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BACTEC Lytic 10/F</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BACTEC PEDS Plus/F</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">14/14***</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert SA Aerobic</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">16/16**</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert SN Anaerobic</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">16/17**</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert FA FAN Aerobic</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert FN FAN Anaerobic</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">BacT/Alert PF Pediatric FAN</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">17/17**</td></tr><tr><td colspan="1" rowspan="1">Versa Trek Redox 1</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">8/9'</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr><tr><td colspan="1" rowspan="1">Versa</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">9/9</td></tr></table>

\*This set of test runs also contained 1 "Invalid" run \*\*This set of test runs also contained 2 "Invalid" runs \*\*\*\*This set of test runs also contained 4 "Invalid" runs 'This set of test runs contained 1 false positive result for mecA "This set of test runs contained 1 false positive Staphylococcus, species undetermined result

A false positive “S. aureus, mecA Present” result was observed in one test run from one Versa Trek Redox 1 bottle for S. aureus, mecA- 11632. The discrepant result is thought to be a contamination event, since all other samples (8/9) gave the correct “S. aureus present, mecA Absent” call, including 2/3 correct results from the same bottle with the discrepant call. All test results did result in a correct S. aureus result, suggesting that the bottle type did not interfere with the assay.

A false positive “Staphylococcus Positive, Staphylococcus species OTHER than $S .$ . aureus or S. lugdunensis, mecA absent” result was observed for $E$ . faecalis 29212 in one test run from one BacT/Alert SN Anaerobic bottle. The discrepant result is also thought to be a random contamination event, because the 16 other “valid” test runs returned a correct “Staphylococcus Negative” result, including 2/3 correct initial results and 3/3 correct re- test results from the same bottle with the discrepant call.

The only other exceptions observed during this study were 9 ‘Invalid’ call runs that are noted in the table above, all with $E .$ . faecalis 29212. When tested with $E .$ . faecalis 29212, 1 invalid run was observed for BACTEC PEDS Plus/F, 1 invalid run for BacT/Alert SA Aerobic, and 1 invalid run for BacT/Alert PF Pediatric FAN bottle types. In all of these cases, extra test runs were performed to evaluate any possible interference with the assay in a Negative sample (evaluating SPC only). Re-test with 9 cards resulted in an additional 3 invalid runs for BACTEC PEDS Plus/F, 1 invalid run for BacT/Alert SA Aerobic, and 1 invalid run for BacT/Alert SN Anaerobic (retested due to false positive contamination result). Overall, it appears that the bottle types with invalid results have an elevated invalid rate $( 5 - 2 8 \% )$ compared to the prospective study $( 2 \% )$ .

# k. Fresh versus Frozen Study:

A study was performed to assess the tolerance of the Staph ID/R Blood Culture Panel to correctly identify cultures stored frozen at $- 2 0 ^ { \circ } \mathrm { C }$ . In this study, 5 different strains used for the Staph ID/R Blood Culture QC Panel were tested: S. aureus, mecA+ ATCC 33592, S. epidermidis, mecA $^ +$ ATCC 700562, S. epidermidis, mecA- ATCC 12228, S. lugdunensis mecA-, ATCC 49576, and $E .$ . faecalis ATCC 29212 (Negative). In addition, 4 strains for fresh vs frozen Reference Site testing were used in the study: S. aureus, mecA $^ +$ ATCC BAA-1682, S. epidermidis, mecA $^ +$ ATCC 700562, S. lugdunensis mecA-, ATCC 49576, and $E .$ . faecalis ATCC 29212 (Negative).

For preparation of these samples, isolate cultures were prepared in TSB broth, incubated overnight and the CFU concentrations for each strain were estimated by optical density measurements. BACTEC Plus Aerobic/F blood bottles containing negative blood were inoculated with each culture at approximately 30-100 CFU/bottle and incubated until alarm positivity in a BACTEC Blood Culture System. Alarm positive bottles were pulled at bottle ring or incubated past positivity $< 8$ hours, consistent with timeframes used during the specimen stability studies. Alarm positive blood cultures were gram stained, diluted, plated on agar plates and colonies counted the following day to confirm concentrations of each bottle.

Samples were tested in the Staph ID/R Blood Culture Panel after bottle ring, but before freezing, and aliquots from the bottles were stored frozen at $- 2 0 ^ { \circ } \mathrm { C }$ . The frozen aliquots were thawed and tested before 1 month and after 5 months storage and tested in the Staph ID/R Blood Culture Panel.

The Staph ID/R Blood Culture Panel results for the fresh vs frozen sample stability study were $100 \%$ concordant with expected results for all strains tested. Frozen storage did not impact performance in the Staph ID/R Blood Culture Panel.

Fresh vs. Frozen Sample Stability Study (Staph ID/R Blood Culture Panel)   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Species, mecAstatus</td><td rowspan=1 colspan=1>ATCCStrain #</td><td rowspan=1 colspan=1>Sample Input(CFU/mL)</td><td rowspan=1 colspan=1>Fresh(Correct)</td><td rowspan=1 colspan=1>Frozen, &lt;1month(Correct)</td><td rowspan=1 colspan=1>Frozen, ≥ 5months(Correct)</td></tr><tr><td rowspan=1 colspan=1>QC1</td><td rowspan=1 colspan=1>S. aureus, mecA+</td><td rowspan=1 colspan=1>33592</td><td rowspan=1 colspan=1>1.7x108</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2//2</td></tr><tr><td rowspan=1 colspan=1>QC2</td><td rowspan=1 colspan=1>S. epidermidis, mecA-</td><td rowspan=1 colspan=1>12228</td><td rowspan=1 colspan=1>3.9x107</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>QC3</td><td rowspan=1 colspan=1>S. lugdunensis, mecA-</td><td rowspan=1 colspan=1>49576</td><td rowspan=1 colspan=1>3.0x106</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>QC4</td><td rowspan=1 colspan=1>S. epidermidis,mecA+</td><td rowspan=1 colspan=1>700562</td><td rowspan=1 colspan=1>6.9x107</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>QC5</td><td rowspan=1 colspan=1>E. faecalis, mecA-</td><td rowspan=1 colspan=1>29212</td><td rowspan=1 colspan=1>3.2x108</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>RP1</td><td rowspan=1 colspan=1>S. aureus, mecA+</td><td rowspan=1 colspan=1>BAA-1682</td><td rowspan=1 colspan=1>5.2x106</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>RP2</td><td rowspan=1 colspan=1>S. epidermidis,mecA+</td><td rowspan=1 colspan=1>700562</td><td rowspan=1 colspan=1>1.9x107</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>RP3</td><td rowspan=1 colspan=1>S. lugdunensis, mecA-</td><td rowspan=1 colspan=1>49576</td><td rowspan=1 colspan=1>4.9x107</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>RP4</td><td rowspan=1 colspan=1>E. faecalis, mecA-</td><td rowspan=1 colspan=1>29212</td><td rowspan=1 colspan=1>1.1x109</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>2/2</td></tr></table>

A parallel study was performed to assess the tolerance of the Staph ID/R Blood Culture Panel reference methods to correctly identify cultures stored frozen at $- 2 0 ^ { \circ } \mathrm { C }$ . Reference site validation samples were shipped at room temperature or frozen and tested in the same assays used during prospective studies (i.e., BD Phoenix ID for species identification and cefoxitin disc diffusion for methicillin resistance). The samples shipped at room temperature were tested within 18 hours of bottle ring, consistent with the specimen stability study conditions. Frozen aliquots were stored frozen $( - 2 0 ^ { \circ } \mathrm { C } )$ , consistent with conditions used for the prospective study, and thawed when ready to be tested. Frozen aliquots were thawed and tested at 1, 30, 60, 90 and 120 days.

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison:

N/A

3. Clinical Studies:

A method comparison study was conducted at three external, geographically-diverse clinical study sites in the U.S. during an eight month period in 2014–2015 to evaluate the comparative performance of the Staph ID/R Blood Culture Panel assay with the PA500 Portrait™ Analyzer System to applicable conventional biochemical and culture reference microbiology methods. Eligible study subjects included individuals receiving routine care requiring blood culture testing. Blood culture specimens were collected from the patients and incubated on the BACTEC continuous monitoring blood culture system. Bottles that were flagged positive by the instrument were Gram stained and then bottles confirmed to contain gram-positive cocci in clusters (GPCC) or gram-positive cocci in singles (GPC) were then tested with the Staph ID/R Blood Culture Panel and compared to traditional laboratory reference methods (i.e., culture followed by testing blood culture isolates with conventional biochemical, BD Phoenix, and cefoxitin disc diffusion testing). Cefoxitin disc diffusion tests were used as the reference method for confirming mecA mediated resistance. A total of 853 samples were collected for all three sites combined. Twentytwo (22) specimens were excluded from the Staph ID/R Blood Culture Panel clinical study dataset. The remaining 831 clinical specimens met the inclusion criteria and were used in the prospective study to evaluate the performance of the Staph ID/R Blood Culture Panel. A total of 762 prospective samples were tested originally in the clinical trial, while the remaining 69 archived frozen specimens were tested after the prospective clinical trial

# Clinical evaluation sites in the United States

<table><tr><td colspan="2" rowspan="1">Clinical Site 1 - 3</td><td colspan="1" rowspan="1">City, State</td><td colspan="1" rowspan="1">GeographicArea</td><td colspan="1" rowspan="1">InstitutionType</td><td colspan="1" rowspan="1">PrincipalInvestigator</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Indiana University School of Medicine (IU)</td><td colspan="1" rowspan="1">Indianapolis, IN</td><td colspan="1" rowspan="1">Midwest</td><td colspan="1" rowspan="1">Hospital Lab</td><td colspan="1" rowspan="1">Gerald Denys, PhD</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Primary Children's Medical Center (PCMC)</td><td colspan="1" rowspan="1">Salt Lake City,UT</td><td colspan="1" rowspan="1">West</td><td colspan="1" rowspan="1">Hospital Lab</td><td colspan="1" rowspan="1">Judy Daly, PhD</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">TriCore Reference Laboratories</td><td colspan="1" rowspan="1">Albuquerque, NM</td><td colspan="1" rowspan="1">Southwest</td><td colspan="1" rowspan="1">ReferenceLaboratory</td><td colspan="1" rowspan="1">Stephen Young, PhD</td></tr><tr><td colspan="2" rowspan="1">Additional Collection Site for PCMC</td><td colspan="1" rowspan="1">City, State</td><td colspan="1" rowspan="1">GeographicArea</td><td colspan="1" rowspan="1">InstitutionType</td><td colspan="1" rowspan="1">PrincipalInvestigator</td></tr><tr><td>ARUP Laboratories</td><td>Salt Lake City, UT</td><td>West</td><td>Reference Laboratory</td><td>Marc Couturier, PhD</td></tr><tr><td>Microbiology Reference Site</td><td></td><td></td><td></td><td></td></tr><tr><td>Medical College of Wisconsin</td><td>Milwaukee, WI</td><td>Midwest</td><td>Reference Laboratory</td><td>Nate Ledeboer, PhD</td></tr></table>

In addition, 102 Staph ID/R Blood Culture Panel assays were performed on a ‘Low Prevalence’ panel of contrived or ‘simulated’ blood culture specimens, consisting of low prevalence Staphylococcus species and gram-positive negatives. These specimens were prepared by spiking blood culture bottles containing whole blood with bacterial suspensions of bacterial isolates. Prepared blood culture bottles were then grown to positivity on the BACTEC blood culture system until flagged positive. Gram stain was performed to verify the presence of gram-positive cocci in clusters (GPCC) or grampositive cocci in singles (GPC) and then testing was performed with the Staph ID/R Blood Culture Panel.

The following tables provide the clinical performance of the Staph ID/R Blood Culture Panel for organism identification of overall Staphylococcus species, S. aureus, S. epidermidis, S. lugdunensis, and Staphylococcus species undetermined (i.e., not S. aureus, or S. lugdunensis) as determined by the reference methods. For each organism identification, Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) of the Staph ID/R Blood Culture Panel assay are calculated based upon the $\%$ agreement to the reference methods.

Demographic Summary for Prospective Arm of the Staph ID/R Blood Culture Panel   

<table><tr><td rowspan=1 colspan=4>Prospective Study Specimens: Total Specimens - 831</td></tr><tr><td rowspan=1 colspan=2>Sex</td><td rowspan=1 colspan=2>Number of Specimens</td></tr><tr><td rowspan=1 colspan=2>Male</td><td rowspan=1 colspan=2>454</td></tr><tr><td rowspan=1 colspan=2>Female</td><td rowspan=1 colspan=2>377</td></tr><tr><td rowspan=1 colspan=1>Patient Age</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Supplemental</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>≤ 1 year</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>92</td></tr><tr><td rowspan=1 colspan=1>1 - 18 years</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>19 - 35 years</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>36 - 55 years</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>56 - 70 years</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>221</td></tr><tr><td rowspan=1 colspan=1>≥ 71 years</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>206</td></tr></table>

Positive percent agreement (PPA) was calculated as $100 \%$ x $\left( \mathrm { T P / T P + F N } \right)$ . True positive (TP) indicates that both Staph ID/R Blood Culture Panel and the reference/comparator method had a positive result for a specific analyte, and false negative (FN) indicates that the Staph ID/R Blood Culture Panel result was negative while the reference/comparator method was positive. Clinical specificity or negative percent agreement (NPA) was calculated as $100 \%$ x $( \mathrm { T N } / \mathrm { T N } + \mathrm { F P } )$ . True negative (TN) indicates that both Staph ID/R Blood Culture Panel and the reference/comparator method had a negative result for a specific analyte, and false positive (FP) indicates that the Staph ID/R Blood Culture Panel result was positive while the reference/comparator method was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. The results are summarized in the following tables.

# Summary of Clinical Performance of Staph ID/R Blood Culture Panel versus Reference Method(s) – Prospective and Simulated Blood Cultures.

<table><tr><td rowspan=2 colspan=6>% AgreementAll sites combined                                                            TP/TP +     PPA                     NPATN/TN+TPFN      95% CI                  95% CI</td></tr><tr><td rowspan=1 colspan=1>TP/TP +FN</td><td rowspan=1 colspan=1>PPA95% CI</td><td rowspan=1 colspan=1>TN/TN+TP</td><td rowspan=1 colspan=1>NPA95% CI</td></tr><tr><td rowspan=3 colspan=1>Detection of Staphylococcus aureus</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>211/214</td><td rowspan=1 colspan=1>98.6%96.0 - 99.5%</td><td rowspan=1 colspan=1>548/551</td><td rowspan=1 colspan=1>99.5%98.4 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>102/102</td><td rowspan=1 colspan=1>100.0%96.4 - 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>211/214</td><td rowspan=1 colspan=1>98.6%96.0 - 99.5%</td><td rowspan=1 colspan=1>650/653</td><td rowspan=1 colspan=1>99.5%98.7 - 99.8%</td></tr><tr><td rowspan=3 colspan=1>Detection of Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%43.9 - 100%</td><td rowspan=1 colspan=1>761/762</td><td rowspan=1 colspan=1>99.9%99.3 - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%88.7 - 100%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%94.9 - 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%89.6 - 100%</td><td rowspan=1 colspan=1>833/834</td><td rowspan=1 colspan=1>99.9%99.3 - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Detection of Staphylococcus species OTHER than S. aureusor S. lugdunensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>444/449</td><td rowspan=1 colspan=1>98.9%97.4 - 99.5%</td><td rowspan=1 colspan=1>307/316</td><td rowspan=1 colspan=1>97.2%94.7 - 98.5%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Detection of mecA with Staphylococcus aureus</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>68/72</td><td rowspan=1 colspan=1>94.4%86.6 - 97.8%</td><td rowspan=1 colspan=1>682/690</td><td rowspan=1 colspan=1>98.8%97.7- 99.4%</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100.0%90.1 - 100.0%</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%89.9 - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>103/107</td><td rowspan=1 colspan=1>96.3%90.8 - 98.5%</td><td rowspan=1 colspan=1>716/724</td><td rowspan=1 colspan=1>98.9%97.8- 99.4%</td></tr><tr><td rowspan=1 colspan=1>Detection of mecA with Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>762/762</td><td rowspan=1 colspan=1>100.0%99.5 - 100%</td></tr><tr><td rowspan=1 colspan=1>Detection of mecA with Staphylococcus species OTHERthan S. aureus or S. lugdunensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>243/262</td><td rowspan=1 colspan=1>92.7%88.1 - 97.1%</td><td rowspan=1 colspan=1>481/500</td><td rowspan=1 colspan=1>96.2%92.4 - 98.0%</td></tr></table>

Summary of Staphylococcus Genus-level Analyte for (Staphylococcus species OTHER than S. aureus or S. lugdunensis’) versus Reference Method(s).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=2 colspan=4>% Agreement (95% CI)Prospective          Simulated</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=2>Prospective</td></tr><tr><td rowspan=1 colspan=1>S. arlettae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. auricularis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. capitis</td><td rowspan=1 colspan=1>32/35</td><td rowspan=1 colspan=1>91.4%77.6-97.0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. carnosus</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%20.7-100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. cohnii</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%34.2-100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. equorum</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%34.2-100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus</td><td rowspan=1 colspan=1>12/15</td><td rowspan=1 colspan=1>80.0%54.8-93.0%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>83/97</td><td rowspan=1 colspan=1>85.6%77.2-91.2%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. intermedius</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. pettenkoferi</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100%64.6-100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. saprophyticus</td><td rowspan=1 colspan=1>4/8</td><td rowspan=1 colspan=1>50.0%21.5-78.5%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. schleiferi</td><td rowspan=1 colspan=1>0/2</td><td rowspan=1 colspan=1>0.0%0-0.7%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. sciuri</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. simulans</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%34.2-100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. species</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%34.2-100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. warneri</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%67.6-100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr><tr><td rowspan=1 colspan=1>S. xylosus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%43.9-100%</td></tr></table>

# Polymicrobials – Mixed Specimen Combinations Detected by Staph ID/R Blood Culture Panel and Reference Method(s).

<table><tr><td rowspan=1 colspan=1>Staph ID/R Result</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Reference Result</td><td rowspan=1 colspan=2>Discrepant Result Description</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Species dentification</td><td rowspan=1 colspan=1>mecAResut</td><td rowspan=1 colspan=1>Organlsm 1</td><td rowspan=1 colspan=1>CefoxitinResult</td><td rowspan=1 colspan=1>Organism 2</td><td rowspan=1 colspan=1>CefoxitinResult</td><td rowspan=1 colspan=1>Organism 3</td><td rowspan=1 colspan=1>SpeciesIdentification</td><td rowspan=1 colspan=1>mecA Result</td></tr><tr><td rowspan=1 colspan=11>u</td></tr><tr><td rowspan=1 colspan=1>Site 2(aly)</td><td rowspan=1 colspan=1>DALY125</td><td rowspan=1 colspan=1>S. aureus in mixed Staphnion (NOT . lugd)</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S, oureus</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA cll</td></tr><tr><td rowspan=1 colspan=11>ba</td></tr><tr><td rowspan=9 colspan=1>Site 1,IU(Deenys)</td><td rowspan=1 colspan=1>DNYS031</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus or S. lugdunensis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S, aureus</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>FN for S. aoureus;Corect for Staph call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS033</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. udunersis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S, capitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>No growth</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct specie cal</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS047</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. ugdunensis</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S, copitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS071</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. lugdunersis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. haemolyticus</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct speces call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS123</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. lugduenss</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS191</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. lugdunensls</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS202</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. lugdunersis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>FN mecA</td></tr><tr><td rowspan=1 colspan=1>DNYS288</td><td rowspan=1 colspan=1>Staph, species OTHER thanS. aureus or S. lugdunensis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S, capitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. pettenkoferi</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>FN mecA</td></tr><tr><td rowspan=1 colspan=1>DNYS297</td><td rowspan=1 colspan=1>Staph, species OTHER thanS. aureus orS. ugduenss</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S, hominils</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1>S. copitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=4 colspan=1>Site 3,Tr rICore(Young)</td><td rowspan=1 colspan=1>YNG109</td><td rowspan=1 colspan=1>Staph, species OTHER than. aureus or S. lugdunensis</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. copitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>YNG129</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. ugdunensls</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. warmeri</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>YNG172</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. oureus or S. lugdunenss</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1>S. hoemolyticus</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>YNG197</td><td rowspan=1 colspan=1>Staph. species OTHER thanS. aureus orS. lugdunenss</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1>S. copitis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct species call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=6></td><td rowspan=1 colspan=5></td></tr><tr><td rowspan=2 colspan=1>Site 1, IU( (enys)</td><td rowspan=1 colspan=1>DNYS026</td><td rowspan=1 colspan=1>S. aureus in mixed Staphnecon NOT . ug</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FP for S. cureus;Correct fo Staph call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>DNYS045</td><td rowspan=1 colspan=1>S. aureus in mixed Staphnection NT S </td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. oureus</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct for s.oureus; FP for mixed</td><td rowspan=1 colspan=1>FP mecA</td></tr><tr><td rowspan=2 colspan=1>Site 3, TriCore(Young)</td><td rowspan=1 colspan=1>YNG207</td><td rowspan=1 colspan=1>S. aureus in mixed Staphncion NOT S.</td><td rowspan=1 colspan=1>Present</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Resistant</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FP for S. aureus;Correct for Staph call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr><tr><td rowspan=1 colspan=1>YNG299</td><td rowspan=1 colspan=1>S. aureus in mixed StaphneconNOT .</td><td rowspan=1 colspan=1>Absent</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Sensitive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FP for S. oureus;Correct for Staph call</td><td rowspan=1 colspan=1>Correct mecA call</td></tr></table>

# N. Instrument Name:

The Great Basin PA500 Portrait™ Analyzer System

# O. System Descriptions:

1. Modes of Operation:

# See Device Description (Section I) above

2. Software

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _X___ or No

3. Specimen Identification: N/A

4. Specimen Sampling and Handling: N/A

5. Calibration: N/A

6. Quality Control: See Section M (i) above

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.