# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K153544

# B. Purpose for Submission:

New device 510k clearance for the cobas® Influenza A/B & RSV Nucleic Acid Test for Use on the cobas® Liat System (cobas® Liat Influenza A/B & RSV) and the cobas® Influenza A/B & RSV Quality Control Kit

# C. Measurand:

Target RNA sequences of a well-conserved region of the matrix gene of influenza A (Inf A), the nonstructural protein (NSP) gene of influenza B (Inf B), and the matrix gene of respiratory syncytial virus (RSV).

# D. Type of Test:

Multiplexed nucleic acid assay for the qualitative detection of Influenza A, Influenza B and RSV RNA from Nasopharyngeal Swab (NPS) specimens, including nucleic acid isolation and multiplex real-time RT-PCR amplification using the cobas® Liat System.

# E. Applicant:

IQuum, Inc. (a wholly-owned subsidiary of Roche Molecular Systems Inc.)

# F. Proprietary and Established Names:

1) cobas® Influenza A/B & RSV Nucleic Acid Test for Use on the cobas® Liat System (cobas® Liat Influenza A/B & RSV)

2) cobas® Influenza A/B & RSV Quality Control Kit

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code(s):

OCC - Respiratory virus panel nucleic acid assay system OZE - Influenza A and influenza B multiplex nucleic acid assay OOI - Real time nucleic acid amplification system JJX - Single (specified) analyte controls (assayed and unassayed)

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

1) The cobas® Influenza A/B & RSV Nucleic Acid Test for Use on the cobas® Liat System (cobas® Liat Influenza A/B & RSV) is an automated multiplex real-time RT-PCR assay for the rapid in vitro qualitative detection and discrimination of influenza A virus, influenza B virus and respiratory syncytial virus (RSV) RNA in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. The test is intended for use as an aid in the diagnosis and differentiation of influenza A, influenza B, and RSV in humans and is not intended to detect influenza C.

Negative results do not preclude influenza virus or RSV infection and should not be used as the sole basis for treatment or other patient management decisions. Conversely, positive results do not ruleout bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease.

Performance characteristics for influenza A were established during the 2013-2014 and the 2014- 2015 influenza seasons when influenza A/H3 and A/H1N1 pandemic were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2) The cobas® Influenza A/B & RSV Quality Control Kit contains External Controls for use with the cobas® Liat Influenza A/B & RSV assay. External Controls are run during the Add cobas® Liat Influenza A/B & RSV Tube Lot procedure. Additional External Controls should be tested in accordance with local, state, federal and/or accrediting organization requirements as applicable.

2. Indication(s) for use:

Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: cobas® Liat System

# I. Device Description:

# Overview

The cobas® Liat Influenza A/B & RSV Nucleic Acid Test for Use on the cobas® Liat System (cobas $^ \mathrm { \textregistered }$ Liat Influenza A/B & RSV) is a rapid, automated in vitro diagnostic test for the qualitative detection of influenza A, influenza B, and RSV RNA in nasopharyngeal swab (NPS) specimens eluted in viral transport media.

The assay targets a well-conserved region of the matrix gene of influenza A (Inf A target), the non-structural protein gene of influenza B (Inf B target), and the matrix gene of RSV (RSV target). An Internal Process Control (IPC) is also included. The IPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the sample preparation and RT-PCR.

The assay utilizes a single-use disposable cobas® Liat Tube that holds the sample purification and PCR reagents, and hosts the sample preparation and PCR processes. The cobas® Liat Tube uses a flexible tube as a sample vessel. It contains all required unit dose reagents pre-packed in tube segments, separated by peelable seals, in the order of reagent use.

The cobas® Liat System automates and integrates sample purification, nucleic acid amplification, and detection of the target sequence in biological samples. The cobas® Liat System performs all assay steps from clinical sample and reports assay result automatically. During the testing process, multiple sample processing actuators of the cobas® Liat System compress the cobas® Liat Tube to selectively release reagents from tube segments, move the sample from one segment to another, and control reaction volume, temperature, and time to conduct sample preparation, nucleic acid extraction, target enrichment, inhibitor removal, nucleic acid elution, and real-time PCR. An embedded microprocessor controls and coordinates the actions of these sample processors to perform all required assay steps within the closed cobas® Liat Tube.

# Materials Provided

The cobas® Liat Influenza A/B & RSV Assay Pack (Cat # 07341903190) contains sufficient reagents to process 20 specimens or quality control samples. The pack contains 20 sets of a cobas® Liat Influenza A/B & RSV Tube and a transfer pipette. A Package Insert Barcode Card (Cat # 34- 04895) with lot-specific barcode is also included.

# Equipment

cobas® Liat System (Cat # 07341920190) (Analyzer)

# Materials Required but Not Provided

Acceptable specimen collection kits include Universal Transport Medium (UTM) Swab Collection Kits (BD Cat # 220531 or Copan Cat # 305C), each kit containing a Collection Swab and a tube containing $3 ~ \mathrm { m L }$ of UTM, or equivalent.

cobas® Liat Influenza A/B & RSV Quality Control Kit, Cat# 07402686190, containing: cobas® Liat Influenza A/B & RSV Positive Control (Positive Control), Cat # 20-04515 cobas® Liat Influenza A/B & RSV Dilution UTM (Dilution UTM), Cat # 20-04514 Transfer Pipette, Cat # 20-03795 Control Kit Barcode Card, Cat # 34-04894 Negative Control Barcode Label, Cat # 34-04535 Positive Control Barcode Label, Cat # 34-04534

# Quality Control

The cobas® Liat Influenza A/B & RSV assay has three controls: (1) internal process control, (2) positive control, and (3) negative control.

# Internal Process Control

The internal process control (IPC) is comprised of a bacteriophage MS2 that is pre-packed in each cobas® Liat tube. The IPC is designed to be detected at a cycle threshold $\mathrm { ( C t ) }$ of ${ \sim } 3 1$ on the cobas® Liat Influenza A/B & RSV assay, similar to the Ct of approximately 4x Limit of Detection (LOD) concentration of target viruses. When conducting an assay, IPC is first mixed with a sample and then it goes through all the test processes to monitor both the sample preparation and RT-PCR performance. The IPC RNA is detected in a separate channel by MS2 specific primers and probe. If IPC Ct does not meet the acceptance criteria and Inf A, Inf B, and RSV are not detected, the assay run is called “Invalid” to avoid false negative results due to excessive sample inhibition or system operation outside the normal range.

# Positive Control

Positive control is provided in the cobas® Liat Influenza A/B & RSV assay Quality Control Kit. The positive control is comprised of a pooled sample of inactivated Influenza A, Influenza B, and RSV. The individual inactivated viruses are pooled to make the positive control solution, and then dispensed and “freeze-dried” in unit-dose quantities in individual Positive Control tubes. To use the positive control, an operator transfers a unit dose of Universal Transport Media (UTM) from the Dilution UTM tube into the Positive Control tube using an included transfer pipette to dissolve and mix the dried positive control. The entire mixture is then transfer into the cobas® Liat tube, and the cobas® Liat tube is run on a cobas® Liat Analyzer according to the Instructions for Use.

The positive control is required to be run during the “Add Liat Tube Lot” process, in which the cobas® Liat tube lot and end user site procedures are checked. Additional positive control runs may be performed by the end user to confirm the performance of a cobas® Liat Analyzer and a cobas® Liat tube lot through detection of Influenza A, Influenza B and RSV targets, or as required by the end user’s quality control standards.

# Negative Control

Negative control is provided in the cobas® Liat Influenza A/B & RSV assay Quality Control Kit. The negative control is comprised of UTM. The solution is provided in unit dose quantity and labeled as Dilution UTM. To use the negative control, an operator transfers the entire contents of the Dilution UTM tube into the cobas® Liat tube using a transfer pipette and runs the assay following the Instructions for Use.

The negative control is required to be run during the “Add Liat Tube Lot” process, in which potential contamination and end user site procedures are checked. Additional negative control runs may be performed by the end user to check if there is contamination resulting in a false positive result, or as required by the end user’s quality control standards.

# Workflow

To perform the cobas® Liat Influenza A/B & RSV assay, an operator first collects a nasopharyngeal swab and places the swab into UTM. The operator transfers the sample into cobas® Liat Influenza A/B & RSV assay tube using a transfer pipette, and scans the tube barcode to identify the test and the sample barcode to code the sample ID with the assay run on the cobas® Liat System. The cobas® Liat Tube is then inserted into the cobas® Liat System. The system performs all the test steps and outputs interpreted results (e.g., Influenza A Detected, Influenza B Not Detected, RSV Not Detected) in ${ \sim } 2 0$ minutes. A report of the interpreted results can be viewed on the cobas® Liat System’s LCD screen, and printed directly through a USB or network connected printer. No reagent preparation or additional steps are required other than adding the sample to the cobas® Liat Tube. Because all the reagents are contained within the cobas® Liat Tube and no sample or reagent needs to be removed from the tube, cross-contamination between samples is minimized.

# Results Interpretation

The results are interpreted by the cobas® Liat System software from measured fluorescent signals and real time curve recognition algorithm. No manual data interpretation or reviewing PCR curves is required. The cobas® Liat Analyzer automatically interprets the results from measured fluorescent signals. Embedded algorithms determine the PCR cycle threshold (Ct), and evaluate the Ct against a valid range (i.e., minimum value to assay cutoff) to generate a positive or negative PCR result. Additionally, pattern recognition algorithms inspect the baseline, logarithmic, and plateau phases of the PCR curve to determine if the curve is typical/normal or abnormal. If a PCR curve is determined to be abnormal, the result may be called “Invalid” or “Indeterminate.” This pattern recognition provides additional error checking capabilities to ensure the quality of the results. In cases where one of Inf A, Inf B, and RSV is detected, but the PCR curve pattern is abnormal, the result is reported as “Invalid” to avoid false positive results. When at least two of Inf A, Inf B and RSV are detected but the PCR curve pattern for one target is abnormal, the target(s) with the normal curve pattern is called “Detected” while the target with the abnormal curve is called “Indeterminate”.

All possible final test results and the respective follow-up actions are described in Table 1 below:

Table 1: Interpretation of Results   

<table><tr><td rowspan=1 colspan=2>Result Report</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza A Not Detected</td><td rowspan=1 colspan=1>Negative test for Influenza A(no Influenza A RNA detected)</td></tr><tr><td rowspan=1 colspan=1>Influenza A Detected</td><td rowspan=1 colspan=1>Positive test for Influenza A(Influenza A RNA present)</td></tr><tr><td rowspan=1 colspan=1>Influenza A IndeterminateRepeat Assay</td><td rowspan=1 colspan=1>Presence or absence of Influenza A cannot be determined. Repeatassay with same sample or, if possible, new sample.</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Influenza B Not Detected</td><td rowspan=1 colspan=1>Negative test for Influenza B(no Influenza B RNA detected)</td></tr><tr><td rowspan=1 colspan=1>Influenza B Detected</td><td rowspan=1 colspan=1>Positive test for Influenza B(Influenza B RNA present)</td></tr><tr><td rowspan=1 colspan=1>Influenza B IndeterminateRepeat Assay</td><td rowspan=1 colspan=1>Presence or absence of Influenza B cannot be determined. Repeatassay with same sample or, if possible, new sample.</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>RSV Not Detected</td><td rowspan=1 colspan=1>Negative test for RSV(no RSV RNA detected)</td></tr><tr><td rowspan=1 colspan=1>RSV Detected</td><td rowspan=1 colspan=1>Positive test for RSV(RSV RNA present)</td></tr><tr><td rowspan=1 colspan=1>RSV IndeterminateRepeat Assay</td><td rowspan=1 colspan=1>Presence or absence of RSV cannot be determined. Repeat assaywith same sample or, if possible, new sample.</td></tr><tr><td rowspan=1 colspan=2>Assay Invalid. Repeat Assay</td><td rowspan=2 colspan=1>Presence or absence of Influenza A, Influenza B, and RSV cannotbe determined. Repeat assay with same sample or, if possible, newsample.</td></tr><tr><td rowspan=1 colspan=2>[Error]. Assay Aborted</td></tr></table>

If the test result is “Indeterminate” or “Invalid”, the assay should be repeated with the same patient specimen, or if possible, with a newly collected specimen. Specimens that have repeat “Indeterminate” or “Invalid” results should be sent to a laboratory for confirmatory testing.

If an assay is aborted due to run error, or if an assay is aborted by user, the test should be repeated with the same sample. Or, if possible, the test should be repeated with a new sample. Manufacturer Service Representative should be contacted if repeat “Errors” are reported.

Dual infections of Influenza A and Influenza B are rare. If the test result is “Influenza A Detected” and “Influenza B Detected”, the assay should be repeated with the same patient specimen, or if possible, with a newly collected specimen. Specimens that have repeat “Influenza A Detected” and “Influenza B Detected” results should be sent to a laboratory for confirmatory testing.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Hologic Prodesse ProFlu+™

2. Predicate 510(k) number: K132129

<table><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">cobas® Influenza A/B &amp; RSV Nucleic AcidTest for Use on the cobas® Liat System(K153544)</td><td colspan="1" rowspan="1">Hologic Prodesse ProFlu+TM (K132129)</td></tr><tr><td colspan="1" rowspan="1">Assay Analyte</td><td colspan="1" rowspan="1">Influenza A, Influenza B, and RSV</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The cobas® Influenza A/B &amp; RSV Nucleic AcidTest for Use on the cobas® Liat System (cobas®Liat Influenza A/B &amp; RSV) is an automatedmultiplex real-time RT-PCR assay for the rapidin vitro qualitative detection and discriminationof influenza A virus, influenza B virus andrespiratory syncytial virus (RSV) RNA innasopharyngeal swab specimens from patientswith signs and symptoms of respiratory infectionin conjunction with clinical and epidemiologicalrisk factors. The test is intended for use as an aidin the diagnosis and differentiation of influenzaA, influenza B, and RSV in humans and is notintended to detect influenza C.Negative results do not preclude influenza virusor RSV infection and should not be used as thesole basis for treatment or other patientmanagement decisions. Conversely, positiveresults do not rule-out bacterial infection or co-infection with other viruses. The agent detectedmay not be the definite cause of disease.Performance characteristics for influenza A wereestablished during the 2013-2014 and the 2014-2015 influenza seasons when influenza A/H3and A/H1N1 pandemic were the predominantinfluenza A viruses in circulation. When otherinfluenza A viruses are emerging, performancecharacteristics may vary.If infection with a novel influenza A virus issuspected based on current clinical andepidemiological screening criteria recommendedby public health authorities, specimens should becollected with appropriate infection controlprecautions for novel virulent Influenza virusesand sent to state or local health department fortesting. Viral culture should not be attempted inthese cases unless a BSL 3+ facility is availableto receive and culture specimens.</td><td colspan="1" rowspan="1">The ProFlu+TM Assay is a multiplex Real-TimePCR (RT-PCR) in vitro diagnostic test for therapid and qualitative detection anddiscrimination of Influenza A Virus, Influenza BVirus, and Respiratory Syncytial Virus (RSV)nucleic acids isolated and purified fromnasopharyngeal (NP) swab specimens obtainedfrom symptomatic patients. This test is intendedfor use to aid in the differential diagnosis ofInfluenza A, Influenza B and RSV viralinfections in humans and is not intended todetect Influenza C.Negative results do not preclude influenza orRSV virus infection and should not be used asthe sole basis for treatment or other managementdecisions. Conversely, positive results do notrule-out bacterial infection or co-infection withother viruses. The agent detected may not be thedefinite cause of disease. The use of additionallaboratory testing and clinical presentation mustbe considered in order to obtain the finaldiagnosis of respiratory viral infection.Performance characteristics for Influenza AVirus were established when Influenza A/H3and A/H1 were the predominant Influenza Aviruses in circulation (2006 - 2007 respiratoryseason). Performance characteristics forInfluenza A were confirmed when InfluenzaA/H1, Influenza A/H3, and Influenza A/2009H1N1 were the predominant Influenza A virusesin circulation (2008 and 2009). When otherInfluenza A viruses are emerging, performancecharacteristics may vary.If infection with a novel Influenza A virus issuspected based on current clinical andepidemiological screening criteria recommendedby public health authorities, specimens shouldbe collected with appropriate infection controlprecautions for novel virulent Influenza virusesand sent to state or local health department fortesting. Viral culture should not be attempted inthese cases unless a BSL 3+ facility is availableto receive and culture specimens.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasopharyngeal swab eluted in VTM</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">InternalControl</td><td colspan="1" rowspan="1">Yes for sample preparation/nucleic acidextraction and RT-PCR performance using</td><td colspan="1" rowspan="1">Yes for RT-PCR performance only using in vitrotranscribed RNA. Extraction control is</td></tr><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">cobas® Influenza A/B &amp; RSV Nucleic AcidTest for Use on the cobas® Liat System(K153544)</td><td colspan="1" rowspan="1">Hologic Prodesse ProFlu+TM (K132129)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">encapsulated RNA.</td><td colspan="1" rowspan="1">recommended but not provided.</td></tr><tr><td colspan="1" rowspan="1">Influenza AViral Target</td><td colspan="1" rowspan="1">Well conserved region of the matrix gene</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Influenza BViral Target</td><td colspan="1" rowspan="1">Well conserved region of the non-structureprotein (NSP) gene</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">RSV ViralTarget</td><td colspan="1" rowspan="1">Well conserved region of the matrix gene</td><td colspan="1" rowspan="1">Polymerase gene</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethod</td><td colspan="1" rowspan="1">Integrated silica-magnetic bead-based nucleicacid extraction</td><td colspan="1" rowspan="1">Silica-magnetic bead-based nucleic acidextraction using Roche MagNA Pure LCSystem, or bioMérieux NucliSENS easyMag</td></tr><tr><td colspan="1" rowspan="1">Assay Method</td><td colspan="1" rowspan="1">RT-PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Multiplex assay using different reporter dyes foreach target</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Result</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">AssayInstrument</td><td colspan="1" rowspan="1">cobas® Liat System</td><td colspan="1" rowspan="1">Roche MagNA Pure LC System, or bioMérieuxNucliSENS easyMagCepheid SmartCycler II Real Time Instrument</td></tr><tr><td colspan="1" rowspan="1">Self-containedSystem</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">All AssayReagentsContained inDisposable</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Sample VolumeDetection</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">AutomatedAssay</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">ErrorDiagnosticSystem</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">PCR CurvePatternRecognition</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">AutomatedResultInterpretation</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">User</td><td colspan="1" rowspan="1">Hospital nurse and CLIA moderate complexitylaboratory technologist.Users in various CLIA waived settings, such asnurses and medical assistants at emergencyrooms, urgent care clinics, and outpatient clinics,including physician's offices.</td><td colspan="1" rowspan="1">CLIA high complexity laboratory technologist.</td></tr><tr><td colspan="1" rowspan="1">TestAvailability</td><td colspan="1" rowspan="1">Random access, on-demand test</td><td colspan="1" rowspan="1">Batch processing</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">~20 minutes</td><td colspan="1" rowspan="1">≥4 hours</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff – Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses. Document issued on July 15, 2011. Docket number FDA-2008-D-0095.

# L. Test Principle:

The cobas® Liat Influenza A/B & RSV assay uses well established nucleic acid test chemistry and assay protocol for viral RNA detection. The sample preparation methodology is based on chaotropic agent-based lysis and magnetic particle nucleic acid purification. First, the NPS sample is diluted and mixed with an internal process control (IPC) (encapsulated RNA). Chaotropic and proteolytic reagent then disrupts the three dimensional structure in macromolecules such as proteins and nucleic acids in the sample, and denatures them. Second, nucleic acids are isolated from lysates through binding to the surface of silica magnetic beads in the presence of a chaotropic salt, which removes water from hydrated molecules in solution. Third, the beads are separated from the lysates using a magnetic field, and the lysate removed. Fourth, the beads with captured nucleic acids are washed to remove possible inhibitors in the sample. Finally, the captured nucleic acids are eluted under low-salt conditions into a small volume of elution buffer

Eluted viral RNA is first transcribed into cDNA using reverse transcriptase. This cDNA then undergoes a polymerase chain reaction (PCR) where the reaction mixture is repeatedly heated to denature the nucleic acid, and cooled to allow annealing of primers and extension of annealed primers by DNA polymerase to logarithmically amplify the specific region of the cDNA. Dual-labeled fluorogenic hydrolysis (TaqMan) probes anneal to the specific target sequences located between the binding regions of forward and reverse primers. During the extension phase of the PCR cycle, the $5 ^ { \prime }$ nuclease activity of the polymerase degrades the probes, causing the reporter dyes to separate from the quenchers, thus generating fluorescent signals. Fluorescence intensities are monitored at each PCR cycle. When the fluorescence intensities exceed predetermined thresholds, cycle threshold $\mathrm { ( C t ) }$ values are generated for the specific analyte corresponding to the fluorescence channel.

All these sample preparation/nucleic acids extractions and real-time PCR amplification processes are conducted in a closed cobas® Liat Tube in ${ \sim } 2 0$ minutes.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Precision/Reproducibility:

# Reproducibility Study

A reproducibility study was conducted assessing the total variability of the cobas® Liat Influenza $\mathrm { A } / \mathrm { B }$ & RSV assay across operators, study sites, testing days, cobas® Liat Analyzers, and cobas® Liat assay tube lots. The cobas® Liat Influenza A/B & RSV assay was evaluated at three CLIA waived sites. Two (2) operators at each of the three sites tested a 10-member reproducibility panel in triplicate on five different days, for a total of ${ \sim } 9 0 0$ runs (10 panel members $\times 3$ replicates $\times 2$ operators $\times 5$ days $\times 3$ sites). The reproducibility panel contained a true negative; and a high negative, a low positive and a moderate positive of each of Influenza A, Influenza B and RSV.

The reproducibility panel samples were constructed by spiking viruses of known titer into negative NPS in UTM sample matrix. A/Brisbane/59/07, B/Malaysia/2506/04, and RSV-A 2006 Isolate with known $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ titers were obtained from a vendor. The moderate positive, low positive, and high negative concentrations used for each of the strains corresponded to $\mathrm { C } _ { 1 0 0 }$ (4xLOD), C95 (1xLOD), and $\mathrm { C } _ { 5 }$ (as determined by Probit analysis using LOD data), respectively. The true negative sample comprised negative NPS sample matrix in UTM. For a given virus, the expected result for the true negative and the high negative panel member is “Not Detected”, while the expected result for the low positive and moderate positive panel member is “Detected”.

Three (3) CLIA waived sites and six operators (two operators per site) participated in this reproducibility study. The six operators consisted of two nurses, three medical assistants, and one administrative assistant with no formal medical laboratory training. All operators had limited or no training or hands-on experience in conducting laboratory testing.

The six operators at the three sites tested the members of the reproducibility panel in triplicate on five non-consecutive days. Three (3) cobas® Liat Analyzers were used at each site for a total of nine cobas® Liat Analyzers. Each site also used a different lot of cobas® Liat Influenza A/B & RSV assay tubes for a total of three lots.

Percent agreement with expected result, mean Ct, and Ct $\% \mathrm { C V }$ for each site are show in Table 2 to Table 4 below:

Table 2: Influenza A Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31 /31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNegative (C5)</td><td rowspan=1 colspan=1>29 / 30</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>29 / 30</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>88 / 90 (97.8%)</td><td rowspan=1 colspan=1>92.3% -99.4%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31 /31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 /30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>29 / 291</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>89 / 89 (100.0%)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>RSV HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>RSV LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0% -100.0%</td></tr><tr><td rowspan=1 colspan=1>RSV ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>299 / 300 (99.7%)</td><td rowspan=1 colspan=3>303 / 303 (100.0%)</td><td rowspan=1 colspan=3>298 / 299 (99.7%)</td><td rowspan=1 colspan=1>900 / 902(99.8%)</td><td rowspan=1 colspan=1>99.2% -99.9%</td></tr></table>

30 Flu B Low Positive (C95) replicates yielded an “Assay Invalid. Repeat Assay.” result, and was not repeated.

Table 3: Influenza B Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31 /31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 /91 (100.0%)</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Flu A LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>Flu A ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>Flu B HighNegative (C5)</td><td rowspan=1 colspan=1>29 / 30</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 91 (98.9%)</td><td rowspan=1 colspan=1>94.0% - 99.8%</td></tr><tr><td></td><td rowspan=1 colspan=1>Flu B LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>29 / 291</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>89 / 89 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>Flu B ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>RSV HighNegative (Cs)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td></td><td rowspan=1 colspan=1>RSV LowPositive (C95)</td><td rowspan=1 colspan=1>30 /30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>RSV ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>299 / 300 (99.7%)</td><td rowspan=1 colspan=3>303 / 303 (100.0%)</td><td rowspan=1 colspan=3>299 / 299 (100.0%)</td><td rowspan=1 colspan=1>901 / 902 (99.9%)</td><td rowspan=1 colspan=1>99.4% - 100.0%</td></tr></table>

30 Flu B Low Positive $( \mathrm { C } 9 5 )$ replicates yielded an “Assay Invalid. Repeat Assay.” result, and was not repeated.

Table 4: RSV Reproducibility Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>Ave.Ct</td><td rowspan=1 colspan=1>Ct %CV</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30 /30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31 /31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>31 /31</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>91 / 91 (100.0%)</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B LowPositive (C95)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>29 / 291</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>89 / 89 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>RSV HighNegative (C5)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>RSV LowPositive (C95)</td><td rowspan=1 colspan=1>29 / 30</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>90 / 91 (98.9%)</td><td rowspan=1 colspan=1>94.0% - 99.8%</td></tr><tr><td rowspan=1 colspan=1>RSV ModeratePositive (C100)</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>90 / 90 (100.0%)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>299 / 300 (99.7%)</td><td rowspan=1 colspan=3>303 / 303 (100.0%)</td><td rowspan=1 colspan=3>299 / 299 (100.0%)</td><td rowspan=1 colspan=1>901 / 902 (99.9%)</td><td rowspan=1 colspan=1>99.4% - 100.0%</td></tr></table>

30 Flu B Low Positive $( \mathrm { C } 9 5 )$ replicates yielded an “Assay Invalid. Repeat Assay.” result, and was not repeated.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

External Positive and Negative Controls

Positive and negative controls are provided in the cobas® Liat Influenza A/B & RSV assay Quality Control Kit. Refer to the “Device Description” section of this summary for details regarding these controls.

External positive and negative controls are required to be run during the Add cobas® Liat Influenza A/B & RSV Tube Lot procedure. Additional External positive and negative controls may be tested in accordance with local, state, federal, and/or accrediting organization requirements as applicable.

During the clinical study testing both prospective and retrospective specimens (during the 2014 to 2015 influenza season), quality control testing using the external positive and negative controls was performed per the product Instructions for Use at all three study sites. A total of 173 external positive controls and 173 external negative controls were tested during the clinical study. Two (2) external positive controls (2/173, $1 . 2 \%$ , $9 5 \%$ CI, $0 . 3 \% - 4 . 1 \% )$ and three external negative controls (3/173, $1 . 7 \%$ , $9 5 \%$ CI, $0 . 6 \% - 5 . 0 \%$ ) failed initially, but succeeded on repeat testing per the product Instructions for Use.

# Specimen Stability Study

Specimen stability was evaluated at temperatures specified in the cobas® Liat Influenza A/B & RSV

Package Insert. Influenza A, Influenza B and RSV strains at $3 \mathbf { x }$ LOD were spiked into negative nasopharyngeal swab in UTM matrix, stored at $- 8 0 \mathrm { { } ^ { \circ } C }$ and $4 ^ { \circ } \mathrm { C }$ , and tested at different time points.

General recommendations for the storage and transport of nasopharyngeal swab in UTM specimens with regard to nucleic acid-based influenza and RSV testing are to store specimens refrigerated (2- $8 ^ { \circ } \mathrm { C } )$ for up to 72 hours prior to testing. For long duration transport or storage, specimens should be frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ or colder and transported on dry ice (i.e. $- 7 8 . 5 ^ { \circ } \mathrm { C } )$ (CDC Influenza Specimen Collection, CS246972; Copan Universal Transport Media System Package Insert, 44A-Rev.07061). The cobas® Liat Influenza A/B & RSV assay Package Insert specifies the same storage and transport conditions. This specimen stability study evaluated the effect of these specimen storage temperatures on influenza and RSV detection by the cobas® Liat Influenza A/B & RSV assay.

Each temperature condition was tested using a sample panel containing a positive sample $( 3 \mathrm { x } \ L \mathrm { O D } )$ and a negative sample. The positive sample contained A/Brisbane/59/2007 at $6 . 0 \times 1 0 ^ { - 3 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L } )$ , B/Malaysia/2506/04 at $1 . 2 \times 1 0 ^ { - 2 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ , and RSV-A 2006 Isolate at $1 . 2 { \times } 1 0 ^ { 0 } \mathrm { T C I D } { \scriptstyle 5 0 } / \mathrm { m L } \nonumber$ spiked into negative NPS in UTM sample matrix at $3 \mathbf { x }$ LOD concentration. Influenza and RSV strains used were from the same source as identified in the “Detection Limit” section of this summary. The negative sample contained negative NPS in UTM sample matrix.

Five (5) positive and five negative samples were tested using the cobas® Liat Influenza A/B & RSV assay immediately after spiking. Other samples were stored at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for 30 days; and at $4 \mathrm { { } ^ { \circ } C }$ for 2, 3, and 7 days. At each time point, another five positive and five negative samples were tested using the cobas® Liat Influenza A/B & RSV assay. Samples stored at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ were thawed prior to testing.

A total of 60 runs were performed in this study. Runs were performed on 14 cobas® Liat Analyzers and used 4 lots of cobas® Liat tubes.

Table 5 below shows the results from this study. The cobas® Liat Influenza A/B & RSV assay yielded the expected result for positive (“Influenza A Detected, Influenza B Detected, RSV Detected”) and negative (“Influenza A Not Detected, Influenza B Not Detected, RSV Not Detected”) samples in all replicates tested at each time point.

Table 5: Specimen Stability Study Results   

<table><tr><td rowspan=1 colspan=1>Temp.</td><td rowspan=1 colspan=1>Time(Days)</td><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>InfA CtAve.</td><td rowspan=1 colspan=1>Inf BCtAve.</td><td rowspan=1 colspan=1>RSVCtAve.</td><td rowspan=1 colspan=1>IPCCtAve.</td><td rowspan=1 colspan=1>Inf ADetected</td><td rowspan=1 colspan=1>Inf ANotDetected</td><td rowspan=1 colspan=1>Inf BDetected</td><td rowspan=1 colspan=1>Inf BNotDetected</td><td rowspan=1 colspan=1>RSVDetected</td><td rowspan=1 colspan=1>RSVNotDetected</td></tr><tr><td rowspan=4 colspan=1>-80</td><td rowspan=2 colspan=1>0</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=2 colspan=1>30</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=8 colspan=1>4C</td><td rowspan=2 colspan=1>0</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=2 colspan=1>7</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td></tr></table>

Overall, the study results show that nasopharyngeal swab samples may be stored at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for up to 30 days and at $4 \mathrm { { } ^ { \circ } C }$ for up to 7 days prior to testing. The results support the package insert instructions that specimens may be refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ for up to 72 hours (3 days); and that, for long duration transport or storage, specimens should be frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ or colder and transported on dry ice.

# Performance using Fresh vs. Frozen Specimen

The cobas® Liat Influenza A/B & RSV assay was tested by comparing its performance using fresh and frozen specimens. One (1) influenza A (A/Brisbane/59/07), one influenza B (B/Malaysia/2506/04), and one RSV strain (RSV-A 2006 Isolate) was individually spiked into nasopharyngeal swab in UTM matrix at different viral loads, including levels near the LOD and levels reflecting the clinical sample range. Influenza and RSV strains used were from the same source as identified in the “Detection Limit” section of this summary. For each strain, at least 60 samples were tested immediately while at least another 60 samples were frozen at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for 7 days, thawed and then tested. Ten (10) negative samples were also tested fresh and after being frozen at - $8 0 \mathrm { { } ^ { \circ } C }$ for 7 days. The negative sample consisted of negative NPS in UTM sample matrix.

A total of 381 runs were performed in this study. Runs were performed on 26 cobas® Liat Analyzers using one lot of cobas® Liat tubes.

Table 6 below shows the results from the study:

Table 6: Specimen Fresh vs. Frozen Study Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>x LOD</td><td rowspan=2 colspan=1>Conc.(TCID50/mL)</td><td rowspan=1 colspan=5>Fresh</td><td rowspan=1 colspan=5>Frozen</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveCallRate</td><td rowspan=1 colspan=1>CtAvg.</td><td rowspan=1 colspan=1>CtSD</td><td rowspan=1 colspan=1>Ct%CV</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveCallRate</td><td rowspan=1 colspan=1>CtAvg.</td><td rowspan=1 colspan=1>CtSD</td><td rowspan=1 colspan=1>Ct%CV</td></tr><tr><td rowspan=5 colspan=2>InfluenzaA/Brisbane/59/07</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.0 x 10-3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.0 x 10-2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4.0 x 10-1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>4.0 x 100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>20000</td><td rowspan=1 colspan=1>4.0 x 101</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>19.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=5 colspan=2>InfluenzaB/Malaysia/2506/04</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2 x 10-2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8.0 x 10-2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.3%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>8.0 x 10-1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>8.0 x 100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>20.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>20000</td><td rowspan=1 colspan=1>8.0 x 101</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=5 colspan=2>RSV A 2006 Isolate</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2 x 100</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8.0 x 100</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>8.0 x 101</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>8.0 x 102</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>22.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>20000</td><td rowspan=1 colspan=1>8.0 x 103</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.9%</td></tr></table>

Note: Ct Average (Ave.) and Ct Standard Deviation (SD) for Negative sample is that of the IPC.

Fresh and frozen positive samples demonstrated $100 \%$ detection across all levels of viral load, including those near LOD. Ct values were also very similar between fresh and frozen samples. Fresh and frozen negative samples all demonstrated $0 \%$ detection. The results showed that freezing and thawing samples had no effect on the detection of Influenza A, Influenza B, and RSV by the cobas® Liat Influenza A/B & RSV assay.

# d. Detection Limit:

The Limit of Detection (LOD) of the cobas® Liat Influenza A/B & RSV assay was tested using three strains of Influenza A (A/Brisbane/10/2007, A/Brisbane/59/2007, and A/NY/01/2009), two strains of Influenza B (B/Malaysia/2506/04 and B/Florida/04/06), and two strains of RSV (RSV-A 2006 isolate and RSV-B Ch93(18)-18). The LOD was determined by limiting dilution studies using these titered viruses. The viruses were spiked into nasopharyngeal swab (NPS) in UTM sample matrix, and then tested using the cobas® Liat Influenza A/B & RSV assay. The LOD was determined as the lowest virus concentration that was detected $2 9 5 \%$ of the time (i.e., concentration at which at least 19 out of 20 replicates tested positive).

Virus strains (Table 7 below) were cultured and re-titered using $\mathrm { T C I D } _ { 5 0 }$ procedure by a third party laboratory before their use in LOD determination. Each characterized strain was first adjusted to the highest $\log _ { 1 0 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in UTM as diluent.

Table 7: Influenza and RSV Strains and Viral Titers Used in the LOD Study   

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Virus Type</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>1.4 x 105</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>Influenza A/H1N1</td><td rowspan=1 colspan=1>1.4 x 105</td></tr><tr><td rowspan=1 colspan=1>A/NY/01/2009</td><td rowspan=1 colspan=1>Influenza A/H1N1 pdm</td><td rowspan=1 colspan=1>7.2 x 105</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>Influenza B Yamagata Lineage</td><td rowspan=1 colspan=1>1.4 x 105</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Influenza B Victoria Lineage</td><td rowspan=1 colspan=1>2.5 x 105</td></tr><tr><td rowspan=1 colspan=1>RSV-A 2006 isolate</td><td rowspan=1 colspan=1>RSV Type A</td><td rowspan=1 colspan=1>6.6 x 106</td></tr><tr><td rowspan=1 colspan=1>RSV-B Ch93(18)-18</td><td rowspan=1 colspan=1>RSV Type B</td><td rowspan=1 colspan=1>1.3 x 106</td></tr></table>

To prepare the testing sample, the virus at different concentrations was spiked into a pooled NPS in UTM sample matrix. NPS samples were collected throughout the study period from healthy donors that did not show any flu-like symptoms. The swabs were eluted in UTM, pooled, and then tested to confirm that the matrix pool was negative for Influenza A, Influenza B and RSV. Virus was spiked into the NPS in UTM sample matrix at no more than $5 \%$ the final volume to prepare the test sample.

For each run, the spiked sample was tested using the cobas® Liat Influenza A/B & RSV assay following the Instructions for Use. Dilutions of each virus strain were first tested in triplicates. The lowest concentration at which all three replicates tested positive was treated as the tentative LOD for each strain. The LOD was then confirmed by testing at least 20 replicates with concentrations at the tentative LOD. The confirmed LOD of each strain was determined as the lowest concentration resulting in positive detection of at least 19 out of 20 replicates. If more than one replicate is detected as negative, a two-fold or higher concentration of virus was tested with 20 additional replicates. The process was iterated until the concentration at which the virus was detected in at least 19 of 20 replicates.

A total of 263 runs were performed in this study. Runs were performed on 22 cobas® Liat Analyzers using three lots of cobas® Liat assay tubes.

The LOD Study results are presented in Table 8 below:

Table 8: LOD Study Results   

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Stock VirusConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>Test VirusLODConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>PositiveCall Rate</td><td rowspan=1 colspan=1>Ct Ave.</td><td rowspan=1 colspan=1>Ct %CV</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>1.4 x 105</td><td rowspan=1 colspan=1>2.0 x 10-2</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>1.4 x 105</td><td rowspan=1 colspan=1>2.0 x 10-3</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>A/NY/01/2009</td><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>7.2 x 105</td><td rowspan=1 colspan=1>2.0 x 10-2</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>Inf B</td><td rowspan=1 colspan=1>1.4 x 105</td><td rowspan=1 colspan=1>2.0 x 10-3</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>2.1%</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Inf B</td><td rowspan=1 colspan=1>2.5 x 105</td><td rowspan=1 colspan=1>4.0 x 10-3</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>RSV-A 2006 isolate</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>6.6 x 106</td><td rowspan=1 colspan=1>4.0 x 10-1</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>RSV-B Ch93(18)-18</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1.3 x 106</td><td rowspan=1 colspan=1>4.0 x 10-1</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>2.7%</td></tr></table>

# e. Analytical Reactivity:

An analytical reactivity study was conducted evaluating the ability of the cobas® Liat Influenza A/B & RSV assay to detect influenza and RSV strains representing temporal and geographical diversity. The cobas® Liat Influenza A/B & RSV assay was evaluated for reactivity with 28 Influenza A strains, 15 Influenza B strains, and 7 RSV strains. Influenza A strains included 14 Influenza A/H1 strains (including 3 H1N1 pdm09 strains), 12 Influenza A/H3 strains (including one H3N2v strain), one Influenza A/H7N9 strain, and one Influenza A/H5N1 reassortant strain. Influenza B strains included strains from both the Victoria lineage and Yamagata lineage. RSV strains included both RSV Type A and Type B strains.

Cultured viral strains were obtained from vendors. All strains were frozen and were accompanied by Certificate of Analysis including $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ titer information. Each viral strain under test was serially diluted in 10 fold increments in UTM using the same procedure as that for the LOD study. The virus was then spiked in negative NPS in UTM sample matrix, and the spiked sample was tested using the cobas® Liat Influenza A/B & RSV assay following the Instructions for Use. Each viral strain was tested at the LOD for strains used in LOD studies, and at “near LOD” level for other strains. To estimate “near LOD” level, each strain was first tested at $1 . 0 { \times } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $1 . 0 \times 1 0 ^ { 2 }$ $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ , and then higher or lower concentrations (as appropriate) until all triplicate runs at a concentration were detected, and gave a Ct between 31 and 34. Given that the $\mathrm { C t }$ at LOD was 31-34 based on LOD study data, a Ct value in this range was used to estimate the “near LOD” level.

A total of 243 runs were performed on 20 cobas® Liat Analyzers using three lots of cobas® Liat tubes.

Table 9 below shows the strains tested for reactivity and the “near LOD” concentration of each strain that yielded 3 / 3 positive results.

Table 9: Analytical Reactivity Study Results   

<table><tr><td colspan="1" rowspan="1">Virus Strain</td><td colspan="1" rowspan="1">Type / Subtype</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Inf AResult</td><td colspan="1" rowspan="1">Inf BResult</td><td colspan="1" rowspan="1">RSVResult</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">5.0×101 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Anhui/1/2013</td><td colspan="1" rowspan="1">Influenza A/H7N9 (Eurasianlineage)</td><td colspan="1" rowspan="1">1.0×103 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">2.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">2.0×10-3 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/X0810301/2013(H5N1)-PR8-IDCDC-RG34B</td><td colspan="1" rowspan="1">Influenza A/H5N1 reassortant</td><td colspan="1" rowspan="1">2.5×101 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/FM/1/47</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/H3/Perth/16/09</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">2.5×10-1 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">1.0×102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Indiana/8/2011</td><td colspan="1" rowspan="1">Influenza A/H3N2v</td><td colspan="1" rowspan="1">5.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">4.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/MRC2</td><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×101 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/NY/01/2009</td><td colspan="1" rowspan="1">Influenza A/H1N1 pdm09</td><td colspan="1" rowspan="1">2.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">Influenza A/H1N1 pdm09</td><td colspan="1" rowspan="1">2.5×10-2 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/NY/03/2009</td><td colspan="1" rowspan="1">Influenza A/H1N1 pdm09</td><td colspan="1" rowspan="1">2.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">5.0×100 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Solomon Island/3/2006</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Swine/1976/31</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×101 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Swine/Iowa/15/30</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Texas/50/2012</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">1.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">1.0×102 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Victoria/361/2011</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">2.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">Influenza A/H1N1</td><td colspan="1" rowspan="1">1.0×103 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">Influenza A/H3N2</td><td colspan="1" rowspan="1">5.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">B/Brisbane/60/2008</td><td colspan="1" rowspan="1">Influenza B (Victoria lineage)</td><td colspan="1" rowspan="1">1.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06</td><td colspan="1" rowspan="1">Influenza B (Yamagataiineage)</td><td colspan="1" rowspan="1">2.0×10-3 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">Influenza B (Yamagatalineage)</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.0×100 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/HongKong/5/72</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.5×10-1 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.5×10-1 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">Influenza B (Victoria lineage)</td><td colspan="1" rowspan="1">4.0×10-3 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Mass/3/66</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×101 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">B/Massachusetts/2/2012</td><td colspan="1" rowspan="1">Influenza B (Yamagatalineage)</td><td colspan="1" rowspan="1">5.0×10-3 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">B/Nevada/03/2011</td><td colspan="1" rowspan="1">Influenza B (Victoria lineage)</td><td colspan="1" rowspan="1">2.5×10-1 CEID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×100 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Texas/6/2011</td><td colspan="1" rowspan="1">Influenza B (Yamagatalineage)</td><td colspan="1" rowspan="1">1.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/1/2010</td><td colspan="1" rowspan="1">Influenza B (Yamagatalineage)</td><td colspan="1" rowspan="1">5.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">RSV A 2006 isolate</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">4.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A Long</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0×102 TCID50/mL</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A2</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0×10° TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B 9320</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.0×10 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B Ch93(18)-18</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">4.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B Wash/18537</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.0×100 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B WV/14617/85</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr></table>

Data demonstrates that the cobas® Liat Influenza $\mathrm { A } / \mathrm { B }$ & RSV assay is reactive against all strains at the specified concentrations tested.

The cobas® Liat Influenza A/B & RSV assay was shown to be reactive against viral strains with titers ranging from $1 0 ^ { - 2 }$ to $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ . The observed difference in titers is most likely due to the fact that $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ determination and RT-PCR testing measure different targets. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ is the measure of infectious virus titer, or the virus required to produce a cytopathic effect in $50 \%$ of inoculated tissue culture cells. This is a measure of only live, intact virus, and takes into account the infection and cytopathtic efficiency of the virus for the host cells. On the other hand, PCR-based assay detect nucleic acids, including that from intact and non-intact virions, and free nucleic acids in solution. Because of this measurement difference, $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ can vary widely for a substantially equivalent quantity of RNA, and vice versa, resulting in the difference observed. Also, titering methods have been reported to have an inherent variability of up to one order of magnitude, which also may contribute to the observed difference (Propagation and purification of influenza virus, GE Healthcare Life Sciences,   
Application note 28-9843-41 AB,   
https://www.gelifesciences.com/gehcls_images/GELS/Related%20Content/Files/1354807438862/lit doc28984341_20130610001258.pdf).

In addition, based on in silico analysis, there is no evidence to suggest that the observed difference in analytical reactivity is principally caused by sequence mismatch between the primer/probes of the cobas® Liat Influenza A/B & RSV assay to the strains tested.

# f. Analytical Specificity:

A cross reactivity study was carried out to evaluate the cobas® Liat Influenza A/B & RSV assay’s potential cross-reactivity with microorganisms that may be present in nasopharyngeal swab samples. The cobas® Liat Influenza A/B & RSV assay was evaluated against a panel containing human genomic DNA and 35 microorganisms. Bacteria and Candida albicans were tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $\geq 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } ,$ , or the highest available concentration. For Mycobacterium tuberculosis and Mycoplasma pneumoniae, testing was performed with genomic DNA due to difficulties in propagation of these bacteria. Each microorganism was spiked into negative NPS in UTM matrix at the concentration indicated in Table 10 below. The spiked volume did not exceed $5 \%$ of the sample volume. The spiked samples were tested using the cobas® Liat Influenza A/B & RSV assay in triplicate. A control of negative NPS in UTM matrix and no microorganisms was also tested.

A total of 114 runs were performed on 13 cobas® Liat Analyzers using 6 lots of cobas® Liat assay tubes.

Table 10 below shows the cross reactivity study results:

Table 10: Cross Reactivity Study Results   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Inf A Result</td><td colspan="1" rowspan="1">Inf B Result</td><td colspan="1" rowspan="1">RSV Result</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">9.0×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">1.4×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">2.5×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus</td><td colspan="1" rowspan="1">1.4×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">8.0×103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">8.0×104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Enterovirus 68</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">7.0×103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">3.7×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">7.5×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">4.5×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus Type 1A</td><td colspan="1" rowspan="1">8.0×105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">8.0×104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">8.0×104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Varicella-Zoster Virus</td><td colspan="1" rowspan="1">4.4×103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">2.2×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">4.2×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">8.0×104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">3.6×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">2.3×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">6.7×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">2.5×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">2.8×106 copies/mL1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">2.9×106 copies/mL1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria elongate</td><td colspan="1" rowspan="1">2.0×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">2.2×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">2.3×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2.4×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.8×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.5×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">4.3×106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">1.0×104 copies/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

1 Testing was performed with genomic DNA due to difficulties in propagation of these bacteria.

The cobas® Liat Influenza A/B & RSV assay showed no cross reactivity for the human genomic DNA or the microorganisms at the concentrations tested.

# g. Potentially Interfering Substances:

The cobas® Liat Influenza A/B & RSV assay was evaluated with potentially interfering substances that may be encountered in nasopharyngeal swab specimens. Medically and/or physiologically relevant concentrations of potential interferents (Table 11 below) were tested with two influenza A strains, two influenza B strains, and two RSV strains at $3 \mathbf { x }$ LOD $( 1 0 ^ { 0 } - 1 0 ^ { - 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ .

Table 11: Interfering Substances Panel   

<table><tr><td rowspan=1 colspan=1>Potentially Interfering Substances</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Mucin: bovine submaxillary gland,type I-S</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal spray  Afrin</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal corticosteroids  Veramyst</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal gel  Zicam</td><td rowspan=1 colspan=1>Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulphur</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Throat lozenges, oral anesthetic andanalgesic  Cepacol</td><td rowspan=1 colspan=1>Benzocaine, Menthol</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment  Bactroban</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug  Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug  Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>7.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antimicrobial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/ml</td></tr></table>

The potentially interfering substances were evaluated at a concentration equivalent to: $( 1 ) { \geq } 3$ times the average concentration found in serum (or nasal secretion for mucin), (2) $5 \% \%$ for liquids or 5 $\mathrm { m g / m L }$ for solids, or (3) the highest soluble concentration in UTM.

Interfering substance testing was performed on a sample panel containing a negative sample and six positive samples (Table 12). The negative sample contained negative NPS in UTM sample matrix. The positive samples contained two influenza A strains, two influenza B strains, and two RSV strains, each spiked into negative NPS in UTM sample matrix at its $3 \mathbf { x }$ LOD concentration.

Table 12: Concentration of Influenza and RSV Strains Used in Potentially Interfering Substances Study   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>6.0×10-2</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>6.0×10-3</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>6.0×10-3</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>1.2×10-2</td></tr><tr><td rowspan=1 colspan=1>RSV-A 2006 Isolate</td><td rowspan=1 colspan=1>1.2×100</td></tr><tr><td rowspan=1 colspan=1>RSV-B Ch93(18)-18</td><td rowspan=1 colspan=1>1.2×100</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr></table>

Each interfering substance was spiked into each sample and tested using the cobas® Liat Influenza A/B & RSV assay in triplicate. Controls containing the samples and no interfering substance were also tested.

A total of 231 runs were performed on 18 cobas® Liat Analyzers using one lot of cobas® Liat tubes.

Table 13 below shows the potentially interfering substances study results:

Table 13: Potentially Interfering Substances Study Results   

<table><tr><td rowspan=1 colspan=1>Potentially</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>A/Brisbane/1017</td><td rowspan=1 colspan=2>A/Brisbane/59/07</td><td rowspan=1 colspan=2>B/Florida/04/06</td><td rowspan=1 colspan=2>B/Malaysia/2506/04</td><td rowspan=1 colspan=2>RSV A 2006Isolate</td><td rowspan=1 colspan=2>RSV BCh93(18)-18</td><td rowspan=1 colspan=2>Negative</td></tr><tr><td rowspan=1 colspan=1>InerferingSubstances</td><td rowspan=1 colspan=1>Conc.</td><td rowspan=1 colspan=1>Inf ADetected</td><td rowspan=1 colspan=1>Inf ACt.Ave.</td><td rowspan=1 colspan=1>Inf ADetected</td><td rowspan=1 colspan=1>Inf ACt.Ave.</td><td rowspan=1 colspan=1>Inf BDetected</td><td rowspan=1 colspan=1>Inf BCt.Ave.</td><td rowspan=1 colspan=1>Inf BDetected</td><td rowspan=1 colspan=1>Inf BCt.Ave.</td><td rowspan=1 colspan=1>RSVDetected</td><td rowspan=1 colspan=1>RSVCt.Ave.</td><td rowspan=1 colspan=1>RSVDetected</td><td rowspan=1 colspan=1>RSV Ct.Ave.</td><td rowspan=1 colspan=1>Inf A NDInf B NDRSV ND</td><td rowspan=1 colspan=1>IPCCt.Ave.</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.5</td></tr><tr><td rowspan=1 colspan=1>Afrin</td><td rowspan=1 colspan=1>5%(v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.5</td></tr><tr><td rowspan=1 colspan=1>Bactroban</td><td rowspan=1 colspan=1>5mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.8</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>5%(v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td></tr><tr><td rowspan=1 colspan=1>Cepacol</td><td rowspan=1 colspan=1>5mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.5</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>5mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.4</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>5mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>Tamiflu</td><td rowspan=1 colspan=1>7.5mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.8</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4ug/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.5</td></tr><tr><td rowspan=1 colspan=1>Veramyst</td><td rowspan=1 colspan=1>5%(v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.6</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>5%(v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.2</td></tr></table>

Results show that substances at the concentrations tested did not interfere in the detection of influenza A, influenza B and RSV strains.

# h. Potential Microbial Interference Study:

A potentially interfering microorganism study was conducted to evaluate whether microorganisms that may be present in nasopharyngeal swab samples can interfere in the detection of Influenza A, Influenza B or RSV by the cobas® Liat Influenza $\mathrm { A } / \mathrm { B }$ & RSV assay. The panel of human genomic DNA and 35 microorganisms tested in the cross-reactivity study was tested for potential interference. Bacteria and Candida albicans were tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $\ge 1 0 ^ { 5 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ or the highest available concentration, in the presence of one Influenza A strain, one Influenza B strain and one RSV strain at $3 \mathbf { x }$ LOD concentrations in negative nasopharyngeal swab in UTM matrix. For Mycobacterium tuberculosis and Mycoplasma pneumoniae, testing was performed with genomic DNA due to difficulties in propagation of these bacteria.

Interfering microorganism testing was performing using a positive sample comprised of A/Brisbane/59/2007 at $6 . 0 \times 1 0 ^ { - 3 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ , B/Malaysia/2506/04 at $1 . \bar { 2 } \times 1 0 ^ { - 2 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ , and RSV-A 2006 Isolate at $1 . 2 { \times } 1 0 ^ { 0 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ , spiked into negative NPS in UTM sample matrix (3x LOD concentration). Each potentially interfering microorganism was spiked into each sample at the concentration indicated in Table 14. The spiked volume did not exceed $5 \%$ of the sample volume. The spiked samples were tested using the cobas® Liat Influenza $\mathrm { A } / \mathrm { B }$ & RSV assay in triplicate. A control comprised of the positive sample and no interfering microorganism was also tested.

A total of 111 runs were performed on 12 cobas® Liat Analyzers using three lots of cobas® Liat assay tubes

Table 14: Potentially Interfering Microorganisms Study Results   

<table><tr><td rowspan=2 colspan=1>Microorganism</td><td rowspan=2 colspan=1>Test Concentration</td><td rowspan=1 colspan=3>One Flu A, One Flu B &amp; One RSV strain at 3x LOD</td></tr><tr><td rowspan=1 colspan=1>Inf A Result</td><td rowspan=1 colspan=1>Inf B Result</td><td rowspan=1 colspan=1>RSV Result</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>9.0×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7</td><td rowspan=1 colspan=1>1.4×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>4.5×104 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>2.5×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus</td><td rowspan=1 colspan=1>1.4×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus 229E</td><td rowspan=1 colspan=1>8.0×103 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus OC43</td><td rowspan=1 colspan=1>8.0×104 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Enterovirus 68</td><td rowspan=1 colspan=1>1.0×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>7.0×103 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 1</td><td rowspan=1 colspan=1>3.7×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 2</td><td rowspan=1 colspan=1>7.5×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 3</td><td rowspan=1 colspan=1>4.5×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus Type 1A</td><td rowspan=1 colspan=1>8.0×105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>8.0×104 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>8.0×104 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Varicella-Zoster Virus</td><td rowspan=1 colspan=1>4.4×103 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>2.2×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>4.2×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>8.0×104 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium sp</td><td rowspan=1 colspan=1>3.6×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.9×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>2.3×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus sp</td><td rowspan=1 colspan=1>1.9×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>6.7×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>2.5×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>2.8×106 copies/mL1</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2.9×106 copies/mL1</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>2.0×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>2.2×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2.3×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>2.4×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.9×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1.8×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>2.5×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>4.3×106 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>1.0×104 copies/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr></table>

1 Testing was performed with genomic DNA due to difficulties in propagation of these bacteria.

Results show that the presence of human genomic DNA or the microorganisms at the concentrations tested did not interfere with the detection of Influenza A, Influenza B, or RSV.

i. Carry-Over Study:

A study was conducted to demonstrate that the single-use, self-contained cobas® Liat assay tube and the cobas® Liat System are not susceptible to carry-over contamination when alternating high positive and negative samples are tested. High positive samples consisted of an influenza A virus spiked into negative nasopharyngeal swab in UTM matrix at a concentration greater than that expected in $9 5 \%$ or more of specimens of diseased patients in the intended use population. In clinical studies, the lowest $\mathrm { C t }$ values of Influenza A detected specimens, which corresponds to the highest concentration specimens, was approcimately12 to 13. The highest titer available for Influenza A/Brisbane/59/07 allowed for a test concentration of $7 . 1 \times 1 0 ^ { 3 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ . This test concentration gave a Ct of appromately12.7. To test the worst case scenario, this highest available concentration of Influenza A was used as the high positive sample concentration. Negative samples contained negative nasopharyngeal swab in UTM matrix.

Twenty (20) high positive and 20 negative samples were tested on each of two cobas® Liat Systems (i.e. 40 high positive and 40 negative samples in total). High positive and negative samples were run alternating analyzer-to-analyzer and run-to-run. All 40 high positive samples tested were correctly reported as “Influenza A Detected”. All 40 negative samples tested were correctly reported as “Influenza A Not Detected”. There was no carry-over or cross contamination observed during this study.

# j. Assay cut-off:

The assay cutoff for each analyte in terms of cycle threshold $\mathrm { ( C t ) }$ was determined based on two considerations: (1) the Ct cutoff should be greater than the latest Ct values of positive runs at the LOD of the virus strains tested to ensure analytical sensitivity; and (2) the Ct cutoff should be early enough to minimize the risk of false positive detection to ensure the specificity for target detection.

For the Influenza A, A/Brisbane/59/07 had the latest Ct in the LOD studies, generating an average LOD Ct of 34.1 with a standard deviation of 0.97. As such, the latest expected Ct value of positive runs at the LOD was calculated to be 36.7. The earliest Ct for negative samples was 37.2. Therefore, the Ct cutoff for Influenza A was set at 37.0.

Similarly, for Influenza B, B/Florida/04/06 had the latest Ct in LOD studies, generating an average LOD Ct of 33.7 with a standard deviation of 0.72. As such, the latest expected Ct value of positive runs at the LOD was calculated to be 35.6. The earliest Ct for negative samples was 42.0. Therefore, the Ct cutoff for Influenza B was set at 38.5.

For RSV, RSV-B Ch 93(18)-18 had the latest Ct in LOD studies, generating an average LOD Ct of 33.0 with a standard deviation of 0.90. As such, the latest expected Ct value of positive runs at the LOD was calculated to be 35.4. The earliest Ct for negative samples was 37.7. Therefore, the Ct cutoff for RSV was set at 37.3.

These Ct cut-offs were further verified in clinical studies.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Performance of the cobas® Liat Influenza A/B & RSV assay was evaluated against the comparator method of an FDA-cleared nucleic acid test in a prospective clinical study.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Study:

Clinical performance characteristics of the cobas® Liat Influenza A/B & RSV assay were evaluated in multi-site prospective studies during the 2013-2014 and the 2014-2015 influenza seasons in the U.S. A total of 12 investigational sites (CLIA waived sites) throughout the U.S. participated in the study. To be enrolled in the study, patients had to be presenting at the participating study sites with flu-like symptoms, and provided informed consents.

Nasopharyngeal swab (NPS) samples were prospectively collected and tested at 6 of the 12 sites in the 2013-2014 influenza season from January 2014 to May 2014, and at 9 of the 12 sites in the 2014- 2015 influenza season from October 2014 to April 2014. Three (3) of the 12 sites participated in the study during both time periods.

A FDA-cleared multiplexed nucleic acid amplification test for Influenza A, B, and RSV was performed by a central reference laboratory as the comparator method for this study.

One NPS sample, placed into UTM after specimen collection, was used for the cobas® Liat Influenza A/B & RSV assay testing according to the product instructions for use. The remaining sample in UTM was labeled and stored frozen at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for later shipment to the central reference laboratory for comparator testing.

PCR and bi-directional sequencing were used to investigate discordant results between the cobas® Liat Influenza A/B & RSV assay and the comparator test. Nested PCR was used to increase the sensitivity of the PCR/sequencing method which uses primers that are distinguished from those used in the cobas® Liat Influenza A/B & RSV assay.

A total of 1421 nasopharyngeal swab (NPS) specimens eluted in VTM were enrolled in this study. Of those, 41 specimens did not meet eligibility criteria. Additionally, 17 and 13 specimens were excluded due to invalid results from the cobas® Liat Influenza A/B & RSV assay and the comparator assay, respectively. A total of 1350 prospective NPS specimens were included in the performance analysis. Patient age and gender distribution for all the 1421 prospective specimens is presented in Table 15 below.

Table 15: Patient Demographics – Prospective Study   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>346</td><td rowspan=1 colspan=1>24.3%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>428</td><td rowspan=1 colspan=1>30.1%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>514</td><td rowspan=1 colspan=1>36.2%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>9.4%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1421</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>692</td><td rowspan=1 colspan=1>48.7%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>51.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>636</td><td rowspan=1 colspan=1>100%</td></tr></table>

Compared to the comparator test, the performance of the cobas® Liat Influenza A/B & RSV assay for influenza A, influenza B and RSV are presented in Table 16, Table 17, and Table 18 below, respectively:

Table 16: Influenza A Performance – Prospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>Flu A (Prospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>47a</td><td rowspan=1 colspan=1>219</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1128</td><td rowspan=1 colspan=1>1131</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>1175</td><td rowspan=1 colspan=1>1350</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 98.3% (172/175), 95%CI: 95.1% to 99.4%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 96.0% (1128/1175), 95%CI: 94.7% to 97.0%</td></tr></table>

a 41 Liat positive, comparator negative specimens were tested by PCR/sequencing. Of these, 18 were positive and 23 were negative by PCR/sequencing.

Table 17: Influenza B Performance – Prospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>Flu B (Prospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>8 a</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1300</td><td rowspan=1 colspan=1>1302</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>1308</td><td rowspan=1 colspan=1>1350</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 95.2% (40/42), 95%CI: 84.2% to 98.7%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 99.4% (1300/1308), 95%CI: 98.8% - 99.7%</td></tr></table>

a 6 Liat positive, comparator negative specimens were tested by PCR/sequencing. Of these, 5 were positive and 1 was negative by PCR/sequencing.

Table 18: RSV Performance – Prospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>RSV (Prospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>16 a</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1235</td><td rowspan=1 colspan=1>1238</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>1251</td><td rowspan=1 colspan=1>1350</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 97.0% (96/99), 95%CI: 91.5% to 99.0%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 98.7% (1235/1251), 95%CI: 97.9% to 99.2%</td></tr></table>

a 15 Liat positive, comparator negative specimens were tested by PCR/sequencing. Of these, 3 were positive and 12 were negative by PCR/sequencing.

Compared to the comparator test, the performance of the cobas® Liat Influenza A/B & RSV assay for influenza A, influenza B and RSV in the prospective study was also analyzed by study site and by patient age group. The cobas® Liat Influenza A/B & RSV assay demonstrated similar performance for influenza A, influenza B and RSV across study sites and patient age groups.

The Liat Influenza A/B & RSV assay detected six mixed influenza A, B, and RSV infections in this prospective clinical evaluation. One (1) sample tested positive for influenza A and influenza B by the Liat Influenza A/B & RSV assay, while the comparator assay and/or PCR-sequencing results were negative for influenza A and influenza B. Four (4) samples tested positive for influenza A and RSV, while the comparator assay was positive for influenza A and RSV for two of these samples, positive for influenza A and negative for RSV for one sample, and negative for influenza A and positive for RSV for one sample; PCR-sequencing of the two discrepant samples yielded negative results. One (1) sample tested positive for influenza B and RSV, and the comparator assay result was also positive for influenza B and RSV. No samples tested positive for influenza A, B, and RSV.

The following statement is included in the “Reasons to Repeat the Assay” section of the Product

Insert: “Dual infections of Influenza A and Influenza B are rare. If the test result is “Influenza A Detected” and “Influenza B Detected”, the assay should be repeated with the same patient specimen, or if possible, with a newly collected specimen. Specimens that have repeat “Influenza A Detected” and “Influenza B Detected” results should be sent to a laboratory for confirmatory testing.”

# b. Retrospective Study:

In addition to the prospective clinical study, retrospective NPS samples were obtained from two reference laboratories. Specimens obtained were collected in the 2013 – 2014 influenza season. All samples were archived at $- 8 0 \mathrm { { } ^ { \circ } C }$ upon receipt at IQuum. A total of 300 selected retrospective samples were distributed to and tested at 3 of the 12 sites. The retrospective samples were worked into the daily workload of those sites for testing.

The same FDA-cleared multiplexed nucleic acid amplification test for Influenza A, B and RSV that was used in the prospective study was also the comparator test for this study.

The same PCR and bi-directional sequencing method that was used in the prospective study for investigating discordances between the cobas® Liat Influenza A/B & RSV assay and the comparator method was also utilized in this study.

Of the 300 enrolled NPS specimens, no cobas® Liat Influenza A/B & RSV assay result was obtained for five specimens due to invalid results. These specimens were excluded from the data analysis, but are reported as a part of the invalid rate. Three (3) specimens had an invalid comparator result (i.e. target and internal control not detected), and were removed from the performance analyses.

As such, 292 retrospective NPS specimens were available for performance analyses.

Compared to the comparator test, the performance of the cobas® Liat Influenza A/B & RSV assay for influenza A, influenza B and RSV in this retrospective study are presented in Table 19, Table 20, and Table 21 below, respectively:

Table 19: Influenza A Performance – Retrospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>Flu A (Retrospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>2 a</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>292</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 98.7% (76/77), 95%CI: 93.0% to 99.8%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 99.1% (213/215), 95%CI: 96.7% to 99.7%</td></tr></table>

a 1 Liat positive, comparator negative specimen was tested by PCR/sequencing. This sample was negative by PCR/sequencing.

Table 20: Influenza B Performance – Retrospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>Flu B (Retrospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>194</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>292</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 99.0% (97/98), 95%CI: 94.4% to 99.8%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 99.5% (193/194), 95%CI: 97.1% to 99.9%</td></tr></table>

Table 21: RSV Performance – Retrospective NPS Specimens   

<table><tr><td rowspan=1 colspan=1>RSV (Retrospective)</td><td rowspan=1 colspan=1>Comparator Positive</td><td rowspan=1 colspan=1>Comparator Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Liat Positive</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>7 a</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Liat Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>202</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>292</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement (PPA) = 98.8% (83/84), 95%CI: 93.6% to 99.8%</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement (NPA) = 96.6% (201/208), 95%CI: 93.2% to 98.4%</td></tr></table>

a 6 Liat positive, comparator negative specimens were tested by PCR/sequencing. Of these, all 6 were positive by PCR/sequencing.

Compared to the comparator test, the performance of the cobas® Liat Influenza A/B & RSV assay for influenza A, influenza B and RSV in the retrospective study was also analyzed by study site. The cobas® Liat Influenza A/B & RSV assay demonstrated similar performance for influenza A, influenza B and RSV across study sites.

The Liat Influenza A/B & RSV assay detected two mixed influenza A, B, and RSV infections in retrospective clinical evaluation. Both samples tested positive for influenza B and RSV by the Liat Influenza A/B & RSV assay, while the comparator assay was positive for influenza B and RSV for one of these samples, and positive for influenza B and negative for RSV for one sample; PCR-sequencing of the discrepant sample yielded positive result for RSV.

c. cobas® Liat Influenza A/B & RSV Invalid Rate Analysis:

During the clinical study testing prospective and retrospective clinical NPS in UTM specimens, the cobas® Liat Influenza A/B & RSV assay initial invalid rate was $1 . 8 \%$ (29/1,656 specimens, $9 5 \%$ CI: $1 . 2 \% - 2 . 5 \%$ ). Of these 29 specimens with initial invalid results, 5 specimens had 2 invalid or aborted runs, 16 specimens had 1 invalid run and were not repeated due to unavailability of residual samples, and 8 specimens had an initial invalid run and a repeat test per product instructions for use yielded a valid result.

4. Clinical cut-off:

Not applicable.

5. Expected values:

In the cobas® Liat Influenza A/B & RSV assay prospective clinical study (described in the “Clinical Studies” section above), a total of 313 NPS specimens were determined to be evaluable during the 2013- 2014 influenza season from January 2014 to May 2014. The number and percentage of influenza A, influenza B and RSV positive cases per specified age group, as determined by the cobas® Liat Influenza A/B & RSV assay, are presented in Table 22, Table 23 and Table 24 below, respectively:

Table 22: Influenza A Positives by the cobas® Liat Influenza A/B & RSV per Patient Age Group - 2013 to 2014 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza APositives</td><td rowspan=1 colspan=1>Influenza A Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.7%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>8.1%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.7%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11.1%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>6.1%</td></tr></table>

Table 23: Influenza B Positives by cobas® Liat Influenza A/B & RSV per Patient Age Group - 2013 to 2014 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza BPositives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>8.9%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11.1%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6.4%</td></tr></table>

Table 24: RSV Positives by cobas® Liat Influenza A/B & RSV per Patient Age Group - 2013 to 2014 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza BPositives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5.9%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.7%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>4.2%</td></tr></table>

A total of $1 0 4 8 \mathrm { N P S }$ specimens were determined to be evaluable during the 2014-2015 influenza season from October 2014 to April 2015. The number and percentage of influenza A, influenza B and RSV positive cases per specified age group, as determined by the cobas® Liat Influenza A/B & RSV assay, are presented in Table 25, Table 26, and Table 27 below, respectively:

Table 25: Influenza A Positives by the cobas® Liat Influenza A/B & RSV per Patient Age Group - 2014 to 2015 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza APositives</td><td rowspan=1 colspan=1>Influenza A Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>11.4%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>27.5%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>16.5%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>24.4%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1048</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>19.5%</td></tr></table>

Table 26: Influenza B Positives by cobas® Liat Influenza A/B & RSV per Patient Age Group - 2014 to 2015 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza BPositives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3.9%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1048</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2.8%</td></tr></table>

Table 27: RSV Positives by cobas® Liat Influenza A/B & RSV per Patient Age Group - 2014 to 2015 Influenza Season   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number of Influenza BPositives</td><td rowspan=1 colspan=1>Influenza B Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>29.2%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1048</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>9.5%</td></tr></table>

# N. Instrument Name:

cobas® Liat System

# O. System Descriptions:

1. Modes of Operation:

The cobas® Liat System functions as a point-of-care platform with sample-to-answer capabilities in which all sample processing steps as well as detection are carried out using a single-use, disposable cobas® Liat Tube.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_ __ or No

3. Specimen Identification:

A sample barcode is scanned or a sample ID entered into the cobas® Liat System during the assay run process. After scanning the sample barcode, the corresponding sample is loaded directly into a cobas® Liat Tube using a transfer pipette. After the tube is capped, the tube barcodes is scanned by the cobas® Liat System and the tube is inserted into the system to start the test.

4. Specimen Sampling and Handling:

Specimen sampling and handling during the assay is controlled automatically using multiple sample processing modules contained within the cobas® Liat System. The sample processing modules are composed of two assemblies, a moving side assembly comprised of multiple sample processing plungers and clamps and a fixed side assembly. When performing an assay, a cobas® Liat Tube is inserted into the tube slot of a cobas® Liat System. The plungers and clamps selectively compress the cobas® Liat Tube segments against the fixed side assembly to release reagents from the segments, move the sample from one segment to another, and control reaction conditions.

5. Calibration:

Not required. The cobas® Liat Tube is single use component of a closed system.

6. Quality Control: Quality control is addressed for each specific FDA-cleared assay to be run on the instrument.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.