# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

# C. Measurand:

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1700: Culture medium for antimicrobial susceptibility tests.

2. Classification: Class II

3. Product code: JSO: Culture Media, Antimicrobial Susceptibility Test, Excluding Mueller Hinton Agar

4. Panel:

83: Microbiology

# H. Intended Use:

1. Intended use(s):

HardyCHROM ESBL is a selective and differential chromogenic medium which is intended for the qualitative and presumptive detection from stool specimens of: 1) Enterobacteriaceae that are potentially non-susceptible to ceftazidime and cefpodoxime; and 2) Extended-spectrum beta-lactamase (ESBL)-producing Escherichia coli, Klebsiella pneumoniae and Klebsiella oxytoca.

The test is performed on stool specimens from patients at risk of harboring Enterobacteriaceae that are non-susceptible to ${ \bf { \dot { 3 } } ^ { r d } }$ generation cephalosporins or ESBLproducing $E .$ . coli, $K .$ . pneumoniae and $K .$ . oxytoca, and is intended as an aid in the detection, identification of colonization and control of these bacteria in a healthcare setting. HardyCHROM ESBL is not intended to diagnose infection or to guide or monitor treatment for infections. Results can be interpreted after incubation for 18-24 hours. Subculture to non-selective medium is required for confirming identification, antimicrobial susceptibility testing and epidemiological typing.

A lack of growth or the absence of pink, blue or yellow/gold colonies on HardyCHROM ESBL does not preclude the presence of Enterobacteriaceae that are non-susceptible to $3 ^ { \mathrm { r d } }$ generation cephalosporins or ESBL producing organisms.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only.

Do not incubate plates in a $\mathrm { C O } _ { 2 }$ atmosphere.

Cultures on HardyCHROM ESBL should be incubated at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ for 18-24 hours. Analytical and clinical studies showed reduced specificity with cultures incubated longer than 24 hours. Do not incubate more than 24 hours.

A lack of growth or the absence of pink, blue or yellow/gold colonies on HardyCHROM ESBL does not preclude the presence of Enterobacteriaceae that are non-susceptible to $3 ^ { \mathrm { r d } }$ generation cephalosporins.

It is important to subculture isolated colonies to non-selective medium to confirm the identity and susceptibility of pink or blue colonies from HardyCHROM ESBL when screening for $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae.

A lack of growth or the absence of pink or blue colonies on HardyCHROM ESBL does not preclude the presence of ESBL-producing organisms.

It is important to subculture isolated colonies to non-selective medium to confirm the identity, susceptibility and ESBL-phenotype of pink or blue colonies from HardyCHROM ESBL when screening for ESBL-producing microorganisms.

4. Special instrument requirements:

None

# I. Device Description:

HardyCHROM ESBL is a selective and differential chromogenic medium that contains extended-spectrum $\beta$ -lactam antimicrobial agents. The medium is intended for use in screening of stool specimens for:

1) Detection of $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae; 2) Detection and differentiation of strains of $E$ . coli and $K .$ . pneumoniae/K. oxytoca that produce an Extended-Spectrum $\beta$ -Lactamase (ESBL).

All isolates must be subcultured for confirmatory identification, antimicrobial susceptibility testing and, if required, epidemiological typing.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Bio-Rad VRESelect   
2. Predicate 510(k) number(s): K122187

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceHardyCHROM ESBLK160512</td><td colspan="1" rowspan="1">PredicateBio-Rad VRESelectK122187</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">HardyCHROM ESBL is aselective and differentialchromogenic medium which isintended for the qualitative andpresumptive detection from stoolspecimens of: 1)Enterobacteriaceae that arepotentially non-susceptible toceftazidime and cefpodoxime;and 2) Extended-spectrum beta-lactamase (ESBL)-producingEscherichia coli, Klebsiellapneumoniae and Klebsiellaoxytoca.The test is performed on stoolspecimens from patients at riskof harboring Enterobacteriaceaethat are non-susceptible to 3rdgeneration cephalosporins orESBL-producing E. coli, K.pneumoniae and K. oxytoca, andis intended as an aid in thedetection, identification ofcolonization and control of thesebacteria in a healthcare setting.HardyCHROM ESBL is notintended to diagnose infection orto guide or monitor treatment forinfections. Results can beinterpreted after incubation for18-24 hours. Subculture to non-selective medium is required forconfirming identification,antimicrobial susceptibilitytesting and epidemiologicaltyping.A lack of growth or the absenceof pink, blue or yellow/goldcolonies on HardyCHROMESBL does not preclude thepresence of Enterobacteriaceaethat are non-susceptible to rgeneration cephalosporins orESBL producing organisms.</td><td colspan="1" rowspan="1">VRESelect is a selective anddifferential chromogenicmedium, containing 8 μg/mL ofvancomycin, for the qualitativedetection of gastrointestinalcolonization of vancomycin-resistant Enterococcus faecium(VREfm) and vancomycin-resistant Enterococcus faecalis(VREfs) and to aid in theprevention and control ofvancomycin-resistantEnterococcus (VRE) inhealthcare settings. The test isperformed on rectal swabs, orfecal specimens from patients tobe screened for VREcolonization. VRESelect is notintended to diagnose VREinfection nor to guide or monitortreatment of infection. Resultscan be interpreted after 24 to 28hours incubation. Subculture tonon-selective media (e.g.,Trypticase Soy Agar with 5%sheep blood) is needed forfurther identification,susceptibility testing andepidemiological typing.</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Bacterial growth with enzymaticmetabolism of chromogenicsubstrate</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Stool</td><td colspan="1" rowspan="1">Rectal swab or stool</td></tr><tr><td colspan="1" rowspan="1">Inoculation</td><td colspan="1" rowspan="1">Direct</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretation</td><td colspan="1" rowspan="1">Manual, visual</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Subculture</td><td colspan="1" rowspan="1">Required for confirmation ofidentity and antimicrobialsusceptibility testing</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceHardyCHROM ESBLK160512</td><td rowspan=1 colspan=1>PredicateBio-Rad VRESelectK122718</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>Cephalosporin non-susceptibility:3rd generation cephalosporinnon-susceptibleEnterobacteriaceaeESBL:ESBL-producing strains of E.coli, K. pneumoniae and K.oxytoca</td><td rowspan=1 colspan=1>Vancomycin resistant E. faecalisand E. faecium</td></tr><tr><td rowspan=1 colspan=1>Reference Method</td><td rowspan=1 colspan=1>Growth in Trypticase Soyenrichment broth withceftazidime or cefotaxime,followed by subculture toMacConkey agar withbiochemical identification andantimicrobial susceptibilitytesting of isolated colonies</td><td rowspan=1 colspan=1>Growth on Bile Esculin AzideAgar with 6µg/mL Vancomycinfollowed by biochemicalidentification and vancomycinMIC</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI. Performance Standards for Antimicrobial Susceptibility Testing. $2 5 ^ { \mathrm { t h } }$ ed. CLSI supplement M100S. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.

# L. Test Principle:

HardyCHROM ESBL is a selective and differential medium for the detection of bacteria belonging to the Enterobacteriaceae family that are non-susceptible to $3 ^ { \mathrm { r d } }$ generation cephalosporins, including those that produce Extended Spectrum $\beta$ -Lactamases.

The selective components in HardyCHROM ESBL culture medium are designed to inhibit the growth of yeasts and gram-positive bacteria, as well as gram-negative bacteria that are sensitive to $3 ^ { \mathrm { r d } }$ generation cephalosporins. Organisms that grow on HardyCHROM ESBL are differentiated based on the metabolism of chromogenic substrates. ESBL-producing strains of $K .$ . pneumoniae and $K .$ . oxytoca exhibit blue colonies, whereas those of $E$ . coli appear pink.

Colonies of Enterobacteriaceae that are non-susceptible to $3 ^ { \mathrm { r d } }$ generation cephalosporins may appear pink, blue or yellow/gold.

Testing is performed by direct inoculation of the stool onto the culture medium. Cultures are read after incubation of the plates for 18-24 hours at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ in an aerobic atmosphere. All presumptive cephalosporin non-susceptible and ESBL-producing isolates must be subcultured for identification and antimicrobial susceptibility testing according to established methods.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of results obtained with HardyCHROM ESBL was evaluated in a study conducted at 3 sites using blinded panels comprised of duplicate suspensions of 10 strains of bacteria in transport medium. The panel members included five $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible strains of Enterobacteriaceae $E$ . coli (2), Enterobacter cloacae (1), Klebsiella pneumoniae (1), and Klebsiella oxytoca (1) at $1 0 ^ { 3 } \mathrm { C F U / m L }$ ) and five susceptible strains of bacteria (1 each of $E .$ . coli, Staphylococcus aureus, Staphylococcus epidermidis, Shigella flexneri and Shigella sonnei at $1 0 ^ { 6 } \mathrm { C F U / m L }$ ). The bacterial suspensions were prepared and shipped overnight to the test sites on ice prior to each day of testing. At the test sites, a $1 0 \mu \mathrm { L }$ loop of each bacterial suspension was used to inoculate a plate of HardyCHROM ESBL culture medium, which was then incubated for 24 hours at $3 5 \mathrm { { ^ \circ C } }$ prior to being read by 2 independent operators. Cultures were inoculated on each of 5 days at each site (2 replicates x 5 days $_ { \textrm { X 2 } }$ operators x 3 sites $= 6 0$ results per strain). The results of the study are summarized in Tables 1 and 2 according to the respective criteria for detection of $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae and ESBL-producing $E$ . coli, $K .$ . pneumoniae/K. oxytoca.

Table 1. Reproducibility Study for detection of $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae, stratified by day   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Positive Agreement</td><td rowspan=1 colspan=3>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HardyCHROMPositive</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HardyCHROMNegative</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>40 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>40 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Day 5</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>267</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>99.6</td></tr></table>

Expected colony colors: Pink, Blue or Yellow/Gold 1 8 samples at Site 1 (4 from each operator) were excluded due to an error in preparation $^ 2 2 0$ positive and 20 negative samples excluded from analysis due to control failure at Site 3 Note: No $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible organisms expected to produce yellow/gold colonies were included in the study

Table 2. Summary of the results of the Reproducibility Study for detection of ESBLproducing $E .$ . coli and $K .$ . pneumoniae/K. oxytoca, stratified by day   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Positive Agreement</td><td rowspan=1 colspan=3>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HardyCHROMPositive</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HardyCHROMNegative</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>32 2</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>48 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>Day 5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>336</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>83.3</td></tr></table>

Expected colony colors: $E$ . coli: Pink; $K .$ . pneumoniae/K. oxytoca: Blue 1 8 samples at Site 1 (4 from each operator) were excluded due to an error in preparation $^ 2 1 6$ positive and 24 negative samples excluded from analysis due to control failure at Site 3 Note: Samples that contained $E$ . cloacae and which exhibited blue colonies on HardyCHROM ESBL were considered “false positive” for ESBL-producing $E$ . coli and $K .$ . pneumoniae/K. oxytoca. If these samples are omitted from the analysis, negative agreement was 279/280 $( 9 9 . 6 \% )$ .

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

External Controls External control strains of confirmed ESBL phenotype were tested on day of the Clinical Study as described in Section M(3), below. Instructions for appropriate testing of External Controls are included in the Package Insert.

Table 3. Recommended strains for use as External Controls and expected results   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Phenotype</td><td rowspan=1 colspan=1>ATCC No.</td><td rowspan=1 colspan=1>Expected Result</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ESBL Negative</td><td rowspan=1 colspan=1>25922</td><td rowspan=1 colspan=1>No growth</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ESBL Positive</td><td rowspan=1 colspan=1>51983</td><td rowspan=1 colspan=1>Pink Colonies</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ESBL Positive</td><td rowspan=1 colspan=1>51983</td><td rowspan=1 colspan=1>Blue Colonies</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ESBL Positive</td><td rowspan=1 colspan=1>700603</td><td rowspan=1 colspan=1>Blue Colonies</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>3dgenerationcephalosporin non-susceptible</td><td rowspan=1 colspan=1>BAA-856</td><td rowspan=1 colspan=1>Yellow/Gold Colonies</td></tr></table>

ATCC: American Type Culture Collection

# Specimen Stability

A study was conducted to determine the acceptable duration of storage of stool specimens prior to inoculation on HardyCHROM ESBL. Testing was performed with “raw”, unpreserved stool and stool in Culture & Sensitivity (C&S) Medium Transport. One strain each of ESBL-producing and $3 ^ { \mathrm { r d } }$ generation cephalosporin nonsusceptible $E .$ . coli, $K .$ . pneumoniae and $K .$ . oxytoca was included in the study. Each species was used to seed 3 ESBL-negative stool specimens (raw or in transport medium) which were then then stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ or ambient temperature $( 2 3 - 2 4 ^ { \circ } \mathrm { C } )$ . At specified intervals, samples of each stool specimen were inoculated onto HardyCHROM ESBL. Acceptable results were defined as recovery of the target organism with ${ < } 1 \log _ { 1 0 }$ decrease in colony counts compared to baseline. The acceptance criteria were met for all 3 species in raw, unpreserved stool after storage for up to 4 hours at ambient temperature or up to 168 hours (7 days) at $2 { - } 8 ^ { \circ } \mathrm { C }$ . For stool specimens diluted in C&S Medium, the acceptance criteria were met after storage for up to 24 hours at ambient temperature and up to 7 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ . Appropriate storage conditions for stool specimens prior to plating on HardyCHROM ESBL are included in the Package Insert.

# d. Detection limit:

# Limit of Detection

The analytical sensitivity of HardyCHROM ESBL was determined for two strains of each of E. coli, $K .$ . pneumoniae, $K .$ . oxytoca and $E .$ . cloacae (Table 4). Suspensions of each organism were diluted in stool matrix and plated on HardyCHROM ESBL. Parallel dilutions were also prepared in Tryptic Soy Broth/saline and plated on blood agar. All culture plates were incubated aerobically at $3 5 \mathrm { { ^ \circ C } }$ and read after 24 hours. The LOD was estimated based on the highest dilution of the parental bacterial suspension at which growth was visible on the HardyCHROM medium and assuming a standard inoculum of $1 0 \mu \mathrm { L }$ stool per plate. The estimated LOD was verified by testing 5 additional replicates of each bacterial strain at the LOD target level. For each strain the LOD was estimated to be $1 0 ^ { 3 } \mathrm { C F U / m L }$ stool (10 CFU/plate).

Table 4. Bacterial strains used to determine the analytical LOD of HardyCHROM ESBL   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Strain 1</td><td rowspan=2 colspan=1>β-lactamase Genotype</td><td rowspan=1 colspan=3>AntimicrobialSusceptibility</td><td rowspan=2 colspan=1>EBSLPheenotype2</td><td rowspan=2 colspan=1>3rd GenerationCephalosporinSusceptibility 3</td></tr><tr><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>TZ</td><td rowspan=1 colspan=1>CPD</td></tr><tr><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>856282</td><td rowspan=1 colspan=1>SHV-12(2be); TEM1(2b)</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>871550</td><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>900691</td><td rowspan=1 colspan=1>CTX-M-8</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>ESBL+</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>937062</td><td rowspan=1 colspan=1>TEM-19(2be)</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>Non-ESBL</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>846222</td><td rowspan=1 colspan=1>TEM-52(2be)</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>ESBL+</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>870581</td><td rowspan=1 colspan=1>CTX-M-30; CTX-M-75</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>ESBL+</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>866353</td><td rowspan=1 colspan=1>CTX-M-12</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>ESBL+</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>871347</td><td rowspan=1 colspan=1>SHV-2(2be)</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>ESBL+</td><td rowspan=1 colspan=1>Non-Susceptible</td></tr></table>

CT: Cefotaxime (S: $\vdots \leq 1 ; \operatorname { I } = 2$ ; $\mathtt { R } \ge 4 \mu \mathrm { g } / \mathrm { m L } )$ ); TZ: Ceftazidime $\mathbf { ( S ; \le 4 ; I = 8 }$ ; $\scriptstyle \mathrm { R : \geq } 1 6 \mu \mathrm { g } / \mathrm { m L }$ ); CPD: Cefpodoxime $( \mathrm { S } ; \geq 2 1 ; \mathrm { I } = 1 8 - 2 0 $ ; R ≤17mm) S: Susceptible; I: Intermediate: R: Resistant 1 All strains were obtained from IHMA (International Health Management Associates) 2 As determined by cefotaxime-cefotaxime/clavulanic acid (CT/CTL) and ceftazidime-ceftazidime/clavulanic acid (TZ/TZL) 3 Defined as I or R for any one of CT, TZ or CPD

# Analytical Reactivity/Inclusivity

The ability of HardyCHROM ESBL to recover potential ESBL-producing and/or $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible strains of $E$ . coli, K. oxytoca and $K .$ . pneumoniae was determined by testing a representative panel of strains with known cefotaxime, ceftazidime and cefpodoxime MICs, known genotypic resistance markers and confirmed ESBL phenotypes. The study included a total of 54 ESBL producing strains as shown in Table 5, all $( 1 0 0 \% )$ of which produced the expected colony color on HardyCHROM ESBL when inoculated at a concentration of 10 CFU/plate. Fiftythree of 54 strains $( 9 8 . 1 \% )$ were also non-susceptible to at least one $3 ^ { \mathrm { r d } }$ generation cephalosporin and produced colony colors on HardyCHROM ESBL that were consistent with this phenotype.

Table 5. Summary of the genotypic characteristics of the strains of $E$ . coli, $K .$ . pneumoniae and K. oxytoca evaluated in the Analytical Reactivity Study   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>β-lactamase Genotypes Represented</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>TEM (6, 12, 19, 21, 29, 210, 214, 215), CTX-M (1, 2, 3, 8, 14, 15,24, 27, 28, 40, 55, 75, 79, 116, 125, 130), TEM-OSBL</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>SHV (2, 11, 14, 18, 31, 55, 83, 89, 90, 108, 120, 133, 173, 178, 179,180, 182), TEM (4, 11, 129), CTX-M (12, 14, 15, 22, 38, 40, 64,74, 124), VEB-1, SHV-OSBL, TEM-OSBL</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>TEM(52), CTX-M (22, 30, 75), SHV (7, 12), DHA-1</td></tr></table>

Note: 53/54 strains $( 9 8 . 1 \% )$ were also non-susceptible to at least one ${ \overline { { 3 ^ { \mathrm { r d } } } } }$ generation cephalosporin and produced results on HardyCHROM ESBL that were consistent with this phenotype (i.e., pink or blue colonies). The exception was 1 strain of $K .$ pneumoniae that was ESBL positive and carried the SHV-11 and TEM-11 $\beta$ -lactamase genes but which was susceptible to ceftazidime, cefotaxime and cefpodoxime.

In addition to the strains of ESBL-producing organisms in Table 5, testing was also conducted with 21 representative strains of Citrobacter, Enterobacter, Escherichia, Hafnia, Klebsiella, Morganella, Proteus, Providencia, Raoultella, Serratia and Shigella that were non-susceptible to at least one $3 ^ { \mathrm { r d } }$ generation cephalosporin. Of these, 15 strains $( 7 1 . 4 \% )$ exhibited pink, blue or yellow-gold colonies on HardyCHROM ESBL after incubation for 24 hours. Colonies of the remaining 6 strains $H .$ alvei (1), $M .$ morganella (2), $P .$ . stuartii (2) and S. sonnei (1)) were recovered but did not exhibit one of the expected colony colors. The potential to obtain false negative results with HardyCHROM ESBL due to a lack of growth or failure to develop the expected colony color is noted in the Intended Use and is mitigated by including a Limitation in the device labeling (see Section H(3), above).

# e. Analytical specificity:

The analytical specificity of HardyCHROM ESBL was evaluated by testing a panel of organisms that are phylogenetically related to ESBL-producing species or that are likely to be found in stool specimens. Testing was performed by streaking $1 0 \mu \mathrm { L }$ of a $1 0 ^ { 8 } \mathrm { \dot { C } F U / m L }$ suspension of each organism on HardyCHROM ESBL and incubating at $3 5 \mathrm { { ^ \circ C } }$ for 24 hours. The organisms tested are listed in Tables 6 and 7 for Enterobacteriaceae $( \mathtt { n } = 5 2 )$ ) and non-Enterobacteriaceae $( \mathtt { n } = 4 7$ ), respectively.

The majority of non-Enterobacteriaceae species did not grow on HardyCHROM ESBL medium and none exhibited pink, blue or yellow/gold colonies. Of the Enterobacteriaceae, 45/45 $( 1 0 0 \% )$ of those that were $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible either did not grow on HardyCHROM ESBL or produced colonies that were not pink, blue or yellow/gold. Three carbapenem resistant and $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible strains ( $E .$ . coli (1) and $K .$ . pneumoniae (2)) produced pink and blue colonies, as expected. Four strains (1 each of $C .$ braakii, $C .$ . freundii, $E .$ . cloacae and Y. kristensenii) that were non-susceptible to cefpodoxime were not recovered on HardyCHROM ESBL. These results are mitigated by including appropriate Limitations in the device labeling (see Section H(3), above).

Table 6. Summary of results from testing strains of Enterobacteriaceae in the HardyCHROM ESBL Analytical Specificity Study   

<table><tr><td rowspan=2 colspan=1>Genus or Species</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=3>Number Non-susceptible</td><td rowspan=2 colspan=1>Growth onHardyCHROMESBL1</td></tr><tr><td rowspan=1 colspan=1>Cefotaxime</td><td rowspan=1 colspan=1>Ceftazidime</td><td rowspan=1 colspan=1>Cefpodoxime</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2}$</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Cronobacter</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 3$</td></tr><tr><td rowspan=1 colspan=1>Hafnia</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Morganella</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Providencia</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Raoultella</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td></tr></table>

NA: Not available   
Number with Pink or Blue colonies after 24 hours 2 1 C. braakii and 1 C. freundii; both with intermediate susceptibility to cefpodoxime 3 Carbapenem resistant and resistant to cefotaxime, ceftazidime and cefpodoxime; ESBL non-determinable 4 Both strains carbapenem resistant and resistant to cefotaxime, ceftazidime and cefpodoxime; 1 ESBL non-producer and 1 ESBL nondeterminable 5 Y. kristensenii

Table 7. Summary of non-Enterobacteriaceae species tested in the HardyCHROM ESBL Analytical Specificity Study   

<table><tr><td rowspan=1 colspan=2>Species Name 1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanii 2</td><td rowspan=1 colspan=1>Lactobacillus gasseri</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Lactobacillus lactis</td></tr><tr><td rowspan=1 colspan=1>Aspergillus sp.</td><td rowspan=1 colspan=1>Listeria grayi</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>Micrococcus luteus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Pediococcus acidilactici</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ssp. jejuni</td><td rowspan=1 colspan=1>Penicillium aurantiogriseum</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Penicillium chrysogenum</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Penicillium rubens 2</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>Pseudomonas fluorescens 2</td></tr><tr><td rowspan=1 colspan=1>Enterococcus cassiflavus</td><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>Staphylococcus aureus (3)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (3)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (2)</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>Streptococcus agalactiae (2)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>Streptococcus gallolyticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus saccharolyticus</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Geotrichum candidum 2</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Geotrichum guilliermondii 2</td><td rowspan=1 colspan=1>Vibrio cholerae</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1></td></tr></table>

1 The number of strains tested is shown in parenthesis if ${ > } 1$ 2 Growth observed on HardyCHROM ESBL but colonies not pink, blue or yellow/gold

f. Assay cut-off: Not applicable.

g. Assay Interference:

# Interfering Substances

A study was conducted to determine the ability to recover target organisms in the presence of potentially interfering substances. In order to represent a “worst case” scenario, the study was performed with representative strains of ESBL-producing and/or $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible E. coli, $K .$ . pneumoniae, K. oxytoca, $E$ . cloacae and $P .$ . mirabilis seeded into “raw”, unpreserved stool at levels close to the LOD of HardyCHROM ESBL. The absence of interference was determined by the recovery of each species with the expected colony color and $< 1$ $\log _ { 1 0 }$ reduction in colony count compared with a control condition without the interfering substance. A list of the potentially interfering substances tested and their concentrations is provided in Table 8. None of the substances was found to interfere with recovery of the target organisms at the concentrations tested.

Table 8. Substances tested for potential interference with growth on HardyCHROM ESBL   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration (%) 1</td></tr><tr><td rowspan=4 colspan=1>Antifungal</td><td rowspan=1 colspan=1>Nystop (Nystatin)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Lotrimin (Clotrimazole)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Lotrimin Ultra (Butenafine Hydrochloride)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Lamisil (Terbinafine Hydrochloride 1%)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=2 colspan=1>Antiseptic</td><td rowspan=1 colspan=1>Bactine (Benzalkonium Chloride)</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Biologic</td><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Contraceptive</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=9 colspan=1>GI Medication</td><td rowspan=1 colspan=1>Pepto-Bismol (Bismuth Subsalicylate)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Prilosec OTC (Omeprazole)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Alka-Seltzer (Sodium carbonate/potassium carbonate)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Mylanta (Al(OH)3)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Tums (CaCO3)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Rolaids (Mg(OH)2)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Milk of Magnesia (Mg(OH)2)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Dulcolax (Sodium picosulfate solution)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>Immodium AD (Loperamide)</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=4 colspan=1>Lubricant</td><td rowspan=1 colspan=1>Mineral oil</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Petroleum jelly</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Fleet (Glycerin)</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>KY Jelly</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=3 colspan=1>Other</td><td rowspan=1 colspan=1>C&amp;S Medium Transport</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>Physiological Saline</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Tween 80 (Polysorbate 80)</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=2 colspan=1>Topical Medication</td><td rowspan=1 colspan=1>Preparation H (Hemorrhoid Cream)</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Cortizone 10 (Hydrocortizone)</td><td rowspan=1 colspan=1>10%</td></tr></table>

w/v or $\mathbf { v } / \mathbf { v }$ , as appropriate

# Interfering Microorganisms

A study was performed to demonstrate the ability to discriminate ESBL-producing target organisms from non-target species that are capable of growing on HardyCHROM ESBL. Plates of HardyCHROM ESBL were inoculated with one strain each of E. coli, $K .$ . pneumoniae and $K .$ . oxytoca ${ \bf \tilde { \tau } } _ { \sim 1 0 } { } ^ { 2 }$ CFU/plate) in the presence and absence of high concentrations $( > 1 0 ^ { 6 }$ CFU/plate) of various “co-infecting” species that grew on HardyCHROM ESBL in the Analytical Specificity Study (Table 9).

Table 9. Species of potentially interfering microorganisms included in the Interference Study   

<table><tr><td rowspan=1 colspan=2>Species Name</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>Pediococcus acidilactici</td></tr><tr><td rowspan=1 colspan=1>Geotrichum candidum</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Aspergillus sp.</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td></tr><tr><td rowspan=1 colspan=1>Penicillium rubens</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>Penicillium chrysogenum</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr></table>

All species listed had growth on HardyCHROM ESBL in the Analytical Specificity Study, although none produced pink, blue or yellow-gold colonies

In the presence of high levels of Acinetobacter baumannii, the number of colonies $E .$ . coli observed was substantially diminished compared to that obtained in the absence of the interfering species. In addition, in the presence of A. baumannii and Pseudomonas fluorescens, colonies of K. pneumoniae appeared smaller than those grown under control conditions. No other anomalies were observed in the presence of any of the potentially interfering microorganisms. The potential for an adverse effect on the performance of HardyCHROM from the presence of high concentrations of $A$ . baumannii and $P .$ . fluorescens is mitigated by including a Limitation in the device labeling.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

Comparison of Fresh vs Frozen Stool   
To enable use of frozen stool specimens for analytical testing, a study was performed to compare the performance of HardyCHROM ESBL with fresh and frozen stool matrix. Stool samples (raw or diluted 1:5 in Culture & Sensitivity Medium Transport, Cary Blair Formula) were stored overnight at $2 { - } 8 ^ { \circ } \mathrm { C }$ or frozen at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ , equilibrated to ambient temperature and then inoculated with representative strains of ESBLproducing and/or $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae. A list of the organisms tested is provided in Table 10. An aliquot of each sample was plated on HardyCHROM ESBL culture medium. The number and color of colonies obtained were recorded after incubation of the culture plates for 18 and 24 hours. All the organisms tested were successfully recovered with the expected colony color from ${ \ge } 9 / 1 0$ fresh or frozen stool samples. No important differences in colony counts between cultures from fresh and frozen stool samples were observed. These results support the use of frozen stool samples (raw or diluted in C&S Medium Transport, Cary Blair Formula) for analytical testing.

Table 10. Organisms tested in the comparison of fresh and frozen stool matrix   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>IHMA Strain 1</td><td rowspan=1 colspan=1>ESBL</td><td rowspan=1 colspan=1>3rd GenerationCephalosporin</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>946852</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Non-susceptible</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>846222</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Non-susceptible</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>871347</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Non-susceptible</td></tr><tr><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>856282</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Non-susceptible</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>862552</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Non-susceptible</td></tr></table>

1 International Health Management Associates strain designation

3. Clinical studies:

# a. Clinical Sensitivity:

The performance of HardyCHROM ESBL was evaluated in a prospective Clinical Study conducted at three geographically diverse sites in the US. Recovery of ESBLproducing $E$ . coli and $K .$ . pneumoniae/K. oxytoca from stool samples on HardyCHROM ESBL was compared to the results of a reference method comprised of enrichment culture of stool in Trypticase Soy Broth containing ceftazidime $( 1 \mu \mathrm { g / m L } )$ or cefotaxime $( 1 \mu \mathrm { g / m L } )$ , followed by subculture to MacConkey agar. The same study was also used to evaluate the performance of HardyCHROM ESBL for recovery of cephalosporin non-susceptible Enterobacteriaceae. Organisms that grew on either MacConkey agar or HardyCHROM ESBL were identified by standard methods. Cephalosporin susceptibility and confirmed ESBL phenotype were determined using an FDA cleared MIC test and/or by disk diffusion according to CLSI M100.

A total of 1,687 stool samples were enrolled in the study. Of these, 128 were excluded from the analysis of performance due to protocol deviations. Isolates from a total of 1,559 samples were included in the analysis of performance.

To supplement testing of prospectively collected clinical specimens, a total of 50 contrived specimens were also evaluated. These were prepared by spiking organisms of known ESBL and cephalosporin susceptibility phenotype directly into stool matrix at a concentration of $3 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 3 } \dot { \mathrm { C F U / m L } }$ . The results obtained on HardyCHROM ESBL were compared to those obtained by the same reference methods used for the prospective clinical specimens.

# Detection of $3 ^ { r d }$ Generation Cephalosporin Non-Susceptible Enterobacteriaceae

Prospectively Collected Clinical Specimens The performance of HardyCHROM ESBL for recovery of $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae from prospectively collected clinical specimens is shown in Table 11. The same sensitivity and specificity were observed when plates were read after 18 and 24 hours. Table 12 shows the agreement between the colony color observed on the HardyCHROM ESBL plates and the identity and $3 ^ { \mathrm { r d } }$ generation cephalosporin susceptibility of the isolates recovered.

Table 11. Performance of HardyCHROM ESBL with prospectively collected clinical specimens in comparison to the reference method, stratified by clinical site   

<table><tr><td rowspan=1 colspan=5>3rd Generation Cephalosporin Non-susceptible Enterobacteriaceae</td></tr><tr><td rowspan=2 colspan=2>Site 1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>526</td><td rowspan=1 colspan=1>535</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>541</td><td rowspan=1 colspan=1>635</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>85/94 = 90.4% (82.8-94.9%) 3</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>526/541 = 97.2% (95.5-98.3%)</td></tr><tr><td rowspan=2 colspan=2>Site 2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>678</td><td rowspan=1 colspan=1>684</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>703</td><td rowspan=1 colspan=1>780</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>71/77 = 92.2% (84.0-96.4%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>678/703 = 96.4% (94.8-97.6%)</td></tr><tr><td rowspan=2 colspan=2>Site 3</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>217</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>270</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>638</td><td rowspan=1 colspan=1>660</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>691</td><td rowspan=1 colspan=1>930</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>217/239 = 90.8% (86.5-93.8%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>638/691 = 92.3% (90.1-94.1%)</td></tr><tr><td rowspan=2 colspan=2>Overall</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=7 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>466</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>1842</td><td rowspan=1 colspan=1>1879</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>410</td><td rowspan=1 colspan=1>1935</td><td rowspan=1 colspan=1>2345</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>373/410 = 91.0% (87.8-93.4%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>1842/1935 = 95.2% (94.1-96.1%)</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=3>373/466 = 80.0% (76.2%-83.4%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=3>1842/1879 = 98.0% (97.3-98.6%)</td></tr></table>

PPV: Positive Predictive value; NPV: Negative Predictive Value Note: The same sensitivity and specificity were observed when plates were read after 18 and 24 hours 1 Pink, Blue or Yellow/Green colonies 2 Colonies other than Pink, Blue or Yellow/Green, or No Growth $9 5 \%$ Score Confidence Interval

Table 12. Agreement between colony color observed on HardyCHROM ESBL with prospectively collected clinical specimens and the identity and $3 ^ { \mathrm { r d } }$ generation cephalosporin susceptibility of the isolates recovered from the HardyCHROM ESBL culture medium   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>3rd Generation CephalosporinNon-susceptible Enterobacteriaceae</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=7 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>466</td></tr><tr><td rowspan=1 colspan=1>Negative2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1858</td><td rowspan=1 colspan=1>1879</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>469</td><td rowspan=1 colspan=1>1876</td><td rowspan=1 colspan=1>2345</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=3>448/469 = 95.5% (93.3-97.1%)</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=3>1858/1876 = 99.0% (98.5-99.4%)</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=3>448/466 = 96.1% (94.0-97.5%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>1858/1879 = 98.9% (98.3-99.3%)</td><td rowspan=1 colspan=2>1858/1879 = 98.9% (98.3-99.3%)</td></tr></table>

PPA: Positive Percent Agreement; NPA: Negative Percent Agreement; PPV: Positive Predictive Value; NPV: Negative Predictive Value 1 Pink, Blue or Yellow/Green colonies 2 Colonies other than Pink, Blue or Yellow/Green or No Growth

# Contrived Specimens

Table 13 shows the agreement between colony color observed on HardyCHROM ESBL and the identity and $3 ^ { \mathrm { r d } }$ generation cephalosporin susceptibility of the isolates recovered from contrived specimens as determined by the reference method. There were no differences in positive or negative agreement when plates were read either 18 or 24 hours after inoculation.

Table 13. Agreement between the colony color observed on HardyCHROM ESBL with contrived specimens and organism identity and cephalosporin susceptibility as determined by the reference method   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>3rd Generation CephalosporinNon-susceptible Enterobacteriaceae</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=3>47/48 = 97.9% (89.1-99.6%)</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=3>3/5 = 60.0% (23.1-88.2%)</td></tr></table>

PPA: Positive Percent Agreement; NPA: Negative Percent Agreement Note: The same positive and negative agreement were observed when plates were read after 18 and 24 hours 1 Pink, Blue or Yellow/Green colonies 2 Colonies other than Pink, Blue or Yellow/Green or No Growth

# Detection of ESBL Producing E. coli and K. pneumoniae/K. oxytoca

Prospectively Collected Clinical Specimens The performance of HardyCHROM ESBL for recovery of ESBL-producing strains of E. coli and $K .$ . pneumoniae/K. oxytoca from prospectively collected clinical samples is shown in Table 14. The same sensitivity and specificity were observed when plates were read after 18 and 24 hours. Table 15 summarizes HardyCHROM ESBL performance according to the species of ESBL-producing organisms recovered by the reference method. Table 16 shows a comparison of the morphology observed on HardyCHROM ESBL and the identity and ESBL phenotype of the isolates recovered from the HardyCHROM ESBL plates as determined by conventional methods.

Table 14. Performance of HardyCHROM ESBL with prospectively collected clinical specimens in comparison to the reference method, stratified by clinical site   

<table><tr><td rowspan=1 colspan=5>ESBL Producing E. coli and K. pneumoniae/K. oxytoca</td></tr><tr><td rowspan=2 colspan=2>Site 1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>456</td><td rowspan=1 colspan=1>457</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>523</td><td rowspan=1 colspan=1>550</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>26/27 = 96.3% (81.7-99.3%) 3</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>456/523 = 87.2% (%)</td></tr><tr><td rowspan=2 colspan=2>Site 2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>564</td><td rowspan=1 colspan=1>564</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>629</td><td rowspan=1 colspan=1>654</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>25/25 = 100% (86.7-100%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>564/629 = 89.7% (87.0-91.9%)</td></tr><tr><td rowspan=2 colspan=2>Site 3</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>471</td><td rowspan=1 colspan=1>475</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>738</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>115/119 = 96.6% (91.7-98.7%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>471/619 = 76.1% (72.6-79.3%)</td></tr><tr><td rowspan=2 colspan=2>Overall</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=7 colspan=1>HardyCHROMESBL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>446</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1894</td><td rowspan=1 colspan=1>1899</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>2174</td><td rowspan=1 colspan=1>2345</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>166/171 = 97.1% (93.3-98.7%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>1894/2174 = 87.1% (85.6-88.5%)</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=3>166/446 = 37.2% (32.9-41.8%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=3>1894/1899 = 99.7% (99.4-99.9%)</td></tr></table>

PPV: Positive Predictive Value; NPV: Negative Predictive Value Note: The same sensitivity and specificity were observed when plates were read at 18 and 24 hours 1 Pink, or Blue colonies 2 Colonies other than Pink or Blue, or No Growth 3 $9 5 \%$ Score Confidence Interval

Table 15. Summary of HardyCHROM ESBL performance with prospectively collected clinical specimens in comparison to the reference method, stratified by ESBL-producing species   

<table><tr><td rowspan=1 colspan=1>ESBL-producing Species</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>110/11595.7%(90.2-98.1%) 1</td><td rowspan=1 colspan=1>110/22449.1%(42.6-55.6%)</td><td rowspan=1 colspan=1>2116/223094.9%(93.9-95.7%)</td><td rowspan=1 colspan=1>2116/212199.8%(99.4-99.9%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae/K. oxytoca</td><td rowspan=1 colspan=1>56/56100%(93.6-199%)</td><td rowspan=1 colspan=1>56/22225.2%(20.0-31.3%)</td><td rowspan=1 colspan=1>2123/228992.7%(91.6-93.7%)</td><td rowspan=1 colspan=1>2123/2123100%(99.8-100%)</td></tr><tr><td rowspan=1 colspan=1>E. coli andK. pneumoniae/K. oxytocaCombined</td><td rowspan=1 colspan=1>166/17197.1%(93.3-98.7%)</td><td rowspan=1 colspan=1>166/44637.2%(32.9-41.8%)</td><td rowspan=1 colspan=1>1894/217487.1%(85.6-88.5%)</td><td rowspan=1 colspan=1>1894/189999.7%(99.4-99.9%)</td></tr></table>

PPV: Positive Predictive Value; NPV: Negative Predictive Value 1 $9 5 \%$ Score Confidence Interval

Table 16. Agreement between the colony color observed on HardyCHROM ESBL with prospectively collected clinical specimens and the identity and ESBL phenotype of the isolates recovered from the HardyCHROM ESBL culture medium   

<table><tr><td rowspan=1 colspan=2>HardyCHROM ESBL</td><td rowspan=2 colspan=1>Incubation</td><td rowspan=1 colspan=2>Agreement (% with 95% CI)</td></tr><tr><td rowspan=1 colspan=1>Colony Color</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>Pink</td><td rowspan=2 colspan=1>E. coli</td><td rowspan=1 colspan=1>18 hours</td><td rowspan=1 colspan=1>123/12697.6%(93.2-99.2%)</td><td rowspan=1 colspan=1>2118/221995.4%(94.5-99.2%)</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>123/12697.6%(93.2-99.2%)</td><td rowspan=1 colspan=1>2125/221995.8%(94.8-96.5%)</td></tr><tr><td rowspan=2 colspan=1>Blue</td><td rowspan=2 colspan=1>K. pneumoniae/K. oxytoca</td><td rowspan=1 colspan=1>18 hours</td><td rowspan=1 colspan=1>61/6298.4%(91.4-99.7%)</td><td rowspan=1 colspan=1>2122/228392.9%(91.8-93.9%)</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>61/6298.4%(91.4-99.7%)</td><td rowspan=1 colspan=1>2115/229392.6%(91.5-93.6%)</td></tr></table>

$9 5 \%$ Score Confidence Interval

# Contrived Specimens

Table 17 shows the agreement between colony color observed on HardyCHROM ESBL and the identity and ESBL phenotype of the isolates recovered from contrived specimens as determined by the reference method. There were no differences in positive or negative agreement when plates were read either 18 or 24 hours after inoculation.

Table 17. Agreement between the colony color observed on HardyCHROM ESBL with contrived specimens and organism identity and ESBL phenotype as determined by the reference method   

<table><tr><td rowspan=1 colspan=2>HardyCHROM ESBL</td><td rowspan=1 colspan=2>Agreement (% with 95% CI)1, 2</td></tr><tr><td rowspan=1 colspan=1>Colony Color</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Pink</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>19/19100%(83.2-100%)</td><td rowspan=1 colspan=1>31/3491.2%(77.0-97.0%)</td></tr><tr><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>K. pneumoniae/K. oxytoca</td><td rowspan=1 colspan=1>26/26100%(87.1-100%)</td><td rowspan=1 colspan=1>25/2792.6%(76.6-97.9%)</td></tr></table>

1 $9 5 \%$ Score Confidence Interval 2 The same positive and negative agreement were observed when plates were read after 18 and 24 hours

b. Clinical specificity: Refer to Section M3(a), above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the prospective Clinical Study described in Section M(3), above, the prevalence of $3 ^ { \mathrm { r d } }$ generation cephalosporin non-susceptible Enterobacteriaceae that was observed on HardyCHROM ESBL was $2 3 . 9 \%$ . The prevalence of ESBL-producing $E .$ . coli and $K .$ . pneumoniae/K. oxytoca as determined using HardyCHROM ESBL was $1 0 . 6 \%$ .

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.