# SPECIAL 510(k): Device Modification OIR Decision Summary

RE: K161556

This 510(k) submission contains information/data on modifications made to the SUBMITTER’S own Class II device requiring 510(k). The following items are present and acceptable:

1. The name and 510(k) number of the SUBMITTER’S previously cleared device.

Trade Name: CDC Human Influenza Virus Real-time RT-PCR Diagnostic Panel, Influenza A Subtyping Kit   
510(k) Number: k140851

2. Submitter’s statement that the INDICATION/INTENDED USE of the modified device, called “CDC Human Influenza Virus Real-time RT-PCR Diagnostic Panel, Influenza A Subtyping Kit VER2”, as described in its labeling HAS NOT CHANGED along with the proposed labeling which includes instructions for use, and package labeling.

Submitter states in the labeling, 510(k) Summary, and in the submission that the intended use of the modified device has not changed from its predicate.

3. A description of the device MODIFICATIONS, including clearly labeled diagrams and assay instructions in sufficient detail to demonstrate that the FUNDAMENTAL SCIENTIFIC TECHNOLOGY of the modified device has not changed.

# The modifications presented in this special 510k include:

a) Redesign of the reverse primer and probe sequences for A/pdmH1N1 assay to better detect currently circulating influenza A(H1N1)pdm09 viruses belonging to the 6B.1 genetic clade;   
b) Evaluation of ZENTM Double-Quenched probes (InfA-P, H3-P, pdmInfA-P, pdmH1-P, and RP-P) as an alternate option to the currently cleared BHQ probes;   
c) Elimination of the influenza A/H1 primers/probe (H1-F, H1-R, H1-P) set of reagents because this influenza virus is no longer circulating and the assay for its detection is unnecessary;   
d) Elimination of references and instructions for A/H1 from the IFU statement, package labels, software, and labeling;   
e) Replace positive control PIPC (inactive A/H1, A/H3, A/H1pdm09, B) with SPIC (inactive A/H3, A/H1pdm09, B);   
f) Updating nomenclature of influenza A virus subtypes to align with WHO nomenclature. Designations A/H1, A/H3 and A/H1pdm updated to A(H1N1), A(H3N2) and A(H1N1)pdm09;

g) Updates to the device labeling:

Addition of study results demonstrating the performace of modified assays (modified primers and probes); Replacing LoD data for influenza A/(H3/N2) strains A/New York/55/2004 and A/Wisconsin/67/2005 with data for A/Perth/16/2009 and A/Victoria/361/2011 strains;

Addition of reactivity data with potential pandemic influenza A swine H1N1v viruses A/Texas/14/2008, A/Ohio/09/2015, and A/Minnesota/19/2011; • Strains A/Maryland/12/1991 and A/Swine/Wisconsin/125/1997 were removed from the reactivity table as per sponsor indication that they no longer circulate;

4. Comparison Information (similarities and differences) to applicant’s legally marketed predicate device including, labeling, intended use, physical characteristics, and software is shown in the table below.

<table><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">Predicate:CDC Human Influenza Virus Real-Time RT- PCR Diagnostic PanelDiagnostic Panel, Influenza ASubtyping Kit (K140851)</td><td colspan="1" rowspan="1">Modified Device:CDC Human Influenza Virus Real-Time RT- PCR Diagnostic PanelDiagnostic Panel, Influenza ASubtyping Kit (VER 2) (K161556)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Influenza A Subtyping Kit containsreagents and controls of the CDCHuman Influenza Virus Real-Time RT-PCR Diagnostic Panel and is intendedfor use in real-time RT-PCR (rRT-PCR)assays on an Applied Biosystems (ABI)7500 Fast Dx Real-Time PCRinstrument in conjunction with clinicaland epidemiological information:For determination of the subtype ofseasonal human influenza A viruses asseasonal A/H1, A/H3, and/orA/H1pdm09 from viral RNA in upperrespiratory tract clinical specimens(including nasopharyngeal swabs[NPS], nasal swabs [NS], throat swabs[TS], nasal aspirates [NÅ], nasalwashes [NW] and dualnasopharyngeal/throat swabs[NPS/TS]) and lower respiratory tractspecimens (including bronchoalveolarlavage [BAL], bronchial wash [BW],tracheal aspirate [TA], sputum, andlung tissue) from human patients withsigns and symptoms of respiratoryinfection and/or from viral culture;To provide epidemiologic informationfor surveillance of circulating influenzaviruses.Performance characteristics forinfluenza were established during aseason when seasonal influenzaviruses A/H1 and A/H3 were thepredominant influenza A viruses in</td><td colspan="1" rowspan="1">The Influenza A Subtyping Kit containsreagents and controls of the CDCHuman Influenza Virus Real-Time RT-PCR Diagnostic Panel and is intendedfor use in real-time RT-PCR (rRT-PCR)assays on an Applied Biosystems (ABI)7500 Fast Dx Real-Time PCRinstrument in conjunction with clinicaland epidemiological information:For determination of the subtype ofseasonal human influenza A viruses asseasonal A(H3), and/or A(H1)pdm09from viral RNA in upper respiratorytract clinical specimens (includingnasopharyngeal swabs [NPS], nasalswabs [NS], throat swabs [TS], nasalaspirates [NA], nasal washes [NW] anddual nasopharyngeal/throat swabs[NPS/TS]) and lower respiratory tractspecimens (including bronchoalveolarlavage [BAL], bronchial wash [BW],tracheal aspirate [TA], sputum, andlung tissue) from human patients withsigns and symptoms of respiratoryinfection and/or from viral culture;To provide epidemiologic informationfor surveillance of circulating influenzaviruses.Performance characteristics forinfluenza were established during aseason when seasonal influenzaviruses A(H1N1) and A(H3N2) werethe predominant influenza A viruses incirculation and during a season when</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">circulation and during a season whenthe A/H1pdm09 influenza virus was thepredominant influenza A virus incirculation. Performance characteristicsmay vary with other emerging influenzaA viruses.Negative results do not precludeinfluenza virus infection and should notbe used as the sole basis for treatmentor other patient managementdecisions. Conversely, positive resultsdo not rule out bacterial infection or co-infection with other viruses. The agentdetected may not be the definite causeof disease.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentinfluenza viruses and sent to state orlocal health department for testing.Viral culture should not be attemptedunless a BSL 3+ facility is available toreceive and culture specimens.Text box in IFU:All users, analysts, and any personreporting results from use of this deviceshould be trained to perform andinterpret the results from this procedureby a competent instructor prior to use.CDC Influenza Division will limit thedistribution of this device to only thoseusers who have successfullycompleted a training course providedby CDC instructors or designees.</td><td colspan="1" rowspan="1">the A(H1N1)pdm09 influenza virus wasthe predominant influenza A virus incirculation. Performance characteristicsmay vary with other emerging influenzaA viruses.Negative results do not precludeinfluenza virus infection and should notbe used as the sole basis for treatmentor other patient managementdecisions. Conversely, positive resultsdo not rule out bacterial infection or co-infection with other viruses. The agentdetected may not be the definite causeof disease.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentinfluenza viruses and sent to state orlocal health department for testing.Viral culture should not be attemptedunless a BSL 3E facility is available toreceive and culture specimens.Text box in IFU:All users, analysts, and any personreporting results from use of this deviceshould be trained to perform andinterpret the results from this procedureby a competent instructor prior to use.CDC Influenza Division will limit thedistribution of this device to only thoseusers who have successfullycompleted a training course providedby CDC instructors or designees.</td></tr><tr><td colspan="1" rowspan="1">OrganismDetected</td><td colspan="1" rowspan="1">Universal influenza A viruses (animaland human), Swine-origin influenza Aviruses, Influenza A subtypes:seasonal A(H1), A(H3), A(H1)pdm09</td><td colspan="1" rowspan="1">Universal influenza A viruses (animaland human), Swine-origin influenza Aviruses, Influenza A subtypes:seasonal A(H3) and A(H1)pdm09</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Nasopharyngeal swabs, nasal swabs,throat swabs, nasal aspirates, nasal</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">washes and dualnasopharyngeal/throat swabs,bronchoalveolar lavages, bronchialaspirates, bronchial washes, trachealaspirates, sputum, and lung tissue.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethod</td><td colspan="1" rowspan="1">•QIAamp® DSP Viral RNA Mini Kit,QiagenMagNA Pure Compact -Nucleic AcidIsolation Kit I, RocheMagNA Pure Compact - RNAIsolation Kit, RocheMagNA Pure LC - Total Nucleic AcidKit, RocheQiagen QIAcube - QIAamp® DSPViral RNA Mini Kit, QiagenNucliSENS® easyMAG®, bioMerieux</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Enzyme MasterMix</td><td colspan="1" rowspan="1">Invitrogen SuperScript ™M II Platinum®One-Step Quantitative RT-PCR Kit(with or without ROX)ORQuanta BioSciences qScript ™ One-Step qRT-PCR Kit, Low ROX</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Real-time RT-PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Fast DxReal-Time PCR Instrument with SDSsoftware version 1.4</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">ProbeQuenchingMolecule</td><td colspan="1" rowspan="1">Black Hole Quencher Probe (BHQ-1)</td><td colspan="1" rowspan="1">ZEN Double-Quenched Probe (InfA,H3, pdmInfA, pdmH1, and RP assays)OR Black Hole Quencher Probe (InfA,H3, pdmInfA, pdmH1, and RP assays)</td></tr><tr><td colspan="1" rowspan="1">Oligonucleotides</td><td colspan="1" rowspan="1">H1 Assay-Targets a region of the HAgeneH3 assay-Targets a region of the HAgene pdmH1 assay-Targets a region ofthe HA geneInfA assay-Targets a conserved regionof the matrix gene in Influenza Aviruses</td><td colspan="1" rowspan="1">Gene targets of the oligonucleotideassays are the same as the predicate;minor changes to the pdmH1oligonucleotide sequences have beenmade; the H1 assay is not included inthis version of theInfluenza A Subtyping Kit</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pdmInfA-Targets a conserved region ofthe nucleoprotein gene in InfluenzaA(H1N1) pdm09 viruses</td><td colspan="1" rowspan="1"></td></tr></table>

# 5. A Design Control Activities Summary was present which includes:

a) Identification of Risk Analysis method used to assess the impact of the modification on the device and its components, and the results of the analysis

The risk analysis for the CDC Influenza A Subtyping Kit was conducted to verify that the device modifications do not present increased or new risks to the user. The following potential risk factors were identified: 1) the modified oligonucleotides may demonstrate variable reactivity among virus strains, and 2) that labeling changes may result in testing errors leading to inaccurate or delayed assay results. Mitigations for risk #1 included analytical reactivity verification testing. For risk #2, the proposed mitigations are to distribute to the end users an updated labeling (package insert) for the Influenza A Subtyping Kit and a Product Update Communication explaining the modifications and communicating the availability of the updated Influenza A Subtyping Kit. Proposed mitigations should bring the identified hazards to an acceptable level.

b) Based on the Risk Analysis, an identification of the verification activities required, including tests used and acceptance criteria to be applied.

# Verification of Performance

The following studies were conducted to mitigate the risk of variable reactivity of the modified primers and probes:

# Analytical sensitivity

A study was conducted to compare the approximate Limit of Detection (LoD; range finding study, $\mathsf { n } { = } 3$ per each analyte concentration) between cleared and modified RT-PCR reaction mixtures. The viruses tested were the historic virus A/California/07/2009 and the recent virus A/West Virginia/01/2016. The latter strain includes the point mutations that cause aberrant results with the current cleared A(H1)pdm09 assay. Characterized viruses of a known $50 \%$ infectious dose titer (EID50/mL or ${ \mathsf { T C l D 5 0 } } / { \mathsf { m L } } ,$ ) were extracted, and the RNA was serially diluted and tested in order to determine an estimated LoD (the lowest concentration where 3 of 3 replicates are detected). These studies were conducted with two RT-PCR enzyme systems cleared for use with the CDC Influenza A Sybtyping Kit (Invitrogen Superscript and Quanta qscript).

The results of the range-finding LoD study indicate equivalent reactivity between the cleared and modified reaction mixtures for the historic virus A/California/07/2009. The results for A/West

Virginia/01/2016 confirm that the modifed A(H1)pdm09 assay detects the virus bearing the point mutation that was not detected by the current A(H1)pdm09 assay.

The estimated LoD was confirmed by testing extraction replicates $( \mathsf { n } { = } 2 0 )$ of the highest virus dilution (of the range-finding study) where greater than or equal to $9 5 \%$ of replicates tested positive. These studies were conducted with two RT-PCR enzyme systems cleared for use with the CDC Influenza A Sybtyping Kit (Invitrogen Superscript and Quanta qscript). The following lowest concentration where the InfA, pdmInfA, and pdmH1 primer and probe sets demonstrate uniform detection was reported as the LOD.

<table><tr><td rowspan=2 colspan=1>Influenza VirusTested</td><td rowspan=2 colspan=1>Influenza Strain Designation</td><td rowspan=1 colspan=2>LoD (ID50/mL)</td></tr><tr><td rowspan=1 colspan=1>InvitrogenSuperScript</td><td rowspan=1 colspan=1>Quanta qScript</td></tr><tr><td rowspan=2 colspan=1>A(H1)pdm09</td><td rowspan=1 colspan=1>A/West Virginia/01/2016</td><td rowspan=1 colspan=1>100.9</td><td rowspan=1 colspan=1>100.9</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>103.1</td><td rowspan=1 colspan=1>103.8</td></tr></table>

A comparison study was conducted to demonstrate LOD equivalency for the A/H3 assays between the currently cleared BHQ probe and the ZEN probe. The RNA was extracted from A/Hong Kong/4801/2014 virus, was serially deluted, and three replicates per dilution were tested. The results indicate similar analytical sensitivity between the assays with ZEN and BHQ-1 probes.

# Inclusivity

An inclusivity study was conducted to demonstrate the capability of the modified primer/probe RT-PCR mixtures (InfA, pdmInfA and pdmH1 with ZEN probes) to detect influenza A(H1)pdm09 viruses representative of different geographic locations and phylogenetic clades. Inclusivity testing was performed with ten representative H1pdm09 viruses at or near the established LoD. The viruses were grown to high titer, harvested, and serially diluted to near the LoD of the assays. The diluted viruses were extracted and tested $\scriptstyle \mathtt { n } = 3$ replicates) to demonstrate reactivity.

The Influenza A Subtyping Kit was reactive with all H1pdm09 isolates tested. The inclusivity results are presented in the table below.

<table><tr><td rowspan=1 colspan=1>Influenza Virus Strain A(H1N1)pdm09 Identification</td><td rowspan=1 colspan=1>ID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/California/04/2009</td><td rowspan=1 colspan=1>102.9</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>103.5</td></tr><tr><td rowspan=1 colspan=1>A/Colorado/14/2012</td><td rowspan=1 colspan=1>101.1</td></tr><tr><td rowspan=1 colspan=1>A/Florida/27/2011</td><td rowspan=1 colspan=1>101.9</td></tr></table>

Page 7 of 10   

<table><tr><td rowspan=1 colspan=1>A/Florida/62/2014</td><td rowspan=1 colspan=1>102.2</td></tr><tr><td rowspan=1 colspan=1>A/Maryland/13/2012</td><td rowspan=1 colspan=1>101.0</td></tr><tr><td rowspan=1 colspan=1>A/Minnesota/03/2011</td><td rowspan=1 colspan=1>103.9</td></tr><tr><td rowspan=1 colspan=1>A/North Carolina/4/2014</td><td rowspan=1 colspan=1>103.3</td></tr><tr><td rowspan=1 colspan=1>A/Utah/13/2016</td><td rowspan=1 colspan=1>101.5</td></tr><tr><td rowspan=1 colspan=1>A/Washington/24/2012</td><td rowspan=1 colspan=1>102.5</td></tr></table>

# Analytical Specificity

Analytical specificity was evaluated by testing influenza A(H1) virus strains representing diverse geographic locations. Samples were tested in triplicate using RNA extracted from high titer preparations of viruses $\mathrm { \Lambda } ( \geq 1 0 ^ { 6 } \ : \mathrm { | D 5 0 / m L \rangle }$ ). Cross-reactivity testing was conducted with two enzyme systems (Invitrogen Superscript and Quanta qscript) and one extraction method cleared for use with the Influenza A Subtyping Kit. The results are presented in the table below.

<table><tr><td rowspan=2 colspan=1>Strain DesignationSubtype</td><td rowspan=2 colspan=1>ID50/mL</td><td rowspan=1 colspan=3>SuperScript</td><td rowspan=1 colspan=3>qScript</td></tr><tr><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdmH1</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdmH1</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07A(H1N1)</td><td rowspan=1 colspan=1>108.4</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/2001A(H1N1)</td><td rowspan=1 colspan=1>108.1</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/lowa/1/2006A(H1N1v)</td><td rowspan=1 colspan=1>108.2</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td></tr><tr><td rowspan=1 colspan=1>A/Texas/14/2008A(H1N1v)</td><td rowspan=1 colspan=1>108.3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td></tr><tr><td rowspan=1 colspan=1>A/Ohio/09/2015A(H1N1v)</td><td rowspan=1 colspan=1>107.7</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td></tr><tr><td rowspan=1 colspan=1>A/Minnesota/19/2011A(H1N2v)</td><td rowspan=1 colspan=1>107.1</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1>(+)3/3</td><td rowspan=1 colspan=1></td></tr></table>

The reactivity data for three animal viruses, A/Texas/14/2008, A/Ohio/09/2015 and A/Minnesota/19/2011 were added to the reactivity table in the labeling.

# Performance testing with archived clinical specimens

The performance when testing clinical specimens was evaluated for the modified Influenza A Subtyping Kit (VER2) including all ZEN quencher probes and the pdmH1 reagents with modified reverse primer and probe sequences. A clinical specimen set was pre-selected from archives of past influenza seasons 2011-2012, 2013-2014 and 2015-2016. The specimens were characterized as positive or negative for influenza using the cleared predicate device or genetic sequence analysis. One group of specimens included forty-two specimens from the 2015-2016 influenza season that produced aberrant results with the current, cleared pdmH1 assay, but were confirmed to be positive with A(H1)pdm09 influenza virus by genetic sequence analysis. These specimens were tested to validate reactivity with the modified InfA, pdmInfA, and pdmH1 assays. A second group of thirty-two specimens, previously confirmed to be positive for A(H3) influenza virus, was tested to validate reactivity with the InfA, and H3 assays containing the ZENTM double-quenched probe. A final group included specimens that were previously confirmed to be negative for influenza A by the cleared InfA assay. This testing was conducted with two enzyme systems (Invitrogen Superscript and Quanta qscript) and one extraction method cleared for use with the Influenza A Subtyping Kit.

The results are summarized in the four tables below.

Retrospective Positive Clinical Study Results-A(H1)pdm09 Comparison   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>InvitrogenSuperScript</td><td rowspan=1 colspan=2>Quanta qScript</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1># ofPositives</td><td rowspan=1 colspan=1>% PositiveAgreement(95% CI)</td><td rowspan=1 colspan=1># ofPositives</td><td rowspan=1 colspan=1>% PositiveAgreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>BW</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1(85.5-99.5)</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0(89.6-100.0)</td></tr><tr><td rowspan=1 colspan=1>NW</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0(51.0-100.0)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0(51.0-100.0)</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.00(34.2-100.0)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.00(34.2-100.0)</td></tr></table>

Retrospective Positive Clinical Study Results-A(H3) Comparison   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Invitrogen SuperScript</td><td rowspan=1 colspan=2>Quanta qScript</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1># ofPositives</td><td rowspan=1 colspan=1>% PositiveAgreement(95% CI)</td><td rowspan=1 colspan=1># ofPositives</td><td rowspan=1 colspan=1>% PositiveAgreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0(20.7-100.0)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0(20.7-100.0)</td></tr><tr><td rowspan=1 colspan=1>NPS, NS</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0(88.7-100.0)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.00(88.7-100.00)</td></tr><tr><td rowspan=1 colspan=1>NW</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0(20.7-100.0)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0(20.7-100.0)</td></tr></table>

Retrospective Negative Clinical Study Results-A(H1)pdm09 Comparison   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>InvitrogenSuperScript</td><td rowspan=1 colspan=2>Quanta qScript</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1># ofNegatives</td><td rowspan=1 colspan=1>% NegativeAgreement(95% CI)</td><td rowspan=1 colspan=1># ofNegatives</td><td rowspan=1 colspan=1>% NegativeAgreement(95% Cl)</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100.00(93.2-100.0)</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100.00(93.1-100.0)</td></tr></table>

Retrospective Negative Clinical Study Results-A(H3) Comparison   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Invitrogen SuperScript</td><td rowspan=1 colspan=2>Quanta qScript</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1># ofNegatives</td><td rowspan=1 colspan=1>% NegativeAgreement(95% CI)</td><td rowspan=1 colspan=1># ofNegatives</td><td rowspan=1 colspan=1>% NegativeAgreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100.00(88.3-100.0)</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.00(87.9-100.0)</td></tr></table>

The results demonstrate high positive and negative percent agreement between the cleared and the modified assays.

# 6. Conclusion

The labeling for this modified subject device has been reviewed to verify that the indication/intended use for the device is unaffected by the modification. In addition, the submitter’s description of the particular modifications and the comparative information between the modified and unmodified devices demonstrate that the fundamental scientific technology has not changed. The submitter has provided the design control information as specified in The New 510(k) Paradigm and on this basis, I recommend the device be determined substantially equivalent to the previously cleared device.