# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM

A. 510(k) Number:

K161814

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay for use with the Solana instrument.

# C. Measurand:

Target RNA sequences of the matrix gene of influenza A virus and influenza B virus.

# D. Type of Test:

The Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay is a qualitative assay utilizing reverse transcription of RNA to DNA followed by an isothermal helicase-dependent amplification of DNA sequences.

# E. Applicant:

Quidel Corporation 2005 East State Street, Suite 100 Athens, OH 45701 USA

F. Proprietary and Established Names:

Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code:

OCC Respiratory viral panel multiplex nucleic acid assay OZE Influenza A and influenza B multiplex nucleic acid assay OOI Real time nucleic acid amplification system

4. Panel:

83- Microbiology

# H. Intended Use:

1. Intended use(s):

The Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay is a qualitative in vitro diagnostic test for the detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Performance characteristics for influenza A were established during the spring of 2016 when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as the intended use, above

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Solana® Instrument

# I. Device Description:

The Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay amplifies and detects viral RNA present in viral transport media containing nasopharyngeal or nasal swab specimens obtained from symptomatic patients.

The assay consists of two major steps: 1) specimen preparation, and 2) amplification and detection of target sequences specific to influenza A and/or influenza B using isothermal Reverse Transcriptase - Helicase-Dependent Amplification (RT-HDA) in the presence of target-specific fluorescence probes.

A patient nasal or nasopharyngeal swab specimen in viral transport media is transferred to a Process Buffer Tube subjected to heat treatment at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes and mixed. The processed sample is transferred to a Reaction Tube. The Reaction Tube contains lyophilized RT-HDA reagents, dNTPs, primers and probes. Once rehydrated with the processed sample, the Reaction Tube is placed in the Solana instrument for amplification and detection of influenza A and influenza B-specific target sequences. In the Solana instrument, the target sequences are amplified by influenza A and influenza B specific primers and detected by influenza A and influenza B specific fluorescence probes, respectively. A competitive process control (PRC) is included in the Process Buffer Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure, or device failure. The PRC target is amplified by influenza B specific primers and detected by a PRC specific fluorescence probe.

The two target probes and PRC probe are labeled with a quencher on one end and a fluorophore on the other end. In addition, the two target probes and PRC probe have one or more bases that are comprised of ribonucleic acid. Upon annealing to influenza A, influenza B or PRC amplicons, the fluorescence probes are cleaved by RNaseH2 and the fluorescence signal increases due to physical separation of fluorophore from quencher. Solana measures and interprets the fluorescent signal, using on-board method-specific algorithms. Solana then reports the test results to the user on its display screen, and it can print out the results via an integrated printer.

The assay kit consists of 48 Process Buffer tubes and 48 Reaction Tubes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

2. Predicate 510(k) number(s): K131728

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Solana® Influenza A+B Assay</td><td>Lyra® Influenza A+B Assay (k131728)</td></tr><tr><td>Intended Use</td><td>The Solana® Influenza A+B Assay is a qualitative in vitro diagnostic test for the detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus. Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Performance characteristics for influenza A were established during the spring of 2016 when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL 3+ facility is available to receive and</td><td>The Lyra® Influenza A+B assay is a multiplex Real Time RT-PCR assay for the in vitro qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus. Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Performance characteristics for influenza A were established during the 2011 and 2013 influenza seasons when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Solana® Influenza A+B Assay</td><td colspan="1" rowspan="1">Lyra® Influenza A+B Assay(k131728)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">culture specimens.</td><td colspan="1" rowspan="1">cases unless a BSL 3+ facility isavailable to receive and culturespecimens.The assay can be performed usingeither the Life TechnologiesQuantStudioTM Dx; the AppliedBiosystems® 7500 Fast Dx, or theCepheid SmartCycler® II.</td></tr><tr><td colspan="1" rowspan="1">SampleTypes</td><td colspan="1" rowspan="1">Nasal swabs and nasopharyngealswabs</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionTechniques</td><td colspan="1" rowspan="1">Automated multiplex assay usingdifferent reporter dyes for each target</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Solana® Influenza A+B Assay</td><td colspan="1" rowspan="1">Lyra® Influenza A+B Assay(k131728)</td></tr><tr><td colspan="1" rowspan="1">Viral Targets</td><td colspan="1" rowspan="1">Influenza A: Matrix Gene;Influenza B: Matrix Gene</td><td colspan="1" rowspan="1">Influenza A: Matrix Gene;Influenza B: conserved influenza Bsequence within the neuraminidasegene</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Reverse Transcriptase - Helicase-Dependent Amplification (RT-HDA)of viral nucleic acid targets</td><td colspan="1" rowspan="1">Real Time PCR-based system fordetecting the presence or absence ofviral RNA</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethods</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">bioMérieux easyMAG® AutomatedMagnetic Extraction Reagents</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Solana</td><td colspan="1" rowspan="1">Life Technologies QuantStudioTMDx, the Applied Biosystems® 7500Fast Dx, or the CepheidSmartCycler® II</td></tr><tr><td colspan="1" rowspan="1">InternalControl</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff: Statistical Guidance on Reporting Results from   
Studies Evaluating Diagnostic Tests (Final, 3/13/2007)   
http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm071287.pdf   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document:   
Respiratory Viral Panel Multiplex Nucleic Acid Assay   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc   
m180307.htm Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) –   
http://www.fda.gov/cdrh/oivd/guidance/1588.pdf. Guidance for Industry and Food and Drug Administration Staff - eCopy Program for Medical Device (December 2012)   
http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo cuments/UCM313794.pdf

# L. Test Principle:

The Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ assay detects viral RNA in nasal and nasopharyngeal specimens. A specimen is collected by a healthcare provider and is placed in viral transport medium to be transported to a clinical laboratory. Once received in the laboratory, the sample is vortexed and $5 0 \mathrm { - \mu L }$ of the specimen is transferred to a Process Buffer Tube containing lysis buffer. The sample is heated to lyse the cells and expose the RNA. An aliquot of the diluted sample is transferred to the Solana Reaction Tube containing lyophilized HDA reagents, dNTPs, primers, and probes, which are rehydrated upon addition of the diluted sample. The Reaction Tube is placed in the Solana instrument for amplification of the influenza A and influenza B-specific target sequences. The detection is achieved through fluorescence from specific fluorophore labeled probes. The entire reaction takes place at $6 5 \mathrm { { ^ \circ C } }$ . A competitive process control (PRC) is included in the lysis tube to monitor sample processing, inhibitory substances, reagent failure, or device failure. The PRC target is amplified by influenza B specific primers and detected by a PRC specific fluorescent probe.

Materials provided with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay kit:

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Quantity</td><td rowspan=1 colspan=1>Storage</td></tr><tr><td rowspan=1 colspan=1>Process Buffer</td><td rowspan=1 colspan=1>48 tubes/kit 1.55 mL</td><td rowspan=1 colspan=1>2° to 8</td></tr><tr><td rowspan=1 colspan=1>Reaction Tubes</td><td rowspan=1 colspan=1>48 tubes/kit</td><td rowspan=1 colspan=1>2°C to 8°</td></tr></table>

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

All analytical studies were conducted in clinical matrix consisting of a pool of nasopharyngeal samples collected by medical personnel from volunteer donors during the winter of 2016. The samples were collected with flocked swabs and then placed into viral transport media. All samples were screened for Influenza A and B negativity with a FDA-cleared molecular device followed by the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay prior to pooling. The pooled negative matrix was stored at ${ - 2 0 ^ { \circ } \mathrm { C } }$ or at $\le - 7 0 ^ { \circ } \mathrm { C }$ for long period storage.

An approximate limit of detection (LoD) concentration for the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ assay was established during earlier range finding studies. The final LoD was evaluated in a study using three influenza A and two influenza B strains that were cultured and titered prior to the study. The viruses were serially diluted in negative nasal matrix to concentrations ranging from below to above the predicted LoD levels. The dilutions were tested in 20 replicates with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay according to the instructions for use. The study was performed using multiple Solana instruments. The experiments included a positive and a negative control for each setup. The LoD was defined as concentration where at least 19 of the 20 replicates were positive for the given virus. The summary of results is shown below.

Limit of Detection for the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay   

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=2>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06 (H1N1)</td><td rowspan=1 colspan=2>7.5x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009(H1N1pdm)</td><td rowspan=1 colspan=2>4.7x102TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012(H3N2)</td><td rowspan=1 colspan=1>6.3x100</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/08 (Victoria)</td><td rowspan=1 colspan=1>8.5x101</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012 (Yamagata)</td><td rowspan=1 colspan=1>3.3x10</td><td rowspan=1 colspan=1>TCID50/mL</td></tr></table>

# b. Precision (Repeatability)

The repeatability of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay was evaluated by testing a four sample panel in triplicate twice daily for twelve non-consecutive days by two operators. Each of the spiked panel members contained Influenza A (A/California/07/2009) and influenza B (B/Brisbane/60/08). A negative sample consisting of unspiked clinical matrix was included in the panel. The samples were prepared at the following targeted concentration:

1. Low positive (1x LoD)   
2. Moderate Positive (2x LoD)   
3. High Negative (below LoD, expected positivity $20 \%$ of the time)   
4. Negative (matrix, no virus spiked)

The summary of the results is shown below.

Repeatability of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>% Detection</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>LowPositive</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>4.7x10</td><td rowspan=1 colspan=1>4.7x102 TCID50/mL</td><td rowspan=1 colspan=1>100% (72/72)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>8.5x10</td><td rowspan=1 colspan=1>8.5x10TCID50/mL</td><td rowspan=1 colspan=1>100%(72/72)</td></tr><tr><td rowspan=2 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>9.4x102</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>100%(72/72)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1.7x102</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>100%(72/72)</td></tr><tr><td rowspan=2 colspan=1>HighNegative</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=2>1.4x10^{2TCID50/mL</td><td rowspan=1 colspan=1>49%(35/72)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>2.6 x101</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>83%(60/72)</td></tr><tr><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=2>No virus</td><td rowspan=1 colspan=1>0% (0/72)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=2>No virus</td><td rowspan=1 colspan=1>0% (0/72)</td></tr></table>

# c. Reproducibility

The reproducibility of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay was evaluated by testing the foursample panel (as described above) in triplicate twice daily for five non-consecutive days by two operators at three testing locations.

A four sample panel consisting of three levels of a combined influenza A and influenza B contrived samples and a negative contrived sample were tested in this study. Influenza A and influenza B viruses (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) were diluted in negative nasal matrix to $2 \mathrm { \Delta x }$ LOD for moderate positive, $1 \mathrm { \Delta x }$ LOD for low positive and to C20 to C80 for high negative / low positive (as described above). Negative nasal matrix without spiked virus was used for the negative sample. The Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay was used according to the instructions for use.

Panels and controls were tested at each site by two operators per instrument for five days, with each sample tested in three replicates, for a total of 90 results per concentration level for each virus for each instrument (2 operators $_ { \textrm { X } 5 }$ days x 3 sites x 3 replicates).

<table><tr><td colspan="10" rowspan="1">Reproducibility Summary for Solana Influenza A+B Assay</td></tr><tr><td colspan="1" rowspan="3">−</td><td colspan="6" rowspan="1">SITE</td><td colspan="2" rowspan="3">OverallPercentAgreement</td><td colspan="1" rowspan="3">95%ConfidenceInterval</td></tr><tr><td colspan="2" rowspan="1">Site #1</td><td colspan="2" rowspan="1">Site #2</td><td colspan="2" rowspan="1">Site #3</td></tr><tr><td colspan="1" rowspan="1">#Detectedpositive/# tested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td><td colspan="1" rowspan="1">#Detectedpostive/# tested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td><td colspan="1" rowspan="1">#Detectedpoositive/# tested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/07/2009High Negative(1.4x102)TCID50/mL)</td><td colspan="1" rowspan="1">10/30</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">25/30</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">23/30</td><td colspan="1" rowspan="1">76.7</td><td colspan="1" rowspan="1">58/90</td><td colspan="1" rowspan="1">64.4</td><td colspan="1" rowspan="1">54.1 to 73.6</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/07/2009Low Positive(4.7x10{TCID50/mL)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.5 to 100</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/07/2009Moderate Positive(9.4x102TCID50/mL)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.5 to 100</td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Brisbane/60/08High Negative (2.6 x10TCID50/L</td><td colspan="1" rowspan="1">9/30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">5/30</td><td colspan="1" rowspan="1">16.7</td><td colspan="1" rowspan="1">10/30</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">24/90</td><td colspan="1" rowspan="1">26.7</td><td colspan="1" rowspan="1">18.6 to 36.6</td></tr><tr><td colspan="1" rowspan="1">#Detectedppossitive/p#ested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td><td colspan="1" rowspan="1">#Detectedppositive/# tested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td><td colspan="1" rowspan="1">#Detectedpositivel/# tested</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Brisbane/60/08Low Positive(8.5x101TCID50/mL)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.5 to 100</td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Brisbane/60/08Moderate Positive(1.7x102TCID50/mL)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.5 to 100</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.5 to 100</td></tr><tr><td colspan="5" rowspan="1">Controls</td><td colspan="5" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A PositiveT CControl</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.2 to 100</td></tr><tr><td colspan="1" rowspan="1">Influenza B PositiveControl</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.2 to 100</td></tr><tr><td colspan="1" rowspan="1">Influenza ANegative Control</td><td colspan="1" rowspan="1">0/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.2 to 100</td></tr><tr><td colspan="1" rowspan="1">Influenza BPositive Control</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.2 to 100</td></tr></table>

d. Linearity/assay reportable range: Not applicable.

e. Sample Stability and Evaluation of Collection Media

Sample stability when stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ was evaluated using six different transport media: BD/Copan UTM, Remel M4, Remel M4RT, Remel M5, Remel M6, or Copan eSwab. Pre-titered viral stocks of one strain of each influenza A and influenza B (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) were added to six transport media (each containing pooled negative matrix), at a final concentration of 2x LoD. Each inoculated medium was divided into 6 aliquots and placed for storage at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ until tested with the Solana $^ \mathrm { \textregistered }$ Influenza $\mathbf { A } { + } \mathbf { B }$ assay. The samples were processed according to the instructions for use. Each transport medium was tested in 3 replicates at Day 0, 24 hours, 48 hours, 72 hours, Day 7, and Day 9. The results showed that influenza A and influenza B are stable in transport media BD UTM, Remel M4, Remel M4RT, Remel M5, and Remel M6 at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 9 days.

Influenza A and influenza B are stable in Copan eSwab transport medium at 2 to $8 \mathrm { { } ^ { \circ } C }$ for up to 48 hours; this was added to the warnings section in the package insert.

# f. Fresh vs. Frozen Stability

The stability of frozen viral RNA stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ was evaluated by testing a four sample panel consisting of three levels of a combined influenza A and influenza B contrived samples in clinical matrix; a negative sample consisted of nasal matrix without spiked virus. Influenza A and Influenza B (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) were diluted to $2 \mathbf { x }$ LOD for moderate positive, 1x LOD for low positive, and to C20 to C80 concentration for a high negative.

Eight replicates of each panel member were tested at the time of preparation. The qualitative result was recorded for each replicate. After storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ for 78 days, eight replicates of each panel member were tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay according to the instructions for use and the qualitative result was recorded for each replicate. Using the Qiagen Studio Software, Detection times (Dt) were established for each positive replicate (this information is not available to the user).

<table><tr><td rowspan=1 colspan=2>Sample</td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Avg.DetectionTime(min.)</td><td rowspan=1 colspan=1>Avg.DetectionTime (min.)</td><td rowspan=2 colspan=1>%ChangeinDetectionTime</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>FrozenStorage</td></tr><tr><td rowspan=3 colspan=2>InfluenzaA</td><td rowspan=1 colspan=1>Low Pos.</td><td rowspan=1 colspan=2>4.7x102 TCID50/mL</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>-4.6%</td></tr><tr><td rowspan=1 colspan=1>Mod. Pos.</td><td rowspan=1 colspan=2>9.4x102 TCID50/mL</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>+3.3%</td></tr><tr><td rowspan=1 colspan=1>High Neg.</td><td rowspan=1 colspan=2>1.4 x102 TCID50/mL</td><td rowspan=1 colspan=1>31.7*</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>+5.0%</td></tr><tr><td rowspan=3 colspan=2>InfluenzaB</td><td rowspan=1 colspan=1>Low Pos.</td><td rowspan=1 colspan=2>8.5x10TCID50/mL</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>+1.6%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mod. Pos</td><td rowspan=1 colspan=2>1.7x102TCID50/mL</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>High Neg.</td><td rowspan=1 colspan=1>2.6 x101</td><td rowspan=1 colspan=1>b TCID50/mL</td><td rowspan=1 colspan=1>29.6**</td><td rowspan=1 colspan=1>28.8***</td><td rowspan=1 colspan=1>-2.7%</td></tr></table>

\* Two of the eight replicates were negative \*\* One of the eight replicates was negative \*\*\* Two of the eight replicates were negative

The results presented in the table above showed that there is no apparent loss in the detectable levels of the viral RNA with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay when samples are stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ for up to 78 days.

# g. Processed Sample Stability - Prior to Heating

Sample stability was evaluated for specimens that were processed with the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Process Buffer and then placed in storage at three temperatures (2 to $8 \mathrm { { } ^ { \circ } C }$ , room temperature, and $- 2 0 ^ { \circ } \mathrm { C }$ ) prior the heat step and analysis with the Solana instrument.

Test samples consisted of pooled negative matrix spiked with Influenza A and B strains (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) at $2 \mathbf { x }$ LOD concentrations. The samples $( 5 0 \mu \mathrm { L } )$ were added to the Process Buffer and vortexed for 5 seconds, according to the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay procedure. The tubes were then placed at each of the designated temperatures for a defined time. All samples were tested in three replicates at each time point: 0 (analyzed without delay), 3 minutes, 10 minutes, 1 hour, 4 hours, 24 hours, and 48 hours at each of the storage temperature conditions. The Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ assay results were positive for all samples tested at each time point after storage at the three evaluated temperatures. The results showed that samples remain reactive after the buffer processing when stored for further analysis for up to 48 hours at 2 to $8 \mathrm { { } ^ { \circ } C }$ , $2 5 \mathrm { { ^ \circ C } }$ and ${ - 2 0 ^ { \circ } \mathrm { C } }$ .

# h. Processed Sample Stability - Post heating

Sample stability was evaluated for specimens that were added to the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Process Buffer, and heated, following by storage at three temperatures (2 to $8 \mathrm { { } ^ { \circ } C }$ , room temperature, and $- 2 0 ^ { \circ } \mathrm { C }$ ) before the analysis step with the Solana instrument. Test samples consisted of pooled negative matrix spiked with Influenza A and B strains (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) at $2 \mathbf { x }$ LOD concentrations. The samples were added to the Process Buffer followed by heating according to the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay procedure, and then placed at each of the designated temperatures for a defined time. All samples were tested in three replicates at each time point: 0 (analyzed without delay), 3 minutes, 10 minutes, 1 hour, 4 hours, 24 hours, and 48 hours at each of the storage temperature conditions. The Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ assay results were positive for all samples tested at each time point after storage at the three evaluated temperatures. The results showed that samples remain reactive after the buffer processing and heating steps when stored for further analysis for up to 48 hours at 2 to $8 \mathrm { { } ^ { \circ } C }$ , $2 5 \mathrm { { ^ \circ C } }$ and ${ } _ { - 2 0 } \mathrm { { \circ C } }$ .

# i. Assay Cutoff

The cutoff value for the Solana $^ \mathrm { \textregistered }$ Influenza $\mathbf { A } { + } \mathbf { B }$ assay was determined based on specific parameters of the amplification curve calculated from negative samples and contrived positive samples at concentrations near the LoD. These parameters included the slope of the amplification curve for samples near the LoD (“slope threshold”) and the time required to reach the slope threshold. The cutoff was set based on the longest amount of time to obtain a positive amplification result.

# j. Analytical Reactivity (inclusivity)

The reactivity of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay was evaluated against multiple strains of influenza A and influenza B viruses. The influenza panel consisted of 14 influenza A strains, and 8 Influenza B strains diluted in clinical matrix to concentrations near the level of detection (LoD) of the assay. The samples were tested in triplicate with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. The study utilized multiple Solana instruments and each run included a positive and a negative control.

All influenza A strains tested positive in all three replicates at $2 . 3 \mathrm { x } 1 0 ^ { 3 }$ T $\mathrm { \ C I D } _ { 5 0 } / \mathrm { m L }$ except for influenza A/Port Chalmers/1/73, which was positive at $1 . 4 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All influenza B strains tested positive in all three replicates at $2 . 6 \mathrm { x } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ except for influenza B/Florida/07/2004 and B/Florida/04/2006 strains, which were positive at $7 . 7 \mathrm { x } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

<table><tr><td rowspan=1 colspan=4>Influenza Virus Strains Evaluated for Inclusivity</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Subtype/Lineage</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Inclusive (Yes or No)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Influenza A</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>H1N1p</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/FM/1/47</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/SolomonIslands/3/06</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/NewCaledonia/20/1999</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/11</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/PortChalmers/1/73</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.4x104*</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>A/WS/33</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/2004</td><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>7.7x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>7.7x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/25/06/04</td><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>2.6x102</td><td rowspan=1 colspan=1>Yes</td></tr></table>

$^ { * } \mathrm { A }$ review of the Port Chalmers sequence with the Solana primers/probes does not indicate any potential issues. The apparent increased concentration may be due to the starting concentration of the virus that was slightly lower than published.

Due to restrictions and availability of a number of influenza A strains, in silico analysis was performed for three additional strain designations:

A total of four H3N2v (1 human strain and 3 swine) sequences were analyzed in silico. All four sequences demonstrated $100 \%$ homology. A total of $3 4 0 \mathrm { H } 5 \mathrm { N } 1$ strains were analyzed in silico. Three hundred thirty-nine (339) strains in the database demonstrated $2 9 5 \%$ overall homology and $2 8 8 \%$ homology to any individual primer or probe sequence. One H5N1 strain demonstrated an overall homology of $8 8 \%$ and $2 8 2 \%$ homology to any individual primer or probe sequence.

A total of $1 6 4 \mathrm { H } 7 \mathrm { N } 9$ sequences were analyzed in silico. All 164 sequences demonstrated $100 \%$ homology.

The sponsor conducted an in silico inclusivity analysis for 14 influenza A viruses found in birds (avian viruses). The primers and probe were aligned to the 27 sequences yielded by the search and the homology percentages were calculated. The Solana Flu A primers and probe are $90 \mathrm { - } 1 0 0 \%$ homologous to the genomes of the tested avian strains shown in the table below.

<table><tr><td rowspan=1 colspan=2>Avian Non-clinical Restricted Influenza A Viruses</td></tr><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Mallard/NY/6750/78 (H2N2)</td></tr><tr><td rowspan=1 colspan=1>H7N3</td><td rowspan=1 colspan=1>A/Chicken/NJ/15086-3/94 (H7N3)</td></tr><tr><td rowspan=1 colspan=1>H9N2</td><td rowspan=1 colspan=1>A/Chicken/NJ/12220/97 (H9N2)</td></tr><tr><td rowspan=1 colspan=1>H4N8</td><td rowspan=1 colspan=1>A/Mallard/OH/338/86 (H4N8)</td></tr><tr><td rowspan=1 colspan=1>H6N2</td><td rowspan=1 colspan=1>A/Chicken/CA/431/00 (H6N2)</td></tr><tr><td rowspan=1 colspan=1>H8N4</td><td rowspan=1 colspan=1>A/Blue Winged Teal/LA/B174/86 (H8N4)</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG5</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>A/GWT/LA/169GW/88 (H10N7)</td></tr><tr><td rowspan=1 colspan=1>H11N9</td><td rowspan=1 colspan=1>A/Chicken/NJ/15906-9/96 (H11N9)</td></tr><tr><td rowspan=1 colspan=1>H12N5</td><td rowspan=1 colspan=1>A/Duck/LA/188D/87 (H12N5)</td></tr><tr><td rowspan=1 colspan=1>H13N6</td><td rowspan=1 colspan=1>A/Gull/MD/704/77 (H13N6)</td></tr><tr><td rowspan=1 colspan=1>H14N5</td><td rowspan=1 colspan=1>A/Mallard/GurjevRussia/262/82 (H14N5)</td></tr><tr><td rowspan=1 colspan=1>H15N9</td><td rowspan=1 colspan=1>A/Shearwater/Australia/2576/79 (H15N9)</td></tr><tr><td rowspan=1 colspan=1>H16N3</td><td rowspan=1 colspan=1>A/Shorebird/DE/172/2006(H16N3)</td></tr></table>

# k. Analytical Specificity

Cross-Reactivity

A study was conducted to demonstrate that the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay does not produce false positive results in the presence of microorganisms that may be present in specimens collected from nasal passages of patients symptomatic for influenza. A panel of 44 microorganisms (24 bacteria, 1 yeast, 19 viruses) was tested in the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay (in the absence of influenza A or influenza B). All microorganisms tested were pre-titered. Each organism was spiked into pooled negative nasal matrix such that the concentration was $\geq \mathrm { 1 x 1 0 ^ { 5 } \ T C I D } _ { 5 0 } / \mathrm { \dot { m } L }$ for viruses or $\geq \mathrm { { l x 1 0 ^ { 6 } C F U / m L } }$ for bacteria, except for Chlamydophila pneumoniae $( 5 \mathrm { x } 1 0 ^ { 4 } \mathrm { C F U / m L } )$ ), Streptococcus pneumoniae $( 1 \mathrm { x } 1 \dot { 0 } ^ { 5 } \mathrm { C F U / m L } )$ ), and Enterovirus 71 $( 2 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ because of lower stock titers.

The samples were tested in 3 replicates with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay according to the instructions for use. The study was performed using multiple Solana instruments. The experiments included a positive and a negative control for each setup.

All 44 microorganisms tested negative for influenza A and influenza B using the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. The organisms tested and their concentrations are listed in the table below.

<table><tr><td colspan="2" rowspan="1">Potential Cross-reactive Organisms</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">5.0x104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.0x104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.0x103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">HSV 1 MacIntyre Strain</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">HSV 2 G strain</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus A1</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.0x10° CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1.0x103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">5.0x104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.0x104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">HSV 1 MacIntyre Strain</td><td colspan="1" rowspan="1">1.0x10 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">HSV 2 G strain</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus A1</td><td colspan="1" rowspan="1">1.0x10 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1.0x10 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">1.0x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.0x10° CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td></tr></table>

It was also shown that high concentrations of influenza A did not cross-react with influenza B and high concentrations of influenza B did not cross-react with influenza A in the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. Each virus evaluated for cross-reactivity was spiked into pooled negative nasal matrix at $\mathrm { 1 x 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$ and the samples were tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay in 3 replicates according to the instructions for use. No cross-reactivity was observed. The results are summarized below.

<table><tr><td rowspan=1 colspan=1>Influenza Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Solana Result</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA/Taiwan/42/06</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td><td rowspan=1 colspan=1>Influenza A PositiveInfluenza B Negative</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB/Panama/45/90</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td><td rowspan=1 colspan=1>Influenza A NegativeInfluenza B Positive</td></tr></table>

# Microbial Interference

A study was conducted to demonstrate that organisms that may be present in specimens collected from nasal passages of patients symptomatic for influenza do not interfere with the detection of influenza A or influenza B in the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ Assay.

A panel of 44 potentially interfering microorganisms (24 bacteria, 1 yeast and 19 viruses) was tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay in triplicate in the presence of influenza A and influenza B viruses. Each microorganism was diluted in negative nasal matrix to the desired concentration ( ${ 1 0 } ^ { 6 }$ or higher $\mathrm { C F U / m L }$ for bacteria and yeast, and $1 0 ^ { 5 }$ or higher pfu/mL or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses). Each sample was spiked with influenza A and influenza B viruses (Influenza A/California/07/2009 and Influenza B/Brisbane/60/08, respectively) for a final concentration equal to $2 \mathbf { x }$ LOD. Each sample was tested in 3 replicates with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay according to the instructions for use.

The study also included samples designed to test whether the detection of influenza A is affected in the presence of Influenza B, and vice versa. One sample was spiked with influenza A at $\mathrm { \bar { 1 } . 0 x 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$ concentration in the presence of influenza B at a low concentration $2 \mathrm { x }$ LoD) and another sample was spiked with influenza B at $1 . 0 \mathrm { x } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ concentration in the presence of influenza A at a low concentration (2x LoD).

No microbial interference was observed. The organisms included in the interference study are shown in the table below.

# Organisms Tested in the Interference Study

<table><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">Legionella pneumophila</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11</td><td colspan="1" rowspan="1">Measles</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Metapneumovirus A1</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Mumps</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">Mycobacterium avium</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">Parainfluenza Type 1</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">Parainfluenza Type 2</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">Parainfluenza Type 3</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">HSV 1 MacIntyre Strain</td><td colspan="1" rowspan="1">Respiratory syncytial virus</td></tr><tr><td colspan="1" rowspan="1">HSV 2 G strain</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Taiwan/42/06</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Panama/45/90</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr></table>

The study also included samples designed to evaluate whether the detection of influenza A is affected in the presence of a high concentration of influenza B, and vice versa. One sample was spiked with influenza A at $1 . 0 \mathrm { x } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ concentration and with influenza $\mathrm { B }$ at a low concentration $( 2 \mathrm { x } \ \mathrm { L o D } )$ . Similarly, another sample was spiked with influenza B at $1 . 0 \mathrm { x } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ concentration and with influenza A at a low concentration $2 \mathrm { x }$ LoD). Each sample was tested in triplicate with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. No inhibition was observed. The results are summarized below.

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Viral Strains</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Solana Result</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>InfluenzaA/Taiwan/42/06</td><td rowspan=1 colspan=1>1.70E+02TCID50/mL</td><td rowspan=2 colspan=1>Influenza A PositiveInfluenza B Positive</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB/Panama/45/90</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>InfluenzaA/Taiwan/42/06InfluenzaB/Panama/45/90</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL1.50E+03TCID50/mL</td><td rowspan=1 colspan=1>Influenza A PositiveInfluenza B Positive</td></tr></table>

# Interfering Substances

The performance of Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay was evaluated with potentially interfering substances that may be present in nasal and nasopharyngeal specimens. The potentially interfering substances were evaluated with influenza A (A/Mexico/ 4108/2009) and influenza B (Influenza B/Brisbane/60/08) at concentrations of 2x LOD. There was no evidence of interference caused by the substances tested at the concentrations shown below.

<table><tr><td colspan="3" rowspan="1">Potential Interfering Substances</td></tr><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">Mucin protein</td><td colspan="1" rowspan="1">2.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Afrin - nasal spray</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Saline nasal spray</td><td colspan="1" rowspan="1">Saline</td><td colspan="1" rowspan="1">15.0%</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">15.0%</td></tr><tr><td colspan="1" rowspan="1">Flonase</td><td colspan="1" rowspan="1">Fluticasone</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Zicam Gentle Allergy ReliefNasalGel</td><td colspan="1" rowspan="1">Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulfur</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">12.0 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">2.2 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">282.0 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">2.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Chloraseptic</td><td colspan="1" rowspan="1">Benzocaine, Menthol</td><td colspan="1" rowspan="1">0.68 g/mL</td></tr><tr><td colspan="1" rowspan="1">Amantadine hydrochloride</td><td colspan="1" rowspan="1">Amantadine hydrochloride</td><td colspan="1" rowspan="1">282.0 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Nasocort Allergy 24 hour</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Sinus Buster Nasal Spray</td><td colspan="1" rowspan="1">Capsicum annuum (Capsaicin)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">NasalCrom Nasal AllergySpray</td><td colspan="1" rowspan="1">Cromolyn Sodium</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Rhinocort</td><td colspan="1" rowspan="1">Budesonide (Glucocorticoid)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Air-Vita Allergy Multi-Symptom Relief</td><td colspan="1" rowspan="1">Allium cepa, Ambrosia artemisiaefolia, Apismellifica, Chamomilla, Eucalyptol,Eucalyptus globulus, Euphrasia officinalis,Galphimia glauca, Histaminumhydrochloricum, Natrum muriaticum, Nuxvomica, Quercus robur, Silicea, Wyethiahelenioides</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Ipratropium bromide</td><td colspan="1" rowspan="1">Ipratropium bromide</td><td colspan="1" rowspan="1">10.0 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Olopatadine hydrochloride</td><td colspan="1" rowspan="1">Olopatadine hydrochloride</td><td colspan="1" rowspan="1">10.0 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Amantadine hydrochloride</td><td colspan="1" rowspan="1">Amantadine hydrochloride</td><td colspan="1" rowspan="1">282 ng/mL</td></tr></table>

# l. Carryover

A study was conducted to verify that the Solana® Influenza $\mathbf { A } { + } \mathbf { B }$ assay is not prone to sample cross-contamination and amplicon carryover. Six highly concentrated samples were run in alternating sequence with 6 negative samples in 5 separate runs on the Solana instrument. Positive samples were contrived in nasal clinical matrix at concentrations greater or equal to $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { \dot { T } C I D } _ { 5 0 } / \mathrm { m L }$ for influenza A (A/Taiwan/42/06) and for influenza B (B/Panama/45/90). Negative samples consisted of pooled negative nasal matrix. All positive influenza A and influenza B samples tested positive and all negative samples tested negative in all runs. No carryover was observed. The results are summarized below.

Carryover Study   

<table><tr><td rowspan=1 colspan=1>Input</td><td rowspan=1 colspan=1># Run</td><td rowspan=1 colspan=1># Positive</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Matrix</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

# m. Controls

The Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay contains an internal process control that is included in the lysis buffer tube and is used to detect HDA inhibitory substances in clinical samples and to confirm the integrity of assay reagents and the operation of the Solana instrument. The internal control monitors the entire extraction, amplification and detection process.

The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Control Set is available as an accessory to the assay and consists of a mixture of highly purified, inactivated strains of Influenza A/New Caledonia/20/99 (H1N1), Influenza B/Florida/04/06 and an influenza RNA‐free matrix. The controls are intended to be used in the same manner as patient samples and monitor for substantial reagent and instrument failure. The Positive control is manufactured at concentrations near the LoD of the assay. The Negative control serves to detect reagent or environment contamination by influenza A or B RNA or amplicons.

# 2. Clinical study:

A multi-center field clinical study was conducted between February and April 2016 to evaluate the performance characteristics of the Solana $^ \mathrm { \textregistered }$ Influenza $\mathbf { A } { + } \mathbf { B }$ Assay with nasal and nasopharyngeal swabs from 1473 patients with signs and symptoms of respiratory infection. A single nasal or nasopharyngeal swab specimen (302 and 1171, respectively) was collected from each patient and placed in viral transport media (BD/Copan UTM, Remel M5, or Remel M6). All specimens were processed and tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay on the Solana instrument at the clinical sites. Of the 1473 collected specimens, 742 specimens were tested fresh, while 731were frozen and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ prior to testing with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay.

An aliquot of each fresh specimen was transported at $2 \mathrm { { } ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ to a central location for testing by the comparator methods. Sensitivity and specificity estimates were calculated using a Composite Reference Method that consisted of direct specimen fluorescent antibody (DSFA) test and culture with DFA (using R-Mix Too mixed cells and the D3 Duet™ DFA Influenza A / Respiratory Virus Screening Kit from Diagnostic Hybrids). A specimen was considered positive if either test was positive.

Each sample was also tested with a FDA cleared molecular influenza $\mathrm { A } / \mathrm { B }$ assay for calculations of percent agreement between the two devices. The testing with the molecular comparator involved extraction with the NucliSENS® easyMAG® followed by testing with a FDA-cleared Influenza $\mathbf { A } { + } \mathbf { B }$ molecular assay.

The gender and age of the patients enrolled in the study are shown below.

<table><tr><td rowspan=1 colspan=3>Study Population by Gender and Age</td></tr><tr><td rowspan=1 colspan=1>Gender*</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>672</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>197</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>167</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>328</td><td rowspan=1 colspan=1>197</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>111</td></tr></table>

\*Three specimens did not have gender or age provided.

a. Clinical Performance of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay against Culture with DFA and DSFA

There were 1473 specimens tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay and with the comparator, viral culture with DFA and also with DSFA. A specimen was recorded as positive for influenza A or B if either comparator test was positive. All comparator testing was performed on fresh specimens within 72-hours of their collection. Of the 1473 specimens collected at the clinical sites, 742 were tested fresh with the Solana assay while 731 were frozen and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ prior to testing with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. Of the specimens tested, 65 specimens were excluded from data analysis (15 specimens were contaminated or toxic in the cell culture and 50 specimens were invalid in the Solana Assay) leaving 1408 evaluable specimens for calculations of sensitivity and specificity, as shown below.

<table><tr><td rowspan=1 colspan=8>Performance Characteristics of the Solana Influenza A+B Assay for Influenza ACompared to Culture and DSFA (Across all Sites Combined)</td></tr><tr><td rowspan=1 colspan=1>SampleCategory</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Sensitivity%(95% CI)</td><td rowspan=1 colspan=1>Specificity%(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>709</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>503</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.9(96.1 to 99.7)</td><td rowspan=1 colspan=1>95.4(93.3 to 96.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>699</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.3(95.2 to 99.4)</td><td rowspan=1 colspan=1>94.8(92.6 to 96.4)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1408</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>51*</td><td rowspan=1 colspan=1>996</td><td rowspan=1 colspan=1>5**</td><td rowspan=1 colspan=1>98.6(96.8 to 99.4)</td><td rowspan=1 colspan=1>95.1(93.7 to 96.3)</td></tr></table>

\* Of the 51 discordant specimens (Solana Positive/Culture and DSFA Negative), 28 were positive by an alternate FDA cleared molecular assay. $^ { * * } \mathrm { O f }$ the 5 discordant specimens (Solana Negative/Culture and DSFA Positive) 2 were positive by an alternate FDA cleared molecular assay.

Influenza B   

<table><tr><td rowspan=1 colspan=8>Performance Characteristics of the Solana Influenza A+B Assay for Influenza BCompared to Culture and DSFA (Across all Sites Combined)</td></tr><tr><td rowspan=1 colspan=1>SourceCategory</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Sensitivity%(95% CI)</td><td rowspan=1 colspan=1>Specificity%(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>709</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>646</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(94.2 to 100)</td><td rowspan=1 colspan=1>99.8(99.1 to 100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>699</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>668</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(85.7 to 100)</td><td rowspan=1 colspan=1>98.8(97.7 to 99.4)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1408</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>9*</td><td rowspan=1 colspan=1>1314</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(95.7 to 100)</td><td rowspan=1 colspan=1>99.3(98.7 to 99.6)</td></tr></table>

\*Of the nine discordant specimens (Solana Positive/Culture and DSFA Negative), two were positive by an alternate FDA cleared molecular assay.

# b. Clinical Performance of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay Compared with FDA Cleared Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

There were 1473 specimens tested with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay and with a FDAcleared influenza $\mathbf { A } { + } \mathbf { B }$ molecular comparator assay. All comparator testing was performed on fresh specimens within 72-hours of their collection and utilized extraction of nucleic acids with the NucliSENS® easyMAG®. Of the 1473 specimens collected at the clinical sites, 742 were tested fresh with the Solana assay while 731 were frozen and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ prior to testing with the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay. Eighty specimens were excluded from data analysis due to invalid results (31 specimens were invalid in the comparator assay $( 2 . 1 \% )$ and 50 specimens were invalid in the Solana® Assay $( 3 . 4 \% )$ ; one specimen was invalid in both assays). The positive percent agreement (PPA) and the negative percent agreement (NPA) of the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay results for influenza A and influenza B, as compared with an FDA cleared molecular comparator for the remaining 1393 evaluable specimens are shown below.

Influenza A   

<table><tr><td rowspan=1 colspan=8>Percent Agreement of the Solana® Influenza A+B Assay for Influenza ACompared to FDA cleared Molecular Assay (Across all Sites Combined)</td></tr><tr><td rowspan=1 colspan=1>SampleCategory</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=1>NPA (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>710</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>96.5 (93.0 to 98.3)</td><td rowspan=1 colspan=1>98.2 (98.7 to 99.1)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>683</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>475</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>97.8 (94.5 to 99.2)</td><td rowspan=1 colspan=1>95.2 (92.9 to 96.7)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1393</td><td rowspan=1 colspan=1>375</td><td rowspan=1 colspan=1>33*</td><td rowspan=1 colspan=1>974</td><td rowspan=1 colspan=1>11**</td><td rowspan=1 colspan=1>97.2 (95.0 to 98.4)</td><td rowspan=1 colspan=1>96.7 (95.4 to 97.7)</td></tr></table>

\* Of the 33 discordant specimens (Solana Positive/Comparator Negative), nine specimens were positive by culture/DSFA. \*\* Of the 11 discordant specimens (Solana Negative/ Comparator Positive), two were positive by culture/DSFA.

# Influenza B

<table><tr><td rowspan=1 colspan=8>Percent Agreement of the Solana® Influenza A+B Assay for Influenza BCompared to FDA cleared Molecular Assay (Across all Sites Combined)</td></tr><tr><td rowspan=1 colspan=1>SourceCategory</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=1>NPA (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>710</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>647</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100 (93.7 to 100)</td><td rowspan=1 colspan=1>99.1 (98.0 to 99.6)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>683</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>652</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100 (85.7 to 100)</td><td rowspan=1 colspan=1>98.8 (97.6 to 99.4)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1393</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>1299</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100 (95.4 to 100)</td><td rowspan=1 colspan=1>98.9 (98.2 to 99.4)</td></tr></table>

\*Of the 14 discordant specimens (Solana Positive/ Comparator Negative), seven were positive by culture/DSFA.

b. Other clinical supportive data: Not applicable.

4. Clinical cut-off:

Not applicable.

5. Expected Values

The expected values for the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay were established during a prospective study conducted between February and April 2016 collected and tested at five clinical sites across the United States. The positivity rate for the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ assay (combined across the sites) based on 1420 evaluable specimens is shown below, stratified by age.

<table><tr><td rowspan=1 colspan=7>Expected Values (N=1420)</td></tr><tr><td rowspan=2 colspan=1>Age Group</td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=1 colspan=3>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Numberof Patients</td><td rowspan=1 colspan=1>NumberofPositives</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>NumberofPatients</td><td rowspan=1 colspan=1>NumberofPositives</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>24.1%</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>6.9%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>16.2%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>37.9%</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3.4%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>15.3%</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2%</td></tr></table>

The prospective clinical study had a dual infection rate for Influenza A and Influenza B of $0 . 2 \%$ (3/1420) as determined by the Solana Influenza $\mathbf { A } { + } \mathbf { B }$ Assay. All three of these dual detections were only positive for influenza A by culture and DSFA and also by an alternate molecular comparator.

N. Instrument Name: Solana $^ \mathrm { \textregistered }$ Instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No X

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

The Solana® instrument platform was originally reviewed under k150868. The additional information was provided in support of the Solana $^ \mathrm { \textregistered }$ Influenza $\mathbf { A } { + } \mathbf { B }$ Assay and was reviewed and found acceptable;

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification:

Specimens are identified by scanning a barcode or by manual entry.

4. Specimen Sampling and Handling:

Swab specimens are collected in transport medium. After vortexing, $5 0 \mu \mathrm { l }$ of the expressed specimen is transferred to a lysis tube. After heat lysis, $5 0 \mu \mathrm { l }$ of lysed specimen is transferred to a reaction tube for automated amplification and detection.

5. Calibration:

The end user is not required to calibrate the instrument.

6. Quality Control:

Process Control:

The process control is used to monitor sample processing, to detect HDA inhibitory substances, to confirm the integrity of assay reagents and the operation of the Solana instrument. The process control is included in the Reaction Mix tube and is run with each sample.

# External Controls:

The external controls are available from Quidel as an accessory to the assay. The controls consist of one vial containing Positive Control and one vial containing Negative Control. These controls are intended to monitor pre-analytical and environmental factors that could substantially affect reagent integrity or the instrument function. The negative control is used to detect reagent or environmental contamination of the system by influenza A or B RNA or amplicons.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.