# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K162444 B. Purpose for Submission:

To obtain a substantial Equivalence Determination for a new 510(k) application for the Cepheid Xpert MRSA NxG Assay on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s and GeneXpert Infinity-80 systems).

# C. Measurand:

DNA sequences for:

(1) the mecA and mecC genes for methicillin resistance;   
(2) the junction of the staphylococcal cassette chromosome mec (SCCmec) and the Staphylococcus aureus orfX gene.

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for amplification and detection of DNA target sequences for methicillin-resistant Staphylococcus aureus (MRSA).

# E. Applicant:

Cepheid

F. Proprietary and Established Names: Xpert MRSA NxG

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640: Antimicrobial test powder

2. Classification: Class II

3. Product code:

NQX: System, nucleic acid amplification test, DNA, methicillin resistant Staphylococcus aureus, direct specimen OOI: Instrumentation for clinical multiplex test systems

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The Xpert® MRSA NxG Assay, performed on the GeneXpert® Instrument Systems, is a qualitative in vitro diagnostic test intended for the detection of methicillin-resistant Staphylococcus aureus (MRSA) DNA directly from nasal swabs in patients at risk for nasal colonization. The test utilizes automated real-time polymerase chain reaction (PCR) for the amplification of MRSA-specific DNA targets and fluorogenic target-specific hybridization probes for the real-time detection of the amplified DNA. The Xpert MRSA NxG Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings. The Xpert MRSA NxG Assay is not intended to diagnose, guide, or monitor treatment for MRSA infections, or provide results of susceptibility to methicillin. A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s):

Prescription use only.

The Xpert MRSA NxG Assay may generate a false positive MRSA (MRSA DETECTED) result when testing a nasal specimen with a mixture of organisms containing both methicillin-resistant coagulase-negative Staphylococcus and an empty cassette SA.

The Xpert MRSA NxG Assay may generate a false negative result (MRSA NOT DETECTED) in the event of a co-colonization that contains both methicillin-resistant Staphylococcus aureus (MRSA) and an empty cassette Staphylococcus aureus (SA). This may occur in rare cases when the titer of an empty cassette SA organism is substantially higher than that of the MRSA organism.

Assay interference may be observed in the presence of Nasonex $( \geq 5 0 \% \mathrm { v } / \mathrm { v } )$ ), Flonase $( \geq 5 0 \% \mathrm { v } / \mathrm { v } )$ ), and Beconase $( \geq 4 0 \% \mathrm { v } / \mathrm { v } )$ ).

4. Special instrument requirements:

The Xpert MRSA NxG assay is for use on the GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s and GeneXpert Infinity-80 instrument systems.

# I. Device Description:

The Xpert MRSA NxG Assay is an automated real-time polymerase chain reaction (PCR) in vitro diagnostic test for qualitative detection of methicillin-resistant Staphylococcus aureus (MRSA) directly from nasal swab specimens of patients at risk for colonization with MRSA in a healthcare setting.

The Xpert MRSA NxG Assay is performed on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems), which automate sample preparation, amplification and real-time detection in single-use, disposable cartridges.

Each Xpert MRSA NxG cartridge contains primers and probes to detect sequences within the mecA and mecC genes, as well as insertion of the staphylococcal cassette chromosome mec (SCCmec) at the S. aureus chromosome attB site. The assay incorporates a Sample Processing Control (SPC) and a Probe Check Control (PCC) to monitor the integrity of the reagents and process workflow.

The GeneXpert Instrument Systems have between 1 and 80 randomly accessible modules, depending upon the instrument, that are each capable of performing separate sample preparation and real-time PCR assays. Each module contains a syringe drive for dispensing fluids, an ultrasonic horn for lysing cells, and a thermocycler unit for real-time PCR amplification and detection.

Once the instrument is loaded and the test initiated, all the steps associated with sample processing, PCR amplification/detection and result interpretation occur automatically. The final result report can be viewed and/or printed.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX™ MRSA XT 2. Predicate 510(k) number(s): K133605

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Cepheid Xpert MRSANxG Assay (K162444)</td><td rowspan=1 colspan=1>BD MAX MRSA XTAssay (K133605)</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>21 CFR 866.1640</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>NQX</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Device Class</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=7 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The Xpert MRSA NxGAssay, performed on theGeneXpert InstrumentSystems, is a qualitative invitro diagnostic testintended for the detection ofmethicillin-resistantStaphylococcus aureus(MRSA) DNA directly fromnasal swabs in patients atrisk for nasal colonization.The test utilizes automatedreal-time polymerase chainreaction (PCR) for theamplification of MRSA-specific DNA targets andfluorogenic target-specific</td><td rowspan=7 colspan=1>The BD MAX MRSA XTassay performed on the BDMAX System is anautomated qualitative invitro diagnostic test for thedirect detection ofmethicillin-resistantStaphylococcus aureus(MRSA) DNA from nasalswabs in patients at risk fornasal colonization. The testutilizes real-timepolymerase chain reaction(PCR) for the amplificationof MRSA DNA andfluorogenic target-specifichybridization probes for thedetection of the amplifiedDNA. The BD MAXMRSA XT assay isintended to aid in theprevention and control ofMRSA infections inhealthcare settings. It is notintended to diagnoseMRSA infections nor guideor monitor treatment forMRSA infections. Anegative result does notpreclude nasalcolonization. Concomitantcultures are necessary torecover organisms forepidemiological typing orfor further susceptibilitytesting.</td></tr><tr><td rowspan=1 colspan=1>hybridization probes for thereal-time detection of theamplified DNA. The XpertMRSA NxG Assay is</td></tr><tr><td rowspan=1 colspan=1>intended to aid in theprevention and control ofMRSA infections in</td></tr><tr><td rowspan=1 colspan=1>healthcare settings. The</td></tr><tr><td rowspan=1 colspan=1>Xpert MRSA NxG Assay is</td><td rowspan=1 colspan=1>s nor gu</td></tr><tr><td rowspan=1 colspan=1>not intended to diagnose,guide, or monitor treatmentfor MRSA infections, orprovide results ofsusceptibility to methicillin.</td></tr><tr><td rowspan=1 colspan=1>A negative result does notpreclude MRSA nasalcolonization. Concomitantcultures are necessary torecover organisms forepidemiological typing or</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Cepheid Xpert MRSANxG Assay (K162444)</td><td rowspan=1 colspan=1>BD MAX MRSA XTAssay (K133605)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>for further susceptibilitytesting.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mode of detectingmethicillin resistance</td><td rowspan=1 colspan=1>Concomitant detection ofthe mecA or mecC genesand the junction of theSCCmec cassette with the S.aureus orfX gene</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Nasal swabs</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technological Principles</td><td rowspan=1 colspan=1>PCR amplification and real-time detection usingfluorogenic reporter probes</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Sample Processing Control</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Cepheid Xpert MRSANxG Assay (K162444)</td><td rowspan=1 colspan=1>BD MAX MRSA XTAssay (K133605)</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>Cepheid GeneXpertInstrument Systems(GeneXpert Dx, GeneXpertInfinity-48, GeneXpertInfinity-48s, and GeneXpertInfinity-80 systems)</td><td rowspan=1 colspan=1>BD MAX System</td></tr><tr><td rowspan=1 colspan=1>Early Assay TerminationFunction</td><td rowspan=1 colspan=1>Yes (for positive samples)</td><td rowspan=1 colspan=1>No</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

1. CLSI. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - $3 ^ { r d }$ Edition. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

2. CLSI. User Verification of Precision and Estimation of Bias; Approved Guideline - $3 ^ { r d }$ Edition. CLSI document EP15-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

3. CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - $2 ^ { n d }$ Edition. CLSI document EP17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

4. CLSI. Molecular Diagnostic Methods for Infectious Diseases - $3 ^ { r d }$ Edition. CLSI report MM03. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.

5. CLSI. Interference Testing in Clinical Chemistry; Approved Guideline - $2 ^ { n d }$ Edition. CLSI document EP07-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2005.

6. CLSI. Performance Standards for Antimicrobial Disk Susceptibility Tests; Approved Standard - $. l 2 ^ { t h }$ Edition. CLSI document M02-A12. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.   
7. CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - $2 ^ { n d }$ Edition. CLSI document EP12-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.   
8. CLSI. Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria That Grow Aerobically; Approved Standard - $I O ^ { t h }$ Edition. CLSI document M07-A10. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.   
9. CLSI. Performance Standards for Antimicrobial Susceptibility Testing - $2 \mathcal { 4 } ^ { t h }$ Edition. CLSI supplement M100S. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.   
10. ASTM D4169-09, Standard Practice for Performance Testing of Shipping Containers and Systems, 2014.   
11. EN 13640: Stability Testing of In Vitro Diagnostic Reagents; June, 2002.   
12. Guidance for Industry and FDA Staff: Class II Special Controls Guidance Document - Instrumentation for Clinical Multiplex Test Systems. FDA Guidance Instrumentation for Clinical Multiplex Test System; March 10, 2005.   
13. Guidance for Industry and FDA Staff: Guidance for the Content of Pre-market Submissions for Software Contained in Medical Devices; May 11, 2005.   
14. Guidance for Sponsors, Institutional Review boards, Clinical Investigators and FDA Staff: Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable; April 25, 2006.   
15. Guidance for Industry, FDA Reviewers and Compliance on: Off-the-Shelf Software Use in Medical Devices; September 9, 1999.   
16. Guidance for Industry: Cybersecurity for Networked Medical Devices Containing Offthe-Shelf (OTS) Software; January, 2005.   
17. Guidance for Industry and FDA Staff: Content of Premarket Submissions for Management of Cybersecurity in Medical Devices; October 2, 2014.   
18. General Principles of Software Validation; Final Guidance for Industry and FDA Staff; January 11, 2002.   
19. Federal Communication Commission Part 15, Subparts A and B.   
20. Federal Communication Commission Part 18.   
21. IEC CISPR-11 Industrial, Scientific, and Medical Equipment Radio Disturbance Characteristics - Limits and Methods of Measurement; 2004.   
22. IEC CISPR-22 Information technology equipment - Radio Disturbance Characteristics - Limits and Methods of Measurement; 2006.   
23. IEC 61010-1 Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 1, General Requirements; 2001.   
24. EN 61010-1 Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 1, General Requirements; 2001.   
25. UL 61010-1 Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 1, General Requirements; 2004.   
26. IEC 61010-2-101 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 2-101: Particular requirements for In-Vitro Diagnostic (IVD) Medical Equipment; 2002.   
27. EN 61010-2-101 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 2-101: Particular requirements for In-Vitro Diagnostic (IVD) Medical Equipment; 2002.   
28. EN 61326-1 Electrical Equipment for Measurement, Control, and Laboratory Use – EMC Requirements: General Requirements; 2006.   
29. EN 61326-2-6 Electrical Equipment for Measurement, Control, and Laboratory Use – EMC Requirements – Part 2-6; 2006.   
30. EN 55011 Industrial, Scientific and Medical (ISM) Radio-Frequency Equipment – Electromagnetic Disturbance Characteristics; 2010.   
31. CAN/CSA C22.2 No. 61010-1 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 1: General Requirements; 2004.   
32. CAN/CSA C22.2 No. 61010-2-101 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 2-101: Particular Requirements for InVitro Diagnostic (IVD) Medical Equipment; 2004.   
33. LVD (Low Voltage Directive) 2014/35/EU.   
34. WEEE Directive 2002/96/EC.   
35. EMC (Electromagnetic Compatibility) Directive, 2004/108/EC.

# L. Test Principle:

The Xpert MRSA NxG Assay is performed on the GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems) which automate nucleic acid extraction, amplification and detection in single-use, disposable cartridges. Each cartridge contains primers and probes for detection of the mecA and mecC genes that encode methicillin resistance, as well as the junction of the SCCmec cassette and the S. aureus orfX open reading frame that is formed by insertion of the cassette into the bacterial chromosome at the attB integration site.

Each cartridge also includes a Sample Processing Control and Probe Check Control that are designed to monitor reagent and process integrity.

Nasal swab specimens for testing in the Xpert MRSA NxG Assay are collected using either the Cepheid Sample Collection Device (dual rayon swab) or the Copan Liquid Amies Elution Swab (ESwab™), or equivalent, and are transported to the laboratory or designated GeneXpert testing area. For the Cepheid Sample Collection Device, one swab is placed directly into a tube containing $2 \mathrm { m L }$ of Elution Reagent and, after a brief vortex, the entire volume is transferred to the Xpert MRSA NxG cartridge. For samples collected using the ESwab, $3 0 0 \mu \mathrm { L }$ of the Amies transport medium is mixed with $2 \mathrm { m L }$ of Elution Reagent, after which all $2 . 3 \mathrm { m L }$ of liquid are transferred to the test cartridge.

The operator initiates a test from the user interface and loads the cartridge into the GeneXpert instrument, after which all process steps are performed automatically. Once complete, the final result report can be viewed and printed. Results are reported as MRSA DETECTED, MRSA NOT DETECTED, ERROR, INVALID or NO RESULT (Table 1). Instructions for retesting are provided for samples with indeterminate results. For a report of MRSA DETECTED, detection of both the mecA/mecC and SCCmec targets is required.

The Xpert MRSA NxG Assay has an Early Termination Feature whereby results for samples that are strongly positive for MRSA are reported prior to completion of the full number of PCR cycles designated in the assay definition file.

Table 1. Summary of Xpert MRSA NxG result interpretation   

<table><tr><td colspan="1" rowspan="1">Result</td><td colspan="1" rowspan="1">Interpretation</td></tr><tr><td colspan="1" rowspan="1">MRSA DETECTED</td><td colspan="1" rowspan="1">MRSA DNA is detectedMRSA:•mecA/mecC and SCCmec targets have a Ct within the valid range•Sample Processing Control:•Not applicable - the SPC signal is not used in the result interpretationalgorithm if MRSA is detectedProbe Check Control:PASS: All probe check results pass</td></tr><tr><td colspan="1" rowspan="1">MRSA NOT DETECTED</td><td colspan="1" rowspan="1">MRSA DNA is not detectedMRSA Possible scenarios:•Target DNA for neither mecA/mecC nor SCCmec is detectedTarget DNA for mecA/mecC is detected and target DNA for SCCmecis not detectedTarget DNA for mecA/mecC is not detected and target DNA forSCCmec is detectedSample Processing Control:•PASS - SPC has a Ct within the valid range and neither mecA/mecCnor SCCmec target DNA are detectedIf either mecA/mecC or SCCmec exhibit a valid Ct, the SPC result isignoredProbe Check Control:PASS: All probe check results pass</td></tr><tr><td colspan="1" rowspan="1">INVALID</td><td colspan="1" rowspan="1">Presence or absence of MRSA target DNA (mecA/mecC or SCCmec) cannot bedetermined. Repeat the test.MRSA:Target DNA for neither mecA/mecC nor SCCmec is detected•Sample Processing Control:FAIL: SPC Ct is not within the valid rangeProbe Check Control:PASS: All probe check results pass</td></tr><tr><td colspan="1" rowspan="1">ERROR</td><td colspan="1" rowspan="1">Presence or absence of MRSA target DNA (mecA/mecC or SCCmec) cannot bedetermined. Repeat the test.MRSA:mecA/mecC: No ResultSCCmec: No ResultSample Processing Control:No Result•Probe Check Control:FAIL: one or more Probe Check results failed•If the Probe Check Control passed, the error was caused by a systemcomponent failure</td></tr><tr><td>NO RESULT</td><td>Presence or absence of MRSA target DNA (mecA/mecC or SCCmec) cannot be determined. Repeat the test. A NO RESULT indicates that insufficient data were collected. MRSA: mecA/mecC: No Result SCCmec: No Result Sample Processing Control: No Result</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

To estimate the components of variance associated with the Xpert MRSA NxG Assay a Reproducibility Study was conducted at 3 sites, with 2 operators per site. Each operator tested a 5 member panel of samples twice per day over 6 days using 3 lots of reagents (3 sites x 2 operators/site x 2 panels/day x 2 replicates/panel member x 6 days $= 1 4 4$ results/panel member). Each site conducted testing on a different GeneXpert System; Site 1: Infinity-80, Site 2: GeneXpert Dx; Site 3: Infinity-48. Samples were prepared in simulated nasal matrix and frozen until testing. Separate studies were conducted to demonstrate equivalent assay performance with simulated and natural nasal matrix as well as with fresh and frozen samples (Section M(2)(b)). The panel members for the Reproducibility Study are summarized in Table 2.

Table 2. Summary of Reproducibility Study panel members   

<table><tr><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Multiple of LOD</td><td rowspan=1 colspan=1>CFU/swab1</td></tr><tr><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>SCCmec Type XILGA2512</td><td rowspan=1 colspan=1>Low Positive-1</td><td rowspan=1 colspan=1>1.1X</td><td rowspan=1 colspan=1>285</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive-1</td><td rowspan=1 colspan=1>3.2X</td><td rowspan=1 colspan=1>855</td></tr><tr><td rowspan=2 colspan=1>SCCmec Type IIN315}$</td><td rowspan=1 colspan=1>Low Positive-2</td><td rowspan=1 colspan=1>0.8X</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive-2</td><td rowspan=1 colspan=1>2.3X</td><td rowspan=1 colspan=1>375</td></tr></table>

1 Based on the workflow for the Cepheid Sample Collection Device (rayon swab) 2 mecC positive 3 mecA positive

Percent agreement for strain LGA251 was $100 \%$ (144/144) at both target levels. For strain N315, agreement was $100 \%$ (144/144) for the Moderate Positive samples and $9 8 . 6 \%$ (142/144) for the Low Positive samples. For negative samples, agreement was $100 \%$ (144/144). ANOVA (Analysis Of Variance) was performed to assess the variance components of the Ct values for the mecA/mecC, SCCmec and SPC targets (Table 3). The total variation for all targets was between 0.8 and $1 . 1 \mathrm { C t } ( 2 . 6 \substack { - 3 . 8 \% } )$ .

The Xpert MRSA NxG Assay demonstrated acceptable reproducibility across sites, GeneXpert systems, operators, panel members and reagent lots.

Table 3. Summary of Ct variance components observed in the Reproducibility Study $\mathrm { \Delta } _ { \mathrm { n } } = 1 4 4$ per sample)   

<table><tr><td rowspan=3 colspan=1>MRSAStrain</td><td rowspan=3 colspan=3>Sample</td><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=13>Ct</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Between Day</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>Within Assay</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=3>Negative</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=6 colspan=1>LGA2511</td><td rowspan=2 colspan=3>LowPositive</td><td rowspan=1 colspan=1>Mec</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>SCCmec</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=4 colspan=3>Moderate Positive</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Mec</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>SCCmec</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=4 colspan=1>N3152</td><td rowspan=2 colspan=3>LowPosiive</td><td rowspan=1 colspan=1>Mec</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>SCCmec</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=3>Moderate</td><td rowspan=1 colspan=1>Mec</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=3> Psitive</td><td rowspan=1 colspan=1>SCCmec</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td></tr></table>

NA: Not Applicable; SPC: Sample Processing Control; SD: Standard Deviation; $\% \mathrm { C V }$ : Percent Coefficient of Variation 1 SCCmec Type XI, mecC positive 2 SCCmec Type II, mecA positive Note: There were a total of 12 indeterminate results over the course of the study (11 reported as “Error” and 1 as “Invalid”). All 12 samples produced valid test results upon repeat.

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Controls

Commercially available External Positive and Negative Controls were run each day that specimens were tested during the two prospective Clinical Studies to evaluate the performance of the Xpert MRSA NxG Assay (Section M(3)). Before testing of any clinical specimens, the expected results had to be obtained with both External Controls. The workflow for testing of the External Controls was the same in both studies and is described in the Xpert MRSA NxG Assay Package Insert. A summary of the External Control results from both studies, as well as the two studies combined, is provided in Table 4.

Table 4. Summary of External Control performance in the prospective Clinical Studies   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Study 1</td><td rowspan=1 colspan=2>Study 2</td><td rowspan=1 colspan=2>Studies 1 &amp; 2 Combined</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Tested</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>327</td><td rowspan=1 colspan=1>334</td></tr><tr><td rowspan=1 colspan=1>Expected Result</td><td rowspan=1 colspan=1>2091</td><td rowspan=1 colspan=1>2142</td><td rowspan=1 colspan=1>1063</td><td rowspan=1 colspan=1>1074</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>321</td></tr><tr><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>95.4</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>96.1</td></tr></table>

Study 1: Evaluation of performance with Copan Sample Collection Device (rayon swab); Study 2: Evaluation of performance with ESwab 1 MRSA Not Detected: 3; Error: 7 2 MRSA Detected: 2; Error: 5; Invalid: 1; No Result: 1 3 Error: 2 4 Error: 3; Invalid: 1

An additional evaluation of the recommended External Controls was conducted by testing 3 lots of each control material with 3 lots of Xpert MRSA NxG Assay reagents. Of 180 External Negative Controls tested, 174 $( 9 6 . 7 \% )$ produced the expected result on initial testing, as did 177/180 $( 9 8 . 3 \% )$ External Positive Controls.

Internal Controls   
Each Xpert MRSA NxG Assay includes two types of Internal Control to monitor the   
performance of the system

Sample Processing Control (SPC): Designed to monitor bacterial lysis and sample inhibition. Test results are reported as INVALID if the SPC fails.

Probe Check Control (PCC): Fluorescence measurement to monitor probe rehydration and integrity, as well as filling of the reaction tube. Test results are reported as ERROR if the PCC fails.

The performance of the Internal Controls was verified by testing various potential failure modes due to errors in the manufacture of the Xpert MRSA NxG Assay reagents or the assay procedure. Under the conditions tested, the Internal Controls failed appropriately and no false positive or false negative results for the detection of MRSA were reported.

# Sample Stability

The stability of nasal swab specimens for use with the Xpert MRSA NxG Assay was determined analytically by testing two strains of MRSA that were seeded onto swabs at target levels close to the LOD of the assay. MRSA negative samples were also included in order to assess the effect of sample storage on the performance of the SPC. The study was performed with both the Cepheid Sample Collection Device (rayon swab) and ESwab using simulated nasal matrix (Section M(2)(b)). Samples were placed at different temperatures and tested at specified intervals. Results were analyzed qualitatively and by ANOVA to assess differences in Ct values for the mecA/mecC, SCCmec and SPC targets over time. No MRSA false negative or false positive results were observed during the course of the study and there were no important differences in Ct values between samples tested at baseline (Day 0) and those tested at any time point in the study. This study indicates that samples collected with both swab types are stable for up to 24 hours at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ and up to 7 days at 2- $8 \mathrm { { } ^ { \circ } C }$ .

# Cartridge Hold Time

After loading onto the GeneXpert Infinity System, assay cartridges may have to wait for a GeneXpert module to become available before being processed. A study was therefore conducted to determine the maximum permissible interval between addition of a sample to an Xpert MRSA NxG Assay cartridge and testing on the GeneXpert Infinity System. The study was performed with MRSA positive (3X LOD) and negative samples prepared with simulated nasal matrix. The samples were added to Xpert MRSA NxG Assay cartridges that were held for up to 5 hours under different environmental conditions prior to testing (i.e., ambient temperature/ humidity, $2 5 ^ { \circ } \mathrm { C } / 7 5 \%$ relative humidity or $3 5 ^ { \circ } \mathrm { C }$ ). The expected results were obtained at each time point and there were no important differences in Ct values under any of the conditions tested. The results of the study support the recommended maximum interval of 4.5 hours from sample addition to the Xpert MRSA NxG Assay cartridges to processing on the GeneXpert Infinity System.

# d. Detection limit:

# Determination of the Limit of Detection

The analytical sensitivity of the Xpert MRSA NxG Assay was determined by testing dilutions of organisms representing 13 different MRSA genotypes in simulated nasal matrix (Section M(2)(b)). The Limit of Detection (LOD) was determined for each strain using logistic regression according to the guidance in CLSI Document EP17- A2. Testing was performed with both specimen collection and transport devices at five different concentrations of each strain. The LOD point estimates were confirmed by testing an additional 20 replicates of each strain at the claimed LOD target level. At least 19/20 replicates had to produce positive results in order for the LOD to be considered confirmed. A summary of the results of the study is presented in Table 5.

This study demonstrated that, with the Cepheid Sample Collection Device (rayon swab), the Xpert MRSA NxG Assay will produce a positive result $2 9 5 \%$ of the time with nasal swabs containing 302 CFU/swab. With the Eswab specimen collection and transport device, the Xpert MRSA NxG Assay will produce a positive result $2 9 5 \%$ of the time with nasal swabs containing 812 CFU/swab.

Table 5. Summary of Xpert MRSA NxG LOD values   

<table><tr><td rowspan=2 colspan=1>SCCmec Type</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>PFGE Type</td><td rowspan=1 colspan=2>LOD (CFU/swab; 95% CI)</td></tr><tr><td rowspan=1 colspan=1>Rayon Swab 1</td><td rowspan=1 colspan=1>ESwab</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>64/4176</td><td rowspan=1 colspan=1>USA500</td><td rowspan=1 colspan=1>91 (72, 136)</td><td rowspan=1 colspan=1>343 (285, 469)</td></tr><tr><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>N315</td><td rowspan=1 colspan=1>USA100</td><td rowspan=1 colspan=1>161 (127, 236)</td><td rowspan=1 colspan=1>218 (184, 293)</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>11373</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>64 (50, 96)</td><td rowspan=1 colspan=1>254 (215, 338)</td></tr><tr><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>Fin 7</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>302 (256, 392)</td><td rowspan=1 colspan=1>812 (656, 1145)</td></tr><tr><td rowspan=1 colspan=1>IVa</td><td rowspan=1 colspan=1>MW2</td><td rowspan=1 colspan=1>USA400</td><td rowspan=1 colspan=1>58 (46, 84)</td><td rowspan=1 colspan=1>167 (134, 245)</td></tr><tr><td rowspan=1 colspan=1>IVa</td><td rowspan=1 colspan=1>NRS384</td><td rowspan=1 colspan=1>USA300</td><td rowspan=1 colspan=1>182 (143, 282)</td><td rowspan=1 colspan=1>563 (470, 733)</td></tr><tr><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>ST59</td><td rowspan=1 colspan=1>USA1000</td><td rowspan=1 colspan=1>102 (85, 138)</td><td rowspan=1 colspan=1>465 (378, 671)</td></tr><tr><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>HDE288</td><td rowspan=1 colspan=1>USA800</td><td rowspan=1 colspan=1>42 (32, 64)</td><td rowspan=1 colspan=1>89 (71, 128)</td></tr><tr><td rowspan=1 colspan=1>VII</td><td rowspan=1 colspan=1>JCSC6082</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>128 (95, 235)</td><td rowspan=1 colspan=1>245 (201,338)</td></tr><tr><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>WA MRSA-16</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>163 (139, 233)</td><td rowspan=1 colspan=1>631 (520, 851)</td></tr><tr><td rowspan=1 colspan=1>IX</td><td rowspan=1 colspan=1>JCSC6945</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>169 (142, 227)</td><td rowspan=1 colspan=1>377 (311, 533)</td></tr><tr><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>JCSC6943</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>97 (86, 119)</td><td rowspan=1 colspan=1>166 (149, 215)</td></tr><tr><td rowspan=1 colspan=1>XI</td><td rowspan=1 colspan=1>LGA251</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>266 (219, 358)</td><td rowspan=1 colspan=1>734 (597, 998)</td></tr></table>

PFGE: Pulsed Field gel Electrophoresis Cepheid Sample Collection Device (rayon swab)

# Analytical Reactivity/Inclusivity

A study was conducted to demonstrate the ability of the Xpert MRSA NxG Assay to detect strains of MRSA isolated from different geographical regions and with different genotypic and phenotypic characteristics. A total of 196 strains were tested including representatives of Cooper & Feil Groups 1A, 1B and 2, SCCmec types I, IA, II, III, IIIA, III-Hg, IV, IVa, IVb, IVc, IVd, V, VI, VII, VIII, IX, X and XI, 48 different spa-types, 12 different Pulsed Field Gel Electrophoresis (PFGE) types and 16 clonal complexes. Heteroresistant strains and the mecC variant strain LGA251 were also tested, as was a “Challenge Panel” of 59 strains with known oxacillin MIC values ranging from 0.5 to $> 3 2 \mu \varrho / \mathrm { m L }$ . All 196 strains were successfully detected in simulated nasal matrix at a concentration of 906 CFU/swab (3X the highest LOD shown in Table 5 for the Cepheid Sample Collection Device (rayon swab)). These results are acceptable.

# e. Analytical specificity:

Cross-reactivity Study The analytical specificity of the Xpert MRSA NxG Assay was evaluated by testing a panel of 152 potentially cross-reactive microorganisms, including 40 strains of methicillin susceptible S. aureus (MSSA), 7 strains of Borderline Oxacillin Resistant S. aureus (BORSA) and 57 Gram positive bacteria, of which 34 were other Staphylococcus spp. Also included in the study were 25 strains of Gram negative bacteria, 3 yeast species, 17 viruses and human cells.

Note: Included in the panel of microorganisms were two SCCmec “empty cassette” variants of S. aureus (1 MSSA and 1 BORSA).

Each strain was tested in triplicate in the presence of simulated nasal matrix using the workflow for the Cepheid Sample Collection Device (rayon swab) as representative of the “worst case” scenario (i.e., potential for the highest concentration of crossreactive organisms in the amplification reaction). Bacteria and yeast were tested at a concentration of ${ \geq } 7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/swab. Viruses and human DNA were tested at the equivalent of $\geq 2 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { s w a l }$ b and $2 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ cells/swab, respectively.

The expected results were obtained from all the microorganisms tested, as well as the human cells, and there was no evidence of the potential for false-positive results due to cross-reaction with non-target organisms.

# Bioinformatic Analysis

The specificity of the Xpert MRSA NxG Assay primers and probes for the targeted sequences was also evaluated through extensive in silico analysis which showed that positive results with species other than S. aureus is unlikely to occur, although the potential for detection with methicillin resistant strains of S. argenteus, a recently described species of Staphylococcus that is closely related to S. aureus, is noted in the Package Insert.

# Carry-over Contamination Study

To determine the potential for sample-to-sample contamination using the Xpert MRSA NxG Assay, testing was performed with alternating “high positive” $( \geq 1 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ CFU/mL of Elution Reagent) and negative samples. The study was conducted on two GeneXpert Systems with 20 high positive and 21 negative samples per instrument. No false-positive results were obtained and the results were therefore determined to be acceptable.

# f. Assay cut-off:

Thresholds and cut-offs for the Xpert MRSA NxG Assay were initially established through pre-clinical testing using nasal samples collected with the Cepheid Sample Collection Device (rayon swab). These settings were validated in two prospective Clinical Studies performed at US and ex-US sites (Section M(3)). The valid Ct ranges for the mecA/mecC, SCCmec and SPC targets are shown in Table 6.

Table 6. Valid Ct ranges for Xpert MRSA NxG targets   

<table><tr><td rowspan=2 colspan=1>Target Sequence</td><td rowspan=1 colspan=2>Valid Ct Range</td></tr><tr><td rowspan=1 colspan=1>Minimum</td><td rowspan=1 colspan=1>Maximum</td></tr><tr><td rowspan=1 colspan=1>mecA/mecC</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>SCCmec</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>36</td></tr></table>

SPC: Sample Processing Control Note: Both the mecA/mecC and SCCmec targets are required to have a valid Ct in order for a sample to be reported as MRSA DETECTED.

In addition to fixed algorithm parameters within the software Assay Definition File for the Xpert MRSA NxG Assay, Lot Specific Parameters (LSP) are also generated for every lot of assay reagents to account for potential variations in production. The LSP are embedded within the barcode of each assay cartridge and are transferred to the GeneXpert instrument system when the barcode is scanned.

# g. Assay Interference:

The potential for interference with the Xpert MRSA NxG Assay was evaluated with endogenous and exogenous substances that may be present in nasal swab samples, as well as a representative panel of commensal bacterial species. Testing was performed using simulated nasal matrix (Section M(2)(b)) in the presence of two strains of MRSA (N315: SCCmec Type II; mecA positive and LGA251: SCCmec Type XI; mecC positive) at approximately 3X LOD using the workflow for the Cepheid Sample Collection Device (rayon swab) as representative of the “worst case” scenario (i.e., the highest potential concentration of the interfering substance/organism). MRSA negative samples were also included to assess the potential for interference with the SPC. In addition to analysis of qualitative test results, ANOVA was used to assess differences in Ct values for the mecA/mecC, SCCmec and SPC targets in relation to appropriate control conditions without any interfering substance or organisms present.

# Potentially Interfering Substances

All MRSA positive and negative samples $\mathrm { i } \mathtt { n } = 8$ per condition) were correctly identified in the presence of each of the potentially inhibitory substances shown in Table 7. However, notable differences in $\mathrm { C t }$ values relative to the control were observed with three substances (Beconase, Flonase and Nasonex), all of which were retested and at lower concentration at which there was no evidence of assay interference. The potential for assay interference with Flonase, Beconase and Nasonex is included as a Limitation in the Package Insert. The results of this study are acceptable,

Table 7. Potentially interfering substances tested with the Xpert MRSA NxG Assay   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>Aneferin Decongestant Spray</td><td rowspan=1 colspan=1>0.05% Oxymetazoline Hydrochloride</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Azelastin Antihistamine Spray</td><td rowspan=1 colspan=1>0.1% Azelastine Hydrochloride</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Bactroban</td><td rowspan=1 colspan=1>2% Mupirocin</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Beconase® AQ</td><td rowspan=1 colspan=1>0.05% or 3.6x10-5g Beclomethasone</td><td rowspan=1 colspan=1>50% (v/v) 240% v230% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>100% Human Blood</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Finafta Multioral</td><td rowspan=1 colspan=1>7.5% Benzocaine</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Flonase</td><td rowspan=1 colspan=1>0.05% Fluticasone Propionate</td><td rowspan=1 colspan=1>50% (v/v) 240% (v/v)</td></tr><tr><td rowspan=1 colspan=1>FluMist</td><td rowspan=1 colspan=1>Live intranasal influenza virus vaccine</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Flunisolide Nasal Solution USP, 0.025%</td><td rowspan=1 colspan=1>0.03% or 1.9x10-5g Flunisolide</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Mucous (Mucin)</td><td rowspan=1 colspan=1>Porcine mucin representing denselyglycosylated proteins (mucous)</td><td rowspan=1 colspan=1>7% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Nasacort® AQ</td><td rowspan=1 colspan=1>0.06% or 4.4x10-5g Triamcinolone acetonide</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>NasalCrom Allergy Symptom Controller</td><td rowspan=1 colspan=1>5.2mg Cromolyn Sodium</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasonex (Nasal Allergy SymptomMedication, inhaled nasal steroid)</td><td rowspan=1 colspan=1>0.05% Mometasone Furoate Monohydrate</td><td rowspan=1 colspan=1>50% (v/v) 240%(v/v)</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine Decongestant Spray</td><td rowspan=1 colspan=1>0.5% Phenylephrine Hydrochloride</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>5mg Zanamivir</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Rhinocort aqua®</td><td rowspan=1 colspan=1>0.06% or 4.4x10&quot;g Budesonide</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Moisturizing Spray</td><td rowspan=1 colspan=1>0.65% Sodium Chloride</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>TobraDex</td><td rowspan=1 colspan=1>0.3% Tobramycin0.1% Dexamethasone</td><td rowspan=1 colspan=1>50% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Zicam Nasal Gel(Upper Respiratory AllergySymptom Relief)</td><td rowspan=1 colspan=1>4x, 12x, 30x Luffa operculata12x, 30x Galphimia glauca12x, 30x, 200x Histaminum hydrochloricum12x, 30x, 200x Sulphur</td><td rowspan=1 colspan=1>50% (v/v)</td></tr></table>

1 Concentration on the swab $7 5 \mu \mathrm { L }$ absorption volume) 2 Shown to be inhibitory (delayed Ct values for one or both MRSA targets and/or the SPC)

Microbial Interference Study   
The list of commensal microorganisms tested is shown in Table 8. The Xpert MRSA NxG Assay correctly identified all the MRSA positive and negative samples in the study $\mathtt { \tilde { n } } = 4$ per condition) and there were no statistically significant differences in Ct values between the test samples and controls. The results of this study showed that there was no evidence of inhibition of the Xpert MRSA NxG Assay in the presence of high concentrations of the commensal bacterial species tested.

Table 8. Species of commensal bacteria tested with the Xpert MRSA NxG Assay in the Microbial Interference Study   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Gram Stain</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium bovis</td><td rowspan=1 colspan=1>ATCC 7715</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td><td rowspan=1 colspan=1>ATCC 9007</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>ATCC43628</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC700111</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>ATCC 29905</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (MSSA)</td><td rowspan=1 colspan=1>15280</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis (MSSE)</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>ATCC 25175</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 6303</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Note: All commensal species were tested at $\overline { { 2 2 \mathrm { ~ x ~ } 1 0 ^ { 6 } } }$ CFU/swab

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

Comparison of Fresh and Frozen Cell Stocks Samples for analytical evaluation of assay performance were prepared using MRSA cell stocks that were grown on Tryptic Soy Agar plates, suspended in phosphate buffered saline containing $1 5 \% \mathrm { ( v / v ) }$ glycerol and frozen at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ . The concentration of the stocks was determined by performing viable counts. Plate counts performed before and after freezing at ${ - 8 0 ^ { \circ } C }$ demonstrated negligible difference indicating that the integrity of the MRSA cells was maintained during the freeze-thaw process.

Comparison of Performance with Natural and Simulated Matrices In order to provide a sufficient quantity of specimen matrix for testing, a simulated nasal matrix was used for the majority of Analytical Studies. The formulation of the simulated matrix is shown in Table 9. To represent a worst case scenario, each sample collection swab was seeded with $7 5 \mu \mathrm { L }$ of simulated matrix (equal to the absorptive capacity of the swab head).

Table 9. Formulation of simulated nasal matrix   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Porcine mucin</td><td rowspan=1 colspan=1>5% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole human blood</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Phosphate Buffered Saline</td><td rowspan=1 colspan=1>1X</td></tr><tr><td rowspan=1 colspan=1>Glycerol</td><td rowspan=1 colspan=1>15% v/v</td></tr></table>

To assess the suitability of the simulated matrix for use in analytical testing, a comparison study was conducted to confirm that the Xpert MRSA NxG Assay performed similarly in the presence of natural and simulated nasal matrix. Testing was performed with two MRSA strains at different levels, close to the LOD of the assay and with both types of specimen collection device (rayon swab and ESwab). The results summarized in Tables 10 (Cepheid Sample Collection Device (rayon swab)) and 11 (Eswab) show that the percentage of positive results at each target level met the pre-defined acceptance criteria and therefore that the assay exhibited similar analytical sensitivity for the detection of MRSA with both natural and simulated matrix. Ct values for the mecA/mecC, SCCmec and SPC targets were also similar in both matrices. These results support the use of simulated matrix in the Analytical Studies to characterize the performance of the Xpert MRSA NxG Assay.

Table 10. Comparison of natural and simulated matrix: Cepheid Sample Collection Device (rayon swab)   

<table><tr><td rowspan=1 colspan=13>SCCmec Type ⅡI Strain N315 (LOD = 161 CFU/swab)</td></tr><tr><td rowspan=3 colspan=2>Matrix</td><td rowspan=3 colspan=1>TargetLevell</td><td rowspan=3 colspan=1>N</td><td rowspan=3 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=6>Ct Value</td></tr><tr><td rowspan=2 colspan=1>Observed</td><td rowspan=2 colspan=1>Expected</td><td rowspan=1 colspan=2>mecA/mecC2</td><td rowspan=1 colspan=2>SCCmec²2</td><td rowspan=1 colspan=2>SPC3</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=5 colspan=2>Natural</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=2>Simulated</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.4</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=13>SCCmec Type XI Strain LGA251 (LOD = 266 CFU/swab)</td></tr><tr><td rowspan=3 colspan=2>Matrix</td><td rowspan=3 colspan=1>Target Level</td><td rowspan=3 colspan=1>N</td><td rowspan=3 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=6>Ct Value</td></tr><tr><td rowspan=2 colspan=1>Observed</td><td rowspan=2 colspan=1>Expected</td><td rowspan=1 colspan=2>mecA/mecC{</td><td rowspan=1 colspan=2>SCCmec²2</td><td rowspan=1 colspan=2>SPC3</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=5 colspan=2>Natural</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=2>Simulated</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

NA: Not Applicable; SD: Standard Deviation; SPC: Sample Processing Control 1 Multiple of Limit of Detection (LOD) Actual concentrations tested (CFU/swab): Strain N315: ${ < } 1 \mathrm { X }$ : 75; 2X: 300; 5X: 750; 10X: 1500 Strain LGA251: <1X: 125; 2X: 500; 5X: 1250; 10X: 2500 2 Values shown are for samples that were MRSA-positive by the Xpert MRSA NxG Assay 3 Values shown are for samples that were MRSA-negative by the Xpert MRSA NxG Assay

Table 11. Comparison of natural and simulated matrix: ESwab   

<table><tr><td rowspan=1 colspan=13>SCCmec Type II Strain N315 (LOD = 218 CFU/swab)</td></tr><tr><td rowspan=3 colspan=2>Matrix</td><td rowspan=3 colspan=1>TargetLevel</td><td rowspan=3 colspan=1>N</td><td rowspan=3 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=6>Ct Value</td></tr><tr><td rowspan=2 colspan=1>Observed</td><td rowspan=2 colspan=1>Expected</td><td rowspan=1 colspan=2>mecA/mecC²</td><td rowspan=1 colspan=2>SCCmec²</td><td rowspan=1 colspan=2>SPC^{3</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=5 colspan=2>Natural</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=2>Simulated</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Simulated</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=13>SCCmec Type XI Strain LGA251 (LOD = 734 CFU/swab)</td></tr><tr><td rowspan=3 colspan=2>Matrix</td><td rowspan=3 colspan=1>TargetLevel</td><td rowspan=3 colspan=1>N</td><td rowspan=3 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=6>Ct Value</td></tr><tr><td rowspan=2 colspan=1>Observed</td><td rowspan=2 colspan=1>Expected</td><td rowspan=1 colspan=2>mecA/mecC2</td><td rowspan=1 colspan=2>SCCmec²</td><td rowspan=1 colspan=2>SPC{3</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=3 colspan=2>Natural</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.58</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>ral</td><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=2>Simulated</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5-95</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

NA: Not Applicable; SD: Standard Deviation; SPC: Sample Processing Control 1 Multiple of Limit of Detection (LOD) Actual concentrations tested (CFU/swab): Strain N315: ${ < } 1 \mathrm { X }$ : 100; 2X: 400; 5X: 1000; 10X: 2000 Strain LGA251: ${ < } 1 \mathrm { X }$ : 350; 2X: 1400; 5X: 3500; 10X: 7000 2 Values shown are for samples that were MRSA-positive by the Xpert MRSA NxG Assay 3 Values shown are for samples that were MRSA-negative by the Xpert MRSA NxG Assay

Comparison of Performance with Fresh and Frozen Samples To support use of frozen samples to characterize the analytical performance of the Xpert MRSA NxG Assay, a study was conducted to compare fresh matrix and matrix that was subjected to up to two freeze-thaw cycles. Samples were prepared by diluting MRSA cell stocks to the appropriate concentration in simulated matrix (Table 9). The samples were either tested immediately or after freezing at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ and thawing. To challenge the assay, the study was performed by adding $7 5 \mu \mathrm { L }$ of the simulated sample matrix to $2 \mathrm { m L }$ of Elution Buffer and by transferring the entire

volume of the mixture to the Xpert MRSA NxG Assay cartridge. The results presented in Tables 12 and 13 show that the expected proportion of positive results was obtained at each target level and that there were negligible differences Ct values between fresh samples and those that were frozen and thawed up to two times. The data therefore support use of frozen samples in the Analytical Studies.

Table 12. Fresh vs Frozen Sample Study: SCCmec Type II Strain N315   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>TargetLevel</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=2>mecA/mecC2</td><td rowspan=1 colspan=2>SCCmec^{</td><td rowspan=1 colspan=2>SPC</td></tr><tr><td rowspan=1 colspan=1>Observed</td><td rowspan=1 colspan=1>Expected</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=5 colspan=1>FRESH</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=1>1XFreeze-Thaw</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.80</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.70</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=1>2XFreeze-Thaw</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.81</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

NA: Not Applicable; SD: Standard Deviation; SPC: Sample Processing Control Multiple of Limit of Detection $( \mathrm { L O D } = 1 6 1 $ CFU/rayon swab) Actual concentrations tested $( \mathrm { C F U } / 7 5 \mu \mathrm { L }$ matrix): 0.25X: 35; 0.5X: 75; 1X: 150; 2X: 300 2 Values shown are for samples that were MRSA-positive by the Xpert MRSA NxG Assay 3 Values shown are for samples that were MRSA-negative by the Xpert MRSA NxG Assay

Table 13. Fresh vs Frozen Sample Study: SCCmec Type XI Strain LGA251   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>TargetLevel</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MRSAPositive</td><td rowspan=1 colspan=2>% Positive</td><td rowspan=1 colspan=2>mecA/mecC^2</td><td rowspan=1 colspan=2>SCCmec^{</td><td rowspan=1 colspan=2>SPC</td></tr><tr><td rowspan=1 colspan=1>Observed</td><td rowspan=1 colspan=1>Expected</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=5 colspan=1>FRESH</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=1>1XFreeze-Thaw</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=1>2XFreeze-Thaw</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>0.25X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>0.5X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>20-80</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>1X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

NA: Not Applicable; SD: Standard Deviation; SPC: Sample Processing Control 1 Multiple of Limit of Detection $\mathrm { ( L O D } = 2 6 6$ CFU/rayon swab) Actual concentrations tested $( \mathrm { C F U } / 7 5 \mu \mathrm { L }$ matrix): 0.25X: 60; 0.5X: 125; 1X: 250; 2X: 500 2 Values shown are for samples that were MRSA-positive by the Xpert MRSA NxG Assay 3 Values shown are for samples that were MRSA-negative by the Xpert MRSA NxG Assay

# 3. Clinical studies:

a. Clinical Sensitivity:

The performance of the Xpert MRSA NxG Assay was evaluated in two separate prospective, multi-center Clinical Studies using nasal swab specimens collected from subjects at risk for nasal colonization with MRSA. In the first study, testing was performed with samples collected using the Cepheid Sample Collection Device (rayon swab). In the second study, testing was performed with samples collected using the Copan Liquid Amies Elution Swab Collection and Transport System (ESwab).

The reference method for both studies comprised a combination of direct culture on MRSA selective chromogenic medium and enriched culture in Trypticase Soy Broth (TSB) containing $6 . 5 \%$ NaCl, followed by subculture onto blood agar and MRSA selective chromogenic medium. Identification of presumptive MRSA colonies was confirmed by Gram stain, catalase and coagulase testing. Susceptibility to cefoxitin was determined by disk diffusion using a $3 0 \mu \mathrm { g }$ disk according to the criteria described in CLSI Document M100-S24.1

Performance with Cepheid Sample Collection Device (rayon swab) The evaluation of performance with the Cepheid Sample Collection Device (rayon swab) was performed with specimens collected at 8 geographically diverse sites (6 US and 2 ex-US). A total of 1183 specimens were initially enrolled in the study, of which 77 were excluded due to protocol deviations (74), site policy (1) or change of mind by the donor (2). Of the remaining 1106 samples, 31 $( 2 . 8 \% )$ produced an indeterminate result on initial testing (ERROR: 15; INVALID: 9; NO RESULT: 7). Three (3) samples also produced indeterminate results on repeat testing for a final indeterminate rate of $0 . 3 \%$ (3/1106). The performance of the Xpert MRSA NxG Assay with the Cepheid Sample Collection Device (rayon swab) in comparison to the reference method is summarized in Tables 14 and 15.

Table 14. Xpert MRSA NxG Assay performance with the Cepheid Sample Collection Device (rayon swab) vs Reference Method   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Reference Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Xpert MRSANxG Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>141</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>951</td><td rowspan=1 colspan=1>962</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>981</td><td rowspan=1 colspan=1>1103</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>111/122 = 91.0% (95% CI: 84.6 - 94.9%)</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>951/981 = 96.9%(95%CI: 95.7 - 97.9%)</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>111/141 = 78.7% (95% CI: 71.3 - 84.7%)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>951/962 = 98.9% (95% CI: 98.0 - 99.4%)</td></tr></table>

1 30/30 specimens with Xpert MRSA NxG false positive results were also MRSA culture negative upon repeat subculture of the enrichment broth 2 11/11 specimens with Xpert MRSA $\mathrm { N x G }$ false negative results were MRSA culture positive upon repeat subculture of the enrichment broth

Table 15. Xpert MRSA NxG Assay performance with the Cepheid Sample Collection Device (rayon swab) vs Reference Method stratified by site   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Samples(%)</td><td rowspan=2 colspan=1>Culture Positive(% Prevalence)</td><td rowspan=1 colspan=4>Percent (95%CI)</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>90(8.2)</td><td rowspan=1 colspan=1>39(43.3)</td><td rowspan=1 colspan=1>87.2(73.3-94.4)</td><td rowspan=1 colspan=1>90.2(79.0-95.7)</td><td rowspan=1 colspan=1>87.2(73.3-94.4)</td><td rowspan=1 colspan=1>90.2(79.0-95.7)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>78(7.1)</td><td rowspan=1 colspan=1>3(3.8)</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>100(95.1-100)</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>100(95.1-100)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>228(20.7)</td><td rowspan=1 colspan=1>23(10.1)</td><td rowspan=1 colspan=1>100(85.6-100)</td><td rowspan=1 colspan=1>97.1(93.8-98.7)</td><td rowspan=1 colspan=1>79.3(61.6-90.2)</td><td rowspan=1 colspan=1>100(98.1-100)</td></tr><tr><td rowspan=1 colspan=1>D1</td><td rowspan=1 colspan=1>172(15.6)</td><td rowspan=1 colspan=1>11(6.4)</td><td rowspan=1 colspan=1>100(74.1-100)</td><td rowspan=1 colspan=1>97.5(93.8-99.0)</td><td rowspan=1 colspan=1>73.3(48.1-89.1)</td><td rowspan=1 colspan=1>100(97.6-100)</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>125(11.3)</td><td rowspan=1 colspan=1>7(5.6)</td><td rowspan=1 colspan=1>100(64.6-100)</td><td rowspan=1 colspan=1>97.5(92.8-99.1)</td><td rowspan=1 colspan=1>70.0(39.7-89.2)</td><td rowspan=1 colspan=1>100(96.8-100)</td></tr><tr><td rowspan=1 colspan=1>F1</td><td rowspan=1 colspan=1>5(0.5)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>NotApplicable</td><td rowspan=1 colspan=1>100(56.7-100)</td><td rowspan=1 colspan=1>NotApplicable</td><td rowspan=1 colspan=1>100(56.7-100)</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>125(11.3)</td><td rowspan=1 colspan=1>6(4.8)</td><td rowspan=1 colspan=1>83.3(43.7-96.7)</td><td rowspan=1 colspan=1>97.5(92.9-99.1)</td><td rowspan=1 colspan=1>62.5(30.6-86.3)</td><td rowspan=1 colspan=1>99.1(95.3-99.8)</td></tr><tr><td rowspan=1 colspan=1>H{2</td><td rowspan=1 colspan=1>280(25.4)</td><td rowspan=1 colspan=1>33(11.8)</td><td rowspan=1 colspan=1>84.8(69.1-93.4)</td><td rowspan=1 colspan=1>96.4(93.2-98.1)</td><td rowspan=1 colspan=1>75.7(59.9-86.6)</td><td rowspan=1 colspan=1>97.9(95.3-99.1)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1103(100)</td><td rowspan=1 colspan=1>122(11.1)</td><td rowspan=1 colspan=1>91.0(84.6-94.9)</td><td rowspan=1 colspan=1>96.9(95.7-97.9)</td><td rowspan=1 colspan=1>78.7(71.3-84.7)</td><td rowspan=1 colspan=1>98.9(98.0-99.4)</td></tr></table>

PPV: Positive Predictive Value; NPV: Negative Predictive Value 1 ex-US sites 2 Also participated in evaluation of performance with the ESwab sample collection device (Tables 16 and 17)

# Performance with ESwab

The evaluation of performance with the Eswab was performed with specimens collected at 6 geographically diverse sites in the US. A total of 918 specimens were initially enrolled in the study of which 56 were excluded due to protocol deviations (54) or contaminated/inconclusive cultures (4). Of the remaining 862 samples, 16 $( 1 . 9 \% )$ produced an indeterminate result on initial testing (ERROR: 8; INVALID: 8). Due to insufficient sample volume, retesting was not performed in this study. The final indeterminate rate was therefore $1 . 9 \%$ (16/862). The performance of the Xpert MRSA NxG Assay with the ESwab in comparison to the reference method is summarized in Tables 16 and 17.

Table 16. Xpert MRSA NxG Assay performance with the ESwab vs Reference Method   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Reference Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Xpert MRSANxG Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>730</td><td rowspan=1 colspan=1>737</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>748</td><td rowspan=1 colspan=1>846</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>91/98 = 92.9% (95% CI: 86.0 - 96.5%)</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>730/748 = 97.6% (95% CI: 96.2 - 98.5%)</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>91/109 = 83.5% (95% CI: 75.4  89.3%)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>748/846 = 99.1% (95% CI: 98.1 - 99.5%)</td></tr></table>

1 17/18 specimens with Xpert MRSA NxG false positive results were also MRSA culture negative upon repeat subculture of the enrichment broth 2 6/7 specimens with Xpert MRSA NxG false negative results were MRSA culture positive upon repeat subculture of the enrichment broth

Table 17. Xpert MRSA NxG Assay performance with the ESwab vs Reference Method stratified by site   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Samples(%)</td><td rowspan=2 colspan=1>Culture Positive(% Prevalence)</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>231(27.3)</td><td rowspan=1 colspan=1>20(8.7)</td><td rowspan=1 colspan=1>95.0(76.4-99.1)</td><td rowspan=1 colspan=1>96.2(92.7-98.1)</td><td rowspan=1 colspan=1>70.4(51.5-84.1)</td><td rowspan=1 colspan=1>99.5(97.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>139(16.4)</td><td rowspan=1 colspan=1>6(4.3)</td><td rowspan=1 colspan=1>100(61.0-100)</td><td rowspan=1 colspan=1>99.2(95.9-99.9)</td><td rowspan=1 colspan=1>85.7(48.7-97.4)</td><td rowspan=1 colspan=1>100(97.2-100)</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>136(16.1)</td><td rowspan=1 colspan=1>4(2.9)</td><td rowspan=1 colspan=1>75.0(30.1-95.4)</td><td rowspan=1 colspan=1>100(97.2-100)</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>99.2(95.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>15(1.8)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>NotApplicable</td><td rowspan=1 colspan=1>80.0(54.8-93.0)</td><td rowspan=1 colspan=1>NotApplicable</td><td rowspan=1 colspan=1>100(75.8-100)</td></tr><tr><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>168(19.9)</td><td rowspan=1 colspan=1>52(31.0)</td><td rowspan=1 colspan=1>96.2(87.0-98.9)</td><td rowspan=1 colspan=1>95.7(90.3-98.1)</td><td rowspan=1 colspan=1>90.9(80.4-96.1)</td><td rowspan=1 colspan=1>98.2(93.8-99.5)</td></tr><tr><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>157(18.6)</td><td rowspan=1 colspan=1>16(10.2)</td><td rowspan=1 colspan=1>81.3(57.0-93.4)</td><td rowspan=1 colspan=1>99.3(96.1-99.9)</td><td rowspan=1 colspan=1>92.9(68.5-98.7)</td><td rowspan=1 colspan=1>97.9(94.0-99.3)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>846(100)</td><td rowspan=1 colspan=1>98(11.6)</td><td rowspan=1 colspan=1>92.9(86.0-96.5)</td><td rowspan=1 colspan=1>97.6(96.2-98.5)</td><td rowspan=1 colspan=1>83.5(75.4-89.3)</td><td rowspan=1 colspan=1>99.1(98.1-99.5)</td></tr></table>

PPV: Positive Predictive Value; NPV: Negative Predictive Value Also participated in evaluation of performance with the Cepheid Sample Collection Device (rayon swab) (Tables 15 and 16)

Analysis of the data from the two Clinical Studies showed that there was no statistically significant difference in sensitivity or specificity using the two different specimen collection and transport devices. A summary of the clinical performance of the Xpert MRSA NxG Assay with both the Cepheid Sample Collection Device (rayon swab) and ESwab combined in relation to the reference culture method is shown in Table 18.

Table 18. Xpert MRSA NxG Assay performance with the Cepheid Sample Collection Device (rayon swab) and ESwab combined vs Reference Method   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Reference Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Xpert MRSANxG Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>250</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1681</td><td rowspan=1 colspan=1>1699</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>1729</td><td rowspan=1 colspan=1>1949</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>202/220 = 91.8% (95% CI: 87.4 - 94.8%)</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>1681/1729 = 97.2% (95% CI: 96.3 - 97.9%)</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>202/250 = 80.8% (95% CI: 75.5 - 85.2%)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>1729/1949 = 98.9% (95% CI: 98.3 - 99.3%)</td></tr></table>

The Clinical Study demonstrated that the performance of the Xpert MRSA NxG assay for the detection of MRSA in nasal swab samples is acceptable and substantially equivalent to that of the predicate device.

b. Clinical specificity: Refer to Section M(3)(a): Clinical Sensitivity.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The performance of the Xpert MRSA NxG Assay was evaluated in two prospective Clinical Studies at multiple sites in the US and ex-US. The prevalence of MRSA observed in each of the studies, as determined by the Xpert assay, is shown in Tables 19 and 20 stratified by the age and gender of the subjects.

Table 19. Prevalence of MRSA observed using the Cepheid Sample Collection Device (rayon swab)   

<table><tr><td rowspan=3 colspan=1>Age(years)</td><td rowspan=1 colspan=3>Male</td><td rowspan=1 colspan=3>Female</td><td rowspan=1 colspan=3>All Subjects</td></tr><tr><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td></tr><tr><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1(33.3)</td><td rowspan=1 colspan=1>1(33.3)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1(20.0)</td><td rowspan=1 colspan=1>1(20.0)</td></tr><tr><td rowspan=1 colspan=1>6-12</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td></tr><tr><td rowspan=1 colspan=1>13-21</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1(5.3)</td><td rowspan=1 colspan=1>1(5.3)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1(12.5)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2(7.4)</td><td rowspan=1 colspan=1>1(3.7)</td></tr><tr><td rowspan=1 colspan=1>22-65</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=1>67(13.6)</td><td rowspan=1 colspan=1>62(12.6)</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>29(12.0)</td><td rowspan=1 colspan=1>27(11.2)</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>96(13.1)</td><td rowspan=1 colspan=1>89(12.1)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>34(13.3)</td><td rowspan=1 colspan=1>24(9.4)</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>8(10.8)</td><td rowspan=1 colspan=1>7(9.5)</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>42(12.7)</td><td rowspan=1 colspan=1>31(9.4)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>772</td><td rowspan=1 colspan=1>102(13.2)</td><td rowspan=1 colspan=1>87(11.3)</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>39(11.8)</td><td rowspan=1 colspan=1>35(10.6)</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>141(12.8)</td><td rowspan=1 colspan=1>122(11.1)</td></tr></table>

Xpert: Xpert MRSA NxG Assay Reference: Combined direct and enriched reference culture method (Section M(3))

Table 20. Prevalence of MRSA observed using the ESwab sample collection device   

<table><tr><td rowspan=3 colspan=1>Age(years)</td><td rowspan=1 colspan=3>Male</td><td rowspan=1 colspan=3>Female</td><td rowspan=1 colspan=3>All Subjects</td></tr><tr><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=2>Number (%)MRSA Positive</td></tr><tr><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>Xpert</td><td rowspan=1 colspan=1>Reference</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0(NA)</td><td rowspan=1 colspan=1>0(NA)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>1(100)</td></tr><tr><td rowspan=1 colspan=1>6-12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0(NA)</td><td rowspan=1 colspan=1>0(NA)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td></tr><tr><td rowspan=1 colspan=1>13-21</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1(5.0)</td><td rowspan=1 colspan=1>1(5.0)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3(15.0)</td><td rowspan=1 colspan=1>3(15.0)</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>4(10.0)</td><td rowspan=1 colspan=1>4(10.0)</td></tr><tr><td rowspan=1 colspan=1>22-65</td><td rowspan=1 colspan=1>320</td><td rowspan=1 colspan=1>47(14.7)</td><td rowspan=1 colspan=1>39(12.2)</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>25(7.9)</td><td rowspan=1 colspan=1>22(7.0)</td><td rowspan=1 colspan=1>635</td><td rowspan=1 colspan=1>72(11.3)</td><td rowspan=1 colspan=1>61(9.6)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>15(18.8)</td><td rowspan=1 colspan=1>16(20.0)</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>17(19.1)</td><td rowspan=1 colspan=1>16(18.0)</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>32(18.9)</td><td rowspan=1 colspan=1>32(18.9)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>64(15.2)</td><td rowspan=1 colspan=1>57(13.5)</td><td rowspan=1 colspan=1>424</td><td rowspan=1 colspan=1>45(10.6)</td><td rowspan=1 colspan=1>41(9.7)</td><td rowspan=1 colspan=1>846</td><td rowspan=1 colspan=1>109(12.9)</td><td rowspan=1 colspan=1>98(11.6)</td></tr></table>

NA: Not applicable Xpert: Xpert MRSA NxG Assay Reference: Combined direct and enriched reference culture method (Section M(3))

# N. Instrument Name:

GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity48s and GeneXpert Infinity-80)

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ____X____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No ___ _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: Barcode scan or manual entry.

4. Specimen Sampling and Handling:

Nasal swab samples may be collected using either the Cepheid Sample Collection Device (rayon swab) or the Copan Liquid Amies Elution Swab (ESwab™).

Cepheid Sample Collection Device (rayon swab)   
One swab is used for the Xpert MRSA NxG Assay and the second swab is retained for repeat testing. The swab shaft is broken off into a vial of Xpert MRSA NxG Elution Reagent and vortexed to elute the target organisms (if present). Using a transfer pipette, the entire volume of Elution Reagent is then added to the Sample Chamber of the Xpert MRSA NxG Assay cartridge and the test initiated.

# ESwab

The Eswab tube containing the swab is vortexed to elute the sample and $3 0 0 \mu \mathrm { L }$ of the ESwab medium is added to a vial of Xpert MRSA NxG Elution Reagent using a transfer pipette. The entire volume of liquid from the Elution Reagent vial is then transferred into the Sample Chamber of the Xpert MRSA NxG Assay cartridge and the test is initiated.

5. Calibration:

No calibration by the user is required.

6. Quality Control:

Quality control is addressed for each separately cleared assay to be run on the instrument. See Section M(1)(c) for information on internal and external controls.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.