# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K162274

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the Solana® Strep Complete Assay from throat specimens.

# C. Measurand:

Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus) and Streptococcus dysgalactiae (pyogenic Group C and G $\beta$ -hemolytic Streptococcus nucleic acids).

# D. Type of Test:

The Solana Strep Complete Assay is a helicase-dependent amplification (HDA) in vitro diagnostic test for the rapid and qualitative detection and differentiation of Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus) and Streptococcus dysgalactiae (pyogenic Group C and G $\beta$ -hemolytic Streptococcus) nucleic acids isolated from throat swab specimens obtained from symptomatic patients.

# E. Applicant:

Quidel Corporation

F. Proprietary and Established Names: Solana Strep Complete Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.2680 - Streptococcus spp. nucleic acid-based assay

2. Classification: Class II

3. Product code:

PGX - Group C and G Beta-Hemolytic Streptococcus Nucleic Acid Amplification System

4. Panel: 83- Microbiology

# H. Intended Use:

1. Intended use(s):

The Solana Strep Complete Assay is a rapid in vitro diagnostic test, using isothermal amplification technology (helicase-dependent amplification, HDA), for the qualitative detection and differentiation of Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus) and Streptococcus dysgalactiae (pyogenic Group C and G $\beta$ -hemolytic Streptococcus) nucleic acids isolated from throat swab specimens obtained from patients with signs and symptoms of pharyngitis, such as sore throat. The Solana Strep Complete Assay is intended for use only with the the Solana instrument.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): For in vitro diagnostic use only. For prescription use only.

4. Special instrument requirements: Solana™ Instrument

# I. Device Description:

The Solana Strep Complete assay detects Streptococcus pyogenes (GAS; $\beta$ -hemolytic Group A Streptococci) and Streptococus dysgalactiae DNA isolated from throat swab specimens obtained from patients with the signs and symptoms of pharyngitis. The assay consists of three major steps: 1) sample processing 2) specimen preparation, and 3) amplification and detection of target sequences specific to GAS and Streptococus dysgalactiae, respectively, using isothermal Helicase-Dependent Amplification (HDA) in the presence of target-specific fluorescence probe.

Patient specimen on a throat swab is transferred to a Lysis Tube and subjected to heat treatment at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. The heat-treated sample is added to a Dilution Tube, and then transferred to a Reaction Tube. The Reaction Tube contains lyophilized HDA reagents, dNTPs, primers and probes. Once rehydrated with the diluted sample, the Reaction Tube is placed in the Solana for amplification and detection of GAS- and Streptococus dysgalactiae - specific target sequences. Target sequences are amplified by analyte-specific primers and detected using fluorescence probes included in each Reaction Tube. Two competitive process controls (PRC) are included in the Lysis Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. The PRC target is amplified by target-specific primers and detected by a PRC specific fluorescence probe.

Solana instrument platform measures and interprets the fluorescent signal, using on-board method-specific algorithms. Solana instrument will then report the test results to the user on the display screen. Results can also be printed out. The total time for assay completion from specimen processing to result is $\leq 2 5$ minutes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra Direct Strep Assay   
2. Predicate 510(k) number(s): k133883

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Solana Strep Complete Assay(K162274)</td><td colspan="1" rowspan="1">Lyra Direct Strep Assay(K133883)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Solana Strep CompleteAssay is a rapid in vitrodiagnostic test, using isothermalamplification technology(helicase-dependentamplification, HDA), for thequalitative detection anddifferentiation of Streptococcuspyogenes (Group A β-hemolyticStreptococcus) andStreptococcus dysgalactiae(pyogenic Group C and G β-hemolytic Streptococcus) nucleicacids isolated from throat swabspecimens obtained frompatients with signs andsymptoms of pharyngitis, suchas sore throat. The Solana StrepComplete Assay is intended foruse only with the the Solanainstrument.</td><td colspan="1" rowspan="1">The Lyra Direct Strep Assay is aReal-Time PCR in vitrodiagnostic test for the qualitativedetection and differentiation ofGroup A β-hemolyticStreptococcus (Streptococcuspyogenes) and pyogenic Group Cand G β-hemolytic Streptococcusnucleic acids isolated from throatswab specimens obtained frompatients with signs andsymptoms of pharyngitis, such asa sore throat. The assay does notdifferentiate between pyogenicGroups C and G β-hemolyticStreptococcus.All negative test results shouldbe confirmed by bacterialculture, because negative resultsdo not preclude Group A, C or GStrep infection and should not beused as the sole basis fortreatment.The assay is intended for use inhospital, reference, or statelaboratory settings. The device isnot intended for point-of-careuse.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Throat swab specimens</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Lysis</td><td colspan="1" rowspan="1">Manual, heat-based</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnique</td><td colspan="1" rowspan="1">Automatically detectsfluorescence after dissociation offluorophore from quencherduring amplification</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Solana Strep Complete Assay(K162274)</td><td colspan="1" rowspan="1">Lyra Direct Strep Assay(k133883)</td></tr><tr><td colspan="1" rowspan="1">DNAAmplificationTechnology</td><td colspan="1" rowspan="1">Isothermal Helicase-DependentAmplification (HDA)</td><td colspan="1" rowspan="1">Real-Time Polymerase ChainReaction (RT-PCR)</td></tr><tr><td colspan="1" rowspan="1">GAS TargetSequenceDetected</td><td colspan="1" rowspan="1">78 base pair (bp) sequence S.pyogenes genome, resident in theDNase B (sdaB) gene67 bp sequenceProtein G gene in the S.dysgalactiae (pyogenicGroup C and G β-hemolyticStreptococcus) genome</td><td colspan="1" rowspan="1">99 base pair (bp) sequence inthe putative competence(comX1.1) gene in the S.pyogenes genome188bp sequence in the tagatose-6-phosphate kinase (lacC) geneconserved in Group C and Gstreptococcal genomes</td></tr><tr><td colspan="1" rowspan="1">Testing Time</td><td colspan="1" rowspan="1">25 minutes</td><td colspan="1" rowspan="1">60-70 minutes</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">SolanaT</td><td colspan="1" rowspan="1">ABI 7500 Fast DXThermocycler</td></tr><tr><td colspan="1" rowspan="1">Reagents/Components</td><td colspan="1" rowspan="1">Dry heating blocks, DilutionBuffer, Lysis Buffer, ReactionTubes</td><td colspan="1" rowspan="1">LyraTM Direct Strep MasterMix, Process Buffer, andRehydration SolutionABI 7500 Fast Dx 96-well PCRPlate, optical plate films andplate centrifugeDry heating block</td></tr><tr><td colspan="1" rowspan="1">PerformanceCharacteristics</td><td colspan="1" rowspan="1">GAS* Sensitivity:98.9%[95% CI: 98.3% - 99.2%]GAS Specificity:99.5%[95% CI: 99.1% - 99.7%]Pyo GCS/GGS* S. dysgalactiaeSensitivity:100%[95% CI: 95.3% - 100%]Specificity:99.5%[95% CI: 99.1% - 99.7%]</td><td colspan="1" rowspan="1">GAS* Sensitivity:96.5%[95% CI: 91.3% - 98.6%]GAS Specificity:98.0%[95% CI: 97.0% - 98.6%]Pyo GCS/GGS* Sensitivity:95.7%[95% CI: 88.1% - 98.5%]Pyo GCS/GGS Specificity:98.3%[95% CI: 97.4% - 98.9%]</td></tr></table>

${ } ^ { * } \mathrm { G A S } =$ Group A Streptococcus; Pyo GCS/GGS $=$ Pyogenic Group C/G Streptococcus

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable.

# L. Test Principle:

Patient specimen on a throat swab is transferred to a Lysis Tube and subjected to heat treatment at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. The heat-treated sample is added to a Dilution Tube, and then transferred to two Reaction Tubes; one for GAS and one for S. dysgalactiae. Each Reaction Tube contains lyophilized helicase-dependent amplification (HDA) reagents, dNTPs, primers and probes for GAS or S. dysgalactiae. Once rehydrated with the diluted sample, the Reaction Tube is placed in Solana for amplification and detection of GAS- and S. dysgalactiae-specific target sequences. Target sequences are amplified by analyte-specific primers and detected using fluorescence probes included in each Reaction Tube. Two competitive process controls (PRC) are included in the Lysis Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. The PRC target is amplified by target-specific primers and detected by a PRC specific fluorescence probe.

The target and PRC probes are labeled with a quencher on one end and a fluorophore on the other end (see Table 1 for associated fluorophores).

<table><tr><td rowspan=1 colspan=2>Table 1: Solana Strep Complete Probe Labels</td></tr><tr><td rowspan=1 colspan=1>Detectant</td><td rowspan=1 colspan=1>Fluorescence Dye/Channel</td></tr><tr><td rowspan=1 colspan=1>GAS or S. dysgalactiae Probe</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Process Controls (PRC)</td><td rowspan=1 colspan=1>CY5</td></tr><tr><td rowspan=1 colspan=1>Reference (differentiates GAS from S.dysgalactiae reaction mix)</td><td rowspan=1 colspan=1>ROX</td></tr></table>

The target and PRC fluorescent probes have a RNA linker that is cleaved by RNaseH2 upon annealing to GAS, S. dysgalactiae or PRC amplicons, and the fluorescence signal increases due to separation of fluorophore from quencher. The Solana Instrument measures and interprets the fluorescent signal, using on-board method-specific algorithms. Solana instrument will then report the test results to the user on its display screen, and it can print out the results via a printer.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility Study

The between-laboratory reproducibility of the Solana Strep Complete assay was evaluated at three sites (two external clinical sites and one in-house) using a panel of four contrived specimens:

1) Negative (N)   
2) High Negative (HN), 0.3 x LoD   
3) Low Positive (LP), 1 x LoD and

# 4) Moderate Positive (MP), 3.0 x LoD

Contrived specimens were made by diluting frozen bacterial stocks of GAS and S. dysgalactiae to the concentration noted above. The LoD values were based on the values obtained in the LoD study described in section $\mathbf { M } . 1 . d$ below. See section $\mathbf { M } . 2 . b$ below for details on the matrix equivalency and freeze-thaw study.

Three replicates of each panel member for each analte and the controls were processed and tested on the Solana Strep Complete assay at each site by two operators for five non-consecutive days using three instruments (2 operators x 3 replicates x 5 days x 3 sites $= 9 0$ results per concentration for each analyte). The LoD values were based on the values obtained in the LoD study. The reproducibility study results are acceptable. The results are shown in the Table 2

<table><tr><td rowspan=1 colspan=10>Table 2: Solana Strep Complete Inter-Laboratory Reproducibility</td></tr><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>PanelID</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>Combined</td></tr><tr><td rowspan=1 colspan=1>DetectedPPo/Total</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>DetectedT Pos/Total</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>DetectedPos/Total</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>DetectedPos/Total</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=4 colspan=1>GAS(ATCC19615)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>13/30</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>36/90</td><td rowspan=1 colspan=1>40.0%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=4 colspan=1>S.dysgalactiaesubspequisimilis(ATCC10009)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>6/30</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5/30</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>21/90</td><td rowspan=1 colspan=1>23.3%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

These results met the pre-defined acceptance criteria for the various panels and are described in labeling.

# Precision Study

Within-laboratory, inter-instrument precision was evaluated for the Solana Strep Complete Assay using the same panel of contrived specimens as in the reproducibility study above.

The study was conducted over 12 days at one site (in-house) with two operators (2 operators $\textbf { x } 1 2$ days x 3 replicates $= 7 2$ results per panel member). One operator used one instrument and the other used two instruments. The results are shown in Table 3.

<table><tr><td rowspan=1 colspan=8>Table 3: Precision</td></tr><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Panel ID</td><td rowspan=1 colspan=2>Operator 1</td><td rowspan=1 colspan=2>Operator 2</td><td rowspan=1 colspan=2>Combined</td></tr><tr><td rowspan=1 colspan=1>DetectedPos/Total</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>DetectedPos/Total</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>DetectedPos/Total</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=4 colspan=1>GAS(ATCC 19615)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>16/36</td><td rowspan=1 colspan=1>44.4%</td><td rowspan=1 colspan=1>15/36</td><td rowspan=1 colspan=1>41.7%</td><td rowspan=1 colspan=1>31/72</td><td rowspan=1 colspan=1>43.1%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=4 colspan=1>S. dysgalactiaesubsp equisimilis(ATCC 10009)</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>6/36</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>14/36</td><td rowspan=1 colspan=1>38.9%</td><td rowspan=1 colspan=1>20/72</td><td rowspan=1 colspan=1>27.8%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0/72</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

The precision study results met the pre-defined acceptance criteria for LP, MP, and PC specimens. The results of this study are acceptable and these results are described in labeling.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Solana Strep Complete assay incorporates process controls (PRC) in the lysis buffer tube that is used to monitor sample processing and evaluate the presence of substances inhibitory to helicase dependent amplification (HDA) and to confirm the integrity of assay reagents. The Quidel Molecular $\mathbf { A } + \mathbf { G }$ Streptococci Control Set #M111, which contains positive and negative controls, serves as an external processing and extraction controls for the Solana Strep Complete assay and were run each day of testing.

Studies were performed to determine the stability of specimens collected using the following routinely used swab systems: nylon, rayon and polyester swabs in Amies media, and rayon and polyester swabs in Stuart media, and rayon swabs in Amies gel. Freshly grown stocks of GAS or S. dysgalactiae of known titer were used to spike the swabs listed above. Triplicate testing for each analyte with each condition with each collection/transport system listed above demonstrated that specimens can be stored at $2 5 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ for 2 days and then at 2 to $8 ~ ^ { \circ } C$ for up to 6 more days prior to being tested in the Solana Strep Complete assay. A separate study was performed where the spiked samples were stored at $\mathrm { \leq } { - } 1 5 \mathrm {  { ^ \circ C } }$ or $\le - 7 0 ^ { \mathrm { o } } \mathrm { C }$ for a minimum of 32 days before testing.

# d. Detection limit:

The limit of detection (LoD) of the Solana Strep Complete assay was determined using contrived stocks of two GAS strains, one strain of Group C S. dysgalactiae subsp. equisimilis and one strain of Group G S. dysgalactiae subsp. equisimilis. For each strain, stocks were prepared by suspending and diluting freshly grown colonies on to $5 \%$ sheep blood tryptic soy agar. Colony counts from preliminary serial dilutions were associated with optical density to determine the LoD. The LoD was defined as the point at which at least $9 5 \%$ of all replicates tested positive $( \mathrm { C } _ { 9 5 } )$ . The GAS strains were serially diluted then spiked onto swabs. For each strain, the LoD was confirmed by testing 20 replicates for each of three dilutions at $0 . 3 { \mathrm { ~ x ~ } }$ LoD, $_ { 1 \mathrm { ~ x ~ } }$ LoD and $3 \mathrm { ~ x ~ }$ LoD. Each dilution for each strain was confirmed by plating on $5 \%$ sheep blood tryptic soy agar, incubating and counting colonies. The LoD study results are shown in Table 4.

<table><tr><td rowspan=1 colspan=3>Table 4: LoD for GAS and Streptococus dysgalactiae Groups C and G</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>CFU/ml</td></tr><tr><td rowspan=1 colspan=1>Group A Streptococcal strain 1(Streptococcus pyogenes)</td><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=1 colspan=1>1.5 x 104</td></tr><tr><td rowspan=1 colspan=1>Group A Streptococcal strain 2(Streptococcus pyogenes)</td><td rowspan=1 colspan=1>ATCC 12344</td><td rowspan=1 colspan=1>8.5 x 104</td></tr><tr><td rowspan=1 colspan=1>Pyogenic Group G Streptococcal(Streptococcus dysgalactiae subsp. equisimilis)</td><td rowspan=1 colspan=1>ATCC 12394</td><td rowspan=1 colspan=1>5.7 x 10</td></tr><tr><td rowspan=1 colspan=1>Pyogenic Group C Streptococcal(Streptococcus dysgalactiae subsp. equisimilis)</td><td rowspan=1 colspan=1>ATCC 10009</td><td rowspan=1 colspan=1>7.1 x 104</td></tr></table>

Based these studies, the highest observed LoD for Streptococcus pyogenes and Streptococcus dysgalactiae using the Solana Strep Complete Assay is $8 . 5 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ $\mathrm { C F U / m L }$ and $7 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , respectively. These LoD values are described in the labeling and they were used in determining the spiking levels for all subsequent analytical studies.

# e. Analytical reactivity:

Inclusivity studies were conducted with seven GAS strains, 14 Group C S. dysgalactiae strains and 11 Group G S. dysgalactiae strains (in addition to the four strains from the LoD studies above) against three different reagent lots with three instruments. Swabs were prepared as described in ${ \bf M } . 1 . d$ above and run with the Solana Strep Complete assay at the concentrations described in Table 5. The inclusivity study results and the final organism concentrations tested are shown in Table 5.

<table><tr><td rowspan=1 colspan=4>Table 5: Group A β-hemolytic Streptococcus Inclusivity</td></tr><tr><td rowspan=1 colspan=1>Strain #</td><td rowspan=1 colspan=1>ATCC/ CCUG**#</td><td rowspan=1 colspan=1>LancfieldType*</td><td rowspan=1 colspan=1>ConcentrationDetected CFU/ml</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC BAA-595</td><td rowspan=7 colspan=1>GAS</td><td rowspan=7 colspan=1>8.5 x 104</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 12384</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 49399</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC 700294</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>CCUG 33409</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CCUG 39158</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>CCUG 53553</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 9542</td><td rowspan=14 colspan=1>GCS</td><td rowspan=14 colspan=1>7.1 x 104</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 12388</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 35666</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>CCUG 1483</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>CCUG 6713</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CCUG 27479</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>CCUG 27480</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>CCUG 27658</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>CCUG 27659</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>CCUG 27664</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>CCUG 28115</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>CCUG 28116</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>CCUG 28238</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>CCUG 48477</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 6644</td><td rowspan=11 colspan=1>GGS</td><td rowspan=11 colspan=1>7.1 x 104</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CCUG 502</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>CCUG 27482</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>CCUG 27483</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>CCUG 33802</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CCUG 15679</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>CCUG 15680</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>CCUG 21557</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>CCUG 24070</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>CCUG 26147</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>CCUG 27477</td></tr></table>

\*All GCS and GGS tested were S. disgalactiae subsp. equisimilis or S. disgalactiae subsp dysgalactiae \*\*ATCC: American Type Culture Collection; CCUG: Culture Collection, University of Gӧteborg

Three results were obtained for each of the strains tested. Results showed that each strain was detected $100 \%$ of the time. These study results are acceptable and they are described in the labeling.

# f. Analytical specificity:

# i. Microbial cross-reactivity

An in silico BLAST analysis of primers used in the Solana Complete Strep assay against the NCBI database against 60 potential interfering virus/organisms did not show evidence of cross-reactivity.

The Solana Complete Strep assay was tested in the presence of 45 other microorganisms commonly found in throat specimens. Each potentially interfering microorganism was tested in the presence of $2 \textbf { x }$ LoD of each of the GAS and S. dysgalactiae organisms used in the LoD studies in ${ \bf M } . 1 . d$ above, in the presence of clinically relevant levels of viruses (e.g., $1 0 ^ { 5 } \mathrm { p f u / m l } )$ and bacteria/yeast (e.g., $1 0 ^ { 6 } \mathrm { { c f u } / \mathrm { { m L } } }$ ). All strain combinations were spiked into contrived negative matrix. The strains included in the cross-reactivity study are shown in Table 6.

<table><tr><td rowspan=1 colspan=3>Table 6: Strains Included in Cross-Reactivity</td></tr><tr><td rowspan=1 colspan=3>Strain</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffii</td><td rowspan=1 colspan=1>Legionella jordanis</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium haemolyticum</td><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Moraxella cartarrhalis</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>Neisseria subflava</td><td rowspan=1 colspan=1>Streptococcus oralis</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Peptostreptococcus micros</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheria</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Streptococcus sanguinis</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Staphylococcus aureus MRSA</td><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Adenovirus Type 1</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae type A</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Adenovirus Type 11 (Slobitski)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumonia</td><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>Parainfluenza Type 4B (VR-1377)</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Streptococcus bovis</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>Streptococcus canis</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Streptococcus gordonii(Virdans type)</td><td rowspan=1 colspan=1>Rhinovirus Type 15 (1734)</td></tr></table>

Of the 45 microorganisms tested that might be found in throat specimens, Klebsiella pneumoniae, Serratia marcescens and Enterococcus faecalis each cross-reacted once out of six times tested (triplicate testing was repeated for each cross-reactive strain) with the Sonala Strep Complete assay.

These study results are acceptable and they are described in the labeling.

# ii. Microbial interference

Microbial interference studies were conducted in triplicate with each of the 45 bacteria, yeast and viruses listed above in Table 6 in triplicate in the presence of clinically relevant levels of viruses (e.g., $1 0 ^ { 5 } \mathrm { P F U / m l }$ ) and bacteria/yeast (e.g., $1 0 ^ { 6 }$ $\mathrm { C F U / m l } _ { , }$ ) or higher. All of the strains used in the LoD studies in $\mathbf { M } . 1 . d$ above strains were prepared as described in the cross-reactivity studies in $\mathbf { M . l } f . i$ above along with the spiked onto the swab at a $2 \textbf { x }$ LoD concentration. These studies were conducted using three instruments. None of the bacteria, yeast or viruses demonstrated interference with the detection of the Group A Streptococcal strains in any of the replicates tested.

These study results are acceptable and they are described in the labeling.

# iii. Interfering substances

28 chemical and biological substances were evaluated for potential to interfere with the Solana Strep Complete assay, including blood $( 5 \% \mathrm { v / v } )$ and human saliva $( 1 0 \% \mathrm { v / v ) }$ . Each substance was tested in triplicate using the strains used in the LoD studies in ${ \bf M } . 1 . d$ above at $2 \times \mathrm { L o D }$ at medically relevant concentrations. None of the substances tested were found to interfere with the Solana Strep Complete assay in any of the replicates tested.

These study results are acceptable and they are described in the labeling.

# iv. Carryover/Cross-contamination

A study to assess carryover contamination was conducted by three operators using three different Solana Instruments with one GAS strain (ATCC 19615) and one $S .$ . dysgalactiae subsp equisimilis strain (ATCC 10009). Fifty high positive contrived specimens for each analyte (at ${ \sim } 2 0 0 0 \mathrm { x }$ LoD) and 50 negative contrived specimens were spiked onto swabs and tested with the Solana Strep Complete assay. Five positive and five negative swabs were tested in an alternating order. No carryover/cross contamination was observed in the course of this study.

These study results are acceptable and they are described in the labeling.

# g. Assay cut-off:

Cut-off values are based on set time thresholds for amplification and were determined based on results from preliminary LoD studies and initial clinical specimen testing. The cut-offs were set based on the longest time observed to achieve a valid GAS and S. dysgalactiae results from these studies.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

A comparison study was conducted between negative clinical matrix and a contrived negative matrix in order to validate the use of the contrived negative matrix in place of a clinical negative matrix for use in the analytical studies (described in section M1 above).

Contrived negative matrix consisted of liquid Amies transport medium, Porcine Gastric Mucin (PGM), Phosphate Buffered Saline (PBS), Bovine Serum Albumin and sodium azide. The matrix comparison study results are shown in Table 7.

<table><tr><td rowspan=1 colspan=6>Table 7: Matrix Comparison Study</td></tr><tr><td rowspan=2 colspan=2>Panel ID</td><td rowspan=1 colspan=2>ContrivedNegative Matrix</td><td rowspan=1 colspan=2>Pooled NegativeClinical Matrix</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=1 colspan=1>GAS(ATCC 19615)</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococusdysgalactiaesubsp equisimilis(ATCC 12394)</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

These studies demonstrate that the contrived negative matrix is equivalent to a clinical matrix. These study results are acceptable and support the use of contrived matrix to facilitate the conduct of analytical studies.

Fresh contrived specimens and frozen specimens were prepared as described in the LoD studies $( { \bf M } . 1 . d )$ and precision studies (M.1.a) respectively and compared. The results of the fresh versus frozen study are shown in Table 8.

<table><tr><td rowspan=1 colspan=6>Table 8: Fresh Versus Frozen Study</td></tr><tr><td rowspan=2 colspan=2>Panel ID</td><td rowspan=1 colspan=2>ContrivedNegative Matrix</td><td rowspan=1 colspan=2>Pooled NegativeClinical Matrix</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=3 colspan=1>GAS(ATCC 19615)</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.3 x LoD</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>47/72</td><td rowspan=1 colspan=1>65%</td></tr></table>

These studies demonstrate that freezing does not appear to impact results for the Solana Strep Complete assay, in the context of the contrived analytical studies submitted in this submission. These study results are acceptable and support the use of frozen specimens in contrived matrix to facilitate the conduct of analytical studies.

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the Solana Strep Complete assay was demonstrated with 2688 prospectively collected fresh throat specimens at three sites across the United States. The assay was evaluated for the qualitative detection and differentiation of GAS and S. dysgalactiae using nucleic acids isolated from the patients’ throat swabs. A single specimen was collected per patient. Samples were collected using Polyester or Rayon Swab and transported in liquid Amies or liquid Stuart non-nutritive medium.

All 2688 fresh throat swab specimens were processed for culture and identification of GAS, GCS and GGS at the testing sites using well-accepted microbiological methods. Cultured isolates were typed by latex agglutination. $\beta$ -hemolytic isolates that were typed as Group C or G were subcultured and the species were determined using an FDA-cleared MALDI-TOF assay. Swab transport fluid was also tested using another FDA-cleared nucleic acid amplification test (NAAT) and cultured at a central reference laboratory.

A specimen was considered positive if culture from the swab, swab transport fluid or the results from another FDA cleared assay were positive for GAS and/or GCS/GGS.

Two results were determined to be invalid in these studies. The invalid rate observed during the clinical prospective study for this device was $0 . 0 7 \%$ (2/2688). The breakdown of performance by analyte based on the 2686 valid swab specimens is summarized in Tables 9 and 10.

<table><tr><td colspan="4" rowspan="1">Table 9: Clinical Performance Data for the Solana StrepComplete vs. Dual Composite Culture and NAAT forGroup A β-hemolytic Streptococcus</td></tr><tr><td colspan="4" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">475</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">500</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2180</td><td colspan="1" rowspan="1">2186</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">481</td><td colspan="1" rowspan="1">2205</td><td colspan="1" rowspan="1">2686</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 98.8% (475/481) 95% CI (97.3%-99.4%)Specificity: 98.9% (2180/2205) 95% CI (98.3%-99.2%)</td></tr><tr><td colspan="4" rowspan="1">Site 1</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">94</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">679</td><td colspan="1" rowspan="1">681</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">683</td><td colspan="1" rowspan="1">775</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 97.8% (90/92) 95% CI (92.4%-99.4%)Specificity: 99.4% (679/683) 95% CI (98.5%-99.8%)</td></tr><tr><td colspan="4" rowspan="1">Site 2</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">510</td><td colspan="1" rowspan="1">511</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">516</td><td colspan="1" rowspan="1">601</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 98.8%% (84/85) 95% CI (93.6%-99.8%)Specificity: 98.8%% (510/516) 95% CI (97.5%-99.5%)</td></tr><tr><td colspan="4" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">103</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">492</td><td colspan="1" rowspan="1">495</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">495</td><td colspan="1" rowspan="1">598</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 97.1% (100/103) 95% CI (91.8%-99.0%)Specificity: 99.4% (492/495) 95% CI (98.2%-99.8%)</td></tr><tr><td colspan="4" rowspan="1">Site 4</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">95</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">254</td><td colspan="1" rowspan="1">254</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">266</td><td colspan="1" rowspan="1">349</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100% (37/39) 95% CI (95.6%-100%)Specificity: 95.5% (254/266) 95% CI (92.3%-97.4%)</td></tr><tr><td colspan="4" rowspan="1">Site 5</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">GAS</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">118</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">245</td><td colspan="1" rowspan="1">245</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">245</td><td colspan="1" rowspan="1">363</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100% (118/118) 95% CI (96.8%-100%)Specificity: 100% (245/245) 95% CI (98.5%-100%)</td></tr></table>

These study results are acceptable and they are described in the labeling.

<table><tr><td colspan="4" rowspan="1">Table 10: Clinical Performance Data for the SolanaStrep Complete vs. Dual Composite Culture and NAATfor Pyogenic Group C and G β-hemolytic Streptococcusdysgalactiae</td></tr><tr><td colspan="4" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">92</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2594</td><td colspan="1" rowspan="1">2594</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">2608</td><td colspan="1" rowspan="1">2686</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100% (78/78) 95% CI (95.3%-100%)Specificity: 99.5% (2594/2608) 95% CI (99.1%-99.7%)</td></tr><tr><td colspan="4" rowspan="1">Site 1</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">739</td><td colspan="1" rowspan="1">739</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">743</td><td colspan="1" rowspan="1">775</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100.0% (32/32) 95% CI (89.3%-100.0%)Specificity: 96.6% (739/743) 95% CI (98.6%-99.8%)</td></tr><tr><td colspan="4" rowspan="1">Site 2</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">580</td><td colspan="1" rowspan="1">580</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">585</td><td colspan="1" rowspan="1">601</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 100% (16/16) 95% CI (80.6%-100%)Specificity: 99.1% (593/605) 95% CI (98.0%-99.6%)</td></tr><tr><td colspan="4" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">568</td><td colspan="1" rowspan="1">568</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">572</td><td colspan="1" rowspan="1">598</td></tr><tr><td colspan="4" rowspan="1">Sensitivity:100% (26/26) 95% CI (87.1%-100%)Specificity: 99.3% (568/572) 95% CI (98.2%-99.7%)</td></tr><tr><td colspan="4" rowspan="1">Site 4</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">347</td><td colspan="1" rowspan="1">347</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">347</td><td colspan="1" rowspan="1">349</td></tr><tr><td colspan="4" rowspan="1">Sensitivity:100% (2/2) 95% CI (34.2%-100%)Specificity: 100% (347/347) 95% CI (98.9%-100.0%)</td></tr><tr><td colspan="4" rowspan="1">Site 5</td></tr><tr><td colspan="1" rowspan="2">Solana StrepComplete</td><td colspan="3" rowspan="1">Composite Dual Culture and NAAT</td></tr><tr><td colspan="1" rowspan="1">GCS/GGS</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">360</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">361</td><td colspan="1" rowspan="1">363</td></tr><tr><td colspan="4" rowspan="1">Sensitivity:100% (2/2) 95% CI (34.2%-100%)Specificity: 99.7% (361/360) 95% CI (98.4%-100.0%)</td></tr></table>

All pyogenic Group C and Group $\mathrm { G }$ streptococcal species found during this assay belong to the species S. dysgalactiae. These study results are acceptable and they are described in the labeling.

The external quality control isolates used in these studies were from the Quidel Molecular $\mathbf { A } { + } \mathbf { G }$ Control Set #M111 consisting of Streptococcus pyogenes Z018 (for the GAS primer set) and Streptococcus dysgalactiae Z068 (for the GCS/GGS primer set), which serve as processing and extraction controls. The control isolates were tested with acceptable results.

All GAS positive external controls were detected accurately ( $100 \%$ , 180/180) by the Solana Strep Complete assay. All S. dysgalactiae positive controls were detected accurately ( $100 \%$ , 180/180). All negative controls were not detected $( 0 \% , 0 / 1 8 0 )$ . There were no invalid control results.

$b$ . Clinical specificity: See Tables 9 and 10 above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

The overall prevalence of GAS or GCS/GGS in patients tested during this study based on culture results alone was $1 6 . 0 \%$ (431/2686) for GAS and $2 . 4 \% ( 6 5 / 2 6 8 6 )$ for GCS/GGS. The overall prevalence of GAS or GCS/GGS in patients tested during this study based on a combination of culture results and another FDA-cleared NAAT assay was $1 7 . 9 \%$ (481/2686) for GAS and $2 . 9 \%$ (78/2686) for GCS/GGS. All clinical specimens collected during this study were collected between February, 2016 and July 2016.

# N. Instrument Name:

Solana™ Instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No X

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No _

3. Specimen Identification: Specimens are identified by scanning a barcode or by manual entry.

4. Specimen Sampling and Handling:

Swab specimens are expressed in transport medium. $5 0 \mu \mathrm { l }$ of the expressed specimen are transferred to a lysis tube. After heat lysis, $5 0 \mu \mathrm { l }$ of lysed specimen is transferred to a reaction tube for automated amplification and detection. See section I above for more information.

# 5. Calibration:

The end user is not required to calibrate the instrument. Automated calibration happens by comparing between the measured magnitude of the optical signal of and an integrated calibration standard and the expected magnitude of the optical signal.

6. Quality Control:

See section M.1.c for information on internal and external controls.   
See section M.3.a for information on external control performance during clinical trials.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.