# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY ONLY TEMPLATE

# B. Purpose for Submission:

To obtain a substantial equivalence determination for a new device.

C. Measurand: Nucleic acids from:

• Campylobacter (C. coli/ C. jejuni)   
• Salmonella   
• Shiga toxin 1 (stx1)   
• Shiga toxin 2 (stx2)   
• Escherichia coli serotype O157 (when stx1 or stx2 are detected)   
• Shigella

D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) coupled with chip based detection

# E. Applicant:

Great Basin Scientific, Inc.

# F. Proprietary and Established Names:

Assay: Great Basin Stool Bacteria Pathogens Panel Instrument: Portrait Analyzer System, Model: PA500

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990, Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assay

2. Classification: Class II

PCH – Gastrointestinal Pathogen Panel Multiplex Nucleic Acid-Based Assay System PCI – Gastrointestinal Pathogen Panel Multiplex Nucleic Acid-Based Assay System OOI – Real time nucleic acid amplification system

4. Panel:

83 - Microbiology

# H. Intended Use:

The Great Basin Stool Bacterial Pathogens Panel is a multiplexed, qualitative test for the detection and identification of DNA targets of enteric bacterial pathogens. The Stool Bacterial Pathogens Panel detects nucleic acids from:

• Campylobacter (C. coli/C. jejuni)   
• Salmonella   
• Shiga toxin 1 (stx1)   
• Shiga toxin 2 (stx2)   
• Escherichia coli serotype O157   
• Shigella

Shiga toxin genes are found in Shiga toxin-producing strains of $E$ . coli (STEC/EHEC/VTEC) and Shigella dysenteriae. The E. coli O157 test result is only reported if a Shiga toxin gene (stx1 and/or $s t x 2$ ) is also detected.

The Stool Bacterial Pathogens Panel is performed directly from Cary Blair or C&S Medium preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis, or colitis and is performed on the Portrait™ Analyzer.

The test is intended for use as an aid in the diagnosis of specific agents of gastrointestinal illness in conjunction with clinical and epidemiological information; however, it is not to be used to monitor these infections. Positive results do not rule out co-infection with other organisms and may not be the definitive cause of patient illness. Negative test results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test, or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease. Concomitant culture is necessary if organism recovery or further typing of bacterial agents is desired.

2. Indication(s) for use: See Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: For use with the PA500 Portrait Analyzer System

# I. Device Description:

The Great Basin Stool Bacterial Pathogens Panel on the PA500 Portrait™ System utilizes automated, hot-start PCR amplification technology to amplify specific nucleic acid sequences that are then detected using hybridization probes immobilized on a modified silicon chip surface, in a single-use, self-contained test cartridge.

An aliquot of the specimen (stool preserved in stool transport media) is first processed using the Sample Preparation Device (SPD). An aliquot of the eluate obtained from the SPD is loaded into the sample port of the SBPP Test Cartridge.

Genomic DNA is extracted from microbial cells and diluted to reduce potential inhibitors of the PCR. During the PCR process, biotin-labeled primers direct the amplification of specific nucleic acid sequences within a conserved region for identification of: a bacterial sample processing control (SPC), Campylobacter coli/Campylobacter jejuni, Salmonella spp., Shigella spp., Shiga toxin 1, Shiga toxin 2, and E. coli serotype O157.

Following PCR, biotin-labeled, amplified target DNA sequences are hybridized to sequence specific probes immobilized on the silicon chip surface, and incubated with anti-biotin antibody conjugated to the horseradish peroxidase enzyme (HRP). The unbound conjugate is washed away, and tetramethylbenzidine (TMB) is added to produce a colored precipitate at the location of the probe/target sequence complex. The resulting signal is detected by the automated Portrait™ Optical Reader within the PA500 Portrait™ Analyzer System. The SPC undergoes the same extraction, amplification, and detection steps as the sample in order to monitor for inhibitory substances, as well as process inefficiency due to instrument or reagent failure. No operator intervention is required once the sample is loaded into the sample port, and the Stool Bacterial Pathogens Panel cartridge is loaded into the Portrait™ Analyzer.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Nanosphere Verigene Enteric Pathogens Nucleic Acid Test

2. Predicate 510(k) number(s): K140083

# 3. Comparison with predicate:

<table><tr><td></td><td rowspan=1 colspan=1>Features/Characteristics</td><td rowspan=1 colspan=1>Stool Bacterial Pathogens Panel(SBPP)</td><td rowspan=1 colspan=1>Predicate DeviceVerigene® EP (K140083)</td></tr><tr><td></td><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Great Basin Scientific, Inc.</td><td rowspan=1 colspan=1>Nanosphere</td></tr><tr><td></td><td rowspan=1 colspan=1>Trade Name</td><td rowspan=1 colspan=1>Great Basin Stool Bacterial Pathogen Panel(SBPP)</td><td rowspan=1 colspan=1>Verigene Enteric Pathogen Nucleic AcidTest</td></tr><tr><td></td><td rowspan=1 colspan=1>510(k) Number</td><td rowspan=1 colspan=1>K163571</td><td rowspan=1 colspan=1>K140083</td></tr><tr><td></td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=2 colspan=2>Qualitative/QuantitativeIntended Use/Indications for Use</td><td rowspan=1 colspan=1>Qualitative/Quantitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>The Great Basin Stool Bacterial PathogensPanel is a multiplexed, qualitative test forthe detection and identification of DNAtargets of enteric bacterial pathogens. TheStool Bacterial Pathogens Panel detectsnucleic acids from:Campylobacter (C. coli and C. jejuni)SalmonellaShiga toxin 1 (stxl)Shiga toxin 2 (stx2)Escherichia coli serotype O157ShigellaShiga toxin genes are found in Shiga toxin-producing strains of E. coli(STEC/EHEC/VTEC) and Shigelladysenteriae. The E. coli O157 test result isonly reported if a Shiga toxin gene (stx1and/or stx2) is also detected.The Stool Bacterial Pathogens Panel isperformed directly from Cary Blair or C&amp;SMedium preserved stool specimens fromsymptomatic patients with suspected acutegastroenteritis, enteritis, or colitis and isperformed on the Portrait™M Analyzer.The test is intended for use as an aid in thediagnosis of specific agents ofgastrointestinal illness in conjunction withclinical and epidemiological information.Positive results do not rule out co-infectionwith other organisms and may not be thedefinitive cause of patient illness. Negativetest results in the setting of clinical illnesscompatible with gastroenteritis may be dueto infection by pathogens that are notdetected by this test, or non-infectiouscauses such as ulcerative colitis, irritablebowel syndrome, or Crohn&#x27;s disease.Concomitant culture is necessary iforganism recovery or further typing ofbacterial agents is desired.</td><td rowspan=1 colspan=1>The Verigene Enteric Pathogens NucleicAcid Test (EP) is a multiplexed, qualitativetest for simultaneous detection andidentification of common pathogenicenteric bacteria, viruses, and geneticvirulence markers from liquid or soft stoolpreserved in Cary-Blair medium, collectedfrom individuals with signs and symptomsof gastrointestinal infection. The test isperformed on the automated NanosphereVerigene System utilizing reversetranscription (RT), polymerase chainreaction (PCR), and array hybridization todetect specific gastrointestinal microbialnucleic acid gene sequences associatedwith the following pathogenic bacteria andviruses:• Campylobacter Group (composed ofC. coli, C. jejuni, and C. lari)•Salmonella speciesShigella species (including S.dysenteriae, S. boydii, S. sonnei, andS. flexneri)Vibrio Group (composed of V.cholerae and V. parahaemolyticus)Yersinia enterocolitica•Norovirus GI/GIIRotavirus AIn addition, EP detects the Shiga toxin 1gene and Shiga toxin 2 gene virulencemarkers. Shiga toxin producing E. coli(STEC) typically harbor one or both genesthat encode for Shiga toxins 1 and 2.EP is indicated as an aid in the diagnosis ofspecific agents of gastrointestinal illness, inconjunction with other clinical, laboratory,and epidemiological information; however,is not to be used to monitor theseinfections. EP also aids in the detection andidentification of acute gastroenteritis in thecontext of outbreaks.Due to the limited number of positivespecimens collected for certain organismsduring the prospective clinical study,performance characteristics for Yersinia</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>species were primarily established withcontrived specimens. Concomitant cultureis necessary for organism recovery andfurther typing of bacterial agents.EP results should not be used as the solebasis for diagnosis, treatment, or otherpatient management decisions. Confirmedpositive results do not rule out co-infectionwith other organisms that are not detectedby this test, and may not be the sole ordefinitive cause of patient illness. NegativeEP results in the setting of clinical illnesscompatible with gastroenteritis may be dueto infection by pathogens that are notdetected by this test or noninfectious causessuch as ulcerative colitis, irritable bowelsyndrome, or Crohn&#x27;s disease</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Human Stool sample preserved in CaryBlair or C&amp;S Preservation and TransportMedia</td><td rowspan=1 colspan=1>Human Stool sample preserved in Cary-Blair Medium</td></tr><tr><td rowspan=1 colspan=1>Sample Lysis and DNAExtraction</td><td rowspan=1 colspan=1>Automated sample lysis and DNAextraction in a self-contained cartridge</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Amplification Technology</td><td rowspan=1 colspan=1>Multiplex polymerase chain reaction (PCR)</td><td rowspan=1 colspan=1>Reserve transcription (RT) polymerasechange reaction (PCR)</td></tr><tr><td rowspan=1 colspan=1>Detection Technology</td><td rowspan=1 colspan=1>Colorimetric target specific hybridizationto probe on a chip surface, optical reader,automated software with built-in resultinterpretation.</td><td rowspan=1 colspan=1>Gold/Silver nanoparticle probe detection ofbacterial-specific DNA on complementaryoligo-microarray. Optical light scatterdetection of gold-silver aggregates</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>One internal processing control (wholeorganism)  complete assay control</td><td rowspan=1 colspan=1>Two internal processing controls(hybridization control and extraction/assaycontrol</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>PA500 Portrait Analyzer</td><td rowspan=1 colspan=1>Verigene Reader and Processor SP</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>&lt;2 hours</td><td rowspan=1 colspan=1>~2 hours</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guideline: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens

# L. Test Principle:

The PA500 Portrait Analyzer System is a fully automated system that includes: the Portrait Analyzer, single-use Stool Bacterial Pathogen Panel Cartridges, and the Portrait Data Analysis Software Program. The Portrait System is designed to perform automated sample preparation, PCR, and optical chip-based detection with integrated data analysis in less than two hours. The Portrait System was granted 510(k) clearance for the Portrait Toxigenic C. difficile Assay (K113358), Portrait GBS Assay (K143312), Staph ID/R Blood Culture Panel (K152470) and the Shiga Toxin Direct Test (K152955).

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

A Reproducibility Study was conducted at 3 clinical study sites; 2 external and 1 internal. The study incorporated several variables including 6 different operators (2 per site), 70 different Portrait Analyzers and 10 different cartridge lots.

A panel consisting of 7 different samples was tested in triplicate over 5 nonconsecutive days by each operator. Each analyte detected in the Stool Bacterial Pathogens Panel (SBPP) was included as a low positive (1.5X LoD) and a moderate positive (3X LoD) in the reproducibility panel. A single negative was also included.

The overall results of the Reproducibility Study are summarized in the table below. There w $1 s \geq 9 5 \%$ agreement of positive results for analytes that were present in low concentrations in the samples (1.5X LoD) and $100 \%$ agreement of positive results for analytes present at moderate concentration (3X LoD). There was $100 \%$ agreement in negative results from the negative sample.

# Summary of Reproducibility Results

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=3 colspan=1>Campylobactercoli/jejuni</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>100%(90/90)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100%(90/90)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.8% (540/541)</td></tr><tr><td rowspan=3 colspan=1>Salmonella</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>96.7%(87/90)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100%(90/90)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.8%b (540/541)</td></tr><tr><td rowspan=3 colspan=1>Shiga toxin 1</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>97.8%(90/92)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100% (91/91)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (540/540)</td></tr><tr><td rowspan=3 colspan=1>Shiga toxin 2</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>95.7%(88/92)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100% (91/91)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (540/540)</td></tr><tr><td rowspan=3 colspan=1>E. coliSerotype O157</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>97.8%(90/92)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100%(91/91)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (540/540)</td></tr><tr><td rowspan=3 colspan=1>Shigella</td><td rowspan=1 colspan=1>1.5X LoD</td><td rowspan=1 colspan=1>100%(90/90)</td></tr><tr><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>100%(90/90)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.8% (540/541)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (540/540)</td></tr></table>

b. Linearity/assay reportable range:

Not applicable (N/A)

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

External Controls:

External positive and negative controls are intended to monitor for correct procedural technique and reagent integrity. External controls are not provided by Great Basin.

External Controls should be treated as if they are patient specimens.   
The External Negative Control is intended to detect reagent or environmental contamination by target nucleic acids.   
All External Controls should yield the expected results, with no internal control failures (INVALID results).   
The following External Control materials are recommended to allow the user to choose the most appropriate controls for the lab’s Quality Control Plan:

External Negative Control may be any of the following:

- an aliquot of pure Cary Blair or C&S media   
- a fresh (never frozen), previously characterized sample known to be negative for all assay targets   
- a frozen, raw stool specimen that has been thawed and then preserved and known to be negative for all assay targets

External Positive Control: commercially-available ATCC strains for all assay targets are available, and may be pooled and diluted in stool preservation media and processed as a patient specimen.

Specimen Processing Control (SPC):

The SPC controls for all analytical steps in the procedure, including DNA extraction, PCR amplification of target DNA sequences, hybridization, and detection on the chip surface. The SPC consists of B. subtilis cells lyophilized directly onto the frit of the SPD, and is processed along with the stool sample. The SPC detects inhibition of PCR reactions, ensuring the reaction conditions are appropriate and that the amplification reagents are functional. The SPC signal should be ”Detected” in a sample in which target sequences were ”Not Detected“ for all Stool Bacterial Pathogens Panel targets, and can be ”Detected“ or ”Not Detected” in samples for which at least one target sequence of the SBPP was ”Detected“.

Specimen Stability:

The recommended storage time and temperature conditions for Cary-Blair or C&S preserved stool specimens, prior to testing via the SBPP are as follows:

• Refrigerated storage $( 2 ^ { \circ } - 8 ^ { \circ } \mathrm { C } )$ for up to 120 hours (5 days) • Room temperature (RT) $( 2 0 ^ { \circ } { - } 2 5 ^ { \circ } \mathrm { C } )$ storage for up to 2 hours • The combination of 2-hour storage at room temperature $( 2 0 ^ { \circ } { - } 2 5 ^ { \circ } \mathrm { C } )$ followed by refrigerated storage for up to 120 hours.

To assess the stability of the SBPP nucleic acid targets under the recommended storage conditions, a Specimen Stability Study was conducted to evaluate each suggested time and temperature condition. The Specimen Stability Study tested 8 strains for which the LoD was experimentally measured:

• Campylobacter coli (ATCC 43486)   
• Campylobacter jejuni (ATCC 49943)   
• Escherichia coli (ATCC BAA-2196, stx1+/stx2+/non-O157)   
• Escherichia coli (ATCC 43895, stx1+/stx2+/O157+)   
• Salmonella bongori (ATCC 43975)   
• Salmonella enterica (ATCC 13311)   
• Shigella flexneri (ATCC 25929)   
• Shigella sonnei (ATCC 29930)

The contrived positive samples were prepared by spiking fresh cultures of each organism into similar pooled, preserved, clinical negative stool matrix. The results of the specimen stability study support the recommended storage time and temperature conditions for Cary-Blair or C&S preserved stool specimens Results are summarized in the tables below:

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=6>Correct SBPP Result</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ATCCID</td><td rowspan=1 colspan=1>SampleInput≤3X LoD(CFU/mL)</td><td rowspan=1 colspan=1>Fresh(&lt; 30 min)</td><td rowspan=1 colspan=1>2 hr RT(20°-25C)</td><td rowspan=1 colspan=1>24 hr2°-8</td><td rowspan=1 colspan=1>48 hr2°-8</td><td rowspan=1 colspan=1>72 hr2°-8°</td><td rowspan=1 colspan=1>2 hr RT(20-25°C) +6 days 2°-8°C</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>43486</td><td rowspan=1 colspan=1>5.1 x 10^{3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>49943</td><td rowspan=1 colspan=1>7.8 x 102</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/non-O157)</td><td rowspan=1 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>3.4 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/0157+)</td><td rowspan=1 colspan=1>43895</td><td rowspan=1 colspan=1>3.9 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>43975</td><td rowspan=1 colspan=1>6.5 x 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>13311</td><td rowspan=1 colspan=1>5.9 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>25929</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>29930</td><td rowspan=1 colspan=1>4.4 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# Specimen Stability Study Results

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=4>Correct SBPP Result</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ATCCID</td><td rowspan=1 colspan=1>Concentration≤3X LoD(CFU/mL)</td><td rowspan=1 colspan=1>Fresh(&lt; 30 min)</td><td rowspan=1 colspan=1>96 hr 2°-8°C</td><td rowspan=1 colspan=1>120 hr 2°-8°C</td><td rowspan=1 colspan=1>2 hr RT (20°-25C) +5 days 2°-8°C</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>43486</td><td rowspan=1 colspan=1>2.1 x 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>49943</td><td rowspan=1 colspan=1>4.4 x 10^3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/non-O157)</td><td rowspan=1 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>2.1 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/0157+)</td><td rowspan=1 colspan=1>43895</td><td rowspan=1 colspan=1>3.8 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>43975</td><td rowspan=1 colspan=1>6.5 x 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3a</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>13311</td><td rowspan=1 colspan=1>5.4 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>25929</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>29930</td><td rowspan=1 colspan=1>2.3 x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>ied Salmonella</td><td rowspan=1 colspan=2>but additionally detected Shiga</td><td rowspan=1 colspan=2></td></tr></table>

Fresh versus Frozen:

A Fresh vs. Frozen Study was performed to support the use of frozen, transport media preserved stool specimens in SBPP for the Frozen Retrospective and Reproducibility Studies, as well as for discordant resolution of prospective samples.

The Fresh vs. Frozen Study tested the performance of the SBPP on contrived positive samples that were subject to three (3) freeze/thaw cycles. Demonstration of specimen stability after the third freeze/thaw is important because generation of a Frozen Retrospective panel from archived clinical specimens requires a two freeze thaws prior to shipment to clinical sites. Clinical sites will perform the third thawing of archived specimens prior to testing. Samples tested in the Reproducibility Study were subjected to two freeze/thaws.

Contrived positive samples for the Fresh vs. Frozen Study were prepared with fresh bacterial cells spiked into a pooled, preserved, negative, clinical stool. The panel for the Fresh vs. Frozen Study was comprised of eight (8) SBPP target strains that included:

· Campylobacter coli (ATCC 43486 cadF+)   
· Campylobacter jejuni (ATCC 49943 cadF+)   
· Escherichia coli (ATCC BAA-2196 stx1+/stx2+/non-O157)   
· Escherichia coli (ATCC 43895 stx1+/stx2+/O157+)   
· Salmonella bongori (ATCC 43975 invA+)   
· Salmonella enterica (ATCC 13311 invA+)   
· Shigella flexneri (ATCC 25929 ipaH+) Shigella sonnei (ATCC 29930 ipaH+)

The E. coli (ATCC BAA-2196), E. coli (ATCC 43895), S. flexneri (ATCC 25929) and S. sonnei (ATCC 29930) strains were stable following three (3) freeze thaw cycles at ${ < } 1 \mathrm { X }$ LoD, 1X LoD, and 5X LoD.

Freeze/thaw had a negative impact on contrived samples spiked with Salmonella species, S. enterica (ATCC 13311) and S. bongori (ATCC 43975), and the Campylobacter species, C. coli (ATCC 43486) and $C .$ jejuni (ATCC 49943) .

The results of the Fresh vs. Frozen study are summarized in the tables below.

Fresh vs. Frozen Study Summary (E. coli and Shigella sp)   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>ATCC ID</td><td rowspan=2 colspan=1>Input Conc.(X LOD)</td><td rowspan=2 colspan=1>CFU/mL</td><td rowspan=1 colspan=4>Correct SBPP Results</td></tr><tr><td rowspan=1 colspan=1>T0= FreshPre-Freeze</td><td rowspan=1 colspan=1>T1= 1XFreeze/Thaw Freeze/Thaw</td><td rowspan=1 colspan=1>T2 = 2XFreeze/Thaw Freeze/Thaw</td><td rowspan=1 colspan=1>T3 =3XFreeze/Thaw</td></tr><tr><td rowspan=3 colspan=1>E. coli(stx1+/stx2+/non-0157)</td><td rowspan=3 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>&lt;1X LOD</td><td rowspan=1 colspan=1>2.0 x 103</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>1X LOD</td><td rowspan=1 colspan=1>1.5 x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>5X LOD</td><td rowspan=1 colspan=1>5.0 x 104</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=3 colspan=1>E. coli(stx1+/stx2+/0157+)</td><td rowspan=3 colspan=1>43895</td><td rowspan=1 colspan=1>&lt;1X LOD</td><td rowspan=1 colspan=1>2.0 x 103</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>1/5</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=1 colspan=1>1X LOD</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>5X LOD</td><td rowspan=1 colspan=1>8.0 x 104</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=3 colspan=1>Shigella flexneri</td><td rowspan=3 colspan=1>25929</td><td rowspan=1 colspan=1>&lt;1X LOD</td><td rowspan=1 colspan=1>1.4 x 103</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>3/5</td></tr><tr><td rowspan=1 colspan=1>1X LOD</td><td rowspan=1 colspan=1>2.3 x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>5X LOD</td><td rowspan=1 colspan=1>7.0 x 104</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=3 colspan=1>Shigella sonnei</td><td rowspan=3 colspan=1>29930</td><td rowspan=1 colspan=1>&lt;1X LOD</td><td rowspan=1 colspan=1>1.4 x 103</td><td rowspan=1 colspan=1>1/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>2/5</td></tr><tr><td rowspan=1 colspan=1>1X LOD</td><td rowspan=1 colspan=1>2.1 x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>5X LOD</td><td rowspan=1 colspan=1>7.0 x 104</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr></table>

Fresh vs. Frozen Study Summary (Salmonella spp.)   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>ATCCID</td><td rowspan=2 colspan=1>Input Conc.(XLMD)</td><td rowspan=2 colspan=1>FUmL</td><td rowspan=1 colspan=3>Correct SBPPResults</td></tr><tr><td rowspan=1 colspan=1>T0= FreshPre-Freeze</td><td rowspan=1 colspan=1>T1= 1XFreeze/Thaw</td><td rowspan=1 colspan=1>T2 = 2XFreeze/Thaw</td></tr><tr><td rowspan=3 colspan=1>Salmonellaenterica</td><td rowspan=3 colspan=1>13311</td><td rowspan=1 colspan=1>&lt;1XLD</td><td rowspan=1 colspan=1>5.6x 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=1>1XLOD</td><td rowspan=1 colspan=1>21x 104</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>5XL0D</td><td rowspan=1 colspan=1>1.1 x 105</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=3 colspan=1>Salmonellaenterica(repeated)</td><td rowspan=3 colspan=1>13311</td><td rowspan=1 colspan=1>&lt;1XLD</td><td rowspan=1 colspan=1>28x 103</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1X1D</td><td rowspan=1 colspan=1>29x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5XLD</td><td rowspan=1 colspan=1>1.3x 105</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5ab</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Salmonellabongori</td><td rowspan=3 colspan=1>43975</td><td rowspan=1 colspan=1>&lt;1XLD</td><td rowspan=1 colspan=1>1.2x 103</td><td rowspan=1 colspan=1>1/5</td><td rowspan=1 colspan=1>1/5</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=1 colspan=1>1XLD</td><td rowspan=1 colspan=1>26x 103</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>5XLD</td><td rowspan=1 colspan=1>1.3x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=2 colspan=1>Salmonellabongori(repeated)</td><td rowspan=2 colspan=1>43975</td><td rowspan=1 colspan=1>1XLD</td><td rowspan=1 colspan=1>23x 104</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>3/10</td></tr><tr><td rowspan=1 colspan=1>5XLD</td><td rowspan=1 colspan=1>1.1 x 105</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>4/5</td></tr></table>

a One replicate in this set correctly identified Salmonella, but additionally detected $C .$ . coli/jejuni. b One replicate in this set correctly identified Salmonella , but additionally detected $C .$ . coli/jejuni & Shiga toxin 1

Media Equivalency:   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>ATCCID</td><td rowspan=2 colspan=1>InputConc.(XLM)</td><td rowspan=2 colspan=1>CFUnL</td><td rowspan=1 colspan=3>Corredt SBPPResuts</td></tr><tr><td rowspan=1 colspan=1>T0= Fresh Pre-Freeze</td><td rowspan=1 colspan=1>T1= 1XFreeze/Thaw</td><td rowspan=1 colspan=1>T2 = 2XFreeze/Thaw</td></tr><tr><td rowspan=3 colspan=1>Campylobactercoli</td><td rowspan=3 colspan=1>43486</td><td rowspan=1 colspan=1>&lt;XLD</td><td rowspan=1 colspan=1>4.2x 102</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>1/5</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>1XLD</td><td rowspan=1 colspan=1>1.8x 103</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5XLOD</td><td rowspan=1 colspan=1>1.1 X104</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=3 colspan=1>Campylobacterjejuni</td><td rowspan=3 colspan=1>49943</td><td rowspan=1 colspan=1>&lt;XLM</td><td rowspan=1 colspan=1>20x 101</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>1XLD</td><td rowspan=1 colspan=1>24x 103</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>5XLD</td><td rowspan=1 colspan=1>1.2x 104</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=3 colspan=1>Campylobacterjejuni(repeated)</td><td rowspan=3 colspan=1>49943</td><td rowspan=1 colspan=1>&lt;1XLD</td><td rowspan=1 colspan=1>1.0x 102</td><td rowspan=1 colspan=1>5/8</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1XLD</td><td rowspan=1 colspan=1>1.5x 103</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>4/10</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5XLD</td><td rowspan=1 colspan=1>1.3x 104</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>1/5</td></tr></table>

\* This set of test runs also contained 1 "Invalid" Run

A Media Equivalency Study was conducted to demonstrate equivalent SBPP performance in five (5) prevalent stool preservation media types:

• Thermo Scientific™ Remel™ Cary Blair Transport Medium Meridian™ Para-Pak® Enteric Plus Transport Medium • Meridian™ Para-Pak® Culture & Sensitivity (C&S) Medium Thermo Scientific™ Protocol™ Cary Blair Medium C Thermo Scientific™ Protocol™ Culture & Sensitivity (C&S) Medium

To generate the unique stool matrices for each media type, raw clinical stool specimens, that previously tested negative for SBPP targets, were preserved in each preservation medium, listed above, per the Manufacturer’s instructions. The subsequent stool matrices were used as the base for contrived positive samples, and the clinical negative sample, for each media type.

The Media Equivalency Study was designed to demonstrate equivalent SBPP test performance in the 5 media types, by evaluating eight (8) strains for which LoD was initially measured. The 8 strains included:

C. coli (ATCC 43486)   
• C. jejuni (ATCC 49443)   
• E. coli (ATCC BAA-2196 stx1+/stx2+/non-O157)   
• E. coli (ATCC 43895 stx1+/stx2+/O157+) S. bongori (ATCC 43975)   
• S. enterica (ATCC 13311) S. flexneri (ATCC 25929)S. sonnei (ATCC 29930)

All stool transport media tested: Remel™ Cary Blair Transport Medium; Protocol™ Cary Blair Medium; Protocol™ Culture & Sensitivity (C&S) Medium; Para-Pak $^ \mathrm { \textregistered }$ Enteric Plus Transport Medium; and Para- Pak $^ \mathrm { \textregistered }$ Culture & Sensitivity (C&S) Medium, performed as expected, and each test medium type demonstrated equivalent performance to all other test media. At 5X LoD, for all strains tested, there was $100 \%$ agreement with the expected results in all five (5) media types. Likewise, at 2X LoD there was $2 9 5 \%$ agreement with the expected results for all strains tested across all five (5) media types. Also, as expected, the percent agreement for strains below LoD $_ { < 1 \mathrm { X } }$ LoD) varied from $10 \%$ to $90 \%$ across these five (5) media types. All the media types tested, therefore, are compatible with the SBPP Test.

Media Equivalency Study   

<table><tr><td rowspan=1 colspan=3>Results</td><td rowspan=1 colspan=5>Correct SBPP Results for Different Media Types</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>ATCCID</td><td rowspan=1 colspan=1>Sample InputConc. (X LoD)</td><td rowspan=1 colspan=1>Remel CaryProtocol CaryBlair</td><td rowspan=1 colspan=1>Remel CaryProtocol CaryBlair</td><td rowspan=1 colspan=1>ProtocolC&amp;S</td><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>Para-PakEnteric Plus</td></tr><tr><td rowspan=3 colspan=1>Campylobacter coli</td><td rowspan=3 colspan=1>43486</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>4/10</td><td rowspan=1 colspan=1>5/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30*</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>Campylobacter jejuni</td><td rowspan=3 colspan=1>49443</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>5/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>3/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10a</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>E. coli(stx1+/stx2+/non-0157)</td><td rowspan=3 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>8/10*</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>4/10</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>29/30*</td><td rowspan=1 colspan=1>29/30*</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/9*</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=3 colspan=1>E. coli(stx1+/stx2+/0157+)</td><td rowspan=3 colspan=1>43895</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>3/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>Salmonella bongori</td><td rowspan=3 colspan=1>43975</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>4/11</td><td rowspan=1 colspan=1>2/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>5/10</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>Salmonella enterica</td><td rowspan=3 colspan=1>13311</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>6/9</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>46/50</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>19/20*</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>Shigella flexneri</td><td rowspan=3 colspan=1>25929</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>4/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30c</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30d</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=3 colspan=1>Shigella sonnei</td><td rowspan=3 colspan=1>29930</td><td rowspan=1 colspan=1>&lt;1X</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>10/11b</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>8/10</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>28/29</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=1 colspan=1>5X</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Clinical Negative</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr></table>

\* This set of tests also contained 1 "Invalid" run a One replicate detected $C .$ jejuni correctly, but additionally detected Shiga toxin 2 (Protocol Cary Blair Media) b One replicate detected $S _ { \cdot }$ sonnei correctly, but additionally detected Shiga toxin 1, Shiga toxin2, and Serotype O1 (Protocol Cary Blair Media) c One replicate detected S. flexneri correctly, but additionally detected $C .$ coli/jejuni (Protocol C&S) d One replicate in this set correctly identified S. flexneri , but additionally detected $C .$ . coli/jejuni (Para-Pak Enteric

# d. Detection limit:

The limit of detection (LoD) of the SBPP was assessed by testing 10 bacterial strains which included all the target organisms detected in the SBPP. The LoDs for each strain tested is shown in the table below

# Limit of Detection by Analyte

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ATCC ID</td><td rowspan=1 colspan=1>SBPP TargetGene Present</td><td rowspan=1 colspan=1>LoD(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>43486</td><td rowspan=1 colspan=1>cadF</td><td rowspan=1 colspan=1>1.8 x 103</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>49943</td><td rowspan=1 colspan=1>cadF</td><td rowspan=1 colspan=1>1.3 x 103</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>BAA-2215</td><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>5.7 x 103</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>51435</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>4.4 x 103</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>stx1, stx2</td><td rowspan=1 colspan=1>1.1 x 104</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>43895</td><td rowspan=1 colspan=1>stx1, stx2, 0157</td><td rowspan=1 colspan=1>1.6 x 104</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>43975</td><td rowspan=1 colspan=1>invA</td><td rowspan=1 colspan=1>2.5 x 103</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>13311</td><td rowspan=1 colspan=1>invA</td><td rowspan=1 colspan=1>1.9 x 104</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>25929</td><td rowspan=1 colspan=1>ipaH</td><td rowspan=1 colspan=1>5.2 x 103</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>29930</td><td rowspan=1 colspan=1>ipaH</td><td rowspan=1 colspan=1>1.4 x 104</td></tr></table>

# Inclusivity/ Reactivity:

The analytical reactivity of the SBPP was assessed by testing an additional 91 well characterized bacterial strains representing the organisms detected in the SBPP. Organisms were diluted to a concentration of 2X LoD into pooled negative preserved clinical stool matrix. The additional organisms shown in the table below were detected in the SBBP.

<table><tr><td rowspan=1 colspan=1>Organism/Serotype</td><td rowspan=1 colspan=1>TotalNumber</td><td rowspan=1 colspan=1>Concentration 2XLoD (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.6 x 103</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni (subsp. jejuni, subsp. doylei)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.6 x 103</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica (subsp. enterica: Typhi, Newport, Choleraesuis,Stanley, Heidelber, Muenchen, Paratyphi B, Bareilly, Kentucky, SaintPaul, Tennessee, Paratyphi A, Typhimurium, Choleraesuis, arizonae,Typhimurium, Dublin, houtenae, Virchow, Agona, Bristol, Montevideo,Infantis, Mississippi, subsp. indica, subsp. salame, subsp. diarizonae,subsp. houtanae), Salmonella bongori</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>3.8 x 104</td></tr><tr><td rowspan=1 colspan=1>Shigella boydij (Serotypes 1, 1a, 2, 2a, 3, 5, 6, 8), Shigella sonnei,Shegella dysenteriae (Serotypes 1, 2, 3, 12, 13)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.8 x 104</td></tr><tr><td rowspan=1 colspan=3>Shiga-toxin producing E. coli</td></tr><tr><td rowspan=1 colspan=1>O11:H8, O26:H11, O21:H19, O111</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.2 X 104</td></tr><tr><td rowspan=1 colspan=1>0111, O26:H11, O45:H2, O45:H2, O123:H25, O103:H2, O103:H11</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4 X 104</td></tr><tr><td rowspan=1 colspan=1>091:H21, 0113:H21, O145:H28, O145:H25, O121:H19, O104:H4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>8.8 x 103</td></tr><tr><td rowspan=1 colspan=1>0157:H7, 0157:H7, 0157:NM, 0157:H7:K, 0157:H7, 0157:NM,0157:H7, 0157:NM</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.2 x 104</td></tr><tr><td rowspan=1 colspan=3>Shiga-toxin producing Shigella dysentariae</td></tr><tr><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.4 x 104</td></tr></table>

# e. Analytical specificity:

The potential for cross-reactivity was evaluated in an Exclusivity Study, by testing non-target organisms commonly found in stool. The study included 100 organisms phylogenetically related to targeted organisms as well as other bacteria, fungi/yeast, parasites, viruses, and human genomic DNA (84 bacterial strains, 3 yeast, 3 parasites, 9 viruses and human genomic DNA). For those isolates that were classified as Biosafety level III, or unable to be cultured via standard clinical microbiology techniques, genomic DNA was tested in place of whole organism.

Each non-target organism or nucleic acid was prepared in pooled, preserved, negative, clinical stool matrix. All bacterial and yeast strains were tested at concentrations $\geq 1 . 0 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ . Genomic DNA (gDNA) templates, viral strains, and parasites, were tested at $\geq 1 ~ \mathrm { u g / m L }$ , ${ \geq } 1 \mathrm { x } 1 0 ^ { 6 }$ copies/mL, or ${ \ge } 1 \mathrm { \dot { x } } 1 0 ^ { 5 }$ T $\mathrm { \ C I D } _ { 5 0 } / \mathrm { m L }$ , respectively.

All but one of the organisms shown gave the expected “Not Detected” result indicating that there was no cross-reactivity in the SBPP. The one exception was Enteroinvasive Escherichia coli (EIEC) which was “Detected” in the SBPP. This cross-reactivity was expected since the gene target used to detect Shigella spp. (ipaH) is also present EIEC.

# Organisms Tested for Exclusivity

<table><tr><td colspan="4" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Genus</td><td colspan="1" rowspan="1">Species</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia</td><td colspan="1" rowspan="1">defectiva</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">penneri</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">baumannii</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">vulgaris</td></tr><tr><td colspan="1" rowspan="1">Aeromonas</td><td colspan="1" rowspan="1">hydrophila</td><td colspan="1" rowspan="1">Providencia</td><td colspan="1" rowspan="1">alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus</td><td colspan="1" rowspan="1">tetradius</td><td colspan="1" rowspan="1">Providencia</td><td colspan="1" rowspan="1">rettgeri</td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">cereus</td><td colspan="1" rowspan="1">Providencia</td><td colspan="1" rowspan="1">stuartii</td></tr><tr><td colspan="1" rowspan="1">Bacteriodes</td><td colspan="1" rowspan="1">fragilis</td><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Bacteriodes</td><td colspan="1" rowspan="1">vulgatus</td><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">putida</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium</td><td colspan="1" rowspan="1">adolescentis</td><td colspan="1" rowspan="1">Ruminococcus</td><td colspan="1" rowspan="1">bromii</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium</td><td colspan="1" rowspan="1">bifidum</td><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium</td><td colspan="1" rowspan="1">longum</td><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">marcescens</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">curvus (gDNA)</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">aureus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">fetus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">epidermidis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">fetus subsp.venerealis</td><td colspan="1" rowspan="1">Stenotrophomonas</td><td colspan="1" rowspan="1">maltophilia</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">hyointestinalis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">agalactiae</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">lari (5 strains)</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">dysgalactiae</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">upsaliensis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">intermedius</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">amalonaticus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">pyogenes</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">freundii</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">uberis</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">difficile</td><td colspan="1" rowspan="1">Trabulsiella</td><td colspan="1" rowspan="1">guamensis</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">histolyticum</td><td colspan="1" rowspan="1">Veillonella</td><td colspan="1" rowspan="1">parvula</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">perfringens</td><td colspan="1" rowspan="1">Vibrio</td><td colspan="1" rowspan="1">cholera</td></tr><tr><td colspan="1" rowspan="1">Clostridium</td><td colspan="1" rowspan="1">sordellii</td><td colspan="1" rowspan="1">Vibrio</td><td colspan="1" rowspan="1">parahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">aerogenes</td><td colspan="1" rowspan="1">Vibrio</td><td colspan="1" rowspan="1">vulnificus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">cloacae</td><td colspan="1" rowspan="1">Yersinia</td><td colspan="1" rowspan="1">bercovieri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">cecorum</td><td colspan="1" rowspan="1">Yersinia</td><td colspan="1" rowspan="1">enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">faecalis</td><td colspan="1" rowspan="1">Yersinia</td><td colspan="1" rowspan="1">pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">faecium</td><td colspan="1" rowspan="1">Yersinia</td><td colspan="1" rowspan="1">rohdei</td></tr><tr><td colspan="1" rowspan="1">EAEC Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="2" rowspan="1">Fungi</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="2" rowspan="1">Candida albicans</td></tr><tr><td colspan="1" rowspan="1">EIEC Escherichia</td><td colspan="1" rowspan="1">coli (3 strains)</td><td colspan="2" rowspan="1">Candida catenulata</td></tr><tr><td colspan="1" rowspan="1">ETEC Escherichia</td><td colspan="1" rowspan="1">coli</td><td colspan="2" rowspan="1">Saccharomyces boulardii</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="1" rowspan="1">fergusonii</td><td colspan="2" rowspan="1">Viruses and Parasites</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="1" rowspan="1">hermannii</td><td colspan="2" rowspan="1">Adenovirus Type 2 (gDNA)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium</td><td colspan="1" rowspan="1">varium</td><td colspan="2" rowspan="1">Adenovirus type 40, strain Dugan (gDNA)</td></tr><tr><td colspan="1" rowspan="1">Gardnerella</td><td colspan="1" rowspan="1">vaginalis</td><td colspan="2" rowspan="1">Adenovirus type 41, strain Tak</td></tr><tr><td colspan="1" rowspan="1">Helicobacter</td><td colspan="1" rowspan="1">fennelliae</td><td colspan="2" rowspan="1">Coxsackie B4</td></tr><tr><td colspan="1" rowspan="1">Helicobacter</td><td colspan="1" rowspan="1">pylori</td><td colspan="2" rowspan="1">Cryptosporidium parvum (gDNA)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">pneumoniae</td><td colspan="2" rowspan="1">Entamoeba histolytica (gDNA)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella</td><td colspan="1" rowspan="1">oxytoca</td><td colspan="2" rowspan="1">Enterovirus (RNA)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">acidophilus</td><td colspan="2" rowspan="1">Giardia intestinalis (gDNA)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">casei</td><td colspan="2" rowspan="1">Norovirus GI (synthetic RNA)</td></tr><tr><td colspan="1" rowspan="1">Leminorella</td><td colspan="1" rowspan="1">grimonti</td><td colspan="2" rowspan="1">Norovirus GII (synthetic RNA)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">grayi</td><td colspan="2" rowspan="1">Rotavirus</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">innocua</td><td colspan="2" rowspan="1">Rotavirus A (RNA)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">monocytogenes</td><td colspan="2" rowspan="1">Human genomic DNA</td></tr><tr><td colspan="1" rowspan="1">Morganella</td><td colspan="1" rowspan="1">morganii</td><td colspan="2" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus</td><td colspan="1" rowspan="1">anaerobius</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas</td><td colspan="1" rowspan="1">shigelloides</td><td colspan="2" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Porphyromonas</td><td colspan="1" rowspan="1">asaccharolytica</td></tr><tr><td colspan="1" rowspan="1">Prevotella</td><td colspan="1" rowspan="1">melaninogenicus</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">mirabilis</td></tr></table>

# Competitive Inhibition:

To evaluate potential for competitive interference in the SBPP, combinations of the 8 SBPP target organisms representative of potential dual infections, were tested. The organisms included: $C .$ . coli (ATCC 43486), $C .$ . jejuni (ATCC 49943), $E$ . coli (ATCC BAA- $2 1 9 6 \ s t x I + / s t x 2 + )$ ), $E .$ . coli (ATCC 4 $3 8 9 5 \ s t x l + / s t x 2 + / \mathrm { O } 1 5 7 +$ ), S. bongori (ATCC 43975), S. enterica (ATCC 13311), S. flexneri (ATCC 25929), and S. sonnei (ATCC 29930). The panels were designed such that one organism of each bacterial species was present at a low titer (2X LoD) with a second organism present at a high titer $\mathrm { \dot { ( > 1 0 ^ { 6 } C F U / m L ) } }$ ). The combinations and concentrations tested along with the study results are shown in the table below.

Competitive inhibition was observed for Salmonella when $E$ . coli $( s t x I + / s t x 2 + / \mathrm { O } 1 5 7 + )$ was present at concentrations $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ .

<table><tr><td rowspan=2 colspan=1>Organism at LowTiter(2X) LoD</td><td rowspan=1 colspan=8>Organisms at High Titer: ≥106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>C. coli(ATCC43486)</td><td rowspan=1 colspan=1>C.jejuni(ATCC49943)</td><td rowspan=1 colspan=1>E. coli(stx1+/stx2+/non-0157)(ATCCBAA-2196)</td><td rowspan=1 colspan=1>E. coli(stx1+/stx2+/0157+)(ATCC43895)</td><td rowspan=1 colspan=1>S.bongori(ATCC43975)</td><td rowspan=1 colspan=1>S.enterica(ATCC13311)</td><td rowspan=1 colspan=1>S.flexneri(ATCC25929)</td><td rowspan=1 colspan=1>S.sonnei(ATCC29930)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli(ATCC 43486)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni(ATCC 49943)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/0157-)(ATCC BAA-2196)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(stx1+/stx2+/0157+)(ATCC 43895)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori(ATCC 43975)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/6a</td></tr><tr><td rowspan=2 colspan=1>Salmonella enterica(ATCC 13311)</td><td rowspan=2 colspan=1>7/9b</td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td><td rowspan=1 colspan=1>14/19c</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>3/3</td><td rowspan=2 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>6/6d</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri(ATCC 25929)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei (ATCC29930)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/6e</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

a In 1/3 replicates, 'high titer' Shigella sonnei was not detected and contamination with a Campylobacter sp. was noted. An additional 3 replicates were tested and the expected result was obtained for both analytes, in all replicates.

b For a 'low titer' Salmonella enterica and 'high titer' Campylobacter coli sample, the SBPP did not detect Salmonella in 2/3 replicates, although Campylobacter was correctly identified in all cases. An additional 6 replicates were tested, and the expected result was obtained for both analytes, in all replicates.

c For a 'low titer' Salmonella enterica and 'high titer' Escherichia coli (ATCC 43895, $\dot { \geq } 1 0 ^ { 6 } \mathrm { C F U / m L }$ ) sample, the SBPP did not detect Salmonella in 1/3 replicates. An additional 16 replicates were tested and 12/16 detected 'low titer' Salmonella.

d We decreased the concentration of the 'high titer' $E .$ . coli to $1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ in combination with 'low titer' Salmonella and tested 6 replicates. The expected result was obtained for both analytes, in all replicates.

$^ { \mathrm { ~ e ~ } } \mathrm { I n } \ 1 / 3$ replicates, 'low titer' Shigella sonnei was not detected, although Shiga Toxin 1 & 2 was detected in all cases. An additional 3 replicates were tested, and the expected result obtained for both analytes, in all replicates.

# Non-microbial Interference:

Potential interference in the SBPP from 19 different substances that are common stool contaminants, or likely to be present in patients with diarrhea were evaluated in an Interfering Substances Study. Each substance was added to a positive stool prepared by adding a single SBPP target organism to pooled, negative, preserved clinical stool at $\leq 3 \mathrm { X }$ LoD. The organisms tested represented each analyte detected by the SBPP and included: $C .$ . coli (ATCC 43486), $C .$ . jejuni (ATCC 49943), $E$ . coli (ATCC BAA-2196 stx1+/stx2+), $E$ . coli (ATCC 43895

$s t x I + / s t x 2 + / \mathrm { O } 1 5 7 + \AA ^ { , }$ ), S. bongori (ATCC 43975), S. enterica (ATCC 13311), S.   
flexneri (ATCC 25929), and S. sonnei (ATCC 29930).

A clinical, negative, un-spiked stool matrix was also tested as a control to evaluate potential interference with the internal assay control in the absence of analyte. The list of potentially interfering substances along with the concentrations tested is shown below. No interference in the SBPP was observed for the substances tested shown in the table below at the concentrations listed.

# Potential Intefereins Substances Tested

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>Ampicillin</td><td rowspan=1 colspan=1>50 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Bacitracin Zinc ointment</td><td rowspan=1 colspan=1>50 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Benzalkonium chloride, ethanol (moist towelettes)</td><td rowspan=1 colspan=1>9.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Bovine Mucin</td><td rowspan=1 colspan=1>6.25 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Calcium carbonate</td><td rowspan=1 colspan=1>200 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>50% v/v</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>75 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Imodium</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Kaopectate</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Milk of Magnesia</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>50% v/v</td></tr><tr><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>9.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Pork Mucin</td><td rowspan=1 colspan=1>6.25 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>9.7 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>10% v/v</td></tr></table>

# Microbial Interference:

The potential for cross-reactivity in a mixed infection was evaluated in a Microbial Interference Study. A panel of non-target gastrointestinal pathogens commonly encountered in stool was tested in the presence of each of the analytes detected in the SBPP.

Potentially interfering bacteria/fungi, viruses, and DNA were added at ${ \geq } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ , ${ \geq } 1$ x $1 0 ^ { 6 }$ copies $ { \mathrm { / m L } }$ , $\mathrm { a n d } \geq 1 ~ \mu \mathrm { g / m L }$ , respectively, to pooled, negative, preserved, clinical stool with a single SBPP target analyte added at $\leq 3 \mathrm { X }$ LoD. In total, 21 unique bacterial strains, 2 yeast, 2 parasites, 3 viruses and human genomic DNA were tested for microbial interference with the 8 SBPP target strains.

No interference from non-target organisms was observed in the SBPP for any of the samples tested.

# Organisms Tested for Microbial Interference

<table><tr><td rowspan=1 colspan=2>Bacteria</td><td rowspan=1 colspan=1>Yeasts, Parasites, Viruses andDNA</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophilia</td><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Saccharomyces boulardii</td></tr><tr><td rowspan=1 colspan=1>Bacteroides vulgatus</td><td rowspan=1 colspan=1>Klebsiella pneumonia</td><td rowspan=1 colspan=1>Entamoeba histolytica (gDNA)</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium bifidum</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Giardia intestinalis (gDNA)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (toxinA/B)</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Enterovirus 71 (RNA)</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>Norovirus G1 (synthetic RNA)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Rotavirus (RNA)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Human genomic DNA</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (non-STEC O157)</td><td rowspan=1 colspan=1>Vibrio cholera</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Carryover/Cross-Contamination:

To evaluate potential carry-over/cross-contamination of the SBPP a Carry-over Study was conducted. Contrived stool samples containing a high concentration of an analyte were alternated with clinical negative stool samples in 10 consecutive testing rounds. The high positive sample was generated by adding previously frozen and quantified enriched broth cultures into pooled negative, preserved, clinical stool at a final concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ . The organisms used in this study were C. coli (ATCC 43486), $E .$ . coli (ATCC 43895 $\mathrm { s } t x I + / s t x 2 + / \mathrm { O } 1 5 7 + )$ , S. bongori (ATCC 43975), and S. flexneri (ATCC 25929). The negative sample was stool from symptomatic patients that previously tested negative for all SBPP targets.

The alternating pattern of 10 test rounds of high positive and negative samples were performed in direct succession on 4 different analyzers. In total, 80 SBPP tests were performed: 40 tests of a high positive sample and 40 tests of a negative sample. No carry-over or cross contamination was observed in the SBPP.

f. Assay cut-off: N/A

2. Comparison studies:

a. Method comparison with predicate device: N/A b. Matrix comparison: See M1c.3.

3. Clinical studies:

a. Clinical Sensitivity:

The clinical evaluation of the SBPP consisted of a prospective and a frozen retrospective method comparison study. An additional study using selected fresh positive Salmonella samples was also performed.

The prospective method comparison study was conducted to compare the performance of the SBPP to standard stool culture-based methods for identification of Campylobacter jejuni /Campylobacter coli, Escherichia coli serotype O157 Salmonella spp., Shiga toxin 1 Shiga toxin 2, and Shigella spp. The study was conducted at four external, geographicallydiverse U.S. clinical study sites (Midwest, Northeast, Southwest and West) from July, 2016 through November, 2016. The specimens enrolled in the study were excess remnants of preserved stool samples collected from symptomatic individuals suspected of gastrointestinal infection that were processed according to routine standard care. A total of 1506 samples were collected for all four sites combined. Subsequent to enrollment, 24 samples were excluded from the data set leaving 1479 samples included in the analysis.

In addition, frozen archived de-identified specimens that were previously characterized as positive or negative by the standard of care method used at the institution (historical result) were obtained. The historical result for each sample was first confirmed by an FDA cleared Nucleic Acid Amplification Test (NAAT) prior to enrolling the sample in the study. A total of 150 frozen samples were included in the panel. The SBPP results were compared to the historical result.

To further increase the number of positive Salmonella samples evaluated, additional fresh samples selected as positive by the standard of care method used by that site were collected and tested.

The positive (PPA) and negative (NPA) percent agreement for each of the three studies (Prospective Study - all sites combined, Frozen Retrospective Sample study and Selected Salmonella study) along with the $9 5 \%$ Confidence Intervals are shown in table below.

<table><tr><td rowspan=2 colspan=2>Specimen</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>% Agreement (95% Cl)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1479</td><td rowspan=1 colspan=1>96.4% (82.3-99.4)27/281</td><td rowspan=1 colspan=1>99.2% (98.6-99.5)1439/1451²</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=3 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1479</td><td rowspan=1 colspan=1>83.3% (55.2-95.3)10/12³</td><td rowspan=1 colspan=1>99.6% (99.1-99.8)1461/14674</td></tr><tr><td rowspan=1 colspan=1>Fresh Selected</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>92.9% (77.4-98.0)26/28</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>94.4% (81.9-98.5)34/36</td><td rowspan=1 colspan=1>100.0% (97.8-100.0)170/170</td></tr><tr><td rowspan=2 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1479</td><td rowspan=1 colspan=1>100.0% (20.7-100.0)1/1</td><td rowspan=1 colspan=1>99.5% (99.0-99.8)1471/14785</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>100.0% (88.3-100.0)29/29</td><td rowspan=1 colspan=1>100.0% (97.9-100.0)177/177</td></tr><tr><td rowspan=2 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1479</td><td rowspan=1 colspan=1>100.0% (20.7-100.0)1/1</td><td rowspan=1 colspan=1>99.4% (98.8-99.7)1469/14786</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>100.0% (89.0-100.0)31/31</td><td rowspan=1 colspan=1>100.0% (97.9-100.0)175/175</td></tr><tr><td rowspan=2 colspan=1>E. coli Serotype 0157</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100% (51.0-100.0)4/4</td><td rowspan=1 colspan=1>75.0% (46.8-91.1)9/127</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100.0% (81.6 -100.0)17/17</td><td rowspan=1 colspan=1>100.0% (89.0-100.0)31/31</td></tr><tr><td rowspan=2 colspan=1>Shigella</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1479</td><td rowspan=1 colspan=1>100% (56.6-100.0)5/5</td><td rowspan=1 colspan=1>99.1% (98.4-99.4)1460/14748</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>94.7% (75.4-99.1)18/19</td><td rowspan=1 colspan=1>100.0% (98.0-100.0)187/187</td></tr></table>

1 Campylobacter DNA sequence was detected in 0/1false negative specimens when tested with an alternate,   
FDA-cleared NAAT   
2 Campylobacter DNA sequence was detected in 10/12 false positives specimens when tested with an   
alternate, FDA-cleared NAAT   
3 Salmonella DNA sequence was detected in $2 / 2$ false negative specimens when tested with an alternate,   
FDA-cleared NAAT   
4 Salmonella DNA sequence was detected in $^ { 6 / 6 }$ false positives specimens tested with an alternate, FDA  
cleared NAAT   
5 Stx1 DNA sequence was detected in $6 / 7$ false positive specimens when tested with an alternate, FDA  
cleared NAAT   
6 Stx2 DNA sequence was detected in 8/9 false positive specimens when tested with an alternate, FDA  
cleared NAAT   
$^ 7 ~ E .$ . coli O157 DNA sequence was detected in $3 / 3$ false positive specimens when tested with an alternate,   
FDA-cleared NAAT   
8 Shigella DNA sequence was detected in 14/14 false positive specimens when tested with an alternate,   
FDA-cleared NAAT

# Detection of Multiple Target Analytes

The tables below identify samples from the Prospective Study for which multiple target analytes were detected either by SBPP or the Reference Test. Each table also shows the analytes that were discrepant for a given sample.

# Samples with Multiple Analytes as Determined by SBPP

<table><tr><td rowspan=2 colspan=1>Sample ID</td><td rowspan=1 colspan=2>Multiple Target Analytes as Determined by SBPP</td><td rowspan=2 colspan=1>Discrepant Results(Analytes not Detected byReference Test)</td></tr><tr><td rowspan=1 colspan=1>SBPPDetected Analytes</td><td rowspan=1 colspan=1>Reference Test DetectedAnalytes</td></tr><tr><td rowspan=1 colspan=1>MCW-113</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>E. coli Serotype O157</td><td rowspan=1 colspan=1>Stx 2</td></tr><tr><td rowspan=1 colspan=1>LACNY-272</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>E. coli Serotype O157</td><td rowspan=1 colspan=1>Stx 2</td></tr><tr><td rowspan=1 colspan=1>LACNY-349</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>E. coli Serotype O157</td><td rowspan=1 colspan=1>Stx 2</td></tr><tr><td rowspan=1 colspan=1>LACNY-467</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>No Analytes Detected</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td></tr><tr><td rowspan=1 colspan=1>TRCL-416</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>No Discrepant Results</td></tr><tr><td rowspan=1 colspan=1>TRCL-177</td><td rowspan=1 colspan=1>Stx2Salmonella</td><td rowspan=1 colspan=1>No Analytes Detected</td><td rowspan=1 colspan=1>Stx2Salmonella</td></tr><tr><td rowspan=1 colspan=1>CHLA-122</td><td rowspan=1 colspan=1>Stx1E. coli Serotype O157</td><td rowspan=1 colspan=1>No Analytes Detected</td><td rowspan=1 colspan=1>Stx1E. coli Serotype O157</td></tr><tr><td rowspan=1 colspan=1>CHLA-43</td><td rowspan=1 colspan=1>Stx1Shigella</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Stx1</td></tr><tr><td rowspan=1 colspan=1>MCW-142</td><td rowspan=1 colspan=1>Stx1E. coli Serotype O157Shigella</td><td rowspan=1 colspan=1>No Analytes Detected</td><td rowspan=1 colspan=1>Stx1E. coli Serotype O157Shigella</td></tr><tr><td rowspan=1 colspan=1>CHLA-4</td><td rowspan=1 colspan=1>Stx1Stx2</td><td rowspan=1 colspan=1>No Analytes Detected</td><td rowspan=1 colspan=1>Stx1Stx2</td></tr><tr><td rowspan=1 colspan=1>CHLA-143</td><td rowspan=1 colspan=1>Stx1Stx2</td><td rowspan=1 colspan=1>Stx1</td><td rowspan=1 colspan=1>Stx2</td></tr></table>

# Samples with Multiple Analytes as Determined by Reference Test

<table><tr><td rowspan=2 colspan=1>Sample ID</td><td rowspan=1 colspan=2>Multiple Target Analytes as Determined by theReference Test</td><td rowspan=2 colspan=1>Discrepant Results(Analytes not Detected bySBPP)</td></tr><tr><td rowspan=1 colspan=1>Reference Test DetectedAnalytes</td><td rowspan=1 colspan=1>SBPPDetected Analytes</td></tr><tr><td rowspan=1 colspan=1>TRCL-416</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>Stx2E. coli Serotype O157</td><td rowspan=1 colspan=1>No Discrepant Results</td></tr></table>

b. Clinical specificity: See M3a.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

In the SBPP clinical study, 1479 prospectively collected fresh specimens which were obtained from four geographically distributed U.S. healthcare institutions (Midwest, Northeast, Southwest, West) were included in the method comparison study. The number and percentage of positive SBPP results for each analyte (overall prevalence rate) are shown in below. The prevalence rate obtained in routine use may vary depending on the institution, geographic location and patient population.

Prevalence Rates of Analytes Detected by SBPP Observed in Clinical Studies   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>Shigella</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.3% (8/348)</td><td rowspan=1 colspan=1>0.3% (1/348)</td><td rowspan=1 colspan=1>0.6% (2/348)</td><td rowspan=1 colspan=1>0.9% (3/348)</td><td rowspan=1 colspan=1>0.6% (2/348)</td><td rowspan=1 colspan=1>2.0% (7/348)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.8% (19/505)</td><td rowspan=1 colspan=1>0.1% (5/505)</td><td rowspan=1 colspan=1>0.4% (2/505)</td><td rowspan=1 colspan=1>0.6% (3/505)</td><td rowspan=1 colspan=1>0.6% (3/505)</td><td rowspan=1 colspan=1>0.4% (2/505)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.7% (8/465)</td><td rowspan=1 colspan=1>1.5% (7/465)</td><td rowspan=1 colspan=1>0.0% (0/465)</td><td rowspan=1 colspan=1>0.4% (2/465)</td><td rowspan=1 colspan=1>0.2% (1/465)</td><td rowspan=1 colspan=1>0.9% (4/465)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.5% (4/161)</td><td rowspan=1 colspan=1>1.9% (3/161)</td><td rowspan=1 colspan=1>2.5% (4/161)</td><td rowspan=1 colspan=1>1.2% (2/161)</td><td rowspan=1 colspan=1>0.6% (1/161)</td><td rowspan=1 colspan=1>3.7% (6/161)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2.6%(39/1479)</td><td rowspan=1 colspan=1>1.1%(16/1479)</td><td rowspan=1 colspan=1>0.5%(8/1479)</td><td rowspan=1 colspan=1>0.7%(10/1479)</td><td rowspan=1 colspan=1>0.5%(7/1479)</td><td rowspan=1 colspan=1>1.3%(19/1479)</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.