# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K163636   
B. Purpose for Submission: New device

# C. Measurand:

Adenovirus, Coronavirus, Human Metapneumovirus (hMPV), Human Rhinovirus/Enterovirus (HRV/HEV), Influenza A, Influenza A H1 (Flu A/H1), Influenza A H1-2009 (Flu A/H1-2009), Influenza A H3 (Flu A/H3), Influenza B (Flu B), Parainfluenza virus 1 (PIV 1), Parainfluenza virus 2 (PIV 2), Parainfluenza virus 3 (PIV 3), Parainfluenza virus 4 (PIV 4), Respiratory Syncytial Virus A (RSV A), Respiratory Syncytial Virus B (RSV B), Chlamydia pneumoniae, and Mycoplasma pneumoniae nucleic acids target sequences.

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the GenMark ePlex instrument for the simultaneous qualitative in vitro detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) collected in viral transport media and obtained from individuals suspected of respiratory tract infections.

E. Applicant: GenMark Diagnostics, Incorporated

F. Proprietary and Established Names: ePlex® Respiratory Pathogen (RP) Panel Common Name: ePlex $\mathrm { R P } ^ { \mathbb { B } }$ Panel

G. Regulatory Information:

<table><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Regulation Section</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OTG</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZE</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZX</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZY</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OQW</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3332Reagents for Detectionof Specific NovelInfluenza A Viruses</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation forClinical Multiplex TestSystems</td><td colspan="1" rowspan="1">Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The ePlex® Respiratory Pathogen (RP) Panel is a multiplexed nucleic acid in vitro diagnostic test intended for use on the ePlex® Instrument for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals exhibiting signs and symptoms of respiratory tract infection.

The following virus types, subtypes, and bacteria are identified using the ePlex® RP Panel: adenovirus, coronavirus, human metapneumovirus, human rhinovirus/enterovirus, influenza A, influenza A H1, influenza A H1-2009, influenza A H3, influenza B, parainfluenza virus 1, parainfluenza virus 2, parainfluenza virus 3, parainfluenza virus 4, respiratory syncytial virus (RSV) A, respiratory syncytial virus (RSV) B, Chlamydia pneumoniae, and Mycoplasma pneumoniae.

The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory tract infection aids in the diagnosis of respiratory infection when used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the organism(s) detected by the ePlex® RP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the genetic similarity between human rhinovirus and enterovirus, the ePlex® RP Panel cannot reliably differentiate them. If differentiation is required, an ePlex® RP Panel positive human rhinovirus/enterovirus result should be followed-up using an alternative method (e.g., cell culture or sequence analysis).

Performance characteristics for influenza A were established when influenza A H1- 2009 and A H3 were the predominant influenza A viruses in circulation. Performance of detecting influenza A may vary if other influenza A strains are circulating or a novel influenza A virus emerges. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL- $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: ePlex instrument

# I. Device Description:

The ePlex Respiratory Pathogen (RP) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solidphase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RTPCR), and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological specimens occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, singlestranded capture probe immobilized on the working gold electrode surface. Singlestranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

A summary of the ePlex RP Panel nucleic acid targets is presented in Table 1 below.

Table 1: A summary of the ePlex RP Targets   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target Gene(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Hexon protein (L3) and Penton protein (L2)</td></tr><tr><td rowspan=1 colspan=1>Coronaviruses (229E, HKU1, NL63, and OC43)</td><td rowspan=1 colspan=1>Nucleoprotein (N)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Nucleocapsid (N)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Matrix protein (M)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>Hemagglutinin (HA)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RNA polymerase subunit PB1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Hemagglutinin-neuraminidase (HN)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Hemagglutinin-neuraminidase (HN)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Hemagglutinin-neuraminidase (HN)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Hemagglutinin-neuraminidase (HN)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>5&#x27;- Untranslated Region (UTR)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>Nucleocapsid (N)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>Nucleocapsid (N)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Major Outer Membrane Protein A (OmpA)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumonia</td><td rowspan=1 colspan=1>Cytadhesin (P1)</td></tr></table>

# Materials Provided:

Each ePlex Respiratory Pathogen Panel kit contains sufficient reagents to test 12 samples (EA001012):

· ePlex Respiratory Pathogen Panel Cartridge (12) · Sample Delivery Device – RP Panel; $2 0 0 \mu \mathrm { l }$ (12)

# Materials Needed but Not Provided:

Equipment

GenMark ePlex instrument and software Pipettes calibrated to deliver $2 0 0 \mu \mathrm { l }$ · Vortex mixer Printer (Optional)

Consumables

Pipette tips, aerosol resistant, RNase/DNase-free   
Disposable, powder free gloves   
$10 \%$ bleach for decontamination of appropriate surfaces   
$70 \%$ ethanol or isopropyl alcohol

# Interpretation of Results

Results interpretation of the ePlex Respiratory Pathogen Panel is performed by the ePlex instrument, and the interpretation of results on the ePlex RP Panel Detection Report for each targeted analyte is summarized in Table 2 below.

Table 2: Interpretation of Results on the ePlex RP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Target Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Target Detected</td><td rowspan=1 colspan=1>The test was completed successfully, and thetarget has generated signal above its definedthreshold, and the Internal Control was reported asPASS.</td><td rowspan=1 colspan=1>All results are displayed on theRP Panel Detection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>Multiple TargetsDetected</td><td rowspan=1 colspan=1>The test was completed successfully, and multipletargets have generated signals above its definedthreshold, and the Internal Control was reported asPASS.</td><td rowspan=1 colspan=1>All results are displayed on theRP Panel Detection Report.Test is valid, report results.Detection of more than 3pathogens may indicatecontamination. Re-test of thesample is recommended toconfirm results.</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>The test was completed successfully, and thetarget did not generate signal above its definedthreshold, and the Internal Control was reported asPASS.</td><td rowspan=1 colspan=1>All results are displayed on theRP Panel Detection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The test has not successfully completed, andresults for this test are not valid. This is often dueto an instrument or software error or failure of aninternal control.</td><td rowspan=1 colspan=1>No results are displayed on theRP Panel Detection Report.Test is not valid, repeat test.</td></tr></table>

The ePlex RP Panel detects Influenza A and the A/H1, A/H1-2009, and A/H3 subtypes using unique assays for each. Interpretation of results for Influenza A are described in Table 3 below.

Table 3: Possible Assay Results for Influenza A and the Corresponding Interpretation   

<table><tr><td colspan="1" rowspan="1">Results forInfluenza A andSubtypes</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Results on Report</td><td colspan="1" rowspan="1">Recommended Action</td></tr><tr><td colspan="1" rowspan="1">Influenza A detected,at least one subtype(A/H1, A/H1-2009,or A/H3) reported asdetected.</td><td colspan="1" rowspan="1">This is an expectedresult.</td><td colspan="1" rowspan="1">Result reported asinfluenza A andinfluenza A subtypedetected.</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A detected,all subtypes (A/H1,A/H1-2009, andA/H3) reported as notdetected</td><td colspan="1" rowspan="1">Low virus titers canresult in detection ofinfluenza A matrixwithout a subtype.Detection of influenza Amatrix without a subtypecan also indicate thepresence of a novelstrain.</td><td colspan="1" rowspan="1">Result reported asinfluenza Adetected (nosubtype detected)</td><td colspan="1" rowspan="1">Re-test to confirm result.If the re-test provides adifferent result, test thesample a third time to ensurethe accuracy of the result. Ifthe re-test provides the sameresult, then the function of theePlex RP Panel cartridgeshould be verified by testingwith appropriate externalpositive control materials(known positive samples forinfluenza A H1, influenza AH3 and influenza A H1-2009), and a negative controlshould also be run to test forpotential PCR-productcontamination. If the ePlexRP Panel accurately identifiesthe external and negativecontrols, contact theappropriate public healthauthorities for confirmatorytesting.</td></tr><tr><td colspan="1" rowspan="1">Influenza A detectedand more than onesubtype (A/H1,A/H1-2009, or A/H3)reported as detected.</td><td colspan="1" rowspan="1">Sample is co-infectedwith multiple influenzasubtypes.Infections with multiplesubtypes of influenza arepossible but rare.A live intranasalmultivalent influenzavirus vaccine may cause</td><td colspan="1" rowspan="1">Result reported asinfluenza A andmultiple subtypesdetected.</td><td colspan="1" rowspan="1">Re-test to confirm result.If the re-test result confirmsthe original result, it isrecommended that the samplebe further investigated using adifferent FDA-clearedinfluenza A subtyping assay.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">false positive results forinfluenza A, A/H1,A/H3, A/H1-2009, and/orinfluenza B.Contamination hasoccurred.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A notdetected, at least onesubtype (A/H1,A/H1-2009, or A/H3)reported as detected.</td><td colspan="1" rowspan="1">Low virus titers canresult in detection ofinfluenza A subtypewithout the influenza Amatrix.Detection of influenza Asubtype without theinfluenza A matrix canalso indicate the presenceof a novel strain.</td><td colspan="1" rowspan="1">Influenza A(subtype) detected.Re-testing of thissample to confirminfluenza A(subtype) isrecommended.Refer to packageinsert for additionalinformation.</td><td colspan="1" rowspan="1">Re-test to confirm result.If the re-test result confirmsthe original result, theinfluenza A subtype isconsidered positive.It is recommended that thesample be further investigatedusing a different FDA-clearedinfluenza A subtyping assayand/or sending the residualsample to local public healthlaboratory for further testing.</td></tr></table>

# ePlex RP Panel Test Reports

There are several different reports that are available on the ePlex instrument. Results are provided in a printable format, may be viewed electronically, or may be exported for additional analysis. Reports can be customized with account specific information such as the address, logo, and institution specific footers on each report.

# Detection Report

The ePlex RP Panel Detection Report includes the results for each individual sample run on the ePlex instrument. The Summary section indicates the overall test result and lists all detected analytes in that sample. The Results section includes a list of all analytes on the panel with an individual result for each. Results for each analyte are reported as Detected, Not Detected, or Invalid (displayed as a red x); results for the Internal Control are reported as PASS, FAIL, INVALID, or N/A.

# External Control Report

The ePlex RP Panel External Control Report is generated for an external control that has been pre-defined in the ePlex RP Panel software. The Summary section indicates the overall result (Pass or Fail status) and lists all detected analytes for that external control. The Results section includes a list of all panel analytes with the result, expected result, and Pass/Fail status for each. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x). An analyte is reported as Pass if the actual result matches the expected result (as defined for that control); an analyte is reported as Fail if the actual result does not match the expected result. If the actual results for each analyte match the expected results (all analytes reported as Pass), the overall result for the external control is reported as Pass in the Summary section. If the actual result for any analyte does not match the expected result, the overall result for the external control is reported as Fail in the Summary section.

# Summary Report

The Summary Report allows the operator to use defined searchable criteria to create customized reports, using specified analytes, dates, range of dates, sample, external control, test bay, or operator.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray® Respiratory Panel (RP)

2. Predicate K number(s): K160068

3. Comparison with predicate(s):

<table><tr><td colspan="3" rowspan="1">Similarities or Differences</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">ePlex Respiratory Pathogen (RP)Panel (K163636)</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel (RP)(K160068)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">NPS (in transport media)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Adenovirus, Coronavirus, HumanMetapneumovirus, Influenza A,Influenza A H1, Influenza A H3,Influenza A H1-2009, Influenza B,Parainfluenza Virus 1, ParainfluenzaVirus 2, Parainfluenza Virus 3,Parainfluenza Virus 4, HumanRhinovirus/Enterovirus, RespiratorySyncytial Virus A, , RespiratorySyncytial Virus B, Chlamydiapneumoniae, and Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Same, except that it also detects anddifferentiates Coronavirus 229E,Coronavirus HKU1, CoronavirusNL63, and Coronavirus OC43; it alsodetects Bordetella pertussis; it does notdifferentiate Respiratory SyncytialVirus A and Respiratory SyncytialVirus B</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid amplificationtest</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">ePlex Instrument</td><td colspan="1" rowspan="1">FilmArray 2.0 or FilmArray® Torch</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">Reagents contained within cartridge toallow sample lysis and nucleic acidextraction, RT-PCR amplification, andhybridization-based electrochemicaldetection reagents.</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followed byhigh resolution melting analysis toconfirm identity of amplified product.</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Room temperature</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access rawdata.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Eight internal controls to demonstratethat sample extraction, amplificationand detection processes functioned asintended.</td><td colspan="1" rowspan="1">Two controls are included in eachreagent pouch to control for sampleprocessing and both stages of PCR andmelt analysis.</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

None

# L. Test Principle:

The ePlex instrument automates all aspects of nucleic acid testing including extraction, amplification, and detection, combining electrowetting and GenMark’s eSensor technology in a single-use cartridge. eSensor technology is based on the principles of competitive DNA hybridization and electrochemical detection, which is highly specific and is not based on fluorescent or optical detection.

Electrowetting, or digital microfluidics, uses electrical fields to directly manipulate discrete droplets on the surface of a hydrophobically coated printed circuit board (PCB). Sample and reagents are moved in a programmable fashion in the ePlex cartridge to complete all portions of the sample processing from nucleic acid extraction to detection.

A sample is loaded onto the ePlex cartridge and nucleic acids are extracted and purified from the specimen via magnetic solid phase extraction. For RNA targets, a reverse transcription step is performed to generate complementary DNA from the RNA, followed by PCR to amplify the targets. Exonuclease digestion creates single-stranded DNA in preparation for eSensor detection.

The target DNA is mixed with ferrocene-labeled signal probes that are complementary to the specific targets on the panel. Target DNA hybridizes to its complementary signal probe and capture probes, which are bound to gold-plated electrodes, as shown below in Figure 1.

![](images/85e327936ace65c94344d5b9980a4cd6a94584f772d8dc131be555d84402ebec.jpg)  
Figure 1: Hybridization complex. Target-specific capture probes are bound to the gold electrodes in the eSensor microarray on the ePlex cartridge. The amplified target DNA hybridizes to the capture probe and to a complementary ferrocene-labeled signal probe. Electrochemical analysis determines the presence or absence of targets using voltammetry.

The presence of each target is determined by voltammetry which generates specific electrical signals from the ferrocene-labeled signal.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility Study :

A multisite reproducibility study of the ePlex RP Panel was conducted at three testing sites (two external sites and one internal site) across major potential sources of variability, such as site-to-site, lot-to-lot, day-to-day, and operator-to-operator. One ePlex instrument per study site with either three or four towers was employed in this reproducibility study. Two operators performed testing at each site on six days (five nonconsecutive days) with three unique lots of ePlex RP Panel cartridges. A reproducibility study panel consisting of three panel members with six organisms that represent seven RP Panel targets (multi-spiked1) at three concentrations (moderate positive - 3x LoD, low positive - 1x LoD, and negative) was tested in triplicate. The six representative organisms in the reproducibility study panel included adenovirus, coronavirus OC43 (CoV-OC43), human metapneumovirus, influenza A H3, parainfluenza virus 1, and RSV A; organisms were diluted in natural clinical matrix (pooled, negative nasopharyngeal swab in VTM samples). Negative samples consisted of the same natural clinical matrix (pooled, negative nasopharyngeal swab in VTM samples) only. Each contrived sample was divided into aliquots and stored frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ prior to testing. Each operator tested nine samples (i.e., three member reproducibility panel in triplicate) each day; each panel member was tested 108 times (3 replicates x 3 sites $_ { \textrm { X 2 } }$ operators x 3 lots $_ { \textrm { X 2 } }$ days of testing/operator/lot) for a minimum of 324 tests.

Summary results for the ePlex RP Panel reproducibility study are provided in Table 4 below.

Table 4: Reproducibility of ePlex RP Panel Results   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">T able 4: Reproducibility of ePlex RP Panel Results</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3 a</td><td colspan="1" rowspan="1">All Sites(95% CI)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Vir</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(6.0E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/10800%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)(2.0E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">34/3694.4%</td><td colspan="1" rowspan="1">28/3580.0%</td><td colspan="1" rowspan="1">98/10791.6%(84.8%-95.5%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="3">Coronavirus</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(1.5E+03TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)(5.0E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/35100%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="2">HumanMetapneumovirus</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(6.8E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)(2.3E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/35100%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3 a</td><td colspan="1" rowspan="1">All Sites(95% CI)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">104/10797.2%</td><td colspan="1" rowspan="1">320/32399.1%(97.3%-99.7%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(1.5E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/361000%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">Low Positive(1× LoD)5.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/35100%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1-2009</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/32300%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/1071000%</td><td colspan="1" rowspan="1">323/323100%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="3">Influenza AH3</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(1.5E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/3600%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)(5.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/35100%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/323100%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVirus 1</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(1.2E+0TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)4.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/3500%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 2</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/323100%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">106/10799.1%</td><td colspan="1" rowspan="1">322/32399.7%(98.3%-99.9%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/323100%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="3">RespiratorySyncytialVirus A</td><td colspan="1" rowspan="1">ModeratePositive(3× LoD)(4.5E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)(1.5E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">35/35100%</td><td colspan="1" rowspan="1">107/107100%(96.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">36/36100%</td><td colspan="1" rowspan="1">108/108100%(96.6%-100%)</td></tr><tr><td colspan="1" rowspan="1">RespiratorySyncytialVirus B</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/32300%(98.8%-100%)</td></tr><tr><td colspan="7" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/107100%</td><td colspan="1" rowspan="1">323/323100%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">108/108100%</td><td colspan="1" rowspan="1">107/10899.1%</td><td colspan="1" rowspan="1">106/10799.1%</td><td colspan="1" rowspan="1">321/32399.4%(97.8%-99.8%)</td></tr></table>

a 1/324 reproducibility panel samples had initial and re-tested invalid results (tested at Site 3) and was excluded from this analysis.

Over the course of the reproducibility study, a total of three different ePlex instruments and 66 different ePlex bays (one 4-tower ePlex instrument each at Site 1 and Site 2, and one 3-tower ePlex instrument at Site 3, each tower has six bays) were used by six different operators at three different sites. All bays were utilized at least once during the conduct of this study.

In total, 334 runs were attempted with one pre-flight check failure. Of the 333 runs that initiated, all runs completed and valid results were obtained from 323 out of the 333 runs that were initiated $( 3 2 3 / 3 3 3 , 9 7 . 0 \% )$ .

Of the 324 reproducibility panel samples initially tested, 315 $( 9 7 . 2 \%$ ; $9 5 \%$ CI: $9 4 . 8 \% - 9 8 . 5 \% )$ had valid results and nine $( 2 . 8 \%$ ; $9 5 \%$ CI: $1 . 5 \% - 5 . 2 \%$ ) had invalid results. Initial validity rates were similar across the testing sites: $9 6 . 3 \%$ at Site 1, $9 8 . 1 \%$ at Site 2, $9 7 . 2 \%$ at Site 3. Initially invalid samples were re-tested and eight had valid results. The final validity rate was $9 9 . 7 \%$ ( $9 5 \%$ CI: $9 8 . 3 \% 9 9 . 9 \%$ ).

b. Linearity/assay reportable range:   
Not applicable, qualitative assay   
c. Traceability, Stability, Expected values (controls, calibrators, or   
methods):

# Assay Controls

# Internal Controls

Each ePlex RP Panel cartridge includes internal controls that monitor performance of each step of the testing process. A DNA control verifies extraction, amplification and detection of DNA targets, and RNA controls verify amplification and detection of RNA targets. Each amplification reaction on the cartridge has at least one internal control and in each reaction either the internal control or a target must generate signal above the defined threshold for a valid test result. Internal control results are interpreted by the ePlex software and displayed on ePlex RP Panel Reports as Internal Control with a result of PASS, FAIL, N/A, or INVALID. Table 5 below includes detailed interpretation of Internal Control results.

Table 5: Internal Control Results   

<table><tr><td colspan="1" rowspan="1">InternalControlResult</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">PASS</td><td colspan="1" rowspan="1">The internal control or a target from eachamplification reaction has generated signal abovethe threshold.The test was completed and internal controls weresuccessful, indicating valid results weregenerated.</td><td colspan="1" rowspan="1">All results are displayed onthe RP Panel DetectionReport.Test is valid, report results.</td></tr><tr><td colspan="1" rowspan="1">FAIL</td><td colspan="1" rowspan="1">Neither the internal control nor any target in atleast one amplification reaction generates signalabove the threshold.The test was completed but at least one internalcontrol was not detected, indicating that resultsare not valid.</td><td colspan="1" rowspan="1">No results are displayed onthe RP Panel DetectionReport.Test is not valid, repeat thetest using a new cartridge.</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">The internal control in every amplificationreaction does not generate signal above thethreshold, but a target in every amplificationreaction does generate signal above the threshold.The test was completed and internal controls werenot successful, however detection of signal abovethe threshold for a target in every amplificationreaction indicates valid results were generated.</td><td colspan="1" rowspan="1">All results are displayed onthe RP Panel DetectionReport.Test is valid, report results.</td></tr><tr><td colspan="1" rowspan="1">INVALID</td><td colspan="1" rowspan="1">An error has occurred during processing thatprevents analysis of signal data.The test has not successfully completed andresults for this test are not valid. This is often dueto an instrument or software error.</td><td colspan="1" rowspan="1">No results are displayed onthe RP Panel DetectionReport.Test is not valid, repeat thetest using a new cartridge.</td></tr></table>

# External Controls

External controls are not provided with the ePlex RP Panel. However, during the prospective clinical evaluation of the ePlex RP Panel, each testing site was provided with positive and negative external control samples. There were four external positive controls (A, B, C, D) and one external negative control. The positive controls together represent all the targets on the ePlex RP Panel. On each day of testing with the ePlex RP Panel, each site tested at least one positive and one negative control. Sites rotated through the four positive controls throughout the study.

Table 6: External Controls Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>External Controls</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>Positive Control A</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus, Influenza A H3, PIV 4, RSV A, and C.pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Positive Control B</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus, Human Metapneumovirus, Influenza A H1-2009, PIV 2, and RSV B</td></tr><tr><td rowspan=1 colspan=1>Positive Control C</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus, Influenza A H1, PIV 1, and PIV</td></tr><tr><td rowspan=1 colspan=1>Positive Control D</td><td rowspan=1 colspan=1>Coronavirus, Human Rhinovirus/Enterovirus, Influenza B, and M.pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Negative (Not Detected)</td></tr></table>

Over 104 days of investigational testing across five testing sites, there were 209 external controls tested with valid results: 27 Positive Control A, 24 Positive Control B, 27 Positive Control C, 25 Positive Control D, and 106 Negative Control. Of these, 203/209 had the expected results. The remaining six controls had the expected results except as follows:

two Negative Controls each had one false positive (FP) result: one for Adenovirus, one for Coronavirus

one Negative Control had four FP results: Coronavirus, Human   
Rhinovirus/Enterovirus, Influenza B, and Mycoplasma pneumoniae. It is likely the site inadvertently tested a Positive Control D sample and not a Negative Control since these four organisms are the expected results for Positive Control D   
two Positive Controls (A and B) each had a FP result for Human   
Rhinovirus/Enterovirus   
one Positive Control A had a false negative (FN) result for PIV 4.

The sponsor is also recommending the following in the product package insert: “Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Viral transport medium can be used as the negative control. Previously characterized positive samples or viral transport medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.”

# Specimen Stability

The ePlex RP Panel package insert claims that clinical specimens can be stored at room temperature $( 1 5 - 3 0 ^ { \circ } \mathrm { C } )$ for up to 12 hours or refrigerated at $4 \mathrm { { } ^ { \circ } C }$ for up to 10 days after collection in transport media. Specimens can also be stored at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ or - ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for six months with up to two freeze/thaw cycles.

These clinical specimen stability claims are supported by analytical study which investigated the effect of different specimen storage conditions on specimen integrity (as determined by performance on the ePlex RP Panel).

A representative panel of six analytes (Table 7) was spiked into a pooled nasopharyngeal (NP) sample to approximately 1x LoD concentrations. It is noted that the six-analyte test sample generates seven results for the ePlex RP Panel.

Table 7: Representative Analytes Contained in Pooled NP Sample for Analytical Studies   

<table><tr><td rowspan=1 colspan=1>Analyte Strain</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Expected Positive Result on theePlex RP Panel</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>H3N2 Brisbane/10/07</td><td rowspan=1 colspan=1>Flu A and Flu A H3</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>2006 Isolate</td><td rowspan=1 colspan=1>RSV-A</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>PIV-1</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>B2 Peru1-2002</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 7</td><td rowspan=1 colspan=1>Adenovirus</td></tr></table>

Baseline condition (T0) was established with 20 replicates. Each subsequent time point/test condition was assessed with 10 replicates unless otherwise noted. The overall study design (identifying all of the test conditions assessed in the study) is summarized in Table 8 below.

Table 8: Study Design – Specimen Storage and Testing Time points   

<table><tr><td rowspan=1 colspan=1>Storage Condition</td><td rowspan=1 colspan=12>Time Points</td></tr><tr><td rowspan=1 colspan=13>Ambient Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>15°-30</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>4h</td><td rowspan=1 colspan=1>8h</td><td rowspan=1 colspan=1>12h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=13>Refrigerated Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>2 - 8</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>3d</td><td rowspan=1 colspan=1>5d</td><td rowspan=1 colspan=1>7d</td><td rowspan=1 colspan=1>10d</td><td rowspan=1 colspan=1>14d</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=13>Frozen Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>-20 &amp; -80</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1w</td><td rowspan=1 colspan=1>2w</td><td rowspan=1 colspan=1>1m</td><td rowspan=1 colspan=1>3m</td><td rowspan=1 colspan=1>6m</td><td rowspan=1 colspan=1>12m</td><td rowspan=1 colspan=1>18m</td><td rowspan=1 colspan=1>24m</td><td rowspan=1 colspan=1>30m</td><td rowspan=1 colspan=1>36m</td><td rowspan=1 colspan=1>42m</td></tr></table>

Note: Shaded boxes indicate future time points that have not yet been tested.

The specimen stability study acceptance criteria were the following:

Performance at each time point will be considered acceptable if the positivity rate for each analyte (either from the first 10 replicates or from the final 20 replicates in case of additional testing) is equivalent to the positivity rate for the corresponding analyte at time point T0 (baseline time point). “Equivalent” is defined as positivity rates that differ by $5 \%$ or less. If a time point did not meet the positivity acceptance criteria, the next time point was evaluated. If the subsequent time point met the acceptance criteria, the study was continued. If two consecutive time points failed, the specimen stability was determined to be the last time point when the acceptance criteria were met.

The specimen stability assessment data are summarized in Table 9 to Table 13 below.

Table 9: Ambient Storage Conditions (Positivity and Mean Signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>TO</td><td rowspan=1 colspan=2>4 Hrs</td><td rowspan=1 colspan=2>8 Hrs</td><td rowspan=1 colspan=2>12 Hrs</td><td rowspan=1 colspan=2>24 Hrs</td></tr><tr><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>279.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>318.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>306.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>309.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>158.9</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>852.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>872.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>659.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>651.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>576.0</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>619.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>669.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>567.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>604.4</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>500.5</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>421.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>448.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>373.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>404.1</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>259.8</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>619.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>645.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>629.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>661.8</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>388.2</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>376.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>405.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>299.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>361.6</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>296.7</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>776.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>820.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>745.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>790.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>776.8</td></tr></table>

Table10: Refrigerated Storage Conditions (Positivity)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=7>% Positivity</td></tr><tr><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1 Day</td><td rowspan=1 colspan=1>3 Days</td><td rowspan=1 colspan=1>5 Days</td><td rowspan=1 colspan=1>7 Days</td><td rowspan=1 colspan=1>10 days</td><td rowspan=1 colspan=1>14 days</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

Table 11: Refrigerated Storage Conditions (Mean Signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=7>Mean Signal (nA)</td></tr><tr><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1 Day</td><td rowspan=1 colspan=1>3 Days</td><td rowspan=1 colspan=1>5 Days</td><td rowspan=1 colspan=1>7 Days</td><td rowspan=1 colspan=1>10 days</td><td rowspan=1 colspan=1>14 days</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>330.6</td><td rowspan=1 colspan=1>159.6</td><td rowspan=1 colspan=1>270.5</td><td rowspan=1 colspan=1>269.1</td><td rowspan=1 colspan=1>322.0</td><td rowspan=1 colspan=1>351.0</td><td rowspan=1 colspan=1>174.6</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>725.4</td><td rowspan=1 colspan=1>691.8</td><td rowspan=1 colspan=1>719.3</td><td rowspan=1 colspan=1>586.1</td><td rowspan=1 colspan=1>844.0</td><td rowspan=1 colspan=1>762.7</td><td rowspan=1 colspan=1>546.7</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>589.3</td><td rowspan=1 colspan=1>522.0</td><td rowspan=1 colspan=1>589.9</td><td rowspan=1 colspan=1>577.8</td><td rowspan=1 colspan=1>600.2</td><td rowspan=1 colspan=1>588.5</td><td rowspan=1 colspan=1>462.8</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>404.0</td><td rowspan=1 colspan=1>361.4</td><td rowspan=1 colspan=1>347.9</td><td rowspan=1 colspan=1>328.8</td><td rowspan=1 colspan=1>391.7</td><td rowspan=1 colspan=1>352.4</td><td rowspan=1 colspan=1>245.3</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>475.6</td><td rowspan=1 colspan=1>613.1</td><td rowspan=1 colspan=1>453.8</td><td rowspan=1 colspan=1>576.9</td><td rowspan=1 colspan=1>413.7</td><td rowspan=1 colspan=1>558.9</td><td rowspan=1 colspan=1>411.2</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>328.7</td><td rowspan=1 colspan=1>277.5</td><td rowspan=1 colspan=1>319.1</td><td rowspan=1 colspan=1>248.2</td><td rowspan=1 colspan=1>341.4</td><td rowspan=1 colspan=1>280.2</td><td rowspan=1 colspan=1>187.1</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>767.1</td><td rowspan=1 colspan=1>669.9</td><td rowspan=1 colspan=1>752.2</td><td rowspan=1 colspan=1>696.8</td><td rowspan=1 colspan=1>815.8</td><td rowspan=1 colspan=1>802.1</td><td rowspan=1 colspan=1>531.0</td></tr></table>

Table 12: Frozen Storage Conditions (Positivity)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>TO</td><td rowspan=1 colspan=2>1 Week</td><td rowspan=1 colspan=2>2 Weeks</td><td rowspan=1 colspan=2>1 Month</td><td rowspan=1 colspan=2>3 Months</td><td rowspan=1 colspan=2>6 Months</td></tr><tr><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95*</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr></table>

\* When the first 10 replicates were tested at the one week time point, RSV-A was missed in 1/10 replicates. Ten additional replicates were tested and the combined percent positivity for RSV-A was $9 5 \%$ .

Table 13: Frozen Storage Conditions (Mean Signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>T0</td><td rowspan=1 colspan=2>1 Week</td><td rowspan=1 colspan=2>2 Weeks</td><td rowspan=1 colspan=2>1 Month</td><td rowspan=1 colspan=2>3 Months</td><td rowspan=1 colspan=2>6 Months</td></tr><tr><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>-20</td><td rowspan=1 colspan=1>-80°</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>330.6</td><td rowspan=1 colspan=1>260.7</td><td rowspan=1 colspan=1>281.3</td><td rowspan=1 colspan=1>282.6</td><td rowspan=1 colspan=1>209.6</td><td rowspan=1 colspan=1>249.5</td><td rowspan=1 colspan=1>301.9</td><td rowspan=1 colspan=1>286.2</td><td rowspan=1 colspan=1>274.0</td><td rowspan=1 colspan=1>308.4</td><td rowspan=1 colspan=1>349.0</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>725.4</td><td rowspan=1 colspan=1>901.0</td><td rowspan=1 colspan=1>665.5</td><td rowspan=1 colspan=1>595.7</td><td rowspan=1 colspan=1>626.0</td><td rowspan=1 colspan=1>621.6</td><td rowspan=1 colspan=1>723.5</td><td rowspan=1 colspan=1>574.4</td><td rowspan=1 colspan=1>640.2</td><td rowspan=1 colspan=1>714.7</td><td rowspan=1 colspan=1>708.1</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>589.3</td><td rowspan=1 colspan=1>587.7</td><td rowspan=1 colspan=1>518.8</td><td rowspan=1 colspan=1>497.5</td><td rowspan=1 colspan=1>516.0</td><td rowspan=1 colspan=1>525.9</td><td rowspan=1 colspan=1>561.7</td><td rowspan=1 colspan=1>458.9</td><td rowspan=1 colspan=1>515.7</td><td rowspan=1 colspan=1>523.2</td><td rowspan=1 colspan=1>567.2</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>404.0</td><td rowspan=1 colspan=1>395.2</td><td rowspan=1 colspan=1>384.1</td><td rowspan=1 colspan=1>345.2</td><td rowspan=1 colspan=1>369.0</td><td rowspan=1 colspan=1>298.8</td><td rowspan=1 colspan=1>318.0</td><td rowspan=1 colspan=1>251.7</td><td rowspan=1 colspan=1>254.0</td><td rowspan=1 colspan=1>267.2</td><td rowspan=1 colspan=1>298.0</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>475.6</td><td rowspan=1 colspan=1>422.0</td><td rowspan=1 colspan=1>465.6</td><td rowspan=1 colspan=1>577.1</td><td rowspan=1 colspan=1>567.7</td><td rowspan=1 colspan=1>464.4</td><td rowspan=1 colspan=1>555.1</td><td rowspan=1 colspan=1>422.8</td><td rowspan=1 colspan=1>470.1</td><td rowspan=1 colspan=1>484.7</td><td rowspan=1 colspan=1>478.9</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>328.7</td><td rowspan=1 colspan=1>349.6</td><td rowspan=1 colspan=1>324.1</td><td rowspan=1 colspan=1>303.2</td><td rowspan=1 colspan=1>296.2</td><td rowspan=1 colspan=1>272.5</td><td rowspan=1 colspan=1>273.1</td><td rowspan=1 colspan=1>234.4</td><td rowspan=1 colspan=1>226.0</td><td rowspan=1 colspan=1>237.7</td><td rowspan=1 colspan=1>265.5</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>767.1</td><td rowspan=1 colspan=1>802.9</td><td rowspan=1 colspan=1>987.1</td><td rowspan=1 colspan=1>666.5</td><td rowspan=1 colspan=1>291.6</td><td rowspan=1 colspan=1>621.1</td><td rowspan=1 colspan=1>677.2</td><td rowspan=1 colspan=1>476.3</td><td rowspan=1 colspan=1>515.1</td><td rowspan=1 colspan=1>473.2</td><td rowspan=1 colspan=1>487.2</td></tr></table>

The data from the specimen stability studies supported the following specimen stability claims:

· storage under ambient conditions $( 1 5 \mathrm { - } 3 0 ^ { \circ } \mathrm { C } )$ for up to 12 hours in VTM · storage under refrigerated conditions $( 4 ^ { \circ } \mathrm { C } )$ for up to 10 days in VTM storage under frozen conditions ( ${ - 2 0 ^ { \circ } \mathrm { C } }$ and $- 8 0 ^ { \circ } \mathrm { C }$ ) for up to six months

# Fresh vs. Frozen Study

An analytical study was conducted to demonstrate clinical specimen stability after subjecting samples to one or two freeze-thaw cycles. A representative panel of six analytes (Table 7 above) was spiked into pooled negative NPS in VTM to approximately 1x LoD concentrations and tested fresh after one and after two freeze-thaw cycles at 10 replicates each.

The specimen freeze-thaw study results are summarized in Table 14 below.

Table 14: Freeze-Thaw Study (Positivity and Mean Signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>T0</td><td rowspan=1 colspan=2>T</td><td rowspan=1 colspan=2>22</td></tr><tr><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>229.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>355.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>379.1</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>790.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>720.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>714.0</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>622.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>588.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>593.9</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>424.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>389.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>410.1</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>585.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>385.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>355.1</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>365.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>351.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>361.4</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>816.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>703.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>685.1</td></tr></table>

The data from the specimen freeze-thaw study supported the following specimen stability claim:

specimens in VTM are stable after two freeze-thaw cycles

# In-Cartridge Sample Stability Study

An analytical study was carried out to demonstrate that the cartridge can be stored at room temperature $( 1 5 - 3 0 ^ { \circ } \mathrm { C } )$ for two hours after the sample has been loaded prior to running the cartridge in an ePlex bay.

A representative panel of six analytes (Table 7 above) was spiked into pooled negative NPS as matrix. Each target mix was tested at a final concentration of approximately 1x LoD. Forty (40) cartridges were loaded following package insert instructions. Twenty (20) of these cartridges were run within 10 minutes of loading; the other 20 cartridges were kept for two hours at room temperature and then run following package insert instructions.

The acceptance criteria for this analytical study were the following:

The sample is stable at room temperature after it has been loaded into the cartridge for up to 120 minutes prior to running the cartridge. Stability is defined as equal positivity rates for each target in each of the tested conditions that will be compared to each other. Equivalent positivity rate is defined as positivity rates that differ by $5 \%$ or less.

The in-cartridge sample stability study results are summarized in Table 15 and Table 16 below.

Table 15: In-Cartridge Sample Stability Study Comparison of Positivity Rate at 10 and 120 Minutes   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>% Positive</td></tr><tr><td rowspan=1 colspan=1>Sample loaded after 10 min</td><td rowspan=1 colspan=1>Sample loaded after 120 min</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>95% (19/20)</td></tr></table>

Table 16: In-Cartridge Sample Stability Study Comparison of Mean and Mean minus 2 SD at 10 and 120 Minutes   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Cutoff(nA)</td><td rowspan=1 colspan=2>Sample loaded after 10 min</td><td rowspan=1 colspan=2>Sample loaded after 120 min</td></tr><tr><td rowspan=1 colspan=1>mean signal(nA)</td><td rowspan=1 colspan=1>mean signalminus 2SD(nA)</td><td rowspan=1 colspan=1>mean signal(nA)</td><td rowspan=1 colspan=1>mean signalminus 2SD(nA)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>314.8</td><td rowspan=1 colspan=1>181.2</td><td rowspan=1 colspan=1>225.0</td><td rowspan=1 colspan=1>10.1</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>712.9</td><td rowspan=1 colspan=1>420.9</td><td rowspan=1 colspan=1>688.6</td><td rowspan=1 colspan=1>568.9</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>588.5</td><td rowspan=1 colspan=1>535.6</td><td rowspan=1 colspan=1>580.1</td><td rowspan=1 colspan=1>438.7</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>413.0</td><td rowspan=1 colspan=1>313.7</td><td rowspan=1 colspan=1>353.8</td><td rowspan=1 colspan=1>167.7</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>542.5</td><td rowspan=1 colspan=1>333.3</td><td rowspan=1 colspan=1>561.0</td><td rowspan=1 colspan=1>369.1</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>339.6</td><td rowspan=1 colspan=1>254.9</td><td rowspan=1 colspan=1>382.7</td><td rowspan=1 colspan=1>235.3</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>774.8</td><td rowspan=1 colspan=1>558.8</td><td rowspan=1 colspan=1>605.5</td><td rowspan=1 colspan=1>270.8</td></tr></table>

The study results demonstrated that the cartridge can be stored at room temperature $( 1 5 - 3 0 ^ { \circ } \mathrm { C } )$ for two hours after the sample has been loaded prior to running the cartridge in an ePlex bay.

# Simulated vs. Natural NPS in VTM Specimen Study

An analytical study was conducted to verify sample matrix equivalency between Remel M5 VTM (simulated matrix) and pooled negative nasopharyngeal swabs (NPS) collected in commercially available VTM (natural matrix) for the ePlex RP Panel for targets that are tested at 10x LoD and above.

A representative panel of six analytes (Table 7 above) was spiked either into Remel M5 VTM or pooled negative NPS at a final concentration of approximately $1 0 \mathrm { x }$ LoD. A negative control was run in duplicate for VTM and the NPS pool. Twenty (20) cartridge runs of spiked NPS matrix and 20 cartridge runs of spiked VTM were performed, as were two negative controls with each set of 20 runs (i.e., 44 cartridge runs were performed).

The acceptance criteria for this analytical study were the following:

The assay will be determined to not be impacted by sample matrix if the positivity rate for each target is equivalent to the positivity rate for the corresponding target in the simulated matrix control. Equivalent is defined as positivity rates that differ by $5 \%$ or less.

The sample matrix equivalence study results are summarized in Table 17 below.

Table 17: Sample Matrix Equivalence Study Positivity rate and mean signals for all targets in VTM and NPS   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=4>Remel M5 VTM</td><td rowspan=1 colspan=4>NPS in VTM</td><td rowspan=2 colspan=1>Cut offin nA</td><td rowspan=2 colspan=1>%differencein positivityrate</td></tr><tr><td rowspan=1 colspan=1>%positive</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>SD(nA)</td><td rowspan=1 colspan=1>MeannA-</td><td rowspan=1 colspan=1>%positive</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>SD(nA)</td><td rowspan=1 colspan=1>MeannA-2SD</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>308.9</td><td rowspan=1 colspan=1>125.5</td><td rowspan=1 colspan=1>57.9</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>317.8</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>122.2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>724.5</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>537.1</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>763.0</td><td rowspan=1 colspan=1>112.6</td><td rowspan=1 colspan=1>537.8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>RSV-A</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>631.3</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>434.1</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>623.7</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>465.7</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>466.2</td><td rowspan=1 colspan=1>77.6</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>474.1</td><td rowspan=1 colspan=1>71.7</td><td rowspan=1 colspan=1>330.8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>484.5</td><td rowspan=1 colspan=1>133.5</td><td rowspan=1 colspan=1>217.5</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>502.9</td><td rowspan=1 colspan=1>138.3</td><td rowspan=1 colspan=1>226.3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adv B</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>371.0</td><td rowspan=1 colspan=1>47.2</td><td rowspan=1 colspan=1>276.6</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>354.3</td><td rowspan=1 colspan=1>52.4</td><td rowspan=1 colspan=1>249.5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>790.3</td><td rowspan=1 colspan=1>90.2</td><td rowspan=1 colspan=1>609.9</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>736.2</td><td rowspan=1 colspan=1>106.7</td><td rowspan=1 colspan=1>522.8</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td></tr></table>

The results from the sample matrix equivalency study demonstrated the equivalency between Remel M5 VTM and natural nasopharyngeal swabs (NPS) collected in Remel M5 VTM for targets at approximately 10x LoD.

# d. Detection limit:

The limit of detection (LoD), or analytical sensitivity was identified and verified for each viral and bacterial target on the ePlex RP Panel using quantified reference strains/isolates. Serial dilutions were prepared in a natural clinical matrix (pooled, negative nasopharyngeal swab in VTM samples) with one or more organisms per series, and at least 20 replicates per target were tested in the verification. The limit of detection was defined as the lowest concentration at which each target is detected at least $9 5 \%$ of the time. The confirmed LoD for each ePlex RP Panel organism is shown in Table 18 below.

Table 18: LoD Results Summary   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Concentration</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 1 (C)</td><td rowspan=1 colspan=1>1.0E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Type 4 (E)</td><td rowspan=1 colspan=1>2.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Type 7 (B)</td><td rowspan=1 colspan=1>2.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>HKU1a</td><td rowspan=1 colspan=1>5.0E+04 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>7.5E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>5.0E+02 TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>A1 IA3-2002</td><td rowspan=1 colspan=1>2.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A2 IA14-2003</td><td rowspan=1 colspan=1>2.0E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B1 Peru2-2002</td><td rowspan=1 colspan=1>2.0E+02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B2 Peru1-2002</td><td rowspan=1 colspan=1>2.3E+02 TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Enterovirus Type 68 (2007)</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>1.5E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus B14</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus C a</td><td rowspan=1 colspan=1>1.0E+05 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>NY/01/2009</td><td rowspan=1 colspan=1>1.0E-01 TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>5.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 Brisbane/10/07</td><td rowspan=1 colspan=1>5.0E+01TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>1.0E-01 TCID50/mL</td></tr><tr><td rowspan=3 colspan=1>Influenza B (Victoria Lineage)</td><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Montana/5/2012</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=3 colspan=1>Influenza B (Yamagata Lineage)</td><td rowspan=1 colspan=1>B/Massachusetts/02/2012</td><td rowspan=1 colspan=1>1.0E+02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>1.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>4.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>5.0E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>5.0E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>3.0E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>2006 Isolate</td><td rowspan=1 colspan=1>1.5E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>2.0E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>3.0E+02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>FH strain of Eaton Agent [NCTC 10119]</td><td rowspan=1 colspan=1>3.0E+02 CCU/mL</td></tr></table>

a Clinical samples confirmed positive for coronavirus HKU1 and human rhinovirus C by bi-directional sequencing and quantified by real-time RT-PCR were used for determination of LoD.

# e. Analytical Reactivity:

A panel of 101 strains/isolates representing the genetic, temporal, and geographic diversity of each target on the ePlex RP Panel was evaluated to demonstrate analytical reactivity. Each strain/isolate was tested in triplicate at 3x LoD in natural clinical matrix (pooled, negative nasopharyngeal swab in VTM samples); if the organism was not detected at this concentration, testing of higher concentrations was performed. Additional in silico analysis was also performed on a subset of ePlex RP Panel organisms.

Results of analytical reactivity study, including in silico analysis results, are shown in Tables 19 to Table 29.

Table 19: Analytical Reactivity (Inclusivity) Results for Adenovirus   

<table><tr><td rowspan=1 colspan=1>Adenovirus Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Type 31</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=8 colspan=1>B</td><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type 11</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>De Wit Type 14</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Ch.79 Type 16</td><td rowspan=1 colspan=1>2.0E+02 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=1 colspan=1>Type 21</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Compton Type 34</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Holden Type 35</td><td rowspan=1 colspan=1>6.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Wan Type 50</td><td rowspan=1 colspan=1>2.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x b</td></tr><tr><td rowspan=3 colspan=1>C</td><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type 5</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type 6</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>D</td><td rowspan=1 colspan=1>Type 26</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type 37</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>F</td><td rowspan=1 colspan=1>Type 40 Dugan</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type 41/ Strain Tak</td><td rowspan=1 colspan=1>3.0E+03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a In silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles).

b In silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.

Table 20: Analytical Reactivity (Inclusivity) Results for Human Metapneumovirus   

<table><tr><td rowspan=1 colspan=1>Metapneumovirus Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>Peru6-2003 G, B2</td><td rowspan=1 colspan=1>6.8E+02 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 21: Analytical Reactivity (Inclusivity) Results for Human Rhinovirus/Enterovirus   

<table><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD Detected</td></tr><tr><td rowspan=14 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Type A2</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type A7</td><td rowspan=1 colspan=1>1.5E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>Type A16</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type A18</td><td rowspan=1 colspan=1>1.5E+02 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=1 colspan=1>Type A34</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type A57</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type A77</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>277G</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type B3</td><td rowspan=1 colspan=1>1.5E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>Type B17</td><td rowspan=1 colspan=1>1.5E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>Type B42</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type B83</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Type B84</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>FO2-2547</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Type 71</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=8 colspan=1>Coxsackievirus</td><td rowspan=1 colspan=1>A9</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A10</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A21</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A24</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>1.0E+02 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=1 colspan=1>B3</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B4</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B5</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=4 colspan=1>Echovirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E6</td><td rowspan=1 colspan=1>1.0E+01TCID50/mL</td><td rowspan=1 colspan=1>10x b</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>1.0E+01TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Poliovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0E+02 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr></table>

a In silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes. b In silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles).

# Table 22: Analytical Reactivity (Inclusivity) Results for Influenza A

Note: Due to different assays for influenza A matrix and influenza A subtypes on the ePlex RP Panel, if different LoDs are observed for inclusivity for a Flu A matrix vs. a subtype, the differences are noted in the Multiple of LoD Detected column.

<table><tr><td rowspan=1 colspan=1>Influenza A Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD Detected</td></tr><tr><td rowspan=7 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/FM/1/47</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x (Influenza A matrix) a10000x H1 subtypeb</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>9.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>9.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x (Influenza A matrix)H1 subtype not detected</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x (Influenza A matrix) aH1 subtype not detected</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>9.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x (Influenza A matrix)H1 subtype not detected</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/3/2006</td><td rowspan=1 colspan=1>9.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>9.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>30x f</td></tr><tr><td rowspan=5 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=5 colspan=1>1.5E+02 TCID50/mL</td><td rowspan=5 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td></tr><tr><td rowspan=1 colspan=1>A/Nanchang/933/95</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/05</td></tr><tr><td rowspan=8 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>A/California/7/2009</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x 8$</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/09</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/NY/02/2009</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x h</td></tr><tr><td rowspan=1 colspan=1>A/Swine NY/03/2009</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Swine/Iowa/15/30</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x (Influenza A matrix)100,000x (H1-2009 subtype) i</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x j</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC2/2009</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>100x j</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC3/2009</td><td rowspan=1 colspan=1>1.0E+02 TCID50/mL</td><td rowspan=1 colspan=1>1,000x j</td></tr></table>

a In silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles).

b In silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.   
$^ \mathrm { c } \mathrm { H } 1 { - } 2 0 0 9 $ subtype was detected in this seasonal influenza A H1 strain at $3 0 \mathrm { x }$ LoD.

d In silico analysis revealed little homology between this non-contemporary strain sequence and the H1 signal probe/capture probe sequences.

e In silico analysis revealed little homology between this non-contemporary influenza strain sequence and the H1 primer sequences.

f For Influenza A matrix, in silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { ( T C I D _ { 5 0 } }$ value is based only on infectious virus particles). For H1 subtype, in silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.

g For Influenza A matrix, in silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes. For H1 subtype, in silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles).

h For Influenza A matrix, in silico analysis revealed good homology to primers and probes. Lower sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain

$\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles). For H1-2009 subtype, in silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.

i In silico analysis revealed little homology between the strain sequence and the H1 or H1-2009 primer, signal probe and capture probe sequences.

j No sequence data was available to investigate lower sensitivity of the influenza A 2009 H1N1 A/Virginia/ATCC1/2009, A/Virginia/ATCC2/2009 and A/Virginia/ATTC3/2009 strains.

Table 23: Analytical Reactivity (Inclusivity) Results for Influenza A Strains Titered with Methods Different From the Reference Strain   

<table><tr><td rowspan=1 colspan=1>Influenza A Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Deteced</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>1.6E+02 CEID50/mL (Influenza A matrix)1.6E+08 CEID50/mL (H1 subtype)</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>1.6E+02 CEID50/mL (Influenza A matrix)1.6E+05 CEID50/mL (H1 subtype)</td></tr><tr><td rowspan=3 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Aichi/2/68 H3N2</td><td rowspan=1 colspan=1>1.6E+03 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>Alice (vaccine) A/England/42/72</td><td rowspan=1 colspan=1>5.0E+00 EID50/mL (Influenza A matrix)5.00E+01 EID50/mL (H3 subtype)</td></tr><tr><td rowspan=1 colspan=1>MRC-2 Recombinant Strain</td><td rowspan=1 colspan=1>8.9E+02 CEID50/mL (Influenza A matrix)8.9E+03 CEID50/mL (H3 subtype)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Washington/24/2012 (A/H1 pdm09)</td><td rowspan=1 colspan=1>3.2E+03 EID50/mL (Influenza A matrix)3.2E+02 EID50/mL (H1-2009 subtype)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N2</td><td rowspan=1 colspan=1>Kilbourne F63: A/NWS/34 (HA) xA/Rockefeller Institute/5/57 (NA),Reassortant NWS-F- Matrix</td><td rowspan=1 colspan=1>8.89 x 101 CEID50/mL (Influenza A matrix)No subtype detected a</td></tr><tr><td rowspan=1 colspan=1>Influenza A H5N8</td><td rowspan=1 colspan=1>A/Gyrfalcon/Washington/41088-6/2014BPL</td><td rowspan=1 colspan=1>1.6E+03 EID50/mL (Influenza A matrix)No subtype detected</td></tr><tr><td rowspan=1 colspan=1>Influenza A H5N2</td><td rowspan=1 colspan=1>A/NorthernPintail/Washington/40964/2014 BPL</td><td rowspan=1 colspan=1>2.5E+03 EID50/mL (Influenza A matrix)No subtype detected b</td></tr><tr><td rowspan=1 colspan=1>Influenza A H7N9</td><td rowspan=1 colspan=1>A/ANHUI/1/2013</td><td rowspan=1 colspan=1>7.9E+03 EID50/mL (Influenza A matrix)No subtype detected c</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2v</td><td rowspan=1 colspan=1>A/Indiana/21/2012</td><td rowspan=1 colspan=1>2.5E+04 EID50/mL (Influenza A matrix andH3 subtype)</td></tr></table>

a In silico analysis revealed little homology between this non-contemporary strain sequence and the H1 Signal Probe/Capture Probe sequences. b Detection of the H5 Subtype not expected c Detection of the H7 Subtype not expected NOTE: $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L } =$ Chick Embryo Infectious Dose; $\mathrm { E I D } _ { 5 0 } / \mathrm { m L } = \mathrm { E g g }$ Infectious Dose

For human, avian, and swine influenza A strains that were not available for wet testing on the ePlex RP Panel, in silico analysis was performed. Bioinformatics analysis was used to predict a result based on the number and location of mismatches in the primers, capture probes, and signal probes found in the ePlex RP Panel relative to an alignment of GenBank sequences. The in silico analysis results are summarized in Table 24 below.

Table 24: Predicted (in silico) Reactivity (Inclusivity) Results for Influenza A   

<table><tr><td colspan="1" rowspan="1">Influenza ASubtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBank ID</td><td colspan="1" rowspan="1">Predicted ePlexResult</td></tr><tr><td colspan="1" rowspan="5">H2N2</td><td colspan="1" rowspan="2">Human</td><td colspan="1" rowspan="1">A/Albany/20/1957(H2N2)</td><td colspan="1" rowspan="1">CY022014</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Kilbourne F38: A/Korea/426/68 (HA, NA) xA/Puerto Rico/8/34</td><td colspan="1" rowspan="1">CY037296</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="3">Avian</td><td colspan="1" rowspan="1">A/chicken/New York/13828-3/1995(H2N2)</td><td colspan="1" rowspan="1">CY014822</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Japan/305/1957(H2N2)</td><td colspan="1" rowspan="1">CY014977</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Korea/426/1968(H2N2)</td><td colspan="1" rowspan="1">CY031596</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H4N6</td><td colspan="1" rowspan="16">Avian</td><td colspan="1" rowspan="1">A/Blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td><td colspan="1" rowspan="1">CY063978</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="20">H5N1</td><td colspan="1" rowspan="1">A/Peregrine falcon/Aomori/7/2011</td><td colspan="1" rowspan="1">AB629716</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/West Bengal/239022/2010</td><td colspan="1" rowspan="1">CY061305</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/West Bengal/193936/2009</td><td colspan="1" rowspan="1">GU272009</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Hunan/1/2009</td><td colspan="1" rowspan="1">HM172150</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Hunan/8/2008</td><td colspan="1" rowspan="1">GU182162</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/West Bengal/106181/2008</td><td colspan="1" rowspan="1">GU083632</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Primorsky/85/2008</td><td colspan="1" rowspan="1">FJ654298</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/West Bengal/82613/2008</td><td colspan="1" rowspan="1">GU083648</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Duck/France/080036/2008</td><td colspan="1" rowspan="1">CY046185</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Vietnam/G12/2008</td><td colspan="1" rowspan="1">AB593450</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Thailand/PC-340/2008</td><td colspan="1" rowspan="1">EU620664</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Great egret/Hong Kong/807/2008</td><td colspan="1" rowspan="1">CY036240</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Rook/Rostov-on-Don/26/2007(H5N1)</td><td colspan="1" rowspan="1">EU814504</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Turkey/VA/505477-18/2007(H5N1)</td><td colspan="1" rowspan="1">GU186510</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Bangladesh/1151-10/2010(H5N1)</td><td colspan="1" rowspan="1">HQ156766</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="4">Human</td><td colspan="1" rowspan="1">A/Bangladesh/3233/2011</td><td colspan="1" rowspan="1">CY088772</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/R0405050/2007(H5N1)</td><td colspan="1" rowspan="1">HQ200572</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/S1211394/2008</td><td colspan="1" rowspan="1">HQ200597</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/486/97(H5N1)</td><td colspan="1" rowspan="1">AF255368</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Swine</td><td colspan="1" rowspan="1">A/Swine/East Java/UT6010/2007(H5N1)</td><td colspan="1" rowspan="1">HM440124</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="11">H5N2</td><td colspan="1" rowspan="11">Avian</td><td colspan="1" rowspan="1">A/Duck/Pennsylvania/10218/1984(H5N2)</td><td colspan="1" rowspan="1">AB286120</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/American black duck/Illinois/08OS2688/2008</td><td colspan="1" rowspan="1">CY079453</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/American green-wingedteal/California/HKWF609/2007</td><td colspan="1" rowspan="1">CY033447</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Canada goose/New York/475813-2/2007</td><td colspan="1" rowspan="1">GQ923358</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Blue-winged teal/Saskatchewan/22542/2007</td><td colspan="1" rowspan="1">CY047705</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Taiwan/A703-1/2008</td><td colspan="1" rowspan="1">AB507267</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Duck/France/080032/2008</td><td colspan="1" rowspan="1">CY046177</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Duck/New York/481172/2007</td><td colspan="1" rowspan="1">GQ117202</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Gadwall/Altai/1202/2007</td><td colspan="1" rowspan="1">CY049759</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Louisiana/476670-4/2007</td><td colspan="1" rowspan="1">GQ923390</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Waterfowl/Colorado/476466-2/2007</td><td colspan="1" rowspan="1">GQ923374</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H5N3</td><td colspan="1" rowspan="2">Avian</td><td colspan="1" rowspan="1">A/Duck/Singapore/F119/3/1997(H5N3)</td><td colspan="1" rowspan="1">GU052803</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H6N1</td><td colspan="1" rowspan="1">A/Duck/PA/486/1969(H6N1)</td><td colspan="1" rowspan="1">EU743287</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Influenza ASubtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBank ID</td><td colspan="1" rowspan="1">Predicted ePlexRResult</td></tr><tr><td colspan="1" rowspan="1">H6N2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">A/Mallard/Czech Republic/15902-17K/2009(H6N2)</td><td colspan="1" rowspan="1">HQ244433</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="9">H7N2</td><td colspan="1" rowspan="8">Avian</td><td colspan="1" rowspan="1">A/Chicken/Hebei/1/2002</td><td colspan="1" rowspan="1">AY724263</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/PA/149092-1/02</td><td colspan="1" rowspan="1">AY241609</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/NJ/294508-12/2004</td><td colspan="1" rowspan="1">EU743254</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/New York/23165-6/2005</td><td colspan="1" rowspan="1">CY031077</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Muscovy duck/New York/23165-13/2005</td><td colspan="1" rowspan="1">CY033226</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Muscovy duck/New York/87493-3/2005</td><td colspan="1" rowspan="1">CY034791</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Netherlands/29/2006</td><td colspan="1" rowspan="1">CY043833</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Northern shoveler/California/JN1447/2007</td><td colspan="1" rowspan="1">CY076873</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">Human</td><td colspan="1" rowspan="1">A/New York/107/2003(H7N2)</td><td colspan="1" rowspan="1">EU587373</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H7N3</td><td colspan="1" rowspan="1">A/Canada/rv504/2004(H7N3)</td><td colspan="1" rowspan="1">CY015007</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="8">H7N7</td><td colspan="1" rowspan="7">Avian</td><td colspan="1" rowspan="1">A/American green-wingedteal/Mississippi/090S046/2009</td><td colspan="1" rowspan="1">CY079309</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Germany/R28/03</td><td colspan="1" rowspan="1">AJ619676</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Netherlands/1/03</td><td colspan="1" rowspan="1">AY340091</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/California/HKWF1971/2007</td><td colspan="1" rowspan="1">CY033383</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Korea/GH171/2007</td><td colspan="1" rowspan="1">FJ959087</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Mute swan/Hungary/5973/2007</td><td colspan="1" rowspan="1">GQ240816</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Northern shoveler/Mississippi/ 09OS643/2009</td><td colspan="1" rowspan="1">CY079413</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">A/Netherlands/219/03(H7N7)</td><td colspan="1" rowspan="1">AY340089</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="5">H7N9</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">A/Shanghai/1/2013(H7N9)</td><td colspan="1" rowspan="1">EPI439493</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="4">Avian</td><td colspan="1" rowspan="1">A/Northernshoveler/Mississippi/110S145/2011(H7N9)</td><td colspan="1" rowspan="1">CY133650</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy turnstone/DelawareBay/220/1995(H7N9)</td><td colspan="1" rowspan="1">CY127254</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Turkey/Minnesota/1/1988(H7N9)</td><td colspan="1" rowspan="1">CY014787</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Blue-winged teal/Ohio/566/2006(H7N9)</td><td colspan="1" rowspan="1">CY024819</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">H9N2</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">A/Hong Kong/1073/99(H9N2)</td><td colspan="1" rowspan="1">AJ278647</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="3">Avian</td><td colspan="1" rowspan="1">A/Turkey/Wisconsin/1/1966(H9N2)</td><td colspan="1" rowspan="1">CY014664</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">A/chicken/Germany/N/1949(H10N7)</td><td colspan="1" rowspan="1">GQ176135</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">H11N9</td><td colspan="1" rowspan="1">A/Duck/Memphis/546/1974(H11N9)</td><td colspan="1" rowspan="1">GQ257441</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="3">H1N1</td><td colspan="1" rowspan="1">Swine</td><td colspan="1" rowspan="1">A/Swine/Wisconsin/1/1971(H1N1)</td><td colspan="1" rowspan="1">CY022414</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="8">Human</td><td colspan="1" rowspan="2">A/California/UR06-0393/2007(H1N1)</td><td colspan="1" rowspan="1">CY026540</td><td colspan="1" rowspan="2">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">CY026539</td></tr><tr><td colspan="1" rowspan="2">H1N2</td><td colspan="1" rowspan="2">A/New York/297/2003(H1N2)</td><td colspan="1" rowspan="1">CY002664</td><td colspan="1" rowspan="2">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">CY002665</td></tr><tr><td colspan="1" rowspan="4">H1N1 (2009)</td><td colspan="1" rowspan="2">A/Aalborg/INS133/2009(H1N1)</td><td colspan="1" rowspan="1">CY063606</td><td colspan="1" rowspan="2">Influenza A H1-2009</td></tr><tr><td colspan="1" rowspan="1">CY063607</td></tr><tr><td colspan="1" rowspan="2">A/South Carolina/02/2010(H1N1)</td><td colspan="1" rowspan="1">KC781370</td><td colspan="1" rowspan="2">Influenza A H1-2009</td></tr><tr><td colspan="1" rowspan="1">KC781372</td></tr><tr><td colspan="1" rowspan="2">H1N2</td><td colspan="1" rowspan="2">Swine</td><td colspan="1" rowspan="1">A/Swine/Hong Kong/NS857/2001(H1N2)</td><td colspan="1" rowspan="1">GQ229350</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Sweden/1021/2009(H1N2)</td><td colspan="1" rowspan="1">GQ495135</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Influenza ASubtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBank ID</td><td colspan="1" rowspan="1">Predicted ePlexResult</td></tr><tr><td colspan="1" rowspan="1">H3N1</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/Blue-winged teal/ALB/452/1983(H3N1)</td><td colspan="1" rowspan="1">CY004635</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="28">H3N2v</td><td colspan="1" rowspan="21">Human</td><td colspan="1" rowspan="2">A/Iowa/07/2011(H3N2)</td><td colspan="1" rowspan="1">JQ070760</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ290177</td></tr><tr><td colspan="1" rowspan="2">A/Iowa/08/2011(H3N2)</td><td colspan="1" rowspan="1">JQ070768</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ290167</td></tr><tr><td colspan="1" rowspan="2">A/Iowa/09/2011(H3N2)</td><td colspan="1" rowspan="1">JQ070776</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ290183</td></tr><tr><td colspan="1" rowspan="2">A/Indiana/08/2011(H3N2)</td><td colspan="1" rowspan="1">JQ070800</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ070795</td></tr><tr><td colspan="1" rowspan="2">A/Maine/06/2011(H3N2)</td><td colspan="1" rowspan="1">JN866181</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JN866186</td></tr><tr><td colspan="1" rowspan="1">A/Maine/07/2011(H3N2)</td><td colspan="1" rowspan="1">JN992746</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Pennsylvania/09/2011(H3N2)</td><td colspan="1" rowspan="1">JN655534</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Pennsylvania/11/2011(H3N2)</td><td colspan="1" rowspan="1">JN655540</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">A/Pennsylvania/10/2011(H3N2)</td><td colspan="1" rowspan="1">JN655550</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">A/West Virginia/06/2011(H3N2)</td><td colspan="1" rowspan="1">JQ290159</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ290164</td></tr><tr><td colspan="1" rowspan="1">A/West Virginia/07/2011(H3N2)</td><td colspan="1" rowspan="1">JQ348839</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">A/Indiana/10/2011(H3N2)</td><td colspan="1" rowspan="1">KJ942592</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JQ070787</td></tr><tr><td colspan="1" rowspan="2">A/Boston/38/2008(H3N2)</td><td colspan="1" rowspan="1">CY044580</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">CY044581</td></tr><tr><td colspan="1" rowspan="5">Swine</td><td colspan="1" rowspan="1">A/swine/NY/A01104005/2011(H3N2v)</td><td colspan="1" rowspan="1">JN940422</td><td colspan="1" rowspan="1">Influenza A H3</td></tr><tr><td colspan="1" rowspan="2">A/Maine/06/2011(H3N2)</td><td colspan="1" rowspan="1">JN866181</td><td colspan="1" rowspan="1">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JN866186</td><td colspan="1" rowspan="1">Influenza A H3</td></tr><tr><td colspan="1" rowspan="2">A/Indiana/08/2011(H3N2)</td><td colspan="1" rowspan="1">JN655558</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">JN638733</td></tr><tr><td colspan="1" rowspan="10">Avian</td><td colspan="1" rowspan="2">A/American black duck/North Carolina/675-075/2004(H3N2)</td><td colspan="1" rowspan="1">GU051135</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">GU051136</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">H3N5</td><td colspan="1" rowspan="2">A/Mallard/Netherlands/2/1999(H3N5)</td><td colspan="1" rowspan="1">CY060261</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">CY060264</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">H3N6</td><td colspan="1" rowspan="2">A/American black duck/NewBrunswick/25182/2007(H3N6)</td><td colspan="1" rowspan="1">CY047696</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">CY047697</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">H3N7</td><td colspan="1" rowspan="2">A/Northernshoveler/California/HKWF1367/2007(H3N7)</td><td colspan="1" rowspan="1">CY033372</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">CY033375</td><td colspan="1" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="2">H3N8</td><td colspan="1" rowspan="2">A/American blackduck/Washington/699/1978(H3N8)</td><td colspan="1" rowspan="1">GU052300</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">GU052299</td></tr></table>

Table 25: Analytical Reactivity (Inclusivity) Results for Influenza B   

<table><tr><td rowspan=1 colspan=1>Influenza B Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=4 colspan=1>Influenza B(Yamagata Lineage)</td><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=2 colspan=1>Influenza B(Victoria Lineage)</td><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>1.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>100x b</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Influenza B(Lineage unknown)</td><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a No sequence data available. Lower sensitivity may be a result of mismatches in the assay primers and/or probes. In addition, the reduced sensitivity may be the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { T C I D } _ { 5 0 }$ value is based only on infectious virus particles). b In silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.

Table 26: Analytical Reactivity (Inclusivity) Results for Parainfluenza Virus   

<table><tr><td rowspan=1 colspan=1>Parainfluenza Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD Detected</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>1.2E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>1.5E+02 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C-243</td><td rowspan=1 colspan=1>5.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>9.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a In silico analysis revealed that lower sensitivity may be a result of mismatches in the assay primers and/or probes.

Table 27: Analytical Reactivity (Inclusivity) Results for Respiratory Syncytial Virus   

<table><tr><td rowspan=1 colspan=2>RSV Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD Detected</td></tr><tr><td rowspan=2 colspan=2>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>4.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=2>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>6.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Wash/18537/62</td><td rowspan=1 colspan=1>6.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>WV/14617/85</td><td rowspan=1 colspan=1>6.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 28: Analytical Reactivity (Inclusivity) Results for Chlamydia pneumoniae   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=2 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>9.0E+02 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>TWAR strain 2043</td><td rowspan=1 colspan=1>9.0E+02 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 29: Analytical Reactivity (Inclusivity) Results for Mycoplasma pneumoniae   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=7 colspan=2>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>[Bru]</td><td rowspan=1 colspan=1>9 x 102 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>M129-B170</td><td rowspan=1 colspan=1>9 x 102 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>9 x 102 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[M52]</td><td rowspan=1 colspan=1>9 x 102 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[Mac]</td><td rowspan=1 colspan=1>9 x 1022 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=1 colspan=1>3 x 1044 CCU/mL</td><td rowspan=1 colspan=1>100x a</td></tr><tr><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>3 x 104CCU/mL</td><td rowspan=1 colspan=1>100x b</td></tr></table>

a No sequence data available. Lower sensitivity may be a result of mismatches in the assay primers and/or probes. In addition, the reduced sensitivity may be the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { C C U / m l }$ value is based only on live bacteria). $^ { \mathrm { ~ b ~ } } I n$ silico analysis revealed good homology to primers and probes. The reduced sensitivity is likely the result of incorrect estimation of genetic material present in the culture of this or the reference strain $\mathrm { ( C C U / m l }$ value is based only on live bacteria).

All of the 101 strains/isolates wet tested for inclusivity were detected by the ePlex RP Panel.

f. Analytical Specificity/Cross-reactivity Evaluation:

# On-Panel Organisms Testing

Potential cross-reactivity of each viral and bacterial target on the ePlex RP Panel was evaluated at high concentrations (i.e., $1 . 0 \mathrm { E } { + 0 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or ${ > } 1 . 0 \mathrm { E } { + } 0 5$ $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for viruses; $1 . 0 \mathrm { E } { + 0 6 }$ CFU/mL or $\mathrm { C C U / m L }$ for bacterial isolates, or $1 . 0 \mathrm { E } { + 0 6 }$ copies $/ \mathrm { m L }$ for in vitro transcripts) of quantified strains/isolates diluted in VTM. In vitro transcripts for coronavirus HKU1 was diluted in PBS. Table 30 below summarizes the results of the on-panel viral and bacterial strains/isolates tested.

Table 30: Cross-reactivity Study with ePlex RP Panel Target Organisms   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Adenovirus A</td><td colspan="1" rowspan="1">Type 31</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B</td><td colspan="1" rowspan="1">Type 7A</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus D</td><td colspan="1" rowspan="1">Type 9</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus E</td><td colspan="1" rowspan="1">Type 4</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F</td><td colspan="1" rowspan="1">Type 41</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">HKU1 in vitro transcript</td><td colspan="1" rowspan="1">1.0E+06 copies/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">NL63</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Type 68 2007 isolate</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">A/NY/01/2009</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2v a</td><td colspan="1" rowspan="1">A/Indiana/21/2012</td><td colspan="1" rowspan="1">2.5E+05 EID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H5N2 b</td><td colspan="1" rowspan="1">A/Northern PintailWashington/40964/14BPL</td><td colspan="1" rowspan="1">2.5E+05 EID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H5N8c</td><td colspan="1" rowspan="1">A/Gyrfalcon/Washington/410886/2014 BPL</td><td colspan="1" rowspan="1">1.6E+05 EID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza A H7N9 d</td><td colspan="1" rowspan="1">A/ANHUI/1/2013</td><td colspan="1" rowspan="1">7.9E+05 EID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/Florida/02/06</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">2006 Isolate</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">CH93(18)-18</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">AR-39</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">FH strain of Eaton Agent[NCTC 10119]</td><td colspan="1" rowspan="1">1.0E+06 CCU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr></table>

a Influenza A H3N2v detected as Influenza A, Influenza A H3 b Influenza A H5N2 detected as Influenza A c Influenza A H5N8 detected as Influenza A d Influenza A H7N9 detected as Influenza A

No cross-reactivity was observed between any of the on-panel viruses or bacteria.

# Off-Panel Organisms Testing

Potential cross-reactivity of viruses, bacteria, and fungi that are not targets on the ePlex RP Panel was evaluated at high concentrations (i.e., $1 . 0 \mathrm { E } { + 0 } 5 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or copies/mL for viruses, $1 . 0 \mathrm { E } ^ { + 0 6 }$ CFU/mL for bacterial and yeast isolates, or $1 . 0 \mathrm { E } { + 0 6 }$ copies/mL for plasmid DNA or genomic RNA) by diluting quantified strains/isolates in VTM. Plasmid for Bocavirus and genomic RNA for MERS- $\mathrm { \ C o V }$ were diluted in PBS. Table 31 below summarizes the results of the strains/isolates tested.

Table 31: Cross-reactivity with Organisms Not Targeted by the ePlex RP Panel (Exclusivity)   

<table><tr><td colspan="2" rowspan="1">(Exclasivly)</td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Cross-Reactivityesults</td></tr><tr><td colspan="2" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">18323 [NCTC 10739]</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">ATCC 15311</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 25416</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15126</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">MERS-CoV</td><td colspan="1" rowspan="1">EMC/2012 a</td><td colspan="1" rowspan="1">1.0E+05 copies/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">ATCC 13812</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">AD 169</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Strain B95-8</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 10279</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 43065</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Herpes Simplex Virus</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Human bocavirus</td><td colspan="1" rowspan="1">Bocavirus plasmid b</td><td colspan="1" rowspan="1">1.0E+06 copies/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 51504</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 314</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">ATCC 8014</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Philadelphia-1</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Measles</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 23246</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Isolate 2</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">ATCC 25177</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Neisseria meningiditis</td><td colspan="1" rowspan="1">ATCC 13077</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 29193</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">ATCC 33277</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 33420</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus aureus (MRSA)</td><td colspan="1" rowspan="1">NRS384</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus aureus (MSSA)</td><td colspan="1" rowspan="1">ATCC 25923</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus epidermidis(MRSE)</td><td colspan="1" rowspan="1">ATCC 35983</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus epidermidis (MSSE)</td><td colspan="1" rowspan="1">ATCC 49134</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29970</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">ATCC 35666</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 15914</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 49619</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="2" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">8.9E+03 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr></table>

a Extracted genomic RNA b Plasmid does not contain full length viral genome.

No cross-reactivity was observed between any of the off-panel viruses, bacteria or fungi with the ePlex RP Panel targets.

# g. Assay cut-off:

Analytical studies were conducted to establish the signal boundaries for all targets and controls of the ePlex RP Panel.

Positive and negative NPS samples were obtained from internal and external sources. Supplemental samples were contrived for analytes that were underrepresented in the existing sample set. Contrived samples included viral and bacterial cultures prepared in negative NPS. Contrived positive samples were formulated using cultured analytes that were diluted into negative NPS matrix. For each target, the signals for positive and negative tests were analyzed. An appropriate boundary was established wherein specificity and sensitivity were maximized. The analysis was verified by ROC analysis. The final boundary set points are listed in Table 32 below.

Table 32: ePlex RP Panel Boundary Set Points   

<table><tr><td colspan="1" rowspan="1">Target Name</td><td colspan="1" rowspan="1">Boundary (nA)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Adenovirus E</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C (Penton)</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="1">Adenovirus (Pan)</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus A/C</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus B</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 4a5</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">20/30 a</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Control Name</td><td colspan="1" rowspan="1">Boundary(nA)</td></tr><tr><td colspan="1" rowspan="1">Control 1 (IC4 - hMPV)</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Control 2 (IC2  PIV2)</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Control 4 (IC1  H3)</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Control 6 (IC2  Flu A)</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Control 7 (IC2  229E)</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Control 8 (IC1  PIV3)</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Control 3 (Pombe  Bact)</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Control 5 (Pombe  Adv)</td><td colspan="1" rowspan="1">20</td></tr></table>

a Due to elevated background signal on RSV B in the presence of RSV A virus, there is a conditional threshold for RSV B for when an RSV A signal is detected. In the absence of RSV A signal above $1 0 \mathrm { n A }$ , the boundary set point for RSV B is $2 0 \mathrm { n A }$ ; if an RSV A signal is detected at greater than 10 nA on a cartridge, the boundary set point for RSV B is set to $3 0 \mathrm { n A }$ .

# h. Interfering Substances:

Substances commonly found in respiratory samples, substances that could be introduced during specimen collection, or medications commonly used to treat congestion, allergies, or asthma symptoms that could potentially interfere with the ePlex RP Panel were individually evaluated in an analytical study. To simulate clinical samples, quantified representative viral and bacterial strains were diluted to 1x LoD in a natural clinical matrix (pooled, negative nasopharyngeal swab specimens) and tested in triplicate for negative and positive interference. Natural clinical matrix (pooled, negative nasopharyngeal swab samples) with no organisms added was used as a control.

All substances and organisms tested for interference were shown to be compatible with the ePlex RP Panel. No potentially interfering substances or microorganisms were found to inhibit the ePlex RP Panel at the concentrations tested in Table 33 below.

Table 33: List of Substances/Organisms and Concentrations for Testing in the Interfering Substances Study   

<table><tr><td rowspan=1 colspan=1>Potentially Interfering Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Control Sample Matrix a</td><td rowspan=1 colspan=1>Becton Dickinson UVT</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Transport Medium a</td><td rowspan=1 colspan=1>Copan eSwab (Liquid Amies media)</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=4 colspan=1>Viral Transport Medium a</td><td rowspan=1 colspan=1>MicroTest M4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>MicroTest M4-RT</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>MicroTest M5</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>MicroTest M6</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Flocked Swabs</td><td rowspan=1 colspan=1>Copan Minitip in UVT</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Copan Regular Tip in UVT</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Blood (human)</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Human gDNA</td><td rowspan=1 colspan=1>50 ng/rxn</td></tr><tr><td rowspan=1 colspan=1>Throat lozenges, oral anesthetic andanalgesic</td><td rowspan=1 colspan=1>Benzocaine, menthol</td><td rowspan=1 colspan=1>26% w/v</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=3 colspan=1>Nasal sprays or drops</td><td rowspan=1 colspan=1>Phenylephrine HCl (Neo-Synephrine®)</td><td rowspan=1 colspan=1>1.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Oxymetazoline HCl (Afrin®)</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Sodium chloride</td><td rowspan=1 colspan=1>0.8% w/v</td></tr><tr><td rowspan=1 colspan=1>Antibacterial, systemic</td><td rowspan=1 colspan=1>Tobramycin b</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>2% w/v</td></tr><tr><td rowspan=6 colspan=1>Nasal corticosteroids</td><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Budesonide (Rhinocort®)</td><td rowspan=1 colspan=1>0.9% v/v</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone (Nasacort®)</td><td rowspan=1 colspan=1>1.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Fluticasone (Flonase®)</td><td rowspan=1 colspan=1>1.5% v/v</td></tr><tr><td rowspan=4 colspan=1>ZICAM® Allergy Relief Nasal Gel</td><td rowspan=1 colspan=1>Luffa opperculata</td><td rowspan=4 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Sulfur</td></tr><tr><td rowspan=1 colspan=1>Galphimia glauca</td></tr><tr><td rowspan=1 colspan=1>Histaminum hydrochloricum</td></tr><tr><td rowspan=2 colspan=1>Anti-viral drugs</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>550 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>142 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1.0E+05 TCID50/mL</td></tr><tr><td rowspan=6 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=6 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diptheriae</td></tr></table>

a Testing of media was done by adding a negative NPS collected in the specified media and diluting in the natural clinical matrix. b At concentrations greater than $1 \%$ weight/volume in the sample, tobramycin was found to inhibit assay performance.

Note: Nasal influenza vaccines (e.g., FluMist) were not evaluated in this study, but are predicted to be reactive with the ePlex RP Panel Influenza A (including subtype) and Influenza B assays. Therefore, contamination of specimens with vaccine or recent administration of the vaccine prior to NPS specimen collection could lead to accurate detection by the ePlex RP Panel of the viruses contained in the vaccine, but would not represent infection by those agents.

# i. Co-Detected Organisms Study:

Detection of more than one clinically relevant viral and/or bacterial organism in a sample was evaluated in an analytical study with the ePlex RP Panel, using a natural clinical matrix (pooled, negative nasopharyngeal swab samples) spiked with two ePlex RP Panel targeted organisms: one organism at a low concentration (i.e., $1 { - } 3 \mathbf { x }$ LoD) and the second organism at a high concentration (i.e., $1 . 0 \mathrm { E } { + 0 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ .

Study results are summarized in Table 34 below.

Table 34: Detection of Co-Infections   

<table><tr><td colspan="1" rowspan="1">ePlex RP PanelResultOrganism 1</td><td colspan="1" rowspan="1">Organism 1(High Titer)</td><td colspan="1" rowspan="1">ePlex RP PanelResultOrganism 2</td><td colspan="1" rowspan="1">Organism 2(Low Titer)</td><td colspan="1" rowspan="1">Organism 2Multiple of LoD</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Adenovirus B</td><td colspan="1" rowspan="1">2.0E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">5.0E+01 TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">5.0E+01TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">6.0E-01TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">1.0E-01TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">3.0E-01TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">5.0E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">1 x 10-1TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">7.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">2.0E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">2.3E+02TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.5E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0E+05TCID50/mL</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">2.0E+00TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr></table>

The study results demonstrated the ability of the ePlex RP Panel to detect two clinically relevant co-infecting organisms in a sample at both high and low concentrations.

# j. Carry-Over Contamination:

The carryover/cross-contamination rate of the ePlex RP Panel and ePlex instrument was evaluated using a checkerboard approach by running high positive and negative samples interspersed in all bays of a four-tower ePlex instrument (i.e., 24 bays total) over five separate runs on five separate days. Quantified parainfluenza virus 3 was prepared in VTM at a high concentration $( 1 . 0 \mathrm { E } { + 0 5 } \ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ to simulate a clinically relevant high positive and was tested as a representative target organism. VTM was used to represent negative samples. On each round of testing, 24 ePlex RP Panel cartridges were evaluated. $100 \%$ of parainfluenza 3-positive samples generated a result of Detected and $100 \%$ of parainfluenza 3-negative samples generated a parainfluenza 3 result of No Target Detected, indicating no carryover or cross-contamination was observed between bays or within bays with the ePlex RP Panel when testing samples consecutively or in adjacent bays with an ePlex instrument .

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# Prospective Clinical Study

The clinical performance of the ePlex RP Panel was established during multi-center clinical studies conducted at five distinct U.S. test sites. Each test site was

representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

Clinical performance of the ePlex RP Panel was evaluated testing clinical nasopharyngeal swab (NPS) samples prospectively-collected in VTM at eight geographically distinct U.S. clinical sites in two phases. From March 2013 through August 2014 (Phase 1), NPS in VTM samples were prospectively-collected from all comers meeting the study eligibility criteria at four specimen collection sites and immediately frozen $\mathrm { N } { = } 2 2 1 8$ specimens) for later testing as prospective archived/frozen (Category II) specimens. From September 2016 through October 2016 (Phase 2), NPS in VTM samples were prospectively-collected from all comers meeting the study eligibility criteria and tested fresh at five specimen collection and testing sites $\scriptstyle ( \mathrm { N } = 5 1 4$ specimens) as prospective fresh (Category I) specimens. Category II specimens were distributed to sites beginning in September 2016. Study sites also began testing Category I specimens at this time. At each site, Category II specimens were thawed and tested according to the study procedures as time permitted over the remaining duration of the clinical study.

A total of 2732 prospective specimens (Category I and II) were collected across the two phases. Prior to the start of investigational testing the ePlex RP Panel, 263 specimens were withdrawn (251 specimens had sample handling deviations, nine specimens were outside of the protocol allowed timelines, two specimens had insufficient volume, and one specimen had incomplete documentation) from the study. Of the 2469 remaining prospective specimens eligible for testing, 2462 (511 Category I and 1951 Category II specimens) were evaluable. Samples with final, valid results and a valid comparator result were considered evaluable. Seven (7) prospective specimens were not evaluable due to the lack of final, valid ePlex RP Panel results and were excluded from performance evaluations.

Demographic information for the 2462 evaluable prospective specimens is described in Table 35 below.

Table 35: Subject Demographic Data for All Evaluable Prospective Specimens by Collection Site $\scriptstyle ( \mathbf { N } = 2 4 6 2 )$   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>All SitesN=2462n (%)</td><td rowspan=1 colspan=1>Site 1N=165</td><td rowspan=1 colspan=1>Site 2N=248</td><td rowspan=1 colspan=1>Site 3N=350</td><td rowspan=1 colspan=1>Site 4N=892</td><td rowspan=1 colspan=1>Site 5N=345</td><td rowspan=2 colspan=1>Site 6N=101n (%)</td><td rowspan=2 colspan=1>Site 7N=161n (%)</td><td rowspan=2 colspan=1>Site 8N=200n (%)</td></tr><tr><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td></tr><tr><td rowspan=1 colspan=10>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>1247 (50.6)</td><td rowspan=1 colspan=1>96 (58.2)</td><td rowspan=1 colspan=1>118 (47.6)</td><td rowspan=1 colspan=1>186 (53.1)</td><td rowspan=1 colspan=1>450 (50.4)</td><td rowspan=1 colspan=1>188 (54.5)</td><td rowspan=1 colspan=1>43 (42.6)</td><td rowspan=1 colspan=1>84 (52.2)</td><td rowspan=1 colspan=1>82 (41.0)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1215 (49.4)</td><td rowspan=1 colspan=1>69 (41.8)</td><td rowspan=1 colspan=1>130 (52.4)</td><td rowspan=1 colspan=1>164 (46.9)</td><td rowspan=1 colspan=1>442 (49.6)</td><td rowspan=1 colspan=1>157 (45.5)</td><td rowspan=1 colspan=1>58 (57.4)</td><td rowspan=1 colspan=1>77 (47.8)</td><td rowspan=1 colspan=1>118 (59.0)</td></tr><tr><td rowspan=1 colspan=10>Age (years)</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>388 (15.8)</td><td rowspan=1 colspan=1>17 (10.3)</td><td rowspan=1 colspan=1>21 (8.5)</td><td rowspan=1 colspan=1>74 (21.1)</td><td rowspan=1 colspan=1>164 (18.4)</td><td rowspan=1 colspan=1>45 (13.0)</td><td rowspan=1 colspan=1>28 (27.7)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>36 (18.0)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>325 (13.2)</td><td rowspan=1 colspan=1>12 (7.3)</td><td rowspan=1 colspan=1>22 (8.9)</td><td rowspan=1 colspan=1>62 (17.7)</td><td rowspan=1 colspan=1>64 (7.2)</td><td rowspan=1 colspan=1>100 (29.0)</td><td rowspan=1 colspan=1>39 (38.6)</td><td rowspan=1 colspan=1>16 (9.9)</td><td rowspan=1 colspan=1>10 (5.0)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>321 (13.0)</td><td rowspan=1 colspan=1>15 (9.1)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>38 (10.9)</td><td rowspan=1 colspan=1>82 (9.2)</td><td rowspan=1 colspan=1>116 (33.6)</td><td rowspan=1 colspan=1>34 (33.7)</td><td rowspan=1 colspan=1>18 (11.2)</td><td rowspan=1 colspan=1>12 (6.0)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>926 (37.6)</td><td rowspan=1 colspan=1>87 (52.7)</td><td rowspan=1 colspan=1>131 (52.8)</td><td rowspan=1 colspan=1>98 (28.0)</td><td rowspan=1 colspan=1>385 (43.2)</td><td rowspan=1 colspan=1>55 (15.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>92 (57.1)</td><td rowspan=1 colspan=1>78 (39.0)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>502 (20.4)</td><td rowspan=1 colspan=1>34 (20.6)</td><td rowspan=1 colspan=1>68 (27.4)</td><td rowspan=1 colspan=1>78 (22.3)</td><td rowspan=1 colspan=1>197 (22.1)</td><td rowspan=1 colspan=1>29 (8.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>32 (19.9)</td><td rowspan=1 colspan=1>64 (32.0)</td></tr></table>

A total of five test sites participated in the prospective clinical evaluation of ePlex RP Panel. One test site had a 4-tower ePlex instrument (24 bays) and the remaining four test sites had a 3-tower ePlex instrument (18 bays). A total of 13 ePlex RP Panel cartridge lots were used in the prospective clinical evaluation.

The performance of the ePlex RP Panel was evaluated by comparing the ePlex RP Panel results with those from an FDA-cleared multiplexed respiratory pathogen panel (i.e., the main comparator) and an analytically validated PCR test with bidirectional sequencing for confirmation of RSV subtypes.

# Prospective Clinical Study Performance

Positive percent agreement (PPA) for each panel organism was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result was one where the “detected” ePlex RP Panel result matched the “detected” comparator method result, while a TN result was one where a “negative” ePlex RP Panel result matched a “negative” comparator method result. A FN result was one where the “negative” ePlex RP Panel result did not match the “detected” comparator method result, while a FP result was one where the “detected” ePlex RP Panel result did not match the “negative” comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

Samples for which FP and/or FN results (i.e., discrepant results) were obtained when comparing the ePlex RP Panel results to the comparator method results were further investigated. The discrepancy investigation was mainly conducted by performing analytically validated PCR/Sequencing assays.

The ePlex RP Panel prospective performance data in positive percent and negative percent agreements against the comparator methods (all sites combined) are presented in the following Table 36 by analyte:

Table 36: ePlex RP Panel Prospective Clinical Performance Summary   

<table><tr><td colspan="2" rowspan="2">Organism</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="3">Adenovirus a</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">40.9-92.9</td><td colspan="1" rowspan="1">499/503</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.0-99.7</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">48/53</td><td colspan="1" rowspan="1">90.6</td><td colspan="1" rowspan="1">79.7-95.9</td><td colspan="1" rowspan="1">1874/1898</td><td colspan="1" rowspan="1">98.7</td><td colspan="1" rowspan="1">98.1-99.1</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">54/61</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">78.2-94.3</td><td colspan="1" rowspan="1">2373/2401</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">98.3-99.2</td></tr><tr><td colspan="1" rowspan="3">Coronavirus b</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100</td><td colspan="1" rowspan="1">503/504</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">89/110</td><td colspan="1" rowspan="1">80.9</td><td colspan="1" rowspan="1">72.6-87.2</td><td colspan="1" rowspan="1">1828/1841</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.8-99.6</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">96/117</td><td colspan="1" rowspan="1">82.1</td><td colspan="1" rowspan="1">74.1-88.0</td><td colspan="1" rowspan="1">2331/2345</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">99.0-99.6</td></tr><tr><td colspan="1" rowspan="3">hMPV</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">107/113</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">88.9-97.5</td><td colspan="1" rowspan="1">1832/1838</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">107/113</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">88.9-97.5</td><td colspan="1" rowspan="1">2343/2349</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="3">HRV/EV d</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">176/183</td><td colspan="1" rowspan="1">96.2</td><td colspan="1" rowspan="1">92.3-98.1</td><td colspan="1" rowspan="1">316/328</td><td colspan="1" rowspan="1">96.3</td><td colspan="1" rowspan="1">93.7-97.9</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">317/336</td><td colspan="1" rowspan="1">94.3</td><td colspan="1" rowspan="1">91.3-96.4</td><td colspan="1" rowspan="1">1544/1615</td><td colspan="1" rowspan="1">95.6</td><td colspan="1" rowspan="1">94.5-96.5</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">493/519</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">92.8-96.6</td><td colspan="1" rowspan="1">1860/1943</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">94.7-96.5</td></tr><tr><td colspan="1" rowspan="3">FluA</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">106/111</td><td colspan="1" rowspan="1">95.5</td><td colspan="1" rowspan="1">89.8-98.1</td><td colspan="1" rowspan="1">1836/1840</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">106/111</td><td colspan="1" rowspan="1">95.5</td><td colspan="1" rowspan="1">89.8-98.1</td><td colspan="1" rowspan="1">2347/2351</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.6-99.9</td></tr><tr><td colspan="1" rowspan="3">FluA H1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1951/1951</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2462/2462</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">FluA H1-2009</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">70/71</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">92.4-99.8</td><td colspan="1" rowspan="1">1874/1880</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">70/71</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">92.4-99.8</td><td colspan="1" rowspan="1">2385/2391</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.5-99.9</td></tr><tr><td colspan="1" rowspan="3">FluA H3 g</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">34/37</td><td colspan="1" rowspan="1">91.9</td><td colspan="1" rowspan="1">78.7-91.2</td><td colspan="1" rowspan="1">1914/1914</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">34/37</td><td colspan="1" rowspan="1">91.9</td><td colspan="1" rowspan="1">78.7-91.2</td><td colspan="1" rowspan="1">2425/2425</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">FluB h</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">509/510</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">58/65</td><td colspan="1" rowspan="1">89.2</td><td colspan="1" rowspan="1">79.4-94.7</td><td colspan="1" rowspan="1">1882/1886</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.5-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">59/66</td><td colspan="1" rowspan="1">89.4</td><td colspan="1" rowspan="1">79.7-94.8</td><td colspan="1" rowspan="1">2391/2396</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.5-99.9</td></tr><tr><td colspan="1" rowspan="3">PIV1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">510/510</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3</td><td colspan="1" rowspan="1">1926/1927</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">80.5-99.3</td><td colspan="1" rowspan="1">2436/2437</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">PIV2</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">66.7-98.6</td><td colspan="1" rowspan="1">497/498</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100</td><td colspan="1" rowspan="1">1941/1942</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">21/22</td><td colspan="1" rowspan="1">95.5</td><td colspan="1" rowspan="1">78.2-99.2</td><td colspan="1" rowspan="1">2438/2440</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">PIV3 </td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">506/506</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">94/104</td><td colspan="1" rowspan="1">90.4</td><td colspan="1" rowspan="1">83.2-94.7</td><td colspan="1" rowspan="1">1842/1847</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">99/109</td><td colspan="1" rowspan="1">90.8</td><td colspan="1" rowspan="1">83.9-94.9</td><td colspan="1" rowspan="1">2348/2353</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.5-99.9</td></tr><tr><td colspan="1" rowspan="3">PIV4j</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">503/508</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">97.7-99.6</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">1944/1946</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100</td><td colspan="1" rowspan="1">2447/2454</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">56.5-98.0</td><td colspan="1" rowspan="1">501/501</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">27/31</td><td colspan="1" rowspan="1">87.1</td><td colspan="1" rowspan="1">71.1-94.9</td><td colspan="1" rowspan="1">1917/1918</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">35/40</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">73.9-94.5</td><td colspan="1" rowspan="1">2418/2419</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">RSV B</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">59.6-98.2</td><td colspan="1" rowspan="1">500/500</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">81/86</td><td colspan="1" rowspan="1">94.2</td><td colspan="1" rowspan="1">87.1-97.5</td><td colspan="1" rowspan="1">1861/1863</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">90/96</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">87.0-97.1</td><td colspan="1" rowspan="1">2361/2363</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">C. pneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">11.8-76.9</td><td colspan="1" rowspan="1">1945/1946</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">11.8-76.9</td><td colspan="1" rowspan="1">2456/2457</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">M. pneumoniae m</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">507/508</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">37.6-96.4</td><td colspan="1" rowspan="1">1945/1946</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">52.9-97.8</td><td colspan="1" rowspan="1">2452/2454</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr></table>

a Adenovirus was not detected in 3/7 FN specimens using PCR/Sequencing. Adenovirus was detected in 13/28 FP specimens using PCR/Sequencing.   
b Twenty (20) FN prospective-frozen specimens were repeat tested with the comparator method and 12 had Coronavirus detected. Of these 12 specimens, 11 were repeat tested with the ePlex RP Panel and 3 had Coronavirus detected. Coronavirus was not detected in 2/21 FN specimens using PCR/Sequencing. Coronavirus was detected in 3/13 FP specimens using PCR/sequencing. One (1) FP specimen was not tested by PCR/Sequencing.   
c Human Metapneumovirus was not detected in 1/6 FN specimens using PCR/Sequencing. Human Metapneumovirus was detected in 4/6 FP specimens using PCR/sequencing.   
d Human rhinovirus/enterovirus was not detected in 7/26 FN specimens using PCR/Sequencing. Human   
rhinovirus/enterovirus was detected in 42/83 FP samples using PCR/sequencing.   
e Influenza A comparator results contain 71 specimens with A H1-2009, 37 specimens with A H3, and 3 specimens with no subtype detected. Influenza A was not detected in 1/3 FN samples using PCR/Sequencing. Two (2) FN specimens were not tested by PCR/Sequencing. Influenza A was detected in 1/4 FP specimens using   
PCR/Sequencing.   
f Influenza A H1-2009 was detected in $4 / 6$ FP specimens using PCR/Sequencing.   
g Influenza A H3 was not detected in 1/3 FN specimens using PCR/Sequencing.   
h Influenza B was not detected in 3/7 FN specimens using PCR/Sequencing. Influenza B was detected in 2/4 FP specimens using PCR/Sequencing. One (1) FP specimen was not tested by PCR/Sequencing   
i Parainfluenza virus 3 was not detected in 3/10 FN specimens using PCR/Sequencing. Parainfluenza virus 3 was detected in 4/5 FP specimens using PCR/Sequencing.   
j Parainfluenza virus 4 was detected in 3/5 FP specimens using PCR/Sequencing. 2 FP specimens were not tested by PCR/Sequencing.   
k RSV B was detected in 1/2 FP specimens using PCR/Sequencing.   
$^ { 1 } C .$ . pneumoniae was not detected in 1/3 FN specimens using PCR/Sequencing. C. pneumoniae was detected in the 1 FP specimen using PCR/Sequencing.   
m M. pneumoniae was not detected in the 1 FN specimen using PCR/Sequencing. M. pneumoniae was detected in the 1 FP specimen using PCR/Sequencing. One (1) FP specimen was not tested by PCR/Sequencing.

# Prospective Clinical Study Mixed Infection Analysis

The ePlex RP Panel identified a total of 135 prospective specimens with multiple organisms detected, or $5 . 5 \%$ (135/2462) of all prospective specimens. Of these, 118 (118/2462, $4 . 8 \%$ ) had two organisms, 14 (14/2462, $0 . 6 \%$ ) had three organisms, and $3 ( 3 / 2 4 6 2 , 0 . 1 \% )$ had four organisms detected. Of the 135 specimens with multiple organisms detected, 58 included one or more organisms that had not been detected by the comparator method (i.e., false positive results).

All distinct co-infection combinations as detected by the ePlex RP Panel during the prospective clinical evaluation are presented in Table 37 below.

Table 37: Distinct Co-Detection Combinations Detected by the ePlex RP Panel in the Prospective Clinical Specimens   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by theePlex RP Panel</td><td colspan="1" rowspan="2">Total NumberOf Co-detections(% of samples)</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-detections</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) a</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">Flu A (unk)</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1), Flu A (unk)(1), Flu B (1),HRV/EV (1</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">Flu A H3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1), Flu B (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1), FluA H1-2009 (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV 3 (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0.12%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV (2)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18 (0.73%)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">ADV (6), HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">Mpneum</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV 1 (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1), PIV 4 (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">PIV 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV (2)</td></tr><tr><td colspan="1" rowspan="1">CPneum</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (0.16%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">12 (0.49%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">CoV (1), HRV/EV (4)</td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0.12%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0.12%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoV (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A (unk)</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Flu A (unk) (1)</td></tr><tr><td colspan="1" rowspan="1">Flu AH3</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu AH3</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (0.16%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">HRV/EV (2)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FluA09H1</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">FluA09H1</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0.08%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5 (0.20%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0.12%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7 (0.28%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">HRV/EV (1), PIV 2 (2)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11 (0.45%)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">HRV/EV (5)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (0.16%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">PIV 4 (4)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5 (0.20%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11 (0.45%)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HRV/EV (6)</td></tr><tr><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV 4 (1)</td></tr><tr><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.04%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="4" rowspan="1">Total Number of Co-Detections</td><td colspan="1" rowspan="1">135 (5.5%)</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">64/290 b</td></tr><tr><td colspan="4" rowspan="1">Total Number with 2 Organisms Detected</td><td colspan="1" rowspan="1">118 (4.8%)</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">49/236</td></tr><tr><td colspan="4" rowspan="1">Total Number with 3 Organisms Detected</td><td colspan="1" rowspan="1">14 (0.6%)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">8/42</td></tr><tr><td colspan="4" rowspan="1">Total Number with 4 Organisms Detected</td><td colspan="1" rowspan="1">3 (0.1%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7/12</td></tr></table>

Note: $\mathbf { A D V } =$ adenovirus, $\operatorname { C o V } =$ coronavirus, $\mathrm { H M P V = }$ human metapneumovirus, ${ \mathrm { H R V / E V } } =$ human rhinovirus/enterovirus, Flu $\equiv$ Influenza, (unk) $\mid =$ unknown subtype, $\mathrm { P I V } =$ parainfluenza, ${ \mathrm { R S V } } =$ respiratory syncytial virus, Cpneum ${ } = C .$ . pneumoniae, Mpneum $\mathsf { \Omega } = M .$ . pneumoniae

a A discrepant organism is defined as one that was detected by the ePlex RP Panel but not by the comparator method.

$^ { \mathrm { ~ b ~ } } 6 4$ discrepant organisms were investigated using PCR/Sequencing; the discrepant organism was detected in 20/64 cases:

-In 8/18 specimens, adenovirus was detected by PCR/Sequencing.   
-In 1/4 specimens, coronavirus was detected by PCR/Sequencing.   
-In 7/25 specimens, human rhinovirus/enterovirus was detected by PCR/Sequencing.   
-In 1/1 specimen, influenza A H1-2009 was detected by PCR/Sequencing.   
- $- \mathrm { I n } 1 / 1$ specimen, parainfluenza virus 3 was detected by PCR/Sequencing.   
-In 2/6 specimens, parainfluenza virus 4 was detected by PCR/Sequencing.

Additional distinct co-infection combinations detected by the comparator method, but not detected by the ePlex RP Panel in the prospective clinical evaluation are presented in Table 38 below.

Table 38: Additional Distinct Co-infection Combinations Detected by the Comparator Method, but not detected by the ePlex RP Panel in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=3>Distinct Co-Detection CombinationsDetectedby the Comparator Method</td><td rowspan=2 colspan=1>Total NumberOf Co-detections(% of samples)</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-detections</td><td rowspan=2 colspan=1>DiscrepantOrganism(s) a,b</td></tr><tr><td rowspan=1 colspan=1>Organism 1</td><td rowspan=1 colspan=1>Organism 2</td><td rowspan=1 colspan=1>Organism 3</td></tr><tr><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ADV (1), CoV (1)</td></tr><tr><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4 (0.16%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ADV (4)</td></tr><tr><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1), PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ADV (1)</td></tr><tr><td rowspan=1 colspan=1>CPneum</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CPneum (1)</td></tr><tr><td rowspan=1 colspan=1>CPneum</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CPneum (1)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>FluA H1-2009</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0.08%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV (2)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV (1)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6 (0.24%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CoV (4), HRV/EV (2)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV (1)</td></tr><tr><td rowspan=1 colspan=1>CoV</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3 (0.12%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>CoV (2), RSV B (1)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Flu A H3 (1), PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>FluA H1-2009</td><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HMPV (1), HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0.08%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>HRV/EV (2)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1 (0.04%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0.08%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>RSV A (2)</td></tr></table>

Note: $\mathbf { A D V } =$ adenovirus, $\operatorname { C o V } =$ coronavirus, $\mathrm { H M P V = }$ human metapneumovirus, ${ \mathrm { H R V / E V } } =$ human rhinovirus/enterovirus, Flu= Influenza, $\mathrm { P I V } =$ parainfluenza, ${ \mathrm { R S V } } =$ respiratory syncytial virus, Cpneum ${ \bf \Omega } = C .$ pneumoniae, Mpneum $\mathsf { \Omega } = M .$ . pneumoniae

a A discrepant organism is defined as one that was detected by the comparator method but not by the ePlex RP Panel. $^ \mathrm { ~ b ~ } 3 6 / 3 6$ discrepant organisms were investigated using PCR/Sequencing; the discrepant organism was not detected in 10/36 cases:

-In 2/6 samples, adenovirus was not detected by PCR/Sequencing.   
-In 1/2 samples, Chlamydophila pneumoniae was not detected by PCR/Sequencing.   
-In 1/11 samples, coronavirus was not detected by PCR/Sequencing.   
-In 5/8 samples, human rhinovirus/enterovirus was not detected by PCR/Sequencing.   
-In 1/1 sample, influenza A H3 was not detected by PCR/Sequencing.

# Retrospective Clinical Study

Some of the organisms on the ePlex RP Panel were of low prevalence and were not encountered in sufficiently large numbers during the prospective evaluation to adequately demonstrate system performance. To supplement the number of positives for targets that were not sufficiently represented in the prospective collection, additional nasopharyngeal swab in VTM specimens were retrospectively collected from six clinical sites.

A total of 535 nasopharyngeal swab in VTM specimens that had previously tested positive for one or more of the target organisms during standard-of-care (SOC) testing were collected and stored frozen. Prior to the start of investigational testing, 11 specimens were withdrawn due to incompliance to the study protocol, and 52 specimens were withdrawn because the organisms present had sufficient representation in other samples. In addition, the composition and integrity of the pre-selected specimens were also confirmed with the same comparator method employed in the prospective clinical study (i.e., an FDA-cleared multiplexed respiratory pathogen panel). As the result of this confirmation testing using the comparator method, 26 additional specimens were excluded because the original SOC testing positive results for the intended organisms were not confirmed when tested with the comparator method.

The remaining 446 retrospective specimens were eligible for testing in this study, all 446 specimens were evaluable. The retrospective specimens were distributed approximately equally across the same five testing sites that participated in the prospective evaluation of the ePlex RP Panel. Prospective and retrospective samples were intermixed and testing sites were blinded to the identity of the samples.

Demographic information for the 446 eligible retrospective specimens is described in Table 39 below.

Table 39: Subject Demographic Data for Retrospective Specimens by Collection Site $\scriptstyle ( \mathbf { N } = 4 4 6 )$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN=446n (%)</td><td rowspan=1 colspan=1>Site 1N=1n (%)</td><td rowspan=1 colspan=1>Site 2N=1n (%)</td><td rowspan=1 colspan=1>Site 3N=129n (%)</td><td rowspan=1 colspan=1>Site 4N=18n (%)</td><td rowspan=1 colspan=1>Site 5N=131n (%)</td><td rowspan=1 colspan=1>Site 6N=166n (%)</td></tr><tr><td rowspan=1 colspan=8>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>232 (52.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>76 (58.9)</td><td rowspan=1 colspan=1>11 (61.1)</td><td rowspan=1 colspan=1>68 (51.9)</td><td rowspan=1 colspan=1>76 (45.8)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>214 (48.0)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>53 (41.1)</td><td rowspan=1 colspan=1>7 (38.9)</td><td rowspan=1 colspan=1>63 (48.1)</td><td rowspan=1 colspan=1>90 (54.2)</td></tr><tr><td rowspan=1 colspan=8>Age (years)</td></tr><tr><td rowspan=1 colspan=1>0 − 1</td><td rowspan=1 colspan=1>122 (27.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>24 (18.6)</td><td rowspan=1 colspan=1>5 (27.8)</td><td rowspan=1 colspan=1>56 (42.7)</td><td rowspan=1 colspan=1>37 (22.3)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>107 (24.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>51 (39.5)</td><td rowspan=1 colspan=1>3 (16.7)</td><td rowspan=1 colspan=1>16 (12.2)</td><td rowspan=1 colspan=1>36 (21.7)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>59 (13.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (7.0)</td><td rowspan=1 colspan=1>2 (11.1)</td><td rowspan=1 colspan=1>19 (14.5)</td><td rowspan=1 colspan=1>29 (17.5)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>99 (22.2)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>11 (8.5)</td><td rowspan=1 colspan=1>8 (44.4)</td><td rowspan=1 colspan=1>31 (23.7)</td><td rowspan=1 colspan=1>48 (28.9)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65</td><td rowspan=1 colspan=1>59 (13.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>34 (26.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (6.9)</td><td rowspan=1 colspan=1>16 (9.6)</td></tr></table>

# Retrospective Clinical Study Performance

The ePlex RP Panel retrospective performance data in positive percent and negative percent agreements against the comparator methods are presented in the following Table 39 by analyte:

Table 39: ePlex RP Panel Retrospective Clinical Study Performance Summary   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN +FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus a</td><td rowspan=1 colspan=1>55/56</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>90.6-99.7</td><td rowspan=1 colspan=1>386/390</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>97.4-99.6</td></tr><tr><td rowspan=1 colspan=1>Coronavirus b</td><td rowspan=1 colspan=1>121/138</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>81.2-92.2</td><td rowspan=1 colspan=1>307/307</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>71.4</td><td rowspan=1 colspan=1>35.9-91.8</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>37/41</td><td rowspan=1 colspan=1>90.2</td><td rowspan=1 colspan=1>77.5-96.1</td><td rowspan=1 colspan=1>384/402</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>93.0-97.1</td></tr><tr><td rowspan=1 colspan=1>Influenza A d</td><td rowspan=1 colspan=1>75/82</td><td rowspan=1 colspan=1>91.5</td><td rowspan=1 colspan=1>83.4-95.8</td><td rowspan=1 colspan=1>363/363</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>446/446</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009 e</td><td rowspan=1 colspan=1>27/31</td><td rowspan=1 colspan=1>87.1</td><td rowspan=1 colspan=1>71.1-94.9</td><td rowspan=1 colspan=1>415/415</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>45/51</td><td rowspan=1 colspan=1>88.2</td><td rowspan=1 colspan=1>76.6-94.5</td><td rowspan=1 colspan=1>394/394</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>445/445</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1 g</td><td rowspan=1 colspan=1>43/48</td><td rowspan=1 colspan=1>89.6</td><td rowspan=1 colspan=1>77.8-95.5</td><td rowspan=1 colspan=1>396/397</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.6-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>46/51</td><td rowspan=1 colspan=1>90.2</td><td rowspan=1 colspan=1>79.0-95.7</td><td rowspan=1 colspan=1>395/395</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>444/444</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>69.9-97.2</td><td rowspan=1 colspan=1>426/426</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>25/27</td><td rowspan=1 colspan=1>92.6</td><td rowspan=1 colspan=1>76.6-97.9</td><td rowspan=1 colspan=1>414/414</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>RSV Bh</td><td rowspan=1 colspan=1>21/22</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>78.2-99.2</td><td rowspan=1 colspan=1>419/419</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>445/445</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae </td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100</td></tr></table>

<table><tr><td>a Adenovirus was not detected in the 1 FN specimen using PCR/Sequencing. Adenovirus was detected in 2/4 FP specimens using PCR/Sequencing. bn (1) unintendecoronavirus positivespecimen by the sourc laboratory was not confirmed by the comparator method and therefore was excluded from the performance calculation for coronavirus. Coronavirus was not detected in 2/16 FN specimens using PCR/Sequencing. One (1) FN specimen was not tested by PCR/Sequencing.</td></tr><tr><td>cThree (3)unintended HRV/EV positiv specimens by the source laboratory were not confirmed by the comparator method and therefore were excluded from the performance calculation for HRV/EV. O () uendefuenzaA posiv sece byhe souraboratory was ot conimed byhe copa meod andthereore was excluderom h perora calculation olunza. Influenza was not detee in 3/7 FN specimens using PCR/Sequencing. Influenza A comparator results contain 31 specimens with A H1-2009</td></tr><tr><td>and 51 specimens with A H3 detected. eInfluenza A H1-2009 was not detected in 2/4 FN specimens using PCR/Sequencing. fOne (1) unintended influenza A H3 positive specimen by the source laboratory was not confirmed by the coparator method and therefore was excluded from the performance calculation or influenza A H3. InfluenzaA</td></tr><tr><td>H3 was not detected in 1/6 FN specimens using PCR/Sequencing. O (nitended   posiv een y e our laoratoy was ot cnfirme b he cparatored</td></tr><tr><td>and therefore was excluded from the performance calculation for PIV 1. PIV 1 was not detected in 2/5 FN specimens using PCR/Sequencing. h Five (5) unintended RSV positive specimens by the comparator method were not confirmed by PCR/Sequencing with regard to determining RSV subtypes and therefore was excluded from the performance calculations for RSV</td></tr></table>

# Mock Clinical Study

Influenza A H1 and Chlamydia pneumoniae are of such rarity that both prospective and retrospective testing efforts were insufficient to demonstrate system performance for these two organisms. To supplement the prospective and retrospective data, an evaluation of contrived specimens was performed at the same five testing sites that participated in the prospective and retrospective evaluations of the ePlex RP Panel. Contrived clinical specimens were prepared using individual unique negative residual clinical NPS in VTM specimens. For each target, at least 50 individual specimens were prepared with half of the samples spiked at approximately 2 -10x LoD and the remaining samples spiked with target spanning the expected clinical range. Contrived samples were tested along with other clinical samples and/or contrived samples that were negative for the corresponding target.

There were 327 contrived specimens created and tested to supplement low prevalence targets on the RP Panel; 104 specimens contained one or more low prevalence organisms (i.e., Influenza A H1 and Chlamydia pneumoniae) and 223 specimens were negative for these organisms. All 327 contrived specimens were tested with the ePlex RP Panel and 326 were evaluable. The one unevaluable specimen did not have a final, valid ePlex RP Panel result after repeat testing.

Mock clinical study results are summarized for these two low prevalence organisms in Table 40 below.

Table 40: ePlex RP Panel Performance Testing Contrived Specimens   

<table><tr><td rowspan="2">Organism</td><td colspan="4">Positive Percent Agreement (PPA)</td><td colspan="3">Negative Percent Agreement (NPA)</td></tr><tr><td>× LoD</td><td>TP/(TP + FN)</td><td>%</td><td>95% CI</td><td>TN/(TN + FP)</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="5">Influenza A H1</td><td>10</td><td>25/25</td><td>100</td><td>86.7-100</td><td rowspan="5">275/275</td><td rowspan="5">100</td><td rowspan="5">98.6-100</td></tr><tr><td>50</td><td>8/8</td><td>100</td><td>67.6-100</td></tr><tr><td>100</td><td>9/9</td><td>100</td><td>70.1-100</td></tr><tr><td>1000</td><td>9/9</td><td>100</td><td>70.1-100</td></tr><tr><td>Combined</td><td>51/51</td><td>100</td><td>93.0-100</td></tr><tr><td rowspan="5">Chlamydia pneumoniae</td><td>2</td><td>25/25</td><td>100</td><td>86.7-100</td><td rowspan="5">274/274</td><td rowspan="5">100 98.6-100</td></tr><tr><td>5</td><td>9/9</td><td>100</td><td>70.1-100</td></tr><tr><td>10</td><td>9/9</td><td>100</td><td>70.1-100</td></tr><tr><td>100</td><td>9/9</td><td>100</td><td>70.1-100</td></tr><tr><td>Combined</td><td>52/52</td><td>100</td><td>93.1-100</td></tr></table>

# Clinical Study ePlex Instrument Performance

There were 3242 specimens eligible for ePlex RP Panel testing (i.e., 2469 prospective specimens, 446 retrospective specimens, and 327 contrived specimens) in the clinical evaluations of the ePlex RP Panel. Influenza A with no subtype was detected in 11 specimens. These 11 specimens underwent an additional ePlex RP Panel test per the product instructions for use to identify the subtype. In addition, 28 specimens that had been withdrawn were inadvertently sent for investigational testing. In total, there were 3281 specimens initially tested using the ePlex RP Panel.

Of the 3281 specimens initially tested, 3127 $( 9 5 . 3 \%$ , $9 5 \%$ CI: $9 4 . 5 \% - 9 6 . 0 \% )$ had valid results and 154 $( 4 . 7 \%$ , $9 5 \%$ CI: $4 . 0 \% 6 - 5 . 5 \%$ had invalid results. Initial validity rates were similar across the five testing sites. The 154 initially invalid specimens were re-tested per the product instructions for use at the same or a different testing site and 146 had valid results. The final validity rate was $9 9 . 8 \%$ (3273/3281) $9 5 \%$ CI: $9 9 . 5 \% . 9 9 . 9 \%$ ). Final validity rates were also similar across the testing sites.

Of the 162 invalid results (154 initial invalid and 8 re-test invalid), 161 were due to an assay algorithm failure and one was due to operator error. For the one invalid due to an operator error, the cartridge ejected due to a pre-flight failure but the operator did not notice until the following morning. The cartridge was re-inserted and testing completed; however, the sample was in the cartridge for longer than indicated in the product instructions for use so the initial result was invalidated by the operator and the specimen was re- tested.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex RP Panel in nasopharyngeal swab in VTM specimens. 2462 prospective specimens were collected at eight collection sites in two phases from patients of all ages and genders presenting with signs and/or symptoms of respiratory infection. In the first phase from March 2013 through August 2014, 1951 prospective specimens were collected and frozen; from September 2016 through October 2016, 511 prospective specimens were collected and tested fresh (never frozen). The expected values of individual analytes based on ePlex RP Panel results in prospective specimens for each phase are summarized in Table 41 to Table 44 below.

Table 41: Expected Value (As Determined by ePlex RP Panel) Summary by Age Group in the Prospective Clinical Evaluation (Phase 1: March 2013 – August 2014)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>All Ages(N=1951)n (%)</td><td rowspan=1 colspan=1>Age 0-1(N=315)n (%)</td><td rowspan=1 colspan=1>Age &gt;1-5(N=250)n (%)</td><td rowspan=1 colspan=1>Age &gt;5-21(N=246)n (%)</td><td rowspan=1 colspan=1>Age &gt;21-65(N=745)n (%)</td><td rowspan=1 colspan=1>Age &gt;65(N=395)n (%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>72 (3.7)</td><td rowspan=1 colspan=1>31 (9.8)</td><td rowspan=1 colspan=1>24 (9.6)</td><td rowspan=1 colspan=1>7 (2.8)</td><td rowspan=1 colspan=1>7 (0.9)</td><td rowspan=1 colspan=1>3 (0.8)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>102 (5.2)</td><td rowspan=1 colspan=1>19 (6.0)</td><td rowspan=1 colspan=1>18 (7.2)</td><td rowspan=1 colspan=1>16 (6.5)</td><td rowspan=1 colspan=1>32 (4.3)</td><td rowspan=1 colspan=1>17 (4.3)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>113 (5.8)</td><td rowspan=1 colspan=1>22 (7.0)</td><td rowspan=1 colspan=1>28 (11.2)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>31 (4.2)</td><td rowspan=1 colspan=1>26 (6.6)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>388 (19.9)</td><td rowspan=1 colspan=1>113 (35.9)</td><td rowspan=1 colspan=1>94 (37.6)</td><td rowspan=1 colspan=1>58 (23.6)</td><td rowspan=1 colspan=1>87 (11.7)</td><td rowspan=1 colspan=1>36 (9.1)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>110 (5.6)</td><td rowspan=1 colspan=1>6 (1.9)</td><td rowspan=1 colspan=1>18 (7.2)</td><td rowspan=1 colspan=1>20 (8.1)</td><td rowspan=1 colspan=1>49 (6.6)</td><td rowspan=1 colspan=1>17 (4.3)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>76 (3.9)</td><td rowspan=1 colspan=1>4(1.3)</td><td rowspan=1 colspan=1>13 (5.2)</td><td rowspan=1 colspan=1>14 (5.7)</td><td rowspan=1 colspan=1>37 (5.0)</td><td rowspan=1 colspan=1>8 (2.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>34 (1.7)</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>5 (2.0)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>12 (1.6)</td><td rowspan=1 colspan=1>10 (2.5)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>62 (3.2)</td><td rowspan=1 colspan=1>4(1.3)</td><td rowspan=1 colspan=1>9 (3.6)</td><td rowspan=1 colspan=1>10 (4.1)</td><td rowspan=1 colspan=1>24 (3.2)</td><td rowspan=1 colspan=1>15 (3.8)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>24 (1.2)</td><td rowspan=1 colspan=1>4(1.3)</td><td rowspan=1 colspan=1>12 (4.8)</td><td rowspan=1 colspan=1>4 (1.6)</td><td rowspan=1 colspan=1>3 (0.4)</td><td rowspan=1 colspan=1>1 (0.3)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>10 (0.5)</td><td rowspan=1 colspan=1>4 (1.3)</td><td rowspan=1 colspan=1>4 (1.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>99 (5.1)</td><td rowspan=1 colspan=1>31 (9.8)</td><td rowspan=1 colspan=1>20 (8.0)</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>27 (3.6)</td><td rowspan=1 colspan=1>18 (4.6)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>7(0.4)</td><td rowspan=1 colspan=1>3 (1.0)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>28 (1.4)</td><td rowspan=1 colspan=1>13 (4.1)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>4 (1.0)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>83 (4.3)</td><td rowspan=1 colspan=1>33 (10.5)</td><td rowspan=1 colspan=1>19 (7.6)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>15 (2.0)</td><td rowspan=1 colspan=1>10 (2.5)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>3 (0.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>1 (0.3)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>5 (0.3)</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 42: Expected Value (As Determined by ePlex RP Panel) Summary by Age Group in the Prospective Clinical Evaluation (Phase 2: September 2016 – October 2016)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>All Ages(N=511)n (%)</td><td rowspan=1 colspan=1>Age 0-1(N=73)n (%)</td><td rowspan=1 colspan=1>Age &gt;1-5(N=75)n (%)</td><td rowspan=1 colspan=1>Age &gt;5-21(N=75)n (%)</td><td rowspan=1 colspan=1>Age &gt;21-65(N=181)n (%)</td><td rowspan=1 colspan=1>Age &gt;65(N=107)n (%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>10 (2.0)</td><td rowspan=1 colspan=1>3 (4.1)</td><td rowspan=1 colspan=1>4 (5.3)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>1 (0.9)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>2 (2.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>4 (2.2)</td><td rowspan=1 colspan=1>1 (0.9)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>188 (36.8)</td><td rowspan=1 colspan=1>37 (50.7)</td><td rowspan=1 colspan=1>40 (53.3)</td><td rowspan=1 colspan=1>33 (44.0)</td><td rowspan=1 colspan=1>58 (32.0)</td><td rowspan=1 colspan=1>20 (18.7)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>13 (2.5)</td><td rowspan=1 colspan=1>3 (4.1)</td><td rowspan=1 colspan=1>4 (5.3)</td><td rowspan=1 colspan=1>3 (4.0)</td><td rowspan=1 colspan=1>2 (1.1)</td><td rowspan=1 colspan=1>1 (0.9)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5 (1.0)</td><td rowspan=1 colspan=1>2 (2.7)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>1 (1.4)</td><td rowspan=1 colspan=1>4 (5.3)</td><td rowspan=1 colspan=1>2 (2.7)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>5 (6.8)</td><td rowspan=1 colspan=1>3 (4.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9 (1.8)</td><td rowspan=1 colspan=1>3 (4.1)</td><td rowspan=1 colspan=1>4 (5.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>2 (2.7)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 43: Expected Value (As Determined by ePlex RP Panel) Summary by Specimen Collection Site in the Prospective Clinical Evaluation (Phase 1: March 2013 – August 2014)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>All Sites(N=1951)n (%)</td><td rowspan=1 colspan=1>Site 1(N=165)n (%)</td><td rowspan=1 colspan=1>Site 2(N=248)n (%)</td><td rowspan=1 colspan=1>Site 3(N=350)n (%)</td><td rowspan=1 colspan=1>Site 4(N=892)n (%)</td><td rowspan=1 colspan=1>Site 5(N=296)n (%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>72 (3.7)</td><td rowspan=1 colspan=1>4 (2.4)</td><td rowspan=1 colspan=1>8 (3.2)</td><td rowspan=1 colspan=1>28 (8.0)</td><td rowspan=1 colspan=1>23 (2.6)</td><td rowspan=1 colspan=1>9 (3.0)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>102 (5.2)</td><td rowspan=1 colspan=1>8 (4.8)</td><td rowspan=1 colspan=1>11 (4.4)</td><td rowspan=1 colspan=1>32 (9.1)</td><td rowspan=1 colspan=1>29 (3.3)</td><td rowspan=1 colspan=1>22 (7.4)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>113 (5.8)</td><td rowspan=1 colspan=1>10 (6.1)</td><td rowspan=1 colspan=1>23 (9.3)</td><td rowspan=1 colspan=1>27 (7.7)</td><td rowspan=1 colspan=1>30 (3.4)</td><td rowspan=1 colspan=1>23 (7.8)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>388 (19.9)</td><td rowspan=1 colspan=1>27 (16.4)</td><td rowspan=1 colspan=1>33 (13.3)</td><td rowspan=1 colspan=1>61 (17.4)</td><td rowspan=1 colspan=1>185 (20.7)</td><td rowspan=1 colspan=1>82 (27.7)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>110 (5.6)</td><td rowspan=1 colspan=1>5 (3.0)</td><td rowspan=1 colspan=1>21 (8.5)</td><td rowspan=1 colspan=1>48 (13.7)</td><td rowspan=1 colspan=1>19 (2.1)</td><td rowspan=1 colspan=1>17 (5.7)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>76 (3.9)</td><td rowspan=1 colspan=1>3 (1.8)</td><td rowspan=1 colspan=1>22 (8.9)</td><td rowspan=1 colspan=1>31 (8.9)</td><td rowspan=1 colspan=1>5 (0.6)</td><td rowspan=1 colspan=1>15 (5.1)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>34 (1.7)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>18 (5.1)</td><td rowspan=1 colspan=1>12 (1.3)</td><td rowspan=1 colspan=1>2 (0.7)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>62 (3.2)</td><td rowspan=1 colspan=1>9 (5.5)</td><td rowspan=1 colspan=1>9 (3.6)</td><td rowspan=1 colspan=1>9 (2.6)</td><td rowspan=1 colspan=1>19 (2.1)</td><td rowspan=1 colspan=1>16 (5.4)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>24 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (1.4)</td><td rowspan=1 colspan=1>2 (0.2)</td><td rowspan=1 colspan=1>17 (5.7)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>10 (0.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>99 (5.1)</td><td rowspan=1 colspan=1>13 (7.9)</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>28 (8.0)</td><td rowspan=1 colspan=1>41 (4.6)</td><td rowspan=1 colspan=1>14 (4.7)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>7 (0.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>4 (0.4)</td><td rowspan=1 colspan=1>2 (0.7)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>28 (1.4)</td><td rowspan=1 colspan=1>4 (2.4)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>7 (2.0)</td><td rowspan=1 colspan=1>4 (0.4)</td><td rowspan=1 colspan=1>7 (2.4)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>83 (4.3)</td><td rowspan=1 colspan=1>6 (3.6)</td><td rowspan=1 colspan=1>15 (6.0)</td><td rowspan=1 colspan=1>24 (6.9)</td><td rowspan=1 colspan=1>15 (1.7)</td><td rowspan=1 colspan=1>23 (7.8)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>3 (0.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>2 (0.2)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>5 (0.3)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.3)</td></tr></table>

Table 44: Expected Value (As Determined by ePlex RP Panel) Summary by Specimen Collection Site in the Prospective Clinical Evaluation (Phase 2: September 2016 – October 2016)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>All Sites(N=511)n (%)</td><td rowspan=1 colspan=1>Site 5(N=49)n (%)</td><td rowspan=1 colspan=1>Site 6(N=101)n (%)</td><td rowspan=1 colspan=1>Site 7(N=161)n (%)</td><td rowspan=1 colspan=1>Site 8(N=200)n (%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>10 (2.0)</td><td rowspan=1 colspan=1>2 (4.1)</td><td rowspan=1 colspan=1>3 (3.0)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>2 (1.0)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.0)</td><td rowspan=1 colspan=1>4 (2.5)</td><td rowspan=1 colspan=1>2 (1.0)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>188 (36.8)</td><td rowspan=1 colspan=1>24 (49.0)</td><td rowspan=1 colspan=1>49 (48.5)</td><td rowspan=1 colspan=1>62 (38.5)</td><td rowspan=1 colspan=1>53 (26.5)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>1 (2.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.5)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>13 (2.5)</td><td rowspan=1 colspan=1>2 (4.1)</td><td rowspan=1 colspan=1>4 (4.0)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>4 (2.0)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5 (1.0)</td><td rowspan=1 colspan=1>2 (4.1)</td><td rowspan=1 colspan=1>2 (2.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.5)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>1 (2.0)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>4 (2.5)</td><td rowspan=1 colspan=1>2 (1.0)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (7.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9 (1.8)</td><td rowspan=1 colspan=1>1 (2.0)</td><td rowspan=1 colspan=1>4 (4.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (2.0)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (3.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.5)</td></tr></table>

N. Instrument Name: ePlex Instrument

# O. System Descriptions:

1. Modes of Operation:

After samples have been loaded in the ePlex RP Panel cartridge, the remaining processing steps are executed under control of the ePlex instrument.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification: Specimen identification can be entered manually or via barcode.

4. Specimen Sampling and Handling:

The ePlex RP Panel is intended for use with nasopharyngeal swab (NPS) collected in VTM specimens. The operator removes an ePlex RP Panel cartridge pouch and one Sample Delivery Device from Kit packaging. The operator first writes the accession ID or place a barcode label with accession ID on the ePlex RP Panel cartridge and the Sample Delivery Device, and then pipette $2 0 0 \mu \mathrm { l }$ of specimen into the Sample Delivery Device using a calibrated pipette, close the Sample Delivery Device, and vortex it for 10 seconds. The operator subsequently removes the white cover from the tip of the Sample Delivery Device, invert the Sample Delivery Device and dispense the entire volume $( \sim 3 5 0 ~ \mu \mathrm { L } )$ by squeezing the vial and dispensing the drops into the sample loading port of the ePlex RP Panel cartridge. The operator then close the sample loading port by sliding the cap over the port and firmly pushing down on the cap to securely seal the sample delivery port, scan the ePlex RP Panel cartridge using the barcode reader provided with the ePlex instrument, and insert the cartridge into any available bay on the ePlex instrument, indicated by a flashing, white LED light. The test then begins automatically when the cartridge has been inserted into the bay and the pre-run check (cartridge initialization) is completed, indicated by a blue LED light.

5. Calibration:

Not applicable

6. Quality Control:

# Internal Controls

Each ePlex RP Panel cartridge includes internal controls that monitor performance of each step of the testing process. A DNA control verifies extraction, amplification and detection of DNA targets, and RNA controls verify amplification and detection of RNA targets. Each amplification reaction on the cartridge has at least one internal control and in each reaction either the internal control or a target must generate signal above the defined threshold for a valid test result. Internal control results are interpreted by the ePlex software and displayed on ePlex RP Panel Reports as Internal Control with a result of PASS, FAIL, N/A or INVALID.

# External Controls

External controls are not provided with the ePlex RP Panel. However, the sponsor is making the following recommendations regarding running external controls in the product package insert:

“Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Viral transport medium can be used as the negative control. Previously characterized positive samples or viral transport medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.”

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.