# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION

A. 510(k) Number: K163260   
B. Purpose for Submission: To obtain a substantial equivalence determination for the SeptiCyte LAB   
C. Measurand: Four mRNA transcript immune biomarkers: LAMP1, CEACAM4, PLA2G7, PLAC8   
D. Type of Test: Reverse transcription $^ +$ quantitative PCR (RT-qPCR)   
E. Applicant: Immunexpress, Inc.   
F. Proprietary and Established Names: SeptiCyte LAB

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3215; Device to detect and measure non-microbial analyte(s) in human clinical specimens to aid in assessment of patients with suspected sepsis

2. Classification: Class II (Special Controls)

3. Product code: PRE

4. Panel: 83: Microbiology

# H. Intended Use:

1. Intended use(s):

The SeptiCyte LAB is a gene expression assay using reverse transcription polymerase chain reaction to quantify the relative expression levels of host response genes isolated from whole blood collected in the PAXgene Blood RNA Tube. The SeptiCyte LAB is used in conjunction with clinical assessments and other laboratory findings as an aid to differentiate infection-positive (sepsis) from infection-negative systemic inflammation in patients suspected of sepsis on their first day of ICU admission. The SeptiCyte LAB generates a score (SeptiSCORE) that falls within one of four discrete Interpretation Bands based on the increasing likelihood of infection-positive systemic inflammation. The SeptiCyte LAB is intended for in-vitro diagnostic use.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): Blood must be collected in PAXgene Blood RNA Tube (K042613)

4. Special instrument requirements:

Applied Biosystems7500 FastDx Real-Time PCR Instrument QIAcube (optional, for automated RNA extraction) Spectrophotometer (e.g., Nanodrop)

# I. Device Description:

The SeptiCyte LAB is an in vitro diagnostic test for simultaneous amplification and detection of four RNA transcripts (LAMP1, CEACAM4, PLA2G7, PLAC8) using total RNA extracted from human blood. The test has been designed, manufactured, and validated for use on the ABI 7500 Fast Dx real-time thermal cycler. Each SeptiCyte LAB kit includes, in two boxes, the reagents sufficient for up to 12 patient samples.

The specimen used for the SeptiCyte LAB is a $2 . 5 ~ \mathrm { m L }$ sample of whole blood collected by venipuncture using the PAXgene collection tubes within the PAXgene Blood RNA System (Qiagen, kit catalogue # 762164; Becton Dickinson, Collection Tubes catalogue number 762165; FDA 510k number K042613). A white blood count (WBC) of $2 . 7 \dot { \mathrm { ~ X ~ } } 1 0 ^ { 5 } \mathrm { { W B C / m L } }$ or greater must be verified prior to testing patients.

Total RNA isolation is performed using the procedures specified in the PAXgene Blood RNA kit (a component of the PAXgene Blood RNA System). The purified total RNA must be evaluated for concentration (A260 indicating a concentration o $\mathrm { f } \geq 2 \mathrm { n g / \mu L }$ and purity (as estimated by A260/A280 ratio $\ge 1 . 6$ ). RNA specimens may need to be adjusted in concentration to facilitate a constant input volume of $1 0 ~ \mu \mathrm { L }$ . If purified total RNA is ${ < } 2 \ \mathrm { n g / }$ $\mu \mathrm { L }$ then the sample is too dilute to use. If purified total RNA concentration is ${ \tt > } 5 0 \ \mathrm { n g / \mu L }$ and $\leq 5 0 0 ~ \mathrm { n g / \mu L }$ , the specimen must be diluted 10-fold with SeptiCyte LAB Diluent ( $1 \mu \mathrm { L }$ RNA into $9 \mu \mathrm { L }$ Diluent) to $\leq 5 0 ~ \mathrm { n g / \mu L }$ to prevent exceeding $5 0 0 \mathrm { n g }$ total input into the RT reaction. If total RNA concentration is ${ > } 5 0 0 ~ \mathrm { n g / \mu L }$ , the specimen must be diluted 50-fold with SeptiCyte LAB Diluent ( $1 \mu \mathrm { L }$ RNA into $4 9 \mu \mathrm { L }$ Diluent). Patient RNA samples are normalized to concentrations between $2 \mathrm { n g / \mu L }$ to $5 0 ~ \mathrm { n g / \mu L }$ using SeptiCyte LAB Diluent to result in a 20 ng to $5 0 0 \mathrm { n g }$ of RNA input per RT reaction for the SeptiCyte LAB . Purified total RNA should be tested immediately after extraction or stored frozen in single-use portions at or below $- 7 0 \mathrm { { } ^ { \circ } C }$ until ready for testing.

The SeptiCyte LAB uses quantitative, real-time determination of the amount of each transcript in the sample based on the detection of fluorescence by the qPCR instrument (Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, Applied Biosystems, Foster City, CA, catalogue number 4406985; FDA 510k number K082562). The SeptiCyte LAB includes an RT step which converts the Extracted RNA to cDNA. The cDNA is immediately run in the qPCR portion of the test.

Transcripts CEACAM4, LAMP1, and PLAC8 are amplified, detected, and quantified in a multiplex reaction. There is a separate reaction (singleplex) well for PLA2G7. The reported score is calculated based on the difference between two pairwise combinations of the four markers, using SeptiCyte Analysis software which accompanies the SeptiCyte LAB . The software is designed to analyze .sds run files from the ABI 7500 Fast Dx instrument, compute the likelihood of sepsis for each patient sample, and generate a separate Patient Report for each sample.

The SeptiCyte LAB quantifies the gene expression levels of four host immune Response mRNA targets, to produce a quantitative score (the SeptiScore) ranging from 0 – 10. The SeptiScore is interpreted by means of four Interpretation Bands, which directly correlate with an increased likelihood of sepsis. The SeptiCyte LAB is a quantitative blood test that combines the results of four RNA transcripts (CEACAM4, LAMP1, PLA2G7, and PLAC8) as a single numerical result.

PAXgene Blood RNA tube extraction utilizes manual extraction with the IVD version of the QIAGEN PAXgene Blood RNA Kit. Both standard or accelerated RNA extraction can be performed. The standard PAXgene Blood RNA tube processing , which is performed according to manufacturer instructions, includes the 2 hour room temperature incubation prior to extraction while the accelerated processing does not include the 2 hour room temperature incubation prior to extraction. The median turnaround time for the SeptiCyte LAB including all steps from the start of the blood draw to the report generation, but excluding the standard 2 hour pre-incubation step was $4  { \mathrm { h } } 2 3  { \mathrm { m i n } }$ . If the 2 hour preincubation in included, the turnaround time results are about 6.5 hours. Semi-automated extraction PAXgene Blood RNA Tube, using the QIAGEN QIAcube System can also be utilized.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

# B·R·A·H·M·S PCT Sensitive KRYPTOR

2. Predicate 510(k) number(s): DEN150009

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=5>Predicate Comparison</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=3>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=3>SeptiCyte LAB(K163260)</td><td rowspan=1 colspan=1>B·R·A·H·M·S PCT Sensitive KRYPTOR(DEN150009)</td></tr><tr><td rowspan=3 colspan=1>Intended Use/Indications forUse</td><td rowspan=3 colspan=3>The SeptiCyte LAB is a geneexpression assay using reversetranscription polymerase chainreaction to quantify the relativeexpression levels of hostresponse genes isolated from wholeblood collected in the PAXgeneBlood RNA Tube. The SeptiCyteLAB is used in conjunction withclinical assessments and otherlaboratory findings as an aid todifferentiate infection-positive(sepsis) from infection-negativesystemic inflammation in patientssuspected of sepsis on their first dayof ICU admission. The SeptiCyteLAB generates a score(SeptiSCORE ) that falls within oneof four discrete Interpretation Bandsbased on the increasing likelihoodof infection-positive systemicinflammation. The SeptiCyte LAB isintended for in-vitro diagnostic use.</td><td rowspan=3 colspan=1>The B·R·A·H·M·S PCT sensitive KRYPTOR is animmunofluorescent assay using Time-ResolvedAmplified Cryptate Emission (TRACE) technologyto determine the concentration of PCT(procalcitonin) in human serum and EDTA orheparin plasma. The B·R·A·H·M·S PCT sensitiveKRYPTOR is intended to be performed on theB·R·A·H·M·S KRYPTOR analyzer family.The B·R·A·H·M·S PCT sensitive KRYPTOR isintended for use in conjunction with other laboratoryfindings and clinical assessments to aid in the riskassessment of critically ill patients on their first dayof Intensive Care Unit (ICU) admission forprogression to severe sepsis and septic shock.The B·R·A·H·M·S PCT sensitive KRYPTOR is alsointended for use to determine the change in PCTlevel over time as an aid in assessing the cumulative28-day risk of all-cause mortality in conjunctionwith other laboratory findings and clinicalassessments for patients diagnosed with severesepsis or septic shock in the ICU or when obtained inthe emergency deparent or other medical wards priorto ICU admission.Procalcitonin (PCT) is a biomarker associated withthe inflammatory response to bacterial infection thataids in the risk assessment of critically ill patients ontheir first day of Intensive Care Unit (ICU)admission for progression to severe sepsis and septicshock. The percent change in PCT level over timealso aids in the prediction of cumulative 28-daymortality in patients with severe sepsis and septicshock.PCT level on the first day of ICU admission above2.0 µg/L is associated with a higher risk forprogression to severe sepsis and/or septic shock thana PCT level below 0.5 μg/L.A PCT level that declines ≤ 80% from the day thatsevere sepsis or septic shock was clinicallydiagnosed (Day 0) to four days after clinicaldiagnosis (Day 4) is associated with highercumulative 28-day risk of all-cause mortality than adecline &gt; 80%. The combination of the PCT level (≤</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr></tr></table>

<table><tr><td rowspan=1 colspan=3>Predicate Comparison</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>SeptiCyte LAB(K163260)</td><td rowspan=1 colspan=1>B·R·A·H·M·S PCT Sensitive KRYPTOR(DEN150009)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.0 ug/L or &gt; 2.0 µg/L) at initial diagnosis of severesepsis or septic shock with the patient&#x27;s clinicalcourse and the change in PCT level over time untilDay 4 provides important additional informationabout the mortality risk. The PCT level on Day 1(the day after severe sepsis or septic shock is firstclinically diagnosed) can be used to calculate thepercent change in PCT level at Day 4 if the Day 0measurement is unavailable.</td></tr><tr><td rowspan=1 colspan=1>Intended UsePopulation</td><td rowspan=1 colspan=1>Critically ill patients on their first dayof Intensive Care Unit (ICU)admission</td><td rowspan=1 colspan=1>1. Critically ill patients on their first day of IntensiveCare Unit (ICU) admission;2. Patients in the emergency deparent or othermedical wards prior to ICU admission;3. Patients on day 0 to day 4 after clinical diagnosisof severe sepsis or septic shock.</td></tr><tr><td rowspan=1 colspan=1>Site of Use</td><td rowspan=1 colspan=1>To be performed by a trained operatorin a professional setting, such as ahospital central laboratory.</td><td rowspan=1 colspan=1>To be performed by a trained operator in aprofessional setting, such as a hospital centrallaboratory.</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>2.5 mL peripheral whole blood,collected in PAXgene Blood RNATube</td><td rowspan=1 colspan=1>Human serum, plasma (EDTA, heparin)</td></tr><tr><td rowspan=1 colspan=1>SpecimenProcessing</td><td rowspan=1 colspan=1>1. Manual extraction of material inPAXgene Blood RNA Tube, usingthe IVD version of the QIAGENPAXgene Blood RNA Kit2. Semi-automated extractionPAXgene Blood RNA Tube, usingthe QIAGEN QIAcube System</td><td rowspan=1 colspan=1>Fully automated, closed system (BRAHMSKRYPTOR analyzer) which can only operateutilizing custom reagents</td></tr><tr><td rowspan=1 colspan=1>Analyte(s)</td><td rowspan=1 colspan=1>Four mRNA transcript immunebiomarkers: LAMP1, CEACAM4,PLA2G7, PLAC8</td><td rowspan=1 colspan=1>Procalcitonin (PCT)</td></tr><tr><td rowspan=1 colspan=1>Assay Method</td><td rowspan=1 colspan=1>Reverse transcription + quantitativePCR (RT-qPCR)</td><td rowspan=1 colspan=1>Quantitative immunofluorescent assay</td></tr><tr><td rowspan=1 colspan=1>Assay Principle</td><td rowspan=1 colspan=1>Quantitative gene expression assay,based on real-time generation offluorescence from hydrolysis of dye-quencher hydrolysis probes duringcycles of PCR amplification ofnucleic acid templates</td><td rowspan=1 colspan=1>Sandwich immunoassay</td></tr><tr><td rowspan=1 colspan=1>DetectionMethod</td><td rowspan=1 colspan=1>Multi-channel fluorescence detection</td><td rowspan=1 colspan=1>Time-Resolved Amplified Cryptate Emission(TRACE) technology, which measures a specificfluorescence signature, with time delay, that isproportional to the antigen (PCT) concentration.</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>High Positive Control, Low PositiveControl and Negative Control foreach in vitro transcripts (LAMP1,CEACAM4, PLA2G7, PLAC8(IVTs) formulated in neutral buffer,with IVT concentrations designed toproduce high, medium or lowSeptiSCORE values</td><td rowspan=1 colspan=1>Two PCT controls formulated in defibrinated humanplasma</td></tr></table>

<table><tr><td></td><td rowspan=1 colspan=3>Predicate Comparison</td></tr><tr><td></td><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td></td><td rowspan=1 colspan=1>SeptiCyte LAB(K163260)</td><td rowspan=1 colspan=1>B·R·A·H·M·S PCT Sensitive KRYPTOR(DEN150009)</td></tr><tr><td></td><td rowspan=1 colspan=1>InstrumentPlatform</td><td rowspan=1 colspan=1>Applied Biosystems 7500 Fast DxReal-Time PCR Instrument</td><td rowspan=1 colspan=1>BRAHMS KRYPTOR analyzer family</td></tr><tr><td></td><td rowspan=1 colspan=1>Result Output</td><td rowspan=1 colspan=1>SeptiScore, calculated from theexpression levels of the four mRNAanalytes LAMP1, CEACAM4,PLA2G7, PLAC8, and describing therelative likelihood of sepsis vs.Systemic Inflammatory ResponseSyndrome (SIRS).</td><td rowspan=1 colspan=1>Calculated estimate of concentration of circulatingPCT, in units of ng/mL</td></tr><tr><td></td><td rowspan=1 colspan=1>ReagentStability</td><td rowspan=1 colspan=1>1. Reagents in original kit boxunopened at -20±5 °C: up to thestated expiration date;2. Reagents once opened: up to threefreeze/thaw cycles</td><td rowspan=1 colspan=1>1. In original shipping containers unopened at 2-8C: up to the stated expiration date;2. After opening, onboard at 2-8 °C: 29 days</td></tr><tr><td rowspan=1 colspan=2>Limitations</td><td rowspan=1 colspan=1>1. The diagnostic value of SeptiCyteLAB in the setting of sepsis has notbeen validated in US patients youngerthan 18.2. Not to be used if patient has WBCcount &lt; 2 x 105 cells/mL.3. For prescription use only.4. The SeptiCyte LAB should beevaluated in context of all laboratoryfindings and the total clinical status ofthe patient. In cases where thelaboratory results do not agree withthe clinical picture or history,additional tests should be performed.5. A signal is generated for PLAC8 inthe presence of gDNA and absence ofreverse transcriptase enzyme.6. If the RNA input per RT reactionfor the SeptiCyte LAB is not withinthe linear range (i.e., 20 ng (2 ng/uL)to 500ng (50 ng/uL) ), the controlswill indicate that the run passed evenif the input RNA for the sample is notwithin the linear range.</td><td rowspan=1 colspan=1>1. Citrate plasma tubes should not be used as PCTconcentrations are underestimated.2. The prognostic value of PCT in the setting ofsepsis has not been validated in US patients youngerthan 18.3. Improper handling of the reagents may falsify thetest results.4. Increased PCT levels may not always be related tosystemic infection. These conditions include, but arenot limited to:Patients experiencing major trauma and/or recentsurgical procedure including extracorporealcirculation or burns;Patients under treaent with OKT3 antibodies, OK-432, interleukins, TNF-alpha and other drugsstimulating the release of pro-inflammatorycytokines or resulting in anaphylaxis;Patients diagnosed with active medullary C-cellcarcinoma, small cell lung carcinoma, or bronchialcarcinoid;Patients with acute or chronic viral hepatitis and/ordecompensated severe liver cirrhosis (Child-PughClass C);Patients with prolonged or severe cardiogenicshock, prolonged severe organ perfusionanomalies or after resuscitation from cardiacarrest;Patients receiving peritoneal dialysis orhemodialysis treatment;Patients with biliary pancreatitis, chemicalpneumonitis or heat stroke;Patients with invasive fungal infections (e.g.,candidiasis, aspergillosis ) or acute attacks ofplasmodium falciparum malaria; andNeonates during the first 2 days of life.</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Predicate Comparison</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>SeptiCyte LAB(K163260)</td><td rowspan=1 colspan=1>B·R·A·H·M·S PCT Sensitive KRYPTOR(DEN150009)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5. The results of the B·R· A·H·M·S PCT sensitiveKRYPTOR should be evaluated in context of alllaboratory findings and the total clinical status of thepatient. In cases where the laboratory results do notagree with the clinical picture or history, additionaltests should be performed.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

<table><tr><td rowspan=1 colspan=1>Standard Number</td><td rowspan=1 colspan=1>Standard Name</td></tr><tr><td rowspan=1 colspan=1>EP5-A3</td><td rowspan=1 colspan=1>Evaluation of Precision of Quantitative MeasurementProcedures</td></tr><tr><td rowspan=1 colspan=1>EP07-A2</td><td rowspan=1 colspan=1>Interference Testing in Clinical Chemistry</td></tr><tr><td rowspan=1 colspan=1>EP17-A2</td><td rowspan=1 colspan=1>Evaluation of Detection Capability for ClinicalLaboratory Measurement Procedures</td></tr><tr><td rowspan=1 colspan=1>EP24-A2</td><td rowspan=1 colspan=1>Assessment of Diagnostic Accuracy of LaboratoryTests Using Receiver Operating Characteristic Curves</td></tr><tr><td rowspan=1 colspan=1>EP-25</td><td rowspan=1 colspan=1>Evaluation of Stability of In Vitro Diagnostic Reagents</td></tr><tr><td rowspan=1 colspan=1>BS EN ISO14971:2012</td><td rowspan=1 colspan=1>Medical devices — Application of risk management tomedical devices.</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Guidance</td></tr><tr><td rowspan=1 colspan=1>FDA Guidance on Off-the-Shelf Software Use in Medical Devices (issuedSeptember 9,1999)</td></tr><tr><td rowspan=1 colspan=1>General Principles of Software Validation; Final Guidance for Industry and FDAStaff (issued January 11, 2002)</td></tr><tr><td rowspan=1 colspan=1>FDA Guidance for the Content of Premarket Submissions for Software Containedin Medical Devices (issued 11 May 2005)</td></tr><tr><td rowspan=1 colspan=1>Code of Federal Regulations Title 21 Part 11, Electronic records; electronicsignatures</td></tr><tr><td rowspan=1 colspan=1>FDA Guidance for Industry Computerized Systems Used in ClinicalInvestigations (issued May 2007)</td></tr></table>

# L. Test Principle:

The test employs reverse transcription quantitative PCR (RT-qPCR) to measure relative expression levels of four host response genes (CEACAM4, LAMP1, PLA2G7, and PLAC8) from RNA extracted from the patient whole blood collected in the PAXgene Blood RNA collection tube.

The SeptiCyte Analysis Software analyzes the relative expression levels of the four genes resulting in a composite SeptiScore that ranges from 0 to 10, with a score of 10 indicating a higher probability of infection-positive inflammation (i.e. sepsis) as opposed to infectionnegative SIRS. The reported score is calculated based on the difference between two pairwise combinations of the four markers, using the interpretive software provided.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The intermediate precision and reproducibility of the SeptiCyte LAB was evaluated at three analytical laboratories using pooled blood samples. Results are summarized in Table 1A and 1B below. The blood sample pools were constructed to yield High $( \sim 9 )$ , Medium $( \sim 5 )$ , or Low $( \sim 0 . 5 )$ SeptiSCORE values. Identical replicates (prepared as aliquots from the blood sample pools) were tested in triplicate over five nonconsecutive days with multiple operators.

When intermediate precision was assessed at the Ct level, the SD was $< 1 . 3$ cycles and the CV was $< 7 \%$ . The SD for the SeptiSCORE was $\leq 0 . 3 7$ Score units. The CV, when measurable, for the SeptiSCORE was $\leq 1 0 \%$ .

Table 1A: Intermediate Precision of the SeptiCyte LAB Test   

<table><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Target orSeptiSCORE</td><td colspan="1" rowspan="1">SamplePool</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Mean Ct orSeptiSCORE</td><td colspan="1" rowspan="1">SD Ct orSeptiSCORE</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">22.16</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">5.18</td><td colspan="1" rowspan="1">0.50</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">22.09</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.42</td><td colspan="1" rowspan="1">0.14</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">21.05</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">0.05</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">23.93</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">5.29</td><td colspan="1" rowspan="1">0.55</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">24.06</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">23.81</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.48</td><td colspan="1" rowspan="1">0.15</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">29.54</td><td colspan="1" rowspan="1">1.22</td><td colspan="1" rowspan="1">4.13</td><td colspan="1" rowspan="1">0.53</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">27.63</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">24.75</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">1.02</td><td colspan="1" rowspan="1">0.11</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18.93</td><td colspan="1" rowspan="1">1.22</td><td colspan="1" rowspan="1">6.47</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20.12</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">0.09</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">21.11</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">0.09</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">8.84</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">4.18</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">5.53</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">5.87</td><td colspan="1" rowspan="1">0.14</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">0.11</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">19.66</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.81</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">19.64</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">0.06</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">19.36</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.03</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">0.15</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.20</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">0.08</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.31</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">0.08</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">26.59</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">0.13</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">24.85</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.95</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">0.14</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">16.17</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">0.08</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">17.66</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">0.09</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">19.59</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">1.42</td><td colspan="1" rowspan="1">0.17</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">8.05</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">3.24</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">4.62</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">4.03</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">22.09</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.87</td><td colspan="1" rowspan="1">0.18</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">22.35</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">2.11</td><td colspan="1" rowspan="1">0.21</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">21.50</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.93</td><td colspan="1" rowspan="1">0.19</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">23.94</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">2.22</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">24.40</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">2.52</td><td colspan="1" rowspan="1">0.27</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">24.35</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">2.08</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">28.94</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">27.38</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">24.88</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18.75</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.63</td><td colspan="1" rowspan="1">0.13</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20.30</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">1.85</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">21.52</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">1.59</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">8.35</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">3.50</td><td colspan="1" rowspan="1">0.13</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">5.03</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">6.25</td><td colspan="1" rowspan="1">0.14</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">0.11</td></tr></table>

$\ast _ { \mathrm { C V } }$ not calculated when mean SeptiSCORE value is close to zero due to inclusion of both positive and negative values.

When reproducibility was assessed at the $\mathrm { C t }$ level the SD was $< 1 . 4$ cycles and the CV was $< 8 \%$ . The SD for the SeptiSCORE was $\leq 0 . 5$ Score units. The CV, when measurable, for the SeptiSCORE was $< 1 0 \%$ .

Table1B: Reproducibility of the SeptiCyte LAB Test   

<table><tr><td colspan="1" rowspan="1">Target orSeptiSCORE</td><td colspan="1" rowspan="1">Sample Pool</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Mean Ct orSeptiSCORE</td><td colspan="1" rowspan="1">SD Ct orSeptiSCORE</td><td colspan="1" rowspan="1">CV (%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">21.64</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">5.81</td><td colspan="1" rowspan="1">0.35</td></tr><tr><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">21.70</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">5.09</td><td colspan="1" rowspan="1">0.31</td></tr><tr><td colspan="1" rowspan="1">CEACAM4</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">20.87</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">4.10</td><td colspan="1" rowspan="1">0.24</td></tr><tr><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">23.55</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">4.91</td><td colspan="1" rowspan="1">0.32</td></tr><tr><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">23.83</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">3.90</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">LAMP1</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">23.70</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">3.58</td><td colspan="1" rowspan="1">0.24</td></tr><tr><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">28.71</td><td colspan="1" rowspan="1">1.35</td><td colspan="1" rowspan="1">4.71</td><td colspan="1" rowspan="1">0.38</td></tr><tr><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">26.97</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">4.10</td><td colspan="1" rowspan="1">0.31</td></tr><tr><td colspan="1" rowspan="1">PLA2G7</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">24.43</td><td colspan="1" rowspan="1">0.81</td><td colspan="1" rowspan="1">3.33</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">18.31</td><td colspan="1" rowspan="1">1.34</td><td colspan="1" rowspan="1">7.31</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">19.70</td><td colspan="1" rowspan="1">1.07</td><td colspan="1" rowspan="1">5.42</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">PLAC8</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">20.95</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">3.70</td><td colspan="1" rowspan="1">0.22</td></tr><tr><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">HIGH</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">8.48</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">5.21</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">MEDIUM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">5.15</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">8.82</td><td colspan="1" rowspan="1">0.13</td></tr><tr><td colspan="1" rowspan="1">SeptiSCORE</td><td colspan="1" rowspan="1">LOW</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">*</td><td colspan="1" rowspan="1">0.09</td></tr></table>

$\ast _ { \mathrm { C V } }$ not calculated because SeptiSCORE values include both positive and negative values.

# b. Lot-toLot Precision Study Design:

The lot-to-lot component of the precision study was to determine the extent of variation in assay performance, due to differences in manufactured lots of the SeptiCyte LAB reagents. The study consisted of three Branches (A, B and C). Branch A analyzed 60 replicate blood samples from a single pool that was created by combining blood from nine healthy individuals. Branch B analyzed 60 replicate blood samples from each of three pools of clinical samples constructed to produce high, medium and low SeptiSCORE values, respectively. Branch C consisted of a two-part analysis. 50 replicates of purified healthy pool RNA were tested with each kit (Lot2 and Lot3) and 15 replicates of each of three mixtures of IVTs (Ct-Max,Ct-Mid,CtMin) formulated to give maximum, medium or minimum Ct values for all four transcripts were tested with kit Lot 1, Lot 2 and Lot 3. An additional study with 30 replicates using the purified healthy pool RNA used in Branch C was used to test three lots of reagents. were tested with kit Lots 1, 2 and 3.

No significant performance differences were detected for three lots of test kit reagents when used to test purified healthy pool RNA.

SeptiSCORE differences of ${ \leq } 0 . 8 8$ units are detected when reagent lots are tested with synthetic IVT mixtures. These differences were within the manufacturing tolerances for the SeptiCyte LAB reagents.

# c. Linearity/assay reportable range:

A panel of clinical specimens was tested over a comprehensive range of RNA input amounts (1000, 750, 600, 500, 400, 300, 200, 100, 50, and $2 0 \mathrm { n g }$ per RT reaction). The testing demonstrated a linear response for all SeptiCyte LAB transcripts at RNA input amounts up to 50 ng across a panel samples. The testing demonstrated a strong linear response for all SeptiCyte LAB transcripts at RNA input from $2 0 \mathrm { n g } \left( 2 \mathrm { n g / u L } \right)$ to $5 0 0 \mathrm { n g }$ $( 5 0 \mathrm { n g / u L } )$ across a panel samples. The results validated the reportable range of $2 0 \mathrm { n g } \left( 2 \mathrm { n g } / \mathrm { u L } \right)$ to $5 0 0 \mathrm { n g }$ ( $5 0 \mathrm { n g / u L ) }$ of RNA input per RT reaction for the SeptiCyte LAB .

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

Single tests each of the High Positive Controls (HPC), Low Positive Control (LPC) and Negative Control (NC), must be included in each run. The run is valid if no flags appear for any of these controls.   
• A single test of the No Template Control (NTC) is included in each run and No Score generated results in a valid result.   
• The SeptiCyte LAB High Positive Control, Low Positive Control and Negative Control must yield SeptiSCORE values correlating with the ranges listed in Table 2 below. If any of the SeptiCyte LAB Controls are out-ofrange, the entire batch run is invalid and an error report is generated.

The software compares each control specimen (HPC, LPC, NC) to its expected result. Controls are run as a single replicate.

Table 2: Control Expected Results   

<table><tr><td rowspan=1 colspan=1>Designation</td><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>Expected result</td></tr><tr><td rowspan=1 colspan=1>HPC</td><td rowspan=1 colspan=1>High PositiveControl</td><td rowspan=1 colspan=1>Band 4 (Score ≥8.0)</td></tr><tr><td rowspan=1 colspan=1>LPC</td><td rowspan=1 colspan=1>Low PositiveControl</td><td rowspan=1 colspan=1>Band 2, 3 or 4 (6.5 ≤Score ≥ 3.1)</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Band 1 (Score &lt; 3.1)</td></tr><tr><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>No TemplateControl</td><td rowspan=1 colspan=1>Pass (no ScoreGenerated)</td></tr></table>

If all Controls are valid, then the batch run is valid and specimen results are reported.

If HPC, LPC, NC and/or NTC fail, the batch run is invalid and no data is reported for the clinical specimens and an error report is generated. This determination is made automatically by the interpretive software. The user confirms by visual inspection of the run data. For this failure, the batch run is repeated starting with either a new RNA preparation or starting at the RT reaction step.

In the clinical study, each plate contained a full set of controls HPC, LPC, NC, NTC.   
All controls passed for all runs.

If the RNA input per RT reaction for the SeptiCyte LAB is not within the linear range (i.e., $2 0 \mathrm { n g } \left( 2 \mathrm { n g } / \mathrm { u L } \right)$ to $5 0 0 \mathrm { n g } \left( 5 0 \mathrm { n g } / \mathrm { u L } \right)$ ), the controls will indicate that the run passed even if the input RNA for the sample is not within the linear range.

# Real time stability

Three manufactured lots of SeptiCyte LAB were found to be stable for 16 months when stored under the label stated conditions ( $- 1 5 ^ { \mathrm { { \circ } } } \mathrm { C }$ to $- 3 0 ^ { \circ } \mathrm { C }$ ).

# Open vial stability

The SeptiCyte LAB kit consists of three component types: Base Reagents (BR), Target Specific Reagents (TSR), and Controls (HPC, LPC, NC). Freeze-thaw stability was assessed for each component type. This study validated all claims relating to open vial stability when the freeze-thaw occurs between a ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ freezer and ambient bench temperature:

BR Stability: The BRs in the SeptiCyte LAB kit, consisting of Diluent, RT Buffer, RT Enzyme Mix, qPCR Buffer and qPCR Enzyme Mix, are stable for are stable for up to 12 freeze-thaw cycles. TSR Stability: The TSRs in the SeptiCyte LAB kit, consisting of Primer/Probe Mix A, Primer/Probe Mix B, are stable up to four freeze-thaw cycles. • Test Control Stability: The SeptiCyte LAB Controls, consisting of HPC, LPC and NC, are stable for up to four freeze-thaw cycles.

# Shipping stability

Shipping stability testing consisted of testing kits under different shipping configurations and testing the shipping containers(shippers) for resistance to dropping and crushing. Human blood peripheral leukocyte RNA (Clontech 636592) and the SeptiCyte LAB controls (HPC, LPC, NC, and NTC) were used to assess the shipping stability. The SeptiCyte LAB is stable under real-time and simulated shipping conditions. The change in Ct values for shipping conditions tested was $\leq + / - 0 . 2 2$ cycles for all four transcripts. The change in SeptiSCORE was $\underline { { < } } \mathrm { + } / \mathrm { - } 0 . 5$ units.

# e. Detection limit:

To determine the LoD and LoQ, leukocyte-depleted blood (LDB) was spiked with serial dilutions of white blood cells (WBCs) and 20 replicates of each level of WBC tested with the SeptiCyte LAB Test. The lowest level at which 19/20 replicates generated a SeptiSCORE is reported as the LoD. The LoQ is the lowest level of WBC for which 19/20 replicates generate a SeptiSCORE with a standard deviation of 1.0 Score units or less. The LoD and LoQ were both determined to be $0 . 2 7 3 \mathrm { X } 1 0 ^ { 6 }$ WBC/mL blood. Additionally, a leukocyte depleted blood (LDB) sample was included and tested with 10 replicates to demonstrate that a SeptiSCORE would not be generated for any of the LDB replicates.

Table 3: Score Summary Statistics for LoD and LoQ (LoD and LoQ are shaded)   

<table><tr><td rowspan=1 colspan=1>WBC concentration(cells/mL blood)</td><td rowspan=1 colspan=1>TotalReplicatesScored</td><td rowspan=1 colspan=1>ScoreMean</td><td rowspan=1 colspan=1>ScoreSD</td><td rowspan=1 colspan=1>Score%CV</td></tr><tr><td rowspan=1 colspan=1>5.45 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6.89</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>2.67</td></tr><tr><td rowspan=1 colspan=1>2.73 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.42</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>3.04</td></tr><tr><td rowspan=1 colspan=1>1.82 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.46</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>3.18</td></tr><tr><td rowspan=1 colspan=1>0.909 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.30</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>3.32</td></tr><tr><td rowspan=1 colspan=1>0.455 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.49</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>4.42</td></tr><tr><td rowspan=1 colspan=1>0.273 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.30</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>5.29</td></tr><tr><td rowspan=1 colspan=1>0.091 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7.53</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>9.53</td></tr><tr><td rowspan=1 colspan=1>LDB</td><td rowspan=1 colspan=1>No Score</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# $f .$ Analytical specificity:

# Interfering Substances

Based on the CLSI Guidance EP07-A2, “Interference Testing in Clinical Chemistry” the SeptiCyte LAB was evaluated in the presence of interfering substances. Each substance was added to a blood sample in a PAXgene Blood RNA tube. No interference was found for any of the substances in Table 4 and Table 5 at the concentration listed.

Table 4: Interference Testing Levels   

<table><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Concentration in PAXgene Blood RNATube(pre- RNA extraction)</td></tr><tr><td rowspan=1 colspan=1>Rheumatoid Factor</td><td rowspan=1 colspan=1>45 IU/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>3000 U/L</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>1.18 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>20 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>500 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>69 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Cefotaxime</td><td rowspan=1 colspan=1>673 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Dopamine</td><td rowspan=1 colspan=1>5.87 μmol/L</td></tr><tr><td rowspan=1 colspan=1>CRP</td><td rowspan=1 colspan=1>4 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Noradrenaline orNorepinephrine</td><td rowspan=1 colspan=1>700 pg/mL</td></tr><tr><td rowspan=1 colspan=1>Dobutamine</td><td rowspan=1 colspan=1>11.2 µg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>20 g/dL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>5 g/dL</td></tr><tr><td rowspan=1 colspan=1>Furosemide</td><td rowspan=1 colspan=1>181 μmol/L</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>15 pg/mL</td></tr><tr><td rowspan=1 colspan=1>LBP</td><td rowspan=1 colspan=1>45 μg/mL</td></tr></table>

A preliminary study indicated that the SeptiSCORE for the following substances was significantly different (by Student’s t-test) from the control. These substances were retested at the following levels and resulted in no substantial effects across the range tested. (Note: The final amount of Albumin and Triglycerides was calculated by summing the endogenous levels present in the test blood and the exogenously added amount.)

Table 5: Additional Interferent Testing Levels   

<table><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Concentration in PAXgeneBlood RNA Tube(pre- RNA extraction)</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>2.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>1.25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>0.625 μg/mL</td></tr><tr><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>15 pg/mL</td></tr><tr><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>7.5 pg/mL</td></tr><tr><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>3.75 pg/mL</td></tr><tr><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>1.875 pg/mL</td></tr><tr><td rowspan=1 colspan=1>LBP</td><td rowspan=1 colspan=1>45 μg/mL</td></tr><tr><td rowspan=1 colspan=1>LBP</td><td rowspan=1 colspan=1>22.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>LBP</td><td rowspan=1 colspan=1>11.25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>LBP</td><td rowspan=1 colspan=1>5.625 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>5 g/dL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>2.5 g/dL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>1.25 g/dL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>0.625 g/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>500 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>375 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>250 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>125 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>62.5 mg/dL</td></tr></table>

# Integrity of the No Template Control (NTC)

Contamination of any general-purpose test components (e.g., with human cellular RNA), or non-template-directed cross-reactivity of primers and probes could generate a defined Ct value. Accordingly, this study investigated the performance of the SeptiCyte LAB when challenged with 40 replicates of the NTC. Diluent consists of purified water that contains no deoxyribonuclease (DNAse) or ribonuclease (RNAse). The panel also included controls (HPC, LPC and NC) which were used to verify SeptiCyte LAB kit function. All three controls were correctly determined and all NTCs (i.e., $4 0 / 4 0 )$ had an “undetermined” Ct value for each template; meaning no Ct values and no SeptiSCORE were obtained. Results demonstrated that the SeptiCyte LAB does not generate Ct values or a SeptiSCORE when NTC (i.e., a blank) is processed by the test.

# Fluorescence Channel Specificity

To demonstrate that the primers and probes for each transcript (CEACAM4, LAMP1, PLAC8) were not cross-detected in the neighboring three fluorescence channels used by the other multiplexed transcripts in the SeptiCyte LAB , testing of In-vitro transcripts (IVTs) corresponding to each of the four targets was prepared by in-vitro transcription from synthetic gBlocks DNA templates (Integrated DNA Technologies). Testing for PLA2G7 was also performed although this was considered supplementary as the PLA2G7 reaction is singleplex in a separate reaction well. Concentrations of in-vitro transcripts within the stock preparations were calculated in terms of copies $\mu \mathrm { L }$ , using the theoretical transcript molecular weight and the measured RNAconcentration $( \mathrm { n g / \mu L } )$ . Stock preparations of each IVT product were formulated in an inert matrix consisting of polyadenylicacid(poly(A)). For this study, poly(A) was used as matric as it mimics the polyelectrolyte and chemical composition of clinical RNA samples and is a simple homopolymer and would not introduce a risk of sequence-specific cross-reactivity.

Each IVT was tested both in the triplex reaction (Primer/ProbeMixA,specific for CEACAM4,LAMP1, PLAC8) and in the simplex reaction (Primer/ProbeMix B, specific for PLA2G7). Ten replicates of each reaction were run at $\mathrm { 1 X 1 0 ^ { 6 } }$ IVT copies input per reaction.

All controls (HPC, LPC, NC, NTC) were correctly determined. In all cases (40/40) when an IVT was correctly paired with the corresponding Primer/ProbeMix, a single Ct value was observed in the expected channel. No Ct value’s ( 0/120) were observed in an inappropriate reaction or channel.

The study demonstrated that the primers and probes for the individual transcripts (CEACAM4, LAMP1, PLAC8) are not cross-detected in the neighboring three fluorescence channels used by the other multiplexed transcripts in the SeptiCyte LAB Test. Additionally, the presence of poly(A) carrier does not lead to detectable signal in any fluorescent channel.

# Absence of ReactivityToward Genomic DNA

Genomic DNA(gDNA) samples were tested with and without reverse transcriptase enzyme to evaluate nonspecificity of the SeptiCyte LAB with respect to the requirement for RNA template and absence of reactivity towards gDNA. Studies demonstrated that the SeptiCyte LAB does not generate a SeptiSCORE when gDNA is used as the sole source of input material for the test.

Testing was conducted over two days and included 10 replicates of the two commercially-procured total human gDNA samples, each tested at $1 0 \mathrm { n g }$ and $1 0 0 \mathrm { n g }$ input per RT reaction. Each sample was tested with and without reverse transcriptase enzyme in the RT reaction, followed by qPCR using either theTriplex Primer/Primer Mix A (specific for CEACAM4,LAMP1, PLAC8) or the Simplex Primer/Probe MixB (specific for PLA2G7) at the qPCRstep. HPC, LPC and NC controls were included to verify SeptiCyte LAB kit function. NTC controls were included for detection of potential contamination and carry-over events.

Results demonstrated that controls performed as expected depending on the presence or absence of reverse transcriptase enzyme. No SeptiSCORE was determined when gDNA was used as input for all conditions tested.

When gDNA was used as a template, although no SeptiSCORE could be calculated, individual Ct values were observed for PLAC8. Further investigation was conducted. The EP400NL pseudogene was experimentally tested for activity in the SeptiCyte™ LAB as it has significant homology to the PLAC8 coding sequence. Pure EP400NL DNA in the form of agBlock double-stranded template was used as input in the SeptiCyte LAB . No Ct for the PLAC8 reaction was detected, when up to $1 0 ^ { 8 }$ copies of the EP400NLg Block template was used as input to the SeptiCyte LAB.

Additionally, while the EP400NL pseudogene has significant homology to the PLAC8 coding sequence, the production of Ct values from the EP400NL pseudogene would require both amplification and probe binding across mismatches. In-silico sequence analysis determined this cause to be unlikely.

Although it remains unclear why a signal is generated for PLAC8 in the presence of gDNA and absence of reverse transcriptase enzyme, this occurrence impacts neither the normal use of the SeptiCyte LAB . Normal use of the test includes reverse transcriptase and the amount of residual gDNA is expected to be lower than $1 0 ~ \mathrm { n g }$ per RT reaction. In addition,the other three transcripts (LAMP1, CEACAM4, PLA2G7) were not detected. The SeptiCyte Analysis software is designed so that the presence of an acceptable Ct value for all four transcripts is needed to generate a SeptiSCORE.

In the absence of reverse transcriptase, the study found that PLAC8 reactions yielded high Ct values $( > 3 1$ cycles) at gDNA inputs of either 10 ng or100 ng. The likely cause of low level PLAC8 detection in the presence of gDNA template and absence of reverse transcriptase enzyme remains unresolved. However, the reverse transcriptase should never be omitted from the recommended assay procedure. A message of an invalid result would be returned when only the PLAC8 transcript was detected.

A limitation was added to the labeling describing that a signal is generated for PLAC8 in the presence of gDNA and absence of reverse transcriptase enzyme.

# Requirement for Reverse Transcriptase

Four RNA samples were analyzed by the SeptiCyte LAB with and without reverse transcriptase in order to evaluate the requirement for reverse transcriptase enzyme. The study included 10 replicates of the four RNAs tested at $1 0 0 0 \mathrm { n g }$ RNA input per RT reaction. Each sample was tested with and without the SeptiCyte RT Enzyme mix followed by the usual qPCR step (i.e., Triplex Primer/Primer Mix A (CEACAM4,LAMP1,PLAC8) and Simplex Primer/Probe Mix B (PLA2G7)). HPC, LPC and NC were included to verify the SeptiCyte LAB function. The study

demonstrated that the SeptiCyte LAB does not generate a SeptiSCORE for HPC, LPC or NC controls or from clinical specimens, when reverse transcriptase is omitted from the RT step of the test.

# g. Recovery:

The validated sample type for SeptiCyte LAB is a $2 . 5 ~ \mathrm { m L }$ peripheral blood sample collected in a PAXgene Blood RNA tube. The recovery study determined the variability in RNA yield for this collection tube when processed from different collection sites and when processed either with the inclusion or omission of a twohour pre-incubation step, before the extraction step. The study also determined the percentage of PAXgene blood RNA samples that will yield sufficient RNA to run the test. The specimens included in the recovery study were blood samples collected in PAXgene Blood RNA tubes. Six groups of samples were evaluated and demonstrated the following results:

Table 6: Summary of Recovery Study Results   

<table><tr><td rowspan=1 colspan=1>Sample Set Description</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(ng/uL)</td><td rowspan=1 colspan=1>SD(ng/uL)</td><td rowspan=1 colspan=1>N&lt;2ng/uL</td><td rowspan=1 colspan=1>N&lt;5ng/uL</td></tr><tr><td rowspan=1 colspan=1>Expanded Clinical Dataset, processed Site 1</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Clinical Dataset, processed at Site 1</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Healthy donors, processed at Site 1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Clinical Dataset, processed at Site 2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Turn —Around —Time study samples,standard&#x27; processing at Site 3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Turn —Around —Time study samples,accelerated² processing at Site 3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr></table>

1 Standard PAXgene blood RNA Kit method (as per product instructions) 2 Accelerated Method omits a two-hour pre-incubation step in the PAXgene blood RNA extraction method

No significant differences in the yield of purified RNA was found between sample groups. Across all groups, $6 2 7 / 6 3 0 = 9 9 . 5 \%$ of samples produced sufficient purified RNA to perform the SeptiCyte LAB.

# h. Carryover:

Carryover contamination was investigated using fifteen clinical samples. The samples were chosen to represent SeptiSCORE values distributed across the range of clinically observed SeptiSCORE values. The samples were received in the form of PAXgene blood RNA tubes and total RNA was isolated from the tubes using the PAXgene isolation kit. Both the RT and qPCR steps were evaluated.

RT step: RNA samples $( \mathrm { n } { = } 5 )$ spanning the range of SeptiSCORE were tested, each in quadruplicate. An input of $2 0 0 \mathrm { n g }$ RNA was used in each RT reaction. A total of $6 0 \mathrm { R T }$ reactions containing RNA template were interspersed with 60 NTCs, in an alternating row pattern on three plates.

• qPCR step: Testing was conducted on a single day, over three qPCR plates.

All 15 RNA samples (tested in quadruplicate) produced acceptable SeptiSCORE values. No cross-contamination of any of the NTC wells was observed for the RT or qPCR plate configurations. As there were $1 2 0 \mathrm { N T C }$ wells in the qPCR plates, this implies that the frequency of carryover contamination events is $< 1 / 1 2 0$ (0.8 ).

# i PAXgene Blood RNA Tube Stability:

Two PAXgene Blood RNA tubes were collected from each of nine donors and incubated for at least two hours at room temperature according to the manufacturer’s procedure. One tube was processed immediately after the two-hour pre-incubation; the RNA was extracted according to the PAXgene Blood RNA extraction kit protocol and analyzed by the SeptiCyte LAB. The other tube was frozen and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ . After 11 months, the frozen sample was thawed and the RNA was extracted according to the manufacturer’s protocol and analyzed by the SeptiCyte LAB. The SeptiSCORE values for the two RNA samples were compared by regression analysis. The study demonstrated that blood collected and stored in PAXgene Blood RNA tubes is stable for subsequent use in the SeptiCyte LAB for at least 11 months when stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

# $j$ Stability of Extracted RNA:

The SeptiSCORE values for 20 RNA samples were compared before and after the RNA was frozen at $- 7 0 ^ { \circ } \mathrm { C } \left( \pm 5 ^ { \circ } \mathrm { C } \right)$ for 20 weeks. The RNA evaluated was extracted from samples and tested directly with the SeptiCyte LAB as part of the Precision Study (see 1a above). After running the test, the RNAs were frozen at $- 7 0 ^ { \circ } \mathrm { C } \left( \pm 5 ^ { \circ } \mathrm { C } \right)$ . At 20 weeks after freezing, the RNAs were thawed and tested again with the SeptiCyte LAB (frozen). The SeptiSCORE values from the fresh RNAs were compared to the SeptiSCORE values for the frozen RNAs. The SeptiSCORE differences were all $< 1$ unit. Linear regression indicated strong correlation $\textstyle \left( { \mathrm { R } } ^ { 2 } \right)$ $\left. \ - 0 . 9 9 1 \right.$ ) demonstrating that RNA extracted from PAXgene Blood RNA tubes is stable for the SeptiCyte LAB after being frozen at $\mathrm { - 7 0 ^ { \circ } C }$ $( { \pm } 5 ^ { \circ } \mathrm { C } )$ for 20 weeks.

# k. RNA Automation Study:

The manual and automated RNA (QiaCube) extractions were evaluated using two operators that processed a total of 48 identical sample aliquots across three PAXgene Blood RNA Kit lots. Each operator performed two manual and two automated PAXgene Blood RNA extractions per day for each of six non-consecutive PAXgene Blood RNA extractions according to the manufacturer’s procedure for manual or automated extraction (QiaCube), as described in the PAXgene Blood RNA Kit Handbook (Version 2). Extracted RNA samples were analyzed using a standard quantity of $1 0 0 \mathrm { n g } \mathrm { R N A }$ input into the reverse transcription step of the SeptiCyte LAB . Extracted RNAs were measured for nucleic acid concentration (A260) and purity (A260:A280 ratio) by Nanodrop spectrophotometer. Manually isolated RNA concentration ranged from $1 1 . 8 – 7 0 . 2 \mathrm { n g / \mu L }$ (median $6 0 . 2 \ \mathrm { n g / \mu L }$ ), with A260:A280 ranging from 1.9-2.1 (median 2.02). Automated extraction RNA concentration ranged from $6 5 . 3 \mathrm { - } 9 7 . 3 ~ \mathrm { n g / \mu L }$ (median $7 4 . 3 ~ \mathrm { n g / \mu L }$ , with A260:A280 ranging from 1.97-2.12 (median 2.02). Overall, a slightly higher median RNA yield was observed for the

automated compared to the manual RNA extraction. The purity, as assessed by A260:A280, was similar for both extraction methods. Significant differences between manual and automated RNA extraction methods for the SeptiCyte LAB result. CVs were investigated using analysis of variance (ANOVA) with a mixed-model approach. All statistical analyses were carried out using custom R scripts within the R package lme4 The precision of the SeptiCyte LAB was found to be equivalent across manual and automated (QIAcube) PAXgene Blood RNA extraction procedures. The CV values for all four target Cts and SeptiSCORE values for the automated extraction were within 1 of those for the manual extraction. The CV values also aligned to those determined in the Precision-Intermediate Study (See 1a above) using only manual extraction.

For the clinical validation of the automated vs manual processing method, two types of samples were analyzed: Pooled clinical samples (Part 1) and individual clinical samples (Part 2). In Part I, pooled samples were selected to produce High, Medium and Low (H, M, L) SeptiSCORE values as part of Branch B of the Precision Study. The same six subject PAXgene tubes per pool (H, M, L) for a total of 18 tubes were used. Samples were stored frozen at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 16 months before use. For Part 2, the SeptiSCORE values from the VENUS subjects were reviewed and 10 from each of the four SeptiSCORE Interpretation Bands were selected (total $= 4 0$ ). The duplicate PAXgene™ Blood tubes from each of the 40 subjects were thawed, extracted and analyzed. Samples were extracted using the automated method and examined for sources of variability between two operators over four nonconsecutive days. The automatically extracted samples were compared to the SeptiSCORE values obtained with manually-extracted RNA previously determined by three laboratory sites and reported in the clinical study. The SeptiSCORE values for manually extracted samples were compared through linear regression to their automatically extracted equivalents. The SeptiSCORE values determined from automated and manually extracted samples were compared for all the clinical samples from both Part 1 and Part 2 and found to be highly correlated $\mathrm { T } ^ { 2 } = 0 . 9 8 )$ . The SeptiSCORE values for each of the pools analyzed in Part 1 (H, M, and L), were within the range of SeptiSCORE values observed in the precision study.

l. Assay cut-off: See clinical cut-off.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

# a. Clinical Sensitivity:

Clinical performance of the SeptiCyte LAB was evaluated through observational, non interventional, prospective clinical trials (Molecular Diagnosis and Risk Stratification of Sepsis (MARS- NCT01905033) and Septic Gene Expression Using SeptiCyte (VENUS - NCT02127502)) which were conducted across eight clinical sites in the United States and Europe. This evaluation included PAX gene Blood RNA tube samples from 447 subjects $( \mathrm { n } { = } 1 9 8$ $\scriptstyle \mathtt { n } = 6 4$ from MARS and $\mathrm { n } { = } 1 3 5$ from MARs expanded) subjects and $\mathrm { n } { = } 2 4 9$ VENUS subjects $\mathrm { \Pi } ^ { - 1 2 9 }$ from VENUS and $\mathrm { n } { = } 1 2 0$ from VENUS expanded)) recruited from adult patients (18-89 years old) admitted to the ICU exhibiting at least two SIRS criteria. The demographics of the clinicaly study population are found in Table 7a and 7b below.

The SeptiSCORE for each subject was compared to a Retrospective Physician Diagnosis(RPD) comparator based on clinical case review by medical experts who independently reviewed clinical information about each subject while remaining blinded to the SeptiCyte LAB results. Case reviews occurred after the patient was discharged from the hospital. The RPD Case Review Panel based their diagnosis on clinical data. Two RPD methods were used:

1. Forced RPD: Subjects were stratified by diagnosis (Systemic Inflammatory Response Syndrome (SIRS) or sepsis) according to the majority opinion of the RPD Case Review Panelists. The category of Indeterminate was not allowed; all subjects were forced into either the SIRS or sepsis categories,with no subjects excluded. The performance evaluation of the SeptiCyte LAB was based on the Forced RPD.

2. Consensus RPD: Subjects were stratified by diagnosis (SIRS, sepsis or indeterminate) according to the majority opinion of three expert reviewers. Indeterminate cases $\mathrm { ( n { = } } 3 7 \mathrm { ) }$ ) were those for which a definitive diagnosis could not be reached by expert review and were not included in the analysis.

The Consensus RPD is also presented and should be considered supplementary information. An analysis of the study indeterminates found that these patients did not cluster as a group, but rather span the SeptiSCORE spectrum (i.e., the complete range of scores from 0-10 was observed). Patients classified as indeterminates do not appear to differ in any meaningful way from that of the rest of the intended use population, except that the clinical variables recorded for these subjects was insufficiently clear to support a confident RPD.

Table 7a: Summary of Demographics from Subjects within the Complete Clinical Dataset $\scriptstyle ( \mathrm { n = 4 4 7 } )$ )   

<table><tr><td colspan="2" rowspan="2">Demographic &amp; Clinical (categorical)</td><td colspan="1" rowspan="1">Sepsis (Forced)</td><td colspan="1" rowspan="1">SIRS(Forced)</td></tr><tr><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="2">Gender</td><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">88 (19.7)</td><td colspan="1" rowspan="1">104 (23.3)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">114 (25.5)</td><td colspan="1" rowspan="1">141 (31.5)</td></tr><tr><td colspan="1" rowspan="1">Ethnicity</td><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">13 (2.9)</td><td colspan="1" rowspan="1">11 (2.5)</td></tr><tr><td colspan="1" rowspan="5">−</td><td colspan="1" rowspan="1">Black (American or European of Africandescent)</td><td colspan="1" rowspan="1">37 (8.3)</td><td colspan="1" rowspan="1">51 (11.4)</td></tr><tr><td colspan="1" rowspan="1">Hispanic</td><td colspan="1" rowspan="1">5 (1.1)</td><td colspan="1" rowspan="1">7 (1.6)</td></tr><tr><td colspan="1" rowspan="1">Pacific Islander</td><td colspan="1" rowspan="1">0 (0)</td><td colspan="1" rowspan="1">1 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">2 (0.4)</td><td colspan="1" rowspan="1">4 (0.9)</td></tr><tr><td colspan="1" rowspan="1">White (Caucasian)</td><td colspan="1" rowspan="1">145 (32.4)</td><td colspan="1" rowspan="1">171 (38.3)</td></tr><tr><td colspan="1" rowspan="8">Site</td><td colspan="1" rowspan="1">MARS site 1</td><td colspan="1" rowspan="1">90 (20.1)</td><td colspan="1" rowspan="1">108 (24.2)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 1</td><td colspan="1" rowspan="1">4 (0.9)</td><td colspan="1" rowspan="1">3 (0.7)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 2</td><td colspan="1" rowspan="1">56 (12.5)</td><td colspan="1" rowspan="1">69 (15.4)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 3</td><td colspan="1" rowspan="1">14 (3.1)</td><td colspan="1" rowspan="1">25 (5.6)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 4</td><td colspan="1" rowspan="1">3 (0.7)</td><td colspan="1" rowspan="1">1 (0.2)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 5</td><td colspan="1" rowspan="1">14 (3.1)</td><td colspan="1" rowspan="1">12 (2.7)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 6</td><td colspan="1" rowspan="1">17 (3.8)</td><td colspan="1" rowspan="1">20 (4.5)</td></tr><tr><td colspan="1" rowspan="1">VENUS trial site 7</td><td colspan="1" rowspan="1">4 (0.9)</td><td colspan="1" rowspan="1">7 (1.6)</td></tr><tr><td colspan="1" rowspan="10">Source ofadmission</td><td colspan="1" rowspan="1">Emergency department</td><td colspan="1" rowspan="1">1 (0.2)</td><td colspan="1" rowspan="1">4 (0.9)</td></tr><tr><td colspan="1" rowspan="1">Emergency department, other hospital</td><td colspan="1" rowspan="1">82 (18.3)</td><td colspan="1" rowspan="1">103 (23)</td></tr><tr><td colspan="1" rowspan="1">Intensive Care Unit, other hospital</td><td colspan="1" rowspan="1">3 (0.7)</td><td colspan="1" rowspan="1">13 (2.9)</td></tr><tr><td colspan="1" rowspan="1">Nursing ward</td><td colspan="1" rowspan="1">3 (0.7)</td><td colspan="1" rowspan="1">2 (0.4)</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">8 (1.8)</td><td colspan="1" rowspan="1">3 (0.7)</td></tr><tr><td colspan="1" rowspan="1">Nursing ward, other hospital</td><td colspan="1" rowspan="1">39 (8.7)</td><td colspan="1" rowspan="1">41 (9.2)</td></tr><tr><td colspan="1" rowspan="1">Operating theatre</td><td colspan="1" rowspan="1">17 (3.8)</td><td colspan="1" rowspan="1">18 (4)</td></tr><tr><td colspan="1" rowspan="1">Post-anesthesia care facility</td><td colspan="1" rowspan="1">26 (5.8)</td><td colspan="1" rowspan="1">9 (2)</td></tr><tr><td colspan="1" rowspan="1">Coronary care Facility</td><td colspan="1" rowspan="1">2 (0.4)</td><td colspan="1" rowspan="1">4 (0.9)</td></tr><tr><td colspan="1" rowspan="1">Emergency department (via operatingtheatre)</td><td colspan="1" rowspan="1">11 (2.5)</td><td colspan="1" rowspan="1">18 (4)</td></tr><tr><td colspan="1" rowspan="4">Impression onDischarge(Consensus of 3x PrincipalInvestigators)</td><td colspan="1" rowspan="1">Definite Infection</td><td colspan="1" rowspan="1">103 (23)</td><td colspan="1" rowspan="1">0 (0)</td></tr><tr><td colspan="1" rowspan="1">Possible Infection</td><td colspan="1" rowspan="1">46 (10.3)</td><td colspan="1" rowspan="1">11 (2.5)</td></tr><tr><td colspan="1" rowspan="1">Probable Infection</td><td colspan="1" rowspan="1">41 (9.2)</td><td colspan="1" rowspan="1">2 (0.4)</td></tr><tr><td colspan="1" rowspan="1">SIRS</td><td colspan="1" rowspan="1">12 (2.7)</td><td colspan="1" rowspan="1">232 (51.9)</td></tr><tr><td colspan="1" rowspan="2">PCT</td><td colspan="1" rowspan="1">Not available</td><td colspan="1" rowspan="1">38 (8.5)</td><td colspan="1" rowspan="1">61 (13.6)</td></tr><tr><td colspan="1" rowspan="1">Available</td><td colspan="1" rowspan="1">164 (36.7)</td><td colspan="1" rowspan="1">184 (41.2)</td></tr><tr><td colspan="1" rowspan="2">Death2</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">170 (38.1)</td><td colspan="1" rowspan="1">218 (48.9)</td></tr><tr><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">31 (7)</td><td colspan="1" rowspan="1">27 (6.1)</td></tr><tr><td colspan="1" rowspan="2">Ventilator</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">121 (27.1)</td><td colspan="1" rowspan="1">137 (30.6)</td></tr><tr><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">81 (18.1)</td><td colspan="1" rowspan="1">108 (24.2)</td></tr></table>

1 Regardless of infection type (Gram positive, Gram negative or viral) there was no apparent differences observed in SeptiCyte score. Insufficient fungal infections were observed to evaluate effect on SeptiSCORE. $^ 2 \mathrm { N } { = } 4 4 6$ . Mortality data unavailable for one subject.

Table 7b: Summary of Demographics from Subjects within the Complete Clinical Dataset $\scriptstyle ( \mathrm { n = 4 4 7 } )$   

<table><tr><td rowspan=2 colspan=2>Demographic &amp; Clinical (categorical)</td><td rowspan=1 colspan=1>Sepsis(Consensus)</td><td rowspan=1 colspan=1>SIRS(Consensus)</td><td rowspan=1 colspan=1>Indeterminate(Consensus)</td></tr><tr><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>78 (17.4)</td><td rowspan=1 colspan=1>100 (22.4)</td><td rowspan=1 colspan=1>14 (3.1)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>102 (22.8)</td><td rowspan=1 colspan=1>130 (29.1)</td><td rowspan=1 colspan=1>23 (5.1)</td></tr><tr><td rowspan=6 colspan=1>Ethnicity</td><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>12 (2.7)</td><td rowspan=1 colspan=1>11 (2.5)</td><td rowspan=1 colspan=1>1 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Black (American or European ofAfrican descent)</td><td rowspan=1 colspan=1>30 (6.7)</td><td rowspan=1 colspan=1>48 (10.7)</td><td rowspan=1 colspan=1>10 (2.2)</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>5 (1.1)</td><td rowspan=1 colspan=1>7 (1.6)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Pacific Islander</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>4 (0.9)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>White (Caucasian)</td><td rowspan=1 colspan=1>131 (29.3)</td><td rowspan=1 colspan=1>159 (35.6)</td><td rowspan=1 colspan=1>26 (5.8)</td></tr><tr><td rowspan=8 colspan=1>Site</td><td rowspan=1 colspan=1>MARS site 1</td><td rowspan=1 colspan=1>81 (18.1)</td><td rowspan=1 colspan=1>100 (22.4)</td><td rowspan=1 colspan=1>17 (3.8)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 1</td><td rowspan=1 colspan=1>52 (11.6)</td><td rowspan=1 colspan=1>64 (14.3)</td><td rowspan=1 colspan=1>9 (2)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 2</td><td rowspan=1 colspan=1>3 (0.7)</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 3</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>3 (0.7)</td><td rowspan=1 colspan=1>2 (0.4)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 4</td><td rowspan=1 colspan=1>14 (3.1)</td><td rowspan=1 colspan=1>24 (5.4)</td><td rowspan=1 colspan=1>1 (0.2)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 5</td><td rowspan=1 colspan=1>4 (0.9)</td><td rowspan=1 colspan=1>7 (1.6)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 6</td><td rowspan=1 colspan=1>11 (2.5)</td><td rowspan=1 colspan=1>12 (2.7)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=1 colspan=1>VENUS trial site 7</td><td rowspan=1 colspan=1>13 (2.9)</td><td rowspan=1 colspan=1>19 (4.3)</td><td rowspan=1 colspan=1>5 (1.1)</td></tr><tr><td rowspan=10 colspan=1>Source ofadmission</td><td rowspan=1 colspan=1>Emergency department</td><td rowspan=1 colspan=1>103 (23)</td><td rowspan=1 colspan=1>136 (30.4)</td><td rowspan=1 colspan=1>26 (5.8)</td></tr><tr><td rowspan=1 colspan=1>Emergency department, other hospital</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>1 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Intensive Care Unit, other hospital</td><td rowspan=1 colspan=1>18 (4.1)</td><td rowspan=1 colspan=1>22 (4.9)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=1 colspan=1>Nursing ward</td><td rowspan=1 colspan=1>33 (7.4)</td><td rowspan=1 colspan=1>10 (2.3)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>4 (0.9)</td><td rowspan=1 colspan=1>9(2)</td><td rowspan=1 colspan=1>1 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Nursing ward, other hospital</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>4 (0.9)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Operating theatre</td><td rowspan=1 colspan=1>11 (2.5)</td><td rowspan=1 colspan=1>15 (3.4)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=1 colspan=1>Post-anesthesia care facility</td><td rowspan=1 colspan=1>3 (0.6)</td><td rowspan=1 colspan=1>15 (3.3)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Coronary care Facility</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>4 (0.9)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Emergency department (via operatingtheatre)</td><td rowspan=1 colspan=1>3 (0.7)</td><td rowspan=1 colspan=1>13 (2.9)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=4 colspan=1>Impressionon Discharge(Consensus of3 x PrincipalInvestigators)</td><td rowspan=1 colspan=1>Definite Infection</td><td rowspan=1 colspan=1>100 (22.4)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=1 colspan=1>Possible Infection</td><td rowspan=1 colspan=1>36 (8.1)</td><td rowspan=1 colspan=1>5 (1.1)</td><td rowspan=1 colspan=1>16 (3.6)</td></tr><tr><td rowspan=1 colspan=1>Probable Infection</td><td rowspan=1 colspan=1>39 (8.7)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>4 (0.9)</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>5(1.1)</td><td rowspan=1 colspan=1>225 (50.3)</td><td rowspan=1 colspan=1>14 (3.1)</td></tr><tr><td rowspan=2 colspan=1>PCT</td><td rowspan=1 colspan=1>Not available</td><td rowspan=1 colspan=1>37 (8.3)</td><td rowspan=1 colspan=1>58 (13)</td><td rowspan=1 colspan=1>4 (0.9)</td></tr><tr><td rowspan=1 colspan=1>Available</td><td rowspan=1 colspan=1>143 (32)</td><td rowspan=1 colspan=1>172 (38.5)</td><td rowspan=1 colspan=1>33 (7.4)</td></tr><tr><td rowspan=2 colspan=1>Death2</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>151 (33.9)</td><td rowspan=1 colspan=1>203 (45.5)</td><td rowspan=1 colspan=1>34 (7.6)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>28 (6.3)</td><td rowspan=1 colspan=1>27 (6.1)</td><td rowspan=1 colspan=1>3 (0.7)</td></tr><tr><td rowspan=2 colspan=1>Ventilator</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>111 (24.8)</td><td rowspan=1 colspan=1>128 (28.6)</td><td rowspan=1 colspan=1>19 (4.3)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>69 (15.4)</td><td rowspan=1 colspan=1>102 (22.8)</td><td rowspan=1 colspan=1>18 (4)</td></tr></table>

1 Regardless of infection type (Gram positive, Gram negative or viral) there was no apparent differences observed in SeptiCyte score. Insufficient fungal infections were observed to evaluate effect on SeptiSCORE . $^ 2 { \mathrm { N } } { = } 4 4 6$ . Mortality data unavailable for one subject.

The distribution of scores for each RPD and stratified patient population are presented in Tables 8 and 9 below.

Table 8: Forced RPD Distribution $\scriptstyle ( \mathrm { N } = 4 4 7 )$ .   

<table><tr><td rowspan=3 colspan=2>Race</td><td rowspan=1 colspan=8>SeptiSCORE Interpretation Band (SeptiSCORE)</td></tr><tr><td rowspan=1 colspan=2>1 (0-3.0)</td><td rowspan=1 colspan=2>2 (3.1-4.4)</td><td rowspan=1 colspan=2>3 (4.5-5.9)</td><td rowspan=1 colspan=2>4 (6-10)</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>73</td></tr><tr><td rowspan=1 colspan=1>Pacific Islander</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 9: Consensus RPD Distribution $\scriptstyle ( \mathrm { N } = 4 4 7 )$ )   

<table><tr><td rowspan=3 colspan=2>Race</td><td rowspan=1 colspan=12>SeptiSCORE Interpretation Band (SeptiSCORE)</td></tr><tr><td rowspan=1 colspan=3>1 (1-3.0)</td><td rowspan=1 colspan=3>2 (3.1-4.4)</td><td rowspan=1 colspan=3>3 (4.5-5.9)</td><td rowspan=1 colspan=3>4 (6-10)</td></tr><tr><td rowspan=1 colspan=1>IND*</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>PacificIslander</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr></table>

\*Indeterminate

The clinical study results show a relationship between SeptiSCORE and the increasing likelihood of sepsis across each SeptiSCORE Interpretation Band. The Likelihood Ratios (LR) in Tables 10 - 13 (below) were calculated using the standard definition where LR equals the probability that an individual with disease has the test result divided by the probability that an individual without disease has the test result. This formula was applied to each SeptiSCORE Interpretation Band separately. These predictive values depend on the likelihood ratios and the prevalence of disease. Laboratories and other users should establish their own reference intervals for their patient populations using the SeptiCyte LAB to reflect potential sources of variability, such as patient gender, race, age, and preparation techniques. As higher SeptiSCORE values were observed in this sample of African American/European of African descent, additional analysis was performed for this subgroup (Tables 11 and 13 below).

Due to administrative errors, Tables 11, 12, 16 and 17 have been updated. In Table 11, the following were updated in the interpretation column: For Band 4, $86 \%$ was updated to $87 \%$ ; for Band 2, $22 \%$ was updated to $1 9 \%$ , and for Band 1, $16 \%$ was updated to $0 \%$ . Additionally, the likelihood ratio for Band 2 was updated from 0.32 to 0.33. In Table 12 (same as Table 17), the interpretation column of Band 3 was updated from $47 \%$ to $48 \%$ and the Band 2 interpretation column was updated from $19 \%$ to $20 \%$ . In Table 16, the interpretation column of Band 3 was updated from $48 \%$ to $49 \%$ .

Table 10: Likelihood Ratio’s for all complete clinical study population using the Forced Diagnosis $\scriptstyle ( \mathrm { N } = 4 4 7 )$   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretationBand</td><td rowspan=2 colspan=1>Sepsis</td><td rowspan=2 colspan=1>SIRS</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihoodRatio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Sepsis is &gt; 82%likely</td><td rowspan=1 colspan=1>18%</td><td rowspan=1 colspan=1>82%</td><td rowspan=1 colspan=1>5.61</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>Sepsis is &gt; 49%likely</td><td rowspan=1 colspan=1>51%</td><td rowspan=1 colspan=1>49%</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>Sepsis is &gt; 22%likely</td><td rowspan=1 colspan=1>78%</td><td rowspan=1 colspan=1>22%</td><td rowspan=1 colspan=1>0.35</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>Sepsis is &lt; 16%likely</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>16%</td><td rowspan=1 colspan=1>0.23</td></tr></table>

Table 11: Likelihood Ratio’s for African American or European of African descent SeptiSCORE values using the Forced Diagnosis $( \mathrm { N } { = } 8 8 )$   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretationBand</td><td rowspan=2 colspan=1>Sepsis</td><td rowspan=2 colspan=1>SIRS</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihoodRatio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Sepsis is &gt; 87%likely</td><td rowspan=1 colspan=1>13%</td><td rowspan=1 colspan=1>87%</td><td rowspan=1 colspan=1>9.0</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Sepsis is &gt; 26%likely</td><td rowspan=1 colspan=1>74%</td><td rowspan=1 colspan=1>26%</td><td rowspan=1 colspan=1>0.49</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Sepsis is &gt; 19%likely</td><td rowspan=1 colspan=1>81%</td><td rowspan=1 colspan=1>19%</td><td rowspan=1 colspan=1>0.33</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Sepsis is &lt; 0 %likely</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 12: Likelihood Ratio’s values for complete clinical study population using the Consensus Diagnosis\* $\scriptstyle ( \ N = 4 1 0 )$   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretation Band</td><td rowspan=2 colspan=1>IND*</td><td rowspan=2 colspan=1>Sepsis</td><td rowspan=2 colspan=1>SIRS</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihood Ratio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Sepsis is &gt; 84%likely</td><td rowspan=1 colspan=1>16%</td><td rowspan=1 colspan=1>84%</td><td rowspan=1 colspan=1>6.71</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>Sepsis is &gt; 48%likely</td><td rowspan=1 colspan=1>52%</td><td rowspan=1 colspan=1>48%</td><td rowspan=1 colspan=1>1.17</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>Sepsis is &gt; 20%likely</td><td rowspan=1 colspan=1>80%</td><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>Sepsis is &lt; 11%likely</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>0.16</td></tr></table>

Table 13: Likelihood Ratio‘s for African American or European of African descent SeptiSCORE values using the Consensus Diagnosis\* $\left( \mathrm { N } { = } 7 8 \right)$   
Figure 1: Probability of Sepsis for Complete Clinical Dataset   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretation Band</td><td rowspan=2 colspan=1>IND*</td><td rowspan=2 colspan=1>Sepsis</td><td rowspan=2 colspan=1>SIRS</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihood Ratio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Sepsis is &gt; 89%likely</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>12.8</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Sepsis is &gt; 11%likely</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Sepsis is &gt; 17%likely</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>17%</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Sepsis is &lt; 0%likely</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.0</td></tr></table>

\*Indeterminate patients excluded for Sepsis Likelihood Ratio calculation.

![](images/f84ea049e1f97b0385ded9e3f5e1824eaabd7e340b88154d2f0906b0217d91ed.jpg)

The clinical data showed a relationship between SeptiSCORE and the increasing likelihood of sepsis across each SeptiSCORE Interpretation Band. An $80 \%$

Confidence Interval $80 \%$ CI) of the probability of sepsis was calculated for every SeptiSCORE Interpretation Band using an exact binomial test via the R function binom.test (implementing Clopper & Pearson, 1934). Upper and lower limits of the $80 \%$ CIs of the probability of sepsis for non-adjacent SeptiSCORE Interpretation Bands were compared in order to determine if any of the non-adjacent SeptiSCORE Interpretation Bands had overlapping $80 \%$ CIs. The probability of a sepsis diagnosis as determined by Consensus RPD (left panel); Forced RPD (right panel) is shown across each of the four SeptiSCORE Interpretation Bands for the Complete Clinical Dataset. As shown in Figure 1 above, non-adjacent SeptiSCORE Interpretation Bands displayed non-overlapping $80 \%$ confidence intervals (CIs) for probability of sepsis. The point estimate for each Band is the observed sepsis diagnosis probability; whiskers are the upper and lower $80 \%$ CI boundaries.The probability of sepsis exhibited a monotonic increase that correlates with SeptiSCORE for both the Forced RPD and the Consensus RPD.

To further evaluate whether the SeptiCyte LAB t is effective and will provide clinically significant results in the intended use population, the SeptiCyte LAB was also analyzed with respect to available clinical and laboratory variables (see Table 14 below) from the Complete Clinical Dataset using a backwards-elimination (BWElimination) variable selection procedure, combined with a logistic regression decision rule. The backwards-elimination was used to determine which clinical variables best discriminate sepsis from SIRS. Area Under Curve (AUC) was used as the metric for assessing model performance for each analysis. Models were constructed using the Forced or Consensus RPD and the top five clinical variables contributing to the AUC were identified. For all logistic regression models, BWElimination and k-fold cross-validation were used to determine if the test provided diagnostic clinical utility beyond that provided by other clinical variables and laboratory assessments available within the first\~24 hours of the suspicion of sepsis. The backward elimination technique started with the full model including all independent variables. Models were constructed including or excluding Procalcitonin (PCT). At each step, the effect showing the smallest contribution to the model was deleted. The logistic regression model for discriminating between sepsis and SIRS found SeptiSCORE to be the most significant variable for each model. This observation was found for both Consensus RPD and Forced RPD.

TABLE 14: List of Variables Evaluated in Logistic Regression Modeling   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Variable</td><td colspan="1" rowspan="1">Brief Description of Variable*</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Age</td><td colspan="1" rowspan="1">Age of patient</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Race</td><td colspan="1" rowspan="1">Race of patient: African descent orNon-African descent</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Gender</td><td colspan="1" rowspan="1">Gender of patient</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Glucose (Maximum)</td><td colspan="1" rowspan="1">Peak blood glucose concentration</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">HeartRate (Minimum)</td><td colspan="1" rowspan="1">Lowest heart rate</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HeartRate (Maximum)</td><td colspan="1" rowspan="1">Peak heart rate</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Infection status: Present</td><td colspan="1" rowspan="1">Has an infection been confirmed?</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Infection status: Fungal?</td><td colspan="1" rowspan="1">Is a fungal infection present?</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Infection status: Gram negative?</td><td colspan="1" rowspan="1">Is a Gram negative infection present?</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Infection status: Gram positive? Is a Gram positive infection present?</td><td colspan="1" rowspan="1">Infection status: Gram positive? Is a Gram positive infection present?</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Infection status: Mixed?</td><td colspan="1" rowspan="1">Is a mixed infection (i.e.,morethanoneCategory of infectious agent)</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Infection status: Viral?</td><td colspan="1" rowspan="1">Isaviralinfectionpresent?</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">MeanArterialPressure(MAPpMaximum)</td><td colspan="1" rowspan="1">Peak mean arterial blood pressure</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">SIRS Criteria</td><td colspan="1" rowspan="1">Number of SIRS criteria observed</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Temperature (Maximum)</td><td colspan="1" rowspan="1">Peak body temperature</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Temperature (Minimum)</td><td colspan="1" rowspan="1">Lowest body temperature</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">WBC (Maximum)</td><td colspan="1" rowspan="1">Maximum WBC count</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">WBC (Minimum)</td><td colspan="1" rowspan="1">Minimum WBC count</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">PCT/Procalcitonin**</td><td colspan="1" rowspan="1">Plasma PCT concentration (log2)</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">SeptiSCORE**</td><td colspan="1" rowspan="1">The SeptiCyte LAB result</td></tr></table>

\* All variables were measured and recorded within the first 24 hours of subject enrollment except Infection status. \*\*Variables that were not used in every model built.

b. Clinical specificity: See Clinical sensitivity   
c. Other clinical supportive data (when a. and b. are not applicable):

In the analysis of blood samples obtained from 297 apparently healthy subjects, slightly higher SeptiSCORE values were observed African Americans as compared to Whites, Hispanics and Asians. Results are summarized in the Table 15 below.

Table 15: SeptiCyte LAB Results for Healthy Subject Cohort   

<table><tr><td rowspan=3 colspan=2>Race</td><td rowspan=1 colspan=4>SeptiSCORE Interpretation Band (SeptiSCORE)</td></tr><tr><td rowspan=1 colspan=1>1 (1-3.0)</td><td rowspan=1 colspan=1>2 (3.1-4.4)</td><td rowspan=1 colspan=1>3 (4.5-5.9)</td><td rowspan=1 colspan=1>4 (6-10)</td></tr><tr><td rowspan=1 colspan=1>Healthy</td><td rowspan=1 colspan=1>Healthy</td><td rowspan=1 colspan=1>Healthy</td><td rowspan=1 colspan=1>Healthy</td></tr><tr><td rowspan=2 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>44.0</td><td rowspan=1 colspan=1>44.0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>49.0</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=2 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>58.6</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>20.1</td><td rowspan=1 colspan=1>62.5</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>0.7</td></tr></table>

4. Clinical cut-off:

SeptiSCORE cut-off values were established prior to the clinical trial. The following SeptiSCORE Interpretation tables are the results from non-interventional observational clinical trials, the VENUS trial (Clinicaltrials.gov identifier: NCT02127502) and (MARS) clinical trial (NCT01905033) using either the Forced Diagnosis (Table 16) or the Consensus Diagnosis (Table 17). The SeptiSCORE values for African Americans included in the clinical study and apparently healthy African Americans (not part of the clinical study) showed there may be a small positive bias (higher SeptiSCORE) when compared to other racial and ethnic groups.

Table 16: Recommendations for interpretation of SeptiSCORE values using the Forced Diagnosis   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretation Band</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihoodRatio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>Sepsis is &gt; 82% likely</td><td rowspan=1 colspan=1>18%</td><td rowspan=1 colspan=1>82%</td><td rowspan=1 colspan=1>5.61</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>Sepsis is &gt; 49% likely</td><td rowspan=1 colspan=1>51%</td><td rowspan=1 colspan=1>49%</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>Sepsis is &gt; 22% likely</td><td rowspan=1 colspan=1>78%</td><td rowspan=1 colspan=1>22%</td><td rowspan=1 colspan=1>0.35</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>Sepsis is &lt; 16% likely</td><td rowspan=1 colspan=1>84%</td><td rowspan=1 colspan=1>16%</td><td rowspan=1 colspan=1>0.23</td></tr></table>

Table 17: Recommendations for interpretation of SeptiSCORE values using the Consensus Diagnosis   

<table><tr><td rowspan=2 colspan=1>SeptiSCOREInterpretation Band</td><td rowspan=2 colspan=1>Interpretation</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>SepsisLikelihoodRatio</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>Sepsis</td></tr><tr><td rowspan=1 colspan=1>Band 4 (6-10)</td><td rowspan=1 colspan=1>Sepsis is &gt; 84% likely</td><td rowspan=1 colspan=1>16%</td><td rowspan=1 colspan=1>84%</td><td rowspan=1 colspan=1>6.71</td></tr><tr><td rowspan=1 colspan=1>Band 3 (4.5-5.9)</td><td rowspan=1 colspan=1>Sepsis is &gt; 48% likely</td><td rowspan=1 colspan=1>52%</td><td rowspan=1 colspan=1>48%</td><td rowspan=1 colspan=1>1.17</td></tr><tr><td rowspan=1 colspan=1>Band 2 (3.1-4.4)</td><td rowspan=1 colspan=1>Sepsis is &gt; 20% likely</td><td rowspan=1 colspan=1>80%</td><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>Band 1 (0-3.0)</td><td rowspan=1 colspan=1>Sepsis is &lt; 11% likely</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>0.16</td></tr></table>

# 5. Expected values/Reference range:

See Clinical cut-off. Predictive values depend on the likelihood ratios and the prevalence of disease. Laboratories and other users should establish their own reference intervals for their patient populations using the SeptiCyte LAB to reflect potential sources of variability, such as patient gender, race, age, and preparation techniques.

# N. Instrument Name:

The SeptiCyte LAB uses the Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, Applied Biosystems, Foster City, CA, catalogue number 4406985; FDA 510k number K082562.

SeptiCyte Analysis software.

# O. System Descriptions:

1. Modes of Operation: See Device Description (Section I) above

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes ___X_ _ or No

2. Software: FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types: Yes ____X_ __ or No

Note: The software V1.3 is intended to be run only using Windows 7 (not Windows 8.1) and was validated to run only using Windows 7.

3. Specimen Identification:

The Sample IDs (for 1, 2 or 3 samples) for a run are entered into the ABI-7500 Fast Dx setup screen, by the laboratory operator when setting up the run. The laboratory operator types in the sample IDs through the keyboard attached to the ABI-7500 Fast Dx console. The SeptiCyte LAB does not provide any provision for data entry other than by the keyboard. This does not preclude the use of a bar code reader if the laboratory has a reader that can be attached to the ABI-7500 Fast Dx and used as a data input device. This, however, would be part of the laboratory operations, and would occur outside of the instructions provided with the SeptiCyte LAB. The sample IDs are automatically embedded in the SDS file, along with the associated real-time RT-qPCR data, as created by the ABI-7500 Fast Dx during the run. SDS file is exported by the ABI-7500 Fast Dx and manually transferred to the computer running SeptiCyte Analysis software. SDS file is read by the SeptiCyte Analysis Software. The SeptiCyte Analysis Software parses the contents of the SDS file, and uses each embedded Sample ID to search in the local HL7 directory for the first instance of an HL7 file that contains the corresponding Sample ID tag within the appropriate HL7 file element. Once the SeptiCyte Analysis software locates the corresponding HL7 file, it reads the contents of this file, and then populates the relevant fields on the patient report with the following patient demographic information: Patient Name, Medical Report , Blood Collection Date, Sample ID. Alternatively, if no corresponding HL7 file is available in the local HL7 directory, then the SeptiCyte Analysis software generates a report with only the Sample ID field populated taking information from the ABI-7500 Fast Dx SDS file. In such a case, the Sample ID would need to be manually linked to the patient demographic information after the report was generated.

The SeptiCyte LAB Analysis Software imports patient demographic information from the LIMS system via file system transfer. Further the software exports patient and report information to the LIMS system.

4. Specimen Sampling and Handling: See Section I- Device description

5. Calibration: No calibrators are run as part of the SeptiCyte LAB protocol.

6. Quality Control: See Section 1C - Traceability, Stability, Expected values (controls, calibrators, or methods):

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

1. The submitted information in this premarket notification is complete and supports a substantial equivalence decision.