# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

To obtain a substantial equivalent determination for the MALDI Biotyper CA System.

C. Measurand: See Intended Use.

# D. Type of Test:

The MALDI Biotyper CA System is a qualitative in vitro diagnostic device intended for the identification of Gram-negative bacterial colonies cultured from human specimens. The device is comprised of an ionization source, a mass analyzer and a spectral database. The device is indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of Gram-negative bacterial infections.

# E. Applicant:

Bruker Daltonik GmbH

# F. Proprietary and Established Names:

Trade Name: MALDI Biotyper CA (MBT-CA) System, MBT smart CA System

Common Names: MBT-CA, System, mass spectrometry

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3361 Instrumentation for clinical multiplex test   
systems   
2. Classification: Class II (special controls)   
3. Product code: PEX   
4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The MALDI Biotyper CA System is a mass spectrometer system using matrix-assisted laser desorption/ionization - time of flight (MALDI-TOF) for the identification of microorganisms cultured from human specimens.

The MALDI Biotyper CA System is a qualitative in vitro diagnostic device indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of bacterial and yeast infections.

The following organisms are claimed:

<table><tr><td colspan="2">Bacteria:</td></tr><tr><td>Abiotrophia defectiva</td><td>Achromobacter xylosoxidans</td></tr><tr><td>Acinetobacter baumannii/ nosocomialis group Acinetobacter calcoaceticus</td><td></td></tr><tr><td>Acinetobacter haemolyticus</td><td>Acinetobacter johnsonii</td></tr><tr><td>Acinetobacter junii</td><td>Acinetobacter lwoffii</td></tr><tr><td>Acinetobacter pittii</td><td>Acinetobacter radioresistens</td></tr><tr><td>Acinetobacter ursingii</td><td>Actinomyces europaeus</td></tr><tr><td>Actinomyces funkei</td><td>Actinomyces graevenitzii</td></tr><tr><td>Actinomyces hyovaginalis</td><td>Actinomyces meyeri</td></tr><tr><td>Actinomyces neuii</td><td>Actinomyces odontolyticus</td></tr><tr><td>Actinomyces oris</td><td>Actinomyces radingae</td></tr><tr><td>Actinomyces turicensis</td><td>Actinomyces urogenitalis</td></tr><tr><td>Actinotignum schalii group</td><td>Aerococcus sanguinicola</td></tr><tr><td>Aerococcus urinae</td><td>Aerococcus viridans</td></tr><tr><td>Aeromonas salmonicida</td><td>Aeromonas sp[7]</td></tr><tr><td>Aggregatibacter actinomycetemcomitans</td><td>Aggregatibacter aphrophilus</td></tr><tr><td>Aggregatibacter segnis</td><td>Alcaligenes faecalis</td></tr><tr><td>Alloiococcus otitis</td><td>Alloscardovia omnicolens</td></tr><tr><td>Anaerococcus murdochii</td><td>Anaerococcus vaginalis</td></tr><tr><td>Arthrobacter cumminsii</td><td>Bacteroides caccae</td></tr><tr><td>Bacteroides fragilis</td><td>Bacteroides nordii</td></tr><tr><td>Bacteroides ovatus group</td><td>Bacteroides pyogenes</td></tr><tr><td>Bacteroides salyersiae</td><td>Bacteroides stercoris group</td></tr><tr><td>Bacteroides thetaiotaomicron group</td><td>Bacteroides uniformis</td></tr><tr><td>Bacteroides vulgatus group</td><td>Bifidobacterium breve</td></tr><tr><td>Bordetella group [3]</td><td>Bordetella hinzii</td></tr><tr><td>Brevibacterium casei</td><td>Brevundimonas diminuta group</td></tr><tr><td>Burkholderia cepacia complex [13]</td><td>Burkholderia gladioli</td></tr><tr><td>Burkholderia multivorans</td><td>Campylobacter coli</td></tr><tr><td>Campylobacter jejuni</td><td>Campylobacter ureolyticus</td></tr><tr><td>Capnocytophaga ochracea</td><td>Capnocytophaga sputigena</td></tr><tr><td>Chryseobacterium gleum</td><td>Chryseobacterium indologenes</td></tr><tr><td>Citrobacter amalonaticus complex</td><td>Citrobacter freundii complex</td></tr><tr><td>Citrobacter koseri</td><td>Clostridium beijerinckii</td></tr><tr><td>Clostridium bifermentans</td><td>Clostridium butyricum</td></tr><tr><td>Clostridium clostridioforme group</td><td>Clostridium difficile</td></tr><tr><td>Clostridium innocuum</td><td>Clostridium paraputrificum</td></tr><tr><td>Bacteria:</td><td></td></tr><tr><td>Clostridium perfringens</td><td>Clostridium ramosum</td></tr><tr><td>Clostridium septicum</td><td>Clostridium sordellii</td></tr><tr><td>Clostridium sporogenes /</td><td>Clostridium tertium</td></tr><tr><td>Clostridium botulinum (group I) Corynebacterium accolens</td><td>Corynebacterium afermentans group</td></tr><tr><td>Corynebacterium amycolatum</td><td>Corynebacterium aurimucosum group</td></tr><tr><td>Corynebacterium bovis</td><td>Corynebacterium coyleae</td></tr><tr><td>Corynebacterium diphtheriae</td><td>Corynebacterium freneyi</td></tr><tr><td>Corynebacterium glucuronolyticum</td><td>Corynebacterium glutamicum</td></tr><tr><td>Corynebacterium jeikeium</td><td>Corynebacterium kroppenstedtii</td></tr><tr><td>Corynebacterium macginleyi</td><td>Corynebacterium minutissimum</td></tr><tr><td>Corynebacterium mucifaciens / ureicelerivorans group</td><td>Corynebacterium propinquum</td></tr><tr><td>Corynebacterium pseudodiphtheriticum</td><td>Corynebacterium pseudotuberculosis</td></tr><tr><td>Corynebacterium resistens</td><td>Corynebacterium riegelii</td></tr><tr><td>Corynebacterium striatum group</td><td>Corynebacterium tuberculostearicum</td></tr><tr><td>Corynebacterium ulcerans</td><td>Corynebacterium urealyticum</td></tr><tr><td>Corynebacterium xerosis</td><td>Cronobacter sakazakii group</td></tr><tr><td>Cupriavidus pauculus group</td><td>Delftia acidovorans group</td></tr><tr><td>Dermabacter hominis</td><td>Dermacoccus nishinomiyaensis</td></tr><tr><td>Edwardsiella tarda</td><td>Eikenella corrodens</td></tr><tr><td>Elizabethkingia meningoseptica group</td><td>Enterobacter aerogenes</td></tr><tr><td>Enterobacter amnigenus</td><td>Enterobacter cloacae complex</td></tr><tr><td>Enterococcus avium</td><td>Enterococcus casseliflavus</td></tr><tr><td>Enterococcus durans</td><td>Enterococcus faecalis</td></tr><tr><td>Enterococcus faecium</td><td>Enterococcus gallinarum</td></tr><tr><td>Enterococcus hirae</td><td>Enterococcus mundtii</td></tr><tr><td>Enterococcus raffinosus</td><td>Escherichia coli</td></tr><tr><td>Escherichia hermannii</td><td>Escherichia vulneris</td></tr><tr><td>Ewingella americana</td><td>Facklamia hominis</td></tr><tr><td>Finegoldia magna</td><td>Fluoribacter bozemanae</td></tr><tr><td>Fusobacterium canifelinum</td><td>Fusobacterium necrophorum</td></tr><tr><td>Fusobacterium nucleatum</td><td>Gardnerella vaginalis</td></tr><tr><td>Gemella haemolysans</td><td>Gemella morbillorum</td></tr><tr><td></td><td>Granulicatella adiacens</td></tr><tr><td>Gemella sanguinis Haemophilus haemolyticus</td><td>Haemophilus influenzae</td></tr><tr><td>Haemophilus parahaemolyticus group</td><td>Haemophilus parainfluenzae</td></tr><tr><td>Hafnia alvei</td><td>Helcococcus kunzii</td></tr><tr><td>Kingella denitrificans</td><td>Kingella kingae</td></tr><tr><td>Klebsiella oxytoca / Raoultella ornithinolytica</td><td>Klebsiella pneumoniae</td></tr><tr><td>Klebsiella variicola</td><td>Kocuria kristinae</td></tr><tr><td>Kytococcus sedentarius</td><td>Lactobacillus gasseri</td></tr><tr><td>Lactobacillus jensenii</td><td>Lactobacillus rhamnosus</td></tr><tr><td>Lactococcus garvieae</td><td>Lactococcus lactis</td></tr><tr><td>Leclercia adecarboxylata</td><td>Legionella longbeachae</td></tr><tr><td>Legionella pneumophila</td><td>Leuconostoc citreum</td></tr><tr><td>Leuconostoc mesenteroides</td><td>Leuconostoc pseudomesenteroides</td></tr><tr><td>Listeria monocytogenes</td><td>Macrococcus caseolyticus</td></tr><tr><td>Mannheimia haemolytica group</td><td>Micrococcus luteus</td></tr><tr><td>Micrococcus lylae Moraxella sg Branhamella catarrhalis*</td><td>Mobiluncus curtisii</td></tr><tr><td></td><td>Moraxella sg Moraxella nonliquefaciens *</td></tr><tr><td>Moraxella sg Moraxella osloensis*</td><td>Morganella morganii</td></tr><tr><td>Myroides odoratimimus</td><td>Myroides odoratus</td></tr><tr><td>Neisseria bacilliformis</td><td>Neisseria cinerea</td></tr><tr><td>Neisseria elongata</td><td>Neisseria flavescens / subflava group</td></tr><tr><td>Neisseria gonorrhoeae</td><td>Neisseria lactamica</td></tr><tr><td>Neisseria meningitidis</td><td>Neisseria sicca group</td></tr><tr><td>Neisseria weaveri</td><td>Nocardia brasiliensis</td></tr><tr><td>Nocardia cyriacigeorgica</td><td>Nocardia farcinica group</td></tr><tr><td>Nocardia nova</td><td>Nocardia otitidiscaviarum</td></tr><tr><td>Ochrobactrum anthropi</td><td>Oligella ureolytica</td></tr><tr><td>Oligella urethralis</td><td>Pantoea agglomerans</td></tr><tr><td>Parabacteroides distasonis</td><td>Parabacteroides goldsteinii</td></tr><tr><td>Parabacteroides johnsonii/merdae group</td><td>Parvimonas micra</td></tr><tr><td>Pasteurella multocida</td><td>Pediococcus acidilactici</td></tr><tr><td>Pediococcus pentosaceus</td><td>Peptoniphilus harei group</td></tr><tr><td>Peptostreptococcus anaerobius</td><td>Plesiomonas shigelloides</td></tr><tr><td>Pluralibacter gergoviae</td><td>Porphyromonas gingivalis</td></tr><tr><td>Porphyromonas somerae</td><td>Prevotella bivia</td></tr><tr><td>Prevotella buccae</td><td>Prevotella denticola</td></tr><tr><td>Prevotella intermedia</td><td>Prevotella melaninogenica</td></tr><tr><td colspan="2">Bacteria:</td></tr><tr><td>Propionibacterium acnes</td><td>Proteus mirabilis</td></tr><tr><td>Proteus vulgaris group</td><td>Providencia rettgeri</td></tr><tr><td>Providencia stuartii</td><td>Pseudomonas aeruginosa</td></tr><tr><td>Pseudomonas fluorescens group</td><td>Pseudomonas oryzihabitans</td></tr><tr><td>Pseudomonas putida group</td><td>Pseudomonas stutzeri</td></tr><tr><td>Ralstonia pickettii</td><td>Rhizobium radiobacter</td></tr><tr><td>Rothia aeria</td><td>Rothia dentocariosa</td></tr><tr><td>Rothia mucilaginosa</td><td>Salmonella sp**</td></tr><tr><td>Serratia fonticola</td><td>Serratia liquefaciens</td></tr><tr><td>Serratia marcescens</td><td>Serratia odorifera</td></tr><tr><td>Serratia plymuthica</td><td>Serratia rubidaea</td></tr><tr><td>Sphingobacterium multivorum</td><td>Sphingobacterium spiritivorum</td></tr><tr><td>Sphingomonas paucimobilis group</td><td>Staphylococcus aureus</td></tr><tr><td>Staphylococcus auricularis</td><td>Staphylococcus capitis</td></tr><tr><td>Staphylococcus caprae</td><td>Staphylococcus carnosus</td></tr><tr><td>Staphylococcus cohnii</td><td>Staphylococcus delphini</td></tr><tr><td>Staphylococcus epidermidis</td><td>Staphylococcus equorum</td></tr><tr><td>Staphylococcus felis</td><td>Staphylococcus haemolyticus</td></tr><tr><td>Staphylococcus hominis</td><td>Staphylococcus intermedius</td></tr><tr><td>Staphylococcus lentus</td><td>Staphylococcus lugdunensis</td></tr><tr><td>Staphylococcus pasteuri</td><td>Staphylococcus pettenkoferi</td></tr><tr><td>Staphylococcus pseudintermedius</td><td>Staphylococcus saccharolyticus</td></tr><tr><td>Staphylococcus saprophyticus</td><td>Staphylococcus schleiferi</td></tr><tr><td>Staphylococcus sciuri</td><td>Staphylococcus simulans</td></tr><tr><td>Staphylococcus vitulinus</td><td>Staphylococcus warneri</td></tr><tr><td>Staphylococcus xylosus</td><td>Stenotrophomonas maltophilia</td></tr><tr><td>Streptococcus agalactiae</td><td>Streptococcus anginosus</td></tr><tr><td>Streptococcus canis</td><td>Streptococcus constellatus</td></tr><tr><td>Streptococcus dysgalactiae</td><td>Streptococcus equi</td></tr><tr><td>Streptococcus gallolyticus</td><td>Streptococcus gordonii</td></tr><tr><td>Streptococcus intermedius</td><td>Streptococcus lutetiensis</td></tr><tr><td>Streptococcus mitis/oralis roup</td><td>Streptococcus mutans</td></tr><tr><td>Streptococcus parasanguinis</td><td>Streptococcus pneumoniae</td></tr><tr><td>Streptococcus pyogenes</td><td>Streptococcus salivarius / vestibularis</td></tr><tr><td>Streptococcus sanguinis</td><td>group Streptococcus sobrinus</td></tr><tr><td>Streptococcus thermophilus</td><td>Sutterella wadsworthensis</td></tr><tr><td>Trueperella bernardiae</td><td>Turicella otitidis</td></tr><tr><td>Vagococcus fluvialis</td><td>Veillonella parvula group</td></tr><tr><td>Vibrio parahaemolyticus</td><td>Vibrio vulnificus</td></tr><tr><td>Weeksella virosa</td><td>Yersinia enterocolitica</td></tr><tr><td>Yersinia frederiksenii</td><td>Yersinia intermedia</td></tr><tr><td>Yersinia kristensenii</td><td>Yersinia pseudotuberculosis</td></tr><tr><td>* = subgenus</td><td></td></tr><tr><td>** = species</td><td></td></tr></table>

<table><tr><td colspan="2">Yeasts:</td></tr><tr><td>Candida albicans</td><td>Candida boidinii</td></tr><tr><td>Candida dubliniensis</td><td>Candida duobushaemulonii</td></tr><tr><td>Candida famata</td><td>Candida glabrata</td></tr><tr><td>Candida guilliermondii</td><td>Candida haemulonis</td></tr><tr><td>Candida inconspicua</td><td>Candida intermedia</td></tr><tr><td>Candida kefyr</td><td>Candida krusei</td></tr><tr><td>Candida lambica</td><td>Candida lipolytica</td></tr><tr><td>Candida lusitaniae</td><td>Candida metapsilosis</td></tr><tr><td>Candida norvegensis</td><td>Candida orthopsilosis</td></tr><tr><td>Candida parapsilosis</td><td>Candida pararugosa</td></tr><tr><td>Candida pelliculosa</td><td>Candida tropicalis</td></tr><tr><td>Candida valida</td><td>Candida zeylanoides</td></tr><tr><td>Cryptococcus gattii</td><td>Cryptococus neoformans var grubii*</td></tr><tr><td>Cryptococcus neoformans var neoformans *</td><td>Cyberlindnera jadinii</td></tr><tr><td>Geotrichum candidum</td><td>Geotrichum capitatum</td></tr><tr><td>Kloeckera apiculata</td><td>Malassezia furfur</td></tr><tr><td>Malassezia pachydermatis</td><td>Pichia ohmeri</td></tr><tr><td>Rhodotorula mucilaginosa</td><td>Saccharomyces cerevisiae</td></tr><tr><td>Trichosporon asahii</td><td>Trichosporon inkin</td></tr><tr><td>Trichosporon mucoides group</td><td></td></tr><tr><td colspan="2">* = variety</td></tr></table>

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s):

The MALDI Biotyper CA System is for prescription use only in accordance with 21 CFR

4. Special instrument requirements:

Mass Spectrometer: microflex LT/SH mass spectrometer

Target Plates: US IVD 48 Spot Target

Reagents:

• US IVD Bacterial Test Standard (BTS) • US IVD HCCA portioned ( $\mathfrak { a }$ -Cyano-4-hydroxycinnamic acid)

Database: MALDI Biotyper for Clinical Applications (MBT-CA)

Software:

• MBT-CA System Software Package: MBT-CA System client software displaying the user interface MBT-CA System Server MBT-CA System DB Server   
• flexControl Software Package: GTPS firmware flexControl acquisition software

# I. Device Description:

The MBT-CA System consists of the microflex LT/SH mass spectrometer, reference library, kit reagents (US IVD HCCA, US IVD Bacterial Test Standard), US IVD 48 Spot Target or MBT Biotarget 96 US IVD plate, and software. The MALDI Biotyper CA System with closed safety covers is a Class 1 Laser product. With the safety cover opened it becomes a Class 4 Laser product. The laser is a $3 3 7 \mathrm { n m }$ fixed focus, nitrogen laser. The MALDI Biotyper CA System is also referred to as the MBT-CA System.

The reference library, MALDI Biotyper for Clinical Applications, includes type strains, clinical strains and culture collection strains. The MALDI Biotyper CA System reference library was established by analyzing the type strain from each claimed species combined with 5 to 10 additional strains from the same species provided by clinical laboratories or different commercial strain collections for a total of 528 strains. Library mass spectra used for matching contain up to 70 peaks. The MALDI Biotyper for Clinical Applications is also referred to as the MBT-CA.

US IVD HCCA portioned $\mathbf { \dot { a } }$ -Cyano-4-hydroxycinnamic acid) is a solution that is used when processing test organisms for identification on the MALDI Biotyper CA System. US IVD HCCA is reconstituted in accordance with instructions provided using recommended solvent. $1 . 0 \mu \mathrm { L }$ of the matrix is added to the spot with the sample and allowed to dry.

US IVD Bacterial Test Standard (BTS) is an in-vitro-diagnostic product used for quality control and validation of the microflex LT/SH mass spectrometers. US IVD BTS contains a manufactured extract of Escherichia coli DH5 alpha that demonstrates a characteristic peptide and protein profile mass spectrum, when tested on the MALDI Biotyper CA System. US IVD BTS is spiked with two additional proteins that extend the upper boundary of the mass range of the US IVD BTS. The overall mass range covered by US IVD BTS is 3.6 to $1 7 \mathrm { k D a }$ .

US IVD 48 Spot Target plates are reusable steel plates which have been developed for the preparation and identification of test organisms using the MALDI Biotyper CA System. The target allows for the identification of 48 test organisms. The target has five cross-joint positions which should be used for US IVD BTS control. Target plate cleaning is performed after each run.

MALDI Biotyper CA System client software displays a user-interface which guides the user through the MALDI Biotyper CA System workflow. The MALDI Biotyper CA System client also interfaces to the flexControl software for automated acquisition of mass spectra on the microflex LT/SH instrument.

The MALDI Biotyper CA System server communicates with the MALDI Biotyper CA System client and the MBT-DB server. It performs preprocessing on acquired spectra, and matches peaks lists against the Main Spectrum (reference pattern, (MSP)) for matching and calculates the score value (log (score)).

The MBT-DB server stores all information for the MALDI Biotyper CA System. The MBTDB maintains spectra data (creation information and mass/intensity lists), project data (results of defined and executed runs), method data (parameter lists for spectra preprocessing and identification), user management data, reference patterns and other peak lists plus additional maintenance data.

GTPS firmware communicates with the flexControl PC software, controls and monitors the vacuum, moves the sample carrier and performs the docking of the target plate, controls and monitors high voltages in the ion source, generates trigger signals, and monitors instrument status.

The flexControl acquisition software communicates with the MALDI Biotyper CA System client, loads automatic run jobs, communicates with the GTPS firmware, communicates with the laser in the microflex LT/SH instrument, sets the acquisition parameters in the digitizer and reads the acquired data from the digitizer, performs automated data acquisition, evaluates acquired spectra, adjusts the laser power during automatic data acquisition, performs a recalibration of the time-of-flight to mass transformation, stored acquired spectra on disk and performs source cleaning. The flexControl software does not display a user interface.

The optional Honeywell (Hyperion $1 3 0 0 \mathrm { g } )$ Barcode Reader USB cable is connected to the MALDI Biotyper CA System computer. The barcode reader scans the unique ten-digit target ID which appears in the Target ID box on the target plate. After the target ID has been entered, the a new Run page opens and the ten-digit target ID appears as the Plate Id and is appended to the Run name. Sample identifications are entered into the computer corresponding to the target plate position for that run.

# J. Substantial Equivalence Information:

1. Predicate device name(s): MALDI Biotyper CA System

2. Predicate 510(k) number(s): K142677

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">SIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">NEW DEVICE (K163536)MALDI Biotyper CA System</td><td colspan="1" rowspan="1">PREDICATE DEVICE (K142677)MALDI Biotyper CA System</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PEX</td><td colspan="1" rowspan="1">PEX</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The MALDI Biotyper CA System is a massspectrometer system using matrix-assistedlaser desorption/ionization - time of flight(MALDI-TOF) for the identification ofmicroorganisms cultured from humanspecimens.The MALDI Biotyper CA System is aqualitative in vitro diagnostic deviceindicated for use in conjunction with otherclinical and laboratory findings to aid in thediagnosis of bacterial and yeast infections.</td><td colspan="1" rowspan="1">The MALDI Biotyper CA System is a massspectrometer system using matrix-assistedlaser desorption/ionization - time of flight(MALDI-TOF) for the identification ofmicroorganisms cultured from humanspecimens.The MALDI Biotyper CA System is aqualitative in vitro diagnostic deviceindicated for use in conjunction with otherclinical and laboratory findings to aid in thediagnosis of bacterial and yeast infections.</td></tr><tr><td>Characteristic</td><td>NEW DEVICE (K163536) MALDI Biotyper CA System</td><td>PREDICATE DEVICE (K142677) MALDI Biotyper CA System</td></tr><tr><td>Sample type</td><td>Isolated colony from any patient sample source. Acceptable media: Columbia blood agar with 5% sheep blood (Gram-negative bacteria) Trypticase soy agar with 5% sheep blood (Gram-negative bacteria, Gram- positive bacteria, yeasts) Chocolate agar (Gram-negative bacteria, Gram-positive bacteria) MacConkey Agar (Gram-negative bacteria) Columbia CNA agar with 5% sheep blood (Gram-positive bacteria) Brucella Agar with 5% horse blood (Gram-negative anaerobic bacteria, Gram- positive anaerobic bacteria) positive anaerobic bacteria) CDC anaerobe Agar with 5% sheep</td><td>Isolated colony from any patient sample source. Acceptable media: Columbia blood agar with 5% sheep blood (Gram-negative bacteria) Trypticase soy agar with 5% sheep blood (Gram-negative bacteria, Gram- positive bacteria, yeasts) Chocolate agar (Gram-negative bacteria, Gram-positive bacteria) MacConkey Agar (Gram-negative bacteria) Columbia CNA agar with 5% sheep blood (Gram-positive bacteria) Brucella Agar with 5% horse blood (Gram-negative anaerobic bacteria, Gram-</td></tr><tr><td>Type of Test Matrix</td><td>CDC anaerobe laked sheep blood Agar with kanamycin and vancomycin (Gram-negative anaerobic bacilli) Bacteroides bile esculin Agar with amikacin (Bacteroides species) Clostridium difficile Agar with 7% sheep blood (Clostridium difficile) Sabouraud-Dextrose Agar (Yeasts) Brain Heart Infusion Agar (Yeasts) Campylobacter Agar with 5 Antimicrobics and 10% Sheep Blood (Campylobacter species) Bordet Gengou Agar with 15% sheep blood (Bordetella species) Automated Mass Spectrometry System αCyano-4-hydroxycinnamic acid</td><td>anaerobic bacteria, Gram-positive anaerobic bacteria) CDC anaerobe laked sheep blood Agar with kanamycin and vancomycin (Gram-negative anaerobic bacilli) Bacteroides bile esculin Agar with amikacin (Bacteroides species) Clostridium difficile Agar with 7% sheep blood (Clostridium difficile) Sabouraud-Dextrose Agar (Yeasts) Brain Heart Infusion Agar (Yeasts) Campylobacter Agar with 5 Antimicrobics and 10% Sheep Blood (Campylobacter species) Bordet Gengou Agar with 15% sheep blood (Bordetella species) Automated Mass Spectrometry System α-Cyano-4-hydroxycinnamic acid</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">NEW DEVICE (K163536)MALDI Biotyper CA System</td><td colspan="1" rowspan="1">PREDICATE DEVICE (K142677)MALDI Biotyper CA System</td></tr><tr><td colspan="1" rowspan="1">Method ofTesting</td><td colspan="1" rowspan="1">Bacteria &amp; Yeast: Direct testingIf after initial analysis the log(score) isreported at &lt;2.00, organisms may beprocessed using the Extraction (Ext)procedure or extended Direct Transfer(eDT, 70% aqueous formic acid)procedure. If eDT procedure still yields log(score) &lt;2.00, organisms may be processedvia Ext procedure.</td><td colspan="1" rowspan="1">Bacteria &amp; Yeast: Direct testingIf after initial analysis the log(score) isreported at &lt;2.00, organisms may beprocessed using the Extraction (Ext)procedure or extended Direct Transfer(eDT, 70% aqueous formic acid) procedure.If eDT procedure still yields log (score)&lt;2.00, organisms may be processed via Extprocedure.</td></tr><tr><td colspan="1" rowspan="1">ResultReporting</td><td colspan="1" rowspan="1">Organism identification is reported with highconfidence if the log(score) is ≥2.00An organism identification is reportedwith low confidence if the log (score) isbetween 1.70 and &lt;2.00.</td><td colspan="1" rowspan="1">Organism identification is reported with highconfidence if the log(score) is ≥2.00An organism identification is reportedwith low confidence if the log (score) isbetween 1.70 and &lt;2.00.</td></tr><tr><td colspan="1" rowspan="1">MatchingAlgorithm</td><td colspan="1" rowspan="1">Calculates matches by comparing a newspectrum against each single reference entryof a reference database.</td><td colspan="1" rowspan="1">Calculates matches by comparing a newspectrum against each single reference entryof a reference database.</td></tr><tr><td colspan="1" rowspan="1">Mass range</td><td colspan="1" rowspan="1">2,000 - 20,000 m/z</td><td colspan="1" rowspan="1">2,000 - 20,000 m/z</td></tr><tr><td colspan="1" rowspan="1">Calibration</td><td colspan="1" rowspan="1">Bruker US IVD Bacterial Test Standard</td><td colspan="1" rowspan="1">Bruker US IVD Bacterial Test Standard</td></tr><tr><td colspan="1" rowspan="1">Database</td><td colspan="1" rowspan="1">MALDI Biotyper Reference Library forClinical Applications (MBT-CA)-update</td><td colspan="1" rowspan="1">MALDI Biotyper Reference Library forClinical Applications (MBT-CA)</td></tr></table>

<table><tr><td rowspan=1 colspan=3>DIFFERENCES</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>NEW DEVICE (K163536)MALDI Biotyper CA System</td><td rowspan=1 colspan=1>PREDICATE DEVICE (K142677)MALDI Biotyper CA System</td></tr><tr><td rowspan=1 colspan=1>System Update</td><td rowspan=1 colspan=1>System claim additional organisms but noadditional changes</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Media</td><td rowspan=1 colspan=1>Additional Media validated:•  Buffered Charcoal Yeast ExtractAgar (Legionella species)Buffered Charcoal Yeast ExtractSelective Agar with polymyxin,anisomycin and vancomycin(Nocardia species)Modified Thayer-Martin Agar(Neisseria species)</td><td rowspan=1 colspan=1>See Similarities</td></tr><tr><td rowspan=1 colspan=1>MALDI-TOFMS instruments</td><td rowspan=1 colspan=1>Bruker microflex LT/SH (benchtop)Bruker microflex LT/SH smart (benchtop)</td><td rowspan=1 colspan=1>Bruker microflex LT/SH (benchtop)</td></tr><tr><td rowspan=1 colspan=1>MALDI TargetPlates</td><td rowspan=1 colspan=1>US IVD 48 Spot Target(48 positions reusable steel targets)MBT Biotarget 96 US IVD(96 positions disposable targets)</td><td rowspan=1 colspan=1>US IVD 48 Spot Target(48 positions reusable steel targets)</td></tr></table>

The differences do not affect substantial equivalence of the predicate and the new divice. Both systems are mass spectrometers using matrix-assisted laser desorption/ionization-time to flight (MALDI-TOF) for the identification of microorganisms cultured from human specimens. The differences noted above do not impact the intended use and do not raise new questions related to the safety and effectiveness of the test (new) device.

K. Standard/Guidance Document Referenced (if applicable):   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Standards No.</td><td rowspan=1 colspan=1>RecognitionNo.(FDA)</td><td rowspan=1 colspan=1>Standards Title</td><td rowspan=1 colspan=1>Date</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CLSI MM-18A</td><td rowspan=1 colspan=1>7-192</td><td rowspan=1 colspan=1>Criteria for Identification of Bacteria andFungi by DNA Target Sequencing;Approved Guideline, 1st Edition</td><td rowspan=1 colspan=1>4/28/2008</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CLSI EP09-A2-IR</td><td rowspan=1 colspan=1>7-92</td><td rowspan=1 colspan=1>Method Comparison and Bias EstimationUsing Patient Samples; Approved Guideline--Second Edition (Interim Revision),</td><td rowspan=1 colspan=1>7/30/2010</td></tr></table>

# L. Test Principle:

Biochemical methods are currently the most commonly used approach for the identification of microorganisms. Organisms are tested against a range of reagents and organism identification is based on a microorganism's reaction to these reagents.

The MBT-CA System uses a different methodology for organism identification based on unique protein patterns of the microorganisms obtained from mass spectrometry. The test organism's spectrum (a pattern of mass peaks) is compared with a reference spectra library (database). Using biostatistical analysis, a probability ranking of the organism identification is generated. The probability ranking is represented as a log (score) between 0.00 and 3.00. Organism identification is reported with high confidence if the log (score) is ${ \ge } 2 . 0 0$ . An organism identification is reported with low confidence if the log (score) is between 1.70 and $< 2 . 0 0$ .

Organisms to be identified with the MBT-CA System should be isolated for purity on appropriate isolation media.

Direct Transfer procedure (DT): An individual colony from a subculture plate is transferred to a selected position on a US IVD 48 Spot Target plate or a MBT Biotarget 96 US IVD Target plate (targets) and overlaid with US IVD HCCA portioned (matrix). The standard solvent $50 \%$ acetonitrile $/ 4 7 . 5 \% \mathrm { H } 2 \mathrm { O } / 2 . 5 \%$ trifluoroacetic acid) in the matrix solution extracts proteins (mainly ribosomal proteins, which are present in high concentration) from the microorganisms. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System. If after initial analysis the log (score) is reported as $< 2 . 0 0$ , organisms can be processed using the extended Direct Transfer (eDT) procedure or the Extraction procedure and analysis repeated. If eDT is employed and log (score) is reported as $< 2 . 0 0$ , reanalysis via the Extraction procedure may be used.

extended Direct Transfer procedure (eDT): If DT analysis yields a $( \log ( \mathrm { s c o r e } ) < 2 . 0 0 )$ result, an individual colony from a subculture plate may be transferred to a selected position on a target and overlaid with $70 \%$ aqueous formic acid solution. The target is air-dried and then matrix is overlaid. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System. If a high confidence result is not achieved (log (score) is reported at $< 2 . 0 0$ ), organisms can be processed using the Extraction procedure and analysis repeated.

Extraction procedure $( E x t )$ : If after initial analysis and eDT procedure the log (score) is reported at $< 2 . 0 0$ , organisms are processed using the Extraction procedure and analysis repeated. For this purpose, isolated colonies from the subculture plate are extracted in accordance with instructions of the user manual (Ext sample preparation procedure). An aliquot of extracted material is transferred to a selected position on a target, air-dried and then overlaid with matrix material. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System.

MALDI-TOF Analysis: Samples are analyzed using MALDI (matrix-assisted laser desorption/ionization) TOF (time of- flight) mass spectrometry. The matrix transfers protons onto the extracted proteins and absorbs UV light. A laser in the MALDI- TOF mass spectrometer irradiates the matrix sample composite, causing evaporation and release of positively charged intact proteins and peptides ("soft" ionization technique). These ions are electrostatically accelerated over a short distance and arrive in the flight tube at a mass-dependent speed. As different proteins/peptides have different masses, ions arrive at the detector at different times (time of flight). The system measures the time (in the nanosecond range) between pulsed acceleration and the corresponding detector signal, the speed is converted into an exact molecular mass. The mass-to-charge ratio of an ion is proportional to the square of its drift time.

Highly abundant microbial proteins (mainly ribosomal proteins) result in a mass spectrum with characteristic mass and intensity distribution. It is species-specific for many bacteria and is interpreted as a molecular fingerprint to identify the test organism.

Data acquisition is controlled with MBT-CA System software. The spectrum of the unknown organism is first transformed into a peak list. Using a biostatistical algorithm, this peak list is compared to the reference peak lists of organisms in the reference library (database) and a log (score) is generated. A higher log (score) indicates a higher degree of similarity to the organism in the reference library. Organism identification is reported with high confidence if the log (score) is ${ \ge } 2 . 0 0$ . An organism identification is reported with low confidence if the log (score) is between 1.70 and $< 2 . 0 0$ .

The log (score) ranges, defined in the MBT-CA System, are indicative of the probability of organism identification. Results should be reviewed by a trained microbiologist and final organism identification should be based on all relevant information available. This information includes but is not limited to: Gram staining, colony morphology, growth characteristics, sample matrix, etc.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility

The reproducibility study for Gram-negative and Gram-positive aerobic bacteria, Gramnegative and Gram-positive microaerophilic bacteria, Gram-negative and Gram-positive anaerobic bacteria and yeasts was carried out to confirm day-to-day reproducibility and precision of the MALDI Biotyper CA System at different clinical study sites. The study was conducted for five days with two runs and two operators each day per clinical site.

The sources of variability tested were:

Two operators/each clinical study site Three clinical study sites

At least two target plates/each clinical study site • Four microflex LT/SH instruments

Ten well-characterized organisms were chosen for this study and tested in duplicate via Direct Transfer procedure in accordance with product instructions. When the DT log (score) was $< 2 . 0 0$ , per product instructions, the test organism was tested following the extended Direct Transfer and Extraction procedure. The results are shown in the table below.

Table: Reproducibility Study Summary:   
\* REPRO-02: Organism skipped as MBT-CA ID applying DT procedure was not confirmed via eDT and Ext procedure.   

<table><tr><td rowspan=1 colspan=1>Blinded Test Organism</td><td rowspan=1 colspan=1>ReproducibilityPanel</td><td rowspan=1 colspan=1>Site A:MBT-CA ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>Site B:MBT-CA ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>Site C:MBT-CAID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>Dermabacter hominis</td><td rowspan=1 colspan=1>REPRO-01</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>REPRO-02*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Nocardia farcinica group</td><td rowspan=1 colspan=1>REPRO-03</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>REPRO-04</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Clostridium tertium</td><td rowspan=1 colspan=1>REPRO-05</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Facklamia hominis</td><td rowspan=1 colspan=1>REPRO-06</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides caccae</td><td rowspan=1 colspan=1>REPRO-07</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Trueperella bernardiae</td><td rowspan=1 colspan=1>REPRO-08</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>REPRO-09</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>REPRO-10</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>179/180 (99%)</td><td rowspan=1 colspan=1>179/180 (99%)</td><td rowspan=1 colspan=1>179/180 (99%)</td></tr></table>

The data showed that $9 9 \%$ of all test organisms were correctly identified with a log (score) ${ \ge } 2 . 0 0$ at each test site after final extraction testing confirming reproducibility of the MALDI Biotyper CA System.

The reproducibility of the MALDI Biotyper smart CA System was evaluated at one laboratory site (in-house). Reproducibility was assessed with ten well-characterized organisms that were chosen for this study and tested in duplicate via Direct Transfer procedure in accordance with MALDI Biotyper CA System instruction for use. When the DT log (score) was $< 2 . 0 0$ , per product instructions, the test organism was tested following the extended Direct Transfer and Extraction procedure. The study was conducted for five days with two runs and two operators each day per clinical site. Results are summarized below:

Table: Reproducibility Study Summary:   

<table><tr><td>Blinded Test Organism</td><td>Reproducibility Panel</td><td>Site C: MBT-smart CA ID (DT+eDT+Ext)</td></tr><tr><td>Dermabacter hominis</td><td>REPRO-01</td><td>20/20 (100%)</td></tr><tr><td>Listeria monocytogenes</td><td>REPRO-02*</td><td>N/A</td></tr><tr><td>Nocardia farcinica group</td><td>REPRO-03</td><td>20/20 (100%)</td></tr></table>

\* REPRO-02: Organism skipped as MBT-CA ID applying DT procedure was not confirmed via eDT and Ext procedure.   

<table><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>REPRO-04</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Clostridium tertium</td><td rowspan=1 colspan=1>REPRO-05</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Facklamia hominis</td><td rowspan=1 colspan=1>REPRO-06</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides caccae</td><td rowspan=1 colspan=1>REPRO-07</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Trueperella bernardiae</td><td rowspan=1 colspan=1>REPRO-08</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>REPRO-09</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>REPRO-10</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>180/180 (100%)</td></tr></table>

The data show that $100 \%$ of all test organisms were correctly identified with a log (score) ${ > } 2 . 0 0$ at each test site after final extraction testing confirming reproducibility of the MALDI Biotyper smart CA System.

b. Linearity/assay reportable range:

Not applicable, qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Calibrator:

US IVD BTS is used for mass spectrum calibration and optimization as well as a performance control for the identification of microorganisms with the MALDI Biotyper CA System. US IVD BTS contains a manufactured extract of Escherichia coli DH5 alpha that demonstrates a characteristic peptide and protein profile mass spectrum, when tested on the MALDI Biotyper CA System. US IVD BTS is spiked with two additional proteins that extend the upper boundary of the mass range of the US IVD BTS. The overall mass range covered by US IVD BTS is 3.6 to $1 7 \mathrm { k D a }$ . Two US IVD BTS control positions on a US IVD 48 Spot Target are selected and inoculated with US IVD BTS solution. The US IVD BTS solution is allowed to dry at room temperature and then overlaid with reconstituted US IVD HCCA portioned solution. If US IVD BTS does not meet all required performance specifications, the test run will be invalid. If US IVD BTS is not inoculated onto a target prior to processing, the test run will be invalid.

# Controls:

Klebsiella pneumoniae, Haemophilus influenzae, Proteus vulgaris, Pseudomonas aeruginosa, and Esherichia coli, were used as controls. Of the 193 Quality Control runs conducted during the course of the method comparison study, there were five instances where a Quality Control organism failed to yield an expected result. As a result, all isolates included in that plate run were repeated using a fresh QC organism transfer. The overall plate repeat rate was $2 . 6 \%$ (5/193).

d. Assay cut-off:

The assay cut-off was established and reported in 510(k) K142677.

e. Detection limit:

The Limit of Detection/Dynamic Range study for Gram-negative bacteria was previously performed and reported in 510(k) K130831 (pp. 46-50). The Limit of Detection/Dynamic Range study for Gram-positive bacteria and yeasts was performed and reported in 510(k) K142677.

f. Analytical specificity:

The Interference & Specificity Study was previously established and reported in 510(k) K130831.

g. Sample stability studies:

Sample Stability after Matrix Overlay:

The sample stability study on target plates for Gram-negative bacteria was previously validated and reported in 510(k) K130831. The sample stability on target plates for Grampositive bacteria and yeasts was validated and reported in 510(k) K142677.

US IVD Bacterial Test Standard (BTS)

BTS Stability was established and described in 510(k) K130831.

HCCA portioned (Matrix) Stability

HCCA portioned (Matrix) Stability was established and described in 510(k) K130831.

Target plates stability:

Target plate stability was established and described in 510(k) K142677.

h. Carry-Over and Cross Contamination:

The carry-over, cross-contamination and target cleaning study was previously performed and reported in 510(k) K130831.

# i. Influence of Agar Media

The validation of sample preparation of test organism to demonstrate that culture media inoculated onto US IVD 48 Spot targets with or without an organism present does not interfere with system performance was previously performed and reported in 510(k) K130831.

# j. Organism Stability

Media and Colony Stability

In accordance with device instructions for use, primary or secondary isolation plates of recommended media [Trypticase Soy Agar with $5 \%$ sheep blood (TSA), Buffered Charcoal Yeast Extract Agar (BCYE), Buffered Charcoal Yeast Extract Selective Agar with polymyxin, anisomycin and vancomycin (BCYE/PAV) and Modified Thayer-Martin Agar (MTM))] may be held for up to 12 hours at room temperature prior to testing on the MALDI Biotyper CA System.

In the previous submission (K142677) TSA medium was only validated for Gram-positive /Gram-negative aerobic bacteria and yeast. This submission was intended to establish the suitability of this medium for Gram-positive $/ .$ -negative microaerophilic and anaerobic bacteria.

Testing was conducted using one Gram-negative bacterium, one microaerophilic

Gram-negative bacterium, one Gram-positive bacterium, one microaerophilic Grampositive bacterium, two Legionella species, three Nocardia species and three Neisseria species, at varying incubation time points in replicates of eight. Additionally, thirteen clinical isolates derived from Neisseria species were tested in duplicate at varying incubation time points. Results are summarized below:

Table: TSA Summary: Anaerobic Gram-negative/Gram-positive Bacteria (applying MBT-CA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ex)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ex)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table: TSA Summary: Anaerobic Gram-negative/Gram-positive Bacteria (applying eDT and Ext procedure)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>&gt;2.00 ID (eDT)</td><td rowspan=1 colspan=1>false ID (eDT)</td><td rowspan=1 colspan=1>&gt;2.00 ID (Ext)</td><td rowspan=1 colspan=1>false ID (Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr></table>

Table: TSA Summary: Microaerophilic Gram-negative/Gram-positive Bacteria (applying MBTCA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C,, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table: TSA Summary: Microaerophilic Gram-negative/Gram-positive Bacteria (applying eDT and Ext procedure)   

<table><tr><td colspan="1" rowspan="1">Testing Condition</td><td colspan="1" rowspan="1">&gt;2.00 ID (eDT)</td><td colspan="1" rowspan="1">false ID (eDT)</td><td colspan="1" rowspan="1">&gt;2.00 ID (Ext)</td><td colspan="1" rowspan="1">false ID (Ext)</td></tr><tr><td colspan="1" rowspan="1">18h/37°C, 0h/25°C</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td colspan="1" rowspan="1">18h/37°C,12h/25°C</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td colspan="1" rowspan="1">24h/37°C,0h/25°C</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td colspan="1" rowspan="1">24h/37°C,12h/25°C</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td>48h/37°C, 0h/25°C</td><td>4/4</td><td>0/4</td><td>4/4</td><td>0/4</td></tr><tr><td>48h/37°C, 12h/25°C</td><td>4/4</td><td>0/4</td><td>4/4</td><td>0/4</td></tr></table>

Table: BCYE Summary: Legionella organisms (applying MBT-CA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>18h/37°C,12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C,,0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C,,12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C,, 0h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 1,12h/25°C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table: BCYE Summary: Legionella organisms (applying eDT and Ext procedure)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>&gt;2.00 ID (eDT)</td><td rowspan=1 colspan=1>false ID (eDT)</td><td rowspan=1 colspan=1>&gt;2.00 ID (Ext)</td><td rowspan=1 colspan=1>false ID (Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td></tr></table>

Table: BCYE/PAV Summary: Nocardia organisms (applying MBT-CA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ex)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>18h/37°C,12h/25°</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C,0h/25°C</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C,, 0h/25°C</td><td rowspan=1 colspan=1>19/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°</td><td rowspan=1 colspan=1>20/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table: BCYE/PAV Summary: Nocardia organisms (applying eDT and Ext procedure)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>&gt;2.00 ID (eDT)</td><td rowspan=1 colspan=1>false ID (eDT)</td><td rowspan=1 colspan=1>&gt;2.00 ID (Ext)</td><td rowspan=1 colspan=1>false ID (Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>24h/37°C,, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr></table>

Table: MTM Summary: Neisseria organisms (applying MBT-CA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>17/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>19/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>20/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>20/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr></table>

Table: MTM Summary: Neisseria organisms (applying eDT and Ext procedure)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>&gt;2.00 ID (eDT)</td><td rowspan=1 colspan=1>false ID (eDT)</td><td rowspan=1 colspan=1>&gt;2.00 ID (Ext)</td><td rowspan=1 colspan=1>false ID (Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>24h/37°C,12h/25°C</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>0/6</td></tr></table>

The study results confirm that the following culture media can be used on the MALDI Biotyper CA System (18 – 48 hours incubation at $3 7 \left( \pm 2 \right) { } ^ { \circ } { \mathrm { C } } )$ :

Anaerobic Gram-negative/-positive bacteria and microaerophilic Gram-negative/Grampositive bacteria:

Trypticase Soy Agar with $5 \%$ sheep blood (TSA)

Legionella organisms:

• Buffered Charcoal Yeast Extract Agar (BCYE)

Nocardia organisms:

Buffered Charcoal Yeast Extract Selective Agar with polymyxin, anisomycin and vancomycin (BCYE/PAV)

The cultivation of Neisseria using Modified Thayer-Martin Agar (MTM) should not be longer than 24 hours.

# Organism Stability prior to MALDI Biotyper CA System Analysis

The organism stability study for Gram-negative bacteria prior to MALDI Biotyper CA analysis was previously validated and reported in 510(k) K130831. Organism stability for Gram-positive bacteria and yeasts was previously established and reported in 510(k) K142677.

Other supportive Instrument Performance Characteristics

Mixed Culture:

The validation of mixed cultures derived from a target organism and varying amounts of non-target organisms was previously performed and reported in 510(k) K130831.

# Viability Study

Viability studies with gram negative rods mixed with matrix on the target plate were not performed. The user is advised to consider all samples, microbial cultures and inoculated products as infectious. Aseptic techniques and usual precautions for handling organisms should be observed throughout the MALDI Biotyper CA System workflow according to "CLSI M29-A, Protection of Laboratory Workers from Occupationally Acquired Infections; Approved Guideline - Current revision". For additional handling precautions, refer to "Biosafety in Microbiological and Biomedical Laboratories - CDC/NIH -Latest edition".

Resolution of the Acinetobacter baumannii complex:

A study was performed to evaluate the resolution of the Acinetobacter baumannii complex to the species level after final extraction procedure. Sixty-six (66) clinical isolates of Acinetobacter baumannii complex (members: A. baumannii, A. calcoaceticus, A. nosocomialis, A. pittii) collected during initial Clinical Method Comparison protocol (see K130831) and identified by 16S rRNA and/or protein gene sequencing were re-tested using Direct Transfer, extended Direct Transfer and Extraction procedure in parallel. The results are summarized below:

Table: MBT-CA ID of clinical isolates from Acinteobacter baumannii complex   

<table><tr><td rowspan=1 colspan=1>Testsequence</td><td rowspan=1 colspan=1>Highconfidence ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>highconfidence ID(eDT)</td><td rowspan=1 colspan=1>false ID(eDT)</td><td rowspan=1 colspan=1>highconfidence ID(Ext)</td><td rowspan=1 colspan=1>false ID(Ext)</td></tr><tr><td rowspan=1 colspan=1>1st Day</td><td rowspan=1 colspan=1>65/66</td><td rowspan=1 colspan=1>0/66</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>0/66</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>0/66</td></tr><tr><td rowspan=1 colspan=1>2nd Day</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>0/66</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>0/66</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>0/66</td></tr></table>

The study results demonstrated that the Acinetobacter baumannii complex cannot be completely resolved applying MBT-CA System workflow. The reference library/software was updated to report complex Acinetobacter baumannii and Acinetobacter nosocomialis under the Acinetobacter baumannii/nosocomialis group. After formation of an Acinetobacter baumannii/nosocomialis group unambiguous MBT-CA identification with high confidence $( \log ( \mathrm { s c o r e } ) \geq 2 . 0 0 )$ of Acinetobacter baumannii/nosocomialis group, Acinetobacter calcoaceticus and Acinetobacter pittii is possible if the final extraction procedure has been applied. A matching hint will be included in the package insert which contains the following alert: The displayed species should be considered a member of the Acinetobacter baumannii complex. For organisms identified by the MBT-CA System as

Acinetobacter calcoaceticus, Acinetobacter pittii or Acinetobacter   
baumannii/nosocomialis group the full Extraction procedure (Ext) is mandatory for secure species differentiation.

Cultivation of yeast organisms at $3 7 ^ { \circ } \mathrm { C }$ :

A study was performed showing the general applicability of the MBT-CA System workflow for identification of yeasts cultivated at $3 7 \ : ( \pm 2 ) ^ { \circ } \mathrm { C }$ . Testing was conducted using three yeast species at varying incubation time points in replicates of eight. After initial testing, isolates were further tested at room temperature after twelve hours post-incubation. The results are summarized below:

Evaluation of cultivation time for yeast organisms on TSA media at $3 7 ^ { \circ } \mathrm { C }$ (applying MBT-CA System workflow)   

<table><tr><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>≥2.00 ID(DT)</td><td rowspan=1 colspan=1>false ID(DT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT)</td><td rowspan=1 colspan=1>false ID(DT+eDT)</td><td rowspan=1 colspan=1>≥2.00 ID(DT+eDT+Ext)(DT+eDT+Ext)</td><td rowspan=1 colspan=1>false ID(DT+eDT+Ext)(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>18h/37°C, 0h/25°</td><td rowspan=1 colspan=1>4/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>7/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>18h/37°C,,12h/25°C</td><td rowspan=1 colspan=1>6/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>10/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>14/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>24h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>12/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 0h/25°C</td><td rowspan=1 colspan=1>9/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>13/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr><tr><td rowspan=1 colspan=1>48h/37°C, 12h/25°C</td><td rowspan=1 colspan=1>9/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>20/24</td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/24</td></tr></table>

The study results confirmed the acceptability of $3 7 \ : ( \pm 2 ) ^ { \circ } \mathrm { C }$ cultivation of yeasts and sample colony stability of up to 12 hours.

Biological / Technical Equivalency Study - MBT-CA smart System:

A study was performed to demonstrate equivalence of the MBT-CA output when using MALDI-TOF mass spectrometers equipped with smartbeam laser technology and nitrogen laser technology.

A panel of thirty four species which are part of the MBT-CA library were measured, with the 34 species representing Gram negative, Gram positive, and yeast. All three sample preparation techniques (DT, eDT, Ext) were used in parallel (eDT and Ext were always additionally performed independent from the DT result). Each sample preparation technique (DT, eDT, Ext) was spotted eight times onto the MALDI target. The measurement was performed on two nitrogen laser instruments and three smart laser instruments.

Overall 4080 spectra were collected in this study. The performance of all spectra for each single species, each single sample prep (DT, eDT and Ext), each MBT-CA result (no ID “red”, low confidence ID “yellow”, high confidence ID “green”) for each instrument type is summarized in the table below.

Table: Biological / Technical Equivalency Study Results; Overall performance of 4080 measured samples. “High confidence ID” (green), “low confidence ID” (yellow) and “no ID” (red) are shown.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Nitrogen (544)</td><td rowspan=1 colspan=3>Smart (816)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Nitrogen (544)</td><td rowspan=1 colspan=3>Smart (816)</td></tr><tr><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>399</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>34.4%</td><td rowspan=1 colspan=1>13.2%</td><td rowspan=1 colspan=1>52.4%</td><td rowspan=1 colspan=1>36.8%</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>48.9%</td></tr><tr><td rowspan=1 colspan=1>eDT</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>446</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>633</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>eDT</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>11.2%</td><td rowspan=1 colspan=1>82.0%</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>14.7%</td><td rowspan=1 colspan=1>77.6%</td></tr><tr><td rowspan=1 colspan=1>Ext</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>803</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Ext</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>98.4%</td></tr><tr><td rowspan=2 colspan=15>∑22613612703632501835         13.8%   8.3%  77.8%  14.8%  10.2%  75.0%</td></tr><tr><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>1270</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>1835</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>13.8%</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>77.8%</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>75.0%</td></tr></table>

The study results demonstrated equivalence between the Nitrogen Laser System and the Smartbeam Laser System.

# Equivalence Study MBT Biotarget 96 US IVD (Nitrogen Laser):

Studies were performed to verify and validate the use of the MBT Biotarget 96 US IVD in conjunction with the MBT-CA System (Nitrogen Laser). The study goal was to show the equivalency between the cleared US IVD MSP 48 Target Polished Steel and the new MBT Biotarget 96 US IVD. The functionality and performance of the MBT-CA System using the MBT Biotarget 96 US IVD under varying conditions was shown. Test runs were performed always in parallel using US IVD MSP 48 Target Polished Steel plates. The following studies were performed:

# Repeatability and Precision:

The repeatability and precision study of the MBT Biotarget 96 US IVD was evaluated using three different testing-operators, three unique Biotarget 96 lots, three unique HCCA lots, three unique BTS lots and two unique MBT-CA Systems. The tests were performed on six working-days with two assay runs per day (microbial samples prepared as triplicate). The data showed $100 \%$ of test organisms were correctly MBT-CA identified at high confidence level [log (score) ${ \geq } 2 . 0 ]$ applying the MBT-CA System workflow (combination of DT, eDT and Ext procedure). The rate of MBT-CA System false identification was $0 \%$ for all test samples after final sample preparation procedure.

The study results demonstrate that both target types performed similarly.

Tolerance Range (Dynamic Range):

The Limit of Detection study was designed to establish the estimated dynamic range of sample size of Gram-negative and Gram-positive bacteria and yeast enabling automated mass spectra acquisition and species-identification using the Biotarget 96. Ten different species were analyzed representing Gram positive, Gram negative bacteria and yeast. All sample preparation techniques (DT, eDT and Ext) were conducted. Up to five dilutions steps were performed for each scenario. Cell density and cell concentration were estimated by measuring the optical density of the suspension at a wavelength of $6 0 0 \mathrm { n m }$ .

The rate of MBT-CA System false identifications at low confidence level [log(score) ${ \geq } 1 . 7 0 ]$ was $0 \%$ . The estimated dynamic range (limit of detection) of bacteria and yeast onto the MBT Biotarget 96 US IVD / US IVD MSP 48 Target Polished Steel required for MBT-CA System identification did not show significant differences.

Study results demonstrated that the estimated dynamic range for the direct (or cell equivalents using extraction) technique is $\mathsf { 5 x 1 0 } ^ { \mathsf { \overline { { 5 } } } }$ cells/uL.

Sample Stability prior to Matrix application:

This study was conducted to assess sample stability of microbial material (bacteria and yeasts isolates) on Biotarget 96 prior to addition of HCCA solution. Five species were analyzed on two instruments. Measurement was applied directly after matrix application and after 15, 30, 60 and 120 minutes and was carried out at ambient temperature $( 2 3 \pm 2 )$ $^ \circ \mathrm { C }$ to simulate working conditions. All sample preparation techniques (DT, eDT and Ext) were conducted. Each sample was spotted and measured eight times. For comparison, evaluation of sample stability was performed in parallel using Steel Targets. The rate of MBT-CA System false identifications was $0 \%$ for all tests.

The study results demonstrate that both target types performed similarly.

# Sample Stability post Matrix application:

This study was conducted to access the stability of the transferred microbial samples after addition of HCCA solution on a Biotarget 96 / Steel Target the MBT-CA workflow (DT, eDT, Ext) was tested in parallel. Two temperature conditions were analyzed in this study ( $2 1 ^ { \circ } \mathrm { C }$ and $2 5 ^ { \circ } \mathrm { C }$ ). All measurements were conducted using five different species on two MALDI instruments. All sample preparation techniques (DT, eDT and Ext) were performed and each sample was spotted eight times on both target types. The ready prepared sample stability was analyzed immediately after matrix application (standard condition) and after 4h, 8h and 24h of aging.

No MBT-CA false identifications occurred. The study results demonstrate that both target types performed similarly.

# Validation Study - Validation of 50 Representative Claimed Species:

This study was conducted to validate the Biotarget 96 performance using 50 different organisms 22 Gram-negative aerobic/anaerobic bacteria, 22 Gram-positive aerobic/anaerobic bacteria and six which are content of the FDA cleared MBT-CA System software and reference library. Single sample preparations for each of the test organisms were carried out using MBT-CA System preparation procedures (DT, eDT and Ext) in parallel. For comparison purposes, this validation study was performed in parallel using six steel Targets. In this study 50 claimed species representing Gram negative aerobic / anaerobic, Gram positive aerobic / anaerobic bacteria and yeast were analyzed. All sample preparation techniques (DT, eDT and Ext) were conducted. The rate of MBT-CA false identifications was $0 \%$ for this test. The overall identification rate of test samples at high confidence level (log (score) ${ \ge } 2 . 0$ ) was $100 \%$ .

The study results demonstrate that both target types performed similarly.

Technical Study - Mass Accuracy/Target Edge Effects:

This study was designed for validation of Biotarget 96 in terms of "mass accuracy" using BTS, MBTCA System software and reference library and at least two microflex LT/SH mass spectrometer systems. Targets were prepared with dissolved BTS, dried and then overlaid with HCCA solution.

Two parameters were evaluated during this study: Target Flatness (the mass accuracy was used as an indicator for target flatness) and edge effects (comparison of mass accuracy from edge positions vs inner target positions). For evaluation, one prominent reference protein (e.g. $6 2 5 5 . 4 \ : \mathrm { m / z } )$ within the mass range of the BTS was analyzed concerning mass reproducibility. Additionally, the log (scores) of each BTS spectrum were calculated as an indicator of identification performance.

Equivalence Study MBT Biotarget 96 US IVD (Smartbeam Laser):

The study design of this study was identical to the previous section (Equivalence Study MBT Biotarget 96 US IVD (Nitrogen Laser)).

The following six studies were performed:

• Repeatability / Precision   
• Tolerance range (dynamic range) - limit of detection (LOD)   
• Sample stability prior to matrix application   
• Sample stability post matrix application   
• Mass Accuracy / Target Edge Effects   
• Identification of 50 FDA cleared organisms according to MBT-CA System workflow.

Generally it can be stated that the technical performance between the MBT Biotarget 96 US IVD compared to the US IVD MSP 48 Target Polished Steel was comparable.

# Nocardia Study

This study was designed to verify and validate the measurement and identification of different strains and different species of the genus Nocardia by using the MBT-CA System as well as the MBT smart CA System in parallel. In addition the study was performed using the US IVD MSP 48 Target Polished Steel and the MBT Biotarget 96 US IVD in parallel. The results of this study can be evaluated as analytical study for identification performance of the MBT-CA System for the genus Nocardia and additionally as demonstration of equivalence between the “MBT-CA and MBT smart CA Systems” as well as equivalence of the “MBT Biotarget 96 US IVD and US IVD MSP 48 Target Polished Steel”.

Analytical Studies:

• Measurement of 30 strains covering 6 Nocardia species.   
• DT, eDT and Ext was used for sample preparation.   
• Each sample preparation technique was spotted 8 times on the targets.   
• Each spot was measured twice.   
• MALDI targets “US IVD MSP 48 Target Polished Steel” and “MBT Biotarget 96 US IVD” was used.   
• Measurement on MBT smart CA System and MBT-CA System.   
• Nocardia were measured after 24h, 48h and after 5 days of cultivation.

The identification performance of Nocardia using the standard identification MBT-CA workflow is shown in the diagram below. The general log(score) distribution was similar and the equivalence for all examined components (MBT-CA System, MBT smart CA System, MBT Biotarget 96 US IVD, and US IVD MSP 48 Target Polished Steel) was shown.

![](images/a1cd2790289577d9b1edc30636947b181dd8b66152197a3ad1dc1e48499dbd1f.jpg)  
Table: An example all 5760 log (scores) after 48h cultivation

From 5760 spectra derived log(scores) after 48h cultivation overlaid. Each “log(score) lane” represents 480 single log(scores).

Data showed that the MBT smart System and the MBT System, as well as the MBT Biotarget 96 US IVD and US IVD MSP 48 Target Polished Steel had equivalent performance

Comparison studies:

Method comparison with predicate device:

Not applicable. Refer to the Clinical Studies section of this document.

Matrix comparison:

Not applicable

Clinical studies:

Clinical Sensitivity:

Challenge Panel

To demonstrate intra-laboratory performance, a challenge panel of 46 organisms (12 Gram-positive aerobic bacteria, eight Gram-negative aerobic bacteria, four Gram-positive microaerophilic bacteria, five Gram-negative microaerophilic bacteria, 10 Gram-positive anaerobic bacteria, four Gram-negative anaerobic bacteria, three yeasts) was tested at three study sites. All of the 46 organisms included in the panel were selected from stored organisms tested during the clinical study. A reference laboratory prepared the panel. Organism identifications were blinded to test sites. Each site tested the challenge panel member via Direct Transfer, extended Direct Transfer and Extraction procedure in parallel. Results are summarized below:

Table: Challenge Panel Study Summary   

<table><tr><td rowspan=2 colspan=1>Test procedure</td><td rowspan=1 colspan=1>Site A*</td><td rowspan=1 colspan=1>Site B**</td><td rowspan=1 colspan=1>Site C***</td></tr><tr><td rowspan=1 colspan=3>MALDI Biotyper CA System</td></tr><tr><td rowspan=1 colspan=1>DT method</td><td rowspan=1 colspan=1>43/45 (96%)</td><td rowspan=1 colspan=1>38/44 (86%)</td><td rowspan=1 colspan=1>36/41 (88%)</td></tr><tr><td rowspan=1 colspan=1>eDT method</td><td rowspan=1 colspan=1>44/45 (98%)</td><td rowspan=1 colspan=1>36/44 (82%)</td><td rowspan=1 colspan=1>38/41(93%)</td></tr><tr><td rowspan=1 colspan=1>Ext method</td><td rowspan=1 colspan=1>43/45 (96%)</td><td rowspan=1 colspan=1>36/44 (82%)</td><td rowspan=1 colspan=1>39/41(95%)</td></tr><tr><td rowspan=1 colspan=1>MBT-CA workflow</td><td rowspan=1 colspan=1>45/45 (100%)</td><td rowspan=1 colspan=1>40/44 (91%)</td><td rowspan=1 colspan=1>40/41 (98%)</td></tr></table>

\* One (1) sample was not identified due to isolate failure to grow. $^ { * * }$ Two (2) samples were not identified due to isolate failure to grow. $^ { \ast \ast \ast }$ Five (5) samples were not identified due to isolate failure to grow.

# Method Comparison:

To demonstrate performance of the MALDI Biotyper CA (MBT-CA) System and the MALDI Biotyper smart CA (MBT-smart CA), a method comparison study was performed at four US clinical test sites and in-house laboratory. Fresh and stored organisms were tested on the MALDI Biotyper CA System in accordance to manufacturer’s instructions for use. All organisms included in the study were sub-cultured for purity. Testing on the MBT-CA System was done from a freshly isolated colony.

All organisms included in the study and tested by US study sites were also sub-cultured onto an agar slant or appropriate media for isolation and shipped to the study reference laboratory. The reference laboratory stored all organisms included in the study and when requested, sent organisms to the sequencing reference laboratory for 16S rRNA or ITS sequencing and protein gene sequencing when requested. In-house laboratory and some testing sites processed their own isolates to the sequencing reference laboratory. The results are summarized below:

Table: Overall Performance   

<table><tr><td rowspan=1 colspan=5>Overall Performance</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>1906</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>181) +2)</td><td rowspan=1 colspan=1>2055</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>531 +4</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>15)</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1932</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2091</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highconfidence ID</td><td rowspan=1 colspan=1>high &amp; lowconfidence ID</td></tr><tr><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Discordant results:

1) MBT-CA Organism $\mathrm { I D } > 2 . 0$ ; Correct Genus ID - Incorrect Species ID (17 isolates); Reference Method reported a different organism.   
2) MBT-CA Organism $\mathrm { I D } > 2 . 0$ ; Incorrect Genus ID (4 isolates); Reference Method reported a different organism. 3) MBT-CA Organism ID $( \ge 1 . 7 ; < 2 . 0 )$ ; Correct Genus ID - Incorrect Species ID (3 isolates); Reference Method reported a different organism.   
4) MBT-CA Organism ID $( \ge 1 . 7 ; < 2 . 0 )$ ; Incorrect Genus ID (1 isolate); Reference Method reported a different organism.   
5) MBT-CA Organism ID $< 1 . 7$ ; no ID (4 isolates); Reference Method reported a species from the included in the MBT-CA Reference Library.

<table><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA21.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>1906</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>91.2%</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>2037</td><td rowspan=1 colspan=2>27</td><td rowspan=1 colspan=2>14</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=2>97.4%</td><td rowspan=1 colspan=2>1.29%</td><td rowspan=1 colspan=2>0.67%</td><td rowspan=1 colspan=2>0.19%</td><td rowspan=1 colspan=2>0.19%</td><td rowspan=1 colspan=2>0.05%</td><td rowspan=1 colspan=2>0.19%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=1>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=1>Incorrect: Genus ID</td><td rowspan=1 colspan=1>no ID</td></tr><tr><td rowspan=1 colspan=1>2064</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>0.86%</td><td rowspan=1 colspan=1>0.24%</td><td rowspan=1 colspan=1>0.19%</td></tr></table>

# Table: Overall Performance – Bacteria:

<table><tr><td rowspan=1 colspan=5>PerformanceBACTERIA</td></tr><tr><td rowspan=1 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>1827</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1978</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1850</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2011</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highresolution</td><td rowspan=1 colspan=1>high &amp; lowresolution</td></tr><tr><td rowspan=1 colspan=1>98.54%</td><td rowspan=1 colspan=1>99.85%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=1 colspan=8>EY</td><td rowspan=1 colspan=3>98.54%</td><td rowspan=1 colspan=3>99.85%</td><td rowspan=1 colspan=2>n/a</td></tr><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=5>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=5>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=3>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=3>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=16>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=2>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=2>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>1829</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>90.9%</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=2>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=2>0.2%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>1960</td><td rowspan=1 colspan=2>25</td><td rowspan=1 colspan=2>14</td><td rowspan=1 colspan=3>4</td><td rowspan=1 colspan=3>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=2>97.5%</td><td rowspan=1 colspan=2>1.24%</td><td rowspan=1 colspan=2>0.70%</td><td rowspan=1 colspan=3>0.20%</td><td rowspan=1 colspan=3>0.20%</td><td rowspan=1 colspan=2>0.05%</td><td rowspan=1 colspan=2>0.15%</td></tr><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=5>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=5>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=4>1985</td><td rowspan=1 colspan=5>18</td><td rowspan=1 colspan=5>5</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=4>98.7%</td><td rowspan=1 colspan=5>0.90%</td><td rowspan=1 colspan=5>0.25%</td><td rowspan=1 colspan=2>0.15%</td></tr></table>

Table: Overall Performance – Gram Negative Bacteria:   

<table><tr><td rowspan=1 colspan=5>Performance Gram negative bacteria</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>761</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>765</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highconfidence ID</td><td rowspan=1 colspan=1>high &amp; lowconfidence ID</td></tr><tr><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>754</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>3</td><td rowspan=1 colspan=2>3</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=2>98.6%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.39%</td><td rowspan=1 colspan=2>0.39%</td><td rowspan=1 colspan=2>0.52%</td><td rowspan=1 colspan=2>0.13%</td><td rowspan=1 colspan=2>0.00%</td></tr><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=4>754</td><td rowspan=1 colspan=4>6</td><td rowspan=1 colspan=4>5</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=4>98.6%</td><td rowspan=1 colspan=4>0.78%</td><td rowspan=1 colspan=4>0.65%</td><td rowspan=1 colspan=2>0.00%</td></tr></table>

Table: Overall Performance – Gram Positive Bacteria:   

<table><tr><td rowspan=1 colspan=5>Performance Gram positive bacteria</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>1101</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1217</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1124</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1246</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highconfidence ID</td><td rowspan=1 colspan=1>high &amp; lowconfidence ID</td></tr><tr><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>1101</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>88.4%</td><td rowspan=1 colspan=1>8.4%</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=2>1206</td><td rowspan=1 colspan=2>25</td><td rowspan=1 colspan=2>11</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=2>96.8%</td><td rowspan=1 colspan=2>2.01%</td><td rowspan=1 colspan=2>0.88%</td><td rowspan=1 colspan=2>0.08%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.24%</td></tr><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=4>1231</td><td rowspan=1 colspan=4>12</td><td rowspan=1 colspan=4>0</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=4>98.8%</td><td rowspan=1 colspan=4>0.96%</td><td rowspan=1 colspan=4>0.00%</td><td rowspan=1 colspan=2>0.24%</td></tr></table>

Table: Overall Performance - Yeast:   

<table><tr><td rowspan=1 colspan=5>Performance YEAST</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies/ genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>80</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highconfidence ID</td><td rowspan=1 colspan=1>high &amp; lowconfidence ID</td></tr><tr><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>77</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=2>96.3%</td><td rowspan=1 colspan=2>2.50%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>0.00%</td><td rowspan=1 colspan=2>1.25%</td></tr><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=4>79</td><td rowspan=1 colspan=4>0</td><td rowspan=1 colspan=4>0</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=4>98.8%</td><td rowspan=1 colspan=4>0.00%</td><td rowspan=1 colspan=4>0.00%</td><td rowspan=1 colspan=2>1.25%</td></tr></table>

# Performance - single species:

Interpretation of Bruker Performance Report “Single Species”:

The performance table of the single species Arthrobacter cumminsii was selected exemplarily to illustrate the meanings of the single sectors within the table:

<table><tr><td rowspan=1 colspan=5>Arthrobacter cumminsii</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8#(1)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=5># One isolate was identified with high confidence by eDT procedure only. Matching hint isincluded in the labeling.</td></tr></table>

<table><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=3>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies/genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=3>I</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=3>III</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=3>G</td><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=7>comment field (J)</td></tr></table>

The MBT-CA results as well as the results of the reference algorithm (gold standard) were subdivided in three parts. The MBT-CA results can be interpreted as “high confidence species ID”, “low confidence species ID” and (combined) “incorrect ID” and “no ID”. The reference algorithm ID was subdivided in “high resolution species” if the results was unambiguous and “low resolution species / genus” if the result was not unambiguous. Additionally the result of the reference algorithm could be different from the MBT-CA result (“negative”).

<table><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;high confidence species ID&quot; and the result of the reference algorithmwas unambiguous (both reported A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>Ⅱ</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;high confidence species ID&quot; and the result of the reference algorithmwas not unambiguous (the MBT-CA reported A. cumminsii and the reference algorithmreported A. cumminsii and A. albus equivalently).</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;low confidence species ID&quot; and the result of the reference algorithmwas unambiguous (both reported A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;1ow confidence species ID&quot; and the result of the reference algorithmwas not unambiguous (the MBT-CA reported A. cumminsii and the reference algorithmreported A. cumminsii and A. albus equivalently).</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;high confidence species ID&quot; and the result of the reference algorithmwas different (the MBT-CA would report A. cumminsii but the reference algorithm wouldreport a different species, false positive for A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;low confidence species ID&quot; and the result of the reference algorithmwas different (the MBT-CA would report A. cumminsii but the reference algorithm wouldreport a different species, false positive for A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;incorrect ID&quot; or &quot;no ID&quot; and the result of the reference algorithmwas unambiguous (the MBT-CA would report a different species or no ID but the referencealgorithm would report A. cumminsii, false negative for A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>The MBT-CA reported &quot;incorrect ID&quot; or &quot;no ID&quot; and the result of the reference algorithmwas not unambiguous (the MBT-CA would report a different species or no ID but thereference algorithm would report A. cumminsii, false negative for A. cumminsii).</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>Room for further information.</td></tr></table>

In addition to the performance table further tables with deeper differentiation of the incorrect results were created for demonstrating the MBT-CA performance. The incorrect MBT-CA results were divided into “correct genus, incorrect species”, “incorrect genus” and “no ID”.

The transition, accumulation and counting of data are shown below:

<table><tr><td rowspan=1 colspan=5>Single Species Performance Table</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>lI</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (≥1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=5>Single Species Performance Table</td></tr><tr><td rowspan=2 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Organism ID ≥ 2.0(High Confidence)</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>A + B +E1 + E2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism ID (≥1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>C + D +F1 + F2</td><td rowspan=1 colspan=1>→</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (21.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=5 colspan=1>species</td><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA21.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>E1</td><td rowspan=1 colspan=1>E2</td><td rowspan=1 colspan=1>F1</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>H</td></tr><tr><td rowspan=6 colspan=1>Arthrobactercumminsii</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>57.1%</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>12</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=2>85.7%</td><td rowspan=1 colspan=2>14.3%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td></tr><tr><td rowspan=1 colspan=4>14</td><td rowspan=2 colspan=10>cYanmaGellowb</td></tr><tr><td rowspan=1 colspan=4>100.0%</td></tr></table>

<table><tr><td rowspan=4 colspan=1>species</td><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA≥2.0</td><td rowspan=1 colspan=2>MBT-CA≥1.7 to &lt;2.0</td><td rowspan=1 colspan=2>MBT-CA&lt;1.7</td></tr><tr><td rowspan=1 colspan=14>RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td><td rowspan=1 colspan=1>high</td><td rowspan=1 colspan=1>low</td></tr><tr><td rowspan=6 colspan=1>Arthrobactercumminsii</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>57.1%</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>12</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=2>85.7%</td><td rowspan=1 colspan=2>14.3%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.0%</td></tr><tr><td rowspan=1 colspan=4>14</td><td rowspan=2 colspan=10></td></tr><tr><td rowspan=1 colspan=4>100.0%</td></tr></table>

All incorrect or “no $\mathrm { I D } ^ { \bullet }$ results of the performance tables from the single species (sections E and $\mathrm { F }$ in the above left table) were separately considered in the fields A, B, C, D, E1, E2, F1, F2, $\mathbf { G }$ and $\mathrm { H }$ in the tables above right and below. E1 and F1 results were not observed in the entire method comparison study.

<table><tr><td colspan="1" rowspan="4">Bacteria</td><td colspan="4" rowspan="1">Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td colspan="4" rowspan="1">Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td colspan="4" rowspan="1">Incorrect: GenusID</td><td colspan="2" rowspan="1">no ID</td></tr><tr><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to &lt;2.0</td><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to &lt;2.0</td><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to&lt;2.0</td><td colspan="2" rowspan="1">MBT-CA&lt;1.7</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">RESC</td><td colspan="2" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Abiotrophia defectiva</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">17</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">17</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Acinetobacter baumanniinosocomialis group</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">51</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">51</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Acinetobactercalcoaceticus</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">5</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Acinetobacter pittii</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">32</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">32</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinotignum schaaliigroup</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">11.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">18</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">18</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces europaeus</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">97.1%</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">35</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">35</td><td colspan="10" rowspan="2">EY</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces funkei</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">3</td><td colspan="10" rowspan="2">EY</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Actinomyces graevenitzii</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces hyovaginalis</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">2</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces radingae</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">20</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">20</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces turicensis</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">40</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">40</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Actinomyces urogenitalis</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Aerococcus sanguinicola</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">31</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">31</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Aerococcus viridans</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">32</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">32</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Aggregatibacteractinomycetemcomitans</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">6</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">RES</td><td colspan="2" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">highlow</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Aggregatibacteraphrophilus</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">13</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">13</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Aggregatibacter segnis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">42.9%</td><td colspan="1" rowspan="1">57.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"> 0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">7</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Alloiococcus otitis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0% 0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Alloscardovia omnicolens</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Anaerococcus murdochii</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">7</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Arthrobacter cumminsii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">28.6%</td><td colspan="1" rowspan="1">57.1%</td><td colspan="1" rowspan="1">7.1%</td><td colspan="1" rowspan="1">7.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">12</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">85.7%</td><td colspan="2" rowspan="1">14.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">14</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Bacteroides caccae</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">27</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">27</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Bacteroides nordii</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">87.5%</td><td colspan="1" rowspan="1">12.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Bacteroides pyogenes</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">10</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">10</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Bacteroides salyersiae</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0% 0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">7</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Bacteroides stercorisgroup</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">19</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">19</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Bifidobacterium breve</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">94.1%</td><td colspan="1" rowspan="1">5.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">17</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">17</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium beijerinckii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium bifermentans</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">81.8%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">18.2%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">81.8%</td><td colspan="2" rowspan="1">18.2%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">11</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium butyricum</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">16</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">16</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridiumclostridioforme group</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">14</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">14</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium innocuum</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">34</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">34</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridiumparaputrificum</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">17</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">17</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium ramosum</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">37</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">37</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr></table>

<table><tr><td colspan="1" rowspan="4">Bacteria</td><td colspan="4" rowspan="1">Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td colspan="5" rowspan="1">Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td colspan="4" rowspan="1">Incorrect: GenusID</td><td colspan="2" rowspan="1">no ID</td></tr><tr><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to &lt;2.0</td><td colspan="3" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to &lt;2.0</td><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to&lt;2.0</td><td colspan="2" rowspan="1">MBT-CA&lt;1.7</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">highlow</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Clostridium septicum</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">32</td><td colspan="2" rowspan="1">0</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">32</td><td colspan="11" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium sordellii</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">21</td><td colspan="2" rowspan="1">0</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">21</td><td colspan="11" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Clostridium sporogenes/C. botulinum (group I)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">7.7%</td></tr><tr><td colspan="2" rowspan="1">12</td><td colspan="2" rowspan="1">0</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1</td></tr><tr><td colspan="2" rowspan="1">92.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">7.7%</td></tr><tr><td colspan="4" rowspan="1">12</td><td colspan="11" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">92.3%</td></tr><tr><td colspan="1" rowspan="6">Clostridium tertium</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">39</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">97.5%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">2.5%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">39</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="6">Corynebacteriumaccolens</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">36</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">36</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="5" rowspan="1">RESOLUTION REFERENCE ALGORITHM</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="3" rowspan="1">lowhigh</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Corynebacteriumafermentans group</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">83.3%</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">6</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Corynebacterium coyleae</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">77.8%</td><td colspan="1" rowspan="1">22.2%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">18</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">18</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Corynebacterium freneyi</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Corynebacteriumglutamicum</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Corynebacteriummucifaciens /ureicelerivorans group</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">83.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">16.7%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">83.3%</td><td colspan="3" rowspan="1">16.7%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">6</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REF</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Corynebacteriumpseudotuberculosis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Corynebacterium resistens</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">2</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">2</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Dermabacter hominis</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">3.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">29</td><td colspan="3" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">96.7%</td><td colspan="3" rowspan="1">3.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">30</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Enterococcus durans</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">6</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Enterococcus mundtii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Enterococcus raffinosus</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">34</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">34</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Escherichia hermannii</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">86.2%</td><td colspan="1" rowspan="1">6.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">3.4%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">3.4%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">27</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">93.1%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">3.4%</td><td colspan="2" rowspan="1">3.4%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">27</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">93.1%</td></tr><tr><td colspan="1" rowspan="6">Escherichia vulneris</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">83.3%</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">12</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">12</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Ewingella americana</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">57.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">42.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">57.1%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">42.9%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">57.1%</td></tr><tr><td colspan="1" rowspan="6">Facklamia hominis</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">17</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">17</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Fluoribacter bozemanae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">2</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">2</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Gemella morbillorum</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">96.2%</td><td colspan="1" rowspan="1">3.8%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">26</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">26</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Helcococcus kunzii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Kingella denitrificans</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">94.5%</td><td colspan="1" rowspan="1">4.4%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">1.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">90</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">98.9%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">1.1%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">90</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">98.9%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Klebsiella variicola</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">9</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Lactobacillus gasseri</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">12.9%</td><td colspan="1" rowspan="1">83.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">30</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">96.8%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">3.2%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">30</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">96.8%</td></tr><tr><td colspan="1" rowspan="6">Lactobacillus jensenii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">24</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">24</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">39</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">39</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">94.7%</td><td colspan="1" rowspan="1">5.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">19</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">19</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REF</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Legionella longbeachae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Legionella pneumophila</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">33</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">33</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Leuconostoc citreum</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Leuconostocpseudomesenteroides</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Listeria monocytogenes</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">25</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">25</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">high low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Mannheimia haemolyticagroup</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">13</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">13</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Micrococcus lylae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Mobiluncus curtisii</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria bacilliformis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria cinerea</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">2</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">2</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Neisseria elongata</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria flavescens /subflavagroup</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">56</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">56</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">45</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">45</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria lactamica</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">91.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">8.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">11</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">91.7%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">8.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">11</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">91.7%</td></tr><tr><td colspan="1" rowspan="6">Neisseria meningitidis</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">92.1%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">5.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">35</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">92.1%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">5.3%</td><td colspan="2" rowspan="1">2.6%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">35</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">92.1%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="5" rowspan="1">RESOLUTION REFERENCE ALGORITHM</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="3" rowspan="1">lowhigh</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Neisseria sicca group</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">30</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">30</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Neisseria weaveri</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">9</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Nocardia brasiliensis</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">37.5%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">12.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">14</td><td colspan="3" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">87.5%</td><td colspan="3" rowspan="1">12.5%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">16</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Nocardia cyriacigeorgica</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">87.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">13.0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">20</td><td colspan="3" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">87.0%</td><td colspan="3" rowspan="1">13.0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">23</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Nocardia farcinica group</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">89.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">10.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">26</td><td colspan="3" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">89.7%</td><td colspan="3" rowspan="1">10.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">29</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REF</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Nocardia nova</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">87.5%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">6.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">6.3%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">28</td><td colspan="3" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">2</td></tr><tr><td colspan="2" rowspan="1">87.5%</td><td colspan="3" rowspan="1">6.3%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">6.3%</td></tr><tr><td colspan="5" rowspan="1">30</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">93.8%</td></tr><tr><td colspan="1" rowspan="6">Nocardia otitidiscaviarum</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Ochrobactrum anthropi</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">40%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">10%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">90%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">10%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">9</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">90%</td></tr><tr><td colspan="1" rowspan="6">Parabacteroidesgoldsteinii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">2</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">2</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Parabacteroides johnsoni/ merdae group</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">13</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">13</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Parvimonas micra</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">95.1%</td><td colspan="1" rowspan="1">4.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">41</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">41</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Pediococcus acidilactici</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">10</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">10</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Pluralibacter gergoviae</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">10</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">10</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Porphyromonas somerae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">1</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">1</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Ralstonia pickettii</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">9</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REF</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Serratia fonticola</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">7</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Serratia odorifera</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Sphingobacteriummultivorum</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Sphingobacteriumspiritivorum</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">7</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Sphingomonaspaucimobilis group</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">16</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">16</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus delphini</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Staphylococcusintermedius</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus lentus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">3</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus sciuri</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">73.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">26.7%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">11</td><td colspan="3" rowspan="1">4</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">73.3%</td><td colspan="3" rowspan="1">26.7%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">15</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus xylosus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">40%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">3</td><td colspan="3" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">60%</td><td colspan="3" rowspan="1">40%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="5" rowspan="1"></td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="3" rowspan="1">lowhigh</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1"> low</td></tr><tr><td colspan="1" rowspan="6">Streptococcus canis</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">18</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">18</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Streptococcus equi</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">6</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Streptococcusparasanguinis</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">73.3%</td><td colspan="1" rowspan="1">6.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">20%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">36</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">9</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">80%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">20%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">36</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="6">Streptococcus sanguinis</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">32</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">32</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Streptococcus sobrinus</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REFE</td><td colspan="2" rowspan="1">RENCE ALGO</td><td colspan="2" rowspan="1">RITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Streptococcus salivariusvestibularis group</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">92.2%</td><td colspan="1" rowspan="1">6.3%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">63</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">98.4%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">1.6%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">63</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">98.4%</td></tr><tr><td colspan="1" rowspan="6">Streptococcusthermophilus</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">4</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">4</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Trueperella bernardiae</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">27</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">27</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Turicella otitidis</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">11.1%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">24</td><td colspan="3" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">88.9%</td><td colspan="3" rowspan="1">11.1%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">27</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Vagococcus fluvialis</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">LUTION REF</td><td colspan="2" rowspan="1">RENCE ALG</td><td colspan="2" rowspan="1">ORITHM</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="2" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">hig</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">higl</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Veillonella parvula group</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">33</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">33</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Weeksella virosa</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">10</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">10</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Yersinia frederiksenii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Yersinia intermedia</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">8</td><td colspan="10" rowspan="2">PY</td></tr><tr><td colspan="5" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Yersinia kristensenii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">57.1%</td><td colspan="1" rowspan="1">42.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">7</td><td colspan="3" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="3" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="5" rowspan="1">7</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="5" rowspan="1">100%</td></tr></table>

<table><tr><td colspan="1" rowspan="4">Yeast</td><td colspan="4" rowspan="1">Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td colspan="4" rowspan="1">Correct: Genus IDIncorrect: Species IDor Group / ComplexID</td><td colspan="4" rowspan="1">Incorrect: Genus ID</td><td colspan="2" rowspan="1">no ID</td></tr><tr><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to&lt;2.0</td><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to&lt;2.0</td><td colspan="2" rowspan="1">MBT-CA≥2.0</td><td colspan="2" rowspan="1">MBT-CA≥1.7 to&lt;2.0</td><td colspan="2" rowspan="1">MBT-CA&lt;1.7</td></tr><tr><td colspan="14" rowspan="1">RESOLUTION REFERENCE ALGORITHM</td></tr><tr><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td><td colspan="1" rowspan="1">high</td><td colspan="1" rowspan="1">low</td></tr><tr><td colspan="1" rowspan="6">Candida intermedia</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">10%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">90%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">10%</td></tr><tr><td colspan="4" rowspan="1">9</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">90%</td></tr><tr><td colspan="1" rowspan="6">Candida zeylanoides</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">6</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Cyberlindnera jadinii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">5</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">5</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Malassezia furfur</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">75.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">25.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">6</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">75.0%</td><td colspan="2" rowspan="1">25.0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">8</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Malasseziapachydermatis</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">9</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">9</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Rhodotorulamucilaginosa</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">20</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">20</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Trichosporon inkin</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">8</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">8</td><td colspan="10" rowspan="2">Y</td></tr><tr><td colspan="4" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Trichosporon mucoidesgroup</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">14</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="2" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">100%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td><td colspan="2" rowspan="1">0%</td></tr><tr><td colspan="4" rowspan="1">14</td><td colspan="10" rowspan="2"></td></tr><tr><td colspan="4" rowspan="1">100%</td></tr></table>

# Table: Overall Incidence of Incorrect Identifications

<table><tr><td rowspan=1 colspan=4>Correct: Genus IDCorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Correct: Genus IDIncorrect: Species ID orGroup / Complex ID</td><td rowspan=1 colspan=4>Incorrect: Genus ID</td><td rowspan=1 colspan=2>no ID</td></tr><tr><td rowspan=1 colspan=4>REFERENCE ALGORITHM</td><td rowspan=1 colspan=10>REFERENCE ALGORITHM</td></tr><tr><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>T0 Wres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td><td rowspan=1 colspan=1>highres.</td><td rowspan=1 colspan=1>lowres.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>E1</td><td rowspan=1 colspan=1>E2</td><td rowspan=1 colspan=1>F1</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>G1</td><td rowspan=1 colspan=1>G2</td></tr><tr><td rowspan=1 colspan=1>1904</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>91.1%</td><td rowspan=1 colspan=1>6.2%</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=2>2034</td><td rowspan=1 colspan=2>28</td><td rowspan=1 colspan=2>17</td><td rowspan=1 colspan=2>3</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=2>97.3%</td><td rowspan=1 colspan=2>1.3%</td><td rowspan=1 colspan=2>0.8%</td><td rowspan=1 colspan=2>0.1%</td><td rowspan=1 colspan=2>0.2%</td><td rowspan=1 colspan=2>0.0%</td><td rowspan=1 colspan=2>0.2%</td></tr></table>

<table><tr><td>MBT-CA</td><td>Reference Method</td><td>Code</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Correct genus ID - Incorrect species ID</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=17 colspan=1>1)</td><td rowspan=1 colspan=1>Bacteroides stercoris</td><td rowspan=1 colspan=1>Bacteroides eggerthii</td><td rowspan=8 colspan=1>A(8x)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Klebsiella variicola</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus gasseri</td><td rowspan=1 colspan=1>Lactobacillus johnsonii</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Neisseria polysaccharea</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Neisseria polysaccharea</td></tr><tr><td rowspan=1 colspan=1>Sphingomonas paucimobilis</td><td rowspan=1 colspan=1>Sphingomonas zeae</td></tr><tr><td rowspan=1 colspan=1>Sphingomonas paucimobilis</td><td rowspan=1 colspan=1>Sphingomonas zeae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius ssp salivarius</td><td rowspan=1 colspan=1>Streptococcus thermophilus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td><td rowspan=9 colspan=1>B(9x)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>Streptococcus sp. (Viridans)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>Neisseria cinerea</td><td rowspan=2 colspan=1>C(2x)</td></tr><tr><td rowspan=1 colspan=1>Ochrobactrum anthropi</td><td rowspan=1 colspan=1>Ochrobactrum intermedium</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Neisseria sp.</td><td rowspan=1 colspan=1>D</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Incorrect genus ID</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>2)</td><td rowspan=1 colspan=1>Ewingella americana</td><td rowspan=1 colspan=1>Rahnella sp.</td><td rowspan=4 colspan=1>E2(4x)</td></tr><tr><td rowspan=1 colspan=1>Ewingella americana</td><td rowspan=1 colspan=1>Rahnella sp.</td></tr><tr><td rowspan=1 colspan=1>Ewingella americana</td><td rowspan=1 colspan=1>Rahnella sp.</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>Enterobacter sp.</td></tr><tr><td rowspan=1 colspan=1>4)</td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>Enterobacter sp.</td><td rowspan=1 colspan=1>F2</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>no ID</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>5)</td><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>Nocardia nova</td><td rowspan=3 colspan=1>G1(3x)</td></tr><tr><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>Nocardia nova</td></tr><tr><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>Candida intermedia</td></tr><tr><td rowspan=1 colspan=1>no ID</td><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>G2</td></tr></table>

If an indication for the possibility of cross-matching patterns was found, the organism was included in the matching hint table found in the package labeling and as found below.

# Table: Matching Hint Table

<table><tr><td colspan="1" rowspan="1">W Htt</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">a</td><td colspan="1" rowspan="1">Achromobacter xylosoxidans</td><td colspan="1" rowspan="1">A. xylosoxidans</td><td colspan="1" rowspan="1">A.xylosoxidans;A.denitrificans;A.insolitus;A.marplatensis;A.ruhlandii;A.spanius</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.All species associated withthe displayed identificationhave been reported asisolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">by</td><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">A. calcoaceticus</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">The displayed species shouldbe considered as member ofthe Acinetobacter baumanniicomplex.For organisms identified bythe MBT-CA System asAcinetobacter calcoaceticus;Acinetobacter pittii orAcinetobacterbaumannii/nosocomialisgroup the full Extractionprocedure (Ext) is mandatoryfor secure speciesdifferentiation.</td></tr><tr><td colspan="1" rowspan="1">bz</td><td colspan="1" rowspan="1">Acinetobacter pittii</td><td colspan="1" rowspan="1">A. pittii</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">The displayed species shouldbe considered as member ofthe Acinetobacter baumanniicomplex.For organisms identified bythe MBT-CA System asAcinetobacter calcoaceticus;Acinetobacter pittii orAcinetobacterbaumannii/nosocomialisgroup the full Extractionprocedure (Ext) is mandatoryfor secure speciesdifferentiation.</td></tr><tr><td colspan="1" rowspan="1">Wt H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">ca</td><td colspan="1" rowspan="1">Acinetobacter baumanniinosocomialis group</td><td colspan="1" rowspan="1">A. baumannii;A. nosocomialis</td><td colspan="1" rowspan="1">A. baumannii;A. nosocomialis</td><td colspan="1" rowspan="1">The displayed species shouldbe considered as member ofthe Acinetobacter baumanniicomplex.For organisms identified bythe MBT-CA System asAcinetobacter calcoaceticus;Acinetobacter pittii orAcinetobacter baumannii/nosocomialis group the fullExtraction procedure (Ext) ismandatory for secure speciesdifferentiation.</td></tr><tr><td colspan="1" rowspan="1">Z</td><td colspan="1" rowspan="1">Actinomyces oris</td><td colspan="1" rowspan="1">A. oris</td><td colspan="1" rowspan="1">A. oris;A. naeslundii;A.viscosus</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cb</td><td colspan="1" rowspan="1">Actinotignum schaalii group</td><td colspan="1" rowspan="1">A. schaalii;A. sanguinis</td><td colspan="1" rowspan="1">A. schaalii;A. sanguinis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">aa</td><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">A. viridans</td><td colspan="1" rowspan="1">A. viridans;A. urinaeequi</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">ab</td><td colspan="1" rowspan="1">Aeromonas salmonicida</td><td colspan="1" rowspan="1">A. salmonicida</td><td colspan="1" rowspan="1">A. salmonicida;A. bestiarum;A. molluscorum;A. rivuli;A. encheleia</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">W H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">c</td><td colspan="1" rowspan="1">Aeromonas sp[7]</td><td colspan="1" rowspan="1">A.allosaccharophilaA.caviaeA. culicicolaA.hydrophilaA.ichthiosmiaA.sobriaA. veronii</td><td colspan="1" rowspan="1">A.allosaccharophila;A.aquariorum;A.caviae;A.culicicola;A.enteropelogenes;A.fluvialis;A.hydrophila;A.ichthiosmia;A.jandaei;A.media;A.punctata;A.rivuli;A.sanarellii;A.sobria;A.taiwanensis;A.veronii</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether. A.hydrophila;A. caviae and A.sobria(A.veronii bv sobria) are themost frequently reportedspecies associated withhuman infection.A.ichthiosmia is considered asynonym of A.veronii andA.punctata is considered asynonym of A.caviae.</td></tr><tr><td colspan="1" rowspan="1">cc</td><td colspan="1" rowspan="1">Arthrobacter cumminsii</td><td colspan="1" rowspan="1">A. cumminsii</td><td colspan="1" rowspan="1">A. cumminsii;A. albus</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bacteroides stercoris group</td><td colspan="1" rowspan="1">B. stercoris</td><td colspan="1" rowspan="1">B. stercoris;B. eggerthii</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ac</td><td colspan="1" rowspan="1">Bacteroides ovatus group</td><td colspan="1" rowspan="1">B. ovatus;B. xylanisolvens</td><td colspan="1" rowspan="1">B. ovatus;B. xylanisolvens</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ad</td><td colspan="1" rowspan="1">Bacteroidesthetaiotaomicron group</td><td colspan="1" rowspan="1">B. thetaiotaomicron;B. faecis</td><td colspan="1" rowspan="1">B. thetaiotaomicron;B. faecis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ae</td><td colspan="1" rowspan="1">Bacteroides vulgatus group</td><td colspan="1" rowspan="1">B. vulgatus;B. dorei</td><td colspan="1" rowspan="1">B. vulgatus;B. dorei</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">af</td><td colspan="1" rowspan="1">Bordetella group[3]</td><td colspan="1" rowspan="1">B. bronchiseptica;B. parapertussis;B. pertussis</td><td colspan="1" rowspan="1">B. bronchiseptica;B. parapertussis;B. pertussis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether.</td></tr><tr><td colspan="1" rowspan="1">ag</td><td colspan="1" rowspan="1">Brevundimonas diminutagroup</td><td colspan="1" rowspan="1">B. diminuta;B. naejangsanensis</td><td colspan="1" rowspan="1">B. diminuta;B. naejangsanensis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">W </td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">d</td><td colspan="1" rowspan="1">Burkholderia gladioli</td><td colspan="1" rowspan="1">B. gladioli</td><td colspan="1" rowspan="1">B.gladioli;B.glumae;B.caryopylii</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.B. glumae and B. caryopyliihave not been reported asisolated with humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">cd</td><td colspan="1" rowspan="1">Burkholderia multivorans</td><td colspan="1" rowspan="1">B. multivorans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The displayed species shouldbe considered as member ofthe Burkholderia cepaciacomplex.</td></tr><tr><td colspan="1" rowspan="1">e</td><td colspan="1" rowspan="1">Burkholderia cepaciacomplex [13]</td><td colspan="1" rowspan="1">B. ambifaria;B. anthina;B. cenocepacia;B. cepacia;B. diffusa;B. dolosa;B. lata;B. latens;B. metallica;B. pyrrocinia;B. seminalis;B. stabilis;B. vietnamiensis</td><td colspan="1" rowspan="1">B.ambifaria;B.anthina;B.cenocepacia;B.cepacia;B.diffusa;B.dolosa;B.lata;B.latens;B.metallica;B.pyrrocinia;B.seminalis;B.stabilis;B.vietnamiensis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">ah</td><td colspan="1" rowspan="1">Chryseobacterium gleum</td><td colspan="1" rowspan="1">C. gleum</td><td colspan="1" rowspan="1">C. gleum;C. bernardetii</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">f</td><td colspan="1" rowspan="1">Citrobacter amalonaticuscomplex</td><td colspan="1" rowspan="1">C. amalonaticus;C. farmeri</td><td colspan="1" rowspan="1">C. amalonaticus;C. farmeri</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult; whereas they aregrouped together. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">g</td><td colspan="1" rowspan="1">Citrobacter freundii complex</td><td colspan="1" rowspan="1">C. braakii;C. freundii;C. gillenii;C. murliniae;C. rodentium;C. sedlakii;C. werkmannii;C. youngae</td><td colspan="1" rowspan="1">C.braakii;C.freundii;C.gillenii;C.murliniae;C.rodentium;C.sedlakii;C.werkmannii;C.youngae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether. C. rodentium hasnot been reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">ce</td><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">C. beijerinckii</td><td colspan="1" rowspan="1">C. beijerinckii;C. diolis;C. roseum;C.saccharoperbutylacetonicum</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cf</td><td colspan="1" rowspan="1">Clostridium clostridioformegroup</td><td colspan="1" rowspan="1">C. clostridioforme;C. bolteae</td><td colspan="1" rowspan="1">C. clostridioforme;C. bolteae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">cg</td><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">C. sporogenes</td><td colspan="1" rowspan="1">C. sporogenes;C. botulinum (group I)</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.C. botulinum is not includedin the MBT-CA database. C.botulinum is a select agentand should be ruled out;handle isolate with extremecaution and handle inaccordance with local;state;and federal accreditingorganizations' requirementsas applicable.</td></tr><tr><td colspan="1" rowspan="1">Nut H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">ch</td><td colspan="1" rowspan="1">Corynebacteriumafermentans group</td><td colspan="1" rowspan="1">C. afermentans;C. pilbarense</td><td colspan="1" rowspan="1">C. afermentans;C. pilbarense</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ak</td><td colspan="1" rowspan="1">Corynebacteriumaurimucosum group</td><td colspan="1" rowspan="1">C. aurimucosum</td><td colspan="1" rowspan="1">C. aurimucosum;C. singulare</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ci</td><td colspan="1" rowspan="1">Corynebacteriummucifaciens ureicelerivoransgroup</td><td colspan="1" rowspan="1">C. mucifaciens;C. ureicelerivorans</td><td colspan="1" rowspan="1">C. mucifaciens;C. ureicelerivorans</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">al</td><td colspan="1" rowspan="1">Corynebacterium striatumgroup</td><td colspan="1" rowspan="1">C. striatum;C. simulans</td><td colspan="1" rowspan="1">C. striatum;C. simulans</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">am</td><td colspan="1" rowspan="1">Cronobacter sakazakii group</td><td colspan="1" rowspan="1">C. sakazakii;C. dublinensis;C. muytjensii;C. turicensis</td><td colspan="1" rowspan="1">C. sakazakii;C. dublinensis;C. muytjensii;C. turicensis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">an</td><td colspan="1" rowspan="1">Cupriavidus pauculus group</td><td colspan="1" rowspan="1">C. pauculus;C. metallidurans</td><td colspan="1" rowspan="1">C. pauculus;C. metallidurans</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is not</td></tr><tr><td colspan="1" rowspan="1">Nut Wi</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">possible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ao</td><td colspan="1" rowspan="1">Delftia acidovorans group</td><td colspan="1" rowspan="1">D. acidovorans;D. lacustris;D. litopenaei;D. tsuruhatensis</td><td colspan="1" rowspan="1">D. acidovorans;D. lacustris;D. litopenaei;D. tsuruhatensis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">ap</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">E. tarda</td><td colspan="1" rowspan="1">E. tarda;E. hoshinae;E. ictaluri</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">aq</td><td colspan="1" rowspan="1">Elizabethkingiameningoseptica group</td><td colspan="1" rowspan="1">E. meningoseptica;E. anophelis;E. miricola</td><td colspan="1" rowspan="1">E. meningoseptica;E. anophelis;E. miricola</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">h</td><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">E. asburiae;E. cancerogenus;E. cloacae;E. hormaechei;E. kobei;E. ludwigii</td><td colspan="1" rowspan="1">E. asburiae;E. cancerogenus;E. cloacae;E. cowanii;E. hormaechei;E. kobei;E. ludwigii;E. mori;E. nimipressuralis;E. soli</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether.E. mori and E. soli have notbeen reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">i</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. albertii;E. coli;E. fergusonii;Shigella spp.</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.All species associated withthe displayed identificationhave been reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">j</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">H. influenza;</td><td colspan="1" rowspan="1">Based on 16S rRNA gene</td></tr><tr><td colspan="1" rowspan="1">W H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">H. aegyptius</td><td colspan="1" rowspan="1">sequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">as</td><td colspan="1" rowspan="1">Haemophilusparahaemolyticus group</td><td colspan="1" rowspan="1">H. parahaemolyticus;H.paraphrohaemolyticuS</td><td colspan="1" rowspan="1">H. parahaemolyticus;H.paraphrohaemolyticus</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">k</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">H. alvei</td><td colspan="1" rowspan="1">H. alvei;H. paralvei;Obesumbacteriumproteus</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Obesumbacterium proteus ismost commonly associatedwith brewery spoilage andhas not reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae;K. granulomatis;K. singaporensis</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.All species associated withthe displayed identificationhave been isolated fromhuman specimens; K.pneumoniae is the mostcommon species reported asisolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">cj</td><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">K. variicola;K. granulomatis;K. singaporensis</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">W Ht</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">m</td><td colspan="1" rowspan="1">Klebsiella oxytocaRaoultellaornithinolytica</td><td colspan="1" rowspan="1">K. oxytocaR.ornithinolytica</td><td colspan="1" rowspan="1">K. oxytoca;R. ornithinolytica;R. planticola</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult; whereas they aregrouped together. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">ck</td><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">L. gasseri</td><td colspan="1" rowspan="1">L. gasseri;L. taiwanensis</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cl</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">L. jensenii</td><td colspan="1" rowspan="1">L. jensenii;L. fornicalis;L. psittaci</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cm</td><td colspan="1" rowspan="1">Leuconostoc citreum</td><td colspan="1" rowspan="1">L. citreum</td><td colspan="1" rowspan="1">L. citreum;L. holzapfelii</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cn</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">L. monocytogenes</td><td colspan="1" rowspan="1">L. monocytogenes;L. inocua;L. ivanovii;L. marthii;L. seeligeri;L. welshimeri</td><td colspan="1" rowspan="1">Secure species differentiationfor Listeria species isdifficult since several specieswithin the genus Listeria arevery closely related. Alsoalternative methods (e.g.DNA sequencing) show lowdiscrimatory power. ForListeria identified bytheMBT-CA it isrecommended to proceedwith full extractionprocedure (Ext) for finalidentification.</td></tr><tr><td colspan="1" rowspan="1">co</td><td colspan="1" rowspan="1">Mannheimia haemolyticagroup</td><td colspan="1" rowspan="1">M. haemolytica</td><td colspan="1" rowspan="1">M. haemolytica;M. glucosida</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Moraxella sg Moraxellaosloensis</td><td colspan="1" rowspan="1">M. osloensis</td><td colspan="1" rowspan="1">Moraxella osloensis;Enhydrobacter</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure species</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">aerosaccus</td><td colspan="1" rowspan="1">differentiation between thedisplayed species is difficult.The rare speciesEnhydrobacter aerosaccus isclosely related to Moraxellaosloensis; and has notreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">M. morganii;M. psychrotolerans</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Both species associated withthe displayed identificationhave been reported as isolatedfrom human specimens.</td></tr><tr><td colspan="1" rowspan="1">au</td><td colspan="1" rowspan="1">Myroides odoratimimus</td><td colspan="1" rowspan="1">M. odoratimimus</td><td colspan="1" rowspan="1">M. odoratimimus;M. profundi</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cp</td><td colspan="1" rowspan="1">Neisseria flavescenssubflava group</td><td colspan="1" rowspan="1">N. flavescens;N. perflava;N. subflava</td><td colspan="1" rowspan="1">N. flavescens;N. perflava;N. subflava</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">cq</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">N. gonorrhoeae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">If the reference methodreported N. gonorrhoeae NOfalse identification of theMBT-CA was observedduring clinical tests.</td></tr><tr><td colspan="1" rowspan="1">cr</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In rare cases apathogenicNeisseria species could beidentified as N.meningitidiswith the MBT-CA. If thereference method reportedN.meningitidis NO falseidentification of the MBT-CA was observed duringclinical tests.</td></tr><tr><td colspan="1" rowspan="1">cs</td><td colspan="1" rowspan="1">Neisseria sicca group</td><td colspan="1" rowspan="1">N. macacae;N. mucosa;N. sicca</td><td colspan="1" rowspan="1">N. macacae;N. mucosa;N. sicca;Morococcus</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup is not possible by theMBT-CA System or</td></tr><tr><td colspan="1" rowspan="1">Wu H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cerebrosus</td><td colspan="1" rowspan="1">reference method (16S rRNAgene sequencing). Thereforethey are grouped together.</td></tr><tr><td colspan="1" rowspan="1">ct</td><td colspan="1" rowspan="1">Nocardia brasiliensis</td><td colspan="1" rowspan="1">N. brasiliensis</td><td colspan="1" rowspan="1">N. brasiliensis;N. iowensis;N. vulneris</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">cu</td><td colspan="1" rowspan="1">Nocardia farcinica group</td><td colspan="1" rowspan="1">N. farcinica;N. kroppenstedtii</td><td colspan="1" rowspan="1">N. farcinica;N. kroppenstedtii</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup is not possible by theMBT-CA System orreference method (16S rRNAgene sequencing). Thereforethey are grouped together.</td></tr><tr><td colspan="1" rowspan="1">cV</td><td colspan="1" rowspan="1">Ochrobactrum anthropi</td><td colspan="1" rowspan="1">O. anthropi</td><td colspan="1" rowspan="1">O. anthropi;O. lupini</td><td colspan="1" rowspan="1">Based on protein genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">P. agglomerans</td><td colspan="1" rowspan="1">P. agglomerans;P. anthophila;P. brenneri;P. conspicua;P. eucalypti;P. vagans</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.All species associated withthe displayed identificationhave been reported asisolated from humanspecimens. Pantoeaagglomerans; is the mostcommonly reported Pantoeaspecies isolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">CW</td><td colspan="1" rowspan="1">Parabacteroides johnsoniimerdae group</td><td colspan="1" rowspan="1">P. johnsonii;P. merdae</td><td colspan="1" rowspan="1">P. johnsonii;P. merdae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">Wt Wit</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">av</td><td colspan="1" rowspan="1">Peptoniphilus harei group</td><td colspan="1" rowspan="1">P. harei</td><td colspan="1" rowspan="1">P. harei;P. indolicus</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">q</td><td colspan="1" rowspan="1">Proteus vulgaris group</td><td colspan="1" rowspan="1">P. hauseri;P. penneri;P. vulgaris</td><td colspan="1" rowspan="1">P. hauseri;P. penneri;P. vulgaris</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">r</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">P. rettgeri</td><td colspan="1" rowspan="1">P. rettgeri;P. alcalifaciens;P. burhodogranariea;P. heimbachae;P. rustigianii;P. vermicola</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.P. burhodogranariea and P.vermicola have not beenreported as isolated fromhuman specimens</td></tr><tr><td colspan="1" rowspan="1">We Hh</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">Pseudomonas fluorescensgroup</td><td colspan="1" rowspan="1">P. azotoformans;P. brenneri;P. cedrina;P. congelans;P. corrugata;P. extremorientalis;P. fluorescens;P. gessardii;P. libanensis;P. mandelii;P. marginalis;P. migulae;P. mucidolens;P. orientalis;P. poae;P. rhodesiae;P. synxantha;P. tolaasii;P. trivialis;P. veronii</td><td colspan="1" rowspan="1">P. azotoformans;P. brenneri;P. cedrina;P. congelans;P. corrugata;P. extremorientalis;P. fluorescens;P. gessardii;P. libanensis;P. mandelii;P. marginalis;P. migulae;P. mucidolens;P. orientalis;P. poae;P. rhodesiae;P. synxantha;P. tolaasii;P. trivialis;P. veronii</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes basedon 16S rRNA genesequencing is difficult;whereas they are groupedtogether. Members of the P.fluorescens group areenvironmental organisms. P.fluorescens is the mostcommonly isolated speciesreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">aw</td><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td><td colspan="1" rowspan="1">P. oryzihabitans</td><td colspan="1" rowspan="1">P. oryzihabitans;P. oleovorans;P. psychrotolerans</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">t</td><td colspan="1" rowspan="1">Pseudomonas putida group</td><td colspan="1" rowspan="1">P. fulva;P. monteilii;P. mosselii;P. plecoglossicida;P. putida</td><td colspan="1" rowspan="1">P. fulva;P. monteilii;P. mosselii;P. plecoglossicida;P. putida</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroups or complexes based on16S rRNA gene sequencing isdifficult;whereas they are groupedtogether. Members of the P.putida group areenvironmental organisms. P.putida is the most commonlyisolated species reported asisolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">ax</td><td colspan="1" rowspan="1">Rhizobium radiobacter</td><td colspan="1" rowspan="1">R. radiobacter</td><td colspan="1" rowspan="1">R. radiobacter;R. massilense;R. leguminosarum</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">u</td><td colspan="1" rowspan="1">Salmonella sp</td><td colspan="1" rowspan="1">Salmonella sp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Identification ispossible on genuslevel only.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">S. fonticola</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Wu Ht</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">S. liquefaciens</td><td colspan="1" rowspan="1">S. liquefaciens;S. proteamaculans;S. grimesii;S. plymuthica;S. ficaria</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.All species associated withthe displayed identificationhave been reported asisolated from humanspecimens.</td></tr><tr><td colspan="1" rowspan="1">W</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">S. marcescens;S. nematodiphila;S. ureilytica</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.S. nematodiphila and S.ureilytica have not beenreported as isolated fromhuman specimens.</td></tr><tr><td colspan="1" rowspan="1">ay</td><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">S. plymuthica</td><td colspan="1" rowspan="1">S. plymuthica;S. grimesii;S. proteamaculans;S. quinivorans</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Sphingomonas paucimobilisgroup</td><td colspan="1" rowspan="1">S. paucimobilis</td><td colspan="1" rowspan="1">S. paucimobilis;S. zeae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by the MBT-CA is notpossible although thereference method (16S rRNAgene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">az</td><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">S. carnosus</td><td colspan="1" rowspan="1">S. carnosus;S. condimenti;S. piscifermentans</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">bc</td><td colspan="1" rowspan="1">Staphylococcus vitulinus</td><td colspan="1" rowspan="1">S. vitulinus</td><td colspan="1" rowspan="1">S. vitulinus;S. fleurettii</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">S. maltophilia;Pseudomonas beteli;Pseudomonashibiscola;Pseudomonasgeniculata</td><td colspan="1" rowspan="1">S. maltophilia;Pseudomonas beteli;Pseudomonashibiscola;Pseudomonasgeniculata</td><td colspan="1" rowspan="1">S. maltophilia;P. beteli;P. hibiscola;P. geniculata aresynonymously usedtaxonomical names.</td></tr><tr><td colspan="1" rowspan="1">Wu H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">bd</td><td colspan="1" rowspan="1">Streptococcus lutetiensis</td><td colspan="1" rowspan="1">S. lutetiensis</td><td colspan="1" rowspan="1">S. lutetiensis;S. infantarius ssp.coli;S. infantarius sspinfantarius</td><td colspan="1" rowspan="1">The species designationStreptococcus lutetiensis andStreptococcus pasteurianushave been proposed for S.bovis biotype II.1 and biotypeII.2 strains;respectively (Poyart et al;2002). However;S. lutetiensis exhibits bothphenotypic and geneticsimilarity to S. infantariussubsp. coli (Schlegel et al.:2000); and preliminaryresults show a closerelationship between S.pasteurianus and S.gallolyticus.</td></tr><tr><td colspan="1" rowspan="1">CX</td><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">S. parasanguinis</td><td colspan="1" rowspan="1">S. parasanguinis;S. australis;S. infantis;S. lactarius;S. rubneri</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Displayed species aremembers of the"Streptococcus viridansgroup".</td></tr><tr><td colspan="1" rowspan="1">be</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">For organisms identified bythe MBT-CA asStreptococcus pneumoniae orStreptococcus mitis/oralisgroup; it is recommended toproceed with full extraction(EXT) for finalidentification.</td></tr><tr><td colspan="1" rowspan="1">cy</td><td colspan="1" rowspan="1">Streptococcus salivariusvestibularis group</td><td colspan="1" rowspan="1">S. salivarius;S. vestibularis</td><td colspan="1" rowspan="1">S. salivarius;S. vestibularis</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by MBT-CA is notpossible although thereference method (proteingene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.S. salivarius is also known asS. salivarius ssp salivarius.</td></tr><tr><td colspan="1" rowspan="1">CZ</td><td colspan="1" rowspan="1">Streptococcus thermophilus</td><td colspan="1" rowspan="1">S. thermophilus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">S. thermophilus is alsoknown as S. salivarius sspthermophilus.</td></tr><tr><td colspan="1" rowspan="1">Wrt Ht</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">bg</td><td colspan="1" rowspan="1">Streptococcus mitis oralisgroup</td><td colspan="1" rowspan="1">S. mitis;S. oralis</td><td colspan="1" rowspan="1">S. mitis;S. oralis;S. australis;S. cristatus;S. dentisani;S. infantis;S. oligofermentans;S. pseudopneumoniae;S. rubneri;S. sanguinis;S. tigurinus</td><td colspan="1" rowspan="1">For organisms identified bythe MBT-CA asStreptococcus pneumoniae orStreptococcus mitis/oralisgroup; it is recommended toproceed with full extraction(EXT) for finalidentification.During clinical trials severalspecies were identified withthe reference algorithm (lowconfidence) for isolatesidentified as "S. mitis oralisgroup" with the MBT-CA.The following species arementioned: Streptococcusmitis;S. oralis;S. australis;S. cristatus;S. dentisani;S. infantis;S. mitis;S. oligofermentans;S. pseudopneumoniae;S. rubneri;S. sanguinis and S. tigurinus.They may be identified bythe Streptococcus mitisoralis reference spectrum inthe MBT-CA.</td></tr><tr><td colspan="1" rowspan="1">da</td><td colspan="1" rowspan="1">Veillonella parvula group</td><td colspan="1" rowspan="1">V. parvula</td><td colspan="1" rowspan="1">V. parvula;V. denticariosi;V. dispar;V. rugosae</td><td colspan="1" rowspan="1">Differentiation betweenmembers of the displayedgroup by MBT-CA is notpossible although thereference method (proteingene sequencing) is able todistinguish the mentionedspecies. Therefore they aregrouped together.</td></tr><tr><td colspan="1" rowspan="1">bh</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">V. parahaemolyticus</td><td colspan="1" rowspan="1">V. parahaemolyticus;V. alginolyticus;V. harveyi;V. campbellii;V. natriegens;V. rotiferianus</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.</td></tr><tr><td colspan="1" rowspan="1">W H</td><td colspan="1" rowspan="1">Species / Group / Complex</td><td colspan="1" rowspan="1">Strains Included inDatabase</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification.Confirmatory tests arerequired todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint</td></tr><tr><td colspan="1" rowspan="1">y</td><td colspan="1" rowspan="1">Yersinia pseudotuberculosis</td><td colspan="1" rowspan="1">Y. pseudotuberculosis</td><td colspan="1" rowspan="1">Y. pseudotuberculosis;Y. pestis;Y. similis</td><td colspan="1" rowspan="1">Based on 16S rRNA genesequencing a secure speciesdifferentiation between thedisplayed species is difficult.Y. pestis is not included in theMBT-CA database. Y. pestisis a select agent and should beruled out; handle isolate withextreme caution and handle inaccordance with local;state;and federal accreditingorganizations' requirementsas applicable. All speciesassociated with the displayedidentification have beenreported as isolated fromhuman specimens.</td></tr></table>

Table: Matching Hint Table / Synonym Table Yeasts   

<table><tr><td colspan="1" rowspan="1">W c</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">strains used fordatabase creation</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification. Basedon ITS sequencing asecure speciesdifferentiationbetween thedisplayed species isdifficult.Confirmatory testsare required todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint;Synonyms(anamorph or teleomorph)</td></tr><tr><td colspan="1" rowspan="1">bi</td><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">C. famata</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Debaryomyces hansenii</td></tr><tr><td colspan="1" rowspan="1">bj</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">C. guilliermondii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. guilliermondii;Meyerozyma caribbica;Meyerozyma guilliermondii;Pichia guilliermondii;Pichia caribbica</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida inconspicua</td><td colspan="1" rowspan="1">C. inconspicua</td><td colspan="1" rowspan="1">C. inconspicua;C. norvegensis;P. cactophilia</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">bk</td><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">C. kefyr</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. kefyr;Kluyveromyces marxianus</td></tr><tr><td colspan="1" rowspan="1">bl</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">C. krusei</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. krusei;Issatchenkia orientalis;Pichia kudriavzevii</td></tr><tr><td colspan="1" rowspan="1">bm</td><td colspan="1" rowspan="1">Candida lambica</td><td colspan="1" rowspan="1">C. lambica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. lambica;Pichia fermentans</td></tr><tr><td colspan="1" rowspan="1">bn</td><td colspan="1" rowspan="1">Candida lipolytica</td><td colspan="1" rowspan="1">C. lipolytica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. lipolytica;Yarrowia lipolytica</td></tr><tr><td colspan="1" rowspan="1">bo</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">C. lusitaniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. lusitaniae;Clavispora lusitaniae</td></tr><tr><td colspan="1" rowspan="1">bp</td><td colspan="1" rowspan="1">Candida norvegensis</td><td colspan="1" rowspan="1">C. norvegensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. norvegensis;Pichia norvegensis</td></tr><tr><td colspan="1" rowspan="1">bq</td><td colspan="1" rowspan="1">Candida pelliculosa</td><td colspan="1" rowspan="1">C. pelliculosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. pelliculosa;Pichia anomala;Wickerhamomyces anomalus</td></tr><tr><td colspan="1" rowspan="1">br</td><td colspan="1" rowspan="1">Candida valida</td><td colspan="1" rowspan="1">C. valida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C. valida;Pichia membranifaciens</td></tr><tr><td colspan="1" rowspan="1">bs</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">C. gattii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Cryptococcus gattii;Cryptococus bacillisporus;Filobasidiella bacillispora</td></tr><tr><td colspan="1" rowspan="1">db</td><td colspan="1" rowspan="1">Cyberlindnera jadinii</td><td colspan="1" rowspan="1">C. jadinii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida utilis</td></tr><tr><td colspan="1" rowspan="1">bt</td><td colspan="1" rowspan="1">Geotrichum candidum</td><td colspan="1" rowspan="1">G. candidum</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Dipodascus geotrichum;Galactomyces geotrichum;G. candidum</td></tr><tr><td colspan="1" rowspan="1">bu</td><td colspan="1" rowspan="1">Geotrichum capitatum</td><td colspan="1" rowspan="1">G. capitatum</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Dipodascus capitatus;G. capitatum;Magnusiomyces capitatus</td></tr><tr><td colspan="1" rowspan="1">W Hit</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">strains used fordatabase creation</td><td colspan="1" rowspan="1">Different species arepotentially associatedwith the displayedidentification. Basedon ITS sequencing asecure speciesdifferentiationbetween thedisplayed species isdifficult.Confirmatory testsare required todifferentiate betweenlisted organisms.</td><td colspan="1" rowspan="1">Matching Hint;Synonyms(anamorph or teleomorph)</td></tr><tr><td colspan="1" rowspan="1">bv</td><td colspan="1" rowspan="1">Kloeckera apiculata</td><td colspan="1" rowspan="1">K. apiculata</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Hanseniaspora uvarum;K. apiculata</td></tr><tr><td colspan="1" rowspan="1">bw</td><td colspan="1" rowspan="1">Pichia ohmeri</td><td colspan="1" rowspan="1">P. ohmeri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida guilliermondii varmembranaefaciens;Kodamaea ohmeri;Pichia ohmeri</td></tr><tr><td colspan="1" rowspan="1">bx</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">S. cerevisiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida robusta;S. cerevisiae</td></tr><tr><td colspan="1" rowspan="1">dc</td><td colspan="1" rowspan="1">Trichosporon mucoidesgroup</td><td colspan="1" rowspan="1">T. dermatis,T. mucoides</td><td colspan="1" rowspan="1">T. mucoides;T. dermatis</td><td colspan="1" rowspan="1">Differentiation between membersof the displayed group by MBT-CA is not possible. The referencemethod (ITS sequencing) is alsonot able to distinguish thementioned species. Therefore theyare grouped together.</td></tr></table>

$k .$ Clinical specificity: See clinical sensitivity results l. Other clinical supportive data (when a. and $b .$ . are not applicable):

2. Clinical cut-off: See Assay cut-off

3. Expected values/Reference range: See Assay cut-off

# N. Instrument Name:

MALDI Biotyper CA (MBT-CA) System, MBT smart CA System

O. System Descriptions: Also see K130831

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes X or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification:

The user manually enters the specimen identification information into the MALDI Biotyper CA System. The user first defines active sample positions (see section 4.2.1 of User Manual) and US IVD BTS control positions (see section 4.2.2 of User Manual). The defined sample positions are required to have a user entered and valid sample identifier in the Id column along with an optional description. All inoculated positions must contain a valid ID before the run can proceed. After all sample positions have been input, at least two US IVD BTS control positions must be defined. It is recommended that the cross-joint positions on the target are used as quality control positions.

Alternatively, the user can use an optional Honeywell (Hyperion $1 3 0 0 \mathrm { g }$ ) Barcode Reader USB cable is connected to the MALDI Biotyper CA System computer. The barcode reader scans the unique ten-digit target ID which appears in the Target ID box on the target plate. After the target ID has been entered, the a new Run page opens and the ten-digit target ID appears as the Plate Id and is appended to the Run name. Sample identifications are entered into the computer corresponding to the target plate position for that run.

# 4. Specimen Sampling and Handling:

After incubation of bacteria on recommended isolation media for $1 8 { - } 2 4 \mathrm { h }$ at $( 3 7 ^ { \circ } \mathrm { C }$ $\pm 2 ^ { \circ } \mathrm { C } )$ , colonies are stable for up to $^ { 1 2 \mathrm { ~ h ~ } }$ when held at room temperature.

Using a sterile colony transfer device, smear isolated colonies of bacteria as a thin film directly onto a sample position on a cleaned target. Inoculating an appropriate amount of the test organism onto the target is important. Excessive or insufficient amounts of inoculum may impact organism identification. The User Manual visually illustrates suitable and unsuitable inoculum amounts of organism on target.

Each of the sample positions and US IVD BTS control positions are overlayed with 1 $\mu \mathrm { L }$ US IVD HCCA portioned solution. Use a new pipette tip to add matrix to each inoculated sample position. Dry the inoculated plate at room temperature. The inoculated MALDI target plate is now ready for use. For stability see Sections M.1.j and M.1.k above.

If the MALDI BIOTYPER CA System identification of the test organism does not result in a bacterial identification with a log(score) value of $\geq 2 . 0$ , repeat testing using the extraction procedure in Section 3.6 of the User Manual. Briefly, the extraction of a sample from an isolated colony consists of multiple washing cycles of ethanol and water followed by addition of formic acid and acetonitrile followed by a centrifugation step. After centrifugation, a sample of the supernatant is applied to the target plate, dried and loaded into the MALDI Biotyper CA System for analysis.

5. Calibration: See K130831

6. Quality Control: See Section M.1.c above.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10, 21 CFR 801.109, and special controls of regulation 21 CFR 866. 3361.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.