# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K163390

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for a new device, iC-GPC Assay on the iC-System.

# C. Measurand:

Nucleic acid sequences of the following gram positive organisms and associated resistance markers: Staphylococcus aureus, Staphylococcus epidermidis, Enterococcus faecalis, Enterococcus faecium, mecA, vanA and vanB.

# D. Type of Test:

Qualitative in vitro diagnostic test utilizing polymerase chain reaction (PCR) for the multiplex amplification of specific targets followed by detection of the amplified products with microarray hybridization.

# E. Applicant

iCubate, Inc.

F. Proprietary and Established Names:

iC-GPC Assay iC-System

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3365, Multiplex nucleic acid assay for identification of microorganisms and resistance markers from positive blood cultures.

2. Classification:

Class II

3. Product codes:

PAM: Gram positive bacteria and their resistance markers NSU: Instrumentation for clinical multiplex test systems

4. Panel:

Microbiology

# H. Intended Use:

1. Intended use(s):

The iCubate iC-GPC Assay for use on the iC-System is a qualitative, multiplexed, in vitro diagnostic test for the detection and identification of potentially pathogenic gram positive bacteria, which may cause bloodstream infection (BSI). The iC-GPC Assay is performed directly on positive blood cultures, confirmed by Gram stain to contain gram positive cocci. Cultures demonstrating mixed Gram stain results should not be tested on the assay. The iC-GPC Assay is validated for use with select BACTEC, BacT/ALERT and VersaTREK blood culture bottles. The iC-GPC Assay is indicated for use in conjunction with other clinical and laboratory findings, such as blood culture isolate identification and antimicrobial susceptibility testing, to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor bloodstream infections.

The iC-GPC Assay detects organism DNA and identifies the following bacterial species and resistance markers:

<table><tr><td>Bacterial Species</td><td>Resistance Markers</td></tr><tr><td>Staphylococcus epidermidis Streptococcus pneumoniae</td><td>vanA- associated with vancomycin</td></tr></table>

The iC-GPC Assay detects the mecA resistance marker, inferring mecA-mediated methicillin resistance, and the vanA and vanB resistance markers, inferring vanA/vanBmediated vancomycin resistance. In mixed growth, the iC-GPC Assay does not specifically attribute van-mediated vancomycin resistance to either $E$ . faecalis or $E$ . faecium, or mecA-mediated methicillin resistance to either $S .$ aureus or $S .$ epidermidis.

Sub-culturing of positive blood cultures is necessary to recover organisms for susceptibility testing, identification of organisms not detected by the iC-GPC Assay, differentiation of mixed growth, association of antimicrobial resistance marker genes to a specific organism, or for epidemiological typing.

2. Indication(s) for use: Same as the Intended Use

3. Special conditions for use statement(s):

For prescription use only The iC-GPC Assay has only been validated for use with the following blood culture bottles: BD BACTEC Standard/10 Aerobic/F, BD BACTEC Standard/10 Anaerobic/F, BD BACTEC Plus Aerobic/F, BD BACTEC Plus Anaerobic/F, BD BACTEC Lytic/10 Anaerobic/F, VersaTREK REDOX 1, VersaTREK REDOX 2, BacT/ALERT SA Standard Aerobic, BacT/ALERT FA Aerobic FAN, and BacT/ALERT FA Plus Aerobic.

4. Special instrument requirements: For use on the iC-System

# I. Device Description:

The iC-GPC Assay detects gram positive bacteria and resistance markers from positive blood culture specimens that are determined to contain gram positive cocci by Gram stain.

The assay utilizes PCR for the multiplex amplification of specific gram positive bacterial and resistance marker targets and detects the amplified targets with microarray hybridization. The assay utilizes proprietary ARM-PCR (Amplicon Rescued Multiplex PCR) technology allowing multiple targets to be amplified in one reaction. Testing is performed using a selfcontained disposable cassette that is processed by the iC-Processor. After processing, the cassette is read using the iC-Reader. Final results are generated by the iC-Reader aided by computer software for data acquisition, analysis and display in the iC-Report.

The iC-GPC Cassettes are provided in individually sealed plastic pouches which must be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

The analytes detected and reported by the iC-GPC Assay include the following:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>iC-GPC Targets</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis (SE)</td><td rowspan=1 colspan=1>gseA, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (SA)</td><td rowspan=1 colspan=1>nuc, mecA</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae (SPN)</td><td rowspan=1 colspan=1>lytA</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (EFLS)</td><td rowspan=1 colspan=1>ddl, vanB</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (EFCM)</td><td rowspan=1 colspan=1>fcm, vanA</td></tr></table>

Materials Provided:

• iC-GPC Cassettes (shipped and stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ )

Materials required but not provided (instruments and equipment):

• iC-Processor   
• iC-Reader   
• iC-Report Software   
• $2 { - } 8 ^ { \circ } \mathrm { C }$ Refrigerator   
• ${ \le } 7 0 ^ { \circ } \mathrm { C }$ Freezer   
• Automated blood culture monitoring system   
• Micro-pipettes & tips   
• Blood culture bottles   
• Gram staining reagents

Test report and interpretation:

After reading a cassette, data is transferred from the iC-Reader to the iMac computer. Data is analyzed using iC-Report software and a final result is generated. The user may view or print the final result.

iC-GPC provides a qualitative result for the presence (“Detected”) or absence (“Not Detected”) of iC-GPC target organisms. Detected organisms and resistance markers are reported separately. If a resistance marker is identified without an associated organism concurrently detected, the resistance marker will not be reported. Positive mecA resistance marker results will only be reported for specimens that are also positive for Staphylococcus aureus or Staphylococcus epidermidis. Positive vanA and vanB resistance marker results will only be reported for specimens that are also positive for Enterococcus faecium or Enterococcus faecalis.

If multiple organisms are detected, all organisms detected will be reported with the following disclaimer:

‘The report is indeterminate; the sample may contain multiple organisms. Isolate the organisms and perform alternate tests for organism identification and potential resistance.’

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene Gram Positive Blood Culture NucleicAcid Test (BC-GP)

2. Predicate 510(k) number:

K122514

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate (K122514)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative multiplexed invitro diagnostic device forthe detection andidentification of potentiallypathogenic gram-positivebacteria and associatedresistance markers frompositive blood culturebottles.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Organisms and resistancemarkers detected</td><td colspan="1" rowspan="1">Staphylococcus aureus,Staphylococcus epidermidis,Streptococcus pneumoniae,Enterococcus faecalis,Enterococcus faecium,mecA, vanA and vanB</td><td colspan="1" rowspan="1">Same plus additionalorganisms (see differencestable below)</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Positive Blood Culturespecimens containing grampositive cocci</td><td colspan="1" rowspan="1">Positive Blood Culturespecimens containing grampositive bacteria</td></tr><tr><td colspan="1" rowspan="1">Specimen Processing andPurification</td><td colspan="1" rowspan="1">Automated by instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Device includes one internalpositive control of heat-killed Bacillusthuringiensis. Externalcontrols are not provided.</td><td colspan="1" rowspan="1">Device includes twointernal controls, onecontaining intact Bacillussubtilis and the othercontaining an assay-specific DNA target.External controls are notprovided.</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate (K122514)</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">iC-System</td><td colspan="1" rowspan="1">Verigene System</td></tr><tr><td colspan="1" rowspan="1">Organisms and ResistanceMarkers Detected</td><td colspan="1" rowspan="1">Staphylococcus aureusStaphylococcusepidermidisStreptococcus pneumoniaeEnterococcus faecalisEnterococcus faeciummecAvanAvanB</td><td colspan="1" rowspan="1">Same plus the followingadditional organisms:Staphylococcus speciesStaphylococcuslugdenensisStreptococcus spp.Streptococcus, pyogenesStreptococcus agalactiaeStreptococcus anginosusgroupEnterococcus faecalisEnterococcus faeciumListeria spp.</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Heat lysis followed by two-step ARM-PCR,microarray hybridizationand detection utilizingfluorescently labeledprobes.</td><td colspan="1" rowspan="1">Cell lysis and magneticbead-based bacterial DNAisolation followed bydetection andidentification of bacterial-specific DNA in amicroarray format usinggold nanoparticle probe-based hybridizationtechnology.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">iC-Processor and iC-Reader</td><td colspan="1" rowspan="1">Verigene Reader andProcessor SP</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">4.5 hours</td><td colspan="1" rowspan="1">2.5 hours</td></tr></table>

# K. Standard/Guidance Document Referenced:

None cited

# L. Test Principle:

The iC-GPC Assay is a multiplexed molecular assay that uses proprietary ARM-PCR (Amplicon Rescued Multiplex PCR) technology. The following chemical reactions take place in the iC-GPC Cassette:

1. DNA extraction – Nucleic acid extraction is performed using heat to lyse the gram positive bacteria and release DNA to be used in subsequent multiplex ARM-PCR amplification.

2. $1 ^ { \mathrm { s t } }$ stage multiple PCR – The ARM‐PCR protocol includes two steps of amplification.

The first step of amplification utilizes a set of nested sequence‐specific primers designed for each target. These primers are used to enrich targets as well as incorporate tags and priming sites for the $2 ^ { \mathrm { n d } }$ stage PCR reaction.

3. $2 ^ { \mathrm { n d } }$ stage PCR‐ Following the $1 ^ { \mathrm { s t } }$ stage PCR, a $2 ^ { \mathrm { n d } }$ stage PCR is conducted using universal primers specific for tags incorporated into the amplicon during the $1 ^ { \mathrm { s t } }$ stage reaction. The $2 ^ { \mathrm { n d } }$ stage PCR exponentially amplifies the target sequence and prepares it for microarray hybridization by enriching for single strand amplicons.

4. Microarray Hybridization‐ Upon completion of the $2 ^ { \mathrm { n d } }$ stage PCR, amplified DNA is bound by gene specific fluorescently labeled probes and hybridizes to positions on the microarray specific for each target. Binding to a specific spot is measured by emission from the fluorescently labeled probes upon excitation with a laser in the iC‐ Reader and is interpreted by iC‐Report to generate a data report.

5. The iC‐Report software controls the operation of the iC‐Processor and iC‐Reader, analyzes data, and generates a test report after a run. The entire process takes approximately four and a half hours.

# M. Performance Characteristics:

1. Analytical performance:

a. Evaluation of organism concentration at bottle positivity (bottle ring)

A study was performed to establish the concentration of each iC-GPC Assay target organism at initial bottle positivity. A total of nineteen organisms were evaluated. Organisms were inoculated into BD BACTEC Plus Aerobic blood culture bottles with human blood added. Bottles were allowed to incubate on the blood culture system until initial bottle positivity. Bottles were removed from the incubator within two hours of bottle ring after which plating and subsequent colony counts were performed to confirm organism concentrations. A minimum of five bottles were evaluated for each strain. The average concentrations at bottle ring are presented in Table 1 below. These concentrations, representative of the levels that may be observed in a clinical setting, are equivalent to or greater than the respective target LoDs (see below) and were used to determine target organism concentrations for analytical studies.

Table 1: Target Organism Concentrations at Bottle Positivity   

<table><tr><td colspan="1" rowspan="1">Organism/Strain ID</td><td colspan="1" rowspan="1">Average Concentration (CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">SE 700566</td><td colspan="1" rowspan="1">2.68 × 107</td></tr><tr><td colspan="1" rowspan="1">SE 35984</td><td colspan="1" rowspan="1">2.04 × 107</td></tr><tr><td colspan="1" rowspan="1">SE 12228</td><td colspan="1" rowspan="1">2.59 × 107</td></tr><tr><td colspan="1" rowspan="1">SE 49134</td><td colspan="1" rowspan="1">1.01 × 107</td></tr><tr><td colspan="1" rowspan="1">SA 700699</td><td colspan="1" rowspan="1">5.00 × 108</td></tr><tr><td colspan="1" rowspan="1">SA BAA 1768</td><td colspan="1" rowspan="1">5.24 × 107</td></tr><tr><td colspan="1" rowspan="1">SA BAA 977</td><td colspan="1" rowspan="1">1.14 × 108</td></tr><tr><td colspan="1" rowspan="1">SA 25923</td><td colspan="1" rowspan="1">6.06 × 108</td></tr><tr><td colspan="1" rowspan="1">SPN 6301</td><td colspan="1" rowspan="1">5.40 × 106</td></tr><tr><td colspan="1" rowspan="1">SPN 700673</td><td colspan="1" rowspan="1">5.95 × 106</td></tr><tr><td colspan="1" rowspan="1">EFLS 51299</td><td colspan="1" rowspan="1">4.72 × 1010</td></tr><tr><td colspan="1" rowspan="1">EFLS 700802</td><td colspan="1" rowspan="1">2.22 × 108</td></tr><tr><td colspan="1" rowspan="1">EFLS JMI 12536</td><td colspan="1" rowspan="1">2.27 × 108</td></tr><tr><td colspan="1" rowspan="1">EFLS 29212</td><td colspan="1" rowspan="1">5.53 × 1010</td></tr><tr><td colspan="1" rowspan="1">EFLS BAA 2128</td><td colspan="1" rowspan="1">5.77 × 107</td></tr><tr><td colspan="1" rowspan="1">EFCM 700221</td><td colspan="1" rowspan="1">1.16 × 109</td></tr><tr><td colspan="1" rowspan="1">EFCM 51559</td><td colspan="1" rowspan="1">5.87 × 108</td></tr><tr><td colspan="1" rowspan="1">EFCM 35667</td><td colspan="1" rowspan="1">2.86 × 107</td></tr><tr><td colspan="1" rowspan="1">EFCM BAA 2127</td><td colspan="1" rowspan="1">2.70 × 108</td></tr></table>

# b. Reproducibility

A multi-site reproducibility study was conducted to assess the site‐to‐site, operator‐to operator, and lot‐to‐lot reproducibility of the iC‐GPC Assay. A representative panel of iC‐GPC target organisms and one non‐target organism were tested at two concentrations, the concentration present at bottle ring and the concentration present after eight hours incubation post bottle ring. Testing was performed by two independent operators at each of three sites, two external and one in‐house. The panel was tested in replicates of three across five, non‐consecutive days. Testing was evaluated across three cassette lots and four iC‐Systems.

Five organisms, representing each of the iC‐GPC Assay targets, plus one non‐target organism were evaluated (see table 2 below). Blood culture bottles were inoculated with low organism concentrations and appropriate volumes of human blood and incubated on the automated blood culture instrument until signaled as positive by the instrument. Within one hour after initial positivity, an aliquot was removed from each bottle. Smaller aliquots were prepared and frozen at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ until testing was performed. The bottle was returned to the blood culture instrument for approximately eight additional hours of incubation. After this extended incubation, the bottle was removed, and aliquots were prepared and frozen at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ until testing.

Table 2: Reproducibility Test Panel   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>iC-GPC Targets</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>ATCC700566</td><td rowspan=1 colspan=1>gseA, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCC700699</td><td rowspan=1 colspan=1>nuc,mecA</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>ATCC6301</td><td rowspan=1 colspan=1>lytA</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC51299</td><td rowspan=1 colspan=1>ddl, vanB</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC700221</td><td rowspan=1 colspan=1>fcm,vanA</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumstriatum</td><td rowspan=1 colspan=1>MCW</td><td rowspan=1 colspan=1>---</td></tr></table>

Testing was performed by two operators at each of three sites; two external and one internal. Study specimens were randomized and blinded and tested over five nonconsecutive days. Each panel member was tested in triplicate per site, per day and per operator. Three lots of iC-GPC Cassettes were included in the study.

Results from testing are shown in Table 3 below. Two false negative results were observed, one bottle negative for both S. aureus and mecA. Six false positive results were observed, five $E .$ . faecalis and one Staphylococcus epidermidis/mecA. All false positive results occurred in blood culture bottles containing the non-targeted organism Corynebacterium striatum, with three false positive results (two $E$ . faecalis and one $S .$ . epidermidis/mecA) occurring at initial bottle ring and the three additional false positive results (all $E .$ . faecalis) occurring after eight hours incubation post bottle ring. A total of 15 tests $( 1 . 4 \% )$ generated invalid results and were not included in the data analysis.

Table 3: iC-GPC Reproducibility Study Results by Target Organism and Concentration   

<table><tr><td colspan="1" rowspan="1">Organism/GeneTarget/Concentration</td><td colspan="1" rowspan="1">OverallPerformance</td><td colspan="1" rowspan="1">OverallPerformance%[95% CI]</td><td colspan="1" rowspan="1">FalseNegatives</td><td colspan="1" rowspan="1">FalsePositives</td><td colspan="1" rowspan="1">PositiveControls CheckFailures</td><td colspan="1" rowspan="1">SystemFailures</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis (gseA)Bottle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/976(0.10%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis (gseA)Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/975(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">S. aureus (nuc)Bottle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">S. aureus (nuc)Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9[93.97-99.80]</td><td colspan="1" rowspan="1">1/90(1.12%)</td><td colspan="1" rowspan="1">0/975(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae (lytA)Bottle Ring</td><td colspan="1" rowspan="1">88/88</td><td colspan="1" rowspan="1">100.0[95.82-100.0]</td><td colspan="1" rowspan="1">0/88(0.00%)</td><td colspan="1" rowspan="1">0/977(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae (lytA)Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis (ddl)Botle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">2/976(0.20%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis (ddl)Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86, 100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">3/976(0.31%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. faecium (fcm)Bottle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/975(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. faecium (fcm)Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">mecABottle Ring</td><td colspan="1" rowspan="1">177/178</td><td colspan="1" rowspan="1">99.4[96.89-99.90]</td><td colspan="1" rowspan="1">1/178(0.56%)</td><td colspan="1" rowspan="1">1/887(0.11%)</td><td colspan="1" rowspan="1">1/180(0.56%)</td><td colspan="1" rowspan="1">1/180(0.56%)</td></tr><tr><td colspan="1" rowspan="1">mecABottle Ring + 8 hours</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0[97.91-100.0]</td><td colspan="1" rowspan="1">0/180(0.00%)</td><td colspan="1" rowspan="1">0/885(0.00%)</td><td colspan="1" rowspan="1">0/180(0.00%)</td><td colspan="1" rowspan="1">0/180(0.00%)</td></tr><tr><td colspan="1" rowspan="1">vanABottle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/975(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">vanABottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">vanBBottle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">vanBBottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86, 100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/976(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr></table>

Results from the reproducibility study demonstrated that the iC-GPC Assay generates reproducible results for each targeted organism and resistance marker in positive blood culture specimens containing organism concentrations present at bottle ring and after eight hours additional incubation.

c. Linearity/assay reportable range: Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

Specimen stability:

A study was conducted to evaluate the stability of positive blood culture specimens stored under various conditions for use with the iC-GPC Assay. Samples replicates were prepared with six different organisms, representing each of the iC‐GPC Assay targets and one non‐target organism, Corynebacterium striatum (see Table 4 below). Samples were prepared by inoculating low concentrations of organisms into blood culture bottles with human blood added followed by incubation on an automated blood culture instrument until positivity.

Table 4: Specimen Stability Test Panel   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>iC-GPC Targets</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC700566</td><td rowspan=1 colspan=1>gseA, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCC700699</td><td rowspan=1 colspan=1>nuc, mecA</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC6301</td><td rowspan=1 colspan=1>lytA</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC51299</td><td rowspan=1 colspan=1>ddl, vanB</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC700221</td><td rowspan=1 colspan=1>fcm, vanA</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>MCW</td><td rowspan=1 colspan=1>---</td></tr></table>

Samples were stored under three temperature conditions: incubated $( 3 5 - 3 7 ^ { \circ } \mathrm { C }$ , on the automated blood culture instrument), ambient temperature $( 2 0 { - } 2 5 ^ { \circ } \mathrm { C } )$ , and refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ . Samples were tested in triplicate for each organism and storage condition at each of the following time points: 8, 24, and 48 hours beyond initial bottle positivity. For each time point and storage condition, results of the study demonstrated $100 \%$ detection for all bottle replicates containing targeted organisms and $100 \%$ negative results for bottles containing the non-targeted organism.

Results from the specimen stability study support the following stability claims for positive specimens stored under the following conditions: incubated $( 3 5 - 3 7 ^ { \circ } \mathrm { C } )$ , room temperature $( 2 0 { - } 2 5 ^ { \circ } \mathrm { C } )$ , and refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 48 hours beyond initial bottle positivity.

Fresh versus frozen and freeze/thaw studies:

A study was performed to evaluate the effect of freezing blood culture specimens on performance of iC-GPC Assay. The same panel of organisms presented in Table 4 above was evaluated in contrived blood culture specimens with human blood added. Samples were evaluated with organism concentrations both at bottle ring and eight hours post bottle ring concentrations. For each organism and organism concentration, three replicates were evaluated after both two and four weeks of frozen storage between $- 1 0 ^ { \circ } \mathrm { C }$ and $- 3 0 ^ { \circ } \mathrm { C }$ . Study results demonstrated $100 \%$ detection of samples containing targeted organisms and $100 \%$ negative results for bottles containing the non-targeted organism for each time point and organism concentration evaluated.

Additional testing was conducted with the same sample panel to evaluate the effect on positive blood culture specimens subjected to one or two freeze-thaw cycles. Again, positive blood culture samples with organism concentrations at bottle ring and eight hours post bottle ring concentrations were evaluated. Results of the study showed $100 \%$ detection as expected and demonstrated no effect on iC-GPC Assay performance for blood culture specimens subjected to two freeze/thaw cycles.

Results from the fresh versus frozen and the freeze/thaw studies demonstrated equivalent iC-GPC Assay performance between fresh and frozen specimens and for specimens subjected to one or two freeze thaw cycles. These results support the use of frozen clinical specimens in the clinical study as well as frozen contrived specimens in the reproducibility study.

# Assay controls:

Internal DNA amplification and processing control: An internal positive control of heat‐killed Bacillus thuringiensis is loaded into the test sample prior to DNA extraction. The positive control is lysed, amplified using primers specific for a $B .$ . thuringiensis cry gene, processed, and hybridized to the array at the same time as the sample. Reporting of the positive control in the iC‐Report confirms the assay was successful and ran to completion. In cases where targets are present at high concentrations, it is possible the target sequence will be preferentially amplified over the positive control. If the positive control is not detected on the array but a target is reported, the assay is still considered valid. If neither positive control nor target is reported, the assay is considered to have failed; an end report of “Positive controls check failed” will be generated.

Recommended External Controls: The iC-GPC Assay does not include external controls. The following recommendations for external control testing are included in the iC-GPC package insert:

Cultured organisms for each of the iC‐GPC Assay targets should be tested occasionally in accordance with laboratory defined quality control requirements. It is recommended that external quality controls are performed before testing newly manufactured cassette lots and any time system performance is in question. A fresh culture of recommended quality control organisms should be isolated on TSA $5 \%$ sheep blood agar plates or equivalent media. From the growth plate, transfer the organisms into saline, and adjust organism concentration to 0.5 MacFarland or $1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m l }$ equivalent. Then, test in accordance to iC‐GPC Assay instructions. If the external control fails, the test should be repeated. The strains listed in Table 5 can be used for external control testing.

Table 5: External Controls, Recommended Organisms   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ATCC ID Number</td><td rowspan=1 colspan=1>Target</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>6301</td><td rowspan=1 colspan=1>S. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>51299</td><td rowspan=1 colspan=1>E. faecalis andvanB</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>700699</td><td rowspan=1 colspan=1>S. aureus andmecA</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>700221</td><td rowspan=1 colspan=1>E. faecium andvanA</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>700566</td><td rowspan=1 colspan=1>S. epidermidis andmecA</td></tr></table>

External control testing, clinical study: A total of 551 quality control samples were evaluated in the clinical study and 24 quality control failures were observed.

Six quality control failures were due to incorrect organism calls with three samples generating false positive results and three samples generating false negative results. A total of 17 quality control samples generated internal control failures or instrument failures: twelve positive control check failures, two processor errors and two array registration errors. One additional sample was processed but not read by the site.

Quality control failures stratified by targeted organism are shown in Table 6 below.

Table 6: Quality Control Results, Clinical Study   

<table><tr><td rowspan=1 colspan=1>QCTesting</td><td rowspan=1 colspan=2>S. aureus(nuc, mecA)</td><td rowspan=1 colspan=2>S. epidermidis(gseA, mecA)</td><td rowspan=1 colspan=2>S. pneumoniae(lytA)</td><td rowspan=1 colspan=2>E. faecalis(ddl1, vanB)</td><td rowspan=1 colspan=2>E. faecium(fcm, vanA)</td><td rowspan=1 colspan=1>Total TestsPerformed</td><td rowspan=1 colspan=1>PercentagePass/Fail</td></tr><tr><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>95.3%</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>91.8%</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>527</td><td rowspan=1 colspan=1>95.6%</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>4.4%</td></tr></table>

# e. Detection limit:

A study was conducted to evaluate the Limit of Detection (LoD) of the iC-GPC Assay for each targeted analyte. A total of 19 bacterial strains were evaluated with a minimum of two strains per iC‐GPC Assay target. The strains evaluated are shown in Table 7.

Table 7: LoD Test Panel   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>iC-GPC Targets</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC700566</td><td rowspan=1 colspan=1>gseA, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC35984</td><td rowspan=1 colspan=1>gseA, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC12228</td><td rowspan=1 colspan=1>gseA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC49134</td><td rowspan=1 colspan=1>gseA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCC700699</td><td rowspan=1 colspan=1>nuc, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCCBAA1768</td><td rowspan=1 colspan=1>nuc, mecA</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCCBAA977</td><td rowspan=1 colspan=1>nuc</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCC25923</td><td rowspan=1 colspan=1>nuc</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>ATCC6301</td><td rowspan=1 colspan=1>lytA</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>ATCC700673</td><td rowspan=1 colspan=1>lytA</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC51299</td><td rowspan=1 colspan=1>dl, vanB</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>dl, vanB</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>JMI 12536</td><td rowspan=1 colspan=1>ddl, vanA</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC 29212</td><td rowspan=1 colspan=1>ddl</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCCBAA2128</td><td rowspan=1 colspan=1>dl</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC700221</td><td rowspan=1 colspan=1>fcm, vanA</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC51559</td><td rowspan=1 colspan=1>fcm, vanA</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC35667</td><td rowspan=1 colspan=1>fcm</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCCBAA2127</td><td rowspan=1 colspan=1>fcm</td></tr></table>

The LoD study included preliminary estimations of target LoDs based on testing of 10 sample replicates of each strain at several organism concentrations. Additional testing was performed with three organism concentrations spanning the estimated LoD, with replicates of 20 tested on three different lots for each dilution. All samples were prepared in a blood culture media/blood matrix.

The final LoD for each strain was calculated as the lowest organism concentration (CFU/mL) demonstrating a detection rate of approximately $9 5 \%$ . Final LoD organism concentrations were determined by plating and subsequent colony counts. A total of 91 runs with positive controls check failures $( 2 . 1 1 \% )$ and 21 system failures $( 0 . 4 9 \% )$ were observed in the study and were excluded from the data analysis. Results from LoD testing are shown in Table 8.

Table 8: Final LoDs for Targeted Analytes   

<table><tr><td rowspan=1 colspan=1>iC-GPC Organism target</td><td rowspan=1 colspan=1>Defined LoD (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.6 × 106 - 1.7 × 107</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1.7 × 106  4.4 × 106</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>7.4 × 105  9.5 × 106</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1.3 × 106  6.0 × 106</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>3.0 × 105  5.8 × 106</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>4.9 × 106 - 7.9 × 106</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>7.2 × 105  1.1 × 107</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>3.9 × 106  5.8 × 106</td></tr></table>

# f. Analytical reactivity (Inclusivity)

The analytical reactivity (inclusivity) for each iC-GPC Assay target was evaluated using multiple well-characterized and clinically relevant strains chosen to represent temporal, geographic, and genetic diversity. Inclusivity panel organisms included the following:

• 5 Staphylococcus epidermidis, mecA negative strains   
• 6 Staphylococcus epidermidis, mecA positive strains   
• 5 Staphylococcus aureus, mecA negative strains   
• 58 Staphylococcus aureus, mecA positive strains, representing various pulse- field gel electrophoresis types and including the following: - Vancomycin-intermediate SA (VISA) - Panton-Valentine Leukocidin (PVL)-producing SA - ATCC 43300 (hetero-resistant, mecA positive)   
• 2 borderline oxacillin-resistant Staphylococcus aureus strains (BORSA)   
• 10 Streptococcus pneumoniae strains   
• 5 Enterococcus faecalis, vanA/vanB negative strains   
• 5 Enterococcus faecalis, vanA or vanB positive strains   
• 5 Enterococcus faecium, vanA/vanB negative strains   
• 5 Enterococcus faecium, vanA or vanB positive strains

A total of 106 organisms were evaluated for iC-GPC Assay Inclusivity testing with testing performed in triplicate. Strains were tested near the target LoD (2-3x LoD) or at the lowest level of bottle positivity. Colony counts were used to confirm final concentration tested. In the event of a false negative result, the strain was retested near the target LoD in replicates of ten.

Results from the study showed that all expected targets were detected by the iC-GPC Assay with the exception of two false negative results: one false negative for $S$ . epidermidis and one false negative S. aureus/mecA. Repeat testing of the two strains generated 10/10 replicates with the expected positive results. In this study there were four runs with positive controls check failures $( 1 . 1 3 \% )$ and one system error $( 0 . 2 8 \% )$ . Results of inclusivity testing are summarized in Table 9 below.

Table 9: Inclusivity Study Results   

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Other IDs/Notes</td><td colspan="1" rowspan="1">ReplicatesDetected/Total Replicates</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Z318</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Z0801689</td><td colspan="1" rowspan="1">HER 1292</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">ATCC 700583</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Z291</td><td colspan="1" rowspan="1">ATCC 14990</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Z049</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">Z0801651</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">Z256</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">Z257</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">Z258</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">12/13</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">Z259</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis, mecA (+)</td><td colspan="1" rowspan="1">ATCC 29887</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Z021</td><td colspan="1" rowspan="1">ATCC 6538P</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">ATCC 12600</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Z0801675</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Z057</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Z153</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">HM-466</td><td colspan="1" rowspan="1">131</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">HM-467</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-10129</td><td colspan="1" rowspan="1">TCH60</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-10189</td><td colspan="1" rowspan="1">HFH-30364, USA400, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-10192</td><td colspan="1" rowspan="1">HFH-30106, non USA100-1100</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-13524</td><td colspan="1" rowspan="1">H342087</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-28983</td><td colspan="1" rowspan="1">S0385</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-45872</td><td colspan="1" rowspan="1">HIP07930, USA600</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-45880</td><td colspan="1" rowspan="1">LIM1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-45890</td><td colspan="1" rowspan="1">BR 5, VISA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46062</td><td colspan="1" rowspan="1">H2138 (isolate 10)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46063</td><td colspan="1" rowspan="1">P1V44, VISA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46072</td><td colspan="1" rowspan="1">1078, USA700</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46080</td><td colspan="1" rowspan="1">AIS 2006061, USA1000,PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46081</td><td colspan="1" rowspan="1">HIP12899, USA1100, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46218</td><td colspan="1" rowspan="1">GA-442, USA700</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46070</td><td colspan="1" rowspan="1">USA300-0114</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46171</td><td colspan="1" rowspan="1">CA-126, USA100</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46172</td><td colspan="1" rowspan="1">CA-127, USA300, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46177</td><td colspan="1" rowspan="1">CA-347, USA600</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46180</td><td colspan="1" rowspan="1">CA-409, USA200</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46182</td><td colspan="1" rowspan="1">CA-513, USA800</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46191</td><td colspan="1" rowspan="1">CO-34, USA300, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46197</td><td colspan="1" rowspan="1">CO-72, USA800</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46199</td><td colspan="1" rowspan="1">CT-110, USA100</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46207</td><td colspan="1" rowspan="1">CT-58, USA500</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46215</td><td colspan="1" rowspan="1">GA-356</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46221</td><td colspan="1" rowspan="1">GA-656, USA800</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46223</td><td colspan="1" rowspan="1">GA-92, USA300, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46224</td><td colspan="1" rowspan="1">NY-315, USA600</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46250</td><td colspan="1" rowspan="1">OR-130, USA100</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46251</td><td colspan="1" rowspan="1">OR-131, USA200</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46261</td><td colspan="1" rowspan="1">TN-112, USA300</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-46269</td><td colspan="1" rowspan="1">TN-82, USA200</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41875</td><td colspan="1" rowspan="1">M0001</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41876</td><td colspan="1" rowspan="1">M0006</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41877</td><td colspan="1" rowspan="1">M0055</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41878</td><td colspan="1" rowspan="1">M0102</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41879</td><td colspan="1" rowspan="1">M0108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41880</td><td colspan="1" rowspan="1">M0197</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41881</td><td colspan="1" rowspan="1">M0200</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41882</td><td colspan="1" rowspan="1">M0288</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41883</td><td colspan="1" rowspan="1">M0334</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41887</td><td colspan="1" rowspan="1">M0663</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41889</td><td colspan="1" rowspan="1">M0934</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41890</td><td colspan="1" rowspan="1">M0943</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41895</td><td colspan="1" rowspan="1">M1510</td><td colspan="1" rowspan="1">12/13</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-41896</td><td colspan="1" rowspan="1">M1565</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-10187</td><td colspan="1" rowspan="1">HFH-29994, USA100</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-13525</td><td colspan="1" rowspan="1">F338081</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-13526</td><td colspan="1" rowspan="1">W342179</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-13533</td><td colspan="1" rowspan="1">S247312</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-13546</td><td colspan="1" rowspan="1">SU-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-30544</td><td colspan="1" rowspan="1">HI049, USA300, PVL+</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">NR-45924</td><td colspan="1" rowspan="1">LinR #12</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">ATCC 700698</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">ATCC BAA44</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. aureus, mecA (+)</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">Hetero-resistant</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">MCW 109</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">BORSA</td><td colspan="1" rowspan="1">MCW 141</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z022</td><td colspan="1" rowspan="1">Clinical isolate, serotype 19F</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z073</td><td colspan="1" rowspan="1">Clinical isolate, serotype 19F</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z319</td><td colspan="1" rowspan="1">Clinical isolate, serotype 12F</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z278</td><td colspan="1" rowspan="1">ATCC BAA-255, R6 (non-virulent)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z279</td><td colspan="1" rowspan="1">PHE NCTC 11910, serotype23F</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z280</td><td colspan="1" rowspan="1">PHE NCTC 11897, serotype9V</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z282</td><td colspan="1" rowspan="1">ATCC 33400, NCTC 7465,serotype 1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z295</td><td colspan="1" rowspan="1">ATCC 49619, serotype 19F</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z261</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">Z262</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Z0801637</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">ATCC 33186</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">ATCC 49532</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Z289</td><td colspan="1" rowspan="1">ATCC 19433</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Z266</td><td colspan="1" rowspan="1">ATCC 6055</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis, vanB (+)</td><td colspan="1" rowspan="1">Z0801693</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis, vanA (+)</td><td colspan="1" rowspan="1">Z324</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis, vanA (+)</td><td colspan="1" rowspan="1">Z267</td><td colspan="1" rowspan="1">PHE NCTC 12201- vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis, vanA (+)</td><td colspan="1" rowspan="1">Z269</td><td colspan="1" rowspan="1">PHE NCTC 12203- vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecalis, vanB (+)</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Z322</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">ATCC 8459</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Z265</td><td colspan="1" rowspan="1">ATCC 9756</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Z290</td><td colspan="1" rowspan="1">ATCC 19434</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Z320</td><td colspan="1" rowspan="1">ATCC 19634</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium, vanA (+)</td><td colspan="1" rowspan="1">Z0801892</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium, vanB (+)</td><td colspan="1" rowspan="1">Z323</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium, vanA (+)</td><td colspan="1" rowspan="1">Z270</td><td colspan="1" rowspan="1">PHE NCTC 12202- vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium, vanA (+)</td><td colspan="1" rowspan="1">Z271</td><td colspan="1" rowspan="1">PHE NCTC 12204- vanA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. faecium, vanA (+)</td><td colspan="1" rowspan="1">Z260</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">3/3</td></tr></table>

Note: The iC-GPC software logic only allows reporting of vanA $^ { \prime } B$ targets with Enterococcus faecium and Enterococcus faecalis. Therefore the vanA target will not be reported for positive specimens containing S. aureus strains that carry this marker (i.e., VISA/VRSA strains).

# g. Analytical specificity (Exclusivity)

The analytical specificity of the iC-GPC Assay was evaluated by testing a comprehensive panel of non-targeted microorganisms that may be encountered in positive blood cultures. Exclusivity panel members included organisms phylogenetically related to iC-GPC target organisms, organisms commonly recovered from positive blood cultures as well as common blood culture contaminants. A total of 94 exclusivity organisms were tested. Potential cross-reactivity was evaluated by testing panel organisms at high concentrations in blood culture bottle media with the appropriate volume of human blood added according to bottle instructions. In the event of a false positive result, additional replicates were evaluated. Samples excluded from the sample set for data analyses were 17 samples with positive controls check failures $( 4 . 4 \% )$ and 3 samples with system errors $( 0 . 8 \% )$ .

Due to false positive $E .$ . faecalis results observed with samples containing Streptococcus bovis (strain Z167), a second strain of Streptococcus bovis (ATCC 9145) was also tested. In a total of 14 sample replicates containing Streptococcus bovis, 11 samples generated false positive results for the $E$ . faecalis target indicating cross reactivity of this organisms with the iC‐ GPC Assay target. The package insert includes a limitation indicating cross-reactivity of Streptococcus bovis with the Enterococcus faecalis target of the iC-GPC Assay.

Exclusivity results are presented in Table 10 below; performance is based on the observation of expected negative results.

Table 10: Exclusivity Results   

<table><tr><td colspan="1" rowspan="1">Exclusivity Organism</td><td colspan="1" rowspan="1">Test Concentration(CFU/mL, or as designated,bottle ring or TCID50/mL)</td><td colspan="1" rowspan="1">Exclusivity(observednegative results)</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">3.09 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">6.55 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">5.70 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">5.05 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">3.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">1.01 × 1010</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">3.55 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">8.10 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">7.30 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">4.20 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">2.55 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">2.24 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">8.00 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida catenulata</td><td colspan="1" rowspan="1">1.14 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">7.05 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1.15 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">1.03 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">2.82 × 107</td><td colspan="1" rowspan="1">5/61</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">3.15 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">7.65 × 107</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">2.35 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">6.40 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">3.29 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">1.70 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (NAP-1 toxigenic)</td><td colspan="1" rowspan="1">2.44 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">2.97× 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium oedematiens (novyi)</td><td colspan="1" rowspan="1">5.70 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">5.95 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">3.79 × 108</td><td colspan="1" rowspan="1">5/62</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">2.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">4.19 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus</td><td colspan="1" rowspan="1">1 × 106.34 (TCID50)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">1.08 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1 × 105.07 (TCID50)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">1 × 107.77 (TCID50)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">5.05 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">1.42 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">6.50 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2.72 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">5.85 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">9.60 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">3.80 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">6.10 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">3.20 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">1.04 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">4.67 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">1 × 106.10 (TCID50)</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.27 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">3.09 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">1.25 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">7.25 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2.27× 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">3.45 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kytococcus schroeteri</td><td colspan="1" rowspan="1">1.85 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">5.30 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum subsp.plantarum</td><td colspan="1" rowspan="1">1.53 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">1.24 × 108</td><td colspan="1" rowspan="1">5/63</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">2.57 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">1.22 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroides</td><td colspan="1" rowspan="1">3.17 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">1.13 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Oerskovia enterophila</td><td colspan="1" rowspan="1">1.33 × 1010</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">2.82 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Planococcus citreus</td><td colspan="1" rowspan="1">1.14 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">9.75 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">9.25 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">1.11 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.13 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">1.62 × 109</td><td colspan="1" rowspan="1">10/114</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">1.05 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">1.48 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">2.14 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">4.95 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rothia mucilaginosus</td><td colspan="1" rowspan="1">1.60 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp (typhimurium)</td><td colspan="1" rowspan="1">4.45 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">4.05 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">2.12 × 10</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">1.95 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">1.87 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">8.55 × 107</td><td colspan="1" rowspan="1">5/65</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">3.05 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">6.00 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">1.36 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">2.67 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi subsp. coagulans</td><td colspan="1" rowspan="1">2.67 × 108</td><td colspan="1" rowspan="1">5/6</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">6.48× 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">6.05 × 108</td><td colspan="1" rowspan="1">13/147</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">4.65 × 108</td><td colspan="1" rowspan="1">13/148</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">5.50 × 108</td><td colspan="1" rowspan="1">3/149</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">8.35 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">Bottle ring + 8 hours</td><td colspan="1" rowspan="1">11/1210</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Bottle ring + 8 hours</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td><td colspan="1" rowspan="1">Bottle ring + 8 hours</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">8.20 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">8.85 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">7.45 × 107</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1. Candida krusei: 1/3 false positive $E .$ . faecalis in initial testing, 3/3 repeats negative.   
2. Corynebacterium amycolatum: 1/3 false positive $S .$ . epidermidis in initial testing, 3/3 repeats negative.   
3. Lactobacillus reuteri: 1/3 false positive mecA in initial testing, 3/3 repeats negative. Note that mecA would not be reported as the S. aureus and $S _ { \cdot }$ . epidermidis targets were negative as expected.   
4. Providencia alcalifaciens: 1/3 false positive S. aureus in initial testing, 8/8 repeats negative.   
5. Staphylococcus intermedius: 1/3 false positive mecA in initial testing, 3/3 repeats negative. Note that mecA would not be reported as the $S _ { \cdot }$ . aureus and S. epidermidis targets were negative as expected.   
6. Staphylococcus schleiferi subsp. coagulans: 1/3 false positive mecA in initial testing, 3/3 repeats negative. Note that mecA would not be reported as the S. aureus and S. epidermidis targets were negative as expected.   
7. Streptococcus agalactiae: 1/3 false positive mecA in initial testing, 11/11 repeats negative. Note that mecA would not be reported as the $S _ { \cdot }$ . aureus and S. epidermidis targets were negative as expected.   
8. Streptococcus anginosus: 1/2 false positive $E .$ . faecalis in initial testing, 12/12 repeats negative.   
9. Streptococcus bovis: 1/2 false positive $E .$ . faecalis in initial testing, 10/12 false positive $E$ . faecalis in repeat testing   
10. Streptococcus intermedius: 1/3 false positive $E .$ . faecalis in initial testing, 9/9 repeats negative

# h. Microbial interference

A study was conducted to evaluate the performance of the iC-GPC Assay for samples containing mixtures of targeted and non-targeted organisms. Samples were prepared in a blood culture media/blood matrix with target organisms near the assay LoD mixed with non-targeted organisms at high concentrations ranging from $1 \dot { 0 } ^ { 6 } \ – 1 0 ^ { 9 }$ $\mathrm { C F U / m L }$ or at concentrations present at eight hours post bottle ring. A total of 85 different non-targeted microorganisms were evaluated in combination with each of five targeted organism strains. Testing was performed in triplicate.

For those organism combinations generating initial false negative results, additional samples were tested in replicates of 10 with the target organism near target LoD. If additional false negative results were observed, testing was repeated with the target organism at bottle ring concentrations. For those combinations with unexpected initial false negative results, additional replicate testing at the same concentration or at a higher bottle ring concentration generated $100 \%$ positive results with the iC-GPC Assay. Samples excluded from the sample set for data analysis included 22 samples with positive controls check failures $( 1 . 4 \% )$ and 10 samples with system errors $( 0 . 7 \% )$ . Microbial interference study results are presented in Table 11 below; performance is based on all expected targets detected.

Table 11: Microbial Interference Study Results   

<table><tr><td colspan="1" rowspan="3">Exclusivity Organism</td><td colspan="1" rowspan="3">TestConcentration(F/mL)</td><td colspan="1" rowspan="1">SE</td><td colspan="1" rowspan="1">SA</td><td colspan="1" rowspan="1">SPN</td><td colspan="1" rowspan="1">EFLS</td><td colspan="1" rowspan="1">EFCM</td></tr><tr><td colspan="5" rowspan="1">Test Concentration for initial three replicates (CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">1.4 × 107</td><td colspan="1" rowspan="1">1.4 × 107</td><td colspan="1" rowspan="1">BottleRing</td><td colspan="1" rowspan="1">1.2 × 107</td><td colspan="1" rowspan="1">1.2 × 107</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">1.54× 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">3.28 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">2.85 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">2.53 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/131</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.75 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">1.78 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">3.65 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">2.10 × 10</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">1.28 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">18/22</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1.12 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">4.00 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida catenulate</td><td colspan="1" rowspan="1">5.68 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">3.53 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">5.75 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">5.15 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1.41 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">20/223</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1.58 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">3.83 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">1.18 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/64</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">3.20 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1.64 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">8.50 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (NAP-1toxigenic)</td><td colspan="1" rowspan="1">1.22 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/65</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">1.48 × 107</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium oedematiens(novyi)</td><td colspan="1" rowspan="1">2.85 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">2.98 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">1.89 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.25 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">2.09 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">2.53 × 10</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">3.25 × 10</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.36 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">2.93 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">4.80 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">1.90 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">3.05 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">Exclusivity Organism</td><td colspan="1" rowspan="3">TestConcentration(CFU/mL)</td><td colspan="1" rowspan="1">SE</td><td colspan="1" rowspan="1">SA</td><td colspan="1" rowspan="1">SPN</td><td colspan="1" rowspan="1">EFLS</td><td colspan="1" rowspan="1">EFCM</td></tr><tr><td colspan="5" rowspan="1">Test Concentration for initial three replicates (CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">1.4 × 107</td><td colspan="1" rowspan="1">1.4 × 107</td><td colspan="1" rowspan="1">BottleRing</td><td colspan="1" rowspan="1">1.2 × 107</td><td colspan="1" rowspan="1">1.2 × 107</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">1.60 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">5.18 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">2.33 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6.35 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">1.54 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/127</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">6.23 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3.63 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.13 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">1.73 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/138</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kytococcus schroeteri</td><td colspan="1" rowspan="1">9.25 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">2.65 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarumsubsp.plantarum</td><td colspan="1" rowspan="1">7.63 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">6.18 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.28 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">6.08 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroides</td><td colspan="1" rowspan="1">1.58 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">5.63 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/129</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Oerskovia enterophila</td><td colspan="1" rowspan="1">6.63 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">1.41 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">4.88 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">4.63 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/1310</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/1211</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">5.55 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/1312</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">5.63 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">8.08 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">5.25× 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">7.40 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/613</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.07× 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">2.48 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rothia mucilaginosus</td><td colspan="1" rowspan="1">8.00 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp (typhimurium)</td><td colspan="1" rowspan="1">2.23 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">2.03 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">21/2314</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">1.06 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/1315</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">9.75 × 106</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/1216</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">9.33 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">4.28 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">1.53 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">3.00 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">6.78 × 108</td><td colspan="1" rowspan="1">12/1317</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">1.33 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi subsp.coagulans</td><td colspan="1" rowspan="1">1.22 × 109</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">3.20 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">3.03 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">2.33 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/1318</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">2.75 × 108</td><td colspan="1" rowspan="1">5/619</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">4.18 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">8.25 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Bottle ring + 8hours</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspseudopneumoniae</td><td colspan="1" rowspan="1">Bottle ring + 8hours</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">4.10 × 108</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">4.43 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/1320</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">3.73 × 107</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1. 1 false negative vanB in initial testing (2/3), 10/10 repeats passed   
2. 2 false negative S. aureus and 1 false negative mecA in initial testing (1/3), 2 false negative $S .$ . aureus and 2 false negative   
mecA in repeat testing near LoD (8/10). No false negatives in repeat testing at bottle ring concentration (9/9).   
3. 1 false negative vanB in initial testing (2/3), 1 false negative vanB in repeat testing near LoD (9/10). No false negatives in   
repeat testing at bottle ring concentration (9/9).   
4. 1 false positive mecA in initial testing (2/3), 3/3 repeats passed. Note that mecA would not be reported as the $S$ aureus and $S _ { \cdot }$ .   
epidermidis targets were negative as expected.   
5. 1 false positive S. epidermidis in initial testing (2/3), 3/3 repeats passed   
6. 1 false positive S. aureus in initial testing (2/3), 3/3 repeats passed   
7. 1 false negative vanB in initial testing (2/3), 9/9 repeats passed   
8. 1 false negative S. aureus and 1 false negative mecA in initial testing (2/3), 10/10 repeats passed   
9. 1 false negative S. aureus and 1 false negative mecA in initial testing (2/3), 9/9 repeats passed   
10. 1 false negative S. aureus in initial testing (2/3), 10/10 repeats passed   
11. 1 false negative vanB in initial testing (2/3), 9/9 repeats passed   
12. 2 false negative mecA in initial testing (1/3), 10/10 repeats passed   
13. 1 false positive S. epidermidis in initial testing (2/3), 3/3 repeats passed   
14. 1 false negative mecA in initial testing (2/3), 1 false negative S. aureus and 1 false negative mecA in repeat testing near LoD   
(9/10). No false negatives in repeat testing at bottle ring concentration (9/9).   
15. 1 false negative S. aureus and 1 false negative mecA in initial testing (2/3), 10/10 repeats passed   
16. 1 false negative S. pneumoniae in initial testing (1/2), 10/10 repeats passed at bottle ring concentration.   
17. 1 false negative mecA in initial testing (2/3), 10/10 repeats passed   
18. 1 false negative E. faecium in initial testing (2/3), 10/10 repeats passed   
19. 1 false positive E. faecalis in initial testing (2/3), 3/3 repeats passed   
20. 1 false negative S. aureus in initial testing (2/3), 10/10 repeats passed

# i. Competitive inhibition

Performance of the iC-GPC Assay was evaluated with combinations of target analytes that may be found in mixed positive blood cultures. One target organism was tested at a concentration near LoD (“low organism”) in combination with a second target organism at a concentration of bottle ring $+ 8$ hours (“high organism”).

Combinations of five organisms representing each of the iC-GPC Assay targets were tested, for a total of 20 combinations. Competitive inhibition results are summarized in Table 12 below. Results represent the number of detected targeted organisms out of the total number of replicates tested. No positive control failures or other system errors were observed in the study.

Due to competitive inhibition, low concentration organism targets were detected in only 3/20 $( 1 5 \% )$ of organism combinations. All high concentration iC-GPC targeted organisms were detected in all organism combinations. A limitation was placed in the package insert indicating the following:

Due to competitive inhibition, target organisms present near LoD concentrations may not be detected by the iC-GPC Assay when a second target organism is present at higher concentrations.

Table 12: Competitive Inhibition Study Results   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Target Performance</td></tr><tr><td rowspan=1 colspan=1>LoD Strain</td><td rowspan=1 colspan=1>High Strain</td><td rowspan=1 colspan=1>Low Organism(~2-3x LoD)</td><td rowspan=1 colspan=1>Low ResistanceMarker(~2-3x LoD)</td><td rowspan=1 colspan=1>High Organism(Bottle ring+ 8 hours)</td><td rowspan=1 colspan=1>High ResistanceMarker (Bottlering + 8 hours)</td></tr><tr><td rowspan=4 colspan=1>S. epidermidis(SE/mecA)</td><td rowspan=1 colspan=1>SA/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3*</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3*</td></tr><tr><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>EFLS/vanB</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>EFCM/vanA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>S. aureus(SA/mecA)</td><td rowspan=1 colspan=1>SE/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3*</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3*</td></tr><tr><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>EFLS/vanB</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>EFCM/vanA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>S. pneumoniae(SPN)</td><td rowspan=1 colspan=1>SE/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>SA/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>EFLS/vanB</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>EFCM/vanA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>E. faecalis(EFLS/vanB)</td><td rowspan=1 colspan=1>SE/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>SA/mecA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>EFCM/vanA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>E. faecium(EFCM/vanA)</td><td rowspan=1 colspan=1>SE/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>SA/mecA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>EFLS/vanB</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>13/60 (21.7%)</td><td rowspan=1 colspan=1>25/48 (52.1%)</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>48/48 (100%)</td></tr></table>

\*The source of the resistance marker cannot be distinguished between high concentration organism and low concentration organism.

# $j$ . Blood culture bottle equivalency

Commonly used blood culture bottle media types were evaluated to demonstrate that variability in media composition does not interfere with iC-GPC Assay performance. A total of 19 representative strains of iC-GPC targeted organisms plus one non-target organism were tested in 10 bottled blood culture media types. Target organisms were tested near LoD concentrations ( $2 { - } 3 \mathbf { x }$ LoD). Expected performance for targeted organisms was based on all expected targets identified and no false positive targets detected. Expected performance for non-target organisms was based on expected negative results. A total of 10 runs with positive controls check failures $( 1 . 2 3 \% )$ and five system errors $( 0 . 6 2 \% )$ were observed in the study and excluded from the data analysis.

Performance of the iC-GPC Assay for all BACTEC and VersaTREK bottle types and for BacT/ALERT SA Standard Aerobic bottles met the acceptance criteria $6 \% \geq 9 5 \%$ accurate results, demonstrating acceptable analytical performance for these bottle types with the iC-GPC Assay (see table 13 below).

False positive results for Enterococcus faecium were observed in 15/59 $( 2 5 . 4 \% )$ of BacT/ALERT FA Aerobic FAN and 13/60 $( 2 1 . 7 \% )$ of BacT/ALERT FA Plus Aerobic bottles for specimens prepared with target organisms present at $2 { - } 3 \mathbf { x }$ LoD. Additional testing for both bottle types was performed with higher organism concentrations of 5-10x LoD for each targeted organism. This additional testing resulted in false positive $E .$ . faecium results for 8/59 $( 1 3 . 6 \% )$ and 5/60 $( 8 . 3 \% )$ of BacT/ALERT FA Aerobic FAN and BacT/ALERT FA Plus Aerobic bottles respectively.

The potential cause of false positive $E .$ . faecium results was further investigated using multiple lots of uninoculated BacT/Alert FA Aerobic FAN and FA Plus Aerobic bottles. Results from this testing showed that false positive results continued to occur for these two bottle types with the iC-GPC Assay. Further testing of inoculated BacT/Alert FA Aerobic FAN and FA Plus Aerobic blood culture bottles was performed using a PCR assay that detects a different $E .$ . faecium DNAs sequence. This testing generated positive PCR results that were subsequently confirmed by sequencing and therefore confirming that the uninoculated bottle media contained $E .$ . faecium nucleic acids. Results from the investigation indicate that the iC-GPC Assay false positive $E$ . faecium results observed in BacT/Alert FA Aerobic FAN and BacT/Alert FA Plus Aerobic media in the analytical bottle equivalency study are not likely due to a limitation of the iC-GPC Assay but rather are due to the presence of remnant nucleic acids and/or non-viable $E .$ . faecium organisms in the culture media.

Although analytical testing showed significant numbers of false positive $E$ . faecium results for samples prepared in BacT/Alert FA Aerobic FAN and BacT/Alert FA Plus blood culture media, there were no false positive $E .$ . faecium results observed in clinical positive blood culture specimens for these two bottle types (i.e., no false positive $E .$ . faecium results observed in 96 BacT/Alert FA Aerobic FAN and for 18 BacT/Alert FA Plus blood culture bottles).

Because of the observed false positive results observed in analytical testing, the result interpretation section as well as the limitations section of the package insert indicate that all positive $E .$ . faecium results should be confirmed using alternative methods when testing BacT/ALERT FA Aerobic FAN or FA Plus Aerobic blood culture bottles.

The following limitations are included in the package insert regarding testing of BacT/Alert blood culture bottles with the iC-GPC Assay:

There is an increased risk of $E .$ . faecium false positives when the iC-GPC Assay is used with bioMérieux BacT/ALERT FA Aerobic FAN and BacT/ALERT FA Plus Aerobic blood culture bottles due to remnant nucleic acids or non-viable organisms that have been reported to be present in the culture media. When using these bottle types, positive $E .$ . faecium test results should be confirmed using alternative methods.

The iC-GPC Assay has not been validated for use with bioMérieux BacT/ALERT SN Standard Anaerobic, FN Anaerobic FAN, and FN Plus Anaerobic blood culture bottles due to an increased risk of $E$ . faecium false positives from nucleic acids or non-viable organisms present in the culture media. Anaerobic BacT/ALERT bottles including SN Standard Anaerobic, FN Anaerobic FAN, and FN Plus Anaerobic should not be used with the iC-GPC Assay.

Study results from the blood culture equivalency study are presented in Table 13.

Table 13: iC-GPC Assay Blood Culture Equivalency Study Results   

<table><tr><td colspan="1" rowspan="1">Blood Culture Bottle Type</td><td colspan="1" rowspan="1">OverallPerformance (%)</td><td colspan="1" rowspan="1">TargetPerformance(%)</td><td colspan="1" rowspan="1">FalseNegatives (%)</td><td colspan="1" rowspan="1">Non-TargeterPeformance(%)</td><td colspan="1" rowspan="1">False Positives(%)</td></tr><tr><td colspan="1" rowspan="1">BACTECTM Plus Aerobic</td><td colspan="1" rowspan="1">69/70 (98.6%)</td><td colspan="1" rowspan="1">66/67 (98.5%)</td><td colspan="1" rowspan="1">1/67 (1.5%)1</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">0/70 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Plus Anaerobic</td><td colspan="1" rowspan="1">70/72 (97.2%)</td><td colspan="1" rowspan="1">67/69 (97.1%)</td><td colspan="1" rowspan="1">1/69 (1.4%)2</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">1/72 (1.4%)3</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Aerobic</td><td colspan="1" rowspan="1">62/63 (98.4%)</td><td colspan="1" rowspan="1">59/60 (98.3%)</td><td colspan="1" rowspan="1">1/60 (1.7%)4</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">0/63 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC StandardAnaerobic</td><td colspan="1" rowspan="1">77/80 (96.3%)</td><td colspan="1" rowspan="1">74/77 (96.1%)</td><td colspan="1" rowspan="1">2/77 (2.6%)5</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">1/80 (1.3%)6</td></tr><tr><td colspan="1" rowspan="1">BACTEC Lytic/10Anaerobic</td><td colspan="1" rowspan="1">78/81 (96.3%)</td><td colspan="1" rowspan="1">75/78 (96.2%)</td><td colspan="1" rowspan="1">2/78 (2.6%)7</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">1/81 (1.2%)8</td></tr><tr><td colspan="1" rowspan="1">VersaTREK REDOX 1</td><td colspan="1" rowspan="1">60/60 (100.0%)</td><td colspan="1" rowspan="1">57/57 (100.0%)</td><td colspan="1" rowspan="1">0/57 (0.0%)</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">0/60 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">VersaTREK REDOX 2</td><td colspan="1" rowspan="1">59/59 (100.0%)</td><td colspan="1" rowspan="1">56/56 (100.0%)</td><td colspan="1" rowspan="1">0/56 (0.0%)</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">0/59 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT SA StandardAerobic</td><td colspan="1" rowspan="1">60/60 (100.0%)</td><td colspan="1" rowspan="1">57/57 (100.0%)</td><td colspan="1" rowspan="1">0/57 (0.0%)</td><td colspan="1" rowspan="1">3/3 (100.0%)</td><td colspan="1" rowspan="1">0/60 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FA AerobicFAN (2-3x LoD)</td><td colspan="1" rowspan="1">44/59 (74.6%)</td><td colspan="1" rowspan="1">44/56 (78.6%)</td><td colspan="1" rowspan="1">0/56 (0.0%)</td><td colspan="1" rowspan="1">0/3 (0.0%)</td><td colspan="1" rowspan="1">15/59 (25.4%)9</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FA Aerobic</td><td colspan="1" rowspan="1">51/59 (86.4%)</td><td colspan="1" rowspan="1">51/57 (89.5%)</td><td colspan="1" rowspan="1">0/57 (0.0%)</td><td colspan="1" rowspan="1">0/2 (0.0%)</td><td colspan="1" rowspan="1">8/59 (13.6%)10</td></tr><tr><td colspan="1" rowspan="1">Blood Culture Bottle Type</td><td colspan="1" rowspan="1">OverallPerformance (%)</td><td colspan="1" rowspan="1">TargetPerformance(%)</td><td colspan="1" rowspan="1">FalseNegatives (%)</td><td colspan="1" rowspan="1">Non-TargetPerformance(%)</td><td colspan="1" rowspan="1">False Positives(%)</td></tr><tr><td colspan="1" rowspan="1">FAN (5-10x LoD)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FA PlusAerobic (2-3x LoD)</td><td colspan="1" rowspan="1">47/60 (78.3%)</td><td colspan="1" rowspan="1">46/57 (80.7%)</td><td colspan="1" rowspan="1">0/57 (0.0%)</td><td colspan="1" rowspan="1">1/3 (33.3%)</td><td colspan="1" rowspan="1">13/60 (21.7%)11</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FA PlusAerobic (5-10x LoD)</td><td colspan="1" rowspan="1">55/60 (91.7%)</td><td colspan="1" rowspan="1">54/57 (94.7%)</td><td colspan="1" rowspan="1">0/57 (0.0%)</td><td colspan="1" rowspan="1">1/3 (33.3%)</td><td colspan="1" rowspan="1">5/60 (8.3%)12</td></tr></table>

1. 1 false negative mecA   
2. 1 false negative S. epidermidis   
3. 1 false positive E. faecalis   
4. 1 false negative S. epidermidis   
5. 1 false negative S. epidermidis, 1 false negative vanA   
6. 1 false positive S. epidermidis   
7. 1 false negative S. epidermidis, 1 false negative S. pneumoniae   
8. 1 false positive $E .$ . faecalis/vanB   
9. 15 false positive $E$ . faecium   
10. 8 false positive E. faecium   
11. 12 false positive $E$ . faecium, 1 false positive S. aureus

# k. Interfering substances

iC-GPC Assay performance was evaluated in the presence of potentially inhibiting substances that may be present in blood and blood culture media. Samples containing five representative target organisms plus one gram positive, non-target organism (Group B Streptococcus) were evaluated. Samples were prepared with targeted organisms at near LoD concentrations with each potential interferent at “worst-case” concentrations that exceed the potentially highest concentration encountered in blood culture bottles inoculated with indicated blood volumes. Antibiotics were tested at five times the concentration expected in whole blood.

Target organism performance was based on all expected targets detected, while nontarget organism performance was based on expected negative results. In the event of a false negative result, the organism/interferent was retested in replicates of ten. In the event of a false positive result or other system failure, the organism/interferent was retested in replicates of three. For any discordant results observed both initially and with additional replicates, additional samples were evaluated with decreasing inhibitor concentrations until interference was no longer observed.

Interference testing was performed in BD BACTEC Plus Aerobic blood culture bottle media with a sodium polyanetholesulfonate (SPS) concentration of $0 . 0 5 \% \mathrm { w } / \mathrm { v }$ . Additional SPS at concentrations greater than $0 . 0 3 \% \mathrm { w } / \mathrm { v }$ was found to interfere with iC-GPC Assay performance. Total protein (gamma-globulin and albumin) in concentrations greater than ${ 3 \mathrm { g / d L } }$ was also found to interfere with iC-GPC Assay performance. The presence of these substances in positive blood culture specimens may result in false negative results or positive controls check failures.

A total of 10 positive controls check failures $( 2 . 6 9 \% )$ and three system errors $( 0 . 8 1 \% )$ were observed during the study and results from these samples were excluded from the data analysis. Table 14 includes results for those substances that did not demonstrate interference initially or after testing of additional replicates. Table 15 includes results for those substances that demonstrated interference.

Table 14: Interference Study Results – Non-interfering Substances   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Target Performance</td></tr><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Test</td><td rowspan=1 colspan=1>SE</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>EFLS</td><td rowspan=1 colspan=1>EFCM</td><td rowspan=1 colspan=1>Group B</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>14 g/L</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/61</td></tr><tr><td rowspan=1 colspan=1>Triglyceride</td><td rowspan=1 colspan=1>1000 mg/dL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Conjugated bilirubin</td><td rowspan=1 colspan=1>20 mg/dL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>12/132</td><td rowspan=1 colspan=1>12/133</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Unconjugated bilirubin</td><td rowspan=1 colspan=1>20 mg/dL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Human genomic DNA</td><td rowspan=1 colspan=1>1 x 106 cells/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>12/134</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>100 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>12/135</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Piperacillin</td><td rowspan=1 colspan=1>160 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Meropenem</td><td rowspan=1 colspan=1>80 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Cefepime</td><td rowspan=1 colspan=1>80 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>12/136</td><td rowspan=1 colspan=1>11/127</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>80 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>125 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Gentamicin</td><td rowspan=1 colspan=1>100 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>11/128</td><td rowspan=1 colspan=1>12/149</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

1. 1 false positive $E .$ . faecalis, 3/3 repeats passed 2. 1 false negative S. pneumoniae, 10/10 repeats passed 3. 1 false negative vanB, 10/10 repeats passed 4. 1 false negative vanB, 10/10 repeats passed 5. 1 false negative S. pneumoniae, 10/10 repeats passed 6. 1 false negative mecA, 10/10 repeats passed 7. 1 false negative S. pneumoniae, 9/9 repeats passed 8. 1 false negative S. pneumoniae, 9/9 repeats passed 9. 1 false negative vanB, 1 false positive S. epidermidis, 11/11 repeats passed

Table 15: Interfering Substance Study – Interfering Substances   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Target Performance</td></tr><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Test</td><td rowspan=1 colspan=1>SE</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>SPN</td><td rowspan=1 colspan=1>EFLS</td><td rowspan=1 colspan=1>EFCM</td><td rowspan=1 colspan=1>Group B</td></tr><tr><td rowspan=1 colspan=1>Total Protein(γ-globulin + albumin)</td><td rowspan=1 colspan=1>3 g/dL</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/31</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Total Protein(γ-globulin + albumin)</td><td rowspan=1 colspan=1>6g/dL*</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=1 colspan=1>SPS</td><td rowspan=1 colspan=1>0.03% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>12/132</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/63</td></tr><tr><td rowspan=1 colspan=1>SPS</td><td rowspan=1 colspan=1>0.04% w/v*</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

1. 1 false negative vanB, test not repeated 2. 1 false negative S. aureus/mecA, 10/10 repeats passed 3. 1 false positive mecA, 3/3 repeats passed

# l. Carryover/cross-contamination

The iC-System is a closed system, designed to reduce the potential for carryover or cross-contamination between samples. A study was conducted to evaluate the potential for false positive results from carryover or cross-contamination when high positive samples are tested consecutively with negative samples. Two representative iC-GPC target organisms, Staphylococcus aureus ATCC BAA977 (nuc positive, mecA negative) and Enterococcus faecium ATCC 700221 (fcm positive, vanA positive), were tested as representative high positive samples. A gram positive nontarget organism, Corynebacterium striatum, was used as the negative sample. All organisms were tested at high concentrations, eight hours beyond initial bottle positivity. Positive, target organism samples were tested in an alternating pattern with negative, non-target organism samples across eight blades of an iC-Processor. Performance was based on accurate iC-GPC Assay target detection. No false positive results were observed.

# m. Assay cut-off:

The iC-GPC Assay identifies and reports detected analytes based on the relative intensity levels of fluorescent markers (amplified DNA bound by gene specific fluorescently labeled probes) hybridized to a glass microarray within the iC-Cassette. The assay software analyzes data captured from image processing and compares fluorescent signals from analyte-specific sensors in the sensor array to background fluorescence.

# 2. Clinical studies:

# a. Method comparison with FDA-cleared multiplex assay:

A method comparison study was performed at four geographically dispersed clinical sites to evaluate the performance of the iC-GPC Assay for use on the iC-System. Sites tested 966 leftover, de-identified blood culture specimens from aerobic and anaerobic blood culture bottles flagged as positive by their respective continuous monitoring blood culture system. Three of the commonly used blood culture systems were included in the study: BD BACTEC, bioMérieux BacT/ALERT, and Thermo Fisher VersaTREK. The following blood culture bottle types were evaluated: BD BACTEC Standard Aerobic/Anaerobic, VersaTREK REDOX 1/2, BacT/ALERT SA Standard Aerobic, and BacT/ALERT FA Aerobic FAN.

Blood culture specimens flagged as positive by the instrument were confirmed using Gram stain to contain gram positive cocci prior to being entered into the study. Specimens demonstrating mixed Gram stain results were excluded as indicated by the assay instructions for use.

Performance of the iC-GPC Assay was evaluated as compared to an FDA-cleared multiplex assay that detects gram positive organisms and associated resistance

markers directly from positive blood culture specimens. PCR and bi-directional sequencing were also performed for all specimens with positive results for resistance markers by either the iC-GPC Assay or an FDA-cleared multiplex assay. Results from this additional testing were used only for discordant analysis with results included as footnotes under the performance tables.

Additionally, performance of the iC-GPC Assay was compared to conventional reference methods including subculture, identification of isolates using biochemical methods or MALDI-TOF identification and phenotypic antimicrobial susceptibility testing (AST).

An additional 168 contrived specimens were tested for low prevalence organisms and resistance markers. Contrived specimens were prepared using a variety of clinical strains and type strains of S. pneumoniae, $E .$ . faecalis, and $E$ . faecium. Confirmation of strain identities was confirmed by sequencing for all clinical isolates as well as isolates containing vanA/vanB resistance markers. Contrived specimens were prepared in BD BACTEC Plus Aerobic blood culture bottles with the appropriate volume of blood added in accordance with bottle instructions. Organisms were spiked into bottles at low concentrations and incubated until flagged as positive by the instrument. Aliquots of specimens were frozen and provided to the three clinical sites for testing.

Of the 966 positive blood culture specimens evaluated in the study, a total of 53 specimens were excluded from the study results. Of the remaining 913 specimens remaining, 879 were fresh prospective specimens and 34 were frozen prospective specimens.

The total specimens excluded from the iC-GPC Assay Method Comparison Study $\left( \mathrm { n } { = } 5 3 \right)$ are listed by site and reason for exclusion in Table 16. The most common reasons for exclusion included repeat iC-GPC errors, repeat comparator assay errors, or exclusion of specimens with mixed Gram stain results.

Table 16: Withdrawn Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Reason for Withdrawal</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Mixed GramStain</td><td rowspan=1 colspan=1>RepeatiC-GPC Error</td><td rowspan=1 colspan=1>RepeatBC-GP Error</td><td rowspan=1 colspan=1>No BC-GP ID/Other</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>53</td></tr></table>

When performance of the iC-GPC Assay was compared to the FDA-cleared multiplex assay, there was no apparent difference in performance noted between the four study sites or between the three blood culture systems. Performance for all positive bottle types/systems combined is presented in Tables 17-24 respectively for detection of the S. aureus, S. epidermidis, S. pneumoniae, $E .$ . faecalis, $E .$ . faecium, mecA, vanA and vanB iC-GPC Assay targets as compared to the comparator assay. Results are

stratified by prospectively tested fresh specimens, prospectively collected/retrospectively tested frozen specimens and contrived specimens. Table 22 includes performance for detection of mecA with results linked to detection of S. aureus and S. epidermidis combined. Tables 23 and 24 include performance for detection of vanA and vanB respectively with results linked to detection of $E$ . faecalis and $E .$ . faecium combined.

Table 17: iC-GPC Assay Performance: Staphylococcus aureus (nuc)   

<table><tr><td rowspan=1 colspan=5>aureus (huc)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Grocd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>96.9%249/257(94.0-98.4)</td><td rowspan=1 colspan=1>99.7%620/622(98.8-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%9/9(70.1-100)</td><td rowspan=1 colspan=1>100%25/25(86.7-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>97.0%258/266*(94.2-98.5)</td><td rowspan=1 colspan=1>99.7%645/647**(98.9-99.9)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%168/168(97.8-100)</td></tr></table>

$^ { * } \mathrm { O f }$ the 8 observed false negative $S .$ . aureus results by the iC-GPC Assay, 7/8 were positive for S. aureus by PCR/bi-directional sequencing. $^ { * * } \mathrm { O f }$ the 2 observed false positive $S _ { \ast }$ . aureus results by the iC-GPC Assay, 1/2 was positive for $S _ { \cdot }$ . aureus by PCR/bi-directional sequencing. One sample was not available for discordant analysis.

<table><tr><td rowspan=1 colspan=5>Table 18: C-GPC Assay Performance: Staphylococcusepidermidis (gseA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=2>Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jroecd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>98.3%227/231(95.6-99.3)</td><td rowspan=1 colspan=1>98.0%635/648(96.6-98.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%4/4(51.0-100)</td><td rowspan=1 colspan=1>96.7%29/30(83.3-99.4)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>98.3%231/235*(95.7-99.3)</td><td rowspan=1 colspan=1>97.9%664/678**(96.6-98.8)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%168/168(97.8-100)</td></tr></table>

${ } ^ { * } \mathrm { O f }$ the 4 observed false negative S. epidermidis results by the iC-GPC Assay, 3/4 were positive for S. epidermidis by PCR/bi-directional sequencing. $^ { * * } \mathrm { O f }$ the 14 observed false positive S. epidermidis results by the iC-GPC Assay, 9/14 were positive for $S _ { \cdot }$ . epidermidis by PCR/bi-directional sequencing.

Table 19: iC-GPC Assay Performance: Streptococcus   

<table><tr><td rowspan=1 colspan=5>pneumoniae (lytA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jrocrd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>82.6%*19/23(62.9-93.0)</td><td rowspan=1 colspan=1>99.9%855/856(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%4/4(51.0-100)</td><td rowspan=1 colspan=1>100%30/30(88.6-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>85.2%23/27*(67.5-94.1)</td><td rowspan=1 colspan=1>99.9%885/886**(99.4-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>100%36/36(90.4-100)</td><td rowspan=1 colspan=1>100%132/132(97.2-100)</td></tr></table>

$^ { * } \mathrm { O f }$ the 4 observed false negative S. pneumoniae results by the iC-GPC Assay, 0/4 were positive for $S .$ . pneumoniae by PCR/bi-directional sequencing and 0/4 culture isolates obtained from these specimens were positive for $S _ { \cdot }$ . pneumoniae by MALDI ID. ${ \ast } { \ast } _ { \mathrm { O f } }$ the 1 observed false positive S. pneumoniae result by the iC-GPC Assay, $1 / 1$ was positive for $S .$ pneumoniae by PCR/bi-directional sequencing.

Table 20: iC-GPC Assay Performance: Enterococcus faecalis (ddl)   

<table><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jrod</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>96.6%56/58(88.3-99.0)</td><td rowspan=1 colspan=1>99.9%820/821(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%3/3(43.8-100)</td><td rowspan=1 colspan=1>100%31/31(89.0-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>96.7%59/61*(88.8-99.1)</td><td rowspan=1 colspan=1>99.9%851/852**(99.3-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>100%90/90(95.9-100)</td><td rowspan=1 colspan=1>100%78/78(95.3-100)</td><td rowspan=1 colspan=1></td></tr></table>

$^ { * } \mathrm { O f }$ the 2 observed false negative E. faecalis results by the iC-GPC Assay, 1/2 was positive for $E$ . faecalis by PCR/bi-directional sequencing. $^ { * * } \mathrm { O f }$ the 1 observed false positive $E .$ . faecalis result by the iC-GPC Assay, 0/1 was positive for $E .$ . faecalis by PCR/bi-directional sequencing

Table 21: iC-GPC Assay Performance: Enterococcus faecium   

<table><tr><td rowspan=1 colspan=5>(fcm)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jrocsed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>96.4%27/28(82.3-99.4)</td><td rowspan=1 colspan=1>99.8%849/851(99.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(20.7-100)</td><td rowspan=1 colspan=1>100%33/33 (89.6-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>96.6%28/29*(82.8-99.4)</td><td rowspan=1 colspan=1>99.8%882/884**(99.2-99.9)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>100%42/42(91.6-100)</td><td rowspan=1 colspan=1>100%126/126(97.0-100)</td></tr></table>

$^ { * } \mathrm { O f }$ the 1 observed false negative $E$ . faecium result by the iC-GPC Assay, 1/1 was positive for $E$ . faecium by PCR/bi-directional sequencing. $^ { * * } \mathrm { O f }$ the 2 observed false positive $E .$ . faecium results by the iC-GPC Assay, 1/2 were positive for $E$ . faecium by PCR/bi-directional sequencing.

<table><tr><td rowspan=1 colspan=5>Table 22: iC-GPC Assay Performance: mecA</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jrocd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>96.1%269/280(93.1-97.8)</td><td rowspan=1 colspan=1>98.2%588/599(96.7-99.0)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%5/5(56.6-100)</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>96.1%274/285*(93.2-97.8)</td><td rowspan=1 colspan=1>98.2%617/628**(96.9-99.0)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%168/168(97.8-100)</td></tr></table>

${ } ^ { * } \mathrm { O f }$ the 11 observed false negative mecA results by the iC-GPC Assay, 10/11 were positive for mecA by PCR/bi-directional sequencing. $\ast \ast _ { \mathrm { O f } }$ the 11 observed false positive mecA results by the iC-GPC Assay, 8/11 were positive for mecA by PCR/bi-directional sequencing.

Table 23: iC-GPC Assay Performance: vanA   

<table><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jrocsd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>94.7%18/19(75.4-99.1)</td><td rowspan=1 colspan=1>99.4%855/860(98.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%1/1(20.7-100)</td><td rowspan=1 colspan=1>100%33/33(89.6-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>95.0%19/20*(76.4-99.1)</td><td rowspan=1 colspan=1>99.4%888/893**(98.7-99.8)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>100%40/40(91.2-100)</td><td rowspan=1 colspan=1>100%128/128(97.1-100)</td></tr></table>

$^ { * } \mathrm { O f }$ the 1 observed false negative vanA result by the iC-GPC Assay, 1/1 was positive for vanA by PCR/bi-directional sequencing. $^ { * * } \mathrm { O f }$ the five observed false positive results by the iC-GPC Assay, 2/5 were positive were vanA by PCR/bi-directional sequencing.

<table><tr><td rowspan=1 colspan=5>Table 24: iC-GPC Assay Performance: vanB</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>FDA-ClearedMultiplexAssay</td></tr><tr><td rowspan=3 colspan=1>Jroecd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>879</td><td rowspan=1 colspan=1>100%2/2(34.2-100)</td><td rowspan=1 colspan=1>99.9%876/877(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%34/34 (89.8-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>100%2/2(34.2-100)</td><td rowspan=1 colspan=1>99.9%910/911*(99.4-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>100%56/56(93.6-100)</td><td rowspan=1 colspan=1>100%112/112(96.7-100)</td></tr></table>

$^ { * } \mathrm { O f }$ the 1 observed false positive vanB result by the iC-GPC Assay, 0/1 was positive for vanB by PCR/bi-directional sequencing

Tables 25-30 include performance of the iC-GPC Assay as compared to an FDA-cleared multiplex assay for each targeted resistant marker linked to detection of specific targeted organisms. Tables 25 and 26 include stratified performance for detection of mecA with S. aureus and S. epidermidis respectively. Tables 27-30 include stratified performance for vanA and vanB for detection with $E$ . faecalis and $E .$ . faecium. Performance tables include all prospectively collected specimens, both fresh and frozen combined.

<table><tr><td colspan="7" rowspan="1">Table 25: Detection of mecA with S. aureus, Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">Staphylococcusaureus/mecA</td><td colspan="5" rowspan="1">FDA-Cleared Comparator Assay</td></tr><tr><td colspan="2" rowspan="1">SA+/mecA+</td><td colspan="1" rowspan="1">SA+/mecA-</td><td colspan="1" rowspan="1">SA-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iC-GPC</td><td colspan="1" rowspan="1">SA+/mecA+</td><td colspan="2" rowspan="1">111</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">114</td></tr><tr><td colspan="1" rowspan="1">SA+/mecA-</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">144</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">146</td></tr><tr><td colspan="1" rowspan="1">SA-</td><td colspan="2" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">645</td><td colspan="1" rowspan="1">653</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">115</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">647</td><td colspan="1" rowspan="1">913</td></tr><tr><td colspan="5" rowspan="1">Staphylococcus aureus/mecA</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SA+/mecA+)</td><td colspan="2" rowspan="1">96.5% (111/115)</td><td colspan="1" rowspan="1">91.4%</td><td colspan="1" rowspan="1">98.6%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SA+/mecA-)</td><td colspan="2" rowspan="1">95.4% (144/151)</td><td colspan="1" rowspan="1">90.7%</td><td colspan="1" rowspan="1">97.7%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SA-)</td><td colspan="2" rowspan="1">99.7% (645/647)</td><td colspan="1" rowspan="1">98.9%</td><td colspan="1" rowspan="1">99.2%</td></tr><tr><td colspan="7" rowspan="1">Table 26: Detection of mecA with S. epidermidis, Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">Staphylococcusepidermidis/mecA</td><td colspan="5" rowspan="1">FDA-Cleared Comparator Assay</td></tr><tr><td colspan="2" rowspan="1">SE+/mecA+</td><td colspan="1" rowspan="1">SE+/mecA-</td><td colspan="1" rowspan="1">SE-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iC-GPC</td><td colspan="1" rowspan="1">SE+/mecA+</td><td colspan="2" rowspan="1">163</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">171</td></tr><tr><td colspan="1" rowspan="1">SE+/mecA-</td><td colspan="2" rowspan="1">4</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">74</td></tr><tr><td colspan="1" rowspan="1">SE-</td><td colspan="2" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">664</td><td colspan="1" rowspan="1">668</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">170</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">678</td><td colspan="1" rowspan="1">913</td></tr><tr><td colspan="5" rowspan="1">Staphylococcus epidermidis/mecA</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE+/mecA+)</td><td colspan="2" rowspan="1">95.9% (163/170)</td><td colspan="1" rowspan="1">91.8%</td><td colspan="1" rowspan="1">98.0%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE+/mecA-)</td><td colspan="2" rowspan="1">95.4% (62/65)</td><td colspan="1" rowspan="1">87.3%</td><td colspan="1" rowspan="1">98.4%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE-)</td><td colspan="2" rowspan="1">97.9% (664/678)</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">98.8%</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Table 27: Detection of vanA with E. faecalis, Prospective Specimens</td></tr><tr><td rowspan=2 colspan=2>Enterococcus faecalis/vanA</td><td rowspan=1 colspan=5>FDA-Cleared Comparator Assay</td></tr><tr><td rowspan=1 colspan=2>Efs+/vanA+</td><td rowspan=1 colspan=1>Efs+/vanA-</td><td rowspan=1 colspan=1>Efs-</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=4 colspan=1>iC-GPC</td><td rowspan=1 colspan=1>Efs+/vanA+</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Efs+/vanA-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Efs-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>851</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>852</td><td rowspan=1 colspan=1>913</td></tr><tr><td rowspan=1 colspan=5>Enterococcus faecalis/vanA</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efs+/vanA+)</td><td rowspan=1 colspan=2>(0/0)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efs+/vanA-)</td><td rowspan=1 colspan=2>93.4% (57/61)</td><td rowspan=1 colspan=1>84.3%</td><td rowspan=1 colspan=1>97.4%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efs-)</td><td rowspan=1 colspan=2>99.9% (851/852)</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>99.9%</td></tr></table>

<table><tr><td colspan="7" rowspan="1">Table 28: Detection of vanB with E. faecalis, Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">Enterococcus faecalis/vanB</td><td colspan="5" rowspan="1">FDA-Cleared Comparator Assay</td></tr><tr><td colspan="2" rowspan="1">Efs+/vanB+</td><td colspan="1" rowspan="1">Efs+/vanB-</td><td colspan="1" rowspan="1">Efs-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iC-GPC</td><td colspan="1" rowspan="1">Efs+/vanB+</td><td colspan="2" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Efs+/vanB-</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">57</td></tr><tr><td colspan="1" rowspan="1">Efs-</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">851</td><td colspan="1" rowspan="1">853</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">2</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">852</td><td colspan="1" rowspan="1">913</td></tr><tr><td colspan="5" rowspan="1">Enterococcus faecalis/vanB</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs+/vanB+)</td><td colspan="2" rowspan="1">100.0% (2/2)</td><td colspan="1" rowspan="1">34.2%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs+/vanB-)</td><td colspan="2" rowspan="1">96.6% (57/59)</td><td colspan="1" rowspan="1">88.5%</td><td colspan="1" rowspan="1">99.1%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs-)</td><td colspan="2" rowspan="1">99.9% (851/852)</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">99.9%</td></tr><tr><td colspan="7" rowspan="1">Table 29: Detection of vanA with E. faecium, Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">Enterococcus faecium/vanA</td><td colspan="5" rowspan="1">FDA-Cleared Comparator Assay</td></tr><tr><td colspan="2" rowspan="1">Efm+/vanA+</td><td colspan="1" rowspan="1">Efm+/vanA-</td><td colspan="1" rowspan="1">Efm-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iC-GPC</td><td colspan="1" rowspan="1">Efm+/vanA+</td><td colspan="2" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">Efm+/vanA-</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Efm-</td><td colspan="2" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">882</td><td colspan="1" rowspan="1">883</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">20</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">884</td><td colspan="1" rowspan="1">913</td></tr><tr><td colspan="5" rowspan="1">Enterococcus faecium/vanA</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efm+/vanA+)</td><td colspan="2" rowspan="1">95.0% (19/20)</td><td colspan="1" rowspan="1">76.4%</td><td colspan="1" rowspan="1">99.1%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efm+/vanA-)</td><td colspan="2" rowspan="1">77.8% (7/9)</td><td colspan="1" rowspan="1">45.3%</td><td colspan="1" rowspan="1">93.7%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efm-)</td><td colspan="2" rowspan="1">99.8% (882/884)</td><td colspan="1" rowspan="1">99.2%</td><td colspan="1" rowspan="1">99.9%</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Table 30: Detection of vanB with E. faecium: Prospective Specimens</td></tr><tr><td rowspan=2 colspan=2>Enterococcus faecium/vanB</td><td rowspan=1 colspan=5>FDA-Cleared Comparator Assay</td></tr><tr><td rowspan=1 colspan=2>Efm+/vanB+</td><td rowspan=1 colspan=1>Efm+/vanB-</td><td rowspan=1 colspan=1>Efm-</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=4 colspan=1>iC-GPC</td><td rowspan=1 colspan=1>Efm+/vanB+</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Efm+/vanB-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Efm-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>882</td><td rowspan=1 colspan=1>883</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>884</td><td rowspan=1 colspan=1>913</td></tr><tr><td rowspan=1 colspan=5>Enterococcus faecium/vanB</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm+/vanB+)</td><td rowspan=1 colspan=2>(0/0)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm+/vanB-)</td><td rowspan=1 colspan=2>96.6% (28/29)</td><td rowspan=1 colspan=1>82.8%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm-)</td><td rowspan=1 colspan=2>99.8% (882/884)</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>99.9%</td></tr></table>

Tables 31-33 include performance of the iC-GPC Assay for detection of vanA and vanB in contrived specimens prepared with $E$ . faecalis or $E$ . faecium strains that carry either vanA or vanB resistance markers

<table><tr><td rowspan=1 colspan=5>Table 31: Detection of vanA with E. faecalis:Contrived Specimens</td></tr><tr><td rowspan=2 colspan=2>vanA/E. faecalis</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=3 colspan=1>iCubateiC-GPC</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>152</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=3>vanA/E. faecalis</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Positive % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>80.6%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=2>Negative % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 32: Detection of vanA with E. faecium:Contrived Specimens</td></tr><tr><td rowspan=2 colspan=2>vanA/E. faecium</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=3 colspan=1>iCubateiC-GPC</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=3>vanA/E. faecium</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Positive % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.2%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=2>Negative % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 33: Detection of vanB with E. faecalis:Contrived Specimens</td></tr><tr><td rowspan=2 colspan=2>vanB/E. faecalis</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=3 colspan=1>iCubateiC-GPC</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=3>vanB/E. faecalis</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Positive % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.6%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=2>Negative % Agreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

# b. Performance of iC-GPC compared to traditional culture and susceptibility testing

Performance of the iC-GPC Assay was also evaluated compared to results from traditional laboratory reference methods that included subculture of positive bottles to agar media, identification of culture isolates using conventional biochemical or MALDI-TOF identification and antimicrobial susceptibility testing (AST). AST testing was comprised of cefoxitin disc testing for all Staphylococcus isolates (phenotypic reference method for mecA) and vancomycin Etests for all Enterococcus faecalis and Enterococcus faecium (phenotypic reference method for vanA/vanB). Performance of the iC-GPC Assay in comparison to culture and AST testing is presented in Tables 34-40 for each targeted analyte (S. aureus, S. epidermidis, S. pneumoniae, E. faecalis, E. faecium, mecA, vanA and vanB). Performance is stratified by prospectively tested fresh specimens and prospectively collected, retrospectively tested frozen specimens.

Table 34: iC-GPC Assay Performance: Staphylococcus aureus (nuc)   

<table><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;Biochemicals</td></tr><tr><td rowspan=3 colspan=1>Jrocd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>849</td><td rowspan=1 colspan=1>95.8%250/261(92.6-97.6)</td><td rowspan=1 colspan=1>99.8%587/588(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%9/9(70.1-100)</td><td rowspan=1 colspan=1>100%23/23(85.7-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>881</td><td rowspan=1 colspan=1>95.9%259/270(92.9-97.7)</td><td rowspan=1 colspan=1>99.8%610/611(99.1-100)</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 36: iC-GPC Assay Performance: Streptococcuspneumoniae (lytA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;Biochemicals</td></tr><tr><td rowspan=3 colspan=1>Jrocd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>856</td><td rowspan=1 colspan=1>100%18/18(82.4-100)</td><td rowspan=1 colspan=1>99.9%837/838(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>100%3/3(43.8-100)</td><td rowspan=1 colspan=1>96.7%29/30(83.3-99.4)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>889</td><td rowspan=1 colspan=1>100%21/21(84.5-100)</td><td rowspan=1 colspan=1>99.8%866/868(99.2-99.9)</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 35: iC-GPC Assay Performance: Staphylococcusepidermidis (gseA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;Biochemicals</td></tr><tr><td rowspan=3 colspan=1>Jrod</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>849</td><td rowspan=1 colspan=1>95.9%213/222(92.5-97.9)</td><td rowspan=1 colspan=1>96.5%605/627(94.7-97.7)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%3/3(43.8-100)</td><td rowspan=1 colspan=1>96.6%28/29(82.8-99.4)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>881</td><td rowspan=1 colspan=1>96.0%216/225(92.6-97.9)</td><td rowspan=1 colspan=1>96.5%633/656(94.8-97.7)</td></tr></table>

Table 37: iC-GPC Assay Performance: Enterococcus faecalis (ddl)   

<table><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=1 colspan=1>Negative(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;Biochemicals</td></tr><tr><td rowspan=3 colspan=1>Jroced</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>858</td><td rowspan=1 colspan=1>96.6%57/59(88.5-99.1)</td><td rowspan=1 colspan=1>99.9%798/799(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>100%3/3(43.8-100)</td><td rowspan=1 colspan=1>100%30/30(88.6-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>891</td><td rowspan=1 colspan=1>96.8%60/62(89.0-99.1)</td><td rowspan=1 colspan=1>99.9%828/829(99.3-100)</td></tr></table>

Table 38: iC-GPC Assay Performance: Enterococcus faecium (fcm)   

<table><tr><td rowspan=1 colspan=5>(fcm)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;Biochemicals</td></tr><tr><td rowspan=3 colspan=1>Jroecd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>858</td><td rowspan=1 colspan=1>96.4%27/28(82.3-99.4)</td><td rowspan=1 colspan=1>99.6%827/830(98.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>100%1/1(20.7-100)</td><td rowspan=1 colspan=1>100%32/32(89.3-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>891</td><td rowspan=1 colspan=1>96.6%28/29(82.8-99.4)</td><td rowspan=1 colspan=1>99.7%859/862(99.0-99.9)</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 39: iC-GPC Assay Performance: vanA/B (vancomycinresistance)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture,Biochemicals&amp; Etest</td></tr><tr><td rowspan=3 colspan=1>Jroecsd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>855</td><td rowspan=1 colspan=1>87.5%21/24(69.-95.7)</td><td rowspan=1 colspan=1>99.8%829/831(99.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>100%1/1(20.7-100)</td><td rowspan=1 colspan=1>100%32/32(89.3-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>888</td><td rowspan=1 colspan=1>88.0%*22/25(70.0-95.8)</td><td rowspan=1 colspan=1>99.8%861/863(99.2-99.9)</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 40: iC-GPC Assay Performance: mecA/methicillinresistance</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=4 colspan=1>Culture,Biochemicals&amp; CefoxitinDisc</td></tr><tr><td rowspan=3 colspan=1>Jroecsed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>848</td><td rowspan=1 colspan=1>90.0%252/280(85.9-93.0)</td><td rowspan=1 colspan=1>97.0%551/568(95.3-98.1)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%5/5(56.6-100)</td><td rowspan=1 colspan=1>100%27/27(87.5-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>880</td><td rowspan=1 colspan=1>90.2%257/285(86.2-93.1)</td><td rowspan=1 colspan=1>97.1%578/595(95.5-98.2)</td></tr></table>

\*Of the 28 observed false negative mecA results by the iC-GPC Assay, 14/28 were positive for mecA by multiple FDA-cleared multiplex assays. Repeat AST testing also confirmed 14 false negative samples to be cefoxitin sensitive. See note below.

Tables 41-44 include performance of the iC-GPC Assay for detection of mecA and vanA/vanB in comparison to traditional culture and AST testing for specific organism targets. Results include all fresh and frozen prospective clinical specimens with performance stratified for detection of mecA with S. aureus, detection of mecA with S. epidermidis, detection of vanA/vanB with $E .$ . faecalis and detection of vanA/vanB with E. faecium.

NOTE: There were 21 observed false negative mecA results by the iC-GPC Assay for specimens identified to contain methicillin-resistant S. aureus by conventional identification methods and cefoxitin disc testing (See Table 41 below). Of these 21 specimens, five were false negative due to a missed detection for the S. aureus target. Of the remaining 16 false negative results for mecA, all were also mecA negative by the FDA-cleared multiplex assay used in the method comparison study. Further investigation into these false negative results revealed that 14 of the specimens were from a single clinical site. Additional testing of isolates from the 14 false negative specimens was performed. Isolates were tested with an alternate real-time PCR assay that detects mecA and mecC. This assay confirmed the isolates to be negative for both mecA and mecC. Cefoxitin disc testing was also repeated on the 14 specimen isolates. Upon repeat testing, all isolates were found to be susceptible to cefoxitin with a zone size of $2 2 \mathrm { m m }$ which is very close to the cutoff of $2 1 \mathrm { m m }$ for susceptibility/resistance. Results from the additional testing confirm that the iC-GPC result of mecA negative was correct for 14 of the 21 false negative mecA results.

<table><tr><td rowspan=1 colspan=7>Table 41: Detection of mecA with S. aureus: Prospective Specimens</td></tr><tr><td rowspan=2 colspan=2>Staphylococcus aureus/mecA</td><td rowspan=1 colspan=5>Culture, Biochemicals &amp; Cefoxitin Disc Testing</td></tr><tr><td rowspan=1 colspan=2>SA+/mecA+</td><td rowspan=1 colspan=1>SA+/mecA-</td><td rowspan=1 colspan=1>SA-</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=4 colspan=1>iCubateiC-GPC</td><td rowspan=1 colspan=1>SA+/mecA+</td><td rowspan=1 colspan=2>113</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>114</td></tr><tr><td rowspan=1 colspan=1>SA+/mecA-</td><td rowspan=1 colspan=2>16</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>SA-</td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>610</td><td rowspan=1 colspan=1>621</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=2>134</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>611</td><td rowspan=1 colspan=1>880</td></tr><tr><td rowspan=1 colspan=5>Staphylococcus aureus/mecA</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=3>% Agreement (SA+/mecA+)</td><td rowspan=1 colspan=2>84.3%* (113/134)</td><td rowspan=1 colspan=1>77.2%</td><td rowspan=1 colspan=1>89.5%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (SA+/mecA-)</td><td rowspan=1 colspan=2>94.8% (128/135)</td><td rowspan=1 colspan=1>89.7%</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (SA-)</td><td rowspan=1 colspan=2>99.8% (610/611)</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>100%</td></tr></table>

\*See note above

<table><tr><td colspan="7" rowspan="1">Table 42: Detection of mecA with S. epidermidis: Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">Staphylococcusepidermidis/mecA</td><td colspan="5" rowspan="1">Culture, Biochemicals &amp; Cefoxitin Disc Testing</td></tr><tr><td colspan="2" rowspan="1">SE+/mecA+</td><td colspan="1" rowspan="1">SE+/mecA-</td><td colspan="1" rowspan="1">SE-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iCubateiC-GPC</td><td colspan="1" rowspan="1">SE+/mecA+</td><td colspan="2" rowspan="1">144</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">169</td></tr><tr><td colspan="1" rowspan="1">SE+/mecA-</td><td colspan="2" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="1">SE-</td><td colspan="2" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">632</td><td colspan="1" rowspan="1">641</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">151</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">655</td><td colspan="1" rowspan="1">880</td></tr><tr><td colspan="5" rowspan="1">Staphylococcus epidermidis/mecA</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE+/mecA+)</td><td colspan="2" rowspan="1">95.4% (144/151)</td><td colspan="1" rowspan="1">90.7%</td><td colspan="1" rowspan="1">97.7%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE+/mecA-)</td><td colspan="2" rowspan="1">81.2% (60/74)</td><td colspan="1" rowspan="1">70.7%</td><td colspan="1" rowspan="1">88.4%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (SE-)</td><td colspan="2" rowspan="1">96.5% (632/655)</td><td colspan="1" rowspan="1">94.8%</td><td colspan="1" rowspan="1">97.7%</td></tr><tr><td colspan="7" rowspan="1">Table 43: Detection of vanA/vanB with E. faecalis: Prospective Specimens</td></tr><tr><td colspan="2" rowspan="2">E. faecalis/vanA/vanB</td><td colspan="5" rowspan="1">Culture, Biochemicals &amp; Vancomycin AST</td></tr><tr><td colspan="2" rowspan="1">Efs+/vanR</td><td colspan="1" rowspan="1">Efs+/vanS</td><td colspan="1" rowspan="1">Efs-</td><td colspan="1" rowspan="1">TOTAL</td></tr><tr><td colspan="1" rowspan="4">iCubateiC-GPC</td><td colspan="1" rowspan="1">Efs+/van+</td><td colspan="2" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Efs+/van-</td><td colspan="2" rowspan="1">2</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">55</td></tr><tr><td colspan="1" rowspan="1">Efs-</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">828</td><td colspan="1" rowspan="1">830</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="2" rowspan="1">3</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">829</td><td colspan="1" rowspan="1">888</td></tr><tr><td colspan="5" rowspan="1">E. faecalis/vanA/vanB</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs+/van+)</td><td colspan="2" rowspan="1">33.3% (1/3)</td><td colspan="1" rowspan="1">6.2%</td><td colspan="1" rowspan="1">79.2%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs+/van-)</td><td colspan="2" rowspan="1">94.6% (53/56)</td><td colspan="1" rowspan="1">85.4%</td><td colspan="1" rowspan="1">98.2%</td></tr><tr><td colspan="3" rowspan="1">% Agreement (Efs-)</td><td colspan="2" rowspan="1">99.9% (828/829)</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">100%</td></tr></table>

Table 44: Detection of vanA/vanB with E. faecium: Prospective Specimens   

<table><tr><td rowspan=1 colspan=7>Table 44: Detectionl of vanA/vanB with E. Jaecium: Prospective Speclilells</td></tr><tr><td rowspan=2 colspan=2>E. faecalis/vanA/vanB</td><td rowspan=1 colspan=5>Culture, Biochemicals &amp; Vancomycin AST</td></tr><tr><td rowspan=1 colspan=2>Efm+/vanR</td><td rowspan=1 colspan=1>Efm+/vanS</td><td rowspan=1 colspan=1>Efm-</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=4 colspan=1>iCubateiC-GPC</td><td rowspan=1 colspan=1>Efm+/van+</td><td rowspan=1 colspan=2>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Efm+/van-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Efm-</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>856</td><td rowspan=1 colspan=1>857</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=2>21</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>859</td><td rowspan=1 colspan=1>888</td></tr><tr><td rowspan=1 colspan=5>E. faecium/vanA/vanB</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm+/van+)</td><td rowspan=1 colspan=2>95.2% (20/21)</td><td rowspan=1 colspan=1>77.3%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm+/van-)</td><td rowspan=1 colspan=2>87.5% (7/8)</td><td rowspan=1 colspan=1>52.9%</td><td rowspan=1 colspan=1>97.8%</td></tr><tr><td rowspan=1 colspan=3>% Agreement (Efm-)</td><td rowspan=1 colspan=2>99.7% (856/859)</td><td rowspan=1 colspan=1>99.0%</td><td rowspan=1 colspan=1>99.9%</td></tr></table>

# $b$ . Analysis of Mixed Culture Results:

In the method comparison study, there were seven mixed culture specimens that were detected by the iC-GPC Assay, the FDA-cleared multiplex comparator assay, or with both assays. Tables 45 and 46 list the mixed target combinations detected by iC-GPC Assay and/or the comparator assay. There was one discrepant specimen for which the iC-GPC Assay detected a target that was not detected by the comparator assay. There was one discrepant specimen for which the comparator assay detected targets that were not detected by iC-GPC. It is noted that the assay labeling includes a limitation that due to competitive inhibition, target organisms present near LoD concentrations may not be detected by the iC-GPC Assay when a second target organism is present at higher concentrations.

Table 45: Multiple Organism Detections by iC-GPC as Compared to FDA-cleared multiplex assay   

<table><tr><td rowspan=1 colspan=6>Multiple Detections by iC-GPC</td><td rowspan=2 colspan=1>TotalTargetsDetected</td><td rowspan=2 colspan=1>DiscrepantTargets</td><td rowspan=2 colspan=1>Discrepant Results(Targets Not Detectedby FDA-ClearedMultiplex Assay</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>Target 4</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1081</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1149</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1150</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>vanA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1289</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1403</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfacalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>TGH</td><td rowspan=1 colspan=1>4029</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table 46: Multiple Organism Detections by FDA-cleared multiplex assay as compared to iC-GPC   

<table><tr><td rowspan=1 colspan=6>Multiple Detections by FDA-Cleared Multiplex Assay</td><td rowspan=2 colspan=1>TotalTargetsDetected</td><td rowspan=2 colspan=1>DiscrepantTargets</td><td rowspan=2 colspan=1>Discrepant Results(Targets NotD Detected byi-GPC</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>Target 4</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1081</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1149</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1150</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1289</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1403</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>MCW</td><td rowspan=1 colspan=1>2134</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Staphylococcusaureus, mecA</td></tr><tr><td rowspan=1 colspan=1>TGH</td><td rowspan=1 colspan=1>4029</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table 47 includes a listing of specimens generating iC-GPC positive results for multiple organism targets as compared to results from reference culture methods. Of the six mixed culture specimens detected by the iC-GPC Assay, there were two discrepant specimens in which the iC-GPC Assay detected a target that was not detected by culture, biochemicals, and AST testing.

Table 47: Multiple Organism Detections by iC-GPC as Compared to Culture   

<table><tr><td rowspan=1 colspan=6>Multiple Detections by FDA-cleared FDA Assay</td><td rowspan=2 colspan=1>TotalTargetsDetected</td><td rowspan=2 colspan=1>DiscrepantTargets</td><td rowspan=2 colspan=1>Discrepant Results(Targets NotDDetected byCulture/AST)</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>Target 4</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1081</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1149</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1150</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Enterococcusfaecium, vanA</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1289</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>vanB (E-TestIntermediate)</td></tr><tr><td rowspan=1 colspan=1>LAC</td><td rowspan=1 colspan=1>1403</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>TGH</td><td rowspan=1 colspan=1>4029</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table 48 includes a listing of specimens for which multiple organisms were recovered by traditional reference culture methods. There were 56 mixed culture specimens identified by culture and standard of care isolate identification, including organisms not detected by the iC-GPC Assay. There were a total of 12 discrepant specimens for which the reference methods identified isolates that were not detected by iC-GPC (false negatives). There were nine discrepant specimens for which iC-GPC detected targets that were not detected by the reference methods (false positives).

Table 48: Multiple Organisms recovered by Culture as Compared to iC-GPC Assay   

<table><tr><td colspan="6" rowspan="1">Multiple Detections by Culture, Biochemicals &amp; AST</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Frequency</td><td colspan="1" rowspan="1">No. w/DiscrepantTargets</td><td colspan="1" rowspan="1">Discrepant Target</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus aureus,mecA,Enterococcus faecalis,Staphylococcusepidermidis*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td><td colspan="1" rowspan="1">Enterococcus faecium,vanA</td><td colspan="1" rowspan="1">Candidaglabrata</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2">Staphylococcus aureus,mecA</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Stenotrophomonasrhizophila</td><td colspan="1" rowspan="1">Pseudomonasfulva</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Staphylococcusepidermidis (2), mecA(2)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Frequency</td><td colspan="1" rowspan="1">No. w/Discrepant Targets</td><td colspan="1" rowspan="1">Discrepant Target</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcus hominis,mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">mecA*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus, mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus, mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">mecA*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcus capitus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Staphylococcus capitus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcus capitus,mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcus warneri,mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Staphylococcus species,mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis, mecA</td><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Bacillus circulans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuse idermidis</td><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Low Disc: Strep.mitis/oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td><td colspan="1" rowspan="1">Unidentified Organism</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcus aureus,mecA</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="1">Aerococcusviridans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterococcus faecalis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus, mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis,mecA</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Staphylococcusepidermidis (2)*,Enterococcus faecalis*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis,mecA</td><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepiddermidis*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Low Disc: Streptococcusmitis/oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Staphylococcusepidermidis*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lentus,mecA</td><td colspan="1" rowspan="1">Low Disc: Strep.mitis/oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohniissp urealyticus, mecA</td><td colspan="1" rowspan="1">Staphylococcus cohnii sspcohnii, mecA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr></table>

\*false positive result

3. Clinical cut-off:

Not applicable

4. Expected values:

A total of 913 prospectively collected fresh and frozen blood culture specimens were obtained from four geographically dispersed clinical sites. The number and percentage of positive cases (positivity rate) determined by the iC-GPC Assay stratified by U.S. state for each of the organisms and resistance markers detected by the assay are presented below. Overall, the iC-GPC Assay detected at least one organism in $6 6 . 9 \%$ (611/913) prospectively collected specimens and at least one resistance marker in $3 3 . 4 \%$ (305/913) prospectively collected specimens. Expected values are presented in Table 49.

Table 49: Positivity by the iC-GPC Assay as Observed in the Clinical Study   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=1>Sites</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>TOTAL n</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>913</td></tr><tr><td rowspan=2 colspan=1>S. aureus</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>266</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>28.7%</td><td rowspan=1 colspan=1>21.2%</td><td rowspan=1 colspan=1>37.1%</td><td rowspan=1 colspan=1>32.3%</td><td rowspan=1 colspan=1>29.1%</td></tr><tr><td rowspan=2 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>235</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>21.0%</td><td rowspan=1 colspan=1>31.6%</td><td rowspan=1 colspan=1>26.6%</td><td rowspan=1 colspan=1>21.9%</td><td rowspan=1 colspan=1>25.7%</td></tr><tr><td rowspan=2 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=2 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>6.7%</td></tr><tr><td rowspan=2 colspan=1>E. faecium</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>3.2%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=1>TOTAL n</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>913</td></tr><tr><td rowspan=2 colspan=1>mecA</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>285</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>25.7%</td><td rowspan=1 colspan=1>33.8%</td><td rowspan=1 colspan=1>32.7%</td><td rowspan=1 colspan=1>37.5%</td><td rowspan=1 colspan=1>31.2%</td></tr><tr><td rowspan=2 colspan=1>vanA</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=2 colspan=1>vanB</td><td rowspan=1 colspan=1>POSITIVEn</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>% Positivity</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.2%</td></tr></table>

# N. Instrument Name:

iC-System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No _X

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No ___X_

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X_ _ or No

3. Specimen Identification: Patient information is manually entered into the iMac computer for each iC-GPC test. For each patient specimen, the user then selects “Link Cassette” in the iC-System software and then scans the Cassette barcode using a barcode scanner.

4. Specimen Sampling and Handling:

Positive blood culture specimens are extracted from the blood culture bottle using a syringe and placed into a sterile tube. A gram stain is then performed to determine the presence of gram positive cocci. If gram positive cocci are present and is not mixed, the user then pipets the well-mixed specimen into the bottom of the sample well in the iCGPC Cassette. The user closes the cassette cap and locks the sliding lid into place. The loaded cassette is then placed onto the iC-Processor. When processing is complete, the Cassette automatically ejects and then the user inserts the iC-Cassette into the iC-Reader where the iC-Cassette is read. Results are automatically transferred to the iC-Software and subsequently reported.

5. Calibration:

The user is not required to calibrate the instrument.

6. Quality Control:

See Section M(1)(d) above for information on internal and external controls.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.