Bruker Daltonik GmbH Mr. David Cromwick Director of Quality 40 Manning Rd Billerica, MA 01821 US

Re: K163536 Trade/Device Name: MALDI Biotyper CA (MBT-CA) System, MBT smart CA System Regulation Number: 21 CFR 866.3361 Regulation Name: Mass spectrometer system for clinical use for the identification of microorganisms Regulatory Class: II Product Code: PEX Dated: December 16, 2016 Received: December 16, 2016

Dear Mr. Cromwick:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA‚Äôs issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act‚Äôs requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of

medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Division of Industry and Consumer Education at its toll-free number (800) 638 2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, ‚ÄúMisbranding by reference to premarket notification‚Äù (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH‚Äôs Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

# Kristian M. Roth -S

Uwe Scherf, M.Sc., Ph.D.'PS Director Division of Microbiology Devices Office of In Vitro Diagnostics and Radiological Health Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K163536

Device Name MALDI Biotyper CA System

Indications for Use (Describe) The MALDI Biotyper CA System is a mass spectrometer systems using matrix-assisted laser desorption/ionization - time of flight (MALDI-TOF) for the identification of microorganisms cultured from human specimens.

The MALDI Biotyper CA System is a qualitative in vitro diagnostic device indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of bacterial and yeast infections.

The following organisms are claimed:

Bacteria:   
Abiotrophia defectiva   
Achromobacter xylosoxidans   
Acinetobacter baumannii / nosocomialis group   
Acinetobacter calcoaceticus   
Acinetobacter haemolyticus   
Acinetobacter johnsonii   
Acinetobacter junii   
Acinetobacter lwoffii   
Acinetobacter pittii   
Acinetobacter radioresistens   
Acinetobacter ursingii   
Actinomyces europaeus   
Actinomyces funkei   
Actinomyces graevenitzii   
Actinomyces hyovaginalis   
Actinomyces meyeri   
Actinomyces neuii   
Actinomyces odontolyticus   
Actinomyces oris   
Actinomyces radingae   
Actinomyces turicensis   
Actinomyces urogenitalis   
Actinotignum schaalii group   
Aerococcus sanguinicola   
Aerococcus urinae   
Aerococcus viridans   
Aeromonas salmonicida   
Aeromonas sp[7]   
Aggregatibacter actinomycetemcomitans   
Aggregatibacter aphrophilus   
Aggregatibacter segnis   
Alcaligenes faecalis   
Alloiococcus otitis   
Alloscardovia omnicolens   
Bacteroides caccae   
Bacteroides fragilis   
Bacteroides nordii   
Bacteroides ovatus group   
Bacteroides pyogenes   
Bacteroides salyersiae   
Bacteroides stercoris group   
Bacteroides thetaiotaomicron group   
Bacteroides uniformis   
Bacteroides vulgatus group   
Bifidobacterium breve   
Bordetella group[3]   
Bordetella hinzii   
Brevibacterium casei   
Brevundimonas diminuta group   
Burkholderia cepacia complex [13]   
Burkholderia gladioli   
Burkholderia multivorans   
Campylobacter coli   
Campylobacter jejuni   
Campylobacter ureolyticus   
Capnocytophaga ochracea   
Capnocytophaga sputigena   
Chryseobacterium gleum   
Chryseobacterium indologenes   
Citrobacter amalonaticus complex   
Citrobacter freundii complex   
Citrobacter koseri   
Clostridium beijerinckii   
Clostridium bifermentans   
Clostridium butyricum   
Clostridium clostridioforme group   
Clostridium difficile   
Clostridium innocuum   
Clostridium paraputrificum   
Clostridium perfringens   
Clostridium ramosum   
Clostridium septicum   
Clostridium sordellii   
Clostridium sporogenes / Clostridium botulinum (group I)   
Clostridium tertium   
Corynebacterium accolens   
Corynebacterium afermentans group   
Corynebacterium amycolatum   
Corynebacterium aurimucosum group   
Corynebacterium bovis   
Corynebacterium coyleae   
Corynebacterium diphtheriae   
Corynebacterium freneyi   
Corynebacterium glucuronolyticum   
Corynebacterium glutamicum   
Corynebacterium jeikeium   
Corynebacterium kroppenstedtii   
Corynebacterium macginleyi   
Corynebacterium minutissimum   
Corynebacterium mucifaciens / ureiceler   
Corynebacterium propinquum   
Corynebacterium pseudodiphtheriticum   
Corynebacterium pseudotuberculosis   
Corynebacterium resistens   
Corynebacterium riegelii   
Corynebacterium striatum group   
Corynebacterium tuberculostearicum   
Corynebacterium ulcerans   
Corynebacterium urealyticum   
Corynebacterium xerosis   
Cronobacter sakazakii group   
Cupriavidus pauculus group   
Delftia acidovorans group   
Dermabacter hominis   
Dermacoccus nishinomiyaensis   
Edwardsiella tarda   
Eikenella corrodens   
Elizabethkingia meningoseptica group   
Enterobacter aerogenes   
Enterobacter amnigenus   
Enterobacter cloacae complex   
Enterococcus avium   
Enterococcus casseliflavus   
Enterococcus durans   
Enterococcus faecalis   
Enterococcus faecium   
Enterococcus gallinarum   
Enterococcus hirae   
Enterococcus mundtii   
Enterococcus raffinosus   
Escherichia coli   
Escherichia hermannii   
Escherichia vulneris   
Ewingella americana   
Facklamia hominis   
Finegoldia magna   
Fluoribacter bozemanae   
Fusobacterium canifelinum   
Fusobacterium necrophorum   
Fusobacterium nucleatum   
Gardnerella vaginalis   
Gemella haemolysans   
Gemella morbillorum   
Gemella sanguinis   
Granulicatella adiacens   
Haemophilus haemolyticus   
Haemophilus influenzae   
Haemophilus parahaemolyticus group   
Haemophilus parainfluenzae   
Hafnia alvei   
Helcococcus kunzii   
Kingella denitrificans   
Kingella kingae   
Klebsiella oxytoca / Raoultella ornithinolyt   
Klebsiella pneumoniae   
Klebsiella variicola   
Kocuria kristinae   
Kytococcus sedentarius   
Lactobacillus gasseri   
Lactobacillus jensenii   
Lactobacillus rhamnosus   
Lactococcus garvieae   
Lactococcus lactis   
Leclercia adecarboxylata   
Legionella longbeachae   
Legionella pneumophila   
Leuconostoc citreum   
Leuconostoc mesenteroides   
Leuconostoc pseudomesenteroides   
Listeria monocytogenes   
Macrococcus caseolyticus   
Mannheimia haemolytica group   
Micrococcus luteus   
Micrococcus lylae   
Mobiluncus curtisii   
Moraxella sg Branhamella catarrhalis\*   
Moraxella sg Moraxella nonliquefaciens\*   
Moraxella sg Moraxella osloensis\*   
Morganella morganii   
Myroides odoratimimus   
Myroides odoratus   
Neisseria bacilliformis   
Neisseria cinerea   
Neisseria elongata   
Neisseria flavescens / subflava group   
Neisseria gonorrhoeae   
Neisseria lactamica   
Neisseria meningitidis   
Neisseria sicca group   
Neisseria weaveri   
Nocardia brasiliensis   
Nocardia cyriacigeorgica   
Nocardia farcinica group   
Nocardia nova   
Nocardia otitidiscaviarum   
Ochrobactrum anthropi   
Oligella ureolytica   
Oligella urethralis   
Pantoea agglomerans   
Parabacteroides distasonis   
Parabacteroides goldsteinii   
Parabacteroides johnsonii / merdae group   
Parvimonas micra   
Pasteurella multocida   
Pediococcus acidilactici   
Pediococcus pentosaceus   
Peptoniphilus harei group   
Peptostreptococcus anaerobius   
Plesiomonas shigelloides   
Pluralibacter gergoviae   
Porphyromonas gingivalis   
Porphyromonas somerae   
Prevotella bivia   
Prevotella buccae   
Prevotella denticola   
Prevotella intermedia   
Prevotella melaninogenica   
Propionibacterium acnes   
Proteus mirabilis   
Proteus vulgaris group   
Providencia rettgeri   
Providencia stuartii   
Pseudomonas aeruginosa   
Pseudomonas fluorescens group   
Pseudomonas oryzihabitans   
Pseudomonas putida group   
Pseudomonas stutzeri   
Ralstonia pickettii   
Rhizobium radiobacter   
Rothia aeria   
Rothia dentocariosa   
Rothia mucilaginosa   
Salmonella sp\*\*   
Serratia fonticola   
Serratia liquefaciens   
Serratia marcescens   
Serratia odorifera   
Serratia plymuthica   
Serratia rubidaea   
Sphingobacterium multivorum   
Sphingobacterium spiritivorum   
Sphingomonas paucimobilis group   
Staphylococcus aureus   
Staphylococcus auricularis   
Staphylococcus capitis   
Staphylococcus caprae   
Staphylococcus carnosus   
Staphylococcus cohnii   
Staphylococcus delphini   
Staphylococcus epidermidis   
Staphylococcus equorum   
Staphylococcus felis   
Staphylococcus haemolyticus   
Staphylococcus hominis   
Staphylococcus intermedius   
Staphylococcus lentus   
Staphylococcus lugdunensis   
Staphylococcus pasteuri   
Staphylococcus pettenkoferi   
Staphylococcus pseudintermedius   
Staphylococcus saccharolyticus   
Staphylococcus saprophyticus   
Staphylococcus schleiferi   
Staphylococcus sciuri   
Staphylococcus simulans   
Staphylococcus vitulinus   
Staphylococcus warneri   
Staphylococcus xylosus   
Stenotrophomonas maltophilia   
Streptococcus agalactiae   
Streptococcus anginosus   
Streptococcus canis   
Streptococcus constellatus   
Streptococcus dysgalactiae   
Streptococcus equi   
Streptococcus gallolyticus   
Streptococcus gordonii   
Streptococcus intermedius   
Streptococcus lutetiensis   
Streptococcus mitis / oralis group   
Streptococcus mutans   
Streptococcus parasanguinis   
Streptococcus pneumoniae   
Streptococcus pyogenes   
Streptococcus salivarius / vestibularis group   
Streptococcus sanguinis   
Streptococcus sobrinus   
Streptococcus thermophilus   
Sutterella wadsworthensis   
Trueperella bernardiae   
Turicella otitidis   
Vagococcus fluvialis   
Veillonella parvula group   
Vibrio parahaemolyticus   
Vibrio vulnificus   
Weeksella virosa   
Yersinia enterocolitica   
Yersinia frederiksenii   
Yersinia intermedia   
Yersinia kristensenii   
Yersinia pseudotuberculosis

$\ast _ { \mathrm { s g } } =$ subgenus $* * _ { \mathrm { s p } } =$ species

Yeasts:   
Candida albicans   
Candida boidinii   
Candida dubliniensis   
Candida duobushaemulonii

<table><tr><td>Candida famata</td><td></td></tr><tr><td>Candida glabrata</td><td></td></tr><tr><td>Candida guilliermondii</td><td></td></tr><tr><td>Candida haemulonis</td><td></td></tr><tr><td>Candida inconspicua</td><td></td></tr><tr><td>Candida intermedia</td><td></td></tr><tr><td>Candida kefyr Candida krusei</td><td></td></tr><tr><td>Candida lambica</td><td></td></tr><tr><td>Candida lipolytica</td><td></td></tr><tr><td>Candida lusitaniae</td><td></td></tr><tr><td>Candida metapsilosis</td><td></td></tr><tr><td>Candida norvegensis</td><td></td></tr><tr><td>Candida orthopsilosis</td><td></td></tr><tr><td>Candida parapsilosis</td><td></td></tr><tr><td>Candida pararugosa</td><td></td></tr><tr><td>Candida pelliculosa</td><td></td></tr><tr><td>Candida tropicalis</td><td></td></tr><tr><td>Candida valida</td><td></td></tr><tr><td>Candida zeylanoides</td><td></td></tr><tr><td>Cryptococcus gattii</td><td></td></tr><tr><td>Cryptococcus neoformans var grubii*</td><td></td></tr><tr><td>Cryptococcus neoformans var neoformans*</td><td></td></tr><tr><td>Cyberlindnera jadinii</td><td></td></tr><tr><td>Geotrichum candidum</td><td></td></tr><tr><td>Geotrichum capitatum</td><td></td></tr><tr><td>Kloeckera apiculata</td><td></td></tr><tr><td>Malassezia furfur</td><td></td></tr><tr><td>Malassezia pachydermatis</td><td></td></tr><tr><td>Pichia ohmeri</td><td></td></tr><tr><td>Rhodotorula mucilaginosa</td><td></td></tr><tr><td>Saccharomyces cerevisiae</td><td></td></tr><tr><td>Trichosporon asahii</td><td></td></tr><tr><td>Trichosporon inkin</td><td></td></tr><tr><td>Trichosporon mucoides group</td><td></td></tr></table>

\*var $=$ variety

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

‚ÄúAn agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.‚Äù

# 10. 510(k) SUMMARY

# Date of Summary:

June 21, 2017

# Product Name

MBT-CA System

Sponsor

Bruker Daltonik GmbH Fahrenheitstrasse 4 28359 Bremen, Germany

# Correspondent

Bruker Daltonik GmbH   
Markus Kostrzewa, Vice President Clinical Mass Spectrometry R&D   
Fahrenheitstrasse 4   
28359 Bremen, Germany   
Phone: $+ 4 9$ (0)421-2205 1258 Fax: $+ 4 9$ (0)421-2205 1400   
Email: Markus.Kostrzewa@bruker.com

# Device Identification

Trade or Proprietary Name: Common or Usual Name:

MALDI Biotyper CA System   
System, mass spectrometry, maldi tof, microorganism identification,   
cultured isolates   
PEX   
21 CFR 866.3361 Instrumentation for clinical multiplex test systems   
Class II   
Microbiology

Product Code: Regulation Section: Device Class: Panel:

# Substantial Equivalency

The MBT-CA System (K163536) is substantially equivalent to the MBT-CA System (K142677). Table 1 compares the characteristics of the MBT-CA System (New Device) and the MBT-CA System (Predicate Device).

Table 1: Substantial Equivalency Table   

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">NEW DEVICEMBT-CA System (K163536)</td><td colspan="1" rowspan="1">PRIMARY PREDICATE DEVICEMBT-CA System (K142677)</td></tr><tr><td colspan="1" rowspan="1">Product Codes</td><td colspan="1" rowspan="1">PEX</td><td colspan="1" rowspan="1">PEX</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The MALDI Biotyper CA System is a massspectrometer systems using matrix-assistedlaser desorption/ionization - time of flight(MALDI-TOF) for the identification ofmicroorganisms cultured from humanspecimens.The MALDI Biotyper CA System is aqualitative in vitro diagnostic devicesindicated for use in conjunction with otherclinical and laboratory findings to aid in thediagnosis of bacterial and yeast infections.</td><td colspan="1" rowspan="1">The MALDI Biotyper CA System is a massspectrometer system using matrix-assistedlaser desorption/ionization - time of flight(MALDI-TOF) for the identification ofmicroorganisms cultured from humanspecimens.The MALDI Biotyper CA System is aqualitative in vitro diagnostic deviceindicated for use in conjunction with otherclinical and laboratory findings to aid in thediagnosis of bacterial and yeast infections.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Isolated colony from any patient samplesource.Acceptable media:Columbia blood agar with 5% sheepblood (Gram-negative bacteria)Trypticase soy agar with 5% sheep blood(Gram-negative bacteria, Gram-positivebacteria, yeasts)Chocolate agar (Gram-negative bacteria,Gram-positive bacteriaMacConkey Agar (Gram-negativebacteria)Columbia CNA agar with 5% sheep blood(Gram-positive bacteria)Brucella Agar with 5% horse blood (Gram-negative anaerobic bacteria, Gram-positive anaerobic bacteria)CDC anaerobe Agar with 5% sheep blood(Gram-negative anaerobic bacteria,Gram-positive anaerobic bacteria)CDC anaerobe 5% sheep blood Agar withphenylethyl alcohol (Gram-negativeanaerobic bacteria, Gram-positiveanaerobic bacteria)CDC anaerobe laked sheep blood Agarwith kanamycin and vancomycin (Gram-negative anaerobic bacteria)Bacteroides bile esculin Agar withamikacin (Bacteroides species)Clostridium difficile Agar with 7% sheep</td><td colspan="1" rowspan="1">Isolated colony from any patient samplesource.Acceptable media:Columbia blood agar with 5% sheepblood (Gram-negative bacteria)Trypticase soy agar with 5% sheep blood(Gram-negative bacteria, Gram-positivebacteria, yeasts)Chocolate agar (Gram-negative bacteria,Gram-positive bacteriaMacConkey Agar (Gram-negativebacteria)Columbia CNA agar with 5% sheep blood(Gram-positive bacteria)Brucella Agar with 5% horse blood (Gram-negative anaerobic bacteria, Gram-positive anaerobic bacteria)CDC anaerobe Agar with 5% sheep blood(Gram-negative anaerobic bacteria,Gram-positive anaerobic bacteria)CDC anaerobe 5% sheep blood Agar withphenylethyl alcohol (Gram-negativeanaerobic bacteria, Gram-positiveanaerobic bacteria)CDC anaerobe laked sheep blood Agarwith kanamycin and vancomycin (Gram-negative anaerobic bacteria)Bacteroides bile esculin Agar withamikacin (Bacteroides species)Clostridium difficile Agar with 7% sheep</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">blood (Clostridium difficile)Sabouraud-Dextrose Agar (Yeasts)Brain Heart Infusion Agar (Yeasts)Campylobacter Agar with 5 Antimicrobicsand 10% Sheep Blood (CampylobacterspeciesBordet Gengou Agar with 15% sheepblood (Bordetella species)plus three (3) additionally validatedmedia (see differences).</td><td colspan="1" rowspan="1">blood (Clostridium difficile)Sabouraud-Dextrose Agar (Yeasts)Brain Heart Infusion Agar (Yeasts)Campylobacter Agar with 5 Antimicrobicsand 10% Sheep Blood (CampylobacterspeciesBordet Gengou Agar with 15% sheepblood (Bordetella species)</td></tr><tr><td colspan="1" rowspan="1">Type of Test</td><td colspan="1" rowspan="1">Automated Mass Spectrometry System</td><td colspan="1" rowspan="1">Automated Mass Spectrometry System</td></tr><tr><td colspan="1" rowspan="1">Matrix</td><td colspan="1" rowspan="1">Œ±-Cyano-4-hydroxycinnamic acid</td><td colspan="1" rowspan="1">Œ±-Cyano-4-hydroxycinnamic acid</td></tr><tr><td colspan="1" rowspan="1">Method ofTesting</td><td colspan="1" rowspan="1">Bacteria &amp; Yeast: Direct testingIf after initial analysis the log(score) isreported at &lt;2.00, organisms may beprocessed using the Extraction (Ext)procedure or extended Direct Transfer (eDT,70% aqueous formic acid) procedure. If eDTprocedure still yields log(score) &lt;2.00,organisms may be processed via Extprocedure.</td><td colspan="1" rowspan="1">Bacteria &amp; Yeast: Direct testingIf after initial analysis the log(score) isreported at &lt;2.00, organisms may beprocessed using the Extraction (Ext)procedure or extended Direct Transfer (eDT,70% aqueous formic acid) procedure. If eDTmethod still yields log(score) &lt;2.00,organisms may be processed via Extprocedure.</td></tr><tr><td colspan="1" rowspan="1">Result Reporting</td><td colspan="1" rowspan="1">Organism identification is reported withhigh confidence if the log(score) is ‚â•2.00.An organism identification is reported withlow confidence if the log(score) is between1.70 and &lt;2.00.</td><td colspan="1" rowspan="1">Organism identification is reported withhigh confidence if the log(score) is ‚â•2.00.An organism identification is reported withlow confidence if the log(score) is between1.70 and &lt;2.00.</td></tr><tr><td colspan="1" rowspan="1">MatchingAlgorithm</td><td colspan="1" rowspan="1">Calculates matches by comparing a newspectrum against each single referenceentry of a reference database.</td><td colspan="1" rowspan="1">Calculates matches by comparing a newspectrum against each single referenceentry of a reference database.</td></tr><tr><td colspan="1" rowspan="1">Recorded massrange</td><td colspan="1" rowspan="1">2,000 - 20,000 m/z</td><td colspan="1" rowspan="1">2,000 - 20,000 m/z</td></tr><tr><td colspan="1" rowspan="1">Calibration</td><td colspan="1" rowspan="1">Bruker US IVD Bacterial Test Standard (BTS)</td><td colspan="1" rowspan="1">Bruker US IVD Bacterial Test Standard (BTS)</td></tr><tr><td colspan="1" rowspan="1">Database</td><td colspan="1" rowspan="1">MALDI Biotyper Reference Library forClinical Applications (MBT-CA) - update</td><td colspan="1" rowspan="1">MALDI Biotyper Reference Library forClinical Applications (MBT-CA)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">NEW DEVICEMBT-CA System (K163536)</td><td colspan="1" rowspan="1">PRIMARY PREDICATE DEVICEMBT-CA System (K142677)</td></tr><tr><td colspan="1" rowspan="1">System Update</td><td colspan="1" rowspan="1">System claims additional organisms but noadditional changes.</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Media</td><td colspan="1" rowspan="1">Additional Media validated:Buffered Charcoal Yeast Extract Agar(Legionella species)Buffered Charcoal Yeast Extract SelectiveAgar with polymyxin, anisomycin andvancomycin (Nocardia species)Modified Thayer-Martin Agar (Neisseriaspecies)</td><td colspan="1" rowspan="1">See Similarities</td></tr><tr><td colspan="1" rowspan="1">MALDI-TOF MSinstruments</td><td colspan="1" rowspan="1">Bruker microflex LT/SH (benchtop)Bruker microflex LT/SH smart (benchtop)</td><td colspan="1" rowspan="1">Bruker microflex LT/SH (benchtop)</td></tr><tr><td colspan="1" rowspan="1">MALDI TargetPlates</td><td colspan="1" rowspan="1">US IVD 48 Spot Target(48 positions reusable steel targets)MBT Biotarget 96 US IVD(96 positions disposable targets)</td><td colspan="1" rowspan="1">US IVD 48 Spot Target(48 positions reusable steel targets)</td></tr></table>

The differences do not affect substantial equivalence of the MBT-CA System and MBT-CA System (K142677). Both systems are mass spectrometers using matrix-assisted laser desorption/ionization-time to flight (MALDI-TOF) for the identification of microorganisms cultured from human specimens. The differences noted above do not impact the intended use and do not raise questions as to the safety and effectiveness of the test (new) device.

# Intended Use

The MALDI Biotyper CA System is a mass spectrometer system using matrix-assisted laser desorption/ionization - time of flight (MALDI-TOF) for the identification of microorganisms cultured from human specimens.

The MALDI Biotyper CA System is a qualitative in vitro diagnostic device indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of bacterial and yeast infections.

# Methodology

Biochemical methods are currently the most commonly used methods for the identification of microorganisms. Organisms are tested against a range of reagents and organism identification is based on a microorganism's reaction to these reagents.

The MBT-CA System uses a different methodology for organism identification based on unique protein patterns of the microorganisms obtained from mass spectrometry. The test organism's spectrum (a pattern of mass peaks) is compared with a reference spectra library (database). Using biostatistical analysis, a probability ranking of the organism identification is generated. The probability ranking is represented as a log(score) between 0.00 and 3.00. Organism identification is reported with high confidence if the log(score) is $\ge 2 . 0 0$ . An organism identification is reported with low confidence if the log(score) is between 1.70 and $< 2 . 0 0$ .

Organisms to be identified with the MBT-CA System should be isolated for purity on appropriate isolation media.

Direct Transfer procedure (DT): An individual colony from a subculture plate is transferred to a selected position on a US IVD 48 Spot Target plate or a MBT Biotarget 96 US IVD Target plate (targets) and overlaid with US IVD HCCA portioned (matrix). The standard solvent ( $50 \%$ acetonitrile $/ 4 7 . 5 \% \mathsf { H } _ { 2 } \mathsf { O } / 2 . 5 \%$ trifluoroacetic acid) in the matrix solution extracts proteins (mainly ribosomal proteins, which are present in high concentration) from the microorganisms. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System. If after initial analysis the log(score) is reported as $< 2 . 0 0$ , organisms can be processed using the extended Direct Transfer (eDT) procedure or the Extraction procedure and analysis repeated. If eDT is employed and log(score) is reported as $< 2 . 0 0$ , reanalysis via the Extraction procedure may be used.

extended Direct Transfer procedure (eDT):

If DT analysis yields a [log(score) $< 2 . 0 0 ]$ result, an individual colony from a subculture plate may be transferred to a selected position on a target and overlaid with $70 \%$ aqueous formic acid solution. The target is air-dried and then matrix is overlaid. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System. If a high confidence result is not achieved [log(score) is reported at $< 2 . 0 0 ]$ , organisms can be processed using the Extraction procedure and analysis repeated.

Extraction procedure $( E x t )$ : If after initial analysis and eDT procedure the log(score) is reported at $\mathtt { < } 2 . 0 0$ , organisms are processed using the Extraction procedure and analysis repeated. For this purpose, isolated colonies from the subculture plate are extracted in accordance with instructions of the user manual (Ext sample preparation procedure). An aliquot of extracted material is transferred to a selected position on a target, air-dried and then overlaid with matrix material. When dried matrix crystallizes, the inoculated target is ready to be analyzed on the MBT-CA System.

# MALDI-TOF Analysis:

Samples are analyzed using MALDI (matrix-assisted laser desorption/ionization) TOF (timeof-flight) mass spectrometry. The matrix transfers protons onto the extracted proteins and absorbs UV light. A laser in the MALDI- TOF mass spectrometer irradiates the matrix sample composite, causing evaporation and release of positively charged intact proteins and peptides ("soft" ionization technique). These ions are electrostatically accelerated over a short distance and arrive in the flight tube at a mass-dependent speed. As different proteins/peptides have different masses, ions arrive at the detector at different times (time of flight). The system measures the time (in the nanosecond range) between pulsed acceleration and the corresponding detector signal, the speed is converted into an exact molecular mass. The mass-to-charge ratio of an ion is proportional to the square of its drift time.

Highly abundant microbial proteins (mainly ribosomal proteins) result in a mass spectrum with characteristic mass and intensity distribution. It is species-specific for many bacteria and is interpreted as a molecular fingerprint to identify the test organism.

Data acquisition is controlled with MBT-CA System software. The spectrum of the unknown organism is first transformed into a peak list. Using a biostatistical algorithm, this peak list is compared to the reference peak lists of organisms in the reference library (database) and a log(score) is generated. A higher log(score) indicates a higher degree of similarity to the organism in the reference library. Organism identification is reported with high confidence if the log(score) is $\ge 2 . 0 0$ . An organism identification is reported with low confidence if the log(score) is between 1.70 and $< 2 . 0 0$ .

The log(score) ranges, defined in the MBT-CA System, are indicative of the probability of organism identification. Results should be reviewed by a trained microbiologist and final organism identification should be based on all relevant information available. This information includes but is not limited to: Gram staining, colony morphology, growth characteristics, sample matrix, etc.

# Performance Data

# Media and Colony Stability

With the inclusion of further Gram-negative and Gram-positive microaerophilic, additional Gram-negative and Gram-positive anaerobic and further Gram-negative and Gram-positive aerobic bacteria and yeasts, a study on the following media was conducted to confirm acceptability of the recommended agar and the stability of the colony for up to 12 hours storage at room temperature prior to analysis:

ÔÇ∑ Trypticase Soy Agar with $5 \%$ sheep blood (TSA) Buffered Charcoal Yeast Extract Agar (BCYE) Buffered Charcoal Yeast Extract Selective Agar with polymyxin, anisomycin and vancomycin (BCYE/PAV)   
ÔÇ∑ Modified Thayer-Martin Agar (MTM)

In the previousl submission (K142677) TSA medium was only validated for Gram-positive /- negative aerobic bacteria and yeast. Goal for TSA medium in this submission was to show the suitability of this medium for Gram-positive /-negative microaerophilic and anaerobic bacteria.

Testing was conducted using one (1) Gram-negative bacterium, one (1) microaerophilic Gram-negative bacterium, one (1) Gram-positive bacterium, one (1) microaerophilic Grampositive bacterium, two (2) Legionella species, three (3) Nocardia species and three (3) Neisseria species, at varying incubation time points in replicates of eight (8). Additionally, thirteen (13) clinical isolates derived from Neisseria species were tested in duplicate at varying incubation time points.

Conclusion Media and Colony Stability: Three (3) additional media could be validated successfully. TSA medium could be validated for all kinds of organisms. Cultivation of Neisseria on Modified Thayer-Martin Agar (MTM) should not be longer than 24 hours plus 12 hours storage at room temperature.

# Resolution of the Acinetobacter baumannii complex

This study served to evaluate an improved species resolution power of the MBT-CA System of the Acinetobacter baumannii complex (members: A. baumannii, A. calcoaceticus, A. nosocomialis, A. pittii) down to the species level after final extraction procedure. Sixty-six (66) clinical isolates of Acinetobacter baumannii complex collected during initial Clinical Method Comparison protocol (see K130831) and identified by 16S rRNA and/or protein gene sequencing were re-tested using Direct Transfer, extended Direct Transfer and Extraction procedure in parallel.

The study results showed that the Acinetobacter baumannii complex cannot be completely but partially resolved applying MBT-CA System workflow. The reference library/software will be updated to complex Acinetobacter baumannii and Acinetobacter nosocomialis under the Acinetobater baumannii/nosocomialis group. After formation of an Acinetobacter baumannii/nosocomialis group unambiguous MBT-CA identification with high confidence [log(score) $2 2 . 0 0 ]$ of Acinetobacter baumannii/nosocomialis group, Acinetobacter calcoaceticus and Acinetobacter pittii is possible if final Extraction procedure has been applied. A matching hint will be included in the system package insert which contains the following alert: The displayed species should be considered a member of the Acinetobacter baumannii complex. For organisms identified by the MBT-CA System as Acinetobacter calcoaceticus, Acinetobacter pittii or Acinetobacter baumannii/nosocomialis group the full Extraction procedure (Ext) is mandatory for secure species differentiation.

# Cultivation of yeast organisms at $3 7 ^ { \circ } \mathsf { C }$

This study served to show the general applicability of the MBT-CA System workflow for identification of yeasts cultivated at 37 $( \pm 2 ) ^ { \circ } \mathsf C$ . Testing was conducted using three (3) yeast species at varying incubation time points in replicates of eight (8). After initial testing, isolates were further tested at room temperature after twelve (12) hours post-incubation.

The study results confirmed the acceptability of 37 $( \pm 2 ) ^ { \circ } \mathsf C$ cultivation of yeasts and sample colony stability of up to 12 hours.

# Reproducibility:

The reproducibility study for Gram-negative and Gram-positive aerobic bacteria, Gramnegative and Gram-positive microaerophilic bacteria, Gram-negative and Gram-positive anaerobic bacteria and yeasts was carried out to confirm day-to-day reproducibility and precision of the MALDI Biotyper CA System at different clinical study sites. The study was conducted for five (5) days with two (2) runs [two (2) operators] each day per clinical site. The sources of variability tested were:

ÔÇ∑ Two (2) operators/each clinical study site ÔÇ∑ Three (3) clinical study sites ÔÇ∑ At least two (2) target plates/each clinical study site Four (4) microflex LT/SH instruments

Ten (10) well-characterized organisms were chosen for this study and tested in duplicate via Direct Transfer procedure in accordance with product instructions. When the DT log(score) was $< 2 . 0 0$ , per product instructions, the test organism was tested following the extended Direct Transfer and Extraction procedure.

Note: As REPRO-2 was identified as Listeria monocytogenes after applying DT procedure test sites should have proceeded with testing of eDT and Ext procedure in parallel in accordance with the study protocol. Test sites did not conduct additional testing, therefore REPRO-02 was excluded from data analysis.

Table 2: Reproducibility Study Summary   

<table><tr><td rowspan=1 colspan=1>Blinded Test Organism</td><td rowspan=1 colspan=1>ReproducibilityPanel</td><td rowspan=1 colspan=1>Site A:MBT-CA ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>Site B:MBT-CA ID(DT+eDT+Ext)</td><td rowspan=1 colspan=1>Site C:MBT-CA ID(DT+eDT+Ext)</td></tr><tr><td rowspan=1 colspan=1>Dermabacter hominis</td><td rowspan=1 colspan=1>REPRO-01</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>REPRO-02*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Nocardia farcinica group</td><td rowspan=1 colspan=1>REPRO-03</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>REPRO-04</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Clostridium tertium</td><td rowspan=1 colspan=1>REPRO-05</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Facklamia hominis</td><td rowspan=1 colspan=1>REPRO-06</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides caccae</td><td rowspan=1 colspan=1>REPRO-07</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Trueperella bernardiae</td><td rowspan=1 colspan=1>REPRO-08</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>REPRO-09</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>REPRO-10</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>179/180 (99%)</td><td rowspan=1 colspan=1>179/180 (99%)</td><td rowspan=1 colspan=1>179/180 (99%)</td></tr></table>

\*REPRO-02: Organism skipped as MBT-CA ID applying DT procedure was not confirmed via eDT and Ext procedure.

Conclusion Reproducibility: $9 9 \%$ of test organisms were correctly identified with a log(score) $\ge 2 . 0 0$ at each clinical test site. In addition, no isolates were falsely identified. Thus, data confirm reproducibility and precision of the whole MALDI Biotyper CA System independent from:

ÔÇ∑ Clinical Site   
ÔÇ∑ System operators   
ÔÇ∑ microflex LT/SH instruments Target plate

# Challenge Panel:

A panel of 46 organisms (12 Gram-positive aerobic bacteria, 8 Gram-negative aerobic bacteria, 4 Gram-positive microaerophilic bacteria, 5 Gram-negative microaerophilic bacteria, 10 Gram-positive anaerobic bacteria, 4 Gram-negative anaerobic bacteria, 3 yeasts) was tested at three (3) study sites. All of the forty-six (46) organisms included in the panel were selected from stored organisms tested during the clinical study. The study interim reference laboratory prepared the panel. Organism identifications were blinded to test sites. Each site tested the challenge panel member via Direct Transfer, extended Direct Transfer and Extraction procedure in parallel.

Table 3: Challenge Panel Study Summary   

<table><tr><td rowspan=1 colspan=1>Test procedure</td><td rowspan=1 colspan=1>Site A*</td><td rowspan=1 colspan=1>Site B**</td><td rowspan=1 colspan=1>Site C***</td></tr><tr><td rowspan=1 colspan=1>DT method</td><td rowspan=1 colspan=1>43/45 ( 96%)</td><td rowspan=1 colspan=1>38/44 (86%)</td><td rowspan=1 colspan=1>36/41 (88%)</td></tr><tr><td rowspan=1 colspan=1>eDT method</td><td rowspan=1 colspan=1>44/45 ( 98%)</td><td rowspan=1 colspan=1>36/44 (82%)</td><td rowspan=1 colspan=1>38/41 (93%)</td></tr><tr><td rowspan=1 colspan=1>Ext method</td><td rowspan=1 colspan=1>43/45 ( 96%)</td><td rowspan=1 colspan=1>36/44 (82%)</td><td rowspan=1 colspan=1>39/41 (95%)</td></tr><tr><td rowspan=1 colspan=1>MBT-CA workflow</td><td rowspan=1 colspan=1>45/45 (100%)</td><td rowspan=1 colspan=1>40/44 (91%)</td><td rowspan=1 colspan=1>40/41 (98%)</td></tr></table>

\* One (1) sample was not identified due to isolate failure to grow. $^ { * * }$ Two (2) samples were not identified due to isolate failure to grow. $* * *$ Five (5) samples were not identified due to isolate failure to grow.

Conclusion Challenge Panel:

Testing of the challenge panel confirms reliability of the MBT-CA System workflow independently from

ÔÇ∑ Clinical Site   
ÔÇ∑ Test-operator   
ÔÇ∑ microflex LT/SH mass spectrometer system   
ÔÇ∑ US IVD 48 Spot Target plate

# Biological / Technical Equivalency Study - MBT-CA smart System

Objective of this validation was to demonstrate equivalence of the MBT-CA output when using MALDI-TOF mass spectrometers equipped with smartbeam laser technology and nitrogen laser technology.

A panel of thirty four (34) species which are part of the MBT-CA library were measured. The 34 species represent Gram negative, Gram positive, and yeast. All three sample preparation techniques (DT, eDT, Ext) were used in parallel (eDT and Ext were always additionally performed independent from the DT result). Each sample preparation technique (DT, eDT, Ext) was spotted 8 times onto the MALDI target. The measurement was performed on two nitrogen laser instruments and 3 smart laser instruments.

Overall 4080 spectra were represented in this study. The performance of all spectra for each single species, each single sample prep (DT, eDT and Ext), each MBT-CA result (no ID ‚Äúred‚Äù, low confidence ID ‚Äúyellow‚Äù, high confidence ID ‚Äúgreen‚Äù) for each instrument type is summarised in Table 4.

Table 4: Overall performance of 4080 measured samples. ‚ÄúHigh confidence $| \mathsf { D } ^ { \prime }$ (green), ‚Äúlow confidence $| \mathsf { D } ^ { \prime }$ (yellow) and ‚Äúno ID‚Äù (ref) are shown. The table (left) shows the total numbers and table (right) the percentage.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Nitrogen (544)</td><td rowspan=1 colspan=3>Smart (816)</td><td rowspan=1 colspan=1>‚Ä≤</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Nitrogen (544)</td><td rowspan=1 colspan=3>Smart (816)</td></tr><tr><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>399</td><td rowspan=1 colspan=1>‚Ä≤</td><td rowspan=1 colspan=1>DT</td><td rowspan=1 colspan=1>34.4%</td><td rowspan=1 colspan=1>13.2%</td><td rowspan=1 colspan=1>52.4%</td><td rowspan=1 colspan=1>36.8%</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>48.9%</td></tr><tr><td rowspan=1 colspan=1>eDT</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>446</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>633</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>eDT</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>11.2%</td><td rowspan=1 colspan=1>82.0%</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>14.7%</td><td rowspan=1 colspan=1>77.6%</td></tr><tr><td rowspan=1 colspan=1>Ext</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>803</td><td rowspan=1 colspan=1>‚Ä≤</td><td rowspan=1 colspan=1>Ext</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>98.4%</td></tr><tr><td rowspan=2 colspan=1>‚àë</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>1270</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>1835</td><td rowspan=1 colspan=2>‚àë</td><td rowspan=1 colspan=1>13.8%</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>77.8%</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>75.0%</td></tr></table>

As an example, the row ‚ÄúExt‚Äù shows a similar percentage of ‚Äúhigh confidence results‚Äù between both systems ( $9 9 . 1 \%$ for Nitrogen Laser versus $9 8 . 4 \%$ for Smartbeam Laser).

Conclusion: Based on these results the equivalence between the Nitrogen Laser System and the Smartbeam Laser System could be shown.

# Equivalence Study MBT Biotarget 96 US IVD (Nitrogen Laser)

This study was designed to verify and validate the use of the MBT Biotarget 96 US IVD in conjunction with the MBT-CA System (Nitrogen Laser). Study goal was to show the equivalency between the already cleared US IVD MSP 48 Target Polished Steel and the new MBT Biotarget 96 US IVD. The functionality and performance of the MBT-CA System using the MBT Biotarget 96 US IVD under varying conditions was shown. Test runs were performed always in parallel using US IVD MSP 48 Target Polished Steel plates.

The following six studies were performed:

Repeatability / Precision   
ÔÇ∑ Tolerance range (dynamic range) - limit of detection (LOD)   
ÔÇ∑ Sample stability prior to matrix application   
ÔÇ∑ Sample stability post matrix application Mass Accuracy / Target Edge Effects Identification of 50 FDA cleared organisms according to MBT-CA System workflow.

Repeatability and Precision

In this study, approximately one thousand (1000) measurements were carried out. In total, seven hundred and twenty (720) MBT-CA System identifications (10 species, triplicates, two target plates, 1 operator per day, 2 runs per day, 6 days) were performed. $100 \%$ of test organisms were correctly MBT-CA identified at high confidence level $[ \mathsf { l o g } ( \mathsf { s c o r e } ) \mathsf { \geq } 2 . 0 ]$ applying the MBT-CA System workflow (combination of DT, eDT and Ext procedure). The rate of MBT-CA System false identification was $0 \%$ for all test samples after final sample preparation procedure.

Both target types performed similarly.

Tolerance Range (Dynamic Range) - Limit of Detection (LOD)   
In this study, approximately one thousand five hundred (1500) measurements were performed. 10 different species were analysed representing Gram positive, Gram negative bacteria and yeast. All sample preparation techniques (DT, eDT and Ext) were conducted. Up to 5 dilutions steps were performed for each scenario. The rate of MBT-CA System false identifications at least at low confidence level $[ | 0 \boldsymbol { \mathrm { g } } ( s \mathsf { c o r e } ) \ge 1 . 7 0 ]$ was $0 \%$ . The estimated dynamic range (limit of detection) of bacteria and yeast onto the MBT Biotarget 96 US IVD / US IVD MSP 48 Target Polished Steel required for MBT-CA System identification did not show significant differences.

As a common limit of detection the amount of about $5 ^ { * } 1 0 ^ { 5 }$ cells (or cell equivalents using extraction) which must be placed on a target spot could be defined. Since a sharp boundary is hard to define, some exceptions occurred during this study. But these exceptions occurred for different species, different sample preparation techniques and different target types. Therefore there is no general trend for a better limit of detection for one kind of target.

Therefore the equivalence between the MBT Biotarget 96 US IVD / US IVD MSP 48 Target Polished Steel could be shown.

Sample Stability prior to Matrix application

In this study, approximately two thousand five hundred (2500) measurements were carried out. Five species were analyzed on two instruments. Measurement was applied directly after matrix application and after 15, 30, 60 and 120 minutes. All sample preparation techniques (DT, eDT and Ext) were conducted. Each sample was spotted and measured 8 times. The rate of MBT-CA System false identifications was $0 \%$ for all tests.

The performance was similar for both target types.

Sample Stability post Matrix application

In this study, approximately three thousand (3000) measurements were performed. Two temperature conditions were analysed in this study $2 1 ^ { \circ } \mathsf { C }$ and $2 5 ^ { \circ } \mathsf { C }$ ). All measurements were conducted using 5 different species on two MALDI instruments. All sample preparation techniques (DT, eDT and Ext) were performed and each sample was spotted 8 times on both target types. The ready prepared sample stability was analyzed immediately after matrix application (standard condition) and after 4h, 8h and 24h of aging.

No MBT-CA false identifications occurred. The performance of the MBT Biotarget 96 US IVD as compared to that of the US IVD MSP 48 Target Polished Steel could be shown.

Validation Study - Validation of 50 Representative Claimed Species   
Approximately three hundred (300) measurements were carried out. In this study 50 claimed species representing Gram negative aerobic / anaerobic, Gram positive aerobic / anaerobic bacteria and yeast were analysed. All sample preparation techniques (DT, eDT and Ext) were conducted. The rate of MBT-CA false identifications was $0 \%$ for this test. The overall identification rate of test samples at high confidence level (log(score) ${ \geq } 2 . 0 $ ) was $100 \%$ .

General performance of the MBT Biotarget 96 US IVD and the US IVD MSP 48 Target Polished Steel was comparable.

Technical Study - Mass Accuracy / Target Edge Effects   
In this study, approximately one thousand five hundred (1500) measurements were carried out. This study was designed for validation of the MBT Biotarget 96 US IVD in terms of "mass accuracy / target flatness‚Äù using BTS, MBT-CA System software and reference library and at least two (2) microflex LT/SH mass spectrometer systems. Validation was performed on five (5) MBT Biotarget 96 US IVD and five (5) US IVD MSP 48 Target Polished Steel. Targets were entirely prepared with dissolved BTS, dried and then overlaid with HCCA matrix.

Two parameters were evaluated during this study: Target Flatness (the mass accuracy was used as an indicator for target flatness) and edge effects (comparison of mass accuracy from edge positions vs inner target positions). For evaluation, one prominent reference protein (e.g. $6 2 5 5 . 4 \ : \mathrm { m } / z $ ) within the mass range of the BTS was analyzed concerning mass reproducibility. Additionally the log(scores) of each BTS spectrum were calculated as an indicator of identification performance.

Generally it can be stated that the technical performance between the MBT Biotarget 96 US IVD compared to the US IVD MSP 48 Target Polished Steel was comparable.

# Equivalence Study MBT Biotarget 96 US IVD (Smartbeam Laser)

The study design of this study was identical to the previous chapter [Equivalence Study MBT Biotarget 96 US IVD (Nitrogen Laser)]. In contrast to the 10,000 measurements in the previous chapter using two nitrogen laser systems, in this study an additional 5000 measurements were performed on one MBT smart CA System (Smartbeam Laser) to further prove the equivalence of both target types.

The following six studies were performed:

Repeatability / Precision Tolerance range (dynamic range) - limit of detection (LOD)   
ÔÇ∑ Sample stability prior to matrix application   
ÔÇ∑ Sample stability post matrix application Mass Accuracy / Target Edge Effects Identification of 50 FDA cleared organisms according to MBT-CA System workflow.

Generally it can be stated that the technical performance and equivalence between the MBT Biotarget 96 US IVD compared to the US IVD MSP 48 Target Polished Steel could be shown again in this study.

# Nocardia Study

This study was designed to verify and validate the measurement and identification of different strains and different species of the genus Nocardia by using the MBT-CA System as well as the MBT smart CA System in parallel. In addition the study was performed using the US IVD MSP 48 Target Polished Steel and the MBT Biotarget 96 US IVD in parallel. The results of this study can be evaluated as analytical study for identification performance of the MBT-CA System for the genus Nocardia and additionally as demonstration of equivalence between the ‚ÄúMBT-CA and MBT smart CA Systems‚Äù as well as equivalence of the ‚ÄúMBT Biotarget 96 US IVD and US IVD MSP 48 Target Polished Steel‚Äù.

# Analytical Studies:

ÔÇ∑ Measurement of 30 strains covering 6 Nocardia species.   
ÔÇ∑ DT, eDT and Ext was used for sample preparation.   
ÔÇ∑ Each sample preparation technique was spotted 8 times on the targets.   
ÔÇ∑ Each spot was measured twice.   
ÔÇ∑ MALDI targets ‚ÄúUS IVD MSP 48 Target Polished Steel‚Äù and ‚ÄúMBT Biotarget 96 US IVD‚Äù were used.   
ÔÇ∑ Measurement on MBT smart CA System and MBT-CA System. Nocardia were measured after 24h, 48h and after 5 days of cultivation.

Altogether about 15,000 measurements were performed (30 strains \* 3 sample preparations $^ \ast 8$ target spots $^ \ast 2$ measurements per spot \* 2 target types $^ \ast 2$ instrument types \* three cultivation times ). The identification performance of Nocardia using the standard identification MBT-CA workflow was excellent. The general log(score) distribution was similar and the the equivalence for all examined components (MBT-CA System, MBT smart CA System, MBT Biotarget 96 US IVD, and US IVD MSP 48 Target Polished Steel) was shown successfully.

As an example all 5760 log(scores) after 48h cultivation are shown in the following diagram:

![](images/b4d9bb1506fa6a0778b34491c18364d1fa88a67644cbc681cb3feb8c136ea657.jpg)

Figure 1: 5760 spectra derived log(scores) after 48h cultivation overlaid. Each ‚Äúlog(score) lane‚Äù represents 480 single log(scores). The ‚Äúfocal points‚Äù are very similar.

# Conclusion:

Studies found that the MBT smart System and the MBT System, as well as the MBT Biotarget 96 US IVD and US IVD MSP 48 Target Polished Steel had comparable performance.

# Method Comparison:

To demonstrate performance of the MALDI Biotyper CA (MBT-CA) System, a method comparison study was performed at four (4) US clinical test sites and in-house laboratory. 2,091 fresh and stored organisms were tested on the MALDI Biotyper CA System in accordance with manufacturer‚Äôs instructions for use. All organisms included in the study were sub-cultured for purity. Testing on the MBT-CA System was done from a freshly isolated colony.

All organisms included in the study and tested by US study sites were also sub-cultured onto an agar slant or appropriate media for isolation and shipped to the study interim reference laboratory. The interim reference laboratory stored all organisms included in the study and when requested, sent organisms to the sequencing reference laboratory for 16S rRNA or ITS sequencing and protein gene sequencing when requested. In-house laboratory and some testing sites processed their own isolates to the sequencing reference laboratory.

The following Gram-negative, Gram-positive and yeast species are additionally included in the reference library (please refer to K130831 and K142677 for previously claimed organisms).

Table 5: Claimed Organisms   

<table><tr><td colspan="1" rowspan="1">Organisms</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter_baumannii_nosocomialis group</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter pittii</td></tr><tr><td colspan="1" rowspan="1">Actinomyces europaeus</td></tr><tr><td colspan="1" rowspan="1">Actinomyces funkei</td></tr><tr><td colspan="1" rowspan="1">Actinomyces graevenitzii</td></tr><tr><td colspan="1" rowspan="1">Actinomyces hyovaginalis</td></tr><tr><td colspan="1" rowspan="1">Actinomyces radingae</td></tr><tr><td colspan="1" rowspan="1">Actinomyces turicensis</td></tr><tr><td colspan="1" rowspan="1">Actinomyces urogenitalis</td></tr><tr><td colspan="1" rowspan="1">Actinotignum_ schaalii group</td></tr><tr><td colspan="1" rowspan="1">Aerococcus sanguinicola</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter actinomycetemcomitans</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter segnis</td></tr><tr><td colspan="1" rowspan="1">Alloiococcus otitis</td></tr><tr><td colspan="1" rowspan="1">Alloscardovia omnicolens</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus murdochii</td></tr><tr><td colspan="1" rowspan="1">Arthrobacter cumminsii</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td></tr><tr><td colspan="1" rowspan="1">Bacteroides nordii</td></tr><tr><td colspan="1" rowspan="1">Bacteroides pyogenes</td></tr><tr><td colspan="1" rowspan="1">Bacteroides salyersiae</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris group</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium breve</td></tr><tr><td colspan="1" rowspan="1">Candida intermedia</td></tr><tr><td colspan="1" rowspan="1">Candida zeylanoides</td></tr><tr><td colspan="1" rowspan="1">Clostridium beijerinckii</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td></tr><tr><td colspan="1" rowspan="1">Clostridium_clostridioforme group</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td></tr><tr><td colspan="1" rowspan="1">Clostridium paraputrificum</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes/ Clostridium botulinum (group I)</td></tr><tr><td colspan="1" rowspan="1">Clostridium tertium</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium accolens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium_afermentans group</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium coyleae</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium freneyi</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium glutamicum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium_mucifaciens_ureicelerivorans group</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium resistens</td></tr><tr><td colspan="1" rowspan="1">Cyberlindnera jadinii</td></tr><tr><td colspan="1" rowspan="1">Dermabacter hominis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td></tr><tr><td colspan="1" rowspan="1">Enterococcus mundtii</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td></tr><tr><td colspan="1" rowspan="1">Ewingella americana</td></tr><tr><td colspan="1" rowspan="1">Facklamia hominis</td></tr><tr><td colspan="1" rowspan="1">Fluoribacter bozemanae</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td></tr><tr><td colspan="1" rowspan="1">Helcococcus kunzii</td></tr><tr><td colspan="1" rowspan="1">Kingella denitrificans</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus jensenii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc citreum</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc pseudomesenteroides</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Malassezia furfur</td></tr><tr><td colspan="1" rowspan="1">Malassezia pachydermatis</td></tr><tr><td colspan="1" rowspan="1">Mannheimia_haemolytica group</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisii</td></tr><tr><td colspan="1" rowspan="1">Neisseria bacilliformis</td></tr><tr><td colspan="1" rowspan="1">Neisseria cinerea</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td></tr><tr><td colspan="1" rowspan="1">Neisseria_flavescens_subflava group</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1">Neisseria_sicca group</td></tr><tr><td colspan="1" rowspan="1">Neisseria weaveri</td></tr><tr><td colspan="1" rowspan="1">Nocardia brasiliensis</td></tr><tr><td colspan="1" rowspan="1">Nocardia cyriacigeorgica</td></tr><tr><td colspan="1" rowspan="1">Nocardia_farcinica group</td></tr><tr><td colspan="1" rowspan="1">Nocardia nova</td></tr><tr><td colspan="1" rowspan="1">Nocardia otitidiscaviarum</td></tr><tr><td colspan="1" rowspan="1">Ochrobactrum anthropi</td></tr><tr><td colspan="1" rowspan="1">Parabacteroides goldsteinii</td></tr><tr><td colspan="1" rowspan="1">Parabacteroides_johnsonii_merdae group</td></tr><tr><td colspan="1" rowspan="1">Parvimonas micra</td></tr><tr><td colspan="1" rowspan="1">Pediococcus acidilactici</td></tr><tr><td colspan="1" rowspan="1">Pluralibacter gergoviae</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas somerae</td></tr><tr><td colspan="1" rowspan="1">Ralstonia pickettii</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td></tr><tr><td colspan="1" rowspan="1">Serratia odorifera</td></tr><tr><td colspan="1" rowspan="1">Sphingobacterium multivorum</td></tr><tr><td colspan="1" rowspan="1">Sphingobacterium spiritivorum</td></tr><tr><td colspan="1" rowspan="1">Sphingomonas paucimobilis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus delphini</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lentus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus sciuri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus xylosus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus canis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equi</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus_salivarius_vestibularis group</td></tr><tr><td colspan="1" rowspan="1">Streptococcus thermophilus</td></tr><tr><td colspan="1" rowspan="1">Trichosporon inkin</td></tr><tr><td colspan="1" rowspan="1">Trichosporon_mucoides group</td></tr><tr><td colspan="1" rowspan="1">Trueperella bernardiae</td></tr><tr><td colspan="1" rowspan="1">Turicella otitidis</td></tr><tr><td colspan="1" rowspan="1">Vagococcus fluvialis</td></tr><tr><td colspan="1" rowspan="1">Veillonella_parvula group</td></tr><tr><td colspan="1" rowspan="1">Weeksella virosa</td></tr><tr><td colspan="1" rowspan="1">Yersinia frederiksenii</td></tr><tr><td colspan="1" rowspan="1">Yersinia intermedia</td></tr><tr><td colspan="1" rowspan="1">Yersinia kristensenii</td></tr></table>

<table><tr><td></td></tr><tr><td></td></tr><tr><td></td></tr><tr><td>THIS SPACE INTENTIONALLY LEFT BLANK</td></tr><tr><td></td></tr><tr><td></td></tr><tr><td></td></tr><tr><td></td></tr></table>

Table 6 to Table 8 below show the overall isolate performance.

Table 6: Overall Isolate Performance   

<table><tr><td rowspan=1 colspan=5>Overall Performance - CLAIM 3</td></tr><tr><td rowspan=1 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Organism ID ‚â• 2.0(High Confidence)</td><td rowspan=1 colspan=1>1904</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2055</td></tr><tr><td rowspan=1 colspan=1>Organism ID (‚â•1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (‚â•1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1930</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2091</td></tr></table>

<table><tr><td colspan="2">Positive</td><td rowspan="2">Negative</td></tr><tr><td></td><td>high &amp; low</td></tr><tr><td>high resolution</td><td>resolution</td><td></td></tr><tr><td>98.45%</td><td>99.81%</td><td>n/a</td></tr></table>

Table 7: Overall Bacteria Performance   

<table><tr><td rowspan=1 colspan=5>Performance BACTERIA</td></tr><tr><td rowspan=1 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Organism ID ‚â• 2.0(High Confidence)</td><td rowspan=1 colspan=1>1827</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1978</td></tr><tr><td rowspan=1 colspan=1>Organism ID (‚â•1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (‚â•1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1850</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2011</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highresolution</td><td rowspan=1 colspan=1>high &amp; lowresolution</td></tr><tr><td rowspan=1 colspan=1>98.54%</td><td rowspan=1 colspan=1>99.85%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Table 8: Overall Yeast Performance   

<table><tr><td rowspan=1 colspan=5>Performance YEAST</td></tr><tr><td rowspan=1 colspan=1>MBT-CA RESULT</td><td rowspan=1 colspan=3>REFERENCE ALGORITHM</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>high resolutionspecies</td><td rowspan=1 colspan=1>low resolutionspecies / genus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Organism ID ‚â• 2.0(High Confidence)</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Organism ID (‚â•1.7; &lt;2.0)(Low Confidence)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>- INCORRECT MBT-CA ID (‚â•1.7)- NO ID (&lt;1.7)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>80</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>highresolution</td><td rowspan=1 colspan=1>high &amp; lowresolution</td></tr><tr><td rowspan=1 colspan=1>96.25%</td><td rowspan=1 colspan=1>98.75%</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# Statement of Safety and Efficacy

The data presented clearly demonstrate the safety and efficacy of the MBT-CA System as compared to the reference algorithm, when the instructions for use are followed.