# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K163085   
B. Purpose for Submission: To obtain a substantial equivalence determination for a new device   
C. Measurand: tcdB gene of toxigenic Clostridium difficile   
D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) assay   
E. Applicant: Focus Diagnostics, Inc.: DBA DiaSorin Molecular LLC   
F. Proprietary and Established Names: SimplexaTM C. difficile Direct SimplexaTM C. difficile Positive Control Pack

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130, Clostridium difficile toxin gene amplification assay 21 CFR 862.2570 - Instrumentation for clinical multiplex test systems   
2. Classification: II

3. Product code: OZN, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

Simplexa™ C. difficile Direct   
The Focus Diagnostics Simplexa™ C. difficile Direct assay is intended for use on the Integrated Cycler instrument for the detection of Clostridium difficile toxin B gene (tcdB) present in liquid or unformed stool samples from individuals suspected of C. difficile infection (CDI). This test aids in the diagnosis of illness resulting from CDI.   
The assay is for professional and prescription use only. Simplexa™ C. difficile Positive Control Pack   
The Simplexa™ C. difficile Positive Control Pack is intended to be used as a control with the Simplexa™ C. difficile Direct kit.   
This control is not intended for use with other assays or systems.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: 3M Integrated Cycler with Integrated Cycler Studio Software

# I. Device Description:

The Simplexa C. difficile Direct assay system is a real-time PCR system that enables the direct amplification and detection of toxigenic C. difficile DNA from unprocessed liquid or unformed stool specimens that have not undergone nucleic acid extraction. The system consists of the Simplexa C. difficile Direct assay, the Integrated Cycler (with Integrated Cycler Studio Software), the Direct Amplification Disc and associated accessories.

Materials provided:

Simplexa C. difficile Direct Reaction Mix

Materials provided separately:

Direct Amplification Disc Kit Simplexa C. difficile Sample Prep Kit

Materials required but not supplied:

· Integrated Cycler with Integrated Cycler Studio Software version 6.0 or higher   
· Simplexa C. difficile Positive Control Pack   
· $5 0 \mu \mathrm { L }$ fixed volume pipette   
· Sterile, nuclease-free disposable pipette tips with filters   
· Freezer (manual defrost) at $^ { - 1 0 }$ to ${ } _ { - 3 0 } \mathrm { { } ^ { \circ } C }$ (for kit component) Refrigerator at 2 to $8 ~ ^ { \circ } \mathrm { C }$ (for specimens) Disposable, powder-free gloves

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX Cdiff

2. Predicate 510(k) number(s): K130470

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceSimplexa C. difficile Direct</td><td colspan="1" rowspan="1">PredicateBD MAX Cdiff (K130470)</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">Detection of C. difficiletoxin gene sequences inliquid or unformed stoolsamples from individualssuspected of C. difficileinfection (CDI). Intended toaid in the diagnosis of CDI.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay target</td><td colspan="1" rowspan="1">C. difficile toxin B gene(tcdB)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Liquid or unformed stool</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay technology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample extraction</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test interpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Qualitative/Quantitative</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceSimplexa C. difficile Direct</td><td colspan="1" rowspan="1">PredicateBD MAX Cdiff (K130470)</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Real-time PCR withbifunctional fluorescentprimer-probes</td><td colspan="1" rowspan="1">Real-time PCR usingfluorogenic target-specifichybridization probes</td></tr><tr><td colspan="1" rowspan="1">Instrument platform</td><td colspan="1" rowspan="1">3M Integrated Cycler</td><td colspan="1" rowspan="1">BD MAX System</td></tr><tr><td colspan="1" rowspan="1">Samples/controls per run</td><td colspan="1" rowspan="1">Eight</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Positive control</td><td colspan="1" rowspan="1">Positive Control Packprovided separately</td><td colspan="1" rowspan="1">User provided</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff: Administrative Procedures for CLIA Categorization, March 12, 2014

Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k), August 12, 2005

Off-The-Shelf Software Use in Medical Devices, September 9, 1999

Cybersecurity for Networked Medical Devices Containing Off-The-Shelf (OTS) Software, January 14, 2005

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005

General Principles of Software Validation, January 11, 2002

# L. Test Principle:

The Simplexa C. difficile Direct assay system is a real-time PCR system that enables the direct amplification and detection of toxigenic C. difficile DNA from unprocessed liquid or unformed stool specimens that have not undergone nucleic acid extraction. In the Simplexa C. difficile Direct assay, bi-functional fluorescent probe primers are used together with corresponding reverse primers to amplify C. difficile and DNA internal control targets. The assay targets a sequence which is in a well conserved region of C. difficile toxin B gene (tcdB). A DNA internal control is used to detect PCR failure and/or inhibition.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility

Reproducibility testing for the Simplexa C. difficile Direct was performed at three sites with a coded and randomized sample panel that included a positive control, a negative control (TE-stool matrix without analyte) and four contrived samples spiked into a negative stool - TE buffer matrix. As shown in Table 1, the four contrived positive samples consisted of a low positive (LP) and a medium positive (MP) for each of two strains of toxigenic C. difficile (ATCC 43255 and NAP1A). Each contrived sample was prepared by spiking a specific organism concentration into negative stool - TE matrix. Both positive and negative controls were also tested during this study.

Table 1. Reproducibility Sample Panel   

<table><tr><td rowspan=1 colspan=1>Sample Panel Member</td><td rowspan=1 colspan=1>Strain at Spiked Concentration</td></tr><tr><td rowspan=1 colspan=1>ATCC43255 - LP</td><td rowspan=1 colspan=1>ATCC43255 at 0.95 CFU/mL (1 X LoD)</td></tr><tr><td rowspan=1 colspan=1>ATCC43255 - MP</td><td rowspan=1 colspan=1>ATCC43255 at 3.8 CFU/mL (4 X LoD)</td></tr><tr><td rowspan=1 colspan=1>NAP1A-LP</td><td rowspan=1 colspan=1>NAP1A at 0.43 CFU/mL (1 X LoD)</td></tr><tr><td rowspan=1 colspan=1>NAP1A - MP</td><td rowspan=1 colspan=1>NAP1A at 1.7CFU/mL (4 X LoD)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Each sample panel member was coded, randomized, and tested in triplicate per run for two runs per day for a total of five non-consecutive days per site. Two runs per day were performed by each operator. Therefore, a total of 90 replicates (three replicates x two runs x five days x three sites) were tested for each sample panel member. Two Integrated Cycler instruments were used at each site for a total of six instruments overall.

One negative control replicate and one NAP1A sample replicate produced invalid results with the test device due to insufficient specimen volume upon initial testing. Both samples were retested according to the labelled instructions for use, and both samples produced the expected result upon retest.

Test results were acceptable and did not vary between sites, operators, days, runs, or instruments. Combined results for all sites are presented in the Table 2.

Table 2. Site-to-Site Reproducibility Results   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=1 colspan=1>ATCC43255- LP</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 - 100%</td></tr><tr><td rowspan=1 colspan=1>ATCC43255 - MP</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9 - 100%</td></tr><tr><td rowspan=1 colspan=1>NAP1A -LP</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>98.9%(89/90)</td><td rowspan=1 colspan=1>94.0 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>NAP1A-MP</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9 - 100%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9 - 100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9 - 100%</td></tr></table>

1Expected results of negative sample is “Negative” for C. difficile.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Controls:

The Simplexa C. difficile Positive Control Pack consists of inactivated C. difficile, and is intended to be used as an external control for the Simplexa C. difficile Direct kit. The positive control pack is provided separately. Focus recommends testing external controls once per day.

Sample Prep Buffer should be used as a No Template Control (NTC). A NTC should be tested for every control run in which a positive control is tested.

A DNA internal control (DNA IC) template with a specific fluorescent primers/probe set is included in each reaction mix and is used to detect PCR failure and/or inhibition. Failure of the DNA IC will cause the software to report an invalid result when toxigenic C. difficile DNA has not been detected in the sample. In case of an invalid result, the sample should be re-prepared and the assay repeated. A new Sample Prep Swab and Sample Prep Buffer should be used.

Expected control results are listed in Table 3. If the controls are not within these parameters, patient results should be considered invalid and the assay repeated.

Table 3. Expected Control Results   

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>C. difficile tcdBGene ExpectedResult</td><td rowspan=1 colspan=1>DNA IC</td></tr><tr><td rowspan=1 colspan=1>Simplexa C. difficileDirect Positive Control</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not applicable2</td></tr><tr><td rowspan=1 colspan=1>No Template Control</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Valid</td></tr></table>

1Typical Ct values for the Positive Control range between 28 to $< 3 4$ . 2 Detection of the Simplexa DNA IC is not required for a valid result when toxigenic $C .$ . difficile DNA is detected.

Positive Controls and No Template Controls were tested daily at each site during the prospective clinical study. A total of 458 Positive controls and $4 5 9 \mathrm { N o }$ Template Controls were tested across all sites and produced valid expected initial results for 456 $( 9 9 . 6 \% )$ Positive Controls and 451 $( 9 8 . 3 \% )$ No Template Controls. After QC retesting, $100 \%$ of the unexpected QC results resolved to the expected result.

d. Detection limit:

# Analytical sensitivity/ Limit of detection

The Limit of Detection (LoD) for the Simplexa C. difficile Direct was determined using a panel consisting of negative stool-TE matrix spiked with two strains of toxigenic $C .$ . difficile (ATCC 43255 and NAP1A). The bacterial stocks were serially diluted, and a total of 32 replicates were tested at each dilution. The lowest concentration of $C .$ . difficile where at least $9 5 \%$ of the replicates were detected was determined to be the LoD for each strain. The results are shown in Table 4 below.

Table 4. Simplexa $C .$ . difficile Direct LoD   

<table><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Strain</td><td rowspan=1 colspan=1>LoD Concentration (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>0.95</td></tr><tr><td rowspan=1 colspan=1>NAP1A</td><td rowspan=1 colspan=1>0.43</td></tr></table>

# Analytical reactivity

The Simplexa C. difficile Direct assay was evaluated for analytical reactivity to 32 strains of toxigenic $C .$ . difficile from a variety of toxinotypes.

Analytical reactivity samples were prepared by spiking regrown and retitered strains into negative stool-TE matrix at 4X LoD of the ATCC 43255 strain (approximately 4 $\mathrm { C F U / m L }$ ). Each sample was tested in triplicate. If at least one of three replicates produced a negative assay result, higher concentrations were tested until all three replicates were detected. Of the 32 tested strains, 22 strains were detected at 4 $\mathrm { C F U / m L }$ , eight strains were detected at $8 - 3 2 \mathrm { C F U / m L }$ , and two strains were

detected at 256 – 512 CFU/mL. The results are shown in Table 5.

Table 5. Inclusivity/Reactivity   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Conc(CFU/mL)</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>Toxin</td><td rowspan=1 colspan=1>Ribotype</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 43596</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>012</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 43598</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>A-B+</td><td rowspan=1 colspan=1>017</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 43600</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>014</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 51695</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 9689</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1382</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>012</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1805</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>027</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1870</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>027</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1871</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1873</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>053</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile CCUG 8864(CCUG 20309)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>A-B+</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile IS81</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>034</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R1880</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>086</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R7771</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>VII</td><td rowspan=1 colspan=1>A-B+</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R8366</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R8637</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>IX</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>019</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R9367</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>XIII</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>070</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R9385</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>XV</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>122</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R10456</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>058</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R10725</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>078</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R10842</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>045</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R10870</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>XIV</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile R12425</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 17858</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>054</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 43594</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>005</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 17857</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 700792</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>005</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1875</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>078</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile ATCC 43599</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>001</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1872</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1814</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>251</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile BAA-1874</td><td rowspan=1 colspan=1>512</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>A+B+</td><td rowspan=1 colspan=1>002</td></tr></table>

e. Analytical specificity:

Cross reactivity

The cross reactivity of the Simplexa $C .$ . difficile Direct assay was evaluated by wet testing 127 organisms that are closely related to $C .$ . difficile, or that cause similar clinical symptoms, or that are present as normal flora in the specimen type of interest. Samples were prepared by spiking each of the 127 organisms at clinically relevant concentrations into TE-stool matrix. Clinically relevant concentrations were defined $\mathrm { a s { \geq } 1 0 ^ { 6 } C F U / m L }$ for bacteria and fungi and $\ge \mathrm { 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$ $\mathrm { ( P F U / m L ) }$ ) for viruses, or other industry accepted units. Each sample was tested in triplicate. If at least one of the three replicates was detected as positive with the assay, an additional five replicates of the same cross reactant was tested. The organisms are listed in Table 6.

Cross reactivity was not observed. One replicate of Clostridium noyii produced a positive result; however, testing with an additional five replicates produced negative results.

Campylobacter jejuni was tested during validation. However, it is unknown if subspecies jejuni was tested. In silico NCBI BLAST analysis was performed for $C .$ . jejuni subsp jejuni did not predict cross reactivity with the Simplexa C. difficile Direct kit.

The organisms Clostridium botulinum, Desulfovibrio piger, and Ruminococcus bromii were not available for wet testing. These organisms were evaluated for potential cross reactivity with the assay primers and probes by in silico NCBI BLAST analysis of genomic DNA sequences. The in silico analysis did not predict cross reactivity with the Simplexa C. difficile Direct kit for these species.

Table 6. Cross reactivity   

<table><tr><td colspan="2" rowspan="1">Organisms</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">Enterococcus faecium vanA</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Enterococcus gallinarum vanC</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Enterococcus hirae</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">Enterococcus raffinosus</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Enterovirus 71</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. Faecalis</td><td colspan="1" rowspan="1">Escherichia coli ATCC 11775</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">Escherichia coli ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus ATCC 11778</td><td colspan="1" rowspan="1">Escherichia coli ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus ATCC 13472</td><td colspan="1" rowspan="1">Escherichia coli O157:H7(ATCC 700927)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">Escherichia fergusonii</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Escherichia hermannii</td></tr><tr><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">Flavonifractor plautii (deposited as Clostridiumorbiscindens)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">Fusobacterium varium</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">Gemella morbillorum</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Hafnia alvei</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Helicobacter fennelliae</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Helicobacter pylori</td></tr><tr><td colspan="1" rowspan="1">Candida catenulate</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp. Pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Lactococcus lactis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Leminorella grimontii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">Listeria grayi</td></tr><tr><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">Listeria innocua</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">Norovirus G2</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td></tr><tr><td colspan="1" rowspan="1">Clostridium chauvoei</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile ATCC 43593</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile ATCC 43601</td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td></tr><tr><td colspan="1" rowspan="1">Clostridium fallax</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">Proteus penneri</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">Providencia rettgeri</td></tr><tr><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Clostridium paraputrificum</td><td colspan="1" rowspan="1">Pseudomonas putida</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Rotavirus, strain Wa</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Arizonae</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Enterica serovarCholeraesuis</td></tr><tr><td colspan="1" rowspan="1">Clostridium sepiticum</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Enterica serovarTyphimimurium</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Clostridium sphenoides</td><td colspan="1" rowspan="1">Serratia marcescens subsp. marcescens</td></tr><tr><td colspan="1" rowspan="1">Clostridium spiroforme</td><td colspan="1" rowspan="1">Shigella boydii</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Clostridium symbiosum</td><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Clostridium tertium</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus AD-169</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">Streptococcus uberis</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Vibrio cholerae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">White Blood Cells (Human)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">Yersinia bercovieri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Yersinia rohdei</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1"></td></tr></table>

# Microbial interference:

The same 127 potentially interfering organisms that were tested for cross reactivity were also evaluated for potential interference with the detection of toxigenic $C .$ . difficile DNA. Two strains of toxigenic $C .$ . difficile (ATCC 42355 and NAP1A) were tested at two to four times LoD in triplicate in the presence of potentially interfering organisms. If at least one replicate was not detected, an additional five replicates were tested.

The Simplexa C. difficile Direct kit successfully detected toxigenic C. difficile DNA in the presence of these organisms at the concentrations tested. Strain NAP1A was detected in seven out of eight replicates of Clostridium septicum and Clostridium sordellii, and six out of eight replicates of Abiotrophia defective. All replicates of both strains NAP1A and ATCC 43255 were detected successfully in the presence of all other organisms tested.

# Interference:

A study was conducted to evaluate the effects of exogenous and endogenous substances that could potentially interfere with assay performance. A total of 33 potentially interfering substances were co-spiked into TE-stool matrix with toxigenic C. difficile strains ATCC 43255 or NAP1A spiked at two to four times the LoD. Endogenous interferents were spiked at the highest possible endogenous level, and exogenous interferents were spiked at approximately $10 \%$ of the recommended dose. The samples were tested in triplicate. If at least one replicate showed evidence of interference, an additional five replicates were tested. For substances that showed evidence of interference, successively lower concentrations were tested to determine the minimally interfering level. The results are summarized in Table 7 below.

Four substances produced evidence of interference at the highest concentrations tested. Colace tested at $1 0 \% \mathrm { w / v }$ and $5 \% \mathrm { w / v }$ produced invalid results in all replicates. When the Colace concentration was reduced to $2 . 5 \% \ \mathrm { w / v }$ , five out of eight $( 6 2 . 5 \% )$ replicates of C. difficile strain ATCC 42355 and seven out of eight $( 8 7 . 5 \% )$ replicates of $C .$ . difficile strain NAP1A were detected successfully. The remaining replicates were not detected.

Naproxen tested at $1 4 ~ \mathrm { m g / m L }$ produced false negative results in four out of five replicates of strains ATCC 42355 and NAP1A. All replicates of both strains were detected when the naproxen concentration was reduced to seven $\mathrm { m g / m L }$ .

All replicates of the ATCC 42355 strain were detected at a Dulcolax concentration of $10 \%$ w/v. However, Dulcolax at $1 0 \% \mathrm { w / v }$ , $5 \%$ w/v, $2 . 5 \% \mathrm { w } / \mathrm { v }$ , and $1 . 2 5 \%$ w/v produced invalid results for all replicates of strain NAP1A. Dulcolax at $0 . 6 2 5 \%$ w/v produced invalid results in four out of eight replicates and a Not Detected result in one replicate of strain NAP1. All replicates of strain NAP1 were detected when Dulcolax concentration was reduced to $0 . 3 1 2 5 \% \ \mathrm { w / v }$ .

All replicates of the ATCC 42355 strain were detected in the presence of $0 . 2 ~ \mathrm { m g / m L }$ Milk of Magnesia. Five out of eight replicates of strain NAP1were detected at this concentration of Milk of Magnesia.

Table 7. Interference   

<table><tr><td colspan="1" rowspan="2">InterferingSubstance</td><td colspan="1" rowspan="2">InterferingSubstanceConcentration</td><td colspan="1" rowspan="2">Active Ingredient</td><td colspan="2" rowspan="1">%Detection(#Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">ATCC 43255</td><td colspan="1" rowspan="1">NAP1A</td></tr><tr><td colspan="1" rowspan="1">Afrin</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Oxymetazolinehydrochloride 0 5%</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Antacid and Anti-gas generic</td><td colspan="1" rowspan="1">0.1 mg/mL</td><td colspan="1" rowspan="1">Aluminumhydroxide/Magnesiumhydroxide</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Anusol Plus(Pramoxine HCL)</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Pramoxinehydrochloride 1% andzinc sulfatemonohydrate 0.5%</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Barium Sulfate</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">Barium sulfate (98% forsuspension)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Benzalkonium(Towlettes)</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">Benzalkonium chloride,Ethanol</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">Glucose, Hormones,Enzymes, Ions, Iron,etc.</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Colace1</td><td colspan="1" rowspan="1">2.5 % w/v</td><td colspan="1" rowspan="1">Ducosate sodium USP100 mg</td><td colspan="1" rowspan="1">62.5%(5/8)</td><td colspan="1" rowspan="1">87.5%(7/8)</td></tr><tr><td colspan="1" rowspan="1">Dramamine</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Dimenhydrinate</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Dulcolax2</td><td colspan="1" rowspan="1">10% w/v forATCC 43255 &amp;0.3125% w/v forNAP1A</td><td colspan="1" rowspan="1">Bisacodyl USP 5 mg</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Gynol II,Nonoxynol 9</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Nonoxynol 9 (4 %)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Hydrocortisonecream</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Imodium</td><td colspan="1" rowspan="1">0 05 mg/mL</td><td colspan="1" rowspan="1">Loperamidehydrochloride</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">KY Jelly</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Mesalazine rectalsuspension (4 g/60 mL)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">14 mg/mL</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Milk of Magnesia</td><td colspan="1" rowspan="1">0.2 mg/mL</td><td colspan="1" rowspan="1">Magnesium hydroxide</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">62.5%(5/8)</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">Mineral oil</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Monistat 7</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Miconazole nitrate 2%(100 mg)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">3 mg/mL</td><td colspan="1" rowspan="1">Immunoglobulins,Lysozyme, Polymers,etc.</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Naproxen3</td><td colspan="1" rowspan="1">7 mg/mL</td><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10000 USPunits/ml</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Palmitic Acid</td><td colspan="1" rowspan="1">2 mg/mL</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol</td><td colspan="1" rowspan="1">0.175 mg/mL</td><td colspan="1" rowspan="1">Bismuth subsalicylate</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.1 mg/mL</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">SPF 30 Sunscreen</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">Avobenzone 3%,homosalate 8%,octisalate 5%,octocrylene 4%, andoxybenzone 4%</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">SPF 50 Sunscreen</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">Avobenzone 3%,homosalate 13%,octisalate 5%,octocrylene 7%, andoxybenzone 4%</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Stearic Acid</td><td colspan="1" rowspan="1">4 mg/mL</td><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Tums</td><td colspan="1" rowspan="1">0.1 mg/mL</td><td colspan="1" rowspan="1">Calcium Carbonate</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Vagisil</td><td colspan="1" rowspan="1">10% w/v</td><td colspan="1" rowspan="1">Benzocaine USP (20% -External analgesic),Resorcinol (3% -External analgesic)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">1.4 mg/mL</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">10 %w/v</td><td colspan="1" rowspan="1">White petrolatum USP(100%)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Witch hazel</td><td colspan="1" rowspan="1">Liquid from 1wipe</td><td colspan="1" rowspan="1">Witch hazel</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="5" rowspan="1">Colace also tested at two higher concentrations (10 % and 5 %w/v) produced "Invalid" results inall replicates.2 Dulcolax also tested at 10 %, 5 % 2.5 %, and 1.25 %w/v produced "Invalid" results for allreplicates of strain NAP1A. Dulcolax at 0.625% w/v produced "Invalid" results in four out ofeight replicates and a "Not Detected" result in one replicate of strain NAP1A.3 Naproxen also tested at 14 mg/mL produced "Not Detected" results in four out of five replicatesof strains NAP1A and ATCC 43255.</td></tr></table>

f. Assay cut-off:

The initial assay cut-off was established during a period of feasibility testing. The assay cut-off was confirmed by analysis of the results from the limit of detection and the prospective clinical studies.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the Simplexa C. difficile Direct test was established in a prospective multi-site investigation. Two thousand three hundred and fifty one (2351) samples were prospectively collected from five geographically diverse sites between December 3, 2015 and June 10, 2016 from patients with signs and symptoms of $C .$ . difficile infection. Each site enrolled fresh unformed stool specimens that were left over from routine clinical C. difficile infection diagnostic testing. Two thousand three hundred and thirty (2330) were evaluable on the Simplexa $C .$ . difficile Direct and the Direct Culture method. Two thousand three hundred and thirty six (2336) were evaluable on the Simplexa C. difficile Direct and the Combined Direct and Enriched Culture methods.

Of the 2336 samples, 1233 $( 5 2 . 8 \% )$ were collected from females and 1103 $( 4 7 . 2 \% )$ were collected from males. The age distribution of the subjects is shown in Table 8.

Table 8. Age distribution   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Less than 6</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>6 Yrs to 21 Yrs</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>22 Yrs to 59 Yrs</td><td rowspan=1 colspan=1>1073</td><td rowspan=1 colspan=1>45.9</td></tr><tr><td rowspan=1 colspan=1>More than 60 Yrs</td><td rowspan=1 colspan=1>1125</td><td rowspan=1 colspan=1>48.2</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>2336</td><td rowspan=1 colspan=1>100</td></tr></table>

Samples were tested with the Simplexa $C .$ difficile Direct at the collection sites, and the comparator culture methods were performed at one central laboratory. Discrepant analysis was performed using FDA cleared nucleic acid amplification test (NAAT) results provided by the clinical sites.

The initial invalid rate of the clinical prospective study using Simplexa™ C. difficile Direct was $0 . 6 4 \%$ (15/2351 samples). After repeat testing, 3/15 samples remained invalid, thus resulting in a final invalid rate of $0 . 1 3 \%$ (3/2351 samples).

Comparison with combined direct and enriched culture

The performance of the Simplexa $C .$ difficile Direct test compared with the combined direct and enriched culture result is shown in Table 9.

Table 9. Comparison with combined direct and enriched culture   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Combined Direct &amp; Enriched Culture</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Direct</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>336</td><td rowspan=1 colspan=1>96a</td><td rowspan=1 colspan=1>432</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>55b</td><td rowspan=1 colspan=1>1849</td><td rowspan=1 colspan=1>1904</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>1945</td><td rowspan=1 colspan=1>2336</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>85.93%</td><td rowspan=1 colspan=1>(336/391)</td><td rowspan=1 colspan=1>95% CI    82.1%, 89 %</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95 6%</td><td rowspan=1 colspan=1>(1849/1945)</td><td rowspan=1 colspan=1>95% CI    94 %, 95.9%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>77.8%</td><td rowspan=1 colspan=1>(336/432)</td><td rowspan=1 colspan=1>95% CI   73.6%,81.4%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>(1849/1904)</td><td rowspan=1 colspan=1>95% CI   96.3%, 97.8%</td></tr></table>

a 59/96 discrepant samples were positive and 37/96 were negative for $C .$ . difficile toxin gene DNA when tested with an FDA cleared molecular test. $^ \mathrm { ~ b ~ } 4 3 / 5 5$ discrepant samples were negative and 12/55 were positive for $C .$ . difficile toxin gene DNA when tested with an FDA cleared molecular test.

# Comparison with direct culture

The performance of the Simplexa C.difficile Direct test compared with direct culture is shown in Table 10 and is reported as positive percent agreement (PPA) and negative percent agreement (NPA).

Table 10. Comparison with direct culture   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Direct ToxigenicCulture</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Direct</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>163a</td><td rowspan=1 colspan=1>428</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>12b</td><td rowspan=1 colspan=1>1890c</td><td rowspan=1 colspan=1>1902</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>277</td><td rowspan=1 colspan=1>2053</td><td rowspan=1 colspan=1>2330</td></tr></table>

<table><tr><td>PPA</td><td>95.7%</td><td>(265/277)</td><td>95% CI</td><td>92.6%, 97.5%</td></tr><tr><td>NPA</td><td>92.1%</td><td>(1890/2053)</td><td>95% CI</td><td>90.8%, 93.2%</td></tr></table>

$^ { \mathrm { a } } 1 1 6 / 1 6 3$ discrepant samples were positive, 46/163 were negative, and 1/163 was indeterminate for $C$ difficile toxin gene DNA when tested with an FDA cleared molecular test.   
$^ \mathrm { b } 8 / 1 2$ discrepant samples were negative and 4/12 were positive for $C .$ . difficile toxin gene DNA when tested with an FDA cleared molecular test.   
c Direct culture results were not available from 6 specimens due to plating errors. Overall results were obtained by enriched culture for these 6 samples and so included in the Combined Culture results.

Sample subsets from the multi-site investigational study were also tested with three commercially available FDA cleared NAATs as additional information. The percent agreement (PPA, NPA) of the Simplexa C.difficile Direct test to each of the three cleared NAATs is shown in Tables 11 through 13.

Table 11. Comparison with cleared NAAT1   

<table><tr><td rowspan=1 colspan=1>cYanmaGYellowb</td><td rowspan=1 colspan=3>NAAT1</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Direct</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>683</td><td rowspan=1 colspan=1>691</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>707</td><td rowspan=1 colspan=1>829</td></tr></table>

<table><tr><td>PPA</td><td>93.4%</td><td>(114/122)</td><td>95% CI</td><td>87.6%, 96.6%</td></tr><tr><td>NPA</td><td>96.6%</td><td>(683/707)</td><td>95% CI</td><td>95 %, 97.7%</td></tr></table>

Table 12. Comparison with cleared NAAT2   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>NAAT2</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Direct</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>591</td><td rowspan=1 colspan=1>600</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>629</td><td rowspan=1 colspan=1>776</td></tr></table>

<table><tr><td>PPA</td><td>93.9%</td><td>(138/147)</td><td>95% CI</td><td>88.8%, 96.7%</td></tr><tr><td>NPA</td><td>94 %</td><td>(591/629)</td><td>95% CI</td><td>91.8%, 95.6%</td></tr></table>

Table 13. Comparison with cleared NAAT3   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>NAAT3</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Direct</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>584</td><td rowspan=1 colspan=1>604</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>589</td><td rowspan=1 colspan=1>721</td></tr></table>

<table><tr><td>PPA</td><td>84.8%</td><td>(112/132)</td><td>95% CI</td><td>77.8%,</td><td>90 %</td></tr><tr><td>NPA</td><td>99.2%</td><td>(584/589)</td><td>95% CI</td><td>98 %,</td><td>99.6%</td></tr></table>

b. Clinical specificity:

See section M3a

c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the prospective clinical study conducted at five geographically diverse sites, 2336 samples were collected from 1233 $( 5 2 . 8 \% )$ female and 1103 $( 4 7 . 2 \% )$ male patients. Patient ages ranged from less than one years old to 106 years old, and 2198 $( 9 4 \% )$ were 22 years old or older. Out of the total number of evaluable samples, 391 $( 1 6 . 7 \% )$ samples were culture positive by toxigenic $C .$ . difficile combined direct and enriched culture. The percent positivity with the Simplexa $C .$ . difficile Direct varied from $1 4 . 3 \%$ to $2 1 . 4 \%$ between the clinical sites.

N. Instrument Name: 3M Integrated Cycler

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No ___X_

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification: Specimens are identified by scanning or typing in the sample identifier.

4. Specimen Sampling and Handling:

The test is intended for use with liquid or unformed human stool specimens. Stool specimens should be collected following standard laboratory procedures and precautions. Specimens may be stored at 2 to $2 5 ~ ^ { \circ } \mathrm { C }$ . Stool samples must be tested within 48 hours of collection.

5. Calibration:

End-user calibration for the Integrated Cycler is not necessary. Calibration of the optical modules (excitation and emission gain settings) is performed during the manufacturing process and the values are stored in the instrument firmware.

6. Quality Control:

See section M.1.c for information on internal and external controls.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision