# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K170491   
B. Purpose for Submission: To obtain a substantial equivalence determination for a new device   
C. Measurand: tcdA gene of toxigenic Clostridium difficile   
D. Type of Test: Qualitative Helicase-Dependent Amplification (HDA) assay   
E. Applicant: Quidel Corporation   
F. Proprietary and Established Names: Solana C. difficile Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130, Clostridium difficile toxin gene amplification assay

2. Classification: II

3. Product code: OZN

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Solana C. difficile assay is an in vitro diagnostic test for the direct, qualitative detection of the Clostridium difficile Toxin A gene (tcdA) in unformed stool specimens of patients suspected of having Clostridium difficile infection (CDI). The Solana C. difficile assay is intended for use as an aid in diagnosis of CDI. The assay utilizes helicase‐dependent amplification (HDA) for the amplification of a highly conserved fragment of the Toxin A gene sequence. The Solana C. difficile Assay is intended for use only with the Solana instrument.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only For in vitro diagnostic use only

4. Special instrument requirements: Solana instrument

# I. Device Description:

The Solana C. difficile Assay combines sample processing and Helicase‐Dependent Amplification (HDA) performed in the Solana instrument for the detection of toxigenic Clostridium difficile directly from CDI‐suspected diarrheal specimens. The assay components include the Solana instrument, neonatal flocked swabs for specimen transfer, Lysis Tubes, Dilution Tubes, and Reaction Tubes. The Reaction Tubes contain lyophilized HDA reagents, dNTPs, primers and probes. The Lysis Tubes contain Lysis Buffer and include a competitive process control (PRC) to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. Solana amplifies and detects the target sequence and reports the test results to the user. A maximum of 12 tests can be performed on a single Solana instrument.

Materials provided:

Neonatal flocked swabs · Lysis Tubes Dilution Tubes Reaction Tubes

Materials required but not provided:

External controls for $C .$ . difficile (e.g., Quidel Molecular C. difficile Control Set, which contains positive and negative controls, serves as an external processing and extraction control)

· Sterile DNase‐free filter‐blocked or positive displacement micropipettor tips   
· Micropipettor   
· Stopwatch or timer   
· Scissors or a blade   
· Heat block capable of $9 5 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ temperature   
· Solana workflow tray and transfer rack   
· Solana instrument   
· Thermometer

# J. Substantial Equivalence Information:

1. Predicate device name(s): Portrait Toxigenic C. difficile Assay

2. Predicate 510(k) number(s): K113358 (DEN120013)

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceSolana C. difficile Assay</td><td colspan="1" rowspan="1">PredicatePortrait Toxigenic C.difficile Assay(K113358/DEN120013)</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Solana C. difficile assayis an in vitro diagnostic testfor the direct, qualitativedetection of the Clostridiumdifficile Toxin A gene(tcdA) in unformed stoolspecimens of patientssuspected of havingClostridium difficileinfection (CDI). The SolanaC. difficile assay is intendedfor use as an aid in diagnosisof CDI. The assay utilizeshelicase-dependentamplification (HDA) for theamplification of a highlyconserved fragment of theToxin A gene sequence. TheSolana C. difficile Assay isintended for use only with</td><td colspan="1" rowspan="1">Portrait Toxigenic C. difficileAssay, a prescription deviceunder 21 CFR Part 801.109that is indicated for thedetection of toxigenicClostridium difficile inhuman fecal samplescollected from patientssuspected of havingClostridium difficileinfection (CDI). The testutilizes automated blockedprimer enabled helicase-dependent amplification(bpHDA) to detect toxingene sequences associatedwith toxin producing C.difficile. The PortraitToxigenic C. difficile Assayis intended as an aid in the</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">the Solana instrument.</td><td colspan="1" rowspan="1">diagnosis of CDI.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Liquid or unformed stool</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Qualitative/Quantitative</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay technology</td><td colspan="1" rowspan="1">Isothermal helicase-dependent nucleic acidamplification</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample extraction</td><td colspan="1" rowspan="1">Not required</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection method</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceSolana C. difficile Assay</td><td rowspan=1 colspan=1>PredicatePortrait Toxigenic C.difficile Assay(K113358/ DEN120013))</td></tr><tr><td rowspan=1 colspan=1>Assay target</td><td rowspan=1 colspan=1>Toxin A gene (tcdA)</td><td rowspan=1 colspan=1>Toxin B gene (tcdB)</td></tr><tr><td rowspan=1 colspan=1>Instrument platform</td><td rowspan=1 colspan=1>Solana instrument</td><td rowspan=1 colspan=1>Portrait Analyzer</td></tr><tr><td rowspan=1 colspan=1>Samples/controls per run</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>one</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guideline Document: Toxin Gene Amplification Assays for the Detection of Clostridium difficile, August 27, 2015.

# L. Test Principle:

A small amount of specimen is transferred to a Lysis Tube using a swab. The Lysis Tube is then subjected to heat‐treatment at $9 5 ~ ^ { \circ } \mathrm { C }$ for five minutes. The heat‐treated sample is added to a Dilution Tube, and then transferred to a Reaction Tube. The Reaction Tube contains lyophilized HDA reagents, dNTPs, primers and probes. Once rehydrated with the diluted sample, the Reaction Tube is placed in Solana for amplification and detection of target sequence. In Solana, the target sequence is amplified by specific primers and detected by a specific fluorescence probe included in the Reaction Tube. A competitive process control (PRC) is included in the Lysis Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. The PRC is amplified by the target‐specific primers and detected by a PRC specific fluorescence probe. The target and PRC probes are labeled with a quencher on one end and a fluorophore on the other end. Upon annealing to target or PRC amplicons, the fluorescence signal increases due to physical separation of fluorophore from quencher. Solana measures and interprets the fluorescent signal, using on‐board method‐specific algorithms. Solana then report the test results to the user on its display screen, and it can print out the results via a printer.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

To evaluate the reproducibility of the Solana C. difficile Assay, a blinded and randomized study panel was tested at one internal site and two external clinical sites. The panel was contrived in negative stool matrix and consisted of negative samples and positive samples spiked with toxigenic $C .$ difficile at the following three organism concentration levels: moderate positive $( 3 . 4 \mathrm { ~ x ~ } 1 \dot { 0 } ^ { 3 } \mathrm { C F U / m L } )$ , low positive $( 1 . 7 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ $\mathrm { C F U / m L }$ ), and high negative $( \dot { 4 } . 8 \mathrm { ~ x ~ } 1 0 ^ { 2 } \mathrm { \ C F U / m L } )$ ). Each site tested the panel and external positive and negative controls in triplicate for five non-consecutive days. Testing was conducted by two operators at each site with two runs per day.

The Solana C. difficile Assay produced the expected results in $9 8 . 9 \%$ (89/90) of the low positive samples, $100 \%$ (90/90) of the moderate positive samples, and $100 \%$ (90/90) of the negative samples. The assay produced positive results in $4 7 . 8 \%$ (43/90) of the high negative samples which was within the expected range of 20 to $80 \%$ positive results. The results did not vary between sites, days, or runs.

The external positive and negative controls produced the expected results for all replicates and no invalid control results were observed during the study. The reproducibility study results were acceptable. The results are summarized in Table 1.

Table 1. Site-to-Site Reproducibility Results   

<table><tr><td rowspan=1 colspan=1>Sites</td><td rowspan=1 colspan=2>Site #1</td><td rowspan=1 colspan=2>Site #2</td><td rowspan=1 colspan=2>Site #3</td><td rowspan=2 colspan=2>OverallPercentAgreement</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>#expectedresults/#tested</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>#expectedresults/#tested</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>#expectedresults/#tested</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>12/30</td><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>19/30</td><td rowspan=1 colspan=1>63.3%</td><td rowspan=1 colspan=1>12/30</td><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>43/90</td><td rowspan=1 colspan=1>47.8%</td><td rowspan=1 colspan=1>37.8 - 58.0%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>94 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95 - 100%</td></tr></table>

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Specimen Stability: An analytical study was conducted to assess the specimen storage capabilities of the Solana C. difficile Assay by testing contrived stool samples that were stored under

conditions that may be observed in a clinical setting. Specimen stability was also assessed in the prospective clinical study that tested 852 specimens stored for up to three days at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ . The results of clinical and analytical studies supported the storage of unpreserved stool specimens at 2 to $8 ^ { \mathrm { o } } \mathrm { C }$ for up to three days before testing.

# Controls:

The Solana C. difficile assay incorporates several controls to monitor assay performance.

A competitive process control (PRC) is included in the Lysis Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. The PRC is amplified by the target‐specific primers and detected by a PRC specific fluorescence probe.

External positive and negative controls for toxigenic C. difficile (such as the Quidel Molecular C. difficile Control Set available separately) are intended to be tested and processed as if they were patient specimens. The external positive control is intended to monitor substantial reagent and instrument failure. The external negative control is used to detect reagent or environmental contamination (or carry-over) by C. difficile DNA or amplicon.

It is recommended that the reactivity of each new lot and each new shipment of the Solana C. difficile Assay be verified by testing external controls on receipt and before the new lot or shipment are put into use. External control tests should be performed thereafter in accordance with appropriate federal, state and local guidelines. The Solana C. difficile assay should not be used in patient testing if the external controls do not produce the correct results.

During the prospective clinical study, external positive and negative controls were tested daily at each site. A total of 80 positive controls and 80 negative controls were tested across all sites, and all controls produced the expected results. Out of a total of 854 specimens tested during the study, two $( 0 . 2 \% )$ produced invalid Solana C. difficile Assay results and both specimens remained invalid after repeat testing.

d. Detection limit:

# Analytical sensitivity/ Limit of detection

The Limit of Detection (LoD) of the Solana C. difficile Assay was determined using serial dilutions of two toxigenic $C .$ . difficile strains, ATCC BAA‐1805 and CCUG 20309 spiked in negative stool matrix. The LoD is defined as the lowest concentration of C. difficile in the sample at which $9 5 \%$ of all replicates tested positive. The LoD was confirmed by testing an additional 20 replicates of each strain at the LoD concentration in two additional lots of reagents. The results of the LoD study are summarized in Table 2.

Table 2. Limit of detection   

<table><tr><td rowspan=1 colspan=1>Stool Matrix</td><td rowspan=1 colspan=1>C. difficile Strains</td><td rowspan=1 colspan=1>Strain LOD</td></tr><tr><td rowspan=2 colspan=1>Unpreserved Stool</td><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>9.13 x 103 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>CCUG 20309</td><td rowspan=1 colspan=1>4.90 x 10³ CFU/mL</td></tr></table>

Serial dilutions of a preparation of quantified $C .$ . difficile genomic DNA, BAA‐ 1382DQ™ spiked in lysis buffer were also tested with the Solana C. difficile Assay. The genomic DNA was detected in $9 5 \%$ or more of all replicates at a concentration equivalent to 15 copies per assay.

# Analytical reactivity (Inclusivity):

The inclusivity of the Solana C. difficile Assay was evaluated by testing an additional twenty three strains of toxigenic C. difficile representing multiple toxinotypes. Solana C. difficile Assay testing was performed on three replicates of each strain spiked in negative stool matrix at a concentration near the LoD $( 1 . 8 3 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m L }$ , 2X LOD concentration as determined for ATCC BAA-1805). All twenty-three additional strains were detected in all replicates by the Solana C. difficile Assay at this concentration.

Table 3. Inclusivity/Reactivity   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Toxinotype</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1805*</td><td rowspan=1 colspan=1>III</td></tr><tr><td rowspan=1 colspan=1>CCUG 20309*</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1870</td><td rowspan=1 colspan=1>IIIb</td></tr><tr><td rowspan=1 colspan=1>CCUG 37770</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1875</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>ATCC43598</td><td rowspan=1 colspan=1>VIII</td></tr><tr><td rowspan=1 colspan=1>CCUG37774</td><td rowspan=1 colspan=1>XXIII</td></tr><tr><td rowspan=1 colspan=1>CCUG9004</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1874</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCC43600</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1871</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1803</td><td rowspan=1 colspan=1>IIIc</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1872</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCC700792</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCC43599</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CCUG60276</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>CCUG60275</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>CCUG37778</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>CCUG37777</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>CCUG37776</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>CCUG37773</td><td rowspan=1 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC17857</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCC43594</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ATCC 43596</td><td rowspan=1 colspan=1>0</td></tr></table>

# e. Analytical specificity:

# Cross reactivity/ Microbial interference

The analytical specificity of the Solana C. difficile Assay was evaluated by testing a panel of sixty‐eight bacterial, viral and yeast microorganisms and human DNA representing enteric pathogens, flora or nucleic acids commonly present in the intestine. Microorganisms were tested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ or higher for bacteria and yeast, $1 0 ^ { 5 } \ : \mathrm { P F U / m L }$ (or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) or higher for viruses, and ${ 1 0 } ^ { 6 }$ copies/mL for human DNA. The microorganisms were mixed with pooled negative stool matrix and tested directly for cross‐reactivity. The microorganisms were also tested in stool matrix in the presence of two strains of toxigenic $C .$ difficile (CCUG 20309 or ATCC BAA 1805 tested separately) at 2X LOD for potential microbial interference. Three replicates were tested for each sample. Potential cross reactivity of Clostridium botulinum with assay primers and probes was evaluated in silico analysis of genomic DNA sequences since this organism was not available for wet testing. In silico analysis did not predict C. botulinum cross reactivity with the assay primers. No cross reactivity or microbial interference was observed with any of the wet tested panel members. Table 4 lists the microorganisms evaluated in these studies.

Table 4. Cross reactivity/ Microbial Interference Panel   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Identification</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">ATCC 49176</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC19606</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC7966</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">ATCC15554</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC13472</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC25285</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC43479</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni sub sp jejuni</td><td colspan="1" rowspan="1">ATCC33292</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC10231</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC8090</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">ATCC638</td></tr><tr><td colspan="1" rowspan="1">Clostridium botulinum</td><td colspan="1" rowspan="1">In silicoanalysis</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">CCRI-11128</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">ATCC19398</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">ATCC19402</td></tr><tr><td colspan="1" rowspan="1">Clostridium orbiscindens</td><td colspan="1" rowspan="1">ATCC49531</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ATCC13124</td></tr><tr><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">ATCC35704</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">ATCC12464</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">ATCC 9714</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">Z077</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 6329</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 9284</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 33098</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 36938</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 43123</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 47545</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">CCUG 59819</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43593</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">ATCC 43601</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">ATCC 15579</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 23511</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">ATCC 700927</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">ATCC 43504</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 33497</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC BAA-389</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Salmonella choleraesuis (typhimurium)</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ATCC 11835</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12973</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coxsackie virus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1"></td></tr></table>

# Interference:

The performance of Solana C. difficile Assay was evaluated in the presence of exogenous and endogenous substances that may be present in stool specimens and that could potentially interfere with assay performance. A panel of thirty‐two substances was tested in triplicate with the Solana C. difficile Assay in the presence or absence of two strains of toxigenic C. difficile (CCUG 20309 or ATCC BAA 1805) at 2X LOD. There was no evidence of interference observed at the concentrations tested. Table 5 lists the substances tested in this study.

Table 5. Interfering Substances Panel   

<table><tr><td colspan="1" rowspan="1">Substance Name</td><td colspan="1" rowspan="1">Active Ingredients</td><td colspan="1" rowspan="1">TestConcentration</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Cortizone 10</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Fleet Glycerin Suppositories</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Desitin</td><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Anusol Plus</td><td colspan="1" rowspan="1">pramoxine hydrochloride andzinc sulfate monohydrate</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Cortizone 10</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Fleet Glycerin Suppositories</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Desitin</td><td colspan="1" rowspan="1">Zinc Oxide</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Anusol Plus</td><td colspan="1" rowspan="1">pramoxine hydrochloride andzinc sulfatemonohydrate</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Tums</td><td colspan="1" rowspan="1">Calcium Carbonate</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Equate Antacid Max Strength</td><td colspan="1" rowspan="1">Aluminum hydroxide,Magnesium hydroxide</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Mesalazine RectalSuspensionEnema</td><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Fleet Mineral Oil Enema</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Gynol II Vaginal Contraceptive</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Imodium AD</td><td colspan="1" rowspan="1">Loperamide HCl</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Pepto Bismol</td><td colspan="1" rowspan="1">Bismuth subsalicylate</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Ex-Lax</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">12.5 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">12.5 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Polysporin</td><td colspan="1" rowspan="1">Bacitracin and Polymyxin B</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">12.5 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Tucks personal cleaning pads</td><td colspan="1" rowspan="1">Witch hazel</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Benzalkonium ChlorideTowelettes</td><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">Immunoglobulins, Lysozyme,Polymers, etc.</td><td colspan="1" rowspan="1">3.5%</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">Glucose, Hormones, Enzymes,Ions, Iron, etc.</td><td colspan="1" rowspan="1">10%</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">12.5 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Steric Acid</td><td colspan="1" rowspan="1">Steric Acid</td><td colspan="1" rowspan="1">12.5 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Triglyceride Mix (C2 - C10)</td><td colspan="1" rowspan="1">Triglyceride</td><td colspan="1" rowspan="1">10%</td></tr></table>

# Carryover – Cross Contamination

A study was conducted to evaluate the Solana C. difficile Assay for potential carryover and cross contamination. Two samples were prepared: a high positive sample consisting of toxigenic C. difficile strain CCUG 20309 at $4 . 9 \mathrm { \ : x \ : i 0 ^ { 6 } C F U / m L }$ in negative stool matrix, and a negative sample consisting of unspiked stool matrix. Positive and negative samples were tested in alternating fashion on the Solana C. difficile Assay by two operators over 10 runs for a total of 50 positive and 50 negative results. No carryover was observed in the study.

# f. Assay cut-off:

The cut-off values for the Solana C. difficile Assay were based on specific parameters of the amplification curve calculated from results from preliminary LoD studies and initial clinical specimen testing. The cut-offs were set based on the longest amount of time to obtain a positive amplification result.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. See M3 below.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity:

Performance characteristics of the Solana C. difficile Assay were established during a prospective study conducted from November 2016 to February 2017. Eight hundred fifty-four left-over specimens from patients suspected of having CDI were collected from three distinct geographical sites across the United States. The patient ages

ranged from less than two years old to $> 6 0$ years, and 799 $( 9 3 . 6 \% )$ were 22 years old or older. The specimens were tested raw with the Solana C. difficile Assay at the sites on the day of collection or after storage for up to three days at 2 to $8 ^ { \circ } \mathrm { C }$ . The Solana results were compared to a “broth enhanced” toxigenic bacterial culture and separately compared to a FDA-cleared molecular test. An alternate FDA-cleared molecular test was used for discrepant analysis.

# Comparison with broth enhanced toxigenic bacterial culture

Eight hundred fifty-four (854) raw specimens were tested by both the Solana C. difficile Assay and broth enhanced toxigenic culture. For the toxigenic culture method, samples were inoculated into chopped-meat glucose (CMG) broth and after 48-hours sub-cultured onto CCFA-HB plates. Suspicious colonies were further characterized and C. difficile identified colonies were sub-cultured in CMG broth for subsequent cytotoxin testing. Two specimens $( 0 . 2 \% )$ were invalid in the Solana C. difficile Assay when tested according to the Solana C. difficile Assay draft instructions for use. Both specimens remained invalid upon repeat testing and were removed from further analysis. The combined results of the remaining eight hundred fifty-two specimens are summarized in Table 6.

Table 6. Comparison with broth enhanced culture   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Broth EnhancedToxigenic Culture</td></tr><tr><td rowspan=1 colspan=1>Solana C. difficile Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Grand Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>6*</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8**</td><td rowspan=1 colspan=1>731</td><td rowspan=1 colspan=1>739</td></tr><tr><td rowspan=1 colspan=1>Grand Total</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>737</td><td rowspan=1 colspan=1>854</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>93.04%</td><td rowspan=1 colspan=1>(107/115)</td><td rowspan=1 colspan=1>95% CI: 86.87%, 96.43%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>99.19%</td><td rowspan=1 colspan=1>(731/737)</td><td rowspan=1 colspan=1>95% CI: 98.24%,99.63%</td></tr></table>

\* Three of the six specimens were positive for $C .$ . difficile toxin gene DNA by an alternate FDA cleared nucleic acid amplification test. $^ { * * }$ Six of the eight specimens were positive for $C .$ . difficile toxin gene DNA by an alternate FDA cleared nucleic acid amplification test.

# Comparison with an FDA cleared nucleic acid amplification test (NAAT)

The performance of the Solana C. difficile Assay was compared with an FDA cleared NAAT for all eight hundred fifty-two specimens with valid Solana results. The combined results reported as positive percent agreement (PPA) and negative percent agreement (NPA) are summarized in Table 7.

Table 7. Comparison with an FDA cleared NAAT   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>FDA Cleared NAAT</td></tr><tr><td rowspan=1 colspan=1>Solana C. difficile Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Grand Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>16*</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3**</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>739</td></tr><tr><td rowspan=1 colspan=1>Grand Total</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>752</td><td rowspan=1 colspan=1>852</td></tr></table>

<table><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>97.0%</td><td rowspan=1 colspan=1>(97/100)</td><td rowspan=1 colspan=1>95% CI: 91.55%, 98.98%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>97.87%</td><td rowspan=1 colspan=1>(736/752)</td><td rowspan=1 colspan=1>95% CI: 96.57%, 98.69%</td></tr></table>

\* Twelve of the 16 specimens were positive for $C$ . difficile toxin gene DNA by an alternate FDA cleared NAAT   
$^ { * * }$ Two of the three specimens were positive for $C .$ . difficile toxin gene DNA by an alternate FDA cleared NAAT.   
b. Clinical specificity: See section M3a.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

The expected values of the Solana C. difficile Assay were established during a prospective study conducted between November 2016 to February 2017. Eight hundred fifty-four (854) specimens used for this study were collected from patients suspected of having Clostridium difficile infection (CDI) at three distinct geographical sites across the United States. A single specimen was collected per patient. The specimens were processed and tested with Solana C. difficile Assay on the Solana instrument at the sites.

Patient age, gender, and the percent positive results observed with the Solana C. difficile Assay for the combined sites are shown in Table 8.

Table 8. Combined Sites – Age and Gender Distributions   

<table><tr><td rowspan=1 colspan=1>Age/Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total % positive with theSolana C. difficile Assay</td></tr><tr><td rowspan=1 colspan=1>≤ 2 years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16.7% (1/6)</td></tr><tr><td rowspan=1 colspan=1>3 to 11 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20.0% (2/10)</td></tr><tr><td rowspan=1 colspan=1>12 to17years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>7.1% (1/14)</td></tr><tr><td rowspan=1 colspan=1>18 to21years</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>24.0% (6/25)</td></tr><tr><td rowspan=1 colspan=1>22 to59 years</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>14.2% (48/337*)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>461</td><td rowspan=1 colspan=1>12.0% (55/460*)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>358</td><td rowspan=1 colspan=1>854</td><td rowspan=1 colspan=1>13.3% (113/852**)</td></tr></table>

\* One specimen was invalid $^ { * * }$ Two specimens total were invalid

N. Instrument Name: Solana instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes _ or No ___X_

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No ___X_

2. Software:

The Solana® Instrument Software was reviewed and cleared as part of submission K150868. The additional information was provided in support of the Solana C. difficile Assay was reviewed and found acceptable.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No

3. Specimen Identification: Specimens are identified by scanning a barcode or by manual entry.

4. Specimen Sampling and Handling:

Raw stool specimens are sampled with the neonatal flocked swabs provided and transferred to a lysis tube. After heat lysis, $5 0 \mu \mathrm { L }$ of lysed sample is transferred to a dilution tube. Subsequently, $5 0 \mu \mathrm { L }$ of diluted sample is transferred to a reaction tube for automated amplification and detection.

5. Calibration:

The end user is not required to calibrate the instrument.

6. Quality Control:

See section M1c for information on internal and external controls.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.