May 2, 2017

BECTON, DICKINSON AND COMPANY   
LAURA STEWART   
REGULATORY AFFAIRS SPECIALISTS   
7 LOVETON CIRCLE   
SPARKS MD 21152

Re: K170308 Trade/Device Name: BD Max Extended Enteric Bacterial Panel, BD Max System Regulation Number: 21 CFR 866.3990 Regulation Name: Gastrointestinal microorganism multiplex nucleic acid-based assay. Regulatory Class: II Product Code: PCH Dated: January 31, 2017 Received: February 1, 2017

Dear Ms. Stewart:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA‚Äôs issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act‚Äôs requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Division of Industry and Consumer Education at its toll-free number (800) 638 2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, ‚ÄúMisbranding by reference to premarket notification‚Äù (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH‚Äôs Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours,

# Kristian M. Roth -S

Uwe Scherf, M.Sc., Ph.D.For: Director Division of Microbiology Devices Office of In Vitro Diagnostics and Radiological Health Center for Devices and Radiological Health

Enclosure

<table><tr><td>DEPARTMENT OF HEALTH AND HUMAN SERVICES Food and Drug Administration Indications for Use</td><td>Form Approved: OMB No. 0910-0120 Expiration Date: January 31, 2017 See PRA Statement below.</td></tr><tr><td>510(k) Number (if known)</td><td></td></tr><tr><td>K170308 Device Name</td><td></td></tr><tr><td>BD MAXTM Extended Enteric Bacterial Panel</td><td></td></tr><tr><td>Indications for Use (Describe)</td><td></td></tr></table>

7KH%'0\$;¬å([WHQGHG(QWHULF%DFWHULDO3DQHOSHUIRUPHGRQWKH%'0\$;6\VWHPLVDQ DXWRPDWHGLQYLWURGLDJQRVWLFWHVWIRUWKHGLUHFWTXDOLWDWLYHGHWHFWLRQDQGGLIIHUHQWLDWLRQRIHQWHULFEDFWHULDO SDWKRJHQV,WLVXVHGLQFRQMXQFWLRQZLWKWKH%'0\$;(QWHULF%DFWHULDO3DQHODVDQRSWLRQDO0DVWHU0L[ 7KH%'0\$;¬å([WHQGHG(QWHULF%DFWHULDO3DQHOGHWHFWVQXFOHLFDFLGVIURP

x 3OHVLRPRQDVVKLJHOORLGHV   
x 9LEULR9YXOQLILFXV9SDUDKDHPRO\WLFXVDQG9FKROHUDH (QWHURWR[LJHQLF(VFKHULFKLDFROL(7(&KHDWODELOHHQWHURWR[LQ/7KHDWVWDEOH HQWHURWR[LQ67JHQHV   
C <HUVLQLDHQWHURFROLWLFD

7HVWLQJLVSHUIRUPHGRQVRIWWRGLDUUKHDOXQSUHVHUYHGVWRROVSHFLPHQVRU&DU\%ODLUSUHVHUYHG VWRROVSHFLPHQVIURPV\PSWRPDWLFSDWLHQWVZLWKVXVSHFWHGDFXWHJDVWURHQWHULWLVHQWHULWLVRUFROLWLV7KH WHVWLVSHUIRUPHGGLUHFWO\RQWKHVSHFLPHQXWLOL]LQJUHDOWLPHSRO\PHUDVHFKDLQUHDFWLRQ3&5IRUWKH DPSOLILFDWLRQRIUHOHYDQWJHQHWDUJHW'1\$7KHWHVWXWLOL]HVIOXRURJHQLFJHQHVSHFLILFK\EULGL]DWLRQ SUREHVIRUWKHGHWHFWLRQRIWKHDPSOLILHG'1\$

7KLVWHVWLVLQWHQGHGIRUXVHLQFRQMXQFWLRQZLWKFOLQLFDOSUHVHQWDWLRQODERUDWRU\ILQGLQJVDQG HSLGHPLRORJLFDOLQIRUPDWLRQDVDQDLGLQWKHGLIIHUHQWLDOGLDJQRVLVRI3OHVLRPRQDVVKLJHOORLGHV9LEULR 9YXOQLILFXV9SDUDKDHPRO\WLFXVDQG9FKROHUDH,(QWHURWR[LJHQLF(VFKHULFKLDFROL(7(&/767 DQG<HUVLQLDHQWHURFROLWLFDLQIHFWLRQV5HVXOWVRIWKLVWHVWVKRXOGQRWEHXVHGDVWKHVROHEDVLVIRU GLDJQRVLVWUHDWPHQWRURWKHUSDWLHQWPDQDJHPHQWGHFLVLRQV3RVLWLYHUHVXOWVGRQRWUXOHRXWFRLQIHFWLRQ ZLWKRWKHURUJDQLVPVWKDWDUHQRWGHWHFWHGE\WKLVWHVWDQGPD\QRWEHWKHVROHRUGHILQLWLYHFDXVHRI SDWLHQWLOOQHVV1HJDWLYHUHVXOWVLQWKHVHWWLQJRIFOLQLFDOLOOQHVVFRPSDWLEOHZLWKJDVWURHQWHULWLVPD\EHGXH WRLQIHFWLRQE\SDWKRJHQVWKDWDUHQRWGHWHFWHGE\WKLVWHVWRUQRQLQIHFWLRXVFDXVHVVXFKDVXOFHUDWLYH FROLWLVLUULWDEOHERZHOV\QGURPHRU&URKQ¬∂VGLVHDVH

<table><tr><td colspan="2">Type of Use (Select one or both, as applicable)</td></tr><tr><td>Prescription Use (Part 21 CFR 801 Subpart D)</td><td>‚ñ°Over-The Counter Use (21 CFR 801 Subpart C)</td></tr></table>

# CONTINUE ON A SEPARATE PAGE IF NEEDED

7KLVVHFWLRQDSSOLHVRQO\WRUHTXLUHPHQWVRIWKH3DSHUZRUN5HGXFWLRQ\$FWRI

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELO

7KHEXUGHQWLPHIRUWKLVFROOHFWLRQRILQIRUPDWLRQLVHVWLPDWHGWRDYHUDJHKRXUVSHUUHVSRQVHLQFOXGLQJWKHWLPH WRUHYLHZLQVWUXFWLRQVVHDUFKH[LVWLQJGDWDVRXUFHVJDWKHUDQGPDLQWDLQWKHGDWDQHHGHGDQGFRPSOHWHDQGUHYLHZ WKHFROOHFWLRQRILQIRUPDWLRQ6HQGFRPPHQWVUHJDUGLQJWKLVEXUGHQHVWLPDWHRUDQ\RWKHUDVSHFWRIWKLVLQIRUPDWLRQ FROOHFWLRQLQFOXGLQJVXJJHVWLRQVIRUUHGXFLQJWKLVEXUGHQWR

'HSDUWPHQWRI+HDOWKDQG+XPDQ6HUYLFHV )RRGDQG'UXJ\$GPLQLVWUDWLRQ 2IILFHRI&KLHI,QIRUPDWLRQ2IILFHU 3DSHUZRUN5HGXFWLRQ\$FW35\$6WDII 35\$6WDII#IGDKKVJRY ¬≥\$QDJHQF\PD\QRWFRQGXFWRUVSRQVRUDQGDSHUVRQLVQRWUHTXLUHGWRUHVSRQGWRDFROOHFWLRQRI LQIRUPDWLRQXQOHVVLWGLVSOD\VDFXUUHQWO\YDOLG20%QXPEHU¬¥

# 5. 510(k) Summary

BD MAX‚Ñ¢ Extended Enteric Bacterial Panel (xEBP)   
Summary Preparation Date:   
1/30/2017   
Submitted by:   
BD Diagnostic Systems   
Becton, Dickinson and Company   
7 Loveton Circle   
Sparks, Maryland 21152   
Contact:   
Laura Stewart   
Regulatory Affairs Specialist   
Tel: 410-316-4435   
Fax: 410-316-4188   
Email: laura_stewart@bd.com   
Proprietary Names:   
For the instrument:   
BD MAX‚Ñ¢ System   
For the assay:   
BD MAX‚Ñ¢ Extended Enteric Bacterial Panel (xEBP)

# Common Names:

For the instrument:   
Bench-top molecular diagnostics workstation   
For the assay:   
Gastrointestinal Bacterial Panel Multiplex Nucleic Acid-Based Assay System   
Enteric Bacterial Panel   
Enteric Bacterial Nucleic Acid Test   
Enteric Bacterial identification and differentiation system   
Enteric assay   
Enteric test

# Regulatory Information

Regulation section: 866.3990 ‚Äì Gastrointestinal microorganism multiplex nucleic acid-based assay.

Classification: Class II

Panel: Microbiology (83)

Product Code(s):   
PCI- Gastrointestinal Bacterial Panel Multiplex Nucleic Acid-Based Assay System PCH- Gastrointestinal Pathogen Panel Multiplex Nucleic Acid-Based Assay System OOI- Real Time Nucleic Acid Amplification System

Predicate Device BioFire Diagnostics FilmArray Gastrointestinal (GI) Panel [510(k) K160459]

Device Establishment   
GeneOhm Sciences Canada, Inc. (BD Diagnostics)   
2555 Boul. du Parc-Technologique   
Quebec, QC G1P 4S5   
Canada

Registration Number: 3007420875

# Performance Standards

Class II Special Controls Guideline: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens, November 2, 2015.

# Intended Use

The BD MAX‚Ñ¢ Extended Enteric Bacterial Panel performed on the BD MAX‚Ñ¢ System, is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric bacterial pathogens. It is used in conjunction with the BD MAX Enteric Bacterial Panel as an optional Master Mix. The BD MAX Extended Enteric Bacterial Panel detects nucleic acids from

‚Ä¢ Plesiomonas shigelloides Vibrio (V. vulnificus, V. parahaemolyticus, and V. cholerae) Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable enterotoxin (ST) genes   
‚Ä¢ Yersinia enterocolitica

Testing is performed on soft to diarrheal unpreserved stool specimens or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of relevant gene target DNA. The test utilizes fluorogenic gene-specific hybridization probes for the detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Plesiomonas shigelloides, Vibrio (V. vulnificus, V. parahaemolyticus, and $V .$ cholerae), Enterotoxigenic Escherichia coli (ETEC) LT/ST and Yersinia enterocolitica infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn‚Äôs disease.

Special Conditions for Use Statement: For prescription use.

Special Instrument Requirements: BD MAX‚Ñ¢ System

# Device Description

The BD MAX‚Ñ¢ Extended Enteric Bacterial Panel performed on the BD MAX‚Ñ¢ System, is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric bacterial pathogens. It is used in conjunction with the BD MAX Enteric Bacterial Panel as an optional Master Mix. The BD MAX Extended Enteric Bacterial Panel detects nucleic acids from

Plesiomonas shigelloides Vibrio (V. vulnificus, V. parahaemolyticus, and V. cholerae) Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable enterotoxin (ST) genes   
‚Ä¢ Yersinia enterocolitica

Testing is performed on unpreserved or Cary-Blair preserved soft to diarrheal stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis.

The BD MAX‚Ñ¢ Extended Enteric Bacterial Panel (BD MAX‚Ñ¢ xEBP) is composed of a single Master Mix that must be used in conjunction with the BD MAX‚Ñ¢ Enteric Bacterial Panel (BD MAX EBP), performed on the BD MAX‚Ñ¢ System. The BD MAX xEBP is performed with the Master Mix contained in the BD MAX xEBP reagent kit and the EBP reagents, including the Master Mix contained in the BD MAX‚Ñ¢ EBP reagent kit. The BD MAX xEBP assay and accompanying Assay Definition File (ADF) were developed to allow inclusion of both BD MAX EBP and BD MAX xEBP master mix reagents in one test run.

The BD MAX‚Ñ¢ System and the BD MAX‚Ñ¢ Extended Enteric Bacterial Panel is run with the instrument with associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes. The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX‚Ñ¢ System software automatically interprets test results. A test result may be called as POS (Positive), NEG (Negative), or UNR (Unresolved) for each of the assay‚Äôs targets, based on the amplification status of the target and of the Sample Processing Control. IND (Indeterminate) or INC (Incomplete) results are due to BD MAX‚Ñ¢ System failure.

# Test Principle

A stool specimen is collected and transported to the laboratory in a dry, clean container (for unpreserved specimens) or in Cary-Blair transport media. The specimen is vortexed for 15 seconds and then a $1 0 ~ \mu \mathrm { L }$ loop is used to inoculate a BD MAX‚Ñ¢ Extended Enteric Bacterial Panel and a BD MAX‚Ñ¢ Extended Enteric Bacterial Panel Sample Buffer Tube included in the BD MAX‚Ñ¢ Enteric Bacterial Panel kit. The Sample Buffer Tube is closed with a septum cap, vortexed and transferred to the BD MAX‚Ñ¢ System. A worklist is created and the Sample Buffer Tubes, the BD MAX Enteric Bacterial Panel Unitized Reagent Strip (containing both the BD MAX‚Ñ¢ EBP and BD MAX‚Ñ¢ xEBP master mix reagents) and the BD MAX‚Ñ¢ PCR Cartridges are loaded onto the BD MAX‚Ñ¢ System.

Following enzymatic bacterial cell lysis at elevated temperature, the released nucleic acids are captured on magnetic beads. The beads, with the bound nucleic acids, are washed using Wash Buffer and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized using Neutralization Buffer and transferred to the Master Mix tubes to rehydrate the PCR reagents. After rehydration, the BD $\mathbf { M A X T M }$ System dispenses a fixed volume of PCR-ready solution containing extracted nucleic acids into the BD MAX‚Ñ¢ PCR Cartridge. Microvalves in the BD MAX‚Ñ¢ PCR Cartridge are sealed by the system prior to initiating PCR to contain the amplification mixture, thus preventing evaporation and amplicon contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan $^ \mathrm { \textregistered }$ ) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect the amplicons of the enteric bacterial targets (Plesiomonas shigelloides, Vibrio ( $V .$ vulnificus, V. parahaemolyticus, and V. cholerae), heat labile and heat stabile (LT/ST) ETEC (Enterotoxigenic $E .$ . coli) and Yersinia enterocolitica) and the Sample Processing Control amplicons in five different optical channels of the BD MAX‚Ñ¢ System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the cDNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at each cycle, and interprets the data at the end of the program to report the final results.

Table 1 shows the similarities and Table 2 shows the differences between the BD MAX‚Ñ¢ Extended Enteric Bacterial Panel and the predicate device.   
Table 1: Similarities Comparison to Predicate Device   

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">BD MAXTM xEBP</td><td colspan="1" rowspan="1">FilmArray GI Panel (K160459)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD MAXTM Extended Enteric Bacterial Panelperformed on the BD MAX System, is an automatedin vitro diagnostic test for the direct qualitativedetection and differentiation of enteric bacterialpathogens. It is used in conjunction with the BDMAX EntericBacterial Panel as an optional Master Mix. The BDMAX Extended Enteric Bacterial Panel detectsnucleic acids fromPlesiomonas shigelloidesVibrio (V. vulnificus, V. parahaemolyticus, and V.cholerae)Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable enterotoxin (ST)genesYersinia enterocoliticaTesting is performed on soft to diarrheal unpreservedstool specimens or Cary-Blair preserved stoolspecimens from symptomatic patients withsuspected acute gastroenteritis, enteritis or colitis.The test is performed directly on the specimen,utilizing real-time polymerase chain reaction (PCR)for the amplification of relevant gene target DNA.The test utilizes fluorogenic gene-specifichybridization probes for the detection of theamplified DNA.This test is intended for use, in conjunction withclinical presentation, laboratory findings, andepidemiological information, as an aidin the differential diagnosis of Plesiomonasshigelloides, Vibrio (V. vulnificus, V.parahaemolyticus, and V. cholerae), EnterotoxigenicEscherichia coli (ETEC) LT/ST and Yersiniaenterocolitica infections. Results of this test shouldnot be used as the sole basis for diagnosis, treatment,or other patient management decisions. Positiveresults do not rule out co-infection with otherorganisms that are not detected by this test, and maynot be the sole or definitive cause of patient illness.Negative results in the setting of clinical illnesscompatible with gastroenteritis may be due toinfection by pathogens that are not detected by thistest or non-infectious causes such as ulcerativecolitis, irritable bowel syndrome, or Crohn's disease.</td><td colspan="1" rowspan="1">The FilmArray Gastrointestinal (GI) Panel is aqualitative multiplexed nucleic acid-based in vitrodiagnostic test intended for use with FilmArraysystems. The FilmArray GI Panel is capable of thesimultaneous detection and identification of nucleicacids from multiple bacteria, viruses, and parasitesdirectly from stool samples in Cary Blair transportmedia obtained from individuals with signs and/orsymptoms of gastrointestinal infection. Thefollowing bacteria (including several diarrheagenicE. coli/Shigella pathotypes), parasites, and virusesare identified using the FilmArray GI Panel:- Campylobacter (C. jejuni/C. coli/C. upsaliensis)Clostridium difficile (C. difficile) toxin A/B- Plesiomonas shigelloides- Salmonella- Vibrio (V. parahaemolyticus/V. vulnificus/V.cholerae), including specific identification ofVibrio cholerae- Yersinia enterocolitica- Enteroaggregative Escherichia coli (EAEC)- Enteropathogenic Escherichia coli (EPEC)- Enterotoxigenic Escherichia coli (ETEC) lt/st- Shiga-like toxin-producing Escherichia coli(STEC) stx1/stx2 (including specific identificationof the E. coli O157 serogroup within STEC)- Shigella/Enteroinvasive Escherichia coli (EIEC)- Cryptosporidium- Cyclospora cayetanensis- Entamoeba histolytica- Giardia lamblia (also known as G. intestinalisand G. duodenalis)- Adenovirus F 40/41- Astrovirus- Norovirus GI/GII- Rotavirus A- Sapovirus (Genogroups I, II, IV, and V)The FilmArray GI Panel is indicated as an aid inthe diagnosis of specific agents of gastrointestinalillness and results are meant to be used inconjunction with other clinical, laboratory, andepidemiological data. Positive results do not ruleout co-infection with organisms not included in theFilmArray GI Panel. The agent detected may not bethe definite cause of the disease.Concomitant culture is necessary for organism</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">BD MAXTMxEBP</td><td colspan="1" rowspan="1">FilmArray GI Panel (K160459)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">‚àí</td><td colspan="1" rowspan="1">recovery and further typing of bacterial agents.This device is not intended to monitor or guidetreatment for C. difficile infection.Due to the small number of positive specimenscollected for certain organisms during theprospective clinical study, performancecharacteristics for E. coli O157, Plesiomonasshigelloides, Yersinia enterocolitica, Astrovirus,and Rotavirus A were established primarily withretrospective clinical specimens.Performance characteristics for Entamoebahistolytica, and Vibrio (V. parahaemolyticus, V.vulnificus, and Vibrio cholerae) were establishedprimarily using contrived clinical specimens.Negative FilmArray GI Panel results in the settingof clinical illness compatible with gastroenteritismay be due to infection by pathogens that are notdetected by this test or non-infectious causes suchas ulcerative colitis, irritable bowel syndrome, orCrohn's disease.A gastrointestinal microorganism multiplex nucleicacid-based assay also aids in the detection andidentification of acute gastroenteritis in the contextof outbreaks.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Cary-Blair preserved stoolUnpreserved soft to diarrheal stool</td><td colspan="1" rowspan="1">Cary-Blair preserved stool.Not claimed (see Differences below)</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCRDetection: fluorogenic target-specific hybridization.</td><td colspan="1" rowspan="1">Amplification: PCRDetection: non target-specific fluorescent dye</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">‚Ä¢Plesiomonas shigelloides‚Ä¢ Vibrio (V. vulnificus, V. parahaemolyticus, and V.cholerae)‚Ä¢Enterotoxigenic Escherichia coli heat labile andheat stabile (LT/ST) (ETEC)‚Ä¢Yersinia enterocolitica</td><td colspan="1" rowspan="1">‚Ä¢ Plesiomonas shigelloides‚Ä¢ Vibrio (V. parahaemolyticus/V. vulnificus/ V.cholerae), including specific identification ofVibrio cholerae‚Ä¢Enterotoxigenic Escherichia coli (ETEC) lt/stYersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofTest Results</td><td colspan="1" rowspan="1">Automated: BD MAX System diagnostic software</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Analysis Platform</td><td colspan="1" rowspan="1">BD MAXTM System</td><td colspan="1" rowspan="1">Film Array Instrument</td></tr><tr><td colspan="1" rowspan="1">PCR Samplepreparation</td><td colspan="1" rowspan="1">Automated:BD MAXTM System</td><td colspan="1" rowspan="1">Automated:Film Array Instrument</td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">TaqMan¬Æ Probe</td><td colspan="1" rowspan="1">Fluorescent double stranded DNA binding dye (LCGreen Plus)</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample Processing Control (SPC)</td><td colspan="1" rowspan="1">Two controls are included in each reagent pouch tocontrol for sample processing and both stages ofPCR and melt analysis.</td></tr></table>

Table 2: Differences Comparison to Predicate Device   

<table><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td>BD MAXTMxEBP</td><td>FilmArray GI Panel</td></tr><tr><td>Specimen Type</td><td>Unpreserved soft to diarrheal stool</td><td>Not claimed Other organisms detected:</td></tr><tr><td colspan="3">Organisms Detected</td></tr><tr><td rowspan="10">Listed in device Similarities above.</td><td rowspan="9"></td><td>Campylobacter (C. jejuni/C. coli/C. upsaliensis)</td></tr><tr><td>Clostridium difficile (C. difficile) toxin A/B</td></tr><tr><td>Salmonella</td></tr><tr><td>Enteroaggregative Escherichia coli (EAEC) Enteropathogenic Escherichia coli (EPEC)</td></tr><tr><td>Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2</td></tr><tr><td>(including specific identification of the E. coli O157</td></tr><tr><td>serogroup within STEC)</td></tr><tr><td>Shigella/ Enteroinvasive Escherichia coli (EIEC)</td></tr><tr><td>Cryptosporidium</td></tr><tr><td></td><td></td></tr><tr><td></td><td>Cyclospora cayetanensis</td></tr><tr><td></td><td>Entamoeba histolytica</td></tr><tr><td></td><td>Giardia lamblia (also known as G. intestinalis and G.</td></tr><tr><td></td><td>duodenalis)</td></tr><tr><td></td><td>Adenovirus F 40/41 Astrovirus</td></tr><tr><td></td><td></td></tr><tr><td></td><td>Norovirus GI/GII</td></tr><tr><td></td><td>Rotavirus A</td></tr><tr><td></td><td>Sapovirus (Genogroups I, II, IV, and V)</td></tr></table>

# Analytical Performance

# Precision

Within-laboratory precision was evaluated for the BD MAX‚Ñ¢ Extended Enteric Bacterial Panel at one (1) internal site. Testing was performed over 12 days, with two runs per day (one each by 2 operators), for a total of 24 runs. The Precision panel members were divided into four (4) concentration categories, based upon organism concentration relative to the LoDs established for each of the assay targets and expected correct percent positive/negative. The panel members contained Vibrio cholerae, Plesiomonas shigelloides, ETEC and Yersinia enterocolitica. The following values were used as spike levels for the target organisms contained in each panel member:

‚Ä¢ Moderate Positive (MP): ${ \ge } 2$ to $\leq 3 \mathrm { x }$ LoD; positive approximately $9 5 \%$ of the time ‚Ä¢ Low Positive (LP): ${ \geq } 1$ to $< 2 \mathbf { x }$ LoD; positive approximately $9 5 \%$ of the time ‚Ä¢ High Negative (HN): C20-80 LoD; negative between 20 and $80 \%$ of the time True Negative (TN): No target; negative $100 \%$ of the time

Each panel member was spiked with negative unpreserved stool matrix. True negative samples contained no target. High negative samples were spiked with target organisms below the analytical LoD of the assay; however, the HN samples were expected to yield a positive result in approximately $20 \%$ to $80 \%$ of the replicates due the inherent sensitivity of the PCR assays. Results are summarized by target and concentration in Table 3.

Table 3:   
Precision study result Using One Lot of the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td></tr><tr><td rowspan=1 colspan=1>HNa</td><td rowspan=1 colspan=1>58.328/48(44.3, 71.2)</td><td rowspan=1 colspan=1>45.822/48(32.6, 59.7)</td><td rowspan=1 colspan=1>41.720/48(28.8, 55.7)</td><td rowspan=1 colspan=1>47.923/48(34.5, 61.7)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>97.947/48(89.1, 99.6)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>97.947/48(89.1, 99.6)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

# Reproducibility

The Site-to-Site reproducibility study was performed at three (3) clinical sites using one (1) reagent lot. Two (2) operators performed 2 runs per day, over five (5) distinct days (consecutive or not), for a total of 30 runs. The panels used were the same as described under the Precision heading, above.

The overall site-to-site reproducibility percent agreement was $100 \%$ for the TN category for all targets, and ranged from 30.0 to $4 8 . 9 \%$ , 97.8 to $100 \%$ and 98.9 to $100 \%$ for the HN, LP and MP categories, respectively. Results are summarized in Table 4. The quantitative reproducibility results across sites by target are presented in Table 5 and Table 6. Ct. Score is an internal criterion used to determine final assay results and was selected as a means of assessing quantitative assay reproducibility. Mean Ct. Score and the mean Cycle EP values with variance components (SD and $\%$ CV) are shown in Table 5 and Table 6.

Table 4: Site-to-Site Reproducibility Results Using One Lot of the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td></tr><tr><td rowspan=1 colspan=1>HNa</td><td rowspan=1 colspan=1>48.944/90(38.8, 59.0)</td><td rowspan=1 colspan=1>30.027/90(21.5, 40.1)</td><td rowspan=1 colspan=1>35.632/90(26.4, 45.8)</td><td rowspan=1 colspan=1>46.742/90(36.7, 56.9)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>97.888/90(92.3, 99.4)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Table 5: Quantitative Site-to-Site Reproducibility for all Targets and Sample Processing Control   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Category</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between DayWithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=6 colspan=1>Vibrio</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>805.1</td><td rowspan=1 colspan=1>334.64</td><td rowspan=1 colspan=1>41.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>103.89</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>350.40</td><td rowspan=1 colspan=1>43.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>2725.7</td><td rowspan=1 colspan=1>398.88</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>303.13</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>250.16</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>588.42</td><td rowspan=1 colspan=1>21.6</td><td rowspan=1 colspan=1>812.29</td><td rowspan=1 colspan=1>29.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>3076.6</td><td rowspan=1 colspan=1>459.04</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>127.59</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>92.07</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>485.26</td><td rowspan=1 colspan=1>15.8</td></tr><tr><td rowspan=6 colspan=1>Plesiomonasshigelloides</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>596.5</td><td rowspan=1 colspan=1>284.02</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>284.02</td><td rowspan=1 colspan=1>47.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>2349.7</td><td rowspan=1 colspan=1>332.60</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>250.45</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>416.36</td><td rowspan=1 colspan=1>17.7</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>2320.9</td><td rowspan=1 colspan=1>342.77</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>313.96</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>86.03</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>416.63</td><td rowspan=1 colspan=1>18.0</td><td rowspan=1 colspan=1>630.12</td><td rowspan=1 colspan=1>27.1</td></tr><tr><td rowspan=3 colspan=1>Yersinia</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=3 colspan=1>enterocolitica</td><td rowspan=3 colspan=1>EndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>599.6</td><td rowspan=1 colspan=1>232.01</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>139.57</td><td rowspan=1 colspan=1>23.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>121.25</td><td rowspan=1 colspan=1>20.2</td><td rowspan=1 colspan=1>296.66</td><td rowspan=1 colspan=1>49.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1191.1</td><td rowspan=1 colspan=1>272.51</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>107.96</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>121.52</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>317.31</td><td rowspan=1 colspan=1>26.6</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1264.2</td><td rowspan=1 colspan=1>363.00</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>139.76</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>388.98</td><td rowspan=1 colspan=1>30.8</td></tr><tr><td rowspan=6 colspan=1>ETEC</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>976.0</td><td rowspan=1 colspan=1>577.46</td><td rowspan=1 colspan=1>59.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>577.46</td><td rowspan=1 colspan=1>59.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>2049.6</td><td rowspan=1 colspan=1>511.41</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>61.48</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>121.96</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>529.33</td><td rowspan=1 colspan=1>25.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>2640.0</td><td rowspan=1 colspan=1>491.69</td><td rowspan=1 colspan=1>18.6</td><td rowspan=1 colspan=1>217.12</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>263.58</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>598.65</td><td rowspan=1 colspan=1>22.7</td></tr><tr><td rowspan=2 colspan=1>SPC</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>6020.7</td><td rowspan=1 colspan=1>672.52</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>148.61</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>688.74</td><td rowspan=1 colspan=1>11.4</td></tr></table>

Table 6: Quantitative Site-to-Site Reproducibility Results Summary   

<table><tr><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=3>Vibrio</td><td rowspan=1 colspan=3>P. shigelloides</td><td rowspan=1 colspan=3>Y. enterocolitica</td><td rowspan=1 colspan=3>ETEC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=4 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=4 colspan=1>CycleEP</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>805.1</td><td rowspan=1 colspan=1>2725.7</td><td rowspan=1 colspan=1>3076.6</td><td rowspan=1 colspan=1>596.5</td><td rowspan=1 colspan=1>2349.7</td><td rowspan=1 colspan=1>2320.9</td><td rowspan=1 colspan=1>599.6</td><td rowspan=1 colspan=1>1191.1</td><td rowspan=1 colspan=1>1264.2</td><td rowspan=1 colspan=1>976.0</td><td rowspan=1 colspan=1>2049.6</td><td rowspan=1 colspan=1>2640.0</td><td rowspan=1 colspan=1>6020.7</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>350.40</td><td rowspan=1 colspan=1>812.29</td><td rowspan=1 colspan=1>485.26</td><td rowspan=1 colspan=1>284.02</td><td rowspan=1 colspan=1>416.36</td><td rowspan=1 colspan=1>630.12</td><td rowspan=1 colspan=1>296.66</td><td rowspan=1 colspan=1>317.31</td><td rowspan=1 colspan=1>388.98</td><td rowspan=1 colspan=1>577.46</td><td rowspan=1 colspan=1>529.33</td><td rowspan=1 colspan=1>598.65</td><td rowspan=1 colspan=1>688.74</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>43.5</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>49.5</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>59.2</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>11.4</td></tr></table>

The Lot-to-lot reproducibility study was performed at one (1) site using three (3) reagent lots. Two (2) operators performed 2 runs per day, over five (5) distinct days (consecutive or not), for a total of 30 runs. The panels used were the same as described under the Precision heading, above. Results from 5 days of the accuracy and precision study were used to comprise data for one lot of reagents for the Lot-toLot study.

The overall Lot-to-lot reproducibility percent agreements were $100 \%$ for TN, and ranged from 23.3 to $4 1 . 1 \%$ , 97.8 to $100 \%$ and 98.9 to $100 \%$ for the HN, LP and MP respectively. Results are summarized in Table 7. The quantitative results across lots and by target are presented in Table 8 and Table 9. Ct. Score and the Cycle EP, an internal criteria used to determine a final assay result, was selected as a means of assessing quantitative assay reproducibility. Mean Ct. Score and the mean Cycle EP values with variance components (SD and $\%$ CV) are shown in Table 8 and Table 9.

Table 7: Lot-to-lot Reproducibility Results for BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td></tr><tr><td rowspan=2 colspan=1>HNa</td><td rowspan=2 colspan=1>41.137/90(31.5, 51.4)</td><td rowspan=1 colspan=1>28.926/90</td><td rowspan=1 colspan=1>23.321/90</td><td rowspan=2 colspan=1>41.137/90(31.5, 51.4)</td></tr><tr><td rowspan=1 colspan=1>(20.5, 39.0)</td><td rowspan=1 colspan=1>(15.8, 33.1)</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=2 colspan=1>10090/90(95.9, 100)</td><td rowspan=2 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>10090/90</td><td rowspan=2 colspan=1>97.888/90(92.3, 99.4)</td></tr><tr><td rowspan=1 colspan=1>(95.9, 100)</td></tr><tr><td rowspan=3 colspan=1>MP</td><td rowspan=3 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td><td rowspan=2 colspan=1>10090/90</td></tr><tr><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>89/90</td></tr><tr><td rowspan=1 colspan=1>(95.9, 100)</td><td rowspan=1 colspan=1>(94.0, 99.8)</td><td rowspan=1 colspan=1>(95.9, 100)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Table 8: Quantitative Lot-to-lot for all Targets and SPC   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Category</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between DayWithinn Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=6 colspan=1>Vibrio</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>931.1</td><td rowspan=1 colspan=1>369.81</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>369.81</td><td rowspan=1 colspan=1>39.7</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>3124.7</td><td rowspan=1 colspan=1>189.01</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>70.08</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>135.26</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>242.75</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>3033.5</td><td rowspan=1 colspan=1>207.01</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>53.48</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>101.18</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>236.53</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=6 colspan=1>Plesiomonasshigelloides</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>686.9</td><td rowspan=1 colspan=1>320.86</td><td rowspan=1 colspan=1>46.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>104.23</td><td rowspan=1 colspan=1>15.2</td><td rowspan=1 colspan=1>337.36</td><td rowspan=1 colspan=1>49.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>2444.1</td><td rowspan=1 colspan=1>411.42</td><td rowspan=1 colspan=1>16.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>269.42</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>491.79</td><td rowspan=1 colspan=1>20.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>2428.7</td><td rowspan=1 colspan=1>170.72</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>140.04</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>353.06</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>416.43</td><td rowspan=1 colspan=1>17.1</td></tr><tr><td rowspan=3 colspan=1>Yersinia</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=3 colspan=1>enterocolitica</td><td rowspan=3 colspan=1>EndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>731.3</td><td rowspan=1 colspan=1>252.85</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>154.25</td><td rowspan=1 colspan=1>21.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>12.15</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>296.44</td><td rowspan=1 colspan=1>40.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1339.7</td><td rowspan=1 colspan=1>140.87</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>72.45</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>158.41</td><td rowspan=1 colspan=1>11.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1370.3</td><td rowspan=1 colspan=1>162.58</td><td rowspan=1 colspan=1>11.9</td><td rowspan=1 colspan=1>25.73</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>19.29</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>165.73</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=6 colspan=1>ETEC</td><td rowspan=3 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=3 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>838.3</td><td rowspan=1 colspan=1>308.54</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>105.56</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>64.26</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>332.37</td><td rowspan=1 colspan=1>39.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>2056.8</td><td rowspan=1 colspan=1>267.41</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>323.86</td><td rowspan=1 colspan=1>15.7</td><td rowspan=1 colspan=1>419.99</td><td rowspan=1 colspan=1>20.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>2244.2</td><td rowspan=1 colspan=1>315.85</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>332.48</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>458.59</td><td rowspan=1 colspan=1>20.4</td></tr><tr><td rowspan=2 colspan=1>SPC</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>CycleEndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>5544.7</td><td rowspan=1 colspan=1>567.21</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>282.58</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>633.70</td><td rowspan=1 colspan=1>11.4</td></tr></table>

Table 9: Quantitative Lot-to-lot Reproducibility Results Summary   

<table><tr><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=3>Vibrio</td><td rowspan=1 colspan=3>P. shigelloides</td><td rowspan=1 colspan=3>Y. enterocolitica</td><td rowspan=1 colspan=3>ETEC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=4 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.30</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=4 colspan=1>CycleEP</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>931.1</td><td rowspan=1 colspan=1>3124.7</td><td rowspan=1 colspan=1>3033.5</td><td rowspan=1 colspan=1>686.9</td><td rowspan=1 colspan=1>2444.1</td><td rowspan=1 colspan=1>2428.7</td><td rowspan=1 colspan=1>731.3</td><td rowspan=1 colspan=1>1339.7</td><td rowspan=1 colspan=1>1370.3</td><td rowspan=1 colspan=1>838.3</td><td rowspan=1 colspan=1>2056.8</td><td rowspan=1 colspan=1>2244.2</td><td rowspan=1 colspan=1>5544.7</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>369.81</td><td rowspan=1 colspan=1>242.75</td><td rowspan=1 colspan=1>236.53</td><td rowspan=1 colspan=1>337.36</td><td rowspan=1 colspan=1>491.79</td><td rowspan=1 colspan=1>416.43</td><td rowspan=1 colspan=1>296.44</td><td rowspan=1 colspan=1>158.41</td><td rowspan=1 colspan=1>165.73</td><td rowspan=1 colspan=1>332.37</td><td rowspan=1 colspan=1>419.99</td><td rowspan=1 colspan=1>458.59</td><td rowspan=1 colspan=1>633.70</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>49.1</td><td rowspan=1 colspan=1>20.1</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>11.4</td></tr></table>

# Storage and Stability

‚Ä¢ Collected specimens, either unpreserved stool or stool stored in $1 5 ~ \mathrm { m L }$ Cary-Blair transport media should be kept between $2 \ { } ^ { \circ } \mathrm { C }$ and $2 5 ~ ^ { \circ } \mathrm { C }$ during transport. Protect against exposure to excessive heat.   
‚Ä¢ Specimen can be stored for up to 120 hours (5 days) at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or for up to 24 hours at $2 { - } 2 5 ~ ^ { \circ } \mathrm { C }$ before testing.   
‚Ä¢ BD MAX Extended Enteric Bacterial Panel Master Mix is stable at $2 { - } 2 5 ~ ^ { \circ } \mathrm { C }$ through the stated expiration date. Do not use expired components.   
‚Ä¢ BD MAX Extended Enteric Bacterial Master Mix Tubes are provided in sealed pouches. To protect product from humidity, immediately re-seal after opening. Master Mix tubes are stable for up to 14 days at $2 { - } 2 5 ~ ^ { \circ } \mathrm { C }$ after initial opening and re-sealing of the pouch. Unreconstituted Master Mix tubes are stable for up to 24 hours at $2 { - } 2 5 ~ ^ { \circ } \mathrm { C }$ after being removed from their protective pouch.

# Controls

External Control materials are not provided by BD. External Positive and Negative Controls are not used by the BD MAX System software for the purpose of sample test result interpretation. External Controls are treated as if they were patient samples. However, Quality Control strains and procedures are included in the package insert. Various types of External Controls are recommended to allow the user to select the most appropriate for their laboratory quality control program:

a. External Negative Control: Commercially available control material or a previously characterized sample known to be negative. BD recommends that the External negative Control be prepared prior to the External Positive Control in order to reduce the potential for contamination as a result of control preparation.

b. External Positive Control: Commercially available control materials, such as the ATCC strains listed below, or previously characterized samples known to be positive:

‚Ä¢ Yersinia enterocolitica (ATCC 9610) ‚Ä¢ Vibrio cholerae (ATCC 14033) ‚Ä¢ Vibrio parahaemolyticus (ATCC 17802) ‚Ä¢ ETEC (ATCC 35401) ‚Ä¢ Plesiomonas shigelloides (ATCC 14029)

The assay includes a Specimen Processing Control (SPC) that is present in the Extraction Tube. The Sample Processing Control monitors the efficiency of DNA capture, washing and elution during the sample processing steps, as well as the efficiency of DNA target amplification and detection during PCR analysis.

# Analytical Sensitivity

The analytical sensitivity (Limit of Detection or LoD) for the BD MAX Extended Enteric Bacterial Panel was determined as follows: Each target organism was prepared and quantified from culture prior to inclusion in this study. Individual inoculating loops were dipped into each organism preparation and were then transferred to a Sample Buffer Tube already containing fecal matrix (preserved or unpreserved) that was pre-determined to be negative for all the targets detected by the BD MAX Extended Enteric Bacterial Panel. Each organism was tested with a minimum of 24 replicates per sample type (preserved or unpreserved), by 2 operators, using 3 different production lots of the BD MAX Extended Enteric Bacterial Panel. The LoD for a specific organism was confirmed by testing at least 24 additional replicates at the determined LoD concentration. Analytical sensitivity (LoD), defined as the lowest concentration at which greater than $9 5 \%$ of all replicates are expected to test positive, ranged from 34 to 539 CFU/SBT and 3,434 to $5 3 , 8 5 2 \mathrm { C F U / m L }$ (in stool) for unpreserved specimens for both strains and 79 to 257 CFU/SBT and 7,860 to 25,712 CFU/mL (in stool) for preserved specimens (Table 10).

Table 10: BD MAX‚Ñ¢ Extended Enteric Bacterial Panel Limit of Detection for Individual Target   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Unpreserved (1 strain)</td><td rowspan=1 colspan=2>Unpreserved (2nd strain)</td><td rowspan=1 colspan=2>Cary-Blair Preserved (1t strain)</td></tr><tr><td rowspan=1 colspan=1>Target organism</td><td rowspan=1 colspan=1>LoDa(CFU/SBT)</td><td rowspan=1 colspan=1>LoDa(CFU/mL instool</td><td rowspan=1 colspan=1>LoDa(CFU/SBT)</td><td rowspan=1 colspan=1>LoDa(CFU/mLin stool)</td><td rowspan=1 colspan=1>LoDa(CFU/SBT)</td><td rowspan=1 colspan=1>LoDa(CFU/mL instool</td></tr><tr><td rowspan=1 colspan=1>PlesiomonasshigelloidesATCC 14029b;ATCC 14030c</td><td rowspan=1 colspan=1>458</td><td rowspan=1 colspan=1>45,752</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>15,481</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>25,712</td></tr><tr><td rowspan=1 colspan=1>YersiniaenterocoliticaATCC 9610b;CCUG 4588c</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>20,900</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>31,099</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>22,723</td></tr><tr><td rowspan=1 colspan=1>ETEC ST/LTATCC 35401b;CCUG 47194c</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>3,434</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>53,852</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>13,706</td></tr><tr><td rowspan=1 colspan=1>Vibrio choleraeATCC 14033bENF 9786c</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>14,942</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>4,344</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>25,238</td></tr><tr><td rowspan=1 colspan=1>VibrioparahaemolyticusATCC17802b;ENF 5887c</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>20,708</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>8,031</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>12,424</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificusATCC 27562b 2;ENF 10727c</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>13,093</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>7,959</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>7,860</td></tr></table>

a LoD concentrations are expressed in CFU/SBT and $\overline { { \mathrm { C F U / m L } } }$ , except for Vibrio, which is expressed in cells/SBT and cells/mL. b 1st strain c 2nd strain

# Analytical Inclusivity

A variety of BD MAX Extended Enteric Bacterial Panel assay target strains were included in this study. Strain selection criteria included prevalence, serotype and geographic location, where appropriate. Sixty-nine (69) strains were tested, including strains from public collections and well-characterized clinical isolates.

Inclusivity testing included 10 strains of Plesiomonas shigelloides, 10 strains of Yersinia enterocolitica, 36 strains of Vibrio (cholerae, parahaemolyticus and vulnificus) and 13 strains of ETEC LT/ST. The strains were tested at $< 3 \mathrm { ~ x ~ }$ LoD (Limit of Detection) of the corresponding strain in unpreserved stool matrix. The BD MAX Extended Enteric Bacterial Panel correctly identified 68 of the 69 strains tested upon initial testing. One strain of ETEC ST/LT (CCUG 38088) did not meet acceptance criteria and was further evaluated. This strain was titrated and tested to determine the minimum concentration sufficient for detection. Upon repeat, the CCUG 38088 strain of ETEC ST/LT was detected at $1 0 \mathrm { ~ x ~ L o D }$ .

# Analytical Specificity (Cross-Reactivity and Exclusivity)

The BD MAX Extended Enteric Bacterial Panel was performed on samples containing phylogenetically related species and other organisms (bacteria, viruses, parasites and yeast) likely to be found in stool specimens. The bacterial cells, yeasts, parasites and viruses were tested in the Sample Buffer Tube at $\geq 1 0 ^ { 6 }$ CFU, cells or genome equivalents $\mathrm { \Delta } ^ { \prime } \mathrm { m L }$ in stool, $\mathrm { o r } \geq 1 0 ^ { 5 } \mathrm { P F U / m L }$ in stool or $\mathrm { T C I D 5 0 / m L }$ in stool or equivalent amount of RNA/DNA/PCR reaction. Overall, 184 organisms were tested.

‚Ä¢ Most bacterial strains, yeast, parasites and viruses tested produced negative results with the BD MAX Extended Enteric Bacterial Panel. Two (2) strains of Vibrio mimicus, associated with human disease, produced positive results with the BD MAX Extended Enteric Bacterial Panel. However, no positive result was recorded at $\leq 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ cells/mL in stool with these two strains.   
‚Ä¢ The following 8 Vibrio species, NOT associated with infections in humans and therefore unlikely to be encountered in human stool, were detected by the BD MAX Extended Enteric Bacterial Panel: V. brasiliensis, V. campbellii, V. harveyi, V. hispanicus, V. nereis, V. pacini, V. rotiferianus and V. tubiashii.

Based upon in silico analysis, the following Vibrio species could be detected by the BD MAX Extended Enteric Bacterial Panel:

2 Vibrio species, NOT associated with infections in humans and therefore unlikely to be encountered in human stool: V. coralliilyticus and Moritella marina (formerly known as Vibrio marinus).   
‚Ä¢ One (1) uncategorized Vibrio species associated with infections in humans, Vibrio HENC.

# Interfering Substances

Nineteen (19) biological and chemical substances that may occasionally be present in stool specimens were evaluated for potential interference with the BD MAX Extended Enteric Bacterial Panel. Included in this study was an Antibiotics Mixture, which consisted of a combination of 8 different antibiotics tested simultaneously (each antibiotic at a concentration that may be found in a stool sample). Nystatin cream was found to interfere at levels above $3 . 1 ~ \mathrm { m g / m L }$ . Spermicidal lubricant and hydrocortisone cream were found to interfere at levels above $2 . 5 ~ \mathrm { m g / m L }$ . Vagisil was found to interfere at levels above $0 . 9 2 ~ \mathrm { m g / m L }$ . Results demonstrated no reportable interference with any other substance tested (Table 11).

In addition, microorganisms that may be endogenously present in stool specimens were evaluated for potential interference with the BD MAX Extended Enteric Bacterial Panel. Ten (10) organisms were tested at high concentration $( > 2 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of stool). Results demonstrated no reportable interference with any microorganism tested (Table 12).

Table 11: Endogenous and Commercial Exogenous Substances Tested with the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=1 colspan=1>Brand Name of Description</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Brand Name of Description</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Fecal Fat</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Spermicial Lubricant</td><td rowspan=1 colspan=1>P</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Diaper Rash Cream</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Vagisil¬Æ</td><td rowspan=1 colspan=1>P</td></tr><tr><td rowspan=1 colspan=1>Whole Human Blood</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Laxatives</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone Cream</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>Anti-Diarrheal (liquid)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Antiseptic Towelettes</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Anti-Diarrheal (pill)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Enema</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antibiotics Mixture</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hemorrhoidal Gel</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antacids</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Nystatin Cream</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>Non-Steroidal Anti-Inflamatory(NSAID)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Topical Antibiotic</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

P: Potential interference with the BD MAX Extended Enteric Bacterial Panel at high concentrations. NI: No reportable interference with the BD MAX Extended Enteric Bacterial Panel.

Table 12: Microorganisms Tested for Interference with the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (stx1/stx2)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Bacteroides thetaiotaomicron</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Ruminococcus bromii</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>NI</td></tr></table>

NI: No reportable interference with the BD MAX Extended Enteric Bacterial Panel

# Carryover/Cross-Contamination

A study was conducted to investigate within-run carryover and between-run carryover while processing samples with high bacterial load of analytes in the BD $\mathbf { M A X T M }$ Extended Enteric Bacterial Panel. A panel made of one high positive member containing one target organism and one negative member was used to prepare numerous samples. The enterotoxigenic Escherichia coli strain was used to represent the high positive panel member $( \mathrm { \tilde { \sim } 1 \mathrm { ~ x ~ } } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SBT). The negative member did not contain any target analyte. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run by alternating negative and positive samples. Three (3) operators performed 3 consecutive runs across 3 BD MAX instruments for a total of nine (9) runs containing 24 samples. Of the 108 negative samples tested in this study, one (1) sample produced a positive ETEC ST/LT result.

# Mixed Infection/Competitive Interference

The mixed infection/competitive interference study was designed to evaluate the ability of the BD MAX Extended Enteric Bacterial Panel to detect low positive results in the presence of other targets at high concentrations. A mix of three (3) out of four (4) organisms (Plesiomonas shigelloides, Yersinia enterocolitica, ETEC ST/LT, and Vibrio cholerae) were prepared at $< 2 \tt x$ their respective LoD to serve as low targets in the BD MAX Extended Enteric Bacterial Panel Sample Buffer Tube. The fourth BD MAX Enteric Bacterial Panel analyte (absent from the low targets mix) was spiked into the Sample Buffer Tube at a concentration $\geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { \dot { C } F U / m L }$ (representing a high load target) along with $1 0 ~ \mu \mathrm { L }$ of unpreserved stool and tested to simulate mixed infections. In the presence of high loads of Plesiomonas shigelloides $( \geq$ $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } $ ) and Vibrio cholerae $( \geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { { c e l l s / m L } ) }$ ), all three (3) organisms corresponding to their respective simulated mixed infection preparations were successfully detected by the BD MAX Extended Enteric Bacterial Panel. Successful detection of all three (3) low target organisms by the BD MAX Extended Enteric Bacterial Panel was achieved in the presence of Yersinia enterocolitica at $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ $\mathrm { C F U / m L }$ and ETEC ST/LT at $9 . 4 4 \times 1 0 ^ { 2 }$ CFU/mL.

# Clinical Performance Studies

Clinical performance characteristics of the BD MAX Extended Enteric Bacterial Panel were determined in a multi-site investigational study. The study involved a total of six (6) geographically diverse clinical centers where stools specimens were collected as part of routine patient care, enrolled into the trial, and tested with the BD MAX Extended Enteric Bacterial Panel. Specimens were obtained from pediatric or adult patients suspected of acute bacterial gastroenteritis, enteritis or colitis, for which stool culture had been ordered by healthcare provider. The reference method for both prospective fresh and prospective frozen specimens, was a combination of bacterial culture followed by alternate PCR assay and bi-directional sequencing on presumptive positive isolates for Yersinia enterocolitica, Vibrio and Plesiomonas shigelloides. For ETEC, the comparator method was two alternate PCR assays and bidirectional sequencing performed from the stool specimens. For retrospective specimens, the historical results were recorded at the collection site. The historical results were confirmed using an alternate PCR assay and bi-directional sequencing as part of the composite comparator method in order to confirm the presence of the target DNA.

A total of 2264 prospective specimens (882 unpreserved and 1382 Cary-Blair preserved) and 146 retrospective specimens (87 unpreserved and 59 Cary-Blair preserved) were enrolled in the clinical evaluation for a total of 2410 specimens enrolled. All test results from the BD MAX Extended Enteric Bacterial Panel and the comparator method were single results, no coinfections were detected. Table 13 describes the number of compliant specimens enrolled by patient age and specimen type with a total of 2403 compliant specimens overall. Table 14 through Table 22 describe the performance characteristics of the BD MAX Extended Enteric Bacterial Panel that were observed during the clinical trial.

Table 13: Compliant Clinical Trial Enrollment Summary by Age Group and Specimen Type   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Cary-Blairpreserved</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0-1 month1 month to 2 years2-12</td><td rowspan=1 colspan=1>6250311</td><td rowspan=1 colspan=1>066164</td><td rowspan=1 colspan=1>6316475</td></tr><tr><td rowspan=1 colspan=1>13-1819-21Over 21Unknown</td><td rowspan=1 colspan=1>1414467116</td><td rowspan=1 colspan=1>85236215</td><td rowspan=1 colspan=1>22667129221</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1439</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>2403</td></tr></table>

# Vibrio Performance Results

For the Cary-Blair preserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $100 \%$ and $9 9 . 6 \%$ of the Vibrio prospective positive and negative specimens, respectively, and $100 \%$ of the retrospective positive and negative specimens. For the unpreserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $9 9 . 8 \%$ of the Vibrio negative specimens. No unpreserved prospective positive was found for Vibrio, therefore no performance can be estimated. $100 \%$ and $9 7 . 8 \%$ of the retrospective positive and negative specimens respectively was identified (Table 14).

As Vibrio prevalence is low, an evaluation of contrived samples was performed to supplement data collected in the study. These were prepared by spiking four (4) different strains for each species of Vibrio detected by BD MAX Extended Enteric Bacterial Panel in negative stool matrix. Strains were spiked at various clinically relevant loads and randomly distributed among three (3) clinical sites for BD MAX Extended Enteric Bacterial Panel testing. A positive agreement of $100 \%$ was obtained across the tested loads. Results are shown in Table 15.

Table 14: Vibrio ‚Äì Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Reference Method</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Cary-Blair preserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7</td><td rowspan=3 colspan=1>100(34.2, 100)</td><td rowspan=3 colspan=1>99.6(99.1, 99.8)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1351</td><td rowspan=1 colspan=1>1351</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1356</td><td rowspan=1 colspan=1>1358</td></tr><tr><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>100(34.2, 100)</td><td rowspan=3 colspan=1>100(80.6, 100)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=6 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>99.8(99.2, 99.9)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>866</td><td rowspan=1 colspan=1>866</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>868</td><td rowspan=1 colspan=1>868</td></tr><tr><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=3 colspan=1>100(34.2, 100)</td><td rowspan=3 colspan=1>97.8(88.7, 99.6)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td></tr></table>

While the BD MAX Extended Enteric Bacterial Panel is designed to detect $V .$ . paraheamoliticus, V. cholerae and $V .$ . vulnificus, the panel does not report results to the species level. The Vibrio species identification was obtained by sequencing of the alternate PCR performed from the reference method presumptive positive isolate. During the study, equal number of $V .$ cholerae and $V .$ . parahaemoliticus were obtained and no $V .$ vulnificus was obtained. Three (3) $V .$ . parahaemoliticus and three (3) V. cholerae were obtained.

Table 15: Vibrio Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Expected Result</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=3 colspan=1>100(92.6, 100)</td><td rowspan=3 colspan=1>100(97.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=3 colspan=1>100(92.6, 100)</td><td rowspan=3 colspan=1>100(97.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr></table>

# Plesiomonas shigelloides Performance Results

For the Cary-Blair preserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $9 9 . 9 \%$ of the Plesiomonas shigelloides prospective negative specimens, and $100 \%$ of the retrospective positive and negative specimens. No Cary-Blair prospective positive $P .$ . shigelloides were identified; therefore no performance can be estimated. For the unpreserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $9 9 . 9 \%$ of the $P$ . shigelloides negative specimens. No prospective positive $P .$ . shigelloides were identified; therefore no performance can be estimated. $100 \%$ and $9 7 . 9 \%$ of the retrospective positive and negative specimens respectively was identified (Table 16).

As $P .$ . shigelloides prevalence is low, an evaluation of contrived samples was performed to supplement data collected in the study. These were prepared by spiking 12 different strains of

P. shigelloides in negative stool matrix. Strains were spiked at various clinically relevant loads and randomly distributed among three (3) clinical sites for BD MAX Extended Enteric Bacterial Panel testing. A positive agreement of $100 \%$ was obtained across the tested loads. Results are shown in Table 17.

Table 16: Plesiomonas shigelloides ‚Äì Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Reference Method</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Cary-Blairpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>99.9(99.5, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1355</td><td rowspan=1 colspan=1>1355</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1357</td><td rowspan=1 colspan=1>1357</td></tr><tr><td rowspan=3 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=3 colspan=1>100(51, 100)</td><td rowspan=3 colspan=1>100(90.8, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=6 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>99.9(99.3, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>864</td></tr><tr><td rowspan=3 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=3 colspan=1>100(43.9, 100)</td><td rowspan=3 colspan=1>97.9(88.9, 99.6)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>50</td></tr></table>

Table 17: Plesiomonas shigelloides Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Expected Result</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=3 colspan=1>100(92.6, 100)</td><td rowspan=3 colspan=1>100(97.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>49</td><td rowspan=3 colspan=1>100(92.6, 100)</td><td rowspan=3 colspan=1>99.3(96.2, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>143</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr></table>

a Sample XW0007C was initially found positive for Plesiomonas shigelloides, but found negative for this target once retested from the SBT (Discrepant analysis).

# Yersinia enterocolitica Performance Results

For the Cary-Blair preserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $9 9 . 9 \%$ of the Yersinia enterocolitica prospective negative specimens and $100 \%$ of the retrospective negative specimens. No Cary-Blair prospective and retrospective positive Y. enterocolitica were identified, therefore no performance can be estimated. For the unpreserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $100 \%$ of the Y. enterocolitica negative prospective specimens. No prospective positive Y. enterocolitica were identified, therefore no performance can be estimated. $100 \%$ of the retrospective positive and negative specimens was identified (Table 18).

As Y. enterocolitica prevalence is low, an evaluation of contrived samples was performed to supplement data collected in the study. These were prepared by spiking twelve (12) different strains of Y. enterocolitica in negative stool matrix. Strains were spiked at various clinically relevant loads and randomly distributed among three (3) clinical sites for BD MAX Extended Enteric Bacterial Panel testing. A positive agreement of $100 \%$ was obtained across the tested loads, except for one Cary-Blair preserved sample type at clinical load which the agreement obtained is $9 7 . 9 \%$ (Results are shown in Table 19).

Table 18: Yersinia enterocolitica ‚Äì Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Reference Method</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Cary-Blairpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>99.9(99.6, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1341</td><td rowspan=1 colspan=1>1341</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1342</td><td rowspan=1 colspan=1>1342</td></tr><tr><td rowspan=3 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>100(89.3, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=6 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>No Data forCalculation</td><td rowspan=3 colspan=1>100(99.6, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=2 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=3 colspan=1>100% (70.1%,100%)</td><td rowspan=3 colspan=1>100(92.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>56</td></tr></table>

Table 19: Yersinia enterocolitica Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Expected Result</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td><td rowspan=3 colspan=1>97.9(89.1, 99.6)</td><td rowspan=3 colspan=1>100(97.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=3 colspan=1>100(92.6, 100)</td><td rowspan=3 colspan=1>100(97.4, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr></table>

b Sample XW0351C was initially found negative for Yersinia enterocolitica, but found positive for this target once retested from the SBT (Discrepant analysis).

# Enterotoxigenic E. coli (ETEC LT/ST) Performance Results

For the Cary-Blair preserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $100 \%$ and $9 9 . 8 \%$ of the ETEC prospective positive and negative specimens, respectively, and $100 \%$ of the retrospective positive and negative specimens. For the unpreserved specimen type, the BD MAX Extended Enteric Bacterial Panel identified $100 \%$ and $9 9 . 9 \%$ of the ETEC prospective positive and negative specimens, respectively, and $90 \%$ and $9 6 . 3 \%$ of the retrospective positive and negative specimens respectively was identified (Table 20).

Table 20: Enterotoxigenic E. coli (ETEC LT/ST) ‚Äì Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=3>Comparator Method</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=2 colspan=1>NPA (%)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Cary-Blairpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13</td><td rowspan=3 colspan=1>100(72.2, 100)</td><td rowspan=3 colspan=1>99.8(99.3, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1348</td><td rowspan=1 colspan=1>1348</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1351</td><td rowspan=1 colspan=1>1361</td></tr><tr><td rowspan=3 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=1>100(56.6, 100)</td><td rowspan=3 colspan=1>100(87.9, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=6 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td><td rowspan=3 colspan=1>100(80.6, 100)</td><td rowspan=3 colspan=1>99.9(99.3, 100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>818</td><td rowspan=1 colspan=1>818</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>819</td><td rowspan=1 colspan=1>835</td></tr><tr><td rowspan=2 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=3 colspan=1>90(59.6, 98.2)</td><td rowspan=3 colspan=1>96.3(81.7, 99.3)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>37</td></tr></table>

The ETEC toxin identification was obtained by sequencing of the alternate PCR performed from stool specimens (Table 21). While the BD MAX Extended Enteric Bacterial Panel is designed to detect the toxin types described below, the panel does not report results to the species or toxin level.

Table 21: ETEC Performance per Toxin Observed During the Clinical Trial   

<table><tr><td rowspan=3 colspan=2>ETEC</td><td rowspan=2 colspan=1>LT</td><td rowspan=1 colspan=3>ST</td><td rowspan=2 colspan=1>Unknown</td></tr><tr><td rowspan=1 colspan=1>ST</td><td rowspan=1 colspan=1>STp</td><td rowspan=1 colspan=1>STh</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Specimen Origin</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td></tr><tr><td rowspan=2 colspan=1>Cary-BlairPreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(5/5)(56.6, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(1/1)(20.7, 100)</td><td rowspan=1 colspan=1>100(3/3)(43.9, 100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(5/5)(56.6, 100)</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td><td rowspan=1 colspan=1>100(3/3)(43.9, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td><td rowspan=1 colspan=1>80.0(4/5)(37.6, 96.4)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>0</td></tr></table>

c ETEC toxin was detected but the specific species not identified.

# Discrepant Results

There were nineteen (19) discrepant results in the clinical trial. Eighteen (18) falsepositive (FP) and one (1) false-negative (FN) result. Three (3) retrospective specimens, one (1) ETEC FN, one (1) ETEC FP and one (1) Vibrio FP, were not tested in the discrepant analysis, due to limited specimen volume. Of the sixteen (16) FP discrepant specimens tested, there were seven (7) FP Vibrio, four (4) P. shigelloides, one (1) Y. enterocolitica and four (4) ETEC. Five (5) out of the seven (7) Vibrio discrepant specimens were negative for both BD MAX Extended Enteric Bacterial Panel and the comparator method. The two (2) remaining Vibrio FP specimens, obtained a positive status in the BD MAX Extended Enteric Bacterial Panel coupled with a negative status in the comparator method testing in the discordance study.

Out of the four (4) P. shigelloides discrepant results, one (1) was negative for both BD MAX Extended Enteric Bacterial Panel and the comparator method. Two (2) specimens obtained a positive status on the BD MAX Extended Enteric Bacterial Panel coupled with a negative status in the comparator method testing in the discordance study. They were also both positive for another target on the BD MAX Extended Enteric Bacterial Panel, ETEC and Y. enterocolitica, respectively. One (1) specimen, a Retrospective specimen, was Vibrio true positive in the study and $P .$ . shigelloides FP. In discrepant analysis this specimen was $P .$ . shigelloides negative for the comparator method and positive for Vibrio and $P .$ . shigelloides (1/3). The Y. enterocolitica FP specimen was negative for both BD MAX Extended Enteric Bacterial Panel and the comparator method in discrepant analysis. One (1) of the six (6) ETEC FP was negative for both BD MAX Extended Enteric Bacterial Panel and the comparator method. Two (2) ETEC specimens obtained a positive status on the BD MAX Extended Enteric Bacterial Panel coupled with an ETEC positive status in the RM testing in the discordance study. One (1) ETEC specimen obtained a positive status on the BD MAX Extended Enteric Bacterial Panel coupled with a negative status in the comparator method testing in the discordance study.

# Non-Reportable

The initial unresolved rate was calculated when considering the unresolved rate of both BD MAX Enteric Bacterial Panel and BD MAX Extended Enteric Bacterial Panel assays. Of the 2264 prospective specimens initially evaluated, $3 . 1 \%$ of the Cary-Blair preserved and $3 . 9 \%$ of the unpreserved specimens initially reported as unresolved. The unresolved rate following a valid repeat test was calculated when considering BD MAX Extended Enteric Bacterial Panel only; $0 . 1 \%$ of the prospective Cary-Blair preserved specimens and $0 . 3 \%$ of the prospective unpreserved specimens remained unresolved. Of all the specimens initially evaluated with both BD MAX Enteric Bacterial Panel and BD MAX Extended Enteric Bacterial Panel assays, $1 . 0 \%$ of the CaryBlair preserved and $1 . 5 \%$ of the unpreserved initially reported as Indeterminate. Following a valid repeat test, $0 . 1 \%$ of the Cary-Blair preserved and none of the unpreserved remained Indeterminate. No incompletes were reported during this study (Table 22).

Table 22: Non-reportable Rates   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=1 colspan=3>Unresolved Rate</td><td rowspan=1 colspan=2>Indeterminate Rate</td><td rowspan=1 colspan=1>Incomplete Rate</td></tr><tr><td rowspan=1 colspan=1>InitialxEBP(95% CI)</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalxEBP(95% CI)</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blairpreserved</td><td rowspan=1 colspan=1>2.435/1430(1.8, 3.4)</td><td rowspan=1 colspan=1>3.144/1430(2.3, 4.1)</td><td rowspan=1 colspan=1>0.11/1427(0.0, 0.4)</td><td rowspan=1 colspan=1>1.015/1430(0.6, 1.7)</td><td rowspan=1 colspan=1>0.11/1427(0.0, 0.4)</td><td rowspan=1 colspan=1>00/1430(0.0, 0.3)</td></tr><tr><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>2.221/958(1.4, 3.3)</td><td rowspan=1 colspan=1>3.937/958(2.8, 5.3)</td><td rowspan=1 colspan=1>0.33/958(0.1, 0.9)</td><td rowspan=1 colspan=1>1.514/958(0.9, 2.4)</td><td rowspan=1 colspan=1>0.0/958(0.0, 0.4)</td><td rowspan=1 colspan=1>00/958(0.0, 0.4)</td></tr></table>