# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K170557 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequence in $c f b$ gene of Streptococcus agalactiae (Group B Streptococcus, GBS)

# E. Applicant:

F. Proprietary and Established Names: GenePOC GBS LB Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3740—Streptococcal spp. serological reagents

2. Classification: Class I, non-exempt

3. Product code: NJR—Nucleic Acid Amplification Assay System, Group B Streptococcus, Direct Specimen

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The GenePOC GBS LB assay performed on the revogene instrument is a qualitative in vitro diagnostic test designed to detect Group B Streptococcus (GBS) DNA from 18-24 hour LIM broth enrichments of vaginal/rectal specimen swabs obtained from pregnant women. The GenePOC GBS LB assay utilizes automated sample processing and realtime polymerase chain reaction (PCR) to detect a cfb gene sequence specific to the Streptococcus agalactiae genome.

The GenePOC GBS LB assay is indicated for the identification of antepartum GBS colonization and does not provide susceptibility results. It is not intended to diagnose or monitor treatment of GBS infection. Culture isolates are needed for performing susceptibility testing as recommended for penicillin-allergic women.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

The GenePOC GBS LB Assay is run on the revogene instrument, which is capable of automated nucleic acid extraction, amplification, real-time PCR detection of target nucleic acid sequences with data analysis using a GBS-specific assay definition file. The final software version reported for this 510(k) submission was V3.1.5 and 0.3.4 for the firmware.

Note: Refer to K170558 for additional information regarding the revogene instrument and related software.

# I. Device Description:

The GenePOC GBS LB Assay is a single-use test for the qualitative detection of GBS DNA from enriched vaginal/rectal swab specimens using real-time PCR technology and fluorogenic oligonucleotide probes. The assay is automated and utilizes a microfluidic cartridge for the simultaneous detection of the target GBS DNA and the internal process control $( \mathrm { P r C } )$ DNA on the revogene instrument.

The GenePOC GBS LB Assay is composed of a GBS-specific disposable microfluidic cartridge (PIE), Sample Buffer Tube (SBT) and Disposable Transfer Tool (DTT). These components are used to lyse and dilute the sample, amplify, and detect GBS nucleic acid from vaginal/rectal swabs following LIM Broth enrichment. User intervention is required for sample preparation, adding LIM Broth enriched samples into the SBT, transferring the

sample into the cartridge, and loading/unloading the cartridge into the revogene instrument. Once the sample is added into the cartridge, the process is then fully automated. Each test kit contains 24 individual pouches.

The revogene instrument can process from one up to a maximum of 8 samples simultaneously in the same run. On completion of a run, the results are reported by the revogene instrument from measured fluorescent signals and embedded calculation algorithms. For the GBS application, two signals are processed: GBS target and $\mathrm { P r C }$ . The output results can be reported as positive, negative, indeterminate, or unresolved. On completion of a run, the user removes the used cartridges and disposes of them in normal biological waste. Results may be viewed, printed, transferred, and/or stored by the user.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX GBS Assay

2. Predicate 510(k) number(s): K111860

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">GenePOC GBS LB Assay(Subject Device)K170557</td><td colspan="1" rowspan="1">BD MAX GBS Assay(Predicate Device)K111860</td></tr><tr><td colspan="3" rowspan="1">SIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The GenePOC GBS LB assayperformed on the revogeneinstrument is a qualitative in vitrodiagnostic test designed to detectGroup B Streptococcus (GBS) DNAfrom 18-24 hour LIM brothenrichments of vaginal/rectalspecimen swabs obtained frompregnant women. The GenePOC GBSLB assay utilizes automated sampleprocessing and real-time polymerasechain reaction (PCR) to detect a cfbgene sequence specific to theStreptococcus agalactiae genome.The GenePOC GBS LB assay isindicated for the identification ofantepartum GBS colonization anddoes not provide susceptibilityresults. It is not intended to diagnose</td><td colspan="1" rowspan="1">The BD MAX GBS Assay asimplemented on the BD MAXSystem is a qualitative in vitrodiagnostic test designed to detectGroup B Streptococcus (GBS) DNAin Lim Broth cultures afterincubation for greater than or equalto (&gt;)18 hours, obtained fromvaginal-rectal swab specimens fromantepartum pregnant women. Thetest incorporates automated DNAextraction to isolate the targetnucleic acid from the specimen andreal-time polymerase chain reaction(PCR) to detect a 124 bp region ofthe cfb gene sequence of theStreptococcus agalactiaechromosome. Results from the BDMAX GBS Assay can be used as anaid in determining colonization</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">or monitor treatment of GBSinfection. Culture isolates are neededfor performing susceptibility testingas recommended for penicillin-allergic women.</td><td colspan="1" rowspan="1">status in antepartum women.The BD MAX GBS Assay does notprovide susceptibility results.Cultured isolates are needed forperforming susceptibility testing asrecommended for penicillin-allergicwomen. Subculture to solid mediafor additional testing whenindicated.The BD MAX System is intendedfor in vitro diagnostic (IVD) use inperforming FDA cleared orapproved nucleic acid testing inclinical laboratories. The BD MAXSystem is capable of automatedextraction and purification of nucleicacids from multiple specimen typesas well as the automatedamplification and detection of targetnucleic acid sequences byfluorescence-based PCR.</td></tr><tr><td colspan="1" rowspan="1">Specimen type</td><td colspan="1" rowspan="1">Vaginal-Rectal Swab (Enriched LIMBroth)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationMethod</td><td colspan="1" rowspan="1">Manual; vaginal/rectal swab enrichedovernight in LIM Broth, then analiquot of culture added to samplebuffer</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DNA Extraction</td><td colspan="1" rowspan="1">Automated by instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: Real-Time PCRDetection: Fluorogenic</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Automated using software</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Internal ProcessControl</td><td colspan="1" rowspan="1">To help monitor presence of potentialinhibitory substances as well as anysystem or reagent failures</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">External Control</td><td colspan="1" rowspan="1">Materials available commercially</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">DIFFERENCES</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DNA Target</td><td colspan="1" rowspan="1">190 bp region of the cfb gene</td><td colspan="1" rowspan="1">124 bp region of cfb gene</td></tr><tr><td colspan="1" rowspan="1">Probe Design</td><td colspan="1" rowspan="1">TaqMan</td><td colspan="1" rowspan="1">Scorpion</td></tr><tr><td colspan="1" rowspan="1">GBS AssayFormat</td><td colspan="1" rowspan="1">Fully integrated sample processingand PCR reaction/detection in acartridge</td><td colspan="1" rowspan="1">Requires combination of a reagentstrip and a PCR cartridge</td></tr><tr><td colspan="1" rowspan="1">Single Use</td><td colspan="1" rowspan="1">Cartridge (PIE) can be used once</td><td colspan="1" rowspan="1">Cartridge can be used twice(contains 24 test channels)</td></tr><tr><td colspan="1" rowspan="1">Instrument OpticalChannels</td><td colspan="1" rowspan="1">revogene contains 4 channels</td><td colspan="1" rowspan="1">BD MAX Instrument contains 6optical channels</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI Guideline EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline.

# L. Test Principle:

Vaginal-rectal swab specimens are collected and used to inoculate LIM Broth. After incubation at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ for 18-24 hours in ambient air, $1 5 \mu \mathrm { l }$ of the enriched broth culture is transferred to the SBT. After vortexing the SBT for 15 seconds, approximately $1 5 0 ~ \mu \mathrm { l }$ of the inoculated sample buffer is transferred into the GenePOC microfluidic cartridge (PIE) using the DTT. The loaded GBS LB cartridge is placed into the revogene for further sample processing. No operator intervention is necessary once the clinical sample is loaded onto the revogene.

Each GBS LB microfluidic cartridge is a completely integrated and self-contained cartridge. Each sample is transferred by centrifugation from one microfluidic chamber to the next in sequence, where all reagents specific for the PCR reaction are incorporated and dried within the PCR wells. The step-wise process includes sample homogenization, lysis of cells, and specimen dilution followed by subsequent real-time PCR steps within 1 PCR well in the cartridge. An internal Process Control $( \mathrm { P r } \mathrm { C } )$ is contained in the homogenization chamber and is present in every test to monitor the analytical process (including sample lysis, dilution and nucleic acid amplification, and detection) for the presence of potential inhibitory substances as well as system or reagent failures. The amplified products are detected in real time using target-specific TaqMan chemistry-based probes. The GBS LB-specific designed primers and probe detect a target region of 190 base pairs of the $c f b$ gene of the Streptococcus agalactiae genome. The results are reported by the revogene from measured fluorescent signals and embedded calculation algorithms.

# Interpretation of Sample Results

The analysis software associated with the revogene instrument determines results for GBS and $\mathrm { P r C }$ based on the amplification cycle (Ct) values provided in the assay protocol file. The

GenePOC GBS LB Assay performed on the revogene requires the use of a Ct cut-off and a threshold on RFU to report results. All assay outcomes are described below in Table 1.

Table 1. Summary of all Potential Results for the GenePOC GBS LB Assay and Interpretation   

<table><tr><td rowspan=1 colspan=1>Symbol Displayedon User Screen</td><td rowspan=1 colspan=1>Overall ResultReported</td><td rowspan=1 colspan=1>Interpretation ofPrC Result</td><td rowspan=1 colspan=1>Interpretation of Result</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive or Negative</td><td rowspan=1 colspan=1>Sample contains GBS target DNA. Theprocess control may not amplify in thepresence of high load of GBS target DNA.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>No GBS target DNA detected. Requirespositive PrC result.</td></tr><tr><td rowspan=1 colspan=1>?</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Amplification/detection failure for theprocess control as well as for the GBStarget. Repeat testing must be performedusing the original corresponding enrichedLIM Broth specimen or inoculated SBTwithin the defined timeframe as describedin the package insert.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>No reportable result due to possibleinstrument errors. Repeat testing must beperformed using the originalcorresponding enriched LIM Brothspecimen or inoculated SBT within thedefined timeframe as described in thepackage insert.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the GenePOC GBS LB Assay was evaluated by testing three reagent lots of the assay on three revogene instruments (one per site) with two operators at each of three sites for five days. A blinded and randomized reproducibility panel containing three GBS strains was tested in triplicate at two concentrations in negative matrix: low positive (735 CFU/ml) and moderate positive (1,125 CFU/ml). A Precision Study was based on the within-site variability over five days. The GBS strains used in the panel included two hemolytic strains (ATCC 12400 and ATCC 12403) and one non-hemolytic strain (ATCC 13813). Two true negative samples (LIM Broth matrix previously inoculated with vaginal/rectal swabs that was determined to be GBS negative) were also included as panel members. Across three sites, a total of 720 replicates were tested with the GenePOC GBS LB Assay. For site-to-site reproducibility, the overall percent agreement with expected result (i.e., negative for true negative and positive for moderate positive and low positive samples) was $59 5 \%$ for all GBS strains tested at low positive and high positive concentration. The true negatives yielded expected results $100 \%$ of the time. During the Reproducibility and Precision Study, one unresolved result and one indeterminate result were reported for a rate of $0 . 1 \%$ for the indeterminate rate and the unresolved rate. The results from the site-to site Reproducibility Study for the GenePOC GBS LB Assay are presented in Table 2 below. Bacterial cell numbers in the suspensions were verified by colony counts.

Table 2. Summary of the Percent Agreement Analysis across all Sites   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=1 colspan=3>Observed/Expected</td><td rowspan=2 colspan=1>All Sites</td><td rowspan=2 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>ATCC 13813,Low Positive</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%</td></tr><tr><td rowspan=1 colspan=1>ATCC 13813,High Positive</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87/90</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>ATCC 12403,LowPositive</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8%</td></tr><tr><td rowspan=1 colspan=1>ATCC 12403,High Positive</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%</td></tr><tr><td rowspan=1 colspan=1>ATCC 12400,Low Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ATCC 12400,High Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td></tr></table>

The quantitative results per panel member for between-site, between-operator and between-day reproducibility for $\mathrm { C t }$ values are summarized in Table 3 below. Table 4 shows the $\% \mathrm { C V }$ from the Precision Study. The Reproducibility/Precision Studies are acceptable.

Table 3. Summary of the Overall SD and $\%$ CV for the Ct values in the Reproducibility Study   

<table><tr><td rowspan=2 colspan=1>Target Load</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2>Ct GBS</td><td rowspan=1 colspan=2>Inter-site</td><td rowspan=1 colspan=2>Inter-operator</td><td rowspan=1 colspan=2>Inter-day</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Low Positive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>14.4</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>7.3</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>16.1</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>14.3</td></tr><tr><td rowspan=3 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>15.0</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>14.9</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>14.4</td></tr><tr><td rowspan=2 colspan=1>Target Load</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Ct PrC</td><td rowspan=1 colspan=2>Inter-site</td><td rowspan=1 colspan=1>Inter-o</td><td rowspan=1 colspan=1>perator.</td><td rowspan=1 colspan=2>Inter-day</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>12.1</td></tr></table>

Table 4. Summary of the SD and $\%$ CV for the $\mathrm { C t }$ values for the Within-Site Precision Study   

<table><tr><td rowspan=3 colspan=1>Target Load</td><td rowspan=3 colspan=1>Strain</td><td rowspan=1 colspan=12>Mean Ct, SD, and % CV for GBS</td></tr><tr><td rowspan=1 colspan=4>Site 1</td><td rowspan=1 colspan=4>Site 2</td><td rowspan=1 colspan=4>Site 3</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=1>Low Positive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=3 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>5.8</td></tr><tr><td rowspan=3 colspan=1>Target Load</td><td rowspan=3 colspan=1>Strain</td><td rowspan=1 colspan=12>Mean Ct, SD, and % CV for PrC</td></tr><tr><td rowspan=1 colspan=4>Site 1</td><td rowspan=1 colspan=4>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>Low Positive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.3</td></tr></table>

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Controls There are three types of controls for the GenePOC GBS LB Assay including:

Internal Processing Control: Each GenePOC GBS LB Assay cartridge contains an Internal Process Control $( \mathrm { P r } \mathrm { C } )$ . The $\mathrm { P r C }$ is lysed, amplified, and detected along with each specimen tested and monitors the efficacy of the DNA extraction and PCR amplification processes.

External Controls: External controls should be tested according to guidelines or requirements of local, provincial and/or federal regulations or accreditation organizations. Commercial control material (e.g., Streptococcus agalactiae ATCC 13813) can be used as a positive external control. It is recommended that bacterial strains be freshly prepared in LIM Broth. Transfer a $1 5 \mu \mathrm { l }$ aliquot of an 18-24 hour culture in LIM broth into a SBT. Pure LIM Broth is recommended for use as a negative external control. The GenePOC GBS LB Assay does not include external positive and negative controls.

During the clinical study, positive and negative external controls were tested with the GenePOC GBS LB Assay daily. Overall, it was reported that $9 9 . 5 \%$ of the external controls yielded the expected results (189/190). The submitted external control data are acceptable.

# Reagent Stability

A Reagent Stability Study was conducted with three GenePOC GBS LB reagent lots and three revogene instruments. Stability of the kits stored at both $4 \mathrm { { } ^ { \circ } C }$ and $2 5 \mathrm { { ^ \circ C } }$ was evaluated at the following time points: Day 0, Day 14, Month 1, Month 2, Month 4, and Month 5. The three reagent lots were not produced on the same date and, therefore, testing was staggered. For every stability point, six cartridges per condition were processed. All SBT were inoculated with $1 5 \mu \mu 1$ of LIM Broth containing GBS gDNA.

Results of the study indicated that the GenePOC GBS LB Assay reagents were stable up to 4 months with conditioning at $4 \mathrm { { } ^ { \circ } C }$ and $2 5 \mathrm { { ^ \circ C } }$ . Testing of all lots is ongoing at GenePOC until the 15 month time point is reached.

# LIM Broth Stability and SBT Stability

For both the LIM Broth Culture and inoculated SBT Stability Studies, performance of the GenePOC GBS LB Assay was assessed when broth cultures and SBT were stored at room temperature and $2 - 8 ^ { \circ } \mathrm { C }$ across several time points: 0, 1, 2, 3, 5, 7, and 12 days. All testing was performed on three revogene instruments. One strain of GBS was tested at $1 , 1 2 5 \mathrm { C F U / m l }$ and prepared in negative matrix for the positive samples. An aliquot of negative matrix was used to prepare negative samples. Four replicates were tested at each time point and storage condition per lot across three lots. Results from the nested stability study at room temperature $( 2 5 \pm 2 ^ { \circ } \mathrm { C } )$ showed that enriched LIM Broth culture could be stored at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for up to 48 hrs before inoculating SBT (then held at room temperature for an extra 3 days) and still detect 4/4 GBS samples per condition. Two unexpected results were obtained in the room temperature stability test. One sample during the room temperature study yielded an unresolved assay result; it was re-tested and yielded a positive result. Also, one replicate yielded a false negative GBS result with LIM Broth stored for 48 hrs at room temperature only, which was considered a worst-case scenario. For the LIM Broth stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ up to 3 days, all tested samples produced the expected results.

LIM Broth culture stored for 3 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ was then used to inoculate SBT and held up to 12 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ . All SBT samples produced the expected results when stored for 12 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ . In addition, all positive and negative external controls yielded the expected results during the study. The initial rate of unresolved results was $0 . 3 \%$ (1/336). No indeterminate results were obtained during the study. Table 5 shows results of the stability studies with LIM Broth culture and inoculated SBT. The results support claims for the following in the package insert:

LIM Broth Culture stored at $2 5 \mathrm { { ^ \circ C } }$ for up to 2 days or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 3 days. · Inoculated SBT stored at $2 5 \mathrm { { ^ \circ C } }$ for up to 3 days or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 5 days.

Table 5. LIM Broth and Inoculated SBT Stability Results   

<table><tr><td rowspan=1 colspan=1>Parameter</td><td rowspan=1 colspan=1>Storage Temperature</td><td rowspan=1 colspan=1>Results</td></tr><tr><td rowspan=1 colspan=1>LIM Broth culture storageand conservation withoutrefrigeration</td><td rowspan=2 colspan=1>25 ± °2C</td><td rowspan=1 colspan=1>Stable up to 2 days(48 hrs)</td></tr><tr><td rowspan=1 colspan=1>Inoculated SBTconservation and storagewithout refrigeration</td><td rowspan=1 colspan=1>Stable up to 3 days(72 hrs)</td></tr><tr><td rowspan=1 colspan=1>LIM Broth cultureconservation and storagewith refrigeration</td><td rowspan=2 colspan=1>4 ± 3°</td><td rowspan=1 colspan=1>Stable up to 3 days</td></tr><tr><td rowspan=1 colspan=1>Inoculated SBTconservation and storagewith refrigeration</td><td rowspan=1 colspan=1>Stable up to 12 days</td></tr></table>

# PIE Stability

To establish the sample stability after loading the sample into a cartridge, a true negative and one strain of GBS (ATCC 12403) spiked at $1 , 1 2 5 \mathrm { C F U / m l }$ in SB were tested at room temperature in duplicate across three time points: 0, 60 minutes, and 120 minutes. Assays were performed using three reagent lots and three revogene instruments. All samples yielded the expected results. A sample stability of 60 minutes in the PIE will be claimed. No unresolved or indeterminate results were obtained in the study.

# d. Detection limit:

# LoD

A Limit of Detection (LoD) study was performed to evaluate the analytical sensitivity of the GenePOC GBS LB Assay using two representative strains of GBS (ATCC 12403 and ATCC 13813). Preliminary LoD concentrations were determined by serially diluting strains in negative clinical LIM Broth matrix from 5,000 $\mathrm { C F U / m l }$ down to 150 CFU/ml per strain. For each strain, eight replicates were tested at each dilution across three GenePOC GBS LB assay lots. The observed LoD of each GBS strain was determined as the lowest concentration that had a positivity rate of $\geq$ $9 5 \%$ . While the preliminary LoD for ATCC 13813 GBS strain was estimated to be at $2 0 0 \mathrm { C F U / m l }$ , the LOD for ATCC 12403 GBS strain was determined to be 375 $\mathrm { C F U / m l }$ . The analytical studies were performed using these two strains based on the final assay LoD claims. Only one sample was reported as unresolved for an initial unresolved rate of $0 . 3 \%$ (1/384). Repeat testing yielded a valid result.

The LoD concentrations determined in the preliminary study were confirmed with the same GBS reference strains (diluted to the preliminary LoD concentrations) and tested with twenty-four (24) replicates. A third GBS strain was added (ATCC 12400) to the confirmation panel, where the LoD was estimated to be 375 CFU/ml. The final LoDs values for these strains are presented in Table 6 below.

Table 6. Limit of Detection of the GenePOC GBS LB Assay   

<table><tr><td rowspan=1 colspan=1>GBSStrain</td><td rowspan=1 colspan=1>LoDConcentration(CUml)</td><td rowspan=1 colspan=1>Observed/Expected(% Agreement)</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>375</td><td rowspan=1 colspan=1>23/24(96%)</td></tr><tr><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>23/24 (96%)</td></tr><tr><td rowspan=1 colspan=1>ATCC12400</td><td rowspan=1 colspan=1>375</td><td rowspan=1 colspan=1>23/24 (96%)</td></tr></table>

# Analytical Reactivity

The analytical reactivity (inclusivity) of the GenePOC GBS LB Assay was evaluated with a panel of twelve Streptococcus agalactiae strains representing 11 serotypes (Ia, Ib, Ic, II, III, IV, V, VI, VII, VIII, IX) and one non-hemolytic strain. Each isolate was tested with twenty-four (24) replicates in negative matrix at a concentration near the LoD based on the average CFU/ml from the three isolates tested in the LoD study. If the positivity rate for GBS was not $100 \%$ , sample GBS strain concentrations were increased until all replicates were positive. Table 7 below provides a list of GBS strains tested and the test concentration where $100 \%$ of the replicates were positive for GBS.

Table 7. Analytical Reactivity Study   

<table><tr><td rowspan=1 colspan=1>GBS Strain</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Concentration at which 100%positivityobserveda</td></tr><tr><td rowspan=1 colspan=1>ATCC 12400</td><td rowspan=1 colspan=1>Serotype Ia</td><td rowspan=1 colspan=1>735 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC51487</td><td rowspan=1 colspan=1>Serotype Ib</td><td rowspan=1 colspan=1>5,625 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC27591</td><td rowspan=1 colspan=1>Serotype Ic</td><td rowspan=1 colspan=1>1,875 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC12973</td><td rowspan=1 colspan=1>Serotype II</td><td rowspan=1 colspan=1>735CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC12403</td><td rowspan=1 colspan=1>SerotypeIII</td><td rowspan=1 colspan=1>375 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC49446</td><td rowspan=1 colspan=1>Serotype IV</td><td rowspan=1 colspan=1>2,625 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-611</td><td rowspan=1 colspan=1>SerotypeV</td><td rowspan=1 colspan=1>1,875CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2671</td><td rowspan=1 colspan=1>SerotypeVI</td><td rowspan=1 colspan=1>1,125CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2670</td><td rowspan=1 colspan=1>SerotypeVII</td><td rowspan=1 colspan=1>2,625CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2669</td><td rowspan=1 colspan=1>SerotypeVIII</td><td rowspan=1 colspan=1>3,750 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2668</td><td rowspan=1 colspan=1>Serotype IX</td><td rowspan=1 colspan=1>1,125 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>ATCC13813</td><td rowspan=1 colspan=1>Non-Hemolytic</td><td rowspan=1 colspan=1>500 CFU/ml</td></tr></table>

aThe higher concentrations needed to obtain $100 \%$ positivity are not a concern as the enriched LIM Broth culture is expected to have GBS organism concentrations well-above the assay LoD.

# e. Analytical specificity:

A study was conducted to determine the cross-reactivity of 75 microorganisms (or gDNA/RNA) that represented various non-GBS groups of Streptococci, along with other bacteria, parasites, and viruses normally found in vaginal and anal flora, with the GenePOC GBS LB Assay. Bacteria and yeasts were tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / m l }$ in SB, whereas viruses, parasites and human DNA were tested at $\ge 1 0 ^ { 5 } \mathrm { D N A }$ or RNA $\mathrm { c p / m l }$ in SB. Samples were tested in triplicate and spiked into negative enriched LIM Broth.

The results for all replicates in the cross-reactivity study were negative, demonstrating that the organisms or gDNA/RNA tested at these concentrations did not cross react with the GenePOC GBS LB Assay. During the study, no unresolved or indeterminate results were obtained with any of the organisms tested. Table 8 shows the cross-reactivity panel tested for this study.

Table 8. Cross-Reactivity Panel   

<table><tr><td colspan="2" rowspan="1">Bacteria (strain or gDNA)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Mycoplasma genitalium gDNA</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">Mycoplasma hominis gDNA</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">Propionibacterium acnes</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium breve</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium linens</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovar Dublin</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis gDNA</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovarMinneapolis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovartyphimurium</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovar Newport</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Staphylococcus aureus (Cowan)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">Streptococcus anginosus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Streptococcus bovis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae subsp. disgalactiae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae equisimilis</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Streptococcus oralis</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">Streptococcus uberis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus delbreuckii</td><td colspan="1" rowspan="1">Ureaplasma urealyticum gDNA</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Yeasts</td><td></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida parapsilosis</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Candida tropicalis</td></tr><tr><td colspan="1" rowspan="1">Viruses (gDNA or RNA)</td><td></td></tr><tr><td colspan="1" rowspan="1">HerpesSimplexVirus-1 gDNA</td><td colspan="1" rowspan="1">Norovirus GII RNA</td></tr><tr><td colspan="1" rowspan="1">HerpesSimplexVirus-2 gDNA</td><td colspan="1" rowspan="1">Human Papillomavirus (HPV) gDNA</td></tr><tr><td colspan="1" rowspan="1">Parasites (gDNA)</td><td></td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis gDNA</td><td colspan="1" rowspan="1">Trichomonas vaginalis gDNA</td></tr></table>

# f. Interfering Studies

# Microbial Interference

This study was performed to evaluate potential interference from common microorganisms (associated with vaginal/rectal flora) with the detection of GBS by the GenePOC GBS LB Assay. All testing was completed with three reagent lots and three revogene instruments. For testing, non-target organisms were grouped into 7 pools according to Table 9 shown below. Pools were diluted to obtain a concentration of $1 0 ^ { 5 }$ or $1 0 ^ { 6 } \mathrm { C F U / m l }$ (or $\mathrm { c p / m l } )$ ) of SB and tested in triplicate. Each of the seven pools was a tested in the presence of negative LIM Broth culture with two GBS strains. Both ATCC 13813 and ATCC 12403 were tested at $7 3 5 \mathrm { C F U / m l }$ . No interference from non-target organisms was observed, except for GBS strains in the presence with Enterococcus faecalis (ATCC 19433), Enterococcus faecium (ATCC 19434), and Lactobacillus acidophilus (ATCC 4356). Interference with Enterococcus faecalis (ATCC 19433) was observed with both GBS strains at $1 0 ^ { 6 } \mathrm { C F U / m l }$ . No interference was observed with both GBS strains when Enterococcus faecalis was lowered to $1 0 ^ { 5 } \mathrm { C F U / m l }$ . For Enterococcus faecium, all three replicates of GBS strain ATCC 12403 were detected at $1 0 ^ { 4 } \mathrm { C F U / m l }$ . At $1 0 ^ { 5 } \mathrm { C F U / m l }$ of Lactobacillus acidophilus, GBS strain ATCC 12403 was inhibited. When the concentration of Lactobacillus acidophilus was lowered to $1 0 ^ { 4 } \mathrm { C F U / m l }$ , all three replicates of GBS strain ATCC 12403 could be detected. All pools, when tested in the absence of GBS, yielded the expected negative result with the assay. No interference with the internal process control was observed when microorganisms were present in high loads ( $1 0 ^ { 5 }$ or $1 0 ^ { 6 } \mathrm { C F U / m l }$ of SB). The initial rate of unresolved results obtained during the study was $0 . 8 \%$ (1/133). This unresolved sample replicate was successfully repeated. No indeterminate results were obtained. The potential for high concentrations of $E$ . faecalis, $E .$ . faecium, and $L$ . acidophilus to inhibit the detection of GBS is noted as a Limitation in the device labeling. The results of the Microbial Interference Study are acceptable.

Table 9. List of Seven Pools for the Microbial Interference Study   

<table><tr><td rowspan=1 colspan=1>Poola</td><td rowspan=1 colspan=1>Strains</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Enterococcus faecalis, ATCC 19433</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Enterococcus faecium, ATCC 19434</td></tr><tr><td rowspan=5 colspan=1>3</td><td rowspan=1 colspan=1>Acinetobacter baumannii, ATCC 19606</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis, ATCC 27370</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis, ATCC 25285</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni, ATCC 33560</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile, ATCC 9689</td></tr><tr><td rowspan=5 colspan=1>4</td><td rowspan=1 colspan=1>Corynebacterium genitalium, ATCC 33031</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli, ATCC 11775</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis, ATCC 49145</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca, ATCC 8724</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus, ATCC 4356</td></tr><tr><td rowspan=6 colspan=1>5</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae, ATCC 43069</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcus anaerobius, ATCC 27337</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium acnes, ATCC 11827</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa, ATCC 35554</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica serovar Dublin,ATCC 39184</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens, ATCC 13880</td></tr><tr><td rowspan=6 colspan=1>6</td><td rowspan=1 colspan=1>Shigella sonnei, ATCC 29930</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus, ATCC 33592</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis, ATCC 14990</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes, ATCC 19615</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica, ATCC 27729</td></tr><tr><td rowspan=1 colspan=1>Candida albicans, ATCC 20735</td></tr><tr><td rowspan=1 colspan=1>Poola</td><td rowspan=1 colspan=1>gDNA or RNA</td></tr><tr><td rowspan=5 colspan=1>7</td><td rowspan=1 colspan=1>Chlamydia trachomatis, VR-878D</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus-1, ATCC VR-539D</td></tr><tr><td rowspan=1 colspan=1>Human Papillomavirus (HPV), ATCC VR-3240SD</td></tr><tr><td rowspan=1 colspan=1>Trichomonas vaginalis, ATCC 30001</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium, ATCC BAA-2641S</td></tr></table>

aEven if a pool only contained one microorganism, the term “pool” was kept all along the list.

# Interfering Substances

This study was conducted to evaluate the potential interference of a panel of 31 endogenous and exogenous substances with the GenePOC GBS LB Assay.

Potentially interfering substances were distributed into 11 pools for testing. Samples were prepared in triplicate with one of two strains of GBS (ATCC 13813 and ATCC 12403) in negative LIM Broth and diluted to $7 3 5 \mathrm { C F U / m l }$ in SB in the presence of the potential interferent. LIM Broth served as a negative sample. Interfering substances were added at concentrations that would be expected to be observed among patient specimens. Interference was observed against GBS strain ATCC 12403 in the

presence of Pool E $\mathrm { 1 . 5 \mu g / m l }$ loperamide hydrochloride, $4 5 ~ { \mu \mathrm { g / m l } }$ bismuth subsalicylate, and $0 . 7 3 5 ~ { \mu \mathrm { g / m l } }$ sennosides). When tested individually, no interference on the detection of GBS strain ATCC 12403 was observed with the substances from Pool E. Interference pools used for the study are highlighted in Table 10 and Table 11 below. Fecal fat was not tested due to lack of supply. One indeterminate result was obtained for a positive external control to yield an initial indeterminate rate of $0 . 7 5 \%$ (1/133). After re-testing, the control yielded a valid positive result. No unresolved results were obtained during the study. Eight samples were excluded due to: errors in the placement of the PIE cartridge, wrong sample tested, or wrong PIE lot tested. All samples were re-tested according to the study protocol and the re-test results were used to replace the previous results. Results of the Interference Study are acceptable.

Table 10. List of Exogenous Interferents Tested and Grouping into the Six Pools   

<table><tr><td colspan="1" rowspan="1">Pool</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Substances</td><td colspan="1" rowspan="1">CommercialName</td><td colspan="1" rowspan="1">ActiveIngredient</td><td colspan="1" rowspan="1">Absorption,Substances</td><td colspan="1" rowspan="1">Concentrationin SBT</td></tr><tr><td colspan="1" rowspan="5">A</td><td colspan="1" rowspan="1">SI1</td><td colspan="1" rowspan="1">Fungicide</td><td colspan="1" rowspan="1">Micatin</td><td colspan="1" rowspan="1">Miconazolenitrate cream</td><td colspan="1" rowspan="1">Topical, cream</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="1">SI2</td><td colspan="1" rowspan="1">Hemorrhoidcooling gel</td><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">PhenylephrineHCI</td><td colspan="1" rowspan="1">Topical, gel</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="1">SI3</td><td colspan="1" rowspan="1">Lubricating gel</td><td colspan="1" rowspan="1">K-YPersonalLubricant</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Topical, gel</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="1">SI4</td><td colspan="1" rowspan="1">Body powder</td><td colspan="1" rowspan="1">Vagisildeodorantpowder</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Topical, powder</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="1">SI5</td><td colspan="1" rowspan="1">Moisturizinglotion</td><td colspan="1" rowspan="1">Aveenomoisturizinglotion</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Topical, cream</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="4">B</td><td colspan="1" rowspan="1">SI6</td><td colspan="1" rowspan="1">Body oil</td><td colspan="1" rowspan="1">NeutrogenaBody oil</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Topical, Oil</td><td colspan="1" rowspan="1">0.023% v/v</td></tr><tr><td colspan="1" rowspan="1">SI7</td><td colspan="1" rowspan="1">DeodorantSpray</td><td colspan="1" rowspan="1">Summer'sEve Spray</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Topical, spray</td><td colspan="1" rowspan="1">0.023% v/v</td></tr><tr><td colspan="1" rowspan="1">SI8</td><td colspan="1" rowspan="1">Enemas</td><td colspan="1" rowspan="1">Life BRANDHeavyMineral OilUSP</td><td colspan="1" rowspan="1">Mineral oil</td><td colspan="1" rowspan="1">Topical, oil</td><td colspan="1" rowspan="1">0.023% v/v</td></tr><tr><td colspan="1" rowspan="1">SI9</td><td colspan="1" rowspan="1">Antimicrobials</td><td colspan="1" rowspan="1">Canesten</td><td colspan="1" rowspan="1">Clotrimazole</td><td colspan="1" rowspan="1">Topical, cream</td><td colspan="1" rowspan="1">0.023% w/v</td></tr><tr><td colspan="1" rowspan="4">C</td><td colspan="1" rowspan="1">SI10</td><td colspan="1" rowspan="1">Enemas</td><td colspan="1" rowspan="1">Pentasa</td><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">Oral, powder</td><td colspan="1" rowspan="1">0.495 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI11</td><td colspan="1" rowspan="1">RadiologyOralCompounds</td><td colspan="1" rowspan="1">BariumSulfate</td><td colspan="1" rowspan="1">BariumSulfate</td><td colspan="1" rowspan="1">Oral, powder</td><td colspan="1" rowspan="1">0.113 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI12</td><td colspan="1" rowspan="1">GastritisMedications</td><td colspan="1" rowspan="1">Nexium</td><td colspan="1" rowspan="1">EsomeprazoleMagnesium</td><td colspan="1" rowspan="1">Oral, powder</td><td colspan="1" rowspan="1">0.011 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI13</td><td colspan="1" rowspan="1">Antimicrobials</td><td colspan="1" rowspan="1">Flagyl</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Oral, powder</td><td colspan="1" rowspan="1">0.016 μg/ml</td></tr><tr><td colspan="1" rowspan="3">D</td><td colspan="1" rowspan="1">SI14</td><td colspan="1" rowspan="1">Non-SteroidalAnti-InflammatoryMedications</td><td colspan="1" rowspan="1">Aleve</td><td colspan="1" rowspan="1">NaproxenSodium</td><td colspan="1" rowspan="1">Oral, pill</td><td colspan="1" rowspan="1">0.071 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI15</td><td colspan="1" rowspan="1">Antimicrobials</td><td colspan="1" rowspan="1">DIFLUCANOne</td><td colspan="1" rowspan="1">Fluconazol</td><td colspan="1" rowspan="1">Oral, pill</td><td colspan="1" rowspan="1">0.020 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI16</td><td colspan="1" rowspan="1">GastritisMedications</td><td colspan="1" rowspan="1">Tums</td><td colspan="1" rowspan="1">CalciumCarbonate</td><td colspan="1" rowspan="1">Oral, pill</td><td colspan="1" rowspan="1">1.200 μg/ml</td></tr><tr><td colspan="1" rowspan="3">E</td><td colspan="1" rowspan="1">SI17</td><td colspan="1" rowspan="1">Anti-DiarrhealMedication</td><td colspan="1" rowspan="1">Imodium</td><td colspan="1" rowspan="1">LoperamideHydrochloride</td><td colspan="1" rowspan="1">Oral, pill</td><td colspan="1" rowspan="1">0.023 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI18</td><td colspan="1" rowspan="1">Anti-DiarrhealMedication</td><td colspan="1" rowspan="1">PeptoBismol</td><td colspan="1" rowspan="1">Bismuthsubsalicylate</td><td colspan="1" rowspan="1">Oral, liquid</td><td colspan="1" rowspan="1">0.675 μg/ml</td></tr><tr><td colspan="1" rowspan="1">SI19</td><td colspan="1" rowspan="1">Laxatives</td><td colspan="1" rowspan="1">Senokot</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">Oral, pill</td><td colspan="1" rowspan="1">0.011 μg/ml</td></tr><tr><td colspan="1" rowspan="3">F</td><td colspan="1" rowspan="1">SI20</td><td colspan="1" rowspan="1">Spermicidal</td><td colspan="1" rowspan="1">Trojan withSpermicidalLubricantCondom</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">Condom</td><td colspan="1" rowspan="1">0.023 % v/v</td></tr><tr><td colspan="1" rowspan="1">SI21</td><td colspan="1" rowspan="1">MoistTowelettes</td><td colspan="1" rowspan="1">EquateFlushableMoist Wipes</td><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">Moist wipes</td><td colspan="1" rowspan="1">0.023 % v/v</td></tr><tr><td colspan="1" rowspan="1">SI22</td><td colspan="1" rowspan="1">MoistTowelettes</td><td colspan="1" rowspan="1">Wet Ones</td><td colspan="1" rowspan="1">BenzalkoniumChloride</td><td colspan="1" rowspan="1">Moist wipes</td><td colspan="1" rowspan="1">0.023 % v/v</td></tr></table>

Table 11. List of Endogenous Interferents Tested and Grouping into the Five Pools   

<table><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Substances</td><td rowspan=1 colspan=1>Concentration in SBT</td></tr><tr><td rowspan=2 colspan=1>G</td><td rowspan=1 colspan=1>SI23</td><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=1 colspan=1>SI24</td><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>15000 cells/ml</td></tr><tr><td rowspan=2 colspan=1>H</td><td rowspan=1 colspan=1>SI25</td><td rowspan=1 colspan=1>Amniotic Fluid</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=1 colspan=1>SI26</td><td rowspan=1 colspan=1>Mucous</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=2 colspan=1>I</td><td rowspan=1 colspan=1>SI27</td><td rowspan=1 colspan=1>Seminal Fluid</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=1 colspan=1>SI28</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=2 colspan=1>J</td><td rowspan=1 colspan=1>SI29</td><td rowspan=1 colspan=1>Feces</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=1 colspan=1>SI30</td><td rowspan=1 colspan=1>Meconium</td><td rowspan=1 colspan=1>0.023% v/v</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>SI31</td><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>4.650 ng/ml</td></tr></table>

# g. Carry-over/Cross Contamination

Carry-over and cross contamination for the GenePOC GBS LB Assay was assessed with two operators conducting a total of 20 runs of 8 samples in each run. Samples were tested in an alternating pattern with high positive samples andnegative samples (within-run and between-run). No carry-over and cross contamination was observed. The overall percent agreement was $100 \%$ for positive and negative samples. No unexpected results, including unresolved or indeterminate results, were obtained during the study.

# h. Assay cut-off:

The assay cut-off was established and validated by evaluating the performance of the GenePOC GBS LB Assay on the revogene instrument in two separate studies. A total of 258 LIM specimens were tested with the GenePOC GBS LB Assay and with the Reference Culture Method. In the first study, the assay cut-off and thresholds for the GenePOC GBS LB Assay were established by testing 133 fresh specimens previously enriched in LIM Broth. Testing was conducted on one revogene instrument with three unique GBS LB PIE lots. In the second study, testing was performed to validate the cut-offs from an additional 125 frozen LIM specimens. Testing was conducted on one revogene with the same three GBS LB PIE lots utilized for Study 1. The cut-off values for a positive result for both the internal process control and GBS were set at Ct values ${ < } 4 5$ and an EP Threshold at 125. Using these cut-offs, a sensitivity of $9 8 . 3 \%$ (58/59) $9 5 \%$ CI: $9 1 . 0 { - } 9 9 . 7 \% )$ and a specificity of $9 9 . 0 \%$ (197/199)( $9 5 \%$ CI: $9 6 . 4 \substack { - 9 9 . 7 \% }$ ) were reported after testing the 258 specimens.

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable. Please refer to 3a below.

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity:

GenePOC conducted a prospective multi-center trial at 4 geographically diverse clinical trial sites (two Canadian and two US clinical sites). Vaginal/rectal swab specimens were collected from pregnant women at 35-37 weeks of gestation for whom GBS diagnostic procedures were indicated and ordered. Vaginal/rectal specimen swabs were collected and cultured according to established guidelines; swabs were placed in LIM Broth and incubated for 18-24 hrs at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ in ambient air or $5 \% \mathrm { C O } _ { 2 }$ . Residual de-identified LIM Broth cultures were used for testing with the GenePOC GBS LB Assay. A total of 839 vaginal/rectal swab specimens initially met the study criteria. Sixty-eight (68) specimens were shown to be non-compliant for the following reasons:

· Transport and storage times exceeded study protocol requirements (1) Samples did not have all test results as required by the study protocol (4) Reference Method and/or GenePOC GBS LB Assay not performed according to the clinical trial protocol (63)

For the Reference Culture Method, LIM Broth cultures were subcultured to a nonselective blood agar, and all colonies with a characteristic appearance suggestive of GBS were screened to confirm the presence of GBS using established laboratory methods: gram stain, catalase, latex agglutination, and CAMP test. The CAMP test was only used at one site in addition to the other biochemical tests listed. If GBS was not presumptively identified after incubation for 18-24 hrs on sheep blood agar plates, the plates were further incubated overnight and then re-inspected to identify suspect colonies. Originally, the Reference Culture Method revealed 156 positive specimens. To improve the recovery of GBS by the Reference Culture Method, a supplemental study was conducted with frozen LIM Broth culture aliquots (in glycerol) from all negative samples by the Reference Culture Method $\mathrm { ( n { = } } 6 1 5$ ). Aliquots were subcultured onto non-selective agar and again examined for the presence of GBS colonies. GBS identification was confirmed by biochemical testing according to the clinical reference method protocol. Any positive GBS samples identified by this method were combined with the results of the first culture. No GenePOC GBS LB Assay results were changed for the supplemental study. For six isolates in the supplemental study, results were excluded due to inconclusive culture, the frozen LIM Broth tube was not found, or the frozen LIM Broth tube was not stored with glycerol; therefore, the original Reference Culture Method results for the six samples were included in the analysis instead of excluding these samples from the final analysis. With the addition of the secondary study, the Reference Culture Method was considered to be a Composite Reference Method. A total of 14 additional reference positive samples were identified by the supplemental study and added to previous results for a total of 170 reference positive results [156 (original) $+ 1 4$ (supplemental study)].

The GenePOC GBS LB Assay was performed according to the package insert. A 15 $\mu \mathrm { l }$ aliquot of enriched LIM Broth culture was transferred to a sample buffer tube (SBT), and a $1 5 0 ~ \mu \mathrm { l }$ was loaded into each PIE. The assay was run on the revogene instrument. Residual SBT with sample was stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for repeat testing, if needed. One hundred thirty-seven (137) GenePOC GBS LB runs out of one hundred thirty-nine (139) were valid $9 8 . 6 \%$ success rate). The overall initial unresolved and indeterminate rates were $0 . 7 4 \%$ (6/812) and $1 . 4 8 \%$ (12/812), respectively. The final non-reportable result rate was $0 . 2 5 \%$ for the GenePOC GBS LB Assay. Results of the GenePOC GBS LB Assay were compared to a Composite Reference Culture Method as shown in Table 12.

Table 12. Clinical Performance Data for the GenePOC GBS LB Assays. Composite Reference Culture Method (all sites)   

<table><tr><td rowspan=2 colspan=1>GBS LB Assay</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>27b</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>574</td><td rowspan=1 colspan=1>581</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>601</td><td rowspan=1 colspan=1>771</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:: 95.9% (95%CI: 91.7 - 98.0 %)Specificity:: 95.5% (95%CI: 93.5 - 96.9 %)</td></tr></table>

a 5 out of 7 false negative GenePOC GBS LB results were tested on a FDA-cleared molecular device and yielded negative results. An FDA-cleared GBS NAAT was used to test all samples at only one site. $^ b I O$ out 27 false positive GenePOC GBS LB results were tested on a FDA-cleared molecular device and yielded positive results. An FDA-cleared GBS NAAT was used to test all samples at only one site.

b. Clinical Specificity: See above   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

In the investigational study with the GenePOC GBS LB Assay on the revogene (K170558), the overall GBS prevalence rate as determined by the Composite Reference Method (defined above) was $2 2 . 0 \%$ (170/771).

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.