# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K170604   
B. Purpose for Submission: New device

# C. Measurand:

Adenovirus, Human Metapneumovirus (hMPV), Influenza A (Flu A), Influenza A subtype H1 (Flu A H1), Influenza A subtype H1- 2009 (Flu A H1-2009), Influenza A subtype H3 (Flu A H3), Influenza B (Flu B), Coronavirus 229E (CoV-229E), Coronavirus HKU1 (CoVHKU1), Coronavirus NL63 (CoV-NL63), Coronavirus OC43 (CoV-OC43), Parainfluenza virus 1 (PIV 1), Parainfluenza virus 2 (PIV 2), Parainfluenza virus 3 (PIV 3), Parainfluenza virus 4 (PIV 4), Human Rhinovirus/Enterovirus (HRV/HEV), Respiratory Syncytial Virus (RSV), Bordetella parapertussis (IS1001), Bordetella pertussis (ptxP), Chlamydia pneumoniae, and Mycoplasma pneumoniae nucleic acids target sequences.

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the FilmArray $^ { \mathfrak { R } } 2 . 0$ or FilmArray® Torch systems for the simultaneous qualitative in vitro detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) collected in viral transport media and obtained from individuals suspected of respiratory tract infections.

E. Applicant: BioFire Diagnostics, LLC

F. Proprietary and Established Names: FilmArray Respiratory Panel 2 (RP2) Common Name: FilmArray RP2

G. Regulatory Information:

<table><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Regulation Section</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OTG</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZE</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZX</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZY</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZZ</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">001</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation forClinical Multiplex TestSystems</td><td colspan="1" rowspan="1">Clinical Chemistry(75)</td></tr><tr><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation forClinical Multiplex TestSystems</td><td colspan="1" rowspan="1">Clinical Chemistry(75)</td></tr></table>

# H. Intended Use:

1. Intended use: The FilmArray® Respiratory Panel 2 (RP2) is a multiplexed nucleic acid test intended for use with FilmArray® 2.0 or FilmArray® Torch systems for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray® RP2:

· Adenovirus   
· Coronavirus 229E   
· Coronavirus HKU1   
· Coronavirus NL63   
· Coronavirus OC43   
· Human Metapneumovirus   
Human Rhinovirus/Enterovirus Influenza A, including subtypes H1, H1-2009, and H3   
· Influenza B   
· Parainfluenza Virus 1 Parainfluenza Virus 2   
Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus   
· Bordetella parapertussis (IS1001)   
· Bordetella pertussis $( p t x P )$   
· Chlamydia pneumoniae   
Mycoplasma pneumoniae

The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms: the agent(s) detected by the FilmArray® RP2 may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the genetic similarity between Human Rhinovirus and Enterovirus, the FilmArray® RP2 cannot reliably differentiate them. A positive FilmArray® RP2 Rhinovirus/Enterovirus result should be followed up using an alternate method (e.g., cell culture or sequence analysis) if differentiation is required.

Performance characteristics for Influenza A were established when Influenza A H1-2009 and A H3 were the predominant Influenza A viruses in circulation. Performance of detecting Influenza A may vary if other Influenza A strains are circulating or a novel Influenza A virus emerges. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health departments for

testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: FilmArray 2.0 or FilmArray Torch systems

# I. Device Description:

The FilmArray Respiratory Panel 2 (RP2) is designed to simultaneously identify 21 different potential pathogens (see the Intended Use section) of respiratory tract infection from a single NPS specimen in approximately 45 minutes. FilmArray RP2 is compatible with BioFire Diagnostics’ (BioFire) PCR-based in vitro diagnostic FilmArray 2.0 and FilmArray Torch systems for infectious disease testing. A specific software module (i.e., FilmArray RP2 pouch module) is used to perform FilmArray RP2 testing on these systems.

A test is initiated by loading Hydration Solution into one port of the FilmArray pouch and a NPS sample (in transport media) mixed with the provided Sample Buffer into the other port of the FilmArray RP2 pouch and placing it in a FilmArray instrument. The FilmArray pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically-controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (RT-PCR) reaction, PCR1. The products from first stage PCR are then diluted and combined with a fresh, primerfree master mix and a fluorescent double stranded DNA binding dye (LC Green® Plus, BioFire Diagnostics, LLC). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The second stage PCR, or nested PCR, PCR2, is performed in singleplex fashion in each well of the array. At the conclusion of the second stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the second stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Materials Provided with the FilmArray RP2 Kit:

Each kit contains sufficient reagents to test 6 samples (6-test kit; RFIT-ASY-0130) or 30 samples (30-test kit; RFIT-ASY-0129):

· Individually-packaged FilmArray RP2 pouches Single-use $( 1 . 0 \mathrm { m L } )$ Sample Buffer ampoules Single-use pre-filled $( 1 . 5 \mathrm { m L }$ ) Hydration Injection Vials (blue) Single-use Sample Injection Vials (red) Individually-packaged Transfer Pipettes

# Materials Needed but Not Provided with the FilmArray RP2 Kit:

· $10 \%$ bleach solution

FilmArray system including:

· FilmArray 2.0 or FilmArray Touch and accompanying software · FilmArray Pouch Loading Station

# Interpretation of Results

When PCR2 is complete, the FilmArray instrument performs a DNA melting analysis on the PCR products and measures the fluorescence signal generated in each well. The FilmArray Software then performs several analyses and assigns a final assay result. The steps in the analyses are described below.

Analysis of melt curves   
The FilmArray Software evaluates the DNA melt curve for each well of the PCR2 array   
to determine if a PCR product was present in that well. If the melt profile indicates the   
presence of a PCR product, then the analysis software calculates the melting   
temperature (Tm) of the curve and compares it against the expected Tm range for the   
assay. If the software determines that the Tm falls inside the assay-specific Tm range, the melt curve is called positive. If the software determines that the melt curve is not in the appropriate Tm range, the melt curve is called negative.   
Analysis of replicates   
Once melt curves have been identified, the software evaluates the three replicates for   
each assay to determine the assay result. For an assay to be called positive, at least two   
of the three associated melt curves must be called positive, and the Tm for at least two of the three positive melt curves must be similar (i.e., within $1 ^ { \circ } \mathrm { C }$ ). Assays that do not   
meet these criteria are called negative.

For the following organisms detected by the FilmArray RP2, the organism is reported as “Detected” if a single corresponding assay is positive.

Coronavirus 229E · Coronavirus HKU1 · Coronavirus NL63 Coronavirus OC43 · Human Metapneumovirus Human Rhinovirus/Enterovirus · Influenza B Parainfluenza Virus 1 Parainfluenza Virus 2 Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus Bordetella parapertussis (IS1001) Bordetella pertussis (ptxP) Chlamydia pneumoniae · Mycoplasma pneumoniae

The test results for Adenovirus and Influenza A (including subtyping) depend on the interpretation of results from more than one corresponding assay. Interpretation and actions for these results are provided below.

· Adenovirus The FilmArray RP2 pouch contains five different assays (Adeno2, Adeno3, Adeno6, Adeno7.1, and Adeno8) for the detection of Adenovirus. The FilmArray Software interprets each of these assays independently and the results are combined as a final test result for the virus. If one or any combination of assays is positive, the test report result will be Adenovirus “Detected”. If all assays are negative, the test report result will be Adenovirus “Not Detected”. Influenza A and Subtyping The assays in the FilmArray RP2 are designed to both detect Influenza A and to differentiate commonly occurring hemagglutinin subtypes. To accomplish this, the FilmArray RP2 uses two Influenza A assays, FluA-pan-1 and FluA-pan-2, and three subtyping assays, FluA-H1-2, FluA-H1-2009, and FluA-H3, directed at the respective hemagglutinin gene. Each of the individual assays is interpreted independently and the test result reported for Influenza A is based on the combined results of the five assays as outlined in Table 1.

Table 1: Possible Assay Results for Influenza A and Corresponding Interpretation   

<table><tr><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>FluA-panAssays(n=2)</td><td rowspan=1 colspan=1>FluA-H1-2</td><td rowspan=1 colspan=1>FluA-H1-2009</td><td rowspan=1 colspan=1>FluA-H3</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Influenza A Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=2 colspan=1>Multiple infectionsare possible butrare a, retest toconfirm result b</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A (no subtypedetected)</td><td rowspan=1 colspan=1>2 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Retest</td></tr><tr><td rowspan=1 colspan=1>Influenza A Equivocal</td><td rowspan=1 colspan=1>1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>Retest</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 Equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3 Equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009Equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

a The FilmArray RP2 can simultaneously detect multiple influenza viruses contained in multivalent vaccines. b Repeated multiple positives should be further confirmed by other FDA cleared Influenza subtyping tests.

Influenza A (no subtype detected):

If both of the FluA-pan assays are positive, but none of the hemagglutinin subtyping assays are positive, then the interpretation is Influenza A (no subtype detected). This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the presence of a novel Influenza A strain. In both cases, the sample in question should be retested. If the retest provides a different result, test the sample a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the RP2 pouches should be verified by testing with appropriate external control materials (known positive samples for Influenza A H1, Influenza A H3 and Influenza A H1-2009), and a negative control should also be run to test for PCR-product contamination. If the FilmArray RP2 accurately identifies the external and negative controls, contact the appropriate public health authorities for confirmatory testing.

# FilmArray RP2 Test Report

The FilmArray RP2 test report is automatically displayed upon completion of a run and can be printed or saved as a PDF file. Each report contains a Run Summary, a Result Summary, and a Run Details section. An example of the test report is presented below:

<table><tr><td colspan="6">FilmArray Respiratory Panel 2</td></tr><tr><td colspan="6">Run Summary</td></tr><tr><td>Sample ID:</td><td></td><td>RP2ex_33_Equiv</td><td>Run Date:</td><td></td><td>18 Jan 2017 21 PM</td></tr><tr><td>Equivocal:</td><td>Detected:</td><td>Adenovirus Influenza A</td><td></td><td>Controls:</td><td>Passed</td></tr><tr><td colspan="6">Result Summary</td></tr><tr><td colspan="6">Viruses Detected Adenovirus</td></tr><tr><td>Not Detected Not Detected Not Detected Not Detected</td><td>Coronavirus 229E Coronavirus HKU1 Coronavirus NL63</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="6"></td></tr><tr><td>Not Detected</td><td>Coronavirus OC43 Human Metapneumovirus</td><td></td><td></td><td></td><td></td></tr><tr><td>Not Detected Equivocal</td><td></td><td>Human Rhinovirus/Enterovirus</td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td>Influenza A</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Influenza B</td><td></td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Parainfluenza Virus 1</td><td></td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td>Parainfluenza Virus 2</td><td></td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td>Parainfluenza Virus 3</td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td>Parainfluenza Virus 4</td><td></td><td></td><td></td></tr><tr><td></td><td>Respiratory Syncytial Virus</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="6"></td></tr><tr><td>Not Detected</td><td>Bordetella parapertussis (IS1001)</td><td>Bacteria</td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td>Bordetella pertussis (ptxP)</td><td></td><td></td><td></td></tr><tr><td>Not Detected</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Chlamydia pneumoniae Mycoplasma pneumoniae</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="6">Not Detected</td></tr><tr><td>Run Details</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Run Status:</td><td>Pouch: RP2 v1.1</td><td></td><td>Protocol:</td><td>NPS2 v3.1 JDoe</td><td></td></tr><tr><td></td><td>Completed</td><td></td><td>Operator:</td><td></td><td></td></tr><tr><td></td><td>Serial No..</td><td>6265525</td><td>Instrument:</td><td></td><td>TM8CCF3</td></tr><tr><td></td><td>Lot No.:</td><td>161013E</td><td></td><td></td><td></td></tr></table>

Run Summary

The Run Summary section of the test report provides the Sample ID, time and date of the run, control results and an overall summary of the test results. Any organism with a “Detected” result will be listed in the corresponding field of the summary. If all of the organism assays were negative then “None” will be displayed in the Detected field. All Influenza A equivocal results (refer to Table 2) will be displayed in the Equivocal field. Controls are listed as “Passed”, “Failed” or “Invalid”. Table 2 below provides additional information for each of the possible control field results.

Table 2: Interpretation of Controls Field on the FilmArray RP2 Test Report   

<table><tr><td colspan="1" rowspan="1">ControlResult</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">Passed</td><td colspan="1" rowspan="1">The run was successfully completedANDBoth pouch controls were successful.</td><td colspan="1" rowspan="1">NoneReport the results provided on the test report</td></tr><tr><td colspan="1" rowspan="1">Failed</td><td colspan="1" rowspan="1">The run was successfully completedBUTAt least one of the pouch controls(RNA Process Control and/or PCR2Control) failed.</td><td colspan="1" rowspan="1">Repeat the test using a new pouch.If the error persists, contact Technical Supportfor further instruction.</td></tr><tr><td>Control Result</td><td>Explanation</td><td>Action</td></tr><tr><td>Invalid</td><td>The controls are invalid because the run did not complete. (Typically this indicates a software or hardware error).</td><td>Note any error codes displayed during the run and the Run Status field in the Run Details section of the report. Refer to the appropriate FilmArray Operator's Manual or contact Technical Support for further instruction. Once the error is resolved, repeat the test or repeat the test using another instrument.</td></tr></table>

· Results Summary

The Result Summary section of the test report lists the result for each target tested by the panel. Possible results for each organism except for Influenza A and subtyping are “Detected”, “Not Detected”, or “Invalid”.

Table 3 below provides an explanation for each interpretation and any follow-up necessary to obtain a final result.

Table 3: Reporting of Results and Required Actions   

<table><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Detected a</td><td rowspan=1 colspan=1>The run was successfully completedANDThe pouch controls were successful (Passed)ANDThe assay(s) for the organism were POSITIVE(i.e., met the requirements for a positive result described inthe Interpretation of Results section above)</td><td rowspan=1 colspan=1>Report results.</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>The run was successfully completedANDThe pouch controls were successful (Passed)ANDThe assay(s) for the organism were NEGATIVE(i.e., did not meet the requirements for a positive resultdescribed in the Assay Interpretation section above)</td><td rowspan=1 colspan=1>Report results.</td></tr><tr><td rowspan=1 colspan=1>Equivocal</td><td rowspan=1 colspan=1>The run was successfully completedANDThe pouch controls were successful (Passed)ANDThe combination of positive and negative assay results forInfluenza A were inconclusive(see Table 2)</td><td rowspan=1 colspan=1>Retest the originalspecimen using a newpouch and report theresults of the retest.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The pouch controls were not successful (Failed)ORThe run was not successful(Run Status displayed as: Aborted, Incomplete, InstrumentError or Software Error)</td><td rowspan=1 colspan=1>See Table 3.</td></tr></table>

a If four or more organisms are detected in a specimen, retesting is recommended to confirm the polymicrobial result.

# Run Details

The Run Details section provides additional information about the run including: pouch information (type, lot number, and serial number), Run Status (Completed,

Incomplete, Aborted, Instrument Error, Instrument Communication Error, or Software Error), the protocol that was used to perform the test, the identity of the operator that performed the test, and the instrument used to perform the test.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Respiratory Panel (RP)

2. Predicate K number(s): K160068

3. Comparison with predicate(s):

<table><tr><td rowspan=1 colspan=3>Similarities or Differences</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel 2 (RP2)(K170604)</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel (RP)(K160068)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>NPS (in transport media)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus 229E, CoronavirusHKU1, Coronavirus NL63, CoronavirusOC43, Human Metapneumovirus, InfluenzaA, Influenza A subtype H1, Influenza Asubtype H3, Influenza A subtype H1-2009,Influenza B, Parainfluenza Virus 1,Parainfluenza Virus 2, Parainfluenza Virus 3,Parainfluenza Virus 4, HumanRhinovirus/Enterovirus, Respiratory SyncytialVirus, Bordetella parapertussis, Bordetellapertussis, Chlamydia pneumoniae, andMycoplasma pneumoniae</td><td rowspan=1 colspan=1>Same, except that it does not detectBordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>RNA/DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>FilmArray 2.0 or FilmArray Torch</td><td rowspan=1 colspan=1>FilmArray, FilmArray 2.0 or FilmArrayTorch</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>About 45 minutes</td><td rowspan=1 colspan=1>About 1 hour</td></tr><tr><td rowspan=1 colspan=1>Reagent Storage</td><td rowspan=1 colspan=1>Room temperature</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TestInterpretation</td><td rowspan=1 colspan=1>Automated test interpretation and reportgeneration. User cannot access raw data.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>Same</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

· FDA guidance document issued on August 27, 2014, titled “Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices”

· FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay”

· FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays”

· FDA guidance document issued on October 9, 2009, titled “Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays”

· FDA guidance document issued on March 13, 2007, titled “Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests”

· FDA guidance document issued on July 15, 2011, titled “Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses”

· FDA guidance document issued on April 25, 2005, titled “Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable”

· FDA guidance document issued on May 11, 2005, titled “Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices”

· FDA guidance document issued on September 9, 1999, titled “Off-The-Shelf Software Use in Medical Devices”

· FDA guidance document issued on January 11, 2002, titled “General Principle of Software Validation”

· FDA guidance document issued on November 30, 2004, titled “Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use”

· FDA guidance document issued on January 1, 2000, titled “Guidance for Industry and FDA on Alternative to Certain Prescription Device Labeling Requirements”

· Interference Testing in Clinical Chemistry; Clinical and Laboratory Standards Institute (CLSI Approved Guideline – Second Addition, EP7-A2 (2005)

· User Protocol for Evaluation of Qualitative Test Performance; Clinical and Laboratory Standards Institute (CLSI) Approved Guideline – Second Edition, EP12-A2 (January 2008)

· Molecular Diagnostic Methods for Infectious Diseases; Clinical and Laboratory Standards Institute (CLSI) Approved Guideline, MM3-A2 (February 2006)

· Evaluation of Stability of In Vitro Diagnostic Reagents; Clinical and Laboratory Standards Institute (CLSI) Approved Guideline, EP25-A (September 2009)

# L. Test Principle:

The FilmArray instrument, software, and pouch work together to perform sample lysis and purification, amplification, and detection of nucleic acid.

One of the primary functions of a FilmArray instrument is to drive the various steps in the testing process by moving liquids to appropriate locations within the pouch. Liquids are moved from the fitment to the blisters of the pouch by means of pneumatic actuators in the instrument piston block which press on the syringe-like plunging devices in the pouch fitment. There are 12 pistons that operate in a specified sequence to deliver reagents to the appropriate blisters in the pouch when they are needed.

Within the pouch, liquids are moved by using bladders and hard seals to exert pressure on the exterior of the pouch, such that the instrument is never in contact with the liquids contained in the pouch. The bladders are inflatable elastomeric membranes used to ‘squish’ the pouch blisters, thus forcing the liquid out of the compressed blister and along any connecting channels. The hard-seals are piston driven actuators used to direct the flow of liquids from the blisters by pinching shut the channels and blocking flow while pinched.

Thermal interactions between the pouch and instrument play a crucial role in the amplification of target nucleic acids. Temperature control is driven by a pair of numerically controlled Peltier devices; solid-state thermal control instruments fitted with calibrated temperature sensors and protective circuitry. These Peltier devices heat and cool to perform the first and second stage PCR reactions and carefully control the temperature across the array during DNA melting. The instrument uses a blue LED to illuminate the second stage PCR array and a digital camera to record fluorescence generated in the second stage PCR. The optical system is designed to detect the fluorescence signal generated during DNA melting.

The instrument communicates with the computer and the FilmArray software using Ethernet cables. The software provides instructions to the instrument to control each of the steps described above.

A detailed explanation of specific steps in the testing process is provided below:

# 1. Sample Lysis and Purification

Nucleic acid purification occurs in the first four blisters of the pouch using magnetic bead technology.

# a. Sample Lysis

Prior to loading the sample into the pouch, nucleases are inactivated by mixing the sample with a denaturing buffer (FilmArray Sample Buffer). The sample/sample buffer mixture is then loaded into the pouch via the injection port and pouch vacuum pulls the liquid into the sample well of the pouch fitment.

During the addition of the sample/sample buffer mixture, the template for the RNA Process Control is rehydrated and introduced into the reaction mixture. The RNA Process Control targets an mRNA of Schizosaccharomyces pombe, which is freezedried into the sample well of each pouch. The S. pombe is processed in parallel with the patient sample through each step of the test including nucleic acid extraction, reverse transcription (RT), PCR1, PCR2, and DNA melting. A positive result for the RNA Process Control indicates that all steps in the test are functioning properly.

The instrument activates a piston located above the fitment to move the sample/sample buffer mixture from the fitment into the trapezoidal sample lysis blister and then heatseals the fitment to prevent sample loss. The sample lysis blister contains ceramic beads. The instrument then activates the bead beater assembly which rotates a metal bar that strikes the pouch for 60 seconds to lyse organisms in the sample by creating high speed impacts between the sample and beads (bead beating). At the conclusion of the bead beating process, cells and organisms are lysed and the cell contents, including the nucleic acids, are released into the reaction mixture.

# b. Nucleic Acid Isolation

The instrument inflates the appropriate bladder and moves the lysed sample into the magnetic bead blister. Here, the liberated nucleic acids are captured by adsorption to silica-magnetic beads.

The instrument then uses a retractable magnet positioned adjacent to the blister to hold the beads against the inside of the blister while they are washed to remove proteins, cell debris, and other potential PCR inhibitors. The instrument moves the wash buffer from the fitment into the appropriate blisters by exerting pressure on pistons located above the fitment.

After the washes are completed, an elution buffer is moved from the fitment to the appropriate blister resulting in the nucleic acids being released from the beads. The instrument then moves the purified nucleic acid solution to the 1st stage PCR blister while the beads and other waste products are pushed back to the trapezoidal blister.

# 2. Reverse Transcription and 1st Stage Multiplex PCR

In the $1 ^ { \mathrm { s t } }$ stage PCR blister, liquid containing the purified nucleic acid rehydrates a freeze-dried reagent pellet containing all of the outer primers. A PCR master mix, containing all components needed for PCR and reverse transcription (RT), is moved from the fitment to an adjoining blister. A Peltier device is in contact with these two blisters and the instrument performs a “hot-start” PCR and RT by preheating the blisters containing the purified sample and the PCR master mix. Once the appropriate temperature is reached, the contents of the two blisters are mixed and the RT step and thermo-cycling is initiated.

Since the FilmArray RP2 includes RNA viruses, an RT step is needed to convert the viral RNA into cDNA that can be amplified by PCR. Both the RT and the first stage of the nested PCR reaction are performed using the same outer primers and master mix.

# 3. Dilution, $2 ^ { \mathrm { n d } }$ Stage PCR and DNA Melt Analysis

Following completion of the RT and $1 ^ { \mathrm { s t } }$ stage PCR steps, a second singleplex PCR is carried out. To accomplish this, the instrument dilutes the products of the 1st stage PCR with fresh PCR master mix containing a double stranded DNA binding dye (LC Green® Plus, BioFire Diagnostics, LLC). This solution is distributed over the $2 ^ { \mathrm { n d } }$ stage PCR array, where it rehydrates the dried primers in each well. The individual wells in the array contain primers for different assays (each assay is present in triplicate wells of the array) that target specific nucleic acid sequences from each of the pathogens or control templates. The primers in the PCR2 array are “nested” or internal to the specific PCR

products of the $1 ^ { \mathrm { s t } }$ stage multiplex reaction. A second Peltier device is responsible for driving the PCR2 reaction and for controlling temperature during DNA melting. The product of the $2 ^ { \mathrm { n d } }$ stage PCR is visualized with the fluorescent LC Green® Plus dye. At the conclusion of PCR2, the temperature of the array is gradually increased and the fluorescence in each well is monitored and analyzed to generate a melt curve. The instrument then transfers images and temperature measurements to the FilmArray software for analysis.

The PCR2 array also contains a control assay, called the PCR2 Control, which is comprised of a specific set of PCR2 assay primers along with the corresponding template pre-spotted into three specific wells of the array. Failure of the PCR2 Control invalidates the run and indicates a test failure that is specific to the PCR2 step of the testing process.

# 4. Data Analysis and Result Reporting

The temperature at which a specific PCR product melts (melting temperature or Tm) is consistent and predictable. The FilmArray software automatically evaluates the results from replicate wells of each assay for the detection of amplicons with a specific Tm, which denotes the presence of specific bacterial or viral targets. The FilmArray software uses the following steps to interpret the melt curve data generated from each FilmArray RP2 assay:

# a. Analysis of Melt Curves

First, the FilmArray RP2 Melt Detector performs a set of basic calculations on the melt data to determine if a PCR reaction occurred in each well. If the melt profile indicates that a PCR product is present, then the analysis software calculates one or two Tm values, depending on the number of melt curves present in the data, and the Tm values are compared against an expected melt range for the associated assay. If the software determines that the melt is positive and the melt curve falls inside the assay’s specific melt range, then the curve is called positive. If the software determines that the melt is negative or that it is not in the appropriate range, then the curve is called negative.

# b. Analysis of Replicates

Next, the analysis software evaluates the replicates for each assay (target and control) to determine if the assay is positive or negative. To be called positive, at least two of the three wells associated with an assay must have a positive melt curve and the Tm for the positive curves must be similar (i.e., within $1 ^ { \circ } \mathrm { C }$ ). Assays with replicates that do not meet these criteria are called negative.

# c. Analysis of Controls

Results for control assays are compared to their expected values and assigned a single pass or fail result for each control. Pouch-specific rules define how control failures affect interpretations. The default rule specifies that any control failure invalidates the entire run. For the FilmArray RP2, failure of the RNA Process Control or the PCR2 Control is interpreted as a control failure and all target assays (regardless of the assay result) are assigned a test result of invalid.

# d. Interpretation of Assay Results

Once the results for the individual assays are determined, the software applies interpretation rules to determine the final test results. For many organisms, the target is determined to be present or absent if a single associated assay is positive or negative, respectively. For these organisms, the final test result is either “Detected” (for positive results), “Not Detected” (for negative results) or “Invalid” (when either control fails or the run fails). The FilmArray RP2 also includes test results that rely on the results of multiple assays. See the Interpretation of Results section for more information on interpreting these test results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility Study :

A reproducibility study was conducted at three testing sites on a combination of FilmArray 2.0 and FilmArray Torch systems. The study incorporated a range of potential variation introduced by site (three testing sites), day (five different days), operator (at least two per site), system, instrument or Torch module (at least three per site/sample), and pouch lot (at least three).

A total of four contrived NPS samples containing known quantities of various RP2 analytes (Table 4 below) were prepared in simulated NPS in VTM sample matrix1. The contrived samples contained various combinations of 12 different FilmArray RP2 analytes2, each at three different concentrations, Negative, Low Positive $\left( 1 \times \mathrm { L o D } \right)$ , and Moderate Positive $( 3 \mathrm { { x L o D } ) }$ . Note that the negative data were acquired from samples not spiked with a particular analyte (i.e., negative data for the analytes in Sample#1 and $\# 2$ were obtained from Sample#3 and #4, and vice-versa).

Table 4: Reproducibility Test Panel for the FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain/Serotype</td><td rowspan=1 colspan=1>Source/ ID</td><td rowspan=1 colspan=1>Limit of Detection(LoD) Concentration</td><td rowspan=1 colspan=1>Sample #1Concentration(1xLoD)</td><td rowspan=1 colspan=1>Sample #2Concentration(3xLoD)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>0C43</td><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>5.6E+02 Copies/mL(3.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>5.6E+02 Copies/mL(3.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>1.7E+03 Copies/mL(9.0E+01TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>Zeptometrix0810015CF</td><td rowspan=1 colspan=1>3.0E+01 Copies/mL(5.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>3.0E+01 Copies/mL(5.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>9.0E+01 Copies/mL(1.5E+00TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C2</td><td rowspan=1 colspan=1>Species CSerotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>3.7E+01 Copies/mL(2.0E+00 TCID50/mL)</td><td rowspan=1 colspan=1>3.7E+01 Copies/mL(2.0E+00 TCID50/mL)</td><td rowspan=1 colspan=1>1.1E+02 Copies/mL(6.0E+00TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/PortChalmers/1/73</td><td rowspan=1 colspan=1>ATCC VR-810</td><td rowspan=1 colspan=1>2.1E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>2.1E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>6.3E+01 Copies/mL(3.0E-01 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>Zeptometrix0810012CFN</td><td rowspan=1 colspan=1>3.8E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>3.8E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>1.1E+02 Copies/mL(3.0E-01 TCID5/mL)</td></tr><tr><td rowspan=1 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>A747</td><td rowspan=1 colspan=1>Zeptometrix081461</td><td rowspan=1 colspan=1>5.8E+01 IS1001Copies/mL(4.1E+01 CFU/mL)</td><td rowspan=1 colspan=1>5.8E+01 IS1001Copies/mL(4.1E+01 CFU/mL)</td><td rowspan=1 colspan=1>1.7E+02 IS1001Copies/mL (1.2E+02CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain/Serotype</td><td rowspan=1 colspan=1>Source/ ID</td><td rowspan=1 colspan=1>Limit of Detection(LoD)</td><td rowspan=1 colspan=1>Sample #3(1xLoD)</td><td rowspan=1 colspan=1>Sample #4(3xLoD)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>6.6E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>6.6E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td rowspan=1 colspan=1>2.0E+02 Copies/mL(3.0E-01 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>Zeptometrix0810255CF a</td><td rowspan=1 colspan=1>3.4E+01 Copies/mL(5.0E+00 TCID50/mL)</td><td rowspan=1 colspan=1>3.4E+01 Copies/mL(5.0E+00 TCID50/mL)</td><td rowspan=1 colspan=1>1.0E+02 Copies/mL(1.5E+01TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>Type 4a</td><td rowspan=1 colspan=1>Zeptometrix0810060CF</td><td rowspan=1 colspan=1>1.6E+03 Copies/mL(5.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>1.6E+03 Copies/mL(5.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>4.8E+03 Copies/mL(1.5E+02TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Type 16, A1IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>1.2E+03 Copies/mL(1.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>1.2E+03 Copies/mL(1.0E+01 TCID50/mL)</td><td rowspan=1 colspan=1>3.6E+03 Copies/mL(3.0E+01TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>9.0E+00 Copies/mL(2.0E-02 TCID50/mL)</td><td rowspan=1 colspan=1>9.0E+00 Copies/mL(2.0E-02 TCID50/mL)</td><td rowspan=1 colspan=1>2.7E+01 Copies/mL(6.0E-02 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis b</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix0801459</td><td rowspan=1 colspan=1>1.0E+03 CFU/mL</td><td rowspan=1 colspan=1>1.0E+03 CFU/mL</td><td rowspan=1 colspan=1>3.0E+03 CFU/mL</td></tr></table>

a Formerly Zeptometrix 0810037CF. b For $B$ . pertussis, the FilmArray RP2 amplifies a single-copy target and commercially available $\mathrm { \ q P C R }$ assays typically target the multi-copy IS481 sequences, therefore RP2 testing was performed based on the $\mathrm { C F U / m L }$ and an equivalent Copies/mL was not determined for this analyte.

Once prepared, each sample of the reproducibility study panel was tested with the FilmArray RP2 to confirm that it contained the intended analytes at the intended concentration and then divided into single-use aliquots $( 4 0 0 ~ \mu \mathrm { L } )$ and stored frozen (≤ - $7 0 ^ { \circ } \mathrm { C }$ ) until the day of testing.

After being distributed to the sites, six replicates of each sample were tested on five different days on various FilmArray Torch modules (Site A, Site C) or FilmArray 2.0 instruments (Site B, Site C). Sites A and C conducted testing with at least three Torch modules per sample, while sites B and C utilized at least three different FilmArray 2.0 instruments per sample. Note that Site C tested a total of 12 replicates of each sample per day, with six replicates tested on the Torch system and six replicates tested on the FilmArray 2.0 system.

Daily testing was performed by at least two different operators per system and site, and three different pouch lots were used on rotating days so that data from all variables were distributed between reagent lots. For any required retest per the Instructions for Use, another aliquot of the same sample was tested on the same day by the same operator using the same system, instrument or Torch module, and pouch lot. Results of the valid retest were used as the final test result for the sample aliquot.

In total, 120 data points per sample (over a total of 480 valid runs) were obtained, with 60 data points per sample per system (i.e., FilmArray 2.0 and FilmArray Torch systems), 30 data points per sample per Site A and B, and 60 data points at Site C.

Over the course of the reproducibility study, a total of 15 different FilmArray 2.0 instruments and 19 different FilmArray Torch modules (four Torch bases) were used by seven different operators at three different sites. Valid results were obtained from 480 out of the 489 runs that were initiated (480/489, $9 8 . 2 \%$ ). The majority of invalid runs (8/9) were associated with a Control failure, while one invalid run was due to an instrument error (Table 5 below).

Table 5: Performance of the FilmArray Systems and RP2 Controls during the Reproducibility Study   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Runs</td><td rowspan=1 colspan=1>Control Failure(Percentage)</td><td rowspan=1 colspan=1>Instrument Errors(Percentage)</td><td rowspan=1 colspan=1>Software Errors(Percentage)</td></tr><tr><td rowspan=1 colspan=1>FilmArray 2.0</td><td rowspan=1 colspan=1>247</td><td rowspan=1 colspan=1>6(2.4%)</td><td rowspan=1 colspan=1>1(0.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>FilmArray Torch</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>2(0.8%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>8a(1.6%)</td><td rowspan=1 colspan=1>1(0.2%)</td><td rowspan=1 colspan=1>0(0.0%)</td></tr></table>

Seven control failures occurred on pouch lot 349116, while one occurred on pouch lot 347416.

A summary of the reproducibility study results, percent $( \% )$ agreement with the expected Detected or Not Detected result, for each analyte (by site and system) is provided in Table 6 below.

Table 6: Reproducibility of FilmArray RP2 Results on FilmArray Torch and FilmArray 2.0 Systems   

<table><tr><td colspan="1" rowspan="1">I abic 0. kcp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="7" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="3" rowspan="1">FilmArray Torch</td><td colspan="3" rowspan="1">FilmArray 2.0</td><td colspan="1" rowspan="2">AllSites/Systems(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)1.1E+02Copies/mL(6.0E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)3.7+01Copes/mL(2.0E+0TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/3010%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">119/12099.2%(95.4%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus229E</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1">CoronavirusHKU1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)1.7E+03Copies/mL(9.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">58/6096.7%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">117/12097.5%(92.9%-99.5%</td></tr><tr><td colspan="1" rowspan="2">Coronavirus0C43</td><td colspan="1" rowspan="1">Low Positive(1× LoD)5.6E+02Copies/mL(3.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">27/3090.0%</td><td colspan="1" rowspan="1">57/6095.0%</td><td colspan="1" rowspan="1">117/12097.5%(92.9%-99.5%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">CoronavirusNL63</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="7" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="3" rowspan="1">FilmArray Torch</td><td colspan="3" rowspan="1">FilmArray 2.0</td><td colspan="1" rowspan="2">AllSites/Systems(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td></tr><tr><td colspan="1" rowspan="3">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)3.6E+03Copies/mL(3.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)1.2E+03Copies/mL(1.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">28/3093.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">58/6096.7%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)1.1E+02Copies/mL.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">28/3093.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">58/6096.7%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)3.8+01Copies/mL(1.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/12000%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">Influenza AH3</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)6.3E+01Copies/mL(3.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">119/12099.2%(95.4%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)2.1E+01Copies/mL(1.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1-2009</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)1.0E+02Copies/mL(1.5E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)3.4E+01Copies/mL(5.0E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/4801000%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">ParainfluenzaVirus 2</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)9.0E+01Copies/mL(1.5E+00TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/3010%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)3.0E+01Copies/mL(5.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">27/3090.0%</td><td colspan="1" rowspan="1">57/6095.0%</td><td colspan="1" rowspan="1">116/12096.7%(91.7%-99.1%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="4" rowspan="1">Agreement with Expected Result</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">FilmArray Torch</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">FilmArray 2.0</td><td colspan="1" rowspan="2">AllSites/Systems(95% C)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td></tr><tr><td colspan="1" rowspan="3">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)4.8E+0Copies/mL(1.5E+02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)1.6E+03Copies/mL(5.0E+01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/6010%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">RespiratorySyncytialVirus</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)2.7E+1Copies/mL(6.0E-02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)9.0E+00Copies/mL(2.0E-02TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Bordetellaparapertussis</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)1.8E+02 IS1001Copies/mL(1.2E+02 CFU/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">119/12099.2%(95.4%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)6.0E+01 IS1001Copies/mL(4.1E+01 CFU/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">24/30 a80.0%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">53/60a88.3%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">112/12093.3%(87.3%-97.1%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="7" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">FilmArray Torch</td><td colspan="3" rowspan="1">FilmArray 2.0</td><td colspan="1" rowspan="2">AllSites/Systems(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemSub-Total</td></tr><tr><td colspan="1" rowspan="3">Bordetellapertussis</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)3.0E+03 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)1.0E+03 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">28/3093.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">58/6096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">118/12098.3%(94.1%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="3">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">Moderate Positive(3× LoD)2.0E+02Copies/mL(3.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%(97.0%-100%)</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1× LoD)6.6E+01Copies/mL(1.0E-01TCID50/mL)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">28/3093.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">58/6096.7%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">117/12097.5%(92.9%-99.5%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%(98.5%-100%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">240/240100%</td><td colspan="1" rowspan="1">480/480100%(99.2%-100%)</td></tr><tr><td colspan="3" rowspan="1">Overall Agreement with the Expected ResultAll Analytes and All Test Levels(95% Confidence Interval)</td><td colspan="7" rowspan="1">9,562/9,60099.6%(99.5% - 99.7%)</td></tr></table>

a Data from Site A were further reviewed by the unique site-specific variables including test day, Torch module, and operator. No correlation could be found between the Not Detected results and any one or more of these variables. The Not Detected results at Site A were found to be statistically non-significant $( \mathtt { p } { > } 0 . 0 5$ by Fisher’s exact test) and therefore do not appear to indicate a siteor system-dependent variance in precision of the FilmArray RP2 Bordetella parapertussis (IS1001) results.

The reproducibility (standard deviation) of melting temperature (Tm) for the amplification products generated by the FilmArray RP2 was also evaluated, with a Tm standard deviation for each assay $\mathrm { o } \mathrm { \bar { f } } \pm 0 . 5 ^ { \circ } \mathrm { C } ^ { 3 }$ or less observed within and between the FilmArray 2.0 and FilmArray Torch systems (Table 7 below).

Table 7: Reproducibility of Tm $( ^ { \circ } \mathbf { C } )$ For Select FilmArray RP2 Assays on FilmArray Torch and FilmArray 2.0 Systems   

<table><tr><td rowspan=4 colspan=1>Analyte</td><td rowspan=4 colspan=1>FilmArrayRP2 Assay</td><td rowspan=1 colspan=10>Tm Observed (C)a</td></tr><tr><td rowspan=1 colspan=4>FilmArray Torch</td><td rowspan=1 colspan=4>FilmArray 2.0</td><td rowspan=2 colspan=2>AllSites/Systems</td></tr><tr><td rowspan=1 colspan=2>Site A</td><td rowspan=1 colspan=2>Site C</td><td rowspan=1 colspan=2>Site B</td><td rowspan=1 colspan=2>Site C</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDev</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDev</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDev</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDev</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StDev</td></tr><tr><td rowspan=2 colspan=1>Controls</td><td rowspan=1 colspan=1>YeastRNA</td><td rowspan=1 colspan=1>82.3</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>82.2</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>± 0.3</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>76.1</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>75.9</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>76.0</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>75.8</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>75.9</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=10>VIRUSES</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adeno2</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>±0.1</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>Adeno6</td><td rowspan=1 colspan=1>89.6</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>89.2</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>±0.3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>CoV-OC43-2</td><td rowspan=1 colspan=1>80.7</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>± 0.1</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>80.5</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>± 0.2</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>78.2</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>± 0.3</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>84.3</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>84.2</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>84.3</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>84.1</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>84.2</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=3 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>FluA-pan1</td><td rowspan=1 colspan=1>84.2</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>±0.1</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>83.8</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>FluA-pan2</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>±0.1</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>FluA-H3</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>80.4</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>80.3</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>80.4</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>80.2</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>80.3</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>83.2</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>83.1</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>83.2</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>83.0</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>83.1</td><td rowspan=1 colspan=1>±0.2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>PIV4</td><td rowspan=1 colspan=1>77.1</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>77.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>77.2</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>77.0</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>77.1</td><td rowspan=1 colspan=1>±0.3</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVVirus</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>± 0.2</td></tr><tr><td></td><td></td><td></td><td></td><td rowspan=1 colspan=7>BACTERIA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>IS1001</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>± 0.2</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ptxP</td><td rowspan=1 colspan=1>88.6</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>88.2</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>88.4</td><td rowspan=1 colspan=1>±0.3</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>Cpne</td><td rowspan=1 colspan=1>79.6</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>± 0.3</td><td rowspan=1 colspan=1>79.3</td><td rowspan=1 colspan=1>± 0.2</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>± 0.3</td></tr></table>

a Mean Tm values are calculated from a combination of Tm values obtained at the $3 \times \mathrm { L o D }$ and $1 \times \mathrm { L o D }$ concentrations.

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

Two process controls are included in each pouch:

# RNA Process Control

The RNA Process Control assay targets an RNA transcript from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and becomes rehydrated when sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, reverse transcription, PCR1, dilution, PCR2, and DNA melting. A positive control result indicates that all steps carried out in the FilmArray RP2 pouch were successful.

# PCR2 Control

The PCR2 Control assay detects a DNA target that is dried into wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

Both control assays must be positive for the test run to pass. If the controls fail, the sample should be retested using a new pouch.

The FilmArray Software automatically fails the run if the melting temperature (Tm) for either the RNA Process Control or the PCR2 Control is outside of an acceptable range (i.e., $8 0 . 3 \substack { - 8 4 . 3 ^ { \circ } \mathrm { C } }$ for the RNA Process Control and $7 3 . 8 – 7 7 . 8 ^ { \circ } \mathrm { C }$ for the PCR2 Control). If required by local, state, or accrediting organization quality control requirements, users can monitor the system by trending Tm values for the control assays and maintaining records according to standard laboratory quality control practices.

# External Controls

External controls are not provided with the FilmArray RP2. However, five frozen (- $7 0 ^ { \circ } \mathrm { C } )$ external control mixes (ECMs) (see Table 8 below) were prepared and provided to the clinical study sites for testing during the prospective clinical trial and the clinical study testing contrived specimens. FilmArray operators were required to complete a valid ECM run (correct results obtained) on each day of clinical specimen testing (tested on a rotating basis).

Table 8: External Control Mixes (ECMs) Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>External ControlMixes</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>ECM 1</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus 229E, Human Metapneumovirus, Influenza A H1,Influenza B, and Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>ECM 2</td><td rowspan=1 colspan=1>Coronavirus NL63, Influenza A H3, Parainfluenza Virus 3, RespiratorySyncytial Virus and Bordetella pertussis (ptxP)</td></tr><tr><td rowspan=1 colspan=1>ECM 3</td><td rowspan=1 colspan=1>Coronavirus HKU1, Influenza A H1-2009, Parainfluenza Virus 4, andBordetella parapertussis (IS1001)</td></tr><tr><td rowspan=1 colspan=1>ECM 4</td><td rowspan=1 colspan=1>Coronavirus OC43, Human Rhinovirus/Enterovirus, Parainfluenza Virus 1,Parainfluenza Virus 2, and Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>ECM 5</td><td rowspan=1 colspan=1>Negative (Not Detected)</td></tr></table>

The sponsor is also recommending the following in the product package insert: “Good laboratory practice recommends running external positive and negative controls regularly. Transport media can be used as an external negative control. Previously characterized positive samples or negative samples spiked with well characterized organisms can be used as external positive controls. External controls should be used in accordance with the appropriate accrediting organization requirements, as applicable.”

# Specimen Stability

FilmArray RP2 testing requires approximately $0 . 3 ~ \mathrm { m L }$ of NPS in Viral Transport Medium (VTM) specimen, collected according to a standard technique. Samples in VTM should be tested as soon as possible, though they may be stored at room

temperature $( 1 5 . 2 5 ^ { \circ } \mathrm { C } )$ for up to four hours, under refrigeration $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ ) for up to three days, or frozen $( \le - 1 5 ^ { \circ } \mathrm { C }$ or $\le - 7 0 ^ { \circ } \mathrm { C }$ ) for up to 30 days.

Detailed documentation concerning NPS in VTM sample storage and transport was provided in the original FilmArray RP submissions (K103175, K110764, and K120267). The results of the analytical study conducted for the original FilmArray RP submissions supported the claim that NPS specimens in Viral Transport Medium (VTM) can be stored for up to 4 hours at room temperature $( 1 8 - 3 0 ^ { \circ } \mathrm { C } )$ , 3 days in the refrigerator $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ or 30 days in the freezer $( < - 1 5 ^ { \circ } \mathrm { C } )$ without affecting the accuracy of FilmArray RP test results. The specimen stability claims for the FilmArray RP2 remain unchanged.

# Fresh vs. Frozen Study

Detailed documentation concerning fresh vs. frozen NPS in VTM specimens was provided in the original FilmArray RP submissions (K103175, K110764, and K120267). The results of the analytical study conducted for the original FilmArray RP submissions demonstrated that preservation of samples (by freezing at $\le - 7 0 ^ { \circ } \mathrm { C }$ ) does not affect the accuracy of test results compared to freshly collected or freshly prepared samples. Therefore, it is appropriate to utilize frozen archived prospective and retrospective clinical samples in the evaluation of FilmArray RP2 to supplement the prospective (fresh) clinical study data, and to use frozen simulated samples in analytical studies for this submission.

# Single-Spiked vs. Multi-Spiked Specimen Study

Detailed documentation concerning analytical study results used to determine whether the presence of multiple organisms in a specimen would affect the system LoD was provided in the original FilmArray RP submissions (K103175, K110764, and K120267). The results of the analytical study demonstrated that the LoD for each target analyte from testing single-spiked and multi-spiked specimens were comparable. Therefore, it is appropriate to utilize the multi-spiked approach in conducting the analytical studies and the clinical study testing contrived clinical specimens in support of this submission.

# Simulated vs. Natural NPS in VTM Specimen Study

Detailed documentation concerning analytical study results used to determine whether the simulated NPS in VTM sample matrix and the natural NPS in VTM sample matrix were equivalent with regard to FilmArray RP test analytical sensitivity was provided in the original FilmArray RP submissions (K103175, K110764, and K120267). Result of the analytical study demonstrated that the simulated NPS in VTM sample matrix and the natural NPS in VTM sample matrix were equivalent with regard to FilmArray RP test ability to detect target analytes. Therefore, it is appropriate to utilize the simulated NPS in VTM sample matrix in conducting the analytical studies in support of this submission.

# Quantification of Nucleic Acids Derived from Viral and Bacterial Cultures

The majority of the FilmArray RP2 analytes require specific technique-dependent methods for culturing and quantification. Consistency in the quantification method of isolates and strains within and between species is important to mitigate inconsistencies in molecular detection that may be caused by variability of quantification methods based on infectivity or viability of microorganisms. Therefore, an analytical study was conducted to determine the nucleic acids concentration (copies/mL) of bacterial and virus stocks used in the analytical and clinical studies of the FilmArray RP2 using commercially available quantitative real-time PCR (qPCR) assay kits.

# d. Detection limit:

Limit of detection (LoD) estimation and confirmation studies were carried out with contrived samples in simulated NPS in VTM (Remel M4 transport medium) matrix designed to resemble a natural clinical NPS in VTM specimen. An equivalence study was performed previously which demonstrated that the simulated matrix was equivalent to the natural clinical NPS matrix and did not impact FilmArray RP test performance. Refer to the “Simulated vs. Natural NPS in VTM Specimen Study” section.

Representative isolates of respiratory viruses and bacteria were selected to make contrived samples in order to obtain positive results for every assay on the panel. In some cases, testing of more than one isolate/strain per analyte was performed to assess LoD for clinically important species or variants, specifically when more than one assay was needed to detect the expected diversity of an analyte (e.g. Adenovirus).

An estimated LoD concentration for each analyte was first determined by testing contrived samples serially diluted ten-fold to span at least four concentrations ( $1 0 \times$ to $0 . 0 1 \times )$ bracketing the anticipated LoD. Five replicates were tested at each dilution, with additional dilutions tested, if needed, to reach a concentration at which loss in detection could be observed. The estimated LoD concentration was subsequently confirmed by testing 20 replicates on the FilmArray 2.0 and 20 replicates on the FilmArray Torch, for a total of 40 replicates at $1 { \times } \mathrm { L o D }$ . An additional 20 replicates per system were tested at a 10-fold lower concentration, for a total of 40 replicates at $0 . 1 { \times } \mathrm { L o D }$ . The required criteria for confirmation of LoD was a detection rate of at least $9 5 \%$ at $1 \times \mathrm { L o D }$ $( \geq 1 9 / 2 0$ per system) and a detection rate of less than $9 5 \%$ below LoD $( \leq 1 9 / 2 0$ detected per system at $0 . 1 { \times } \mathrm { L o D } _ { \rho } ^ { \ }$ ).

A multi-spiked approach was employed in both the LoD estimation and the LoD confirmation studies where samples spiked with up to five analytes at various concentrations were tested in the LoD studies. An equivalence study was performed previously which demonstrated that the multi-spiked samples were equivalent to the single-spiked samples and employing the multi-spiking approach did not impact FilmArray RP test performance. Refer to the “Single-Spiked vs. Multi-Spiked

Specimen Study” section.

LoD confirmation criteria were met for each analyte on both the FilmArray 2.0 and FilmArray Torch systems at the same concentration. The confirmed LoD concentrations are shown in Table 9 below with detected results at the $1 \times$ (shaded) and $0 . 1 \times$ concentrations presented separately for each system and also combined. The LoD concentration for most analytes is reported in Copies $/ \mathrm { m L }$ , as determined primarily by commercially-available quantitative PCR and RT-PCR assays. Refer to the “Quantification of Nucleic Acids Derived from Viral and Bacterial Cultures” section. In addition, viable units for the specific cultures tested ( $50 \%$ tissue culture infectious dose $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) or colony forming unit (CFU/mL)) based on the values obtained from the respective Certificate of Analysis (CoA) are also presented.

Table 9: Limit of Detection Confirmation Testing Results for Each RP2 Analyte at $\mathbf { 1 } \times \mathbf { L o D }$ and ${ \bf 0 . 1 } \times { \bf L o D }$ on FilmArray 2.0, FilmArray Torch, and Combined   

<table><tr><td colspan="1" rowspan="1">RP2 Analyte</td><td colspan="1" rowspan="1">Isolate/Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">FilmArray2.0</td><td colspan="1" rowspan="1">FilmArrayTorch</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="24">Adenovirus</td><td colspan="1" rowspan="4">AdenovirusSpecies A, Serotype 18ATCC VR-19</td><td colspan="1" rowspan="2">7.6E+03 Copies/mL(5.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">7.6E+02 Copies/mL(5.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">32/40</td></tr><tr><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="4">AdenovirusSpecies B, Serotype 7AZeptometrix0810021CF</td><td colspan="1" rowspan="2">3.9E+01 Copies/mL(5.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">3.9E+00 Copies/mL(5.0E-03 TCID50/mL)</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">26/40</td></tr><tr><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="4">AdenovirusSpecies C, Serotype 2ATCC VR-846</td><td colspan="1" rowspan="2">3.7E+01 Copies/mL(2.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">3.7E+00 Copies/mL(2.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">26/40</td></tr><tr><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="4">AdenovirusSpecies D, Serotype 37Zeptometrix0810119CF</td><td colspan="1" rowspan="2">9.0E+00 Copies/mL(5.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">9.0E-01 Copies/mL(5.0E-03 TCID50/mL)</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">23/40</td></tr><tr><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">58%</td></tr><tr><td colspan="1" rowspan="4">AdenovirusSpecies E, Serotype 4aS. Carolina/2004, UIRF</td><td colspan="1" rowspan="2">3.0E+03 Copies/mL(1.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">38/40</td></tr><tr><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="2">3.0E+02 Copies/mL(1.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">7/40</td></tr><tr><td colspan="1" rowspan="1">5%</td><td colspan="1" rowspan="1">30%</td><td colspan="1" rowspan="1">18%</td></tr><tr><td colspan="1" rowspan="4">AdenovirusSpecies F, Serotype 41Tak, ATCC VR-930</td><td colspan="1" rowspan="2">1.2E+02 Copies/mL(1.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.2E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">19/40</td></tr><tr><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">35%</td><td colspan="1" rowspan="1">48%</td></tr><tr><td colspan="1" rowspan="4">Coronavirus 229E</td><td colspan="1" rowspan="4">Coronavirus 229EATCC VR-740</td><td colspan="1" rowspan="2">6.5E+01 Copies/mL(4.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">6.5E+00 Copies/mL(4.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">31/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">78%</td></tr><tr><td colspan="1" rowspan="4">CoronavirusHKU1</td><td colspan="1" rowspan="4">Coronavirus HKU1Clinical specimen a</td><td colspan="1" rowspan="2">2.0E+03 RNACopies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">2.0E+02 RNACopies/mL</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">24/40</td></tr><tr><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">60%</td></tr><tr><td colspan="1" rowspan="4">Coronavirus NL63</td><td colspan="1" rowspan="4">Coronavirus NL63BEI NR-470</td><td colspan="1" rowspan="2">5.4E+01 Copies/mL(2.5E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">5.4E+00 Copies/mL(2.5E-02 TCID50/mL)</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">22/40</td></tr><tr><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">55%</td></tr><tr><td colspan="1" rowspan="4">Coronavirus OC43</td><td colspan="1" rowspan="4">Coronavirus OC43ATCC VR-759</td><td colspan="1" rowspan="2">5.6E+02 Copies/mL(3.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">5.6E+01 Copies/mL(3.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">25/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">63%</td></tr><tr><td colspan="1" rowspan="4">HumanMetapneumovirus</td><td colspan="1" rowspan="4">HumanMetapneumovirus16, Type A1 IA10-2003Zeptometrix</td><td colspan="1" rowspan="2">1.2E+03 Copies/mL(1.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.2E+02 Copies/mL(1.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">33/40</td></tr><tr><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">83%</td></tr><tr><td colspan="1" rowspan="8">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="4">EnterovirusSpecies D, Serotype 68ATCC VR-1823</td><td colspan="1" rowspan="2">2.6E+01 Copies/mL(3.0E+02 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">2.6E+00 Copies/mL(3.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">23/40</td></tr><tr><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">58%</td></tr><tr><td colspan="1" rowspan="4">Human RhinovirusType 1AZeptometrix0810012CFN</td><td colspan="1" rowspan="2">3.8E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">3.8E+00 Copies/mL(1.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">24/40</td></tr><tr><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">65%</td><td colspan="1" rowspan="1">60%</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1</td><td colspan="1" rowspan="4">Influenza A H1N1A/New Caledonia/20/99Zeptometrix081036CF</td><td colspan="1" rowspan="2">1.4E+02 Copies/mL(1.0E+03 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.4E+01 Copies/mL(1.0E+02 TCID50/mL)</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">21/40</td></tr><tr><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">53%</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1-2009</td><td colspan="1" rowspan="4">Influenza AH1N1pdm09A/Swine/NY/03/2009Zeptometrix</td><td colspan="1" rowspan="2">3.3E+02 Copies/mL(5.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">3.3E+01 Copies/mL(5.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">23/40</td></tr><tr><td colspan="1" rowspan="1">40%</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">58%</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3</td><td colspan="1" rowspan="4">Influenza H3N2A/Port Chalmers/1/73ATCC VR-810</td><td colspan="1" rowspan="2">2.1E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">2.1E+00 Copies/mL(1.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">17/40</td></tr><tr><td colspan="1" rowspan="1">40%</td><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">43%</td></tr><tr><td colspan="1" rowspan="4">Influenza B</td><td colspan="1" rowspan="4">Influenza BB/FL/04/06Zeptometrix0810255CF</td><td colspan="1" rowspan="2">3.4E+01 Copies/mL(5.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">3.4E+00 Copies/mL(5.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">23/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">58%</td></tr><tr><td colspan="1" rowspan="4">ParainfluenzaVirus 1</td><td colspan="1" rowspan="4">Parainfluenza Virus 1Type 1Zeptometrix0810014CF</td><td colspan="1" rowspan="2">1.0E+03 Copies/mL(5.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.0E+02 Copies/mL(5.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">31/40</td></tr><tr><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">78%</td></tr><tr><td colspan="1" rowspan="4">ParainfluenzaVirus 2</td><td colspan="1" rowspan="4">Parainfluenza Virus 2Type 2Zeptometrix0810015CF</td><td colspan="1" rowspan="2">3.0E+01 Copies/mL(5.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">3.0E+00 Copies/mL(5.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">12/40</td></tr><tr><td colspan="1" rowspan="1">15%</td><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">30%</td></tr><tr><td colspan="1" rowspan="4">ParainfluenzaVirus 3</td><td colspan="1" rowspan="4">Parainfluenza Virus 3Type 3Zeptometrix0810016CF</td><td colspan="1" rowspan="2">3.8E+01 Copies/mL(2.5E+00 TCID50/mL)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">3.8E+00 Copies/mL(2.5E-01 TCID50/mL)</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">19/40</td></tr><tr><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">48%</td></tr><tr><td colspan="1" rowspan="4">ParainfluenzaVirus 4</td><td colspan="1" rowspan="4">Parainfluenza Virus 4Type 4aZeptometrix0810060CF</td><td colspan="1" rowspan="2">1.6E+03 Copies/mL(5.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.6E+02 Copies/mL(5.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">26/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="4">RespiratorySyncytial Virus</td><td colspan="1" rowspan="4">Respiratory SyncytialVirusType AZeptometrix0810040ACF</td><td colspan="1" rowspan="2">9.0E+00 Copies/mL(2.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">9.0E-01 Copies/mL(2.0E-03 TCID50/mL)</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">15/40</td></tr><tr><td colspan="1" rowspan="1">35%</td><td colspan="1" rowspan="1">40%</td><td colspan="1" rowspan="1">38%</td></tr><tr><td colspan="1" rowspan="4">Bordetellaparapertussis(IS1001)</td><td colspan="1" rowspan="4">B. parapertussisA747Zeptometrix 0801461</td><td colspan="1" rowspan="2">6.0E+01 IS1001Copies/mL b (4.1E+01CFII/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">6.0E+00 IS1001CoCoies/ml b(4.1E+00 CFU/mL)</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">16/40</td></tr><tr><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">35%</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="4">Bordetella pertussis(pxP)</td><td colspan="1" rowspan="4">B. pertussisA639Zeptometrix 0801459</td><td colspan="1" rowspan="2">1.0E+03 CFU/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">1.0E+02 CFU/mL</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">24/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">60%</td></tr><tr><td colspan="1" rowspan="4">Chlamydiapneumoniae</td><td colspan="1" rowspan="4">C. pneumoniaeTW183ATCC VR-2282</td><td colspan="1" rowspan="2">6.6E+01 Copies/mL(1.0E-01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">39/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">98%</td></tr><tr><td colspan="1" rowspan="2">6.6E+00 Copies/mL(1.0E-02 TCID50/mL)</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">21/40</td></tr><tr><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">53%</td></tr><tr><td colspan="1" rowspan="4">Mycoplasmapneumoniae</td><td colspan="1" rowspan="4">M. pneumoniaeM129Zeptometrix 0801579</td><td colspan="1" rowspan="2">4.6E+02 Copies/mLc(1.0E+01 TCID50/mL)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">4.6E+01 Copies/mLc(1.0E+00 TCID50/mL)</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">24/40</td></tr><tr><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">60%</td></tr></table>

a A cultured isolate of Coronavirus HKU1 was not available for testing. LoD for Coronavirus HKU1 was therefore determined by testing dilutions of a clinical NPS specimen known to contain the virus. The amount of viral RNA in the specimen (in RNA copies $/ \mathrm { m L }$ ) was determined by real-time RT-PCR against a standard curve. b IS1001 sequences can be present in more than one copy per cell, so the relationship between $\mathrm { C F U / m L }$ and copies/mL may vary from strain to strain and culture to culture. LoD was determined based on the copy number of IS1001 measured by an independent quantitative real-time PCR assay and for this culture. c The copy number of Mycoplasma pneumoniae was determined by a multicopy assay (16S rRNA) and may not accurately reflect the number of copies detected by the single-copy FilmArray RP2 gene target.

# e. Analytical Reactivity:

The analytical reactivity/inclusivity of the FilmArray RP2 for each reported analyte was initially assessed by in silico analysis and subsequently by testing unique strains or isolates. Isolate/strain selection emphasized variants of known human clinical

significance, including relevant species, serotypes, genotypes, isolates collected at various times and from different locations, recently circulating or newly emerging variants, or any strains of particular interest based on sequence alignments, as available. For more genetically diverse analytes a larger collection of isolates were selected in order to better demonstrate and characterize the analytical reactivity of the assay(s) with the expected diversity of the analyte.

Each isolate/strain was prepared and tested as a contrived sample in Remel M4 Transport Medium at a concentration near the LoD $\left( 3 { \times } \mathrm { L o D } \right)$ in triplicate, with one replicate on each of three different reagent lots. The majority of isolates were tested based on molecular quantification in units of copies $/ \mathrm { m L }$ , though a few were tested in viable units of $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C F U / m L }$ . Any isolate that was not reliably detected in $3 / 3$ or $4 / 5$ replicates at the $3 { \times } \mathrm { L o D }$ level, was tested again at $1 0 { \times } \mathrm { L o D }$ or a higher concentration, as needed, until detection was observed or a limitation on assay analytical reactivity with that isolate was identified.

Analytical reactivity wet testing results are summarized in Table 10 to Table 21 below.

Table 10: Adenovirus Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Species a</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC VR-863</td><td rowspan=1 colspan=1>[Huie/Massachusetts]</td><td rowspan=1 colspan=1>3x</td><td rowspan=21 colspan=1>AdenovirusDetected</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC VR-19</td><td rowspan=1 colspan=1>[Washington DC/1954]</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Zeptometrix 0810073CF</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=11 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810062CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7A</td><td rowspan=1 colspan=1>Zeptometrix 0810021CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7d/d2</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[Iowa/2001]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7h</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[Iowa/1999]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[Wisconsin/2005]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[Missouri/2005]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-17</td><td rowspan=1 colspan=1>[CH.79/Saudi Arabia/1955]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[Missouri/2005]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-716</td><td rowspan=1 colspan=1>[Compton/1972]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>ATCC VR-718</td><td rowspan=1 colspan=1>[Holden]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>ATCC VR-1602</td><td rowspan=1 colspan=1>[Wan/Amsterdam/1988]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810050CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>[Adenoid 6]</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810020CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC VR-6</td><td rowspan=1 colspan=1>[Tonsil 99/Washington DC]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810069CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810115CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Zeptometrix 0810119CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td></tr></table>

<table><tr><td rowspan=2 colspan=1>E</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>Univ of Iowa ResearchFoundation</td><td rowspan=1 colspan=1>[S Carolina/2004]</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810070CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>F</td><td rowspan=2 colspan=1>40</td><td rowspan=1 colspan=1>Zeptometrix 0810084CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>NCPV 0101141v</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>41</td><td rowspan=1 colspan=1>ATCC VR-930</td><td rowspan=1 colspan=1>[Tak/73-3544/Netherlands/1973]</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810085CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr></table>

a In silico analysis of available sequences predicts that the FilmArray RP2 will also react with Adenovirus B55, C57, species D serotypes, and G52.

Table 11: Coronavirus Isolates/Specimens Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>CoronavirusType</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=2 colspan=1>Coronavirus 229EDetected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810229CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>HKU1</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>[Utah/2015]</td><td rowspan=1 colspan=1>1x</td><td rowspan=5 colspan=1>Coronavirus HKU1Detected</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>[Utah/2015]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>[Utah/2015]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>[S. Carolina/2010]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>[Detroit/2010]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI NR-470 a</td><td rowspan=1 colspan=1>[Amsterdam/2003]</td><td rowspan=1 colspan=1>1x</td><td rowspan=2 colspan=1>Coronavirus NL63Detected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>OC43</td><td rowspan=1 colspan=1>ATCC VR-759 b</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=2 colspan=1>Coronavirus OC43Detected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810024CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Organism obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID, NIH: Human Coronavirus NL63, NR-470. b Discontinued part number; see ATCC VR-1558.

Table 12: Human Metapneumovirus Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=2>[Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td><td rowspan=1 colspan=2>[Iowa10/2003]</td><td rowspan=1 colspan=1>1x</td><td rowspan=13 colspan=1>HumanMetapneumovirusDetected</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Zeptometrix 0810160CF</td><td rowspan=1 colspan=2>[Iowa3/2002]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810163CF</td><td rowspan=1 colspan=2>[Iowa14/2003]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Zeptometrix 0810164CF</td><td rowspan=1 colspan=2>[Iowa27/2004]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810156CF</td><td rowspan=1 colspan=2>[Peru2/2002]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810158CF</td><td rowspan=1 colspan=2>[Peru3/2003]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Univ. of Iowa Research Foundation</td><td rowspan=1 colspan=2>[Iowa7/2003]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=6 colspan=1>B2</td><td rowspan=3 colspan=1>4</td><td rowspan=3 colspan=1>Zeptometrix 0810157CF</td><td rowspan=3 colspan=2>[Peru1/2002]</td><td rowspan=3 colspan=1>3x</td></tr><tr></tr><tr><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810159CF</td><td rowspan=1 colspan=2>[Peru6/2003]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Zeptometrix 0810162CF</td><td rowspan=1 colspan=2>[Iowa18/2003]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Univ. of Iowa Research Foundation</td><td rowspan=1 colspan=1>[Iowa16/2003]</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 13: Human Rhinovirus and Enterovirus Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Species </td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=3>Human Rhinovirus</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=8 colspan=1>A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix0810012CFN</td><td rowspan=1 colspan=1>[1A]</td><td rowspan=1 colspan=1>1x</td><td rowspan=15 colspan=1>HumanRhinovirus/EnterovirusDetected</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-482</td><td rowspan=1 colspan=1>[HGP]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>[68-CV11]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-283</td><td rowspan=1 colspan=1>[11757/WashingtonDC/1960]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-507 b</td><td rowspan=1 colspan=1>[137-3]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>[Ch47]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>R</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>[130-63]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>[50-525-CV54]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=7 colspan=1>B</td><td rowspan=2 colspan=1>3</td><td rowspan=2 colspan=1>ATCC VR-483</td><td rowspan=2 colspan=1>[FEB]</td><td rowspan=2 colspan=1>3x</td></tr><tr></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-284</td><td rowspan=1 colspan=1>[1059/S Carolina/1959]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC VR-1663</td><td rowspan=1 colspan=1>[33342/N Carolina/1959]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ATCC VR-1137</td><td rowspan=1 colspan=1>[5870]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>ATCC VR-338</td><td rowspan=1 colspan=1>[56822]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>ATCC VR-1193</td><td rowspan=1 colspan=1>[Baylor 7]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Enterovirus</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Coxsackievirus 10</td><td rowspan=1 colspan=1>ATCC VR-168</td><td rowspan=1 colspan=1>[NY/1950]</td><td rowspan=1 colspan=1>3x</td><td rowspan=11 colspan=1>HumanRhinovirus/EnterovirusDetected</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>ATCC VR-1432</td><td rowspan=1 colspan=1>[H]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix 0810017CF</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Zeptometrix 0810074CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>Zeptometrix 0810075CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix 0810076CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix 0810077CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix 0810023CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>ATCC VR-850</td><td rowspan=1 colspan=1>[Kuykendall/California/1952]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC VR-583</td><td rowspan=1 colspan=1>[DN-19/Texas/1963]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>[US/MO/2014-18947]</td><td rowspan=1 colspan=1>1x</td></tr></table>

a In silico analysis of available sequences predicts that the FilmArray RP2 will react with all currently characterized species and serotypes of Human Rhinovirus and Enterovirus, including Polioviruses. b Discontinued part number; see ATCC VR-1365.

Table 14: Influenza A Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=2>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=13 colspan=1>H1N1</td><td rowspan=9 colspan=1>Human</td><td rowspan=1 colspan=1>Zeptometrix 0810036CF</td><td rowspan=1 colspan=1>[New Caledonia/20/1999]</td><td rowspan=1 colspan=1>1x</td><td rowspan=13 colspan=1>Influenza AH1 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-219</td><td rowspan=1 colspan=1>[NWS/1933]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-95</td><td rowspan=1 colspan=1>[PR/8/1934]</td><td rowspan=1 colspan=1>10x a</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-96</td><td rowspan=1 colspan=1>[Weiss/1943]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-97</td><td rowspan=1 colspan=1>[FM/1/1947]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-98</td><td rowspan=1 colspan=1>[Mal/302/1954]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-546</td><td rowspan=1 colspan=1>[Denver/1/1957]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810036CFN</td><td rowspan=1 colspan=1>[Solomon Isl/03/2006]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810244CF</td><td rowspan=1 colspan=1>[Brisbane/59/2007]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Swine</td><td rowspan=2 colspan=1>ATCC VR-333</td><td rowspan=2 colspan=1>[A/Swine/Iowa/15/1930]</td><td rowspan=2 colspan=1>3x</td></tr><tr></tr><tr><td rowspan=1 colspan=1>ATCC VR-99</td><td rowspan=1 colspan=1>[A/Swine/1976/1931]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-897</td><td rowspan=1 colspan=1>[A/New Jersey/8/76 (Hsw1N1)]</td><td rowspan=1 colspan=1>10x a</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=2>Isolate ID/Source</td><td rowspan=1 colspan=2>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>Recombinant</td><td rowspan=1 colspan=1>BEI NR-9677 b</td><td rowspan=1 colspan=2>[Kilbourne F63, A/NWS/1934 (HA) xA/Rockefeller Institute/5/1957 (NA)]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=9 colspan=1>H1N1pdm09</td><td rowspan=9 colspan=1>Human</td><td rowspan=1 colspan=1>Zeptometrix 0810249CFN</td><td rowspan=1 colspan=1>[SwineNY/03/2009]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>Influenza AH1-2009Detected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810248CFN</td><td rowspan=1 colspan=1>[SwineNY/01/2009]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810109CFN</td><td rowspan=1 colspan=2>[SwineNY/02/2009]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810109CFJ</td><td rowspan=1 colspan=1>[Canada/6294/2009]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810165CF</td><td rowspan=1 colspan=2>[California/07/2009]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810166CF</td><td rowspan=1 colspan=2>[Mexico/4108/2009]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-19823</td><td rowspan=1 colspan=2>[Netherlands/2629/2009]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44345 d</td><td rowspan=1 colspan=2>[Hong Kong/H090-761-V1(0)/2009]</td><td rowspan=1 colspan=1>10x e</td></tr><tr><td rowspan=1 colspan=1>BEI NR-42938</td><td rowspan=1 colspan=2>[Georgia/F32551/2012]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=9 colspan=1>H3N2</td><td rowspan=8 colspan=1>Human</td><td rowspan=1 colspan=1>ATCC VR-810</td><td rowspan=1 colspan=2>[Port Chalmers/1/1973]</td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>Influenza AH3 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-776</td><td rowspan=1 colspan=2>[Alice (live attenuated vaccine)]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810238CF</td><td rowspan=1 colspan=2>[Texas/50/2012]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-547</td><td rowspan=1 colspan=2>[Aichi/2/1968]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-544</td><td rowspan=1 colspan=2>[Hong Kong/8/1968]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-822</td><td rowspan=1 colspan=2>[Victoria/3/1975]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td><td rowspan=1 colspan=2>[Wisconsin/67/2005]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810138CF</td><td rowspan=1 colspan=2>[Brisbane/10/2007]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Recombinant</td><td rowspan=1 colspan=1>ATCC VR-777</td><td rowspan=1 colspan=2>[MCR2(A/England/42/72xA/PR8/34)]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=2>[Ohio/2012]</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Influenza AH3 Detected</td></tr><tr><td rowspan=2 colspan=1>H2N2</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>BEI NR-2775 </td><td rowspan=1 colspan=2>[Japan/305/1957]</td><td rowspan=1 colspan=1>10x e</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Recombinant</td><td rowspan=1 colspan=1>BEI NR-9679 h</td><td rowspan=1 colspan=2>[Korea/426/1968xPuerto Rico/8/1934 ]</td><td rowspan=1 colspan=1>10x </td><td rowspan=1 colspan=1>(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>H2N3</td><td rowspan=9 colspan=1>Avian</td><td rowspan=1 colspan=1>MRI Globali</td><td rowspan=1 colspan=2>Mallard/Alberta/79/2003</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Influenza AEquivocal</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>MRI Global</td><td rowspan=1 colspan=2>A/Chicken/Yunnan/1251/2003</td><td rowspan=1 colspan=1>3x</td><td rowspan=7 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>MRI Global i</td><td rowspan=1 colspan=2>Northern pintail/Washington/40964/2014</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>BEI NR-9682 </td><td rowspan=1 colspan=2>A/Duck/Singapore/645/1997</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N8</td><td rowspan=1 colspan=1>MRI Global</td><td rowspan=1 colspan=2>Gyrfalcon/Washington/41088-6/2014</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=2 colspan=1>H7N7</td><td rowspan=2 colspan=1>MRI Global</td><td rowspan=2 colspan=2>A/Netherlands/219/2003</td><td rowspan=2 colspan=1>3x</td></tr><tr></tr><tr><td rowspan=1 colspan=1>H7N9</td><td rowspan=1 colspan=1>MRI Global</td><td rowspan=1 colspan=2>A/Anhui/01/2013</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>BEI NR-2765 k</td><td rowspan=1 colspan=2>Chicken/Germany/N/49</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Influenza AEquivocal</td></tr></table>

a Reported as Influenza A (no subtype detected) at $3 \times \mathrm { L o D }$ . b Genomic RNA obtained through the NIH Biodefense and Emerging Infections Research Resources Respiratory NAID, NIH Kilbourne F63: A/NWS/1934 (HA) x A/Rockefeller Institute/5/1957 (NA) (H1N2), Reassortant NWS-F, NR9677. c Organism obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Netherlands/2629/2009 (H1N1)pdm09, NR-19823. d Organism obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Hong Kong/H090-761-V1(0)/2009 (H1N1)pdm09, NR-44345. e Reported as Influenza A Equivocal or Influenza A (no subtype detected) at $3 \times \mathrm { L o D }$ . f Organism obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Georgia/F32551/2012 (H1N1)pdm09, NR-42938. g Genomic RNA obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID, NIH: Genomic RNA from Influenza A Virus, A/Japan/305/1957 (H2N2), NR-2775. h Genomic RNA obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID, NIH: Genomic RNA from Kilbourne F38: A/Korea/426/1968 (HA, NA) x A/Puerto Rico/8/1934 (H2N2), NR-9679. i Isolate provided and tested by MRI Global, Kansas City, MO.

j Genomic RNA obtained through the NIH Biodefense and Emerging Infections Research Resources Repository NIAID, NIH: Genomic RNA from Kilbourne F181: A/duck/Singapore/645/1997 (H5N3), Wild Type, NR-9682. k Genomic RNA obtained through the NIH Biodefense and Emerging Infections Research Resources Repository NIAID, NIH: Genomic RNA from Influenza A Virus, A/chicken/Germany/N/1949 (H10N7), NR-2765

Table 15: Influenza B Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Lineage</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC VR-101</td><td rowspan=1 colspan=1>[Lee/1940]</td><td rowspan=1 colspan=1>3x</td><td rowspan=13 colspan=1>Influenza BDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-102</td><td rowspan=1 colspan=1>[Allen/1945]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-103</td><td rowspan=1 colspan=1>[GL/1739/1954]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-296</td><td rowspan=1 colspan=1>[1/Maryland/1959]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-295</td><td rowspan=1 colspan=1>[2/Taiwan/1962]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-786</td><td rowspan=1 colspan=1>[Brigit/Russia/1969]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>Victoria</td><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>[5/Hong Kong/1972]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810258CF</td><td rowspan=1 colspan=1>[2506/Malaysia/2004]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC 2005743348</td><td rowspan=1 colspan=1>[1/Ohio/2005]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Yamagata</td><td rowspan=1 colspan=1>Zeptometrix 0810256CF</td><td rowspan=1 colspan=1>[07/Flordia/2004]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810255CF</td><td rowspan=1 colspan=1>[04/Flordia/2006]</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810241CF</td><td rowspan=1 colspan=1>[1/Wisconsin/2010]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810239CF</td><td rowspan=1 colspan=1>[2/Massachusetts/2012]</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 16: Parainfluenza Virus Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=2>1</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>ParainfluenzaVirus 1 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>[C-35/Washington DC/1957]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3226 a</td><td rowspan=1 colspan=1>[C39]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-48680 b</td><td rowspan=1 colspan=1>[FRA/29221106/2009]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=2 colspan=1>ParainfluenzaVirus 2 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>[Greer/Ohio/1955]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=2>3</td><td rowspan=1 colspan=1>Zeptometrix 0810016CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=3 colspan=1>ParainfluenzaVirus 3 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>[C-243/Washington DC/1957]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>[NIH 47885, Wash/47885/57]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>4</td><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>ParainfluenzaVirus 4 Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1378</td><td rowspan=1 colspan=1>[M-25/1958]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>B</td><td rowspan=1 colspan=1>Zeptometrix 0810060BCF</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>[CH-19503/WashingtonDC/1962]</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Discontinued part number. b Obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 1, HPIV1/FRA/29221106/2009, NR-48680. c Obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 3, NIH 47885, NR-3233.

Table 17: Respiratory Syncytial Virus Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF</td><td rowspan=1 colspan=1>[2006]</td><td rowspan=1 colspan=1>1x</td><td rowspan=7 colspan=1>RespiratorySyncytial VirusDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>[Long/Maryland/1956]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>[A2/Melbourne/1961]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B</td><td rowspan=1 colspan=1>Zeptometrix 0810040CF</td><td rowspan=1 colspan=1>[Ch-93 (18)-18]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>[WV/14617/1985]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>[9320/Massachusetts/1977]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>[18537/Washington DC/1962]</td><td rowspan=1 colspan=1>10x</td></tr></table>

Table 18: Bordetella parapertussis (and Bordetella bronchiseptica) Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Zeptometrix 0801461</td><td rowspan=1 colspan=1>[A747]</td><td rowspan=1 colspan=1>1x</td><td rowspan=6 colspan=1>Bordetellaparapertussis(IS1001)Detected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801462</td><td rowspan=1 colspan=1>[E595]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15237</td><td rowspan=1 colspan=1>[NCTC 10853]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15311</td><td rowspan=1 colspan=1>[NCTC 5952]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-587</td><td rowspan=1 colspan=1>[12822/Germany/1993]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica(containing IS1001) a</td><td rowspan=1 colspan=1>NRRL B-59909</td><td rowspan=1 colspan=1>[MBORD849/Pig/Netherlands]</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Reactivity with IS1001 sequences in $B$ . bronchiseptica represents the intended reactivity of the assay, however the analyte will be inaccurately reported as $B _ { \cdot }$ . parapertussis. The assay does not react with IS1001-like sequences in $B$ . holmesii (see Analytical Reactivity).

Table 19: Bordetella pertussis Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain]</td><td rowspan=1 colspan=1>xLoD Detected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801459</td><td rowspan=1 colspan=1>[A639]</td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>Bordetellapertussis (ptxP)Detected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801460</td><td rowspan=1 colspan=1>[E431]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 8467</td><td rowspan=1 colspan=1>[F]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 9340</td><td rowspan=1 colspan=1>[5,17921]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>[18323/NCTC 10739]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 10380</td><td rowspan=1 colspan=1>[10-536]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 51445</td><td rowspan=1 colspan=1>[CNCTC Hp 12/63,623]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-589</td><td rowspan=1 colspan=1>[Tohama]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1335</td><td rowspan=1 colspan=1>[MN2531]</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 20: Chlamydia pneumoniae Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>xLoD Detected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>[TW-183/Taiwan/1965]</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>ChlamydiapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1310</td><td rowspan=1 colspan=1>[CWL-029]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1360</td><td rowspan=1 colspan=1>[CM-1/Georgia]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>[AR-39/Seattle/1983]</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 21: Mycoplasma pneumoniae Isolates Tested and Detected by FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>[Strain]</td><td rowspan=1 colspan=1>xLoD Detected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0801579</td><td rowspan=1 colspan=1>[M129]</td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>MycoplasmapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 29342</td><td rowspan=1 colspan=1>[M129-B7]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 29085</td><td rowspan=1 colspan=1>[PI 1428]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 15531</td><td rowspan=1 colspan=1>[FH strain of Eaton Agent [NCTC 10119]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15492</td><td rowspan=1 colspan=1>[Mac]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>unknown</td><td rowspan=1 colspan=1>ATCC15293</td><td rowspan=1 colspan=1>[M52]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC15377</td><td rowspan=1 colspan=1>[Bru]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC39505</td><td rowspan=1 colspan=1>[Mutant 22]</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 49894</td><td rowspan=1 colspan=1>[UTMB-10P]</td><td rowspan=1 colspan=1>3x</td></tr></table>

All isolates wet tested (177/177 $( 1 0 0 \% )$ in the analytical reactivity study were amplified by the appropriate assay(s) and detected by the FilmArray RP2 at concentrations within $1 0 \times$ LoD.

Analytical reactivity was demonstrated with a wide variety of isolates/strains, including over 25 Adenovirus isolates of different species and serotypes, 25 different Human Rhinovirus or Enterovirus species and serotypes, and several isolates of the major Influenza A subtypes that typically infect humans. Due to the public health concerns related to zoonotic transmission of influenza A viruses to humans (primarily swine and avian lineages), several isolates of swine and avian influenza A viruses were also tested at a concentration equal to $3 \times$ the lowest Influenza A LoD.

The expected outcomes of Influenza A H1, Influenza A Equivocal, and Influenza A (no subtype detected) results were reported for influenza A viruses of swine and avian origin, depending on the strain, RNA sequence, and concentration tested. As expected, the majority of zoonotic and recombinant viruses of non-H1, H1-2009, or H3 hemagglutinin subtypes were reported as Influenza A (no subtype detected). However, one (avian H2N3) virus was reported as Influenza A Equivocal at lower concentrations of $3 \times$ and $1 0 \times \mathrm { L o D }$ and Influenza A (no subtype detected) at a higher concentration of $1 0 0 \times$ LoD. Another avian isolate (H10N7) was reported as Influenza A Equivocal at all concentrations tested, up to $1 0 0 \times$ LoD.

$f .$ Analytical Specificity/Cross-reactivity Evaluation:

Analytical specificity (cross-reactivity) of the FilmArray RP2 was evaluated by challenging the system with contrived samples containing a high concentration of the potentially cross-reacting organism. On-panel organisms were tested to assess the potential for intra-panel cross-reactivity (e.g., does the Coronavirus OC43 assay cross-react with Coronavirus HKU1, etc.), while off-panel organisms (those not intended to be detected by the panel) were tested to assess the potential for non-specific amplification of respiratory flora or other respiratory pathogens that may be present in an NPS specimen. Organisms for off-panel testing were selected based on a combination of several factors including 1) relatedness to specific species detected by the FilmArray RP2 (i.e., near-neighbors), 2) clinical relevance (cause respiratory symptoms) or likelihood of being present in NPS specimens, and 3) genetic similarity to FilmArray RP2 assay primers, as determined by in silico analyses.

Each organism was tested in Remel M4 transport medium on three different lots of FilmArray RP2 pouches and at the highest concentration possible based on the stock concentration of the cultured isolate. The final concentrations of analytes in the samples were typically ${ \geq } 1 . 0 \mathrm { E } { + } 0 6 $ CFU/ml for bacteria and fungi and $\ge 1 . 0 \mathrm { E } + 0 5 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses (calculated to be between ${ \sim } 7 . 0 \mathrm { E } { + } 0 1 -$ $4 . 0 \mathrm { E } { + } 0 5$ fold higher than the LoD of the FilmArray RP2 assays).

# On-Panel Organisms Testing

A group of 22 on-panel analytes were tested at high concentration to assess the potential for intra-panel cross-reactivity. The organisms tested are shown in Table 22 below and results, including cross-reactivity observed or predicted for three of the on-panel organisms are discussed below.

Table 22: On-panel Organisms Tested by the FilmArray RP2 for Evaluation of Analytical Specificity (Organisms with the potential for non-specific amplification by another FilmArray RP2 assay are shown in bold type)   

<table><tr><td rowspan=1 colspan=3>Viruses</td><td rowspan=1 colspan=1>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Bordetella parapertussis&quot;c,d</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229 E</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>dBordetella pertussis</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1a</td><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Influenza A H1N1pdm09</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterovirus (Echovirus 6)</td><td rowspan=1 colspan=1>Influenza A Hsw1N1</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr></table>

a Two different clinical specimens were tested, containing up to $8 . 9 \mathrm { E } { + 0 8 }$ RNA copies/mL of Coronavirus HKU1. b Swine origin Hsw1N1 (A/New Jersey/8/1976 ; ATCC VR-897) was variably detected as either Influenza A H1 or Influenza A H1-2009 at a concentration of $8 . 9 \mathrm { E } { + 0 6 }$ CEID50/mL; indicating the potential for cross-reactivity with the FluA H1-2009 assay at higher concentrations. c Contains pertussis toxin pseudogene sequences that are predicted to be amplified by the $p t x P$ assay and reported as Bordetella pertussis $( p t x P )$ Detected when present at very high concentrations (i.e., ${ > } 1 . 2 \mathrm { E } { + } 0 9$ CFU/mL). d Non-specific amplification by the HRV/EV assay, generating a Human Rhinovirus/Enterovirus Detected result, was observed when testing $B$ . pertussis at high concentration (i.e., ${ \ge } 4 . 5 \mathrm { E } { + } 0 7$ CFU/mL). Similar cross-reactivity is predicte to be possible with $B .$ . parapertussis, but was not observed when tested up to $1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL.

Testing of two different clinical NPS specimens known to contain high concentrations of Coronavirus HKU1 (i.e., up to $8 . 9 \mathrm { E } { + 0 8 }$ Copies/mL or $> 4 . 0 \mathrm { E } \substack { + 0 . 5 \times \mathrm { L o D } }$ ) did not generate inaccurate Coronavirus OC43 Detected results. These data support the conclusion that the intra-panel cross-reactivity observed between the Coronavirus OC43 assay contained in the FilmArray RP and high concentrations of Coronavirus HKU1 $_ { ( > 1 . 0 \mathrm { E } + 0 8 }$ Copies/mL) has been eliminated by the redesigned Coronavirus OC43 assay (CoV-OC43- 2) contained in the FilmArray RP2.

Based on sequence analysis and experience with the same assay in the FilmArray RP, cross-reactivity was expected between the assay for the detection of Bordetella pertussis $( p t x P )$ and non-pertussis Bordetella strains that carry a pertussis toxin pseudogene (e.g. $B .$ . parapertussis and the offpanel species B. bronchiseptica) when present at concentrations ${ > } 1 . 0 \mathrm { E } { + } 0 6 $ CFU/mL. However, reaction conditions for the FilmArray RP2 were determined to be less permissive for this non-specific interaction and the predicted cross-reactivity was not observed when testing on-panel $B$ . parapertussis at a much higher concentration (i.e., ${ \geq } 1 . 2 \mathrm { E } { + } 0 9$ CFU/mL). (Note: also see B. bronchiseptica testing in off-panel testing section below).

Though not observed, the potential for cross-reactivity with $B$ . parapertussis will be described as possible, but in the FilmArray RP2, the true organism ( $B$ . parapertussis) will also be identified and reported.

The FilmArray RP2 includes assays to distinguish classical human Influenza A H1 and the pandemic H1-2009 variant derived from swine. However, due to sequence similarity, some reactivity of the H1-2009 assay may be observed with historical and/or novel H1N1 strains of swine origin. This was demonstrated by testing a swine origin influenza A Hsw1N1 isolate from the 1970s (A/New Jersey/8/1976) which was detected and reported as Influenza A H1 in one replicate and as Influenza A H1-2009 in two replicates when tested at a high concentration of $8 . 9 \mathrm { E + 0 6 } \mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ .

Finally, the FilmArray RP2 assay for detection of Human Rhinovirus/Enterovirus (HRV/EV) contains a primer pair that, despite several mismatches, is able to amplify sequences of the oxidoreductase gene found in Bordetella pertussis as well as selected published $B$ . parapertussis and $B$ . bronchiseptica sequences. The predicted non-specific amplification was confirmed when testing $B$ . pertussis at a concentration of $4 . 5 \mathrm { E } { + } 0 7$ CFU/mL and higher. The cross-reactivity was not observed when testing $B$ . parapertussis (or the off-panel organism $B .$ . bronchiseptica, see below) at a concentration of $1 . 2 \mathrm { E } { + } 0 9 $ $\mathrm { C F U / m L }$ , but it may still be possible. Bordetella infections are less common than the “common cold’ caused by Rhinoviruses (and Enteroviruses). Therefore not only should this crossreactivity be rarely observed, but the presumably more significant Bordetella pertussis infection should be appropriately identified for treatment, with possible extra care for a concurrent Rhinovirus infection associated with the non-specific result.

# Off-Panel Organisms Testing

In silico analysis of assay specificity was supplemented with wet testing of 50 off-panel bacteria, viruses, and fungi at high concentrations (typically ${ \geq } 1 . 0 \mathrm { E } { + } 0 6 $ CFU/ml for bacteria and fungi and ${ \ge } 1 . 0 \mathrm { E } { + } 0 5$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses). The organisms tested are shown in Table 23 below and results are discussed below.

Table 23: Off-panel Organisms Tested by the FilmArray RP2 for Evaluation of Analytical Specificity (Organisms with the potential for non-specific amplification by a FilmArray RP2 assay are shown in bold type)   

<table><tr><td colspan="3" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Bocavirus</td></tr><tr><td colspan="1" rowspan="1">Bordetella avium</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td></tr><tr><td colspan="1" rowspan="1">Bordetella hinzii</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Measles Virus</td></tr><tr><td colspan="1" rowspan="1">Legionella bozemanii</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Mumps</td></tr><tr><td>Legionella dumofii</td><td>Lactobacillus plantarum</td><td>Staphylococcus epidermidis</td><td>Middle East Respiratory Syndrome Coronavirus</td></tr><tr><td>Legionella feeleii</td><td>Moraxella catarrhalis</td><td>Stenotrophomonas maltophilia</td><td></td></tr><tr><td>Legionella longbeacheae</td><td>Mycoplasma genitalium</td><td>Streptococcus pneumoniae</td><td>Fungi/Yeast</td></tr><tr><td>Legionella micdadei</td><td>Mycoplasma hominis</td><td>Streptococcus agalactiae</td><td>Candida albicans</td></tr><tr><td>Legionella pneumophila</td><td>Mycoplasma orale</td><td>Streptococcus pyogenes</td><td>Cryptococcus neoformans</td></tr><tr><td>Chlamydia trachomatis</td><td>Mycobacterium tuberculosis</td><td>Streptococcus salivarus</td><td>Aspergillus fumigatus</td></tr><tr><td>Corynebacterium diphteriae</td><td>Neisseria elongata</td><td>Ureaplasma urealyticum</td><td>Aspergillus flavus</td></tr></table>

Non-specific amplification of pertussis toxin pseudogene sequences by the $\overline { { p t x P } }$ assay is possible at very high concentrations b $( \dot { \geq } 1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL), generating a Bordetella pertussis $( p t x P )$ Detected result. Non-specific amplification by the HRV/EV assay, generating a Human Rhinovirus/Enterovirus Detected result, is predicted at high concentrations but was not observed when tested up to c $1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL. Strains of Bordetella bronchiseptica that carry $\mathrm { I S } I O O I$ insertion sequences will be amplified by the IS1001 assay as intended, but misidentified by the FilmArray RP2 as Bordetella parapertussis (IS1001). Heat-inactivated viral culture obtained through BEI Resources, NIAID, NIH: Middle East Respiratory Syndrome Coronavirus (MERS-CoV), EMC/2012, Heat-Inactivated, NR-50171.

The only non-specific interaction observed in testing of the off-panel organisms was a Bordetella pertussis $( p t x P )$ Detected result in one of three replicates of B. bronchiseptica tested at a concentration of $1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL. This cross-reactivity has been observed at lower concentrations in the FilmArray RP and was predicted based on the presence of pertussis toxin pseudogene sequences in this species. Though not observed, Human Rhinovirus/Enterovirus assay cross-reactivity with some sequences of $B$ . bronchiseptica is predicted by in silico analyses. In addition, certain strains of $B$ . bronchiseptica are known to carry the IS1001 insertion sequences that are most common to $B$ . parapertussis. In these cases, the FilmArray RP2 assay for detection of Bordetella parapertussis (IS1001) will amplify the correct targeted sequence, but will misidentify the organism as Bordetella parapertussis rather than $B .$ . bronchiseptica. As B. bronchiseptica is a very rare human pathogen (more commonly associated with veterinary cases of canine “kennel cough”), the probability of inaccurate Bordetella pertussis $( p t x P )$ results or inaccurate Bordetella parapertussis (IS1001) results due to cross-reactivity with B. bronchiseptica is low.

# Analytical Specificity Evaluation Conclusion

Analytical specificity testing has demonstrated that the majority of the FilmArray RP2 assays are highly specific for the organisms they are designed to detect. Cross-reactivity that was identified will occur almost exclusively at high organism levels (i.e., ${ > } 1 . 0 \mathrm { E } { + } 0 7$ units $/ \mathrm { m L }$ ) and most events are associated with near-neighbor species that carry the same genes or highly similar sequences as the targeted organism. Overall, the likelihood, risk, and impact of the FilmArray RP2 non-specific interactions are predicted to be minor. All identified FilmArray RP2 cross-reactivity cases are indicated in the product Instructions for Use, as a precaution to minimize misinterpretation of results. A summary is provided in Table 24 below.

Table 24: Predicted and Observed Cross-Reactivity of the FilmArray RP2   

<table><tr><td rowspan=1 colspan=1>Cross-reactive Organism</td><td rowspan=1 colspan=1>FilmArray RP2 Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Non-pertussis Bordetellaspecies (e.g., Bordetellaparapertussis or B.bronchiseptica ²)</td><td rowspan=1 colspan=1>Bordetella pertussis (</td><td rowspan=1 colspan=1>The Bordetella pertussis (ptxP) assay canamplify pertussis toxin pseudogenesequences in B. bronchiseptica and B.parapertussis. Cross-reactivity is observedat high concentration (≥1.2E+09CFU/mL).</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica (with IS1001 sequences)</td><td rowspan=1 colspan=1>Bordetella parapertussis(IS1001)</td><td rowspan=1 colspan=1>Some strains of B. bronchiseptica (rarelyisolated from humans) do carry IS1001insertion sequences identical to thosecarried by most strains of B.parapertussis. These sequences will beamplified by the IS1001 assay andreported by FilmArray RP2 as Bordetellaparapertussis (IS1001).</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis and otherBordetella species</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirusd,e</td><td rowspan=1 colspan=1>The Human Rhinovirus/Enterovirus assaymay amplify off-target sequences found instrains of B. pertussis, B. bronchisepticaand B. parapertussis. Cross-reactivitywith B. pertussis was observed at aconcentration of 4.5E+07 CFU/mL orhigher.</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1(swine origin)</td><td rowspan=1 colspan=1>Influenza A H1-2009 </td><td rowspan=1 colspan=1>The Influenza A H1-2009 assay mayreact with H1 hemagglutinin genesequences from viruses of swine origin.RP2 results will be either Influenza A H1or Influenza A H1-2009, depending onthe strain and concentration in thesample.</td></tr></table>

a B. bronchiseptica infection is rare in humans and more common in domesticated animals (‘kennel cough’). b Cross-reactivity was observed only when tested at a high concentration $( \geq 1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL). c Cross-reactivity between the Bordetella pertussis (ptxP) assay and $B$ . parapertussis will be reported as a co-detection (Bordetella parapertussis (IS1001) Detected and Bordetella pertussis (ptxP) Detected); while cross-reactivity with most strains of $B$ . bronchiseptica (that do not carry IS1001) will be reported only as Bordetella pertussis (ptxP) Detected. d Cross-reactivity with $B$ . pertussis was observed when tested at a concentration of $4 . 5 \mathrm { E } { + } 0 7$ CFU/mL and higher. Cross-reactivity with $B$ . parapertussis and B. bronchiseptica is predicted based on in silico analysis, but was not observed when tested at a concentration of $1 . 2 \mathrm { E } { + } 0 9 $ CFU/mL. e Cross-reactivity between the Human Rhinovirus/Enterovirus assays and $B _ { \cdot }$ . pertussis or $B$ . parapertussis will be reported as a co-detection (Bordetella pertussis (ptxP) Detected and Human Rhinovirus/Enterovirus Detected or Bordetella parapertussis (IS1001) Detected and Human Rhinovirus/Enterovirus Detected); while cross-reactivity with most strains of $B$ . bronchiseptica (that do not carry IS1001) will be reported (falsely) only as Human Rhinovirus/Enterovirus Detected. f Swine origin Hsw1N1 (A/New Jersey/8/1976 ; ATCC VR-897) was detected as either Influenza A H1 or Influenza A H1-2009 at a concentration of $8 . 9 \mathrm { E } { + 0 6 }$ CEID50/mL.

# g. Assay cut-off:

The FilmArray RP2 is part of BioFire Diagnostics’ FilmArray system. The FilmArray system is designed to interpret the test data and automatically report the test results to the operator. The FilmArray system uses the results of the Melt Detector to determine each test result. The Melt Detector is part of the FilmArray Analysis Software and assigns a positive or negative result to each reaction on the array through analysis of the melt data collected during the test. These positive and negative results are combined in the FilmArray Analysis Software (using the replicate, assay and interpretation rules) to report the presence or absence of each pathogen in the panel.

To determine the result for each well, the Melt Detector was developed to analyze the melt data. The Melt Detector analyzes the melt data for each well independently, and utilizes curve-specific characteristics (e.g., shape, signal-to-noise, etc.) to assign a positive or negative call to each curve. To maximize the sensitivity and specificity of the Melt Detector, the algorithm was tuned against a large data set comprising typical and atypical melting curves (i.e., training data set) with expert annotation (positive or negative calls) during the development of the original FilmArray RP.

To evaluate the Melt Detector performance for the FilmArray RP2, the observed sensitivity and specificity rates for the individual melt curves and assay calls were reported and analyzed. These sensitivity and specificity rates were determined by comparing the FilmArray RP2 test results obtained from testing well-characterized samples collected as part of the clinical evaluation and analytic testing of the FilmArray RP2 panel, to expert annotation.

For individual melt curves, the observed sensitivity and specificity, as compared to expert annotation, of the Melt Detector is $9 9 . 6 5 \%$ and $9 9 . 9 4 \%$ , respectively. For the Analysis Software, the observed sensitivity and specificity of the assay calls, as compared to expert annotation, are greater than $9 9 . 8 0 \%$ for sensitivity and $9 9 . 9 6 \%$ specificity. These rates are comparable to the observed rates reported in the previous melt detector validation studies.

# h. Interfering Substances:

An analytical study was performed to assess the potential inhibitory effects of exogenous and endogenous substances and competitive microorganisms that may be commonly found in nasopharyngeal specimens (NPS). This study also assessed the potential inhibitory effects of disinfecting/cleaning substances and specimen collection materials.

For this study, potentially interfering substances were selected for evaluation based upon whether the substance may normally be found in NPS specimens or may be introduced into specimens during NPS collection or subsequent handling and testing. This included endogenous substances that may be found in specimens at normal or elevated levels, such as blood, mucus/mucin, human genomic DNA, and various commensal or infectious microorganisms (both on-panel and off-panel). Exogenous substances that may be present in specimens, such as medications, treatments, or topical applications for soothing symptoms associated with respiratory infections were also included. Lastly, substances such as disinfectants (e.g., bleach and ethanol) and various swabs and transport media that could contact specimens during collection or testing were evaluated as potential technique- specific interfering substances.

Each tested substance was added to a contrived sample at a concentration similar to or greater than the level expected to be found in a clinical NPS specimen. The contrived sample contained a mix of five different FilmArray RP2 analytes, each present at a concentration near the limit of detection (LoD), i.e., 2-3xLoD. Contrived sample with no substance added served as a positive control (no interference) on each day of testing, and a potentially interfering substance in negative sample matrix served as a negative or substance-only control. Samples containing test substances were evaluated for effects of the substance on the internal pouch control assays as well as effects on the ability of the FilmArray RP2 to provide accurate organism test results compared to the positive control samples.

In total, 39 different endogenous and exogenous substances, potentially competing microorganisms, specimen collection materials (swabs and media), and disinfecting agents were evaluated in this study (see Table 25 below).

Table 25: Results for Potentially Interfering Substances Tested on the FilmArray RP2   

<table><tr><td colspan="1" rowspan="1">Substance Tested</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="3" rowspan="1">Endogenous Substances</td></tr><tr><td colspan="1" rowspan="1">Human Whole Blood</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human Mucin (Sputum)</td><td colspan="1" rowspan="1">1 swab/mL sample</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">2.0E+01 ng/μL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Competitive Mcroorgarisms</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.7E+04 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Adenovirus A12</td><td colspan="1" rowspan="1">8.9E+05 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">6.6E+05 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">5.8E+08 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">1.6E+07 TCID50/mLaa</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.0E+07 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">4.2E+04 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2.5E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.7E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">6.2E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td><td colspan="1" rowspan="1">1.6E+06 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">1.2E+06 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Exogenous Substances b</td></tr><tr><td colspan="1" rowspan="1">Tobramycin (systemic antibiotic)</td><td colspan="1" rowspan="1">0.6E-01 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mupirocin(active ingredient in anti-bacterial ointment)</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray with Preservatives(0.65% NaCl, Phenylcarbinol, Benzalkonium</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Nasal Decongestant Spray(Oxymetazoline HCl 0.05%, Benzalkonium</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Analgesic ointment (Vicks®VapoRub®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Petroleum Jelly (Vaseline®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Snuff (Tobacco)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Disinfecting/Cleaning Substances</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">1% and 2% v/v[512, 1024 ppm chlorine]</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">Disinfecting wipes (ammonium chloride d)</td><td colspan="1" rowspan="1">1/ in2</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">7% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">DNAZap (AmbionTM AM9891G &amp; AM9892G)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">RNaseZap (AmbionTM AM9782)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Specimen Collection Miterials</td></tr><tr><td colspan="1" rowspan="1">Rayon Swabs (Copan 168C)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Nylon Flocked Swabs (Copan 553C)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Polyester Swabs (Copan 175KS01)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Calcium Alginate Swabs (Puritan 25-801 A 50)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">M4® Transport Medium(Remel R12500, 3mL/tube)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">M4-RT® Transport Medium(Remel R12506, 3 mL/tube)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">M5® Transport Medium(Remel R12516, 3 mL/tube)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">M6™ Transport Medium(Remel R12535, 1.5 mL/tube)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Universal Viral Transport vial(BD 220220, 3 mL/tube)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sigma-Virocult" Viral Collection and TransportSystem  Swabs</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Copan ESwab™" Sample Collection and DeliverySystem  Swabs and</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr></table>

Liquid Amies Medium (Copan 482C)a A different lot of this isolate was also tested at a lower concentration of $3 . 2 \mathrm { E } { + } 0 4 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ with no interference observed. b Nasal influenza vaccines (e.g. FluMist) were not evaluated, but are predicted to be reactive with the FilmArray RP2 Influenza A (subtype) and Influenza B assays. c Not Detected results were reported for several analytes after incubation of the sample with $2 \%$ bleach for 10 minutes or overnight. It was concluded that interference resulted primarily from damage to the organisms/nucleic acids in the sample, rather than inhibition or interference with pouch function(s). ${ \bf { \check { n } } } _ { \bf { \check { n } } }$ -Alkyl (C14, $60 \%$ , C16, $30 \%$ , C12, $5 \%$ , C18, $5 \%$ ) Dimethyl Benzyl Ammonium Chloride - $0 . 1 8 4 \%$ nAlkyl (C12, $6 8 \%$ , C14, $32 \%$ ) Dimethyl Ethylbenzyl Ammonium Chloride - $0 . 1 8 4 \%$

Testing showed that none of the substances evaluated had an effect on the FilmArray RP2 control assays and no interference with pouch function was identified. However, it was demonstrated that exposure of samples to bleach prior to testing could damage the organisms/nucleic acids in the sample and generate inaccurate FilmArray RP2 test results (i.e., lack of analyte detection), depending on the concentration and/or length of time the bleach was allowed to interact with the sample.

# Interfering Substances Evaluation Conclusion

No interference was observed when testing samples containing potentially inhibitory biological substances (e.g., blood etc.) or high levels of potentially competing microorganisms. Similarly, detection near LoD was robust in samples prepared in a variety of transport media types or when exposed to various swabs that may be used for NPS collection. Saline, decongestants, ointments or other substances that could be introduced into the sample also had no effect on the function of the FilmArray RP2.

The only limitation is related to potential damage to the organisms in the sample caused by bleach that could lead to false negative results. A warning describing the potential for sample damage caused by bleach will be included in the product Instructions for Use.

Note: Nasal influenza vaccines (e.g., FluMist) were not evaluated in this study, but are predicted to be reactive with the FilmArray RP2 Influenza A (including subtype) and Influenza B assays. Therefore, contamination of specimens with vaccine or recent administration of the vaccine prior to NPS specimen collection could lead to accurate detection by the FilmArray RP2 of the viruses contained in the vaccine, but would not represent infection by those agents.

# i. Carry-Over Contamination:

A formal carry-over study in support of this regulatory submission for the FilmArray RP2 was not performed, since carry-over studies with high positive samples followed by negative samples have been performed for other FDA-cleared FilmArray Panels (i.e., FilmArray RP, BCID, and GI) for both the FilmArray 2.0 and the FilmArray Torch systems. No carry-over has been observed.

j. Comparator Assay for B. parapertussis Analytical Validation

In the prospective clinical study for the FilmArray RP2, the FilmArray RP2 results were compared to the results from standard of care testing using the same FDA-cleared multiplexed respiratory pathogens panel performed at the clinical sites for all the FilmArray RP2 analytes except for Bordetella parapertussis.

Two well-validated PCR assays for Bordetella parapertussis (IS1001) followed by bidirectional sequencing (designed to give at least 200 base pairs of sequence information) were performed as the comparator method for Bordetella parapertussis (IS1001). The two comparator PCR assays target the same gene as the FilmArray RP2 assay (i.e., the IS1001 genetic element), but the primers identify sequences that do not overlap with the FilmArray RP2 assays. The two PCR comparator assays used in the clinical studies were designed by BioFire as published assays could not be found that provided adequate amplicon length or quality suitable for sequencing. Additionally, the assays were nested in order to match the sensitivity of the FilmArray RP2 assay.

Validation testing demonstrated that the PCR followed by sequencing assays had similar analytical reactivity performance to the FilmArray RP2 assay, and a LoD that was within 5-fold of the FilmArray RP2 assay LoD (this was considered "equivalent analytical sensitivity").

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# Prospective Clinical Study

The clinical performance of the FilmArray RP2 was established during a multi-center study conducted at three geographically distinct U.S. study sites during portions of the 2015-2016 and 2016-2017 respiratory illness seasons. Each study location was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

Residual NPS specimens in VTM meeting the following eligibility criteria were prospectively collected and tested using the FilmArray RP2 during the clinical study:

Inclusion criteria:

Specimen is residual NPS in VTM left over from standard of care (SOC) testing under clinician order for respiratory pathogen analysis using the FilmArray RP, an FDA-cleared multiplexed respiratory pathogens panel   
· Specimen has been held at room temperature for less than or equal to four hours or at $4 \mathrm { { } ^ { \circ } C }$ for less than or equal to three days before enrollment (FilmArray RP2 testing or archiving of specimen aliquots at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ must be completed within this window)   
· At least $1 . 5 \mathrm { m L }$ of specimen is remaining after standard of care testing and available for use in the study

Exclusion criteria:

Specimen other than NPS in VTM (e.g., nasopharyngeal aspirate, anterior or mid-turbinate swab, oropharyngeal swab, NPS collected in medium other than VTM) Specimen was not tested with the comparator test as part of patient care Specimen cannot be tested within the defined storage parameters

Insufficient specimen volume for testing

A waiver of the informed consent requirement was obtained from the Institutional Review Boards (IRBs) at each study site for the use of residual NPS specimens. Each residual specimen collected for the study was assigned a unique Study Code Number (SCN). The SCN was used to de-identify the specimen aliquots used for FilmArray RP2 testing and comparator PCR/sequencing testing, and to provide select clinical data to the sponsor. The SCN was recorded in a key which linked each SCN to subject identification information to allow collection of subject demographic information, hospitalization status, and the comparator test result(s) provided for subject care. Access to this key was limited to specific site personnel who de-identified study specimens and who had no knowledge of the FilmArray RP2 results.

The following information was recorded on the Case Report Form (CRF) for each subject from whom a specimen was enrolled:

· Age group and sex Date of specimen collection Subject hospitalization status (Outpatient, Emergency Department, or Hospitalized) SOC comparator test result(s)

A total of 1635 residual NPS specimens in VTM were prospectively acquired initially for the clinical study. Between January and March 2016, specimens were prospectively collected from all comers meeting the study eligibility criteria and immediately frozen $\scriptstyle ( \mathrm { N } = 6 9 5$ specimens) for later testing as prospective archived/frozen (Category II) specimens. Between September and November 2016, specimens were prospectively collected from all comers meeting the study eligibility criteria and tested fresh $\scriptstyle ( \mathrm { N } = 9 4 0$ specimens) as prospective fresh (Category I) specimens. Category II specimens were distributed to sites beginning in September 2016. Study sites also began testing Category I specimens at this time. At each site, Category II specimens were thawed and tested according to the study procedures as time permitted over the remaining duration of the clinical study.

Table 26: Participating Study Sites for the FilmArray RP2 Clinical Evaluation   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Study SiteLocation</td><td rowspan=1 colspan=1>EnrolledPopulation</td><td rowspan=1 colspan=1>Category I SpecimenCollected(September November 2016)</td><td rowspan=1 colspan=1>Category II SpecimenCollected(January - March2016)</td><td rowspan=1 colspan=1>Total Number ofProspectiveSpecimens(Category I and II)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Salt Lake City,UT</td><td rowspan=1 colspan=1>Predominantlypediatric</td><td rowspan=1 colspan=1>350</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>600</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Chicago, IL</td><td rowspan=1 colspan=1>Pediatric andadult</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>244</td><td rowspan=1 colspan=1>530</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Columbus, OH</td><td rowspan=1 colspan=1>Predominantlypediatric</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>505</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>940</td><td rowspan=1 colspan=1>695</td><td rowspan=1 colspan=1>1635</td></tr></table>

Of the total of 1635 prospective NPS specimens in VTM that were initially acquired for the clinical study, 23 were excluded from the final performance data analysis (see Table 27 below). The most common reason for specimen exclusion was one instance of a site failing to run an ECM for a given day of testing (10 specimens excluded). The second most common reason for specimen exclusion was the testing of specimens outside the 3-day refrigerated storage window (nine specimens). The third most common reason for exclusion was that the specimen was found to not meet the inclusion criteria after the specimen had been enrolled (three specimens).

Table 27: Summary of Specimens Excluded from the Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Reason for Exclusion</td><td rowspan=1 colspan=1>Specimens</td></tr><tr><td rowspan=1 colspan=1>Did not meet inclusion criteria</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Specimen of incorrect sample type</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Comparator test not requested by clinician for standardof care testing</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>FilmArray RP2 test could not be completed within 3-day window</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>No external control run for test date</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>FilmArray RP2 run failure  unable to be retested dueto volume limitation</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23/1635 (1.4%)</td></tr></table>

The final data set of the prospective clinical study consisted of 1612 specimens. Table 28 below provides a summary of demographic information for the 1612 specimens included in the data analysis of the prospective clinical study.

Table 28: Demographic Summary for Prospective FilmArray RP2 Clinical Evaluation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=2 colspan=1>S</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>867 (54%)</td><td rowspan=1 colspan=1>331 (57%)</td><td rowspan=1 colspan=1>271 (51%)</td><td rowspan=1 colspan=1>265 (53%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>745 (46%)</td><td rowspan=1 colspan=1>250 (43%)</td><td rowspan=1 colspan=1>256 (49%)</td><td rowspan=1 colspan=1>239 (47%)</td></tr><tr><td rowspan=4 colspan=1>A</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>885 (55%)</td><td rowspan=1 colspan=1>379 (65%)</td><td rowspan=1 colspan=1>170 (32%)</td><td rowspan=1 colspan=1>336 (67%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>331 (21%)</td><td rowspan=1 colspan=1>132 (23%)</td><td rowspan=1 colspan=1>89 (17%)</td><td rowspan=1 colspan=1>110 (22%)</td></tr><tr><td rowspan=1 colspan=1>22 - 49 years</td><td rowspan=1 colspan=1>128 (8%)</td><td rowspan=1 colspan=1>27 (5%)</td><td rowspan=1 colspan=1>79 (15%)</td><td rowspan=1 colspan=1>22 (4%)</td></tr><tr><td rowspan=1 colspan=1>50+ years</td><td rowspan=1 colspan=1>268 (17%)</td><td rowspan=1 colspan=1>43 (7%)</td><td rowspan=1 colspan=1>189 (36%)</td><td rowspan=1 colspan=1>36 (7%)</td></tr><tr><td rowspan=3 colspan=1>Shs</td><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>329 (20%)</td><td rowspan=1 colspan=1>77 (13%)</td><td rowspan=1 colspan=1>66 (13%)</td><td rowspan=1 colspan=1>186 (37%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>640 (40%)</td><td rowspan=1 colspan=1>229 (39%)</td><td rowspan=1 colspan=1>197 (37%)</td><td rowspan=1 colspan=1>214 (42%)</td></tr><tr><td rowspan=1 colspan=1>Emergency</td><td rowspan=1 colspan=1>643 (40%)</td><td rowspan=1 colspan=1>275 (47%)</td><td rowspan=1 colspan=1>264 (50%)</td><td rowspan=1 colspan=1>104 (21%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>1612</td><td rowspan=1 colspan=1>581</td><td rowspan=1 colspan=1>527</td><td rowspan=1 colspan=1>504</td></tr></table>

The performance of the FilmArray RP2 was evaluated by comparing the FilmArray RP2 test results with those from an FDA-cleared multiplexed respiratory pathogen panel as well as with results from two analytically-validated PCR followed by bidirectional sequencing assays for $B$ . parapertussis (this analyte is not detected by the FDA-cleared multiplexed respiratory pathogen panel). The $B$ . parapertussis comparator assays were designed to amplify a different sequence than that amplified by the FilmArray RP2. Any specimen that had bi-directional sequencing data meeting pre

defined quality acceptance criteria that matched organism-specific sequences deposited in the NCBI GenBank database (www.ncbi.nlm.nih.gov) with acceptable E-values was considered Positive. Any specimen that tested negative by both of the comparator assays was considered Negative.

# Prospective Clinical Study System Performance

The overall success rate for initial specimen tests in the prospective study was $9 9 . 3 \%$ (1611/1623) ( $9 5 \%$ CI: $9 8 . 7 \% - 9 9 . 6 \%$ ; 12 tests were unsuccessful (one due to an incomplete test, one due to an instrument error, and ten due to control failures). Two tests $( 2 / 1 6 2 3 ; 0 . 1 \% )$ did not complete on the initial run, resulting in an instrument success rate of $9 9 . 9 \%$ (1621/1623) ( $9 5 \%$ CI: $9 9 . 6 \% - 1 0 0 \%$ for initial specimen tests. Both specimens were retested and valid results were produced after a single retest. Ten tests $( 1 0 / 1 6 2 1 ; 0 . 6 \% )$ did not produce valid pouch controls, resulting in a pouch control success rate of $9 9 . 4 \%$ (1611/1621) $9 5 \%$ CI: $9 8 . 9 \% - 9 9 . 7 \% )$ for completed runs in the initial specimen tests. Nine of the 10 invalid specimens were retested and produced valid control results after a single retest; one could not be retested due to insufficient specimen volume.

# Prospective Clinical Study Performance

Positive Percent Agreement (PPA) for each analyte was calculated as $100 \%$ x $\left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the FilmArray RP2 and the comparator method had a positive result for this specific analyte, and false negative (FN) indicates that the FilmArray RP2 result was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $100 \% \mathrm { ~ x ~ }$ (TN / (TN $+ \left. { F P } \right)$ ). True negative (TN) indicates that both the FilmArray RP2 and the comparator method had negative results, and a false positive (FP) indicates that the FilmArray RP2 result was positive but the comparator result was negative. The exact binomial twosided $9 5 \%$ confidence interval was calculated.

Samples for which false positive and/or false negative results (i.e., discrepant results) were obtained when comparing the FilmArray RP2 results to the comparator method results were further investigated. The discrepancy investigation was mainly conducted by performing independent molecular tests with primers that are different from that of the FilmArray RP2 and/or comparator method retesting.

The FilmArray RP2 prospective performance data in positive percent and negative percent agreements against the comparator methods (all sites combined) are presented by analyte in the following Table 29:

Table 29: FilmArray RP2 Prospective Clinical Performance Summary   

<table><tr><td colspan="2" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP +FN</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN +FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="3">Adenovirus a</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">36/38</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">82.7-98.5</td><td colspan="1" rowspan="1">850/880</td><td colspan="1" rowspan="1">96.6</td><td colspan="1" rowspan="1">95.2-97.6</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">34/36</td><td colspan="1" rowspan="1">94.4</td><td colspan="1" rowspan="1">81.9-98.5</td><td colspan="1" rowspan="1">640/658</td><td colspan="1" rowspan="1">97.3</td><td colspan="1" rowspan="1">95.7-98.3</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">70/74</td><td colspan="1" rowspan="1">94.6</td><td colspan="1" rowspan="1">86.9-97.9</td><td colspan="1" rowspan="1">1490/1538</td><td colspan="1" rowspan="1">96.9</td><td colspan="1" rowspan="1">95.9-97.6</td></tr><tr><td colspan="1" rowspan="3">CoV-229E b</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">909/913</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.9-99.8</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">48.7-97.4</td><td colspan="1" rowspan="1">686/687</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5</td><td colspan="1" rowspan="1">1595/1600</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.9</td></tr><tr><td colspan="1" rowspan="3">CoV-HKU1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">917/917</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100</td><td colspan="1" rowspan="1">640/652</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">96.8-98.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.8-100</td><td colspan="1" rowspan="1">1557/1569</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.7-99.6</td></tr><tr><td colspan="1" rowspan="3">CoV-NL63 d</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">917/918</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100</td><td colspan="1" rowspan="1">645/654</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">97.4-99.3</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100</td><td colspan="1" rowspan="1">1562/1572</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="3">CoV-OC43</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">11/13</td><td colspan="1" rowspan="1">84.6</td><td colspan="1" rowspan="1">57.8-95.7</td><td colspan="1" rowspan="1">904/905</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">22/28</td><td colspan="1" rowspan="1">78.6</td><td colspan="1" rowspan="1">60.5-89.8</td><td colspan="1" rowspan="1">662/666</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.5-99.8</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">33/41</td><td colspan="1" rowspan="1">80.5</td><td colspan="1" rowspan="1">66.0-89.8</td><td colspan="1" rowspan="1">1566/1571</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.9</td></tr><tr><td colspan="1" rowspan="3">hMPV f</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">913/913</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">68/70</td><td colspan="1" rowspan="1">97.1</td><td colspan="1" rowspan="1">90.2-99.2</td><td colspan="1" rowspan="1">616/624</td><td colspan="1" rowspan="1">98.7</td><td colspan="1" rowspan="1">97.5-99.3</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">73/75</td><td colspan="1" rowspan="1">97.3</td><td colspan="1" rowspan="1">90.8-99.3</td><td colspan="1" rowspan="1">1529/1537</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.0-99.7</td></tr><tr><td colspan="1" rowspan="3">HRV/EV</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">320/328</td><td colspan="1" rowspan="1">97.6</td><td colspan="1" rowspan="1">95.3-98.8</td><td colspan="1" rowspan="1">532/590</td><td colspan="1" rowspan="1">90.2</td><td colspan="1" rowspan="1">87.5-92.3</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">105/108</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">92.1-99.1</td><td colspan="1" rowspan="1">567/586</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">95.0-97.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">425/436</td><td colspan="1" rowspan="1">97.5</td><td colspan="1" rowspan="1">95.5-98.6</td><td colspan="1" rowspan="1">1099/1176</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">91.9-94.7</td></tr><tr><td colspan="1" rowspan="3">FluA h</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">915/915</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">75/75</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.1-100</td><td colspan="1" rowspan="1">616/616</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">78/78</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.3-100</td><td colspan="1" rowspan="1">1531/1531</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">FluA H1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">918/918</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">691/691</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1609/1609</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">FluA H1-2009</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">918/918</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.1-100</td><td colspan="1" rowspan="1">617/617</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.1-100</td><td colspan="1" rowspan="1">1535/1535</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">FluA H3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">915/915</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">690/690</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">1605/1605</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">FluB i</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">918/918</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100</td><td colspan="1" rowspan="1">678/680</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">98.9-99.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100</td><td colspan="1" rowspan="1">1596/1598</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="1" rowspan="3">PIV1 j</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">913/913</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">689/690</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100</td><td colspan="1" rowspan="1">1602/1603</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">TP/(TP +FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN +FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="3">PIV2 k</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">88.9-99.6</td><td colspan="1" rowspan="1">863/871</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.2-99.5</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">694/694</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">88.9-99.6</td><td colspan="1" rowspan="1">1557/1565</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.0-99.7</td></tr><tr><td colspan="1" rowspan="3">PIV3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/42</td><td colspan="1" rowspan="1">95.2</td><td colspan="1" rowspan="1">84.2-98.7</td><td colspan="1" rowspan="1">867/876</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.1-99.5</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">690/691</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">43/45</td><td colspan="1" rowspan="1">95.6</td><td colspan="1" rowspan="1">85.2-98.8</td><td colspan="1" rowspan="1">1557/1567</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="3">PIV4 m</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100</td><td colspan="1" rowspan="1">910/912</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.2-99.9</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">686/691</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.3-99.7</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100</td><td colspan="1" rowspan="1">1596/1603</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.1-99.8</td></tr><tr><td colspan="1" rowspan="3">RSV n</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">44/45</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">88.4-99.6</td><td colspan="1" rowspan="1">867/873</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.5-99.7</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">131/131</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.2-100</td><td colspan="1" rowspan="1">545/563</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">95.0-98.0</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">175/176</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">96.9-99.9</td><td colspan="1" rowspan="1">1412/1436</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">97.5-98.9</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="3">B. parapertussis(IS1001) °</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">37.6-96.4</td><td colspan="1" rowspan="1">913/913</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">692/692</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">48.7-97.4</td><td colspan="1" rowspan="1">1605/1605</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">B. pertussis(ptxP) P</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">915/916</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">693/693</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">20.8-93.9</td><td colspan="1" rowspan="1">1608/1609</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="3">C. pneumoniae 9</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">915/916</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">691/691</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100</td><td colspan="1" rowspan="1">1606/1607</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="3">M. pneumoniae "</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100</td><td colspan="1" rowspan="1">897/901</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.9-99.8</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">48.7-97.4</td><td colspan="1" rowspan="1">686/687</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3</td><td colspan="1" rowspan="1">1583/1588</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.9</td></tr></table>

e Of the eight FN specimens, six were TP for CoV-HKU1. They were confirmed to be due to a known cross-reactivity with CoV-HKU1 by the comparator method. All six specimens were negative for CoV-OC43 when tested with two independent PCR assays; the remaining two FN specimens were negative for CoV-OC43 when tested using an independent molecular method. CoV-OC43 was detected in 2/5 FP specimens upon comparator method retest.

f Both FN specimens were negative for hMPV when tested using an independent molecular method. hMPV was detected in 6/8 FP specimens during discrepancy investigation; one was detected using an independent molecular method and five were detected upon comparator method retest.

g HRV/EV was detected in 5/11 FN specimens during discrepancy investigation; one was detected using an independent molecular method and four were detected upon FilmArray RP2 retest. HRV/EV was detected in 33/77 FP specimens during discrepancy investigation; four were detected using an independent molecular method and 29 were detected upon comparator method retest.

h Three specimens were excluded from influenza A analysis: one with a comparator method result of Influenza A (No Subtype Detected) and two FilmArray RP2 Influenza A (Equivocal) detections.

# Prospective Clinical Study Mixed Infection Analysis

FilmArray RP2 reported a total of 245 specimens with distinctive multiple organism detections ( $1 5 . 2 \%$ of all specimens, 245/1612; and $2 4 . 0 \%$ of all positive specimens, 245/1020; Table 30). The majority of multiple detections $( 1 9 0 / 2 4 5 ; 7 7 . 6 \% )$ contained two organisms, while $2 0 . 0 \%$ (49/245) contained three organisms, $1 . 6 \%$ (4/245) contained four organisms, $0 . 4 \%$ (1/245) contained five organisms, and $0 . 4 \%$ (1/245) contained six organisms. Out of the 245 specimens with multiple detections, 124 specimens $( 5 0 . 6 \%$ ; 124/245) were concordant with the comparator methods. One hundred twenty-one (121) specimens $( 4 9 . 4 \%$ ; 121/245) contained one or more organisms that had not been detected by the comparator methods (i.e., false positive results).

The three organisms that were most prevalent in multiple detections were also the three most prevalent organisms in the study as a whole (i.e. HRV/EV, RSV, and adenovirus).

Table 30: Prevalence of Analytes in Multiple Detections as determined by the FilmArray RP2   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="2" rowspan="1">Prevalence in Multiple Detections(N=245)</td></tr><tr><td colspan="3" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">34.7%</td></tr><tr><td colspan="1" rowspan="1">CoV-229E</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">16.7%</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">12.7%</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">7.8%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">13.5%</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">61.2%</td></tr><tr><td colspan="1" rowspan="1">FluA H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.7%</td></tr><tr><td colspan="1" rowspan="1">FluA H3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td></tr><tr><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2.0%</td></tr><tr><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">6.1%</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">8.6%</td></tr><tr><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.9%</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">42.9%</td></tr><tr><td colspan="3" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">B. parapertussis (IS1001)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">B. pertussis (ptxP)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.9%</td></tr></table>

All distinct co-infection combinations as detected by the FilmArray RP2 during the prospective clinical study are presented in Table 31 below.

Table 31: Distinct Co-infection Combinations Detected by the FilmArray RP2 in the Prospective Clinical Trial   

<table><tr><td colspan="6" rowspan="1">Distinct Co-infection CombinationsDetected by the FilmArray RP2</td><td colspan="1" rowspan="2">TotalCo-infections</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infections a</td><td colspan="1" rowspan="2">Discrepant Analyte(s) a</td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1">Analyte 4</td><td colspan="1" rowspan="1">Analyte 5Analyte 6</td><td colspan="1" rowspan="1">Analyte 5Analyte 6</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Adenovirus (15), HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">HRV/EV (3), RSV (4)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">CoV-HKU1 (4), RSV (3)</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">CoV-NL63 (2), RSV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">HRV/EV (6), PIV2 (2)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HRV/EV (3), PIV3 (4)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Adenovirus (4), RSV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Adenovirus (2), HRV/EV (3), RSV(1)</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoV-NL63 (2)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoV-HKU1 (1), HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">HRV/EV (3)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus (2), RSV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (1), CoV-HKU1 (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus (3), PIV3 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">hMPV (1), PIV4 (2)</td></tr><tr><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (1), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1), M. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoV-HKU1 (2)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (1), PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-229E</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-229E</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-229E (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoV-NL63 (2)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1), PIV4 (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV4 (1), RSV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1), HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluA H3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-229E</td><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-229E (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-NL63 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-HKU1 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV4 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-0C43 (1)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1), HRV/EV (1), FluB (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV1 (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1), PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-229E</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">:</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV4 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-OC43 (1)</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">CoV-OC43 (1)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">FluB (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluA H1-2009</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluA H3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV/EV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">B. parapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="6" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">245</td><td colspan="1" rowspan="1">121</td><td colspan="1" rowspan="1">135/554</td></tr><tr><td colspan="6" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">190</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">91/380</td></tr><tr><td colspan="6" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">35/147</td></tr><tr><td colspan="6" rowspan="1">Total Quadruple Infections</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5/16</td></tr><tr><td colspan="6" rowspan="1">Total Quintuple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2/5</td></tr><tr><td colspan="6" rowspan="1">Total Sextuple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2/6</td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by FilmArray RP2 but not detected by the comparator method. Of the 135 discrepant analytes (out of 554 total analytes), 66 $( 4 8 . 9 \% )$ were observed as being present in the specimen during discrepancy investigation; 43/135 $( 3 1 . 9 \% )$ were observed using an independent molecular method and 27/135 $( 2 0 . 0 \% )$ were observed upon comparator method retest.

Additional distinct co-infection combinations detected by the comparator method(s), but not detected by the FilmArray RP2 in the prospective clinical trial are presented in Table 32 below.

Table 32: Additional Distinct Co-infection Combinations Detected by the Comparator Method(s), but not detected by the FilmArray RP2 in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=4>Distinct Co-infection Combinations</td><td rowspan=1 colspan=1>TotalSpecimenswith</td><td rowspan=1 colspan=1>Number ofSpecimens withDiscrepant</td><td rowspan=1 colspan=1>DiscrepantAnalyte(s) a</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>FluA H1-2009</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1)</td></tr><tr><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>FluA H1-2009</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV-229E (1)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>CoV-OC43</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>CoV-OC43 (3)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>CoV-OC43</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV-OC43 (2)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>CoV-OC43</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV-0C43 (1)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/EV (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>B. parapertussis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>B. parapertussis</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV2 (1), PIV3</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Total Co-infections</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>18/91</td></tr><tr><td rowspan=1 colspan=1>Total Double Infections</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13/74</td></tr><tr><td rowspan=1 colspan=1>Total Triple Infections</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4/9</td></tr><tr><td rowspan=1 colspan=1>Total Quadruple Infections</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/8</td></tr></table>

a This table includes only distinct co-infections that were detected by the comparator method(s) but not by FilmArray RP2; the remaining co-infections detected by the comparator method(s) are already represented in Table 32 above. b Of the six FilmArray RP2 FN specimens, all were TP for CoV-HKU1. They were confirmed to be due to a known cross-reactivity with CoV-HKU1 of the comparator method. All six specimens were negative for CoV-OC43 when tested with two independent PCR assays.

# Retrospective Clinical Study

Some of the analytes on the FilmArray RP2 were of low prevalence and were not encountered in sufficiently large numbers during the prospective study to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective specimens was performed at BioFire Diagnostics. These specimens were archived NPS in VTM specimens that were selected because they had previously tested positive for one of the following analytes at the source laboratory: coronavirus 229E, influenza A H1, influenza A H3, influenza B, parainfluenza virus 1, parainfluenza virus 4, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, parainfluenza virus 2, parainfluenza virus 3, and Mycoplasma pneumoniae.

A total of 217 clinical specimens were initially received for testing in this retrospective study (Table 34 below). In preparation for testing, a study code number (SCN) was assigned to each specimen and a key was created in order to randomize them such that operators were blinded to the expected test result.

Prior to testing with the FilmArray RP2, the composition/integrity of the pre-selected specimens was confirmed with the same comparator methods employed in the prospective clinical study (i.e., PCR followed by bi-directional sequencing assays for $B$ . parapertussis or an FDA-cleared multiplexed respiratory pathogen panel for all other analytes). Specimens were divided into two different groups for testing based on the method of confirmation testing performed: all specimens containing analytes on the FDA-cleared multiplexed respiratory pathogens panel comparator method were tested in Group 1 and specimens containing $B _ { \cdot }$ . parapertussis were tested in Group 2. Negative NPS specimens were included in each group. Table 33 below shows the number of specimens for each analyte (and negatives) that were tested with the FilmArray RP2 in this study.

Table 33: Archived Specimens Used in the Retrospective Clinical Study   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Number of Specimens</td></tr><tr><td colspan="2" rowspan="1">Group 1</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1 a</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">18</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">197 a</td></tr><tr><td colspan="2" rowspan="1">Group 2</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Total Overall</td><td colspan="1" rowspan="1">217</td></tr></table>

One specimen contained two analytes of interest

The FDA-cleared multiplexed respiratory pathogen panel comparator method was performed on 197 of the 217 achieved specimens (Group 1). One of the 197 specimens was excluded from performance analysis because of an invalid RP2 run with insufficient volume to retest. Additionally, two of the 197 specimens were also excluded from performance analysis because a valid FDA-cleared multiplexed respiratory pathogen panel comparator method confirmation result was not obtained and there was insufficient specimen volume for retesting: one comparator run was incomplete and the other comparator run had a control failure. FilmArray RP2 results for these specimens are shown below in Table 34. Valid comparator method and FilmArray RP2 results were obtained for 194 of the 197 archived specimens (Group 1).

Table 34: Specimens Excluded Because of Invalid Comparator Test and Insufficient Volume for Retesting   

<table><tr><td rowspan=1 colspan=1>SCN</td><td rowspan=1 colspan=1>Analyte Detected at the SourceLaboratory</td><td rowspan=1 colspan=1>Comparator MethodConfirmationResult</td><td rowspan=1 colspan=1>FilmArray RP2Result</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0004</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0127</td><td rowspan=1 colspan=1>PIV4</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>PIV4</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0170</td><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>C. pneumoniae</td></tr></table>

The $B _ { \cdot }$ . parapertussis PCR followed by bi-directional sequencing comparator assays were performed on 20 of the 217 achieved specimens (Group 2). The FDA-cleared multiplexed respiratory pathogens panel comparator method was not performed on Group 2 specimens. Valid comparator method and FilmArray RP2 results were obtained for 20 of these 20 archived specimens.

In addition, all Group 1 and Group 2 positive archived specimens (as determined at the source laboratory) that were not confirmed by the respective comparator method were also excluded from the performance calculation for each of the respective analyte. FilmArray RP2 results for these excluded specimens are shown below in Table 35..

Table 35: Specimens Excluded Because of Unconfirmed Comparator Method Results   

<table><tr><td rowspan=1 colspan=1>SCN</td><td rowspan=1 colspan=1>Analyte Detected at the SourceLaboratory</td><td rowspan=1 colspan=1>Comparator MethodConfirmationResult</td><td rowspan=1 colspan=1>FilmArray RP2Result</td></tr><tr><td rowspan=1 colspan=4>Samples Excluded from RP2 CoV-229E Performance Analysis Because of Unconfirmed Comparator Method CoV-229E</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0026</td><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0049</td><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td><td rowspan=1 colspan=1>CoV-229E</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0055</td><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0143</td><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>CoV-229E Not Detected</td><td rowspan=1 colspan=1>CoV-229E</td></tr><tr><td rowspan=1 colspan=4>Samples Excluded from RP2 Flu B Performance Analysis Because of Unconfirmed Comparator Method Flu B Result</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0199</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Influenza B Not Detected</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=4>Samples Excluded from RP2 PIV 2 Performance Analysis Because of Unconfirmed Comparator Method PIV 2 Result</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0014</td><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>PIV 2 Not Detected</td><td rowspan=1 colspan=1>PIV 2 Not Detected</td></tr><tr><td rowspan=1 colspan=4>Samples Excluded from RP2 B. pertussis Performance Analysis Because of Unconfirmed Comparator Method B.</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0028</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0036</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0052</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0087</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0097</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0131</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td><td rowspan=1 colspan=1>B. pertussis Not Detected</td></tr><tr><td rowspan=1 colspan=1>Samples Excluded from</td><td rowspan=1 colspan=1>RP2 C. pneumoniae Performance Ana</td><td rowspan=1 colspan=2>ysis Because of Unconfirmed Comparator Method C.</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0145</td><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>C. pneumoniae Not</td><td rowspan=1 colspan=1>C. pneumoniae</td></tr><tr><td rowspan=1 colspan=2>Samples Excluded from RP2 M. pneumoniae Performance Analysis Because of Unconfirmed Comparator Method M.</td><td rowspan=1 colspan=2>ysis Because of Unconfirmed Comparator Method M.</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0094</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M. pneumoniae Not</td><td rowspan=1 colspan=1>M. pneumoniae Not</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0104</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M. pneumoniae Not</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0105</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M. pneumoniae Not</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0138</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M. pneumoniae Not</td><td rowspan=1 colspan=1>M. pneumoniae Not</td></tr><tr><td rowspan=1 colspan=1>026979-ARC-0149</td><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M. pneumoniae Not</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr></table>

Table 36: Analyte Confirmation Summary for All Group 1 and Group 2 Specimens   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Number of Specimens</td><td colspan="1" rowspan="1">Number of Specimens Confirmed bythe Comparator Method (%)</td></tr><tr><td colspan="2" rowspan="1">Group 1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">15 (78.9%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">17 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">16 (94.1%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1 a</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">16 (94.1%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">17 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16 (100%)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">25 (80.6%)</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">16 (94.1%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae a</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">16 (76.2%)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4 (100%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">194 a</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Group 2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16 (100%)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4 (100%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td></tr><tr><td colspan="1" rowspan="1">Total Overall</td><td colspan="1" rowspan="1">214</td><td colspan="1" rowspan="1"></td></tr></table>

One specimen contained two analytes of interest

A summary of the available demographic information for the 214 valid specimens is provided in 37 below.

Table 37: Available Demographic Summary for All Valid Retrospective Specimens Included in the Retrospective Study   

<table><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>75 (35%)</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>81 (38%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>58 (27%)</td></tr><tr><td rowspan=1 colspan=1>Age Range</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>78 (36%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>46 (21%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22 - 49 years</td><td rowspan=1 colspan=1>13 (6%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>50+ years</td><td rowspan=1 colspan=1>19 (9%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>58 (27%)</td></tr><tr><td rowspan=1 colspan=2>Total Specimens</td><td rowspan=1 colspan=1>214</td></tr></table>

# Retrospective Clinical Study Performance

The FilmArray RP2 retrospective performance data expressed as positive percent and negative percent agreements against the comparator methods are presented by analyte in Table 38:

Table 38: FilmArray RP2 Retrospective Clinical Study Performance Summary   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN +FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="7" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">189/194</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">94.1-98.9</td></tr><tr><td colspan="1" rowspan="1">CoV- 229E a</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100</td><td colspan="1" rowspan="1">175/175</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">194/194</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">192/192</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">194/194</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">192/193</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">97.1-99.9</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">18/19</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">75.4-99.1</td><td colspan="1" rowspan="1">168/175</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">92.0-98.0</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">85.1-100</td><td colspan="1" rowspan="1">172/172</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.8-100</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">191/191</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009-H1</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">193/193</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">82.4-100</td><td colspan="1" rowspan="1">176/176</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">Influenza B b</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">177/177</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">178/178</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2 c</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">177/177</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100</td><td colspan="1" rowspan="1">175/177</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">96.0-99.7</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100</td><td colspan="1" rowspan="1">174/177</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">95.1-99.4</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">191/192</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">97.1-99.9</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis(IS1001) d</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis (ptxP)e</td><td colspan="1" rowspan="1">25/26</td><td colspan="1" rowspan="1">96.2</td><td colspan="1" rowspan="1">81.1-99.3</td><td colspan="1" rowspan="1">160/162</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">95.6-99.7</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae </td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100</td><td colspan="1" rowspan="1">176/176</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.9-100</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae 8</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">171/173</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">95.9-99.7</td></tr></table>

a Four of 19 CoV-229E positive archived specimens by the source laboratory were not confirmed by the comparator method and therefore were excluded from the performance calculation for CoV-229E .

b One of the 17 Influenza B positive archived specimens by the source laboratory was not confirmed by the comparato method and therefore was excluded from the performance calculation for Influenza B.

c One of the 17 Parainfluenza Virus 2 positive archived specimens the source laboratory was not confirmed by the comparator method and therefore was excluded from the performance calculation for Parainfluenza Virus 2 .

d The comparator $B$ . parapertussis PCR followed by sequencing assays were performed on 20 achieved specimens only (Group 2). The comparator method for the other analytes was not performed on these 20 specimens.

e Six of the $3 1 ~ B$ . pertussis positive archived specimens by the source laboratory were not confirmed by the comparator method and therefore were excluded from the performance calculation for $B$ . pertussis.

f One of the $1 7 ~ C .$ . pneumoniae positive archived specimens by the source laboratory was not confirmed by the comparator method and therefore was excluded from the performance calculation for $C .$ . pneumoniae.

g Five of the $2 1 M .$ . pneumoniae positive archived specimens by the source laboratory were not confirmed by the comparator method and therefore were excluded from the performance calculation for $M .$ . pneumoniae.

# Testing of Contrived Clinical Specimens

Influenza A H1 is of such rarity that both prospective and retrospective testing efforts were insufficient to demonstrate system performance. To supplement the prospective and retrospective data, an evaluation of contrived specimens was performed at one of the three clinical sites participated in the prospective evaluation. Contrived clinical specimens were prepared using individual unique residual NPS specimens that had previously tested negative by the FDA-cleared multiplexed respiratory pathogens panel (i.e., the same test as the comparator method employed in the prospective and retrospective clinical evaluations) at the source laboratory.

Spiking was performed using multiple quantified isolates of Influenza A H1. At least 25 of the contrived positive specimens had analyte concentrations at $2 \times$ the limit of detection (LoD), while the remaining 25 contrived positive specimens were at additional concentrations that spanned the clinically relevant range which was based on FilmArray RP2 Cp observations of influenza A (A H1, A H-2009, and H3) from the prospective and archived specimen studies. Contrived positive specimens were prepared and randomized along with 50 un-spiked influenza A H1 negative specimens such that the analyte status of each contrived specimen was unknown to the users performing the testing. The results of the FilmArray RP2 testing are presented in 39 below.

Table 39: FilmArray RP2 Performance Testing Contrived Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=4>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>× LoD</td><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22/23 a</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>79.0-99.2</td><td rowspan=6 colspan=1>50/50</td><td rowspan=6 colspan=1>100</td><td rowspan=6 colspan=1>92.9-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>72.3-100</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6-100</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6-100</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>47/48 a</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>89.1-99.6</td></tr></table>

a The FN specimen was spiked with influenza A/Weiss/43; this strain was detected at all other concentrations. Two specimens (also spiked with influenza A/Weiss/43) had a result of Influenza A Equivocal or Influenza A H1 Equivocal and were excluded from Influenza A H1 performance calculation.

4. Clinical cut-off: Not applicable   
5. Expected values/Reference range:

Table 40: Expected Value (As Determined by FilmArray RP2) Summary by Collection Site for the FilmArray RP2 Prospective Clinical Evaluation (Category II Archived Prospective Specimens) (January 2016 – March 2016)   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="1">Overall (n=694)</td><td colspan="2" rowspan="1">Site 1 (n=250)Salt Lake City,UT</td><td colspan="2" rowspan="1">Site 2 (n=243)Chicago, IL</td><td colspan="2" rowspan="1">Site 3 (n=201)Columbus, OH</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue (%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue(%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue(%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue (%)</td></tr><tr><td colspan="9" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">7.5%</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">7.2%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">8.2%</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">7.0%</td></tr><tr><td colspan="1" rowspan="1">CoV- 229E</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1.0%</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">7.8%</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">11.2%</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">6.6%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">7.1%</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">9.6%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">7.0%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4.0%</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">3.7%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">4.1%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4.0%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">11.0%</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">10.4%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">10.3%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">12.4%</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">124</td><td colspan="1" rowspan="1">17.9%</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">17.2%</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">18.1%</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">18.4%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">10.8%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">11.1%</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">18.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009-H1</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">10.7%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">11.1%</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">18.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3.0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1.0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1.0%</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">149</td><td colspan="1" rowspan="1">21.5%</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">23.6%</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">21.0%</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">19.4%</td></tr><tr><td></td><td></td><td></td><td colspan="1" rowspan="1">Bacteria</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis (IS1001)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis(ptxP)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr></table>

Table 41: Expected Value (As Determined by FilmArray RP2) Summary by Collection Site for the FilmArray RP2 Prospective Clinical Evaluation (Category I Fresh Prospective Specimens) (September 2016 – November 2016)   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="1">Overall (n=918)</td><td colspan="2" rowspan="1">Site 1 (n=331)Salt Lake City,UT</td><td colspan="2" rowspan="1">Site 2 (n=284)Chicago, IL</td><td colspan="2" rowspan="1">Site 3 (n=303)Columbus, OH</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue (%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue(%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue(%)</td><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">ExpectedValue (%)</td></tr><tr><td colspan="9" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">7.2%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">7.6%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">11.2%</td></tr><tr><td colspan="1" rowspan="1">CoV- 229E</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">CoV-HKU1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">CoV-NL63</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">CoV-OC43</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.3%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">378</td><td colspan="1" rowspan="1">41.2%</td><td colspan="1" rowspan="1">146</td><td colspan="1" rowspan="1">44.1%</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">24.3%</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">53.8%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009-H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">5.9%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2.4%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">4.6%</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">10.9%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">5.3%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">6.0%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">4.6%</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">5.3%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.3%</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">5.4%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2.7%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">11.9%</td></tr><tr><td colspan="9" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.3%</td></tr><tr><td colspan="1" rowspan="1">parapertussis (IS1001)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis(ptxP)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.0%</td></tr></table>

Table 42: Expected Value (As Determined by FilmArray RP2) Summary by Age Group for the FilmArray RP2 Prospective Clinical Evaluation (Category I and II Prospective Specimens) (January 2016 – March 2016 and September 2016 – November 2016)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Overall(N=1612)</td><td rowspan=1 colspan=2>≤5 years(N=885)</td><td rowspan=1 colspan=2>6-21 years(N=331)</td><td rowspan=1 colspan=2>22-49 years(N=128)</td><td rowspan=1 colspan=2>50+ years(N=268)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=10>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>10.8%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.7%</td></tr><tr><td rowspan=1 colspan=1>CoV- 229E</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>CoV-NL63</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>CoV-OC43</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>502</td><td rowspan=1 colspan=1>31.1%</td><td rowspan=1 colspan=1>379</td><td rowspan=1 colspan=1>42.8%</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>26.6%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009-H1</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.7%</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>12.3%</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>4.9%</td></tr><tr><td rowspan=1 colspan=11>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis (IS1001)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis (ptxP)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td></tr></table>

Table 43: Expected Value (Multiple Detections As Determined by FilmArray RP2) Summary by Age Group for the FilmArray RP2 Prospective Clinical Evaluation (Category I and II Prospective Specimens) (January 2016 – March 2016 and September 2016 – November 2016)   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Overall(N=1612</td><td rowspan=1 colspan=1>≤5 years(N=885)</td><td rowspan=1 colspan=1>6-21 years(N=331)</td><td rowspan=1 colspan=1>22-49 years(N=128)</td><td rowspan=1 colspan=1>50+ years(N=268</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/EV</td><td rowspan=1 colspan=1>30 (1.9%)</td><td rowspan=1 colspan=1>27 (3.1%)</td><td rowspan=1 colspan=1>3 (0.9%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + RSV</td><td rowspan=1 colspan=1>22 (1.4%)</td><td rowspan=1 colspan=1>22 (2.5%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1 + RSV</td><td rowspan=1 colspan=1>13 (0.8%)</td><td rowspan=1 colspan=1>12 (1.4%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>CoV-NL63 + RSV</td><td rowspan=1 colspan=1>13 (0.8%)</td><td rowspan=1 colspan=1>12 (1.4%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV2</td><td rowspan=1 colspan=1>11 (0.7%)</td><td rowspan=1 colspan=1>9(1.0%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV3</td><td rowspan=1 colspan=1>11 (0.7%)</td><td rowspan=1 colspan=1>10 (1.1%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + RSV</td><td rowspan=1 colspan=1>10 (0.6%)</td><td rowspan=1 colspan=1>8 (0.9%)</td><td rowspan=1 colspan=1>2 (0.6%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/EV + RSV</td><td rowspan=1 colspan=1>9 (0.6%)</td><td rowspan=1 colspan=1>9 (1.0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>CoV-NL63 + HRV/EV</td><td rowspan=1 colspan=1>8 (0.5%)</td><td rowspan=1 colspan=1>7 (0.8%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1 + HRV/EV</td><td rowspan=1 colspan=1>5 (0.3%)</td><td rowspan=1 colspan=1>3 (0.3%)</td><td rowspan=1 colspan=1>2 (0.6%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>CoV-OC43 + HRV/EV</td><td rowspan=1 colspan=1>5 (0.3%)</td><td rowspan=1 colspan=1>5 (0.6%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>hMPV + HRV/EV</td><td rowspan=1 colspan=1>5 (0.3%)</td><td rowspan=1 colspan=1>5 (0.6%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

# N. Instrument Name:

FilmArray 2.0 or FilmArray Torch System

# O. System Descriptions:

1. Modes of Operation:

After samples and hydration reagent have been placed in the reagent pouch, the remaining processing steps are executed under control of the instrument.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: Specimen identification can be entered manually or via a barcode.

4. Specimen Sampling and Handling:

The FilmArray RP2 is intended for use with nasopharyngeal swab (NPS) collected in VTM specimens. The operator places a Hydration Injection Vial and a Sample Injection Vial into the FilmArray Pouch Loading Station. The operator first hydrates the test pouch with the Hydration Injection Vial and then adds Sample Buffer into the Sample Injection Vial using the provided Sample Buffer ampoule. Using a transfer pipette provided in the kit, the operator adds $\sim 3 0 0 \mu \mathrm { l }$ of specimen into the Sample Injection Vial, closes the Sample Injection Vial, removes the Sample Injection Vial containing the sample mixture from the Loading Station, inverts the vial at least three times to mix, and then inserts it into the Loading Station port where the proper amount of specimen is pulled into the FilmArray RP2 pouch by vacuum. The FilmArray RP2 pouch is then placed in the FilmArray 2.0 instrument or the available module of a FilmArray Torch system for testing.

5. Calibration: Not applicable

6. Quality Control:

Two process controls are included in each FilmArray RP2 pouch:

# RNA Process Control

The RNA Process Control assay targets an RNA transcript from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and becomes rehydrated when sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, reverse transcription, PCR1, dilution, PCR2, and DNA melting. A positive control result indicates that all steps carried out in the FilmArray RP2 pouch were successful.

# PCR2 Control

The PCR2 Control assay detects a DNA target that is dried into wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

Both control assays must be positive for the test run to pass. If the controls fail, the sample should be retested using a new pouch.

The FilmArray Software automatically fails the run if the melting temperature (Tm) for either the RNA Process Control or the PCR2 Control is outside of an acceptable range (i.e., $8 0 . 3 \substack { - 8 4 . 3 ^ { \circ } \mathrm { C } }$ for the RNA Process Control and $7 3 . 8 – 7 7 . 8 ^ { \circ } \mathrm { C }$ for the PCR2 Control). If required by local, state, or accrediting organization, users can monitor the system by trending Tm values for the control assays and maintaining records according to standard laboratory quality control practices.

External controls are not provided with the FilmArray RP2. However, the sponsor is making the following recommendations in the product package insert regarding running external controls:

“Good laboratory practice recommends running external positive and negative controls regularly. Transport media can be used as an external negative control. Previously

characterized positive samples or negative samples spiked with well characterized organisms can be used as external positive controls. External controls should be used in accordance with the appropriate accrediting organization requirements, as applicable.”

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.