# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K170308

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for the BD MAX Extended Enteric Bacterial Panel on the BD MAX System.

C. Measurand: Target DNA sequences for:

· Plesiomonas shigelloides   
· Vibrio (V. vulnificus, V. parahaemolyticus, and V. cholerae)   
· Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable enterotoxin (ST) genes   
· Yersinia enterocolitica

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for the amplification and detection of DNA from Plesiomonas shigelloides, Vibrio (V. vulnificus, V.   
parahaemolyticus, and V. cholerae), Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable   
enterotoxin (ST) genes and Yersinia enterocolitica .

# E. Applicant:

Becton Dickinson and Company, BD Diagnostic Systems

# F. Proprietary and Established Names:

BD MAX Extended Enteric Bacterial Panel (xEBP) BD MAX System

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 – Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification:

Class II

3. Product code: PCI, PCH, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use:

The BD MAX Extended Enteric Bacterial Panel performed on the BD MAX System, is an   
automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric bacterial pathogens. It is used in conjunction with the BD MAX Enteric Bacterial Panel as an optional Master Mix. The BD MAX Extended Enteric Bacterial Panel detects nucleic acids from   
· Plesiomonas shigelloides   
· Vibrio (V. vulnificus, V. parahaemolyticus, and V. cholerae)   
· Enterotoxigenic Escherichia coli (ETEC) heat-labile enterotoxin (LT)/ heat-stable   
· enterotoxin (ST) genes   
· Yersinia enterocolitica

Testing is performed on soft to diarrheal unpreserved stool specimens or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of relevant gene target DNA. The test utilizes fluorogenic gene-specific hybridization probes for the detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Plesiomonas shigelloides, Vibrio (V. vulnificus, V. parahaemolyticus, and V. cholerae), Enterotoxigenic Escherichia coli (ETEC) LT/ST and Yersinia enterocolitica infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out coinfection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For Prescription Use Only

4. Special instrument requirements: The assay is performed on the BD MAX System.

# I. Device Description:

The BD MAX System and the BD MAX Extended Enteric Bacterial Panel are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, real-time PCR master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes (SBT). The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX System software automatically interprets test results.

# Brief Explanation of the Procedure

A soft to diarrheal stool is collected and transported to the laboratory. After the stool has been homogenized, a disposable inoculating loop is used to collect a $1 0 ~ \mu \mathrm { L }$ aliquot of the stool material and the contents of the loop are dispensed into a SBT. The SBT is closed with a septum cap and vortexed. A worklist is created and the SBT, the BD MAX Enteric Bacterial Panel Unitized Reagent Strips (URS) and the BD MAX PCR cartridge are loaded onto the BD MAX System. The BD MAX System automates sample preparation including cell lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. Amplified targets are detected with hydrolysis probes labeled with quenched fluorophores. The amplification, detection and interpretation of the signals are done automatically by the BD MAX System.

Reagents Provided with the BD MAX Extended Enteric Bacterial Panel

BD MAX Extended Enteric Bacterial Panel Master Mix: Dried PCR Master Mix containing DNA polymerase, nucleotides, target and Sample Processing Controlspecific probe and primers.

· BD MAX Enteric Bacterial Panel (BD Diagnostic Systems Catalog no. 442963)   
· BD MAX PCR Cartridges (BD Diagnostic Systems Catalog no. 437519)   
· VWR Multi-Tube Vortexer (VWR catalog no. 58816-115)   
· Vortex Genie 2 or equivalent (VWR, Cat. No. 58815-234)NALGENE $^ \mathrm { \textregistered }$   
· Cryogenic Vial Holder (VWR, catalog no. 66008-783) Rack compatible with a multi-tube vortex mixer (e.g., Cryogenic Vial Holder or equivalent)   
· Disposable $1 0 ~ \mu \mathrm { L }$ inoculating loops (BD Diagnostic Systems Catalog no. 220216) Lab coat and disposable gloves, powderless Sterile scissors (optional)   
· Stopwatch or timer

# J. Substantial Equivalence Information:

1. Predicate device name(s): BioFire Diagnostics FilmArray Gastrointestinal (GI) Panel

2. Predicate 510(k) number(s): K160459

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">BD MAX xEBP</td><td colspan="1" rowspan="1">FilmArray GI Panel (K160459)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD MAX Extended Enteric BacterialPanel performed on the BD MAX System, isan automated in vitro diagnostic test for thedirect qualitative detection and differentiationof enteric bacterial pathogens. It is used inconjunction with the BD MAX EntericBacterial Panel as an optional Master Mix. TheBD MAX Extended Enteric Bacterial Paneldetects nucleic acids fromPlesiomonas shigelloidesVibrio (V. vulnificus, V. parahaemolyticus,and V. cholerae)•  Enterotoxigenic Escherichia coli (ETEC)heat-labile enterotoxin (LT)/ heat-stable•  enterotoxin (ST) genes2   Yersinia enterocoliticaTesting is performed on soft to diarrhealunpreserved stool specimens or Cary-Blair</td><td colspan="1" rowspan="1">The FilmArray Gastrointestinal (GI) Panel is aqualitative multiplexed nucleic acid-based in vitrodiagnostic test intended for use with FilmArraysystems. The FilmArray GI Panel is capable of thesimultaneous detection and identification of nucleicacids from multiple bacteria, viruses, and parasitesdirectly from stool samples in Cary Blair transportmedia obtained from individuals with signs and/orsymptoms of gastrointestinal infection. Thefollowing bacteria (including several diarrheagenicE. coli/Shigella pathotypes), parasites, and virusesare identified using the FilmArray GI Panel:- Campylobacter (C. jejuni/C. coli/C. upsaliensis)- Clostridium difficile (C. difficile) toxin A/B- Plesiomonas shigelloides- Salmonella- Vibrio (V. parahaemolyticus/V. vulnificus/V.cholerae), including specific identification ofVibrio cholerae- Yersinia enterocolitica- Enteroaggregative Escherichia coli (EAEC)- Enteropathogenic Escherichia coli (EPEC)- Enterotoxigenic Escherichia coli (ETEC) lt/st</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">preserved stool specimens from symptomaticpatients with suspected acute gastroenteritis,enteritis or colitis. The test is performeddirectly on the specimen, utilizing real-timepolymerase chain reaction (PCR) for theamplification of relevant gene target DNA.The test utilizes fluorogenic gene-specifichybridization probes for the detection of theamplified DNA.This test is intended for use, in conjunctionwith clinical presentation, laboratory findings,and epidemiological information, as an aid inthe differential diagnosis of Plesiomonasshigelloides, Vibrio Group (V. vulnificus, V.parahaemolyticus, and V. cholerae),Enterotoxigenic Escherichia coli (ETEC)LT/ST and Yersinia enterocolitica infections.Results of this test should not be used as thesole basis for diagnosis, treatment, or otherpatient management decisions. Positive resultsdo not rule out coinfection with otherorganisms that are not detected by this test, andmay not be the sole or definitive cause ofpatient illness. Negative results in the setting ofclinical illness compatible with gastroenteritismay be due to infection by pathogens that arenot detected by this test or non-infectiouscauses such as ulcerative colitis, irritablebowel syndrome, or Crohn's disease.</td><td colspan="1" rowspan="1">Shiga-like toxin-producing Escherichia coli(STEC) stx1/stx2 (including specific identificationof the E. coli O157 serogroup within STEC)- Shigella/Enteroinvasive Escherichia coli (EIEC)- Cryptosporidium- Cyclospora cayetanensis- Entamoeba histolytica- Giardia lamblia (also known as G. intestinalisand G. duodenalis)- Adenovirus F 40/41- Astrovirus- Norovirus GI/GIIRotavirus A- Sapovirus (Genogroups I, II, IV, and V)The FilmArray GI Panel is indicated as an aid inthe diagnosis of specific agents of gastrointestinalillness and results are meant to be used inconjunction with other clinical, laboratory, andepidemiological data. Positive results do not ruleout co-infection with organisms not included in theFilmArray GI Panel. The agent detected may not bethe definite cause of the disease.Concomitant culture is necessary for organismrecovery and further typing of bacterial agents.This device is not intended to monitor or guidetreatment for C. difficile infection.Due to the small number of positive specimenscollected for certain organisms during theprospective clinical study, performancecharacteristics for E. coli O157, Plesiomonasshigelloides, Yersinia enterocolitica, Astrovirus,and Rotavirus A were established primarily withretrospective clinical specimens.Performance characteristics for Entamoebahistolytica, and Vibrio (V. parahaemolyticus, V.vulnificus, and Vibrio cholerae) were establishedprimarily using contrived clinical specimens.Negative FilmArray GI Panel results in the settingof clinical illness compatible with gastroenteritismay be due to infection by pathogens that are notdetected by this test or non-infectious causes suchas ulcerative colitis, irritable bowel syndrome, orCrohn's disease.A gastrointestinal microorganism multiplex nucleicacid-based assay also aids in the detection andidentification of acute gastroenteritis in the contextof outbreaks.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Cary-Blair preserved stoolUnpreserved soft to diarrheal stool</td><td colspan="1" rowspan="1">Cary-Blair preserved stool.Not claimed (see Differences below)</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCRDetection: fluorogenic target-specific hybridization.</td><td colspan="1" rowspan="1">Amplification: PCRDetection: non target-specific fluorescent dye</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Plesiomonas shigelloidesVibrio (V. vulnificus, V. parahaemolyticus, and V.cholerae)•Enterotoxigenic Escherichia coli heat labile andheat stabile (LT/ST) (ETEC)• Yersinia enterocolitica</td><td colspan="1" rowspan="1">Plesiomonas shigelloidesVibrio (V. parahaemolyticus/V. vulnificus/V.cholerae), including specific identification ofVibrio choleraeEnterotoxigenic Escherichia coli (ETEC) lt/stYersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofTest Results</td><td colspan="1" rowspan="1">Automated: BD MAX System diagnostic software</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Analysis Platform</td><td colspan="1" rowspan="1">BD MAX System</td><td colspan="1" rowspan="1">Film Array Instrument</td></tr><tr><td colspan="1" rowspan="1">PCR Samplepreparation</td><td colspan="1" rowspan="1">Automated:BD MAX System</td><td colspan="1" rowspan="1">Automated:Film Array Instrument</td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">TaqMan Probe</td><td colspan="1" rowspan="1">Fluorescent double stranded DNA binding dye (LCGreen Plus)</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample Processing Control (SPC)</td><td colspan="1" rowspan="1">Two controls are included in each reagent pouch tocontrol for sample processing and both stages ofPCR and melt analysis.</td></tr></table>

<table><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td>BD MAX xEBP</td><td>FilmArray GI Panel</td></tr><tr><td>Specimen Type</td><td>Unpreserved soft to diarrheal stool</td><td>Not claimed Other organisms detected:</td></tr><tr><td>Organisms Detected</td><td>Listed in device Similarities above.</td><td>Campylobacter (C. jejuni/C. coli/C. upsaliensis) Clostridium difficile (C. difficile) toxin A/B Salmonella Enteroaggregative Escherichia coli (EAEC) Enteropathogenic Escherichia coli (EPEC) Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli O157 serogroup within STEC) Shigella/ Enteroinvasive Escherichia coli (EIEC) Cryptosporidium Cyclospora cayetanensis Entamoeba histolytica Giardia lamblia (also known as G. intestinalis and G. duodenalis) Adenovirus F 40/41 Astrovirus Norovirus GI/GII</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guideline: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool

# L. Test Principle:

A stool specimen is collected and transported to the laboratory in a dry, clean container (for unpreserved specimens) or in Cary-Blair transport media. The specimen is vortexed for 15 seconds and then a $1 0 \mu \mathrm { L }$ loop is used to inoculate a BD MAX Extended Enteric Bacterial Panel and a BD MAX Extended Enteric Bacterial Panel Sample Buffer Tube included in the BD MAX Enteric Bacterial Panel kit. The Sample Buffer Tube is closed with a septum cap, vortexed and transferred to the BD MAX System. A worklist is created and the Sample Buffer Tubes, the BD MAX Enteric Bacterial Panel Unitized Reagent Strip (containing both the BD MAX EBP and BD MAX xEBP master mix reagents) and the BD MAX PCR Cartridges are loaded onto the BD MAX System.

Following enzymatic bacterial cell lysis at elevated temperature, the released nucleic acids are captured on magnetic beads. The beads, with the bound nucleic acids, are washed using Wash Buffer and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized using Neutralization Buffer and transferred to the Master Mix tubes to rehydrate the PCR reagents. After rehydration, the BD MAX System dispenses a fixed volume of PCRready solution containing extracted nucleic acids into the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed by the system prior to initiating PCR to contain the amplification mixture, thus preventing evaporation and amplicon contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect the amplicons of the enteric bacterial targets (Plesiomonas shigelloides, Vibrio ( $V .$ vulnificus, V. parahaemolyticus, and V. cholerae), heat labile and heat stabile (LT/ST) ETEC (Enterotoxigenic $E$ . coli) and Yersinia enterocolitica) and the Sample Processing Control amplicons in five different optical channels of the BD MAX System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the cDNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at each cycle, and interprets the data at the end of the program to report the final results.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

Precision

Within-laboratory precision was evaluated for the BD MAX Extended Enteric Bacterial Panel at one (1) internal site. Testing was performed over 12 days, with two runs per day (one each by 2 operators), for a total of 24 runs. The Precision panel members were divided into four (4) concentration categories, based upon organism concentration relative to the LoDs established for each of the assay targets and expected correct percent positive/negative. The panel members contained Vibrio cholerae, Plesiomonas shigelloides, ETEC and Yersinia enterocolitica. The following values were used as spike levels for the target organisms contained in each panel member:

· Moderate Positive (MP): ${ \ge } 2$ to $\leq 3 \mathbf { x }$ LoD; positive approximately $9 5 \%$ of the time · Low Positive (LP): ${ \geq } 1$ to $< 2 \mathbf { x }$ LoD; positive approximately $9 5 \%$ of the time · High Negative (HN): C20-80 LoD; negative between 20 and $80 \%$ of the time

Each panel member was spiked with negative unpreserved stool matrix. True negative samples contained no target. High negative samples were spiked with target organisms below the analytical LoD of the assay; however, the HN samples were expected to yield a positive result in approximately $20 \%$ to $80 \%$ of the replicates due the inherent sensitivity of the PCR assays. Results are summarized by target and concentration below.

Precision study result Using One Lot of the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td><td rowspan=1 colspan=1>10096/96(96.2, 100)</td></tr><tr><td rowspan=1 colspan=1>HNa</td><td rowspan=1 colspan=1>58.328/48(44.3, 71.2)</td><td rowspan=1 colspan=1>45.822/48(32.6, 59.7)</td><td rowspan=1 colspan=1>41.720/48(28.8, 55.7)</td><td rowspan=1 colspan=1>47.923/48(34.5, 61.7)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>97.947/48(89.1, 99.6)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>10048/48(92.6, 100)</td><td rowspan=1 colspan=1>97.947/48(89.1, 99.6)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

# Site-to-site Reproducibility

The Site-to-Site reproducibility study was performed at three (3) clinical sites using one (1) reagent lot. Two (2) operators performed 2 runs per day, over five (5) distinct days (consecutive or not), for a total of 30 runs. The panels used were the same as described under the Precision heading, above.

The overall site-to-site reproducibility percent agreement was $100 \%$ for the TN category for all targets, and ranged from 30.0 to $4 8 . 9 \%$ , 97.8 to $100 \%$ and 98.9 to $100 \%$ for the HN, LP and MP categories, respectively. Results are summarized in Table 1. The quantitative reproducibility results across sites by target are presented in Table 2. Ct. Score is an internal criterion used to determine final assay results and was selected as a means of assessing quantitative assay reproducibility. Mean Ct. Score and the mean Cycle EP values with variance components (SD and $\%$ CV) are shown in Table 2.

Table 1: Site-to-Site Reproducibility Results Using One Lot of the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td></tr><tr><td rowspan=1 colspan=1>HNa</td><td rowspan=1 colspan=1>48.944/90(38.8, 59.0)</td><td rowspan=1 colspan=1>30.027/90(21.5, 40.1)</td><td rowspan=1 colspan=1>35.632/90(26.4, 45.8)</td><td rowspan=1 colspan=1>46.742/90(36.7, 56.9)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>97.888/90(92.3, 99.4)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>98.989/90(94.0, 99.8)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Table 2: Quantitative Site-to-Site Reproducibility Results Summary   

<table><tr><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=3>Vibrio</td><td rowspan=1 colspan=3>P. shigelloides</td><td rowspan=1 colspan=3>Y. enterocolitica</td><td rowspan=1 colspan=3>ETEC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=4 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=4 colspan=1>CycleEP</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>805.1</td><td rowspan=1 colspan=1>2725.7</td><td rowspan=1 colspan=1>3076.6</td><td rowspan=1 colspan=1>596.5</td><td rowspan=1 colspan=1>2349.7</td><td rowspan=1 colspan=1>2320.9</td><td rowspan=1 colspan=1>599.6</td><td rowspan=1 colspan=1>1191.1</td><td rowspan=1 colspan=1>1264.2</td><td rowspan=1 colspan=1>976.0</td><td rowspan=1 colspan=1>2049.6</td><td rowspan=1 colspan=1>2640.0</td><td rowspan=1 colspan=1>6020.7</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>350.40</td><td rowspan=1 colspan=1>812.29</td><td rowspan=1 colspan=1>485.26</td><td rowspan=1 colspan=1>284.02</td><td rowspan=1 colspan=1>416.36</td><td rowspan=1 colspan=1>630.12</td><td rowspan=1 colspan=1>296.66</td><td rowspan=1 colspan=1>317.31</td><td rowspan=1 colspan=1>388.98</td><td rowspan=1 colspan=1>577.46</td><td rowspan=1 colspan=1>529.33</td><td rowspan=1 colspan=1>598.65</td><td rowspan=1 colspan=1>688.74</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>43.5</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>49.5</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>59.2</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>11.4</td></tr></table>

The Lot-to-lot reproducibility study was performed at one (1) site using three (3) reagent lots. Two (2) operators performed 2 runs per day, over five (5) distinct days (consecutive or not), for a total of 30 runs. The panels used were the same as described under the Precision heading, above. Results from 5 days of the accuracy and precision study were used to comprise data for one lot of reagents for the Lot-toLot study.

The overall Lot-to-lot reproducibility percent agreements were $100 \%$ for TN, and ranged from 23.3 to $4 1 . 1 \%$ , 97.8 to $100 \%$ and 98.9 to $100 \%$ for the HN, LP and MP respectively. Results are summarized in Table 3. The quantitative results across lots and by target are presented in Table 8 and Table 6. Cycle threshold score and the Cycle EP, an internal criteria used to determine a final assay result, was selected as a means of assessing quantitative assay reproducibility. Mean Ct. Score and the mean Cycle EP values with variance components (SD and $\%$ CV) are shown in Table 4.

Table 3: Lot-to-lot Reproducibility Results for BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Vibrio(95% CI)</td><td rowspan=1 colspan=1>P. shigelloides(95% CI)</td><td rowspan=1 colspan=1>Y. enterocolitica(95% CI)</td><td rowspan=1 colspan=1>ETEC(95% CI)</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td><td rowspan=1 colspan=1>100180/180(97.9, 100)</td></tr><tr><td rowspan=1 colspan=1>HNa</td><td rowspan=1 colspan=1>41.137/90 (31.5,51.4)</td><td rowspan=1 colspan=1>28.926/90 (20.5,39.0)</td><td rowspan=1 colspan=1>23.321/90 (15.8,33.1)</td><td rowspan=1 colspan=1>41.137/90 (31.5,51.4)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>97.888/90 (92.3,99.4)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td><td rowspan=1 colspan=1>98.989/90 (94.0,99.8)</td><td rowspan=1 colspan=1>10090/90(95.9, 100)</td></tr></table>

a For the True Negative (TN) and High Negative (HN) categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Table 4: Quantitative Lot-to-lot Reproducibility Results Summary   

<table><tr><td rowspan=2 colspan=1>PCRMetric</td><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=3>Vibrio</td><td rowspan=1 colspan=3>P. shigelloides</td><td rowspan=1 colspan=3>Y. enterocolitica</td><td rowspan=1 colspan=3>ETEC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>TN</td></tr><tr><td rowspan=4 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.30</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=4 colspan=1>CycleEP</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>931.1</td><td rowspan=1 colspan=1>3124.7</td><td rowspan=1 colspan=1>3033.5</td><td rowspan=1 colspan=1>686.9</td><td rowspan=1 colspan=1>2444.1</td><td rowspan=1 colspan=1>2428.7</td><td rowspan=1 colspan=1>731.3</td><td rowspan=1 colspan=1>1339.7</td><td rowspan=1 colspan=1>1370.3</td><td rowspan=1 colspan=1>838.3</td><td rowspan=1 colspan=1>2056.8</td><td rowspan=1 colspan=1>2244.2</td><td rowspan=1 colspan=1>5544.7</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>369.81</td><td rowspan=1 colspan=1>242.75</td><td rowspan=1 colspan=1>236.53</td><td rowspan=1 colspan=1>337.36</td><td rowspan=1 colspan=1>491.79</td><td rowspan=1 colspan=1>416.43</td><td rowspan=1 colspan=1>296.44</td><td rowspan=1 colspan=1>158.41</td><td rowspan=1 colspan=1>165.73</td><td rowspan=1 colspan=1>332.37</td><td rowspan=1 colspan=1>419.99</td><td rowspan=1 colspan=1>458.59</td><td rowspan=1 colspan=1>633.70</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>49.1</td><td rowspan=1 colspan=1>20.1</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>11.4</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

External Positive Controls are intended to monitor for substantial reagent failure and External Negative Controls are used to detect reagent or environmental contamination (or carry-over) from positive specimens or nucleic acids (amplicon). External controls are not provided with the BD MAX Extended Enteric Bacterial Panel; however recommendations for control preparation and testing are provide in the package insert.

During the clinical study, external positive and negative controls were included in each run and in the case of a failure of either or both external controls, testing of all samples included in the run were repeated from the stored SBTs. Negative external controls consisted of saline. Positive controls were prepared as organism suspensions as detailed in the table below. Each positive control was tested on a rotating basis. BD MAX xEBP External Controls

External Negative and Positive Control Panels   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>ATCC Strain</td><td rowspan=1 colspan=1>SuspensionPreparation in Saline</td><td rowspan=1 colspan=1>Final Concentration inReady to use 2 mL Tubes</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>9610</td><td rowspan=4 colspan=1>0.5 McF</td><td rowspan=1 colspan=1>1.5 X 105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Vibrio choleraeorVibrio parahaemolyticus</td><td rowspan=1 colspan=1>14033or17802</td><td rowspan=1 colspan=1>1.5 X 105 CFU/mLor1.0 X 105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>14029</td><td rowspan=1 colspan=1>3.0 X 10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>35401</td><td rowspan=1 colspan=1>1.5 X 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>-</td></tr></table>

Each testing site included one (1) positive external control and one (1) negative external control in each BD MAX run. A total of 186 runs were performed. Of those, 171 runs $( 9 1 . 9 \% )$ passed. The pass rate ranged between $8 5 . 3 \%$ and $9 5 . 7 \%$ between sites. All samples included in a run with EC failure were repeated or remained invalid if it was a final run. Runs with valid External Controls remained high throughout the trial.

Internal Specimen Processing Control:

Each BD MAX Extended Enteric Bacterial Extraction Tube contains a Sample Processing Control (SPC) which is a plasmid containing a synthetic target DNA sequence. The SPC monitors the efficiency of DNA capture, washing and elution during the sample processing steps, as well as the efficiency of DNA amplification and detection during PCR analysis. If the SPC result fails to meet the acceptance criteria, the test result of will be reported as Unresolved (UNR). An Unresolved result is indicative of sample-associated inhibition or reagent failure. The user is instructed to repeat any sample reported as Unresolved.

Specimen Stability Study: (Preservation of DNA in stool or Cary-Blair Media)

Pooled unpreserved or Cary-Blair preserved negative stool specimens were spiked with BD MAX xEBP targets and tested according to the storage conditions described in the table below. An additional storage of 3 hours at room temperature was also applied to every SBT following inoculation in order to mimic a full run of the BD MAX xEBP, representing the normal customer workflow. To further represent normal workflow, SBT septum caps were pierced before storage at the appropriate temperature (including the initial 3-hour storage period).

Storage Conditions   

<table><tr><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=1>In Collection Device orTransportMedium</td><td rowspan=1 colspan=1>In SBT</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>No storage</td></tr><tr><td rowspan=1 colspan=1>1A</td><td rowspan=4 colspan=1>24h at25°C(Unpreserved and Cary-Blair preserved samples )</td><td rowspan=1 colspan=1>+ 48h at 25°C</td></tr><tr><td rowspan=1 colspan=1>1B</td><td rowspan=1 colspan=1>+ 48h + 10% at 25°C</td></tr><tr><td rowspan=1 colspan=1>1C</td><td rowspan=1 colspan=1>+ 120h at 2-8°C</td></tr><tr><td rowspan=1 colspan=1>1D</td><td rowspan=1 colspan=1>+ 120h + 20% at 2-8°C</td></tr><tr><td rowspan=1 colspan=1>2A</td><td rowspan=4 colspan=1>120h at 2-8°C(Unpreserved and Cary-Blair preserved samples )</td><td rowspan=1 colspan=1>+ 48h at 25°C</td></tr><tr><td rowspan=1 colspan=1>2B</td><td rowspan=1 colspan=1>+ 48h + 10% at 25°C</td></tr><tr><td rowspan=1 colspan=1>2C</td><td rowspan=1 colspan=1>+ 120h at 2-8°C</td></tr><tr><td rowspan=1 colspan=1>2D</td><td rowspan=1 colspan=1>+ 120h + 20% at 2-8°C</td></tr></table>

For each xEBP target and for all test conditions $\ge 9 5 \%$ expected results for positive and negative samples were required.

Only positive and negative results were included to determine the percentage of detection at each concentration. All non-reportable results (UNR/IND/INC) and unexpected results (ex: false positive in a channel not being evaluated) were required to be repeated from the SBT. Valid results upon repeat were included in the analysis. If the non-reportable/unexpected result did not resolve upon repeat, it was excluded from the study.

# Cary-Blair Preserved Samples:

The acceptance criteria were met for positive and negative samples at all time-points analyzed (T0 to 1D) following an initial incubation time of $2 4 \mathrm { h }$ at $2 5 \mathrm { { ^ \circ C } }$ in the collection device, except for one condition:

ETEC negative condition (1C): 24h at $2 5 \mathrm { ^ \circ C }$ plus 120h at $2 { - } 8 ^ { \circ } C$ resulted in $91 \%$ (51/56). The failure was investigated and determined to be a result of poor handling while preparing the samples for the study. Specifically, it was determined that the same pipette tip that was used for manually piercing the positive tubes was used for piercing the negative tubes, resulting in contamination. These failures are not linked to assay performance as is further demonstrated with passing results at the last time point (1D) (24h at $2 5 \mathrm { ^ \circ C }$ plus $1 2 0 h + 2 0 \%$ at $2 { - } 8 ^ { \circ } C )$ ).

The acceptance criteria were met for positive and negative samples at all time-points analyzed (2A to 2D) following an initial incubation time of $1 2 0 \mathrm { h }$ at $2 { - } 8 ^ { \circ } \mathrm { C }$ in the

collection device, except for two conditions, both representing buffer time point conditions:

· Plesiomonas shigelloides positive condition (2B): 120h at $2 { - } 8 ^ { \circ } \mathrm { C }$ plus 48h $+ 1 0 \%$ at $2 5 \mathrm { { ^ \circ C } }$ resulted in $93 \%$ (26/28); · Vibrio (V. cholerae, V. vulnificus and V. parahaemolyticus) positive condition (2D): 120h at $2 { - } 8 ^ { \circ } \mathrm { C }$ plus 120h $+ 2 0 \%$ at $2 { - } 8 ^ { \circ } \mathrm { C }$ resulted in $93 \%$ (26/28).

Considering that a trend was not observed, the results for Plesiomonas shigelloides are likely due to the variability of quantification at low levels, resulting in insufficient low target spike and thus not indicative of stability. The method for quantification is based on plate count, for which up to $30 \%$ of variation can be expected. For Vibrio, if the failure was truly due to stability, it is reasonable to expect a failure at condition 2B, whereby the samples were held at a worst case room temperature condition.

# Unpreserved Samples:

The acceptance criteria were met for positive and negative samples at all time-points analyzed (T0 to 1D) following an initial incubation time of $2 4 \mathrm { h }$ at $2 5 \mathrm { { ^ \circ C } }$ in the collection device, except for one condition:

ETEC negative condition (1C): 24h at $2 5 \mathrm { ^ \circ C }$ plus 120h $+ 2 0 \%$ at $2 { - } 8 ^ { \circ } C$ resulted in $93 \%$ (52/56). These samples were prepared by the same operator and therefore impacted by the same handling error as described above for ETEC negative condition (1) in preserved matrix.

The acceptance criteria were met for negative and positive samples at all time-points analyzed following an initial incubation time of $1 2 0 \mathrm { h }$ at $2 { - } 8 ^ { \circ } \mathrm { C }$ in the collection device, except for one condition:

Vibrio (V. cholerae, V. vulnificus and V. parahaemolyticus) positive condition (2B): 120h at $2 { - } 8 ^ { \circ } \mathrm { C }$ plus 48 $h + 1 0 \%$ at $2 5 \mathrm { { ^ \circ C } }$ resulted in $93 \%$ (26/28).

If the failure was truly due to stability, it is reasonable to expect a failure at condition 1B, whereby the samples were held at a worst case room temperature initial hold for 24 hours followed by room temperature storage for an additional $4 8 +$ hours.

# d. Detection limit:

The analytical sensitivity (Limit of Detection or LoD) for the BD MAX Extended Enteric Bacterial Panel was determined as follows: Each target organism was prepared and quantified from culture prior to inclusion in this study. Individual inoculating loops were dipped into each organism preparation and were then transferred to a Sample Buffer Tube already containing fecal matrix (preserved or unpreserved) that was pre-determined to be negative for all the targets detected by the BD MAX Extended Enteric Bacterial Panel. Each organism was tested with a

minimum of 24 replicates per sample type (preserved or unpreserved), by 2 operators, using 3 different production lots of the BD MAX Extended Enteric Bacterial Panel. The LoD for a specific organism was confirmed by testing at least 24 additional replicates at the determined LoD concentration. Analytical sensitivity (LoD), defined as the lowest concentration at which greater than $9 5 \%$ of all replicates are expected to test positive, ranged from 34 to 534 CFU/SBT and 3,434 to $5 3 , 8 5 2 \mathrm { C F U / m L }$ (in stool) for unpreserved specimens for both strains and 79 to 257 CFU/SBT and 7,860 to 25,712 CFU/mL (in stool) for preserved specimens.

BD MAX Extended Enteric Bacterial Panel Limit of Detection for Individual Target   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Unpreserved (1 strain)</td><td rowspan=1 colspan=2>Unpreserved (2 strain)</td><td rowspan=1 colspan=2>Cary-Blair Preserved (1t strain)</td></tr><tr><td rowspan=1 colspan=1>Target organism</td><td rowspan=1 colspan=1>LoD1(CFU/SBT)</td><td rowspan=1 colspan=1>LoD(CFU/mL instool)</td><td rowspan=1 colspan=1>LoD1(CFU/SBT)</td><td rowspan=1 colspan=1>LoD&#x27;(CFU/mL instool)</td><td rowspan=1 colspan=1>LoD1(CFU/SBT)</td><td rowspan=1 colspan=1>LoD(CFU/mL instool)</td></tr><tr><td rowspan=1 colspan=1>PlesiomonasshigelloidesATCC 14029; ATCC14030b</td><td rowspan=1 colspan=1>458</td><td rowspan=1 colspan=1>45,752</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>15,481</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>25,712</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocoliticaATCC 9610²; CCUG48</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>20,900</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>31,099</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>22,723</td></tr><tr><td rowspan=1 colspan=1>ETEC ST/LTATCC 35401²; CCUG47194</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>3,434</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>53,852</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>13,706</td></tr><tr><td rowspan=1 colspan=1>Vibrio choleraeATCC 14033²; ENF9786</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>14,942</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>4,344</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>25,238</td></tr><tr><td rowspan=1 colspan=1>VibrioparahaemolyticusATCC17802°; ENF 87</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>20,708</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>8,031</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>12,424</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificusATCC 27562² 2; ENF10727</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>13,093</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>7,959</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>7,860</td></tr></table>

1LoD concentrations are expressed in CFU/SBT and CFU/mL, except for Vibrio. for which it is expressed in cells/SBT and cells/mL. a1st strain b2nd strain

# e. Analytical Reactivity (Inclusivity):

A variety of BD MAX Extended Enteric Bacterial Panel assay target strains were included in this study. Strain selection criteria included prevalence, serotype and geographic location, where appropriate. Sixty-nine (69) strains were tested, including strains from public collections and well-characterized clinical isolates.

Inclusivity testing included 10 strains of Plesiomonas shigelloides, 10 strains of Yersinia enterocolitica, 36 strains of Vibrio (cholerae, parahaemolyticus and vulnificus) and 13 strains of ETEC LT/ST. The strains were tested at $< 3 \mathrm { ~ x ~ }$ LoD (Limit of Detection) of the corresponding strain in unpreserved stool matrix. The BD MAX Extended Enteric Bacterial Panel correctly identified 68 of the 69 strains tested upon initial testing. One strain of ETEC ST/LT (CCUG 38088) did not meet acceptance criteria and was further evaluated. This strain was titrated and tested to determine the minimum concentration sufficient for detection. Upon repeat, the CCUG 38088 strain of ETEC ST/LT was detected at $1 0 \mathrm { x }$ LoD.

Three replicates of each strain were tested at $2 . 9 9 \mathrm { x }$ LoD in unpreserved stool matrix. Three different combinations of reagent lots and 3 lots of negative unpreserved stool matrix were used.

Analytical Reactivity Results in Unpreserved Stool   

<table><tr><td colspan="1" rowspan="2">SBT</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">ATCC orother collectionnumber</td><td colspan="2" rowspan="1">Assay</td><td colspan="1" rowspan="1">result at</td><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">Non-Reportable</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Yersi</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">Vibrio</td><td colspan="1" rowspan="1">Plesio</td><td colspan="1" rowspan="1">Straindetected</td><td colspan="1" rowspan="1">IND</td><td colspan="1" rowspan="1">UNR</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">SBToverall</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="10">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ATCC 14030</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ENF 9782</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">ENF 9000</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">ENF 10445</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">ENF 15744</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">CCUG 7041A</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">CCUG 9221</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">CCUG 14309</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">CCUG 14597</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="10">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">CCUG 4588</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">ATCC 9610</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">ATCC 27729</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">CCUG 8050</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">CCUG 8232</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">CCUG 8234</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="6">Vibrioparahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">ENF 5887</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">ATCC BAA-241</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">ATCC 33847</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">ATCC 49529</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">ATCC 43996</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">ATCC 33846</td><td colspan="1" rowspan="1">0/2</td><td colspan="1" rowspan="1">0/2</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">0/2</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">ATCC BAA-242</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">ENF 2508</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">ENF 8159</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">CCUG 67711</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">CCUG 34902</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="12">Vibriovulnificus</td><td colspan="1" rowspan="1">ATCC 27562</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">ATCC 29306</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">ENF 10442</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">ATCC 29307</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">ATCC 43382</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">ATCC BAA-86</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">CCUG 38297</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">CCUG 47321</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="12">Vibriocholerae</td><td colspan="1" rowspan="1">ATCC 14033</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">ENF 9786</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">ENF 9792</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">ATCC 9458</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">ATCC 9459</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">CCUG 2573</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">CCUG 2569</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">CCUG 4070</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">CCUG 21589</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">CCUG 56875</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">CCUG 53725</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">CCUG 14542</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">SBT</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">ATCC orother collectionnumber</td><td colspan="5" rowspan="1">Assay result at 2.99x</td><td colspan="3" rowspan="1">Non-Reportable</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Yersi</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">Vibrio</td><td colspan="1" rowspan="1">Plesio</td><td colspan="1" rowspan="1">Straindetected</td><td colspan="1" rowspan="1">IND</td><td colspan="1" rowspan="1">UNR</td><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">SBToverall</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="13">Escherichiacoli (ETEC)</td><td colspan="1" rowspan="1">ATCC 35401</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">ATCC 37218</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">ATCC 43896</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">ENF 15629</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Zeptometix801624</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">PennState 5 2215</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">PennState 911633</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">PennState 5 0038</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">PennState 6 0671</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">PennState 6 1182</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">PennState 131186</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">CCUG 47194</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">CCUG 38088</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">No1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr></table>

1 Further testing was performed as defined in the protocol

# f. Analytical specificity:

The BD MAX Extended Enteric Bacterial Panel was performed on samples containing phylogenetically related species and other organisms (bacteria, viruses, parasites and yeast) likely to be found in stool specimens. The bacterial cells, yeasts, parasites and viruses were tested in the Sample Buffer Tube at $\geq 1 0 ^ { \overline { { 6 } } }$ CFU, cells or genome equivalents/mL in stool, $\mathrm { o r } \geq 1 0 ^ { 5 } \mathrm { P F U / m L }$ in stool or $\mathrm { T C I D 5 0 / m L }$ in stool or equivalent amount of RNA/DNA/PCR reaction. Overall, 184 organisms were tested and are listed in the table below.

Most of bacterial strains, yeast, parasites and viruses tested   
produced negative results with the BD MAX Extended Enteric Bacterial Panel.   
Two (2) strains of Vibrio mimicus, associated with human   
disease, produced positive results with the BD MAX Extended Enteric Bacterial Panel. However, no positive result was recorded at $\leq 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ cells/mL in stool with those two strains.   
The following 8 Vibrio species, NOT associated with infections in humans and therefore unlikely to be encountered in human stool, were detected by the BD MAX Extended Enteric Bacterial Panel: V. brasiliensis, V. campbellii, V. harveyi, V. hispanicus, V. nereis, V. pacini, V. rotiferianus and V. tubiashii.

Based upon in silico analysis, the following Vibrio species could be detected by the BD MAX Extended Enteric Bacterial Panel:

· 2 Vibrio species, NOT associated with infections in humans and therefore unlikely to be encountered in human stool: V.

coralliilyticus and Moritella marina (formerly known as Vibrio marinus).

One (1) uncategorized Vibrio species associated with infections in humans, namely Vibrio HENC.

Organisms Tested to Determine the BD Max Extended Enteric Bacterial Panel Specificity   

<table><tr><td colspan="3" rowspan="1">Organisms</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Cryptosporidium hominis</td><td colspan="1" rowspan="1">Proteus penneri</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11 Slobitski</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 14</td><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 18</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Rotavirus</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 2</td><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">Ruminococcus bromii</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 3</td><td colspan="1" rowspan="1">Encephalitozoon cuniculi</td><td colspan="1" rowspan="1">Salmonella bongori</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 31</td><td colspan="1" rowspan="1">Encephalitozoon hellum</td><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 4</td><td colspan="1" rowspan="1">Entamoeba dispar</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">Entamoeba gingivalis</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovarenteriditis</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">Entamoeba invadens</td><td colspan="1" rowspan="1">Serratia fonticola</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 5 Adenoid 75</td><td colspan="1" rowspan="1">Entamoeba ranarum</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 8</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Serratia marcescens subsp. marcescens</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">Enterobacter cloacae subsp. cloacae</td><td colspan="1" rowspan="1">Shewanella algae</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Shigella boydii</td></tr><tr><td colspan="1" rowspan="1">Aeromonas schubertii</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">Shigella flexneri</td></tr><tr><td colspan="1" rowspan="1">Aeromonas veronii</td><td colspan="1" rowspan="1">Enterovirus type 69</td><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Shimwellia blattae</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">Staphylococcus aureus subsp. aureus</td></tr><tr><td colspan="1" rowspan="1">Arcobacter butzleri</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Arcobacter cryaerophilus</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Astrovirus Type 4</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Giardia muris</td><td colspan="1" rowspan="1">Streptococcus salivarius subsp. salivarius</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Bacteroides thetaiomicron</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td></tr><tr><td colspan="1" rowspan="1">Bacteroides vulgatus</td><td colspan="1" rowspan="1">Helicobacter fennelliae</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifidum</td><td colspan="1" rowspan="1">Hepatitis A virus</td><td colspan="1" rowspan="1">Vibrio alginolyticus</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longum subsp. longum</td><td colspan="1" rowspan="1">Herpes Simplex Virus Type 1</td><td colspan="1" rowspan="1">Vibrio brasiliensis</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">Human adenovirus 22</td><td colspan="1" rowspan="1">Vibrio campbellii</td></tr><tr><td colspan="1" rowspan="1">Bovine enterovirus Type 6</td><td colspan="1" rowspan="1">Human Adenovirus 50</td><td colspan="1" rowspan="1">Vibrio comitans</td></tr><tr><td colspan="1" rowspan="1">Bovine parvovirus</td><td colspan="1" rowspan="1">Human Coxsackievirus A 9</td><td colspan="1" rowspan="1">Vibrio diazotrophicus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Human Coxsackievirus B 1</td><td colspan="1" rowspan="1">Vibrio fluvialis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus subsp. fetus</td><td colspan="1" rowspan="1">Human Coxsackievirus B 3</td><td colspan="1" rowspan="1">Vibrio harveyi</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">Human echovirus 1</td><td colspan="1" rowspan="1">Vibrio hispanicus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp. jejuni</td><td colspan="1" rowspan="1">Human herpesvirus 2</td><td colspan="1" rowspan="1">Vibrio mimicus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">Human rhinovirus 1A</td><td colspan="1" rowspan="1">Vibrio natriegens</td></tr><tr><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Vibrio nereis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp. pneumoniae</td><td colspan="1" rowspan="1">Vibrio pacinii</td></tr><tr><td colspan="1" rowspan="1">Campylobacter sputorum biovar sputorum</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Vibrio rotiferianus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter ureolyticus</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">Vibrio sinaloensis</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">Lactococcus lactis subsp. lactis</td><td colspan="1" rowspan="1">Vibrio tubiashii</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">Yersinia bercovieri</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Yersinia frederiksenii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Megamonas hypermegale</td><td colspan="1" rowspan="1">Yersinia intermedia</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Megasphaeara elsdenii</td><td colspan="1" rowspan="1">Yersinia kristensenii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">Morganella morganii subsp. morganii</td><td colspan="1" rowspan="1">Yersinia mollaretii</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Yersinia rohdei</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">Norovirus Groupe I</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">Norovirus Groupe II</td><td colspan="1" rowspan="7"></td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Parabacteroides merdae</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">Pentatrichomonas hominis</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">Photobacterium damselae subsp. damselae</td></tr><tr><td></td><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td></tr></table>

# g. Matrix equivalence Study

N/A

# h. Interference Study

Nineteen (19) biological and chemical substances that may occasionally be present in stool specimens were evaluated for potential interference with the BD MAX Extended Enteric Bacterial Panel. Included in this study was an Antibiotics Mixture, which consisted of a combination of 8 different antibiotics tested simultaneously (each antibiotic at a concentration that may be found in a stool sample).

Nystatin cream was found to interfere at levels above $3 . 1 ~ \mathrm { m g / m L }$ . Spermicidal lubricant and hydrocortisone cream were found to interfere at levels above 2.5 $\mathrm { m g / m L }$ . Vagisil was found to interfere at levels above $0 . 9 2 \ : \mathrm { m g / m L }$ . Results demonstrated no reportable interference with any other substance tested.

In addition, microorganisms that may be endogenously present in stool specimens were evaluated for potential interference with the BD MAX Extended Enteric Bacterial Panel. Ten (10) organisms were tested at high concentration $\mathrm { ( > 1 0 ^ { 6 } C F U / m L }$ of stool). Results demonstrated no reportable interference with any microorganism tested.

Endogenous and Commercial Exogenous Substances Tested with the BD MAX Extended Enteric Bacterial Panel   

<table><tr><td rowspan=1 colspan=1>Brand Name of Description</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Brand Name of Description</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Fecal Fat</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Spermicial Lubricant</td><td rowspan=1 colspan=1>P</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Diaper Rash Cream</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Vagisil®</td><td rowspan=1 colspan=1>P</td></tr><tr><td rowspan=1 colspan=1>Whole Human Blood</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Laxatives</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone Cream</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>Anti-Diarrheal (liquid)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Antiseptic Towelettes</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Anti-Diarrheal (pill)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Enema</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antibiotics Mixture</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Hemorrhoidal Gel</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=1>Antacids</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Nystatin Cream</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>Non-Steroidal Anti-Inflamatory(NSAID)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Topical Antibiotic</td><td rowspan=1 colspan=1>NI</td><td rowspan=1 colspan=2>EY</td></tr></table>

P: Potential interference with the BD MAX Extended Enteric Bacterial Panel at high concentrations. NI: No reportable interference with the BD MAX Extended Enteric Bacterial Panel.

# Microorganisms Tested for Interference with the BD MAX Extended Enteric Bacteria Panel

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (stx1/stx2)</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Bacteroides thetaiotaomicron</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Ruminococcus bromii</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>NI</td></tr></table>

NI: No reportable interference with the BD MAX Extended Enteric Bacterial Panel.

# i. Fresh versus Frozen Study:

The objective of this study was to evaluate target stability after two (2) freeze/thaw cycles in target-spiked preserved and unpreserved specimens.

Individual Preserved and unpreserved negative stool specimens were used to generate the target-spiked positive specimens. A stool specimen placed into a modified CaryBlair medium for transport and storage is defined as “preserved” matrix; a stool specimen stored in its natural state is defined as “unpreserved” matrix.

Enteric bacterial organisms representing test panel targets (Vibrio cholera, Enterotoxigenic Escherichia coli (ETEC), Yersinia enterocolitica, and Plesiomonas shigelloides) were prepared as a stock enteric organism target mix and spiked into 2 aliquots from each negative specimen in accordance with target loads illustrated in the table below. The concentrations were chosen to test low positive and moderate positive specimens (i.e., concentrations near the LoD; 1.99X and 3X the analytical LoD, respectively) as well as a high LoD values (6X and 9X LoD).

Target Levels and Number of Replicates per Freeze/Thaw Cycle\*   

<table><tr><td rowspan=2 colspan=1>Target Level(x LOD)</td><td rowspan=1 colspan=2>Number of Replicates per Target</td></tr><tr><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Preserved</td></tr><tr><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Total # Replicates</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>96</td></tr></table>

\* Testing was performed using a conditional testing sequence algorithm. This algorithm only mandated testing the lowest concentration (1.99x LoD) of replicates.

Baseline testing was performed for each individual, target-spiked sample following preparation and prior to storage or subjected to freeze/thaw cycles. Specimens were

then stored at ${ } _ { - 8 0 } \mathrm { { } ^ { \circ } C }$ for one (1) month and subjected to two (2) freeze/thaw cycles.   
After each freeze/thaw cycle the specimens were sampled and tested.

The study was performed by four (4) operators, using three (3) lots of BD MAX xEBP reagents and eight (8) BD MAX Systems. Testing was performed using a conditional testing sequence algorithm. If $59 5 \%$ of the $1 . 9 9 \mathrm { { x } }$ LoD samples were gave the expected result, then no additional testing was performed.

All targets (Vibrio cholera, Enterotoxigenic Escherichia coli (ETEC), Yersinia enterocolitica, and Plesiomonas shigelloides) yielded $100 \%$ agreement with expected results, for all specimens, across the conditions tested. The BD MAX xEBP met the performance requirements for all targets with fresh and frozen specimens. Study results are shown in the table below for each freeze/thaw cycle time point

Summary of Specimen Freeze/Thaw Study Results   

<table><tr><td rowspan=1 colspan=3>Condition</td><td rowspan=1 colspan=4>Target Positive Matrix Proportion Positive (%) andnumber of samples tested</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Freeze/ThawCycles</td><td rowspan=1 colspan=1>Load(xLOD)</td><td rowspan=1 colspan=1>Vibriocholera</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Plesiomonasshigelloides</td></tr><tr><td rowspan=6 colspan=1>Unpreserved</td><td rowspan=4 colspan=1>0(fresh/baseline)</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=6 colspan=1>Preserved</td><td rowspan=4 colspan=1>0(fresh/baseline)</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td><td rowspan=1 colspan=1>10023/23a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td><td rowspan=1 colspan=1>10024/24</td></tr></table>

a Proportion positive rates were calculated based on total sample number after removal of IND results, which were excluded

Based on the study results, two (2) specimen freeze/thaw cycles do not affect the performance of BD MAX xEBP. The BD MAX xEBP was able to detect $100 \%$ proportion positive for all enteric bacterial targets spiked in preserved, unpreserved, negative, clinical stool specimens before and after undergoing multiple freeze/thaw cycles.

# j. Carryover / Cross-Contamination

A study was conducted to investigate within-run carryover and between-run carryover while processing specimens with high bacterial loads of Enterotoxigenic Escherichia coli (ETEC) in the BD MAX extended Enteric Bacterial Panel. Positive samples were prepared in SBT with an unpreserved stool ETEC was present at ${ \sim } 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ . The negative member did not contain any target analyte. Negative samples were prepared in SBT with negative unpreserved stool matrix.

Three different reagent lots along with three (3) lots of negative unpreserved stool matrix were used. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run by alternating negative and positive samples. Three (3) operators performed 3 consecutive runs across 3 BD MAX Instruments for a total of nine (9) runs containing 24 samples (total of 108 negative and 108 high positive samples).

Of a total of 108 ETEC positive samples and 108 ETEC negative samples, $100 \%$ and $9 9 . 1 \%$ expected assay results were obtained, respectively. One ETEC false positive result was observed among the negative samples $0 . 9 \%$ false positive).

# k. Mixed Infection Study:

A mixed infection/competitive interference study was performed to evaluate the ability of the BD MAX Extended Enteric Bacterial Panel to detect low positive results in the presence of other targets at high concentrations. Each of the targeted organisms was evaluated at low concentrations (1.99x LoD) when present in samples containing another targeted organisms at a high concentrations $( \ge \mathrm { { l x } 1 0 ^ { 6 } \ C F U / m L ) }$ . Samples were prepared with unpreserved stool matrix. A total of 24 replicates were tested for each of the sample mixes described in the table below.

Mixed infection Target Combinations   

<table><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>High Target</td><td rowspan=1 colspan=1>Low Load Target</td></tr><tr><td rowspan=3 colspan=1>#1</td><td rowspan=3 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td></tr><tr><td rowspan=1 colspan=1>ETEC</td></tr><tr><td rowspan=3 colspan=1>#2</td><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td></tr><tr><td rowspan=1 colspan=1>ETEC</td></tr><tr><td rowspan=3 colspan=1>#3</td><td rowspan=3 colspan=1>Vibriocholerae</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>ETEC</td></tr><tr><td rowspan=3 colspan=1>#4</td><td rowspan=3 colspan=1>ETEC</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td></tr></table>

In the presence of high loads of Plesiomonas shigelloides $\mathrm { ( \ge 1 \ x 1 0 ^ { 6 } C F U / m L ) }$ and Vibrio cholerae $( \geq 1 \overline { { \mathrm { ~ x ~ } } } 1 0 ^ { 6 }$ cells/mL), all 3 organisms corresponding to their respective simulated mixed infection preparations were successfully detected by the BD MAX Extended Enteric Bacterial Panel. Successful detection of all 3 low target organisms by the BD MAX Extended Enteric Bacterial Panel was achieved in the presence of Yersinia enterocolitica at $1 . 0 \mathrm { { x } 1 0 ^ { 4 } \ C F U / m L }$ and ETEC ST/LT at $9 . { \dot { 4 } } 4 \mathrm { ~ x ~ } 1 0 ^ { 2 } \mathrm { C F U / m L }$ .

# Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison:

3. Clinical Study:

Clinical performance characteristics of the BD MAX Extended Enteric Bacterial Panel were determined in a multi-site investigational study. The study involved a total of six (6) geographically diverse clinical centers where stools specimens were collected as part of routine patient care, enrolled into the trial, and tested with the BD MAX Extended Enteric Bacterial Panel. Specimens were obtained from pediatric or adult patients suspected of acute bacterial gastroenteritis, enteritis or colitis, for which stool culture had been ordered by healthcare provider. The reference method for identification of Yersinia enterocolitica, Vibrio and Plesiomonas shigelloides in fresh and frozen prospective specimens, was growth on culture media to identify potential colonies based on appearance and oxidase testing followed by a PCR assay and bidirectional sequencing on potential colonies for definitive identification. For ETEC, the comparator method was two PCR assays performed on the stool specimens followed by bi-directional sequencing of the product. For retrospective specimens, the historical results were recorded at the collection site. The historical results were confirmed using a PCR assay and bi-directional sequencing as part of the composite comparator method in order to confirm the presence of the target DNA.

A total of 2264 prospective specimens (882 unpreserved and 1382 Cary-Blair preserved) and 146 retrospective specimens (87 unpreserved and 59 Cary-Blair preserved) were enrolled in the clinical evaluation for a total of 2410 specimens enrolled. All test results from the BD MAX Extended Enteric Bacterial Panel and the comparator method were single results, no coinfections were detected. Table 4 describes the number of compliant specimens enrolled by patient age and specimen type with a total of 2403 compliant specimens overall. Table 5 through 13 describe the performance characteristics of the BD MAX Extended Enteric Bacterial Panel that were observed during the clinical trial.

Compliant Specimens Tested by Age Group and Specimen Type   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Cary-Blairpreserved</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=2 colspan=1>0-1 month1 month to 2 years2-1213-18</td><td rowspan=2 colspan=1>6250311141</td><td rowspan=1 colspan=1>0</td><td rowspan=2 colspan=1>6316475226</td></tr><tr><td rowspan=1 colspan=1>6616485</td></tr><tr><td rowspan=1 colspan=1>19-21Over 21Unknown</td><td rowspan=1 colspan=1>4467116</td><td rowspan=1 colspan=1>236215</td><td rowspan=1 colspan=1>67129221</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1439</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>2403</td></tr></table>

The following tables include the BD MAX Extended Enteric Bacterial Panel clinical study results stratified by prospective (fresh) and retrospective (frozen) unpreserved specimens as well as prospective and retrospective Cary-Blair preserved specimens. Assay performance for each targeted analyte is calculated as compared to the comparator method (CM) for prospective specimens and to the historical result (confirmed by alternate PCR and bi-directional sequencing) for retrospective specimens.

Vibrio – Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=3>SpecimenOrigin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>ReferenceMethod</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair preserved</td><td rowspan=3 colspan=3>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1351</td><td rowspan=1 colspan=1>1351</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1356</td><td rowspan=1 colspan=1>1358</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>PPA (95% CI): 100% (34.2%,,100%)NPA (95% CI): 99.6% (99.1%, 99.8%)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>Cary-Blair preserved</td><td rowspan=3 colspan=3>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>PPA (95% CI): 100% (34.2%, 100%)NPA (95% CI): 100% (80.6%, 100%)</td><td rowspan=1 colspan=1>I): 10(</td><td rowspan=1 colspan=2>PPA (95% CI): 100% (34.2%, 100%)NPA (95% CI): 100% (80.6%, 100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=3>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>866</td><td rowspan=1 colspan=1>866</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>868</td><td rowspan=1 colspan=1>868</td></tr><tr><td rowspan=1 colspan=6>No Data for PPA CalculationNPA (95% CI): 99.8% (99.2%, 99.9%)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=3>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 100% (34.2%, 100%)NPA (95% CI): 97.8% (88.7%, 99.6%)</td><td rowspan=1 colspan=2>PPA (95% CI): 100% (34.2%, 100%)</td><td rowspan=1 colspan=1>9.6%</td></tr></table>

Vibrio Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 100%(92.6%,100%)NPA: 100%(97.4%,100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (92.6%,100%)NPA: 100% (97.4%, 100%)</td></tr></table>

Plesiomonas shigelloides – Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1355</td><td rowspan=1 colspan=1>1355</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1357</td><td rowspan=1 colspan=1>1357</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>No Data for PPA CalculationNPA: 99.9% (99.5%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (51%, 100%)NPA: 100% (90.8%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>864</td></tr><tr><td rowspan=1 colspan=6>No Data for PPA CalculationNPA: 99.9% (99.3%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (43.9%, 100%)NPA: 97.9% (88.9%, 99.6%)</td></tr></table>

Plesiomonas shigelloides Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (92.6%, 100%)NPA: 100% (97.4%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>49</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>143</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (92.6%, 100%)NPA: 99.3% (96.2%, 99.9%)</td></tr></table>

1Sample XW0007C was initially found positive for Plesiomonas shigelloides, but found negative for this target once retested from the SBT (Discrepant analysis).

Yersinia enterocolitica – Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen OriginBD MAX</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1341</td><td rowspan=1 colspan=1>1341</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1342</td><td rowspan=1 colspan=1>1342</td></tr><tr><td rowspan=1 colspan=6>No Data for PPA CalculationNPA: 99.9% (99.6%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=6>No Data for PPA CalculationNPA: 100% (89.3%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=6>No Data for PPA CalculationNPA: 100% (99.6%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (70.1%, 100%)NPA: 100% (92.4%, 100%)</td></tr></table>

Yersinia enterocolitica Contrived Samples Results per Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 97.9% (89.1%, 99.6%)NPA: 100% (97.4%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>192</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (92.6%, 100%)NPA: 100% (97.4%, 100%)</td></tr></table>

2Sample XW0351C was initially found negative for Yersinia enterocolitica, but found positive for this target once retested from the SBT (Discrepant analysis).

Enterotoxigenic E. coli (ETEC LT/ST) – Overall Performance   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>ComparatorMethod</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1348</td><td rowspan=1 colspan=1>1348</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1351</td><td rowspan=1 colspan=1>1361</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (72.2%, 100%)NPA: 99.8% (99.3%, 99.9%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (56.6%,100%)NPA: 100% (87.9%, 100%</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective(Fresh+Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>818</td><td rowspan=1 colspan=1>818</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>819</td><td rowspan=1 colspan=1>835</td></tr><tr><td rowspan=1 colspan=6>PPA: 100% (80.6%, 100%)NPA: 99.9% (99.3%, 100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective(Frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=6>PPA: 90% (59.6%, 98.2%)NPA: 96.3% (81.7%, 99.3%)</td></tr></table>

ETEC Performance per Toxin Observed During the Clinical Trial   

<table><tr><td rowspan=3 colspan=2>ETEC</td><td rowspan=2 colspan=1>LT</td><td rowspan=1 colspan=3>ST</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ST</td><td rowspan=1 colspan=1>STp</td><td rowspan=1 colspan=1>STh</td><td rowspan=1 colspan=1>Unknown*</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SpecimenOrigin</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td><td rowspan=1 colspan=1>EstimatePercent(95% CI)</td></tr><tr><td rowspan=2 colspan=1>Cary-BlairPreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(5/5)(56.6, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(1/1)(20.7, 100)</td><td rowspan=1 colspan=1>100(3/3)(43.9, 100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(5/5)(56.6, 100)</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>100(4/4)(51.0, 100)</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td><td rowspan=1 colspan=1>100(3/3)(43.9, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>100(7/7)(64.6, 100)</td><td rowspan=1 colspan=1>80.0(4/5)(37.6, 96.4)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>100(2/2)(34.2, 100)</td><td rowspan=1 colspan=1>0</td></tr></table>

Co-infections Observed in the Prospective Clinical Study

No co-infections were observed during the prospective clinical study.

# Non-Reportable Rate

The initial unresolved rate was calculated when considering the unresolved rate of both BD MAX Enteric Bacterial Panel and BD MAX Extended Enteric Bacterial Panel assays. Of the 2264 prospective specimens initially evaluated, $3 . 1 \%$ of the Cary-Blair preserved and $3 . 9 \%$ of the unpreserved specimens initially reported as unresolved. The unresolved rate following a valid repeat test was calculated when considering BD MAX Extended Enteric Bacterial Panel only $0 . 1 \%$ of the prospective Cary-Blair preserved specimens and $0 . 3 \%$ of the prospective unpreserved specimens remained unresolved. Of all the specimens initially evaluated with both BD MAX Enteric Bacterial Panel and BD MAX Extended Enteric Bacterial Panel assays, $1 . 0 \%$ of the Cary-Blair preserved and $1 . 5 \%$ of the unpreserved initially reported as Indeterminate. Following a valid repeat test, $0 . 1 \%$ of the Cary-Blair preserved and none of the unpreserved remained Indeterminate. No incomplete tests were reported during this study.

# Non-reportable Rates

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=1 colspan=3>Unresolved Rate</td><td rowspan=1 colspan=2>Indeterminate Rate</td><td rowspan=1 colspan=2>Incomplete Rate</td><td rowspan=1 colspan=3>Total Rate</td></tr><tr><td rowspan=1 colspan=1>InitialxEBP(95% CI)</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalxEBP(95%CI</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>InitialxEBP(95%CI</td><td rowspan=1 colspan=1>InitialEBP+xEBP(95% CI)</td><td rowspan=1 colspan=1>FinalxEBP(95% CI)</td></tr><tr><td rowspan=2 colspan=1>Cary-BlairPreserved</td><td rowspan=2 colspan=1>2.4%35/1430(1.8%,,3.4%)</td><td rowspan=2 colspan=1>3.1%44/1430(2.3%,4.1%)</td><td rowspan=1 colspan=1>0.1%1/1427</td><td rowspan=1 colspan=1>1.0%15/1430</td><td rowspan=1 colspan=1>0.1%1/1427</td><td rowspan=1 colspan=1>0.0%0/1430</td><td rowspan=1 colspan=1>0.0%0/1427</td><td rowspan=2 colspan=1>3.5%50/1430(2.7%,4.6%)</td><td rowspan=2 colspan=1>4.1%59/1430(3.2%,5.3%)</td><td rowspan=2 colspan=1>0.1%2/1427(0.0%,0.%</td></tr><tr><td rowspan=1 colspan=1>(0.0%,0.4%)</td><td rowspan=1 colspan=1>(0.6%,1.7%)</td><td rowspan=1 colspan=1>(0.0%,0.%</td><td rowspan=1 colspan=1>(0.0%,0.3%</td><td rowspan=1 colspan=1>(0.0%,0.3%</td></tr><tr><td rowspan=2 colspan=1>Unpreserved</td><td rowspan=2 colspan=1>2.2%21/958(1.4%,3.3%</td><td rowspan=2 colspan=1>3.9%37/958(2.8%,5.3%</td><td rowspan=1 colspan=1>0.3%3/958</td><td rowspan=1 colspan=1>1.5%14/958</td><td rowspan=1 colspan=1>0.0%0/958</td><td rowspan=1 colspan=1>0.0%0/958</td><td rowspan=1 colspan=1>0.0%0/958</td><td rowspan=1 colspan=1>3.7%35/958</td><td rowspan=1 colspan=1>5.3%51/958</td><td rowspan=2 colspan=1>0.3%3/958(0.1%,0.9%)</td></tr><tr><td rowspan=1 colspan=1>(0.1%,0.9%)</td><td rowspan=1 colspan=1>(0.9%,2.4%)</td><td rowspan=1 colspan=1>(0.0%,0.4%)</td><td rowspan=1 colspan=1>(0.0%,0.4%)</td><td rowspan=1 colspan=1>(0.0%,0.4%)</td><td rowspan=1 colspan=1>(2.6%,5.0%)</td><td rowspan=1 colspan=1>(4.1%,6.9%)</td></tr><tr><td rowspan=3 colspan=1>Overall</td><td rowspan=3 colspan=1>2.3%56/2388(1.8%,3.0%)</td><td rowspan=1 colspan=1>3.4%81/2388</td><td rowspan=1 colspan=1>0.2%4/2385</td><td rowspan=1 colspan=1>1.2%29/2388</td><td rowspan=1 colspan=1>0.0%1/2385</td><td rowspan=1 colspan=1>0.0%0/2388</td><td rowspan=1 colspan=1>0.0%0/2385</td><td rowspan=1 colspan=1>3.6%85/2388</td><td rowspan=1 colspan=1>4.6%110/2388</td><td rowspan=1 colspan=1>0.2%5/2385</td></tr><tr><td rowspan=2 colspan=1>(2.7%,4.2%)</td><td rowspan=2 colspan=1>(0.1%,0.4%)</td><td rowspan=2 colspan=1>(0.8%,1.7%)</td><td rowspan=2 colspan=1>(0.0%,0.2%)</td><td rowspan=1 colspan=1>(0.0%,</td><td rowspan=2 colspan=1>(0.0%,0.2%</td><td rowspan=1 colspan=1>(2.9%,</td><td rowspan=2 colspan=1>(3.8%,5.5%)</td><td rowspan=2 colspan=1>(0.1%,0.5%)</td></tr><tr><td rowspan=1 colspan=1>0.2%)</td><td rowspan=1 colspan=1>4.4%)</td></tr></table>

4. Clinical cut-off:

N/A

N. Instrument Name: BD MAX System

# O. System Descriptions:

1. Modes of Operation:

The BD MAX System fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification: Specimens are labeled with a barcode.

4. Specimen Sampling and Handling:

A disposable inoculating loop is used to place $1 0 ~ \mu \mu 1$ of the unpreserved or Cary-Blair stool specimen into a SBT which is then vortexed and placed onto the system.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control: See Section M.1c above.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.