# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# C. Measurand:

Target RNA sequences from norovirus genogroup I and genogroup II

# D. Type of Test:

An in vitro molecular diagnostic test for the qualitative detection and differentiation of norovirus genogroup I and genogroup II RNA from stool specimens

# E. Applicant:

F. Proprietary and Established Names: RIDA GENE Norovirus GI/GII

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990

2. Classification: Class II

3. Product code: PIQ, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The RIDA GENE Norovirus GI/GII assay, performed on the Applied Biosystems 7500 Fast Dx System, is a real-time RT-PCR in vitro diagnostic test for the qualitative detection and differentiation of norovirus genogroup I (GI) and II (GII) RNA from raw or unpreserved stool specimens collected from individuals with signs and symptoms of

acute gastroenteritis.

The RIDA GENE Norovirus GI/GII assay is intended for use as an aid in the differential diagnosis of norovirus genogroup I and II infections in patients symptomatic for gastroenteritis in conjunction with clinical evaluation, laboratory findings, and epidemiological information. The assay aids in the detection and identification of norovirus infections as the cause of acute gastroenteritis in sporadic cases as well as in the context of outbreaks.

Negative results do not preclude a norovirus infection and should not be used as the sole basis for diagnosis.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

· Applied Biosystems 7500 Fast Dx software versions (1.4 or 1.4.1) bioMérieux NucliSENS easyMAG software versions (2.0 or 2.1)

# I. Device Description:

The RIDA GENE Norovirus GI/GII is a real-time RT-PCR in vitro diagnostic test for the qualitative detection and differentiation of norovirus genogroup I (GI) and II (GII) RNA in human stool specimens. Sample preparation/RNA extraction and amplification/real-time detection are performed on two separate instruments.

An internal control RNA (ICR) is added to each sample prior to extraction and RNA extraction is performed on the bioMérieux NucliSENS easyMAG instrument with bioMérieux NucliSENS Nucleic Acid Extraction Reagents.

The amplification/real-time detection is performed on an Applied Biosystems 7500 Fast Dx System. Reverse transcription and real-time PCR are completed in the same reaction plate well under optimized conditions. The detection is carried out in a one-step real-time RTPCR format.

The RIDA GENE Norovirus GI/GII assay kit contains sufficient reagents for 100 reactions.   
The kit is stored at ${ } - 2 0 { } ^ { \circ } \mathrm { C }$ .

# J. Substantial Equivalence Information:

1. Predicate device name(s): Xpert Norovirus

2. Predicate 510(k) number(s):

K142501

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td></td><td>Device</td><td>Predicate</td></tr><tr><td>Item Intended Use</td><td>RIDA GENE Norovirus GI/GII (K171511) The RIDA GENE Norovirus GI/GII assay, performed on the Applied</td><td>Xpert Norovirus (K142501) The Cepheid Xpert Norovirus Assay, performed on the GeneXpert</td></tr><tr><td></td><td>Biosystems 7500 Fast Dx System, is a real-time RT-PCR in vitro diagnostic test for the qualitative detection and differentiation of norovirus genogroup I (GI) and II (GII) RNA from raw or unpreserved stool specimens collected from individuals with signs and symptoms of acute gastroenteritis. This RIDA GENE Norovirus GI/GII assay is intended for use as an aid in the differential diagnosis of norovirus genogroup I and II infections in patients symptomatic for gastroenteritis in conjunction with clinical evaluation, laboratory findings, and epidemiological information. The assay aids in the detection and identification of norovirus infections as the cause of acute gastroenteritis in sporadic cases as well as in the context of outbreaks. Negative results do not preclude a norovirus infection and should not be</td><td>Instrument Systems, is a qualitative in vitro diagnostic test for the identification and differentiation of norovirus genogroup I and genogroup II RNA from raw or unpreserved unformed stool specimens collected from individuals with symptoms of acute gastroenteritis. The test utilizes automated real-time reverse transcriptase polymerase chain reaction (RT-PCR) to detect norovirus RNA. The Xpert Norovirus Assay is intended to aid in the diagnosis of norovirus infections when used in conjunction with clinical evaluation, laboratory findings, and epidemiological information. The assay also aids in the detection and identification of norovirus infections in the context of outbreaks Same</td></tr><tr><td>Specimen</td><td>used as the sole basis for diagnosis. Human stool</td><td></td></tr><tr><td>Types Test Principle</td><td>Real-time reverse transcriptase polymerase chain reaction (RT-PCR)</td><td>Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">RIDA GENE Norovirus GI/GII (K171511</td><td colspan="1" rowspan="1">Xpert Norovirus (K142501)</td></tr><tr><td colspan="1" rowspan="1">Assay Control(s)</td><td colspan="1" rowspan="1">Bacteriophage MS2 as Internal ControlRNA (ICR) in each sample. Positive control(PC) and negative controls (NC) processedwith each batch of samples.</td><td colspan="1" rowspan="1">Sample processing control (SPC) andprobe check control (PCC) integrated inassay/instrument system. Externalcontrols available but not provided</td></tr><tr><td colspan="1" rowspan="1">Extraction</td><td colspan="1" rowspan="1">bioMérieux NucliSENS easyMAG NucleicAcid Extraction System</td><td colspan="1" rowspan="1">Self-contained and automated in theGeneXpert Cartridge and GeneXpertInstrument Systems. No reagentpreparation - all reagents are contained inthe cartridge.</td></tr><tr><td colspan="1" rowspan="1">InstrumentSystem</td><td colspan="1" rowspan="1">Applied Biosystems 7500 FAST Dx System</td><td colspan="1" rowspan="1">Cepheid GeneXpert Instrument Systems(GeneXpert Dx, GeneXpert Infinity-48and GeneXpert Infinity-80)</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not Applicable

# L. Test Principle:

The RIDA GENE Norovirus GI/GII assay is a real-time RT-PCR in vitro diagnostic test for the simultaneous qualitative detection and differentiation of norovirus GI and GII RNA in human stool specimens. The assay also detects an internal control RNA (ICR, bacteriophage MS2) that is added to each sample prior to extraction. Nucleic acid extraction and amplification/real-time detection are completed on separate instruments. Following nucleic acid extraction with the NucliSENS easyMag extraction system from bioMérieux, the assay is performed on an Applied Biosystems 7500 Fast Dx System.

The isolated RNA is transcribed into cDNA by a reverse transcriptase. Gene fragments specific for norovirus GI and GII are subsequently amplified by real-time PCR. The amplified targets are detected with hydrolysis (TaqMan) probes, which are labeled at one end with a quencher and at the other end with a fluorescent reporter dye (fluorophore). In the presence of a target, the probe(s) hybridize to it and, during the extension step, the Taqpolymerase breaks the reporter-quencher proximity. Upon excitation by the Applied Biosystems 7500 FAST Dx’s halogen light source, the reporter emits a distinct fluorescent signal which is detected by the optical unit of the Applied Biosystems 7500 FAST Dx System. Hence, depending on the target sequence present (genogroup GI, GII or both), one, two or none of the reporters on the norovirus specific probes emits light to be detected by the instrument. Fluorophores are chosen in a way that their excitation and emission wavelengths do not overlap and signals are readily discriminated by the software. The fluorescence signal increases with the amount of formed amplicons.

RIDA GENE Norovirus GI/GII Probe Labels   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Dye</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>Cy5</td></tr><tr><td rowspan=1 colspan=1>Internal Control RNA (ICR)</td><td rowspan=1 colspan=1>VIC</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The reproducibility of the RIDA GENE Norovirus GI/GII assay was evaluated at 3 sites. A panel of 5 samples with varying concentrations of norovirus GI and norovirus GII that included negative, moderate positive and low positive samples was tested in triplicates two times on each of five different days by two operators at each of the three sites (5 days x 2 times/day x 3 replicates x 3 sites). The same kit lot of RIDA GENE Norovirus GI/GII was used at each of the 3 testing sites. Results are presented in the table below.

Reproducibility of the RIDA GENE Norovirus GI/GII   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Panel Member ID</td><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean (Ct)</td><td rowspan=1 colspan=1>SD (Ct)*</td><td rowspan=1 colspan=1>CV (%)**</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GI low positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GII low positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GI low positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GII low positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.5%</td></tr></table>

\*SD: Standard deviation, CV: Coefficient of variation \*\*For negative samples no coefficient of variation is calculated

The precision of the RIDA GENE Norovirus GI/GII Assay was evaluated internally using a panel of 5 samples with varying concentrations of Norovirus GI and Norovirus GII that included negative, moderate positive and low positive samples. The samples were tested in triplicates over 12 days with two runs per day by two operators at one study site (12 days $_ { \textrm { X 2 } }$ times/day x 3 replicates). Three independent kit lots were used for the precision study. The Inter-lot precision was calculated over those three kit lots. Results are presented in the tables below.

Intra-Site Precision of the RIDA GENE Norovirus GI/GII   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=2 colspan=1>Channe1</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=3>Intra-Site kit lot 1</td><td rowspan=1 colspan=3>Intra-Site kit lot 2</td><td rowspan=1 colspan=3>Intra-Site kit lot 3</td></tr><tr><td rowspan=1 colspan=1>Mean(Ct)</td><td rowspan=1 colspan=1>SD(Ct)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>Mean(Ct)</td><td rowspan=1 colspan=1>SD(Ct)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>Mean(Ct)</td><td rowspan=1 colspan=1>SD(Ct)</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GIlow positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GIIlow positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>24.5*</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>24.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GI lowpositive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GII lowpoositive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>31.0*</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>23.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>23.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.6%</td></tr></table>

SD: Standard deviation, CV: Coefficient of variation For negative samples no coefficient of variation is calculated $^ { * 1 }$ replicate was undetermined because of a pipetting error; replicate was retested

# Inter-Lot Precision of the RIDA GENE Norovirus GI/GII

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=3>Inter-Lot</td></tr><tr><td rowspan=1 colspan=1>Mean (Ct)</td><td rowspan=1 colspan=1>SD (Ct)*</td><td rowspan=1 colspan=1>CV (%)**</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GI low positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GII low positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>24.7#</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Norovirus GI low positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Norovirus GII low positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>31.2#</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.5%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Norovirus GImoderate positive</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Norovirus GIImoderate positive</td><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>23.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.4%</td></tr></table>

$^ { * } \mathrm { S D }$ : Standard deviation, $\ast \ast C \mathrm { V }$ : Coefficient of variation $\boldsymbol { : }$ For negative samples no coefficient of variation is calculated. $^ { \# } 1$ replicate is undetermined because of a pipetting error; replicate was retested

b. Linearity/assay reportable range:

Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The RIDA GENE Norovirus GI/GII assay incorporates an Internal Control RNA (ICR; bacteriophage MS2), a positive control (non-infectious in vitro transcribed RNA containing a portion of the targeted norovirus GI and GII genes) and a negative control (Tris buffer) to monitor reagent and assay performance. The positive and negative controls should be included in every PCR run. The Internal Control RNA (ICR) should be used during extraction and amplification in the assay.

Control types used to monitor:

Positive control: Substantial reagent failure including primer and probe integrity · Negative control: Reagent and/or environmental contamination ICR: Inhibitors in the extracted specimen; assures adequate amplification

# d. Detection limit:

The limit of detection (LoD) was performed to evaluate the analytical sensitivity of the RIDA GENE Norovirus GI/GII assay using a dilution series of two native fecal samples, one genogroup I and one genogroup II, diluted into a negative stool matrix.

The genogroup of the two native samples was determined by conventional RT-PCR followed by bi-directional sequencing. The norovirus RNA copy numbers in the dilution series of fecal samples were determined by using two standard curves containing either GI or GII transcripts quantified by real-time RT-PCR. Measurements for the standard curve were performed in duplicate. The dilution series of samples were tested by the RIDA GENE Norovirus GI/GII assay applying the quantified standard curve described above.

The preliminary limit of detection of the RIDA GENE Norovirus GI/GII was established based on the RNA copy number that gave a minimum of 2 out of 3 positive test results. The LoD concentration determined from the preliminary study was further verified by testing 20 replicates at the the LoD concentration. The LoD was confirmed if all of those replicates yielded positive results.

The LoDs of the RIDA GENE Norovirus GI/GII for the two genogroups are presented in the table below.

Limit of detection of the RIDA GENE Norovirus GI/GII   

<table><tr><td rowspan=1 colspan=1>Norovirus Genogroup</td><td rowspan=1 colspan=1>Limit of detection</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>6.5x 105copies/g stool</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>2.5x 105copies/g stool</td></tr></table>

# e. Analytical Reactivity (Inclusivity):

The analytical reactivity of the RIDA GENE Norovirus GI/GII was evaluated using twenty four genotypes representing both norovirus genogroups (GI and GII). The studies were carried out with positive clinical stool specimens representing norovirus GI and GII genotypes at low (near LoD) and at high concentration. Negative pooled clinical stool matrix was used for preparing the dilutions of norovirus clinical stool samples. Positive clinical stool specimens were tested for eight (8) norovirus GI genotypes and sixteen (16) norovirus GII genotypes including the recently circulating Norovirus GII.4 Sydney strain and the previously dominant Norovirus GII.4 New Orleans strain with the RIDA GENE Norovirus GI/GII assay. The results of the analytical reactivity (inclusivity) study demonstrated that the RIDA GENE Norovirus GI/GII assay appropriately identified all the temporally and geographically diverse samples selected to represent the range of norovirus genotypes. The table below shows the data for the analytical reactivity studies.

Analytical Reactivity of the RIDA GENE Norovirus GI/GII   

<table><tr><td colspan="1" rowspan="2">Subgroup</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Norovirus</td></tr><tr><td colspan="1" rowspan="1">GI</td><td colspan="1" rowspan="1">GII</td></tr><tr><td colspan="1" rowspan="1">GI.1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.2</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.3</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.4</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.5</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.6</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.7</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GI.8</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">GII.1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.4 Sydney</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.4 New Orleans</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.6</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.7</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.8</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.9</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.12</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.13</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.14</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.15</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.16</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">GII.17</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr></table>

# f. Analytical specificity (Cross-reactivity):

The analytical specificity of the RIDA GENE Norovirus GI/GII Assay was evaluated by testing a panel of 69 organisms, consisting of 56 bacteria, 1 fungus, 8 viruses, and 4 parasites representing common gastroenteritis pathogens or those potentially encountered in stool. The analytical specificity study included testing of bacterial cultures at ${ 1 0 } ^ { 6 }$ to ${ 1 0 } ^ { 9 } \mathrm { c f u / m l } .$ , parasite and fungi cultures at $1 0 ^ { 7 }$ to ${ 1 0 } ^ { 9 }$ cfu/ml, and viral cell culture supernatants at $1 0 ^ { 5 }$ to $1 0 ^ { 9 } \mathrm { { p f u } / \mathrm { { m l } } }$ in the stool specimens.

The samples were extracted using the NucliSENS easyMAG System and tested on the Applied Biosystems 7500 Fast Dx platform. The analytical specificity of the RIDA GENE Norovirus GI/GII Assay was $100 \%$ . Results are shown in the table below.

Analytical Specificity of the RIDA GENE Norovirus GI/GII   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">NorovirusGI</td><td colspan="1" rowspan="1">NorovirusGII</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">NorovirusGI</td><td colspan="1" rowspan="1">NorovirusGII</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffi</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 40</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 41</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pleisomonas shigelloides</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Astrovirus type 1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Astrovirus type 4</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rotavirus G1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Rotavirus G2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rotavirus G3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rotavirus G4</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rotavirus G9</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Salmonella agona</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Salmonella enteritidis</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Sapovirus GI.1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Sapovirus GIV</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(0157:H7; EHEC)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Sapovirus GV</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(0157; vtx1, vtx2, eae)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (O26:H-; EPEC)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(O26:H11; vtx2, eae)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (08;vtx1)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O103</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Steptococcus agalactiae</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O111</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Steptococcus dysgalactiae</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O121</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O145</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O45</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

# g. Interference:

The performance of the RIDA GENE Norovirus GI/GII assay was evaluated with potentially interfering substances that may be present in stool. The substances were added to 9 fecal samples containing the analyte: 3 low positive GI, 3 low positive GII and 3 negative. Low positive stool specimens contained norovirus GI and GII at a concentration of $2 { - } 3 \mathbf { x }$ LoD. For each sample a common batch was prepared and subsequently aliquoted into 12 portions (11 substances and 1 reference measurement). An interfering substance was added to each sample and tested in triplicates. As a reference one aliquot of each sample was measured without addition of a substance.

None of the potentially interfering substances interfered with detection of norovirus concentrations near the LoD of the RIDA GENE Norovirus GI/GII.

Potentially Interfering Substances in the RIDA GENE Norovirus GI/GII   

<table><tr><td colspan="1" rowspan="1">Substance name</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">Acetaminophen (analgesic)</td><td colspan="1" rowspan="1">6.0 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Amoxicillin (antibiotic)</td><td colspan="1" rowspan="1">4.5 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Aspartame (artificial sweetener)</td><td colspan="1" rowspan="1">4.8 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate (radiocontrast agent</td><td colspan="1" rowspan="1">18.5 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Human blood (maybe found in patient stool)</td><td colspan="1" rowspan="1">5.0 % (v/w)</td></tr><tr><td colspan="1" rowspan="1">Ibuprofen (analgesic)</td><td colspan="1" rowspan="1">3.6 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Loperamide (anti-diarrhea drug)</td><td colspan="1" rowspan="1">0.02 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Metronidazole (antibiotic)</td><td colspan="1" rowspan="1">3.0 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Mucin (mucilage)</td><td colspan="1" rowspan="1">5.0 % (w/w)</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol (anti-diarrheal drug)</td><td colspan="1" rowspan="1">6.3 % (v/w)</td></tr><tr><td colspan="1" rowspan="1">Stearic acid / Palmitic acid (1:1) (fatty acids)</td><td colspan="1" rowspan="1">40.0 % (w/w)</td></tr></table>

# h. Carry-Over/Cross-Contamination:

An internal carry-over/cross-contamination study was performed with the RIDA GENE Norovirus GI/GII assay in association with the NucliSENS easyMAGSystem and the Applied Biosystems 7500 Fast Dx platform. In this study, one high positive norovirus GII sample was measured 47 times alternating with a negative sample measured 47 times on the same plate. This setup was repeated for a total of 5 runs (negative adjacent positive). The results showed no carry-over/cross-contamination.

2. Comparison studies:

a. Method comparison with predicate device:

The clinical performance evaluation was performed against a composite reference test method.

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable

b. Clinical specificity: Not Applicable

c. Other clinical supportive data (when a. and b. are not applicable):

Clinical Performance

Performance characteristics of the RIDA GENE Norovirus GI/GII assay were established during a prospective, multi-center study conducted at four institutions in the U.S. from February 2014 to April 2015. The study specimens consisted of raw or unpreserved unformed stool specimens from subjects with symptoms of acute gastroenteritis. A total of 769 specimens were collected at the four different study sites. Of the 769 samples, 50 samples could not be used for study analysis. An additional 332 samples were prospectively collected during various outbreaks at two institutions and were tested (September 2016 to April 2017); of the 332 samples, 300

samples provided valid results.

A total of 1019 specimens provided valid results with the norovirus GI and GII by the RIDA GENE Norovirus GI/GII assay. The RIDA GENE Norovirus GI/GII performance was compared to a composite reference method performed at the Centers for Disease Control and Prevention (CDC; Atlanta, GA, US). The composite reference method consisted of conventional RT-PCR followed by bi-directional sequencing for two different target regions of norovirus.

The RIDA GENE Norovirus GI/GII demonstrated $9 1 . 8 \%$ PPA and $9 9 . 1 \%$ NPA for detection of norovirus GI, relative to the composite reference method. The RIDA GENE Norovirus GI/GII assay demonstrated $9 4 . 7 \%$ PPA and $98 \%$ NPA for detection of norovirus GII. Results are presented in the tables below.

Norovirus Genogroup I vs. Composite Reference Method   

<table><tr><td rowspan=2 colspan=1>RIDA GENE Norovirus GI/GII</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>949</td><td rowspan=1 colspan=1>954</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>958</td><td rowspan=1 colspan=1>1019</td></tr></table>

<table><tr><td>Percent Positive Agreement; (95 % CI)</td><td>91.8 % (56/61); (81.9 % - 97.3 %)</td></tr><tr><td>Percent Negative Agreement; (95 % CI)</td><td>99.1 % (949/958); (98.2% - 99.6 %)</td></tr></table>

Norovirus Genogroup II vs. Composite Reference Method   

<table><tr><td rowspan=2 colspan=1>RIDA GENE Norovirus GI/GII</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>229</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>778</td><td rowspan=1 colspan=1>790</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>794</td><td rowspan=1 colspan=1>1019</td></tr></table>

<table><tr><td>Percent Positive Agreement; (95 % CI)</td><td>94.7 % (213/225); (90.9 % - 97.2 %)</td></tr><tr><td>Percent Negative Agreement; (95 % CI)</td><td>98.0 % (778/794); (96.7 % - 98.8 %)</td></tr></table>

# 4. Expected values/Reference range:

The prevalence of norovirus GI and norovirus GII with the RIDA GENE Norovirus GI/GII was calculated for 719 prospectively collected fresh, raw or unpreserved stool specimens from a multicenter study. The overall number and percentage of positive prospective norovirus GI and norovirus GII as determined by the RIDA GENE Norovirus GI/GII assay was $4 . 2 \%$ (30/719) and $3 1 . 3 \%$ (225/719), respectively, as presented in the following table.

Overall Prevalence of GI and GII   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>RIDA GENE Norovirus</td></tr><tr><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>4.2 %b (30/719)</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>31.3 % (225/719)</td></tr></table>

The number and percentage of Norovirus GI and Norovirus GII cases, as defined by the RIDA GENE Norovirus GI/GII assay, calculated by age group and gender are presented respectively in the following tables.

Observed Prevalence of GI and GII by Age Group   

<table><tr><td rowspan=1 colspan=1>Age(Years)</td><td rowspan=1 colspan=1>No. of GIPositives</td><td rowspan=1 colspan=1>GIObservedPrevalence%</td><td rowspan=1 colspan=1>No. of GIIPositives</td><td rowspan=1 colspan=1>GIIObservedPrevalence%</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>13/191</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>36/191</td><td rowspan=1 colspan=1>18.8</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>7/88</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>31/88</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>22 - 59*</td><td rowspan=1 colspan=1>7/130</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>45/130</td><td rowspan=1 colspan=1>34.6</td></tr><tr><td rowspan=1 colspan=1>≥60#</td><td rowspan=1 colspan=1>3/310</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>113/310</td><td rowspan=1 colspan=1>36.5</td></tr></table>

$^ { * 1 }$ sample is GI and GII positive $^ { \# } 3$ samples are GI and GII positive

Observed Prevalence of GI and GII by Gender   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>No. of GIPositives</td><td rowspan=1 colspan=1>GIObservedPrevalence%</td><td rowspan=1 colspan=1>No. of GIIPositives</td><td rowspan=1 colspan=1>GIIObservedPrevalence%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>14/272</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>136/272</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>16/447</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>89/447</td><td rowspan=1 colspan=1>19.9</td></tr></table>

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.