# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K171974

# B. Purpose for Submission:

New device; to obtain market clearance for the Solana RSV+hMPV Assay when performed on the Solana instrument.

# C. Measurand:

RSV RNA: Matrix Gene hMPV RNA: Fusion Protein Gene

# D. Type of Test:

Reverse Transcriptase - Helicase-Dependent Amplification (RT-HDA)

# E. Applicant:

Quidel Corporation 2005 East State Street, Suite 100 Athens, OH 45701 USA

F. Proprietary and Established Names: Solana RSV+hMPV Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980 Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code(s):

Primary: OCC Respiratory Viral Panel Multiplex Nucleic Acid Assay Secondary: OEM Human Metapneumovirus (hMPV) RNA Assay System

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Solana RSV+hMPV Assay is a qualitative in vitro diagnostic test for the detection and differentiation of RSV and hMPV viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of RSV and hMPV viral infections in humans in conjunction with clinical and epidemiological risk factors. This test is not intended to differentiate the two subtypes of RSV or the four genetic sub-lineages of hMPV.

Negative results do not preclude RSV infection and/or hMPV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease. The use of additional laboratory testing and clinical presentation must be considered in order to obtain the final diagnosis of respiratory viral infection.

2. Indication(s) for use: As a diagnostic aid in patients with signs and symptoms of respiratory infection.

3. Special conditions for use statement(s):

For in vitro diagnostic use only For prescription use only

4. Special instrument requirements:

Solana Instrument

# I. Device Description:

An aliquot $( 5 0 \mu \mathrm { L } )$ of patient nasal or nasopharyngeal swab specimen collected in viral transport media is transferred to a Process Buffer Tube. One Process Buffer Tube, which contains a competitive process control (PRC), is required for each specimen or control sample to be tested. The tubes are vortexed and heated in a heat block at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. After a $2 ^ { \mathrm { n d } }$ vortexing step, $5 0 ~ \mu \mathrm { L }$ of the heated sample is transferred to a Reaction Tube which rehydrates its contents. The Reaction Tubes are then placed in the Solana instrument for amplification and detection. Upon annealing to RSV, hMPV or PRC amplicons, the fluorescence probes are cleaved by RNaseH2 and the fluorescence signal increases due to physical separation of fluorophore from quencher. The Solana software automatically interprets the fluorescent signal and the test results are displayed on the Solana result screen in approximately 40 minutes. The following RSV and hMPV combinations of results may be displayed:

Table 1. Interpretation of Results   

<table><tr><td rowspan=1 colspan=1>Assay Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>RSV POSITIVEhMPV NEGATIVE</td><td rowspan=1 colspan=1>RSV RNA detected</td></tr><tr><td rowspan=1 colspan=1>RSV NEGATIVEhMPV POSITIVE</td><td rowspan=1 colspan=1>hMPV RNA detected</td></tr><tr><td rowspan=1 colspan=1>RSV POSITIVEhMPV POSITIVE</td><td rowspan=1 colspan=1>RSV RNA detected andhMPV RNA detected</td></tr><tr><td rowspan=1 colspan=1>RSV NEGATIVEhMPV NEGATIVE</td><td rowspan=1 colspan=1>No RSV RNA detected/PRC detected andNo hMPV RNA detected/PRC detected</td></tr><tr><td rowspan=1 colspan=1>RSV INVALIDhMPV INVALID</td><td rowspan=1 colspan=1>No RSV or hMPV RNA and No PRC detected; forinvalid test results, re-process another aliquot of the samesample or obtain a new sample and re-test.</td></tr></table>

The results may also be printed out via an attached printer.

The assay kit contains 48 Process Buffer Tubes and 48 Reaction Tubes. A maximum of 12 tests may be performed in one run.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra RSV+hMPV Assay   
2. Predicate 510(k) number(s): K131813   
3. Comparison with predicate:

Table 2. Comparison with the predicate   

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Solana RSV+hMPV Assay</td><td colspan="1" rowspan="1">Lyra RSV+hMPV Assay(k131813)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Solana RSV+hMPV Assay isa qualitative in vitro diagnostictest for the detection anddifferentiation of RSV and hMPVviral RNA in nasal andnasopharyngeal swabs frompatients with signs and symptomsof respiratory infection. This testis intended for use as an aid in thedifferential diagnosis of RSV andhMPV viral infections in humansin conjunction with clinical andepidemiological risk factors. Thistest is not intended todifferentiate the two subtypes ofRSV or the four genetic sub-lineages of hMPV.Negative results do not precludeRSV infection and/or hMPVinfection and should not be usedas the sole basis for diagnosis,treatment or other patientmanagement decisions.Conversely, positive results donot rule-out bacterial infection orco-infection with other viruses.The agent detected may not bethe definite cause of disease. Theuse of additional laboratorytesting and clinical presentationmust be considered in order toobtain the final diagnosis ofrespiratory viral infection.</td><td colspan="1" rowspan="1">The Lyra RSV + hMPV Assay is amultiplex Real-Time PCR (RT-PCR) assay for the qualitativedetection and identification ofrespiratory syncytial virus (RSV)and human metapneumovirus(hMPV) ribonucleic acid (RNA)extracted from nasal andnasopharyngeal swab specimensfrom patients with signs andsymptoms of respiratory infection.This in vitro diagnostic test isintended to aid in the differentialdiagnosis of RSV and hMPVinfections in humans in conjunctionwith clinical and epidemiologicalrisk factors. This test is notintended to differentiate the twosubtypes of RSV or the four geneticsub-lineages of hMPV.Negative results do not precludeRSV infection and/or hMPVinfection and should not be used asthe sole basis for diagnosis,treatment or other patientmanagement decisions.Conversely, positive results do notrule-out bacterial infection or co-infection with other viruses. Theagent detected may not be thedefinite cause of disease. The use ofadditional laboratory testing andclinical presentation must beconsidered in order to obtain thefinal diagnosis of respiratory viralinfection.The Quidel Molecular RSV + hMPVAssay can be performed using eitherthe Life Technologies QuantStudioDx RT-PCR Instrument, the AppliedBiosystems 7500 Fast Dx RT-PCR</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Instrument, or the CepheidSmartCycler II System.</td></tr><tr><td colspan="1" rowspan="1">SampleTypes</td><td colspan="1" rowspan="1">Nasal and nasopharyngeal swabscollected in transport media.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnology</td><td colspan="1" rowspan="1">Automated multiplex assay usingfluorescent signal.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Differences</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Viral Targets</td><td colspan="1" rowspan="1">RSV: Matrix gene;hMPV: Fusion protein gene</td><td colspan="1" rowspan="1">RSV: L viral polymerase and NS2geneshMPV: RNA polymerase gene</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Reverse Transcriptase - Helicase-Dependent Amplification (RT-HDA)</td><td colspan="1" rowspan="1">Real Time RT-PCR-based system</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethod</td><td colspan="1" rowspan="1">No extraction</td><td colspan="1" rowspan="1">bioMérieux easyMAG AutomatedMagnetic Extraction Reagents</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Solana</td><td colspan="1" rowspan="1">Life Technologies QuantStudio Dx,the Applied Biosystems 7500 FastDx, or the Cepheid SmartCycler II</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document:   
Respiratory Viral Panel Multiplex Nucleic Acid Assay -   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc   
m180307.htm

Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays – https://www.fda.gov/RegulatoryInformation/Guidances/ucm180308.htm

Guidance for Industry and FDA Staff: Statistical Guidance on Reporting Results from   
Studies Evaluating Diagnostic Tests (Final, 3/13/2007)   
http://www.fda.gov/downloads/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm071287.pdf Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) –   
http://www.fda.gov/cdrh/oivd/guidance/1588.pdf.

Guidance for Industry and Food and Drug Administration Staff - eCopy Program for Medical Device (December 2012)

# L. Test Principle:

The assay consists of two major steps: 1) specimen preparation, and 2) amplification and detection of target sequences specific to RSV and/or hMPV using isothermal Reverse Transcriptase - Helicase-Dependent Amplification (RT-HDA) in the presence of targetspecific fluorescence probes. The cells in a patient nasal or nasopharyngeal swab specimen are lysed during a heating step releasing viral nucleic acids when present. An aliquot of the processed sample is then added to the lyophilized RT-HDA reagents, dNTPs, primers and probes contained in the Reaction Tube. The Reaction Tube is then placed in the Solana instrument for amplification and detection of RSV and hMPV specific target sequences. In the Solana instrument, the target sequences are amplified by RSV and hMPV specific primers and detected by RSV and hMPV specific fluorescence probes, respectively. A competitive process control (PRC) is included in the Process Buffer Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure, or instrument failure. The PRC target is amplified by RSV and hMPV specific primers and detected by a PRC specific fluorescence probe. The two target probes and the PRC probe are labeled with a quencher on one end and a fluorophore on the other end. In addition, the two target probes and the PRC probe incorporate one or more RNA bases. Upon annealing to RSV, hMPV or PRC amplicons, the fluorescence probes are cleaved by RNaseH2 and the fluorescence signal increases due to physical separation of fluorophore from quencher. Solana measures and interprets the fluorescent signal, using on-board method-specific algorithms. The results are displayed on the instrument screen and can be printed out via an attached printer.

# M. Performance Characteristics (if/when applicable):

# 1. Analytical Performance:

The analytical studies were conducted in pooled negative clinical matrix consisting of nasopharyngeal samples collected by medical personnel from volunteer donors during the winter of 2017. The samples were collected with flocked swabs, placed into viral transport media and screened with the Solana RSV+hMPV Assay. Only the negative samples were pooled to generate clinical matrix for use in the analytical studies. The pooled negative matrix was stored at ${ - 2 0 ^ { \circ } \mathrm { C } }$ or at $\le - 7 0 ^ { \circ } \mathrm { C }$ for long period storage.

# a. Precision/Reproducibility:

Precision

The precision of the Solana RSV+hMPV assay was evaluated in an internal study testing a panel of contrived samples over 12 days (non-consecutive). Testing was performed by two operators and each sample was tested in 3 replicates.

The test samples were prepared by spiking two strains of RSV (RSV A and RSV B) and two strains of hMPV (hMPV 20 Type A2 and hMPV 4 Type B2) into contrived nasal matrix to targeted concentrations for low positive (at LoD, or $\mathrm { C } _ { 9 5 }$ ), a moderate positive (at $2 \mathbf { x }$ LoD, or $\mathrm { C } _ { 1 0 0 }$ ), and a high negative (at $0 . 2 \mathbf { x }$ LoD, or $\mathrm { C } _ { 2 0 } { - } \mathrm { C } _ { 8 0 } ,$ ) for each virus. Negative nasal matrix was used as a negative sample, for a total of four test samples.

Table 3. Precision Test Panel   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Set 1/0perator 1</td></tr><tr><td rowspan=1 colspan=1>Sample 1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1.6 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 2</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>7.9 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 3</td><td rowspan=1 colspan=1>High Negative RSV/hMPV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1.6 x 103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>2.4 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Set 2/0perator 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>2.4 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>4.6 x 10^3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 2</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>2.3 x 10^³3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample 3</td><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>2.4 x 102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=2>4.6 x 102</td></tr><tr><td rowspan=1 colspan=1>Sample 4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=2>N/A</td></tr></table>

Table 4. Solana RSV+hMPV Assay Precision/Detection Rate   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Low Pos.</td><td rowspan=1 colspan=1>Mod. Pos.</td><td rowspan=1 colspan=1>High Neg.</td><td rowspan=1 colspan=1>Neg.</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>18/36</td><td rowspan=1 colspan=1>0/36</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>24/36</td><td rowspan=1 colspan=1>0/36</td></tr><tr><td rowspan=1 colspan=1>Total RSV</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>58.3% (42/72)</td><td rowspan=1 colspan=1>100% (0/72)</td></tr><tr><td rowspan=1 colspan=1>hMPV A</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>11/36</td><td rowspan=1 colspan=1>0/36</td></tr><tr><td rowspan=1 colspan=1>hMPV B</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>16/36</td><td rowspan=1 colspan=1>0/36</td></tr><tr><td rowspan=1 colspan=1>Total hMPV</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>37.5% (27/72)</td><td rowspan=1 colspan=1>100% (0/72)</td></tr></table>

# Reproducibility

Reproducibility of the Solana RSV+hMPV assay was evaluated in a study testing a panel of contrived precision samples (as described above) across three testing sites over 5 days (non-consecutive). Testing was performed by two operators and each sample was tested in 3 replicates. Positive and negative controls were run in triplicate along with the panels.

Table 5. Solana $\mathbf { R S V + h M P V }$ Assay Reproducibility Summary   

<table><tr><td rowspan=1 colspan=8>RSV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Low Pos.</td><td rowspan=1 colspan=2>Mod. Pos.</td><td rowspan=1 colspan=2>High Neg.</td><td rowspan=1 colspan=1>Neg.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>7/15</td><td rowspan=1 colspan=1>9/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>1/15</td><td rowspan=1 colspan=1>5/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>5/15</td><td rowspan=1 colspan=1>9/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>No.Detected</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>13/45</td><td rowspan=1 colspan=1>23/45</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=5>% Agreement with Expected Results (Across Sites)*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>71.1</td><td rowspan=1 colspan=1>48.9</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>56.6-82.3</td><td rowspan=1 colspan=1>35.0-63.0</td><td rowspan=1 colspan=1>96.0-100</td></tr><tr><td rowspan=1 colspan=8>hMPV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Low Pos.</td><td rowspan=1 colspan=2>Mod. Pos.</td><td rowspan=1 colspan=2>High Neg.</td><td rowspan=1 colspan=1>Neg.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV A</td><td rowspan=1 colspan=1>hMPV B</td><td rowspan=1 colspan=1>hMPV A</td><td rowspan=1 colspan=1>hMPV B</td><td rowspan=1 colspan=1>hMPV A</td><td rowspan=1 colspan=1>hMPV B</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>4/15</td><td rowspan=1 colspan=1>6/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>1/15</td><td rowspan=1 colspan=1>6/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>5/15</td><td rowspan=1 colspan=1>6/15</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>No.Detected</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>10/45</td><td rowspan=1 colspan=1>18/45</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>with Expected Results</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>60.0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>92.1-100</td><td rowspan=1 colspan=1>63.7-87.5</td><td rowspan=1 colspan=1>45.5-73.0</td><td rowspan=1 colspan=1>96.0-100</td></tr></table>

\* An expected result for the high negative sample is a negative result.

Table 6. Controls Reproducibility   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>RSV PositiveControl</td><td rowspan=1 colspan=1>RSV NegativeControl</td><td rowspan=1 colspan=1>hMPVPositiveControl</td><td rowspan=1 colspan=1>hMPVNegativeControl</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>No. Detected</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>0/90</td></tr><tr><td rowspan=1 colspan=5>% Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9-100</td><td rowspan=1 colspan=1>95.9-100</td><td rowspan=1 colspan=1>95.9-100</td><td rowspan=1 colspan=1>96.0-100</td></tr></table>

b. Linearity/assay Reportable Range:

N/A

b. Traceability, Stability, Expected values (controls, calibrators, or methods):

Sample Stability

A study was conducted to evaluate specimen stability of RSV and hMPV viruses when stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ or at $- 7 0 \mathrm { { } ^ { \circ } C }$ in various types of transport media. The following transport media containing negative clinical matrix were evaluated: Copan UTM, Remel M4, Remel M5, Remel M6, Remel M4RT, and Copan ESwab transport media. Two strains of RSV and two strains of hMPV were spiked into each transport media targeting near LoD concentrations. The virus concentrations used in the study are shown below:

Table 7. Stability Test Panel   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>RSV A strain A2 (VR-1540)</td><td rowspan=1 colspan=1>2.37 x 104</td></tr><tr><td rowspan=1 colspan=1>RSVBstrainWash/18537/62 (VR-1580)</td><td rowspan=1 colspan=1>1.18 x 103</td></tr><tr><td rowspan=1 colspan=1>hMPV16 TypeA1 strain IA10-2003</td><td rowspan=1 colspan=1>1.11 x 103</td></tr><tr><td rowspan=1 colspan=1>hMPV 5 TypeB1 strain Peru3-2003</td><td rowspan=1 colspan=1>1.14 x 104</td></tr></table>

All test samples were aliquoted and placed in storage at the designated temperature. A negative sample (unspiked) of each media was also aliquoted and placed in storage. Each sample was tested in the Solana $\mathrm { R S V + h M P V }$ assay at time 0. One set of samples was placed in a refrigerator (at $2 { - } 8 ^ { \circ } \mathrm { C } $ ) and the second set was placed in a freezer at $- 7 0 \mathrm { { } ^ { \circ } C }$ . Each transport media stored in the refrigerator was tested in the Solana $\mathrm { R S V + h M P V }$ assay in triplicate after storage for 24 hours, 48 hours, 72 hours, 8 days and 10 days. Each transport media stored in the freezer was tested in the Solana $\mathrm { R S V + h M P V }$ assay in triplicate after storage for 1 week, 2 weeks 3 weeks, 4 weeks, 5 weeks, 10 weeks and 15 weeks.

A positive and negative control was included in each experiment and generated expected results. All positive specimens generated positive results for RSV and hMPV at all time points and all negative specimens remained negative at all time points for both storage conditions. The data support specimen stability claim for RSV and hMPV samples up to 8 days when stored in a refrigerator (at $2 { - } 8 ^ { \circ } \mathrm { C }$ ) and up to 10 weeks when frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

# Processed Sample Stability-Prior to Heating

Sample stability was evaluated for specimens that were processed with the Solana Process Buffer and then placed in storage at three temperatures (2 to $8 ^ { \circ } \mathrm { C }$ , room temperature, and $- 2 0 ^ { \circ } \mathrm { C }$ ) prior to the heat lysis step and analysis with the Solana instrument.

Test samples consisted of pooled negative matrix spiked with RSV and hMPV viruses targeting near LoD concentrations (RSV A at $2 . 3 7 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 4 }$ and hMPV B1 at $1 . 1 4 \times 1 0 ^ { 4 } $ . The samples $( 5 0 \mu \mathrm { L } )$ were added to the Process Buffer tube and vortexed for 5 seconds, according to the Solana $\mathrm { R S V + h M P V }$ assay procedure. The tubes were then placed at each of the designated temperatures for a defined period of time before performing the heating step of the procedure. All samples were evaluated and tested in the Solana assay in three replicates at each time point: 3 minutes, 10 minutes, 1 hour, 4 hours, 24 hours, and 48 hours after storage at each of the temperature conditions. One set of samples was further processed without delay through the heating, amplification and detection steps (0 time point). The Solana RSV+hMPV assay results were positive for all samples tested at each time point after storage at the three evaluated temperatures. A positive and negative control was included in each experiment and generated expected results. The results showed that samples remain stable in the Process Buffer when stored prior to further analysis for up to 48 hours at 2 to $8 \mathrm { { } ^ { \circ } C }$ , $2 5 \mathrm { { ^ \circ C } }$ and ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ .

# Processed Sample Stability-Post Heating

Sample stability was evaluated for specimens that were processed with the Solana RSV+hMPV assay through the heating step and then placed in storage at three temperatures (2 to $8 \mathrm { { } ^ { \circ } C }$ , $2 5 \mathrm { { ^ \circ C } }$ (room temperature), and $- 2 0 ^ { \circ } \mathrm { C }$ ) prior to placing the tubes in the Solana instrument for amplification and detection.

Test samples consisted of pooled negative matrix spiked with RSV and hMPV viruses targeting near LoD concentrations (RSV A at $2 . 3 7 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 4 }$ and hMPV B1 at $1 . 1 4 \times 1 0 ^ { 4 } $ . The samples $( 5 0 \mu \mathrm { L } )$ were added to the Process Buffer tube and vortexed for 5 seconds, and then subjected to the heating step at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. After the heating step the tubes were placed in storage at three temperatures: 2 to $8 \mathrm { { } ^ { \circ } C }$ , room temperature $( 2 5 ^ { \circ } \mathrm { C } )$ , and ${ } _ { - 2 0 } \mathrm { { \circ C } }$ . The tubes were kept at the designated temperatures for: 3 minutes, 10 minutes, 1 hour, 4 hours, 24 hours, and 48 hours before placing in the Solana instrument for amplification and detection. One set of samples was analyzed without delay (0 time point). All samples were evaluated and tested in three replicates at each time point after storage at each of the temperature conditions. The Solana $\mathrm { R S V + h M P V }$ assay results were positive for all samples tested demonstrating that the assay performance is not affected if samples are stored for up to 48 hours after the heating step, at the temperatures tested (2 to $8 \mathrm { { } ^ { \circ } C }$ , $2 5 \mathrm { { ^ \circ C } }$ or $- 2 0 ^ { \circ } \mathrm { C }$ ) before further analysis with the Solana instrument. A positive and negative control was included in each experiment and generated expected results.

Effect of Varying the Length of the Heating Interval - Robustness Study

The Solana $\mathrm { R S V + h M P V }$ assay procedure requires that samples are subjected to lysis by heating at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. This study evaluated the performance of the assay when the heating process was either shortened or prolonged, outside of the indicated 5 minutes. The test samples were prepared by spiking nasal clinical matrix with RSV and with hMPV to near LoD concentrations (RSV A at $2 . 3 7 \times 1 0 ^ { 4 }$ and hMPV B1 at $1 . 1 4 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ ). After addition of $5 0 \mu \mathrm { L }$ of each sample to the Process Buffer, the samples were heated to $9 5 \mathrm { { ^ \circ C } }$ and subjected to heating for either 1 minute, 3 minutes, 5 minutes, 7 minutes, 10

minutes or 12 minutes. Each sample was evaluated and tested in three replicates in the Solana RSV+hMPV assay. All samples generated expected positive results demonstrating that accurate results are obtained if the samples are heated anywhere between 1 to 12 minutes. A positive and negative control was included in each experiment and generated expected results.

Range of Heating Temperatures - Robustness Study

The Solana $\mathrm { R S V + h M P V }$ assay procedure requires that samples are subjected to a lysis step by heating at $9 5 \mathrm { { ^ \circ C } }$ for 5 minutes. This study evaluated the performance of the assay when the heating temperature is outside of the specified $9 5 \mathrm { { ^ \circ C } }$ . The test samples were prepared by spiking nasal clinical matrix with RSV and with hMPV to near Lode concentrations (RSV A at $2 . 3 7 \times 1 0 ^ { 4 }$ and hMPV B1 at $1 . 1 4 \mathrm { ~ x ~ } 1 0 ^ { 4 } )$ . After addition of 50 all of each sample to the Process Buffer, the samples were heated to one of the following heating temperatures: $5 5 \mathrm { { ^ \circ C } }$ , $6 5 \mathrm { { ^ \circ C } }$ , $7 5 \mathrm { { } ^ { \circ } C }$ , $8 5 ^ { \circ } \mathrm { C }$ , $9 5 \mathrm { { ^ \circ C } }$ , and $1 0 5 \mathrm { { ^ \circ C } }$ . . Each sample was evaluated and tested in three replicates in the Solana $\mathrm { R S V + h M P V }$ assay. All samples generated expected positive results demonstrating that accurate results are obtained if the heating temperature ranges from $5 5 \mathrm { { ^ \circ C } }$ to $1 0 5 \mathrm { { ^ \circ C } }$ for 5 minutes. A positive and negative control was included in each experiment and generated expected results.

# Fresh vs. Frozen Study

To demonstrate the stability of RSV and hMPV nucleic acids in clinical samples, the sponsor selected 30 samples that were tested fresh (within 72 hours of collection) in the study and then frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ . These samples (10 positive for RSV, 10 positive for hMPV and 10 negative samples) were retested with the Solana assay and with the comparator assay after the study was completed. The time range of storage of the selected samples was from 17 to 29 weeks. The qualitative results for all samples remained unchanged after storage at $- 7 0 ^ { \circ } \mathrm { C }$ , i.e., all positive samples tested positive and all negative samples tested negative in both assays. The Ct values obtained with the comparator assay when testing fresh samples positive for RSV ranged from 23.1 to 35.8, with a median Ct value of 27.0. The Ct values obtained with the comparator assay when testing fresh samples positive for hMPV ranged from 23.1 to 35.8, with a median Ct value of 31.2. The change in $\mathrm { C t }$ values obtained with the comparator assay testing samples positive for RSV after storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ ranged from -0.8 to 2.3. The change in Ct values obtained with the comparator assay testing samples positive for hMPV after storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ ranged from $- 4 . 4$ to 3.7. The data showed that there was no significant change in $\mathrm { C t }$ values over time when stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ for the samples tested.

# Carry-over Study

A carryover study was conducted to verify that cross-contamination and amplicon carryover does not occur when using the Solana assay. Six high positive samples were run in alternating sequence with six negative samples in five separate runs on the Solana instrument. The high positive samples were prepared in nasal clinical matrix at a concentration of $1 . { \overset { - } { 0 } } { \overset { \cdot } { \mathrm { x } } } 1 0 ^ { 5 }$ for each virus, RSV A and hMPV B. There was no carryover observed as all 30 negative samples tested negative even when tested immediately after a high positive sample.

# Controls

The Solana RSV+hMPV Assay contains an internal process control that is included in the reagent mix in the Reaction Tube and is used to detect HDA inhibitory substances in clinical samples and to confirm the integrity of assay reagents and the operation of the Solana instrument.

The Solana RSV+hMPV Control Set is available as an accessory to the assay and consists of two vials of Positive Controls containing inactivated strains of RSV B and hMPV B2, and one vial of a Negative Control made of RSV and hMPV RNA-free matrix. The controls are intended to be used in the same manner as patient samples. The external Positive Control is intended to monitor for substantial reagent and instrument failure. The external Negative Control serves to detect reagent or environment contamination by RSV and hMPV RNA or amplicons.

# d. Analytical Sensitivity (Limit of Detection):

The analytical limit of detection (LoD) of the Solana RSV+hMPV Assay was evaluated using two subtypes of RSV (RSV A and RSV B) and four genetic sub-lineages of hMPV (hMPV A1, hMPV A2, hMPV B1 and hMPV B2). Each pre-titered virus stock was diluted in the negative clinical matrix to a concentration that was 10x the estimated LoD (previously determined). The stocks were further serially diluted and each dilution was tested in 20 replicates with the Solana RSV+hMPV assay according to the instructions for use. The study included multiple Solana instruments. The LoD for each organism was determined to be the viral concentration from the highest dilution where at least 19 out of 20 replicates $( \geq 9 5 \% )$ were positive in the Solana assay for that analyte. The results of the study are shown below.

Table 8. Results of the LoD Study   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2>LoD (TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>7.9x103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>3.9x102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV TypeA1</td><td rowspan=1 colspan=1>3.7x102</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV TypeA2</td><td rowspan=1 colspan=1>1.2x104</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPVTypeB1</td><td rowspan=1 colspan=1>3.8x103</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV TypeB2</td><td rowspan=1 colspan=1>2.3x103</td><td rowspan=1 colspan=1></td></tr></table>

# e. Analytical Reactivity (inclusivity)

The reactivity of the Solana $\mathrm { R S V + h M P V }$ Assay was evaluated with four additional strains of RSV (two RSV A and two RSV B) and with four additional strains of hMPV (one each of hMPV A1, hMPV A2, hMPV B1 and hMPV B2) at concentrations near the level of detection (LoD) of the assay. Pre-titered RSV and hMPV strains were serially diluted in negative nasal matrix targeting concentrations of 2x LOD. The dilutions of the viruses were tested in three replicates in the Solana RSV+hMPV Assay according to the instructions for use. The study was performed using multiple Solana instruments. The experiments included a positive and a negative control for each setup. All RSV and hMPV strains were detected in all three replicates at the concentrations shown below.

Table 9. Analytical Reactivity Results   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>RSV A, strain Long (VR-26)</td><td rowspan=1 colspan=1>1.6x104</td></tr><tr><td rowspan=1 colspan=1>RSV A, strain 4/2015 Isolate #1</td><td rowspan=1 colspan=1>1.6x104</td></tr><tr><td rowspan=1 colspan=1>RSV B, strain 9320 (VR-955)</td><td rowspan=1 colspan=1>7.9x102</td></tr><tr><td rowspan=1 colspan=1>RSV B, strain WV/14617/85 (VR-1400)</td><td rowspan=1 colspan=1>7.9x102</td></tr><tr><td rowspan=1 colspan=1>hMPV 9 Type A1, strain IA3-2002</td><td rowspan=1 colspan=1>7.4x102</td></tr><tr><td rowspan=1 colspan=1>hMPV 27 Type A2, strain IA27-2004</td><td rowspan=1 colspan=1>2.4x104</td></tr><tr><td rowspan=1 colspan=1>hMPV 3 Type B1, strain Peru2-2002</td><td rowspan=1 colspan=1>7.6x103</td></tr><tr><td rowspan=1 colspan=1>hMPV 18 Type B2, strain IA18-2003</td><td rowspan=1 colspan=1>4.5x103</td></tr></table>

Additionally, the sponsor conducted an in silico analysis of the hMPV and RSV primers and probes used in the assay to determine the inclusivity of various strains of these viruses based on the hMPV and RSV sequences available from NCBI. The primer and probe sequences demonstrated $100 \%$ homology with the majority of viral sequences found in the NCBI database.

# f. Analytical specificity:

# Cross-Reactivity

The Solana $\mathrm { R S V + h M P V }$ Assay was evaluated for potential cross-reactivity (false positive results) with organisms that may be found in the nasal passages of patients with symptoms of respiratory tract infection.

A panel of 46 microorganisms (25 bacteria, 1 yeast, 20 viruses) was tested in the Solana RSV+hMPV Assay. All microorganisms tested were pre-titered. Each organism was spiked into pooled negative nasal matrix such that the concentration was ${ \ge } 1 \mathrm { x } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses or $\geq 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and yeast.

The samples were tested in 3 replicates with the Solana RSV+hMPV Assay according to the instructions for use. The study was performed using multiple Solana instruments. The experiments included a positive and a negative control for each setup. No cross-reactivity was observed with the organisms shown below at the concentrations tested.

Table 10. Potential Cross-reactive Organisms   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">Units</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (VR-977)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus, Type 71</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">HSV 2 G strain</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">hMPV Peru1-2002, B21</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Texas/50/2012</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Panama/45/90</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus Type 7</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">RSV A2 (VR-1540)2</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td></tr></table>

1All three replicates tested positive for hMPV and negative for RSV in the Solana RSV+hMPV assay. 2All three replicates tested negative for hMPV and positive for RSV in the Solana $\mathrm { R S V + h M P V }$ assay.

# Microbial Interference

A study was conducted to demonstrate that organisms that may be present in specimens collected from nasal passages of patients symptomatic of respiratory tract infections do not interfere with the detection of RSV or hMPV when tested with the Solana assay.

A panel of the same 46 microorganisms described above was tested with the Solana RSV+hMPV Assay in the presence of low concentrations of RSV and hMPV. Each potential interfering microorganism was spiked into negative nasal matrix for a final concentration o $: \geq 1 \mathrm { x } 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ for viruses or $\geq 1 \mathrm { { x } } 1 0 ^ { 6 } \mathrm { { C F U / m L } }$ for bacteria and yeast. This study was conducted with two strains of RSV and two strains of hMPV at concentrations shown below.

Table 11. Microbial Interference Strains Tested   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Concentration Tested(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>RSV A, A2</td><td rowspan=1 colspan=1>2.37 x 104</td></tr><tr><td rowspan=1 colspan=1>RSV B, Wash/18537/62</td><td rowspan=1 colspan=1>1.18 x 103</td></tr><tr><td rowspan=1 colspan=1>hMPV 16 Type A1, IA10-2003</td><td rowspan=1 colspan=1>1.11 x 103</td></tr><tr><td rowspan=1 colspan=1>hMPV 5 Type B1, Peru3-2003</td><td rowspan=1 colspan=1>1.14 x 104</td></tr></table>

Each sample was tested in the Solana $\mathrm { R S V + h M P V }$ assay in three replicates using multiple Solana instruments. The runs also included the target strains (above) without any other organisms, as a control.

No microbial interference was observed for any of the samples at the concentrations tested.

The study also demonstrated that there is no competitive interference between RSV and hMPV when both organisms are present. The table below shows the RSV and hMPV strains and concentrations evaluated in this study.

Table 12. Competitive Interference Strains Tested   

<table><tr><td rowspan=1 colspan=1>Target Virus</td><td rowspan=1 colspan=1>ConcentrationTested(TCID50/mL)</td><td rowspan=1 colspan=1>Inhibitory Virus</td></tr><tr><td rowspan=1 colspan=1>RSV A, A2 (VR-1540)</td><td rowspan=1 colspan=1>2.37 x 104</td><td rowspan=1 colspan=1>hMPV B2 (1.0 x 105 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>RSV B, Wash/18537/62 (VR-1580)</td><td rowspan=1 colspan=1>1.18 x 103</td><td rowspan=1 colspan=1>hMPV B2 (1.0 x 105 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>hMPV 16 Type A1, IA10-2003</td><td rowspan=1 colspan=1>1.11 x 103</td><td rowspan=1 colspan=1>RSV A2 (1.0 x 105 TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>hMPV 5 Type B1, Peru3-2003</td><td rowspan=1 colspan=1>1.14 x 104</td><td rowspan=1 colspan=1>RSV A2 (1.0 x 105 TCID50/mL)</td></tr></table>

# Interference from Substances

Stocks of 20 potentially interfering substances were prepared in a buffered bovine albumin (BSA) solution. Each substance was spiked into pooled negative nasal matrix to a targeted concentration and tested in the presence of RSV (RSV A, A2 (VR-1540) and RSV B, Wash/18537/62 (VR-1580)) or hMPV (hMPV 16 Type A1, IA10-2003 and hMPV 5 Type B1, Peru3-2003) at concentrations of $3 \mathbf { x }$ LoD for each virus. Each substance was tested in 3 replicates in the Solana assay. The study was conducted using multiple Solana instruments. Each run included a positive and a negative control. There was no interference observed with any of the substances tested at the concentrations shown below.

Table 13. Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">2.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Afrin - nasal spray (Oxymetazoline)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Saline nasal spray (Sodium chloride)</td><td colspan="1" rowspan="1">15.0%</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine (Phenylephrinehydrochloride)</td><td colspan="1" rowspan="1">15.0%</td></tr><tr><td colspan="1" rowspan="1">Flonase (Fluticasone)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Zicam Gentle Allergy Relief NasalGel(Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulfur)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Bactroban (Mupirocin)</td><td colspan="1" rowspan="1">12.0 mg/mL</td></tr><tr><td colspan="1" rowspan="1">TamiFlu (Oseltamivir)</td><td colspan="1" rowspan="1">2.2 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Relenza (Zanamivir)</td><td colspan="1" rowspan="1">282.0 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Tobramycin (Tobramycin sulfate)</td><td colspan="1" rowspan="1">2.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Chloraseptic Spray (Benzocaine, Menthol)</td><td colspan="1" rowspan="1">0.68 g/mL</td></tr><tr><td colspan="1" rowspan="1">Symmetrel (Amantadine hydrochloride)</td><td colspan="1" rowspan="1">282.0 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Nasocort Allergy 24 hour (Triamcinolone)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Sinus Buster Nasal Spray(Capsaicin)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">NasalCrom Nasal Allergy Spray (CromolynSodium)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Rhinocort (Budesonide/Glucocorticoid)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Air-Vita Allergy Multi-Symptom Relief(various plant extracts)</td><td colspan="1" rowspan="1">5.0%</td></tr><tr><td colspan="1" rowspan="1">Atrovent Nasal Spray (Ipratropium bromide)</td><td colspan="1" rowspan="1">10.0 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Patanase Nasal Spray (Olopatadinehydrochloride)</td><td colspan="1" rowspan="1">10.0 mg/mL</td></tr></table>

# g. Assay cut-off:

The preliminary threshold settings for the Solana $\mathrm { R S V + h M P V }$ Assay were determined during the development stage and were based on the analysis of the fluorescent signal obtained with known positive and negative clinical specimens, as well as analysis of contrived samples used in the determination of LoD. The RSV cutoff determination involved 24 positive and 62 negative specimens and the hMPV cutoff 23 positive and 61 negative specimens, as determined by the comparator assay. The specific parameters used for the determination of the cutoff for each analyte included the slope of the amplification curve for samples near the LoD (“slope threshold”), the time required to reach the slope threshold and the duration of the evaluation of the generated signal. The cutoff settings were established and used during the clinical study.

2. Comparison studies:

a. Method comparison with predicate device: See below under the “Clinical studies” section.

b. Matrix comparison:

Not applicable.

3. Clinical studies:

A prospective clinical study was conducted between January and May 2017, testing specimens collected from 2064 patients with signs and symptoms of respiratory infection. Four specimens were removed from the study due to protocol deviations (inappropriate specimen type) leaving a total of 2060 specimens for evaluation. Three hundred (300) nasal and 1760 nasopharyngeal swab specimens collected in transport media were tested with the Solana $\mathrm { R S V + h M P V }$ assay at six clinical testing sites. Comparison testing was performed on all specimens at a central clinical laboratory using an FDA cleared molecular assay detecting RSV and hMPV. Of those, 769 specimens (collected between April and May 2017) were tested fresh and 1291 specimens (collected between January and May 2017) were frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ and shipped to clinical sites for testing with the Solana assay. An aliquot of each sample was sent (on cold packs or dry ice as needed) to the central laboratory for testing with the comparator assay.

The samples were transported and stored at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 72 hours or frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ after 72 hours until testing.

The gender and demographics for the study population are shown below.

Table 14. Study Population Demographics   

<table><tr><td rowspan=1 colspan=3>Fresh Samples</td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>380</td><td rowspan=1 colspan=1>389</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>22 to 50 years</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>51 to 65 years</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Frozen Samples</td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>673</td><td rowspan=1 colspan=1>618</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>22 to 50 years</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>51 to 65 years</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>115</td></tr></table>

# Data analysis

Of the 2060 samples evaluated, 14 specimens were invalid in the Solana assay (remained invalid upon repeat testing) and were excluded from data analysis for an invalid rate of $0 . 7 \%$ (14/2060), with the $9 5 \%$ CI $( 0 . 4 \% - 1 . 1 \% )$ . The remaining 2046 samples were included in the final data analysis.

The positive percent agreement (PPA) and the negative percent agreement (NPA) of the Solana $\mathrm { R S V + h M P V }$ assay for RSV and for hMPV, when compared to an FDA cleared comparator assay is shown below. The performance is presented by sample storage category (fresh or frozen) and combined.

Table 15. Solana RSV+hMPV Assay Performance for RSV vs. FDA Cleared Molecular Assay   

<table><tr><td rowspan=1 colspan=8>RSV</td></tr><tr><td rowspan=1 colspan=1>SampleStorage</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=1>NPA (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>747</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.3%(66.7%-98.6%)</td><td rowspan=1 colspan=1>100%(99.5%- 100%)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1286</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1143</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>95.8%(91.1% -98.0%)</td><td rowspan=1 colspan=1>99.9%(99.5%- 100%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2046</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1890</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>95.5%(91.0%-97.8%)</td><td rowspan=1 colspan=1>99.9%(91.0%-97.8%)</td></tr></table>

1One specimen positive for RSV in the Solana assay and negative by the Comparator was positive for RSV by an alternative FDA-cleared molecular device. $^ 2 \mathrm { S i x }$ of the seven specimens negative for RSV in the Solana assay and positive by the Comparator, were positive for RSV by an alternative FDA-cleared molecular device.

Table 16. Solana RSV+hMPV Assay Performance for hMPV vs. FDA Cleared Molecular Assay   

<table><tr><td rowspan=1 colspan=8>hMPV</td></tr><tr><td rowspan=1 colspan=1>SampleStorage</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>733</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.0%(80.5%-99.3%)</td><td rowspan=1 colspan=1>99.7%(99.0%-99.9%)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1286</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1220</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.4%(87.3%-98.4%)</td><td rowspan=1 colspan=1>99.9%(99.5%-100%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2046</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1953</td><td rowspan=1 colspan=1>4{2</td><td rowspan=1 colspan=1>95.6%(89.1%-98.3%)</td><td rowspan=1 colspan=1>99.8%(99.6%-99.9%)</td></tr></table>

1Three specimens positive for hMPV in the Solana assay and negative by the Comparator were negative for hMPV by an alternative FDA-cleared molecular device. 2Four specimens negative for hMPV in the Solana assay and positive by the Comparator, were positive for hMPV by an alternative FDA-cleared molecular device.

4. Clinical cut-off: Not applicable.

The positivity rate of RSV and hMPV was established during the prospective study conducted between January and May 2017, based on 2046 (760 fresh and 1286 frozen) evaluable specimens collected from patients with signs and symptoms of respiratory tract infection across six clinical sites in the United States. The majority of samples collected between January and May 2017 were frozen before testing. The majority of samples collected between April and May 2017 were tested fresh. The positivity rate of the Solana RSV+hMPV assay during the study period is shown below for the frozen samples and for the fresh samples, stratified by age.

Table 17. Solana $\mathbf { R S V + }$ hMPV Positivity Rate during the Clinical Study   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>RSV</td><td rowspan=1 colspan=2>hMPV</td></tr><tr><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>TotalPositive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=6>Fresh Specimens (N=760) (collected Apr-May 2017)</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5.9%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6.1%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>22 to 50 years</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>51 to 65 years</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.1%</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=6>Frozen Specimens (N=1286) (collected Jan-May 2017)</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>36.2%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6.9%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>15.4%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8.8%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.3%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>22 to 50 years</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>51 to 65 years</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.9%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.0%</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>278</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.2%</td></tr></table>

# N. Instrument Name:

Solana Instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No X_

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X_

2. Software:

The Solana instrument platform was originally reviewed under K150868. The additional information was provided in support of the Solana RSV+hMPV Assay and was reviewed and found acceptable.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification: Specimens are identified by scanning a barcode or by manual entry.

4. Specimen Sampling and Handling:

Swab specimens are collected in viral transport medium. After vortexing, $5 0 \mu \mathrm { l }$ of the expressed specimen is transferred to a lysis tube. After heat lysis, $5 0 \mu \mathrm { l }$ of the specimen is transferred to a reaction tube on the Solana instrument for automated amplification and detection.

5. Calibration:

The end user is not required to calibrate the instrument.

6. Quality Control:

a. Process Control

A competitive process control (PRC) is included in the Process Buffer Tube to monitor sample processing, inhibitory substances in clinical samples, reagent failure or device failure. The PRC target is amplified by RSV and hMPV specific primers and detected by a PRC specific fluorescence probe.

a. External Controls:

The external controls are available from Quidel as an accessory to the assay. The control kit includes two vials of Positive Controls (inactivated strains of RSV B and of hMPV8 B2) and one vial of a Negative Control (RSV and hMPV RNA-free matrix). These controls are intended to monitor pre-analytical and environmental factors that could substantially affect reagent integrity or the instrument function. The negative control is used to detect reagent or environmental contamination of the system by RSV or hMPV RNA or amplicons.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.