# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K171441   
B. Purpose for Submission: To obtain a Substantial Equivalence Determination for the ARIES C. difficile Assay   
C. Measurand: Conserved regions of the Clostridium difficile toxin A (tcdA) and toxin B (tcdB) genes   
D. Type of Test: Qualitative real-time Polymerase Chain Reaction (PCR)   
E. Applicant: Luminex Corporation

# F. Proprietary and Established Names:

ARIES C. difficile Assay Complete Kit ARIES C. difficile Assay Protocol File Kit ARIES C. difficile Assay Kit ARIES Stool Resuspension Kit

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130: Clostridium difficile toxin gene amplification assay

2. Classification: Class II

3. Product code: OZN: C. difficile toxin gene amplification assay OOI: Real time nucleic acid amplification system

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The ARIES C. difficile Assay is a real-time polymerase chain reaction (PCR) based qualitative in vitro diagnostic test for the direct detection of toxigenic Clostridium difficile (C. difficile) nucleic acid in unpreserved, unformed (liquid or soft) stool specimens obtained from patients suspected of having Clostridium difficile infection (CDI). The test targets the C. difficile toxin A gene (tcdA) and toxin B gene (tcdB) and is indicated for use as an aid in the diagnosis of $C .$ . difficile infection (CDI).

The ARIES C. difficile Assay is indicated for use with ARIES Systems.

2. Indication(s) for use:

Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only.

The ARIES C. difficile Assay is for use with unpreserved, liquid or soft human stool specimens from symptomatic patients. Performance characteristics with other specimen types or patients without symptoms of infection with toxigenic C. difficile have not been established.

False negative results may occur due to the presence of sequence variation in the regions of the tcdA and tcdB genes targeted by the ARIES C. difficile assay.

4. Special instrument requirements:

The ARIES C. difficile Assay is indicated for use with Luminex ARIES Systems.

# I. Device Description:

The ARIES C. difficile Assay is a Polymerase Chain Reaction (PCR)-based qualitative in vitro diagnostic test system that consists of the ARIES System or the ARIES M1 System with the associated ARIES Software, a stool resuspension kit, an assay-specific test cassette, and an assay-specific Protocol File. The ARIES C. difficile Assay cassette is a disposable, single-use device that contains nucleic acid purification reagents, an internal Sample Processing Control (SPC), and an assay-specific master mix for the detection of the $C .$ . difficile toxin A (tcdA) and toxin B (tcdB) genes. The assay is for use on unpreserved, liquid or soft stool specimens from patients suspected of $C .$ . difficile infection.

The stool specimens are processed using the ARIES Stool Resuspension Kit which includes flocked swabs for sample transfer, Stool Resuspension Tubes containing pre-processing beads, and Stool Resuspension Buffer. To process a specimen, a flocked swab is used to transfer an aliquot of stool to a Stool Resuspension Tube containing the appropriate volume of Resuspension Buffer. The swab head is broken off into the tube which is then vortexed and centrifuged. An aliquot of the supernatant from the pre-processed sample is then added to the assay cassette which is loaded into the ARIES instrument for automated nucleic acid extraction, amplification and detection.

The assay cassette includes a Sample Processing Control (SPC) that is extracted and processed with the patient specimen. The SPC is designed to monitor DNA recovery, amplification and detection.

The extracted nucleic acid and SPC are transferred through the assay cassette by magnetic beads. The eluted sample is then used to rehydrate lyophilized PCR reagents that are specific for the tcdA and tcdB genes and SPC. Each primer pair is labeled with a different fluorophore and is detected in a different optical channel of the ARIES Systems. During PCR amplification, synthetic quencher nucleotides are incorporated into the amplified products that result in a decrease in fluorescence in the corresponding optical channel when target DNA is present. Following amplification, melt curve analysis is performed to confirm the identity of the amplicons. Results are interpreted automatically using parameters in the ARIES C. difficile Assay Protocol File as either “Toxigenic C. difficile Positive”, “Toxigenic C. difficile Negative” or “Invalid, and may be reported from the ARIES Software or from the optional SYNCT Software desktop application.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Quidel Molecular Direct C. difficile Assay

2. Predicate 510(k) number(s): K123998

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device (K171441)</td><td colspan="1" rowspan="1">Predicate (K123998)</td></tr><tr><td colspan="1" rowspan="1">Luminex ARIESC. difficile Assay</td><td colspan="1" rowspan="1">Quidel Molecular DirectC. difficile Assay</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3130</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OZN</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The ARIES C. difficileAssay is a real-timepolymerase chain reaction(PCR) based qualitative invitro diagnostic test for thedirect detection of toxigenicClostridium difficile (C.difficile) nucleic acid inunpreserved, unformed(liquid or soft), stoolspecimens obtained frompatients suspected of havingClostridium difficileinfection (CDI). The testtargets the C. difficile toxinA gene (tcdA) and toxin Bgene (tcdB) and is indicatedfor use as an aid in thediagnosis of C. difficileinfection (CDI).The ARIES C. difficileAssay is indicated for usewith ARIES Systems.</td><td colspan="1" rowspan="1">The Quidel MolecularDirect C. difficile Assay isa qualitative, multiplexedin vitro diagnostic test forthe direct detection of toxinA gene (tcdA) or toxin Bgene (tcdB) sequences oftoxigenic strains ofClostridium difficile fromunformed (liquid or soft)stool specimens collectedfrom patients suspected ofhaving Clostridiumdifficile-AssociatedDisease (CDAD).The Quidel MolecularDirect C. difficile Assay isa real-time PCR test andutilizes proprietary samplepreparation withfluorescently labeledprimers and probes.The assay can beperformed using either theLife TechnologiesQuantStudio Dx; theApplied Biosystems 7500Fast Dx, or the CepheidSmartCycler II, to detectthe toxin gene sequencesassociated with toxin-producing C. difficilestrains.The assay is intended to beperformed directly on</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CDAD-suspected stoolspecimens, and is indicatedfor use as an aid in thediagnosis of CDAD.</td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">Toxin A (tcdA) and toxin B(tcdB) genes of C. difficile</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Unpreserved, unformed(liquid or soft) stool</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Format</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=1>Device (K171441)</td><td rowspan=1 colspan=1>Predicate (K123998)</td></tr><tr><td rowspan=1 colspan=1>Luminex ARIESC. difficile Assay</td><td rowspan=1 colspan=1>Quidel Molecular DirectC. difficile Assay</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>ARIES System or ARIESM1 System</td><td rowspan=1 colspan=1>Life TechnologiesQuantStudio Dx, AppliedBioSystems 7500 Fast Dxor Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=1>Assay Format</td><td rowspan=1 colspan=1>Pre-fabricated, unitizedreagent cassette for singleuse</td><td rowspan=1 colspan=1>Manually prepared PCRtubes or plates</td></tr><tr><td rowspan=1 colspan=1>Detection Chemistry</td><td rowspan=1 colspan=1>Fluorescently labeledprimers with quencher-labeled nucleotidesDecrease in fluorescenceover time</td><td rowspan=1 colspan=1>Fluorescently-labeledhydrolysis probesIncrease in fluorescenceover time</td></tr><tr><td rowspan=1 colspan=1>Result Interpretation</td><td rowspan=1 colspan=1>Cycle threshold valuecoupled with melt curveanalysis</td><td rowspan=1 colspan=1>Cycle threshold value</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline -Third Edition. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

CLSI. Interference Testing in Clinical Chemistry; Approved Guideline - Second Edition.   
CLSI document EP07-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2005.

CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. CLSI document EP12-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. User Verification of Precision and Estimation of Bias; Approved Guideline - Third

Edition. CLSI document EP15-A3. Wayne, PA: Clinical and Laboratory Standards Institute;   
2014.

CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - Second Edition. CLSI document EP17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

CLSI. Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline - Second Edition. CLSI document EP24-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2011.

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

CLSI. Molecular Diagnostic Methods for Infectious Diseases – Third Edition. CLSI report MM03. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.

CLSI. Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline. CLSI document MM13-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2005.

ISO 14971. Medical devices - Application of risk management to medical devices.

# L. Test Principle:

The ARIES C. difficile Assay is performed on unpreserved, unformed (liquid or soft) stool from patients suspected of C. difficile infection. A swab is used to transfer a portion of the stool specimen to a Stool Resuspension Tube containing Resuspension Buffer and preprocessing beads. The tube is vortexed briefly to homogenize the sample and centrifuged to sediment the beads and particulate matter. An aliquot of the supernatant is then added to an ARIES C. difficile Assay cassette which is loaded into the magazine of the ARIES System for automated processing. Up to 6 cassettes can be loaded in a single magazine. Within the instrument, a barcode on the ARIES C. difficile Assay cassette is automatically scanned to associate the cassette with the appropriate assay protocol that provides all the necessary information to perform the test, analyze the data and generate the result report.

Each assay cassette contains all the reagents necessary for nucleic acid extraction, amplification and detection for one sample and includes a Sample Processing Control to monitor reagent and process integrity. Processing of the sample within the assay cassette is fully automated and there is no direct contact between the sample or reagents and the instrument, reducing the potential for contamination.

The extracted nucleic acids are used to rehydrate the lyophilized PCR Master Mix which contains specific, fluorescently labeled primers for each toxin gene target and the SPC. The fluorescently labeled primers contain the synthetic nucleotide base 2'-deoxy-5-methyl

isocytidine (iC). During PCR amplification, incorporation of quencher-modified 2'- deoxyisoguanosine triphosphate (iG) into the nascent DNA strand opposite the iC residues results in a target-specific reduction of fluorescence with each successive cycle as amplicons accumulate. The ARIES Systems monitor the decrease in fluorescence in real-time and use the changes in signal over the course of the reaction to calculate amplification cycle $\mathrm { ( C t ) }$ values. At the end of the reaction the amplification products undergo melt curve analysis to verify their identity. A combination of metrics for the toxin A (tcdA) and toxin B (tcdB) genes of $C .$ . difficile and the SPC is used for result interpretation. If either toxin gene is detected, the sample is reported as “Toxigenic C. difficile Positive”. If neither of the toxin genes is detected and the SPC signal is valid, the sample is reported as “Toxigenic $C .$ . difficile Negative”, otherwise the result is “Invalid”.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The Site-to-Site Reproducibility, Within Laboratory Precision/Repeatability and Lot-to-Lot Reproducibility Studies described below were conducted using an assay workflow that did not include pre-processing of the stool specimens. However, the LoD of the ARIES $C .$ . difficile Assay was shown to be similar in terms of CFU/test cassette with and without preprocessing, and the data from these studies were therefore considered acceptable. Assay reproducibility with pre-processing was demonstrated over multiple days in studies performed to evaluate the stability of stool specimens prior to testing (Section M(1)(c)).

# Site-to-Site Reproducibility

The reproducibility of the ARIES $C .$ . difficile Assay between sites was evaluated in a study performed by two operators on two instruments at each of three sites over a period of five days. Each operator tested a blinded panel of $C .$ . difficile positive and negative samples using the same lot of reagents (3 sites $\mathrm { \Delta X } 2$ operators $\mathrm { ~ X ~ 5 ~ }$ days X 3 replicates $= 9 0$ data points per panel member). The panels consisted of two strains of toxigenic $C .$ difficile each of which was tested at Moderate, Low and High Negative target levels, in addition to $C .$ . difficile negative samples. The results of the study demonstrated acceptable reproducibility from siteto-site at target levels close to the limit of detection (LoD) of the assay (Table 1).

Table 1. Summary of results from the ARIES $C .$ . difficile Assay Site-to-Site Reproducibility Study, stratified by site and overall   

<table><tr><td rowspan=2 colspan=1>C. difficileATCC Strain</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=4>Number Positive/Number Tested (%)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=3 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive1X LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>28/30(93.3)</td><td rowspan=1 colspan=1>21/30(70.0)</td><td rowspan=1 colspan=1>27/30(90.0)</td><td rowspan=1 colspan=1>76/90(84.4)</td></tr><tr><td rowspan=3 colspan=1>BAA-1871</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/301(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive1X LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29/30(96.7)</td><td rowspan=1 colspan=1>89/90(98.9)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>23/30(76.7)</td><td rowspan=1 colspan=1>28/30(93.3)</td><td rowspan=1 colspan=1>24/30(80.0)</td><td rowspan=1 colspan=1>75/90(83.3)</td></tr><tr><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/30(0.0)</td><td rowspan=1 colspan=1>1/30(3.3)</td><td rowspan=1 colspan=1>0/30(0.0)</td><td rowspan=1 colspan=1>1/90(1.1)</td></tr></table>

LoD: Limit of Detection (CFU/test cassette) 1 1/30 samples was reported as Invalid on initial testing; reported as Positive upon repeat

# Within Laboratory Precision/Repeatability

Within laboratory precision/repeatability of the ARIES C. difficile Assay was evaluated by testing a panel of samples in triplicate on multiple ARIES instruments over a period of 12 days (3 replicates $\mathrm { ~ X ~ } 1 2 \mathrm { ~ d a y s } = 3 6$ replicates per panel member). The panel members were prepared using same two strains of $C .$ . difficile and target levels as those in the Site-to-Site Reproducibility Study, above. The results of the study demonstrated acceptable repeatability and precision from day-to-day with target levels at or above the LoD (Table 2).

Table 2. Summary of results from the Within Laboratory Precision/Repeatability Study for the ARIES C. difficile Assay   

<table><tr><td rowspan=1 colspan=1>C.difficileATCC Strain</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Positive/Tested (%)</td></tr><tr><td rowspan=3 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>36/36(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive1X LoD</td><td rowspan=1 colspan=1>36/36(100)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>24/36(66.7)</td></tr><tr><td rowspan=3 colspan=1>BAA-1871</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>36/36(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive1X LoD</td><td rowspan=1 colspan=1>35/36(97.2)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>9/36(25.0)</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/36(0.0)</td></tr></table>

LoD: Limit of Detection (CFU/test cassette) 2/36 samples produced Invalid results on initial testing; both reported as Positive upon repeat

# Lot-to-Lot Reproducibility

The lot-to-lot reproducibility of the ARIES $C .$ . difficile Assay was evaluated by testing nine replicates of each member of a panel of $C .$ . difficile positive and negative samples with each of three lots of reagents (9 replicates $\mathrm { ~ X ~ } 3 ~ \mathrm { l o t s } = 2 7$ replicates per panel member). The panels were prepared using two strains of $C .$ . difficile at Moderate, Low and High Negative target levels. The results are summarized in Table 3 and show acceptable performance with each lot of reagents.

Table 3. Summary of results from the ARIES $C .$ . difficile Assay Lot-to-Lot Reproducibility Study, stratified by reagent lot and overall   

<table><tr><td rowspan=2 colspan=1>C. difficileATCC Strain</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=4>Positive/Tested (%)</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=3 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>27/27(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive2X LoD</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>27/27(100)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>8/9(88.9)</td><td rowspan=1 colspan=1>26/27(96.3)</td></tr><tr><td rowspan=3 colspan=1>BAA-1871</td><td rowspan=1 colspan=1>Moderate Positive10X LoD</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>27/27(100)</td></tr><tr><td rowspan=1 colspan=1>Low Positive2X LoD</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>9/9(100)</td><td rowspan=1 colspan=1>27/27(100)</td></tr><tr><td rowspan=1 colspan=1>High Negative0.1X LoD</td><td rowspan=1 colspan=1>6/9(66.7)</td><td rowspan=1 colspan=1>8/9(88.9)</td><td rowspan=1 colspan=1>4/9(44.4)</td><td rowspan=1 colspan=1>18/27(66.7)</td></tr><tr><td rowspan=1 colspan=1>NotApplicable</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/92(0.0)</td><td rowspan=1 colspan=1>0/9(0.0)</td><td rowspan=1 colspan=1>0/9(0.0)</td><td rowspan=1 colspan=1>0/27(0.0)</td></tr></table>

LoD: Limit of Detection (CFU/test cassette) 1 $1 / 9$ samples was reported as Invalid on initial testing; reported as Positive upon repeat 2 $1 / 9$ samples reported as Invalid on initial testing; reported as Negative upon repeat

# b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Specimen Stability

The stability of stool specimens for use with the ARIES C. difficile Assay was evaluated analytically by testing pooled $C .$ . difficile negative stool specimens that were seeded with an enumerated suspension of cultured $t c d A ^ { + } / t c d B ^ { + }$ organisms and stored under different conditions. Unseeded stool was included to assess the effect of specimen storage on the performance of the SPC. Six C. difficile positive and negative assay replicates were tested at each stability time point. The results of these studies supported the stability of stool specimens for up to 4 hours at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ , 7 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ or 3 months at $\mathrm { { \le - 7 0 } ^ { \circ } C }$ .

An additional study was conducted to verify the ability of the ARIES assay to detect $C .$ . difficile in seeded specimens after multiple freeze-thaw cycles. The expected results were obtained at each target level after 5 freeze-thaw cycles.

The Specimen Stability Studies required testing of multiple samples with standardized target levels over multiple days. The results of these studies therefore provided additional data to demonstrate acceptable reproducibility and/or repeatability of the ARIES C. difficile Assay when using the specimen pre-processing procedure.

# Reagent Stability

The shelf-life of the ARIES C. difficile Assay cassettes was evaluated in a real-time stability study performed on three lots of reagents that were stored either refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ or at room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ . The results from the study support assignment of an expiration date 12 months from the day of manufacture for the assay cassettes when stored under the recommended conditions.

A separate study demonstrated that ARIES C. difficile Assay cassettes were stable to up to 10 hours at ambient temperature when removed from their primary packaging.

Sample Processing Control Each ARIES C. difficile Assay cassette contains a Sample Processing Control (SPC) that is designed to monitor nucleic acid extraction and PCR amplification. Samples that are negative for toxigenic C. difficile by the ARIES assay and in which the SPC is not detected are reported as “Invalid” and must be retested beginning with the primary stool sample.

# External Controls

External Controls should be tested according to guidelines or requirements of local, provincial and/or federal regulations or accreditation organizations. A toxigenic reference strain of $C .$ . difficile or clinical isolate may be used as a Positive Control. Another species of Clostridium or a non-toxigenic strain of C. difficile or may be used as a Negative Control. Alternatively, clinical specimens that are known to be positive or negative for toxigenic C. difficile may be used as Positive and Negative External Controls, respectively.

External Positive and Negative Controls were tested on a daily basis during the prospective Clinical Study described in Section M(3)(a). The Positive External Control comprised a standardized suspension of a strain of toxigenic C. difficile. The negative External Control comprised a strain of Clostridium sordellii. On initial testing, 184/187 $( 9 8 . 4 \% )$ Positive and 183/187 $( 9 7 . 9 \% )$ ) Negative External Controls produced the expected results.

# d. Detection limit:

# Limit of Detection

The Limit of Detection (LoD) of the ARIES C. difficile Assay was determined for five strains of C. difficile by testing various dilutions of enumerated cell stocks in stool matrix. The LoD was defined as the lowest concentration tested at which $2 9 5 \%$ of assay replicates produced positive results (Table 4).

Table 4. Analytical sensitivity of the ARIES $C .$ . difficile Assay   

<table><tr><td rowspan=2 colspan=1>C. difficileATCC Strain</td><td rowspan=2 colspan=1>Toxinotype</td><td rowspan=2 colspan=1>tcdA/tcdBGenes {2</td><td rowspan=1 colspan=2>Limit of Detection (CFU)</td></tr><tr><td rowspan=1 colspan=1>Per mL Stool</td><td rowspan=1 colspan=1>Per Cassette</td></tr><tr><td rowspan=1 colspan=1>BAA-1871</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>43598</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>-/+</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>BAA-1812</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>BAA-1803</td><td rowspan=1 colspan=1>IIIc</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>18.6</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>BAA-1870</td><td rowspan=1 colspan=1>IIIb</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>0.6</td></tr></table>

ATCC: American Type Culture Collection; CFU: Colony Forming Units 1 Outbreak-associated Pulsed Field Gel Electrophoresis type NAP1 2 As reported by ATCC

# Inclusivity (Analytical Reactivity)

The inclusivity of the ARIES C. difficile Assay was evaluated by testing 15 additional strains of C. difficile at levels close to the LoD of the assay (Table 5). All $( 1 0 0 \% )$ 15 strains were successfully detected.

Table 5. Strains of $C .$ . difficile used to evaluate the inclusivity of the ARIES C. difficile Assay   

<table><tr><td rowspan=1 colspan=1>C. difficileATCC Strain</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>tcdA/tcdB Genes1</td></tr><tr><td rowspan=1 colspan=1>17858</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>43255</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>43600</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>700792</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1382</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1806</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1808</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1811</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1814</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>+/-</td></tr><tr><td rowspan=1 colspan=1>BAA-1815</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1872</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1873</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1874</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-1875</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>BAA-2156</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>+/+</td></tr></table>

ATCC: American Type Culture Collection 1 As reported by ATCC

# Bioinformatic Analysis

The inclusivity of the ARIES C. difficile Assay primers for the tcdA and tcdB gene targets was analyzed in silico using the Basic Local Alignment Search Tool (BLAST). The targeted regions of tcdA and tcdB are generally well conserved, although the potential for false-negative ARIES C. difficile assay results due to sequence variation is noted as a Limitation in the device labeling. In silico analysis demonstrated that the ARIES C. difficile Assay is likely to detect strains of Toxinotype X although this was not demonstrated functionally.

e. Analytical specificity:

# Cross-reactivity Study

The analytical specificity of the ARIES C. difficile Assay was evaluated by testing a panel of 61 organisms and viruses that may be found in stool specimens (Table 6). Testing was performed in triplicate with each potentially cross-reactive species at $\overline { { \geq 1 0 } } ^ { 6 } \mathrm { C F U / m L }$ of Stool Resuspension Buffer for bacteria and yeast and $\geq 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } \mathrm { \dot { / } m L }$ for viruses (or the highest concentration attainable). Human DNA was also tested at a concentration of $5 \mu \mathrm { g / m L }$ of Stool Resuspension Buffer.

On initial testing, 1/3 replicates for $C .$ . bifermentans and $E .$ . coli were reported as positive for toxigenic C. difficile. Upon repeat testing of both organisms, 3/3 replicates gave negative results with the ARIES C. difficile Assay.

Invalid results were obtained on initial testing of A. baumannii (3/3 replicates) and $S .$ . marcescens (2/3 replicates) at ${ > } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . Both produced negative results when retested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ .

The results of the Cross-reactivity Study are noted in the device labeling.

Table 6. Organisms and viruses tested for potential cross-reaction in the ARIES $C .$ . difficile Assay   

<table><tr><td rowspan=1 colspan=4>Bacteria and Yeast (CFU/mL of Stool Resuspension Buffer)</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia defectiva</td><td rowspan=1 colspan=1>5.8 x 107</td><td rowspan=1 colspan=1>Enterococcus faecalis (vanB)</td><td rowspan=1 colspan=1>5.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii 1</td><td rowspan=1 colspan=1>4.8 x 107</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>9.8 x 106</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>7.5 x 107</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>5.2 x 108</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis subsp. faecalis</td><td rowspan=1 colspan=1>1.1 x 109</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>1.3 x 107</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus 2</td><td rowspan=1 colspan=1>4.9 x 106</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>4.7 x 108</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>2.4 x 108</td><td rowspan=1 colspan=1>Non-toxigenic Clostridium difficile (4)</td><td rowspan=1 colspan=1>0.3-1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>2.6 x 107</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>2.3 x 106</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni 2</td><td rowspan=1 colspan=1>1.4 x 106</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>1.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1.3 x 107</td><td rowspan=1 colspan=1>Porphyromonas asaccharolytica</td><td rowspan=1 colspan=1>3.7 x 106</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii ²</td><td rowspan=1 colspan=1>1.5 x 108</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>2.1 x 106</td></tr><tr><td rowspan=1 colspan=1>Clostridium bifermentans4</td><td rowspan=1 colspan=1>5.3 x 107</td><td rowspan=1 colspan=1>Proteus mirabilis 2</td><td rowspan=1 colspan=1>1.4 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium butyricum</td><td rowspan=1 colspan=1>2.2 x 107</td><td rowspan=1 colspan=1>Providencia alcalifaciens</td><td rowspan=1 colspan=1>2.1 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium haemolyticum 5</td><td rowspan=1 colspan=1>1.3 x 105</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2.0 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium novyi</td><td rowspan=1 colspan=1>1.4 x 108</td><td rowspan=1 colspan=1>Salmonella enterica (typhimurium)</td><td rowspan=1 colspan=1>6.0 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium orbiscindens 6</td><td rowspan=1 colspan=1>3.1 x 107</td><td rowspan=1 colspan=1>Salmonella enterica subsp. arizonae</td><td rowspan=1 colspan=1>5.8 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>5.3 x 106</td><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica</td><td rowspan=1 colspan=1>2.6 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium scindens</td><td rowspan=1 colspan=1>8.9 x 107</td><td rowspan=1 colspan=1>Serratia liquefaciens 2</td><td rowspan=1 colspan=1>5.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium septicum</td><td rowspan=1 colspan=1>8.1 x 106</td><td rowspan=1 colspan=1>Serratia marcescens 3</td><td rowspan=1 colspan=1>9.7 x 107</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellii</td><td rowspan=1 colspan=1>1.6 x 106</td><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>2.3 x 108</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>5.2 x 107</td><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>1.6 x 108</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O26:H4 2</td><td rowspan=1 colspan=1>1.8 x 108</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>1.2 x 108</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157:H7 4</td><td rowspan=1 colspan=1>2.1 x 108</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>5.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>4.4 x 107</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>8.8 x 108</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>8.3 x 107</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>3.0 x 108</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>1.1 x 108</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ΓCID50/mL of S</td><td rowspan=1 colspan=1>Viruses (TCID50/mL of Stool Resuspension Buffer)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus (Type 7A)</td><td rowspan=1 colspan=1>5.1 x 106</td><td rowspan=1 colspan=1>Enterovirus (Type 71) 5</td><td rowspan=1 colspan=1>2.1 x 104</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus (Type A16)</td><td rowspan=1 colspan=1>2.0 x 106</td><td rowspan=1 colspan=1>Norovirus Group I</td><td rowspan=1 colspan=1>4.3 x 106</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (Type AD-169)</td><td rowspan=1 colspan=1>5.7 x 105</td><td rowspan=1 colspan=1>Norovirus Group II</td><td rowspan=1 colspan=1>4.3 x 106</td></tr><tr><td rowspan=1 colspan=1>Echovirus Type 11</td><td rowspan=1 colspan=1>2.9 x 106</td><td rowspan=1 colspan=1>Rotavirus 5</td><td rowspan=1 colspan=1>8.5 x 103</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(μg/mL of Stool</td><td rowspan=1 colspan=2>Other (μg/mL of Stool Resuspension Buffer)</td></tr><tr><td rowspan=1 colspan=1>Human DNA 2</td><td rowspan=1 colspan=1>5</td><td></td><td></td></tr></table>

Note: Concentrations shown are per milliliter of Stool Resuspension Buffer 1 3/3 replicates reported as Invalid on initial testing; upon retest at $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ , 3/3 replicates reported as Negative 2 1/3 replicates reported as Invalid on initial testing; upon retest at the same concentration $1 / 1$ replicates reported as Negative 3 2/3 replicates reported as Invalid on initial testing; upon retest at $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ , $5 / 6$ replicates reported as Negative, 1/6 reported as Invalid 4 1/3 replicates reported as Positive; upon retest at the same concentration, 3/3 replicates reported as Negative 5 Tested at the highest concentration attainable 6 Flavonifactor plautii

# Bioinformatic Analysis

In silico analysis was performed to determine the potential for cross-reaction of the ARIES C. difficile Assay primers with C. botulinum which was not available for laboratory testing. No significant homology was observed and no cross-reaction with C. botulinum is predicted.

Contamination Study

The potential for false-positive results with the ARIES C. difficile Assay due to within run or between run cross-contamination was evaluated by testing an alternating series of C. difficile “high positive” $\mathrm { 1 0 ^ { 6 } C F U / m L }$ of Stool Resuspension Buffer) and negative samples in successive instrument runs. The expected results were obtained for all $C .$ . difficile positive and $C .$ . difficile negative samples (30/30 each). These results are acceptable.

# f. Assay cut-off:

The ARIES C. difficile Assay result algorithm uses a combination of parameters based on cycle threshold, amplicon melting temperature and fluorescence intensity for the tcdA and tcdB gene targets and SPC to report results as either toxigenic C. difficile Positive, Negative or Invalid. The algorithm parameters were established through Receiver Operator Characteristic (ROC) analysis using known toxigenic C. difficile positive and negative clinical specimens. The final parameters were validated in the prospective Clinical Study described in Section M(3)(a).

# g. Assay interference:

# Potentially Interfering Substances

The potential for interference with the ARIES C. difficile Assay was evaluated with endogenous and exogenous substances that may be present in stool specimens. Testing was performed in the presence of two strains of toxigenic C. difficile (ATCC BAA-1870 and BAA-1871, both of which are positive for both tcdA and tcdB+) as well as with C. difficile negative samples.

All C. difficile positive and negative samples produced the expected results, with the exception of those tested in the presence of $3 . 5 \%$ w/v mucin in which case 5/6 positive samples were incorrectly reported as toxigenic $C .$ . difficile negative (Table 7). However, when tested at a lower concentration $( 0 . 3 5 \% \mathrm { w / v ) }$ no interference in the presence of mucin was observed. The potential for false negative results in the presence of mucin ${ > } 0 . 3 5 \% \ \mathrm { w / v }$ is noted as a Limitation in the device labeling.

Table 7. Substances evaluated for potential interference with the ARIES $C .$ . difficile Assay   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration in Stool</td></tr><tr><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>1.3% w/v</td></tr><tr><td rowspan=1 colspan=1>Fecal fat (Triglyceride)</td><td rowspan=1 colspan=1>20.0% w/v</td></tr><tr><td rowspan=1 colspan=1>Fecal fat (Cholesterol)</td><td rowspan=1 colspan=1>4.9% w/v</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin (tarry stool)</td><td rowspan=1 colspan=1>12.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Hydrocortizone cream</td><td rowspan=1 colspan=1>2.0% w/v</td></tr><tr><td rowspan=1 colspan=1>Imodium 2</td><td rowspan=1 colspan=1>0.63% w/v</td></tr><tr><td rowspan=1 colspan=1>Kaopectate</td><td rowspan=1 colspan=1>0.1% w/v</td></tr><tr><td rowspan=1 colspan=1>Metronidazole 1, 3</td><td rowspan=1 colspan=1>140 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Moist towelettes (Benzalkonium chloride)</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucin 44</td><td rowspan=1 colspan=1>0.35% w/v</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol</td><td rowspan=1 colspan=1>0.1% w/v</td></tr><tr><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>2.0% w/v</td></tr><tr><td rowspan=1 colspan=1>Vagisil anti-itch cream</td><td rowspan=1 colspan=1>2.0% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>20% v/v</td></tr></table>

1 1/3 replicates for BAA-1871 was negative for tcdA 2 2/3 replicates for BAA-1870 were negative for tcdA Although positive, delayed Ct values were observed for tcdB with both BAA-1870 and BAA-1871 In the presence of $3 . 5 \% \ \mathrm { W / V }$ mucin in stool, 2/3 replicates for BAA-1870 and 3/3 replicates for BAA-1871 were reported as toxigenic $C .$ difficile negative. No interference was observed when mucin was tested at $0 . 3 5 \%$ w/v. 5 2/3 replicates for both BAA-1870 and BAA-1871 were negative for tcdA

# Microbial Interference

The potential for interference with the ARIES C. difficile Assay by organisms and viruses that may be present in stool specimens was investigated using the same list of species that was evaluated for potential cross-reactivity (refer to Section M(1)(e) and Table 6). Testing was performed in triplicate with each potentially interfering species in the presence of low levels of each of two strains of toxigenic $C .$ . difficile, close to the LOD of the assay. The potentially interfering species were tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ of Stool Resuspension Buffer for bacteria and yeast and $\dot { \geq } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses. Human DNA was also tested at a concentration of $5 \mu \mathrm { g / m L }$ of Stool Resuspension Buffer. For all replicates, the expected results were obtained with both strains of $C .$ . difficile with the exception of those listed in Table 8 for which repeat testing was performed. On initial testing with $C .$ . difficile strain ATCC BAA-1870, one false negative result was obtained in the presence of both $S .$ . marcescens and human DNA, although repeat testing produced the expected positive results. The results of the Microbial Interference Study are included in the device labeling and the potential for false negative results in the presence of very high concentrations of $S _ { \cdot }$ . marcescens is noted as a Limitation.

Table 8. Summary of unexpected results obtained in the Microbial Interference Study   

<table><tr><td rowspan=3 colspan=1>Species/Substance</td><td rowspan=1 colspan=6>ARIES C. difficile Assay Results: Initial (Repeat)</td></tr><tr><td rowspan=1 colspan=3>C. difficile ATCC BAA-1870</td><td rowspan=1 colspan=3>C. difficile ATCC BAA-1871</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3(1/1)</td><td rowspan=1 colspan=1>0/3(0/1)</td><td rowspan=1 colspan=1>1/3(0/1)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>2/3(1/1)</td><td rowspan=1 colspan=1>0/3(0/1)</td><td rowspan=1 colspan=1>1/3(0/1)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>2/3(1/1)</td><td rowspan=1 colspan=1>0/3(0/1)</td><td rowspan=1 colspan=1>1/3(0/1)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>2/3(3/3)</td><td rowspan=1 colspan=1>1/3(0/3)</td><td rowspan=1 colspan=1>0/3(0/3)</td><td rowspan=1 colspan=1>2/3(4/4)</td><td rowspan=1 colspan=1>0/3(0/4)</td><td rowspan=1 colspan=1>1/3(0/4)</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>2/3(2/3)</td><td rowspan=1 colspan=1>1/3(0/3)</td><td rowspan=1 colspan=1>0/3(1/3) 2</td><td rowspan=1 colspan=1>2/3(1/1)</td><td rowspan=1 colspan=1>0/3(0/1)</td><td rowspan=1 colspan=1>1/3(0/1)</td></tr></table>

1 Initially tested at $9 . 7 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L }$ ; retested at $\overline { { 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } } }$ 2 1/1 replicates produced a positive result upon additional retest

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity:

The performance of the ARIES C. difficile Assay was evaluated in a prospective Clinical Study at four geographically diverse sites in the US using left-over de-identified unpreserved, unformed stool specimens from patients suspected of having C. difficile infection. A total of 1021 specimens were enrolled in the study, of which 37 did not meet the prescribed criteria for inclusion in the analysis of performance [failure to meet patient inclusion criteria (2); absence of reference result (26); improper documentation (4); performance of the ARIES $C .$ . difficile Assay by untrained operators (3); incorrect specimen storage prior to ARIES $C .$ . difficile testing (2)]. Results from a total of 984 specimens were therefore available for analysis, of which 472 $( 4 8 . 0 \% )$ were from males and 512 $( 5 2 . 0 \% )$ were from females. The age distribution of subjects is shown in Table 9.

Table 9. Age distribution of subjects enrolled in the ARIES $C .$ difficile Clinical Study   

<table><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=1 colspan=1>≤2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>2-11</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>12-21</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>47.0</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>490</td><td rowspan=1 colspan=1>49.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>984</td><td rowspan=1 colspan=1>100</td></tr></table>

The results obtained with the ARIES $C .$ difficile Assay were compared to those obtained by direct and enriched toxigenic $C .$ . difficile culture performed at a central laboratory. Specimens for culture were transported/stored at ambient temperature in tubes containing Anaerobic Tissue Transport Medium (ATTM). All cultures were inoculated within 72 hours of specimen collection. The direct culture method comprised anaerobic incubation of cycloserine-cefoxitin-fructose agar (CCFA) plates for up to 48 hours, followed by phenotypic identification of suspected colonies of C. difficile and testing of confirmed isolates for cytotoxin production. For the enriched culture method, broth containing cycloserinecefoxitin-mannitol with taurocholate and lysozyme (CCM-TAL) was incubated anaerobically for 48 hours before sub-culture to CCFA plates. As for the direct culture method, colonies that were identified phenotypically as C. difficile were tested for cytotoxin production.

For the ARIES $C .$ . difficile Assay, the majority of specimens (875/984; $8 8 . 9 \%$ ) were stored refrigerated at $2 { - } 8 ^ { \circ } \mathrm { C }$ prior to testing. There were also 109 specimens $( 1 1 . 1 \% )$ for which testing could not be performed with 36 hours of collection, and which were therefore stored frozen at ${ } _ { - 8 0 } \mathrm { { } ^ { \circ } C }$ prior to analysis.

Forty-three (43) specimens $( 4 . 3 \% )$ required repeat testing with the ARIES C. difficile Assay due to initial Invalid results (18; $1 . 8 \%$ ), improper specimen storage/processing (10; $1 . 1 \% )$ , aborted runs $( 1 0 ; 1 . 1 \% )$ or specimen mix-up $( 5 ; 0 . 5 \% )$ . Upon re-testing, 38 specimens were reported as either toxigenic $C .$ . difficile positive or negative but five specimens still produced Invalid results for a final Invalid rate of $0 . 5 \%$ (5/984). The number of specimens available for evaluation of performance was therefore 979. The performance of the ARIES C. difficile Assay in comparison to direct toxigenic culture and to the combined results from direct and enriched toxigenic culture is shown in Tables 10 and 11, respectively, and was found to be acceptable.

Table 10. Clinical performance of the ARIES C. difficile Assay in comparison to direct toxigenic culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Direct Toxigenic Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>ARIESC. difficileAssay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>651</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>$2{}$</td><td rowspan=1 colspan=1>809</td><td rowspan=1 colspan=1>811</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>874</td><td rowspan=1 colspan=1>979 3</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>103/105 = 98.1% (95% CI: 93.3-99.5%)</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>809/874 = 92.6% (95% CI: 90.6-94.1%)</td></tr></table>

30/65 $( 4 6 . 2 \% )$ specimens were positive by enriched culture and 33/65 $( 5 0 . 8 \% )$ were positive by a validated PCR/bidirectional sequencing assay (including 18 that were also positive by enriched culture); in total, 45/65 $( 6 9 . 2 \% )$ specimens were positive by enriched culture and/or PCR/bidirectional sequencing 2 1/2 $( 5 0 \% )$ specimens was positive by enriched culture and by a validated PCR/bidirectional sequencing assay 3 Five (5) specimens yielded Invalid results after repeat testing with the ARIES $C .$ . difficile Assay and were excluded from the analysis; of these, one (1) specimen was positive by direct toxigenic culture

Table 11. Clinical performance of the ARIES $C .$ difficile Assay in comparison to direct and enriched toxigenic culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Direct and Enriched Toxigenic Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>ARIESC. difficileAssay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>811</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>832</td><td rowspan=1 colspan=1>979 3</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>133/147 = 90.5% (95% CI: 84.6-94.2%)</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>797/832 = 95.8% (95% CI: 94.2-97.0%)</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>133/168 = 79.2%</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>797/811 = 98.3%</td></tr></table>

1 15/35 $( 4 2 . 9 \% )$ specimens were positive by a validated PCR/bidirectional sequencing assay 2 1/14 $( 7 . 1 \% )$ specimens was positive by a validated PCR/bidirectional sequencing assay 3 Five (5) specimens yielded Invalid results after repeat testing with the ARIES $C .$ . difficile Assay and were excluded from the analysis; of these, 1 was positive by direct and enriched toxigenic culture

The performance of the ARIES C. difficile Assay at each clinical site in comparison to direct toxigenic culture and to the combined results from direct and enriched toxigenic culture is shown in Tables 12 and 13, respectively.

Table 12. Performance of the ARIES C. difficile Assay by clinical site in comparison to direct toxigenic culture   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Samples (%)</td><td rowspan=2 colspan=1>Direct ToxigenicCulture Positive (%)</td><td rowspan=1 colspan=2>Percent Agreement(95% Score Confidence Interval)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>206(21.0)</td><td rowspan=1 colspan=1>31(15.0)</td><td rowspan=1 colspan=1>96.8(83.8-99.4)</td><td rowspan=1 colspan=1>92.6(87.7-95.6)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>1841(18.8)</td><td rowspan=1 colspan=1>19(11.5)</td><td rowspan=1 colspan=1>94.7(75.4-99.1)</td><td rowspan=1 colspan=1>93.9(89.2-96.7)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>190(19.4)</td><td rowspan=1 colspan=1>18(9.5)</td><td rowspan=1 colspan=1>100(82.4-100)</td><td rowspan=1 colspan=1>91.9(86.8-95.1)</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>399(40.8)</td><td rowspan=1 colspan=1>37(9.3)</td><td rowspan=1 colspan=1>100(90.6-100)</td><td rowspan=1 colspan=1>92.3(89.0-94.6)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>979(100)</td><td rowspan=1 colspan=1>105(10.7)</td><td rowspan=1 colspan=1>98.1(93.3-99.5)</td><td rowspan=1 colspan=1>92.6(90.6-94.1)</td></tr></table>

Five (5) samples that produced ARIES $C$ . difficile Assay Invalid results were excluded from the analysis of performance: Site A: 1 (direct toxigenic culture positive); Site B: 1 (direct toxigenic culture negative); Site C: 3 (all direct toxigenic culture negative)

Table 13. Performance of the ARIES $C .$ . difficile Assay by clinical site in comparison to direct and enriched toxigenic culture   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Samples (%)</td><td rowspan=2 colspan=1>Direct/EnrichedToxigenicCulture Positive (%)</td><td rowspan=1 colspan=2>Percent(95% Score Confidence Interval)</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>206(21.0)</td><td rowspan=1 colspan=1>45(21.8)</td><td rowspan=1 colspan=1>82.2(68.7-90.7)</td><td rowspan=1 colspan=1>96.3(92.1-98.3)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>1841(18.8)</td><td rowspan=1 colspan=1>25(13.6)</td><td rowspan=1 colspan=1>92.0(75.0-97.8)</td><td rowspan=1 colspan=1>96.9(92.9-98.7)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>190(19.4)</td><td rowspan=1 colspan=1>24(12.6)</td><td rowspan=1 colspan=1>95.8(79.8-99.3)</td><td rowspan=1 colspan=1>94.6(90.0-97.1)</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>399(40.8)</td><td rowspan=1 colspan=1>53(13.3)</td><td rowspan=1 colspan=1>94.3(84.6-98.1)</td><td rowspan=1 colspan=1>95.7(93.0-97.4)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>979(100)</td><td rowspan=1 colspan=1>147(15.0)</td><td rowspan=1 colspan=1>90.5(84.6-94.2)</td><td rowspan=1 colspan=1>95.8(94.2-97.0)</td></tr></table>

Five (5) samples that produced ARIES $C .$ . difficile Assay Invalid results were excluded from the analysis of performance: Site A: 1 (direct and enriched toxigenic culture positive); Site B: 1 (direct and enriched toxigenic culture negative); Site C: 3 (all direct and enriched toxigenic culture negative)

ARIES C. difficile Assay performance was evaluated separately for fresh and frozen specimens and the results are shown in Table 14. Although there was a difference in positive percent agreement between fresh and frozen specimens in comparison to direct toxigenic culture, performance in relation to direct and enriched toxigenic culture in combination was similar for the two types of sample. This is acceptable.

Table 14. Performance of the ARIES C. difficile Assay with refrigerated or frozen stool specimens   

<table><tr><td rowspan=1 colspan=6>Comparison to Direct Toxigenic Culture</td></tr><tr><td rowspan=2 colspan=1>SampleStorage</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=2>Culture (%)</td><td rowspan=1 colspan=2>Percent Agreement(95% Score Confidence Interval)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Refrigerated</td><td rowspan=1 colspan=1>875</td><td rowspan=1 colspan=1>92(10.5)</td><td rowspan=1 colspan=1>783(89.5)</td><td rowspan=1 colspan=1>100(96.0-100)</td><td rowspan=1 colspan=1>92.8(90.8-94.5)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>13(12.5)</td><td rowspan=1 colspan=1>91(87.5)</td><td rowspan=1 colspan=1>84.6(57.8-95.7)</td><td rowspan=1 colspan=1>90.1(82.3-94.7)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>105(10.7)</td><td rowspan=1 colspan=1>796(81.3)</td><td rowspan=1 colspan=1>98.1(93.3-99.5)</td><td rowspan=1 colspan=1>92.6(90.6-94.1)</td></tr><tr><td rowspan=1 colspan=6>Comparison to Direct/Enriched Toxigenic Culture</td></tr><tr><td rowspan=2 colspan=1>SampleStorage</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=2>Culture (%)</td><td rowspan=1 colspan=2>Percent(95% Score Confidence Interval)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>Refrigerated</td><td rowspan=1 colspan=1>875</td><td rowspan=1 colspan=1>128(14.6)</td><td rowspan=1 colspan=1>747(85.4)</td><td rowspan=1 colspan=1>90.6(84.3-94.6)</td><td rowspan=1 colspan=1>95.7(94.0-97.0)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>19(18.3)</td><td rowspan=1 colspan=1>85(81.7)</td><td rowspan=1 colspan=1>89.5(68.6-97.1)</td><td rowspan=1 colspan=1>96.5(90.1-98.8)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>147(15.0)</td><td rowspan=1 colspan=1>832(85.0)</td><td rowspan=1 colspan=1>90.5(84.6-94.2)</td><td rowspan=1 colspan=1>95.8(94.2-97.0)</td></tr></table>

Storage condition prior to testing with the ARIES C. difficile Assay; all cultures were performed on stool specimens stored at room temperature in Anaerobic Tissue Transport Medium

b. Clinical specificity: Refer to Section M(3)(a), above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

The prevalence of C. difficile in the prospective Clinical Study described in Section M(3)(a) is shown Table 15, stratified by patient age and clinical setting. Overall prevalence as determined by the ARIES C. difficile Assay was $1 7 . 2 \%$ (168/979) whereas by the reference enriched culture method it was $1 5 . 0 \%$ (147/979).

Table 15. Prevalence of $C .$ . difficile in the prospective Clinical Study as determined by the ARIES C. difficile Assay   

<table><tr><td rowspan=2 colspan=1>Age Range(years)</td><td rowspan=1 colspan=4>% Prevalence (Positive/Tested)</td></tr><tr><td rowspan=1 colspan=1>Hospital</td><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>Other 1</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>≤2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>(0/1)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0(0/1)</td></tr><tr><td rowspan=1 colspan=1>2-11</td><td rowspan=1 colspan=1>0(0/2)</td><td rowspan=1 colspan=1>0(0/1)</td><td rowspan=1 colspan=1>0(0/1)</td><td rowspan=1 colspan=1>0(0/4)</td></tr><tr><td rowspan=1 colspan=1>12-21</td><td rowspan=1 colspan=1>38.5(5/13)</td><td rowspan=1 colspan=1>7.1(1/14)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>22.2(6/27)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>14.3(33/230)</td><td rowspan=1 colspan=1>18.5(41/222)</td><td rowspan=1 colspan=1>0(0/8)</td><td rowspan=1 colspan=1>16.1(74/460)</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>16.8(39/232)</td><td rowspan=1 colspan=1>18.3(44/241)</td><td rowspan=1 colspan=1>35.7(5/14)</td><td rowspan=1 colspan=1>18.1(88/487)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>16.1(77/477)</td><td rowspan=1 colspan=1>18.0(86/479)</td><td rowspan=1 colspan=1>21.7(5/23)</td><td rowspan=1 colspan=1>17.2(168/979)</td></tr></table>

NA: Not applicable 1 Emergency Department or Extended Care Facility

# N. Instrument Name:

ARIES Systems (ARIES System or ARIES M1 System)

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification: Specimen identification numbers can be entered manually or using a barcode scanner.

4. Specimen Sampling and Handling:

A portion of the test sample is transferred manually to a tube for pre-processing (resuspension in buffer and centrifugation). An aliquot of the supernatant from the preprocessed sample is added manually to the ARIES C. difficile Assay test cassette for automated nucleic acid extraction, PCR amplification/detection and result interpretation.

5. Calibration:

Calibration is performed by Luminex service personnel using ARIES System Verification Cassettes.

6. Quality Control: Refer to Section M(1)(c).

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.