Vela Diagnostics USA Inc.   
Donald Henton   
Dir. Regulatory Affairs North America 353C US Route 46 West   
Suite 250   
Fairfield, New Jersey 07004

Re: K172509 Trade/Device Name: Sentosa SA201 HSV 1/2 Qualitative PCR Test Regulation Number: 21 CFR 866.3305 Regulation Name: Herpes simplex virus serological assays Regulatory Class: Class II Product Code: OQO Dated: December 29, 2017 Received: January 4, 2018

Dear Donald Henton:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR

Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice   
(https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/) and CDRH Learn   
(http://www.fda.gov/Training/CDRHLearn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (http://www.fda.gov/DICE) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

# Sincerely, Uwe Scherf -S

Uwe Scherf, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

<table><tr><td>DEPARTMENT OF HEALTH AND HUMAN SERVICES Food and Drug Administration Indications for Use</td><td>Form Approved: OMB No. 0910-0120 Expiration Date: January 31, 2017 See PRA Statement below.</td></tr><tr><td colspan="2">510(k) Number (if known) K172509</td></tr><tr><td colspan="2"></td></tr><tr><td colspan="2">Device Name Sentosa® SA201 HSV-1/2 PCR Test</td></tr><tr><td colspan="2"></td></tr></table>

Indications for Use (Describe)

# Intended Use

The Sentosa® SA201 HSV-l/2 PCR Test is a real-time PCR-based qualitative in vitro diagnostic test for detection and differentiation of Herpes Simplex Virus (HSV-l and HSV-2) DNA from male and female skin lesions from anogenital or oral sites. The test is intended for use as an aid in diagnosis of herpes infection in symptomatic patients.

Warning: The Sentosa® SA201 HSV-l/2 PCR Test is not FDA cleared for use with cerebrospinal fluid (CSF). The test is not intended to be used for prenatal screening.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 510(k) Summary

Date summary prepared: 1/31/2018

# 510(k) Submitter/Holder

Vela Diagnostics USA, Inc. 353C US Route 46 West Suite 250 Fairfield, NJ 07004

# Contact

Donald Henton   
Director Regulatory Affairs North America   
Telephone: 973-369-3578   
Fax: 973-521-7077   
Email: donald.henton@veladx.com

# Name of Device

Trade Name: Sentosa® SA201 HSV-1/2 PCR Test   
Catalog Numbers: 300216   
Common Name: Herpes Simplex Virus Nucleic Acid Amplification Assay   
Classification Name: Herpes simplex virus serological assays (21 CFR $\ S$ 866.3305, Class II, OQO).

# Predicate Device

The Sentosa® SA201 HSV-1/2 PCR Test was compared to and found to be substantially equivalent to the following product of comparable type in commercial distribution:

Trade Name: IMDx HSV-1/2 for Abbott m2000 Assay   
Common Name: Herpes Simplex Virus Nucleic Acid Amplification Assay 510(k) Number: K140198 (cleared 05/13/2014)   
Manufacturer: Intelligent Medical Devices, Inc.

Table 1. Sentosa® SA201 HSV-1/2 PCR Test and Predicate Similarities Comparison   

<table><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="5" rowspan="1">Predicate</td><td colspan="1" rowspan="1">Similarities /Differences</td></tr><tr><td colspan="1" rowspan="1">Characteristics</td><td colspan="1" rowspan="1">SentosaSA201 HSV1/2PCR Test (K172509)</td><td colspan="5" rowspan="1">IMDx HSV-1/2 forAbbott m2000 Assay(K140198)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3305</td><td colspan="5" rowspan="1">21 CFR 866.3305</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OQO</td><td colspan="5" rowspan="1">OQO</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="5" rowspan="1">Class II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="4">Intended use</td><td colspan="1" rowspan="4">The Sentosa® SA201HSV-1/2 PCR Test is areal-time     PCR-basedqualitative   in   vitrodiagnostic   test   fordetection               anddifferentiation of HerpesSimplex Virus(HSV-1and HSV-2) DNA frommaleand female skinlesions from anogenitalor oral sites. The test isintended for use as an aidin diagnosis of herpesinfection in symptomaticpatients.Warning: The Sentosa®SA201 HSV-1/2 PCRTest is not FDA clearedfor       use        withcerebrospinal        fluid(CSF). The test is notintended to be used forprenatal screening.</td><td colspan="5" rowspan="4">TheIMDxHSV-1/2for Abbott  m2000assayis anin vitrodiagnostic test for thedirect,      qualitativedetection           anddifferentiation      ofHerpes Simplex Virustype 1 (HSV-1) andtype 2 (HSV-2) DNAfrom male and femaleskin  lesions   fromanogenital  or  oralsites.    Thetest isintended for use as anaid in the diagnosis ofHSV  infection   insymptomaticpatients.The assay is intendedto be run on the Abbottm2000     instrumentsystem.Warning:The IMDxHSV-1/2forAbbottm2000assayisnotFDA-cleared for usewith     cerebrospinalfluid (CSF). The assayis not intended for pre-natal screening.</td><td colspan="1" rowspan="4">The intended useis  the   sameexcept the notedsystems they aredesigned to berun on.The warningisthe same.</td></tr><tr><td colspan="2" rowspan="2"></td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">or  oral</td></tr><tr><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Real-time PCR DNAamplification</td><td colspan="5" rowspan="1">Real-time PCR DNAamplification</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detectionand differentiation ofHSV-1 and HSV-2DNA</td><td colspan="1" rowspan="1">Qualitative detectionand differentiation ofHSV-1 and HSV-2DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Male and female skinlesions from anogenitalor oral sites</td><td colspan="1" rowspan="1">Male and female skinlesions fromanogenital or oral sites</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Device Description

The Sentosa® SA201 HSV-1/2 PCR Test is a $( 4 \mathrm { x } 2 4 )$ configuration contains reagents and enzymes for specific amplification of a 104 bp (base-pair) fragment of the UL30 gene common to both HSV1 and HSV2, and specific probes for the direct detection and differentiation of HSV1 and HSV2 amplicons, respectively. Pathogen detection by PCR is based on the amplification of specific regions of the pathogen genome. In real-time PCR, the amplified product is detected via fluorescent dyes, which are usually linked to oligonucleotide probes that bind specifically to the target sequences. Real-time monitoring of the fluorescence intensities during a PCR run allows the detection of the accumulating product. Amplification of the targets occurs in three channels: green, orange and red on the Sentosa® SA201. Output is recorded as the increase of fluorescence over time in comparison to background signal. Monitoring the fluorescence intensities during the PCR run allows the detection of the accumulating product without having to re-open the reaction tubes after the PCR run.

The Sentosa® SA201 HSV-1/2 PCR Test workflow starts with extraction of nucleic acids from samples (anogenital or oral swabs) using the Sentosa® SX Virus Total Nucleic Acid Kit on the Sentosa® SX101 instrument. Following extraction, the instrument will automatically set up the PCR with the extracted nucleic acids in a 96-well PCR plate. Subsequently, the 96-well PCR plate is sealed and transferred to the Sentosa ® SA201 for PCR amplification, followed by data analysis.

The Sentosa® Link facilitates data transfer between the Sentosa® SX101, the Sentosa® SA201 Reporter and existing LIS/LIMS (laboratory information systems) in the clinical lab. The Sentosa® SX101 instrument communicates with Sentosa® SA201 thermocycler. This creates a user environment that links the SX101 and the Sentosa® SA201 to facilitate automated workflow to export results in a LIS/LIMS-compatible format.

# Intended Use

The Sentosa® SA201 HSV-1/2 PCR Test is a real-time PCR-based qualitative in vitro diagnostic test for detection and differentiation of Herpes Simplex Virus (HSV-1 and HSV-2) DNA from male and female skin lesions from anogenital or oral sites. The test is intended for use as an aid in diagnosis of herpes infection in symptomatic patients.

# Technological Characteristics

The Sentosa® SA201 HSV-1/2 PCR Test contains reagents and enzymes for specific amplification of a 104 bp fragment of UL30 gene common to both HSV1 and HSV2, and specific probes for the direct detection and differentiation of HSV1 and HSV2 amplicons, respectively. Pathogen detection by PCR is based on the amplification of specific regions of the pathogen genome. In real-time PCR, the amplified product is detected via fluorescent dyes, which are usually linked to oligonucleotide probes that bind specifically to the target sequences. Real-time monitoring of the fluorescence intensities during a PCR run allows the detection of the accumulating product. Amplification of the targets occurs in three channels: green, orange and red on the Sentosa® SA201. Output is recorded as the increase of fluorescence over time in comparison to background signal. Monitoring the fluorescence intensities during the PCR run allows the detection of the accumulating product without having to re-open the reaction tubes after the PCR run.

The Sentosa® SA201 HSV-1/2 PCR Test workflow starts with extraction of nucleic acids from samples (anogenital or oral swabs) using the Sentosa® SX Virus Total Nucleic Acid Kit on the Sentosa® SX101 instrument. Following extraction, the instrument will automatically set up the PCR with the extracted nucleic acids in a 96-well PCR plate. Subsequently, the 96-well PCR plate is sealed and transferred to the Sentosa® SA201 for PCR amplification, followed by data analysis.

Drivers are installed on the Sentosa® Link to connect the Sentosa SX101 instrument and the Sentosa® SA201 thermocycler. This creates a user environment that links the SX101 and the Sentosa® SA201 to facilitate automated workflow to export results in a LIS/LIMS-compatible format.

The Sentosa® SA201 HSV-1/2 PCR Test uses the Sentosa® SX101 hardware and Sentosa® SA201 hardware, along with the associated consumables to operate as an automated sample extraction, PCR setup, amplification and reporting system (referred to as workflow for short).

# Sentosa® SA201 HSV1/2 PCR Test

![](images/19b4a29afd03f104f5a3490f6c1fe98a1860c3799b16b41eed5a0681227e825d.jpg)  
Figure 1. Sentosa® SA201 HSV-1/2 PCR Test Workflow Overview.

![](images/5450cafaeea86755527bc7debbda0ed82405e4bd6a05d71a0fdf5863f0794f2b.jpg)  
Figure 2. Sentosa® SA201 HSV-1/2 PCR Test Workflow Flowchart.

# Performance

# Analytical Studies:

1. Analytical sensitivity: The limit of detection (LoD) was assessed for the Sentosa® SA201 HSV-1/2 PCR Test using two strains of HSV-1 (MacIntyre and KOS) and two strains of HSV-2 (MS and G). The LoD is defined as the HSV titer $\mathrm { ( T C I D 5 0 / m L ) }$ ) detected with a probability of $9 5 \%$ or greater and was determined by Probit analysis using HSV-1 MacIntyre and HSV-2 MS. The results of the analytical sensitivity of the Sentosa® SA201 HSV-1/2 PCR Test are summarized in the following table.

Table 2. Analytical sensitivity - Limit of Detection (LoD) HSV-1 and HSV-2   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>HSV-1 MacIntyre</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>HSV-1 KOS</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>4</td></tr></table>

2. Precision: The study was conducted over a period of five (5) days with three (3) reagent lots, five (5) operators, and four (4) instrument systems at Vela Research Singapore. The test materials used in the study included NC, PC, and negative sample, 3xLoD HSV1, 1.5xLoD HSV1, 3xLoD HSV2 and 1.5xLoD HSV2. Each concentration was assayed three (3) times per run for 20 runs, resulting in 60 replicates (three (3) replicates/run x four (4) run/day x five (5) days x one (1) site $= 6 0$ replicates). The precision analysis was based on $\% \mathrm { C V }$ of Ct values, and is presented in Table 3 below. The quality control analysis was based on the mean and standard deviations (SD) of the $\mathrm { C t }$ values. The demonstrated agreement with the results being $100 \%$ for all tested types, and the $\% \mathrm { C V }$ being (less than) $< 1 0 \%$ .

Table 3. Precision   

<table><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Channel</td><td colspan="1" rowspan="1">Agreement(%)</td><td colspan="1" rowspan="1">95%CI*</td><td colspan="1" rowspan="1">Mean Ct</td><td colspan="1" rowspan="1">SD</td></tr><tr><td colspan="1" rowspan="1">1.5x LoD HSV-1a</td><td colspan="1" rowspan="1">Green</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">31.79</td><td colspan="1" rowspan="1">0.81</td></tr><tr><td colspan="1" rowspan="1">3x LoD HSV-1 a</td><td colspan="1" rowspan="1">Green</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">30.72</td><td colspan="1" rowspan="1">0.90</td></tr><tr><td colspan="1" rowspan="1">1.5x LoD HSV-2 b</td><td colspan="1" rowspan="1">Orange</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">30.25</td><td colspan="1" rowspan="1">1.09</td></tr><tr><td colspan="1" rowspan="1">3x LoD HSV-2 b</td><td colspan="1" rowspan="1">Orange</td><td colspan="1" rowspan="1">59/59(100%)*</td><td colspan="1" rowspan="1">93.89% -100%</td><td colspan="1" rowspan="1">29.35</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="1">NC</td><td colspan="1" rowspan="1">Red</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">29.08</td><td colspan="1" rowspan="1">2.24</td></tr><tr><td colspan="1" rowspan="1">PC</td><td colspan="1" rowspan="1">Green</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">26.09</td><td colspan="1" rowspan="1">1.10</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Orange</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">28.51</td><td colspan="1" rowspan="1">0.69</td></tr><tr><td colspan="1" rowspan="1">Negative sample</td><td colspan="1" rowspan="1">Red</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">93.98% -100%</td><td colspan="1" rowspan="1">28.98</td><td colspan="1" rowspan="1">2.03</td></tr></table>

$^ { \mathrm { a } } 1 . 5 \mathrm { x }$ LoD HSV-1 is $6 0 \mathrm { T C I D } 5 0 / \mathrm { m L }$ and $3 \mathbf { x }$ LoD HSV-1 is 120 TCID50/mL $^ { \mathrm { b } } 1 . 5 \mathrm { x }$ LoD HSV-2 is $6 ~ \mathrm { T C I D } 5 0 / \mathrm { m L }$ and $3 \mathbf { x }$ LoD HSV-2 is $1 2 \mathrm { T C I D } 5 0 / \mathrm { m L }$ \* Total number of samples is less than 60 due to exclusion of 1 invalid sample

3. Reproducibility: The Reproducibility Study was to demonstrate the performance of the Sentosa® SA201 HSV-1/2 PCR Test using simulated samples (with spiked-in HSV1/2 virus) using multiple sites, multiple instruments, multiple operators, and multiple lots using the matrix. The study demonstrated excellent agreement across the sites, operators, and systems with the results meeting the acceptance for all the tested types, and the $\% \mathrm { C V }$ being (less than) $<$ $10 \%$ . The results are summarized in the following Table 4.

Table 4. Reproducibility   

<table><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% CI*</td><td rowspan=1 colspan=1>Mean Ct ± SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-1a</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>31.45 ±1.65</td><td rowspan=1 colspan=1>5.25%</td></tr><tr><td rowspan=1 colspan=1>3x LoD HSV-1a</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>30.23 ±1.01</td><td rowspan=1 colspan=1>3.34%</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-2b</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>29.29 ±0.47</td><td rowspan=1 colspan=1>1.60%</td></tr><tr><td rowspan=1 colspan=1>3x LoD HSV-2b</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>28.01 ±1.18</td><td rowspan=1 colspan=1>4.21%</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>26.93 ±0.60</td><td rowspan=1 colspan=1>2.23%</td></tr><tr><td rowspan=2 colspan=1>PC</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>25.80 ±0.26</td><td rowspan=1 colspan=1>1.01%</td></tr><tr><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>28.32 ±0.32</td><td rowspan=1 colspan=1>1.13%</td></tr><tr><td rowspan=1 colspan=1>Negative sample</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>27.17 ±1.14</td><td rowspan=1 colspan=1>4.20%</td></tr><tr><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>95.91% - 100%</td><td rowspan=1 colspan=1>26.81 ±0.55</td><td rowspan=1 colspan=1>2.05%</td></tr></table>

4. Analytical Reactivity / Cross-reactivity: The Analytical Reactivity and Specificity Study was to demonstrate the analytical reactivity and cross-reactivity (analytical specificity) performance of the Sentosa® SA201 HSV-1/2 PCR Test with the Sentosa® workflow using high and low concentration HSV1 and HSV2 samples (with competing organisms). The results in the study showed that the 55 organisms, which are closely related to HSV1/2 or present in oral or genital swab samples at high concentrations showed no cross reactivity to HSV1/2 detection using the Sentosa® SA201 HSV-1/2 PCR Test. It also showed no interference with the detection of low concentration levels of HSV1 or HSV2. The results do suggest that C. glabrata, S. aureus, S. epidermidis, P. melaninogenica and HHV6 might lead to slight competitive inhibition from HSV1 to HSV2. The results also showed no cross-reactivity within the multiplex panel (HSV1 and HSV2) in the presence of either high concentration HSV1 or high concentration HSV2 strains.

Table 5. The Analytical Reactivity / Cross-reactivity tested organisms   

<table><tr><td rowspan=1 colspan=2>Potential interfering organisms</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata, NCYC 388</td><td rowspan=1 colspan=1>Human papillomavirus type 16</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii, 2208</td><td rowspan=1 colspan=1>Human papillomavirus type 18</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Human DNA</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwofii</td><td rowspan=1 colspan=1>Human herpesvirus (HHV6)</td></tr><tr><td rowspan=1 colspan=1>Actinomyces isralii, serotype 1</td><td rowspan=1 colspan=1>Human herpesvirus 4, B95-8</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1, strain Adenoid 71</td><td rowspan=1 colspan=1>Klebsiella pneumonia, NCTC 9633</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7, strain Gomen</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis, VPI2553</td><td rowspan=1 colspan=1>Maraxella catarrhalis, strain 20</td></tr><tr><td rowspan=1 colspan=1>Candida albicans, strain 132</td><td rowspan=1 colspan=1>Mobiluncus curtisii, BV 345-16</td></tr><tr><td rowspan=1 colspan=1>Candida krusei, NRRL Y-6</td><td rowspan=1 colspan=1>Mobiluncus muleris, BV 64-5</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis, NRRL-Y-12969</td><td rowspan=1 colspan=1>Mycoplasma hominis, PG21</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis, PK233</td><td rowspan=1 colspan=1>Neisseria gonorrhea, B-585</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis, UW-57/Cx</td><td rowspan=1 colspan=1>Neisseria meningitides, serogroup B</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile, strain 4118</td><td rowspan=1 colspan=1>Prevotella melaninogenica, B282</td></tr><tr><td rowspan=1 colspan=1>CMV, AD-169</td><td rowspan=1 colspan=1>Rubella virus</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium, 392-1</td><td rowspan=1 colspan=1>Staphylococcus aureus, F-182</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans, 52</td><td rowspan=1 colspan=1>Staphylococcus aureus, FDA 209</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae, QC strain</td><td rowspan=1 colspan=1>Staphylococcus epidermidis, 255-01B</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis, AGR329</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus,LRA27.02</td></tr><tr><td rowspan=1 colspan=1>Enterovirus type 71, BrCr</td><td rowspan=1 colspan=1>Streptococcus mitis, NCTC 12261</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus/Human herpesvirus 4,strain P-3</td><td rowspan=1 colspan=1>Streptococcus mutans, UA159</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O103, strain NCDCH515b</td><td rowspan=1 colspan=1>Streptococcus pneumonia, CIP104225</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum subspnecrophorum, VPI2553</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum subspnucleatum, 1612A</td><td rowspan=1 colspan=1>SV40 (Simian virus 40)</td></tr><tr><td rowspan=1 colspan=1>Gardnerella/Haemophilus vaginalis, 317</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Haemophilus ducreyi, CIP 542</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Hepatitis A virus, strain PA21</td><td rowspan=1 colspan=1>Varicella-Zoster Virus (VZV)</td></tr><tr><td rowspan=1 colspan=1>HIV-1, Group M Subtype C</td><td rowspan=1 colspan=1></td></tr></table>

5. Interfering Substances: The Interfering Substances and Competitive Interference Study to evaluate the effect of potentially interfering substances and competitive interference on the performance of the Sentosa® SA201 HSV1/2 PCR Test was conducted over a period of ten (10) days with one (1) reagent lot, five (5) operators, and four (4) instrument workflow systems. The test materials used in this study included 31 substances commonly found in genital and oral specimens, which were dissolved into virus transport media (VTM) at approximately 3xLoD for both HSV1 and HSV2 to approximately 10 times of the normal active concentration. Each material was assayed in triplicate. The highest concentration for each substance with no interfering effect on device performance was reported. The test materials used in the competitive interference study were HSV1 and HSV2 spiked-in samples with different combinations of HSV1 and HSV2 concentrations. Each combination was tested in 20 replicates. The virus with the highest concentration that resulted in $100 \%$ detection of another virus was recorded, as well as the highest equal concentration combination of the two viruses. No interfering effects on the performance at concentration levels higher (5 to 10 times) than the normal active concentration of the interference substances tested. The HSV1 concentration that resulted in $100 \%$ detection rate of 3xLoD HSV2 was at least 50xLoD. The HSV2 concentration that resulted in $100 \%$ detection rate of 3xLoD HSV1 was at least 10xLoD but less than 50xLoD. No mutual interference was observed in HSV1 and HSV2 equal concentration combinations to at least 500xLoD of each virus.

Table 6. Interfering Substances and Competitive Interference Study tested substances   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Active Ingredients</td><td colspan="1" rowspan="1">Tested concentration</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">Acycloguanosine (10%)</td><td colspan="1" rowspan="1">7mg/mL</td></tr><tr><td colspan="1" rowspan="1">Whole blood with EDTA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Female urine</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Male urine</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">3.3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Saliva</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">K.Y. Jelly lubricant</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Feminine wash</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Xylocaine 5%</td><td colspan="1" rowspan="1">Lidocaine (50mg)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Stannous fluoride (0.454%)</td><td colspan="1" rowspan="1">0.53% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Desitin maximum originalpaste</td><td colspan="1" rowspan="1">Zinc Oxide (40%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Mentholatum lip balm</td><td colspan="1" rowspan="1">Menthol (0.7%), Camphor(1.7%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Listerine anti-bacterialmouthwash</td><td colspan="1" rowspan="1">Eucalyptol (0.092%), Menthol(0.042%), Methy Salicylate(0.060%), Thymol (0.064%)</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Douche</td><td colspan="1" rowspan="1">Providone-iodine (10% w/v)</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">YeastGard</td><td colspan="1" rowspan="1">Candida albicans 27X* HPUS,Candida parapsilosis 27X*HPUSPulsatilla 27* HPUS</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Vaginal contraceptive gel</td><td colspan="1" rowspan="1">Nonoxynol-9 (4%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Vaginal contraceptive gel</td><td colspan="1" rowspan="1">Nonoxynol-9 (3%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Monistat-7</td><td colspan="1" rowspan="1">Miconazole nitrate vaginalcream (2%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Fleet</td><td colspan="1" rowspan="1">Benzalkonium (0.06%w/w)/EdTA</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Gyno-Trosyd</td><td colspan="1" rowspan="1">Tioconazole (100mg)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole 1% Cream</td><td colspan="1" rowspan="1">Clotrimazole (50mg, 1%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Anti-Inch Cream</td><td colspan="1" rowspan="1">Lidocaine ph. Eur. (2% w/w)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Abreva</td><td colspan="1" rowspan="1">Docosanol (10%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Buffy coat</td><td colspan="1" rowspan="1">White blood cell</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Talcum powder</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Seminal fluid</td><td colspan="1" rowspan="1">Seminal fluid</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Feces</td><td colspan="1" rowspan="1">Feces</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Corn starch</td><td colspan="1" rowspan="1">Corn starch</td><td colspan="1" rowspan="1">1.25 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Paracetamol</td><td colspan="1" rowspan="1">Acetamidophenol</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Aspirin</td><td colspan="1" rowspan="1">Acetylsalicyclic acid</td><td colspan="1" rowspan="1">10 mg/mL</td></tr></table>

6. Carry-over and Cross-contamination: The Cross-contamination and Carry-over studies were to evaluate the potential cross-contamination during extraction and on subsequent runs on the performance of the Sentosa ® SA201 HSV1/2 Test with the Sentosa® SX101 workflow instrument in accordance with CLSI MM17-A (2008). These carry-over and crosscontamination studies were conducted over a period of four (4) days with one (1) reagent lot, two (2) operators, and two (2) Sentosa® workflow systems. The test materials used in the study included negative samples and $1 \mathrm { x } 1 0 ^ { \mathord { \wedge } 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ HSV1 (2500xLoD) for the crosscontamination study, a NC, a PC, and negative samples for the carry-over contamination study. Three (3) run matrices were used, two (2) for the cross–contamination study and one (1) for the carry-over contamination study. The results showed that the overall contamination rate of the twelve runs was $0 \%$ at a HSV1 positive sample concentration of $1 \mathrm { x } 1 0 ^ { \sim } 5$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All 96 positive samples were detected as positive, and the 183 negative samples were detected as negative (no amplification signal noted).

# Clinical Studies

1. Clinical Study: This study utilized residual anogenital lesion or oral lesion samples from male and female patients with signs and symptoms of HSV infections collected from 8 sites in the USA. Samples were either tested at the same facility at which they were obtained, or were shipped to a different testing site. Each sample was tested with Sentosa® SA201 HSV-1/2 PCR Test (4 test sites) and the $\mathrm { E L V I S } ^ { \mathrm { \textregistered } }$ HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System reference test (3 test sites). A total of 2684 samples were enrolled in the study. 389 samples were excluded, leaving a total of 2295 samples in the final analysis. Out of these, 317 were oral lesions and 1978 were anogenital lesions.

The clinical performance of Sentosa® SA201 HSV-1/2 PCR Test on oral samples compared to the ELVIS® HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System (ELVIS) for sensitivity and specificity is summarized in the following tables. Discordant samples were tested using bi-directional sequencing analysis.

Table 6. HSV-1 results for anogenital lesions   

<table><tr><td rowspan=2 colspan=1>SentosaSA201 HSV-1/2      PCRTest</td><td rowspan=1 colspan=3>ELVIS Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>281</td><td rowspan=1 colspan=1>54b</td><td rowspan=1 colspan=1>335</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>9a</td><td rowspan=1 colspan=1>1237</td><td rowspan=1 colspan=1>1246</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>1291</td><td rowspan=1 colspan=1>1581</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>96.90%</td><td rowspan=1 colspan=1>94.21%</td><td rowspan=1 colspan=1>98.36%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.82%</td><td rowspan=1 colspan=1>94.58%</td><td rowspan=1 colspan=1>96.78%</td></tr></table>

a From sequencing analysis, 4 discordant samples (HSV-1 positive by ELVIS and HSV-1 negative by Sentosa®) were in agreement with Sentosa® results, 2 were in agreement with ELVIS results and 2 were not in agreement with Sentosa® nor ELVIS results. 1 discordant sample was not tested due to insufficient volume.

b From sequencing analysis, 44 discordant samples (HSV-1 negative by ELVIS and HSV-1 positive by Sentosa®) were in agreement with Sentosa® results and 10 were in agreement with ELVIS results.

Table 7. HSV-2 results for anogenital lesions   

<table><tr><td rowspan=2 colspan=1>SentosaSA201 HSV-1/2      PCRTest</td><td rowspan=1 colspan=3>ELVIS Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>147d</td><td rowspan=1 colspan=1>538</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6c</td><td rowspan=1 colspan=1>1434</td><td rowspan=1 colspan=1>1440</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1581</td><td rowspan=1 colspan=1>1978</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>98.49%</td><td rowspan=1 colspan=1>96.74%</td><td rowspan=1 colspan=1>99.31%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>90.70%</td><td rowspan=1 colspan=1>89.17%</td><td rowspan=1 colspan=1>92.04%</td></tr></table>

c From sequencing analysis, 4 discordant samples (HSV-2 positive by ELVIS and HSV-2 negative by Sentosa®) were in agreement with Sentosa® results and 2 were in agreement with ELVIS results.

d Out of 147 discrepant samples, 142 were unique samples (3 samples were ELVIS HSV-1 positive/HSV-2 negative and Sentosa® HSV-1 negative/HSV-2 positive and were doublecounted in the 9 samples that are ELVIS HSV-1 positive and Sentosa® HSV-1 negative. 2 samples were ELVIS HSV-1 negative/HSV-2 negative and Sentosa® HSV-1 positive/HSV-2 positive. These were double-counted in the 54 samples that are ELVIS HSV-1 negative and Sentosa HSV-1 positive.) From sequencing analysis, 119 discordant samples (HSV-2 negative by ELVIS and HSV-2 positive by Sentosa®) were in agreement with Sentosa results and 20 were in agreement with ELVIS results. Three discordant samples was not tested due to insufficient volume.

HSV-1 results for anogenital lesions $\mathrm { ( N { = } 1 } 5 8 1$ , 397 samples tested as HSV-2 positive by ELVIS were excluded from the 1978 anogenital lesion samples).

Table 8. HSV-1 results for oral lesions   

<table><tr><td rowspan=2 colspan=1>SentosaSA201HSV-1/2PCR Test</td><td rowspan=1 colspan=3>ELVIS Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>203</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>314</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.36%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>86.38%</td><td rowspan=1 colspan=1>81.41%</td><td rowspan=1 colspan=1>90.19%</td></tr></table>

e From sequencing analysis, 27 discordant samples (HSV-1 negative by ELVIS and HSV-1 positive by Sentosa®) were in agreement with Sentosa® results and 4 were in agreement with ELVIS results. One discordant sample was not tested due to insufficient volume.

Table 9. HSV-2 results for oral lesions   

<table><tr><td rowspan=2 colspan=1>SentosaSA201 HSV-1/2     PCRTest</td><td rowspan=1 colspan=3>ELVIS Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1f</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>314</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>317</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Lower95% CI</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>66.67%</td><td rowspan=1 colspan=1>20.77%</td><td rowspan=1 colspan=1>93.85%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>99.68%</td><td rowspan=1 colspan=1>98.22%</td><td rowspan=1 colspan=1>99.94%</td></tr></table>

f From sequencing analysis, one sample (HSV-2 positive/HSV-1 negative by ELVIS and HSV-2 negative/HSV-1 positive by Sentosa®) was in agreement with Sentosa® results. g One discordant sample (HSV-1/2 negative by ELVIS and HSV-2 positive by Sentosa®) was not tested due to insufficient volume.   
HSV-1 results for oral lesions $\scriptstyle ( \mathrm { N } = 3 1 4$ , 3 samples tested as HSV-2 positive by ELVIS were excluded from the 317 oral lesion samples)

Table 10. Distribution of samples according to demographics for anogenital lesions as tested by Sentosa® SA201 HSV-1/2 PCR Test   

<table><tr><td rowspan=2 colspan=1>Age(years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0 - 10</td><td rowspan=1 colspan=1>2/277.4%</td><td rowspan=1 colspan=1>1/372.7%</td><td rowspan=1 colspan=1>3/644.7%</td><td rowspan=1 colspan=1>0/270.0%</td><td rowspan=1 colspan=1>0/370.0%</td><td rowspan=1 colspan=1>0/640.0%</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>59/18831.4%</td><td rowspan=1 colspan=1>8/4020.0%</td><td rowspan=1 colspan=1>67/22829.4%</td><td rowspan=1 colspan=1>51/23222.0%</td><td rowspan=1 colspan=1>6/4314.0%</td><td rowspan=1 colspan=1>57/27520.7%</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>126/40031.5%</td><td rowspan=1 colspan=1>19/10318.4%</td><td rowspan=1 colspan=1>145/50328.8%</td><td rowspan=1 colspan=1>173/53832.2%</td><td rowspan=1 colspan=1>55/14637.7%</td><td rowspan=1 colspan=1>228/68433.3%</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>50/25419.7%</td><td rowspan=1 colspan=1>7/7010.0%</td><td rowspan=1 colspan=1>57/32417.6%</td><td rowspan=1 colspan=1>71/30023.7%</td><td rowspan=1 colspan=1>14/7817.9%</td><td rowspan=1 colspan=1>85/37822.5%</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>27/18514.6%</td><td rowspan=1 colspan=1>4/2814.3%</td><td rowspan=1 colspan=1>31/21314.6%</td><td rowspan=1 colspan=1>58/22126.2%</td><td rowspan=1 colspan=1>11/3630.6%</td><td rowspan=1 colspan=1>69/25726.8%</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>22/10521.0%</td><td rowspan=1 colspan=1>1/273.7%</td><td rowspan=1 colspan=1>23/13217.4%</td><td rowspan=1 colspan=1>41/13929.5%</td><td rowspan=1 colspan=1>12/3138.7%</td><td rowspan=1 colspan=1>53/17031.2%</td></tr><tr><td rowspan=1 colspan=1>61 - 70</td><td rowspan=1 colspan=1>5/618.2%</td><td rowspan=1 colspan=1>2/1216.7%</td><td rowspan=1 colspan=1>7/739.6%</td><td rowspan=1 colspan=1>21/7727.3%</td><td rowspan=1 colspan=1>4/1428.6%</td><td rowspan=1 colspan=1>25/9127.5%</td></tr><tr><td rowspan=1 colspan=1>71 - 80</td><td rowspan=1 colspan=1>2/229.1%</td><td rowspan=1 colspan=1>0/90.0%</td><td rowspan=1 colspan=1>2/316.5%</td><td rowspan=1 colspan=1>12/3040.0%</td><td rowspan=1 colspan=1>3/1127.3%</td><td rowspan=1 colspan=1>15/4136.6%</td></tr><tr><td rowspan=1 colspan=1>81 - 90</td><td rowspan=1 colspan=1>0/60.0%</td><td rowspan=1 colspan=1>0/40.0%</td><td rowspan=1 colspan=1>0/100.0%</td><td rowspan=1 colspan=1>3/933.3%</td><td rowspan=1 colspan=1>2/633.3%</td><td rowspan=1 colspan=1>5/1533.3%</td></tr><tr><td rowspan=1 colspan=1>&gt;90</td><td rowspan=1 colspan=1>0/00.0%</td><td rowspan=1 colspan=1>0/30.0%</td><td rowspan=1 colspan=1>0/30.0%</td><td rowspan=1 colspan=1>0/00.0%</td><td rowspan=1 colspan=1>1/333.3%</td><td rowspan=1 colspan=1>1/333.3%</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>293/124823.5%</td><td rowspan=1 colspan=1>42/33312.6%</td><td rowspan=1 colspan=1>335/158121.2%</td><td rowspan=1 colspan=1>430/157327.3%</td><td rowspan=1 colspan=1>108/40526.7%</td><td rowspan=1 colspan=1>538/197827.2%</td></tr></table>

Table 11. Distribution of samples according to demographics for oral lesions as tested by Sentosa® SA201 HSV-1/2 PCR Test   

<table><tr><td rowspan=2 colspan=1>Age(years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0 - 10</td><td rowspan=1 colspan=1>11/2544.0%</td><td rowspan=1 colspan=1>10/2835.7%</td><td rowspan=1 colspan=1>21/5339.6%</td><td rowspan=1 colspan=1>0/250.0%</td><td rowspan=1 colspan=1>0/290.0%</td><td rowspan=1 colspan=1>0/540.0%</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>7/1353.8%</td><td rowspan=1 colspan=1>5/1827.8%</td><td rowspan=1 colspan=1>12/3138.7%</td><td rowspan=1 colspan=1>0/130.0%</td><td rowspan=1 colspan=1>0/180.0%</td><td rowspan=1 colspan=1>0/310.0%</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>8/4020.0%</td><td rowspan=1 colspan=1>6/2623.1%</td><td rowspan=1 colspan=1>14/6621.2%</td><td rowspan=1 colspan=1>0/400.0%</td><td rowspan=1 colspan=1>0/260.0%</td><td rowspan=1 colspan=1>0/660.0%</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>5/2321.7%</td><td rowspan=1 colspan=1>5/1631.3%</td><td rowspan=1 colspan=1>10/3925.6%</td><td rowspan=1 colspan=1>1/244.2%</td><td rowspan=1 colspan=1>0/160.0%</td><td rowspan=1 colspan=1>1/402.5%</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>9/1947.4%</td><td rowspan=1 colspan=1>2/633.3%</td><td rowspan=1 colspan=1>11/2544.0%</td><td rowspan=1 colspan=1>0/190.0%</td><td rowspan=1 colspan=1>0/60.0%</td><td rowspan=1 colspan=1>0/250.0%</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>7/2528.0%</td><td rowspan=1 colspan=1>5/955.6%</td><td rowspan=1 colspan=1>12/3435.3%</td><td rowspan=1 colspan=1>0/250.0%</td><td rowspan=1 colspan=1>0/90.0%</td><td rowspan=1 colspan=1>0/340.0%</td></tr><tr><td rowspan=1 colspan=1>61 - 70</td><td rowspan=1 colspan=1>4/1822.2%</td><td rowspan=1 colspan=1>7/1450.0%</td><td rowspan=1 colspan=1>11/3234.4%</td><td rowspan=1 colspan=1>1/185.6%</td><td rowspan=1 colspan=1>0/140.0%</td><td rowspan=1 colspan=1>1/323.1%</td></tr><tr><td rowspan=1 colspan=1>71 - 80</td><td rowspan=1 colspan=1>9/1369.2%</td><td rowspan=1 colspan=1>7/1450.0%</td><td rowspan=1 colspan=1>16/2759.3%</td><td rowspan=1 colspan=1>0/130.0%</td><td rowspan=1 colspan=1>1/156.7%</td><td rowspan=1 colspan=1>1/283.6%</td></tr><tr><td rowspan=1 colspan=1>81 - 90</td><td rowspan=1 colspan=1>1/333.3%</td><td rowspan=1 colspan=1>2/450.0%</td><td rowspan=1 colspan=1>3/742.9%</td><td rowspan=1 colspan=1>0/30.0%</td><td rowspan=1 colspan=1>0/40.0%</td><td rowspan=1 colspan=1>0/70.0%</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>61/17934.1%</td><td rowspan=1 colspan=1>49/13536.3%</td><td rowspan=1 colspan=1>110/31435.0%</td><td rowspan=1 colspan=1>2/1801.1%</td><td rowspan=1 colspan=1>1/1370.7%</td><td rowspan=1 colspan=1>3/3170.9%</td></tr></table>

Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the Sentosa® SA201 HSV-1/2 PCR Test are shown below. These calculations for hypothetical prevalence are based on overall sensitivity and specificity per sample type from the clinical study results. For HSV-1, these calculations are based upon an overall sensitivity and specificity of $9 6 . 9 0 \%$ and $9 5 . 8 2 \%$ , respectively, for anogenital swabs and $1 0 0 . 0 \%$ and $8 6 . 3 8 \%$ , respectively, for oral swabs. For HSV-2, these calculations are based upon an overall sensitivity and specificity of $9 8 . 4 9 \%$ and $9 0 . 7 0 \%$ , respectively, for anogenital swabs and $6 6 . 6 7 \%$ and $9 9 . 6 8 \%$ , respectively, for oral swabs.

Table 12. Prevalence vs hypothetical Predictive Values   

<table><tr><td rowspan=3 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=4>Anogenital</td><td rowspan=1 colspan=4>Oral</td></tr><tr><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>32.12%</td><td rowspan=1 colspan=1>99.93%</td><td rowspan=1 colspan=1>17.77%</td><td rowspan=1 colspan=1>99.97%</td><td rowspan=1 colspan=1>13.03%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>80.96%</td><td rowspan=1 colspan=1>99.32%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>54.96%</td><td rowspan=1 colspan=1>99.83%</td><td rowspan=1 colspan=1>35.79%</td><td rowspan=1 colspan=1>99.91%</td><td rowspan=1 colspan=1>27.87%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>91.64%</td><td rowspan=1 colspan=1>98.27%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>72.03%</td><td rowspan=1 colspan=1>99.64%</td><td rowspan=1 colspan=1>54.06%</td><td rowspan=1 colspan=1>99.82%</td><td rowspan=1 colspan=1>44.93%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.86%</td><td rowspan=1 colspan=1>96.42%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>85.28%</td><td rowspan=1 colspan=1>99.20%</td><td rowspan=1 colspan=1>72.58%</td><td rowspan=1 colspan=1>99.59%</td><td rowspan=1 colspan=1>64.73%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.12%</td><td rowspan=1 colspan=1>92.29%</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>90.86%</td><td rowspan=1 colspan=1>98.63%</td><td rowspan=1 colspan=1>81.95%</td><td rowspan=1 colspan=1>99.29%</td><td rowspan=1 colspan=1>75.88%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.89%</td><td rowspan=1 colspan=1>87.47%</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>93.92%</td><td rowspan=1 colspan=1>97.89%</td><td rowspan=1 colspan=1>87.59%</td><td rowspan=1 colspan=1>98.90%</td><td rowspan=1 colspan=1>83.04%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>99.29%</td><td rowspan=1 colspan=1>81.77%</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>95.86%</td><td rowspan=1 colspan=1>96.87%</td><td rowspan=1 colspan=1>91.37%</td><td rowspan=1 colspan=1>98.36%</td><td rowspan=1 colspan=1>88.01%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>99.52%</td><td rowspan=1 colspan=1>74.94%</td></tr></table>

2. HSV-2 oral lesion contrived specimen study: A contrived specimen study was performed to provide additional performance data for detection of HSV-2 in oral samples. A study was performed to test 30 HSV-2-contrived oral lesion samples, along with 15 HSV-1 positive oral lesion samples and 15 HSV-1/HSV-2 negative samples. In the test, all 30 HSV-2-contrived oral lesion samples were identified as HSV-2 positive, and all 15 HSV-1 positive and 15 HSV-1/2 negative samples were correctly identified.

# Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.