# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K172569   
B. Purpose for Submission: To obtain a substantial equivalence determination for the GenePOC CDiff assay   
C. Measurand: tcdB gene of toxigenic Clostridium difficile   
D. Type of Test: Real-time Polymerase chain reaction (rtPCR)   
E. Applicant: GenePOC Inc.   
F. Proprietary and Established Names: GenePOC CDiff

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3130 - Clostridium difficile toxin gene amplification assay

2. Classification: Class II

3. Product code: OZN

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The GenePOC CDiff assay performed on the revogene instrument is a qualitative in vitro diagnostic test that utilizes automated sample processing and real-time polymerase chain reaction (PCR) to detect the toxin B (tcdB) gene of toxigenic Clostridium difficile ( $C .$ difficile) in unformed (liquid or soft) stool specimens obtained from patients suspected of having C. difficile infection (CDI). The GenePOC CDiff assay is intended to aid in the diagnosis of CDI.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: revogene instrument

# I. Device Description:

The GenePOC CDiff assay is a single-use test for the qualitative detection of the toxin B (tcdB) gene of toxigenic C. difficile in unformed (liquid or soft) stool specimens. The GenePOC CDiff assay kit is comprised of the disposable CDiff microfluidic cartridges (PIE), Disposable Transfer Loops (DTL), Sample Buffer Tubes (SBT), and individual pouches of the following materials: Disposable Transfer Tools (DTT), Sample Buffer Tube (SBT) and CDiff PIE. These components are used to suspend the sample, extract, amplify, and detect $C .$ . difficile nucleic acid. A Process Control $( \mathrm { P r } \mathrm { C } )$ is also incorporated into each PIE to verify sample processing and amplification steps. The $\mathrm { P r C }$ allows for the verification of potential inhibitor substances as well as microfluidic, instrument or reagent failure. The GenePOC CDiff assay is designed to be used on the revogene. The revogene is an instrument that automates sample homogenization, sample dilution, cells lysis, DNA amplification and detection of the amplified PCR products.

Each GenePOC CDiff assay kit provides components for 24 tests. User intervention is required for sample preparation, transferring the stool specimen with the DTL into the SBT, using the DTT to transfer the sample into the CDiff PIE, and loading/unloading the PIE into the revogene carousel. Each PIE is a completely integrated closed device in which a sample is dispensed and processed through different microfluidic chambers and channels that allow for the sample processing and subsequent real-time PCR steps.

Upon completion of a run, the results are computed by the revogene from measured fluorescent signals and embedded calculation algorithms. The output results include positive, negative, indeterminate, and unresolved. Upon completion of a run, the user removes the used cartridges and disposes of them in normal biological waste. Results may be viewed,

printed, transferred, and/or stored by the user.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX CDiff Assay

2. Predicate 510(k) number(s): K130470

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceGenePOC CDiff(K172569)</td><td rowspan=1 colspan=1>PredicateBD MAX Cdiff Assay(K130470)</td></tr><tr><td rowspan=1 colspan=1>Assay target</td><td rowspan=1 colspan=1>tcdB gene</td><td rowspan=1 colspan=1>same</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Unformed (liquid or soft)stool</td><td rowspan=1 colspan=1>same</td></tr><tr><td rowspan=1 colspan=1>Assay format</td><td rowspan=1 colspan=1>Real-time PCR</td><td rowspan=1 colspan=1>same</td></tr><tr><td rowspan=1 colspan=1>Detection probes</td><td rowspan=1 colspan=1>TaqMan Probes</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample extraction</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>same</td></tr><tr><td rowspan=1 colspan=1>Result interpretation</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceGenePOC CDiff(K172569)</td><td rowspan=1 colspan=1>PredicateBD MAX Cdiff Assay(K130470)</td></tr><tr><td rowspan=1 colspan=1>Instrument platform</td><td rowspan=1 colspan=1>revogene</td><td rowspan=1 colspan=1>BD MAX</td></tr><tr><td rowspan=1 colspan=1>Samples per run</td><td rowspan=1 colspan=1>Up to eight</td><td rowspan=1 colspan=1>Up to 24</td></tr><tr><td rowspan=1 colspan=1>Assay run time</td><td rowspan=1 colspan=1>~ 70 minutes</td><td rowspan=1 colspan=1>2 - 3 hours</td></tr><tr><td rowspan=1 colspan=1>Sample volume</td><td rowspan=1 colspan=1>5 μL inoculating loop</td><td rowspan=1 colspan=1>10 μL inoculating loop</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI Guideline EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline

# L. Test Principle:

The GenePOC revogene automates and integrates nucleic acid extraction and amplification, and detection of the target sequence in complex samples using real-time PCR. A liquid or soft stool specimen is collected using a standard stool collection device. Using the DTL, a disposable $5 \mu \mathrm { L }$ inoculating loop dipped into the stool specimen, stool material is transferred into SBT. After vortexing, approximately $1 5 0 \mu \mathrm { L }$ of the inoculated sample buffer is transferred into the GenePOC microfluidic cartridge using the DTT. The loaded CDiff

cartridge is placed into the revogene for further sample processing. No operator intervention is necessary once the clinical sample is loaded onto the revogene.

Each CDiff microfluidic cartridge (PIE) is a completely integrated and self-contained device. Each sample is sequentially transferred by centrifugation from one microfluidic chamber to the next and all reagents specific for the PCR reaction are incorporated and dried within the PCR wells. The stepwise process includes sample homogenization, specimen dilution, and lysis of cells followed by the subsequent real-time PCR steps within one PCR well in the cartridge. An internal Process Control $( \mathrm { P r } \mathrm { C } )$ is contained in the homogenization chamber and is therefore present in every test to verify critical steps of the analytical process (including sample homogenization, dilution, sample lysis, and nucleic acid amplification and detection) for the presence of potential inhibitory substances as well as system or reagent failures. The amplified products are detected in real time using target-specific TaqMan chemistry-based probes. The CDiff specific designed primers and probe detect a target region of 262 base pairs (bp) of the toxin B gene (tcdB) of C. difficile. The results are computed by the system from measured fluorescent signals and embedded calculation algorithms. Results may be viewed, printed, transferred, and/or stored by the user.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Single site and multi-site precision studies were conducted to determine BetweenLaboratory Reproducibility, Between-Lot Reproducibility, and Within-Laboratory Precision.

Between-Site Reproducibility:

Between-site reproducibility study was performed at three sites, by two operators per site, over five distinct days using one CDiff assay kit lot. A total of 120 negative sample replicates and 90 positive sample replicates were tested for each strain and concentration. Two toxigenic C. difficile strains were tested: ATCC 43255 (toxinotype 0, ribotype 087) and ATCC BAA-1805 (toxinotype IIIb, NAP1/ribotype 027).

A five-member panel was contrived using a pool of $C .$ . difficile-negative stool matrix at the following target concentrations:

Low positive (LP) at 1X to $< 2 \mathrm { X }$ LoD · Moderate positive (MP) at $\geq 2 \mathrm { X }$ to 3X LoD True negative (TN) samples without toxigenic C. difficile

The LP concentrations tested were $2 , 4 3 8 \mathrm { C F U / m L }$ and $2 , 9 2 5 \mathrm { C F U / m L }$ of SB for ATCC 43255 and ATCC BAA-1805, respectively. The MP concentrations tested were 3,750 and 4,500 CFU/mL of SB for ATCC 43255 and ATCC BAA-1805, respectively.

For the between-site reproducibility, the overall percent agreement was $100 \%$ for TN,

LP and MP samples of strain ATCC BAA-1805. The overall percent agreement for LP and MP samples of strain ATCC 43255 was $9 4 . 4 \%$ and $9 6 . 7 \%$ , respectively. The qualitative results, mean Cycle threshold $\mathrm { ( C t ) }$ values, and variance (SD and $\% C V$ ) are shown in Table 1.

Table 1. Between-Site Reproducibility   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=4>Assay Results/Expected</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=2>Ct values*</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Site 01</td><td rowspan=1 colspan=1>Site 02</td><td rowspan=1 colspan=1>Site 03</td><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>Total[95% CI]</td><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>LP</td><td rowspan=1 colspan=1>ATCC43255</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>85/90</td><td rowspan=1 colspan=1>94.4%[87.5%-98.2%]</td><td rowspan=1 colspan=1>38.48</td><td rowspan=1 colspan=1>1.56</td><td rowspan=1 colspan=1>4.05</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1805</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%[96.7%-100%]</td><td rowspan=1 colspan=1>36.37</td><td rowspan=1 colspan=1>2.74</td><td rowspan=1 colspan=1>7.54</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>57/6095%[86.1-99.0%]</td><td rowspan=1 colspan=1>58/6096.7%[88.5-99.6]</td><td rowspan=1 colspan=1>60/60100%[95.1-100]</td><td rowspan=1 colspan=1>175/180</td><td rowspan=1 colspan=1>97.2%[93.6%-99.1%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=3 colspan=1>MP</td><td rowspan=1 colspan=1>ATCC43255</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>87/90</td><td rowspan=1 colspan=1>96.7%[90.6%-99.3%]</td><td rowspan=1 colspan=1>37.68</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>4.32</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1805</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%[96.7%-100%]</td><td rowspan=1 colspan=1>36.57</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>4.52</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>59/6098.3%[91.1-100%]</td><td rowspan=1 colspan=1>59/6098.3%[91.1%- 100%]</td><td rowspan=1 colspan=1>59/6098.3%[91.1%-100%</td><td rowspan=1 colspan=1>177/180</td><td rowspan=1 colspan=1>98.3%[95.2%-99.7%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40/40100%[92.8-100%][</td><td rowspan=1 colspan=1>40/40100%][92.8%- 100%][92.8%-100%]</td><td rowspan=1 colspan=1>40/40100%</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[97.5%-100%]</td><td rowspan=1 colspan=1>33.09</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>3.09</td></tr></table>

\* For the LP and MP categories, Ct values reported are for the toxigenic C. difficile target. For the TN category, Ct values reported are for the $\mathrm { P r C }$ .

# Between-Lot Reproducibility

Between-lot reproducibility study was performed at one site, by two operators, over 15 days using three CDiff assay kit lots (five days per kit lot) testing the same panel described for between-site reproducibility. The overall percent agreement was $100 \%$ for ATCC BAA-1805 panel members TN and MP and $9 8 . 9 \%$ for the LP panel member. The overall percent agreement for ATCC 43255 panel members LP and MP was $9 1 . 1 \%$ and $9 6 . 7 \%$ respectively. The qualitative results and $\mathrm { C t }$ value analysis are shown in Table 2.

Table 2. Between-Lot Reproducibility   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=4>Assay Results/Expected</td><td rowspan=2 colspan=1>AgreementTotal[95% CI]</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=2>Ct values*</td></tr><tr><td rowspan=1 colspan=1>Site 01</td><td rowspan=1 colspan=1>Site 02</td><td rowspan=1 colspan=1>Site 03</td><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>LP</td><td rowspan=1 colspan=1>ATCC43255</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>82/90</td><td rowspan=1 colspan=1>91.1%[83.2%-96.1%]</td><td rowspan=1 colspan=1>38.57</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>4.46</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1805</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%[94.0%-100%]</td><td rowspan=1 colspan=1>37.02</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>5.35</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>57/6095%[86.1-99.0%</td><td rowspan=1 colspan=1>56/6093.3%[83.8-98.2%]</td><td rowspan=1 colspan=1>58/6096.7%[88.5-99.6%]</td><td rowspan=1 colspan=1>171/180</td><td rowspan=1 colspan=1>95.0%[90.7%-97.7%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=3 colspan=1>MP</td><td rowspan=1 colspan=1>ATCC43255</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>87/90</td><td rowspan=1 colspan=1>96.7%[90.6%-99.3%]</td><td rowspan=1 colspan=1>37.55</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>5.75</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1805</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%[96.7%-100%]</td><td rowspan=1 colspan=1>36.86</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>3.56</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>59/6098.3%[91.1-100%]</td><td rowspan=1 colspan=1>60/60100%[95.1-100%]</td><td rowspan=1 colspan=1>58/6096.7%[88.5-99.6%]</td><td rowspan=1 colspan=1>177/180</td><td rowspan=1 colspan=1>98.3%[95.2%-99.7%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40/40100%[92.8-100%]</td><td rowspan=1 colspan=1>40/40100%[92.8-100%]</td><td rowspan=1 colspan=1>40/40100%[92.8-100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[97.5%-100%]</td><td rowspan=1 colspan=1>33.21</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>2.71</td></tr></table>

\* For the LP and MP categories, Ct values reported are for the toxigenic $C$ . difficile target. For the TN category, Ct values reported are for the $\mathrm { P r C }$ .

# Within-Site Precision/Repeatability

A within-site precision study was performed at one site, by two operators, over 12 days, with one CDiff assay kit lot testing the same panel used for between-site reproducibility. Precision study results for the TN and the LP ATCC BAA-1805 strain panel members demonstrated $100 \%$ agreement and $9 7 . 2 \%$ for MP. Precision study results for LP and MP ATCC 43255 panel members demonstrated agreement of $8 8 . 9 \%$ and $9 5 . 8 \%$ respectively. The qualitative results and Ct value analysis are shown in Table 3.

Table 3. Within-Laboratory Precision   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>AssayResults/Expected</td><td rowspan=2 colspan=1>Agreement (%)[95% CI]</td><td rowspan=1 colspan=3>Ct values**</td></tr><tr><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>LP</td><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>64/72</td><td rowspan=1 colspan=1>88.9[79.3%-95.1%]</td><td rowspan=1 colspan=1>38.64</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>4.10</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>100[95.9%-100%]</td><td rowspan=1 colspan=1>36.93</td><td rowspan=1 colspan=1>2.41</td><td rowspan=1 colspan=1>6.54</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>135/143</td><td rowspan=1 colspan=1>94.4[89.3%-97.6%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=3 colspan=1>MP</td><td rowspan=1 colspan=1>ATCC 43255</td><td rowspan=1 colspan=1>69/72</td><td rowspan=1 colspan=1>95.8[88.3%-99.1%]</td><td rowspan=1 colspan=1>37.29</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>3.77</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>70/72</td><td rowspan=1 colspan=1>97.2[90.3%-99.7%]</td><td rowspan=1 colspan=1>36.73</td><td rowspan=1 colspan=1>1.43</td><td rowspan=1 colspan=1>3.88</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>139/144</td><td rowspan=1 colspan=1>96.5[92.1%-98.9%]</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95/95</td><td rowspan=1 colspan=1>100[96.9%-100%]</td><td rowspan=1 colspan=1>33.04</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>3.05</td></tr></table>

\* For the LP and MP categories, Ct values reported are for the toxigenic C. difficile target. For the TN category, Ct values reported are for the $\mathrm { P r C }$ .

b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The GenePOC CDiff assay uses three types of controls: an internal process control $( \mathrm { P r } \mathrm { C } )$ , a positive external control (PEC), and negative external control (NEC). The $\mathrm { P r C }$ is included in each CDiff PIE cartridge. The $\mathrm { P r C }$ is extracted, amplified, and detected along with each specimen tested, and it verifies the efficacy of the dilution, cell lysis, PCR amplification, and detection processes.

Quality control procedures monitor the accuracy and precision of the analytical process. Each laboratory must establish the number, type and frequency of testing control materials per applicable regulations or accrediting agencies. It is recommended that one PEC and one NEC should be run at least daily until adequate process validation is achieved with the CDiff assay on the revogene in each laboratory setting.

External Control materials are not provided by GenePOC. External Controls are not used by the revogene software for the purpose of sample test result interpretation. The PEC monitors for substantial reagent failure, and the NEC monitors for environmental contamination or carryover. External Controls are treated as if they are specimens.

For the PEC and NEC, GenePOC recommends using commercially available control materials (e.g., ATCC 43255, a C. difficile strain bearing the tcdB gene can be used for the PEC, and ATCC 43593, a non- toxigenic C. difficile strain can be used for the

NEC). It is recommended that the positive bacterial strain be freshly prepared in saline to a turbidity of 0.5 McFarland from isolated colonies and subsequently diluted $\%$ in saline before its addition into the SBT with the DTL. The negative strain can be prepared similarly while added directly to the SBT without the dilution step. Alternatively, a previously characterized stool specimens positive or negative for toxigenic $C .$ . difficile could also be used as the PEC and NEC, respectively.

One PEC and one NEC were processed each day of analytical and clinical testing and with the use of each new PIE kit lot. Of the total 403 external controls that were run across all sites in the clinical study, 391 $( 9 7 . 0 \% )$ were valid, nine $( 2 . 2 \% )$ failed due to non-reportable results, and three $( 0 . 7 \% )$ failed due to false results.

# d. Detection limit:

The analytical sensitivity (Limit of Detection or LoD) of the CDiff assay was determined using negative stool matrix previously spiked with various concentrations of toxigenic C. difficile bacterial suspensions. The testing to establish and confirm the LoD included three GenePOC CDiff reagent lots, six revogene instruments, and two operators. There were four runs per instrument per day, and the confirmation study was conducted over four days. Two strains of toxigenic $C .$ difficile (ATCC 43255, Ribotype 087, Toxinotype 0, and ATCC BAA-1805, NAP 1, Ribotype 027, Toxinotype IIIb) were tested in 24 replicates per concentration in the confirmation study. The LoD was defined as the lowest concentration at which $9 5 \%$ or more of all replicates tested positive. The LoD was determined at $1 { , } 5 0 0 \mathrm { C F U / m L }$ of SB for both strains.

# Analytical inclusivity (reactivity):

The analytical inclusivity of the GenePOC CDiff assay was evaluated by testing 20 strains of toxigenic C. difficile from various geographic origins representing eight different toxinotypes. Quantitated cultures of each C. difficile strain were diluted in negative stool matrix, and three replicates per strain were tested using three different CDiff kit lots. All strains were detected at 3,750 CFU/mL of SB (2-3xLOD). The results are summarized in Table 4.

In addition, an in silico analysis was performed to assess inclusivity of the primers and probe of the CDiff assay target for 52 toxigenic C. difficile strains listed in the National Center for Biotechnology Information (NCBI) database. The alignment results showed no mismatch with the selected 52 sequences. The analysis predicted the detection of these toxigenic C. difficile strains.

Table 4. Inclusivity   

<table><tr><td colspan="1" rowspan="1">Toxigenic C. difficile Strain</td><td colspan="1" rowspan="1">Toxinotype, Toxin</td></tr><tr><td colspan="1" rowspan="1">ATCC 9689</td><td colspan="1" rowspan="1">Toxinotype 0,,A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCC700792</td><td colspan="1" rowspan="1">Toxinotype 0,$A+{$B+$</td></tr><tr><td colspan="1" rowspan="1">ATCC17858</td><td colspan="1" rowspan="1">Toxinotype 0, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1382</td><td colspan="1" rowspan="1">Toxinotype 0, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCC 51695</td><td colspan="1" rowspan="1">Toxinotype 0,,A+,B+</td></tr><tr><td colspan="1" rowspan="1">ATCC43600</td><td colspan="1" rowspan="1">Toxinotype0,$A+ B+$</td></tr><tr><td colspan="1" rowspan="1">ATCC43599</td><td colspan="1" rowspan="1">Toxinotype0,$A+$B+$</td></tr><tr><td colspan="1" rowspan="1">ATCC43596</td><td colspan="1" rowspan="1">Toxinotype0,$A+{$B+$</td></tr><tr><td colspan="1" rowspan="1">ATCC43594</td><td colspan="1" rowspan="1">Toxinotype0,A+{}B+</td></tr><tr><td colspan="1" rowspan="1">ATCC17857</td><td colspan="1" rowspan="1">Toxinotype 0, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCC43598</td><td colspan="1" rowspan="1">ToxinotypeVIII, A-, B+</td></tr><tr><td colspan="1" rowspan="1">CCUG8864</td><td colspan="1" rowspan="1">ToxinotypeX, A-, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1870</td><td colspan="1" rowspan="1">Toxinotype IIIb, NAP1, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1812</td><td colspan="1" rowspan="1">Toxinotype XII, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1803</td><td colspan="1" rowspan="1">Toxinotype IIIc, NAP1, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1814</td><td colspan="1" rowspan="1">Toxinotype XXII, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1804</td><td colspan="1" rowspan="1">Toxinotype 0, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1875</td><td colspan="1" rowspan="1">Toxinotype V, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-2155</td><td colspan="1" rowspan="1">ToxinotypeXXII, A+, B+</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1873</td><td colspan="1" rowspan="1">Toxinotype 0, A+, B+</td></tr></table>

# e. Analytical specificity:

# Cross reactivity:

The cross-reactivity of the CDiff assay was assessed by testing high concentrations of 58 non-targeted organisms (50 bacteria, one yeast, seven viruses) and DNA preparations that may be found in unformed stool specimens (see Table 5, Table 6, and Table 7) . The test panel included commensal and pathogenic microorganisms from the intestinal tract, species phylogenetically related to C. difficile, nonpathogenic strains of $C .$ . difficile; and human DNA. Bacteria and yeast were tested at a load of $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB. Nucleic acids from viruses and human DNA were tested at a load o $\mathrm { \Delta } \mathrm { f } \geq 1 0 ^ { 5 } \mathrm { D N A }$ or RNA copies $/ \mathrm { m L }$ of SB. The test panel was contrived by spiking quantitated cell cultures or nucleic acid solutions in a $C .$ . difficile-negative liquid stool matrix. Each organism was tested in triplicate.

Under the conditions of the study, Clostridium sordellii gave a false positive result by the CDiff assay for one replicate out of three at $1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB, but no false positives were observed when an additional three replicates were tested at $1 0 ^ { 5 }$ $\mathrm { C F U / m L }$ of SB.

False positive results were observed in one replicate out of six each for Clostridium novyi and Clostridium scindens $1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB; however, no false positives were observed when an additional three replicates were tested each at $1 0 ^ { 5 } \mathrm { C F U / m L }$ of SB. Similarly, a false positive result was observed for one out of three replicates of Enterococcus faecalis tested at $2 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ of SB; however, no false positive results were observed for an additional three replicates tested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB. None of the other test panel members produced positive results when tested with the CDiff assay.

Potential cross reactivity with Coxsackievirus was analyzed in silico as no acceptable extracts were available for wet testing. The in silico analysis predicted that

Coxsackievirus strains should react with the CDiff assay.

Table 5. Cross Reactivity Panel - Non-toxigenic C. difficile, other Clostridium sp.   

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td><td rowspan=1 colspan=1>ATCC 43593</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td><td rowspan=1 colspan=1>ATCC 43601</td></tr><tr><td rowspan=1 colspan=1>Clostridium bifermentans</td><td rowspan=1 colspan=1>ATCC 638</td></tr><tr><td rowspan=1 colspan=1>Clostridium butyricum</td><td rowspan=1 colspan=1>ATCC 860</td></tr><tr><td rowspan=1 colspan=1>Clostridium haemolyticum</td><td rowspan=1 colspan=1>ATCC 9650</td></tr><tr><td rowspan=1 colspan=1>Clostridium novyia</td><td rowspan=1 colspan=1>ATCC 19402</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>ATCC 13124</td></tr><tr><td rowspan=1 colspan=1>Clostridium scindensa</td><td rowspan=1 colspan=1>ATCC 35704</td></tr><tr><td rowspan=1 colspan=1>Clostridium septicum</td><td rowspan=1 colspan=1>ATCC 12464</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellib</td><td rowspan=1 colspan=1>ATCC 9714</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>ATCC 15579</td></tr><tr><td rowspan=1 colspan=1>Flavonifractor plautii(anc. design. Clostridium orbiscindens)</td><td rowspan=1 colspan=1>ATCC 49531</td></tr></table>

a Samples produced false positive reactions in one replicate out of six tested at ${ \sim } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB. No false positive reactions were observed for three replicates tested at $1 0 ^ { 5 } \mathrm { C F U / m L }$ of SB. b Samples produced false positive reactions in one replicate out of three tested at ${ \sim } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB. No false positive reactions were observed for three replicates tested at $1 0 ^ { 5 } \mathrm { C F U / m L }$ of SB.

Table 6. Cross Reactivity Panel - Other Bacteria   

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 14029</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC 7966</td><td colspan="1" rowspan="1">Porphyromonasasaccharolytica</td><td colspan="1" rowspan="1">ATCC 25260</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 14579</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 33583</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni sub sp. jejuni</td><td colspan="1" rowspan="1">ATCC 33560</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 35554</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni(anc. design. Campylobacter coli)</td><td colspan="1" rowspan="1">ATCC 43479</td><td colspan="1" rowspan="1">Salmonella enterica subsp.arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovarCholeraesuis</td><td colspan="1" rowspan="1">ATCC 7001</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovarTyphimurium</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp. cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">Serratia marcescens subsp.marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 19433</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 11775</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">CCRI-7792</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">CCRI-22391</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">ATCC 43504</td><td colspan="1" rowspan="1">Staphylococcus aureussubsp. aureus</td><td colspan="1" rowspan="1">ATCC 33592</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">Streptococcussp.(S.agalactiae)</td><td colspan="1" rowspan="1">ATCC 12973</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 7644</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr></table>

Table 7. Cross Reactivity Panel - Viruses and Other Organisms   

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>ID</td><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus 1 (DNA)</td><td rowspan=1 colspan=1>ATCC VR-1D</td><td rowspan=1 colspan=1>Enterovirus (RNA)</td><td rowspan=1 colspan=1>ATCC VR-1823D</td></tr><tr><td rowspan=1 colspan=1>Rotavirus (RNA)</td><td rowspan=1 colspan=1>ATCC VR-2018DQ</td><td rowspan=1 colspan=1>Echovirus (RNA)</td><td rowspan=1 colspan=1>ATCC VR-1734D</td></tr><tr><td rowspan=1 colspan=1>Norovirus (RNA)</td><td rowspan=1 colspan=1>ATCC VR-3235SD (GII)</td><td rowspan=1 colspan=1>HumanHerpesvirus 5</td><td rowspan=1 colspan=1>ATCC VR-538D</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus</td><td rowspan=1 colspan=1>In silico</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>TQMN controlgenomic DNA</td><td rowspan=1 colspan=2></td></tr></table>

# Microbial Interference

A microbial interference study was conducted to assess the potential inhibitory effect of 30 non-targeted microorganisms (27 bacteria, one yeast and two viruses) that may be found in stool specimens (Table 8). Microorganisms were selected from the cross reactivity panel. Pools of two to six organisms were prepared in negative liquid stool matrix and tested in four replicates each in the presence or absence of two toxigenic C. difficile strains (ATCC 43255 and ATCC BAA-1805). The potentially interfering organisms were spiked at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ of SB for bacteria and yeast and at ${ \geq } 1 0 ^ { 5 }$ $\mathrm { c p / m L }$ of SB for viruses. Toxigenic $C .$ . difficile strains were spiked at 2-3X LoD (ATCC 43255 at $3 , 7 5 0 \mathrm { C F U / m L }$ of SB, and ATCC BAA-1805 at $4 { , } 5 0 0 \mathrm { C F U / m L }$ of SB). When potential interference was observed with pooled organisms, additional replicates of the organisms were tested individually at the same spiked concentration.

No interference was observed with the GenePOC CDiff assay for the detection of $C .$ . difficile strain ATCC BAA-1805, and no interference was observed for the detection of the assay PrC.

Pool 3 and pool 5 showed potentially inhibitory effects on the detection of the toxigenic $C .$ . difficile strain ATCC 43255 for one replicate out of four and two replicates out of four, respectively. When each of the bacteria from these pools were tested individually in duplicate at the same spiked concentration, no interference was observed for the detection of either toxigenic C. difficile strain or for the $\mathrm { P r C }$ .

Table 8. Microbial Interference Panel   

<table><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>Potentially Interfering Microorganisms</td></tr><tr><td rowspan=5 colspan=1>1</td><td rowspan=1 colspan=1>Aeromonas hydrophila</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni(Campylobacter coli)</td></tr><tr><td rowspan=6 colspan=1>2</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellii</td></tr><tr><td rowspan=5 colspan=1>3a</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae subsp. cloacae</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O157:H7</td></tr><tr><td rowspan=1 colspan=1>Helicobacter pylori</td></tr><tr><td rowspan=6 colspan=1>4</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica serovarCholeraesuis</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica serovarTyphimurium</td></tr><tr><td rowspan=6 colspan=1>56</td><td rowspan=1 colspan=1>Shigella boydii</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus subsp. aureus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=2 colspan=1>6</td><td rowspan=1 colspan=1>Rotavirus RNA</td></tr><tr><td rowspan=1 colspan=1>Norovirus RNA</td></tr></table>

a Potential interference observed with the detection of ATCC 43255 only in one pool replicate out of four. No interference was observed when pooled organisms were re-tested individually. b Potential interference observed with the detection of ATCC 43255 only in two pool replicates out of four. No interference was observed when pooled organisms were re-tested individually.

# Interfering Substances

An interfering substances study was conducted to assess the potentially inhibitory effects of 16 exogenous (Table 9) and five endogenous substances (Table 10) that may be present in stool specimens. The substances were tested in the presence or absence of two strains of toxigenic C. difficile (ATCC 43255 and ATCC BAA-1805) in negative stool matrix in SBTs. Substances were spiked at their high concentrations expected in stool specimens, and C. difficile strains were spiked at 2-3X LoD (ATCC 43255 at 3,750 CFU/mL of SB, and ATCC BAA-1805 at 4,500 CFU/mL of SB).

Calcium carbonate (e.g. Tums) and aluminum hydroxide/magnesium hydroxide (e.g. Stomaax) showed a potentially inhibitory effect on the detection of toxigenic $C .$ . difficile when either of these substances was present in SBT at a concentration of 5 $\mathrm { m g / m L }$ $( 0 . 5 \% \mathrm { w } / \mathrm { v } )$ and $5 ~ \mu \mathrm { L / m L }$ $( 0 . 5 \% \mathrm { v / v } )$ , respectively. When tested at $0 . 5 ~ \mathrm { m g / m L }$ $( 0 . 0 5 \% \mathrm { w } / \mathrm { v } )$ or $0 . 5 ~ \mu \mathrm { L / m L }$ $( 0 . 0 5 \% \mathrm { v } / \mathrm { v } )$ respectively, these substances showed no reportable interference with the CDiff assay. No interference was observed for the remaining exogenous or endogenous substances at the tested concentrations.

Table 9. Exogenous Substance Interference Panel   

<table><tr><td rowspan=1 colspan=1>Substance (commercial name)</td><td rowspan=1 colspan=1>Active Ingredient(s)</td><td rowspan=1 colspan=1>Concentration oramount in SBT</td></tr><tr><td rowspan=1 colspan=1>Vaginal antifungal / anti-itch(Nystatin)</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Creams / ointments(Personnelle Hydrocortisone cream)</td><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Anti-hemorrhoidal creams / ointments(Preparation H)</td><td rowspan=1 colspan=1>Phenylephrine HCI</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Antacids (Tums)</td><td rowspan=1 colspan=1>Calcium Carbonate</td><td rowspan=1 colspan=1>0.05% w/v*</td></tr><tr><td rowspan=1 colspan=1>Antacids (Stomaax)*</td><td rowspan=1 colspan=1>Aluminum Hydroxideand MagnesiumHydroxide</td><td rowspan=1 colspan=1>0.05% v/v*</td></tr><tr><td rowspan=1 colspan=1>Enemas(Life BRAND Heavy Mineral Oil USP)</td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Enemas(Mesalazine)</td><td rowspan=1 colspan=1>Mesalazine,5- Aminosalicylic Acid</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Condom with spermicidal lubricant(Trojan)</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>Square of 2 mm2</td></tr><tr><td rowspan=1 colspan=1>Anti-diarrheal medication(Pepto Bismol)</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Anti-diarrheal medication (Imodium)</td><td rowspan=1 colspan=1>LoperamideHydrochloride</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Laxatives (Senokot)</td><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Oral and topical antibiotics(Vancomycin)</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Oral and topical antibiotics(Metronidazole)</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Non-steroidal anti-inflammatory(Aleve)</td><td rowspan=1 colspan=1>Naproxen Sodium</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Moist towelettes(Equate Flushable Moist Wipes)</td><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>Square of 2 mm2</td></tr><tr><td rowspan=1 colspan=1>Moist towelettes (Wet Ones)</td><td rowspan=1 colspan=1>Benzalkonium Chloride</td><td rowspan=1 colspan=1>Square of 2 mm2</td></tr></table>

Calcium carbonate and aluminum hydroxide/magnesium hydroxide showed a potentially inhibitory effect on the detection of toxigenic C. difficile when these substances were present in SBT at a concentration of $0 . 5 \%$ w/v and $0 . 5 \%$ v/v, respectively.

Table 10. Endogenous Substance Interference Panel   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration oramount in SBT</td></tr><tr><td rowspan=1 colspan=1>Fecal fat, triglycerides mix (C2-C10)</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Fecal fat, Palmitic acid</td><td rowspan=1 colspan=1>1.0% v/v</td></tr><tr><td rowspan=1 colspan=1>Fecal fat, Stearic acid</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>0.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>0.5% v/v</td></tr></table>

# Carry-Over and Cross-Contamination

Carry-over and cross-contamination were evaluated by running two studies using three different revogene instruments: two instruments were utilized for the carry-over study and one instrument was used for the cross-contamination study. Ten runs were performed for each study without cleaning between runs. Each study tested high positive $\mathrm { ( \geq 1 0 ^ { 7 } C F U / m L }$ of SB) toxigenic $C .$ . difficile and negative samples contrived in stool matrix.

For the within-run cross-contamination study, four replicates of high positive samples were alternated with four negative samples in each run, and the positions were alternated between each run $\scriptstyle \mathrm { n = 4 0 }$ replicates each sample). For the between-run carry-over study, a run of eight replicates of high positive samples was followed by a run of eight negative samples over five iterations for a total of ten runs $\scriptstyle \mathtt { n } = 4 0$ replicates each sample). No evidence of amplicon carry-over or sample cross contamination was observed.

# Specimen stability

Specimen stability was evaluated within the context of the clinical studies, and it was also assessed analytically by testing pre-characterized positive and negative clinical samples under different storage time and temperature conditions. The results supported the stool specimen storage instructions described in the package insert.

Stool specimens can be stored at $2 5 \mathrm { { ^ \circ C } }$ for up to two days, or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to four days. Inoculated SBT can be stored at $2 5 \mathrm { { ^ \circ C } }$ for up to two days, or at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to three days.

# f. Assay cut-off:

The assay cut-off was established by testing and analyzing clinical stool samples that were known to be positive or negative for toxigenic $C .$ . difficile. The cut-off was confirmed by the analysis of the results from the limit of detection and prospective clinical studies.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the GenePOC CDiff assay was established in a prospective multi-site investigation conducted at seven geographically diverse sites across the US (five sites) and Canada (two sites) in comparison with a combined direct and broth enriched culture method for toxigenic C. difficile. Leftover, deidentified, unformed (liquid or soft) stool specimens were collected from symptomatic subjects suspected of CDI.

Two sets of specimens were evaluated. One specimen set was prospectively collected and tested fresh at six sites with the GenePOC CDiff assay. A second specimen set consisted of specimens that had been prospectively collected in a previous trial and stored frozen prior to testing. Frozen specimens were distributed to three sites for testing with the GenePOC CDiff assay. The reference culture method was performed for each freshly collected specimen (prior to any specimen freezing) at a central reference laboratory.

Specimens that met the study inclusion criteria and that did not meet any of the exclusion criteria were enrolled. A total of 863 fresh specimens and 1718 frozen specimens were initially collected. Of these, 798 fresh specimens and 1665 frozen specimens were eligible for enrollment. Of the eligible specimens, 797 fresh and 1664 frozen specimens had valid results for direct toxigenic culture, broth enriched toxigenic culture, and the GenePOC CDiff assay. These specimens were included in the performance calculations. The population demographics for both specimen sets are described in Table 11.

Table 11. Study Population Demographics of Compliant Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Frozen</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>n = 797</td><td rowspan=1 colspan=1>n = 1664</td></tr><tr><td rowspan=1 colspan=3>Source of sample</td></tr><tr><td rowspan=1 colspan=1>Inpatient</td><td rowspan=1 colspan=1>617 (77.4%)</td><td rowspan=1 colspan=1>1187 (71.3%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>123 (15.4%)</td><td rowspan=1 colspan=1>297 (17.8%)</td></tr><tr><td rowspan=1 colspan=1>Emergency room</td><td rowspan=1 colspan=1>57 (7.2%)</td><td rowspan=1 colspan=1>177 (10.6%)</td></tr><tr><td rowspan=1 colspan=1>Missing</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3 (0.2%)</td></tr><tr><td rowspan=1 colspan=3>Age Class</td></tr><tr><td rowspan=1 colspan=1>&lt;2</td><td rowspan=1 colspan=1>4 (0.5%)</td><td rowspan=1 colspan=1>5 (0.3%)</td></tr><tr><td rowspan=1 colspan=1>3-18</td><td rowspan=1 colspan=1>30 (3.8%)</td><td rowspan=1 colspan=1>75 (4.5%)</td></tr><tr><td rowspan=1 colspan=1>19-60</td><td rowspan=1 colspan=1>399 (50.1%)</td><td rowspan=1 colspan=1>800 (48.1%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 60</td><td rowspan=1 colspan=1>364 (45.7%)</td><td rowspan=1 colspan=1>784 (47.1%)</td></tr></table>

Of the 798 fresh and 1665 frozen eligible specimens that were compliant at the specimen and PCR level according to the study protocol, 12 and 28 were respectively reported indeterminate at initial testing ( $1 . 5 \%$ for the fresh specimens and $1 . 7 \%$ for the frozen specimens) and one frozen specimen remained indeterminate following repeat testing. The indeterminate rate after repeat testing was $0 . 0 \%$ (0/798) for the fresh specimen and $0 . 1 \%$ (1/1665) for the frozen specimens.

The overall initial non-reportable rate (including all indeterminate an unresolved results) for the fresh specimens was $2 . 6 \%$ (21/798), which resolved to $0 . 1 \%$ (1/798) after repeat testing. The overall initial non-reportable rate for the frozen specimens was $2 . 5 \%$ (41/1665), which resolved to $0 . 1 \%$ (1/1665) after repeat testing. Nonreportable assay results (UNR and IND) were excluded from the final performance calculations.

The sensitivity, specificity, Positive Predictive Value (PPV) and Negative Predictive Value (NPV) values were calculated by comparing GenePOC CDiff assay results with the combined results of direct and enriched culture method (Reference Method). Discrepant analysis was performed on a portion of specimens with discordant results between the GenePOC CDiff assay and the culture reference method using FDAcleared molecular tests at each site. The performance of the GenePOC CDiff assay was also reported in comparison to direct culture as positive percent agreement (PPA) and negative percent agreement (NPA).

Comparison with Combined Direct and Broth Enriched Culture The clinical performance of the GenePOC CDiff assay compared to the combined direct and broth enriched toxigenic culture results are presented in Table 12 for fresh specimens and Table 13 for frozen specimens.

Table 12. Combined Direct and Enriched Culture Comparison – Fresh Specimens   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=2>Combined Directand EnrichedCulture Result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>GenePOCCDiff Assay</td><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>20a</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>22b</td><td rowspan=1 colspan=1>664</td><td rowspan=1 colspan=1>686</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>684</td><td rowspan=1 colspan=1>797</td></tr><tr><td rowspan=1 colspan=4>EY</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>80.5%</td><td rowspan=1 colspan=2>(91/113)</td><td rowspan=1 colspan=2>72.3% to86.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=2>(664/684)</td><td rowspan=1 colspan=2>95.5% to98.1%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>82.0%</td><td rowspan=1 colspan=2>(91/111)</td><td rowspan=1 colspan=2>73.8% to88.0%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=2>(664/686)</td><td rowspan=1 colspan=2>95.2% to 97.9%</td></tr></table>

a Of the 20 specimens with false positive GenePOC CDiff test results relative to composite reference culture, eight were positive by an FDA-cleared NAAT for $C .$ . difficile toxin gene DNA.

b Of the 22 specimens with false negative GenePOC CDiff test results relative to composite reference culture, 13 were negative and four were positive by an FDA-cleared NAAT for $C$ . difficile toxin gene DNA.

Table 13. Combined Direct and Enriched Culture Comparison - Frozen Specimens   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=3>Combined Directand EnrichedCulture Result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>GenePOCCDiff Assay</td><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>192</td><td rowspan=1 colspan=2>39a</td><td rowspan=1 colspan=1>231</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>28b</td><td rowspan=1 colspan=2>1405</td><td rowspan=1 colspan=1>1433</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=2>1444</td><td rowspan=1 colspan=1>1664</td></tr><tr><td rowspan=1 colspan=5>EY</td><td rowspan=1 colspan=2>95%CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>87.3%</td><td rowspan=1 colspan=3>(192/220)</td><td rowspan=1 colspan=2>82.2% to91.1%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=3>(1405/1444)</td><td rowspan=1 colspan=2>96.3% to98.0%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>83.1%</td><td rowspan=1 colspan=3>(192/231)</td><td rowspan=1 colspan=2>77.8% to87.4%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=3>(1405/1433)</td><td rowspan=1 colspan=2>97.2% to 98.6%</td></tr></table>

a Of the 39 specimens with false positive GenePOC CDiff test results relative to composite reference culture, 17 were positive by an FDA-cleared NAAT for $C .$ . difficile toxin gene DNA. b Of the 28 specimens with false negative GenePOC CDiff test results relative to composite reference culture, 14 were negative and 12 were positive by an FDA-cleared NAAT for $C .$ . difficile toxin gene DNA.

Comparison with Direct Culture   
The performance of the GenePOC CDiff assay compared to direct toxigenic culture results are presented in Table 14 for fresh specimens and Table 15 for frozen specimens.   
b. Clinical specificity: See section M3a   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

Table 14. Direct Culture Comparison- Fresh Specimens   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=2>Direct CultureResult</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>GenePOCCDiff Assay</td><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>683</td><td rowspan=1 colspan=1>686</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>731</td><td rowspan=1 colspan=1>797</td></tr><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>95%CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=2>(63/66)</td><td rowspan=1 colspan=2>87.5% to 98.4%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>93.4%</td><td rowspan=1 colspan=2>(683/731)</td><td rowspan=1 colspan=2>91.4% to 95.0%</td></tr></table>

Table 15. Direct Culture Comparison - Frozen Specimens   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=2>Direct CultureResult</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>GenePOCCDiff Assay</td><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>231</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1425</td><td rowspan=1 colspan=1>1433</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>1496</td><td rowspan=1 colspan=1>1664</td></tr><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=2>95%CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=2>(160/168)</td><td rowspan=1 colspan=2>90.9% to 97.6%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95.3%</td><td rowspan=1 colspan=2>(1425/1496)</td><td rowspan=1 colspan=2>94.1% to 96.2%</td></tr></table>

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the clinical studies, a total of 2 461 specimens were enrolled during two prospective collections at eight geographically diverse clinical sites, from subjects ranging in age from less than two years old to more than 60 years old. Of these specimens, 333 were positive based on the combined results of direct and enriched toxigenic culture for an overall observed prevalence of $1 3 . 5 \%$ [333/2461; $9 5 \% C \mathrm { I }$ : $1 2 { , } 2 - 1 5 \% ]$ . The overall percentage of positive results observed with the GenePOC CDiff assay with these specimens was $1 1 . 5 \%$ [283/2461; $9 5 \% C \mathrm { I }$ : $1 0 , 3 - 1 2 , 8 \%$ .

N. Instrument Name: revogene instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X_ __ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

The instrument has two barcode readers to identify reagents and patient specimens. Barcodes are used to identify patient specimens, and the SBT and PIE are pre-labeled with a unique barcode to identify both specimen and assay. This provides traceability of the sample ID to the PIE ID, SBT ID, and assay ID.

# 4. Specimen Sampling and Handling:

Samples are manually prepared and transferred to an assay specific PIE for analysis. Briefly, the user is required to pipette the patient specimen into the SBT, transfer the sample into the PIE, and load the PIEs into the instrument. All further specimen handling is automated.

# 5. Calibration:

The system is factory calibrated by the manufacturer, and will undergo performance qualification testing on-site during annual preventive maintenance. If qualification testing results determine significant drift, the instrument will be returned to the manufacturer for re-calibration.

# 6. Quality Control:

Each PIE contains an internal process control $( \mathrm { P r } \mathrm { C } )$ to verify amplification, quality of assay reagents, and sample processing effectiveness and to detect potential inhibition or instrument failures. The PrC is lysed, amplified, and detected along with each specimen tested and monitors the efficacy of the DNA extraction and PCR amplification processes. Commercial control material can be used as a Positive External Control in accordance with applicable regulations or accrediting agencies.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.