# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K172509 B. Purpose for Submission: Clearance of New Device

# C. Measurand:

Target DNA sequences from Herpes Simplex Virus type 1 (HSV-1) and Herpes Simplex Virus type 2 (HSV-2)

# D. Type of Test:

An in vitro molecular diagnostic test for the qualitative detection and differentiation of HSV-1 and HSV-2 DNA from swab specimens collected from anogenital or oral skin lesions

E. Applicant: Vela Diagnostics USA, Inc.

F. Proprietary and Established Names: Sentosa SA201 HSV-1/2 PCR Test

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Sentosa SA201 HSV-l/2 PCR Test is a real-time PCR-based qualitative in vitro diagnostic test for detection and differentiation of Herpes Simplex Virus (HSV-l and HSV-2) DNA from male and

female skin lesions from anogenital or oral sites. The test is intended for use as an aid in diagnosis of herpes infection in symptomatic patients.

Warning: The Sentosa SA201 HSV-l/2 PCR Test is not FDA cleared for use with cerebrospinal fluid (CSF). The test is not intended to be used for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Sentosa SA201 Thermocycler for PCR amplification Sentosa SX101 Instrument for Nucleic Acid Extraction

# I. Device Description:

The Sentosa SA201 HSV-1/2 PCR Test uses the Sentosa SX101 hardware and Sentosa SA201 hardware, along with the associated consumables to operate as an automated sample extraction, PCR setup, amplification and reporting system described under Section L. Test Principle below.

The Sentosa SA201 HSV-1/2 PCR Test workflow starts with extraction of nucleic acids from samples (genital or oral swabs) using the Sentosa SX Virus Total Nucleic Acid Kit on the Sentosa SX101 instrument. Following extraction, the instrument automatically sets up the PCR with the extracted nucleic acids in a 96-well PCR plate. Subsequently, the 96-well PCR plate is sealed and transferred to the Sentosa SA201 for PCR amplification, followed by data analysis.

There are 2 kits (1 assay kit and 1 nucleic acid extraction kit) that are required to perform the Sentosa SA201 HSV-1/2 PCR Test. The kits and components are described below.

Sentosa SA201 HSV-1/2 Qualitative PCR Test Kit (4x24) - Item No. 300216   

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Capcolor</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity(tube)</td><td rowspan=1 colspan=1>Volumetube</td></tr><tr><td rowspan=1 colspan=1>HSV1/2Qal M</td><td rowspan=1 colspan=1>Geen</td><td rowspan=1 colspan=1>Mx1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>70uL</td></tr><tr><td rowspan=1 colspan=1>DNAM(24)</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>Mx2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>400μL</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>Negativecontrol (NC)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>300μL</td></tr><tr><td rowspan=1 colspan=1>HSV1/2Qal PC</td><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>Positivecontrol (PC)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>300μL</td></tr><tr><td rowspan=1 colspan=1>EC3</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>Extractioncontrol</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>200μL</td></tr></table>

Sentosa SX Virus Total Nucleic Acid Kit v2.0 (4x24) - Item No. 300353   

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity (tube)</td><td rowspan=1 colspan=1>Amount</td></tr><tr><td rowspan=1 colspan=1>Virus A1 (24)</td><td rowspan=1 colspan=1>Proteinase K solution</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>350 μL</td></tr><tr><td rowspan=1 colspan=1>Virus A2 (24)</td><td rowspan=1 colspan=1>Magnetic beads</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>600 μL</td></tr><tr><td rowspan=1 colspan=1>Virus A3 (24)</td><td rowspan=1 colspan=1>Lyophilized carrier RNA</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100 μg</td></tr><tr><td rowspan=1 colspan=1>Virus A4 (24)</td><td rowspan=1 colspan=1>Carrier RNA buffer</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>200 μL</td></tr><tr><td rowspan=1 colspan=1>Virus B1 (24)</td><td rowspan=1 colspan=1>Lysis buffer</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6 mL</td></tr><tr><td rowspan=1 colspan=1>Virus B2 (24)</td><td rowspan=1 colspan=1>Binding buffer</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17 mL</td></tr><tr><td rowspan=1 colspan=1>Virus B3 (24)</td><td rowspan=1 colspan=1>Washing buffer 1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17 mL</td></tr><tr><td rowspan=1 colspan=1>Virus B4 (24)</td><td rowspan=1 colspan=1>Washing buffer 2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17 mL</td></tr><tr><td rowspan=1 colspan=1>Virus B6 (24)</td><td rowspan=1 colspan=1>Elution buffer</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6 mL</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): IMDx HSV-1/2 for Abbott $m 2 0 0 0$ (Intelligent Medical Devices, Inc.)

2. Predicate 510(k) number(s): K140198

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Characteristics</td><td colspan="1" rowspan="1">Sentosa SA201HSV-1/2 PCR Test(K172509)</td><td colspan="1" rowspan="1">IMDx HSV-1/2 forAbbott m2000 Assay(K140198)</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3305</td><td colspan="1" rowspan="1">21 CFR 866.3305</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OQO</td><td colspan="1" rowspan="1">OQO</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Sentosa SA201 HSV-1/2 PCR Test is a real-timePCR-based qualitative invitro diagnostic test fordetection anddifferentiation of HerpesSimplex Virus (HSV-1 andHSV-2) DNA from maleand female skin lesionsfrom anogenital or oralsites. The test is intendedfor use as an aid indiagnosis of herpesinfection in symptomaticpatients.Warning: The SentosaSA201 HSV-1/2 PCR Testis not FDA cleared for usewith cerebrospinal fluid(CSF). The test is notintended to be used forprenatal screening.</td><td colspan="1" rowspan="1">The IMDx HSV-1/2 forAbbott m2000 assay is anin vitro diagnostic test forthe direct, qualitativedetection anddifferentiation of HerpesSimplex Virus type 1(HSV-1) and type 2(HSV-2) DNA from maleand female skin lesionsfrom anogenital or oralsites. The test is intendedfor use as an aid in thediagnosis of HSVinfection in symptomaticpatients. The assay isintended to be run on theAbbott m2000 instrumentsystem.Warning: The IMDxHSV-1/2 for Abbottm2000 assay is not FDA-cleared for use withcerebrospinal fluid (CSF).The assay is not intendedfor pre-natal screening.</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Real-time PCR DNAamplification</td><td colspan="1" rowspan="1">Real-time PCR DNAamplification</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative detection anddifferentiation of HSV-1and HSV-2 DNA</td><td colspan="1" rowspan="1">Qualitative detection anddifferentiation of HSV-1and HSV-2 DNA</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Male and female skinlesions from anogenital ororal sites</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Characteristics</td><td colspan="1" rowspan="1">Sentosa SA201HSV-1/2 PCR Test(K172509)</td><td colspan="1" rowspan="1">IMDx HSV-1/2 forAbbott m2000 Assay(K140198)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">HSV-1 and HSV-2 UL30gene common sequences</td><td colspan="1" rowspan="1">HSV-1 Glycoprotein Dgene and HSV-2 UL30gene</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Sentosa SX101 Instrumentfor Nucleic Acid Extrcationand Sentosa SA201Thermo-cycler for PCRamplification</td><td colspan="1" rowspan="1">Sample extraction andreal-time PCRamplification/detectionusing the Abbott m2000system.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): Not Applicable

# L. Test Principle:

The Sentosa SA201 HSV-1/2 PCR Test (4x24) contains reagents and enzymes for specific amplification of a 104 bp fragment of UL30 gene common to both HSV-1 and HSV-2, and specific probes for the direct detection and differentiation of HSV-1 and HSV-2 amplicons, respectively. Pathogen detection by PCR is based on the amplification of specific regions of the pathogen genome. In real-time PCR, the amplified product is detected via fluorescent dyes, which are usually linked to oligonucleotide probes that bind specifically to the target sequences. Real-time monitoring of the fluorescence intensities during a PCR run allows the detection of the accumulating product. Amplification of the targets occurs in three channels: green, orange and red on the Sentosa SA201. Output is recorded as the increase of fluorescence over time in comparison to background signal. Monitoring the fluorescence intensities during the PCR run allows the detection of the accumulating product without having to re-open the reaction tubes after the PCR run.

# Sample Preparation

HSV-1/2 nucleic acids are extracted from swabs in universal virus transport medium (UTM) automatically using the Sentosa SX101 instrument. Briefly, the samples are collected in $1 . 5 ~ \mathrm { m L }$ Sentosa SX Safe-Lock Tubes and loaded onto the Sentosa SX101 worktable together with relevant reagents from Sentosa SX Virus Total Nucleic Acid Kit v2.0 for extraction. Using the instrument, the samples are automatically lysed to release the nucleic acids which are then washed and eluted in the Sentosa SA 96-Well Optical Plate. PCR set-up is subsequently performed.

# Reagent Preparation

The Sentosa SA201 HSV-1/2 PCR Test comprises reagents required for PCR amplification. The reagent tubes are loaded onto the Sentosa SX101 worktable and the Sentosa SX101 instrument automatically combines the reagents. The resultant master mix is subsequently dispensed into the Sentosa SA 96-Well Optical Plate where it is mixed with the eluted nucleic acids.

# Amplification

PCR is performed using the Sentosa SA201 instrument. The recombinant DNA polymerase in DNA4 M2 and primers in HSV-1/2 Qual M1 of the Sentosa SA201 HSV-1/2 PCR Test are used to amplify the target DNA sequence. The first step of PCR amplification is the denaturation and separation of the double stranded DNA at high temperature. The second step involves the annealing of the specific primers to the targeted site on the single stranded DNA. This is followed by the third step where extension of the primers/DNA synthesis occurs with the DNA polymerase. These steps occur during each cycle of the PCR amplification and multiple cycles will increase the amplified PCR products exponentially.

# Detection

The Sentosa SA201 instrument detects the amplified products via fluorophore emission. In realtime PCR, the amplified product is detected via fluorescent dyes, which are conjugated to oligonucleotide probes that bind specifically to the target sequences.

The fluorophore is released during PCR amplification when the probe binds to the complementary sequence of the target and gets hydrolyzed and released. PCR product is detected accordingly from the free probes. Amplification of the targets occurs in three channels: green (HSV-1), orange (HSV-2) and red (extraction control, EC) on Sentosa SA201 Real-Time PCR instrument. Output is recorded as the increase of fluorescence over time relative to background signal.

Real-time monitoring of the fluorescence intensities during a PCR run allows the detection of the accumulating product. Monitoring the fluorescence intensities during the PCR run at each cycle allows the detection of the accumulating product without having to re-open the reaction tubes after the PCR run.

# Sentosa SA201 HSV-1/2 PCR Test workflow

The Sentosa SA201 HSV-1/2 PCR Test workflow starts with extraction of nucleic acids from samples (genital or oral swabs) using the Sentosa SX Virus Total Nucleic Acid Kit on the Sentosa SX101 instrument. Following extraction, the instrument will automatically set up the PCR with the extracted nucleic acids in a 96-well PCR plate. Subsequently, the 96-well PCR plate is sealed and transferred to the Sentosa SA201 for PCR amplification, followed by data analysis.

Drivers are installed on the Sentosa Link to connect the Sentosa SX101 instrument and the Sentosa SA201 thermocycler. This creates a user environment that links the SX101 and the Sentosa SA201 to facilitate automated workflow to export results in a LIS/LIMScompatible format.

The Sentosa SA201 HSV-1/2 PCR Test uses the Sentosa SX101 hardware and Sentosa SA201 hardware, along with the associated consumables to operate as an automated sample extraction, PCR setup, amplification and reporting system as shown in the figures below.

# Sentosa® SA201 HSV1/2 PCR Test

#

![](images/4a8410534752fcf0a708b20b3e08117016aad3f1598fbfe35caa974b7b9ffcac.jpg)

![](images/a83890dc1b8ecac8e88f988543d1e1ee1d79f38c87645a6320a50345f31972ba.jpg)

# Data analysis

The following tables provide the basis of result interpretation for the Sentosa SA201 Reporter in the “Automated data analysis and result interpretation”.

Ct value ranges

Fluorescence channel Green detects HSV-1, fluorescence channel Orange detects HSV-2, and fluorescence channel Red detects the extraction control (EC). The following table defines the Ct values validity criteria and results per channel for the Negative Control, Positive Control and any samples.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Greenchannel(HSV-1)</td><td rowspan=1 colspan=1>Orangechannel(HSV-2)</td><td rowspan=1 colspan=1>Red channel(EC)</td></tr><tr><td rowspan=1 colspan=1>Negativecontrol</td><td rowspan=1 colspan=1>&lt;5,&gt;40or no Ct (-)</td><td rowspan=1 colspan=1>&lt;5,&gt;40or no Ct (-)</td><td rowspan=1 colspan=1>25-35 (+)</td></tr><tr><td rowspan=1 colspan=1>Positivecontrol</td><td rowspan=1 colspan=1>22 - 30 (+)</td><td rowspan=1 colspan=1>25-32 (+)</td><td rowspan=1 colspan=1>&lt;25, &gt;35 or no Ct (-)OR25-35 (+)</td></tr><tr><td rowspan=1 colspan=1>NegativeSample</td><td rowspan=1 colspan=1>&lt;5, &gt;40 orno Ct (-)</td><td rowspan=1 colspan=1>&lt;5, &gt;40 orno Ct (-)</td><td rowspan=1 colspan=1>25-35 (+)</td></tr><tr><td rowspan=1 colspan=1>PositiveSample</td><td rowspan=1 colspan=1>5-40 (+)</td><td rowspan=1 colspan=1>5-40 (+)</td><td rowspan=1 colspan=1>&lt;25, &gt;35 or no Ct (-)OR25-35 (+)</td></tr></table>

# Result interpretation

<table><tr><td rowspan=1 colspan=3>Results observed per channel</td><td rowspan=2 colspan=1>Result Interpretation</td></tr><tr><td rowspan=1 colspan=1>HSV-1Green</td><td rowspan=1 colspan=1>HSV-2Orange</td><td rowspan=1 colspan=1>ECRed</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>HSV-1 and HSV-2DNAnotdetected</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>HSV-1 DNA detected* and HSV-2 DNA not detected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>HSV-2 DNA detected* and HSV-1 DNA not detected</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>HSV-1   and   HSV-2   DNAdetected*</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample invalid</td></tr></table>

# Performance Characteristics:

1. Analytical performance:

# a. Precision/Reproducibility:

The precision of Sentosa SA201 HSV-1/2 PCR Test was assessed using a sevenprecision panel consisting of the assay negative control, assay positive control and 2 strains of HSV: HSV-1 MacIntyre and HSV-2 Strain G, diluted in universal viral transport medium. Other than the controls, panel members were formulated with a single HSV strain present (HSV-1 or HSV-2) at two concentrations; Positive (3x LoD) and Low Positive ( $1 . 5 \mathrm { ~ x ~ }$ LoD). A seventh panel member labeled Negative (universal viral transport medium) was prepared using the universal viral transport medium only. The data were used to determine the mean $\mathrm { C t }$ , standard deviation (SD) and the coefficient of variation $( \% C V )$ for each target and the extraction control.

For the within-laboratory precision study, the seven-member panel was tested in replicates of three, four runs per day for a total of five days. The tests were conducted by five alternating operators using three Sentosa SX101, four Sentosa SA201 and three Sentosa SA201 HSV-1/2 PCR Test lots.

Summary of Within-laboratory Precision study results   

<table><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-1</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>31.79</td><td rowspan=1 colspan=1>0.81</td></tr><tr><td rowspan=1 colspan=1>3x LoDHSV-1</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>30.72</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-2</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>30.25</td><td rowspan=1 colspan=1>1.09</td></tr><tr><td rowspan=1 colspan=1>3x LoD HSV-2</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>59/59 100%)*</td><td rowspan=1 colspan=1>93.89 - 100%</td><td rowspan=1 colspan=1>29.35</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>28.98</td><td rowspan=1 colspan=1>2.03</td></tr><tr><td rowspan=1 colspan=1>Assay NegativeControl</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>29.08</td><td rowspan=1 colspan=1>2.24</td></tr><tr><td rowspan=2 colspan=1>Assay PositiveControl</td><td rowspan=2 colspan=1>GreenOrange</td><td rowspan=2 colspan=1>60/60 (100%)</td><td rowspan=2 colspan=1>93.98 - 100%</td><td rowspan=1 colspan=1>26.09</td><td rowspan=1 colspan=1>1.10</td></tr><tr><td rowspan=1 colspan=1>28.51</td><td rowspan=1 colspan=1>0.69</td></tr></table>

\* Total number of sample is 59 due to exclusion of 1 invalid sample

Reproducibility:

For site-to-site reproducibility, the reproducibility test panel samples were tested in replicates of three, two runs per day by two operators for a total of five days in three clinical sites in the USA. Each clinical site used one Sentosa SX101, one Sentosa SA201, one unique lot of Sentosa SA201 HSV-1/2 PCR Test and one unique lot of Sentosa SX Virus Total Nucleic Acid kit. Thus the reproducibility study tested three different lots of assay and extraction kits. The results are summarized in the

following table.

<table><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean Ct±SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-1</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>31.45 ±1.65</td><td rowspan=1 colspan=1>5.25%</td></tr><tr><td rowspan=1 colspan=1>3x LoD HSV-1</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>30.23 ±1.01</td><td rowspan=1 colspan=1>3.34%</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD HSV-2</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>29.29 ±0.47</td><td rowspan=1 colspan=1>1.60%</td></tr><tr><td rowspan=1 colspan=1>3x LoD HSV-2</td><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>28.01 ±1.18</td><td rowspan=1 colspan=1>4.21%</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>26.93 ±0.60</td><td rowspan=1 colspan=1>2.23%</td></tr><tr><td rowspan=2 colspan=1>PC</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>25.80 ±0.26</td><td rowspan=1 colspan=1>1.01%</td></tr><tr><td rowspan=1 colspan=1>Orange</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>28.32 ±0.32</td><td rowspan=1 colspan=1>1.13%</td></tr><tr><td rowspan=1 colspan=1>Negative sample</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>27.17 ±1.14</td><td rowspan=1 colspan=1>4.20%</td></tr><tr><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.91 - 100%</td><td rowspan=1 colspan=1>26.81 ±0.55</td><td rowspan=1 colspan=1>2.05%</td></tr></table>

b. Linearity/assay reportable range: Not applicable c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Positive Control (PC): The PC supplied in a tube is placed into the Sentosa SX101 sample rack during the workflow run. It consists of two linearized plasmids with sequences for HSV-1 and HSV-2 detection by primers and probes in the Sentosa SA201 HSV-1/2 goes through extraction and PCR setup as any sample in the run. Thus, it also monitors for instrument/workflow issues, such as inhibition or pipetting errors due to instrument/workflow failure.

Negative Control (NC): The NC is nucleic acid-free water supplied in a tube that is placed into the Sentosa SX101 sample rack during the workflow run. It can monitor for substantial PCR reagent failure since the NC goes through extraction and PCR setup as any sample in the run. Thus, it also monitors for instrument/workflow issues, such as reagent and/or environmental contamination or pipetting errors due to instrument/workflow failure.

Extraction Control (EC): The EC supplied in a tube is placed into the Sentosa SX101 during the workflow run. It consists of one linearized plasmid with sequence for an unrelated tobacco mosaic virus that is detected by EC primers and probe in the

Sentosa SA201 HSV-1/2 PCR Test reagents. The EC is added to the lysis buffer, which is used in the extraction of all samples, NC and PC before PCR setup. Thus EC monitors for sample extraction failure or and/or inhibition, and also instrument/workflow issues, such as inhibition or pipetting errors due to instrument/workflow failure.

# d. Detection limit:

(i) A Limit of Detection (LoD) study was performed to evaluate the analytical sensitivity of the Sentosa SA201 HSV-1/2 PCR Test using two representative strains of HSV-1 (McIntyre & KOS) and two representative strains of HSV-2 (MS & G). The study included serial dilutions of quantified viral cultures of HSV-1 MacIntyre, HSV-1 KOS, HSV-2 MS, and HSV-2 G. and the preliminary LoD was determined. The LoD for each strains was confirmed by testing at a level within the LoD range with 20 replicates. The LoD analytical sensitivity study was conducted over a period of five (5) days with three (3) reagent lots, four (4) operators and four (4) instrument systems. The LoD for each strain was determined using probit analysis.

The LoD (with equal or more than $9 5 \%$ positive detection rate) for HSV-1 was 40 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and for HSV-2 was $4 ~ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , and the LoB (with equal or less than $5 \%$ positive detection rate) for HSV-1 was $0 . 8 1 ~ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and for HSV-2 was $0 . 0 4 ~ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The final LoDs are presented in the table below.

# LoD of Sentosa SA201 HSV-1/2 PCR Test

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD (TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>HSV-1MacIntyre</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>HSV-1KOS</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>HSV-2MS</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>HSV-2Strain G</td><td rowspan=1 colspan=1>4</td></tr></table>

(ii) The analytical reactivity of the Sentosa SA201 HSV-1/2 PCR Test was assessed to determined whether the test could detect a diverse range of HSV-1 and HSV-2 strains. A total of 40 clinical isolates (20 HSV-1 and 20 HSV-2) were collected from male and female genital and oral lesions from different locations that were quantitated and diluted to1x LoD. The Sentosa SA201 HSV-1/2 PCR Test detected all 40 strains tested at 1x LoD.

# e. Analytical specificity/Coss-reactivity:

A panel of microorganism that may be present in patient specimens was tested to determined whether these microorganisms interfered with the detection of HSV-1 or HSV-2 or were cross-reactive with the Sentosa SA201 HSV-1/2 Qualitative PCR Test. Organisms were tested at a target concentration of approximately $1 \mathrm { ~ x ~ } \ : 1 0 ^ { 6 }$ CFU/mL for bacteria and fungi or $1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ for viruses in the absence and presence each of $1 . 5 \mathrm { x }$ LoD of each of two HSV strains: HSV-1 MacIntyre or HSV2 MS. None of the potential interfering organisms cross-reacted or interfered with the detection of any of the HSV strains by the Sentosa SA201 HSV-1/2 PCR Test. In addition, there was no cross-reactivity within the multiplex panel (HSV-1 and HSV2) in the presence of high concentration of HSV-1 or HSV-2.

Cross-Reactivity Panel Tested   

<table><tr><td rowspan=1 colspan=1>Cross-Reactlvity Panel Tested</td><td rowspan=1 colspan=1>Cross-Reactlvity Panel Tested</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata, NCYC 388</td><td rowspan=1 colspan=1>Human papillomavirus type 16</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii, 2208</td><td rowspan=1 colspan=1>Human papillomavirus type 18</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Human DNA</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwofii</td><td rowspan=1 colspan=1>Human herpesvirus (HHV6)</td></tr><tr><td rowspan=1 colspan=1>Actinomyces isralii, serotype 1</td><td rowspan=1 colspan=1>Human herpesvirus 4, B95-8</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1, strain Adenoid 71</td><td rowspan=1 colspan=1>Klebsiella pneumonia, NCTC 9633</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7, strain Gomen</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis, VPI2553</td><td rowspan=1 colspan=1>Maraxella catarrhalis, strain 20</td></tr><tr><td rowspan=1 colspan=1>Candida albicans, strain 132</td><td rowspan=1 colspan=1>Mobiluncus curtisii, BV 345-16</td></tr><tr><td rowspan=1 colspan=1>Candida krusei, NRRL Y-6</td><td rowspan=1 colspan=1>Mobiluncus muleris, BV 64-5</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis, NRRL-Y-12969</td><td rowspan=1 colspan=1>Mycoplasma hominis, PG21</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis, PK233</td><td rowspan=1 colspan=1>Neisseria gonorrhea, B-585</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis, UW-57/Cx</td><td rowspan=1 colspan=1>Neisseria meningitides, serogroup B</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile, strain 4118</td><td rowspan=1 colspan=1>Prevotella melaninogenica, B282</td></tr><tr><td rowspan=1 colspan=1>CMV, AD-169</td><td rowspan=1 colspan=1>Rubella virus</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium, 392-1</td><td rowspan=1 colspan=1>Staphylococcus aureus, F-182</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans, 52</td><td rowspan=1 colspan=1>Staphylococcus aureus, FDA 209</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae, QC strain</td><td rowspan=1 colspan=1>Staphylococcus epidermidis, 255-01B</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis, AGR329</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus, LRA27.02</td></tr><tr><td rowspan=1 colspan=1>Enterovirus type 71, BrCr</td><td rowspan=1 colspan=1>Streptococcus mitis, NCTC 12261</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus/Human herpesvirus 4,strain P-3</td><td rowspan=1 colspan=1>Streptococcus mutans, UA159</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O103, strain NCDC</td><td rowspan=1 colspan=1>Streptococcus pneumonia, CIP 104225</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorumsubsp necrophorum,</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatumsubsp nucleatum, 1612A</td><td rowspan=1 colspan=1>SV40 (Simian virus 40)</td></tr><tr><td rowspan=1 colspan=1>Gardnerella/Haemophilus vaginalis, 317</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Haemophilus ducreyi, CIP 542</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Hepatitis A virus, strain PA21</td><td rowspan=1 colspan=1>Varicella-Zoster Virus (VZV)</td></tr><tr><td rowspan=1 colspan=1>HIV-1, Group M Subtype C</td><td rowspan=1 colspan=1></td></tr></table>

# f. Interfering Substances

A panel of 31 substances that may be present in oral/genital patient specimens was tested to determine whether these substances interfered with the performance of the Sentosa SA201 HSV-1/2 PCR Test. Two strains of HSV: HSV-1 MacIntyre and HSV-2

MS, were diluted to approximately 3x LoD in universal viral transport medium and spiked with each potentially inhibitory substance. None of the substances showed an inhibitory effect on the detection of HSV-1 or HSV-2 by the Sentosa SA201 HSV-1/2 PCR Test.

Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=3>Iliterlerig Substalices Tested</td></tr><tr><td rowspan=1 colspan=1>Potentially InterferingSubstance</td><td rowspan=1 colspan=1>Active Ingredients</td><td rowspan=1 colspan=1>Concentration ofS ubstance</td></tr><tr><td rowspan=1 colspan=1>Acyclovir</td><td rowspan=1 colspan=1>Acycloguanosine (10%)</td><td rowspan=1 colspan=1>7.0 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Whole blood with EDTA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Female urine</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Male urine</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>3.3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Saliva</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>K.Y. Jelly lubricant</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Feminine wash</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Xylocaine 5%</td><td rowspan=1 colspan=1>Lidocaine (50mg)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Toothpaste</td><td rowspan=1 colspan=1>Stannous fluoride (0.454%)</td><td rowspan=1 colspan=1>0.53% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Desitin maximumoriginal paste</td><td rowspan=1 colspan=1>Zinc Oxide (40%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Mentholatum lip balm</td><td rowspan=1 colspan=1>Menthol (0.7%), Camphor (1.7%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Listerine anti-bacterialmouthwash</td><td rowspan=1 colspan=1>Eucalyptol (0.092%), Menthol (0.042%),Methy Salicylate (0.060%), Thymol(0.064%)</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>7.0 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Douche</td><td rowspan=1 colspan=1>Providone-iodine (10% w/v)</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Yeast Gard</td><td rowspan=1 colspan=1>Candida albicans 27X* HPUS, Candidaparapsilosis 27X* HPUS Pulsatilla 27*HPUS</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Vaginal contraceptive gel</td><td rowspan=1 colspan=1>Nonoxynol-9 (4%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Vaginal contraceptive gel</td><td rowspan=1 colspan=1>Nonoxynol-9 (3%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Monistat-7</td><td rowspan=1 colspan=1>Miconazole nitrate vaginal cream (2%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Fleet</td><td rowspan=1 colspan=1>Benzalkonium (0.06% w/w)/EDTA</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Gyno-Trosyd</td><td rowspan=1 colspan=1>Tioconazole (100mg)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole 1% Cream</td><td rowspan=1 colspan=1>Clotrimazole (50mg, 1%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Anti-Inch Cream</td><td rowspan=1 colspan=1>Lidocaine ph. Eur. (2% w/w)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Abreva</td><td rowspan=1 colspan=1>Docosanol (10%)</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Buffy coat</td><td rowspan=1 colspan=1>White blood cell</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Talcum powder</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Seminal fluid</td><td rowspan=1 colspan=1>Seminal fluid</td><td rowspan=1 colspan=1>7.0 % (v/v)</td></tr><tr><td rowspan=1 colspan=1>Feces</td><td rowspan=1 colspan=1>Feces</td><td rowspan=1 colspan=1>7.0 % (w/v)</td></tr><tr><td rowspan=1 colspan=1>Corn starch</td><td rowspan=1 colspan=1>Corn starch</td><td rowspan=1 colspan=1>1.25 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Paracetamol</td><td rowspan=1 colspan=1>Acetamidophenol</td><td rowspan=1 colspan=1>5.0 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Aspirin</td><td rowspan=1 colspan=1>Acetylsalicyclic acid</td><td rowspan=1 colspan=1>10 mg/mL</td></tr></table>

# g. Cross-contamination and carryover-contamination

The Cross-contamination and Carry-over studies were assessed to evaluate the potential cross-contamination during extraction and on subsequent runs on the performance of the Sentosa SA201 HSV1/2 Test with the Sentosa SX101 workflow instrument. The carryover and cross-contamination studies were conducted over a period of four (4) days with one (1) reagent lot, two (2) operators, and two (2) Sentosa workflow systems. The test materials used in the study included negative samples and $1 \mathrm { x } 1 0 ^ { \sim } 5$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ HSV1 (2500xLoD) for the cross-contamination study, a NC, a PC, and negative samples for the carry-over contamination study. Three (3) run matrices were used, two (2) for the cross– contamination study and one (1) for the carry-over contamination study. The results showed that there was no contamination. All 96 positive samples were detected as positive, and the 183 negative samples were detected as negative (no amplification signal noted).

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

The clinical performance evaluation was performed against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable b. Clinical specificity: Not applicable c. Other clinical supportive data (when a. and b. are not applicable):

The performance of the Sentosa SA201 HSV-1/2 PCR Test was compared with the ELVIS HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance:

A total of 2684 samples were collected throughout the study from eight sample collection sites in the United States from 2016 – 2017 and then tested at four locations in the United States to evaluate the performance of the Sentosa SA201 HSV-1/2 PCR

Test. Concurrently the samples were tested by the reference method (ELVIS HSV ID and $\mathrm { D } ^ { 3 }$ Typing Test System) for performance comparison. A total of 389 samples were excluded due to no/invalid ELVIS/Sentosa SA201 HSV-1/2 PCR Test results, wrong lesion type and various administrative errors.

Of the 2295 valid samples, 1978 were anogenital lesions and 317 were oral lesions. In the ELVIS reference test, a sample that was HSV-2 positive could not be used to type for HSV-1. Thus, the total number of HSV-2 positive samples is deducted from the total sample number for HSV-1 analysis.

Three hundred and ninety seven (397) anogenital prospective specimens identified as HSV-2 positive by ELVIS viral culture were removed from the initial 1978 anogenital specimens for the calculation of the HSV-1 clinical performance. Due to low prevalence of HSV-2 in oral specimens, only three (3) oral specimens identified as HSV-2 positive by ELVIS viral culture were removed from the initial 317 oral specimens for the calculation of the HSV-1 clinical performance.

Results from the prospective studies are presented in the tables below.

HSV-1 Results for Anogenital Specimens   

<table><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>281</td><td rowspan=1 colspan=1>54b</td><td rowspan=1 colspan=1>335</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>9a</td><td rowspan=1 colspan=1>1237</td><td rowspan=1 colspan=1>1246</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>1291</td><td rowspan=1 colspan=1>1581</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>Lower 95% CI</td><td rowspan=1 colspan=1>Upper 95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>96.90%</td><td rowspan=1 colspan=1>94.21%</td><td rowspan=1 colspan=1>98.36%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95.82%</td><td rowspan=1 colspan=1>94.58%</td><td rowspan=1 colspan=1>96.78%</td></tr></table>

aFrom sequencing analysis, 4 discordant samples (HSV-1 positive by ELVIS and HSV-1 negative by Sentosa) were in agreement with Sentosa results, 2 were in agreement with ELVIS results and 2 were not in agreement with Sentosa nor ELVIS results. 1 discordant sample was not tested due to insufficient volume. bFrom sequencing analysis, 44 discordant samples (HSV-1 negative by ELVIS and HSV-1 positive by Sentosa) were in agreement with Sentosa results and 10 were in agreement with ELVIS results.

HSV-2 Results for Anogenital Specimens   

<table><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>147d</td><td rowspan=1 colspan=1>538</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1434</td><td rowspan=1 colspan=1>1440</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1581</td><td rowspan=1 colspan=1>1978</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>Lower 95% CI</td><td rowspan=1 colspan=1>Upper 95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>98.49%</td><td rowspan=1 colspan=1>96.74%</td><td rowspan=1 colspan=1>99.31%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>90.70%</td><td rowspan=1 colspan=1>89.17%</td><td rowspan=1 colspan=1>92.04%</td></tr></table>

cFrom sequencing analysis, 4 discordant samples (HSV-2 positive by ELVIS and HSV-2 negative by Sentosa) were in agreement with Sentosa results and 2 were in agreement with ELVIS results. eOut of 147 discrepant samples, 142 were unique samples (3 samples were ELVIS HSV-1 positive/HSV-2 negative and Sentosa HSV1 negative/HSV2 positive and were double-counted in the 9 samples that are ELVIS HSV-1 positive and Sentosa HSV1 negative. 2 samples were ELVIS HSV-1 negative/HSV-2 negative and Sentosa HSV1 positive/HSV-2 positive. These were double-counted in the 54 samples that are ELVIS HSV-1 negative and Sentosa HSV-1 positive.) From sequencing analysis, 119 discordant samples (HSV2 negative by ELVIS and HSV2 positive by Sentosa) were in agreement with Sentosa results and 20 were in agreement with ELVIS results. Three discordant samples was not tested due to insufficient volume

HSV-1 Results for Oral Specimens   

<table><tr><td rowspan=2 colspan=1>HSV-1</td><td rowspan=1 colspan=4>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=2>32</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>203</td><td rowspan=1 colspan=1>203</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=2>235</td><td rowspan=1 colspan=1>314</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Estimate</td><td rowspan=1 colspan=1>Lower 95% CI Upper 95% CI</td><td rowspan=1 colspan=1>Lower 95% CI Upper 95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=2>100.00%</td><td rowspan=1 colspan=1>95.36%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=2>86.38%</td><td rowspan=1 colspan=1>81.41%</td><td rowspan=1 colspan=1>90.19%</td></tr></table>

cFrom sequencing analysis, 27 discordant samples (HSV-1 negative by ELVIS and HSV-1 positive by Sentosa) were in agreement with Sentosa results and 4 were in agreement with ELVIS results. One discordant sample was not tested due to insufficient volume.

HSV-2 Results for Oral Specimens   

<table><tr><td rowspan=2 colspan=1>HSV-2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1f</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>314</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>317</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>Lower 95% CI Upper 95% CI</td><td rowspan=1 colspan=1>Lower 95% CI Upper 95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>66.67%</td><td rowspan=1 colspan=1>20.77%</td><td rowspan=1 colspan=1>93.85%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>99.68%</td><td rowspan=1 colspan=1>98.22%</td><td rowspan=1 colspan=1>99.94%</td></tr></table>

fFrom sequencing analysis, one sample (HSV2 positive/HSV1 negative by ELVIS and HSV2 negative/HSV1 positive by Sentosa) was in agreement with Sentosa results. gOne discordant sample (HSV1/2 negative by ELVIS and HSV2 positive by Sentosa) was not tested due to insufficient volume.

# HSV-2 oral lesion contrived specimen results:

A contrived specimen study was performed to provide additional performance data for detection of HSV-2 in oral samples. Thirty (30) contrived HSV-2 positive oral samples were prepared by spiking HSV-2 virus into HSV-negative oral samples. HSV-2 was spiked in HSV-negative oral samples at concentrations $1 . 5 ~ \mathrm { X }$ LoD, 3 X LoD, $1 0 ~ \mathrm { X }$ LoD, 100 X LoD, 1,000 X LoD and 10,000 X LoD. In addition, fifteen (15) HSV-1 positive oral lesion samples and 15 HSV-1/HSV-2 negative oral samples. All 30 HSV2 contrived oral lesion samples were detected in all contrived samples at all concentration.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Prevalence: The observed prevalence of HSV-1 and HSV-2 in the prospective clinical study of the Sentosa SA201 HSV-1/2 PCR Test clinical study varied between age groups for both oral lesions and anogenital lesions, and is shown in the tables below. The prevalence rates for HSV-1 were individually established as $2 1 . 2 \%$ (335/1581) for anogenital samples and $3 5 . 0 \%$ (110/314) for oral samples. The prevalence rates for HSV2 were individually established as $2 7 . 2 \%$ (538/1978) for anogenital samples and $0 . 9 \%$ (3/317) for oral samples.

Distribution of samples according to demographics for anogenital lesions as tested by Sentosa SA201 HSV-1/2 PCR Test   

<table><tr><td rowspan=2 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0 - 10</td><td rowspan=1 colspan=1>2/27(7.4%)</td><td rowspan=1 colspan=1>1/37(2.7%)</td><td rowspan=1 colspan=1>3/64(4.7%)</td><td rowspan=1 colspan=1>0/27(0.0%)</td><td rowspan=1 colspan=1>0/37(0.0%)</td><td rowspan=1 colspan=1>0/64(0.0%)</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>59/188(31.4%)</td><td rowspan=1 colspan=1>8/40(20.0%)</td><td rowspan=1 colspan=1>67/228(29.4%)</td><td rowspan=1 colspan=1>51/232(22.0%)</td><td rowspan=1 colspan=1>6/43(14.0%)</td><td rowspan=1 colspan=1>57/275(20.7%)</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>126/400(31.5%)</td><td rowspan=1 colspan=1>19/103(18.4%)</td><td rowspan=1 colspan=1>145/503(28.8%)</td><td rowspan=1 colspan=1>173/538(32.2%)</td><td rowspan=1 colspan=1>55/146(37.7%)</td><td rowspan=1 colspan=1>228/684(33.3%)</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>50/254(19.7%)</td><td rowspan=1 colspan=1>7/70(10.0%)</td><td rowspan=1 colspan=1>57/324(17.6%)</td><td rowspan=1 colspan=1>71/300(23.7%)</td><td rowspan=1 colspan=1>14/78(17.9%)</td><td rowspan=1 colspan=1>85/378(22.5%)</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>27/185(14.6%)</td><td rowspan=1 colspan=1>4/28(14.3%)</td><td rowspan=1 colspan=1>31/213(14.6%)</td><td rowspan=1 colspan=1>58/221(26.2%)</td><td rowspan=1 colspan=1>11/36(30.6%)</td><td rowspan=1 colspan=1>69/257(26.8%)</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>22/105(21.0%)</td><td rowspan=1 colspan=1>1/27(3.7%)</td><td rowspan=1 colspan=1>23/132(17.4%)</td><td rowspan=1 colspan=1>41/139(29.5%)</td><td rowspan=1 colspan=1>12/31(38.7%)</td><td rowspan=1 colspan=1>53/170(31.2%)</td></tr><tr><td rowspan=1 colspan=1>61 - 70</td><td rowspan=1 colspan=1>5/61(8.2%)</td><td rowspan=1 colspan=1>2/12(16.7%)</td><td rowspan=1 colspan=1>7/73(9.6%)</td><td rowspan=1 colspan=1>21/77(27.3%)</td><td rowspan=1 colspan=1>4/14(28.6%)</td><td rowspan=1 colspan=1>25/91(27.5%)</td></tr><tr><td rowspan=1 colspan=1>71 - 80</td><td rowspan=1 colspan=1>2/22(9.1%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>2/31(6.5%)</td><td rowspan=1 colspan=1>12/30(40.0%)</td><td rowspan=1 colspan=1>3/11(27.3%)</td><td rowspan=1 colspan=1>15/41(36.6%)</td></tr><tr><td rowspan=1 colspan=1>81 - 90</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/10(0.0%)</td><td rowspan=1 colspan=1>3/9(33.3%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>5/15(33.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;90</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>0/0(0.0%)</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>1/3(33.3%)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>293/1248(23.5%)</td><td rowspan=1 colspan=1>42/333(12.6%)</td><td rowspan=1 colspan=1>335/1581(21.2%)</td><td rowspan=1 colspan=1>430/1573(27.3%)</td><td rowspan=1 colspan=1>108/405(26.7%)</td><td rowspan=1 colspan=1>538/1978(27.2%)</td></tr></table>

Distribution of samples according to demographics for oral lesions as tested by Sentosa SA201 HSV-1/2 PCR Test   

<table><tr><td rowspan=2 colspan=1>Age (years)</td><td rowspan=1 colspan=3>HSV-1</td><td rowspan=1 colspan=3>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=2 colspan=1>0 - 10</td><td rowspan=2 colspan=1>11/25(44.0%)</td><td rowspan=2 colspan=1>10/28(35.7%)</td><td rowspan=1 colspan=1>21/53</td><td rowspan=1 colspan=1>0/25</td><td rowspan=2 colspan=1>0/29(0.0%)</td><td rowspan=2 colspan=1>0/54(0.0%)</td></tr><tr><td rowspan=1 colspan=1>(39.6%)</td><td rowspan=1 colspan=1>(0.0%)</td></tr><tr><td rowspan=1 colspan=1>11 - 20</td><td rowspan=1 colspan=1>7/13(53.8%)</td><td rowspan=1 colspan=1>5/18(27.8%)</td><td rowspan=1 colspan=1>12/31(38.7%)</td><td rowspan=1 colspan=1>0/13(0.0%)</td><td rowspan=1 colspan=1>0/18(0.0%)</td><td rowspan=1 colspan=1>0/31(0.0%)</td></tr><tr><td rowspan=1 colspan=1>21 - 30</td><td rowspan=1 colspan=1>8/40(20.0%)</td><td rowspan=1 colspan=1>6/26(23.1%)</td><td rowspan=1 colspan=1>14/66(21.2%)</td><td rowspan=1 colspan=1>0/40(0.0%)</td><td rowspan=1 colspan=1>0/26(0.0%)</td><td rowspan=1 colspan=1>0/66(0.0%)</td></tr><tr><td rowspan=1 colspan=1>31 - 40</td><td rowspan=1 colspan=1>5/23(21.7%)</td><td rowspan=1 colspan=1>5/16(31.3%)</td><td rowspan=1 colspan=1>10/39(25.6%)</td><td rowspan=1 colspan=1>1/24(4.2%)</td><td rowspan=1 colspan=1>0/16(0.0%)</td><td rowspan=1 colspan=1>1/40(2.5%)</td></tr><tr><td rowspan=1 colspan=1>41 - 50</td><td rowspan=1 colspan=1>9/19(47.4%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>11/25(44.0%)</td><td rowspan=1 colspan=1>0/19(0.0%)</td><td rowspan=1 colspan=1>0/6(0.0%)</td><td rowspan=1 colspan=1>0/25(0.0%)</td></tr><tr><td rowspan=1 colspan=1>51 - 60</td><td rowspan=1 colspan=1>7/25(28.0%)</td><td rowspan=1 colspan=1>5/9(55.6%)</td><td rowspan=1 colspan=1>12/34(35.3%)</td><td rowspan=1 colspan=1>0/25(0.0%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>0/34(0.0%)</td></tr><tr><td rowspan=1 colspan=1>61 - 70</td><td rowspan=1 colspan=1>4/18(22.2%)</td><td rowspan=1 colspan=1>7/14(50.0%)</td><td rowspan=1 colspan=1>11/32(34.4%)</td><td rowspan=1 colspan=1>1/18(5.6%)</td><td rowspan=1 colspan=1>0/14(0.0%)</td><td rowspan=1 colspan=1>1/32(3.1%)</td></tr><tr><td rowspan=1 colspan=1>71 - 80</td><td rowspan=1 colspan=1>9/13(69.2%)</td><td rowspan=1 colspan=1>7/14(50.0%)</td><td rowspan=1 colspan=1>16/27(59.3%)</td><td rowspan=1 colspan=1>0/13(0.0%)</td><td rowspan=1 colspan=1>1/15(6.7%)</td><td rowspan=1 colspan=1>1/28(3.6%)</td></tr><tr><td rowspan=1 colspan=1>81 - 90</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>2/4(50.0%)</td><td rowspan=1 colspan=1>3/7(42.9%)</td><td rowspan=1 colspan=1>0/3(0.0%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>0/7(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;90</td><td rowspan=1 colspan=1>61/179(34.1%)</td><td rowspan=1 colspan=1>49/135(36.3%)</td><td rowspan=1 colspan=1>110/314(35.0%)</td><td rowspan=1 colspan=1>2/180(1.1%)</td><td rowspan=1 colspan=1>1/137(0.7%)</td><td rowspan=1 colspan=1>3/317(0.9%)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>11/25(44.0%)</td><td rowspan=1 colspan=1>10/28(35.7%)</td><td rowspan=1 colspan=1>21/53(39.6%)</td><td rowspan=1 colspan=1>0/25(0.0%)</td><td rowspan=1 colspan=1>0/29(0.0%)</td><td rowspan=1 colspan=1>0/54(0.0%)</td></tr></table>

Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the Sentosa SA201 HSV-1/2 PCR Test are shown below. These calculations for hypothetical prevalence are based on overall sensitivity and specificity per sample type from the clinical study results. For HSV-1, these calculations are based upon an overall sensitivity and specificity of $9 6 . 9 0 \%$ and $9 5 . 8 2 \%$ , respectively, for anogenital swabs and $1 0 0 . 0 \%$ and $8 6 . 3 8 \%$ , respectively, for oral swabs. For HSV-2, these calculations are based upon an overall sensitivity and specificity of $9 8 . 4 9 \%$ and $9 0 . 7 0 \%$ , respectively, for anogenital swabs and $6 6 . 6 7 \%$ and $9 9 . 6 8 \%$ , respectively, for oral swabs.

Prevalence vs hypothetical Predictive Values   

<table><tr><td rowspan=3 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=4>Anogenital</td><td rowspan=1 colspan=4>Oral</td></tr><tr><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>32.12%</td><td rowspan=1 colspan=1>99.93%</td><td rowspan=1 colspan=1>17.77%</td><td rowspan=1 colspan=1>99.97%</td><td rowspan=1 colspan=1>13.03%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>80.96%</td><td rowspan=1 colspan=1>99.32%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>54.96%</td><td rowspan=1 colspan=1>99.83%</td><td rowspan=1 colspan=1>35.79%</td><td rowspan=1 colspan=1>99.91%</td><td rowspan=1 colspan=1>27.87%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>91.64%</td><td rowspan=1 colspan=1>98.27%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>72.03%</td><td rowspan=1 colspan=1>99.64%</td><td rowspan=1 colspan=1>54.06%</td><td rowspan=1 colspan=1>99.82%</td><td rowspan=1 colspan=1>44.93%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.86%</td><td rowspan=1 colspan=1>96.42%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>85.28%</td><td rowspan=1 colspan=1>99.20%</td><td rowspan=1 colspan=1>72.58%</td><td rowspan=1 colspan=1>99.59%</td><td rowspan=1 colspan=1>64.73%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.12%</td><td rowspan=1 colspan=1>92.29%</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>90.86%</td><td rowspan=1 colspan=1>98.63%</td><td rowspan=1 colspan=1>81.95%</td><td rowspan=1 colspan=1>99.29%</td><td rowspan=1 colspan=1>75.88%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>98.89%</td><td rowspan=1 colspan=1>87.47%</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>93.92%</td><td rowspan=1 colspan=1>97.89%</td><td rowspan=1 colspan=1>87.59%</td><td rowspan=1 colspan=1>98.90%</td><td rowspan=1 colspan=1>83.04%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>99.29%</td><td rowspan=1 colspan=1>81.77%</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>95.86%</td><td rowspan=1 colspan=1>96.87%</td><td rowspan=1 colspan=1>91.37%</td><td rowspan=1 colspan=1>98.36%</td><td rowspan=1 colspan=1>88.01%</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>99.52%</td><td rowspan=1 colspan=1>74.94%</td></tr></table>

# N. Instrument Name:

Sentosa SA201 Thermocycler for PCR amplification Sentosa SX101 Instrument for Nucleic Acid Extrcation

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X_____ or No _

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No ___X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

The Sentosa SA201 HSV-1/2 PCR Test is performed on the Sentosa SX101 and the Sentosa SA201 Real-Time PCR Instruments. The Sentosa SX application “24-1L HSV$I \_ { 2 } ^ { \dag }$ is installed in the Sentosa SX software prior to performing the assay.

The sponsor states: Refer to the Sentosa SA201 HSV-1/2 PCR Test User Manual for detailed operating procedures for the test. For detailed information on software and instrument operations, refer to the Sentosa SX101 User Manual, Sentosa Link User Manual and the Sentosa SA201 Real-Time PCR Instrument Reference Guide for detailed procedures.

3. Specimen Identification:

Patient ID/Sample ID is labeled with a unique barcode, which is tracked by using the Sentosa Link software to prevent re-use and track positive sample identification.

4. Specimen Sampling and Handling: Not Applicable

5. Calibration:

The Sentosa SA201 Real-Time PCR Instrument undergoes a single calibration during manufacturing. No additional calibration is performed by the ende use.

6. Quality Control:

The extraction control (EC) consists of one linearized plasmid with sequence for an unrelated tobacco mosaic virus that is detected by EC primers and probe in the Sentosa SA201 HSV-1/2 PCR Test reagents. The EC is added to the lysis buffer, which is used in the extraction of all samples, NC and PC before PCR setup. Thus EC monitors for sample extraction failure or and/or inhibition, and also instrument/workflow issues, such as inhibition or pipetting errors due to instrument/workflow failure.

Two controls are provided by Vela to the user in the Sentosa SA201 HSV-1/2 PCR Test: a tube of negative control (NC) and a tube of positive control (PC) that undergo the full sample workflow, enabling monitoring of the performance of the assay.

The NC is nucleic acid-free water supplied in a tube that is placed into the Sentosa SX101 sample rack during the workflow run. The NC will monitor instrument or workflow issues, such as reagent and/or environmental contamination, as the presence of HSV-1 and / or HSV-2 DNA will not be detected in the negative control (NC).

The positive control (PC) consists of DNA target sequences for HSV-1 and HSV-2 detection by primers and probes in the Sentosa SA201 HSV-1/2 PCR Test (4x24). It can monitor substantial PCR reagent failure since the PC goes through extraction and PCR set-up, alike any sample in the run. It will also monitor instrument / workflow issues, such as inhibition or pipetting errors due to instrument/workflow failure.

Run Validity Criteria:

Run: Whole run on the Sentosa SA 96-Well Optical Plate Sample: Single sample in one well of Sentosa SA 96-Well Optical Plate

The software automatically determines run validity and sample result. The software will invalidate a run if either or both controls (negative and positive) have invalid results based on the table in Ct value ranges.   
· A run may be invalidated by an operator if technical, operator, or instrument difficulties are observed and documented while performing the assay.   
· An invalid run must be repeated.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

The software documentation was reviewed and found to be acceptable. The sponsor provided documentation to support that the device was designed , developed, and maintained under appropriate software lifecycle processes.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.