# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

Substantial equivalence determination for the T2Bacteria Panel on the T2Dx Instrument

# C. Measurand:

The assay amplifies and detects nucleic acids of the following species:   
Enterococcus faecium   
Escherichia coli   
Klebsiella pneumoniae   
Pseudomonas aeruginosa   
Staphylococcus aureus

# D. Type of Test:

The T2Bacteria Panel, performed on the T2Dx Instrument, is a molecular diagnostic assay for the detection of the above listed bacterial species from whole blood specimens obtained from patients with signs and symptoms of bacterial bloodstream infections.

E. Applicant: T2 Biosystems, Inc.   
F. Proprietary and Established Names: T2Bacteria Panel

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3960

2. Classification: Class II

3. Product code(s): QBX, NSU

4. Panel: 83 - Microbiology

# H. Intended Use:

1. Intended use(s):

The T2Bacteria Panel run on the T2Dx Instrument is a qualitative T2 magnetic resonance (T2MR) test for the direct detection of bacterial species in $\mathrm { K } _ { 2 }$ EDTA human whole blood specimens from patients with suspected bacteremia. The T2Bacteria Panel identifies five species of bacteria: Enterococcus faecium, Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa and Staphylococcus aureus.

The T2Bacteria Panel is indicated as an aid in the diagnosis of bacteremia and results should be used in conjunction with other clinical and laboratory data. Concomitant blood cultures are necessary to recover organisms for susceptibility testing or further identification, and for organisms not detected by the T2Bacteria Panel.

Results from the T2Bacteria Panel are not intended to be used as the sole basis for diagnosis, treatment, or other patient management decisions in patients with suspected bacteremia.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: The T2Bacteria Panel is performed on the T2Dx Instrument

# I. Device Description:

The T2Bacteria panel is a qualitative molecular diagnostic assay that employs a whole blood compatible PCR amplification followed by T2 magnetic resonance (T2MR) detection. The T2Bacteria Panel is performed on the T2Dx Instrument which executes all steps after specimen loading. A $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimen containing a minimum of $3 ~ \mathrm { m L }$ is loaded directly onto the T2Bacteria Sample Inlet, which is then placed on the T2Bacteria Cartridge along with the T2Bacteria Reagent Tray. The Cartridge and Reagent Tray contain the lysis reagent, the internal control, primers, enzyme, buffer and probe-coupled superparamagnetic particles for each bacterial target. The blood specimen is mixed with the red blood cell lysing reagent and the bacterial cells are concentrated by centrifugation. The internal control is added to the concentrated bacterial cells; a bead-beating process then lyses the bacterial cells. The supernatant containing the DNA from the lysed bacterial cells and the internal control are amplified using the target and internal control-specific primers. The generated amplicons are aliquoted into individual tubes containing target-specific conjugated particles for $E$ . faecium, E. coli, K. pneumoniae, $P _ { - }$ . aeruginosa, S. aureus and the internal control. Bacterial DNA is amplified with target-specific primers and amplicons are hybridized to target-specific probes attached to superparamagnetic particles causing clustering of the particles. The hybridization occurring in individual tubes is analyzed in the MR reader and a signal for each target is generated and is detected by T2 Magnetic Resonance (T2MR) indicating the presence of the target organism. Up to seven specimens can be loaded onto the T2Dx instrument at the same time. When running the first specimen in a series or a single specimen, the result is reported in 3.5 hours from the time the specimen is loaded onto the instrument. The results are interpreted by the device software as valid or invalid (based on the result of the internal control or target detections), and if valid, results are reported as “Positive” or “Target not Detected” for each specific target. For $E .$ . coli, results are reported as Positive, Indeterminate or “Target not Detected”. An Indeterminate result is a valid result, but the presence or absence of $E$ . coli in the specimen cannot be definitively assessed.

# J. Substantial Equivalence Information:

1. Predicate device name(s): T2Candida Panel

2. Predicate 510(k) number(s): DEN140019

3. Comparison with predicate:

Table 1. Comparison with the Predicate   

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceK172708T2Bacteria Panel</td><td colspan="1" rowspan="1">PredicateDEN140019T2Candida Panel</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The T2Bacteria Panel run on theT2Dx Instrument is a qualitative T2magnetic resonance (T2MR) test forthe direct detection of bacterialspecies in K2EDTA human wholeblood specimens from patients withsuspected bacteremia. TheT2Bacteria Panel identifies fivespecies of bacteria: Enterococcusfaecium, Escherichia coli, Klebsiellapneumoniae, Pseudomonasaeruginosa and Staphylococcusaureus. The T2Bacteria Panel isindicated as an aid in the diagnosisof bacteremia and results should beused in conjunction with otherclinical and laboratory data.Concomitant blood cultures arenecessary to recover organisms forsusceptibility testing or furtheridentification and for organisms notdetected by the T2Bacteria Panel.Results from the T2Bacteria Panelare not intended to be used as thesole basis for diagnosis, treatment,or other patient managementdecisions in patients with suspectedbacteremia.</td><td colspan="1" rowspan="1">The T2Candida Panel and T2DxInstrument is a qualitative T2Magnetic Resonance (T2MR)assay for the direct detection ofCandida species in EDTA humanwhole blood specimens frompatients with symptoms of, ormedical conditions predisposingthe patient to, invasive fungalinfections. The T2Candida Panelidentifies five species of Candidaand categorizes them into thefollowing three species groups:Candida albicans and/or Candidatropicalis Candida parapsilosisCandida glabrata and/or Candidakrusei. The T2Candida panel doesnot distinguish between C.albicans and C. tropicalis. TheT2Candida panel does notdistinguish between C. glabrataand C. krusei. The T2Candidapanel is indicated for thepresumptive diagnosis ofcandidemia. The T2Candida Panelis performed independent of bloodculture. Concomitant bloodcultures are necessary to recoverorganisms for susceptibility testingor for further identification.The T2Candida positive andnegative external controls areintended to be used as qualitycontrol samples with theT2Candida Panel when run on theT2Dx Instrument system. Thesecontrols are not intended for usewith other assays or systems.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">4 mL whole blood collected in ablood collection tube with EDTAanticoagulant</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Platform</td><td colspan="1" rowspan="1">T2Dx</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Nucleic acid amplification followedby T2 magnetic resonance detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Throughput</td><td colspan="1" rowspan="1">Single cartridge test with randomaccess with seven drawers on T2Dx</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Test CartridgeFormat</td><td rowspan=1 colspan=1>T2Bacteria Test Cartridge anddisposables</td><td rowspan=1 colspan=1>T2Candida Test Cartridge anddisposables</td></tr><tr><td rowspan=1 colspan=1>Reagent Trays</td><td rowspan=1 colspan=1>T2Bacteria Test reagents specific fordetection of bacteria</td><td rowspan=1 colspan=1>T2Candida Test reagents specificfor detection of Candida</td></tr><tr><td rowspan=1 colspan=1>Targets</td><td rowspan=1 colspan=1>T2Bacteria Panel tests for sixdifferent species of bacteriaimplicated in bacteremia:Enterococcus faecium, Escherichiacoli, Klebsiella pneumoniae,Pseudomonas aeruginosa andStaphylococcus aureus</td><td rowspan=1 colspan=1>T2Candida Panel tests for fivedifferent species of Candidacommonly associated withcandidemia: Candida albicansand/or Candida tropicalis,Candida parapsilosis, Candidaglabrata and/or Candida krusei</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

IEC 61010-1:2001, (Second Edition). Safety requirements for electrical equipment for measurement control and laboratory use – General requirements. 2001 IEC61010-2-010:2003 (Second Edition). Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 2-010. Particular requirements for laboratory equipment for the heating of materials. 2003 IEC 61010-2-081:2001 (First Edition) $) + \mathbf { A } 1 { : } 2 0 0 3$ . Safety requirements for electrical equipment for measurement, control and laboratory use – Part 2-081: Particular requirements for automatic and semiautomatic laboratory equipment for analysis and other purposes. 2001 · IEC 61010-2-101:2002 (Second Edition). Safety requirements for electrical equipment for measurement, control, and laboratory use. Particular requirements for in vitro diagnostic (IVD) medical equipment. 2002. IEC 61326-1:2005. Electrical equipment for measurement, control and laboratory use – EMC requirements – Part 1: General requirements. 2005 IEC 61326-2-6:2005. Electrical equipment for measurement, control and laboratory use – EMC requirements – Part 2-6: Particular requirements – In vitro diagnostic (IVD)

medical equipment. 2005   
CLSI EP17-A2. Protocols for Determination of Limits of Detection and Limits of Quantitation; Approved Guideline. 2013.   
CLSI EP07-A2. Interference Testing in Clinical Chemistry. 2007   
CLSI EP05-A3. Evaluation of Precision Performance of Quantitative Measurement Methods. 2014   
CLSI EP25-A. Evaluation of Stability of In Vitro Diagnostic Reagents. 2013.   
CLSI EP12-A2. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition. 2014.   
ISTA 7E. Testing Standard for Thermal Transport Packaging Used in Parcel Delivery System Shipment. 2010

# L. Test Principle:

During processing on the T2Dx instrument, pathogens are concentrated directly in whole blood, then lysed to release the target DNA. Bacterial DNA is amplified with target-specific primers and amplicons are hybridized to target-specific probes attached to superparamagnetic particles causing clustering of the particles. A signal is detected by T2 Magnetic Resonance (T2MR) indicating the presence of the target organism. The Internal Control on the Panel monitors performance for each patient sample or control.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multicenter reproducibility study was performed to determine the run to run, reagent lot, day to day and site to site reproducibility. Testing was performed at three sites (two external and one internal) with a panel of five target species, each tested in triplicate at two concentrations (1-2X LoD and 3-4X LoD) using two reagent lots. Testing was performed for six non-consecutive days with at least two operators per site for a total of 36 replicates per sample per site.

The reproducibility panel was comprised of each target species spiked in fresh human whole blood specimens in double-spiked samples ( $K .$ . pneumoniae and S. aureus) or triple-spiked samples (E. coli, $P .$ . aeruginosa and $E$ . faecium). (The applicability of using multi-spiked samples was evaluated in a separate study, see below.) Bacterial levels were confirmed by colony count testing of the original suspension used for spiking. A total of 108 negative blood samples were included in reproducibility panel. Reproducibility results were acceptable with a range of $9 7 . 2 \%$ to $100 \%$ agreement in detection with expected results for each of the target species. A summary of the reproducibility results across sites, reagents and operators is shown in Table 2 below.

Table 2. Summary of Reproducibility Results Across Sites, Reagents and Operators   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc.</td><td rowspan=1 colspan=1>TestSite</td><td rowspan=1 colspan=1>No.Detected</td><td rowspan=1 colspan=1>No. NotDetected</td><td rowspan=1 colspan=1>AgreementwithExpected</td></tr><tr><td rowspan=8 colspan=1>E. faecium</td><td rowspan=4 colspan=1>1-2X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>107/10899.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=4 colspan=1>3-4X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=8 colspan=1>E. coli</td><td rowspan=4 colspan=1>1-2X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>3-4X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=2 colspan=1>108/108</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=2 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=8 colspan=1>K.pneumoniae</td><td rowspan=4 colspan=1>1-2X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>3-4X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=8 colspan=1>P.aeruginosa</td><td rowspan=4 colspan=1>1-2X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>3-4X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=8 colspan=1>S. aureus</td><td rowspan=4 colspan=1>1-2X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>105/10897.2%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=4 colspan=1>3-4X LoD</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=4 colspan=1>108/108100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc.</td><td rowspan=1 colspan=1>TestSite</td><td rowspan=1 colspan=1>No.Detected</td><td rowspan=1 colspan=1>No. NotDetected</td><td rowspan=1 colspan=1>AgreementwithExpected</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35</td><td rowspan=4 colspan=1>105/10897.2%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>All sites</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>105</td></tr></table>

b. Linearity/assay reportable range:

Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The T2Bacteria Panel utilizes two types of controls, the internal control and the external controls. The Internal Control (IC) is automatically introduced into each specimen during sample processing on the T2Dx instrument and is carried through the lysis, amplification and detection steps of the assay. The IC monitors the amplification and detection process and detects the presence of inhibitors in the specimen.

The external controls include the QCheck Positive Kit and the QCheck Negative Kit. The QCheck Positive Kit contains two sample blends of bacterial species, the SaKp Positive Control (which contains cells of S. aureus and $K .$ . pneumoniae) and the EcPaEfm Positive Control (which contains cells of $E$ . coli, $P .$ . aeruginosa and $E$ . faecium) suspended in a buffer-based solution. These positive controls individually interrogate the respective, complementary probe-coated particles in the detection reactions. Results for the positive controls are reported for only the detection channels for species included in the control. The QCheck Negative Kit contains vials of a buffer-based solution which are used to detect contamination of the T2Bacteria regents or the instrument. Results of the negative control are reported for all five detection channels.

External controls were run each day of the analytical and clinical studies. Summaries of the quality control results obtained at all sites during the clinical study are provided in Tables 3 to 5 below.

There were a total of 4394 negative control results obtained in the clinical study at the testing sites; 4375 of these were valid and 19 were invalid. Using the original cut-off for $E .$ . coli, $9 9 . 2 \%$ (4340/4375) gave negative results. There were a total of 35 false positive results $( 0 . 8 \% )$ including 25 false positive results for $E$ . coli. To address the occurrence of false positive results obtained for $E$ . coli, the original $E$ . coli cut-off for the QCheck Negative Control was modified (increased); cut-off values for other target analytes were unchanged. Using the modified cut-off for $E .$ . coli, $9 9 . 4 \%$ (4348/4375) gave acceptable negative results. With the revised cut-off, there was a total of 27 false positive results with the negative controls, including 17 false positive results for $E$ . coli. Results for the negative controls with both the original and revised cut-offs are shown in Table 3 below.

Table 3. Results Obtained with the QCheck Negative Controla,b   

<table><tr><td rowspan=3 colspan=1>Site</td><td rowspan=3 colspan=1>No.validTests</td><td rowspan=1 colspan=3>E. coli original cut-off</td><td rowspan=1 colspan=3>E. coli revised cut-off</td></tr><tr><td rowspan=2 colspan=1>No.Correct</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>No. FPc</td><td rowspan=1 colspan=1>No.</td><td rowspan=2 colspan=1>%Correct</td><td rowspan=2 colspan=1>No. FPd(%)</td></tr><tr><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>Correct</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>625</td><td rowspan=1 colspan=1>620</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>5 (0.8)</td><td rowspan=1 colspan=1>621</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>4 (0.6)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>2 (1.3)</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>1 (0.7)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>2 (0.6)</td><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>2 (0.6)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>427</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>3 (0.7)</td><td rowspan=1 colspan=1>428</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>2 (0.5)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>726</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>9 (1.2)</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>6 (0.8)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>2 (1.1)</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>1 (0.5)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>350</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>3 (0.9)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>245</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>3 (1.2)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>3(1.2)</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>3 (1.2)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>329</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>404</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>404</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>1 (0.3)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>1(0.3)</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>1 (0.3)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4375</td><td rowspan=1 colspan=1>4340</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>35 (0.8%)</td><td rowspan=1 colspan=1>4348</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>27 (0.6%)</td></tr></table>

a Abbreviations: FP, False positive b Includes results for all detection channels from prospective specimen and reproducibility testing during the clinical study c False positives included: $2 5 E .$ . coli results, $8 P$ . aeruginosa results and $2 K .$ . pneumoniae results. d False positives included: $1 7 E .$ . coli results, $8 P$ . aeruginosa results and $2 K .$ . pneumoniae results.

There were a total of 846 results obtained with the SaKp Positive control during the course of the clinical study at the testing sites; 836 of these were valid and 10 were invalid. A total of 834 $( 9 9 . 3 \% )$ of the results provided the expected detection. Two false negative results (one each for $K .$ . pneumoniae and S. aureus) were observed. Results obtained with the SaKp control are shown in Table 4 below.

Table 4. Results obtained with the QCheck SaKp Positive Control a,b   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>No. valid Tests</td><td rowspan=1 colspan=1>No. Correct</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>No. FN (%)c</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>2 (1.4)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>836</td><td rowspan=1 colspan=1>834</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>2 (0.2)</td></tr></table>

a Abbreviations: FN, False negative b Includes results for all detection channels from prospective specimen and reproducibility testing during the clinical study c False negatives included $1 K .$ . pneumoniae result, $1 \ S .$ . aureus result

There were a total of 1357 results obtained with the EcPaEfm Positive control run during the course of the clinical study at the testing sites; 1347 of these were valid and 10 were invalid. A total of 1338 $( 9 9 . 3 \% )$ of the results provided the expected detections. Nine false negative results (five for $E$ . coli and four for $P .$ . aeruginosa) were observed. Results obtained with the EcPaEfm control are shown in Table 5 below.

Table 5. Results obtained with the QCheck EcPaEfm Positive Control a,b   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>No. validTests</td><td rowspan=1 colspan=1>No. Correct</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>No. FN (%)c</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>3 (2.3)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>1 (1.6)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>1 (1.3)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>2 (2.0)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>1 (0.8)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>1 (1.0)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1347</td><td rowspan=1 colspan=1>1338</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>9 (0.7%)</td></tr></table>

a Abbreviations: FN, False negative b Includes results for all detection channels from prospective specimen and reproducibility testing during the clinical study c False negatives include $5 E .$ . coli results and $4 P$ . aeruginosa results

# d. Detection limit:

LoD testing was performed using two strains of each target bacterial species inoculated into human whole blood. The determination of the LoD included an initial screening phase and a confirmatory phase. In the initial phase, double- or triplespiked samples were prepared at four concentrations (1, 3, 9 and $1 0 \mathrm { C F U / m L }$ ) and a preliminary LoD was identified. (The applicability of using multi-spiked samples was evaluated in a separate study; see below.) To confirm the preliminary LoD, a minimum of 20 replicates of both strains of each species was tested at increasing concentrations until a positivity rate of $9 5 \%$ was achieved. The preliminary LoD for each species was confirmed using two reagent lots. For two species, $K .$ . pneumoniae and $P .$ . aeruginosa, the preliminary LoD was not confirmed with both strains of each species tested; the higher concentration that provided $\ge 9 5 \%$ positivity for both strains was confirmed as the LoD. For $E .$ . coli, the preliminary LoD of $1 0 \mathrm { C F U / m L }$ did not provide $\ge 9 5 \%$ positivity during the confirmatory testing. Confirmatory testing was repeated and the LoD was confirmed at 11 CFU/mL. Results of the confirmatory LoD testing are shown in Table 6 below.

Table 6. Confirmatory LoD Testing for Target Species   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=2>Strain 1</td><td rowspan=1 colspan=2>Strain 2</td><td rowspan=2 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>ReagentLot 1No. Tested/No Correct(%)</td><td rowspan=1 colspan=1>ReagentLot 2No. Tested/No Correct (%)</td><td rowspan=1 colspan=1>ReagentLot 1No. Tested/No Correct(%)</td><td rowspan=1 colspan=1>ReagentLot 2No. Tested/No Correct(%)</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>19/20 (95)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>19/20 (95)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>19/20 (95)a</td><td rowspan=1 colspan=1>20/20 (100)a</td><td rowspan=1 colspan=1>20/20 (100)b</td><td rowspan=1 colspan=1>20/20 (100)b</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>20/20 (100)c</td><td rowspan=1 colspan=1>20/20 (100)c</td><td rowspan=1 colspan=1>20/20 (100)d</td><td rowspan=1 colspan=1>20/20 (100)d</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>2</td></tr></table>

a Tested at 2 CFU/mL b Tested at 1 CFU/mL c Tested at 3 CFU/mL d Tested at 5 CFU/mL

# e. Single species spike, multi-species spike equivalence

A comparison of the equivalence of results obtained for human blood samples spiked with a single bacterial species vs. samples spiked with three bacterial species was performed to determine the applicability of testing multiple species in a single blood specimen for the analytical studies.

A single strain each of the five target bacterial species was tested at concentrations of 1-2 X LoD and 3-4 X LoD as controls. Four replicates of each species at each concentration level were evaluated. Multi-spiked samples included the combinations of $K .$ pneumoniae/S. aureus and E. faecium/E. coli/

P. aeruginosa; each species in each combination was tested at two concentrations, 1-2 X LoD and 3-4 X LoD.

Samples containing two or three bacterial species gave positive results for each species; signals obtained from the single-spiked samples were equivalent to the signals obtained from the multi-spiked samples.

# f. Analytical specificity:

Analytical exclusivity testing of the T2Bacteria Panel was conducted to assess the cross reactivity of the panel with non-panel species tested at $1 0 ^ { 3 } \mathrm { C F U / m L }$ for fungi and bacteria, and $1 0 ^ { 3 } \dot { \mathrm { T C I D } } _ { 5 0 }$ for viral pathogens. Bacterial species that were shown to be potentially cross-reactive at the initial concentration were then tested in triplicate at more physiologically relevant concentrations of 100, 33 and $1 0 \mathrm { C F U / m L }$ . Isolates for testing were selected based on ${ \geq } 9 6$ primer homology and $2 8 4 \%$ probe homology with species identified in a BLAST search. Species that showed no crossreactivity with T2Bacteria Panel target organisms when tested at a concentration of $1 0 ^ { 3 } \mathrm { C F U / m L }$ or $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 }$ are shown in Table 7 below. Species that showed cross reactivity with T2Bacteria Panel members when tested at concentrations of $\geq 1 0$ $\mathrm { C F U / m L }$ are shown in Table 8.

By in silico analysis, Yersinia pestis was determined to be non-cross reactive (Table 7); Klebsiella quasipneumoniae and S. argenteus were determined to be cross reactive with $K .$ . pneumoniae and $S .$ aureus, respectively (Table 8).

Table 7. Organisms with No Cross-Reactivity with T2Bacteria Panel Targets   

<table><tr><td rowspan=1 colspan=6>Fungi</td></tr><tr><td rowspan=1 colspan=1>Aspergillusfumigatus</td><td rowspan=1 colspan=1>Aspergillus niger</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Candidaparapsilosis</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Cryptococcusaalbidus</td><td rowspan=1 colspan=1>Cryptococcusneoformans</td><td rowspan=1 colspan=1>Fusariummoniliforme</td><td rowspan=1 colspan=1>Fusariumoxysporum</td><td rowspan=1 colspan=1>Rhizomucormeihei</td></tr><tr><td rowspan=1 colspan=2>Rhizopus microsporus</td><td rowspan=1 colspan=2>Rhizopus oryzae</td><td rowspan=1 colspan=2>Rhodotorula glutinis</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Viruses</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Adenovirustype 1</td><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Epstein-Barr Virus</td><td rowspan=1 colspan=1>Hepatitis A Virus</td><td rowspan=1 colspan=2>Hepatitis B Virus</td></tr><tr><td rowspan=1 colspan=2>Herpes Simplex Virus 1</td><td rowspan=1 colspan=2>Herpes Simplex Virus 2</td><td rowspan=1 colspan=2>Varicella Zoster Virus</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Gram Positive Bacteria</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Actinomycesisraelii</td><td rowspan=1 colspan=1>Clostridiumsphenoides</td><td rowspan=1 colspan=1>Enterococcusavium</td><td rowspan=1 colspan=1>Enterococcuscaccae</td><td rowspan=1 colspan=1>Enterococcuscasseliflavus</td><td rowspan=1 colspan=1>Enterococcuscecorum</td></tr><tr><td rowspan=1 colspan=1>Enterococcusdispar</td><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Enterococcusgallinarum</td><td rowspan=1 colspan=1>Enterococcusgilvus</td><td rowspan=1 colspan=1>Enterococcushirae</td><td rowspan=1 colspan=1>Enterococcusitalicus</td></tr><tr><td rowspan=1 colspan=1>Enterococcusmalodoratu</td><td rowspan=1 colspan=1>Enterococcusmundtii</td><td rowspan=1 colspan=1>Enterococcuspallens</td><td rowspan=1 colspan=1>Enterococcuspseudoavium</td><td rowspan=1 colspan=1>Enterococcusraffinosus</td><td rowspan=1 colspan=1>Finegoldiamagna</td></tr><tr><td rowspan=1 colspan=1>Lactobacillusacidophilus</td><td rowspan=1 colspan=1>Lactococcuslactis</td><td rowspan=1 colspan=1>Leuconostocmesenteroides</td><td rowspan=1 colspan=1>Listeriamonocytogenes</td><td rowspan=1 colspan=1>Parvimonasmicra</td><td rowspan=1 colspan=1>Pediococcuspentosaceus</td></tr><tr><td rowspan=1 colspan=1>Peptoniphilusharei</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>Propionibacteriumacnes</td><td rowspan=1 colspan=1>Staphylococcusauricularis</td><td rowspan=1 colspan=1>Staphylococcuscapitis</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>Staphylococcushominis</td><td rowspan=1 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>Staphylococcussaprophyticus</td><td rowspan=1 colspan=1>Staphylococcuswarneri</td><td rowspan=1 colspan=1>Staphylococcusxylosus</td></tr><tr><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>Streptococcusanginosus</td><td rowspan=1 colspan=1>Streptococcusbovis</td><td rowspan=1 colspan=1>Streptococcusconstellatus</td><td rowspan=1 colspan=1>Streptococcusdysgalactiae</td><td rowspan=1 colspan=1>Streptococcusmutans</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=2>Streptococcus salivarius</td><td rowspan=1 colspan=2>Streptococcusoralis</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Gram Negative Bacteria</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus</td><td rowspan=1 colspan=1>Acinetobacterlwoffi</td><td rowspan=1 colspan=1>Acinetobacternosocomialis</td><td rowspan=1 colspan=1>Acinetobacterpittii</td><td rowspan=1 colspan=1>Acinetobacterradioresistans</td><td rowspan=1 colspan=1>Aeromonashydrophila</td></tr><tr><td rowspan=1 colspan=1>Bacteroidesfragilis</td><td rowspan=1 colspan=1>Burkholderiacepacia</td><td rowspan=1 colspan=1>Chryseobacteriumindologenes</td><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>Cupriaviduspauculus</td></tr><tr><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>Fusobacteriumnecrophorum</td><td rowspan=1 colspan=1>Fusobacteriumnucleatum</td><td rowspan=1 colspan=1>Klebsiellaoxytoca</td></tr><tr><td rowspan=1 colspan=1>Leptotrichiatrevisanii</td><td rowspan=1 colspan=1>Moraxellacatarrhalis</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Myroides odoratus</td><td rowspan=1 colspan=1>Ochrobactrumanthropi</td><td rowspan=1 colspan=1>Oligellaurethralis</td></tr><tr><td rowspan=1 colspan=1>Pantoeaagglomerans</td><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>Providenciastuartii</td><td rowspan=1 colspan=1>Pseudomonasalcaligenes</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasfluorescens</td><td rowspan=1 colspan=1>Pseudomonasluteola</td><td rowspan=1 colspan=1>Pseudomonasoryzihabitans</td><td rowspan=1 colspan=2>Pseudomonas pseudoalcaligenes(oleovorans)</td><td rowspan=1 colspan=1>Pseudomonasputida</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasstutzeri</td><td rowspan=1 colspan=1>Ralstoniapicketti</td><td rowspan=1 colspan=1>Raoultellaornithinolytica</td><td rowspan=1 colspan=1>Raoultellaplanticola</td><td rowspan=1 colspan=2>Salmonella enterica Enteritidis</td></tr><tr><td rowspan=1 colspan=1>SalmonellaentericaTyphimurium</td><td rowspan=1 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>Shewanellaputrefaciens</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=2>Weeksella virosa</td></tr><tr><td rowspan=1 colspan=2>Yersinia pseudotuberculosis</td><td rowspan=1 colspan=2>Yersinia pestisa</td><td rowspan=1 colspan=2></td></tr></table>

a Determined to be non-cross reactive by in silico analysis

Table 8. Organisms with Cross Reactivity with T2Bacteria Targets   

<table><tr><td rowspan=1 colspan=1>T2Bacteria PanelTarget</td><td rowspan=1 colspan=1>Cross Reactive SpeciesPredicted by Wet Testing</td><td rowspan=1 colspan=1>Cross Reactive Species Predictedby In Silico Analysis</td></tr><tr><td rowspan=1 colspan=1>E faecium</td><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=7 colspan=1>NA</td></tr><tr><td rowspan=6 colspan=1>E. coli</td><td rowspan=1 colspan=1>Escherichia albertii</td></tr><tr><td rowspan=1 colspan=1>Escherichia fergusonii</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniaea</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>S. argenteusa</td></tr></table>

a Determined to be cross-reactive by in silico analysis

For one species, Enterovirus Type 68, 0 of 3 replicates tested at $1 , 0 0 0 \mathrm { \ T C I D } _ { 5 0 } / \mathrm { m L }$ were positive for $E .$ . coli; 1 of 6 replicates tested at 316 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ were positive on the $E .$ coli channel of the Panel. This species was deemed not cross reactive.

# g. Assay cut-off:

To establish the limit of blank (LoB) and cut-off values for the bacterial target and internal controls detections, negative blood specimens from healthy donors and blood specimens from patients suspected of bacteremia were tested using multiple instruments and reagent lots across multiple days. T2MR signals were evaluated to establish the upper limit of the signal distribution for negative samples; the T2MR signal values that encompassed $\geq 9 9 \%$ of all signals from negative samples was determined to be the LoB.

The assay cut-off values were determined on a per channel basis using spiked positive and negative samples; the lower limit of the signal distribution for each target channel and for the internal control measurements determined the assay cut-off. Established cut-offs were reevaluated utilizing data obtained with prospectively-collected and contrived samples evaluated during the clinical study to validate the cut-off with a larger specimen pool; analysis verified the cut-off for all targets except $E$ . coli. The cut-off for detection of $E$ . coli was redefined using the larger data set and all results were reanalyzed with the new cut-off. The cut-off values were established to produce $\geq 9 5 . 6 \%$ specificity and $\geq 9 0 . 0 \%$ sensitivity.

Because of the occurrence of false positive results with $E$ . coli in the clinical study, the cut-off for $E .$ . coli was reassessed to include an indeterminate range for which the presence or absence of $E$ . coli cannot be definitively assessed.

# h. Analytical Sensitivity

A total of 57 clinical isolates were evaluated for inclusivity in the T2Bacteria Panel. Isolates were selected to represent variations in antimicrobial resistance, and phylogenic, temporal and geographic diversity. Isolates were inoculated into human whole blood at a concentration of 2-3X LoD. Testing of each isolate was performed in triplicate. In the event of a false negative result, testing was repeated with 20 replicates. Results of inclusivity testing combined with results from contrived specimen testing in the clinical study (for which 50 unique isolates of each target species were tested) indicate that the T2Bacteria Panel can detect a variety of strains of target organisms with $5 9 5 \%$ accuracy. Results of the inclusivity study are shown in Table 9 below.

Table 9. Results of Inclusivity Testing   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>No. Strains Tested</td><td rowspan=1 colspan=1>No. Positive/Total (%)</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>33/33 (100)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>K.pneumoniae</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>77/79 (97.5)</td></tr><tr><td rowspan=1 colspan=1>P.aeruginosa</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>39/39 (100)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>42/44 (95.5)</td></tr></table>

# i. Co-infection Studies

A competitive inhibition study was performed to evaluate the ability of the T2Bacteria Panel to detect target bacterial species present at 1-2X LoD in the presence of other clinically relevant organisms (on- and off- panel) that may be present in a co-infection. Three combinations of organisms were tested in human whole blood samples: 1) samples containing two target bacterial species, each in concentrations of 1-2X LoD (low concentrations); 2) samples containing one target bacterial species at a concentration of 1-2X LoD (low concentration) and a second target bacterial species at a concentration of 1000 CFU/mL (high concentration); and 3) samples containing one target bacterial species at a concentration of 1-2 X LoD (low concentration) and a non-target bacterial species or yeast species at a concentration of $1 0 0 0 \mathrm { C F U / m L }$ (high concentration). Four replicates of each combination were initially tested; any combination resulting in a non-detection was repeated with 20 replicates. If ${ \leq } 9 5 \%$ of the target species were detected in the 20 replicates, the concentration of the competing organisms was decreased to determine the level at which the detection was not inhibited.

On Panel Combinations. All target organisms were detected with four replicates at all concentrations except for the following combinations: $E .$ . coli (low concentration)/ $P .$ . aeruginosa (high concentration), $P .$ aeruginosa (low concentration)/ $E .$ coli (low concentration) and $P .$ . aeruginosa (low concentration)/S. aureus (low concentration). Repeat testing of 20 replicates of these combinations resulted in $1 0 0 \%$ detection. For the combination of $P .$ . aeruginosa (low concentration) combined with $E .$ . coli (high concentration), $P .$ . aeruginosa was initially detected in $2 / 4$ replicates; repeat testing

showed detection of 18/20 replicates. This combination was further evaluated with decreasing concentrations of $E .$ coli (100 CFU, 33 CFU and 10 CFU); both $P .$ . aeruginosa and $E .$ . coli were detected in all replicates with these concentrations. A limitation was included in the labeling indicating that high concentrations of $E .$ . coli in a blood specimen may prevent the detection of low concentrations of $P .$ aeruginosa when present in the same specimen.

Non-Panel Combinations. Combinations of non-panel and panel species were tested as noted above. Non-panel member species were selected based on clinical relevance and included Staphylococcus epidermidis, Streptococcus pneumoniae, Enterobacter cloacae, Streptococcus salivarius and Candida albicans. Non-panel species were tested at a concentration of $1 0 0 0 \mathrm { C F U / m L }$ . Initial testing with four replicates showed $100 \%$ detection for all targets except $P .$ . aeruginosa with S. pneumoniae for which $P$ . aeruginosa was detected in three of the four replicates. Additional testing with 20 replicates of this combination resulted in $100 \%$ detection.

# j. Interfering Substances

An interfering substances study was performed to determine and characterize the effects of potential endogenous and exogenous interfering substances commonly found in blood on the performance of the T2Bacteria Panel. Interferents were screened as pools of interferents in human whole blood samples; potential interferents were added in high concentrations to represent worst case scenarios; samples were tested in triplicate. Samples were inoculated with T2Bacteria target species at a concentration of 2-3 X LoD and samples without interferents were include as controls. Interferents that caused a difference in signal from a sample without interferents were further analyzed individually in blood samples.

Evaluation of potential endogenous substances included 13 interferents that represented underlying disease conditions (Table 10). None of the endogenous substances tested showed interference with the T2Bacteria assay.

Table 10. Endogenous Substances (Representing Human Underlying Conditions) Evaluated with the T2Bacteria Panel   

<table><tr><td colspan="1" rowspan="1">Underlying Sourceor Condition</td><td colspan="1" rowspan="1">EndogenousInterferent</td><td colspan="1" rowspan="1">ReferenceLevel</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">InterferenceNoted</td></tr><tr><td colspan="1" rowspan="1">Leukocytosis</td><td colspan="1" rowspan="1">Human DNA (BuffyCoat)</td><td colspan="1" rowspan="1">WBC 4.5-11 X106 cells/mL</td><td colspan="1" rowspan="1">2.08 X 10' to 2.48 X107 cells/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="2">Icterus</td><td colspan="1" rowspan="1">Bilirubin(conjugated)</td><td colspan="1" rowspan="1">0-3.4 μmol/L</td><td colspan="1" rowspan="1">342 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Bilirubin(unconjugated)</td><td colspan="1" rowspan="1">5-21 μmol/L</td><td colspan="1" rowspan="1">342 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="2">Hepatitis</td><td colspan="1" rowspan="1">ALT</td><td colspan="1" rowspan="1">5-40 U/L</td><td colspan="1" rowspan="1">120 U/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">AST</td><td colspan="1" rowspan="1">9-48 U/L</td><td colspan="1" rowspan="1">144 U/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Hemolysis</td><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">11-17.4 g/dL</td><td colspan="1" rowspan="1">&gt;20 g/dL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Lipemia</td><td colspan="1" rowspan="1">Intralipid (to mimictriglycerides)</td><td colspan="1" rowspan="1">0.34-3.7mmol/L</td><td colspan="1" rowspan="1">3270 mg/dl</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="2">Hyperproteinemia</td><td colspan="1" rowspan="1">Protein (albumin)</td><td colspan="1" rowspan="1">39-51 g/L</td><td colspan="1" rowspan="1">60 g/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Immunoglobulin G</td><td colspan="1" rowspan="1">6-13 g/L</td><td colspan="1" rowspan="1">60 g/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="2">Renal Failure</td><td colspan="1" rowspan="1">Creatinine</td><td colspan="1" rowspan="1">6-13 mg/L</td><td colspan="1" rowspan="1">50 mg/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Urea</td><td colspan="1" rowspan="1">1.1-14.3mmol/L</td><td colspan="1" rowspan="1">42.9 mmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Multiple</td><td colspan="1" rowspan="1">Circulating humanDNA</td><td colspan="1" rowspan="1">3.1-730.5ng/mL(plasma)</td><td colspan="1" rowspan="1">2.2 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">SystemicInflammatoryResponse</td><td colspan="1" rowspan="1">Lactoferrin (87kDa)</td><td colspan="1" rowspan="1">0.005-2.5 μM</td><td colspan="1" rowspan="1">7.5 μmol/L</td><td colspan="1" rowspan="1">No</td></tr></table>

A total of 28 exogenous potentially interfering substances were tested at high concentrations (Table 11). Only Feraheme was shown to interfere with the T2Bacteria assay at the test concentration of $6 1 8 ~ \mu \mathrm { g / m L }$ . Further testing was performed with lower concentrations of Feraheme; concentrations of Feraheme $\geq 2 1 \ \mu \mathrm { g / m L }$ were shown to interfere with the performance of the T2Bacteria Panel. The interference of Feraheme with the T2Bacteria Panel was noted as a limitation in the device labeling.

Table 11. Exogenous Substances Evaluated with the T2Bacteria Panel   

<table><tr><td colspan="1" rowspan="1">Exogenous Interferent</td><td colspan="1" rowspan="1">Reference Level</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">1.8 mg/mL</td><td colspan="1" rowspan="1">5.4 mg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Feraheme</td><td colspan="1" rowspan="1">206 μg/mL</td><td colspan="1" rowspan="1">618 μg/mL</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Heparin</td><td colspan="1" rowspan="1">350-1000 U/L</td><td colspan="1" rowspan="1">3000 U/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Iopamidol</td><td colspan="1" rowspan="1">10-225 mL cumulativeinjections perindividual</td><td colspan="1" rowspan="1">180 μL per 4 mLVacutainer</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">MRI Contrast Agent:gadopentetate dimeglumine</td><td colspan="1" rowspan="1">0.01 mmol/kg</td><td colspan="1" rowspan="1">1.5mM</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.51 μg/mL</td><td colspan="1" rowspan="1">1.53 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Lisinopril</td><td colspan="1" rowspan="1">0.25 μmol/L</td><td colspan="1" rowspan="1">0.74 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cytarabine</td><td colspan="1" rowspan="1">10.8 μg/mL</td><td colspan="1" rowspan="1">32.4 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Amphotericin B Trihydrate</td><td colspan="1" rowspan="1">80 μg/mL</td><td colspan="1" rowspan="1">240 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Caspofungin</td><td colspan="1" rowspan="1">8.0 - 17.6 μg/mL</td><td colspan="1" rowspan="1">52.8 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Fluconazole</td><td colspan="1" rowspan="1">65.2-81.5 μmol/L</td><td colspan="1" rowspan="1">245 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">35-234 μmol/L</td><td colspan="1" rowspan="1">701μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Micafungin</td><td colspan="1" rowspan="1">30 μg/mL</td><td colspan="1" rowspan="1">90 mg/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">7.59-50.6 μmol/L</td><td colspan="1" rowspan="1">152μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Azithromycin</td><td colspan="1" rowspan="1">0.4-5.1 μmol/L</td><td colspan="1" rowspan="1">15.3μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cefazolin, sodium salt</td><td colspan="1" rowspan="1">37.4-881 µmol/L</td><td colspan="1" rowspan="1">2.643mmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cefepime Hydrochloride</td><td colspan="1" rowspan="1">164 μg/mL</td><td colspan="1" rowspan="1">492 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cefoxitin Sodium Salt</td><td colspan="1" rowspan="1">60 μg/mL</td><td colspan="1" rowspan="1">180 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ceftazidime Pentahydrate</td><td colspan="1" rowspan="1">46.9 mg/L 162.3 mg/L</td><td colspan="1" rowspan="1">487 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">1.51-15.1 µmol/L</td><td colspan="1" rowspan="1">30.2 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Clindamycin</td><td colspan="1" rowspan="1">4.0-29.7μmol/L</td><td colspan="1" rowspan="1">89.1 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Gentamycin sulfate</td><td colspan="1" rowspan="1">10.5-20.9 μmol/L</td><td colspan="1" rowspan="1">21 μmol/L</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Imipenem/Cilastatin(Primaxin)</td><td colspan="1" rowspan="1">Imipenem 21-83μg/mLCilastin 31-88 μg/mL</td><td colspan="1" rowspan="1">528 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Linezolid</td><td colspan="1" rowspan="1">18.6 μg/mL</td><td colspan="1" rowspan="1">55.8 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Meropenem trihydrate</td><td colspan="1" rowspan="1">55-62 μg/mL</td><td colspan="1" rowspan="1">186 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Piperacillin</td><td colspan="1" rowspan="1">39 μg/mL</td><td colspan="1" rowspan="1">117 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Tazobactam</td><td colspan="1" rowspan="1">6.3 μg/mL</td><td colspan="1" rowspan="1">18.9 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">12.4-27.6 µmol/L</td><td colspan="1" rowspan="1">103 μg/mL</td><td colspan="1" rowspan="1">No</td></tr></table>

# k. Carryover/Cross Contamination Studies

A carryover/cross contamination study was performed to evaluate potential assay contamination caused by operator or carryover in the $\mathrm { T } 2 \mathrm { D } \mathbf { x }$ instrument. Whole blood specimens were spiked at concentrations of $1 0 0 \mathrm { C F U / m L }$ (61 samples) and 1000 $\mathrm { C F U / m L }$ (56 samples) and were loaded on the $\mathrm { T } 2 \ \mathrm { D x }$ instrument alternately with negative whole blood specimens (123 negative specimens).

Of the 123 negative specimens, 3 $( 2 . 4 \% )$ gave false positive results; two with $E$ . coli $( 1 . 6 \% )$ and one with $K .$ . pneumoniae $( 0 . 8 \% )$ . The occurrence of false positive results was evaluated in follow-up studies related to reagent contamination (see below).

# l. Specimen Stability

The stability of $\mathrm { K } _ { 2 } \mathrm { E D T A }$ blood specimens when stored at room temperature $( 1 5 { - } 3 0 ^ { \circ } \mathrm { C } )$ , refrigerator temperature $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ and frozen $( { \leq } 6 5 ^ { \circ } \mathrm { C } )$ using samples spiked at a concentration of $2 0 \mathrm { C F U / m L }$ and tested over a range of days was evaluated. In addition, studies were performed to evaluate the effect of storage at room temperature followed by refrigeration. Results indicated that storage at all evaluated temperatures and tested within the recommended times provided expected results for target and internal control detections. The acceptable storage times and temperatures included in the device labeling are shown in Table 12.

Table 12. Recommended Storage Times and Temperatures for Blood Specimens Prior to Testing with the T2Bacteria Panel   

<table><tr><td rowspan=1 colspan=1>Temperature</td><td rowspan=1 colspan=1>Storage Time from collection</td></tr><tr><td rowspan=1 colspan=1>Room Temperature (15 - 25° C)</td><td rowspan=1 colspan=1>Up to 12 hours</td></tr><tr><td rowspan=1 colspan=1>Refrigeration Temperature (2 - 8° C)</td><td rowspan=1 colspan=1>Up to 72 hours</td></tr><tr><td rowspan=1 colspan=1>Room Temperature (15 - 25° C),then Refrigeration (2 - 8° C)</td><td rowspan=1 colspan=1>Room Temperature up to 12 hoursRefrigeration up to 72 hours</td></tr><tr><td rowspan=1 colspan=1>Freezer Temperature ≤65° C</td><td rowspan=1 colspan=1>Up to 3 months</td></tr></table>

To determine the equivalence of fresh vs. frozen specimens, human whole blood samples were spiked with 20 replicates of two strains of each target species at various concentrations around the LoD concentration and tested with two lots of reagents (a total of 980 tests). A total of 70 tests were also performed for a single target organism at 5X LoD concentrations. All samples were tested with the T2Bacteria Panel at the time of preparation (fresh) and after freezing for at least 24 hours. Results from both preparations showed $100 \%$ detection with both specimen types at the established LoD. In addition, a portion of the contrived samples (minimum of 15 for each target species) and eight prospectively collected T2 blood specimens with positive concurrently-collected blood cultures were tested fresh and frozen. Results were equivalent for all samples tested.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

Because of the expected low prevalence of positive blood cultures for the target bacterial species, evaluation of the sensitivity of the T2Bacteria Panel required the use of contrived samples. Contrived sample testing was conducted at three clinical sites. A total of 300 organism-spiked contrived samples and 50 negative samples were prepared using human whole blood specimens collected in $\mathrm { K } _ { 2 } \mathrm { E D T A }$ tubes from healthy adults and were verified to be negative for the T2Bacteria target species. Each contrived sample was prepared using a single unique bacterial strain; a total of 50 unique strains for each target species were tested at specific concentrations for each target organism. Samples were spiked at four concentration levels, ${ < } 1 \mathrm { C F U / m L }$ , 1-10 CFU/mL, 11-30 CFU/mL and 31-100 CFU/mL as shown in Table 13 below. The concentrations were determined to approximate clinically relevant concentrations of organisms in a blood specimen. Because of the higher LoD for $E$ . coli as compared to other target species, only 22 samples were prepared for $E$ . coli at or above the LoD. All samples were tested in a blinded fashion and negative samples were tested randomly throughout the study.

Table 13. Organism Concentrations in Contrived Samples   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>LoDCFU/mL</td><td rowspan=1 colspan=5>Number of Contrived Samples</td></tr><tr><td rowspan=1 colspan=1>&lt;1CFU/mL</td><td rowspan=1 colspan=1>1-10CFU/mL</td><td rowspan=1 colspan=1>11-30CFU/mL</td><td rowspan=1 colspan=1>31-100CFU/mL</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>350</td></tr></table>

Testing all target species with the T2Bacteria Panel at concentrations greater than LoD showed PPA $5 9 0 \%$ for all species (Range $9 0 . 0 - 1 0 0 \%$ ); results for $E$ . coli were determined using the final modified cut-off (Table 14).

Of the 250 results obtained for the negative samples included in the contrived study, 248 were negative for all targets. Two results were false positive for $E .$ . coli.

Table 14. T2 Results Obtained for Contrived Samples   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>LoD</td><td rowspan=1 colspan=2>&lt;LoD</td><td rowspan=1 colspan=2>≥ LoD</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>60.0% (6/10)</td><td rowspan=1 colspan=1>31.3 - 83.2</td><td rowspan=1 colspan=1>100.0% (40/40)</td><td rowspan=1 colspan=1>91.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>67.8% (19/20)</td><td rowspan=1 colspan=1>49.3  82.1</td><td rowspan=1 colspan=1>90.9 (20/22)</td><td rowspan=1 colspan=1>72.2 - 97.5a</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>50.0% (5/10)</td><td rowspan=1 colspan=1>23.7 - 76.3</td><td rowspan=1 colspan=1>100.0% (40/40)</td><td rowspan=1 colspan=1>91.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>63.6% (7/11)</td><td rowspan=1 colspan=1>35.4  84.8</td><td rowspan=1 colspan=1>97.4 (38/39)</td><td rowspan=1 colspan=1>86.8 - 99.5</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2% (2/11)</td><td rowspan=1 colspan=1>5.1 - 47.7</td><td rowspan=1 colspan=1>92.3 (36/39)</td><td rowspan=1 colspan=1>79.7 - 97.3</td></tr></table>

a For samples spiked with $E$ . coli at concentrations $> 1$ CFU/mL, PPA is 90.0% (36/40) (95% CI, 77.0 – 96.0); one sample gave an indeterminate result (1/40).

# b. Clinical specificity:

The specificity of the T2Bacteria Panel was determined by a prospective study comparing the results of the T2Bacteria Panel with results from a standard of care blood culture collected from the same draw at the same anatomical site. The comparison study was performed at 11 geographically diverse clinical sites in the U.S. A total of 2430 subjects were enrolled, specimens from 1003 subjects were excluded, with the majority excluded due to blood not collected, blood specimens frozen and not tested (to decrease the number of frozen specimens vs. fresh specimens), insufficient blood volume for testing and bloods for which the storage limits were exceeded. A total of 1427 prospectively-collected specimens were tested and included in the comparative study. For each participant, standard of care blood cultures were drawn followed by collection of three $\mathrm { K } _ { 2 } \mathrm { E D T A }$ blood tubes (Tubes A,

B and C) for testing with the T2Bacteria Panel. Tube A was tested with the T2Bacteria Panel and the remaining tubes were frozen for future analysis. Three blood culture systems were used in the study; a total of 39 blood cultures gave positive results for the T2Bacteria targeted organisms. Organism identification was performed on all positive blood cultures using standard of care methods. Of the 1427 specimens 672 were fresh specimens and 755 were frozen. Results obtained with the T2Bacteria Panel showed a specificity range of 95.0 to $9 9 . 4 \%$ depending on the target (Table 15).

Table 15. T2Bacteria Results Obtained in the Prospective Study as Compared to Blood Culture   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>100.0% (1/1)</td><td rowspan=1 colspan=1>20.7 -100</td><td rowspan=1 colspan=1>99.4% (1417/1426)</td><td rowspan=1 colspan=1>98.8 - 99.7</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>90.9% (10/11)a</td><td rowspan=1 colspan=1>62.3 - 98.4</td><td rowspan=1 colspan=1>95.0% (1345/1416)b</td><td rowspan=1 colspan=1>93.7 - 96.0</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>100.0% (6/6)</td><td rowspan=1 colspan=1>61.0 - 100.0</td><td rowspan=1 colspan=1>98.5% (1399/1421)</td><td rowspan=1 colspan=1>97.7 - 99.0</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>56.6 - 100.0</td><td rowspan=1 colspan=1>97.7 (1389/1422)</td><td rowspan=1 colspan=1>96.8 - 98.3</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>81.3% (13/16)c</td><td rowspan=1 colspan=1>57.0 - 93.4</td><td rowspan=1 colspan=1>98.0% (1383/1411)</td><td rowspan=1 colspan=1>97.1 - 98.6</td></tr></table>

a $E$ . coli detected with retest using Tube B (residual specimen, drawn at the same time as the original blood specimen). b Eight additional specimens gave indeterminate results $^ \mathrm { c } S .$ . aureus detected with retest using Tube B (residual specimen, drawn at the same time as the original blood specimen.

In the clinical study, there were no prospectively collected specimens that contained two or more target species; the ability of the T2Bacteria Panel to detect more than one species in a single blood specimen was not evaluated. A limitation was added to the device labeling.

Although the sensitivity (PPA) and specificity (NPA) of the T2Bacteria Panel was acceptable, false positive results were obtained at multiple clinical sites during the prospective arm of the clinical study. These occurred with multiple lots of cartridges and reagents and from both fresh and frozen specimens. The majority of false positive results were obtained for the $E .$ . coli and $P .$ . aeruginosa targets.

A review of other culture results from patients for whom a false positive T2Bacteria result was obtained was performed. Various data that may provide insight on the validity of the false positive results was reviewed and additional testing was conducted. Positive blood cultures for the same target species obtained within $\pm 1 4$ days of the T2Bacteria test were considered as strong evidence of bacteremia; positive cultures from other sources were considered less predictive of bacteremia (Table 16)

To further evaluate the validity of the false positive results, alternative amplification and sequencing was performed on a frozen, residual blood tube (Tube B of the original specimen draw) collected from each patient for whom a false positive T2Bacteria result was obtained and for whom no other culture from any source

(collected within $1 4 \pm$ days of the T2 positive specimen) was positive with the same detected organism. Samples that were positive by amplification and sequencing for the same species detected by the T2Bacteria Panel were also considered strong evidence of bacteremia (Table 16).

Analysis of the additional data indicated that $4 4 . 5 \%$ of the false positives were associated with strong evidence of bacteremia, $1 4 . 8 \%$ were weakly associated with bacteremia and $4 0 . 6 \%$ were not associated with bacteremia.

Table 16. Analysis of T2 Positive/Blood Culture Negative Specimensa   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>T2 Pos/BC NegTotal</td><td rowspan=1 colspan=1>OtherBCPositiveb</td><td rowspan=1 colspan=1>Amplificationand SequencePositivec</td><td rowspan=1 colspan=1>No.Associatedwith StrongEvidence of(%)</td><td rowspan=1 colspan=1>No.Associatedwith OtherEvidenceofInfection(%)</td><td rowspan=1 colspan=1>No.Associatedwith NoEvidenceofInfection(%)</td></tr><tr><td rowspan=1 colspan=1>Efm</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4/9(44.4)</td><td rowspan=1 colspan=1>3/9(33.3)</td><td rowspan=1 colspan=1>2/9(22.2)</td></tr><tr><td rowspan=1 colspan=1>Eci</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>21/63(33.3)</td><td rowspan=1 colspan=1>8/63(12.7)</td><td rowspan=1 colspan=1>34/63(54.0)</td></tr><tr><td rowspan=1 colspan=1>Kp</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>14/22(63.6)</td><td rowspan=1 colspan=1>3/22(13.6)</td><td rowspan=1 colspan=1>5/22(22.7)</td></tr><tr><td rowspan=1 colspan=1>Pa</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11/33(33.3)</td><td rowspan=1 colspan=1>4/33(12.1)</td><td rowspan=1 colspan=1>18/33(54.5)</td></tr><tr><td rowspan=1 colspan=1>Sa</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>19/28(67.9)</td><td rowspan=1 colspan=1>5/28(17.9)</td><td rowspan=1 colspan=1>4/28(14.3)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>69/155(44.5)</td><td rowspan=1 colspan=1>23/155(14.8)</td><td rowspan=1 colspan=1>63/155(40.7)</td></tr></table>

a Abbreviations: Efm, $E .$ . faecium; Eci, $E$ . coli; Kp, K. pneumoniae; Pa, $P$ . aeruginosa; Sa, S. aureus; BC, blood culture b Blood cultures positive for the T2 species detected other than the paired blood culture and collected within $\pm \nobreakspace 1 4 \nobreakspace$ days of the T2 specimen c Amplification and sequencing performed on blood samples drawn at the same time as collection of the T2 specimen (Tube B) and positive for the species detected by the panel. d Strong evidence, see text. e Other evidence, see text.

Analysis of the results obtained during the analytical studies and with negative controls performed during the analytical and clinical studies also showed a higher than expected number of false positive results (especially for $E$ . coli) that mirrored the results obtained with clinical specimens (Table 3 and Table 17).

Table 17. False Positive Results Obtained in the Analytical Studies   

<table><tr><td rowspan=2 colspan=1>Analytical Study</td><td rowspan=1 colspan=5>Percent False Positives by Targeta</td></tr><tr><td rowspan=1 colspan=1>Efm</td><td rowspan=1 colspan=1>Eci</td><td rowspan=1 colspan=1>Kp</td><td rowspan=1 colspan=1>Pa</td><td rowspan=1 colspan=1>Sa</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Single vs. Multi Spike</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Interfering Substances</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Analytical Reactivity</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Competitive Inhibition</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Analytical Specificity</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Reproducibility</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.1</td></tr></table>

a Abbreviations: Efm, E. faecium; Eci, E. coli; Kp, $K .$ . pneumoniae; Pa, $P$ . aeruginosa; Sa, S. aureus

An investigation into the root cause identified reagent contamination as the likely source for the false positive results from patients with no evidence of infection. T2 Biosystems implemented improved reagent testing methods and release criteria with the overall goal of reducing the rate of false positive results that can be attributed to reagents contamination. To evaluate the effect of the improved testing methods, the false positive rate obtained with testing negative controls during the clinical study (875 negative control runs) was compared to the false positive rate obtained with an additional 286 negative control runs. Results showed that after these improvements the false positive rate potentially caused by reagent contamination was $\leq 1 . 1 \%$ for all targets (Table 18).

Table l8. Positivity of T2Bacteria Panel in QCheck Negative Samples Before and After Reagent Improvements   

<table><tr><td rowspan="2">Improvements</td><td colspan="5">No. Positive/No. Tests (%)a</td></tr><tr><td>Efm</td><td>Eci</td><td>Kp</td><td>Pa</td><td>Sa</td></tr><tr><td>Beforeb</td><td>0/875 (0.0)</td><td>17/875 (1.9)</td><td>2/875 (0.2)</td><td>8/875 (0.9)</td><td>0/875 (0.0)</td></tr><tr><td>Afterc</td><td>0/286 (0.0)</td><td>3/286 (1.1)</td><td>0/286 (0.0)</td><td>1/286 (0.3)</td><td>0/286 (0.0)</td></tr></table>

a Abbreviations: Efm, $E .$ . faecium; Eci, E. coli; Kp, K. pneumoniae; Pa, P. aeruginosa; Sa, S. aureus b Negative controls run during the clinical study c Negative controls run post study

Additional testing of archived samples (Tubes B or C) for 43 samples that were originally positive for $E .$ . coli (30 samples) or $P$ . aeruginosa (13 samples) and not associated with other evidence of infections gave negative results for all samples. Two of the archived samples were positive for $P .$ . aeruginosa from patients who were originally T2 positive for $P .$ . aeruginosa, suggesting that these 2 samples were most likely true positive results (with negative blood cultures) and the remaining samples were most likely false positive results for $P .$ . aeruginosa.

After reagent improvements, additional testing was completed on 286 QCheck negative controls and $1 2 0 \mathrm { K } _ { 2 } \mathrm { E D T A }$ blood samples obtained from healthy donors. False positives were observed only for $E .$ . coli $( 1 . 7 \% )$ and $P$ . aeruginosa $( 1 . 7 \% )$ in the blood samples and only for $E$ . coli $( 1 . 0 \% )$ and $P$ . aeruginosa $( 0 . 3 \% )$ in the QCheck negative controls.

To address the possibility of false positive results with $E .$ . coli, an indeterminate zone was instituted for interpretation of results for this species only. The device labeling indicates that an indeterminate result (while valid) cannot be considered positive or negative, and no antimicrobial therapy decisions should be based on this result.

The rate of invalid results and instrument failures that occurred during the clinical study were $0 . 4 \%$ (8/1991 tests) and $3 . 1 \%$ (61/1991 tests), respectively.

c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the prospective clinical study a total of 1427 paired blood cultures and T2Bacteria specimens were evaluated. There were a total of 36 positive blood cultures and 199 positive T2Bacteria results. Positive T2Bacteria results were evaluated as noted above.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Parts 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.