# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM

A. 510(k) Number:

K173798

# B. Purpose for Submission:

Clearance of New Device. The same devices was cleared under K150962 for genital lesion samples. In this submission the sponsor is obtaining clearance for the same device with cutaneous and mucocutaneous lesion samples.

# C. Measurand:

Target DNA Sequences from conserved regions of Herpes Simplex Virus Type 1 (HSV- 1) and Herpes Simplex Virus Type 2 (HSV- 2)

# D. Type of Test:

Real-Time PCR Assay

# E. Applicant:

DiaSorin Molecular LLC.

# F. Proprietary and Established Names:

Simplexa HSV 1 & 2 Direct MOL2150 and Simplexa HSV 1 & 2 Positive Control Pack MOL2160

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3309

2. Classification: Class II

3. Product code: PGI

4. Panel: Microbiology

# H. Intended Use:

1. Intended use(s):

Simplexa HSV 1 & 2 Direct

The DiaSorin Molecular Simplexa HSV 1 & 2 Direct assay is intended for use on the LIAISON MDX instrument for the qualitative detection and differentiation of herpes simplex virus (HSV-1 and HSV-2) DNA present in mucocutaneous and cutaneous lesion swabs from patients with signs and symptoms of HSV-1 or HSV-2 infection. This test is an aid in the differential diagnosis of HSV-1 and HSV-2 infections.

The assay is not intended for use as a screening test for the presence of HSV-1 and HSV2 in blood or blood products. The assay is for professional use only.

Simplexa HSV 1 & 2 Positive Control Pack

The Simplexa HSV 1 & 2 Positive Control Pack is intended to be used as a control with the Simplexa HSV 1 & 2 Direct kit.

This control is not intended for use with other assays or systems.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: LIAISON MDX (with LIAISON MDX Studio Software)

# I. Device Description:

The Simplexa HSV 1 & 2 Direct assay system is a real-time PCR that enables the direct amplification, detection and differentiation of HSV-1 and/or HSV-2 DNA from unprocessed cutaneous and mucocutaneous lesion swab specimens without nucleic acid extraction. The system consists of the Simplexa HSV 1 & 2 Direct assay, the LIAISON MDX (with LIAISON MDX Studio Software), the Direct Amplification Disc and associated accessories.

In the Simplexa HSV 1 & 2 Direct assay, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify HSV-1, HSV-2 and internal control targets. Well conserved regions of the HSV-1 and HSV-2 DNA polymerase genes are targeted to identify HSV-1 and HSV-2 DNA respectively in the specimen. An internal control is used to detect PCR failure and/or inhibition.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Lyra Direct HSV $1 + 2 / \mathrm { V } Z \mathrm { V }$ Assay

2. Predicate 510(k) number(s): K133448

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=5>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=3>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=12 colspan=1>Indications For Use</td><td rowspan=1 colspan=3>The DiaSorin MolecularSimplexa HSV 1 &amp; 2 Directassay is intended for use onthe LIAISON MDXinstrument for thequalitative detection and</td><td rowspan=12 colspan=1>The Lyra Direct HSV 1 +2/VZV Assay is an in vitromultiplex Real-Time PCRtest for qualitativedetection anddifferentiation of herpessimplex virus type 1,herpes simplex virus type2, and varicella-zoster virusDNA isolated and purifiedfrom cutaneous ormucocutaneous lesionsamples obtained fromsymptomatic patientssuspected of active herpessimplex virus 1, herpessimplex virus 2 and/orvaricella-zoster infection.The Lyra Direct HSV 1 +2/VZV Assay is intended toaid in the diagnosis ofherpes simplex virus 1,herpes simplex virus 2 andvaricella-zoster virus activecutaneous ormucocutaneous infections.</td></tr><tr><td rowspan=1 colspan=3>differentiation of herpes</td><td rowspan=1 colspan=2>differentiation of herpessimplex virus (HSV-1 and</td></tr><tr><td rowspan=2 colspan=3>HSV-2) DNA present inmucocutaneous and</td></tr><tr><td rowspan=2 colspan=3>mucocutaneous andcutaneous lesion swabs</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3>from patients with signs and</td></tr><tr><td rowspan=1 colspan=3>symptoms of HSV-1 or</td></tr><tr><td rowspan=1 colspan=3>HSV-2 infection. This test is</td></tr><tr><td rowspan=1 colspan=3>an aid in the differential</td></tr><tr><td rowspan=1 colspan=2>diagnosis of HSV-1 and</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=3>diagnosis of HSV-1 andHSV-2 infections.</td></tr><tr><td rowspan=1 colspan=1>herpes simplex virus 2 and</td></tr><tr><td rowspan=1 colspan=1>Extraction Technology</td><td rowspan=1 colspan=3>None</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Amplification Technology</td><td rowspan=1 colspan=3>PCR-based system for</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>detecting the presence orabsence of viral DNA inclinical specimens.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Detection Technology</td><td rowspan=1 colspan=1>Multiplex assay usingdifferent reporter dyes foreach target.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Samples Type</td><td rowspan=1 colspan=1>Cutaneous andMucocutaneous LesionSamples</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Assay Targets</td><td rowspan=1 colspan=1>Well conserved region ofthe polymerase genes forHSV-1 and HSV-2.</td><td rowspan=1 colspan=1>HSV-1: glycoprotein G,HSV-2: glycoprotein G,VZV: ORF6: DNA-helicase primase.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff: Administrative Procedures for CLIA Categorization, May 12, 2014

Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k), July 07, 2015

Off-The-Shelf Software Use in Medical Devices, September 9, 1999

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005

General Principles of Software Validation, January 11, 2002

Cybersecurity for Networked Medical Devices Containing Off-The-Shelf (OTS) Software, January 14, 2005

# L. Test Principle:

The Simplexa HSV 1 & 2 Direct assay system is a real-time PCR that enables the direct amplification, detection and differentiation of HSV-1 and/or HSV-2 DNA from unprocessed cutaneous and mucocutaneous lesion swab specimens without nucleic acid extraction. The system consists of the Simplexa HSV 1 & 2 Direct assay, the LIAISON MDX (with LIAISON MDX Studio Software), the Direct Amplification Disc and associated accessories.

In the Simplexa HSV 1 & 2 Direct assay, bi-functional fluorescent probe-primers are used together with corresponding reverse primers to amplify HSV-1, HSV-2 and internal control targets. Well conserved regions of the HSV-1 and HSV-2 DNA polymerase genes are targeted to identify HSV-1 and HSV-2 DNA respectively in the specimen. An internal control is used to detect PCR failure and/or inhibition. The principle of the assay and procedural steps are described below:

# Specimen Collection

Acceptable specimen type is swab samples collected from herpetic cutaneous and mucocutaneous lesions and stored in BD UVT, Remel M4, Remel M4RT, Remel M5, Remel M6 or UTM transport media. Follow the manufacturer’s package inserts for collection media and acceptable swab types. Do not use calcium alginate swabs, as they may contain substances that inhibit PCR testing. Specimens should be transported on ice and stored at 2 to $8 ~ ^ { \circ } \mathrm { C }$ for up to 7 days post collection. If there is a greater than 7 days delay before processing of the specimen, store specimen at $- 7 0 ~ ^ { \circ } \mathrm { C }$ .

# Real-time PCR Instrument Setup

Refer to the LIAISON MDX Operator Manual for details on how to configure the LIAISON MDX Studio Software to add an assay definition, set up and analyze runs on the LIAISON MDX.

# Direct Amplification Disc Loading and Real-Time PCR Amplification

1. Select samples that need to be tested.   
2. Thaw Reaction Mix vials at room temperature (approximate range 18 to $2 5 ~ ^ { \circ } \mathrm { C }$ ). Thaw one Reaction Mix vial for each sample or control to be tested.   
3. Scan the barcode on the Simplexa HSV 1 & 2 Direct Reaction Mix vial or barcode card.   
4. Scan the disc barcode on the Direct Amplification Disc (DAD).   
5. Scan or type in each sample identifier.   
6. For one wedge at a time, peel the adhesive foil back to expose the Reaction (R) and Sample (SAMPLE) wells without completely removing the adhesive foil cover (Figure 1 & 2). Avoid touching the underside of the foil that will be in contact with the wells and disc surface.   
7. Ensure that the Reaction Mix is completely thawed. Briefly spin down the tubes as needed. (Do not vortex the Reaction Mix).   
8. Use the fixed volume pipette to transfer $5 0 ~ \mu \mathrm { L }$ of the Reaction Mix into Reaction (R) well.   
9. Use the fixed volume pipette to transfer $5 0 ~ \mu \mathrm { L }$ of sample or control; pipette sample or control into Sample well (SAMPLE).   
10. Cover the wedge sealing the wells with the peeled adhesive foil, pressing down firmly near the edge of the wedge. If the original foil is torn do not load the wells in the wedge. Instead load another wedge.   
11. Tear off the tab portion of the foil cover along the perforation.   
12. Repeat steps 6 to 11 for the next sample(s).   
13. Load the sealed DAD into the LIAISON MDX and start the run.

![](images/6544a695b0c815a6cd37a776181259ebf01ac7a09919cc80d587b4d655f5e0c8.jpg)

![](images/c8b662ff0aa47fad05814dab5733fcb1ad1f5553794a631546f600acfddb429a.jpg)  
Figure 2 - Sample [SAMPLE] and Reaction [R] Wells

# Quality Control

SimplexaTM HSV 1 & 2 Positive Control Pack (MOL2160) may be used as an external control for QC testing, training or proficiency testing. Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations and standard good laboratory practice. Refer to the SimplexaTM HSV 1 & 2 Positive Control (IFUC.US.MOL2160) for instructions on testing the positive control.

# Expected Control Results

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>DNA Internal Control(DNA IC)</td></tr><tr><td rowspan=1 colspan=1>Simplexa HSV 1 &amp; 2Positive Control</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not applicable2</td></tr><tr><td rowspan=1 colspan=1>No Template Control(NTC)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Valid</td></tr></table>

1 Typical Ct values for the Positive Control range between 25 to ${ \le } 4 0$ . 2 Detection of the Simplexa DNA Internal Control (DNA IC) is not required for a valid result when HSV is detected.

# Interpretation of Results

pon completion of the run, the software automatically calculates and displays results.

For each accession ID (Sample ID) entered, the software displays a result (“Detected”, “Not Detected”, “Invalid” or $^ { \cdot \varsigma } \mathrm { E C } 5 0 0 ^ { \circ }$ , EC505 and EC515) for HSV-1 and HSV-2.

a. “Detected” result points to the presence of HSV-1 and/or HSV-2 DNA in the patient sample.   
b. “Not Detected” result points to the absence of HSV-1 and/or HSV-2 DNA in the patient sample.   
c. “Invalid” result points to the inability to determine presence or absence of HSV-1 and/or HSV-2 DNA in the patient sample. This result may be due to 1) DNA Internal Control (DNA IC) failure, or 2) failure to detect sufficient specimen. The sample needs to be re-tested. See “Invalid Results” section below.   
d. “EC500” Data processing error due to noise, weak or late amplification in the signal. Repeat the sample. If the problem persists, contact Technical Service.   
e. “EC505” Insufficient information to determine whether amplification was present. If the problem persists, contact Technical Service.   
f. “EC515” Internal control amplification is not within specification. Result is invalid, repeat the sample. If the problem persists, contact Technical Service.

# Invalid Results

In case of an “Invalid” result, re-test the sample with a new Reaction Mix vial from the same kit or a new kit. If the problem is unresolved, contact DiaSorin Molecular Technical Services department.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility studies were not conducted again for the purposes of this submission and the results from original submission K150962 are shown below.

Reproducibility for the Simplexa HSV 1 & 2 Direct assay was evaluated. Three investigative sites assessed the device's inter-site, inter-day and inter/intra-assay reproducibility. Each of the laboratories tested the positive control and a panel of five contrived sample pools including a low (approximately 1-2 times LoD) and medium positive (approximately 2-4 times LoD) for each analyte and a high negative. The high negative sample contained a small amount of HSV-1 and HSV-2, and it was designed to be negative approximately $9 5 \%$ of the time. The assays were performed in triplicate on 5 different days. Each site had 2 operators; each operator assayed the entire sample panel and positive control once per day, for a total of 2 sets of data per day.

Reproducibility Results for the Simplexa HSV1&2 Direct Assay from submission K150962   
a. Expected Results of HSV-2 Low Positive, HSV-2 Medium Positive and High Negative samples are “Negative” for HSV-1.   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementWithExpectedResults</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=7 colspan=1>HSV-1Result</td><td rowspan=1 colspan=1>HSV-1 LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-1MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30/30)a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.7%(29/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>98.9%(89/0) a</td><td rowspan=1 colspan=1>94.0 to99.8%</td></tr><tr><td rowspan=1 colspan=1>HSV-2MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.7%(29/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0%(30/30) a</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>98.9%(89//90)a</td><td rowspan=1 colspan=1>94.0 to99.8%</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>96.7%(29/30)a</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>93.3%(28/30) a</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>90.0%(27/30) a</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>93.3%(84/90) a</td><td rowspan=1 colspan=1>86.2 to96.9%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>99.4% (179/180)</td><td rowspan=1 colspan=3>98.3% (177/180)</td><td rowspan=1 colspan=3>97.8% (175/179)</td><td rowspan=1 colspan=1>98.5%(531/539)</td><td rowspan=1 colspan=1>97.1 to99.2%</td></tr></table>

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">Sample</td><td colspan="3" rowspan="1">Site - 1</td><td colspan="3" rowspan="1">Site - 2</td><td colspan="3" rowspan="1">Site - 3</td><td colspan="1" rowspan="2">Total %AgreementWithExpectedResults</td><td colspan="1" rowspan="2">95%CI</td></tr><tr><td colspan="1" rowspan="1">%AgreementWithExpectedesults</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">%AgreementWithExpectedResults</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">%AgreementWithExpectedesults</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">Total%CV</td></tr><tr><td colspan="1" rowspan="5">HSV-2Result</td><td colspan="1" rowspan="1">HSV-1 LowPositive</td><td colspan="1" rowspan="1">100.0%(0//30))</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(0/30) </td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">96.7%(29/30)   b</td><td colspan="1" rowspan="1">41.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">98.9%(89 8/90) b</td><td colspan="1" rowspan="1">94.0 to99.8%</td></tr><tr><td colspan="1" rowspan="1">HSV-1MediumPositive</td><td colspan="1" rowspan="1">100.0%(30/30b</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30b</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/300) b</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%( 90/90)b</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-2 LowPositive</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">90.0%(27/30)</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1">93.3%(28/30)</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">94.4%(85/90)</td><td colspan="1" rowspan="1">87.6 to97.6%</td></tr><tr><td colspan="1" rowspan="1">HSV-2MediumPositive</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">96.7%(29/30) b</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">86.7%(26/30) b</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">100.0%(0 /30)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">94.4%(8 (85/90)</td><td colspan="1" rowspan="1">87.6 to97.6%</td></tr><tr><td colspan="1" rowspan="2">b</td><td colspan="1" rowspan="1">PositiveControl</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">30.2</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">30.1</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">100.0%(29/29)</td><td colspan="1" rowspan="1">29.9</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">100.0%(89/89)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">TotalAgreement</td><td colspan="3" rowspan="1">99.4% (179/180)</td><td colspan="3" rowspan="1">96.1% (173/180)</td><td colspan="3" rowspan="1">98.9% (176/179)</td><td colspan="1" rowspan="1">98.0%(528/539)</td><td colspan="1" rowspan="1">96.4 to98.9%</td></tr></table>

b Expected Results of HSV-1 Low Positive, HSV-1 Medium Positive and High Negative samples are “Negative” for HSV-2.

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>Total %AgreementWithExpectedResults</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>%AgreementWithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=7 colspan=1>DNAICResult</td><td rowspan=1 colspan=1>HSV-1LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.%%</td></tr><tr><td rowspan=1 colspan=1>HSV-1MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2LowPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2MediumPositive</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>100.0% (180/180)</td><td rowspan=1 colspan=3>100.0% (180/180)</td><td rowspan=1 colspan=3>100.0% (179/179)</td><td rowspan=1 colspan=1>100.0%(539/539)</td><td rowspan=1 colspan=1>96.4 to98.9%</td></tr></table>

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): No changes were made from the clearance of K150962.   
d. Assay cut-off: No changes were made from the clearance of K150962.

e. Detection limit: No changes were made from the clearance of K150962.

f. Analytical reactivity:

No changes were made from the clearance of K150962.

g. Analytical specificity:

# Cross-Reactivity (Analytical Specificity)

The Simplexa HSV 1 & 2 Direct assay’s analytical specificity was evaluated by testing the ability to exclusively identify HSV-1 and HSV-2 viruses with no crossreactivity to organisms that are closely related, or cause similar clinical symptoms or may be present on cutaneous or mucocutaneous swabs. A total of 71 potential crossreactants were spiked into negative cutaneous and mucocutaneous swab matrix and assayed in triplicate. No cross-reactivity was observed.

<table><tr><td colspan="1" rowspan="2">No.</td><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">Tested Concentration</td><td colspan="2" rowspan="1">Qualitative Result(#Detected/#Total)</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Baseline</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">35/35</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Bacteroides ureolyticus**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1.00 x 106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">1.00 x 106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1.00 x 106 IFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 x 106 IFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Coronavirus (HCoV OC43)</td><td colspan="1" rowspan="1">1.00 x 103 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Coxsackievirus B (CVB-1)</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.00 x 10° CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.00 x 10 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Epstein Barr Virus (B95-8)</td><td colspan="1" rowspan="1">1.00 x 105 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Escherichia coli O157H7</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Haemophilus ducreyi**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Haemophilus influenzae (Type A)</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Hepatitis B</td><td colspan="1" rowspan="1">1.00 x 105 IU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Hepatitis C</td><td colspan="1" rowspan="1">1.00 x 10 IU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">HHV-6 (Z29 Strain)</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">HHV-7 SB</td><td colspan="1" rowspan="1">1.00 x 103 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">HIV-1 IIIB</td><td colspan="1" rowspan="1">1.00 x 105 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">HIV-2 NIHZ*</td><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">HPV18 Recombinant</td><td colspan="1" rowspan="1">1.00 x 105 PFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">1.00 x 103 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Mycoplasma genitalium**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1.00 x 106 CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Mycoplasma orale**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.00 x 106 CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">Mycoplasma salivarium**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">Respiratory syncytial virus A</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">Respiratory syncytial virus B</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">Rubella</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">Salmonella enteritidis**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA),ATCC 700699</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE), ATCC 29887</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">Streptococcus pyogenes, M1</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1.00 x 10tachyzooites/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">Treponema pallidum**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">1.00 x 106trophozoites/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1.00 x 106 CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">VZV</td><td colspan="1" rowspan="1">1.00 x 105 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr></table>

\* Quantified material was not available to test; instead the vendor provided a culture fluid with a known Ct value. The site was directed to dilute the stock to a relevant $\mathrm { C t }$ value; 1:50 dilution factor. \*\*Microorganism was not available for testing therefore in silico NCBI BLAST analysis was performed and found no cross reactivity. $\mathrm { N } / \mathrm { A } = \mathrm { N o t }$ applicable

# Interference

The performance of the Simplexa HSV 1 & 2 Direct assay was evaluated with potentially interfering substances that may be present on cutaneous and mucocutaneous swabs at the concentrations indicated in the table below. A total of 24 potentially interfering substances were tested in a low positive HSV-1 and HSV-2 sample (4 times LoD) in negative cutaneous and mucocutaneous swab matrix containing male and female cutaneous and mucocutaneous swabs and assayed in triplicate. No interference was observed.

<table><tr><td rowspan=2 colspan=1>Potential Interferent</td><td rowspan=2 colspan=1>InterferentConcentration</td><td rowspan=1 colspan=2># Detected/# Total</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>10 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Buffy coat</td><td rowspan=1 colspan=1>7% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Carmex Original Lip Balm (Camphor, 1.7%;Menthol, 0.7%)</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>10 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Chlorpheniramine maleate</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Cold-EEZE Cold Remedy plus Throat (ZincumGluconicum 2X)</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Cornstarch</td><td rowspan=1 colspan=1>1.25 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Desitin (Zinc Oxide, 40%)</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Dextromethorphan hydrobromide</td><td rowspan=1 colspan=1>10 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Foscarnet</td><td rowspan=1 colspan=1>1.25 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Ganciclovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Lanacane Benzethonium chloride, 0.2%;Benzocaine, 20%)</td><td rowspan=1 colspan=1>7% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Lip Clear Lysine (Zinc Oxide, 1.2%)</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Listerine (Eucalyptol, 0.092%; Menthol, 0.042%;Methyl salicylate, 0.060%; Thymol, 0.064%)</td><td rowspan=1 colspan=1>7% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Miconazole 3 (Miconazole nitrate, 2%)</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Seminal fluid</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Spermicide</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Tioconazole</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Toothpaste</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Valganciclovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>KY Jelly</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# Competitive Interference

Competitive interference was studied to evaluate the effects of clinically relevant co-infections with each of the analytes detected by the Simplexa HSV 1 & 2 Direct assay. The study assessed whether a high concentration of one virus in the sample could potentially affect the Simplexa HSV 1 & 2 Direct assay performance for another target present at low levels. A low sample was contrived at approximately 4 times LoD for each target (HSV-1 McIntryre strain and HSV-2 G strain), and a baseline Ct was determined for each sample. Each potential concomitant infecting virus was spiked into the low level sample and assayed in triplicate. Baseline sample results are also shown below. No competitive interference was observed.

<table><tr><td rowspan=1 colspan=2>Baseline (Low Level)</td><td rowspan=1 colspan=2>Competitive Interferent(High Concentration)</td><td rowspan=1 colspan=2>Qualitative Results(#Detected/#Total)</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>1.00 x 106</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=2></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>1.71 x 103</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# Inhibition By Other Microorganisms

The Simplexa HSV 1 & 2 Direct assay was evaluated by testing the ability to identify HSV-1 and HSV-2 viruses when other potentially inhibitory organisms are present. The panel of 71 potentially inhibitory organisms was individually spiked into a pool with a low concentration (approximately 4 times LoD) of HSV-1 and HSV-2 in cutaneous and mucocutaneous swab matrix. Each microorganism sample was initially tested in triplicate and if any one of the replicates was “Not Detected” for either the HSV-1 or the HSV-2 targets then five additional replicates would be tested to confirm if any inhibition was caused by the microorganism. If the majority of replicates $( > 4 / 8 )$ were “Not Detected” then an inhibitory effect would be determined. None of the microorganisms caused ${ > } 4 / 8$ of the replicates to be “Not Detected”. In silico NCBI BLAST analysis was performed for 7 potential inhibitory organisms.

<table><tr><td colspan="1" rowspan="2">No.</td><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">Tested Concentration</td><td colspan="2" rowspan="1">Qualitative Result(#Detected/#Total)</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Baseline</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">35/35</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Bacteroides ureolyticus**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1.00 x 10° IFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 X 106 IFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Coronavirus (HCoV OC43)</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Coxsackievirus B (CVB-1)</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.00 X 103 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Epstein Barr Virus (B95-8)</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Escherichia coli O157H7</td><td colspan="1" rowspan="1">1.00 X 10° CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.00 X 10° CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Haemophilus ducreyi**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Haemophilus influenzae (Type A)</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Hepatitis B</td><td colspan="1" rowspan="1">1.00 X 103 IU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Hepatitis C</td><td colspan="1" rowspan="1">1.00 X 103 IU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">HHV-6 (Z29 Strain)</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">HHV-7 SB</td><td colspan="1" rowspan="1">1.00 X 103 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">HIV-1 IIIB</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">HIV-2 NIHZ*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">HPV18 Recombinant</td><td colspan="1" rowspan="1">1.00 X 105 PFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Mycoplasma genitalium**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1.00 X 106 CCU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Mycoplasma orale**</td><td colspan="1" rowspan="1">Not Applicable</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.00 x 106 CCU/mL</td><td colspan="1" rowspan="1">4/8</td><td colspan="1" rowspan="1">5/8</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">Mycoplasma salivarium**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">Respiratory syncytial virus A</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">Respiratory syncytial virus B</td><td colspan="1" rowspan="1">1.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">Rubella</td><td colspan="1" rowspan="1">1.00 X 105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">Salmonella enteritidis**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA), ATCC700699</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">Staphylococcus epidermidis (MRSE),ATCC 29887</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">Streptococcus pyogenes, M1</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1.00 X 106tachyzooites/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">Treponema pallidum**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">1.00 X 106trophozoites/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1.00 X 106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">VZV</td><td colspan="1" rowspan="1">1.00 X 105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

\* Quantified material was not available to test; instead the vendor provided a culture fluid with a known Ct value. The site was directed to dilute the stock to a relevant $\mathrm { C t }$ value; 1:50 dilution factor. \*\*Microorganism was not available for testing therefore in silico NCBI BLAST analysis was performed and found no inhibition. $\mathrm { N } / \mathrm { A } = \mathrm { N o t }$ applicable.

h. Sample stability studies: No changes were made from the clearance of K150962.   
i. Stability studies:

No changes were made from the clearance of K150962.

j. Carryover contamination: No changes were made from the clearance of K150962.   
k. Fresh vs Frozen: No changes were made from the clearance of K150962.

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable

b. Clinical specificity: Not Applicable

c. Other clinical supportive data (when a. and b. are not applicable):

Clinical Agreement

# Prospective Study 1 – Cutaneous and Mucocutaneous Swab Sample Type K150962

A total of 718 cutaneous and mucocutaneous lesion swab samples were prospectively collected May 28, 2014 through December 4, 2014 from patients with signs and symptoms of herpes simplex virus (HSV) infection from 6 geographically diverse locations. Of the 718 samples collected, 9 samples were removed from the analysis because they were either not tested or had invalid results on the 3 assays (Simplexa HSV 1 & 2 Direct, culture, or bi-directional sequencing). Of the 709 remaining samples, 13 samples were removed from the analysis because they were not tested on the tests included in the composite comparator method sufficient to generate a final comparator result. A total of 696 samples were used for the analysis. Samples were tested on Simplexa HSV 1 & 2 Direct at the collection sites, culture samples were sent to a central lab, and the sequencing samples were sent to DiaSorin Molecular. All samples were either tested fresh or frozen within 72 hours of sample collection. Aliquots were made from each sample; 1 aliquot was tested on Simplexa HSV 1 & 2 Direct at the collection site, and the remainder was sent to DiaSorin Molecular. One aliquot was sent for culture testing at an external site, 1 aliquot was used for bidirectional sequencing, and the last aliquot was held as a retain. Any sample that was not collected and frozen within 72 hours was disqualified.

The testing included 332 total runs with 638 evaluable controls, and 5 invalid control pairs. There were 718 patient samples of which 2 were not evaluable due to internal control failure, 5 not evaluable due to invalid runs, 1 was not evaluable due to wrong sample type, 1 not evaluable due to testing on a commercial instrument, 5 Insufficient Volume Errors, 3 not evaluable because of daily control issues, and 5 non-enrollable patient samples not meeting acceptance criteria (tested $> 7 2$ hrs post collection).

# Prospective Study 2 – Cutaneous and Mucocutaneous Swab Sample Type K173798

A total of 514 cutaneous and mucocutaneous lesion swab samples were prospectively collected July 24, 2017 through October 11, 2017. These samples were collected from patients with signs and symptoms of herpes simplex virus (HSV) infection from 4 geographically diverse sites. Of the 514 samples, 511 samples were evaluable on Simplexa HSV 1 & 2 Direct, 512 were evaluable by the culture method, and 510 were evaluable by the bi-directional sequencing method. Samples were tested on Simplexa HSV 1 & 2 Direct at the collection sites, culture samples were sent to a central lab, and the sequencing samples were sent to DiaSorin Molecular. All samples were either tested fresh or frozen within 72 hours of sample collection. Aliquots were made from each sample; 1 aliquot was tested on Simplexa HSV 1 & 2 Direct at the collection site, and the remainder was sent to DiaSorin Molecular. One aliquot was sent for culture testing at an external site, 1 aliquot was used for bi-directional sequencing, and the last aliquot was held as a retain. Any sample that was not collected and frozen within 72 hours was disqualified.

The testing included 153 total Runs with 250 evaluable controls, and 3 invalid control pairs. There were 514 patient samples of which 2 were not evaluable due to EC505 codes from both the HSV-1 (FAM) & HSV-2 (CFR610) channel, 2 Insufficient Volume errors, 18 samples not evaluable because of daily control issues (7 samples were non-evaluable due to invalid PC control runs and 11 samples were nonevaluable due to no daily control runs) and 1 was a non-enrollable patient sample that did not meet the acceptance criteria (tested $> 7 2$ hrs post collection).

Retrospective Study – Cutaneous and Mucocutaneous Swab Sample Type A total of 174 Cutaneous HSV-1 swabs, 174 Mucocutaneous HSV-2 swabs and 17 samples from unknown locations, were retrospectively collected June 6, 2011 to May 17, 2014 and February 21, 2017 through July 17, 2017. All samples were tested using the composite comparator method.

The clinical performance of the Simplexa HSV 1 & 2 Direct assay was evaluated by comparing the positive and negative percent agreement to a composite comparator algorithm consisting of culture, bi-directional sequencing and a FDA cleared NAAT. All samples yielding a positive result by either sequencing or culture were tested on an FDA cleared NAAT and a 2 out of 3 rule was used to determine the final composite results. All sites collected and tested the swab samples on the Simplexa HSV-1 and HSV-2 Direct and sent samples to a central lab for culture testing. For culture, each sample was tested for HSV- 2 first, and if positive for HSV-2, no further testing was performed. Samples that were HSV-2 culture negative were further tested for HSV-1 culture positivity. Dual positives could not be identified in the culture assay.

The available retained samples were sent to DiaSorin Molecular and tested in a validated bi-directional sequencing assay. Results for Simplexa HSV 1 & 2 Direct compared to the composite comparator algorithm are presented in the tables that follow when combined with the data from the second sample set.

# Prospective Results Composite Reference Method HSV-1 Cutaneous Swabs

The table below shows HSV-1 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method based on the observed prevalence in the study population for HSV-1 for Cutaneous swabs.

Prospective Results Composite Reference Method HSV-1 Mucocutaneous Swabs   

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Cutaneous for HSV-1)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>219</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(30/30)95% CI: 88.7% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>96.3%(182/189)95% CI: 92.6% to98.5%</td></tr></table>

The table below shows HSV-1 positive and negative percent Agreement (PPA and NPA) vs. Composite Reference Method based on the observed prevalence in the study population for HSV-1 for Mucocutaneous swabs.

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Mucocutaneous for HSV-1)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>180</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>703</td><td rowspan=1 colspan=1>706</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>721</td><td rowspan=1 colspan=1>887</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>98.2%(162/165)95% CI: 94.4% to99.6%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>97.5%(703/721)95% CI: 96.1% to98.4%</td></tr></table>

# Prospective Results Composite Reference Method HSV-2 Cutaneous Swabs

The table below shows HSV-2 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method based on the observed prevalence in the study population for HSV-2 for Cutaneous swabs.

Prospective Results Composite Reference Method HSV-2 Mucocutaneous Swabs   

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Cutaneous for HSV-2)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>183</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>219</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>97.0%(32/33)95% CI: 84.4% to99.5%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>97.9%(182/186)95% CI: 94.6% to99.2%</td></tr></table>

The table below shows HSV-2 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method based on the observed prevalence in the study population for HSV-2 for Mucocutaneous swabs.

<table><tr><td colspan="4" rowspan="1">Clinical Agreement - (Mucocutaneous for HSV-2)</td></tr><tr><td colspan="1" rowspan="2">Simplexa HSV 1 &amp; 2 DirectResults</td><td colspan="3" rowspan="1">Composite Reference Method</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">193</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">216</td></tr><tr><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">669</td><td colspan="1" rowspan="1">670</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">692</td><td colspan="1" rowspan="1">886</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">%PPA</td><td colspan="1" rowspan="1">99.5%(193/194)95% CI: 97.1% to100.0%</td><td colspan="1" rowspan="1">%NPA</td><td colspan="1" rowspan="1">96.7%(669/692)95% CI: 95.1% to97.8%</td></tr></table>

# Retrospective Results Composite Reference Method HSV-1 Cutaneous Swabs

The table below shows HSV-1 positive and negative percent agreement (PPA and NPA) vs.   
Composite Reference Method for retrospectively collected HSV-1 for Cutaneous swabs. The table below shows HSV-1 positive and negative percent Agreement (PPA and NPA) vs.   
Composite Reference Method for retrospectively collected HSV-1 for Mucocutaneous swabs.

Retrospective Results Composite Reference Method HSV-1 Mucocutaneous Swabs   

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Cutaneous for HSV-1)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>118</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(26/26)95% CI: 87.1% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>98.9%(91/92)95% CI: 94.1% to99.8%</td></tr></table>

Retrospective Results Composite Reference Method HSV-2 Cutaneous Swabs The table below shows HSV-2 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method for retrospectively collected HSV-2 for Cutaneous swabs.   

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Mucocutaneous for HSV-1)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>146</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(32/32)95% CI: 89.3% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>99.1%(113/114)95% CI: 95.2% to100.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Cutaneous for HSV-2)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>118</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(29/29)95% CI: 88.3% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>100.0%(89/89)95% CI: 95.9% to100.0%</td></tr></table>

# Retrospective Results Composite Reference Method HSV-2 Mucocutaneous Swabs

The table below shows HSV-2 positive and negative percent Agreement (PPA and NPA) vs.   
Composite Reference Method for retrospectively collected HSV-2 for Mucocutaneous swabs.

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Mucocutaneous for HSV-2)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>124</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>146</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(22/22)95% CI: 85.1% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>100.0%(124/124)95% CI: 97.0% to100.0%</td></tr></table>

# Retrospective Results Composite Reference Method HSV-1 Unknown Locations Cutaneous Swabs

The table below shows HSV-1 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method for retrospectively collected HSV-1 for Cutaneous swabs from unknown locations.

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Unknown Sample Locations for HSV-1)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(3/3)95% CI: 43.9% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>100.0%(12/12)95% CI: 75.8% to100.0%</td></tr></table>

# Retrospective Results Composite Reference Method HSV-2 Unknown Locations Cutaneous Swabs

The table below shows HSV-2 positive and negative percent agreement (PPA and NPA) vs. Composite Reference Method for retrospectively collected HSV-2 for Cutaneous swabs from unknown locations.

<table><tr><td rowspan=1 colspan=4>Clinical Agreement - (Unknown Sample Locations for HSV-2)</td></tr><tr><td rowspan=2 colspan=1>Simplexa HSV 1 &amp; 2 DirectResults</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%PPA</td><td rowspan=1 colspan=1>100.0%(3/3)95% CI: 43.9% to100.0%</td><td rowspan=1 colspan=1>%NPA</td><td rowspan=1 colspan=1>100.0%(12/12)95% CI: 75.8% to100.0%</td></tr></table>

4. Clinical cut-off:

Not Applicable

5. Expected values/Reference range:

The observed expected values using the Simplexa HSV 1 & 2 Direct assay are presented below.   
The data is stratified by age, gender and lesion location.

# Cutaneous and Mucocutaneous Lesion Swabs by Age and Gender

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>Age Group</td><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=2>Simplexa HSV 1&amp; 2Direct HSV-1 Results</td><td rowspan=1 colspan=2>Simplexa HSV 1&amp; 2Direct HSV-2 Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=4 colspan=1>Female</td><td rowspan=1 colspan=1>≤18 years</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>18.8%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>15.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;18 to 21 years</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>35.4%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>25.0%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 years</td><td rowspan=1 colspan=1>779</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>18.1%</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>26.2%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1003</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>19.8%</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>24.7%</td></tr><tr><td rowspan=4 colspan=1>Male</td><td rowspan=1 colspan=1>≤18 years</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>30.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>&gt;18 to 21 years</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>45.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>22.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;21 years</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>13.4%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>19.7%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>21.1%</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>17.2%</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1212</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>23.4%</td></tr></table>

# Cutaneous and Mucocutaneous Lesion Swabs by Lesion Location

<table><tr><td rowspan=2 colspan=1>Skin Type</td><td rowspan=2 colspan=1>LesionLocation</td><td rowspan=2 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=2>Simplexa HSV 1&amp; 2Direct HSV-1 Results</td><td rowspan=1 colspan=2>Simplexa HSV 1&amp; 2Direct HSV-2 Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=3 colspan=1>Cutaneous</td><td rowspan=1 colspan=1>Genital</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>22.8%</td></tr><tr><td rowspan=1 colspan=1>Skin</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>13.0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7.6%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>17.5%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>16.7%</td></tr><tr><td rowspan=8 colspan=1>Mucocutaneous</td><td rowspan=1 colspan=1>Anorectal</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>37.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>12.5%</td></tr><tr><td rowspan=1 colspan=1>Genital</td><td rowspan=1 colspan=1>854</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>19.6%</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>27.2%</td></tr><tr><td rowspan=1 colspan=1>Nasal</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Ocular</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Oral</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>39.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Urethra</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>940</td><td rowspan=1 colspan=1>196</td><td rowspan=1 colspan=1>20.9%</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>25.0%</td></tr><tr><td rowspan=2 colspan=1>Unknown</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>15.9%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>25.0%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>15.9%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>25.0%</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>1212</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>23.4%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.