# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K173263

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the Xpert Carba-R Assay on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems) in the qualitative detection of the blaKPC, $b l a _ { \mathrm { N D M } } , b l a _ { \mathrm { V I M } } , b l a _ { \mathrm { O X A - } 4 8 }$ , and $b l a _ { \mathrm { I M P } }$ gene sequences from perirectal swab specimens.

# C. Measurand:

Target DNA sequence of the following genes: blaKPC, blaNDM, blaVIM, blaOXA-48, and blaIMP

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay

# E. Applicant:

Cepheid

F. Proprietary and Established Names:

Proprietary Name: Xpert Carba-R

Common Name: Xpert Carba-R Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 (Antimicrobial susceptibility test powder)

2. Classification: Class II

3. Product code(s):

POC-System, nucleic acid amplification test, DNA, antimicrobial resistance marker, direct specimen

PMY-System, nucleic acid amplification test, DNA, carbapenem non-susceptible gram negative organism, colony

OOI-Real-time nucleic acid amplification system

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The Xpert Carba-R Assay, performed on the GeneXpert Instrument Systems, is a qualitative in vitro diagnostic test designed for the detection and differentiation of the $b l a _ { \mathrm { \tiny K P C } } , b l a _ { \mathrm { \tiny N D M } } , b l a _ { \mathrm { \tiny V I M } } , b l a _ { \mathrm { \tiny O X A - 4 8 } } .$ and $b l a _ { \mathrm { { I M P } } }$ gene sequences associated with carbapenemnon-susceptibility. The test utilizes automated real-time polymerase chain reaction (PCR).

The Xpert Carba-R Assay is intended as an aid to infection control in the detection of carbapenem-non-susceptible bacteria that colonize patients in healthcare settings. A negative Xpert Carba-R Assay result does not preclude the presence of other resistance mechanisms.

The Xpert Carba-R Assay is for use with the following sample types:

# Pure Colonies

The assay is performed on carbapenem-non-susceptible pure colonies of Enterobacteriaceae, Acinetobacter baumannii, or Pseudomonas aeruginosa, when grown on blood agar or MacConkey agar. For testing pure colonies, the Xpert Carba-R Assay should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing.

The identification of a $b l a _ { \mathrm { \tiny { I M P } } } , b l a _ { \mathrm { \tiny { N D M } } } ,$ or $b l a _ { \mathrm { v m } }$ metallo-beta-lactamase gene (i.e., the genes that encode the IMP, NDM, and VIM metallo-beta-lactamases, respectively) may be used as an aid to clinicians in determining appropriate therapeutic strategies for patients with known or suspected carbapenem-non- susceptible bacterial infections.

# Rectal and Perirectal Swab Specimens

The assay is performed on rectal and perirectal swab specimens from patients at risk for intestinal colonization with carbapenem-non-susceptible bacteria. Concomitant cultures are necessary to recover organisms for epidemiological typing, antimicrobial susceptibility testing, and for further confirmatory bacterial identification.

The Xpert Carba-R Assay, when performed on rectal and perirectal swab specimens, is not intended to guide or monitor treatment for carbapenem-non-susceptible bacterial infections or to determine infection from carbapenem-non-susceptible bacteria.

2. Indication(s) for use: Same as the Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

The Xpert Carba-R Assay detects blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ from rectal and perirectal swab specimens and is not for bacterial identification or to report susceptibility status.

The detection of assay targets by the Xpert Carba-R Assay does not indicate the presence of viable organisms containing the resistance marker.

The Xpert Carba-R Assay is not a sub-typing tool and does not report variants of the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ genes.

Antimicrobial agents, such as new beta-lactam/beta-lactamase inhibitor combinations, have varying activity against bacteria producing different types of beta-lactamases. Xpert Carba-R Assay results showing the presence of $b l a _ { \mathrm { I M P } } , b l a _ { \mathrm { V I M } }$ , and $b l a _ { \mathrm { N D M } }$ metallo-betalactamase genes from pure colonies of the claimed organisms may be helpful in determining a therapeutic strategy that includes beta-lactam/beta-lactamase inhibitor combinations.

4. Special instrument requirements:

The Xpert Carba-R Assay uses PCR technology on the GeneXpert Instrument Systems, which extract, amplify, and detect the target DNA.

# I. Device Description:

The Xpert Carba-R Assay is an automated real-time polymerase chain reaction (PCR) in vitro diagnostic test for the qualitative detection of the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ gene sequences from pure colonies of target organisms, rectal swab specimens, and perirectal swab specimens. The Xpert Carba-R Assay is intended as an aid for infection control to monitor the spread of carbapenem-non-susceptible organisms in healthcare settings. The assay is performed on Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems). The GeneXpert Instrument Systems utilize single-use, disposable cartridges (Xpert Carba-R cartridges) containing PCR reagents that allow for automated sample preparation, amplification, and real-time detection of gene targets in approximately 50 minutes. A Sample

Processing Control (SPC) and a Probe Check Control (PCC) have been incorporated into the assay design to address key failure modes that could result in a false negative determination.

The GeneXpert Instrument Systems (GeneXpert Dx Systems and the GeneXpert Infinity Systems) have 1 to 80 randomly accessible modules, depending upon the instrument, that are each capable of performing separate sample processing and real-time PCR tests. Because the cartridges are self-contained and specimens never come into contact with working parts of the instrument modules, cross-contamination between samples is minimized.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Xpert Carba-R Assay   
2. Predicate 510(k) number(s): K160901   
3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Predicate Device</td><td colspan="1" rowspan="1">Subject Device</td></tr><tr><td colspan="1" rowspan="1">Cepheid Xpert Carba-R Assay(K160901)</td><td colspan="1" rowspan="1">Cepheid Xpert Carba-R Assay(K173263)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Xpert Carba-R Assay,performed on the GeneXpertInstrument Systems, is a qualitativein vitro diagnostic test designed forthe detection and differentiation ofthe blakpc, blandm, blavm, blaoxa-48,and blamp gene sequences associatedwith carbapenem-non-susceptibility.The test utilizes automated real-timepolymerase chain reaction (PCR).The Xpert Carba-R Assay is intendedas an aid to infection control in thedetection of carbapenem-non-susceptible bacteria that colonizepatients in healthcare settings. TheXpert Carba-R Assay is not intendedto guide or monitor treatment forcarbapenem-non-susceptiblebacterial infections. A negative XpertCarba-R Assay result does notpreclude the presence of otherresistance mechanisms.</td><td colspan="1" rowspan="1">The Xpert Carba-R Assay, performedon the GeneXpert InstrumentSystems, is a qualitative in vitrodiagnostic test designed for thedetection and differentiation of theblakc, blaNDm, blavm, blaoxA-48, andblamp gene sequences associated withcarbapenem-non-susceptibility. Thetest utilizes automated real-timepolymerase chain reaction (PCR).The Xpert Carba-R Assay is intendedas an aid to infection control in thedetection of carbapenem-non-susceptible bacteria that colonizepatients in healthcare settings. Anegative Xpert Carba-R Assay resultdoes not preclude the presence ofother resistance mechanisms.The Xpert Carba-R Assay is for usewith the following sample types:Pure ColoniesThe assay is performed on</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The Xpert Carba-R Assay is for usewith the following sample types:Rectal Swab SpecimensThe assay is performed on rectalswab specimens from patients at riskfor intestinal colonization withcarbapenem-non-susceptiblebacteria. Concomitant cultures arenecessary to recover organisms forepidemiological typing,antimicrobial susceptibility testing,and for further confirmatory bacterialidentification.Pure ColoniesThe assay is performed oncarbapenem-non-susceptible purecolonies of Enterobacteriaceae,Acinetobacter baumannii, orPseudomonas aeruginosa, whengrown on blood agar or MacConkeyagar. For testing pure colonies, theXpert Carba-R Assay should be usedin conjunction with other laboratorytests including phenotypicantimicrobial susceptibility testing.</td><td colspan="1" rowspan="1">carbapenem-non-susceptible purecolonies of Enterobacteriaceae,Acinetobacter baumannii, orPseudomonas aeruginosa, whengrown on blood agar or MacConkeyagar. For testing pure colonies, theXpert Carba-R Assay should be usedin conjunction with other laboratorytests including phenotypicantimicrobial susceptibility testing.The identification of a blamp, blaNDM,or blavim metallo-beta-lactamase gene(i.e., the genes that encode the IMP,NDM, and VIM metallo-beta-lactamases, respectively) may be usedas an aid to clinicians in determiningappropriate therapeutic strategies forpatients with known or suspectedcarbapenem-non- susceptible bacterialinfections.Rectal and Perirectal SwabSpecimensThe assay is performed on rectal andperirectal swab specimens frompatients at risk for intestinalcolonization with carbapenem-non-susceptible bacteria. Concomitantcultures are necessary to recoverorganisms for epidemiological typing,antimicrobial susceptibility testing,and for further confirmatory bacterialidentification.The Xpert Carba-R Assay, whenperformed on rectal and perirectalswab specimens, is not intended toguide or monitor treatment forcarbapenem-non-susceptible bacterialinfections or to determine infectionfrom carbapenem-non-susceptiblebacteria.</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Fully-automated nucleic acidamplification (DNA); real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge</td><td colspan="1" rowspan="1">Disposable single-use, multi-chambered fluidic cartridge</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">TaqMan Probes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal sample processing control(SPC); Probe check control (PCC);External controls available</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">blakpc, blanDm, blavIm, blaoxA-48, andblamp gene sequences</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">GeneXpert Instrument System(includes GeneXpert Dx, Infinity-48,Infinity-48s, and Infinity-80)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time to obtain test results</td><td colspan="1" rowspan="1">Approximately 50 minutes to results</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretationof test results</td><td colspan="1" rowspan="1">Diagnostic software of the GeneXpertInstrument System</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Predicate Device</td><td rowspan=1 colspan=1>Subject Device</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Cepheid Xpert Carba-R Assay(K160901)</td><td rowspan=1 colspan=1>Cepheid Xpert Carba-R Assay(K173263)</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>rectal swab specimens</td><td rowspan=1 colspan=1>perirectal swab specimen</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. ASTM D4169-09, Standard Practice for Performance Testing of Shipping Containers and Systems.   
2. CLSI EP5-A3, Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline—Third Edition, 2014   
3. CLSI EP15-A3, User Verification of Performance for Precision and Trueness; Approved Guideline–Third Edition, 2014   
4. CLSI M02-A11. Performance standards for Antimicrobial Disk Susceptibility Tests; Eleventh Edition, 2012   
5. CLSI M07-A9. Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; Approved standard—Ninth Edition, 2012   
6. CLSI M07-A10. Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; Approved standard—Tenth Edition, 2015   
7. CLSI M100-S24. Performance Standard for Antimicrobial Susceptibility Testing; Approved standard—Twenty-fourth Informational Supplement, 2014   
8. CLSI MM3-A2, Molecular Diagnostic Methods for Infectious Disease; Approved Guideline–Second Edition, 2006   
9. EN 13640, Stability Testing of in vitro Diagnostic Reagents, June 2002   
10. General Principles of Software Validation; Final Guidance for Industry and FDA Staff, issued January 11, 2002   
11. Guidance for Industry and FDA Staff–Format for Traditional and Abbreviated 510(k)s, issued August 12, 2005   
12. Guidance for Industry and FDA Staff-Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems, issued on March 10, 2005   
13. Guidance for Industry and FDA Staff–Content of Premarket Submissions for Management of Cybersecurity in Medical Devices, issued on October 2, 2014   
14. Guidance for Industry-Cybersecurity for Networked Medical Devices Containing Offthe-Shelf (OTS) Software, issued January 14, 2005   
15. Guidance for Industry and FDA Staff–Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, issued May 11, 2005   
16. Guidance for Industry, FDA Reviewers and Compliance on Guidance for Off-the-Shelf Software Use in Medical Devices; issued September 9, 1999   
17. Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff-Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable, issued April 25, 2006

# L. Test Principle:

The Xpert Carba-R Assay cartridges contain reagents for the detection of blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ gene sequences from perirectal swab specimens. Each swab is resuspended in $5 \mathrm { m L }$ of Sample Reagent. The sample is vortexed then transferred $( 1 . 7 \mathrm { m L } )$ ) to the sample chamber of a disposable Xpert Carba-R Assay cartridge. The user initiates a test from the system-user interface and places the cartridge into the GeneXpert instrument platform, which performs hands-off real-time, multiplex PCR for the detection of target sequences.

Results of the assay run are interpreted by the GeneXpert Instrument System software from measured fluorescent signals and embedded calculation algorithms. The results are automatically generated at the end of the process in a report that can be viewed and printed. Basic users see test results reported as “red” highlighted for DETECTED results and “green” highlighted for NOT DETECTED results. Additional results that can be reported include: INVALID, ERROR, and NO RESULT.

# Interpretation of Results

The Xpert Carba-R Assay provides test results for the IMP, VIM, NDM, KPC, and OXA-48 target DNA sequences. A Sample Processing Control (SPC) and a Probe Check Control (PCC) have been designed for the assay as internal controls to enable the GeneXpert Instrument System to detect specific failure modes related to assay performance. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for any of the IMP, VIM, NDM, KPC, and OXA-48 targets, or SPC target, a probe check error is reported and the test will not continue. The assay also reports if the test has an INVALID, ERROR or NO RESULT. Under these conditions, the test will need to be repeated using a new sample and cartridge. Retest procedures are described in the Xpert Carba-R package insert. An interpretation table for test results is shown in Table 1.

Table 1. Interpretation of Test Results for the Xpert Carba-R Assay   

<table><tr><td rowspan=1 colspan=1>Result Report</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>DETECTED</td><td rowspan=1 colspan=1>Target(s): For a valid &quot;DETECTED&quot; test result, PCRamplification of the target DNA sequence gives Ct value(s)within the valid range and a fluorescence endpoint above thethreshold setting for IMP, VIM, NDM, KPC, and/or OXA-48;SPC: Not applicable (if at least one target detected); PCC: PASS;all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>NOT DETECTED</td><td rowspan=1 colspan=1>Target(s): For a valid &quot;NOT DETECTED&quot; test result, no validCt(s) are reported for the IMP, VIM, NDM, KPC, and/or OXA-48target DNA sequences; SPC: PASS, PCR amplification of theSPC DNA sequence gives a Ct value within the valid range and afluorescence endpoint above the threshold setting; PCC: PASS;all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>ERROR</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC: NORESULT; PCC: FAIL*, one or more of the probe check resultsfailed. *If the probe check passed, the error is caused by a systemcomponent failure.</td></tr><tr><td rowspan=1 colspan=1>INVALID</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC:FAIL, No PCR amplification of the SPC DNA sequence or theSPC Ct is not within valid range and the fluorescence endpoint isbelow threshold setting; PCC: PASS; all probe check results pass.</td></tr><tr><td rowspan=1 colspan=1>NO RESULT</td><td rowspan=1 colspan=1>Target(s): Presence or absence of IMP, VIM, NDM, KPC, andOXA-48 target DNA sequences cannot be determined; SPC: NORESULT; PCC: Not applicable.A &quot;NO RESULT&quot; indicates that insufficient data were collected.For example, the operator stopped a test that was in progress or apower failure occurred.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multi-center reproducibility study of the Xpert Carba-R Assay was conducted using samples suspended in perirectal matrix. An 11-member panel was tested that included: 1) two different organisms for each of the five resistance genes detected by the Xpert Carba-R Assay and 2) one negative sample for all five gene targets. Each organism was spiked into a pooled negative perirectal matrix at low positive $( \mathord { \sim } 1 \mathrm { X }$ LoD) and moderate positive levels (2-3X LoD). To measure site-to-site reproducibility, the (11)-member panel was tested in replicates of four each day at three (3) sites over a six day testing period with two operators per site. Three lots of

Xpert Carba-R Assay cartridges were used at each testing site. A total of 1584 samples suspended in perirectal matrix were evaluated, where 144 replicates for each of the 11 different panel members were tested. In perirectal matrix, $9 9 . 4 \%$ (1574/1584) of samples were successful and produced the expected result on the first attempt. Four (4) ERROR cases, four (4) INVALID results, and two (2) NO RESULT outcomes were reported. All ten samples yielded valid results upon repeat testing. The results of the reproducibility study are summarized in Table 2 below.

Table 2. Reproducibility of the 11-Member Perirectal Sample Panel   

<table><tr><td rowspan=2 colspan=1>Resistance Gene</td><td rowspan=1 colspan=3>Site 1a</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>% TotalAgreement bySample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>IMP Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>IMP Low Pos</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>96.5%(139/144)</td></tr><tr><td rowspan=1 colspan=1>VIM Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>VIM Low Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>91.7%(44/48)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>83.3%(20/24)</td><td rowspan=1 colspan=1>89.6%(43/48)</td><td rowspan=1 colspan=1>92.4%(133/144)</td></tr><tr><td rowspan=1 colspan=1>NDM Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>NDM Low Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>87.5%(21/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>93.8%45/48)</td><td rowspan=1 colspan=1>97.9%(141/144)</td></tr><tr><td rowspan=1 colspan=1>KPC Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>KPC Low Pos</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>91.7%44/48)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>93.8%(135/144)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Low Pos</td><td rowspan=1 colspan=1>87.5%(21/24)</td><td rowspan=1 colspan=1>87.5%(21/24)</td><td rowspan=1 colspan=1>87.5%(42/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>93.8%(135/144)</td></tr></table>

aEach site tested one GeneXpert Instrument—GeneXpert Dx, Infinity-80, or Infinity-48.

The reproducibility of the Xpert Carba-R Assay was also evaluated by assessing the fluorescent signal (expressed in Ct values) for each target detected. The mean, standard deviation (SD), and coefficient of variation (CV) between sites, days, and operators for each panel member are presented in Table 3 below.

Table 3. Reproducibility of the Fluorescent Signal   

<table><tr><td rowspan=2 colspan=1>ResistanceGene(sample)</td><td rowspan=2 colspan=1>AssayChannel(Analyte)</td><td rowspan=2 colspan=1>N^d</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenSite</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>WithinAssay</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>IMP Mod Pos</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>IMP Low Pos</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>VIM Mod Pos</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>VIM Low Pos</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>NDM Mod Pos</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>NDM Low Pos</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>KPC Mod Pos</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>KPC Low Pos</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Mod Pos</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Low Pos</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.9</td></tr></table>

aResults with non-zero Ct values out of 144.

At ${ \sim } 1 \mathrm { X }$ LoD, the expected target was not detected in some samples (See Table 2 above). The lowest $\%$ total agreement, $9 2 . 4 \%$ (133/144), was observed with the low positive VIM sample. All moderate positive samples (2-3X LoD) gave the expected result. All negative samples were correctly identified as negative (144/144). Of the 76 external control samples run, $9 8 . 7 \%$ (75/76) gave a result on the first attempt that was the expected result. Retest of the one indeterminate control sample was successful. Agreement between sites, operators, and lots was evaluated using Fisher’s Exact test. The data presented in Table 2 and Table 3 demonstrated an acceptable reproducibility for the Xpert Carba-R Assay on the GeneXpert Instrument Systems.

b. Linearity/assay reportable range: Not Applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Controls

Commercially-available external controls can also be run in accordance with local, state, and federal accrediting organizations, as applicable. For each day of the study, an external negative control and two types of experimental positive controls were tested. One external positive control consisted of $E$ . coli cells containing a plasmid with an insert carrying amplicon sequences from all five Xpert Carba-R target analyte genes (Multivalent External Positive Control). The second external positive control consisted of individual carbapenemase-producing bacteria, each harboring only one of the Xpert Carba-R target carbapenemase genes. On each testing day, one negative control, the five-gene construct positive control, and two of five individual bacterial positive controls were tested. Of the 970 external control samples run, $9 9 . 2 \%$ (962/970) gave a valid result on the first attempt. Retesting of the eight indeterminate controls gave the expected results.

External controls include the following bacteria harboring target genes:

Multivalent External Positive Control—External positive control (inactivated Escherichia coli carrying plasmid with KPC, NDM, VIM, IMP, and OXA-48 gene sequences)

Individual Positive Controls · K. pneumoniae KPC (ATCC BAA-1705) K. pneumoniae NDM (ATCC BAA-2146) K. pneumoniae VIM (NCTC 13439) K. pneumoniae OXA-48 (NCTC 13442) Escherichia coli IMP (NCTC 13476)

External Negative Control—Inactivated E. coli containing a plasmid with no resistance gene inserted.

# Internal Control (IC) Reaction Analysis

Internal controls enable the system to detect specific failure modes that could potentially result in an incorrect test result. Each Xpert Carba-R Assay includes a Sample Processing Control (SPC) and Probe Check Control (PCC) pre-loaded in the cartridge and provided with the assay.

# Sample Processing Control (SPC)

The SPC contains Bacillus globigii that is included in each cartridge to verify adequate processing of the sample. The SPC verifies that lysis of bacteria has occurred if the organisms are present and verifies the effectiveness of each sample preparation step—reaction tube filling, reaction components are present and functioning, and monitoring for the presence of potential inhibitor(s) in the PCR assay. Test results are reported as INVALID if the SPC fails to meet the valid minimum or maximum Ct specification.

Probe Check control (PCC)

The PCC verifies reagent rehydration, real-time PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for any of the IMP, VIM, NDM, KPC, and OXA-48 targets or SPC target, a probe check error is reported and the test will not continue. If a probe check error is reported, the test may be repeated using a new sample, new cartridge, and new reagents.

# Stability Studies

A number of studies were conducted to establish the specimen stability of perirectal swabs with the Xpert Carba-R Assay. This study was performed to: 1) establish the specimen stability for the perirectal swabs after specimen collection but prior to transfer to the Xpert Carba-R Sample Reagent and 2) establish the specimen stability of perirectal swabs eluted into Xpert Carba-R Sample Reagent.

# Stability of Perirectal Swabs

In order to support a claim for the period of time that perirectal swabs could be held before transfer to the Xpert Carba-R Sample Reagent and further processing, a mixture of five carbapenemase-producing bacteria (at approximately 3X LoD) was spiked onto negative perirectal matrix swabs to create known positive samples. Both positive and negative matrix swab samples were tested with the Xpert Carba-R Assay after being stored for 1, 3, 5, and 7 days at $2 ^ { \circ } \mathrm { C }$ , $8 ^ { \circ } \mathrm { C }$ , $1 5 \mathrm { { ^ \circ C } }$ and $2 8 ^ { \circ } \mathrm { C }$ . Four (4) replicate negative matrix swab samples and 14 replicate positive matrix swab samples were tested per time point and temperature condition; at the time of testing, swabs were then added to Xpert Carba-R Sample Reagent. Eight replicate positive and negative matrix swab samples were tested at time $\scriptstyle { \mathrm { t } } = 0$ .

Under the conditions of this study, negative specimens at all storage conditions and temperatures tested were correctly identified using the Xpert Carba-R Assay. For positive specimens, 229 of 232 replicates (with all five targets) were DETECTED. Three samples had one target missed. When retested, 2 of the 3 samples yielded the expected result (one sample was not retested). The data supports that perirectal swabs are stable when stored at $8 ^ { \circ } \mathrm { C } - 2 8 ^ { \circ } \mathrm { C }$ for up to seven (7) days from the time of collection prior to transfer to Xpert Carba-R Sample Reagent to initiate testing with the Xpert Carba-R Assay.

Stability after Perirectal Swabs Resuspended in Xpert Carba-R Sample Reagent To establish a claim for the stability of perirectal swabs in the Xpert Carba-R Sample Reagent, pooled negative perirectal swab matrix was spiked at approximately 3X LoD with a mixture of five carbapenemase-producing bacteria that harbored targets of the Xpert Carba-R Assay. Aliquots of the positive swab matrix (with bacteria) in Sample Reagent were prepared to create positive samples. Negative samples in the study consisted of aliquots of negative pooled swab matrix containing Xpert Carba-R Sample Reagent. Positive and negative swab matrix samples were tested with the Xpert Carba-R Assay after being stored for 1, 3, 5 and 7 days at $2 \mathrm { { } ^ { \circ } C }$ , $8 \mathrm { { } ^ { \circ } C }$ , $1 5 ^ { \circ } \mathrm { C }$ , and $2 8 ^ { \circ } \mathrm { C }$ . Four (4) replicate negative matrix samples and 14 replicate positive matrix swab samples were tested per time point and temperature condition. Positive and negative matrix swab replicates were also tested at time $\scriptstyle { \mathrm { t } } = 0$ .

Under the conditions of this study, positive and negative specimens at all storage conditions and temperatures tested were correctly reported using the Xpert CarbaR Assay. The data supports that sample material from perirectal swabs are stable in Xpert Carba-R Sample Reagent for up to 5 days when stored at $2 ^ { \circ } \mathrm { C } - 2 8 ^ { \circ } \mathrm { C }$ prior to testing with the Xpert Carba-R Assay.

Of the 655 runs in the stability study, four (4) runs $( 0 . 6 1 \% )$ produced indeterminate GeneXpert results—(2) NO RESULT, (1) ERROR, and (1) INVALID. Three runs were repeated yielding the expected Xpert Carba-R Assay result. The one invalid run was not repeated.

The stability studies support the following storage conditions prior to testing with the Xpert Carba-R Assay: 1) up to 5 days at $1 5 ^ { \circ } \mathrm { C } - 2 8 ^ { \circ } \mathrm { C }$ for perirectal swab specimens in transport tubes before transfer to Xpert Carba-R Sample Reagent and 2) up to 5 days at $2 ^ { \circ } \mathrm { C } { - } 2 8 ^ { \circ } \mathrm { C }$ for perirectal swab specimens in Xpert Carba-R Sample Reagent.

# d. Detection limit:

Using whole organisms seeded into perirectal matrix, a study was conducted to determine the Limit of Detection (LoD) of the Xpert Carba-R Assay for carbapenemase-producing organisms harboring the blaKPC, blaNDM, blaVIM, blaOXA-48, and $b l a _ { \mathrm { I M P } }$ gene sequence targets. The LoD was defined as the lowest concentration (reported as cells/swab and cells/mL in Sample Reagent) of sample that can be reproducibly distinguished from negative samples with $9 5 \%$ confidence. Pre-screened negative perirectal matrix was used to confirm that the Limit of Blank (LoB) was zero. Two strains harboring each assay target were tested. Point estimates and twosided $9 5 \%$ confidence intervals for the analytical LoD (reported as CFU/swab) were determined using probit regression analysis. The LoD estimate was verified by preparing two independent dilutions of each bacterial culture to the point estimate LoD value. Twenty measurements (10 from each dilution) were tested, and the study was completed using at least two unique lots of Xpert Carba-R Assay reagents.

Of the 2,815 runs in the LoD study to estimate the LoD of various targets in perirectal matrix (excluding the external controls), 33 runs $( 1 . 2 \% )$ provided indeterminate GeneXpert results (4 NO RESULT, 8 ERROR, 21 INVALID). All 33 runs were repeated yielding Carba-R valid results as expected. A total of 190 of 192 external control runs provided valid GeneXpert results. Repeat testing yielded expected results for the external controls.

All the data was collected with the GeneXpert Dx software version $4 . 4 \mathrm { a }$ on the GeneXpert Dx GX-IV and GX-XVI instruments and with the Infinity Xpertise software version 6.1 on the Infinity-80 instrument. Table 4 shows the estimated LoD results by Probit Analysis for the test panel in perirectal matrix. The LoDs for assay targets tested in perirectal swabs were similar to the results previously determined for rectal swab (K160901).

Table 4. LoD for Organisms in Perirectal Matrix Harboring Carbapenemase Genes using the Xpert Carba-R Assay in 5 mL Sample Reagent   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Target Gene</td><td colspan="1" rowspan="1">LOD ClaimCFU/swaba</td><td colspan="1" rowspan="1">Estimated LoD InSample ReagentCFU/mLb</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">635</td><td colspan="1" rowspan="1">127</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">901</td><td colspan="1" rowspan="1">180</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">VIM-4</td><td colspan="1" rowspan="1">446</td><td colspan="1" rowspan="1">89</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">358</td><td colspan="1" rowspan="1">72</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1303</td><td colspan="1" rowspan="1">261</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">223</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">137</td><td colspan="1" rowspan="1">27</td></tr></table>

a. Colony counts per swab confirmed by plating of each organism. b. Estimated LoD for swab in 5 mL of Sample Reagent using 0.2 conversion factor: $\phantom { - } 0 . 2 x$ CFU/swab

# Analytical Reactivity

Using whole organisms seeded into perirectal matrix, an Inclusivity Study was conducted to test reactivity of the Xpert Carba-R Assay with 72-member wellcharacterized bacterial isolates. The panel consisted of the following molecular resistance marker groups: (11) $b l a _ { \mathrm { K P C } }$ isolates, (13) $b l a _ { \mathrm { N D M } }$ isolates, (11) blaVIM isolates, (8) blaOXA-48 isolates, (5) $b l a _ { \mathrm { N D M / O X A - 1 8 1 } }$ isolates, (6) blaOXA-181 isolates, (17) $b l a _ { \mathrm { I M P } }$ isolates, and (1) $b l a _ { \mathrm { K P C / V I M } }$ isolate. Strains were seeded at 3X LoD and tested in pooled negative perirectal swab matrix. For a list of strains tested during the Analytical Reactivity Study, please refer to Table 5 below.

Table 5. Strains Tested in Analytical Reactivity   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConfirmedGeneticResistanceMarker</td><td rowspan=1 colspan=1>ConcentrationTested(CFU/mL)</td><td rowspan=1 colspan=1>Mean Ct</td></tr><tr><td rowspan=1 colspan=5>KPC Isolates</td></tr><tr><td rowspan=1 colspan=1>NCTC 13438</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-3</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>35.8</td></tr><tr><td rowspan=1 colspan=1>31551</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>34.5</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=1 colspan=1>CFVL</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>34.4</td></tr><tr><td rowspan=1 colspan=1>KBM18</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=1 colspan=1>COL</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>33.1</td></tr><tr><td rowspan=1 colspan=1>BM9</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-3</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>CGNC</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>PA3</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.3</td></tr><tr><td rowspan=1 colspan=1>PA-COL</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>34.4</td></tr><tr><td rowspan=1 colspan=1>164-3</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>33.7</td></tr><tr><td rowspan=1 colspan=3>NDM Isolates</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2146</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>34262</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>33.2</td></tr></table>

IMP Isolates   

<table><tr><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ConfirmedGeneticResistanceMarker</td><td colspan="1" rowspan="1">ConcentrationTested(CFU/mL)</td><td colspan="1" rowspan="1">Mean Ct</td></tr><tr><td colspan="1" rowspan="1">GEN</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">130</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">3047</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">33.3</td></tr><tr><td colspan="1" rowspan="1">7892</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">34.2</td></tr><tr><td colspan="1" rowspan="1">CAN</td><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">EGY</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">NDM-2</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">35.6</td></tr><tr><td colspan="1" rowspan="1">I5</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-4</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">405</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-5</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">CF-ABE</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">34.0</td></tr><tr><td colspan="1" rowspan="1">73999</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">39365</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">VIM Isolates</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">VIM-10</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">31.2</td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">130</td><td colspan="1" rowspan="1">31.8</td></tr><tr><td colspan="1" rowspan="1">NCTC 13440</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="1">758</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">32.0</td></tr><tr><td colspan="1" rowspan="1">PA-87</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">32.6</td></tr><tr><td colspan="1" rowspan="1">B92A</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">31.0</td></tr><tr><td colspan="1" rowspan="1">Coll</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">32.7</td></tr><tr><td colspan="1" rowspan="1">BM19</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">32.7</td></tr><tr><td colspan="1" rowspan="1">KOW7</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">VIM-4</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">DIH</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-19</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">33.6</td></tr><tr><td colspan="1" rowspan="1">MSH2014-3</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA Isolates</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">34.2</td></tr><tr><td colspan="1" rowspan="1">OM11</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.4</td></tr><tr><td colspan="1" rowspan="1">501</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">DUW</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">OM22</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">34.3</td></tr><tr><td colspan="1" rowspan="1">BOU</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">34.6</td></tr><tr><td colspan="1" rowspan="1">TUR</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">34.0</td></tr><tr><td colspan="1" rowspan="1">11670</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">MSH2014-64</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">280</td><td colspan="1" rowspan="1">36.5</td></tr><tr><td colspan="1" rowspan="1">MSH2014-72</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">35.0</td></tr><tr><td colspan="1" rowspan="1">166643</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">34.3</td></tr><tr><td colspan="1" rowspan="1">42194</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">35.3</td></tr><tr><td colspan="1" rowspan="1">CDC0051</td><td colspan="1" rowspan="1">Klebsiella ozaenaea</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">35.5</td></tr><tr><td></td><td></td><td colspan="3" rowspan="1">IMP. Isolates</td></tr><tr><td colspan="1" rowspan="1">NCTC 13476</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">34.4</td></tr><tr><td colspan="1" rowspan="1">695</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">1720</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">2340</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">34.3</td></tr><tr><td colspan="1" rowspan="1">IMPBMI</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="1">6852</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">33.8</td></tr><tr><td colspan="1" rowspan="1">Yonsei_1</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">34.7</td></tr><tr><td colspan="1" rowspan="1">Yonsei_2</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">34.6</td></tr><tr><td colspan="1" rowspan="1">70450-1</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">3994</td><td colspan="1" rowspan="1">Pseudomonas spp.</td><td colspan="1" rowspan="1">IMP-10</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">MKAM</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">5344</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">CDC0161</td><td colspan="1" rowspan="1">Enterobacteraerogenesa</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">36.3</td></tr><tr><td colspan="1" rowspan="1">3985</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-11</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">34.4</td></tr><tr><td colspan="1" rowspan="1">4032</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-6</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="1">3424</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-7</td><td colspan="1" rowspan="1">1x106</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">32443</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-13</td><td colspan="1" rowspan="1">1x106</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">0092</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-14</td><td colspan="1" rowspan="1">1x106</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">Isolates with more than one genetic marker target</td></tr><tr><td colspan="1" rowspan="1">GR-04/KP-69</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2/VIM</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">B108A</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">33.9, 36.6</td></tr><tr><td colspan="1" rowspan="1">KP-OMA3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.5, 35.4</td></tr><tr><td colspan="1" rowspan="1">1300920</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">33.3, 36.1</td></tr><tr><td colspan="1" rowspan="1">MSH2014-69</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM/OXA-181</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">33.4, 33.2</td></tr><tr><td colspan="1" rowspan="1">C10192-DISCS</td><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">NDM/OXA-181</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">34.4, 36.1</td></tr></table>

aThese organisms were not tested as bacterial isolates.

Under the conditions of this study, 69 of 72 carbapenemase-producing bacterial strains were detected with the Xpert Carba-R Assay. Three carbapenemase-producing bacterial strains (IMP-7, IMP-13, and IMP-14) were not detected with the Xpert Carba-R Assay even at $1 0 ^ { 6 } \mathrm { C F U / m L }$ . Of the 219 Xpert Carba-R Assay tests, excluding controls, one test $( 0 . 5 \% )$ was reported as an indeterminate result (1 ERROR). The test was successfully repeated. All 36 external positive and negative external control runs provided valid GeneXpert results. For a summary of these results, please refer to Table 6 below.

Table 6. Summary of Variants Detected by Wet Testing or Predicted to be Detected Based on In Silico Analysis.   

<table><tr><td rowspan=2 colspan=1>Marker(or TraditionalSubgroup)</td><td rowspan=1 colspan=3>Wet testing</td><td rowspan=2 colspan=1>Not tested but predicted to bedetected based onin silico analysis</td></tr><tr><td rowspan=1 colspan=1>No. ofSampleswith Target</td><td rowspan=1 colspan=1>Type(s)Detected</td><td rowspan=1 colspan=1>Type(s) notDetected</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>KPC-2, 3, 4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KPC-5, 6, 7, 8, 9, 10, 11, 12, 13,14, 15, 16</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>NDM-1, 2, 4, 5</td><td rowspan=1 colspan=1>cYanmaGeaYellowb</td><td rowspan=1 colspan=1>NDM-3, 6, 7, 8, 9</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>VIM-1, 2, 4, 10,19</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>VIM-5, 6, 7, 8, 9, 11, 12, 13, 14,15, 16, 17, 18, 20, 23, 24, 25, 26,27, 28, 29, 30, 31, 32, 33, 34, 35,36, 37, 38</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>OXA-48,OXA-181(OXA-48 variant)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-162, 163, 204, 232, 244,245, 247</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>IMP-1 (9 strains),IMP-2, 4, 6, 10,11</td><td rowspan=1 colspan=1>IMP-7b, 13c,14b</td><td rowspan=1 colspan=1>IMP-3, 8, 9, 13°, 19, 20, 21, 22, 24,25, 27, 28, 30, 31, 33, 37, 40, 42</td></tr></table>

aIMP-4 gene (Enterobacter aerogenes) was detected at much higher CFU/mL $\overline { { ( 5 \thinspace x \thinspace I 0 ^ { 4 } \thinspace C F U / m L ) } }$ than other IMP variants detected by the assay. $^ { b } I M P { - } 7$ and IMP-14 genes (Pseudomonas aeruginosa) were not detected by the assay and were not predicted to be detected by in silico analysis (Limitations in package insert). cIMP-13 gene (Klebsiella pneumoniae): Although predicted to be detected by in silico analysis, the IMP-13 gene was not detected by the assay (Limitation in package insert).

# e. Analytical specificity:

Using whole organisms seeded into perirectal matrix, the analytical specificity of the Xpert Carba-R Assay was evaluated with a panel of 62 well-characterized carbapenem-susceptible bacteria or bacteria with carbapenem non-susceptibility due to genes or mechanisms other than the Xpert Carba-R target genes. Panel members were resuspended in negative perirectal matrix. A set of 31 commensal/enteric microorganisms was also evaluated in the study, as well as human genomic DNA. Bacteria were seeded into negative matrix at ${ \geq } \mathrm { { i } } 0 ^ { 6 } \mathrm { { C F U / m L } }$ (Peptostreptococcus anaerobius tested at $5 \mathrm { ~ x ~ } 1 0 ^ { 5 } \ : \mathrm { \bar { C } F U / m L ) }$ in triplicate and tested with the Xpert Carba-R Assay. Viruses were tested $\mathrm { a t } > 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ or greater than $2 . 5 \times \mathrm { ~ i 0 ~ } ^ { 7 }$ RNA copies/mL. Human cells were tested at $\mathrm { 1 \ x 1 \ 0 ^ { 5 } \ c e l l s / m L }$ . For the Analytical Specificity Study panel, please refer to Table 7 and Table 8 below.

Table 7. Analytical Specificity Panel with Organisms Having a Resistance Mechanism other than Targets of the Xpert Carba-R Assay   

<table><tr><td>Organism</td><td>Strain ID</td><td>Confirmed Resistance Mechanism(s)a</td><td>Carbapenem Susceptibility (S/I/R)b</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">CTX-M (-1, type 15 like,TEM)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13465</td><td colspan="1" rowspan="1">CTX-M (25)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate (810)</td><td colspan="1" rowspan="1">OmpC/OmpF deficient,TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Clinical isolate (1698)</td><td colspan="1" rowspan="1">TEM (WT+164S)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate (5557)</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">kpn5</td><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">kpn12</td><td colspan="1" rowspan="1">TEM; SHV; CTX-M</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">eco1</td><td colspan="1" rowspan="1">TEM; CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Clinical isolate (eco2)</td><td colspan="1" rowspan="1">CTX-M (2); TEM; OXA-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Clinical isolate (cor1)</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Clinical isolate (hpp21)</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">fer29</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">gut25</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">Clinical isolate (3209)</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Shigella flexnerii</td><td colspan="1" rowspan="1">Clinical isolate (3331)</td><td colspan="1" rowspan="1">CTX-M (2); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">PA_3</td><td colspan="1" rowspan="1">AmpC; CTX-M-15; TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate (32189)</td><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate (32443)</td><td colspan="1" rowspan="1">CTX-M (1, -type 15 like);SHV</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">32598</td><td colspan="1" rowspan="1">CTX-M (-1, -type 15 like);SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">33560</td><td colspan="1" rowspan="1">CTX-M (15); SHV-11;TEM-1</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">33603</td><td colspan="1" rowspan="1">SHV-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate (33617)</td><td colspan="1" rowspan="1">SHV-27</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Clinical isolate (33643)</td><td colspan="1" rowspan="1">SHV (-5, -55); TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34430</td><td colspan="1" rowspan="1">SHV; TEM; CTX-M-15</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34680</td><td colspan="1" rowspan="1">TEM; CTX-M-2</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">34732</td><td colspan="1" rowspan="1">CTX-M (15); SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">PA_174</td><td colspan="1" rowspan="1">GX-/Culture+; SHV; TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Clinical isolate(STU645)</td><td colspan="1" rowspan="1">SHV (WT+238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">STU 669</td><td colspan="1" rowspan="1">SHV (WT+238S+240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">C3015</td><td colspan="1" rowspan="1">AmpC (CMY II); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">RI_100</td><td colspan="1" rowspan="1">AmpC (DHA); SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Strain ID</td><td colspan="1" rowspan="2">Confirmed ResistanceMechanism(s)a</td><td colspan="3" rowspan="1">CarbapenemSusceptibility (S//)</td></tr><tr><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B4A</td><td colspan="1" rowspan="1">SHV (WT + 238S +240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B13A</td><td colspan="1" rowspan="1">SHV (WT + 238S +240K)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">RI_474</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">B71</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">DD82A</td><td colspan="1" rowspan="1">SHV (WT + 238S +240K</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B100</td><td colspan="1" rowspan="1">CTX-M (-1, type-15 like);SHV (WT+238S); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">135B</td><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">B157</td><td colspan="1" rowspan="1">SHV; TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">T2914280</td><td colspan="1" rowspan="1">CTX-M (-1, -15); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">DD188</td><td colspan="1" rowspan="1">TEM (104K + 164S)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">DD189</td><td colspan="1" rowspan="1">AmpC (ACT/MIR)</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">B198B</td><td colspan="1" rowspan="1">CTX-M (-1, type -15 like);TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">T3019989-1</td><td colspan="1" rowspan="1">CTXM (-1, type-15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">T3019989-2</td><td colspan="1" rowspan="1">CTX-M (-1, type-15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ENC-THAI14</td><td colspan="1" rowspan="1">VEB-1, TEM</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CB154006</td><td colspan="1" rowspan="1">CTX-M (9); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">S35766</td><td colspan="1" rowspan="1">AmpC(ACT/MIR)</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">X1856910</td><td colspan="1" rowspan="1">AmpC (ACT/MIR); TEM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">W3758164</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SV: TEM.</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">X2135758</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">W3809535</td><td colspan="1" rowspan="1">CTX-M (-1, -15 like);SHV</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC0064</td><td colspan="1" rowspan="1">SPM</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0099</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0121</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0122</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0123</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0124</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0130</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC0131</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Strain ID</td><td colspan="1" rowspan="2">Confirmed ResistanceMechanism(s)a</td><td colspan="3" rowspan="1">CarbapenemSusceptibility (S/I/R)</td></tr><tr><td colspan="1" rowspan="1">ETPb</td><td colspan="1" rowspan="1">IMPb</td><td colspan="1" rowspan="1">MEMb</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaegroup</td><td colspan="1" rowspan="1">CDC0132</td><td colspan="1" rowspan="1">IMI</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">CDC0164</td><td colspan="1" rowspan="1">IMI</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr></table>

a Presence of these markers of resistance was determined by individual PCR assays, DNA sequence analysis, or by other research-based methods. ${ } ^ { b } S / I / \dot { R } =$ Susceptible/Intermediate/Resistant; $E T P = E$ rtapenem, $I M P =$ Imipenem, $M E M = .$ Meropenem

Table 8. Panel of Commensal and Other Enteric Microorganisms Tested in this Study and Human DNA   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 29212</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 700603</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">ATCC 9689</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 700621</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 9756</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC BAA-747</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 33128</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 49948</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 51331</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27028</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 49809</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 49037</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">CCUG 29780 / ATCC 12401</td></tr><tr><td colspan="1" rowspan="1">Bifidiobacterium adolescentis</td><td colspan="1" rowspan="1">ATCC 15703</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 51697</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">CCUG 34787</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">CCUG 418</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">CCUG 33629</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">CCUG 17874</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">CCUG 33548</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">CCUG 6325</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">CCUG 43594/ATCC 33560</td></tr><tr><td colspan="2" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B Type 7A/NY</td><td colspan="1" rowspan="1">MRVP/ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71/NY</td><td colspan="1" rowspan="1">MRVP/ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">Norovirus GII</td><td colspan="1" rowspan="1">Clinical Sample -Cepheid</td></tr><tr><td colspan="2" rowspan="1">Human DNA</td></tr><tr><td colspan="1" rowspan="1">Bladder Cell Carcinoma (hgDNA)</td><td colspan="1" rowspan="1">ATCC HTB-4</td></tr></table>

Of the 94 potentially cross-reactive organisms and nucleic acid tested, including organisms exhibiting antibiotic resistance mechanisms other than production of KPC, NDM, VIM, IMP and OXA-48, none were detected with the Xpert Carba-R Assay. Of the 283 tests in perirectal matrix, including controls, one $( 0 . 4 \% )$ run was indeterminate (1 INVALID). The indeterminate run was successfully repeated and was reported as NOT DETECTED for all five targets (KPC, NDM, VIM, IMP and OXA48) as expected. All external positive and negative controls were correctly reported as expected.

All the data were collected on the GeneXpert Dx (GX-IV) instrument using the GeneXpert Dx software version 4.4a.

# f. Assay cut-off:

For IMP, VIM, NDM, KPC, and OXA-48 gene targets, the valid cycle threshold $\mathrm { ( C t ) }$ range was 3.0 to 38.0. For the SPC, the valid Ct range was set from 3.0 to 40.0. A Ct value outside the valid range is reported as NOT DETECTED. The Ct cut-offs are included as automatic calculations in the assay definition file (ADF) provided with the Xpert Carba-R Assay. Assay cut-off values have not changed from those described in K152614.

# g. Interfering substances:

A study was conducted to assess the inhibitory effects of substances potentially encountered in perirectal swab specimens on the performance of the Xpert Carba-R Assay. Twenty-four substances were evaluated at “worst case scenario” concentrations. Eight replicate positive samples were tested per substance. Negative samples consisted of pooled negative perirectal swab matrix with/without the interfering substance (not seeded with carbapenemase-producing organisms). Controls consisted of positive and negative samples with no interfering substances added. Positive samples were prepared from a mix of five carbapenemase-producing organisms harboring KPC, NDM, VIM, IMP and OXA-48 gene sequences seeded into pooled negative perirectal swab matrix to give concentrations that were 3X analytical LoD (Table 9). The list of potentially interfering substances tested is shown in Table 10.

Table 9. Organisms used to Prepare the Mixed Positive Sample   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target Gene</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>VIM</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>IMP-1</td></tr></table>

Table 10. Potentially Interfering Substances Tested   

<table><tr><td colspan="1" rowspan="1">Substance ID</td><td colspan="1" rowspan="1">Substance/Class</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">Aleve (1)</td><td colspan="1" rowspan="1">Non-steroidal anti-inflammatorymedication</td><td colspan="1" rowspan="1">Naproxen</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate (2)</td><td colspan="1" rowspan="1">Imaging compound</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.25% and 0.1%w/v</td></tr><tr><td colspan="1" rowspan="1">Antimicrobial (3)</td><td colspan="1" rowspan="1">Antibiotic (oral)</td><td colspan="1" rowspan="1">Cephalexin</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Antimicrobial(4)</td><td colspan="1" rowspan="1">Antibiotic (oral)</td><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Condom (5)</td><td colspan="1" rowspan="1">Condom withspermicidal lubricant</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">1 condoma</td></tr><tr><td colspan="1" rowspan="1">Cortizone (6)</td><td colspan="1" rowspan="1">Creams/ointment/suppositories</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">ExLax (7)</td><td colspan="1" rowspan="1">Laxative</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Fecal Fat (8)</td><td colspan="1" rowspan="1">Lipids</td><td colspan="1" rowspan="1">Stearic acid/Palmiticacid/Cholesterol</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Imodium (9)</td><td colspan="1" rowspan="1">Anti-diarrhealmedication</td><td colspan="1" rowspan="1">Loperamidehydrochloride/bismuthsubsalicylate</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Kaopectate (10)</td><td colspan="1" rowspan="1">Anti-diarrhealmedication</td><td colspan="1" rowspan="1">Loperamidehydrochloride/bismuthsubsalicylate</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">K-Y Jelly (11)</td><td colspan="1" rowspan="1">Topical cream</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Milk of Magnesia(12)</td><td colspan="1" rowspan="1">Antacids</td><td colspan="1" rowspan="1">Calciumcarbonate/aluminumhydroxide/magnesiumhydroxide/simethicone</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil-enema(13)</td><td colspan="1" rowspan="1">Enemas</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Neosporin (14)</td><td colspan="1" rowspan="1">Antibiotic (topical)</td><td colspan="1" rowspan="1">Polymixin B/Neomycin/Bacitracin</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Nystatin (15)</td><td colspan="1" rowspan="1">Anti-fungal/anti-itch Vaginal</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Pepcid (16)</td><td colspan="1" rowspan="1">Antacid</td><td colspan="1" rowspan="1">Famotidine</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol (17)</td><td colspan="1" rowspan="1">Anti-diarrhealmedication</td><td colspan="1" rowspan="1">Loperamidehydrochloride/bismuthsubsalicylate</td><td colspan="1" rowspan="1">0.25%, 0.1%,0.05%, 0.025%,0.01% w/v</td></tr><tr><td colspan="1" rowspan="1">Petroleum jelly(18)</td><td colspan="1" rowspan="1">Topical cream</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Preparation H (19)</td><td colspan="1" rowspan="1">Anti-hemorrhoidcreams/ointments</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Prilosec (20)</td><td colspan="1" rowspan="1">Acid reducer;antacid</td><td colspan="1" rowspan="1">Oemprazole</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Saline-enema (21)</td><td colspan="1" rowspan="1">Enemas</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Tagamet (22)</td><td colspan="1" rowspan="1">Antacid</td><td colspan="1" rowspan="1">Cimetidine</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Vagisil (23)</td><td colspan="1" rowspan="1">Anti-fungal/anti-itch Vaginal</td><td colspan="1" rowspan="1">Benzocaine, resorcinol</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Wet Ones (24)</td><td colspan="1" rowspan="1">Moist Towelettes</td><td colspan="1" rowspan="1">Benzalkoniumchloride, ethanol</td><td colspan="1" rowspan="1">1 pieceb</td></tr></table>

aOne condom added to 40 mL swab matrix. bOne piece (5 inch $x \ 7 – I / 2$ inch) added to 40 mL swab matrix.

Results showed that the majority of the substances tested at their potentially highest concentration relative to the controls did not interfere with the Xpert Carba-R Assay. Under the conditions of this study, 22 of the 24 potentially interfering substances tested at their potentially highest concentration $( 0 . 2 5 \% \mathrm { w / v } )$ did not interfere with the Xpert Carba-R Assay. Inhibitory effects were observed with two substances, barium sulfate and Pepto-Bismol, when tested at $0 . 2 5 \%$ w/v in perirectal swab matrix. Interference with the Xpert Carba-R Assay may be observed with barium sulfate at $>$ $0 . 1 \% \mathrm { w / v }$ and Pepto-Bismol at $> 0 . 0 2 5 \% \mathrm { ~ W ~ }$ /v. Of the 468 tests (not including the external controls), 22 tests provided indeterminate GeneXpert results (3 ERROR and 19 INVALID). All 22 indeterminate GeneXpert results were successfully repeated.

Each of the external controls gave the expected GeneXpert test result. All the data were collected using the GeneXpert Dx software version 4.4a on the GeneXpert Dx (GX-IV and GX-XVI) instruments.

# Competitive Interference

To evaluate the potential competitive inhibitory effect of multiple carbapenemaseproducing organisms on the performance of the Xpert Carba-R Assay in perirectal swab specimens, a study was performed by testing various combinations of carbapenemase-producing organisms seeded at high and low concentrations into natural matrix. High concentrations of organisms corresponded to ${ \sim } 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and low concentrations corresponded to 2X LoD. An inhibitory effect was observed for two out of five targets (IMP and NDM) when a low concentration of each target was present in combination with a high concentration of another assay target. No inhibitory effect was observed for the VIM, KPC, and OXA-48 targets. Table 11 reports the number of target replicates that were detected in the competitive interference study with perirectal swab matrix.

Table 11. Number of Correct Results for Combinations in the Competitive Interference Study with the Xpert Carba-R Assay using a Perirectal Swab Matrix   

<table><tr><td rowspan=2 colspan=1>SampleNmber</td><td rowspan=2 colspan=1>Combination</td><td rowspan=1 colspan=5>Number of DETECTED results/Number of replicates</td><td rowspan=2 colspan=1>Inhibited(Yes orNb)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>OXA</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High KPC/High NDM/Low VIM</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>High KPC/High NDM/Low OXA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>High KPC/High NDM/Low IMP</td><td rowspan=1 colspan=1>1/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Yes/IMP</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>High VIM/High OXA/Low KPC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>High VIM/High OXA/Low NDM</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>High VIM/High OXA/Low IMP</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>High IMP/Low KPC</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>High IMP/Low NDM</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Yes/NDM</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>High IMP/Low VIM</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>High IMP/Low OXA</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>High OXA/Low VIM</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>High VIM/Low OXA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>Nb</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>High KPC/Low NDM</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>NA</td></tr></table>

aTarget was considered inhibited if 6 or fewer replicates were detected per sample.

For those targets not detected at 2X LoD (6 or fewer replicates detected), an additional study was performed where the low target concentration was increased to 4X LoD to evaluate the competitive inhibitory effect. No inhibitory effect was observed for the two targets (IMP and NDM) at 4X LoD in the presence of high concentrations of the other targets for the Xpert Carba-R Assay.

# h. Carry-over:

The purpose of the Carry-Over Study was to determine the carry-over rate of contamination in negative samples due to the nucleic acid extraction and amplification of high positive samples in the GeneXpert cartridge. In this study, a negative sample was tested in a GeneXpert module immediately following the testing of a high titer positive sample in the same GeneXpert module. The high positive sample was composed of inactivated $E .$ . coli cells containing a plasmid with all five Xpert Carba-R target analyte genes (KPC, NDM, VIM, IMP and OXA-48 targets) diluted in Sample Reagent with perirectal swab matrix to a concentration of $1 \mathrm { ~ x ~ } \mathrm { { 1 0 ^ { 6 } } \mathrm { { C F U / m L } } }$ . The testing pattern was repeated 25 times on two GeneXpert modules for a total of 102 tests (25 high positive samples and 26 negative samples per module). For each day of the study, an external negative control and two types of experimental positive controls were tested as described previously. All 52 negative samples reported NOT DETECTED results for all five Xpert Carba-R Assay targets as expected. All positive samples correctly reported all Xpert Carba-R targets as DETECTED. Study results indicated no evidence of sample or amplicon carry-over contamination in the GeneXpert modules.

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity:

Prospective Study #1—MultiCenter Study Performance characteristics of the Xpert Carba-R Assay were evaluated in a multicenter study using perirectal swab specimens. The positive percent agreement (PPA) and negative percent agreement (NPA) of the Xpert Carba-R Assay were evaluated relative to a Composite Reference Method. Due to limited historical information on the detection of carbapenem non-susceptible organisms harboring assay targets by this method, which consisted of Reference Culture $^ +$ PCR and Sequencing of the amplification product, performance was reported as PPA and NPA instead of sensitivity and specificity. Five geographically diverse sites were selected (across the United States), and prospectively collected paired perirectal swab specimens were collected from subjects who were hospitalized or in a long-term care facility. Highly soiled perirectal swabs were excluded from the study. Contrived specimens were included in the study due to the expected low prevalence for some Xpert Carba-R

Assay target genes.

For the Reference Culture, the second swab specimen was inoculated into $1 1 ~ \mathrm { m L }$ MacConkey enrichment broth containing a $1 0 ~ \mu \ g$ meropenem disk and incubated overnight at $3 5 \mathrm { { ^ \circ C } }$ . An aliquot of MacConkey broth culture was spread onto a MacConkey agar plate, and a $1 0 ~ \mu \ g$ meropenem disk was placed on the plate. After an overnight incubation at $3 5 \mathrm { { ^ \circ C } }$ , the zone of clearing was measured. If growth was observed within a $2 8 \mathrm { m m }$ zone (including the meropenem disk), species identification of the organisms was performed after subculture of colonies to sheep blood agar. Organism susceptibility status (susceptible, intermediate or resistant) to meropenem, ertapenem and imipenem was determined using CLSI standard test methods (M07- A9) and the interpretive criteria found in the FDA drug label and CLSI M100-S24. Carbapenem non-susceptible organisms were then subcultured to sheep blood agar (with a meropenem disk placed between the $1 ^ { \mathrm { s t } }$ and $2 ^ { \mathrm { n d } }$ streak) and incubated overnight at $3 5 \mathrm { { ^ \circ C } }$ . Three to five well-isolated colonies of the same morphotype, as described in CLSI M07-A9, were collected and sent for sequencing.

DNA from the carbapenem non-susceptible isolates was purified, quantified, and amplified using primers specific to all 5 target genes that were validated and amplify a larger region than the Xpert Carba-R primers. The production of the appropriate sized amplification products was confirmed on Agilent 2100 Bioanalyzer. If no bands were shown on the Bioanalyzer for any of the five target genes, the isolate was not sent for sequence analysis and the Reference Method result was considered negative for the five target genes. PCR and sequencing was not performed for specimens where there was no growth in the 28 mm zone, if the results of antimicrobial susceptibility testing identified a susceptible isolate, or if organisms had intrinsic resistance to all the carbapenems tested (e.g. Stenotrophomonas maltophilia).

A total of 963 prospective specimens were initially enrolled in the clinical study, of which 947 were eligible for inclusion. From the 947 eligible specimens, 924 were included in the final dataset. The ineligible/excluded specimens are described below:

(10) excluded strain type based on testing design [Stenotrophomonas   
maltophilia (8) and Pseudomonas putida (1), Pseudomonas stutzeri (1)]   
(10) Ineligible, soiled   
(6) invalid GX control   
(4) shipping delay   
(2) Ineligible, duplicate   
(2) patients had colostomy   
(1) Ineligible, consent   
(1) insufficient number of swabs collected   
(1) GX indeterminate not retested   
(1) Ineligible, sample type   
(1) GX 2 timed indeterminate

Performance of the Xpert Carba-R Assay was assessed separately for each type of resistance marker target and compared to the Reference Culture $^ +$ Sequencing result. With the perirectal swab specimens, $9 8 . 9 \%$ (914/924) of specimens yielded a result on the first run. Four (4) INVALID, (5) ERROR, and (1) NO RESULT were observed in the first run, but yielded a valid result upon repeat testing.

In addition to the prospective study, well-characterized isolates carrying each assay target were tested in a contrived study. Strains were re-suspended in negative perirectal matrix before testing with the Xpert Carba-R Assay at 1X, 3X, and 10X LoD concentrations. The following numbers of unique isolates were evaluated in the study spanning multiple gram negative species (Table 12 and Table 13):

Table 12. Number of Unique Bacterial Strains Tested by Target and Concentration Level   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>3X LoDa</td><td rowspan=1 colspan=1>10X LoDa</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>Negativesb</td><td rowspan=1 colspan=3>15</td></tr></table>

aStrains tested at 3X and 10X LoD were chosen from the unique strains tested at 1X LoD. bThirty negative samples were part of the study panel (also includes replicates tested).

Table. 13. Various Species Tested in the Contrived Study by Target   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Various Organismsincluded in the Study</td></tr><tr><td colspan="1" rowspan="8">VIM</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="5">IMP</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas stutzeri</td></tr><tr><td colspan="1" rowspan="2">NDM</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter spp.</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">Empedobacter brevis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td></tr><tr><td colspan="1" rowspan="6">KPC</td><td colspan="1" rowspan="1">Citrobacter koseri</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="4">OXA-48</td><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="8">No target</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td></tr></table>

Discordant samples were tested with an alternate PCR method when the Xpert CarbaR Assay result was positive and the Reference Method result was negative. An aliquot of the MacConkey broth was used to extract DNA and amplify any potential targets by PCR before sending for DNA sequencing. These results were not used to change the original performance data (See results in the footnotes to Table 14).

Study results of the Xpert Carba-R Assay compared to the Reference Method are shown in Table 14 stratified by individual target for prospective and contrived studies.

Table 14. Clinical Performance Data for the Xpert Carba-R Assay vs. Reference Culture $^ +$ Sequencing   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA%(5 CC)</td><td rowspan=1 colspan=1>NPA% 95 C)</td></tr><tr><td rowspan=5 colspan=1>Prospective</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.0%(99.6-100.0)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100.0%(9.6-100.0)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>923</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(20.7-100.0)</td><td rowspan=1 colspan=1>100.0%(99.6-100.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>918</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(34.2-100.0)</td><td rowspan=1 colspan=1>99.6%(98.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>924</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>922</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(20.7-00.0)</td><td rowspan=1 colspan=1>99.9%(99.4-100.0)</td></tr><tr><td rowspan=5 colspan=1>Contrived</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(95.4-100.0)</td><td rowspan=1 colspan=1>100.0%(98.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>350</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(95.5100.0)</td><td rowspan=1 colspan=1>100.0%(98.9-100.)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(95.4-100.0)</td><td rowspan=1 colspan=1>100.0%(98.100.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(95.4-100.0)</td><td rowspan=1 colspan=1>100.0%(98.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(95.4-100.0)</td><td rowspan=1 colspan=1>100.0%(98.9-100.0)</td></tr></table>

a0 of the 4 FPs was determined to be TP after discordant analysis. $^ b 0$ of the 1FPs was determined to be TP after discordant analysis.

The Xpert Carba-R Assay performance by specific organism group is shown in Table 15 for the prospective study.

Table 15. Xpert Carba-R Results (by Organism and Target) vs Reference Culture $^ +$ Sequencing (Prospective Study)   

<table><tr><td colspan="8" rowspan="1">PROSPECTIVE STUDY</td></tr><tr><td colspan="1" rowspan="1">Organisma</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FP</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">PPA(95% CI)</td><td colspan="1" rowspan="1">NPA(95% CI)</td></tr><tr><td colspan="1" rowspan="5">E. aerogenes(n=1)</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(20.7-100)</td></tr><tr><td colspan="1" rowspan="2">K. pneumoniae(n=3)b</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">100%(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">100%(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(20.7-100)</td><td colspan="1" rowspan="1">100%(34.2-100)</td></tr><tr><td colspan="1" rowspan="5">P. aeruginosa(n=28)</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(87.9-100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(87.9-100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(87.9-100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">96.4%(82.3-99.4)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(87.9-100)</td></tr></table>

aN refers to the total numbers of specimens where an organism was identified. This number includes samples that were PCR positive and PCR negative for the presence of assay targets. Acinetobacter baumannii (1) was recovered but did not contain target sequences by the Reference Method or by the Xpert Carba-R Assay. ${ } ^ { b } I n$ (1) Klebsiella pneumoniae isolate, multiple targets were detected by the Reference Method— NDM/OXA-48; Both targets were also detected by the Xpert Carba-R Assay.

External controls for the Xpert Carba-R Assay consisted of one negative sample, one sample positive for all (5) targets of the assay, and five different positive controls each containing a single target of the assay. The negative control and five-target positive controls were run on each day that study samples were tested, along with two of the single-target positive controls (on a rotating basis). Study samples were not run until correct results were obtained for each of the four controls. External control data was compiled across all sites and overall QC results were acceptable.

# Prospective Study #2—Single Site Study (high prevalence site)

The use of rectal swab specimens was previously cleared (K160901). Since only a few organisms with assay targets were recovered in the prospective study with perirectal swab specimens, a second study was conducted to examine the detection of assay targets from both rectal and perirectal swab specimens from the same patient. This study design was considered feasible as similar LoDs for assay targets were reported for organisms tested in rectal and perirectal swab matrix. To demonstrate equivalence between the perirectal swab specimens and rectal swab specimens, fresh prospectively collected rectal and perirectal swab specimens were enrolled from the same patient at one site. Paired swab sets were used to collect specimens from each patient. One paired swab set was used to collect the perirectal swab specimen, and a second paired swab set was used to collect the rectal swab specimen. The perirectal swab specimen was collected first followed by the rectal swab specimen. One swab from each set was used for Xpert Carba-R testing. The second swab from each set was saved for culture and susceptibility testing when either (or both) the perirectal or rectal swab specimen was positive for one or more targets by the Xpert Carba-R

Assay. No culture was performed if perirectal and rectal swab specimens were both negative by the Xpert Carba-R Assay. If high agreement between the two specimen types was not observed, the study protocol allowed for additional testing of the specimens with discordant results and true positive specimens to show equivalent performance of the Xpert Carba-R Assay with both specimen types in the context of the Reference Culture $^ +$ Sequencing Method. Because the PPA and NPA performance was determined acceptable, this conditional testing was not needed to help evaluate device performance.

A total of 207 specimens were enrolled in the clinical study. Of the 207 eligible specimens, 201 specimens were included in the final dataset. Six swab specimens (4- perirectal and 2-rectal swab specimens) were excluded due to indeterminate results from the Xpert Carba-R Assay. The performance (PPA and NPA) of the Xpert CarbaR Assay using perirectal swab specimens was determined relative to the results of the Xpert Carba-R Assay using rectal swab specimens from the same patient (Table 16). Relative to the Xpert Carba-R Assay rectal swab specimen result, the perirectal swab specimens demonstrated a PPA and NPA of $9 4 . 7 \%$ $9 5 \% C \mathrm { I }$ : 75.4-99.1) and $9 7 . 8 \%$ ( $9 5 \% C \mathrm { I }$ : 94.5-99.1), respectively.

Table 16. Xpert Carba-R Assay-Perirectal Swab Specimens vs Rectal Swab Specimens   

<table><tr><td rowspan=1 colspan=5>Xpert Carba-R AssayRectal Swab Specimens</td></tr><tr><td rowspan=4 colspan=1>Xpert Carba-R Assay Perirectal SwabSpecimens</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>18a</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>179</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>201</td></tr><tr><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=2>94.7% (95%CI: 75.4-99.1)</td></tr><tr><td rowspan=1 colspan=3>NPA</td><td rowspan=1 colspan=2>97.8% (95%CI: 94.5-99.1)</td></tr></table>

a For one specimen, Xpert testing on the rectal swab was positive for KPC and OXA-48 but the perirectal swab was positive for OXA-48 only. The specimen was culture negative using both rectal and perirectal swabs. Sequence results from the MacConkey broths were negative with the perirectal swab and OXA-48 positive for the rectal swab.

$^ { b } 2$ of 4 were culture positive with both rectal and perirectal swabs and sequence results from isolates were both OXA-48 positive; 1 of 4 was culture negative and sequence negative for both rectal and perirectal swabs; 1 of 4 was culture positive but sequencing was not available due to: 1) the isolate not saved (rectal swab discrepant testing) and 2) results of culture interpreted as carbapenem susceptible, and per protocol, sequencing was not performed (perirectal swab discrepant testing).

cCulture negative for both rectal and perirectal swabs; sequence results from MacConkey broths were both OXA-48 Positive.

b. Clinical specificity: See comments in 3a above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

In the multi-site prospective clinical study, $0 . 3 \%$ of the 924 perirectal swab specimens (3/924) contained a carbapenem non-susceptible organism with at least one of the assay gene targets (IMP, VIM, NDM, KPC, OXA-48) by the Reference Method. By the Xpert Carba-R Assay, $0 . 9 \%$ of the 924 perirectal swab specimens (8/924) showed at least one target detected by the Xpert Carba-R Assay.

N. Instrument Name: GeneXpert Instrument Systems

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X_____ or No _

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No ___ _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification: Similar to previously cleared system.

4. Specimen Sampling and Handling:

Specific instructions should be followed for the collection of perirectal swab specimens. The user refers to a Reference Diagram supplied by Cepheid to determine the acceptability of swab specimens. Any specimens that are highly soiled per the Reference

Diagram must be excluded from analysis. Swabs are then placed in the appropriate collection device. One swab is added to Sample Reagent for Xpert Carba-R Assay testing. An aliquot of sample $( 1 . 7 \mathrm { m L } )$ is then transferred to the sample chamber of the disposable, single-use fluidic cartridge (Xpert Carba-R cartridge). The user initiates a test from the system user interface and places the cartridge into the GeneXpert Instrument System. Additional sample preparation, amplification, and real-time detection are all fully-automated and completed by the instrument system.

# 5. Calibration:

The Xpert Check kit is used by the customer or by Cepheid personnel to perform the calibration check of the instrument. The Xpert Check is not provided with the instrument since the instrument is originally calibrated by Cepheid. A calibration check is recommended on an annual basis. In the GeneXpert Operator’s Manual (Calibration Section), the user is instructed to contact Cepheid Technical Support for information about calibration.

6. Quality Control:

Quality control is addressed for each separately cleared assay to be run on the instrument.   
See section M1(c) for information on internal and external controls.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.