# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K173330   
B. Purpose for Submission: To obtain a Substantially Equivalent determination for the PanNAT Shiga Toxin-producing E. coli (STEC) Test   
C. Measurand: Conserved regions of the Shiga toxin genes $( s t x 1 , s t x 2 )$ and the $E$ . coli O157 O-antigen gene cluster $( f c l )$   
D. Type of Test: Qualitative real-time Polymerase Chain Reaction (PCR)   
E. Applicant: Micronics, Inc.   
F. Proprietary and Established Names: PanNAT STEC Test   
G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990: Gastrointestinal microorganism multiplex nucleic acid-based assay

2. Classification: Class II (Special Controls)

3. Product code: PCI: Gastrointestinal Bacterial Panel Multiplex Nucleic Acid-Based Assay System PCH: Gastrointestinal Pathogen Panel Multiplex Nucleic Acid-Based Assay System

# OOI: Real Time Nucleic Acid Amplification System

4. Panel:

83 - Microbiology

# H. Intended Use:

1. Intended use(s):

The Micronics PanNAT STEC Test is a qualitative, in vitro nucleic acid amplificationbased test intended for the simultaneous detection and identification of the stx1 and $s t x 2$ Shiga toxin genes and the O-antigen gene cluster of $E$ . coli O157 (fcl). Testing is performed in a unitized cartridge on the PanNAT System on soft to diarrheal unpreserved or Cary-Blair preserved stool specimens from individuals with signs and symptoms of gastrointestinal infection. The PanNAT STEC Test is intended for use in conjunction with clinical presentation, laboratory findings and epidemiological risk factors, as an aid in detection of specific Shiga-toxin expressing strains of $E$ . coli $( ^ { \circ } \mathrm { S T E C } ^ { \circ } )$ from patients with diarrhea.

The results of this test should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Positive PanNAT STEC Test results do not rule out the potential for coinfection with other pathogens that are not detected by this device and may not be indicative of the sole or definitive cause of patient illness. Negative PanNAT STEC Test results in the context of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

The PanNAT STEC Test may exhibit reduced sensitivity and/or failure detection failure with stx1d, stx2b, stx2f and stx2g. These subtypes of Shiga toxin are infrequently isolated from human specimens.

4. Special instrument requirements:

The PanNAT STEC Test is indicated for use with the PanNAT System.

# I. Device Description:

The PanNAT STEC Test is a cartridge-based real-time PCR-based assay for the detection of the Shiga toxin genes $s t x 1$ and $s t x 2$ and the Escherichia coli O157-specific gene cluster, fcl. The assay is performed on the PanNAT System which comprises an instrument and software for automated sample processing, nucleic acid amplification and detection and result interpretation. The PanNAT STEC Test is for use on preserved (Cary-Blair Transport Medium) or unpreserved soft or diarrheal stool specimens from subjects with signs and symptoms of gastrointestinal infection.

The PanNAT STEC Test kit contains components to process and analyze up to 25 specimens and includes PanNAT Sample Transfer Packs (comprised of a swab, Sample Buffer Tube and Adaptor Cap) and PanNAT STEC Test Cartridges.

To perform the PanNAT STEC Test, a swab is used to transfer part of the stool specimen to a Sample Buffer Tube which is then fitted with an Adaptor Cap and attached to the inlet port of a PanNAT STEC Test Cartridge. The cartridge-tube assembly is then inserted into the PanNAT System for automated processing.

The PanNAT STEC Test cartridge contains all the components necessary to perform the assay and includes separate reaction mixtures for each target analyte $( s t x 1 , s t x 2$ and O157) that are dried in different wells. Each reaction mixture also contains primers and a detector probe for an endogenous human gene that is extracted from the sample and co-amplified with the target analyte (if present). There are three reaction mixtures per target analyte. The first three reaction mixtures (one per target) are rehydrated with the processed sample and analyzed to report the presence or absence of the target nucleic acids. The remaining wells are used as Positive and Negative Controls and receive none of the processed sample but instead are rehydrated with buffer. Each of the control wells must produce the expected results for the results from the test wells to be reported.

Detection of the amplification reactions occurs in real-time using quenched fluorescent probes and results are reported automatically.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD Max Enteric Bacterial Panel   
2. Predicate 510(k) number(s): K140111

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">PanNAT STEC Test(K173330)</td><td colspan="1" rowspan="1">BD MAX EntericBacterial Panel(K140111)</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3990</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Primary Product Code</td><td colspan="1" rowspan="1">PCI</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Micronics PanNATSTEC Test is a qualitative,in vitro nucleic acidamplification-based testintended for thesimultaneous detection andidentification of the stx1and stx2 Shiga toxin genesand the O-antigen genecluster of E.coli O157 (fcl).Testing is performed in aunitized cartridge on thePanNAT System on soft todiarrheal unpreserved orCary-Blair preserved stoolspecimens from individualswith signs and symptomsof gastrointestinalinfection. The PanNATSTEC Test is intended foruse in conjunction withclinical presentation,laboratory findings andepidemiological riskfactors, as an aid indetection of specific Shiga-toxin expressing strains ofE. coli ("STEC") frompatients with diarrhea.The results of this testshould not be used as thesole basis for diagnosis,treatment or other patientmanagement decisions.</td><td colspan="1" rowspan="1">The BD MAX EntericBacterial Panel performedon the BD MAX System isan automated in vitrodiagnostic test for thedirect qualitative detectionand differentiation ofenteric bacterial pathogens.The BD MAX EntericBacterial Panel detectsnucleic acids from:Salmonella spp.Campylobacter spp.(jejuni and coli)Shigella spp. /Enteroinvasive E. coli(EIEC)Shiga toxin 1 (stx1) /Shiga toxin 2 (stx2) genes(found in Shiga toxin-producing E. coli[STEC]) as well asShigella dysenteriae,which can possess aShiga toxin gene (stx)that is identical to thestx1 gene of STEC.Testing is performed onunpreserved soft todiarrheal stool specimensor Cary-Blair preservedstool specimens fromsymptomatic patients withsuspected acute</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive PanNAT STECTest results do not rule outthe potential forcoinfection with otherpathogens that are notdetected by this device andmay not be indicative ofthe sole or definitive causeof patient illness. NegativePanNAT STEC Test resultsin the context of clinicalillness compatible withgastroenteritis may be dueto infection by pathogensthat are not detected by thistest or non-infectiouscauses such as ulcerativecolitis, irritable bowelsyndrome, or Crohn'sdisease.</td><td colspan="1" rowspan="1">gastroenteritis, enteritis orcolitis. The test isperformed directly on thespecimen, utilizing real-time polymerase chainreaction (PCR) for theamplification of spaO, aCampylobacter specific tufgene sequence, ipaH andstx1/stx2. The test utilizesfluorogenic sequence-specific hybridizationprobes for detection of theamplified DNA.This test is intended foruse, in conjunction withclinical presentation,laboratory findings, andepidemiologicalinformation, as an aid inthe differential diagnosis ofSalmonella, Shigella/EIEC,Campylobacter and Shigatoxin-producing E. coli(STEC) infections. Resultsof this test should not beused as the sole basis fordiagnosis, treatment, orother patient managementdecisions. Positive resultsdo not rule out co-infectionwith other organisms thatare not detected by thistest, and may not be thesole or definitive cause ofpatient illness. Negativeresults in the setting ofclinical illness compatiblewith gastroenteritis may bedue to infection bypathogens that are not</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">detected by this test or non-infectious causes such asulcerative colitis, irritablebowel syndrome, orCrohn's disease.</td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">Shiga toxin genes stx1 andstx2</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Soft or diarrheal stool:unpreserved or preservedin Cary-Blair medium</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Format</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">PanNAT STEC Test(K173330)</td><td colspan="1" rowspan="1">BD MAX EntericBacterial Panel(K140111)</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">PanNAT System</td><td colspan="1" rowspan="1">BD MAX</td></tr><tr><td colspan="1" rowspan="1">Test Format</td><td colspan="1" rowspan="1">Integrated cartridge forsample processing, DNAamplification/detection</td><td colspan="1" rowspan="1">Separate consumables forsample processing andamplification/detection</td></tr><tr><td colspan="1" rowspan="1">DNA Extraction</td><td colspan="1" rowspan="1">Silica membranes</td><td colspan="1" rowspan="1">Magnetic affinity beads</td></tr><tr><td colspan="1" rowspan="1">Liquid Movement</td><td colspan="1" rowspan="1">Pneumatic, microfluidic</td><td colspan="1" rowspan="1">Automated pipettor</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Hybridization probes</td><td colspan="1" rowspan="1">Hydrolysis probes</td></tr><tr><td colspan="1" rowspan="1">Process Control</td><td colspan="1" rowspan="1">Endogenous human DNA</td><td colspan="1" rowspan="1">Exogenous</td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">Shiga toxin genes stx1and stx2 (differentiated)E. coli O157 fcl genecluster</td><td colspan="1" rowspan="1">Specific DNA sequencesfrom:Shiga toxin genes stx1and stx2 (notdifferentiated)Salmonella spp.Campylobacter spp.(jejuni and coli)Shigella spp. /Enteroinvasive E. coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(EIEC)</td></tr><tr><td colspan="1" rowspan="1">Positive/Negative Controls</td><td colspan="1" rowspan="1">Integrated within testcartridge</td><td colspan="1" rowspan="1">Performed in separate teststrips/amplification wells</td></tr><tr><td colspan="1" rowspan="1">Batch Size</td><td colspan="1" rowspan="1">1 per run</td><td colspan="1" rowspan="1">Up to 24 per run</td></tr><tr><td colspan="1" rowspan="1">Run Time</td><td colspan="1" rowspan="1">~1 hour 10 minutes</td><td colspan="1" rowspan="1">~3 hours</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. CLSI. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - $3 ^ { r d }$ Edition. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

2. CLSI. Interference Testing in Clinical Chemistry; Approved Guideline - $2 ^ { n d }$ Edition. CLSI document EP07-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2005.

3. CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - $2 ^ { n d }$ Edition. CLSI document EP17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

4. ISO 14971. Medical device – Application of risk management to medical devices.

5. CLSI. Molecular Diagnostic Methods for Infectious Diseases - $3 ^ { r d }$ Edition. CLSI report MM03. Wayne, PA: Clinical and Laboratory Standards Institute; 2015.

6. IEC 61010-2-101 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 2-101: Particular requirements for In-Vitro Diagnostic (IVD) Medical Equipment; 2002.

7. IEC 61010-2-010 Safety Requirements for Electrical Equipment for Measurement, Control and Laboratory Use – Part 010: Particular requirements for laboratory equipment for the heating of materials; 2014.

8. IEC 61010-2-081 Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Part 2-081: Particular requirements for automatic and semi-automatic laboratory equipment for analysis and other purposes; 2015.

9. Class II Special Controls Guideline: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens; Guideline for Industry and FDA Staff.

# L. Test Principle:

The PanNAT STEC Test is performed on soft or diarrheal stool specimens from patients suspected of gastrointestinal infection. The specimens may be preserved in Cary-Blair transport medium or unpreserved. To perform the test, a swab is used to transfer an aliquot of the stool specimen to a Sample Buffer Tube which is sealed with an Adaptor Cap and

inserted into the sample port of the PanNAT STEC Test cartridge. The cartridge is then loaded into the PanNAT System for automated processing.

Each test cartridge contains all the reagents necessary for nucleic acid extraction, amplification and detection for one sample. Processing of the sample within the cartridge is fully automated and fluid movement occurs via pneumatic pressure. Following bacterial lysis using chaotropic salts and detergent, DNA is captured on silica membranes and the eluate is used to rehydrate the amplification reagents.

Each cartridge contains nine amplification wells, three per target gene, that contain dried PCR reagents. Each well includes a pair of primers and a detector probe for an endogenous human gene that acts as a Process Control for nucleic acid recovery and amplification. One chamber corresponding to each target analyte $( s t x 1 , s t x 2$ and O157) is rehydrated with the extracted sample. The remaining chambers are rehydrated with buffer and serve as controls. The three Positive Control wells contain recombinant plasmid DNA bearing the target region for each of the target analytes as well as the Process Control. The three Negative Control wells contain no target or Process Control DNA. The expected results must be obtained with each Positive and Negative Control well for patient results to be reported. In addition, to report a negative result for a target analyte, the Process Control must be detected in the corresponding well, otherwise the result is reported as “Invalid.” The results are interpreted automatically by the instrument and may be viewed on-screen, printed or exported via USB (Universal Serial Bus) flash drive. A summary of the algorithm for result interpretation is provided in Table 1.

Table 1. Result interpretation for the PanNAT STEC Test   

<table><tr><td rowspan=2 colspan=1>Positive/NegativeControl Wells</td><td rowspan=1 colspan=2>Sample Well</td><td rowspan=2 colspan=1>Interpretation 2</td></tr><tr><td rowspan=1 colspan=1>Target Analyte</td><td rowspan=1 colspan=1>Process Control</td></tr><tr><td rowspan=4 colspan=1>Valid 3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid Assay</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td></tr><tr><td rowspan=4 colspan=1>Invalid 4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid Assay</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Invalid Assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Invalid Assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Invalid Assay</td></tr></table>

1 stx1, stx2 or O157 2 For Positive/Negative results for the target analytes to be reported, each of the Sample Wells (stx1, stx2 and O157) must produce a Valid result 3 Valid means that all 3 Positive and all 3 Negative Control wells generated the expected results 4 Invalid means that one or more Positive or Negative Control wells generated an unexpected result

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the PanNAT STEC Test between sites was evaluated in a study that was performed by two operators at each of three sites using a total of 13 PanNAT Systems. Over multiple days, each operator tested 15 replicates of a blinded 9- member panel of unpreserved stool samples containing different levels of the PanNAT STEC Test target analytes (3 sites X 2 operators X 9 panel members X 15 replicates $= 8 1 0$ PanNAT STEC Test results; 90 per panel member). The panel members used in the study are summarized in Table 2. A different lot of reagents was used at each of the testing sites. A summary of the qualitative test results obtained with each panel member, stratified by site/reagent lot is shown in Table 3 and an analysis of cycle numbers for each of the target analytes and the Process Control is provided in Table 4.

Table 2. Panel members evaluated in the PanNAT STEC Test Reproducibility Study   

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>E. coli Strain</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Target Level</td><td rowspan=1 colspan=1>CFU/mL 1</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>TW14003</td><td rowspan=2 colspan=1>stx1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>3 x 106</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=1>TW08023</td><td rowspan=2 colspan=1>stx2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>3 x 10b</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=2 colspan=1>CDC B6914-MS1(ATCC 43888)</td><td rowspan=2 colspan=1>0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>3 x 106</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=2 colspan=1>EDL933(ATCC 43895)</td><td rowspan=2 colspan=1>stx1/stx2/0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>3 x 106</td></tr></table>

$\overline { { { 1 \mathrm { ~ x ~ } 1 0 } ^ { 6 } \mathrm { C F U / m L } = 1 X } }$ LoD; $\overline { { 3 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } = 3 \mathrm { X } } }$ LoD (Section M(1)(d))

There was a total of 33/810 $( 4 . 1 \% )$ Invalid results during the study. In addition, run errors occurred with 4/810 $( 0 . 5 \% )$ samples. Therefore, a total of 773 valid PanNAT STEC Test results was included in the analysis of device reproducibility (Tables 3 and 4).

For each panel member, agreement with expected results was assessed collectively for all three analytes. For panel members $\# 1 . 7$ and $\# 9$ , agreement with the expected results for all three analytes was $2 9 7 . 7 \%$ . For panel member $\# 8$ , the Low Positive sample for the $s t x 1 ^ { + } / s t x 2 ^ { + } / 0 1 5 7 ^ { + } E$ . coli O157 strain EDL933, the agreement at Sites A and B was $9 3 . 3 \%$ (28/30) and $9 6 . 4 \%$ (27/28), respectively, whereas at Site C it was $7 6 . 7 \%$ (23/30). The seven (7) discordant results at Site C were investigated. Of these, two (2) were due to failure to detect stx1 $\mathit { s t x 1 }$ agreement 28/30, $9 3 . 3 \%$ , four (4) were due to failure to detect $s t x 2$ ( $s t x 2$ agreement 26/30, $8 6 . 6 \%$ and one (1) was due to failure to detect O157 (O157 agreement 29/30, $9 6 . 7 \%$ ). In each case, the sample was correctly reported positive for the other two analytes and the observed discordance in the composite results (for stx1, $s t x 2$ and O157 combined) was therefore not due to a systematic failure for any single analyte. Overall, the reproducibility of the PanNAT STEC Test was determined to be acceptable.

Table 3. Summary of qualitative test results from the PanNAT STEC Test Reproducibility Study, stratified by site/reagent lot   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Analyte(s)</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=4>Agreement (%)1,2</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>28/28(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>26/26(100)</td><td rowspan=1 colspan=1>84/84(100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>stx1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>28/28(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>26/28(92.9)</td><td rowspan=1 colspan=1>83/85(97.6)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>27/27(100)</td><td rowspan=1 colspan=1>29/30(96.7)</td><td rowspan=1 colspan=1>25/25(100)</td><td rowspan=1 colspan=1>81/82(98.8)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=1>stx2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>27/27(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>28/30(93.3)</td><td rowspan=1 colspan=1>84/86(97.7)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>28/28(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>87/87(100)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=2 colspan=1>0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>28/29(96.6)</td><td rowspan=1 colspan=1>87/88(98.9)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>28/28(100)</td><td rowspan=1 colspan=1>29/29(100)</td><td rowspan=1 colspan=1>87/87(100)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=2 colspan=1>stx1/stx2/0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>28/30(93.3)</td><td rowspan=1 colspan=1>27/28(96.4)</td><td rowspan=1 colspan=1>23/30 3(76.7)</td><td rowspan=1 colspan=1>78/88(88.6)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>26/27(96.3)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>86/87(98.9)</td></tr></table>

PanNAT Systems used: Site A: 9; Site B: 3; Site C: 6; 5 of the PanNAT Systems were used at more than one site (3 at Sites A and C; 2 at Sites A and B) 1 In comparison to the expected results for the panel member for all three analytes combined 2 Each site used a different reagent lot: site and reagent lot are therefore confounded 3 Of the 7 discordant results, 2 were due to failure to detect stx1, 4 were due to failure to detect $s t x 2$ and 1 was due to failure to detect O157. In each case, the sample was reported positive for the other two analytes.

Table 4. Summary of mean cycle numbers for panel members in the PanNAT STEC Test Reproducibility Study   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Analyte(s)</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=6>Mean Cycle Number (Standard Deviation)</td></tr><tr><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>PrC</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>PrC</td><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>PrC</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.9(0.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.8(1.1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.5(0.9)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>stx1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>35.6(0.7)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.1(1.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.0(0.9)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>34.5(1.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.0(0.8)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.7(0.9)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=1>stx2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.1(0.9)</td><td rowspan=1 colspan=1>35.3(0.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.7(1.0)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.1(0.8)</td><td rowspan=1 colspan=1>33.7(0.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.9(0.8)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=2 colspan=1>0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.1(0.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.0(0.9)</td><td rowspan=1 colspan=1>35.5(0.9)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.1(1.1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29.0(1.0)</td><td rowspan=1 colspan=1>34.2(0.7)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=2 colspan=1>stx1/stx2/0157</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>36.1(0.9)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>35.3(1.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>35.4(0.8)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>34.6(0.8)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>33.6(1.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>33.8(0.9)</td><td rowspan=1 colspan=1></td></tr></table>

PrC: Process Control Note: Specimens with false negative results were excluded from the calculation of mean and standard deviation

The PanNAT STEC Test demonstrated acceptable reproducibility across sites, PanNAT Systems, operators, panel members and reagent lots.

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Specimen Stability

The stability of preserved and unpreserved stool specimens for use with the PanNAT STEC Test was evaluated analytically by testing contrived samples that were seeded with an enumerated suspension of $E$ . coli O157 strain EDL933 $\stackrel { \cdot } { ( } s t x 1 ^ { + } / s t x 2 ^ { + } / f c l ^ { + } )$ at levels close the LoD of the assay. The samples were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and tested at predefined intervals. Five assay replicates were tested at each time point. The results of the study support the claimed stability of preserved and unpreserved stool specimens at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 7 days.

The results of additional testing that was performed to establish the stability of frozen, unpreserved stool specimens are described in Section M(2)(b).

Process Control The PanNAT STEC Test Process Control is comprised of an endogenous human gene that is co-extracted from the stool specimen and co-amplified along with the with the target analytes (if present). The Process Control is designed to monitor for the adequacy of sample addition to the test cartridge as well as reagent and process integrity.

# Integrated Positive/Negative Controls

Each PanNAT STEC Test cartridge includes integrated amplification wells for Positive and Negative Controls. These wells contain the primers and probes for detection of stx1, stx2 and O157 target genes as well as the Process Control. In addition, the Positive Control wells contain plasmid DNA that carries the target sequences for each of the PanNAT STEC Test analytes; whereas the Negative Control wells contain no target DNA for any of the target analytes. Each of the Positive and Negative Control wells must produce the expected result in order for sample results to be reported. If one or more of the control wells fails the test sample is reported as “Invalid.” Refer to Table 1, above, for additional information regarding the interpretation of PanNAT STEC Test results.

External Positive/Negative Controls

External Controls should be tested according to guidelines or requirements of local, provincial and/or federal regulations or accreditation organizations.

Commercially prepared External Positive and Negative Controls were tested throughout the prospective Clinical Study described in Section M(3)(a). The Positive External Control comprised a standardized suspension of $E$ . coli ATCC 43895 strain EDL933 $\mathrm { \langle 5 \mathrm { ~ x ~ } 1 0 ^ { 6 } C F U / m L }$ ) and a human cell line $\left( 4 . 8 – 9 . 6 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { c e l l s / m L } \right)$ ). The negative External Control comprised diluent containing the human cell line alone. A total of 170 pairs of External Controls were tested over the course of the study. On initial testing, 6/170 $( 3 . 5 \% )$ Positive Controls and 12/170 $( 7 . 1 \% )$ Negative Controls produced Invalid results. Of the remainder, 162/164 $( 9 8 . 7 \% )$ Positive Controls and 158/158 $( 1 0 0 \% )$ Negative Controls produced the expected results.

As an alternative to commercially prepared controls, a strain of $E$ . coli that is known to harbor the genes targeted by the PanNAT STEC Test (e.g., ATCC 43895 strain EDL933) may be used as a Positive Control. Another strain of $E$ . coli that does not carry any of the target genes (e.g., E. coli K12) may used as a Negative Control. Clinical specimens that are known to be positive or negative for the PanNAT STEC Test analytes may also be used as Positive and Negative External Controls, respectively.

# d. Detection limit:

# Limit of Detection

The Limit of Detection (LoD) of the PanNAT STEC Test was initially estimated using seven Shiga toxin positive/negative strains of O157 and non-O157 E. coli that were diluted preserved and unpreserved stool matrix at three different concentrations. The estimated LoD for each strain was the lowest concentration at which all of three replicates produced positive results. For two strains, the LoD was then confirmed by testing additional replicates at the estimated LoD concentration. The LoD was defined as the concentration at which $2 9 5 \%$ of replicates produced a positive result for both strains. The results of the study are summarized in Tables $\underline { { \boldsymbol { 5 } } }$ and $\underline { { \mathbf { 6 } } }$ for unpreserved and preserved stool specimens, respectively. The confirmed LoD with unpreserved stool specimens was $\mathrm { { \dot { 1 } { \cdot } x 1 0 ^ { 6 } C F U / \mathrm { { \dot { m } L } } } }$ and with preserved specimens in Cary-Blair medium it was $5 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ (equivalent to $\mathrm { i } . 7 5 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { \dot { C } F U / m L }$ of raw, undiluted stool).

Table 5. Summary of results from the PanNAT STEC Test LoD Study with unpreserved stool specimens   

<table><tr><td rowspan=3 colspan=1>E. coli Strain</td><td rowspan=2 colspan=3>Genotype</td><td rowspan=1 colspan=4>Number Positive/Tested</td></tr><tr><td rowspan=1 colspan=3>LoD Titration</td><td rowspan=1 colspan=1>LODConfirmation</td></tr><tr><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>1 x 105CFU/mL</td><td rowspan=1 colspan=1>1 x 106CFU/mL</td><td rowspan=1 colspan=1>1 x 107CFU/mL</td><td rowspan=1 colspan=1>1 x 106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>EDL933</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>92-3265</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B6914-MS1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>92-3099</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MT#2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M105-14</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3007-85</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>2/21(100%)</td><td rowspan=1 colspan=1>28/28(100%)</td></tr></table>

1/28 assay replicates was reported as Invalid and no retest was performed

Table 6. Summary of results from the PanNAT STEC Test LoD Study with stool specimens preserved in Cary-Blair Transport Medium   

<table><tr><td rowspan=3 colspan=1>E. coliStrain</td><td rowspan=2 colspan=3>Genotype</td><td rowspan=1 colspan=5>Number Positive/Tested</td></tr><tr><td rowspan=1 colspan=3>LoD Titration</td><td rowspan=1 colspan=2>LoD Confirmation</td></tr><tr><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>2.9 x 104CFU/mL</td><td rowspan=1 colspan=1>2.9 x 105CFU/mL</td><td rowspan=1 colspan=1>2.9 x 106CFU/mL</td><td rowspan=1 colspan=1>2.9 x 105CFU/mL</td><td rowspan=1 colspan=1>5 x 10}$CFU/mL</td></tr><tr><td rowspan=1 colspan=1>EDL933</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>2/2 2(100%)</td><td rowspan=1 colspan=1>18/19 3, 5(94.7%)</td><td rowspan=1 colspan=1>28/28(100%)</td></tr><tr><td rowspan=1 colspan=1>92-3265</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B6914-MS1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>92-3099</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MT#2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M105-14</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>3007-85</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>17/19 4, 5(89.5%)</td><td rowspan=1 colspan=1>28/28(100%)</td></tr></table>

Note: Retesting of samples with Invalid or Incomplete results was not performed 1 Preserved specimens were prepared by diluting 1 volume of raw stool in 2.5 volumes of Cary-Blair transport medium. 2 1/3 assay replicates was reported as Invalid 3 1/20 assay replicates was reported as Incomplete 4 1/20 replicates was reported as Invalid 5 In the initial confirmation study with preserved specimens, both strains yielded ${ < } 9 5 \%$ proportion positive. The confirmation study was therefore repeated at a higher target level $\mathrm { ~ 5 ~ x ~ } \mathrm { 1 \dot { 0 } ^ { 5 } }$ vs $2 . 9 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { \ : \dot { C } F U / m L }$ ).

# Inclusivity (Analytical Reactivity)

The inclusivity of the PanNAT STEC Test was evaluated by testing 24 wellcharacterized strains of $E$ . coli including seven strains of serogroup 0157 and 17 strains of Shiga toxin producing non-O157 serogroups (six of which non-typeable). An O-antigen deficient laboratory strain of $E$ . coli (K12) was included as a Negative Control. Each strain was tested in triplicate in unpreserved stool matrix at a concentration equivalent to 3X LoD $( 3 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ . The results of the study are summarized in Table 7. Three strains failed to produce the expected results due to mismatches in the target regions for the PanNAT STEC Test $s t x 1 / 2$ primers and/or probes. A Limitation was therefore added to the device labeling to indicate that false negative results may occur due to the presence of sequence variations in the regions of the stx1 and $s t x 2$ genes targeted by the PanNAT STEC Test.

Table 7. Strains of $E .$ . coli evaluated in the PanNAT STEC Test Inclusivity Study   

<table><tr><td rowspan=2 colspan=1>E. coli Strain</td><td rowspan=2 colspan=1>Serogroup</td><td rowspan=1 colspan=3>Genotype (subtype) 1</td><td rowspan=2 colspan=1>PanNATSTEC TestCorrect</td></tr><tr><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>0157</td></tr><tr><td rowspan=1 colspan=1>ATCC 35150</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 43888</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 43889</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 43890</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 43894</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 700376</td><td rowspan=1 colspan=1>0157:NM</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC 700377</td><td rowspan=1 colspan=1>0157:NM</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2326</td><td rowspan=1 colspan=1>0104:H4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>DEC142</td><td rowspan=1 colspan=1>0165:H25</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+ (2c/d+)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>DEC171</td><td rowspan=1 colspan=1>ONT:H25</td><td rowspan=1 colspan=1>+(1a)</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>DEC99</td><td rowspan=1 colspan=1>0121:H19</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(2a)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>MSU TW01664</td><td rowspan=1 colspan=1>0145:H16</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>MSUJ TW06315</td><td rowspan=1 colspan=1>0111:NM</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>MSUTW08023</td><td rowspan=1 colspan=1>0121:H19</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>MSU TW14003</td><td rowspan=1 colspan=1>045:H2</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW00971</td><td rowspan=1 colspan=1>O26:H11</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW05672</td><td rowspan=1 colspan=1>ONT:H24</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW05997</td><td rowspan=1 colspan=1>0103:NM</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW07731</td><td rowspan=1 colspan=1>0146:H21</td><td rowspan=1 colspan=1>+(1c)</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/32</td></tr><tr><td rowspan=1 colspan=1>TW07740</td><td rowspan=1 colspan=1>0146:H21</td><td rowspan=1 colspan=1>+ (1d)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/3 3</td></tr><tr><td rowspan=1 colspan=1>TW07754</td><td rowspan=1 colspan=1>0146:H21</td><td rowspan=1 colspan=1>+(1d)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/3 3</td></tr><tr><td rowspan=1 colspan=1>TW07929</td><td rowspan=1 colspan=1>ONT:HNT</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW07930</td><td rowspan=1 colspan=1>ONT:HNM</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TW08111</td><td rowspan=1 colspan=1>ONT:H49</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td></tr></table>

stx1 and stx2 subtypes are listed in parenthesis if known 2 3/3 replicates were reported positive for $s t x 1$ but negative for $s t x 2$ . Sequence analysis showed that the $s t x 2$ variant carried by this strain most closely resembled $s t x 2 \mathrm { b }$ which is not detected by the PanNAT STEC Test. 3 Sequence analysis showed the presence of mismatches in the $s t x 1$ target region with the PanNAT STEC Test detector probe. Both TW07740 and TW07754 were initially characterized as subtype $s t x 1 { \mathrm { c } }$ but further analysis suggested that they more closely resemble strains of subtype stx1d which is detected with reduced sensitivity by the PanNAT STEC Test.

# Bioinformatic Analysis

The inclusivity of the PanNAT STEC Test primers and probes for the stx1, stx2 and fcl gene targets was analyzed in silico using the Basic Local Alignment Search Tool (BLAST). The targeted regions these genes were generally well conserved, although the potential for false-negative PanNAT STEC Test results due to sequence variation in the stx1 and $s t x 2$ genes and with certain subtypes is noted as a Limitation in the device labeling.

# e. Analytical specificity:

# Cross-reactivity Study

The analytical specificity of the PanNAT STEC Test was evaluated by testing a panel of 49 organisms and viruses that may be found in stool specimens (Table 8). Testing was performed in triplicate by spiking unpreserved stool with each potentially crossreactive species at ${ \geq } 1 0 ^ { 6 }$ CFU or organisms/mL for bacteria, yeast and parasites and $\geq 1 0 ^ { 5 } \mathrm { T C I \hat { D } _ { 5 0 } / m L }$ for viruses. Negative PanNAT STEC Test results were obtained

with all replicates for all the organisms and viruses tested except for a strain of Shigella dysenteriae that carries the stxA gene, a closely related homologue of $E$ . coli stx1, and which is therefore expected to produce a positive result for this analyte. The potential for the PanNAT STEC Test to give positive results for Shiga toxin genes with strains of Shigella or other Enterobacteriaceae is noted as a Limitation in the device labeling.

Table 8. Panel of microorganisms and viruses evaluated for potential cross-reaction and/or interference in the PanNAT STEC Test   

<table><tr><td rowspan=1 colspan=2>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Morganella morganii</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>Clostridium parabotulinum</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Shigella dysenteriae (stxA) 2</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Shigella flexneri</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli K12</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O111a (EAEC)</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O111a:2 (EPEC)</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O29:NM (EIEC)</td><td rowspan=1 colspan=1>Vibrio spp.</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli O8:H19 (ETEC)</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermanii</td><td rowspan=1 colspan=1>Yersinia pseudotuberculosis</td></tr><tr><td rowspan=1 colspan=1>Hafnia alvei</td><td></td></tr><tr><td rowspan=1 colspan=2>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 18</td><td rowspan=1 colspan=1>Echovirus 11</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Enterovirus 68</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B4</td><td rowspan=1 colspan=1>Norovirus (recombinant)</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Rotavirus</td></tr><tr><td rowspan=1 colspan=2>Parasites and Yeast</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Entamoeba histolytica (lysate)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Giardia lamblia</td></tr></table>

EAEC: Enteroaggregative Escherichia coli; EPEC: Enteropathogenic Escherichia coli; Enteroinvasive Escherichia coli; ETEC: Enterotoxigenic Escherichia coli 1 Cross-reactivity Study: 1/3 assay replicates reported as Incomplete; not retested 2 Cross-reactivity Study: $3 / 3$ replicates reported as positive for stx1, negative for stx2 and O157. This result is expected. 3 Microbial Interference Study: 1/3 assay replicates reported as Incomplete; not retested (refer to Section M(1)(g))

# Bioinformatic Analysis

In silico analysis was performed to determine the potential for cross-reaction of the PanNAT STEC Test primers and probes with non-target nucleic acids. The potential for cross-reaction with the stxA gene of Shigella spp. is noted in the device labeling. Otherwise, no significant homology/complementarity was observed that was predicted to produce false positive results.

# Contamination Study

The potential for false-positive results with the PanNAT STEC Test due to run-to-run contamination was evaluated by testing an alternating series of $E$ . coli O157 $( s t x 1 ^ { + } / s t x 2 ^ { + } / \mathrm { O } 1 5 7 ^ { + } )$ “high positive” $( 1 \bar { 0 } ^ { 7 } \mathrm { C F U / m L } )$ and negative unprocessed stool samples in successive runs on three PanNAT Systems. One negative sample was reported as Invalid and not retested. The expected results were obtained for all the remaining “high positive” (15/15; $100 \%$ ) and negative samples (14/14; $100 \%$ ). These results are acceptable.

# $f .$ Assay cut-off:

The methodology for threshold calculation was optimized through analysis of ${ \mathrm { > } } 1 8 0 0$ contrived specimens and validated in the Clinical Studies described in Section M(3)(a). Individual thresholds for each cartridge well and optical channel are determined based on an analysis of the changes in fluorescence during the early cycles of PCR amplification. A cycle number is then assigned where the rate of fluorescence accumulation exceeds the applicable threshold for the well and optical channel, and the accumulation of fluorescence is determined to be consistent with amplification. The decision algorithm described in Table 1 is then applied to determine the result status of the sample.

# g. Assay interference:

# Interfering Substances

The potential for interference with the PanNAT STEC Test was evaluated with 25 endogenous and exogenous substances that may be found in stool specimens (Table 9). The study was performed using two strains of Shiga toxin producing $E$ . coli O157 (EDL931 and EDL933; both $s t x 1 ^ { + } / s t x 2 ^ { + } / f c l ^ { + } )$ at 1X and 3X LoD ( $\mathrm { ~ \bar { ~ } 1 ~ x ~ } 1 \bar { 0 } ^ { 6 }$ and $3 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m L } )$ in unpreserved stool samples, as well as with unpreserved stool samples that were negative for each of the target analytes.

False negative results were obtained in the presence of three substances at the concentrations tested: $5 \%$ (v/v) Imodium AD, $10 \%$ (w/v) Gaviscon and $4 8 ~ \mathrm { m g / m L }$ stearic acid. In each case, only $1 / 3$ replicates under the test condition produced a negative result and in two cases (with Imodium AD and stearic acid), the failures were at the 1X LoD target level. No false positive results were observed under any of the test conditions.

A total of 9 Invalid results were observed during the study in association with 8 of the potentially interfering substances that were tested. Although none of the samples with Invalid results was retested, there was no evidence of systemic failures with any individual substance.

Overall, the results of the Interfering Substances Study are considered acceptable.

Table 9. Substances evaluated for potential interference with the PanNAT STEC Test   

<table><tr><td rowspan=1 colspan=4>Potential Source</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration inUnpreserved Stool</td></tr><tr><td rowspan=5 colspan=4>Endogenous</td><td rowspan=1 colspan=1>Whole Blood (EDTA)</td><td rowspan=1 colspan=1>40% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>35 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>48 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>10 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Bile</td><td rowspan=1 colspan=1>250 mg/mL</td></tr><tr><td rowspan=1 colspan=4>Exogenous</td><td rowspan=1 colspan=1>Kaopectate</td><td rowspan=1 colspan=1>3.5% v/v</td></tr><tr><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>Pepto Bismol</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Imodium AD</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Petroleum Jelly</td><td rowspan=1 colspan=1>500 mg/mL</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Baby Wipes</td><td rowspan=1 colspan=1>50% v/v</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Hydrocortisone Cream</td><td rowspan=1 colspan=1>500 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Diaper Rash Cream</td><td rowspan=1 colspan=1>500 mg/mL</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Stearic Acid 2</td><td rowspan=1 colspan=1>48 mg/mL</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Palmitic Acid</td><td rowspan=1 colspan=1>48 mg/mL</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=11 colspan=1></td><td rowspan=1 colspan=1>Gaviscon</td><td rowspan=1 colspan=1>100 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Milk of Magnesia</td><td rowspan=1 colspan=1>100 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Aleve</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>95 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>500 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>50 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>4.3 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>50 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>12.5 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Sennica Laxative</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Diaper Eluent</td><td rowspan=1 colspan=1>25% v/v</td></tr></table>

EDTA: Ethylenediaminetetraacetic acid (anticoagulant) 1 1/3 replicates for $E$ . coli O157 strain EDL933 at 1X LoD produced a negative result $( s t x 1 ^ { + } / s t x 2 ^ { + } / \mathrm { O } 1 5 7 ^ { - } )$ 2 1/3 replicates for $E$ . coli O157 strain EDL931 at 1X LoD produced a negative result $( s t x 1 ^ { - } / s t x 2 ^ { - } / \mathrm { O } 1 5 7 ^ { - } )$ 3 $1 / 3$ replicates for $E$ . coli O157 strain EDL933 at 3X LoD produced a negative result $( s t x 1 ^ { + } / s t x 2 ^ { + } / \mathrm { O } 1 5 7 ^ { - } )$

# Microbial Interference

The potential for interference with the PanNAT STEC Test by organisms that may be present in stool specimens was investigated using the same list of species that was evaluated for potential cross-reactivity (refer to Section M(1)(e) and Table 8).

Testing was performed with each potentially interfering species in triplicate using unpreserved stool specimens in the presence of $E .$ . coli O157 strain EDL933 (ATCC 43895; $s t x 1 ^ { + } / s t x 2 ^ { + } / \bar { \bf O } 1 5 7 ^ { + } )$ at 1X LoD $( 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ). The potentially interfering species were tested at ${ 1 0 } ^ { 6 } \mathrm { C F U }$ or organisms/mL for bacteria, yeast and parasites and $\mathrm { 1 0 ^ { 5 } P F U / m L }$ for viruses. No interference was observed with any of the microorganisms or viruses tested, although the potential for false-positive results with species of Shigella that carry the $s t x A$ gene homolog of $s t x 1$ is noted in the device labeling.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

To support use of frozen, retrospective unpreserved specimens to establish clinical performance and in analytical testing, a study was conducted to compare the results obtained from testing of “fresh” and frozen stool samples at different target levels. Fresh (never frozen) unpreserved stool was spiked with $E$ . coli O157 strain EDL933 $( s t x 1 ^ { + } / s t x 2 ^ { + } / \mathrm { O } 1 5 7 ^ { + } )$ at different target levels, aliquoted and either tested on the day of preparation or after different periods of storage at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ . The results of the study are presented in Table 10 and showed that freezing of specimens for up to 68 weeks at - $8 0 \mathrm { { } ^ { \circ } C }$ had no effect on PanNAT STEC Test results. These results are acceptable.

Table 10. Summary of results from testing of unpreserved “fresh” and frozen stool samples   

<table><tr><td rowspan=3 colspan=1>Target Level(CFU/mL)</td><td rowspan=1 colspan=4>Agreement with Expected Results (%)</td></tr><tr><td rowspan=2 colspan=1>Fresh&quot;</td><td rowspan=1 colspan=3>Duration of Storage at -80°C</td></tr><tr><td rowspan=1 colspan=1>4 Weeks</td><td rowspan=1 colspan=1>10 Weeks</td><td rowspan=1 colspan=1>68 Weeks</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>57/57(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/61 1, 2(98.4)</td></tr><tr><td rowspan=1 colspan=1>1.5X LoD(1.5 x 106)</td><td rowspan=1 colspan=1>18/18(100)</td><td rowspan=1 colspan=1>18/18(100)</td><td rowspan=1 colspan=1>18/18(100)</td><td rowspan=1 colspan=1>18/18(100)</td></tr><tr><td rowspan=1 colspan=1>5X LoD(5 x 106)</td><td rowspan=1 colspan=1>20/20(100)</td><td rowspan=1 colspan=1>21/21(100)</td><td rowspan=1 colspan=1>21/21(100)</td><td rowspan=1 colspan=1>21/21(100)</td></tr><tr><td rowspan=1 colspan=1>100X LoD(1 x 108)</td><td rowspan=1 colspan=1>19/19 3(100)</td><td rowspan=1 colspan=1>21/21(100)</td><td rowspan=1 colspan=1>20/20(100)</td><td rowspan=1 colspan=1>21/21(100)</td></tr></table>

1 1 replicate reported as Invalid 2 1 false positive reported for $s t x 1$ 2 replicates reported as Invalid

3. Clinical studies:

a. Clinical Sensitivity:

Prospective Clinical Study The performance of the PanNAT STEC Test was evaluated in a prospective Clinical Study that was conducted at six sites in the U.S. using leftover de-identified stool specimens obtained from subjects who presented with symptoms of acute diarrhea. The specimens were either unpreserved (“fresh”) or preserved in Cary-Blair transport medium. Each specimen was divided into three aliquots of $2 5 0 \mu \mathrm { L }$ each: #1: for analysis with the PanNAT STEC Test (stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 7 days); $\# 2$ : for testing by the comparator culture method (stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to three days or

longer at $- 7 0 ^ { \circ } \mathrm { C } \mathrm { \Omega }$ ); #3: for analysis in the event of discordant results between the PanNAT STEC Test and the reference method.

The reference culture method comprised separate procedures for growth and detection of Shiga-toxin producing $E .$ . coli (STEC) and $E$ . coli O157. For detection of STEC, $5 0 \mu \mathrm { L }$ of unpreserved or $1 7 5 \mu \mathrm { L }$ of preserved stool was inoculated into $5 \mathrm { m L }$ MacConkey enrichment broth which was incubated at $3 7 { \pm } 2 ^ { \circ } \mathrm { C }$ for 16-24 hours before testing with an immunoassay for the presence of Shiga Toxin 1 (ST1) and Shiga Toxin 2 (ST2) according to the manufacturer’s instructions. For detection of $E$ . coli O157, an aliquot of the specimen was streaked onto a Sorbitol MacConkey agar plate that was incubated at $3 6 { \pm } 2 ^ { \circ } \mathrm { C }$ for 18-24 hours. Clear or beige colonies that were presumptively identified as $E$ . coli O157 were sub-cultured, serotyped by latex agglutination and, if positive for the O157 antigen, their identity was confirmed by biochemical analysis.

A total of 1331 specimens from unique subjects were initially enrolled in the prospective Clinical Study. Of these, 30 (29 preserved and one (1) unpreserved) were excluded from the analysis of performance due inadvertent data loss (21) or protocol deviation (9). Of the remaining 1301, 99 $( 7 . 6 \% )$ initially produced Invalid PanNAT STEC Test results. Per protocol, all 99 specimens were retested and 78/99 $( 7 8 . 8 \% )$ produced valid results for a final Invalid rate of $1 . 6 \%$ (21/1301). Of the 1280 specimens with valid PanNAT STEC Test results, 1184 $( 9 2 . 5 \% )$ were preserved in Cary-Blair Transport Medium, while 96 $( 7 . 5 \% )$ were unpreserved stool specimens.

The results from testing of prospectively collected stool specimens with the PanNAT STEC Test in comparison to the reference method are shown in Tables 11, 12 and 13.

Table 11. Results from testing prospectively collected stool specimens preserved in CaryBlair Transport Medium with the PanNAT STEC Test in comparison to the reference method   

<table><tr><td rowspan=2 colspan=2>Detection of stx1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1172</td><td rowspan=1 colspan=1>1172</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1180</td><td rowspan=1 colspan=1>1184</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (4/4); 95% CI: 51.0-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>99.3% (1172/1180); 95% CI: 98.7-99.7%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>33.3% (4/12)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>100% (1172/1172)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>0.3% (4/1184)</td></tr><tr><td rowspan=2 colspan=2>Detection of stx2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7^2}$</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1172</td><td rowspan=1 colspan=1>1172</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1179</td><td rowspan=1 colspan=1>1184</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (5/5); 95% CI: 56.6-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>99.4% (1172/1179); 95% CI: 98.8-99.7%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>41.7% (5/12)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>100% (1172/1172)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>0.4% (5/1184)</td></tr><tr><td rowspan=2 colspan=2>Detection of E. coli O157</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3 $</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1179</td><td rowspan=1 colspan=1>1179</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1181</td><td rowspan=1 colspan=1>1184</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (3/3); 95% CI: 43.9-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>99.8% (1179/1181); 95% CI: 99.4-100%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>60.0% (3/5)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>100% (1179/1179)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>0.3% (3/1184)</td></tr><tr><td rowspan=1 colspan=5>95% CI: 95% score confidence interval</td></tr></table>

1 7/8 positive for stx1 by PCR/bidirectional sequencing 2 $3 / 7$ positive for $s t x 2$ by PCR/bidirectional sequencing 3 $3 / 3$ positive for $s t x 2$ ; negative for stx1 4 1/2 positive for $E$ . coli O157 by PCR/bidirectional sequencing

Table 12. Results from testing prospectively collected unpreserved stool specimens with the PanNAT STEC Test in comparison to the reference method   

<table><tr><td rowspan=2 colspan=2>Detection of stx1</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>Not applicable</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>100% (96/96); 95% CI: 96.2-100%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>Not applicable</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>100% (96/96)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>0% (0/96)</td></tr><tr><td rowspan=2 colspan=2>Detection of stx2</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (2/2); 95% CI: 34.2-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>98.9% (93/94); 95% CI: 94.2-99.8%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>66.7% (2/3)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>100% (93/93)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>2.1% (2/96)</td></tr><tr><td rowspan=2 colspan=2>Detection of E. coli O157</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>$2{2}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>94</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (2/2); 95% CI: 34.2-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>100% (94/94); 95% CI: 96.1-100%</td></tr><tr><td rowspan=1 colspan=2>Positive Predictive Value</td><td rowspan=1 colspan=3>100% (2/2)</td></tr><tr><td rowspan=1 colspan=2>Negative Predictive Value</td><td rowspan=1 colspan=3>100% (94/94)</td></tr><tr><td rowspan=1 colspan=2>Prevalence</td><td rowspan=1 colspan=3>2.1% (2/96)</td></tr></table>

$9 5 \%$ CI: $9 5 \%$ score confidence interval 1 $1 / 1$ positive for $s t x 2$ by PCR/bidirectional sequencing 2 2/2 positive for $s t x 2$ ; negative for stx1

Table 13. Results from testing of prospectively collected stool specimens with the PanNAT STEC Test, stratified by target analyte/analyte combination   

<table><tr><td rowspan=2 colspan=1>Analyte(s) 1</td><td rowspan=1 colspan=2>Agreement</td></tr><tr><td rowspan=1 colspan=1>Preserved Specimens</td><td rowspan=1 colspan=1>Unpreserved Specimens</td></tr><tr><td rowspan=1 colspan=1>stx1 Only</td><td rowspan=1 colspan=1>100%(4/4)</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx2 Only</td><td rowspan=1 colspan=1>100%(2/2)</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>0157 Only</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx1 + 0157</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx2 + 0157</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(2/2)</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2 + 0157</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>98.7%(1160/1175) 2</td><td rowspan=1 colspan=1>98.9%(93/94) 3</td></tr></table>

1 As determined by the reference method 2 Of the 15 specimens with false positive results by the PanNAT STEC Test, 8 were positive for stx1, 5 for $s t x 2$ and 2 for both $s t x 2$ and O157 3 1/94 negative specimens was positive by the PanNAT STEC Test for stx2

# Analysis of Retrospective Specimens

Due to the low prevalence of the target analytes in the prospective Clinical Study, additional testing was performed with retrospective (frozen), unpreserved stool specimens that were characterized using a well validated PCR/bidirectional sequencing assay to confirm the presence/absence of the stx1, stx2 and/or O157 targets. One hundred and fourteen (114) retrospective specimens were initially enrolled in the study of which eight (8) were excluded from the analysis of performance due to the absence of a result for the comparator PCR/bidirectional sequencing method (7) or because previous enrollment (1). On initial testing, 8/106 $( 7 . 5 \% )$ retrospective specimens produced Invalid results with the PanNAT STEC Test. Per protocol, all were retested and 5/8 $( 6 2 . 5 \% )$ produced valid results upon repeat, for a final invalid rate of $2 . 8 \%$ (3/106). Therefore, a total of 103 retrospective specimens produced evaluable results with both the comparator method and the PanNAT STEC Test. The results of the study are summarized in Tables 14 and 15.

Table 14. Results from testing retrospective (frozen), unpreserved specimens with the PanNAT STEC Test   

<table><tr><td rowspan=2 colspan=2>Detection of stx1</td><td rowspan=1 colspan=3>PCR/Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>88.5% (23/26); 95% CI: 71.0-96.0%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>97.4% (75/77); 95% CI: 91.0-99.3%</td></tr><tr><td rowspan=2 colspan=2>Detection of stx2</td><td rowspan=1 colspan=3>PCR/Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3^{2$</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>90.0% (27/30); 95% CI: 74.4-96.5%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>98.6% (72/73); 95% CI: 92.6-99.8%</td></tr><tr><td rowspan=2 colspan=2>Detection of E. coli O157</td><td rowspan=1 colspan=3>PCR/Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3 \ }$</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>1024</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>89.7% (26/29); 95% CI: 73.6-96.4%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>98.6% (72/73); 95% CI: 92.6-99.8%</td></tr></table>

$9 5 \%$ CI: $9 5 \%$ score confidence interval 1 $3 / 3$ specimens were negative by standard of care culture but positive by PCR/bidirectional sequencing 2 2/3 specimens were negative by standard of care culture but positive by PCR/bidirectional sequencing 3 1/3 specimens was negative by standard of care culture but positive by PCR/bidirectional sequencing 4 Insufficient PCR product was obtained from one specimen to enable bidirectional sequencing for the O157 target

Table 15. Results from testing retrospective (frozen), unpreserved stool specimens with the PanNAT STEC Test, stratified by target analyte/analyte combination   

<table><tr><td rowspan=1 colspan=1>Analyte(s) 1</td><td rowspan=1 colspan=1>Agreement</td></tr><tr><td rowspan=1 colspan=1>stx1 Only</td><td rowspan=1 colspan=1>25.0%(1/4)</td></tr><tr><td rowspan=1 colspan=1>stx2 Only</td><td rowspan=1 colspan=1>0.0%(0/2)</td></tr><tr><td rowspan=1 colspan=1>0157 Only</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx1 + 0157</td><td rowspan=1 colspan=1>0.0%(0/1)</td></tr><tr><td rowspan=1 colspan=1>stx2 + 0157</td><td rowspan=1 colspan=1>85.7%(6/7)</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2 + 0157</td><td rowspan=1 colspan=1>95.2%(20/21)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>98.5%(67/68) 2</td></tr></table>

1 As determined by PCR/bidirectional sequencing 2 1/68 negative specimens was positive by the PanNAT STEC Test for stx1, $s t x 2$ and O157

# Analysis of Contrived Specimens

To supplement the prospective Clinical Study and analysis of retrospective specimens, additional testing was also performed with 16 contrived (spiked) unpreserved stool specimens, each of which was spiked with an enumerated stock of a different strain of Shiga-toxin producing $E$ . coli or $E .$ . coli O157:H7 to a final concentration of $5 \mathrm { ~ x ~ } 1 0 ^ { \overline { { 6 } } } \mathrm { C F U / m i }$ of stool (5X LoD). The contrived specimens were frozen and tested at the clinical sites, interspersed with retrospective specimens (above). No Invalid results were obtained with the contrived specimens (Invalid rate $0 / 1 6 = 0 \%$ ). The results of the study are summarized in Tables 16 and 17.

Table 16. Results from testing frozen, contrived specimens with the PanNAT STEC Test   

<table><tr><td rowspan=2 colspan=2>Detection of stx1</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (12/12); 95% CI: 75.8-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>100% (4/4); 95% CI: 51.0-100%</td></tr><tr><td rowspan=2 colspan=2>Detection of stx2</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (8/8); 95% CI: 67.6-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>100% (8/8); 95% CI: 67.6-100%</td></tr><tr><td rowspan=2 colspan=2>Detection of E. coli O157</td><td rowspan=1 colspan=3>Expected Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>PanNATSTEC Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=3>100% (2/2); 95% CI: 34.2-100%</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement</td><td rowspan=1 colspan=3>100% (14/14); 95% CI: 78.5-100%</td></tr></table>

$9 5 \%$ CI: $9 5 \%$ score confidence interval

Table 17. Results from testing contrived, unpreserved stool specimens with the PanNAT STEC Test, stratified by target analyte/analyte combination   

<table><tr><td rowspan=1 colspan=1>Analytes</td><td rowspan=1 colspan=1>Positive Agreement</td></tr><tr><td rowspan=1 colspan=1>stx1 Only</td><td rowspan=1 colspan=1>100%(7/7)</td></tr><tr><td rowspan=1 colspan=1>stx2 Only</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>0157 Only</td><td rowspan=1 colspan=1>100%(1/1)</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2</td><td rowspan=1 colspan=1>100%(4/4)</td></tr><tr><td rowspan=1 colspan=1>stx1 + 0157</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx2 + 0157</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>stx1 + stx2 + 0157</td><td rowspan=1 colspan=1>100%(1/1)</td></tr></table>

Expected results based on genetic determinants known to be present within each strain

b. Clinical specificity: Refer to Section M(3)(a), above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable

4. Expected values/Reference range:

The performance of the PanNAT STEC Test was evaluated in a prospective Clinical Study conducted at six (6) sites in the US (Section M(3)(a)). The overall prevalence of stx1, stx2 and $E$ . coli O157 in stool specimens was $0 . 9 \%$ , $1 . 2 \%$ and $0 . 5 \%$ , respectively as determined by the PanNAT assay and $0 . 3 \%$ , $0 . 5 \%$ and $0 . 4 \%$ respectively as determined by the reference culture method. In Table 18, the prevalence of each analyte as determined by the PanNAT assay is stratified by the age of the subjects.

Table 18. Prevalence of stx1, $s t x 2$ and $E$ . coli O157 positive subjects observed in the prospective PanNAT STEC Test Clinical Study, stratified by age of the subjects   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=3>PanNAT STEC Test Positive(% Prevalence)</td></tr><tr><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>E. coli O157</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1 (0.7)</td><td rowspan=1 colspan=1>5 (3.3)</td><td rowspan=1 colspan=1>4 (2.7)</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>4 (2.1)</td><td rowspan=1 colspan=1>3 (1.6)</td><td rowspan=1 colspan=1>1 (0.5)</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>495</td><td rowspan=1 colspan=1>5 (1.1)</td><td rowspan=1 colspan=1>6 (1.2)</td><td rowspan=1 colspan=1>2 (0.4)</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>2 (0.5)</td><td rowspan=1 colspan=1>1 (0.2)</td><td rowspan=1 colspan=1>0 (0)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1280</td><td rowspan=1 colspan=1>12 (0.9)</td><td rowspan=1 colspan=1>15 (1.2)</td><td rowspan=1 colspan=1>7 (0.5)</td></tr></table>

N. Instrument Name: PanNAT System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No X

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification: Specimens are identified by scanning or typing in the applicable patient identification code.

4. Specimen Sampling and Handling:

Using the swab provided in the PanNAT STEC Sample transfer pack, the operator transfers a portion of the stool specimen to a Sample Buffer Tube, seals the tube using an Adaptor Cap and attaches the cap to the inlet port of the PanNAT STEC Test cartridge. The operator then inserts the cartridge with attached Sample Buffer Tube into the

PanNAT System for automated processing.

5. Calibration:

The PanNAT System is factory calibrated and does not require user calibration. A system check using Positive and Negative Controls is recommended if the instrument is moved after initial installation.

6. Quality Control: Refer to Section M(1)(c).

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable and the special controls for this device type under 21 CFR 866.3990.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.