BioFire Diagnostics, LLC   
Kristen Kanack   
Senior Vice President, Regulatory & Clinical Affairs   
515 Colorow Drive   
Salt Lake City, Utah 84108

Re: K180966 Trade/Device Name: FilmArray Pneumonia Panel Regulation Number: 21 CFR 866.3985 Regulation Name: Device to detect and identify microorganisms and associated resistance marker nucleic acids directly in respiratory specimens Regulatory Class: Class II Product Code: QDP

Dear Kristen Kanack:

The Food and Drug Administration (FDA) is sending this letter to notify you of an administrative change related to your previous substantial equivalence (SE) determination letter dated November 9th, 2018. Specifically, FDA is updating this SE Letter as an administrative correction. because FDA has created a new product code to better categorize your device technology.

Please note that the 510(k) submission was not re-reviewed. For questions regarding this letter please contact Kristian Roth, Ph.D., OHT7: Office of In Vitro Diagnostics and Radiological Health,

Sincerely, Digitally signed by Kristian M. Roth -   
Kristian M. Roth -S Date: 2021.06.24 08:49:06 -04'00'   
Kristian Roth, Ph.D.   
Deputy Director (Acting)   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics and Radiological Health   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

# November 9, 2018

BioFire Diagnostics, LLC   
Kristen Kanack   
Senior Vice President, Regulatory & Clinical Affairs   
515 Colorow Drive   
Salt Lake City, Utah 84108

Re: K180966 Trade/Device Name: FilmArray Pneumonia Panel Regulation Number: 21 CFR 866.3985 Regulation Name: Device to detect and identify microorganisms and associated resistance marker nucleic acids directly in respiratory specimens Regulatory Class: Class II Product Code: QBH Dated: April 12, 2018 Received: April 13, 2018

Dear Kristen Kanack:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's

requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/CombinationProducts/GuidanceRegulatoryInformation/ucm597488.htm); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice   
(https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/) and CDRH Learn   
(http://www.fda.gov/Training/CDRHLearn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (http://www.fda.gov/DICE) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Steven R. Gitterman -S for

Uwe Scherf, Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K180966

Device Name FilmArray Pneumonia Panel

Indications for Use (Describe)   
The FilmArray $\textsuperscript { \textregistered }$ Pneumonia Panel is a multiplexed nucleic acid test intended for use with FilmArray $^ \mathrm { \textregistered }$ , FilmArray $^ \mathrm { \textregistered }$ 2.0, or FilmArray $\textsuperscript { \textregistered }$ Torch systems for the simultaneous detection and identification of multiple respiratory viral and bacterial or endotracheal aspirates) or bronchoalveolar lavage (BAL)-like specimens (BAL or mini-BAL) obtained from individuals suspected of lower respiratory tract infection.

The following bacteria are reported semi-quantitatively with bins representing approximately $1 0 ^ { \mathord { \wedge } } 4 ~ 1 0 ^ { \mathord { \wedge } } 5$ , $1 0 \AA 6$ ,b $: \geq 1 0 ^ { \land } 7$ nucleic acid from these common bacteria within a specimen:

Bacteria reported with bins of $1 0 ^ { \mathord { \wedge } } 4 ~ 1 0 ^ { \mathord { \wedge } } 5$ $1 0 ^ { \wedge } 6$ , ${ \geq } 1 0 ^ { \land } 7$ copies/mL

Acinetobacter calcoaceticus-baumannii complex Enterobacter cloacae complex   
• Escherichia coli   
•Haemophilus influenzae   
Klebsiella aerogenes   
Klebsiella oxytoca   
• Klebsiella pneumoniae group   
Moraxella catarrhalis   
Proteus spp.   
Pseudomonas aeruginosa   
Serratia marcescens   
Staphylococcus aureus   
Streptococcus agalactiae   
Streptococcus pneumoniae   
Streptococcus pyogenes

The following atypical bacteria, viruses, and antimicrobial resistance genes are reported qualitatively:

Atypical Bacteria Chlamydia pneumoniae Legionella pneumophila Mycoplasma pneumoniae

Viruses   
Adenovirus   
Coronavirus   
Human Metapneumovirus   
Human Rhinovirus/Enterovirus   
Influenza A   
Influenza B   
Parainfluenza Virus   
Respiratory Syncytial Virus

# Antimicrobial Resistance Genes

CTX-M   
IMP   
KPC   
NDM   
OXA-48-like   
VIM   
•mecA/C and MREJ

Thetteu e he abbcalyie levidl u treatment, or other patient management decisions.

Ne $1 0 \textdegree$ c/mb.Detectionalyedes not uleut o-fection witotheanims; the agent()det y til o valcululoen adraym e he valt pat  poss respiratory tract infection.

D causative agent of pneumonia. Semi-quantitative bin (copies $/ \mathrm { m L }$ ) results generated by the FilmArray Pneumonia Panel are not equivalent to $\mathrm { C F U / m L }$ and do not consistently correlate with the quantity of bacterial analytes compared to $\mathrm { C F U / m L }$ F ul /L the relativebundancebacteri asdetermined by ulture (FU/mL).Clinical correlation s dvisedtodet significance of semi-quantitative bin (copies/mL) for clinical management.

The antiirobil resistance gene detected may r may not b associated with the agent() responsible or sese. Neuibn antimicrobials, as multiple mechanisms of antimicrobial resistance exist.

Antimicrobial resistance can occur via multiple mechanisms. A "Not Detected" result for a genetic marker of reuark venim C lus should beused in conjunction with culture results or determination  bacterial susceptibilityorresistane.

/ analysis) if differentiation is required.

genus, complex, or group results if desired, to identify bacterial pathogens present below the $1 0 ^ { \wedge } 4$ copies/mL bin if desired, and for antimicrobial susceptibility testing.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

Teurenohoctioationatverhours sec time o reviw instructions, searh existing data sources, gatherand maintain the data needed and coplet a review the collection finformation. Send comments regarding this burden estimateor anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 510(k) Summary BioFire Diagnostics, LLC

FilmArray® Pneumonia Panel

# Introduction:

According to the requirements of 21 CFR 807.92, the following information provides sufficient detail to understand the basis for a determination of substantial equivalence.

# Submitted by:

BioFire Diagnostics, LLC 515 Colorow Drive Salt Lake City, UT 84108

Telephone: 801-736-6354   
Facsimile: 801-588-0507

Contact: Kristen J. Kanack, ext. 1330

Date Submitted: April 12, 2018

# Device Name and Classification:

Trade Name: FilmArray Pneumonia Panel

Regulation Number: 21 CFR 866.3985

Classification Name: Device to detect and identify microorganisms and associated resistance marker nucleic acids directly in respiratory specimens

Product Code: QDP

# Predicate Device:

DEN170047 – Curetis Unyvero LRT Application

# Intended Use:

The FilmArray® Pneumonia Panel is a multiplexed nucleic acid test intended for use with FilmArray®, FilmArray® 2.0, or FilmArray® Torch systems for the simultaneous detection and identification of multiple respiratory viral and bacterial nucleic acids, as well as select antimicrobial resistance genes, in sputum-like specimens (induced or expectorated sputum, or endotracheal aspirates) or bronchoalveolar lavage (BAL)-like specimens (BAL or mini-BAL) obtained from individuals suspected of lower respiratory tract infection.

The following bacteria are reported semi-qualitatively with bins representing approximately $1 0 ^ { \wedge } 4 .$ , $1 0 ^ { \wedge } 5$ , $1 0 \AA 6$ , or ${ \ge } 1 0 ^ { \wedge } 7$ genomic copies of bacterial nucleic acid per milliliter (copies/mL) of specimen, to aid in estimating relative abundance of nucleic acid from these common bacteria within a specimen:

<table><tr><td colspan="3">Bacteria reported with bins of 10^4, 10^5, 10^6, or ≥10^7 copies/mL</td></tr><tr><td>Acinetobacter calcoaceticus-baumannii complex</td><td>Klebsiella oxytoca</td><td>Serratia marcescens</td></tr><tr><td>Enterobacter cloacae complex</td><td>Klebsiella pneumoniae group</td><td>Staphylococcus aureus</td></tr><tr><td>Haemophilus influenzae</td><td>Moraxella catarrhalis</td><td>Streptococcus agalactiae</td></tr><tr><td>Escherichia coli</td><td>Proteus spp.</td><td>Streptococcus pneumoniae</td></tr><tr><td>Klebsiella aerogenes</td><td>Pseudomonas aeruginosa</td><td>Streptococcus pyogenes</td></tr></table>

The following atypical bacteria, viruses, and antimicrobial resistance genes are reported qualitatively:

<table><tr><td colspan="3">Atypical Bacteria</td></tr><tr><td>Chlamydia pneumoniae</td><td>Legionella pneumophila</td><td>Mycoplasma pneumoniae</td></tr><tr><td></td><td>Viruses</td><td></td></tr><tr><td>Adenovirus Coronavirus Human Metapneumovirus</td><td>Human Rhinovirus/Enterovirus Influenza A Influenza B</td><td>Parainfluenza Virus Respiratory Syncytial Virus</td></tr><tr><td></td><td>Antimicrobial Resistance Genes</td><td></td></tr><tr><td>CTX-M IMP KPC</td><td>NDM OXA-48-like VIM</td><td>mecA/C and MREJ</td></tr></table>

The detection and identification of specific viral and bacterial nucleic acids, as well as the estimation of relative abundance of nucleic acid from common bacterial analytes, within specimens collected from individuals exhibiting signs and/or symptoms of a respiratory infection, aids in the diagnosis of lower respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, pathogens below the limit of detection, or in the case of bacterial analytes, present at levels below the lowest reported $1 0 \mathord { \sim } 4$ copies $/ \mathrm { m L }$ bin. Detection of analytes does not rule out coinfection with other organisms: the agent(s) detected by the FilmArray Pneumonia Panel may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible lower respiratory tract infection.

Detection of bacterial nucleic acid may be indicative of colonizing or normal respiratory flora and may not indicate the causative agent of pneumonia. Semi-quantitative Bin (copies/mL) results generated by the FilmArray Pneumonia Panel are not equivalent to $\mathrm { C F U / m L }$ and do not consistently correlate with the quantity of bacterial analytes compared to CFU/mL. For specimens with multiple bacteria detected, the relative abundance of nucleic acids (copies/mL) may not correlate with the relative abundance of bacteria as determined by culture $\mathrm { ( C F U / m L ) }$ ). Clinical correlation is advised to determine significance of semi-quantitative Bin (copies/mL) for clinical management.

The antimicrobial resistance gene detected may or may not be associated with the agent(s) responsible for disease. Negative results for these antimicrobial resistance gene assays do not indicate susceptibility to corresponding classes of antimicrobials, as multiple mechanisms of antimicrobial resistance exist.

Antimicrobial resistance can occur via multiple mechanisms. A “Not Detected” result for a genetic marker of antimicrobial resistance does not indicate susceptibility to associated antimicrobial drugs or drug classes. A “Detected” result for a genetic marker of antimicrobial resistance cannot be definitively linked to the microorganism(s) detected. Culture is required to obtain isolates for antimicrobial susceptibility testing, and FilmArray Pneumonia Panel results should be used in conjunction with culture results for determination of bacterial susceptibility or resistance.

Due to the genetic similarity between human rhinovirus and enterovirus, the test cannot reliably differentiate them. A positive Rhinovirus/Enterovirus result should be followed up using an alternate method (e.g., cell culture or sequence analysis) if differentiation is required.

Culture is required to identify pathogens not detected by the FilmArray Pneumonia Panel, to further speciate analytes in genus, complex, or group results if desired, to identify bacterial pathogens present below the $1 0 \mathord { \sim } 4$ copies/mL bin if desired, and for antimicrobial susceptibility testing.

# Device Description:

The FilmArray Pneumonia Panel is designed to simultaneously identify 26 potential pathogens of lower respiratory tract infection (LRTI) and associated antimicrobial resistance (AMR) genes from a sputumlike (induced and expectorated sputum as well as endotracheal aspirate, ETA) or bronchoalveolar lavage (BAL)-like (BAL and mini-BAL) specimens obtained from individuals with signs and/or symptoms of lower respiratory tract infection in a time ${ \bf \Pi } ^ { ' }$ hour) that allows the test results to be used in determining appropriate patient treatment and management. FilmArray Pneumonia Panel is compatible with BioFire Diagnostics’ (BioFire) PCR-based in vitro diagnostic FilmArray, FilmArray 2.0, and FilmArray Torch systems for infectious disease testing. A specific software module (i.e. FilmArray Pneumonia Panel pouch module) is used to perform FilmArray Pneumonia Panel testing on these systems.

<table><tr><td rowspan=1 colspan=1>Bacteria  Quantitative Results</td><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>blacTx-M (Extended spectrum beta-lactamase (ESBL))</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>blamp (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>blakpc (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>mecA/mecC and MREJ (Methicillin resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>blanDm (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>blaOxa48-like (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>blavm (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Bacteria (Atypical) - Qualitative Results</td><td rowspan=1 colspan=1>Parainfluenza Virus</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=3 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr></table>

A test is initiated by loading Hydration Solution into one port of the FilmArray pouch and a sputum-like or BAL-like sample mixed with the provided Sample Buffer into the other port of the FilmArray Pneumonia Panel pouch and placing it in a FilmArray instrument. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically-controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green $\textsuperscript { \textregistered }$ Plus, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The 2nd stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the 2nd stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the 2nd stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Substantial Equivalence:

The FilmArray Pneumonia Panel is substantially equivalent to the Curetis Unyvero LRT Application (DEN170047), which was recently cleared on April 03, 2018 and determined to be a Class II device under the classification product code QBH.

Table 1 compares the FilmArray Pneumonia Panel to the Curetis Unyvero LRT Application and outlines the similarities and differences between the two systems.

Table 1. Comparison of the FilmArray Pneumonia Panel and the Curetis Unyvero LRT Application)   

<table><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">New Device:FilmArray Pneumonia Panel</td><td colspan="1" rowspan="1">Predicate:Curetis Unyvero LRT Application(DEN170047)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Sputum-like (induced or expectoratedsputum, endotracheal aspirates) and BAL-like (BAL or mini-BAL) specimens fromindividuals (of all ages) with signs and/orsymptoms of lower respiratory tractinfection</td><td colspan="1" rowspan="1">Endotracheal aspirate specimens from adulthospitalized patients with suspected lowerrespiratory tract infections</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Bacteria: Acinetobacter calcoaceticus-baumannii complex, Enterobacter cloacaecomplex, Escherichia coli , Haemophilusinfluenzae, Klebsiella aerogenes, Klebsiellaoxytoca, Klebsiella pneumoniae group,Moraxella catarrhalis, Proteus spp.,Pseudomonas aeruginosa, Serratiamarcescens, Staphylococcus aureus,Streptococcus agalactiae, Streptococcuspneumoniae, Streptococcus pyogenesAtypical Bacteria: Chlamydia pneumoniae,Legionella pneumophila, MycoplasmapneumoniaeAntimicrobial Resistance Genes: CTX-M,IMP, KPC, mecA/C + MREJ, NDM,OXA48-like, VIMViruses: Adenovirus, Coronavirus, HumanMetapneumovirus, HumanRhinovirus/Enterovirus, Influenza A,Influenza B, Parainfluenza virus,Respiratory Syncytial virus</td><td colspan="1" rowspan="1">Bacteria: Acinetobacter spp., Citrobacterfreundii, Enterobacter cloacae complex,Escherichia coli, Haemophilus influenzae,Klebsiella oxytoca, Klebsiella pneumoniae,Klebsiella variicola, Moraxella catarrhalis,Morganella morganii, Proteus spp.,Pseudomonas aeruginosa, Serratiamarcescens, Staphylococcus aureus,Stenotrophomonas maltophilia,Streptococcus pneumoniaeAtypical Bacteria: Chlamydia pneumoniae,Legionella pneumophila, Mycoplasmapneumonia,Antimicrobial Resistance Genes: CTX-M,KPC, mecA, NDM, OXA-23, OXA-24,OXA-48, OXA-58, TEM, VIM</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">DNA only</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result types</td><td colspan="1" rowspan="1">Semi-quantitative detection with bin values(in 1-log rounded copies/mL bins) forBacteriaQualitative for Atypical Bacteria,Antimicrobial Resistance Genes, andViruses</td><td colspan="1" rowspan="1">Qualitative for all analytes</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray, FilmArray 2.0, or FilmArrayTorch</td><td colspan="1" rowspan="1">Unyvero System</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">About 1 hour</td><td colspan="1" rowspan="1">About 4-5 hours</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Room temperature</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td colspan="1" rowspan="1">One control is included in each cartridge tomonitor for the presence of PCR inhibitorsand enables the system to detect any failuresin the testing process.</td></tr><tr><td colspan="1" rowspan="1">UserComplexity</td><td colspan="1" rowspan="1">Moderate/Low</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Summary of Performance Data

# Clinical Performance

The clinical performance of the FilmArray Pneumonia Panel was established during a multi-center study conducted at eight geographically distinct U.S. study sites from October 2016 to July 2017. A total of 904 residual BAL (821 BAL and 83 mini-BAL) and 925 residual sputum ( 478 sputum and 447 ETA) specimens were acquired for the prospective clinical study. FilmArray Pneumonia Panel performance in BAL and mini-BAL was similar, as was performance in sputum and ETA; therefore these sample types are not stratified further in performance tables. A total of 58 BAL and 89 sputum specimens were excluded from the final data analysis. The most common reasons for specimen exclusion for both specimen types was reference culture unable to be performed, the specimen was found to not meet the inclusion criteria after the specimen had been enrolled, or the study site was unable to complete the Case Report Form (CRF). The final data set consisted of 846 BAL and 836 sputum specimens. Table 2 and Table 3 provide a summary of demographic information for the specimens included in the prospective study.

Table 2. Overall and Per Site Demographic Analysis for BAL Specimens   

<table><tr><td rowspan=1 colspan=11>BAL</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1a</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>Site 8</td></tr><tr><td rowspan=2 colspan=1>5</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>480 (57%)</td><td rowspan=1 colspan=1>80 (59%)</td><td rowspan=1 colspan=1>7 (54%)</td><td rowspan=1 colspan=1>138 (55%)</td><td rowspan=1 colspan=1>21 (68%)</td><td rowspan=1 colspan=1>75 (61%)</td><td rowspan=1 colspan=1>82 (52%)</td><td rowspan=1 colspan=1>27 (61%)</td><td rowspan=1 colspan=1>50 (55%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>366 (43%)</td><td rowspan=1 colspan=1>55 (41%)</td><td rowspan=1 colspan=1>6 (46%)</td><td rowspan=1 colspan=1>113 (45%)</td><td rowspan=1 colspan=1>10 (32%)</td><td rowspan=1 colspan=1>48 (39%)</td><td rowspan=1 colspan=1>76 (48%)</td><td rowspan=1 colspan=1>17 (39%)</td><td rowspan=1 colspan=1>41 (45%)</td></tr><tr><td rowspan=5 colspan=1>A</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>23 (3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>5 (38%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>15 (48%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>3 (2%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>6 - 17 years</td><td rowspan=1 colspan=1>27 (3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>8 (62%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>13 (42%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>4 (3%)</td><td rowspan=1 colspan=1>1 (2%)</td><td rowspan=1 colspan=1>1 (1%)</td></tr><tr><td rowspan=1 colspan=1>18 - 34 years</td><td rowspan=1 colspan=1>70 (8%)</td><td rowspan=1 colspan=1>18 (13%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>17 (7%)</td><td rowspan=1 colspan=1>3 (10%)</td><td rowspan=1 colspan=1>10 (8%)</td><td rowspan=1 colspan=1>10 (6%)</td><td rowspan=1 colspan=1>5 (11%)</td><td rowspan=1 colspan=1>7(8%)</td></tr><tr><td rowspan=1 colspan=1>35 - 65 years</td><td rowspan=1 colspan=1>470 (56%)</td><td rowspan=1 colspan=1>78 (58%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>152 (61%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>70 (57%)</td><td rowspan=1 colspan=1>88 (56%)</td><td rowspan=1 colspan=1>27 (61%)</td><td rowspan=1 colspan=1>55 (60%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>255 (30%)</td><td rowspan=1 colspan=1>38 (28%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>82 (33%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>43 (35%)</td><td rowspan=1 colspan=1>53 (34%)</td><td rowspan=1 colspan=1>11 (25%)</td><td rowspan=1 colspan=1>28 (31%)</td></tr><tr><td rowspan=3 colspan=1>Bea mn</td><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>666 (79%)</td><td rowspan=1 colspan=1>116 (86%)</td><td rowspan=1 colspan=1>12 (92%)</td><td rowspan=1 colspan=1>223 (89%)</td><td rowspan=1 colspan=1>9 (29%)</td><td rowspan=1 colspan=1>82 (67%)</td><td rowspan=1 colspan=1>118 (75%)</td><td rowspan=1 colspan=1>25 (57%)</td><td rowspan=1 colspan=1>81 (89%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>159 (19%)</td><td rowspan=1 colspan=1>18 (13%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>28 (11%)</td><td rowspan=1 colspan=1>22 (71%)</td><td rowspan=1 colspan=1>31 (25%)</td><td rowspan=1 colspan=1>39 (25%)</td><td rowspan=1 colspan=1>14 (32%)</td><td rowspan=1 colspan=1>7(8%)</td></tr><tr><td rowspan=1 colspan=1>Emergency</td><td rowspan=1 colspan=1>21 (2%)</td><td rowspan=1 colspan=1>1 (1%)</td><td rowspan=1 colspan=1>1 (8%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>10 (8%)</td><td rowspan=1 colspan=1>1 (1%)</td><td rowspan=1 colspan=1>5(11%)</td><td rowspan=1 colspan=1>3 (3%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>846</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>91</td></tr></table>

a Subject age could not be determined for one specimen from Site 1

Table 3. Overall and Per Site Demographic Analysis for Sputum Specimens   
BioFire Diagnostics $5 1 0 ( \mathrm { k } )$ FilmArray Pneumonia Panel   

<table><tr><td colspan="11" rowspan="1">Sputum</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td><td colspan="1" rowspan="1">Site 6</td><td colspan="1" rowspan="1">Site 7</td><td colspan="1" rowspan="1">Site 8</td></tr><tr><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">481 (58%)</td><td colspan="1" rowspan="1">66 (59%)</td><td colspan="1" rowspan="1">54 (54%)</td><td colspan="1" rowspan="1">136 (56%)</td><td colspan="1" rowspan="1">97 (61%)</td><td colspan="1" rowspan="1">14 (82%)</td><td colspan="1" rowspan="1">31 (53%)</td><td colspan="1" rowspan="1">34 (74%)</td><td colspan="1" rowspan="1">49 (47%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">355 (42%)</td><td colspan="1" rowspan="1">45 (41%)</td><td colspan="1" rowspan="1">46 (46%)</td><td colspan="1" rowspan="1">105 (44%)</td><td colspan="1" rowspan="1">61 (39%)</td><td colspan="1" rowspan="1">3 (18%)</td><td colspan="1" rowspan="1">28 (47%)</td><td colspan="1" rowspan="1">12 (26%)</td><td colspan="1" rowspan="1">55(53%)</td></tr><tr><td colspan="1" rowspan="5">BA</td><td colspan="1" rowspan="1">≤ 5 years</td><td colspan="1" rowspan="1">138 (17%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">49 (49%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">80 (51%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (4%)</td><td colspan="1" rowspan="1">7 (7%)</td></tr><tr><td colspan="1" rowspan="1">6 - 17 years</td><td colspan="1" rowspan="1">107 (13%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">35 (35%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">64 (41%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">2 (4%)</td><td colspan="1" rowspan="1">6 (6%)</td></tr><tr><td colspan="1" rowspan="1">18 - 34 years</td><td colspan="1" rowspan="1">86 (10%)</td><td colspan="1" rowspan="1">15 (14%)</td><td colspan="1" rowspan="1">16 (16%)</td><td colspan="1" rowspan="1">20 (8%)</td><td colspan="1" rowspan="1">13 (8%)</td><td colspan="1" rowspan="1">1 (6%)</td><td colspan="1" rowspan="1">6 (10%)</td><td colspan="1" rowspan="1">5 (11%)</td><td colspan="1" rowspan="1">10 (10%)</td></tr><tr><td colspan="1" rowspan="1">35 - 65 years</td><td colspan="1" rowspan="1">284 (34%)</td><td colspan="1" rowspan="1">51 (46%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">133 (55%)</td><td colspan="1" rowspan="1">1 (1%)</td><td colspan="1" rowspan="1">6 (35%)</td><td colspan="1" rowspan="1">36 (61%)</td><td colspan="1" rowspan="1">20 (43%)</td><td colspan="1" rowspan="1">37 (36%)</td></tr><tr><td colspan="1" rowspan="1">&gt; 65 years</td><td colspan="1" rowspan="1">221 (26%)</td><td colspan="1" rowspan="1">45 (41%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">88 (37%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">10 (59%)</td><td colspan="1" rowspan="1">17 (29%)</td><td colspan="1" rowspan="1">17 (37%)</td><td colspan="1" rowspan="1">44 (42%)</td></tr><tr><td colspan="1" rowspan="3">Ce in</td><td colspan="1" rowspan="1">Hospitalized</td><td colspan="1" rowspan="1">682 (82%)</td><td colspan="1" rowspan="1">106 (95%)</td><td colspan="1" rowspan="1">64 (64%)</td><td colspan="1" rowspan="1">219 (91%)</td><td colspan="1" rowspan="1">105 (66%)</td><td colspan="1" rowspan="1">12 (71%)</td><td colspan="1" rowspan="1">52 (88%)</td><td colspan="1" rowspan="1">23 (50%)</td><td colspan="1" rowspan="1">101 (97%)</td></tr><tr><td colspan="1" rowspan="1">Outpatient</td><td colspan="1" rowspan="1">73 (9%)</td><td colspan="1" rowspan="1">2 (2%)</td><td colspan="1" rowspan="1">14 (14%)</td><td colspan="1" rowspan="1">18 (7%)</td><td colspan="1" rowspan="1">24 (15%)</td><td colspan="1" rowspan="1">2 (12%)</td><td colspan="1" rowspan="1">5(8%)</td><td colspan="1" rowspan="1">7(15%)</td><td colspan="1" rowspan="1">1 (1%)</td></tr><tr><td colspan="1" rowspan="1">Emergency</td><td colspan="1" rowspan="1">81 (10%)</td><td colspan="1" rowspan="1">3 (3%)</td><td colspan="1" rowspan="1">22 (22%)</td><td colspan="1" rowspan="1">4 (2%)</td><td colspan="1" rowspan="1">29 (18%)</td><td colspan="1" rowspan="1">3 (18%)</td><td colspan="1" rowspan="1">2 (3%)</td><td colspan="1" rowspan="1">16 (35%)</td><td colspan="1" rowspan="1">2 (2%)</td></tr><tr><td colspan="10" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td><td colspan="1" rowspan="1">Site 6</td><td colspan="1" rowspan="1">Site 7</td><td colspan="1" rowspan="1">Site 8</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">836</td><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">241</td><td colspan="1" rowspan="1">158</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">104</td></tr></table>

All specimens were evaluated with the FilmArray Pneumonia Panel at clinical study sites. Refrigerated specimen aliquots were sent to a central reference laboratory for quantitative reference culture (qRefCx) and frozen specimen aliquots were also sent to BioFire for evaluation by polymerase chain reaction (PCR)/sequencing-based comparator methods.

The reference methods used in this study were as follows:

Bacterial analytes were compared to qRefCx to evaluate sensitivity and specificity, and the method was considered positive for the presence of the organism of interest if it was recovered in culture and enumerated at a level of 3162 $( 1 0 ^ { \setminus { 3 . 5 } ) }$ CFU/mL or greater.

Bacterial analytes were also evaluated by comparison to a single PCR assay for the organism of interest followed by a quantitative molecular assay that included sequencing (qMol) to assess FilmArray bin reporting performance. Atypical bacteria and viruses were compared to two conventional PCR assays followed by bidirectional sequencing. For specimens with an applicable bacteria detected by FilmArray, AMR genes were compared to a single PCR assay (from the specimen) followed by sequencing. A specimen was considered to be positive for an analyte if bi-directional sequencing data meeting predefined quality acceptance criteria matched organism-specific sequences deposited in the NCBI GenBank database (www.ncbi.nlm.nih.gov) with acceptable E-values. When two PCR comparator assays were used, any specimen that tested negative by both of the comparator assays was considered Negative.

Positive Percent Agreement (PPA) or Sensitivity for each analyte was calculated as $1 0 0 \% \textbf { X }$ $\mathrm { ( T P / \Omega / T P + }$ FN)). True positive (TP) indicates that both the FilmArray Pneumonia Panel and the comparator method had a positive result for this specific analyte, and false negative (FN) indicates that the FilmArray Pneumonia Panel result was negative while the comparator result was positive. Negative Percent Agreement (NPA) or Specificity was calculated as $100 \%$ x (TN / $\left( \mathrm { T N } + \mathrm { F P } \right)$ ). True negative (TN) indicates that both the FilmArray Pneumonia Panel and the comparator method had negative results, and a false positive (FP) indicates that the FilmArray Pneumonia Panel result was positive but the comparator result was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. Samples for which false positive and/or false negative results (i.e., discrepant results) were obtained when comparing the FilmArray Pneumonia Panel results to the comparator method results were further investigated. The discrepancy investigations were primarily performed as follows: for discrepancies between the Pneumonia Panel and reference culture for bacterial analytes, discrepancies were first examined to see if qRefCx or FilmArray had observed the analyte but reported it as “negative” or “Not Detected” because it was below the detection threshold. If this did not resolve the discrepancy, the results of qMol testing were considered. And if these methods still did not resolve the discrepancy, it was then investigated in the same manner as other analytes that used molecular comparator (i.e. using multiple additional molecular assays followed by sequence analysis). The results of SOC testing were also considered. The prospective clinical study results are summarized in Table 4 and Table 5 for BAL and sputum specimens, respectively.

Table 4. FilmArray Pneumonia Panel Clinical Performance Summary for BAL Specimensa   
BioFire Diagnostics $5 1 0 ( \mathrm { k } )$ FilmArray Pneumonia Panel   

<table><tr><td rowspan=1 colspan=4>BAL</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Reference</td><td rowspan=1 colspan=1>Sensitivity/PPA</td><td rowspan=1 colspan=1>Specificity/NPA</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Method</td><td rowspan=1 colspan=1>TP/TP +FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN +FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=8>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complexb</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>839/846</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.3-99.6%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>48.7-97.4%</td><td rowspan=1 colspan=1>832/839</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.3-99.6%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complexd</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>64.6-98.5%</td><td rowspan=1 colspan=1>822/834</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>97.5-99.2%</td></tr><tr><td rowspan=1 colspan=1>Escherichia colie</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100%</td><td rowspan=1 colspan=1>826/834</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>98.1-99.5%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>764/836</td><td rowspan=1 colspan=1>91.4</td><td rowspan=1 colspan=1>89.3-93.1%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>835/844</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.0-99.4%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae grouph</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>79.6-100%</td><td rowspan=1 colspan=1>819/831</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>97.5-99.2%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>817/846</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>95.1-97.6%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>837/841</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>98.8-99.8%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosak</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.4-100%</td><td rowspan=1 colspan=1>772/810</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>93.6-96.6%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td><td rowspan=1 colspan=1>834/840</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>98.5-99.7%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusm</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>88.9-99.6%</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>91.2</td><td rowspan=1 colspan=1>89.1-93.0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae&quot;</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>821/845</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>95.8-98.1%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>817/841</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>95.8-98.1%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenesp</td><td rowspan=1 colspan=1>qRefCx</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>838/844</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>98.5-99.7%</td></tr><tr><td rowspan=1 colspan=8>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae¶</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>844/845</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>833/833</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>841/842</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td><td rowspan=1 colspan=1>837/837</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>18/21</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>65.4-95.0%</td><td rowspan=1 colspan=1>810/823</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>97.3-99.1%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirust</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td><td rowspan=1 colspan=1>836/837</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus&quot;</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>52/54</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>87.5-99.0%</td><td rowspan=1 colspan=1>771/782</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>97.5-99.2%</td></tr><tr><td rowspan=1 colspan=1>Influenza Av</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>830/833</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.9-99.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza Bw</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>43.6-97.0%</td><td rowspan=1 colspan=1>837/838</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>16/18</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>67.2-96.9%</td><td rowspan=1 colspan=1>824/826</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.1-99.9%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>PCR/Seq</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>841/841</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5-100%</td></tr></table>

a The performance measures of sensitivity and specificity only refer to the bacterial analytes for which the gold-standard of qRefCx was used as the reference method. Performance measures of PPA and NPA refer to all other analytes, for which PCR/sequencing assays were used as comparator methods.

b Evidence of ACB complex was found in all seven FP specimens; one was enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx and six were detected by qMol.

$^ \mathrm { ~ c ~ } K .$ . aerogenes was identified in the single FN specimen in SOC culture. Evidence of $K$ . aerogenes was found in all seven FP specimens; four were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx and three were detected by qMol.

$^ { \mathrm { ~ d ~ } } E$ . cloacae complex was observed in the single FN specimen below the $1 0 { \sim } 4$ bin by the FilmArray Pneumonia Panel. Evidence of $E$ . cloacae complex was found in all $1 2 \mathrm { F P }$ specimens; six were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, five were detected by qMol, and one was detected using an additional molecular method.

e Evidence of $E$ . coli was found in all eight FP specimens; six were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx and two were detected by qMol.

Evidence of $H .$ . influenzae was found in all $7 2 \mathrm { F P }$ specimens; seven were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 56 were detected by qMol, eight were detected using an additional molecular method, and one was identified in SOC culture.

g Evidence of $K$ . oxytoca was found in all nine FP specimens; three were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, five were detected by qMol, and one was detected using an additional molecular method.

h Evidence of $K .$ pneumoniae group was found in all $1 2 \ \mathrm { F P }$ specimens; seven were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, four were detected by qMol, and one was detected using an additional molecular method.

I Evidence of $M$ . catarrhalis was found in all $2 9 \mathrm { F P }$ specimens; two were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 25 were detected by qMol, and two were detected using an additional molecular method.

Evidence of Proteus spp. was found in all four FP specimens; three were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx and one was detected by qMol.

k Evidence of P. aeruginosa was found in all 38 FP specimens; 19 were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 16 were detected by qMol, and thre were detected using an additional molecular method.

l Evidence of S. marcescens was found in all six FP specimens; four were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx and two were detected by qMol.

$^ { \mathrm { ~ m ~ } } S .$ . aureus was detected in the single FN specimen using an additional molecular method. Evidence of S. aureus was found in 69/70 FP specimens; 29 were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 30 were detected by qMol, eight were detected using an additional molecular method, and two were identified in SOC culture.

n Evidence of S. agalactiae was found in all $2 4 \ \mathrm { F P }$ specimens; seven were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 13 were detected by qMol, and four were detected using an additional molecular method.

o Evidence of S. pneumoniae was found in all $2 4 ~ \mathrm { F P }$ specimens; five were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 18 were detected by qMol, and one was detected using an additional molecular method.

p Evidence of $S .$ . pyogenes was found in all six FP specimens; two were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, three were detected by qMol, and one was detected using an additional molecular method.

q The single FP specimen was negative for $C .$ . pneumoniae when tested with additional molecular methods during discrepancy investigation.

r The single FP specimen was negative for $M .$ . pneumoniae when tested with additional molecular methods during discrepancy investigation.

was detected in $2 / 3 \ \mathrm { F N }$ and $8 / 1 3 \ \mathrm { F P }$ specimens using an additional molecular method.

The single FP specimen was negative for hMPV when tested with additional molecular methods during discrepancy investigation.

u HRV/EV was detected in both FN specimens using an additional molecular method. HRV/EV was detected in 8/11 FP specimens during discrepancy investigation; seven were detected using an additional molecular method and one was detected upon FilmArray Pneumonia Panel retest.

v FluA was detected in $2 / 3 \ \mathrm { F P }$ specimens using an additional molecular method.

w FluB was detected in the single FN specimen upon FilmArray Pneumonia Panel retest. FluB was detected in the single FP specimen using an additional molecular method.

x PIV was detected in both FN and both FP specimens using an additional molecular method.

Table 5. FilmArray Pneumonia Panel Clinical Performance Summary for Sputum Specimensa   
BioFire Diagnostics 510(k) FilmArray Pneumonia Panel   

<table><tr><td colspan="8" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">ReferenceMMethod</td><td colspan="3" rowspan="1">Sensitivity/PPA</td><td colspan="3" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP +FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/TN +FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumannii complexb</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9</td><td colspan="1" rowspan="1">62.3-98.4%</td><td colspan="1" rowspan="1">807/825</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">96.6-98.6%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">30.1-95.4%</td><td colspan="1" rowspan="1">823/832</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">98.0-99.4%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5%</td><td colspan="1" rowspan="1">803/824</td><td colspan="1" rowspan="1">97.5</td><td colspan="1" rowspan="1">96.1-98.3%</td></tr><tr><td colspan="1" rowspan="1">Escherichia colie</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3%</td><td colspan="1" rowspan="1">787/812</td><td colspan="1" rowspan="1">96.9</td><td colspan="1" rowspan="1">95.5-97.9%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">67.2-96.9%</td><td colspan="1" rowspan="1">727/818</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">86.5-90.9%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">817/827</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">97.8-99.3%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae grouph</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">21/23</td><td colspan="1" rowspan="1">91.3</td><td colspan="1" rowspan="1">73.2-97.6%</td><td colspan="1" rowspan="1">769/813</td><td colspan="1" rowspan="1">94.6</td><td colspan="1" rowspan="1">92.8-95.9%</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">761/831</td><td colspan="1" rowspan="1">91.6</td><td colspan="1" rowspan="1">89.5-93.3%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.j</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td><td colspan="1" rowspan="1">813/821</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.1-99.5%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosak</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">103/106</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">92.0-99.0%</td><td colspan="1" rowspan="1">673/730</td><td colspan="1" rowspan="1">92.2</td><td colspan="1" rowspan="1">90.0-93.9%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">26/27</td><td colspan="1" rowspan="1">96.3</td><td colspan="1" rowspan="1">81.7-99.3%</td><td colspan="1" rowspan="1">782/809</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">95.2-97.7%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusm</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">111/112</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">95.1-99.8%</td><td colspan="1" rowspan="1">631/724</td><td colspan="1" rowspan="1">87.2</td><td colspan="1" rowspan="1">84.5-89.4%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae"</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">793/827</td><td colspan="1" rowspan="1">95.9</td><td colspan="1" rowspan="1">94.3-97.0%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">785/820</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">94.1-96.9%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenesp</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">825/830</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.6-99.7%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Atypical Bacteria</td><td colspan="1" rowspan="1">—</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">835/835</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila¶</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">826/826</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">52.9-97.8%</td><td colspan="1" rowspan="1">827/827</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenoviruss</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">13/17</td><td colspan="1" rowspan="1">76.5</td><td colspan="1" rowspan="1">52.7-90.4%</td><td colspan="1" rowspan="1">815/817</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Coronavirust</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">28/32</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">71.9-95.0%</td><td colspan="1" rowspan="1">796/802</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.4-99.7%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus"</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">20/21</td><td colspan="1" rowspan="1">95.2</td><td colspan="1" rowspan="1">77.3-99.2%</td><td colspan="1" rowspan="1">812/813</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirusy</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.2-100%</td><td colspan="1" rowspan="1">717/730</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">97.0-99.0%</td></tr><tr><td colspan="1" rowspan="1">Influenza Aw</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td><td colspan="1" rowspan="1">819/822</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.9-99.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza Bx</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td><td colspan="1" rowspan="1">821/823</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">TP/TP +FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN +FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virusy</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">28/29</td><td colspan="1" rowspan="1">96.6</td><td colspan="1" rowspan="1">82.8-99.4%</td><td colspan="1" rowspan="1">804/806</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus2</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.8-100%</td><td colspan="1" rowspan="1">787/791</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">98.7-99.8%</td></tr></table>

a The performance measures of sensitivity and specificity only refer to the bacterial analytes for which the gold-standard of qRefCx was used as the referenc method. Performance measures of PPA and NPA refer to all other analytes, for which PCR/sequencing assays were used as comparator methods.

b The isolate recovered from the single FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Pseudomonas fluorescen during discrepancy investigation. Evidence of ACB complex was found in all $1 8 \ \mathrm { F P }$ specimens; 15 were detected by $\mathbf { q M o l }$ , two were detected using a additional molecular method, and one was identified in SOC culture.

c The isolate recovered from the single FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Hafnia paralvei during discrepancy investigation. Evidence of $K$ . aerogenes was found in all nine FP specimens; three were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, five were detected by qMol, and one was detected using an additional molecular method.

$^ { \mathrm { d } } E$ . cloacae complex was detected in the single FN specimen using an additional molecular method. Evidence of $E$ . cloacae complex was found in all 21 FP specimens; four were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 16 were detected by qMol, and one was detected using an additional molecular method.

$^ \mathrm { ~ c ~ } E .$ . coli was observed in the single FN specimen below the $1 0 { \sim } 4$ bin by the FilmArray Pneumonia Panel. Evidence of $E$ . coli was found in all $2 5 \ \mathrm { F P }$ specimens; six were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 14 were detected by qMol, and five were detected using an additional molecular method.

$^ { \mathrm { f } } H ,$ influenzae was detected in $1 / 2 \ \mathrm { F N }$ specimens by qMol. The isolate recovered from the other FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Haemophilus haemolyticus during discrepancy investigation. Evidence of $H .$ influenzae was found in all 91 FP specimens; four were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 78 were detected by qMol, seven were detected using an additional molecular method, and two were identified in SOC culture.

g Evidence of $K$ . oxytoca was found in all $1 0 \mathrm { F P }$ specimens; three were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, five were detected by qMol, and two were detected using an additional molecular method.

${ ^ { \mathrm { ~ h ~ } } K } .$ . pneumoniae group was detected in $1 / 2 \ \mathrm { F N }$ specimens by qMol. The other FN appeared to be a result of a specimen swap at the central reference laboratory. Evidence of $K$ . pneumoniae group was found in 43/44 FP specimens; 15 were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 21 were detected by qMol, and seven were detected using an additional molecular method.

Evidence of M. catarrhalis was found in all ${ } ^ { 7 0 } \mathrm { F P }$ specimens; one was enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 63 were detected by qMol, five were detected using an additional molecular method, and one was identified in SOC culture.

j Evidence of Proteus spp. was found in all eight FP specimens; two were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, four were detected by qMol, and two were detected using an additional molecular method.

$^ { \mathrm { ~ k ~ } } P .$ . aeruginosa was observed in $1 / 3 \ \mathrm { F N }$ specimens below the $1 0 \mathord { \sim } 4$ bin by the FilmArray Pneumonia Panel. The isolates recovered from the other two FN specimens were misidentified by qRefCx; molecular testing of the isolates identified one as Pseudomonas denitrificans and the other as Pseudomonas fluorescens during discrepancy investigation. Evidence of $P$ . aeruginosa was found in all 57 FP specimens; 21 were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 33 were detected by qMol, two were detected using an additional molecular method, and one was identified in SOC culture.

$^ { 1 } S .$ marcescens was observed in the single FN specimen below the $1 0 { \sim } 4$ bin by the FilmArray Pneumonia Panel. Evidence of S. marcescens was found in 26/27 FP specimens; seven were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 16 were detected by qMol, and three were detected using an additiona molecular method.

$^ { \mathrm { ~ m ~ } } S .$ aureus was observed in the single FN specimen below the $1 0 \mathord { \sim } 4$ bin by the FilmArray Pneumonia Panel. Evidence of $S _ { \cdot }$ . aureus was found in all 93 FP specimens; 43 were enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx, 43 were detected by qMol, three were detected using an additional molecular method, and four were identified in SOC culture.

n Evidence of S. agalactiae was found in all 34 FP specimens; five were enumerated below $1 0 { \sim } 3 . 5$ CFU/mL by qRefCx, 24 were detected by qMol, and fi were detected using an additional molecular method

Evidence of S. pneumoniae was found in all $3 5 \mathrm { F P }$ specimens; one was enumerated below $1 0 ^ { \wedge } 3 . 5 \mathrm { C F U / m L }$ by qRefCx and 34 were detected by qMol.

p Evidence of S. pyogenes was found in all five FP specimens; four were detected by qMol and one was detected using an additional molecular method.

$^ { \mathrm { q } } L$ . pneumophila was detected in the single FN specimen using an additional molecular method.

The single FN specimen was negative for M. pneumoniae when tested during discrepancy investigatio s AdV was detected in all four FN and 1/2 FP specimens using an additional molecular method.

${ } ^ { \mathfrak { t } } \mathbf { C o V }$ was detected in all four FN and $3 / 6 \mathrm { F P }$ specimens using an additional molecular method.

u hMPV was detected in the single FN specimen using an additional molecular method. The single FP specimen was negative for hMPV when tested during discrepancy investigation.

v HRV/EV was detected in 12/13 FP specimens during discrepancy investigation; 11 were detected using an additional molecular method and one was detected upon FilmArray Pneumonia Panel retest.

w FluA was detected in all three FP specimens using an additional molecular method.

x Both FP specimens were negative for FluB when tested with additional molecular methods during discrepancy investigation.

PIV was detected in the single FN and 1/2 FP specimens using an additional molecular method.

BioFire Diagnostics $5 1 0 ( \mathrm { k } )$ FilmArray Pneumonia Panel

A total of 156 BAL specimens and 295 sputum specimens received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel and reported results for CTX-M, IMP, KPC, NDM, OXA-48-like, and VIM. A total of 116 BAL specimens and 204 sputum specimens received a FilmArray Pneumonia Panel Staphylococcus aureus Detected result and reported results for mecA/C and MREJ. Performance of the Pneumonia Panel AMR gene assays was calculated by comparing results of qMol direct from these specimens and is shown in Table 6 (five BAL and four sputum specimens were excluded from qMol analysis due to invalid comparator results).

Table 6. FilmArray Pneumonia Panel Clinical Performance Summary – AMR Genes (comparator method: qMol direct from specimen)a   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=6>BAL</td><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP +FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN +FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TP/(TP +FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN +FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>CTX-Mb</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>48.7-97.4%</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4-100%</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>49.0-94.3%</td><td rowspan=1 colspan=1>280/281</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>KPCc</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>148/149</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.3-99.9%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>284/284</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>mecA/CandMREJd</td><td rowspan=1 colspan=1>40/45</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>76.5-95.2%</td><td rowspan=1 colspan=1>64/70</td><td rowspan=1 colspan=1>91.4</td><td rowspan=1 colspan=1>82.5-96.0%</td><td rowspan=1 colspan=1>94/98</td><td rowspan=1 colspan=1>95.9</td><td rowspan=1 colspan=1>90.0-98.4%</td><td rowspan=1 colspan=1>91/104</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>79.8-92.5%</td></tr><tr><td rowspan=1 colspan=1>NDMe</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>149/150</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.3-99.9%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>289/290</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.1-99.9%</td></tr></table>

a Performance in this summary table is calculated when any applicable organism is detected in the sample.

d Evidence of mecA/C and/or SCCmec cassette genetic elements was found in all five FN BAL and all four FN sputum specimens by an additional molecular method; three of these also had a MRSA isolate recovered via qRefCx or SOC culture. Evidence of mecA $\scriptstyle { \mathcal { C } }$ and/or SCCmec cassette genetic elements was found in 5/6 FP BAL and all $1 3 \mathrm { \ F P }$ sputum specimens; nine had a MRSA isolate recovered via qRefCx or SOC culture, and nine additional specimens had evidence of mecA/C and/or SCCmec cassette genetic elements by an additional molecular method.

e NDM was detected in the single FN BAL specimen using an additional molecular method; P. aeruginosa was recovered from the specimen and was resistant to carbapenems but carried only KPC. The single FP BAL specimen was negative for NDM when tested with additional molecular methods during discrepancy investigation.

f The single FP sputum specimen was negative for VIM when tested with additional molecular methods during discrepancy investigation.

qRefCx isolated one or more applicable gram-negative bacteria from 127 of the 156 BAL specimens and 230 of the 295 sputum specimens that received a FilmArray Pneumonia Panel Detected result for an applicable gram-negative bacterium. Additionally, qRefCx isolated S. aureus from 75 of 116 BAL specimens and 154 of 204 sputum specimens that received a FilmArray Pneumonia Panel Staphylococcus aureus Detected result. The method used to assess correlation of the CTX-M, IMP, KPC, NDM, OXA-48-like, and VIM results (Table 7 and Table 8) or the mecA/C and MREJ results (Error! Reference source not found. and Error! Reference source not found.) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 7. CTX-M, IMP, KPC, NDM, and VIM Performance Table (PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=14>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result(Filmarray)</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>CTX-M</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(any resistance gene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteriaDetected)a</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>121/123(98.4%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>127/127(1000%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>124/126(98.4%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>127/127(100%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>127/1271000%)</td><td rowspan=1 colspan=1>5/5(100%[56.6-100%]</td><td rowspan=1 colspan=1>118/122(6.7%)[91.9-98.7%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=2 colspan=1>12</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>8/8(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>12/12(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>12/12(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>12/12</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>8/8(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>77(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>77(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>77(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>7/7(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>$00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>42/43(97.7%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>43/43(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>43/43(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>43/43(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>43/43(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>42/43(97.7%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(10%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>6/6(10%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>27/28(96.4%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28(100%)</td><td rowspan=1 colspan=1>1/1c(100%)</td><td rowspan=1 colspan=1>25/27d(92.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28(100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>24/27(88.9%)</td></tr></table>

a An additional nine specimens had no applicable bacteria detected by FilmArray, but had one or more applicable bacteria isolated by qRefCx; no resistance markers were identified in the cultured isolate(s) by PCR/seq from these specimens b One specimen E. cloacae complex and K. pneumoniae group detected by FilmArray (E. cloacae complex isolated by qRefCx; CTX-M was not identified in this isolate by PCR/seq) $^ \mathrm { c } \mathrm { E } ,$ cloacae complex and P. aeruginosa detected by FilmArray and isolated by qRefCx (KPC identified from the E. cloacae isolate by PCR/seq) d One specimen A. calcoaceticus-baumannii complex and K. pneumoniae group detected by FilmArray (A. calcoaceticus-baumannii complex isolated by qRefCx; KPC was not identified in this isolate by PCR/seq); one specimen Proteus spp. and P. aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx; KPC was not identified in this isolated by PCR/seq)

Table 8. CTX-M, IMP, KPC, NDM, and VIM Performance Table (PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan="2"></td><td rowspan="2"></td><td colspan="8"></td><td colspan="4"></td></tr><tr><td colspan="2">CTX-M</td><td colspan="2">IMP</td><td colspan="2">KPC</td><td colspan="2">NDM</td><td colspan="2">VIM</td><td colspan="2">Overall (any resistance gene)</td></tr><tr><td rowspan="2">Applicable Bacteria Result Overall</td><td rowspan="2">N</td><td colspan="2">PPA</td><td colspan="2">PPA</td><td colspan="2">PPA NPA</td><td colspan="2">PPA</td><td colspan="2">PPA NPA</td><td colspan="2">PPA NPA</td></tr><tr><td>3/4</td><td>NPA 221/226</td><td>1/1</td><td>NPA 229/229</td><td>5/6</td><td>223/224</td><td>0/0</td><td>NPA 230/230</td><td>1/1</td><td>229/229</td><td>9/11b</td><td>214/219</td></tr><tr><td>(any applicable bacteria Detected)a</td><td>230</td><td>(75.0%)</td><td>(977.8%)</td><td>(100%)</td><td>(100%)</td><td>(83.3%)</td><td>(99.6%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(81.8%) [52.3-94.9%]</td><td>(97.7%) [94.8-99.0%]</td></tr><tr><td>Acinetobacter calcoaceticus- baumannii complex</td><td>5c</td><td>0/0</td><td>5/5 (100%)</td><td>00</td><td>5/5 (100%)</td><td>0/0 (-)</td><td>5/5 (100%)</td><td>0/0 (-)</td><td>5/5 (100%)</td><td>00</td><td>5/5 (100%)</td><td>0/0</td><td>5/5</td></tr><tr><td>Enterobacter cloacae complex</td><td>7d</td><td>(-</td><td>7/7</td><td>00</td><td>7/7</td><td>00</td><td>7/7</td><td></td><td>7/7</td><td>00</td><td>7/7</td><td>(-) 0/0</td><td>(100%) 7/7</td></tr><tr><td></td><td></td><td>00 1/1</td><td>(100%) 9/9</td><td>0/0</td><td>(100%) 10/10</td><td>0/0</td><td>(100%) 10/10</td><td>00 0/0</td><td>(100%) 10/10</td><td>0/0</td><td>(100%) 10/10</td><td>(-) 1/1</td><td>(100%) 9/9</td></tr><tr><td>Echerichia coli</td><td>10</td><td>(100%)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td></tr><tr><td>Klebsiella aerogenes</td><td>3</td><td>0/0 (-</td><td>3/3 (100%)</td><td>0/0 (-)</td><td>3/3 (100%)</td><td>0/0 ()</td><td>3/3 (100%%)</td><td>0/0 (-)</td><td>3/3 (100%)</td><td>0/0 (-)</td><td>3/3 (100%)</td><td>0/0</td><td>3/3</td></tr><tr><td></td><td></td><td>0/0</td><td>4/4</td><td></td><td>4/4</td><td>0/0</td><td>4/4</td><td>0</td><td>4/4</td><td>00</td><td>4/4</td><td>(-) 00</td><td>(100%)</td></tr><tr><td>Klebsiella oxytoca</td><td>4</td><td></td><td>(100%)</td><td>00</td><td></td><td>()</td><td>(100%)</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4/4</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>(100%)</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td>(100%)</td><td></td><td></td><td></td><td>(100%)</td><td></td><td></td><td></td><td>(100%)</td></tr><tr><td></td><td></td><td>(-)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=14>Sputum</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>CTX-M</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(any resistance gene)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>b</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>(10% </td><td rowspan=1 colspan=1>00% </td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>202 </td><td rowspan=1 colspan=1>(10 </td><td rowspan=1 colspan=1>00% </td><td rowspan=1 colspan=1>$00 </td><td rowspan=1 colspan=1>202 </td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>202 </td><td rowspan=1 colspan=1>(10 %)</td><td rowspan=1 colspan=1>101%) )</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>0 </td><td rowspan=1 colspan=1> 5/67(97.0%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>68/68( (100%)</td><td rowspan=1 colspan=1>$0 </td><td rowspan=1 colspan=1>67/67(10%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>68/68(0 (1%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>68/68(0 (1%)</td><td rowspan=1 colspan=1>0/2(0%%</td><td rowspan=1 colspan=1>64/66(97.0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>14c</td><td rowspan=1 colspan=1>$7 </td><td rowspan=1 colspan=1>14/14(100%))</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(10% )</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(10% )</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>(10 )</td><td rowspan=1 colspan=1>130 </td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>88/91 (96.7%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>92/92(10%)</td><td rowspan=1 colspan=1>4/4h(10%)</td><td rowspan=1 colspan=1>87/88(98.9%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>92/92(100%)</td><td rowspan=1 colspan=1>1/1j(100%)</td><td rowspan=1 colspan=1>91/91(100%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>84/87(96.6%)</td><td rowspan=1 colspan=1></td></tr></table>

a An additional 19 specimens had no applicable bacteria detected by FilmArray, but had one or more applicable bacteria isolated by qRefCx; CTX-M was identified by $\mathrm { P C R / s e q }$ in one specimen (E. coli isolated by qRefCx), but no other resistance markers were identified in the cultured isolate(s) by PCR/seq from these specimens b One specimen had presence of dual AMR genes (KPC and VIM) c An A. calcoaceticus-baumannii isolate was not recovered by qRefCx for one specimen d An E. cloacae isolate was not recovered by qRefCx for one specimen e A S. marcescens isolate was not recovered by qRefCx for one specimen $^ \mathrm { f } \ E$ . coli and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (CTX-M identified in the E. coli isolate by PCR/seq) g One specimen A. calcoaceticus-baumannii complex, K. pneumoniae group, Proteus spp., and $P .$ . aeruginosa detected by FilmArray (K. pneumoniae group and $P$ . aeruginosa isolated by qRefCx; CTX-M was not identified in either of these isolates by PCR/seq); one specimen $E$ . coli, $K .$ . pneumoniae group, and $P$ . aeruginosa detected by FilmArray (E. coli and $P$ . aeruginosa isolated by qRefCx; CTX-M was not identified in either of these isolated by $\mathrm { P C R } / \mathrm { s e q } )$ ; one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray $P .$ . aeruginosa isolated by qRefCx; CTX-M was not identified in this isolate by PCR/seq) h One specimen $E$ . coli and $K .$ . pneumoniae group detected by FilmArray and isolated by qRefCx (KPC identified in the $K .$ . pneumoniae isolate by PCR/seq); one specimen P. aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (KPC identified in the S. marcescens isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, K. aerogenes, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, $K$ . pneumoniae, and $P$ . aeruginosa isolated by qRefCx; KPC identified in the K. pneumoniae isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, $K$ . pneumoniae group, Proteus spp., and $P$ . aeruginosa detected by FilmArray $K .$ . pneumoniae and $P$ . aeruginosa isolated by qRefCx; KPC identified in the K. pneumoniae isolate by PCR/seq) i One specimen K. pneumoniae group and $P$ . aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx; KPC was not identified in this isolate by PCR/seq) j One specimen A. calcoaceticus-baumannii complex, K. aerogenes, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, K. pneumoniae, and P. aeruginosa isolated by qRefCx; VIM identified in the P. aeruginosa isolate by PCR/seq)

RefCx isolated one or more applicable gram-negative bacteria from 79 of the 94 BAL specimens and 131 of the 196 sputum specimens that eceived a FilmArray Pneumonia Panel Detected result for an applicable gram-negative bacterium for OXA-48-like. The method used to assess correlation of the OXA-48-like results (Table 9 and Table 10) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 9. OXA-48-like Performance Table (PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.4-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100%</td></tr></table>

Table 10. OXA-48-like Performance Table (PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>79.6-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>131/131</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.2-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td></tr></table>

qRefCx isolated S. aureus from 75 of 116 BAL specimens and 154 of 204 sputum specimens that received a FilmArray Pneumonia Panel Staphylococcus aureus Detected result. The method used to assess correlation of the mecA/C and MREJ results (Table 11 and Table 12) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 11. mecA/C and MREJ 3x3 Performance Table (qRefCx & PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=6>BAL</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureusPCR/seq: mecA/C</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>730</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>799</td><td rowspan=1 colspan=1>846</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>76.4-99.1%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>71.9-96.1%</td></tr><tr><td rowspan=1 colspan=3>Org-</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>91.2%</td><td rowspan=1 colspan=1>89.1-93.0%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>799/826</td><td rowspan=1 colspan=1>46/846</td></tr><tr><td rowspan=1 colspan=1>(95.0%)</td><td rowspan=1 colspan=1>(96.7%)</td><td rowspan=1 colspan=1>(5.4%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>773/819</td><td rowspan=1 colspan=1>70/846</td></tr><tr><td rowspan=1 colspan=1>(88.9%)</td><td rowspan=1 colspan=1>(94.4%)</td><td rowspan=1 colspan=1>(8.3%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>116/846</td></tr><tr><td rowspan=1 colspan=1>(97.9%)</td><td rowspan=1 colspan=1>(91.2%)</td><td rowspan=1 colspan=1>(13.7%)</td></tr></table>

Table 12. mecA/C and MREJ 3x3 Performance Table (qRefCx & PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureusPCR/seq: mecA/C</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>836</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8-100%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>49/54</td><td rowspan=1 colspan=1>90.7%</td><td rowspan=1 colspan=1>80.1-96.0%</td></tr><tr><td rowspan=1 colspan=3>Org-</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>87.2%</td><td rowspan=1 colspan=1>84.5-89.4%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>729/778</td><td rowspan=1 colspan=1>107/836</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(93.7%)</td><td rowspan=1 colspan=1>(12.8%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>49/54</td><td rowspan=1 colspan=1>734/782</td><td rowspan=1 colspan=1>97/836</td></tr><tr><td rowspan=1 colspan=1>(90.7%)</td><td rowspan=1 colspan=1>(93.9%)</td><td rowspan=1 colspan=1>(11.6%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>111/112</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>204/836</td></tr><tr><td rowspan=1 colspan=1>(99.1%)</td><td rowspan=1 colspan=1>(87.2%)</td><td rowspan=1 colspan=1>(24.4%)</td></tr></table>

FilmArray Pneumonia Panel CTX-M reporting was also compared to the standard phenotypic extended spectrum $\beta$ -lactamase (ESBL) activity testing methods performed in conjunction with qRefCx. Standard phenotypic ESBL activity was only reported for $E .$ . coli and Klebsiella spp. by the central reference laboratory.

Of the 156 BAL specimens that received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel, 53 specimens received a Detected result for $E$ . coli, K. oxytoca, and/or $K .$ . pneumoniae; qRefCx isolated $E$ . coli, K. oxytoca, and/or $K .$ . pneumoniae from 43 of these specimens. Of the 295 sputum specimens that received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel, 114 specimens received a Detected result for $E$ . coli, $K .$ . oxytoca, and/or $K .$ . pneumoniae; qRefCx isolated $E$ . coli, K. oxytoca, and/or $K .$ . pneumoniae from 71 of these specimens. The correlation between FilmArray Pneumonia Panel reporting of CTX-M in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen are stratified by each applicable associated organism in Table 13 and Table 14Error! Reference source not found..

Table 13. CTX-M Performance Table (comparison to phenotypic AST methods for BAL specimens)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>37.6-96.4%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr></table>

Table 14. CTX-M Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>4/7</td><td rowspan=1 colspan=1>57.1</td><td rowspan=1 colspan=1>25.1-84.2%</td><td rowspan=1 colspan=1>63/64</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>91.7-99.7%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>20.8-93.9%</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>26/27</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>81.7-99.3%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>1/2a</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100%</td></tr></table>

a One specimen $E$ . coli and $K .$ . oxytoca detected by FilmArray and isolated by qRefCx (ESBL activity identified in the $E$ . coli isolate by qRefCx AST); one specimen $E$ . coli and K. pneumoniae group detected by FilmArray $E .$ coli isolated by qRefCx and ESBL activity identified by qRefCx AST).

The FilmArray Pneumonia Panel carbapenem resistance gene reporting was also compared to the standard phenotypic carbapenem susceptibility testing methods performed in conjunction with qRefCx. In accordance with current CLSI guidelines, standard phenotypic ertapenem susceptibility is not reported for $A$ . calcoaceticus-baumannii complex; therefore carbapenem susceptibility is only based on meropenem susceptibility for this organism. Resistance or intermediate-resistance to either ertapenem or meropenem constituted carbapenem resistance for this analysis. The correlation between FilmArray Pneumonia Panel reporting of the carbapenem resistance genes in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen are stratified by each applicable associated organism in Table 15 through Table 18.

Table 15. IMP, KPC, NDM, and VIM Performance Table (comparison to phenotypic AST methods for BAL specimens)   

<table><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>BAL</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>cYanmaGlowb</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(any carbapenem resistancegene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>126a</td><td rowspan=1 colspan=1>0/17(0%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>3/17(17.6%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>0/17(0%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>0/17(0%%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>3/17(17.6%)[6.2-41.0%]</td><td rowspan=1 colspan=1>109/109(10(1%))[96.6-100%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>9/9(10%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/0(</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>$00 </td><td rowspan=1 colspan=1>12/12(1000%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>12/12(10%)</td><td rowspan=1 colspan=1>$00 </td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>7/7(100%%)</td><td rowspan=1 colspan=1>0/0(</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-</td><td rowspan=1 colspan=1>7/7(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(</td><td rowspan=1 colspan=1>14/14(100%%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(10%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>012 </td><td rowspan=1 colspan=1>30/30(1(100%)</td><td rowspan=1 colspan=1>012 </td><td rowspan=1 colspan=1>30/30(1010%)</td><td rowspan=1 colspan=1>012 </td><td rowspan=1 colspan=1>30/30(10 (1%)</td><td rowspan=1 colspan=1>012 </td><td rowspan=1 colspan=1>30/30(1010%)</td><td rowspan=1 colspan=1>0/12(%</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>23/23(100%)</td><td rowspan=1 colspan=1>3/5(60.0%)</td><td rowspan=1 colspan=1>23/23(100%)</td><td rowspan=1 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>23/23(10(10%)</td><td rowspan=1 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>23/23(100%)</td><td rowspan=1 colspan=1>3/5b(60.0%)</td><td rowspan=1 colspan=1>23/23(100%)</td></tr></table>

The isolate recovered from one specimen (P. aeruginosa) did not yield valid AST results on the VITEK instrument

b Of the three specimens that were concordant: one specimen A. calcoaceticus-baumannii complex and $K .$ . pneumoniae group detected by FilmArray (A. calcoaceticus-baumannii isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by $\mathrm { P C R / s e q ) }$ ; one specimen $E$ . cloacae complex and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $E$ . cloacae isolate by qRefCx AST, and KPC identified in the $E .$ . cloacae isolate by $\mathrm { P C R } / \mathrm { s e q } )$ ; one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray ( $P .$ . aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by $\mathrm { P C R / s e q }$ ). Of the two specimens that were not concordant: one specimen A. calcoaceticus-baumannii complex and $P .$ . aeruginosa detected by FilmArray ( $P .$ . aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by $\mathrm { P C R / s e q ) }$ ; one specimen $E$ . cloacae complex and $K .$ . aerogenes detected by FilmArray ( $E .$ cloacae isolated by qRefCx, carbapenem resistance identified by qRefCx AST and KPC not identified in the isolate by $\mathrm { P C R } / \mathrm { s e q } )$ .

Table 16. IMP, KPC, NDM, and VIM Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=12>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(any carbapenem resistancegene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(an applicable bacteiaDetd</td><td rowspan=1 colspan=1>229a</td><td rowspan=1 colspan=1>0/35(% %)</td><td rowspan=1 colspan=1>194/194(00%)</td><td rowspan=1 colspan=1>6/35(17.1%)</td><td rowspan=1 colspan=1>193/194(99.5%)</td><td rowspan=1 colspan=1>0/35(%)</td><td rowspan=1 colspan=1>194/194(00%)</td><td rowspan=1 colspan=1>1/35(2.9%)</td><td rowspan=1 colspan=1>194/194(00%)</td><td rowspan=1 colspan=1>6/35b(17.1%)[8.1-32.7%]</td><td rowspan=1 colspan=1>193/194( 9.5%)9 7.1-9.9%</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>5c</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%</td><td rowspan=1 colspan=1>0/2(%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>7d</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(10%)</td><td rowspan=1 colspan=1>0/1(0%%</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/1(0%)%</td><td rowspan=1 colspan=1>6/6(10%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>3/3(100%%)</td><td rowspan=1 colspan=1>$00</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(</td><td rowspan=1 colspan=1>3/3(100%%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0/2(%)</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>0/2(0%%</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>0/2(0%%</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>19/19(100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00 </td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>0/16(0%)</td><td rowspan=1 colspan=1>51/51(10(10%)</td><td rowspan=1 colspan=1>01 </td><td rowspan=1 colspan=1>51/51(100%%)</td><td rowspan=1 colspan=1>0/16(%)</td><td rowspan=1 colspan=1>51/51(100%%)</td><td rowspan=1 colspan=1>0/16(%%</td><td rowspan=1 colspan=1>51/51(100%)</td><td rowspan=1 colspan=1>0/16(0%%</td><td rowspan=1 colspan=1>51/51(100%%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>14e</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%))</td><td rowspan=1 colspan=1>0/1(0%%</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%%)</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1% </td><td rowspan=1 colspan=1>4/13(30.8%)</td><td rowspan=1 colspan=1>78/(98.7%)</td><td rowspan=1 colspan=1>03 </td><td rowspan=1 colspan=1>79/(100%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td><td rowspan=1 colspan=1>79/79(10(1%)</td><td rowspan=1 colspan=1>4/13(30.8%)</td><td rowspan=1 colspan=1>78/79(98.7%)</td></tr></table>

a The isolate recovered from one specimen $P .$ . aeruginosa) did not yield valid AST results on the VITEK instrument b One specimen had presence of dual AMR genes (KPC and VIM) $^ \mathrm { c } _ { \mathrm { \tiny ~ A n } , A }$ . calcoaceticus-baumannii isolate was not recovered by qRefCx for one specimen $^ { \mathrm { ~ d ~ } } { \ A } { \ n } \ E$ . cloacae isolate was not recovered by qRefCx for one specimen e A S. marcescens isolate was not recovered by qRefCx for one specimen

Of the four specimens that were concordant: one specimen A. calcoaceticus-baumannii complex, K. aerogenes, K. pneumoniae group, and P. aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, K. pneumoniae, and P. aeruginosa isolated by qRefCx, carbapenem resistance identified in all three isolates by qRefCx AST, KPC identified in the K. pneumoniae isolate by PCR/seq, and VIM identified in the P. aeruginosa isolate by $\mathrm { P C R / s e q ) }$ ; one specimen A. calcoaceticus-baumannii complex, K. pneumoniae group, Proteus spp., and P. aeruginosa detected by FilmArray (K. pneumoniae and P. aeruginosa isolated by qRefCx, carbapenem resistance identified in the K. pneumoniae isolate by qRefCx AST, and KPC identified in the K. pneumoniae isolate by PCR/seq); one specimen E. coli and K. pneumoniae group detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the K. pneumoniae isolate by qRefCx AST and KPC identified in the isolate by PCR/seq); one specimen P. aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the S. marcescens isolate by qRefCx AST and KPC identified in the isolate by PCR/seq). Of the nine specimens that were not concordant: one specimen A. calcoaceticus-baumannii complex and Proteus spp. detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the A. calcoaceticus-baumannii isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen E. cloacae complex and P. aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by $\mathrm { P C R / s e q ) }$ ; one specimen E. coli and P. aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen K. pneumoniae group and P. aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the K. pneumoniae isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen Proteus spp. and P. aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen P. aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the S. marcescens isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen P. aeruginosa and S. marcescens detected by FilmArray (P. aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, E. coli, K. pneumoniae group, and P. aeruginosa detected by FilmArray (E. coli and P. aeruginosa isolated by qRefCx, carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST, and KPC not identified in either isolate by $\mathrm { P C R / s e q ) }$ ; one specimen A. calcoaceticus-baumannii complex, Proteus spp., P. aeruginosa, and S. marcescens detected by FilmArray (Proteus spp. and P. aeruginosa isolated by qRefCx; carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq).

Table 17. OXA-48-like Performance Table (comparison to phenotypic AST methods for BAL specimens)   
a E. cloacae complex and K. aerogenes detected by FilmArray (E. cloacae complex isolated by qRefCx and carbapenem resistance identified by qRefCx AST)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>77/77</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.2-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>79.6-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/1a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td></tr></table>

Table 18. OXA-48-like Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>121/121</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/3a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100%</td></tr></table>

a One specimen E. coli and K. pneumoniae group detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $\kappa$ . pneumoniae group isolate by qRefCx AST); one specimen K. aerogenes and $\kappa$ pneumoniae group detected by FilmArray $\boldsymbol { { \cal K } }$ pneumoniae group isolated by qRefCx and carbapenem resistance identified by qRefCx AST); one specimen $\kappa .$ . pneumoniae group and Proteus spp. detected by FilmArray (K. pneumoniae group isolated by qRefCx and carbapenem resistance identified by qRefCx AST)

The FilmArray Pneumonia Panel mecA/C and MREJ reporting was also compared to the standard phenotypic cefoxitin susceptibility testing methods performed in conjunction with qRefCx. The correlation between FilmArray Pneumonia Panel reporting of mecA/C and MREJ in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen is shown in Error! Reference source not found. and Table 19 and Table 20Error! Reference source not found..

Table 19. mecA/C and MREJ 3x3 Performance Table (qRefCx & phenotypic AST on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=6>BAL</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureusqRefCx Phenotypic AST: cefoxitin susceptibility</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>730</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>799</td><td rowspan=1 colspan=1>846</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ /Res+</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>75.4-99.1%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>24/28</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>68.5-94.3%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>91.2%</td><td rowspan=1 colspan=1>89.1-93.0%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>799/827</td><td rowspan=1 colspan=1>46/846</td></tr><tr><td rowspan=1 colspan=1>(94.7%)</td><td rowspan=1 colspan=1>(96.6%)</td><td rowspan=1 colspan=1>(5.4%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>24/28</td><td rowspan=1 colspan=1>772/818</td><td rowspan=1 colspan=1>70/846</td></tr><tr><td rowspan=1 colspan=1>(85.7%)</td><td rowspan=1 colspan=1>(94.4%)</td><td rowspan=1 colspan=1>(8.3%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>116/846</td></tr><tr><td rowspan=1 colspan=1>(97.9%)</td><td rowspan=1 colspan=1>(91.2%)</td><td rowspan=1 colspan=1>(13.7%)</td></tr></table>

Table 20. mecA/C and MREJ 3x3 Performance Table (qRefCx & phenotypic AST on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureusqRefCx Phenotypic AST: cefoxitin susceptibility</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>836</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1-99.7%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>48/52</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>81.8-97.0%</td></tr><tr><td rowspan=1 colspan=3>Org-</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>87.2%</td><td rowspan=1 colspan=1>84.5-89.4%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>728/776</td><td rowspan=1 colspan=1>107/836</td></tr><tr><td rowspan=1 colspan=1>(98.3%)</td><td rowspan=1 colspan=1>(93.8%)</td><td rowspan=1 colspan=1>(12.8%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>48/52</td><td rowspan=1 colspan=1>735/784</td><td rowspan=1 colspan=1>97/836</td></tr><tr><td rowspan=1 colspan=1>(92.3%)</td><td rowspan=1 colspan=1>(93.8%)</td><td rowspan=1 colspan=1>(11.6%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>111/112</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>204/836</td></tr><tr><td rowspan=1 colspan=1>(99.1%)</td><td rowspan=1 colspan=1>(87.2%)</td><td rowspan=1 colspan=1>(24.4%)</td></tr></table>

The FilmArray Pneumonia Panel bin performance compared to the quantitative molecular assay (qMol) comparator is shown for BAL (Table 21) and sputum (Table 22). The qMol values are broken into onelog ranges correlating to the reported semi-quantitative FilmArray Pneumonia Panel bins. The relationship between qMol quantitative bins in copies/mL and traditional culture quantification in $\mathrm { C F U / m L }$ is unknown.

Table 21. FilmArray Pneumonia Panel Overall bin performance for BAL Specimens (qMol)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qMol Binned Valuesa(copies/mL)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5</td><td rowspan=1 colspan=1>10^4.0</td><td rowspan=1 colspan=1>10^5.0</td><td rowspan=1 colspan=1>10^6.0</td><td rowspan=1 colspan=1>≥10^7.0</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>R(/do)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>12025</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>12067</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>12025/12082(99.5%)</td><td rowspan=1 colspan=1>48/109(44.0%)</td><td rowspan=1 colspan=1>57/116(49.1%)</td><td rowspan=1 colspan=1>40/103(38.8%)</td><td rowspan=1 colspan=1>112/130(86.2%)</td><td rowspan=2 colspan=1>12540</td></tr><tr><td rowspan=1 colspan=4>257/458 (56.1%)</td></tr></table>

a Shaded cells indicate results considered concordant between the FilmArray Pneumonia Panel and qMol.

Table 22. FilmArray Pneumonia Panel Overall bin performance for Sputum Specimens (qMol)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qMol Range of Values a(copies//mL)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5</td><td rowspan=1 colspan=1>&lt;10^4.0</td><td rowspan=1 colspan=1>&lt;10^5.0</td><td rowspan=1 colspan=1>&lt;10^6.0</td><td rowspan=1 colspan=1>≥10^7.0</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>R (T/!don)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>11392</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11498</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>197</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>174</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>337</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>11392/11489(99.2%)</td><td rowspan=1 colspan=1>87/209(41.6%)</td><td rowspan=1 colspan=1>104/202(51.5%)</td><td rowspan=1 colspan=1>88/184(47.8%)</td><td rowspan=1 colspan=1>288/336(67.9%)</td><td rowspan=2 colspan=1>12420</td></tr><tr><td rowspan=1 colspan=4>567/931 (60.9%)</td></tr></table>

a Shaded cells indicate results considered concordant between the FilmArray Pneumonia Panel and qMol.

The FilmArray Pneumonia Panel bin performance compared to qRefCx quantification and semiquantification values provided by the source lab is shown for BAL and sputum in Table 23 - Table 34. Data is shown for overall performance, as well as for some individual organisms. In these tables, the values reported by culture are broken into ranges. A FilmArray Pneumonia Panel bin result is considered concordant if the culture value is within $0 . 5 ~ \mathrm { l o g }$ of the bin boundary. For example, the $1 0 ^ { \wedge } 5$ FilmArray Pneumonia Panel bin $( 1 0 ^ { \land } 4 . 5 \ – 1 0 ^ { \land } 5 . 5 )$ is concordant with the culture range of $1 0 ^ { \mathord { \wedge } } 4 \mathord { - } 1 0 ^ { \mathord { \wedge } } 6$ CFU/mL.

Table 23. FilmArray Pneumonia Panel Overall bin performance for BAL Specimens (qRefCx)   
BioFire Diagnostics 510(k) FilmArray Pneumonia Panel   

<table><tr><td colspan="8" rowspan="1">BAL</td></tr><tr><td colspan="2" rowspan="1">qRefCx Range of Values(Predicted FilmArray Bin)</td><td colspan="1" rowspan="1">ND to&lt;10^3.5(ND)</td><td colspan="1" rowspan="1">10^3.5 to&lt;10^4.0(10^4)</td><td colspan="1" rowspan="1">10^4.0 to&lt;10^5.0(10^4 or 10^5)</td><td colspan="1" rowspan="1">10^5.0 to&lt;10^6.0(10^5 or 10^6)</td><td colspan="1" rowspan="1">10^6.0 to&lt;10^7.0(10^6 or 10^7)</td><td colspan="1" rowspan="1">≥10^7.0(≥10^7)</td></tr><tr><td colspan="1" rowspan="3">Fa1</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">12202</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">10^4</td><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">10^5</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">10^6</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">≥10^7</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="2" rowspan="2">% concordant</td><td colspan="1" rowspan="2">12202/12530(97.4%)</td><td colspan="1" rowspan="1">1/32(3.1%)</td><td colspan="1" rowspan="1">14/69(20.3%)</td><td colspan="1" rowspan="1">2/34(5.9%)</td><td colspan="1" rowspan="1">12/13(92.3%)</td><td colspan="1" rowspan="1">12/12(100%)</td></tr><tr><td colspan="5" rowspan="1">41/160 (25.6%)</td></tr></table>

Table 24. FilmArray Pneumonia Panel Overall bin performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to&lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to&lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to&lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to&lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>RmWr</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>11596</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>11596/12143(95.5%)</td><td rowspan=1 colspan=1>14/64(21.9%)</td><td rowspan=1 colspan=1>37/167(22.2%)</td><td rowspan=1 colspan=1>35/90(38.9%)</td><td rowspan=1 colspan=1>53/54(98.1%)</td><td rowspan=1 colspan=1>20/22(90.9%)</td></tr><tr><td rowspan=1 colspan=5>159/397 (40.1%)</td></tr></table>

Table 25. FilmArray Pneumonia Panel H. influenzae bin performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to&lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to&lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to&lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to&lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rmm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>764</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>764/836(91.4%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/1(100%)</td></tr><tr><td rowspan=1 colspan=5>1/10 (10.0%)</td></tr></table>

Table 26. FilmArray Pneumonia Panel H. influenzae bin performance for Sputum Specimens (qRefCx)   
BioFire Diagnostics 510(k) FilmArray Pneumonia Panel   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(2107)</td></tr><tr><td rowspan=5 colspan=1>Rm Wrm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>727</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>727/818(88.9%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/12(0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td></tr><tr><td rowspan=1 colspan=5>3/18 (16.7%)</td></tr></table>

Table 27. FilmArray Pneumonia Panel P. aeruginosa bin performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rrm Rm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>772</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>772/810(95.3%)</td><td rowspan=1 colspan=1>0/4(0.0%)</td><td rowspan=1 colspan=1>2/15(13.3%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td></tr><tr><td rowspan=1 colspan=5>7/36 (19.4%)</td></tr></table>

Table 28. FilmArray Pneumonia Panel P. aeruginosa bin performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>673</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>673/730(92.2%)</td><td rowspan=1 colspan=1>3/15(20.0%)</td><td rowspan=1 colspan=1>7/42(16.7%)</td><td rowspan=1 colspan=1>5/24(20.8%)</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=5>40/106 (37.7%)</td></tr></table>

Table 29.FilmArray Pneumonia Panel S. aureus bin performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>729/799(91.2%)</td><td rowspan=1 colspan=1>0/11(0.0%)</td><td rowspan=1 colspan=1>0/19(0.0%)</td><td rowspan=1 colspan=1>1/9(11.1%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td></tr><tr><td rowspan=1 colspan=5>9/47 (19.1%)</td></tr></table>

Table 30. FilmArray Pneumonia Panel S. aureus bin performance for Sputum Specimens (qRefCx)   
BioFire Diagnostics 510(k) FilmArray Pneumonia Panel   

<table><tr><td colspan="8" rowspan="1">Sputum</td></tr><tr><td colspan="2" rowspan="1">qRefCx Range of Values(Predicted FilmArray Bin)</td><td colspan="1" rowspan="1">ND to&lt;10^3.5(ND)</td><td colspan="1" rowspan="1">10^3.5 to &lt;10^4.0(10^4)</td><td colspan="1" rowspan="1">10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td colspan="1" rowspan="1">10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td colspan="1" rowspan="1">10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td colspan="1" rowspan="1">≥10^7.0(≥10^7)</td></tr><tr><td colspan="1" rowspan="4">Rw rm</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">631</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">10^4</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">10^5</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">10^6</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">≥10^7</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="2" rowspan="2">% concordant</td><td colspan="1" rowspan="2">631/724(87.2%)</td><td colspan="1" rowspan="1">1/18(5.6%)</td><td colspan="1" rowspan="1">11/45(24.4%)</td><td colspan="1" rowspan="1">13/28(46.4%)</td><td colspan="1" rowspan="1">12/13(92.3%)</td><td colspan="1" rowspan="1">7/8(87.5%)</td></tr><tr><td colspan="5" rowspan="1">44/112 (39.3%)</td></tr></table>

Table 31. FilmArray Pneumonia Panel S. pneumoniae bin performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rm m</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>817</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>817/841(97.1%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>0/0(-%)</td><td rowspan=1 colspan=1>0/0(-%)</td></tr><tr><td rowspan=1 colspan=5>2/5 (40%)</td></tr></table>

Table 32. FilmArray Pneumonia Panel S. pneumoniae bin performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range of Values(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;10^3.5(ND)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td></tr><tr><td rowspan=5 colspan=1>Rmrm</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>785</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>785/820(95.7%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td></tr><tr><td rowspan=1 colspan=5>9/16 (56.3%)</td></tr></table>

Table 33. FilmArray Pneumonia Panel Bin Performance Summary for BAL Specimens comparted to qRefCx   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=1 colspan=1>qRefCx Range of Values(Concordant FilmArray Bin)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/7(14.3%)</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/12(16.7%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/4(0%)</td><td rowspan=1 colspan=1>1/7(14.3%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/12(16.7%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/10(10.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/7(42.9%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/2(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>8/15</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

Table 34. FilmArray Pneumonia Panel Bin Performance Summary for Sputum Specimens comparted to qRefCx   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=1 colspan=1>qRefCx Range of Values(Concordant FilmArray Bin)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(10^4)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(60.0%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(53.3%)</td></tr><tr><td rowspan=2 colspan=1>Moraxella catarrhalis</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>0/0(-)</td></tr><tr><td rowspan=1 colspan=1>(-)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>3/5(60.0%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>0/4(0%)</td><td rowspan=1 colspan=1>2/15(13.3%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>7/36(19.4%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/6(16.7%)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>0/11(0.0%)</td><td rowspan=1 colspan=1>0/19(0%)</td><td rowspan=1 colspan=1>1/9(11.1%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>9/47(19.1%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/1(0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/5(40.0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=1 colspan=1>qRefCx Range of Values(Concordant FilmArray Bin)</td><td rowspan=1 colspan=1>10^3.5 to &lt;10^4.0(104)</td><td rowspan=1 colspan=1>10^4.0 to &lt;10^5.0(10^4 or 10^5)</td><td rowspan=1 colspan=1>10^5.0 to &lt;10^6.0(10^5 or 10^6)</td><td rowspan=1 colspan=1>10^6.0 to &lt;10^7.0(10^6 or 10^7)</td><td rowspan=1 colspan=1>≥10^7.0(≥10^7)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>1/5(20.0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/11(45.5%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>3/6(50.0%)</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>6/12(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td><td rowspan=1 colspan=1>2/5(40.0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>8/24(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/12(0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/18(16.7%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/4(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/9(44.4%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>3/6(50.0%)</td><td rowspan=1 colspan=1>2/7(28.6%)</td><td rowspan=1 colspan=1>5/7(71.4%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>12/23(52.2%)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/5(40.0%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>5/10(50.0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/15(53.3%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3/15(20.0%)</td><td rowspan=1 colspan=1>7/42(16.7%)</td><td rowspan=1 colspan=1>5/24(20.8%)</td><td rowspan=1 colspan=1>19/19(100%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>40/106(37.7%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/4(0%)</td><td rowspan=1 colspan=1>4/12(33.3%)</td><td rowspan=1 colspan=1>3/6(50.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>12/27(44.4%)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1/18(5.6%)</td><td rowspan=1 colspan=1>11/45(24.4%)</td><td rowspan=1 colspan=1>13/28(46.4%)</td><td rowspan=1 colspan=1>12/13(92.3%)</td><td rowspan=1 colspan=1>7/8(87.5%)</td><td rowspan=1 colspan=1>44/112(39.3%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>1/5(20.0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/9(11.1%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>9/16(56.3%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>3/6(50.0%)</td></tr></table>

A ‘rank order’ analysis was performed on polymicrobial specimens to compare the relative abundance of each analyte within a specimen as reported by qRefCx to the order reported by the FilmArray

Pneumonia Panel. In this analysis, there were 20 specimens with two or more organisms reported by qRefCx for BAL and 84 for sputum. In these specimens, the FilmArray Pneumonia Panel was in agreement with qRefCx for the most abundant organism $4 5 . 0 \%$ of the time (9/20) for BAL and $4 1 . 7 \%$ of the time (35/84) for sputum. For the second most abundant organism, the FilmArray Pneumonia Panel was in agreement with qRefCx $3 0 . 0 \%$ of the time (6/20) for BAL and $2 6 . 2 \%$ of the time (22/84) for sputum, and was in agreement for the third most abundant organism $7 . 7 \%$ of the time (1/13) for BAL and $3 . 8 \%$ of the time (2/52) for sputum.

Table 35. Concordance of Abundance in Polymicrobial Specimens (as compared to qRefCx)   

<table><tr><td rowspan=2 colspan=1>Ranking Performance</td><td rowspan=1 colspan=3>BAL</td><td rowspan=1 colspan=3>Sputum</td></tr><tr><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Most Abundant</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>41.7%</td></tr><tr><td rowspan=1 colspan=1>Second Most Abundant</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>26.2%</td></tr><tr><td rowspan=1 colspan=1>Third Most Abundant</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>3.8%</td></tr></table>

‘Rank concordance’ analysis was performed on polymicrobial specimens to determine the ability of the FilmArray Pneumonia Panel to measure relative abundance of the nucleic acid for an organism with respect to other organisms in the specimen, as compared to qRefCx. In this analysis, the detected organisms from individual polymicrobial specimens (110 BAL and 246 sputum) were ranked in descending order based on their quantification values from qRefCx. The rank determined by the FilmArray Pneumonia Panel bin result was compared to the qRefCx ranking. False positives results were always considered discordant (i.e. only exact rank matches are considered concordant).

Table 36. Concordance of Organism Ranking in Polymicrobials Specimens (as compared to qRefCx)   

<table><tr><td rowspan=2 colspan=1>Ranking Performance</td><td rowspan=1 colspan=3>BAL</td><td rowspan=1 colspan=3>Sputum</td></tr><tr><td rowspan=1 colspan=1>Concordant</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Concordant</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>28.0%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>64.3%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>36.0%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>46.2%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>35.9%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>19.1%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>16.1%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>44.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>44.4%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>29.4%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>34.9%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>5.1%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>25.0%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>68.0%</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>55.1%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>39.5%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>54.5%</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>41.8%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>31.6%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>28.6%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>17.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>18.9%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>40.0%</td></tr></table>

The overall success rate for initial specimen tests in the prospective study was $9 8 . 1 \%$ (1764/1798); 34 tests were unsuccessful (two due to an incomplete test and 32 due to control failures). Two tests (2/1798; $0 . 1 \% )$ did not complete on the initial run, resulting in an instrument success rate of $9 9 . 9 \%$ (1796/1798) for initial specimen tests. Both specimens were able to be retested and valid results were produced after a single retest.

Thirty-two (32) tests (32/1764; $1 . 8 \%$ ) did not produce valid pouch controls, resulting in a pouch control success rate of $9 8 . 2 \%$ (1764/1796) for completed runs in the initial specimen tests. Twenty-eight (28) of the 32 invalid specimens were able to be retested; 25 produced valid control results after a single retest, while the remaining three did not produce valid control results after retesting and were not able to be retested further due to insufficient specimen volume; four were not able to be retested at all due to insufficient specimen volume.Two additional studies were also conducted to demonstrate all aspects of clinical performance (see Testing of Preselected Archived Specimens and

Testing of Contrived Specimens below).

# Testing of Preselected Archived Specimens

Some of the analytes on the FilmArray Pneumonia Panel were of low prevalence and were not encountered in large enough numbers during the prospective study to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective specimens was performed at BioFire.

A total of 171 frozen archived specimens were received for testing from external laboratories. Eighteen (18) specimens were negatives (13 BAL and 5 sputum) and 153 specimens (139 BAL and 14 sputum) contained at least one analyte of interest. Twenty-two (22) specimens contained two or more analytes of interest. The set included specimens known to be positive for: Acinetobacter calcoaceticus-baumannii complex, Klebsiella aerogenes, Klebsiella oxytoca, Proteus spp., Serratia marcescens, Streptococcus pyogenes, Chlamydia pneumoniae, Legionella pneumophila, adenovirus, human metapneumovirus, influenza A, influenza B, parainfluenza virus (PIV), respiratory syncytial virus, various gram-negative bacteria with an extended-spectrum $\beta$ -lactamase (ESBL) phenotype, and various gram-negative bacteria with a carbapenem resistant phenotype.

Prior to testing with the FilmArray Pneumonia Panel, the composition/integrity of the specimens was first confirmed with confirmatory molecular methods. At the completion of testing, four specimens were excluded due to invalid confirmation tests; these specimens had insufficient volume for retesting. Results from the remaining 18 negative and 149 positive specimens (containing 173 analytes) are presented here.

The reported analyte (as determined by the source laboratory) was confirmed for 117 analytes (107 in BAL and 10 in sputum) of the expected positive results (117/173; $6 7 . 6 ~ \%$ ) of the expected positive

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

results (116/173; $6 7 . 1 \ \%$ ). More than three quarters of the unconfirmed analytes were specimens previously identified as positive for gram-negative bacteria exhibiting phenotypic ESBL or carbapenamase activity (44/57; $7 7 . 2 \% )$ ). This is expected since this phenotypic activity can be conferred by alternative mechanisms beyond the antibiotic resistance genes found on the FilmArray Pneumonia Panel. Specimens with unconfirmed (or unexpected) analytes were excluded from performance calculations for that particular analyte and FilmArray test results are reported separately.

The FilmArray Pneumonia Panel demonstrated positive percent agreement (PPA) of $100 \%$ with previous laboratory results for 11 of 14 analytes tested in BAL specimens (Table 37) and all of the analytes tested in sputum specimens (Table 38Error! Reference source not found.). The exceptions were Klebsiella aerogenes, Proteus spp. and RSV with a PPA of $5 0 . 0 \%$ , $8 0 . 0 \%$ , and $9 3 . 8 \%$ respectively, due to one false negative (FN) for each analyte. While PPA for most analytes was $100 \%$ , an insufficient number of specimens were tested for all but influenza A, parainfluenza virus, and respiratory syncytial virus in BAL. Contrived specimen testing was used to demonstrate performance for these analytes in an additional contrived study (see

Testing of Contrived Specimens). Negative percent agreement (NPA) was $100 \%$ for all analytes except parainfluenza virus in BAL and influenza A in sputum; however, NPA was more thoroughly evaluated in the prospective clinical study.

Table 37. FilmArray Pneumonia Panel Archived BAL Specimen Performance Data Summary   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Quantitative Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>37.6-96.4%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.6-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.3-100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.7-100%</td></tr><tr><td rowspan=1 colspan=7>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td></tr><tr><td rowspan=1 colspan=7>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.9-100%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.7-100%</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.5-100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100%</td><td rowspan=1 colspan=1>77/77</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.2-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>84.5-100%</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.7-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>86/86</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.7-100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td><td rowspan=1 colspan=1>68/69</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>92.2-99.7%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>15/16</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>71.7-98.9%</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1-100%</td></tr></table>

Table 38. FilmArray Pneumonia Panel Archived Sputum Specimen Performance Data Summary   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Quantitative Bacteria</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=7>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100%</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

# Testing of Contrived Specimens

A prospective clinical evaluation of the FilmArray Pneumonia Panel was performed during the 2016- 2017 respiratory infection season at several geographically diverse clinical laboratories. Over 1600 specimens were analyzed from subjects whose specimens were submitted for microbial evaluation of lower respiratory tract pathogens. In the prospective study, some analytes were of insufficient prevalence to adequately demonstrate system performance and additional archived, preselected positive specimens containing rare analytes were also tested. Several analytes were so rare that both prospective and archived testing efforts were insufficient to demonstrate system performance. In this study, contrived clinical specimens were created to evaluate the sensitivity and specificity of the FilmArray Pneumonia Panel assays for these rare analytes (Table 39Error! Reference source not found.).

Table 39. Contrived Clinical Specimen Analytes   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Matrix</td></tr><tr><td rowspan=1 colspan=1>BAL</td><td rowspan=1 colspan=1>Sputum</td></tr><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Antibiotic Resistance Markers</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr></table>

Contrived specimens $\scriptstyle ( \mathrm { N = 1 } 1 2 5$ ) were spiked using residual clinical samples that were pre-screened with the FilmArray Pneumonia Panel and found to be negative for the analytes of interest. Specimens were spiked with a variety of different isolates/strains for each organism at concentrations spanning observed ranges in clinical specimens. Different isolates of organisms were used from those used in analytical testing when possible. Samples positive for one analyte served as negatives for other analytes.

For the majority of analytes reported qualitatively, at least 25 of the contrived positive specimens had analyte concentrations at $2 \times$ the limit of detection (LoD), while the remaining specimens were tested at additional concentrations that spanned clinically observed ranges. The “clinically observed range” was based on data from previous FilmArray Pneumonia Panel positive test results (e.g. observations from the prospective or archived studies). If a clinically observed range could not be determined for a particular analyte, specimens were spiked at various factors of LoD. If the stock concentration of organism did not allow for spiking at the highest level, the highest achievable level was used. For bacteria reported with binned values, specimens were spiked at various concentrations starting just below and then spanning the reported levels (i.e. $1 0 ^ { \wedge } 3$ to ${ \ge } 1 0 ^ { \wedge } 7$ copies per milliliter (mL)).

Specimens were prepared and randomized at BioFire such that the analyte status of each contrived specimen was unknown to the users performing the testing. BAL and sputum specimens were analyzed separately; however, the preparation and testing for both matrices were identical. Contrived specimens were frozen, then distributed to prospective study sites and tested according to the prospective clinical study protocol alongside clinical (non-contrived) specimens.

The positive percent agreement (PPA) and negative percent agreement (NPA) for the FilmArray Pneumonia Panel assays were determined using standard binomial sampling statistics. In this study, a success was defined as agreement between the known composition of the contrived specimen and the FilmArray Pneumonia Panel result; i.e., a positive FilmArray Pneumonia Panel result for spiked samples (True Positive, TP) and a negative FilmArray Pneumonia Panel result for un-spiked samples (True Negative, TN).

The results of the 1125 specimens tested in this study are summarized in Error! Reference source not found. for BAL and Error! Reference source not found. for sputum below.

The majority of analytes in both specimen types met the performance goals of $90 \%$ PPA with an $80 \%$ lower bound of the $9 5 \%$ CI and $9 8 \%$ NPA with a $9 5 \%$ lower bound of the $9 5 \%$ CI. The exceptions being influenza A spiked into BAL and Klebsiella aerogenes spiked into BAL and sputum. Influenza A spiked in BAL demonstrated $8 6 \%$ PPA in part due to two missed detections at $0 . 2 \times \mathrm { L o D }$ and two additional missed detections at $2 \times \mathrm { L o D }$ from a strain that may have been under-quantified. However, FilmArray performance goals for influenza A in BAL were achieved in the archived study. Klebsiella aerogenes spiked in both sample types demonstrated $8 5 . 5 \%$ PPA in part due to five missed detections in BAL and four missed detections in sputum at a range of concentrations from a Klebsiella aerogenes strain that demonstrated poor reactivity with the FilmArray Pneumonia Panel.

Samples spiked with bacterial analytes just below the $1 0 ^ { \wedge } 4$ copies/mL binned value (i.e., near $1 . 0 0 \mathrm { E } { + 0 3 } $ ), and samples spiked with other analytes below their LoD (i.e., near $0 . 2 \times \mathrm { L o D } \mathrm { , }$ ), produced the expected unreliable detection.

Table 40. FilmArray Pneumonia Panel Performance of Contrived BAL Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Sensitivity/PPA</td><td rowspan=1 colspan=3>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=1>598/598</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenesa</td><td rowspan=1 colspan=1>47/55</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>73.8-92.4%</td><td rowspan=1 colspan=1>592/594</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.8-99.9%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>46/50</td><td rowspan=1 colspan=1>92.0</td><td rowspan=1 colspan=1>81.2-96.8%</td><td rowspan=1 colspan=1>604/604</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=1>603/603</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6%</td><td rowspan=1 colspan=1>604/604</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6%</td><td rowspan=1 colspan=1>597/597</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=7>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=1>604/604</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>599/599</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=1>603/604</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>50/53</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>84.6-98.1%</td><td rowspan=1 colspan=1>568/569</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.0-100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>597/598</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza Ab</td><td rowspan=1 colspan=1>43/50</td><td rowspan=1 colspan=1>86.0</td><td rowspan=1 colspan=1>73.8-93.0%</td><td rowspan=1 colspan=1>585/585</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza Bc</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=1>588/589</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.0-100%</td></tr><tr><td rowspan=1 colspan=7>Antibiotic Resistance Markers</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Sensitivity/PPA</td><td rowspan=1 colspan=3>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>130/130</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1-100%</td><td rowspan=1 colspan=1>323/324</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.1-100%</td><td rowspan=1 colspan=1>412/412</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=1>400/400</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=1>404/404</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=1>307/307</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.8-100%</td><td rowspan=1 colspan=1>399/399</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100%</td></tr></table>

a Five FN specimens were spiked with an K. aerogenes (previously $E$ . aerogenes) strain (ATCC 29751) that demonstrated poor reactivity with the FilmArray Pneumonia Panel (see Error! Reference source not found.). b Two FN specimens were spiked with an influenza A strain that may have been under-quantified c Three FN specimens were spiked with an influenza B strain that may have been under-quantified

Table 41. FilmArray Pneumonia Panel Performance of Contrived Sputum Specimen   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Sensitivity/PPA</td><td rowspan=1 colspan=3>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenesa</td><td rowspan=1 colspan=1>47/55</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>73.8-92.4%</td><td rowspan=1 colspan=1>513/513</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=1>516/516</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=7>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6%</td><td rowspan=1 colspan=1>521/521</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>521/521</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=1>521/521</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>50/52</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>87.0-98.9%</td><td rowspan=1 colspan=1>494/494</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.2-100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.0-100%</td><td rowspan=1 colspan=1>520/520</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza Ab</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=1>517/521</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.0-99.7%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>48/51</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>84.1-98.0%</td><td rowspan=1 colspan=1>516/517</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=1 colspan=7>Antibiotic Resistance Markers</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>121/122</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>95.6-99.9%</td><td rowspan=1 colspan=1>289/290</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.1-99.9%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>43/44</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>88.2-99.6%</td><td rowspan=1 colspan=1>381/381</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.4-100%</td><td rowspan=1 colspan=1>360/361</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=1>372/372</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.0-100%</td><td rowspan=1 colspan=1>232/232</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.6-100%</td><td rowspan=1 colspan=1>369/369</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100%</td></tr></table>

a Four FN specimens were spiked with an $K .$ aerogenes (previously $E .$ . aerogenes) strain (ATCC 29751) that demonstrated poor reactivity with the FilmArray Pneumonia Panel (see Error! Reference source not found.). b Two FN specimens were spiked with an influenza A strain that may have been under-quantified c Two FN specimens were spiked with an influenza B strain that may have been under-quantified

# Testing of Polymicrobial Contrived Specimens

Additionally, two sets of individual BAL $\left( \mathrm { N } { = } 6 0 \right)$ ) and sputum $\left( \mathrm { N } { = } 6 0 \right)$ ) specimens were multi-spiked with randomized low, medium and high relative concentrations of either A. baumannii, E. cloacae, and $E$ . coli or $K .$ . oxytoca, $P$ . mirabilis, and $S .$ marcescens. As shown in Table 42 and Table 43, the majority of the spiked organisms were reported at the expected relative low, medium, or high bin level by the

FilmArray Pneumonia Panel. In four BAL specimens, $E .$ cloacae was intended to be spiked at a medium level but was reported in a in a high $( { \geq } 1 0 ^ { \land } 7 )$ bin. Also, one specimen spiked with a high level of $P$ . mirabilis was not detected (i.e. a false negative result for $P .$ . mirabilis).

Table 42. Polymicrobial Sputum Specimen Results   

<table><tr><td rowspan=1 colspan=6>Organism Spiked Into Sputum</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FilmArray Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td></tr><tr><td rowspan=3 colspan=1>Se kd</td><td rowspan=1 colspan=1>Low(104 copies/mL)</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Medium(105.5 copies/mL)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>High(107 copies/mL)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td></tr></table>

Table 43. Polymicrobial BAL Specimen Results   

<table><tr><td rowspan=1 colspan=6>Organism Spiked Into BAL</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FilmArray Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td></tr><tr><td rowspan=3 colspan=1>Se et</td><td rowspan=1 colspan=1>Low(104 copies/mL)</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Medium(105.5copies/mL)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>High(107 copies/mL)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59b</td></tr></table>

a Quantified at the bin boundary and reported $\mathrm { a s } { \mathord {  } } { = } 1 0 ^ { \mathord { \wedge } } 7$ b One false negative result

# Selected Analytical Studies

# Limit of Detection

A limit of detection (LoD) was established for atypical bacteria and viruses detected by the FilmArray Pneumonia Panel. LoD was estimated by testing dilutions of contrived BAL or sputum samples containing known concentrations of organisms. Confirmation of LoD was achieved by testing at least 20 replicates per samples type on FilmArray, FilmArray 2.0 and FilmArray Torch systems (60 replicates total per sample type). LoD concentration was confirmed when the analyte was detected in at least $9 5 \%$ of the replicates tested.

The confirmed LoD for each atypical bacterium or virus (including a LoD for more than one isolate of the more genetically diverse viruses) is listed in Table 44Error! Reference source not found.. LoD concentration is based on quantification of each culture in viable units $\mathrm { ( T C I D 5 0 / m L }$ or $\mathrm { C F U / m L }$ ) and a corresponding molecular LoD concentration (DNA or RNA copies/mL) is provided based on quantitative real-time or digital PCR.

Table 44. Summary of Limit of Detection (LoD) for FilmArray Pneumonia Panel Atypical Bacteria and Viruses   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>IsolateStrain/Serotype/Source ID</td><td rowspan=1 colspan=2>LoD Concentrationa</td></tr><tr><td rowspan=1 colspan=1>Viable Units</td><td rowspan=1 colspan=1>Molecular (DNA or RNA)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW183ATCC VR-2282</td><td rowspan=1 colspan=1>5.0E-01 TCID50/mL b</td><td rowspan=1 colspan=1>3.3E+02 copies/mLb</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Philadelphia-1ATCC 33152</td><td rowspan=1 colspan=1>5.0E+02 CFU/mL</td><td rowspan=1 colspan=1>1.6E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129Zeptometrix 0801579</td><td rowspan=1 colspan=1>7.5E+01 TCID50/mL b</td><td rowspan=1 colspan=1>3.5E+03 copies/mLb</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=6 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species A (A18)ATCC VR-19</td><td rowspan=1 colspan=1>5.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>9.2E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Species B (B3)Zeptometrix 0810062CF</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>1.8E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Species C (C2)ATCC VR-846</td><td rowspan=1 colspan=1>5.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>7.5E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Species D (D37)Zeptometrix 0810119CF</td><td rowspan=1 colspan=1>2.5E-01 TCID50/mLb</td><td rowspan=1 colspan=1>2.9E+03 copies/mL b</td></tr><tr><td rowspan=1 colspan=1>Species E (E4)Zeptometrix 0810070CF</td><td rowspan=1 colspan=1>1.0E-01 TCID50/mLb</td><td rowspan=1 colspan=1>3.5E+04 copies/mLb</td></tr><tr><td rowspan=1 colspan=1>Species F (F41)ATCC VR-930</td><td rowspan=1 colspan=1>5.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>5.5E+03 copies/mL</td></tr><tr><td rowspan=4 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>229EATCC VR-740</td><td rowspan=1 colspan=1>5.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>8.1E+01 copies/mL</td></tr><tr><td rowspan=1 colspan=1>HKU1Clinical Specimend</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04 copies/mL</td></tr><tr><td rowspan=1 colspan=1>NL63BEI NR-470</td><td rowspan=1 colspan=1>2.5E+00 TCID50/mLc</td><td rowspan=1 colspan=1>5.4E+02 copies/mLc</td></tr><tr><td rowspan=1 colspan=1>OC43ATCC VR-759</td><td rowspan=1 colspan=1>5.0E+02 TCID50/mLc</td><td rowspan=1 colspan=1>9.3E+03 copies/mLc</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>16 Type A1Zeptometrix 0810161CF</td><td rowspan=1 colspan=1>5.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>5.9E+03 copies/mL</td></tr><tr><td rowspan=2 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>RhinovirusType 1AZeptometrix 810012CFN</td><td rowspan=1 colspan=1>1.5E+01 TCID50/mL b</td><td rowspan=1 colspan=1>6.6E+03 copies/mL b</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6Zeptometrix 0810076CF</td><td rowspan=1 colspan=1>1.0E+02 TCID50/mL</td><td rowspan=1 colspan=1>5.7E+02 copies/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H1N1pdm09 A/SwineNY/03/09Zeptometrix 0810249CF</td><td rowspan=1 colspan=1>2.5E+00 TCID50/mLc</td><td rowspan=1 colspan=1>1.7E+03 copies/mLc</td></tr><tr><td rowspan=1 colspan=1>H3N2A/Port Chalmers/1/73ATCC VR-810</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL b</td><td rowspan=1 colspan=1>2.1E+02 copies/mL b</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06Zeptometrix 0810255CF</td><td rowspan=1 colspan=1>5.0E+00 TCID50/mLc</td><td rowspan=1 colspan=1>4.2E+02 copies/mLc</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>Type 1Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>2.5E+01 TCID50/mL</td><td rowspan=1 colspan=1>5.2E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Type 2Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>2.5E+01 TCID50/mLc</td><td rowspan=1 colspan=1>1.5E+03 copies/mLc</td></tr><tr><td rowspan=1 colspan=1>Type 3Zeptometrix 0810016CF</td><td rowspan=1 colspan=1>2.5E+01 TCID50/mLc</td><td rowspan=1 colspan=1>3.8E+02 copies/mLc</td></tr><tr><td rowspan=1 colspan=1>Type 4AZeptometrix 0810060CF</td><td rowspan=1 colspan=1>2.5E+02 TCID50/mL</td><td rowspan=1 colspan=1>8.1E+03 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Type AZeptometrix 0810040ACF</td><td rowspan=1 colspan=1>1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>4.3E+02 copies/mL</td></tr></table>

a The listed concentration was confirmed with $2 9 5 \%$ detection on each FilmArray system in artificial BAL (aBAL) and/or sputum. b LoD confirmation $( \geq 9 5 \%$ detection) was achieved at a 2 to 5-fold lower concentration in aBAL. c LoD confirmation $( \geq 9 5 \%$ detection) was achieved at a 2 to 5-fold lower concentration in sputum.   
BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

Note: LoD concentrations of the cultured viruses and the obligate intracellular atypical bacteria (C. pneumoniae and $M .$ . pneumoniae) are provided in units of $\mathrm { T C I D } _ { 5 0 }$ ( $50 \%$ Tissue Culture Infectious Dose). $\mathrm { T C I D } { 5 0 }$ is an indirect measure of viral or bacterial concentration based on infectivity and cytotoxicity and will therefore vary considerably depending on technique and methodology (including cell type, culture media and conditions, cytotoxicity of the virus, etc.). It is not appropriate to make determinations on relative sensitivity of different molecular assays for detection of viruses and bacteria based on LoD values measured in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Concentrations are also presented in copies $\mathrm { \Delta } / \mathrm { m L }$ based upon independent quantitative PCR assays $( \mathsf { q } ^ { \mathrm { p } } \mathrm { C R } )$ or digital PCR. Note that the accuracy of qPCR assays may also be affected by assay conditions, the standard reference material, and sequence variance between strains.

No assay-specific LoD concentrations were determined for the bacterial analytes. For bacteria, the FilmArray Pneumonia Panel reports a Detected result when the estimated bacterial nucleic acid abundance is ${ \ge } 1 0 ^ { \land 3 . 5 }$ copies $/ \mathrm { m L }$ , and the panel reports a Not Detected result if there is no amplification or the estimated bacterial nucleic acid abundance is $< 1 0 \land 3 . 5$ copies $/ \mathrm { m L }$ . Each assay was determined to be linear in relation to input concentration (slope $\approx 1 . 0$ and coefficient of determination (A $\mathrm { d j } \ \mathrm { R } ^ { 2 } ) > 0 . 9 5 )$ ) and estimates of nucleic acid abundance and corresponding bin results were determined to be accurate within $0 . 5 \mathrm { - l o g _ { 1 0 } }$ copies/mL when compared to a copies/mL input concentration determined by digital PCR.No assay-specific LoD concentrations were determined for the antimicrobial resistance (AMR) gene assays. AMR genes are reported as Detected when an applicable bacterium is Detected and the assay for the AMR gene is positive. Positive AMR gene assay results were observed in $2 9 5 \%$ of 90 replicates when the applicable bacterium was tested at a concentration ${ \ge } 1 0 ^ { \wedge } 3 . 5$ copies $\mathrm { \Delta } / \mathrm { m L }$ in the precision evaluation (see Precision (Reproducibility) below).

# Inclusivity

Analytical reactivity of FilmArray Pneumonia Panel assays was evaluated via a combination of empirical (wet) testing and in silico analysis of sequences available in public databases. Testing was performed on a collection of more than 350 genetically diverse viruses, bacteria, and antimicrobial resistance genes. The tested isolates represent relevant species, subspecies, strains, serotypes, or genotypes as well as temporal and geographic diversity for each of the panel analytes. Each isolate was tested in triplicate at concentrations near LoD or the lowest reportable level for the analyte. In silico analysis of sequence data was also used to make predictions of assay reactivity for less common strains or serotypes and AMR gene types that were not tested but that may be detected by the FilmArray Pneumonia Panel assays.

Atypical bacteria and viruses were tested and detected at concentrations within $3 \times \mathrm { L o D }$ (Table 46Error! Reference source not found. - Table 56Error! Reference source not found.). Bacteria were tested at a concentration of $1 . 0 \mathrm { E } { + } 0 4$ copies/mL (based on digital PCR of a single-copy gene in the bacterial chromosome) and the majority of isolates $( 9 4 . 4 \% )$ were detected with the expected bin result (Table 57Error! Reference source not found. - Table 71Error! Reference source not found.) and when the bacterium was detected, the appropriate AMR gene(s) were also detected (Table 72Error! Reference source not found. - Table 79Error! Reference source not found.).

Limitations on assay reactivity (based on wet testing observations) with specific viral and bacterial isolates or sequences and AMR gene types or sequences are noted in Table 45Error! Reference source not found.. Most limitations are associated with single-base sequence variants under one or more assay primers. Additional predicted limitations on reativity based on in-silico sequence analysis are provided in the footnotes and analyte-specific tables below.

Note: FilmArray Pneumonia Panel Influenza A and Influenza B assays are predicted to react with attenuated viruses used in vaccines.   
Table 45. Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays Observed in Testing   

<table><tr><td rowspan=1 colspan=1>Limitation</td><td rowspan=1 colspan=1>ObservedReesult</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain/Isolate/Variant</td></tr><tr><td rowspan=4 colspan=1>Minor</td><td rowspan=4 colspan=1>Detectedmay be under-reported byone bin (≤10-fold)</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Enterobacter hormaechei(ATCC 49162)a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniae subsp.quasipneumoniae(DSM 28211)b</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Moraxella catarrhalis(ATCC 23246)c</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Streptococcus pyogenes(ATCC 19615)</td></tr><tr><td rowspan=6 colspan=1>Major</td><td rowspan=3 colspan=1>Detectedmay be under-reported bytwo or more bins (&gt;10-fold)</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Enterobacter asburiae(ATCC 35953, 35954, 35955, and 35957)d</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>Klebsiella (Enterobacter) aerogenes(ATCC 29751)e</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>MREJ type XV</td></tr><tr><td rowspan=3 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>Acinetobacter nosocomialis(ATCC 700472)f</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa(ATCC 25619)s</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>MREJ type xviiiMREJ type xix</td></tr></table>

a Minor limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for two $E .$ . hormaechei sequences $( 3 . 0 \% )$ from public databases. b Minor limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for one $K .$ . quasipneumoniae sequence $( 2 0 . 0 \% )$ from public databases. Additional minor or major limitation predicted for four $K$ . pneumoniae sequences $( 0 . 3 \% )$ from public databases. c Minor limitation observed for this isolate. Additional minor or major limitation predicted for two $M$ . catarrhalis sequences $( 3 . 3 \% )$ ) from public databases. d Major limitation observed or predicted for these isolates due to sequence variance under primer. Similar limitation predicted for five $E$ . asburiae sequences $( 1 0 . 9 \% )$ , eight $E$ . cloacae sequences $( 2 . 2 \% )$ , and two $E$ . ludwigii sequences $( 1 6 . 7 \% )$ from public databases. e Major limitation observed for this isolate due sequence variance under primers. Similar limitation predicted for six $K .$ aerogenes sequences $( 4 . 3 \% )$ from public databases f ATCC 700472 could not be confirmed as A. nosocomialis by sequence and may be a non-Acinetobacter calcoaceticus-baumannii complex species that has been mis-identified. g Major limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for one $P$ . aeruginosa sequence $\left( 0 . 7 \% \right)$ from public databases.

Table 46. Adenovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotype </td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=22 colspan=1>Adenovirus</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC VR-19 [Washington D.C./1954]</td><td rowspan=1 colspan=1>9.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=22 colspan=1>AdenovirusDetected</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC VR-863 [Huie/Massachusetts]</td><td rowspan=1 colspan=1>2.7E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Zeptometrix 0810073CF</td><td rowspan=1 colspan=1>2.7E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=12 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810062CF</td><td rowspan=1 colspan=1>1.8E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-7 [Gomen/California/1954]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7A</td><td rowspan=1 colspan=1>Zeptometrix 0810021CF</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7d/d2</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation [Iowa/2001]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7h</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation [Iowa/1999]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC VR-12 [Slobitski/Massachusetts]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-15 [De Wit/Netherlands/1955]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-17 [CH.79/Saudi Arabia/1955]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>ATCC VR-1833 [128/Saudi Arabia/1956]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-716 [Compton/1972]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>ATCC VR-718 [Holden]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>ATCC VR-1602 [Wan/Amsterdam/1988]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>C</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-846 [Adenoid 6]</td><td rowspan=1 colspan=1>7.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810050CF</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810020CF</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC VR-6 [Tonsil 99]</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>D</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Zeptometrix 08100119CF</td><td rowspan=1 colspan=1>5.8E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810069CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810115CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotype a</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=7 colspan=1></td><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810070CF</td><td rowspan=1 colspan=1>1.7E+04</td><td rowspan=1 colspan=1>1x</td><td rowspan=7 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC VR-1572 [RI-67/Missouri/1952-1953]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation [S Carolina/2004]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=4 colspan=1>F</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>ATCC VR-930 [Tak 73-3544/Netherlands/1973]</td><td rowspan=1 colspan=1>5.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>NCPV 0101141v</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Zeptometrix 0810084CF</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Zeptometrix 0810085CF</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a In silico analysis of available sequences predicts that the FilmArray Pneumonia Panel will also react with Adenovirus B55, C57, all species D serotypes, and G52.

Table 47. Coronavirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=11 colspan=1>Coronavirus</td><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>8.1E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=11 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810229CF</td><td rowspan=1 colspan=1>2.4E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>HKU1a</td><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Detroit/2010]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [S Carolina/2010]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI NR-470b [Amsterdam/2003]</td><td rowspan=1 colspan=1>2.7E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td><td rowspan=1 colspan=1>8.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>OC43</td><td rowspan=1 colspan=1>ATCC VR-759c</td><td rowspan=1 colspan=1>4.6E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810024CF</td><td rowspan=1 colspan=1>1.4E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a No cultured isolates of Coronavirus HKU1 were available for testing. Five clinical NPS specimens containing Coronavirus HKU1 were collected from different regions of the US in 2010 and 2015, quantified molecularly, and tested. b Organism obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID, NIH: Human Coronavirus NL63, NR470. c Discontinued part #. See ATCC VR-1558.

Table 48. Human Metapneumovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=2 colspan=1>Serotype</td><td rowspan=2 colspan=2>Source [Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(TCID50/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=11 colspan=1>HumanMetapneumovirus</td><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=2>Zeptometrix 0810161CF [Iowa10/2003]</td><td rowspan=1 colspan=1>5.0E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=11 colspan=1>HumanMetapneumovirusDetected</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=2>Zeptometrix 0810160CF [Iowa3/2002]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=2>Zeptometrix 0810163CF [Iowa14/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=2>Zeptometrix 0810164CF [Iowa27/2004]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>b</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>Zeptometrix0810156CF [Peru2/2002]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=2>Zeptometrix 0810158CF [Peru3/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=5 colspan=1>B2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>Zeptometrix 0810159CF [Peru6/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>4</td><td rowspan=2 colspan=1>Zeptometrix 0810157CF [Peru1/2002]</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>1.5E+02</td><td rowspan=2 colspan=1>3x</td></tr><tr></tr><tr><td rowspan=2 colspan=1>18</td><td rowspan=2 colspan=1>Zeptometrix 0810162CF [Iowa18/2003]</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>1.0E+02</td><td rowspan=2 colspan=1>3x</td></tr><tr></tr></table>

Table 49. Human Rhinovirus and Enterovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotype</td><td rowspan=2 colspan=2>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Human Rhinovirusa</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>Zeptometrix 0810012CFN [1A]</td><td rowspan=1 colspan=1>2.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=14 colspan=1>HumanRhinovirus/EnterovirusDetected</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>ATCC VR-482 [HGP]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=2>ATCC VR-1601 [68-CV11]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=2>ATCC VR-283 [11757/Washington DC/1960]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=2>ATCC VR-507b [137-3]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=2>ATCC VR-1600 [Ch47]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=2>ATCC VR-1187 [130-63]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=2>ATCC VR-1195 [50-525-CV54]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>ATCC VR-483 [FEB]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=2>ATCC VR-284 [1059/S Carolina/1959]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=2>ATCC VR-1663 [33342/N Carolina/1959]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ATCC VR-1137 [5870]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>ATCC VR-338 [56822]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=2>ATCC VR-1193 [Baylor 7]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=7>Enterovirus</td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Coxsackievirus 10</td><td rowspan=1 colspan=2>ATCC VR-168 [NY/1950]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=2 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=2>ATCC VR-1432 [H]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotype</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix 0810017CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=9 colspan=1>Detected</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Zeptometrix 0810074CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>Zeptometrix 0810075CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix 0810076CF</td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix 0810077CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix 0810023CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>ATCC VR-850 [Kuykendall/California/1952]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC VR-583 [DN-19/Texas/1963]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>ATCC VR-1823 [US/MO/2014-18947]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

a The concentration used for Human Rhinovirus isolate testing was based on $3 \times$ the Enterovirus LoD concentration $5 . 7 \mathrm { E } { + } 0 2$ copies/mL). b Discontinued part #; see ATCC VR-1365.

Table 50. Influenza A Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=40 colspan=1>Influenza A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=40 colspan=1>Influenza ADetected</td></tr><tr><td rowspan=9 colspan=1>H1N1</td><td rowspan=1 colspan=1>ATCC VR-219 [NWS/1933]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-95 [PR/8/1934a]</td><td rowspan=1 colspan=1>1.0E+03</td><td rowspan=1 colspan=1>1.5xa</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-96 [Wiess/1943]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-97 [FM/1/1947]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-98 [Mal/302/1954]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-546 [Denver/1/1957]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810036CF [New Caledonia/20/1999]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810036CFN [Solomon Islands/3/2006]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810244CF [Brisbane/59/2007]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>BEI NR-3478b [Kilbourne F63 A/NWS/1934 (HA) xA/Rockefeller Institute/5/1957 (NA)]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=7 colspan=1>H1N1pdm09</td><td rowspan=1 colspan=1>Zeptometrix 0810249CF [SwineNY/03/2009]</td><td rowspan=1 colspan=1>6.6E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810109CFJ [Canada/6294/2009]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810165CF [California/07/2009]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810166CF [Mexico/4108/2009]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-19823c [Netherlands/2629/2009]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-42938d [Georgia/F32551/2012]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44345® [Hong Kong/H090-761-V1(0)/2009]</td><td rowspan=1 colspan=1>1.0E+03</td><td rowspan=1 colspan=1>1.5xa</td></tr><tr><td rowspan=8 colspan=1>H3N2</td><td rowspan=1 colspan=1>ATCC VR-810 [Port Chalmers/1/1973]</td><td rowspan=1 colspan=1>1.0E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-776 [Alice (live attenuated vaccine)]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810238CF [Texas /50/2012]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-547 [Aichi/2/1968]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-544 [Hong Kong/8/1968]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-822 [Victoria/3/1975]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810252CF [Wisconsin/67/2005]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810138CF [Brisbane/10/2007]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H3N2v</td><td rowspan=1 colspan=1>ODHL1 [Ohio/2012]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>BEI NR-2775f [Japan/305/1957]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H2N3</td><td rowspan=1 colspan=1>MRIGlobalg [Mallard/Alberta/79/2003]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>MRIGlobal8 [Chicken/Yunnan/1251/2003]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>MRIGlobalg [Northern pintail/Washinton/40964/2014]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>BEI NR-9682h [Duck/Singapore/645/97]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H5N8</td><td rowspan=1 colspan=1>MRIGlobal&amp; [Gyrfalcon/Washing-ton/41088-6/2014]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H7N7</td><td rowspan=1 colspan=1>MRIGlobal8 [Netherlands/219/2003]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H7N9</td><td rowspan=1 colspan=1>MRIGlobal [Anhui/01/2013]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>BEI NR-2765i [Chicken/Germany/N/49]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Swine</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>H1N1 Swine</td><td rowspan=1 colspan=1>ATCC VR-333 [Swine/Iowa/15/1930]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-99 [Swine/1976/1931]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

a 1.5x the LoD for Influenza A H1N1pdm09 Zeptometrix 0810109CFN [SwineNY/03/2009] (6.6E+02 copies/mL). b Genomic RNA obtained through BEI Resources NAID, NIH: Kilbourne F63: A/NWS/1934 (HA) $\mathbf { X }$ A/Rockefeller Institute/5/1957 (NA) (H1 Reassortant NWS-F, NR-9677. c Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Netherlands/2629/2009 (H1N1)pdm09, NR-19823 d Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Georgia/F32551/2012 (H1N1)pdm09, NR-42938. e Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Hong Kong/H090-761-V1(0)/2009 (H1N1)pdm09, NR-44345. f Genomic RNA obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/Japan/305/1957 (H2N2), NR-2775. g Isolate provided and tested by MRI Global, Kansas City, MO. BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

Genomic RNA obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/chicken/Germany/N/1949 (H10N7), NR-2765.

Table 51. Influenza B Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Lineage</td><td rowspan=2 colspan=1>[Strain/Location/Year], Source</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=6 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC VR-101 [Lee/1940]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=13 colspan=1>Influenza BDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-102 [Allen/1945]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-103 [GL/1739/1954]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-296 [1/Maryland/1959]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-295 [2/Taiwan/1962]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-786 [Brigit/Russia/1969]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=7 colspan=1>Influenza B</td><td rowspan=3 colspan=1>Victoria</td><td rowspan=1 colspan=1>ATCC VR-823 [5/Hong Kong/1972]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810258CF [2506/Malaysia/2004]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC 2005743348 [1/0hio/2005]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Yamagata</td><td rowspan=1 colspan=1>Zeptometrix 0810256CF [07/Florida/2004]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810255CF [04/Florida/2006]</td><td rowspan=1 colspan=1>2.1E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810241CF [1/Wisconsin/2010]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810239CF [2/Massachusetts/2012]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 52. Parainfluenza Virus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=2>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=12 colspan=1>ParainfluenzaVirus</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=2>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>5.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=12 colspan=1>Parainfluenza VirusDetected</td></tr><tr><td rowspan=1 colspan=2>BEI NR-48680a [FRA/29221106/2009]</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC VR-94 [C-35/Washington DC/1957]</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=2>ATCC VR-92 [Greer/Ohio/1955]</td><td rowspan=1 colspan=1>8.9E+02</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=2>Zeptometrix 0810016CF</td><td rowspan=1 colspan=1>1.9E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233b [NIH 47885, Wash/47885/57]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-93 [C-243/Washington DC/1957]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>4</td><td rowspan=1 colspan=2>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>8.1E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=2>ATCC VR-1378 [M-25/1958]</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>Zeptometrix 0810060BCF</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC VR-1377 [CH-19503/Washington DC/1962]</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Virus obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 1, HPIV1/FRA/29221106/2009, NR-48680. b Virus obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 3, NIH 47885, NR-3233.

Table 53. Respiratory Syncytial Virus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=7 colspan=1>RespiratorySyncytial Virus</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF [2006]</td><td rowspan=1 colspan=1>4.3E+02</td><td rowspan=1 colspan=1>1x</td><td rowspan=7 colspan=1>RespiratorySyncytial VirusDDtected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-26 [Long/Maryland/1956]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1540 [A2/Melbourne/1961]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B</td><td rowspan=1 colspan=1>Zeptometrix 0810040CF [Ch-93 (18)-18]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400 [WV/14617/1985]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955 [9320/Massachusetts/1977]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1580 [18537/Washington DC/1962]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 54. Chlamydia pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Source [Strain]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=4 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>ATCC VR-2282 [TW-183/Taiwan/1965]</td><td rowspan=1 colspan=1>6.7E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>ChlamydiapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1310 [CWL-029]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1360 [CM-1/Georgia]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 53592 [AR-39/Seattle/1983]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 55. Legionella pneumophila Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Species/Subspecies</td><td rowspan=2 colspan=1>Serogroup</td><td rowspan=2 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(CFU/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=2 colspan=1>L. pneumophila</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 33152 [Philadelphia-1]</td><td rowspan=1 colspan=1>5.0E+02</td><td rowspan=1 colspan=1>1x</td><td rowspan=6 colspan=1>LegionellapneumophilaDetected</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 33155 [Bloomington-2]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>L. pneumophila subsp. fraseri</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC 33156 [Los Angeles-1]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC 33216 [Dallas 1E]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>L. pneumophila subsp. pascullei</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC 33737 [U8W]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>L. pneumophila subsp. pneumophila</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>ATCC 43283 [Leiden 1]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=1>Species/Subspecies</td><td rowspan=1 colspan=1>Serogroup</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=2>Test Concentration(CFU/mL)       xLoD</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC 43703 [1169-MN-H]</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr></table>

Table 56. Mycoplasma pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=2>Source [Strain]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=9 colspan=1>Mycoplasmapneumoniae</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=2>Zeptometrix 0801579 [M129]</td><td rowspan=1 colspan=1>1.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>MycoplasmapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=2>ATCC 29342 [M129-B7]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC 29085 [PI 1428]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>ATCC 15531-TTR [FH strain of Eaton Agent [NCTC10119]]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC 15492 [Mac]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>unknown</td><td rowspan=1 colspan=1>ATCC 15293 [M52]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15377 [Bru]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC 39505 [Mutant 22]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2>ATCC 49894 [UTMB-10P]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 57. Acinetobacter calcoaceticus-baumannii complex Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>A. baumannii</td><td rowspan=1 colspan=2>ATCC 9955 [6-561]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=13 colspan=1>Acinetobactercalcoaceticus-baumannii complexDetected</td></tr><tr><td rowspan=1 colspan=2>ATCC 19606 [2208 Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 17961 [CDC 7788]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>AR-Bank #0033</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>GRE 1153064</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>GRE 1062081</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=3 colspan=1>A. calcoaceticus</td><td rowspan=1 colspan=2>ATCC 51432</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 23055 [46]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 14987 [HO-1]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. calcoaceticus subsp. anitratus</td><td rowspan=1 colspan=2>ATCC 15308 [NCTC 7844]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. pittii</td><td rowspan=1 colspan=1>ATCC 19004 [57.071.228 ]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. nosocomialis a</td><td rowspan=1 colspan=1>ATCC 17903 [NCTC 8102]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A.seifertii</td><td rowspan=1 colspan=2>CCUG 34785</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a A. nosocomialis ATCC 700472 was not detected at any concentration. Sequencing suggests the isolate may be mis-identified.

Table 58. Enterobacter cloacae complex Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>E. cloacae</td><td rowspan=1 colspan=2>ATCC 49141 [AmMs 204]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=13 colspan=1>Enterobactercloacae complexDetected</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1143 [Entb 55M]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2341 [1101152]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>AR-Bank #0154</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NCTC 13464</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>E. cloacae subsp. cloacae</td><td rowspan=1 colspan=2>ATCC 13047 [Type Strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. cloacae subsp. dissolvens</td><td rowspan=1 colspan=2>ATCC 23373Da [ICPB ED105]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>E. asburiae</td><td rowspan=1 colspan=2>ATCC 35953b [CDC 1497-78 Type Strain ]</td><td rowspan=1 colspan=1>1.0E+06b</td></tr><tr><td rowspan=1 colspan=1>ATCC 35957b [CDC 570-83]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+06b</td></tr><tr><td rowspan=2 colspan=1>E. hormaechei</td><td rowspan=1 colspan=1>ATCC 49162b [CDC 992-77]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+05b</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2082</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>E. kobei</td><td rowspan=1 colspan=2>CCUG 59410</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>E. ludwigii</td><td rowspan=1 colspan=2>CCUG 23050</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a Genomic DNA from $E$ . cloacae subsp. dissolvens. b See Table 45Error! Reference source not found. for limitation.

Table 59. Escherichia coli Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=13 colspan=1>E. coli</td><td rowspan=1 colspan=1>ATCC 25922 [FDA strain Seattle 1946]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=13 colspan=1>Escherichia coliDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 43888 [CDC B568-73]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0061</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0086</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0137</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0150</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0162</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062016</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1252008</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1252009</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1256018</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801905 [Z136]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 29930 [WRAIR I virulent]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

Table 60. Haemophilus influenzae Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>Type a</td><td rowspan=1 colspan=1>ATCC 9006 [AMC 36-A-3 [610, PCM 2436]]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=8 colspan=1>Haemophilusinfluenzae Detected</td></tr><tr><td rowspan=1 colspan=1>Type b</td><td rowspan=1 colspan=1>ATCC 10211 [AMC 36-A-1 [572]], Biotype 1</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Type c</td><td rowspan=1 colspan=1>ATCC 49699 [C 9007]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Type d</td><td rowspan=1 colspan=1>ATCC 9008 [AMC 36-A-6 [611]]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Type e</td><td rowspan=1 colspan=1>ATCC 8142 [AMC 36-A-7 [595, NCTC 8472]]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Type f</td><td rowspan=1 colspan=1>ATCC 700223 [GA1264]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Biogroupaegyptius</td><td rowspan=1 colspan=1>ATCC 11116 [180-a [NCTC 8502]]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Non-typeable</td><td rowspan=1 colspan=1>ATCC 51907 [Rd [KW20]]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

NOTE: The Hinfluenzae assay will not react with strains that do not carry the hpd gene128.

Table 61. Klebsiella (Enterobacter) aerogenes Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>K. aerogenes</td><td rowspan=1 colspan=1>ATCC 13048 [NCTC 10006]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=6 colspan=1>KlebsiellaaerogenesDetected</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0062</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0074</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0161</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1254066</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 29751² [MULB-250]</td><td rowspan=1 colspan=1>1.0E+07a</td></tr></table>

a Previously known as Enterobacter aerogenes. b See Table 45Error! Reference source not found. for limitation.

Table 62. Klebsiella oxytoca Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=12 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>ATCC 13182 [479-2 Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=12 colspan=1>Klebsiella oxytocaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 43086 [Pasco 201]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 49131 [AmMS 101]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 700324 [LBM 90.11.033]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 8724 [NRRL B-199]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0147</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 10678</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 14611</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1254054</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 63. Klebsiella pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=15 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705 [ART 2008133]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=21 colspan=1>KlebsiellapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0068</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0075</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0076</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0079</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0080</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank # 0097</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0107</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0153</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062084</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1355030</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 328</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NCTC 13465</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801886</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae subsp. ozaenae</td><td rowspan=1 colspan=1>ATCC 11296 [AMC 35-E-5]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae subsp. pneumoniae</td><td rowspan=1 colspan=1>ATCC 13883 [NCTC 9633]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae subsp. rhinoscleromatis</td><td rowspan=1 colspan=1>ATCC 13884 [NCTC 5046]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. quasipneumoniae subsp. quasipneumoniae</td><td rowspan=1 colspan=1>DSM 28211a [01A030, SB11]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=1>K. quasipneumoniae subsp. simipneumoniae</td><td rowspan=1 colspan=1>DSM 28212 [07A044, SB30]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. variicola</td><td rowspan=1 colspan=1>ATCC BAA-830 [F2R9]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a See Table 45Error! Reference source not found. for limitation.

Table 64. Moraxella catarrhalis Isolates Tested and Detected   

<table><tr><td>Organism</td><td>Source [Strain]</td><td>Test concentration (copies/mL)</td><td>Result</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=5 colspan=1>M. catarrhalis</td><td rowspan=1 colspan=1>ATCC 25238 [Ne 11]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=5 colspan=1>Moraxella catarrhalisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 25240 [N9]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 8176 [20]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 23246 a [NCTC 4103]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=1>ATCC 49143 [Am MS 116]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a See Table 45Error! Reference source not found. for limitation.

Table 65. Proteus spp. Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>ATCC 29906 [1003]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=13 colspan=1>Proteus spp.Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC 33583 [571101]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 35659 [LRA 08 01 73]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0156</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0159</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1254053</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>P. hauseri</td><td rowspan=1 colspan=1>ATCC 13315 [NCTC 4175 Strain Lehmann]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 700826 [CDC 1732-80]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>P. penneri</td><td rowspan=1 colspan=1>ATCC 33519 [Type Strain CDC 1808-73]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 35197 [CDC 1655-67]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=3 colspan=1>P. vulgaris</td><td rowspan=1 colspan=1>ATCC 29905</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 33420</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 27973 [CDC 1787-64-SC1]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 66. Pseudomonas aeruginosa Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=11 colspan=1>P. aeruginosaa</td><td rowspan=1 colspan=1>ATCC 10145 [ MDB strain BU 277 type strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=11 colspan=1>Pseudomonas aeruginosaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1744 [109246]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 19429 [NCTC 6750]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 27853 [Boston 41501]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0054</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0092</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0100</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0103</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0111</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Creighton University PS28</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

aeruginosa ATCC 25619 was not detected at any concentration tested. See Table 45Error! Reference source not found. for limitation

Table 67. Serratia marcescens Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>ATCC 13880 [Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>Serratia marcescensDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 27137 [CDC 3100-71]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 43297 [3G]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-885 [Type strain KRED]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1659005</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1659004</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 68. Staphylococcus aureus Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=3>Source [train] [PFGE Type if appliable]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=16 colspan=1>S. aureus</td><td rowspan=1 colspan=3>Staphylococcus aureus representing PFGE Types USA100-USA1200</td><td rowspan=1 colspan=1></td><td rowspan=16 colspan=1>Staphylococcus aureusDetected</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS705 [PFGE USA100]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS701 [PFGE USA200]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1717 [PFGE USA300]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS683 [PFGE USA300]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS662 [PFGE USA300]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS707 [PFGE USA300]</td><td rowspan=1 colspan=1>PFGE USA300</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1707 [PFGE USA400]</td><td rowspan=1 colspan=1>[PFGE USA400</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PFGE USA500</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS648 [PFGE USA600]</td><td rowspan=1 colspan=1>NARSA NRS648 [PFGE USA600]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PFGE USA700</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS668 [PFGE USA800]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1749 [PFGE USA900 96:308]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1759 [PFGE USA900 N7129]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1700 [PFGE USA1000]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>BEI NR-46081 [PFGE USA1100 HIP12899]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain] [PFGE Type if applicable]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=41 colspan=1></td><td rowspan=1 colspan=1>ATCC BAA-1765 [PFGE USA1200 102-04]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=41 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1691 [Not USA100-1100]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Methicillin Sensitive Staphylococcus aureus (MSSA)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 10832 [W0od 46]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 14154 [Rose]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 12600 [NCTC Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 25923 [Seattle/1945]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 29213 [Wichita]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2421 [Mass/2010]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Rennes 1060728</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062519 [SCCmec Type: III/ MREJ xix]b</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Borderline Resistant Staphylococcus aureus (BORSA)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SUN1 [Sunnybrook]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Methicillin Resistant Staphylococcus aureus (MRSA)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 43300 [F182 Kansas / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2422 [Worcester MA/2010 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1720 [MRSA252 / SCCmec Type: II/ PFGEUSA200]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS745 [CA-629 / SCCmec Type: V]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-38 [E2125 / SCCmec Type: I]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS686 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-44 [HPV107 / SCCmec Type: I / PFGE: Iberian]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-41 [NYBK2464 / SCCmec Type: II / PFGE 100]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS385 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-42 [HDE288 / SCCmec: Type VI/ PFGE 800]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-39 [HUSA304 / SCCmec Type: III]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-40 [CPS22 / SCCmec Type: III]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062264 [SCCmec Type: IV / MREJ type iv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1055015 [SCCmec Type: IVa / MREJ type vi]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0759084 [SCCmec Type: IV / MREJ type v]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0860042 [SCCmec Type: III / MREJ type vii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1052034 [MREJ ix]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1151100 [SCCmec Type: IV / MREJ type xi]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0960006 [MREJ type xii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1055017 [SCCmec Type: IVa / MREJ type xii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0759163 [MREJ type xiv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062373 [MREJ type xv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1057114 [MREJ type xvii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062292 [MREJ type xviii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Methicillin Resistant Staphylococcus aureus (MRSA) - mecC+</td><td rowspan=1 colspan=1>RSA) -ecC+</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2312 [M10/0061 / SCCmec Type: XI / mecC]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2313 [M10/0148 / SCCmec Type: XI / mecC</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a Staphylococcus aureus ATCC BAA-1717 was not detected at $1 . 0 \mathsf { E } ^ { + } 0 4 $ copies/mL but was detected at $1 . 0 \mathsf { E } ^ { + 0 5 }$ copies/mL with accurate bin results. The expected detection and $1 0 ^ { \wedge } 4$ bin result was reported in $\mathtt { \ge 5 0 \% }$ of additional replicates tested at $1 . 0 \mathsf { E } ^ { + 0 4 }$ copies/mL and no limitation on reactivity could be identified based on the isolate sequence.

b MREJ type xix characterized as MSSA129.

Table 69. Streptococcus agalactiae Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>S. agalactiae</td><td rowspan=1 colspan=2>NCTC 8017 [MK 104 P]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>Streptococcus agalactiaeDetected</td></tr><tr><td rowspan=1 colspan=2>ATCC 13813 [Ia/c Type Strain]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 12403 [III Typing Strain D136C]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 12386 [Grouping strain O90R]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-611 [V 2603 V/R]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2669 [VIII 5030-08]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Clinical Isolate [Utah/2010/CI03]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 70. Streptococcus pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 6303</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>Streptococcus pneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC BAA-341 [SPN1439-106]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>11A</td><td rowspan=1 colspan=1>NCTC 11900 [Gorman]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>S</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC 700672 [VH14]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>19A</td><td rowspan=1 colspan=1>ATCC 700673 [Hungary 19A-6]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>Non-capsulated</td><td rowspan=2 colspan=1>ATCC BAA-255 [R6]</td><td rowspan=2 colspan=1>1.0E+04</td></tr><tr></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td></td><td>unknown</td><td>ATCC BAA-1409 [62076]</td><td>1.0E+04</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

Table 71. Streptococcus pyogenes Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>ATCC 12344 [Typing strain T1, NCIB 11841, SF 130]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=8 colspan=1>Streptococcus pyogenesDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 12348 [Typing strain S43 Type 6]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 12384 [Typing strain C203 Type 3]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 19615a [Bruno]</td><td rowspan=1 colspan=1>1.0E+06a</td></tr><tr><td rowspan=1 colspan=1>ATCC 700294 [SF370; M1 GAS [M-type 1 T-type 1]]</td><td rowspan=1 colspan=1>1.0E+05b</td></tr><tr><td rowspan=1 colspan=1>ATCC 49399 [QC A62]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-595 [MGAS 315, serotype M3]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-947 [MGAS 5005, serotype M1]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a See Table 45 Error! Reference source not found. for limitation.   
b Streptococcus pyogenes ATCC 700294 was detected in 3/5 replicates at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies $/ \mathrm { m L }$ with $1 0 \mathord { \sim } 4$ copies $/ \mathrm { m L }$ bin results and 3/3 replicates at $1 . 0 \mathrm { E } { + } 0 5$ copies $/ \mathrm { m L }$ with $1 0 { \sim } 5$ copies $/ \mathrm { m L }$ bin results. The expected detection and $1 0 { \sim } 4$ bin result was reported in $2 5 0 \%$ of additional replicates tested at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies $/ \mathrm { m L }$ and no limitation on reactivity could be identified based on isolate sequence.

The following tables (Table 72 - Table 79) describe the reactivity of the AMR genes assays with different AMR gene types in various host bacteria. Results are shown for the isolates tested as well as predictions of reactivity with untested AMR gene types based on in silico analysis of sequences retrieved from public databases from June 2016 to Sept 2016.

Table 72. Isolates Containing mecA/C and MREJ Tested and Detected   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=3>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=37 colspan=2>S. aureus</td><td rowspan=1 colspan=4>Methicillin Sensitive Staphylococcus aureus (MSSA)containing SCCmec cassette (non-functional mecA variant)</td><td rowspan=37 colspan=1>mecA/C and MREJDetected</td></tr><tr><td rowspan=1 colspan=4>ATCC BAA-2421 [Mass/2010]                                1.0E+04</td></tr><tr><td rowspan=1 colspan=4>Methicillin Resistant Staphylococcus aureus (MRSA)(Characterized SCCmec Types)</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS705 [NY-12 / SCCmec Type: I]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS701 [MN-082 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1717 [TCH1516 / SCCmec Type: IVa]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS683 [GA-298 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS662 [CO-34 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS707 [NY-155 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1707 [MW2 /SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS691 [GA-62 /SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS648 [CA-347 /SCCmec Type:II or IV]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS689 [GA-442 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS668 [CO-72 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1700 [HFH-33798 / SCCmec Type: IVb]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>BEI NR-46081b (NRSA NRS484) [HIP12899 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1691 [HFH-30137 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC 43300 [F182 Kansas / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-2422 [Worcester MA/2010 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-1720 [MRSA252 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS745 [CA-629 / SCCmec Type: IV or V]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=4>Methicillin Resistant Staphylococcus aureus (MRSA)(Characterized MREJ Types)</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-38 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS686 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-44 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-41 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>NARSA NRS385 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-42 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-39 [MREJ type iii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-40 [MREJ type iv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>GRE 1062264 [MREJ type iv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>GRE 1055015 [MREJ type vi]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>GRE 0860042 [MREJ type vii]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>GRE 1052034 [MREJ type ix]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>GRE 1055017 [MREJ type xii]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>GRE 1062373 [MREJ type xv]c</td><td rowspan=1 colspan=1>1.0E+06b</td></tr><tr><td rowspan=1 colspan=2>GRE 1057114 [MREJ type xvii]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=2>GRE 1062292 [MREJ type xvii]</td><td rowspan=1 colspan=1>3.3E+08</td><td rowspan=2 colspan=1>mecA/C and MREJ NotDetected</td></tr><tr><td rowspan=1 colspan=2>GRE 1062519 [MREJ type xix]c,d</td><td rowspan=1 colspan=1>1.0E+07</td></tr><tr><td rowspan=1 colspan=3>Methicillin Resistant Staphylococcus aureus (MRSA)(SCCmec Type: XI / mecC / mecALGA2s1 variants)</td><td rowspan=5 colspan=1>mecA/C and MREJDetected</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2312 [M10/0061 / SCCmec Type: XI / mecC]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2313 [M10/0148 / SCCmec:Type XI / mecC ]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>S. argenteus</td><td rowspan=1 colspan=2>Methicillin Resistant Staphylococcus argenteus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>DSM 28299 [MSHR-1132]</td><td rowspan=1 colspan=1>1.00E+05</td></tr></table>

a mecA/C and MREJ assays positive in less than three replicates at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies $/ \mathrm { m L }$ , no sequence based limitation on reactivity identified. b Bacteria obtained through NARSA for distribution by BEI Resources, NIAID, NIH: Staphylococcus aureus, Strain HIP12899, NR-46081 c See Table 45Error! Reference source not found. for limitation. d MREJ type xix characterized MSSA129.

Table 73. In Silico Reactivity Predictions for mecA/C and MREJ   

<table><tr><td rowspan=1 colspan=2>mecA/Ca,b</td><td rowspan=1 colspan=3>MREJd</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Reduced Reactivityor Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Reduced Reactivityor Not Detected</td><td rowspan=1 colspan=1>UnknownReactivity(no sequences)</td></tr><tr><td rowspan=2 colspan=1>mecA in S. aureus</td><td rowspan=2 colspan=1>mecA in some isolatesof S. capitis, S. kloosii</td><td rowspan=1 colspan=1>MREJ i, iα - viie</td><td rowspan=1 colspan=1>MREJ ixf</td><td rowspan=1 colspan=1>MREJ viii</td></tr><tr><td rowspan=1 colspan=1>MREJ xi-xiv</td><td rowspan=1 colspan=1>MREJ xv8</td><td rowspan=1 colspan=1>MREJ x</td></tr><tr><td rowspan=2 colspan=1>mecC in S. aureus</td><td rowspan=1 colspan=1>and S. vitulinus</td><td rowspan=1 colspan=1>MREJ xvi - xviii</td><td rowspan=1 colspan=1>MREJ xviii</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=2 colspan=1>mecC in S. sciuri</td><td rowspan=2 colspan=1>MREJ inS. argenteus</td><td rowspan=1 colspan=1>MREJ xix, xxh</td></tr><tr><td rowspan=1 colspan=1>mecA and mecC innon-aureus staphylococci(including S. argenteus)</td><td rowspan=1 colspan=1>MREJ in non-aureusstaphylococci andother speciesd</td></tr></table>

a July 2016; analysis of 1,257 database mecA sequences from S. aureus and 14 mecC sequences from S. aureus, as well as mecA and mecC sequences from non-aureus staphylococci. b mecC is also referenced as SCCmec XI and mecALGA251. c Limited or reduced reactivity predicted for 2/1,257 mecA sequences from S. aureus $( 0 . 2 \% )$ . d June 2016; analysis of approximately 1,450 typed MREJ database sequences from $S .$ aureus and untyped sequences from S. aureus, non-aureus staphylococci and non-staphylococcus species (Bacillus cereus, bacillus thuringiensis, Macrococcus caseolyticus, Clostridium acidurici, and Rummeliibacillus stabekisii). e Limited or reduced reactivity predicted for 1/141 MREJ iii sequences $\left( 0 . 7 \% \right)$ ; normal reactivity observed for the isolate of MREJ iii tested (see Table 72Error! Reference source not found.). f Limited or reduced reactivity predicted for 2/8 MREJ ix sequences $( 2 5 . 0 \% )$ ; normal reactivity observed for the isolate of MREJ ix tested (see Table 72) g Reduced reactivity predicted by in silico analysis and observed with the isolate of MREJ xv tested (see Table 72Error! Reference source not found.). h MREJ xix and xx were not included in the assay design because they were identified from methicillin-sensitive S. aureus129.

Table 74. Isolates Containing the $\pmb { b } / \pmb { a } _ { \mathsf { C T X - M } }$ gene Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>CTX-M Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=5>Isolates Tested</td></tr><tr><td rowspan=5 colspan=1>CTX-M</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0137a</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=14 colspan=1>CTX-M Detected</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>GRE 1254054</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=3 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0068a</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0153a</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1355030</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0162</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0107</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-8</td><td rowspan=1 colspan=1>K. aerogenes</td><td rowspan=1 colspan=1>GRE 1254066</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>E.coli</td><td rowspan=1 colspan=1>AR-Bank #0086</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>E.cloacae</td><td rowspan=1 colspan=1>NCTC 13464</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0079</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td><td rowspan=1 colspan=1>E.coli</td><td rowspan=1 colspan=1>Zeptometrix 0801905</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-22</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>GRE 1254053</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>CTX-M-25</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>NCTC 13465</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=5>In silico Reactivity PredictionsaUnknown ReactivityDetected                              Not Detected(no sequences)</td></tr><tr><td rowspan=1 colspan=2>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=1>CTX-M-1  CTX-M-117</td><td rowspan=1 colspan=2>CTX-M-150</td><td rowspan=1 colspan=1>CTX-M-151</td><td rowspan=1 colspan=1>CTX-M-118</td><td rowspan=1 colspan=1>CTX-M-143</td></tr><tr><td rowspan=1 colspan=1>CTX-M-121  CTX-M-126</td><td rowspan=1 colspan=2>CTX-M-152</td><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>CTX-M-119</td><td rowspan=1 colspan=1>CTX-M-145</td></tr><tr><td rowspan=1 colspan=1>CTX-M-129 - CTX-M-132</td><td rowspan=1 colspan=2>CTX-M-155 - CTX-M-177</td><td rowspan=1 colspan=1>CTX-M-120</td><td rowspan=1 colspan=1>CTX-M-146</td></tr><tr><td rowspan=1 colspan=1>CTX-M-134</td><td rowspan=1 colspan=2>CTX-M-179 - CTX-M-185</td><td rowspan=1 colspan=1>CTX-M-127</td><td rowspan=1 colspan=1>CTX-M-149</td></tr><tr><td rowspan=1 colspan=1>CTX-M-136 - CTX-M-139</td><td rowspan=4 colspan=2></td><td rowspan=1 colspan=1>CTX-M-128</td><td rowspan=1 colspan=1>CTX-M-153</td></tr><tr><td rowspan=1 colspan=1>CTX-M-141  CTX-M-142</td><td rowspan=1 colspan=1>CTX-M-133</td><td rowspan=1 colspan=1>CTX-M-154</td></tr><tr><td rowspan=1 colspan=1>CTX-M-144</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-135</td><td rowspan=1 colspan=1>CTX-M-178</td></tr><tr><td rowspan=1 colspan=1>CTX-M-147  CTX-M-148</td><td rowspan=1 colspan=1>CTX-M-140</td><td rowspan=1 colspan=1></td></tr></table>

a July 2016; analysis of over 1,200 database CTX-M sequences (typed and untyped).

Table 75. Isolates Containing the $\Delta I a _ { \vert M P }$ gene Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>IMP Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration (copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=5>Isolates Tested</td></tr><tr><td rowspan=3 colspan=1>IMP</td><td rowspan=1 colspan=1>K. aerogenes</td><td rowspan=1 colspan=1>AR-Bank #0161</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=9 colspan=1>IMP Detected</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1062016</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0080</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-1a</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0103</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-3a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1252008</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>GRE 1062081</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>GRE 1062084</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-9</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1252009</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0092</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=5>In silico Reactivity Predictionsb</td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=1>Reduced Reactivityor Not Detected</td><td rowspan=1 colspan=1>Unknown Reactivity(no sequences)</td></tr><tr><td rowspan=2 colspan=1>IMP-1 - IMP-30a</td><td rowspan=2 colspan=1>IMP-40 - IMP-45</td><td rowspan=2 colspan=1>IMP-58 - IMP- 60</td><td rowspan=1 colspan=1>IMP-31</td><td rowspan=1 colspan=1>IMP-36</td></tr><tr><td rowspan=1 colspan=1>IMP-35</td><td rowspan=1 colspan=1>IMP-39</td></tr><tr><td rowspan=2 colspan=1>IMP-32 - IMP-34</td><td rowspan=2 colspan=1>IMP-48 - IMP-49</td><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>IMP-46</td></tr><tr><td rowspan=1 colspan=1>IMP-47</td></tr><tr><td rowspan=2 colspan=1>IMP 37 - IMP-38</td><td rowspan=2 colspan=1>IMP-51- IMP-56</td><td rowspan=1 colspan=1>IMP-50</td></tr><tr><td rowspan=1 colspan=1>IMP-57</td></tr></table>

a Limited or reduced reactivity predicted for 1/36 $( 2 . 8 \% )$ of IMP-1 and 1/3 $( 3 3 . 3 \% )$ of IMP-3 sequences. b June 2016; analysis of over 220 database IMP sequences (typed and untyped).

Table 76. Isolates Containing the $\scriptstyle b _ { \pmb { i } } _ { \pmb { a } _ { \mathsf { K P C } } }$ gene Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>KPC Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=5>Isolates Tested</td></tr><tr><td rowspan=6 colspan=1>KPC</td><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>ATCC BAA-2341</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=15 colspan=1>KPC Detected</td></tr><tr><td rowspan=1 colspan=1>E. hormaechi</td><td rowspan=1 colspan=1>ATCC BAA-2082</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AR-Bank #0156</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>AR-Bank #0147</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0097</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=5 colspan=1>KPC-2</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix 0801886</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>JMI 328</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>KPC-3</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0061</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>KPC-5</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Creighton University PS28</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>In silico Reactivity Predictionsa</td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Unknown Reactivity(no sequences)</td></tr><tr><td rowspan=1 colspan=1>KPC-1-19</td><td rowspan=1 colspan=1>KPC-21-22</td><td rowspan=1 colspan=1>KPC-24-26</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>KPC-20KPC-23</td></tr></table>

a August 2016; analysis of approximately 1,125 database KPC sequences (typed and untyped).

Table 77. Isolates Containing the $\pmb { b } / \mathsf { a } _ { \mathsf { N D M } }$ gene Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>NDM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Isolates Tested</td><td></td><td></td></tr><tr><td rowspan=3 colspan=1>NDM</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0162</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=3 colspan=1>NDM Detected</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0153</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0068</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=1>NDM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AR-Bank #0159</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NDM-1a</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>AR-Bank #0033</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>GRE 1153064</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0150</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0137</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>In silico Reactivity Predictions</td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=2>Not Detected</td></tr><tr><td rowspan=1 colspan=1>NDM-1a</td><td rowspan=1 colspan=1>NDM-7</td><td rowspan=1 colspan=1>NDM-13</td><td rowspan=6 colspan=2>None</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>NDM-8</td><td rowspan=1 colspan=1>NDM-14</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td><td rowspan=1 colspan=1>NDM-9</td><td rowspan=1 colspan=1>NDM-15</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=1 colspan=1>NDM-10</td><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>NDM-11</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NDM-6</td><td rowspan=1 colspan=1>NDM-12</td></tr></table>

a Limited or reduced reactivity is predicted for $3 / 4 3 0 \mathrm { N D M - 1 }$ sequences $( 0 . 7 \% )$ b June 2016; analysis of 900 database NDM sequences (typed and untyped).

Table 78. Isolates Containing the $b 1 a _ { 0 } \times a - 4 8$ and like genes Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>OXA-48-like Typea</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td></td><td></td><td rowspan=1 colspan=3>Isolates Tested and Detected</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>K. aerogenes</td><td rowspan=1 colspan=1>AR-Bank #0074</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=6 colspan=1>OXA-48-like Detected</td></tr><tr><td rowspan=2 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>GRE 1411136</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>GRE 1411137</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-162</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>GRE 1355030</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0068</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-232</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0075</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>In silico Reactivity Predictionsa</td><td></td><td></td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Not Detectedb</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>OXA-204</td><td rowspan=1 colspan=1>OXA-370</td><td rowspan=1 colspan=1>OXA-163c</td><td rowspan=1 colspan=1>OXA-438c</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>OXA-232</td><td rowspan=1 colspan=1>OXA-484</td><td rowspan=1 colspan=1>OXA-247c</td><td rowspan=1 colspan=1>OXA-439c</td></tr><tr><td rowspan=1 colspan=1>OXA-162</td><td rowspan=1 colspan=1>OXA-244</td><td rowspan=1 colspan=1>OXA-505</td><td rowspan=1 colspan=1>OXA-405c</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>OXA-245</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-416</td></tr><tr><td rowspan=1 colspan=1>OXA-199</td><td rowspan=1 colspan=1>OXA-252</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-436c</td></tr></table>

a June 2016; analysis of 165 database OXA-48-like sequences (typed and untyped). b Sequence analysis predicts that the listed OXA-48-like types will not be detected. Non-OXA-48-like types (e.g. OXA-23-like, OXA-40/24-like, OXA-51- like, and OXA-58-like, OXA-143a-like and OXA-143-like) will also not be detected. c Deletion variants with altered carbapenem hydrolysis activity, as described for OXA- $1 6 3 ^ { 1 3 0 }$ .

Table 79. Isolates Containing the blaVIM gene Tested and Detected, and In Silico Reactivity Predictions   

<table><tr><td rowspan=1 colspan=1>VIM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=5>Isolates Tested</td></tr><tr><td rowspan=3 colspan=1>VIM</td><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>AR-Bank #0154</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>VIM Detected</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0111</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank 0076</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-2a</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0100</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0054</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-7</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1256018</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-10</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=5>In silico Reactivity Predictions</td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=1>Reduced Reactivityor Not Detected</td><td rowspan=1 colspan=1>Unknown Reactivity(no sequences)</td></tr><tr><td rowspan=3 colspan=1>VIM-1  VIM-20a</td><td rowspan=3 colspan=1>VIM-23 - VIM-47</td><td rowspan=3 colspan=1>VIM-49 - VIM-51</td><td rowspan=1 colspan=1>VIM-39</td><td rowspan=1 colspan=1>VIM-21</td></tr><tr><td rowspan=1 colspan=1>VIM-45</td><td rowspan=1 colspan=1>VIM-22</td></tr><tr><td rowspan=1 colspan=1>VIM-46</td><td rowspan=1 colspan=1>VIM-48</td></tr></table>

a Limited or reduced reactivity is predicted for 2/177 VIM-2 sequences $\left( 1 . 1 \% \right)$ . b September 2016; analysis of over 600 database VIM sequences (typed and untyped).

# Analytical Specificity (Cross-Reactivity and Exclusivity)

There is a risk of false positive results due to non-specific amplification and/or cross-reactivity with organisms found in the respiratory tract. The potential for non-specific amplification and detection by BioFire Diagnostics 510(k)   
FilmArray Pneumonia Panel

the FilmArray Pneumonia Panel assays was evaluated by in silico analyses of available sequences and by empirical (wet) testing of high concentrations of organisms in contrived samples. The observed and predicted cross-reactivities for organisms closely related to those detected by the panel and unrelated organisms that may be present in lower respiratory specimens are summarized in Table 80. Erroneous results due to cross-reactivity with organisms that were not evaluated or new variant sequences that emerge is also possible.

On-panel organisms were tested to assess the potential for intra-panel cross-reactivity (Table 81Error! Reference source not found.). Off-panel organisms included species of the same genus or otherwise genetically related to organisms detected by the panel, as well as normal flora and pathogens that may be present in sputum-like and BAL-like specimens (Table 82). Antimicrobial resistance genes were also evaluated in conjunction with on and off panel host organisms.

The final concentration of analyte in the sample (typically ${ \ge } 1 . 0 \mathrm { E } { + } 0 7$ CFU/mL for bacteria and fungi and ${ \ge } 1 . 0 \mathrm { E } { + } 0 5$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses) represented levels ${ \sim } 1 0 0 - 1 0 0 { , } 0 0 0$ fold higher than the LoD or lowest reportable level of the FilmArray Pneumonia Panel assays.

Table 80. Observed and Predicted Cross-Reactivity of FilmArray Pneumonia Panel Assays   

<table><tr><td rowspan=1 colspan=1>FilmArray Pneumonia Panel Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism</td></tr><tr><td></td><td rowspan=1 colspan=1>Closely-Related Species</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia fergusoniia</td></tr><tr><td rowspan=1 colspan=1>Shigella species (S. boydii, S. dysenteriae, S. flexneri, S. sonnei)a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Klebsiella michiganensisa</td></tr><tr><td rowspan=2 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus argenteusb</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schweitzeri</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas putidad</td></tr><tr><td rowspan=1 colspan=2>Unrelated Species</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Bordetella species</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virusf</td><td rowspan=1 colspan=1>Aspergillus niger</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus laurentii</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus uniguttulatus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Stenotrophomonas acidaminiphila</td></tr><tr><td rowspan=1 colspan=1>CTX-Mh</td><td rowspan=1 colspan=1>Acinetobacter schindleriBurkholderia vietnamiensis</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli.k</td><td rowspan=1 colspan=1>Lelliottia amnigena (Enterobacter amnigenus)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td></tr><tr><td rowspan=1 colspan=1>Enterobacter ludwigii</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td></tr></table>

# able 81. On Panel Organisms Tested for Evaluation of FilmArray Pneumonia Panel Analytical Specificity

False positive results were observed when testing the species shown in bold.

Table 82. Off-Panel Bacteria Tested or Evaluated by In Silico Analysis for FilmArray Pneumonia Panel Analytical Specificity   

<table><tr><td rowspan=1 colspan=4>ON-PANEL</td></tr><tr><td rowspan=1 colspan=4>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniae</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Enterobacter ludwigir</td><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter nosocomialis</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter pittii</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Proteus hauseri</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Enterobacter asburiae</td><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaea</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Proteus penneri</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CTX-M(Klebsiella oxytoca)</td><td rowspan=1 colspan=1>KPC(Klebsiella pneumoniae)</td><td rowspan=1 colspan=1>OXA-48-like(Serratia marcescens)</td><td rowspan=1 colspan=1>mecA and MREJ(Staphylococcus aureus)</td></tr><tr><td rowspan=1 colspan=1>IMP(Pseudomonas aeruginosa)</td><td rowspan=1 colspan=1>NDM(Acinetobacter baumannii)</td><td rowspan=1 colspan=1>VIM(Enterobacter cloacae)</td><td rowspan=1 colspan=1></td></tr></table>

Table 80Error! Reference source not found. for cross-reactivity information.

False positive results were observed when testing the species shown in bold.   

<table><tr><td rowspan=1 colspan=4>OFF-PANEL</td></tr><tr><td rowspan=1 colspan=4>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia defectiva</td><td rowspan=1 colspan=1>Escherichia fergusonira</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>Shigella boydii a</td></tr><tr><td rowspan=1 colspan=1>Achromobacterxylosoxidans</td><td rowspan=1 colspan=1>Escherichia hermanii</td><td rowspan=1 colspan=1>Mycoplasma bovis</td><td rowspan=1 colspan=1>Shigella dysenteriaea</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter haemolyticus</td><td rowspan=1 colspan=1>Escherichia vulneris</td><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>Shigella flexneria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter johnsonii</td><td rowspan=1 colspan=1>Fluoribacter dumoffei</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Shigella sonner</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter junii</td><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>Mycoplasma orale</td><td rowspan=1 colspan=1>Staphylococcus argenteusa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwolfi</td><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Staphylococcus capitis</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter radioresistens</td><td rowspan=1 colspan=1>Granulicatella adiacens</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>Staphylococcus caprae</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter schindleri²</td><td rowspan=1 colspan=1>Haemophilus ducreyi</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Staphylococcus cohnii</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter ursingii</td><td rowspan=1 colspan=1>Haemophilus haemolyticus</td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>Staphylococcus haemolyticus</td></tr><tr><td rowspan=1 colspan=1>Actinobacillusactinomycetemcomitans</td><td rowspan=1 colspan=1>Haemophilus parahaemolyticus</td><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>Staphylococcus epidermidis(mecA)</td></tr><tr><td rowspan=1 colspan=1>Actinobacillus hominis</td><td rowspan=1 colspan=1>Haemophilus parainfluenzae</td><td rowspan=1 colspan=1>Nocardia asteroides</td><td rowspan=1 colspan=1>Staphylococcus hominis</td></tr><tr><td rowspan=1 colspan=1>Actinobacillus ureae</td><td rowspan=1 colspan=1>Haemophilus parasuis</td><td rowspan=1 colspan=1>Nocardia brasilensis</td><td rowspan=1 colspan=1>Staphylococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Actinomyces isrealii</td><td rowspan=1 colspan=1>Haemophilus sputorumb</td><td rowspan=1 colspan=1>Pantoea agglomerans</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td></tr><tr><td rowspan=1 colspan=1>Actinomyces naeslundii</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>Pasteurella multocida</td><td rowspan=1 colspan=1>Staphylococcus lutrae</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Hafnia paralvei</td><td rowspan=1 colspan=1>Pediococcus acidilactici</td><td rowspan=1 colspan=1>Staphylococcus pasteuri</td></tr><tr><td rowspan=1 colspan=1>Bacteriodes fragilis</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>Staphylococcus pseudointermedius</td></tr><tr><td rowspan=1 colspan=1>Bordatella bronchiseptica</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Pluralibacter gergoviae</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>Bordatella parapertussis</td><td rowspan=1 colspan=1>Klebsiella michiganensisa</td><td rowspan=1 colspan=1>Porphyromonas gingivalis</td><td rowspan=1 colspan=1>Staphylococcus schleiferi</td></tr><tr><td rowspan=1 colspan=1>Bordatella pertussisa</td><td rowspan=1 colspan=1>Kluyvera intermedia</td><td rowspan=1 colspan=1>Prevotella intermedia</td><td rowspan=1 colspan=1>Staphylococcus schweitzeri²</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>Kluyvera ascorbata</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>Staphylococcus sciuri</td></tr><tr><td rowspan=1 colspan=1>Burkholderia mallei</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Prevotella oralis</td><td rowspan=1 colspan=1>Staphylococcus warneri</td></tr><tr><td rowspan=1 colspan=1>Burkholderia multivorans</td><td rowspan=1 colspan=1>Leclercia adecarboxylata</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Staphylococcus xylosus</td></tr><tr><td rowspan=1 colspan=1>Burkholderia pseudomallei</td><td rowspan=1 colspan=1>Legionellabozemanii</td><td rowspan=1 colspan=1>Providencia rettgeri(OXA-48-like)</td><td rowspan=1 colspan=1>Stenotrophomonasacidaminiphila²</td></tr><tr><td rowspan=1 colspan=1>Cardiobacterium hominis</td><td rowspan=1 colspan=1>Legionella cincinnatiensis</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Cedecea davisae</td><td rowspan=1 colspan=1>Legionella feeleii</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td><td rowspan=1 colspan=1>Stenotrophomonas nitritireducens</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Legionella lansingensis</td><td rowspan=1 colspan=1>Pseudomonas luteola</td><td rowspan=1 colspan=1>Stenotrophomonas rhizophila</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila psittaci</td><td rowspan=1 colspan=1>Legionella longbeachae</td><td rowspan=1 colspan=1>Pseudomonas nitroreducens</td><td rowspan=1 colspan=1>Streptococcus   equi   subsp.Zooepidemicus</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii (KPC)</td><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>Pseudomonas oryzihabitans</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri(OXA-48-like)</td><td rowspan=1 colspan=1>Legionella wadsworthii</td><td rowspan=1 colspan=1>Pseudomonas pertucinogena</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>Lelliottia nimipressuralis</td><td rowspan=1 colspan=1>Pseudomonas putida² (IMP)</td><td rowspan=1 colspan=1>Streptococcus oralis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter werkmanii (VIM)</td><td rowspan=1 colspan=1>Lelliottia amnigenaa(Enterobacter amnigenus)</td><td rowspan=1 colspan=1>Pseudomonas stutzeri</td><td rowspan=1 colspan=1>Streptococcus parasanguinis</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Leuconostoc lactis</td><td rowspan=1 colspan=1>Ralstonia pickettii</td><td rowspan=1 colspan=1>Streptococcus pseudopneumoniae</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Raoultella ornithinolytica</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diptheriae</td><td rowspan=1 colspan=1>Macrococcus caseolyticus</td><td rowspan=1 colspan=1>Raoultella planticola</td><td rowspan=1 colspan=1>Streptococcus sanguinis</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=1 colspan=4>OFF-PANEL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>Raoultella terrigena</td><td rowspan=1 colspan=1>Streptococcus tigurinus</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiptherticum</td><td rowspan=1 colspan=1>Moraxella equi</td><td rowspan=1 colspan=1>Rhodococcus equi</td><td rowspan=1 colspan=1>Streptomyces anulatus</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium urealyticum</td><td rowspan=1 colspan=1>Moraxella lacunata</td><td rowspan=1 colspan=1>Rothia mucilaginosa</td><td rowspan=1 colspan=1>Treponema denticola</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>Moraxella lincolnii</td><td rowspan=1 colspan=1>Salmonella enterica (CTX-M)</td><td rowspan=1 colspan=1>Ureaplasma parvum</td></tr><tr><td rowspan=1 colspan=1>Eikenella corrodens</td><td rowspan=1 colspan=1>Moraxella nonliquiefaciens</td><td rowspan=1 colspan=1>Serratia fonticola</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cancerogenus</td><td rowspan=1 colspan=1>Morganella morganii (NDM)</td><td rowspan=1 colspan=1>Serratia liquefaciens</td><td rowspan=1 colspan=1>Vagococcus fluvialis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter massiliensis</td><td rowspan=1 colspan=1>Mycobacterium africanumb</td><td rowspan=1 colspan=1>Serratia odorifera</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Enterobacter soli</td><td rowspan=1 colspan=1>Mycobacterium bovis</td><td rowspan=1 colspan=1>Serratia plymuthica</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Mycobacterium caprae</td><td rowspan=1 colspan=1>Serratia rubidaea</td><td rowspan=1 colspan=1>Yersinia pseudotuberculosis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Mycobacterium microtib</td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Bocavirus</td><td rowspan=1 colspan=1>Hantavirus</td><td rowspan=1 colspan=1>Human Papillomavirus (HPV)</td><td rowspan=1 colspan=1>Mumps virus</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Herpes simplex virus 1</td><td rowspan=1 colspan=1>Influenza C</td><td rowspan=1 colspan=1>Varicella zoster virus</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr virus</td><td rowspan=1 colspan=1>Human Immunodeficiency Virus(HIV)</td><td rowspan=1 colspan=1>Middle East Respiratory SyndromeCoronavirus (MERS-CoV)</td><td rowspan=1 colspan=1>Severe Acute RespiratorySyndrome Coronavirus (SARS-CoV)</td></tr><tr><td rowspan=1 colspan=1>German Measles Virus (Rubella)</td><td rowspan=1 colspan=1>Measles Virus (Rubeola)</td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=1>Coccidioides posadasii</td><td rowspan=1 colspan=1>Fusarium kyushense</td><td rowspan=1 colspan=1>Pneumocystis carinii</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>Cryptococcus albidus</td><td rowspan=1 colspan=1>Histoplasma capsulatum</td><td rowspan=1 colspan=1>Pneumocystis jirovecii</td></tr><tr><td rowspan=1 colspan=1>Aspergillus niger</td><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>Paecilomyces variottii</td><td rowspan=1 colspan=1>Pneumocystis murina</td></tr><tr><td rowspan=1 colspan=1>Aspergillus terreus</td><td rowspan=1 colspan=1>Cryptococcus laurentiia</td><td rowspan=1 colspan=1>Paracoccidodes brasiliensis</td><td rowspan=1 colspan=1>Rhizopus microsporus</td></tr><tr><td rowspan=1 colspan=1>Blastomyces dermatitidis</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Penicillium chrysogenum</td><td rowspan=1 colspan=1>Scedosporium apiospermum</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Cryptococcus uniguttalatusa</td><td rowspan=1 colspan=1>Penicillium marneffei</td><td rowspan=1 colspan=1>Scedosporium prolificans</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Filobasidium capsuligenum</td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>AmpC(Klebsiella (Enterobacter)aerogenes)</td><td rowspan=1 colspan=1>OXA-24/40 (non-48-like)(Acinetobacter baumannii)</td><td rowspan=1 colspan=1>SME(Serratia marcescens)</td><td rowspan=1 colspan=1>TEM(Escherichia coli)</td></tr><tr><td rowspan=1 colspan=1>CMY (II)(Escherichia coli)</td><td rowspan=1 colspan=1>SHV(Klebsiella pneumoniae)</td><td rowspan=1 colspan=1>SPM(Pseudomonas aeruginosa)</td><td rowspan=1 colspan=1>VAN(Staphylococcus aureus)</td></tr><tr><td rowspan=1 colspan=1>ompK36 [SHV-12,OMPC](Klebsiella pneumoniae)</td><td rowspan=1 colspan=2>SCCmec variant lacking mecA or mecC (Staphylococcus aureus)d</td><td rowspan=1 colspan=1></td></tr></table>

a See Table 80Error! Reference source not found. for cross-reactivity information.

b Analytical specificity was evaluated only by in silico analysis of whole genome or partial genome sequences in public databases. No cross-reactivity is predicted based on the sequences analyzed.

c Tested at a concentration less than $1 . 0 \mathrm { E } { + 0 7 }$ CFU/mL and also evaluated by in silico analysis. Cross-reactivity was not observed in testing nor predicted based on the sequences analyzed.

d Methicillin-sensitive isolate of S. aureus (Rennes 1060728) that has MREJ sequence but no mecA or mecC gene (empty cassette). Staphylococcus aureus was reported as Detected and the mecA/C and MREJ result was Not Detected

# Precision (Reproducibility)

Precision (Reproducibility) testing was performed with contrived BAL samples over multiple days at three laboratory locations (sites) on a combination of FilmArray, FilmArray 2.0 and FilmArray Torch systems. The testing incorporated a range of potential variation introduced by operator, system, instrument or Torch module, concentration and reagent lot, for a total of 30 tests per system and 90 total replicates per sample/concentration.

Evaluation of the reproducibility of Detected/Not Detected results for atypical bacteria and viruses included samples containing combinations of five different analytes, at Negative, Low Positive $\left( 1 \times \mathrm { L o D } \right)$ , and Moderate Positive $\scriptstyle ( 3 \times \mathbf { L o D } )$ concentrations. Negative results were obtained from samples that were not spiked with the analyte (see evaluation of precision for bacterial analytes below).

A summary of results (percent $( \% )$ agreement with the expected Detected or Not Detected result) for atypical bacteria and viruses (by site and system) is provided in Table 83Error! Reference source not found..

Table 83. Reproducibility of FilmArray Pneumonia Panel Atypical Bacteria and Virus Results   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>Concentration Tested</td><td rowspan=3 colspan=1>Expected Result</td><td rowspan=1 colspan=4>Agreement with Expected Result</td><td></td></tr><tr><td rowspan=1 colspan=1>FilmArray</td><td rowspan=1 colspan=1>FilmArray 2.0</td><td rowspan=1 colspan=1>FilmArray Torch</td><td rowspan=2 colspan=1>All Sites/Systems[95% CI]</td><td></td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Atypical Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>2,340/2,34000%[99.8%-100%]</td><td></td></tr><tr><td rowspan=3 colspan=1>Legionella pneumophilaPhiladelphia-1ATCC 33152</td><td rowspan=1 colspan=1>Moderate Positive3× LoD1.5E+03 CFU/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>Low Positive1× LoD5.0E+02 CFU/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>2,160/2,16000%[99.8%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>2,340/2,340100%[99.8%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=3 colspan=1>AdenovirusSpecies B Serotype 3ZeptoMetrix 0810062CF</td><td rowspan=1 colspan=1>Moderate Positive3× LoD3.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>Low Positive1× LoD1.0E+00 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>2,160/2,160100%[99.8%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>776/7809.5%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>2,336/2,34099.8%[99.6%-100%]</td><td></td></tr><tr><td rowspan=3 colspan=1>Human Metapneumovirus16 Type A1ZeptoMetrix 0810161CF</td><td rowspan=1 colspan=1>Moderate Positive3× LoD1.5E+02 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>Low Positive1× LoD5.0E+01 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.9%[94.0%-100%]</td><td></td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>2,160/2,16000%[99.8%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>779/78099.9%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>779/78099.9%</td><td rowspan=1 colspan=1>2,338/2,34099.9%[99.7%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=3 colspan=1>Influenza A3N2A/Port Chalmers/1/73ATCC VR-810</td><td rowspan=1 colspan=1>Moderate Positive3× LoD1.5E+00 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>Low Positive1× LoD0.5E-01 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.9%[94.0%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>2,160/2,160100%[99.8%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>2,340/2,340100%[99.8%-100%]</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza VirusType 2ZeptoMetrix 0810015CF</td><td rowspan=1 colspan=1>Moderate Positive3× LoD7.5E+01 TCID5/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td><td rowspan=1 colspan=1>|</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>Concentration Tested</td><td rowspan=3 colspan=1>Expected Result</td><td rowspan=1 colspan=4>Agreement with Expected Result</td></tr><tr><td rowspan=1 colspan=1>FilmArray</td><td rowspan=1 colspan=1>FilmArray 2.0</td><td rowspan=1 colspan=1>FilmArray Torch</td><td rowspan=2 colspan=1>All Sites/Systems95% CI]</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Low Positive1× LoD2.5E+01 TCID50/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%[96.0%-100%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>720/720100%</td><td rowspan=1 colspan=1>2,160/2,160100%[99.8%-100%]</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/780100%</td><td rowspan=1 colspan=1>780/78010%</td><td rowspan=1 colspan=1>2,340/2,340100%[99.8%-100%]</td></tr></table>

Precision for bacterial analytes was measured at each concentration as 1) precision of bin results and 2) reproducibility of analyte detection. When a sample containing one or more bacteria is tested repeatedly, the precision of the bin results (probability that each replicate will receive the same bin result) will vary based on the concentration of nucleic acid measured and the relation of that concentration to the limits of each bin. Bin precision may be as low as $50 \%$ for values at a bin limit and precision will increase (up to $90 \%$ or higher) as the distance of the measured value from a bin limit increases. The precision of FilmArray Pneumonia Panel bin results will follow the model illustrated in Figure 1Error! Reference source not found.:

• $5 9 0 \%$ at a bin center (Scenario 1) ${ \sim } 6 0 - 9 0 \%$ between a bin limit and bin center (Scenario 2)   
• ${ \sim } 5 0 \%$ at bin limits (Scenario 3)

![](images/e452beea8c985d0597175b727547b79c7a33dc9336b56370c16c1159b2675b9b.jpg)  
Figure 1. Model for Precision of FilmArray Pneumonia Panel Bin Results Top: The probability (0.0 – 1.0) of the same bin results for each replicate tested varies based on proximity of the measured value to a bin limit. Bottom: Expected distribution of bin results at different mean measured values.

Samples containing bacteria and corresponding antimicrobial resistance (AMR) genes were tested at six different concentrations over the reportable range and below. A summary of the bin precision (percent $( \% )$ of replicates reported in each bin) and the reproducibility of detection is shown at each concentration tested in Table 84Error! Reference source not found..

Table 84. Reproducibility of FilmArray Pneumonia Panel Bacterial Bin Results on FilmArray, FilmArray 2.0 and FilmArray Torch Grey shading indicates the expected bin results based on the analyte concentration and bold font indicates the bin with the greatest percentage of results at each concentration. BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Concentration(log1o copies/mL)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>% Replicates Reported in Each Bin Result</td><td rowspan=2 colspan=1>TotalDetected</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=7 colspan=1>Acinetobacter baumannii(NDM-1)AR-BANK#0033</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>87/90(96.7%)</td><td rowspan=1 colspan=1>3/90(3.3%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>82/90(91.1%)</td><td rowspan=1 colspan=1>8/90(8.9%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>80/90(88.9%)</td><td rowspan=1 colspan=1>9/90(10.0%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>74/90(82.2%)</td><td rowspan=1 colspan=1>15/90(16.7%)</td><td rowspan=1 colspan=1>75/9083.3%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/901.1%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Enterobacter cloacae(VIM)AR-BANK#0154</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/90(95.6%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>80/90(88.9%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/9095.6%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>83/90(92.2%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>85/90(94.4%)</td><td rowspan=1 colspan=1>5/905.6%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Escherichia coli(IMP)GRE 1062016</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>7/90(7.8%)</td><td rowspan=1 colspan=1>82/90(91.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10/90(11.1%)</td><td rowspan=1 colspan=1>80/90(88.9%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>12/90(13.3%)</td><td rowspan=1 colspan=1>77/90(85.6%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>15/90(16.7%)</td><td rowspan=1 colspan=1>74/90(82.2%)</td><td rowspan=1 colspan=1>16/9017.8%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>HaemophilusinfluenzaeATCC 10211</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>35/90(48.9%)</td><td rowspan=1 colspan=1>55/90(61.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>49/90(54.4%)</td><td rowspan=1 colspan=1>40/90(44.4%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>41/90(45.6%)</td><td rowspan=1 colspan=1>49/90(54.4%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>42/90(46.7%)</td><td rowspan=1 colspan=1>48/90(53.3%)</td><td rowspan=1 colspan=1>42/9046.7%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=6 colspan=1>Klebsiella aerogenes(Enterobacter aerogenes)ATCC 13048</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>65/90(72.2%)</td><td rowspan=1 colspan=1>25/90(27.8%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>52/90(57.8%)</td><td rowspan=1 colspan=1>38/90(42.2%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>38/90(42.2%)</td><td rowspan=1 colspan=1>51/90(56.7%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>33/90(36.7%)</td><td rowspan=1 colspan=1>57/90(63.3%)</td><td rowspan=1 colspan=1>33/9036.7%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/901.1%</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Concentration(log10 copies/mL)</td><td rowspan=1 colspan=4>% Replicates Reported in Each Bin Result</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>TotalDetected</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Klebsiella oxytoca(CTX-M)GRE 1254054</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>84/90(93.3%)</td><td rowspan=1 colspan=1>3/90(3.3%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/90(2.2%)</td><td rowspan=1 colspan=1>88/9097.8%</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>88/90(97.8%)</td><td rowspan=1 colspan=1>2/902.2%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Klebsiella pneumoniae(KPC)AR-BANK#0097</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>12/90(13.3%)</td><td rowspan=1 colspan=1>78/90(86.7%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>15/90(16.7%)</td><td rowspan=1 colspan=1>75/90(83.3%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>23/90(25.6%)</td><td rowspan=1 colspan=1>66/90(73.3%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15/90(16.7%)</td><td rowspan=1 colspan=1>75/90(83.3%)</td><td rowspan=1 colspan=1>15/9016.7%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Moraxella catarrhalisATCC 8176</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>26/90(28.9%)</td><td rowspan=1 colspan=1>64/90(71.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>83/90(92.2%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/90(95.6%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/90(95.6%)</td><td rowspan=1 colspan=1>4/904.4%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Proteus mirabilisATCC 35659</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>88/90(97.8%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/90(2.2%)</td><td rowspan=1 colspan=1>88/9097.8%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>27/90(30.0%)</td><td rowspan=1 colspan=1>63/90(70.0%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>26/90(28.9%)</td><td rowspan=1 colspan=1>64/90(71.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>14/90(15.6%)</td><td rowspan=1 colspan=1>75/90(83.3%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28/90(31.1%)</td><td rowspan=1 colspan=1>62/90(68.9%)</td><td rowspan=1 colspan=1>28/9031.1%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=5 colspan=1>Pseudomonas aeruginosaATCC 10145</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>20/90(22.2%)</td><td rowspan=1 colspan=1>70/90(77.8%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>24/90(26.7%)</td><td rowspan=1 colspan=1>66/90(73.3%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>,</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>16/90(17.8%)</td><td rowspan=1 colspan=1>74/90(82.2%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>14/90(15.6%)</td><td rowspan=1 colspan=1>76/90(84.4%)</td><td rowspan=1 colspan=1>14/9015.6%</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Concentration(log10 copies/mL)</td><td rowspan=1 colspan=4>% Replicates Reported in Each Bin Result</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>TotalDetected</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Serratia marcescens(OXA-48-like)GRE 1659005</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>2/90(2.2%)</td><td rowspan=1 colspan=1>88/90(97.8%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7/90(7.8%)</td><td rowspan=1 colspan=1>83/90(92.2%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>83/90(92.2%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>84/90(93.3%)</td><td rowspan=1 colspan=1>6/906.7%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Staphylococcus aureussubsp. aureus(mecA/ and MREJ)ATCC 43300</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2/1260(0.2%)</td><td rowspan=1 colspan=1>1258/1260(99.8%)</td><td rowspan=1 colspan=1>2/12600.2%</td></tr><tr><td rowspan=7 colspan=1>Streptococcus agalactiaeATCC 13813</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>88/90(97.8%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>86/90(95.6%)</td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/9095.6%</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/90(3.3%)</td><td rowspan=1 colspan=1>87/90(96.7%)</td><td rowspan=1 colspan=1>3/903.3%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=7 colspan=1>Streptococcus pneumoniaeATCC 6303</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1>89/9098.9%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>0/900.0%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr><tr><td rowspan=4 colspan=1>Streptococcus pyogenesATCC 49399</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>5/90(5.6%)</td><td rowspan=1 colspan=1>84/90(93.3%)</td><td rowspan=1 colspan=1>1/90(1.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/90(4.4%)</td><td rowspan=1 colspan=1>86/90(95.6%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Concentration(log10 copies/mL)</td><td rowspan=1 colspan=5>% Replicates Reported in Each Bin Result</td><td rowspan=2 colspan=1>TotalDetected</td></tr><tr><td rowspan=1 colspan=1>≥10^7</td><td rowspan=1 colspan=1>10^6</td><td rowspan=1 colspan=1>10^5</td><td rowspan=1 colspan=1>10^4</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/90(3.3%)</td><td rowspan=1 colspan=1>87/90(96.7%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>16/90(18.9%)</td><td rowspan=1 colspan=1>74/90(81.1%)</td><td rowspan=1 colspan=1>16/9017.8%</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1800/1800(100%)</td><td rowspan=1 colspan=1>0/18000.0%</td></tr></table>

The precision of the antimicrobial resistance (AMR) genes was measured as the reproducibility of analyte detection on each system and overall, presented in Error! Reference source not found. as the percent of replicates that are detected at concentrations of the associated bacterium that are within the reportable range, or below the reportable range, as well as the percent agreement with the expected Not Detected result in unspiked samples.

Table 85. Reproducibility of FilmArray Pneumonia Panel Antimicrobial Resistance Gene Results on FilmArray, FilmArray 2.0 and FilmArray Torch   

<table><tr><td rowspan=3 colspan=1>AMR GeneOrganism</td><td rowspan=3 colspan=1>Concentration ofOrganism(log10 copies/mL)</td><td rowspan=3 colspan=1>ExpectedResult</td><td rowspan=1 colspan=4>Agreement with the Expected Result</td></tr><tr><td rowspan=1 colspan=1>FilmArray</td><td rowspan=1 colspan=1>FilmArray 2.0</td><td rowspan=1 colspan=1>FilmArrayTorch</td><td rowspan=2 colspan=1>All Systems/Sites5% Cl]</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=3 colspan=1>CTX-MKlebsiella oxytocaGRE 1254054</td><td rowspan=1 colspan=1>Reportable Range(3.5 - 7.5)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>149/150a99.3%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>449/450a99.8%[98.8%-99.9%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.5)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>0/300.0%</td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1>0/300.0%</td><td rowspan=1 colspan=1>1/901.1%[0.03%-6.0%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>599/60099.8%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1799/180099.9%[99.7%-100%]</td></tr><tr><td rowspan=3 colspan=1>IMPEscherichia coliGRE 1062016</td><td rowspan=1 colspan=1>Reportable Range(4.0 - 7.0)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>360/36000%[99.0%-100%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.0-3.0)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>10/6016.7%</td><td rowspan=1 colspan=1>9/6015.0%</td><td rowspan=1 colspan=1>3/605.0%</td><td rowspan=1 colspan=1>22/18012.2%[7.8%-17.9%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1800/1800100%[99.8%-100%]</td></tr><tr><td rowspan=3 colspan=1>KPCKlebsiella pneumoniaeAR-Bank#0097</td><td rowspan=1 colspan=1>Reportable Range(4.0 - 7.0)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>119/120b99.2%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>359/360b99.7%[98.5%-100%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.0-3.0)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>14/6023.3%</td><td rowspan=1 colspan=1>12/6020.0%</td><td rowspan=1 colspan=1>9/6015.0%</td><td rowspan=1 colspan=1>35/18019.4%[13.9%-25.0%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1800/1800100%[99.8%-100%]</td></tr><tr><td rowspan=3 colspan=1>mecA/C and MREJStaphylococcus aureusATCC 43300</td><td rowspan=1 colspan=1>Reportable Range(4.0 - 7.0)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>119/120b99.2%</td><td rowspan=1 colspan=1>118/120b98.3%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>357/360b99.2%[97.6%-99.8%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.0-3.0)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>0/600.0%</td><td rowspan=1 colspan=1>0/600.0%</td><td rowspan=1 colspan=1>0/600.0%</td><td rowspan=1 colspan=1>0/1800%[0.0%-2.0%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>1260/1260100%[99.7%-100%]</td></tr><tr><td rowspan=3 colspan=1>NDMAcinetobacterbaumanniiAR-Bank#0033</td><td rowspan=1 colspan=1>Reportable Range(3.5 - 7.5)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>149/150a99.3%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>449/450a99.8%[98.8%-100%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.5)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1>0/300.0%</td><td rowspan=1 colspan=1>2/902.2%[0.3%-7.8%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>599/60099.8%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1799/180099.9%[99.7%-100%]</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like Serratia</td><td rowspan=1 colspan=1>Reportable Range</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>119/120b</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>359/360b</td></tr></table>

BioFire Diagnostics 510(k) FilmArray Pneumonia Panel

<table><tr><td rowspan=3 colspan=1>AMR GeneOrganism</td><td rowspan=3 colspan=1>Concentration ofOrganism(log1o copies/mL)</td><td rowspan=3 colspan=1>ExpectedResult</td><td rowspan=1 colspan=4>Agreement with the Expected Result</td></tr><tr><td rowspan=1 colspan=1>FilmArray</td><td rowspan=1 colspan=1>FilmArray 2.0</td><td rowspan=1 colspan=1>FilmArrayTorch</td><td rowspan=2 colspan=1>All Systems/Sites[95% C</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=3 colspan=1>marcescensGRE 1659005</td><td rowspan=1 colspan=1>(4.0 - 7.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.70%[98.5%-100%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.0-3.0)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>14/6023.3%</td><td rowspan=1 colspan=1>12/6020.0%</td><td rowspan=1 colspan=1>9/6015.0%</td><td rowspan=1 colspan=1>35/18019.4%[13.9%-26.0%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>598/60099.7%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1798/180099.9%[99.6%-100%]</td></tr><tr><td rowspan=3 colspan=1>VIMEnterobacter cloacaeAR-BANK#0154</td><td rowspan=1 colspan=1>Reportable Range(4.0 - 7.0)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>360/36000%[99.0%-100%]</td></tr><tr><td rowspan=1 colspan=1>Below ReportableRange(2.0-3.0)</td><td rowspan=1 colspan=1>Detected(Variable)</td><td rowspan=1 colspan=1>10/6016.7%</td><td rowspan=1 colspan=1>9/6015.0%</td><td rowspan=1 colspan=1>3/605.0%</td><td rowspan=1 colspan=1>22/18012.2%[7.8%-17.9%]</td></tr><tr><td rowspan=1 colspan=1>None(No Analyte)</td><td rowspan=1 colspan=1>N/A orNot Detected</td><td rowspan=1 colspan=1>599/60099.8%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>600/600100%</td><td rowspan=1 colspan=1>1799/180099.9%[99.7%-100%]</td></tr></table>

a CTX-M and NDM Not Detected results observed at the corresponding bacterial concentration of $4 . 5 \log _ { 1 0 } $ copies/mL. b KPC, mecA/C and MREJ, and OXA-48-like Not Detected results observed at the corresponding bacterial concentration of $4 . 0 \log _ { 1 0 } \mathrm { { \Omega } } _ { 1 0 }$ copies/mL.

# Interference

Potentially interfering substances that could be present in BAL-like or sputum-like specimens or that may be introduced during specimen collection and testing were evaluated for their effect on FilmArray Pneumonia Panel performance. Substances included endogenous substances that may be found in specimens at normal or elevated levels (e.g. blood, mucus/mucin, human genomic DNA), various commensal or infectious microorganisms, medications, a variety of sample processing substances and substances used to clean, decontaminate, or disinfect work areas. The performance of the FilmArray Pneumonia Panel has not been established with all potentially interfering medications for the treatment of lower respiratory tract infections. The effect of interfering substances has only been evaluated for those listed in Table 86. Interference from substances that were not evaluated could lead to erroneous results.

Each substance was added to contrived samples containing representative qualitatively reported organisms and representative organisms with bin reporting. Qualitatively reported organisms were at concentrations near $( 2 - 3 \times )$ LoD and those with bin reporting were present at $4 . 0 \log _ { 1 0 }$ (copies $/ \mathrm { m L }$ ) (e.g. in the lowest reported bin). The concentration of substance added to the samples (Table 86) was equal to or greater than the highest level expected to be in BAL-like or Sputum-like specimens.

Four of the evaluated substances were found to interfere with the ability of the FilmArray Pneumonia Panel to report accurate analyte results; Bleach, MycoPrep, $2 \% \ \mathrm { N a O H }$ , and $5 \%$ Oxalic acid. Each of these substances contain chemicals known to react with nucleic acids, altering their chemical structure. The interference observed was related to the inability to detect the chemically modified nucleic acids. Treatment of specimens with these substances prior to FilmArray Pneumonia Panel testing may result in loss of analyte detection, therefore samples that have been in contact with these substances should not be tested using the FilmArray Pneumonia Panel. None of the other substances tested were shown to interfere with the FilmArray Pneumonia Panel results, however, testing of specimens that have been centrifuged or pre-treated by addition of enzyme, media, mucolytic agent, or decontaminating substances is not recommended..

Table 86. Evaluation of Potentially Interfering Substances on the FilmArray Pneumonia Panel   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>Testing Outcome</td></tr><tr><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>60 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>HC1 (gastric acid)</td><td rowspan=1 colspan=1>5 mmol/L</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>2 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Human Cells (K-562 cell line)</td><td rowspan=1 colspan=1>3.8E+06 cells/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Immunoglobulins (IgG)</td><td rowspan=1 colspan=1>60 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>16 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td></td><td rowspan=1 colspan=1>Exogenous Substances</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Albuterol (bronchodilator)</td><td rowspan=1 colspan=1>1.7 μmol/L</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Benzocaine (Orajel)</td><td rowspan=1 colspan=1>1.0 % w/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Epinephrine (hormone, bronchodilator)</td><td rowspan=1 colspan=1>8.3 μg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Galphimia glauca (Homeopathic remedy)</td><td rowspan=1 colspan=1>1.0 % w/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Guaifenesin (expectorant)</td><td rowspan=1 colspan=1>15.2 mmol/L</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Lidocaine</td><td rowspan=1 colspan=1>5.1 mmol/L</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Menthol and cetylpyridinium chloride(Cepacol Mouthwash)</td><td rowspan=1 colspan=1>1.0% v/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Mupirocin (antibiotic)</td><td rowspan=1 colspan=1>6.0 ng/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Nicotine</td><td rowspan=1 colspan=1>6.2 μmol/L</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Pentamidine (antimicrobial)</td><td rowspan=1 colspan=1>1.5 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine hydrochloride (decongestant)</td><td rowspan=1 colspan=1>0.3 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Tobramycin sulfate (antibiotic)</td><td rowspan=1 colspan=1>30 mg/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Zanamivir (influenza antiviral)</td><td rowspan=1 colspan=1>426 ng/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Competitive Microorganisms</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Actinobacillus actinomycetecomitans</td><td rowspan=1 colspan=1>3.8E+07 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>5.5E+07 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>1.7E+07 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>2.5E+05 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>1.4E+06 copies/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1.8E+07 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>8.1E+06 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>3.5E+04 copies/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.9E+07 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>5.9E+06 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>5.5E+06 CFU/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=1>8.7E+07copies/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Disinfection/Cleaning Substances</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Reagent Alcohol</td><td rowspan=1 colspan=1>7.0%</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Bleach</td><td rowspan=1 colspan=1>1.0% v/v(600 ppm chlorine)</td><td rowspan=1 colspan=1>Interference Observed a</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample Processing Materialsa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Copan Snotbuster (active ingredient DTT)</td><td rowspan=1 colspan=1>50.0% v/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Sputolysin (active ingredient DTT)</td><td rowspan=1 colspan=1>50.0% v/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>SPUTASOL (active ingredient DTT + salts)</td><td rowspan=1 colspan=1>50.0% v/v</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>MycoPrep (active ingredient NaOH + NALC)</td><td rowspan=1 colspan=1>50.0% v/v</td><td rowspan=1 colspan=1>Interference Observed b</td></tr><tr><td rowspan=1 colspan=1>NaOH (decontaminant)</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>Interference Observed </td></tr><tr><td rowspan=1 colspan=1>Oxalic Acid (decontaminant)</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>Interference Observed b</td></tr></table>

a FilmArray Pneumonia Panel testing of lower respiratory specimens that have been processed or treated with these or other substances (e.g. trypsin) has not been validated and is not recommended. b Pouch controls passed but Not Detected results were reported for one or more analytes after incubation of the sample with substance. Substance(s) are known to chemically interact with and damage nucleic acids (DNA and/or RNA) to prevent amplification.