Applied BioCode Inc. Robert Di Tullio Regulatory Consultant 10020 Pioneer Blvd. Suite 102 Santa Fe Springs, CA 90670

Re: K180041 Trade/Device Name: BioCode Gastrointestinal Pathogen Panel (GPP) Regulation Number: 21 CFR 866.3990 Regulation Name: Gastrointestinal Microorganism Multiplex Nucleic Acid-based Assay Regulatory Class: Class II Product Code: PCH, OOI Dated: January 08, 2018 Received: January 09, 2018

Dear Mr. Di Tullio:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the

electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801 and Part 809), please contact the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely,

# Steven R. Gitterman -S for

Uwe Scherf, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K180041

Device Name BioCode Gastrointestinal Pathogen Panel (GPP)

Indications for Use (Describe)   
The BioCode Gastrointestinal Pathogen Panel (GPP) is a qualitative, gastrointestinal microorganism   
multiplexed nucleic acid-based assay capable of detecting of nucleic acids from the following organisms in   
unpreserved stool and Cary-Blair media:

Cryptosporidium (C. parvum, C. hominis)

Entamoeba histolytica

Escherichia coli (E. coli) O157

Enterotoxigenic E. coli (ETEC) LT/ST

Enteroaggregative E. coli (EAEC)

Norovirus GI/GII

Rotavirus A

Shiga-like Toxin producing E. coli (STEC) stx1/stx2

Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC • Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus), specific identification of $V .$ parahaemolyticus

• Yersinia enterocolitica

The BioCode Gastrointestinal Pathogen Panel (GPP) is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the BioCode Gastrointestinal Pathogen Panel (GPP). The agent detected may not be the cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.   
This device is not intended to monitor or guide treatment for C. difficile infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Campylobacter spp., E. coli O157, Shigella/EIEC, Yersinia enterocolitica, and Adenovirus 40/41 were established primarily with retrospective clinical specimens.

Performance characteristics for Entamoeba histolytica, and Vibrio spp. (V. parahaemolyticus, V. vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens.

Over-The-Counter Use (21 CFR 801 Subpart C)

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# APPLIED BIOCODE, INC. 10020 PIONEER BLVD. SUITE 102 SANTA FE SPRINGS, CA 90670, USA

# 510(k) SUMMARY

Introduction: According to the requirements of 21 CFR 807.92, the following provides sufficient information to understand the basis for a determination of substantial equivalence.

Submitted by: Applied BioCode, Inc. 10020 Pioneer Blvd. Suite 102 Santa Fe Springs, CA 90670

# Contact:

Robert Di Tullio Regulatory Consultant rditullio@apbiocode.com Telephone: 310 801 1235 Fax: 323 372 3816

Date Submitted: August 22, 2018

Trade Name: BioCode Gastrointestinal Pathogen Panel

Classification Name and Regulation Number: Gastrointestinal microorganism multiplex nucleic acid-based assay (21 CFR 866.3990)

Predicate Device: K121454 – Luminex xTAG® Gastrointestinal Pathogen Panel (GPP)

# Intended Use:

The BioCode Gastrointestinal Pathogen Panel (GPP) is a qualitative, gastrointestinal microorganism multiplexed nucleic acid-based assay capable of detecting of nucleic acids from the following organisms in unpreserved stool and Cary-Blair media:

 Adenovirus 40/41

 Campylobacter (C. jejuni, C. coli)   
 Clostridium difficile (C. difficile) toxin A/B (from fresh specimens only)  Cryptosporidium (C. parvum, C. hominis)   
 Entamoeba histolytica   
 Escherichia coli (E. coli) O157   
 Enterotoxigenic E. coli (ETEC) LT/ST   
 Enteroaggregative E. coli (EAEC)   
 Giardia lamblia (also known as G. intestinalis and G. duodenalis)  Norovirus GI/GII   
 Rotavirus A   
 Salmonella spp.   
 Shiga-like Toxin producing E. coli (STEC) stx1/stx2   
 Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC   
 Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus), specific id  Yersinia enterocolitica

The BioCode Gastrointestinal Pathogen Panel (GPP) is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the BioCode Gastrointestinal Pathogen Panel (GPP). The agent detected may not be the cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.   
This device is not intended to monitor or guide treatment for C. difficile infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Campylobacter spp., E. coli O157, Shigella/EIEC, Yersinia enterocolitica, and Adenovirus 40/41 were established primarily with retrospective clinical specimens.

Performance characteristics for Entamoeba histolytica, and Vibrio spp. (V. parahaemolyticus, V. vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Adenovirus 40/41, Campylobacter, E. coli O157, Shigella/EIEC, Yersinia enterocolitica, and Giardia lamblia were established additionally with retrospective clinical specimens. Performance characteristics for Entamoeba histolytica, Giardia lamblia, Yersinia enterocolitica and Vibrio (V. parahaemolyticus, V. vulnificus, and V. cholerae) were established primarily using contrived clinical specimens.

A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

# Device Description:

The BioCode Gastrointestinal Pathogen Panel (GPP) is a multiplex nucleic acid test designed to be used with the BioCode MDx 3000 system. The BioCode MDx 3000 is an automated system that integrates PCR amplification, target capture, signal generation and optical detection for multiple gastrointestinal pathogens from a single stool specimen, either unpreserved or in Cary Blair. Stool specimens are processed and nucleic acids extracted with easyMAG, an automated system. Once the PCR plate is set up and sealed, all other operations are automated on MDx 3000. The BioCode Gastrointestinal Pathogen Panel simultaneously tests for 17 pathogens (see table below) from unpreserved stool specimens or stool collected in Cary-Blair transport medium. Results from the BioCode Gastrointestinal Pathogen Panel test are available within less than 5 hours.

Bacteria, Viruses, and Parasites Detected by the BioCode Gastrointestinal Pathogen Panel

<table><tr><td>Bacteria</td><td>Parasites</td></tr><tr><td>Campylobacter spp. (C. jejuni, C. coli) Clostridium difficile toxin A/B Enteroaggregative E. coli (EAEC)</td><td>Cryptosporidium spp.(C. parvum/C. hominis) Entamoeba histolytica</td></tr><tr><td rowspan="2">Enterotoxigenic E. coli (ETEC): LT/ST Shiga-toxin producing E. coli (STEC): stx1/stx2 E.coli 0157</td><td>Giardia lamblia/intestinalis</td></tr><tr><td>Viruses Adenovirus 40/41</td></tr><tr><td>Shigella spp. /Enteroinvasive E.coli (EIEC) Salmonella spp.</td><td>Norovirus GI/GII Rotavirus A</td></tr><tr><td>Vibrio parahaemolyticus</td><td></td></tr><tr><td>Vibrio spp (not parahaemolyticus)</td><td>RNA Internal Control</td></tr><tr><td>Yersinia enterocolitica</td><td></td></tr></table>

# Device Comparison:

Comparison of the BioCode Gastrointestinal Pathogen Panel with the Predicate Device   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Proposed Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">Applied BioCode, Inc. BioCode Gastrointestinal Pathogen Panel</td><td colspan="1" rowspan="1">XTAG®GastrointestinalPathogen Panel (GPP)</td></tr><tr><td colspan="1" rowspan="1">Common Name</td><td colspan="1" rowspan="1">Gastrointestinal Microorganism Multiplex Nucleic acid-based assay</td><td colspan="1" rowspan="1">GastrointestinalMicroorganismMultiplex Nucleicacid-based assay</td></tr><tr><td colspan="1" rowspan="1">510(k) No.</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">K121454</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21CFR 866.3990</td><td colspan="1" rowspan="1">21CFR 866.3990</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PCH, 00I</td><td colspan="1" rowspan="1">PCH, 0OI</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">||</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td>Intended Use</td><td>The BioCode Gastrointestinal Pathogen Panel (GPP) is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with the BioCode MDx 3000 Instrument. The BioCode GPP is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites extracted directly from unpreserved stool samples or stool preserved in Cary-Blair transport medium obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following bacteria (including several diarrheagenic E. coli/Shigella pathotypes), parasites, and viruses are identified using the BioCode GPP: Campylobacter (C. jejuni/C. coli), Clostridium difficile (C. difficile) toxin A/B, Salmonella spp, Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio parahaemolyticus, Yersinia enterocolitica, Enteroaggregative Escherichia coli (EAEC), Enterotoxigenic Escherichia coli (ETEC) It/st, E. coli O157 serogroup, Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2, Shigella/ Enteroinvasive Escherichia coli (EIEC), Cryptosporidium spp, Entamoeba histolytica, Giardia lamblia (also known as G. intestinalis and G. duodenalis), Adenovirus F 40/41, Norovirus Gl/GIl, Rotavirus A. The BioCode GPP is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. For In Vitro Diagnostic Use Only. For Prescription Use Only. Positive results do not rule out co-infection with organisms not included in the BioCode GPP. The agent detected may not be the definite cause of the disease. Negative results in the setting of clinical illness compatiblewith gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease. Concomitant culture is necessary for organism recovery and further typing of bacterial agents. This device is not intended to monitor or guide treatment for C. difficile infection. Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Adenovirus 40/41, Campylobacter, E. coli O157, Shigella/EIEC, Yersinia enterocolitica, and Giardia lamblia were established additionally with retrospective clinical specimens. Performance characteristics for Entamoeba histolytica, Giardia lamblia, Yersinia enterocolitica and Vibrio (V. parahaemolyticus, V. vulnificus, and V. cholerae) were established primarily using contrived clinical specimens. A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.</td><td>Equivalent</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Nucleic Acid Purification SystemBioCode MDx 3000</td><td colspan="1" rowspan="1">Nucleic AcidPurification SystemPCR ThermocyclerLuminex 100/200™or           MAGPIXinstruments</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Unpreserved stool and stool in Cary- Blair Media</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Externally Sourced -</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Methodology</td><td colspan="1" rowspan="1">Multiplex RT-PCR and probe hybridization followed by fluorescence detection anddecoding of barcoded magnetic beads (BMB) that are coupled to biotinylatedproducts with streptavidin conjugate</td><td colspan="1" rowspan="1">Multiplex RT-PCR andmultiplex TSPEfollowed byfluorescence-activated sorting oflabeled beads coupledto streptavidin-conjugatedbiotinylated products.</td></tr><tr><td colspan="1" rowspan="1">Calibrators</td><td colspan="1" rowspan="1">Internal Calibration</td><td colspan="1" rowspan="1">External CalibrationKit</td></tr></table>

# Summary of Performance Characteristics of the

# BioCode GPP Clinical Performance

# Testing of Prospectively collected Specimens (Categories I and II)

A clinical investigational study was performed in which a total of 1558 leftover, de-identified samples were prospectively collected from patients who underwent stool sample collection for clinical indications at four (4) investigational sites, with each collecting approximately 400 samples. The sites were located in the United States, with a broad geographic representation. The sites were located in the Northeast (Baltimore, MD), Southeast (Tampa, FL), Midwest (Memphis, TN), and West (Los Angeles, CA). In addition testing was performed on archived known positives and contrived specimens to further augment the sample numbers. Each raw stool specimen was de-identified at the site and the de-identified leftover sample divided into aliquots and either tested freshly on the ABC IUO assay at the site, or frozen and tested at a later date at the site. Each stool specimen in Cary-Blair was similarly de-identified at the site and the de-identified leftover sample aliquoted into 4 aliquots and tested freshly on the ABC IUO assay at the site. All prospectively collected samples had age, sex, and therapy status collected by the site. Enrollment of samples covered multiple calendar seasons beginning January 2015 and ending in August 2017, within which to enroll the specimens displaying due diligence in attempting to cover all panel targets with native specimens. The results of the clinical study follow.

# Demographic data for prospective specimens (fresh and frozen).

<table><tr><td rowspan=1 colspan=1>Prospective Study Specimens</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total Specimens</td><td rowspan=1 colspan=1>1558</td></tr><tr><td rowspan=1 colspan=1>Gender (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FemaleMale</td><td rowspan=1 colspan=1>778/1558 (49.9)780/1558 (51.1)</td></tr><tr><td rowspan=1 colspan=1>Age Category (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt; 5 year6-21 yrs22-59 yrs60+ yrs</td><td rowspan=1 colspan=1>140/1558 (9.0)237/1558 (15.2)718/1558 (46.1)463/1558 (29.7)</td></tr><tr><td rowspan=1 colspan=1>Status (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>InpatientOutpatient</td><td rowspan=1 colspan=1>1212/1558 (77.8)346/1558 (22.2)</td></tr></table>

Clinical Sites collected both Category I and II see the following table for the breakdown.

# Breakdown of prospective specimen collection by site.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Unpreserved(Fresh)</td><td rowspan=1 colspan=1>Unpreserved(Frozen)</td><td rowspan=1 colspan=1>Cary-Blair(Fresh)</td></tr><tr><td rowspan=1 colspan=1>Site 001</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>350</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 002</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 003</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 006</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>361</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>960</td><td rowspan=1 colspan=1>361</td></tr></table>

Table. Comparator (reference) methods for prospective clinical study.

<table><tr><td rowspan=1 colspan=1>Target Pathogen/Toxin</td><td rowspan=1 colspan=1>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni, C. coli)</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (C. difficile) toxin A/B</td><td rowspan=1 colspan=1>FDA cleared NAAT</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium (C. parvum, C. hominis)</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (E. coli) 0157</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) LT/ST</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia /intestinalis</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing assays</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Shiga-like Toxin producing E. coli (STEC) stx1/stx2</td><td rowspan=1 colspan=1>Enrichment culture/FDA cleared antigen test</td></tr><tr><td rowspan=1 colspan=1>Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus)</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Culture</td></tr></table>

Clinical sensitivity/positive agreement was calculated as TP $\gamma / ( \mathsf { T P } + \mathsf { F N } )$ . TP $=$ true positive or positive by both the reference and BioCode GPP; $\mathsf { F N } =$ false negative or negative by BioCode GPP only. Clinical specificity/negative agreement was calculated as TN/ $\left( \mathsf { T N } + \mathsf { F P } \right)$ . ${ \mathsf { T N } } =$ true negative or negative by both the reference and BioCode GPP; $\mathsf { F P } =$ false positive or positive by BioCode GPP only. The exact binomial twosided $9 5 \%$ confidence interval was calculated. The results stratified by sample type and storage method are presented in the tables below.

Table. Summary of Clinical Study Results (Prospective specimens) for Unpreserved Stool (Fresh).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp.a</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5, 100.0)</td><td rowspan=1 colspan=1>234/236 (99.2)</td><td rowspan=1 colspan=1>(96.97, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficileb</td><td rowspan=1 colspan=1>26/27 (96.3)</td><td rowspan=1 colspan=1>(81.0, 99.9)</td><td rowspan=1 colspan=1>208/210 (99.1)</td><td rowspan=1 colspan=1>(96.6, 99.9)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>237/237 (100.0)</td><td rowspan=1 colspan=1>(98.46, 100.0)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5, 100.0)</td><td rowspan=1 colspan=1>234/234 (100.0)</td><td rowspan=1 colspan=1>(98.44, 100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2, 100.0)</td><td rowspan=1 colspan=1>229/232 (98.7)</td><td rowspan=1 colspan=1>(96.27, 99.7)</td></tr><tr><td rowspan=1 colspan=1>STECd</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>235/237 (99.2)</td><td rowspan=1 colspan=1>(96.99, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Salmonella sppe</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2, 100.0)</td><td rowspan=1 colspan=1>232/234 (99.2)</td><td rowspan=1 colspan=1>(96.95, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5, 100.0)</td><td rowspan=1 colspan=1>233/236 (98.7)</td><td rowspan=1 colspan=1>(96.33, 99.7)</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>236/237 (99.6)</td><td rowspan=1 colspan=1>(97.67, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>237/237 (100.0)</td><td rowspan=1 colspan=1>(98.46, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocoliticah</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>236/237 (99.6)</td><td rowspan=1 colspan=1>(97.67, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5, 100.0)</td><td rowspan=1 colspan=1>234/234 (100.0)</td><td rowspan=1 colspan=1>(98.44, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>235/235 (100.0)</td><td rowspan=1 colspan=1>(98.44, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>234/235 (99.6)</td><td rowspan=1 colspan=1>(97.65, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>233/235 (99.2)</td><td rowspan=1 colspan=1>(96.96, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GIl</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>235/235 (100.0)</td><td rowspan=1 colspan=1>(98.44, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>234/235 (99.6)</td><td rowspan=1 colspan=1>(97.65, 100.0)</td></tr></table>

a - Campylobacter spp: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and 1 of 2 confirmed as positive.   
b - Clostridium difficile: The 1 false negative compared to the FDA Cleared NAAT reference test produced high Ct (Ct 35.0). c - ETEC: The 3 false positives compared to bidirectional sequencing were not confirmed as positives by an additional round of sequencing.   
d - STEC: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and both confirmed as positive.   
e– Salmonella spp: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and both confirmed as positives.   
f - Shigella/EIEC: The 3 false positives compared to the culture reference method were tested by bidirectional sequencing, and all 3 confirmed as positives.   
g - Vibrio parahaemolyticus: The 1 false positive sample compared to the culture reference method was tested by bidirectional sequencing and confirmed as positive.   
h - Yersinia enterocolitica: The 1 false positive sample compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive.   
ij – Giardia lamblia: The 1 false positive to bidirectional sequencing was not confirmed as positive by 2 additional rounds of sequencing. j - Adenovirus 40/41: The 2 false positives to bidirectional sequencing were not confirmed as positives by an additional round of sequencing.

Table. Summary of Clinical Study Results (Prospective specimens) for Unpreserved Stool (Frozen).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp.a</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2, 100.0)</td><td rowspan=1 colspan=1>936/952 (98.3)</td><td rowspan=1 colspan=1>(97.3, 99.0)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficileb</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157c</td><td rowspan=1 colspan=1>1/2 (50.0)</td><td rowspan=1 colspan=1>(1.3, 98.7)</td><td rowspan=1 colspan=1>950/954 (99.6)</td><td rowspan=1 colspan=1>(98.9, 99.9)</td></tr><tr><td rowspan=1 colspan=1>EAECd</td><td rowspan=1 colspan=1>25/29 (86.2)</td><td rowspan=1 colspan=1>(68.3, 96.1)</td><td rowspan=1 colspan=1>916/919 (99.7)</td><td rowspan=1 colspan=1>(99.1, 99.9)</td></tr><tr><td rowspan=1 colspan=1>ETECe</td><td rowspan=1 colspan=1>7/10 (70.0)</td><td rowspan=1 colspan=1>(34.8, 93.3)</td><td rowspan=1 colspan=1>934/939 (99.5)</td><td rowspan=1 colspan=1>(98.8, 99.8)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2, 100.0)</td><td rowspan=1 colspan=1>918/919 (99.9)</td><td rowspan=1 colspan=1>(99.4, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp</td><td rowspan=1 colspan=1>18/22 (81.8)</td><td rowspan=1 colspan=1>(59.7, 94.8)</td><td rowspan=1 colspan=1>926/934 (99.1)</td><td rowspan=1 colspan=1>(98.3, 99.6)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIECh</td><td rowspan=1 colspan=1>4/5 (80.00)</td><td rowspan=1 colspan=1>(28.4, 99.5)</td><td rowspan=1 colspan=1>940/951 (98.8)</td><td rowspan=1 colspan=1>(97.9, 99.4)</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>955/957 (99.8)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>956/956 (100.0)</td><td rowspan=1 colspan=1>(99.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>951/956 (99.5)</td><td rowspan=1 colspan=1>(98.8, 99.8)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>7/7 (100.0)</td><td rowspan=1 colspan=1>(59.0, 100.0)</td><td rowspan=1 colspan=1>941/941 (100.0)</td><td rowspan=1 colspan=1>(99.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>948/948 (100.0)</td><td rowspan=1 colspan=1>(99.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lambliak</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.8, 100.0)</td><td rowspan=1 colspan=1>940/946 (99.4)</td><td rowspan=1 colspan=1>(98.6, 99.8)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41&#x27;</td><td rowspan=1 colspan=1>7/10 (70.0)</td><td rowspan=1 colspan=1>(34.8, 93.3)</td><td rowspan=1 colspan=1>935/938 (99.7)</td><td rowspan=1 colspan=1>(99.1, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>39/39 (100.0)</td><td rowspan=1 colspan=1>(91.0, 100.0)</td><td rowspan=1 colspan=1>913/917 (99.6)</td><td rowspan=1 colspan=1>(98.9, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>19/20 (95.0)</td><td rowspan=1 colspan=1>(75.1, 99.9)</td><td rowspan=1 colspan=1>928/936 (99.2)</td><td rowspan=1 colspan=1>(98.3, 99.6)</td></tr></table>

a - Campylobacter spp: The 16 false positives compared to the culture reference method were tested by bidirectional sequencing, and 8 of 16 confirmed as positives. b - Clostridium difficile: C. difficile testing must be performed with fresh specimens only, not with previously frozen specimens. c - E. coli O157: The one false negative compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive. The 4 false positive samples compared to the culture reference method were tested by bidirectional sequencing, and 3 of 4 confirmed as positives.

d - EAEC: The 4 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 3 of the 4 confirmed as positives. 2 of the 3 false positives could not be repeated due to low sample volume. The remaining 1 was not detected by addition rounds of sequencing.

e - ETEC: The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. 1 of the 5 false positives could not be repeated due to low sample volume. The remaining 4 false positives were not confirmed as positives by an additional round of sequencing.

f - STEC: The 1 false positive compared to the culture reference method was tested by bidirectional sequencing and confirmed as positive.

g – Salmonella spp: The 4 false negatives compared to the culture reference method were tested by bidirectional sequencing, and 1 of 4 could not be confirmed as positives. The 8 false positive samples compared to the culture reference method were tested by bidirectional sequencing and 6 of 8 confirmed as positives.

h - Shigella/EIEC: The 1 false negative compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive. The 11 false positive samples compared to the culture reference method were tested by bidirectional sequencing, and 10 of 11 confirmed as positives.

i - Vibrio parahaemolyticus: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing

and, 1 of 2 confirmed as positive.   
j - Yersinia enterocolitica: The 5 false positives compared to the culture reference method were tested by bidirectional sequencing and, 3 of 5 confirmed as positive.   
k – Giardia lamblia: The 4 false positives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing: none were confirmed as positives.   
l - Adenovirus 40/41: The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. The 3 false positives were not confirmed as positives by an additional round of sequencing.

Table. Summary of Clinical Study Results (Prospective specimens) for Native Cary-Blair Samples.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp.a</td><td rowspan=1 colspan=1>2/3 (66.7)</td><td rowspan=1 colspan=1>(9.4, 99.2)</td><td rowspan=1 colspan=1>347/358 (96.9)</td><td rowspan=1 colspan=1>(94.6, 98.5)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficileb</td><td rowspan=1 colspan=1>37/38 (97.4)</td><td rowspan=1 colspan=1>(86.2, 99.9)</td><td rowspan=1 colspan=1>318/322 (98.8)</td><td rowspan=1 colspan=1>(96.9, 99.7)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157c</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>359/361 (99.5)</td><td rowspan=1 colspan=1>(98.0, 99.9)</td></tr><tr><td rowspan=1 colspan=1>EAECd</td><td rowspan=1 colspan=1>17/18 (94.4)</td><td rowspan=1 colspan=1>(72.71, 99.9)</td><td rowspan=1 colspan=1>336/341 (98.5)</td><td rowspan=1 colspan=1>(96.6, 99.5)</td></tr><tr><td rowspan=1 colspan=1>ETECe</td><td rowspan=1 colspan=1>13/14 (92.9)</td><td rowspan=1 colspan=1>(66.13, 99.8)</td><td rowspan=1 colspan=1>343/345 (99.4)</td><td rowspan=1 colspan=1>(97.9, 99.9)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>359/361 (99.5)</td><td rowspan=1 colspan=1>(98.0, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp</td><td rowspan=1 colspan=1>4/5 (80.0)</td><td rowspan=1 colspan=1>(28.36, 99.5)</td><td rowspan=1 colspan=1>354/356 (99.4)</td><td rowspan=1 colspan=1>(98.0, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIECh</td><td rowspan=1 colspan=1>1/2 (50.0)</td><td rowspan=1 colspan=1>(1.26, 98.7)</td><td rowspan=1 colspan=1>356/359 (99.2)</td><td rowspan=1 colspan=1>(97.6, 99.8)</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>361/361 (100.0)</td><td rowspan=1 colspan=1>(99.0, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>361/361 (100.0)</td><td rowspan=1 colspan=1>(99.0, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>357/361 (98.9)</td><td rowspan=1 colspan=1>(97.2, 99.7)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium sppi</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.24, 100.0)</td><td rowspan=1 colspan=1>354/356 (99.4)</td><td rowspan=1 colspan=1>(98.0, 99.9)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>359/359 (100.0)</td><td rowspan=1 colspan=1>(99.0, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lambliak</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>357/358 (99.7)</td><td rowspan=1 colspan=1>(98.5, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>359/359 (100.0)</td><td rowspan=1 colspan=1>(99.0, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII&#x27;</td><td rowspan=1 colspan=1>6/7 (85.7)</td><td rowspan=1 colspan=1>(42.13, 99.6)</td><td rowspan=1 colspan=1>354/354 (100.0)</td><td rowspan=1 colspan=1>(99.0, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>360/360 (100.0)</td><td rowspan=1 colspan=1>(98.98, 100.0)</td></tr></table>

a – Campylobacter spp. The 1 false negative compared to reference culture method was tested by bidirectional sequencing and confirmed positive. The 11 false positives compared to reference culture method were tested by bidirectional sequencing, and 11 of 11 confirmed as positives.

b – Clostridium difficile. The 1 false negative compared to the FDA cleared NAAT reference method produced high Ct (35).

c - E. coli O157. The 2 false positives compared to reference culture method were tested by bidirectional sequencing, and 2 of 2 confirmed as positives.

d – EAEC. The 1 false negative compared to bidirectional sequencing was tested by 2 additional rounds of sequencing and confirmed as positive. The 4 of 5 false positives were not detected by an addition round of sequencing.

– ETEC. The 1 false negative compared to bidirectional sequencing was tested by 2 additional rounds of sequencing, and was not confirmed as positive. 1 of 2 false positives was confirmed as positive by 2 additional rounds of sequencing.

f – STEC. The 2 false positives compared to reference culture method were tested by bidirectional sequencing, and 2 of 2 confirmed as positives.

g – Salmonella spp. The 1 false negative compared to the reference culture method was tested by bidirectional sequencing and confirmed as positive. The 2 false positives compared to reference culture method were tested by bidirectional sequencing and 1 of 2 confirmed as positive.   
h – Shigella/EIEC. The 1 false negative compared to the reference culture method was tested by bidirectional sequencing and could not be confirmed as positive. The 3 false positives compared to reference culture method were tested by bidirectional sequencing, and all 3 confirmed as positives.   
i - Yersinia enterocolitica. The 4 false positives compared to the reference culture method were tested by bidirectional sequencing, and none were confirmed as positive.   
j – Cryptosporidium spp. The 2 false positives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing and both confirmed as positives.   
k – Giardia lamblia. The 1 false positive compared to bidirectional sequencing was not confirmed as positive by 2 additional rounds of sequencing.   
l - Norovirus GI/GII. The 1 false negative compared to bidirectional sequencing produced a high Ct (37) which indicates that this sample is low positive.

During the prospective clinical study $2 . 6 \%$ (41/1558) of samples were invalid for lack of RNA-IC signal on initial testing. The invalid rate after reflex testing for the prospective study was approximately $0 . 2 \%$ (3/1558).

# Analysis of mixed infections in the prospective study

The BioCode GPP detected a total of 49 samples with mixed infections in the prospective clinical study. This represents $3 . 1 \%$ of the total number of specimens (49/1558). 40 were double infections, 8 were triple infections, and 1 was quadruple infection. The most common pathogens in co-infections were with EAEC (22/49, $4 4 . 9 \% )$ and ETEC (18/49, $3 6 . 7 \% )$ . The most common co-infection combinations detected by the BioCode GPP in the prospective clinical study are summarized in the table below.

Most prevalent multiple detection combinations (5 or more instances) from clinical evaluation.   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>EAEC + ETEC</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile + Salmonella spp</td><td rowspan=1 colspan=1>5</td></tr></table>

Clinical co-infection combinations detected by BioCode GPP (unpreserved stool).   

<table><tr><td rowspan=1 colspan=4>Distinct Co-Infection Combinations Detected by BioCode GPP</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Analyte_1</td><td rowspan=1 colspan=1>Analyte_2</td><td rowspan=1 colspan=1>Analyte_3</td><td rowspan=1 colspan=1>Analyte_4</td><td rowspan=1 colspan=1>TotalCo-infections</td><td rowspan=1 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=1 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus 40/41 (x1)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Campylobacter spp (x1)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1>C. difficile</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ETEC (x1)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Campylobacter spp (x1)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Giardia lamblia (x1)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>E.coli 0157 (x1)</td></tr><tr><td rowspan=1 colspan=1>E.coli O157</td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>E.coli 0157 (x1)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella/EIEC (x1)</td></tr></table>

<table><tr><td></td><td rowspan=1 colspan=4>Distinct Co-Infection Combinations Detected by BioCode GPP</td><td rowspan=1 colspan=3></td></tr><tr><td></td><td rowspan=1 colspan=1>Analyte_1</td><td rowspan=1 colspan=1>Analyte_2</td><td rowspan=1 colspan=1>Analyte_3</td><td rowspan=1 colspan=1>Analyte_4</td><td rowspan=1 colspan=1>TotalCo-infections</td><td rowspan=1 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=1 colspan=1>Discrepant Analyte(s)</td></tr><tr><td></td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella/EIEC (x1)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ETEC (x1)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>EAEC (x1);ETEC (x1)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Rotavirus A (x1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella/EIEC (x1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus GI/GII (x1)</td></tr><tr><td rowspan=1 colspan=1>′</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Rotavirus A (x1);STEC (x1);</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>V. parahaemolyticus</td><td rowspan=1 colspan=1>Y. enterocolitica</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Total Co-infections</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Double Infections</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Triple Infections</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td></tr></table>

Clinical co-infection combinations detected by BioCode GPP (Cary-Blair).   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Infection Combinations Detected by BioCode GPP</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Analyte_1</td><td colspan="1" rowspan="1">Analyte_2</td><td colspan="1" rowspan="1">Analyte_3</td><td colspan="1" rowspan="1">Analyte_4</td><td colspan="1" rowspan="1">TotalCo-infections</td><td colspan="1" rowspan="1">Number ofDiscrepantCo-infections</td><td colspan="1" rowspan="1">Discrepant Analyte(s)</td></tr><tr><td colspan="1" rowspan="1">C. difficile</td><td colspan="1" rowspan="1">Salmonella spp</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">E.coli 0157</td><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E.coli O157 (x1);STEC (x1)</td></tr><tr><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EAEC (x1)</td></tr><tr><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">ETEC</td><td colspan="1" rowspan="1">Y. enterocolitica</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ETEC (x1);Y. enterocolitica (x2)</td></tr><tr><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EAEC (x1)</td></tr><tr><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">STEC</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">STEC (x1)</td></tr><tr><td colspan="4" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Double Infections</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Triple Infections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Quadruple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr></table>

Clinical co-infection combinations detected by reference methods (unpreserved stool).   

<table><tr><td rowspan=1 colspan=3>Distinct Co-Infection CombinationsDetected by Reference Methods</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Analyte_1</td><td rowspan=1 colspan=1>Analyte_2</td><td rowspan=1 colspan=1>Analyte_3</td><td rowspan=1 colspan=1>TotalCo-infections</td><td rowspan=1 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=1 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus40/41</td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus 40/41 (x1)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>EAEC (x1)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>EAEC (x1);ETEC (x2)</td></tr><tr><td rowspan=1 colspan=3>Total Co-infections</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Double Infections</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Triple Infections</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr></table>

Clinical co-infection combinations detected by reference methods (Cary-Blair).   

<table><tr><td rowspan=1 colspan=3>Distinct Co-Infection CombinationsDetected by Reference Methods</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Analyte_1</td><td rowspan=1 colspan=1>Analyte_2</td><td rowspan=1 colspan=1>Analyte_3</td><td rowspan=1 colspan=1>TotalCo-infections</td><td rowspan=1 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=1 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella/EIEC (x1)</td></tr><tr><td rowspan=1 colspan=3>Total Co-infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Double Infections</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Triple Infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr></table>

# Testing of inoculated Cary-Blair specimens from previously frozen prospective specimens

To supplement the number of prospective Cary-Blair specimens. 400 unpreserved stool samples from sites 1 and 2 were thawed and inoculated into Cary-Blair. 3 were removed from the study for improper storage prior to testing. 2 were invalid for RNA IC failure in the unpreserved stool. The following table summarizes the comparison of the results from the initial unpreserved testing and subsequent inoculated results.

Summary of Inoculated Cary-Blair or unpreserved samples Vs. reference method. Agreements were calculated compared to reference method testing of unpreserved stool. Reference testing was not repeated after samples were inoculated to Cary-Blair.   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>(n)</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=2 colspan=1>Campylobacter spp.a</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>385/392 (98.21)</td><td rowspan=1 colspan=1>(96.36, 99.28)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>388/394 (98.48)</td><td rowspan=1 colspan=1>(96.72, 99.44)</td></tr><tr><td rowspan=2 colspan=1>E. coli 0157b</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>1/2 (50.00)</td><td rowspan=1 colspan=1>(1.26, 98.74)</td><td rowspan=1 colspan=1>389/393 (98.98)</td><td rowspan=1 colspan=1>(97.41, 99.72)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1/2 (50.00)</td><td rowspan=1 colspan=1>(1.26, 98.74)</td><td rowspan=1 colspan=1>391/395 (98.99)</td><td rowspan=1 colspan=1>(97.43, 99.72)</td></tr><tr><td rowspan=2 colspan=1>Enteroaggregative E. coli(AEC)</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>12/14 (85.71)</td><td rowspan=1 colspan=1>(57.19, 98.22)</td><td rowspan=1 colspan=1>378/380 (99.47)</td><td rowspan=1 colspan=1>(98.11, 99.94)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>12/14 (85.71)</td><td rowspan=1 colspan=1>(57.19, 98.22)</td><td rowspan=1 colspan=1>382/382 (100.0)</td><td rowspan=1 colspan=1>(99.04, 100.0)</td></tr><tr><td rowspan=2 colspan=1>EnterotoxigenicE. coli (ETEC)d</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>4/6 (66.67)</td><td rowspan=1 colspan=1>(22.28, 95.67)</td><td rowspan=1 colspan=1>387/388 (99.74)</td><td rowspan=1 colspan=1>(98.57, 99.99)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>4/6 (66.67)</td><td rowspan=1 colspan=1>(22.28, 95.67)</td><td rowspan=1 colspan=1>386/390 (98.97)</td><td rowspan=1 colspan=1>(97.39, 99.72)</td></tr><tr><td rowspan=2 colspan=1>Shiga toxin-producing E.coli (STEC)</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>359/359 (100.0)</td><td rowspan=1 colspan=1>(98.98, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>361/361 (100.0)</td><td rowspan=1 colspan=1>(98.98, 100.0)</td></tr><tr><td rowspan=2 colspan=1>Salmonella spp.e</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>5/6 (83.33)</td><td rowspan=1 colspan=1>(35.88, 99.58)</td><td rowspan=1 colspan=1>385/389 (98.97)</td><td rowspan=1 colspan=1>(97.39, 99.72)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>6/6 (100.0)</td><td rowspan=1 colspan=1>(54.07, 100.0)</td><td rowspan=1 colspan=1>389/391 (99.49)</td><td rowspan=1 colspan=1>(98.16, 99.94)</td></tr><tr><td rowspan=2 colspan=1>Shigella/ EIECf</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>389/394 (98.73)</td><td rowspan=1 colspan=1>(97.06, 99.59)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50, 100.0)</td><td rowspan=1 colspan=1>391/396 (98.74)</td><td rowspan=1 colspan=1>(97.08, 99.59)</td></tr><tr><td rowspan=2 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>395/395 (100.0)</td><td rowspan=1 colspan=1>(99.07, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>397/397 (100.0)</td><td rowspan=1 colspan=1>(99.08, 100.0)</td></tr><tr><td rowspan=2 colspan=1>Vibrio spp. (notparahaemolyticus)</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>395/395 (100.0)</td><td rowspan=1 colspan=1>(99.07, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>397/397 (100.0)</td><td rowspan=1 colspan=1>(99.08, 100.0)</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica®</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>394/395 (99.75)</td><td rowspan=1 colspan=1>(98.60, 99.99)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>396/397 (99.75)</td><td rowspan=1 colspan=1>(98.60, 99.99)</td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>392/392 (100.0)</td><td rowspan=1 colspan=1>(99.06, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.81, 100.0)</td><td rowspan=1 colspan=1>394/394 (100.0)</td><td rowspan=1 colspan=1>(99.07, 100.0)</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>394/394 (100.0)</td><td rowspan=1 colspan=1>(99.07, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>396/396 (100.0)</td><td rowspan=1 colspan=1>(99.07, 100.0)</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>(n)</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=2 colspan=1>Giardia lambliah</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>391/394 (99.24)</td><td rowspan=1 colspan=1>(97.79, 99.84)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>394/396 (99.49)</td><td rowspan=1 colspan=1>(98.19, 99.94)</td></tr><tr><td rowspan=2 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>394</td><td rowspan=1 colspan=1>3/6 (50.00)</td><td rowspan=1 colspan=1>(11.81, 88.19)</td><td rowspan=1 colspan=1>388/388 (100.0)</td><td rowspan=1 colspan=1>(99.05, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/6 (33.33)</td><td rowspan=1 colspan=1>(4.33, 77.72)</td><td rowspan=1 colspan=1>385/390 (98.72)</td><td rowspan=1 colspan=1>(97.03, 99.58)</td></tr><tr><td rowspan=2 colspan=1>Norovirus(GI/GII)</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>28/28 (100.0)</td><td rowspan=1 colspan=1>(87.66, 100.0)</td><td rowspan=1 colspan=1>364/367 (99.18)</td><td rowspan=1 colspan=1>(97.63, 99.83)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>28/28 (100.0)</td><td rowspan=1 colspan=1>(87.66, 100.0)</td><td rowspan=1 colspan=1>364/369 (98.64)</td><td rowspan=1 colspan=1>(96.87, 99.56)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>11/12 (91.67)</td><td rowspan=1 colspan=1>(61.52, 99.79)</td><td rowspan=1 colspan=1>380/383 (99.22)</td><td rowspan=1 colspan=1>(97.73, 99.84)</td></tr><tr><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>11/12 (91.67)</td><td rowspan=1 colspan=1>(61.52, 99.79)</td><td rowspan=1 colspan=1>380/385 (98.70)</td><td rowspan=1 colspan=1>(97.00, 99.58)</td></tr></table>

a – Campylobacter spp. Unpreserved: The 6 false positives compared to reference culture method were tested by bidirectional sequencing and 4 of 6 confirmed as positives. Cary-Blair: The 7 false positives compared to the reference culture method were tested by bidirectional sequencing and 4 of 7 confirmed as positives.

b - E. coli O157. Unpreserved: The one false negative compared to the reference culture method was tested by bidirectional sequencing and could not be confirmed as positive. The 4 false positives compared to the reference culture method were tested by bidirectional sequencing and 3 of 4 confirmed as positives. Cary-Blair: The one false negative compared to the reference culture method was tested by bidirectional sequencing and could not be confirmed as positive. The 4 false positives compared to reference culture method were tested by bidirectional sequencing and 3 of 4 confirmed as positives.

c - EAEC. Unpreserved: The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 1 of the 2 confirmed as positive. Cary-Blair: The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 1 of the 2 confirmed as positive. The 2 false positives compared to bidirectional sequencing were not confirmed as positive by 2 additional rounds of sequencing.

d - ETEC. Unpreserved: The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. None of 4 false positives were confirmed as positive by 2 additional rounds of sequencing. Cary-Blair: The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. The 1 false positive compared to bidirectional sequencing was not available for confirmation testing.

e – Salmonella spp. Unpreserved: The 2 false positives compared to the reference culture method were tested by bidirectional sequencing and 1 of 2 confirmed as positives. Cary-Blair: The one false negative compared to the reference culture method was tested by bidirectional sequencing and confirmed as positive. The 4 false positives compared to the reference culture method were tested by bidirectional sequencing and 2 of 4 confirmed as positives.

f - Shigella/EIEC. Unpreserved: The 5 false positives compared to the reference culture method were tested by bidirectional sequencing and 4 of 5 confirmed as positives. Cary-Blair: The 5 false positives compared to the reference culture method were tested by bidirectional sequencing and 4 of 5 confirmed as positives.

g - Yersinia enterocolitica. Unpreserved: The 1 false positive compared to the reference culture method was tested by bidirectional sequencing and confirmed as positive. Cary-Blair: The 1 false positive compared to the reference culture method were tested by bidirectional sequencing and confirmed as positive.

h - Giardia lamblia. Unpreserved: The 2 false positives compared to bidirectional sequencing were not confirmed as positive by 2 additional rounds of sequencing. Cary-Blair: The 3 false positives compared to bidirectional sequencing were not confirmed as positive by 2 additional rounds of sequencing.

i – Adenovirus 40/41. Unpreserved: The 4 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 1 of 4 was confirmed as positive. The 5 false positives were not confirmed as positives by an additional round of sequencing. Cary-Blair: The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none confirmed as positive.

# Testing of Pre-selected Archived Specimens (Category III)

Several analytes were not encountered or had low prevalence in the clinical study. To supplement the results of the prospective clinical study, 260 preselected archived specimens were assayed. These were archived clinical specimens that were previously tested positive by different methods. Prior to testing with the Applied BioCode Gastrointestinal Pathogen Panel, the presence of the expected analyte was verified in each specimen using analyte-specific PCR followed by bi-directional sequencing performed at Applied BioCode, Inc. The specimens were randomized with negative specimens, such that the users performing the BioCode GPP were blinded to the expected test result. A summary of the demographic information of the tested samples and the results of the BioCode GPP testing are presented in Tables below.

Demographic summary for archived specimens   

<table><tr><td>Prospective Study Specimens</td><td></td></tr><tr><td>Total Specimens</td><td>260</td></tr><tr><td>Gender (n/N(%))</td><td></td></tr><tr><td>Female</td><td>123/260 (47.3)</td></tr><tr><td>Male Age Category (n/N(%))</td><td>137/260 (52.7)</td></tr><tr><td>&lt; 5 year</td><td>54/260 (20.8)</td></tr><tr><td>6-21 yrs</td><td>46/260 (17.7)</td></tr><tr><td>22-59 yrs</td><td>123/260 (47.3)</td></tr><tr><td></td><td></td></tr><tr><td>60+ yrs</td><td>37/260 (14.2)</td></tr></table>

Summary of Clinical specimen Results (Archived specimens)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Campylobacter sppa</td><td rowspan=1 colspan=1>38/40 (95.0)</td><td rowspan=1 colspan=1>(83.1, 99.4)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>19/19 (100.0)</td><td rowspan=1 colspan=1>(82.4, 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.55, 100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>20/20 (100.0)</td><td rowspan=1 colspan=1>(83.2, 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>STECb</td><td rowspan=1 colspan=1>30/33 (90.9)</td><td rowspan=1 colspan=1>(75.7, 98.1)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.c</td><td rowspan=1 colspan=1>29/30 (96.7)</td><td rowspan=1 colspan=1>(82.8, 99.9)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Shigella/ EIECd</td><td rowspan=1 colspan=1>43/45 (95.6)</td><td rowspan=1 colspan=1>(84.9, 99.5)</td><td rowspan=1 colspan=1>151/152 (99.3)</td><td rowspan=1 colspan=1>(96.4, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.24, 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp.e</td><td rowspan=1 colspan=1>16/19 (84.2)</td><td rowspan=1 colspan=1>(60.4, 96.6)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lambliaf</td><td rowspan=1 colspan=1>25/26 (96.2)</td><td rowspan=1 colspan=1>(83.2, 99.9)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>26/26 (100.0)</td><td rowspan=1 colspan=1>(86.8, 100.0)</td><td rowspan=1 colspan=1>151/152 (99.3)</td><td rowspan=1 colspan=1>(96.4, 100.0)</td></tr></table>

Distribution of Mixed Infections in Archived positives.

Prevalence of Analytes in Mixed Infections in Archived positives.   

<table><tr><td rowspan=1 colspan=1>Analytes DetectedSimultaneously</td><td rowspan=1 colspan=1>n/N (%)</td></tr><tr><td rowspan=1 colspan=1>Total Mixed Infections</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>71/84 (84.5)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11/84 (13.1)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2/84 (2.4)</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Prevalence in Mixed InfectionN = 84</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp</td><td rowspan=1 colspan=1>7/84</td><td rowspan=1 colspan=1>8.3</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>28/84</td><td rowspan=1 colspan=1>33.3</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>30/84</td><td rowspan=1 colspan=1>35.7</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>18/84</td><td rowspan=1 colspan=1>21.4</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>17/84</td><td rowspan=1 colspan=1>20.2</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>33/84</td><td rowspan=1 colspan=1>39.3</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp</td><td rowspan=1 colspan=1>12/84</td><td rowspan=1 colspan=1>14.3</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>4/84</td><td rowspan=1 colspan=1>4.8</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>6/84</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>6/84</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>11/84</td><td rowspan=1 colspan=1>13.1</td></tr><tr><td rowspan=1 colspan=1>Norovirus Gl/GllI</td><td rowspan=1 colspan=1>4/84</td><td rowspan=1 colspan=1>4.8</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>7/84</td><td rowspan=1 colspan=1>8.3</td></tr></table>

# Most Prevalence Multiple Detection Combinations (5 or more instances) in Archived positives.

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157+STEC</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>EAEC+ETEC</td><td rowspan=1 colspan=1>10</td></tr></table>

# Testing of Contrived Specimens (Category IV)

For some analytes both prospective and archived testing were insufficient to demonstrate system performance. To supplement the prospective and archived data, contrived specimens were assayed. The contrived specimens were positive for Giardia, E. histolytica, Yersinia enterocolitica, Vibrio parahaemolyticus, and Vibrio spp. These contrived clinical specimens were prepared using specimens that had previously tested negative for all BioCode GPP analytes. Specimens were spiked at levels of up to 3X LOD $7 \%$ of contrived specimens) or greater using multiple strains for each organism. Positive samples of each were prepared, and randomized by mixing with negative samples before testing. A total of 612 samples, 485 positives, were tested. The results of the BioCode GPP testing are presented in the Table below.

Summary of contrived specimen results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NP</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AgreementRaten/N (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>AgreementRaten/N (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>88/96 (91.7)</td><td rowspan=1 colspan=1>(84.2, 96.3)</td><td rowspan=1 colspan=1>516/516 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp. (not parahaemolyticus)</td><td rowspan=1 colspan=1>82/94 (87.2)</td><td rowspan=1 colspan=1>(78.8, 93.2)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>40/47 (85.1)</td><td rowspan=1 colspan=1>(72.3, 92.6)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>42/47 (89.4)</td><td rowspan=1 colspan=1>(77.4, 95.4)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>95/98 (96.9)</td><td rowspan=1 colspan=1>(91.3, 99.4)</td><td rowspan=1 colspan=1>514/514 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>96/99 (97.1)</td><td rowspan=1 colspan=1>(91.4, 99.4)</td><td rowspan=1 colspan=1>507/513 (98.8)</td><td rowspan=1 colspan=1>(97.5, 99.6)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>94/98 (95.9)</td><td rowspan=1 colspan=1>(89.9, 98.9)</td><td rowspan=1 colspan=1>513/514 (99.8)</td><td rowspan=1 colspan=1>(98.9, 100.0)</td></tr></table>

a - All false negative specimens were tested by PCR/bidirectional sequencing and none could be confirmed as positives (not positive via sequencing). It is likely that these were either prepared incorrectly or degraded during shipping and handling.

Detailed breakdown of contrived testing samples   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>Straincharacterization</td><td rowspan=1 colspan=1>LoD(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>ConcentrationPrepared(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>Multipleof LOD</td><td rowspan=1 colspan=1>Replicatestested</td><td rowspan=1 colspan=1>Falsenegative</td><td rowspan=1 colspan=1>Falsenegativedistribution</td></tr><tr><td rowspan=4 colspan=1>Vibrioparahaemolyticus</td><td rowspan=4 colspan=1>ATCC17802</td><td rowspan=4 colspan=1>strain EB101</td><td rowspan=4 colspan=1>5.00E+01</td><td rowspan=1 colspan=1>1.50E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Vibrioparahaemolyticus</td><td rowspan=5 colspan=1>BEI NR-21991</td><td rowspan=5 colspan=1>01:K56 strain10295</td><td rowspan=5 colspan=1>7.50E+02</td><td rowspan=1 colspan=1>1.50E+02</td><td rowspan=1 colspan=1>0.2x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>3</td><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>3.15</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5.00E+03</td><td rowspan=1 colspan=1>6.3x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>Straincharacterization</td><td rowspan=1 colspan=1>LoD(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>ConcentrationPrepared(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>Multipleof LOD</td><td rowspan=1 colspan=1>Replicatestested</td><td rowspan=1 colspan=2>Falsenegative</td><td rowspan=1 colspan=1>Falsenegativedistribution</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.90E+03</td><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Vibrioparahaemolyticus</td><td rowspan=3 colspan=1>BEI NR-21990</td><td rowspan=3 colspan=1>04:K12 strain48057</td><td rowspan=3 colspan=1>5.00E+01</td><td rowspan=1 colspan=1>1.50E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>3</td><td rowspan=2 colspan=2>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.30E+04</td><td rowspan=1 colspan=1>460x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=5 colspan=1>Vibrioparahaemolyticus</td><td rowspan=5 colspan=1>BEI NR-22002</td><td rowspan=5 colspan=1>03:K6 strainTX2103</td><td rowspan=5 colspan=1>1.71E+02</td><td rowspan=1 colspan=1>1.50E+02</td><td rowspan=1 colspan=1>0.88x</td><td rowspan=1 colspan=1>9</td><td rowspan=2 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>5.13E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>14.6x</td><td rowspan=1 colspan=1>4</td><td rowspan=3 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5.00E+03</td><td rowspan=1 colspan=1>29x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.71E+05</td><td rowspan=1 colspan=1>1000x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>Vibrioparahaemolyticus</td><td rowspan=4 colspan=1>Zepto0801903</td><td rowspan=4 colspan=1>strain Z134</td><td rowspan=4 colspan=1>5.00E+01</td><td rowspan=1 colspan=1>1.50E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>5.00E+03</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.50E+04</td><td rowspan=1 colspan=1>300x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.92E+05</td><td rowspan=1 colspan=1>3840x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Vibrio cholerae</td><td rowspan=5 colspan=1>ATCC25870</td><td rowspan=5 colspan=1>0:1 Vibrio, vib+</td><td rowspan=5 colspan=1>4.90E+02</td><td rowspan=1 colspan=1>1.47E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>4</td><td rowspan=4 colspan=2>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4.90E+04</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>9.80E+04</td><td rowspan=1 colspan=1>200x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>9.80E+05</td><td rowspan=1 colspan=1>2000x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Vibrio cholerae</td><td rowspan=5 colspan=1>BEI NR-146</td><td rowspan=5 colspan=1>0:1 strain 2125same as ATTC39050 EL Tor</td><td rowspan=5 colspan=1>1.47E+03</td><td rowspan=1 colspan=1>1.47E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>6</td><td rowspan=2 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4.44E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.48E+04</td><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4.90E+04</td><td rowspan=1 colspan=1>33x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.47E+05</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=6 colspan=1>Vibrio cholerae</td><td rowspan=6 colspan=1>BEI NR-149</td><td rowspan=6 colspan=1>0:2 (non-O:1,non- 0:139strain) Nanking32/123</td><td rowspan=6 colspan=1>9.20E+03</td><td rowspan=1 colspan=1>1.47E+03</td><td rowspan=1 colspan=1>0.16x</td><td rowspan=1 colspan=1>3</td><td rowspan=5 colspan=2>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>9.80E+03</td><td rowspan=1 colspan=1>1.1x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>1.47E+04</td><td rowspan=1 colspan=1>1.6x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>2.7x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.76E+04</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9.20E+04</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Vibrio vulnificus</td><td rowspan=2 colspan=1>ATCC27562</td><td rowspan=2 colspan=1>(vib+) strain324 CDC B9629</td><td rowspan=2 colspan=1>4.90E+02</td><td rowspan=1 colspan=1>1.47E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>1.96E+03</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>Vibrio vulnificus</td><td rowspan=4 colspan=1>ATCC29306</td><td rowspan=4 colspan=1>strain CDCA1402</td><td rowspan=4 colspan=1>2.45E+03</td><td rowspan=1 colspan=1>7.35E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>7.35E+04</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.23E+05</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.45E+05</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>Bilups-1803-68,</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>Straincharacterization</td><td rowspan=1 colspan=1>LoD(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>ConcentrationPrepared(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>Multipleof LOD</td><td rowspan=1 colspan=1>Replicatestested</td><td rowspan=1 colspan=3>Falsenegative</td><td rowspan=1 colspan=1>Falsenegativedistribution</td></tr><tr><td rowspan=3 colspan=1>enterocolitica</td><td rowspan=3 colspan=1>23715</td><td rowspan=3 colspan=1>biotype2,serotype 0:8</td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>4.50E+04</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>4</td><td rowspan=3 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.50E+05</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.48E+07</td><td rowspan=1 colspan=1>9900x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=3 colspan=1>ATCC29913</td><td rowspan=3 colspan=1>serotype O:8CDC 497-70, BEINR-207</td><td rowspan=3 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6.00E+03</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7.50E+04</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=3 colspan=1>ATCC9610</td><td rowspan=3 colspan=1>0:8 biovar 1strain NCTC12982</td><td rowspan=3 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>7.50E+04</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>7</td><td rowspan=2 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.50E+05</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=3 colspan=1>BEI NR-206</td><td rowspan=3 colspan=1>0:8 ATCC 27729strain WA</td><td rowspan=3 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.00E+04</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4.50E+04</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=3 colspan=1>BEI NR-212</td><td rowspan=3 colspan=1>0:3 strain NCTC11175</td><td rowspan=3 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.50E+05</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4.50E+05</td><td rowspan=1 colspan=1>300x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Yersiniaenterocolitica</td><td rowspan=2 colspan=1>BEI NR-213</td><td rowspan=2 colspan=1>0:9 Strain NCTC11174</td><td rowspan=2 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>1.50E+05</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Entamoebahistolytica</td><td rowspan=5 colspan=1>BEI NR-176</td><td rowspan=5 colspan=1>HB-301:NIH</td><td rowspan=5 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>9.30E-01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>1.24E+00</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>2</td><td rowspan=4 colspan=3>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9.30E+00</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9.30E+01</td><td rowspan=1 colspan=1>300x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>Entamoebahistolytica</td><td rowspan=3 colspan=1>BEI NR-177</td><td rowspan=3 colspan=1>200:NIH</td><td rowspan=3 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>9.30E-01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.55E+01</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>8</td><td rowspan=2 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>Entamoebahistolytica</td><td rowspan=4 colspan=1>BEI NR-178</td><td rowspan=4 colspan=1>HM-1:IMSS</td><td rowspan=4 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>9.30E-01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6.20E+00</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=3>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>9.30E+01</td><td rowspan=1 colspan=1>300x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=6 colspan=1>Entamoebahistolytica</td><td rowspan=6 colspan=1>BEI NR-179</td><td rowspan=6 colspan=1>Rahman</td><td rowspan=6 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>9.30E-01</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>9.30E+00</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>1.55E+01</td><td rowspan=3 colspan=1>50x</td><td rowspan=3 colspan=1>3</td><td rowspan=3 colspan=1></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=3>0</td><td></td></tr><tr><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.00E+01</td><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Giardia lamblia</td><td rowspan=2 colspan=1>BEI NR-9232</td><td rowspan=2 colspan=1>Mario strain</td><td rowspan=2 colspan=1>1.81E+03</td><td rowspan=1 colspan=1>3.62E+03</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5.42E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=3>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>Straincharacterization</td><td rowspan=1 colspan=1>LoD(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>ConcentrationPrepared(CFU/mL oroocysts/mL)</td><td rowspan=1 colspan=1>Multipleof LOD</td><td rowspan=1 colspan=1>Replicatestested</td><td rowspan=1 colspan=1>Falsenegative</td><td rowspan=1 colspan=1>Falsenegativedistribution</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1>9.04E+03</td><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>3</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.28E+04</td><td rowspan=1 colspan=1>7x</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.81E+04</td><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6.92E+04</td><td rowspan=1 colspan=1>38x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=6 colspan=1>Giardia lamblia</td><td rowspan=6 colspan=1>BEI NR-9234</td><td rowspan=6 colspan=1>D.Hall strain</td><td rowspan=6 colspan=1>1.81E+03</td><td rowspan=1 colspan=1>5.42E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9.94E+03</td><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>1.63E+04</td><td rowspan=1 colspan=1>9x</td><td rowspan=1 colspan=1>2</td><td rowspan=4 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.81E+04</td><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.62E+04</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4.52E+04</td><td rowspan=1 colspan=1>25x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Giardia lamblia</td><td rowspan=5 colspan=1>BEI NR-9235</td><td rowspan=5 colspan=1>DAN strain</td><td rowspan=5 colspan=1>1.81E+03</td><td rowspan=1 colspan=1>5.42E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>9.04E+03</td><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>2</td><td rowspan=4 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.62E+04</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5.42E+04</td><td rowspan=1 colspan=1>30x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6.36E+04</td><td rowspan=1 colspan=1>35x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=4 colspan=1>Giardia lamblia</td><td rowspan=4 colspan=1>WaterborneP101</td><td rowspan=4 colspan=1>Lot# 160428</td><td rowspan=4 colspan=1>1.81E+03</td><td rowspan=1 colspan=1>5.42E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2.71E+04</td><td rowspan=1 colspan=1>15x</td><td rowspan=1 colspan=1>6</td><td rowspan=3 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.62E+04</td><td rowspan=1 colspan=1>20x</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3.75E+04</td><td rowspan=1 colspan=1>21x</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr></table>

# Performance of Controls During Clinical Trials

During clinical evaluation of the BioCode GPP, at least one negative control was included on each run. Negative controls were S.T.A.R. Buffer, or well characterized negative specimens. The negative controls completed all processing steps (pretreatment/extractions/amplification/detection). It was recommended that external controls consisting of 4 pools of inactivated organisms (see table) be assayed on a rotating basis with the exception of Giardia (Waterborne Inc.), all controls were prepared by ZeptoMetrix (cat no. NATGIP-ABC; Giardia is now also available with that catalog number).

<table><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Dilution Factor</td></tr><tr><td rowspan=3 colspan=1>Pool 1</td><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>NAP1</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>WA</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Z004</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=5 colspan=1>Pool 2</td><td rowspan=1 colspan=1>Escherichia coli (EAEC)</td><td rowspan=1 colspan=1>92.0147; EAEC</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>DS4-868</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Norovirus Gll</td><td rowspan=1 colspan=1>recombinant</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=4 colspan=1>Pool 3</td><td rowspan=1 colspan=1>Adenovirus Type 41</td><td rowspan=1 colspan=1>TAK</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli0157/STEC</td><td rowspan=1 colspan=1>EDL933</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>Waterborne Inc.</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>Z005</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=4 colspan=1>Pool 4</td><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidiumparvum</td><td rowspan=1 colspan=1>lowa</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (ETEC)</td><td rowspan=1 colspan=1>ETEC; ST+, LT+</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Norovirus Gl</td><td rowspan=1 colspan=1>recombinant</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr></table>

Performance of Controls during Clinical Trials (including Reproducibility testing)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site 001(valid/total)</td><td rowspan=1 colspan=1>Site 002(valid/total)</td><td rowspan=1 colspan=1>Site 003(valid/total)</td><td rowspan=1 colspan=1>Site 006(valid/total)</td><td rowspan=1 colspan=1>All sites(valid/total)</td></tr><tr><td rowspan=1 colspan=1>Pool 1</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>11/14a</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>28/31</td></tr><tr><td rowspan=1 colspan=1>Pool 2</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>Pool 3</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Pool 4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>24/27b</td><td rowspan=1 colspan=1>34/35c</td><td rowspan=1 colspan=1>38/39d</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>111/116</td></tr></table>

a – The same pool 1 extract was being repeated freeze-thawed. It failed 3 times in a row for low signals, while the RNA IC signals were normal in the other samples and NC for the plate. When the group performed fresh extractions the PC signals returned to normal. b – 2 failed for False positive signals and 1 for RNA IC not being detected c – 1 RNA IC not detected d – 1 RNA IC not detected

# General Performance of Assay During Clinical Trials

Accounting of valid, partially invalid, and invalid runs.   

<table><tr><td rowspan=1 colspan=1>Run Description</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>% of Total</td></tr><tr><td rowspan=1 colspan=1>Valid runs with complete results</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>49.5</td></tr><tr><td rowspan=1 colspan=1>Valid runs with RNA-IC failures for oneor more samplesa</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>33.6</td></tr><tr><td rowspan=1 colspan=1>Partially or completely invalid runs</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>16.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>100</td></tr></table>

a – All invalid results were reflex tested according to the IFU, and all but 3 were resolved as valid results.

Summary of issues causing partially or completely invalid runs.   

<table><tr><td rowspan=1 colspan=1>Reason for failure</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>% of Total</td></tr><tr><td rowspan=1 colspan=1>User error</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>Instrument/Alignment a</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>Negative Control contamination</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>Software installation error b</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>Reagent storage/ handlingc</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>Unknown reason</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>Total Invalid runs</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>16.8</td></tr></table>

a MDx 3000 Alignment error at one clinical site accounted for 3 consecutive failed runs before it was corrected. b Unapproved Software for remote access was installed that resulted in a software error for 2 runs software was removed and issue did not repeat. c Reagent storage/handling error at one site accounted for 2 consecutive failed runs.

# Clinical Specificity – Microbial Detection in Asymptomatic Volunteers

In order to determine baseline levels for each analyte included in the BioCode Gastrointestinal Pathogen Panel in individuals who are not exhibiting signs and symptoms of infectious gastroenteritis, 125 clinical stool samples were collected from healthy asymptomatic donors. These are defined as donors not exhibiting signs and symptoms, or on antibiotics (for symptoms) during the previous 30 days. Asymptomatic donors from two sites, Tampa General Hospital (clinical site 2) and University of Maryland (clinical site 3) and various age groups were included in this study and the demographic information for the donors is shown in the table below. PCR inhibition, as determined by results of the assay internal control (bacteriophage MS2), was observed for two samples $( 1 . 6 \% )$ . After re-running this sample in accordance with the package insert instructions for use, inhibition was still observed in, so no result was reported. A total of 26 samples were positive. The results are summarized in the Table below.

Table. Demographic information for Asymptomatic Volunteers.

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>58</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1-5</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>&gt;60</td><td rowspan=1 colspan=1>60</td></tr></table>

Table. Detections in Asymptomatic Volunteers-Stratified by Age   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>&lt; 5 yrs</td><td rowspan=1 colspan=1>6-21 yrs</td><td rowspan=1 colspan=1>22-59 yrs</td><td rowspan=1 colspan=1>60+ yrs</td></tr><tr><td rowspan=1 colspan=1>All Negative</td><td rowspan=1 colspan=1>1 (100.0%)</td><td rowspan=1 colspan=1>3 (100.0%)</td><td rowspan=1 colspan=1>49 (80.33%)</td><td rowspan=1 colspan=1>46 (76.67%)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>9 (14.75%)</td><td rowspan=1 colspan=1>11 (18.33%)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>1 (1.67%)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>2 (3.28%)</td><td rowspan=1 colspan=1>0 (0.00%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>1 (1.67%)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>1 (1.64%)</td><td rowspan=1 colspan=1>0 (0.00%)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GIl</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>0 (0.00%)</td><td rowspan=1 colspan=1>1 (1.67%)</td></tr></table>

# ANALYTICAL PERFORMANCE

The results of the analytical studies summarized in the following paragraphs met the acceptance criteria and successfully demonstrated the analytical performance characteristics of the proposed BioCode Gastrointestinal Pathogen Panel assay.

# REPRODUCIBILITY STUDY

A study was performed to assess the Reproducibility of the BioCode Gastrointestinal Pathogen Panel on the BioCode MDx 3000. This study was designed to assess intra-assay (within run), Inter-assay (run-torun), day-to-day and site-to-site reproducibility. One lot of reagents was assayed at 3 sites by 2 operators on 1 instrument per site for 5 days (total of 10 runs per site; 90 replicates total). The reproducibility panel consisted of 7 contrived samples (sample 7 a negative control) extracted in triplicate and each assayed in singlet. The samples consisted of combinations of 12 representative targets at 1.5x LoD (Low) and $3 { \tt x }$ LoD (Medium). Reproducibility was $> 9 9 \%$ .

Qualitative results from Reproducibility study. All results are as expected with the exception of one false positive for Giardia lamblia and one false negative for STEC.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Concentration Level</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Medium Positive</td><td rowspan=1 colspan=2>Low Positive</td><td rowspan=1 colspan=2>Negative</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Detectionn/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Detectionn/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lambia</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>449/450(99.78)</td><td rowspan=1 colspan=1>(98.77,99.99)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>89/90(98.89)</td><td rowspan=1 colspan=1>( 93.96,99.97)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>( 95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>( 99.18,100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidiumparvum</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.98,100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.18,100.0)</td></tr></table>

# Premarket Notification 510(k)

Quantitative results from Reproducibility study. Fluorescent signals from BMBs with the same barcode are sorted and calculated to generate median fluorescence index (MFI) for each analyte (shown below). The presence or absence of a pathogen is determined relative to the validated assay cutoff by MFI.

<table><tr><td rowspan=2 colspan=1>TargetAnalyte/ Probe</td><td rowspan=2 colspan=1>AnalyteConcentration</td><td rowspan=2 colspan=1>AnalyteMean</td><td rowspan=1 colspan=2>Repeatability</td><td rowspan=1 colspan=2>Between Runs</td><td rowspan=1 colspan=2>Between Days</td><td rowspan=1 colspan=2>Between Sites</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>9174</td><td rowspan=1 colspan=1>1853</td><td rowspan=1 colspan=1>20.2</td><td rowspan=1 colspan=1>622</td><td rowspan=1 colspan=1>6.78</td><td rowspan=1 colspan=1>1464</td><td rowspan=1 colspan=1>15.95</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2442</td><td rowspan=1 colspan=1>26.62</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>11572</td><td rowspan=1 colspan=1>1204</td><td rowspan=1 colspan=1>10.41</td><td rowspan=1 colspan=1>469</td><td rowspan=1 colspan=1>4.05</td><td rowspan=1 colspan=1>1980</td><td rowspan=1 colspan=1>17.11</td><td rowspan=1 colspan=1>1190</td><td rowspan=1 colspan=1>10.28</td><td rowspan=1 colspan=1>2647</td><td rowspan=1 colspan=1>22.87</td></tr><tr><td rowspan=1 colspan=1>tcdA (C. difficile)</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>5144</td><td rowspan=1 colspan=1>1407</td><td rowspan=1 colspan=1>27.35</td><td rowspan=1 colspan=1>1306</td><td rowspan=1 colspan=1>25.4</td><td rowspan=1 colspan=1>689</td><td rowspan=1 colspan=1>13.39</td><td rowspan=1 colspan=1>882</td><td rowspan=1 colspan=1>17.15</td><td rowspan=1 colspan=1>2222</td><td rowspan=1 colspan=1>43.2</td></tr><tr><td rowspan=1 colspan=1>tcdA (C. difficile)</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>8802</td><td rowspan=1 colspan=1>2381</td><td rowspan=1 colspan=1>27.05</td><td rowspan=1 colspan=1>1444</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>1645</td><td rowspan=1 colspan=1>18.69</td><td rowspan=1 colspan=1>1337</td><td rowspan=1 colspan=1>15.19</td><td rowspan=1 colspan=1>3499</td><td rowspan=1 colspan=1>39.76</td></tr><tr><td rowspan=1 colspan=1>tcdB (C. difficile)</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>11595</td><td rowspan=1 colspan=1>2004</td><td rowspan=1 colspan=1>17.29</td><td rowspan=1 colspan=1>847</td><td rowspan=1 colspan=1>7.31</td><td rowspan=1 colspan=1>2293</td><td rowspan=1 colspan=1>19.78</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3161</td><td rowspan=1 colspan=1>27.27</td></tr><tr><td rowspan=1 colspan=1>tcdB (C. difficil)</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>16441</td><td rowspan=1 colspan=1>1878</td><td rowspan=1 colspan=1>11.42</td><td rowspan=1 colspan=1>2219</td><td rowspan=1 colspan=1>13.49</td><td rowspan=1 colspan=1>2804</td><td rowspan=1 colspan=1>17.05</td><td rowspan=1 colspan=1>682</td><td rowspan=1 colspan=1>4.15</td><td rowspan=1 colspan=1>4096</td><td rowspan=1 colspan=1>24.91</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>30206</td><td rowspan=1 colspan=1>2436</td><td rowspan=1 colspan=1>8.06</td><td rowspan=1 colspan=1>2164</td><td rowspan=1 colspan=1>7.16</td><td rowspan=1 colspan=1>2516</td><td rowspan=1 colspan=1>8.33</td><td rowspan=1 colspan=1>3342</td><td rowspan=1 colspan=1>11.07</td><td rowspan=1 colspan=1>5302</td><td rowspan=1 colspan=1>17.55</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>20485</td><td rowspan=1 colspan=1>1024</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3824</td><td rowspan=1 colspan=1>18.67</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2839</td><td rowspan=1 colspan=1>13.86</td><td rowspan=1 colspan=1>4871</td><td rowspan=1 colspan=1>23.78</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>17155</td><td rowspan=1 colspan=1>2043</td><td rowspan=1 colspan=1>11.91</td><td rowspan=1 colspan=1>2850</td><td rowspan=1 colspan=1>16.62</td><td rowspan=1 colspan=1>1987</td><td rowspan=1 colspan=1>11.58</td><td rowspan=1 colspan=1>1030</td><td rowspan=1 colspan=1>6.01</td><td rowspan=1 colspan=1>4160</td><td rowspan=1 colspan=1>24.25</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>20850</td><td rowspan=1 colspan=1>1699</td><td rowspan=1 colspan=1>8.15</td><td rowspan=1 colspan=1>2666</td><td rowspan=1 colspan=1>12.79</td><td rowspan=1 colspan=1>1949</td><td rowspan=1 colspan=1>9.35</td><td rowspan=1 colspan=1>2447</td><td rowspan=1 colspan=1>11.74</td><td rowspan=1 colspan=1>4448</td><td rowspan=1 colspan=1>21.33</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>6115</td><td rowspan=1 colspan=1>1654</td><td rowspan=1 colspan=1>27.05</td><td rowspan=1 colspan=1>1487</td><td rowspan=1 colspan=1>24.31</td><td rowspan=1 colspan=1>1337</td><td rowspan=1 colspan=1>21.87</td><td rowspan=1 colspan=1>1043</td><td rowspan=1 colspan=1>17.06</td><td rowspan=1 colspan=1>2797</td><td rowspan=1 colspan=1>45.74</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>9707</td><td rowspan=1 colspan=1>2492</td><td rowspan=1 colspan=1>25.68</td><td rowspan=1 colspan=1>1968</td><td rowspan=1 colspan=1>20.28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1669</td><td rowspan=1 colspan=1>17.19</td><td rowspan=1 colspan=1>3588</td><td rowspan=1 colspan=1>36.96</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>9927</td><td rowspan=1 colspan=1>2690</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>7.98</td><td rowspan=1 colspan=1>2956</td><td rowspan=1 colspan=1>29.78</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>9.73</td><td rowspan=1 colspan=1>4188</td><td rowspan=1 colspan=1>42.18</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>12421</td><td rowspan=1 colspan=1>1540</td><td rowspan=1 colspan=1>12.39</td><td rowspan=1 colspan=1>2018</td><td rowspan=1 colspan=1>16.25</td><td rowspan=1 colspan=1>3126</td><td rowspan=1 colspan=1>25.16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4026</td><td rowspan=1 colspan=1>32.42</td></tr><tr><td rowspan=1 colspan=1>ST-a (ETEC)</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>14272</td><td rowspan=1 colspan=1>3469</td><td rowspan=1 colspan=1>24.31</td><td rowspan=1 colspan=1>3562</td><td rowspan=1 colspan=1>24.96</td><td rowspan=1 colspan=1>3910</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6326</td><td rowspan=1 colspan=1>44.32</td></tr><tr><td rowspan=1 colspan=1>ST-a (ETEC)</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>21453</td><td rowspan=1 colspan=1>4151</td><td rowspan=1 colspan=1>19.35</td><td rowspan=1 colspan=1>2943</td><td rowspan=1 colspan=1>13.72</td><td rowspan=1 colspan=1>4869</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>2322</td><td rowspan=1 colspan=1>10.83</td><td rowspan=1 colspan=1>7416</td><td rowspan=1 colspan=1>34.57</td></tr><tr><td rowspan=1 colspan=1>ST-b (ETEC)</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>13193</td><td rowspan=1 colspan=1>3246</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>1992</td><td rowspan=1 colspan=1>15.1</td><td rowspan=1 colspan=1>3719</td><td rowspan=1 colspan=1>28.19</td><td rowspan=1 colspan=1>733</td><td rowspan=1 colspan=1>5.56</td><td rowspan=1 colspan=1>5373</td><td rowspan=1 colspan=1>40.73</td></tr><tr><td rowspan=1 colspan=1>ST-b (ETEC)</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>19807</td><td rowspan=1 colspan=1>3205</td><td rowspan=1 colspan=1>16.18</td><td rowspan=1 colspan=1>2141</td><td rowspan=1 colspan=1>10.81</td><td rowspan=1 colspan=1>4756</td><td rowspan=1 colspan=1>24.01</td><td rowspan=1 colspan=1>1829</td><td rowspan=1 colspan=1>9.23</td><td rowspan=1 colspan=1>6389</td><td rowspan=1 colspan=1>32.26</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>19049</td><td rowspan=1 colspan=1>1970</td><td rowspan=1 colspan=1>10.34</td><td rowspan=1 colspan=1>1729</td><td rowspan=1 colspan=1>9.08</td><td rowspan=1 colspan=1>1647</td><td rowspan=1 colspan=1>8.65</td><td rowspan=1 colspan=1>2326</td><td rowspan=1 colspan=1>12.21</td><td rowspan=1 colspan=1>3872</td><td rowspan=1 colspan=1>20.33</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>20020</td><td rowspan=1 colspan=1>1246</td><td rowspan=1 colspan=1>6.22</td><td rowspan=1 colspan=1>1474</td><td rowspan=1 colspan=1>7.36</td><td rowspan=1 colspan=1>2036</td><td rowspan=1 colspan=1>10.17</td><td rowspan=1 colspan=1>2156</td><td rowspan=1 colspan=1>10.77</td><td rowspan=1 colspan=1>3538</td><td rowspan=1 colspan=1>17.67</td></tr><tr><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>16291</td><td rowspan=1 colspan=1>3347</td><td rowspan=1 colspan=1>20.54</td><td rowspan=1 colspan=1>5757</td><td rowspan=1 colspan=1>35.34</td><td rowspan=1 colspan=1>2353</td><td rowspan=1 colspan=1>14.44</td><td rowspan=1 colspan=1>5161</td><td rowspan=1 colspan=1>31.68</td><td rowspan=1 colspan=1>8747</td><td rowspan=1 colspan=1>53.69</td></tr><tr><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>21117</td><td rowspan=1 colspan=1>2977</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>6931</td><td rowspan=1 colspan=1>32.82</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5593</td><td rowspan=1 colspan=1>26.49</td><td rowspan=1 colspan=1>9390</td><td rowspan=1 colspan=1>44.47</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>21865</td><td rowspan=1 colspan=1>2384</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>3282</td><td rowspan=1 colspan=1>15.01</td><td rowspan=1 colspan=1>2727</td><td rowspan=1 colspan=1>12.47</td><td rowspan=1 colspan=1>2302</td><td rowspan=1 colspan=1>10.53</td><td rowspan=1 colspan=1>5402</td><td rowspan=1 colspan=1>24.71</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>26561</td><td rowspan=1 colspan=1>2992</td><td rowspan=1 colspan=1>11.26</td><td rowspan=1 colspan=1>3871</td><td rowspan=1 colspan=1>14.58</td><td rowspan=1 colspan=1>2549</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>2903</td><td rowspan=1 colspan=1>10.93</td><td rowspan=1 colspan=1>6234</td><td rowspan=1 colspan=1>23.47</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45792</td><td rowspan=1 colspan=1>4269</td><td rowspan=1 colspan=1>9.32</td><td rowspan=1 colspan=1>3274</td><td rowspan=1 colspan=1>7.15</td><td rowspan=1 colspan=1>2960</td><td rowspan=1 colspan=1>6.46</td><td rowspan=1 colspan=1>5063</td><td rowspan=1 colspan=1>11.06</td><td rowspan=1 colspan=1>7959</td><td rowspan=1 colspan=1>17.38</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>45552</td><td rowspan=1 colspan=1>4673</td><td rowspan=1 colspan=1>10.26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4124</td><td rowspan=1 colspan=1>9.05</td><td rowspan=1 colspan=1>7622</td><td rowspan=1 colspan=1>16.73</td><td rowspan=1 colspan=1>9846</td><td rowspan=1 colspan=1>21.61</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>25298</td><td rowspan=1 colspan=1>3754</td><td rowspan=1 colspan=1>14.84</td><td rowspan=1 colspan=1>2212</td><td rowspan=1 colspan=1>8.74</td><td rowspan=1 colspan=1>3314</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>3639</td><td rowspan=1 colspan=1>14.38</td><td rowspan=1 colspan=1>6573</td><td rowspan=1 colspan=1>25.98</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>27966</td><td rowspan=1 colspan=1>2649</td><td rowspan=1 colspan=1>9.47</td><td rowspan=1 colspan=1>3123</td><td rowspan=1 colspan=1>11.17</td><td rowspan=1 colspan=1>2672</td><td rowspan=1 colspan=1>9.56</td><td rowspan=1 colspan=1>3226</td><td rowspan=1 colspan=1>11.54</td><td rowspan=1 colspan=1>5859</td><td rowspan=1 colspan=1>20.95</td></tr></table>

# LIMIT OF DETECTION (LoD)

A study was performed to assess the performance of the BioCode Gastrointestinal Pathogen Panel on the BioCode MDx 3000 at the Limit of Detection (LoD) for both unpreserved Stool and Cary- Blair specimens. In this study the BioCode GPP was tested with quantified bacteria, virus or parasite stocks (note Norovirus GI and Norovirus GII were tested by CDC). For initial screening, four replicates of each concentration in negative stool and Cary-Blair were extracted on the easyMAG System and tested in singlet with the BioCode GPP on the BioCode MDx 3000 system to estimate LoD. The LoD was confirmed by extracting 20 replicates of each sample type and testing each in singlet for a total of 20 replicates at or near presumptive LoD. LoD for each stock was defined as the lowest concentration with $29 5 \%$ detection of 20 replicates (19 out of 20), and was determined separately unpreserved stool and Cary-Blair preserved stool.

Limit of Detection (LoD) for BioCode GPP.   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Unpreserved Stool LoD</td><td rowspan=1 colspan=1>UnpreservedStoolDetection</td><td rowspan=1 colspan=1>Cary-Blair Stool LoD</td><td rowspan=1 colspan=1>Cary-BlairStoolDetection</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 33559</td><td rowspan=1 colspan=1>5.6 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.6 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>7.0 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.0 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(toxinotype 0)</td><td rowspan=1 colspan=1>ATCC 9689</td><td rowspan=1 colspan=1>1.9 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.9 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(toxinotype III; Nap1)</td><td rowspan=1 colspan=1>Zeptometrix0801619cf</td><td rowspan=1 colspan=1>8.3 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.3 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli092:H33 (EAEC)</td><td rowspan=1 colspan=1>STEC TW04440</td><td rowspan=1 colspan=1>1.4 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.4 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enteroinvasive E. coli029:NM (EIEC)</td><td rowspan=1 colspan=1>ATCC 43892</td><td rowspan=1 colspan=1>3.6 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.5 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli078:H11 H10407 (ETEC)</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>5.6 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.6 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>SGSC 4900</td><td rowspan=1 colspan=1>1.4 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica ssp.enterica</td><td rowspan=1 colspan=1>ATCC 14028</td><td rowspan=1 colspan=1>2.2 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.1 x 10³ CFU/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>2.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>2.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>ATCC 700376</td><td rowspan=1 colspan=1>3.3 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.3 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>4.4 x 10²2 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.7 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>ATCC 25870</td><td rowspan=1 colspan=1>4.9 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4.9 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td><td rowspan=1 colspan=1>1.3 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.0 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Unpreserved Stool LoD</td><td rowspan=1 colspan=1>UnpreservedStoolDetection</td><td rowspan=1 colspan=1>Cary-Blair Stool LoD</td><td rowspan=1 colspan=1>Cary-BlairStoolDetection</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 23715</td><td rowspan=1 colspan=1>1.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>UKRC</td><td rowspan=1 colspan=1>1.3 x 104oocysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.3 x 104oocysts/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>waterborneP102C</td><td rowspan=1 colspan=1>3.1 x 103oocysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.1 x 10³3oocysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolyticaHB-301:NIH</td><td rowspan=1 colspan=1>BEI NR-178</td><td rowspan=1 colspan=1>3.1 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.1 x 10-1cysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis(aka G. lamblia)</td><td rowspan=1 colspan=1>waterborneP101</td><td rowspan=1 colspan=1>1.8 x 10³ cysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.8 x 10³ cysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>Zeptometrix0810084</td><td rowspan=1 colspan=1>1.0 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.0 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41 (TAK)</td><td rowspan=1 colspan=1>Zeptometrix0810085</td><td rowspan=1 colspan=1>9.4 x 10-2TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.5 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>2.5 x 10³ TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>6.2 x 10²2TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus Gla</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>6.4 x 105$copies/gram</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>6.5 x 10 5$copies/gram</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus Glla</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>5.2 x 104copies/gram</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.96 x 104copies/gram</td><td rowspan=1 colspan=1>20/20</td></tr></table>

# INTERFERING SUBSTANCES

A study was performed to demonstrate the accuracy of the BioCode GPP on the BioCode MDx 3000 in the presence of potentially inhibiting substances or microorganisms. Each member of the interfering substance panel (ISP) was added to prescreened negative stool sample spiked with representative members of the BioCode GPP at 3X LoD and a negative matrix only pre-screened negative stool. One parasite, one virus, one gram-positive bacterium, and one gram-negative bacterium were used as representatives for this study. Each sample was tested with and without potentially interfering substances. Each sample was extracted in triplicate on the easyMag System and tested in singlet with the BioCode GPP on the BioCode MDx 3000 system. Concentrations were determined by reviewing 510k summary results of previous GI panel clinical trials and FDA Guidance. No interfering substances or microorganisms were identified.

Interference Test Panel.   

<table><tr><td>Sample Name</td><td>Organism</td><td>Source</td></tr><tr><td>Sample A</td><td>Clostridium difficile (NAP1) Cryptosporidium parvum</td><td>Zeptometrix 0801619cf Waterborne P102C</td></tr><tr><td>Sample B</td><td>Human Rotavirus A Escherichia coli 10C-3114 (STEC)</td><td>ATCC VR-2018 ATCC BAA-2217</td></tr><tr><td>Sample C</td><td>Negative Matrix Only</td><td>N/A</td></tr></table>

Potential Microbial Interferents. No inhibition or unexpected results were observed in the presence of high titer for the organisms in the table below.   

<table><tr><td rowspan=1 colspan=1>Microbial Interferent</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>Interference Yes(Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>No Interferent (Control)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilisa</td><td rowspan=1 colspan=1>Zeptometrix 0801583</td><td rowspan=1 colspan=1>1 x 106$</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Blastocystis hominisb</td><td rowspan=1 colspan=1>ATCC 50752</td><td rowspan=1 colspan=1>$1 x 10{}$</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 14053</td><td rowspan=1 colspan=1>1 x 10}$</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile non-toxigenic</td><td rowspan=1 colspan=1>ATCC 700057</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>1x106</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli nonpathogenic</td><td rowspan=1 colspan=1>ATCC BAA-97</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 39324</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces boulardii</td><td rowspan=1 colspan=1>ATCC MYA-796</td><td rowspan=1 colspan=1>1 x 10$</td><td rowspan=1 colspan=1>N</td></tr></table>

$\mathtt { a } - 1 / 3$ wells of B. fragilis in the negative stool only sample produced a false positive result for Adenovirus 40/41. An additional 5 extractions were then repeated with no false positives. b – Blastocystis hominis is titered in cells/mL.

# Premarket Notification 510(k)

Potential Interfering Substances.   

<table><tr><td rowspan=1 colspan=1>Substance Interferent</td><td rowspan=1 colspan=1>Brand/Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Interference Yes(Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>No Interferent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Blood (EDTA)</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>40% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Ampicillin</td><td rowspan=1 colspan=1>Ampicillin</td><td rowspan=1 colspan=1>152 µmol/L</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Sodium hypochlorite</td><td rowspan=1 colspan=1>Bleach (10%)</td><td rowspan=1 colspan=1>50% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>Mineral oil, USP</td><td rowspan=1 colspan=1>50% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>Hydrocortisone cream</td><td rowspan=1 colspan=1>50% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Loperamide hydrochloride</td><td rowspan=1 colspan=1>Imodium</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>Senokot</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Magnesium Hydroxide, Aluminum Hydroxide</td><td rowspan=1 colspan=1>Maalox</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>14 mg/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Benzalkonium chloride, Ethanol</td><td rowspan=1 colspan=1>Moist towelettes</td><td rowspan=1 colspan=1>50% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mono, di, triglycerides mix</td><td rowspan=1 colspan=1>Supelco</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>3 mg/ml</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Naproxen sodium</td><td rowspan=1 colspan=1>Naproxen sodium</td><td rowspan=1 colspan=1>14mg/ml</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Polymyxin B sulfate, bacitracin zinc, Neomycin</td><td rowspan=1 colspan=1>Neosporin</td><td rowspan=1 colspan=1>50% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>1000 U/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bismuth subsalicylate</td><td rowspan=1 colspan=1>Pepto-Bismol</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Petrolatum</td><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Calcium carbonate</td><td rowspan=1 colspan=1>Tums</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>12.5 mg/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

# ANALYTICAL REACTIVITY/INCLUSIVITY

A study was performed to verify Analytical Reactivity/Inclusivity of the BioCode Gastrointestinal Pathogen Panel. Different strains were selected that represent various temporal, geographic, and genetic diversity for each analyte. This study tested a panel of titered stocks for relevant organisms diluted in Pre-Screened Negative Stools at 3X LoD. Stocks not detected at 3X LoD, if applicable, were tested at higher concentrations. Due to a lack of titered specimens, Adenovirus 40/41 clinical samples and Cryptosporidium DNA from the Cryptosporidium reference unit were used (Public Health England). Norovirus GI and GII genotypes and the Rotavirus vaccine strain were tested by the CDC. All of the organisms were detected at the concentrations indicated.

Shiga-like toxin producing E. coli (STEC) stx1/stx2 and E. coli O157 Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiplesof LoDDetected</td></tr><tr><td rowspan=5 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>E. coli 0157:H45</td><td rowspan=1 colspan=1>STEC SC373/2 TW07922</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:HNM</td><td rowspan=1 colspan=1>STEC DA-26 TW07952</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:H7</td><td rowspan=1 colspan=1>STEC 93-111 TW04863</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O157: H7</td><td rowspan=1 colspan=1>STEC MI06-19 TW14301</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:HNT</td><td rowspan=1 colspan=1>STEC DA-27 TW07953</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiplesof LoDDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>E. coli O157:H7 Strain EDL933</td><td rowspan=1 colspan=1>BEI NR-11</td><td rowspan=1 colspan=1>9.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=17 colspan=1>Shiga toxinproducing E. coli(STEC)</td><td rowspan=1 colspan=1>E. coli O26:H11</td><td rowspan=1 colspan=1>STEC 2332/00 TW08998</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O113:H21</td><td rowspan=1 colspan=1>STEC CL-15 TW02318</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O45:H2</td><td rowspan=1 colspan=1>STEC DEC11C DEC11C</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O103:H2</td><td rowspan=1 colspan=1>STEC 107-226 TW07881</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O104:H21</td><td rowspan=1 colspan=1>STEC G5506 TW04909</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O111:H2</td><td rowspan=1 colspan=1>STEC RD8 TW06296</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O111:H8</td><td rowspan=1 colspan=1>STEC DEC8B</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O26:NM</td><td rowspan=1 colspan=1>STEC DA-22 TW07948</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0145:NM</td><td rowspan=1 colspan=1>ATCC BAA-2192</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0146:21</td><td rowspan=1 colspan=1>STEC DEC16E TW01383</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O45:H2</td><td rowspan=1 colspan=1>STEC MI05-14 TW14003</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O121:19</td><td rowspan=1 colspan=1>STEC MDCH-4 TW07614</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O121:NM</td><td rowspan=1 colspan=1>STEC DA-37 TW07972</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli *0104:H4</td><td rowspan=1 colspan=1>ATCC BAA-2326</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 045:H2</td><td rowspan=1 colspan=1>ATCC BAA-2193</td><td rowspan=1 colspan=1>7.5 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O26: H11</td><td rowspan=1 colspan=1>BAA-2196</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O0121:H19</td><td rowspan=1 colspan=1>BAA-2219</td><td rowspan=1 colspan=1>7.5 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Enterotoxigenic E. coli (ETEC), Enteroaggregative E. coli (EAEC) Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=4 colspan=1>EnteroaggregativeE. coli (EAEC)</td><td rowspan=1 colspan=1>E. coli O44:H18</td><td rowspan=1 colspan=1>STEC 042TW04393</td><td rowspan=1 colspan=1>4.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O111a,111b:K58:H21</td><td rowspan=1 colspan=1>ATCC 29552</td><td rowspan=1 colspan=1>4.08 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0104:H4</td><td rowspan=1 colspan=1>ATCC BAA-2326</td><td rowspan=1 colspan=1>4.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O3:K2a</td><td rowspan=1 colspan=1>BEI NR-102</td><td rowspan=1 colspan=1>4.08 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Shigella/Enteroinvasive E. coli (EIEC) Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=5 colspan=1>EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1>E. coli O121</td><td rowspan=1 colspan=1>ATCC BAA-2190</td><td rowspan=1 colspan=1>1.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0124:HNM</td><td rowspan=1 colspan=1>STEC 929-78TW16574</td><td rowspan=1 colspan=1>1.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli 0136:H</td><td rowspan=1 colspan=1>STEC LT-41TW06139</td><td rowspan=1 colspan=1>1.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O285A:HNM</td><td rowspan=1 colspan=1>BEI NR-101</td><td rowspan=1 colspan=1>1.08 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>E. coli O15</td><td rowspan=1 colspan=1>ATCC 49105</td><td rowspan=1 colspan=1>1.08 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>Shigella boydii (Type 1)</td><td rowspan=1 colspan=1>ATCC 9207</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii, (Type 2)</td><td rowspan=1 colspan=1>BEI NR-521</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 7)</td><td rowspan=1 colspan=1>ATCC 9905</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 20)</td><td rowspan=1 colspan=1>ATCC BAA-1247</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 3)</td><td rowspan=1 colspan=1>ATCC 8702</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Shigella dysenteriae (Type 1)</td><td rowspan=1 colspan=1>BEI NR-520</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=4 colspan=1>Shigelladysenteriae</td><td rowspan=1 colspan=1>Shigella dysenteriae (Type 3)</td><td rowspan=1 colspan=1>ATCC 9751</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Type 2)</td><td rowspan=1 colspan=1>ATCC 9750</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Type 5)</td><td rowspan=1 colspan=1>ATCC 9764</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Type 12)</td><td rowspan=1 colspan=1>ATCC 49552</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>Shigella flexneri, strain 24570(Type 2a)</td><td rowspan=1 colspan=1>BEI NR-517</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri, strain 2457T(Type 2a)</td><td rowspan=1 colspan=1>BEI NR-518</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 2b)</td><td rowspan=1 colspan=1>ATCC 12022</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 6)</td><td rowspan=1 colspan=1>ATCC 12025</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 1b)</td><td rowspan=1 colspan=1>ATCC 9380</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 25931</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 11060</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 9290</td><td rowspan=1 colspan=1>1.31 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29029</td><td rowspan=1 colspan=1>1.31 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Salmonella Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. arizonae</td><td rowspan=1 colspan=1>Salmonella enterica subsp.Enterica</td><td rowspan=1 colspan=1>ATCC 13314</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. salamae</td><td rowspan=1 colspan=1>serovar Tranoroa</td><td rowspan=1 colspan=1>ATCC 700148</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=23 colspan=1>Salmonella entericasubsp. enterica</td><td rowspan=1 colspan=1>serovar Montevideo</td><td rowspan=1 colspan=1>ATCC 7001BAA-710</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Enteritidis</td><td rowspan=1 colspan=1>SGSC4901</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Enteritidis</td><td rowspan=1 colspan=1>ATCC 4931</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Oranienburg</td><td rowspan=1 colspan=1>SGSC4079</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Paratyphi B var.L(+)tartrate+</td><td rowspan=1 colspan=1>SGSC4150</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Typhimurium</td><td rowspan=1 colspan=1>SGSC1412</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Saintpaul</td><td rowspan=1 colspan=1>SGSC2512</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar S. typhimurium LT2</td><td rowspan=1 colspan=1>SGSC2666</td><td rowspan=1 colspan=1>6.45 x 103 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Newport</td><td rowspan=1 colspan=1>SGSC2493</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Newport</td><td rowspan=1 colspan=1>ATCC 6962</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Muenchen</td><td rowspan=1 colspan=1>SGSC2490</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Agona</td><td rowspan=1 colspan=1>SGSC2458</td><td rowspan=1 colspan=1>6.45 x 103 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Javiana</td><td rowspan=1 colspan=1>SGSC4917</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Schwarzengrund</td><td rowspan=1 colspan=1>SGSC2514</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Heidelberg</td><td rowspan=1 colspan=1>SGSC2480</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Infantis</td><td rowspan=1 colspan=1>SGSC2484</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Montevideo</td><td rowspan=1 colspan=1>SGSC2487</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Thompson</td><td rowspan=1 colspan=1>SGSC 2519</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Hadar</td><td rowspan=1 colspan=1>SGSC4965</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Mississippi</td><td rowspan=1 colspan=1>SGSC4078</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Paratyphi A</td><td rowspan=1 colspan=1>SGSC2499</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Choleraesuis</td><td rowspan=1 colspan=1>SGSC4770</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Choleraesuis</td><td rowspan=1 colspan=1>ATCC 13312</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>serovar Dublin</td><td rowspan=1 colspan=1>SGSC4157</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Braenderup</td><td rowspan=1 colspan=1>ATCC® 700136</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Bareilly</td><td rowspan=1 colspan=1>ATCC® 9115</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>serovar Typhi</td><td rowspan=1 colspan=1>Zeptometrix0801933</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. II</td><td rowspan=1 colspan=1>Salmonella enterica subs. II</td><td rowspan=1 colspan=1>SGSC3039</td><td rowspan=1 colspan=1>6.45 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IIIa</td><td rowspan=1 colspan=1>Salmonella enterica subs. IIla</td><td rowspan=1 colspan=1>SGSC3061</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IIIb</td><td rowspan=1 colspan=1>Salmonella enterica subs. IIIb</td><td rowspan=1 colspan=1>SGSC3068</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IV</td><td rowspan=1 colspan=1>Salmonella enterica subs. IV</td><td rowspan=1 colspan=1>SGSC3074</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. VI</td><td rowspan=1 colspan=1>Salmonella enterica subs VI</td><td rowspan=1 colspan=1>SGSC3116</td><td rowspan=1 colspan=1>6.45 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Campylobacter inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=8 colspan=1>Campylobacter spp.</td><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>BEI NR-399</td><td rowspan=1 colspan=1>2.10 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>BEI NR-400</td><td rowspan=1 colspan=1>2.10 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. doylei</td><td rowspan=1 colspan=1>ATCC 49350</td><td rowspan=1 colspan=1>2.10 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. doylei</td><td rowspan=1 colspan=1>ATCC 49349</td><td rowspan=1 colspan=1>2.10 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43478</td><td rowspan=1 colspan=1>1.68 x 10² CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43485</td><td rowspan=1 colspan=1>1.68 x 10² CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>BEI HM-296</td><td rowspan=1 colspan=1>1.68 x 10² CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43484</td><td rowspan=1 colspan=1>1.68 x 10² CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Vibrio spp inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=11 colspan=1>Vibrio spp. (notparahaemolyticus)</td><td rowspan=1 colspan=1>Vibrio cholerae (0:1 Inaba,Biotype E1 Tor)</td><td rowspan=1 colspan=1>BEI NR-147</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (0:1 Inaba,Biotype E1 Tor)</td><td rowspan=1 colspan=1>BEI NR-146</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae(O:2)</td><td rowspan=1 colspan=1>BEI NR-149</td><td rowspan=1 colspan=1>1.47 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (0:4)</td><td rowspan=1 colspan=1>BEI NR-151</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (0:139)</td><td rowspan=1 colspan=1>BEI NR-144</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3X</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (O:1 Ogawa)</td><td rowspan=1 colspan=1>ATCC 14035</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 27562</td><td rowspan=1 colspan=1>1.47 x 10³3CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC BAA-88</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 43382</td><td rowspan=1 colspan=1>4.9 x 10³ CFU/mL</td><td rowspan=1 colspan=1>10X</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 29306</td><td rowspan=1 colspan=1>1.47 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 29307</td><td rowspan=1 colspan=1>1.47 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-21990</td><td rowspan=1 colspan=1>3.75 x 101 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR- 21991</td><td rowspan=1 colspan=1>3.75 x 101 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-22002</td><td rowspan=1 colspan=1>3.75 x 101 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Zepto 0801903</td><td rowspan=1 colspan=1>3.75 x 101CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-22013</td><td rowspan=1 colspan=1>3.75 x 10² CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Yersinia enterocolitica inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=5 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica (0:8)</td><td rowspan=1 colspan=1>ATCC 9610</td><td rowspan=1 colspan=1>4.43 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica (0:3)</td><td rowspan=1 colspan=1>BEI NR-212</td><td rowspan=1 colspan=1>4.43 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica (0:8)</td><td rowspan=1 colspan=1>BEI NR-206</td><td rowspan=1 colspan=1>4.43 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 29913</td><td rowspan=1 colspan=1>4.43 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>BEI NR-213</td><td rowspan=1 colspan=1>4.43 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Clostridium difficile inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>TOXINOTYPE</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=17 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC 43255-FZ</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC 700792-FZ</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1382-fz</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC 51695-fz</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC 43599-fz</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 43596-fz</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC 17858-fz</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 43594</td><td rowspan=1 colspan=1>2.48 x 103 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 43600</td><td rowspan=1 colspan=1>2.48 x 103 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 17857</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1871</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>0A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1872</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>VIII A-B+</td><td rowspan=1 colspan=1>ATCC 43598</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>III A+B+ (Nap1)</td><td rowspan=1 colspan=1>ATCC BAA-1805</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>XXII A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1814</td><td rowspan=1 colspan=1>2.48 x 10³ CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>III A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1870</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>V A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1875</td><td rowspan=1 colspan=1>2.48 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Adenovirus 40/41 inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration Detected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=3 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Human adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>lot #K1220A</td><td rowspan=1 colspan=1>3.0E-01 TCID50/mL</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP143</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP258</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=7 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP170</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP174</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP 276</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP288</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP309</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP329</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP446</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr></table>

Norovirus GI/GII inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=8 colspan=1>*Norovirus Gl</td><td rowspan=1 colspan=1>GI.1</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.2</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.3</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 108 copies/gram</td><td rowspan=1 colspan=1>300x</td></tr><tr><td rowspan=1 colspan=1>GI.4</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.5</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.6</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.7</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GI.8</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.93 x 106 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=14 colspan=1>aNorovirus Gll</td><td rowspan=1 colspan=1>GII.1</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.2</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GII.3</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GII.4 New Orleans</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GII.4 Sydney</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.5</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.6</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.7</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.8</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GlI.12</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.13</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>GII.14</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GII.16</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 106 copies/gram</td><td rowspan=1 colspan=1>30x</td></tr><tr><td rowspan=1 colspan=1>GII.17</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.57 x 105 copies/gram</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

Norovirus GI/GII inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiples ofLoDDetected</td></tr><tr><td rowspan=14 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Rotavirus A (DS-1)</td><td rowspan=1 colspan=1>ATCCVR-2550</td><td rowspan=1 colspan=1>7.44 x 10³3 IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 89-12C2)</td><td rowspan=1 colspan=1>ATCC VR-2272</td><td rowspan=1 colspan=1>7.44 x 10³3 IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (WISC2)</td><td rowspan=1 colspan=1>ATCCVR-2417</td><td rowspan=1 colspan=1>7.44 x 10³3 IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 408)</td><td rowspan=1 colspan=1>ATCC VR-2273</td><td rowspan=1 colspan=1>7.44 x 10³3 IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 248)</td><td rowspan=1 colspan=1>ATCC VR-2274</td><td rowspan=1 colspan=1>7.44 x 10³3 IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HU)</td><td rowspan=1 colspan=1>ATCC VR-1546</td><td rowspan=1 colspan=1>7.44 x 10³ IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV CJN)</td><td rowspan=1 colspan=1>ATCC VR-2275</td><td rowspan=1 colspan=1>7.44 x 10³ IU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G1</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>Ct=20</td><td rowspan=7 colspan=1>N/Aa</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G2</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>Ct= 22</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G3</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>Ct= 26</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G4</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>Ct=21</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G9</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>Ct= 20</td></tr><tr><td rowspan=1 colspan=1>Rota vaccine strain</td><td rowspan=1 colspan=1>RotaTeq vaccine</td><td rowspan=1 colspan=1>1.00 x 108 pfu/mL</td></tr><tr><td rowspan=1 colspan=1>Rota vaccine strain</td><td rowspan=1 colspan=1>Rotarix vaccine</td><td rowspan=1 colspan=1>1.00 x 106 CCID5o/mL</td></tr></table>

a – Tested at CDC at indicated concentration at CDC, concentration cannot be related to LoD.

# Cryptosporidium spp inclusivity results.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration Detected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=5 colspan=1>C. parvum</td><td rowspan=1 colspan=1>DNA subtype /llaA17G1R1</td><td rowspan=1 colspan=1>UKCR UK28</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /llaA15G2R1</td><td rowspan=1 colspan=1>UKCR UK29</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /llaA19G1R1</td><td rowspan=1 colspan=1>UKCR UK30</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /lldA22G1</td><td rowspan=1 colspan=1>UKCR UK31</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /lldA15G1</td><td rowspan=1 colspan=1>UKCR UK32</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=4 colspan=1>C. hominis</td><td rowspan=1 colspan=1>DNA subtype laA14R3</td><td rowspan=1 colspan=1>UKCR UKH14</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype IdA18</td><td rowspan=1 colspan=1>UKH12</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA subtype IbA10G2</td><td rowspan=1 colspan=1>UKH13</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>DNA E. coli O103:H2</td><td rowspan=1 colspan=1>NR2520</td><td rowspan=1 colspan=1>Clinical Sample unknown</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Entamoeba histolytica inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=4 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>200:NIH</td><td rowspan=1 colspan=1>BEI NR-177</td><td rowspan=1 colspan=1>9.36 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>HB-301:NIH</td><td rowspan=1 colspan=1>BEI NR-176</td><td rowspan=1 colspan=1>9.36 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Rahman</td><td rowspan=1 colspan=1>BEI NR-179</td><td rowspan=1 colspan=1>9.36 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>H-303:NIH</td><td rowspan=1 colspan=1>BEI NR-180</td><td rowspan=1 colspan=1>9.36 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

Giardia lamblia inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=4 colspan=1>Giardia lamblia/intestinalis</td><td rowspan=1 colspan=1>Egypt-4</td><td rowspan=1 colspan=1>BEI NR-9231</td><td rowspan=1 colspan=1>5.42 x 10³ cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Mario</td><td rowspan=1 colspan=1>BEI NR-9232</td><td rowspan=1 colspan=1>5.42 x 10³ cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>D.Hall</td><td rowspan=1 colspan=1>BEI NR-9234</td><td rowspan=1 colspan=1>5.42 x 10³ cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>DAN</td><td rowspan=1 colspan=1>BEI NR-9235</td><td rowspan=1 colspan=1>5.42 x 10³3 cysts/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# ANALYTICAL SPECIFICITY/CROSS REACTIVITY

A study was performed to verify that the BioCode GPP does not detect DNA or RNA from organisms commonly found in stool specimens or from organisms that can cause similar clinical symptoms. In addition, on-panel organisms were tested at high concentrations to insure there is no cross-reactivity with other panel targets. This study tested a panel of titered stocks and genomic DNA extracts for relevant organisms. Organisms that were not available for wet testing were analyzed in silico comparing the whole organism sequence against all primers to assess potential for cross reactivity. Analysis was conducted using BlastN and Primer Blast programs.

Cross-reactivity was not observed with microorganisms tested in this study except for the following.

Empirical testing and in silico sequence analysis indicate that the Vibrio spp assay may also react with some less common Vibrio species (i.e., V. alginolyticus, and V. mimicus).

Empirical testing and in silico sequence analysis indicate a potential for cross-reactivity with Y. bercovieri, Y. frederiksenii, Y. intermedia and Y. mollaretii near the established LoD for Y. entericolitica $( \mathsf { \tilde { \Delta } } 1 . 5 \times 1 0 ^ { 3 } \mathsf { C F U / m L } )$ . Y. rohdei was also detected when present at high levels $( > 6 . 8 \times 1 0 ^ { 4 } \thinspace \mathsf { C F U / m L } )$ . These species are in the Y. enterocolitica group and are suspected human pathogens.

Shiga toxin (stx; identical to stx1 of STEC) is found in Shigella dysenteriae; therefore, a BioCode GPP report with positive test results for Shiga-like toxin-producing E. coli (STEC) and Shigella/Enteroinvasive E. coli (EIEC) in the same sample may indicate the presence of S. dysenteriae.

Empirical testing with a gene fragment construct and in silico sequence analysis do not predict cross reactivity with the closely related E. dispar.

Empirical testing has demonstrated that these assays will detect recombinant viruses included in Rotavirus vaccines.

Empirical testing indicates potential for cross reactivity with C. cuniculus and C. meleagridis with the Cryptosporidium assay.

# Premarket Notification 510(k)

# No Cross reactivity was observed for the organisms tested below.

<table><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Aeromonas jandaei</td><td rowspan=1 colspan=1>Egglerthella lenta</td><td rowspan=1 colspan=1>Proteus penneri</td></tr><tr><td rowspan=1 colspan=1>Aeromonas media</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Aeromonas trota</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Providencia alcalifaciens</td></tr><tr><td rowspan=1 colspan=1>Aeromonas caviae</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Enteropathogenic Escherichia coli</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwoffii</td><td rowspan=1 colspan=1>Escherichia coli Non pathogenic</td><td rowspan=1 colspan=1>Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Escherichia coli Non pathogenic</td><td rowspan=1 colspan=1>Saccharomyces boulardii</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Escherichia coli Non pathogenic</td><td rowspan=1 colspan=1>Serratia liquefaciens</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Bacteroidesthetaiotaomicron</td><td rowspan=1 colspan=1>Escherichia vulneris</td><td rowspan=1 colspan=1>Shewanella algae</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium breve</td><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter fetus</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Campylobacterhyointestinalis</td><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>Grimontia hollisae (formerly vibrio)</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensis</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Cedecea davisae</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Citrobacteramalonaticus</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Trabulsiella guamensis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile non-toxigenic</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>Vibrio alginolyticusa</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile non-toxigenic</td><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>Vibrio fluvialis</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile non-toxigenic</td><td rowspan=1 colspan=1>Leminorella grimontii</td><td rowspan=1 colspan=1>Vibrio mimicusa</td></tr><tr><td rowspan=1 colspan=1>Clostridium histolyticum</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Yersinia bercovierib</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Yersinia frederikseniib</td></tr><tr><td rowspan=1 colspan=1>Clostridium septicum</td><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Yersinia intermediab</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellii</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Yersinia mollaretiib</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>Porphyromonas asaccharolytica</td><td rowspan=1 colspan=1>Yersinia pseudotuberculosis</td></tr><tr><td rowspan=1 colspan=1>Clostridium tetani</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>Yersinia rohdeib</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>canmaGellowb</td></tr></table>

a – Detected as Vibrio spp at high titers, see full submission for details. b – Detected as Yersinia enterocolitica at high titers, see full submission for details.

# Premarket Notification 510(k)

No Cross reactivity was observed for the viruses and parasites tested below.   

<table><tr><td rowspan=1 colspan=2>Viruses</td><td rowspan=1 colspan=1>Parasites</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3</td><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>Cryptosporidium cuniculus (DNA)b</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 4</td><td rowspan=1 colspan=1>Enterovirusa</td><td rowspan=1 colspan=1>Cryptosporidium felis (DNA)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7a</td><td rowspan=1 colspan=1>Echovirus 11a</td><td rowspan=1 colspan=1>Cryptosporidium meleagridis b</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 8</td><td rowspan=1 colspan=1>HSV Type 2</td><td rowspan=1 colspan=1>Cryptosporidium meleagridis (DNA)b</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 14</td><td rowspan=1 colspan=1>Norovirus GIV</td><td rowspan=1 colspan=1>Cryptosporidium muris</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37</td><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>Cryptosporidium ubiquitum (DNA)</td></tr><tr><td rowspan=1 colspan=1>Astrovirus type 1a</td><td rowspan=1 colspan=1>Sapovirus Gla</td><td rowspan=1 colspan=1>Encephalitozoon cuniculi</td></tr><tr><td rowspan=1 colspan=1>Astrovirus type 4a</td><td rowspan=1 colspan=1>Sapovirus GIVa</td><td rowspan=1 colspan=1>Encephalitozoon hellem</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Sapovirus GVa</td><td rowspan=1 colspan=1>Encephalitozoon intestinalis</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Parasites</td><td rowspan=1 colspan=1>Giardia muris</td></tr><tr><td rowspan=1 colspan=1>Coxsackie virus A16a</td><td rowspan=1 colspan=1>Blastocystis hominisa</td><td rowspan=1 colspan=1>Pentatrichomonas hominis</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (CMV)</td><td rowspan=1 colspan=1>Cryptosporidium canis (DNA)</td><td rowspan=1 colspan=1></td></tr></table>

a – tested at CDC b – Detected as Cryptosporidium spp.

# No Cross reactivity was observed for BioCode GPP targets.

<table><tr><td rowspan=1 colspan=3>BioCode GPP Targets</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Enterotoxigenic E. coli O78:H11H10407 (ETEC)</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni spp. jejuni</td><td rowspan=1 colspan=1>Shiga toxin producing E. coli (STEC)</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (toxinotype 0)</td><td rowspan=1 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficileoxinotype Il;Nap1)</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>Entamoeba histolytica HB-301:NIH</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli O92:H33(EAEC)</td><td rowspan=1 colspan=1>Salmonella enterica ssp. enterica</td><td rowspan=1 colspan=1>Giardia intestinalis (aka G. Iamblia)</td></tr><tr><td rowspan=1 colspan=1>Enteroinvasive E. coli O29:NM (EIEC)</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Adenovirus 40 (dugan)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rotavirus A</td></tr></table>

No Cross reactivity was predicted by in silico analysis.   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Parasites</td></tr><tr><td rowspan=1 colspan=1>Anaerococcus tetradius</td><td rowspan=1 colspan=1>Eubacterium cylindroides</td><td rowspan=1 colspan=1>Ancylostoma duodenale</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td><td rowspan=1 colspan=1>Eubacterium rectale</td><td rowspan=1 colspan=1>Ascaris lumbricoides</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium longum</td><td rowspan=1 colspan=1>Megamonas hypermegale</td><td rowspan=1 colspan=1>Balantidium coli</td></tr><tr><td rowspan=1 colspan=1>Campylobacter concisus</td><td rowspan=1 colspan=1>Methanobrevibacter smithii</td><td rowspan=1 colspan=1>Chilomastix mesnili</td></tr><tr><td rowspan=1 colspan=1>Campylobacter curvus</td><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Cryptosporidium bovis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter gracilis</td><td rowspan=1 colspan=1>Ruminococcus bromii</td><td rowspan=1 colspan=1>Cryptosporidium canis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter helveticus</td><td rowspan=1 colspan=1>Ruminococcusflavefaciens</td><td rowspan=1 colspan=1>Cryptosporidium cuniculus </td></tr><tr><td rowspan=1 colspan=1>Campylobacter hominis</td><td rowspan=1 colspan=1>Ruminococcus obeum</td><td rowspan=1 colspan=1>Cryptosporidium felis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>Selenomonas ruminantium</td><td rowspan=1 colspan=1>Cryptosporidium fetus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter mucosalis</td><td rowspan=1 colspan=1>Vibrio cincinnatiensis</td><td rowspan=1 colspan=1>Cryptosporidium meleagridisb</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Campylobacter rectus</td><td rowspan=1 colspan=1>Vibrio furnissii</td><td rowspan=1 colspan=1>Cryptosporidium muris</td></tr><tr><td rowspan=1 colspan=1>Campylobactershowae</td><td rowspan=1 colspan=1>Vibrio metschnikovii</td><td rowspan=1 colspan=1>Cryptosporidium ryanae</td></tr><tr><td rowspan=1 colspan=1>Campylobactersputorum</td><td rowspan=1 colspan=1>Yersinia kristensenii</td><td rowspan=1 colspan=1>Cryptosporidium xiaoi</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Dientamoeba fragilis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter ureolyticus</td><td rowspan=1 colspan=1>Norovirus GIV</td><td rowspan=1 colspan=1>Endolimax nana</td></tr><tr><td rowspan=1 colspan=1>Clostridium acetobutylicum</td><td rowspan=1 colspan=1>Rotavirus B</td><td rowspan=1 colspan=1>Entamoeba coli</td></tr><tr><td rowspan=1 colspan=1>Clostridium methylpentosum</td><td rowspan=1 colspan=1>Rotavirus Ca</td><td rowspan=1 colspan=1>Entamoeba dispar</td></tr><tr><td rowspan=1 colspan=1>Clostridium novyi</td><td rowspan=1 colspan=1>Rotavirus D</td><td rowspan=1 colspan=1>Entamoeba hartmanni</td></tr><tr><td rowspan=1 colspan=1>Clostridium ramosum</td><td rowspan=1 colspan=1>Rotavirus E</td><td rowspan=1 colspan=1>Entamoeba moshkovskii</td></tr><tr><td rowspan=1 colspan=1>Collinsella aerofaciens</td><td rowspan=1 colspan=1>Rotavirus F</td><td rowspan=1 colspan=1>Entamoeba polecki</td></tr><tr><td rowspan=1 colspan=1>Desulfovibrio piger</td><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1></td></tr></table>

a – Cross-reactivity predicted with Porcine Rotavirus C strains only, no cross-reactivity with human Rotavirus C. b - C. Cuniculus and C. meleagridis cross reactivity for Cryptosporidium spp assay observed in lab testing was not predicted by in silico analysis (Primer Blast).

# COMPETITIVE INHIBITION

A study was performed to evaluate the potential for inhibition in samples with mixed infections. Prescreened negative stool was spiked with one target at high concentration $( \ge 1 0 ^ { 6 } \mathsf { C F U / m L }$ for bacteria and ${ \ge } 1 0 ^ { 5 }$ units/mL for viruses or parasites) and two targets at medium concentration $( \leq 3 \times \mathsf { L o D } )$ . Common co-infections were determined by reviewing results of previous GI Panel clinical trials from 510(k) summaries, publications/posters and internal clinical sample testing. Each sample was extracted in triplicate on the easyMag and each extraction tested in singlet with the Gastrointestinal Pathogen Panel on the BioCode MDx 3000 system. No competitive inhibition was observed.

# Competitive inhibition testing results.

<table><tr><td rowspan=1 colspan=1>PanelDesignation</td><td rowspan=1 colspan=1>Viral/Bacteria Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Screening Titer</td><td rowspan=1 colspan=1>TargetProbe</td><td rowspan=1 colspan=1>AverageMFI</td><td rowspan=1 colspan=1>ScreeningResult(n of 3)</td></tr><tr><td rowspan=3 colspan=1>Cl-1</td><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Zepto 801619</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106CFU/mL</td><td rowspan=1 colspan=1>tcdB</td><td rowspan=1 colspan=1>23969</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.44X10³TCID5o/mL</td><td rowspan=1 colspan=1>Rota</td><td rowspan=1 colspan=1>33899</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.02x10³CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>11585</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CI-2</td><td rowspan=1 colspan=1>092:H33 Escherichiacoli (EAEC)</td><td rowspan=1 colspan=1>STEC JM221TW04440</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106 CFU/mL</td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>26578</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.02X10³ CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>10680</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Zepto 801619</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>5.7x10² CFU/mL</td><td rowspan=1 colspan=1>tcdB</td><td rowspan=1 colspan=1>8775</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>CI-3</td><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>27671</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>Zepto 801619</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>5.7x10² CFU/mL</td><td rowspan=1 colspan=1>tcdB</td><td rowspan=1 colspan=1>8876</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>PanelDesignation</td><td rowspan=1 colspan=1>Viral/Bacteria Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Screening Titer</td><td rowspan=1 colspan=1>TargetProbe</td><td rowspan=1 colspan=1>AverageMFI</td><td rowspan=1 colspan=1>ScreeningResult(n of 3)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.44X10³TCID5o/mL</td><td rowspan=1 colspan=1>Rota</td><td rowspan=1 colspan=1>31617</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Cl-4</td><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3x106CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>26059</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>092:H33 Escherichiacoli (EAEC)</td><td rowspan=1 colspan=1>STEC JM221TW04440</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>4.08X10³ CFU/mL</td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>22175</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp .jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>2.1X10³ CFU/mL</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>32286</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Cl-5</td><td rowspan=1 colspan=1>Campylobacter jejunisubsp. Doylei</td><td rowspan=1 colspan=1>ATCC 49349</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106CFU/mL</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>24738</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.02X10³ CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>7382</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>092:H33 Escherichiacoli (EAEC)</td><td rowspan=1 colspan=1>STEC JM221TW04440</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>4.08X10³ CFU/mL</td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>22488</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CI-6</td><td rowspan=1 colspan=1>092:H33 Escherichiacoli (EAEC)</td><td rowspan=1 colspan=1>STEC JM221TW04440</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3x106CFU/mL</td><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>24968</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisub sp .jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>2.1X10³ CFU/mL</td><td rowspan=1 colspan=1>Campy</td><td rowspan=1 colspan=1>33202</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliE2348/69 (EPEC)</td><td rowspan=1 colspan=1>STEC TW06375</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.02X10³ CFU/mL</td><td rowspan=1 colspan=1>EPEC</td><td rowspan=1 colspan=1>10342</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Cl-7</td><td rowspan=1 colspan=1>Shiga-toxin producingE. coli (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106CFU/mL</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>34913</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>waterborne P101</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>5.42x10³ cysts/mL</td><td rowspan=1 colspan=1>G.lam</td><td rowspan=1 colspan=1>37282</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>1.31X10³ CFU/mL</td><td rowspan=1 colspan=1>Shig</td><td rowspan=1 colspan=1>10518</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CI-8</td><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>waterborne P101</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x105 cysts/mL</td><td rowspan=1 colspan=1>G.lam</td><td rowspan=1 colspan=1>12832</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>1.31X10³ CFU/mL</td><td rowspan=1 colspan=1>Shig</td><td rowspan=1 colspan=1>12530</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shiga-toxin producingE. coli (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.5X10³CFU/mL</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>29785</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Cl-9</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3.0x106CFU/mL</td><td rowspan=1 colspan=1>Shig</td><td rowspan=1 colspan=1>12483</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Shiga-toxin producingE. coli (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>7.5X10³CFU/mL</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>31890</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>waterborne P101</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>5.42X10³ cysts/mL</td><td rowspan=1 colspan=1>G.lam</td><td rowspan=1 colspan=1>31227</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# CROSS CONTAMINATION/SAMPLE CARRYOVER

A study was performed to demonstrate the absence of carryover or cross-contamination when using the BioCode Gastrointestinal Pathogen Panel in conjunction with the easyMag. High-positive samples (EAEC at $1 0 ^ { 6 } \mathsf { C F U / m L } )$ ) were tested alternating with no-template control samples in a

# Premarket Notification 510(k)

“checkerboard” pattern. Samples were extracted checkerboard and assayed in singlet. The study consisted of five complete runs from extraction to BioCode MDx 3000 results on one instrument. No cross contamination was observed.

# FRESH VS. FROZEN STABILITY

Since many of the unpreserved samples in three of the four sites using 60 spiked unpreserved stool specimens to assess fresh vs frozen specimen stability. Specimens were contrived and tested at time 0 and after $\geq 2$ freeze thaw cycles. Samples were tested at Low positive (1X-1.5X LoD) spiked into 7 negative stool samples. Cary-Blair specimens were not subjected to frozen storage during the clinical study, therefore there were not tested. Frozen samples displayed stability throughout the length of the study.

Fresh Vs. Frozen results.   

<table><tr><td rowspan=1 colspan=1>Acceptance Criteria</td><td rowspan=1 colspan=1>Results</td></tr><tr><td rowspan=1 colspan=1>95% replicates for all positive targets in the contrived sample should beValid and Detected (≥60/63)</td><td rowspan=1 colspan=1>63/63</td></tr><tr><td rowspan=1 colspan=1>95% of negative targets should be Valid and not detected (≥60/63)</td><td rowspan=1 colspan=1>633/63</td></tr></table>

a – Sample Target was detected, but was invalid for RNA IC.

Analysis of results from Fresh Vs Frozen study.   

<table><tr><td rowspan=1 colspan=3>Analysis</td><td rowspan=1 colspan=1>Delta Fresh -Frozen</td><td rowspan=1 colspan=1>PercentDecrease</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp . jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>FF1</td><td rowspan=1 colspan=1>3835</td><td rowspan=1 colspan=1>14%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli 10C-3114 (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>FF2</td><td rowspan=1 colspan=1>3019</td><td rowspan=1 colspan=1>12%</td></tr><tr><td rowspan=1 colspan=1>Human rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>FF3</td><td rowspan=1 colspan=1>4437</td><td rowspan=1 colspan=1>13%</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td><td rowspan=1 colspan=1>FF4</td><td rowspan=1 colspan=1>1260</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>078:H11 Escherichia coli strain H10407 (ETEC) STa</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>FF5</td><td rowspan=1 colspan=1>2503</td><td rowspan=1 colspan=1>8%</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>FF6</td><td rowspan=1 colspan=1>2365</td><td rowspan=1 colspan=1>20%</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. Enterica</td><td rowspan=1 colspan=1>ATCC 14028</td><td rowspan=1 colspan=1>FF7</td><td rowspan=1 colspan=1>-148</td><td rowspan=1 colspan=1>-2%</td></tr><tr><td rowspan=1 colspan=1>Human adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>FF8</td><td rowspan=1 colspan=1>610</td><td rowspan=1 colspan=1>3%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>waterborne P102M</td><td rowspan=1 colspan=1>FF9</td><td rowspan=1 colspan=1>3134</td><td rowspan=1 colspan=1>14%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>FF10</td><td rowspan=1 colspan=1>1719</td><td rowspan=1 colspan=1>6%</td></tr></table>

# SPECIMEN STABILITY

A study was performed to assess the specimen stability limitations for the optimal performance of the BioCode Gastrointestinal Pathogen Panel on the BioCode MDx 3000. This study employed the use of spiked specimens to assess the following storage conditions:

Samples in Cary-Blair - 0, 2, 4 and 6 days at room temperature $( 2 0 - 2 5 ^ { \circ } \mathsf { C } )$ ; 2, 4 and 6 days at $2 { - } 8 ^ { \circ } \mathsf C$   
Unpreserved Stool - 0, 2, 4 and 6 days at ${ 2 - 8 ^ { \circ } C }$   
Fresh vs. Frozen (- $\mathsf { - 9 0 ^ { \circ } C }$ to $- 6 0 ^ { \circ } \mathsf { C } )$ ( $_ { 2 \times }$ freeze thaws) - 30, 60 and 90 days at $\mathsf { - 9 0 ^ { \circ } C }$ to $\scriptscriptstyle - 6 0 ^ { \circ } \mathsf { C }$   
S.T.A.R. buffer after pretreatment (SK38 Tubes prior to extraction) - 24 hours at $2 { - } 8 ^ { \circ } \mathsf C$   
Fresh vs. Frozen (- $\mathfrak { g o c }$ to $- 6 0 ^ { \circ } \mathsf { C } )$ (2x freeze thaws) - 30, 60 and 90 days at $\mathsf { - 9 0 ^ { \circ } C }$ to $\scriptscriptstyle - 6 0 ^ { \circ } \mathsf { C }$   
Extracted Nucleic Acid - 24 hours at $2 { - } 8 ^ { \circ } \mathsf C$   
Fresh vs. Frozen (- $\mathfrak { g o c }$ to $- 6 0 ^ { \circ } \mathsf { C } )$ ( $_ { 2 \times }$ freeze thaws) - 30, 60 and 90 days at $\mathsf { - 9 0 ^ { \circ } C }$ to $\frown 6 0 ^ { \circ } \mathsf { C }$

# Premarket Notification 510(k)

One parasite, one virus, one-gram positive bacterium, and one-gram negative bacterium was used for testing. Spiked and clinical samples were tested at $\sim 3 \mathsf { X }$ LoD with two organisms in each sample. Concentrated organism stocks were serially diluted in Cary-Blair and combined with prescreened negative stool and each condition was assayed with 3 replicates at each time point. Samples displayed stability under the various conditions throughout the length of the study.

![](images/e483f0dbafe1d88c5028f67574040dceaaa48b273d0fbf03acbada1c6995745e.jpg)

# Premarket Notification 510(k)

![](images/b12af28f437c4205388791a0d9bae66db55e978c40d4c8c48cbe7e050ea2c04d.jpg)  
Figure. Graphic display of MFI results at indicated storage temperatures and timepoints.   
Figure. Graphic display of MFI results at indicated storage temperatures and timepoints.

# CONCLUSION

The intended use and fundamental scientific technology of the BioCode Gastrointestinal Pathogen Panel Assay is substantially equivalent to the predicate device. Clinical and non-clinical studies have established that the BioCode Gastrointestinal Pathogen Panel is substantially equivalent to the predicate device.