# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM

A. 510(k) Number: K180559   
B. Purpose for Submission: Clearance of New Device.   
C. Measurand: Target DNA Sequences from conserved regions of Herpes Simplex Virus Type 1 (HSV- 1) and Herpes Simplex Virus Type 2 (HSV- 2)   
D. Type of Test: Qualitative Real-Time PCR Assay   
E. Applicant: ELITechGroup Inc. Molecular Diagnostics   
F. Proprietary and Established Names: HSV 1&2 ELITe MBG Assay   
G. Regulatory Information:

1. Regulation section:   
21CFR 866.3309

2. Classification: Class II

3. Product code: PGI

4. Panel: 83 - Microbiology

# H. Intended Use:

1. Intended use(s):

The HSV 1&2 ELITe $\mathbf { M G B } ^ { \circledast }$ Assay is a real-time polymerase chain reaction (PCR) based qualitative in vitro diagnostic test for the direct detection and differentiation of Herpes Simplex Virus 1 and 2 (HSV-1 and HSV-2) DNA in cutaneous or mucocutaneous lesion swab specimens from patients with signs and symptoms of HSV-1 or HSV-2 infection. This test is an aid in the differential diagnosis of HSV-1 and HSV-2 infections.

The HSV 1&2 ELITe MGB Assay is not FDA cleared for use with cerebrospinal fluid (CSF) specimens. The assay is not intended to be used for prenatal screening or for screening blood or blood products.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: The HSV 1&2 ELITe MGB Assay is indicated for use on the ELITe InGenius $\textsuperscript { \textregistered }$ System.

# I. Device Description:

The HSV 1&2 ELITe MGB Assay is a multiplexed qualitative in vitro diagnostic Real-Time PCR Assay that uses unique primer sets and single uniquely labeled probes to amplify and detect:

• The Herpes Simplex Virus (HSV) genotype 1; HSV-1 glycoprotein D encoding gene, The HSV-2 glycoprotein G encoding gene, and   
An Internal Control.

Intended for the direct detection of HSV DNA in symptomatic male and female patients using DNA purified from swab specimens collected from cutaneous or mucocutaneous lesions from individuals with herpetic lesions.

The HSV 1&2 ELITe MGB Assay is used on the ELITe InGenius system. The ELITe InGenius system is a bench top instrument integrating all required hardware, reagents and software components to perform nucleic acid sample preparation and real-time PCR operations.

The ELITe InGenius system can process 1 to 12 samples in 12 parallel tracks, and samples may be loaded in primary tubes or in secondary tubes provided. The system utilizes a universal, cassette-based process for sample extraction and allows for multiple and independent PCRs to be performed from a single nucleic acid eluate. The unused eluates can be saved for future retesting or archiving.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ARIES $\textsuperscript { \textregistered }$ HSV 1&2 Assay

2. Predicate 510(k) number(s): K151906

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Indications For Use</td><td colspan="1" rowspan="1">The HSV 1&amp;2 ELITe MGB®Assay is a real-timepolymerase chain reaction(PCR) based qualitative invitro diagnostic test for thedirect detection anddifferentiation of HerpesSimplex Virus 1 and 2 (HSV-1 and HSV-2) DNA incutaneous or mucocutaneouslesion swab specimens frompatients with signs andsymptoms of HSV-1 or HSV-2 infection. This test is an aidin the differential diagnosis ofHSV-1 and HSV-2 infections.The HSV 1&amp;2 ELITe MGBAssay is not FDA cleared foruse with cerebrospinal fluid(CSF) specimens. The assayis not intended to be used forprenatal screening or forscreening blood or bloodproducts.</td><td colspan="1" rowspan="1">The ARIES® HSV 1&amp;2Assay is a real-timepolymerase chain reaction(PCR) based qualitative invitro diagnostic test for thedirect detection anddifferentiation of HerpesSimplex Virus 1 and 2 (HSV1 and HSV 2) DNA incutaneous or mucocutaneouslesion specimens fromsymptomatic patients. Thetest is indicated for use as anaid in diagnosis of HSVinfection in symptomaticpatients. The ARIES® HSV1&amp;2 Assay is indicated foruse on the ARIES® System.WARNING: The ARIES®HSV 1&amp;2 Assay is not FDAcleared for use withcerebrospinal fluid (CSF).The assay is not intended tobe used for prenatalscreening.</td></tr><tr><td colspan="1" rowspan="1">Extraction Technology</td><td colspan="1" rowspan="1">Same as predicate</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Same as predicate</td><td colspan="1" rowspan="1">Qualitative Real-time PCR</td></tr><tr><td colspan="1" rowspan="1">Samples Type</td><td colspan="1" rowspan="1">Same as predicate</td><td colspan="1" rowspan="1">Male and female cutaneousand mucocutaneous lesionswab samples</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">DNA sequences from HSV-1glycoprotein D gene andHSV-2 glycoprotein G gene.</td><td colspan="1" rowspan="1">DNA sequences from HerpesSimplex Virus type 1 (HSV-1) and Herpes Simplex Virustype 2 (HSV-2)- differenttarget</td></tr><tr><td colspan="1" rowspan="1">Detection Technology</td><td colspan="1" rowspan="1">Multiplex assay with pairedreporter and quencherfluorescence labeled probesand different reporter dyes foreach target. Measures increasein assay fluorescence witheach PCR cycle.</td><td colspan="1" rowspan="1">Pairs fluorescent-labeledprimers with quencherlabeled nucleotides.Measures decrease in assayfluorescence with each PCRcycle.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Evaluating Substantial Equivalence in Premarket Notifications $5 1 0 ( \mathrm { k } )$   
2. Statistical Guidance on Reporting Results from studies evaluating diagnostic tests   
3. Guidance Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (DRAFT 10-2-14)   
4. Guidance for Clinical Investigators, Sponsors, and IRBs - Adverse Event Reporting to IRBs - Improving Human Subject Protection   
5. Guidance for Evaluation and Reporting of Age-, Race-, and Ethnicity-Specific Data in Medical Device Clinical Studies (1500626) (09-12-17)   
6. Guidance for Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable   
7. Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices   
8. Guidance for Refuse to Accept Policy for $5 1 0 ( \mathrm { k } ) \mathrm { s }$   
9. EP05-A3 Vol. 34 No. 13 - Evaluation of Precision Performance of Quantitative   
10. Measurement Methods; Approved Guideline-Third Edition   
11. EP12-A2 Vol. 28 No. 3 - User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition   
12. EP17-A2 Vol. 32 No. 8 - Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - Second Edition   
13. EP24-A2 Vol. 31 No. 23 - Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline - Second Edition   
14. EP25-A Vol. 29 No. 20 - Evaluation of Stability of in Vitro Diagnostic Reagents; Approved Guideline   
15. M29-A4 Vol. 34 No. 8 - Protection of Laboratory Workers from Occupationally Acquired Infections; Approved Guideline - 4th Edition   
16. MM03 - Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline, 3rd Edition   
17. MM06-A2 Vol. 30 No. 22 - Quantitative Molecular Methods for Infectious Diseases; Approved Guideline Second Edition   
18. MM09-A2 Vol. 34 No.4 - Nucleic Acid Sequencing Methods in Diagnostic Laboratory Medicine; Approved Guideline   
19. MM13-A Vol. 25 No. 31 - Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline   
20. MM17 Ed 2 Vol. 38 No. 9 - Verification and Validation of Multiplex Nucleic Acid Assays (NATS); Approved Guideline

# L. Test Principle:

The HSV 1&2 ELITe MGB Assay system is comprised of three major processes:

(1) automated preparation of unprocessed sample to extract nucleic acids from primary swab specimens using the ELITe InGenius SP 200 Extraction Cartridge,   
(2) PCR amplification of target DNA sequences using HSV-1 and HSV-2 specific primers,   
(3) real-time detection of fluorescent-labeled HSV-1 and HSV-2 specific oligonucleotide detection probes.

An Internal Control (IC), containing unrelated randomized DNA sequence, is added to all samples prior to extraction and monitors the integrity of the reagents, equipment function, and the presence of inhibitors in the samples. A positive signal in the Internal Control channel in the absence of HSV DNA indicates that the PCR has not been inhibited.

The amplification reagents, Positive Control and Internal Control are packaged as part of the HSV 1&2 ELITe MGB Assay.

The following table shows the result interpretation:

Summary of the Run Validity Criteria and Result Interpretation Algorithm:   

<table><tr><td rowspan=1 colspan=1>HSV-1 CT valueAP593,Channel 4</td><td rowspan=1 colspan=1>HSV-2 Ct valueFAM,Channel 1</td><td rowspan=1 colspan=1>IC Cr valueAP525,Channel 2</td><td rowspan=1 colspan=1>Status</td><td rowspan=1 colspan=1>HSV-1Result</td><td rowspan=1 colspan=1>HSV-2Result</td></tr><tr><td rowspan=2 colspan=1>UndeterminedCT ≥ 45.0</td><td rowspan=2 colspan=1>UndeterminedCT &gt; 45.0</td><td rowspan=1 colspan=1>CT≤ 32.0</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Undeterminedor CT &gt; 32.0</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=2 colspan=1>DeterminedCT≤ 45.0</td><td rowspan=1 colspan=1>UndeterminedCT &gt; 45.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=2 colspan=1>Valid</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>DeterminedCT≤ 45.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>UndeterminedCT &gt; 45.0</td><td rowspan=1 colspan=1>DeterminedCT≤ 45.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr></table>

The Sample run is valid when the conditions reported in the table below are met.

<table><tr><td rowspan=1 colspan=1>1) HSV 1&amp;2 ELITe MGB Positive Control</td><td rowspan=1 colspan=1>Status</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2 ELITe MGB Positive Control</td><td rowspan=1 colspan=1>APPROVED</td></tr><tr><td rowspan=1 colspan=1>2) HSV 1&amp;2 ELITe MGB Negative Control</td><td rowspan=1 colspan=1>Status</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2 ELITe MGB Negative Control</td><td rowspan=1 colspan=1>APPROVED</td></tr></table>

ELITe InGenius software reports as target DNA being “Detected” or “Not Detected”. For a valid run, the specimen results are interpreted as follows.

ELITe InGenius Software Output and Interpretation   

<table><tr><td rowspan=1 colspan=2>Results of Sample Run</td><td rowspan=2 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>HSV1 Result</td><td rowspan=1 colspan=1>HSV2 Result</td></tr><tr><td rowspan=1 colspan=1>HSV-1 DNANot Detected</td><td rowspan=1 colspan=1>HSV-2 DNANot Detected</td><td rowspan=1 colspan=1>HSV-1 and HSV-2 negative</td></tr><tr><td rowspan=1 colspan=1>HSV-1 DNANot Detected</td><td rowspan=1 colspan=1>HSV-2DNA Detected</td><td rowspan=1 colspan=1>HSV-1 negative and HSV-2 positive</td></tr><tr><td rowspan=1 colspan=1>HSV-1 DNADetected</td><td rowspan=1 colspan=1>HSV-2 DNANot Detected</td><td rowspan=1 colspan=1>HSV-1 positive and HSV-2 negative</td></tr><tr><td rowspan=1 colspan=1>HSV-1 DNADetected</td><td rowspan=1 colspan=1>HSV-2 DNADetected</td><td rowspan=1 colspan=1>HSV-1 positive and HSV-2 positive</td></tr><tr><td rowspan=1 colspan=2>Invalid - Retest Sample</td><td rowspan=1 colspan=1>Not valid assay result due to Internal Controlfailure (Incorrect extraction or presence ofinhibitor). Specimen Should be retested</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the HSV 1&2 ELITe MGB Assay was evaluated in a multi-site investigation using contrived clinical samples. HSV test panels were prepared by spiking HSV-1 (Maclntyre strain) or HSV-2 (MS strain) virus into Universal transport media (UTM) at the concentrations of ${ < } 1 \times \mathrm { L o D }$ , $1 \times \mathrm { L o D }$ and $3 \times \mathrm { L o D }$ . HSV-1 and HSV-2 negative panel members were included as panel member controls. The reproducibility panel composition is shown in the table below:

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">Description of Contents</td><td colspan="1" rowspan="1">Viral Load</td><td colspan="1" rowspan="1">Expected PositivityRate</td></tr><tr><td colspan="1" rowspan="1">M1</td><td colspan="1" rowspan="1">HSV-1 C50 (High Negative) in UTM</td><td colspan="1" rowspan="1">&lt;1× LOD</td><td colspan="1" rowspan="1">20-80% positive</td></tr><tr><td colspan="1" rowspan="1">M2</td><td colspan="1" rowspan="1">HSV-1 C95 (Low Positive) in UTM</td><td colspan="1" rowspan="1">1× LOD</td><td colspan="1" rowspan="1">≥95% positive</td></tr><tr><td colspan="1" rowspan="1">M3</td><td colspan="1" rowspan="1">HSV-1 C100 (Moderate Positive) inUTM</td><td colspan="1" rowspan="1">2-3 × LOD</td><td colspan="1" rowspan="1">100% positive</td></tr><tr><td colspan="1" rowspan="1">M4</td><td colspan="1" rowspan="1">HSV-2 C50 (High Negative) in UTM</td><td colspan="1" rowspan="1">&lt;1× LOD</td><td colspan="1" rowspan="1">20-80% positive</td></tr><tr><td colspan="1" rowspan="1">M5</td><td colspan="1" rowspan="1">HSV-2 C95 (Low Positive) in UTM</td><td colspan="1" rowspan="1">1× LOD</td><td colspan="1" rowspan="1">≥95% positive</td></tr><tr><td colspan="1" rowspan="1">M6</td><td colspan="1" rowspan="1">HSV-2 C100 (Moderate Positive) inUTM</td><td colspan="1" rowspan="1">2-3 × LOD</td><td colspan="1" rowspan="1">100% positive</td></tr><tr><td colspan="1" rowspan="1">M7</td><td colspan="1" rowspan="1">HSV Negative in UTM</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100% negative</td></tr></table>

Panels were tested at 3 sites by 2 operators per site with 1 run per operator per day, for 10 nonconsecutive days using a single lot of HSV 1&2 ELITe MGB Assay. Testing was performed on a minimum of 90 (30 per site) replicates per panel member. Lot-to-Lot variability was assessed only at EGI MDx using three lots of HSV 1&2 ELITe MGB Assay. Controls were run daily and were included in the first run of the day.

The results are presented in the following table.

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Sample</td><td colspan="3" rowspan="1">Site - 1</td><td colspan="3" rowspan="1">Site - 2</td><td colspan="3" rowspan="1">Site - 3</td><td colspan="1" rowspan="2">% Agree-mentwithExpectedResults</td><td colspan="1" rowspan="2">95% CI</td></tr><tr><td colspan="1" rowspan="1">% Agree-ment withExpectedResults</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">%Agree-mment withExpectedResults</td><td colspan="1" rowspan="1">Avg:Ct</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">%Agree-mment withExpectedResults</td><td colspan="1" rowspan="1">Avg:Ct</td><td colspan="1" rowspan="1">Total%CV</td></tr><tr><td colspan="1" rowspan="7">HSV-1Result</td><td colspan="1" rowspan="1">HSV-1Low Pos</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">1.70%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">2.10%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">2.00%</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-1Mod Pos</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">1.30%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">5.20%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">1.50%</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-2Low Pos</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(29/29)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(89/89)</td><td colspan="1" rowspan="1">95.6 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-2Mod Pos</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSVNeg</td><td colspan="1" rowspan="1">100.0%(60/60)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(38/38)</td><td colspan="1" rowspan="1">41.4</td><td colspan="1" rowspan="1">2.50%</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(138/138)</td><td colspan="1" rowspan="1">97.3 to100.0%</td></tr><tr><td colspan="1" rowspan="1">PosControl</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">27.5</td><td colspan="1" rowspan="1">1.30%</td><td colspan="1" rowspan="1">100.0%(5/5)</td><td colspan="1" rowspan="1">27.5</td><td colspan="1" rowspan="1">1.20%</td><td colspan="1" rowspan="1">100.0%(5/5)</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0.80%</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">91.2 to100.0%</td></tr><tr><td colspan="1" rowspan="1">TotalAgreement</td><td colspan="3" rowspan="1">100.0%(210/210)</td><td colspan="3" rowspan="1">100.0%(162/162)</td><td colspan="3" rowspan="1">100.0%(165/165)</td><td colspan="1" rowspan="1">100.0%(537/537)</td><td colspan="1" rowspan="1">98.2 to100.0%</td></tr><tr><td colspan="13" rowspan="1">a) Expected Results of HSV-2 Low Positive, HSV-2 Moderate Positive and HSV Negative samplesare "Negative" for HSV-1.</td></tr><tr><td colspan="1" rowspan="3">Target</td><td colspan="1" rowspan="3">Sample</td><td colspan="3" rowspan="1">Site - 1</td><td colspan="3" rowspan="1">Site - 2</td><td colspan="3" rowspan="1">Site - 3</td><td colspan="1" rowspan="3">%Agree-ment withExpectedResults</td><td colspan="1" rowspan="3">95%CI</td></tr><tr><td colspan="1" rowspan="2">%Agree-ment withExpectedResults</td><td colspan="1" rowspan="2">Avg.Ct</td><td colspan="1" rowspan="2">Total%CV</td><td colspan="1" rowspan="2">%Agree-ment withExpectedResults</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="2">Total%CV</td><td colspan="1" rowspan="2">%Agree-ment withExpectedResults</td><td colspan="1" rowspan="2">Avg•Ct</td><td colspan="1" rowspan="2">Total%CV</td></tr><tr><td colspan="1" rowspan="1">Avg•Ct</td></tr><tr><td colspan="1" rowspan="7">HSV-2Result</td><td colspan="1" rowspan="1">HSV-1Low Pos</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-1Mod Pos</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(30/30)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-2Low Pos</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">3.10%</td><td colspan="1" rowspan="1">100.0%(29/29)</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">2.30%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">1.90%</td><td colspan="1" rowspan="1">100.0%(89/89)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSV-2Mod Pos</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">1.30%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">35.95</td><td colspan="1" rowspan="1">1.60%</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">2.30%</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">HSVNeg</td><td colspan="1" rowspan="1">100.0%(60/60)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(38/38)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(40/40)*</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0%(138/138)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">PosControl</td><td colspan="1" rowspan="1">100.0%(30/3)</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">1.30%</td><td colspan="1" rowspan="1">100.0%(5/5)</td><td colspan="1" rowspan="1">27.4</td><td colspan="1" rowspan="1">1.50%</td><td colspan="1" rowspan="1">100.0%(5/5)</td><td colspan="1" rowspan="1">26.8</td><td colspan="1" rowspan="1">1.40%</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">95.9 to100.0%</td></tr><tr><td colspan="1" rowspan="1">TotalAgreement</td><td colspan="3" rowspan="1">100.0%(210/210)</td><td colspan="3" rowspan="1">100.0%(162/162)</td><td colspan="3" rowspan="1">100.0%(165/165)</td><td colspan="1" rowspan="1">100.0%(537/537)</td><td colspan="1" rowspan="1">95.9 to10.0%</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site - 1</td><td rowspan=1 colspan=3>Site - 2</td><td rowspan=1 colspan=3>Site - 3</td><td rowspan=2 colspan=1>% Agree-ment withExpectedResults</td><td rowspan=2 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>% Agree-ment withExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>TotaI%CV</td><td rowspan=1 colspan=1>% Agree-ment withExpectedResults</td><td rowspan=1 colspan=1>Avg•Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>% Agree-ment withExpectedResults</td><td rowspan=1 colspan=1>Avg•Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=7 colspan=1>ICResult</td><td rowspan=1 colspan=1>HSV-1Low Pos</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>3.80%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>2.00%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.70%</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-1Mod Pos</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>2.30%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>2.80%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.90%</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2Low Pos</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=1>100.0%(29/29)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>2.20%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.60%</td><td rowspan=1 colspan=1>100.0%(89/89)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSV-2Mod Pos</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.20%</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.60%</td><td rowspan=1 colspan=1>100.0%(90/90)</td><td rowspan=1 colspan=1>95.9 to100.0%</td></tr><tr><td rowspan=1 colspan=1>HSVNeg</td><td rowspan=1 colspan=1>100.0%(60/60)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.10%</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.90%</td><td rowspan=1 colspan=1>100.0%(38/38)</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.90%</td><td rowspan=1 colspan=1>100.0%(138/138)</td><td rowspan=1 colspan=1>97.3 to100.0%</td></tr><tr><td rowspan=1 colspan=1>PosControl</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>1.40%</td><td rowspan=1 colspan=1>100.0%(5/5)</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>2.10%</td><td rowspan=1 colspan=1>100.0%(5/5)</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.90%</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>91.2 to100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>100.0%(210/210)</td><td rowspan=1 colspan=3>100.0%(164/164)</td><td rowspan=1 colspan=3>100.0%(163/163)</td><td rowspan=1 colspan=1>100.0%(537/537)</td><td rowspan=1 colspan=1>98.2 to100.0%</td></tr></table>

b) Expected Results of HSV-1 Low Positive, HSV-1 Moderate Positive and HSV Negative samples are “Negative” for HSV-2.

$b$ . Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): The stability of the HSV 1&2 ELITe MGB Assay was evaluated in a real-time

and the data support a shelf-life claim of 10 months for the HSV 1&2 ELITe MGB Assay. A reagent freeze thaw stability study was conducted with 3 reagent lots and demonstrated that the HSV 1&2 ELITe MGB Assay is stable when subjected to 8 freeze-thaw cycles.

# d. Detection limit:

The limit of detection (LoD) of the HSV 1&2 ELITe MGB Assay was determined using 4 HSV strains (two for each target). Quantitated viral strains were obtained and serially diluted into HSV negative pooled human oral cheek matrix in UTM. All dilutions were tested, and the LoD was determined using Probit (Logit) Data Analysis software (Analyse-it for Microsoft Excel v4.80.2, Logistic Function model). LoD for each strain represents the lowest viral titer in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ at which a positive result can be detected at $\ge 9 5 \%$ confidence. The LoD for each strain was then verified by testing 20 replicates. The results are presented in the following table.

Limit of Detection Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Isolate/Strain</td><td rowspan=1 colspan=1>CellLine</td><td rowspan=1 colspan=1>Qualitativeresults#detected/Total</td><td rowspan=1 colspan=1>Mean CT ±SDfrom detectedreplicates</td><td rowspan=1 colspan=1>1×LoDTCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>MacIntyrestrain</td><td rowspan=1 colspan=1>Vero</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>37.91 ± 0.69</td><td rowspan=1 colspan=1>59.0TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>Isolate #15(Zeptometrix)</td><td rowspan=1 colspan=1>Vero</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>39.94 ± 0.95</td><td rowspan=1 colspan=1>1.5TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>MS strain</td><td rowspan=1 colspan=1>Vero</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>37.90 ± 0.92</td><td rowspan=1 colspan=1>5.4TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>Isolate #2(Zeptometrix)</td><td rowspan=1 colspan=1>Vero</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>38.67 ± 1.03</td><td rowspan=1 colspan=1>0.3TCID50/mL</td></tr></table>

A swab elution efficiency study was also performed by using the same HSV negative pooled human cheek swab matrix and the HSV-1 MacIntyre strain used to verify the LoB. This study showed $100 \%$ elution efficiency from the Copan regular flocked swab compared with the same volume of material directly spiked into UTM.

# e. Analytical Reactivity:

The analytical reactivity (Inclusivity) of the HSV 1&2 ELITe MGB Assay was tested on 44 well characterized commercially available HSV-1 and HSV-2 isolates. All strains were tested with the HSV 1&2 ELITE MGB Assay as spiked samples in UTM prepared at close to LoD (3xLOD) level. For HSV-1 and HSV-2 viral isolates not detected (negative) at $3 { \times } \mathrm { L o D }$ concentrations, $2 \times$ incremental concentrations were tested, the lowest detectable level was determined, and the final test concentration was reported. All tested HSV-1 and HSV-2 isolates were detected by the HSV 1&2 ELITe MGB Assay at concentrations of 16.2 – 354 TCID50/mL.

<table><tr><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Estimated1×LoD(TCID50/mL)</td><td colspan="1" rowspan="1">×LoDTested</td><td colspan="1" rowspan="1">Final TestConc.(TCID50/mL)</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HSV-1 MacIntyre Strain</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">HSV-1 Isolate #1</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">HSV-1 Isolate #2</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">HSV-1 Isolate #3</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">HSV-1 Isolate #4</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HSV-1 Isolate #5</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="2">HSV-1 Isolate #6</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">354</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">HSV-1 Isolate #7</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">HSV-1 Isolate #8</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">HSV-1 Isolate #9</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">HSV-1 Isolate #10</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">HSV-1 Isolate #11</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">HSV-1 Isolate #12</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">HSV-1 Isolate #13</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">HSV-1 Isolate #14</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">HSV-1 Isolate #15</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">HSV-1 Isolate #16</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">HSV-1 Isolate #17</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">HSV-1 Isolate #18</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">HSV-1 Isolate #19</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">21</td><td colspan="1" rowspan="2">HSV-1 Isolate #20</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">354</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">HSV-1 Isolate #21</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">177</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">HSV-2 MS Strain</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">HSV-2 Isolate #1</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">HSV-2 Isolate #2</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">HSV-2 Isolate #3</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">HSV-2 Isolate #4</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">HSV-2 Isolate #5</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">HSV-2 Isolate #6</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">HSV-2 Isolate #7</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">31</td><td colspan="1" rowspan="2">HSV-2 Isolate #8</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">HSV-2 Isolate #9</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">HSV-2 Isolate #10</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">34</td><td colspan="1" rowspan="2">HSV-2 Isolate #11</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">35</td><td colspan="1" rowspan="2">HSV-2 Isolate #12</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">HSV-2 Isolate #13</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Estimated1×LoD(TCID50/mL)</td><td colspan="1" rowspan="1">xLoDTested</td><td colspan="1" rowspan="1">Final TestConc.(TCID50/mL)</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">12×</td><td colspan="1" rowspan="1">64.8</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">24×</td><td colspan="1" rowspan="1">129.6</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">37</td><td colspan="1" rowspan="2">HSV-2 Isolate #14</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">38</td><td colspan="1" rowspan="2">HSV-2 Isolate #15</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">39</td><td colspan="1" rowspan="2">HSV-2 Isolate #16</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">40</td><td colspan="1" rowspan="3">HSV-2 Isolate #17</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">12x</td><td colspan="1" rowspan="1">64.8</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">HSV-2 Isolate #18</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">42</td><td colspan="1" rowspan="3">HSV-2 Isolate #19</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">12×</td><td colspan="1" rowspan="1">64.8</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">43</td><td colspan="1" rowspan="3">HSV-2 Isolate #20</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">6×</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">12×</td><td colspan="1" rowspan="1">64.8</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">HSV-2 Isolate #21</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">3×</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# $f .$ Analytical specificity:

# Cross-Reactivity

Potential cross-reactivity of the HSV 1&2 ELITe MGB Assay was evaluated by testing 49 different microbial species that are either genetically related to HSV1 and HSV2 or cause similar clinical symptoms or may be present in the cutaneous and mucocutaneous sites tested by this device. For each of the 49 potential cross reactants, the sample to be tested was prepared from quantified stock diluted to the required concentration using UTM. The potential cross reactants were tested, the concentrations were evaluated and the results are presented in the following table:

<table><tr><td colspan="1" rowspan="2">No.</td><td colspan="1" rowspan="2">Potential Cross-Reactants</td><td colspan="1" rowspan="2">TestedConcentration</td><td colspan="2" rowspan="1">Qualitative Result(#Detected/#Total)</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">HSV-2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus type 2</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">500 ng/swab</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Human Herpes Virus 6</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Human Herpes Virus 7</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Human papilloma virus 16</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Human papilloma virus 18</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Herpes Simplex Virus 1(HSV-1), isolate 20, ZMC</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Herpes Simplex Virus 2(HSV-2), isolate 20. ZMC</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Mobiluncus curtsii</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Neisseria gonorrhea</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Rubella Virus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Staphylococcus aureusMMSA)</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE)</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1×106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1×106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1×106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">1×106CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Varicella-Zoster Virus(VZV)</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr></table>

# Microbial Interference

The microbial interference was evaluated in the presence of either HSV-1 or HSV-2 spiked at $3 { \times } \mathrm { L o D }$ in UTM and the 49 organisms indicated in the cross reactivity section above. Each microorganism was tested either at $1 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L }$ or higher for bacterial isolates, or at $1 \times 1 0 ^ { 5 }$ T $\mathrm { \ ' C I D 5 0 / m L }$ or higher for viruses. None of the nontarget organisms that are reasonably expected to be found in cutaneous and mucocutaneous swab samples interfered with the detection of HSV-1 or HSV-2 species. The only exception is that amplification of HSV-1 is completely inhibited in the presence of HSV-2 titers of $1 \times 1 0 ^ { 3 }$ or higher. This observed HSV-2 interference is reported as a limitation in the device package insert.

# Competitive Interference

Competitive interference was studied to evaluate the effects of possible clinically relevant co-infection with both HSV-1 and HSV-2 using HSV 1&2 ELITe MGB Assay.

The study assessed whether a high concentration of one virus in the sample could potentially affect the HSV 1&2 ELITe MGB Assay performance for another target present at low levels. A low positive sample was contrived at approximately $3 { \times } \mathrm { L o D }$ for each target (HSV-1 McIntryre strain and HSV-2 MS strain), and a baseline Ct was determined for each sample. Each potential concomitant infecting virus was spiked into the low level sample and assayed in triplicate.

Competitive interference of HSV-1 with HSV-2 was observed at HSV-2 titers of $1 \times 1 0 ^ { \bar { 3 } }$ , $1 \times 1 0 ^ { 4 }$ , $1 \times 1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ . This observed interference information is included as a limitation in the device package insert.

No competitive interference of HSV-2 with HSV-1 at the levels tested was observed.   
The results of the testing are shown in the table below.

Competitive Interference of HSV-1 and HSV-2 targets at unequal concentrations   

<table><tr><td rowspan=1 colspan=2>Baseline (Low Level)</td><td rowspan=1 colspan=2>Competitive Interferent(High Concentration)</td><td rowspan=1 colspan=2>Qualitative Results(#Detected/#Total)</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>100000</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>10000</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>100000</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>10000</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>HSV-1McIntyre</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Additionally, in a separate study both strains were tested at similar or equal concentrations of $3 { \times } \mathrm { L o D }$ , $1 \times 1 0 ^ { 3 }$ and $1 \times 1 0 ^ { 5 }$ , and no competitive interference was observed.

Competitive Interference of HSV-1 & HSV-2 targets at equal concentrations   

<table><tr><td rowspan=1 colspan=2>HSV-1 Concentration</td><td rowspan=1 colspan=2>HSV-2 Concentration</td><td rowspan=1 colspan=2>Qualitative Results(#Detected/#Total)</td><td rowspan=1 colspan=2>QuantitativeResults (%CV)</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>1×105</td><td rowspan=1 colspan=1>HSV-2MS</td><td rowspan=1 colspan=1>1×105</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3.02 %</td><td rowspan=1 colspan=1>1.64 %</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>1×103</td><td rowspan=1 colspan=1>HSV-2MS</td><td rowspan=1 colspan=1>1×103</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>1.09 %</td><td rowspan=1 colspan=1>2.95 %</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>177(3×LoD)</td><td rowspan=1 colspan=1>HSV-2MS</td><td rowspan=1 colspan=1>16.2(3×LoD)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>1.74 %</td><td rowspan=1 colspan=1>1.88 %</td></tr></table>

# Interfering Substances

The performance of the HSV 1&2 ELITe MGB Assay was evaluated with potentially interfering substances that could be encountered in lesion swab specimens obtained from cutaneous and mucocutaneous locations. A total of 33 substances were individually spiked into negative UTM containing one of each HSV-1 and HSV-2 isolates at 3X LoD level and tested in triplicate with the HSV 1&2 ELITe MGB Assay. There were no invalid results. No interference was observed (see table below).

<table><tr><td rowspan=2 colspan=1>Potential Interferent</td><td rowspan=2 colspan=1>InterferentConcentration</td><td rowspan=1 colspan=3>#Detected/#Total</td></tr><tr><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1>IC</td></tr><tr><td rowspan=1 colspan=1>Whole blood with EDTA</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Buffy coat</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Acyclovir</td><td rowspan=1 colspan=1>2.5 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Casein</td><td rowspan=1 colspan=1>7 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Female urine</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Male urine</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>K-Y Brand jelly</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Douche</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Spermicide</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yeast·Gard</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Monistat 1</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Monistat 3</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Vagisil Cream</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Tioconazole 1</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Rite Aid Feminine Wash,Sensitive Skin</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole-7 vaginal cream</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Oral Analgesic Gel</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Listerine antiseptic mouthwash</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Abreva</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Carmex lip balm</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Releev cold sore treatment</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Lip Clear lysine</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Toothpaste</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Wal-Finate</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Cold-Eeze</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Non-GMO Corn Starch</td><td rowspan=1 colspan=1>1.25 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Zinc Oxide Ointment</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Cough DM</td><td rowspan=1 colspan=1>10mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Lanacane Max Strength anti-itch cream</td><td rowspan=1 colspan=1>7% w/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Seminal fluid</td><td rowspan=1 colspan=1>7% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Foscarnet sodium</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# g. Assay cut-off:

The assay cut-off analysis was performed on a separate set of 141 clinical samples collected from 3 clinical sites. Each clinical sample was evaluated using HSV 1&2 ELITe MGB Assay in conjunction with the ELITe InGenius instrument and a

composite reference method (FDA-cleared real-time PCR assay combined with PCR amplification and bi-directional sequencing). Both targets in clinical samples were detected up to cycle 45. Therefore $\mathrm { C } _ { \mathrm { T } }$ of 45 was established as a diagnostic assay cutoff for both HSV-1 and HSV-2 targets.

# h. Sample stability studies:

This study assessed both sample stability and sample freeze-thaw stability. The samples for the stability evaluation were prepared by spiking both the HSV-1 and HSV-2 vendor quantitated viral stocks (HSV-1 MacIntyre strain and HSV-2 MS strain) in UTM, M4, M4RT, M5 and M6 media.

Each stability sample set consisted of:

• 5 replicates spiked at $3 { \times } \mathrm { L o D }$ ,   
5 replicates spiked at $1 \times 1 0 ^ { 3 }$ TCID50/mL, and   
5 replicates spiked at $1 \times 1 0 ^ { 5 }$ TCID50/mL (15 replicates total for each sample set).

The stability of each sample set was assessed by sample incubation at $+ 4 ^ { \circ } \mathrm { C }$ for 1 week. All HSV-1 and HSV-2 samples were confirmed to be stable in UTM, M4, M4RT, M5 and M6 media for 1 week at $+ 4 ^ { \circ } \mathrm { C }$ .

The storage conditions were also validated by re-testing previously analyzed clinical samples that were stored in a $- 8 0 ^ { \circ } \mathrm { C }$ freezer $( { \leq } - 7 0 ^ { \circ } \mathrm { C } )$ for minimum of 4 months. Sample concentrations covered HSV clinical range. Ten HSV-1 or HSV-2 positive samples were tested for each media (except M6 for which only 7 HSV-positive samples were available). Positivity of all samples was confirmed after 4 month storage in a $- 8 0 ^ { \circ } \mathrm { C }$ freezer $( { \leq } - 7 0 ^ { \circ } \mathrm { C } )$ .

A freeze thaw study was performed using 5 sample sets prepared as in the above using UTM, M4, M4RT, M5 and M6 media. All samples were subjected to 3 freezethaw cycles. All the samples were tested with the HSV 1&2 ELITe MGB Assay on the ELITe InGenius. The data obtained show that HSV-1 and HSV-2 viruses are stable after 3 freeze-thaw cycles in UTM, M4, M4RT, M5 and M6 media.

# i. Carryover contamination:

The sample-to-sample carry-over from positive samples into the negative samples for the HSV 1&2 ELITe MGB Assay was studied by performing 5 integrated checkerboard runs; HSV-1 high positive at $2 . 5 { \times } 1 0 ^ { 7 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ concentration and negative samples (UTM) interspersed, which were compared to the overall background noise from 2 negative sample runs. All 5 checkerboard runs and 2 complete negative runs were performed by 1 operator. The high positive sample concentration in this study exceeded $9 5 \%$ of the concentration levels obtained from samples of infected patients in the study. No carry-over and cross contamination was

observed (the results are shown below).

Carry-Over and Cross-Contamination Results   

<table><tr><td rowspan=2 colspan=1>Run description</td><td rowspan=1 colspan=2>PositiveSamples</td><td rowspan=1 colspan=2>NegativeSamples</td></tr><tr><td rowspan=1 colspan=1># Neg</td><td rowspan=1 colspan=1>% Neg.</td><td rowspan=1 colspan=1># Pos.</td><td rowspan=1 colspan=1>% Pos.</td></tr><tr><td rowspan=1 colspan=1>Run#1, BLANK</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0 /10</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#2,Checkerboard</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#3,Checkerboard</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#4,BLANK</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0 /10</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#5,Checkerboard</td><td rowspan=1 colspan=1>0 /6</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0 /6</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#6,Checkerboard</td><td rowspan=1 colspan=1>0 /6</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Run#7,Checkerboard</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>All runs</td><td rowspan=1 colspan=1>0 /29</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0 /50</td><td rowspan=1 colspan=1>0 %</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison:

Because all analytical studies were conducted in the UTM and clinical studies were conducted using UTM, M4, M4RT, M5 and M6 media, a the matrix comparison study was performed. The matrix comparison study was conducted using contrived sample panel made by spiking HSV-1 or HSV-2 quantitated viral organisms into each of the recommended media: UTM, M4, M4RT, M5 and M6. Each sample set consisted of 3 replicates spiked at $3 { \times } \mathrm { L o D }$ , 3 replicates spiked at $1 \times 1 0 ^ { 3 }$ T $\mathrm { \dot { C } I D 5 0 / m L }$ , and 3 replicates spiked at $\bar { 1 } { \times } 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ (9 replicates total for each sample set). Each sample was processed on the InGenius using the HSV 1&2 ELITe MGB Assay. All replicates in all media were detected and shown comparable results. All tested media showed comparable performance as shown in the table below.

<table><tr><td rowspan=2 colspan=1>Target/Channel</td><td rowspan=2 colspan=1>SampleTiterTCID50/mL</td><td rowspan=1 colspan=5>Sample Matrix</td><td rowspan=2 colspan=1>AllMediaAvgCT</td><td rowspan=2 colspan=1>AllMediaStDev</td><td rowspan=2 colspan=1>AllMedia%CV</td></tr><tr><td rowspan=1 colspan=1>UTM,AvgCT</td><td rowspan=1 colspan=1>M4,AvgCT</td><td rowspan=1 colspan=1>M4RT,AvgCT</td><td rowspan=1 colspan=1>M5,AvgCT</td><td rowspan=1 colspan=1>M6,AvgCT</td></tr><tr><td rowspan=3 colspan=1>HSV-2CH1,FAM</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>27.15</td><td rowspan=1 colspan=1>26.76</td><td rowspan=1 colspan=1>26.42</td><td rowspan=1 colspan=1>26.82</td><td rowspan=1 colspan=1>27.23</td><td rowspan=1 colspan=1>26.88</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.21%</td></tr><tr><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>33.59</td><td rowspan=1 colspan=1>33.76</td><td rowspan=1 colspan=1>33.51</td><td rowspan=1 colspan=1>34.15</td><td rowspan=1 colspan=1>33.77</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.74%</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>36.32</td><td rowspan=1 colspan=1>35.56</td><td rowspan=1 colspan=1>35.96</td><td rowspan=1 colspan=1>35.54</td><td rowspan=1 colspan=1>36.14</td><td rowspan=1 colspan=1>35.91</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.97%</td></tr><tr><td rowspan=2 colspan=1>HSV-1CH4,</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>22.02</td><td rowspan=1 colspan=1>21.13</td><td rowspan=1 colspan=1>20.82</td><td rowspan=1 colspan=1>20.77</td><td rowspan=1 colspan=1>20.63</td><td rowspan=1 colspan=1>21.08</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>2.66%</td></tr><tr><td rowspan=2 colspan=1>CH4,AP593</td><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>28.01</td><td rowspan=1 colspan=1>28.47</td><td rowspan=1 colspan=1>27.97</td><td rowspan=1 colspan=1>28.58</td><td rowspan=1 colspan=1>26.72</td><td rowspan=1 colspan=1>27.95</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.64%</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>35.84</td><td rowspan=1 colspan=1>36.07</td><td rowspan=1 colspan=1>37.02</td><td rowspan=1 colspan=1>35.43</td><td rowspan=1 colspan=1>34.69</td><td rowspan=1 colspan=1>35.81</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.39%</td></tr></table>

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable

Not Applicable

c. Other clinical supportive data (when a. and b. are not applicable):

To evaluate the clinical performance of the HSV 1&2 ELITe MGB Assay, device performance was compared to a composite reference method. It consisted of an FDA cleared assay and a validated HSV 1&2 PCR followed by bi-directional sequencing of gel electrophoresis-positive samples). Validated HSV 1&2 PCR targeted genomic regions distinct from the HSV 1&2 ELITe MGB Assay. A positive result by the composite reference method is defined as a positive by the FDA cleared PCR or the validated sequencing. Two negative results are needed to confirm a negative)

A total of 1,174 left-over prospectively collected archived swab samples from cutaneous (546) and mucocutaneous (628) lesions from symptomatic patients were collected and evaluated in the study.

The samples were tested with HSV 1&2 ELITe MGB Assay and the Composite Reference Method. Out of the 1,174 tested samples, 2 samples were found invalid by the ELITe MGB Assay and were excluded from the performance analysis tables. Out of the 1172 remaining samples 1 additional invalid sample result for HSV1 and 2 additional invalid sample results for HSV2 by the composite reference method were removed from the performance analysis tables. Therefore for HSV1, 1171 samples analyzed and for HSV2 1170 samples were analyzed.

# HSV-1 Positive/Negative Percent Agreements (PPA/NPA) - Summary of the Results:

The PPA/NPA performance of HSV 1&2 ELITe MGB Assay when compared to the Composite Reference Method in detection of HSV-1 DNA in cutaneous and mucocutaneous lesions is summarized in the tables below:

<table><tr><td rowspan=1 colspan=4>Summary of HSV-1 Results for Valid Cutaneous Lesion Samples (N=545)</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2 ELITe MGB Kit</td><td rowspan=1 colspan=3>Composite reference method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>85</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>460</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>466</td><td rowspan=1 colspan=1>545</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.2-99.8%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>96.9-99.3%</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=4>Summary of HSV-1 Results for Valid Mucocutaneous Lesion Samples (N=626)</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2 ELITe MGB Kit</td><td rowspan=1 colspan=3>Composite reference method</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>487</td><td rowspan=1 colspan=1>488</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>626</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>95.7-99.9%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>95.8-98.6%</td><td rowspan=1 colspan=1></td></tr></table>

# HSV-2 Positive/Negative Percent Agreements (PPA/NPA) - Summary of the Results:

The PPA/NPA performance of HSV 1&2 ELITe MGB Assay when compared to the Composite Reference Method in detection of HSV-2 DNA in cutaneous and mucocutaneous lesions is summarized in the table below:

<table><tr><td rowspan=1 colspan=4>Summary of HSV-2 Results for Valid Cutaneous Lesion Samples (N=545)</td></tr><tr><td rowspan=1 colspan=1>HSV 1&amp;2 ELITe MGB Kit</td><td rowspan=1 colspan=3>Composite reference method</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>414</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>545</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>96.2%</td><td rowspan=1 colspan=1>91.3-98.3%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>96.9-99.3%</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="4" rowspan="1">Summary of HSV-2 Results for Valid Mucocutaneous Lesion Samples (N=625)</td></tr><tr><td colspan="1" rowspan="1">HSV 1&amp;2 ELITe MGB Kit</td><td colspan="3" rowspan="1">Composite reference method</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">164</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">172</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">449</td><td colspan="1" rowspan="1">453</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">457</td><td colspan="1" rowspan="1">625</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">97.6%</td><td colspan="1" rowspan="1">94.0-99.1%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">98.2%</td><td colspan="1" rowspan="1">96.6-99.1%</td><td colspan="1" rowspan="1"></td></tr></table>

# Contrived Sample Study:

Due to the difficulty in obtaining sufficient HSV-2 positive oral samples, testing for HSV-2 was supplemented by using a contrived panel. The panel consisted of 75 individual negative cheek swab samples collected in Universal Transport Media (UTM) and spiked with HSV-2 at concentrations of $3 { \times } \mathrm { L o D }$ , $8 { \times } \mathrm { L o D }$ , $4 0 { \times } \mathrm { L o D }$ , $2 0 0 { \times } \mathrm { L o D }$ and $1 0 0 0 { \times } \mathrm { L o D }$ (10 of each), 10 HSV-1 Positive samples spiked at $1 0 { \times } \mathrm { L o D }$ and 15 HSV-1/HSV-2 Negative Oral Samples

All panel members were randomized, blinded to the tester and tested with HSV 1&2 ELITe MGB Assay on the ELITe InGenius instrument according to the clinical study protocol.

The HSV-2 Oral Contrived Panel Study revealed that 49 out of 50 oral HSV-2 contrived samples were positive using HSV 1&2 ELITe MGB Assay $9 8 \%$ detection). All 10 HSV-1 Positive samples confirmed $100 \%$ positivity.

4. Clinical cut-off:

Not Applicable

# 5. Expected values/Reference range:

The observed expected values for HSV-1 and HSV-2 in the study population using the HSV 1&2 ELITe MGB Assay were calculated and are presented for the combined sample set stratified by age group, gender and by lesion source in the tables below. A total of 6 dual positives for HSV1 and HSV2 were detected by the ELITe MGB Assay and one of the samples was confirmed as dual positive by the composite reference method.

# Cutaneous and Mucocutaneous HSV 1&2 Prevalence by Age and Gender

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>Age Group</td><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGBAssay HSV-1 results</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGBAssay HSV-2 results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=7 colspan=1>Female</td><td rowspan=1 colspan=1>&lt;20</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>42.9%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>28.6%</td></tr><tr><td rowspan=1 colspan=1>20-29</td><td rowspan=1 colspan=1>244</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>27.9%</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>28.7%</td></tr><tr><td rowspan=1 colspan=1>30-39</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>16.8%</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>31.5%</td></tr><tr><td rowspan=1 colspan=1>40-49</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14.4%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25.8%</td></tr><tr><td rowspan=1 colspan=1>50-59</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>20.5%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>27.3%</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>17.1%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24.4%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>737</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>22.1%</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>28.0%</td></tr><tr><td rowspan=7 colspan=1>Male</td><td rowspan=1 colspan=1>&lt;20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10.0%</td></tr><tr><td rowspan=1 colspan=1>20-29</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>17.4%</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>22.9%</td></tr><tr><td rowspan=1 colspan=1>30-39</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>21.4%</td></tr><tr><td rowspan=1 colspan=1>40-49</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>31.3%</td></tr><tr><td rowspan=1 colspan=1>50-59</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>20.5%</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>21.3%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>434</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>13.8%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>22.4%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Gender not identified</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=2>ALL</td><td rowspan=1 colspan=1>1172</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>25.9%</td></tr></table>

Cutaneous HSV 1&2 Expected Values by Lesion Source   

<table><tr><td rowspan=2 colspan=1>Lesion Source</td><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGBAssayHSV-1 results</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGB AssayHSV-2 results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>Genital/Anogenital</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>15.3%</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>31.5%</td></tr><tr><td rowspan=1 colspan=1>Skin lesion</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>15.8%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>17.8%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>545</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>15.6%</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>24.0%</td></tr></table>

Mucocutaneous HSV 1&2 Expected Values by Lesion Source   

<table><tr><td rowspan=2 colspan=1>Lesion Source</td><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGBAssay HSV-1 results</td><td rowspan=1 colspan=2>HSV 1&amp;2 ELITe MGBAssay HSV-2 results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>Genital/Vaginal/Cervical</td><td rowspan=1 colspan=1>501</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>21.8%</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>32.5%</td></tr><tr><td rowspan=1 colspan=1>Oral</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>28.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.7%</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7.4 %</td></tr><tr><td rowspan=1 colspan=1>Anorectal</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>41.7%</td></tr><tr><td rowspan=1 colspan=1>Urethral</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Ocular</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Nasal</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50.0 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 %</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>627</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>22.0%</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>27.4%</td></tr></table>

# N. Instrument Name:

ELITe InGenius system

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes _X___ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No ___X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

3. Specimen Identification: Not Applicable

4. Specimen Sampling and Handling: Not Applicable

5. Calibration: Not Applicable

6. Quality Control: Not Applicable

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

The HSV 1&2 ELITe MGB Assay is used on the ELITe InGenius system. The ELITe InGenius system is a bench top instrument integrating all required hardware, reagent and software components to perform nucleic acid sample preparation and real-time PCR operations.

The ELITe InGenius system can process from 1 to 12 samples in 12 parallel tracks, and samples may be loaded in primary tubes or in secondary tubes provided. The system utilizes a universal, cassette-based process for sample extraction, and allows for multiple and independent PCRs to be performed from a single nucleic acid eluate. The system can operate in three different modes: nucleic acid extraction only, PCR amplification only, or nucleic acid extraction with PCR amplification.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.