# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K180041

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the BioCode Gastrointestinal Pathogen Panel (GPP) for the detection of microbial nucleic acids extracted from human stool specimens.

# C. Measurands:

Target nucleic acid sequences of the following gastrointestinal microorganisms:

· Campylobacter (C. jejuni/C. coli)   
· Clostridium difficile (C. difficile) toxin A/B   
· Salmonella spp.   
· Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio parahaemolyticus   
· Yersinia enterocolitica   
· Enteroaggregative Escherichia coli (EAEC)   
· Enterotoxigenic Escherichia coli (ETEC) lt/st   
· E. coli O157 serogroup   
· Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2   
Shigella/ Enteroinvasive Escherichia coli (EIEC)   
· Cryptosporidium spp.   
· Entamoeba histolytica   
Giardia lamblia (also known as G. intestinalis and G. duodenalis)   
· Adenovirus F 40/41   
· Norovirus GI/GII   
· Rotavirus A   
D. Type of Test: Qualitative nucleic acid multiplex test   
E. Applicant: Applied BioCode   
F. Proprietary and Established Names:

# BioCode Gastrointestinal Pathogen Panel (GPP)

# G. Regulatory Information:

1.Regulation section:   
866.3990   
Gastrointestinal microorganisms multiplex nucleic acid-based assay

2.Classification: Class II

3.Product code: PCH, OOI

4.Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BioCode Gastrointestinal Pathogen Panel (GPP) is a qualitative, gastrointestinal microorganism multiplexed nucleic acid-based assay capable of detecting of nucleic acids from the following organisms in unpreserved stool and Cary-Blair media:

Adenovirus 40/41 Campylobacter (C. jejuni, C. coli) Clostridium difficile (C. difficile) toxin A/B (from fresh specimens only) Cryptosporidium (C. parvum, C. hominis) Entamoeba histolytica Escherichia coli (E. coli) O157 Enterotoxigenic E. coli (ETEC) LT/ST Enteroaggregative E. coli (EAEC) Giardia lamblia (also known as G. intestinalis and G. duodenalis) · Norovirus GI/GII Rotavirus A Salmonella spp. Shiga-like Toxin producing $E .$ . coli (STEC) stx1/stx2 Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus), specific identification of V. parahaemolyticus Yersinia enterocolitica

The BioCode Gastrointestinal Pathogen Panel (GPP) is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out coinfection with organisms not included in the BioCode Gastrointestinal Pathogen Panel (GPP).

The agent detected may not be the cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn's disease.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.   
This device is not intended to monitor or guide treatment for C. difficile infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Campylobacter spp., E. coli O157, Shigella/EIEC, Yersinia enterocolitica, and Adenovirus 40/41 were established primarily with retrospective clinical specimens.

Performance characteristics for Entamoeba histolytica, and Vibrio spp. (V. parahaemolyticus, V. vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens.

Indication(s) for use:

Same as intended use.

1. Special conditions for use statement(s): For prescription use only.

2. Special instrument requirements: For use with the BioCode MDx 3000 instrument.

# I. Device Description:

The BioCode Gastrointestinal Pathogen Panel (GPP)is a multiplex nucleic acid test designed to be used with the BioCode MDx 3000 instrument. The BioCode Gastrointestinal Pathogen Panel (GPP) is an automated system that integrates PCR amplification, target capture, signal generation and optical detection for multiple gastrointestinal pathogens from a single stool specimen, either unpreserved or in Cary Blair medium. Stool specimens are processed and nucleic acids are extracted with the bioMérieux NucliSENSE easyMag. Once the PCR plate is set up and sealed, all other operations are automated. The BioCode Gastrointestinal Pathogen Panel (GPP) simultaneously tests for 17 pathogens from unpreserved stool specimens or stool preserved in

Cary-Blair medium. Results from the BioCode Gastrointestinal Pathogen Panel (GPP) test are available within 5 hours.

# J. Substantial Equivalence Information:

1. Predicate device name(s): xTAG Gastrointestinal Pathogen Panel

2. Predicate 510(k) number(s): K121454

3. Comparison with predicate: See table below.

Table 1. Comparison with Predicate Device   

<table><tr><td>Name Intended Use The xTAG Gastrointestinal Pathogen Panel</td><td>Predicate: Luminex xTAG GPP K121454</td><td>Applied BioCode Gastrointestinal Pathogen Panel (GPP)</td></tr><tr><td></td><td>(GPP) is a multiplexed nucleic acid test intended for the simultaneous qualitative detection and identification of multiple viral, parasitic, and bacterial nucleic acids in human stool specimens from individuals with signs and symptoms of infectious colitis or gastroenteritis. The following pathogen types, subtypes and toxin genes are identified using the xTAG GPP: Campylobacter spp. (C. jejuni, C. coli and C. lari only) Clostridium difficile (C. difficile) toxin A/B Cryptosporidium (C. parvum and C. hominis only) Escherichia coli (E. coli) O157 Enterotoxigenic Escherichia coli (ETEC) LT/ST Giardia (G. lamblia only - also known as G. intestinalis and G. duodenalis) Norovirus GI/GII Rotavirus A Salmonella Shiga-like Toxin producing E. coli (STEC) stx 1/stx 2 Shigella (S. boydii, S. sonnei, S. flexneri and S. dysenteriae) The detection and identification of specific gastrointestinal microbial nucleic acid from individuals exhibiting signs and symptoms of gastrointestinal infection aids in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation, laboratory findings and epidemiological information. A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks. xTAG GPP positive results are presumptive and must be confirmed by FDA-cleared tests or other acceptable reference methods. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Confirmed positive results do not rule out coinfection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative xTAG Gastrointestinal Pathogen Panel results in the setting of clinical illness compatible with gastroenteritis may be due to infection by or non-infectious causes such as ulcerative pathogens that are not detected by this test or colitis, irritable bowel syndrome, or Crohn's</td><td>The BioCode Gastrointestinal Pathogen Panel (GPP) is a qualitative, gastrointestinal microorganism multiplexed nucleic acid- based assay capable of detecting of nucleic acids from the following organisms in unpreserved stool and Cary-Blair media: •Adenovirus 40/41 •Campylobacter (C. jejuni, C. coli) •Clostridium difficile (C. difficile) toxin A/B (from fresh specimens only) •Cryptosporidium (C. parvum, C. hominis) •Entamoeba histolytica •Escherichia coli (E. coli) O157 •Enterotoxigenic E. coli (ETEC) LT/ST •Enteroaggregative E. coli (EAEC) •Giardia lamblia (also known as G. intestinalis and G. duodenalis) Norovirus GI/GII •Rotavirus A Salmonella spp. •Shiga-like Toxin producing E. coli (STEC) stx1/stx2 •Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC •Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus), specific identification of V. parahaemolyticus •Yersinia enterocolitica The BioCode Gastrointestinal Pathogen Panel (GPP) is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the BioCode Gastrointestinal Pathogen Panel (GPP). The agent detected may not be the cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">use with the Luminex 100/200 instrument.</td><td colspan="1" rowspan="1">This device is not intended to monitor orguide treatment for C. difficile infection.Due to the small number of positivespecimens collected for certain organismsduring the prospective clinical study,performance characteristics forCampylobacter spp., E. coli O157,Shigella/EIEC, Yersinia enterocolitica, andAdenovirus 40/41 were establishedprimarily with retrospective clinicalspecimens.Performance characteristics for Entamoebahistolytica, and Vibrio spp. (V.parahaemolyticus, V. vulnificus, and Vibriocholerae) were established primarily usingcontrived clinical specimens.</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Unpreserved stool and stool in Cary-Blairmedium</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Externally sourced</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Nucleic acid purification system PCRthermocycler Luminex 100/200 or MAGPIX</td><td colspan="1" rowspan="1">BioCode MDx 3000</td></tr><tr><td colspan="1" rowspan="1">AssayMethodology</td><td colspan="1" rowspan="1">Multiplex PCR and multiplex TSPEfollowed by fluorescence-activated sorting oflabeled beads coupled to streptavidin-conjugated biotinylated products</td><td colspan="1" rowspan="1">Multiplex PCR and probe hybridizationfollowed by fluorescence detection anddecoding of barcoded magnetic beads(BMB) that are coupled to biotinylatedproducts with streptavidin conjugate</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Approximately 5 hours</td><td colspan="1" rowspan="1">Approximately 5 hours</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration.</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable.

# L. Test Principle:

The BioCode Gastrointestinal Pathogen Panel (GPP) is an automated system that integrates PCR amplification, target capture, signal generation and optical detection for multiple gastrointestinal pathogens from a single stool specimen. Stool specimens are processed and nucleic acids are extracted with the bioMérieux NucliSENSE easyMAG automated system. Once the PCR plate is set up and sealed, all other operations are automated.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Reproducibility:

A study was performed to assess the Reproducibility of the BioCode Gastrointestinal Pathogen Panel (GPP) on the BioCode MDx 3000. This study was designed to assess intraassay (within run), inter-assay (run-to-run), day-to-day and site-to-site reproducibility. One lot of reagents was assayed at 3 sites by 2 operators on 1 instrument per site for 5 days (total of 10 runs per site; 90 replicates total).

The reproducibility panel consisted of 7 contrived samples (sample 7 was a negative control) extracted in triplicate and each assayed in singlet. The samples consisted of combinations of 12 representative targets at $1 . 5 \mathrm { x }$ LoD (Low) and $3 \mathbf { x }$ LoD (Medium). Reproducibility for each analyte and panel member was $> 9 9 \%$ . Results from the study are shown in Table 2.

Table 2. Qualitative results from Reproducibility study.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Concentration Level</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Medium Positive</td><td rowspan=1 colspan=2>Low Positive</td><td rowspan=1 colspan=2>Negative</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Detectionn/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Detectionn/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9-100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9-100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>449/450(99.78)</td><td rowspan=1 colspan=1>(98.7 -99.9)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9-100.0)</td><td rowspan=1 colspan=1>89/90(98.89)</td><td rowspan=1 colspan=1>(93.96-99.97)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidiumparvum</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>90/90(100.0)</td><td rowspan=1 colspan=1>(95.9 -100.0)</td><td rowspan=1 colspan=1>450/450(100.0)</td><td rowspan=1 colspan=1>(99.2 -100.0)</td></tr></table>

# b. Fresh versus Frozen Study:

Since many of the unpreserved samples underwent a freeze thaw prior to testing with the BioCode Gastrointestinal Pathogen Panel (GPP), a study was performed using 60 spiked unpreserved stool specimens to assess fresh vs frozen specimen stability. Specimens were contrived and tested initially as fresh as well as after $\geq 2$ freeze thaw cycles. Samples were tested at low positive (1X - 1.5X LoD) spiked into 7 negative stool samples. CaryBlair specimens were not subjected to frozen storage during the clinical study, therefore they were not tested. Results for the study are presented in Table 3 with mean fluorescence intensities listed in Table 4 along with the normalized values. Typical raw signal for positive values are greater than 18,000 mean fluorescence intensity units. Frozen samples displayed stability throughout the length of the study as exemplified by changes in mean fluorescent intensity of less than $20 \%$ .

Table 3. Fresh Vs. Frozen Summary.   

<table><tr><td rowspan=1 colspan=1>Acceptance Criteria</td><td rowspan=1 colspan=1>Results</td></tr><tr><td rowspan=1 colspan=1>95% replicates for all positive targets in the contrived sample should be Validand Detected (≥60/63)</td><td rowspan=1 colspan=1>63/63</td></tr><tr><td rowspan=1 colspan=1>95% of negative targets should be Valid and not detected (≥60/63)</td><td rowspan=1 colspan=1>63/63a</td></tr></table>

a – Sample Target was detected but was invalid for RNA IC.

Table 4. Analysis of results from Fresh Vs Frozen study.   

<table><tr><td rowspan=1 colspan=3>Analysis</td><td rowspan=1 colspan=1>Delta Fresh -Frozen</td><td rowspan=1 colspan=1>PercentDecrease</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>FF1</td><td rowspan=1 colspan=1>3835</td><td rowspan=1 colspan=1>14%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli 10C-3114 (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>FF2</td><td rowspan=1 colspan=1>3019</td><td rowspan=1 colspan=1>12%</td></tr><tr><td rowspan=1 colspan=1>Human rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>FF3</td><td rowspan=1 colspan=1>4437</td><td rowspan=1 colspan=1>13%</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td><td rowspan=1 colspan=1>FF4</td><td rowspan=1 colspan=1>1260</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>O78:H11 Escherichia coli strain H10407 (ETEC)</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>FF5</td><td rowspan=1 colspan=1>2503</td><td rowspan=1 colspan=1>8%</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>FF6</td><td rowspan=1 colspan=1>2365</td><td rowspan=1 colspan=1>20%</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica subsp. Enterica</td><td rowspan=1 colspan=1>ATCC 14028</td><td rowspan=1 colspan=1>FF7</td><td rowspan=1 colspan=1>-148</td><td rowspan=1 colspan=1>-2%</td></tr><tr><td rowspan=1 colspan=1>Human adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>FF8</td><td rowspan=1 colspan=1>610</td><td rowspan=1 colspan=1>3%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>waterborne P102M</td><td rowspan=1 colspan=1>FF9</td><td rowspan=1 colspan=1>3134</td><td rowspan=1 colspan=1>14%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>FF10</td><td rowspan=1 colspan=1>1719</td><td rowspan=1 colspan=1>6%</td></tr></table>

# c. Specimen stability:

A study was performed to establish the specimen stability limitations of the BioCode Gastrointestinal Pathogen Panel (GPP). This study employed the use of spiked specimens to assess the following storage conditions:

· Samples in Cary-Blair - 0, 2, 4 and 6 days at room temperature $( 2 0 - 2 5 ^ { \circ } \mathrm { C } )$ ); 2, 4 and 6 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ Unpreserved Stool - 0, 2, 4 and 6 days at $2 { - } 8 ^ { \circ } \mathrm { C }$   
· Fresh vs. Frozen (- $- 9 0 ^ { \circ } \mathrm { C }$ to $- 6 0 ^ { \circ } \mathrm { C }$ ) (2x freeze thaws) - 30, 60 and 90 days at $- 9 0 \mathrm { { } ^ { \circ } C }$ to - $6 0 ^ { \circ } \mathrm { C }$ S.T.A.R. buffer after pretreatment (SK38 Tubes prior to extraction) - 24 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ Fresh vs. Frozen ( $- 9 0 \mathrm { { } ^ { \circ } C }$ to $- 6 0 ^ { \circ } \mathrm { C }$ ) (2x freeze thaws) - 30, 60 and 90 days at $- 9 0 ^ { \circ } \mathrm { C }$ to $- 6 0 ^ { \circ } \mathrm { C }$ Extracted Nucleic Acid

- 24 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$

· Fresh vs. Frozen ( $- 9 0 ^ { \circ } \mathrm { C }$ to $- 6 0 ^ { \circ } \mathrm { C }$ ) (2x freeze thaws) - 30, 60 and 90 days at $- 9 0 \mathrm { { } ^ { \circ } C }$ to - $6 0 ^ { \circ } \mathrm { C }$

One parasite ( $C .$ . parvum), one virus (Rotavirus A), one gram-positive bacterium ( $C .$ . difficile), and one gram-negative bacterium (STEC) were used as representative panel analyte constituents for this study. Spiked and clinical samples were tested at ${ \sim } 3 \mathrm { X }$ LoD with two organisms in each sample.

Concentrated organism stocks were serially diluted in Cary-Blair and combined with prescreened negative stool and each condition was assayed with 3 replicates at each time point. Samples displayed acceptable stability under the all conditions listed above.

# d. Linearity/assay reportable range:

Not applicable because the BioCode Gastrointestinal Pathogen Panel (GPP) is qualitative.

e. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Control Summary:

Four pools of external controls were used on a rotating basis throughout testing. Each of these pools consisted of inactivated organisms sourced from Waterborne Inc. or Zeptometrix as shown in Table 5. Results from control testing performed in the clinical and reproducibility studies are shown in Table 6.

Table 5. Summary of Controls   

<table><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Dilution Factor</td></tr><tr><td rowspan=3 colspan=1>Pool 1</td><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>NAP1</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>WA</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Z004</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=5 colspan=1>Pool 2</td><td rowspan=1 colspan=1>Escherichia coli (EAEC)</td><td rowspan=1 colspan=1>92.0147; EAEC</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>DS4-868</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>recombinant</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=4 colspan=1>Pool 3</td><td rowspan=1 colspan=1>Adenovirus Type 41</td><td rowspan=1 colspan=1>TAK</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliO157/STEC</td><td rowspan=1 colspan=1>EDL933</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>Waterborne Inc.</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>Z005</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=4 colspan=1>Pool 4</td><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Iowa</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (ETEC)</td><td rowspan=1 colspan=1>ETEC; ST+, LT+</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>recombinant</td><td rowspan=1 colspan=1>Natrol (ZeptoMetrix)</td><td rowspan=1 colspan=1>1/4</td></tr></table>

Table 6. Performance of Controls during Clinical Trials (including Reproducibility testing)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site 001(valid/total)</td><td rowspan=1 colspan=1>Site 002(valid/total)</td><td rowspan=1 colspan=1>Site 003(valid/total)</td><td rowspan=1 colspan=1>Site 006(valid/total)</td><td rowspan=1 colspan=1>All sites(valid/total)</td></tr><tr><td rowspan=1 colspan=1>Pool 1</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>11/14</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>28/31</td></tr><tr><td rowspan=1 colspan=1>Pool 2</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>23/23</td></tr><tr><td rowspan=1 colspan=1>Pool 3</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Pool 4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>15/15</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>38/39</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>111/116</td></tr></table>

# $f .$ Limit of detection:

A study was performed to assess the performance of the BioCode Gastrointestinal Pathogen Panel (GPP) at the Limit of Detection (LoD) for both unpreserved Stool and Cary- Blair specimens. In this study the GI Panel was tested with quantified stocks of bacteria, virus or parasites. For initial screening, four replicates of each concentration in negative stool and Cary-Blair were extracted on the easyMag System and tested in singlet with the BioCode Gastrointestinal Pathogen Panel (GPP) to estimate LoD. The LoD was confirmed by extracting 20 replicates of each sample type and testing each in singlet for a total of 20 replicates at or near the presumptive LoD. LoD for each stock was defined as the lowest concentration with $2 9 5 \%$ detection of 20 replicates (19 out of 20) and was determined separately for unpreserved stool and Cary-Blair preserved stool. Results for LoD testing are shown in Table 7 below.

Table 7. Limits of Detection in CFU/mL   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Unpreserved StoolLoD</td><td rowspan=1 colspan=1>UnpreservedStoolDetection</td><td rowspan=1 colspan=1>Cary-Blair StoolLoD</td><td rowspan=1 colspan=1>Cary-BlairStoolDetection</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 33559</td><td rowspan=1 colspan=1>5.6 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.6 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>ATCC 33292</td><td rowspan=1 colspan=1>7.0 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.0 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(toxinotype 0)</td><td rowspan=1 colspan=1>ATCC 9689</td><td rowspan=1 colspan=1>1.9 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.9 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile(toxinotype III; Nap1)</td><td rowspan=1 colspan=1>Zeptometrix0801619cf</td><td rowspan=1 colspan=1>8.3 x 10² CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.3 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coliO92:H33 (EAEC)</td><td rowspan=1 colspan=1>STEC TW04440</td><td rowspan=1 colspan=1>1.4 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.4 x 10³3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enteroinvasive E. coli029:NM (EIEC)</td><td rowspan=1 colspan=1>ATCC 43892</td><td rowspan=1 colspan=1>3.6 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.5 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli078:H11 H10407 (ETEC)</td><td rowspan=1 colspan=1>ATCC 35401</td><td rowspan=1 colspan=1>5.6 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.6 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>SGSC 4900</td><td rowspan=1 colspan=1>1.4 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.5 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica ssp.enterica</td><td rowspan=1 colspan=1>ATCC 14028</td><td rowspan=1 colspan=1>2.2 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.1 x 103 CFU/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC)</td><td rowspan=1 colspan=1>ATCC BAA-2217</td><td rowspan=1 colspan=1>2.5 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>2.5 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>ATCC 700376</td><td rowspan=1 colspan=1>3.3 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.3 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>4.4 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.7 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>ATCC 25870</td><td rowspan=1 colspan=1>4.9 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4.9 x 102 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td><td rowspan=1 colspan=1>1.3 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.0 x 101 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 23715</td><td rowspan=1 colspan=1>1.5 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.5 x 103 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>UKRC</td><td rowspan=1 colspan=1>1.3 x 104oocysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.3 x 104oocysts/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>waterborne P102C</td><td rowspan=1 colspan=1>3.1 x 103oocysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.1 x 103oocysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolyticaHB-301:NIH</td><td rowspan=1 colspan=1>BEI NR-178</td><td rowspan=1 colspan=1>3.1 x 10-1 cysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>3.1 x 10-1cysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis(aka G. lamblia)</td><td rowspan=1 colspan=1>waterborne P101</td><td rowspan=1 colspan=1>1.8 x 103 cysts/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.8 x 103 cysts/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>Zeptometrix0810084</td><td rowspan=1 colspan=1>1.0 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.0 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41 (TAK)</td><td rowspan=1 colspan=1>Zeptometrix0810085</td><td rowspan=1 colspan=1>9.4 x 10-2TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>7.5 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>ATCC VR-2018</td><td rowspan=1 colspan=1>2.5 x 103TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>6.2 x 102TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus GIa</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>6.4 x 105copies/gram</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>6.5 x 105copies/gram</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Norovirus GIIa</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>5.2 x 104copies/gram</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.96 x 104copies/gram</td><td rowspan=1 colspan=1>20/20</td></tr></table>

# g. Analytical Reactivity/Inclusivity:

A study was performed to verify analytical reactivity of the BioCode Gastrointestinal Pathogen Panel (GPP). Different strains were selected that represent various temporal, geographic, and genetic diversity for each analyte. This study tested a panel of titered stocks for relevant organisms diluted in pre-screened negative stools at 3X LoD. Stocks not detected at 3X LoD, if applicable, were tested at higher concentrations. Due to a lack of titered specimens, Adenovirus 40/41 clinical samples and Cryptosporidium DNA from the Cryptosporidium reference unit were used (Public Health England). Norovirus GI and GII genotypes and the Rotavirus vaccine strain were tested by the CDC. All of the organisms were detected at the concentrations indicated. The strains evaluated are shown in Tables 8-21 below.

Table 8. Shiga-like toxin producing $E _ { \ast }$ . coli (STEC) stx1/stx2 and E. coli O157 Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=6 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>E. coli O157:H45</td><td rowspan=1 colspan=1>STEC SC373/2 TW07922</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:HNM</td><td rowspan=1 colspan=1>STEC DA-26TW07952</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:H7</td><td rowspan=1 colspan=1>STEC 93-111 TW04863</td></tr><tr><td rowspan=1 colspan=1>E. coli O157: H7</td><td rowspan=1 colspan=1>STEC MI06-19 TW14301</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157:HNT</td><td rowspan=1 colspan=1>STEC DA-27 TW07953</td></tr><tr><td rowspan=1 colspan=1>E. coli O157:H7 Strain EDL933</td><td rowspan=1 colspan=1>BEI NR-11</td></tr><tr><td rowspan=17 colspan=1>Shiga toxinproducing E. coli(STEC)</td><td rowspan=1 colspan=1>E. coli O26:H11</td><td rowspan=1 colspan=1>STEC 2332/00 TW08998</td></tr><tr><td rowspan=1 colspan=1>E. coli O113:H21</td><td rowspan=1 colspan=1>STEC CL-15 TW02318</td></tr><tr><td rowspan=1 colspan=1>E. coli O45:H2</td><td rowspan=1 colspan=1>STEC DEC11C DEC11c</td></tr><tr><td rowspan=1 colspan=1>E. coli O103:H2</td><td rowspan=1 colspan=1>STEC 107-226 TW07881</td></tr><tr><td rowspan=1 colspan=1>E. coli O104:H21</td><td rowspan=1 colspan=1>STEC G5506 TW04909</td></tr><tr><td rowspan=1 colspan=1>E. coli O111:H2</td><td rowspan=1 colspan=1>STEC RD8 TW06296</td></tr><tr><td rowspan=1 colspan=1>E. coli O111:H8</td><td rowspan=1 colspan=1>STEC DEC8B</td></tr><tr><td rowspan=1 colspan=1>E.coli O26:NM</td><td rowspan=1 colspan=1>STEC DA-22 TW07948</td></tr><tr><td rowspan=1 colspan=1>E.coli 0145:NM</td><td rowspan=1 colspan=1>ATCC BAA-2192</td></tr><tr><td rowspan=1 colspan=1>E. coli O146:21</td><td rowspan=1 colspan=1>STECDEC16E TW01383</td></tr><tr><td rowspan=1 colspan=1>E. coli O45:H2</td><td rowspan=1 colspan=1>STECMI05-14 TW14003</td></tr><tr><td rowspan=1 colspan=1>E. coli O121:19</td><td rowspan=1 colspan=1>STEC MDCH-4 TW07614</td></tr><tr><td rowspan=1 colspan=1>E. coli O121:NM</td><td rowspan=1 colspan=1>STEC DA-37 TW07972</td></tr><tr><td rowspan=1 colspan=1>E. coli *0104:H4</td><td rowspan=1 colspan=1>ATCC BAA-2326</td></tr><tr><td rowspan=1 colspan=1>E. coli 045:H2</td><td rowspan=1 colspan=1>ATCC BAA-2193</td></tr><tr><td rowspan=1 colspan=1>E. coli O26: H11</td><td rowspan=1 colspan=1>BAA-2196</td></tr><tr><td rowspan=1 colspan=1>E. coli O121:H19</td><td rowspan=1 colspan=1>BAA-2219</td></tr></table>

Table 9. Enterotoxigenic $E .$ . coli (ETEC), Enteroaggregative E. coli (EAEC) Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=4 colspan=1>EnteroaggregativeE. coli (EAEC)</td><td rowspan=1 colspan=1>E. coli O44:H18</td><td rowspan=1 colspan=1>STEC 042 TW04393</td></tr><tr><td rowspan=1 colspan=1>E. coli O111a,111b:K58:H21</td><td rowspan=1 colspan=1>ATCC 29552</td></tr><tr><td rowspan=1 colspan=1>E. coli O104:H4</td><td rowspan=1 colspan=1>ATCC BAA-2326</td></tr><tr><td rowspan=1 colspan=1>E. coli O3:K2a</td><td rowspan=1 colspan=1>BEI NR-102</td></tr></table>

Table 10. Shigella/Enteroinvasive E. coli (EIEC) Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=5 colspan=1>EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1>E. coli O121</td><td rowspan=1 colspan=1>ATCC BAA-2190</td></tr><tr><td rowspan=1 colspan=1>E. coli O124:HNM</td><td rowspan=1 colspan=1>STEC 929-78 TW16574</td></tr><tr><td rowspan=1 colspan=1>E. coli O136:H</td><td rowspan=1 colspan=1>STEC LT-41 TW06139</td></tr><tr><td rowspan=1 colspan=1>E. coli O285A:HNM</td><td rowspan=1 colspan=1>BEI NR-101</td></tr><tr><td rowspan=1 colspan=1>E. coli O15</td><td rowspan=1 colspan=1>ATCC 49105</td></tr><tr><td rowspan=5 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>Shigella boydii (Type 1)</td><td rowspan=1 colspan=1>ATCC 9207</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii, (Type 2)</td><td rowspan=1 colspan=1>BEI NR-521</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 7)</td><td rowspan=1 colspan=1>ATCC 9905</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 20)</td><td rowspan=1 colspan=1>ATCC BAA-1247</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii (Type 3)</td><td rowspan=1 colspan=1>ATCC 8702</td></tr><tr><td rowspan=5 colspan=1>Shigelladysenteriae</td><td rowspan=1 colspan=1>Shigella dysenteriae (Type 1)</td><td rowspan=1 colspan=1>BEI NR-520</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Type 3)</td><td rowspan=1 colspan=1>ATCC 9751</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae(Type 2)</td><td rowspan=1 colspan=1>ATCC 9750</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae(Type 5)</td><td rowspan=1 colspan=1>ATCC 9764</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae (Type 12)</td><td rowspan=1 colspan=1>ATCC 49552</td></tr><tr><td rowspan=5 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>Shigella flexneri, strain 24570(Type 2a)</td><td rowspan=1 colspan=1>BEI NR-517</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri, strain 2457T(Type 2a)</td><td rowspan=1 colspan=1>BEI NR-518</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 2b)</td><td rowspan=1 colspan=1>ATCC 12022</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 6)</td><td rowspan=1 colspan=1>ATCC 12025</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri (Type 1b)</td><td rowspan=1 colspan=1>ATCC 9380</td></tr><tr><td rowspan=4 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 25931</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 11060</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 9290</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29029</td></tr></table>

Table 11. Salmonella Inclusivity results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. arizonae</td><td rowspan=1 colspan=1>Salmonella enterica subsp.Enterica</td><td rowspan=1 colspan=1>ATCC 13314</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. salamae</td><td rowspan=1 colspan=1>serovar Tranoroa</td><td rowspan=1 colspan=1>ATCC 700148</td></tr><tr><td rowspan=27 colspan=1>Salmonella entericasubsp. enterica</td><td rowspan=1 colspan=1>serovar Montevideo</td><td rowspan=1 colspan=1>ATCC 7001 BAA-710</td></tr><tr><td rowspan=1 colspan=1>serovar Enteritidis</td><td rowspan=1 colspan=1>SGSC4901</td></tr><tr><td rowspan=1 colspan=1>serovar Enteritidis</td><td rowspan=1 colspan=1>ATCC 4931</td></tr><tr><td rowspan=1 colspan=1>serovar Oranienburg</td><td rowspan=1 colspan=1>SGSC4079</td></tr><tr><td rowspan=1 colspan=1>serovar Paratyphi B var.L(+)tartrate+</td><td rowspan=1 colspan=1>SGSC4150</td></tr><tr><td rowspan=1 colspan=1>serovar Typhimurium</td><td rowspan=1 colspan=1>SGSC1412</td></tr><tr><td rowspan=1 colspan=1>serovar Saintpaul</td><td rowspan=1 colspan=1>SGSC2512</td></tr><tr><td rowspan=1 colspan=1>serovar S. typhimurium LT2</td><td rowspan=1 colspan=1>SGSC2666</td></tr><tr><td rowspan=1 colspan=1>serovar Newport</td><td rowspan=1 colspan=1>SGSC2493</td></tr><tr><td rowspan=1 colspan=1>serovar Newport</td><td rowspan=1 colspan=1>ATCC 6962</td></tr><tr><td rowspan=1 colspan=1>serovar Muenchen</td><td rowspan=1 colspan=1>SGSC2490</td></tr><tr><td rowspan=1 colspan=1>serovar Agona</td><td rowspan=1 colspan=1>SGSC2458</td></tr><tr><td rowspan=1 colspan=1>serovar Javiana</td><td rowspan=1 colspan=1>SGSC4917</td></tr><tr><td rowspan=1 colspan=1>serovar Schwarzengrund</td><td rowspan=1 colspan=1>SGSC2514</td></tr><tr><td rowspan=1 colspan=1>serovar Heidelberg</td><td rowspan=1 colspan=1>SGSC2480</td></tr><tr><td rowspan=1 colspan=1>serovar Infantis</td><td rowspan=1 colspan=1>SGSC2484</td></tr><tr><td rowspan=1 colspan=1>serovar Montevideo</td><td rowspan=1 colspan=1>SGSC2487</td></tr><tr><td rowspan=1 colspan=1>serovar Thompson</td><td rowspan=1 colspan=1>SGSC 2519</td></tr><tr><td rowspan=1 colspan=1>serovar Hadar</td><td rowspan=1 colspan=1>SGSC4965</td></tr><tr><td rowspan=1 colspan=1>serovar Mississippi</td><td rowspan=1 colspan=1>SGSC4078</td></tr><tr><td rowspan=1 colspan=1>serovar Paratyphi A</td><td rowspan=1 colspan=1>SGSC2499</td></tr><tr><td rowspan=1 colspan=1>serovar Choleraesuis</td><td rowspan=1 colspan=1>SGSC4770</td></tr><tr><td rowspan=1 colspan=1>serovar Choleraesuis</td><td rowspan=1 colspan=1>ATCC 13312</td></tr><tr><td rowspan=1 colspan=1>serovar Dublin</td><td rowspan=1 colspan=1>SGSC4157</td></tr><tr><td rowspan=1 colspan=1>serovar Braenderup</td><td rowspan=1 colspan=1>ATCC® 700136</td></tr><tr><td rowspan=1 colspan=1>serovar Bareilly</td><td rowspan=1 colspan=1>ATCC® 9115</td></tr><tr><td rowspan=1 colspan=1>serovar Typhi</td><td rowspan=1 colspan=1>Zeptometrix0801933</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. II</td><td rowspan=1 colspan=1>Salmonella enterica subs. II</td><td rowspan=1 colspan=1>SGSC3039</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IIIa</td><td rowspan=1 colspan=1>Salmonella enterica subs. IIIa</td><td rowspan=1 colspan=1>SGSC3061</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IIIb</td><td rowspan=1 colspan=1>Salmonella enterica subs. IIIb</td><td rowspan=1 colspan=1>SGSC3068</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. IV</td><td rowspan=1 colspan=1>Salmonella enterica subs. IV</td><td rowspan=1 colspan=1>SGSC3074</td></tr><tr><td rowspan=1 colspan=1>Salmonella entericasubsp. VI</td><td rowspan=1 colspan=1>Salmonella enterica subs VI</td><td rowspan=1 colspan=1>SGSC3116</td></tr></table>

Table 12. Campylobacter inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=8 colspan=1>Campylobacter spp.</td><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>BEI NR-399</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. jejuni</td><td rowspan=1 colspan=1>BEI NR-400</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. doylei</td><td rowspan=1 colspan=1>ATCC 49350</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni subsp. doylei</td><td rowspan=1 colspan=1>ATCC 49349</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43478</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43485</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>BEI HM-296</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC 43484</td></tr></table>

Table 13. Vibrio spp inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=11 colspan=1>Vibrio spp. (notparahaemolyticus)</td><td rowspan=1 colspan=1>Vibrio cholerae (O:1 Inaba,Biotype E1 Tor)</td><td rowspan=1 colspan=1>BEI NR-147</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (O:1 Inaba,Biotype E1 Tor)</td><td rowspan=1 colspan=1>BEI NR-146</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae(O:2)</td><td rowspan=1 colspan=1>BEI NR-149</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (O:4)</td><td rowspan=1 colspan=1>BEI NR-151</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (O:139)</td><td rowspan=1 colspan=1>BEI NR-144</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae (O:1 Ogawa)</td><td rowspan=1 colspan=1>ATCC 14035</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 27562</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC BAA-88</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 43382</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 29306</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>ATCC 29307</td></tr><tr><td rowspan=5 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-21990</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR- 21991</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-22002</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Zepto 0801903</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>BEI NR-22013</td></tr></table>

Table 14. Yersinia enterocolitica inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=5 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia enterocolitica (O:8)</td><td rowspan=1 colspan=1>ATCC 9610</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica(0:3)</td><td rowspan=1 colspan=1>BEI NR-212</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica (O:8)</td><td rowspan=1 colspan=1>BEI NR-206</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 29913</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>BEI NR-213</td></tr></table>

Table 15. Clostridium difficile inclusivity results.   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>TOXINOTYPE</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=15 colspan=2>Clostridium difficile</td><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 43255-FZ</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 700792-FZ</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1382-fz</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 51695-fz</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC43599-fz</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC43596-fz</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC17858-fz</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC43594</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC43600</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC 17857</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1871</td></tr><tr><td rowspan=1 colspan=1>0 A+B+</td><td rowspan=1 colspan=1>ATCC BAA-1872</td></tr><tr><td rowspan=1 colspan=1>VIII A-B+</td><td rowspan=1 colspan=1>ATCC 43598</td></tr><tr><td rowspan=1 colspan=1>III A+B+ (Nap1)</td><td rowspan=1 colspan=1>ATCCBAA-1805</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>XXII A+B+</td><td rowspan=1 colspan=1>ATCCBAA-1814</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>III A+B+</td></tr><tr><td rowspan=1 colspan=1>VA+B+</td><td rowspan=1 colspan=1>ATCC BAA-1875</td></tr></table>

Table 16. Adenovirus 40/41 inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=3 colspan=1>Adenovirus 40</td><td rowspan=1 colspan=1>Human adenovirus 40 (dugan)</td><td rowspan=1 colspan=1>lot #K1220A</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP143</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP258</td></tr><tr><td rowspan=7 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP170</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID# SP174</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID# SP 276</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID# SP288</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID# SP309</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP329</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>sample ID # SP446</td></tr></table>

Table 17. Norovirus GI/GII inclusivity results.   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=8 colspan=2>*Norovirus GI</td><td rowspan=1 colspan=1>GI.1</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.2</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.3</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.4</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.5</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.6</td><td rowspan=1 colspan=1>ClinicalSpecimen</td></tr><tr><td rowspan=1 colspan=1>GI.7</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GI.8</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=14 colspan=2>aNorovirus GII</td><td rowspan=1 colspan=1>GII.1</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.2</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.3</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.4 New Orleans</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.4 Sydney</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>GII.5</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>GII.6</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.7</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.8</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.12</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.13</td><td rowspan=1 colspan=1>Clinicall Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.14</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.16</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr><tr><td rowspan=1 colspan=1>GII.17</td><td rowspan=1 colspan=1>Clinical Specimen</td></tr></table>

Table 18. Rotavirus inclusivity results.   

<table><tr><td rowspan=1 colspan=4>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=6 colspan=4></td><td rowspan=1 colspan=1>Rotavirus A (DS-1)</td><td rowspan=1 colspan=1>ATCC VR-2550</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 89-12C2)</td><td rowspan=1 colspan=1>ATCC VR-2272</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (WISC2)</td><td rowspan=1 colspan=1>ATCC VR-2417</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 408)</td><td rowspan=1 colspan=1>ATCC VR-2273</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A (HRV 248)</td><td rowspan=1 colspan=1>ATCC VR-2274</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>Rotavirus A (HU)</td></tr><tr><td rowspan=8 colspan=4>Rotavirus A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rotavirus A (HRV CJN)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G1</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G2</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G3</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G4</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A G9</td><td rowspan=1 colspan=1>Clinical Sample</td></tr><tr><td rowspan=1 colspan=1>Rota vaccine strain</td><td rowspan=1 colspan=1>RotaTeq vaccine</td></tr><tr><td rowspan=1 colspan=1>Rota vaccine strain</td><td rowspan=1 colspan=1>Rotarix vaccine</td></tr></table>

Table 19. Cryptosporidium spp inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=5 colspan=1>C. parvum</td><td rowspan=1 colspan=1>DNA subtype /IIaA17G1R1</td><td rowspan=1 colspan=1>UKCR UK28</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /IIaA15G2R1</td><td rowspan=1 colspan=1>UKCRUK29</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /IIaA 19G1R1</td><td rowspan=1 colspan=1>UKCRUK30</td></tr><tr><td rowspan=1 colspan=1>DNA subtype /IIdA22G1</td><td rowspan=1 colspan=1>UKCRUK31</td></tr><tr><td rowspan=1 colspan=1>DNA subtype/IIdA15G1</td><td rowspan=1 colspan=1>UKCRUK32</td></tr><tr><td rowspan=3 colspan=1>C. hominis</td><td rowspan=1 colspan=1>DNA subtype IaA14R3</td><td rowspan=1 colspan=1>UKCR UKH14</td></tr><tr><td rowspan=1 colspan=1>DNA subtype IdA18</td><td rowspan=1 colspan=1>UKH12</td></tr><tr><td rowspan=1 colspan=1>DNA subtype IbA10G2</td><td rowspan=1 colspan=1>UKH13</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>DNA E. coli O103:H2</td><td rowspan=1 colspan=1>NR2520</td></tr></table>

Table 20. Entamoeba histolytica inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=4 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>200:NIH</td><td rowspan=1 colspan=1>BEI NR-177</td></tr><tr><td rowspan=1 colspan=1>HB-301:NIH</td><td rowspan=1 colspan=1>BEI NR-176</td></tr><tr><td rowspan=1 colspan=1>Rahman</td><td rowspan=1 colspan=1>BEI NR-179</td></tr><tr><td rowspan=1 colspan=1>H-303:NIH</td><td rowspan=1 colspan=1>BEI NR-180</td></tr></table>

Table 21. Giardia lamblia inclusivity results.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=4 colspan=1>Giardia lamblia/intestinalis</td><td rowspan=1 colspan=1>Egypt-4</td><td rowspan=1 colspan=1>BEI NR-9231</td></tr><tr><td rowspan=1 colspan=1>Mario</td><td rowspan=1 colspan=1>BEI NR-9232</td></tr><tr><td rowspan=1 colspan=1>D.Hall</td><td rowspan=1 colspan=1>BEI NR-9234</td></tr><tr><td rowspan=1 colspan=1>DAN</td><td rowspan=1 colspan=1>BEI NR-9235</td></tr></table>

# h. Analytical Cross-reactivity:

A study was performed to verify that the BioCode Gastrointestinal Pathogen Panel (GPP) does not detect DNA or RNA from organisms commonly found in stool specimens or from organisms that can cause similar clinical symptoms. In addition, onpanel organisms were tested at high concentrations to insure there is no cross-reactivity with other panel targets. This study tested a panel of titered stocks and genomic DNA extracts for relevant organisms. Organisms that were not available for wet testing were analyzed in silico comparing the whole organism sequence against all primers to assess potential for cross reactivity. Analysis was conducted using BlastN and Primer Blast programs. Results from evaluation of cross-reactivity are shown in Tables 21-25.

Cross-reactivity was not observed with microorganisms tested in this study except for the

following:

Y. enterocolitica: Wet testing and in silico sequence analysis indicate a potential for cross-reactivity with Y. bercovieri, Y. frederiksenii, Y. intermedia and Y. mollaretii near the established LoD for Y. enterocolitica $( \sim 1 . 5 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { C F U / m L } )$ ). Y. rohdei was also detected when present at high levels $\left( > 6 . 8 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m L } \right)$ . These species are in the Y. enterocolitica group and are suspected human pathogens.

Shiga toxin: Shiga toxin $( s t x )$ , that is identical to stx1 of STEC is found in Shigella dysenteriae; therefore, a BioCode Gastrointestinal Pathogen Panel (GPP) GI Panel report with positive test results for Shiga-like toxin-producing $E .$ . coli (STEC) and Shigella/Enteroinvasive $E$ . coli (EIEC) in the same sample may indicate the presence of Shigella dysenteriae since the genetic relatedness of these two pathogens indicate they could constitute a single species but they are maintained as separately named entities for the purposes of epidemiology and clinical medicine.

Rotavirus: Wet testing has demonstrated that the GPP assay will detect recombinant viruses included in Rotavirus vaccines.

Cryptosporidium: Wet testing demonstrated cross reactivity with $C .$ . cuniculus and $C .$ meleagridis .

E. histolytica: Wet testing with a gene fragment construct and in silico sequence analysis do not predict cross reactivity with the closely related E. dispar.

The following tables (Table 22, Table 23, and table 24) present a listing of organisms for which there was no observed cross-reactivity:

Table 22. Bacteria tested for Cross-reactivity.   

<table><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Aeromonas jandaei</td><td rowspan=1 colspan=1>Egglerthella lenta</td><td rowspan=1 colspan=1>Proteus penneri</td></tr><tr><td rowspan=1 colspan=1>Aeromonas media</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Aeromonas trota</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Providencia alcalifaciens</td></tr><tr><td rowspan=1 colspan=1>Aeromonas caviae</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanni</td><td rowspan=1 colspan=1>Enteropathogenic Escherichia coli</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>Escherichia coli Non pathogenic</td><td rowspan=1 colspan=1>Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>Saccharomyces boulardii</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Escherichia vulneris</td><td rowspan=1 colspan=1>Serratia liquefaciens</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Bacteroides thetaiotaomicron</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Shewanella algae</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium breve</td><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter fetus</td><td rowspan=1 colspan=1>Grimontia hollisae (formerly vibrio)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter hyointestinalis</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensis</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Cedecea davisae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>Trabulsiella guamensis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile non-toxigenic</td><td rowspan=1 colspan=1>Leminorella grimontii</td><td rowspan=1 colspan=1>Vibrio alginolyticus a</td></tr><tr><td rowspan=1 colspan=1>Clostridium histolyticum</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Vibrio fluvialis</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Vibrio mimicusa</td></tr><tr><td rowspan=1 colspan=1>Clostridium septicum</td><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Yersinia bercovieri b</td></tr><tr><td rowspan=1 colspan=1>Clostridium sordellii</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Yersinia frederiksenii b</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>Porphyromonas asaccharolytica</td><td rowspan=1 colspan=1>Yersinia intermedia b</td></tr><tr><td rowspan=1 colspan=1>Clostridium tetani</td><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>Yersinia mollaretii b</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Yersinia pseudotuberculosis</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Yersinia rohdei b</td></tr></table>

a Detected as Vibrio spp. at high titers, see full submission for details. b Detected as Yersinia enterocolitica at high titers.

Table 23. Parasites tested for Cross-reactivity.   

<table><tr><td rowspan=1 colspan=2>Viruses</td><td rowspan=1 colspan=1>Parasites</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3</td><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>Cryptosporidium cuniculus (DNA)a</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 4</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Cryptosporidium felis (DNA)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7</td><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Cryptosporidium meleagridis1</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 8</td><td rowspan=1 colspan=1>HSV Type 2</td><td rowspan=1 colspan=1>Cryptosporidium meleagridis (DNA)a</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 14</td><td rowspan=1 colspan=1>Norovirus GIV</td><td rowspan=1 colspan=1>Cryptosporidium muris</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37</td><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>Cryptosporidium ubiquitum (DNA)</td></tr><tr><td rowspan=1 colspan=1>Astrovirus type 1</td><td rowspan=1 colspan=1>Sapovirus GI</td><td rowspan=1 colspan=1>Encephalitozoon cuniculi</td></tr><tr><td rowspan=1 colspan=1>Astrovirus type 4</td><td rowspan=1 colspan=1>Sapovirus GIV</td><td rowspan=1 colspan=1>Encephalitozoon hellem</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Sapovirus GV</td><td rowspan=1 colspan=1>Encephalitozoon intestinalis</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Parasites</td><td rowspan=1 colspan=1>Giardia muris</td></tr><tr><td rowspan=1 colspan=1>Coxsackie virus A16</td><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>Pentatrichomonas hominis</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>Toxoplasma gondii</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (CMV)</td><td rowspan=1 colspan=1>Cryptosporidium canis (DNA)</td><td rowspan=1 colspan=1></td></tr></table>

a Detected as Cryptosporidium spp.

Table 24. Microorganisms tested for Cross-reactivity.   

<table><tr><td rowspan=1 colspan=3>GI Panel Targets</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Enterotoxigenic E. coli O78:H11H10407 (ETEC)</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni spp. jejuni</td><td rowspan=1 colspan=1>Shiga toxin producing E. coli (STEC)</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (toxinotype 0 )</td><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (toxinotype III;Nap1)</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>Entamoeba histolytica HB-301:NIH</td></tr><tr><td rowspan=1 colspan=1>gregative E. coli O92:H33 (EAEC)</td><td rowspan=1 colspan=1>Salmonella enterica ssp. enterica</td><td rowspan=1 colspan=1>Giardia intestinalis (aka G. lamblia)</td></tr><tr><td rowspan=1 colspan=1>Enteroinvasive E. coli O29:NM(EIEC)</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>Adenovirus 40 (dugan)</td></tr><tr><td rowspan=1 colspan=1>cYanmaGelowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rotavirus A</td></tr></table>

Table 25. Analysis of microorganisms in silico for Cross-reactivity.   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Parasites</td></tr><tr><td rowspan=1 colspan=1>Anaerococcus tetradius</td><td rowspan=1 colspan=1>Eubacterium cylindroides</td><td rowspan=1 colspan=1>Ancylostoma duodenale</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td><td rowspan=1 colspan=1>Eubacterium rectale</td><td rowspan=1 colspan=1>Ascaris lumbricoides</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium longum</td><td rowspan=1 colspan=1>Megamonas hypermegale</td><td rowspan=1 colspan=1>Balantidium coli</td></tr><tr><td rowspan=1 colspan=1>Campylobacter concisus</td><td rowspan=1 colspan=1>Methanobrevibacter smithii</td><td rowspan=1 colspan=1>Chilomastix mesnili</td></tr><tr><td rowspan=1 colspan=1>Campylobacter curvus</td><td rowspan=1 colspan=1>Peptoniphilus asaccharolyticus</td><td rowspan=1 colspan=1>Cryptosporidium bovis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter gracilis</td><td rowspan=1 colspan=1>Ruminococcus bromii</td><td rowspan=1 colspan=1>Cryptosporidium canis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter helveticus</td><td rowspan=1 colspan=1>Ruminococcus flavefaciens</td><td rowspan=1 colspan=1>Cryptosporidium cuniculus b</td></tr><tr><td rowspan=1 colspan=1>Campylobacter hominis</td><td rowspan=1 colspan=1>Ruminococcus obeum</td><td rowspan=1 colspan=1>Cryptosporidium felis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari</td><td rowspan=1 colspan=1>Selenomonas ruminantium</td><td rowspan=1 colspan=1>Cryptosporidium fetus</td></tr><tr><td rowspan=1 colspan=1>Campylobacter mucosalis</td><td rowspan=1 colspan=1>Vibrio cincinnatiensis</td><td rowspan=1 colspan=1>Cryptosporidium meleagridis b</td></tr><tr><td rowspan=1 colspan=1>Campylobacter rectus</td><td rowspan=1 colspan=1>Vibrio furnissii</td><td rowspan=1 colspan=1>Cryptosporidium muris</td></tr><tr><td rowspan=1 colspan=1>Campylobacter showae</td><td rowspan=1 colspan=1>Vibrio metschnikovii</td><td rowspan=1 colspan=1>Cryptosporidium ryanae</td></tr><tr><td rowspan=1 colspan=1>Campylobacter sputorum</td><td rowspan=1 colspan=1>Yersinia kristensenii</td><td rowspan=1 colspan=1>Cryptosporidium xiaoi</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensis</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Dientamoeba fragilis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter ureolyticus</td><td rowspan=1 colspan=1>Norovirus GIV</td><td rowspan=1 colspan=1>Endolimax nana</td></tr><tr><td rowspan=1 colspan=1>Clostridium acetobutylicum</td><td rowspan=1 colspan=1>Rotavirus B</td><td rowspan=1 colspan=1>Entamoeba coli</td></tr><tr><td rowspan=1 colspan=1>Clostridium methylpentosum</td><td rowspan=1 colspan=1>Rotavirus C a</td><td rowspan=1 colspan=1>Entamoeba dispar</td></tr><tr><td rowspan=1 colspan=1>Clostridium novyi</td><td rowspan=1 colspan=1>Rotavirus D</td><td rowspan=1 colspan=1>Entamoeba hartmanni</td></tr><tr><td rowspan=1 colspan=1>Clostridium ramosum</td><td rowspan=1 colspan=1>Rotavirus E</td><td rowspan=1 colspan=1>Entamoeba moshkovskii</td></tr><tr><td rowspan=1 colspan=1>Collinsella aerofaciens</td><td rowspan=1 colspan=1>Rotavirus F</td><td rowspan=1 colspan=1>Entamoeba polecki</td></tr><tr><td rowspan=1 colspan=1>Desulfovibrio piger</td><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1></td></tr></table>

a Cross-reactivity predicted with Porcine Rotavirus C strains only, no cross-reactivity with human Rotavirus C. b C. cuniculus and $C .$ . meleagridis cross reactivity for Cryptosporidium spp assay observed in lab testing was not predicted by in silico analysis (Primer Blast).

# i. Interference:

Interfering substances:

A study was performed to demonstrate the accuracy of the BioCode Gastrointestinal

Pathogen Panel (GPP) in the presence of potentially inhibiting substances. The substances tested were:

EDTA blood $( 4 0 \% \mathrm { w } / \mathrm { v } )$ Ampicillin $( 1 5 2 \mu \mathrm { m o l / L } )$ $10 \%$ bleach $( 5 0 \% \mathrm { w } / \mathrm { v } )$ cholesterol $( 5 \% \mathrm { w } / \mathrm { v } )$ mineral oil $( 5 0 \% \mathrm { w } / \mathrm { v } )$ hydrocortisone $( 5 0 \% \mathrm { w } / \mathrm { v } )$ Imodium $( 5 \% \mathrm { w } / \mathrm { v } )$ Senokot $( 5 \% \mathrm { w } / \mathrm { v } )$ Maalox $( 5 \% \mathrm { w / v } )$ metronidazole $\mathrm { 1 4 m g / m L }$

benzalkonium chloride ( $50 \%$ w/v)   
Supleco $( 5 \% \mathrm { w / v } )$ , mucin $( 3 ~ \mathrm { m g / m L } )$   
naproxen sodium $\mathrm { 1 4 m g / m L }$ )   
Neosporin ( $50 \%$ w/v)   
nystatin $( 1 0 0 0 \mathrm { U / m L } )$   
Pepto-Bismol $( 5 \% \mathrm { w } / \mathrm { v } )$   
Preparation H $[ ( 5 \% \mathrm { w } / \mathrm { v } )$   
Tums $( 5 \% \mathrm { w } / \mathrm { v } )$   
vancomycin ( $1 2 . 5 \mathrm { m g / m L }$ )

Each member of the interfering substance panel was added to prescreened negative stool sample spiked with representative members of the BioCode Gastrointestinal Pathogen Panel (GPP) at 3X LoD and a negative matrix only pre- screened negative stool. One parasite ( $C .$ . parvum), one virus (Rotavirus A), one gram positive bacterium (C. difficile), and one gram negative bacterium (STEC) was used as representative analytes for this study. Each sample was tested with and without potentially interfering substances. Each sample was extracted in triplicate tested in singlet with the GI Panel on the BioCode Gastrointestinal Pathogen Panel (GPP).

Concentrations were determined by reviewing $5 1 0 ( \mathrm { k } )$ summary results of previous GI panel clinical trials and CLSl-EP1-A2. None of the substances listed above returned interfering results at the concentrations tested.

# Microbial Inhibition:

The accuracy of the BioCode Gastrointestinal Pathogen Panel (GPP) in the presence of potentially inhibiting microorganisms was assessed in the following study. The interfering organisms tested were Bacteroides fragilis $\mathrm { 1 \times 1 0 ^ { 6 } C F U / m L }$ ), Blastocystis hominis $( 1 \mathrm { ~ x ~ } \bar { 1 0 ^ { 5 } }$ CFU/mL), Candida albicans $1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ ), Clostridium difficile non-toxigenic $\boldsymbol { \mathbf { \rho } } _ { 1 \mathrm { ~ X ~ } 1 0 \theta }$ $\mathrm { C F U / m L }$ ), Enterococcus faecalis $( 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ), Escherichia coli non-pathogenic ( $_ { \textrm { 1 x } }$ $1 0 ^ { 6 } \mathrm { C F U / m L }$ ), Pseudomonas aeruginosa $( 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ), and Saccharomyces boulardi (1 $\mathrm { \Delta x \ 1 0 ^ { 5 } C F U / m L }$ ).Each member of the interfering microorganism panel was added to prescreened negative stool sample spiked with representative members of the BioCode Gastrointestinal Pathogen Panel (GPP) GI Pathogen Panel at 3X LoD and a negative matrix only pre- screened negative stool. One parasite (C. parvum), one virus (Rotavirus A), one gram positive bacterium (C. difficile), and one gram negative bacterium (STEC) was used as representatives for this study. Each sample was tested with and without potentially interfering microorganisms. Each sample was extracted in triplicate and tested in singlet with the GI Panel on the BioCode Gastrointestinal Pathogen Panel (GPP).

Concentrations were determined by reviewing $5 1 0 \mathrm { k }$ summary results of previous GI panel clinical trials and CLSl-EP1-A2. None of the microorganisms listed above returned interfering results at the concentrations tested.

# Competitive Interference:

A study was performed to evaluate the potential for inhibition in samples with mixed infectious agents. Prescreen negative stool was spiked with one target at high concentration $( \ge 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and ${ \geq } 1 0 ^ { 5 }$ units/mL for viruses or parasites) and two targets at low concentration $( \le 3 \mathrm { L o D } )$ . Common co-infections were determined by reviewing results of previous GI Panel clinical trials from $5 1 0 \mathrm { k }$ summaries, publications/posters and internal clinical sample testing. Each sample was extracted in triplicate on the easyMAG and each extraction tested in singlet with the BioCode GPP on the BioCode MDx 3000 system. No inhibition was observed.

Table 26. Competitive inhibition testing results.   

<table><tr><td colspan="1" rowspan="2">Panel Designation</td><td colspan="1" rowspan="2">Viral/Bacteria Strain</td><td colspan="1" rowspan="2">Source</td><td colspan="1" rowspan="2">Level</td><td colspan="1" rowspan="2">Titer Tested</td><td colspan="1" rowspan="2">TargetProbe</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">(n of 3Detected)</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 1</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Zeptometrix801619</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">tcdB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">ATCC VR-2018</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.44X103TCID50/mL</td><td colspan="1" rowspan="1">Rota</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.02x103CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 2</td><td colspan="1" rowspan="1">092:H33Escherichia coli(EAEC)</td><td colspan="1" rowspan="1">STEC JM221TW04440</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.02X103CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Zeptometrix801619</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">5.7x102CFU/mL</td><td colspan="1" rowspan="1">tcdB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 3</td><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Zeptometrix801619</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">5.7x102CFU/mL</td><td colspan="1" rowspan="1">tcdB</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">ATCC VR-2018</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.44X103TCID50/mL</td><td colspan="1" rowspan="1">Rota</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 4</td><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3x106CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">092:H33Escherichia coli(EAEC)</td><td colspan="1" rowspan="1">STEC JM221TW04440</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">4.08X103CFU/mL</td><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC 33292</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">2.1X103CFU/mL</td><td colspan="1" rowspan="1">Campy</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 5</td><td colspan="1" rowspan="1">Campylobacter jejunisubsp.Doylei</td><td colspan="1" rowspan="1">ATCC 49349</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">Campy</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.02X103CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">092:H33Escherichia coli(EAEC)</td><td colspan="1" rowspan="1">STEC JM221TW04440</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">4.08X103CFU/mL</td><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Panel Designation</td><td colspan="1" rowspan="1">Viral/Bacteria Strain</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">TargetProbe</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 6</td><td colspan="1" rowspan="1">092:H33Escherichia coli(EAEC)</td><td colspan="1" rowspan="1">STEC JM221TW04440</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3x106CFU/mL</td><td colspan="1" rowspan="1">EAEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejunisubsp. jejuni</td><td colspan="1" rowspan="1">ATCC 33292</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">2.1X103CFU/mL</td><td colspan="1" rowspan="1">Campy</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coliE2348/69 (EPEC)</td><td colspan="1" rowspan="1">STEC TW06375</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.02X103CFU/mL</td><td colspan="1" rowspan="1">EPEC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 7</td><td colspan="1" rowspan="1">Shiga-toxin producing E.coli (STEC)</td><td colspan="1" rowspan="1">ATCC BAA-2217</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">stx2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">waterborne P101</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">5.42X103cysts/mL</td><td colspan="1" rowspan="1">G.lam</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">1.31X103CFU/mL</td><td colspan="1" rowspan="1">Shig</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 8</td><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">waterborne P101</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x105cysts/mL</td><td colspan="1" rowspan="1">G.lam</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">1.31X103CF/mML</td><td colspan="1" rowspan="1">Shig</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Shiga-toxin producing E.coli (STEC)</td><td colspan="1" rowspan="1">ATCC BAA-2217</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.5X103CFU/mL</td><td colspan="1" rowspan="1">stx2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 9</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3.0x106CFU/mL</td><td colspan="1" rowspan="1">Shig</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Shiga-toxin producing E.coli (STEC)</td><td colspan="1" rowspan="1">ATCC BAA-2217</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">7.5X103CFU/mL</td><td colspan="1" rowspan="1">stx2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">waterborne P101</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">5.42X103cysts/mL</td><td colspan="1" rowspan="1">G.lam</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# j. Carry-Over/Cross-Contamination study:

The BioCode Gastrointestinal Pathogen Panel (GPP) assay was evaluated for the presence of contamination due to carry over in known negative specimens by alternating high positive samples with negative samples. No evidence of carry-over contamination was observed.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison:

3. Clinical studies:

A total of 1558 leftover, de-identified samples were prospectively collected from patients who underwent stool sample collection for clinical indications of diarrhea caused by gastrointestinal infection at four (4) investigational sites, with each collecting approximately 400 samples. In addition testing was performed on archived known positives and contrived specimens to further augment the sample numbers. Each raw stool specimen was de-identified at the site and the de-identified leftover sample was divided into aliquots and either tested freshly on the BioCode Gastrointestinal Pathogen Panel (GPP) at the site, or frozen and tested at a later date at the site. Each stool specimen in Cary-Blair was similarly de-identified at the site and the de-identified leftover sample aliquoted into 4 aliquots and tested freshly on the BioCode Gastrointestinal Pathogen Panel (GPP) at the site. All prospectively collected samples had age, sex, and therapy status collected by the site. Enrollment of samples covered multiple calendar seasons beginning January 2015 and ending in August 2017. During the prospective clinical study $2 . 6 \%$ (41/1558) of samples were invalid for lack of RNA-IC signal on initial testing. The invalid rate after repeat testing was approximately $0 . 2 \%$ (3/1558). The results of the clinical study are presented in the tables below.

The performance for C. difficile for frozen specimens was not equivalent to the predicate device. Therefore, samples suspected of $C .$ . difficile presence should only be tested fresh. The assay software requires selection of sample type (fresh or frozen) and results for $C .$ . difficile are reported only for fresh specimens.

Table 27. Demographic data for prospective specimens (fresh and frozen).   

<table><tr><td rowspan=1 colspan=1>Prospective Study Specimens</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total Specimens</td><td rowspan=1 colspan=1>1558</td></tr><tr><td rowspan=1 colspan=1>Gender (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FemaleMale</td><td rowspan=1 colspan=1>778/1558 (49.9)780/1558 (51.1)</td></tr><tr><td rowspan=1 colspan=1>Age Category (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt; 5 year6-21 yrs22-59 yrs60+ yrs</td><td rowspan=1 colspan=1>140/1558 (9.0)237/1558 (15.2)718/1558 (46.1)463/1558 (29.7)</td></tr><tr><td rowspan=1 colspan=1>Status (n/N(%))</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>InpatientOutpatient</td><td rowspan=1 colspan=1>1212/1558 (77.8)346/1558 (22.2)</td></tr></table>

Table 28. Breakdown of prospective specimen collection by site.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Unpreserved(Fresh)</td><td rowspan=1 colspan=1>Unpreserved(Frozen)</td><td rowspan=1 colspan=1>Cary-Blair(Fresh)</td></tr><tr><td rowspan=1 colspan=1>Site 001</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>350</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 002</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>347</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 003</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 006</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>361</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>960</td><td rowspan=1 colspan=1>361</td></tr></table>

Table 29. Comparator (reference) methods for prospective clinical study.   

<table><tr><td rowspan=1 colspan=1>Target Pathogen/Toxin</td><td rowspan=1 colspan=1>Reference Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni, C. coli)</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (C. difficile) toxin A/B</td><td rowspan=1 colspan=1>FDA cleared NAAT</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium (C. parvum, C. hominis)</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (E. coli) O157</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) LT/ST</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli (EAEC)</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia /intestinalis</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Composite result of two PCR/sequencing</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Shiga-like Toxin producing E. coli (STEC) stx1/stx2</td><td rowspan=1 colspan=1>Enrichment culture/FDA cleared antigen test</td></tr><tr><td rowspan=1 colspan=1>Shigella (S. boydii, S. sonnei, S. flexneri, S. dysenteriae)/EIEC</td><td rowspan=1 colspan=1>Enrichment culture</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp. (V. cholerae, V. parahaemolyticus, V. vulnificus)</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Culture</td></tr></table>

Table 30. Summary of Clinical Study Results (Prospective specimens) for Unpreserved Stool (Fresh).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp. a</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>234/236 (99.2)</td><td rowspan=1 colspan=1>(97.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile b</td><td rowspan=1 colspan=1>26/27 (96.3)</td><td rowspan=1 colspan=1>(81.0 - 99.9)</td><td rowspan=1 colspan=1>208/210 (99.1)</td><td rowspan=1 colspan=1>(96.6 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>237/237 (100.0)</td><td rowspan=1 colspan=1>(98.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>234/234 (100.0)</td><td rowspan=1 colspan=1>(98.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC c</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2 - 100.0)</td><td rowspan=1 colspan=1>229/232 (98.7)</td><td rowspan=1 colspan=1>(96.4 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>STEC d</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>235/237(99.2)</td><td rowspan=1 colspan=1>(97.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Salmonella sppe</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2 - 100.0)</td><td rowspan=1 colspan=1>232/234(99.2)</td><td rowspan=1 colspan=1>(97.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC f</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>233/236 (98.7)</td><td rowspan=1 colspan=1>(96.3 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus g</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>236/237 (99.6)</td><td rowspan=1 colspan=1>(97.7 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>237/237 (100.0)</td><td rowspan=1 colspan=1>(98.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica h</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>236/237 (99.6)</td><td rowspan=1 colspan=1>(97.7 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>234/234 (100.0)</td><td rowspan=1 colspan=1>(98.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>235/235 (100.0)</td><td rowspan=1 colspan=1>(98.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>234/235 (99.6)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41 j</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>233/235 (99.2)</td><td rowspan=1 colspan=1>(97.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>235/235 (100.0)</td><td rowspan=1 colspan=1>(98.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5 - 100.0)</td><td rowspan=1 colspan=1>234/235 (99.6)</td><td rowspan=1 colspan=1>(97.7 - 100.0)</td></tr></table>

a - Campylobacter spp: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and 1 of 2 confirmed as positive.   
b - Clostridium difficile: The 1 false negative compared to the FDA Cleared NAAT reference test produced high Ct $\left( \mathrm { C t } 3 5 . 0 \right)$ . c - ETEC: The 3 false positives compared to bidirectional sequencing were not confirmed as positives by an additional round of sequencing.   
d - STEC: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and both confirmed as positive.   
e– Salmonella spp: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing, and both confirmed as positives.   
f - Shigella/EIEC: The 3 false positives compared to the culture reference method were tested by bidirectional sequencing, and all 3 confirmed as positives.   
g - Vibrio parahaemolyticus: The 1 false positive sample compared to the culture reference method was tested by bidirectional sequencing and confirmed as positive.   
h - Yersinia enterocolitica: The 1 false positive sample compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive.   
ij – Giardia lamblia: The 1 false positive to bidirectional sequencing was not confirmed as positive by 2 additional rounds of sequencing.   
j - Adenovirus 40/41: The 2 false positives to bidirectional sequencing were not confirmed as positives by an additional round of sequencing.

Table 31. Summary of Clinical Study Results (Prospective specimens) for Unpreserved Stool (Frozen).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp. a</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2 - 100.0)</td><td rowspan=1 colspan=1>936/952 (98.3)</td><td rowspan=1 colspan=1>(97.3 - 99.0)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157 b</td><td rowspan=1 colspan=1>1/2 (50.0)</td><td rowspan=1 colspan=1>(1.3 - 98.7)</td><td rowspan=1 colspan=1>950/954 (99.6)</td><td rowspan=1 colspan=1>(98.9 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>EAEC c</td><td rowspan=1 colspan=1>25/29 (86.2)</td><td rowspan=1 colspan=1>(68.3 - 96.1)</td><td rowspan=1 colspan=1>916/919 (99.7)</td><td rowspan=1 colspan=1>(99.1 99.9)</td></tr><tr><td rowspan=1 colspan=1>ETEC d</td><td rowspan=1 colspan=1>7/10 (70.0)</td><td rowspan=1 colspan=1>(34.8 - 93.3)</td><td rowspan=1 colspan=1>934/939 (99.5)</td><td rowspan=1 colspan=1>(98.8 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>STEC </td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2 - 100.0)</td><td rowspan=1 colspan=1>918/919 (99.9)</td><td rowspan=1 colspan=1>(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp f</td><td rowspan=1 colspan=1>18/22 (81.8)</td><td rowspan=1 colspan=1>(59.7 - 94.8)</td><td rowspan=1 colspan=1>926/934 (99.1)</td><td rowspan=1 colspan=1>(98.3 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC g</td><td rowspan=1 colspan=1>4/5 (80.00)</td><td rowspan=1 colspan=1>(28.4 - 99.5)</td><td rowspan=1 colspan=1>940/951 (98.8)</td><td rowspan=1 colspan=1>(97.9 - 99.4)</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus h</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>955/957 (99.8)</td><td rowspan=1 colspan=1>(99.3 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>956/956 (100.0)</td><td rowspan=1 colspan=1>(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>951/956 (99.5)</td><td rowspan=1 colspan=1>(98.8 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>7/7 (100.0)</td><td rowspan=1 colspan=1>(59.0 - 100.0)</td><td rowspan=1 colspan=1>941/941 (100.0)</td><td rowspan=1 colspan=1>(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>948/948 (100.0)</td><td rowspan=1 colspan=1>(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia j</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.8 - 100.0)</td><td rowspan=1 colspan=1>940/946 (99.4)</td><td rowspan=1 colspan=1>(98.6 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41 k</td><td rowspan=1 colspan=1>7/10 (70.0)</td><td rowspan=1 colspan=1>(34.8 - 93.3)</td><td rowspan=1 colspan=1>935/938 (99.7)</td><td rowspan=1 colspan=1>(99.1 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>39/39 (100.0)</td><td rowspan=1 colspan=1>(91.0 - 100.0)</td><td rowspan=1 colspan=1>913/917 (99.6)</td><td rowspan=1 colspan=1>(98.9 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>19/20 (95.0)</td><td rowspan=1 colspan=1>(75.1 - 99.9)</td><td rowspan=1 colspan=1>928/936 (99.2)</td><td rowspan=1 colspan=1>(98.3 - 99.6)</td></tr></table>

a - Campylobacter spp: The 16 false positives compared to the culture reference method were tested by bidirectional sequencing, and 8 of 16 confirmed as positives.

b - E. coli O157: The one false negative compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive. The 4 false positive samples compared to the culture reference method were tested by bidirectional sequencing, and 3 of 4 confirmed as positives.

c - EAEC: The 4 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 3 of the 4 confirmed as positives. 2 of the 3 false positives could not be repeated due to low sample volume. The remaining 1 was not detected by addition rounds of sequencing.

d - ETEC: The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. 1 of the 5 false positives could not be repeated due to low sample volume. The remaining 4 false positives were not confirmed as positives by an additional round of sequencing.

e - STEC: The 1 false positive compared to the culture reference method was tested by bidirectional sequencing and confirmed as positive.

f – Salmonella spp: The 4 false negatives compared to the culture reference method were tested by bidirectional sequencing, and 1 of 4 could not be confirmed as positives. The 8 false positive samples compared to the culture reference method were tested by bidirectional sequencing and 6 of 8 confirmed as positives.

g - Shigella/EIEC: The 1 false negative compared to the culture reference method was tested by bidirectional sequencing and could not be confirmed as positive. The 11 false positive samples compared to the culture reference method were tested by bidirectional sequencing, and 10 of 11 confirmed as positives.

h - Vibrio parahaemolyticus: The 2 false positives compared to the culture reference method were tested by bidirectional sequencing and, 1 of 2 confirmed as positive.

- Yersinia enterocolitica: The 5 false positives compared to the culture reference method were tested by bidirectional sequencing and, 3 of 5 confirmed as positive.

$\mathrm { j - }$ Giardia lamblia: The 4 false positives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing: none were confirmed as positives.

$\mathbf { k }$ - Adenovirus 40/41: The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. The 3 false positives were not confirmed as positives by an additional round of sequencing.

Table 32. Summary of Clinical Study Results (Prospective specimens) for Native CaryBlair Samples.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp. a</td><td rowspan=1 colspan=1>2/3 (66.7)</td><td rowspan=1 colspan=1>(9.4 - 99.2)</td><td rowspan=1 colspan=1>347/358 (96.9)</td><td rowspan=1 colspan=1>(94.6 - 98.5)</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile b</td><td rowspan=1 colspan=1>37/38 (97.4)</td><td rowspan=1 colspan=1>(86.2 - 99.9)</td><td rowspan=1 colspan=1>318/322 (98.8)</td><td rowspan=1 colspan=1>(96.9 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157 c</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>359/361 (99.5)</td><td rowspan=1 colspan=1>(98.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>EAEC d</td><td rowspan=1 colspan=1>17/18 (94.4)</td><td rowspan=1 colspan=1>(72.71 - 99.9)</td><td rowspan=1 colspan=1>336/341 (98.5)</td><td rowspan=1 colspan=1>(96.6 - 99.5)</td></tr><tr><td rowspan=1 colspan=1>ETEC </td><td rowspan=1 colspan=1>13/14 (92.9)</td><td rowspan=1 colspan=1>(66.13 - 99.8)</td><td rowspan=1 colspan=1>343/345 (99.4)</td><td rowspan=1 colspan=1>(97.9 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>STEC f</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>359/361 (99.5)</td><td rowspan=1 colspan=1>(98.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp g</td><td rowspan=1 colspan=1>4/5 (80.0)</td><td rowspan=1 colspan=1>(28.36 - 99.5)</td><td rowspan=1 colspan=1>354/356 (99.4)</td><td rowspan=1 colspan=1>(98.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC h</td><td rowspan=1 colspan=1>1/2 (50.0)</td><td rowspan=1 colspan=1>(1.26 - 98.7)</td><td rowspan=1 colspan=1>356/359 (99.2)</td><td rowspan=1 colspan=1>(97.6 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>361/361 (100.0)</td><td rowspan=1 colspan=1>(99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>361/361 (100.0)</td><td rowspan=1 colspan=1>(99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>357/361 (98.9)</td><td rowspan=1 colspan=1>(97.2 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp </td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.24 - 100.0)</td><td rowspan=1 colspan=1>354/356 (99.4)</td><td rowspan=1 colspan=1>(98.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>359/359 (100.0)</td><td rowspan=1 colspan=1>(99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia *</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50 - 100.0)</td><td rowspan=1 colspan=1>357/358 (99.7)</td><td rowspan=1 colspan=1>(98.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>359/359 (100.0)</td><td rowspan=1 colspan=1>(99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>6/7 (85.7)</td><td rowspan=1 colspan=1>(42.13 - 99.6)</td><td rowspan=1 colspan=1>354/354 (100.0)</td><td rowspan=1 colspan=1>(99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.50 - 100.0)</td><td rowspan=1 colspan=1>360/360 (100.0)</td><td rowspan=1 colspan=1>(98.98 - 100.0)</td></tr></table>

a – Campylobacter spp. The 1 false negative compared to reference culture method was tested by bidirectional sequencing and confirmed positive. The 11 false positives compared to reference culture method were tested by bidirectional sequencing, and 11 of 11 confirmed as positives.

b – Clostridium difficile. The 1 false negative compared to the FDA cleared NAAT reference method produced high Ct (35). c - E. coli O157. The 2 false positives compared to reference culture method were tested by bidirectional sequencing, and 2 of 2 confirmed as positives.

d – EAEC. The 1 false negative compared to bidirectional sequencing was tested by 2 additional rounds of sequencing and confirmed as positive. The 4 of 5 false positives were not detected by an addition round of sequencing.

e – ETEC. The 1 false negative compared to bidirectional sequencing was tested by 2 additional rounds of sequencing and was not confirmed as positive. 1 of 2 false positives was confirmed as positive by 2 additional rounds of sequencing.

f – STEC. The 2 false positives compared to reference culture method were tested by bidirectional sequencing, and 2 of 2 confirmed as positives.

g – Salmonella spp. The 1 false negative compared to the reference culture method was tested by bidirectional sequencing and confirmed as positive. The 2 false positives compared to reference culture method were tested by bidirectional sequencing and 1 of 2 confirmed as positive.

h – Shigella/EIEC. The 1 false negative compared to the reference culture method was tested by bidirectional sequencing and could not be confirmed as positive. The 3 false positives compared to reference culture method were tested by bidirectional sequencing, and all 3 confirmed as positives.

i - Yersinia enterocolitica. The 4 false positives compared to the reference culture method were tested by bidirectional sequencing, and none were confirmed as positive.

j – Cryptosporidium spp. The 2 false positives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing and both confirmed as positives.

k – Giardia lamblia. The 1 false positive compared to bidirectional sequencing was not confirmed as positive by 2 additional rounds of sequencing.

l - Norovirus GI/GII. The 1 false negative compared to bidirectional sequencing produced a high Ct (37) which indicates that this sample is low positive.

# Testing of inoculated Cary-Blair specimens from previously frozen prospective specimens

To supplement the number of prospective Cary-Blair specimens, 400 unpreserved stool samples from sites 1 and 2 were thawed and inoculated into Cary-Blair. Three were removed from the study for improper storage prior to testing. Two were invalid for RNA IC failure in the unpreserved stool. The following table summarizes the performance of BioCode Gastrointestinal Pathogen Panel (GPP) for these inoculated Cary-Blair specimens with performance calculated based on a comparison to reference/comparator methods.

Table 33. Summary of Inoculated Cary-Blair specimens versus reference/comparator methods.   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>(n)</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp. a</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.8 -100.0)</td><td rowspan=1 colspan=1>388/394(98.5)</td><td rowspan=1 colspan=1>(96.7 -99.4</td></tr><tr><td rowspan=1 colspan=1>E. coli O157 b</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1/2 (50.0)</td><td rowspan=1 colspan=1>(1.3 - 98.7)</td><td rowspan=1 colspan=1>391/395(99.0)</td><td rowspan=1 colspan=1>(97.4 -99.7)</td></tr><tr><td rowspan=1 colspan=1>Enteroaggregative E. coli(EAEC) c</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>12/14(85.7)</td><td rowspan=1 colspan=1>(57.2 -98.2)</td><td rowspan=1 colspan=1>382/382(100.0)</td><td rowspan=1 colspan=1>(99.0 -100.0)</td></tr><tr><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC) d</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>4/6 (66.7)</td><td rowspan=1 colspan=1>(22.3 -95.7)</td><td rowspan=1 colspan=1>386/390(99.0)</td><td rowspan=1 colspan=1>(97.4-99.7)</td></tr><tr><td rowspan=1 colspan=1>Shiga toxin-producing E. coli(STEC)</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.8 -100.0)</td><td rowspan=1 colspan=1>361/361(100.0)</td><td rowspan=1 colspan=1>(99.0 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp e</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>6/6 (100.0)</td><td rowspan=1 colspan=1>(54.1-100.0)</td><td rowspan=1 colspan=1>389/391(99.5)</td><td rowspan=1 colspan=1>(98.2 -99.9</td></tr><tr><td rowspan=1 colspan=1>Shigella/ EIECf</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>1/1 (100.0)</td><td rowspan=1 colspan=1>(2.5-100.0)</td><td rowspan=1 colspan=1>391/396(98.7)</td><td rowspan=1 colspan=1>(97.1-99.6)</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>397/397(100.0)</td><td rowspan=1 colspan=1>(99.1 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp. (notparahaemolyticus)</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>397/397(100.0)</td><td rowspan=1 colspan=1>(99.1-100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica g</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>396/397(99.8)</td><td rowspan=1 colspan=1>(98.6 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/2 (100.0)</td><td rowspan=1 colspan=1>(15.8 -100.0)</td><td rowspan=1 colspan=1>394/394(100.0)</td><td rowspan=1 colspan=1>(99.1 -100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>396/396(100.0)</td><td rowspan=1 colspan=1>(99.1-100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia h</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>394/396(99.5)</td><td rowspan=1 colspan=1>(98.2- 99.9)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41i</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>2/6 (33.3)</td><td rowspan=1 colspan=1>(4.3 - 77.7)</td><td rowspan=1 colspan=1>385/390(98.7)</td><td rowspan=1 colspan=1>(97.0 -99.6)</td></tr><tr><td rowspan=1 colspan=1>Norovirus (GI/GII)</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>28/28(100.0)</td><td rowspan=1 colspan=1>(87.7-100.0)</td><td rowspan=1 colspan=1>364/369(98.6)</td><td rowspan=1 colspan=1>(96.9 -99.6)</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>Cary-Blair(Inoculated)</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>11/12(91.7)</td><td rowspan=1 colspan=1>(61.5 -99.8)</td><td rowspan=1 colspan=1>380/385(98.7)</td><td rowspan=1 colspan=1>(97.0 -99.6)</td></tr></table>

a – Campylobacter spp. The 7 false positives compared to the reference culture method were tested by bidirectional sequencing and 4 of 7 confirmed as positives.

b - E. coli O157. The one false negative compared to the reference culture method was tested by bidirectional sequencing and could not be confirmed as positive. The 4 false positives compared to reference culture method were tested by bidirectional sequencing and 3 of 4 confirmed as positives.

c - EAEC. The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; 1 of th 2 confirmed as positive. The 2 false positives compared to bidirectional sequencing were not confirmed as positive by 2 additional rounds of sequencing.

d - ETEC. The 2 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none were confirmed as positives. The 1 false positive compared to bidirectional sequencing was not available for confirmation testing.

e – Salmonella spp. The one false negative compared to the reference culture method was tested by bidirectional sequencing and confirmed as positive. The 4 false positives compared to the reference culture method were tested by bidirectional sequencing and 2 of 4 confirmed as positives.

f - Shigella/EIEC. The 5 false positives compared to the reference culture method were tested by bidirectional sequencing and 4 of 5 confirmed as positives.

g - Yersinia enterocolitica. The 1 false positive compared to the reference culture method were tested by bidirectional sequencing and confirmed as positive.

h - Giardia lamblia. The 3 false positives compared to bidirectional sequencing were not confirmed as positive by 2 additional rounds of sequencing.

i – Adenovirus 40/41. The 3 false negatives compared to bidirectional sequencing were tested by 2 additional rounds of sequencing; none confirmed as positive.

# Testing of Pre-selected Archived Specimens (Category III)

Several analytes were not encountered or had low prevalence in the clinical study. To supplement the results of the prospective clinical study, 260 preselected archived specimens were assayed. These were archived clinical specimens that were previously tested positive by different methods. Prior to testing with the Applied BioCode Gastrointestinal Pathogen Panel (GPP), the presence of the expected analyte was verified in each specimen using analyte-specific PCR followed by bi-directional sequencing performed at Applied BioCode, Inc. The specimens were randomized with negative specimens, such that the users performing the BioCode Gastrointestinal Pathogen Panel (GPP) were blinded to the expected test result. A summary of results of the BioCode Gastrointestinal Pathogen Panel (GPP) testing are presented in the tables below.

Table 34. Demographics of Pre-selected Archived Specimens   

<table><tr><td>Prospective Study Specimens</td><td></td></tr><tr><td>Total Specimens</td><td>260</td></tr><tr><td>Gender (n/N(%))</td><td></td></tr><tr><td>Female</td><td>123/260 (47.3)</td></tr><tr><td>Male</td><td>137/260 (52.7)</td></tr><tr><td>Age Category (n/N(%)) &lt; 5 year</td><td>54/260 (20.8)</td></tr><tr><td>6-21 yrs</td><td>46/260 (17.7)</td></tr><tr><td>22-59 yrs</td><td>123/260 (47.3)</td></tr><tr><td>60+yrs</td><td>37/260 (14.2)</td></tr></table>

Table 35. Summary of Clinical Specimen Results (Archived specimens)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreementn/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Campylobacter spp.</td><td rowspan=1 colspan=1>38/40 (95.0)</td><td rowspan=1 colspan=1>(83.1 - 99.4)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>E.coli 0157</td><td rowspan=1 colspan=1>19/19 (100.0)</td><td rowspan=1 colspan=1>(82.4 - 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>20/20 (100.0)</td><td rowspan=1 colspan=1>(83.2 - 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>30/33 (90.9)</td><td rowspan=1 colspan=1>(75.7 - 98.1)</td><td rowspan=1 colspan=1>152/152(100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>29/30 (96.7)</td><td rowspan=1 colspan=1>(82.8 - 99.9)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Shigella/ EIEC</td><td rowspan=1 colspan=1>43/45 (95.6)</td><td rowspan=1 colspan=1>(84.9 - 99.5)</td><td rowspan=1 colspan=1>151/152 (99.3)</td><td rowspan=1 colspan=1>(96.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>3/3 (100.0)</td><td rowspan=1 colspan=1>(29.2 - 100.0)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp.</td><td rowspan=1 colspan=1>16/19 (84.2)</td><td rowspan=1 colspan=1>(60.4 - 96.6)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>25/26 (96.2)</td><td rowspan=1 colspan=1>(83.2 - 99.9)</td><td rowspan=1 colspan=1>152/152 (100.0)</td><td rowspan=1 colspan=1>(97.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 40/41</td><td rowspan=1 colspan=1>26/26 (100.0)</td><td rowspan=1 colspan=1>(86.8 - 100.0)</td><td rowspan=1 colspan=1>151/152 (99.3)</td><td rowspan=1 colspan=1>(96.4 - 100.0)</td></tr></table>

# Testing of Contrived Specimens (Category IV)

For some analytes both prospective and archived testing were insufficient to establish performance. To supplement the prospective and archived data, contrived specimens were assayed. The contrived specimens were positive for Giardia, E. histolytica, Yersinia enterocolitica, Vibrio parahaemolyticus, and Vibrio spp. These contrived clinical specimens were prepared using individual clinical specimens that had

previously tested negative for all GI panel analytes. Specimens were spiked at levels of up to 3X LOD $( \sim 5 0 \%$ of contrived specimens) or greater using multiple strains for each organism. Positive samples of each were prepared and randomized by mixing with negative samples before testing. A total of 612 samples, 485 positives, were tested. The results of the BioCode Gastrointestinal Pathogen Panel (GPP) testing are presented in the table below.

Table 36. Summary of contrived specimen results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AgreementRaten/N (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Agreement Raten/N (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>88/96 (91.7)</td><td rowspan=1 colspan=1>(84.2, 96.3)</td><td rowspan=1 colspan=1>516/516 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio spp. (not parahaemolyticus)</td><td rowspan=1 colspan=1>82/94 (87.2)</td><td rowspan=1 colspan=1>(78.8, 93.2)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>40/47 (85.1)</td><td rowspan=1 colspan=1>(72.3, 92.6)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Vibrio vulnificus</td><td rowspan=1 colspan=1>42/47 (89.4)</td><td rowspan=1 colspan=1>(77.4, 95.4)</td><td rowspan=1 colspan=1>518/518 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>95/98 (96.9)</td><td rowspan=1 colspan=1>(91.3,99.4)</td><td rowspan=1 colspan=1>514/514 (100.0)</td><td rowspan=1 colspan=1>(99.3, 100.0)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>96/99 (97.1)</td><td rowspan=1 colspan=1>(91.4, 99.4)</td><td rowspan=1 colspan=1>507/513 (98.8)</td><td rowspan=1 colspan=1>(97.5, 99.6)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>94/98 (95.9)</td><td rowspan=1 colspan=1>(89.9, 98.9)</td><td rowspan=1 colspan=1>513/514 (99.8)</td><td rowspan=1 colspan=1>(98.9, 100.0)</td></tr></table>

# Clinical Specificity – Microbial Detection in Asymptomatic Volunteers

In order to investigate baseline levels for each analyte included in the BioCode Gastrointestinal Pathogen Panel (GPP) in individuals who are not exhibiting signs and symptoms of infectious gastroenteritis, 125 clinical stool samples were collected from healthy asymptomatic donors. These are defined as donors not exhibiting signs and symptoms, or on antibiotics (for symptoms) during the previous 30 days. Asymptomatic donors from two sites and various age groups were included in this study and the demographic information for the donors is shown in the table below. PCR inhibition, as determined by results of the assay internal control (bacteriophage MS2), was observed for one sample $( 0 . 8 \% )$ . After re-running this sample in accordance with the package insert instructions for use, inhibition was still observed, so no result was reported. Therefore, the final data analysis was conducted on 124 of 125 samples collected for this study. A total of 26 samples were positive. The results are summarized in the tables below.

Table 37. Demographic information for Asymptomatic Volunteers.   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>58</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1-5</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>22-65</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>35</td></tr></table>

Table 38. Detections in Asymptomatic Volunteers-Stratified by Age   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>&lt;5 yrs</td><td rowspan=1 colspan=1>6-21 yrs</td><td rowspan=1 colspan=1>22-59 yrs</td><td rowspan=1 colspan=1>60+ yrs</td></tr><tr><td rowspan=1 colspan=1>All Negative</td><td rowspan=1 colspan=1>1 (100.0%)</td><td rowspan=1 colspan=1>3 (100.0%)</td><td rowspan=1 colspan=1>47 (77.1%)</td><td rowspan=1 colspan=1>46 (76.7%)</td></tr><tr><td rowspan=1 colspan=1>Single Infection</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>13 (21.3%)</td><td rowspan=1 colspan=1>14 (23.3%)</td></tr><tr><td rowspan=1 colspan=1>Co-Infections</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (1.6%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Clostridiumdifficile</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>9 (14.8%)</td><td rowspan=1 colspan=1>11 (18.3%)</td></tr><tr><td rowspan=1 colspan=1>EAEC</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (1.7%)</td></tr><tr><td rowspan=1 colspan=1>ETEC</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>2 (3.3%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (1.7%)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (1.6%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (1.7%)</td></tr></table>

# N. Instrument Name:

BioCode Gastrointestinal Pathogen Panel (GPP)

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

No

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes

3. Specimen Identification: Specimen identity is provided by barcode magnetic beads.

4. Specimen Sampling and Handling:

After extraction with the easyMag system and loading samples into a 96well formatted plate, the BioCode MDx 3000 processes all RT-PCR steps automatically.

5. Calibration: Optical calibration of the MDx 3000 is performed twice a year by Applied BioCode. No calibration kit is available.

6. Quality Control:

Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations and good laboratory practices. The BioCode Gastrointestinal Pathogen Panel (GPP) uses an internal control (bacteriophage MS2) which is added to each sample during pre- treatment. The internal control monitors the efficiency of the extraction, reverse transcription, amplification and detection stages of the assay. Positive results may be reported in the absence of RNA IC detection. The BioCode Gastrointestinal Pathogen Panel (GPP) software will suppress negative results for any wells with invalid RNA IC results.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

Samples suspected of C. difficile presence should only be tested fresh. The instrument software has been updated to select for fresh or frozen samples and omit results of $C$ . difficile.

The submitted information in this premarket notification is sufficient to support a substantial equivalence decision for the BioCode Gastrointestinal Pathogen Panel (GPP).